<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic mapping and manipulation: Chapter 7-Making compound mutants
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMgmmcmch7_ArticleID" title="Genetic mapping and manipulation: Chapter 7-Making compound mutants&#xA;">
      <link rel="next" href="#d0e46" title="1.&nbsp;Background">
      <script language="javascript" type="text/javascript" src="/js/history_list.js"></script>
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="makingcompdmutants.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMgmmcmch7_ArticleID"></a>Genetic mapping and manipulation: Chapter 7-Making compound mutants<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">David</span> <span class="surname">Fay<em><span class="remark">
                                 					<sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Department of Molecular Biology, University of Wyoming, Laramie, Wyoming 82071-3944 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward genetics and genetic mapping<br />>>> Genetic mapping and manipulation
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=293.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Genetic+mapping+and+manipulation:+Chapter+7-Making+compound+mutants" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

          <p><a style="color:blue;cursor:pointer" onclick="return kadabra('history');" title="click to expand ..." >Publication History</a></p>
           <ul id="history" style="display:none">
            <li><a href="/chapters/www_makingcompdmutants/makingcompdmutants.html">version 1</a></li>
            <li><a href="/chapters/www_makingcompdmutants.2/makingcompdmutants.html">latest version</a></li>
           </ul>

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e46">1. Background</a></span></dt>
               <dt><span class="sect1"><a href="#d0e60">2. Making double mutants</a></span></dt>
               <dt><span class="sect1"><a href="#d0e216">3. Using the power of counter-selection</a></span></dt>
               <dt><span class="sect1"><a href="#d0e320">4. Crossing mutants into reporter lines</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e46"></a>1.&nbsp;Background
                     </h2>
                  </div>
               </div>
            </div>
            <p>One of the most important genetic skills is the ability to generate double- and triple-mutant strains for phenotypic and genetic
               analysis. These compound mutants can be used for <a href="http://www.wormbook.org/chapters/www_epistasis.2/epistasis.html" target="_blank">epistasis</a> experiments for ordering genes within a genetic pathway or for identifying genetic interactions such as <a href="http://www.wormbook.org/chapters/www_suppressormutations/suppressormutations.html" target="_blank">Suppressor mutations</a> or <a href="http://www.wormbook.org/chapters/www_suppressormutations/suppressormutations.html" target="_blank">Synthetic and enhancer mutations</a> of a given phenotype. Outlined below are a few of the basic methods used for this purpose.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e60"></a>2.&nbsp;Making double mutants
                     </h2>
                  </div>
               </div>
            </div>
            <p>In the simplest case, two mutations (<span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span>) are unlinked, viable, and associated with an easily discernable plate phenotype. In this situation, it is a straightforward
               matter to cross one of the two strains to N2 males (or to generate a homozygous male stock from one of the mutant strains),
               and then cross the resultant heterozygous males to the second mutation to generate the <span class="emphasis"><em><span class="bold"><strong>a/+; b/+</strong></span></em></span> trans-heterozygote. <span class="bold"><strong>A B</strong></span> animals (1/16) can then easily be isolated in the next generation (<a href="#figure1">Figure 1</a>; strategy #1). In cases where <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> mutations do not produce an obvious phenotype or perhaps produce highly similar phenotypes, it may be highly advantageous
               to have one or both mutations linked to a nearby genetic marker (e.g., <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span>). In this case, <span class="emphasis"><em><span class="bold"><strong>a unc/+</strong></span></em></span> males might be crossed to <span class="emphasis"><em><span class="bold"><strong>dpy b</strong></span></em></span> hermaphrodites to generate <span class="emphasis"><em><span class="bold"><strong>a unc/+; dpy b/+</strong></span></em></span> trans-heterozygotes. Following self-fertilization, <span class="bold"><strong>Dpy Unc</strong></span> progeny of the desired phenotype (<span class="emphasis"><em><span class="bold"><strong>a unc; dpy b</strong></span></em></span>) can be isolated (<a href="#figure1">Figure 1</a>; strategy #2). It is important to keep in mind that when using linked markers to follow mutations of interest, the distance
               between the mutation and the visible marker will strongly affect the reliability of the approach. Namely, the farther apart
               the mutation is from the visible marker, the more likely that a recombination event might lead to the mutant allele becoming
               separated from the marker. Thus it is always essential to generate several independent lines and to confirm the correct genotype
               either by direct sequencing of the mutant locus or by obtaining consistent results for multiple independent isolates.
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="gmmcmch7-1.jpg"><img src="gmmcmch7-1_s.jpg" border="2" align="middle" alt="Making compound mutants Figure 7-1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1.&nbsp;</b></p>
               </div>
            </div>
            <p>The real difficulty with creating compound mutants comes when two of the mutations are on the same chromosome. In this situation,
               one can either take a passive or active approach. An example of the passive approach is shown in <a href="#figure2">Figure 2</a>. In essence, you create a trans-heterozygote strain where the mutations of interest are balanced by each other. Then by blindly
               picking sufficient animals of either mutant genotype, you can hope to isolate a low percentage of animals that throw the double-mutant
               chromosome as a result of a crossover event. This can be done either with or without the use of linked markers, depending
               on the phenotype of the individual mutants. This approach is quite effective if the two mutations aren't too closely linked
               and has the advantage of not necessarily requiring any special strain construction beforehand.
            </p>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="gmmcmch7-2.jpg"><img src="gmmcmch7-2_s.jpg" border="2" align="middle" alt="Making compound mutants Figure 7-2"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 2.&nbsp;</b></p>
               </div>
            </div>
            <p>Alternatively, if the two mutations are very closely linked, you may find it difficult or impossible to isolate double-mutant
               animals of the desired genotype. Here it will be best to take an active approach. For this to be feasible, however, it will
               require a bit of prior strain construction. Fortunately, if you have previously generated a strain containing your mutation
               flanked by two visible markers (e.g., for SNP mapping), you should be in a good position to begin your crosses. In this case,
               you will want to carry out crosses to generate a strain such as <span class="emphasis"><em><span class="bold"><strong>unc a dpy/b</strong></span></em></span>, where <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> are the mutations you wish to link in cis and <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span> are visible markers that flank <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> (<a href="#figure3">Figure 3</a>). Depending on the specific locations of the various mutations relative to each other, you will want to isolate either <span class="bold"><strong>Unc-A</strong></span> (<span class="bold"><strong>non-Dpy</strong></span>) or <span class="bold"><strong>A Dpy (non-Unc)</strong></span> recombinant animals and then look for the presence of <span class="bold"><strong>Unc A B</strong></span> or <span class="bold"><strong>B A Dpy</strong></span> progeny in the next generation. This approach is best accomplished if <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> lies outside the region encompassed by <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span>; however, even if <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> is internal, strains of the desired genotype should almost always be obtainable if sufficient recombinants are analyzed.
            </p>
            <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="gmmcmch7-3.jpg"><img src="gmmcmch7-3_s.jpg" border="2" align="middle" alt="Making compound mutants Figure 7-3"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 3.&nbsp;</b></p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e216"></a>3.&nbsp;Using the power of counter-selection
                     </h2>
                  </div>
               </div>
            </div>
            <p>Another very useful trick when working with mutations that are difficult to track based on their plate phenotype is the use
               of the counter-selectable marker. There are many variations to this theme, two of which are outlined in <a href="#figure4">Figure 4</a>. In both cases, a <span class="emphasis"><em><span class="bold"><strong>c/c; m/m</strong></span></em></span> double mutant is sought. In case #1, <span class="emphasis"><em><span class="bold"><strong>m/m</strong></span></em></span> animals display the phenotype <span class="bold"><strong>M</strong></span>, whereas <span class="emphasis"><em><span class="bold"><strong>c/c</strong></span></em></span> mutants fail to display an obvious phenotype. Homozygous <span class="emphasis"><em><span class="bold"><strong>c/c</strong></span></em></span> males are crossed to <span class="emphasis"><em><span class="bold"><strong>m/m; unc/unc</strong></span></em></span> hermaphrodites, where the <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutation is located on the same chromosome and is preferably close to the genetic position of <span class="emphasis"><em><span class="bold"><strong>c</strong></span></em></span>. In subsequent generations, the absence of <span class="bold"><strong>Unc</strong></span> animals can be used to infer that <span class="emphasis"><em><span class="bold"><strong>c</strong></span></em></span> is probably homozygous. In case #2, <span class="emphasis"><em><span class="bold"><strong>m/m</strong></span></em></span> animals display no obvious defects, whereas <span class="emphasis"><em><span class="bold"><strong>c/c</strong></span></em></span> animals display the <span class="bold"><strong>C</strong></span> phenotype. Here, N2 males are first crossed to the <span class="bold"><strong>Unc</strong></span> strain to generate heterozygous <span class="emphasis"><em><span class="bold"><strong>unc/+</strong></span></em></span> males, which are subsequently crossed to c/c hermaphrodites. Males resulting from this latter cross (50% of which will be
               <span class="emphasis"><em><span class="bold"><strong>unc/+; c/+</strong></span></em></span>) are then crossed to <span class="emphasis"><em><span class="bold"><strong>m/m</strong></span></em></span> hermaphrodites, and resultant cross progeny of the genotype <span class="emphasis"><em><span class="bold"><strong>c/+; unc/m</strong></span></em></span> are identified. Finally, <span class="bold"><strong>C</strong></span>   animals that fail to segregate <span class="bold"><strong>Unc</strong></span>  progeny are identified indicating the desired genotype. We also note that in addition to standard genetic markers, balancer
               chromosomes containing integrated <span class="bold"><strong>GFP-expressing arrays</strong></span> can be exceptionally useful for counter selection. In fact, the dominant effect of the GFP means that homozygous <span class="emphasis"><em><span class="bold"><strong>c/c</strong></span></em></span> animals (to site the above case) can be identified directly from the progeny of <span class="emphasis"><em><span class="bold"><strong>c/GFP</strong></span></em></span> heterozygotes.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="gmmcmch7-4.jpg"><img src="gmmcmch7-4_s.jpg" border="2" align="middle" alt="Making compound mutants Figure 7-4"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 4.&nbsp;</b></p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e320"></a>4.&nbsp;Crossing mutants into reporter lines
                     </h2>
                  </div>
               </div>
            </div>
            <p>All the principles described above will apply to the generation of mutant strains containing various integrated GFP or lacZ
               reporters. In the case of GFP, the dominance of the phenotype (green worms) makes these experiments quite straightforward
               to conduct. In fact, in many cases one can distinguish directly GFP/GFP and GFP/+ animals, as the former fluoresce more brightly
               than their heterozygous counterparts. One issue that you may encounter is that a number of integrated GFP lines also contain
               the dominant <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a></strong></span></em></span> marker. This may influence the way in which you design your crosses, as rolling males rarely mate well.
            </p>
            <p>Another issue with integrated reporters is their site of insertion into the genome. Namely, if they reside on the same chromosome
               as your mutation of interest, it may in some cases be quite difficult to generate homozygous strains containing your mutation
               in cis to the integrated reporter. This problem is further exacerbated by the ability of many multi-copy integrated arrays
               to suppress recombination in the vicinity of their insertion site (usually to several map units), thereby rendering the construction
               of certain strains virtually impossible. For some integrated arrays, the insertion site may have been previously mapped to
               a single chromosome, although the precise location of the integration is often not known. Thus, it is typically required that
               one empirically determine whether or not the desired strain is obtainable by setting up the necessary crosses and following
               them through. As described above, it may be advantageous in certain situations to screen for recombinants (using visible markers
               linked to your mutation of interest) that will enable you to select for candidate strains harboring your mutation in cis to
               the reporter insertion. Alternatively, one can sometimes obtain strains containing the same reporter integrated at a less
               problematic location. In any case, one should always check to see if the integration site is known and proceed accordingly.
            </p>
            <p>In some situations, extra-chromosomal arrays containing the reporter of interest may provide an acceptable alternative to
               the use of integrated arrays. These arrays can either be mated into the mutation of interest or created directly in the mutant
               background by injection methods. The obvious disadvantage to using extra-chromosomal arrays for phenotypic studies is their
               tendency towards somatic mosaicism, which can complicate your analysis. When using extrachromosomal arrays, make certain to
               perform your analysis with multiple independent lines using the appropriate controls to ensure consistency in your results.
            </p>
            <p>One phenomenon to be aware of if you are mating an extra-chromosomal array into your mutant background via a male concerns
               the behavior of most extrachromosomal (and even some integrated) arrays during male meiosis. Namely, the arrays tend to pair
               and subsequently disjoin from the lone X chromosome, leading to sperm that contain either the X <span class="emphasis"><em>or</em></span> the array, but not both. The functional consequence of this property is that the large majority of array-containing cross
               progeny tend to be male (~80%), which are not particularly useful if the next step in your strategy is to self the array-containing
               animals. Thus, it may be prudent to set up additional mating plates in these situations to ensure that sufficient array-positive
               hermaphrodites are obtained.
            </p>

         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised April 5, 2006. Published June 14, 2006. This chapter should be cited as: Fay, D. Genetic
                  mapping and manipulation: Chapter 7-Making compound mutants (June 14, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.96.2, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 David Fay. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:davidfay@uwyo.edu" target="_top">davidfay@uwyo.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

