<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Germline genomics
         
         
         
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#Germlinegenomics_ArticleID" title="Germline genomics&#xA;                &#xA;            &#xA;        ">
      <link rel="next" href="#d0e63" title="1.&nbsp;Regulatory mechanisms of germline gene expression">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="germlinegenomics.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="Germlinegenomics_ArticleID"></a>Germline genomics<em><span class="remark">
                                           <sup><a name="d0e10" href="#ftn.d0e10">*</a></sup>
                                       </span></em>
                             
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Valerie</span> <span class="surname">Reinke<em><span class="remark">
                                                     <sup><a name="d0e38" href="#ftn.d0e38">&sect;</a></sup>
                                                 </span></em>
                                       </span><br></span><span class="affiliation"><span class="address">Department of Genetics, Yale University, New Haven, CT 06520 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_germline.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > The germ line
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=260.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Germline+genomics" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e63">1. Regulatory mechanisms of germline gene expression</a></span></dt>
               <dt><span class="sect1"><a href="#d0e76">2. Gene expression profiling of the germ line</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e79">2.1. DNA microarray technology</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e122">2.2. Identification of germline-enriched genes in adult hermaphrodites</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e253">2.3. Analysis of germline gene expression during development</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e379">3. Chromosome bias for genes with germline-enriched expression</a></span></dt>
               <dt><span class="sect1"><a href="#d0e446">4. Integration of germline functional genomic data</a></span></dt>
               <dt><span class="sect1"><a href="#d0e502">5. Future directions</a></span></dt>
               <dt><span class="sect1"><a href="#d0e553">6. Useful links for germline genomics</a></span></dt>
               <dt><span class="sect1"><a href="#d0e573">7. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>The completion of the</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>genome sequence permits the
                     comprehensive examination of the expression and function of genes. Annotation of
                     virtually every encoded gene in the genome allows systematic analysis of those genes
                                 using high-throughput assays, such as microarrays and RNAi. This chapter will center on
                                 the use of microarrays to comprehensively identify genes with enriched expression in the
                                 germ line during development. This knowledge provides a database for further studies
                                 that focus on gene function during germline development or early embryogenesis.
                                 Additionally, a comprehensive overview of germline gene expression can uncover striking
                                 biases in how genes expressed in the germ line are distributed in the genome, leading to
                                 new discoveries of global regulatory mechanisms in the germ line.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e63"></a>1.&nbsp;Regulatory mechanisms of germline gene expression
                     </h2>
                  </div>
               </div>
            </div>
            <p>Regulation of gene expression at both transcriptional and post-transcriptional levels
                           is crucial for the proper specification, proliferation and differentiation of germ
                           cells. At the transcriptional level, certain genes are effectively silenced while
                           other genes are expressed (see <a href="http://www.wormbook.org/chapters/www_specgermline/specgermline.html" target="_blank">Specification of the germ line</a> and <a href="http://www.wormbook.org/chapters/www_germlinechromatin/germlinechromatin.html" target="_blank">Germline chromatin</a>). After transcript production, complex networks of RNA regulatory proteins
                           modulate either the stability and/or translation of mRNAs. These processes combine to effectively control, both
               spatially and
                           temporally, the levels of germline-expressed gene products. Our understanding of these
                           mechanisms in the germ line is still murky. How are genes selected for silencing or
                           expression? How does germline gene expression influence the organization of genes in the
                           genome, and vice versa? How are the dramatic structural changes of meiotic chromosomes
                           coordinated with continued gene expression? What mechanisms control spatial and temporal
                           expression within the germ line?
            </p>
            <p>For those wishing to understand these aspects of germline development, one goal is to
                           identify every germline-expressed gene. With such information in hand, it is possible to
                           analyze gene functions by RNAi and decipher regulatory mechanisms by a combination of
                           bioinformatics and molecular genetics. Such genome-wide studies reveal global trends,
                           and also identify individual genes that might function to control germ cell fate decisions or
                           terminal differentiation of gametes, making them candidates for future functional study.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e76"></a>2.&nbsp;Gene expression profiling of the germ line
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e79"></a>2.1.&nbsp;DNA microarray technology
                        </h3>
                     </div>
                  </div>
               </div>
               <p>DNA microarray technology is frequently used to monitor changes in gene expression
                                  for thousands of genes simultaneously (reviewed in <a href="#bib7">Duggan et al., 1999</a>). Typically, DNA microarrays are common laboratory glass slides upon which
                                  is printed a series of discrete spots of DNA. Each spot corresponds to some portion
                                  of a known gene or predicted open reading frame. For each experiment, a comparison
                                  is made between two different populations of transcripts that have been
                                  reverse-transcribed to cDNA, each in the presence of a distinct fluorescent label.
                                  At each spot, competitive hybridization between the two populations occurs. The
                                  relative intensity of the two fluorescent labels at each spot measures the relative
                                  abundance of a given gene transcript between the two populations (<a href="#figure1">Figure 1</a>). When used in this manner, microarrays provide information
                                  about the steady state levels of mRNAs; they do not directly address the rate at
                                  which a transcript is synthesized or degraded. The <span class="emphasis"><em>C. elegans</em></span>
                                  DNA microarrays that primarily have been used to investigate germline gene
                                  expression contain over 18,000 spots and represent about 90<span class="symbol">%</span> of the currently annotated genes in the
                                  genome (<a href="#bib12">Jiang et al., 2001</a>; see <a href="http://www.wormbook.org/chapters/www_DNAmicroarrays/DNAmicroarrays.html" target="_top">DNA microarrays</a>).
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="germlinegenomicsfig1.jpg"><img src="germlinegenomicsfig1_s.jpg" border="2" align="middle" alt=" Example of microarray hybridization figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. Example of microarray hybridization.&nbsp;</b>A representative portion of a microarray shows the differential signals from two RNA
                                    samples. One RNA sample was reverse transcribed into cDNAs labelled with red
                                    fluorophore, the other RNA sample into cDNAs labelled with green fluorophore, and the
                                    cDNA mixture was hybridized to spots of DNA representing different genes. Selective
                                    hybridization of cDNA from either RNA sample to a DNA spot produces red or green signal;
                                    hybridization of cDNA from both RNA samples produces yellow signal. In this example, red
                                    spots represent RNAs enriched in hermaphrodites with wild type germ lines, and green
                                    spots are RNAs enriched in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-4" target="_blank">glp-4</a>(bn2)</em></span> mutants with greatly diminished germ lines.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e122"></a>2.2.&nbsp;Identification of germline-enriched genes in adult hermaphrodites
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The comprehensive identification of germline-expressed genes will comprise a
                                  molecular definition of germline components, and provide the framework to identify
                                  individual genes involved in specific germline functions. Microarrays are an
                                  excellent platform to achieve such a goal because the expression of all genes in the
                                  genome can be examined simultaneously. Microarray analysis of mutants that perturb
                                  specific aspects of germline development has been very useful in defining sets of
                                  genes with germline-enriched expression. Discussed below are three sets of genes
                                  that have been identified to date: genes with enriched expression in the germline
                                  relative to the soma, genes with enriched expression during spermatogenesis, and
                                  genes with enriched expression during oogenesis.
               </p>
               <p>One of the most straightforward comparisons identified genes with
                                  germline-enriched expression relative to somatic tissues (<a href="#bib20">Reinke et al., 2000</a>; <a href="#bib19">Reinke et al., 2004</a>). A temperature-sensitive
                                  mutation, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-4" target="_blank">glp-4</a>(bn2)</em></span>, causes severe defects in germ cell
                                  proliferation, resulting in gonads with <span class="symbol">&#8764;</span>100-fold fewer germ cells than wild type
                                  (<a href="#bib4">Beanan and Strome, 1992</a>). Microarray comparison of either L4 or
                                  young adult wild type hermaphrodites to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-4" target="_blank">glp-4(ts)</a></em></span>
                                  hermaphrodites of the corresponding stage identified 3144 genes with
                                  germline-enriched expression relative to somatic tissues in hermaphrodites (<span class="symbol">&gt;</span>2x, p<span class="symbol">&lt;</span>0.01). When a similar comparison was done
                                  with wild-type and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-4(ts)</a></em></span> males, 1092 genes showed
                                  significantly enriched expression in the germ line. Previous single-gene analyses
                                  had identified <span class="symbol">&#8764;</span>70 genes that are
                                  expressed and/or function in the hermaphrodite germline. Essentially every one of
                                  these genes was correctly identified as germline-enriched in the microarray
                                  experiments, demonstrating the accuracy and comprehensiveness of the microarray
                                  approach. However, it is important to note that a gene with equivalent expression in
                                  both samples, even if expressed in the germline, will not be identified as a
                                  &#8220;germline gene&#8221;. Thus, these studies are limited to identification of genes with
                                  <span class="emphasis"><em>enriched</em></span> expression in the germ line relative to the soma.
               </p>
               <p>While the identification of genes with germline-enriched expression in the two
                                  sexes is informative, the number of genes identified is formidably large. Further
                                  microarray experiments using mutants that affect gametogenesis helped to group these
                                  genes into distinct categories. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>(lf)</em></span> mutant hermaphrodites
                                  are defective in initiation of spermatogenesis and consequently produce only oocytes
                                  (<a href="#bib15">Nelson et al., 1978</a>), while <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a>(gf)</em></span> mutants
                                  are defective in switching from spermatogenesis to oogenesis and consequently
                                  produce only sperm (<a href="#bib1">Barton et al. 1987</a>). Direct comparison of the
                                  genes differentially expressed between <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>(lf)</em></span> and
                                      <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a>(gf)</em></span> populations defined two sets of genes: those
                                  expressed during spermatogenesis (1343) and those expressed during oogenesis (1652)
                                      (<span class="symbol">&gt;</span>2x, p<span class="symbol">&lt;</span>0.01). Genes with high expression during
                                  spermatogenesis are likely to be involved in spermatocyte specification and
                                  differentiation. Genes with high expression during oogenesis are expected to encode
                                  proteins required for oocyte differentiation as well as maternally provided factors
                                  necessary for proper development of the early embryo.
               </p>
               <p>Combination of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-4" target="_blank">glp-4</a>(ts)</em></span> vs wild type comparison and the
                                  <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>(lf)</em></span> vs <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a>(gf)</em></span> comparison
                                  identifies 4245 genes with expression in the germ line and/or gametes &#8211;
                                  approximately 20<span class="symbol">%</span> of all the genes in
                                  the genome (<a href="#figure2">Figure 2</a>). These genes can be divided into three
                                  groups, spermatogenesis (genes in the green circle), oogenesis (genes in the orange
                                  circle), and intrinsic, which includes germline genes not differentially expressed
                                  during gametogenesis (genes in the blue circle but not in the green or orange
                                  circles). Most of the spermatogenesis-enriched genes identified in hermaphrodites
                                  were also enriched in males, as expected.
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="378">
<tr>
  <td align="center"><p><a href="germlinegenomicsfig2.jpg"><img src="germlinegenomicsfig2_s.jpg" border="2" align="middle" alt=" Germline gene sets figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Venn diagram of germline gene sets.&nbsp;</b>The wild type vs. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-4" target="_blank">glp-4</a></em></span> (diminished germ line) data set was combined with the
                                    <span class="emphasis"><em>fem-1(lf)</em></span> (oocytes only) vs. <span class="emphasis"><em>fem-3(gf)</em></span> (sperm only) data set to generate the displayed
                                    set of genes with germline-enriched expression patterns. Reprinted with permission from
                                    <a href="#bib19">Reinke et al., 2004</a>.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e253"></a>2.3.&nbsp;Analysis of germline gene expression during development
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In addition to expression profiles of mutants, temporal expression profiles gathered
                              from wild type animals can also reveal sets of co-regulated genes. Microarrays have been
                              probed with RNAs prepared at consecutive times of development from wild type animals.
                              This timecourse provides high temporal resolution (every 3 hours for 36 hours) of gene
                              expression from mid-L3 through adulthood (<a href="#bib19">Reinke et al., 2004</a>). During
                              this period, major germline events include entry into meiosis, spermatogenesis,
                              oogenesis and embryogenesis. Because intact animals were used for the timecourse
                              analysis, fluctuations in expression of somatic genes would also be detected. Among the
                              major developmental events in the soma during this period are gonadogenesis (i.e.,
                              generation of uterus, spermathecae and sheath cells) and vulval development. Statistical
                              analysis using ANOVA identified 5083 genes with a significant fluctuation in expression
                              level during the timecourse (p<span class="symbol">&lt;</span>0.01);
                              among these, 2426 genes had also been identified as germline-regulated genes.
               </p>
               <p>To identify sets of co-regulated genes, gene expression data can be grouped together
                              by similarity of expression profiles. One method for grouping genes is hierarchical
                              clustering using the Pearson correlation coefficient to determine relatedness of
                              expression (<a href="#bib8">Eisen et al., 1998</a>). In <a href="#figure3">Figure 3</a>,
                              hierarchical clustering was performed on the data from the timecourse microarrays. Not
                              surprisingly, some of these clusters based on common temporal regulation contain
                              particular classes of germline genes, as defined by the analysis of mutants (Clusters D-F in <a href="#figure3">Figure 3</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="germlinegenomicsfig3.jpg"><img src="germlinegenomicsfig3_s.jpg" border="2" align="middle" alt=" Gene expression analysis in larvae and adults figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 3. Analysis of gene expression in larvae and adults .&nbsp;</b>The top diagram shows germline development during the various stages that were
                                    examined. Black = somatic gonad; orange = proliferating germ cells;
                                    pink = meiotic germ cells; red = differentiating spermatocytes; blue = differentiating
                                    oocytes. The clustergrams below show all genes whose expression changed over the
                                    timecourse examined. Each row represents a gene, and, in the clustergram on the left,
                                    each column represents a different timepoint in the analysis. The clustergram on the
                                    right shows the expression levels of the same genes (in the same order) in wild type,
                                    <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-4" target="_blank">glp-4</a>(ts)</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>(lf)</em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a>(gf)</em></span> hermaphrodites, as well as in wild type and
                                    <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-4" target="_blank">glp-4</a>(ts)</em></span> males. Above the clustergrams, small and large "+" indicate small and large
                                    germ line, "&#8211;" indicates diminished germ line in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-4" target="_blank">glp-4</a>(ts)</em></span> mutants, and &#8220;fem&#8221; indicates
                                    the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a>(lf)</em></span> vs <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a>(gf)</em></span> experiment. Yellow indicates higher expression in staged wild
                                    type, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-4" target="_blank">glp-4</a>(ts)</em></span>, or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a>(gf)</em></span> samples; blue indicates higher expression in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>(lf)</em></span>, or
                                    a reference sample that was used for the timecourse. Clusters of genes with distinct
                                    temporal expression profiles are marked by letters. Cluster D contains almost
                                    exclusively spermatogenesis genes, while intrinsic and oogenesis genes are distributed
                                    among clusters E and F. Clusters A-C primarily contain somatic genes whose expression
                                    varies over time. Reprinted with permission from <a href="#bib19">Reinke et al., 2004</a>.
                     </p>
                  </div>
               </div>
               <p>One prominent cluster identified from the timecourse microarrays contains a set of
                              genes with sharp onset of expression in late L3s and abrupt decline in late L4 (cluster
                              D in the temporal clustergram at left in <a href="#figure3">Figure 3</a>). Examination of this set of genes in
                              the microarray analysis performed on mutants (right side of <a href="#figure3">Figure 3</a>) showed that 99<span class="symbol">%</span> are associated with
                              spermatogenesis. Thus, the spermatogenesis genes are regulated in a fashion that is
                              distinct from the oogenesis and intrinsic genes, and the timing of their expression
                              coincides with the duration of spermatogenesis in late L3/L4 larvae. Additionally,
                              spermatogenesis genes encode different classes of proteins, with more signaling
                              molecules and fewer DNA and RNA metabolism factors (<a href="#figure4">Figure 4</a>). Thus,
                              genes with spermatogenesis-enriched expression are clearly distinct from other
                              germline-enriched genes.
               </p>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="germlinegenomicsfig4.jpg"><img src="germlinegenomicsfig4_s.jpg" border="2" align="middle" alt=" Classification of germline enriched genes figure 4"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 4. Classification of germline-enriched gene sets based on predicted functions.&nbsp;</b>Using Gene Ontology categories, the three sets of germline-enriched genes
                                    (spermatogenesis, oogenesis, and intrinsic), were categorized by predicted function of
                                    their encoded protein products. The oogenesis and intrinsic groups are composed of very
                                    similar proportions of similar types of proteins, while the spermatogenesis group is
                                    distinct. The bars represent the different classifications of nucleic acid binding
                                    proteins. Reprinted with permission from <a href="#bib19">Reinke et al., 2004</a>.
                     </p>
                  </div>
               </div>
               <p>Unlike the spermatogenesis-enriched genes, expression of other germline-enriched genes
                              (in clusters E and F) increased over time without any significant decline. Overall, the
                              temporal regulation of these germline genes was very similar, but minor differences
                              sub-divided them into two groups. Expression of cluster F genes gradually increases in
                              L3 or L4, while expression of cluster E genes has a more abrupt onset at adulthood. The
                              cluster E expression profile is consistent with oogenesis, which initiates in the adult.
                              However, genes in the intrinsic and oogenesis sets are fairly evenly distributed between
                              cluster E and cluster F. Furthermore, the classes of protein functions are also
                              proportioned similarly in the two clusters (<a href="#figure4">Figure 4</a>). Thus, separation
                              of germline genes into &#8220;intrinsic&#8221; and &#8220;oogenesis&#8221; categories is not supported by the
                              temporal analysis, and it is difficult at this point to separate genes specifically
                              required in immature germ cells versus maturing oocytes (see <a href="#bib19">Reinke et al., 2004</a> for discussion of possible reasons).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e379"></a>3.&nbsp;Chromosome bias for genes with germline-enriched expression
                     </h2>
                  </div>
               </div>
            </div>
            <p>Genome-scale views can provide much more than gene lists. A prime example is the
                           discovery of a chromosomal bias in the genomic location of germline-enriched genes,
                           which arose from the microarray studies described above. The germline genes are
                           primarily located on the autosomes; very few reside on the X chromosome (<a href="#bib20">Reinke et al., 2000</a>; <a href="#bib19">Reinke et al., 2004</a>). This bias is particularly
                           strong for spermatogenesis genes, and though less marked for other germline genes, the
                           bias is quite significant.
            </p>
            <p>The biased location of germline-enriched genes, in combination with electron
                           microscopic images of germ cell nuclei in males, led to the hypothesis that the X
                           chromosome might be transcriptionally silent in the germ line. By EM analysis of male
                           germlines, the unpaired X chromosome appears condensed (Goldstein, 1992),
                           a morphology now thought to be similar to the condensed, transcriptionally inactive sex
                           chromosome body observed during meiosis in organisms with single or non-equivalent sex
                           chromosomes (<a href="#bib14">McKee and Handel, 1993</a>). If the X chromosome is essentially
                           transcriptionally inactive in the male germline, then it would be an inhospitable
                           environment for genes needed in that tissue. Thus, genes expressed in the germline would
                           ultimately be under-represented on the X chromosome. Alternatively, some unknown force
                           could prevent germline genes from residing on the X, and consequently, their absence
                           from the X triggers its apparent transcriptional silencing.
            </p>
            <p>Subsequent experiments have supported and extended the hypothesis that the X is
                           silenced in the germline. In the male germline, the unpaired X chromosome is decorated
                           with histone modifications that correlate with transcriptional silencing, and is devoid
                           of modifications that correlate with active gene expression (<a href="#bib13">Kelly et al., 2002</a>; see <a href="http://www.wormbook.org/chapters/www_germlinechromatin/germlinechromatin.html" target="_blank">Germline chromatin</a>). Unexpectedly, the synapsed X chromosomes
                           in hermaphrodites are also apparently silenced throughout most of the germ line, as they
                           largely lack detectable activating modifications (<a href="#bib13">Kelly et al., 2002</a>;
                           see ). However, silencing modifications are not seen on the hermaphrodite Xs as they are
                           in the male (<a href="#bib3">Bean et al., 2004</a>; see <a href="http://www.wormbook.org/chapters/www_germlinechromatin/germlinechromatin.html" target="_blank">Germline
                              chromatin</a>). This milder silencing correlates well with the extent of gene bias
                           against the X chromosome, which is less severe for genes expressed primarily in the
                           hermaphrodite germline (oogenesis genes) compared to those shared with the male germline
                           (spermatogenesis genes). The silencing of the Xs in hermaphrodites was subsequently
                           shown to require the activity of the MES proteins (<a href="#bib9">Fong et al., 2002</a>).
                           Mutations in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mes-2" target="_blank">mes-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mes-3" target="_blank">mes-3</a></em></span>, and
                               <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mes-6" target="_blank">mes-6</a> </em></span>genes causes the X to inappropriately acquire marks of
                           active chromatin, suggesting that transcriptional control of the X chromosome is
                           disrupted (<a href="#bib9">Fong et al., 2002</a>; see <a href="http://www.wormbook.org/chapters/www_specgermline/specgermline.html" target="_blank">Specification of the germ line</a>).
            </p>
            <p>The remarkable insight that an entire chromosome is silenced in a major tissue such as
                           the germ line would have been difficult to achieve if the germ line had been studied
                           only on a gene-by-gene basis. As each individual germline-enriched gene was identified,
                           its location on an autosome would not have been surprising. By examining the expression
                           profiles of many germline genes in parallel, the bias against the X chromosome became
                           obvious, and could be further examined through additional experiments. This example
                           provides an important instance of how emergent properties become apparent from
                           comprehensive molecular depictions.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e446"></a>4.&nbsp;Integration of germline functional genomic data
                     </h2>
                  </div>
               </div>
            </div>
            <p>In addition to the microarray approach described above, other large-scale methods of
                           examining gene expression have been undertaken. An <span class="emphasis"><em>in situ</em></span> hybridization project
                           performed under the direction of Dr. Yuji Kohara examined the expression pattern of
                           several thousand genes at multiple stages of development, from embryos to young adults,
                           specifically in hermaphrodites. Because <span class="emphasis"><em>in situ</em></span> hybridization analysis reveals both
                           location and timing of transcript accumulation, it complements the quantitative data
                           provided by microarrays. Together, these two methods provide insights into spatial,
                           temporal, and condition-dependent control of gene expression. The whole-mount <span class="emphasis"><em>in situ</em></span>
                           studies have successfully identified many genes with germline-enriched expression,
                           perhaps because the germ line has a relatively high rate of mRNA production and/or
                           because the germ line is one of the largest organs in the adult. The spatial aspect of
                           <span class="emphasis"><em>in situ</em></span> hybridization data is particularly informative. For instance, accumulation of
                           oogenesis transcripts from X-linked germline genes is restricted to the proximal germ
                           line, consistent with silencing of the X chromosome in the distal germ line (<a href="#bib13">Kelly et al., 2002</a>, see <a href="http://www.wormbook.org/chapters/www_germlinechromatin/germlinechromatin.html" target="_blank">Germline chromatin</a>).
            </p>
            <p>The identification of germline-expressed genes provides an excellent starting point
                           for analysis of gene function during oogenesis and early embryogenesis. Indeed such
                           studies have begun. Based on an early version of the germline gene set (<a href="#bib20">Reinke et al., 2000</a>), several RNAi screens have been performed, including ones focused
                           on identification of genes required for early embryogenesis (<a href="#bib17">Piano et al., 2002</a>), genes required for correct localization of <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> in the early embryo
                           (<a href="#bib16">Pellettieri et al., 2003</a>; see <a href="http://www.wormbook.org/chapters/www_specgermline/specgermline.html" target="_blank">Specification of the germline</a>), and genes required for
                           chromosome morphogenesis during meiosis (<a href="#bib6">Colai&aacute;covo et al., 2003</a>). A
                           separate study identifying genes expressed during meiosis was followed by RNAi analysis
                           of candidate genes for roles in germline development (<a href="#bib11">Hanazawa et al., 2001</a>). Additionally, protein partners are being identified by systematic yeast
                           two-hybrid analysis (<a href="#bib22">Walhout et al., 2002</a>). Therefore, a tremendous data
                           set is emerging for germline genes, which includes their temporal and spatial
                           expression, biological functions, and protein partners. As this data set becomes more
                           comprehensive, the data will provide a remarkably complete overview of the function and
                           interaction of critical factors and can be used to assemble the genetic regulatory
                           networks underlying germline and embryonic development.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e502"></a>5.&nbsp;Future directions
                     </h2>
                  </div>
               </div>
            </div>
            <p>Many questions about germline development and function remain that will benefit from
                           genomic analyses. Below, some questions are listed along with approaches for answering
                           those questions using DNA microarrays.
            </p>
            <p>How does gene expression differ between different stages of germ cell development?
                           Many additional mutants exist that can be used to further distinguish genes that are
                           expressed in different germ cell types. For instance, a gain-of-function mutation in the
                           GLP-1(Notch) receptor causes excess germline stem cells (<a href="#bib5">Berry et al., 1997</a>; see <a href="http://www.wormbook.org/chapters/www_germlineprolifcontrol/germlineprolifcontrol.html" target="_blank">Germline proliferation and its control</a>). Use of this mutant in
                           microarray experiments will identify genes expressed primarily in the distal
                           proliferating stem cells and better separate these genes from those that are expressed
                           and act in oogenesis.
            </p>
            <p>How many germline-expressed genes still remain to be identified? Because the current
                           microarray studies used intact animals, somatic gene expression has likely masked the
                           germline expression of certain genes. In the future, the use of dissected gonads in
                           microarray experiments will greatly improve the sensitivity with which
                           germline-expressed genes can be identified. Dissected gonads are composed almost
                           entirely of germ cells; with careful dissection, the only somatic cells included are the
                           distal tip cell and sheath cells. Improvements to linear RNA amplification techniques
                           permit the use of very small amounts of RNA (<a href="#bib2">Baugh et al., 2001</a>), and
                           it has been possible to perform microarray experiments with RNA derived from ~50
                           dissected adult gonads. Use of dissected gonads will also improve detection of gene
                           expression changes in mutants with only subtle germline defects.
            </p>
            <p>How do post-transcriptional mechanisms control target mRNAs? Current germline
                           microarray analyses have identified many RNA-binding proteins that are likely to act in
                           the germ line to regulate mRNA translation and/or stability. The RNA targets bound by
                           these proteins can be identified using microarrays. In this experiment, an antibody
                           specific to the RNA binding protein is used to immunoprecipitate the protein and its
                           associated RNAs. These RNAs can then be isolated, labeled and hybridized to a
                           microarray. This approach has worked well for one germline RNA binding protein, <a href="http://www.wormbase.org/db/seq/protein?name=GLD-1;class=protein" target="_blank">GLD-1</a>
                           (see <a href="http://www.wormbook.org/chapters/www_RNAbindingproteins/RNAbindingproteins.html" target="_blank">RNA-binding
                              proteins</a>; M.H. Lee, V. Reinke, and T.
                           Schedl, unpublished data). Over 100 candidate <a href="http://www.wormbase.org/db/seq/protein?name=GLD-1;class=protein" target="_blank">GLD-1</a> mRNA targets have been identified,
                           the vast majority of which have germline-enriched expression.
            </p>
            <p>How do DNA- and chromatin-binding proteins affect germline gene expression? The
                           initial germline microarray experiments described above have identified many candidate
                           transcriptional regulators that likely function in the germ line. RNAi or a genetic
                           deletion of the transcriptional regulator will presumably result in changes in the
                           levels of its transcriptional targets that can be assessed by DNA microarray analysis.
                           These expression data can be correlated with the binding locations of these proteins in
                           the genome. Approaches such as chromatin immunoprecipitation (ChIP) or DNA adenine
                           methylase identification (DamID), which identify or mark DNA sequences bound by specific
                           transcriptional regulators, can be combined with DNA microarray technology to map the
                           bound DNA sequences genome-wide (<a href="#bib18">Pollack and Iyer, 2002</a>; <a href="#bib21">van Steensel and Henikoff, 2003</a>). Because most DNA-binding proteins bind in
                           non-coding, regulatory sequences that flank genes, a microarray that contains intergenic
                           sequences is required for this analysis. Such arrays are currently being built by Jason
                           Lieb at the University of North Carolina at Chapel Hill.
            </p>
            <p>Global genome studies will help to further determine the gene regulatory networks that
                           govern germline function. In addition to microarray-based approaches, systematic RNAi
                           analysis of germline-expressed genes will shed significant light on their roles in
                           germline development (see for example <a href="#bib6">Colai&aacute;covo et al., 2003</a>).
                           Proteomic approaches, such as systematic yeast two-hybrid, mass spectrometry analysis of
                           TAP-immunoprecipitated proteins, and systematic structural analysis (for review, see <a href="#bib23">Zhu et al., 2003</a>), will also play important roles in deciphering the
                           functional relationships among germline gene products. When data from these different
                           functional approaches are combined for germline-expressed genes, we will be
                           well-launched on a comprehensive understanding of germline function.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e553"></a>6.&nbsp;Useful links for germline genomics
                     </h2>
                  </div>
               </div>
            </div>
            <p>
                           <a href="http://wormgermline.yale.edu" target="_top">http://wormgermline.yale.edu</a> (data generated in mutant and timecourse
                           microarray studies can be searched and downloaded).
                           <a href="http://nematode.lab.nig.ac.jp" target="_top">http://nematode.lab.nig.ac.jp</a> (Kohara lab <span class="emphasis"><em>in situ</em></span> database &#8220;NExtDB&#8221;)
                           <a href="http://www.RNAi.org" target="_top">http://www.RNAi.org</a> (database of RNAi experiments including germline gene
                           set) <a href="http://www.wormbase.org" target="_top">http://www.wormbase.org</a> (database of the genomic sequence and useful tools for database mining).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e573"></a>7.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                             <span class="bibliomisc">Barton, M.K., Schedl, T., and Kimble, J., (1987). Gain-of-function mutations of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a></em></span>, a sex-determination gene in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>115</em></span>, 107&#8211;119.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3557107&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                             <span class="bibliomisc">Baugh, L.R., Hill, A.A., Brown, E.L., and Hunter, C.P., (2001). Quantitative analysis of mRNA amplification by <span class="emphasis"><em>in vitro</em></span> transcription. Nuc. Acids Res. <span class="emphasis"><em>29</em></span>, E29.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11222780&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/29.5.e29" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                             <span class="bibliomisc">Bean, C.J., Schaner, C.E., and Kelly, W.G., (2004). Meiotic pairing and imprinted X chromatin assembly in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat Genet. <span class="emphasis"><em>36</em></span>, 100&#8211;105.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14702046&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1283" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                             <span class="bibliomisc">Beanan, M.J., and Strome, S., (1992). Characterization of a germ-line proliferation mutation in <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>116</em></span>, 755&#8211;766.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1289064&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                             <span class="bibliomisc">Berry, L.W., Westlund, B., Schedl, T., (1997). Germ-line tumor formation caused by activation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span>, a <span class="emphasis"><em>Caenorhabditis elegans</em></span> member of the Notch family of receptors. Development <span class="emphasis"><em>124</em></span>, 925&#8211;936.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9043073&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                             <span class="bibliomisc">Colai&aacute;covo, M.P., Stanfield, G.M., Reddy, K.C., Reinke, V., Kim, S.K., and Villeneuve, A.M., (2003). A targeted RNAi screen
                        for genes involved in chromosome morphogenesis and nuclear organization in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> germline. Genetics <span class="emphasis"><em>162</em></span>, 113&#8211;128.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12242227&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                             <span class="bibliomisc">Duggan, D.J., Bittner, M., Chen, Y., Meltzer, P., and Trent, J.M., (1999). Expression profiling using cDNA microarrays. Nat.
                        Genet. <span class="emphasis"><em>21</em></span>, 10&#8211;14.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9915494&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/4434" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                             <span class="bibliomisc">Eisen, M.B., Spellman, P.T., Brown, P.O., and Botstein, D., (1998). Cluster analysis and display of genome-wide expression
                        patterns. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>95</em></span>, 14863&#8211;14868.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9843981&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fong, Y., Bender, L., Wang, W., and Strome, S., (2002). Regulation of the different chromatin states of autosomes and X chromosomes
                        in the germ line of <span class="emphasis"><em>C. elegans</em></span>. Science <span class="emphasis"><em>296</em></span>, 2235&#8211;2238.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12077420&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1070790" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                             <span class="bibliomisc">Goldstein, P., (1982). The synaptonemal complexes of <span class="emphasis"><em>Caenorhabditis elegans</em></span>: pachytene karyotype analysis of male and hermaphrodite wild type and <span class="emphasis"><em>him</em></span> mutants. Chromosoma <span class="emphasis"><em>86</em></span>, 577&#8211;593.</span>
                     <a href="http://dx.doi.org/10.1007/BF00330128" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hanazawa, M., Mochii, M., Ueno, N., Kohara, Y., and Iino Y., (2001). Use of cDNA subtraction and RNA interference screens
                        in combination reveals genes required for germ-line development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>98</em></span>, 8686&#8211;8691.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1447255&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.141004698" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                             <span class="bibliomisc">Jiang, M., Ryu, J., Kiraly, M., Duke, K., Reinke, V., and Kim, S.K., (2001). Genome-wide analysis of developmental and sex-regulated
                        gene expression profiles in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>98</em></span>, 218&#8211;223.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11134517&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.011520898" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kelly, W.G., Schaner, C.E., Dernburg, A.F., Lee, M.H., Kim, S.K., Villeneuve, A.M. and Reinke, V., (2002). X chromosome silencing
                        in the germ line of <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>129</em></span>, 479&#8211;492.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11807039&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                             <span class="bibliomisc">McKee, B.D. and Handel, M.A., (1993). Sex chromosomes, recombination, and chromatin conformation. Chromosoma <span class="emphasis"><em>102</em></span>, 71&#8211;80.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8432196&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00356023" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                             <span class="bibliomisc">Nelson, G.A., Lew, K.K., and Ward, S., (1978). Intersex, a temperature-sensitive mutant of the nematode, <span class="emphasis"><em>C. elegans</em></span>. Dev. Biol. <span class="emphasis"><em>66</em></span>, 386&#8211;409.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=700253&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(78)90247-6" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                             <span class="bibliomisc">Pellettieri, J., Reinke,V., Kim, S.K., and Seydoux, G., (2003). Coordinate activation of maternal protein degradation during
                        the egg-to-embryo transition in <span class="emphasis"><em>C. elegans</em></span>. Dev. Cell <span class="emphasis"><em>5</em></span>, 451&#8211;462.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12242227&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1534-5807(03)00231-4" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                             <span class="bibliomisc">Piano, F., Schetter, A.J., Morton, D.G., Gunsalus, K.C., Reinke, V., Kim, S.K., and Kemphues, K.J. (2002). Gene clustering
                        based on RNAi phenotypes of ovary-enriched genes in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>12</em></span>, 1959&#8211;1964.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12445391&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)01301-5" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                             <span class="bibliomisc">Pollack, J.R., and Iyer, V.R., (2002). Characterizing the physical genome. Nat. Genet. <span class="emphasis"><em>32</em></span>, 515&#8211;521.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12454647&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1035" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                             <span class="bibliomisc">Reinke, V., San Gil, I., Ward, S., and Kazmer, K., (2004). Genome-wide germline-enriched and sex-biased expression profiles
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>131</em></span>, 311&#8211;323.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14668411&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.00914" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                             <span class="bibliomisc">Reinke, V., Smith, H.E., Nance, J., Wang, J., Van Doren, C., Begley, R., Jones, S.J., Davis, E.B., Scherer, S., Ward, S.,
                        and Kim, S.K., (2000). A global profile of germline gene expression in <span class="emphasis"><em>C. elegans</em></span>. Mol. Cell <span class="emphasis"><em>6</em></span>, 605&#8211;616.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11030340&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)00059-9" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                             <span class="bibliomisc">Van Steensel, B., and Henikoff, S., (2003). Epigenomic profiling using microarrays. Biotechniques <span class="emphasis"><em>35</em></span>, 346&#8211;357.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12951776&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                             <span class="bibliomisc">Walhout, A.J.M., Reboul, J., Shtanko, O., Bertin, N., Vaglio, P., Ge, H., Lee, H., Doucette-Stamm, L., Gunsalus, K.C., Schetter,
                        A.G., Morton, D., Kemphues, K.J., Reinke, V., Kim, S.K., Piano, F., and Vidal, M., (2002). Integrating Interactome, Phenome,
                        and Transcriptome Mapping Data for the <span class="emphasis"><em>C. elegans</em></span> Germline. Curr. Biol. <span class="emphasis"><em>12</em></span>, 1952&#8211;1958.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12445390&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)01279-4" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                             <span class="bibliomisc">Zhu H, Bilgin, M., and Snyder, M., (2003). Proteomics. Annu. Rev. Biochem.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14527327&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.biochem.72.121801.161511" target="_blank">Article</a>
                         
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e10" href="#d0e10">*</a></sup>Edited by Judith Kimble and Susan Strome. Last revised March 18, 2005. Published January 20, 2006. This chapter should be
                  cited as: Reinke, V. Germline genomics (January 20, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.74.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Valerie Reinke. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e38" href="#d0e38">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:valerie.reinke@yale.edu" target="_top">valerie.reinke@yale.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

