<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Introduction to nematode evolution and ecology
         
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#FitchIntroNematoEcolEvol_ArticleID" title="Introduction to nematode evolution and ecology&#xA;       ">
      <link rel="next" href="#d0e43" title="1.&nbsp;A model system evolves">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="intronemevolecol.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="FitchIntroNematoEcolEvol_ArticleID"></a>Introduction to nematode evolution and ecology<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                            
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">David H.A.</span> <span class="surname">Fitch<em><span class="remark"><sup><a name="d0e34" href="#ftn.d0e34">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="address">Department of Biology, New York University, New York, NY 10003 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_nematodeevolecol.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Evolution and ecology
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=277.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Introduction+to+nematode+evolution+and+ecology" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e43">1. A model system evolves</a></span></dt>
               <dt><span class="sect1"><a href="#d0e139">2. Nematode relationships to other animals</a></span></dt>
               <dt><span class="sect1"><a href="#d0e455">3. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#d0e460">4. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e43"></a>1.&nbsp;A model system evolves
                     </h2>
                  </div>
               </div>
            </div>
            <p>Sydney Brenner originally conceived <span class="emphasis"><em>Caenorhabditis elegans</em></span> as a model system for building a fundamental understanding of animal genetics, development and behavior, but it is also rapidly
               becoming a respectable model for evolutionary studies. Although <span class="emphasis"><em>C. elegans</em></span> molecular biologists have long appreciated the value of sequence comparisons to identify conserved regions as indicators
               of function, arguably the most interesting aspects of evolution are <span class="emphasis"><em>changes</em></span> in molecular functions, domains of expression, and developmental roles. Elucidating how such changes have shaped functional
               diversity at a variety of levels also has potential for augmenting our understanding of genome function and developmental
               mechanisms. Reciprocally, information from this model system facilitates studies of evolutionary pattern and process. The
               success of such comparative approaches to enhance our understanding obviously depends upon the  availability of material and
               information from multiple related species, as well as different wild populations of <span class="emphasis"><em>C. elegans</em></span>. This section of WormBook reviews much of the recent work to establish a foundation of evolutionary biology and ecology for
               <span class="emphasis"><em>C. elegans</em></span> and other "satellites" of this model system.
            </p>
            <p>Knowing the phylogenetic relationships between <span class="emphasis"><em>C. elegans</em></span> and other nematodes or animals is important for comparative analyses of behavior, morphology, development, molecular mechanisms,
               and genomics. A phylogeny allows the reconstruction of the historical changes that have led to current variation and provides
               a way to test how often convergent changes have occurred. Species phylogenies are also crucial for bioinformatic analyses
               of genomes; they provide a basis for selecting species for comparative genomic sequencing and for testing orthologous and
               paralogous relationships in gene phylogenies, an important foundation for genome annotation and prediction of gene function
               (<a href="#bib10">Eisen, 1998</a>; <a href="#bib11">Eisen and Fraser, 2003</a>). <a href="http://www.wormbook.org/chapters/www_quicktourdiversity/quicktourdiversity.html" target="_blank">A quick tour of nematode diversity and the backbone of nematode phylogeny</a> provides a short tour of general nematode diversity and phylogeny. <a href="http://www.wormbook.org/chapters/www_phylogrhabditids/phylorhab.html" target="_blank">The phylogenetic relationships of <span class="emphasis"><em>C. elegans</em></span> and other rhabditids</a> reviews what is known so far about the closer relationships within the rhabditids and within genus <span class="emphasis"><em>Caenorhabditis</em></span> in particular. The relationship of <span class="emphasis"><em>C. elegans</em></span> to other animals or other major model systems, however, is not covered in these chapters, and is presented in a separate
               part of this introduction ("Animal Relationships" below).
            </p>
            <p>There is substantial variation among rhabditids at genetic and developmental levels. Reconstructing how this variation arose
               is likely to illuminate how developmentally robust systems can nevertheless be modified by evolutionary change, one of the
               most intriguing and fundamental questions in evolutionary biology today. <a href="http://www.wormbook.org/chapters/www_genomevol/genomevol.html" target="_blank">Nematode genome evolution</a> reviews work on the evolution of genome organization and chromosome architecture. <a href="http://www.wormbook.org/chapters/www_evoldevnematode/evoldevnematode.html" target="_blank">Evolution of development in nematodes related to <span class="emphasis"><em>C. elegans</em></span></a> provides an overview of comparative developmental biology using <span class="emphasis"><em>C. elegans</em></span> and other satellite model organisms, such as <span class="emphasis"><em>Pristionchus pacificus</em></span>. These comparative studies reveal that genome architecture and developmental mechanisms can be strikingly diverse, suggesting
               that there is significant plasticity and potential variation available to selection, even for features that superficially
               appear to be highly conserved.
            </p>
            <p>Despite remarkable progress toward a molecular understanding for <span class="emphasis"><em>C. elegans</em></span>, an ecological context for this model system has lagged behind. Substantial molecular work has contributed to a detailed
               picture of the mechanisms involved in dauer formation, chemosensation and other aspects of neurobiology and behavior, innate
               immunity, germline development and sex determination, senescence and developmental timing. Although these features must somehow
               be important for the organism's life and its interaction with the environment, the actual ecological context and evolutionary
               "pressures" to which such features are adaptations have remained quite unclear. <a href="http://www.wormbook.org/chapters/www_naturalvariationgenetics/naturalvariationgenetics.html" target="_blank">Natural variation and population genetics of <span class="emphasis"><em>C. elegans</em></span> </a> reviews work on <span class="emphasis"><em>C. elegans</em></span> population genetics which will help us comprehend, among other things, how this organism migrates, the possible role of mating
               in its natural setting, how variation is related to geography and environment, and the birth-death cycles of populations.
               <a href="http://www.wormbook.org/chapters/www_ecolCaenorhabditis/ecolCaenorhabditis.html" target="_blank">Ecology of <span class="emphasis"><em>Caenorhabditis</em></span> species</a> reviews the potential interactions that <span class="emphasis"><em>C. elegans</em></span> may have with bacteria and pathogens in its environment. <a href="http://www.wormbook.org/chapters/www_intermicrobpath/intermicrobpath.html" target="_blank">Interactions with microbial pathogens</a> reviews current understanding of the ecological context of <span class="emphasis"><em>C. elegans</em></span> and its closest relatives. Future work in these areas is likely to clarify why the biological features of <span class="emphasis"><em>C. elegans</em></span> we like to study in the lab actually exist and why they may be important for the animal itself, thus providing further functional
               insight.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e139"></a>2.&nbsp;Nematode relationships to other animals
                     </h2>
                  </div>
               </div>
            </div>
            <p>In <span class="emphasis"><em>C. elegans</em></span> II, we outlined the 3 possible hypotheses (<a href="#figure1">Figure 1</a>) for relationships between three major model systems, "the" arthropod <span class="emphasis"><em>Drosophila melanogaster</em></span>, "the" vertebrate <span class="emphasis"><em>Mus musculus</em></span>, and "the" nematode <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib13">Fitch and Thomas, 1997</a>). We emphasized that elucidating these relationships was important for making inferences and predictions about which components,
               mechanisms and functions might be unique and derived or ancestrally shared by these or other related model and non-model species,
               such as humans. Of course, additional representatives of animals (including other nematodes) are needed in the phylogenetic
               framework for greater accuracy of such predictions. After we wrote that review, several interesting studies addressed relationships
               among the major animal phyla, and particularly the relationship of nematodes to other animals. There is still considerable
               (even polemical) debate, but additional data and increased analytical sophistication may provide answers in the not-distant
               future. As it is not possible here to review all of the work on animal phylogeny, and because the saga seems far from concluded,
               I highlight only the major findings relevant to the position of nematodes.
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="225">
<tr>
  <td align="center"><p><a href="IntroNemEcoEvofig1.jpg"><img src="IntroNemEcoEvofig1_s.jpg" border="2" align="middle" alt="The three hypotheses for the relationships of three model organisms Figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1. The three hypotheses for the relationships of three model organisms.&nbsp;</b>(A) Mouse and fly are more closely related than either is to <span class="emphasis"><em>C. elegans</em></span>, consistent with the Coelomata hypothesis; (B) Fly and worm are more closely related than either is to mouse, consistent
                     with the Ecdysozoa hypothesis; (C) Worm and mouse are more closely related than either is to fly, not consistent with either
                     of the major competing hypotheses of animal relationships. Modified from Fitch and Thomas (1997).
                  </p>
               </div>
            </div>
            <p>On the basis of complete 18S ribosomal RNA (rRNA) sequences, <a href="#bib2">Aguinaldo et al., 1997</a> proposed that nematodes were related to arthropods in a clade of molting animals they called "Ecdysozoa" (<a href="#figure1">Figure 1</a>b), to the exclusion of deuterostomes (represented by an echinoderm in their study) and some other protostome groups, such
               as molluscs and annelids. This hypothesis differed substantially from the more traditional "Coelomata" hypothesis that placed
               nematodes on a branch diverging before coelomates diverged from one another (i.e., before the divergence of lineages leading
               to mice and flies, <a href="#figure1">Figure 1</a>a). Support for Ecdysozoa depended on excluding all but one nematode, <span class="emphasis"><em>Trichinella spiralis</em></span> (which unfortunately possesses a rRNA sequence with an odd nucleotide composition compared to other nematodes characterized
               so far). However, when the authors included the other nematodes (e.g., <span class="emphasis"><em>C. elegans</em></span>), the nematodes clustered together near the bottom of the tree, consistent with Coelomata, and consistent with data from
               RNA polymerase II (<a href="#bib28">Sidow and Thomas, 1994</a>). Use of <span class="emphasis"><em>Trichinella</em></span> as a representative nematode was justified on the basis that its rRNA sequence evolved more slowly than that of other nematodes,
               such as <span class="emphasis"><em>C. elegans</em></span>. A phenomenon called "long branch attraction" (LBA) can cause taxa with long branches (representing many evolutionary changes)
               to artefactually group with other long branches, particularly those of the outgroup taxa near the root of the tree (<a href="#bib12">Felsenstein, 1978</a>). Of course, the other possible reason that nematodes have long branches is simply that they diverged early from the other
               taxa, as predicted by the Coelomata hypothesis (<a href="#figure2">Figure 2</a>). The putative effect of LBA to provide artefactual support to Coelomata has been central to the debate, along with issues
               of taxon and character sampling.
            </p>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="IntroNemEcoEvofig2.jpg"><img src="IntroNemEcoEvofig2_s.jpg" border="2" align="middle" alt="Figure 2"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 2. Why nematodes might have a long branch.&nbsp;</b>It has been claimed that a phenomenon called "long-branch attraction" (LBA) results in an artefactual placement of nematodes
                     near the base of the bilaterian phylogeny, thus appearing to be consistent with the "Coelomata" hypothesis, and obscuring
                     the phylogenetic signal for "Ecdysozoa". However, the branch (lineage) leading to nematodes could be long for different reasons:
                     (1) the rate of nucleotide change is higher in nematodes than in other taxa (e.g., Ecdysozoa could be true and support for
                     Coelomata is artefactual), or (2) the nematodes branched earlier from the tree than the other taxa (i.e., Coelomata is true
                     after all), or (3) both (i.e., Coelomata is true and the rate is higher in nematodes). One legitimate method for testing for
                     significant LBA is to conduct a relative-rate test; one can then discard data (taxa or sequences) which show a significantly
                     higher rate in nematodes than in other taxa. In the relative-rate test, the distance (amount of change, corrected for superimposed
                     substitutions) between one ingroup taxon (like nematodes, N) and an outgroup (O, e.g. diploblastic animals, yeast, etc.) is
                     compared to the distance between another ingroup taxon (e.g., a non-nematode protostome, P<sub>1</sub>, like <span class="emphasis"><em>Drosophila</em></span>) and the same outgroup. If evolution has occured at equal rates in the different lineages, these distances, NO and P<sub>1</sub>O, should be approximately equal, regardless of which hypothesis (Coelomata or Ecdysozoa) is true. The outgroup, O, thus serves
                     as an unbiased "anchor" for these rate comparisons. Any deviation from this equality is a measure of a rate difference. Data
                     showing such a deviation may be discarded to see if their exclusion results in a change in the phylogenetic topology. It is
                     not legitimate, however, to use a potential ingroup as an anchor for the relative-rate test. For example, if human (or other
                     deuterostome taxon, D) were used, the relative rate test would be biased to favor Ecdysozoa. That is, although the distance
                     between N and D should be the same as that between P<sub>1</sub> and D if Ecdysozoa were true, ND should be <span class="emphasis"><em>greater than</em></span> P<sub>1</sub>D if Coelomata were true and rates were <span class="emphasis"><em>equal</em></span>. Eliminating data where ND<span class="symbol">&gt;</span> P<sub>1</sub>D would thus bias the selection of data to favor Ecdysozoa, even if Coelomata were true.
                  </p>
               </div>
            </div>
            <p>Unfortunately, the index that <a href="#bib2">Aguinaldo et al., 1997</a>, used to classify "long-branch taxa" was the distance between each taxon to the hypothetical last common ancestor of the
               protostomes (P in <a href="#figure2">Figure 2</a>), rather than to a more certain outgroup taxon, such as the cnidarian in their study, as is usually done in relative rate
               tests (<a href="#bib26">Sarich and Wilson, 1967</a>; <a href="#bib32">Wu and Li, 1985</a>; see <a href="#figure2">Figure 2</a>). That is, their method for selecting data to exclude assumed that nematodes diverged from other protostomes after deuterostomes
               diverged (as in the Ecdysozoa hypothesis). This assumption would be incorrect if Coelomata were true. Under the Coelomata
               hypothesis, the distance between nematodes and the hypothetical protostome ancestor is expected to be greater than the distance
               between arthropods and the protostome ancestor, even under constant evolutionary rates (<a href="#figure2">Figure 2</a>). Excluding taxa that show such a  distance ratio would obviously be biased to favor Ecdysozoa. Instead, rates should be
               compared relative to a well-accepted outgroup taxon (O in <a href="#figure2">Figure 2</a>). Under the null hypothesis of constant rates, distances between any extant taxon and the outgroup should be the same; the
               amount of deviation from this equality can therefore be taken as the degree of rate difference (<a href="#figure2">Figure 2</a>).
            </p>
            <p>The same problem applies to data selection in a study which found that 24 of 36 protein orthologs from <span class="emphasis"><em>Saccharomyces cerevisiae</em></span>, <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em>Drosophila melanogaster</em></span>, and human were consistent with Coelomata and only 11 were consistent with Ecdysozoa (<a href="#bib20">Mushegian et al., 1998</a>). The authors argued that LBA could result in artefactual support for Coelomata because most of the proteins supporting Coelomata
               had a longer branch between <span class="emphasis"><em>C. elegans</em></span> and human than between <span class="emphasis"><em>Drosophila</em></span> and human. However, this longer branch for <span class="emphasis"><em>C. elegans</em></span> would be expected even if rates were perfectly clocklike and Coelomata were true (<a href="#figure2">Figure 2</a>). Unfortunately, this study was also hampered by the fact that most of their sets of putative "orthologs" were actually paralogs
               of various sorts, thus introducing another source of possible error for the inference of species relationships (<a href="#bib33">Xie and Ding, 2000</a>). Human was also used as the basis for comparing relative rates in a recent study which included genome-scale data for eleven
               taxa and reported strong support for Ecdysozoa when "long-branch" data were excluded (<a href="#bib9">Dopazo and Dopazo, 2005</a>). Thus, in all of the studies described above, inappropriate use of a potential ingroup taxon to measure relative rates could
               have mistakenly biased the conclusions in favor of Ecdysozoa.
            </p>
            <p>It has been claimed that a phenomenon called "long-branch attraction" (LBA) results in an artefactual placement of nematodes
               near the base of the bilaterian phylogeny, thus appearing to be consistent with the "Coelomata" hypothesis, and obscuring
               the phylogenetic signal for "Ecdysozozoa" <a href="#figure2">Figure 2</a>.
            </p>
            <p>Recent studies using genome-scale numbers of genes have generally supported Coelomata and rejected Ecdysozoa (e.g., <a href="#bib5">Brown et al., 2001</a>; <a href="#bib4">Blair et al., 2002</a>; <a href="#bib31">Wolf et al., 2004</a>). Some of these studies tested for effects of LBA and found no significant effect. The most recent study (<a href="#bib23">Philip et al., 2005</a>), used only single-copy genes from ten completely sequenced genomes to help ensure that only orthologs were compared and
               thus avoid complications of comparing potentially paralogous genes (which would have reflected gene duplication events and
               not the speciation events of interest). Their 780 orthologous genes, as well as the concatenated sequences of five genes meeting
               strict criteria for appropriate phylogenetic signal, strongly support Coelomata, even when sites with the highest evolutionary
               rates are removed. If support for Coelomata were merely due to an artifact of rapid evolution, removing these sites should
               produce support for Ecdysozoa. Removing additional sites with moderate rates results in ambiguity, but not support for Ecdysozoa.
            </p>
            <p>A major criticism of these studies is that the sampling of taxa is small, as might be expected for whole-genome comparisons.
               To determine the effect of both taxon and character sampling, <a href="#bib24">Philippe et al., (2005</a>; reviewed by <a href="#bib17">Jones and Blaxter, 2005</a>) used data from 146 genes and a fairly diverse taxonomic sample of 35 species. In this case, the authors identified the fastest-evolving
               genes by appropriate comparison to the outgroup species and found strong support for Ecdysozoa when these genes were excluded.
               By including or removing taxa, the authors also demonstrated a clear effect of taxon selection. For example, adding hydra
               to the outgroup, which otherwise had only fungi and choanoflagellates, caused <span class="emphasis"><em>C. elegans</em></span> to jump from a position consistent with Coelomata to one consistent with Ecdysozoa. The authors conclude that both accounting
               for LBA effects and including a denser sampling of taxa are required to uncover the phylogenetic signal for Ecdysozoa. This
               effect of taxon addition is explained by the ability of added taxa to "break" long branches and apportion changes more appropriately
               into different lineages, thus providing better phylogenetic information about which states are primitive and which are derived
               (<a href="#bib18">Kim, 1998</a>). Even when only one or a few genes are employed (such as for 18S rRNA), including more taxa has apparently aided resolution,
               generally resulting in support for Ecdysozoa (<a href="#bib14">Giribet and Ribera, 1998</a>; <a href="#bib15">Giribet and Wheeler, 1999</a>; <a href="#bib22">Peterson and Eernisse, 2001</a>). Adding more taxa, however, means that statistically testing the robustness of relationships becomes computationally much
               more time-consuming and most of these taxon-dense analyses do not have such tests.
            </p>
            <p>In this brief review, I have focused on contributions from recent work using multigene data. However, insight might also be
               provided by a few molecular characters which nevertheless have phylogenetic signatures complex enough to have arisen only
               once, such as insertions, deletions, and organization of gene clusters. For example, Hox gene organization may be one such
               complex&#8212;and therefore informative&#8212;feature (<a href="#bib3">Balavoine et al., 2002</a>). The <span class="emphasis"><em>Abd-B</em></span> gene appears to be specific for Ecdysozoa. However, orthology of the <span class="emphasis"><em>C. elegans <a href="http://www.wormbase.org/db/gene/gene?name=php-3" target="_blank">php-3</a></em></span> to <span class="emphasis"><em>Abd-B</em></span> is only very weakly supported (42% bootstrap value; <a href="#bib8">de Rosa et al., 1999</a>). One problem with <span class="emphasis"><em>C. elegans</em></span> as a representative nematode is that it clearly has a highly derived organization of the <span class="emphasis"><em>Hox</em></span> gene cluster relative to other nematodes; genes have been lost and rearrangements have occurred in lineages leading  to <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib1">Aboobaker and Blaxter, 2003</a>). Analyses using other nematode species should therefore provide better phylogenetic information than <span class="emphasis"><em>C. elegans</em></span> for addressing the Ecdysozoa/Coelomata question. Other potentially informative molecular characters are insertions, deletions
               and intragenic rearrangements. In this case, Coelomata appears to be supported (<a href="#bib31">Wolf et al., 2004</a>). On the other hand, intron patterns seem to support Ecdysozoa. In an essentially 4-taxon study, <a href="#bib25">Roy and Gilbert, 2005</a> found that patterns of "intron conservation" (actually shared losses of introns) tended to support Ecdysozoa, under the assumption
               that introns were never regained in the same position. For all of these studies, the problem of low taxon density could be
               problematic. The extreme dynamic of intron loss and gain even within one genus of nematodes, <span class="emphasis"><em>Caenorhabditis</em></span> (<a href="#bib19">Kiontke et al., 2004</a>), suggests one should be wary of assigning homologies to intron presence or absence in taxa as distantly related as different
               phyla.
            </p>
            <p>There are still uncertainties and questions. Because of improvements caused by better taxon sampling, adding more taxa that
               represent more animal phyla could potentially alter the tree topology again. Because Ecdysozoa predicts that "molting animals"
               shared a common ancestor that was itself a molting animal, we might expect that molting in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>Drosophila</em></span> share common mechanisms (although the cuticles that are molted in these animals are very different). In fact, orthologs of
               some of the nuclear hormone receptors (NRs) involved in molting and metamorphosis in insects also function in growth and dauer
               molts in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib16">Gissendanner et al., 2004</a>). The filarial nematode, <span class="emphasis"><em>Dirofilaria immitis</em></span>, has putative orthologs of the ecdysone receptor gene and other ecdysone-regulated genes that are absent from the <span class="emphasis"><em>C. elegans</em></span> genome (<a href="#bib7">Crossgrove et al., 2002</a>; <a href="#bib27">Shea et al., 2004</a>). Of course, filaria are parasites of insects, and a means to respond to insect hormones would have been adaptive and could
               have evolved secondarily. An even more important issue is whether these NR genes or hormone pathways could have been shared
               by much more ancient common ancestry and lost or modified in other lineages. In fact, ecdysteroids are found throughout eukaryotes.
               Such features may not represent derived changes ("apomorphies") supporting Ecdysozoa. As an analogy, just because humans and
               rats have four limbs does not mean that four limbs supports a close relationship between humans and rats, since four limbs
               are shared ("plesiomorphically") by all tetrapods (except where they have been lost secondarily, as in snakes). Cuticular
               shedding also occurs in non-ecdysozoan phyla, such as in some annelids and sipunculans (<a href="#bib6">Brusca and Brusca, 2003</a>), so even the nominal feature of Ecdysozoa cannot be used exclusively to support this taxon.
            </p>
            <p>Even if Ecdysozoa turns out to be the most well-supported hypothesis at the molecular level, there are many anatomical features
               of nematodes that will have to be explained (see <a href="#bib21">Nielsen, 2001</a>; <a href="#bib6">Brusca and Brusca, 2003</a>). Several features presumably present in an ecdysozoan ancestor must have been lost or reduced and/or completely remodeled
               in nematodes (appendages, segmentation, excretory system, nervous system). Although it is conceivable that coeloms have been
               reduced to "pseudocoeloms" several times during animal evolution by heterochrony or similar means and that specialized gas
               exchange systems were eliminated as a response to the evolution of small body size, it is difficult to envisage how nematodes
               could have lost such features as bilateral ganglia after they had evolved in an ancestor (<a href="#bib6">Brusca and Brusca, 2003</a>). As more genomes are sequenced for more representatives of different animal phyla, and the hypotheses are appropriately
               tested with the  increasingly sophisticated tools of phylogenetics, the position of nematodes in the Tree of Life will become
               clearer.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e455"></a>3.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>Many thanks to Thomas B&uuml;rglin for stimulating discussions about the Ecdysozoa debate and to Mark Blaxter, Rob DeSalle, Gonzalo
               Giribet, and Karin Kiontke for their valuable comments on the manuscript.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e460"></a>4.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Aboobaker, A.A., and Blaxter, M.L. (2003). <span class="emphasis"><em>Hox</em></span> gene loss during dynamic evolution of the nematode cluster. Curr. Biol. <span class="emphasis"><em>13</em></span>, 37&#8211;40.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12526742&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)01399-4" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Aguinaldo, A.M., Turbeville, J.M., Linford, L.S., Rivera, M.C., Garey, J.R., Raff, R.A., and Lake, J.A. (1997). Evidence for
                        a clade of nematodes, arthropods and other molting animals. Nature <span class="emphasis"><em>387</em></span>, 489&#8211;493.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9168109&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/387489a0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Balavoine, G., de Rosa, R., and Adoutte, A. (2002). <span class="emphasis"><em>Hox</em></span> clusters and bilaterian phylogeny. Mol. Phylogenet. Evol. <span class="emphasis"><em>24</em></span>, 366&#8211;373.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12220978&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1055-7903(02)00237-3" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Blair, J.E., Ikeo, K., Gojobori, T., and Hedges, S.B. (2002). The evolutionary position of nematodes. BMC Evol. Biol. <span class="emphasis"><em>2</em></span>, 7.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11985779&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2148-2-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Brown, J.R., Douady, C.J., Italia, M.J., Marshall, W.E., and Stanhope, M.J. (2001). Universal trees based on large combined
                        protein sequence data sets. Nat. Genet. <span class="emphasis"><em>28</em></span>, 281&#8211;285.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11431701&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/90129" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Brusca, R.C., and Brusca, G.J. (2003). Invertebrates, 2nd edn (Sunderland, MA: Sinauer Associates), 936 p.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Crossgrove, K., Laudet, V., and Maina, C.V. (2002). <span class="emphasis"><em>Dirofilaria immitis</em></span> encodes <span class="emphasis"><em>Di-nhr-7</em></span>, a putative orthologue of the <span class="emphasis"><em>Drosophila</em></span> ecdysone-regulated <span class="emphasis"><em>E78</em></span> gene. Mol. Biochem. Parasitol. <span class="emphasis"><em>119</em></span>, 169&#8211;177.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11814569&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0166-6851(01)00412-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">de Rosa, R., Grenier, J.K., Andreeva, T., Cook, C.E., Adoutte, A., Akam, M., Carroll, S.B., Balavoine, G. (1999). <span class="emphasis"><em>Hox</em></span> genes in brachiopods and priapulids and protostome evolution. Nature <span class="emphasis"><em>399</em></span>, 772&#8211;776.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10391241&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/21631" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Dopazo, H., and Dopazo, J. (2005). Genome-scale evidence of the nematode-arthropod clade. Genome Biol. <span class="emphasis"><em>6</em></span>, R41.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15892869&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2005-6-5-r41" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Eisen, J.A. (1998). Phylogenomics: improving functional predictions for uncharacterized genes by evolutionary analysis. Genome
                        Res. <span class="emphasis"><em>8</em></span>, 163&#8211;167.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9521918&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Eisen, J.A., and Fraser, C.M. (2003). Phylogenomics: intersection of evolution and genomics. Science <span class="emphasis"><em>300</em></span>, 1706&#8211;1707.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12805538&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1086292" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Felsenstein, J. (1978). Cases in which parsimony or compatibility methods will be positively misleading. Syst. Zool. <span class="emphasis"><em>27</em></span>, 401&#8211;410.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Fitch, D.H.A., and Thomas, W.K. (1997). Evolution. In <span class="emphasis"><em>C. elegans II</em></span>, D.L. Riddle, T. Blumenthal, B.J. Meyer, and J.R. Priess, eds. (Cold Spring Harbor: Cold Spring Harbor Laboratory Press),
                        pp. 815&#8211;850.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Giribet, G., and Ribera, C. (1998). The position of arthropods in the animal kingdom: a search for a reliable outgroup for
                        internal arthropod phylogeny. Mol. Phylogenet. Evol. <span class="emphasis"><em>9</em></span>, 481&#8211;488.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9667996&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/mpev.1998.0494" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Giribet, G., and Wheeler, W.C. (1999). The position of arthropods in the animal kingdom: Ecdysozoa, islands, trees, and the
                        "Parsimony ratchet". Mol. Phylogenet. Evol. <span class="emphasis"><em>3</em></span>, 619&#8211;623.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10620418&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/mpev.1999.0679" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Gissendanner, C.R., Crossgrove, K., Kraus, K.A., Maina, C.V., and Sluder, A.E. (2004). Expression and function of conserved
                        nuclear receptor genes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>266</em></span>, 399&#8211;416.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14738886&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2003.10.014" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jones, M., and Blaxter, M. (2005). Evolutionary biology: animal roots and shoots. Nature <span class="emphasis"><em>434</em></span>, 1076&#8211;1077.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15858555&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/4341076a" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kim, J. (1998). Large-scale phylogenies and measuring the performance of phylogenetic estimators. Syst. Biol. <span class="emphasis"><em>47</em></span>, 43&#8211;60.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12064240&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1080/106351598261021" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kiontke, K., Gavin, N.P., Raynes, Y., Roehrig, C., Piano, F., and Fitch, D.H.A. (2004). <span class="emphasis"><em>Caenorhabditis</em></span> phylogeny predicts convergence of hermaphroditism and extensive intron loss. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 9003&#8211;9008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15184656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0403094101" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mushegian, A.R., Garey, J.R., Martin, J., and Liu, L.X. (1998). Large-scale taxonomic profiling of eukaryotic model organisms:
                        a comparison of orthologous proteins encoded by the human, fly, nematode, and yeast genomes. Genome Res. <span class="emphasis"><em>8</em></span>, 590&#8211;598.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9647634&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Nielsen, C. (2001). Animal Evolution: Interrelationships of the Living Phyla, 2nd edn (Oxford: Oxford University Press).</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Peterson, K.J., and Eernisse, D.J. (2001). Animal phylogeny and the ancestry of bilaterians: inferences from morphology and
                        18S rDNA gene sequences. Evol. Dev. <span class="emphasis"><em>3</em></span>, 170&#8211;205.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11440251&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1046/j.1525-142x.2001.003003170.x" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Philip, G.K., Creevey, C.J., and McInerney, J.O. (2005). The opisthokonta and the ecdysozoa may not be clades: stronger support
                        for the grouping of plant and animal than for animal and fungi and stronger support for the coelomata than ecdysozoa. Mol.
                        Biol. Evol. <span class="emphasis"><em>22</em></span>, 1175&#8211;1184.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15703245&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msi102" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Philippe, H., Lartillot, N., and Brinkmann, H. (2005). Multigene analyses of bilaterian animals corroborate the monophyly
                        of ecdysozoa, lophotrochozoa, and protostomia. Mol. Biol. Evol. <span class="emphasis"><em>22</em></span>, 1246&#8211;1253.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15703236&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msi111" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Roy, S.W., and Gilbert, W. (2005). Resolution of a deep animal divergence by the pattern of intron conservation. Proc. Natl.
                        Acad. Sci. USA <span class="emphasis"><em>102</em></span>, 4403&#8211;4408.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15769859&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0409891102" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sarich, V.M., and Wilson, A.C. (1967). Immunological time scale for hominid evolution. Science <span class="emphasis"><em>158</em></span>, 1200&#8211;1203.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4964406&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Shea, C., Hough, D., Xiao, J., Tzertzinis, G., and Maina, C.V. (2004). An <span class="emphasis"><em>rxr/usp</em></span> homolog from the parasitic nematode, <span class="emphasis"><em>Dirofilaria immitis</em></span>. Gene <span class="emphasis"><em>324</em></span>, 171&#8211;182.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=146932&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.gene.2003.09.032" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sidow, A., and Thomas, W.K. (1994). A molecular evolutionary framework for eukaryotic model organisms. Curr. Biol. <span class="emphasis"><em>4</em></span>, 596&#8211;603.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7953533&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(00)00131-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sudhaus, W., and Fitch, D. (2001). Comparative studies on the phylogeny and systematics of the Rhabditidae (Nematoda). J.
                        Nematol. <span class="emphasis"><em>33</em></span>, 1&#8211;70.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Winnepenninckx, B., Backeljau, T., Mackey, L.Y., Brooks, J.M., De Wachter, R., Kumar, S., and Garey, J.R. (1995). 18S rRNA
                        data indicate that Aschelminthes are polyphyletic in origin and consist of at least three distinct clades. Mol. Biol. Evol.
                        <span class="emphasis"><em>12</em></span>, 1132&#8211;1137.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wolf, Y.I., Rogozin, I.B., and Koonin, E.V. (2004). Coelomata and not Ecdysozoa: evidence from genome-wide phylogenetic analysis.
                        Genome Res. <span class="emphasis"><em>14</em></span>, 29&#8211;36.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14707168&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.1347404" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wu, C.-I, and Li, W.-H. (1985). Evidence for higher rates of nucleotide substitution in rodents than in man. Proc. Natl. Acad.
                        Sci. USA <span class="emphasis"><em>82</em></span>, 1741&#8211;1745.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3856856&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Xie, T., and Ding, D. (2000). Investigating 42 candidate orthologous protein groups by molecular evolutionary analysis on
                        genome scale. Gene <span class="emphasis"><em>261</em></span>, 305&#8211;310.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11167018&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0378-1119(00)00506-0" target="_blank">Article</a>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Lisa R. Girard. Last revised June 08, 2005. Published August 31, 2005. This chapter should be cited as: Fitch, D.H.A.,
                  Introduction to nematode evolution and ecology (August 31, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.19.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2005 David H.A. Fitch. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e34" href="#d0e34">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:david.fitch@nyu.edu" target="_top">david.fitch@nyu.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

