<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Meiosis</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#meiosis" title="Meiosis">
      <link rel="next" href="#sec1" title="1.&nbsp;Abbreviations">
      <script language="javascript" type="text/javascript" src="/js/history_list.js"></script>
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
    <!--#include virtual="/ssi/boilerplate.html" -->
 	<div class="html_to_pdf_link">
		<a href="meiosis.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="meiosis"></a>Meiosis<sup><a name="d0e6" href="#ftn.d0e6">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Kenneth J Hillers<sup>1<span class="italic">&sect;</span></sup>, Verena Jantsch<sup>2<span class="italic">&sect;</span></sup>, Enrique Martinez-Perez<sup>3<span class="italic">&sect;</span></sup>, Judith L Yanowitz<sup>4<span class="italic">&sect;</span></sup><em><span class="remark"><sup><a name="d0e46" href="#ftn.d0e46">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup>Biological Sciences Department, California Polytechnic State University, San Luis Obispo, CA 93407, United States; </span></span><span class="affiliation"><span class="orgname"><sup>2</sup>Department of Chromosome Biology, Max F. Perutz Laboratories, University of Vienna, Vienna Biocenter,1030 Vienna, Austria.;
                           </span></span><span class="affiliation"><span class="orgname"><sup>3</sup>MRC Clinical Sciences Centre, Imperial College, London W12 0NN, United Kingdom; </span></span><span class="affiliation"><span class="orgname"><sup>4</sup>Magee-Womens Research Institute, Department of Obstetrics, Gynecology, and Reproductive Sciences, University of Pittsburgh
                           School of Medicine, Pittsburgh, PA 15213, United States; </span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_germline.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > The germ line
		</a></div><hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Abbreviations</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. Overview</a></span></dt>
               <dt><span class="sect1"><a href="#sec3">3. Chromosome pairing in prophase of meiosis I</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec3.1">3.1. <span class="italic">Cis-</span>acting sequences promote pairing</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3.2">3.2. PCs assemble as nucleoprotein complexes at the nuclear periphery</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3.3">3.3. PC movements promote efficient homolog pairing</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec4">4. Meiotic chromosome structure</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec4.1">4.1. Meiosis-specific cohesin complexes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4.2">4.2. HORMA-domain proteins</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4.3">4.3. The SC</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec5">5. Meiotic recombination</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec5.1">5.1. DSB formation</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5.2">5.2. DSB repair by meiosis-specific adaptation of homologous recombination</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5.3">5.3. The recombination landscape</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5.4">5.4. CO interference, assurance, and homeostasis</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5.5">5.5. Interplay between recombination and chromosome structure</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5.6">5.6. Window of opportunity for CO formation</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec6">6. Bivalent differentiation</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec6.1">6.1. CO-triggered chromosome remodeling</a></span></dt>
                     <dt><span class="sect2"><a href="#sec6.2">6.2. Chromosome condensation</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec7">7. Surveillance mechanisms during prophase I</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec7.1">7.1. Monitoring of SC assembly</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7.2">7.2. Monitoring of recombination progression</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7.3">7.3. Proteins that promote normal prophase I progression</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec8">8. Chromosome segregation during the meiotic divisions</a></span></dt>
               <dt><span class="sect1"><a href="#sec9">9. Summary</a></span></dt>
               <dt><span class="sect1"><a href="#sec10">10. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#sec11">11. <span class="bold"><strong>Table 1. List of meiotic genes discussed in this chapter</strong></span></a></span></dt>
               <dt><span class="sect1"><a href="#sec12">12. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>Sexual reproduction requires the production of haploid gametes (sperm and egg) with only one copy of each chromosome; fertilization
                     then restores the diploid chromosome content in the next generation. This reduction in genetic content is accomplished during
                     a specialized cell division called meiosis, in which two rounds of chromosome segregation follow a single round of DNA replication.
                     In preparation for the first meiotic division, homologous chromosomes pair and synapse, creating a context that promotes formation
                     of crossover recombination events. These crossovers, in conjunction with sister chromatid cohesion, serve to connect the two
                     homologs and facilitate their segregation to opposite poles during the first meiotic division. During the second meiotic division,
                     which is similar to mitosis, sister chromatids separate; the resultant products are haploid cells that become gametes.</strong></span></p>
            <p><span class="bold"><strong>In <span class="italic">Caenorhabditis elegans</span> (and most other eukaryotes) homologous pairing and recombination are required for proper chromosome inheritance during meiosis;
                     accordingly, the events of meiosis are tightly coordinated to ensure the proper execution of these events. In this chapter,
                     we review the seminal events of meiosis: pairing of homologous chromosomes, the changes in chromosome structure that chromosomes
                     undergo during meiosis, the events of meiotic recombination, the differentiation of homologous chromosome pairs into structures
                     optimized for proper chromosome segregation at Meiosis I, and the ultimate segregation of chromosomes during the meiotic divisions.
                     We also review the regulatory processes that ensure the coordinated execution of these meiotic events during prophase I.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Abbreviations
                     </h2>
                  </div>
               </div>
            </div>
            <p><span class="bold"><strong>CO</strong></span>&#8211;crossover, <span class="bold"><strong>DSB</strong></span>&#8211;double strand break, <span class="bold"><strong>HR</strong></span>&#8211;homologous recombination, <span class="bold"><strong>HRR</strong></span>&#8211;homology recognition region, <span class="bold"><strong>IR</strong></span>&#8211;ionizing radiation, <span class="bold"><strong>NCO</strong></span>&#8211;non-crossover, <span class="bold"><strong>NHEJ</strong></span>&#8211;non-homologous end joining, <span class="bold"><strong>PC</strong></span>&#8211;pairing center, <span class="bold"><strong>SC</strong></span>&#8211;synaptonemal complex
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Overview
                     </h2>
                  </div>
               </div>
            </div>
            <p>Sexual reproduction requires the generation of haploid gametes from diploid precursors through the specialized cell division
               program of meiosis. This reduction in ploidy is essential to ensure the restoration of diploidy upon fertilization and requires
               completion of several key events (<a href="#figure1">Figure 1</a>). During early prophase (leptotene and zygotene stages), each chromosome must locate and recognize its appropriate homologous
               pairing partner and align with it. During the zygotene stage, a specialized protein structure called the synaptonemal complex
               (SC) assembles between the aligned chromosomes to hold homologs together; full synapsis of homologs defines the pachytene
               stage. Crossover (CO) recombination events must be completed between the DNA molecules of the aligned and synapsed homologs,
               a process started by the deliberate formation of DNA double strand breaks (DSBs). Crossing over is essential for the formation
               of chiasmata, connections between homologs that become evident upon structural remodeling of chromosomes during later stages
               of meiotic prophase (diplotene and diakinesis). Therefore, there are multiple surveillance mechanisms that act to ensure that
               each homolog pair undergoes an exchange. Late prophase remodeling of chromosome pairs connected by chiasmata results in bivalents
               wherein the connected homologs are oriented away from each other; this promotes bipolar attachment of homologs to the meiosis
               I spindle, leading to segregation of homologous chromosomes at anaphase I. The separation of sister chromatids on the meiosis
               II spindle completes the meiotic program.
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="meiosis_fig1.jpg"><img src="meiosis_fig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b><span class="bold"><strong>Figure 1: Diagram of meiotic events during oogenesis in the <span class="italic">C. elegans</span> germ line.</strong></span>&nbsp;</b>For simplicity, a single pair of homologous chromosomes is shown. DNA replication occurs at the onset of meiosis. During the
                     transition zone in early meiotic prophase (leptotene and zygotene), homolog pairing is achieved and DSBs are formed. By early
                     pachytene, SC assembly is completed, and by late pachytene, the process of homologous recombination leads to the formation
                     of inter-homolog CO events. Following SC disassembly during late prophase, homologous chromosomes remain linked by chiasmata,
                     physical attachments provided by COs in combination with sister chromatid cohesion. In the first meiotic division parental
                     homologs are disjoined to reduce ploidy to the haploid state, and in the second division sister chromatids are disjoined.
                     Recombination leads to genetic exchange as indicated by the different coloring of the chromosomes. Note: during oocyte meiosis,
                     one of the two end products of each division gets extruded as a polar body.
                  </p>
               </div>
            </div>
            <p>Despite the fundamental importance of meiosis in sexual reproduction, many basic questions about the process and the underlying
               mechanisms remain unanswered. Over the past twenty years, <span class="italic">Caenorhabditis elegans</span> has emerged as a major model organism for investigating meiotic mechanisms. Several features of <span class="italic">C. elegans</span> biology have contributed to this emergence. The worm germ line is especially amenable to high-resolution cytological analysis
               of chromosome and nuclear organization in the context of whole mount preparations that preserve 3D nuclear architecture. Importantly,
               each germ line contains a complete time course of meiosis, with nuclei organized in a temporal/spatial gradient corresponding
               to the stages of meiotic prophase (<a href="#figure1">Figure 1</a> and <a href="#figure2">Figure 2</a>). Further, the chromosomal basis of sex determination can be exploited to identify meiotic mutants on the basis of sex chromosome
               missegregation. The availability of worms expressing GFP::histone has also made it possible to screen for mutants based on
               lack of chiasmata connecting homologs at the end of meiotic prophase. Mutant hermaphrodites can still produce a few percent
               euploid survivors even if all six chromosome pairs lack chiasmata, a feature that has greatly facilitated analysis of meiotic
               mutants. Germline mRNA expression profiles have accelerated identification of molecular defects associated with meiotic mutants
               and have provided a basis for identification of candidate genes tested for meiotic roles in targeted RNAi screens. Finally,
               <span class="italic">C. elegans</span> has a robust tradition of investigating the genetic behavior of chromosome rearrangements, which has led to the discovery
               of <span class="italic">cis</span>-acting chromosome features that govern meiotic chromosome behavior.
            </p>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="meiosis_fig2.jpg"><img src="meiosis_fig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b><span class="bold"><strong>Figure 2: The hermaphrodite germ line contains a complete time course of meiotic prophase.</strong></span>&nbsp;</b>Projection of a dissected, three-dimensionally intact hermaphrodite germ line stained with DAPI to visualize chromatin. Stages
                     of meiotic prophase and magnification of a representative nucleus of each meiotic prophase stage are depicted below the germ
                     line. White lines indicate boundaries between germline regions containing nuclei at indicated stages of meiotic prophase.
                     Left end of the germ line contains a compartment of mitotically proliferating cells followed by nuclei undergoing meiotic
                     DNA replication (meiotic entrance). The transition zone contains nuclei in leptotene/zygotene, with chromatin tightly clustered
                     in one half of the nucleus, giving a characteristic half-moon shape appearance. As nuclei move from the transition zone into
                     the pachytene region, paired and aligned homologous chromosomes can be seen distributed around the periphery of each nucleus.
                     Diplotene is characterized by chromatin condensation; at this stage, bivalents (pairs of homologous of chromosomes attached
                     by chiasmata) become visible as 6 separate chromatin structures. Chromosome condensation continues during diakinesis, when
                     the 6 bivalents can be easily visualized within the nucleus. Nuclei move down the gonad tube, transitioning between the stages
                     described above with a speed of approximately one cell row per hour. Inset nuclei from the various stages of prophase I are
                     shown to scale with each other.
                  </p>
               </div>
            </div>
            <p>This chapter begins with a &#8220;parts list&#8221; of meiotic machinery components identified in <span class="italic">C. elegans</span>, followed by a description of the events of meiosis that integrates information about the roles of these components. We will
               focus on oocyte meiosis, as later stages of prophase in oocytes are cytologically more accessible than during spermatocyte
               meiosis. We will discuss the interrelated processes of chromosome movement and pairing (<a href="#sec3" title="3.&nbsp;Chromosome pairing in prophase of meiosis I">Section 3</a>), and the protein complexes that drive the dramatic changes seen in chromosome structure during meiosis (<a href="#sec4" title="4.&nbsp;Meiotic chromosome structure">Section 4</a>; also see <a href="http://www.wormbook.org/chapters/www_germlinechromatin/germlinechromatin.html" target="_top">Germline chromatin</a>). We will then discuss the process of meiotic recombination (<a href="#sec5" title="5.&nbsp;Meiotic recombination">Section 5</a>). Following chiasma formation, late pachytene bivalents differentiate around the site of the chiasma in preparation for subsequent
               segregation (<a href="#sec6" title="6.&nbsp;Bivalent differentiation">Section 6</a>). We round off the chapter with an overview of surveillance mechanisms that monitor meiotic events for proper completion
               (<a href="#sec7" title="7.&nbsp;Surveillance mechanisms during prophase I">Section 7</a>) and a description of the events that occur during meiotic chromosome segregation (<a href="#sec8" title="8.&nbsp;Chromosome segregation during the meiotic divisions">Section 8</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;Chromosome pairing in prophase of meiosis I
                     </h2>
                  </div>
               </div>
            </div>
            <p>Pairing of homologous chromosomes, which occurs during the leptotene and zygotene stages of prophase I (transition zone of
               the germ line, <a href="#figure1">Figure 1</a>), is a crucial event in meiosis. During this highly dynamic process, the two homologous copies of each chromosome find each
               other within the nucleus through an active search process that enables chromosomes to distinguish &#8220;self versus non-self&#8221; and
               assume a side-by-side alignment. This pairing is a necessary prerequisite for CO formation, and thus successful completion
               of meiosis. In many organisms pairing is mediated by tethering chromosome ends to the nuclear periphery where they become
               attached to cytoplasmic motor proteins via SUN/KASH domain protein complexes that span the nuclear envelope. The motor proteins
               then drive chromosome movement that is essential for the timely completion of pairing. In <span class="italic">C. elegans</span> <span class="italic">cis</span>-acting sequences near one end of each chromosome, rather than the telomere itself, assemble a nucleoprotein complex that
               tethers the chromosome ends to the nuclear envelope. These events occur in the transition zone and coincide with chromatin
               adopting a special configuration, with chromosomes pushed to one side of the nucleus opposite the nucleolus, giving the chromatin
               a half-moon shape (<a href="#figure2">Figure 2</a>). Pairing is complete by exit from the transition zone (<a href="#bib89">MacQueen and Villeneuve, 2001</a>). Mutations that disrupt chromosome movements also result in loss of this nuclear reorganization during leptotene/zygotene
               (<a href="#bib89">MacQueen and Villeneuve, 2001</a>; <a href="#bib32">Couteau et al., 2004</a>; <a href="#bib33">Couteau and Zetka, 2005</a>; <a href="#bib94">Martinez-Perez and Villeneuve, 2005</a>; <a href="#bib116">Penkner et al., 2007b</a>). The process of pairing is normally coupled with SC assembly between homologs (<a href="#sec4" title="4.&nbsp;Meiotic chromosome structure">Section 4</a>). However, the chromosome alignment process is genetically separable from synapsis, although both take place along the entire
               length of the chromosome (<a href="#bib114">Pasierbek et al., 2001</a>; <a href="#bib87">MacQueen et al., 2002</a>; <a href="#bib106">Nabeshima et al., 2011</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3.1"></a>3.1.&nbsp;<span class="italic">Cis-</span>acting sequences promote pairing
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Each <span class="italic">C. elegans</span> chromosome has a localized <span class="italic">cis</span>-acting region near one end that plays crucial roles in meiotic chromosome behavior. These regions have been termed HRRs (homology
                  recognition regions) or PCs (pairing centers) (<a href="#bib134">Rosenbluth and Baillie, 1981</a>; <a href="#bib133">Rose et al., 1984</a>; <a href="#bib95">McKim et al., 1988</a>; <a href="#bib59">Herman and Kari, 1989</a>; <a href="#bib169">Villeneuve, 1994</a>). These HRRs/PCs (hereafter, PCs) are each comprised of repetitive DNA sequences (<a href="#bib178">Sanford and Perry, 2001</a>; <a href="#bib121">Phillips et al., 2009</a>). The PC ends of chromosomes are in close proximity to the nuclear envelope in the transition zone, when the chromosome pairing
                  process is initiated (<a href="#figure3">Figure 3</a>) (<a href="#bib49">Goldstein, 1982</a>; <a href="#bib88">MacQueen et al., 2005</a>). Chromosomes lacking PCs display severe defects in pairing and synapsis (<a href="#bib88">MacQueen et al., 2005</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="meiosis_fig3.jpg"><img src="meiosis_fig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b><span class="bold"><strong>Figure 3: Chromosome movements and homolog pairing during early prophase I. (A)</strong></span>.&nbsp;</b>Model depicting the attachment of chromosomes to the nuclear envelope and their movement in transition zone nuclei. Model
                        shows two pairs of homologous chromosomes (one pair depicted in black and a second pair depicted in brown) with their PC-bearing
                        region associated with the nuclear envelope. See key on figure for molecular components. Inset on the left hand side of the
                        panel shows a detailed model of the attachment site of a single chromosome. When chromosomes enter the transition zone they
                        are connected to the nuclear envelope via interactions between PC-associated proteins (PC proteins) and a nuclear envelope-spanning
                        SUN/KASH protein complex. Microtubule-mediated forces in the cytoplasm result in movement of these SUN/KASH complexes, causing
                        movement of attached chromosome ends. Chromosome ends move vigorously, and tend to come together into local clusters (presumably
                        to allow homology between adjacent chromosomes to be assessed). When homologs have identified each other and initiated pairing,
                        the SC (depicted as thin black lines connecting the homologs) is established. Both synapsed and unsynapsed chromosomes continue
                        to move until all chromosomes are synapsed. <span class="bold"><strong>(B)</strong></span> Diagram showing the position of the PC region (grey boxes) on each chromosome and the PC-binding proteins that load to each
                        chromosome.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3.2"></a>3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In <span class="italic">C. elegans</span> meiosis, each PC is bound by one of four C2H2 zinc finger proteins (<a href="http://www.wormbase.org/db/get?name=WP:CE36715;class=Protein" target="_blank"><a href="http://www.wormbase.org/db/get?name=HIM-8;class=Strain" target="_blank">HIM-8</a></a>, <a href="http://www.wormbase.org/db/get?name=WP:CE42593;class=Protein" target="_blank">ZIM-1</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE29521;class=Protein" target="_blank">ZIM-2</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE45931;class=Protein" target="_blank">ZIM-3</a>) (<a href="#figure3">Figure 3</a>). The genes encoding the HIM/ZIM proteins are found within a single operon, ensuring their coordinated germline expression
                  (<a href="#bib122">Phillips et al., 2005</a>; <a href="#bib120">Phillips and Dernburg, 2006</a>). PC repeats are sufficient to recruit ZIM proteins as shown by the injection of plasmids carrying synthetic repeats of the
                  PC motifs (<a href="#bib178">Sanford and Perry, 2001</a>; <a href="#bib121">Phillips et al., 2009</a>). Furthermore, PC proteins are required both for aligning homologs and for homologous synapsis of the specific chromosomes
                  to which they are bound (<a href="#bib122">Phillips et al., 2005</a>; <a href="#bib120">Phillips and Dernburg, 2006</a>; <a href="#bib121">Phillips et al., 2009</a>; <a href="#bib55">Harper et al., 2011</a>; <a href="#bib77">Labella et al., 2011</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE36715;class=Protein" target="_blank"><a href="http://www.wormbase.org/db/get?name=HIM-8;class=Strain" target="_blank">HIM-8</a></a> appears to have an additional role(s) as it is required for elongation of the X chromosomes in transition zone nuclei, where
                  all chromosomes start to occupy extended territories, a process that may facilitate their lengthwise alignment (<a href="#bib106">Nabeshima et al., 2011</a>).
               </p>
               <p>Although functionally similar, the X and autosome PCs differ with respect to the persistence and regulation of nuclear envelope
                  attachment and association of their PC-binding proteins. <a href="http://www.wormbase.org/db/get?name=WP:CE36715;class=Protein" target="_blank"><a href="http://www.wormbase.org/db/get?name=HIM-8;class=Strain" target="_blank">HIM-8</a></a> localizes to X chromosome PCs from premeiotic stages through late pachytene (<a href="#bib122">Phillips et al., 2005</a>). Autosomal PC proteins, by contrast, are detected at PCs primarily from leptotene to early pachytene (<a href="http://www.wormbase.org/db/get?name=WP:CE42593;class=Protein" target="_blank">ZIM-1</a> on chromosomes II and III, <a href="http://www.wormbase.org/db/get?name=WP:CE29521;class=Protein" target="_blank">ZIM-2</a> on chromosome V and <a href="http://www.wormbase.org/db/get?name=WP:CE45931;class=Protein" target="_blank">ZIM-3</a> on chromosomes I and IV). In contrast to <a href="http://www.wormbase.org/db/get?name=WP:CE36715;class=Protein" target="_blank"><a href="http://www.wormbase.org/db/get?name=HIM-8;class=Strain" target="_blank">HIM-8</a></a>, concentration of the autosomal PC proteins at PCs depends on the <a href="http://www.wormbase.org/db/get?name=WP:CE27297;class=Protein" target="_blank">CHK-2</a> protein kinase (<a href="#bib122">Phillips et al., 2005</a>; <a href="#bib120">Phillips and Dernburg, 2006</a>), a master regulator of early events of prophase (<a href="#bib89">MacQueen and Villeneuve, 2001</a>).
               </p>
               <p>The PC nucleoprotein complexes act as recruitment sites for polo kinase, <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> (or <a href="http://www.wormbase.org/db/get?name=WP:CE26649;class=Protein" target="_blank">PLK-1</a>, if <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> is absent). <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> induces structural reorganization of the nuclear envelope (<a href="#bib55">Harper et al., 2011</a>; <a href="#bib77">Labella et al., 2011</a>): the inner and outer nuclear envelope proteins <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> relocate into pronounced aggregates corresponding to the sites where PCs localize to the nuclear envelope (<a href="#figure3">Figure 3</a>) (<a href="#bib117">Penkner et al., 2009</a>; <a href="#bib141">Sato et al., 2009</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> form a functional SUN/KASH protein&#8211;protein interaction module, broadly conserved among eukaryotes (<a href="#bib91">Malone et al., 2003</a>; <a href="#bib44">Fridkin et al., 2004</a>; <a href="#bib116">Penkner et al., 2007b</a>; <a href="#bib102">Minn et al., 2009</a>), that spans the nuclear membranes and connects chromosomes to the cytoskeleton.
               </p>
               <p>If the SUN/KASH interaction is abrogated, <a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> retention at the outer nuclear envelope is lost (<a href="#bib91">Malone et al., 2003</a>; <a href="#bib116">Penkner et al., 2007b</a>) despite the assembly of <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> aggregates with PCs at the inner nuclear envelope (<a href="#bib117">Penkner et al., 2009</a>). Therefore, the trigger for <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> aggregation is transmitted from the nucleus to the cytoplasm where <a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> mirrors <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> aggregates. <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> aggregate formation is independent of DSBs, recombination, pairing, and synapsis, but requires <a href="http://www.wormbase.org/db/get?name=WP:CE27297;class=Protein" target="_blank">CHK-2</a> (<a href="#bib117">Penkner et al., 2009</a>; <a href="#bib141">Sato et al., 2009</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> aggregates at autosomal PC attachment sites are found in leptotene/zygotene (TZ). In contrast the <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> aggregates around the X chromosome PC persist throughout early pachytene. In mid- and late pachytene, most nuclei lack <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> aggregates, despite the presence of a <a href="http://www.wormbase.org/db/get?name=WP:CE36715;class=Protein" target="_blank"><a href="http://www.wormbase.org/db/get?name=HIM-8;class=Strain" target="_blank">HIM-8</a></a> focus at the nuclear envelope (<a href="#bib122">Phillips et al., 2005</a>; <a href="#bib117">Penkner et al., 2009</a>; <a href="#bib141">Sato et al., 2009</a>). Disappearance of <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> aggregates correlates with the establishment of full synapsis and relocalization of <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> from PCs to the SC (<a href="#bib55">Harper et al., 2011</a>; <a href="#bib77">Labella et al., 2011</a>). The recruitment of <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> to PCs and the subsequent formation of dynamic <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> aggregates is essential to ensure faithful SC assembly, as mutants defective in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006311;class=Gene" target="_blank">sun-1</a></span>, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006997;class=Gene" target="_blank">zyg-12</a></span>, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004043;class=Gene" target="_blank">plk-2</a>,</span> or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene" target="_blank">him-8</a>/zim&#916;</span> (lacking all PC-binding proteins) display aberrant synapsis (<a href="#bib116">Penkner et al., 2007b</a>; <a href="#bib141">Sato et al., 2009</a>; <a href="#bib55">Harper et al., 2011</a>; <a href="#bib77">Labella et al., 2011</a>; <a href="#bib176">Woglar et al., 2013</a>).
               </p>
               <p>PCs play a prominent role in promoting homolog recognition, but several lines of evidence suggest that they cannot be the
                  sole determinants of chromosome identity. Some PC proteins (<a href="http://www.wormbase.org/db/get?name=WP:CE42593;class=Protein" target="_blank">ZIM-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE45931;class=Protein" target="_blank">ZIM-3</a>) localize to more than one chromosome; despite this, nonhomologous pairing between different <a href="http://www.wormbase.org/db/get?name=WP:CE42593;class=Protein" target="_blank">ZIM-1</a> or ZIM-3-binding PCs is not detected in wild-type animals (<a href="#bib121">Phillips et al., 2009</a>). Moreover, pre-synaptic alignment has been demonstrated along the entire length of chromosomes (<a href="#bib106">Nabeshima et al., 2011</a>). The chromodomain protein <a href="http://www.wormbase.org/db/get?name=WP:CE20213;class=Protein" target="_blank">MRG-1</a>, present on the autosomes, plays a role in the non-PC mediated homolog alignment and its absence leads to defects in homolog
                  alignment and synapsis of non-PC regions (<a href="#bib39">Dombecki et al., 2011</a>). Mutations in the gene encoding the serine/threonine phosphatase <a href="http://www.wormbase.org/db/get?name=WP:CE23041;class=Protein" target="_blank">PPH-4.1</a> cause defective autosomal chromosome pairing and synapsis between non-homologous chromosomes. The relevant targets for <a href="http://www.wormbase.org/db/get?name=WP:CE23041;class=Protein" target="_blank">PPH-4.1</a> are still unknown (<a href="#bib142">Sato-Carlton et al., 2014</a>). In <span class="italic">C. elegans</span>, homologous recombination is not required for establishment of homolog alignment since <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004985;class=Gene" target="_blank">spo-11</a></span> mutants, in which meiotic DSBs are not formed, are proficient to pair and synapse with their homologs (<a href="#bib37">Dernburg et al., 1998</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3.3"></a>3.3.&nbsp;PC movements promote efficient homolog pairing
                        </h3>
                     </div>
                  </div>
               </div>
               <p>PCs display a highly dynamic behavior during leptotene/zygotene (<a href="#bib117">Penkner et al., 2009</a>; <a href="#bib141">Sato et al., 2009</a>; <a href="#bib11">Baudrimont et al., 2010</a>); in contrast, other chromosomal regions are relatively static (<a href="#bib178">Wynne et al., 2012</a>). During leptotene/zygotene, PCs have a strong tendency to come together into transient local clusters. From there they may
                  continue to move in groups/pairs or dissociate and resume independent movement (<a href="#figure3">Figure 3</a>). Tracking PC movements showed that they comprise both small short-range tracks and long saltatory trajectories (up to 2
                  &micro;m) with an average track length of 0.5 &micro;m and an average speed of 0.19 &micro;m/sec (<a href="#bib117">Penkner et al., 2009</a>; <a href="#bib141">Sato et al., 2009</a>; <a href="#bib11">Baudrimont et al., 2010</a>; <a href="#bib178">Wynne et al., 2012</a>). Chromosomes progressively pair and synapse while progressing through leptotene/zygotene. In spite of this, the characteristics
                  of chromosome movement remain essentially the same (<a href="#bib11">Baudrimont et al., 2010</a>; <a href="#bib178">Wynne et al., 2012</a>), suggesting that PC-mediated chromosome end movement does not cease once synapsis of a given chromosome pair is achieved.
               </p>
               <p>Chromosome movement in the gonad relies on microtubules (<a href="#bib141">Sato et al., 2009</a>; <a href="#bib178">Wynne et al., 2012</a>). Further, <a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> recruits components of the dynein motor complex to the cytoplasmic side of the PC attachments (<a href="#bib141">Sato et al., 2009</a>; <a href="#bib78">Labrador et al., 2013</a>) to mobilize chromosome ends in leptotene/zygotene and to facilitate pairing and synapsis. Dynein knock-down, ATP depletion,
                  or a specific allele affecting the mitochondria-localized <a href="http://www.wormbase.org/db/get?name=WP:CE19499;class=Protein" target="_blank">SPD-3</a> protein (with a likely role in energy production for dynein function) consistently result in reduced pairing and in synapsis
                  with non-homologs (<a href="#bib78">Labrador et al., 2013</a>). However, a direct role of dynein-driven chromosome movements in licensing SC assembly has also been proposed, since severe
                  dynein knockdown inhibited SC assembly (<a href="#bib141">Sato et al., 2009</a>). <a href="#bib141">Sato et al. (2009)</a> further propose that dynein functions to oppose inappropriate chromosome interactions, enabling dissociation of non-homologous
                  chromosomes.
               </p>
               <p>Upon completion of pairing and synapsis, meiocytes enter early pachytene and chromosome clustering is loosened but not completely
                  abrogated. <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> aggregates dissolve around the autosome PCs, whereas they persist longer at the X chromosome PCs, which remain mobile in
                  early pachytene (<a href="#bib117">Penkner et al., 2009</a>; <a href="#bib141">Sato et al., 2009</a>; <a href="#bib178">Wynne et al., 2012</a>). The study of the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00022390;class=Gene" target="_blank">him-19</a></span> mutant revealed that regulation of chromosome end mobilization deteriorates with progressive age in this mutant (<a href="#bib117">Penkner et al., 2009</a>; <a href="#bib165">Tang et al., 2010</a>). The decreased movement and associated increase in nondisjunction with age might suggest an underlying cause for age-related
                  chromosomal abnormalities in systems where meiosis continues throughout one's lifetime, such as during human spermatogenesis.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Meiotic chromosome structure
                     </h2>
                  </div>
               </div>
            </div>
            <p>The meiotic program involves dramatic changes in chromosome structure, which are driven by the regulated association and dissociation
               of different protein complexes from chromosomes. Some of these protein complexes represent meiosis-specific elaborations upon
               structures that occur in mitotically cycling cells. Critically important is the loading of multiple, partially redundant cohesin
               complexes (<a href="#sec4.1" title="4.1.&nbsp;Meiosis-specific cohesin complexes">Section 4.1</a>), which tether sister chromatids together and serve as the basis for assembly of axial elements containing four HORMA-domain
               proteins (<a href="#sec4.2" title="4.2.&nbsp;HORMA-domain proteins">Section 4.2</a>). During pachytene, the axial elements of paired homologous chromosomes are connected by yet another meiosis-specific structure,
               the SC (<a href="#sec4.3" title="4.3.&nbsp;The SC">Section 4.3</a>). Proper assembly of the SC depends on axis assembly, and in turn is required for regulated formation of meiotic COs between
               homologs (and thus, proper chromosome segregation) (<a href="#sec5" title="5.&nbsp;Meiotic recombination">Section 5</a> and <a href="#sec6" title="6.&nbsp;Bivalent differentiation">Section 6</a>). The correct execution of these structural changes is essential to promote pairing, CO formation, and chromosome segregation.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4.1"></a>4.1.&nbsp;Meiosis-specific cohesin complexes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The acquisition of meiosis-specific chromosome features starts with DNA replication, during which sister chromatid cohesion
                  must be established. Meiotic S-phase is twice as long as that of cells in the mitotic compartment of the germ line (<a href="#bib66">Jaramillo-Lambert et al., 2007</a>), perhaps reflecting the time required to adopt the unique chromosome configurations required for meiotic success. <span class="italic">C. elegans</span> expresses three different meiosis-specific cohesin complexes differing in their kleisin subunit: <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a>, and the nearly identical and functionally redundant <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE27409;class=Protein" target="_blank">COH-4</a> (<a href="#bib114">Pasierbek et al., 2001</a>; <a href="#bib149">Severson et al., 2009</a>; <a href="#bib150">Severson and Meyer, 2014</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> is present during meiotic DNA replication&#8212;it provides cohesion independently of DSBs, and its loading onto chromosomes depends
                  on the HORMA-domain protein <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a>, and on <a href="http://www.wormbase.org/db/get?name=WP:CE23033;class=Protein" target="_blank">TIM-1</a> (ortholog of the TIMELESS clock protein) (<a href="#bib114">Pasierbek et al., 2001</a>; <a href="#bib20">Chan et al., 2003</a>; <a href="#bib149">Severson et al., 2009</a>; <a href="#bib150">Severson and Meyer, 2014</a>). In contrast, <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>/4 complexes are only detected in meiotic nuclei following the completion of S-phase, their ability to provide cohesion requires
                  DSBs, and their loading is independent of <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE23033;class=Protein" target="_blank">TIM-1</a> (<a href="#bib150">Severson and Meyer, 2014</a>). Loss of cohesion, evidenced by extensive separation of sister chromatids in diakinesis oocytes, is observed in mutants
                  lacking <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE27409;class=Protein" target="_blank">COH-4</a>, but not in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004333;class=Gene" target="_blank">rec-8</a></span> single or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000592;class=Gene" target="_blank">coh-3</a> <a href="http://www.wormbase.org/db/get?name=WBGene00021560;class=Gene" target="_blank">coh-4</a></span> double mutants (<a href="#bib149">Severson et al., 2009</a>; <a href="#bib168">Tzur et al., 2012</a>; <a href="#bib150">Severson and Meyer, 2014</a>), demonstrating that both <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>/4 complexes contribute to sister chromatid cohesion.
               </p>
               <p>Cohesin loading to meiotic chromosomes is not only essential for sister chromatid cohesion, but also for successful completion
                  of key meiotic prophase events. <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>/4 cohesin complexes associate with axial elements in the transition zone of the germ line, where chromosomes become dramatically
                  elongated compared to mitotic cells (<a href="#bib114">Pasierbek et al., 2001</a>; <a href="#bib56">Hayashi et al., 2007</a>; <a href="#bib106">Nabeshima et al., 2011</a>; <a href="#bib150">Severson and Meyer, 2014</a>). Assembly of axial elements is fully dependent on cohesin loading, as depletion of SMC-1 or <a href="http://www.wormbase.org/db/get?name=WP:CE05664;class=Protein" target="_blank">SCC-3</a> (which are common to all meiotic cohesin complexes), lack of all three meiotic kleisins, or lack of the cohesin loading factor
                  SCC-2, prevent association of HORMA-domain proteins with axial elements and cause severe meiotic defects that prevent CO formation
                  (<a href="#bib113">Pasierbek et al., 2003</a>; <a href="#bib172">Wang et al., 2003</a>; <a href="#bib50">Goodyer et al., 2008</a>; <a href="#bib149">Severson et al., 2009</a>; <a href="#bib85">Lightfoot et al., 2011</a>). Loading of different meiosis-specific cohesin complexes is also required for the assembly of axial elements in mice (reviewed
                  in <a href="#bib96">McNicoll et al., 2013</a>). Furthermore, meiotic defects, including impaired chiasma formation, are present in mutants that display a partial reduction
                  in the overall amount of cohesin associated with axial elements, such as in worms carrying a hypomorphic <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004873;class=Gene" target="_blank">smc-3</a></span> allele or mutants lacking the axis-associated <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a> protein (<a href="#bib12">Baudrimont et al., 2011</a>; <a href="#bib168">Tzur et al., 2012</a>). CO formation is impaired in mutants lacking either <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> or <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>/4, demonstrating that both types of cohesin play important roles during meiotic recombination (<a href="#bib114">Pasierbek et al., 2001</a>; <a href="#bib56">Hayashi et al., 2007</a>; <a href="#bib149">Severson et al., 2009</a>). Moreover, <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>/4 cohesin also play different roles in ensuring SC assembly between homologous chromosomes, since SC assembly is greatly
                  reduced in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000592;class=Gene" target="_blank">coh-3</a> <a href="http://www.wormbase.org/db/get?name=WBGene00021560;class=Gene" target="_blank">coh-4</a></span> double mutants and SC installation is thought to occur between sister chromatids in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004333;class=Gene" target="_blank">rec-8</a></span> mutants (<a href="#bib149">Severson et al., 2009</a>; <a href="#bib150">Severson and Meyer, 2014</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4.2"></a>4.2.&nbsp;HORMA-domain proteins
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Proteins containing a HORMA domain, a protein motif also found in spindle-assembly checkpoint protein MAD2 (<a href="#bib7">Aravind and Koonin, 1998</a>), play fundamental roles in establishing meiotic chromosome structure. The <span class="italic">C. elegans</span> genome encodes four of these proteins&#8212;HIM-3, <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a>, with unique and redundant functions. Collectively, these HORMA-domain proteins regulate almost all key events of meiosis,
                  ranging from homolog recognition to the correct release of sister chromatid cohesion during the meiotic divisions.
               </p>
               <p><a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2 consist mostly of a HORMA domain flanked by short N- and C-terminal regions, while <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> contains a long C-terminal tail that contains four docking sites for <a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a> and two for <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2 (<a href="#bib74">Kim et al., 2014</a>). Therefore, <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> is essential for the assembly of axial elements. In addition, <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2 can also be recruited to axial elements by binding directly to the C-terminal tail of <a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a> (<a href="#bib74">Kim et al., 2014</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> is also required to promote loading of REC-8-containing cohesin complexes (<a href="#bib149">Severson et al., 2009</a>) and to mediate the formation of DSBs (<a href="#bib50">Goodyer et al., 2008</a>), both of which are independent of its requirement to recruit <a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2 (<a href="#bib74">Kim et al., 2014</a>). Interestingly, <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> is the only meiotic HORMA-domain protein that localizes to chromatin in the mitotic compartment of the germ line (<a href="#bib56">Hayashi et al., 2007</a>; <a href="#bib50">Goodyer et al., 2008</a>). Finally, <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> is required for the acquisition of posttranslational modifications on axis-associated chromatin (<a href="#bib34">Couteau and Zetka, 2011</a>). Thus, <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> is not only essential to promote axial element assembly, but may also orchestrate changes in chromatin structure for multiple
                  meiotic prophase events.
               </p>
               <p><a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> are both required to ensure homolog recognition and to promote faithful SC assembly; as a consequence, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001862;class=Gene" target="_blank">him-3</a></span> and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002032;class=Gene" target="_blank">htp-1</a></span> null mutants display severe impairment in CO formation (<a href="#bib185">Zetka et al., 1999</a>; <a href="#bib33">Couteau and Zetka, 2005</a>; <a href="#bib94">Martinez-Perez and Villeneuve, 2005</a>). Although the mechanisms by which these two proteins promote homolog pairing are not well understood, <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> has been proposed to participate in two checkpoint-like mechanisms: the first makes initiation of SC assembly contingent
                  on successful homology recognition, while the second prolongs the homology search process until homolog interactions are stabilized
                  by SC loading (<a href="#bib33">Couteau and Zetka, 2005</a>; <a href="#bib94">Martinez-Perez and Villeneuve, 2005</a>; <a href="#bib153">Silva et al., 2014</a>). A mutant form of <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> that fails to associate with axial elements still supports the delayed exit from early prophase triggered by synapsis defects,
                  suggesting that nuclear soluble <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> may participate in the quality control of SC assembly (<a href="#bib153">Silva et al., 2014</a>). An important difference between <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002032;class=Gene" target="_blank">htp-1</a></span> and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001862;class=Gene" target="_blank">him-3</a></span> null mutants is that SC assembly is dramatically reduced in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001862;class=Gene" target="_blank">him-3</a></span> mutants, whereas lack of <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> results in high levels of SC assembly between non-homologous chromosomes. Non-homologous synapsis is also observed in mutants
                  expressing hypomorphic alleles of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001862;class=Gene" target="_blank">him-3</a></span> (<a href="#bib32">Couteau et al., 2004</a>), demonstrating that both <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a> have regulatory roles in SC assembly. Removal of <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a> (which shares 82% identity at the amino acid level with <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>) from <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002032;class=Gene" target="_blank">htp-1</a></span> mutants results in a dramatic reduction in SC assembly, demonstrating that <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a> can promote homology-independent SC assembly (<a href="#bib33">Couteau and Zetka, 2005</a>; <a href="#bib93">Martinez-Perez et al., 2008</a>). However, the analysis of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002033;class=Gene" target="_blank">htp-2</a></span> single mutants failed to identify obvious meiotic defects; thus, <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a> function appears largely redundant with that of <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> (<a href="#bib93">Martinez-Perez et al., 2008</a>).
               </p>
               <p>In addition to promotion of pairing and synapsis between homologs, <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a> also directly affect meiotic recombination, presumably by making the sister chromatid inaccessible for repair, thereby encouraging
                  repair off the homolog (<a href="#bib32">Couteau et al., 2004</a>; <a href="#bib94">Martinez-Perez and Villeneuve, 2005</a>) (see also <a href="#sec5.2.4" title="5.2.4.&nbsp;Partner choice">Section 5.2.4</a> and <a href="#sec5.5" title="5.5.&nbsp;Interplay between recombination and chromosome structure">Section 5.5</a>). <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002032;class=Gene" target="_blank">htp-1</a></span> mutants show a reduction in <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> foci (see <a href="#sec5" title="5.&nbsp;Meiotic recombination">Section 5</a>) which can be explained by a reduction in the overall number of DSBs formed and/or faster DSB turnover (<a href="#bib33">Couteau and Zetka, 2005</a>; <a href="#bib94">Martinez-Perez and Villeneuve, 2005</a>). Finally, <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a> participate in the two-step release of sister chromatid cohesion during the meiotic divisions, where they act to ensure that
                  a subset of REC-8-containing cohesin remains associated with chromosomes during the first meiotic division (<a href="#bib93">Martinez-Perez et al., 2008</a>; <a href="#bib149">Severson et al., 2009</a>). Of note, the connection between HORMA-domain proteins and the regulation of <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> complexes goes beyond the release of sister chromatid cohesion in the meiotic divisions, since <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002032;class=Gene" target="_blank">htp-1</a> <a href="http://www.wormbase.org/db/get?name=WBGene00002033;class=Gene" target="_blank">htp-2</a></span> double mutants, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001862;class=Gene" target="_blank">him-3</a></span> mutants, and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004333;class=Gene" target="_blank">rec-8</a></span> mutants display premature centriole disengagement during meiosis II in spermatogenesis (<a href="#bib145">Schvarzstein et al., 2013</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4.3"></a>4.3.&nbsp;The SC
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The most prominent, and highly conserved, structural feature of meiotic prophase is the SC. When observed by electron microscopy,
                  the SC appears as a tripartite, ladder-like structure consisting of a central region that contains transverse filaments that
                  bridge the axial elements of paired homologs (<a href="#bib87">MacQueen et al., 2002</a>; <a href="#bib27">Colai&aacute;covo, 2006</a>) (<a href="#figure4">Figure 4</a>). Although structurally similar, with long coiled-coil domains, the major components of the SC in different organisms are
                  poorly conserved at the primary amino acid level. In <span class="italic">C. elegans</span>, four components of the central region of the SC have been identified: <a href="http://www.wormbase.org/db/get?name=WP:CE33925;class=Protein" target="_blank">SYP-1</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE17456;class=Protein" target="_blank">SYP-2</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE41238;class=Protein" target="_blank">SYP-3</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE23830;class=Protein" target="_blank">SYP-4</a>. Analysis of the corresponding single mutants reveals that these four proteins are interdependent for their loading onto
                  chromosomes; consequently, SC assembly is fully abrogated in each one of the single mutants (<a href="#bib87">MacQueen et al., 2002</a>; <a href="#bib28">Colai&aacute;covo et al., 2003</a>; <a href="#bib156">Smolikov et al., 2007b</a>; <a href="#bib158">Smolikov et al., 2009</a>). SC assembly is also abrogated in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002034;class=Gene" target="_blank">htp-3</a></span> mutants and in mutants lacking all three meiotic kleisins (<a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE27409;class=Protein" target="_blank">COH-4</a>) (<a href="#bib50">Goodyer et al., 2008</a>; <a href="#bib149">Severson et al., 2009</a>). Investigation of the interactions among the four SYP proteins by immunoprecipitation, yeast two-hybrid experiments, and
                  immuno-electron microscopy, suggests that <a href="http://www.wormbase.org/db/get?name=WP:CE33925;class=Protein" target="_blank">SYP-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE17456;class=Protein" target="_blank">SYP-2</a> occupy a central position within the SC, while <a href="http://www.wormbase.org/db/get?name=WP:CE41238;class=Protein" target="_blank">SYP-3</a> interacts directly with axis components, and <a href="http://www.wormbase.org/db/get?name=WP:CE23830;class=Protein" target="_blank">SYP-4</a>, the largest of these proteins, interacts with <a href="http://www.wormbase.org/db/get?name=WP:CE41238;class=Protein" target="_blank">SYP-3</a> and reaches into the center of the SC (<a href="#bib143">Schild-Pr&uuml;fert et al., 2011</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="meiosis_fig4.jpg"><img src="meiosis_fig4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b><span class="bold"><strong>Figure 4. Structure of the SC. (A)</strong></span>.&nbsp;</b>TEM images of nuclei from the late pachytene regions of a wild-type germ line (reproduced with permission from <a href="#bib28">Colai&aacute;covo et al., 2003</a>). (A&#8217;) Equatorial section of a wild-type pachytene nucleus containing a very long continuous stretch of SC, visible as a
                        zipper-like track flanked by electron-dense patches of chromatin; the large dark body centered in the upper half of the panel
                        is the nucleolus. (B&#8217;) Detail of SC structure from a different wild-type nucleus. Scale bars equal 500 nm. <span class="bold"><strong>(B)</strong></span> Diagram of SC structure assembled between a pair of homologous chromosomes. Note that each homolog (depicted as a pair of
                        closely associated sister chromatids) assembles an axial element (red), and that central region components (green) link together
                        homologous axial elements. <span class="bold"><strong>(C)</strong></span> Projections of pachytene nuclei stained with anti-HIM-3 and anti-SYP-1 antibodies, counterstained with DAPI and imaged with
                        a DeltaVision system. Note that axial elements from the homologs cannot be resolved as two separate structures since the distance
                        between them (&lt;100 nm) is below the resolution of conventional fluorescence microscopy (200 nm). Scale bar equals 5 &micro;m<span class="bold"><strong>.</strong></span></p>
                  </div>
               </div>
               <p>Once assembled, the SC stabilizes the pairwise association of homologous chromosomes and promotes the formation of interhomolog
                  COs (see also <a href="#sec5.2.4" title="5.2.4.&nbsp;Partner choice">Section 5.2.4</a> and <a href="#sec5.5" title="5.5.&nbsp;Interplay between recombination and chromosome structure">Section 5.5</a>) (<a href="#bib87">MacQueen et al., 2002</a>; <a href="#bib28">Colai&aacute;covo et al., 2003</a>; <a href="#bib155">Smolikov et al., 2007a</a>; <a href="#bib158">Smolikov et al., 2009</a>). In addition, the SC is required for normal levels of CO interference (see <a href="#sec5.4" title="5.4.&nbsp;CO interference, assurance, and homeostasis">Section 5.4</a> and <a href="#sec5.5" title="5.5.&nbsp;Interplay between recombination and chromosome structure">Section 5.5</a>) (<a href="#bib57">Hayashi et al., 2010</a>; <a href="#bib84">Libuda et al., 2013</a>) and to promote normal meiotic progression, as evidenced by the persistence of chromosome clustering in <span class="italic">syp</span> mutants (see <a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>) (<a href="#bib87">MacQueen et al., 2002</a>; <a href="#bib28">Colai&aacute;covo et al., 2003</a>).
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4.3.1"></a>4.3.1.&nbsp;Regulation of SC assembly
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>SC assembly appears to be a cooperative process and occurs preferentially in a pairwise fashion, linking the axis of two chromosomes
                     (<a href="#bib88">MacQueen et al., 2005</a>; <a href="#bib57">Hayashi et al., 2010</a>; <a href="#bib103">Mlynarczyk-Evans et al., 2013</a>). In vivo imaging of GFP-tagged <a href="http://www.wormbase.org/db/get?name=WP:CE41238;class=Protein" target="_blank">SYP-3</a> suggests that synapsis of each homolog pair is initiated only once and that PCs are the primary sites for initiation; once
                     initiated, SC extension continues at a constant rate until full synapsis is achieved (<a href="#bib129">Rog and Dernburg, 2015</a>). Whereas the SC normally assembles only between correctly paired homologs, the SC structure itself is indifferent to homology,
                     as seen by the presence of normal-appearing SC structures between non-homologous chromosomes in some meiotic mutants and in
                     worms heterozygous for chromosome rearrangements (<a href="#bib32">Couteau et al., 2004</a>; <a href="#bib33">Couteau and Zetka, 2005</a>; <a href="#bib94">Martinez-Perez and Villeneuve, 2005</a>; <a href="#bib116">Penkner et al., 2007b</a>; <a href="#bib58">Henzel et al., 2011</a>). Thus, SC assembly must be tightly regulated to ensure a productive outcome. Regulation of SYP protein loading involves
                     axial element components <a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> (see <a href="#sec4.2" title="4.2.&nbsp;HORMA-domain proteins">Section 4.2</a>), but proteins that do not become incorporated into the SC structure have also been reported to control the loading of SYP
                     proteins, both in a chromosome-specific and a nucleus-wide manner. For example, the PC-binding <a href="http://www.wormbase.org/db/get?name=WP:CE36715;class=Protein" target="_blank"><a href="http://www.wormbase.org/db/get?name=HIM-8;class=Strain" target="_blank">HIM-8</a></a>/ZIM proteins are required for proper assembly of SC in a chromosome-specific manner (<a href="#bib122">Phillips et al., 2005</a>; <a href="#bib120">Phillips and Dernburg, 2006</a>; <a href="#bib128">Rog and Dernburg, 2013</a>) (see also <a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>), and in the absence of all PC-binding proteins, SC assembly becomes misregulated and non-homologous SC assembly takes place
                     (<a href="#bib55">Harper et al., 2011</a>; <a href="#bib77">Labella et al., 2011</a>). In addition, the chromosome motions characteristic of early prophase have been shown to be important for proper SC assembly:
                     in their absence, SC assembly often occurs between nonhomologous chromosomes (see <a href="#sec3.3" title="3.3.&nbsp;PC movements promote efficient homolog pairing">Section 3.3</a>). Factors that affect the loading of SYP proteins in a nucleus-wide fashion include: the <a href="http://www.wormbase.org/db/get?name=WP:CE27297;class=Protein" target="_blank">CHK-2</a> kinase, required to induce normal levels of SC assembly in early prophase (<a href="#bib89">MacQueen and Villeneuve, 2001</a>; <a href="#bib94">Martinez-Perez and Villeneuve, 2005</a>); <a href="http://www.wormbase.org/db/get?name=WP:CE48020;class=Protein" target="_blank">HAL-2</a>, a nucleoplasmic protein that prevents the inappropriate loading of SYP proteins, which can interfere with the pairing process
                     (<a href="#bib188">Zhang et al., 2012</a>); <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a>, a component of axial elements that targets protein phosphatase 1 and is required for full SC assembly (<a href="#bib168">Tzur et al., 2012</a>); <a href="http://www.wormbase.org/db/get?name=WP:CE31579;class=Protein" target="_blank">CRA-1</a>, the <span class="italic">C. elegans</span> homolog of the non-catalytic subunit of the N-terminal acetyltransferases B, which is required to promote normal loading
                     of SYP proteins (<a href="#bib157">Smolikov et al., 2008</a>; <a href="#bib46">Gao et al., 2015</a>); the CSN/COP9 signalosome, which is required to prevent the formation of extrachromosomal aggregates of SC proteins during
                     early prophase (<a href="#bib17">Brockway et al., 2014</a>); and the AAA+- ATPase <a href="http://www.wormbase.org/db/get?name=WP:CE01547;class=Protein" target="_blank">PCH-2</a>, which localizes to the SC but is dispensable for SC formation and is proposed to prevent synapsis defects by acting as a
                     &#8220;brake&#8221; that limits the rate of SC assembly to allow time for error correction (<a href="#bib38">Deshong et al., 2014</a>).
                  </p>
                  <p>SC disassembly starts in late pachytene in a process that is linked to the progression of meiotic recombination (see <a href="#sec6" title="6.&nbsp;Bivalent differentiation">Section 6</a>). Interestingly, the SC can be locally disassembled in response to exogenous DNA damage (<a href="#bib34">Couteau and Zetka, 2011</a>) or can be completely disassembled from an X chromosome without a CO (<a href="#bib98">Meneely et al., 2012</a>), demonstrating that the SC is a dynamic structure.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;Meiotic recombination
                     </h2>
                  </div>
               </div>
            </div>
            <p>In most organisms, including <span class="italic">C. elegans</span>, successful chromosome segregation during meiosis requires formation of CO recombination events between homologous chromosomes;
               these COs, in conjunction with sister chromatid cohesion, serve to tether homologs together. These connections between homologs,
               in turn, allow correct (bipolar) attachment of homologs to the meiosis I spindle. Release of a subset of cohesin complexes
               at anaphase I then allows separation of homologs, thereby accomplishing the reduction in ploidy necessary for sexual reproduction.
            </p>
            <p>Meiotic recombination is initiated through introduction of DSBs by the enzyme <a href="http://www.wormbase.org/db/get?name=WP:CE49824;class=Protein" target="_blank">SPO-11</a> (<a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>). Repair of these DSBs can result in formation of COs (<a href="#sec5.2" title="5.2.&nbsp;DSB repair by meiosis-specific adaptation of homologous recombination">Section 5.2</a>) and the role of the meiotic machinery is to ensure that each chromosome receives at least one CO (the obligate CO). COs
               do not occur randomly along the chromosomes: each chromosome has a characteristic distribution of COs (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>). Further, the number of COs per chromosome pair, and their positions relative to each other, are not random, reflecting
               several types of regulation (<a href="#sec5.4" title="5.4.&nbsp;CO interference, assurance, and homeostasis">Section 5.4</a>). In addition, chromosome structure affects meiotic recombination (<a href="#sec5.5" title="5.5.&nbsp;Interplay between recombination and chromosome structure">Section 5.5</a>), as does the meiotic cell cycle (<a href="#sec5.6" title="5.6.&nbsp;Window of opportunity for CO formation">Section 5.6</a>). See <a href="#figure5">Figure 5</a> for an overview of the key steps of meiotic recombination.
            </p>
            <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="meiosis_fig5.jpg"><img src="meiosis_fig5_s.jpg" border="2" align="middle" alt=" figure 5"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b><span class="bold"><strong>Figure 5. Model depicting the key events of meiotic recombination.</strong></span>&nbsp;</b>Homologous chromosomes are represented in red and blue, and the two sister chromatids of each homolog are represented as pairs
                     of double stranded DNA molecules indicated by two parallel lines in close proximity. Proteins required for specific steps
                     of meiotic recombination are indicated on the left and right hand side of the diagram. Note that the existence of some of
                     the recombination intermediates represented in this diagram has not been directly demonstrated in <span class="italic">C. elegans</span>, but are inferred from studies in yeast. The molecular events of recombination are represented in temporal progression starting
                     at the top of the diagram with the formation of a DSB by <a href="http://www.wormbase.org/db/get?name=WP:CE49824;class=Protein" target="_blank">SPO-11</a> in a single chromatid of one of the homologs. Resection of DNA ends and <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> loading promote the invasion of a chromatid from the homologous chromosome, the formation of a D loop, and the start of DNA
                     synthesis. These intermediates can be destabilized by the activity of <a href="http://www.wormbase.org/db/get?name=WP:CE15894;class=Protein" target="_blank">RTEL-1</a>, which leads to repair as NCO products, or they can be stabilized by CO promoting factors that promote the formation of double
                     Holliday junctions. Note that although MSH-4, <a href="http://www.wormbase.org/db/get?name=WP:CE26678;class=Protein" target="_blank">MSH-5</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE43850;class=Protein" target="_blank">ZHP-3</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> are all required for CO formation and eventually become associated with CO-fated recombination events, there are clear differences
                     in the timing of loading of these proteins that are not depicted on this model (see Section 4.2.3 for a detailed description).
                     The asymmetric cleavage of double Holliday junctions by different endonucleases promotes the formation of inter-homolog CO
                     events.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5.1"></a>5.1.&nbsp;DSB formation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Recombination initiates with a DSB, introduced by the conserved topoisomerase-like enzyme <a href="http://www.wormbase.org/db/get?name=WP:CE49824;class=Protein" target="_blank">SPO-11</a> (<a href="#bib37">Dernburg et al., 1998</a>). Mutants defective in genes required for DSB formation, such as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004985;class=Gene" target="_blank">spo-11</a></span>, manifest 12 univalents at diakinesis because the achiasmate homologs condense and separate from one another during diplotene.
                  A hallmark of this class of mutants is that bivalent formation can be restored by exposing the mutant to sources of exogenous
                  breaks such as ionizing radiation (IR). In addition to <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004985;class=Gene" target="_blank">spo-11</a></span> itself, genes that fall into this category include <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001874;class=Gene" target="_blank">him-17</a></span> (<a href="#bib124">Reddy and Villeneuve, 2004</a>), <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00008580;class=Gene" target="_blank">dsb-1</a></span> (<a href="#bib162">Stamper et al., 2013</a>), and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00194892;class=Gene" target="_blank">dsb-2</a></span> (<a href="#bib136">Rosu et al., 2013</a>). Two other genes, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001514;class=Gene" target="_blank">xnd-1</a></span> (<a href="#bib171">Wagner et al., 2010</a>) and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001864;class=Gene" target="_blank">him-5</a></span> (<a href="#bib98">Meneely et al., 2012</a>), influence DSB formation mainly on the X chromosome. <a href="http://www.wormbase.org/db/get?name=WP:CE37459;class=Protein" target="_blank">MRE-11</a> (<a href="#bib24">Chin and Villeneuve, 2001</a>), <a href="http://www.wormbase.org/db/get?name=WP:CE41010;class=Protein" target="_blank">RAD-50</a> (<a href="#bib56">Hayashi et al., 2007</a>), and <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> (<a href="#bib50">Goodyer et al., 2008</a>) are also required for efficient DSB induction, but play additional roles in DSB repair (see below). Finally, <a href="http://www.wormbase.org/db/get?name=WP:CE31579;class=Protein" target="_blank">CRA-1</a> regulates histone acetylation and promotes efficient DSB formation (<a href="#bib46">Gao et al., 2015</a>), suggesting that regulation of chromatin structure by histone modifications is an important modulator of DSB, as observed
                  in other organisms.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5.2"></a>5.2.&nbsp;DSB repair by meiosis-specific adaptation of homologous recombination
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Repair of DSBs during meiosis shares many features in common with homologous recombination (HR) in mitotic cell cycles, with
                  several meiosis-specific modifications. Following DSB formation, 5&#8217; ending DNA strands are resected, leaving 3&#8217; single-stranded
                  DNA flanking the DSB site. A 3&#8217; end then invades an intact double-stranded template DNA molecule (in meiosis, typically the
                  homologous chromosome rather than the sister chromatid, the preferred repair partner in other contexts) and primes DNA synthesis.
                  Following repair synthesis, this intermediate can be disassembled (e.g., through the action of DNA helicases (<a href="#bib112">P&acirc;ques and Haber, 1999</a>)); the released ssDNA end can then re-anneal to the other DSB end, leading to repair of the DSB without CO formation. Alternatively,
                  the intermediate can be stabilized through the action of meiosis-specific CO-promoting factors, leading to formation of more
                  complex, branched CO recombination intermediates (<a href="#figure5">Figure 5</a>). These intermediates must be enzymatically resolved by DNA strand cleavage; this can result in crossing over (reciprocal
                  exchange of information between chromosomes). We depict these meiotic CO-intermediates as double Holliday junctions by analogy
                  to <span class="italic">S. cerevisiae,</span> where recombination intermediate structures have been analyzed using physical assays; in most cases, the existence of specific
                  DSB repair intermediates in <span class="italic">C. elegans</span> is inferred rather than directly demonstrated.
               </p>
               <p>Mutants defective in genes required for DSB repair cannot be rescued by IR. Instead, exposure of repair-deficient mutants
                  to IR leads to significant chromosomal abnormalities including DNA fragmentation, chromosome aggregation and defects in chromosome
                  condensation, which are readily visible in diakinesis-stage oocytes (e.g., <a href="#bib24">Chin and Villeneuve, 2001</a>).
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5.2.1"></a>5.2.1.&nbsp;Resection
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The first step in the repair of meiotic DSBs is resection, the processing of ends into 3&#8217; single-stranded DNA (ssDNA) overhangs.
                     Based on the high degree of conservation between <a href="http://www.wormbase.org/db/get?name=WP:CE49824;class=Protein" target="_blank">SPO-11</a> homologs, it is believed that <a href="http://www.wormbase.org/db/get?name=WP:CE49824;class=Protein" target="_blank">SPO-11</a> becomes covalently attached to the DNA upon DSB formation, and that further processing requires removal of the SPO-11-DNA
                     nucleoprotein complexes to create short 3&#8217; overhangs. The <a href="http://www.wormbase.org/db/get?name=WP:CE37459;class=Protein" target="_blank">MRE-11</a> 5&#8217;-3&#8217; exonuclease is then normally required to generate long 3&#8217; ssDNA tails (<a href="#bib181">Yin and Smolikove, 2013</a>) that ultimately get coated with the <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> recombinase (<a href="#bib6">Alpi et al., 2003</a>; <a href="#bib28">Colai&aacute;covo et al., 2003</a>; <a href="#bib119">Petalcorin et al., 2006</a>). In the absence of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003405;class=Gene" target="_blank">mre-11</a></span>, the exonuclease <a href="http://www.wormbase.org/db/get?name=WP:CE28728;class=Protein" target="_blank">EXO-1</a> has also been shown to contribute to resection of meiotic DSBs, although resection is then delayed until mid-late pachytene
                     (<a href="#bib181">Yin and Smolikove, 2013</a>).
                  </p>
                  <p>Mutants defective in resection, such as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003405;class=Gene" target="_blank">mre-11</a></span> (<a href="#bib24">Chin and Villeneuve, 2001</a>; <a href="#bib181">Yin and Smolikove, 2013</a>) and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004296;class=Gene" target="_blank">rad-50</a></span> (<a href="#bib56">Hayashi et al., 2007</a>), are characterized by impaired loading of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> to SPO-11-dependent DSBs and display diakinesis-stage oocytes with abnormal chromosome aggregations. These aggregates arise
                     from repair by non-homologous end joining (NHEJ). The predominance of HR over NHEJ suggests that active mechanisms suppress
                     NHEJ in the meiotic germ line (<a href="#bib155">Smolikov et al., 2007a</a>; <a href="#bib1">Adamo et al., 2010</a>; <a href="#bib83">Lemmens et al., 2013</a>). A key player in this inhibition is <a href="http://www.wormbase.org/db/get?name=WP:CE43543;class=Protein" target="_blank">COM-1</a> (the Sae2/CtIP homolog (<a href="#bib115">Penkner et al., 2007a</a>)) that actively inhibits the binding of Ku proteins to the DNA ends (<a href="#bib83">Lemmens et al., 2013</a>). The involvement of <a href="http://www.wormbase.org/db/get?name=WP:CE37459;class=Protein" target="_blank">MRE-11</a> in both DSB formation and subsequent resection is another potential mechanism to ensure that DSBs are shunted towards HR.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5.2.2"></a>5.2.2.&nbsp;Strand exchange and formation of downstream repair intermediates
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> nucleoprotein filament serves the important role of promoting the homology search and strand invasion into the homologous
                     duplex DNA (<a href="#figure5">Figure 5</a>). Displacement of the homologous template strand forms a D loop structure wherein the invading 3&#8217; end is used to prime DNA
                     synthesis. COs can result from subsequent formation and cleavage of more complex intermediates (possibly double Holliday junctions)
                     made by second end capture. Non-crossovers (NCOs) are presumed to arise from ejection of the invading strand and repair via
                     synthesis-dependent strand annealing. Failure to execute these steps can allow for aberrant repair via NHEJ and ultimately
                     can cause chromosome fusions.
                  </p>
                  <p>Due to the central role of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> to HR, it is no surprise that <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> filament formation is highly regulated (<a href="#bib125">Rinaldo et al., 2002</a>; <a href="#bib6">Alpi et al., 2003</a>; <a href="#bib28">Colai&aacute;covo et al., 2003</a>; <a href="#bib164">Takanami et al., 2003</a>). The worm homolog of the BRCA2 breast cancer associated protein, <a href="http://www.wormbase.org/db/get?name=WP:CE32718;class=Protein" target="_blank">BRC-2</a><span class="italic">,</span> facilitates nuclear localization of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a>, its loading onto ssDNA (<a href="#bib92">Martin et al., 2005</a>), and D loop formation (<a href="#bib118">Petalcorin et al., 2007</a>). In <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00020316;class=Gene" target="_blank">brc-2</a></span> mutants, RPA-coated ssDNA can be detected; this either reflects the fact that <a href="http://www.wormbase.org/db/get?name=WP:CE04405;class=Protein" target="_blank">RPA-1</a> loads onto resected ssDNA before <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a>, as in other organisms, or is an aberrant outcome resulting from the failure to load <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a>. <a href="http://www.wormbase.org/db/get?name=WP:CE15894;class=Protein" target="_blank">RTEL-1</a> may also influence <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> dynamics through its role as an anti-recombinase, disrupting D loops and promoting alternative pathways of repair (<a href="#bib8">Barber et al., 2008</a>).
                  </p>
                  <p>In order for HR to proceed, <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> must also be removed from the D loop. Three proteins have been thought to promote <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> turnover: <a href="http://www.wormbase.org/db/get?name=WP:CE25143;class=Protein" target="_blank">RAD-54</a> (<a href="#bib99">Mets and Meyer, 2009</a>), the helicase <a href="http://www.wormbase.org/db/get?name=WP:CE37439;class=Protein" target="_blank">HELQ-1</a>/HEL308, and the <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> paralog, <a href="http://www.wormbase.org/db/get?name=WP:CE00094;class=Protein" target="_blank">RFS-1</a> (<a href="#bib173">Ward et al., 2010</a>), with the latter two having a redundant function in wild-type cells. Cytological examination of both <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004298;class=Gene" target="_blank">rad-54</a></span> single mutants and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00021905;class=Gene" target="_blank">helq-1</a> <a href="http://www.wormbase.org/db/get?name=WBGene00004342;class=Gene" target="_blank">rfs-1</a></span> double mutants shows prolonged persistence of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> filaments, indicating the persistence of unrepaired DSBs.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5.2.3"></a>5.2.3.&nbsp;CO designation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Once strand exchange has occurred, additional regulation is required to ensure that a subset of these intermediates become
                     committed to a CO fate. Classically, two CO pathways were described in model systems: the class I pathway, requiring the meiosis-specific
                     homologs of the mismatch repair protein MutS, MSH4 and MSH5, and certain SC associated proteins, is considered responsible
                     for COs subject to interference (see <a href="#sec5.4" title="5.4.&nbsp;CO interference, assurance, and homeostasis">Section 5.4</a>); the class II pathway, dependent on the structure-specific nuclease components MUS81 and MMS4, accounts for the interference-independent
                     COs (reviewed in <a href="#bib76">Kohl and Sekelsky, 2013</a>). In <span class="italic">C. elegans</span>, the class I pathway predominates, as mutations in the respective orthologs, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003420;class=Gene" target="_blank">msh-4</a></span>, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003421;class=Gene" target="_blank">msh-5</a></span> (<a href="#bib175">Winand et al., 1998</a>; <a href="#bib184">Zalevsky et al., 1999</a>; <a href="#bib71">Kelly et al., 2000</a>; <a href="#bib28">Colai&aacute;covo et al., 2003</a>), and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006976;class=Gene" target="_blank">zhp-3</a></span> (Zip3 homologous protein; <a href="#bib64">Jantsch et al., 2004</a>) leads to univalents at diakinesis. It should be noted that MSH-4 and <a href="http://www.wormbase.org/db/get?name=WP:CE26678;class=Protein" target="_blank">MSH-5</a> are first detected broadly in presumptive DSB repair foci that exceed the number of eventual COs; they then become restricted
                     to CO sites by late pachytene (<a href="#bib182">Yokoo et al., 2012</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE43850;class=Protein" target="_blank">ZHP-3</a> is initially present along the entire SC in mid-pachytene and then gradually becomes restricted to CO sites (<a href="#bib14">Bhalla et al., 2008</a>). Thus, additional proteins must function in designating a subset of recombination intermediates for CO fates. Anti-recombinases
                     such as RTEL function at this step to shunt repair intermediates into a NCO outcome (possibly by promoting synthesis-dependent
                     strand annealing); thus, in the absence of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00009124;class=Gene" target="_blank">rtel-1</a></span> function, additional COs are made (<a href="#bib183">Youds et al., 2010</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a>, a cyclin-like protein that is required for CO formation, has been proposed to function in reinforcement of CO designation,
                     as it is predominantly detected as a single focus on each chromosome pair during late pachytene (<a href="#bib182">Yokoo et al., 2012</a>). Additional <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> foci are observed on very large fusion chromosomes, such at the X;IV fusion chromosome mnT12 (<a href="#bib84">Libuda et al., 2013</a>), where multiple COs can be detected genetically (<a href="#bib60">Hillers and Villeneuve, 2003</a>).
                  </p>
                  <p>Loss of function mutations in MSH-4, <a href="http://www.wormbase.org/db/get?name=WP:CE26678;class=Protein" target="_blank">MSH-5</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE43850;class=Protein" target="_blank">ZHP-3</a>, or <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> result in 12 univalents at diakinesis. Irradiation cannot restore bivalent formation in these mutants. The mutants also do
                     not show chromosome fusions indicative of mutations in strand exchange proteins. The kinetics of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> foci turnover is slower in these mutants, but they do disappear prior to diplotene indicating that DSB repair has occurred
                     (<a href="#bib184">Zalevsky et al., 1999</a>; <a href="#bib28">Colai&aacute;covo et al., 2003</a>; <a href="#bib64">Jantsch et al., 2004</a>; <a href="#bib159">Song et al., 2010</a>; <a href="#bib182">Yokoo et al., 2012</a>).
                  </p>
                  <p>The emerging picture of CO formation from studies in <span class="italic">C. elegans</span> and other systems is that an excess of DSBs are made; these get initially bound by repair proteins and putative CO designation
                     proteins and winnowed down to a small number of CO sites. Understanding this selection process is an area of active research
                     in many organisms including <span class="italic">C. elegans.</span></p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5.2.4"></a>5.2.4.&nbsp;Partner choice
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Although many events of DSB repair are conserved between mitosis and meiosis, a major difference is that the homolog is used
                     as a repair template during meiosis, whereas the sister chromatid is the preferred repair template for cells in S and G2 of
                     mitotic cell cycles. The preference for the homolog rather than the sister is thought to be mediated in large part by axial
                     element components (<a href="#sec4.2" title="4.2.&nbsp;HORMA-domain proteins">Section 4.2</a>). Use of the homolog as a recombination partner also appears to be a temporally regulated feature of the meiotic program:
                     access to the homolog as repair template for both CO and non-CO repair is shut down upon transition into the late pachytene
                     stage. Thus, DSBs that are made or persist into late pachytene or beyond are presumed to be repaired using the sister chromatid
                     as a repair template (<a href="#bib28">Colai&aacute;covo et al., 2003</a>; <a href="#bib135">Rosu et al., 2011</a>).
                  </p>
                  <p>During meiosis, an excess of DSBs is formed (relative to the number of COs). Many of these breaks are repaired as NCOs using
                     the homolog as a repair template (<a href="#bib126">Robert et al., 2008</a>; <a href="#bib135">Rosu et al., 2011</a>), but repair using the sister chromatid as a template likely also occurs. The <a href="http://www.wormbase.org/db/get?name=WP:CE27096;class=Protein" target="_blank">SMC-5</a>/6 complex and the <a href="http://www.wormbase.org/db/get?name=WP:CE43190;class=Protein" target="_blank">BRC-1</a> protein have been shown to be important for repair of meiotic DSBs under conditions where inter-homolog CO formation is abrogated
                     and/or in mutant situations when inter-sister repair is the only option (<a href="#bib16">Boulton et al., 2004</a>; <a href="#bib2">Adamo et al., 2008</a>; <a href="#bib15">Bickel et al., 2010</a>). These papers provided evidence that <a href="http://www.wormbase.org/db/get?name=WP:CE27096;class=Protein" target="_blank">SMC-5</a>/6 and <a href="http://www.wormbase.org/db/get?name=WP:CE43190;class=Protein" target="_blank">BRC-1</a> are important for inter-sister repair, but the data are consistent with these proteins also participating in inter-homolog
                     non-CO repair as well.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5.2.5"></a>5.2.5.&nbsp;CO resolution
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The final step of CO recombination is cleavage of CO intermediates to yield mature CO products. <span class="italic">C. elegans</span> contains several resolvase functions that are scaffolded by the SLX-4/<a href="http://www.wormbase.org/db/get?name=WP:CE48043;class=Protein" target="_blank">HIM-18</a> protein (<a href="#bib139">Saito et al., 2009</a>). These can either act redundantly or can substitute for each other to resolve CO intermediates. <a href="http://www.wormbase.org/db/get?name=WP:CE24855;class=Protein" target="_blank">XPF-1</a> (in combination with <a href="http://www.wormbase.org/db/get?name=WP:CE31724;class=Protein" target="_blank">HIM-6</a>/BLM) and <a href="http://www.wormbase.org/db/get?name=WP:CE11216;class=Protein" target="_blank">SLX-1</a> (in combination with <a href="http://www.wormbase.org/db/get?name=WP:CE32609;class=Protein" target="_blank">MUS-81</a>) are proposed to define two mechanistically distinct pathways for resolving CO intermediates (<a href="#bib3">Agostinho et al., 2013</a>; <a href="#bib111">O'Neil et al., 2013</a>; <a href="#bib137">Saito et al., 2013</a>). Double mutants lacking both pathways exhibit defects in post-pachytene chromosome reorganization and disassembly of the
                     SC. Unique to these mutants, diakinesis univalents appear closely apposed and connected by chromatin bridges (<a href="#bib3">Agostinho et al., 2013</a>; <a href="#bib111">O'Neil et al., 2013</a>; <a href="#bib137">Saito et al., 2013</a>). These bridges are ultimately resolved in meiosis II, however, suggesting that additional resolvase functions are present
                     in the worm germ line.
                  </p>
                  <p>The <a href="http://www.wormbase.org/db/get?name=WP:CE31724;class=Protein" target="_blank">HIM-6</a>/BLM helicase may play a central role in ensuring that CO-designated recombination intermediates are eventually resolved into
                     COs. In the absence of <a href="http://www.wormbase.org/db/get?name=WP:CE31724;class=Protein" target="_blank">HIM-6</a> protein, nuclei appear to carry out the early steps of meiotic recombination normally; CO designation also seems normal,
                     as 6 <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> foci can be seen in late pachytene oocytes. However, a subset of these CO-designated intermediates fail to form COs, evidenced
                     by the presence of univalents at diakinesis (<a href="#bib146">Schvarzstein et al., 2014</a>). It may be that <a href="http://www.wormbase.org/db/get?name=WP:CE31724;class=Protein" target="_blank">HIM-6</a> acts to ensure the CO outcome by biasing resolution of intermediates (by either <a href="http://www.wormbase.org/db/get?name=WP:CE24855;class=Protein" target="_blank">XPF-1</a> or <a href="http://www.wormbase.org/db/get?name=WP:CE11216;class=Protein" target="_blank">SLX-1</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE32609;class=Protein" target="_blank">MUS-81</a>) towards CO formation or by protecting CO-designated intermediates from NCO resolution.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5.3"></a>5.3.&nbsp;The recombination landscape
                        </h3>
                     </div>
                  </div>
               </div>
               <p>CO frequency and distribution can be assayed genetically, using either phenotypic markers or DNA sequence polymorphisms; the
                  latter approach allows a more detailed analysis of crossing over, as a higher density of markers can be used in a single experiment
                  (<a href="#bib36">Davis et al., 2005</a>; <a href="#bib61">Hillers and Villeneuve, 2009</a>). Next-generation DNA sequencing has the potential to allow genome-wide determination of the recombination landscape but
                  to date has only been used to probe targeted genomic regions (<a href="#bib70">Kaur and Rockman, 2014</a>). Crossing over can also be analyzed cytologically, by visualizing the number and distribution of <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> foci in late pachytene cells (<a href="#bib182">Yokoo et al., 2012</a>; <a href="#bib84">Libuda et al., 2013</a>).
               </p>
               <p>In many eukaryotes, COs are non-randomly distributed along chromosomes, with a tendency to occur within small regions called
                  hotspots (<a href="#bib109">Nicolas et al., 1989</a>; <a href="#bib10">Baudat et al., 2010</a>). No evidence for CO hotspots has been found in <span class="italic">C. elegans</span>, however, despite considerable efforts (<a href="#bib70">Kaur and Rockman, 2014</a>). Nevertheless, meiotic COs in <span class="italic">C. elegans</span> are not randomly distributed along chromosomes. As described in the WormBook chapter Karyotype, ploidy and gene dosage, the
                  distribution of COs along the autosomes is uneven, with roughly the central third of each autosome having a paucity of events;
                  interestingly, the same region of each autosome has a higher density of genes (the &#8220;central gene cluster&#8221;). Conversely, the
                  arms of each chromosome have a higher frequency of crossing over and a lower gene density, thus the physical and genetic maps
                  of <span class="italic">C. elegans</span> are said to be &#8220;out of alignment&#8221;. This type of CO density domain structure is also observed on the X chromosome, but is
                  much less pronounced (<a href="#bib9">Barnes et al., 1995</a>; <a href="#bib127">Rockman and Kruglyak, 2009</a>). Together, these features indicate that some aspect of CO formation is subject to regional regulation; the DNA sequence
                  determinants of this regulation (if any) are unclear.
               </p>
               <p>Evidence that CO distribution is influenced by genetic factors came initially from the isolation of a mutation in the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004332;class=Gene" target="_blank">rec-1</a></span> gene. This allele alters the distribution of COs along autosomes without changing overall frequency. This redistribution
                  of COs towards the middle of each chromosome aligns the physical and genetic maps of <span class="italic">C. elegans</span> (<a href="#bib131">Rose and Baillie, 1979a</a>; <a href="#bib187">Zetka and Rose, 1995</a>). Mutations in the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001864;class=Gene" target="_blank">him-5</a></span> gene also cause redistribution of meiotic COs along autosomes, resulting in an increased incidence of COs in the central
                  gene cluster; similar to <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004332;class=Gene" target="_blank">rec-1</a></span>, there is not an overall increase of COs along autosomes (<a href="#bib98">Meneely et al., 2012</a>). In contrast to <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004332;class=Gene" target="_blank">rec-1</a></span>, however, <a href="http://www.wormbase.org/db/get?name=WP:CE09062;class=Protein" target="_blank">HIM-5</a> also plays a role in facilitating meiotic DSB formation on the X chromosomes: in a <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001864;class=Gene" target="_blank">him-5</a></span> mutant the frequency of COs on the <span class="italic">X</span> is reduced, with a concomitant increase in X chromosome nondisjunction (the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype).
               </p>
               <p>The phenotypes of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001874;class=Gene" target="_blank">him-17</a></span>, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001514;class=Gene" target="_blank">xnd-1</a></span>, and condensin mutants suggest that chromatin structure and condensation state are also critical determinants of DSB sites.
                  <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001874;class=Gene" target="_blank">him-17</a></span> and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001514;class=Gene" target="_blank">xnd-1</a></span> mutants both show aberrant histone post-translational modifications and are impaired in DSB formation (<a href="#bib124">Reddy and Villeneuve, 2004</a>; <a href="#bib171">Wagner et al., 2010</a>). Both genes are also required for normal localization of <a href="http://www.wormbase.org/db/get?name=WP:CE09062;class=Protein" target="_blank">HIM-5</a>, revealing interplay between chromatin and the recombination landscape (<a href="#bib98">Meneely et al., 2012</a>). Likewise, condensin proteins, which play crucial roles in chromosome condensation during cell divisions, play roles in
                  establishing the wild-type distribution of DSBs and COs. Heterozygosity for a null mutation in any of the subunits of Condensin
                  I (<a href="http://www.wormbase.org/db/get?name=WP:CE31734;class=Protein" target="_blank">DPY-28</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE08398;class=Protein" target="_blank">DPY-26</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE18083;class=Protein" target="_blank">MIX-1</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE03287;class=Protein" target="_blank">SMC-4</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE09790;class=Protein" target="_blank">CAPG-1</a>) or of <a href="http://www.wormbase.org/db/get?name=WP:CE30620;class=Protein" target="_blank">KLE-2</a> (a subunit of Condensin II) leads to HIM-3-dependent alterations in recombination and chromosome structure: COs are redistributed
                  along the <span class="italic">X</span> chromosome, the overall frequency of COs is increased, and pachytene chromosome axis length is increased (<a href="#bib167">Tsai et al., 2008</a>; <a href="#bib99">Mets and Meyer, 2009</a>). Mutants homozygous for a partial loss of function allele of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001087;class=Gene" target="_blank">dpy-28</a></span>, which does not increase CO frequencies, showed that CO distribution correlated with a shift in the distribution of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> foci (<a href="#bib99">Mets and Meyer, 2009</a>). Moreover, the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001087;class=Gene" target="_blank">dpy-28</a></span> partial loss of function allele suppresses the defect in chiasma formation of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001874;class=Gene" target="_blank">him-17</a></span> mutants, suggesting that higher-order chromosome structures promoted by condensin limit DSB formation (<a href="#bib167">Tsai et al., 2008</a>). Together, this set of genes points to a complex interplay between chromosome/ chromatin structure and the meiotic recombination
                  machinery.
               </p>
               <p>Despite these general features, CO frequency and position are not fixed within the population, and the number and frequency
                  of COs differs between the sexes, with maternal age, and with temperature (<a href="#bib62">Hodgkin et al., 1979</a>; <a href="#bib132">Rose and Baillie, 1979b</a>; <a href="#bib186">Zetka and Rose, 1990</a>; <a href="#bib86">Lim et al., 2008</a>; <a href="#bib159">Song et al., 2010</a>). However, in all these situations, the arm/cluster differences in CO distribution remain (<a href="#bib9">Barnes et al., 1995</a>). Thus, at the global level, there are defined regions of the genome that are more or less prone to receive a CO.
               </p>
               <p>CO distribution can also be influenced by events subsequent to DSB formation. DSB sites can be mapped cytologically using
                  anti-RAD-51 antibodies in a mutant such as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004298;class=Gene" target="_blank">rad-54</a></span> that prevents further processing of the filament and effectively traps the recombination intermediate (<a href="#bib99">Mets and Meyer, 2009</a>; <a href="#bib110">Nottke et al., 2011</a>; <a href="#bib135">Rosu et al., 2011</a>; <a href="#bib138">Saito et al., 2012</a>). Cytological examination of RAD-51-marked DSB sites in wild type failed to detect an arm/cluster bias in DSB distribution
                  (<a href="#bib138">Saito et al., 2012</a>), which would be expected if CO distribution were predominantly enforced at the step of DSB formation. Rather, it appears
                  that CO formation in the central gene cluster is actively inhibited at a step post-recruitment of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a>. The resolvase <a href="http://www.wormbase.org/db/get?name=WP:CE11216;class=Protein" target="_blank">SLX-1</a> (<a href="#sec5.2.5" title="5.2.5.&nbsp;CO resolution">Section 5.2.5</a>) appears to function in this inhibition, as mutation of <a href="http://www.wormbase.org/db/get?name=WP:CE11216;class=Protein" target="_blank">SLX-1</a> leads to a redistribution of exchanges along chromosome V: there is an increase in COs in the central gene cluster of chromosome
                  V, with a compensatory decrease in COs in the arm regions (<a href="#bib138">Saito et al., 2012</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5.4"></a>5.4.&nbsp;CO interference, assurance, and homeostasis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Each of the 6 chromosome pairs in <span class="italic">C. elegans</span> has a genetic map size near 50 cM. A map size of 50 cM indicates that each chromosome pair experiences (on average) only
                  one CO per meiosis. Each homolog pair must receive the &#8220;obligate CO&#8221;, to promote accurate chromosome segregation, and different
                  mechanisms operate during meiosis to ensure this outcome. The ability to ensure the formation of at least one CO per homolog
                  pair is known as CO assurance, while the ability to maintain CO number in the face of variability in numbers of DSBs is known
                  as CO homeostasis. As homeostasis is a mechanism that promotes COs at the expense of NCOs in situations where DSB numbers
                  are limiting, CO homeostasis helps to achieve CO assurance, suggesting that these two manifestations of CO regulation are
                  mechanistically linked. Evidence for the operation of these mechanisms during worm meiosis comes from studies in which a limited
                  number of DSBs are exogenously introduced in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004985;class=Gene" target="_blank">spo-11</a></span> mutants. For example, when a single DSB is generated by transposon excision, this event is converted into a CO with high
                  efficiency (<a href="#bib135">Rosu et al., 2011</a>). This suggests that CO assurance can be achieved by ensuring that each chromosome receives at least one DSB. Similarly,
                  when a limited number of DSBs are generated by irradiation, these are also efficiently converted into a single CO per chromosome
                  pair. Further increases in the number of DSBs by increased irradiation does not result in the formation of more <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> marked events, providing evidence for CO homeostasis (<a href="#bib182">Yokoo et al., 2012</a>). In addition, CO homeostasis may explain why only the X chromosome is devoid of a CO in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001864;class=Gene" target="_blank">him-5</a></span> mutants whereas <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> foci are reduced over four-fold genome-wide (<a href="#bib98">Meneely et al., 2012</a>).
               </p>
               <p>In <span class="italic">C. elegans</span>, most chromosome pairs undergo only a single CO per meiosis (<a href="#bib60">Hillers and Villeneuve, 2003</a>; <a href="#bib54">Hammarlund et al., 2005</a>), but when multiple COs have been mapped, they tend to be widely spaced along the chromosome (<a href="#bib60">Hillers and Villeneuve, 2003</a>; <a href="#bib86">Lim et al., 2008</a>; <a href="#bib84">Libuda et al., 2013</a>). This non-random distribution of COs is known as CO interference (<a href="#bib105">Muller, 1916</a>). The fact that most chromosome pairs receive only a single CO suggests that CO interference operates over distances greater
                  than the length of a typical chromosome to discourage additional exchanges once one has occurred. Indeed, studies with fusion
                  chromosomes have demonstrated that end-to-end fusions of two and even three chromosomes still frequently only received a single
                  CO in meiosis (<a href="#bib60">Hillers and Villeneuve, 2003</a>). Further, cytological mapping of COs along the length of <span class="italic">mnT12</span> (an end-to-end fusion of chromosomes <span class="italic">IV</span> and <span class="italic">X</span>) enabled measurement of the distance along chromosomes over which CO interference operates in <span class="italic">C. elegans</span>, and strengthened the idea that interference operates over distances longer than the length of a typical chromosome (<a href="#bib84">Libuda et al., 2013</a>).
               </p>
               <p>While cytological evaluation of CO interference using <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> foci suggests that chromosome pairs rarely have more that one cytologically differentiated CO site, double COs have been
                  characterized using genetic recombination assays. Double CO data for individual autosomes range between 2-10% per chromosome
                  pair (<a href="#bib97">Meneely et al., 2002</a>; <a href="#bib108">Nabeshima et al., 2004</a>; <a href="#bib86">Lim et al., 2008</a>; <a href="#bib38">Deshong et al., 2014</a>; <a href="#bib45">Gabdank and Fire, 2014</a>). Extrapolating these data for all 5 autosomes, this genetic evidence suggests up to 35% of nuclei have at least one autosome
                  that obtains a second CO. These extra COs are cytologically distinct, however, as they are apparently not associated with
                  <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> foci (<a href="#bib182">Yokoo et al., 2012</a>; <a href="#bib38">Deshong et al., 2014</a>). The X chromosome, in contrast, appears particularly immune to double COs in some studies (<a href="#bib86">Lim et al., 2008</a>; <a href="#bib167">Tsai et al., 2008</a>; but see <a href="#bib38">Deshong et al., 2014</a>), perhaps due to its heterochromatic-like state (<a href="#bib72">Kelly et al., 2002</a>). Further, the frequency of double COs differs between oocyte and sperm meiosis (<a href="#bib186">Zetka and Rose, 1990</a>; <a href="#bib86">Lim et al., 2008</a>; <a href="#bib45">Gabdank and Fire, 2014</a>). Data are only available for male sperm: they appear to have less stringent genetic interference compared to oocytes, having
                  both increased numbers of double COs and more closely spaced COs (<a href="#bib86">Lim et al., 2008</a>; <a href="#bib45">Gabdank and Fire, 2014</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5.5"></a>5.5.&nbsp;Interplay between recombination and chromosome structure
                        </h3>
                     </div>
                  </div>
               </div>
               <p>While the analysis of null mutants lacking any of the SYP proteins reveals that these proteins are required for CO/chiasma
                  formation, partial depletion of SYP proteins by RNAi results in an increased incidence of double COs (<a href="#bib57">Hayashi et al., 2010</a>; <a href="#bib84">Libuda et al., 2013</a>). Similarly, a non-null allele of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001862;class=Gene" target="_blank">him-3</a></span> that allows partial synapsis of homologs also results in increased incidence of double COs (<a href="#bib108">Nabeshima et al., 2004</a>). This apparent contradiction can be explained by the proposal that the SC functions in the process of CO interference (<a href="#sec5.4" title="5.4.&nbsp;CO interference, assurance, and homeostasis">Section 5.4</a>). Although the mechanism by which interference is transmitted along chromosomes is not understood, <a href="#bib84">Libuda et al. (2013)</a> found that each CO event is associated with a local extension of the axial element, suggesting that this structural alteration
                  may play a role in impeding the formation of further COs. CO numbers are also increased when the structure of chromosomes
                  is altered by reducing the dose of components of the condensin I complex, although in this case the extra COs are not associated
                  with a <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> marked cytologically differentiated site (<a href="#bib167">Tsai et al., 2008</a>; <a href="#bib99">Mets and Meyer, 2009</a>; <a href="#bib182">Yokoo et al., 2012</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5.6"></a>5.6.&nbsp;Window of opportunity for CO formation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>DSB formation and repair are initiated in the transition zone; <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> foci continue to accumulate through mid-pachytene, suggesting that both the initiation and cessation of DSB formation are
                  controlled. DSB formation has been proposed to be subject to feedback regulation; once all chromosome pairs have a CO-eligible
                  intermediate, DSB formation ceases (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>). In addition, it is clear that as nuclei transition into late pachytene, there is a shift in repair mechanism utilization;
                  even if DSBs are formed in late pachytene germ cells (e.g., by gamma irradiation), these nuclei are incapable of committing
                  to CO formation (<a href="#bib182">Yokoo et al., 2012</a>). Further, during early and mid-pachytene, <a href="http://www.wormbase.org/db/get?name=WP:CE41010;class=Protein" target="_blank">RAD-50</a> is required for <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> loading at DSB sites; after the late pachytene transition, <a href="http://www.wormbase.org/db/get?name=WP:CE41010;class=Protein" target="_blank">RAD-50</a> is no longer required for <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> loading (<a href="#bib56">Hayashi et al., 2007</a>). Finally, access to the homolog as a repair template is also shut down at the transition from mid- to late pachytene (<a href="#bib135">Rosu et al., 2011</a>), presumably reflecting reversion to preferential use of the sister chromatid as a repair partner. This transition to sister
                  repair is accompanied by an ability of DNA damage to trigger localized separation of the chromosome axes, a feature that is
                  antagonized by <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> at earlier stages (<a href="#bib34">Couteau and Zetka, 2011</a>). Ultimately, this transition in DNA repair mode may allow for residual DSBs to be repaired and the nucleus to escape apoptosis
                  induced by the DNA damage checkpoint. The transition zone to mid-pachytene region can thus be considered the window of opportunity
                  for the formation of DSBs and CO-designated events.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;Bivalent differentiation
                     </h2>
                  </div>
               </div>
            </div>
            <p>Following the formation of CO-fated recombination intermediates, meiotic chromosomes undergo two major structural changes
               in preparation for the meiotic divisions: the disassembly of the SC, which occurs in an asymmetric fashion around the CO site;
               and condensation, a process that causes a dramatic reduction in chromosome size (for detailed reviews see <a href="#bib147">Schvarzstein et al., 2010</a>; <a href="#bib177">Wood et al., 2010</a>). Together, these lead to the formation of diakinesis bivalents, highly condensed pairs of homologous chromosomes held together
               by sister chromatid cohesion and COs. Bivalents contain two functional domains: a short arm where sister chromatid cohesion
               will be released during the first meiotic division, and a long arm where sister chromatids will remain attached until the
               onset of the second meiotic division (<a href="#bib5">Albertson and Thomson, 1993</a>; <a href="#bib4">Albertson et al., 1997</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6.1"></a>6.1.&nbsp;CO-triggered chromosome remodeling
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The creation of bivalents with two differentiated functional domains starts in late pachytene with the redistribution of SC
                  and axis-associated proteins that is triggered by CO-fated recombination intermediates (<a href="#bib107">Nabeshima et al., 2005</a>; <a href="#bib19">de Carvalho et al., 2008</a>; <a href="#bib93">Martinez-Perez et al., 2008</a>; <a href="#bib3">Agostinho et al., 2013</a>) (<a href="#figure6">Figure 6</a>). At this stage <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2 and <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a> are depleted from the region of the chromosomes between the CO site and the closest telomere, forming the short arm of the
                  bivalent, while the SYP proteins and <a href="http://www.wormbase.org/db/get?name=WP:CE43850;class=Protein" target="_blank">ZHP-3</a> are depleted from the region between the CO and the farthest telomere, forming the long arm (<a href="#bib107">Nabeshima et al., 2005</a>; <a href="#bib156">Smolikov et al., 2007b</a>; <a href="#bib14">Bhalla et al., 2008</a>; <a href="#bib19">de Carvalho et al., 2008</a>; <a href="#bib93">Martinez-Perez et al., 2008</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>/4-containing cohesin complexes are also largely removed from the long arm of each bivalent by late diakinesis and become
                  restricted to the short arm by prometaphase I (<a href="#bib150">Severson and Meyer, 2014</a>). The short arm region undergoes a further change in protein composition in the two most proximal oocytes in which SYP proteins
                  are removed and <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> kinase is recruited (<a href="#bib68">Kaitna et al., 2002</a>; <a href="#bib130">Rogers et al., 2002</a>; <a href="#bib107">Nabeshima et al., 2005</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> recruitment to diakinesis bivalents is required for homolog segregation during the first meiotic division (<a href="#bib68">Kaitna et al., 2002</a>; <a href="#bib130">Rogers et al., 2002</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> is phosphorylated by <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> in vitro (<a href="#bib130">Rogers et al., 2002</a>), suggesting that <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> phosphorylation may be important for cohesion release in vivo. In fact, <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> staining in the short arm of bivalents becomes weaker during late diakinesis and prometaphase (<a href="#bib130">Rogers et al., 2002</a>; <a href="#bib19">de Carvalho et al., 2008</a>; <a href="#bib55">Harper et al., 2011</a>; <a href="#bib150">Severson and Meyer, 2014</a>), and this reduction in <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> staining requires <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> (<a href="#bib150">Severson and Meyer, 2014</a>). By metaphase I, <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> staining is greatly reduced in the short arm of bivalents (<a href="#bib19">de Carvalho et al., 2008</a>), but use of a different anti-REC-8 antibody shows clearly detectable <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> staining in the short arm of most metaphase I bivalents (<a href="#bib114">Pasierbek et al., 2001</a>; <a href="#bib130">Rogers et al., 2002</a>;; <a href="#bib31">Cortes et al., 2015</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>/4 cohesin complexes that remain bound on the short arm of metaphase I chromosomes must be removed at anaphase onset. A likely
                  candidate for this activity is the conserved protease separase, which is required for proper meiotic chromosome segregation
                  (<a href="#bib154">Siomos et al., 2001</a>). Interestingly, separase is not required for the late prophase removal of <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>/4 cohesin described above (<a href="#bib150">Severson and Meyer, 2014</a>), suggesting that distinct mechanisms mediate cohesin removal during <span class="italic">C. elegans</span> meiosis. Phosphorylation of kleisin subunits is required for their cleavage by separase in yeast mitotic and meiotic cells
                  (reviewed in <a href="#bib52">Haarhuis et al., 2014</a>), and it has been proposed that AIR-2-dependent phosphorylation of <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> complexes may render them sensitive to separase (<a href="#bib130">Rogers et al., 2002</a>; <a href="#bib19">de Carvalho et al., 2008</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> activity in the long arms of diakinesis bivalents is antagonized by <a href="http://www.wormbase.org/db/get?name=WP:CE01319;class=Protein" target="_blank">GSP-2</a> (protein phosphatase 1), whose depletion leads to the premature separation of sister chromatids during anaphase I (<a href="#bib68">Kaitna et al., 2002</a>; <a href="#bib130">Rogers et al., 2002</a>). Similarly, <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2 and <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a> prevent the inappropriate recruitment of <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> to the long arm of the bivalents, and their absence causes untimely removal of <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> and premature release of sister chromatid cohesion during the first meiotic division (<a href="#bib19">de Carvalho et al., 2008</a>; <a href="#bib93">Martinez-Perez et al., 2008</a>). Since the ability of <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a> to antagonize <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> localization depends on the presence of <a href="http://www.wormbase.org/db/get?name=WP:CE01319;class=Protein" target="_blank">GSP-2</a>, and since <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE01319;class=Protein" target="_blank">GSP-2</a> physically interact, <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a> likely acts by recruiting <a href="http://www.wormbase.org/db/get?name=WP:CE01319;class=Protein" target="_blank">GSP-2</a> to chromosomes (<a href="#bib19">de Carvalho et al., 2008</a>; <a href="#bib168">Tzur et al., 2012</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="meiosis_fig6.jpg"><img src="meiosis_fig6_s.jpg" border="2" align="middle" alt=" figure 6"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b><span class="bold"><strong>Figure 6. Asymmetric re-distribution of SC components during bivalent differentiation</strong></span>.&nbsp;</b>Top row shows projections of individual bivalents taken from nuclei at the indicated stages and stained with anti-HTP-1/2
                        and anti-SYP-1 antibodies and counterstained with DAPI. The bottom row shows a diagram of the process of bivalent differentiation.
                        During pachytene <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2 and <a href="http://www.wormbase.org/db/get?name=WP:CE33925;class=Protein" target="_blank">SYP-1</a> colocalize along the whole length of fully synapsed bivalents, but by late pachytene <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2 and <a href="http://www.wormbase.org/db/get?name=WP:CE33925;class=Protein" target="_blank">SYP-1</a> acquire a reciprocal staining pattern, with a CO (or a CO-fated event) marking the boundary of the two domains. SC disassembly
                        and chromosome condensation start during diplotene, when the regions lacking SYP proteins become separated. By diakinesis,
                        bivalents undergo further condensation and the long arms, containing <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2, and short arms, containing <a href="http://www.wormbase.org/db/get?name=WP:CE33925;class=Protein" target="_blank">SYP-1</a>, are clearly differentiated. For a detailed description of bivalent differentiation see <a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6.2"></a>6.2.&nbsp;Chromosome condensation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Concomitant with the changes in protein composition described above, the overall structure of chromosomes is modified by a
                  process of chromatin compaction that starts in late pachytene and continues until the onset of the first meiotic division.
                  This process is mediated by condensin complexes, which are structurally similar to cohesin, containing a core of two SMC proteins
                  that associate with additional factors to modify the topology of chromosomes (reviewed in <a href="#bib177">Wood et al., 2010</a>). The condensin II complex (<a href="http://www.wormbase.org/db/get?name=WP:CE18083;class=Protein" target="_blank">MIX-1</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE03287;class=Protein" target="_blank">SMC-4</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE24117;class=Protein" target="_blank">HCP-6</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE11156;class=Protein" target="_blank">CAPG-2</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE30620;class=Protein" target="_blank">KLE-2</a>) associates with chromosomes in diplotene and is required for the compaction and resolution of chromosomes during diplotene
                  and diakinesis, demonstrating that condensin II is essential for the remodeling of chromosomes during late meiotic prophase
                  (<a href="#bib53">Hagstrom et al., 2002</a>; <a href="#bib21">Chan et al., 2004</a>; <a href="#bib35">Csankovszki et al., 2009</a>; <a href="#bib99">Mets and Meyer, 2009</a>). In diakinesis oocytes, condensin II decorates chromatin on the long and short arms of the bivalents. On the other hand,
                  depletion of condensin I (<a href="http://www.wormbase.org/db/get?name=WP:CE18083;class=Protein" target="_blank">MIX-1</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE03287;class=Protein" target="_blank">SMC-4</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE08398;class=Protein" target="_blank">DPY-26</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE31734;class=Protein" target="_blank">DPY-28</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE09790;class=Protein" target="_blank">CAPG-1</a>) does not induce gross defects in the condensation or individualization of meiotic chromosomes during late prophase, and
                  condensin I components are only observed associated with chromosomes on the short arm of the most proximal oocyte, loading
                  only after nuclear envelope breakdown (<a href="#bib167">Tsai et al., 2008</a>; <a href="#bib35">Csankovszki et al., 2009</a>; <a href="#bib101">Meyer, 2010</a>; <a href="#bib29">Collette et al., 2011</a>; also see X-chromosome dosage compensation). During the meiotic divisions, depletion of condensin I components induces mild
                  defects that are only evident during the second meiotic division (<a href="#bib35">Csankovszki et al., 2009</a>), while depletion of condensin II induces dramatic defects in both meiotic divisions, including chromatin bridges between
                  separating chromosomes in anaphase I and II (<a href="#bib21">Chan et al., 2004</a>; <a href="#bib35">Csankovszki et al., 2009</a>). Thus, the reshaping of chromosomes by condensin during late prophase is an essential aspect of meiosis.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec7"></a>7.&nbsp;Surveillance mechanisms during prophase I
                     </h2>
                  </div>
               </div>
            </div>
            <p>The formation of COs during prophase I of meiosis is essential to promote the accurate segregation of chromosomes into the
               gametes. Therefore, multiple processes that promote the formation of COs, including the establishment of the SC and the progression
               of recombination, are closely monitored. Failures of those processes lead to a meiotic progression delay for the correction
               of errors and/or the induction of apoptosis to eliminate meiocytes where errors persist.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7.1"></a>7.1.&nbsp;Monitoring of SC assembly
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The assembly of the SC plays a central role in mediating meiotic progression. The absence of synapsis on one or more chromosomes
                  causes nuclei to delay exit from zygotene, resulting in a prolongation of chromosome movement and chromosome clustering (<a href="#bib87">MacQueen et al., 2002</a>; <a href="#bib122">Phillips et al., 2005</a>; <a href="#bib18">Carlton et al., 2006</a>; <a href="#bib120">Phillips and Dernburg, 2006</a>; <a href="#bib117">Penkner et al., 2009</a>; <a href="#bib141">Sato et al., 2009</a>; <a href="#bib11">Baudrimont et al., 2010</a>). This delay requires <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002032;class=Gene" target="_blank">htp-1</a>; <a href="http://www.wormbase.org/db/get?name=WBGene00006376;class=Gene" target="_blank">syp-2</a></span> double mutants have no zygotene arrest, leading to the proposal that <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> participates in the generation of an inhibitory signal that blocks exit from zygotene until the SC is installed between all
                  homologous pairs (<a href="#bib94">Martinez-Perez and Villeneuve, 2005</a>). This inhibitory signal appears to involve a soluble pool of <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>, since a mutant version of <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> that fails to associate with axial elements still supports zygotene arrest in SC-deficient mutants (<a href="#bib153">Silva et al., 2014</a>). SC assembly appears to directly antagonize the inhibitory signal that blocks early meiotic progression, as improper SC
                  assembly between sister chromatids in <span class="italic">syp-3(<a href="http://www.wormbase.org/db/get?name=WBVar00088873;class=Variation" target="_blank">me42</a>)</span> mutants results in normal meiotic progression (<a href="#bib156">Smolikov et al., 2007b</a>).
               </p>
               <p><a href="http://www.wormbase.org/db/get?name=WP:CE27297;class=Protein" target="_blank">CHK-2</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> protein kinases have essential roles in organizing and monitoring features of chromosome end mobilization (<a href="#bib55">Harper et al., 2011</a>; <a href="#bib77">Labella et al., 2011</a>). Coincident with chromosome movements in leptotene/zygotene, <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> shows prominent phosphorylation at several residues in its nuclear amino-terminus that are dependent on <a href="http://www.wormbase.org/db/get?name=WP:CE27297;class=Protein" target="_blank">CHK-2</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> (<a href="#bib117">Penkner et al., 2009</a>; <a href="#bib55">Harper et al., 2011</a>; <a href="#bib77">Labella et al., 2011</a>). <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> serine-12 phosphorylation is restricted to <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> molecules concentrated at PCs, whereas <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> phosphorylated at other residues is found throughout the entire nuclear envelope (<a href="#bib117">Penkner et al., 2009</a>). Although PC mobilization still takes place when <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> is rendered non-phosphorylatable, synapsis proceeds at a slower rate. In addition, <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> phosphorylation is required to sustain <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> recruitment to PC attachments in the absence of synapsis; this suggests that phosphorylation of <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> can prolong the time window of chromosome movement (<a href="#bib176">Woglar et al., 2013</a>). The interdependency of <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> phosphorylation by <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> recruitment by phospho-SUN-1 therefore establishes a positive feedback loop to sustain chromosome end-led mobility for as
                  long as necessary to complete synapsis.
               </p>
               <p>In addition to prolonging PC-mediated chromosome movement, the presence of unsynapsed PCs also triggers apoptosis during oocyte
                  meiosis. Induction of this apoptosis by unpaired PCs requires <a href="http://www.wormbase.org/db/get?name=WP:CE01547;class=Protein" target="_blank">PCH-2</a> (<a href="#bib13">Bhalla and Dernburg, 2005</a>). Synapsis at PCs is monitored with the help of <a href="http://www.wormbase.org/db/get?name=WP:CE27781;class=Protein" target="_blank">MES-4</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE30503;class=Protein" target="_blank">MET-1</a> (<a href="http://www.wormbase.org/db/get?name=WP:CE27781;class=Protein" target="_blank">MES-4</a> alone for the X chromosome) (<a href="#bib80">Lamelza and Bhalla, 2012</a>).
               </p>
               <p>In males (chromosomally XO), the single X chromosome remains partnerless during meiosis and thus needs to be shielded from
                  the mechanisms that recognize asynapsed chromosomes and induce delays in meiotic progression (<a href="#bib67">Jaramillo-Lambert et al., 2010</a>). The single X chromosome in males assumes a condensed state and is enriched for repressive chromatin modifiers that are
                  important for shielding the single X from the checkpoint machinery (<a href="#bib22">Checchi and Engebrecht, 2011</a>). Transient pseudosynapsis of X chromosome sister chromatids has been observed and has been proposed to mask the X from being
                  recognized as partnerless (<a href="#bib23">Checchi et al., 2014</a>). Furthermore, <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> aggregates at the single X chromosome in males are not phosphorylated by <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a>, despite its localization to the X PC (<a href="#bib176">Woglar et al., 2013</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7.2"></a>7.2.&nbsp;Monitoring of recombination progression
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Successful completion of meiosis requires formation of COs between each chromosome pair; this in turn requires formation of
                  at least one DSB per homolog pair. However, DSBs also represent possible sources of damage to the genome, and thus, <span class="italic">C. elegans</span> has multiple mechanisms to monitor formation of DSBs and downstream recombination intermediates in order to limit the numbers
                  of DSBs that form, to assure that sufficient DSBs are generated to ensure CO formation, and to prevent germ cells with persistent
                  DNA damage from proceeding to the oocyte meiotic divisions.
               </p>
               <p>A growing body of evidence from multiple organisms suggests that CO intermediates function as a rheostat for the DSB machinery,
                  maintaining DSB formation when levels are low and turning it off when DSBs are high (<a href="#bib81">Lange et al., 2011</a>; <a href="#bib51">Gray et al., 2013</a>; <a href="#bib69">Kauppi et al., 2013</a>; <a href="#bib166">Thacker et al., 2014</a>; <a href="#bib47">Garcia et al., 2015</a>). In <span class="italic">C. elegans</span>, evidence for such a feedback system comes from analysis of <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> and from <a href="http://www.wormbase.org/db/get?name=WP:CE05571;class=Protein" target="_blank">DSB-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE44432;class=Protein" target="_blank">DSB-2</a> (<a href="#bib136">Rosu et al., 2013</a>; <a href="#bib162">Stamper et al., 2013</a>), two proteins that are required for efficient DSB formation and that localize to chromosomes during early meiotic prophase
                  concomitant with DSB formation. Similar to <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> phosphorylation, <a href="http://www.wormbase.org/db/get?name=WP:CE05571;class=Protein" target="_blank">DSB-1</a>/2 chromatin loading depends on <a href="http://www.wormbase.org/db/get?name=WP:CE27297;class=Protein" target="_blank">CHK-2</a> (<a href="#bib136">Rosu et al., 2013</a>; <a href="#bib162">Stamper et al., 2013</a>). In synapsis- or recombination-defective mutants, the zone of <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> phosphorylation and <a href="http://www.wormbase.org/db/get?name=WP:CE05571;class=Protein" target="_blank">DSB-1</a>/2 localization is extended (<a href="#bib162">Stamper et al., 2013</a>; <a href="#bib176">Woglar et al., 2013</a>), suggesting the presence of a checkpoint-like feedback system that monitors formation of CO intermediates and retains nuclei
                  in a DSB permissive state until enough CO-fated events are formed. Failure to form a CO intermediate between the X chromosomes
                  (as seen in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001864;class=Gene" target="_blank">him-5</a></span> and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene" target="_blank">him-8</a></span> mutants) results in delayed meiotic progression, suggesting that this feedback system is capable of detecting the absence
                  of crossing over (or a CO-fated intermediate) on a single chromosome pair (<a href="#bib98">Meneely et al., 2012</a>; <a href="#bib162">Stamper et al., 2013</a>). (<a href="#bib23">Checchi et al., 2014</a>) found that the <a href="http://www.wormbase.org/db/get?name=WP:CE49311;class=Protein" target="_blank">ATM-1</a> protein kinase plays a conserved role in limiting DSB formation, as mutants defective in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000227;class=Gene" target="_blank">atm-1</a></span> have elevated levels of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> foci; however, whether <a href="http://www.wormbase.org/db/get?name=WP:CE49311;class=Protein" target="_blank">ATM-1</a> activation is part of the above-described feedback mechanism or represents a distinct mechanism for limiting DSB numbers
                  remains an open question.
               </p>
               <p>During oogenesis, cells with unrepaired recombination intermediates are subjected to apoptosis dependent on the DNA damage
                  sensors <a href="http://www.wormbase.org/db/get?name=WP:CE29376;class=Protein" target="_blank">MRT-2</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE33217;class=Protein" target="_blank">HPR-9</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE20999;class=Protein" target="_blank">HUS-1</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE29805;class=Protein" target="_blank">CEP-1</a>, the p53 homolog (<a href="#bib48">Gartner et al., 2000</a>; <a href="#bib144">Schumacher et al., 2001</a>); (also see <a href="http://www.wormbook.org/chapters/www_germlinesurvival/germlinesurvival.html" target="_top">Germline survival and apoptosis</a>). Mutants with high levels of resected single stranded DNA, such as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004297;class=Gene" target="_blank">rad-51</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00012767;class=Gene" target="_blank">fcd-2</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00000264;class=Gene" target="_blank">brc-1</a>,</span> or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00020316;class=Gene" target="_blank">brc-2</a></span>, elicit massive apoptosis (<a href="#bib48">Gartner et al., 2000</a>; <a href="#bib92">Martin et al., 2005</a>; <a href="#bib2">Adamo et al., 2008</a>; <a href="#bib1">Adamo et al., 2010</a>). Interestingly, the pro-CO factors MSH-4, <a href="http://www.wormbase.org/db/get?name=WP:CE26678;class=Protein" target="_blank">MSH-5</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE43850;class=Protein" target="_blank">ZHP-3</a> are required for the induction of apoptosis irrespective of DNA damage checkpoint activation, as evaluated by the transcriptional
                  activation of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001170;class=Gene" target="_blank">egl-1</a></span> (<a href="#bib152">Silva et al., 2013</a>). The ability of unrepaired meiotic DSBs to trigger DNA damage-checkpoint-mediated apoptosis also provides a means to cull
                  nuclei that had entered meiotic prophase with <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001882;class=Phenotype" target="_blank">aneuploidy</a> accumulated during the preceding mitotic divisions (<a href="#bib163">Stevens et al., 2013</a>).
               </p>
               <p>In males, where apoptosis is not triggered, many features of damage sensing and signaling nevertheless are activated and thereby
                  contribute to faithful processing of recombination intermediates (<a href="#bib67">Jaramillo-Lambert et al., 2010</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7.3"></a>7.3.&nbsp;Proteins that promote normal prophase I progression
                        </h3>
                     </div>
                  </div>
               </div>
               <p>After meiotic entry (which is covered in Germline proliferation and its control) oocytes remain in G2 throughout their progression
                  along the gonad. Progression through prophase I must be coordinated with the cell cycle and oocyte growth for proper gamete
                  development. <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00009173;class=Gene" target="_blank">prom-1</a></span> mutants enter meiosis and express axis and SC proteins. However, the processes of pairing and synapsis are defective: few
                  nuclei display the asymmetrical chromatin distribution characteristic of the transition zone (leptotene/zygotene), loading
                  of axis and SC proteins is delayed, and the synapsis that eventually does occur is often between nonhomologous chromosomes
                  (<a href="#bib65">Jantsch et al., 2007</a>). One known function of the SCFprom-1 ubiquitin ligase is the degradation of <a href="http://www.wormbase.org/db/get?name=WP:CE33761;class=Protein" target="_blank">CYE-1</a>, which plays a role in the germline proliferative zone (<a href="#bib43">Fox et al., 2011</a>). It is therefore possible <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00009173;class=Gene" target="_blank">prom-1</a></span> connects meiotic events to cell cycle progression.
               </p>
               <p>ERK/MAP kinase <a href="http://www.wormbase.org/db/get?name=WP:CE01583;class=Protein" target="_blank">MPK-1</a> is required for progression to late pachytene; cells lacking <a href="http://www.wormbase.org/db/get?name=WP:CE01583;class=Protein" target="_blank">MPK-1</a> never enter a diplotene-like state (<a href="#bib25">Church et al., 1995</a>; <a href="#bib79">Lackner and Kim, 1998</a>). This is consistent with accumulation of the phosphorylated&#8212;hence activated&#8212;form of <a href="http://www.wormbase.org/db/get?name=WP:CE01583;class=Protein" target="_blank">MPK-1</a> in mid-pachytene (<a href="#bib82">Lee et al., 2007</a>). The transition from homolog-directed DSB repair to sister-directed repair, with the accompanying loss of <a href="http://www.wormbase.org/db/get?name=WP:CE41010;class=Protein" target="_blank">RAD-50</a> dependence for <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> loading (<a href="#sec5.6" title="5.6.&nbsp;Window of opportunity for CO formation">Section 5.6</a>), depends on <a href="http://www.wormbase.org/db/get?name=WP:CE01583;class=Protein" target="_blank">MPK-1</a> (<a href="#bib56">Hayashi et al., 2007</a>). A key determinant of the length of prophase during <span class="italic">C. elegans</span> oogenesis is the translational repressor <a href="http://www.wormbase.org/db/get?name=WP:CE27062;class=Protein" target="_blank">LIN-41</a>, which coordinates oocyte growth with meiotic maturation. In <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003026;class=Gene" target="_blank">lin-41</a></span> mutants pachytene cells prematurely cellularize, diplotene does not occur, and oocytes prematurely enter M phase. This premature
                  M phase entry is elicited in part by translational derepression of a <a href="http://www.wormbase.org/db/get?name=WP:CE00315;class=Protein" target="_blank">CDK-1</a> inhibitor (<a href="#bib160">Spike et al., 2014</a>). For further reading on oocyte maturation and oocyte release from prophase arrest, see Control of oocyte meiotic maturation
                  and fertilization as well as <a href="#bib73">Kim et al., 2013</a>.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec8"></a>8.&nbsp;Chromosome segregation during the meiotic divisions
                     </h2>
                  </div>
               </div>
            </div>
            <p>Proper segregation of chromosomes at meiosis I requires orientation of homologs toward opposite spindle poles and the localized
               release of sister chromatid cohesion from the short arms of the bivalents at anaphase onset (see <a href="#sec6.1" title="6.1.&nbsp;CO-triggered chromosome remodeling">Section 6.1</a>). During metaphase I, outer kinetochore components are seen forming two opposing cup-like structures that encase each half
               of the bivalent (<a href="#bib63">Howe et al., 2001</a>; <a href="#bib104">Monen et al., 2005</a>). This organization promotes the mono-orientation of sister chromatids on the spindle, since sister chromatids of the long
               arm of the bivalent are encased within the same cup-like structure and thus behave as a single unit. Mono-orientation of sister
               chromatids in metaphase I also requires normal sister chromatid cohesion, as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004333;class=Gene" target="_blank">rec-8</a></span> mutants show premature separation of sisters at anaphase I (<a href="#bib149">Severson et al., 2009</a>).
            </p>
            <p>During oogenesis, meiotic chromosome segregation takes place in the context of a barrel-shaped acentriolar spindle that needs
               to be correctly positioned, in close association with the cortex, within the large volume of the fertilized embryo (<a href="#bib5">Albertson and Thomson, 1993</a>). Proper spindle assembly in oocytes requires <a href="http://www.wormbase.org/db/get?name=WP:CE32479;class=Protein" target="_blank">MEI-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE11320;class=Protein" target="_blank">MEI-2</a>, the components of the microtubule severing complex Katanin (<a href="#bib90">Mains et al., 1990</a>; <a href="#bib26">Clark-Maguire and Mains, 1994</a>; <a href="#bib161">Srayko et al., 2000</a>), the kinesin <a href="http://www.wormbase.org/db/get?name=WP:CE07978;class=Protein" target="_blank">KLP-18</a> (<a href="#bib148">Segbert et al., 2003</a>), and the calponin homology-domain protein <a href="http://www.wormbase.org/db/get?name=WP:CE15710;class=Protein" target="_blank">ASPM-1</a> (<a href="#bib170">van der Voet et al., 2009</a>; <a href="#bib30">Connolly et al., 2014</a>). Correct positioning of the meiotic spindle in oocytes is a two-step process that involves translocation of the spindle
               to the cortex, which occurs before metaphase I, followed by a 90&deg; rotation that renders the spindle perpendicular to the cortex
               before the onset of anaphase I (reviewed in <a href="#bib42">Fabritius et al., 2011</a>). Spindle translocation requires kinesin-1, <a href="http://www.wormbase.org/db/get?name=WP:CE32479;class=Protein" target="_blank">MEI-1</a>, and microtubules (<a href="#bib180">Yang et al., 2003</a>; <a href="#bib179">Yang et al., 2005</a>), while spindle rotation is promoted by the anaphase promoting complex, dynein, and a complex formed by <a href="http://www.wormbase.org/db/get?name=WP:CE18951;class=Protein" target="_blank">LIN-5</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE15710;class=Protein" target="_blank">ASPM-1</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE13902;class=Protein" target="_blank">CMD-1</a> (<a href="#bib179">Yang et al., 2005</a>; <a href="#bib41">Ellefson and McNally, 2009</a>; <a href="#bib170">van der Voet et al., 2009</a>).
            </p>
            <p>Detailed imaging of microtubules and kinetochores during meiosis I shows that bivalents are surrounded by robust microtubule
               bundles running along their sides, and that the mid-bivalent region is surrounded by a ring-like structure containing the
               chromosome passenger complex and the chromokinesin <a href="http://www.wormbase.org/db/get?name=WP:CE16631;class=Protein" target="_blank">KLP-19</a> (<a href="#bib174">Wignall and Villeneuve, 2009</a>). This suggests that lateral associations with microtubules (rather than canonical end-on kinetochore attachments) facilitate
               chromosome movements prior to anaphase. Although kinetochores are required for proper chromosome orientation during congression,
               potentially helping to align chromosomes within the lateral bundles, the migration of chromosomes during anaphase I is kinetochore
               independent (<a href="#bib40">Dumont et al., 2010</a>). Instead, lateral microtubule associations remain intact during anaphase to facilitate segregation (<a href="#bib105a">Muscat, et.al. 2015</a>). At anaphase onset, the mid-bivalent ring structure containing <a href="http://www.wormbase.org/db/get?name=WP:CE16631;class=Protein" target="_blank">KLP-19</a> is removed from chromosomes and is left behind in the center of the spindle (<a href="#bib40">Dumont et al., 2010</a>), allowing minus-end forces to drive poleward movement along lateral bundles (Muscat, et.al. 2015). Moreover, spindle elongation
               also facilitates separation, potentially stimulated by another ring component, <a href="http://www.wormbase.org/db/get?name=WP:CE37546;class=Protein" target="_blank">CLS-2</a>/CLASP (<a href="#bib40">Dumont et al., 2010</a>). Kinetochores are disassembled shortly after the onset of anaphase I, and are reassembled by metaphase II. In meiosis II,
               each sister chromatid is encased in a cup-like kinetochore, and <a href="http://www.wormbase.org/db/get?name=WP:CE16631;class=Protein" target="_blank">KLP-19</a> and the chromosome passenger complex localize to a ring-like structure that encircles the region of remaining contact between
               the two sister chromatids (<a href="#bib174">Wignall and Villeneuve, 2009</a>; <a href="#bib40">Dumont et al., 2010</a>).
            </p>
            <p>The processes that mediate meiotic chromosome segregation differ between oogenesis and spermatogenesis (<a href="#bib5">Albertson and Thomson, 1993</a>; <a href="#bib151">Shakes et al., 2009</a>; <a href="#bib174">Wignall and Villeneuve, 2009</a>; <a href="#bib145">Schvarzstein et al., 2013</a>). In contrast to oocyte meiotic spindles, spermatocyte meiotic spindles assume a more traditional configuration. They have
               centrioles at their poles and the metaphase spindles have high microtubule density adjacent to kinetochore domains at the
               ends of the bivalents, indicative of end-on attachments between microtubules and kinetochores. Furthermore, anaphase separation
               of chromosomes in spermatocytes is driven by separation of spindle poles. Finally, completion of the meiotic divisions produces
               markedly different outcomes in spermatogenesis, where four functional haploid gametes are produced, and oogenesis, where one
               haploid gamete and two degenerate polar bodies are formed (also see Control of oocyte meiotic maturation and fertilization).
               Interestingly, the asymmetry of oocyte meiosis can partially correct for pre-existing trisomies, since in triplo-X animals
               the extra X chromosome is preferentially extruded into the first polar body, thus reducing the incidence of aneuploid progeny
               (<a href="#bib31">Cortes et al., 2015</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec9"></a>9.&nbsp;Summary
                     </h2>
                  </div>
               </div>
            </div>
            <p><span class="italic">C. elegans</span> has proven to be a very productive organism for the study of meiosis. The excellent cytology, coupled with the ease of isolation
               of meiotic mutants, has contributed to our understanding of the complex series of events necessary to produce haploid gametes.
               However, understanding the molecular mechanisms that regulate homolog pairing, meiotic recombination, the changes in structure
               that chromosomes undergo during meiotic prophase, the process of chromosome segregation, and the quality control of meiosis
               remain important goals for future studies. The recent development of efficient transgenic tools, combined with super-resolution
               microscopy, long-term in vivo imaging of meiotic chromosomes, rapid methods to map recombination events, and the use of proteomic
               approaches means that <span class="italic">C. elegans</span> will remain an excellent model organism to address these questions over the coming years.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec10"></a>10.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>We thank Barbara Meyer, Anne Villeneuve, and David Greenstein for critically reading the manuscript and members of the Jantsch,
               Martinez-Perez, and Yanowitz labs for comments and discussions. We are grateful to Patricia Hamminger, Sophia Millonigg, Alexander
               Woglar, and Sarah Testori for figure preparation. The authors labs were funded by the FWF (VJ), an MRC core funded grant (E
               M-P), NIH (JLY/VJ &amp; KJH), and the March of Dimes (JLY).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec11"></a>11.&nbsp;<span class="bold"><strong>Table 1. List of meiotic genes discussed in this chapter</strong></span></h2>
                  </div>
               </div>
            </div>
            <p>Genes are organized in alphabetical order and numbers in brackets indicate the section(s) within the main text where the functions
               of a given protein are discussed.
            </p>
            <div class="table"><a name="table1"></a><p class="title"></p>
               <table summary="" border="1">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr valign="bottom">
                        <th align="center" valign="bottom"><span class="bold"><strong>Gene Name</strong></span></th>
                        <th align="center" valign="bottom"><span class="bold"><strong>Ortholog/Homolog</strong></span></th>
                        <th align="center" valign="bottom"><span class="bold"><strong>Molecular Function (known or inferred)</strong></span></th>
                        <th align="center" valign="bottom"><span class="bold"><strong>Subcellular Localization (if known)</strong></span></th>
                        <th align="center" valign="bottom"><span class="bold"><strong>Meiotic Role</strong></span></th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000099;class=Gene" target="_blank">air-2</a></span></td>
                        <td align="center" valign="top">Aurora/ IPL Kinase</td>
                        <td align="center" valign="top">Kinase</td>
                        <td align="center" valign="top">Recruited to the short arm of diakinesis bivalents</td>
                        <td align="center" valign="top">Loss impairs chromosome segregation at meiosis I; phosphorylates <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> in vitro. (<a href="#sec6.1" title="6.1.&nbsp;CO-triggered chromosome remodeling">Section 6.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00008107;class=Gene" target="_blank">aspm-1</a></span></td>
                        <td align="center" valign="top">ASPM</td>
                        <td align="center" valign="top">Interaction with calmodulin and microtubules</td>
                        <td align="center" valign="top">Meiotic spindles</td>
                        <td align="center" valign="top">Required for proper spindle assembly in oocytes and to promote spindle rotation in a complex with <a href="http://www.wormbase.org/db/get?name=WP:CE13902;class=Protein" target="_blank">CMD-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE18951;class=Protein" target="_blank">LIN-5</a>. (<a href="#sec8" title="8.&nbsp;Chromosome segregation during the meiotic divisions">Section 8</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000264;class=Gene" target="_blank">brc-1</a></span></td>
                        <td align="center" valign="top">BRCA1</td>
                        <td align="center" valign="top">DNA repair</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Required for intersister DSB repair (<a href="#sec5.2.4" title="5.2.4.&nbsp;Partner choice">Section 5.2.4</a>); mutation results in high levels of resected ssDNA and massive germline apoptosis (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00020316;class=Gene" target="_blank">brc-2</a></span></td>
                        <td align="center" valign="top">BRCA2</td>
                        <td align="center" valign="top">DNA repair</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Facilitates nuclear entry of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a>, loading of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> on ssDNA, and D loop formation (<a href="#sec5.2.2" title="5.2.2.&nbsp;Strand exchange and formation of downstream repair intermediates">Section 5.2.2</a>). Mutation results in high levels of resected ssDNA and massive germline apoptosis (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00009254;class=Gene" target="_blank">capg-1</a></span></td>
                        <td align="center" valign="top">hCAP-G</td>
                        <td align="center" valign="top">Condensin I complex</td>
                        <td align="center" valign="top">Nuclear localization in mitotic and meiotic prophase nuclei. Associates with short arm of bivalents during late diakinesis
                           and metaphase I, and between sister chromatids at metaphase II
                        </td>
                        <td align="center" valign="top">Heterozygous null mutants show altered distribution of DSBs and COs, increased CO frequencies, and increased pachytene axis
                           length (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>). Required for chromosome segregation (<a href="#sec6.2" title="6.2.&nbsp;Chromosome condensation">Section 6.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00010093;class=Gene" target="_blank">capg-2</a></span></td>
                        <td align="center" valign="top">hCAP-G2</td>
                        <td align="center" valign="top">Condensin II complex</td>
                        <td align="center" valign="top">Decorates chromatin in diakinesis oocytes and during metaphase I and II</td>
                        <td align="center" valign="top">Required for chromosome condensation and segregation. (<a href="#sec6.2" title="6.2.&nbsp;Chromosome condensation">Section 6.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004048;class=Gene" target="_blank">cep-1</a></span></td>
                        <td align="center" valign="top">p53</td>
                        <td align="center" valign="top">DNA binding, transciptional activation</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">DNA damage sensing, involved in DNA damage-induced apoptosis in the germ line. (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000499;class=Gene" target="_blank">chk-2</a></span></td>
                        <td align="center" valign="top">CHK2</td>
                        <td align="center" valign="top">Kinase</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Master regulator of early prophase events. Required for loading of autosomal PC proteins (ZIMs); required for formation of
                           <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> aggregates (<a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>). Required for homolog pairing and normal loading of SC in early prophase (<a href="#sec4.3.1" title="4.3.1.&nbsp;Regulation of SC assembly">Section 4.3.1</a>). Required for N-terminal phosphorylation of <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> shortly after meiotic entry (<a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>). Required for DSB formation and loading of <a href="http://www.wormbase.org/db/get?name=WP:CE05571;class=Protein" target="_blank">DSB-1</a>/2 (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000592;class=Gene" target="_blank">coh-3</a></span></td>
                        <td align="center" valign="top">Rad21L/Rec8</td>
                        <td align="center" valign="top">Meiosis-specific kleisin subunit of cohesin complex</td>
                        <td align="center" valign="top">Chromosome axes</td>
                        <td align="center" valign="top"><a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE27409;class=Protein" target="_blank">COH-4</a> are highly identical and functionally redundant. Loss of <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE27409;class=Protein" target="_blank">COH-4</a> impairs CO formation and SC assembly. Loss of <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE27409;class=Protein" target="_blank">COH-4</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> causes gross defects in sister chromatid cohesion and prevents assembly of axial elements. (<a href="#sec4.1" title="4.1.&nbsp;Meiosis-specific cohesin complexes">Section 4.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00021560;class=Gene" target="_blank">coh-4</a></span></td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000592;class=Gene" target="_blank">coh-3</a></span></td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000592;class=Gene" target="_blank">coh-3</a></span> (above).
                        </td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000592;class=Gene" target="_blank">coh-3</a></span> (above).
                        </td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000592;class=Gene" target="_blank">coh-3</a></span> (above).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00008082;class=Gene" target="_blank">com-1</a></span></td>
                        <td align="center" valign="top">CtlP/Sae2</td>
                        <td align="center" valign="top">DNA repair; resection of DSBs</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Promotes processing of <a href="http://www.wormbase.org/db/get?name=WP:CE49824;class=Protein" target="_blank">SPO-11</a> DSBs to facilitate <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> loading; suppresses NHEJ. (<a href="#sec5.2.1" title="5.2.1.&nbsp;Resection">Section 5.2.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00022172;class=Gene" target="_blank">cosa-1</a></span></td>
                        <td align="center" valign="top">CTND1</td>
                        <td align="center" valign="top">CO designation</td>
                        <td align="center" valign="top">Late pachytene&#8212;one focus per chromosome pair (presumed to mark CO position)</td>
                        <td align="center" valign="top">Designates recombination intermediates for CO formation; required for CO formation. (<a href="#sec5.2.3" title="5.2.3.&nbsp;CO designation">Section 5.2.3</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00020068;class=Gene" target="_blank">cra-1</a></span></td>
                        <td align="center" valign="top">NAA25</td>
                        <td align="center" valign="top">N(alpha)-acetyl-transferase</td>
                        <td align="center" valign="top">Localizes to meiotic nuclei throughout prophase</td>
                        <td align="center" valign="top">Required to promote normal loading of SC proteins (<a href="#sec4.3.1" title="4.3.1.&nbsp;Regulation of SC assembly">Section 4.3.1</a>) and for regulation of histone acetylation and DSB formation (<a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00000871;class=Gene" target="_blank">cye-1</a></span></td>
                        <td align="center" valign="top">CCNE1</td>
                        <td align="center" valign="top">E-type cyclin</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Promotes mitotic proliferation in the distal germ line. Negatively regulated in meiotic cells by <a href="http://www.wormbase.org/db/get?name=WP:CE41801;class=Protein" target="_blank">PROM-1</a> ubiquitin ligase. (<a href="#sec7.3" title="7.3.&nbsp;Proteins that promote normal prophase I progression">Section 7.3</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001085;class=Gene" target="_blank">dpy-26</a></span></td>
                        <td align="center" valign="top">hCAP-H</td>
                        <td align="center" valign="top">Condensin I complex</td>
                        <td align="center" valign="top">Nuclear localization in mitotic and meiotic prophase nuclei. Associates with short arm of metaphase I bivalents and between
                           sister chromatids at metaphase II
                        </td>
                        <td align="center" valign="top">Heterozygous null mutants show altered distribution of DSBs and COs, increased CO frequencies, and increased pachytene axis
                           length (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>). Required for chromosome segregation (<a href="#sec6.2" title="6.2.&nbsp;Chromosome condensation">Section 6.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001087;class=Gene" target="_blank">dpy-28</a></span></td>
                        <td align="center" valign="top">hCAP-D2</td>
                        <td align="center" valign="top">Condensin I complex</td>
                        <td align="center" valign="top">Same as <span class="italic">dpy-</span>26 (above).
                        </td>
                        <td align="center" valign="top">Heterozygotes same as <span class="italic">dpy-</span>26 (above). Homozygous mutants of hypomorph allele show changes in distribution of DSBs and COs without altering CO frequency
                           (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>). Required for chromosome segregation (<a href="#sec6.2" title="6.2.&nbsp;Chromosome condensation">Section 6.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00008580;class=Gene" target="_blank">dsb-1</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Localizes to chromatin in transition zone and early pachytene nuclei</td>
                        <td align="center" valign="top">Involved in DSB formation; regulates competence for DSB formation (<a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>). Localizes to chromosomes, dependent on <a href="http://www.wormbase.org/db/get?name=WP:CE27297;class=Protein" target="_blank">CHK-2</a>. Localization prolonged in synapsis or recombination mutants; suggests a role in maintenance of DSB-permissive state (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00194892;class=Gene" target="_blank">dsb-2</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00008580;class=Gene" target="_blank">dsb-1</a></span> (above).
                        </td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00008580;class=Gene" target="_blank">dsb-1</a></span> (above). Loss of <a href="http://www.wormbase.org/db/get?name=WP:CE44432;class=Protein" target="_blank">DSB-2</a> from chromosomes may correlate with switch from homolog to sister repair (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001170;class=Gene" target="_blank">egl-1</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Positive regulation of apoptosis</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Positive regulator of apoptosis. (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00009731;class=Gene" target="_blank">exo-1</a></span></td>
                        <td align="center" valign="top">exo1</td>
                        <td align="center" valign="top">DNA repair; exonuclease</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Involved in resection of meiotic DSBs. (<a href="#sec5.2.1" title="5.2.1.&nbsp;Resection">Section 5.2.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00012767;class=Gene" target="_blank">fcd-2</a></span></td>
                        <td align="center" valign="top">FANCD2</td>
                        <td align="center" valign="top">DNA repair</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Mutations result in high levels of resected ssDNA and massive germline apoptosis. (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001748;class=Gene" target="_blank">gsp-2</a></span></td>
                        <td align="center" valign="top">PP1CA</td>
                        <td align="center" valign="top">Protein Phosphatase I</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Antagonizes <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> recruitment to chromosomes and physically interacts with <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a>; loss leads to premature sister chromatid separation at Anaphase I. (<a href="#sec6.1" title="6.1.&nbsp;CO-triggered chromosome remodeling">Section 6.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00011819;class=Gene" target="_blank">hal-2</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Localized to the nucleoplasm in all germline nuclei.</td>
                        <td align="center" valign="top">Promotes homolog pairing, prevents inappropriate loading of SYP proteins; required for DSB formation; required for mechanism
                           that couples cessation of chromosome movement with SC assembly. (<a href="#sec4.3.1" title="4.3.1.&nbsp;Regulation of SC assembly">Section 4.3.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001833;class=Gene" target="_blank">hcp-6</a></span></td>
                        <td align="center" valign="top">hCAP-D3</td>
                        <td align="center" valign="top">Condensin II complex</td>
                        <td align="center" valign="top">Associates with chromatin from diplotene, persisting during metaphase I and II</td>
                        <td align="center" valign="top">Required for chromosome condensation and segregation. (<a href="#sec6.2" title="6.2.&nbsp;Chromosome condensation">Section 6.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00021905;class=Gene" target="_blank">helq-1</a></span></td>
                        <td align="center" valign="top">HELQ</td>
                        <td align="center" valign="top">DNA repair; helicase</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Promotes <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> turnover (removal of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> from D loops); redundant with <a href="http://www.wormbase.org/db/get?name=WP:CE00094;class=Protein" target="_blank">RFS-1</a>. (<a href="#sec5.2.2" title="5.2.2.&nbsp;Strand exchange and formation of downstream repair intermediates">Section 5.2.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001862;class=Gene" target="_blank">him-3</a></span></td>
                        <td align="center" valign="top">HORMAD1; Hop1</td>
                        <td align="center" valign="top">HORMA-domain protein; Component of chromosome axis</td>
                        <td align="center" valign="top">Localizes to axial elements throughout meiotic prophase</td>
                        <td align="center" valign="top">HORMA domain flanked by short N- and C- termini; required for homolog recognition. Promotes use of homolog as DSB repair template;
                           absence leads to dramatic reduction in SC. Required to prevent premature centriole disengagement in spermatocyte meiosis II
                           (<a href="#sec4.2" title="4.2.&nbsp;HORMA-domain proteins">Section 4.2</a>.) Non-null mutations can affect chromosome axis integrity and regulation of meiotic recombination (<a href="#sec5.5" title="5.5.&nbsp;Interplay between recombination and chromosome structure">Section 5.5</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001864;class=Gene" target="_blank">him-5</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">promotion of DSB formation</td>
                        <td align="center" valign="top">Localizes to autosomes from mitotic region through late pachytene</td>
                        <td align="center" valign="top">Required for normal DSB formation on <span class="italic">X</span> chromosomes. mutation reduces DSB levels 4-fold, but only <span class="italic">X</span> typically lacks CO. Required for normal distribution of COs genome-wide (<a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>, <a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>, <a href="#sec5.4" title="5.4.&nbsp;CO interference, assurance, and homeostasis">Section 5.4</a>). In absence, chromosome clustering/mobility extended and meiotic progression delayed (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001865;class=Gene" target="_blank">him-6</a></span></td>
                        <td align="center" valign="top">BLM; RecQ</td>
                        <td align="center" valign="top">DNA repair; Holliday junction resolvase</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Resolvase activity (in conjunction with <a href="http://www.wormbase.org/db/get?name=WP:CE24855;class=Protein" target="_blank">XPF-1</a>). Thought to enforce crossover-biased resolution of CO-designated intermediates (<a href="#sec5.2.5" title="5.2.5.&nbsp;CO resolution">Section 5.2.5</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene" target="_blank">him-8</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">X chromosome pairing and synapsis</td>
                        <td align="center" valign="top">Binds X pairing center</td>
                        <td align="center" valign="top">Required for homolog alignment and synapsis of <span class="italic">X.</span> Recruitment site for polo-like kinases <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE26649;class=Protein" target="_blank">PLK-1</a> (<a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>, <a href="#sec4.3" title="4.3.&nbsp;The SC">Section 4.3</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001874;class=Gene" target="_blank">him-17</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Promotion of DSB formation</td>
                        <td align="center" valign="top">Localizes to chromatin throughout the germ line</td>
                        <td align="center" valign="top">Mutants accumulate aberrant histone modifications and display impaired DSB formation. Required for <a href="http://www.wormbase.org/db/get?name=WP:CE09062;class=Protein" target="_blank">HIM-5</a> localization to chromosomes. (<a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>, <a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00011415;class=Gene" target="_blank">him-18</a></span></td>
                        <td align="center" valign="top">SLX4</td>
                        <td align="center" valign="top">DNA repair; Holliday junction resolution</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Scaffold for resolvases. (<a href="#sec5.2.5" title="5.2.5.&nbsp;CO resolution">Section 5.2.5</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00022390;class=Gene" target="_blank">him-19</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Involved in homolog recognition, defects in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00022390;class=Gene" target="_blank">him-19</a></span> mutants worsen with age. (<a href="#sec3.3" title="3.3.&nbsp;PC movements promote efficient homolog pairing">Section 3.3</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001997;class=Gene" target="_blank">hpr-9</a></span></td>
                        <td align="center" valign="top">RAD9</td>
                        <td align="center" valign="top">DNA damage sensing; 9-1-1 complex</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">DNA damage sensing, involved in DNA damage-induced apoptosis in the germ line. (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002032;class=Gene" target="_blank">htp-1</a></span></td>
                        <td align="center" valign="top">HORMAD1; Hop1 (yeast)</td>
                        <td align="center" valign="top">HORMA-domain protein; Component of chromosome axis</td>
                        <td align="center" valign="top">Localizes to axial elements throughout meiotic prophase, but becomes restricted to the long arms of bivalents during late
                           pachytene. Large pool also present as nuclear soluble protein.
                        </td>
                        <td align="center" valign="top">Same as <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a> (below), except <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> is required for homolog recognition, to prevent non-homologous synapsis, and for CO formation. Promotes use of homolog as
                           DSB repair template (<a href="#sec4.2" title="4.2.&nbsp;HORMA-domain proteins">Section 4.2</a>). Thought to help generate inhibitory signal that blocks zygotene exit until all homologous pairs have SC installed (<a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002033;class=Gene" target="_blank">htp-2</a></span></td>
                        <td align="center" valign="top">HORMAD1; Hop1 (yeast)</td>
                        <td align="center" valign="top">HORMA-domain protein; Component of chromosome axis</td>
                        <td align="center" valign="top">Localizes to axial elements throughout meiotic prophase, but becomes restricted to the long arms of bivalents during late
                           pachytene.
                        </td>
                        <td align="center" valign="top">HORMA domain flanked by short N- and C- termini. <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a> are 82% identical; <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a> is largely redundant with <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>. Together <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a> are required for regulated cohesion release during meiotic divisions (<a href="#sec6.1" title="6.1.&nbsp;CO-triggered chromosome remodeling">Section 6.1</a>) and to prevent premature centriole disengagement during meiosis II in spermatogenesis (<a href="#sec4.2" title="4.2.&nbsp;HORMA-domain proteins">Section 4.2</a>). Loss of <a href="http://www.wormbase.org/db/get?name=WP:CE37734;class=Protein" target="_blank">HTP-2</a> does not affect pairing or CO formation (<a href="#sec4.2" title="4.2.&nbsp;HORMA-domain proteins">Section 4.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002034;class=Gene" target="_blank">htp-3</a></span></td>
                        <td align="center" valign="top">HORMAD1; Hop1 (yeast)</td>
                        <td align="center" valign="top">HORMA-domain protein; Component of chromosome axis</td>
                        <td align="center" valign="top">Localizes to nuclei in the mitotic compartment of the germ line. Localizes to axial elements throughout meiotic prophase</td>
                        <td align="center" valign="top">HORMA domain flanked by short N-terminus and long C-terminus. Component of meiotic chromosome axis, required for efficient
                           DSB induction. Required for loading of <a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a>/2, and <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> cohesin, for SC assembly, and for acquisition of posttranslational modifications on axis-associated chromatin. (<a href="#sec4.2" title="4.2.&nbsp;HORMA-domain proteins">Section 4.2</a>, <a href="#sec4.3" title="4.3.&nbsp;The SC">Section 4.3</a>). Promotes DSB formation (<a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>). Antagonizes separation of axial elements in early-mid pachytene (<a href="#sec5.6" title="5.6.&nbsp;Window of opportunity for CO formation">Section 5.6</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002042;class=Gene" target="_blank">hus-1</a></span></td>
                        <td align="center" valign="top">HUS1</td>
                        <td align="center" valign="top">DNA damage sensing; 9-1-1 complex</td>
                        <td align="center" valign="top">Localizes to chromatin following DNA damage</td>
                        <td align="center" valign="top">DNA damage sensing, involved in DNA damage-induced apoptosis in the germ line. (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00016202;class=Gene" target="_blank">kle-2</a></span></td>
                        <td align="center" valign="top">hCAP-H2</td>
                        <td align="center" valign="top">Condensin II complex</td>
                        <td align="center" valign="top">Decorates chromatin in diakinesis oocytes and during metaphase I and II</td>
                        <td align="center" valign="top">Heterozygous null mutants show altered distribution of DSBs and COs, increased CO frequencies, and increased pachytene axis
                           length (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>). Required for chromosome condensation and segregation. (<a href="#sec6.2" title="6.2.&nbsp;Chromosome condensation">Section 6.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002228;class=Gene" target="_blank">klp-18</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Kinesin</td>
                        <td align="center" valign="top">Meiotic spindles; higher concentration at spindle poles</td>
                        <td align="center" valign="top">Required for assembly of acentrosomal spindles in oocytes. (<a href="#sec8" title="8.&nbsp;Chromosome segregation during the meiotic divisions">Section 8</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002229;class=Gene" target="_blank">klp-19</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Chromokinesin</td>
                        <td align="center" valign="top">Forms ring-like structure around mid-bivalent region in meiosis I, and around region of remaining contact between sister chromatids
                           in meiosis I
                        </td>
                        <td align="center" valign="top">Mediates anti-poleward chromosome movement via lateral microtubules. (<a href="#sec8" title="8.&nbsp;Chromosome segregation during the meiotic divisions">Section 8</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00007278;class=Gene" target="_blank">lab-1</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Targets <a href="http://www.wormbase.org/db/get?name=WP:CE01319;class=Protein" target="_blank">GSP-2</a> to chromosomes
                        </td>
                        <td align="center" valign="top">Associates with axial elements and becomes restricted to the long arms of diakinesis bivalents</td>
                        <td align="center" valign="top">Axial element component. Targets phosphatase 1 (<a href="http://www.wormbase.org/db/get?name=WP:CE01319;class=Protein" target="_blank">GSP-2</a>) to chromosomes and antagonizes <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> recruitment (<a href="#sec6.1" title="6.1.&nbsp;CO-triggered chromosome remodeling">Section 6.1</a>). Required for full SC assembly and for normal sister chromatid cohesion (<a href="#sec4.1" title="4.1.&nbsp;Meiosis-specific cohesin complexes">Section 4.1</a>, <a href="#sec4.3.1" title="4.3.1.&nbsp;Regulation of SC assembly">Section 4.3.1</a>). Regulates release of sister chromatid cohesion during the first meiotic division (<a href="#sec6.1" title="6.1.&nbsp;CO-triggered chromosome remodeling">Section 6.1</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003026;class=Gene" target="_blank">lin-41</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Translational repressor</td>
                        <td align="center" valign="top">Present, mostly in cytoplasm, from mid pachytene until oocyte maturation</td>
                        <td align="center" valign="top">Coordinates oocyte growth with meiotic maturation. Required to prevent premature entrance into M phase. (<a href="#sec7.3" title="7.3.&nbsp;Proteins that promote normal prophase I progression">Section 7.3</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003183;class=Gene" target="_blank">mei-1</a></span></td>
                        <td align="center" valign="top">KATNAL1</td>
                        <td align="center" valign="top">Microtubule severing</td>
                        <td align="center" valign="top">Spindles and chromosomes during the meiotic divisions</td>
                        <td align="center" valign="top">With <a href="http://www.wormbase.org/db/get?name=WP:CE11320;class=Protein" target="_blank">MEI-2</a> forms the katanin microtubule severing complex. Required for proper assembly and positioning of meiotic spindle in oocytes.
                           (<a href="#sec8" title="8.&nbsp;Chromosome segregation during the meiotic divisions">Section 8</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003184;class=Gene" target="_blank">mei-2</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Microtubule severing</td>
                        <td align="center" valign="top">Localizes in the cytoplasm of late diakinesis oocytes and to spindles and chromatin during meiotic divisions</td>
                        <td align="center" valign="top">Same as for <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003183;class=Gene" target="_blank">mei-1</a></span> (above).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003222;class=Gene" target="_blank">mes-4</a></span></td>
                        <td align="center" valign="top">WHSC1; SEDB1</td>
                        <td align="center" valign="top">Histone methyltransferase</td>
                        <td align="center" valign="top">Localizes to autosomes in mitotic region, late pachytene, and oocytes</td>
                        <td align="center" valign="top">Involved in monitoring synapsis of autosomes and X chromosomes. (<a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003367;class=Gene" target="_blank">mix-1</a></span></td>
                        <td align="center" valign="top">SMC2</td>
                        <td align="center" valign="top">Condensin I and II complexes</td>
                        <td align="center" valign="top">Associates with chromatin from diplotene (condensin II), and with the short arm of metaphase I bivalents and between sister
                           chomatids during metaphase II (condensin I)
                        </td>
                        <td align="center" valign="top">Heterozygous null mutants show altered distribution of DSBs and COs, increased CO frequencies, and increased pachytene axis
                           length (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>). Required for chromosome condensation and segregation (<a href="#sec6.2" title="6.2.&nbsp;Chromosome condensation">Section 6.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00016603;class=Gene" target="_blank">met-1</a></span></td>
                        <td align="center" valign="top">SETD2</td>
                        <td align="center" valign="top">Histone methyltransferase</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Involved in monitoring synapsis of autosomes. (<a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00019883;class=Gene" target="_blank">met-2</a></span></td>
                        <td align="center" valign="top">SETDB1</td>
                        <td align="center" valign="top">Histone methyltransferase</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Shields univalent <span class="italic">X</span> in males from synapsis checkpoint. (<a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003401;class=Gene" target="_blank">mpk-1</a></span></td>
                        <td align="center" valign="top">ERK</td>
                        <td align="center" valign="top">MAP kinase</td>
                        <td align="center" valign="top">Phosphorylated form (active) present in mid pachytene</td>
                        <td align="center" valign="top">Required for progression to late pachytene; mutants never enter diplotene-like state. Regulates transition in mode of DSB
                           repair. (<a href="#sec7.3" title="7.3.&nbsp;Proteins that promote normal prophase I progression">Section 7.3</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003405;class=Gene" target="_blank">mre-11</a></span></td>
                        <td align="center" valign="top">MRE11A</td>
                        <td align="center" valign="top">DNA repair. Nuclease; forms complex with <a href="http://www.wormbase.org/db/get?name=WP:CE41010;class=Protein" target="_blank">RAD-50</a></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Required for efficient formation of meiotic DSBs and for <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> loading to <a href="http://www.wormbase.org/db/get?name=WP:CE49824;class=Protein" target="_blank">SPO-11</a> DSBs (<a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>, <a href="#sec5.2.1" title="5.2.1.&nbsp;Resection">Section 5.2.1</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003406;class=Gene" target="_blank">mrg-1</a></span></td>
                        <td align="center" valign="top">Chromo-domain protein</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Localizes to autosomes</td>
                        <td align="center" valign="top">Plays a role in non-pairing center mediated homolog alignment. (<a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003420;class=Gene" target="_blank">msh-4</a> (<a href="http://www.wormbase.org/db/get?name=WBGene00001872;class=Gene" target="_blank">him-14</a>)</span></td>
                        <td align="center" valign="top">MSH4</td>
                        <td align="center" valign="top">DNA repair; CO designation</td>
                        <td align="center" valign="top">Early pachytene - multiple foci per chromosome. Late pachytene, one focus per chromosome (presumed to mark CO position)</td>
                        <td align="center" valign="top">Forms a heterodimer with <a href="http://www.wormbase.org/db/get?name=WP:CE26678;class=Protein" target="_blank">MSH-5</a>. Stabilizes CO intermediates to promote CO outcomes (<a href="#sec5.2.3" title="5.2.3.&nbsp;CO designation">Section 5.2.3</a>). Required for induction of apoptosis (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003421;class=Gene" target="_blank">msh-5</a></span></td>
                        <td align="center" valign="top">MSH5</td>
                        <td align="center" valign="top">DNA repair; CO designation</td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003420;class=Gene" target="_blank">msh-4</a></span> (above).
                        </td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003420;class=Gene" target="_blank">msh-4</a></span> (above).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00016602;class=Gene" target="_blank">mus-81</a></span></td>
                        <td align="center" valign="top">MUS81</td>
                        <td align="center" valign="top">DNA repair; Holliday junction resolvase</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Resolvase activity (in conjunction with <a href="http://www.wormbase.org/db/get?name=WP:CE11216;class=Protein" target="_blank">SLX-1</a>). (<a href="#sec5.2.5" title="5.2.5.&nbsp;CO resolution">Section 5.2.5</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00008641;class=Gene" target="_blank">pch-2</a></span></td>
                        <td align="center" valign="top">TRIP13</td>
                        <td align="center" valign="top">AAA-ATPase</td>
                        <td align="center" valign="top">Forms foci in germline nuclei before transition zone, and SC-like tracks during pachytene</td>
                        <td align="center" valign="top">Restricts SC assembly: SC assembly accelerated in mutants (<a href="#sec4.3.1" title="4.3.1.&nbsp;Regulation of SC assembly">Section 4.3.1</a>). Required for induction of apoptosis when synapsis fails (<a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004042;class=Gene" target="_blank">plk-1</a></span></td>
                        <td align="center" valign="top">PLK1</td>
                        <td align="center" valign="top">Polo-like kinase</td>
                        <td align="center" valign="top">Localizes to pairing centers, specially in absence of <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a></td>
                        <td align="center" valign="top">Can partially replace meiotic functions of <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> in promoting pairing and SC assembly. (<a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004043;class=Gene" target="_blank">plk-2</a></span></td>
                        <td align="center" valign="top">PLK1</td>
                        <td align="center" valign="top">Polo-like kinase</td>
                        <td align="center" valign="top">Localizes to pairing centers in early pachytene and to the SC in mid pachytene</td>
                        <td align="center" valign="top">Induces nuclear envelope reorganization; <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> relocate into aggregates at clustered PC ends of chromosomes. Required for N-terminal phosphorylation of <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> shortly after meiotic entry (<a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>). Involved in feedback loop to sustain chromosome end-led mobility until completion of synapsis (<a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00009173;class=Gene" target="_blank">prom-1</a></span></td>
                        <td align="center" valign="top">FBXO47</td>
                        <td align="center" valign="top">F-box protein</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Required for homolog pairing, normal assembly of SC, DSB repair; involved in degradation of <a href="http://www.wormbase.org/db/get?name=WP:CE33761;class=Protein" target="_blank">CYE-1</a>. (<a href="#sec7.3" title="7.3.&nbsp;Proteins that promote normal prophase I progression">Section 7.3</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003417;class=Gene" target="_blank">mrt-2</a></span></td>
                        <td align="center" valign="top">RAD1</td>
                        <td align="center" valign="top">DNA damage sensing; 9-1-1 complex</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">DNA damage sensing, involved in DNA damage-induced apoptosis in the germ line. (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004296;class=Gene" target="_blank">rad-50</a></span></td>
                        <td align="center" valign="top"><a href="http://www.wormbase.org/db/get?name=WBGene00004296;class=Gene" target="_blank">RAD50</a></td>
                        <td align="center" valign="top">DNA repair; forms complex with <a href="http://www.wormbase.org/db/get?name=WP:CE37459;class=Protein" target="_blank">MRE-11</a></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Required for formation and repair of meiotic DSBs (<a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>, <a href="#sec5.2.1" title="5.2.1.&nbsp;Resection">Section 5.2.1</a>). Required for <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> loading to DSBs (and interhomolog CO formation) (<a href="#sec5.2.1" title="5.2.1.&nbsp;Resection">Section 5.2.1</a>), but only in early to mid-pachytene; in late pachytene, <a href="http://www.wormbase.org/db/get?name=WP:CE41010;class=Protein" target="_blank">RAD-50</a> is not required for repair (which is preferentially targeted to sister) (<a href="#sec5.6" title="5.6.&nbsp;Window of opportunity for CO formation">Section 5.6</a>, <a href="#sec7.3" title="7.3.&nbsp;Proteins that promote normal prophase I progression">Section 7.3</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004297;class=Gene" target="_blank">rad-51</a></span></td>
                        <td align="center" valign="top">Rad51; DMC1</td>
                        <td align="center" valign="top">DNA repair; catalyzes strand invasion</td>
                        <td align="center" valign="top">Localizes to sites of DSBs</td>
                        <td align="center" valign="top">Promotes DNA strand invasion (<a href="#sec5.2.1" title="5.2.1.&nbsp;Resection">Section 5.2.1</a>, <a href="#sec5.2.2" title="5.2.2.&nbsp;Strand exchange and formation of downstream repair intermediates">Section 5.2.2</a>, <a href="#sec5.6" title="5.6.&nbsp;Window of opportunity for CO formation">Section 5.6</a>). Mutation results in high levels of resected ssDNA and massive germline apoptosis (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004298;class=Gene" target="_blank">rad-54</a></span></td>
                        <td align="center" valign="top">Rad54</td>
                        <td align="center" valign="top">DNA repair; DEAD-like helicase family</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Promotes removal of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> from recombination intermediates (<a href="#sec5.2.2" title="5.2.2.&nbsp;Strand exchange and formation of downstream repair intermediates">Section 5.2.2</a>). Mutation traps DSBR intermediates as <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> filaments (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004332;class=Gene" target="_blank">rec-1</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Required for normal distribution of COs (but not frequency) (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004333;class=Gene" target="_blank">rec-8</a></span></td>
                        <td align="center" valign="top">Rec8</td>
                        <td align="center" valign="top">Meiosis specific kleisin subunit of cohesin complex</td>
                        <td align="center" valign="top">Chromosome axes</td>
                        <td align="center" valign="top">Required for CO formation and to prevent SC formation between sister chromatids (<a href="#sec4.1" title="4.1.&nbsp;Meiosis-specific cohesin complexes">Section 4.1</a>). Loss of <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE35852;class=Protein" target="_blank">COH-3</a>, and <a href="http://www.wormbase.org/db/get?name=WP:CE27409;class=Protein" target="_blank">COH-4</a> impairs axial element assembly and causes gross defects in sister chromatid cohesion (<a href="#sec4.1" title="4.1.&nbsp;Meiosis-specific cohesin complexes">Section 4.1</a>). Regulated <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> removal during late prophase and metaphase I required for two step-release of cohesion (<a href="#sec6.1" title="6.1.&nbsp;CO-triggered chromosome remodeling">Section 6.1</a>). Required for mono-orientation of sister chromatids on the first meiotic spindle (<a href="#sec8" title="8.&nbsp;Chromosome segregation during the meiotic divisions">Section 8</a>). Required to prevent premature centriole disengagement during meiosis II in spermatogenesis (<a href="#sec4.2" title="4.2.&nbsp;HORMA-domain proteins">Section 4.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004342;class=Gene" target="_blank">rfs-1</a></span></td>
                        <td align="center" valign="top">RAD51C</td>
                        <td align="center" valign="top">DNA repair; interacts with <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top"><a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> paralog; Promotes <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> turnover (removal of <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a>); redundant with <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00021905;class=Gene" target="_blank">helq-1</a></span> (<a href="#sec5.2.2" title="5.2.2.&nbsp;Strand exchange and formation of downstream repair intermediates">Section 5.2.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00017546;class=Gene" target="_blank">rpa-1</a></span></td>
                        <td align="center" valign="top">RPA1</td>
                        <td align="center" valign="top">DNA repair; single-strand binding protein</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Binds to ssDNA at DSB sites in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00020316;class=Gene" target="_blank">brc-2</a></span> mutants; may also bind in WT. (<a href="#sec5.2.2" title="5.2.2.&nbsp;Strand exchange and formation of downstream repair intermediates">Section 5.2.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00009124;class=Gene" target="_blank">rtel-1</a></span></td>
                        <td align="center" valign="top">RTEL1</td>
                        <td align="center" valign="top">DNA repair; DNA helicase</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Anti-recombinase: disrupts D loops in vitro, promotes alternative forms of repair (most likely SDSA; shunts a subset of DSBR
                           intermediates into NCO repair) (<a href="#sec5.2.2" title="5.2.2.&nbsp;Strand exchange and formation of downstream repair intermediates">Section 5.2.2</a>, <a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic">scc-2</span></td>
                        <td align="center" valign="top">SCC2</td>
                        <td align="center" valign="top">Cohesin loader</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Mutants fail to load to cohesin and to assemble axial elements and the SC. (<a href="#sec4.1" title="4.1.&nbsp;Meiosis-specific cohesin complexes">Section 4.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004738;class=Gene" target="_blank">scc-3</a></span></td>
                        <td align="center" valign="top">SCC3</td>
                        <td align="center" valign="top">Cohesin component</td>
                        <td align="center" valign="top">Chromosome axes</td>
                        <td align="center" valign="top">Common to all cohesin complexes. Required for loading of axial element and SC components. (<a href="#sec4.1" title="4.1.&nbsp;Meiosis-specific cohesin complexes">Section 4.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00018909;class=Gene" target="_blank">slx-1</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Resolvase activity (in conjunction with <a href="http://www.wormbase.org/db/get?name=WP:CE32609;class=Protein" target="_blank">MUS-81</a>) (<a href="#sec5.2.5" title="5.2.5.&nbsp;CO resolution">Section 5.2.5</a>). Inhibits CO formation in the central gene clusters of autosomes (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001860;class=Gene" target="_blank">smc-1</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Chromosome axes</td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004738;class=Gene" target="_blank">scc-3</a></span> (above).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004873;class=Gene" target="_blank">smc-3</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Chromosome axes</td>
                        <td align="center" valign="top">Common to all cohesin complexes. Hypomorph allele causes reduction in axis-associated cohesin (<a href="#sec4.1" title="4.1.&nbsp;Meiosis-specific cohesin complexes">Section 4.1</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004874;class=Gene" target="_blank">smc-4</a></span></td>
                        <td align="center" valign="top">SMC4</td>
                        <td align="center" valign="top">Condensin I and II complexes</td>
                        <td align="center" valign="top">Associates with chromatin from diplotene (condensin II), and with the short arm of metaphase I bivalents and between sister
                           chomatids during metaphase II (condensin I)
                        </td>
                        <td align="center" valign="top">Heterozygous null mutants show altered distribution of DSBs and COs, increased CO frequencies, and increased pachytene axis
                           length (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>). Required for chromosome condensation and segregation (<a href="#sec6.2" title="6.2.&nbsp;Chromosome condensation">Section 6.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00016151;class=Gene" target="_blank">smc-5</a></span></td>
                        <td align="center" valign="top">SMC5</td>
                        <td align="center" valign="top">Forms complex with <a href="http://www.wormbase.org/db/get?name=WP:CE46466;class=Protein" target="_blank">SMC-6</a></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Involved in intersister DSB repair and/or interhomolog NCO repair. (<a href="#sec5.2.4" title="5.2.4.&nbsp;Partner choice">Section 5.2.4</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00010056;class=Gene" target="_blank">smc-6</a></span></td>
                        <td align="center" valign="top">SMC6</td>
                        <td align="center" valign="top">Forms complex with <a href="http://www.wormbase.org/db/get?name=WP:CE27096;class=Protein" target="_blank">SMC-5</a></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00010056;class=Gene" target="_blank">smc-6</a></span> (above).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004954;class=Gene" target="_blank">spd-3</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Mitochondrial</td>
                        <td align="center" valign="top">Required for normal chromosome movements during meiosis; mutation results in non-homologous synapsis. (<a href="#sec3.3" title="3.3.&nbsp;PC movements promote efficient homolog pairing">Section 3.3</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004985;class=Gene" target="_blank">spo-11</a></span></td>
                        <td align="center" valign="top">SPO11</td>
                        <td align="center" valign="top">Formation of meiotic DSBs</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Catalyzes formation of meiotic double-strand breaks (<a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>, <a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>, <a href="#sec5.4" title="5.4.&nbsp;CO interference, assurance, and homeostasis">Section 5.4</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006311;class=Gene" target="_blank">sun-1</a></span></td>
                        <td align="center" valign="top">SUN-domain family</td>
                        <td align="center" valign="top">Transmembrane protein, interaction with KASH-domain proteins</td>
                        <td align="center" valign="top">Nuclear envelope, forms aggregates in transition zone nuclei</td>
                        <td align="center" valign="top">Binds lamins and KASH domain proteins forming aggregates in transition zone nuclei at sites of chromosome attachment. Required
                           for homolog pairing and recombination (<a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>). Shows <a href="http://www.wormbase.org/db/get?name=WP:CE27297;class=Protein" target="_blank">CHK-2</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> - dependent phosphorylation at N terminus during leptotene-zygotene. Phospho-SUN-1 required for normal rate of SC assembly
                           (slower in absence) and to sustain <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a> recruitment to chromosome ends (<a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>, <a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>). Phospho <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> persists into pachytene in absence of DSBs or CO intermediates (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006375;class=Gene" target="_blank">syp-1</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Synaptonemal complex central region component</td>
                        <td align="center" valign="top">Localizes along chromosomes during prophase I</td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006376;class=Gene" target="_blank">syp-2</a></span> (below). Additionally, partial depletion alters CO distribution (<a href="#sec5.5" title="5.5.&nbsp;Interplay between recombination and chromosome structure">Section 5.5</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006376;class=Gene" target="_blank">syp-2</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Synaptonemal complex central region component</td>
                        <td align="center" valign="top">Localizes along chromosomes during prophase I</td>
                        <td align="center" valign="top">Structural component of synaptonemal complex required for loading of other SYPs and for CO formation. Stabilizes interhomolog
                           interactions during pachytene; promotes interhomolog DSB repair and normal meiotic progression (<a href="#sec4.3" title="4.3.&nbsp;The SC">Section 4.3</a>, <a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006377;class=Gene" target="_blank">syp-3</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Synaptonemal complex central region component</td>
                        <td align="center" valign="top">Localizes along chromosomes during prophase I</td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006376;class=Gene" target="_blank">syp-2</a></span> (above). Also may interact directly with axis components (<a href="#sec4.3" title="4.3.&nbsp;The SC">Section 4.3</a>). Allele <span class="italic">syp-3(<a href="http://www.wormbase.org/db/get?name=WBVar00088873;class=Variation" target="_blank">me42</a>)</span> leads to SC assembly between sisters (<a href="#sec7.1" title="7.1.&nbsp;Monitoring of SC assembly">Section 7.1</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00019247;class=Gene" target="_blank">syp-4</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Synaptonemal complex central region component</td>
                        <td align="center" valign="top">Localizes along chromosomes during prophase I</td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006376;class=Gene" target="_blank">syp-2</a></span> (above). Interacts with <a href="http://www.wormbase.org/db/get?name=WP:CE41238;class=Protein" target="_blank">SYP-3</a>, spans into center of SC (<a href="#sec4.3" title="4.3.&nbsp;The SC">Section 4.3</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006571;class=Gene" target="_blank">tim-1</a></span></td>
                        <td align="center" valign="top">TIMELESS</td>
                        <td align="center" valign="top">Cohesin loading</td>
                        <td align="center" valign="top">Nuclear localization in mitotic compartment of the germ line and in diplotene and diakinesis oocytes</td>
                        <td align="center" valign="top">Required for loading of <a href="http://www.wormbase.org/db/get?name=WP:CE45697;class=Protein" target="_blank">REC-8</a> cohesin complexes. (<a href="#sec4.1" title="4.1.&nbsp;Meiosis-specific cohesin complexes">Section 4.1</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006840;class=Gene" target="_blank">unc-116</a></span></td>
                        <td align="center" valign="top">Kinesin 1</td>
                        <td align="center" valign="top">Kinesin</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Required for translocation of the oocyte spindle to the cortex of the embryo. (<a href="#sec8" title="8.&nbsp;Chromosome segregation during the meiotic divisions">Section 8</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001514;class=Gene" target="_blank">xnd-1</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">AT-hook containing protein</td>
                        <td align="center" valign="top">localizes to autosomes from mitotic region through late pachytene</td>
                        <td align="center" valign="top">Required for subset of DSBs (mostly on <span class="italic">X</span>) (<a href="#sec5.1" title="5.1.&nbsp;DSB formation">Section 5.1</a>). Mutants accumulate aberrant histone modifications, and result in impaired DSB formation. Required for <a href="http://www.wormbase.org/db/get?name=WP:CE09062;class=Protein" target="_blank">HIM-5</a> localization to chromosomes (<a href="#sec5.3" title="5.3.&nbsp;The recombination landscape">Section 5.3</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00008140;class=Gene" target="_blank">xpf-1</a></span></td>
                        <td align="center" valign="top">ERCC4</td>
                        <td align="center" valign="top">DNA repair; Nucleotide excision repair</td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Resolvase activity (in conjunction with <a href="http://www.wormbase.org/db/get?name=WP:CE31724;class=Protein" target="_blank">HIM-6</a>). (<a href="#sec5.2.5" title="5.2.5.&nbsp;CO resolution">Section 5.2.5</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006976;class=Gene" target="_blank">zhp-3</a></span></td>
                        <td align="center" valign="top">RNF212; Zip3 (yeast)</td>
                        <td align="center" valign="top">RING finger protein; possible SUMO ligase</td>
                        <td align="center" valign="top">Early pachytene - broadly distributed along SC. End of pachytene, diplotene - one focus per chromosome pair (presumed to mark
                           CO position)
                        </td>
                        <td align="center" valign="top">Required for CO formation, role in stabilizing CO-fated intermediates (<a href="#sec5.2.3" title="5.2.3.&nbsp;CO designation">Section 5.2.3</a>). Also required for induction of apoptosis irrespective of DNA damage checkpoint activation (<a href="#sec7.2" title="7.2.&nbsp;Monitoring of recombination progression">Section 7.2</a>).
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00011597;class=Gene" target="_blank">zim-1</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Zinc-finger protein</td>
                        <td align="center" valign="top">Pairing center of chromosomes <span class="italic">II</span> and <span class="italic">III</span> in early prophase nuclei
                        </td>
                        <td align="center" valign="top">Required for homolog alignment and synapsis of <span class="italic">II</span> and <span class="italic">III.</span> Binds to PC ends only from leptotene to early pachytene. Localization depends on <a href="http://www.wormbase.org/db/get?name=WP:CE27297;class=Protein" target="_blank">CHK-2</a>. Recruitment site for polo-like kinases <a href="http://www.wormbase.org/db/get?name=WP:CE22877;class=Protein" target="_blank">PLK-2</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE26649;class=Protein" target="_blank">PLK-1</a>. (<a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00011600;class=Gene" target="_blank">zim-2</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Zinc-finger protein</td>
                        <td align="center" valign="top">Pairing center of chromosome <span class="italic">V</span> in early prophase nuclei
                        </td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00011597;class=Gene" target="_blank">zim-1</a></span> (above) except is required for homolog alignment and synapsis of <span class="italic">V.</span></td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00011601;class=Gene" target="_blank">zim-3</a></span></td>
                        <td align="center" valign="top">&nbsp;</td>
                        <td align="center" valign="top">Zinc-finger protein</td>
                        <td align="center" valign="top">Pairing center of chromosomes <span class="italic">I</span> and <span class="italic">IV</span> in early prophase nuclei
                        </td>
                        <td align="center" valign="top">Same as <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00011597;class=Gene" target="_blank">zim-1</a></span> (above) except is required for homolog alignment and synapsis of <span class="italic">I</span> and <span class="italic">IV.</span></td>
                     </tr>
                     <tr valign="top">
                        <td align="center" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006997;class=Gene" target="_blank">zyg-12</a></span></td>
                        <td align="center" valign="top">KASH-domain family</td>
                        <td align="center" valign="top">Interacts with <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> in nuclear envelope-spanning protein complex
                        </td>
                        <td align="center" valign="top">Nuclear envelope, forms aggregates in transition zone nuclei</td>
                        <td align="center" valign="top">Connects nucleus to cytoskeleton (via dynein). Binds <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a>. Required for homolog pairing and recombination (<a href="#sec3.2" title="3.2.&nbsp;PCs assemble as nucleoprotein complexes at the nuclear periphery">Section 3.2</a>). Recruits Dynein motor complex to cytoplasmic side of <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a> aggregates in leptotene/zygotene; facilitates proper synapsis (<a href="#sec3.3" title="3.3.&nbsp;PC movements promote efficient homolog pairing">Section 3.3</a>).
                        </td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec12"></a>12.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed"><span class="bibliomisc">Adamo, A., Collis, S.J., Adelman, C.A., Silva, N., Horejsi, Z., Ward, J.D., Martinez-Perez, E., Boulton, S.J., and La Volpe,
                        A. (2010). Preventing nonhomologous end joining suppresses DNA repair defects of Fanconi anemia. Mol. Cell <span class="italic">39</span>, 25-35.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.molcel.2010.06.026" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed"><span class="bibliomisc">Adamo, A., Montemauri, P., Silva, N., Ward, J.D., Boulton, S.J., and La Volpe, A. (2008). <a href="http://www.wormbase.org/db/get?name=WP:CE43190;class=Protein" target="_blank">BRC-1</a> acts in the inter-sister pathway of meiotic double-strand break repair. EMBO Rep. <span class="italic">9</span>, 287-292.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/sj.embor.7401167" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed"><span class="bibliomisc">Agostinho, A., Meier, B., Sonneville, R., Jagut, M., Woglar, A., Blow, J., Jantsch, V., and Gartner, A. (2013). Combinatorial
                        regulation of meiotic Holliday junction resolution in <span class="italic">C. elegans</span> by <a href="http://www.wormbase.org/db/get?name=WP:CE31724;class=Protein" target="_blank">HIM-6</a> (BLM) helicase, SLX-4, and the <a href="http://www.wormbase.org/db/get?name=WP:CE11216;class=Protein" target="_blank">SLX-1</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE32609;class=Protein" target="_blank">MUS-81</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE24855;class=Protein" target="_blank">XPF-1</a> nucleases. PLoS Genet. <span class="italic">9</span>, e1003591.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1003591" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed"><span class="bibliomisc">Albertson, D.G., Rose, A.M., and Villeneuve, A.M. (1997). Chromosome organization, mitosis, and meiosis. In <span class="italic">C. Elegans</span> II, D.L. Riddle, T. Blumenthal, B.J. Meyer, and J. Preiss, eds. (Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press),
                        pp. 47-78.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21413226&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed"><span class="bibliomisc">Albertson, D.G., and Thomson, J.N. (1993). Segregation of holocentric chromosomes at meiosis in the nematode, <span class="italic">Caenorhabditis elegans</span>. Chromosome Res. <span class="italic">1</span>, 15&#8211;26.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1007/BF00710603" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed"><span class="bibliomisc">Alpi, A., Pasierbek, P., Gartner, A., and Loidl, J. (2003). Genetic and cytological characterization of the recombination
                        protein <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> in <span class="italic">Caenorhabditis elegans</span>. Chromosoma <span class="italic">112</span>, 6&#8211;16.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1007/s00412-003-0237-5" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed"><span class="bibliomisc">Aravind, L., and Koonin, E.V. (1998). The HORMA domain: a common structural denominator in mitotic checkpoints, chromosome
                        synapsis and DNA repair. Trends Biochem. Sci. <span class="italic">23</span>, 284&#8211;286.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0968-0004(98)01257-2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed"><span class="bibliomisc">Barber, L.J., Youds, J.L., Ward, J.D., McIlwraith, M.J., O'Neil, N.J., Petalcorin, M.I., Martin, J.S., Collis, S.J., Cantor,
                        S.B., Auclair, M., et al. (2008). RTEL1 maintains genomic stability by suppressing homologous recombination. Cell <span class="italic">135</span>, 261&#8211;271.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2008.08.016" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed"><span class="bibliomisc">Barnes, T.M., Kohara, Y., Coulson, A., and Hekimi, S. (1995). Meiotic recombination, noncoding DNA and genomic organization
                        in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">141</span>, 159&#8211;179.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8536965&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed"><span class="bibliomisc">Baudat, F., Buard, J., Grey, C., Fledel-Alon, A., Ober, C., Przeworski, M., Coop, G., and de Massey, B. (2010). PRDM9 is a
                        major determinant of meiotic recombination hotspots in humans and mice. Science <span class="italic">327</span>, 836&#8211;40.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1126/science.1183439" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed"><span class="bibliomisc">Baudrimont, A., Penkner, A., Woglar, A., Machacek, T., Wegrostek, C., Gloggnitzer, J., Fridkin, A., Klein, F., Gruenbaum,
                        Y., Pasierbek, P., et al. (2010). Leptotene/zygotene chromosome movement via the SUN/KASH protein bridge in <span class="italic">Caenorhabditis elegans</span>. PLoS Genet. <span class="italic">6</span>, e1001219.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21124819&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1001219" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed"><span class="bibliomisc">Baudrimont, A., Penkner, A., Woglar, A., Mamnun, Y.M., Hulek, M., Struck, C., Schnabel, R., Loidl, J., and Jantsch, V. (2011).
                        A new thermosensitive <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004873;class=Gene" target="_blank">smc-3</a></span> allele reveals involvement of cohesin in homologous recombination in <span class="italic">C. elegans</span>. PLoS One <span class="italic">6</span>, e24799.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21957461&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pone.0024799" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed"><span class="bibliomisc">Bhalla, N., and Dernburg, A.F. (2005). A conserved checkpoint monitors meiotic chromosome synapsis in <span class="italic">Caenorhabditis elegans</span>. Science <span class="italic">310</span>, 1683&#8211;1686.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16339446&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1126/science.1117468" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed"><span class="bibliomisc">Bhalla, N., Wynne, D.J., Jantsch, V., and Dernburg, A.F. (2008). <a href="http://www.wormbase.org/db/get?name=WP:CE43850;class=Protein" target="_blank">ZHP-3</a> acts at crossovers to couple meiotic recombination with synaptonemal complex disassembly and bivalent formation in <span class="italic">C. elegans</span>. PLoS Genet. <span class="italic">4</span>, e1000235.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18949042&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1000235" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed"><span class="bibliomisc">Bickel, J.S., Chen, L., Hayward, J., Yeap, S.L., Alkers, A.E., and Chan, R.C. (2010). Structural maintenance of chromosomes
                        (SMC) proteins promote homolog-independent recombination repair in meiosis crucial for germ cell genomic stability. PLoS Genet.
                        <span class="italic">6</span>, e1001028.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20661436&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1001028" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed"><span class="bibliomisc">Boulton, S.J., Martin, J.S., Polanowska, J., Hill, D.E., Gartner, A., and Vidal, M. (2004). BRCA1/BARD1 orthologs required
                        for DNA repair in <span class="italic">Caenorhabditis elegans</span>. Curr. Biol. <span class="italic">14</span>, 33&#8211;39.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14711411&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2003.11.029" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed"><span class="bibliomisc">Brockway, H., Balukoff, N., Dean, M., Alleva, B., and Smolikove, S. (2014). The CSN/COP9 signalosome regulates synaptonemal
                        complex assembly during meiotic prophase I of <span class="italic">Caenorhabditis elegans</span>. PLoS Genet. <span class="italic">10</span>, e1004757.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=25375142&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1004757" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed"><span class="bibliomisc">Carlton, P.M., Farruggio, A.P., and Dernburg, A.F. (2006). A link between meiotic prophase progression and crossover control.
                        PLoS Genet. <span class="italic">2</span>, e12.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16462941&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.0020012" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed"><span class="bibliomisc">de Carvalho, C.E., Zaaijer, S., Smolikov, S., Gu, Y., Schumacher, J.M., and Colai&aacute;covo, M.P. (2008). <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a> antagonizes the Aurora B kinase in <span class="italic">C. elegans</span>. Genes Dev. <span class="italic">22</span>, 2869&#8211;2885.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18923084&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.1691208" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed"><span class="bibliomisc">Chan, R.C., Chan, A., Jeon, M., Wu, T.F., Pasqualone, D., Rougvie, A.E., and Meyer, B.J. (2003). Chromosome cohesion is regulated
                        by a clock gene paralogue <a href="http://www.wormbase.org/db/get?name=WP:CE23033;class=Protein" target="_blank">TIM-1</a>. Nature <span class="italic">423</span>, 1002&#8211;1009.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12827206&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/nature01697" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed"><span class="bibliomisc">Chan, R.C., Severson, A.F., and Meyer, B.J. (2004). Condensin restructures chromosomes in preparation for meiotic divisions.
                        J. Cell Biol. <span class="italic">167</span>, 613&#8211;625.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15557118&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1083/jcb.200408061" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed"><span class="bibliomisc">Checchi, P.M., and Engebrecht, J. (2011). <span class="italic">Caenorhabditis elegans</span> histone methyltransferase <a href="http://www.wormbase.org/db/get?name=WP:CE47959;class=Protein" target="_blank">MET-2</a> shields the male X chromosome from checkpoint machinery and mediates meiotic sex chromosome inactivation. PLoS Genet. <span class="italic">7</span>, e1002267.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21909284&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1002267" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed"><span class="bibliomisc">Checchi, P.M., Lawrence, K.S., Van, M. V, Larson, B.J., and Engebrecht, J. (2014). Pseudosynapsis and decreased stringency
                        of meiotic repair pathway choice on the hemizygous sex chromosome of <span class="italic">Caenorhabditis elegans</span> males. Genetics <span class="italic">197</span>, 543&#8211;560.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24939994&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.114.164152" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed"><span class="bibliomisc">Chin, G.M., and Villeneuve, A.M. (2001). <span class="italic">C. elegans <a href="http://www.wormbase.org/db/get?name=WBGene00003405;class=Gene" target="_blank">mre-11</a></span> is required for meiotic recombination and DNA repair but is dispensable for the meiotic G(2) DNA damage checkpoint. Genes
                        Dev. <span class="italic">15</span>, 522&#8211;534.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11238374&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.864101" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed"><span class="bibliomisc">Church, D.L., Guan, K.L., and Lambie, E.J. (1995). Three genes of the MAP kinase cascade, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003186;class=Gene" target="_blank">mek-2</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00003401;class=Gene" target="_blank">mpk-1</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00003401;class=Gene" target="_blank">sur-1</a></span> and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene" target="_blank">let-60</a></span> ras, are required for meiotic cell cycle progression in <span class="italic">Caenorhabditis elegans</span>. Development <span class="italic">121</span>, 2525&#8211;2535.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7671816&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed"><span class="bibliomisc">Clark-Maguire, S., and Mains, P.E. (1994). <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003183;class=Gene" target="_blank">mei-1</a></span>, a gene required for meiotic spindle formation in <span class="italic">Caenorhabditis elegans</span>, is a member of a family of ATPases. Genetics <span class="italic">136</span>, 533&#8211;546.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8150281&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed"><span class="bibliomisc">Colai&aacute;covo, M.P. (2006). The many facets of SC function during <span class="italic">C. elegans</span> meiosis. Chromosoma <span class="italic">115</span>, 195&#8211;211.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16555015&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1007/s00412-006-0061-9" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed"><span class="bibliomisc">Colai&aacute;covo, M.P., MacQueen, A.J., Martinez-Perez, E., McDonald, K., Adamo, A., La Volpe, A., and Villeneuve, A.M. (2003).
                        Synaptonemal complex assembly in <span class="italic">C. elegans</span> is dispensable for loading strand-exchange proteins but critical for proper completion of recombination. Dev. Cell <span class="italic">5</span>, 463&#8211;474.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12967565&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S1534-5807(03)00232-6" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed"><span class="bibliomisc">Collette, K.S., Petty, E.L., Golenberg, N., Bembenek, J.N., and Csankovszki, G. (2011). Different roles for Aurora B in condensin
                        targeting during mitosis and meiosis. J. Cell Sci. <span class="italic">124</span>, 3684&#8211;3694.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22025633&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1242/jcs.088336" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed"><span class="bibliomisc">Connolly, A.A., Osterberg, V., Christensen, S., Price, M., Lu, C., Chicas-Cruz, K., Lockery, S., Mains, P.E., and Bowerman,
                        B. (2014). <span class="italic">Caenorhabditis elegans</span> oocyte meiotic spindle pole assembly requires microtubule severing and the calponin homology domain protein <a href="http://www.wormbase.org/db/get?name=WP:CE15710;class=Protein" target="_blank">ASPM-1</a>. Mol. Biol. Cell <span class="italic">25</span>, 1298&#8211;1311.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24554763&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1091/mbc.E13-11-0687" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed"><span class="bibliomisc">Cortes, D.B., McNally, K.L., Mains, P.E., and McNally, F.J. (2015). The asymmetry of female meiosis reduces the frequency
                        of inheritance of unpaired chromosomes. eLife <span class="italic">4</span>, e06056.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=25848744&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.7554/eLife.06056" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed"><span class="bibliomisc">Couteau, F., Nabeshima, K., Villeneuve, A., and Zetka, M. (2004). A component of <span class="italic">C. elegans</span> meiotic chromosome axes at the interface of homolog alignment, synapsis, nuclear reorganization, and recombination. Curr.
                        Biol. <span class="italic">14</span>, 585&#8211;592.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15062099&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2004.03.033" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed"><span class="bibliomisc">Couteau, F., and Zetka, M. (2005). <a href="http://www.wormbase.org/db/get?name=WP:CE31692;class=Protein" target="_blank">HTP-1</a> coordinates synaptonemal complex assembly with homolog alignment during meiosis in <span class="italic">C. elegans</span>. Genes Dev. <span class="italic">19</span>, 2744&#8211;2756.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16291647&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.1348205" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed"><span class="bibliomisc">Couteau, F., and Zetka, M. (2011). DNA damage during meiosis induces chromatin remodeling and synaptonemal complex disassembly.
                        Dev. Cell <span class="italic">20</span>, 353&#8211;363.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21397846&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.devcel.2011.01.015" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed"><span class="bibliomisc">Csankovszki, G., Collette, K., Spahl, K., Carey, J., Snyder, M., Petty, E., Patel, U., Tabuchi, T., Liu, H., McLeod, I., et
                        al. (2009). Three distinct condensin complexes control <span class="italic">C. elegans</span> chromosome dynamics. Curr. Biol. <span class="italic">19</span>, 9&#8211;19.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19119011&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2008.12.006" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed"><span class="bibliomisc">Davis, M.W., Hammarlund, M., Harrach, T., Hullett, P., Olsen, S., and Jorgensen, E.M. (2005). Rapid single nucleotide polymorphism
                        mapping in <span class="italic">C. elegans</span>. BMC Genomics <span class="italic">6</span>, 118.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16156901&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1186/1471-2164-6-118" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed"><span class="bibliomisc">Dernburg, A.F., McDonald, K., Moulder, G., Barstead, R., Dresser, M., and Villeneuve, A.M. (1998). Meiotic recombination in
                        <span class="italic">C. elegans</span> initiates by a conserved mechanism and is dispensable for homologous chromosome synapsis. Cell <span class="italic">94</span>, 387&#8211;398.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9708740&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0092-8674(00)81481-6" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed"><span class="bibliomisc">Deshong, A.J., Ye, A.L., Lamelza, P., and Bhalla, N. (2014). A quality control mechanism coordinates meiotic prophase events
                        to promote crossover assurance. PLoS Genet. <span class="italic">10</span>, e1004291.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24762417&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1004291" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed"><span class="bibliomisc">Dombecki, C.R., Chiang, A.C.Y., Kang, H.-J., Bilgir, C., Stefanski, N.A., Neva, B.J., Klerkx, E.P., and Nabeshima K. (2011).
                        The chromodomain protein <a href="http://www.wormbase.org/db/get?name=WP:CE20213;class=Protein" target="_blank">MRG-1</a> facilitates SC-independent homologous pairing during meiosis in <span class="italic">Caenorhabditis elegans</span>. Dev. Cell <span class="italic">21</span>, 1092&#8211;1103.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22172672&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.devcel.2011.09.019" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed"><span class="bibliomisc">Dumont, J., Oegema, K., and Desai, A. (2010). A kinetochore-independent mechanism drives anaphase chromosome separation during
                        acentrosomal meiosis. Nat. Cell Biol. <span class="italic">12</span>, 894&#8211;901.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20729837&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/ncb2093" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed"><span class="bibliomisc">Ellefson, M.L., and McNally, F.J. (2009). Kinesin-1 and cytoplasmic dynein act sequentially to move the meiotic spindle to
                        the oocyte cortex in <span class="italic">Caenorhabditis elegans</span>. Mol. Biol. Cell <span class="italic">20</span>, 2722&#8211;2730.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19357192&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1091/mbc.E08-12-1253" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed"><span class="bibliomisc">Fabritius, A.S., Ellefson, M.L., and McNally, F.J. (2011). Nuclear and spindle positioning during oocyte meiosis. Curr. Opin.
                        Cell Biol. <span class="italic">23</span>, 78&#8211;84.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20708397&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.ceb.2010.07.008" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed"><span class="bibliomisc">Fox, P.M., Vought, V.E., Hanazawa, M., Lee, M.-H., Maine, E.M., and Schedl, T. (2011). Cyclin E and <a href="http://www.wormbase.org/db/get?name=WP:CE36652;class=Protein" target="_blank">CDK-2</a> regulate proliferative cell fate and cell cycle progression in the <span class="italic">C. elegans</span> germline. Development <span class="italic">138</span>, 2223&#8211;2234.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21558371&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1242/dev.059535" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed"><span class="bibliomisc">Fridkin, A., Mills, E., Margalit, A., Neufeld, E., Lee, K.K., Feinstein, N., Cohen, M., Wilson, K.L., and Gruenbaum, Y. (2004).
                        Matefin, a <span class="italic">Caenorhabditis elegans</span> germ line-specific SUN-domain nuclear membrane protein, is essential for early embryonic and germ cell development. Proc.
                        Natl. Acad. Sci. U. S. A. <span class="italic">101</span>, 6987&#8211;6992.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15100407&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1073/pnas.0307880101" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed"><span class="bibliomisc">Gabdank, I., and Fire, A.Z. (2014). Gamete-type dependent crossover interference levels in a defined region of <span class="italic">Caenorhabditis elegans</span> chromosome V. G3 <span class="italic">4</span>, 117&#8211;120.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24240780&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed"><span class="bibliomisc">Gao, J., Kim, H.-M., Elia, A.E., Elledge, S.J., and Colai&aacute;covo, M.P. (2015). NatB domain-containing <a href="http://www.wormbase.org/db/get?name=WP:CE31579;class=Protein" target="_blank">CRA-1</a> antagonizes hydrolase ACER-1 linking acetyl-CoA metabolism to the initiation of recombination during <span class="italic">C. elegans</span> meiosis. PLoS Genet. <span class="italic">11</span>, e1005029.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=25768301&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1005029" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed"><span class="bibliomisc">Garcia, V., Gray, S., Allison, R.M., Cooper, T.J., and Neale, M.J. (2015). Tel1(ATM)-mediated interference suppresses clustered
                        meiotic double-strand-break formation. Nature <span class="italic">520</span>, 114&#8211;118.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=25539084&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/nature13993" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed"><span class="bibliomisc">Gartner, A., Milstein, S., Ahmed, S., Hodgkin, J., and Hengartner, M.O. (2000). A conserved checkpoint pathway mediates DNA
                        damage-induced apoptosis and <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001392;class=Phenotype" target="_blank">cell cycle arrest</a> in <span class="italic">C. elegans</span>. Mol. Cell <span class="italic">5</span>, 435&#8211;443.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10882129&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S1097-2765(00)80438-4" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed"><span class="bibliomisc">Goldstein, P. (1982). The synaptonemal complexes of <span class="italic">Caenorhabditis elegans</span>: pachytene karyotype analysis of male and hermaphrodite wild-type and <span class="italic">him</span> mutants. Chromosoma <span class="italic">86</span>, 577&#8211;593.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7172867&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1007/BF00330128" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed"><span class="bibliomisc">Goodyer, W., Kaitna, S., Couteau, F., Ward, J.D., Boulton, S.J., and Zetka, M. (2008). <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> links DSB formation with homolog pairing and crossing over during <span class="italic">C. elegans</span> meiosis. Dev. Cell <span class="italic">14</span>, 263&#8211;274.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18267094&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.devcel.2007.11.016" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed"><span class="bibliomisc">Gray, S., Allison, R.M., Garcia, V., Goldman, A.S.H., and Neale, M.J. (2013). Positive regulation of meiotic DNA double-strand
                        break formation by activation of the DNA damage checkpoint kinase Mec1(ATR). Open Biol. <span class="italic">3</span>, 130019.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23902647&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1098/rsob.130019" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed"><span class="bibliomisc">Haarhuis, J.H.I., Elbatsh, A.M.O., and Rowland, B.D. (2014). Cohesin and its regulation: on the logic of X-shaped chromosomes.
                        Dev. Cell <span class="italic">31</span>, 7&#8211;18.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=25313959&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.devcel.2014.09.010" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed"><span class="bibliomisc">Hagstrom, K.A., Holmes, V.F., Cozzarelli, N.R., and Meyer, B.J. (2002). <span class="italic">C. elegans</span> condensin promotes mitotic chromosome architecture, centromere organization, and sister chromatid segregation during mitosis
                        and meiosis. Genes Dev. <span class="italic">16</span>, 729&#8211;742.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11914278&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.968302" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed"><span class="bibliomisc">Hammarlund, M., Davis, M.W., Nguyen, H., Dayton, D., and Jorgensen, E.M. (2005). Heterozygous insertions alter crossover distribution
                        but allow crossover interference in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">171</span>, 1047&#8211;1056.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16118192&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.105.044834" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed"><span class="bibliomisc">Harper, N.C., Rillo, R., Jover-Gil, S., Assaf, Z.J., Bhalla, N., and Dernburg, A.F. (2011). Pairing centers recruit a Polo-like
                        kinase to orchestrate meiotic chromosome dynamics in <span class="italic">C. elegans</span>. Dev. Cell <span class="italic">21</span>, 934&#8211;947.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22018922&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.devcel.2011.09.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed"><span class="bibliomisc">Hayashi, M., Chin, G.M., and Villeneuve, A.M. (2007). <span class="italic">C. elegans</span> germ cells switch between distinct modes of double-strand break repair during meiotic prophase progression. PLoS Genet. <span class="italic">3</span>, e191.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17983271&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.0030191" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed"><span class="bibliomisc">Hayashi, M., Mlynarczyk-Evans, S., and Villeneuve, A.M. (2010). The synaptonemal complex shapes the crossover landscape through
                        cooperative assembly, crossover promotion and crossover inhibition during <span class="italic">Caenorhabditis elegans</span> meiosis. Genetics <span class="italic">186</span>, 45&#8211;58.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20592266&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.110.115501" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed"><span class="bibliomisc">Henzel, J. V, Nabeshima, K., Schvarzstein, M., Turner, B.E., Villeneuve, A.M., and Hillers, K.J. (2011). An asymmetric chromosome
                        pair undergoes synaptic adjustment and crossover redistribution during <span class="italic">Caenorhabditis elegans</span> meiosis: implications for sex chromosome evolution. Genetics <span class="italic">187</span>, 685&#8211;699.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21212235&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.110.124958" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed"><span class="bibliomisc">Herman, R.K., and Kari, C.K. (1989). Recombination between small X chromosome duplications and the X chromosome in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">121</span>, 723&#8211;737.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2721932&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed"><span class="bibliomisc">Hillers, K.J., and Villeneuve, A.M. (2003). Chromosome-wide control of meiotic crossing over in <span class="italic">C . elegans</span>. Curr. Biol. <span class="italic">13</span>, 1641&#8211;1647.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=13678597&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2003.08.026" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed"><span class="bibliomisc">Hillers, K.J., and Villeneuve, A.M. (2009). Analysis of meiotic recombination in <span class="italic">Caenorhabditis elegans</span>. Methods Mol. Biol. <span class="italic">557</span>, 77&#8211;97.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19799178&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1007/978-1-59745-527-5_7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed"><span class="bibliomisc">Hodgkin, J., Horvitz, H.R., and Brenner, S. (1979). Nondisjunction mutants of the nematode <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">91</span>, 67&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17248881&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed"><span class="bibliomisc">Howe, M., McDonald, K.L., Albertson, D.G., and Meyer, B.J. (2001). <a href="http://www.wormbase.org/db/get?name=WP:CE01367;class=Protein" target="_blank">HIM-10</a> is required for kinetochore structure and function on <span class="italic">Caenorhabditis elegans</span> holocentric chromosomes. J. Cell Biol. <span class="italic">153</span>, 1227&#8211;1238.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11402066&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1083/jcb.153.6.1227" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed"><span class="bibliomisc">Jantsch, V., Pasierbek, P., Mueller, M.M., Schweizer, D., Jantsch, M., and Loidl, J. (2004). Targeted gene knockout reveals
                        a role in meiotic recombination for <a href="http://www.wormbase.org/db/get?name=WP:CE43850;class=Protein" target="_blank">ZHP-3</a>, a Zip3-related protein in <span class="italic">Caenorhabditis elegans</span>. Mol. Cell. Biol. <span class="italic">24</span>, 7998&#8211;8006.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15340062&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1128/MCB.24.18.7998-8006.2004" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed"><span class="bibliomisc">Jantsch, V., Tang, L., Pasierbek, P., Penkner, A., Nayak, S., Baudrimont, A., Schedl, T., Gartner, A., andLoidl, J. (2007).
                        <span class="italic">Caenorhabditis elegans</span> <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00009173;class=Gene" target="_blank">prom-1</a></span> is required for meiotic prophase progression and homologous chromosome pairing. Mol. Biol. Cell <span class="italic">18</span>, 4911&#8211;4920.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17914060&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1091/mbc.E07-03-0243" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed"><span class="bibliomisc">Jaramillo-Lambert, A., Ellefson, M., Villeneuve, A.M., and Engebrecht, J. (2007). Differential timing of S phases, X chromosome
                        replication, and meiotic prophase in the <span class="italic">C. elegans</span> germ line. Dev. Biol. <span class="italic">308</span>, 206&#8211;221.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17599823&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.ydbio.2007.05.019" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed"><span class="bibliomisc">Jaramillo-Lambert, A., Harigaya, Y., Vitt, J., Villeneuve, A., and Engebrecht, J. (2010). Meiotic errors activate checkpoints
                        that improve gamete quality without triggering apoptosis in male germ cells. Curr. Biol. <span class="italic">20</span>, 2078&#8211;2089.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20008570&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2010.10.008" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed"><span class="bibliomisc">Kaitna, S., Pasierbek, P., Jantsch, M., Loidl, J., and Glotzer, M. (2002). The aurora B kinase <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> regulates kinetochores during mitosis and is required for separation of homologous chromosomes during meiosis. Curr. Biol.
                        <span class="italic">12</span>, 798&#8211;812.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12015116&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0960-9822(02)00820-5" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed"><span class="bibliomisc">Kauppi, L., Barchi, M., Lange, J., Baudat, F., Jasin, M., and Keeney, S. (2013). Numerical constraints and feedback control
                        of double-strand breaks in mouse meiosis. Genes Dev. <span class="italic">27</span>, 873&#8211;886.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23599345&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.213652.113" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed"><span class="bibliomisc">Kaur, T., and Rockman, M. V (2014). Crossover heterogeneity in the absence of hotspots in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">196</span>, 137&#8211;148.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24172135&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.113.158857" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed"><span class="bibliomisc">Kelly, K.O., Dernburg, A.F., Stanfield, G.M., and Villeneuve, A.M. (2000). <span class="italic">Caenorhabditis elegans <a href="http://www.wormbase.org/db/get?name=WBGene00003421;class=Gene" target="_blank">msh-5</a></span> is required for both normal and radiation-induced meiotic crossing over but not for completion of meiosis. Genetics <span class="italic">156</span>, 617&#8211;630.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11014811&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib72"></a><p class="bibliomixed"><span class="bibliomisc">Kelly, W.G., Schaner, C.E., Dernburg, A.F., Lee, M.-H., Kim, S.K., Villeneuve, A.M., and Reinke, V. (2002). <span class="italic">X</span>-chromosome silencing in the germline of <span class="italic">C. elegans</span>. Development <span class="italic">129</span>, 479&#8211;492.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11807039&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib73"></a><p class="bibliomixed"><span class="bibliomisc">Kim, S., Spike, C., and Greenstein, D. (2013). Control of oocyte growth and meiotic maturation in <span class="italic">Caenorhabditis elegans</span>. Adv. Exp. Med. Biol. <span class="italic">757</span>, 277&#8211;320.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22872481&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1007/978-1-4614-4015-4_10" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib74"></a><p class="bibliomixed"><span class="bibliomisc">Kim, Y., Rosenberg, S.C., Kugel, C.L., Kostow, N., Rog, O., Davydov, V., Su, T.Y., Dernburg, A.F., and Corbett, K.D. (2014).
                        The chromosome axis controls meiotic events through a hierarchical assembly of HORMA domain proteins. Dev. Cell <span class="italic">31</span>, 487&#8211;502.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=25446517&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.devcel.2014.09.013" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib75"></a><p class="bibliomixed"><span class="bibliomisc">Kimble, J. and Crittenden, S.L. Germline proliferation and its control (August 15, 2005), <span class="italic">WormBook</span>, ed. The <span class="italic">C. elegans</span> Research Community, WormBook, doi/10.1895/wormbook.1.13.1,http://www.wormbook.org.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib76"></a><p class="bibliomixed"><span class="bibliomisc">Kohl, K.P., and Sekelsky, J. (2013). Meiotic and mitotic recombination in meiosis. Genetics <span class="italic">194</span>, 327&#8211;334.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23733849&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.113.150581" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib77"></a><p class="bibliomixed"><span class="bibliomisc">Labella, S., Woglar, A., Jantsch, V., and Zetka, M. (2011). Polo kinases establish links between meiotic chromosomes and cytoskeletal
                        forces essential for homolog pairing. Dev. Cell <span class="italic">21</span>, 948&#8211;958.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22018921&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.devcel.2011.07.011" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib78"></a><p class="bibliomixed"><span class="bibliomisc">Labrador, L., Barroso, C., Lightfoot, J., M&uuml;ller-Reichert, T., Flibotte, S., Taylor, J., Moerman, D.G., Villeneuve, A.M.,
                        and Martinez-Perez, E. (2013). Chromosome movements promoted by the mitochondrial protein <a href="http://www.wormbase.org/db/get?name=WP:CE19499;class=Protein" target="_blank">SPD-3</a> are required for homology search during <span class="italic">Caenorhabditis elegans</span> meiosis. PLoS Genet. <span class="italic">9</span>, e1003497.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23671424&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1003497" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib79"></a><p class="bibliomixed"><span class="bibliomisc">Lackner, M.R., and Kim, S.K. (1998). Genetic analysis of the <span class="italic">Caenorhabditis elegans</span> MAP kinase gene <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003401;class=Gene" target="_blank">mpk-1</a></span>. Genetics <span class="italic">150</span>, 103&#8211;117.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9725833&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib80"></a><p class="bibliomixed"><span class="bibliomisc">Lamelza, P., and Bhalla, N. (2012). Histone methyltransferases <a href="http://www.wormbase.org/db/get?name=WP:CE27781;class=Protein" target="_blank">MES-4</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE30503;class=Protein" target="_blank">MET-1</a> promote meiotic checkpoint activation in <span class="italic">Caenorhabditis elegans</span>. PLoS Genet. <span class="italic">8</span>, e1003089.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23166523&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1003089" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib81"></a><p class="bibliomixed"><span class="bibliomisc">Lange, J., Pan, J., Cole, F., Thelen, M.P., Jasin, M., and Keeney, S. (2011). ATM controls meiotic double-strand-break formation.
                        Nature <span class="italic">479</span>, 237&#8211;240.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22002603&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/nature10508" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib82"></a><p class="bibliomixed"><span class="bibliomisc">Lee, M.-H., Ohmachi, M., Arur, S., Nayak, S., Francis, R., Church, D., Lambie, E., and Schedl, T. (2007). Multiple functions
                        and dynamic activation of <a href="http://www.wormbase.org/db/get?name=WP:CE01583;class=Protein" target="_blank">MPK-1</a> extracellular signal-regulated kinase signaling in <span class="italic">Caenorhabditis elegans</span> germline development. Genetics <span class="italic">177</span>, 2039&#8211;2062.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18073423&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.107.081356" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib83"></a><p class="bibliomixed"><span class="bibliomisc">Lemmens, B.B., Johnson, N.M., and Tijsterman, M. (2013). <a href="http://www.wormbase.org/db/get?name=WP:CE43543;class=Protein" target="_blank">COM-1</a> promotes homologous recombination during <span class="italic">Caenorhabditis elegans</span> meiosis by antagonizing Ku-mediated non-homologous end joining. PLoS Genet. <span class="italic">9</span>, e1003276.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23408909&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1003276" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib84"></a><p class="bibliomixed"><span class="bibliomisc">Libuda, D.E., Uzawa, S., Meyer, B.J., and Villeneuve, A.M. (2013). Meiotic chromosome structures constrain and respond to
                        designation of crossover sites. Nature <span class="italic">502</span>, 703&#8211;706.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/nature12577" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib85"></a><p class="bibliomixed"><span class="bibliomisc">Lightfoot, J., Testori, S., Barroso, C., and Martinez-Perez, E. (2011). Loading of meiotic cohesin by SCC-2 is required for
                        early processing of DSBs and for the DNA damage checkpoint. Curr. Biol. <span class="italic">21</span>, 1421&#8211;1430.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21856158&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2011.07.007" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib86"></a><p class="bibliomixed"><span class="bibliomisc">Lim, J.G.Y., Stine, R.R.W., and Yanowitz, J.L. (2008). Domain-specific regulation of recombination in <span class="italic">Caenorhabditis elegans</span> in response to temperature, age and sex. Genetics <span class="italic">180</span>, 715&#8211;726.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18780748&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.108.090142" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib87"></a><p class="bibliomixed"><span class="bibliomisc">MacQueen, A.J., Colai&aacute;covo, M.P., McDonald, K., and Villeneuve, A.M. (2002). Synapsis-dependent and -independent mechanisms
                        stabilize homolog pairing during meiotic prophase in <span class="italic">C. elegans</span>. Genes Dev. <span class="italic">16</span>, 2428&#8211;2442.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12231631&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.1011602" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib88"></a><p class="bibliomixed"><span class="bibliomisc">MacQueen, A.J., Phillips, C.M., Bhalla, N., Weiser, P., Villeneuve, A.M., and Dernburg, A.F. (2005). Chromosome sites play
                        dual roles to establish homologous synapsis during meiosis in <span class="italic">C. elegans</span>. Cell <span class="italic">123</span>, 1037&#8211;1050.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16360034&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2005.09.034" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib89"></a><p class="bibliomixed"><span class="bibliomisc">MacQueen, A.J., and Villeneuve, A.M. (2001). Nuclear reorganization and homologous chromosome pairing during meiotic prophase
                        require <span class="italic">C. elegans <a href="http://www.wormbase.org/db/get?name=WBGene00000499;class=Gene" target="_blank">chk-2</a></span>. Genes Dev. <span class="italic">15</span>, 1674&#8211;1687.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11445542&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.902601" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib90"></a><p class="bibliomixed"><span class="bibliomisc">Mains, P.E., Kemphues, K.J., Sprunger, S.A., Sulston, I.A., and Wood, W.B. (1990). Mutations affecting the meiotic and mitotic
                        divisions of the early <span class="italic">Caenorhabditis elegans</span> embryo. Genetics <span class="italic">126</span>, 593&#8211;605.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2249759&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib91"></a><p class="bibliomixed"><span class="bibliomisc">Malone, C.J., Misner, L., Le Bot, N., Tsai, M.-C., Campbell, J.M., Ahringer, J., and White, J.G. (2003). The <span class="italic">C. elegans</span> hook protein, <a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a>, mediates the essential attachment between the centrosome and nucleus. Cell <span class="italic">115</span>, 825&#8211;836.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0092-8674(03)00985-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib92"></a><p class="bibliomixed"><span class="bibliomisc">Martin, J.S., Winkelmann, N., Petalcorin, M.I.R., McIlwraith, M.J., and Boulton, S.J. (2005). RAD-51-dependent and -independent
                        roles of a <span class="italic">Caenorhabditis elegans</span> BRCA2-related protein during DNA double-strand break repair. Mol. Cell. Biol. <span class="italic">25</span>, 3127&#8211;3139.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1128/MCB.25.8.3127-3139.2005" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib93"></a><p class="bibliomixed"><span class="bibliomisc">Martinez-Perez, E., Schvarzstein, M., Barroso, C., Lightfoot, J., Dernburg, A.F., and Villeneuve, A.M. (2008). Crossovers
                        trigger a remodeling of meiotic chromosome axis composition that is linked to two-step loss of sister chromatid cohesion.
                        Genes Dev. <span class="italic">22</span>, 2886&#8211;2901.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib94"></a><p class="bibliomixed"><span class="bibliomisc">Martinez-Perez, E., and Villeneuve, A.M. (2005). HTP-1-dependent constraints coordinate homolog pairing and synapsis and promote
                        chiasma formation during <span class="italic">C. elegans</span> meiosis. Genes Dev. <span class="italic">19</span>, 2727&#8211;2743.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.1338505" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib95"></a><p class="bibliomixed"><span class="bibliomisc">McKim, K.S., Howell, A.M., and Rose, A.M. (1988). The effects of translocations on recombination frequency in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">120</span>, 987&#8211;1001.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib96"></a><p class="bibliomixed"><span class="bibliomisc">McNicoll, F., Stevense, M., and Jessberger, R. (2013). Cohesin in gametogenesis. Curr. Top. Dev. Biol. <span class="italic">102</span>, 1&#8211;34.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/B978-0-12-416024-8.00001-5" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib97"></a><p class="bibliomixed"><span class="bibliomisc">Meneely, P.M., Farago, A.F., and Kauffman, T.M. (2002). Crossover distribution and high interference for both the <span class="italic">X</span> chromosome and an autosome during oogenesis and spermatogenesis in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">162</span>, 1169&#8211;1177.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib98"></a><p class="bibliomixed"><span class="bibliomisc">Meneely, P.M., McGovern, O.L., Heinis, F.I., and Yanowitz, J.L. (2012). Crossover distribution and frequency are regulated
                        by <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001864;class=Gene" target="_blank">him-5</a></span> in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">190</span>, 1251&#8211;1266.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.111.137463" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib99"></a><p class="bibliomixed"><span class="bibliomisc">Mets, D.G., and Meyer, B.J. (2009). Condensins regulate meiotic DNA break distribution, thus crossover frequency, by controlling
                        chromosome structure. Cell <span class="italic">139</span>, 73&#8211;86.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2009.07.035" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib100"></a><p class="bibliomixed"><span class="bibliomisc">Meyer, B. J. X-Chromosome dosage compensation (June 25, 2005), <span class="italic">WormBook</span>, ed. The <span class="italic">C. elegans</span> Research Community, WormBook, doi/10.1895/wormbook.1.8.1, http://www.wormbook.org.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib101"></a><p class="bibliomixed"><span class="bibliomisc">Meyer, B.J. (2010). Targeting <span class="italic">X</span> chromosomes for repression. Curr. Opin. Genet. Dev. <span class="italic">20</span>, 179&#8211;189.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.gde.2010.03.008" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib102"></a><p class="bibliomixed"><span class="bibliomisc">Minn, I.L., Rolls, M.M., Hanna-Rose, W., and Malone, C.J. (2009). <a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE32088;class=Protein" target="_blank">ZYG-12</a>, mediators of centrosome-nucleus attachment, are a functional SUN/KASH pair in <span class="italic">Caenorhabditis elegans</span>. Mol. Biol. Cell <span class="italic">20</span>, 4586&#8211;4595.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1091/mbc.E08-10-1034" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib103"></a><p class="bibliomixed"><span class="bibliomisc">Mlynarczyk-Evans, S., Roelens, B., and Villeneuve, A.M. (2013). Evidence that masking of synapsis imperfections counterbalances
                        quality control to promote efficient meiosis. PLoS Genet. <span class="italic">9</span>, e1003963.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1003963." target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib104"></a><p class="bibliomixed"><span class="bibliomisc">Monen, J., Maddox, P.S., Hyndman, F., Oegema, K., and Desai, A. (2005). Differential role of CENP-A in the segregation of
                        holocentric <span class="italic">C. elegans</span> chromosomes during meiosis and mitosis. Nat. Cell Biol. <span class="italic">7</span>, 1248&#8211;1255.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/ncb1331" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib105"></a><p class="bibliomixed"><span class="bibliomisc">Muller, H.J. (1916). The mechanism of crossing-over. Am. Nat. <span class="italic">50</span>, 193&#8211;221, 284&#8211;305, 350&#8211;366, 421&#8211;434.</span>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1086/279553" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib105a"></a><p class="bibliomixed"><span class="bibliomisc">Muscat, C.C., Torre-Santiago, K.M., Tran, M.V., Powers, J.A., and Wignall, S.M. (2015). Kinetochore-independent chromosome
                        segregation driven by lateral microtubule bundles. eLife <span class="italic">4</span>, e06462.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.7554/eLife.06462" target="_blank">Article</a></p>
               </div>
               <div class="bibliomixed"><a name="bib106"></a><p class="bibliomixed"><span class="bibliomisc">Nabeshima, K., Mlynarczyk-Evans, S., and Villeneuve, A.M. (2011). Chromosome painting reveals asynaptic full alignment of
                        homologs and HIM-8-dependent remodeling of <span class="italic">X</span> chromosome territories during <span class="italic">Caenorhabditis elegans</span> meiosis. PLoS Genet. <span class="italic">7</span>, e1002231.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1002231" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib107"></a><p class="bibliomixed"><span class="bibliomisc">Nabeshima, K., Villeneuve, A.M., and Colai&aacute;covo, M.P. (2005). Crossing over is coupled to late meiotic prophase bivalent differentiation
                        through asymmetric disassembly of the SC. J. Cell Biol. <span class="italic">168</span>, 683&#8211;689.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib108"></a><p class="bibliomixed"><span class="bibliomisc">Nabeshima, K., Villeneuve, A.M., and Hillers, K.J. (2004). Chromosome-wide regulation of meiotic crossover formation in <span class="italic">Caenorhabditis elegans</span> requires properly assembled chromosome axes. Genetics <span class="italic">168</span>, 1275&#8211;1292.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib109"></a><p class="bibliomixed"><span class="bibliomisc">Nicolas, A., Treco, D., Schultes, N.P., and Szostak, J.W. (1989). An initiation site for meiotic gene conversion in the yeast
                        <span class="italic">Saccharomyces cerevisiae</span>. Nature <span class="italic">338</span>, 35&#8211;39.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib110"></a><p class="bibliomixed"><span class="bibliomisc">Nottke, A.C., Beese-Sims, S.E., Pantalena, L.F., Reinke, V., Shi, Y., and Colai&aacute;covo, M.P. (2011). <a href="http://www.wormbase.org/db/get?name=WP:CE20240;class=Protein" target="_blank">SPR-5</a> is a histone H3K4 demethylase with a role in meiotic double-strand break repair. Proc. Natl. Acad. Sci. U. S. A. <span class="italic">108</span>, 12805&#8211;12810.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib111"></a><p class="bibliomixed"><span class="bibliomisc">O'Neil, N.J., Martin, J.S., Youds, J.L., Ward, J.D., Petalcorin, M.I.R., Rose, A.M., and Boulton, S.J. (2013). Joint molecule
                        resolution requires the redundant activities of <a href="http://www.wormbase.org/db/get?name=WP:CE32609;class=Protein" target="_blank">MUS-81</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE24855;class=Protein" target="_blank">XPF-1</a> during <span class="italic">Caenorhabditis elegans</span> meiosis. PLoS Genet. <span class="italic">9</span>, e1003582.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23874209&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib112"></a><p class="bibliomixed"><span class="bibliomisc">P&acirc;ques, F., and Haber, J.E. (1999). Multiple pathways of recombination induced by double-strand breaks in <span class="italic">Saccharomyces cerevisiae</span>. Microbiol. Mol. Biol. Rev. <span class="italic">63</span>, 349&#8211;404.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10357855&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib113"></a><p class="bibliomixed"><span class="bibliomisc">Pasierbek, P., F&ouml;dermayr, M., Jantsch, V., Jantsch, M., Schweizer, D., and Loidl, J. (2003). The <span class="italic">Caenorhabditis elegans</span> <a href="http://www.wormbase.org/db/get?name=WP:CE05664;class=Protein" target="_blank">SCC-3</a> homologue is required for meiotic synapsis and for proper chromosome disjunction in mitosis and meiosis. Exp. Cell Res. <span class="italic">289</span>, 245&#8211;255.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14499625&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0014-4827(03)00266-0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib114"></a><p class="bibliomixed"><span class="bibliomisc">Pasierbek, P., Jantsch, M., Melcher, M., Schleiffer, A., Schweizer, D., and Loidl, J. (2001). A <span class="italic">Caenorhabditis elegans</span> cohesion protein with functions in meiotic chromosome pairing and disjunction. Genes Dev. <span class="italic">15</span>, 1349&#8211;1360.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11390355&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.192701" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib115"></a><p class="bibliomixed"><span class="bibliomisc">Penkner, A., Portik-Dobos, Z., Tang, L., Schnabel, R., Novatchkova, M., Jantsch, V., and Loidl, J. (2007a). A conserved function
                        for a <span class="italic">Caenorhabditis elegans</span> Com1/Sae2/CtIP protein homolog in meiotic recombination. EMBO J. <span class="italic">26</span>, 5071&#8211;5082.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18007596&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/sj.emboj.7601916" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib116"></a><p class="bibliomixed"><span class="bibliomisc">Penkner, A., Tang, L., Novatchkova, M., Ladurner, M., Fridkin, A., Gruenbaum, Y., Schweizer, D., Loidl, J., and Jantsch, V.
                        (2007b). The nuclear envelope protein Matefin/<a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> is required for homologous pairing in <span class="italic">C. elegans</span> meiosis. Dev. Cell <span class="italic">12</span>, 873&#8211;885.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17543861&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.devcel.2007.05.004" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib117"></a><p class="bibliomixed"><span class="bibliomisc">Penkner, A.M., Fridkin, A., Gloggnitzer, J., Baudrimont, A., Machacek, T., Woglar, A., Csaszar, E., Pasierbek, P., Ammerer,
                        G., Gruenbaum, Y., and Jantsch, V.l. (2009). Meiotic chromosome homology search involves modifications of the nuclear envelope
                        protein Matefin/<a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a>. Cell <span class="italic">139</span>, 920&#8211;933.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913286&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib118"></a><p class="bibliomixed"><span class="bibliomisc">Petalcorin, M.I.R., Galkin, V.E., Yu, X., Egelman, E.H., and Boulton, S.J. (2007). Stabilization of RAD-51-DNA filaments via
                        an interaction domain in C<span class="italic">aenorhabditis elegans</span> BRCA2. Proc. Natl. Acad. Sci. U. S. A. <span class="italic">104</span>, 8299&#8211;8304.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17483448&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1073/pnas.0702805104" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib119"></a><p class="bibliomixed"><span class="bibliomisc">Petalcorin, M.I.R., Sandall, J., Wigley, D.B., and Boulton, S.J. (2006). CeBRC-2 stimulates D-loop formation by <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> and promotes DNA single-strand annealing. J. Mol. Biol. <span class="italic">361</span>, 231&#8211;242.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16843491&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib120"></a><p class="bibliomixed"><span class="bibliomisc">Phillips, C.M., and Dernburg, A.F. (2006). A family of zinc-finger proteins is required for chromosome-specific pairing and
                        synapsis during meiosis in <span class="italic">C. elegans</span>. Dev. Cell <span class="italic">11</span>, 817&#8211;829.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17141157&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib121"></a><p class="bibliomixed"><span class="bibliomisc">Phillips, C.M., Meng, X., Zhang, L., Chretien, J.H., Urnov, F.D., and Dernburg, A.F. (2009). Identification of chromosome
                        sequence motifs that mediate meiotic pairing and synapsis in <span class="italic">C. elegans</span>. Nat. Cell Biol. <span class="italic">11</span>, 934&#8211;942.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib122"></a><p class="bibliomixed"><span class="bibliomisc">Phillips, C.M., Wong, C., Bhalla, N., Carlton, P.M., Weiser, P., Meneely, P.M., and Dernberg, A.F. (2005). <a href="http://www.wormbase.org/db/get?name=WP:CE36715;class=Protein" target="_blank"><a href="http://www.wormbase.org/db/get?name=HIM-8;class=Strain" target="_blank">HIM-8</a></a> binds to the <span class="italic">X</span> chromosome pairing center and mediates chromosome-specific meiotic synapsis. Cell <span class="italic">123</span>, 1051&#8211;1063.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib123"></a><p class="bibliomixed"><span class="bibliomisc">Qiao, H., Prasada Rao, H.B.D., Yang, Y., Fong, J.H., Cloutier, J.M., Deacon, D.C., et al. (2014). Antagonistic roles of ubiquitin
                        ligase HEI10 and SUMO ligase RNF212 regulate meiotic recombination. Nat. Genet. <span class="italic">46</span>, 194&#8211;199.</span>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/ng.2858" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib124"></a><p class="bibliomixed"><span class="bibliomisc">Reddy, K.C., and Villeneuve, A.M. (2004). <span class="italic">C. elegans</span> <a href="http://www.wormbase.org/db/get?name=WP:CE13489;class=Protein" target="_blank">HIM-17</a> links chromatin modification and competence for initiation of meiotic recombination. Cell <span class="italic">118</span>, 439&#8211;452.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2004.07.026" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib125"></a><p class="bibliomixed"><span class="bibliomisc">Rinaldo, C., Bazzicalupo, P., Ederle, S., Hilliard, M., and La Volpe, A. (2002). Roles for <span class="italic">Caenorhabditis elegans</span> <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004297;class=Gene" target="_blank">rad-51</a></span> in meiosis and in resistance to ionizing radiation during development. Genetics <span class="italic">160</span>, 471&#8211;479.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib126"></a><p class="bibliomixed"><span class="bibliomisc">Robert, V.J., Davis, M.W., Jorgensen, E.M., and Bessereau, J.-L. (2008). Gene conversion and end-joining-repair double-strand
                        breaks in the <span class="italic">Caenorhabditis elegans</span> germline. Genetics <span class="italic">180</span>, 673&#8211;679.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.108.089698" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib127"></a><p class="bibliomixed"><span class="bibliomisc">Rockman, M.V., and Kruglyak, L. (2009). Recombinational landscape and population genomics of <span class="italic">Caenorhabditis elegans</span>. PLoS Genet. <span class="italic">5</span>, e1000419.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1000419" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib128"></a><p class="bibliomixed"><span class="bibliomisc">Rog, O., and Dernburg, A.F. (2013). Chromosome pairing and synapsis during <span class="italic">Caenorhabditis elegans</span> meiosis. Curr. Opin. Cell Biol. <span class="italic">25</span>, 349&#8211;56.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib129"></a><p class="bibliomixed"><span class="bibliomisc">Rog, O., and Dernburg, A.F. (2015). Direct visualization reveals kinetics of meiotic chromosome synapsis. Cell Rep. <span class="italic">10</span>, 1639-1645.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.celrep.2015.02.032" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib130"></a><p class="bibliomixed"><span class="bibliomisc">Rogers, E., Bishop, J.D., Waddle, J.A., Schumacher, J.M., and Lin, R. (2002). The aurora kinase <a href="http://www.wormbase.org/db/get?name=WP:CE24761;class=Protein" target="_blank">AIR-2</a> functions in the release of chromosome cohesion in <span class="italic">Caenorhabditis elegans</span> meiosis. J. Cell Biol. <span class="italic">157</span>, 219&#8211;229.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1083/jcb.200110045" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib131"></a><p class="bibliomixed"><span class="bibliomisc">Rose, A.M., and Baillie, D.L. (1979a). A mutation in <span class="italic">Caenorhabditis elegans</span> that increases recombination frequency more than threefold. Nature <span class="italic">281</span>, 599&#8211;600.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=492325&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/281599a0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib132"></a><p class="bibliomixed"><span class="bibliomisc">Rose, A.M., and Baillie, D.L. (1979b). The effect of temperature and parental age on recombination and nondisjunction in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">92</span>, 409&#8211;418.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17248928&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib133"></a><p class="bibliomixed"><span class="bibliomisc">Rose, A.M., Baillie, D.L., and Curran, J. (1984). Meiotic pairing behavior of two free duplications of linkage group <span class="italic">I</span> in <span class="italic">Caenorhabditis elegans</span>. Mol. Gen. Genet. <span class="italic">195</span>, 52&#8211;56.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6593563&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1007/BF00332723" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib134"></a><p class="bibliomixed"><span class="bibliomisc">Rosenbluth, R.E., and Baillie, D.L. (1981). The genetic analysis of a reciprocal translocation, eT1(<span class="italic">III; V</span>), in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">99</span>, 415&#8211;428.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6953041&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib135"></a><p class="bibliomixed"><span class="bibliomisc">Rosu, S., Libuda, D.E., and Villeneuve, A.M. (2011). Robust crossover assurance and regulated interhomolog access maintain
                        meiotic crossover number. Science <span class="italic">334</span>, 1286&#8211;1289.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22144627&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1126/science.1212424" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib136"></a><p class="bibliomixed"><span class="bibliomisc">Rosu, S., Zawadzki, K.A., Stamper, E.L., Libuda, D.E., Reese, A.L., Dernburg, A.F., and Villeneuve, A.M. (2013). The <span class="italic">C. elegans</span> <a href="http://www.wormbase.org/db/get?name=WP:CE44432;class=Protein" target="_blank">DSB-2</a> protein reveals a regulatory network that controls competence for meiotic DSB formation and promotes crossover assurance.
                        PLoS Genet. <span class="italic">9</span>, e1003674.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23950729&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1003674" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib137"></a><p class="bibliomixed"><span class="bibliomisc">Saito, T.T., Lui, D.Y., Kim, H.-M., Meyer, K., and Colai&aacute;covo, M.P. (2013). Interplay between structure-specific endonucleases
                        for crossover control during <span class="italic">Caenorhabditis elegans</span> meiosis. PLoS Genet. <span class="italic">9</span>, e1003586.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23874210&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1003586" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib138"></a><p class="bibliomixed"><span class="bibliomisc">Saito, T.T., Mohideen, F., Meyer, K., Harper, J.W., and Colai&aacute;covo, M.P. (2012). <a href="http://www.wormbase.org/db/get?name=WP:CE11216;class=Protein" target="_blank">SLX-1</a> is required for maintaining genomic integrity and promoting meiotic noncrossovers in the <span class="italic">Caenorhabditis elegans</span> germline. PLoS Genet. <span class="italic">8</span>, e1002888.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22927825&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1002888" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib139"></a><p class="bibliomixed"><span class="bibliomisc">Saito, T.T., Youds, J.L., Boulton, S.J., and Colai&aacute;covo, M.P. (2009). <span class="italic">Caenorhabditis elegans</span> <a href="http://www.wormbase.org/db/get?name=WP:CE48043;class=Protein" target="_blank">HIM-18</a>/SLX-4 interacts with <a href="http://www.wormbase.org/db/get?name=WP:CE11216;class=Protein" target="_blank">SLX-1</a> and <a href="http://www.wormbase.org/db/get?name=WP:CE24855;class=Protein" target="_blank">XPF-1</a> and maintains genomic integrity in the germline by processing recombination intermediates. PLoS Genet. <span class="italic">5</span>, e1000735.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19936019&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1000735" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib140"></a><p class="bibliomixed"><span class="bibliomisc">Sanford, C., and Perry, M.D. (2001). Asymmetrically distributed oligonucleotide repeats in the <span class="italic">Caenorhabditis elegans</span> genome sequence that map to regions important for meiotic chromosome segregation. Nucleic Acids Res. <span class="italic">29</span>, 2920&#8211;2926.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11452017&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1093/nar/29.14.2920" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib141"></a><p class="bibliomixed"><span class="bibliomisc">Sato, A., Isaac, B., Phillips, C.M., Rillo, R., Carlton, P.M., Wynne, D.J., Kasad, R.A., and Dernburg. A.F. (2009). Cytoskeletal
                        forces span the nuclear envelope to coordinate meiotic chromosome pairing and synapsis. Cell <span class="italic">139</span>, 907&#8211;919.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2009.10.039" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib142"></a><p class="bibliomixed"><span class="bibliomisc">Sato-Carlton, A., Li, X., Crawley, O., Testori, S., Martinez-Perez, E., Sugimoto, A., and Carlton, P.M. (2014). Protein phosphatase
                        4 promotes chromosome pairing and Ssynapsis, and contributes to maintaining crossover competence with increasing age. PLoS
                        Genet. <span class="italic">10</span>, e1004638.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1004638" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib143"></a><p class="bibliomixed"><span class="bibliomisc">Schild-Pr&uuml;fert, K., Saito, T.T., Smolikov, S., Gu, Y., Hincapie, M., Hill, D.E., Vidal, M., McDonald, K., and Colai&aacute;covo,
                        M.P. (2011). Organization of the synaptonemal complex during meiosis in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">189</span>, 411&#8211;421.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.111.132431" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib144"></a><p class="bibliomixed"><span class="bibliomisc">Schumacher, B., Hofmann, K., Boulton, S., and Gartner, A. (2001). The <span class="italic">C. elegans</span> homolog of the p53 tumor suppressor is required for DNA damage-induced apoptosis. Curr. Biol. <span class="italic">11</span>, 1722&#8211;1727.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0960-9822(01)00534-6" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib145"></a><p class="bibliomixed"><span class="bibliomisc">Schvarzstein, M., Pattabiraman, D., Bembenek, J.N., and Villeneuve, A.M. (2013). Meiotic HORMA domain proteins prevent untimely
                        centriole disengagement during <span class="italic">Caenorhabditis elegans</span> spermatocyte meiosis. Proc. Natl. Acad. Sci. U. S. A. <span class="italic">110</span>, E898&#8211;E907.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1073/pnas.1213888110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib146"></a><p class="bibliomixed"><span class="bibliomisc">Schvarzstein, M., Pattabiraman, D., Libuda, D.E., Ramadugu, A., Tam, A., Martinez-Perez, E., Roelens, B., Zawadzki, K.A.,
                        Yokoo, R., Rosu, S., et al. (2014). DNA helicase <a href="http://www.wormbase.org/db/get?name=WP:CE31724;class=Protein" target="_blank">HIM-6</a>/BLM both promotes MutS&#947;-dependent crossovers and antagonizes MutS&#947;-independent interhomolog associations during <span class="italic">Caenorhabditis elegans</span> meiosis. Genetics <span class="italic">198</span>, 193&#8211;207.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.114.161513" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib147"></a><p class="bibliomixed"><span class="bibliomisc">Schvarzstein, M., Wignall, S.M., and Villeneuve, A.M. (2010). Coordinating cohesion, co-orientation, and congression during
                        meiosis: lessons from holocentric chromosomes. Genes Dev. <span class="italic">24</span>, 219&#8211;228.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.1863610" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib148"></a><p class="bibliomixed"><span class="bibliomisc">Segbert, C., Barkus, R., Powers, J., Strome, S., Saxton, W.M., and Bossinger, O. (2003). <a href="http://www.wormbase.org/db/get?name=WP:CE07978;class=Protein" target="_blank">KLP-18</a>, a Klp2 kinesin, is required for assembly of acentrosomal meiotic spindles in <span class="italic">Caenorhabditis elegans</span>. Mol. Biol. Cell <span class="italic">14</span>, 4458&#8211;4469.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1091/mbc.E03-05-0283" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib149"></a><p class="bibliomixed"><span class="bibliomisc">Severson, A.F., Ling, L., Zuylen, V. van, and Meyer, B.J. (2009). The axial element protein <a href="http://www.wormbase.org/db/get?name=WP:CE11330;class=Protein" target="_blank">HTP-3</a> promotes cohesin loading and meiotic axis assembly in <span class="italic">C. elegans</span> to implement the meiotic program of chromosome segregation. Genes Dev. <span class="italic">23</span>, 1763&#8211;1778.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.1808809" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib150"></a><p class="bibliomixed"><span class="bibliomisc">Severson, A.F., and Meyer, B.J. (2014). Divergent kleisin subunits of cohesin specify mechanisms to tether and release meiotic
                        chromosomes. eLife <span class="italic">3</span>, e03467.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.1808809" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib151"></a><p class="bibliomixed"><span class="bibliomisc">Shakes, D.C., Wu, J.-C., Sadler, P.L., Laprade, K., Moore, L.L., Noritake, A., and Chu, D.S. (2009). Spermatogenesis-specific
                        features of the meiotic program in <span class="italic">Caenorhabditis elegans</span>. PLoS Genet. <span class="italic">5</span>, e1000611.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19696886&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1000611" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib152"></a><p class="bibliomixed"><span class="bibliomisc">Silva, N., Adamo, A., Santonicola, P., Martinez-Perez, E., and La Volpe, A. (2013). Pro-crossover factors regulate damage-dependent
                        apoptosis in the <span class="italic">Caenorhabditis elegans</span> germ line. Cell Death Differ. <span class="italic">20</span>, 1209&#8211;1218.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23832114&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/cdd.2013.68" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib153"></a><p class="bibliomixed"><span class="bibliomisc">Silva, N., Ferrandiz, N., Barroso, C., Tognetti, S., Lightfoot, J., Telecan, O., Encheva, V., Faull, P., Hanni, S., Furger,
                        A., et al. (2014). The fidelity of synaptonemal complex assembly is regulated by a signaling mechanism that controls early
                        meiotic progression. Dev. Cell <span class="italic">31</span>, 503&#8211;511.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=25455309&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib154"></a><p class="bibliomixed"><span class="bibliomisc">Siomos, M.F., Badrinath, A., Pasierbek, P., Livingstone, D., White, J., Glotzer, M., and Nasmyth, K. (2001). Separase is required
                        for chromosome segregation during meiosis I in <span class="italic">Caenorhabditis elegans</span>. Curr. Biol. <span class="italic">11</span>, 1825&#8211;1835.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11728305&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0960-9822(01)00588-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib155"></a><p class="bibliomixed"><span class="bibliomisc">Smolikov, S., Eizinger, A., Hurlburt, A., Rogers, E., Villeneuve, A.M., and Colai&aacute;covo, M.P. (2007a). Synapsis-defective mutants
                        reveal a correlation between chromosome conformation and the mode of double-strand break repair during <span class="italic">Caenorhabditis elegans</span> meiosis. Genetics <span class="italic">176</span>, 2027&#8211;2033.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17565963&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.107.076968" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib156"></a><p class="bibliomixed"><span class="bibliomisc">Smolikov, S., Eizinger, A., Schild-Prufert, K., Hurlburt, A., McDonald, K., Engebrecht, J., et al. (2007b). <a href="http://www.wormbase.org/db/get?name=WP:CE41238;class=Protein" target="_blank">SYP-3</a> restricts synaptonemal complex assembly to bridge paired chromosome axes during meiosis in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">176</span>, 2015&#8211;2025.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17565948&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.107.072413" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib157"></a><p class="bibliomixed"><span class="bibliomisc">Smolikov, S., Schild-Pr&uuml;fert, K., and Colai&aacute;covo, M.P. (2008). <a href="http://www.wormbase.org/db/get?name=WP:CE31579;class=Protein" target="_blank">CRA-1</a> uncovers a double-strand break-dependent pathway promoting the assembly of central region proteins on chromosome axes during
                        <span class="italic">C. elegans</span> meiosis. PLoS Genet. <span class="italic">4</span>, e1000088.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18535664&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1000088" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib158"></a><p class="bibliomixed"><span class="bibliomisc">Smolikov, S., Schild-Pr&uuml;fert, K., and Colai&aacute;covo, M.P. (2009). A yeast two-hybrid screen for <a href="http://www.wormbase.org/db/get?name=WP:CE41238;class=Protein" target="_blank">SYP-3</a> interactors identifies <a href="http://www.wormbase.org/db/get?name=WP:CE23830;class=Protein" target="_blank">SYP-4</a>, a component required for synaptonemal complex assembly and chiasma formation in <span class="italic">Caenorhabditis elegans</span> meiosis. PLoS Genet. <span class="italic">5</span>, e1000669.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19798442&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1000669" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib159"></a><p class="bibliomixed"><span class="bibliomisc">Song, A., Labella, S., Korneeva, N.L., Keiper, B.D., Aamodt, E.J., Zetka, M., and Rhoads, R.E. (2010). A <span class="italic">C. elegans</span> eIF4E-family member upregulates translation at elevated temperatures of mRNAs encoding <a href="http://www.wormbase.org/db/get?name=WP:CE26678;class=Protein" target="_blank">MSH-5</a> and other meiotic crossover proteins. J. Cell Sci. <span class="italic">123</span>, 2228&#8211;2237.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20530576&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1242/jcs.063107" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib160"></a><p class="bibliomixed"><span class="bibliomisc">Spike, C.A., Coetzee, D., Eichten, C., Wang, X., Hansen, D., and Greenstein, D. (2014). The TRIM-NHL protein <a href="http://www.wormbase.org/db/get?name=WP:CE27062;class=Protein" target="_blank">LIN-41</a> and the OMA RNA-binding proteins antagonistically control the prophase-to-metaphase transition and growth of <span class="italic">Caenorhabditis elegans</span> oocytes. Genetics <span class="italic">198</span>, 1535&#8211;1558.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=25261698&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1534/genetics.114.168831" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib161"></a><p class="bibliomixed"><span class="bibliomisc">Srayko, M., Buster, D.W., Bazirgan, O.A., McNally, F.J., and Mains, P.E. (2000). <a href="http://www.wormbase.org/db/get?name=WP:CE32479;class=Protein" target="_blank">MEI-1</a>/<a href="http://www.wormbase.org/db/get?name=WP:CE11320;class=Protein" target="_blank">MEI-2</a> katanin-like microtubule severing activity is required for <span class="italic">Caenorhabditis elegans</span> meiosis. Genes Dev. <span class="italic">14</span>, 1072&#8211;1084.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib162"></a><p class="bibliomixed"><span class="bibliomisc">Stamper, E.L., Rodenbusch, S.E., Rosu, S., Ahringer, J., Villeneuve, A.M., and Dernburg, A.F. (2013). Identification of <a href="http://www.wormbase.org/db/get?name=WP:CE05571;class=Protein" target="_blank">DSB-1</a>, a protein required for initiation of meiotic recombination in <span class="italic">Caenorhabditis elegans</span>, illuminates a crossover assurance checkpoint. PLoS Genet. <span class="italic">9</span>, e1003679.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1003679." target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib163"></a><p class="bibliomixed"><span class="bibliomisc">Stevens, D., Oegema, K., and Desai, A. (2013). Meiotic double-strand breaks uncover and protect against mitotic errors in
                        the <span class="italic">C. elegans</span> germline. Curr. Biol. <span class="italic">23</span>, 2400&#8211;2406.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2013.10.015" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib164"></a><p class="bibliomixed"><span class="bibliomisc">Takanami, T., Mori, A., Takahashi, H., Horiuchi, S., and Higashitani, A. (2003). <span class="italic">Caenorhabditis elegans Ce-rdh-1/<a href="http://www.wormbase.org/db/get?name=WBGene00004297;class=Gene" target="_blank">rad-51</a></span> functions after double-strand break formation of meiotic recombination. Chromosome Res. <span class="italic">11</span>, 125&#8211;135.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1023/A:1022863814686" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib165"></a><p class="bibliomixed"><span class="bibliomisc">Tang, L., Machacek, T., Mamnun, Y.M., Penkner, A., Gloggnitzer, J., Wegrostek, C., Konrat, R., Jantsch, M.F., Loidl, J., and
                        Jantsch, V. (2010). Mutations in <span class="italic">Caenorhabditis elegans <a href="http://www.wormbase.org/db/get?name=WBGene00022390;class=Gene" target="_blank">him-19</a></span> show meiotic defects that worsen with age. Mol. Biol. Cell <span class="italic">21</span>, 885&#8211;896.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib166"></a><p class="bibliomixed"><span class="bibliomisc">Thacker, D., Mohibullah, N., Zhu, X., and Keeney, S. (2014). Homologue engagement controls meiotic DNA break number and distribution.
                        Nature <span class="italic">510</span>, 241&#8211;246.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1023/A:1022863814686" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib167"></a><p class="bibliomixed"><span class="bibliomisc">Tsai, C.J., Mets, D.G., Albrecht, M.R., Nix, P., Chan, A., and Meyer, B.J. (2008). Meiotic crossover number and distribution
                        are regulated by a dosage compensation protein that resembles a condensin subunit. Genes Dev. <span class="italic">22</span>, 194&#8211;211.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.1618508" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib168"></a><p class="bibliomixed"><span class="bibliomisc">Tzur, Y.B., de Carvalho, C.E., Nadarajan, S., Bostelen, I. Van, Gu, Y., Chu, D.S., Cheeseman, I.M., Colai&aacute;covo, M.P. (2012).
                        <a href="http://www.wormbase.org/db/get?name=WP:CE15580;class=Protein" target="_blank">LAB-1</a> targets PP1 and restricts Aurora B kinase upon entrance into meiosis to promote sister chromatid cohesion. PLoS Biol. <span class="italic">10</span>, e1001378.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pbio.1001378" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib169"></a><p class="bibliomixed"><span class="bibliomisc">Villeneuve, A.M. (1994). A cis-acting locus that promotes crossing over between <span class="italic">X</span> chromosomes in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">136</span>, 887&#8211;902.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19913287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib170"></a><p class="bibliomixed"><span class="bibliomisc">van der Voet, M., Berends, C.W.H., Perreault, A., Nguyen-Ngoc, T., G&ouml;nczy, P., Vidal, M., Boxem, M., and van den Heuvel, S.
                        (2009). NuMA-related <a href="http://www.wormbase.org/db/get?name=WP:CE18951;class=Protein" target="_blank">LIN-5</a>, <a href="http://www.wormbase.org/db/get?name=WP:CE15710;class=Protein" target="_blank">ASPM-1</a>, calmodulin and dynein promote meiotic spindle rotation independently of cortical <a href="http://www.wormbase.org/db/get?name=WP:CE18951;class=Protein" target="_blank">LIN-5</a>/GPR/G&#945;. Nat. Cell Biol. <span class="italic">11</span>, 269&#8211;277.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=26026148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/ncb1834." target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib171"></a><p class="bibliomixed"><span class="bibliomisc">Wagner, C.R., Kuervers, L., Baillie, D.L., and Yanowitz, J.L. (2010). <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001514;class=Gene" target="_blank">xnd-1</a></span> regulates the global recombination landscape in <span class="italic">Caenorhabditis elegans</span>. Nature <span class="italic">467</span>, 839&#8211;843.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20944745&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/nature09429" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib172"></a><p class="bibliomixed"><span class="bibliomisc">Wang, F., Yoder, J., Antoshechkin, I., and Han, M. (2003). <span class="italic">Caenorhabditis elegans</span> <a href="http://www.wormbase.org/db/get?name=WP:CE33526;class=Protein" target="_blank">EVL-14</a>/PDS-5 and <a href="http://www.wormbase.org/db/get?name=WP:CE05664;class=Protein" target="_blank">SCC-3</a> are essential for sister chromatid cohesion in meiosis and mitosis. Mol. Cell. Biol. <span class="italic">23</span>, 7698&#8211;7707.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14560015&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1128/MCB.23.21.7698-7707.2003" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib173"></a><p class="bibliomixed"><span class="bibliomisc">Ward, J.D., Muzzini, D.M., Petalcorin, M.I.R., Martinez-Perez, E., Martin, J.S., Plevani, P., Cassata, G., Marini, F., and
                        Boulton, S.J. (2010). Overlapping mechanisms promote postsynaptic <a href="http://www.wormbase.org/db/get?name=WP:CE29064;class=Protein" target="_blank">RAD-51</a> filament disassembly during meiotic double-strand break repair. Mol. Cell <span class="italic">37</span>, 259&#8211;272.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20122407&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.molcel.2009.12.026" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib174"></a><p class="bibliomixed"><span class="bibliomisc">Wignall, S.M., and Villeneuve, A.M. (2009). Lateral microtubule bundles promote chromosome alignment during acentrosomal oocyte
                        meiosis. Nat. Cell Biol. <span class="italic">11</span>, 839&#8211;844.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19525937&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/ncb1891" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib175"></a><p class="bibliomixed"><span class="bibliomisc">Winand, N.J., Panzer, J.A., and Kolodner, R.D. (1998). Cloning and characterization of the human and <span class="italic">Caenorhabditis elegans</span> homologs of the <span class="italic">Saccharomyces cerevisiae</span> MSH5 gene. Genomics <span class="italic">53</span>, 69&#8211;80.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9787078&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1006/geno.1998.5447" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib176"></a><p class="bibliomixed"><span class="bibliomisc">Woglar, A., Daryabeigi, A., Adamo, A., Habacher, C., Machacek, T., La Volpe, A., and Jantsch, V. (2013). Matefin/<a href="http://www.wormbase.org/db/get?name=WP:CE11288;class=Protein" target="_blank">SUN-1</a> phosphorylation is part of a surveillance mechanism to coordinate chromosome synapsis and recombination with meiotic progression
                        and chromosome movement. PLoS Genet. <span class="italic">9</span>, e1003335.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23505384&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1003335" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib177"></a><p class="bibliomixed"><span class="bibliomisc">Wood, A.J., Severson, A.F., and Meyer, B.J. (2010). Condensin and cohesin complexity: the expanding repertoire of functions.
                        Nat. Rev. Genet. <span class="italic">11</span>, 391&#8211;404.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20442714&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1038/nrg2794" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib178"></a><p class="bibliomixed"><span class="bibliomisc">Wynne, D.J., Rog, O., Carlton, P.M., and Dernburg, A.F. (2012). Dynein-dependent processive chromosome motions promote homologous
                        pairing in <span class="italic">C. elegans</span> meiosis. J. Cell Biol. <span class="italic">196</span>, 47&#8211;64.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22232701&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1083/jcb.201106022" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib179"></a><p class="bibliomixed"><span class="bibliomisc">Yang, H., Mains, P.E., and McNally, F.J. (2005). Kinesin-1 mediates translocation of the meiotic spindle to the oocyte cortex
                        through <a href="http://www.wormbase.org/db/get?name=WP:CE40247;class=Protein" target="_blank">KCA-1</a>, a novel cargo adapter. J. Cell Biol. <span class="italic">169</span>, 447&#8211;457.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15883196&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1083/jcb.200411132" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib180"></a><p class="bibliomixed"><span class="bibliomisc">Yang, H., McNally, K., and McNally, F.J. (2003). <a href="http://www.wormbase.org/db/get?name=WP:CE32479;class=Protein" target="_blank">MEI-1</a>/katanin is required for translocation of the meiosis I spindle to the oocyte cortex in <span class="italic">C elegans</span>. Dev. Biol. <span class="italic">260</span>, 245&#8211;59.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12885567&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0012-1606(03)00216-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib181"></a><p class="bibliomixed"><span class="bibliomisc">Yin, Y., and Smolikove, S. (2013). Impaired resection of meiotic double-strand breaks channels repair to nonhomologous end
                        joining in <span class="italic">Caenorhabditis elegans</span>. Mol. Cell. Biol. <span class="italic">33</span>, 2732&#8211;2747.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1128/MCB.00055-13" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib182"></a><p class="bibliomixed"><span class="bibliomisc">Yokoo, R., Zawadzki, K.A., Nabeshima, K., Drake, M., Arur, S., and Villeneuve, A.M. (2012). <a href="http://www.wormbase.org/db/get?name=WP:CE41080;class=Protein" target="_blank">COSA-1</a> reveals robust homeostasis and separable licensing and reinforcement steps governing meiotic crossovers. Cell <span class="italic">149</span>, 75&#8211;87.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2012.01.052" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib183"></a><p class="bibliomixed"><span class="bibliomisc">Youds, J.L., Mets, D.G., McIlwraith, M.J., Martin, J.S., Ward, J.D., ONeil, N.J., Rose, A.M., West, S.C., Meyer, B.J., and
                        Boulton, S.J. (2010). <a href="http://www.wormbase.org/db/get?name=WP:CE15894;class=Protein" target="_blank">RTEL-1</a> enforces meiotic crossover interference and homeostasis. Science <span class="italic">327</span>, 1254&#8211;1258.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1126/science.1183112" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib184"></a><p class="bibliomixed"><span class="bibliomisc">Zalevsky, J., MacQueen, A.J., Duffy, J.B., Kemphues, K.J., and Villeneuve, A.M. (1999). Crossing over during <span class="italic">Caenorhabditis elegans</span> meiosis requires a conserved MutS-based pathway that is partially dispensable in budding yeast. Genetics <span class="italic">153</span>, 1271&#8211;1283.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib185"></a><p class="bibliomixed"><span class="bibliomisc">Zetka, M.C., Kawasaki, I., Strome, S., and Muller, F. (1999). Synapsis and chiasma formation in <span class="italic">Caenorhabditis elegans</span> require <a href="http://www.wormbase.org/db/get?name=WP:CE07625;class=Protein" target="_blank">HIM-3</a>, a meiotic chromosome core component that functions in chromosome segregation. Genes Dev. <span class="italic">13</span>, 2258&#8211;2270.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1101/gad.13.17.2258" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib186"></a><p class="bibliomixed"><span class="bibliomisc">Zetka, M.C., and Rose, A.M. (1990). Sex-related differences in crossing over in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">126</span>, 355&#8211;363.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib187"></a><p class="bibliomixed"><span class="bibliomisc">Zetka, M.C., and Rose, A.M. (1995). Mutant <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004332;class=Gene" target="_blank">rec-1</a></span> eliminates the meiotic pattern of crossing over in <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">141</span>, 1339&#8211;1349.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib188"></a><p class="bibliomixed"><span class="bibliomisc">Zhang, W., Miley, N., Zastrow, M.S., MacQueen, A.J., Sato, A., Nabeshima, K., Martinez-Perez, E., Mlynarczyk-Evans, S., Carlton,
                        P.M., and Villeneuve, A.M. (2012). <a href="http://www.wormbase.org/db/get?name=WP:CE48020;class=Protein" target="_blank">HAL-2</a> promotes homologous pairing during <span class="italic">Caenorhabditis elegans</span> meiosis by antagonizing inhibitory effects of synaptonemal complex precursors. PLoS Genet. <span class="italic">8</span>, e1002880.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24107990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1371/journal.pgen.1002880" target="_blank">Article</a>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e6" href="#d0e6">*</a></sup>Edited by Anne Villeneuve and David Greenstein Last revised May 26, 2015. Published in its final form May 4, 2017. This chapter should be
                  cited as: Hillers K.J., Jantsch V., Martinez-Perez E., Yanowitz J.L. Meiosis. (May 4, 2017), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.178.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2016 Kenneth J Hillers, Verena Jantsch, Enrique Martinez-Perez, Judith L Yanowitz. This is an open-access article distributed
                  under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction
                  in any medium, provided the original author and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e46" href="#d0e46">&sect;</a></sup>To whom correspondence should be addressed. Email: <a href="mailto:khillers@calpoly.edu" target="_top">khillers@calpoly.edu</a>; <a href="mailto:verena.jantsch@univie.ac.at" target="_top">verena.jantsch@univie.ac.at</a>; <a href="mailto:enrique.martinez-perez@imperial.ac.uk" target="_top">enrique.martinez-perez@imperial.ac.uk</a>; <a href="mailto:yanowitzjl@mwri.magee.edu" target="_top">yanowitzjl@mwri.magee.edu</a></p>
 <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>