<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Modern techniques for the analysis of chromatin and nuclear organization in C. elegans</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#wb_askjaer_etal_text_jm" title="Modern techniques for the analysis of chromatin and nuclear organization in C. elegans">
      <link rel="next" href="#sec1" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="chromatinanalysis.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="wb_askjaer_etal_text_jm"></a>Modern techniques for the analysis of chromatin and nuclear organization in <span class="emphasis"><em>C. elegans</em></span><sup><a name="d0e8" href="#ftn.d0e8">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Peter Askjaer<sup>1&sect;</sup>, Sevin&ccedil; Ercan<sup>2</sup><sup>&sect;</sup> and Peter Meister<sup>3</sup><em><span class="remark"><sup><a name="d0e42" href="#ftn.d0e42">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup>Andalusian Center for Developmental Biology (CABD), CSIC/JA/Universidad Pablo de Olavide, Seville, Spain; </span></span><span class="affiliation"><span class="orgname"><sup>2</sup>Department of Biology and the Center for Genomics and Systems Biology, New York University, New York, NY 10003; </span></span><span class="affiliation"><span class="orgname"><sup>3Institute of Cell Biology, University of Bern, Bern, Switzerland</sup>; </span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">
 
	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Cell and molecular biology
		</a></div><hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Introduction</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec1-1">1.1. ChIP or DamID, clues for choice</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1-2">1.2. Cell-type specificity of interactions</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec2">2. Chromatin immunoprecipitation in <span class="emphasis"><em>C. elegans</em></span></a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec2-1">2.1. Introduction to ChIP</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-2">2.2. Experimental planning for ChIP</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-3">2.3. ChIP protocols</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec3">3. DamID in <span class="emphasis"><em>C. elegans</em></span></a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec3-1">3.1. Introduction to DamID</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-2">3.2. Experimental planning for DamID</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-3">3.3. DamID protocols</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec4">4. Methods for ChIP and DamID quantification</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec4-1">4.1. qPCR</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4-2">4.2. DNA microarrays</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4-3">4.3. High-throughput sequencing</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4-4">4.4. Data analysis</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec5">5. Live chromatin imaging</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec5-1">5.1. Introduction</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5-2">5.2. Experimental planning</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5-3">5.3. LacI/lacO protocols</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec6">6. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#sec7">7. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>In recent years, <span class="italic">Caenorhabditis elegans</span> has emerged as a new model to investigate the relationships between nuclear architecture, cellular differentiation, and organismal
                     development. On one hand, <span class="italic">C. elegans</span> with its fixed lineage and transparent body is a great model organism to observe gene functions <span class="italic">in vivo</span> in specific cell types using microscopy. On the other hand, two different techniques have been applied in nematodes to identify
                     binding sites for chromatin-associated proteins genome-wide: chromatin immunoprecipitation (ChIP), and Dam-mediated identification
                     (DamID). We summarize here all three techniques together as they are complementary. We also highlight strengths and differences
                     of the individual approaches.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1-1"></a>1.1.&nbsp;ChIP or DamID, clues for choice
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The major difference between ChIP and DamID is that while ChIP captures DNA associated with a protein at the time of crosslinking,
                  DamID also identifies the DNA that has been bound transiently by the protein. While ChIP is based on the immunoprecipitation
                  of the target by a specific antibody, DamID uses direct modification of the DNA by methylating adenines located close to the
                  binding site. Unlike DamID, ChIP can map epigenetic modifications across the genome. DamID has the advantage of being able
                  to identify binding sites when chromatin complexes are not soluble, very dynamic, or in low abundance.
               </p>
               <p>When choosing between ChIP and DamID, a number of points have to be kept in mind, depending on the target protein, the developmental
                  stage, and the biological question.
               </p>
               <p>First, ChIP requires the availability of an antibody for the protein of interest whereas DamID does not rely on antibodies.
                  Concerns about antibody specificity are therefore irrelevant for DamID (for a more detailed discussion, see <a href="#bib21">Greil et al., 2006</a>). However, posttranslational modifications are not detectable by DamID, which is particularly relevant in studies on histone
                  marks.
               </p>
               <p>Second, while ChIP may need to be optimized for each protein, it is relatively straightforward, once a suitable DamID protocol
                  has been established, to analyze several proteins in parallel without further optimization. For example, in a recent tour
                  de force approach, 112 uncharacterized candidate proteins were screened using DamID, which led to the identification of 42
                  novel chromatin associated factors in <span class="emphasis"><em>Drosophila melanogaster</em></span> (<a href="#bib59">van Bemmel et al., 2013</a>).
               </p>
               <p>Third, ChIP physically purifies the DNA interacting with the target, whereas DamID is an enzymatic modification of DNA that
                  gets erased when DNA is replicated. Hence, in early embryos with fast replicating cells, ChIP may be more suited to detect
                  interactions at that developmental stage. Similarly DamID is not suitable to detect dynamic changes in chromatin association
                  during the cell cycle. On the other hand, DamID may be more sensitive to detect transient interactions (<a href="#bib61">Venkatasubrahmanyam et al., 2007</a>).
               </p>
               <p>Fourth, DamID is an <span class="emphasis"><em>in vivo</em></span> technique where methylation marks are deposited in living cells. On one hand, this avoids potential ChIP artifacts due to
                  chemical crosslinking or other fixatives. On the other hand, DamID requires the introduction of foreign DNA encoding the Dam
                  fusion protein and expression level and localization of the Dam fusion protein need special consideration (see below).
               </p>
               <p>In conclusion, depending on the particular protein of interest, either DamID or ChIP may be the more appropriate method for
                  characterization of binding sites of a given protein. Both ChIP and DamID are potent methods for generating genome-wide information
                  about association of proteins with chromatin. Importantly, comparative studies suggest that the two methods produce similar
                  association profiles, which allow integration of both types of data sets (<a href="#bib21">Moorman et al., 2006</a>; <a href="#bib38">Negre et al., 2006</a>). In <span class="emphasis"><em>C. elegans</em></span>, a significant overlap was also observed between DAF-16 target genes identified by ChIP and DamID (<a href="#bib50">Schuster et al., 2010</a>) and between ChIP and DamID profiles for nuclear envelope proteins (<a href="#bib71">Gonz&aacute;lez-Aguilera et al., 2014</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1-2"></a>1.2.&nbsp;Cell-type specificity of interactions
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Both ChIP and DamID are limited by the fact that they use entire animals, hence a mixture of cell types or even developmental
                  stages, as a starting material. This might improve in the future with cell-type specific DamID or nuclei purification, but
                  current protocols have all been developed using entire animals. Making use of the invariant cell lineage of the nematode,
                  adaptation of the <span class="emphasis"><em>lacO</em></span>/lacI system to visualize binding of transcription factors to specific sequence or localization of genes with single-cell
                  resolution complements ChIP and DamID techniques, exploring cell type specificity of locus localization and nuclear organization.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Chromatin immunoprecipitation in <span class="emphasis"><em>C. elegans</em></span></h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-1"></a>2.1.&nbsp;Introduction to ChIP
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Chromatin immunoprecipitation is a technique that is used to study protein-DNA interactions. Although there are variations
                  in ChIP techniques, the most popular ones rely on the reversible crosslinker formaldehyde (<a href="#bib22">Hecht and Grunstein, 1999</a>; <a href="#bib29">Kuo and Allis, 1999</a>). Briefly, a ChIP experiment is composed of the following steps (<a href="#figure1">Figure 1</a>): (1) worms are incubated with formaldehyde that crosslinks chromatin-associated proteins to each other and to DNA; (2) chromatin
                  is sheared to shorter fragments and the protein of interest is immunoprecipitated along with bound DNA; and (3) bound DNA
                  is identified and quantified.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-2"></a>2.2.&nbsp;Experimental planning for ChIP
                        </h3>
                     </div>
                  </div>
               </div>
               <p>ChIP measures DNA-protein interactions that are captured by formaldehyde crosslinking in a population of cells. Since formaldehyde
                  crosslinks protein to protein and protein to DNA, ChIP measures both direct and indirect interactions with DNA. The success
                  of a ChIP experiment for your protein of interest (hereafter &#8220;target&#8221;) depends on many factors. Most important are (1) the
                  nature of the interaction with DNA, (2) abundance of the target, and (3) affinity and specificity of the antibody. Experimental
                  planning will depend on the unique circumstances of your biological question, but below are some common considerations.
               </p>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="chromatinanalysis_fig1.jpg"><img src="chromatinanalysis_fig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                    </table>
                  <div class="caption">
                     <p><b>Figure 1. Experimental steps for chromatin immunoprecipitation (ChIP) analysis of protein-DNA interactions.&nbsp;</b>Embryos are isolated from gravid adults by bleaching and are cross-linked with formaldehyde. Larvae are prepared by breaking
                        frozen popcorn (drops of worms snap-frozen in liquid nitrogen) by grinding or mixer mill before cross-linking. Chromatin is
                        sheared by sonication to a size between 200-1000 bp, and the target protein is immunoprecipitated along with cross-linked
                        DNA using specific antibodies. In parallel, a portion of the extract is taken for Input DNA. Crosslinks are reversed by heat
                        and DNA is purified. ChIP binding is quantified compared to Input DNA using quantitative PCR, DNA microarrays (ChIP-chip)
                        or high-throughput sequencing (ChIP-seq).
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-2-1"></a>2.2.1.&nbsp;Different types of ChIP targets
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Typically, proteins that bind directly to DNA, such as histones and transcription factors, are easier to ChIP, whereas proteins
                     that indirectly bind to DNA and chromatin-modifying enzymes are more difficult to detect by ChIP. Transcription factors that
                     bind to a short DNA sequence motif show a &#8220;focused&#8221; binding. That is, ChIP enrichment is high at the binding site and declines
                     sharply outside the sonication range (&#8764;200-1000 bp) (<a href="#bib39">Niu et al., 2011</a>). ChIP for histone modifications are generally successful if the antibody is specific and has high affinity to the target
                     (<a href="#bib9">Egelhofer et al., 2011</a>; <a href="#bib31">Liu et al., 2011</a>). Some histone modifications and variants, such as H3K4me3 and H2A.Z, exist in a few nucleosomes that are located at specific
                     regions and show more focused binding (<a href="#bib65">Whittle et al., 2008</a>; <a href="#bib31">Liu et al., 2011</a>) compared to other modifications that associate with the genome &#8220;broadly&#8221;, such as those associated with gene bodies like
                     H3K36 methylation (<a href="#bib28">Kolasinska-Zwierz et al., 2009</a>; <a href="#bib19">Rechtsteiner et al., 2010</a>; <a href="#bib31">Liu et al., 2011</a>;). ChIP enrichment signals for proteins with broad binding patterns are typically lower (<a href="#bib23">Ikegami et al., 2010</a>). Some proteins can have focused, broad, and a combination of both binding patterns, such as RNA Polymerase II and the Dosage
                     Compensation Complex (<a href="#bib11">Ercan et al., 2007</a>; <a href="#bib65">Whittle et al., 2008</a>; <a href="#bib10">Ercan et al., 2009</a>; <a href="#bib24">Jans et al., 2009</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-2-2"></a>2.2.2.&nbsp;Antibody selection and validation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>There are two common approaches to selecting an antibody for ChIP. The first one is to use an antibody against the native
                     protein. The advantage of using an antibody against the native protein is the ability to ChIP the target in its wild-type
                     context. The disadvantage is the difficulty of obtaining specific antibodies or controlling for their specificity. Using two
                     different antibodies to different epitopes in the same target, or analyzing different subunits of the same protein complex
                     helps mitigate specificity problems. If possible, it is best to control for the specificity of the antibody in the context
                     of the ChIP experiment, as summarized in <a href="#sec2-2-4" title="2.2.4.&nbsp;ChIP controls">Section 2.2.4</a>, Controls. In our hands, rabbit polyclonal antibodies have been most successful. If possible, one should test the ability
                     of the antibody to immunoprecipitate its target specifically. This is usually a good indication for if the ChIP will work.
                     Antibody specificity should be tested using assays such as western blot and immunofluorescence and comparing wild type to
                     mutant or RNAi signal (<a href="#bib19">Rechtsteiner et al., 2010</a>; <a href="#bib30">Landt et al., 2012</a>). However, some antibodies that work for ChIP may not work in these assays (<a href="#bib9">Egelhofer et al., 2011</a>), and western blot and immunofluorescence analysis do not guarantee that the antibody will be specific within the ChIP assay.
                     A large number of antibodies recognizing histone modifications were tested by the modENCODE groups and the validation data
                     is available at <a href="http://compbio.med.harvard.edu/antibodies/" target="_top">http://compbio.med.harvard.edu/antibodies/</a>.
                  </p>
                  <p>Another approach for immunoprecipitation is to use an affinity tag. In this approach, a transgenic worm expressing the tagged
                     target is used for ChIP (<a href="#bib39">Niu et al., 2011</a>). GFP-3XFlag tag was successfully used for tagging transcription factors (<a href="#bib39">Niu et al., 2011</a>) and for MES-4 histone methylase (<a href="#bib19">Rechtsteiner et al., 2010</a>). Custom made GFP antibodies were used by modENCODE, but a commercial ChIP-grade rabbit polyclonal GFP antibody is available
                     (Abcam, Ab290). If the Flag is exposed, M2 anti-FLAG (Sigma-Aldrich<sup>&reg;</sup> F3165) can also be used (<a href="#bib19">Rechtsteiner et al., 2010</a>). The advantage of using tagged targets is to bypass requirement for a specific antibody and control for specificity using
                     an untagged strain as a negative control. Moreover, the existing GFP-tagged transgenic strains can be used (<a href="#bib49">Sarov et al., 2012</a>). If a tagged protein is chosen as the target, one should consider issues such as tag interference with binding, and expression
                     at incorrect levels or incorrect tissues.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-2-3"></a>2.2.3.&nbsp;Tissue or developmental-stage selection
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Most ChIP experiments have been performed with whole embryos or larva. If your target is found in a specific tissue, it is
                     best to developmentally synchronize worms to enrich for that tissue. Tissue specific ChIP is desirable, but not yet common.
                     It is possible to perform ChIP using a tagged target expressed in the tissue of interest (<a href="#bib73">Kudron et al., 2013</a>), or by purification of nuclei from the tissue of interest (<a href="#bib54">Steiner et al., 2012</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-2-4"></a>2.2.4.&nbsp;ChIP controls
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Choice of negative control differs according to the antibody. If an antibody recognizing the native protein is used, an ideal
                     negative control is repeating the ChIP in a null mutant in which the target is not present. For essential proteins this is
                     not possible, thus an unspecific antibody such as IgG is commonly used as a negative control. If a tagged target is used for
                     ChIP, ChIP in an untagged strain is the best negative control. Positive controls may be chosen based on several genome-wide
                     ChIP data that are available in literature and through <a href="http://www.modencode.org/" target="_top">http://www.modencode.org/</a>, the modENCODE consortium website (<a href="#bib19">Gerstein et al., 2010</a>). The number of experimental replicates depends on the range of difference that needs to be quantified, but a minimum of
                     three biological replicates is desirable. We find that technical replicates are less important as they are often highly correlated
                     with each other.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-2-5"></a>2.2.5.&nbsp;ChIP quantification
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>There are currently three methods for ChIP quantification, ChIP-qPCR, ChIP-chip and ChIP-seq. If the goal for ChIP quantification
                     is to determine amount of binding at a specific locus (e.g., comparing wild type vs. mutant), ChIP-qPCR can be used. If the
                     goal is to determine genome-wide binding sites, ChIP-chip or ChIP-seq are the methods of choice. In all three techniques ChIP
                     enrichment is compared to Input DNA, therefore analysis of the Input DNA is necessary. We discuss qPCR, microarray and sequencing
                     options for ChIP and DamID in the section following DamID below.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-3"></a>2.3.&nbsp;ChIP protocols
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-3-1"></a>2.3.1.&nbsp;Worm collection
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Approximately 100-200 &micro;l of embryos, and &#8764;1-2 ml of larva (adults) give enough extract for a few ChIPs.</p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2-3-1-1"></a>2.3.1.1.&nbsp;Embryo collection
                              </h5>
                           </div>
                        </div>
                     </div>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>Bleach gravid adults until few or no adult pieces are visible.</p>
                           </li>
                           <li>
                              <p>Wash embryos three times with M9.</p>
                           </li>
                           <li>
                              <p>Resuspend &#8764;500 &#956;l embryos in 45 ml M9.</p>
                           </li>
                           <li>
                              <p>Add formaldehyde to a final concentration of 2% (2.8 ml of 37%).</p>
                           </li>
                           <li>
                              <p>Incubate by rocking at room temperature for 30 minutes.</p>
                           </li>
                           <li>
                              <p>Quench formaldehyde by adding glycine to final 125 mM (2.5 ml of 2.5 M), and incubate 5 min.</p>
                           </li>
                           <li>
                              <p>Centrifuge 3000 g for 2 min. Remove supernatant.</p>
                           </li>
                           <li>
                              <p>Resuspend pellet in M9. Centrifuge 3000 g for 2 min. Remove supernatant.</p>
                           </li>
                           <li>
                              <p>Transfer to microfuge tubes with PBS + protease inhibitors, spin at 6000 g for 1 min, remove supernatant.</p>
                           </li>
                           <li>
                              <p>Wash once more with 1 ml PBS + protease inhibitors, spin at 6000 g for 1 min, remove supernatant and store at &#8722;80&deg;C.</p>
                           </li>
                        </ol>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2-3-1-2"></a>2.3.1.2.&nbsp;Larva collection
                              </h5>
                           </div>
                        </div>
                     </div>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>Clean up larva by settling and washing with &#8764;20X volume of M9 three times.</p>
                           </li>
                           <li>
                              <p>Wash larva with equal volume of PBS + protease inhibitors, spin at 3000 g for 1 min, remove supernatant.</p>
                           </li>
                           <li>
                              <p>Resuspend in equal volume of PBS + protease inhibitors.</p>
                           </li>
                           <li>
                              <p>Using 1 ml pipette tip, drip larva mix into a 100 ml beaker that contains liquid nitrogen. Do not hold the tip on top of the
                                 liquid nitrogen very long, the worms may freeze in the tip. This is to make frozen larva &#8220;popcorn&#8221;.
                              </p>
                           </li>
                           <li>
                              <p>Chill a 15 ml tube in liquid nitrogen, and pour popcorn into the 15 ml tube.</p>
                           </li>
                           <li>
                              <p><a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001003;class=Phenotype" target="_blank">Let</a> liquid nitrogen evaporate, but do not thaw the popcorn at any step until crosslinking. Close cap loosely and store at &#8722;80&deg;C.
                              </p>
                           </li>
                        </ol>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-3-2"></a>2.3.2.&nbsp;Extract preparation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2-3-2-1"></a>2.3.2.1.&nbsp;Embryo extract preparation
                              </h5>
                           </div>
                        </div>
                     </div>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>Resuspend &#8764;100 &#956;l of embryos (volume estimated when embryos are collected) in 500 &#956;l FA buffer + 0.1% sarkosyl* + protease/phosphatase
                                 inhibitors.
                              </p>
                           </li>
                           <li>
                              <p>Spin down at top speed for 10 seconds. Remove supernatant.</p>
                           </li>
                           <li>
                              <p>Resuspend in 1 ml FA buffer + 0.1% sarkosyl + protease/phosphatase inhibitors.</p>
                           </li>
                           <li>
                              <p>Dounce on ice (30 strokes) using a glass dounce homogenizer pestle type B.</p>
                           </li>
                           <li>
                              <p>Transfer to a 15 ml polystyrene tube. Make sure that embryos are well suspended and are at the bottom of the tube. Sonicate
                                 using Bioruptor settings: 4&deg;C, 15 min on High, 30 sec on 30 sec off.
                              </p>
                           </li>
                           <li>
                              <p>Transfer samples to microfuge tubes and spin at top speed (13,000-17,000 g) for 15 min at 4&deg;C. Take the supernatant, and put
                                 aside. Resuspend pellet in 0.5 ml FA buffer + 0.1% sarkosyl + protease/phosphatase inhibitors. Sonicate again.
                              </p>
                           </li>
                           <li>
                              <p>Transfer samples to microfuge tubes and spin at top speed for 15 min at 4&deg;C. Take supernatant, combine with the first supernatant.
                                 Discard pellet.
                              </p>
                           </li>
                           <li>
                              <p>Determine the protein concentration of the supernatant by Bio-Rad assay.</p>
                           </li>
                           <li>
                              <p>Take 25 &#956;l aliquot to determine the extent of DNA shearing (DNA purification below).</p>
                           </li>
                           <li>
                              <p>Continue with the next step or aliquot and snap freeze the ChIP extract in liquid nitrogen and store at &#8722;80&deg;C.</p>
                           </li>
                        </ol>
                     </div>
                     <p>*Sarkosyl: this may reduce efficiency of some mouse monoclonal antibodies, but helps to obtain a tight range sonication in
                        Bioruptor.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2-3-2-2"></a>2.3.2.2.&nbsp;Larva extract preparation
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Caution: Do not let the frozen worm popcorn thaw before crosslinking.</p>
                     <p>Estimate popcorn weight.</p>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>Grind worms into a frozen powder. 2 methods are:</p>
                              <div class="orderedlist">
                                 <ol type="1">
                                    <li>
                                       <p>a) Mortar and pestle</p>
                                       <p>Grind popcorn to a fine powder in cryo-mortar and pestle. This takes &#8764;10 min of vigorous grinding. During grinding, periodically
                                          add more liquid nitrogen to keep the sample frozen. Check under dissecting scope that there are no whole worms left. For adults,
                                          breaking on average of 4 pieces is enough.
                                       </p>
                                    </li>
                                    <li>
                                       <p>b) Mixer Mill</p>
                                       <p>You can use a cryo-mill (e.g., from Retsch) to break the worms. Take &#8764;4 grams (2 mls of settled worms) of frozen worm popcorn,
                                          place in prechilled 50 ml cup and add one 25 mm ball. Submerge in liquid nitrogen to chill. Mix for 15 sec at 25 Hz. Open
                                          the chamber and collect sample using a tapered spatula. Check that most worms are broken into a few pieces. If not, break
                                          for another 15 sec at 25 Hz.
                                       </p>
                                    </li>
                                 </ol>
                              </div>
                           </li>
                           <li>
                              <p>Transfer worm powder to 10 volume of 1.1% formaldehyde in PBS + protease and phosphatase inhibitors (1.1 ml of 37% formaldehyde
                                 + 36 ml of PBS).
                              </p>
                           </li>
                           <li>
                              <p>Incubate at room temperature for 10 min with rocking. Make sure all the worms are dissolved.</p>
                           </li>
                           <li>
                              <p>Quench formaldehyde by adding glycine to a final concentration of 125 mM, and incubating 5 min at room temperature.</p>
                           </li>
                           <li>
                              <p>Centrifuge 4000 g for 3 min at 4&deg;C. Remove supernatant.</p>
                           </li>
                           <li>
                              <p>Resuspend pellet in cold PBS + 1 mM PMSF and spin at 4000 g for 3 min.</p>
                           </li>
                           <li>
                              <p>Resuspend pellet in 1 volume of popcorn weight FA buffer + 0.1% sarkosyl + protease/phosphatase inhibitors.</p>
                           </li>
                           <li>
                              <p>Distribute 1 ml into a 15 ml polystyrene tube. Sonicate and process as in the embryo protocol above.</p>
                           </li>
                        </ol>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2-3-2-3"></a>2.3.2.3.&nbsp;Quantifying extract protein
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Add 1 &micro;l of 0, 0.5, 1, 2, 4, 8 mg/ml BSA to 800 &#956;l of water. Add 1 or 2 &#956;l of extract to 800 &#956;l of water. Add 200 &#956;l Bradford
                        reagent (BioRad #500-0006) and incubate at room temperature 5 min. Transfer to spec cuvettes and measure at 595 nm. Plot standard
                        curve and calculate extract concentration.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2-3-2-4"></a>2.3.2.4.&nbsp;Input DNA purification
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Add 1 &#956;l RNaseA to 25 &#956;l of extract and digest RNA at 37&deg;C 1 h. Add 225 &#956;l of ChIP elution buffer (250 mM NaCl, 1% SDS, 10
                        mM Tris.Cl pH 8, 1 mM EDTA). Add 2 &#956;l of proteinase K and incubate at 50&deg;C for 1-2 h. Incubate overnight at in 65&deg;C water
                        bath to reverse crosslinks. Next day purify DNA using Qiagen PCR purification kit. Run DNA on 1% agarose gel to check shearing.
                        The majority of the sheared chromatin should be between 200-1000 bp in length. We do not recommend re-sonicating frozen extract,
                        thus we recommend optimization of the sonication conditions before the experiment.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-3-3"></a>2.3.3.&nbsp;Chromatin immunoprecipitation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Thaw extract on ice. Aliquot 0.5-2.0 mg of extract from embryos, or 2-4 mg of extract from larvae. The amount is chosen based
                              on target amount and how well the antibody works. For high amount of target protein (e.g., histone modifications) or if you
                              get high background signal, try using less extract. Some recommendations are below:
                           </p>
                           <div class="table"><a name="untable1"></a><p class="title">&#8195;</p>
                              <table summary="&#8195;" border="1">
                                 <colgroup>
                                    <col>
                                    <col>
                                    <col>
                                    <col>
                                 </colgroup>
                                 <thead>
                                    <tr valign="bottom">
                                       <th align="left" valign="bottom">Amount (mg)</th>
                                       <th align="left" valign="bottom">Embryo</th>
                                       <th align="left" valign="bottom">L3</th>
                                       <th align="left" valign="bottom">Adult</th>
                                    </tr>
                                 </thead>
                                 <tbody>
                                    <tr valign="top">
                                       <td align="left" valign="top">0.5</td>
                                       <td align="left" valign="top">Histone mod.</td>
                                       <td align="left" valign="top">Histone mod.</td>
                                       <td valign="top">&nbsp;</td>
                                    </tr>
                                    <tr valign="top">
                                       <td align="left" valign="top">1</td>
                                       <td align="left" valign="top">Histone mod. and Pol II</td>
                                       <td align="left" valign="top">Histone mod. and Pol II</td>
                                       <td class="auto-generated">&nbsp;</td>
                                    </tr>
                                    <tr valign="top">
                                       <td align="left" valign="top">2</td>
                                       <td align="left" valign="top">Transcription Factors</td>
                                       <td align="left" valign="top">Transcription Factors</td>
                                       <td align="left" valign="top">Histone mod. and Pol II</td>
                                    </tr>
                                    <tr valign="top">
                                       <td align="left" valign="top">4</td>
                                       <td valign="top">&nbsp;</td>
                                       <td valign="top">&nbsp;</td>
                                       <td align="left" valign="top">Transcription Factors</td>
                                       <td valign="top">&nbsp;</td>
                                    </tr>
                                 </tbody>
                              </table>
                           </div>
                        </li>
                        <li>
                           <p>Add FA buffer + 0.1% sarkosyl + protease/phosphatase inhibitors to bring the volume up to 440 &#956;l.</p>
                        </li>
                        <li>
                           <p>Centrifuge at 4&deg;C 5 min at top speed. Transfer the supernatant to new tubes.</p>
                        </li>
                        <li>
                           <p>Take 5% of the extract as Input. Place at freezer until next day.</p>
                        </li>
                        <li>
                           <p>Add antibody (1-5 &#956;g) to the rest of the sample and incubate while rotating at 4&deg;C overnight.</p>
                        </li>
                        <li>
                           <p>Take 40 &#956;l of protein A (or G, depending on antibody) coupled to sepharose beads (GE Healthcare) or to magnetic beads (Dynabeads<sup>&reg;</sup>, LifeTechnologies) or custom-affinity beads, per ChIP sample and equilibrate using FA buffer. Add 1 ml FA buffer, invert
                              a few times, spin at 2500 g for 1 min, remove supernatant. Repeat this wash a total of four times. After the washes, resuspend
                              the beads in one bed volume of FA buffer.
                           </p>
                        </li>
                        <li>
                           <p>Add 40 &#956;l beads to each ChIP sample and rotate at 4&deg;C, 2 h.</p>
                        </li>
                        <li>
                           <p>Wash beads at room temperature by adding 1 ml of each of the following buffers and incubating on a rotator. Collect beads
                              by spinning for 1 min at 2500 g. (If magnetic or custom beads are used, collect beads by the appropriate method). Remove supernatant
                              by pipetting.
                           </p>
                           <div class="itemizedlist">
                              <ul type="disc">
                                 <li>
                                    <p>&#8226; 2 times FA buffer for 5 min.</p>
                                 </li>
                                 <li>
                                    <p>&#8226; 1 time FA-1 M NaCl for 5 min.</p>
                                    <p>After this wash, transfer beads to new tubes with the next wash buffer.</p>
                                 </li>
                                 <li>
                                    <p>&#8226; 1 time FA-500 mM NaCl for 10 min.</p>
                                 </li>
                                 <li>
                                    <p>&#8226; 1 time TEL buffer for 10 min.</p>
                                 </li>
                                 <li>
                                    <p>&#8226; 2 times TE for 5 min.</p>
                                 </li>
                              </ul>
                           </div>
                        </li>
                        <li>
                           <p>9.  To elute the immunocomplexes, add 125 &#956;l ChIP Elution Buffer and place the tube in a 65&deg;C heat block for 15 min. Vortex
                              briefly every 5 min. Spin down the beads at 6000 g for 1 min and transfer the supernatant to a new tube. Repeat elution and
                              combine supernatants.
                           </p>
                        </li>
                        <li>
                           <p>10. Add 200 &#956;l Elution Buffer to inputs. Add 2 &#956;l of 10 mg/ml Proteinase K to ChIP and Input samples, and incubate &#8764;1 h at
                              50&deg;C heat block.
                           </p>
                        </li>
                        <li>
                           <p>11. Reverse crosslink overnight at 65&deg;C water bath.</p>
                        </li>
                        <li>
                           <p>12. Purify DNA using Qiagen PCR purification kit. Elute in 50 &#956;l Qiagen Elution buffer. We do not typically quantify ChIP
                              DNA, because we found no correlation between amount of DNA and the ChIP signal/noise ratio. When an antibody produces high
                              background, DNA amount would be high, but the specific enrichment may be low.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>For qPCR we use 2 &#956;l (Real-Time) or 5 &#956;l (Agarose gel) of the 50 &#956;l ChIP DNA. For ChIP-seq library preparation, we use half
                     of the ChIP and 10 ng of Input DNA.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2-3-3-1"></a>2.3.3.1.&nbsp;ChIP reagents
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p><span class="underline">FA buffer:</span></p>
                     <p>50 mM HEPES/KOH pH 7.5, 1 mM EDTA, 1% Triton&#8482; X-100, 0.1% sodium deoxycholate; 150 mM NaCl</p>
                     <p><span class="underline">FA-1 M NaCl buffer:</span></p>
                     <p>50 mM HEPES/KOH pH 7.5, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate; 1 M NaCl</p>
                     <p><span class="underline">FA-500 mM NaCl buffer:</span></p>
                     <p>50 mM HEPES/KOH pH 7.5, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate; 500 mM NaCl</p>
                     <p><span class="underline">TEL buffer:</span></p>
                     <p>0.25 M LiCl, 1% NP-40, 1% sodium deoxycholate, 1 mM EDTA, 10 mM Tris-HCl, pH 8.0</p>
                     <p>Prepare 500 ml of FA and TE, 250 ml of FA-500 mM NaCl, FA-1 M NaCl, and TEL buffers, filter sterilize and store at 4&deg;C.</p>
                     <p><span class="underline">ChIP Elution buffer:</span></p>
                     <p>1% SDS, 250 mM NaCl, 10 mM Tris pH 8.0, 1 mM EDTA</p>
                     <p>Prepare 50 ml and store at room temperature.</p>
                     <p>100X protease inhibitors-Calbiochem<sup>&reg;</sup> (cat. #539131)
                     </p>
                     <p>100X protease inhibitors-Calbiochem (cat. #524624)</p>
                     <p>Keep inhibitors on ice, and add to the FA buffer fresh before use.</p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;DamID in <span class="emphasis"><em>C. elegans</em></span></h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-1"></a>3.1.&nbsp;Introduction to DamID
                        </h3>
                     </div>
                  </div>
               </div>
               <p>DamID was developed by Bas van Steensel and Steven Henikoff as an alternative to ChIP for mapping of protein interaction sites
                  within DNA (<a href="#bib60">van Steensel and Henikoff, 2000</a>). DamID is based on the fusion between DNA adenine methyltransferase (Dam) from <span class="emphasis"><em>Escherichia coli</em></span> and a protein of interest that either binds chromatin directly or indirectly as part of a protein complex. When the fusion
                  protein is expressed <span class="emphasis"><em>in vivo</em></span>, the Dam tag will methylate adenines in the immediate vicinity of the binding sites of the chromatin-associated protein.
                  Dam recognizes GATC motifs and creates a unique adenine methylation mark (G<sup>m</sup>ATC) not normally found in the DNA of eukaryotes. Through a series of enzymatic reactions methylated sites are amplified,
                  and can be identified by array or sequencing techniques (<a href="#figure2">Figure 2</a>). The principle of DamID is simple&#8212;genomic DNA is extracted from animals expressing Dam fused to the protein of interest
                  (assay) and animals expressing a fusion of Dam to GFP (control for DNA accessibility). Methylated sites are cut by a restriction
                  enzyme, <span class="emphasis"><em>Dpn</em></span>I, that recognizes specifically G<sup>m</sup>ATC. <span class="emphasis"><em>Dpn</em></span>I cut sites are then ligated to an adaptor before cutting non-methylated GATC sites with <span class="emphasis"><em>Dpn</em></span>II. PCR using the adaptor as a primer amplifies fragments between G<sup>m</sup>ATC sites. Microarray hybridization or sequencing of amplicons reveals the ratio of methylation between the assay and the
                  control.
               </p>
               <p>Many chromatin-associated proteins, such as transcription factors, recognize short DNA sequences, whereas the nuclear lamina
                  contacts domains of few to hundreds of kilobases. ChIP and DamID experiments typically provide resolution in the range of
                  0.5-2 kilobases (<a href="#bib62">Vogel et al., 2007</a>). Shearing of genomic DNA fragments is the main determinant of resolution in ChIP, whereas density of GATC sequences and
                  spreading of Dam activity to nearby recognition sites set the limit in DamID. <span class="emphasis"><em>C. elegans</em></span> has 269,049 GATC sequences per haploid genome (<a href="#bib51">Sha et al., 2010</a>), corresponding to an average density of one site for every 373 bp.
               </p>
               <p>DamID has proven successful in many species, including budding and fission yeast (<a href="#bib61">Venkatasubrahmanyam et al., 2007</a>; <a href="#bib67">Woolcock et al., 2011</a>), <span class="emphasis"><em>Arabidopsis thaliana</em></span> (<a href="#bib18">Germann et al., 2006</a>), <span class="emphasis"><em>D. melanogaster</em></span> (<a href="#bib60">van Steensel and Henikoff, 2000</a>; <a href="#bib59">van Bemmel et al., 2013</a>), and cultured mouse and human cells (<a href="#bib52">Song et al., 2004</a>; <a href="#bib62">Vogel et al., 2006</a>). The versatility of the method is illustrated by the variety of protein classes for which DNA association sites have been
                  mapped, ranging from transcription factors (<a href="#bib40">Orian et al., 2003</a>; <a href="#bib52">Song et al., 2004</a>), repressor elements (<a href="#bib38">Negre et al., 2006</a>; <a href="#bib62">Vogel et al., 2006</a>), components of the RNAi machinery (<a href="#bib67">Woolcock et al., 2011</a>), histones (<a href="#bib1">Braunschweig et al., 2009</a>), nuclear pore proteins (<a href="#bib26">Kalverda and Fornerod, 2010</a>) to nuclear lamina associated proteins (<a href="#bib42">Pickersgill et al., 2006</a>; <a href="#bib53">Steglich et al., 2012</a>).
               </p>
               <p>In <span class="emphasis"><em>C. elegans</em></span>, DamID has been used to identify target genes for the transcription factor DAF-16 (<a href="#bib50">Schuster et al., 2010</a>), as well as to map chromatin anchored to the nuclear envelope (<a href="#bib56">Towbin et al., 2012</a>; <a href="#bib71">Gonz&aacute;lez-Aguilera et al., 2014</a>). In addition, Fire and coworkers used Dam (without fusing it to a chromatin-associated protein) to probe the structure of
                  the <span class="emphasis"><em>C. elegans</em></span> genome for enzymatic accessibility (<a href="#bib51">Sha et al., 2010</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

<tr>
  <td align="center"><p><a href="chromatinanalysis_fig2.jpg"><img src="chromatinanalysis_fig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Principles of DamID analysis of protein-DNA interactions.&nbsp;</b><span class="emphasis"><em>In vivo</em></span> expression of trace amounts of DNA adenine methyltransferase (Dam) fused to a chromatin-associated protein (&#8220;gene-of-interest&#8221;)
                        generates specific methylation marks (G<sup>m</sup>ATC) normally not found in <span class="emphasis"><em>C. elegans</em></span>. As control for DNA accessibility, a strain expressing GFP-Dam is analyzed in parallel. Genomic DNA (gDNA) is purified and
                        incubated with <span class="emphasis"><em>Dpn</em></span>I restriction enzyme to specifically cleave Dam-methylated G<sup>m</sup>ATC sites. Depending on whether DNA arrays (DamID-chip; left) or high-throughput sequencing (DamID-seq; right) is used to
                        identify methylated DNA, different oligonucleotide adaptors are ligated to the cleaved ends. For DamID-chip both samples are
                        digested with <span class="emphasis"><em>Dpn</em></span>II enzyme to cut all non-methylated GATC sequences (for simplicity only one sample is shown). This generates a pool of DNA
                        fragments, of which only the ones that are flanked by two G<sup>m</sup>ATC sequences without intervening non-methylated sites can serve as PCR templates. Finally, amplified DNA fragments from the
                        two strains are identified and quantified by co-hybridization to DNA arrays. Alternatively, for DamID-seq short tags for sequencing
                        are generated by using adaptors with a recognition site for a restriction enzyme that cuts 19-27 nt from the recognition site
                        (e.g., <span class="emphasis"><em>Mme</em></span>I). After digestion, a second adaptor is ligated to the tags, which are then amplified by PCR. One or several purification
                        steps are included to isolate &#8764;120 bp DNA fragments, which are identified by high-throughput sequencing.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-2"></a>3.2.&nbsp;Experimental planning for DamID
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Because DamID is based on ectopic expression of Dam fused to your protein(-s) of interest, it is important to consider carefully
                  which developmental stage and/or tissue type is most relevant for your purposes. Generally, you want to set up your experiments
                  in cells that normally express your protein.
               </p>
               <p>High expression of Dam causes extensive methylation of the adenines in the genome, which increases the background signal and
                  may have adverse effects on cell metabolism. Therefore, keeping expression to a minimum is necessary to obtain a high signal/noise
                  ratio. A typical approach to do this is to express the Dam-target under the control of a weak promoter, such as an inducible
                  promoter in non-inducing conditions (<a href="#bib21">Greil et al., 2006</a>; <a href="#bib50">Schuster et al., 2010</a>).
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-2-1"></a>3.2.1.&nbsp;Design and expression of DamID fusion proteins
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In principle, any protein can be fused to Dam to test for possible direct or indirect DNA interaction which expands the range
                     of proteins for which the method is relevant (<a href="#bib59">van Bemmel et al., 2013</a>). Importantly, the consequences of truncations or point mutations on the DNA association activities of the test protein can
                     be analyzed by comparing DamID profiles from wild-type and mutant Dam fusions. Although experiments are generally performed
                     using full-length proteins, isolated DNA binding domains can be tested as well (<a href="#bib60">van Steensel and Henikoff, 2000</a>). Because expression of the Dam fusion protein should be maintained very low, both genomic DNA and cDNA sequences can be
                     employed for cloning. As in the original <span class="emphasis"><em>Drosophila</em></span> DamID constructs, Dam fusions in <span class="emphasis"><em>C. elegans</em></span> can be driven by heat shock promoters from which basal transcription produces little but sufficient amounts of Dam fusion
                     protein (<a href="#bib50">Schuster et al., 2010</a>; <a href="#bib56">Towbin et al., 2012</a>). Schuster and coworkers used the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002015;class=Gene" target="_blank">hsp-16.1</a></em></span> promoter, and we used the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002018;class=Gene" target="_blank">hsp-16.41</a></em></span> promoter to control expression of Dam::DAF-16 and Dam::LMN-1, respectively. The basal activities of these two promoters have
                     not been evaluated in different tissues, but both promoters drive expression in most if not all cell types (<a href="#bib55">Stringham et al., 1992</a>; <a href="#bib47">Rohner et al., 2013</a>). The Dam protein can be added to either the N- or the C-terminus of the protein of interest. The size of Dam is equivalent
                     to GFP and if any information exists as to whether your protein tolerates better the addition of a N- or a C-terminal tag,
                     this may guide the design of the Dam fusion (see next section). We have used the classical 3&#8217;UTR from the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006789;class=Gene" target="_blank">unc-54</a></em></span> gene in our DamID constructs but choosing other 3&#8217;UTRs could potentially be used to alter the relative signal contribution
                     by different cell types in whole-animal DamID (<a href="#bib33">Mangone et al., 2010</a>). Finally, to minimize variation in expression levels between different DamID strains, we recommend that transgenic strains
                     be generated by MosSCI (<a href="#bib15">Frokjaer-Jensen et al., 2008</a>) or alternative single-copy integration techniques. This may be particular relevant if you plan to compare several fusion
                     proteins.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-2-2"></a>3.2.2.&nbsp;Validation of DamID fusions
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Recombination during transgenesis may lead to production of a truncated fusion protein. For transgenic strains obtained by
                     MosSCI, single copy insertion of full-length transgenes should ideally be validated by Southern blot or PCR-based methods.
                     Additionally, incorrect processing of the chimeric transcripts may lead to aberrant localization or expression compared to
                     their endogenous counterparts. When using <span class="emphasis"><em>hsp</em></span> promoters in the design of DamID strains, the low transcriptional activity that is used during the DamID experiment should
                     not produce detectable signal in immunofluorescence (IF) or Western blot (WB). This is a good control to ensure that your
                     fusion protein is not overexpressed. Upon heat shock however, clear IF and WB signals should be detected and used to control
                     for correct processing and localization.
                  </p>
                  <p>To address correct expression issues, a Myc epitope is typically included as linker between Dam and the protein of interest,
                     although other detection methods, of course, can be used. IF and WB analysis with anti-Myc antibodies are suitable methods
                     to verify correct fusion protein localization and processing, respectively.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-2-3"></a>3.2.3.&nbsp;Biological material for DamID
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>DamID experiments in <span class="emphasis"><em>C. elegans</em></span> were performed in young embryos (<a href="#bib56">Towbin et al., 2012</a>) or adult hermaphrodites (<a href="#bib50">Schuster et al., 2010</a>; <a href="#bib71">Gonz&aacute;lez-Aguilera et al., 2014</a>), although in principle any developmental stage can be studied. Tissue-specific DamID has not been reported yet, but one
                     can imagine purifying nuclei or cells from a particular tissue (<a href="#bib14">Fox et al., 2007</a>; <a href="#bib54">Steiner et al., 2012</a>; <a href="#bib64">Von Stetina et al., 2007</a>) in animals expressing a Dam fusion throughout their body. To prevent amplification of contaminating bacterial DNA in the
                     DamID protocol, using a <span class="emphasis"><em>dam-E. coli</em></span> strain as food source when growing the nematode cultures is recommended (<a href="#bib51">Sha et al., 2010</a>; <a href="#bib56">Towbin et al., 2012</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-2-4"></a>3.2.4.&nbsp;DamID controls
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Accessibility of Dam to methylate its target GATC sequence depends on chromatin compactness and nucleosome positioning (<a href="#bib60">van Steensel and Henikoff, 2000</a>; <a href="#bib51">Sha et al., 2010</a>). &#8220;Dam only&#8221; controls are therefore included in DamID protocols and used as baseline subtraction for fusions between Dam
                     and the protein(-s) to be analyzed. &#8220;Dam only&#8221; controls should be placed under control of the same promoter and 3&#8217;UTR as the
                     Dam fusion genes, and may consist of Dam alone or Dam fused to a fluorescent protein, such as GFP, allowing easy validation
                     of expression. Alternatively, Dam could be fused to a mutant version of the test protein unable to interact with its target
                     DNA (<a href="#bib32">Luo et al., 2011</a>). To serve as control for DNA accessibility, the &#8220;Dam only&#8221; protein should be nuclear. GFP::Dam (i.e., GFP fused to the N-terminus
                     of Dam) fulfills this criteria (<a href="#bib51">Sha et al., 2010</a>; <a href="#bib71">Gonz&aacute;lez-Aguilera et al., 2014</a>), whereas Dam::GFP does not show nuclear localization (<a href="#bib50">Schuster et al., 2010</a>). Being a relatively new technique in the <span class="emphasis"><em>C. elegans</em></span> field, only three DamID data sets have been published that may serve as positive controls (<a href="#bib50">Schuster et al., 2010</a>; <a href="#bib56">Towbin et al., 2012</a>; <a href="#bib71">Gonz&aacute;lez-Aguilera et al., 2014</a>). However, because identification of DNA association sites is usually done by hybridization to high-content arrays followed
                     by detailed bioinformatics analysis, positive controls are both expensive and laborious and are usually omitted. Instead,
                     DamID experiments should include at least three biological replicates (also for the &#8220;Dam only&#8221; control), and one should observe
                     a high correlation coefficient between the replicates (<a href="#bib50">Schuster et al., 2010</a>; <a href="#bib56">Towbin et al., 2012</a>; <a href="#bib71">Gonz&aacute;lez-Aguilera et al., 2014</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-2-5"></a>3.2.5.&nbsp;DamID evaluation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Similarly to ChIP, DamID-amplified DNA fragments can be quantified by qPCR, DNA arrays, or direct sequencing. qPCR requires
                     prior knowledge about the DNA elements associated with the protein of interest and is mostly suitable when a limited number
                     of DNA fragments are to be analyzed (<a href="#bib61">Venkatasubrahmanyam et al., 2007</a>). In contrast, arrays and sequencing methods provide genome-wide information about binding sites. These methods are discussed
                     in more detail below.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-3"></a>3.3.&nbsp;DamID protocols
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-3-1"></a>3.3.1.&nbsp;Generation of DamID strains
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec3-3-1-1"></a>3.3.1.1.&nbsp;Cloning
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Insert your gene (cDNA or genomic fragment) into a suitable vector containing an inducible promoter with low basal activity,
                        the <span class="emphasis"><em>dam</em></span> open reading frame, and a robust 3&#8217;UTR (e.g., the vector pBN61[<span class="emphasis"><em>unc-119(+)</em></span> P<sub><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002018;class=Gene" target="_blank">hsp-16.41</a></em></span></sub><span class="emphasis"><em>::dam::myc::</em></span>MCS<span class="emphasis"><em>::unc-54</em></span> 3&#8217;UTR] (<a href="#bib71">Gonz&aacute;lez-Aguilera et al., 2014</a>)).
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec3-3-1-2"></a>3.3.1.2.&nbsp;Transgenesis
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>We have successfully used the MosSCI method to insert single copies of Dam fusion transgenes into the <span class="emphasis"><em>C. elegans</em></span> genome (<a href="#bib15">Frokjaer-Jensen et al., 2008</a>; <a href="#bib56">Towbin et al., 2012</a>). Alternatively, strains can be generated by bombardment (<a href="#bib50">Schuster et al., 2010</a>), although variability between strains may be higher.
                     </p>
                     <p>Outcross transgenic lines with wild-type nematodes at least twice prior to validation and DamID.</p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-3-2"></a>3.3.2.&nbsp;Validation of DamID fusion proteins
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>For proteins that are expressed during embryogenesis, this life stage provides the best material for validation of localization
                     (by IF) and size of expressed fusion protein (by WB). For larval and adult nematodes the protocols below may need modification.
                     All steps are at room temperature unless indicated otherwise.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec3-3-2-1"></a>3.3.2.1.&nbsp;Immunofluorescence
                              </h5>
                           </div>
                        </div>
                     </div>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>Heat shock young adults for 1 h at 33&deg;C and leave to recover for 2 h at 20&deg;C. Include transgenic animals that have not been
                                 heat shocked and wild type animals as controls.
                              </p>
                           </li>
                           <li>
                              <p>Transfer 6-8 animals into 3 &#956;l of M9 on a polylysine-coated slide (<a href="#bib6">Crittenden and Kimble, 1999</a>). Use more M9 if cutting the animals cannot be done quickly; otherwise all liquid will evaporate. Using two 25-gauge syringe
                                 needles, cut the animals in the middle, allowing embryos to extrude from the animal. Cover the sample with a 12 x 12 mm coverslip.
                                 Gently remove excess liquid with a paper towel if needed; embryos should be squeezed (but not squashed!) by the coverslip.
                              </p>
                           </li>
                           <li>
                              <p>Put slides on metal blocks on dry ice for 15 min or store at -80&deg;C.</p>
                           </li>
                           <li>
                              <p>Flick off cover slip with a scalpel and submerge slides immediately in -20&deg;C methanol&#8212;leave for 15-18 min.</p>
                           </li>
                           <li>
                              <p>Transfer slides to PBST (PBS with 0.1% Tween<sup>&reg;</sup>-20)&#8212;leave for 15 min.
                              </p>
                           </li>
                           <li>
                              <p>Transfer slides to fresh PBST&#8212;leave for 15 min.</p>
                           </li>
                           <li>
                              <p>Dry slides as much as possible but it is critical that the embryos do not dry out, as this will ruin the staining.</p>
                           </li>
                           <li>
                              <p>Make a circle around embryos with Vaseline<sup>&reg;</sup>.
                              </p>
                           </li>
                           <li>
                              <p>Block slides in 40 &#956;l PBST-F (PBST with 10% Fetal Calf Serum)&#8212;leave for 30 min in a wet chamber (e.g., plastic box with wet
                                 paper towels). Remove solution by aspiration.
                              </p>
                           </li>
                           <li>
                              <p>Dilute &#945;-Myc antibody in PBST-F (e.g., Sigma-Aldrich, cat. #C3956 rabbit &#945;-Myc diluted 1:100) and add 40 &#956;l to each slide.
                                 Ideally, include an additional primary antibody as staining control. If your protein localizes to a specific nuclear structure
                                 or compartment, the control antibody should label this too. Leave for 2 h at room temperature or overnight at 4&deg;C in wet chamber.
                              </p>
                           </li>
                           <li>
                              <p>Wash for 1 min in PBST.</p>
                           </li>
                           <li>
                              <p>Wash for 30 min in PBST.</p>
                           </li>
                           <li>
                              <p>Wash for 30 min in PBST.</p>
                           </li>
                           <li>
                              <p>Dry slides as before. Reapply Vaseline.</p>
                           </li>
                           <li>
                              <p>Add 40 &#956;l secondary antibodies diluted in PBST-F (e.g., Invitrogen&#8482;, cat. #A21071 goat anti-rabbit Alexa Fluor<sup>&reg;</sup> 633&#8211;conjugated antibody diluted 1:1000 plus a suitable fluorescently labeled antibody to visualize your control primary antibody).
                                 Leave in darkness for 2 h at room temperature or overnight at 4&deg;C in wet chamber.
                              </p>
                           </li>
                           <li>
                              <p>Wash for 1 min in PBST.</p>
                           </li>
                           <li>
                              <p>Wash for 30 min in PBST.</p>
                           </li>
                           <li>
                              <p>Wash for 30 min in PBST.</p>
                           </li>
                           <li>
                              <p>Rinse quickly with PBS.</p>
                           </li>
                           <li>
                              <p>Dry slides as before. Remove Vaseline.</p>
                           </li>
                           <li>
                              <p>Mount with Mowiol<sup>&reg;</sup> anti-fading mounting medium containing 5 &#956;g/ml Hoechst 33258 (e.g., 30 &#956;l/24 x 24 mm coverslip).
                              </p>
                           </li>
                        </ol>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec3-3-2-2"></a>3.3.2.2.&nbsp;Western blot
                              </h5>
                           </div>
                        </div>
                     </div>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>1.Heat shock two 60 mm NGM plates of synchronized young adults for 1 h at 33&deg;C and leave to recover for 2 h at 20&deg;C. Include
                                 transgenic animals that have not been heat shocked and wild type animals as controls.
                              </p>
                           </li>
                           <li>
                              <p>2.Harvest embryos by standard hypochlorite treatment (1 N NaOH, 30% bleach solution). Dilute a small aliquot (e.g., 1 &#956;l)
                                 to determine number of embryos in each sample and adjust to similar concentration if needed. Approximately 2000 embryos should
                                 be loaded per lane.
                              </p>
                           </li>
                           <li>
                              <p>3.Disrupt embryos by boiling and vortexing in SDS sample buffer together with 0.5 &#956;m-diameter glass beads for 5 min.</p>
                           </li>
                           <li>
                              <p>4.Separate by SDS&#8211;PAGE and transfer to appropriate membrane according to standard protocols.</p>
                           </li>
                           <li>
                              <p>5.Block membrane (e.g., with PBS containing 0.05% Tween-20 and 3% low-fat milk for 30 min).</p>
                           </li>
                           <li>
                              <p>6.Incubate for 2 h with &#945;-Myc primary antibody (e.g., Sigma-Aldrich, cat. #C3956 diluted 1:400).</p>
                           </li>
                           <li>
                              <p>7.Wash membrane with PBS containing 0.05% Tween-20.</p>
                           </li>
                           <li>
                              <p>8.Incubate for 2 h with appropriate secondary antibody (e.g., Sigma-Aldrich cat. #A0545 peroxidase-conjugated anti-rabbit
                                 antibody diluted 1:5000).
                              </p>
                           </li>
                           <li>
                              <p>9.Wash membrane and develop according to manufacture's instructions.</p>
                           </li>
                        </ol>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-3-3"></a>3.3.3.&nbsp;Nematode culture
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>DamID experiments should be performed on synchronized cultures. We prefer to scale up the culture and use liquid medium but
                     nematodes could also be obtained from NGM plates. Approximately 30 mg clean embryo or adult nematode pellet is sufficient
                     for each sample.
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Collect embryos from asynchronous cultures by standard hypochlorite treatment (1 N NaOH, 30% bleach solution).</p>
                        </li>
                        <li>
                           <p>Wash embryos at least three times in S-medium.</p>
                        </li>
                        <li>
                           <p>Leave embryos to hatch overnight at 20&deg;C in S-medium with 5 &#956;g/ml cholesterol and 0.01% Tween-20 in the absence of bacteria.</p>
                        </li>
                        <li>
                           <p>Assay quality and quantity of hatched L1s.</p>
                        </li>
                        <li>
                           <p>Start three independent 50 ml S-medium cultures for each strain with approximately 35,000 L1 larvae. Cultures should contain
                              <span class="emphasis"><em>Dam- E. coli</em></span> (e.g., strain GM119 or SCS110) harvested from 100 ml bacterial culture as food source.
                           </p>
                        </li>
                        <li>
                           <p>Grow cultures with continuous agitation (180 rpm) at 20&deg;C.</p>
                        </li>
                        <li>
                           <p>Monitor cultures and harvest when they are enriched for the life stage to be analyzed (e.g., 53 h for non-gravid young adults;
                              66 h for accumulation of young embryos, etc.).
                           </p>
                           <div class="orderedlist">
                              <ol type="1">
                                 <li>
                                    <p>7.1. Harvest embryos by hypochlorite treatment and wash three times with M9.</p>
                                 </li>
                                 <li>
                                    <p>7.2. Harvest adults by centrifugation and wash 5-7 times with M9 to remove bacteria.</p>
                                 </li>
                              </ol>
                           </div>
                        </li>
                        <li>
                           <p>Make aliquots containing &#8764;30 &#956;l embryo or adult material. Remove excess liquid and snap-freeze in liquid nitrogen before storage
                              at -80&deg;C until further processing.
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-3-4"></a>3.3.4.&nbsp;Purification and amplification of Dam-methylated DNA
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In our experience, the frequency of Dam methylation is significantly lower in embryos compared to adults (C. Gonz&aacute;lez-Aguilera
                     and P. Askjaer, unpublished). We speculate that this is due to the rapid cell divisions in the early embryos, which continuously
                     erase the Dam mark. (DNA replication converts the Dam mark into two hemi-methylated sequences, which are cut much less efficiently
                     by <span class="emphasis"><em>Dpn</em></span>I). Potentially, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002018;class=Gene" target="_blank">hsp-16.41</a></em></span> promoter may also have a higher basal activity in adults versus embryos. This difference in methylation frequency implies
                     that the protocol differs slightly depending on the life stage to be analyzed.
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>1.Purify genomic DNA (gDNA) from &#8764;30 mg embryos or adults, e.g., with Qiagen DNeasy kit including RNase A (cat. #69504 and
                              #19101). Avoid using too much material, as this will reduce the quality of the DNA prep.
                           </p>
                        </li>
                        <li>
                           <p>2.Digest 2.5 &#956;g of adult gDNA with 10 units <span class="emphasis"><em>Dpn</em></span>I (New England Biolabs<sup>&reg;</sup> Inc., cat. #R0176S; cuts G<sup>m</sup>ATC) in 10 &#956;l overnight at 37&deg;C in a thermocycler. Scale up the reaction 2-fold for embryonic gDNA. Include an additional
                              reaction with <span class="emphasis"><em>Dpn</em></span>I for one of the biological samples to be used as ligation control (control A) and a control reaction without <span class="emphasis"><em>Dpn</em></span>I (control B).
                           </p>
                        </li>
                        <li>
                           <p>3.Inactivate <span class="emphasis"><em>Dpn</em></span>I by incubation for 20 min at 80&deg;C.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>Steps 4-13 are relevant for identification of methylation sites by DNA arrays. For high-throughput sequencing, jump to step
                     14.
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>4.Prepare double-stranded adaptors by mixing 50 &#956;l of primers AdRt (5&#8217; CTAATACGACTCACTATAGGGCAGCGTGGTCGCGGCCGAGGA; 100 &#956;M)
                              and 50 &#956;L of AdRb (5&#8217; TCCTCGGCCG; 100 &#956;M). Heat to 95&deg;C and let cool down slowly to room temperature.
                           </p>
                        </li>
                        <li>
                           <p>5.For embryos: Split <span class="emphasis"><em>Dpn</em></span>I digested DNA in two tubes, each with 10 &#956;l.
                           </p>
                        </li>
                        <li>
                           <p>6.Mix 10 &#956;l <span class="emphasis"><em>Dpn</em></span>I digested DNA, 0.8 &#956;l of the annealed adaptors (50 &#956;M), 2 &#956;l of 10x ligation buffer, and 5 units T4 DNA ligase (Roche Applied
                              Science, cat. #10799009001, 5 U/&#956;l), in a volume of 20 &#956;l and incubate at 16&deg;C in a thermocycler for 2 h or overnight. Include
                              also a reaction with <span class="emphasis"><em>Dpn</em></span>I digested DNA but omitting the T4 DNA ligase (control A).
                           </p>
                        </li>
                        <li>
                           <p>7.Inactivate the ligase by incubation for 10 min at 65&deg;C.</p>
                        </li>
                        <li>
                           <p>8.Digest non-methylated GATC sites by incubating the ligation reactions with 10 units <span class="emphasis"><em>Dpn</em></span>II (New England Biolabs Inc., cat. #R0543S) in a volume of 50 &#956;l for 1 h at 37&deg;C. For embryos: Pool the two reactions in a
                              single tube with 100 &#956;l.
                           </p>
                        </li>
                        <li>
                           <p>9.Amplify methylated DNA, using for instance Advantage<sup>&reg;</sup> cDNA Polymerase Mix (Clontech Laboratories, Inc., cat. #639105). Perform also a control reaction without DNA template (control
                              C).
                           </p>
                           <div class="orderedlist">
                              <ol type="1">
                                 <li>
                                    <p>1)28.75 &#956;l ddH<sub>2</sub>O
                                    </p>
                                 </li>
                                 <li>
                                    <p>2)5 &#956;l 10x cDNA PCR reaction buffer (Clontech)</p>
                                 </li>
                                 <li>
                                    <p>3)10 &#956;l of <span class="emphasis"><em>Dpn</em></span>II digested DNA
                                    </p>
                                 </li>
                                 <li>
                                    <p>4)1.25 &#956;l of primer Adr-PCR (5&#8217; GGTCGCGGCCGAGGATC; 50 &#956;M)</p>
                                 </li>
                                 <li>
                                    <p>5)4 &#956;l dNTP mix (Takara Bio Inc., cat. #4030, each 2.5 mM)</p>
                                 </li>
                                 <li>
                                    <p>6)1 &#956;l Advantage cDNA Polymerase Mix</p>
                                    <p>PCR parameters</p>
                                    <div class="orderedlist">
                  <table border="0" width="100%">
                        <colgroup>
                           <col>
                           <col>
                        </colgroup>
                        <tbody>
                           <tr valign="bottom">
                              <td align="left" valign="bottom">
                                       <ol type="1">
                                          <li>
                                             <p>68&deg;C 10 min</p>
                                          </li>
                                          <li>
                                             <p>94&deg;C 1 min</p>
                                          </li>
                                          <li>
                                             <p>65&deg;C 5 min</p>
                                          </li>
                                          <li>
                                             <p>68&deg;C 15 min</p>
                                          </li>
                                          <li>
                                             <p>94&deg;C 1 min</p>
                                          </li>
                                          <li>
                                             <p>65&deg;C 1 min</p>
                                          </li>
                                           </ol>
</td>
                              <td align="left" valign="bottom">
                                       <ol type="1" start="7">
<li>
                                             <p>68&deg;C 10 min</p>
                                          </li>
                                          <li>
                                             <p>Go to step 5 3x</p>
                                          </li>
                                          <li>
                                             <p>94&deg;C 1 min</p>
                                          </li>
                                          <li>
                                             <p>65&deg;C 1 min</p>
                                          </li>
                                          <li>
                                             <p>68&deg;C 2 min</p>
                                          </li>
                                          <li>
                                             <p>Go to step 9 23x (embryos) or 20x (adults).</p>
                                          </li>
                                                                                 </ol>
</td>
                           </tr>
                        </tbody>
                     </table>
                                    </div>
                                 </li>
                              </ol>
                           </div>
                        </li>
                        <li>
                           <p>Check 5 &#956;l of PCR reaction on an agarose gel. Successful amplification of Dam-methylated DNA should produce a smear from 200
                              to 1500 bp. Appearance of discrete bands suggests the presence of a PCR contamination. If a strong smear appears in the control
                              reactions A, B and C, try to reduce the number of cycles in step 12; control lanes do not necessarily need to be blank, but
                              should be fainter than the other lanes.
                           </p>
                        </li>
                        <li>
                           <p>To achieve sufficient material, 6-7 (embryos) or 2 (adults) PCR reactions from the same <span class="emphasis"><em>Dpn</em></span>II digestion are pooled and purified with Qiaquick PCR purification kit (Qiagen).
                           </p>
                        </li>
                        <li>
                           <p>Quantify purified DNA. Typically, 1 &#956;g of DNA is needed for labeling and hybridization, but additional quantities may be required
                              for quality checks, depending on the genomics facility.
                           </p>
                        </li>
                        <li>
                           <p>Label and hybridize to whole genome-tiling array according to the manufacture's instructions. Remember to include a dye-swap
                              for one of the triplicates.
                           </p>
                        </li>
                        <li>
                           <p>For high-throughput sequencing, a modified linker containing a recognition site for enzymes that cut DNA 20-25 bp after the
                              recognition site are ligated to the <span class="emphasis"><em>Dpn</em></span>I-digested ends (e.g., <span class="emphasis"><em>EcoP15</em></span>I or <span class="emphasis"><em>Mme</em></span>I). Different strategies have been developed and the reader should consult the original papers (<a href="#bib32">Luo et al., 2011</a>; <a href="#bib51">Sha et al., 2010</a>). Please note that neither of the two papers identified DNA binding sites for <span class="emphasis"><em>C. elegans</em></span> chromatin-associated proteins, hence, further optimization may be required.
                           </p>
                        </li>
                        <li>
                           <p>Digest with <span class="emphasis"><em>EcoP15</em></span>I or <span class="emphasis"><em>Mme</em></span>I to generate short Dam tags.
                           </p>
                        </li>
                        <li>
                           <p>Purify digested DNA. If the linker used in step 14 is biotin-labeled, purification can be performed with magnetic streptavidin
                              beads.
                           </p>
                        </li>
                        <li>
                           <p>Ligate second linker.</p>
                        </li>
                        <li>
                           <p>Optional: purify correctly ligated DNA based on size by denaturing polyacrylamide gel electrophoresis.</p>
                        </li>
                        <li>
                           <p>PCR amplify Dam fragments using linker-specific primers.</p>
                        </li>
                        <li>
                           <p>Gel-purify DNA of correct size (116-125 bp).</p>
                        </li>
                        <li>
                           <p>Submit fragments for Solexa sequencing.</p>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Methods for ChIP and DamID quantification
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-1"></a>4.1.&nbsp;qPCR
                        </h3>
                     </div>
                  </div>
               </div>
               <p>ChIP or DamID followed by qPCR is a viable option when one knows or has a good guess as to where the protein binds. Real-time
                  PCR is used to quantify the relative amount of ChIP DNA compared to input DNA at the test and control loci. Selecting the
                  test region for qPCR depends on the target. For example, in the case of a DNA-binding transcription factor, one can choose
                  primers encompassing a region of the promoter with a putative DNA sequence motif. Genome wide ChIP data is available for a
                  number of transcription factors (<a href="http://www.modencode.org/" target="_top">http://www.modencode.org/</a>). The negative control locus should be at a region with no expected binding. We recommend expressing ChIP enrichment as percentage
                  of the Input. Some researchers use fold enrichment compared to IgG control, or the negative control loci. In this case, one
                  needs to make sure that the background signal does not vary too much from experiment to experiment.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-2"></a>4.2.&nbsp;DNA microarrays
                        </h3>
                     </div>
                  </div>
               </div>
               <p>For identification and quantification of DNA fragments by array technology, samples from the Dam fusion strain and the &#8220;Dam
                  only&#8221; control strain are processed in parallel and fluorescently labeled with different dyes, followed by co-hybridization
                  on high-density arrays. Similarly for ChIP-chip, ChIP and input DNA are amplified and labeled by two different fluorescent
                  dyes (Cy3 and Cy5) and co-hybridized to microarrays that encompass the whole genome (<a href="#bib2">Buck and Lieb, 2004</a>; <a href="#bib11">Ercan et al., 2007</a>). To avoid potential bias introduced by the two dyes, a dye-swap experiment should be performed with at least one of the
                  biological replicates. Analysis of hybridization data, including normalization and smoothening can be performed with various
                  algorithms. Briefly, fluorescence signal ratio of ChIP to Input or &#8220;Dam fusion&#8221; to &#8220;Dam only&#8221; is a measure of ChIP or DamID
                  enrichment per probe, and due to normalization, enrichment at each locus is relative to the rest of the genome. Further processing
                  of data will typically depend on the class of chromatin-association being investigated. Whole-Genome Tiling Arrays from Roche
                  NimbleGen containing 2.1 million 50 nt probes have been used in both ChIP-chip and DamID experiments, but are no longer produced.
                  Equivalent arrays from other manufactures have not been tested in <span class="emphasis"><em>C. elegans</em></span> DamID, but have been employed in DamID experiments in other organisms as well as in <span class="emphasis"><em>C. elegans</em></span> ChIP studies.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-3"></a>4.3.&nbsp;High-throughput sequencing
                        </h3>
                     </div>
                  </div>
               </div>
               <p>DamID and ChIP DNA can be directly sequenced by high-throughput sequencing. ChIP-seq provides higher resolution and greater
                  dynamic range compared to ChIP-chip, and is currently cheaper since sequencing can be multiplexed, and commercial microarrays
                  are being phased out (<a href="#bib41">Park, 2009</a>; <a href="#bib30">Landt et al., 2012</a>). Multiplexing is done by including a unique identifying index to each sequencing library and pooling libraries for sequencing,
                  for example, in one lane of an Illumina<sup>&reg;</sup> flowcell. For ChIP-seq, having greater numbers of short reads is more important than having long reads. Therefore, single-end
                  sequencing of 36-50 bp by Illumina is a popular method of choice. Transcription factors with more focused binding and fewer
                  binding sites typically require fewer reads, while targets with broad patterns, such as histone modifications, require more
                  reads. In our experience, 10-20 million mapped 50 bp reads are usually enough for a large range of targets in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib72">Kranz et al. 2013</a>). Typically 80-90% of reads from ChIP map to the genome. One lane of Illumina HiSeq2500 typically provides &#8764;150 million mapped
                  reads, thus one can multiplex &#8764;8 samples in a lane. ChIP-seq enrichment score is calculated for each base pair across the
                  genome as read coverage of ChIP compared to Input, and thus reflects relative binding with respect to the whole genome. To
                  accurately determine protein-binding sites by peak calling, it is important to sequence matching Input DNA preferably at same
                  or higher coverage (<a href="#bib4">Chen et al., 2012</a>).
               </p>
               <p>Although most published DamID experiments used DNA microarrays, recent studies have mapped Dam methylation sites with Illumina
                  sequencing (<a href="#bib51">Sha et al., 2010</a>; <a href="#bib32">Luo et al., 2011</a>). These two studies differ in several details, but both included an enzymatic trimming of the methylated fragments to provide
                  20-27 bp DamID sequence tags that were identified by massive sequencing. Approximately 38% of raw sequence reads could be
                  mapped to unique GATC sites in the <span class="emphasis"><em>C. elegans</em></span> genome (<a href="#bib51">Sha et al., 2010</a>), which implies that 10-20 million reads will be appropriate for each sample, thus similar to the situation in ChIP-seq.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-4"></a>4.4.&nbsp;Data analysis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Because data analysis will depend very much on several parameters, such as type of protein being analyzed (transcription factor,
                  histone, nuclear envelope protein, and etc.), methods for data generation (array or sequencing), and extent of bioinformatics
                  assistance available, it is not realistic to give detailed protocols on this aspect here. Please refer to published studies
                  or contact us for further discussion.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;Live chromatin imaging
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-1"></a>5.1.&nbsp;Introduction
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The modENCODE consortium as well as the vast majority of published ChIP and DamID studies use either entire worms or developing
                  embryos. The starting material is therefore a mixture of cell types and/or developmental stages, which hinders cell-type specific
                  nuclear organization studies. One way to overcome this limitation is to make use of the <span class="emphasis"><em>C. elegans</em></span> fixed lineage to carry out imaging-based analysis. Such techniques are low throughput compared to microarray- or sequencing-based
                  studies, and only a limited number of loci/sequences can be studied. Imaging and ChIP/DamID are therefore complementary techniques.
                  Fluorescence <span class="emphasis"><em>in situ</em></span> hybridization is difficult to achieve in <span class="emphasis"><em>C. elegans,</em></span> in particular in differentiated larvae or adults.
               </p>
               <p>All techniques to tag a locus <span class="emphasis"><em>in vivo</em></span> in its chromatin context rely on the recognition of a bacterial- or phage-binding site by its cognate repressor. A number
                  of repressor/binding site couples have been used in a variety of organisms (lacI/<span class="emphasis"><em>lacO;</em></span> tetR/<span class="emphasis"><em>tetO</em></span>; &#955; repressor/operators). In <span class="emphasis"><em>C. elegans,</em></span> the only system used up to now is the <span class="emphasis"><em>lacO</em></span>/lacI couple (<a href="#bib3">Carmi et al., 1998</a>; <a href="#bib25">Kaltenbach et al., 2000</a>; <a href="#bib20">Gonzalez-Serricchio and Sternberg, 2006</a>; <a href="#bib69">Yuzyuk et al., 2009</a>; <a href="#bib36">Meister et al., 2010b</a>; <a href="#bib68">Yuen et al., 2011</a>; <a href="#bib56">Towbin et al., 2012</a>; <a href="#bib5">Cochella and Hobert, 2012</a>; <a href="#bib47">Rohner et al., 2013</a>). The system has been used to answer a variety of biological questions:
               </p>
               <div class="itemizedlist">
                  <ul type="disc">
                     <li>
                        <p>- analyze <span class="emphasis"><em>in vivo</em></span> the interaction of a given protein (e.g., a transcription factor) with its cognate binding site, in a cell-type and developmental
                           stage specific manner (<a href="#bib3">Carmi et al., 1998</a>; <a href="#bib25">Kaltenbach et al., 2000</a>; <a href="#bib58">Updike and Mango, 2006</a>; <a href="#bib27">Kiefer et al., 2007</a>; <a href="#bib69">Yuzyuk et al., 2009</a>).
                        </p>
                     </li>
                     <li>
                        <p>- functionally dissect promoters for transcription factor binding and/or subnuclear positioning (<a href="#bib47">Rohner et al., 2013</a>; <a href="#bib58">Updike and Mango, 2006</a>; <a href="#bib69">Yuzyuk et al., 2009</a>).
                        </p>
                     </li>
                     <li>
                        <p>- analyze mosaic worms using <span class="emphasis"><em>lacO</em></span>/lacI as a tool to identify cells carrying an array (<a href="#bib20">Gonzalez-Serricchio and Sternberg, 2006</a>).
                        </p>
                     </li>
                     <li>
                        <p>- determine cell ploidy by counting the number of <span class="emphasis"><em>lacO</em></span>/lacI spots for a chromosomally integrated array (<a href="#bib20">Gonzalez-Serricchio and Sternberg, 2006</a>).
                        </p>
                     </li>
                     <li>
                        <p>- quantify genome compaction using complex arrays in which binding sites are interspersed with exogenous DNA (<a href="#bib69">Yuzyuk et al., 2009</a>).
                        </p>
                     </li>
                     <li>
                        <p>- analyze <span class="emphasis"><em>de novo</em></span> centromere formation (<a href="#bib68">Yuen et al., 2011</a>).
                        </p>
                     </li>
                     <li>
                        <p>- highlight changes in the organization of the nucleus during development (<a href="#bib36">Meister et al., 2010b</a>).
                        </p>
                     </li>
                     <li>
                        <p>- uncover sequences and signals involved in genome nuclear organization (<a href="#bib36">Meister et al., 2010b</a>; <a href="#bib47">Rohner et al., 2013</a>; <a href="#bib56">Towbin et al., 2012</a>)
                        </p>
                     </li>
                  </ul>
               </div>
               <p>Depending on the biological question, the individual setup of the system is quite different. Here we present first a review
                  of the different published systems, and then recently developed methods and reagents. In the last part, we discuss image acquisition
                  and analysis techniques.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-2"></a>5.2.&nbsp;Experimental planning
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-2-1"></a>5.2.1.&nbsp;LacI/<span class="emphasis"><em>lacO</em></span> systems for monitoring chromatin <span class="emphasis"><em>in vivo</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Two major systems have been used to visualize sequences <span class="emphasis"><em>in vivo</em></span> using lacI/<span class="emphasis"><em>lacO</em></span>. In the first system, hereafter described as &#8220;<span class="emphasis"><em>cis</em></span> tagging&#8221;, a single transgenic array encodes fluorescently tagged lacI and carries <span class="emphasis"><em>lacO</em></span> sites as well as the sequences of interest. Fluorescent lacI therefore binds to the array which encodes it. In the second
                     type, &#8220;<span class="emphasis"><em>trans</em></span> tagging&#8221;, two different transgenes are used. One transgene provides expression of fluorescent lacI but does not contain <span class="emphasis"><em>lacO</em></span> sites, hence is not bound by fluorescent lacI and remains microscopically invisible. A second transgene carrying <span class="emphasis"><em>lacO</em></span> sites as well as the sequences of interest is observed in this case. Both systems require the expression of fluorescently
                     tagged lacI.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-2-2"></a>5.2.2.&nbsp;lacI expression in <span class="emphasis"><em>C. elegans</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>The Lac repressor from the <span class="emphasis"><em>E. coli</em></span> lac operon is expressed as a fusion with a green or cyan fluorescent protein (GFP, CFP), and targeted to the nucleus by an
                     NLS. There are a number of lacI variants currently used for tagging, and these variants differ between organisms for historical
                     reasons. The most widely used in <span class="emphasis"><em>C. elegans</em></span> is lacI<sup>q</sup><span class="emphasis"><em>,</em></span> a lacI version mutated in its C-terminal tetramerization domain that provides high affinity to the <span class="emphasis"><em>lacO</em></span> repeats. In budding yeast, binding of lacI to <span class="emphasis"><em>lacO</em></span> repeats was shown to lead to replication fork passage problems, which in turn can modify gene localization (<a href="#bib7">Dubarry et al., 2011</a>). This can be avoided by using lower affinity versions of lacI (lacI**, <a href="#bib7">Dubarry et al., 2011</a>). In <span class="emphasis"><em>C. elegans,</em></span> no difference in gene positioning has been observed between lacI<sup>q</sup> and lower affinity lacI versions with respect to <span class="emphasis"><em>lacO</em></span> repeats for larger arrays and for bombardment-mediated transgenes (P. Meister, unpublished observations). Expression levels
                     of the GFP-lacI need to be low, as overexpression elevates background fluorescence, enhances non-specific binding, and can
                     cause slow growth or sick animals.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-2-3"></a>5.2.3.&nbsp;<span class="emphasis"><em>Cis</em></span> tagging&#8212;insertion of <span class="emphasis"><em>lacO</em></span>, lacI-expressing constructs and sequences of interest on the same transgenic array
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>Cis</em></span> systems are obtained by microinjection and typically stay extrachromosomal. They are created new for each sequence to be
                     tested. For the expression of lacI, different promoters active in all cells have been used (<a href="#table1" title="Table 1. Cis tagging systems&#8212;lacI expressing constructs and lacO sites present on the same transgene.">Table 1</a>), either inducible (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002016;class=Gene" target="_blank">hsp-16.2</a></em></span>p, <a href="#bib3">Carmi et al., 1998</a>; <a href="#bib20">Gonzalez-Serricchio and Sternberg, 2006</a>) or constitutively expressed (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001088;class=Gene" target="_blank">dpy-30</a></em></span>p, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001898;class=Gene" target="_blank">his-24</a></em></span>p, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019947;class=Gene" target="_blank">htz-1</a></em></span>p, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00000235;class=Gene" target="_blank">baf-1</a></em></span>p, <a href="#bib3">Carmi et al., 1998</a>; <a href="#bib27">Kiefer et al., 2007</a>; <a href="#bib69">Yuzyuk et al., 2009</a>; <a href="#bib13">Fakhouri et al., 2010</a>; <a href="#bib36">Meister et al., 2010b</a>). As for other injected arrays, different levels of complexity can be achieved by adjusting the composition of injected DNA.
                     Simple arrays, composed exclusively of plasmids are highly repetitive and contain up to 300 copies of each of the plasmids
                     used to create them (<a href="#bib36">Meister et al., 2010b</a>). In this case, the single <span class="emphasis"><em>lacO</em></span> site present in each plasmid is sufficient to form of a visible spot, as the total number of plasmids is high. Complexity
                     of the array can be increased by adding, for instance, sonicated fish (salmon or herring) DNA (from 40 to 90% of the total
                     DNA), but in this case a plasmid with <span class="emphasis"><em>lacO</em></span> repeats has to be added to the injection mix (<a href="#bib27">Kiefer et al., 2007</a>; <a href="#bib69">Yuzyuk et al., 2009</a>; <a href="#bib13">Fakhouri et al., 2010</a>).
                  </p>
                  <div class="table"><a name="table1"></a><p class="title">Table 1. <span class="emphasis"><em>Cis</em></span> tagging systems&#8212;lacI expressing constructs and <span class="emphasis"><em>lacO</em></span> sites present on the same transgene.
                     </p>
                     <table summary="Table 1. Cis tagging systems&#8212;lacI expressing constructs and lacO sites present on the same transgene." border="1">
                        <colgroup>
                           <col>
                           <col>
                           <col>
                           <col>
                        </colgroup>
                        <thead>
                           <tr valign="bottom">
                              <th align="left" valign="bottom"><span class="bold"><strong>Laboratory and Reference</strong></span></th>
                              <th align="left" valign="bottom"><span class="bold"><strong>Biological question</strong></span></th>
                              <th align="left" valign="bottom"><span class="bold"><strong>Array composition (misture of plasmids</strong></span></th>
                              <th align="left" valign="bottom">lacI expression system/<span class="emphasis"><em>lacO</em></span> composition*
                              </th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr valign="top">
                              <td align="left" valign="top">
                                 <p>TY</p>
                                 
                                 <p><a href="#bib3">Carmi et al., 1998</a></p>
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>In vivo</em></span> binding of SEX-1 to <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006962;class=Gene" target="_blank">xol-1</a></em></span> promoter
                              </td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO/lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00001079;class=Gene" target="_blank">dpy-20</a></em></span> marker + or &#8211; promoter
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>hsp-16.2p::gfp-lacI</em></span> (induced)
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">
                                 <p>PS</p>
                                 
                                 <p><a href="#bib20">Gonzalez-Serrichio et al., 2006</a></p>
                              </td>
                              <td align="left" valign="top">Array tracking, mosaic analysis</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO/lacI/</em></span>pBluescriptII carrier/<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001079;class=Gene" target="_blank">dpy-20</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene" target="_blank">rol-6</a></em></span> marker
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>hsp-16.2p::gfp-lacI</em></span> (induced)
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">
                                 <p>SM</p>
                                 
                                 <p><a href="#bib27">Kiefer et al., 2007</a></p>
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>In vivo</em></span> binding of PHA-4 to <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003012;class=Gene" target="_blank">lin-26</a></em></span> promoter
                              </td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO/lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00000481;class=Gene" target="_blank">cha-1</a></em></span> marker/promoter/ <span class="emphasis"><em>pha-4p::yfp</em></span> plus herring DNA
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>htz-1p::lacI-cfp</em></span></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">
                                 <p>SM</p>
                                 
                                 <p><a href="#bib69">Yzyuk et al., 2010</a></p>
                              </td>
                              <td align="left" valign="top">Chromatin compaction and cell plasticity</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO/lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00000481;class=Gene" target="_blank">cha-1</a></em></span> marker/promoter/ <span class="emphasis"><em>pha-4p::yfp</em></span> plus herring DNA
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>htz-1p::lacI-cfp</em></span></td>
                           </tr>
                           <tr valign="top">
                              <td valign="top">&nbsp;</td>
                              <td valign="top">&nbsp;</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO/lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00000481;class=Gene" target="_blank">cha-1</a></em></span> marker/promoter/ <span class="emphasis"><em>pha-4p::yfp</em></span> plus herring or salmon DNA
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>his-24p::lacI-cfp</em></span></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">
                                 <p>SM</p>
                                 
                                 <p><a href="#bib13">Fakhouri et al., 2010</a></p>
                              </td>
                              <td align="left" valign="top">Transcription factors cell type specificity and function in chromatin decompaction</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO/lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00000481;class=Gene" target="_blank">cha-1</a></em></span> marker/promoter/ <span class="emphasis"><em>pha-4p::yfp</em></span> plus herring DNA
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>his-24p</em></span> or <span class="emphasis"><em>htz-1p::lacI-cfp</em></span></td>
                           </tr>
                           <tr valign="top">
                              <td valign="top">&nbsp;</td>
                              <td valign="top">&nbsp;</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO/lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00000481;class=Gene" target="_blank">cha-1</a></em></span> marker/promoter/ <span class="emphasis"><em>pha-4p::yfp</em></span> plus herring DNA
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>htz-1p::mCherry-lacI</em></span></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">
                                 <p>GW<sup><span class="bold"><strong><span class="emphasis"><em>&sect;</em></span></strong></span></sup></p>
                                 
                                 <p></p>
                                 
                                 <p><a href="#bib36">Meister et al., 2010b</a></p>
                              </td>
                              <td align="left" valign="top">Nuclear organization changes during development</td>
                              <td align="left" valign="top">1x <span class="emphasis"><em>lacO/lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00003515;class=Gene" target="_blank">myo-3</a>::rfp</em></span> marker
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>baf-1p::gfp-lacI</em></span>**
                              </td>
                           </tr>
                           <tr valign="top">
                              <td colspan="4" align="left" valign="top">
                                 <p><span class="bold"><strong>Transgene type</strong></span>: except where indicated transgenes are extrachromosomal injected arrays. <sup><span class="bold"><strong><span class="emphasis"><em>&sect;</em></span></strong></span></sup>Transgene is an integrated injected array.
                                 </p>
                                 
                                 <p><span class="bold"><strong>lacI expression system: coinjected with and on the same array as the <span class="emphasis"><em>lacO</em></span> sites. <span class="emphasis"><em>lacO</em></span> composition: 256x lacO repeats coinjected except where indicated. **</strong></span>1x <span class="emphasis"><em>lacO</em></span> on each plasmid.
                                 </p>
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <p>The main advantage of <span class="emphasis"><em>cis</em></span> systems is their ease of creation: they only require injection of a single DNA mixture, a standard technique in <span class="emphasis"><em>C. elegans</em></span> laboratories. In these systems, the array carries four different components: (1) an expression construct for a fluorescently
                     labelled protein of interest, (2) its putative binding sequence, (3) <span class="emphasis"><em>lacO</em></span> repeats, and (4) the lacI-expressing plasmid. Colocalization on the array of fluorescent lacI and the fluorescent protein
                     of interest (in different colors) indicates binding of the protein of interest to the sequence. SEX-1 binding to the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006962;class=Gene" target="_blank">xol-1</a></em></span> promoter and a number of rules for PHA-4 binding to its target promoters were characterized with this technique (<a href="#bib3">Carmi et al., 1998</a>; <a href="#bib27">Kiefer et al., 2007</a>; <a href="#bib13">Fakhouri et al., 2010</a>). Using complex arrays, the role of H3K27-dependent chromatin compaction during development was uncovered, which would not
                     have been visible with smaller transgenes (<a href="#bib69">Yuzyuk et al., 2009</a>; <a href="#bib13">Fakhouri et al., 2010</a>).
                  </p>
                  <p>Extrachromosomal arrays are randomly lost during cell division. This has the unique advantage to allow cell-autonomous gene-function
                     analysis, or &#8220;mosaic analysis&#8221;. The <span class="emphasis"><em>lacO/</em></span>lacI spot can be used to identify unequivocally cells carrying the array (which becomes visible) (<a href="#bib20">Gonzalez-Serricchio and Sternberg, 2006</a>).
                  </p>
                  <p>The main disadvantage of injected arrays is that it is difficult to control for their size and the exact ratio of their individual
                     components, especially when adding fish DNA. <span class="emphasis"><em>A</em></span> <span class="emphasis"><em>posteriori</em></span> determination of array composition requires genomic DNA purification and quantitative PCR on the different components. Furthermore,
                     it is almost impossible to determine the exact composition of complex arrays as the total size of the array is impossible
                     to determine precisely.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-2-4"></a>5.2.4.&nbsp;<span class="emphasis"><em>Trans</em></span> tagging&#8212;insertion of the lacI expression system and <span class="emphasis"><em>lacO</em></span>-tagged sequences at separate sites
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A number of transcriptional <span class="emphasis"><em>cis</em></span> effects have been observed in arrays: in a strain carrying a GFP-lacI-expressing array with a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003515;class=Gene" target="_blank">myo-3</a></em></span> muscle-specific marker, lacI is more expressed in muscle cells, while with a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a></em></span> gut marker, the same lacI construct is more expressed in gut cells (<a href="#bib36">Meister et al., 2010b</a>). To avoid such <span class="emphasis"><em>cis effects,</em></span> one can separate the transgenes encoding lacI and marker genes from the sequences of interest tagged with <span class="emphasis"><em>lacO</em></span>: <span class="emphasis"><em>trans</em></span> tagging. Moreover, the latter systems are easy to use as strains can be injected and/or crossed (<a href="#bib36">Meister et al., 2010b</a>; <a href="#bib5">Cochella and Hobert, 2012</a>; <a href="#bib56">Towbin et al., 2012</a>; <a href="#bib47">Rohner et al., 2013</a> ).
                  </p>
                  <p>All published systems are described in <a href="#table2" title="Table 2A. Trans tagging systems&#8212;lacI expressing constructs and lacO sites present on different transgenes*">Table 2A</a> and <a href="#table3" title="Table 2B. Sources for Trans tagging systems">B</a>. Importantly, when creating new fluorescent lacI expression arrays for <span class="emphasis"><em>trans</em></span> tagging, the plasmids used must not contain <span class="emphasis"><em>lacO</em></span> sites (5&#8217;-TTGTTATCCGCTCACAA), even as a single copy. Otherwise lacI will bind these and lead to the formation of a spot (<span class="emphasis"><em>cis</em></span> tagging <a href="#bib20">Gonzalez-Serricchio and Sternberg, 2006</a>; <a href="#bib36">Meister et al., 2010b</a>).
                  </p>
                  <div class="table"><a name="table2"></a><p class="title">Table 2A. <span class="emphasis"><em>Trans</em></span> tagging systems&#8212;lacI expressing constructs and <span class="emphasis"><em>lacO</em></span> sites present on different transgenes*
                     </p>
                     <table summary="Table 2A. Trans tagging systems&#8212;lacI expressing constructs and lacO sites present on different transgenes*" border="1">
                        <colgroup>
                           <col>
                           <col>
                           <col>
                           <col>
                           <col>
                           <col>
                           <col>
                           <col>
                        </colgroup>
                        <thead>
                           <tr valign="bottom">
                              <th align="left" valign="bottom"><span class="bold"><strong>#</strong></span></th>
                              <th align="left" valign="bottom">lacI expression</th>
                              <th align="left" valign="bottom">lacI transgene type*</th>
                              <th align="left" valign="bottom">lacI array composition (mixture of plasmids)</th>
                              <th align="left" valign="bottom">lacI expression system</th>
                              <th align="left" valign="bottom"><span class="emphasis"><em>lacO</em></span> target array composition
                              </th>
                              <th align="left" valign="bottom"><span class="emphasis"><em>lacO</em></span> trangene type*
                              </th>
                              <th align="left" valign="bottom"><span class="emphasis"><em>lacO</em></span> composition
                              </th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr valign="top">
                              <td align="left" valign="top">1</td>
                              <td align="left" valign="top"><span class="emphasis"><em>syIs46</em></span></td>
                              <td align="left" valign="top">Ext.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00001079;class=Gene" target="_blank">dpy-20</a></em></span><sup><span class="emphasis"><em>+</em></span></sup></td>
                              <td align="left" valign="top"><span class="emphasis"><em>dpy-30p::gfp-lacI + hsp16.2p::gfp-lacI</em></span> (induced)
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>256x lacO/promoter/<a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene" target="_blank">rol-6</a><sup>+</sup></em></span> plus salmon DNA
                              </td>
                              <td align="left" valign="top">Ext.</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO</em></span> repeats coinjected
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">2</td>
                              <td align="left" valign="top"><span class="emphasis"><em>syIs46</em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/BluescriptII</em></span> carrier plus herring DNA
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>dpy-30p::gfp-lacI + hsp16.2p::gfp-lacI</em></span> (induced)
                              </td>
                              <td align="left" valign="top"><span class="emphasis"><em>256x lacO/</em></span></td>
                              <td align="left" valign="top">Ext.</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO</em></span> repeats coinjected
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">3</td>
                              <td align="left" valign="top"><span class="emphasis"><em>pxEx214</em></span></td>
                              <td align="left" valign="top">Ext.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene" target="_blank">rol-6</a><sup>+</sup></em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>htz-1p::lacI-cfp</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>256x lacO/promoter/<a href="http://www.wormbase.org/db/get?name=WBGene00000481;class=Gene" target="_blank">cha-1</a><sup>+</sup></em></span> plus herring DNA
                              </td>
                              <td align="left" valign="top">Ext.</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO</em></span> repeats coinjected
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">4</td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00006186;class=Transgene" target="_blank">gwIs39</a></em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a>::gfp</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>baf-1p::gfp-lacI</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene" target="_blank">rol-6</a><sup>+</sup>/<a href="http://www.wormbase.org/db/get?name=WBGene00002016;class=Gene" target="_blank">hsp-16.2</a>::hlh-1</em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004013;class=Gene" target="_blank">pha-4</a></em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top">1x <span class="emphasis"><em>lacO</em></span> on each plasmid
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">5</td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00006186;class=Transgene" target="_blank">gwIs39</a></em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a>::gfp</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>baf-1p::gfp-lacI</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>256x lacO/promoter::mCherry/<a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a><sup>+</sup></em></span></td>
                              <td align="left" valign="top">Bom.</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO</em></span> repeats co-bombarded
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">6</td>
                              <td align="left" valign="top"><span class="emphasis"><em>mels1</em></span></td>
                              <td align="left" valign="top">Bom.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a><sup>+</sup></em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004027;class=Gene" target="_blank">pie-1</a>::gfp-lacI</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>64x lacO/<a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene" target="_blank">rol-6</a><sup>+</sup></em></span></td>
                              <td align="left" valign="top">Ext.</td>
                              <td align="left" valign="top">64x <span class="emphasis"><em>lacO</em></span> repeats coinjected
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">7</td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00006186;class=Transgene" target="_blank">gwIs39</a></em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a>::gfp</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>baf-1p::gfp-lacI</em></span></td>
                              <td align="left" valign="top">MosSCI with <span class="emphasis"><em>256x lacO/<a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a><sup>+</sup></em></span></td>
                              <td align="left" valign="top">MosSCI</td>
                              <td align="left" valign="top"><span class="emphasis"><em>256x lacO</em></span> repeats integrated by MosSCI
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">8</td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00006186;class=Transgene" target="_blank">gwIs39</a></em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a>::gfp</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>baf-1p::gfp-lacI</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>256x lacO/Isy-6</em></span> sequence/<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001250;class=Gene" target="_blank">elt-2</a></em></span>::<span class="emphasis"><em>dsred</em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO</em></span> repeats coinjected
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">9</td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00006186;class=Transgene" target="_blank">gwIs39</a></em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a>::gfp</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>baf-1p::gfp-lacI</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002016;class=Gene" target="_blank">hsp-16.2</a>::gfp</em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top">1x <span class="emphasis"><em>lacO</em></span> on each plasmid
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">10</td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00006186;class=Transgene" target="_blank">gwIs39</a></em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a>::gfp</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>baf-1p::gfp-lacI</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>256x lacO/promoter::mCherry/<a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a><sup>+</sup></em></span></td>
                              <td align="left" valign="top">Bom.</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO</em></span> repeats co-bombarded
                              </td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">11</td>
                              <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00006186;class=Transgene" target="_blank">gwIs39</a></em></span></td>
                              <td align="left" valign="top">Int.</td>
                              <td align="left" valign="top"><span class="emphasis"><em>lacI/<a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a>::gfp</em></span></td>
                              <td align="left" valign="top"><span class="emphasis"><em>baf-1p::gfp-lacI</em></span></td>
                              <td align="left" valign="top">MosSCI with <span class="emphasis"><em>256x lacO/<a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a><sup>+</sup></em></span></td>
                              <td align="left" valign="top">MosSCI</td>
                              <td align="left" valign="top">256x <span class="emphasis"><em>lacO</em></span> repeats integrated by MosSCI
                              </td>
                           </tr>
                           <tr valign="top">
                              <td colspan="8" align="left" valign="top">*Transgene type: Ext.&#8212;Extrachromosomal injected array, Int.&#8212;Integrated injected array; Bom.&#8212;Bombarded transgene; MosSCI&#8212;transgene
                                 integrated by MosSCI.
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="table"><a name="table3"></a><p class="title">Table 2B. Sources for <span class="emphasis"><em>Trans</em></span> tagging systems
                     </p>
                     <table summary="Table 2B. Sources for Trans tagging systems" border="1" width="100%">
                        <colgroup>
                           <col>
                           <col>
                           <col>
                           <col>
                        </colgroup>
                        <thead>
                           <tr valign="bottom">
                              <th align="left" valign="bottom">#</th>
                              <th align="left" valign="bottom"><span class="bold"><strong>Laboratory</strong></span></th>
                              <th align="left" valign="bottom"><span class="bold"><strong>Biological question</strong></span></th>
                              <th align="left" valign="bottom"><span class="bold"><strong>Reference</strong></span></th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr valign="bottom">
                              <td align="left" valign="bottom">1</td>
                              <td align="left" valign="bottom">SM</td>
                              <td align="left" valign="bottom"><span class="emphasis"><em>In vivo</em></span> binding of PHA-4 to <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003012;class=Gene" target="_blank">lin-26</a></em></span> promoter
                              </td>
                              <td align="left" valign="bottom"><a href="#bib25">Kaltenbach et al., 2000</a></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">2</td>
                              <td align="left" valign="top">PS</td>
                              <td align="left" valign="top">Ploidy analysis</td>
                              <td align="left" valign="top"><a href="#bib20">Gonzalez-Serrichio et al., 2006</a></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">3</td>
                              <td align="left" valign="top">SM</td>
                              <td align="left" valign="top"><span class="emphasis"><em>In vivo</em></span> binding of HTZ-1 and transcriptional activation of pharyngeal genes
                              </td>
                              <td align="left" valign="top"><a href="#bib58">Updike and Mango, 2006</a></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">4 and 5</td>
                              <td align="left" valign="top">GW</td>
                              <td align="left" valign="top">Nuclear organization changes during development</td>
                              <td align="left" valign="top"><a href="#bib36">Meister et al., 2010b</a></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">6</td>
                              <td align="left" valign="top">OD</td>
                              <td align="left" valign="top"><span class="emphasis"><em>De novo</em></span> centromere formation
                              </td>
                              <td align="left" valign="top"><a href="#bib68">Yuen et al., 2011</a></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">7</td>
                              <td align="left" valign="top">GW</td>
                              <td align="left" valign="top">Signals for perinuclear heterochromatin anchoring</td>
                              <td align="left" valign="top"><a href="#bib56">Towbin et al., 2012</a></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">8</td>
                              <td align="left" valign="top">OH</td>
                              <td align="left" valign="top">Early asymmetric cell commitment and mRNA locus decompaction</td>
                              <td align="left" valign="top"><a href="#bib5">Cochella and Hobert, 2012</a></td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top">9, 10, and 11</td>
                              <td align="left" valign="top">GW</td>
                              <td align="left" valign="top">Nuclear organization and promoter dissection of  heat-shock induced <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002016;class=Gene" target="_blank">hsp-16.2</a></em></span></td>
                              <td align="left" valign="top"><a href="#bib47">Rohner et al., 2013</a></td>
                           </tr>
                           <tr valign="top">
                              <td colspan="4" align="left" valign="top"><span class="bold"><strong>#</strong></span>Corresponds to the # listed in <a href="#table2" title="Table 2A. Trans tagging systems&#8212;lacI expressing constructs and lacO sites present on different transgenes*">Table 2A</a>.
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-2-5"></a>5.2.5.&nbsp;Published <span class="emphasis"><em>C. elegans</em></span> strains constitutively expressing fluorescent lacI
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Four different strains expressing fluorescent lacI have been published to date and can be used directly by crossing them to
                     the desired <span class="emphasis"><em>lacO</em></span> carrying strain. The first strain, <a href="http://www.wormbase.org/db/get?name=PS2958;class=Strain" target="_blank">PS2958</a>, carries an integrated injected array (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00004670;class=Transgene" target="_blank">syIs46</a></em></span>) in which two plasmids drive expression of GFP-lacI, by the heat-shock inducible <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002016;class=Gene" target="_blank">hsp-16.2</a></em></span> promoter, and the constitutive <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001088;class=Gene" target="_blank">dpy-30</a></em></span> promoter (<a href="#bib20">Gonzalez-Serricchio and Sternberg, 2006</a>). GFP-lacI is expressed from the 20-cell stage onward (likely from the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001088;class=Gene" target="_blank">dpy-30</a></em></span> promoter) and is still visible in some cells in adults. However, the plasmids used to create this array contain <span class="emphasis"><em>lacO</em></span> sites, hence the array is microscopically visible as GFP-lacI binds to it. Two large structures can be observed in all cells
                     in uninduced conditions (P. Meister, unpublished observations). This construct has been used for mosaic analysis and ploidy
                     determination. The second strain carries an extrachromosomal complex array (SM1353 <span class="emphasis"><em>pxEx214</em></span>) with a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene" target="_blank">rol-6</a></em></span> marker and a CFP-lacI expression construct under the transcriptional control of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019947;class=Gene" target="_blank">htz-1</a></em></span> promoter (<a href="#bib58">Updike and Mango, 2006</a>). The array also expresses the histone variant HTZ-1 tagged with YFP. The last two strains express GFP-lacI from integrated
                     transgenes. In GW396, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00006186;class=Transgene" target="_blank">gwIs39</a>,</em></span> an injected integrated array, drives expression of GFP-lacI under transcriptional control of the constitutive <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00000235;class=Gene" target="_blank">baf-1</a></em></span> promoter (<a href="#bib36">Meister et al., 2010b</a>). In this strain, expression is silenced in the germline and GFP-lacI is visible from about the 20-cell stage to late L4
                     (this array has a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a>::gfp</em></span> intestinal marker expressed from late L4/adulthood, which hinders GFP-lacI observation at those stages). Last, the AV696
                     strain carries a bombarded construct (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00018266;class=Transgene" target="_blank">meIs1</a></em></span>), in which GFP-lacI is transcriptionally regulated by a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004027;class=Gene" target="_blank">pie-1</a></em></span> promoter (<a href="#bib68">Yuen et al., 2011</a>). Here, the fluorescent signal decreases rapidly at the beginning of embryogenesis, but is expressed in the germline, thus
                     is useful to locate chromatin loci in the germline or in early embryos. Making a strain carrying both <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00018266;class=Transgene" target="_blank">meIs1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBTransgene00006186;class=Transgene" target="_blank">gwIs39</a></em></span> leads to silencing in <span class="emphasis"><em>trans</em></span> of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004027;class=Gene" target="_blank">pie-1</a></em></span> construct in the germline (P. Meister, unpublished observations; <a href="#bib44">Robert et al., 2005</a>).
                  </p>
                  <p>Choice of the fluorescent lacI expression strain should therefore depend mainly on the developmental stage of the biological
                     question, either germline/early embryos (AV696) or late embryos/larvae/adults (GW396/PS2598/SM1353).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-2-6"></a>5.2.6.&nbsp;Methods for <span class="emphasis"><em>lacO</em></span> insertion into the <span class="emphasis"><em>C. elegans</em></span> genome
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The transgene carrying <span class="emphasis"><em>lacO</em></span> sites is what will be visualized. Different sizes of <span class="emphasis"><em>lacO</em></span> transgenes have been analysed, ranging in size from large injected arrays to single copy integrations. Similar to <span class="emphasis"><em>cis</em></span> tagging, simple arrays are readily identifiable with one <span class="emphasis"><em>lacO</em></span> site per plasmid used to create them (<a href="#bib20">Gonzalez-Serricchio and Sternberg, 2006</a>: <a href="#bib36">Meister et al., 2010b</a>). Complex arrays need additional <span class="emphasis"><em>lacO</em></span> repeats to lead to the formation of a visible spot (<a href="#bib58">Updike and Mango, 2006</a>). Most of the published <span class="emphasis"><em>C. elegans</em></span> simple arrays have been created with plasmids containing a single <span class="emphasis"><em>lacO</em></span> site (e.g., most of the Fire library plasmids contain a single <span class="emphasis"><em>lacO</em></span>). These are readily detected, usually at the nuclear periphery linked to their heterochromatic features and transcriptional
                     silencing (<a href="#bib36">Meister et al., 2010b</a>; <a href="#bib57">Towbin et al., 2010</a>).
                  </p>
                  <p>Smaller transgenes with germline expression can be created by bombardment. In this case, 10-50 copies of the bombarded plasmid
                     are typically integrated, with some variability in the number of the individual co-bombarded plasmids (<a href="#bib36">Meister et al., 2010b</a>; <a href="#bib47">Rohner et al., 2013</a>). A single <span class="emphasis"><em>lacO</em></span> site per plasmid is no longer sufficient for the formation of a spot (compared to the 200-500 <span class="emphasis"><em>lacOs</em></span> present in an injected simple array&#8212;in budding yeast about 24 repeats is the minimal repeat length that can be visualized,
                     yet this depends on the unbound lacI fluorescent background). Hence, <span class="emphasis"><em>lacO</em></span> repeats must be co-bombarded with the plasmids of interest (e.g., 256 repeat, pSR1) (<a href="#bib47">Rohner et al., 2008</a>; <a href="#bib36">Meister et al., 2010b</a>). This method leads to about 50% co-integration rate, with visible transgenes when GFP-lacI is expressed in <span class="emphasis"><em>trans</em></span>. These low-copy transgenes are less subject to silencing and display usually a random localization in embryos (<a href="#bib36">Meister et al., 2010b</a>; <a href="#bib57">Towbin et al., 2010</a>; <a href="#bib47">Rohner et al., 2013</a>).
                  </p>
                  <p>Single targeted genomic insertions of <span class="emphasis"><em>lacO</em></span> arrays can be achieved using MosSCI and derivatives (<a href="#bib15">Frokjaer-Jensen et al., 2008</a>; <a href="#bib56">Towbin et al., 2012</a>; <a href="#bib47">Rohner et al., 2013</a>). Unlike insertion at random loci, this method allows the insertion of different sequences next to the <span class="emphasis"><em>lacO</em></span> array at the same genomic location. This equalizes potential integration site and copy number effects, which are impossible
                     to control for when using microparticle bombardment or injected arrays. As this method is new, two variations are detailed
                     below. Finally, a number of strains carrying a single copy insertion of <span class="emphasis"><em>lacO</em></span> repeats have been created by randomly inserting a modified <span class="emphasis"><em>Mos</em></span> transposon (Christian Fr&oslash;kj&aelig;r-Jensen, personal communication).
                  </p>
                  <p>For all single site insertions, 256 <span class="emphasis"><em>lacO</em></span> repeats (about &#8764;12 kb in size) are used for a sufficient signal to noise ratio of the lacI fluorescence. One needs to be
                     cautious when using plasmids containing <span class="emphasis"><em>lacO</em></span> repeats since such vectors (pSV2-dhfr-8.32, pSR1, <a href="#bib46">Robinett et al., 1996</a>; <a href="#bib47">Rohner et al., 2008</a>) are direct repeats and can recombine during amplification in bacteria. Therefore recA bacterial strains (DH5&#945;/XL1 blue)
                     or recB recJ (SURE) are recommended for amplification/cloning. Moreover, when defreezing bacterial strains transformed with
                     <span class="emphasis"><em>lacO</em></span> repeat plasmids, it is good practice to isolate about 8 single colonies and test the length of <span class="emphasis"><em>lacO</em></span> repeats by restriction enzyme digestion as the repeat stretch has a tendency to shrink in size. In addition, after insertion
                     into the <span class="emphasis"><em>C. elegans</em></span> genome the actual number of sites that is integrated is likely to be lower.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-3"></a>5.3.&nbsp;LacI/lacO protocols
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-3-1"></a>5.3.1.&nbsp;<span class="emphasis"><em>Cis</em></span> tagging&#8212;lacO and lacI expression constructs on the same transgene
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-1-1"></a>5.3.1.1.&nbsp;<span class="emphasis"><em>Cis</em></span> tagging&#8212;simple extrachromosomal arrays
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>For simple arrays, the single <span class="emphasis"><em>lacO</em></span> site present in the plasmid is sufficient for the formation of a visible spot due to the high plasmid copy number. Injection
                        is carried out as for normal arrays, with concentrations of the fluorescent lacI construct ranging from 2.5&#8211;50 ng/&#956;l (see
                        WormBook chapter <a href="http://www.wormbook.org/chapters/www_transformationmicroinjection/transformationmicroinjection.html" target="_top">Transformation and microinjection</a>).
                     </p>
                     <p>Typical concentrations are as follows:</p>
                     <p>-	selectable marker (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003515;class=Gene" target="_blank">myo-3</a>::rfp</em></span> construct)	10-50 ng/&#956;l
                     </p>
                     <p>-	fluorescent lacI expression construct	2.5-50 ng/&#956;l</p>
                     <p>-	sequences of interest	variable (length dependent)</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-1-2"></a>5.3.1.2.&nbsp;<span class="emphasis"><em>Cis</em></span> tagging&#8212;complex arrays
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>For complex arrays, additional <span class="emphasis"><em>lacO</em></span> sites should be added to the injection mixture.
                     </p>
                     <p>Typical concentrations are as follows:</p>
                     <p>-	selectable marker (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00000481;class=Gene" target="_blank">cha-1</a></em></span> rescuing construct)	2 ng/&#956;l
                     </p>
                     <p>-	<span class="emphasis"><em>lacO</em></span> repeat fragment (e.g., pSV-dhfr-8.32 <span class="emphasis"><em>Sph</em></span>I/<span class="emphasis"><em>Kpn</em></span>I)	4.5 ng/&#956;l
                     </p>
                     <p>-	fluorescent lacI expression construct	2.5-50 ng/&#956;l</p>
                     <p>-	sequences of interest	variable (length dependent)</p>
                     <p>-	sheared salmon/herring sperm DNA/<span class="emphasis"><em>E. coli</em></span> DNA	to 100 ng/&#956;l
                     </p>
                     <p>Injection is carried out as described (see <a href="http://www.wormbook.org/chapters/www_transformationmicroinjection/transformationmicroinjection.html" target="_top">Transformation and microinjection</a>).
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-3-2"></a>5.3.2.&nbsp;<span class="emphasis"><em>Trans</em></span> tagging
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-2-1"></a>5.3.2.1.&nbsp;<span class="emphasis"><em>Trans</em></span> tagging&#8212;simple and complex extrachromosomal arrays
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>These arrays are similar to those for <span class="emphasis"><em>cis</em></span> tagging, except that the fluorescent lacI expression construct is omitted and the injected strain expresses a fluorescent
                        lacI (see above). In case addition of metazoan DNA to the complex array is a concern, replacing sheared fish DNA with sheared
                        <span class="emphasis"><em>E. coli</em></span> DNA has been used successfully (<a href="#bib5">Cochella and Hobert, 2012</a>).
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-2-2"></a>5.3.2.2.&nbsp;Bombardment-mediated insertion of lacO arrays next to sequences of interest
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>As for complex arrays, the unique <span class="emphasis"><em>lacO</em></span> site in the plasmids used for bombardment is not sufficient for the formation of a visible spot. Hence, a <span class="emphasis"><em>lacO</em></span> repeats-containing fragment is co-bombarded with the sequences of interest and a selectable marker. Typically, <span class="emphasis"><em>unc-119(<a href="http://www.wormbase.org/db/get?name=WBVar00145093;class=Variation" target="_blank">ed3</a>)</em></span> nematodes are bombarded with 8 &#956;g of a 1:1:1 (w/w) mixture of an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a></em></span> rescuing construct, the sequences of interest and <span class="emphasis"><em>lacO</em></span> repeats (e.g., pSR1 <a href="#bib47">Rohner et al., 2008</a>).
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-2-3"></a>5.3.2.3.&nbsp;Single copy integration of endogenous loci
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p><span class="emphasis"><em>lacO</em></span> constructs exist for ectopic integration of sequences at the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBVar00254893;class=Variation" target="_blank">ttTi5605</a></em></span> chromosome II MosSCI locus (<a href="#bib47">Rohner et al., 2013</a>). These plasmids can be modified by restriction enzyme cloning to insert sequences of interest next to the <span class="emphasis"><em>lacO</em></span> arrays. The resulting plasmid is used as a template to integrate <span class="emphasis"><em>lacO</em></span> repeats at the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBVar00254893;class=Variation" target="_blank">ttTi5605</a></em></span> site. MosSCI is carried out according to the original protocol (<a href="#bib15">Frokjaer-Jensen et al., 2008</a>; <a href="#bib15">Frokjaer-Jensen et al., 2012</a>). Integration of certain constructs, especially longer ones, is more difficult to achieve for unclear reasons, in which case
                        the indirect MosSCI heat-shock procedure is recommended.
                     </p>
                     <p>To tag other loci with <span class="emphasis"><em>lacO</em></span> repeats (e.g., close to your favorite gene/sequence), similar <span class="emphasis"><em>Mos-</em></span>mediated integration can be carried out using a <span class="emphasis"><em>Mos</em></span> insertion located near the site of interest. Due to the resolution of optical microscopes, &#8216;near&#8217; means in the next 20-40
                        kb (<a href="#bib35">Meister et al., 2010a</a>). In some cases, increasing the distance between the <span class="emphasis"><em>Mos</em></span> insertion and the region of interest might even be recommended to avoid possible artifacts due to the integration of <span class="emphasis"><em>lacO</em></span> repeats. Although no such case has been described in the nematode, it is a good control to compare localization obtained
                        with <span class="emphasis"><em>lacO</em></span> integrations with fluorescence <span class="emphasis"><em>in situ</em></span> hybridization of the same locus in a wild-type strain. Approximately 13,000 Mos insertions have been created by the NemaGeneTag
                        project, and are available to the community as unvalidated or validated insertions (<a href="http://elegans.imbb.forth.gr/nemagenetag/" target="_top">http://elegans.imbb.forth.gr/nemagenetag/</a>). All insertions can be browsed (<a href="http://pbil.univ-lyon1.fr/segalat/data/index.php" target="_top">http://pbil.univ-lyon1.fr/segalat/data/index.php</a>) and ordered (<a href="http://ums3421.univ-lyon1.fr/spip.php?article14" target="_top">http://ums3421.univ-lyon1.fr/spip.php?article14</a>) online. This is the most cost-effective manner to create <span class="emphasis"><em>lacO</em></span>-tagged loci, as compared to ZFN/TALEN methods (Transcription-Activator Like Effector Nuclease) (<a href="#bib66">Wood et al., 2011</a>), in which the nuclease is engineered to create a double strand break at a given sequence, providing a break for homologous
                        recombination. CRISPR-based techniques are likely to be similarly efficient to Mos excision for the creation of the site-specific
                        DSB (<a href="#bib70">Dickinson et al., 2013</a>). To achieve site-specific recombination, MosSCI requires homology between the <span class="emphasis"><em>Mos</em></span> insertion site where the break is created by transposon excision and the plasmid, which serves as a template for homologous
                        recombination during break repair. To efficiently achieve the creation of templates with homology surrounding <span class="emphasis"><em>lacO</em></span> repeats, a triple plasmid Gateway<sup>&reg;</sup> (Invitrogen) system with a middle <span class="emphasis"><em>lacO/Cb <a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a></em></span> plasmid has been created (<a href="#bib56">Towbin et al., 2012</a>).
                     </p>
                     <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

<tr>
  <td align="center"><p><a href="chromatinanalysis_fig3.jpg"><img src="chromatinanalysis_fig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>

                        </table>
                        <div class="caption">
                           <p><b>Figure 3. Method for insertion of <span class="emphasis"><em>lacO</em></span> repeats at a given <span class="emphasis"><em>Mos</em></span> insertion site. A.&nbsp;</b>Regions flanking the <span class="emphasis"><em>Mos</em></span> insertions are amplified by PCR using primers a, b, c, and d with Gateway-specific overhangs (dotted, see sequence in the
                              text). Amplified homology stretch is recombined in the Gateway 5&#8217; and 3&#8217; vectors using BP. <span class="bold"><strong>B.</strong></span> The template for recombination is created by LR recombination of the vectors created in A. with the middle <span class="emphasis"><em>lacO/Cb <a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a></em></span> . <span class="bold"><strong>C.</strong></span> <span class="emphasis"><em>Mos-</em></span>mediated Single Copy Integration is carried out in the strain carrying the <span class="emphasis"><em>Mos</em></span> insertion of interest in an <span class="emphasis"><em>unc-119(<a href="http://www.wormbase.org/db/get?name=WBVar00145093;class=Variation" target="_blank">ed3</a>)</em></span> background, which leads to the integration of the <span class="emphasis"><em>lacO/Cb <a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a></em></span> template at the <span class="emphasis"><em>Mos</em></span> locus. <span class="bold"><strong>D.</strong></span> Insertion is tested by PCR using primers located in the <span class="emphasis"><em>lacO/Cb <a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a></em></span> insertion and outside of the homology stretch.
                           </p>
                        </div>
                     </div>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>Cross the <span class="emphasis"><em>Mos</em></span> insertion strain to an <span class="emphasis"><em>unc-119(<a href="http://www.wormbase.org/db/get?name=WBVar00145093;class=Variation" target="_blank">ed3</a>)</em></span> mutant and select for <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000643;class=Phenotype" target="_blank">Unc</a> worms homozygous for the <span class="emphasis"><em>Mos</em></span> insertion of interest (selection by PCR). At this point, it is good to cross in the transgene for the expression of GFP-lacI
                                 to have a visual confirmation of <span class="emphasis"><em>lacO</em></span> integration later (see above for strains).
                              </p>
                           </li>
                           <li>
                              <p>Design primers to amplify left and right sequences located at the 5&#8217; and 3&#8217; of the <span class="emphasis"><em>Mos</em></span> insertion site (<a href="#figure3">Figure 3A</a>). 1.5 kb is sufficient for efficient recombination (Robert and Bessereau,  ). Primers are designed for Gateway BP cloning.
                              </p>
                              <p>Use following 5&#8217; extensions for the following primers a to d (in 5&#8217;-&gt; 3&#8217; direction):</p>
                              <div class="orderedlist">
                                 <ol type="1">
                                    <li>
                                       <p>a: GGGG ACA ACT TTG TAT AGA AAA GTT G&#8212;locus specific sequence</p>
                                    </li>
                                    <li>
                                       <p>b: GGGG AC TGC TTT TTT GTA CAA ACT TG&#8212;locus specific sequence (reverse complement)</p>
                                    </li>
                                    <li>
                                       <p>c: GGGG ACA GCT TTC TTG TAC AAA GTG G&#8212;locus specific sequence</p>
                                    </li>
                                    <li>
                                       <p>d: GGGG AC AAC TTT GTA TAA TAA AGT TG&#8212;locus specific sequence (reverse complement).</p>
                                    </li>
                                 </ol>
                              </div>
                              <p>To minimize amplification errors, we use Phusion<sup>&reg;</sup> polymerase (New England Biolabs, Inc.) according to the manufacturer's instruction with the following cycling conditions:
                                 [98&deg;C, 30&#8217;&#8217;]; [98&deg;C, 10&#8217;&#8217;; 45&deg;C, 30&#8217;&#8217;; 72&deg;C 30&#8217;&#8217;/kb] 5x; [98&deg;C, 10&#8217;&#8217;; 50&deg;C, 30&#8217;&#8217;; 72&deg;C, 30&#8217;&#8217;/kb] 20x; 72&deg;C, 10&#8217; 12&deg;C &#8734;.
                              </p>
                              <p>Test amplicon length and perform BP cloning according to the manufacturer's manual (Invitrogen). In our hands, it is preferable
                                 to transform and plate at least half of the cloning reaction to get a sufficient number of colonies. BP cloning has been highly
                                 efficient, with at least 7 out of 8 colonies testing positive for the desired insertion.
                              </p>
                           </li>
                           <li>
                              <p>Perform the Gateway LR reaction with the 5&#8217;, middle [see note above about <span class="emphasis"><em>lacO</em></span> repeats plasmid], and 3&#8217; clones. Transform at least half of the LR reaction to get a sufficient number of colonies. Test
                                 the plasmids using a PCR across 2 elements of the recombination reaction and prepare minipreps.
                              </p>
                           </li>
                           <li>
                              <p>Test the length of <span class="emphasis"><em>lacO</em></span> sites in the destination vector. The length of the repeats should be about 10 kb to make a visible spot <span class="emphasis"><em>in vivo</em></span>. Normally the <span class="emphasis"><em>lacO</em></span> repeats length does not change drastically during LR recombination, as LR uses a site-specific recombinase and transformation
                                 is carried in DH5&#945; recA strains. It is recommended however to purify plasmids from 8-10 different colonies.
                              </p>
                           </li>
                           <li>
                              <p>Carry on MosSCI according to the original protocol (<a href="#bib15">Frokjaer-Jensen et al., 2008</a>; <a href="#bib15">Frokjaer-Jensen et al., 2012</a>). Use the indirect, heat-shock procedure if the direct insertion is not successful.
                              </p>
                           </li>
                           <li>
                              <p>Check insertion by PCR using primers outside of the homology region and in the inserted transgene. Check for the presence
                                 of a visible spot under the microscope in 2-3 lines. During the homologous recombination procedure, some <span class="emphasis"><em>lacO</em></span> repeats might get lost, which therefore makes the GFP-lacI spot difficult or impossible to see. However, most of the lines
                                 we obtained had a readily visible spot.
                              </p>
                           </li>
                        </ol>
                     </div>
                     <p>Independent of the tagging system used, once lines expressing a fluorescent lacI and carrying <span class="emphasis"><em>lacO-</em></span>tagged constructs have been obtained, there are two ways of imaging the lacI/<span class="emphasis"><em>lacO</em></span> spots. The animals are either fixed and stained or directly imaged on an agarose pad. As procedures for fixing, staining
                        and imaging are similar to published protocols (see WormBook chapter <a href="http://www.wormbook.org/chapters/www_immunohistochemistry/immunohistochemistry.html" target="_top">Immunohistochemistry</a>; <a href="#bib27">Kiefer et al., 2007</a>, see also DamID <a href="#sec3-3-2-1" title="3.3.2.1.&nbsp;Immunofluorescence">Section 3.3.2.1</a> above), we will focus on live imaging of nematode embryos and larvae.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-3-3"></a>5.3.3.&nbsp;Preparing worms and embryos for live imaging
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

<tr>
  <td align="center"><p><a href="chromatinanalysis_fig4.jpg"><img src="chromatinanalysis_fig4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 4. Imaging live embryos carrying <span class="emphasis"><em>lacO</em></span> inserts.&nbsp;</b><span class="bold"><strong>A.</strong></span> Preparation of the agarose pad, using 3 microscope slides. The outer two slides are taped to the bench using thick laboratory
                           tape. Additional layers of tape will determine the thickness of the agarose pad. A drop of molten 2% agarose is placed in
                           the center of the middle slide and immediately covered with a perpendicular slide, creating a flat pad. <span class="bold"><strong>B.</strong></span> Embryos dissected in an watch glass are transferred to the agarose pad using a mouth pipette. Excess buffer is removed with
                           the pipette and embryos are grouped with an eyelash (bar = 20 &micro;m). After covering the embryos with a coverslip, the space
                           around the pad is filled with injection oil and the coverslip is sealed to the slide using Vaseline/lanoline/paraffin or Vaseline
                           alone. <span class="bold"><strong>C.</strong></span> Axial z projection of a stack of optical slices of an embryo expressing GFP-lacI from an integrated large array with <span class="emphasis"><em>lacO</em></span> (<a href="http://www.wormbase.org/db/get?name=GW76;class=Strain" target="_blank">GW76</a>, <span class="emphasis"><em>gwIs4[<a href="http://www.wormbase.org/db/get?name=WBGene00000235;class=Gene" target="_blank">baf-1</a>::GFP-lacI; <a href="http://www.wormbase.org/db/get?name=WBGene00003515;class=Gene" target="_blank">myo-3</a>::RFP]X</em></span>). Two large spots corresponding to the homolog chromosomes are visible in each nucleus (bar = 5 &micro;m). <span class="bold"><strong>D.</strong></span> Axial z projection of a stack of optical slices of an embryo expressing GFP-lacI from an integrated large array without <span class="emphasis"><em>lacO</em></span> and an insertion of <span class="emphasis"><em>lacO</em></span> repeats created using the technique described in Figure 1 (<a href="http://www.wormbase.org/db/get?name=GW440;class=Strain" target="_blank">GW440</a>, <span class="emphasis"><em>gwIs39[<a href="http://www.wormbase.org/db/get?name=WBGene00000235;class=Gene" target="_blank">baf-1</a>::GFP-lacI; <a href="http://www.wormbase.org/db/get?name=WBGene00006929;class=Gene" target="_blank">vit-5</a>::GFP]III gwSi0[lacO/Cb unc-119@ttTi5605]II</em></span>)<span class="emphasis"><em>.</em></span> The GFP-lacI expressing array is not visible as it carries no <span class="emphasis"><em>lacO</em></span> sites. The single copy insertion forms small spots, although these are highly dynamic and appear larger on a z-projection.
                        </p>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-3-1"></a>5.3.3.1.&nbsp;Preparing agarose pads for microscopy
                              </h5>
                           </div>
                        </div>
                     </div>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>Prepare the molding bench (<a href="#figure4">Figure 4A</a>).
                              </p>
                              <p>Put three microscope slides next to each other on a flat surface. Stick both outer slides with tape to the surface, putting
                                 thick laboratory tape on top of the slides. The thickness of the pad will depend on the thickness of the tape glued on the
                                 outer slides. Put 2-3 additional layers of tape on the slides to increase the thickness of the pad.
                              </p>
                           </li>
                           <li>
                              <p>Prepare 2% agarose in water and melt at 95&deg;C. Agarose can be aliquoted in 0.5-1 ml tubes for future use and stored at room
                                 temperature. Re-melting of aliquoted agarose is achieved by putting the solution at 95&deg;C for 5 min followed by brief vortexing.
                                 The molten agarose can be kept longer at 70&deg;C instead of 95&deg;C. At high temperatures, agarose hydrolyzes and does not harden
                                 anymore.
                              </p>
                           </li>
                           <li>
                              <p>Prepare the agarose pad (<a href="#figure4">Figure 4A</a>). Put a drop (50-100 &#956;l) of molten agarose on a slide placed in the middle of the two taped slides. Immediately place a second
                                 slide, perpendicularly to the first one, creating a flat surface. This can be kept for several hours if the upper slide is
                                 not removed (otherwise the pad dries out and becomes unusable).
                              </p>
                           </li>
                        </ol>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-3-2"></a>5.3.3.2.&nbsp;Preparing embryos for live imaging
                              </h5>
                           </div>
                        </div>
                     </div>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>Pick gravid adults with fine forceps, move them to an watch glass filled with M9. In our hands, GFP fluorescence is always
                                 higher when worms have been grown at temperatures above 20&deg;C. This is likely due to GFP variants optimized for mammalian expression
                                 at 37&deg;C, as we also observed that fluorescence is higher after heat-shock at 34&deg;C. Try to avoid taking too much bacteria as
                                 <span class="emphasis"><em>E. coli</em></span> will stick to embryos once dissected and usually has high auto-fluorescence. If needed, move the adults with a platinum pick
                                 to bacteria-free plates prior putting them in M9 buffer.
                              </p>
                           </li>
                           <li>
                              <p>Cut adults in the middle of the worm (approximately at the vulva). Delicately squeeze the head and tail parts to release the
                                 embryos from the adult carcasses.
                              </p>
                           </li>
                           <li>
                              <p>Using a mouth pipette, aspirate the embryos. Mouth pipettes are made using glass capillaries: (1) A 10 &#956;L micro-sampling glass
                                 pipette is heated in the flame of an ethanol burner; (2) Once the glass is soft, pull both ends apart; (3) Break the elongated
                                 ends to create a pipette with an end with a diameter of &#8764;40 &#956;m; (4) Place the pipette in a mouth pipette aspirator (Sigma-Aldrich
                                 cat. #A5177 or building plans available upon request).
                              </p>
                           </li>
                           <li>
                              <p>Transfer the embryos to the agarose pad. Make sure to hold the tip of the mouth pipette vertically during the transfer as
                                 failure to do so will lead to dripping of embryos out of the capillary. Once over the agarose pad, blow in the mouth pipette
                                 to expel the embryos, allow them to settle on the agarose (20 sec), delicately aspirate excess M9 with the mouth pipette.
                                 The M9 layer should be at a minimum height, so that embryos do not move anymore when an eyelash is put on the pad.
                              </p>
                           </li>
                           <li>
                              <p>Group embryos together using an eyelash tip (a flexible eyelash fixed on a tip/toothpick with nail polish). Grouping embryos
                                 together is very helpful for finding the embryos under the microscope and minimizes the search time at high magnification.
                              </p>
                           </li>
                           <li>
                              <p>Put a coverslip on top of the pad/embryos, making sure to avoid trapping too many bubbles, in particular close to the embryos
                                 as bubbles diffract light. The coverslip should be laid at an angle on one side of the pad and then slowly lowered on the
                                 embryos, pushing away bubbles.
                              </p>
                           </li>
                           <li>
                              <p>For longer acquisition times, it is recommended to minimize evaporation (which would flatten the pad, damage embryos and shift
                                 focus). Injection oil (mineral oil, Sigma-Aldrich cat. #M5904) efficiently hinders water exchange between the pad and the
                                 exterior. Injection oil also allows oxygenations so that the embryos can develop and hatch if kept overnight in such a chamber.
                                 Put enough oil to fill the gap between the slide and the coverslip around the agarose pad.
                              </p>
                           </li>
                           <li>
                              <p>In order to avoid movement of embryos due to focusing and sliding of the coverslip/pad, glue the coverslip to the slide using
                                 either a 1:1:1 mixture of Vaseline/lanoline/paraffin (also known as Valap) (on 2 sides only to avoid impairing gas exchanges)
                                 or Vaseline (less solid). Both types of glue melt at 70&deg;C.
                              </p>
                           </li>
                        </ol>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-3-3"></a>5.3.3.3.&nbsp;Preparing larvae/adults for live imaging
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Preparation of agarose pads is similar to the preparation of pads for embryos, except that 0.5 &#956;l 10% NaN<sub>3</sub> is added per 1 mL agarose. This inhibits movement, but also depletes ATP in the worms. This technique is hence not suited
                        for long-term imaging. Alternatively, anesthetizing worms with 400 mM ethanol has been recently shown to deliver similar results
                        (Alicia Woock and Jennifer Cecile, personal communication). Similarly, worms can be anesthetized with 1 mg/ml Tricaine (ethyl
                        3-aminobenzoate methanesulfonate salt) and 0.1 mg/ml of tetramisole hydrochloride dissolved in M9 before transferring them
                        onto the agarose pad (<a href="#bib68">Yuen et al., 2011</a>).
                     </p>
                     <p>When transferring worms to the agarose pad, care should be taken not to transfer too many bacteria as these are autofluorescent.
                        Worms can be washed in M9, spun down and recovered with a pipette before depositing a drop of worm-containing M9 on the pad.
                        Excess M9 is removed with a mouth pipette before putting a coverslip on top of the worms.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-3-4"></a>5.3.4.&nbsp;Imaging live embryos and worms
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Imaging <span class="emphasis"><em>lacO</em></span>-tagged chromatin in embryos is challenging: embryos are relatively thick (about 20 &micro;m) and diffract light; the <span class="emphasis"><em>lacO</em></span>/lacI spot can be small (sub-resolution) and highly dynamic; the fluorescent signal is low as overexpression of lacI is deleterious
                     for the animals; and finally, <span class="emphasis"><em>C. elegans</em></span> embryos and larvae are highly sensitive to focused light.
                  </p>
                  <p>If the fluorescence signal of the <span class="emphasis"><em>lacO</em></span>/lacI spot is bright (e.g., the transgene is large) and embryos/worms do not have to be imaged too frequently, a classical
                     epifluorescence microscope equipped with a mercury lamp is sufficient for imaging (<a href="#bib3">Carmi et al., 1998</a>; <a href="#bib20">Gonzalez-Serricchio and Sternberg, 2006</a>; <a href="#bib5">Cochella and Hobert, 2012</a>). These systems are especially well suited when lineaging is necessary.
                  </p>
                  <p>For most <span class="emphasis"><em>lacO</em></span> constructs, however, confocal imaging is essential or optimal to answer the biological question. Indeed, the observed structure
                     can be small (sub-resolution for single copy inserts) and out-of-focus haze would make it invisible. Two main systems are
                     used to achieve confocal images: point scanning and spinning disk confocal microscopes. In the first system a focused laser
                     beam is moving over the sample at high speed, and emitted light is filtered for out-of-focus photons by passing through a
                     pinhole and acquired by a photomultiplier. These systems are well suited for imaging large arrays (<a href="#bib58">Updike and Mango, 2006</a>). They are however not adapted for repeated imaging as laser power hence light-induced damage is concentrated on a single
                     point.
                  </p>
                  <p>An alternative solution, which minimizes excitation/damage, is the use of spinning disk confocal devices. In spinning disk
                     microscopy, the laser beam is split into hundreds of focused sub-beams, rotating on the sample at high speed. The laser power
                     on each point as well as the time spent by the laser on each point of the sample (dwell time) is minimized, but repeated several
                     hundred times per second. Emitted light is filtered through the pinholes, and photons are acquired using a highly sensitive
                     CCD camera. Image acquisition speed is limited by the frame-reading rate of the camera: the whole field of view can be acquired
                     in as little as 30 milliseconds (the minimal readout time of the camera chip). Using a correct combination of lenses/objective/camera,
                     sampling will meet the Nyquist rate (about 90 nm/pixel). We found that the best system for this application is a spinning
                     disk confocal microscope. To obtain stacks spanning the entire thickness of embryos, higher laser power has to be applied
                     to be able to record light emitted from nuclei located far away (in the z direction) from the objective. Indeed, light from
                     these nuclei is diffracted by the embryo structures located between the objective and them, which greatly reduces the signal.
                     In our microscopy systems, this is usually incompatible with survival of the embryos when imaging repeatedly. For time-lapse
                     experiments, imaging is limited to the 10 &#956;m (in the z direction) closer to the objective, which give the best signal to noise
                     ratio. Chromatin inside the nucleus is dynamic, hence precise positioning of the GFP-lacI/<span class="emphasis"><em>lacO</em></span> spots necessitates a fast acquisition device. Typically, exposure time should not be more than 100 ms per optical slice.
                     As for any live imaging, it is good practice to evaluate whether the development of the worms is disturbed by light exposure.
                     An easy way to check this is to compare illuminated and control embryos at approximately the same stage and follow their development
                     over time using DIC microscopy.
                  </p>
                  <p>Typical images acquired with a spinning disk confocal (TILL Photonics) of embryos carrying two different types of <span class="emphasis"><em>lacO</em></span>-tagged chromatin are shown in <a href="#figure4">Figure 4C-D</a>. Large arrays (about 300-500 copies of the injected plasmids, 3-5 Mb in size, strain <a href="http://www.wormbase.org/db/get?name=GW76;class=Strain" target="_blank">GW76</a>) obtained by gonadal injection, create large heterochromatic structures, which are located at the nuclear periphery due to
                     their epigenetic silencing (<a href="#figure4">Figure 4C</a> and <a href="#bib56">Towbin et al., 2012</a>). In <a href="#figure4">Figure 4D</a>, an embryo with 256 <span class="emphasis"><em>lacO</em></span> repeats integrated as a single copy using the MosSCI technique described above is shown (<a href="http://www.wormbase.org/db/get?name=GW440;class=Strain" target="_blank">GW440</a>, <a href="#bib47">Rohner et al., 2013</a>). In each nucleus, the GFP-lacI spot is smaller and emerges from background nuclear fluorescence created by unbound GFP-lacI.
                     This system has been used to test the function of conserved sequences in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002016;class=Gene" target="_blank">hsp-16.2</a></em></span> promoter on perinuclear pore localization, inserting different mutant versions of the promoter at the same locus.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5-3-5"></a>5.3.5.&nbsp;Basic image analysis
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-5-1"></a>5.3.5.1.&nbsp;Compaction state
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Chromatin compaction state has been correlated with its transcriptional status as well as cell plasticity (<a href="#bib69">Yuzyuk et al., 2009</a>; <a href="#bib13">Fakhouri et al., 2010</a>; <a href="#bib5">Cochella and Hobert, 2012</a>; <a href="#bib47">Rohner et al., 2013</a>). Only large arrays are adapted to this type of measurement as for smaller transgenes de-compaction is either not visible
                        (bombarded transgenes) or not relevant (single <span class="emphasis"><em>lacO</em></span> insertions).
                     </p>
                     <p>In the easiest case, arrays adopt different shapes over time or space. Quantification can therefore be achieved by classifying
                        the shapes and counting the number of cells in which arrays have a specific shape. This has been used to show that complex
                        arrays progressively increase their compaction state during development (<a href="#bib69">Yuzyuk et al., 2009</a>).
                     </p>
                     <p>Alternatively, compaction measurement can be achieved on projections along the z-axis by calculating the ratio of the surface
                        occupied by the array versus the projected nuclear surface (<a href="#bib5">Cochella and Hobert, 2012</a>). Similarly, the 3D volume of a transcriptionally-activated array can be normalized to the volume of a non-transcriptionally-activated
                        array (<a href="#bib47">Rohner et al., 2008</a> ). Finally, <span class="emphasis"><em>ad hoc</em></span> solutions can be developed using image analysis software [Volocity (PerkinElmer) or Imaris (Bitplane)] to measure array volume
                        in different cell types or following a given treatment (<a href="#bib47">Rohner et al., 2008</a> ; <a href="#bib13">Fakhouri et al., 2010</a>).
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec5-3-5-2"></a>5.3.5.2.&nbsp;Radial nuclear localization
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Location of a tagged locus inside the nuclear volume is most easily determined by measuring the position of the locus relative
                        to the nuclear periphery (radial position). The nuclear periphery can be outlined either by labeling a component of the nuclear
                        lamina or the nuclear pore complex, or using the background fluorescence of the unbound GFP-lacI. The first method is more
                        precise as boundaries are sharper, but introducing this fluorescent peripheral marker in the <span class="emphasis"><em>lacO</em></span>/GFP-lacI strain requires an additional worm-crossing step. A number of strains exist, with either tagged lamin (YG66, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00000235;class=Gene" target="_blank">baf-1</a>p::gfp::<a href="http://www.wormbase.org/db/get?name=WBGene00003052;class=Gene" target="_blank">lmn-1</a></em></span>, <a href="#bib34">Mattout et al., 2011</a>) or tagged lamin-associated proteins (<a href="http://www.wormbase.org/db/get?name=XA3507;class=Strain" target="_blank">XA3507</a>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004027;class=Gene" target="_blank">pie-1</a>p::<a href="http://www.wormbase.org/db/get?name=WBGene00002275;class=Gene" target="_blank">lem-2</a>::gfp</em></span>, <a href="#bib17">Galy et al., 2003</a>). Alternatively, staining with an anti-lamin or an anti-nuclear pore can also be used.
                     </p>
                     <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

<tr>
  <td align="center"><p><a href="chromatinanalysis_fig5.jpg"><img src="chromatinanalysis_fig5_s.jpg" border="2" align="middle" alt=" figure 5"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 5.&nbsp;</b>Radial scoring using the three zones methods. <span class="bold"><strong>A.</strong></span> A z stack across the embryo is acquired using a confocal microscope. Typical projection along the z axis is shown for an
                              embryo carrying a <span class="emphasis"><em>lacO</em></span>-tagged locus bound by GFP-lacI (2 spots per nucleus) and expressing a nuclear periphery marked (GFP-LMN-1). <span class="bold"><strong>B.</strong></span> Blow-up of a single nucleus from A. <span class="bold"><strong>C.</strong></span> Determination of the xy plane in which the two copies of the tagged locus are located using maximal fluorescence intensity.
                              Spots appear elongated due to the image acquisition system (point spread function). <span class="bold"><strong>D.</strong></span> For each spot in the plane for which its brightness is maximal, measurements of the diameter of the nucleus (red line) and
                              the shortest distance from the spot to the nuclear rim (black line). The section of the nucleus is binned into 3 zones of
                              equal surface (1, 2 and 3). <span class="bold"><strong>E.</strong></span> Binning of measurement into 3 zones for a number of loci/nuclei. The dark blue series are enriched at the nuclear periphery
                              (zone 1) while the light blue series are enriched in the nuclear interior (zones 2 and 3, red dotted line: random distribution,
                              figure adapted from <a href="#bib36">Meister et al., 2010b</a>). <span class="bold"><strong>F.</strong></span> Comparison of the ideal measurement (shell method) and the zone scoring. The excluded volumes are highlighted (figure adapted
                              from <a href="#bib35">Meister et al., 2010a</a>). A systematic although minor bias is introduced towards enrichment in the nuclear interior.
                           </p>
                        </div>
                     </div>
                     <p>Ideally, one would directly measure the 3D distance between the nuclear rim and the tagged locus. Most of the embryonic nuclei,
                        but also many somatic nuclei in larvae and adults, can be considered spherical. There are exceptions however, for example,
                        muscle cell nuclei, which are elongated along the body axis of the worm. In this case radial position determination requires
                        consideration of the elongated and flat shape of nuclei performed by <span class="emphasis"><em>ad hoc</em></span> software (<a href="#bib36">Meister et al., 2010b</a>).
                     </p>
                     <p>In the case of spherical nuclei, determination of the shortest distance from the spot to the nuclear periphery is hindered
                        by physical constraints. Resolution of the microscope in the optical imaging plane (x and y directions) is about 200 nm, while
                        in the z direction it is often more than 450 nm. Hence, determination of the position of a spot in an xy plane is by far more
                        precise than in the xz or yz planes. Similarly, determining where the nuclear edge is in the z direction at the top or the
                        bottom of the sphere is difficult, even using confocal images. One way to overcome this limitation is to make use of the optical
                        z stacks. If sections are close enough in the z direction (0.2 &#956;m steps), and although the &#8216;real&#8217; spot might lie between two
                        imaging planes, its image will be brighter in one plane (<a href="#figure5">Figure 5C</a>). This plane is chosen as the z section of the nucleus, which contains the spot. In this plane, xy positions can be determined
                        with high precision (about 90 nm) and are used to measure the shortest distance from the nuclear periphery to the spot (<a href="#figure5">Figure 5D</a>). However, this measurement is valid for the central z sections only: spots located at the top 25% or the bottom 25% of the
                        nucleus cannot be used (<a href="#figure5">Figure 5F</a>, discarded volumes). In these volumes, the shortest distance from the nuclear periphery to the spot might lie in the z direction,
                        in which case measuring this distance in the xy plane is imprecise (see above). The cross section of the nucleus in this plane
                        is an ellipsoid or a disc. To quantify the distribution of the spot, this surface is split into three concentric zones of
                        equal area (<a href="#figure5">Figure 5D</a>). Using equal surfaces leads to a thin outermost zone at the nuclear periphery (in terms of radial distances, between 1 and
                        0.82 radius) while zone 2 lies between 0.82 and 0.58 radius. Using the coordinates of the spot and the diameter of the nucleus
                        in the axis of the spot, each spot is assigned to one of the three concentric zones. Scoring position in a number of nuclei
                        gives a score for the locus studied in each of the three zones (<a href="#figure5">Figure 5E</a>). If a given locus does not show any preferential localization, it would be equally distributed among the three zones when
                        scored in a sufficient number of cells (33%, red line, <a href="#figure5">Figure 5E</a>). If the locus shows preferential positioning at the nuclear rim, its 3-zone scoring will show enrichment in zone 1 (<a href="#figure5">Figure 5E</a>, dark blue). Conversely, a centrally positioned locus will show higher enrichment in zone 2 and 3 and depletion in zone 1
                        (<a href="#figure5">Figure 5E</a>, light blue).
                     </p>
                     <p>The three-zone scoring method (zones, <a href="#figure5">Figure 5F</a>) and a zoning method using concentric shells (for which one would be able to precisely measure distances in the z direction;
                        shells, <a href="#figure5">Figure 5F</a>) have been compared mathematically. The systematic error introduced by the first method (<a href="#figure5">Figure 5D</a>) is minimal if spots located in the higher and lower caps are excluded. The error introduced leads to a bias towards the
                        most internal zones and an underestimation of the spots located at the nuclear rim (<a href="#bib35">Meister et al., 2010a</a>).
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>We are indebted to Kohta Ikegami for his comments and clarifications on the methods, Cristina Gonz&aacute;lez-Aguilera for her efforts
               in establishing the DamID protocol in <span class="emphasis"><em>C. elegans</em></span>, Bas van Steensel for reagents and advice. We would like to thank Susan Gasser for continuous support and the Gasser and
               Meister laboratories for discussions, especially Lutz Gehlen for computer simulations.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec7"></a>7.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed"><span class="bibliomisc">Braunschweig, U., Hogan, G.J., Pagie, L., and van Steensel, B. (2009). Histone H1 binding is inhibited by histone variant
                        H3.3. EMBO J. <span class="italic">28</span>, 3635-3645.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19834459&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/emboj.2009.301" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed"><span class="bibliomisc">Buck, M.J., and Lieb, J.D. (2004). ChIP-chip: considerations for the design, analysis, and application of genome-wide chromatin
                        immunoprecipitation experiments. Genomics <span class="italic">83</span>, 349-360.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14986705&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.ygeno.2003.11.004" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed"><span class="bibliomisc">Carmi, I., Kopczynski, J.B., and Meyer, B.J. (1998). The nuclear hormone receptor SEX-1 is an X-chromosome signal that determines
                        nematode sex. Nature <span class="italic">396</span>, 168-173.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9823896&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/24164" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed"><span class="bibliomisc">Chen, Y., Negre, N., Li, Q., Mieczkowska, J.O., Slattery, M., Liu, T., Zhang, Y., Kim, T.K., He, H.H., Zieba, J.<span class="italic">, et al.</span> (2012). Systematic evaluation of factors influencing ChIP-seq fidelity. Nat. Methods <span class="italic">9</span>, 609-614.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22522655&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1985" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed"><span class="bibliomisc">Cochella, L., and Hobert, O. (2012). Embryonic priming of a miRNA locus predetermines postmitotic neuronal left/right asymmetry
                        in <span class="italic">C. elegans</span>. Cell <span class="italic">151</span>, 1229-1242.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23201143&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2012.10.049" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed"><span class="bibliomisc">Crittenden, S.L., and Kimble, J. (1999). Confocal methods for <span class="italic">Caenorhabditis elegans</span>. Methods Mol. Biol.<span class="italic">122</span>, 141-151.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10231789&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1385/1-59259-722-X:141" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed"><span class="bibliomisc">Dickinson, D.J., Ward, J.D., Reiner, D.J., and Goldstein, B. (2013). Engineering the <span class="italic">Caenorhabditis elegans</span> genome using Cas9-triggered homologous recombination. Nat. Methods <span class="italic">10</span>, 1028-1034.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23995389&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://www.nature.com/nmeth/journal/v10/n10/full/nmeth.2641.html" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed"><span class="bibliomisc">Dubarry, M., Loiodice, I., Chen, C.L., Thermes, C., and Taddei, A. (2011). Tight protein-DNA interactions favor gene silencing.
                        Genes Dev. <span class="italic">25</span>, 1365-1370.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21724830&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.611011" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed"><span class="bibliomisc">Duerr, J.S. Immunohistochemistry (June 19, 2006), <span class="italic">WormBook</span>, ed. The <span class="italic">C. elegans</span> Research Community, WormBook, doi/10.1895/wormbook.1.105.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16938559&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed"><span class="bibliomisc">Egelhofer, T.A., Minoda, A., Klugman, S., Lee, K., Kolasinska-Zwierz, P., Alekseyenko, A.A., Cheung, M.S., Day, D.S., Gadel,
                        S., Gorchakov, A.A.<span class="italic">, et al.</span> (2011). An assessment of histone-modification antibody quality. Nat. Struct. Mol. Biol. <span class="italic">18</span>, 91-93.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21131980&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/ 10.1038/nsmb.1972" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed"><span class="bibliomisc">Ercan, S., Dick, L.L., and Lieb, J.D. (2009). The <span class="italic">C. elegans</span> dosage compensation complex propagates dynamically and independently of X chromosome sequence. Curr. Biol. <span class="italic">19</span>, 1777-1787.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16938559&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2009.09.047" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed"><span class="bibliomisc">Ercan, S., Giresi, P.G., Whittle, C.M., Zhang, X., Green, R.D., and Lieb, J.D. (2007). X chromosome repression by localization
                        of the <span class="italic">C. elegans</span> dosage compensation machinery to sites of transcription initiation. Nat. Genet. <span class="italic">39</span>, 403-408.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17293863&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed"><span class="bibliomisc">Evans, T.C., ed. Transformation and microinjection (April 6, 2006), <span class="italic">WormBook</span>, ed. The <span class="italic">C. elegans</span> Research Community, WormBook, doi/10.1895/wormbook.1.108.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16938559&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed"><span class="bibliomisc">Fakhouri, T.H., Stevenson, J., Chisholm, A.D., and Mango, S.E. (2010). Dynamic chromatin organization during foregut development
                        mediated by the organ selector gene PHA-4/FoxA. PLoS Genet. <span class="italic">6</span> e1001060.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20714352&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1001060" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed"><span class="bibliomisc">Fox, R.M., Watson, J.D., Von Stetina, S.E., McDermott, J., Brodigan, T.M., Fukushige, T., Krause, M., and Miller, D.M., 3rd
                        (2007). The embryonic muscle transcriptome of <span class="italic">Caenorhabditis elegans</span>. Genome Biol. <span class="italic">8</span>, R188.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17848203&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2007-8-9-r188" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed"><span class="bibliomisc">Frokjaer-Jensen, C., Davis, M.W., Ailion, M., and Jorgensen, E.M. (2012). Improved Mos1-mediated transgenesis in <span class="italic">C. elegans</span>. Nat. Methods <span class="italic">9</span>, 117-118.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22290181&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/0.1038/nmeth.1865" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed"><span class="bibliomisc">Frokjaer-Jensen, C., Davis, M.W., Hopkins, C.E., Newman, B.J., Thummel, J.M., Olesen, S.P., Grunnet, M., and Jorgensen, E.M.
                        (2008). Single-copy insertion of transgenes in <span class="italic">Caenorhabditis elegans</span>. Nat. Genet. <span class="italic">40</span>, 1375-1383.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18953339&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng.248" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed"><span class="bibliomisc">Galy, V., Mattaj, I.W., and Askjaer, P. (2003). <span class="italic">Caenorhabditis elegans</span> nucleoporins Nup93 and Nup205 determine the limit of nuclear pore complex size exclusion <span class="italic">in vivo</span>. Mol. Biol. Cell <span class="italic">14</span>, 5104-5115.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12937276&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed"><span class="bibliomisc">Germann, S., Juul-Jensen, T., Letarnec, B., and Gaudin, V. (2006). DamID, a new tool for studying plant chromatin profiling
                        in vivo, and its use to identify putative LHP1 target loci. Plant J. <span class="italic">48</span>, 153-163.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16972870&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed"><span class="bibliomisc">Gerstein, M.B., Lu, Z.J., Van Nostrand, E.L., Cheng, C., Arshinoff, B.I., Liu, T., Yip, K.Y., Robilotto, R., Rechtsteiner,
                        A., Ikegami, K.<span class="italic">, et al.</span> (2010). Integrative analysis of the <span class="italic">Caenorhabditis elegans</span> genome by the modENCODE project. Science <span class="italic">330</span>, 1775-1787.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177976&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/ 10.1126/science.1196914" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed"><span class="bibliomisc">Gonz&aacute;lez-Aguilera, C., Ikegami, K., Ayuso, C., de Luis, A., I&ntilde;iguez, M., Cabello, J., Lieb, J.D., and Askjaer, P. (2014) Genome-wide
                        analysis links emerin to neuromuscular junction activity in <span class="italic">Caenorhabditis elegans</span>. Genome Biol. <span class="italic">15</span>, R21.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24490688&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://genomebiology.com/2014/15/2/R21" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed"><span class="bibliomisc">Gonzalez-Serricchio, A.S., and Sternberg, P.W. (2006). Visualization of <span class="italic">C. elegans</span> transgenic arrays by GFP. BMC Genet. <span class="italic">7</span>, 36.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16759392&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2156-7-36" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed"><span class="bibliomisc">Greil, F., Moorman, C., and van Steensel, B. (2006). DamID: mapping of <span class="italic">in vivo</span> protein-genome interactions using tethered DNA adenine methyltransferase. Methods Enzymol. <span class="italic">410</span>, 342-359.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16938559&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0076-6879(06)10016-6" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed"><span class="bibliomisc">Hecht, A., and Grunstein, M. (1999). Mapping DNA interaction sites of chromosomal proteins using immunoprecipitation and polymerase
                        chain reaction. Methods Enzymol. <span class="italic">304</span>, 399-414.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10372373&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed"><span class="bibliomisc">Ikegami, K., Egelhofer, T., Strome, S., and Lieb, J.D. (2010). <span class="italic">Caenorhabditis elegans</span> chromosome arms are anchored to the nuclear membrane via discontinuous association with LEM-2. Genome Biol. <span class="italic">11</span>, R120.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21176223&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2010-11-12-r120" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed"><span class="bibliomisc">Jans, J., Gladden, J.M., Ralston, E.J., Pickle, C.S., Michel, A.H., Pferdehirt, R.R., Eisen, M.B., and Meyer, B.J. (2009).
                        A condensin-like dosage compensation complex acts at a distance to control expression throughout the genome. Genes Dev. <span class="italic">23</span>, 602-618.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19270160&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1751109" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed"><span class="bibliomisc">Kaltenbach, L., Horner, M.A., Rothman, J.H., and Mango, S.E. (2000). The TBP-like factor CeTLF is required to activate RNA
                        polymerase II transcription during <span class="italic">C. elegans</span> embryogenesis. Mol. Cell <span class="italic">6</span>, 705-713.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11030349&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S1097-2765(00)00068-X" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed"><span class="bibliomisc">Kalverda, B., and Fornerod, M. (2010). Characterization of genome-nucleoporin interactions in <span class="italic">Drosophila</span> links chromatin insulators to the nuclear pore complex. Cell Cycle <span class="italic">9</span>, 4812-4817.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21150273&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed"><span class="bibliomisc">Kiefer, J.C., Smith, P.A., and Mango, S.E. (2007). PHA-4/FoxA cooperates with TAM-1/TRIM to regulate cell fate restriction
                        in the <span class="italic">C. elegans</span> foregut. Dev. Biol. <span class="italic">303</span>, 611-624.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17250823&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed"><span class="bibliomisc">Kolasinska-Zwierz, P., Down, T., Latorre, I., Liu, T., Liu, X.S., and Ahringer, J. (2009). Differential chromatin marking
                        of introns and expressed exons by H3K36me3. Nat. Genet. <span class="italic">41</span>, 376-381.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19182803&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng.322" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib72"></a><p class="bibliomixed"><span class="bibliomisc">Kranz, A.L., Jiao, C.Y., Winterkorn, L.H., Albritton S.E., Kramer, M., and Ercan S. (2013). Genome-wide analysis of condensin
                        binding in <span class="italic">Caenorhabditis elegans</span>. Genome Biol. <span class="italic">14</span>, R112.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24125077&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://genomebiology.com/2013/14/10/R112" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib73"></a><p class="bibliomixed"><span class="bibliomisc">Kudron, M., Niu, W., Lu, Z., Wang, G., Gerstein, M., Snyder, M., and Reinke, V. (2013). Tissue-specific direct targets of
                        <span class="italic">Caenorhabditis elegans</span> Rb/E2F dictate distinct somatic and germline programs. Genome Biol. <span class="italic">14</span>, R5.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23347407&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://genomebiology.com/content/14/1/R5" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed"><span class="bibliomisc">Kuo, M.H., and Allis, C.D. (1999). <span class="italic">In vivo</span> cross-linking and immunoprecipitation for studying dynamic Protein:DNA associations in a chromatin environment. Methods <span class="italic">19</span>, 425-433.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10579938&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1006/meth.1999.0879" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed"><span class="bibliomisc">Landt, S.G., Marinov, G.K., Kundaje, A., Kheradpour, P., Pauli, F., Batzoglou, S., Bernstein, B.E., Bickel, P., Brown, J.B.,
                        Cayting, P.<span class="italic">, et al.</span> (2012). ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia. Genome Res. <span class="italic">22</span>, 1813-1831.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22955991&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.136184.111" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed"><span class="bibliomisc">Liu, T., Rechtsteiner, A., Egelhofer, T.A., Vielle, A., Latorre, I., Cheung, M.S., Ercan, S., Ikegami, K., Jensen, M., Kolasinska-Zwierz,
                        P.<span class="italic">, et al.</span> (2011). Broad chromosomal domains of histone modification patterns in <span class="italic">C. elegans</span>. Genome Res. <span class="italic">21</span>, 227-236.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177964&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.115519.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed"><span class="bibliomisc">Luo, S.D., Shi, G.W., and Baker, B.S. (2011). Direct targets of the <span class="italic">D</span>. <span class="italic">melanogaster</span> DSXF protein and the evolution of sexual development. Development <span class="italic">138</span>, 2761-2771.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21652649&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.065227" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed"><span class="bibliomisc">Mangone, M., Manoharan, A.P., Thierry-Mieg, D., Thierry-Mieg, J., Han, T., Mackowiak, S.D., Mis, E., Zegar, C., Gutwein, M.R.,
                        Khivansara, V.<span class="italic">, et al.</span> (2010). The landscape of <span class="italic">C. elegans</span> 3'UTRs. Science <span class="italic">329</span>, 432-435.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20522740&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/ 10.1126/science.1191244" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed"><span class="bibliomisc">Mattout, A., Pike, B.L., Towbin, B.D., Bank, E.M., Gonzalez-Sandoval, A., Stadler, M.B., Meister, P., Gruenbaum, Y., and Gasser,
                        S.M. (2011). An EDMD mutation in <span class="italic">C. elegans</span> lamin blocks muscle-specific gene relocation and compromises muscle integrity. Curr. Biol. <span class="italic">21</span>, 1603-1614.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21962710&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2011.08.030" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed"><span class="bibliomisc">Meister, P., Gehlen, L., Varela, E., Kalck, V., and Gasser, S.M. (2010a). Visualizing yeast chromosomes and nuclear architecture.
                        Methods Enzymol. <span class="italic">470</span>, 537-569.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20946824&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0076-6879(10)70021-5" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed"><span class="bibliomisc">Meister, P., Towbin, B.D., Pike, B.L., Ponti, A., and Gasser, S.M. (2010b). The spatial dynamics of tissue-specific promoters
                        during <span class="italic">C. elegans</span> development. Genes Dev. <span class="italic">24</span>, 766-782.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20395364&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.559610" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed"><span class="bibliomisc">Moorman, C., Sun, L.V., Wang, J., de <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000230;class=Phenotype" target="_blank">Wit</a>, E., Talhout, W., Ward, L.D., Greil, F., Lu, X.J., White, K.P., Bussemaker, H.J., and van Steensel, B. (2006). Hotspots of
                        transcription factor colocalization in the genome of <span class="italic">Drosophila melanogaster</span>. Proc. Natl. Acad. Sci. U. S. A.  <span class="italic">103</span>, 12027-12032.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16880385&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0605003103" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed"><span class="bibliomisc">Negre, N., Hennetin, J., Sun, L.V., Lavrov, S., Bellis, M., White, K.P., and Cavalli, G. (2006). Chromosomal distribution
                        of PcG proteins during <span class="italic">Drosophila</span> development. PLoS Biol. <span class="italic">4</span>, e170.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16613483&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0040170" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed"><span class="bibliomisc">Niu, W., Lu, Z.J., Zhong, M., Sarov, M., Murray, J.I., Brdlik, C.M., Janette, J., Chen, C., Alves, P., Preston, E.<span class="italic">, et al.</span> (2011). Diverse transcription factor binding features revealed by genome-wide ChIP-seq in <span class="italic">C. elegans</span>. Genome Res. <span class="italic">21</span>, 245-254.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177963&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.114587.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed"><span class="bibliomisc">Orian, A., van Steensel, B., Delrow, J., Bussemaker, H.J., Li, L., Sawado, T., Williams, E., Loo, L.W., Cowley, S.M., Yost,
                        C.<span class="italic">, et al.</span> (2003). Genomic binding by the <span class="italic">Drosophila</span> Myc, Max, Mad/Mnt transcription factor network. Genes Dev. <span class="italic">17</span>, 1101-1114.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12695332&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1066903" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed"><span class="bibliomisc">Park, P.J. (2009). ChIP-seq: advantages and challenges of a maturing technology. Nat. Rev. Genet. <span class="italic">10</span>, 669-680.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19736561&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg2641" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed"><span class="bibliomisc">Pickersgill, H., Kalverda, B., de <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000230;class=Phenotype" target="_blank">Wit</a>, E., Talhout, W., Fornerod, M., and van Steensel, B. (2006). Characterization of the <span class="italic">Drosophila</span> <span class="italic">melanogaster</span> genome at the nuclear lamina. Nat. Genet. <span class="italic">38</span>, 1005-1014.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16878134&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1852" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed"><span class="bibliomisc">Rechtsteiner, A., Ercan, S., Takasaki, T., Phippen, T.M., Egelhofer, T.A., Wang, W., Kimura, H., Lieb, J.D., and Strome, S.
                        (2010). The histone H3K36 methyltransferase MES-4 acts epigenetically to transmit the memory of germline gene expression to
                        progeny. PLoS Genet. <span class="italic">6</span>, e1001091.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20824077&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1001091" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed"><span class="bibliomisc">Robert, V., and Bessereau, J.L. (2007). Targeted engineering of the <span class="italic">Caenorhabditis elegans</span> genome following Mos1-triggered chromosomal breaks. EMBO J. <span class="italic">26</span>, 170-183.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15774721&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/sj.emboj.7601463" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed"><span class="bibliomisc">Robert, V.J., Sijen, T., van Wolfswinkel, J., and Plasterk, R.H. (2005). Chromatin and RNAi factors protect the <span class="italic">C. elegans</span> germline against repetitive sequences. Genes Dev. <span class="italic">19</span>, 782-787.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15774721&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.332305" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed"><span class="bibliomisc">Robinett, C.C., Straight, A., Li, G., Willhelm, C., Sudlow, G., Murray, A., and Belmont, A.S. (1996). <span class="italic">In vivo</span> localization of DNA sequences and visualization of large-scale chromatin organization using lac operator/repressor recognition.
                        J. Cell Biol. <span class="italic">135</span>, 1685-1700.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8991083&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.135.6.1685" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed"><span class="bibliomisc">Rohner, S., Gasser, S.M., and Meister, P. (2008). Modules for cloning-free chromatin tagging in <span class="italic">Saccharomyces cerevisae</span>. Yeast <span class="italic">25</span>, 235-239.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18302313&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/yea.1580" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed"><span class="bibliomisc">Rohner, S., Kalck, V., Wang, X., Ikegami, K., Lieb, J.D., Gasser, S.M., and Meister, P. (2013). Promoter- and RNA polymerase
                        II-dependent <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002020;class=Gene" target="_blank">hsp-16</a></span> gene association with nuclear pores in <span class="italic">Caenorhabditis elegans</span>. J. Cell Biol. <span class="italic">200</span>, 589-604.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23460676&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.201207024" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed"><span class="bibliomisc">Sarov, M., Murray, J.I., Schanze, K., Pozniakovski, A., Niu, W., Angermann, K., Hasse, S., Rupprecht, M., Vinis, E., Tinney,
                        M.<span class="italic">, et al.</span> (2012). A Genome-scale resource for <span class="italic">in vivo</span> tag-based protein function exploration in <span class="italic">C. elegans</span>. Cell <span class="italic">150</span>, 855-866.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22901814&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2012.08.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed"><span class="bibliomisc">Schuster, E., McElwee, J.J., Tullet, J.M., Doonan, R., Matthijssens, F., Reece-Hoyes, J.S., Hope, I.A., Vanfleteren, J.R.,
                        Thornton, J.M., and Gems, D. (2010). DamID in <span class="italic">C. elegans</span> reveals longevity-associated targets of DAF-16/FoxO. Mol. Syst. Biol. <span class="italic">6</span>, 399.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20706209&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/msb.2010.54" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed"><span class="bibliomisc">Sha, K., Gu, S.G., Pantalena-Filho, L.C., Goh, A., Fleenor, J., Blanchard, D., Krishna, C., and Fire, A. (2010). Distributed
                        probing of chromatin structure <span class="italic">in vivo</span> reveals pervasive chromatin accessibility for expressed and non-expressed genes during tissue differentiation in <span class="italic">C. elegans</span>. BMC Genomics <span class="italic">11</span>, 465.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20691096&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-11-465" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed"><span class="bibliomisc">Song, S., Cooperman, J., Letting, D.L., Blobel, G.A., and Choi, J.K. (2004). Identification of cyclin D3 as a direct target
                        of E2A using DamID. Mol. Cell. Biol. <span class="italic">24</span>, 8790-8802.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15367695&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.24.19.8790-8802" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed"><span class="bibliomisc">Steglich, B., Filion, G. J., van Steensel, B., and Ekwall, K. (2012). The inner nuclear membrane proteins Man1 and Ima1 link
                        to two different types of chromatin at the nuclear periphery in <span class="italic">S. pombe</span>. Nucleus <span class="italic">3</span>, 77-87.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22156748&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.4161/nucl.18825" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed"><span class="bibliomisc">Steiner, F.A., Talbert, P.B., Kasinathan, S., Deal, R.B., and Henikoff, S. (2012). Cell type-specific nuclei purification
                        from whole animals for genome-wide expression and chromatin profiling. Genome Res. <span class="italic">22</span>, 766-777.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22219512&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.131748.111" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed"><span class="bibliomisc">Stringham, E.G., Dixon, D.K., Jones, D., and Candido, E.P. (1992). Temporal and spatial expression patterns of the small heat
                        shock (<span class="italic">hsp16</span>) genes in transgenic <span class="italic">Caenorhabditis elegans</span>. Mol. Biol. Cell <span class="italic">3</span>, 221-233.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1550963&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed"><span class="bibliomisc">Towbin, B.D., Gonzalez-Aguilera, C., Sack, R., Gaidatzis, D., Kalck, V., Meister, P., Askjaer, P., and Gasser, S.M. (2012).
                        Step-wise methylation of histone H3K9 positions heterochromatin at the nuclear periphery. Cell <span class="italic">150</span>, 934-947.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22939621&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2012.06.051" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed"><span class="bibliomisc">Towbin, B.D., Meister, P., Pike, B.L., and Gasser, S.M. (2010). Repetitive transgenes in <span class="italic">C. elegans</span> accumulate heterochromatic marks and are sequestered at the nuclear envelope in a copy-number- and lamin-dependent manner.
                        Cold Spring Harb. Symp. Quant. Biol. <span class="italic">75</span>, 555-565.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21467137&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/sqb.2010.75.041" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed"><span class="bibliomisc">Updike, D.L., and Mango, S.E. (2006). Temporal regulation of foregut development by HTZ-1/H2A.Z and PHA-4/FoxA. PLoS Genet.
                        <span class="italic">2</span>, e161.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17009877&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.0020161" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed"><span class="bibliomisc">van Bemmel, J.G., Filion, G.J., Rosado, A., Talhout, W., de Haas, M., van Welsem, T., van Leeuwen, F., and van Steensel, B.
                        (2013). A network model of the molecular organization of chromatin in <span class="italic">Drosophila</span>. Mol. Cell <span class="italic">49</span>, 759-771.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23438860&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed"><span class="bibliomisc">van Steensel, B., and Henikoff, S. (2000). Identification of <span class="italic">in vivo</span> DNA targets of chromatin proteins using tethered dam methyltransferase. Nat. Biotechnol. <span class="italic">18</span>, 424-428.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10748524&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/74487" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed"><span class="bibliomisc">Venkatasubrahmanyam, S., Hwang, W.W., Meneghini, M.D., Tong, A.H., and Madhani, H.D. (2007). Genome-wide, as opposed to local,
                        antisilencing is mediated redundantly by the euchromatic factors Set1 and H2A.Z. Proc. Natl. Acad. Sci. U. S. A.  <span class="italic">104</span>, 16609-16614.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17925448&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0700914104" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed"><span class="bibliomisc">Vogel, M. J., Guelen, L., de <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000230;class=Phenotype" target="_blank">Wit</a>, E., Peric-Hupkes, D., Loden, M., Talhout, W., Feenstra, M., Abbas, B., Classen, A. K., and van Steensel, B. (2006). Human
                        heterochromatin proteins form large domains containing KRAB-ZNF genes. Genome Res. <span class="italic">16</span>, 1493-1504.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17038565&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.5391806" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed"><span class="bibliomisc">Vogel, M.J., Peric-Hupkes, D., and van Steensel, B. (2007). Detection of <span class="italic">in vivo</span> protein-DNA interactions using DamID in mammalian cells. Nat. Protoc. <span class="italic">2</span>, 1467-1478.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17545983&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nprot.2007.148" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed"><span class="bibliomisc">Von Stetina, S.E., Watson, J.D., Fox, R.M., Olszewski, K.L., Spencer, W.C., Roy, P.J., and Miller, D.M., 3rd (2007). Cell-specific
                        microarray profiling experiments reveal a comprehensive picture of gene expression in the <span class="italic">C. elegans</span> nervous system. Genome Biol. <span class="italic">8</span>, R135.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17612406&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2007-8-7-r135" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed"><span class="bibliomisc">Whittle, C.M., McClinic, K.N., Ercan, S., Zhang, X., Green, R.D., Kelly, W.G., and Lieb, J.D. (2008). The genomic distribution
                        and function of histone variant HTZ-1 during <span class="italic">C. elegans</span> embryogenesis. PLoS Genet. <span class="italic">4</span>, e1000187.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18787694&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000187" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed"><span class="bibliomisc">Wood, A.J., Lo, T.W., Zeitler, B., Pickle, C.S., Ralston, E.J., Lee, A.H., Amora, R., Miller, J.C., Leung, E., Meng, X.<span class="italic">, et al.</span> (2011). Targeted genome editing across species using ZFNs and TALENs. Science <span class="italic">333</span>, 307.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21700836&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1207773" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed"><span class="bibliomisc">Woolcock, K.J., Gaidatzis, D., Punga, T., and Buhler, M. (2011). Dicer associates with chromatin to repress genome activity
                        in <span class="italic">Schizosaccharomyces pombe</span>. Nat. Struct. Mol. Biol.<span class="italic">18</span>, 94-99.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21151114&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nsmb.1935" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed"><span class="bibliomisc">Yuen, K.W., Nabeshima, K., Oegema, K., and Desai, A. (2011). Rapid <span class="italic">de novo</span> centromere formation occurs independently of heterochromatin protein 1 in <span class="italic">C. elegans</span> embryos. Curr. Biol. <span class="italic">21</span>, 1800-1807.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22018540&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2011.09.016" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed"><span class="bibliomisc">Yuzyuk, T., Fakhouri, T.H., Kiefer, J., and Mango, S.E. (2009). The polycomb complex protein <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003220;class=Gene" target="_blank">mes-2</a></span>/<span class="italic">E(z)</span> promotes the transition from developmental plasticity to differentiation in <span class="italic">C. elegans</span> embryos. Dev. Cell <span class="italic">16</span>, 699-710.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19460346&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.devcel.2009.03.008" target="_blank">Article</a>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Oliver Hobert Last revised May 23, 2013, Published April 2, 2014. This chapter should be cited as: Askjaer P.,
                  et al. Modern techniques for the analysis of chromatin and nuclear organization in <span class="emphasis"><em>C. elegans</em></span> (April 2, 2014), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.169.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2014 Peter Askjaer, Sevin&ccedil; Ercan, Peter Meister. This is an open-access article distributed under the terms of the Creative
                  Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original
                  author and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e42" href="#d0e42">&sect;</a></sup>To whom correspondence should be addressed: <a href="mailto:pask@upo.es" target="_top">pask@upo.es</a>, <a href="mailto:se71@nyu.edu" target="_top">se71@nyu.edu</a>, or <a href="mailto:peter.meister@izb.unibe.ch" target="_top">peter.meister@izb.unibe.ch</a></p>
 <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>