<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Forward and reverse mutagenesis in C. elegans</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WB_Kutscher_ShahamJM_text" title="Forward and reverse mutagenesis in C. elegans">
      <link rel="next" href="#sec1" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="frmutagenesis.pdf">Download PDF version</a>
	</div>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WB_Kutscher_ShahamJM_text"></a>Forward and reverse mutagenesis in <span class="emphasis"><em>C. elegans</em></span><sup><a name="d0e8" href="#ftn.d0e8">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Lena M. Kutscher<sup>1</sup> and Shai Shaham<sup>1</sup><em><span class="remark"><sup><a name="d0e37" href="#ftn.d0e37">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup>Laboratory of Developmental Genetics, The Rockefeller University, New York NY 10065, USA; </span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward and reverse genetics
		</a></div>
<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. Genome-wide mutagenesis</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec2-1">2.1. Strategies for forward genetic screens</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-2">2.2. Chemical mutagenesis</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-3">2.3. Radiation mutagenesis</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-4">2.4. Non-directed insertional mutagenesis using transposons</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec3">3. Target-selected mutagenesis</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec3-1">3.1. Target-selected mutagenesis using PCR</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-2">3.2. TILLING</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-3">3.3. G4 DNA-induced deletion mutagenesis</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec4">4. Gene-targeted mutagenesis</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec4-1">4.1. Transposon-based methods</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4-2">4.2. Zinc finger nucleases, transcription activator-like effector nucleases (TALENs), and the CRISPR/Cas system</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec5">5. Summary</a></span></dt>
               <dt><span class="sect1"><a href="#sec6">6. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#sec7">7. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>Mutagenesis drives natural selection. In the lab, mutations allow gene function to be deciphered. <span class="emphasis"><em>C. elegans</em></span> is highly amendable to functional genetics because of its short generation time, ease of use, and wealth of available gene-alteration
                     techniques. Here we provide an overview of historical and contemporary methods for mutagenesis in <span class="emphasis"><em>C. elegans</em></span>, and discuss principles and strategies for forward (genome-wide mutagenesis) and reverse (target-selected and gene-specific
                     mutagenesis) genetic studies in this animal.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>Mutations are the engine of natural selection and arise spontaneously through a myriad of mechanisms (<a href="#bib91">Nei, 2007</a>). In <span class="emphasis"><em>C. elegans</em></span>, spontaneous point mutations, gene duplications, and deletions arise at rates of 2 &times; 10<sup>&#8722;8</sup>, 3.4 &times; 10<sup>&#8722;7</sup>, and 2.2 &times; 10<sup>&#8722;7</sup> per gene per generation, respectively (<a href="#bib31">Denver et al., 2004</a>; <a href="#bib72">Lipinski et al., 2011</a>). These alterations to the genome result from replication and repair errors, endogenous oxidative damage, and exogenous environmental
               insults (<a href="#bib30">Denver et al., 2009</a>). Cosmic rays bombard the atmosphere of the Earth at a rate of approximately 2 &times; 10<sup>18</sup> particles per second (<a href="#bib96">Reedy et al., 1983</a>), and they can also contribute to mutational load (<a href="#bib4">Babcock and Collins, 1929</a>). <span class="emphasis"><em>C. elegans</em></span> has been used to study the effects of cosmic rays aboard the International Space Station (<a href="#bib128">Zhao et al., 2006</a>). Despite exposure to an average radiation dose of 10-12 mRad/day (approximately 150 times higher than exposure on the surface
               of the Earth (<a href="#bib92">Ohnishi and Ohnishi, 2004</a>)), no significant differences in mutation rates were detected over an 11-day period (<a href="#bib128">Zhao et al., 2006</a>), suggesting that DNA repair mechanisms must correct most mutations efficiently, and that spontaneous alterations to the
               genome may largely result from cell-intrinsic errors.
            </p>
            <p>Mutations are important tools for gene function discovery. Forward and reverse genetic strategies rely on mutations to identify
               and characterize genes involved in a biological process of interest. A causal connection between mutation and phenotype can
               suggest mechanisms of wild-type gene action and can be used for diagnostic prediction. While gene function can be interrogated
               through targeted knockdown of RNAs using RNA interference (RNAi) (<a href="#bib38">Fire et al., 1998</a>), identification of genomic lesions in genes has several advantages. For example, the functions of genes that are not susceptible
               to RNAi can be assessed. Special alleles of genes whose inactivation leads to lethality, or that function redundantly, can
               also be identified (<a href="#bib90">Muller, 1932</a>). Temperature sensitive alleles allow for temporal assessment of gene activity (<a href="#bib50">Hartwell et al., 1974</a>), and single-gene mutations defining an allelic series can be used to study protein structure-function relationships (<a href="#bib58">Jorgensen and Mango, 2002</a>). Finally, while RNAi experiments sometimes give variable results (<a href="#bib61">Kamath and Ahringer, 2003</a>), genomic lesions tend to produce consistent phenotypic changes more suitable for quantitative analysis.
            </p>
            <p>For spontaneous mutations to be an effective tool, roughly 1/(mutation rate) organisms must be screened to identify lesions
               in a particular gene. Often, one must screen well above this number. Although this is easily achieved in bacteria or yeast,
               the number is prohibitively high for other organisms, including <span class="emphasis"><em>C. elegans</em></span>. Fortunately, the work of <a href="#bib89">Muller (1927)</a> demonstrated that mutation rates can be artificially enhanced by a variety of genotoxic agents. Practically, such treatments
               must maintain a balance between administration of sufficient damage (to reduce the numbers of queried animals) and animal
               lethality/sterility. In <span class="emphasis"><em>C. elegans</em></span>, typical mutagenesis compatible with this balance inflicts 10<sup>&#8722;3</sup> to 10<sup>&#8722;4</sup> mutations per gene per generation (<a href="#bib44">Gengyo-Ando and Mitani, 2000</a>).
            </p>
            <p>This chapter provides an overview of historical and current methods for gene mutagenesis in <span class="emphasis"><em>C. elegans</em></span>. Three mutagenesis strategies are commonly used: genome-wide mutagenesis, in which unknown genes controlling a biological
               process are sought (forward genetics); target-selected mutagenesis, in which the entire genome is mutagenized but mutations
               in a known single gene are screened (reverse genetics); and gene-targeted mutagenesis, in which only a single gene is targeted
               for mutagenesis (reverse genetics). Method rationales and protocols are outlined, and are meant to serve as an introduction
               to aid researchers in identifying mutagenesis methods suitable to their goals (<a href="#table1" title="Table 1. Overview of mutagenesis methods. Three types of mutagenesis are discussed in this review: whole-genome, target-selected, and gene-targeted mutagenesis. The different types and variations of each method are given; a general purpose and references for each technique are listed. Advantages and disadvantages of each method are discussed further in the text.">Table 1</a>). Detailed protocols for each method are not presented for most methods, but are referenced and should be carefully followed
               when implemented in the laboratory.
            </p>
            <div class="table"><a name="table1"></a><p class="title">Table 1. <span class="bold">Overview of mutagenesis methods.</span> Three types of mutagenesis are discussed in this review: whole-genome, target-selected, and gene-targeted mutagenesis. The
                  different types and variations of each method are given; a general purpose and references for each technique are listed. Advantages
                  and disadvantages of each method are discussed further in the text.
               </p>
               <table summary="Table 1. Overview of mutagenesis methods. Three types of mutagenesis are discussed in this review: whole-genome, target-selected, and gene-targeted mutagenesis. The different types and variations of each method are given; a general purpose and references for each technique are listed. Advantages and disadvantages of each method are discussed further in the text." border="1">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <tbody>
                     <tr valign="bottom">
                        <td align="left" valign="bottom"><span class="bold"><strong>Method</strong></span></td>
                        <td align="left" valign="bottom"><span class="bold"><strong>Types (references)</strong></span></td>
                        <td align="left" valign="bottom"><span class="bold"><strong>Purposes</strong></span></td>
                     </tr>
                     <tr valign="top">
                        <td colspan="3" align="left" valign="top"><span class="bold"><strong>Genome-wide mutagenesis</strong></span></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">Chemical mutagenesis</td>
                        <td align="left" valign="top">
                           
                           <p>EMS (<a href="#bib15">Brenner, 1974</a>)
                           </p>
                           
                           <p>TMP/UV (<a href="#bib8">Barstead and Moerman, 2006</a>)
                           </p>
                           
                           <p>ENU (<a href="#bib29">De Stasio and Dorman, 2001</a>)
                           </p>
                           
                           <p>formaldehyde (<a href="#bib57">Johnsen and Baillie, 1988</a>)
                           </p>
                           
                           <p>NTG (<a href="#bib49">Greenwald and Horvitz, 1980</a>)
                           </p>
                           
                           <p>DES (<a href="#bib49">Greenwald and Horvitz, 1980</a>)
                           </p>
                           
                           <p>acetaldehyde (<a href="#bib49">Greenwald and Horvitz, 1980</a>)
                           </p>
                           
                           <p>DEO (<a href="#bib3">Anderson and Brenner, 1984</a>)
                           </p>
                           
                           <p>DEB (<a href="#bib115">Trent et al., 1991</a>)
                           </p>
                           
                        </td>
                        <td align="left" valign="top">Forward genetic screens, target-selected mutagenesis</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">Radiation mutagenesis</td>
                        <td align="left" valign="top">
                           
                           <p>Short wave UV (<a href="#bib109">Stewart et al., 1991</a>)
                           </p>
                           
                           <p>IR (<a href="#bib100">Rosenbluth et al., 1985</a>)
                           </p>
                           
                           <p><sup>32</sup>P decay (<a href="#bib5">Babu and Brenner, 1981</a>)
                           </p>
                        </td>
                        <td align="left" valign="top">Forward genetic screens, generate deficiencies</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">Transposon insertional mutagenesis</td>
                        <td align="left" valign="top">
                           
                           <p>Tc1 (<a href="#bib76">Martin et al., 2002</a>)
                           </p>
                           
                           <p><span class="emphasis"><em>Mos1</em></span> (<a href="#bib14">Boulin and Bessereau 2007</a>)
                           </p>
                        </td>
                        <td align="left" valign="top">Forward genetic screens, generate insertion mutants for use in gene-targeted mutagenesis</td>
                     </tr>
                     <tr valign="top">
                        <td colspan="3" align="left" valign="top"><span class="bold"><strong>Target-selected mutagenesis</strong></span></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">PCR-based methods</td>
                        <td align="left" valign="top">
                           
                           <p>restricted extension time (<a href="#bib54">Jansen et al., 1997</a>)
                           </p>
                           
                           <p>poison primer (<a href="#bib35">Edgley et al., 2002</a>)
                           </p>
                           
                           <p>thermostable restriction enzymes (<a href="#bib51">Huang et al., 2006</a>; <a href="#bib121">Wei et al., 2002</a>)
                           </p>
                        </td>
                        <td align="left" valign="top">Recover deletion in specific gene after whole genome mutagenesis</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">TILLING</td>
                        <td align="left" valign="top">(<a href="#bib45">Gilchrist et al., 2006</a>)
                        </td>
                        <td align="left" valign="top">Isolate point mutations/allelic series in a gene of interest</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">G4 DNA-induced mutations</td>
                        <td align="left" valign="top">(<a href="#bib94">Pontier et al., 2009</a>)
                        </td>
                        <td align="left" valign="top">Isolate deletion alleles for genes near G4 genomic site</td>
                     </tr>
                     <tr valign="top">
                        <td colspan="3" align="left" valign="top"><span class="bold"><strong>Gene-targeted mutagenesis</strong></span></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">Transposon-based methods</td>
                        <td align="left" valign="top">
                           
                           <p>Tc1 (<a href="#bib130">Zwaal et al., 1993</a>; <a href="#bib7">Barrett et al., 2004</a>)
                           </p>
                           
                           <p>MosTIC (<a href="#bib99">Robert, 2012</a>)
                           </p>
                           
                           <p>MosDEL (<a href="#bib42">Fr&oslash;kjaer-Jensen et al., 2010</a>; <a href="#bib41">Fr&oslash;kjaer-Jensen et al., 2012</a>)
                           </p>
                        </td>
                        <td align="left" valign="top">Make targeted changes or deletions in gene of interest without mutagenizing entire genome</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">Enzyme-based methods</td>
                        <td align="left" valign="top">
                           
                           <p>ZFNs (<a href="#bib125">Wood et al., 2011</a>)
                           </p>
                           
                           <p>TALENs (<a href="#bib125">Wood et al., 2011</a>)
                           </p>
                           
                           <p>CRISPR/Cas (<a href="#bib137">Friedland et al., 2013</a>; <a href="#bib136">Dickinson et al., 2013</a>)
                           </p>
                        </td>
                        <td align="left" valign="top">Create small indels or repair off a transgene to create mutations</td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Genome-wide mutagenesis
                     </h2>
                  </div>
               </div>
            </div>
            <p>Non-directed genome-wide mutagenesis is commonly used for forward genetic screens, where mutants defective in a biological
               process of interest are sought. Mutagenesis can be accomplished with a variety of chemical agents, each offering a specific
               array of favored lesions and mutation frequencies. Ionizing radiation mutagenesis is used to generate large deletions or complex
               rearrangements. Alternatively, mutagenesis is achieved with transposon insertion, using either the endogenous transposons
               already present in the nematode or an exogenous <span class="emphasis"><em>Mos1</em></span> transposon.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-1"></a>2.1.&nbsp;Strategies for forward genetic screens
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Prior to embarking on genome-wide mutagenesis, a screen must be designed to identify mutant animals defective in the biological
                  process under study. A thorough review of the various types of genetic screens in <span class="emphasis"><em>C. elegans</em></span> has been previously published and should be consulted (<a href="#bib58">Jorgensen and Mango, 2002</a>). The most common screen involves identifying recessive mutations exposed in second-generation (F2) animals of a mutagenized
                  parental strain (P0, <a href="#figure1">Figure 1</a>). Typically, P0 hermaphrodites are mutagenized at the late L4 or early adult stages. F1 progeny, heterozygous for many induced
                  mutations, are then allowed to self, and F2 animals are screened for the phenotype of interest. Many variations on this screen
                  exist, and each screen should be carefully constructed to find enough mutants for the phenotype of interest (<a href="#bib58">Jorgensen and Mango, 2002</a>). Because some genetic screens can be demanding in effort and time, it is prudent to optimize the ratio of F2-to-F1 animals
                  assayed (<a href="#bib105">Shaham, 2007</a>). Calculations for the amount of effort required per screen center on whether F1 or F2 animals must be plated individually.
                  We previously set up a web calculator to aid in evaluating optimal screen parameters (<a href="http://shahamlab.rockefeller.edu/cgi-bin/Genetic_screens/screenfrontpage.cgi" target="_top">http://shahamlab.rockefeller.edu/cgi-bin/Genetic_screens/screenfrontpage.cgi</a>). A detailed description of the theoretical considerations behind this calculator is available (<a href="#bib105">Shaham, 2007</a>). There are also techniques to automate and expedite a forward genetic screen, including the Complex Object Parametric Analyzer
                  and Sorter (COPAS&#8482;), a device that sorts animals into individual wells, and microfluidic devices coupled to automated imaging
                  (<a href="#bib20">Chung et al., 2008</a>; <a href="#bib28">Crane et al., 2012</a>; <a href="#bib33">Doitsidou et al., 2008</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

 <tr>
  <td align="center"><p><a href="frmutagenesis_fig1.jpg"><img src="frmutagenesis_fig1_s.jpg" border="2" align="middle" alt="figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. A simple F2 screen.&nbsp;</b>Depicted here is the simplest variation of an F2 screen, which involves isolating mutants with a phenotype of interest in
                        the second generation after mutagenesis. A population of young L4 hermaphrodites is mutagenized. F1 animals are collected,
                        and F2 single hermaphrodites are isolated, carrying a homozygous mutation in a gene of interest. Obtaining a mutation is a
                        rare event. The details of the scheme depend on the type of mutation sought. For a detailed description of different varieties
                        of F2 screens, consult <a href="#bib58">Jorgensen and Mango (2002)</a>.
                     </p>
                  </div>
               </div>
               <p>Once a mutant is identified, several strategies can be used to isolate the affected gene. In the past, two- and three-point
                  mapping with genetic markers was used to identify the affected region, although this method is not commonly employed today
                  (for a review, see WormBook chapter <a href="http://dx.doi.org/10.1895/wormbook.1.165.1" target="_top">Classical genetic methods</a>). Single nucleotide polymorphism (SNP) mapping using the polymorphic Hawaiian isolate CB4856 is a more common approach (<a href="#bib122">Wicks et al., 2001</a>). Oligonucleotide array comparative genome hybridization (aCGH) has also been used successfully to identify point mutations
                  and deletions in complex pools of genomic DNA (<a href="#bib77">Maydan et al., 2007</a>; <a href="#bib78">Maydan et al., 2009</a>). In this case, custom-made arrays with overlapping 50-mer probes are used to identify SNPs and insertions or deletions (indels).
                  Whole-genome sequencing is now a cost-effective approach to gene mapping, and it can be combined with other mapping methods
                  for rapid gene identification (<a href="#bib34">Doitsidou et al., 2010</a>; <a href="#bib102">Sarin et al., 2008</a>; <a href="#bib129">Zuryn et al., 2010</a>). There is open-source software to aid in whole-genome sequencing data analysis and mutant identification, including Bowtie,
                  CloudMap, and galign, which are specifically tailored to the <span class="emphasis"><em>C. elegans</em></span> genome (<a href="#bib67">Langmead et al., 2009</a>; <a href="#bib66">Langmead and Salzberg, 2012</a>; <a href="#bib82">Minevich et al., 2012</a>; <a href="#bib106">Shaham, 2009</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-2"></a>2.2.&nbsp;Chemical mutagenesis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Chemical mutagenesis provides the easiest and most straightforward way to induce germline mutations at high frequency. Mutation
                  prevalence using chemicals is high enough that a relatively low number of haploid genomes must be interrogated to reach saturation.
                  The most widely used mutagens in <span class="emphasis"><em>C. elegans</em></span> are ethyl methanesulfonate (EMS) and trimethylpsoralen with ultraviolet light (UV/TMP). There are, however, many others with
                  differing mutagenic spectra, including N-ethyl-N-nitrosourea (ENU), formaldehyde, nitrosoguanidine (NTG), diethyl sulfate
                  (DES), acetaldehyde, diepoxybutane (DEB) and diepoxyoctane (DEO). A cocktail of EMS/ENU has also been used to induce a spectrum
                  of mutations (<a href="#bib111">Thompson et al., 2013</a>). Special care should be taken when handling mutagens: working in a chemical hood and wearing proper personal protection
                  gear are a must. Consult the relevant Material Safety Data Sheet (MSDS) and your chemical waste department before beginning
                  a chemical genetic screen.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-2-1"></a>2.2.1.&nbsp;Ethyl methanesulfonate (EMS)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>EMS is the most commonly used mutagen in <span class="emphasis"><em>C. elegans</em></span> and by far the most potent (<a href="#bib15">Brenner, 1974</a>; <a href="#bib39">Flibotte et al., 2010</a>). A forward mutation rate as high as 2.5 &times; 10<sup>&#8722;3</sup> mutations/gene/generation can be achieved with acceptable rates of sterility (<a href="#bib44">Gengyo-Ando and Mitani, 2000</a>). EMS is an alkylating agent and most commonly adds an ethyl group to guanine to form O<sup>6</sup>-ethylguanine (<a href="#bib16">Brookes and Lawley, 1961</a>). This modified guanine pairs inappropriately with thymine in subsequent replications (<a href="#bib27">Coulondre and Miller, 1977</a>). Therefore, mutagenesis is biased towards G/C to A/T transitions, respectively, which often leads to the generation of stop
                     codons (<a href="#bib39">Flibotte et al., 2010</a>). Therefore, these mutations are typically strong loss-of-function or null alleles. Roughly 13% of EMS lesions are deletions
                     or other chromosomal rearrangements (<a href="#bib2">Anderson, 1995</a>).
                  </p>
                  <p><span class="bold"><strong><span class="emphasis"><em>Protocol</em></span></strong></span> (<a href="#bib15">Brenner, 1974</a>):
                  </p>
                  <p>Synchronized L4 or young-adult hermaphrodites are washed with M9 buffer in a sterile 15 ml plastic tube. Spin down animals
                     at 700 &times; <span class="emphasis"><em>g</em></span> for 1 min and remove supernatant. Wash animals at least 3x to remove bacteria. Resuspend animals in 2 ml M9. Add 2 ml of
                     a 2x stock solution of EMS (100 mM). Animals are incubated in a total volume of 4 ml of 50 mM EMS in M9 for 4 h at 20<sup>o</sup>C with constant rotation. EMS should be prepared immediately before use. Animals are then washed 5x with 15 ml M9 to remove
                     trace EMS. For standard F2 screens, plate mutagenized animals using a glass pipette on large NGM plates until F1 animals are
                     gravid. Bleach F1 animals, and plate F2 eggs on NGM plates. Score for the defects of interest at the appropriate developmental
                     stage. EMS is detoxified by addition of 1 M NaOH.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-2-2"></a>2.2.2.&nbsp;Trimethylpsoralen (TMP)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>TMP is a light-sensitive interstrand cross-linking agent, whose mutagenicity is enhanced in the presence of long-wave UV light
                     (<a href="#bib22">Cole, 1970</a>). UV/TMP generates small deletions with an average size of one to three kilobases; however, all base transitions and transversions
                     have been observed experimentally (<a href="#bib39">Flibotte et al., 2010</a>; <a href="#bib44">Gengyo-Ando and Mitani, 2000</a>; <a href="#bib126">Yandell et al., 1994</a>). The forward mutation rate is approximately 1 &times; 10<sup>&#8722;3</sup> mutations/gene/generation, but it depends on total UV exposure (<a href="#bib44">Gengyo-Ando and Mitani, 2000</a>).
                  </p>
                  <p><span class="bold"><strong><span class="emphasis"><em>Protocol</em></span></strong></span> (<a href="#bib8">Barstead and Moerman, 2006</a>):
                  </p>
                  <p>Synchronized L4 animals are washed 5x in 15 ml M9 to remove bacteria. Animals are then resuspended in 580 &micro;l M9 buffer. Twenty
                     microliters of freshly made TMP (1 mg/ml in DMSO) is added to the culture, and animals are incubated in the dark for 15 min.
                     Pipette animals onto dry, unseeded, 9 cm NGM plates. Remove plate lids, and apply 360 &micro;J UV light. A Stratalinker<sup>&reg;</sup> UV Crosslinker (Stratagene) is useful for this step, but calibrated hand-held UV lamps can be used as well (<a href="#bib126">Yandell et al., 1994</a>). Next, add OP50 directly on top of the mutagenized worms and recover for 5 h in the dark at 20<sup>o</sup>C. Continue as for EMS screens. A range of final concentrations of TMP (2 &micro;g/ml to 30 &micro;g/ml) is reported to be successful.
                     It is likely that variations depend on total UV exposure and variability between batches of TMP. Dose-response curves can
                     be used to optimize mutagenesis for each batch (<a href="#bib8">Barstead and Moerman, 2006</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-2-3"></a>2.2.3.&nbsp;N-ethyl-N-nitrosourea (ENU)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>ENU generates an array of molecular lesions, and it is therefore the mutagen of choice if allelic variability is important
                     (<a href="#bib39">Flibotte et al., 2010</a>). The mutagenicity of ENU occurs by transferring its ethyl moiety to nitrogen or oxygen on any of the four nucleotides, creating
                     modified DNA bases that later hinder proper replication (<a href="#bib47">Goth and Rajewsky, 1972</a>; <a href="#bib59">Justice et al., 1999</a>). All codons are equally mutable with ENU, so non-null alleles are often generated (<a href="#bib29">De Stasio and Dorman, 2001</a>). ENU has a slightly higher reversion frequency at around 2.6 &times; 10<sup>&#8722;4</sup>, compared to 1.1 &times; 10<sup>&#8722;4</sup> using EMS (<a href="#bib29">De Stasio and Dorman, 2001</a>). The reversion frequency is the frequency at which a mutated phenotype reverts back to wild type. This is commonly a result
                     of a second site suppressor mutation. Because there are usually fewer suppressor genes, this frequency is lower than the forward
                     mutation rate (<a href="#bib49">Greenwald and Horvitz, 1980</a>).
                  </p>
                  <p><span class="bold"><strong><span class="emphasis"><em>Protocol</em></span></strong></span> (<a href="#bib29">De Stasio and Dorman, 2001</a>):
                  </p>
                  <p>ENU degrades to more toxic products when stored in aqueous solutions (<a href="#bib29">De Stasio and Dorman, 2001</a>). Stock solutions of 50 mM in absolute ethanol should therefore be kept at -20<sup>o</sup>C for no more than 2 weeks. Here it is useful to buy small aliquots of dry ENU and add ethanol directly to the bottle to make
                     the 50 mM solution (M. Edgley, personal communication). Working solutions (1.0 mM diluted in modified M9 buffer lacking ammonium
                     chloride) are made fresh immediately before use (<a href="#bib29">De Stasio and Dorman, 2001</a>). Synchronized late L4/young adult hermaphrodites are incubated with the working solution for 4 h at 20&deg;C with constant rotation.
                     Animals are then washed 5x in M9 before plating on seeded NGM plates. Before discarding, ENU should be inactivated by adding
                     sodium thiosulfate and sodium hydroxide to a final concentration of 10% (w/v) and 1% (w/v), respectively, and incubated at
                     room temperature for 1 h.
                  </p>
                  <p><span class="emphasis"><em>EMS/ENU cocktail Protocol</em></span>: The Million Mutation Project has used a cocktail of EMS and ENU to isolate mutations with a spectrum of lesions (<a href="#bib111">Thompson et al., 2013</a>). The same guidelines for each mutagen are used to prepare the mutagenic solution, and the final concentrations for mutagenesis
                     are 50 mM EMS and 1 mM ENU in M9 (M. Edgley, personal communication). Synchronized L4/young adult hermaphrodites are washed
                     into a 15 ml sterile tube. The animals are resuspended in 2 ml M9, and 2 ml of a 2x stock solution containing 100 mM EMS and
                     2 mM ENU are added. This solution is prepared fresh every time before use. The ENU stock solution (50 mM in absolute ethanol)
                     is prepared and stored the same way as described for ENU mutagenesis.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-2-4"></a>2.2.4.&nbsp;Other chemical mutagens
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The following mutagens are used less often, but they can also be considered for a forward genetic screen. The mutation rates
                     given are rough estimates, as they are usually taken from only one or a few published screens. A thorough review of these
                     uncommon chemical mutagens has been previously published (<a href="#bib2">Anderson, 1995</a>). In all cases, synchronized L4/young adult hermaphrodites are used. Following treatment, mutagenized animals are washed
                     5x in M9 buffer and plated on seeded NGM plates, according to the screen strategy.
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p><span class="underline">Formaldehyde</span> generates deficiencies, as well as point mutations and complex lesions, but it is less effective than EMS (<a href="#bib84">Moerman and Baillie, 1981</a>). Working concentrations of 0.1% (w/v) provide the best trade-off between mutagenic capacity and toxicity (<a href="#bib57">Johnsen and Baillie, 1988</a>). Stock solutions (1% (w/v)) are stored at &#8722;20&deg;C in M9. Hermaphrodites are subjected to mutagenesis for 4 h at 20&deg;C, rotating.
                           </p>
                        </li>
                        <li>
                           <p><span class="underline">Nitrosoguanidine</span> (NTG) is an alkylating agent that most likely causes point mutations by mispairing of bases during DNA replication (<a href="#bib68">Lawley and Thatcher, 1970</a>). It is used at a concentration of 0.5 mg/ml to mutagenize animals at a rate of 1 &times; 10<sup>&#8722;4</sup> mutations per haploid genome (<a href="#bib49">Greenwald and Horvitz, 1980</a>). This rate is about ten times lower than that of EMS. In other organisms, however, NTG is a potent mutagen, suggesting that
                              the best conditions for mutagenesis in <span class="emphasis"><em>C. elegans</em></span> have not yet been determined (<a href="#bib2">Anderson, 1995</a>). Animals are incubated at 20&deg;C for 1 h (<a href="#bib49">Greenwald and Horvitz, 1980</a>).
                           </p>
                        </li>
                        <li>
                           <p><span class="underline">Diethyl sulfate</span> (DES) is a mono-functional alkylating agent, and it has an approximate mutation rate of 5 &times; 10<sup>&#8722;4</sup> mutations per haploid genome (<a href="#bib49">Greenwald and Horvitz, 1980</a>). Based on its chemical structure, it most likely causes point mutations (<a href="#bib16">Brookes and Lawley, 1961</a>). The precise spectrum of lesions, however, is unknown (<a href="#bib49">Greenwald and Horvitz, 1980</a>). Animals are incubated for 2 h in freshly made 1 mM DES diluted in M9 buffer at 20&deg;C (<a href="#bib49">Greenwald and Horvitz, 1980</a>).
                           </p>
                        </li>
                        <li>
                           <p><span class="underline">Acetaldehyde</span> mutagenizes animals at a rate of approximately 1 &times; 10<sup>&#8722;4</sup> mutations per haploid genome. Animals are incubated for 2 h at 20&deg;C in 0.1% acetaldehyde in M9 (<a href="#bib49">Greenwald and Horvitz, 1980</a>).
                           </p>
                        </li>
                        <li>
                           <p><span class="underline">1,2,7,8-diepoxyoctane</span> (DEO) is a bi-functional alkylating agent that can generate large deletions, and it has an approximate mutation rate of 5
                              &times; 10<sup>&#8722;5</sup> mutations per haploid genome. Hermaphrodites are incubated for 3 h in a freshly-made solution of 2 mM DEO in M9 (<a href="#bib3">Anderson and Brenner, 1984</a>).
                           </p>
                        </li>
                        <li>
                           <p><span class="underline">1,2,3,4-diepoxybutane</span> (DEB) has a similar reversion frequency to EMS. 15% of lesions are small deletions of 0.5-3.5 kb. Animals are incubated in
                              1 mM DEB in M9 buffer for 3 h at 20&deg;C (<a href="#bib115">Trent et al., 1991</a>).
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-3"></a>2.3.&nbsp;Radiation mutagenesis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>High-energy radiation mutagenesis is not commonly employed for contemporary forward genetic screens. Researchers should, however,
                  consider such a method if large deletions or complex rearrangements are desired at a relatively high frequency (<a href="#bib89">Muller, 1927</a>).
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-3-1"></a>2.3.1.&nbsp;Short-wave UV
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Short-wave UV can induce chromosomal rearrangements, including deletions, translocations and duplications. DNA absorbs UV
                     light, which causes adjacent pyrimidines to dimerize (<a href="#bib40">Freeman et al., 1989</a>). Lesions are then induced following inaccurate trans-lesion synthesis during replication (<a href="#bib6">Bailly et al., 1994</a>). UV mutagenesis has been used successfully to generate mutations at a rate of 2 &times; 10<sup>&#8722;4</sup> mutations per haploid genome per generation (<a href="#bib49">Greenwald and Horvitz, 1980</a>). However, another study found that UV-C radiation had a low rate of mutation with a decrease in fertility, so more hermaphrodites
                     should be used initially (<a href="#bib44">Gengyo-Ando and Mitani, 2000</a>). The reduction in brood size after irradiation was as high as 20% compared to unirradiated animals (<a href="#bib109">Stewart et al., 1991</a>). Young adult animals on uncovered, unseeded plates are irradiated with 120 W/m<sup>2</sup> at 254 nm. Irradiated animals are allowed to expel eggs that were present prior to mutagenesis and are moved to new plates
                     after 2 h (<a href="#bib109">Stewart et al., 1991</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-3-2"></a>2.3.2.&nbsp;Ionizing radiation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Ionizing radiation (IR) includes X- and &#947;-rays and generates very large deletions that may span several genes at rates of
                     2 &times; 10<sup>&#8722;4</sup> to 9 &times; 10<sup>&#8722;4</sup> (<a href="#bib2">Anderson, 1995</a>; <a href="#bib80">Meneely and Herman, 1979</a>; <a href="#bib107">Sigurdson et al., 1984</a>). IR also induces complex rearrangements including chromosomal duplications, translocations, and inversions. The lesions
                     most likely differ from UV-induced lesions, as IR appears to have no nucleotide bias (<a href="#bib109">Stewart et al., 1991</a>). Synchronized L4/young adult hermaphrodites are irradiated with 1500 Roentgen and plated as needed. This method can recover
                     non-lethal complex lesions (<a href="#bib100">Rosenbluth et al., 1985</a>). Alternatively, an F1 clonal screen can be used to isolate lethal alterations.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-3-3"></a>2.3.3.&nbsp;<sup>32</sup>P decay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Sydney Brenner used <sup>32</sup>P decay to generate some of the first <span class="emphasis"><em>C. elegans</em></span> mutants (<a href="#bib5">Babu and Brenner, 1981</a>; <a href="#bib15">Brenner, 1974</a>). Mutations are generated at rates of 2.8 &times; 10<sup>&#8722;2</sup> to 3.9 &times; 10<sup>&#8722;3</sup> per haploid genome. <sup>32</sup>P is incorporated uniformly into genomic DNA, and mutation is caused by <sup>32</sup>P-transmutation in the genome, rather than by &#946;-particle emission (<a href="#bib5">Babu and Brenner, 1981</a>). <sup>32</sup>P is mixed with unlabeled phosphate and added directly to phosphate-free NGM plates at 10 Ci/mol (not a typo)! The plates
                     are seeded with NA22, a radioactive-resistant strain of bacteria (<a href="#bib2">Anderson, 1995</a>). Freshly hatched larvae are added to the radioactive plates and grown at 15&deg;C for 5 days. F1 animals are collected and plated
                     on standard NGM plates seeded with OP50 as above (<a href="#bib5">Babu and Brenner, 1981</a>). As a considerable amount of radioactive waste is generated using this method, consult your institute's radioactive waste
                     management facility prior to beginning.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-4"></a>2.4.&nbsp;Non-directed insertional mutagenesis using transposons
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Transposons are DNA sequences that hop in and out of a genome with the help of a transposase enzyme, usually encoded by the
                  transposon itself. In <span class="emphasis"><em>C. elegans</em></span>, there are six active endogenous transposons, Tc1-Tc5 and Tc7 (<a href="#bib23">Collins and Anderson, 1994</a>; <a href="#bib24">Collins et al., 1989</a>; <a href="#bib36">Emmons et al., 1983</a>; <a href="#bib70">Levitt and Emmons, 1989</a>; <a href="#bib71">Liao et al., 1983</a>; <a href="#bib97">Rezsohazy et al., 1997</a>; <a href="#bib127">Yuan et al., 1991</a>). Tc1 is the most enriched transposon in the N2 strain, with approximately 30 copies (<a href="#bib36">Emmons et al., 1983</a>). Transposons are not normally active in the germline of N2, but they may become active when genes regulating RNAi and transposition
                  are mutated (<a href="#bib25">Collins et al., 1987</a>; <a href="#bib118">Vastenhouw et al., 2003</a>). Fortunately, an exogenous transposon, <span class="emphasis"><em>Mos1</em></span>, is active in the N2 germline in the presence of <span class="emphasis"><em>Mos1</em></span> transposase (<a href="#bib11">Bessereau et al., 2001</a>). Endogenous transposons and <span class="emphasis"><em>Mos1</em></span> insert randomly into the genome when activated in the appropriate background. Importantly, after insertion and following
                  back-crossing, the transposon provides a molecular tag by which the disrupted gene can be identified using inverse (vectorette)
                  PCR or whole-genome sequencing (<a href="#figure2">Figure 2</a>) (<a href="#bib108">Smith, 2011</a>; <a href="#bib124">Williams et al., 2005</a>). For a comprehensive review, consult Wormbook chapter, <a href="http://www.wormbook.org/chapters/www_transposons/transposons.html" target="_top">Transposons in <span class="emphasis"><em>C. elegans</em></span></a>.
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

 <tr>
  <td align="center"><p><a href="frmutagenesis_fig2.jpg"><img src="frmutagenesis_fig2_s.jpg" border="2" align="middle" alt="figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. An insertional mutagenesis screen for isolating genes disrupted by a transposon.&nbsp;</b>In insertional mutagenesis, transposons (Tc1, and more commonly <span class="emphasis"><em>Mos1</em></span>) are mobilized in an appropriate strain background (mutator background or strain expressing <span class="emphasis"><em>Mos1</em></span>-transposase). The transposon (orange) inserts randomly throughout the genome, via the transposase (gray), typically at a
                        TA dinucleotide. DNA is isolated from animals that exhibit the phenotype of interest. Using inverse PCR, the DNA is cut with
                        a restriction enzyme (RE) that is abundant throughout the genome and self-ligated. PCR is performed using two primers (A and
                        B) within the transposon, elongating in opposite directions. PCR products (final panel) are size selected and sequenced to
                        determine the position of insertion.
                     </p>
                  </div>
               </div>
               <p>While generating mutations by chemical or radiation-induced mutagenesis is simple, identification of the relevant molecular
                  lesion may not be. Major advances in SNP mapping and whole genome sequencing have made gene identification easier, yet these
                  methods can be time consuming and labor intensive. This is especially the case when weakly penetrant or behavioral mutants
                  are studied. In both cases, phenotype can only be established by scoring populations of animals, often requiring F3 or F4
                  animals to be evaluated. In these cases, non-directed insertional mutagenesis using transposons becomes a useful alternative.
               </p>
               <p>There are several caveats to consider before performing a genome-wide screen using insertional mutagenesis. The Bristol reference
                  strain N2 has 30 copies of Tc1 already present in the background, complicating identification of the causal lesion (<a href="#bib36">Emmons et al., 1983</a>). Both Tc1 and <span class="emphasis"><em>Mos1</em></span> have a bias for TA insertion sites; this makes transposons prone to inserting into introns, which may not inactivate the
                  nearby gene or may be spliced out during transcription (<a href="#bib48">Granger et al., 2004</a>; <a href="#bib85">Mori et al., 1988</a>; <a href="#bib101">Rushforth and Anderson, 1996</a>). <span class="emphasis"><em>Mos1</em></span> has about a 10x lower mutation rate than EMS, meaning that ten times as many animals need to be scored to identify a phenotype
                  of interest (<a href="#bib124">Williams et al., 2005</a>). Some mutations also occur as a result of &#8216;hit-and-run&#8217; events, where transposon insertion and subsequent removal causes
                  small indels to disrupt a gene (<a href="#bib14">Boulin and Bessereau, 2007</a>; <a href="#bib124">Williams et al., 2005</a>). These events greatly complicate gene identification. Because of these disadvantages, transposon-based insertional mutagenesis
                  screening is used less often. <span class="emphasis"><em>Mos1-</em></span>based methods are preferred. There are Tc1-derived alleles available from the CGC, so the descriptions below provide an overview
                  of how those alleles were obtained.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-4-1"></a>2.4.1.&nbsp;Tc1-mediated transposition
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>C. elegans</em></span> strains carry endogenous transposons, which have been used to perform a whole-genome insertional mutagenesis screen. Of the
                     six endogenous, active transposons, Tc1 and Tc3 are screened most often, as these have the highest copy number in N2. All
                     transposons, however, are active in most mutator backgrounds (<a href="#bib76">Martin et al., 2002</a>). Tc1 uses TA as an integration site, but with a relaxed target site specificity (<a href="#bib85">Mori et al., 1988</a>).
                  </p>
                  <p>To activate transposition, special mutator strains are used. The most common mutator genes are <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mut-2;class=Gene" target="_blank">mut-2</a>, <a href="http://www.wormbase.org/db/get?name=mut-7;class=Gene" target="_blank">mut-7</a>,</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mut-6;class=Gene" target="_blank">mut-6</a></em></span> (<a href="#bib25">Collins et al., 1987</a>; <a href="#bib62">Ketting et al., 1999</a>; <a href="#bib110">Tabara et al., 1999</a>)<span class="emphasis"><em>. <a href="http://www.wormbase.org/db/get?name=rde-2;class=Gene" target="_blank">rde-2</a>, <a href="http://www.wormbase.org/db/get?name=mut-14;class=Gene" target="_blank">mut-14</a>,</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mut-16;class=Gene" target="_blank">mut-16</a></em></span> are also available (<a href="#bib85">Mori et al., 1988</a>; <a href="#bib112">Tijsterman et al., 2002</a>; <a href="#bib118">Vastenhouw et al., 2003</a>). Because these mutator strains have high rates of transposon hopping, they tend to accumulate mutations quickly, and they
                     should be freshly thawed from frozen stocks rather than propagated on plates (<a href="#bib2">Anderson, 1995</a>).
                  </p>
                  <p><span class="bold"><strong><span class="emphasis"><em>Protocol</em></span></strong></span> (<a href="#bib76">Martin et al., 2002</a>):
                  </p>
                  <p>Prior to screening, a mutator strain is backcrossed four times to the strain that will be mutagenized. Outcrossing limits
                     the number of Tc1 copies already present in the mutator strain and removes any extraneous mutations prior to screening. Animals
                     are allowed to grow for ten generations to accumulate mutations and are then screened for defects of interest (<a href="#bib76">Martin et al., 2002</a>). Once the phenotype is confirmed, strains are immediately outcrossed to the parental strain. This removes the mutator locus
                     and prevents transposon excision from the gene of interest. Transposon insertion sites are identified similar to <span class="emphasis"><em>Mos1</em></span> identification below<span class="emphasis"><em>.</em></span> Since there are multiple copies of Tc1 in the background, SNP mapping is helpful in determining the chromosome of interest
                     and reducing the possible number of candidate genes. As is the case for <span class="emphasis"><em>Mos1</em></span>, most insertions occur in intergenic regions and introns, but a small percentage also appear in exons (<a href="#bib117">van der Linden and Plasterk, 2004</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-4-2"></a>2.4.2.&nbsp;<span class="emphasis"><em>Mos1</em></span>-mediated transposition
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>Mos1</em></span> was originally isolated from <span class="emphasis"><em>Drosophila mauritiana</em></span> (<a href="#bib53">Jacobson et al., 1986</a>). The consensus insertion site is the dinucleotide TA, in the context of an ATATAT sequence. <span class="emphasis"><em>Mos1</em></span>, however, can insert in sequences lacking the consensus site. 50% of insertions map near or within genes (<a href="#bib48">Granger et al., 2004</a>). Intron insertion bias is observed, presumably because these tend to be T/A rich; however, 20% of insertions still occur
                     within coding regions (<a href="#bib48">Granger et al., 2004</a>). Mutagenesis rates are an order of magnitude lower than chemical mutagenesis. As a result, ten times as many F1 animals
                     must be screened to identify lesions in a given gene (<a href="#bib124">Williams et al., 2005</a>). It is advisable to start with a small pilot screen using EMS first. This determines an approximate number of mutants expected
                     and ensures that an insertional screen is feasible (<a href="#bib14">Boulin and Bessereau, 2007</a>). Additionally, 6-15% of transposon-induced mutations arise through a &#8216;hit-and-run&#8217; mechanism, in which the transposon imprecisely
                     excises before the transposase becomes inactive (<a href="#bib14">Boulin and Bessereau, 2007</a>; <a href="#bib124">Williams et al., 2005</a>). On average, 2.5 insertions are generated per line, but as many as ten insertions in a single strain have been detected
                     (<a href="#bib14">Boulin and Bessereau, 2007</a>). <span class="emphasis"><em>Mos1</em></span> has a length of 1.3 kb, so alleles generated tend to be strong loss-of-function. Hypomorphs can be generated if the transposon
                     is inserted into regulatory sequences (<a href="#bib14">Boulin and Bessereau, 2007</a>).
                  </p>
                  <p><span class="bold"><strong><span class="emphasis"><em>Protocol</em></span></strong></span> (<a href="#bib14">Boulin and Bessereau, 2007</a>):
                  </p>
                  <p>The <span class="emphasis"><em>Mos1</em></span> transposase and transposon array are maintained as extrachromosomal arrays in two different strains (EG2762 and EG1470).
                     These are maintained as separate stocks to block spontaneous transposon insertions. EG2762 carries the <span class="emphasis"><em>Mos1</em></span> transposase under the control of a heat-inducible promoter, and a coelomocyte GFP marker. EG1470 has a pharynx GFP marker
                     and the <span class="emphasis"><em>Mos1</em></span> transposon. Immediately prior to the start of the screen, the two strains are crossed to generate doubly transgenic animals.
                     After one generation, the doubly transgenic strain is split into two populations. One population is used to test transposition
                     frequency following heat shock to ensure that silencing of the transposon array in the germline has not occurred. The other
                     is amplified for an additional 2-4 generations. Once transposition has been confirmed, the amplified doubly transgenic animals
                     are subjected to a heat pulse in a 33&deg;C water bath for 1 h, followed by 1 h incubation at 20&deg;C, followed by another 1 h heat
                     shock. Animals are recovered at 20&deg;C for 12 h, transferred onto fresh plates and grown at 20&deg;C. F1 progeny are picked 24 h
                     to 36 h after heat shocks. For a comprehensive protocol and useful tips, see (<a href="#bib14">Boulin and Bessereau, 2007</a>).
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;Target-selected mutagenesis
                     </h2>
                  </div>
               </div>
            </div>
            <p>Target-selected mutagenesis is used to identify mutations in a specific gene from a randomly mutagenized genome. The approach
               involves standard mutagenesis using EMS or UV/TMP, for example, followed by screening using the polymerase chain reaction
               (PCR) or other method to identify animals carrying lesions in the gene of interest.
            </p>
            <p>Two community consortia, the <span class="emphasis"><em>C. elegans</em></span> Knockout Consortium (<a href="http://celeganskoconsortium.omrf.org" target="_top">celeganskoconsortium.omrf.org</a>), and the <span class="emphasis"><em>C. elegans</em></span> National Bioresource Project of Japan (NBRP, <a href="http://www.shigen.nig.ac.jp/c.elegans/mutants/index.jsp" target="_top">http://www.shigen.nig.ac.jp/c.elegans/mutants/index.jsp</a>) are currently using target-selected mutagenesis to identify mutant alleles for every gene in the <span class="emphasis"><em>C. elegans</em></span> genome (<span class="emphasis"><em>C. elegans</em></span> <a href="#bib17">Deletion Mutant Consortium, 2012</a>). Together, over 6,700 deletion or null alleles have been isolated by these groups and are available to the scientific community
               (<span class="emphasis"><em>C. elegans</em></span> <a href="#bib17">Deletion Mutant Consortium, 2012</a>). Strains from the <a href="celeganskoconsortium.omrf.org" target="_top"><span class="emphasis"><em>C. elegans</em></span> Knockout Consortium</a> are available from the <span class="emphasis"><em>Caenorhabditis</em></span> Genetics Center (CGC, <a href="http://www.cbs.umn.edu/research/resources/cgc" target="_top">http://www.cbs.umn.edu/research/resources/cgc</a>) under the allele designation <span class="emphasis"><em>ok</em></span> or <span class="emphasis"><em>gk.</em></span> Strains generated by the <a href="http://www.shigen.nig.ac.jp/c.elegans/mutants/index.jsp" target="_top">NBRP</a>, with allele designation <span class="emphasis"><em>tm</em></span>, are available after submission of a Materials Transfer Agreement (<a href="#bib83">Mitani, 2009</a>). Deletion strains are listed on <a href="http://www.wormbase.org/" target="_top">Wormbase</a>, which should be consulted first to determine if an allele is already available for a gene of interest. If not, one may request
               a deletion strain from the Consortium. Upon receiving a strain, it should be outcrossed to remove irrelevant mutations. It
               is always important to verify the presence of the mutation in a strain. In addition, deletion mutations obtained by target-selection
               can be accompanied by proximal duplications, resulting in obligate heterozygosity for the gene under consideration. In these
               situations, phenotypic assessment of the mutant strain may not reveal the defects associated with complete gene loss. To confirm
               that a gene of interest is responsible for an observed defect, transgenic rescue experiments should be performed and/or multiple
               alleles examined (<span class="emphasis"><em>C. elegans</em></span> <a href="#bib17">Deletion Mutant Consortium, 2012</a>). In addition to the collection of deletions, the Million Mutations Project (<a href="http://genome.sfu.ca/mmp/" target="_top">http://genome.sfu.ca/mmp/</a>) has identified over 800,000 single nucleotide polymorphisms (SNPs) and over 16,000 indels in more than 20,000 genes (<span class="emphasis"><em>C. elegans</em></span> <a href="#bib17">Deletion Mutant Consortium, 2012</a>; <a href="#bib111">Thompson et al., 2013</a>). While some of these SNPs do not affect gene function, many may provide useful structure-function data.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-1"></a>3.1.&nbsp;Target-selected mutagenesis using PCR
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The goal of target-selected mutagenesis is to generate a library containing pools of mutagenized animals that can be screened
                  for specific DNA lesions. Deletion alleles are easier to identify using PCR than point mutations, so UV/TMP and EMS are the
                  preferred mutagens (<a href="#bib69">Lesa, 2006</a>). For library construction, 600,000 hermaphrodites are randomly mutagenized. Once the mutagenized animals become gravid,
                  they are bleached and L1s are allowed to hatch in M9, overnight. The L1s are then subdivided into 1,152 subcultures (55 mm
                  NGM plates) of 500 animals each. After two generations of self-fertilization, 20% of the animals per plate are rinsed off
                  for genomic DNA isolation into twelve 96-well plates (<a href="#bib69">Lesa, 2006</a>). The remaining animals are stored at 15&deg;C for up to six weeks. Although libraries can be frozen, live-animal libraries are
                  less prone to failures in mutant recovery (<a href="#bib69">Lesa, 2006</a>).
               </p>
               <p>Three approaches are commonly used to search for deletions in a gene of interest: restricted PCR conditions, poison primers,
                  and thermostable restriction enzymes. Each approach employs nested PCR primers designed to flank the genomic sequence of interest
                  to increase specificity, and each method favors amplification of deletion products over larger wild-type products (<a href="#bib35">Edgley et al., 2002</a>; <a href="#bib54">Jansen et al., 1997</a>; <a href="#bib121">Wei et al., 2002</a>; <a href="#bib130">Zwaal et al., 1993</a>). All of these methods are prone to high false positive rates (50%-90%); therefore, each pooled sample should be screened
                  in duplicate (<a href="#bib54">Jansen et al., 1997</a>). A comprehensive protocol for library creation and the poison primer PCR technique is available in the <a href="http://www.wormbook.org/chapters/www_introreversegenetics/introreversegenetics.html" target="_top">Reverse genetics</a> chapter in WormBook.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-1-1"></a>3.1.1.&nbsp;Restricted PCR extension time (<a href="#bib54">Jansen et al., 1997</a>)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Each library pool is incubated first with a primer pair (10 pmol per primer) designed 2.5-3.5 kb apart over a region of interest,
                     followed by incubation of a 1:500 dilution of the first PCR with a nested primer pair (<a href="#bib73">Liu et al., 1999</a>). After addition of polymerase, nucleotides and buffer, samples are cycled with short extension times (45 s to 1 min) to
                     favor accumulation of the deletion product (<a href="#bib54">Jansen et al., 1997</a>). DNA polymerases have specific turnover rates (maximum number of nucleotides polymerized in a given amount of time), which
                     vary depending on the type of Taq used (<a href="#bib64">Kornberg and Baker, 1992</a>). For example, if the wild-type (WT) product is 2 kb, the deletion product is 500 bp, and the Taq polymerase has a turnover
                     rate of 1 kb/min, approximately two min will be needed to complete the WT reaction and 30 sec for the deletion product. If
                     the extension time is limited to 45 s, only the deletion product will be amplified. Over many rounds, some WT product is amplified,
                     but at a much slower rate than the deletion. This greatly increases the odds of detecting the less abundant deletion allele.
                     Detected deletions are variable in extent and average around 1.4 kb in size (<a href="#bib73">Liu et al., 1999</a>). Completed reactions are run on an agarose gel, and pools that have a product smaller than the wild-type product are identified.
                     Library animals from this pool are plated either singly or in smaller pools and retested to identify single hermaphrodites
                     carrying the relevant lesion (<a href="#bib54">Jansen et al., 1997</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-1-2"></a>3.1.2.&nbsp;Poison primer method (<a href="#bib35">Edgley et al., 2002</a>)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>This approach was developed to enrich for deletions that are less than 500 bp in size (<a href="#bib35">Edgley et al., 2002</a>). The protocol is similar to that for the restricted PCR extension time method, except that a third primer is added between
                     the external primers of the first PCR (<a href="#figure3">Figure 3</a>). This poison primer is designed to amplify only off of wild-type genomic DNA, competing with the full-length wild-type product,
                     thereby reducing the quantity of wild-type template available for the second PCR. The second set of nested primers is only
                     able to bind the deleted product and the full-length wild-type product, whose total concentration is halved because of the
                     poison primer product. This leads to a 500-fold increase in sensitivity by enhancing the ratio of deletion to wild-type products.
                     Use of a short extension time (45 s) also facilitates synthesis of the deletion product. Because the poison primer determines
                     the location of amplified deletion products, it is usually best to target the 5&#8217; end of a gene to generate null alleles (<a href="#bib35">Edgley et al., 2002</a>).
                  </p>
                  <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

 <tr>
  <td align="center"><p><a href="frmutagenesis_fig3.jpg"><img src="frmutagenesis_fig3_s.jpg" border="2" align="middle" alt="figure 3"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 3. Poison primer method for detecting deletions.&nbsp;</b>This technique enriches rare, deleted DNA over the more abundant, full-length, wild-type product. Primers a, b, and p are
                           used in the first PCR to generate a wild-type template, a mutant template, and a &#8216;poison&#8217; template. In the second PCR, these
                           products are used as template with primers a&#8217; and b&#8217; to increase the signal of the deleted product. Without the poison primer,
                           full-length product dominates the PCR reaction, and the deletion product is below the detection limit. When the poison primer
                           is included, total full-length amplification is decreased, and the deletion product is revealed.
                        </p>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3-1-3"></a>3.1.3.&nbsp;Thermostable restriction enzymes (<a href="#bib51">Huang et al., 2006</a>; <a href="#bib121">Wei et al., 2002</a>)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In this less commonly used method, sequences in a target gene are identified that have a thermostable restriction enzyme recognition
                     site in the interval spanned by the primer pairs (<a href="#bib121">Wei et al., 2002</a>). These enzymes are stable at high temperatures, making them ideal to use with PCR. Pooled genomic DNA is digested for 2
                     h before PCR to cleave genomic DNA and reduce the amount of wild-type template available. The enzymes are also included during
                     the PCR amplifications to continuously cleave any wild-type product. Two sets of nested primers are used. Targeted deletion
                     sequences lack the restriction enzyme site and are not cleaved, thereby increasing the sensitivity of the reaction (<a href="#bib121">Wei et al., 2002</a>). Deletions ranging from 330 bp to 1.7 kb have been found using this method (<a href="#bib51">Huang et al., 2006</a>; <a href="#bib121">Wei et al., 2002</a>). Validated thermostable restriction enzymes include <span class="emphasis"><em>Psp</em></span>GI, <span class="emphasis"><em>Tli</em></span>I, <span class="emphasis"><em>Bst</em></span>UI, <span class="emphasis"><em>Ape</em></span>KI, and <span class="emphasis"><em>Tsp</em></span>45I (<a href="#bib51">Huang et al., 2006</a>; <a href="#bib121">Wei et al., 2002</a>).
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-2"></a>3.2.&nbsp;TILLING
                        </h3>
                     </div>
                  </div>
               </div>
               <p>TILLING (Targeting Induced Local Lesions IN Genomes) allows the identification of point mutations and small indels in a gene
                  of interest (<a href="#bib79">McCallum et al., 2000</a>). Originally developed in <span class="emphasis"><em>Arabidopsis thaliana</em></span>, TILLING uses a single-strand DNA nuclease to determine the location of mismatches after hybridizing a mutated genome to
                  a non-mutated genome (<a href="#bib21">Colbert et al., 2001</a>; <a href="#bib114">Till et al., 2003</a>). While this method has only been used once to isolate point mutations in a gene of interest in <span class="emphasis"><em>C. elegans</em></span>, it has been used more in other organisms. Mutations in a population of <span class="emphasis"><em>C. elegans</em></span> are generated using EMS or ENU (<a href="#bib45">Gilchrist et al., 2006</a>). PCR targeting a gene of interest is performed on genomic DNA derived from many pools of animals (<a href="#figure4">Figure 4</a>). Primers are tagged with different fluorophores, marking the 5&#8217; and 3&#8217; ends of the product with different colors. After
                  PCR, the DNA in each reaction is denatured. Reannealing then allows heteroduplex formation between wild-type and mutant DNA.
                  Reannealed DNA is incubated with a single-strand DNA nuclease, CEL1, derived from celery juice extract (CJE) (<a href="#bib113">Till et al., 2004</a>). CJE cuts heteroduplex DNA where a mismatch or indel creates a single-stranded bulge. Samples are run on denaturing LI-COR
                  gels, and the gels are examined in both fluorescent channels to determine in which pool a mismatch occurs (<a href="#bib45">Gilchrist et al., 2006</a>). In a screen for ten genes (ranging from 788 bp to 9.1 kb in size), 71 mutations were identified. Of these, 59% constituted
                  missense alleles, and 3% were nonsense alleles (<a href="#bib45">Gilchrist et al., 2006</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

 <tr>
  <td align="center"><p><a href="frmutagenesis_fig4.jpg"><img src="frmutagenesis_fig4_s.jpg" border="2" align="middle" alt="figure 4"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 4. TILLING allows identification of point mutations in pools of mutated DNA.&nbsp;</b>Both mutated DNA and wild-type DNA are subjected to PCR, using differentially labeled 5&#8217; and 3&#8217; primers (here in green and
                        red). The PCR products are denatured and reannealed slowly, so that heteroduplex DNA forms between a mutated DNA strand and
                        a wild-type DNA strand. The reannealed PCR products are treated with Cel1, an enzyme that specifically cleaves heteroduplex
                        DNA. The reaction is visualized on a LI-COR gel in two fluorescence channels. When heteroduplex DNA is cut, smaller DNA fragments
                        are seen in both the green and red channels within the same lane. Full-length, uncut product will be prominent at the top
                        of the gel.
                     </p>
                  </div>
               </div>
               <p><span class="bold"><strong><span class="emphasis"><em>Protocol</em></span></strong></span> (<a href="#bib45">Gilchrist et al., 2006</a>):
               </p>
               <p>Individual mutagenized F1 animals are plated singly onto 1500 seeded NGM plates. After the food supply is expended, half of
                  the animals are frozen at &#8722;80&deg;C. The other half are used to generate genomic DNA individually into 96-well plates. To reduce
                  the number of PCR reactions, eight F1 DNA samples are pooled together for analysis. Once a positive pool has been identified,
                  DNA genomic samples are assayed individually to determine from which plate the sample originally arose. PCR primers are designed
                  over an area of 1.5 kb, mainly encompassing exonic regions. Primers can be designed using the program CODDLE to select for
                  regions high in G/C content, which are most prone to mutation by EMS (<a href="#bib45">Gilchrist et al., 2006</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-3"></a>3.3.&nbsp;G4 DNA-induced deletion mutagenesis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The consensus sequence G<sub>3+</sub>N<sub>1-7</sub>G<sub>3+</sub>N<sub>1-7</sub>G<sub>3+</sub> N<sub>1-7</sub>G<sub>3+</sub> folds <span class="emphasis"><em>in vitro</em></span> to form a guanine quadruplex (G4) structure that may block replication <span class="emphasis"><em>in vivo</em></span> (<a href="#bib65">Kruisselbrink et al., 2008</a>). The FANCJ protein (<a href="http://www.wormbase.org/db/get?name=DOG-1;class=Protein" target="_blank">DOG-1</a> in <span class="emphasis"><em>C. elegans</em></span>) is a DNA helicase required for proper replication of such guanine-rich tracts (<a href="#bib65">Kruisselbrink et al., 2008</a>). In <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=dog-1;class=Gene" target="_blank">dog-1</a></em></span> mutants, these tracts are prone to 5&#8217; deletions. There are more than 2,900 guanine quadruplex sites in the <span class="emphasis"><em>C. elegans</em></span> genome, annotated in Wormbase under &#8220;G4 Motif&#8221; in GBrowse, and over 1,600 genes are located 5&#8217; of a G4 site (<a href="#bib94">Pontier et al., 2009</a>). Of ten genes assayed in one study, 11 deletion alleles were generated, most of which were up to several kilobases upstream
                  of the G4 site targeted (<a href="#bib94">Pontier et al., 2009</a>). This method has been used once to generate deletion alleles (<a href="#bib94">Pontier et al., 2009</a>).
               </p>
               <p><span class="bold"><strong><span class="emphasis"><em>Protocol</em></span></strong></span> (<a href="#bib94">Pontier et al., 2009</a>):
               </p>
               <p><span class="emphasis"><em>dog-1(pk2247)</em></span> mutants are picked to individual NGM plates and allowed to grow until no OP50 remains on the plate (<a href="#bib94">Pontier et al., 2009</a>). Half of the animals on each plate are rinsed off with M9 for genomic DNA isolation. Prior to DNA isolation, animals are
                  split into two groups. If a deletion is present in only one of the samples, it probably arose from a somatic event. However,
                  if a deletion is detected in both samples, it is most likely a heritable mutation (<a href="#bib94">Pontier et al., 2009</a>). Deletions are screened using any of the PCR protocols described above. After isolation, the strain is outcrossed to wild-type
                  animals to remove the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=dog-1;class=Gene" target="_blank">dog-1</a></em></span> mutation and any off-target deletions.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Gene-targeted mutagenesis
                     </h2>
                  </div>
               </div>
            </div>
            <p>Gene-targeted mutagenesis methods seek to generate lesions in a specific gene of interest without mutagenesis of the entire
               genome. Nonetheless, strains generated with these methods should still be outcrossed before performing genetic studies, as
               off-target lesions can occasionally occur. In general, however, the mutational load is much lower than that inflicted with
               chemical or radiation mutagenesis.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-1"></a>4.1.&nbsp;Transposon-based methods
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4-1-1"></a>4.1.1.&nbsp;Tc1-mediated gene conversion and deletion
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Transposon-mediated, transgene-directed gene conversion using Tc1 can generate specific alleles in a target gene, albeit at
                     lower frequencies than <span class="emphasis"><em>Mos1</em></span> (<a href="#bib11">Bessereau et al., 2001</a>). Tc1-based methods are not commonly used anymore with the advent of single insertion <span class="emphasis"><em>Mos1</em></span> strains. Some historic alleles were obtained using Tc1, so the following protocol serves as reference for how those alleles
                     were generated. Tc1-based methods require that a transposon is located near or in the gene of interest. Once such a strain
                     is identified, one can either screen for excision events that remove part of the gene, or introduce a mutated transgene to
                     direct gene conversion.
                  </p>
                  <p><span class="bold"><strong><span class="emphasis"><em>Protocol</em></span>:</strong></span></p>
                  <p>For isolating deletion events, animals containing a Tc1 insertion in the gene of interest are crossed into a mutator background
                     (<span class="emphasis"><em>e.g.,</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mut-2;class=Gene" target="_blank">mut-2</a></em></span>), and at least 100 animals are individually plated and allowed to self for two to three generations (<a href="#bib130">Zwaal et al., 1993</a>). Half of the animals on each plate are washed off for DNA isolation, and screened for deletions using PCR. If a deletion
                     is detected, individual animals from the remaining half are plated individually and assayed for the presence of the deletion.
                     Typical excision rates are 10<sup>&#8722;3</sup> per gene per generation (<a href="#bib60">Kagan et al., 1997</a>). Once a deletion has been identified, the strain is outcrossed several times to the wild-type strain to remove <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mut-2;class=Gene" target="_blank">mut-2</a></em></span> and any extraneous mutations.
                  </p>
                  <p>For transgene-directed gene conversion, the protocol is similar, except that a transgene with homology to sequences flanking
                     the transposon insertion site is injected into the gonad first. The transgene is engineered with a point mutation, deletion
                     or GFP insertion (<a href="#bib7">Barrett et al., 2004</a>; <a href="#bib93">Plasterk and Groenen, 1992</a>). Progeny are then screened for homologous recombination from the engineered transgene by PCR. Gene deletions of 0.85 kb
                     to 1.5 kb have been targeted successfully in a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mut-2;class=Gene" target="_blank">mut-2</a></em></span> background, with a frequency of 1/2666 animals undergoing gene conversion with a small deletion template (<a href="#bib7">Barrett et al., 2004</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4-1-2"></a>4.1.2.&nbsp;<span class="emphasis"><em>Mos1</em></span>-based gene conversion and deletion
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>Mos1</em></span> excision-induced Transgene-Instructed gene Conversion (MosTIC) introduces specific point mutations, small deletions, and
                     insertions (<span class="emphasis"><em>e.g.,</em></span> of GFP) into a <span class="emphasis"><em>C. elegans</em></span> gene of interest (<a href="#bib98">Robert and Bessereau, 2007</a>). A strain containing a genomic <span class="emphasis"><em>Mos1</em></span>-insertion near or in a target gene is identified. The <a href="http://elegans.imbb.forth.gr/nemagenetag/" target="_top">NemaGENETAG Consortium</a> created a collection of over 13,000 <span class="emphasis"><em>Mos1</em></span>-tagged alleles (<a href="#bib9">Bazopoulou and Tavernarakis, 2009</a>; <a href="#bib116">Vallin et al., 2012</a>). The <span class="emphasis"><em>Mos1</em></span>-insertion strains are annotated on MosLocator (<a href="http://www.ciml.univ-mrs.fr/applications/MosLocator/" target="_top">http://www.ciml.univ-mrs.fr/applications/MosLocator/</a>) and Wormbase, and they are available to researchers through the Consortium (<a href="http://ums3421.univ-lyon1.fr/" target="_top">http://ums3421.univ-lyon1.fr/</a>). Once the strain of interest is received, it is transformed with a transposase used to remobilize the transposon and with
                     a transgene containing a repair template with a specific mutation in the region of the <span class="emphasis"><em>Mos1</em></span> insertion. Excision of the <span class="emphasis"><em>Mos1</em></span> element following transposase expression can then be repaired by homologous recombination with the provided repair template,
                     thus introducing specific lesions into the genome. Gene conversion events occur at approximately 2 &times; 10<sup>&#8722;4</sup> events per F1 animal isolated, so one would need to screen at least 5000 animals to isolate one successful conversion event
                     (<a href="#bib98">Robert and Bessereau, 2007</a>). A similar strategy, MosDEL, can be used to introduce large deletions at specific sites (<a href="#figure5">Figure 5</a>) (<a href="#bib42">Fr&oslash;kjaer-Jensen et al., 2010</a>). Deletions of up to 25 kb have been generated using the MosDEL method, at efficiencies of approximately 20% of injected worms,
                     such that in 30 positively-selected, transgenic worms, six will carry the targeted deletion (<a href="#bib41">Fr&oslash;kjaer-Jensen et al., 2012</a>). 99.4% of all genes in the <span class="emphasis"><em>C. elegans</em></span> genome are within 25 kb of at least one <span class="emphasis"><em>Mos1</em></span> element in strains generated by the <a href="http://elegans.imbb.forth.gr/nemagenetag/" target="_top">NemaGENETAG</a>, with a median distance of 3.1 kb, essentially making all genes targetable by this method (<a href="#bib42">Fr&oslash;kjaer-Jensen et al., 2010</a>). The related MosSCI method, which also uses homologous repair after excision, can be used to introduce single copies of
                     a transgene into a specific genomic site, but it is not a mutagenic technique (<a href="#bib41">Fr&oslash;kjaer-Jensen et al., 2012</a>; <a href="#bib43">Fr&oslash;kjaer-Jensen et al., 2008</a>).
                  </p>
                  <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

 <tr>
  <td align="center"><p><a href="frmutagenesis_fig5.jpg"><img src="frmutagenesis_fig5_s.jpg" border="2" align="middle" alt="figure 5"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 5. Targeted gene deletions using transposons.&nbsp;</b>To generate a targeted gene deletion, a transposase and a rescuing transgene containing a selection marker (in this case,
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-119;class=Gene" target="_blank">unc-119</a></em></span>) is injected into an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-119;class=Gene" target="_blank">unc-119</a></em></span> mutant animal carrying a transposon insertion near or in a gene of interest. Typically, strains carrying a single <span class="emphasis"><em>Mos1</em></span> insertion near a gene of interest are obtained from the NemaGENETAG Consortium (<a href="#bib9">Bazopoulou and Tavernarakis, 2009</a>; <a href="#bib116">Vallin et al., 2012</a>). Homology-directed repair after a double-strand break (DSB) occurs off the introduced transgene at low frequencies. <span class="emphasis"><em>unc-119(+)</em></span> is inserted into the genome, and in the case of MosDEL, some portion of the gene is deleted, typically the 5&#8217; region of the
                           gene. In MosTIC, the transgene contains a small insertion, deletion, or point mutation, which is incorporated into the genomic
                           DNA.
                        </p>
                     </div>
                  </div>
                  <p><span class="bold"><strong><span class="emphasis"><em>Protocol</em></span>:</strong></span></p>
                  <p><span class="underline">MosTIC:</span> (<a href="#bib99">Robert, 2012</a>) The transgene for repair is designed to contain point mutations, small deletions, or insertions in a region spanning about
                     1 kb around the <span class="emphasis"><em>Mos1</em></span> insertion site. These modifications should be flanked by at least 1.5 kb of genomic sequence homology to facilitate recombination.
                     This repair transgene (50 ng/&micro;l) is introduced into the <span class="emphasis"><em>Mos1-</em></span>containing strain via DNA microinjection, along with a <span class="emphasis"><em>hsp::Transposase</em></span> plasmid (50 ng/&micro;l) and a co-injection marker (<a href="#bib99">Robert, 2012</a>). After a line is established, approximately 200 transgenic animals undergo <span class="emphasis"><em>Mos1</em></span> excision for 1 h at 33&deg;C. Animals are allowed to recover for 1 h at 15&deg;C, and then shifted once more for 1 h to 33&deg;C. After
                     recovery overnight, bleach gravid hermaphrodites and collect the F1s. Plate 100 F1s per plate over many NGM plates and allow
                     to self-fertilize for 2 generations (<a href="#bib99">Robert, 2012</a>). Alternatively, one to five heat-shocked P0 animals may be plated to generate 100 F1 animals per plate (<a href="#bib98">Robert and Bessereau, 2007</a>). After, proceed with a PCR screening strategy as above (<a href="#sec4-1-1" title="4.1.1.&nbsp;Tc1-mediated gene conversion and deletion">Section 4.1.1</a>). One may also screen by phenotype using a different plating strategy if the phenotype is known. The gene conversion event
                     should still be confirmed by PCR, as false positives may arise (<a href="#bib99">Robert, 2012</a>). Expression of the transposase directly under a germline promoter (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=eft-3;class=Gene" target="_blank">eft-3</a></em></span>) rather the heat shock promoter increases efficiency, using a protocol similar to that for MosDEL outlined below (<a href="#bib41">Fr&oslash;kjaer-Jensen et al., 2012</a>; <a href="#bib99">Robert, 2012</a>).
                  </p>
                  <p><span class="underline">MosDEL:</span> (<a href="#bib42">Fr&oslash;kjaer-Jensen et al., 2010</a>) The overall protocol for MosDEL is similar to that for MosTIC. Targeting transgenes used for MosDEL should have one homology
                     arm containing 2 kb of DNA directly adjacent to one side of the <span class="emphasis"><em>Mos1</em></span> insertion site. The other homology arm should have at least 3 kb of DNA targeting sequence upstream, chosen to mark the end
                     point of the deletion (<a href="#bib42">Fr&oslash;kjaer-Jensen et al., 2010</a>). Within the homology arms, GFP, <span class="emphasis"><em>C. briggsae</em></span> <span class="emphasis"><em>unc-119(+),</em></span> or an antibiotic-resistance gene can be inserted as a positive selection marker (<a href="#bib41">Fr&oslash;kjaer-Jensen et al., 2012</a>; <a href="#bib46">Giordano-Santini et al., 2010</a>; <a href="#bib104">Semple et al., 2010</a>; <a href="#bib103">Semple et al., 2012</a>). One advantage of <span class="emphasis"><em>unc-119(+)</em></span> is that it can act as a balancer in case the deletion results in lethality when homozygous. Also, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-119;class=Gene" target="_blank">unc-119</a></em></span> mutant animals are unable to form dauers and die in starvation conditions, which further aids screening (<a href="#bib74">Maduro and Pilgrim, 1995</a>; <a href="#bib95">Praitis et al., 2001</a>). The <span class="emphasis"><em>C. briggsae</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-119;class=Gene" target="_blank">unc-119</a></em></span> sequence is used because it is smaller than the <span class="emphasis"><em>C. elegans</em></span> coding region, but it still rescues <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-119;class=Gene" target="_blank">unc-119</a></em></span> mutants (<a href="#bib42">Fr&oslash;kjaer-Jensen et al., 2010</a>). The injection mix should contain the repair plasmid (50 ng/&micro;l), a <span class="emphasis"><em>Mos1</em></span> transposase under the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=eft-3;class=Gene" target="_blank">eft-3</a></em></span> promoter to drive germ line expression (50 ng/&micro;l, (<a href="#bib41">Fr&oslash;kjaer-Jensen et al., 2012</a>)) and co-injection markers (at least three different, well-expressed markers to reduce false positives) (<a href="#bib41">Fr&oslash;kjaer-Jensen et al., 2012</a>). Deletion mutants are isolated by picking <span class="emphasis"><em>unc-119(+),</em></span> GFP(+), or antibiotic-resistant animals that do not express the co-injection markers. A negative selection marker such as
                     <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00077563?query=peel-1#0-9e-3" target="_blank">peel-1</a></em></span> may be included as a co-injection marker to increase the efficiency of screening after a <span class="emphasis"><em>Mos1</em></span> event (<a href="#bib41">Fr&oslash;kjaer-Jensen et al., 2012</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00077563?query=peel-1#0-9e-3" target="_blank">peel-1</a></em></span> is placed under a heat-shock promoter, and is induced for 2 h at 34&deg;C. Transgenic animals carrying the array are killed,
                     while animals that have lost the transgene survive. This selection will aid in screening for animals that lost the co-injection
                     markers but repaired off the transgene successfully (<a href="#bib41">Fr&oslash;kjaer-Jensen et al., 2012</a>). Deletions are confirmed by PCR and sequencing. The repair template may be engineered to contain Flp recombinase sites,
                     so that the positive selection marker may be removed in somatic cells to create a clean deletion (<a href="#bib119">Vazquez-Manrique et al., 2010</a>).
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-2"></a>4.2.&nbsp;Zinc finger nucleases, transcription activator-like effector nucleases (TALENs), and the CRISPR/Cas system
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Zinc finger nucleases (ZFNs) are engineered chimeric proteins, consisting of tandem zinc finger domains fused to a modified
                  <span class="emphasis"><em>Fok</em></span>I restriction enzyme (<a href="#figure6">Figure 6A</a>) (<a href="#bib63">Kim et al., 1996</a>). The zinc finger domains are engineered to recognize specific sequences. <span class="emphasis"><em>Fok</em></span>I cleaves DNA in the vicinity of the binding site. Typically, three zinc finger domains, recognizing a 9-nt sequence, are
                  fused to <span class="emphasis"><em>Fok</em></span>I. Two such proteins are generated (with a combined recognition sequence of 18 bp) allowing dimeric <span class="emphasis"><em>FokI</em></span> to cleave DNA (<a href="#bib63">Kim et al., 1996</a>). Faulty repair occasionally introduces small indels to disrupt a targeted gene. ZFNs require considerable functional tailoring;
                  therefore, they are expensive to generate.
               </p>
               <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

 <tr>
  <td align="center"><p><a href="frmutagenesis_fig6.jpg"><img src="frmutagenesis_fig6_s.jpg" border="2" align="middle" alt="figure 6"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 6. Molecular scissors can be used to engineer small insertion and deletions into a genome.&nbsp;</b>A. Zinc finger nucleases (ZFNs) consist of three tandem Zinc finger domains (colored umbrellas) fused to a <span class="emphasis"><em>Fok</em></span>I nuclease (gray). <span class="emphasis"><em>Fok</em></span>I acts as a dimer, and specifically cleaves genomic DNA when two ZFNs are targeted in tandem. B. TALENs are similar to ZFNs,
                        except that a single domain recognizes a single nucleotide, so one of four domains is fused together according to the genomic
                        sequence. The nuclease is traditionally <span class="emphasis"><em>Fok</em></span>I (gray), functioning as a dimer, although monomeric TALENs using I-TevI have recently been reported (<a href="#bib12">Beurdeley et al., 2013</a>). C. The CRISPR system uses a single guide RNA (red) to target a region of interest for cleavage. The Cas9 nuclease (green)
                        recognizes aspects of the secondary structure of the single guide RNA and the RNA:DNA hybrid. Cas9 also requires a protospacer
                        motif, NGG, to cleave the DNA.
                     </p>
                  </div>
               </div>
               <p>TALENs function like ZFNs, except that tandem TAL domains are fused to a <span class="emphasis"><em>Fok</em></span>I protein (<a href="#figure6">Figure 6B</a>). TAL domains allow a much greater freedom in choosing a DNA sequence to target (<a href="#bib81">Miller et al., 2011</a>). Each domain is tailored to a specific nucleotide, rather than a tri-nucleotide sequence. The TAL code is modular (<a href="#bib13">Boch et al., 2009</a>; <a href="#bib88">Moscou and Bogdanove, 2009</a>), allowing targeted nucleases to be generated at lower cost using Golden Gate cloning, which compiles multiple DNA fragments
                  in an ordered fashion by traditional cloning techniques (<a href="#bib131">Cermak et al., 2011</a>). An improvement to the system has been reported by fusing a single TAL domain to the catalytic domain of an I-TevI homing
                  endonuclease, allowing the enzyme to function as a monomer (<a href="#bib12">Beurdeley et al., 2013</a>). Although not yet used in <span class="emphasis"><em>C. elegans</em></span>, it may allow TALEN mRNA to be injected at lower concentrations and be more effective (see Protocol below).
               </p>
               <p>In prokaryotes, the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) and CRISPR-associated (Cas) system
                  is an adaptive immune response against viruses and plasmids (reviewed in <a href="#bib123">Wiedenheft et al., 2012</a>). The cell uses an RNA guide to cleave foreign DNA sequences (<a href="#figure6">Figure 6C</a>). The minimum components required to obtain cleavage <span class="emphasis"><em>in vitro</em></span> for the Type II CRISPR system are the Cas9 nuclease from <span class="emphasis"><em>Streptococcus pyogenes</em></span> and a synthetic single guide RNA (sgRNA) composed of a short CRISPR RNA (crRNA) and a trans-activating CRISPR RNA (tracrRNA)
                  (<a href="#bib56">Jinek et al., 2012</a>). Genomic specificity is provided by a 20-bp sequence within the crRNA, whose only constraint is the trinucleotide NGG (Protospacer-Adjacent
                  Motif or PAM) after the 3&#8242; end of the sequence. In genomic editing, after successful targeting by the sgRNA, Cas9 introduces
                  a double-strand break 3 bp upstream of the PAM site, which may introduce indels after error-prone non-homologous end joining
                  (NHEJ) or can repair off an introduced transgene using homology directed repair (HDR). To promote HDR over NHEJ, Cas9 can
                  be converted into a nickase (Cas9n) by a single amino acid substitution (<a href="#bib26">Cong et al., 2013</a>; <a href="#bib32">Mali et al., 2013</a>).  Off-target lesions have been recovered in studies using human cell culture, and single and double mismatches appear to
                  be tolerated in some systems (<a href="#bib139">Fu et al., 2013</a>; <a href="#bib135">Cradick et al., 2013</a>). While no off-target effects have been reported for studies in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib136">Dickinson et al., 2013</a>; <a href="#bib133">Chiu et al., 2013</a>), it is good practice to backcross the mutant animal to remove any possible undesired lesions.  To increase fidelity and
                  reduce off-target hits, two Cas9n enzymes can be used together to target a genomic locus (<a href="#bib142">Ran et al., 2013</a>). One Cas9n nicks the sense strand, while the other nickase targets the anti-sense strand, with an offset of the two sgRNAs
                  between 0 and 20 bp. Double nicking creates 5&#8242; overhangs that may be repaired by NHEJ and introduce lesions. This method was
                  found to greatly decrease off-target lesions in mice zygotes and human cell culture by 50 to 1500 fold (<a href="#bib142">Ran et al., 2013</a>). Additionally, because of the small size of the targeting constructs, multiple loci can be edited at once (<a href="#bib26">Cong et al., 2013</a>).
               </p>
               <p>When ZFNs and TALENs were initially tried in <span class="emphasis"><em>C. elegans</em></span>, germline cleavage was difficult to obtain: ZFN-directed cleavage only occurred in somatic cells (<a href="#bib87">Morton et al., 2006</a>). However, a recently developed protocol uses mRNA constructs for injection into the germ line, and heritable gene changes
                  occur in approximately 5% of progeny (<a href="#bib125">Wood et al., 2011</a>). While only a few studies have used ZFNs and TALENs to generate targeted deletions, these new protocol adjustments greatly
                  improve efficiency. Help designing chimeric proteins can be found at <a href="http://www.zincfingers.org/" target="_top">http://www.zincfingers.org/</a> for ZFNs and at <a href="https://tale-nt.cac.cornell.edu/" target="_top">tale-nt.cac.cornell.edu</a> for TALENs.
               </p>
               <p><span class="bold"><strong>ZFN and TALEN protocol</strong></span> (<a href="#bib125">Wood et al., 2011</a>):
               </p>
               <p>ZFN or TALEN mRNAs that target a gene of interest are injected at low pressure directly into the hermaphrodite gonad (<a href="#bib125">Wood et al., 2011</a>). The mRNAs have 5&#8217; and 3&#8217; UTRs to increase germ line translation efficacy, as well as a 5&#8217; cap and a 3&#8217; polyA tail. ZFN
                  mRNAs are injected at 250 ng/&micro;l per construct (total concentration of 500 ng/&micro;l). TALENs are injected at 1500 ng/&micro;l per construct.
                  F1 animals with the highest likelihood of changes in a target gene arise 10-14 h after injection, with 3.5%-5% of the progeny
                  acquiring a small indel during this 4 h period. F2 animals derived from these F1 animals are screened either by phenotype
                  or by PCR (<a href="#bib125">Wood et al., 2011</a>).
               </p>
               <p><span class="bold"><strong>CRISPR Protocol</strong></span> (<a href="#bib136">Dickinson et al., 2013</a>; <a href="#bib137">Friedland et al., 2013</a>):
               </p>
               <p>CRISPR/Cas9 has been successfully adapted for use in <span class="emphasis"><em>C. elegans</em></span> and allows for rapid mutant recovery, creation of tagged versions of endogenous loci, and the implementation of conditional
                  knock-outs and knock-ins. Generation of the sgRNA requires minimal cloning (<a href="#bib137">Friedland et al., 2013</a>), and transgene assembly is facilitated by Gibson cloning techniques (<a href="#bib160">Gibson et al., 2009</a>). While some constraints on the target DNA may exist beyond the PAM requirement, CRISPR/Cas allows virtually any gene to
                  be removed and edited with precision and fidelity.
               </p>
               <p>The codon-optimized nuclear-localized (NLS) Cas9 protein and sgRNA may be introduced into the germline using a variety of
                  methods, including as a DNA plasmid under the <span class="emphasis"><em>eft-3</em></span> and <span class="emphasis"><em>U6</em></span> promotors, respectively (<a href="#bib132">Chen et al., 2013</a>; <a href="#bib136">Dickinson et al., 2013</a>; <a href="#bib137">Friedland et al., 2013</a>; <a href="#bib143">Tzur et al., 2013</a>), as <span class="emphasis"><em>in vitro</em></span> transcribed RNA (<a href="#bib133">Chiu et al., 2013</a>; <a href="#bib140">Katic and Grosshans, 2013</a>; <a href="#bib141">Lo et al., 2013</a>), a combination of both (<a href="#bib144">Waaijers et al., 2013</a>) or as a recombinant protein (<a href="#bib134">Cho et al., 2013</a>). Mutants can be isolated by phenotype screening or by PCR-based methods described previously. Additionally, a specific lesion
                  or change may be inserted by HDR (<a href="#bib136">Dickinson et al., 2013</a>). For example, one elegant study introduced 4 T/A changes in the <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00003017?query=lin-31#0-9e-3" target="_blank">lin-31</a></em></span> locus to inactivate four MAPK phosphorylation sites and alter vulva development (<a href="#bib136">Dickinson et al., 2013</a>). For a review of the current variations, see (<a href="#bib138">Frokjaer-Jensen, 2013</a>).
               </p>
               <p>For generation of mutant alleles of a gene of interest, clone a target DNA sequence containing G/A(N)<sub>19</sub>NGG, preferably within the coding region and towards the 5&#8242; end of the gene (<a href="#bib137">Friedland et al., 2013</a>) under the U6 promoter. It is also useful if the region of interest to be deleted contains a restriction site for easier
                  mutant screening (<a href="#bib137">Friedland et al., 2013</a>). Inject a DNA plasmid mixture into young adult hermaphrodites using standard techniques. This mixture should contain an
                  NLS-Cas9 under the <span class="emphasis"><em>eft-3</em></span> promoter and a <span class="emphasis"><em>tbb-2</em></span> 3&#8242;UTR for efficient germline expression (50 ng/&#956;l), sgRNA sequence under the U6 promoter (45 ng/&#956;l), and a co-injection marker
                  (e.g., <span class="emphasis"><em>myo-3</em></span>p::mCherry, 5 ng/&#956;l) (<a href="#bib137">Friedland et al., 2013</a>). Isolate F1 animals expressing the co-injection marker and screen their progeny for deletions in the gene of interest using
                  any of the previously discussed techniques.
               </p>
               <p>A similar protocol is used for engineering in specific deletions, point mutations or GFP knock-ins off a transgene following
                  a DSB (<a href="#bib136">Dickinson et al., 2013</a>). The repair transgene should contain at least 1.5 kb of homology regions flanking the desired change, and a selection marker,
                  such as <span class="emphasis"><em>unc-119</em></span>(+) or antibiotic selection, which may itself be flanked by <span class="emphasis"><em>loxP</em></span> sites (<a href="#bib136">Dickinson et al., 2013</a>). A single plasmid containing coding sequences for Cas9 and the sgRNA is injected at 50 ng/&#956;l. The repair template is injected
                  at 10 ng/&#956;l, and co-injection markers, including negative selection using <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00077563?query=peel-1#0-9e-3" target="_blank">peel-1</a></em></span>, are included in the mix. Injected hermaphrodites are singled to individual plates and allowed to grow to starvation, in
                  the case of <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00006843?query=unc-119#0-9e-3" target="_blank">unc-119</a></em></span>(+) selection (<a href="#bib136">Dickinson et al., 2013</a>). Plates containing non-Unc progeny are heat shocked at 34<sup>o</sup>C for 4 h to induce <a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00077563?query=peel-1#0-9e-3" target="_blank"><span class="emphasis"><em>peel-1</em></span></a> negative selection. Non-Unc animals are then singled to establish lines (<a href="#bib136">Dickinson et al., 2013</a>). While current studies in <span class="emphasis"><em>C. elegans</em></span> have used a double-strand break to repair off a transgene, work in mice and human cell cultures suggest that using a nickase
                  version or a double nickase system may increase the efficiency of transgene introduction (<a href="#bib142">Ran et al., 2013</a>). The Cas9 and sgRNA plasmids from (<a href="#bib137">Friedland et al., 2013</a>) and (<a href="#bib136">Dickinson et al., 2013</a>) are available on AddGene.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;Summary
                     </h2>
                  </div>
               </div>
            </div>
            <p>An array of random and directed genome alteration methods are available in <span class="emphasis"><em>C. elegans</em></span>, and the choice of which method to use largely depends on the biological problem at hand. Chemical mutagenesis is generally
               useful for defining genes associated with a particular biological process, as are methods involving random generation of deletions
               and transposon insertions. Reverse genetic methods are utilized when the functions of specific genes are sought. Because all
               of the methods described here rely on random processes for mutation generation, screening of large numbers of animals is generally
               unavoidable. Thus, before embarking on mutagenesis, care should be taken to optimize screen design. Finally, it is imperative
               that the assignment of phenotype to gene be rigorously demonstrated. A good rule to follow is that two independent assays
               should corroborate an assignment. For example, rescue studies and allele identification, multiple alleles, or RNAi and allele
               isolation are all reasonable combinations confirming gene assignment.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>We thank Don Moerman and Mark Edgley for sharing detailed EMS/ENU cocktail mutagenesis protocols. We thank Oliver Hobert for
               soliciting helpful comments on the manuscript. This work was funded in part by NIH grants NS081490, NS064273, and NS073121
               to S.S., and also in part by NRSA Training Grant #GM066699 to L.M.K.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec7"></a>7.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed"><span class="bibliomisc">Ahringer, J. (2006). Reverse genetics. <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.47.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/doi/10.1895/wormbook.1.47.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed"><span class="bibliomisc">Anderson, P. (1995). Mutagenesis. In Methods Cell Biology, H. F. Epstein and D. C. Shakes, ed. (San Diego, CA: Academic Press,
                        Inc.), pp. 31-58.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531732&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed"><span class="bibliomisc">Anderson, P., and Brenner, S. (1984). A selection for myosin heavy chain mutants in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>81</em></span>, 4470-4474.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6589606&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed"><span class="bibliomisc">Babcock, E.B., and Collins, J.L. (1929). Does natural ionizing radiation control rate of mutation? Proc. Natl. Acad. Sci.
                        U. S. A. <span class="emphasis"><em>15</em></span>, 623-628.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16587504&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed"><span class="bibliomisc">Babu, P., and Brenner, S. (1981). Spectrum of <sup>32</sup>P-induced mutants of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mutat. Res. <span class="emphasis"><em>82</em></span>, 269-273.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7266559&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed"><span class="bibliomisc">Bailly, V., Lamb, J., Sung, P., Prakash, S., and Prakash, L. (1994). Specific complex formation between yeast RAD6 and RAD18
                        proteins: a potential mechanism for targeting RAD6 ubiquitin-conjugating activity to DNA damage sites. Genes Dev. <span class="emphasis"><em>8</em></span>, 811-820.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7926769&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.8.7.811" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed"><span class="bibliomisc">Barrett, P.L., Fleming, J.T., and Gobel, V. (2004). Targeted gene alteration in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by gene conversion. Nat. Genet. <span class="emphasis"><em>36</em></span>, 1231-1237.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15502826&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed"><span class="bibliomisc">Barstead, R.J., and Moerman, D.G. (2006). <span class="emphasis"><em>C. elegans</em></span> deletion mutant screening. In <span class="emphasis"><em>C. elegans</em></span>: Methods and Applications, K. Strange, ed. (Totowa, NJ: Humana Press), pp. 51-58.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16988425&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed"><span class="bibliomisc">Bazopoulou, D., and Tavernarakis, N. (2009). The NemaGENETAG initiative: large scale transposon insertion gene-tagging in
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetica <span class="emphasis"><em>137</em></span>, 39-46.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19343510&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s10709-009-9361-3" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed"><span class="bibliomisc">Bessereau, J.L. (2006). Transposons in <span class="emphasis"><em>C. elegans</em></span>. <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.70.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/doi/10.1895/wormbook.1.70.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed"><span class="bibliomisc">Bessereau, J.L., Wright, A., Williams, D.C., Schuske, K., Davis, M.W., and Jorgensen, E.M. (2001). Mobilization of a <span class="emphasis"><em>Drosophila</em></span> transposon in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> germ line. Nature <span class="emphasis"><em>413</em></span>, 70-74.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11544527&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35092567" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed"><span class="bibliomisc">Beurdeley, M., Bietz, F., Li, J., Thomas, S., Stoddard, T., Juillerat, A., Zhang, F., Voytas, D.F., Duchateau, P., and Silva,
                        G.H. (2013). Compact designer TALENs for efficient genome engineering. Nat. Commun. <span class="emphasis"><em>4</em></span>, 1762.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23612303&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ncomms2782" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed"><span class="bibliomisc">Boch, J., Scholze, H., Schornack, S., Landgraf, A., Hahn, S., Kay, S., Lahaye, T., Nickstadt, A., and Bonas, U. (2009). Breaking
                        the code of DNA binding specificity of TAL-type III effectors. Science <span class="emphasis"><em>326</em></span>, 1509-1512.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19933107&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1178811 " target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed"><span class="bibliomisc">Boulin, T., and Bessereau, J.L. (2007). <span class="emphasis"><em>Mos1</em></span>-mediated insertional mutagenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Protoc. <span class="emphasis"><em>2</em></span>, 1276-1287.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17546024&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed"><span class="bibliomisc">Brenner, S. (1974). The genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>77</em></span>, 71-94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4366476&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed"><span class="bibliomisc">Brookes, P., and Lawley, P.D. (1961). The reaction of mono- and di-functional alkylating agents with nucleic acids. Biochem.
                        J. <span class="emphasis"><em>80</em></span>, 496-503.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16748923&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed"><span class="bibliomisc"><span class="emphasis"><em>C. elegans</em></span> Deletion Mutant Consortium. (2012). Large-scale screening for targeted knockouts in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome. G3 <span class="emphasis"><em>2</em></span>, 1415-1425.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23173093&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/g3.112.003830" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib131"></a><p class="bibliomixed"><span class="bibliomisc">Cermak, T., Doyle, E.L., Christian, M., Wang, L., Zhang, Y., Schmidt, C., Baller, J.A., Somia, N.V., Bogdanove, A.J., and
                        Voytas, D.F. (2011). Efficient design and assembly of custom TALEN and other TAL effector-based constructs for DNA targeting.
                        Nucleic Acids Res. <span class="emphasis"><em>39</em></span>, e82.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21493687&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkr218" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed"><span class="bibliomisc">Chang, N., Sun, C., Gao, L., Zhu, D., Xu, X., Zhu, X., Xiong, J.W., and Xi, J.J. (2013). Genome editing with RNA-guided Cas9
                        nuclease in zebrafish embryos. Cell Res. <span class="emphasis"><em>23</em></span>, 465-472.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23528705&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/cr.2013.45" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib132"></a><p class="bibliomixed"><span class="bibliomisc">Chen, C., Fenk, L.A., and de Bono, M. (2013). Efficient genome editing in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by CRISPR-targeted homologous recombination. Nucleic Acids Res. <span class="emphasis"><em>41</em></span>, e193.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24013562&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkt805" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib133"></a><p class="bibliomixed"><span class="bibliomisc">Chiu, H., Schwartz, H.T., Antoshechkin, I., and Sternberg, P.W. (2013). Transgene-Free genome editing in <span class="emphasis"><em>Caenorhabditis elegans</em></span> using CRISPR-Cas. Genetics <span class="emphasis"><em>195</em></span>, 1167-1171.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23979577&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.113.155879" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed"><span class="bibliomisc">Cho, S.W., Kim, S., Kim, J.M., and Kim, J.S. (2013). Targeted genome engineering in human cells with the Cas9 RNA-guided endonuclease.
                        Nat. Biotechnol. <span class="emphasis"><em>31</em></span>, 230-232.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23360966&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nbt.2507" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib134"></a><p class="bibliomixed"><span class="bibliomisc">Cho, S.W., Lee, J., Carroll, D., Kim, J.S., and Lee, J. (2013). Heritable gene knockout in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by direct injection of Cas9-sgRNA ribonucleoproteins. Genetics <span class="emphasis"><em>195</em></span>, 1177-1180.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23979576&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.113.155853" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed"><span class="bibliomisc">Chung, K., Crane, M.M., and Lu, H. (2008). Automated on-chip rapid microscopy, phenotyping and sorting of <span class="emphasis"><em>C. elegans</em></span>. Nat. Methods <span class="emphasis"><em>5</em></span>, 637-643.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18568029&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1227" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed"><span class="bibliomisc">Colbert, T., Till, B.J., Tompa, R., Reynolds, S., Steine, M.N., Yeung, A.T., McCallum, C.M., Comai, L., and Henikoff, S. (2001).
                        High-throughput screening for induced point mutations. Plant Physiol. <span class="emphasis"><em>126</em></span>, 480-484.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11402178&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1104/pp.126.2.480 " target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed"><span class="bibliomisc">Cole, R.S. (1970). Light-induced cross-linking of DNA in the presence of a furocoumarin (psoralen). Studies with phage lambda,
                        <span class="emphasis"><em>Escherichia coli</em></span>, and mouse leukemia cells. Biochim. Biophys. Acta. <span class="emphasis"><em>217</em></span>, 30-39.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4927248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0005-2787(70)90119-X" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed"><span class="bibliomisc">Collins, J., and Anderson, P. (1994). The Tc5 family of transposable elements in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>137</em></span>, 771-781.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8088523&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed"><span class="bibliomisc">Collins, J., Forbes, E., and Anderson, P. (1989). The Tc3 family of transposable genetic elements in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>121</em></span>, 47-55.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2537252&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed"><span class="bibliomisc">Collins, J., Saari, B., and Anderson, P. (1987). Activation of a transposable element in the germ line but not the soma of
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>328</em></span>, 726-728.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3039378&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed"><span class="bibliomisc">Cong, L., Ran, F.A., Cox, D., Lin, S., Barretto, R., Habib, N., Hsu, P.D., Wu, X., Jiang, W., Marraffini, L.A., and Zhang,
                        F. (2013). Multiplex genome engineering using CRISPR/Cas systems. Science <span class="emphasis"><em>339</em></span>, 819-823.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23287718&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1231143" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed"><span class="bibliomisc">Coulondre, C., and Miller, J.H. (1977). Genetic studies of the lac repressor. III. Additional correlation of mutational sites
                        with specific amino acid residues. J. Mol. Biol. <span class="emphasis"><em>117</em></span>, 525-567.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=609095&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0022-2836(77)90056-0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib135"></a><p class="bibliomixed"><span class="bibliomisc">Cradick, T.J., Fine, E.J., Antico, C.J., and Bao, G. (2013). CRISPR/Cas9 systems targeting &#946;-globin and CCR5 genes have substantial
                        off-target activity. Nucleic Acids Res. <span class="emphasis"><em>41</em></span>, 9584-9592.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23939622&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkt714" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed"><span class="bibliomisc">Crane, M.M., Stirman, J.N., Ou, C.Y., Kurshan, P.T., Rehg, J.M., Shen, K., and Lu, H. (2012). Autonomous screening of <span class="emphasis"><em>C. elegans</em></span> identifies genes implicated in synaptogenesis. Nat. Methods <span class="emphasis"><em>9</em></span>, 977-980.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22902935&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.2141" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed"><span class="bibliomisc">De Stasio, E.A., and Dorman, S. (2001). Optimization of ENU mutagenesis of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mutat. Res. <span class="emphasis"><em>495</em></span>, 81-88.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11448645&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S1383-5718(01)00198-X" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed"><span class="bibliomisc">Denver, D.R., Dolan, P.C., Wilhelm, L.J., Sung, W., Lucas-Lledo, J.I., Howe, D.K., Lewis, S.C., Okamoto, K., Thomas, W.K.,
                        Lynch, M., and Baer, C.F. (2009). A genome-wide view of <span class="emphasis"><em>Caenorhabditis elegans</em></span> base-substitution mutation processes. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>106</em></span>, 16310-16314.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19805298&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0904895106" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed"><span class="bibliomisc">Denver, D.R., Morris, K., Lynch, M., and Thomas, W.K. (2004). High mutation rate and predominance of insertions in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> nuclear genome. Nature <span class="emphasis"><em>430</em></span>, 679-682.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15295601&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02697" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed"><span class="bibliomisc">Dicarlo, J.E., Norville, J.E., Mali, P., Rios, X., Aach, J., and Church, G.M. (2013). Genome engineering in <span class="emphasis"><em>Saccharomyces cerevisiae</em></span> using CRISPR-Cas systems. Nucleic Acids Res. <span class="emphasis"><em>41</em></span>, 4336-4343.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23460208&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkt135" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib136"></a><p class="bibliomixed"><span class="bibliomisc">Dickinson, D.J., Ward, J.D., Reiner, D.J., and Goldstein, B. (2013). Engineering the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome using Cas9-triggered homologous recombination. Nat. Methods <span class="emphasis"><em>10</em></span>, 1028-1034.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23995389x&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.2641" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed"><span class="bibliomisc">Doitsidou, M., Flames, N., Lee, A.C., Boyanov, A., and Hobert, O. (2008). Automated screening for mutants affecting dopaminergic-neuron
                        specification in <span class="emphasis"><em>C. elegans</em></span>. Nat. Methods <span class="emphasis"><em>5</em></span>, 869-872.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18758453&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1250" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed"><span class="bibliomisc">Doitsidou, M., Poole, R.J., Sarin, S., Bigelow, H., and Hobert, O. (2010). <span class="emphasis"><em>C. elegans</em></span> mutant identification with a one-step whole-genome-sequencing and SNP mapping strategy. PLoS One <span class="emphasis"><em>5</em></span>, e15435.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21079745&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pone.0015435" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed"><span class="bibliomisc">Edgley, M., D'Souza, A., Moulder, G., McKay, S., Shen, B., Gilchrist, E., Moerman, D., and Barstead, R. (2002). Improved detection
                        of small deletions in complex pools of DNA. Nucleic Acids Res. <span class="emphasis"><em>30</em></span>, e52.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12060690&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gnf051" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed"><span class="bibliomisc">Emmons, S.W., Yesner, L., Ruan, K.S., and Katzenberg, D. (1983). Evidence for a transposon in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Cell <span class="emphasis"><em>32</em></span>, 55-65.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6297788&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0092-8674(83)90496-8" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed"><span class="bibliomisc">Fay, D.S. (2006). Classical genetic methods. <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.165.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/doi/10.1895/wormbook.1.165.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed"><span class="bibliomisc">Fire, A., Xu, S., Montgomery, M.K., Kostas, S.A., Driver, S.E., and Mello, C.C. (1998). Potent and specific genetic interference
                        by double-stranded RNA in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>391</em></span>, 806-811.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9486653&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed"><span class="bibliomisc">Flibotte, S., Edgley, M.L., Chaudhry, I., Taylor, J., Neil, S.E., Rogula, A., Zapf, R., Hirst, M., Butterfield, Y., Jones,
                        S.J., Marra, M.A., Barstead, R.J., and Moerman, D.G. (2010). Whole-genome profiling of mutagenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>185</em></span>, 431-441.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20439774&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.110.116616" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed"><span class="bibliomisc">Freeman, S.E., Hacham, H., Gange, R.W., Maytum, D.J., Sutherland, J.C., and Sutherland, B.M. (1989). Wavelength dependence
                        of pyrimidine dimer formation in DNA of human skin irradiated <span class="emphasis"><em>in situ</em></span> with ultraviolet light. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>86</em></span>, 5605-5609.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2748607&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib137"></a><p class="bibliomixed"><span class="bibliomisc">Friedland, A.E., Tzur, Y.B., Esvelt, K.M., Colaiacovo, M.P., Church, G.M. and Calarco, J.A. (2013). Heritable genome editing
                        in <span class="emphasis"><em>C. elegans</em></span> via a CRISPR-Cas9 system. Nat. Methods <span class="emphasis"><em>10</em></span>, 741-743.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23817069&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.2532" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib138"></a><p class="bibliomixed"><span class="bibliomisc">Fr&oslash;kjaer-Jensen, C. (2013). Exciting prospects for precise engineering of <span class="emphasis"><em>Caenorhabditis elegans</em></span> genomes with CRISPR/Cas9. Genetics <span class="emphasis"><em>195</em></span>, 635-642.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=24190921&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.113.156521" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed"><span class="bibliomisc">Fr&oslash;kjaer-Jensen, C., Davis, M.W., Ailion, M., and Jorgensen, E.M. (2012). Improved <span class="emphasis"><em>Mos1</em></span>-mediated transgenesis in <span class="emphasis"><em>C. elegans</em></span>. Nat. Methods <span class="emphasis"><em>9</em></span>, 117-118.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22290181&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1865" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed"><span class="bibliomisc">Fr&oslash;kjaer-Jensen, C., Davis, M.W., Hollopeter, G., Taylor, J., Harris, T.W., Nix, P., Lofgren, R., Prestgard-Duke, M., Bastiani,
                        M., Moerman, D.G., and Jorgensen, E.M. (2010). Targeted gene deletions in <span class="emphasis"><em>C. elegans</em></span> using transposon excision. Nat. Methods <span class="emphasis"><em>7</em></span>, 451-453.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20418868&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1454" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed"><span class="bibliomisc">Fr&oslash;kjaer-Jensen, C., Davis, M.W., Hopkins, C.E., Newman, B.J., Thummel, J.M., Olesen, S.P., Grunnet, M., and Jorgensen, E.M.
                        (2008). Single-copy insertion of transgenes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Genet. <span class="emphasis"><em>40</em></span>, 1375-1383.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18953339&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng.248" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib139"></a><p class="bibliomixed"><span class="bibliomisc">Fu, Y., Foden, J.A., Khayter, C., Maeder, M.L., Reyon, D., Joung, J.K., and Sander, J.D. (2013). High-frequency off-target
                        mutagenesis induced by CRISPR-Cas nucleases in human cells. Nat. Biotechnol. <span class="emphasis"><em>31</em></span>, 822-826.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23792628&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nbt.2623" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed"><span class="bibliomisc">Gengyo-Ando, K., and Mitani, S. (2000). Characterization of mutations induced by ethyl methanesulfonate, UV, and trimethylpsoralen
                        in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Biochem. Biophys. Res. Commun. <span class="emphasis"><em>269</em></span>, 64-69.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10694478&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1006/bbrc.2000.2260" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib160"></a><p class="bibliomixed"><span class="bibliomisc">Gibson, D.G., Young, L., Chuang, R.Y., Venter, J.C., Hutchison, C.A. 3rd, and Smith, H.O. (2009). Enzymatic assembly of DNA
                        molecules up to several hundred kilobases. Nat. Methods <span class="emphasis"><em>5</em></span>, 343-345.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19363495&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1318" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed"><span class="bibliomisc">Gilchrist, E.J., O'Neil, N.J., Rose, A.M., Zetka, M.C., and Haughn, G.W. (2006). TILLING is an effective reverse genetics
                        technique for <span class="emphasis"><em>Caenorhabditis elegans</em></span>. BMC Genomics <span class="emphasis"><em>7</em></span>, 262.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17049087&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-7-262" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed"><span class="bibliomisc">Giordano-Santini, R., Milstein, S., Svrzikapa, N., Tu, D., Johnsen, R., Baillie, D., Vidal, M., and Dupuy, D. (2010). An antibiotic
                        selection marker for nematode transgenesis. Nat. Methods <span class="emphasis"><em>7</em></span>, 721-723.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20729841&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1494" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed"><span class="bibliomisc">Goth, R., and Rajewsky, M.F. (1972). Ethylation of nucleic acids by ethylnitrosourea-1-<sup>14</sup> C in the fetal and adult rat. Cancer Res. <span class="emphasis"><em>32</em></span>, 1501-1505.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=5063976&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed"><span class="bibliomisc">Granger, L., Martin, E., and Segalat, L. (2004). <span class="emphasis"><em>Mos</em></span> as a tool for genome-wide insertional mutagenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: results of a pilot study. Nucleic Acids Res. <span class="emphasis"><em>32</em></span>, e117.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15310838&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed"><span class="bibliomisc">Greenwald, I.S., and Horvitz, H.R. (1980). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-93;class=Gene" target="_blank">unc-93</a></em></span>(<span class="emphasis"><em>e1500</em></span>): A behavioral mutant of <span class="emphasis"><em>Caenorhabditis elegans</em></span> that defines a gene with a wild-type null phenotype. Genetics <span class="emphasis"><em>96</em></span>, 147-164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6894129&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed"><span class="bibliomisc">Hartwell, L.H., Culotti, J., Pringle, J.R., and Reid, B.J. (1974). Genetic control of the cell division cycle in yeast. Science
                        <span class="emphasis"><em>183</em></span>, 46-51.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4587263&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.183.4120.46" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed"><span class="bibliomisc">Huang, C.G., Agre, P., Strange, K., and Lamitina, T. (2006). Isolation of <span class="emphasis"><em>C. elegans</em></span> deletion mutants following ENU mutagenesis and thermostable restriction enzyme PCR screening. Mol. Biotech. <span class="emphasis"><em>32</em></span>, 83-86.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16382185&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed"><span class="bibliomisc">Hwang, W.Y., Fu, Y., Reyon, D., Maeder, M.L., Tsai, S.Q., Sander, J.D., Peterson, R.T., Yeh, J.R., and Joung, J.K. (2013).
                        Efficient genome editing in zebrafish using a CRISPR-Cas system. Nat. Biotechnol. <span class="emphasis"><em>31</em></span>, 227-229.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23360964&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nbt.2501" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed"><span class="bibliomisc">Jacobson, J.W., Medhora, M.M., and Hartl, D.L. (1986). Molecular structure of a somatically unstable transposable element
                        in <span class="emphasis"><em>Drosophila</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>83</em></span>, 8684-8688.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3022302&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.83.22.8684" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed"><span class="bibliomisc">Jansen, G., Hazendonk, E., Thijssen, K.L., and Plasterk, R.H. (1997). Reverse genetics by chemical mutagenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Genet. <span class="emphasis"><em>17</em></span>, 119-121.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9288111&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng0997-119" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed"><span class="bibliomisc">Jiang, W., Bikard, D., Cox, D., Zhang, F., and Marraffini, L.A. (2013). RNA-guided editing of bacterial genomes using CRISPR-Cas
                        systems. Nat. Biotechnol. <span class="emphasis"><em>31</em></span>, 233-239.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23360965&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nbt.2508" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed"><span class="bibliomisc">Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J.A., and Charpentier, E. (2012). A programmable dual-RNA-guided
                        DNA endonuclease in adaptive bacterial immunity. Science <span class="emphasis"><em>337</em></span>, 816-821.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22745249&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1225829" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed"><span class="bibliomisc">Johnsen, R.C., and Baillie, D.L. (1988). Formaldehyde mutagenesis of the eT1 balanced region in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: dose-response curve and the analysis of mutational events. Mutat. Res. <span class="emphasis"><em>201</em></span>, 137-147.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3419443&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0027-5107(88)90120-0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed"><span class="bibliomisc">Jorgensen, E.M., and Mango, S.E. (2002). The art and design of genetic screens: <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Rev. Genet. <span class="emphasis"><em>3</em></span>, 356-369.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11988761&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg794" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed"><span class="bibliomisc">Justice, M.J., Noveroske, J.K., Weber, J.S., Zheng, B., and Bradley, A. (1999). Mouse ENU mutagenesis. Hum. Mol. Genet. <span class="emphasis"><em>8</em></span>, 1955-1963.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10469849&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/hmg/8.10.1955" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed"><span class="bibliomisc">Kagan, R.M., Niewmierzycka, A., and Clarke, S. (1997). Targeted gene disruption of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> L-isoaspartyl protein repair methyltransferase impairs survival of dauer stage nematodes. Arch. Biochem. Biophys. <span class="emphasis"><em>348</em></span>, 320-328.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9434744&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1006/abbi.1997.0362" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed"><span class="bibliomisc">Kamath, R.S., and Ahringer, J. (2003). Genome-wide RNAi screening in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Methods <span class="emphasis"><em>30</em></span>, 313-321.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12828945&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S1046-2023(03)00050-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib140"></a><p class="bibliomixed"><span class="bibliomisc">Katic, I., and Grosshans, H. (2013). Targeted heritable mutation and gene conversion by Cas9-CRISPR in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>195</em></span>, 1173-1176.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23979578&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.113.155754" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed"><span class="bibliomisc">Ketting, R.F., Haverkamp, T.H., van Luenen, H.G., and Plasterk, R.H. (1999). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mut-7;class=Gene" target="_blank">mut-7</a></em></span> of <span class="emphasis"><em>C. elegans</em></span>, required for transposon silencing and RNA interference, is a homolog of Werner syndrome helicase and RNaseD. Cell <span class="emphasis"><em>99</em></span>, 133-141.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10535732&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0092-8674(00)81645-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed"><span class="bibliomisc">Kim, Y.G., Cha, J., and Chandrasegaran, S. (1996). Hybrid restriction enzymes: zinc finger fusions to <span class="emphasis"><em>Fok</em></span>I cleavage domain. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>93</em></span>, 1156-1160.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8577732&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed"><span class="bibliomisc">Kornberg, A., and Baker, T.A. (1992). DNA Replication (New York: W. H. Freeman and Co.).</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed"><span class="bibliomisc">Kruisselbrink, E., Guryev, V., Brouwer, K., Pontier, D.B., Cuppen, E., and Tijsterman, M. (2008). Mutagenic capacity of endogenous
                        G4 DNA underlies genome instability in FANCJ-defective <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>18</em></span>, 900-905.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18538569&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2008.05.013" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed"><span class="bibliomisc">Langmead, B., and Salzberg, S.L. (2012). Fast gapped-read alignment with Bowtie 2. Nat. Methods <span class="emphasis"><em>9</em></span>, 357-359.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22388286&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed"><span class="bibliomisc">Langmead, B., Trapnell, C., Pop, M., and Salzberg, S.L. (2009). Ultrafast and memory-efficient alignment of short DNA sequences
                        to the human genome. Genome Biol. <span class="emphasis"><em>10</em></span>, R25.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19261174&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed"><span class="bibliomisc">Lawley, P.D., and Thatcher, C.J. (1970). Methylation of deoxyribonucleic acid in cultured mammalian cells by N-methyl-N'-nitro-N-nitrosoguanidine.
                        Biochem. J. <span class="emphasis"><em>116</em></span>, 693-707.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=5435496&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed"><span class="bibliomisc">Lesa, G.M. (2006). Isolation of <span class="emphasis"><em>Caenorhabditis elegans</em></span> gene knockouts by PCR screening of chemically mutagenized libraries. Nat. Protoc. <span class="emphasis"><em>1</em></span>, 2231-2240.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17406462&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nprot.2006.345" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed"><span class="bibliomisc">Levitt, A., and Emmons, S.W. (1989). The Tc2 transposon in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>86</em></span>, 3232-3236.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2541437&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed"><span class="bibliomisc">Liao, L.W., Rosenzweig, B., and Hirsh, D. (1983). Analysis of a transposable element in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>80</em></span>, 3585-3589.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6304721&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib72"></a><p class="bibliomixed"><span class="bibliomisc">Lipinski, K.J., Farslow, J.C., Fitzpatrick, K.A., Lynch, M., Katju, V., and Bergthorsson, U. (2011). High spontaneous rate
                        of gene duplication in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Curr. Biol. <span class="emphasis"><em>21</em></span>, 306-310.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21295484&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2011.01.026" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib73"></a><p class="bibliomixed"><span class="bibliomisc">Liu, L.X., Spoerke, J.M., Mulligan, E.L., Chen, J., Reardon, B., Westlund, B., Sun, L., Abel, K., Armstrong, B., Hardiman,
                        G., King, J., McCague, L., Basson, M., Clover, R., and Johnson, C.D. (1999). High-throughput isolation of <span class="emphasis"><em>Caenorhabditis elegans</em></span> deletion mutants. Genome Res. <span class="emphasis"><em>9</em></span>, 859-867.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10508845&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.9.9.859" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib141"></a><p class="bibliomixed"><span class="bibliomisc">Lo, T.W., Pickle, C.S., Lin, S., Ralston, E.J., Gurling, M., Schartner, C.M., Bian, Q., Doudna, J.A., and Meyer, B.J. (2013).
                        Precise and heritable genome editing in evolutionarily diverse nematodes using TALENs and CRISPR/Cas9 to engineer insertions
                        and deletions. Genetics <span class="emphasis"><em>195</em></span>, 331-348.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23934893&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.113.155382" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib74"></a><p class="bibliomixed"><span class="bibliomisc">Maduro, M., and Pilgrim, D. (1995). Identification and cloning of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-119;class=Gene" target="_blank">unc-119</a></em></span>, a gene expressed in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> nervous system. Genetics <span class="emphasis"><em>141</em></span>, 977-988.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8582641&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib75"></a><p class="bibliomixed"><span class="bibliomisc">Mali, P., Yang, L., Esvelt, K.M., Aach, J., Guell, M., DiCarlo, J.E., Norville, J.E., and Church, G.M. (2013). RNA-guided
                        human genome engineering via Cas9. Science <span class="emphasis"><em>339</em></span>, 823-826.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23287722&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1232033" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib76"></a><p class="bibliomixed"><span class="bibliomisc">Martin, E., Laloux, H., Couette, G., Alvarez, T., Bessou, C., Hauser, O., Sookhareea, S., Labouesse, M., and Segalat, L. (2002).
                        Identification of 1088 new transposon insertions of <span class="emphasis"><em>Caenorhabditis elegans</em></span>: a pilot study toward large-scale screens. Genetics <span class="emphasis"><em>162</em></span>, 521-524.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12242258&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib77"></a><p class="bibliomixed"><span class="bibliomisc">Maydan, J.S., Flibotte, S., Edgley, M.L., Lau, J., Selzer, R.R., Richmond, T.A., Pofahl, N.J., Thomas, J.H., and Moerman,
                        D.G. (2007). Efficient high-resolution deletion discovery in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by array comparative genomic hybridization. Genome Res. <span class="emphasis"><em>17</em></span>, 337-347.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17267812&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.5690307 " target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib78"></a><p class="bibliomixed"><span class="bibliomisc">Maydan, J.S., Okada, H.M., Flibotte, S., Edgley, M.L., and Moerman, D.G. (2009). <span class="emphasis"><em>De Novo</em></span> identification of single nucleotide mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span> using array comparative genomic hybridization. Genetics <span class="emphasis"><em>181</em></span>, 1673-1677.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19189945&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.108.100065" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib79"></a><p class="bibliomixed"><span class="bibliomisc">McCallum, C.M., Comai, L., Greene, E.A., and Henikoff, S. (2000). Targeted screening for induced mutations. Nat. Biotechnol.
                        <span class="emphasis"><em>18</em></span>, 455-457.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10748531&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/74542" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib80"></a><p class="bibliomixed"><span class="bibliomisc">Meneely, P.M., and Herman, R.K. (1979). Lethals, steriles and deficiencies in a region of the X chromosome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>92</em></span>, 99-115.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=574105&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib81"></a><p class="bibliomixed"><span class="bibliomisc">Miller, J.C., Tan, S., Qiao, G., Barlow, K.A., Wang, J., Xia, D.F., Meng, X., Paschon, D.E., Leung, E., Hinkley, S.J., Dulay,
                        G.P., Hua, K.L., Ankoudinova, I., Cost, G.J., Urnov, F.D., Zhang, H.S., Holmes, M.C., Zhang, L., Gregory, P.D., and Rebar,
                        E.J. (2011). A TALE nuclease architecture for efficient genome editing. Nat. Biotechnol. <span class="emphasis"><em>29</em></span>, 143-148.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21179091&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nbt.1755" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib82"></a><p class="bibliomixed"><span class="bibliomisc">Minevich, G., Park, D.S., Blankenberg, D., Poole, R.J., and Hobert, O. (2012). CloudMap: a cloud-based pipeline for analysis
                        of mutant genome sequences. Genetics <span class="emphasis"><em>192</em></span>, 1249-1269.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23051646&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.112.144204" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib83"></a><p class="bibliomixed"><span class="bibliomisc">Mitani, S. (2009). Nematode, an experimental animal in the national BioResource project. Exp. Anim. <span class="emphasis"><em>58</em></span>, 351-356.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19654432&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib84"></a><p class="bibliomixed"><span class="bibliomisc">Moerman, D.G., and Baillie, D.L. (1981). Formaldehyde mutagenesis in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mutat. Res. <span class="emphasis"><em>80</em></span>, 273-279.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7207485&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0027-5107(81)90100-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib85"></a><p class="bibliomixed"><span class="bibliomisc">Mori, I., Benian, G.M., Moerman, D.G., and Waterston, R.H. (1988). Transposable element Tc1 of <span class="emphasis"><em>Caenorhabditis elegans</em></span> recognizes specific target sequences for integration. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>85</em></span>, 861-864.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2829205&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib86"></a><p class="bibliomixed"><span class="bibliomisc">Mori, I., Moerman, D.G., and Waterston, R.H. (1988). Analysis of a mutator activity necessary for germline transposition and
                        excision of Tc1 transposable elements in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>120</em></span>, 397-407.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2829205&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib87"></a><p class="bibliomixed"><span class="bibliomisc">Morton, J., Davis, M.W., Jorgensen, E.M., and Carroll, D. (2006). Induction and repair of zinc-finger nuclease-targeted double-strand
                        breaks in <span class="emphasis"><em>Caenorhabditis elegans</em></span> somatic cells. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>103</em></span>, 16370-16375.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17060623&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0605633103" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib88"></a><p class="bibliomixed"><span class="bibliomisc">Moscou, M.J., and Bogdanove, A.J. (2009). A simple cipher governs DNA recognition by TAL effectors. Science <span class="emphasis"><em>326</em></span>, 1501.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19933106&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1178817" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib89"></a><p class="bibliomixed"><span class="bibliomisc">Muller, H.J. (1927). Artificial transmutation of the gene. Science <span class="emphasis"><em>66</em></span>, 84-87.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17802387&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.66.1699.84" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib90"></a><p class="bibliomixed"><span class="bibliomisc">Muller, H.J. (1932). Further studies on the nature and causes of gene mutations. In Proceedings of the Sixth International
                        Congress of Genetics, D. F. Jones, ed. (Ithaca, NY: Brooklyn Botanic Garden), pp. 213-255.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib91"></a><p class="bibliomixed"><span class="bibliomisc">Nei, M. (2007). The new mutation theory of phenotypic evolution. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>104</em></span>, 12235-12242.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17640887&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0703349104" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib92"></a><p class="bibliomixed"><span class="bibliomisc">Ohnishi, K., and Ohnishi, T. (2004). The biological effects of space radiation during long stays in space. Biol. Sci. Space
                        <span class="emphasis"><em>18</em></span>, 201-205.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15858386&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib93"></a><p class="bibliomixed"><span class="bibliomisc">Plasterk, R.H., and Groenen, J.T. (1992). Targeted alterations of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome by transgene instructed DNA double strand break repair following Tc1 excision. EMBO J. <span class="emphasis"><em>11</em></span>, 287-290.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1740110&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib94"></a><p class="bibliomixed"><span class="bibliomisc">Pontier, D.B., Kruisselbrink, E., Guryev, V., and Tijsterman, M. (2009). Isolation of deletion alleles by G4 DNA-induced mutagenesis.
                        Nat. Methods <span class="emphasis"><em>6</em></span>, 655-657.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19684597&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1362" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib95"></a><p class="bibliomixed"><span class="bibliomisc">Praitis, V., Casey, E., Collar, D., and Austin, J. (2001). Creation of low-copy integrated transgenic lines in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>157</em></span>, 1217-1226.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11238406&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib142"></a><p class="bibliomixed"><span class="bibliomisc">Ran, F.A., Hsu, P.D., Lin, C.Y., Gootenberg, J.S., Konermann, S., Trevino, A.E., Scott, D.A., Inoue, A., Matoba, S., Zhang,
                        Y., and Zhang, F. (2013). Double nicking by RNA-guided CRISPR Cas9 for enhanced genome editing specificity. Cell <span class="emphasis"><em>154</em></span>, 1380-1389.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23992846&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2013.08.021" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib96"></a><p class="bibliomixed"><span class="bibliomisc">Reedy, R.C., Arnold, J.R., and Lal, D. (1983). Cosmic-ray record in solar system matter. Science <span class="emphasis"><em>219</em></span>, 127-135.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17841664&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib97"></a><p class="bibliomixed"><span class="bibliomisc">Rezsohazy, R., van Luenen, H.G., Durbin, R.M., and Plasterk, R.H. (1997). Tc7, a Tc1-hitch hiking transposon in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>25</em></span>, 4048-4054.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9321656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/25.20.4048" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib98"></a><p class="bibliomixed"><span class="bibliomisc">Robert, V., and Bessereau, J.L. (2007). Targeted engineering of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome following <span class="emphasis"><em>Mos1</em></span>-triggered chromosomal breaks. EMBO J. <span class="emphasis"><em>26</em></span>, 170-183.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17159906&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/sj.emboj.7601463" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib99"></a><p class="bibliomixed"><span class="bibliomisc">Robert, V.J. (2012). Engineering the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome by <span class="emphasis"><em>Mos1</em></span>-induced transgene-instructed gene conversion. In Mobile Genetic Elements: Protocols and Genomic Applications, Y. Bigot, ed.
                        (Totowa, NJ: Humana Press), pp. 189-201.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22367873&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/978-1-61779-603-6_11" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib100"></a><p class="bibliomixed"><span class="bibliomisc">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L. (1985). Mutagenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. II. A spectrum of mutational events induced with 1500 r of gamma-radiation. Genetics <span class="emphasis"><em>109</em></span>, 493-511.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3979812&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib101"></a><p class="bibliomixed"><span class="bibliomisc">Rushforth, A.M., and Anderson, P. (1996). Splicing removes the <span class="emphasis"><em>Caenorhabditis elegans</em></span> transposon Tc1 from most mutant pre-mRNAs. Mol. Cell Biol. <span class="emphasis"><em>16</em></span>, 422-429.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8524324&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib102"></a><p class="bibliomixed"><span class="bibliomisc">Sarin, S., Prabhu, S., O'Meara, M.M., Pe'er, I., and Hobert, O. (2008). <span class="emphasis"><em>Caenorhabditis elegans</em></span> mutant allele identification by whole-genome sequencing. Nat. Methods <span class="emphasis"><em>5</em></span>, 865-867.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18677319&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1249" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib103"></a><p class="bibliomixed"><span class="bibliomisc">Semple, J.I., Biondini, L., and Lehner, B. (2012). Generating transgenic nematodes by bombardment and antibiotic selection.
                        Nat. Methods <span class="emphasis"><em>9</em></span>, 118-119.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22290182&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1864" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib104"></a><p class="bibliomixed"><span class="bibliomisc">Semple, J.I., Garcia-Verdugo, R., and Lehner, B. (2010). Rapid selection of transgenic <span class="emphasis"><em>C. elegans</em></span> using antibiotic resistance. Nat. Methods <span class="emphasis"><em>7</em></span>, 725-727.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20729840&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1495" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib105"></a><p class="bibliomixed"><span class="bibliomisc">Shaham, S. (2007). Counting mutagenized genomes and optimizing genetic screens in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. PLoS One <span class="emphasis"><em>2</em></span>, e1117.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17989770&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pone.0001117" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib106"></a><p class="bibliomixed"><span class="bibliomisc">Shaham, S. (2009). galign: a tool for rapid genome polymorphism discovery. PLoS One <span class="emphasis"><em>4</em></span>, e7188.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19779626&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pone.0007188" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib107"></a><p class="bibliomixed"><span class="bibliomisc">Sigurdson, D.C., Spanier, G.J., and Herman, R.K. (1984). <span class="emphasis"><em>Caenorhabditis elegans</em></span> deficiency mapping. Genetics <span class="emphasis"><em>108</em></span>, 331-345.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6500256&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib108"></a><p class="bibliomixed"><span class="bibliomisc">Smith, H.E. (2011). Identifying insertion mutations by whole-genome sequencing. Biotechniques <span class="emphasis"><em>50</em></span>, 96-97.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21486250&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib109"></a><p class="bibliomixed"><span class="bibliomisc">Stewart, H.I., Rosenbluth, R.E., and Baillie, D.L. (1991). Most ultraviolet irradiation induced mutations in the nematode
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span> are chromosomal rearrangements. Mutat. Res. <span class="emphasis"><em>249</em></span>, 37-54.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2067542&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0027-5107(91)90131-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib110"></a><p class="bibliomixed"><span class="bibliomisc">Tabara, H., Sarkissian, M., Kelly, W.G., Fleenor, J., Grishok, A., Timmons, L., Fire, A., and Mello, C.C. (1999). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rde-1;class=Gene" target="_blank">rde-1</a></em></span> gene, RNA interference, and transposon silencing in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>99</em></span>, 123-132.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10535731&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0092-8674(00)81644-X" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib111"></a><p class="bibliomixed"><span class="bibliomisc">Thompson, O., Edgley, M., Strasbourger, P., Flibotte, S., Ewing, B., Adair, R., Au, V., Chaudry, I., Fernando, L., Hutter,
                        H., et al. (2013). The million mutation project: A new approach to genetics in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome Res. <span class="emphasis"><em>23</em></span>, 1749-1762.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23800452&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.157651.113" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib112"></a><p class="bibliomixed"><span class="bibliomisc">Tijsterman, M., Ketting, R.F., Okihara, K.L., Sijen, T., and Plasterk, R.H. (2002). RNA helicase MUT-14-dependent gene silencing
                        triggered in <span class="emphasis"><em>C. elegans</em></span> by short antisense RNAs. Science <span class="emphasis"><em>295</em></span>, 694-697.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11809977&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1067534" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib113"></a><p class="bibliomixed"><span class="bibliomisc">Till, B.J., Burtner, C., Comai, L., and Henikoff, S. (2004). Mismatch cleavage by single-strand specific nucleases. Nucleic
                        Acids Res. <span class="emphasis"><em>32</em></span>, 2632-2641.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15141034&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkh599" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib114"></a><p class="bibliomixed"><span class="bibliomisc">Till, B.J., Reynolds, S.H., Greene, E.A., Codomo, C.A., Enns, L.C., Johnson, J.E., Burtner, C., Odden, A.R., Young, K., Taylor,
                        N.E., et al. (2003). Large-scale discovery of induced point mutations with high-throughput TILLING. Genome Res. <span class="emphasis"><em>13</em></span>, 524-530.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12618384&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.977903" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib115"></a><p class="bibliomixed"><span class="bibliomisc">Trent, C., Purnell, B., Gavinski, S., Hageman, J., Chamblin, C., and Wood, W.B. (1991). Sex-specific transcriptional regulation
                        of the <span class="emphasis"><em>C. elegans</em></span> sex-determining gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=her-1;class=Gene" target="_blank">her-1</a></em></span>. Mech. Dev. <span class="emphasis"><em>34</em></span>, 43-55.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1716965&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0925-4773(91)90090-S" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib143"></a><p class="bibliomixed"><span class="bibliomisc">Tzur, Y.B., Friedland, A.E., Nadarajan, S., Church, G.M., Calarco, J.A., and Colaiacovo, M.P. (2013). Heritable custom genomic
                        modifications in <span class="emphasis"><em>Caenorhabditis elegans</em></span> via a CRISPR-Cas9 System. Genetics <span class="emphasis"><em>195</em></span>, 1181-1185.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23979579&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.113.156075" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib116"></a><p class="bibliomixed"><span class="bibliomisc">Vallin, E., Gallagher, J., Granger, L., Martin, E., Belougne, J., Maurizio, J., Duverger, Y., Scaglione, S., Borrel, C., Cortier,
                        E., et al. (2012). A genome-wide collection of <span class="emphasis"><em>Mos1</em></span> transposon insertion mutants for the <span class="emphasis"><em>C. elegans</em></span> research community. PLoS One <span class="emphasis"><em>7</em></span>, e30482.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22347378&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pone.0030482" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib117"></a><p class="bibliomixed"><span class="bibliomisc">van der Linden, A.M., and Plasterk, R.H.A. (2004). Shotgun cloning of transposon insertions in the genome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Comp. Funct. Genomics <span class="emphasis"><em>5</em></span>, 225-229.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18629154&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/cfg.392" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib118"></a><p class="bibliomixed"><span class="bibliomisc">Vastenhouw, N.L., Fischer, S.E., Robert, V.J., Thijssen, K.L., Fraser, A.G., Kamath, R.S., Ahringer, J., and Plasterk, R.H.
                        (2003). A genome-wide screen identifies 27 genes involved in transposon silencing in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>13</em></span>, 1311-1316.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12906791&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0960-9822(03)00539-6" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib119"></a><p class="bibliomixed"><span class="bibliomisc">Vazquez-Manrique, R.P., Legg, J.C., Olofsson, B., Ly, S., and Baylis, H.A. (2010). Improved gene targeting in <span class="emphasis"><em>C. elegans</em></span> using counter-selection and Flp-mediated marker excision. Genomics <span class="emphasis"><em>95</em></span>, 37-46.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19747540&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.ygeno.2009.09.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib144"></a><p class="bibliomixed"><span class="bibliomisc">Waaijers, S., Portegijs, V., Kerver, J., Lemmens, B.B., Tijsterman, M., van den Heuvel, S., and Boxem, M. (2013). CRISPR/Cas9-Targeted
                        mutagenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>195</em></span>, 1187-1191.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23979586&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.113.156299" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib120"></a><p class="bibliomixed"><span class="bibliomisc">Wang, H., Yang, H., Shivalila, C.S., Dawlaty, M.M., Cheng, A.W., Zhang, F., and Jaenisch, R. (2013). One-step generation of
                        mice carrying mutations in multiple genes by CRISPR/Cas-mediated genome engineering. Cell <span class="emphasis"><em>153</em></span>, 910-918.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23643243&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2013.04.025" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib121"></a><p class="bibliomixed"><span class="bibliomisc">Wei, A., Yuan, A., Fawcett, G., Butler, A., Davis, T., Xu, S.Y., and Salkoff, L. (2002). Efficient isolation of targeted <span class="emphasis"><em>Caenorhabditis elegans</em></span> deletion strains using highly thermostable restriction endonucleases and PCR. Nucleic Acids Res. <span class="emphasis"><em>30</em></span>, e110.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12384612&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gnf109" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib122"></a><p class="bibliomixed"><span class="bibliomisc">Wicks, S.R., Yeh, R.T., Gish, W.R., Waterston, R.H., and Plasterk, R.H. (2001). Rapid gene mapping in <span class="emphasis"><em>Caenorhabditis elegans</em></span> using a high density polymorphism map. Nat. Genet. <span class="emphasis"><em>28</em></span>, 160-164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11381264&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/88878" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib123"></a><p class="bibliomixed"><span class="bibliomisc">Wiedenheft, B., Sternberg, S.H., and Doudna, J.A. (2012). RNA-guided genetic silencing systems in bacteria and archaea. Nature
                        <span class="emphasis"><em>482</em></span>, 331-338.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22337052&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature10886" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib124"></a><p class="bibliomixed"><span class="bibliomisc">Williams, D.C., Boulin, T., Ruaud, A.F., Jorgensen, E.M., and Bessereau, J.L. (2005). Characterization of <span class="emphasis"><em>Mos1</em></span>-mediated mutagenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: a method for the rapid identification of mutated genes. Genetics <span class="emphasis"><em>169</em></span>, 1779-1785.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15654093&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.038265" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib125"></a><p class="bibliomixed"><span class="bibliomisc">Wood, A.J., Lo, T.W., Zeitler, B., Pickle, C.S., Ralston, E.J., Lee, A.H., Amora, R., Miller, J.C., Leung, E., Meng, X., et
                        al. (2011). Targeted genome editing across species using ZFNs and TALENs. Science <span class="emphasis"><em>333</em></span>, 307.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21700836&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1207773" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib126"></a><p class="bibliomixed"><span class="bibliomisc">Yandell, M.D., Edgar, L.G., and Wood, W.B. (1994). Trimethylpsoralen induces small deletion mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>91</em></span>, 1381-1385.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7906415&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.91.4.1381" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib127"></a><p class="bibliomixed"><span class="bibliomisc">Yuan, J.Y., Finney, M., Tsung, N., and Horvitz, H.R. (1991). Tc4, a <span class="emphasis"><em>Caenorhabditis elegans</em></span> transposable element with an unusual fold-back structure. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>88</em></span>, 3334-3338.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1849651&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.88.8.3334" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib128"></a><p class="bibliomixed"><span class="bibliomisc">Zhao, Y., Lai, K., Cheung, I., Youds, J., Tarailo, M., Tarailo, S., and Rose, A. (2006). A mutational analysis of <span class="emphasis"><em>Caenorhabditis elegans</em></span> in space. Mutat. Res. <span class="emphasis"><em>601</em></span>, 19-29.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16765996&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.mrfmmm.2006.05.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib129"></a><p class="bibliomixed"><span class="bibliomisc">Zuryn, S., Le Gras, S., Jamet, K., and Jarriault, S. (2010). A strategy for direct mapping and identification of mutations
                        by whole-genome sequencing. Genetics <span class="emphasis"><em>186</em></span>, 427-430.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20610404&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.110.119230" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib130"></a><p class="bibliomixed"><span class="bibliomisc">Zwaal, R.R., Broeks, A., van Meurs, J., Groenen, J.T., and Plasterk, R.H. (1993). Target-selected gene inactivation in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by using a frozen transposon insertion mutant bank. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>90</em></span>, 7431-7435.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8395047&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Oliver Hobert Last revised January 16, 2014, Published January 17, 2014. This chapter should be cited as: Kutscher L.
                  M., Shaham S. Forward and reverse mutagenesis in <span class="emphasis"><em>C. elegans</em></span> (January 17, 2014), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.167.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2014 Lena M. Kutscher and Shai Shaham. This is an open-access article distributed under the terms of the Creative Commons
                  Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author
                  and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e37" href="#d0e37">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:shaham@rockefeller.edu" target="_top">shaham@rockefeller.edu</a></p>
 <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>