<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Gene duplications and genetic redundancy in C. elegans</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#GeneDupl_ArticleID" title="Gene duplications and genetic redundancy in C. elegans">
      <link rel="next" href="#d0e80" title="1.&nbsp;Gene duplications and genetic redundancy">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>

   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="geneduplication.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="GeneDupl_ArticleID"></a>Gene duplications and genetic redundancy in <span class="emphasis"><em>C. elegans</em></span><sup><a name="d0e12" href="#ftn.d0e12">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Alison</span> <span class="surname">Woollard<em><span class="remark"><sup><a name="d0e40" href="#ftn.d0e40">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="address">Genetics Unit, Department of Biochemistry, University of Oxford, Oxford OX1 3QU, UK
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_geneticsgenomics.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Genetics and genomics<br />>> Genetics
		</a></div><hr />

<a href="http://www.wormbase.org/forums/index.php?board=187.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Gene+duplications+and+genetic+redundancy+in+C.+elegans" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e80">1. Gene duplications and genetic redundancy</a></span></dt>
               <dt><span class="sect1"><a href="#d0e660">2. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#d0e665">3. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>Evolutionary innovation requires genetic raw materials upon which selection can act.
                                 The duplication of genes is of fundamental importance in providing such raw materials.
                                 Gene duplications are very widespread in</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>and appear to arise more frequently
                                 than in either</strong></span> <span class="emphasis"><em><span class="bold"><strong>Drosophila</strong></span></em></span> <span class="bold"><strong>or yeast. It has been proposed that the rate of duplication of
                                 a gene is of the same order of magnitude as the rate of mutation per nucleotide site,
                                 emphasising the enormous potential that gene duplication has for generating substrates
                                 for evolutionary change.</strong></span></p>
            <p><span class="bold"><strong>The fate of duplicated genes is discussed. Complete functional redundancy seems
                                 unstable in the long term. Most models require that equality amongst duplicated genes
                                 must be disrupted if they are to be preserved. There are various ways of achieving
                                 inequality, involving either the nonfunctionalization of one copy, or one copy acquiring
                                 some novel, beneficial function, or both copies becoming partially compromised so that
                                 both copies are required to provide the overall function that was previously provided by
                                 the single ancestral gene. Examples of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>gene duplications that appear to have
                                 followed each of these pathways are considered.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e80"></a>1.&nbsp;Gene duplications and genetic redundancy
                     </h2>
                  </div>
               </div>
            </div>
            <p>The duplication of genes is of paramount importance in providing raw materials for the
                   evolution of genetic diversity. It is therefore of interest to consider <span class="emphasis"><em>C. elegans</em></span> gene
                   duplications in the context of other sequenced genomes. Two recent estimates of the overall
                   extent of gene duplications in <span class="emphasis"><em>C. elegans</em></span> have come up with very similar figures. There are
                   thought to be around 1,200 gene families containing two or more paralogues in <span class="emphasis"><em>C. elegans</em></span>
                   (<a href="#bib3">Cavalcanti et al., 2003</a>; <a href="#bib8">Gu et al., 2002</a>). The total number of
                   genes in the paranome is around 6,000, or <span class="symbol">&#8764;</span> 32<span class="symbol">%</span>  of the genome, where the paranome is defined as
                   the set of proteins that have one or more paralogues, that is, those that are not singletons. In
                   addition, 7.1<span class="symbol">%</span>  of the duplicated genes in the worm are thought to have resulted from block
                   duplications (duplication events involving more than one gene; <a href="#bib3">Cavalcanti et al., 2003</a>). Simple sequence duplications range in size from hundreds of bases to tens of
                   kilobases and copies may be dispersed or clustered (<a href="#bib23">The <span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>). Local clusters of duplicated genes are easily identified and there are 402 such
                   clusters (where a cluster is defined as a group of N genes that are similar within a window of
                   2N genes along the chromosome and where N <span class="symbol">=</span>
                3 or more) distributed throughout the genome (<a href="#bib23">The <span class="emphasis"><em>C.
                            elegans</em></span> Sequencing Consortium, 1998</a>). The worm genome contains twice the number of
                   local gene duplications as <span class="emphasis"><em>Drosophila</em></span> (<a href="#bib21">Rubin et al., 2000</a>), and also differs from
                   the fly genome in terms of the distribution of gene duplicates. Gene duplications are found
                   randomly throughout the fly genome, whereas in the worm duplicated genes are mostly
                   clustered in the recombinogenic segments of the autosomal arms (<a href="#bib21">Rubin et al., 2000</a>).
            </p>
            <p>The rate of origin of gene duplicates in <span class="emphasis"><em>C. elegans</em></span> over the past few hundred
                   thousand years appears to be substantially higher than that for <span class="emphasis"><em>Drosophila</em></span> or yeast (<a href="#bib16">Lynch and
                      Conery, 2000</a>). <a href="#bib16">Lynch and Conery (2000</a>) propose a per-gene rate of duplication of 0.02
                   per million years for <span class="emphasis"><em>C. elegans</em></span>, compared with a rate of 0.002 duplications per gene per
                   million years in <span class="emphasis"><em>Drosophila</em></span> and 0.008 in yeast. <a href="#bib8">Gu et al. (2002)</a> have reported exactly the same figure for <span class="emphasis"><em>C. elegans</em></span>, compared with a rate of only 0.0014 for
                   <span class="emphasis"><em>Drosophila</em></span> (<a href="#bib9">Gu, 2003</a>).
            </p>
            <p>Given the range of values predicted for different species,
                   it has been proposed that 50<span class="symbol">%</span>  of genes in a genome would be expected to duplicate, on average,
                   at least once on time scales of 35 to 350 million years (<a href="#bib16">Lynch and Conery, 2000</a>).
                   Thus, even in the absence of direct amplification of the entire genome (polyploidization), gene
                   duplication has substantial potential for generating substrates for evolutionary innovation.
                   Indeed, it has been proposed that the rate of duplication of a gene is of the same order of
                   magnitude as the rate of mutation per nucleotide site (<a href="#bib16">Lynch and Conery, 2000</a>).
            </p>
            <p> What is the mechanism of gene duplication? Replication slippage and unequal exchange are often
                   invoked as an explanation for closely spaced gene duplicates, but these mechanisms would be
                   expected to give rise to tandem duplicates pointing in the same direction. A recent study found,
                   however, that up to 69<span class="symbol">%</span>  of <span class="emphasis"><em>C. elegans</em></span> duplicate genes reside in the inverse orientation,
                   especially young duplicates (<a href="#bib11">Katju and Lynch, 2003</a>). Inversions are considered by
                   these authors to be part and parcel of the original duplication event, rather than secondary
                   rearrangements (<a href="#bib11">Katju and Lynch, 2003</a>). It has been hypothesized that inverted
                   duplications could be generated by an illegitimate recombination event during DNA replication,
                   involving strand switching by the DNA polymerase or strand misalignment-realignment.
                   RNA-mediated transposition is thought unlikely to have played a significant role in gene
                   duplication within the <span class="emphasis"><em>C. elegans</em></span> genome because of the very small proportion of duplicate genes
                   that lack introns relative to the original copy (<a href="#bib11">Katju and Lynch, 2003</a>). The
                   mechanisms responsible for gene duplication are therefore, in general, unlikely to respect gene
                   boundaries. This is borne out by the Katju and Lynch study, which analysed 290 gene pairs with <span class="symbol">&#8804;</span>
               
                   10<span class="symbol">%</span>  divergence at synonymous sites (Ks) within the <span class="emphasis"><em>C. elegans</em></span> genome. They found that the
                   average duplication span of 1.4 kb is less than the average gene length in <span class="emphasis"><em>C. elegans</em></span> (2.5 kb),
                   suggesting that partial gene duplications are frequent (<a href="#bib11">Katju and Lynch, 2003</a>).
            </p>
            <p>Around half of the <span class="emphasis"><em>C. elegans</em></span> gene duplicates with very low levels of synonymous site
                   substitution were found to also contain unique coding sequence not present in the other copy, in
                   addition to the region of close homology (<a href="#bib11">Katju and Lynch, 2003</a>). Thus,
                   structural heterogeneity between duplicate genes is common. Chimeric duplicates may well have
                   creative potential, especially when they act in conjunction with shuffling events.
            </p>
            <p>What fate
                   awaits duplicate genes? Three alternative theories have been proposed (<a href="#bib15">Lynch, 2002</a>; <a href="#bib16">Lynch and Conery, 2000</a>). The first is the nonfunctionalization of one
                   copy by the accumulation of degenerative mutations. The second is neofunctionalization. In this
                   scenario, one copy acquires a novel, beneficial function and becomes preserved by natural
                   selection, with the other copy retaining the original function. A variation of this scenario is
                   that the two genes could acquire divergent functions but maintain a functional overlap.
                   Selection would then act on these divergent properties and indirectly maintain the functional
                   overlap. Selection could also act on a newly emergent property unique to the combined action of
                   two closely related genes, or on some enhanced efficiency or fidelity achieved by the combined
                   action of two such genes (<a href="#bib24">Thomas, 1993</a>). In the third scenario,
                   subfunctionalization, both copies become partially compromised by the accumulation of mutations,
                   to a point where their total capacity is reduced to the level of the single-copy ancestral gene.
            </p>
            <p>Whatever the mechanism, most models for conserving redundancy over the course of evolution
                   require some degree of symmetry-breaking: equality amongst duplicated genes must be disrupted if
                   they are to be preserved. The fate awaiting most gene duplicates in <span class="emphasis"><em>C. elegans</em></span>, as well as in
                   most other species studied, is likely to be silencing and ultimate loss, rather than
                   preservation (<a href="#bib16">Lynch and Conery, 2000</a>). It has been estimated that the average
                   half-life of a gene duplicate is around 4 million years (<a href="#bib16">Lynch and Conery, 2000</a>).
                   The propensity of recently duplicated genes to become nonfunctional pseudogenes is borne out in
                   a recent study by <a href="#bib17">Mounsey et al. (2002)</a>, who found a lower than expected success rate in generating
                   expression patterns for recently duplicated genes, suggesting that recently duplicated genes are
                   less likely to be expressed. Given our current state of
                   knowledge of gene function in <span class="emphasis"><em>C. elegans</em></span>, it is impossible to estimate the actual proportion of
                   duplicated genes in the genome that are redundant. This will become clearer as sensitized
                   screens and combinatorial RNAi approaches seek to address the general problem of assigning gene
                   function.
            </p>
            <p>One recent study examined the relationship between the prevalence of gene duplications
                   and ontogeny in <span class="emphasis"><em>Caenorhabditis</em></span> species (<a href="#bib2">Castillo-Davis and Hartl, 2002</a>). It was
                   found that genes expressed after embryogenesis had a significantly greater number of duplicates
                   than those expressed early in development. This was found to be true in both the <span class="emphasis"><em>C. elegans</em></span> and
                   <span class="emphasis"><em>C. briggsae</em></span> genomes. For example, 18.36<span class="symbol">%</span>  of early-expressed genes (n<span class="symbol">=</span>1,280) were found to have
                   detectable paralogs in the <span class="emphasis"><em>C. elegans</em></span> genome versus 35.31<span class="symbol">%</span>  of late-expressed genes (n<span class="symbol">=</span>1,014).
                   For <span class="emphasis"><em>C. briggsae</em></span>, the figures are even more striking, 6.7<span class="symbol">%</span>  (n<span class="symbol">=</span>165) and 38.8<span class="symbol">%</span>  (n<span class="symbol">=</span>237),
                   respectively (<a href="#bib2">Castillo-Davis and Hartl, 2002</a>). Therefore, duplicated copies of
                   early-expressed genes appear to be selectively lost. Based on earlier calculations of the rate
                   of origin of gene duplicates in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib16">Lynch and Conery, 2000</a>) and the
                   divergence between <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>, Castillo-Davis and Hartl estimated that more than
                   40<span class="symbol">%</span>  of all genes are expected to have duplicated at least once in both the <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C.
                         briggsae</em></span> lineages since their divergence (<a href="#bib2">Castillo-Davis and Hartl, 2002</a>). The
                   proportion of duplicated genes in the late-expressed class thus falls close to this estimate,
                   whereas the proportion of duplicated genes in the early-expressed class is much lower than
                   expected. It is suggested that the selective loss of duplicates of early-expressed genes
                   reflects developmental constraint (<a href="#bib2">Castillo-Davis and Hartl, 2002</a>). A related
                   study found that gene duplications were more prevalent amongst the set of conserved,
                   slowly-evolving genes versus the set of non-conserved genes in the <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>S. cerevisiae</em></span>
                   genomes, suggesting that slowly evolving genes may be the main source of new genes in eukaryotic
                   genomes (<a href="#bib4">Davis and Petrov, 2004</a>).
            </p>
            <p>11 of 33 of the largest clusters of duplicated
                   genes in <span class="emphasis"><em>C. elegans</em></span> consist of olfactory receptors (<a href="#bib21">Rubin et al., 2000</a>).
                   Olfactory receptors are seven-transmembrane G-protein coupled receptors (GPCRs), each of which
                   specifically recognises a set of odorant and tastant chemicals, allowing the worm to sense and
                   respond to its chemical environment. GPCRs comprise by far the largest multigene family in <span class="emphasis"><em>C.
                         elegans</em></span>, with over 1000 members, making up <span class="symbol">&#8764;</span> 5<span class="symbol">%</span>  of the genome (<a href="#bib1">Bargmann, 1998</a>),
                   although around a third of this group of genes are thought to be pseudogenes (<a href="#bib1">Bargmann, 1998</a>). Thus many duplications in this large multigene family are fated to become
                   nonfunctional. Several recent studies have charted the molecular evolution of some of these
                   genes and found that processes of duplication, diversification and movement that have led to
                   these large gene families are very much ongoing. For example, the 3126 bp inverse orientation
                   duplication that duplicates the 5&#8217; half of gene <a href="http://www.wormbase.org/db/gene/gene?name=T08H10.2" target="_blank">T08H10.2</a> to give rise to pseudogene T08H10.a is
                   thought to have occurred very recently as the duplicated sequences are identical (<a href="#bib19">Robertson, 1998</a>). It has been suggested that strong selective pressures are at work for the
                   continued functionality of these genes because the occurrence of synonymous changes in
                   duplicated GPCR genes is 11-fold higher than the occurrence of amino acid substitutions or
                   nonsynonymous changes (<a href="#bib19">Robertson, 1998</a>; <a href="#bib20">Robertson, 2000</a>). Indeed,
                   it has been proposed that natural selection might somehow favour duplications of genes that are
                   generally involved in responses to environmental stress and pathogens, in organisms facing a
                   challenging and dynamic molecular environment (<a href="#bib14">Lespinet et al., 2002</a>). The massive
                   lineage specific expansion of worm odorant/chemosensory receptors could be just one example of
                   this.
            </p>
            <p>Other large gene families in <span class="emphasis"><em>C. elegans</em></span> include C-type lectins, hormone receptors,
                   collagens and serine/threonine/tyrosine protein kinases. A full description of major
                   protein-coding gene families, including a discussion of family size distribution, can be found
                   in <a href="http://www.wormbook.org/chapters/www_genomclassprot/genomclassprot.html" target="_blank">Genomic classification of protein-coding gene families</a>. An example of a <span class="emphasis"><em>C. elegans</em></span> transcription factor gene family in which
                   duplications abound is the T-box family. There are 21 T-box genes in the <span class="emphasis"><em>C. elegans</em></span> genome
                   (WormBase release WS132, 2004), most of which lack clear orthologs in other species. There are 4
                   pairs of genes that are likely to have arisen from relatively recent duplication events and a
                   functional analysis of 2 of these pairs has been reported. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tbx-37" target="_blank">tbx-37</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tbx-38" target="_blank">tbx-38</a></em></span> (83<span class="symbol">%</span>  amino acid
                   identity) have redundant functions in mesoderm induction in <span class="emphasis"><em>C. elegans</em></span> embryos (<a href="#bib7">Good et al., 2004</a>), whereas <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tbx-8" target="_blank">tbx-8</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tbx-9" target="_blank">tbx-9</a></em></span> (59<span class="symbol">%</span>  amino acid identity), have overlapping, but
                   probably not completely redundant, functions in embryonic morphogenesis (<a href="#bib18">Pocock et al., 2004</a>). The most recent duplication in the T-box gene family would appear to be the one
                   that gave rise to the genes <a href="http://www.wormbase.org/db/gene/gene?name=Y59E9AR.3" target="_blank">Y59E9AR.3</a> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tbx-30" target="_blank">tbx-30</a></em></span>) and <a href="http://www.wormbase.org/db/gene/gene?name=Y59E9AR.5" target="_blank">Y59E9AR.5</a> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tbx-30" target="_blank">tbx-30.1</a></em></span>). These two sequences are
                   100<span class="symbol">%</span>  identical and situated in inverse orientations, <span class="symbol">&#8764;</span> 2 kb apart. Biological function has been
                   ascribed to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tbx-30" target="_blank">tbx-30</a></em></span> (<a href="#bib18">Pocock et al., 2004</a>), but it is unclear at present whether or
                   not <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tbx-30" target="_blank">tbx-30.1</a></em></span> is expressed.
            </p>
            <p>There are several other good examples of duplicated gene pairs that
                   have undergone neofunctionalization. A recent one is the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fbf-1" target="_blank">fbf-1</a>/<a href="http://www.wormbase.org/db/gene/gene?name=fbf-2" target="_blank">fbf-2</a></em></span> gene pair involved in germ
                   line development in <span class="emphasis"><em>C. elegans</em></span>. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fbf-1" target="_blank">fbf-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fbf-2" target="_blank">fbf-2</a></em></span> encode closely related RNA binding proteins that
                   reciprocally regulate the expression of one another in order to modulate the size of the germ
                   line mitotic region (<a href="#bib13">Lamont et al., 2004</a>). Thus, there appears to be a benefit
                   unique to the combined action of these two closely related genes; the duplication event in this
                   case has provided an opportunity for fine-tuning of a developmental process.
            </p>
            <p>A good potential
                   example of the subfunctionalization of a duplicated gene pair in <span class="emphasis"><em>C. elegans</em></span> is the case of the
                   Notch-like receptors <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span>. In <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> posess both overlapping
                   and separate biological functions. When both gene functions are removed, larval lethality
                   results (the Lag &#8211; Lin And Glp phenotype; <a href="#bib12">Lambie and Kimble, 1991</a>). However, in
                   <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span>, the Lag phenotype is seen when only <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> expression is silenced
                   (<a href="#bib22">Rudel and Kimble, 2002</a>). It is suggested that ancestral functions may have been
                   divided between <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> in <span class="emphasis"><em>C. elegans</em></span> after duplication in such a way that their total
                   capacity became reduced to the level of the single-copy ancestral gene. In <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span>, on the other hand, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> appears to have retained this ancestral function following
                   duplication (<a href="#bib22">Rudel and Kimble, 2002</a>).
            </p>
            <p>An interesting example of the apparent
                   gradual evolutionary demise of a duplicated gene is presented in the case of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-4" target="_blank">elt-4</a></em></span>. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-4" target="_blank">elt-4</a></em></span>
                   encodes a truncated GATA type zinc finger transcription factor of just 8.1kDa (72 amino acids),
                   and is situated <span class="symbol">&#8764;</span> 5 kb upstream of the highly conserved <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-2" target="_blank">elt-2</a></em></span>, proposed to be the original copy
                   of the duplicate pair (<a href="#bib5">Fukushige et al., 2003</a>). The bigger <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-2" target="_blank">elt-2</a></em></span> gene is
                   expressed in intestinal cells and is required for intestinal development (<a href="#bib6">Fukushige et al., 1998</a>). While it is clear that <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-4" target="_blank">elt-4</a></em></span> is expressed in the intestine, no effect was
                   found of deleting <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-4" target="_blank">elt-4</a></em></span>, in the gut or elsewhere, either alone or in combination with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-2" target="_blank">elt-2</a></em></span>.
                   Furthermore, experiments in yeast and <span class="emphasis"><em>in vitro</em></span> could not demonstrate any role for ELT-4 in
                   activating or repressing transcription, or indeed any specific DNA binding activity (<a href="#bib5">Fukushige et al., 2003</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-2" target="_blank">elt-2</a></em></span>/<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-4" target="_blank">elt-4</a></em></span> duplication event is thought to have occurred only
                   in the <span class="emphasis"><em>C. elegans</em></span> lineage, after the point at which <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> diverged. Thus,
                   the proposal is that although <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-4" target="_blank">elt-4</a></em></span> may have conferred some selective advantage to <span class="emphasis"><em>C. elegans</em></span> in
                   the past (hence the high level of conservation of the zinc finger domain), its ultimate
                   evolutionary fate will be disappearance from the <span class="emphasis"><em>C. elegans</em></span> genome (<a href="#bib5">Fukushige et al., 2003</a>).
            </p>
            <p>Genome sequence analysis is thus providing the community with far more than simply the informational content of genomes. Gene
               duplications are widespread in the <span class="emphasis"><em>C. elegans</em></span> genome and provide raw material for evolutionary novelty. The data are a snapshot of evolutionary time, from which we can
               glimpse into the past as well as, perhaps, seek to prophesy the future.
                   
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e660"></a>2.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>I would like to thank members of my laboratory and Andreas Russ for useful
                   discussions and the reviewers for helpful comments.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e665"></a>3.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                             <span class="bibliomisc">Bargmann, C.I. (1998). Neurobiology of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome. Science <span class="emphasis"><em>282</em></span>, 2028&#8211;2033.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851919&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.282.5396.2028" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                             <span class="bibliomisc">Castillo-Davis, C.I., and Hartl, D.L. (2002). Genome evolution and developmental constraint in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>19</em></span>, 728&#8211;735.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11961106&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                             <span class="bibliomisc">Cavalcanti, A.R.O., Ferreira, R., Gu, Z., and Li, W.-H. (2003). Patterns of gene duplication in <span class="emphasis"><em>Saccharomyces cerevisiae</em></span> and <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Mol. Evol. <span class="emphasis"><em>56</em></span>, 28&#8211;37.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12569420&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s00239-002-2377-2" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                             <span class="bibliomisc">Davis, J.C., and Petrov, D.A. (2004). Preferential duplication of conserved proteins in eukaryotic genomes. PLoS. Biol. <span class="emphasis"><em>2</em></span>, E55. Epub 2004 Mar 16.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15024414&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0020055" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fukushige, T., Goszczynski, B., Tian, H., and McGhee, J.D. (2003). The evolutionary duplication and probable demise of an
                        endodermal GATA factor in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>165</em></span>, 575&#8211;588.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14573471&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fukushige, T., Hawkins, M.G., and McGhee, J.D. (1998). The GATA-factor <span class="emphasis"><em>elt-2</em></span> is essential for formation of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> intestine, Dev. Biol. <span class="emphasis"><em>198</em></span>, 286&#8211;302.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9659934&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                             <span class="bibliomisc">Good, K., Ciosk, R., Nance, J., Neves, A., Hill, R.J., and Priess, J.R. (2004). The T-box transcription factors TBX-37 and
                        TBX-38 link GLP-1/Notch signaling to mesoderm induction in <span class="emphasis"><em>C. elegans</em></span> embryos, Development <span class="emphasis"><em>131</em></span>, 1967&#8211;1978. Epub 2004 Mar 31.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15056620&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.01088" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                             <span class="bibliomisc">Gu, X. (2003). Evolution of duplicate genes versus genetic robustness against null mutations. Trends Genet. <span class="emphasis"><em>19</em></span>, 354&#8211;356.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12850437&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0168-9525(03)00139-2" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                             <span class="bibliomisc">Gu, Z., Cavalcanti, A., Chen, F.-C., Bouman, P., and Li, W.-H. (2002). Extent of gene duplication in the genomes of <span class="emphasis"><em>Drosophila</em></span>, nematode and yeast. Mol. Biol. Evol. <span class="emphasis"><em>19</em></span>, 256&#8211;262.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11861885&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kamath, R.S., Fraser, A.G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., Kanapin, A., Le Bot, N., Moreno, S., Sohrmann, M.,
                        et al. (2003). Systematic functional analysis of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome using RNAi. Nature <span class="emphasis"><em>421</em></span>, 231&#8211;237.</span>
                     <a href="http://dx.doi.org/10.1038/nature01278" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                             <span class="bibliomisc">Katju, V., and Lynch, M. (2003). The structure and early evolution of recently arisen gene duplicates in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome. Genetics <span class="emphasis"><em>165</em></span>, 1793&#8211;1803.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14704166&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lambie, E.J., and Kimble, J. (1991). Two homologous regulatory genes, <span class="emphasis"><em>lin-12</em></span> and <span class="emphasis"><em>glp-1</em></span>, have overlapping functions. Development <span class="emphasis"><em>112</em></span>, 231&#8211;240.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1769331&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lamont, L.B., Crittenden, S.L., Bernstein, D., Wickens, M., Kimble, J. (2004). FBF-1 and FBF-2 regulate the size of the mitotic
                        region in the <span class="emphasis"><em>C. elegans</em></span> germline. Dev. Cell <span class="emphasis"><em>7</em></span>, 697&#8211;707.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15525531&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2004.09.013" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lespinet, O., Wolf, Y.I., Koonin, E.V., and Aravind, L. (2002). The role of lineage-specific expansion in the evolution of
                        eukaryotes. Genome Res. <span class="emphasis"><em>12</em></span>, 1048&#8211;1059.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12097341&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.174302" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lynch, M. (2002). Genomics. Gene duplication and evolution. Science <span class="emphasis"><em>297</em></span>, 945&#8211;947.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12169715&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1075472" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lynch, M., and Conery, J.S. (2000). The evolutionary fate and consequences of duplicate genes. Science <span class="emphasis"><em>290</em></span>, 1151&#8211;1155.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11073452&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.290.5494.1151" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                             <span class="bibliomisc">Mounsey, A., Bauer, P., and Hope, I.A. (2002). Evidence suggesting that a fifth of annotated <span class="emphasis"><em>Caenorhabditis elegans</em></span> genes may be pseudogenes. Genome Res. <span class="emphasis"><em>12</em></span>, 770&#8211;775.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11997343&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr208802. Article published online before print in April" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                             <span class="bibliomisc">Pocock, R., Ahringer, J., Mitsch, M., Maxwell, S., and Woollard, A. (2004). A regulatory network of T-box genes and the <span class="emphasis"><em>even-skipped</em></span> homologue <span class="emphasis"><em>vab-7</em></span> controls patterning and morphogenesis in <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>131</em></span>, 2373&#8211;2385. Epub 2004 Apr 21.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15102704&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.01110" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                             <span class="bibliomisc">Robertson, H.M. (1998). Two large families of chemoreceptor genes in the nematodes <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span> reveal extensive gene duplication, diversification, movement, and intron loss. Genome Res. <span class="emphasis"><em>8</em></span>, 449&#8211;463.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9582190&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                             <span class="bibliomisc">Robertson, H.M. (2000). The large srh family of chemoreceptor genes in <span class="emphasis"><em>Caenorhabditis</em></span> nematodes reveals processes of genome evolution involving large duplications and deletions and intron gains and losses. Genome
                        Res. <span class="emphasis"><em>10</em></span>, 192&#8211;203.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10673277&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.10.2.192" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                             <span class="bibliomisc">Rubin, G.M., Yandell, M.D., Wortman, J.R., Gabor Miklos, G.L., Nelson, C. R., Hariharan, I.K., Fortini, M.E., Li, P.W., Apweiler,
                        R., Fleischmann, W., et al. (2000). Comparative genomics of the eukaryotes. Science <span class="emphasis"><em>287</em></span>, 2204&#8211;2215.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10731134&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.287.5461.2204" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                             <span class="bibliomisc">Rudel, D., and Kimble, J. (2002). Evolution of discrete Notch-like receptors from a distant gene duplication in <span class="emphasis"><em>Caenorhabditis</em></span>. Evol. Dev. <span class="emphasis"><em>4</em></span>, 319&#8211;333.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12356262&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1046/j.1525-142X.2002.02027.x" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                             <span class="bibliomisc">The <span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium. (1998). Genome sequence of the nematode <span class="emphasis"><em>C. elegans</em></span>: a platform for investigating biology. Science <span class="emphasis"><em>282</em></span>, 2013&#8211;2018.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851916&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                             <span class="bibliomisc">Thomas, J.H. (1993). Thinking about genetic redundancy. Trends Genet. <span class="emphasis"><em>9</em></span>, 395&#8211;399.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8310537&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0168-9525(93)90140-D" target="_blank">Article</a>
                         
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e12" href="#d0e12">*</a></sup>Edited by Philip Anderson and Jonathan Hodgkin. Last revised June 8, 2005. Published June 25, 2005. This chapter should be
                  cited as: Woollard, A. Gene duplications and genetic redundancy in <span class="emphasis"><em>C. elegans</em></span> (June 25, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.2.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2005 Alison Woollard. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e40" href="#d0e40">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:woollard@bioch.ox.ac.uk" target="_top">woollard@bioch.ox.ac.uk</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

