<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>The phylogenetic relationships of Caenorhabditis and other rhabditids</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#phylo_ArticleID" title="The phylogenetic relationships of Caenorhabditis and other rhabditids">
      <link rel="next" href="#d0e154" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="phylorhab.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="phylo_ArticleID"></a>The phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> and other rhabditids<sup><a name="d0e11" href="#ftn.d0e11">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Karin Kiontke<em><span class="remark"><sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup></span></em>, David H. A. Fitch<em><span class="remark"><sup>&sect;</sup></span></em></span><br></span><span class="affiliation"><span class="address">Department of Biology, New York University, New York, NY 10003 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_nematodeevolecol.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Evolution and ecology
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=284.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=The+phylogenetic+relationships+of+Caenorhabditis+and+other+rhabditids" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e154">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#d0e225">2. Rhabditid phylogeny</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e228">2.1. Overview of rhabditid relationships</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e328">2.2. Relationships within <span class="emphasis"><em>Caenorhabditis</em></span>
                              			</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e450">2.3. Strongylids and diplogastrids are rhabditids</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e505">3. Examples of character evolution</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e508">3.1. Reproductive mode</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e571">3.2. Morphological evolution</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e640">3.3. Intron evolution</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e704">3.4. Molecular divergence</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e761">4. Resources for comparative biology using rhabditids</a></span></dt>
               <dt><span class="sect1"><a href="#d0e778">5. Acknowledgments</a></span></dt>
               <dt><span class="sect1"><a href="#d0e783">6. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>is a member of a group of nematodes called rhabditids,
                                 which encompasses a large number of ecologically and genetically diverse species. A new,
                                 preliminary phylogenetic analysis is presented for concatenated sequences of three
                                 nuclear genes for 48 rhabditid and diplogastrid species (including 10</strong></span>
                               <span class="emphasis"><em><span class="bold"><strong>Caenorhabditis</strong></span></em></span> <span class="bold"><strong>species), as well as four species representing
                                 the outgroup. Although many relationships are well-resolved, more data are still needed
                                 to resolve some key relationships, particularly near the base of the rhabditid tree.
                                 There is high confidence for two major clades: (1) a clade comprising</strong></span>
                               <span class="emphasis"><em><span class="bold"><strong>Mesorhabditis</strong></span></em></span> <span class="emphasis"><em><span class="bold"><strong>Parasitorhabditis,</strong></span></em></span>
                               <span class="emphasis"><em><span class="bold"><strong>Pelodera,</strong></span></em></span> <span class="emphasis"><em><span class="bold"><strong>Teratorhabditis</strong></span></em></span> <span class="bold"><strong>plus a few other
                                 species; (2) a large clade (Eurhabditis) comprising most of the remaining rhabditid
                                     genera, including</strong></span> <span class="emphasis"><em><span class="bold"><strong>Caenorhabditis</strong></span></em></span> <span class="bold"><strong>and its sistergroup</strong></span>
                               <span class="emphasis"><em><span class="bold"><strong>Protorhabditis-Prodontorhabditis-Diploscapter.</strong></span></em></span> <span class="bold"><strong>Eurhabditis also contains the parasitic
                                     strongylids, the entomopathogenic</strong></span> <span class="emphasis"><em><span class="bold"><strong>Heterorhabditis,</strong></span></em></span> <span class="bold"><strong>and the
                                     monophyletic group</strong></span> <span class="emphasis"><em><span class="bold"><strong>Oscheius</strong></span></em></span> <span class="bold"><strong>which includes the satellite model organism</strong></span> <span class="emphasis"><em><span class="bold"><strong>O. tipulae.</strong></span></em></span> <span class="bold"><strong>The
                                     relationships within</strong></span> <span class="emphasis"><em><span class="bold"><strong>Caenorhabditis</strong></span></em></span> <span class="bold"><strong>are well resolved. The
                                 analysis also suggests that rhabditids include diplogastrids, to which the second
                                     satellite model organism</strong></span> <span class="emphasis"><em><span class="bold"><strong>Pristionchus pacificus</strong></span></em></span> <span class="bold"><strong>belongs. Genetic
                                     disparity within</strong></span> <span class="emphasis"><em><span class="bold"><strong>Caenorhabditis</strong></span></em></span> <span class="bold"><strong>is as great as that across
                                 vertebrates, suggesting</strong></span> <span class="emphasis"><em><span class="bold"><strong>Caenorhabditis</strong></span></em></span> <span class="bold"><strong>lineages are quickly evolving, ancient, or both. The
                                 phylogenetic tree can be used to reconstruct evolutionary events within rhabditids. For
                                 instance, the reproductive mode changed multiple times from gonochorism to
                                 hermaphroditism, but only once from hermaphroditism to gonochorism. Complete retraction
                                     of the male tail tip, leading to a blunt, peloderan tail, evolved
                                 at least once. Reversions to unretracted tail tips occurred within both major rhabditid
                                 groups. The phylogeny also provides a guide to species which would be good candidates
                                 for future genome projects and comparative studies.</strong></span>
               		
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e154"></a>1.&nbsp;Introduction<sup><a name="d0e157" href="#ftn.d0e157">&#8224;</a></sup></h2>
                  </div>
               </div>
            </div>
            <p>
               			<span class="emphasis"><em>Caenorhabditis</em></span> belongs to a large group of primarily
                           bacteriophagous nematodes called rhabditids (<a href="#figure1">Figure 1</a>). This taxon
                           comprises a great diversity of species, including some parasites (see <a href="http://www.wormbook.org/chapters/www_quicktourdiversity/quicktourdiversity.html" target="_blank">A quick tour of nematode diversity and the backbone of nematode phylogeny</a>). Resolving rhabditid phylogeny is key to uncovering how
                           evolutionary changes occurred in this diversification. A phylogeny for rhabditids also
                           provides an important foundation for comparative biology using <span class="emphasis"><em>C.
                                 elegans</em></span>, "satellite" model systems <span class="emphasis"><em>Pristionchus
                                 pacificus</em></span>, <span class="emphasis"><em>Oscheius tipulae</em></span>, and other
                               <span class="emphasis"><em>Caenorhabditis</em></span> species with sequenced genomes (<span class="emphasis"><em>C.
                                     briggsae</em></span>, <span class="emphasis"><em>C. remanei</em></span>, <span class="emphasis"><em>C.
                                 japonica</em></span>, and <span class="emphasis"><em>Caenorhabditis</em></span> n. sp. represented by
                           strain CB5161 among others; see <a href="http://www.wormbook.org/chapters/www_evoldevnematode/evoldevnematode.html" target="_blank">Evolution of development in nematodes related to <span class="emphasis"><em>C. elegans</em></span></a>). The selection of appropriate representative taxa for further genome
                           sequencing projects also depends on an accurate phylogeny.
            </p>
            <p>Phylogenetic analyses using DNA sequences of small subunit ribosomal RNA (SSU rRNA)
                           genes have revealed some relationships that are inconsistent, as well as some that are
                           consistent, with earlier phylogenies (<a href="#bib8">Fitch et al., 1995</a>; <a href="#bib3">Blaxter et al., 1998</a>; <a href="#bib20">Sudhaus and Fitch, 2001</a>). Many relationships, however,
                           have not been sufficiently resolved. It is therefore important to expand the data pool
                           with regard to both loci and taxa.
            </p>
            <p>Here we report conclusions of an analysis (which will be published elsewhere) using three genes for rhabditids and
                           their closest relatives. In this phylogeny, several branches are well supported. For
                           example, most species belong to either of two major monophyletic groups, here called
                           "Eurhabditis" and "Pleiorhabditis", both of which show high support in this analysis.
                           Eurhabditis includes strongylids and <span class="emphasis"><em>Heterorhabditis</em></span>, as well as
                               <span class="emphasis"><em>Caenorhabditis</em></span> and its sister taxon, the Protorhabditis group.
                           There is also some evidence that diplogastrids (e.g., <span class="emphasis"><em>Pristionchus</em></span>)
                           are part of rhabditids. As we accumulate more data, we are gaining a clearer picture of
                           rhabditid phylogeny. There is still much that remains mysterious, however. In
                           particular, it will be important to establish who are the closest relatives of
                           rhabditids, to sort out the earliest lineages of rhabditids, and to further elucidate
                           the relationships among members of the Eurhabditis group. Only after this is achieved
                           should the much-needed taxonomic revision be initiated. At the moment, however, it is
                           still too early to attempt such a classification because a number of relationships are
                           not sufficiently resolved.
            </p>
            <p>A well-resolved phylogeny is an essential step in inferring the evolutionary changes
                           that have occurred and for discovering possible correlations between such changes. We
                           provide examples of how the phylogeny so far has provided insight into the evolution of
                           reproductive mode, morphological evolution in the male tail tip, the evolution of
                           introns, and molecular divergence. Database and strain resources for rhabditids are
                           being developed to support comparative work with rhabditids.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e225"></a>2.&nbsp;Rhabditid phylogeny
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e228"></a>2.1.&nbsp;Overview of rhabditid relationships
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A preliminary phylogenetic hypothesis for representative rhabditid species and
                                  four species from the outgroup is shown in <a href="#figure1">Figure 1</a>. The analysis
                                  is based on DNA sequences from nearly complete small and large subunit (SSU and LSU)
                                  ribosomal RNA (rRNA) genes and a portion of the RNA Pol II (RNAP2) gene (KK and DF,
                                  unpublished; original data and details of the analysis will be published elsewhere).
               </p>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="522">
<tr>
  <td align="center"><p><a href="phylofig1.jpg"><img src="phylofig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. Rhabditid phylogeny as inferred by weighted parsimony jackknife analysis
                                       using DNA sequences from three genes (SSU and LSU rRNA and RNA polymerase II).&nbsp;</b>Strain designations are given for species available as stocks. Gray taxa names
                                    designate outgroup representatives. Some of the clearer monophyletic groups are
                                    delineated with brackets. Numbers on the branches indicate the percentage of 1,000
                                    jackknife replicates in which the branch appeared (50% of randomly chosen sites were
                                    deleted in each replicate). For each jackknife replicate, four random sequences of taxon
                                    addition were used to generate starting trees. Branches were collapsed where jackknife
                                    values were below 50%. The position of the strongylid clade, inferred from previous
                                    analyses (<a href="#bib3">Blaxter et al., 1998</a>; <a href="#bib11">Fitch and Thomas, 1997</a>), is shown as
                                    a dashed line. (KK and DF, unpublished; details of the analysis will be published
                                    elsewhere.). Right-click or control-click to see larger image.
                     </p>
                  </div>
               </div>
               <p>Two major clades of rhabditids can be distinguished, called here "Pleiorhabditis"
                                  and "Eurhabditis" (<a href="#figure1">Figure 1</a>). Although these two groups were
                                  combined in a single monophyletic group (supported by 87% bootstrap value) in an
                                  earlier analysis using only SSU rRNA genes (<a href="#bib20">Sudhaus and Fitch, 2001</a>), such a clade
                                  does not appear in the present jackknife analysis with additional genes.
                                  Pleiorhabditis comprises several genera of species with a posterior vulva (e.g.,
                                      <span class="emphasis"><em>Mesorhabditis</em></span>, <span class="emphasis"><em>Teratorhabditis</em></span>) and
                                  also the genus <span class="emphasis"><em>Pelodera</em></span> which includes <span class="emphasis"><em>Pelodera</em></span>
                  				<span class="emphasis"><em>strongyloides</em></span>, reported to be a facultative parasite of mammals
                                  (<a href="#bib23">Sudhaus and Schulte, 1988</a>; <a href="#bib13">Jones et al., 1991</a>). Eurhabditis includes
                                  almost all the other rhabditid species, including
                                  <span class="emphasis"><em>Caenorhabditis</em></span>, and thus differs from the Eurhabditis
                                  originally proposed by Sudhaus on the basis of a different view of relationships
                                  (see <a href="#bib20">Sudhaus and Fitch, 2001</a>). Additionally, Eurhabditis includes the "Protorhabditis
                                  group", named after its most prominent members, which is the sister clade to
                                      <span class="emphasis"><em>Caenorhabditis</em></span>. Eurhabditis includes a monophyletic genus
                                      <span class="emphasis"><em>Oscheius</em></span> that comprises two separate clades, one of
                                  leptoderan species, the <span class="emphasis"><em>Insectivora</em></span> group, and another of
                                      peloderan species, the <span class="emphasis"><em>Dolichura</em></span>
                                  group, as proposed by <a href="#bib21">Sudhaus and Hooper (1994</a>; see also <a href="#figure4">Figure 4</a>). One member of this latter group, <span class="emphasis"><em>Oscheius
                                            tipulae</em></span> (represented here by strain CEW1) is used as a satellite
                                  model organism for comparative studies of development (see <a href="http://www.wormbook.org/chapters/www_evoldevnematode/evoldevnematode.html" target="_blank">Evolution of development in nematodes related to <span class="emphasis"><em>C. elegans</em></span></a>; <a href="#bib18">Sommer, 2000</a>) and molecular biology (e.g. <a href="#bib7">Evans et al., 1997</a>). The strongylid parasites of vertebrates and the
                                      <span class="emphasis"><em>Heterorhabditis</em></span> pathogens of insects have also clearly
                                  evolved from within the Eurhabditis clade (see <a href="http://www.wormbook.org/chapters/www_quicktourdiversity/quicktourdiversity.html" target="_blank">A quick tour of nematode diversity and the backbone of nematode phylogeny</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e328"></a>2.2.&nbsp;Relationships within <span class="emphasis"><em>Caenorhabditis</em></span>
                           			
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Relationships are now well-resolved among the <span class="emphasis"><em>Caenorhabditis</em></span>
                                  species which are represented by live strains (<a href="#figure2">Figure 2</a>; <a href="#bib14">Kiontke et al., 2004</a>; <a href="#bib4">Cho et al., 2004</a>). Of these species,
                                      <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span> are closest
                                  relatives. No single species is most closely related to <span class="emphasis"><em>C.
                                        elegans</em></span>, which is the sister species of a clade that includes
                                      <span class="emphasis"><em>C. briggsae</em></span>, <span class="emphasis"><em>C. remanei</em></span>, and
                                      <span class="emphasis"><em>Caenorhabditis</em></span> n. sp. (represented here by strain CB5161).
                                  The closest species to these four <span class="emphasis"><em>Elegans</em></span> group species is
                                      <span class="emphasis"><em>C. japonica</em></span>, which was chosen for a genome sequencing
                                  project in order to provide an outgroup comparison for the four
                                  <span class="emphasis"><em>Elegans</em></span> group species. Another undescribed new species
                                  (represented here by strain PS1010), which has been used in some comparisons (e.g.
                                  <a href="#bib2">Baldwin et al., 1997</a>), diverged earlier than <span class="emphasis"><em>C.
                                        japonica</em></span>. Uncertainty still exists for some
                                  <span class="emphasis"><em>Caenorhabditis</em></span> relationships; instead of being related in the
                                  manner depicted in <a href="#figure2">Figure 2</a>, <span class="emphasis"><em>C.</em></span> sp. PS1010 could be most closely
                                  related to <span class="emphasis"><em>C. drosophilae</em></span> and
                                  <span class="emphasis"><em>Caenorhabditis</em></span> n. sp. (DF5070) (<a href="#bib14">Kiontke et al., 2004</a>). For more about <span class="emphasis"><em>Caenorhabditis</em></span> biodiversity and a
                                  phylogenetic hypothesis incorporating other <span class="emphasis"><em>Caenorhabditis</em></span>
                                  species, see <a href="http://www.wormbook.org/chapters/www_ecolCaenorhabditis/ecolCaenorhabditis.html" target="_blank">Ecology of <span class="emphasis"><em>Caenorhabditis</em></span> species</a>
                                  (also <a href="#bib22">Sudhaus and Kiontke, 1996</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="180">
<tr>
  <td align="center"><p><a href="phylofig2.jpg"><img src="phylofig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Phylogeny of <span class="emphasis"><em>Caenorhabditis</em></span> species represented by
                                       living strains and four outgroup representatives.&nbsp;</b>Black numbers on the branches denote percentage jackknife support from 2,000
                                    replicates for an analysis incorporating all taxa shown using SSU and LSU rRNA genes and
                                    RNAP2 with the third codon positions deleted. Red numbers denote jackknife support in
                                    5,000 replicates for an analysis that included the seven most closely related
                                    <span class="emphasis"><em>Caenorhabditis</em></span> species for SSU and LSU rRNA genes, and coding regions from genes for
                                        RNAP2, <span class="emphasis"><em>par-6</em></span>, and <span class="emphasis"><em>pkc-3</em></span> with the third
                                    codon positions included. Third codon positions were only saturated with substitutions
                                    in pairwise comparisons involving species less closely related to <span class="emphasis"><em>C.
                                          elegans</em></span> than <span class="emphasis"><em>C.</em></span> sp. PS1010. (Modified from <a href="#bib14">Kiontke et al., 2004</a>). Right-click or control-click to see larger image.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e450"></a>2.3.&nbsp;Strongylids and diplogastrids are rhabditids
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Rhabditids, as a monophyletic group, include two taxa that have been previously
                                  classified separately. First, the strongylids are a group of vertebrate parasites
                                  traditionally considered outside of rhabditids. Molecular analyses have placed
                                  strongylids clearly within the Eurhabditis group of rhabditids and as a sister group
                                  of the insect pathogenic <span class="emphasis"><em>Heterorhabditis</em></span> (<a href="#bib11">Fitch and Thomas, 1997</a>; <a href="#bib3">Blaxter et al., 1998</a>; <a href="#bib20">Sudhaus and Fitch, 2001</a>; <a href="#bib6">De
                                     Ley and Blaxter, 2002</a>).
               </p>
               <p>A second group of species traditionally classified outside of rhabditids is the
                                  group designated here informally as diplogastrids (called Diplogastrina,
                                  Diplogastridae, or Diplogasteromorpha in formal classification systems), which
                                  includes the satellite model system, <span class="emphasis"><em>Pristionchus pacificus</em></span>
                                  (see <a href="http://www.wormbook.org/chapters/www_evoldevnematode/evoldevnematode.html" target="_blank">Evolution of development in nematodes related to <span class="emphasis"><em>C. elegans</em></span></a>). Specifically, there appears to be a relationship between diplogastrids
                                  and <span class="emphasis"><em>Rhabditoides inermis</em></span>, a rhabditid (see also <a href="#bib20">Sudhaus and
                                     Fitch, 2001</a>). According to this phylogeny, <span class="emphasis"><em>R. inermis</em></span>
                                  is more closely related to diplogastrids than to its traditional congener, <span class="emphasis"><em>Rhabditoides inermiformis</em></span>, despite their morphological similarities. (Another
                                  species, <span class="emphasis"><em>Rhabditoides regina</em></span>, resides within Pleiorhabditis,
                                  thus making "<span class="emphasis"><em>Rhabditoides</em></span>" polyphyletic.) Furthermore, our
                                  phylogeny suggests that <span class="emphasis"><em>Poikilolaimus</em></span> could be in the most
                                  anciently diverged lineage of rhabditids, placing diplogastrids distinctly within
                                  rhabditids, although the jackknife support (58%) is not strong for this conclusion.
                                  Together, these data suggest that diplogastrids are not a clade separate from
                                  rhabditids, but are instead derived from within rhabditids. These data contradict a
                                  recent proposal based on morphology that bunonematids are the sister taxon of diplogastrids (<a href="#bib12">F&uuml;rst von Lieven, 2002</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e505"></a>3.&nbsp;Examples of character evolution
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e508"></a>3.1.&nbsp;Reproductive mode
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Using the molecular phylogeny, we have traced the evolution of gonochorism (males
                                  and females must cross to propagate), self-fertile hermaphroditism (see <a href="http://www.wormbook.org/chapters/www_sexgermline/sexgermline.html" target="_blank">Sex-determination in the germ line</a>), heterogony (alternating generations of gonochorism and hermaphroditism), and parthenogenesis (sperm, if they exist, do
                  not contribute genetic material; <a href="#figure3">Figure 3</a>). Among the taxa represented in <a href="#figure3">Figure 3</a>, we find that gonochorism is ancestral and that hermaphroditism has arisen at least ten times independently in all the
                                  major rhabditid clades, with only one possible reversal back to gonochorism
                                  occurring within the <span class="emphasis"><em>Dolichura</em></span> species group of
                                      <span class="emphasis"><em>Oscheius</em></span>. Such massive convergence provides an excellent
                                  opportunity to study how significantly developmental constraints limit or channel
                                  evolution.
               </p>
               <p>According to the most parsimonious reconstruction, and assuming reversibility and
                                  equal chances of changes between character states, <span class="emphasis"><em>C. briggsae</em></span>
                                  and <span class="emphasis"><em>C. elegans</em></span> evolved hermaphroditism independently (<a href="#bib14">Kiontke et al., 2004</a>). If we consider that changes from gonochorism to
                                  hermaphroditism have occurred ten times more frequently than the reverse in
                                  rhabditids, it becomes even more plausible that hermaphroditism evolved
                                  independently in these two species. In such a case, there are two possibilities for
                                  evolutionary change: parallelism (in which independent changes affect the same
                                  mechanism) or convergence (in which the changes affect different mechanisms but
                                  yield similar results). In the case of <span class="emphasis"><em>C. elegans</em></span> and
                                      <span class="emphasis"><em>C. briggsae</em></span>, hermaphroditism appears to be convergent
                                  because there is a fundamental difference between the two species in the identity of
                                  <a href="http://www.wormbase.org/db/seq/protein?name=FOG-2;class=protein" target="_blank">FOG-2</a>, for which no single ortholog exists in <span class="emphasis"><em>C. briggsae</em></span>, and
                                  developmental role of <a href="http://www.wormbase.org/db/seq/protein?name=GLD-1;class=protein" target="_blank">GLD-1</a>, which inhibits female fate in the germline of
                                      <span class="emphasis"><em>C. elegans</em></span> but promotes female fate in <span class="emphasis"><em>C.
                                            briggsae</em></span> (<a href="#bib15">Nayak et al., 2005</a>; see <a href="http://www.wormbook.org/chapters/www_sexgermline/sexgermline.html" target="_blank">Sex-determination in the germ line</a>).
               </p>
               <p>We also find that heterogony evolved twice (in
                                  <span class="emphasis"><em>Heterorhabditis</em></span> and an undescribed species represented by
                                  strain SB347) and parthenogenesis evolved once (within the Protorhabditis
                              group).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e571"></a>3.2.&nbsp;Morphological evolution
                        </h3>
                     </div>
                  </div>
               </div>
               <p>As an example of morphological evolution, we have traced the history of changes in
                                  morphogenesis of the male tail tip. In <span class="emphasis"><em>C. elegans</em></span>, the shapes
                                  of the hypodermal cells in the male tail tip change dramatically as the cells fuse
                                  during L4 male morphogenesis and the conically pointed tail tip of the larva
                                  retracts to form the rounded, "peloderan" tail of the adult
                                  male (<a href="#bib16">Nguyen et al., 1999</a>). Tail morphogenesis does not occur in
                                  hermaphrodites or in females of most other rhabditid species. Although males of most
                                  rhabditid species do undergo tail morphogenesis, the tip of the tail does not always
                                  retract. In such cases, the tail tip remains pointed, as in larvae, usually sticking
                                  out beyond the posterior edge of the fan or bursa, if a fan exists. This tail form
                                  is called "leptoderan".
               </p>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="423">
<tr>
  <td align="center"><p><a href="phylofig3.jpg"><img src="phylofig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 3. Evolution of reproductive mode.&nbsp;</b>Reproductive modes were determined for the terminal taxa and mapped onto the
                                    molecular phylogeny depicted in <a href="#figure1">Figure 1</a>. Two <span class="emphasis"><em>Oscheius</em></span> species and
                                    the lineage to strongylids (dashed lines) were added according to their position in
                                    phylogenies obtained with SSU rDNA alone. Red branches represent lineages in which
                                    hermaphroditism evolved; blue branches represent lineages in which gonochorism occurred;
                                    orange indicates heterogonic mode (different generations switch between gonochoristism
                                    and hermaphroditism); green indicates probable or known parthenogens. Character states
                                    for ancestral lineages were inferred by parsimony under the assumption that character
                                    state changes are equally likely and reversible. Right-click or control-click to see larger image.
                     </p>
                  </div>
               </div>
               <p>There have been at least seven independent changes in the shape of the male tail
                                  tip during rhabditid evolution (<a href="#figure4">Figure 4</a>). Because some branches
                                  are still unresolved, the directionality of most of these changes is unclear.
                                  However, the present phylogeny allows us to conclude that at least two changes
                                  occurred from peloderan to leptoderan. Some of the changes in
                                  male tail tips must be convergent because we know that
                                  peloderan tail tips can be constructed in different ways (e.g.,
                                  with or without fusions of the hypodermal cells), as can leptoderan tail tips
                                  (<a href="#bib16">Nguyen et al., 1999</a>; <a href="#bib9">Fitch, 1997</a>; <a href="#bib10">Fitch, 2000</a>). Because of
                                  these convergences, it is clear that there is no monophyletic clade of leptoderan
                                  species or of peloderan species (cf. Peloderinae of <a href="#bib1">Andr&aacute;ssy, 1983</a>). We have found that several other morphological characters (such as
                                  lack of glottoid apparatus) show similar homoplasy at gross morphological levels
                                  (e.g., <a href="#bib20">Sudhaus and Fitch, 2001</a>). Thus, rhabditids provide fertile ground for
                                  investigating the role of developmental plasticity in evolution.
               </p>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="468">
<tr>
  <td align="center"><p><a href="phylofig4.jpg"><img src="phylofig4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 4. Evolution of male tail tip morphogenesis.&nbsp;</b>Orange branches represent evolutionary lineages in which males had unretracted tail
                                    tips, which are therefore pointed in shape and referred to as "leptoderan" if there is a
                                    fan; green represents lineages in which males had retracted tail tips, which are thus
                                    rounded in shape and are referred to as "peloderan," as in <span class="emphasis"><em>C. elegans</em></span>. Gray represents
                                    lineages for which the character state cannot be decided. Using the phylogeny depicted
                                    in <a href="#figure1">Figure 1</a>, ancestral states were inferred by parsimony under the assumption that
                                    character state changes are equally likely and reversible. Right-click or control-click to see larger image.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e640"></a>3.3.&nbsp;Intron evolution
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Using the well-resolved <span class="emphasis"><em>Caenorhabditis</em></span> phylogeny, we inferred
                                  the most parsimonious history of intron gains and losses in a 1,860-bp portion of
                                  the gene coding for the largest subunit of RNAP2 (<a href="#bib14">Kiontke et al., 2004</a>). This phylogeny of closely related species allows finer resolution of
                                  such events than do comparisons among distantly related model organisms, which have
                                  been the main staple of intron evolution fare (e.g. <a href="#bib17">Rogozin et al., 2003</a>). We find that intron evolution is extraordinarily dynamic (<a href="#figure5">Figure 5</a>). In this one region, the number of introns present in a
                                  species can range between 1 and 14. Considering only homologous intron positions at
                                  which a single most parsimonious reconstruction could be made ("unequivocal"
                                  positions) and assuming that intron gains and losses are equally likely, introns
                                  were lost 12 times and gained 4 times (<a href="#bib14">Kiontke et al., 2004</a>). If one
                                  considers the possibility that losses are more likely than gains and that it might
                                  be highly improbable for an intron to be regained in exactly the same spot, it is
                                  possible that as many as 27 losses and at most 3 gains (but at least 1 gain)
                                  occurred. Most of these intron losses occurred in the stem species of the
                                      <span class="emphasis"><em>Caenorhabditis</em></span> clade, and several losses occurred later as
                                  the clade evolved (<a href="#figure5">Figure 5</a>). One intron in <span class="emphasis"><em>C.</em></span> sp. CB5161 is unique
                                  among all rhabditids we have characterized so far, and is thus a clear gain (<a href="#figure5">Figure 5</a>). Characterization of the homologous region in other
                                  rhabditids suggests that intron evolution is very dynamic with periods of losses and
                                  gains (KK and DF, unpub.). Frequent intron loss in other
                                  <span class="emphasis"><em>Caenorhabditis</em></span> genes has also been reported (<a href="#bib4">Cho et al., 2004</a>).
                                  Recent intron gains have also been documented, some of which have sequence
                                  similarity to other introns and could therefore have originated by reverse splicing
                                  (<a href="#bib5">Coghlan and Wolfe, 2004</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="306">
<tr>
  <td align="center"><p><a href="phylofig5.jpg"><img src="phylofig5_s.jpg" border="2" align="middle" alt=" figure 5"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 5. Intron evolution in part of the RNAP2 gene of
                                       <span class="emphasis"><em>Caenorhabditis</em></span> species.&nbsp;</b>The presence (+) or absence (-) of
                                    introns at particular sites (a-q) in the RNAP2 gene is depicted as a matrix (question
                                    marks indicate missing data). For sites a, b, e, g, h, k-o, and q, a single,
                                    unequivocal, most parsimonious history of intron insertion/deletion could be inferred
                                    (rectangles on the branches of the phylogeny). For the other sites, more than one most
                                    parsimonious scenario exists under the assumption that gains and losses are equally
                                    probable. Sites at which a single deletion occurred during
                                    <span class="emphasis"><em>Caenorhabditis</em></span> cladogenesis are marked in dark blue; purple
                                    indicates sites at which two deletions occurred; light blue and orange indicate sites at
                                    which deletions and subsequent reinsertions could have occurred; red indicates a unique
                                    intron insertion. (Modified from <a href="#bib14">Kiontke et al., 2004</a>). Right-click or control-click to see larger image.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e704"></a>3.4.&nbsp;Molecular divergence
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Genome-level comparisons of molecular divergence show that <span class="emphasis"><em>C.
                                        elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> are a little bit less
                                  similar at the molecular level than are human and mouse (<a href="#bib19">Stein et al., 2003</a>). We find the same relative differences when comparing these species
                                  with RNAP2 and rRNA genes alone (<a href="#bib14">Kiontke et al., 2004</a>). Because many
                                  taxa are represented by SSU rRNA sequences, we can use this molecule to compare the
                                  degree of molecular divergences among other vertebrate taxa with the divergences
                                  among rhabditids. The divergence between the least closely related
                                      <span class="emphasis"><em>Caenorhabditis</em></span> species (<span class="emphasis"><em>C. briggsae</em></span>
                                  versus <span class="emphasis"><em>C.</em></span> sp. SB341) is comparable to the divergence between
                                  mouse and zebrafish, encompassing a broad range of vertebrate lineages (<a href="#figure6">Figure 6</a>). Furthermore, species within the Eurhabditis clade are at
                                  least as different at the molecular level as mouse and sea urchin, encompassing a
                                  broad range of deuterostome lineages. It is important to note that the Eurhabditis
                                  clade is but a small fraction of Nematoda. Thus, the nematodes diverged from other
                                  animals very anciently, rates of molecular evolution are much faster in nematodes
                                  than in deuterostomes, or both. Because there is no accurate method for calibrating
                                  a "molecular clock" for nematodes, estimates for dates of divergence will be
                                  extremely unreliable. Complicating this issue, there is considerable rate
                                  heterogeneity among different lineages for some genes (e.g., SSU rRNA; see <a href="#bib14">Kiontke et al., 2004</a>). The best measure of taxonomic difference in rhabditids
                                  is therefore simply the relative amount of sequence difference (as in <a href="#figure6">Figure 6</a>), not some date of divergence which could be very inaccurate.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="288">
<tr>
  <td align="center"><p><a href="phylofig6A.jpg"><img src="phylofig6A_s.jpg" border="2" align="middle" alt=" figure 6A"></a>
  </td>
</tr>
                  </table>
               </div>
               <div class="mediaobject" align="center">
                  <table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="phylofig6B.jpg"><img src="phylofig6B_s.jpg" border="2" align="middle" alt=" figure 6A"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 6. (A) Comparisons of evolutionary divergences in SSU rRNA genes between
                                       species pairs within nematodes (red) and deuterostomes (green).&nbsp;</b>Pairwise divergences were estimated by summing lengths of intervening branches between
                                    taxa in (B); standard deviations (error bars) were calculated as the square root of the
                                    sum of the squares of the SDs of these branches. (B) Likelihood phylograms for SSU rRNA
                                    genes used to calculate the divergences graphed in (A). Species relationships within
                                    each group were assumed. A general time-reversible model (accounting for rate
                                    heterogeneity across sites with some fraction of invariant sites, GTR+I+<span class="symbol">&#915;</span>) was used to estimate branch lengths (substitutions per site
                                        &plusmn; 1 standard deviation). Divergences
                                    between any pair of taxa were then calculated as the sum of the branch lengths
                                    separating those taxa on the tree. (Modified from Kiontke, 2004). Right-click or control-click to see larger image.
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e761"></a>4.&nbsp;Resources for comparative biology using rhabditids
                     </h2>
                  </div>
               </div>
            </div>
            <p>Arguably the most important tools for comparative studies using emerging model systems
                           are the organisms themselves. Genome sequences, interactome maps, cDNA libraries, and
                           other such molecular data and resources are important tools for functional genomics.
                           Ultimately, however, <span class="emphasis"><em>in vivo</em></span> understanding of gene and genome
                           function - particularly with respect to mechanisms and patterns of gene expression,
                           development and behavior - requires analysis in the context of the living system itself.
                           Powerful comparative approaches depend on the availability of multiple, phylogenetically
                           well-defined, living species with clearly defined, standardized strains that can be
                           archived and shared. The <span class="emphasis"><em>Caenorhabditis</em></span> Genetics Center <a href="http://biosci.umn.edu/CGC/CGChomepage.htm" target="_blank">(CGC)</a> provides many stocks of wild strains that can be easily maintained cryogenically. We are
                           also developing a stock collection of rhabditid species, the New York University Rhabditidae Collection (<a href="http://www.nyu.edu/projects/fitch/fresources/fresources.html" target="_blank">NYURC</a>) that is as
                           thoroughly representative of rhabditid phylogeny as possible. The NYURC curates strains
                           that are difficult to culture or archive cryogenically along with data about ecology,
                           biology, morphology, etc., for all rhabditids in culture.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e778"></a>5.&nbsp;Acknowledgments
                     </h2>
                  </div>
               </div>
            </div>
            <p>We thank the National Science Foundation and the Human Frontier Science Program for
                           supporting this work. Although many people have been important to this work, we
                           especially thank Marie-Anne F&eacute;lix, W. Kelley Thomas and Walter Sudhaus for their important collaborations and
               contributions.
                           We thank Byron Adams and Steve Nadler for providing excellent suggestions in their
                           reviews of the manuscript.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e783"></a>6.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                             <span class="bibliomisc">Andr&aacute;ssy, I. (1983). A Taxonomic Review of the Suborder Rhabditina (Nematoda: Secernentia) (Paris: ORSTOM).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                             <span class="bibliomisc">Baldwin, J.G., Frisse, L.M., Vida, J.T., Eddleman, C.D., and Thomas, W.K. (1997). An evolutionary framework for the study
                        of developmental evolution in a set of nematodes related to <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Phylogenet. Evol. <span class="emphasis"><em>8</em></span>, 249&#8211;259.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=97446281&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/mpev.1997.0433" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                             <span class="bibliomisc">Blaxter, M.L., De Ley, P., Garey, J.R., Liu, L.X., Scheldeman, P., Vierstraete, A., Vanfleteren, J.R., Mackey, L.Y., Dorris,
                        M., Frisse, L.M., et al. (1998). A molecular evolutionary framework for the phylum Nematoda. Nature <span class="emphasis"><em>392</em></span>, 71&#8211;75.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9510248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/32160" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                             <span class="bibliomisc">Cho, S., Jin, S.W., Cohen, A., and Ellis, R.E. 2004. A phylogeny of <span class="emphasis"><em>Caenorhabditis</em></span> reveals frequent loss of introns during nematode evolution. Genome Res. <span class="emphasis"><em>14</em></span>, 1207&#8211;1220.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15231741&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.2639304" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                             <span class="bibliomisc">Coghlan, A., and Wolfe, K.H. (2004). Origins of recently gained introns in <span class="emphasis"><em>Caenorhabditis</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 11362&#8211;11367.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15243155&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0308192101" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                             <span class="bibliomisc">De Ley, P., and Blaxter, M. (2002). Systematic position and phylogeny. In The Biology of Nematodes, D.L. Lee, ed. (New York:
                        Taylor &amp; Francis), pp. 1&#8211;30.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                             <span class="bibliomisc">Evans, D., Zorio, D., MacMorris, M., Winter, C.E., Lea, K., and Blumenthal, T. (1997). Operons and SL2 trans-splicing exist
                        in nematodes outside the genus <span class="emphasis"><em>Caenorhabditis</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>94</em></span>, 9751&#8211;9756.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=97420754&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.94.18.9751" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fitch, D.H.A., Bugaj-Gaweda, B., and Emmons, S.W. (1995). 18S ribosomal RNA gene phylogeny for some Rhabditidae related to
                        <span class="emphasis"><em>Caenorhabditis</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>12</em></span>, 346&#8211;358.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7700158&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fitch, D.H.A. (1997). Evolution of male tail development in rhabditid nematodes related to <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Syst. Biol. <span class="emphasis"><em>46</em></span>, 145&#8211;179.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11975351&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fitch, D.H.A. (2000). Evolution of "Rhabditidae" and the male tail. J. Nematol. <span class="emphasis"><em>32</em></span>, 235&#8211;244.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fitch, D.H.A., and Thomas, W.K. (1997). Evolution. In <span class="emphasis"><em>C. elegans</em></span> II, D. L. Riddle, T. Blumenthal, B.J. Meyer and J.R. Priess, eds. (Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory
                        Press), pp. 815&#8211;850.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                             <span class="bibliomisc">F&uuml;rst von Lieven, A. (2002). The sister group of the Diplogastrina (Nematoda). Russ. J. Nematol. <span class="emphasis"><em>10</em></span>, 127&#8211;137.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                             <span class="bibliomisc">Jones, C.C., Rosen, T., and Greenberg, C. (1991). Cutaneous larva migrans due to <span class="emphasis"><em>Pelodera strongyloides</em></span>. Cutis <span class="emphasis"><em>48</em></span>, 123&#8211;126.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1935236&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kiontke, K., Gavin, N.P., Raynes, Y., Roehrig, C., Piano, F., and Fitch, D.H.A. (2004). <span class="emphasis"><em>Caenorhabditis</em></span> phylogeny predicts convergence of hermaphroditism and extensive intron loss. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 9003&#8211;9008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15184656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0403094101" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                             <span class="bibliomisc">Nayak, S., Goree, J., and Schedl, T. (2005). <span class="emphasis"><em>fog-2</em></span> and the evolution of self-fertile hermaphroditism in <span class="emphasis"><em>Caenorhabditis</em></span>. PLoS Biol. 3, e6.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15630478&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0030006" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                             <span class="bibliomisc">Nguyen, C.Q., Hall, D.H., Yang, Y., and Fitch, D.H.A. (1999). Morphogenesis of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> male tail tip. Dev. Biol. <span class="emphasis"><em>207</em></span>, 86&#8211;106.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10049567&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/dbio.1998.9173" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                             <span class="bibliomisc">Rogozin, I.B., Wolf, Y.I., Sorokin, A.V., Mirkin, B.G., and Koonin, E.V. (2003). Remarkable interkingdom conservation of intron
                        positions and massive, lineage-specific intron loss and gain in eukaryotic evolution. Curr. Biol. <span class="emphasis"><em>13</em></span>, 1512&#8211;1517.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12956953&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(03)00558-X" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sommer, R.J. (2000). Comparative genetics: a third model nematode species. Curr. Biol. <span class="emphasis"><em>10</em></span>, R879&#8211;R881.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20566942&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(00)00815-0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                             <span class="bibliomisc">Stein, L.D., Bao, Z., Blasiar, D., Blumenthal, T., Brent, M.R., Chen, N., Chinwalla, A., Clarke, L., Clee, C., Coghlan, A.,
                        <span class="emphasis"><em>et al</em></span>. (2003). The genome sequence of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>: a platform for comparative genomics. PLoS Biol. 1, e45.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14624247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sudhaus, W., and Fitch, D. (2001). Comparative studies on the phylogeny and systematics of the Rhabditidae (Nematoda). J.
                        Nematol. <span class="emphasis"><em>33</em></span>, 1&#8211;70.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sudhaus, W., and Hooper, D.J. (1994). <span class="emphasis"><em>Rhabditis (Oscheius) guentheri</em></span> sp. n., an unusual species with reduced posterior ovary, with observations on the <span class="emphasis"><em>Dolichura</em></span> and <span class="emphasis"><em>Insectivora</em></span> groups (Nematoda: Rhabditidae). Nematologica <span class="emphasis"><em>40</em></span>, 508&#8211;533.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sudhaus, W., and Kiontke, K. (1996). Phylogeny of <span class="emphasis"><em>Rhabditis</em></span> subgenus <span class="emphasis"><em>Caenorhabditis</em></span> (Rhabditidae, Nematoda). J. Zoo. Syst. Evol. Research <span class="emphasis"><em>34</em></span>, 217&#8211;233.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sudhaus, W., and Schulte, F. (1988). <span class="emphasis"><em>Rhabditis (Pelodera) strongyloides</em></span> (Nematoda) als Verursacher von Dermatitis, mit systematischen und biologischen Bemerkungen &uuml;ber verwandte Arten. Zool. Jahrb.
                        (Syst.) <span class="emphasis"><em>115</em></span>, 187&#8211;205.</span>
                         
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e11" href="#d0e11">*</a></sup>Edited by Lisa R. Girard. Last revised July 27, 2005. Published August 11, 2005. This chapter should be cited as: Kiontke,
                  K. and Fitch, D.H.A. The Phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> and other rhabditids (August 11, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.11.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2005 Karin Kiontke and David H. A. Fitch. This is an open-access article distributed under the terms of the Creative Commons
                  Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author
                  and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:kk52@nyu.edu" target="_top">kk52@nyu.edu</a> or <a href="mailto:david.fitch@nyu.edu" target="_top">david.fitch@nyu.edu</a></p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e157" href="#d0e157">&#8224;</a></sup>Terms used: taxon (plural taxa)- a group of phylogenetically related species; monophyletic- a group of species that includes
                  all descendants of one ancestor; paraphyletic- a group of species that includes some but not all descendants of an ancestor;
                  polyphyletic- a group of species that includes descendants of more than one ancestor; clade- a monophyletic taxon.
               </p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

