<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic dissection of developmental pathways
         			
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#epitasis_ArticleID" title="Genetic dissection of developmental pathways&#xA;&#xA;">
      <link rel="next" href="#d0e61" title="1.&nbsp;Introduction">
      <script language="javascript" type="text/javascript" src="/js/history_list.js"></script>
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="epistasis.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="epitasis_ArticleID"></a>Genetic dissection of developmental pathways<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     			<sup><a name="d0e21" href="#ftn.d0e21">&sect;</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Linda S.</span> <span class="surname">Huang<em><span class="remark">
                                 					<sup><a name="d0e41" href="#ftn.d0e41">&#8224;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Department of Biology, University of Massachusetts-Boston, Boston, MA 02125 USA
                           <p></p></span></span></div>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Paul W.</span> <span class="surname">Sternberg</span><br></span><span class="affiliation"><span class="address">Howard Hughes Medical Institute and Division of Biology, California Institute of Technology, Pasadena, CA 91125 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward genetics and genetic mapping
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=300.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Genetic+dissection+of+developmental+pathways" HEIGHT=50 WIDTH=180 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

          <p><a style="color:blue;cursor:pointer" onclick="return kadabra('history');" title="click to expand ..." >Publication History</a></p>
           <ul id="history" style="display:none">
            <li><a href="/chapters/www_epistasis/epistasis.html">version 1</a></li>
            <li><a href="/chapters/www_epistasis.2/epistasis.html">latest version</a></li>
           </ul>

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e61">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#d0e74">2. Epistasis analysis</a></span></dt>
               <dt><span class="sect1"><a href="#d0e279">3. Epistasis analysis of switch regulation pathways</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e282">3.1. Double mutant construction</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e383">3.2. Interpretation of epistasis</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e633">3.3. The importance of using null alleles</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e770">3.4. Use of dominant mutations</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e868">3.5. Complex pathways</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1223">3.6. Genetic redundancy</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1296">3.7. Limits of epistasis</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1566">3.8. Extension to other perturbations</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e1633">4. Extragenic suppressors</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e1642">4.1. Screens for extragenic suppressors</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1742">4.2. Analysis and interpretation of suppressors</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1856">4.3. Silent suppressors</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e1934">5. Prospects: use of new technologies for pathway analysis</a></span></dt>
               <dt><span class="sect1"><a href="#d0e2011">6. Conclusion</a></span></dt>
               <dt><span class="sect1"><a href="#d0e2016">7. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#d0e2021">8. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e61"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>A major tool of developmental geneticists is the ordering of genes in functional pathways. In this chapter, we explain the
               logic behind constructing pathways, starting from the knowledge of the relevant phenotypes associated with the genes of interest,
               assuming that careful analysis of the phenotype has been carried out. We discuss the construction and interpretation of phenotypes
               of double mutants, screening for and analysis of extragenic suppressors, as well as issues regarding complex pathways and
               genetic redundancy. <a href="#bib5">Avery and Wasserman (1992)</a> have provided a brief theoretical discussion of epistasis analysis; here we explain the more practical aspects of how models
               of developmental pathways are built in <span class="emphasis"><em>C. elegans</em></span>.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e74"></a>2.&nbsp;Epistasis analysis
                     </h2>
                  </div>
               </div>
            </div>
            <p>Epistasis is the masking of the phenotype of one mutant by the phenotype of a mutant in another locus. Hence, epistasis analysis
               can be used to determine a functional order of action of two genes, regardless of the directness of the interaction. However,
               epistasis analysis is most informative when the genes analyzed control a common process. Thus, it is important to determine
               the relationships between the mutations of interest before embarking on the construction of a formal genetic pathway.
            </p>
            <p>Two different kinds of pathways exist (<a href="#figure1">Figure 1</a>). Historically, these have been distinguished as regulatory pathways and assembly or metabolic pathways. As both types of
               these pathways require "regulation," for the purpose of this chapter we will refer to the classical regulatory pathway as
               a <span class="bold"><strong>switch regulation pathway</strong></span> and the classical assembly pathway as a <span class="bold"><strong>substrate dependent pathway</strong></span>. The switch regulation pathway involves genes or gene products that can be turned "on" or "off". The different states of
               the genes (or their products) involved this type of pathway determine the outcome of the pathway. Mutations in genes involved
               in this type of pathway will have two distinct and opposite phenotypes, and will also have the ability to bypass the requirement
               for upstream genes. The second type of pathway involves a substrate, where an obligate series of sequential steps are required
               to generate the final outcome; classical examples of this second type of pathway include metabolic pathways and bacteriophage
               morphogenesis. Mutations in genes involved in the substrate dependent pathway will have phenotypes that suggest a progression
               of events. Sometimes determining which of the two types of pathways the genes of interest are involved in requires extensive
               phenotypic analysis of the mutations involved.
            </p>
            <p>Consider for example, vulval differentiation in <span class="emphasis"><em>C. elegans</em></span> (see <a href="http://www.wormbook.org/chapters/www_vulvaldev/vulvaldev.html" target="_blank">Vulval development</a>). In wild-type animals, three of six equipotential vulval precursor cells differentiate to form the vulva. These six precursor
               cells are a subset of the 12 "Pn.p" cells (the posterior daughters of the twelve ectoblasts, <a href="http://www.wormbase.org/db/get?name=P1;class=Cell" target="_blank">P1</a> through <a href="http://www.wormbase.org/db/get?name=P12;class=Cell" target="_blank">P12</a>). Although loss-of-function mutants in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-26" target="_blank">lin-26</a></em></span> (<span class="emphasis"><em>lin</em></span>= <span class="underline"><u>lin</u></span>eage abnormal), <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-39" target="_blank">lin-39</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> (<span class="emphasis"><em>let</em></span> = <span class="underline"><u>let</u></span>hal) are all vulvaless and have the same phenotype when examined under a dissecting microscope, these genes are involved in
               very different aspects of vulval differentiation. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-26" target="_blank">lin-26</a></em></span> is involved in the formation of the Pn.p cells, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-39" target="_blank">lin-39</a></em></span> is involved in determining which of the Pn.p cells become potential vulval precursor cells, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> is involved in the fate decision regarding which Pn.p cells will actually differentiate into vulval tissue. Phenotypic analysis
               under higher magnification, Nomarski optics demonstrates that the vulvaless phenotype of animals mutant in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-26" target="_blank">lin-26</a></em></span> is due to the absence of Pn.p cells; in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-39" target="_blank">lin-39</a></em></span> mutants, the vulval precursor cells are not generated; and in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> mutants, the vulval precursor cells are not differentiating. In this example, the Pn.p cells are acting as the substrate
               while the genes <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-26" target="_blank">lin-26</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-39" target="_blank">lin-39</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> act to change the state of this cell. A mutation in each of these genes represents a step in the pathway that provides the
               appropriate context or substrate for the next step. Here, arrows are used to show a progression of events from one step to
               the next (<a href="#figure1">Figure 1</a>A). Epistasis analysis of substrate dependent pathways will confirm the order of events seen by phenotypic analysis.
            </p>
            <p>On the other hand, mutations involved in a switch regulation pathway should all have the same phenotype or the direct opposite
               phenotype (<a href="#figure1">Figure 1</a>B). In effect, the mutations will represent two states of an event, either ON or OFF. An example of this type of pathway in
               <span class="emphasis"><em>C. elegans</em></span> involves the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> (<span class="emphasis"><em>tra</em></span>= <span class="underline"><u>tra</u></span>nsforming) and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> (<span class="emphasis"><em>her</em></span>= <span class="underline"><u>her</u></span>maphroditization) genes, which are needed for proper sex determination (see <a href="http://www.wormbook.org/chapters/www_somaticsexdeterm/somaticsexdeterm.html" target="_top">Somatic sex determination</a>); these genes are part of a negative regulatory pathway and thus bars are used instead of arrowheads to symbolize the relationship
               between them. In <span class="emphasis"><em>C. elegans</em></span>, the sex of the animals is determined by the number of X chromosomes in an animal; an XX animal is hermaphroditic while an
               XO animal is male. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> loss-of-function mutation causes feminization of XO animals without affecting XX animals. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> loss-of-function mutation causes the opposite phenotype, masculinization of XX animals without affecting XO animals. These
               genes are involved in a switch regulation pathway because XO animals can be either one of two states, male or hermaphrodite;
               the same is true for XX animals.
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="225">
<tr>
  <td align="center"><p><a href="epistasisfig1.jpg"><img src="epistasisfig1_s.jpg" border="2" align="middle" alt="Epitasis General types of pathways figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1. General types of pathways.&nbsp;</b>(A) Substrate Dependent pathway: vulval development. A series of genes (1, 2 and 3) are necessary to produce a series of outcomes
                     (X, Y and Z). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-26" target="_blank">lin-26</a></em></span> is necessary for the development of Pn.p cells. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-39" target="_blank">lin-39</a></em></span> is necessary to select a subset of Pn.p cells to become vulval precursor cells (VPCs). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> is necessary for the VPCs to generate vulval cells.(B) Switch Regulation pathway:  somatic sex determination. A formal regulatory
                     pathway is shown with conditions regulating which of two outcomes (1 or 2) will occur. If the condition is met (e.g., presence
                     of a signal, particular environmental conditions; chromosome constitution), outcome 1 occurs. A specific example is shown
                     at the bottom. High ratio of X chromosomes to autosome sets results in the activation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> via a series of other genes. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> activity inactivates <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> via a series of other genes. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> activity results in female somatic differention; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> inactivity leads to male somatic differentiation. <span class="emphasis"><em>lf</em></span>, loss-of-function mutation; arrows represent positive regulation; bars represent negative regulation.
                  </p>
               </div>
            </div>
            <p>Whether a gene should be placed in the definition of a switch regulation pathway or a substrate dependent pathway depends
               on the phenotype studied or the event assayed. In the study of the pathway containing the receptor tyrosine kinase encoded
               by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span>, its effects on the decision of the vulval precursor cells to take on vulval versus non-vulval fates constitutes a switch
               regulatory pathway. However, synthesis of the <a href="http://www.wormbase.org/db/seq/protein?name=LET-23;class=protein" target="_blank">LET-23</a> receptor would constitute a substrate dependent pathway; one event would lead to another and the presence of <a href="http://www.wormbase.org/db/seq/protein?name=LET-23;class=protein" target="_blank">LET-23</a> is required. Biochemical experiments do not necessarily define a substrate dependent pathway. Studies on how ligand activation
               of <a href="http://www.wormbase.org/db/seq/protein?name=LET-23;class=protein" target="_blank">LET-23</a> leads to a phosphorylation cascade of downstream targets would be a switch regulation pathway; two distinct states are being
               analyzed, for example, phosphorylated and non-phosphorylated. Further, an obligate substrate is not needed in this pathway,
               as activation of a downstream target can bypass the need for <a href="http://www.wormbase.org/db/seq/protein?name=LET-23;class=protein" target="_blank">LET-23</a> (e.g., a gain-of-function mutation in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a> ras</em></span> gene will activate the pathway even in the absence of <a href="http://www.wormbase.org/db/seq/protein?name=LET-23;class=protein" target="_blank">LET-23</a>). It is imperative to distinguish whether the mutations of interest affect a switch regulation versus a substrate dependent
               pathway, as the logic used to order the genes involved is different and thus the resultant pathway constructed can be very
               different depends on a correct assessment. The ordering of genes in a substrate dependent pathway can be determined through
               an examination of the state of the substrate (i.e., the state of the Pn.p cell given above); epistasis analysis serves to
               confirm what is inferred from the phenotypic analysis or to imply the sequence of events. In the next section of this chapter,
               we will consider the logic behind constructing a switch regulation pathway, where pure phenotypic analysis will not allow
               gene ordering.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e279"></a>3.&nbsp;Epistasis analysis of switch regulation pathways
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e282"></a>3.1.&nbsp;Double mutant construction
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Once it is determined that the mutations of interest might constitute a switch regulation pathway, double mutants can be constructed
                  to determine the epistatic relationships between the two mutations. The <span class="emphasis"><em>epistatic</em></span> mutation is the one whose phenotype is displayed in the doubly mutant animal; the mutation whose phenotype is not displayed
                  is <span class="emphasis"><em>hypostatic</em></span> to the other. Thus, an epistasis test can only be performed on two mutations in different loci and with opposite phenotypes;
                  epistatic relationships cannot be determined using mutations with the same phenotypes. However, genes with mutations of the
                  same phenotype can be ordered by epistasis analysis using a mutation in a third gene that acts in the pathway between the
                  two and that causes the opposite phenotype.
               </p>
               <p>We outline three basic strategies for constructing double mutants in <a href="#figure2">Figure 2</a>. The first method is the simplest (<a href="#figure2">Figure 2</a>A) as it involves taking animals carrying the single mutations and using them directly to make the double mutants. Epistasis
                  is determined by the self-progeny of the F2 animals. If animals of phenotype A produce progeny of phenotype A and B while
                  animals of phenotype B only produce progeny of phenotype B, gene B is epistatic to gene A. Gene A would be epistatic to gene
                  B if the opposite were true. Furthermore, the animals of phenotype B from parents of phenotype A will be the desired double
                  mutant; these animals can be cultured to maintain the doubly mutant strain. Although this method is the simplest, it cannot
                  be used if gene A and gene B are tightly linked on the same chromosome.
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="387">
<tr>
  <td align="center"><p><a href="epistasisfig2.jpg"><img src="epistasisfig2_s.jpg" border="2" align="middle" alt="Figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Construction of double mutant strains.&nbsp;</b>(A) No markers. Animals carrying mutations in one gene of interest (gene A) with phenotype A are mated with animals carrying
                        mutations in the other gene of interest (gene B) with phenotype B. The F1 heterozygote (gene a/+; gene b/+) will produce progeny
                        of both phenotype A and phenotype B. Next, F2 animals of phenotype A and phenotype B are individually cultured and their F3
                        progeny examined to determine whether animals of phenotype A produce progeny of phenotype B or <span class="emphasis"><em>vice versa</em></span>. By analyzing both classes, one can ensure that the desired strain will be constructed, even if one's assumption about the
                        double mutant phenotype is incorrect. "?" indicates the allele at a given locus in a particular animal is not inferable from
                        its phenotype or the genotype of its parents. (B) <span class="bold"><strong>Linked markers in <span class="emphasis"><em>cis</em></span></strong></span>. The second method used for epistasis analysis involves using linked markers in <span class="emphasis"><em>cis</em></span> to (on the same chromosome as) the genes of interest. The markers linked to the two genes should be different, closely linked
                        to the gene of interest to avoid recombinants, not have any epistatic relationships to each other, not have any interactions
                        with the genes of interest, and be easily scored as a double mutant. Males carrying mutation <span class="emphasis"><em>b</em></span> with linked marker <span class="emphasis"><em>n</em></span> are mated to hermaphrodites homozygous for mutation <span class="emphasis"><em>a</em></span> and linked recessive marker <span class="emphasis"><em>m</em></span>. Cross-progeny are non-M and heterozygous for <span class="emphasis"><em>a</em></span> and b. Cross-progeny that do not produce N progeny are discarded, as they did not receive the <span class="emphasis"><em>b n</em></span> chromosome from their father. From this heterozygote, hermaphrodites with phenotype M or N  are picked, and the double mutant
                        M N (also homozygous for <span class="emphasis"><em>a</em></span> and <span class="emphasis"><em>b</em></span>) is easily recognized. (C) <span class="bold"><strong>Linked markers in <span class="emphasis"><em>trans</em></span></strong></span>. The best method for determining epistatic relationships involves using markers (m, n) linked in <span class="emphasis"><em>trans</em></span> to (on the homologous chromosome as) the genes of interest. This will eliminate non-specific marker effects while ensuring
                        that the double mutation of the two genes of interest is actually being constructed. Once again, the markers should be fairly
                        close to the gene of interest to reduce the possibility of recombination. This construction is similar to that in part A,
                        except that doubly mutant animals can be ascertained by the method described above and also by their failure to produce progeny
                        carrying the markers linked in <span class="emphasis"><em>trans</em></span>.
                     </p>
                  </div>
               </div>
               <p>The second method used for construction involves using a closely linked marker in <span class="emphasis"><em>cis</em></span> to (on the same chromosome as) each gene of interest (<a href="#figure2">Figure 2</a>B). The construction then involves making the strain carrying both marked chromosomes. The advantage of this method is that
                  the strain can be constructed by simply following the mutations used as markers without regard to the phenotypes in interest.
                  Once the doubly marked animal is obtained, the phenotype of interest can be determined. If the doubly marked animal has phenotype
                  A, gene A is epistatic to gene B and <span class="emphasis"><em>vice versa</em></span>. However, the control experiment must be done to construct double mutants with just the two markers, to make sure there is
                  no effect on the phenotype of interest. Sometimes this method cannot be used because the markers will occlude scoring the
                  phenotypes of interest.
               </p>
               <p>The best method for determining epistatic relationships involves using linked markers in <span class="emphasis"><em>trans</em></span> to (on the homologous chromosome as) the mutations of interest (<a href="#figure2">Figure 2</a>C). This will eliminate non-specific marker effects, while ensuring that the double mutation of the two genes of interest
                  is actually being constructed. The markers should be close to the gene of interest to reduce the possibility of recombination.
                  These markers can either be mutations in an unrelated gene or, better yet, balancer chromosomes (see <a href="#bib10">Edgley et al. (1995)</a> for an explanation of how to use balancers). This construction is similar to the first construction described, except that
                  doubly mutant animals can be ascertained by the methods described above and also by their failure to segregate the <span class="emphasis"><em>trans</em></span> markers.
               </p>
               <p>A combination of methods can also be used. No matter which method is used to construct the double mutant, it is important
                  to demonstrate that the presumptive doubly mutant animal actually carries both mutations of interest. One way that this can
                  be done is by allowing males to mate with the doubly mutant hermaphrodites and picking F1 heterozygous animals. Animals of
                  both phenotype A and phenotype B should be produced by the heterozygous animals, demonstrating that both mutations were present
                  in the putative doubly mutant animal. Also, F1 males (which are presumably cross progeny and thus heterozygous) can be picked
                  and outcrossed to test whether each male actually carries the A and B mutation.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e383"></a>3.2.&nbsp;Interpretation of epistasis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In a switch regulation pathway, the epistatic gene is the downstream gene because these genes are negatively regulating each
                  other. This rule can be derived from analysis of the single and double mutant phenotypes, as illustrated in the epistasis
                  of two genes involved in sex determination, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span>. As mentioned above, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> mutation transforms XX animals into males while the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> mutation transforms XO animals into hermaphrodites. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a>; <a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> double mutant has the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> phenotype; XX animals are males. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> is epistatic to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> and thus downstream of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span>. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> cannot be upstream of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> because the wild-type activity of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> is to activate the hermaphrodite program of development while the wild-type activity of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> is to activate the male program of development. If <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> were upstream of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span>, inactivation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> would eliminate the repression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span>, activating the male program of development. Inactivation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> would lead to the inability to activate the male program of development, resulting in the hermaphrodite program of development.
                  Removal of both <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> would make <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> unable to repress <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> (which does not matter because <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> is inactivated by mutation) and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> unable to activate the male program of development. This would result in the double mutant exhibiting the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span> phenotype. However, the experimental evidence says this is not true, so <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> is downstream of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span>; that is, inactivation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> obviates the need for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=her-1" target="_blank">her-1</a></em></span>.
               </p>
               <p>Comprehensive double mutant analysis with genes involved in a common process allows the construction of a pathway (see <a href="#figure4">Figure 4</a>). For example, during vulval induction, three of six vulval precursor cells (VPCs) normally adopt vulval fates while the
                  other three adopt non-vulval hypodermal fates. By removing the inductive signal through ablation of the signaling cell, the
                  three VPCs that normally become vulva now adopt non-vulval epidermal fates. Mutations that mimic the signaling-cell ablation
                  phenotype (all six VPCs adopt non-vulval hypodermal fates, a.k.a. vulvaless (Vul) mutations) include <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a>,</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-45" target="_blank">lin-45</a></em></span> (see <a href="http://www.wormbook.org/chapters/www_RTKRasMAPKsignaling/RTKRasMAPKsignaling.html" target="_top">RTK/Ras/MAPK Signaling</a> for details of the molecular nature). Mutations also exist that have the opposite phenotype (all six VPCs adopt vulval fates,
                  a.k.a. multivulva (Muv) mutations); these include <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-1" target="_blank">lin-1</a></em></span>. Double mutation construction using Muv and Vul mutations yield the following results:
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-1" target="_blank">lin-1</a></em></span> is epistatic to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span></td>
                     <td><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> is epistatic to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span></td>
                  </tr>
                  <tr>
                     <td><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-1" target="_blank">lin-1</a></em></span> is epistatic to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span></td>
                     <td><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> is epistatic to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span></td>
                  </tr>
                  <tr>
                     <td><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-1" target="_blank">lin-1</a></em></span> is epistatic to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span></td>
                     <td><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span> is epistatic to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span></td>
                  </tr>
                  <tr>
                     <td><span class="emphasis"><em>lin-</em></span>1 is epistatic to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-45" target="_blank">lin-45</a></em></span></td>
                     <td><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-45" target="_blank">lin-45</a></em></span> is epistatic to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span></td>
                  </tr>
               </table>
               <p>Since two states are being assayed, the ability of the VPCs to adopt vulval fates versus their ability to adopt non-vulval
                  fates, the following pathway can be constructed, using the rule of the epistatic gene being the downstream gene (<a href="#figure3">Figure 3</a>):
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="epistasisfig3.jpg"><img src="epistasisfig3_s.jpg" border="2" align="middle" alt="Epitasis figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 3.&nbsp;</b></p>
                  </div>
               </div>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="epistasisfig4.jpg"><img src="epistasisfig4_s.jpg" border="2" align="middle" alt="Epitasis Epistasis analysis figure 4"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 4. Epistasis analysis.&nbsp;</b>The phenotypes of a set of genotypes and the inferences are shown. The inferred pathway leading to the outcome 1 (black oval;
                        *) is shown at the top. The default outcome is 2. Phenotypes are described in two ways in this Figure:  as transformations
                        from 1 to 2 or 2 to 1, or as missing and extra structures, states or parameters. The inference from the phenotype of each
                        genotype is shown. For example, in (ii), the absence of A leads to no 1; thus A promotes *. However, in (iii), the absence
                        of B leads to extra 1; thus B inhibits *.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e633"></a>3.3.&nbsp;The importance of using null alleles
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The examples of epistasis given above result from double mutant analyses using severe loss-of-function recessive alleles.
                  Epistasis analysis is based upon the assumption that the mutations in the genes involved remove the function of that gene
                  in the cell, tissue, or process being analyzed. Thus, it is important to determine by dosage analysis whether the alleles
                  involved are hypermorphs (increased-function), hypomorphs (reduction-of-function), nulls (complete loss-of-function [also
                  known as amorphs]) or neomorphs (novel function) [see <a href="#bib28">Muller (1932</a>) for details; see <a href="#bib34">Sternberg (1990</a>) and <a href="#bib15">Greenwald <span class="symbol">&amp;</span> Horvitz (1980)</a> for a discussion regarding the determination of a gene's null phenotype].
               </p>
               <p>Null mutations are important for epistasis because the logic behind epistasis analysis is only valid if the two mutations
                  used can be presumed to have no assayable activity. For example, consider two genes, A and B, in a regulatory pathway where
                  there exists a null mutation in gene A but there exists no null mutations in gene B, only a hypomorphic mutation. If A is
                  epistatic to B, this would not be as critical, since the normal function of B is to negatively regulate A. In this case, A
                  is downstream of B. So, even if B has residual activity, the double mutant animals do not have functional A activity so the
                  residual B activity cannot regulate A activity.
               </p>
               <p>On the other hand, if B is downstream of A, the double mutant animal may display a misleading phenotype because the B mutation
                  used contains residual activity and the doubly mutant animal has no A activity to regulate this residual activity. Since B
                  and A have opposite phenotypes, having some wild-type B activity will lead to some expression of the A mutant phenotype, depending
                  on how much activity the hypomorphic B mutation retains. Therefore, the outcome of an epistasis experiment where one or both
                  of the mutations used are not null alleles could be co-expression, where both A and B phenotypes are expressed, or, a misleading
                  double mutant phenotype, where the animal displays the A phenotype.
               </p>
               <p>A practical example of this phenomenon can be found in <a href="#bib14">Ferguson et al. (1987)</a>. Epistasis analysis using the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> allele, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n309;class=Variation" target="_blank">n309</a></em></span>, and the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> allele, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n1045;class=Variation" target="_blank">n1045</a></em></span>, resulted in the <span class="emphasis"><em>let-23(<a href="http://www.wormbase.org/db/get?name=n1045;class=Variation" target="_blank">n1045</a>)</em></span>; <span class="emphasis"><em>lin-15(<a href="http://www.wormbase.org/db/get?name=n309;class=Variation" target="_blank">n309</a>)</em></span> doubly mutant animals expressing either the Muv or the Vul phenotype. It is now known that <span class="emphasis"><em>let-23(<a href="http://www.wormbase.org/db/get?name=n1045;class=Variation" target="_blank">n1045</a>)</em></span> makes some wild-type product (Aroian et al., 1994) and that <span class="emphasis"><em>lin-15(<a href="http://www.wormbase.org/db/get?name=n309;class=Variation" target="_blank">n309</a>)</em></span> is not a complete deletion of the locus (<a href="#bib25">Huang et al., 1994</a>; <a href="#bib8">Clark et al., 1994</a>). It is not possible to use a complete null allele of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> to analyze its phenotype in late larval development (without mosaic analysis) because, as the name implies, a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> null is lethal and would not survive to make the vulva. However, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sy97;class=Variation" target="_blank">sy97</a></em></span> allele of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> is a severe loss of function allele with respect to the vulva, such that vulval differentiation is essentially not seen (see
                  <a href="#bib4">Aroian and Sternberg, 1991</a> for an example of tissue-specific activities of a genetic locus). Molecular analysis also identified a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> allele, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1763;class=Variation" target="_blank">e1763</a></em></span>, whose lesion removes more coding sequence than <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n309;class=Variation" target="_blank">n309</a></em></span>. When epistasis analysis is performed using <span class="emphasis"><em>let-23(<a href="http://www.wormbase.org/db/get?name=sy97;class=Variation" target="_blank">sy97</a></em></span>) and <span class="emphasis"><em>lin-15(<a href="http://www.wormbase.org/db/get?name=e1763;class=Variation" target="_blank">e1763</a></em></span>), all the doubly mutant animals display a Vul phenotype (<a href="#bib25">Huang et al., 1994</a>). Of course, the <span class="emphasis"><em>let-23(<a href="http://www.wormbase.org/db/get?name=sy97;class=Variation" target="_blank">sy97</a></em></span>) allele did not exist in 1987 and the molecular nature of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> alleles was not known, so the epistasis was performed using the best of what existed and was known at the time.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e770"></a>3.4.&nbsp;Use of dominant mutations
                        </h3>
                     </div>
                  </div>
               </div>
               <p>As mentioned above, epistasis analysis cannot be performed using mutations that display the same phenotype. This can lead
                  to subsets of the pathway where the order of some genes is not known. Sometimes these genes can be ordered through the use
                  of gain-of-function alleles, which will display the opposite phenotype of the loss-of-function mutations. In these cases,
                  it is important to know that the gain-of-function mutation in the gene of interest increases the normal activity of the gene
                  and does not confer a novel activity.
               </p>
               <p>In the example of the pathway regulating vulval differentiation given above, the genes <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a>, <a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-45" target="_blank">lin-45</a></em></span> cannot be separated by epistasis using hypomorphic alleles, as all display the Vul phenotype. However, there exists a gain-of-function
                  allele of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span>; animals carrying this allele display the Muv phenotype. Double mutant construction using the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span> gain-of-function allele and the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-45" target="_blank">lin-45</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> hypomorphic alleles gives the following results:
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td><span class="bold"><strong>Genotype</strong></span></td>
                     <td><span class="bold"><strong>Phenotype</strong></span></td>
                  </tr>
                  <tr>
                     <td><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-45" target="_blank">lin-45</a>; <a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a>(gf)</em></span></td>
                     <td>Vul</td>
                  </tr>
                  <tr>
                     <td><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a>; <a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a>(gf)</em></span></td>
                     <td>Muv</td>
                  </tr>
               </table>
               <p>Therefore, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span> gain-of-function allele can be used to split the order of action of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-45" target="_blank">lin-45</a></em></span>. These results imply the order (<a href="#figure5">Figure 5</a>):
               </p>
               <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="180">
<tr>
  <td align="center"><p><a href="epistasisfig5.jpg"><img src="epistasisfig5_s.jpg" border="2" align="middle" alt="Epitasis figure 5"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 5.&nbsp;</b></p>
                  </div>
               </div>
               <p>Arrows are used because the wild-type function of the genes is to positively regulate each other. The gain-of-function phenotype
                  of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span> results in the opposite phenotype of hypomorphic <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span> alleles and allowed the ordering of these genes using the logic used for epistasis for switch regulation pathways.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e868"></a>3.5.&nbsp;Complex pathways
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Epistasis analysis does not always result in simple interpretations of gene order. Sometimes this is because the mutations
                  used are not null alleles; the consequences of this are discussed above. Sometimes this is because the mutations used are
                  not part of a switch regulation pathway but instead, part of a complex pathway that combines both switch regulation processes
                  and substrate dependent processes. In fact, complex pathways are more often the rule than the exception. Development proceeds
                  through a series of steps; often, a set of decisions about a fate leads to an execution of a particular fate, which may then
                  lead to decisions about another fate. Development can be viewed as a set of substrate dependent pathways leading to switch
                  regulation pathways which lead to more substrate dependent pathways, and so on.
               </p>
               <p>Because the logic of epistasis differs for switch regulation and substrate dependent pathways, it is important to determine
                  if the mutations involved in a complex pathway are all involved in the same step of the pathway being studied. This is not
                  always obvious and may require careful phenotypic analysis. For example, animals carrying the mutation <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-18" target="_blank">egl-18</a></em></span>(<span class="emphasis"><em>n300</em></span>) are Vul when examined under a dissecting microscope. Because animals carrying the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> mutation are Muv, superficially, Muv and Vul are opposite phenotypes, the epistasis analysis of these two genes seems simple.
                  The doubly mutant animal of genotype <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-18" target="_blank">egl-18</a>; <a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> is Vul (<a href="#bib14">Ferguson, et al., 1987</a>); the simple interpretation is that <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-18" target="_blank">egl-18</a></em></span> is downstream of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span>. However, this is not the case. Careful examination of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-18" target="_blank">egl-18</a></em></span> phenotype reveals its involvement in the generation of the VPCs; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-18" target="_blank">egl-18</a></em></span> animals are Vul because the VPCs are not there and thus cannot make a vulva. The correct interpretation is <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-18" target="_blank">egl-18</a></em></span> is upstream of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span>, operating in the pathway required to generate the VPCs and not in the regulation of VPC fate, as <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> does.
               </p>
               <p>The process of programmed cell death is an excellent example of a complex pathway (see <a href="http://www.wormbook.org/chapters/www_programcelldeath/programcelldeath.html" target="_blank">Programmed cell death</a>; <a href="#figure7">Figure 7</a>). In <span class="emphasis"><em>C. elegans</em></span>, 131 of the 1030 cells generated during development of the adult hermaphrodite undergo programmed cell death. Programmed
                  cell death is a complex process, first involving the specification of whether or not a cell will die, followed by the process
                  of killing the cell, and finishing with the engulfment and degradation of cell corpses. Mutations have been found in genes
                  affecting each of these processes. For example, the genes <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> (<span class="emphasis"><em>ced</em></span>=cell death abnormal), <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span> are involved in the killing process (<a href="#bib12">Ellis, et al., 1991b</a>; <a href="#bib19">Hengartner and Horvitz, 1992</a>). Genes involved in the engulfment of cell corpses include <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-2" target="_blank">ced-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-5" target="_blank">ced-5</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-6" target="_blank">ced-6</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-7" target="_blank">ced-7</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-8" target="_blank">ced-8</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-10" target="_blank">ced-10</a></em></span>; mutations in these genes causes cell corpses to persist. The process of cell death thus consists of the steps outlined in
                  <a href="#figure6">Figure 6</a>.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="epistasisfig6.jpg"><img src="epistasisfig6_s.jpg" border="2" align="middle" alt="Epitasis figure 6"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 6.&nbsp;</b></p>
                  </div>
               </div>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure7"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="epistasisfig7.jpg"><img src="epistasisfig7_s.jpg" border="2" align="middle" alt="Epitasis Mixed switch regulation and substrate dependent pathway figure 7"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 7. Mixed switch regulation and substrate dependent pathway:  programmed cell death.&nbsp;</b>An example of a mixed pathway. Some signal controls activities of either <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span>, resulting in activation of a pathway by which cells die. Inactivation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span> results in extra cells dying. Inactivation of either <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span> results in the survival of cells that would otherwise die. Inactivation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-2" target="_blank">ced-2</a></em></span>, etc. results in cells that die, but whose corpses are not engulfed by surrounding tissue.
                     </p>
                  </div>
               </div>
               <p>Epistasis can only be interpreted using the logic discussed for switch regulation pathways when the genes being analyzed are
                  in a common process. In the example above, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span> are involved in different processes and thus phenotypic analysis will elucidate their order relative to each other such that
                  this analysis will reveal the processes in which these genes are involved and thus reveal their order of action. In this case,
                  an understanding of the events involved in cell death will demonstrate that cells are killed (which requires the action of
                  <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span>) before they are engulfed (which requires the action of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span>).
               </p>
               <p><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span> are involved in regulating the killing of cells. In animals without <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span> function, cells survive that normally die by programmed cell death. Animals without <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span> function die. When <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span> mutant animals also lack either <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span>, they live, suggesting that functional <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span> is necessary for the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span> gene product to function. This interpretation is consistent with the gain-of-function phenotype of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span>, which results in the survival of cells that normally die by programmed cell death (this is the same phenotype as loss-of-function
                  of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span>). These results imply the gene order shown in <a href="#figure8">Figure 8</a>.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure8"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="180">
<tr>
  <td align="center"><p><a href="epistasisfig8.jpg"><img src="epistasisfig8_s.jpg" border="2" align="middle" alt="Epitasis figure 8"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 8.&nbsp;</b></p>
                  </div>
               </div>
               <p>When double mutants are made using mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span>, a gene involved in the engulfment of cell corpses, the doubly mutant animal displays the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> phenotype. However, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> are part of the same substrate dependent pathway but not part of the same switch regulation pathway. For one, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span> do not have the opposite phenotype from one another. Phenotypic analysis would suggest that the cell must first be specified
                  to die by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> before its corpse can be engulfed through the activity of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span>. Also, the engulfment pathway that <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span> regulates is not necessary for causing cell death like <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span>, because mutations that prevent engulfment do not prevent most cell deaths (<a href="#bib18">Hedgecock, et al, 1983</a>; <a href="#bib11">Ellis, et al, 1991a</a>). The action of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> leads to the action of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span>. Thus, the pathway should be drawn as (<a href="#figure9">Figure 9</a>):
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure9"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="epistasisfig9.jpg"><img src="epistasisfig9_s.jpg" border="2" align="middle" alt="Epitasis figure 9"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 9.&nbsp;</b></p>
                  </div>
               </div>
               <p>These examples regarding <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-18" target="_blank">egl-18</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> in the vulval differentiation pathway and the genes in the programmed cell death pathway illustrate the importance of detailed
                  phenotypic characterization of the singly mutant animals in preventing misinterpretation of double mutant phenotypes. Once
                  it is determined that two genes are in a switch regulation pathway, the rule that the epistatic gene resides downstream of
                  the hypostatic gene can be simply applied. In contrast, if two genes are involved in a substrate dependent pathway, double
                  mutant analysis again confirms the order of events that have been visualized, although in this case, the epistatic gene is
                  the upstream gene. Whether or not two genes are involved in a switch regulation process or a substrate dependent process can
                  only be determined by careful analysis of the phenotypes of the mutations involved.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1223"></a>3.6.&nbsp;Genetic redundancy
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Another factor that can complicate the interpretation of epistatic relationships is genetic redundancy. Discussions on the
                  theoretical implications of genetic redundancy can be found in <a href="#bib39">Thomas (1993)</a>  and <a href="#bib38">Tautz (1992)</a>; these issues are beyond the scope of this chapter. In practice, genetic redundancy can be discovered in three ways: by backcrossing
                  newly isolated mutants of interest and discovering that the mutant phenotype is a result of two mutations, by constructing
                  double mutants and discovering a new phenotype, or by undertaking a screen for redundant genes. These methods are discussed
                  below (see <a href="#figure10">Figure 10</a>).
               </p>
               <p>Discovering genetic redundancy by backcross involves examination of the frequency of segregation of the mutation involved
                  from a heterozygous animal (<a href="#figure10">Figure 10</a>A). This method is most often used during the analysis of a new mutation following mutagenesis (for example, see <a href="#bib13">Ferguson and Horvitz (1989)</a>). A recessive phenotype caused by a single gene trait will usually segregate from a heterozygote animal at a frequency of
                  1/4. However, if two genes are required to cause the phenotype, animals expressing this phenotype will segregate from a heterozygous
                  parent at a frequency of 1/16, assuming they are unlinked from each other. Once it is determined that two genes cause the
                  phenotype of interest, the phenotype of each mutation alone should be determined. If these two mutations are genetically redundant,
                  they will only cause the phenotype of interest in combination with each other and should display either a reduced phenotype
                  alone or possibly even no phenotype.
               </p>
               <p>Another method for discovering genetic redundancy involves constructing double mutant animals using existing mutations (<a href="#figure10">Figure 10</a>B). This is usually done when mutations are recovered that display an incomplete phenotype or that display no phenotype with
                  respect to the phenotype of interest (i.e., a silent mutation). This method was extremely informative in the discovery of
                  the two sets of genes involved in the engulfment process during programmed cell death (<a href="#bib11">Ellis, et al., 1991a</a>) and the redundancy seen in the dauer formation pathway (<a href="#bib40">Thomas, et al., 1993</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure10"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="epistasisfig10.jpg"><img src="epistasisfig10_s.jpg" border="2" align="middle" alt="Epitasis Synthetic phenotypes figure 10"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 10. Synthetic phenotypes.&nbsp;</b>(A) Discovery by backcrossing. If two recessive mutations are responsible for the phenotype, then 1/16 progeny of the double
                        heterozygote will display the phenotype. (B) Discovery by construction. (C) Discovery by mutagenesis.
                     </p>
                  </div>
               </div>
               <p>A third method for discovering genetic redundancy involves screening for new mutations that synergize with the original mutation
                  to cause the phenotype of interest (<a href="#figure10">Figure 10</a>C, also see <a href="#bib13">Ferguson and Horvitz, 1989</a>). This is an especially useful technique if it is already known that redundancy exists in the system, through molecular information
                  or by the discovery of genetic redundancy through other methods, as described above.
               </p>
               <p>The formal interpretation of genetic redundancy is that the redundant genes are involved in parallel pathways, regulating
                  the process of interest. For two redundant genes, A and B, that negatively regulate gene C that negatively regulates gene
                  D, the pathway would look like (<a href="#figure11">Figure 11</a>):
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure11"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="225">
<tr>
  <td align="center"><p><a href="epistasisfig11.jpg"><img src="epistasisfig11_s.jpg" border="2" align="middle" alt="Epitasis figure 11"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 11.&nbsp;</b></p>
                  </div>
               </div>
               <p>In this case, the animal carrying mutations in both gene A and gene B would display the opposite phenotype as animals mutant
                  in gene C and would display the same phenotype as animals mutant in gene D. Thus, when epistasis analysis is done with genetically
                  redundant genes, both redundant genes should be eliminated because only then is the phenotype of interest seen. In this case,
                  since gene A only partially negatively regulates gene C, even doing epistasis with a null allele of gene A will not completely
                  eliminate the function of that particular step; both gene A and gene B function need to be completely eliminated to create
                  the phenotype that represents a lack of function at that step. Thus, the triply mutant animal lacking gene A, gene B, and
                  gene C must be constructed for epistasis analysis.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1296"></a>3.7.&nbsp;Limits of epistasis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Although the ordering of genes in pathways using genetic epistasis is a powerful tool, it merely provides a working model
                  for more phenotypic studies or molecular analysis and should not be taken as the &#8220;answer.&#8221; One limitation of epistasis is
                  that it assumes a linearity of events. This is not always the case; as sometimes pathways are branched or contain multiple
                  inputs. Also, some pathways use tissue specific regulators that are not necessarily used in other developmental processes
                  that use a subset of these same genes. Ultimately, these issues can be resolved by careful phenotypic analysis of the genes
                  involved in the pathway, although where to look may not be initially obvious.
               </p>
               <p>One example of a branched pathway in <span class="emphasis"><em>C. elegans</em></span> is the sex determination pathway (see <a href="#figure12">Figure 12</a>). A high X chromosome to autosome ratio (X:A) negatively regulates the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>, <a href="http://www.wormbase.org/db/gene/gene?name=fem-2" target="_blank">fem-2</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a></em></span> genes, which negatively regulate the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> gene. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> functions to activate the developmental program for the female soma while repressing the developmental program for the male
                  soma. However, the role of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>, <a href="http://www.wormbase.org/db/gene/gene?name=fem-2" target="_blank">fem-2</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a></em></span> gene products is not just to regulate <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span>; these genes are also responsible for promoting male germline development and negatively regulating female germline development.
                  In the case of a high X:A ratio, inactive <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>, <a href="http://www.wormbase.org/db/gene/gene?name=fem-2" target="_blank">fem-2</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a></em></span> would lead to female germline development. By recognizing the branch point at <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>, <a href="http://www.wormbase.org/db/gene/gene?name=fem-2" target="_blank">fem-2</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a></em></span> and realizing that germline development is regulated differently than somatic development, the intersex phenotype of animals
                  carrying the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> mutation can be interpreted. XX <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> mutant animals have a male soma because they have no functional <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> to repress male somatic development; these XX animals also have a female germline because <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a>, <a href="http://www.wormbase.org/db/gene/gene?name=fem-2" target="_blank">fem-2</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a></em></span> are repressed by the high X:A ratio.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure12"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="epistasisfig12.jpg"><img src="epistasisfig12_s.jpg" border="2" align="middle" alt="Epitasis Branched pathway: somatic and germline sex determination. figure 12"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 12. Branched pathway: somatic and germline sex determination.&nbsp;</b>The ratio of X to autosomes determines sexual phenotype (see Villeneuve and Meyer (1990) for review). High ratio of X chromosomes
                        to autosomes results in repression of activity of the <span class="emphasis"><em>fem</em></span> genes. For somatic sexual determination, the <span class="emphasis"><em>fem</em></span> genes negatively regulate <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> activity. For germline sex determination, the <span class="emphasis"><em>fem</em></span> genes control sexual identity independently of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span>.
                     </p>
                  </div>
               </div>
               <p>This example also illustrates the existence of tissue specific regulators of a particular pathway. Although <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> is downstream of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a></em></span> in somatic development, it is not so in germline development. Thus, the order of gene action should be determined by epistasis
                  analysis with the genes involved for the particular developmental process of interest, looking at their action in the tissue
                  of interest. Another example of this is the use of the heterochronic genes <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-4" target="_blank">lin-4</a>, <a href="http://www.wormbase.org/db/gene/gene?name=lin-14" target="_blank">lin-14</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-28" target="_blank">lin-28</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-29" target="_blank">lin-29</a></em></span> (<a href="#bib2">Ambros and Moss, 1994</a>). Animals carrying loss-of-function mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-4" target="_blank">lin-4</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-14" target="_blank">lin-14</a></em></span> have widespread defects in temporal control. By examining their effects in the temporal control of the hypodermal seam cells,
                  a regulatory pathway was constructed with the more specifically used heterochronic genes, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-28" target="_blank">lin-28</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-29" target="_blank">lin-29</a></em></span>. A complete description of the construction of this pathway is found in <a href="#bib1">Ambros (1989)</a>; this paper also contains an excellent explanation of how the logic of epistasis analysis was used to construct the pathway.
               </p>
               <p>Another variation on branched pathways are pathways with more than one input (<a href="#figure14">Figure 14</a>). An example of this is the <span class="emphasis"><em>C. elegans</em></span> vulval differentiation pathway. The pathway given in previous sections consists of (<a href="#figure13">Figure 13</a>):
               </p>
               <div class="mediaobject" align="center"><a name="figure13"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="epistasisfig13.jpg"><img src="epistasisfig13_s.jpg" border="2" align="middle" alt="Epitasis figure 13"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 13.&nbsp;</b></p>
                  </div>
               </div>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure14"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="epistasisfig14.jpg"><img src="epistasisfig14_s.jpg" border="2" align="middle" alt="Epitasis Multiple inputs: vulval induction figure 14"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 14. Multiple inputs: vulval induction.&nbsp;</b>(A) Abbreviated pathway of vulval induction (see <a href="#bib36">Sternberg (1993)</a> for review). (B) A "truth table" schema for the combined action of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> to control vulval differentiation.
                     </p>
                  </div>
               </div>
               <p>However, the molecular identification of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> gene product as a growth factor/ligand-like molecule and the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> gene product as a receptor tyrosine kinase suggests that <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> encodes the ligand for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> (see <a href="http://www.wormbook.org/chapters/www_RTKRasMAPKsignaling/RTKRasMAPKsignaling.html" target="_blank">RTKRas/MAP kinase signaling</a>). Furthermore, previous phenotypic characterization of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> demonstrated that although <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> mutant animals exhibit the mutant Muv phenotype even in the absence of the inductive tissue, their VPCs are still capable
                  of responding to the signal. Thus <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> does not belong directly downstream of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> signal, but instead, as a second input upon <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> (<a href="#figure15">Figure 15</a>):
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure15"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="epistasisfig15.jpg"><img src="epistasisfig15_s.jpg" border="2" align="middle" alt="Epitasis figure 15"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 15.&nbsp;</b></p>
                  </div>
               </div>
               <p>The examples of the vulval differentiation pathway and the sex-determination pathway given above underscore the importance
                  of careful phenotypic analysis in the construction of developmental regulatory pathways. This is not to say that epistasis
                  analysis should not be performed until an exhaustive understanding of the mutant phenotype exists; however, pathways constructed
                  through epistasis analysis should be considered models that can change over time, given new information from further phenotypic
                  or molecular analysis.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1566"></a>3.8.&nbsp;Extension to other perturbations
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The logic presented in this chapter can be extended to other non-genetic interactions, such as cell-cell interactions that
                  can be dissected using cell ablations. Rather than remove the function of a gene by mutation, a cell can be removed by laser
                  ablation (<a href="#bib6">Avery and Bargmann, 1995</a>) or by a pharmacological agent . One example of using cell ablations instead of mutation for epistasis is the analysis of
                  cell signalling during male spicule development (<a href="#bib7">Chamberlin and Sternberg, 1993</a>). B<span class="symbol">&#947;</span> and B<span class="symbol">&#948;</span> are two different cell fates adopted by certain progeny of the B cell, a male specific blast cell involved in spicule development.
                  Ablation of the nearby <a href="http://www.wormbase.org/db/get?name=Y.p;class=Cell" target="_blank">Y.p</a> cell (the posterior daughter of the male specific blast cell, Y) leads to a defect in the <span class="symbol">&#948;</span> lineage; ablation of the <span class="bold"><strong>pa</strong></span> cells (the two cells, <a href="http://www.wormbase.org/db/get?name=B.alpa;class=Cell" target="_blank">B.alpa</a> and <a href="http://www.wormbase.org/db/get?name=B.arpa;class=Cell" target="_blank">B.arpa</a>, great-great granddaughters of the blast cell, B; sisters of <span class="symbol">&#947;</span> and <span class="symbol">&#948;</span>) leads to a defect in B<span class="symbol">&#947;</span>. As B<span class="symbol">&#947;</span> and B<span class="symbol">&#948;</span> are alternative cell fates, the ablation of the <a href="http://www.wormbase.org/db/get?name=Y.p;class=Cell" target="_blank">Y.p</a> cell can be considered to cause the opposite phenotypes from that of the ablation of the <span class="bold"><strong>pa</strong></span> cells. The double ablation of both <a href="http://www.wormbase.org/db/get?name=Y.p;class=Cell" target="_blank">Y.p</a> and the <span class="bold"><strong>pa</strong></span> cells has a defect similar to ablation of the <a href="http://www.wormbase.org/db/get?name=Y.p;class=Cell" target="_blank">Y.p</a> cell alone. Using the logic of epistasis, the role of the <span class="bold"><strong>pa</strong></span> cells might be to negatively regulate the effect of Y.p.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1633"></a>4.&nbsp;Extragenic suppressors
                     </h2>
                  </div>
               </div>
            </div>
            <p>To further analyze a pathway, it is often desirable to identify additional genes. One method for doing this involves starting
               with a wild-type animal and screening for additional mutations with the phenotypes of interest. Another method involves isolating
               extragenic suppressor mutations of a previously identified mutant gene in the pathway (see <a href="#bib3">Anderson (1995)</a> for information regarding the different mutagens available). Suppression screens can be better than direct screens because
               the presence of the starting mutation might suppress the lethality of the suppressor mutation, and thus allow the mutation
               to be recovered. Screens for mutations that enhance a partial mutant phenotype can also identify new genes.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1642"></a>4.1.&nbsp;Screens for extragenic suppressors
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Screens for extragenic suppressors seem deceptively simple: a mutation in the pathway is chosen and a screen is performed
                  to look for new mutations that no longer express the mutant phenotype, either in the F1 generation (for dominant suppressors)
                  or in the F2 generation (for recessive suppressors) (see <a href="#figure16">Figure 16</a>). Although the screening process is straightforward, the key to the success of an extragenic suppressor screen involves choosing
                  the most appropriate starting mutation for the purpose of the screen, as the choice of a starting strain affects the suppressor
                  mutations isolated.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure16"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="epistasisfig16.jpg"><img src="epistasisfig16_s.jpg" border="2" align="middle" alt="Epitasis Suppressor screens figure 16"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 16. Suppressor screens.&nbsp;</b>From mutant parents that have been mutagenized, the F1 progeny and F2 grandprogeny are examined for phenotypic revertants,
                        that is animals that do not display the mutant phenotype. Each F1 examined screens two mutagenized gametes, as either the
                        sperm or the ova could carry a mutation. If the F2 are examined, the number of mutagenized gametes is also two per F1. m,
                        starting mutation; sup, suppressor mutation.
                     </p>
                  </div>
               </div>
               <p>One consideration when choosing a starting mutation is to decide what type of mutation to suppress: a null mutation, a loss-of-function
                  mutation, or a gain-of-function mutation. Suppression of a null mutation (X) will allow recovery of new mutations that allow
                  bypass of the gene of interest. These will either be gain-of-function mutations of downstream genes in the same substrate
                  dependent pathway (Y) or loss-of-function mutations of downstream negative regulators (W). However, suppression of a null
                  mutation will not allow recovery of mutations in genes that directly interact with the starting gene's product (<a href="#figure17">Figure 17</a>).
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure17"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="117">
<tr>
  <td align="center"><p><a href="epistasisfig17.jpg"><img src="epistasisfig17_s.jpg" border="2" align="middle" alt="Epitasis figure 17"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 17.&nbsp;</b></p>
                  </div>
               </div>
               <p>On the other hand, suppression of a reduction-of-function mutation (X) will allow recovery of mutations in genes that interact
                  with the gene of interest. A mutation in an interacting gene will be recovered if it removes the negative regulation of an
                  upstream interacting gene (V). Mutations that allow bypass of the gene of interest will also be recovered (W and Y), as with
                  using the null. Upstream genes that, when mutated, increase the activity of the loss-of-function mutation used will also be
                  recovered (Z; see <a href="#figure18">Figure 18</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure18"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="180">
<tr>
  <td align="center"><p><a href="epistasisfig18.jpg"><img src="epistasisfig18_s.jpg" border="2" align="middle" alt="Epitasis figure 18"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 18.&nbsp;</b></p>
                  </div>
               </div>
               <p>Suppression of a gain-of-function mutation can allow the subsequent recovery of the opposite spectra of alleles as suppression
                  of a loss-of-function mutation. For example, loss-of-function mutations of downstream genes will be recovered (Y) as well
                  as gain-of-function mutations of downstream negativly regulated genes (W). These screens can be very powerful (for example,
                  see <a href="#bib42">Wu and Han, 1994</a> and <a href="#bib26">Lackner et al., 1994</a>) regarding suppression of the gain-of-function <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a> ras</em></span> mutations; <a href="#bib31">Riddle (1977)</a> for suppression of dauer defective mutants; and <a href="#bib21">Hodgkin (1986)</a> for suppression of sex determination mutations). However, it is important to know that the starting mutation (X) increases
                  normal gene activity and does not confer a novel activity (<a href="#figure19">Figure 19</a>).
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure19"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="180">
<tr>
  <td align="center"><p><a href="epistasisfig19.jpg"><img src="epistasisfig19_s.jpg" border="2" align="middle" alt="Epitasis figure 19"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 19.&nbsp;</b></p>
                  </div>
               </div>
               <p>For example, screens using an amber suppressible allele will not only yield suppressors of interest, but can also yield mutations
                  in amber suppressor tRNAs. If the suppressors of interest occur at a very low frequency, the other more non-specific mutations
                  may overwhelm the recovery of the desired mutations. Similarly, screens using alleles which are splice site mutations may
                  yield mutations in splicing components as well as the suppressors of interest. Of course, it is not always possible to begin
                  with knowledge of the exact defect of the mutation of interest; nonetheless one can test by construction whether a starting
                  mutation is suppressed by known informational suppressors. e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-7" target="_blank">sup-7</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span> (see below for further details).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1742"></a>4.2.&nbsp;Analysis and interpretation of suppressors
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Analysis of suppressor mutations involves separating the suppressors of interest from intragenic revertants and informational
                  suppressors, testing dominance, checking for phenotypes in an otherwise wild-type background, mapping the suppressor locus,
                  and determining the spectrum of suppression. When a suppressor mutation is isolated, its only known phenotype is that it suppresses
                  the original mutation. Thus, no assumptions can be made about the phenotype of the suppressor on its own; it could have an
                  unexpected phenotype, be lethal, or be silent. Because of this, strains for the analysis of suppressor mutations must carry
                  the starting mutation to allow scoring of the suppressor phenotype. In essence, the strain with the starting mutation is the
                  background strain for further analysis. Besides this, analysis of suppressor mutations proceeds as for any other mutation.
                  For example, to establish linkage with a recessive suppressor mutation, <span class="emphasis"><em>mut/mut; sup/sup</em></span> animals can be crossed to <span class="emphasis"><em>mut/mut; marker/marker</em></span> strains to construct a <span class="emphasis"><em>mut/mut; marker/+; sup/+</em></span> heterozygote, and linkage of <span class="emphasis"><em>sup</em></span> and <span class="emphasis"><em>marker</em></span> observed among the progeny (<span class="emphasis"><em>mut</em></span>=starting mutation, <span class="emphasis"><em>sup</em></span>=suppressor mutation, <span class="emphasis"><em>marker</em></span>=any unrelated marker) (<a href="#figure20">Figure 20</a>A). Dominance can be tested by crossing <span class="emphasis"><em>mut/mut; sup/sup</em></span> males to <span class="emphasis"><em>mut/mut; marker/marker</em></span> hermaphrodites and examining the phenotype of the non-marker cross-progeny (<a href="#figure20">Figure 20</a>B). Similarly, complementation of recessive suppressor mutations can be tested by crossing <span class="emphasis"><em>sup-x/sup-x; mut/mut</em></span> males to <span class="emphasis"><em>sup-y/sup-y; mut/mut; marker/marker</em></span> hermaphrodites.
               </p>
               <div class="mediaobject" align="center"><a name="figure20"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="225">
<tr>
  <td align="center"><p><a href="epistasisfig20.jpg"><img src="epistasisfig20_s.jpg" border="2" align="middle" alt="Epitasis Analysis of suppressor mutations figure 20"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 20. Analysis of suppressor mutations.&nbsp;</b>(A) Linked versus unlinked suppressors. Assuming complete penetrance and recessivity of the <span class="emphasis"><em>sup</em></span> mutation, only <span class="emphasis"><em>mut +/mut sup</em></span> recombinants will be detected among the progeny of animals heterozygous for <span class="emphasis"><em>mut</em></span> and <span class="emphasis"><em>sup</em></span>. These will occur at a frequency of 2p/4 = p/2, where p is the map distance (see <a href="#bib37">Sulston and Hodgkin (1988)</a>). (B) Test for dominance. Animals heterozygous for the <span class="emphasis"><em>sup</em></span> mutation and homozygous for the starting mutation, a, are constructed to determine the phenotype.
                     </p>
                  </div>
               </div>
               <p>Intragenic revertants will be linked; extragenic linked suppressors can be separated by recombination. For analysis of intragenic
                  revertants see <a href="#bib15">Greenwald and Horvitz (1980</a>). Such revertants can be informative with respect to the structure, function or expression of a particular gene, but are
                  not helpful in defining a pathway.
               </p>
               <p>It is important to determine the spectrum of suppression of a suppressor mutation: Does it suppress other alleles of the starting
                  gene? If not, it might suggest informational suppression (see below) or a protein-protein interaction, depending on the nature
                  of the alleles. Does it suppress hypomorphic or null alleles? If it only suppresses hypomorphic alleles, the suppressor might
                  act upstream in a pathway or act in branch of the pathway. Does it suppress other phenotypes of the starting mutation? This
                  would tell you if it acts in all pathways in which your starting mutation functions. Does it suppress mutations in other genes
                  in the pathway, and if so, does it only suppress mutations that lie upstream? Not only can these types of data reveal whether
                  a suppressor is informational, but they also indicate the specificity of the suppressor gene defined. Whether specific or
                  generally acting suppressors are of greater interest is a matter of personal taste.
               </p>
               <p>One class of extragenic suppressors are informational suppressors (reviewed by <a href="#bib22">Hodgkin et al. (1987)</a>). Formally, informational suppressors are allele-specific, gene non-specific, and thus act on particular types of mutations,
                  rather than particular types of gene products. The classic informational suppressor is an amber suppressor, e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-7" target="_blank">sup-7</a></em></span> in <span class="emphasis"><em>C. elegans</em></span>. A new class of informational suppressors, the <span class="emphasis"><em>smg</em></span> genes (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span>), were found by reversion of three independent phenotypes; the <span class="emphasis"><em>smg</em></span> genes affect mRNA stability (<a href="#bib30">Pulak and Anderson, 1993</a>). The informational nature of their suppression was discovered by the realization that three different genetic screens for
                  unrelated pathways identified the same set of suppressor mutations (<a href="#bib23">Hodgkin et al., 1989</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1856"></a>4.3.&nbsp;Silent suppressors
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Once the map position of a suppressor mutation is determined, markers can be used in <span class="emphasis"><em>trans</em></span> to separate the suppressor from the original mutation so that the phenotype of the suppressor alone can be determined. The
                  simplest suppressor mutations to interpret are those that confer a phenotype opposite to the starting mutation, with suppression
                  by virtue of epistasis [e.g., suppression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> multivulva by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span> vulvaless (<a href="#bib17">Han, et al., 1990</a>); suppression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-3" target="_blank">tra-3</a></em></span> masculinization by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-2" target="_blank">fem-2</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a></em></span> feminization (<a href="#bib21">Hodgkin, 1986</a>); suppression of dauer constitutive by dauer defective (<a href="#bib32">Riddle, et al., 1981</a>)]. However, many suppressor mutations have an unrelated phenotype or no phenotype other than suppression of the starting
                  mutation; these are silent suppressors as they have no effect with respect to the phenotype of interest. There are various
                  types of experiments one can carry out but no generalizable formula is available to determine what will be the most informative
                  types of silent suppressor mutations. Note that silent suppressors can identify genes that would otherwise not have been identified.
               </p>
               <p>Some suppressor mutations enhance other mutations that have the opposite phenotype to the starting mutation, suggesting that
                  they indeed control activity of the pathway of interest. Some suppressor mutations may be due to a mutation in two genes and
                  thus have a synthetic phenotype. For example, a double mutant animal carrying two suppressor mutations may display a phenotype
                  opposite to the starting mutation and this suppression phenotype may not be seen unless both suppressor mutations are present.
                  (In <a href="#figure21">Figure 21</a> below, the mutation suppressed would be in the X gene; the redundant suppressors would be in the A and B genes.) The synthetic
                  phenotype seen with the removal of both gene products (i.e., mutations in A and B in <a href="#figure21">Figure 21</a>) can be used for epistasis tests (i.e., with gene Y in <a href="#figure21">Figure 21</a>). If there is clear epistasis of the suppressor double mutant with mutations in the pathway, the redundant suppressors most
                  likely act in a parallel subset of the otherwise linear pathway, with redundant genes A and B defined by suppressors of X
                  (see <a href="#bib40">Thomas et al. (1993)</a> for an example). This pathway would look like: (<a href="#figure21">Figure 21</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure21"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="225">
<tr>
  <td align="center"><p><a href="epistasisfig21.jpg"><img src="epistasisfig21_s.jpg" border="2" align="middle" alt="Epitasis figure 21"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 21.&nbsp;</b></p>
                  </div>
               </div>
               <p>If suppressor mutations suppress hypomorphic alleles but not null alleles, then they are more likely to be a side branch of
                  the main pathway. These mutations are in a side branch because they are unable to bypass the null allele, and thus do not
                  define a downstream component regulated by an upstream gene (see <a href="#bib27">Lee et al., 1994</a> for an example). Silent suppressors that do not show synthetic phenotypes cannot be placed in a pathway using epistasis,
                  but the spectrum of suppression of mutations in the pathway of interest can suggest their role.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1934"></a>5.&nbsp;Prospects: use of new technologies for pathway analysis
                     </h2>
                  </div>
               </div>
            </div>
            <p>Transgenes add to the arsenal of tools for pathway analysis: they can be used for suppression and epistasis. For example,
               high copy number of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> provided by a transgene results in a multivulva phenotype; this phenotype is hypostatic to the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> vulvaless phenotype, consistent with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> acting upstream of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> (<a href="#bib20">Hill and Sternberg, 1992</a>). A powerful technique in the future will be reversion of the dominant effects of transgenes. One potential problem may be
               a high degree of intragenic revertants, that is, deletions of the integrated transgenes. In general this has not happened
               (<a href="#bib45">Hajdu-Cronin et al., 1999</a>, <a href="#bib16">2004</a>; <a href="#bib24">Hsieh et al., 1999</a>).
            </p>
            <p>In <span class="emphasis"><em>Saccharomyces cerevisiae</em></span>, high copy suppression using plasmid libraries is a powerful tool for finding interacting or related genes. In <span class="emphasis"><em>C. elegans</em></span>, similar analysis will be more difficult, but still possible. Due to the larger genome size compared to yeast, it would require
               testing on the order of 500 transgenic lines, each with multiple copies of five cosmids (assuming good representation of the
               genome with 2500 cosmids). The ability to create mutations in genes of interest will be a powerful tool. As genetically defined
               genes are cloned and homologies are seen, other molecularly defined components which may lie in the pathway of interest can
               be cloned through reverse genetics and engineered into a genetic mutation (<a href="#bib29">Plasterk (1995)</a>; see <a href="http://www.wormbook.org/chapters/www_intromethodscellbiology/intromethodscellbiology.html" target="_blank">Methods in cell biology</a> for further details) for further analysis. Finally, from a geneticist's perspective, the genome project makes it more attractive
               to pursue the characterization of silent suppressor mutations because the molecular cloning will be increasingly facile.
            </p>
            <p>Since this chapter was initially published in 1995, the mechanistic interpretation of genetic interactions has received renewed
               attention as genetic epistasis analysis is performed in a large-scale manner in yeast (e.g., <a href="#bib48">Segre et al., 2005</a>; <a href="#bib47">Drees et al., 2005</a>) where systematic double mutant analysis is attempted in a genome-wide scale (e.g., <a href="#bib43">Tong et al., 2004</a>), and in <span class="emphasis"><em>C. elegans</em></span> (e.g., <a href="#bib46">Baugh et al., 2005</a>). From large scale approaches, genetic interaction graphs can be produced which allow for inference about whether genes indeed
               control a common process and also whether parallel pathways may act on a single process. Usually, these large scale versions
               of epistasis analysis rely on a quantitative measure of a simple phenotype for ease of scoring (such as percent viability).
               However, advancing technologies may allow for more complex phenotypic analyses in the future. A complementary approach would
               be to utilize computational approaches to integrate data sets acquired using various experimental methodologies obtained from
               different model organisms to predict previously uncharacterized genetic interactions (e.g., <a href="#bib44">Zhong and Sternberg, 2006</a>). These predicted genetic interactions could then be subjected to extensive phenotypic analyses, as the computational analyses
               would narrow down the number of genetic interactions to be experimentally tested, allowing for more complicated analyses to
               be carried out on a limited number of genes.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e2011"></a>6.&nbsp;Conclusion
                     </h2>
                  </div>
               </div>
            </div>
            <p>The determination of the functional order of genes in a developmental pathway provides a working model for the design of additional
               experiments. New genes can be added to the pathway once they are found, either at a branchpoint or directly in the pathway.
               The gene order can be refined upon receipt of more information about the phenotype of the mutations involved, the isolation
               of new mutations that allow more epistasis analysis to be performed, as well as more information about the molecular and biochemical
               nature of the pathway of interest. Eventually, individual pathways will be connected to other pathways, evolving into a network
               of gene interactions for developmental processes.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e2016"></a>7.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>The authors would like to thank Paul Garrity, Wendy Katz, Ralf Sommer, John DeModena, Marie-Anne F&eacute;lix, Yvonne Hadju-Cronin,
               Howard Lipshitz, Katharine Liu, Anna Newman, and the editors for critical comments. Research in our laboratory discussed here
               was supported by the U. S .P. H. S, the March of Dimes Birth Defects Foundation, and the Howard Hughes Medical Institute,
               with which P. W. S. is an Investigator.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e2021"></a>8.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ambros, V. (1989). A hierarchy of regulatory genes controls a larval to adult developmental switch in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>57</em></span>, 49&#8211;57.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2702689&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(89)90171-2" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ambros, V., and Moss, E.G. (1994). Heterochronic genes and the temporal control of <span class="emphasis"><em>C. elegans</em></span> development. Trends Genet. <span class="emphasis"><em>10</em></span>, 123&#8211;127.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8029828&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0168-9525(94)90213-5" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Anderson, P. (1995). Mutagenesis. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 31&#8211;58.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531732&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Aroian, R.V., and Sternberg, P.W. (1991). Multiple functions of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span>, a <span class="emphasis"><em>C. elegans</em></span> receptor tyrosine kinase gene required for vulval induction. Genetics <span class="emphasis"><em>128</em></span>, 251&#8211;267.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2071015&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Avery, L., and Wasserman, S. (1992). Ordering gene function: the interpretation of epistasis in regulatory hierarchies. Trends
                        Genet. <span class="emphasis"><em>8</em></span>, 312&#8211;316.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1365397&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bargmann, C.I., and Avery, L. (1995). Laser killing of cells in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 225&#8211;250.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531727&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Baugh, L.R., Wen, J.C., Hill, A.A., Slonim, D.K., Brown, E.L., and Hunter, C.P. (2005). Synthetic lethal analysis of <span class="emphasis"><em>Caenorhabditis elegans</em></span> posterior embryonic patterning genes identifies conserved genetic interactions. Genome Biol. <span class="emphasis"><em>6</em></span>, R45.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15892873&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2005-6-5-r45" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chamberlin, H.M., and Sternberg, P.W. (1993). Multiple cell interactions are required for fate specification during male spicule
                        development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>118</em></span>, 297&#8211;323.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8223264&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Clark, S.G., Lu, X., and Horvitz, H.R. (1994). The <span class="emphasis"><em>C. elegans</em></span> locus <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span>, a negative regulator of a tyrosine kinase signaling pathway, encodes two different proteins. Genetics <span class="emphasis"><em>137</em></span>, 987&#8211;997.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7982579&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Drees, B.L., Thorsson, V., Carter, G.W., Rives, A.W., Raymond, M.Z., Avila-Campillo, I., Shannon, P., and Galitski, T. (2005).
                        Derivation of genetic interaction networks from quantitative phenotype data. Genome Biol. <span class="emphasis"><em>6</em></span>, R38.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15833125&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2005-6-4-r38" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Edgley, M.L., Baillie, D.L., Riddle, D.L., and Rose, A.M. (1995). Genetic balancers. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 147&#8211;184.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531724&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ellis, R.E., Jacobson, D.M., and Horvitz, H.R. (1991a). Genes required for the engulfment of cell corpses during programmed
                        cell death in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>129</em></span>, 79&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1936965&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ellis, R.E., Yuan, J. and Horvitz, H.R. (1991b). Mechanisms and functions of Cell Death. Annu. Rev. Cell Biol. <span class="emphasis"><em>7</em></span>, 663&#8211;698.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1809356&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.cb.07.110191.003311" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ferguson, E., and Horvitz, H.R. (1989). The multivulva phenotype of certain <span class="emphasis"><em>C. elegans</em></span> mutants results from defects in two functionally-redundant pathways. Genetics <span class="emphasis"><em>123</em></span>, 109&#8211;121.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2806880&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ferguson, E.L., Sternberg, P.W., and Horvitz, H.R. (1987). A genetic pathway for the specification of the vulval cell lineages
                        of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>326</em></span>, 259&#8211;267.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2881214&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/326259a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Greenwald, I.S., and Horvitz, H.R. (1980). <span class="emphasis"><em>unc-93(<a href="http://www.wormbase.org/db/get?name=e1500;class=Variation" target="_blank">e1500</a>)</em></span>: a behavioral mutant of <span class="emphasis"><em>Caenorhabditis elegans</em></span> that defines a gene with a wild-type null phenotype. Genetics <span class="emphasis"><em>96</em></span>, 147&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6894129&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hajdu-Cronin, Y.M., Chen, W.J., Patikoglou, G., Koelle, M.R., and Sternberg, P.W. (1999). Antagonism between G(o)alpha and
                        G(q)alpha in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: the RGS protein EAT-16 is necessary for G(o)alpha signaling and regulates G(q)alpha activity. Genes Dev. <span class="emphasis"><em>13</em></span>, 1780&#8211;1793.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10421631&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hajdu-Cronin, Y.M., Chen, W.J., and Sternberg, P.W. (2004). The L-type cyclin <a href="http://www.wormbase.org/db/seq/protein?name=CYL-1;class=protein" target="_blank">CYL-1</a> and the heat-shock-factor <a href="http://www.wormbase.org/db/seq/protein?name=HSF-1;class=protein" target="_blank">HSF-1</a> are required for heat-shock-induced protein expression in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>168</em></span>, 1937&#8211;1949.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15611166&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.028423" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Han, M., Aroian, R., and Sternberg, P.W. (1990). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span> locus controls the switch between vulval and non-vulval cell types in <span class="emphasis"><em>C. elegans</em></span>. Genetics <span class="emphasis"><em>126</em></span>, 899&#8211;913.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2076820&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hedgecock, E.M., Sulston, J.E., and Thomson, J.N. (1983). Mutations affecting programmed cell deaths in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>220</em></span>, 1277&#8211;1279.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6857247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hengartner, M.O., Ellis, R.E., and Horvitz, H.R. (1992). <span class="emphasis"><em>Caenorhabditis elegans</em></span> gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span> protects cells from programmed cell death. Nature <span class="emphasis"><em>356</em></span>, 494&#8211;499.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1560823&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/356494a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hill, R.J., and Sternberg, P.W. (1992). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> gene encodes an inductive signal for vulval development in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>358</em></span>, 470&#8211;476.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1641037&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/358470a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J. (1986). Sex determination in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>: analysis if <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-3" target="_blank">tra-3</a></em></span> suppressors and characterization of <span class="emphasis"><em>fem</em></span> genes. Genetics <span class="emphasis"><em>114</em></span>, 15&#8211;52.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3770465&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J., Kondo, K., and Waterston, R.H. (1987). Suppression in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Trends Genet. <span class="emphasis"><em>3</em></span>, 325&#8211;329.</span>
                     <a href="http://dx.doi.org/10.1016/0168-9525(87)90281-2" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J., Papp, A., Pulak, R., Ambros, V., and Anderson, P. (1989). A new kind of informational suppression in the nematode
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>123</em></span>, 301&#8211;313.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2583479&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hsieh, J., Liu, J., Kostas, S.A., Chang, C., Sternberg, P.W., and Fire, A. (1999). The RING finger/B-box factor <a href="http://www.wormbase.org/db/seq/protein?name=TAM-1;class=protein" target="_blank">TAM-1</a> and a retinoblastoma-like protein <a href="http://www.wormbase.org/db/seq/protein?name=LIN-35;class=protein" target="_blank">LIN-35</a> modulate context-dependent gene silencing in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genes Dev. <span class="emphasis"><em>13</em></span>, 2958&#8211;2970.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10580003&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.13.22.2958" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Huang, L.S., Tzou, P., and Sternberg, P.W. (1994). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> locus encodes two negative regulators of <span class="emphasis"><em>C. elegans</em></span> vulval development. Mol. Biol. Cell <span class="emphasis"><em>5</em></span>, 395&#8211;412.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8054684&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lackner, M.R., Kornfeld, K., Miller, L.M., Horvitz, H.R., and Kim, S.K. (1994). A MAP kinase homolog, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mpk-1" target="_blank">mpk-1</a></em></span>, is involved in <span class="emphasis"><em>ras</em></span>-mediated induction of vulval cell fates in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genes Dev. <span class="emphasis"><em>8</em></span>, 160&#8211;173.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8299936&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lee, J., Jongeward, G.D., and Sternberg, P.W. (1994). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span>, a gene required for many aspects of <span class="emphasis"><em>C. elegans</em></span> development and behavior, encodes a clathrin-associated protein. Genes Dev. <span class="emphasis"><em>8</em></span>, 60&#8211;73.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8288128&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Muller, H.J. (1932). Further studies on the nature and causes of gene mutations. Int. Congr. Genet. <span class="emphasis"><em>6</em></span>, 213&#8211;255.
                        		</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Plasterk, R.H. (1995). Reverse genetics: from gene sequence to mutant worm. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 59&#8211;80.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531745&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Pulak, R., and Anderson, P. (1993). mRNA surveillance by the <span class="emphasis"><em>Caenorhabditis elegans smg</em></span> genes. Genes Dev. <span class="emphasis"><em>7</em></span>, 1885&#8211;1897.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8104846&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Riddle, D.L. (1977). A genetic pathway for dauer larva formation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Stadler Genet. Symp. <span class="emphasis"><em>9</em></span>, 101&#8211;120.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Riddle, D.L., Swanson, M.M., and Albert, P.S. (1981). Interacting genes in nematode dauer larva formation. Nature <span class="emphasis"><em>290</em></span>, 668&#8211;671.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7219552&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/290668a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Segre, D., Deluna, A., Church, G.M., and Kishony, R. (2005). Modular epistasis in yeast metabolism. Nat. Genet. <span class="emphasis"><em>37</em></span>, 77&#8211;83.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15592468&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1489" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sternberg, P.W. (1990). Genetic control of cell type and pattern formation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Adv. Genet. <span class="emphasis"><em>27</em></span>, 63&#8211;116.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1971988&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sternberg, P.W. (1990). Genetic control of cell type and pattern formation in <span class="emphasis"><em>C. elegans</em></span>. Adv. Genet. <span class="emphasis"><em>27</em></span>, 63&#8211;115.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sternberg, P.W. (1993). Intercellular signaling and signal transduction in <span class="emphasis"><em>C. elegans</em></span>. Annu. Rev. Genet. <span class="emphasis"><em>27</em></span>, 497&#8211;521.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8122910&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.ge.27.120193.002433" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sulston, J., and Hodgkin, J. (1988). Methods. In: "<span class="emphasis"><em>The Nematode Caenorhabditis elegans</em></span>," W.B. Wood, ed. (Cold Spring Harbor Laboratory), pp. 587&#8211;606.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Tautz, D. (1992). Redundancies, development and the flow of information. Bioessays <span class="emphasis"><em>14</em></span>, 263&#8211;266.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1596275&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/bies.950140410" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Thomas, J.H. (1993). Thinking about genetic redundancy. Trends Genet. <span class="emphasis"><em>9</em></span>, 395&#8211;399.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8310537&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0168-9525(93)90140-D" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Thomas, J.H., Birnby, D.A., and Vowels, J.J. (1993). Evidence for parallel processing of sensory information controlling dauer
                        formation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>134</em></span>, 1105&#8211;1117.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8375650&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                              <span class="bibliomisc">Tong, A.H., Lesage, G., Bader, G.D., Ding, H., Xu, H., Xin, X., Young, J., Berriz, G.F., Brost, R.L., Chang, M., et al. (2004).
                        Global mapping of the yeast genetic interaction network. Science <span class="emphasis"><em>303</em></span>, 808&#8211;813.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14764870&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1091317" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Villeneuve, A.M., and Meyer, B.J. (1990). The regulatory hierarchy controlling sex determination and dosage compensation in
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Adv. Genet. <span class="emphasis"><em>27</em></span>, 117&#8211;188.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2190446&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wu, Y., and Han, M. (1994). Suppression of activated Let-60 Ras protein defines a role of <span class="emphasis"><em>C. elegans</em></span> Sur-1 MAP kinase in vulval differentiation. Genes Dev. <span class="emphasis"><em>8</em></span>, 147&#8211;159.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8299935&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                              <span class="bibliomisc">Zhong, W., and Sternberg, P.W. (2006). Genome-wide prediction of <span class="emphasis"><em>C. elegans</em></span> genetic interactions. Science <span class="emphasis"><em>311</em></span>, 1481&#8211;1484.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16527984&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1123287" target="_blank">Article</a>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by David Fay. WormMethods editor, Victor Ambros. Last revised April 22, 2005. Published June 14, 2006. This chapter
                  should be cited as: Huang, L. S. and Sternberg, P. W. Genetic dissection of developmental pathways (June 14, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.88.2, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e21" href="#d0e21">&sect;</a></sup>Adapted from: Huang, L.S., and Sternberg, P.W. (1995). Genetic Dissection of Developmental Pathways. In Methods Cell Biol.
                  aka <span class="emphasis"><em>C. elegans</em></span>: Modern Biological Analysis of an Organism, H.F. Epstein and D.C. Shakes, eds. (San Diego: Academic Press), <span class="emphasis"><em>48</em></span>, 97&#8211;122.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e41" href="#d0e41">&#8224;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:linda.huang@umb.edu" target="_top">linda.huang@umb.edu</a></p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>


