<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic mapping and manipulation: Chapter 3-Three-point mapping with genetic markers
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMgm3pmch3_ArticleID" title="Genetic mapping and manipulation: Chapter 3-Three-point mapping with genetic markers&#xA;">
      <link rel="next" href="#d0e46" title="1.&nbsp;Basics">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="threepointmapmarkers.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMgm3pmch3_ArticleID"></a>Genetic mapping and manipulation: Chapter 3-Three-point mapping with genetic markers<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">David</span> <span class="surname">Fay<em><span class="remark">
                                 					<sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Department of Molecular Biology, University of Wyoming, Laramie, Wyoming 82071-3944 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward genetics and genetic mapping<br />>>> Genetic mapping and manipulation
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=291.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=3-point+mapping+with+genetic+markers" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

          <!--<p><a style="color:blue;cursor:pointer" onclick="return kadabra('history');" title="click to expand ..." >Publication History</a></p>
           <ul id="history" style="display:none"> -->
          <p><a style="color:blue">Publication History</a></p>
           <ul>
            <li><a href="/chapters/www_threepointmapmarkers/threepointmapmarkers.html">version 1</a></li>
            <li><a href="/chapters/www_classicalgenmethods/classicalgenmethods.html">latest version</a></li>
           </ul>

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e46">1. Basics</a></span></dt>
               <dt><span class="sect1"><a href="#d0e286">2. Saving recombinants</a></span></dt>
               <dt><span class="sect1"><a href="#d0e376">3. Some words of caution</a></span></dt>
               <dt><span class="sect1"><a href="#d0e406">4. Finding and picking recombinants</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e647">4.1. Some other considerations to keep in mind</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e742">5. Exit strategies</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e46"></a>1.&nbsp;Basics
                     </h2>
                  </div>
               </div>
            </div>
            <p>Once you have assigned your mutation to a chromosome, it's time to begin three-point mapping. Three-point mapping is pretty
               much the backbone of worm genetics and is almost always an obligatory step in the process of cloning our mutants. Even SNP
               (single-nucleotide polymorphism) mapping (see below) is really just a high-tech variation of classical three-point mapping
               and is usually preceded by three-point mapping with genetic markers. The basic idea is that we cross our mutation (<span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>) into a strain with two linked morphological markers (<span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span>) that are on the same chromosome as <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>, to generate the <span class="emphasis"><em><span class="bold"><strong>m/a b</strong></span></em></span> heterozygote. We then isolate and follow two classes of recombinant progeny; those that display the <span class="bold"><strong>A</strong></span> phenotype only (<span class="bold"><strong>A non-B</strong></span> recombinants) and those that display only the <span class="bold"><strong>B</strong></span> phenotype only (<span class="bold"><strong>B non-A</strong></span> recombinants). By seeing which of these two classes produce the mutant phenotype (<span class="bold"><strong>M</strong></span>) and by scoring the percentages for each, we can determine whether our mutation lies to the left, to the right, or between
               our set of markers. In the case where the mutation lies in between, we may then determine the approximate distance from each
               marker.
            </p>
            <p><a href="#figure1">Figure 1</a> depicts the outcome of a recombination between markers <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> when <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> lies either to the left or right of the markers. When <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> lies to the left, essentially all <span class="bold"><strong>B non-A</strong></span> recombinant animals will throw 1/4<span class="bold"><strong>B M</strong></span> progeny (as well as <span class="bold"><strong>B</strong></span> and <span class="bold"><strong>A B</strong></span>), whereas <span class="bold"><strong>A non-B</strong></span> recombinant animals will throw only <span class="bold"><strong>A</strong></span> and <span class="bold"><strong>A B</strong></span> progeny. When we see this kind of pattern, we can conclude that <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> lies to the left of <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> or perhaps to the right of <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> but very close. The reason for this is that if <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> were very close to <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> but between <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span>, the frequency of generating the <span class="emphasis"><em><span class="bold"><strong>a m</strong></span></em></span> recombinant chromosome would be very low (also see below). Thus although <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> is most likely to the left of a, we often have this caveat. Greater numbers of recombinants can help to diminish this possibility,
               if not rule it out completely. The situation for <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> lying to the right is simply the reverse.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="gm3pmfigch3-1.jpg"><img src="gm3pmfigch3-1_s.jpg" border="2" align="middle" alt="three-point mapping with genetic markers Figure 3-1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1.&nbsp;</b></p>
               </div>
            </div>
            <p>The mapping described above, though useful, only tells us that <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> is likely to be left or right of our given markers. It doesn't provide any information about how far from these markers <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> might reside. To determine this we need to use markers that flank <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>, as shown in <a href="#figure2">Figure 2</a>. Here we see that depending on the site of the cross over,<span class="bold"><strong>A non-B</strong></span> recombinant animals can in some cases acquire <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> (#1) and in other cases not (#2). The same is true for <span class="bold"><strong>B non-A</strong></span> animals. In three-point mapping, we seek to determine the ratio of recombinant animals that pick up the mutation versus those
               that do not. This ratio provides us with a direct genetic position for the mutation as illustrated in <a href="#figure3">Figure 3</a>.
            </p>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="gm3pmfigch3-2.jpg"><img src="gm3pmfigch3-2_s.jpg" border="2" align="middle" alt="three-point mapping with genetic markers Figure 3-2"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 2.&nbsp;</b></p>
               </div>
            </div>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="gm3pmfigch3-3.jpg"><img src="gm3pmfigch3-3_s.jpg" border="2" align="middle" alt="three-point mapping with genetic markers Figure 3-3"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 3.&nbsp;</b></p>
               </div>
            </div>
            <p>Here markers <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> are in cis and are located 5.0 map units apart, whereas our mutation, <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>, is in trans to <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span>. In the situation on the left, were we to pick <span class="bold"><strong>B non-A</strong></span> recombinant animals, four-fifths or 80% would be expected to carry <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> in cis to b.<span class="bold"><strong>A non-B</strong></span> recombinants, on the other hand, would acquire <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> only one-fifth or 20% of the time. On the right, <span class="bold"><strong>B non-A</strong></span> animals will acquire <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> only 40% of the time, whereas <span class="bold"><strong>A non-B</strong></span> animals will acquire it 60% of the time. Obviously, when picking recombinants from both sides, the numbers should converge
               on a single location, i.e., the frequencies should add up to 100%. These numbers can be used to specifically assign a genetic
               location. For example, in the left diagram, if a were at genetic position 0.0 on the chromosome and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> at 5.0, having 20% of <span class="bold"><strong>A non-B</strong></span> recombinants acquire <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> would lead to a map position assignment of 1.0. Obviously, the greater the number of recombinants scored, the greater the
               certainty of the assignment.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e286"></a>2.&nbsp;Saving recombinants
                     </h2>
                  </div>
               </div>
            </div>
            <p><span class="emphasis"><em>Always save recombinants; they often prove very useful for subsequent mapping</em></span>, not to mention genetic studies where having a linked marker may prove indispensable. <a href="#figure4">Figure 4</a> shows an example of how to use the recombinant chromosome for further mapping (also see <a href="http://www.wormbook.org/chapters/www_mappingdeficienduplicat.2/mappingdeficienduplicat.html" target="_blank">Mapping with deficiencies and duplications</a>). Imagine we are mapping a <span class="emphasis"><em><span class="bold"><strong>ste</strong></span></em></span>  mutation and have placed it between <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> and  <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span>  markers that are separated by 5.0 map units (step 1). The ratios place the <span class="emphasis"><em><span class="bold"><strong>ste</strong></span></em></span>  mutation closer to the <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> marker (10 out of 25 <span class="bold"><strong>Unc non-Dpy</strong></span> recombinant animals threw <span class="bold"><strong>Unc Ste</strong></span> progeny; step 2). We save the<span class="emphasis"><em><span class="bold"><strong>unc ste/unc dpy</strong></span></em></span> strain and cross it to a strain that is homozygous for a <span class="emphasis"><em><span class="bold"><strong>bli</strong></span></em></span> mutation (step 3). We obtain the strain shown in step 4 and then screen for <span class="bold"><strong>Unc non-Ste</strong></span> animals (step 5). In this case, 50% of the <span class="bold"><strong>Unc non-Ste</strong></span> recombinants acquired the <span class="emphasis"><em><span class="bold"><strong>bli</strong></span></em></span> mutation, placing <span class="emphasis"><em><span class="bold"><strong>ste</strong></span></em></span>  and <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutations at an equal distance (but on opposite sides) from <span class="emphasis"><em><span class="bold"><strong>bli.</strong></span></em></span></p>
            <p>In this way, we continue to refine the map position of our mutation. Usually the data from different mapping schemes will
               tend to agree, although not always. This may be due to a number of factors. <span class="emphasis"><em>In general, the farther apart the markers are, the less precise the mapping tends to be</em></span>. Thus we put more weight on data acquired using nearby markers than those that are at some distance. In addition, <span class="emphasis"><em>it is highly advisable to map using markers that have already been cloned</em></span>. This provides a precise chromosomal location and allows one to compare directly the genetic and physical maps. If you have
               no choice but to use a non-cloned mutant for mapping purposes, check <a href="http://www.wormbase.org" target="_blank">Wormbase</a> or journal articles for information regarding how this gene was mapped to its present location.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="gm3pmfigch3-4.jpg"><img src="gm3pmfigch3-4_s.jpg" border="2" align="middle" alt="three-point mapping with genetic markers Figure 3-4"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 4.&nbsp;</b></p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e376"></a>3.&nbsp;Some words of caution
                     </h2>
                  </div>
               </div>
            </div>
            <p>What happens if you initially map your mutation to the wrong chromosome and then try to carry out three-point mapping? Essentially,
               your mutation segregates independently of the recombinant chromosome and will be picked up two-thirds of the time. Thus, if
               for example, 67% of your <span class="bold"><strong>Dpy non-Unc</strong></span> and <span class="bold"><strong>Unc non-Dpy</strong></span> animals throw your mutation, you may want to consider revisiting your two-point mapping data.
            </p>
            <p>Another thing to be aware of is the possibility of either multiple crossovers events ordouble recombinants. Multiple crossovers
               occur when two or more recombination events have taken place on a single chromosome during meiosis. Although such events are
               purportedly extremely rare, they can happen, and the larger the number of recombinants scored, the greater the possibility
               that this could be an issue. For this reason, it is generally wise to stick to markers that are approximately 5.0 map units
               apart when doing three-point mapping. In any case, always be aware of this possibility and refine your interpretations if
               necessary. Double recombinants simply refer to worms that contain two recombinant (non-parental) chromosomes. These are quite
               obvious to spot, as they will only throw recombinant progeny. For example, if you pick a <span class="bold"><strong>Dpy non-Unc</strong></span> and it throws only <span class="bold"><strong>Dpys</strong></span> (no <span class="bold"><strong>Dpy Uncs</strong></span>), then both chromosomes must have been recombinant. You will want to toss such strains as they could contain a mixture of
                <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span>  and <span class="emphasis"><em><span class="bold"><strong>dpy m</strong></span></em></span> chromosomes, which would unduly complicate things. As with multiple cross over events, the chance for double recombinants
               increases as the distance increases between the markers.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e406"></a>4.&nbsp;Finding and picking recombinants
                     </h2>
                  </div>
               </div>
            </div>
            <p>At the most basic level, two things should be anticipated in advance of picking recombinants for mapping: 1) the expected
               frequency of recombinants; and 2) the plate phenotype(s) of the recombinant animals. The first concern is relatively easy
               to calculate. Because you should know the distance between the two genetic markers, the frequency of recombination events
               between these markers can be directly determined. For example, if two markers (<span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span>) are 2.0 map units apart, then a crossover event will occur between <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> in 2% of the chromatid pairs (4% of the tetrads) leading to 1% of the gametes containing an <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span>-only chromosome and 1% containing<span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span>-only chromosome. Because hermaphrodite worms are diploid for all chromosomes, this effectively doubles the chance of acquiring
               a recombinant chromosome in the progeny, as it can come from either the sperm or the oocyte. To detect the recombinant, however,
               it must be over the 'correct' parental chromosome (<span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span>), which will occur only 50% of the time. The end result is that if one is looking specifically for <span class="bold"><strong>A non-B</strong></span> recombinants, and <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> are 2.0 map units apart, then an animal with an <span class="bold"><strong>A non-B</strong></span> phenotype will occur on average about 1% of the time. Likewise, <span class="bold"><strong>B non-A</strong></span> animals will occur 1% of the time. Obviously, if the mapping allows picking of either <span class="bold"><strong>A non-B</strong></span> or <span class="bold"><strong>B non-A</strong></span> non-recombinants, this will effectively double the total number of recombinant animals that can be obtained from a given
               number of plates.
            </p>
            <p>The next step is to recognize and pick the recombinant animals. But first it is important before picking from any plate to
               ask the question: <span class="bold"><strong>Do the animals on this plate display the expected phenotypes?</strong></span> In effect, you are thereby asking: <span class="bold"><strong>Did the parental animal have the correct genotype?</strong></span> This is exceedingly important to determine before picking any recombinants. The reason is that recombination events may have
               occurred in the previous generation such that the cloned parental animal may not have had the correct genotype. For example,
               you may have picked phenotypically wild-type animals from a plate where the parental animal was of genotype <span class="emphasis"><em><span class="bold"><strong>m/a b</strong></span></em></span>. Given that self-progeny with the genotype <span class="emphasis"><em><span class="bold"><strong>m/a b</strong></span></em></span> will be wild type, you might imagine that you are safe in assuming that all wild-type progeny will therefore have genotype
               <span class="emphasis"><em><span class="bold"><strong>m/a b</strong></span></em></span>. But imagine the following two scenarios depicted in <a href="#figure5">Figure 5</a>. In the scenario on the left,<span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> lies to one side of the markers <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span>. A recombination event between the markers and <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> can result in the creation of a wild-type chromosome (+) as well as a triple mutant chromosome (not shown). Therefore, when
               the recombinant + chromosome is paired with one of the parental chromosomes, phenotypically wild-type animals would be generated
               with the genotype <span class="emphasis"><em><span class="bold"><strong>m/+</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>+/a b</strong></span></em></span> (and not the expected <span class="emphasis"><em><span class="bold"><strong>m/a b</strong></span></em></span>). Likewise, animals of <span class="emphasis"><em><span class="bold"><strong>m/a</strong></span></em></span> genotypes (though probably not <span class="emphasis"><em><span class="bold"><strong>m/b</strong></span></em></span>) could arise following a single recombination event between <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and b. For the case on the right, a double recombination event would have to occur to generate a wild-type chromosome, and
               this will admittedly be very rare. A single recombination event, however, could result in either <span class="emphasis"><em><span class="bold"><strong>m/a</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>m/b</strong></span></em></span> animals, which will also appear phenotypically wild type (also see below).
            </p>
            <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="gm3pmfigch3-5.jpg"><img src="gm3pmfigch3-5_s.jpg" border="2" align="middle" alt="three-point mapping with genetic markers Figure 3-5"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 5.&nbsp;</b></p>
               </div>
            </div>
            <p>Clearly, one does not want to pick recombinants from plates where the parental animal had the incorrect genotype. This will
               wreak havoc on one's mapping and lead to incorrect conclusions. The solution is simple: <span class="emphasis"><em>Make sure the phenotypes observed on the plate correspond to the correct parental genotype</em></span>. For example, if the parental animal has the expected genotype <span class="emphasis"><em><span class="bold"><strong>m/a b</strong></span></em></span>, then one should see wild-type animals (<span class="emphasis"><em><span class="bold"><strong>m/a b</strong></span></em></span>), <span class="bold"><strong>M</strong></span> animals (<span class="emphasis"><em><span class="bold"><strong>m/m</strong></span></em></span>), and <span class="bold"><strong>AB</strong></span> animals (<span class="emphasis"><em><span class="bold"><strong>a b/a b</strong></span></em></span>). In addition, it should be possible to find occasional recombinant animals (<span class="bold"><strong>A non-B</strong></span> and <span class="bold"><strong>B non-A</strong></span>), which is exactly what you are looking for. Although simple in practice, fundamental errors by novice mappers are not uncommon.
               For example, some <span class="bold"><strong>Dpy</strong></span> mutants may appear partially Unc, thus, loss of the <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutation could initially go unnoticed. Other markers such as <span class="emphasis"><em><span class="bold"><strong>let</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>egl</strong></span></em></span> may require even greater care to maintain. In the end, strict diligence is the only weapon against such mistakes. <span class="emphasis"><em>Bottom line: Do whatever you consider necessary to ensure that recombinants are obtained only from plates with the correct
                     parental genotype</em></span>.
            </p>
            <p>Recognizing the recombinants that you want may not be trivial! Or it may be, depending on the nature of the mutant phenotypes
               and your level of experience. For example, you acquire a <span class="emphasis"><em><span class="bold"><strong>dpy unc</strong></span></em></span> strain for mapping purposes and the double-mutant animals indeed look both Dpy and Unc, but what will the <span class="bold"><strong>Dpy non-Unc</strong></span> or the <span class="bold"><strong>Unc non-Dpy</strong></span> recombinant animals actually look like? Often one does not have either the <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutation alone for comparison. In the absence of having the single-mutant strains available, the best approach is to read
               up on the descriptions of the single mutant phenotypes, ask experienced members of your lab for advice, and keep handy the
               double mutant strain for comparison to any potential recombinants. Once you have isolated a few true recombinants, finding
               new ones will suddenly get much easier.
            </p>
            <p>How many recombinants should one pick from any given plate? This may depend on several factors. <span class="emphasis"><em>As a rule, be very cautious of plates where you seem to have hit a "gold mine"!</em></span> ("Wow, I can get all 20 recombinants off of one plate!" NOT.) The simplest explanation when encountering such a plate is
               that a recombination event must have occurred in the previous generation to affect the parent. This is precisely the situation
               that was described above. Looking at such plates it will probably be clear that the parent animal did not have the correct
               genotype. In this case, it is permissible to pick a single recombinant animal, as this does represent one legitimate recombination
               event. However, even in cases where most animals correspond to the non-recombinant phenotypes (indicating that a parental
               recombination event did not occur), <span class="emphasis"><em>it is still advisable to pick only 2 or 3 recombinant progeny from any one plate</em></span>. The (perhaps overly paranoid) worry is that a rare mitotic recombination event could have occurred in the germline to generate
               a clone of identical recombinant sperm or oocytes.
            </p>
            <p>Often when looking for recombinants to pick, one will examine the same set of plates for several days in a row. It is a common
               experience that recombinants that are 'invisible' one day will jump out at you the next. Certainly for some types of mutants
               such as <span class="emphasis"><em><span class="bold"><strong>ste</strong></span></em></span>  or <span class="emphasis"><em><span class="bold"><strong>egl</strong></span></em></span>, the recombinant phenotype may only be obvious once animals are well into adulthood. When scanning the same set of plates
               over several days, keep whatever notes necessary to ensure that you don't keep picking your recombinants off the same plate
               without knowing it. Proper note taking and labeling of plates will prevent this from happening.
            </p>
            <p>What happens if you accidentally pick a non-recombinant animal by mistake? No problem, as it should be quite obvious when
               looking at progeny in the next generation that a recombinant was not picked. For example, if you attempted to pick a <span class="bold"><strong>Dpy non-Unc</strong></span> animal and notice several days later that the "recombinant" worm has failed to throw appreciable numbers of <span class="bold"><strong>Dpy non-Unc</strong></span> animals, or is perhaps throwing phenotypically wild-type animals, obviously the parental animal was not a true recombinant.
               Chuck the plate and move on. <span class="bold"><strong>It is better to pick some false recombinants (and eliminate them later) than to miss picking any true recombinants.</strong></span></p>
            <p>A note of caution: <span class="bold"><strong>Make sure that when picking recombinants, you do not carry over contaminating eggs or larvae!</strong></span> This is surprisingly easy to do and will usually ruin your ability to score that particular recombinant since the plate will
               be contaminated with animals of non-recombinant phenotypes. If the plate is crowded, move the recombinant animals to a less
               populated region of the plate in order to clean the recombinant animal of larvae or eggs that may have stuck to its side.
               Sometimes it may even be necessary to transfer the recombinant to a 'clean-up' plate before cloning to its own plate. As a
               second line of defense, always watch the recombinant animal after transferring it to its own plate and destroy any contaminating
               eggs or larvae that may come off. Such procedures become second nature very quickly.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e647"></a>4.1.&nbsp;Some other considerations to keep in mind
                        </h3>
                     </div>
                  </div>
               </div>
               <p>1) The presence of certain phenotypes may prevent that accurate scoring of other phenotypes. For example, the <span class="bold"><strong>Bli</strong></span> (blister) phenotype is often masked (suppressed) by <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>rol</strong></span></em></span> mutations; <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutations may mask the <span class="bold"><strong>Rol phenotype</strong></span>;  <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span>  mutations will usually mask a <span class="bold"><strong>Lon</strong></span> (long) phenotype; certain <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutations may sometimes appear <span class="bold"><strong>Egl</strong></span>, etc. Obviously, there may be a lot to consider, and going into the mapping well informed is essential. Surprisingly, one
                  can sometimes map with mutations that would seem unlikely. For example, it may be possible to identify certain <span class="bold"><strong>UncX non-UncY</strong></span> mutants, depending on the nature of the two <span class="bold"><strong>Unc</strong></span> phenotypes.
               </p>
               <p>2) <span class="bold"><strong>It is highly recommended that you render your phenotypic judgments on animals that have been propagated on clean plates containing
                        sufficient OP50.</strong></span> Many bacterial contaminants can actually prevent the accurate scoring of <span class="bold"><strong>Unc</strong></span> and <span class="bold"><strong>Ste</strong></span> phenotypes in genetic crosses. In addition, starved plates can lead to erroneous conclusions regarding sterility, brood sizes,
                  <span class="bold"><strong>Egl</strong></span>, and other phenotypes. If in doubt, you can always bleach your strains and reassess phenotypes later from clean plates. Better
                  yet, make sure that your plates and strains are clean before initiating the mapping process.
               </p>
               <p>3) Clearly, the <span class="emphasis"><em>most efficient type of three-point mapping would allow for picking of recombinants from "both" directions</em></span>. For example, one can pick <span class="bold"><strong>Unc non-Dpy</strong></span> and <span class="bold"><strong>Dpy non-Unc</strong></span> recombinant animals from a strain with a <span class="emphasis"><em><span class="bold"><strong>dpy unc</strong></span></em></span> chromosome. The benefit of this setup is that it effectively doubles the number of recombinants obtained from a given number
                  of plates, and provides independent mapping information from both types of recombinants (which will hopefully correspond!).
                  It is not uncommon, however, that the markers will dictate that only one of two possible recombinants is picked. For example,
                  when looking for recombinants between <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>let</strong></span></em></span> (lethal) mutations, it will only be possible to identify and pick <span class="bold"><strong>Unc non-Lets</strong></span> for obvious reasons. The same thing occurs when using markers that confer a sterile phenotype, in situations where one of
                  the markers masks the phenotype of the other (e.g., <span class="bold"><strong>Rol non-Uncs</strong></span> versus <span class="bold"><strong>Unc non-Rols</strong></span>), or where penetrance is an issue (see above).
               </p>
               <p>4) Finally, as with all genetics, it is wise to pick more worms than is anticipated to be necessary. The rate-limiting step
                  for all genetics is growth of the animals, and not the time required to transfer a few more worms to plates. Still, there
                  is a limit to how much information can be gleaned from any one cross, and good researchers are always efficient in the use
                  of their time.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e742"></a>5.&nbsp;Exit strategies
                     </h2>
                  </div>
               </div>
            </div>
            <p>When is one done with three-point mapping? Before the advent of SNP mapping, the common strategy was to play out all of one's
               cards. Namely, you would use every available genetic marker or chromosomal rearrangement to minimize the size of the genetic
               region known (or likely) to contain your mutation. Once this was done, cosmid injections to obtain rescue could begin. Of
               course, the paucity of markers in many regions often meant that many cosmids would have to be injected (not a trivial task).
               Even worse, for many regions, complete cosmid coverage was (and is still) very incomplete. As described in <a href="http://www.wormbook.org/chapters/www_SNPsintrotwopointmap/SNPsintrotwopointmap.html" target="_blank">SNPs: introduction and two-point mapping</a>, all this has changed with the advent of SNP mapping, and although a reasonable amount of standard three-point mapping is
               still done, it is not done so exhaustively. Thus to answer the initial query - one usually ceases three-point mapping when
               the mutation has been placed within a 1- to 5-map unit region that is bounded by two solid genetic markers which correspond
               to known locations the physical map (i.e., the markers have been cloned). Having said that, if the region is rich with genetic
               markers, there is no particular reason to stop genetic mapping all together. In some circumstances, it may still be the most
               efficient means by which to narrow down the region of interest to a workable size for injections or candidate cloning.
            </p>
            <p>Regardless of the stage at which you conclude your genetic mapping, you will still want to use your data to find ground zero
               on the physical map; that is, the stretch of DNA where your mutation is predicted to reside. One simple way to do this is
               to construct a complete graphic of the physical map for the region, a process that may require some actual cutting and pasting
               of a printout of the map. One then takes actual physical measurements of the distances between the markers used (e.g., 135
               mm) and, based on the obtained mapping data, finds the point on the chromosome that has been implicated. Note: When generating
               this map, <span class="underline">do not</span> use AceDB, as this program does not currently draw to scale regions containing cosmid gaps. The <a href="http://genome.wustl.edu/genome/celegans/celegans_snp.cgi" target="_blank">SNPs database</a> currently provides an excellent graphic for this purpose, and as you will likely be completing your mapping using SNPs, you
               will want to construct this anyway. Alternatively, one can avoid the graphic map entirely by calculating the predicted site
               of the mutation based on the numerical chromosomal locations of the markers used (e.g., 9,279,450). This is quite straightforward
               and eliminates potential errors associated with any graphic representation. Detailed information for this purpose can be found
               on <a href="http://www.wormbase.org" target="_blank">Wormbase</a>.
            </p>
            <p>It should be pointed out that while it may be satisfying to point to a single base pair on the chromosome as being the most
               likely site of one's mutation, this prediction is only as good as the mapping data. Furthermore, regional variations in recombination
               frequency along the chromosome will lead to significant discrepancies between the actual locations of genes and those predicted
               using these methods. Still, this will allow you to predict better which SNPs might be most appropriate for testing first,
               or if the region is small enough, to compile a list of likely rescuing cosmids and prioritize your order of injections.
            </p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised May 3, 2005. Published February 17, 2006. This chapter should be cited as: Fay, D. Genetic
                  mapping and manipulation: Chapter 3-Three-point mapping with genetic markers (February 17, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.92.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 David Fay. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:davidfay@uwyo.edu" target="_top">davidfay@uwyo.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

