<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic mapping and manipulation: Chapter 5-SNPs: Three-point mapping
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMgmSNP3pmch5_ArticleID" title="Genetic mapping and manipulation: Chapter 5-SNPs: Three-point mapping&#xA;">
      <link rel="next" href="#d0e44" title="1.&nbsp;The basics">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="SNPsintrothreepointmap.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMgmSNP3pmch5_ArticleID"></a>Genetic mapping and manipulation: Chapter 5-SNPs: Three-point mapping<sup><a name="d0e6" href="#ftn.d0e6">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">David</span> <span class="surname">Fay<em><span class="remark">
                                 					<sup><a name="d0e33" href="#ftn.d0e33">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Department of Molecular Biology, University of Wyoming, Laramie, Wyoming 82071-3944 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward genetics and genetic mapping<br />>>> Genetic mapping and manipulation
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=296.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=SNPs:+three-point+mapping" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e44">1. The basics</a></span></dt>
               <dt><span class="sect1"><a href="#d0e218">2. Examples of SNP mapping</a></span></dt>
               <dt><span class="sect1"><a href="#d0e351">3. General SNP mapping strategies</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e44"></a>1.&nbsp;The basics
                     </h2>
                  </div>
               </div>
            </div>
            <p>As described in <a href="http://www.wormbook.org/chapters/www_SNPsintrotwopointmap/SNPsintrotwopointmap.html" target="_blank">SNPs: introduction and two-point mapping</a> single nucleotide polymorphisms (SNPs) provide an exceptionally powerful means for supplementing traditional mapping practices.
               SNP mapping is both conceptually and practically an extension of standard three-point mapping (see <a href="http://www.wormbook.org/chapters/www_threepointmapmarkers/threepointmapmarkers.html" target="_blank">Three-point mapping with genetic markers</a>), and will invariably require some previous genetic mapping of the mutation of interest. The specific strategy for mapping
               mutations by SNP analysis will depend somewhat on the nature of your mutant phenotype. In most instances, it is advantageous
               to generate a chromosome containing your mutation that is flanked by two easily discernable genetic markers such as <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span>. The genetic distance between the flanking markers should be as small as possible, something in the range of a few map units.
               However, some regions of the chromosome will require that you use markers that are separated by significantly greater distances.
               Although not optimal, this is still workable. The issue is that the farther apart the markers are, the more recombinants that
               will ultimately be needed to obtain the same level of mapping refinement. Still, we have successfully used markers as far
               apart as 12 map units to narrow a region down to a handful of genes. Ultimately, it is more important to use easily discernable
               markers than to shave off a few map units of distance. Another point is that it is useful to be able to pick recombinants
               from both directions if at all possible. At the very least, it doubles the number of useful recombinants on the plates and
               can provide an added level of security, provided all your data points are in agreement.
            </p>
            <p>Creating strains for SNP mapping. For viable mutations, triple-mutant chromosomes can easily be obtained by generating balanced
               strains of the <span class="emphasis"><em><span class="bold"><strong>a m/m b</strong></span></em></span> genotype, where <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> is the mutation and <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> are the marker mutations (<a href="#figure1">Figure 1</a>, example #1). One then picks either <span class="bold"><strong>A M</strong></span> or <span class="bold"><strong>M B</strong></span> animals and looks for the appearance of <span class="bold"><strong>A M B</strong></span> animals in some fraction of the self progeny (e.g., from a parent that was <span class="emphasis"><em><span class="bold"><strong>a m b/a m</strong></span></em></span>), indicating the presence of a recombinant <span class="emphasis"><em><span class="bold"><strong>a m b</strong></span></em></span> chromosome. The choice of picking <span class="bold"><strong>A M</strong></span> or <span class="bold"><strong>M B</strong></span> animals will depend on the ease of recognition of the <span class="bold"><strong>A M B</strong></span> animal in either the <span class="bold"><strong>A M</strong></span> or <span class="bold"><strong>M B</strong></span> background. For example, it is somewhat easier to identify <span class="bold"><strong>Dpy Unc</strong></span> animals from plates where most of other animals are <span class="bold"><strong>Unc</strong></span> than the other way around.
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="gmSNP3pmfigch5-1.jpg"><img src="gmSNP3pmfigch5-1_s.jpg" border="2" align="middle" alt=" SNPs: three-point mapping Figure 5-1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1.&nbsp;</b></p>
               </div>
            </div>
            <p>For non-viable mutations, the construction and maintenance of the triple mutant chromosome is somewhat more involved. Here
               you will need to initially construct strains that are either <span class="emphasis"><em><span class="bold"><strong>a m/a b</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>m b/a b</strong></span></em></span> (<a href="#figure1">Figure 1</a>, example #2). Then by blindly picking sufficient animals of the <span class="bold"><strong>A B</strong></span> phenotype, it should be possible to isolate strains that throw 1/4 <span class="bold"><strong>A M B</strong></span>, indicating the <span class="emphasis"><em><span class="bold"><strong>a m b/a b</strong></span></em></span> genotype. Alternatively, you could generate the strain <span class="emphasis"><em><span class="bold"><strong>m b/a c</strong></span></em></span>, where <span class="emphasis"><em><span class="bold"><strong>c</strong></span></em></span> lies just to the right of <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> and confers a phenotype that is distinct from that of either <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span>. By picking <span class="bold"><strong>A-non-C</strong></span> animals, a reasonably high percentage of recombinants should be of the <span class="emphasis"><em><span class="bold"><strong>a m b/a c</strong></span></em></span> genotype (<a href="#figure1">Figure 1</a>, example #3). This approach has the added advantage that the strain generated is at least partially balanced and therefore
               easier to maintain.
            </p>
            <p>On the other hand, if your mutation is inviable but you have already constructed several well-balanced strains that each carry
               the <span class="emphasis"><em><span class="bold"><strong>a m</strong></span></em></span> or the <span class="emphasis"><em><span class="bold"><strong>m b</strong></span></em></span> chromosomes, it is reasonable that you might want to use these strains directly for SNP mapping (also see below). <span class="emphasis"><em>The key here is to have two strains with markers on opposite sides, as this will allow you to move your region in from both
                     directions</em></span>. The same holds true for non-viable mutants when working with a triple-mutant chromosome (<span class="emphasis"><em><span class="bold"><strong>a m b</strong></span></em></span>). This is because the only recombinants that can be propagated as homozygotes (<span class="emphasis"><em><span class="bold"><strong>a/a</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>b/b</strong></span></em></span>) will necessarily have resulted from crossover events that lead to the loss of <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>. Thus, in this situation, it is essential to be able to pick both <span class="bold"><strong>A-non-B</strong></span> and <span class="bold"><strong>B-non-A</strong></span> animals to narrow down your region from both sides.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e218"></a>2.&nbsp;Examples of SNP mapping
                     </h2>
                  </div>
               </div>
            </div>
            <p><a href="#figure2">Figure 2</a> shows the basic flowchart and potential outcomes for mapping a viable mutation that is flanked by the visible markers <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> using four regional SNPs. In this scheme, F1 heterozygous hermaphrodites are generated through crosses and allowed to self
               (step 1). F2 recombinants are then isolated (step 2) and sufficient self progeny are subsequently cloned such that animals
               homozygous for the recombinant chromosome are obtained in the F3 generation (step 3). F4 progeny can then be assayed for the
               presence of N2 or CB4856 sequences at an given SNP locus (step 4). For example, a <span class="bold"><strong>Dpy non-Unc</strong></span> resulting from a crossover at point <span class="bold"><strong>A</strong></span> would be N2 for <span class="bold"><strong>SNP1</strong></span> and would not contain the mutation <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>. Thus, <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> must lie to right of SNP1. Likewise, a <span class="bold"><strong>Dpy non-Unc</strong></span> resulting from a crossover at point <span class="bold"><strong>C</strong></span> would result in an <span class="bold"><strong>M</strong></span> animal that was CB4856 for SNP3 and SNP4. Thus, <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> must reside to the left of both SNP3 and SNP4. In the examples provided in <a href="#figure2">Figure 2</a>, the data from recombinants <span class="bold"><strong>B</strong></span> and <span class="bold"><strong>C</strong></span> establish the tightest endpoints on the left and right respectively. As described above, non-viable mutants may be most conveniently
               mapped using two strains, e.g., <span class="emphasis"><em><span class="bold"><strong>a m</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>m b</strong></span></em></span> (<a href="#figure3">Figure 3</a>). By picking <span class="bold"><strong>A non-M</strong></span> from one strain and <span class="bold"><strong>B non-M</strong></span> from the other, both sides can effectively be pushed in.
            </p>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="gmSNP3pmfigch5-2.jpg"><img src="gmSNP3pmfigch5-2_s.jpg" border="2" align="middle" alt=" SNPs: three-point mapping Figure 5-2"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 2.&nbsp;</b></p>
               </div>
            </div>
            <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="gmSNP3pmfigch5-3.jpg"><img src="gmSNP3pmfigch5-3_s.jpg" border="2" align="middle" alt=" SNPs: three-point mapping Figure 5-3"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 3.&nbsp;</b></p>
               </div>
            </div>
            <p>It is also worth noting that although we mostly use SNP mapping to generate progressively tighter endpoints, valuable three-point
               mapping data can be gleaned through this process. For example, imagine that you have assigned your mutation to the region
               between SNP2 and SNP3 (<a href="#figure2">Figure 2</a>). In doing so, you notice that of the 20 <span class="bold"><strong>Dpy non-Unc</strong></span> recombinants that experienced a crossover in this region (meaning they are all N2 for SNP2 and CB4856 for SNP3), 18 contain
               the mutation <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>. This would suggest that <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> lies relatively close to SNP2, as the large majority of recombination events resulted in the retention of <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> on the <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span> chromosome. Likewise, most <span class="bold"><strong>Unc non-Dpy</strong></span> recombinants isolated in this region (meaning they are N2 for SNP3 and CB4856 for SNP2) would be expected to be <span class="bold"><strong>non-M</strong></span>. This may at first seem counterintuitive. This is because for normal three-point mapping (<a href="http://www.wormbook.org/chapters/www_threepointmapmarkers/threepointmapmarkers.html" target="_blank">Three-point mapping with genetic markers</a>), the mutation is always on the opposite chromosome from the markers, hence a high percentage of <span class="bold"><strong>Dpy-M</strong></span> animals would indicate that <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> lies closer to the <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> side of things. The difference here is that the mutation and markers all start out on the same chromosome, thus a minor revision
               in thinking is required. As always, if this does not make immediate sense, draw it out.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e351"></a>3.&nbsp;General SNP mapping strategies
                     </h2>
                  </div>
               </div>
            </div>
            <p>The optimal strategies for efficient SNP mapping will vary somewhat depending on the nature of the mutant phenotype and on
               the time commitment of the researcher. We have had good success using a two-tiered approach. Initially, we isolate 75-100
               recombinant animals and then in the subsequent generation isolate strains that are homozygous for the recombinant chromosome.
               We then begin the process of testing regional SNPs, initially being somewhat conservative in our choices of locations (meaning
               that we try not to overshoot). We then methodically move in our endpoints, discarding as we go those recombinants that are
               no longer informative. At some point, this initial pool will fail to yield any additional information and we are left with
               just a few recombinants holding down the left and right inner endpoints.
            </p>
            <p>In the second phase of SNP mapping, we go for the "mother load", where several hundred or more recombinants are isolated.
               The strategy is to quickly screen these recombinants using the inner-most (or near inner-most) pair of informative SNPs. For
               reasons described above, it is preferable that these SNPs be CB4856-specific cutters, although well-behaved N2 cutters may
               be workable. <span class="emphasis"><em>Most important is that the chosen SNPs provide consistent and easily interpretable results</em></span>. For carrying out PCR, you can use either the initial recombinant animal (after it has had 1-2 days to lay sufficient eggs)
               or 5 or 6 of its L4 progeny that display the recombinant phenotype.
            </p>
            <p>To avert potential problems, it is advisable to stagger the growth of the recombinant animals as much as possible, either
               by picking recombinants over several days or by maintaining some at 20<span class="symbol">&deg;</span>C and others at 16<span class="symbol">&deg;</span>C. The idea is to avoid having just one or two "days from hell" when all the recombinants must be screened at once. Based
               on the position of the SNPs relative to the markers, we will test first one and then the other SNP for all of our recombinants.
               For example if one is testing <span class="bold"><strong>Dpy non-Unc</strong></span> recombinants and the two innermost SNPs are located closer to the <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutation, it will be most efficient to first test the proximal SNP (relative to <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span>) to determine if it is N2, before bothering to test the distal SNP to see if it is CB4856. Non-informative recombinants are
               immediately pitched, whereas the progeny from the informative recombinants are further cloned to generate homozygous lines.
            </p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e6" href="#d0e6">*</a></sup>Edited by Victor Ambros. Last revised May 4, 2005. Published February 17, 2006. This chapter should be cited as: Fay, D. Genetic
                  mapping and manipulation: Chapter 5-SNPs: Three-point mapping (February 17, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.94.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 David Fay. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e33" href="#d0e33">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:davidfay@uwyo.edu" target="_top">davidfay@uwyo.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

