<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Transformation and microinjection</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMTransform_ArticleID" title="Transformation and microinjection&#xA;       ">
      <link rel="next" href="#d0e45" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
     <div class="html_to_pdf_link">
       <a href="transformationmicroinjection.pdf">Download PDF version</a>
    </div>

      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMTransform_ArticleID"></a>Transformation and microinjection<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                            
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Thomas C. Evans, ed.<em><span class="remark">
                                                      <sup><a name="d0e33" href="#ftn.d0e33">&sect;</a></sup>
                                                    </span></em>
                                          </span><br></span><span class="affiliation"><span class="address">Department of Cell and Developmental Biology, University of Colorado School of Medicine, Aurora, CO 80045 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=324.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Transformation+and+microinjection" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e45">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#d0e102">2. General considerations for DNA transformation</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e105">2.1. Scoreable/selectable marker genes</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e155">2.2. Transgene regulatory elements</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e388">3. Microinjection</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e426">3.1. Protocol 1. Microinjection</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e635">4. DNA transformation: Creating repetitive extrachromosomal arrays by microinjection</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e638">4.1. Introduction</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e686">4.2. Protocol 2. Formation of repetitive arrays by microinjection</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e725">5. DNA transformation: Creating complex arrays by microinjection</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e728">5.1. Introduction</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e759">5.2. Protocol 3. Formation of complex arrays by microinjection</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e795">6. Integration of extrachromosomal arrays</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e798">6.1. Introduction and considerations</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e810">6.2. Protocol 4. Integration of arrays by gamma irradiation</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e841">6.3. Protocol 5. Integration of arrays by UV irradiation</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e863">6.4. Protocol 6. Integration by co-injection with single-stranded oligos</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e878">7. DNA transformation by gene bombardment</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e881">7.1. Introduction</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e957">7.2. Protocol 7. Gene bombardment using rich plates</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e1240">8. Future prospects: Targeted gene replacement</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1259">9. Acknowledgments</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1267">10. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e45"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>DNA transformation and microinjection are essential tools for <span class="emphasis"><em>C. elegans</em></span> research. Transformation is used to clone genes by mutant rescue, to over-express or ectopically express genes, to express
               tagged proteins, to study structure/function of protein domains, and to analyze DNA or RNA regulatory elements. Furthermore,
               transgenes can be powerful tools for the design of new genetic screens. Current transformation techniques generate large extrachromosomal
               DNA arrays, or cause "random" integration of transgenes into the genome. However, homologous recombination into endogenous
               loci can occur at low frequency suggesting that targeted gene replacement is possible (<a href="#bib4">Broverman et al., 1993</a>; <a href="#bib1">Berezikov et al., 2004</a>).
            </p>
            <p>Microinjection is a proven and relatively simple method for introducing DNA into worms (<a href="#bib21">Mello et al., 1991</a>; <a href="#bib20">Mello and Fire, 1995</a>). Moreover, microinjection is a very effective approach to RNA interference (see <a href="http://www.wormbook.org/chapters/www_introreversegenetics/introreversegenetics.html" target="_top">Reverse genetics</a>), and can be used to deliver synthetic mRNAs or other molecules directly to cells (<a href="#bib14">Kimble et al., 1982</a>; <a href="#bib3">Bossinger and Schierenberg, 1992</a>; <a href="#bib6">Evans et al., 1994</a>). For transformation, the relatively new technique of gene bombardment is fast becoming a popular alternative because transgenes
               are often integrated into the genome at low copy number, which offers a number of advantages as described below (<a href="#bib24">Praitis et al., 2001</a>; <a href="#bib1">Berezikov et al., 2004</a>).
            </p>
            <p>This chapter contains protocols for microinjection of <span class="emphasis"><em>C. elegans</em></span>, and for generating transgenic nematode strains by both microinjection and gene bombardment. Protocols for integration of
               extrachromosomal arrays into the genome are also included. This chapter focuses on the methods themselves and on some practical
               considerations but does not include extensive discussions of transgene dynamics, vectors, or theory. For an excellent and
               more thorough analysis of these issues see <a href="#bib20">Mello and  Fire (1995)</a>. In addition, different labs often have distinct preferences and unique modifications of these methods. The protocols below
               generally come from a single lab, although a few alternative modifications are included.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e102"></a>2.&nbsp;General considerations for DNA transformation
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e105"></a>2.1.&nbsp;Scoreable/selectable marker genes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>DNA transformation techniques typically require co-transformation with a scoreable or selectable marker gene. Transformation
                  markers that induce a dominant phenotype allow transformation of any worm strain as long as the host's phenotype does not
                  interfere with the marker-induced phenotype. Other selectable markers rescue lethal or non-lethal mutations and require use
                  of specific mutant strains as transformation hosts. For some mutant rescue experiments, a co-injected marker may not be necessary
                  but is usually advisable as a positive control for transformation. Some commonly used marker genes are listed in <a href="#table1" title="Table 1. Common markers for DNA transformation">Table 1</a>. For microinjection techniques, the popular choice is the pRF4 plasmid (Protocols 2 and 3). This plasmid encodes a mutant
                  collagen (<span class="emphasis"><em>rol-6(<a href="http://www.wormbase.org/db/get?name=su1006;class=Variation" target="_blank">su1006</a></em></span>)) that induces a dominant "roller" phenotype, where animals corkscrew around in circles (<a href="#bib16">Kramer et al., 1990</a>; <a href="#bib21">Mello et al., 1991</a>). This roller phenotype is very easy to spot in a simple dissecting microscope, although there are a few potential disadvantages.
                  The roller phenotype is suppressed in some "dumpy" (<span class="emphasis"><em>dpy</em></span>) and "uncoordinated" (<span class="emphasis"><em>unc</em></span>) mutant backgrounds. In addition, rollers (especially males) have reduced mating efficiency. The twisted cuticle of roller
                  animals can also complicate live analysis of some cell types. Alternatively, one can use an <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a></em></span> anti-sense plasmid, a mutant-rescuing plasmid, or a strongly expressed GFP fusion (<a href="#table1" title="Table 1. Common markers for DNA transformation">Table 1</a>; <a href="#bib8">Fire et al., 1991</a>; <a href="#bib10">Granato et al., 1994</a>; <a href="#bib11">Gu et al., 1998</a>). GFP markers require a dissecting microscope equipped with fluorescent optics. For gene bombardment, an <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> gene rescue system is the method of choice as described in Protocol 7.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e155"></a>2.2.&nbsp;Transgene regulatory elements
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In constructing specific transgenes, non-coding regions must be carefully considered. Obviously important domains include
                  the 5' flanking region with its promoter and transcriptional control elements, the 3' flanking region that might contain additional
                  elements, the 5' UTR with a functional translation start site, and the 3' UTR that provides a cleavage and polyadenylation
                  signal and translational control elements (see <a href="http://www.wormbook.org/chapters/www_transcriptionalregulation/transcriptionalregulation.html" target="_blank">Transcriptional regulation</a>, <a href="http://www.wormbook.org/chapters/www_mechregultranslation/mechregultranslation.html" target="_blank">Mechanism and regulation of translation in <span class="emphasis"><em>C. elegans</em></span></a>). In addition, introns are important because pre-mRNA splicing promotes nuclear export, translation, and stability of mRNAs
                  (<a href="#bib19">Maniatis and Reed, 2002</a>; <a href="#bib22">Nott et al., 2004</a>). Even purely synthetic introns substantially increase expression of reporter transgenes in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib23">Okkema et al., 1993</a>). Furthermore, gene-specific regulatory elements may reside within the first few introns of some genes (see <a href="http://www.wormbook.org/chapters/www_transcriptionalregulation/transcriptionalregulation.html" target="_blank">Transcriptional regulation</a>). Proteins can be expressed from cDNAs in worms, but expression will likely be inefficient unless an intron or two is inserted
                  into the cDNA sequence. Finally, it is important to know if the gene to be studied is trans-spliced to the SL1 or SL2 leader
                  RNAs (see <a href="http://www.wormbook.org/chapters/www_transsplicingoperons/transsplicingoperons.html" target="_top">Trans-splicing and operons</a>). For SL2-spliced genes, which are downstream in multigene operons, promoters may be several kilobases upstream and the primary
                  transcript may encode several genes. Fortunately, most <span class="emphasis"><em>C. elegans</em></span> operons have been identified and can be analyzed using Wormbase (<a href="#bib2">Blumenthal et al., 2002</a>; see <a href="http://www.wormbook.org/chapters/www_transsplicingoperons/transsplicingoperons.html" target="_top">Trans-splicing and operons</a> and <a href="http://www.wormbook.org/chapters/www_webresourcesCelegans/webresourcesCelegans.html" target="_top">Web resources for <span class="emphasis"><em>C. elegans</em></span> studies</a>).
               </p>
               <div class="table"><a name="table1"></a><p class="title"><span class="bold">Table 1. Common markers for DNA transformation</span></p>
                  <table summary="Table 1. Common markers for DNA transformation" border="1">
                     <colgroup>
                        <col align="left">
                        <col align="left">
                        <col align="left">
                        <col align="left">
                        <col align="left">
                     </colgroup>
                     <thead>
                        <tr>
                           <th align="left"><span class="bold"><strong>Marker gene</strong></span></th>
                           <th align="left"><span class="bold"><strong>Plasmid</strong></span></th>
                           <th align="left"><span class="bold"><strong>Marker phenotype</strong></span></th>
                           <th align="left"><span class="bold"><strong>Advantages</strong></span></th>
                           <th align="left"><span class="bold"><strong>Potential issues</strong></span></th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a>(<a href="http://www.wormbase.org/db/get?name=su1006;class=Variation" target="_blank">su1006</a>)</em></span><sup>a</sup></td>
                           <td align="left">pRF4</td>
                           <td align="left">Dominant roller</td>
                           <td align="left">Dominant, easy to detect phenotype; known to be compatible with many co-injected transgenes</td>
                           <td align="left">Suppressed in some mutant backgrounds (<span class="emphasis"><em>unc, dpy</em></span>); reduced mating efficiency; twisted cuticle can interfere with larval lineage analysis
                           </td>
                        </tr>
                        <tr>
                           <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a> antisense</em></span><sup>b</sup></td>
                           <td align="left">pPD10.46</td>
                           <td align="left">Dominant twitcher</td>
                           <td align="left">Similar to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a></em></span></td>
                           <td align="left">Suppressed in some mutant backgrounds; reduced mating in males; not as easy to detect as <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a></em></span></td>
                        </tr>
                        <tr>
                           <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sur-5" target="_blank">sur-5</a>::gfp</em></span><sup>c</sup></td>
                           <td align="left">pTG96_2</td>
                           <td align="left">GFP expression in many somatic cells</td>
                           <td align="left">Dominant visible marker; compatible with most host strains</td>
                           <td align="left">Requires fluorescent dissecting scope; interferes with expression study of other GFP fusions</td>
                        </tr>
                        <tr>
                           <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pha-1" target="_blank">pha-1</a>(wt)</em></span><sup>d</sup></td>
                           <td align="left">pC1,pBX</td>
                           <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pha-1" target="_blank">pha-1</a>(e2123ts)</em></span> rescue
                           </td>
                           <td align="left">Allows selection of transformed worms; likely to work in most mutant backgrounds</td>
                           <td align="left">Requires <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pha-1" target="_blank">pha-1</a>(ts)</em></span> as host strain, which is maintained at 15&deg;C; selection requires growth at 25&deg;C
                           </td>
                        </tr>
                        <tr>
                           <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a>(wt)</em></span><sup>e</sup></td>
                           <td align="left">Various<sup>e</sup></td>
                           <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a>(ed3)</em></span> rescue
                           </td>
                           <td align="left">Allows selection of transformed worms; useful for enrichment of integrants after bombardment</td>
                           <td align="left">Requires <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a>(lf)</em></span> as host strain; selection process takes longer
                           </td>
                        </tr>
                        <tr>
                           <td colspan="5" align="left">References:
                              				
                              <table class="simplelist" border="0" summary="Simple list">
                                 <tr>
                                    <td><sup>a</sup><a href="#bib21">Mello et al., 1991</a>; <a href="#bib15">Kramer, 1996</a></td>
                                 </tr>
                                 <tr>
                                    <td><sup>b</sup><a href="#bib8">Fire et al., 1991</a></td>
                                 </tr>
                                 <tr>
                                    <td><sup>c</sup><a href="#bib11">Gu et al., 1998</a></td>
                                 </tr>
                                 <tr>
                                    <td><sup>d</sup><a href="#bib10">Granato et al., 1994</a></td>
                                 </tr>
                                 <tr>
                                    <td><sup>e</sup><a href="#bib18">Maduro and Pilgrim, 1995</a>; <a href="#bib24">Praitis et al., 2001</a></td>
                                 </tr>
                              </table>
                              
                           </td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e388"></a>3.&nbsp;Microinjection
                     </h2>
                  </div>
               </div>
            </div>
            <p><b>Introduction and considerations.&nbsp;</b>Microinjection is an effective method for creating transgenic animals, for RNAi of selected genes, and for introducing various
               types of molecules directly to cells. For DNA transformation, the easiest approach is to inject DNAs into the distal arm of
               the gonad (<a href="#figure1">Figure 1</a>; <a href="#bib21">Mello et al., 1991</a>; <a href="#bib20">Mello and Fire, 1995</a>). The distal germ line of <span class="emphasis"><em>C. elegans</em></span> contains a central core of cytoplasm that is shared by many germ cell nuclei (see <a href="http://www.wormbook.org/chapters/www_introgermline/introgermline.html" target="_blank">Introduction to the germ line</a>). Therefore, DNAs injected here can be delivered to many progeny. This approach usually leads to the formation of large extrachromosomal
               DNA arrays (<a href="#bib21">Mello et al., 1991</a>). Microinjection directly into oocyte nuclei can induce chromosomal integration of transgenes, but this technique is relatively
               difficult to do (<a href="#bib7">Fire, 1986</a>). For RNAi experiments, most progeny of injected animals can be affected by simply injecting dsRNA into a single gonad or
               intestinal cell because of a very efficient RNA transport system (see <a href="http://www.wormbook.org/chapters/www_Introreversegenetics/Introreversegenetics.html" target="_top">Reverse genetics</a>). While RNAi by feeding is best for high throughput experiments, RNAi by microinjection is more effective for at least some
               genes (see <a href="http://www.wormbook.org/chapters/www_Introreversegenetics/Introreversegenetics.html" target="_top">Reverse genetics</a>). Contrary to common perceptions, nematode microinjection is not hard to learn and does not require the most expensive equipment.
               The microinjection method described below is geared towards generating transgenic worms, but it can be readily adapted for
               injecting a variety of molecules into various locations.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e426"></a>3.1.&nbsp;Protocol 1. Microinjection
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Modified from <a href="#bib21">Mello et al. (1991)</a>; <a href="#bib20">Mello and Fire (1995)</a>.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e439"></a>3.1.1.&nbsp;Equipment
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p><b>Injection table.&nbsp;</b>A heavy table segregated from strong vibrations and air currents is usually sufficient. If vibration is a problem, a heavy
                              metal slab placed on a set of small, partly inflated, tire inner tubes can help.
                           </p>
                        </li>
                        <li>
                           <p><b>Inverted DIC microscope.&nbsp;</b>A Zeiss Axiovert, or equivalent Nikon or Olympus instrument, equipped with standard 5X and 40X Nomarski objectives (non-oil
                              immersion) is sufficient. Top of the line optics are not necessary. The microscope should have a flat, free-sliding glide
                              stage with centered rotation.
                           </p>
                        </li>
                        <li>
                           <p><b>Micromanipulator.&nbsp;</b>Narashige, Leitz, and Zeiss make sufficient manipulators that can hold and position the needle holder. The ideal manipulator
                              will have fine mobility in the X (forward and back), Y (side to side), and Z (vertical) axes, and allow easy alteration of
                              needle angle and position.
                           </p>
                        </li>
                        <li>
                           <p><b>Pressurized injection system with needle holder.&nbsp;</b>The injection needle is placed in a holder with a tight-seal collar, which is then attached by plastic tubing to a regulated
                              pressure source. The pressure regulator is attached to a nitrogen gas tank. A Narishige or Eppendorf microinjection controller
                              equipped with a foot pedal switch is a good but somewhat expensive choice. Tritech (Los Angeles, CA) makes a less expensive
                              system. <a href="#bib21">Mello et al. (1991)</a> describe a homemade system.
                           </p>
                        </li>
                        <li>
                           <p><b>Needle puller.&nbsp;</b>Sutter instruments P-87 and P-97 microelectrode pullers are excellent for making very consistent needles of specific shapes
                              and sizes, although a variety of other microelectrode pullers can be used. Several labs can easily share a single puller.
                           </p>
                        </li>
                        <li>
                           <p><b>A standard dissecting microscope.&nbsp;</b></p>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e481"></a>3.1.2.&nbsp;Materials
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p><b>Microinjection needles.&nbsp;</b>Pull needles using borosilicate glass capillaries (1.0mm OD, 0.75 mm ID) with a fine internal glass filament, which allows
                              backfilling by capillary action (e.g., FLG10 from Dagan Corp., Minneapolis MN; or equivalent from World Precision Inst., Sarasota
                              FL or Clark Inst., Reading, England). Good needles taper quickly (~5-7 mm) to a sharp but open point (<span class="symbol">&#8805;</span>1 um), although each user may develop unique preferences. For needle pullers that produce a closed tip, break open the tip
                              by gently tapping the needle across a glass slide, or by moving the needle against debris on the agarose pad under the microscope
                              (see below).
                           </p>
                        </li>
                        <li>
                           <p><b>Injection pads.&nbsp;</b>Bring 2% agarose in water to a boil, mix well, and place in a heat block. Using a broken Pasteur pipette or a cut-off P200
                              tip, place a drop (~100ul) of hot agarose onto a #1, 50X22-mm glass coverslip. Quickly place a second coverslip on the drop
                              and lightly tap it. Alternatively, place several drops on the first coverslip, which should merge and mostly cover the surface
                              after adding the second coverslip; such pads can be used for several rounds of injection (<a href="#bib20">Mello and Fire, 1995</a>). After the agarose is solidified, peel or slide coverslips apart, and bake the coverslip-pad in a vacuum oven at 80&deg;C for
                              1h to overnight. Alternatively, injection pads can be air dried overnight, but worms may not stick well in humid climates.
                              Bake or even re-bake the pads before use if worm adherence is chronically poor. Higher agarose concentrations (2.5-3%) can
                              increase worm adhesion but will also increase their rate of drying out (see below).
                           </p>
                        </li>
                        <li>
                           <p><b>Injection oil.&nbsp;</b>Series 700 Halocarbon oil (Halocarbon Products, River Edge, NJ). Alternatively, Heavy Paraffin Oil can be used (BDH Chemicals,
                              Poole, England; Gallard-Schlesinger, Carle Place, NY).
                           </p>
                        </li>
                        <li>
                           <p><b>Worm pick.&nbsp;</b>A worm pick with a very flat, narrow, arrow-shaped head works best.
                           </p>
                        </li>
                        <li>
                           <p><b>Recovery buffer and M9 buffer.&nbsp;</b>Recovery buffer: 5mm HEPES pH 7.2, 3 mM CaCl<sub>2</sub>, 3 mM MgCl<sub>2</sub>, 66 mM NaCl, 2.4 mM KCl, 4% Glucose (w/v). Standard M9 Buffer.
                           </p>
                        </li>
                        <li>
                           <p><b>Worms.&nbsp;</b>Well-fed, young to middle-aged (<span class="symbol">&#8805;</span>1day old) gravid hermaphrodites with a full but single row of eggs are best.
                           </p>
                        </li>
                        <li>
                           <p><b>Needle-loading pipettes.&nbsp;</b>Place a 10ul standard glass capillary pipette over a flame until it softens, then remove from flame and quickly pull out and
                              break to generate two drawn out pipettes. Use mouthpiece and tubing for ejecting solution. Alternatively, a p2 pipettor can
                              be used to load needles.
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e543"></a>3.1.3.&nbsp;Microinjection method
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p>Fill a needle-loading pipette by capillary action with <span class="symbol">&#8805;</span> 1 ul of DNA injection mix.
                           </p>
                        </li>
                        <li>
                           <p>Insert the pipette tip in through the back of the injection needle, and expel injection mix onto the needle's internal filament.
                              Watch to see that the injection solution is drawn into the needle tip. Several injection needles can be prepared and stored
                              by resting them across raised clay or wax ridges in a humid box with lid.
                           </p>
                        </li>
                        <li>
                           <p>Place a loaded needle into the needle holder and mount on the manipulator.</p>
                        </li>
                        <li>
                           <p>Position the needle so that the tip is in the center of the microscope's field of view using the 5X objective. Once positioned,
                              move the needle up (using the Z-axis control) so that it is slightly out of focus.
                           </p>
                        </li>
                        <li>
                           <p>Place a drop of oil on an injection pad and place under a dissecting microscope on top of a small Petri plate cover.</p>
                        </li>
                        <li>
                           <p>Scoop one to several worms from a bacteria-free region of an NGM plate with a naked pick and transfer to the oil drop. Avoid
                              contact with the worm's head. Alternatively, first touch the worm pick to the oil, and use the oil droplet to pick up the
                              animals from a bacteria-free region. The idea is to minimize transfer of bacteria to the pad.
                           </p>
                        </li>
                        <li>
                           <p>Flame then cool the worm pick and use it to position the worms in the oil drop, and to gently push them down onto the pad.
                              Orient the worms in rows with their ventral sides facing the same direction (opposite the needle direction). If the worms
                              fail to adhere to the pad, move to a new location or rub the bodies with the pick to remove water or bacteria droplets. If
                              adherence is still a problem re-bake the pads or use thicker or higher concentration agarose pads (see above).
                           </p>
                           <div class="sidebar">
                              <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                                 <h4 class="title">Notes</h4>
                                 <p>a. This step requires practice so that injections can be done relatively quickly. Worms stick to the pads due to the absorption
                                    of water from the worm into the dry agarose. If this goes on too long the worms will desiccate and die. Beginners should inject
                                    only one or two worms at a time. It may also be helpful to establish needle flow before mounting worms on the pad (see step
                                    9 below).
                                 </p>
                                 <p>b. For Paraffin oil, transfer worms one at a time. With the worm on the pick, place one part of the worm in contact with the
                                    pad and then gently remove the pick to allow the worm to situate itself completely on the pad.
                                 </p>
                              </div>
                           </div>
                        </li>
                        <li>
                           <p>Transfer the slide face-up onto the microscope stage. Center the first worm to be injected and focus using the 5X objective.
                              Move the needle down and in close proximity to the dorsal surface of the first animal. Switch to the 40X objective and focus
                              on the worm.
                           </p>
                        </li>
                        <li>
                           <p>First make sure the needle is flowing: Move the needle down into focus. Then move the stage a little to move the worm away
                              from the needle. Apply pressure to test if there is significant flow. If not, the needle is probably clogged. In this case,
                              try a pulse of high pressure (but don't exceed the pressure tolerance of the system, which will cause needle ejection and
                              possibly gasket damage). If this fails to work, focus down onto a piece of debris on the pad and move the needle tip gently
                              down onto the pad surface. With the pressure applied, drag the stage away from the needle; often this will remove junk on
                              the tip and induce flow without breaking the tip. Stop flow and move the needle up off the pad if flow resumes. If this doesn't
                              work, break the tip by gently contacting a particle embedded within the pad while under pressure. Rub the particle across
                              the needle tip by moving the stage. If this doesn't work, or if too much of the needle tip breaks off, mount a new needle.
                              Worms can often tolerate broken needles with fairly large openings, but there is a limit that requires experience to determine.
                           </p>
                        </li>
                        <li>
                           <p>Insert the needle into the worm: Move the first worm back to the center of the field ("moving the worm" always means moving
                              the stage). Position the worm so that its dorsal surface lies at a sharp angle to the needle (~15&deg;-45&deg;; <a href="#figure1">Figure 1</a>). Focus the objective on birefringent particles in the distal core cytoplasm of one gonad arm, at its widest point if possible
                              (<a href="#figure1">Figure 1</a>). Using the manipulator, move the needle tip into the same focal plane directly adjacent to the targeted location. Push the
                              worm up against the needle tip until the cuticle distends inward (i.e. is depressed), and then push the worm slightly back
                              against the needle, into the tip, until the needle inserts into the core. The idea is to "pinch" a bit of cuticle and then
                              push against the pinch, like a hypodermic needle into skin, except the stage is used to move the worm into a stationary needle.
                              Alternatively, one can push the worm into the needle at a right angle, and then gently tap on the manipulator to induce insertion
                              (<a href="#bib20">Mello and Fire, 1995</a>). Note: Usually one gonad is easier to see than the other, but with practice both gonads can usually be injected.
                           </p>
                           <div class="literallayout">
                              <p></p>
                           </div>
                           <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="WMTransformfig1.jpg"><img src="WMTransformfig1_s.jpg" border="2" align="middle" alt="WormBook Worm Methods Transformation and Microinjection Figure 1"></a>
  </td>
</tr>
                              </table>
                              <div class="caption">
                                 <p><b>Figure 1. Microinjection of the <span class="emphasis"><em>C. elegans</em></span> gonad.&nbsp;</b>The optimal position of the injection needle in the cytoplasmic core of the distal germ line is depicted. For DNA transformation,
                                    injection solution should flow in both directions through both the distal and proximal germ line (arrows). Note: The optimal
                                    hermaphrodite would be several hours older than the one shown here, with a larger gonad syncytium that has a larger, more
                                    obvious cytoplasmic core.
                                 </p>
                              </div>
                           </div>
                        </li>
                        <li>
                           <p>Inject the DNA solution. Using the fine X-axis control, position the tip into center of the cytoplasmic core. Apply pressure
                              so that solution flows freely and smoothly in both directions throughout the gonad until the gonad noticeably swells up (<a href="#figure1">Figure 1</a>; see also Figure 1 of <a href="#bib20">Mello and Fire, 1995</a>). If solution does not flow into the core, a gentle tap on the table can often help. For DNA transformation, the best results
                              seem to occur when the gonad is filled seemingly to the point of damage. Stop the flow at this point and pull the worm off
                              the needle. Test the needle for flow again and then move to the next worm and repeat steps 10 and 11. Unclog or break the
                              needle tip if necessary (step 9).
                           </p>
                        </li>
                        <li>
                           <p>Recover the worms: Return the coverslip to the dissecting scope, and add a drop (~20 ul) of recovery buffer on the worms.
                              Make sure buffer surrounds the worms so that they release off of the pad. Transfer coverslip to a large empty Petri dish,
                              cover, and incubate for 10-60 min. Once the worms begin swimming briskly, M9 can be added; add an approximately equal volume
                              of M9 and wait until worms swim briskly again, then repeat several times until the solution is mostly M9. The idea is to gradually
                              change the osmotic strength over several minutes. Note: Worms that have not dried too much can often be recovered quickly
                              by directly adding M9 (<a href="#bib20">Mello and Fire, 1995</a>), although in our experience survival is highest using recovery buffer. Transfer the worms to seeded plates.
                           </p>
                        </li>
                        <li>
                           <p>Troubleshooting. An excellent troubleshooting guide is presented in <a href="#bib20">Mello and Fire (1995)</a>. A few highlights are: (a) If the worm ruptures or dies after injection, the needle may be too big, there was too much tearing,
                              or the worm became too dried out; try injecting only one gonad per animal with a fresh needle. Another less likely cause is
                              a contaminated DNA mix (e.g., phenol). (b) If the animals fail to stick, the pads may be too thin or need to be baked (see
                              above). Also, try drying the worm plates by placing coverless in a hood for a few hours. (c) If the worms dry out too fast,
                              use thinner pads and/or transfer worms to moist fresh plates before injection. (d) If the injections look good but no transformants
                              arise, the DNA prep may be dirty (a common problem) or the transgene may induce lethality (see below).
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e635"></a>4.&nbsp;DNA transformation: Creating repetitive extrachromosomal arrays by microinjection
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e638"></a>4.1.&nbsp;Introduction
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The easiest approach to make transgenic strains is to co-inject two or more DNAs into the distal gonad syncytium; one or more
                  DNAs carrying the intended transgenes and a plasmid carrying a transformation marker. Transgene DNAs can be plasmids, cosmids,
                  phage, YACs, or PCR products. Injected DNAs undergo homologous recombination with each other quite efficiently, so it is not
                  necessary to physically link them before injection (<a href="#bib21">Mello et al., 1991</a>). For this reason, it works best if the injected DNAs share sequence homology in their backbones, although non-homologous
                  recombination also occurs (<a href="#bib20">Mello and Fire, 1995</a>). It is feasible to inject several DNAs and recover animals carrying all injected molecules. Whether the injected DNA is
                  circular or linear does not seem to influence results.
               </p>
               <p>Transgenic animals produced by injection typically carry large extrachromosomal arrays that contain many copies of the co-injected
                  DNAs. These repetitive arrays are usually unstable to cell division but can become inheritable; a fraction of first generation
                  progeny (F1) that contain the transgenes will transmit the array through many subsequent generations often without changes
                  in heritability or expression (for somatic promoters). These heritable arrays still have varying degrees of mitotic instability
                  and incomplete inheritance, although it is possible to integrate transgenic arrays into chromosomes (see Protocols 4-6).
               </p>
               <p><b>Advantages.&nbsp;</b>This technique is relatively fast and efficient for genes expressed in somatic tissues; an experienced person injecting simple
                  plasmids can generate 3-6 independent transgenic lines in 7-10 days from as few as 15-40 injected gonads. (If the transgene
                  causes a deleterious phenotype or if large genomic fragments are used, the success rate can be lower). All that is needed
                  is a plate of well-fed hermaphrodites and a few microliters of purified plasmids, and multiple constructs can be analyzed
                  in a reasonable time frame. In addition, the mitotic instability of these arrays can actually be used to create genetic mosaics,
                  which can then be analyzed to determine the lineages in which a gene functions (<a href="#bib12">Herman, 1995</a>), see <a href="http://www.wormbook.org/chapters/www_geneticmosaics/geneticmosaics.html" target="_top">Genetic mosaics</a>.
               </p>
               <p><b>Disadvantages.&nbsp;</b>(1) The transgene expression pattern may not mimic the endogenous gene, especially for germline-expressed genes. Transgenes
                  in repetitive arrays are strongly silenced in germ cell nuclei (<a href="#bib13">Kelly et al., 1997</a>). In addition, suppressed or ectopic transgene expression is sometimes observed in somatic tissues (<a href="#bib20">Mello and Fire, 1995</a>). Rescue of a null mutant does not assure proper expression since even suppressed or ectopically expressed genes can give
                  rescue. (2) It is difficult to predict and control the level of expression among different arrays. (3) Transgene expression
                  can be variable among siblings of a single strain. Some variability probably relates to mitotic instability of arrays, but
                  even integrated arrays can show expression variability for unknown reasons (<a href="#bib20">Mello and Fire, 1995</a>). (4) Arrays sometimes induce RNAi-like effects that suppress endogenous gene function (<a href="#bib5">Dernburg et al., 2000</a>). This could be due to DNA rearrangements during array formation that cause production of gene product fragments or antisense
                  RNA (T. Evans and J. Kimble, unpublished). DNA rearrangements can also reposition gene regulatory elements leading to mis-regulated
                  transgenes. Rearrangements may also occur during long term passage of some transgenic strains, since sometimes the properties
                  of transgenic animals changes over time (T. Evans, unpublished). In spite of these pitfalls, many well-behaved transgenic
                  strains have been created by this technique. Integration of arrays can minimize problems associated with transmission instability
                  (Protocols 4-6). Therefore, as long as caution is exercised this remains a very important approach to nematode transformation.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e686"></a>4.2.&nbsp;Protocol 2. Formation of repetitive arrays by microinjection
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Modified from <a href="#bib20">Mello and Fire (1995)</a>.
               </p>
               <div class="procedure">
                  <ol type="1">
                     <li>
                        <p>Purify DNAs for injection: Standard alkaline lysis protocols, including commercial kits, are sufficient for plasmids and cosmids
                           although residual contaminants can prevent transformation. To generate clean DNA preps, do extra washes in the commercial
                           column-based procedures and/or further purify DNA by phenol choloroform extraction, G-50 spin column, chloroform extraction,
                           and ethanol precipitation. Miniprep procedures that use LiCl or CTAB precipitation steps are also effective for transgene
                           DNAs (<a href="#bib9">Fire et al., 1990</a>; <a href="#bib20">Mello and Fire, 1995</a>). Protocols for preparing YACs and phage for injection can be found in <a href="#bib20">Mello and Fire (1995)</a>. DNAs can be stored in standard Tris/EDTA (TE) buffer.
                        </p>
                     </li>
                     <li>
                        <p>Mix transgene DNA (to 1-100 ug/ml final concentration) with pRF4 (or other marker; to 100 ug/ml). Dilute DNAs with sterile
                           water or TE. Use lower transgene concentrations if the gene is toxic or if multiple plasmids/cosmids are used. Strive for
                           a total DNA concentration of 100-200 ug/ml. An empty vector (e.g., Bluescript) can be used to reach this goal if desired.
                        </p>
                     </li>
                     <li>
                        <p>Inject 15-50 gonads for each DNA mix as described in Protocol 1.</p>
                     </li>
                     <li>
                        <p>Transfer each injected worm to a separate, seeded NGM plate. Grow at 20-25&deg;C.</p>
                     </li>
                     <li>
                        <p>When F1 progeny reach L3 to L4 stages, pick rollers to new plates, 2-3 rollers per plate. Continue to pick more rollers over
                           the next day or two if necessary. Make sure to label plates to identify the parent for each plate.
                        </p>
                     </li>
                     <li>
                        <p>Look for plates that produce F2 rollers and clone several F2 rollers from those plates. Typically, ~2-15% of F1 rollers will
                           transmit the array to the F2 generation. Most F2 rollers will generate lines that continue to transmit their array at a consistent
                           frequency, which can be 5-95% (30-60% is common). Keep only one strain from each injected parent, since lines from the same
                           parent may not be independent transformants.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e725"></a>5.&nbsp;DNA transformation: Creating complex arrays by microinjection
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e728"></a>5.1.&nbsp;Introduction
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Germline silencing of repetitive transgene arrays depends at least partly on their repetitive sequences (<a href="#bib13">Kelly et al., 1997</a>). To circumvent this problem, fragmented genomic DNA can be added to injection mixes, which presumably limits formation of
                  tandem repeats that silence chromatin in the germ line (<a href="#bib13">Kelly et al., 1997</a>). Such "complex" arrays incorporate transgene DNA, marker DNA, and genomic fragments without prior ligation, presumably due
                  to non-homologous recombination. For this technique, DNA linearization seems to be important. These complex arrays, like repetitive
                  arrays, are carried as heritable extrachromosomal fragments with varying degrees of germline transmission. For unknown reasons,
                  germline expression from complex arrays can disappear after the first few generations even in strains that retain the array.
                  Maintaining animals at 25&deg;C suppresses transgene inactivation (<a href="#bib25">Reese et al., 2000</a>; <a href="#bib26">Strome et al., 2001</a>). A fraction of transgenic strains can maintain strong germline expression for many generations.
               </p>
               <p><b>Advantages.&nbsp;</b>The primary advantage of this technique is that it allows analysis of germline-expressed transgenes that do not work in repetitive
                  arrays. It is relatively quick, inexpensive, and simple compared to gene bombardment protocols described below.
               </p>
               <p><b>Disadvantages.&nbsp;</b>Germline expression can become silenced after a few generations. In addition, because the number of transgenic animals recovered
                  is low and only a small percentage of these will continue express the transgene, many animals must be injected. Complex arrays
                  can also induce phenotypes in some transgenic strains, which probably depends on the combination of genomic fragments in the
                  array. For similar reasons, various ectopic expression patterns are often detected in these transgenic strains.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e759"></a>5.2.&nbsp;Protocol 3. Formation of complex arrays by microinjection
                        </h3>
                     </div>
                  </div>
               </div>
               <p>(From Geraldine Seydoux and <a href="#bib13">Kelly et al., 1997</a>)
               </p>
               <div class="procedure">
                  <ol type="1">
                     <li>
                        <p>Make a DNA injection mix:
                                               
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td>Transgene plasmid, linearized</td>
                              <td>1ug/ml</td>
                           </tr>
                           <tr>
                              <td>pRF4, cut with EcoRI</td>
                              <td>1ug/ml</td>
                           </tr>
                           <tr>
                              <td>N2 genomic DNA, cut with ScaI</td>
                              <td>60ug/ml</td>
                           </tr>
                        </table>
                        <p>
                                          
                        </p>
                     </li>
                     <li>
                        <p>Inject 30&#8211;60 hermaphrodites (50&#8211;100 gonads).</p>
                     </li>
                     <li>
                        <p>Transfer 1-3 injected animals per plate, and grow them at 25&deg;C.</p>
                     </li>
                     <li>
                        <p>Clone F1 rollers to individual plates and grow at 25&deg;C. Typically, 15&#8211;50 F1 rollers will be obtained, and 10&#8211;30% of these
                           will give F2 rollers. Examine these F2 rollers as germline expression often diminishes in subsequent generations. Keep animals
                           at 25&deg;C to minimize this effect.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e795"></a>6.&nbsp;Integration of extrachromosomal arrays
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e798"></a>6.1.&nbsp;Introduction and considerations
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Extrachromosomal arrays can be integrated into a chromosome to help mitigate their genetic instability and variability. Three
                  protocols for integrating arrays are described below. Two methods use irradiation of existing transgenic strains, which presumably
                  induces chromosomal breaks and ligation of arrays to chromosomes during DNA repair. Because of this, mutations can arise in
                  treated animals, so it is wise to outcross the recovered integrated strains by mating with wild type males (see <a href="http://www.wormbook.org/chapters/www_introandbasics/introandbasics.html" target="_blank">Genetic mapping and manipulation</a>). In most cases, integrated arrays retain their multicopy character, thus problems associated with repetitive sequences will
                  remain (see Protocol 2). A third method involves co-injecting transgene DNAs with a single stranded DNA oligonucleotide (<a href="#bib21">Mello et al., 1991</a>). The oligo apparently stimulates random integration and/or suppresses array formation. The primary advantage of this approach
                  is that an integrated line can be directly derived from injected animals, but transformation efficiency is reduced about 10-fold.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e810"></a>6.2.&nbsp;Protocol 4. Integration of arrays by gamma irradiation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>(From Peg MacMorris)</p>
               <div class="procedure">
                  <ol type="1">
                     <li>
                        <p>Choose a transgenic roller line with <span class="symbol">&#8804;</span>50% transmission frequency (if possible). Note: Some people irradiate 2-3 lines to increase the probability of isolating a
                           well-behaved integrant (<a href="#bib20">Mello and Fire, 1995</a>).
                        </p>
                     </li>
                     <li>
                        <p>For each strain, take 200-300 L4 rollers, or a mixed population with several hundred L4 rollers washed into M9 buffer, and
                           irradiate with 3800 Rad from a 137Cesium source. (Each 137Cs source will have it's own instructions).
                        </p>
                     </li>
                     <li>
                        <p>Transfer worms to seeded plates for several hours of recovery. Plate pools of about 20 irradiated young roller adults on 10
                           large (10cm) seeded plates. Grow at 20-25&deg;C.
                        </p>
                     </li>
                     <li>
                        <p>After the food is exhausted, transfer worms by washing each plate with 5ml M9, and drop ~0.1-0.5ml to each of another set
                           of 10 seeded plates. Grow at 20-25&deg;C. Repeat this step at least one time.
                        </p>
                     </li>
                     <li>
                        <p>Clone 10 rollers from each plate and screen the progeny of these animals for clones with 100% roller progeny. (Note: It is
                           important to screen the plates before the worms have starved).
                        </p>
                     </li>
                     <li>
                        <p>
                                           Outcross the suspected integrated lines by mating to N2 males. The integrated array should segregate like
                           a dominant chromosomal mutation.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e841"></a>6.3.&nbsp;Protocol 5. Integration of arrays by UV irradiation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>(From Ji Ying Sze)</p>
               <div class="procedure">
                  <ol type="1">
                     <li>
                        <p>Pick 30 L4 rollers to each of two plates.</p>
                     </li>
                     <li>
                        <p>Place the plates in a Stratagene UV crosslinker (Stratalinker), with the lids removed. Set the power to 300 and push start.
                           (Note: UV bulb energy can decrease with time according to Stratagene).
                        </p>
                     </li>
                     <li>
                        <p>Transfer worms to seeded plates.</p>
                     </li>
                     <li>
                        <p>Clone 200-300 F1 rollers. Then, screen these plates for those that have high transmission to the F2 progeny. Clone 7-8 F2
                           animals from "high transmitting" F1's, clone a few from "intermediate transmitters", and discard the rest. Keep plates producing
                           100% inheritance, which are likely to contain homozygous integrants.
                        </p>
                     </li>
                     <li>
                        <p>Outcross the suspected integrants.</p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e863"></a>6.4.&nbsp;Protocol 6. Integration by co-injection with single-stranded oligos
                        </h3>
                     </div>
                  </div>
               </div>
               <p>(From <a href="#bib21">Mello et al., 1991</a>)
               </p>
               <p>Injection mix: transgene plasmid + co-transformation marker (50-100 ug/ml total) + 1 mg/ml single stranded DNA oligo. The
                  oligo sequence may not be important, but a 50mer that is known to work has been described (<a href="#bib21">Mello et al., 1991</a>). Inject 100-200 gonads as in Protocol 1. Recover worms and place 2-3 worms on seeded plates. This procedure typically yields
                  5-10 heritable lines, one or two of which are integrants with 1-10 transgene copies.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e878"></a>7.&nbsp;DNA transformation by gene bombardment
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e881"></a>7.1.&nbsp;Introduction
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A significant advance in nematode transgenics was the discovery that microparticle bombardment can induce integrative transformation
                  in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib24">Praitis et al., 2001</a>). With this technique, DNA is bound onto gold particles which are then "shot" into worms using a biolistic bombardment instrument
                  or "gene gun". A significant number of transformants generated this way are integrants, and many of these contain only a few
                  copies of the transgene. Perhaps these low copy integrants arise because some DNA-coated particles enter nuclei, since similar
                  integration events are seen following direct injection of DNA into oocyte nuclei (<a href="#bib7">Fire, 1986</a>; <a href="#bib24">Praitis et al., 2001</a>). Particle bombardment will also generate extrachromosomal arrays, but using the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> gene as a selectable marker helps to enrich for integrants (<a href="#bib24">Praitis et al., 2001</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a> (ed3)</em></span> mutant is viable and uncoordinated, but cannot survive dauer formation (<a href="#bib18">Maduro and Pilgrim, 1995</a>). Therefore, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a> (ed3)</em></span> animals transformed with a wild type <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> gene will survive starvation while non-transformed animals and those that have lost unstable arrays will not survive. Many
                  rescued <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> animals grown this way carry transgenes integrated at various non-homologous sites in the genome. It is not known if there
                  are integration hot spots. For this technique, the transgene must be subcloned into the same plasmid with the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> selectable marker.
               </p>
               <p><b>Advantages.&nbsp;</b>Stable integrated transgenic strains can be isolated directly. Moreover, many integrated transgenes do not undergo germline
                  silencing and germline expression can remain stable (<a href="#bib24">Praitis et al., 2001</a>). In addition, low copy integrants are likely to avoid some of the other pitfalls of the repetitive and sometimes jumbled
                  sequences in arrays (see Protocol 2 above). Transgenes integrated at low copy give the researcher more control over gene dosage
                  and strain construction in various genetic backgrounds. Expression variability is also less of a problem with these low copy
                  integrants (<a href="#bib24">Praitis et al., 2001</a>). Finally, scaled-up bombardment allows isolation of strains with homologous gene replacements (<a href="#bib1">Berezikov et al., 2004</a>).
               </p>
               <p><b>Disadvantages.&nbsp;</b>The preparation of nematodes and materials for gene bombardment is significantly more labor and material intensive than for
                  microinjection, and it takes longer to generate transgenic strains. Gene bombardment is relatively expensive, as it requires
                  access to a gene gun (~$17,000) and a steady supply of pricey materials. Nonetheless, the potential payoffs will make these
                  investments worthwhile for some researchers.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e957"></a>7.2.&nbsp;Protocol 7. Gene bombardment using rich plates
                        </h3>
                     </div>
                  </div>
               </div>
               <p>(From C. DeRenzo, D. Ghosh, A. Cuenca, M. Stitzel, and G. Seydoux; adapted from Shai Shaham, personal comm., and <a href="#bib24">Praitis et al., 2001</a>)
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e965"></a>7.2.1.&nbsp;Equipment, materials, and reagents
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>
                                         
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td><span class="underline"><u>Equipment/material</u></span></td>
                        <td><span class="underline"><u>Company</u></span></td>
                        <td><span class="underline"><u>Cat. #</u></span></td>
                     </tr>
                     <tr>
                        <td>Biolistic PDS-1000/He particle delivery system</td>
                        <td>Bio-rad</td>
                        <td>1652257</td>
                     </tr>
                     <tr>
                        <td>Hepta Adapter, biolistic</td>
                        <td>Bio-rad</td>
                        <td>1652225</td>
                     </tr>
                     <tr>
                        <td>Biolistic Macrocarriers</td>
                        <td>Bio-rad</td>
                        <td>1652335  (500 ct)</td>
                     </tr>
                     <tr>
                        <td>1&#956;M gold beads</td>
                        <td>Bio-rad</td>
                        <td>1652263(0.25gm)</td>
                     </tr>
                     <tr>
                        <td>Rupture Discs</td>
                        <td>Bio-rad</td>
                        <td>1652333 (2000psi)</td>
                     </tr>
                     <tr>
                        <td>Hepta Stop Screen</td>
                        <td>Bio-rad</td>
                        <td>1652226 (50ct)</td>
                     </tr>
                     <tr>
                        <td>Spermidine(tissue culture grade)</td>
                        <td>Sigma</td>
                        <td>S-4139  (5.0 gm)</td>
                     </tr>
                     <tr>
                        <td>Nystatin</td>
                        <td>Sigma</td>
                        <td>N1638 (100mls)</td>
                     </tr>
                  </table>
                  <p>
                                    
                  </p>
                  <p><span class="bold"><strong>Enriched Peptone Plates</strong></span> (1 Liter)
                                         
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td>1.2g sodium chloride</td>
                     </tr>
                     <tr>
                        <td>20g peptone</td>
                     </tr>
                     <tr>
                        <td>25g agar</td>
                     </tr>
                     <tr>
                        <td>water to 1 liter</td>
                     </tr>
                  </table>
                  <div class="procedure">
                     <ul>
                        <li>
                           <p>Autoclave, then cool to 55&deg;C and add sterile:
                                                      
                           </p>
                           <table class="simplelist" border="0" summary="Simple list">
                              <tr>
                                 <td>1ml cholesterol (5mg/ml in EtOH)</td>
                              </tr>
                              <tr>
                                 <td>1 ml 1M MgSO<sub>4</sub></td>
                              </tr>
                              <tr>
                                 <td>25 ml 1M potassium phosphate (pH 6.0)</td>
                              </tr>
                              <tr>
                                 <td>water to 1 liter</td>
                              </tr>
                           </table>
                           <p>
                                                     
                           </p>
                        </li>
                     </ul>
                  </div>
                  <p><span class="bold"><strong>Enriched Peptone Plates with nystatin</strong></span> (1 Liter)
                  </p>
                  <div class="procedure">
                     <ul>
                        <li>
                           <p>Use the same recipe as above and add 10mls Nystatin suspension (10,000 units/ml).</p>
                        </li>
                     </ul>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1066"></a>7.2.2.&nbsp;Gene bombardment protocol
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="bold"><strong>Day 1: Master plate</strong></span></p>
                  <p>Seed 50&#8211;100 <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> (<span class="emphasis"><em>ed3</em></span> or <span class="emphasis"><em>ed4</em></span>) worms on an enriched peptone plate with NA22 bacteria spread evenly over the entire plate (use enriched peptone plates with
                     NA22 bacteria throughout this protocol). Either pick worms on several different spots or evenly spread out worms, suspended
                     in M9 buffer. Let the worms grow until they have starved for 2-3 days (~7 days total). The plate now contains numerous worm
                     colonies, which consist mainly of L1s. Always have master plates ready when planning bombardments since <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> worms are slow growing.
                  </p>
                  <p><span class="bold"><strong>Day 7: Amplify worms to 6&#8211;7 plates</strong></span></p>
                  <p>Wash worms off the master plate with M9 buffer. Repeat the wash step until most of the bacteria is gone. Resuspend worms in
                     6-7 ml M9 buffer and spread 1.0 ml onto each of 6-7 plates. Allow worms to grow and starve to L1s (2 days). These plates now
                     contain ~ 2-4 x 10<sup>6</sup> L1s.
                  </p>
                  <p><span class="bold"><strong>Day 9: Amplify worms to 60 plates</strong></span></p>
                  <p>Wash off worms from 6-7 plates with M9 buffer until most of the bacteria is gone. Resuspend worms in 60 ml buffer. Add 1ml
                     worms to each plate; spread worms evenly by holding 5 plates a time and gently rotating the stack. Let the worms grow for
                     2.5-3 days at 20&deg;C or 2 days at 25&deg;C (until they become young adults).
                  </p>
                  <p><span class="bold"><strong>Day 12: DNA preparation</strong></span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Weigh 35-50 mg of 1 &#956;m gold beads (Biorad) into a siliconized 1.5 ml eppendorf tube.</p>
                        </li>
                        <li>
                           <p>Add 1ml 70%EtOH. Vortex 5 minutes. Soak for 15 minutes. Pellet and remove sup.</p>
                        </li>
                        <li>
                           <p>Add 1ml sterile water. Vortex 1 minute. Soak for 1 minute. Pellet and remove sup.</p>
                        </li>
                        <li>
                           <p>Add 1ml sterile water. Vortex 1 minute. Soak for 1 minute. Pellet and remove sup.</p>
                        </li>
                        <li>
                           <p>Add 1ml sterile water. Vortex 1 minute. Soak for 1 minute. Pellet and remove sup.</p>
                        </li>
                        <li>
                           <p>Resuspend in 500 &#956;l sterile 50% glycerol. This bead stock can be used for 2 weeks (or more) and should be stored at 4 &deg;C.</p>
                        </li>
                        <li>
                           <p>Vortex mix for 5 minutes. Transfer 100 &#956;l of bead suspension into 3 eppendorf tubes. Make sure to keep the gold beads in suspension
                              while transferring; vortex the stock suspension at medium speed, stop, and then immediately transfer 100 &#956;l to each tube.
                           </p>
                        </li>
                        <li>
                           <p>For each of the three tubes, add in order while vortexing on medium speed:
                                                  
                           </p>
                           <table class="simplelist" border="0" summary="Simple list">
                              <tr>
                                 <td>10 &#956;l DNA (1 mg/ml)</td>
                              </tr>
                              <tr>
                                 <td>100 &#956;l 2.5 M CaCl<sub>2</sub></td>
                              </tr>
                              <tr>
                                 <td>40&#956;l 0.1 M spermidine (free base, tissue culture grade)</td>
                              </tr>
                           </table>
                           <p>
                                                 
                           </p>
                        </li>
                        <li>
                           <p>Vortex 2 minutes. Soak for 1 minute. Pellet and remove sup.</p>
                        </li>
                        <li>
                           <p>Add 280 &#956;l 70% EtOH. Flick tube to mix. Pellet and remove sup.</p>
                        </li>
                        <li>
                           <p>Add 280 &#956;l 100% EtOH. Flick tube to mix. Pellet and remove sup.</p>
                        </li>
                        <li>
                           <p>Add 96 &#956;l 100% EtOH and resuspend by gently flicking tube. This is your prepped DNA. Do this before preparing worms (below).</p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="bold"><strong>Day 12: Worm preparation</strong></span></p>
                  <p>Wash the 60 plates with M9 buffer into 50ml tubes (~400mls/60plates). Spin at low speed for 1 minute; wash worms until the
                     M9 solution becomes clear and finally transfer worms to a 15ml tube. Spin again at low speed for 1 minute. This should give
                     you a pellet of 2-4 ml packed worms. Remove liquid and resuspend worms to 10ml with M9.
                  </p>
                  <p><span class="bold"><strong>Day 12: Bombardment</strong></span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Using a short pasteur pipette, transfer ~ 1.5 ml of worms to the surface of a dry (1 week post bacterial spreading) enriched
                              peptone plate. Add them drop wise starting at the center and then spiraling around until you reach the edge of the plate.
                              Repeat until all worms are plated (6 plates).
                           </p>
                        </li>
                        <li>
                           <p>Leave the covers off the plates to evaporate the liquid. This should take no more than 15 minutes. If it takes more, the enriched
                              peptone plates are too wet.
                           </p>
                        </li>
                        <li>
                           <p>While the plates dry prepare the gene gun. We use the BioRad Biolistic PDS-1000/He particle delivery system with the Hepta
                              adaptor. This adaptor saves an enormous amount of time and effort. Illustrations and jargon definitions are found in the BioRad
                              manual. Read this manual to be familiar with the procedures described below.
                           </p>
                        </li>
                        <li>
                           <p>Hook the vacuum tubing to a vacuum source port (with plastic adaptor). Open vacuum port. Turn on gene gun. Open helium tank
                              valve (check to make sure that He tank pressure is <span class="symbol">&#8805;</span> 2200psi). Perform a test bombardment by wetting a rupture disk (1500-2000 psi) in isopropanol, placing in retaining cap for
                              hepta adaptor, and tightening onto bombardment chamber.
                           </p>
                        </li>
                        <li>
                           <p>Close door. Pull vacuum to 27 in of Hg. Once it reaches 27 in of Hg press hold. Press Fire button and hold until disk ruptures.
                              Release vacuum (Vent position), open door, unscrew retaining cap and discard the ruptured disk.
                           </p>
                        </li>
                        <li>
                           <p>Place 7 macrocarriers onto the hepta adaptor macrocarrier holder using the special tool. Vortex your DNA preparation at medium
                              speed with cap open. Stop and immediately (quickly) transfer 6 &#956;l of beads onto a macrocarrier and spread it around with your
                              pipette tip. Only spread on area around the hole in the holder. Repeat for all macrocarriers. Let EtOH evaporate (this only
                              takes a few minutes).
                           </p>
                        </li>
                        <li>
                           <p>Place a rupture disk soaked in isopropanol in the retaining cap and tighten. Place hepta stopping screen and macrocarrier
                              holder in chamber as described in manual. Place uncovered worm plate (taped to the sample holder using a rolled piece of adhesive
                              tape to create double sided tape) into lowest rung in bombardment chamber.
                           </p>
                        </li>
                        <li>
                           <p>Evacuate chamber to 26 in Hg. Press Fire button until disk ruptures. Release vacuum (Vent position), and remove plate.</p>
                        </li>
                        <li>
                           <p>Repeat above for all 6 plates.</p>
                        </li>
                        <li>
                           <p>Turn off the vacuum. Close the helium tank valve - make sure no pressure is left in the line. Turn the gene gun power OFF.</p>
                        </li>
                     </ol>
                  </div>
                  <p>**Wipe down all of the gene gun components with 70% ethanol, then autoclave the hepta adaptor and the macro carrier holder
                     components. Wipe down the bombardment chamber with 70% ethanol.
                  </p>
                  <p><span class="bold"><strong>Day 12: Plating worms</strong></span></p>
                  <p>Wash bombarded worms off plates with M9. Place worms in 50ml tubes. Spin 1 minute at low speed. Resuspend worms evenly in
                     2 tubes with 80 ml of M9 media.
                  </p>
                  <p>Plate worms, 1ml per plate, enriched peptone plates with nystatin and seeded with NA22 bacteria. Should end up with ~80 plates.</p>
                  <p>Once plates have dried, place them in a 25&deg; C incubator (we found this to be important for GFP expression of several germline
                     genes, you can grow worms at 20&deg;C for non-GFP transgenes)  and let sit for 10-14 days (preferably in a plastic containers
                     to avoid plates drying out).
                  </p>
                  <p><span class="bold"><strong>Day 22&#8211;26: Screening</strong></span></p>
                  <p>Scan plates for WT animals. Typically, 10-20 plates contain WT animals with anywhere from 1 to many WT animals per plate.</p>
                  <p>From each big plate, clone 3-5 WT worms to individual plates (regular NGM plates with OP50). Place the plates a 25&deg; C for
                     4 days, and then examine the progeny for GFP expression.
                  </p>
                  <div class="sidebar">
                     <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                        <h4 class="title">Notes</h4>
                        <p>Two alternative methods can be used to grow <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> worms for bombardment:
                                                      
                        </p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Worms can be grown on "egg plates" as described (<a href="#bib1">Berezikov et al., 2004</a>).
                                 </p>
                              </li>
                              <li>
                                 <p>Worms can be grown in liquid culture (E. Moss, personal comm.; <a href="#bib17">Lewis and Fleming, 1995</a>). For liquid culture, a 2L baffled flask containing 250ml of complete S medium with 30g/L E. coli is shaken vigorously. Before
                                    bombardment, <span class="emphasis"><em>unc-119(ed3)</em></span> animals are grown at 15&deg;C and fed regularly with E. coli concentrated from a 500ml culture. Gravid worms are isolated by
                                    flotation on 35% sucrose, embryos are isolated by bleach treatment, and synchronized adults are then used for bombardment.
                                    After bombardment, worms are rinsed into four 2L flasks and grown at 25&deg;C. For one week, worms are fed regularly to expand
                                    the populations, and then are allowed to starve in the second week. Worms are then collected by low speed centrifugation and
                                    pipetted onto large plates. Rescued worms are then selected and cloned onto standard plates. (Contributed by Eric Moss).
                                 </p>
                              </li>
                           </ol>
                        </div>
                        <p>
                                                 
                        </p>
                     </div>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1240"></a>8.&nbsp;Future prospects: Targeted gene replacement
                     </h2>
                  </div>
               </div>
            </div>
            <p>Current transformation technology is powerful for the analysis of gene function and regulation in <span class="emphasis"><em>C. elegans</em></span>. However, the holy grail of DNA transformation is targeted gene replacement by homologous recombination. As shown in yeast
               and mice, gene targeting can create null mutations in virtually any selected gene (gene knock-out), and can replace genes
               with designed derivatives (gene knock-in). Homologous recombination of transgenes into endogenous loci was observed in some
               <span class="emphasis"><em>C. elegans</em></span> strains when DNA was injected into oocyte nuclei (<a href="#bib4">Broverman et al., 1993</a>). By scaling up gene bombardment procedures, Berezikov et al. recently isolated a few knock-out homologous integrants, and
               one knock-in replacement (<a href="#bib1">Berezikov et al., 2004</a>). These exciting results suggest that gene targeting may be feasible by microparticle bombardment. A limitation is that only
               ~0.5%-7% of integration events arise from homologous recombination, and it is not clear what parameters might affect these
               rates. Perhaps new transgene designs or isolation of mutants with increased recombination rates will soon lead to routine
               procedures for this powerful technology.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1259"></a>9.&nbsp;Acknowledgments
                     </h2>
                  </div>
               </div>
            </div>
            <p>This chapter would not be possible without the generous contributions of the experts in these methods. Many thanks are due
               to Cyndy DeRenzo, Peg MacMorris, Eric Moss, Geraldine Seydoux, and Ji Ying Sze, for contributing protocols and unpublished
               information, and to David Fay, Andy Fire, Barth Grant, Wade Johnson, Chris Link, Morris Maduro, Craig Mello, and Ding Xue
               for valuable information and advice. Thanks are also due to the entire <span class="emphasis"><em>C. elegans</em></span> community for their collective efforts that make these techniques work. Work in T.C.E's laboratory was supported by grants
               from NSF (9982944, 0345386), U. Colo. Cancer Center and ACS, and NIH (GM20599-03).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1267"></a>10.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Berezikov, E., Bargmann, C.I., and Plasterk, R.H. (2004). Homologous gene targeting in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by biolistic transformation. Nucleic Acids Res. <span class="emphasis"><em>32</em></span>, e40.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14982959&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gnh033" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Blumenthal, T., Evans, D., Link, C.D., Guffanti, A., Lawson, D., Thierry-Mieg, J., Thierry-Mieg, D., Chiu,  W.L., Duke, K.,
                        Kiraly, M., and Kim, S.K. (2002). A global analysis of <span class="emphasis"><em>Caenorhabditis elegans</em></span> operons. Nature <span class="emphasis"><em>417</em></span>, 851&#8211;854.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12075352&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature00831" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bossinger, O., and Schierenberg, E. (1992). Cell-cell communication in the embryo of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>151</em></span>, 401&#8211;409.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1601175&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(92)90180-O" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Broverman, S., MacMorris, M., and Blumenthal, T. (1993). Alteration of <span class="emphasis"><em>Caenorhabditis elegans</em></span> gene expression by targeted transformation. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>90</em></span>, 4359&#8211;4363.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8506273&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Dernburg, A.F., Zalevsky, J., Colaiacovo, M.P., and Villeneuve, A.M. (2000). Transgene-mediated cosuppression in the <span class="emphasis"><em>C. elegans</em></span> germ line. Genes Dev. <span class="emphasis"><em>14</em></span>, 1578&#8211;1583.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10887151&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Evans, T.C., Crittenden, S.L., Kodoyianni, V., and Kimble, J. (1994). Translational control of maternal <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> mRNA establishes an asymmetry in the <span class="emphasis"><em>C. elegans</em></span> embryo. Cell <span class="emphasis"><em>77</em></span>, 83&#8211;194.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8168128&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(94)90311-5" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Fire, A. (1986). Integrative transformation of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. EMBO J. <span class="emphasis"><em>5</em></span>, 2673&#8211;2680.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Fire, A., Albertson, D., Harrison, S.W., and Moerman, D.G. (1991). Production of antisense RNA leads to effective and specific
                        inhibition of gene expression in <span class="emphasis"><em>C. elegans</em></span> muscle. Development <span class="emphasis"><em>113</em></span>, 503&#8211;514.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1782862&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Fire, A., Harrison, S.W., and Dixon, D. (1990). A modular set of lacZ fusion vectors for studying gene expression in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Gene <span class="emphasis"><em>93</em></span>, 189&#8211;198.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2121610&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0378-1119(90)90224-F" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Granato, M., Schnabel, H., and Schnabel, R. (1994). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pha-1" target="_blank">pha-1</a></em></span>, A selectable marker for gene transfer in <span class="emphasis"><em>C. elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>22</em></span>, 1762&#8211;1763.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8202383&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Gu, T., Orita, S., and Han, M. (1998). <span class="emphasis"><em>Caenorhabditis elegans</em></span> <a href="http://www.wormbase.org/db/seq/protein?name=SUR-5;class=protein" target="_blank">SUR-5</a>, a novel but conserved protein, negatively regulates <a href="http://www.wormbase.org/db/seq/protein?name=LET-60;class=protein" target="_blank">LET-60</a> Ras activity during vulval induction. Mol. Cell Biol. <span class="emphasis"><em>18</em></span>, 4556&#8211;4564.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9671465&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Herman, R.K. (1995). Mosaic analysis. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 123&#8211;146.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531723&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Kelly, W.G., Xu, S., Montgomery, M.K., and Fire, A. (1997). Distinct requirements for somatic and germline expression of a
                        generally expressed <span class="emphasis"><em>Caernorhabditis elegans</em></span> gene. Genetics <span class="emphasis"><em>146</em></span>, 227&#8211;238.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9136012&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Kimble, J., Hodgkin, J., Smith, T., and Smith, J. (1982). Suppression of an amber mutation by microinjection of suppressor
                        tRNA in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>299</em></span>, 456&#8211;458.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7121584&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/299456a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Kramer, A. (1996). The structure and function of proteins involved in mammalian pre-mRNA splicing. Annu. Rev. Biochem. <span class="emphasis"><em>65</em></span>, 367&#8211;409.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8811184&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.bi.65.070196.002055" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Kramer, J.M., French, R.P., Park, E.C., and Johnson, J.J. (1990). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a></em></span> gene, which interacts with the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sqt-1" target="_blank">sqt-1</a></em></span> collagen gene to determine organismal morphology, encodes a collagen. Mol. Cell Biol. <span class="emphasis"><em>10</em></span>, 2081&#8211;2089.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1970117&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Lewis, J.A., and Fleming, J.T. (1995). Basic culture methods. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 3&#8211;29.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531730&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Maduro, M., and Pilgrim, D. (1995). Identification and cloning of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span>, a gene expressed in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> nervous system. Genetics <span class="emphasis"><em>141</em></span>, 977&#8211;988.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8582641&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Maniatis, T., and Reed, R. (2002). An extensive network of coupling among gene expression machines. Nature <span class="emphasis"><em>416</em></span>, 499&#8211;506.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11932736&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/416499a" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Mello, C., and Fire, A. (1995). DNA transformation. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 451&#8211;482.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531738&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Mello, C.C., Kramer, J.M., Stinchcomb, D., and Ambros, V. (1991). Efficient gene transfer in <span class="emphasis"><em>C.elegans</em></span>: extrachromosomal maintenance and integration of transforming sequences. EMBO J. <span class="emphasis"><em>10</em></span>, 3959&#8211;3970.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1935914&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Nott, A., Le Hir, H., and Moore, M.J. (2004). Splicing enhances translation in mammalian cells: an additional function of
                        the exon junction complex. Genes Dev. <span class="emphasis"><em>18</em></span>, 210&#8211;222.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14752011&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1163204" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Okkema, P.G., Harrison, S.W., Plunger, V., Aryana, A., and Fire, A. (1993). Sequence requirements for myosin gene expression
                        and regulation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>135</em></span>, 385&#8211;404.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8244003&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Praitis, V., Casey, E., Collar, D., and Austin, J. (2001). Creation of low-copy integrated transgenic lines in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>157</em></span>, 1217&#8211;1226.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11238406&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Reese, K.J., Dunn, M.A., Waddle, J.A., and Seydoux, G. (2000). Asymmetric segregation of <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> in <span class="emphasis"><em>C. elegans</em></span> is mediated by two complementary mechanisms that act through separate <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> protein domains. Mol. Cell <span class="emphasis"><em>6</em></span>, 445&#8211;455.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10983990&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)00043-5" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">	
                     	<span class="bibliomisc">Strome, S., Powers, J., Dunn, M., Reese, K., Malone, C.J., White, J., Seydoux, G., and Saxton, W. (2001). Spindle dynamics
                        and the role of <span class="emphasis"><em><span class="symbol">&#947;</span></em></span>-tubulin in early <span class="emphasis"><em>Caenorhabditis elegans</em></span> embryos. Mol. Biol. Cell <span class="emphasis"><em>12</em></span>, 1751&#8211;1764.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11408582&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised March 04, 2005. Published April 6, 2006. This chapter should be cited as: Evans, T.
                  C., ed. Transformation and microinjection (April 6, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.108.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Thomas C. Evans and contributors. This is an open-access article distributed under the terms of the Creative Commons
                  Attribution License, which permits unrestricted use, distribution, and contributors reproduction in any medium, provided the
                  original author and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e33" href="#d0e33">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:tom.evans@uchsc.edu" target="_top">tom.evans@uchsc.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

