<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic mapping and manipulation: Chapter 10-Suppressor mutations</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMgmsmch10_ArticleID" title="Genetic mapping and manipulation: Chapter 10-Suppressor mutations&#xA;">
      <link rel="next" href="#d0e61" title="1.&nbsp;Basics">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="suppressormutations.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMgmsmch10_ArticleID"></a>Genetic mapping and manipulation: Chapter 10-Suppressor mutations<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">David</span> <span class="surname">Fay<em><span class="remark">
                                 					<sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Department of Molecular Biology, University of Wyoming, Laramie, Wyoming 82071-3944 USA
                           <p></p></span></span></div>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Wade</span> <span class="surname">Johnson<em><span class="remark">
                                 					<sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Howard Hughes Medical Institute, Department of Molecular, Cellular, and Developmental Biology, University of Colorado, Boulder,
                           CO 80309 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward genetics and genetic mapping<br />>>> Genetic mapping and manipulation
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=297.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Suppressor+mutations" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e61">1. Basics</a></span></dt>
               <dt><span class="sect1"><a href="#d0e91">2. Considerations and caveats</a></span></dt>
               <dt><span class="sect1"><a href="#d0e123">3. Suppressor identification strategies</a></span></dt>
               <dt><span class="sect1"><a href="#d0e150">4. Mapping suppressor mutations</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e61"></a>1.&nbsp;Basics
                     </h2>
                  </div>
               </div>
            </div>
            <p>Suppressor and enhancer screens are powerful ways to uncover more information about a known gene or mutation, including the
               identification of other interacting components in a pathway. Very simply, a suppressor screen starts with a known mutation
               and then identifies second-site mutations that suppress the mutant phenotype. Enhancer screens are just the opposite and are
               discussed in the above section along with synthetic mutations. One classic example is the <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></strong></span></em></span> (gf) suppressor screen carried out in recent years by the Han lab. In this screen, <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></strong></span></em></span> worms were mutagenized by <a href="http://www.wormbase.org/db/get?name=EMS;class=Cell" target="_blank">EMS</a> and scored for suppression of the <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></strong></span></em></span> multivulval (<span class="bold"><strong>Muv</strong></span>) mutant phenotype. Suppression of this phenotype leads to either a normal vulva or loss of the vulva, vulvalless (<span class="bold"><strong>Vul</strong></span>). From this screen a variety of genes were isolated and characterized. Many of these genes (or at least the alleles isolated)
               possess silent phenotypes on their own and hence would only have been isolated by such a screen. This is one of the greatest
               strengths of a suppressor screen. Also, all of the genes so far characterized from this screen have led to a greater understanding
               of the ras pathway not only in worms but in other organisms as well.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e91"></a>2.&nbsp;Considerations and caveats
                     </h2>
                  </div>
               </div>
            </div>
            <p>Before starting a suppressor screen, you obviously must have a mutation to suppress. The better characterized the mutation,
               the better able you will be to design an effective screen, as the characteristics of your mutation will affect the types of
               suppressors you can hope to isolate. As a general rule, it is almost always easier to isolate suppressors of weak or partial
               LOF mutants than for strong LOF or null mutants. The reason is that some suppressors are likely to be mutations in negative
               regulators of your gene, possibly at the level of transcription, protein stability, or protein activity. In the case of null
               mutations where your protein has either no activity or isn't being made, such suppressors will be ineffective, as they have
               nothing to work with. For example, if your mutation is a partial loss of function with 20% normal activity, a suppressor that
               increases the abundance or activity of your protein by five-fold will theoretically bring you back to normal levels. The same
               cannot be said if your protein has 0% activity.
            </p>
            <p>The suppressors you obtain will potentially fall into two classes: informational suppressors and functional suppressors. The
               former class includes gene products that will suppress your mutation through a generic mechanism, such as the suppression
               of a stop codon. Other informational suppressors include mRNA degradation mutants (<span class="emphasis"><em><span class="bold"><strong>smg</strong></span></em></span> genes) and protein-degradation mutants (ubiquitination enzymes). For the most part, this class of suppressors is less interesting
               but often cannot be avoided. Knowing the molecular lesion within your mutant of interest will allow you to determine what
               types of informational suppressors you might expect to uncover within your screen. The more relevant class of functional suppressors
               acts through mechanisms that will hopefully shed light on the process or gene of interest.
            </p>
            <p>The question then becomes how can one determine which suppressor mutations are informational and which are functional? By
               setting up a series of basic experiments designed to test the characteristics of the individual suppressor mutations, one
               can easily avoid wasting inordinate amounts of time on suppressors that are non-specific. A variety of experiments could be
               designed to answer this question; a few are listed below. These are not necessarily the only experiments that could be done,
               but they are a good place to start. Each mutant being suppressed will lead to unique experiments that can be carried out.
            </p>
            <p><span class="bold"><strong>First</strong></span>: Cross suppressors to multiple mutant alleles of the gene of interest and assay for suppression. In addition, if RNAi produces
               a phenotype for your mutation, this may provide a complementary approach. Although this may provide a good test for overall
               suppression, keep in mind that suppressors that are not effective on other alleles may still be interesting (i.e., functional)
               because of their allele specificity. This test must be measured against the other tests to determine the validity of the suppressors.
            </p>
            <p><span class="bold"><strong>Second</strong></span>: Use RNAi to disrupt any residual message from the mutant of interest in the mutant:suppressor background. If the suppressors
               are functional and not informational, you may (<span class="emphasis"><em>not</em></span>) see a reversion back to the 100% mutant phenotype. However, the same stipulation stated above for alternative alleles also
               holds true for RNAi. Moreover, certain informational suppressors may act by affecting mRNA degradation, and thus could compromise
               the RNAi pathway. This can potentially complicate interpretation of tests using RNAi.
            </p>
            <p><span class="bold"><strong>Third</strong></span>: In the case of a mutation of interest containing a premature stop codon, cross the suppressor into a strain containing another
               mutant gene with the same premature stop codon (e.g., an opal stop in both cases). Suppression of this unrelated mutated gene
               suggests you have isolated an informational suppressor.
            </p>
            <p><span class="bold"><strong>Fourth</strong></span>: If the gene of interest is within a known biochemical pathway, test other members of this pathway, either by using previously
               identified mutations or RNAi. These experiments should reveal the extent of suppression as being either specific to your gene
               of interest or encompassing the entire pathway.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e123"></a>3.&nbsp;Suppressor identification strategies
                     </h2>
                  </div>
               </div>
            </div>
            <p>Your specific suppressor screen will obviously depend on many factors, such as whether or not the mutation is viable or easy
               to score by its plate phenotype. In the case of embryonic or larval lethal phenotypes, it is useful first to link your mutation
               (<span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>) to a nearby and easily recognizable morphological marker (such as <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span>). The <span class="emphasis"><em><span class="bold"><strong>dpy m</strong></span></em></span> chromosome is then preferably balanced by a regional deficiency or chromosomal rearrangement that discourages recombination
               or by a set of closely spaced morphological markers. The idea is that you want a well-balanced strain where you never see
               viable <span class="bold"><strong>Dpy</strong></span> animals (or can easily tell if a rare recombination event has taken place). The strain is then mutagenized, and plates are
               screened in subsequent generations for the presence of viable <span class="bold"><strong>Dpy</strong></span> animals.
            </p>
            <p>An alternative approach (if your mutation is cloned) would be to rescue your mutant using an extrachromosomal array that contains
               a wild-type copy of the gene as well a marker, such as GFP. One could then mutagenize this resuced strain and look for non-green
               (array-minus) animals that don't show the phenotype. This may be particularly useful for lethal or other easily scorable mutations.
               As an added option, you could use a worm sorter to do much of the work for you (separating any green from non-green animals),
               making this a very powerful approach.
            </p>
            <p>A more modern method for finding suppressors may be through the use of RNAi feeding. Namely, plowing through the "complete"
               genome library and looking for revertants. This approach has the major added advantage that the molecular identity of the
               suppressing clones are immediately known, in contrast to the time consuming procedure of cloning mutations. However, it is
               important to recall that RNAi feeding is not a terribly effective means for inactivating genes, and you may miss a large number
               of potential suppressors. Also, it may be prudent to first attempt a reasonably sized genetic suppressor screen first, to
               determine whether your mutation or allele is likely to be suppressable. If you seem to be getting reasonable numbers of suppressors,
               then investing in an RNAi screen may be more attractive.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e150"></a>4.&nbsp;Mapping suppressor mutations
                     </h2>
                  </div>
               </div>
            </div>
            <p>Another key consideration in suppressor mapping is the creation of mapping strains. Depending on the suppressors isolated,
               many suppressors will be silent (i.e., they will have no observable phenotype on their own). Therefore, to map these silent
               suppressors the original suppressed mutation must be included in all your mapping strains. Another consideration is mating
               difficulties; some mutations may affect male fertility in the homozygous state. If this is the case, then it is always best
               to mate males into the mapping strain rather than into the suppressed strain to obtain heterozygote fertile males. The reason
               for this is to reduce the risk of losing the suppressor. <a href="#figure1">Figure 1</a> shows a typical mating between a specific suppressor and a mapping strain.
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="540">
<tr>
  <td align="center"><p><a href="gmsmch10-1.jpg"><img src="gmsmch10-1_s.jpg" border="2" align="middle" alt=" Suppressor mutations figure 10-1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1.&nbsp;</b></p>
               </div>
            </div>
            <p>The worms isolated in step 3 are then scored for the <span class="bold"><strong>Dpy</strong></span> and <span class="bold"><strong>Unc</strong></span> phenotypes in their progeny. If the suppressor is on the same chromosome as the markers and is very close, then ~100% of
               the worms will <span class="emphasis"><em>not</em></span> throw the markers (i.e., their progeny will not express these phenotypes). If the suppressor is not on the same chromosome,
               then two-thirds of the worms should throw the markers (see <a href="http://www.wormbook.org/chapters/www_twopointmapmarkers.2/twopointmapmarkers.html" target="_blank">Two-point mapping with genetic markers</a> for a better explanation). These numbers, of course, are to be expected if the world were perfect, which it is not. Recombination
               makes this a little more difficult and a little more telling.
            </p>
            <p>Recombination at this step may also supply vital information for suppressors that are on the same chromosome as the markers.
               Because of recombination, a few of the progeny from step 3 may be either <span class="bold"><strong>Dpy</strong></span> or <span class="bold"><strong>Unc</strong></span>. The frequency with which these recombination events arise allows you to map the suppressor to a specific chromosome and
               provides information about the suppressor's direction and distance from the known markers. Three-point mapping is then carried
               out with other markers that should bookend the mutation. For a further and more thorough explanation, please see the <a href="http://www.wormbook.org/chapters/www_twopointmapmarkers/twopointmapmarkers.html" target="_blank">Two-point mapping with genetic markers</a> and <a href="http://www.wormbook.org/chapters/www_threepointmapmarkers/threepointmapmarkers.html" target="_blank">Three-point mapping with genetic markers</a> sections.
            </p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised May 6, 2005. Published February 17, 2006. This chapter should be cited as: Fay, D. and
                  Johnson, W. Genetic mapping and manipulation: Chapter 10-Suppressor mutations (February 17, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.99.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 David Fay and Wade Johnson. This is an open-access article distributed under the terms of the Creative Commons Attribution
                  License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source
                  are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:davidfay@uwyo.edu" target="_top">davidfay@uwyo.edu</a> or <a href="mailto:wade.johnson@colorado.edu" target="_top">wade.johnson@colorado.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

