<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Pre-mRNA splicing and its regulation in Caenorhabditis elegans
         
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#Wormbook_splicing_Zahler_id" title="Pre-mRNA splicing and its regulation in Caenorhabditis elegans&#xA;">
      <link rel="next" href="#sec1" title="1.&nbsp;The importance of alternative splicing">
      <script language="javascript" type="text/javascript" src="/js/history_list.js"></script>
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
   <div class="html_to_pdf_link"> 
       <a href="pre-mRNA.pdf">Download PDF version</a>
   </div>

      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="Wormbook_splicing_Zahler_id"></a>Pre-mRNA splicing and its regulation in <span class="emphasis"><em>Caenorhabditis elegans</em></span><sup><a name="d0e9" href="#ftn.d0e9">*</a></sup>
                     
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Alan M. Zahler<em><span class="remark"><sup><a name="d0e34" href="#ftn.d0e34">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="orgname">Department of Molecular, Cellular and Developmental Biology, Center for Molecular Biology of RNA, Sinsheimer Laboratories,
                           University of California, Santa Cruz, CA 95064 USA; </span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_molecularbiology.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Molecular biology
		</a></div><hr />

<a href="http://www.wormbase.org/forums/index.php?board=204.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Alternative+splicing+in+C.+elegans" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

          <p><a style="color:blue;cursor:pointer" onclick="return kadabra('history');" title="click to expand ..." >Publication History</a></p>
           <ul id="history" style="display:none">
            <li><a href="/chapters/www_altsplicing/altsplicing.html">version 1</a></li>
            <li><a href="/chapters/www_altsplicing.2/pre-mRNA.html">latest version</a></li>
           </ul>

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. The importance of alternative splicing</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. Splicing of messenger RNA precursors</a></span></dt>
               <dt><span class="sect1"><a href="#sec3">3. Constitutive splicing factors in <span class="bolditalic">C. elegans</span></a></span></dt>
               <dt><span class="sect1"><a href="#sec4">4. Alternative pre-mRNA splicing</a></span></dt>
               <dt><span class="sect1"><a href="#sec5">5. Identification of alternatively spliced genes in <span class="bolditalic">C. elegans</span></a></span></dt>
               <dt><span class="sect1"><a href="#sec6">6. Regulation of alternative splicing</a></span></dt>
               <dt><span class="sect1"><a href="#sec7">7. Identifying the elements of the <span class="bolditalic">C. elegans</span> splicing code</a></span></dt>
               <dt><span class="sect1"><a href="#sec8">8. Genetic identification of alternative splicing factors in <span class="emphasis"><em>C. elegans</em></span></a></span></dt>
               <dt><span class="sect1"><a href="#sec9">9. Coupling of alternative splicing to nonsense-mediated decay (AS-NMD)</a></span></dt>
               <dt><span class="sect1"><a href="#sec10">10. Conclusions</a></span></dt>
               <dt><span class="sect1"><a href="#sec11">11. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#sec12">12. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p>Alternative splicing is a common mechanism for the generation of multiple isoforms of proteins. It can function to expand
               the proteome of an organism and can serve as a way to turn off gene expression after transcription. This review focuses on
               splicing, its regulation and the progress in this field achieved through studies in <span class="emphasis"><em>C. elegans</em></span>. Recent experiments, including RNA-Seq to uncover and measure the extent of alternative splicing, comparative genomics to
               identify splicing regulatory elements, and the development of elegant genetic screens using fluorescent reporter constructs,
               have increased our understanding of the <span class="emphasis"><em>cis</em></span>-acting sequences that regulate alternative splicing and the <span class="emphasis"><em>trans</em></span>-acting protein factors that bind to these sequences. The topics covered in this review include constitutive splicing factors,
               identification of alternatively spliced genes<span class="emphasis"><em></em></span>, alternative splicing regulation and the coupling of alternative splicing to nonsense-mediated decay. The significant progress
               towards uncovering the alternative splicing code in this organism is discussed.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;The importance of alternative splicing
                     </h2>
                  </div>
               </div>
            </div>
            <p>In multicellular eukaryotes, many genes show evidence for alternative splicing; introns are removed and exons are spliced
               together in different combinations to yield different multiple distinct messenger RNAs from a single gene. In humans, recent
               studies suggest that ~95% of genes are capable of undergoing alternative splicing (<a href="#bib89">Pan et al., 2008</a>; <a href="#bib126">Wang et al., 2008</a>). Alternative splicing can be highly regulated under developmental-, tissue-, and signal-specific controls leading to the
               regulated generation of multiple protein-coding mRNAs from a single gene (<a href="#bib18">Calarco et al., 2011</a>; <a href="#bib31">Hartmann and Valcarcel, 2009</a>; <a href="#bib61">Licatalosi and Darnell, 2010</a>). This extensive use of alternative splicing can lead to a more highly diverse proteome (<a href="#bib86">Nilsen and Graveley, 2010</a>). In addition, alternative splicing can lead to a downstream change in open reading frame in the mRNA and thus produce a
               premature termination codon (PTC) in one of the isoforms. mRNAs containing PTCs are degraded by the nonsense-mediated decay
               (NMD) machinery. This coupling of alternative splicing and nonsense-mediated decay (AS-NMD) is a post-transcriptional mechanism
               for shutting off gene expression that occurs for roughly 1/3 of human alternative splicing events (<a href="#bib58">Lewis et al., 2003</a>). Alternative splicing occurs for ~25% of genes in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib95">Ramani et al., 2011</a>). <span class="emphasis"><em>C. elegans</em></span> has proven to be a useful model system for identifying targets for alternative splicing, the factors that regulate the process
               and improving our understanding of the mechanisms of this regulation. Like vertebrates, <span class="emphasis"><em>C. elegans</em></span> is an intron-rich organism. Using the Table Browser feature of the UCSC Genome Browser (<a href="#bib43">Karolchik et al., 2011</a>) to examine RefSeq genes (<a href="#bib91">Pruitt et al., 2007</a>), one finds that <span class="emphasis"><em>C. elegans</em></span> transcripts are assembled from an average of 6.4 exons while human transcripts are assembled from an average of 9.7 exons.
               Average exon sizes are similar in length between worms and humans, with a larger mean exon size of 218 nt for worm versus
               145 nt for human (<a href="#bib54">Lander et al., 2001</a>). However, intron size in <span class="emphasis"><em>C. elegans</em></span> is much smaller than in humans, with a peak of intron size clustered near the minimal intron length of 47 nt (56% of <span class="emphasis"><em>C. elegans</em></span> introns are under 100 nt in length), while human introns have a median intron length of 1,023 nt and an average of 3,365
               nt (<a href="#bib54">Lander et al., 2001</a>). This review will examine the work that has been done in <span class="emphasis"><em>C. elegans</em></span> to identify and understand the components of the splicing machinery, the extent of alternative splicing in this organism,
               the sequences in the genome that regulate this splicing, and the factors that regulate specific alternative splicing events.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Splicing of messenger RNA precursors
                     </h2>
                  </div>
               </div>
            </div>
            <p>The machinery that performs pre-mRNA splicing, the removal of introns and the ligation of exons, is called the spliceosome.
               The spliceosome assembles onto the nascent pre-mRNA as it is being transcribed. The two <span class="emphasis"><em>trans</em></span>-esterification reaction steps of pre-mRNA splicing occur in this large multi-component ribonucleoprotein complex (<a href="#bib81">Moore and Proudfoot, 2009</a>; <a href="#bib123">Wahl et al., 2009</a>). The spliceosome assembles onto the pre-mRNA via an ordered binding of its subunits, the uridine-rich small nuclear ribonucleoprotein
               complexes (U snRNPs). Over 100 different proteins are found in the spliceosome, as part of the snRNP complexes or as associated
               factors that bind to the pre-mRNA (<a href="#bib39">Jurica and Moore, 2003</a>; <a href="#bib121">Valadkhan and Jaladat, 2010</a>). snRNP complex assembly is guided by RNA-RNA interactions between the U snRNAs and the pre-mRNA and the snRNAs with each
               other. First U1 binds to the pre-mRNA guided by base-pairing interactions between its 5&#8242; end and the 5&#8242; splice site. U2 binding
               is guided by base pairing interactions with the branchpoint sequence in the intron. Then the U4/U6&#8226;U5 tri-snRNP is recruited
               to the forming spliceosome. Extensive rearrangements of RNA-RNA and RNA-protein interactions then occur in the spliceosome
               to generate a catalytically active form containing U2, U5 and U6 snRNP (<a href="#bib3">Ares and Weiser, 1994</a>; <a href="#bib71">Madhani and Guthrie, 1994</a>). Through genetic experiments in yeast, eight ATP-dependent DExD/H box helicases have been identified that are required for
               the assembly of the spliceosome, for the complex RNA-RNA and RNA-protein interaction rearrangements before, between and after
               the first and second <span class="emphasis"><em>trans</em></span>-esterification reactions, and for the disassembly of the spliceosome (<a href="#bib97">Rocak and Linder, 2004</a>).
            </p>
            <p>Several classes of splicing factor proteins have been identified as being essential for metazoan spliceosome assembly (<a href="#bib10">Blencowe, 2000</a>; <a href="#bib50">Kramer, 1996</a>). The SR protein family of splicing factors is required for the earliest interactions of U1 snRNP with the pre-mRNA and for
               subsequent spliceosome assembly steps (<a href="#bib108">Sanford et al., 2005</a>). SR proteins have redundant functions in promoting constitutive splicing, but certain family members have distinct roles
               in promoting specific alternative splice site usage. SR protein family members have distinct binding specificities for sequences
               within exons that serve as splicing enhancer elements (<a href="#bib20">Cartegni and Krainer, 2002</a>). Analysis of the RNA sequences that bind directly to the human SR protein SF2/ASF (SRSF1) using the technique of cross-linking
               immunoprecipitation (CLIP) have identified preferred exonic sequences for binding to pre-mRNA and a role for shuttling the
               mRNA out to the cytoplasm for translation (<a href="#bib107">Sanford et al., 2008</a>; <a href="#bib109">Sanford et al., 2009</a>). U2 auxiliary factor (U2AF) is a heterodimeric splicing factor composed of 35 and 65 kD subunits in humans (<a href="#bib48">Kielkopf et al., 2004</a>; <a href="#bib133">Zamore et al., 1992</a>). It binds to the polypyrimidine tract and recognizes the AG dinucleotide at the 3&#8242; end of the intron to promote U2 snRNA
               interactions with the intron branchpoint sequence (<a href="#bib75">Merendino et al., 1999</a>; <a href="#bib129">Wu et al., 1999</a>; <a href="#bib136">Zorio and Blumenthal, 1999</a>). U2AF, SR proteins and U1 snRNP assemble onto the pre-mRNA in an ATP-independent manner to form the early (E) complex (<a href="#bib76">Michaud and Reed, 1991</a>; <a href="#bib116">Staknis and Reed, 1994</a>). It is important to note that assembly of the E complex essentially identifies both the 5&#8242; and 3&#8242; splice site for further
               spliceosome assembly, and thus control of this assembly can be a way of regulating alternative splice site choice (<a href="#bib77">Michaud and Reed, 1993</a>). Splicing factors interact with the pre-mRNA, with other splicing factors and with protein components of the snRNPs (<a href="#bib15">Burge et al., 1999</a>; <a href="#bib17">Caceres and Krainer, 1997</a>; <a href="#bib50">Kramer, 1996</a>). This cooperative assembly increases the local concentration of the snRNAs in the vicinity of splice sites and therefore
               is likely to promote the RNA-RNA interactions required for spliceosome assembly.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;Constitutive splicing factors in <span class="bolditalic">C. elegans</span></h2>
                  </div>
               </div>
            </div>
            <p>The spliceosomal snRNAs of <span class="emphasis"><em>C. elegans</em></span> are very similar to their yeast and mammalian counterparts (<a href="#bib119">Thomas et al., 1990</a>). In <span class="emphasis"><em>C. elegans</em></span>, the 5&#8242; end of U1 snRNA is fully complementary to a 5&#8242; splice site sequence of AG|guaaguu (where upper case letters indicate
               the 3&#8242; end of the exon and lower case letters are the 5&#8242; end of the intron). While only a very small fraction of 5&#8242; splice
               sites are a completely complementary match to the 5&#8242; end of U1 snRNA, this sequence represents a consensus 5&#8242; splice site.
               In an alignment of over 28,000 <span class="emphasis"><em>C. elegans</em></span> introns that begin with GU, it was found that for each position in the 5&#8242; splice site region, at least 51% of the introns
               showed a match to this consensus (<a href="#bib45">Kent and Zahler, 2000</a>). The importance of U1 snRNA base-pairing with the 5&#8242; splice site is demonstrated by two extragenic allele-specific suppressors
               of mutations of the canonical G that begins <span class="emphasis"><em>C. elegans</em></span> introns. <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sup-6;class=Gene" target="_blank">sup-6</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sup-39;class=Gene" target="_blank">sup-39</a></em></span> encode two of the 12 U1 snRNA genes in <span class="emphasis"><em>C. elegans</em></span> and were identified in screens as suppressors that can allow mutated splice donors to be recognized by the splicing machinery
               (<a href="#bib99">Roller et al., 2000</a>; <a href="#bib102">Run et al., 1996</a>). These suppressor alleles are compensatory mutations in the 5&#8242; end of U1 snRNA genes that allow them to base pair with the
               mutated 5&#8242; splice sites (<a href="#bib132">Zahler et al., 2004</a>). These U1 snRNA suppressors are informational suppressors, allele-specific gene-nonspecific suppressors, and join the amber
               suppressors and the <span class="emphasis"><em>smg</em></span> genes as members of this group (<a href="#bib83">Mount and Anderson, 2000</a>). A forward genetic screen to identify additional allele-specific suppressors of a 5&#8242; splice site mutation identified three
               non-snRNA suppressors, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=smu-1;class=Gene" target="_blank">smu-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=smu-2;class=Gene" target="_blank">smu-2</a></em></span> and <span class="emphasis"><em>snrp-27</em></span>, all protein coding genes (<a href="#bib23">Dassah et al., 2009</a>). <a href="http://www.wormbase.org/db/get?name=SMU-1;class=Protein" target="_blank">SMU-1</a> is the nematode homolog of mammalian protein fSAP57, and <a href="http://www.wormbase.org/db/get?name=SMU-2;class=Protein" target="_blank">SMU-2</a> is the nematode homolog of mammalian protein RED. The mammalian proteins have been found to be associated with the mammalian
               spliceosomal C complex (the catalytically active form) (<a href="#bib39">Jurica and Moore, 2003</a>). <a href="http://www.wormbase.org/db/get?name=SMU-1;class=Protein" target="_blank">SMU-1</a> and <a href="http://www.wormbase.org/db/get?name=SMU-2;class=Protein" target="_blank">SMU-2</a> were previously identified as playing a role in alternative splicing of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> (<a href="#bib113">Spartz et al., 2004</a>). SNRP-27 is the nematode homolog of the mammalian tri-snRNP 27K protein that is associated with the U4/U6&#8226;U5 tri-snRNP (<a href="#bib27">Fetzer et al., 1997</a>) and the allele identified in this genetic screen is a dominant one (<a href="#bib23">Dassah et al., 2009</a>). Given that <a href="http://www.wormbase.org/db/get?name=SMU-2;class=Protein" target="_blank">SMU-2</a>, <a href="http://www.wormbase.org/db/get?name=SMU-1;class=Protein" target="_blank">SMU-1</a> and SNRP-27 are homologs of proteins that are involved in later assembled stages of the spliceosome in other studies, their
               role as suppressors of cryptic splice site usage suggests that these factors contribute to the fidelity of splice site choice
               at later assembly stages, after the initial identification of 5&#8242; splice sites by U1 snRNP.
            </p>
            <p>In <span class="emphasis"><em>C elegans</em></span>, the snRNP-associated proteins U1A and U2B&#8243; are approximately 50% identical to each other, but are somewhat different from
               their mammalian counterparts or the single <span class="emphasis"><em>Drosophila</em></span> homolog, SNF (<a href="#bib104">Saldi et al., 2007</a>). Saldi et al. showed that the genes that encode these proteins (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rnp-2;class=Gene" target="_blank">rnp-2</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rnp-3;class=Gene" target="_blank">rnp-3</a></em></span>) are co-transcribed in an operon, and that <a href="http://www.wormbase.org/db/get?name=RNP-2;class=Protein" target="_blank">RNP-2</a> is U1 snRNP-associated (U1A) whereas <a href="http://www.wormbase.org/db/get?name=RNP-3;class=Protein" target="_blank">RNP-3</a> is U2 snRNP-associated (U2B&#8243;). In addition to their co-regulation, they also showed that <a href="http://www.wormbase.org/db/get?name=RNP-2;class=Protein" target="_blank">RNP-2</a> and <a href="http://www.wormbase.org/db/get?name=RNP-3;class=Protein" target="_blank">RNP-3</a> are functionally redundant. Both genes must be knocked out before a phenotype is observed, indicating that either can interact
               with both snRNPs when the other is absent.
            </p>
            <p><span class="emphasis"><em>C. elegans</em></span> contains eight different members of the SR protein family called the <span class="emphasis"><em>rsp</em></span> genes (<a href="#bib44">Kawano et al., 2000</a>; <a href="#bib63">Longman et al., 2000</a>; <a href="#bib64">Longman et al., 2001</a>; <a href="#bib82">Morrison et al., 1997</a>). Family members are expressed in all cells at all stages of development (<a href="#bib44">Kawano et al., 2000</a>). There is functional overlap among the family members as studies suggest that, with the exception of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-3;class=Gene" target="_blank">rsp-3</a></em></span>, multiple SR proteins need to be targeted by RNA interference in order for a phenotype to become evident (<a href="#bib44">Kawano et al., 2000</a>; <a href="#bib63">Longman et al., 2000</a>; <a href="#bib64">Longman et al., 2001</a>). Several viable mutant alleles of SR proteins have been generated by the <span class="emphasis"><em>C. elegans</em></span> Gene Knockout Consortium. These include <span class="emphasis"><em>rsp-2(ok639)</em></span> (human homolog is SRSF5/SRp40), <span class="emphasis"><em>rsp-5(ok324)</em></span> (human homolog is SRSF2/SC35) and <span class="emphasis"><em>rsp-6(tm367)</em></span> (human homolog is SRSF7/9G8). All three deletion alleles are viable with no obvious deleterious phenotypes. Using alternative
               splicing-sensitive DNA microarrays, Barberan-Soler et al. were able to detect &gt;2-fold changes in alternative splicing isoform
               ratios for at least 10 target genes each, suggesting that SR proteins in <span class="emphasis"><em>C. elegans</em></span> do play a role in alternative splice site determination (<a href="#bib7">Barberan-Soler et al., 2011</a>). The phenomenon of <span class="emphasis"><em>trans</em></span> splicing is common in nematode species (<a href="#bib1">Allen et al., 2011</a>; <a href="#bib11">Blumenthal, 2005</a>; <a href="#bib14">Bruzik, 1996</a>; <a href="#bib57">Lasda and Blumenthal, 2011</a>). Using splicing competent embryonic extracts from the parasitic nematode <span class="emphasis"><em>Ascaris lumbricoides</em></span>, Sanford and Bruzik showed that SR proteins are essential for the <span class="emphasis"><em>trans</em></span> splicing reaction and that the phosphorylation state and activity level of these proteins changes during early development
               (<a href="#bib105">Sanford and Bruzik, 1999</a>; <a href="#bib106">Sanford and Bruzik, 1999</a>).
            </p>
            <p><span class="emphasis"><em>C. elegans</em></span> has a homolog of each U2AF subunit, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=uaf-1;class=Gene" target="_blank">uaf-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=uaf-2;class=Gene" target="_blank">uaf-2</a></em></span> (<a href="#bib137">Zorio and Blumenthal, 1999</a>; <a href="#bib138">Zorio et al., 1997</a>). <span class="emphasis"><em>C. elegans</em></span> has a short, defined polypyrimidine tract at the 3&#8242; end of its introns (<a href="#bib12">Blumenthal and Steward, 1997</a>). Studies in this organism demonstrated the specificity with which the U2AF heterodimer subunits recognize the polypyrimidine
               tract and 3&#8242; splice site at the end of the intron. The U2AF65 homolog, <a href="http://www.wormbase.org/db/get?name=UAF-1;class=Protein" target="_blank">UAF-1</a>, interacts with the short polypyrimidine tract while the U2AF35 homolog <a href="http://www.wormbase.org/db/get?name=UAF-2;class=Protein" target="_blank">UAF-2</a> recognizes the AG dinucleotide at the 3&#8242; end of the intron (<a href="#bib34">Hollins et al., 2005</a>; <a href="#bib136">Zorio and Blumenthal, 1999</a>). These studies clearly demonstrated that, even though the AG at the 3&#8242; end of the intron is involved in the second <span class="emphasis"><em>trans</em></span>-esterification reaction of splicing, it also has an important role in early spliceosome assembly. <a href="http://www.wormbase.org/db/get?name=UAF-1;class=Protein" target="_blank">UAF-1</a> regulates its own expression. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=uaf-1;class=Gene" target="_blank">uaf-1</a></em></span> message is alternatively spliced to contain an unusual exon with an in-frame stop codon. This exon is unusual because it
               contains 10 matches to the <span class="emphasis"><em>C. elegans</em></span> 3&#8242; splice acceptor sequence UUUUCAG/R, the sequence recognized directly by <span class="emphasis"><em>C. elegans</em></span> U2AF (<a href="#bib136">Zorio and Blumenthal, 1999</a>). MacMorris et al. showed that these splice acceptor repeats cause the pre-mRNA to be retained in the nucleus and thus not
               become a substrate for translation or nonsense-mediated decay. Experiments in which the UUUUCAG/R repeats were placed in a
               GFP transgene indicate that <a href="http://www.wormbase.org/db/get?name=UAF-1;class=Protein" target="_blank">UAF-1</a> is responsible for the nuclear retention of its message, suggesting a model in which an autoregulatory feedback loop controls
               U2AF homeostasis (<a href="#bib70">MacMorris et al., 1999</a>). Genetic suppressor screens have identified viable mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=uaf-1;class=Gene" target="_blank">uaf-1</a></em></span> that can activate changes in splicing of cryptic 3&#8242; splice sites in a target gene (<a href="#bib69">Ma and Horvitz, 2009</a>). In <span class="emphasis"><em>C. elegans</em></span>, there is little available information about the identity of the branchpoint sequence at which lariat formation occurs during
               the first catalytic step of splicing, and no strong branchpoint consensus sequence is found in <span class="emphasis"><em>C. elegans</em></span> introns. Branchpoint formation in other systems has been shown to require splicing factor 1/branchpoint binding protein (SF1/BBP)
               which binds to both U2AF and the branchpoint sequence (<a href="#bib62">Liu et al., 2001</a>; <a href="#bib110">Selenko et al., 2003</a>). The <span class="emphasis"><em>C. elegans</em></span> homolog of SF1/BBP, SFA-1, is essential for development and has strong binding affinity with human U2AF65 protein <span class="emphasis"><em>in vitro</em></span> (<a href="#bib73">Mazroui et al., 1999</a>). SFA-1 binding to <a href="http://www.wormbase.org/db/get?name=UAF-1;class=Protein" target="_blank">UAF-1</a>, which in turn binds strongly to the 3&#8242; splice acceptor sequence, has been proposed as a key step in identifying the weak
               branchpoint for splicing (<a href="#bib34">Hollins et al., 2005</a>). Viable mutations in <span class="emphasis"><em>sfa-1</em></span> were identified in the same screen for suppressors of the <span class="emphasis"><em>unc-93(e1500)</em></span> missense mutation in which <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=uaf-1;class=Gene" target="_blank">uaf-1</a></em></span> suppressors were isolated; this is consistent with <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=uaf-1;class=Gene" target="_blank">uaf-1</a></em></span> and <span class="emphasis"><em>sfa-1</em></span> working together in 3&#8242; splice site recognition, although a role for the <span class="emphasis"><em>sfa-1</em></span> suppressors in altering splice site usage has yet to be demonstrated (<a href="#bib69">Ma and Horvitz, 2009</a>).
            </p>
            <p><span class="emphasis"><em>C. elegans</em></span> has been used as an important model system to study the transition from mitotic proliferation to entry into meiosis in the
               germ line (<a href="#bib25">Ellis and Schedl, 2007</a>; <a href="#bib49">Kimble and Crittenden, 2005</a>). Screens for genes that masculinize the germline of hermaphrodites (loss of the ability to stop the limited sperm meiosis)
               led to identification of the mog (<span class="underline">m</span>asculinization <span class="underline">o</span>f <span class="underline">g</span>ermline) genes (<a href="#bib28">Graham and Kimble, 1993</a>; <a href="#bib29">Graham et al., 1993</a>). Four of the five <span class="emphasis"><em>mog</em></span> genes that have been cloned so far are all homologs of constitutive splicing factors. <a href="http://www.wormbase.org/db/get?name=MOG-1;class=Protein" target="_blank">MOG-1</a>, <a href="http://www.wormbase.org/db/get?name=MOG-4;class=Protein" target="_blank">MOG-4</a> and <a href="http://www.wormbase.org/db/get?name=MOG-5;class=Protein" target="_blank">MOG-5</a> are all homologs of the DEXD/H helicases PRP16, PRP2 and PRP22 respectively (<a href="#bib93">Puoti and Kimble, 1999</a>; <a href="#bib94">Puoti and Kimble, 2000</a>). <a href="http://www.wormbase.org/db/get?name=MOG-2;class=Protein" target="_blank">MOG-2</a> is a homolog of the U2 snRNP protein U2A&#8242; (<a href="#bib134">Zanetti et al., 2011</a>). In oocyte production, <a href="http://www.wormbase.org/db/get?name=GLP-1;class=Protein" target="_blank">GLP-1</a>/Notch activation in pre-meiotic germ cells, in response to signaling from the distal tip cell of the gonad arm, controls
               the mitotic/meiotic proliferation/differentiation of the germ cells through repression of the redundant <a href="http://www.wormbase.org/db/get?name=GLD-1;class=Protein" target="_blank">GLD-1</a> and <a href="http://www.wormbase.org/db/get?name=GLD-2;class=Protein" target="_blank">GLD-2</a> pathways (<a href="#bib49">Kimble and Crittenden, 2005</a>). A screen for enhancers of a weak gain of function allele of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span> identified the worm homolog of the yeast PRP17 splicing factor, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=prp-17;class=Gene" target="_blank">prp-17</a></em></span> (<a href="#bib47">Kerins et al., 2010</a>). Given that both <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=prp-17;class=Gene" target="_blank">prp-17</a></em></span> and the <span class="emphasis"><em>mog</em></span> genes appear to be constitutive splicing factors, Kerins and colleagues performed an RNAi screen against known splicing factors
               and looked for phenotypes of extensive proliferation in the germline. 47/114 factors tested by RNAi showed over-proliferation
               and/or Mog sex determination phenotypes (<a href="#bib47">Kerins et al., 2010</a>). Table 2 and supplemental table S1 of the <a href="#bib47">Kerins et al. 2010</a> () paper are an excellent resource as catalogs of the <span class="emphasis"><em>C. elegans</em></span> homologs of known yeast and mammalian spliceosome components. So why are mutations in the splicing machinery specifically
               associated with the proliferation/meiosis switch and with sex determination? Kerins et al. propose three models to explain
               these observations (<a href="#bib47">Kerins et al., 2010</a>): 1) that one or more genes required for controlling meiosis are spliced inefficiently, and so mutations that make splicing
               less efficient are more likely to show these phenotypes; 2) that one or more genes controlling meiosis are regulated through
               alternative splicing, and that defects that make splicing less efficient change alternative splicing; 3) that pre-mRNA splicing
               may be necessary for assembly of ribonucleoprotein complexes on mRNAs in the nucleus, such as the exon-junction complex, that
               may be important for RNA stability and translational regulation of target genes in the cytoplasm (<a href="#bib47">Kerins et al., 2010</a>). The finding that key developmental processes require efficient splicing opens up important opportunities for further research
               in both developmental biology and splicing regulation.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Alternative pre-mRNA splicing
                     </h2>
                  </div>
               </div>
            </div>
            <p>Alternative splicing is a mechanism for generating multiple mRNA transcripts from a single gene (<a href="#bib42">Kalsotra and Cooper, 2011</a>). Types of alternative splicing include the use of alternative 5&#8242; splice sites, alternative 3&#8242; splice sites, cassette exons,
               retained introns and mutually exclusive exons. These are represented graphically in <a href="#figure1">Figure 1</a>. Alternative splicing often leads to changes in the primary amino acid sequence of the protein, sometimes subtle and sometimes
               quite dramatic. Alternative splicing can control where the site of 3&#8242; end cleavage and polyadenlyation occurs, indicating
               a strong interaction between the splicing and 3&#8242; end formation machineries (<a href="#bib90">Proudfoot, 2011</a>). Many of the genes with alternative isoforms are generated through the use of alternative promoters. Even though these alternative
               promoters will lead to alternative 5&#8242; exons spliced to common downstream exons, the generation of the unique first exon is
               not regulated at the level of alternative splicing but instead by the decisions that govern transcriptional initiation. Alternative
               splicing may also produce isoforms that use different stop codons. An isoform containing a premature termination codon (PTC)
               is a candidate for degradation by the nonsense-mediated decay (NMD) pathway. Evidence for the coupling of alternative splicing
               and NMD was first demonstrated in <span class="emphasis"><em>C. elegans</em></span> in a forward genetic screen for regulators of alternative splicing. Morrison <span class="emphasis"><em>et al</em></span>. showed that two of the genes encoding SR protein splicing factors have alternative isoforms with premature termination codons.
               These alternatively spliced forms were preferentially stabilized in <span class="underline">s</span>uppressor with <span class="underline">m</span>orphogenetic effects on <span class="underline">g</span>enitalia (smg) mutants leading to a change in the steady-state ratio of the alternative isoforms (<a href="#bib82">Morrison et al., 1997</a>). This was the first demonstration that natural substrates for the NMD pathway are alternative isoforms of transcripts. Previously,
               mutant alleles of genes that introduce a PTC were used to identify mutations in smg genes as suppressors (<a href="#bib32">Hodgkin et al., 1989</a>), and a role in suppression of aberrant transcripts caused by mutation had been proposed as the function of the NMD pathway
               (<a href="#bib92">Pulak and Anderson, 1993</a>). Extensive analysis of vertebrate alternative splicing indicates that one third of alternative splicing events leads to
               production of an isoform that is a substrate for the NMD pathway (<a href="#bib58">Lewis et al., 2003</a>). To measure the ratio of alternative mRNA isoforms for a particular gene, scientists determine the relative steady-state
               levels of the isoforms in RNA extracted from cells, tissues or animals by reverse transcription-PCR, quantitative PCR, splicing-sensitive
               DNA microarrays or RNA-seq. When measuring alternatively spliced isoforms it is important to remember that the steady-state
               ratio of the isoforms represents a combination of two variables; the relative amounts of the isoforms generated co-transcriptionally
               by the splicing machinery in the nucleus and the relative stability of those mRNA isoforms in the nucleus and cytoplasm.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
                  <tr>
  <td align="center"><p><a href="Zahler_Pre-mRNA_fig1.jpg"><img src="Zahler_Pre-mRNA_fig1_s.jpg" border="2" align="middle" alt="figure 1"></a>
  </td>
                  </tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1.&nbsp;</b>Types of alternative splicing. In these graphics, exons are represented by boxes and introns by lines. Exon regions included
                     in the messages by alternative splicing are colored while constitutive exons are shown in gray. Promoters are indicated with
                     arrows and polyadenylation sites with AAAA&#8230;
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;Identification of alternatively spliced genes in <span class="bolditalic">C. elegans</span></h2>
                  </div>
               </div>
            </div>
            <p>Initial identification of alternatively spliced genes in <span class="emphasis"><em>C. elegans</em></span> was dependent on comparison of Expressed Sequence Tags (ESTs; libraries of randomly sequenced cDNAs) to the genome sequence
               (<a href="#bib45">Kent and Zahler, 2000</a>). From this initial identification of 844 alternatively spliced genes there has been steady growth in this number through
               constant annotation of genes based on experimental results, additional EST evidence as well as SAGE data analysis (<a href="#bib103">Ruzanov et al., 2007</a>). New data for identifying alternatively spliced genes comes from the new high-throughput RNA sequencing technologies. Ramani
               et al. have reported using RNA-seq to identify alternatively spliced genes in <span class="emphasis"><em>C. elegans</em></span> at various developmental time points (<a href="#bib95">Ramani et al., 2011</a>). In addition to using this approach to confirm the existence of 71% of the previously annotated alternative splicing events
               in <span class="emphasis"><em>C. elegans</em></span>, they report sequence evidence for 2183 new alternatively splicing events, with 759 of these new splice junctions detected
               in 5 or more independent sequencing reads. Their alternative splicing predictions and evidence for developmental regulation
               are accessible in a genome browser format at <span class="underline">http://splicebrowse.ccbr.utoronto.ca/</span>. As of Wormbase release WS226, there are 4732 alternative mRNA isoforms annotated for 20,439 protein-coding genes. Ramani
               and colleagues performed an extrapolation from their data based on 200,000,000 sequence reads to suggest that there are ~5000
               genes that undergo alternative splicing in <span class="emphasis"><em>C. elegans</em></span>, representing ~25% of protein coding genes (<a href="#bib95">Ramani et al., 2011</a>). While this number is less than the percentage of genes predicted to undergo alternative splicing in mammals, it is strong
               indication that alternative splicing is a major gene regulatory process in nematodes. Consistent with the model that alternative
               splicing is a critical process in nematodes, several studies have shown that worm alternative splicing and its regulation
               are under strong evolutionary selective pressure (<a href="#bib8">Barberan-Soler and Zahler, 2008</a>; <a href="#bib37">Irimia et al., 2008</a>; <a href="#bib60">Li et al., 2010</a>; <a href="#bib101">Rukov et al., 2007</a>).
            </p>
            <p><span class="emphasis"><em>C. elegans</em></span> alternatively spliced genes are capable of undergoing developmental regulation. <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=let-2;class=Gene" target="_blank">let-2</a></em></span>, an alpha(2) type IV collagen gene has two mutually exclusive exons; this was the first gene to be shown to undergo a dramatic
               switch in alternative isoform usage to generate embryo- and adult-specific versions of the protein (<a href="#bib112">Sibley et al., 1993</a>). Using custom DNA microarrays designed to detect alternative splicing, Barberan-Soler uncovered 62 genes whose alternative
               splicing showed &gt;4-fold changes in isoform ratios during development (<a href="#bib9">Barberan-Soler and Zahler, 2008</a>). Recently RNA-Seq combined with microarray approaches uncovered 574 temporally-regulated alternative splicing events (<a href="#bib95">Ramani et al., 2011</a>). Examples of sex-specific alternative splicing in nematodes have been uncovered. One isoform of the nuclear hormone receptor
               <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-55;class=Gene" target="_blank">unc-55</a></em></span> is only produced in males (<a href="#bib111">Shan and Walthall, 2008</a>). In hermaphrodites, the <span class="underline">f</span>eminizing locus <span class="underline">o</span>n <span class="underline">X</span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fox-1;class=Gene" target="_blank">fox-1</a></em></span>) gene encodes an RNA binding protein that is a regulator of sex determination and acts as a numerator for counting X chromosomes
               (<a href="#bib33">Hodgkin et al., 1994</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fox-1;class=Gene" target="_blank">fox-1</a></em></span> acts post-transcriptionally to inhibit the expression of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=xol-1;class=Gene" target="_blank">xol-1</a></em></span> gene, the major specifier of male fate (<a href="#bib85">Nicoll et al., 1997</a>). The mammalian homologs of <a href="http://www.wormbase.org/db/get?name=FOX-1;class=Protein" target="_blank">FOX-1</a> were subsequently shown to be key regulators of tissue-specific alternative splicing (<a href="#bib38">Jin et al., 2003</a>; <a href="#bib120">Underwood et al., 2005</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;Regulation of alternative splicing
                     </h2>
                  </div>
               </div>
            </div>
            <p>The spliceosome assembles through an ordered binding of its snRNP subunits onto the pre-mRNA (<a href="#bib15">Burge et al., 1999</a>). Alternative splicing regulation essentially controls where the assembly of a functional spliceosome occurs. Alternatively
               spliced exons often have weak consensus sequences at the 5&#8242; and 3&#8242; ends of the introns, suggesting that additional signals
               are required for recognition of the exon by the splicing machinery (<a href="#bib65">Lopez, 1998</a>). Twenty five years of genomic, genetic, and biochemical experiments have led to the understanding that <span class="emphasis"><em>cis</em></span> splicing regulatory elements on the pre-mRNA are responsible for controlling alternative splicing (<a href="#bib30">Hallegger et al., 2010</a>). These regulatory regions are found in alternative exons or in flanking introns and can be enhancers or silencers of splice
               site usage (<a href="#figure2">Figure 2</a>). These sequence motifs serve as binding sites for protein factors that can enhance or inhibit the ability of the spliceosome
               to recognize the exons. The exonic elements not only encode amino acids but also regulate their own ability to be spliced
               into the mature message. In the "splicing code" hypothesis, the combinatorial assembly of different splicing factors onto
               multiple <span class="emphasis"><em>cis</em></span> elements controls where a functional spliceosome assembles (<a href="#bib127">Wang and Burge, 2008</a>). Since the same pre-mRNA sequence is transcribed in different cell types and the constitutive splicing machinery is present
               in all cells, the regulation of cell type-specific splicing is dependent on the presence and the activity of <span class="emphasis"><em>trans</em></span>-acting protein factors that bind the nascent transcript in a given cell. Catalogs of <span class="emphasis"><em>cis</em></span> splicing elements, as well as models for their potential role in affecting alternative splicing in specific cell types based
               on their position relative to the alternative splicing event, have increased in recent years to the point where most major
               players have likely been identified (<a href="#bib18">Calarco et al., 2011</a>). A recent major advance in computational prediction of mouse tissue-specific alternative splicing has dramatically improved
               our understanding of the role of splicing elements (<a href="#bib5">Barash et al., 2010</a>). Despite improvements in probabilistic models for decoding alternative splicing, mechanistic studies are still required
               to understand how the <span class="emphasis"><em>trans</em></span>-acting factors binding to these <span class="emphasis"><em>cis</em></span> elements act in a combinatorial manner to regulate spliceosome assembly.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
                  <tr>
  <td align="center"><p><a href="Zahler_Pre-mRNA_fig2.jpg"><img src="Zahler_Pre-mRNA_fig2_s.jpg" border="2" align="middle" alt="figure 2"></a>
  </td>
                  </tr>
               </table>
               <div class="caption">
                  <p><b>Figure 2.&nbsp;</b>Elements of the pre-mRNA through which alternative splicing is regulated. Some combination of these regulatory regions can
                     usually be found. Weaker consensus splice sites surrounding the alternative exon, exonic regulatory regions and intronic regulatory
                     regions are indicated.
                  </p>
               </div>
            </div>
            <p>Although the catalogs of <span class="emphasis"><em>cis</em></span> elements and <span class="emphasis"><em>trans</em></span>-acting factors are increasing, the mechanisms by which these control spliceosome assembly have been difficult to resolve.
               For example, splicing factors Fox1, hnRNP L and PTB act through dramatically different mechanisms to silence splicing of target
               exons. FOX1 binding to UGCAUG sequences blocks formation of the spliceosomal E complex at two distinct steps (<a href="#bib135">Zhou and Lou, 2008</a>). HnRNP L blocks tri-snRNP recruitment through interactions with adjacent assembled U1 and U2 snRNPs (<a href="#bib35">House and Lynch, 2006</a>). Interactions between PTB proteins bound to both introns flanking an alternative cassette exon are proposed to loop out
               the exon and prevent its inclusion in the mature message (<a href="#bib59">Li et al., 2007</a>; <a href="#bib122">Wagner and Garcia-Blanco, 2001</a>). Mechanistic studies must take into account the fact that multiple regulatory proteins assemble onto splicing control sequences.
               One example of the challenges in deciphering alternative splicing regulation is in the regulation of the neural-specific N1
               exon of human c-src. The downstream control sequence (DCS), found in the intron downstream of the N1 exon, binds multiple
               factors to regulate N1 splicing; these include hnRNP H, hnRNP F, <span class="underline">K</span>H-type <span class="underline">s</span>plicing <span class="underline">r</span>egulatory <span class="underline">p</span>rotein (KSRP), Fox1 and a neural-specific homolog of PTB (nPTB) (<a href="#bib21">Chou et al., 1999</a>; <a href="#bib22">Chou et al., 2000</a>; <a href="#bib72">Markovtsov et al., 2000</a>; <a href="#bib79">Min et al., 1995</a>; <a href="#bib80">Min et al., 1997</a>; <a href="#bib120">Underwood et al., 2005</a>). Comparisons of the factors that assemble onto this element from neuronal cell nuclear extract vs. epithelial cell nuclear
               extract indicate that a subset of these proteins binds from each extract. Factors in the neuronal extract promote assembly
               of a different complex that is required to activate splicing of the neural-specific exon. Given that only a small number of
               alternative splicing substrates have been analyzed in detail, much more work is required to understand how binding of splicing
               factors to an intron or exon regulates splicing.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec7"></a>7.&nbsp;Identifying the elements of the <span class="bolditalic">C. elegans</span> splicing code
                     </h2>
                  </div>
               </div>
            </div>
            <p>With <span class="emphasis"><em>C. elegans</em></span> as the first completely sequenced animal genome (<a href="#bib16"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>), and with comparative genomic information available, initially from <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib117">Stein et al., 2003</a>) but now from many related nematodes, this model system offers advantages to the identification of alternative splicing regulatory
               elements. After 100 million years of evolutionary divergence, <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> exons have maintained 80% sequence identity, while the introns flanking these exons have diverged rapidly such that it is
               rare to find evolutionarily conserved sequences within them (<a href="#bib45">Kent and Zahler, 2000</a>). However, in many introns flanking alternatively spliced exons there are defined patches of evolutionary conservation. <a href="#figure3">Figure 3</a> shows five examples of alternative splicing showing regions of evolutionary conservation in introns flanking alternative
               exons. It was hypothesized that these distinct evolutionarily conserved intronic sequences had the potential to define key
               splicing regulatory elements (<a href="#bib46">Kent and Zahler, 2000</a>). An analysis of the frequency of pentamer and hexamer sequence occurrence in these highly conserved intronic elements compared
               to their frequency in a total intron database was used to identify hexamer and pentamer words associated with alternative
               splicing in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib40">Kabat et al., 2006</a>). Pentamers and hexamers were chosen for these studies because these represent the known length of preferred RNA binding
               sequences for many RNA binding proteins involved in splicing regulation. Many of these conserved intronic pentamers/hexamers
               are a direct match to human alternative splicing regulatory elements, such as GCAUG for Fox1 (<a href="#bib38">Jin et al., 2003</a>), CUCUCU for PTB (<a href="#bib87">Oberstrass et al., 2005</a>) and GUGUGU for ETR-3 (<a href="#bib26">Faustino and Cooper, 2005</a>), indicating the conserved nature of alternative splicing regulation in animals. In addition to identifying lists of regulatory
               elements, distinct biases were observed as to whether these elements were likely to appear in the intron upstream or the intron
               downstream of the alternative cassette exon. One of the high scoring words in <span class="emphasis"><em>C. elegans</em></span> that has not been identified as a splicing regulatory element in mammals is UCUAUC. This element was demonstrated to regulate
               alternative splicing in combination with a flanking GCAUG <a href="http://www.wormbase.org/db/get?name=FOX-1;class=Protein" target="_blank">FOX-1</a> binding site (<a href="#bib40">Kabat et al., 2006</a>). RNA affinity chromatography techniques were employed to identify proteins in <span class="emphasis"><em>C. elegans</em></span> that bind to this new splicing regulatory element. <span class="emphasis"><em>C. elegans</em></span> cellular extracts were passed over columns containing immobilized UCUAUC RNA sequences. The <span class="emphasis"><em>C. elegans</em></span> homolog of the human hnRNP Q protein, <a href="http://www.wormbase.org/db/get?name=HRP-2;class=Protein" target="_blank">HRP-2</a>, was identified as a high-affinity binder for this sequence and the protein was shown to regulate alternative splicing through
               binding to UCUAUC in introns (<a href="#bib41">Kabat et al., 2009</a>).
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
                  <tr>
  <td align="center"><p><a href="Zahler_Pre-mRNA_fig3.jpg"><img src="Zahler_Pre-mRNA_fig3_s.jpg" border="2" align="middle" alt="figure 3"></a>
  </td>
                  </tr>
               </table>
               <div class="caption">
                  <p><b>Figure 3.&nbsp;</b>Images from the Intronerator genome browser showing alternatively spliced genes (<a href="#bib45">Kent and Zahler, 2000</a>). Gene isoforms predicted by the Wormbase Consortium are shown in blue and WABA homology alignments for <span class="emphasis"><em>C. briggsae</em></span> to this region of the <span class="emphasis"><em>C. elegans</em></span> genome are shown in purple. Dark purple indicates a region of WABA high homology, light purple corresponds to low homology,
                     and white indicates no homology between species (<a href="#bib46">Kent and Zahler, 2000</a>). Regions of alternatively spliced genes: A) W01F3.1, B) ZC477.9, C) ZK637.8, D) H24G06.1 and E) C11D2.6 are shown. Figure
                     adapted from <a href="#bib40">Kabat et al., 2006</a> (<a href="#bib40">Kabat et al., 2006</a>).
                  </p>
               </div>
            </div>
            <p>An alternative approach to discovering new alternative splicing events and their <span class="emphasis"><em>cis</em></span> regulatory elements was done computationally using a support vector machine trained to identify predictive features of <span class="emphasis"><em>C. elegans</em></span> alternative cassette exons (<a href="#bib96">Ratsch et al., 2005</a>). One of the features that allowed the support vector machine to predict alternative splicing was the presence of several
               distinct hexamers in the introns surrounding alternative exons. In a recent approach, RNA-Seq and microarrays were used to
               identify developmentally-regulated alternative splicing events (<a href="#bib95">Ramani et al., 2011</a>). Their analysis of the 50 intronic nucleotides on either side of the alternative cassette exons revealed enriched pentamer
               motifs. There is an obvious strong degree of overlap in the high scoring pentamers and hexamers identified in all three studies
               (<a href="#bib40">Kabat et al., 2006</a>; <a href="#bib95">Ramani et al., 2011</a>; <a href="#bib96">Ratsch et al., 2005</a>). It is clear that progress has been made in the past several years in identifying the most common intronic <span class="emphasis"><em>cis</em></span> regulators of alternative splicing in <span class="emphasis"><em>C. elegans</em></span>. <a href="#table1" title="Table 1. Characterized intronic components of the C. elegans alternative splicing code. The table shows high scoring pentamer and hexamer sequences that were identified as intronic alternative splicing regulators in three different studies (Kabat et al., 2006; Ramani et al., 2011; Ratsch et al., 2005). Biases in the occurrence of the cis element in the intron upstream or downstream of an alternative cassette exon are indicated in the second column. Alternative splicing factors predicted to bind to the cis regulatory elements are indicated in the third column. These high-affinity binding predictions are based on the following C. elegans reports; HRP-2 (Kabat et al., 2009), ASD-2 (Ohno et al., 2008), FOX-1 and ASD-1 (Kuroyanagi et al., 2006), and SUP-12 (Anyanful et al., 2004; Kuroyanagi et al., 2007). The prediction for PTB-1 is based on the binding of its human homolog, PTB, directly to CUCUCU repeats (Oberstrass et al., 2005).">Table 1</a> shows several of the top scoring splicing regulatory hexamers and pentamers from these studies and the <span class="emphasis"><em>trans</em></span>-acting proteins predicted to bind to them. The next challenge to understanding the splicing code is in deciphering how the
               binding of proteins to these elements regulates alternative splicing.
            </p>
            <div class="table"><a name="table1"></a><p class="title"><span class="bold">Table 1.</span> Characterized intronic components of the <span class="emphasis"><em>C. elegans</em></span> alternative splicing code. The table shows high scoring pentamer and hexamer sequences that were identified as intronic alternative
                  splicing regulators in three different studies (<a href="#bib40">Kabat et al., 2006</a>; <a href="#bib95">Ramani et al., 2011</a>; <a href="#bib96">Ratsch et al., 2005</a>). Biases in the occurrence of the <span class="emphasis"><em>cis</em></span> element in the intron upstream or downstream of an alternative cassette exon are indicated in the second column. Alternative
                  splicing factors predicted to bind to the <span class="emphasis"><em>cis</em></span> regulatory elements are indicated in the third column. These high-affinity binding predictions are based on the following
                  <span class="emphasis"><em>C. elegans</em></span> reports; <a href="http://www.wormbase.org/db/get?name=HRP-2;class=Protein" target="_blank">HRP-2</a> (<a href="#bib41">Kabat et al., 2009</a>), ASD-2 (<a href="#bib88">Ohno et al., 2008</a>), <a href="http://www.wormbase.org/db/get?name=FOX-1;class=Protein" target="_blank">FOX-1</a> and <a href="http://www.wormbase.org/db/get?name=ASD-1;class=Protein" target="_blank">ASD-1</a> (<a href="#bib51">Kuroyanagi et al., 2006</a>), and <a href="http://www.wormbase.org/db/get?name=SUP-12;class=Protein" target="_blank">SUP-12</a> (<a href="#bib2">Anyanful et al., 2004</a>; <a href="#bib52">Kuroyanagi et al., 2007</a>). The prediction for <a href="http://www.wormbase.org/db/get?name=PTB-1;class=Protein" target="_blank">PTB-1</a> is based on the binding of its human homolog, PTB, directly to CUCUCU repeats (<a href="#bib87">Oberstrass et al., 2005</a>).
               </p>
               <table summary="Table 1. Characterized intronic components of the C. elegans alternative splicing code. The table shows high scoring pentamer and hexamer sequences that were identified as intronic alternative splicing regulators in three different studies (Kabat et al., 2006; Ramani et al., 2011; Ratsch et al., 2005). Biases in the occurrence of the cis element in the intron upstream or downstream of an alternative cassette exon are indicated in the second column. Alternative splicing factors predicted to bind to the cis regulatory elements are indicated in the third column. These high-affinity binding predictions are based on the following C. elegans reports; HRP-2 (Kabat et al., 2009), ASD-2 (Ohno et al., 2008), FOX-1 and ASD-1 (Kuroyanagi et al., 2006), and SUP-12 (Anyanful et al., 2004; Kuroyanagi et al., 2007). The prediction for PTB-1 is based on the binding of its human homolog, PTB, directly to CUCUCU repeats (Oberstrass et al., 2005)." border="1">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr valign="bottom">
                        <th align="center" valign="bottom"><span class="emphasis"><em>cis</em></span> element
                        </th>
                        <th align="left" valign="bottom">Bias of element occurrence in the intron relative to an alternative cassette exon</th>
                        <th align="center" valign="bottom">Predicted <span class="emphasis"><em>trans</em></span> splicing factor
                        </th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr valign="top">
                        <td align="left" valign="top">UCUAUC</td>
                        <td align="left" valign="top">Upstream Intron</td>
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=HRP-2;class=Protein" target="_blank">HRP-2</a></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">CUAAC</td>
                        <td align="left" valign="top">No Bias</td>
                        <td align="left" valign="top">ASD-2</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">GCAUG</td>
                        <td align="left" valign="top">Downstream Intron</td>
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=FOX-1;class=Protein" target="_blank">FOX-1</a>, <a href="http://www.wormbase.org/db/get?name=ASD-1;class=Protein" target="_blank">ASD-1</a></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">GUGUGU</td>
                        <td align="left" valign="top">Downstream Intron</td>
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=SUP-12;class=Protein" target="_blank">SUP-12</a></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">CUCUCU</td>
                        <td align="left" valign="top">Upstream Intron</td>
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=PTB-1;class=Protein" target="_blank">PTB-1</a></td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec8"></a>8.&nbsp;Genetic identification of alternative splicing factors in <span class="emphasis"><em>C. elegans</em></span></h2>
                  </div>
               </div>
            </div>
            <p>Genetic analysis in <span class="emphasis"><em>C. elegans</em></span> has led to the identification of splicing regulatory pathways. <a href="http://www.wormbase.org/db/get?name=MEC-8;class=Protein" target="_blank">MEC-8</a> is a nuclear protein that contains two RNA recognition motifs (RRMs). Mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span> lead to mechanosensory and other defects. Mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span> are synthetically lethal with viable mutations in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> gene (<a href="#bib68">Lundquist et al., 1996</a>). These viable <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> mutations occur only in exons 17 and 18, which are alternatively spliced (<a href="#bib98">Rogalski et al., 1995</a>). Skipping of exons 17 and 18 is dependent on the function of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span> (<a href="#bib68">Lundquist et al., 1996</a>). <a href="http://www.wormbase.org/db/get?name=MEC-8;class=Protein" target="_blank">MEC-8</a> protein is found in all nuclei in embryos, but expression becomes more limited to specific cell types in larvae (<a href="#bib114">Spike et al., 2002</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> mutant alleles with stop codons in exons 17 and 18 develop a progressive paralysis at later developmental stages; in contrast,
               mutations in other parts of the gene are lethal. Animals containing both viable mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> and viable mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span> have a synthetic lethal phenotype that mimics the embryonic lethality of the lethal <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> alleles (<a href="#bib67">Lundquist and Herman, 1994</a>). In <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a>;unc-52</em></span> double mutant animals, exons 17 and 18 (containing stop codons) cannot be skipped, and this leads to insufficient production
               of full-length <a href="http://www.wormbase.org/db/get?name=UNC-52;class=Protein" target="_blank">UNC-52</a> in embryos and thus to the more severe embryonic lethality. <a href="#figure4">Figure 4</a> summarizes <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> alternative splicing and the specific alternative splicing events promoted by <a href="http://www.wormbase.org/db/get?name=MEC-8;class=Protein" target="_blank">MEC-8</a>. In a genetic screen based on this synthetic lethality, two <span class="underline">s</span>uppressors of the <span class="emphasis"><em><span class="underline">m</span>ec-8; <span class="underline">u</span>nc-52</em></span> synthetic lethality (smu) were identified (<a href="#bib67">Lundquist and Herman, 1994</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=smu-1;class=Gene" target="_blank">smu-1</a></em></span> encodes a nuclear protein that contains five WD motifs, is ubiquitously expressed and is 62% identical to a human spliceosome
               associated protein fSAP57 (<a href="#bib115">Spike et al., 2001</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=smu-2;class=Gene" target="_blank">smu-2</a></em></span> encodes a <span class="emphasis"><em>C. elegans</em></span> homolog of the human RED protein, which has also been shown to be associated with the human spliceosome. <a href="http://www.wormbase.org/db/get?name=SMU-1;class=Protein" target="_blank">SMU-1</a> and <a href="http://www.wormbase.org/db/get?name=SMU-2;class=Protein" target="_blank">SMU-2</a> bind to each other; <a href="http://www.wormbase.org/db/get?name=SMU-2;class=Protein" target="_blank">SMU-2</a> stabilizes <a href="http://www.wormbase.org/db/get?name=SMU-1;class=Protein" target="_blank">SMU-1</a> but <a href="http://www.wormbase.org/db/get?name=SMU-1;class=Protein" target="_blank">SMU-1</a> does not stabilize <a href="http://www.wormbase.org/db/get?name=SMU-2;class=Protein" target="_blank">SMU-2</a>. Based on their homology with components of the mammalian spliceosome, it is hypothesized that <a href="http://www.wormbase.org/db/get?name=SMU-1;class=Protein" target="_blank">SMU-1</a> and <a href="http://www.wormbase.org/db/get?name=SMU-2;class=Protein" target="_blank">SMU-2</a> might interact with the spliceosome and modulate splice site choice (<a href="#bib113">Spartz et al., 2004</a>). A screen for proteins that are <span class="underline">sy</span>nthetic-lethal with <span class="underline">m</span>ec-8 mutations (sym) uncovered another splicing factor, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sym-2;class=Gene" target="_blank">sym-2</a></em></span>, which is a homolog of the human alternative splicing factors hnRNP H and F (<a href="#bib24">Davies et al., 1999</a>; <a href="#bib130">Yochem et al., 2004</a>). In total, analysis of suppressors and enhancers of the viable <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> alleles has led to identification of 4 splicing regulatory factors.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
                  <tr>
  <td align="center"><p><a href="Zahler_Pre-mRNA_fig4.jpg"><img src="Zahler_Pre-mRNA_fig4_s.jpg" border="2" align="middle" alt="figure 4"></a>
  </td>
                  </tr>
               </table>
               <div class="caption">
                  <p><b>Figure 4.&nbsp;</b>Alternative splicing of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span>. Diagram of alternative splicing of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> gene. Every combination of exon inclusion and skipping of exons 16, 17 and 18 has been detected. <a href="http://www.wormbase.org/db/get?name=MEC-8;class=Protein" target="_blank">MEC-8</a> enhances the skipping of exons 17 and 18; isoforms promoted by <a href="http://www.wormbase.org/db/get?name=MEC-8;class=Protein" target="_blank">MEC-8</a> are indicated in red (<a href="#bib68">Lundquist et al., 1996</a>).
                  </p>
               </div>
            </div>
            <p>In addition to a role in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> splicing regulation, <a href="http://www.wormbase.org/db/get?name=MEC-8;class=Protein" target="_blank">MEC-8</a> has also been identified as a regulator of alternative splicing of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-2;class=Gene" target="_blank">mec-2</a></em></span> transcript. <a href="http://www.wormbase.org/db/get?name=MEC-8;class=Protein" target="_blank">MEC-8</a> is required for removal of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-2;class=Gene" target="_blank">mec-2</a></em></span> intron 9, and the sequences required for <a href="http://www.wormbase.org/db/get?name=MEC-8;class=Protein" target="_blank">MEC-8</a> regulation are contained within the intron 9 sequence (<a href="#bib19">Calixto et al., 2010</a>). Taking advantage of the MEC-8-dependent removal of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-2;class=Gene" target="_blank">mec-2</a></em></span> intron 9, and the fact that there is a temperature sensitive <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span> allele, <span class="emphasis"><em>mec-8(u218ts)</em></span>, Calixto and colleagues have engineered a conditional gene expression system. <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-2;class=Gene" target="_blank">mec-2</a></em></span> intron 9 is inserted into a gene of interest and this transgene is then transformed into a <span class="emphasis"><em>u218ts</em></span> background. While the intron is removed efficiently at the permissive temperature, when these worms are raised to the restrictive
               temperature, intron removal ceases and the gene of interest is turned off (<a href="#bib19">Calixto et al., 2010</a>).
            </p>
            <p>Tissue-specific expression of alternative splicing factors has been demonstrated in <span class="emphasis"><em>C. elegans</em></span>. <a href="http://www.wormbase.org/db/get?name=SUP-12;class=Protein" target="_blank">SUP-12</a> is an RRM-containing protein that was identified in a genetic screen for suppressors of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-60;class=Gene" target="_blank">unc-60</a></em></span> mutations. <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-60;class=Gene" target="_blank">unc-60</a></em></span> is alternatively spliced to yield two isoforms, UNC-60A and UNC-60B. <a href="http://www.wormbase.org/db/get?name=UNC-60;class=Protein" target="_blank">UNC-60</a> is a homolog of the actin depolymerizing factor cofilin; UNC-60A and UNC-60B are expressed in non-muscle cells and muscle
               cells respectively. <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sup-12;class=Gene" target="_blank">sup-12</a></em></span> is expressed in muscle cells and is required for the generation of the UNC-60B isoform in those cells by alternative splicing
               (<a href="#bib2">Anyanful et al., 2004</a>). Loss of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sup-12;class=Gene" target="_blank">sup-12</a></em></span> in <span class="emphasis"><em>unc-60B</em></span> mutants allows for splicing to the UNC-60A isoform in muscle cells, and the presence of this cofilin in muscle cells can
               rescue the uncoordination phenotype. <a href="http://www.wormbase.org/db/get?name=SUP-12;class=Protein" target="_blank">SUP-12</a> was shown to function through interactions with GU rich elements in introns (<a href="#bib2">Anyanful et al., 2004</a>). A fluorescent reporter-based genetic screen for factors that alter <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-15;class=Gene" target="_blank">egl-15</a></em></span> alternative splicing also identified <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sup-12;class=Gene" target="_blank">sup-12</a></em></span> as a key splicing regulator, again through interactions with GU-rich intronic elements (<a href="#bib52">Kuroyanagi et al., 2007</a>).
            </p>
            <p>Two neuronal, nuclear RNA binding proteins with a role in synaptic transmission have been identified in <span class="emphasis"><em>C. elegans</em></span>. These are <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-75;class=Gene" target="_blank">unc-75</a></em></span>, which is an ortholog of the human CELF family of alternative splicing factors, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=exc-7;class=Gene" target="_blank">exc-7</a></em></span>, which is a homolog of the <span class="emphasis"><em>Drosophila</em></span> neuronal splicing factor ELAV (<a href="#bib66">Loria et al., 2003</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lec-3;class=Gene" target="_blank">lec-3</a></em></span> has been identified as a gene whose alternative splicing changes in both <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=exc-7;class=Gene" target="_blank">exc-7</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-75;class=Gene" target="_blank">unc-75</a></em></span> mutant strains (<a href="#bib7">Barberan-Soler et al., 2011</a>). A <span class="emphasis"><em>C. elegans</em></span> homolog of the human CUGBP1 splicing factor, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=etr-1;class=Gene" target="_blank">etr-1</a></em></span>, is expressed only in muscle cells and is essential for development (<a href="#bib78">Milne and Hodgkin, 1999</a>). These studies are strongly suggestive of an important role for alternative RNA processing in tissue specification.
            </p>
            <p>Kuroyanagi and colleagues have engineered a creative and powerful genetic screening system with which to analyze alternative
               splicing. They create dual alternative splicing reporter constructs in which each alternative isoform is fused in frame to
               either green or red fluorescent protein (GFP or RFP). Two transgenes containing the alternatively spliced region of interest
               are transformed simultaneously into worms; one version of the transgene has one outcome of the alternative splicing event
               in frame with GFP, while the other transgene version has the other alternative isoform in frame with RFP (<a href="#bib53">Kuroyanagi et al., 2010</a>). They have used this system in forward genetic screens, screening for changes in the GFP:RFP ratio in the worms that are
               indicative of changes in alternative splicing. The have searched for regulators of the alternative splicing of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-15;class=Gene" target="_blank">egl-15</a></em></span> and of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=let-2;class=Gene" target="_blank">let-2</a></em></span> (<a href="#bib51">Kuroyanagi et al., 2006</a>; <a href="#bib52">Kuroyanagi et al., 2007</a>; <a href="#bib88">Ohno et al., 2008</a>). They have discovered <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=asd-1;class=Gene" target="_blank">asd-1</a></em></span> (<span class="underline">a</span>lternative <span class="underline">s</span>plicing <span class="underline">d</span>efective) which is a homolog of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fox-1;class=Gene" target="_blank">fox-1</a></em></span> alternative splicing factor, as a key alternative splicing regulator for <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-15;class=Gene" target="_blank">egl-15</a></em></span> that works through UGCAUG intronic elements (<a href="#bib51">Kuroyanagi et al., 2006</a>). They demonstrated that <a href="http://www.wormbase.org/db/get?name=ASD-1;class=Protein" target="_blank">ASD-1</a> and <a href="http://www.wormbase.org/db/get?name=FOX-1;class=Protein" target="_blank">FOX-1</a> both affect the splicing of this reporter, and that they must have overlapping function since mutations in both genes are
               required to see a visible phenotype in the absence of the sensitive fluorescent splicing reporters. Kuroyanagi et al. also
               identified <a href="http://www.wormbase.org/db/get?name=SUP-12;class=Protein" target="_blank">SUP-12</a> as a regulator of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-15;class=Gene" target="_blank">egl-15</a></em></span> splicing using this screen and showed its affinity for GU-rich elements (<a href="#bib52">Kuroyanagi et al., 2007</a>). A screen for regulators of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=let-2;class=Gene" target="_blank">let-2</a></em></span> developmental alternative splicing regulation uncovered <span class="emphasis"><em>asd-2</em></span>, which is a STAR family RNA binding protein (<a href="#bib88">Ohno et al., 2008</a>). They demonstrated that <span class="emphasis"><em>asd-2</em></span> interacts with a CUAAC element in intron 10. This bi-color fluorescent screening system has proven powerful because it is
               designed to uncover genes directly responsible for splicing regulation of a specific target, and is sensitive enough to detect
               changes in reporter gene alternative splicing that do not otherwise cause detectable phenotypic changes in the worms. <a href="#table2" title="Table 2. C. elegans splicing factors described in this chapter. The closest human homolog of each is indicated in the second column.">Table 2</a> summarizes the splicing factors discussed in this chapter and indicates their closest human homologs.
            </p>
            <div class="table"><a name="table2"></a><p class="title"><span class="bold">Table 2.</span> <span class="emphasis"><em>C. elegans</em></span> splicing factors described in this chapter. The closest human homolog of each is indicated in the second column.
               </p>
               <table summary="Table 2. C. elegans splicing factors described in this chapter. The closest human homolog of each is indicated in the second column." border="1">
                  <colgroup>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr valign="bottom">
                        <th align="left" valign="bottom"><span class="bold"><strong><span class="emphasis"><em>C. elegans</em></span> Splicing Factor</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>Human Homolog</strong></span></th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=asd-1;class=Gene" target="_blank">asd-1</a></em></span></td>
                        <td align="left" valign="top">FOX1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em>asd-2 <a href="http://www.wormbase.org/db/get?name=star-2;class=Gene" target="_blank">star-2</a></em></span></td>
                        <td align="left" valign="top">QK1/Quaking</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=etr-1;class=Gene" target="_blank">etr-1</a></em></span></td>
                        <td align="left" valign="top">CUGBP1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=exc-7;class=Gene" target="_blank">exc-7</a></em></span></td>
                        <td align="left" valign="top">HuD/ELAV-like 4</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fox-1;class=Gene" target="_blank">fox-1</a></em></span></td>
                        <td align="left" valign="top">FOX2</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hrp-1;class=Gene" target="_blank">hrp-1</a></em></span></td>
                        <td align="left" valign="top">hnRNP A1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hrp-2;class=Gene" target="_blank">hrp-2</a></em></span></td>
                        <td align="left" valign="top">hnRNP Q/R</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em>hrpf-1</em></span></td>
                        <td align="left" valign="top">hnRNP F/H</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em>hrpf-2</em></span></td>
                        <td align="left" valign="top">hnRNP F/H</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span></td>
                        <td align="left" valign="top">RBPMS2</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ptb-1;class=Gene" target="_blank">ptb-1</a></em></span></td>
                        <td align="left" valign="top">PTB1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-1;class=Gene" target="_blank">rsp-1</a> <a href="http://www.wormbase.org/db/get?name=srp-5;class=Gene" target="_blank">srp-5</a></em></span></td>
                        <td align="left" valign="top">SRSF4/SRp75</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-2;class=Gene" target="_blank">rsp-2</a> <a href="http://www.wormbase.org/db/get?name=srp-4;class=Gene" target="_blank">srp-4</a></em></span></td>
                        <td align="left" valign="top">SRSF5/SRp40</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-3;class=Gene" target="_blank">rsp-3</a></em></span></td>
                        <td align="left" valign="top">SRSF1/SF2/ASF/SRp30a</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-4;class=Gene" target="_blank">rsp-4</a> <a href="http://www.wormbase.org/db/get?name=srp-2;class=Gene" target="_blank">srp-2</a></em></span></td>
                        <td align="left" valign="top">SRSF2/SC35/SRp30b</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-5;class=Gene" target="_blank">rsp-5</a> <a href="http://www.wormbase.org/db/get?name=srp-3;class=Gene" target="_blank">srp-3</a></em></span></td>
                        <td align="left" valign="top">SRSF2/SC35/SRp30b</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-6;class=Gene" target="_blank">rsp-6</a> <a href="http://www.wormbase.org/db/get?name=srp-1;class=Gene" target="_blank">srp-1</a></em></span></td>
                        <td align="left" valign="top">SRSF7/9G8</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-7;class=Gene" target="_blank">rsp-7</a></em></span></td>
                        <td align="left" valign="top">SFRS12/SRrp86</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-8;class=Gene" target="_blank">rsp-8</a></em></span></td>
                        <td align="left" valign="top">TRA2</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=smu-1;class=Gene" target="_blank">smu-1</a></em></span></td>
                        <td align="left" valign="top">SMU1/fSAP57</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=smu-2;class=Gene" target="_blank">smu-2</a></em></span></td>
                        <td align="left" valign="top">RED</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em>snrp-27</em></span></td>
                        <td align="left" valign="top">tri-snRNP 27K</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sup-12;class=Gene" target="_blank">sup-12</a></em></span></td>
                        <td align="left" valign="top">RBMP38</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sym-2;class=Gene" target="_blank">sym-2</a></em></span></td>
                        <td align="left" valign="top">hnRNP F/H</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-75;class=Gene" target="_blank">unc-75</a></em></span></td>
                        <td align="left" valign="top">CUGBP3</td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <p>The combinatorial interactions of multiple protein factors with the <span class="emphasis"><em>cis</em></span>-elements surrounding a given alternative splicing event lead to an integrated splicing decision. Yet the mechanism of multifactorial
               splicing regulation is poorly understood. Using a splicing-sensitive DNA microarray, Barberan-Soler et al. assayed 352 <span class="emphasis"><em>C. elegans</em></span> alternative cassette exons for changes in embryonic splicing patterns between wild-type and 12 different strains carrying
               viable mutations in splicing factors. In addition to identifying additional substrates for regulation by splicing factors
               such as <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sup-12;class=Gene" target="_blank">sup-12</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=asd-1;class=Gene" target="_blank">asd-1</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fox-1;class=Gene" target="_blank">fox-1</a></em></span>, many alternative splicing events were uncovered which are regulated by multiple splicing factors (<a href="#bib7">Barberan-Soler et al., 2011</a>). Many splicing factors were shown to have the ability to behave as splicing repressors for some alternative cassette exons
               and as splicing activators for others. In addition, an example was uncovered in which the ability of a given alternative splicing
               factor to behave as an enhancer or repressor of a specific splicing event switches during development. The observation that
               splicing factors can change their effects on a substrate during development supports a model in which combinatorial effects
               of multiple factors, both constitutive and developmentally regulated ones, contribute to the overall splicing decision.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec9"></a>9.&nbsp;Coupling of alternative splicing to nonsense-mediated decay (AS-NMD)
                     </h2>
                  </div>
               </div>
            </div>
            <p>Alternative splicing can generate isoforms with differential ability to be substrates for the NMD pathway. An alternative
               splicing-sensitive DNA microarray assay was used to look for changes in the steady state levels of alternative isoforms in
               different NMD mutant backgrounds (<a href="#bib6">Barberan-Soler et al., 2009</a>). In embryos, 73 different alternative splicing events whose isoform ratios changed in an NMD mutant background were identified.
               Strikingly, 59% of these NMD-sensitive alternative splicing events did not introduce a premature termination codon, and thus
               should not have activated the NMD pathway. These were referred to as secondary targets of the NMD pathway. In animals, splicing
               factors are overrepresented in the genes that are regulated by AS-NMD (<a href="#bib55">Lareau et al., 2007</a>; <a href="#bib82">Morrison et al., 1997</a>; <a href="#bib84">Ni et al., 2007</a>; <a href="#bib128">Wollerton et al., 2004</a>), and 12 examples of alternatively spliced <span class="emphasis"><em>C. elegans</em></span> alternative splicing factors regulated by AS-NMD were identified in the microarray experiments (<a href="#bib6">Barberan-Soler et al., 2009</a>). Reports in different systems have shown that several splicing factors have an autoregulatory feedback loop that links AS
               to NMD (<a href="#bib118">Sureau et al., 2001</a>; <a href="#bib128">Wollerton et al., 2004</a>; <a href="#bib131">Zachar et al., 1994</a>). This autoregulation has been linked to regions of high sequence conservation in the pre-mRNA, stressing the evolutionary
               importance of this process (<a href="#bib56">Lareau et al., 2007</a>; <a href="#bib84">Ni et al., 2007</a>). The loss of NMD in <span class="emphasis"><em>C. elegans smg</em></span> mutant strains led to stabilization, and potentially the translation, of PTC-containing isoforms of splicing factor genes.
               Several of these PTC-containing mRNA isoforms, stabilized in the NMD mutants, were shown to enter the polysome pool (<a href="#bib6">Barberan-Soler et al., 2009</a>). Translation of the PTC-containing isoforms of splicing factors, for example translation of the PTC isoform of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rsp-6;class=Gene" target="_blank">rsp-6</a></em></span> SR protein, which would generate a protein with the N-terminal RRM but which would lack the C-terminal SR domain, could produce
               a dominant-negative splicing factor that could compete with the full-length SR protein for pre-mRNA binding but not activate
               splicing once bound. The translation of truncated splicing factor proteins in an NMD mutant background could explain the observed
               changes in alternative splicing of transcripts that are not predicted NMD substrates (<a href="#bib6">Barberan-Soler et al., 2009</a>).
            </p>
            <p>Analysis of alternative splicing coupled to NMD also uncovered PTC-containing transcripts that were inefficiently degraded
               by NMD, as well as PTC-containing mRNAs whose NMD efficiency changed during development. Using sucrose gradient polysome analysis,
               it was observed that PTC-containing mRNAs isoforms that are inefficiently degraded are enriched in monosomes (<a href="#bib6">Barberan-Soler et al., 2009</a>). Three different pathways that PTC-containing transcripts could follow were proposed; strong NMD, inefficient NMD and developmentally
               regulated NMD. Individual representatives of these three different classes are shown in <a href="#figure5">Figure 5</a>. While this figure shows examples of genes that undergo AS-NMD, the features that drive the transcripts to the different
               NMD pathways are unknown. A role for a <span class="emphasis"><em>trans</em></span>-acting factor, the RNA binding protein Pub1p, in protecting specific upstream open reading frame-containing mRNAs from the
               NMD pathway in yeast has been proposed (<a href="#bib100">Ruiz-Echevarria and Peltz, 2000</a>). Recent discoveries that NMD is turned off in cancer cells in their hypoxic microenvironment (<a href="#bib124">Wang et al., 2011</a>), downregulated by myc oncogene overexpression (<a href="#bib125">Wang et al., 2011</a>), and downregulated by expression of microRNA miR-128 in brain (<a href="#bib13">Bruno et al., 2011</a>; <a href="#bib36">Huang et al., 2011</a>) point to the fact the NMD regulation is likely an important mechanism for control of differentiation. Because NMD mutants
               are viable in <span class="emphasis"><em>C. elegans</em></span> but not in mice or flies (<a href="#bib4">Avery et al., 2011</a>; <a href="#bib74">Medghalchi et al., 2001</a>), <span class="emphasis"><em>C. elegans</em></span> is a valuable model system in which to study the interconnection of alternative splicing and the regulation of NMD in animals.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
                  <tr>
  <td align="center"><p><a href="Zahler_Pre-mRNA_fig5.jpg"><img src="Zahler_Pre-mRNA_fig5_s.jpg" border="2" align="middle" alt="figure 5"></a>
  </td>
                  </tr>
               </table>
               <div class="caption">
                  <p><b>Figure 5.&nbsp;</b>Schematic examples of substrates for AS coupled to NMD. Only the alternatively spliced regions to the end of the transcribed
                     region are shown. The splicing of the PTC-containing isoform is shown on top and the full length isoform splicing is shown
                     at the bottom. Position of the PTC and the full-length termination codon (TC) are indicated. The length of the genomic region,
                     from the beginning of the constitutive exon upstream of the alternative cassette exon to the end of the 3&#8242; UTR, is indicated
                     along with the class of NMD involved (strong, inefficient or regulated) (<a href="#bib6">Barberan-Soler et al., 2009</a>). Note that the features of these transcripts that determine which of the NMD classes the PTC-containing substrates will
                     follow are currently unknown.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec10"></a>10.&nbsp;Conclusions
                     </h2>
                  </div>
               </div>
            </div>
            <p>With extensive alternative splicing and a strong degree of regulation, <span class="emphasis"><em>C. elegans</em></span> offers a tractable model system for the study of pre-mRNA splicing regulation. The available bioinformatic, genetic and biochemical
               approaches have led to a coordinated system with the promise to unlock the splicing code for an organism. The availability
               of viable mutants in splicing factors along with viable mutants in components of the NMD pathways will allow for important
               contributions to our understanding of pre-mRNA splicing regulation. With many of the components of the splicing code in place
               for this organism, the next challenges lie in understanding the mechanisms by which the interaction of these factors with
               the pre-mRNA regulates the assembly of the spliceosome and ultimately the choice of splice sites.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec11"></a>11.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>I would like to thank Jim Kent, Massimo Caputi, Brock Roller, Sam Gu, Jennifer Kabat, MaryAnn Dassah, Sergio Barberan-Soler,
               Nicole Lambert and other members of my lab, past and present, who have been fantastic colleagues, and with whom I am truly
               fortunate to have had the privilege to work and learn. The journey of being a scientist is indeed the goal. This chapter is
               dedicated to the memory of David M. Prescott, 1926-2011, who taught me that valuable lesson.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec12"></a>12.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     <span class="bibliomisc">Allen, M. A., Hillier, L. W., Waterston, R. H., and Blumenthal, T. (2011). A global analysis of <span class="emphasis"><em>C. elegans</em></span> trans-splicing. Genome Res. <span class="emphasis"><em>21</em></span>, 255-264.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177958&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.113811.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     <span class="bibliomisc">Anyanful, A., Ono, K., Johnsen, R. C., Ly, H., Jensen, V., Baillie, D. L., and Ono, S. (2004). The RNA-binding protein <a href="http://www.wormbase.org/db/get?name=SUP-12;class=Protein" target="_blank">SUP-12</a> controls muscle-specific splicing of the ADF/cofilin pre-mRNA in <span class="emphasis"><em>C. elegans</em></span>. J. Cell Biol. <span class="emphasis"><em>167</em></span>, 639-647.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15545320&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.200407085" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ares, M., Jr., and Weiser, B. (1994). Rearrangement of snRNA structure during assembly and function of the spliceosome. Prog.
                        Nucleic Acid. Res. Mol. Biol. <span class="emphasis"><em>50</em></span>, 131-159.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7754032&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     <span class="bibliomisc">Avery, P., Vicente-Crespo, M., Francis, D., Nashchekina, O., Alonso, C. R., and Palacios, I. M. (2011). <span class="emphasis"><em>Drosophila</em></span> Upf1 and Upf2 loss of function inhibits cell growth and causes animal death in a Upf3-independent manner. RNA <span class="emphasis"><em>17</em></span>, 624-638.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21317294&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.2404211" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barash, Y., Calarco, J. A., Gao, W., Pan, Q., Wang, X., Shai, O., Blencowe, B. J., and Frey, B. J. (2010). Deciphering the
                        splicing code. Nature <span class="emphasis"><em>465</em></span>, 53-59.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20445623&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature09000" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barberan-Soler, S., Lambert, N. J., and Zahler, A. M. (2009). Global analysis of alternative splicing uncovers developmental
                        regulation of nonsense-mediated decay in <span class="emphasis"><em>C. elegans</em></span>. RNA <span class="emphasis"><em>15</em></span>, 1652-1660.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19617316&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.1711109" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barberan-Soler, S., Medina, P., Estella, J., Williams, J., and Zahler, A. M. (2011). Co-regulation of alternative splicing
                        by diverse splicing factors in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>39</em></span>, 666-674.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20805248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkq767" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barberan-Soler, S., and Zahler, A. M. (2008). Alternative splicing and the steady-state ratios of mRNA isoforms generated
                        by it are under strong stabilizing selection in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>25</em></span>, 2431-2437.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18718918&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msn181" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barberan-Soler, S., and Zahler, A. M. (2008). Alternative splicing regulation during <span class="emphasis"><em>C. elegans</em></span> development: splicing factors as regulated targets. PLoS Genet <span class="emphasis"><em>4</em></span>, e1000001.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18454200&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     <span class="bibliomisc">Blencowe, B. J. (2000). Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases. Trends
                        Biochem. Sci. <span class="emphasis"><em>25</em></span>, 106-110.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10694877&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0968-0004(00)01549-8" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     <span class="bibliomisc">Blumenthal, T. (2005). Trans-splicing and operons. <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.5.1, http://www.wormbook.org.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18050426&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     <span class="bibliomisc">Blumenthal, T., and Steward, K. (1997). RNA processing and gene structure. In <span class="emphasis"><em>C. elegans</em></span> II, D. L. Riddle, T. Blumenthal, B. J. Meyer, and J. R. Priess, eds. (Plainview, NY, Cold Spring Harbor Laboratory Press),
                        pp. 117-145.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21413218&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     <span class="bibliomisc">Bruno, I. G., Karam, R., Huang, L., Bhardwaj, A., Lou, C. H., Shum, E. Y., Song, H. W., Corbett, M. A., Gifford, W. D., Gecz,
                        J.<span class="emphasis"><em>, et al</em></span>. (2011). Identification of a microRNA that activates gene expression by repressing nonsense-mediated RNA decay. Mol. Cell
                        <span class="emphasis"><em>42</em></span>, 500-510.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21596314&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.molcel.2011.04.018" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     <span class="bibliomisc">Bruzik, J. P. (1996). Splicing glue: a role for SR proteins in trans splicing? Microb. Pathog. <span class="emphasis"><em>21</em></span>, 149-155.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8878012&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/mpat.1996.0050" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     <span class="bibliomisc">Burge, C. B., Tuschl, T., and Sharp, P. A. (1999). Splicing of precursors to mRNAs by the spliceosome. In The RNA World, 2nd
                        Edition, R. F. Gesteland, T. R. Cech, and J. F. Atkins, eds. (Cold Spring Harbor, NY, Cold Spring Harbor Laboratory Press),
                        pp. 525-560.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8811184&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     <span class="bibliomisc"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium (1998). Genome sequence of the nematode <span class="emphasis"><em>C. elegans</em></span>: a platform for investigating biology. Science <span class="emphasis"><em>282</em></span>, 2012-2018.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851916&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.282.5396.2012" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     <span class="bibliomisc">Caceres, J. F., and Krainer, A. R. (1997). Mammalian pre-mRNA splicing factors. In Eukaryotic mRNA Processing, A. R. Krainer,
                        ed. (Oxford, U.K., IRL Press), pp. 174-212.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8811184&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     <span class="bibliomisc">Calarco, J. A., Zhen, M., and Blencowe, B. J. (2011). Networking in a global world: establishing functional connections between
                        neural splicing regulators and their target transcripts. RNA <span class="emphasis"><em>17</em></span>, 775-791.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21415141&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.2603911" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     <span class="bibliomisc">Calixto, A., Ma, C., and Chalfie, M. (2010). Conditional gene expression and RNAi using MEC-8-dependent splicing in <span class="emphasis"><em>C. elegans</em></span>. Nat. Methods <span class="emphasis"><em>7</em></span>, 407-411.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20364149&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1445" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cartegni, L., and Krainer, A. R. (2002). Disruption of an SF2/ASF-dependent exonic splicing enhancer in SMN2 causes spinal
                        muscular atrophy in the absence of SMN1. Nat. Genet. <span class="emphasis"><em>30</em></span>, 377-384.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11925564&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng854" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     <span class="bibliomisc">Chou, M. Y., Rooke, N., Turck, C. W., and Black, D. L. (1999). hnRNP H is a component of a splicing enhancer complex that
                        activates a c-<span class="emphasis"><em>src</em></span> alternative exon in neuronal cells. Mol. Cell. Biol. <span class="emphasis"><em>19</em></span>, 69-77.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9858532&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     <span class="bibliomisc">Chou, M. Y., Underwood, J. G., Nikolic, J., Luu, M. H., and Black, D. L. (2000). Multisite RNA binding and release of polypyrimidine
                        tract binding protein during the regulation of <span class="emphasis"><em>c-src</em></span> neural-specific splicing. Mol. Cell <span class="emphasis"><em>5</em></span>, 949-957.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10911989&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)80260-9" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     <span class="bibliomisc">Dassah, M., Patzek, S., Hunt, V. M., Medina, P. E., and Zahler, A. M. (2009). A genetic screen for suppressors of a mutated
                        5&#8242; splice site identifies factors associated with later steps of spliceosome assembly. Genetics <span class="emphasis"><em>182</em></span>, 725-734.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19380478&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.109.103473" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     <span class="bibliomisc">Davies, A. G., Spike, C. A., Shaw, J. E., and Herman, R. K. (1999). Functional overlap between the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span> gene and five <span class="emphasis"><em>sym</em></span> genes in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>153</em></span>, 117-134.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10471705&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ellis, R., and Schedl, T. (2007). Sex determination in the germ line. <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.82.2, http://www.wormbook.org.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18050498&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     <span class="bibliomisc">Faustino, N. A., and Cooper, T. A. (2005). Identification of putative new splicing targets for ETR-3 using sequences identified
                        by systematic evolution of ligands by exponential enrichment. Mol. Cell. Biol. <span class="emphasis"><em>25</em></span>, 879-887.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15657417&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.25.3.879-887.2005" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     <span class="bibliomisc">Fetzer, S., Lauber, J., Will, C. L., and Luhrmann, R. (1997). The [U4/U6.U5] tri-snRNP-specific 27K protein is a novel SR
                        protein that can be phosphorylated by the snRNP-associated protein kinase. RNA <span class="emphasis"><em>3</em></span>, 344-355.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9085842&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     <span class="bibliomisc">Graham, P. L., and Kimble, J. (1993). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mog-1;class=Gene" target="_blank">mog-1</a></em></span> gene is required for the switch from spermatogenesis to oogenesis in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>133</em></span>, 919-931.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8462850&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     <span class="bibliomisc">Graham, P. L., Schedl, T., and Kimble, J. (1993). More <span class="emphasis"><em>mog</em></span> genes that influence the switch from spermatogenesis to oogenesis in the hermaphrodite germ line of <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Dev. Genet. <span class="emphasis"><em>14</em></span>, 471-484.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8111975&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/dvg.1020140608" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hallegger, M., Llorian, M., and Smith, C. W. (2010). Alternative splicing: global insights. FEBS J. <span class="emphasis"><em>277</em></span>, 856-866.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20082635&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1111/j.1742-4658.2009.07521.x" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hartmann, B., and Valcarcel, J. (2009). Decrypting the genome's alternative messages. Curr. Opin. Cell Biol. <span class="emphasis"><em>21</em></span>, 377-386.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19307111&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ceb.2009.02.006" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hodgkin, J., Papp, A., Pulak, R., Ambros, V., and Anderson, P. (1989). A new kind of informational suppression in the nematode
                        <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>123</em></span>, 301-313.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2583479&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hodgkin, J., Zellan, J. D., and Albertson, D. G. (1994). Identification of a candidate primary sex determination locus, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fox-1;class=Gene" target="_blank">fox-1</a></em></span>, on the X chromosome of <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Development <span class="emphasis"><em>120</em></span>, 3681-3689.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7821230&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hollins, C., Zorio, D. A., MacMorris, M., and Blumenthal, T. (2005). U2AF binding selects for the high conservation of the
                        <span class="emphasis"><em>C. elegans</em></span> 3&#8242; splice site. RNA <span class="emphasis"><em>11</em></span>, 248-253.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15661845&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.7221605" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                     <span class="bibliomisc">House, A. E., and Lynch, K. W. (2006). An exonic splicing silencer represses spliceosome assembly after ATP-dependent exon
                        recognition. Nat. Struct. Mol. Biol. <span class="emphasis"><em>13</em></span>, 937-944.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16998487&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nsmb1149" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                     <span class="bibliomisc">Huang, L., Lou, C. H., Chan, W., Shum, E. Y., Shao, A., Stone, E., Karam, R., Song, H. W., and Wilkinson, M. F. (2011). RNA
                        homeostasisgoverned by cell type-specific and branched feedback loops acting on NMD. Mol. Cell <span class="emphasis"><em>43</em></span>, 950-961.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21925383&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.molcel.2011.06.031" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                     <span class="bibliomisc">Irimia, M., Rukov, J. L., Penny, D., Garcia-Fernandez, J., Vinther, J., and Roy, S. W. (2008). Widespread evolutionary conservation
                        of alternatively spliced exons in <span class="emphasis"><em>Caenorhabditis</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>25</em></span>, 375-382.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18048400&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msm262" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                     <span class="bibliomisc">Jin, Y., Suzuki, H., Maegawa, S., Endo, H., Sugano, S., Hashimoto, K., Yasuda, K., and Inoue, K. (2003). A vertebrate RNA-binding
                        protein Fox-1 regulates tissue-specific splicing via the pentanucleotide GCAUG. EMBO J. <span class="emphasis"><em>22</em></span>, 905-912.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12574126&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/emboj/cdg089" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                     <span class="bibliomisc">Jurica, M. S., and Moore, M. J. (2003). Pre-mRNA splicing: awash in a sea of proteins. Mol. Cell <span class="emphasis"><em>12</em></span>, 5-14.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12887888&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(03)00270-3" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kabat, J. L., Barberan-Soler, S., McKenna, P., Clawson, H., Farrer, T., and Zahler, A. M. (2006). Intronic alternative splicing
                        regulators identified by comparative genomics in nematodes. PLoS Comput. Biol. <span class="emphasis"><em>2</em></span>, e86.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16839192&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pcbi.0020086" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kabat, J. L., Barberan-Soler, S., and Zahler, A. M. (2009). <a href="http://www.wormbase.org/db/get?name=HRP-2;class=Protein" target="_blank">HRP-2</a>, the <span class="emphasis"><em>Ceanorhabditis elegans</em></span> homolog of mammalian heterogeneous nuclear ribonucleoproteins Q and R, is an alternative splicing factor that binds to UCUAUC
                        splicing regulatory elements. J. Biol. Chem. <span class="emphasis"><em>284</em></span>, 28490-28497.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19706616&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1074/jbc.M109.023101" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kalsotra, A., and Cooper, T. A. (2011). Functional consequences of developmentally regulated alternative splicing. Nat. Rev.
                        Genet. <span class="emphasis"><em>12</em></span>, 715-729.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21921927&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg3052" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                     <span class="bibliomisc">Karolchik, D., Hinrichs, A. S., and Kent, W. J. (2011). The UCSC Genome Browser. Curr. Protoc. Hum. Genet. <span class="emphasis"><em>Chapter 18</em></span>, Unit18 16.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21975940&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kawano, T., Fujita, M., and Sakamoto, H. (2000). Unique and redundant functions of SR proteins, a conserved family of splicing
                        factors, in <span class="emphasis"><em>Ceanorhabditis elegans</em></span> development. Mech. Dev. <span class="emphasis"><em>95</em></span>, 67-76.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10906451&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0925-4773(00)00339-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kent, W. J., and Zahler, A. M. (2000). Conservation, regulation, synteny, and introns in a large-scale <span class="emphasis"><em>C. briggsae</em></span>-<span class="emphasis"><em>C. elegans</em></span> genomic alignment. Genome Res. <span class="emphasis"><em>10</em></span>, 1115-1125.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=0010958630&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.10.8.1115" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kent, W. J., and Zahler, A. M. (2000). The intronerator: exploring introns and alternative splicing in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>28</em></span>, 91-93.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=0010958630&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/28.1.91" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kerins, J. A., Hanazawa, M., Dorsett, M., and Schedl, T. (2010). <a href="http://www.wormbase.org/db/get?name=PRP-17;class=Protein" target="_blank">PRP-17</a> and the pre-mRNA splicing pathway are preferentially required for the proliferation versus meiotic development decision and
                        germline sex determination in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Dev. Dyn. <span class="emphasis"><em>239</em></span>, 1555-1572.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20419786&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/dvdy.22274" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kielkopf, C. L., Lucke, S., and Green, M. R. (2004). U2AF homology motifs: protein recognition in the RRM world. Genes Dev.
                        <span class="emphasis"><em>18</em></span>, 1513-1526.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15231733&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1206204" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kimble, J., and Crittenden, S. L. (2005). Germline proliferation and its control. <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.13.1, http://www.wormbook.org.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18050413&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kramer, A. (1996). The structure and function of proteins involved in mammalian pre-mRNA splicing. Annu. Rev. Biochem. <span class="emphasis"><em>65</em></span>, 367-409.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8811184&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.bi.65.070196.002055" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kuroyanagi, H., Kobayashi, T., Mitani, S., and Hagiwara, M. (2006). Transgenic alternative-splicing reporters reveal tissue-specific
                        expression profiles and regulation mechanisms <span class="emphasis"><em>in vivo</em></span>. Nat. Methods <span class="emphasis"><em>3</em></span>, 909-915.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17060915&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth944" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kuroyanagi, H., Ohno, G., Mitani, S., and Hagiwara, M. (2007). The Fox-1 family and <a href="http://www.wormbase.org/db/get?name=SUP-12;class=Protein" target="_blank">SUP-12</a> coordinately regulate tissue-specific alternative splicing <span class="emphasis"><em>in vivo</em></span>. Mol. Cell. Biol. <span class="emphasis"><em>27</em></span>, 8612-8621.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17923701&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.01508-07" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kuroyanagi, H., Ohno, G., Sakane, H., Maruoka, H., and Hagiwara, M. (2010). Visualization and genetic analysis of alternative
                        splicing regulation <span class="emphasis"><em>in vivo</em></span> using fluorescence reporters in transgenic <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Nat. Protoc. <span class="emphasis"><em>5</em></span>, 1495-1517.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20725066&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nprot.2010.107" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lander, E. S., Linton, L. M., Birren, B., Nusbaum, C., Zody, M. C., Baldwin, J., Devon, K., Dewar, K., Doyle, M., FitzHugh,
                        W.<span class="emphasis"><em>, et al</em></span>. (2001). Initial sequencing and analysis of the human genome. Nature <span class="emphasis"><em>409</em></span>, 860-921.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11237011&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35057062" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lareau, L. F., Brooks, A. N., Soergel, D. A., Meng, Q., and Brenner, S. E. (2007). The coupling of alternative splicing and
                        nonsense-mediated mRNA decay. Adv. Exp. Med. Biol. <span class="emphasis"><em>623</em></span>, 190-211.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18380348&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/978-0-387-77374-2_12" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lareau, L. F., Inada, M., Green, R. E., Wengrod, J. C., and Brenner, S. E. (2007). Unproductive splicing of SR genes associated
                        with highly conserved and ultraconserved DNA elements. Nature <span class="emphasis"><em>446</em></span>, 926-929.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17361132&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature05676" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lasda, E. L., and Blumenthal, T. (2011). Trans-splicing. Wiley Interdiscip. Rev. RNA <span class="emphasis"><em>2</em></span>, 417-434.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21957027&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lewis, B. P., Green, R. E., and Brenner, S. E. (2003). Evidence for the widespread coupling of alternative splicing and nonsense-mediated
                        mRNA decay in humans. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>100</em></span>, 189-192.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12502788&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0136770100" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed">
                     <span class="bibliomisc">Li, Q., Lee, J. A., and Black, D. L. (2007). Neuronal regulation of alternative pre-mRNA splicing. Nat. Rev. Neurosci. <span class="emphasis"><em>8</em></span>, 819-831.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17895907&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrn2237" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed">
                     <span class="bibliomisc">Li, Y., Breitling, R., Snoek, L. B., van der Velde, K. J., Swertz, M. A., Riksen, J., Jansen, R. C., and Kammenga, J. E. (2010).
                        Global genetic robustness of the alternative splicing machinery in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>186</em></span>, 405-410.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20610403&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.110.119677" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed">
                     <span class="bibliomisc">Licatalosi, D. D., and Darnell, R. B. (2010). RNA processing and its regulation: global insights into biological networks.
                        Nat. Rev. Genet. <span class="emphasis"><em>11</em></span>, 75-87.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20019688&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg2673" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed">
                     <span class="bibliomisc">Liu, Z., Luyten, I., Bottomley, M. J., Messias, A. C., Houngninou-Molango, S., Sprangers, R., Zanier, K., Kramer, A., and
                        Sattler, M. (2001). Structural basis for recognition of the intron branch site RNA by splicing factor 1. Science <span class="emphasis"><em>294</em></span>, 1098-1102.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11691992&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1064719" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed">
                     <span class="bibliomisc">Longman, D., Johnstone, I. L., and Caceres, J. F. (2000). Functional characterization of SR and SR-related genes in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. EMBO J. <span class="emphasis"><em>19</em></span>, 1625-1637.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10747030&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/emboj/19.7.1625" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed">
                     <span class="bibliomisc">Longman, D., McGarvey, T., McCracken, S., Johnstone, I. L., Blencowe, B. J., and Caceres, J. F. (2001). Multiple interactions
                        between SRm160 and SR family proteins in enhancer-dependent splicing and development of <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>11</em></span>, 1923-1933.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11747818&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(01)00589-9" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lopez, A. J. (1998). Alternative splicing of pre-mRNA: developmental consequences and mechanisms of regulation. Annu. Rev.
                        Genet. <span class="emphasis"><em>32</em></span>, 279-305.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9928482&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.genet.32.1.279" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed">
                     <span class="bibliomisc">Loria, P. M., Duke, A., Rand, J. B., and Hobert, O. (2003). Two neuronal, nuclear-localized RNA binding proteins involved
                        in synaptic transmission. Curr. Biol. <span class="emphasis"><em>13</em></span>, 1317-1323.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12906792&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(03)00532-3" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lundquist, E. A., and Herman, R. K. (1994). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span> gene of <span class="emphasis"><em>Ceanorhabditis elegans</em></span> affects muscle and sensory neuron function and interacts with three other genes: <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=smu-1;class=Gene" target="_blank">smu-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=smu-2;class=Gene" target="_blank">smu-2</a></em></span>. Genetics <span class="emphasis"><em>138</em></span>, 83-101.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8001796&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lundquist, E. A., Herman, R. K., Rogalski, T. M., Mullen, G. P., Moerman, D. G., and Shaw, J. E. (1996). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span> gene of <span class="emphasis"><em>C. elegans</em></span> encodes a protein with two RNA recognition motifs and regulates alternative splicing of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> transcripts. Development <span class="emphasis"><em>122</em></span>, 1601-1610.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8625846&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ma, L., and Horvitz, H. R. (2009). Mutations in the <span class="emphasis"><em>Ceanorhabditis elegans</em></span> U2AF large subunit <a href="http://www.wormbase.org/db/get?name=UAF-1;class=Protein" target="_blank">UAF-1</a> alter the choice of a 3&#8242; splice site in vivo. PLoS Genet. <span class="emphasis"><em>5</em></span>, e1000708.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19893607&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000708" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed">
                     <span class="bibliomisc">MacMorris, M. A., Zorio, D. A., and Blumenthal, T. (1999). An exon that prevents transport of a mature mRNA. Proc. Natl. Acad.
                        Sci. U.S.A. <span class="emphasis"><em>96</em></span>, 3813-3818.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10097120&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.96.7.3813" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed">
                     <span class="bibliomisc">Madhani, H. D., and Guthrie, C. (1994). Dynamic RNA-RNA interactions in the spliceosome. Annu. Rev. Genet. <span class="emphasis"><em>28</em></span>, 1-26.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7534458&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.ge.28.120194.000245" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib72"></a><p class="bibliomixed">
                     <span class="bibliomisc">Markovtsov, V., Nikolic, J. M., Goldman, J. A., Turck, C. W., Chou, M. Y., and Black, D. L. (2000). Cooperative assembly of
                        an hnRNP complex induced by a tissue-specific homolog of polypyrimidine tract binding protein. Mol. Cell. Biol. <span class="emphasis"><em>20</em></span>, 7463-7479.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11003644&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.20.20.7463-7479.2000" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib73"></a><p class="bibliomixed">
                     <span class="bibliomisc">Mazroui, R., Puoti, A., and Kramer, A. (1999). Splicing factor SF1 from <span class="emphasis"><em>Drosophila</em></span> and <span class="emphasis"><em>Caenorhabditis</em></span>: presence of an N-terminal RS domain and requirement for viability. RNA <span class="emphasis"><em>5</em></span>, 1615-1631.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10606272&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1017/S1355838299991872" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib74"></a><p class="bibliomixed">
                     <span class="bibliomisc">Medghalchi, S. M., Frischmeyer, P. A., Mendell, J. T., Kelly, A. G., Lawler, A. M., and Dietz, H. C. (2001). <span class="emphasis"><em>Rent1</em></span>, a <span class="emphasis"><em>trans</em></span>-effector of nonsense-mediated mRNA decay, is essential for mammalian embryonic viability. Hum. Mol. Genet. <span class="emphasis"><em>10</em></span>, 99-105.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11152657&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/hmg/10.2.99" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib75"></a><p class="bibliomixed">
                     <span class="bibliomisc">Merendino, L., Guth, S., Bilbao, D., Martinez, C., and Valcarcel, J. (1999). Inhibition of <span class="emphasis"><em>msl-2</em></span> splicing by Sex-lethal reveals interaction between U2AF35 and the 3&#8242; splice site AG. Nature <span class="emphasis"><em>402</em></span>, 838-841.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10617208&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/45602" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib76"></a><p class="bibliomixed">
                     <span class="bibliomisc">Michaud, S., and Reed, R. (1991). An ATP-independent complex commits pre-mRNA to the mammalian spliceosome assembly pathway.
                        Genes Dev. <span class="emphasis"><em>5</em></span>, 2534-2546.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1836445&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.5.12b.2534" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib77"></a><p class="bibliomixed">
                     <span class="bibliomisc">Michaud, S., and Reed, R. (1993). A functional association between the 5&#8242; and 3&#8242; splice site is established in the earliest
                        prespliceosome complex (E) in mammals. Genes Dev. <span class="emphasis"><em>7</em></span>, 1008-1020.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8504926&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.7.6.1008" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib78"></a><p class="bibliomixed">
                     <span class="bibliomisc">Milne, C. A., and Hodgkin, J. (1999). <a href="http://www.wormbase.org/db/get?name=ETR-1;class=Protein" target="_blank">ETR-1</a>, a homologue of a protein linked to myotonic dystrophy, is essential for muscle development in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Curr. Biol. <span class="emphasis"><em>9</em></span>, 1243-1246.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10556089&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(99)80504-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib79"></a><p class="bibliomixed">
                     <span class="bibliomisc">Min, H., Chan, R. C., and Black, D. L. (1995). The generally expressed hnRNP F is involved in a neural-specific pre-mRNA splicing
                        event. Genes Dev. <span class="emphasis"><em>9</em></span>, 2659-2671.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7590243&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.9.21.2659" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib80"></a><p class="bibliomixed">
                     <span class="bibliomisc">Min, H., Turck, C. W., Nikolic, J. M., and Black, D. L. (1997). A new regulatory protein, KSRP, mediates exon inclusion through
                        an intronic splicing enhancer. Genes Dev. <span class="emphasis"><em>11</em></span>, 1023-1036.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9136930&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.11.8.1023" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib81"></a><p class="bibliomixed">
                     <span class="bibliomisc">Moore, M. J., and Proudfoot, N. J. (2009). Pre-mRNA processing reaches back to transcription and ahead to translation. Cell
                        <span class="emphasis"><em>136</em></span>, 688-700.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19239889&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2009.02.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib82"></a><p class="bibliomixed">
                     <span class="bibliomisc">Morrison, M., Harris, K. S., and Roth, M. B. (1997). <span class="emphasis"><em>smg</em></span> mutants affect the expression of alternatively spliced SR protein mRNAs in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>94</em></span>, 9782-9785.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9275202&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.94.18.9782" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib83"></a><p class="bibliomixed">
                     <span class="bibliomisc">Mount, S. M., and Anderson, P. (2000). Expanding the definition of informational suppression. Trends Genet. <span class="emphasis"><em>16</em></span>, 157.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10729829&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0168-9525(99)01964-2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib84"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ni, J. Z., Grate, L., Donohue, J. P., Preston, C., Nobida, N., O'Brien, G., Shiue, L., Clark, T. A., Blume, J. E., and Ares,
                        M., Jr. (2007). Ultraconserved elements are associated with homeostatic control of splicing regulators by alternative splicing
                        and nonsense-mediated decay. Genes Dev. <span class="emphasis"><em>21</em></span>, 708-718.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17369403&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1525507" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib85"></a><p class="bibliomixed">
                     <span class="bibliomisc">Nicoll, M., Akerib, C. C., and Meyer, B. J. (1997). X-chromosome-counting mechanisms that determine nematode sex. Nature <span class="emphasis"><em>388</em></span>, 200-204.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9217163&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/40669" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib86"></a><p class="bibliomixed">
                     <span class="bibliomisc">Nilsen, T. W., and Graveley, B. R. (2010). Expansion of the eukaryotic proteome by alternative splicing. Nature <span class="emphasis"><em>463</em></span>, 457-463.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20110989&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature08909" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib87"></a><p class="bibliomixed">
                     <span class="bibliomisc">Oberstrass, F. C., Auweter, S. D., Erat, M., Hargous, Y., Henning, A., Wenter, P., Reymond, L., Amir-Ahmady, B., Pitsch, S.,
                        Black, D. L., and Allain, F. H. (2005). Structure of PTB bound to RNA: specific binding and implications for splicing regulation.
                        Science <span class="emphasis"><em>309</em></span>, 2054-2057.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16179478&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1114066" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib88"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ohno, G., Hagiwara, M., and Kuroyanagi, H. (2008). STAR family RNA-binding protein ASD-2 regulates developmental switching
                        of mutually exclusive alternative splicing in vivo. Genes Dev. <span class="emphasis"><em>22</em></span>, 360-374.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18230701&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1620608" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib89"></a><p class="bibliomixed">
                     <span class="bibliomisc">Pan, Q., Shai, O., Lee, L. J., Frey, B. J., and Blencowe, B. J. (2008). Deep surveying of alternative splicing complexity
                        in the human transcriptome by high-throughput sequencing. Nat. Genet. <span class="emphasis"><em>40</em></span>, 1413-1415.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18978789&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng.259" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib90"></a><p class="bibliomixed">
                     <span class="bibliomisc">Proudfoot, N. J. (2011). Ending the message: poly(A) signals then and now. Genes Dev. <span class="emphasis"><em>25</em></span>, 1770-1782.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21896654&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.17268411" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib91"></a><p class="bibliomixed">
                     <span class="bibliomisc">Pruitt, K. D., Tatusova, T., and Maglott, D. R. (2007). NCBI reference sequences (RefSeq): a curated non-redundant sequence
                        database of genomes, transcripts and proteins. Nucleic Acids Res. <span class="emphasis"><em>35</em></span>, D61-65.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17130148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkl842" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib92"></a><p class="bibliomixed">
                     <span class="bibliomisc">Pulak, R., and Anderson, P. (1993). mRNA surveillance by the <span class="emphasis"><em>Ceanorhabditis elegans smg</em></span> genes. Genes Dev. <span class="emphasis"><em>7</em></span>, 1885-1897.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8104846&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.7.10.1885" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib93"></a><p class="bibliomixed">
                     <span class="bibliomisc">Puoti, A., and Kimble, J. (1999). The <span class="emphasis"><em>Ceanorhabditis elegans</em></span> sex determination gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mog-1;class=Gene" target="_blank">mog-1</a></em></span> encodes a member of the DEAH-Box protein family. Mol. Cell. Biol. <span class="emphasis"><em>19</em></span>, 2189-2197.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10022905&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib94"></a><p class="bibliomixed">
                     <span class="bibliomisc">Puoti, A., and Kimble, J. (2000). The hermaphrodite sperm/oocyte switch requires the <span class="emphasis"><em>Ceanorhabditis elegans</em></span> homologs of PRP2 and PRP22. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>97</em></span>, 3276-3281.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10737793&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.97.7.3276" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib95"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ramani, A. K., Calarco, J. A., Pan, Q., Mavandadi, S., Wang, Y., Nelson, A. C., Lee, L. J., Morris, Q., Blencowe, B. J., Zhen,
                        M., and Fraser, A. G. (2011). Genome-wide analysis of alternative splicing in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Genome Res. <span class="emphasis"><em>21</em></span>, 342-348.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177968&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.114645.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib96"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ratsch, G., Sonnenburg, S., and Scholkopf, B. (2005). RASE: recognition of alternatively spliced exons in <span class="emphasis"><em>C. elegans</em></span>. Bioinformatics <span class="emphasis"><em>21 Suppl 1</em></span>, i369-i377.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15961480&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/bioinformatics/bti1053" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib97"></a><p class="bibliomixed">
                     <span class="bibliomisc">Rocak, S., and Linder, P. (2004). DEAD-box proteins: the driving forces behind RNA metabolism. Nat. Rev. Mol. Cell. Biol.
                        <span class="emphasis"><em>5</em></span>, 232-241.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14991003&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrm1335" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib98"></a><p class="bibliomixed">
                     <span class="bibliomisc">Rogalski, T. M., Gilchrist, E. J., Mullen, G. P., and Moerman, D. G. (1995). Mutations in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> gene responsible for body wall muscle defects in adult <span class="emphasis"><em>Ceanorhabditis elegans</em></span> are located in alternatively spliced exons. Genetics <span class="emphasis"><em>139</em></span>, 159-169.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7535716&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib99"></a><p class="bibliomixed">
                     <span class="bibliomisc">Roller, A. B., Hoffman, D. C., and Zahler, A. M. (2000). The allele-specific suppressor <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sup-39;class=Gene" target="_blank">sup-39</a></em></span> alters use of cryptic splice sites in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>154</em></span>, 1169-1179.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10757761&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib100"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ruiz-Echevarria, M. J., and Peltz, S. W. (2000). The RNA binding protein Pub1 modulates the stability of transcripts containing
                        upstream open reading frames. Cell <span class="emphasis"><em>101</em></span>, 741-751.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10892745&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(00)80886-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib101"></a><p class="bibliomixed">
                     <span class="bibliomisc">Rukov, J. L., Irimia, M., Mork, S., Lund, V. K., Vinther, J., and Arctander, P. (2007). High qualitative and quantitative
                        conservation of alternative splicing in <span class="emphasis"><em>Ceanorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>24</em></span>, 909-917.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17272679&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msm023" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib102"></a><p class="bibliomixed">
                     <span class="bibliomisc">Run, J. Q., Steven, R., Hung, M. S., van Weeghel, R., Culotti, J. G., and Way, J. C. (1996). Suppressors of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-73;class=Gene" target="_blank">unc-73</a></em></span> gene of <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>143</em></span>, 225-236.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8722777&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib103"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ruzanov, P., Jones, S. J., and Riddle, D. L. (2007). Discovery of novel alternatively spliced <span class="emphasis"><em>C. elegans</em></span> transcripts by computational analysis of SAGE data. BMC Genomics <span class="emphasis"><em>8</em></span>, 447.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18053145&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-8-447" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib104"></a><p class="bibliomixed">
                     <span class="bibliomisc">Saldi, T., Wilusz, C., MacMorris, M., and Blumenthal, T. (2007). Functional redundancy of worm spliceosomal proteins U1A and
                        U2B''. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>104</em></span>, 9753-9757.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17535930&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0701720104" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib105"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sanford, J. R., and Bruzik, J. P. (1999). Developmental regulation of SR protein phosphorylation and activity. Genes Dev.
                        <span class="emphasis"><em>13</em></span>, 1513-1518.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10385619&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.13.12.1513" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib106"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sanford, J. R., and Bruzik, J. P. (1999). SR proteins are required for nematode trans-splicing <span class="emphasis"><em>in vitro</em></span>. RNA <span class="emphasis"><em>5</em></span>, 918-928.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10411135&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1017/S1355838299990234" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib107"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sanford, J. R., Coutinho, P., Hackett, J. A., Wang, X., Ranahan, W., and Caceres, J. F. (2008). Identification of nuclear
                        and cytoplasmic mRNA targets for the shuttling protein SF2/ASF. PLoS ONE <span class="emphasis"><em>3</em></span>, e3369.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18841201&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pone.0003369" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib108"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sanford, J. R., Ellis, J., and Caceres, J. F. (2005). Multiple roles of arginine/serine-rich splicing factors in RNA processing.
                        Biochem. Soc. Trans. <span class="emphasis"><em>33</em></span>, 443-446.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15916537&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1042/BST0330443" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib109"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sanford, J. R., Wang, X., Mort, M., Vanduyn, N., Cooper, D. N., Mooney, S. D., Edenberg, H. J., and Liu, Y. (2009). Splicing
                        factor SFRS1 recognizes a functionally diverse landscape of RNA transcripts. Genome Res. <span class="emphasis"><em>19</em></span>, 381-394.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19116412&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.082503.108" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib110"></a><p class="bibliomixed">
                     <span class="bibliomisc">Selenko, P., Gregorovic, G., Sprangers, R., Stier, G., Rhani, Z., Kramer, A., and Sattler, M. (2003). Structural basis for
                        the molecular recognition between human splicing factors U2AF65 and SF1/mBBP. Mol. Cell <span class="emphasis"><em>11</em></span>, 965-976.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12718882&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(03)00115-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib111"></a><p class="bibliomixed">
                     <span class="bibliomisc">Shan, G., and Walthall, W. W. (2008). Copulation in <span class="emphasis"><em>C. elegans</em></span> males requires a nuclear hormone receptor. Dev. Biol. <span class="emphasis"><em>322</em></span>, 11-20.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18652814&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2008.06.034" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib112"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sibley, M. H., Johnson, J. J., Mello, C. C., and Kramer, J. M. (1993). Genetic identification, sequence, and alternative splicing
                        of the <span class="emphasis"><em>Ceanorhabditis elegans</em></span> alpha 2(IV) collagen gene. J. Cell Biol. <span class="emphasis"><em>123</em></span>, 255-264.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7691828&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.123.1.255" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib113"></a><p class="bibliomixed">
                     <span class="bibliomisc">Spartz, A. K., Herman, R. K., and Shaw, J. E. (2004). <a href="http://www.wormbase.org/db/get?name=SMU-2;class=Protein" target="_blank">SMU-2</a> and <a href="http://www.wormbase.org/db/get?name=SMU-1;class=Protein" target="_blank">SMU-1</a>, <span class="emphasis"><em>Ceanorhabditis elegans</em></span> homologs of mammalian spliceosome-associated proteins RED and fSAP57, work together to affect splice site choice. Mol. Cell.
                        Biol. <span class="emphasis"><em>24</em></span>, 6811-6823.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15254247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.24.15.6811-6823.2004" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib114"></a><p class="bibliomixed">
                     <span class="bibliomisc">Spike, C. A., Davies, A. G., Shaw, J. E., and Herman, R. K. (2002). <a href="http://www.wormbase.org/db/get?name=MEC-8;class=Protein" target="_blank">MEC-8</a> regulates alternative splicing of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> transcripts in <span class="emphasis"><em>C. elegans</em></span> hypodermal cells. Development <span class="emphasis"><em>129</em></span>, 4999-5008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12397108&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib115"></a><p class="bibliomixed">
                     <span class="bibliomisc">Spike, C. A., Shaw, J. E., and Herman, R. K. (2001). Analysis of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=smu-1;class=Gene" target="_blank">smu-1</a></em></span>, a gene that regulates the alternative splicing of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-52;class=Gene" target="_blank">unc-52</a></em></span> pre-mRNA in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Mol. Cell. Biol. <span class="emphasis"><em>21</em></span>, 4985-4995.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11438655&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.21.15.4985-4995.2001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib116"></a><p class="bibliomixed">
                     <span class="bibliomisc">Staknis, D., and Reed, R. (1994). SR proteins promote the first specific recognition of the pre-mRNA and are present together
                        with the U1 small nuclear ribonucleoprotein particle in a general splicing enhancer complex. Mol. Cell. Biol. <span class="emphasis"><em>14</em></span>, 7670-7682.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7935481&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib117"></a><p class="bibliomixed">
                     <span class="bibliomisc">Stein, L. D., Bao, Z., Blasiar, D., Blumenthal, T., Brent, M. R., Chen, N., Chinwalla, A., Clarke, L., Clee, C., Coghlan,
                        A.<span class="emphasis"><em>, et al</em></span>. (2003). The genome sequence of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>: a platform for comparative genomics. PLoS Biol <span class="emphasis"><em>1</em></span>, e45.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14624247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib118"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sureau, A., Gattoni, R., Dooghe, Y., Stevenin, J., and Soret, J. (2001). SC35 autoregulates its expression by promoting splicing
                        events that destabilize its mRNAs. EMBO J. <span class="emphasis"><em>20</em></span>, 1785-1796.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11285241&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/emboj/20.7.1785" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib119"></a><p class="bibliomixed">
                     <span class="bibliomisc">Thomas, J., Lea, K., Zucker-Aprison, E., and Blumenthal, T. (1990). The spliceosomal snRNAs of <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>18</em></span>, 2633-2642.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2339054&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/18.9.2633" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib120"></a><p class="bibliomixed">
                     <span class="bibliomisc">Underwood, J. G., Boutz, P. L., Dougherty, J. D., Stoilov, P., and Black, D. L. (2005). Homologues of the <span class="emphasis"><em>Ceanorhabditis elegans</em></span> Fox-1 protein are neuronal splicing regulators in mammals. Mol. Cell. Biol. <span class="emphasis"><em>25</em></span>, 10005-10016.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16260614&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.25.22.10005-10016.2005" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib121"></a><p class="bibliomixed">
                     <span class="bibliomisc">Valadkhan, S., and Jaladat, Y. (2010). The spliceosomal proteome: at the heart of the largest cellular ribonucleoprotein machine.
                        Proteomics <span class="emphasis"><em>10</em></span>, 4128-4141.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21080498&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/pmic.201000354" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib122"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wagner, E. J., and Garcia-Blanco, M. A. (2001). Polypyrimidine tract binding protein antagonizes exon definition. Mol. Cell.
                        Biol. <span class="emphasis"><em>21</em></span>, 3281-3288.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11313454&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.21.10.3281-3288.2001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib123"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wahl, M. C., Will, C. L., and Luhrmann, R. (2009). The spliceosome: design principles of a dynamic RNP machine. Cell <span class="emphasis"><em>136</em></span>, 701-718.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19239890&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2009.02.009" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib124"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wang, D., Wengrod, J., and Gardner, L. B. (2011). Over-expression of the c-<span class="emphasis"><em>myc</em></span> oncogene inhibits nonsense mediated RNA decay in B-lymphocytes. J. Biol. Chem. <span class="emphasis"><em>286</em></span>, 40038-40043.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21730287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1074/jbc.M111.266361" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib125"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wang, D., Zavadil, J., Martin, L., Parisi, F., Friedman, E., Levy, D., Harding, H., Ron, D., and Gardner, L. B. (2011). Inhibition
                        of nonsense-mediated RNA decay by the tumor microenvironment promotes tumorigenesis. Mol. Cell. Biol. <span class="emphasis"><em>31</em></span>, 3670-3680.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21969377&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.05704-11" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib126"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wang, E. T., Sandberg, R., Luo, S., Khrebtukova, I., Zhang, L., Mayr, C., Kingsmore, S. F., Schroth, G. P., and Burge, C.
                        B. (2008). Alternative isoform regulation in human tissue transcriptomes. Nature <span class="emphasis"><em>456</em></span>, 470-476.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18978772&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature07509" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib127"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wang, Z., and Burge, C. B. (2008). Splicing regulation: from a parts list of regulatory elements to an integrated splicing
                        code. RNA <span class="emphasis"><em>14</em></span>, 802-813.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18369186&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.876308" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib128"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wollerton, M. C., Gooding, C., Wagner, E. J., Garcia-Blanco, M. A., and Smith, C. W. (2004). Autoregulation of polypyrimidine
                        tract binding protein by alternative splicing leading to nonsense-mediated decay. Mol. Cell <span class="emphasis"><em>13</em></span>, 91-100.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14731397&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib129"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wu, S., Romfo, C. M., Nilsen, T. W., and Green, M. R. (1999). Functional recognition of the 3&#8242; splice site AG by the splicing
                        factor U2AF35. Nature <span class="emphasis"><em>402</em></span>, 832-835.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10617206&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/45996" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib130"></a><p class="bibliomixed">
                     <span class="bibliomisc">Yochem, J., Bell, L. R., and Herman, R. K. (2004). The identities of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sym-2;class=Gene" target="_blank">sym-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sym-3;class=Gene" target="_blank">sym-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sym-4;class=Gene" target="_blank">sym-4</a></em></span>, three genes that are synthetically lethal with <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-8;class=Gene" target="_blank">mec-8</a></em></span> in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>168</em></span>, 1293-1306.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15579686&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.029827" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib131"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zachar, Z., Chou, T. B., Kramer, J., Mims, I. P., and Bingham, P. M. (1994). Analysis of autoregulation at the level of pre-mRNA
                        splicing of the suppressor-of-white-apricot gene in <span class="emphasis"><em>Drosophila</em></span>. Genetics <span class="emphasis"><em>137</em></span>, 139-150.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8056305&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib132"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zahler, A. M., Tuttle, J. D., and Chisholm, A. D. (2004). Genetic suppression of intronic +1G mutations by compensatory U1
                        snRNA changes in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>167</em></span>, 1689-1696.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15342508&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.028746" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib133"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zamore, P. D., Patton, J. G., and Green, M. R. (1992). Cloning and domain structure of the mammalian splicing factor U2AF.
                        Nature <span class="emphasis"><em>355</em></span>, 609-614.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1538748&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/355609a0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib134"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zanetti, S., Meola, M., Bochud, A., and Puoti, A. (2011). Role of the <span class="emphasis"><em>C. elegans</em></span> U2 snRNP protein <a href="http://www.wormbase.org/db/get?name=MOG-2;class=Protein" target="_blank">MOG-2</a> in sex determination, meiosis, and splice site selection. Dev. Biol. <span class="emphasis"><em>354</em></span>, 232-241.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21504747&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2011.04.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib135"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zhou, H. L., and Lou, H. (2008). Repression of prespliceosome complex formation at two distinct steps by Fox-1/Fox-2 proteins.
                        Mol. Cell. Biol. <span class="emphasis"><em>28</em></span>, 5507-5516.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18573872&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.00530-08" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib136"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zorio, D. A., and Blumenthal, T. (1999). Both subunits of U2AF recognize the 3&#8242; splice site in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>402</em></span>, 835-838.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10617207&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/45597" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib137"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zorio, D. A., and Blumenthal, T. (1999). U2AF35 is encoded by an essential gene clustered in an operon with RRM/cyclophilin
                        in <span class="emphasis"><em>Ceanorhabditis elegans</em></span>. RNA <span class="emphasis"><em>5</em></span>, 487-494.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10199565&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1017/S1355838299982225" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib138"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zorio, D. A., Lea, K., and Blumenthal, T. (1997). Cloning of <span class="emphasis"><em>Caenorhabditis</em></span> U2AF65: an alternatively spliced RNA containing a novel exon. Mol. Cell. Biol. <span class="emphasis"><em>17</em></span>, 946-953.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9001248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e9" href="#d0e9">*</a></sup>Edited by Thomas Blumenthal. Last Revised November 11, 2011. Published March 21, 2012. This chapter should be cited as: Zahler,
                  A. M. Pre-mRNA splicing and its regulation in <span class="emphasis"><em>Caenorhabditis elegans</em></span> (March 21, 2012), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.31.2, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2012 Alan M. Zahler. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e34" href="#d0e34">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:zahler@ucsc.edu" target="_top">zahler@ucsc.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   </body>
</html>