<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Methods in cell biology
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#cell_ArticleID" title="Methods in cell biology&#xA;">
      <link rel="next" href="#d0e46" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="intromethodscellbiology.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="cell_ArticleID"></a>Methods in cell biology<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Shai</span> <span class="surname">Shaham<em><span class="remark">
                                 				<sup><a name="d0e33" href="#ftn.d0e33">&sect;</a></sup>
                                 			</span></em>
                           			</span>, <span class="lineage">ed.</span><br></span><span class="affiliation"><span class="address">The Rockefeller University, New York, NY 10021, USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=309.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Methods+in+cell+biology" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e46">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#d0e108">2. Visualizing cells and their components</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e116">2.1. Differential interference contrast (DIC or Nomarski) microscopy</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e218">2.2. Polarized light microscopy (Denise Flaherty and Guy Benian)</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e348">2.3. Fluorescence microscopy</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e830">2.4. Electron microscopy</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e1909">3. Protein-protein and protein-DNA interactions</a></span></dt>
               <dt><span class="sect1"><a href="#d0e2311">4. Specific methods in <span class="emphasis"><em>C. elegans</em></span> cell biology</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e2317">4.1. Endocytosis</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e2525">4.2. Chromatin cell biology (Gy&ouml;rgyi Csankovszki and Barbara Meyer)</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e3214">4.3. Programmed cell death (Barbara Conradt, Julia Hatzold, Claus Schertl)</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e3828">5. Embryonic cell culture</a></span></dt>
               <dt><span class="sect1"><a href="#d0e6366">6. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e46"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>Although <span class="emphasis"><em>C. elegans</em></span> is primarily touted for its facile genetics, there has been a burgeoning interest in studying cell biological processes in
               this organism. Strong genetics (<a href="#bib2">Brenner, 1974</a>; <a href="#bib21">Jorgensen and Mango, 2002</a>), the development of fluorescent protein tags (<a href="#bib3">Chalfie et al., 1994</a>; <a href="#bib38">Yang et al., 1996</a>; <a href="#bib40">Zhang et al., 2004</a>), the availability of RNA interference strategies to disrupt gene function (<a href="#bib11">Fire et al., 1998</a>; <a href="#bib35">Timmons and Fire, 1998</a>), and the ability to perform studies on primary cultures of embryonic cells (<a href="#bib4">Christensen et al., 2002</a>; <a href="#bib39">Zhang et al., 2002</a>), have all led to an increase in the number of cell biological problems addressed in the worm. Furthermore, the transparency
               of the organism affords a unique opportunity to study the roles of cell biological processes in a living multicellular animal.
            </p>
            <p>A serious obstacle to studying cell biological phenomena in <span class="emphasis"><em>C. elegans</em></span> is the small size of its cells. However, advances in imaging techniques have allowed faint signals to be significantly amplified,
               and small structures to be visualized, allowing examination of transport, export, and import processes, as well as examination
               of cytoskeletal and chromatin structure.
            </p>
            <p>Here we have compiled a set of protocols that broadly fit under the category of Cell Biology. We begin with a brief discussion
               of various microscopical techniques employed by <span class="emphasis"><em>C. elegans</em></span> researchers to study aspects of the cell. We then describe methods for studying protein-protein and protein-DNA interactions
               in <span class="emphasis"><em>C. elegans</em></span>. We also describe methods used to study specific cell biological problems (e.g. endocytosis, chromatin, programmed cell death).
               Finally, we conclude this chapter with a discussion of primary embryonic cell culture and its uses.
            </p>
            <p>Contributors of sections or protocols are acknowledged in parentheses following the section or protocol titles. In some cases,
               protocols are based on previously published work, which is then cited after the protocol title. The on-line format of this
               chapter should easily allow for revisions and additions to the protocols presented here. Researchers willing to share protocols
               not presented here or with comments on existing protocols are encouraged to submit these to <a href="mailto:shaham@rockefeller.edu" target="_top">shaham@rockefeller.edu</a>.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e108"></a>2.&nbsp;Visualizing cells and their components
                     </h2>
                  </div>
               </div>
            </div>
            <p>To study objects we must interact with them. Generally, the technique of interaction is determined by the size of the object.
               Thus, an object of macroscopic size can be studied by direct contact. However, microscopic objects, such as the cell and its
               organelles, must be studied with agents of similar or smaller size. Cells in <span class="emphasis"><em>C. elegans</em></span> are roughly 3&#8211;30 microns in diameter, thus, light with wavelength in the visible range (~500 nm) is an ideal interacting
               agent. The set up of light microscopes affords a resolution that is about half the wavelength of light employed. Thus, light
               microscopy is useful for examining cells and cellular substructures on the order of 200&#8211;300 nm or larger. However, vesicles
               (often 50 nm in diameter), and other objects of similar or smaller scale cannot be resolved using current setups. Although,
               in principle, light of very short wavelength (e.g. X-rays) could be used to resolve smaller cellular structures, such light
               is too energetic, damaging the cell upon contact. In addition, lenses to focus high energy photons do not exist. High resolution
               can be obtained using electron microscopy. Moving electrons in an electron microscope possess wavelengths on the order of
               0.3 nm. Using an electron microscope, the electrons can be used to form resolved images of cellular structures of about 3
               nm in size.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e116"></a>2.1.&nbsp;Differential interference contrast (DIC or Nomarski) microscopy
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Visible light can be used to examine <span class="emphasis"><em>C. elegans</em></span>, however, in general, bright field and phase-contrast microscopy offers little contrast- making cells and their major components
                  difficult to see. DIC microscopy, however, allows high contrast images to be formed, and is ideal for examining nuclei, nucleoli,
                  and granular structures within <span class="emphasis"><em>C. elegans</em></span> cells (<a href="#bib31">Sulston and Horvitz, 1977</a>; <a href="#bib32">Sulston et al., 1983</a>). In DIC microscopy light passes through a plane polarizer. The resulting polarized light is split into two components using
                  a prism. These components interact with the sample, and are combined above the objective using a second prism. Finally, the
                  transmitted light passes through an analyzer (essentially a polarizer that is polarized at 90 degrees with respect to the
                  original plane of polarization) and on to the observer. The interaction of polarized light with the sample changes its plane
                  of polarization. Thus, only light whose polarization is changed by the sample will be detected by the observer. In this way,
                  highly contrasted images are formed. It should be remembered that shadowing effects seen using DIC microscopy <span class="underline">do not</span> reflect three-dimensional features of the sample. Rather, they reflect the ability of the specimen to interact with polarized
                  light. Current microscopes allow the use of a 100X objective with DIC, giving high magnification and resolution. Some microscopes
                  are equipped with additional lenses, however, these do not afford better resolution.
               </p>
               <p>A protocol for mounting <span class="emphasis"><em>C. elegans</em></span> for observation using DIC microscopy is presented below. This protocol can be used to mount animals for all microscopic techniques
                  utilizing a compound microscope.
               </p>
               <div class="sidebar">
                  <p class="title"><b>Protocol 1: Mounting animals for observation with DIC optics (Monica Driscoll)</b></p>
                  <p><span class="underline">1. Preparation of Agar Pads:</span></p>
                  <p><span class="underline">a. Materials</span></p>
                  <div class="itemizedlist">
                     <ul type="disc">
                        <li>
                           <p>5% agar solution in water, melted and kept molten by placing the tube in a heat block at 65&deg;C</p>
                        </li>
                        <li>
                           <p>Pasteur pipette and bulb</p>
                        </li>
                        <li>
                           <p>2 glass slides with pieces of labeling tape (for example, Fisher #11-880-5-D) taped over both ends to serve as spacers</p>
                        </li>
                        <li>
                           <p>2 clean glass slides</p>
                        </li>
                     </ul>
                  </div>
                  <p><span class="underline">b. Preparation of Pads</span> (See <a href="#figure1">Figure 1</a>):
                  </p>
                  <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="cellfig1.jpg"><img src="cellfig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 1.&nbsp;</b>Preparing agar pads (Monica Driscoll).
                        </p>
                     </div>
                  </div>
                  <p>Place the two taped slides with a clean slide sandwiched between them on a flat surface. Using the Pasteur pipette, place
                     a drop of agar onto the clean slide. Cover the agar with another clean slide placed on top of the three slides in a perpendicular
                     fashion. Press gently so the agar drop is flattened to a circle about 0.4 mm thick (the thickness of the tape spacers). Avoid
                     getting bubbles in the agar since worms will get stuck in them. After the agar solidifies, gently pull out the taped slides,
                     then separate the remaining two slides by sliding one relative to the other. The agar pad should adhere to one of the slides
                     (usually the bottom one). Rest the slide, agar side up, on the bench top.
                  </p>
                  <p>Note: The agar pad should be prepared just before use so that it does not dry out. Alternatively, once you pull out the taped
                     slides, you can wrap the cross shaped slide-agar-slide in a piece of Saran wrap. This way you can keep the pad moist for a
                     couple of hours.
                  </p>
                  <p><span class="underline">2. Mounting Live Animals:</span></p>
                  <p>Place a 1&#8211;2 ul drop of M9 containing 10&#8211;25 mM sodium azide (NaN<sub>3</sub>) onto the center of the agar pad. NaN<sub>3</sub> anesthetizes the worms so that they will not move. The agar pad can also be prepared with anaesthetic included for a final
                     concentration of 2&#8211;10 mM NaN<sub>3</sub> in the agar, instead of in the drop. This generally results in faster anesthetic action. If live worms are needed, NaN<sub>3</sub> can be omitted and bacteria can be added to the drop to allow the worms to feed with little motion.
                  </p>
                  <p>Transfer animals to be observed into the drop. Animals can be transferred using a worm pick or an eyebrow hair fastened to
                     a toothpick with wood glue or clear nail polish. When the hair or pick is moved into the drop, animals float off easily. Alternatively,
                     animals can be spun down in M9 in an eppendorf tube at 1200&#8211;1500 rpm with slow acceleration/deceleration. After the supernatant
                     is discarded, the animals are suspended in 10 ul M9-NaN<sub>3</sub>. 2&#8211;3 ul of the solution with the animals can then be transferred to the pad.
                  </p>
                  <p>Gently lay a coverslip over the animals. Most animals will lie on their left or right sides. Embryos also generally assume
                     stereotypic orientations on agar pads. Between the 4-cell stage and 100&#8211;150 minutes the embryos display either the left or
                     right sides. At gastrulation (150 min) they turn from left to dorsal or from right to ventral. At 350&#8211;400 minutes they return
                     to display left or right sides (<a href="#bib32">Sulston et al., 1983</a>).
                  </p>
                  <p>Animals will stop moving within a minute or two. Animals in sodium azide can recover following incubation in sodium azide
                     of one-hour or less.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e218"></a>2.2.&nbsp;Polarized light microscopy (Denise Flaherty and Guy Benian)
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Polarized light can be used to examine repeated or crystalline structures. In a manner similar to DIC, plane polarized light
                  that passes through a specimen will interact with the specimen and change its plane of polarization, allowing light to be
                  detected through an analyzer located above the specimen and oriented at 90 degrees to the initial plane of polarization (for
                  a more complete description see <a href="http://www.microscopyu.com/articles/polarized/polarizedintro.html" target="_top">http://www.microscopyu.com/articles/polarized/ polarizedintro.html</a>).
               </p>
               <p>The muscles used for locomotion in <span class="emphasis"><em>C. elegans</em></span> reside in the body wall. In the adult, there are 95 spindle shaped cells divided among four quadrants just underlying a basement
                  membrane, hypodermis and cuticle. In each quadrant, the cells are arranged in interlocking pairs. In these muscle cells, myofilaments
                  form a lattice that is restricted to a narrow zone of ~1.5 microns, just underlying the basement membrane and hypodermis.
                  By polarized light microscopy, obvious striations are seen; bright (&#8220;birefringent&#8221;) A-bands alternate with dark I-bands; each
                  I-band contains a row of dense bodies, which are the analogs of Z-discs of vertebrate striated muscle (<a href="#figure2">Figure 2</a>). Because the striations lie at a slightly oblique angle with respect to the long axis of the worm, this muscle is called
                  &#8220;obliquely striated&#8221;. Polarized light is also useful for evaluating the second largest set of muscles, those in the pharynx.
                  Below we present a protocol for observing <span class="emphasis"><em>C. elegans</em></span> muscle using polarized light.
               </p>
               <div class="sidebar">
                  <p class="title"><b>Protocol 2: Observing <span class="emphasis"><em>C. elegans</em></span> muscle using polarized light (Denise Flaherty and Guy Benian)</b></p>
                  <p><span class="underline">Materials and Equipment</span></p>
                  <p>1. Healthy worms</p>
                  <p>2. Light microscope (<span class="underline">not</span> inverted) with:
                  </p>
                  <div class="itemizedlist">
                     <ul type="disc">
                        <li>
                           <p>light source</p>
                        </li>
                        <li>
                           <p>strain free objectives</p>
                        </li>
                        <li>
                           <p>condenser appropriate for bright-field and differential interference contrast microscopy (often labeled H/DIC)</p>
                        </li>
                        <li>
                           <p>an analyzer slider</p>
                        </li>
                        <li>
                           <p>A rotating stage. This would not be necessary if worms had an obliging habit of lying down on the slide, stick straight, at
                              a 45&deg; angle within the field of view.
                           </p>
                        </li>
                     </ul>
                  </div>
                  <p><span class="underline">Method</span></p>
                  <p>Pick one to several adult animals onto a clean glass slide (avoid dust) containing an 18ul drop of M9 buffer. Take care to
                     pick up as little bacteria as possible, as it could obscure the view. It is also often customary to add sodium azide to a
                     final concentration of 20 mM in order to slow the movement of the animal for imaging.
                  </p>
                  <p>Next, cover the animal with a number 1 coverglass, 22 by 40mm, and view the animal under the microscope using the following
                     procedure:
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Be sure that the analyzer slider is in place</p>
                        </li>
                        <li>
                           <p>Set the index of the condenser to &#8220;H/DIC&#8221;</p>
                        </li>
                        <li>
                           <p>Move any color or gray filters out of the way</p>
                        </li>
                        <li>
                           <p>For microscopes with a rotating polarizer, set the rotating polarizer lever on the condenser to 0&deg;.</p>
                        </li>
                        <li>
                           <p>Find the animal on the slide using a low power objective such as 10X.</p>
                        </li>
                        <li>
                           <p>Once the animal has been found, place it in the center of the field of view.</p>
                        </li>
                        <li>
                           <p>Carefully move up to a higher magnification objective such as 40X.</p>
                        </li>
                        <li>
                           <p>Get the animal into focus then close down the luminous field diaphragm so that the edges of the opening are in the field of
                              view.
                           </p>
                        </li>
                        <li>
                           <p>Change the height of the condenser until the edges of the opening are in crisp focus. Then use the condenser centering screws
                              to center the opening in the field of view.
                           </p>
                        </li>
                        <li>
                           <p>Open the luminous field diaphragm wide enough so that the edges go just beyond the field of view.</p>
                        </li>
                        <li>
                           <p>Rotate the stage until the muscle striations are most clearly seen. (This occurs when the striations are at a 45&deg; angle with
                              respect to the field of view).
                           </p>
                        </li>
                        <li>
                           <p>Change the level of light in the field by using the light source dial and/or the condenser diaphragm.</p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Notes</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>getting the right &#8220;balance&#8221; of light is the key to successful polarized light microscopy. As the light from the light source
                              is increased and the light coming through the condenser is decreased, stronger contrast will be achieved. Sometimes this contrast
                              is too sharp, making the muscle look &#8220;grainy&#8221;. If this is the case, begin to reduce the light from the light source and increase
                              the light coming through the condenser. If this is done too much, there will not be enough contrast. Tweak each source of
                              light carefully until the desired polarized light is achieved.
                           </p>
                        </li>
                        <li>
                           <p>Worms that are healthy adults without many developing embryos are the best for polarized light microscopy due to the interference
                              of these embryos over a quadrant of body wall muscle. Similarly, when a quadrant of body wall muscle happens to lie over the
                              reflected arm of the gonad, this is advantageous, since the gonad acts as an empty window through which the muscle can be
                              seen. Skilled practitioners of polarized light microscopy are able to gently roll the worm under the coverslip in order to
                              achieve this effect.
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="cellfig2.jpg"><img src="cellfig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2.&nbsp;</b>The top three panels show polarized light microscopy of adult body wall muscle from wild type and from two muscle-affecting
                        mutants. Each image was captured with a 40X objective and shows portions of about three muscle cells. The inset shows an enlarged
                        view. Wild type muscle shows a high degree of organization with birefringent A-bands alternating with dark I-bands, each containing
                        a row of dense bodies. <span class="emphasis"><em>unc-89(su75)</em></span> exhibits the phenotype of a typical <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-89" target="_blank">unc-89</a></em></span> mutant: reduced organization of the myofilament lattice with an almost &#8220;basketweave&#8221; pattern. <span class="emphasis"><em>unc-98(<a href="http://www.wormbase.org/db/get?name=sf19;class=Variation" target="_blank">sf19</a>)</em></span> displays disorganized myofibrils with characteristic birefringent &#8220;needles&#8221; at the ends of the muscle cells. The bottom panel
                        is an illustration showing the position of the four muscle quadrants (right) and a portion of a single muscle cell emphasizing
                        the myofilament lattice. The oval indicates the plane of view seen by polarized light (Denise Flaherty and Guy Benian).
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e348"></a>2.3.&nbsp;Fluorescence microscopy
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e351"></a>2.3.1.&nbsp;Fluorescent dyes
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Cellular structures and components can be labeled with a variety of fluorescent dyes that can be visualized using a fluorescence
                     microscope. In response to incident light of specified wavelength (produced by a laser or using appropriate filters), fluorescent
                     dye molecules are excited. Decay of the molecule from the excited state to an intermediate excited state produces a photon
                     of a wavelength longer than that of the excitation source, which can easily be detected and separated, using appropriate filters,
                     from the incident light (for a more detailed description see <a href="http://www.microscopyu.com/articles/fluorescence/fluorescenceintro.html" target="_top">http://www.microscopyu.com/articles/fluorescence/fluorescenceintro.html</a>). Below are a number of protocols for using dyes to visualize DNA, proteins and cellular structures. Dyes coupled to antibodies
                     are particularly useful for visualizing the presence and localization of specific proteins within the cell, and several methods
                     for antibody staining are discussed in the chapter on gene expression. Often it is of interest to study whether two cellular
                     molecules physically interact. Fluorescence resonance energy transfer (FRET) has been used to indicate the proximity of two
                     labeled fluorophores. However, this method has not been significantly used in <span class="emphasis"><em>C. elegans</em></span> to date.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e362"></a>2.3.1.1.&nbsp;Visualizing DNA
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>A number of fluorescent dyes can be used to visualize DNA in fixed animals. 4',6'-diamidino-2-phenylindole hydrochloride (DAPI),
                        is by far the most commonly used DNA dye in <span class="emphasis"><em>C. elegans</em></span> (<a href="#figure3">Figure 3</a>).
                     </p>
                     <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="cellfig3.jpg"><img src="cellfig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 3.&nbsp;</b>Adult hermaphrodite stained with DAPI (Jim McCarter and Tim Schedl).
                           </p>
                        </div>
                     </div>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 3: Staining animals with 4',6'-diamidino-2-phenylindole hydrochloride (DAPI) to visualize DNA</b></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Animals are fixed by any of the methods used for antibody staining and left to incubate in a neutral buffer such as PBS or
                                    Tris-Triton.
                                 </p>
                              </li>
                              <li>
                                 <p>To visualize the DNA, samples are incubated with 100 ng/ml of DAPI in the neutral buffer of incubation for about 30 minutes.
                                    Samples are washed 3X in buffer, and mounted for observation either on an agar pad or directly onto glass slides and covered
                                    with a cover slip.
                                 </p>
                              </li>
                              <li>
                                 <p>DAPI is excited by UV light (310 nm) and emits fluorescence light in a broad peak centered around 430 nm.</p>
                              </li>
                           </ol>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e398"></a>2.3.1.2.&nbsp;Labeling exposed neurons with dyes
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Amphid and phasmid neurons can take up lipophilic dyes from the environment (<a href="#bib19">Hedgecock et al., 1985</a>; <a href="#bib28">Perkins et al., 1986</a>). These dyes will label all parts of the neuron. The mechanism of dye uptake is unclear, but seems to correlate with some
                        aspect of neuronal function. Thus, these dyes can be used to visualize neurons and perhaps to study their physiology (<a href="#figure4">Figure 4</a>). The dye fluorescein isothiocyanate (FITC) can be used to label the ADF, ASH, ASI, ASJ, ASK, and ADL cells of the amphid
                        sensilla, and the PHA and PHB neurons of the phasmid sensillum. DiI, DiO, and DiD (Molecular Probes) can be used to visualize
                        the ASI, ADL, ASK, AWB, ASH, and ASJ amphid neurons, as well as the PHA and PHB phasmid neurons (<a href="#bib5">Collet et al., 1998</a>).
                     </p>
                     <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="cellfig4.jpg"><img src="cellfig4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 4.&nbsp;</b>DiI staining of amphid (top) and phasmid (bottom) neurons (Zeynep Altun and Dave Hall).
                           </p>
                        </div>
                     </div>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 4: Staining amphid and phasmid neurons using FITC (based on <a href="#bib19">Hedgecock et al., 1985</a>)</b></p>
                        <p>A stock dye solution containing 20 mg/ml 5-fluorescein isothiocyanate (FITC) in dimethyl formamide can be stored at <span class="symbol">&#8722;</span>20&deg;C indefinitely.
                        </p>
                        <p>50 ul of stock solution is mixed with 200 ul of M9 buffer and applied evenly to the surface of a 10 ml NGM plate preseeded
                           with a lawn of <span class="emphasis"><em>E. coli</em></span>. After two hours, to allow the dye to diffuse into the agar (final concentration 0.1 mg/ml), live animals are transferred
                           to the plate. After staining for 2 hours to overnight, the animals are transferred to a plate devoid of dye for at least 10
                           min to remove free FITC from the intestine.
                        </p>
                        <p>Animals are mounted for observation as described for DIC microscopy. FITC excitation wavelength maximum: 494 nm, emission
                           wavelength maximum: 520 nm.
                        </p>
                     </div>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 5: Staining amphid and phasmid neurons using FITC, DiI, DiO, DiD</b></p>
                        <p>A stock solution of 2 mg/ml FITC in dimethyl formamide is stored at <span class="symbol">&#8722;</span>20&deg; in a foil wrapped tube.
                        </p>
                        <p>Dilute the stock 1:200 in M9. Some dye will precipitate when you do this; don't worry about it. Put 150 &#956;l stain in a microtiter
                           well or eppendorf tube, and use a worm pick to transfer some worms into the dye. Alternatively, wash a plate of worms with
                           M9, concentrate to about 50 &#956;l and add 100 &#956;l dye. Incubate 2&#8211;3 hours at room temperature. Wild-type animals will begin to
                           take up dye within less than an hour.
                        </p>
                        <p>Transfer worms to a fresh plate, and let crawl on a bacterial lawn for about 1 hour to destain. Alternatively, wash the worms
                           3X in M9.
                        </p>
                        <p>Animals are mounted for observation as described for DIC microscopy.</p>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e463"></a>2.3.1.3.&nbsp;Visualizing actin
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Filamentous actin can be readily observed in <span class="emphasis"><em>C. elegans</em></span> muscle by staining with fluorescin-conjugated phalloidin (<a href="#bib30">Strome, 1986</a>). Other actin structures can also be visualized using this method, however, Protocol 7, below, should be used.
                     </p>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 6: Phalloidin staining of F-actin (based on <a href="#bib30">Strome, 1986</a>)</b></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Rinse worms off a plate with S basal, spin briefly in a clinical centrifuge, rinse once with more S basal to remove bacteria.</p>
                              </li>
                              <li>
                                 <p>Put 10 ul of worms in an eppendorf tube, freeze in liquid nitrogen, then immediately place in a speedvac to lyophilize the
                                    worms (takes ~5 min). Add 3&#8211;4 drops ice cold acetone, wait 5 min, remove as much of the acetone as possible and air dry the
                                    remaining acetone. Worms can be stored in this way prior to staining.
                                 </p>
                              </li>
                              <li>
                                 <p>Put 2 U fluorescein-conjugated phalloidin (Molecular Probes) in an eppendorf tube. Speedvac off the methanol to dryness. Add
                                    20 ul &#8220;S mix&#8221; to dissolve the phalloidin, and add this to the dry worms. Let the worms stain at room temp (in the dark to
                                    prevent bleaching) for 0.5&#8211;1 hour.
                                 </p>
                              </li>
                              <li>
                                 <p>Wash the worms 2X in 1 ml of PBBT (PBS + 0.5% BSA+0.5% Tween-20).</p>
                              </li>
                              <li>
                                 <p>Resuspend the worms in 20 ul of 2 ug/ml DAPI in PBS (DAPI is kept frozen as a 1 mg/ml stock). Let the worms sit &gt;5 min.</p>
                              </li>
                              <li>
                                 <p>For viewing, mix a few ul of worm suspension with an equal volume of mounting solution (90% glycerol, 10% PBS, 1 mg/ml phenlyenediamine).</p>
                              </li>
                              <li>
                                 <p>Viewing: illuminating for DAPI fluorescence bleaches the FITC rapidly, so be careful.</p>
                              </li>
                           </ol>
                        </div>
                        <p><span class="underline">S mix</span> (1 ml)
                        </p>
                        <p>743 ul dH20, 250 ul 0.8 M Na phosphate pH 7.5 (recipe: 8.1 ml 1M Na<sub>2</sub>HPO<sub>4</sub>, 1.9 ml 1M NaH<sub>2</sub>PO<sub>4</sub>, 2.5 ml H<sub>2</sub>0), 1 ul 1M MgCl2, 4 ul 1% SDS
                        </p>
                     </div>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 7: Phalloidin staining of F-actin in embryos (Vida Praitis, based on <a href="#bib6">Costa et al., 1997</a>)</b></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Grow up worms to gravid adult stage on high density or NGM plates. Wash off worms with M9. Add 10 ml M9 to the plates and
                                    use a rubber policeman to gently dislodge the eggs from the plates. Pipette eggs into 15 ml tubes. If you want earlier stage
                                    eggs, save adults and bleach, but this is not necessary for morphogenesis-stage embryos and the bleach can interfere with
                                    staining.
                                 </p>
                              </li>
                              <li>
                                 <p>Pellet eggs. Resuspend in ~10 ml M9 with 0.01% Tween 20. Remove supernatant. Repeat for 3 washes total. (The eggs are very
                                    sticky, so include tween or another mild detergent in the M9 to prevent them from sticking to the sides of the tube). Resuspend
                                    in 2 ml culture media.
                                 </p>
                              </li>
                              <li>
                                 <p>Pull up sample with 21g needle in 3 ml syringe. Push out with a 30 g needle. Repeat for total 3 X.</p>
                              </li>
                              <li>
                                 <p>Wash 1X with 1 ml 1X ES/Hepes (no tween). Transfer to a siliconized 1.5 ml eppendorf tube. (The siliconized tube coating helps
                                    prevent eggs from sticking; siliconized tubes can be purchased or made by dipping tubes in 2% dimethyldichlorosilane in 1,1,1-trichloroethane
                                    and drying).
                                 </p>
                              </li>
                              <li>
                                 <p>Pull off supernatant to 120 &#956;l. Add a small amount of Chitinase. Incubate ~10 minutes. Watch eggs on coverslip. Tadpoles should
                                    hatch out of eggshells.
                                 </p>
                              </li>
                              <li>
                                 <p>Add 680 &#956;l of FIX 1 directly to egg sample. Incubate 2 minutes. Spin for 30 s in small microfuge.</p>
                              </li>
                              <li>
                                 <p>Remove Fix 1 and add 1000 &#956;l of FIX 2. Incubate 15 - 20 minutes. Wash 3X with 1 ml culture media. Let eggs settle after 3rd
                                    wash. Remove supernatant.
                                 </p>
                              </li>
                              <li>
                                 <p>Dry a 10 &#956;l sample of Alexa-Phalloidin in the dark (molecular probes). Resuspend Alexa -Phalloidin in 300 &#956;l of 1X PBS.</p>
                              </li>
                              <li>
                                 <p>Add 300 &#956;l Alexa -Phalloidin in PBS to egg sample. Incubate 1 hour in dark.</p>
                              </li>
                              <li>
                                 <p>Wash 3 X with 1 ml culture media.</p>
                              </li>
                              <li>
                                 <p>Pull off supernatant to ~30 &#956;l. Add 30 &#956;l glycerol/anti-bleach. Mix. Spin.</p>
                              </li>
                              <li>
                                 <p>Place 4 &#956;l of pellet/glycerol on slide. Add coverslip. Press very gently between paper towels to remove excess glycerol.</p>
                              </li>
                              <li>
                                 <p>Seal coverslip with nail polish.</p>
                              </li>
                           </ol>
                        </div>
                        <div class="table"><a name="table1"></a><p class="title">Reagents</p>
                           <table summary="Reagents" border="1">
                              <colgroup>
                                 <col align="left">
                                 <col align="center">
                              </colgroup>
                              <tbody>
                                 <tr>
                                    <td align="left">4X Egg Salts</td>
                                    <td align="center">&nbsp;</td>
                                 </tr>
                                 <tr>
                                    <td align="left">472 mM NaCl</td>
                                    <td align="center">5.517 g</td>
                                 </tr>
                                 <tr>
                                    <td align="left">160 mM KCl</td>
                                    <td align="center">2.386 g</td>
                                 </tr>
                                 <tr>
                                    <td align="left">13.6 mM CaCl<sub>2</sub>&middot;2H<sub>2</sub>0
                                    </td>
                                    <td align="center">0.400 g</td>
                                 </tr>
                                 <tr>
                                    <td align="left">13.6 mM MgCl<sub>2</sub>&middot;6H<sub>2</sub>0
                                    </td>
                                    <td align="center">0.553 g</td>
                                 </tr>
                                 <tr>
                                    <td align="left">dH<sub>2</sub>0
                                    </td>
                                    <td align="center">to 200 ml</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                        <p><span class="underline">1X Culture Media</span></p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td>1X Egg Salts</td>
                           </tr>
                           <tr>
                              <td>5 mM Hepes, pH 7.2</td>
                           </tr>
                           <tr>
                              <td>5% Fetal Calf Serum</td>
                           </tr>
                        </table>
                        <p><span class="underline">FIX 1</span> (Make up fresh)
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td>0.60 ml dH<sub>2</sub>0
                              </td>
                           </tr>
                           <tr>
                              <td>2.5 ml 2X Eggs Salts/10 mM Hepes</td>
                           </tr>
                           <tr>
                              <td>0.1 ml 0.5M EGTA</td>
                           </tr>
                           <tr>
                              <td>1.0 ml 16% Formaldehyde (high grade) or use paraformaldehyde</td>
                           </tr>
                           <tr>
                              <td>0.05 ml 10mg/ml Lysolecithin in PBS (L-&#956;-Lysophosphatidylcholine Sigma # L-4129)</td>
                           </tr>
                           <tr>
                              <td>4.25 ml Total Volume</td>
                           </tr>
                        </table>
                        <p><span class="underline">FIX 2</span> (Make up fresh)
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td>1.4 ml dH<sub>2</sub>0
                              </td>
                           </tr>
                           <tr>
                              <td>2.5 ml 2X Eggs Salts/10 mM Hepes</td>
                           </tr>
                           <tr>
                              <td>0.1 ml 0.5M EGTA</td>
                           </tr>
                           <tr>
                              <td>1.0 ml 16% Formaldehyde (high grade)</td>
                           </tr>
                           <tr>
                              <td>5.0 ml Total Volume</td>
                           </tr>
                        </table>
                        <p><span class="underline">Anti-bleach</span></p>
                        <p>Available from molecular probes.</p>
                        <p><span class="underline">Alexa-Phalloidin</span></p>
                        <p>We buy this reagent from Molecular Probes. Alexa dyes produce a more intense signal and do not bleach as rapidly as fluorescein
                           dyes.
                        </p>
                        <p>Samples will eventually deteriorate (I've used it successfully for &gt;1 year), so it is critical to date the reagents, and remember
                           to check the date prior to use.
                        </p>
                        <p>Siliconized Eppendorfs.</p>
                        <p>Chitinase Sigma # C-6137</p>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e687"></a>2.3.2.&nbsp;Fluorescent proteins
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The use of fluorescent proteins to tag cellular proteins has revolutionized cell biology (<a href="#bib41">Zhang et al., 2002</a>). In <span class="emphasis"><em>C. elegans</em></span>, green fluorescent protein (GFP) and its derivatives, and more recently DsRed, have been employed to visualize protein localization,
                     movement, and conformational changes <span class="emphasis"><em>in vivo</em></span> (<a href="#bib3">Chalfie et al., 1994</a>; <a href="#bib25">Miyawaki, 2002</a>). Fusions to any protein can be generated using sets of vectors constructed by Andy Fire and others (<a href="#bib24">Miller et al., 1999</a>). In addition, labeling of specific organelles can be accomplished using pre-made vectors containing GFP fused to specific
                     subcellular targeting sequences. Localization signals that are frequently used include those for the nucleus, mitochondria,
                     and plasma membrane. Details regarding fluorescent proteins can be found in the gene expression chapter.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e714"></a>2.3.3.&nbsp;Fluorescently-labeled antibodies
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Examining protein localization using GFP or DsRed fusion proteins requires injection of corresponding transgenes into animals.
                     Thus, a number of caveats exist in interpreting results from such experiments. For example, expression from transgene arrays
                     is often higher than from the endogenous locus. Furthermore, fluorescent proteins are fairly bulky, thus, fusion proteins
                     may not display appropriate localization. In addition, promoter elements used to drive expression of GFP transgenes are often
                     expressed in cells where the endogenous protein is normally not made, or <span class="emphasis"><em>vice versa</em></span>.
                  </p>
                  <p>Fluorescently-labelled antibodies generated against a protein of choice allow direct examination of endogenous unmodified
                     proteins, giving a clearer assessment of where endogenous expression is localized. Antibodies coupled to gold particles can
                     also be used for electron microscopy to examine subcellular localization in great detail. Antibodies can only label fixed
                     tissues, thus, they cannot be used for real time observation of protein localization dynamics. In addition, fixation often
                     destroys cellular structure, limiting the resolution obtained by immunostaining. Detailed protocols for use of antibodies
                     can be found in the gene expression chapter.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e724"></a>2.3.4.&nbsp;Microscopy of fluorescently-labelled cells
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A number of microscopes are now commercially available for generating and examining fluorescent signals. All are able to resolve
                     fluorescent entities both in space and in time. Differences among these microscopes center on spatial and temporal resolving
                     power.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e729"></a>2.3.4.1.&nbsp;Dissecting fluorescence microscope
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>This binocular microscope allows visualization of highly expressed fluorescent signals at low magnification. The major use
                        of the instrument is to visualize GFP and DsRed-labelled proteins in living worms on the surface of standard agar plates.
                        Generally, use of this microscope allows selection of animals expressing a fusion protein of interest, as well as examination
                        of very coarse cellular features (e.g. presence or absence of a neuronal process) for genetic screens (see for example <a href="#bib29">Shaham and Bargmann, 2002</a>). This method does not allow fine structure to be examined. The microscope consists of a standard binocular dissecting microscope
                        equipped with epifluorescence.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e738"></a>2.3.4.2.&nbsp;Compound fluorescent microscope
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>This microscope has significantly higher resolution and magnification compared to the dissecting fluorescence microscope,
                        and observation can be combined with DIC (<a href="#figure5">Figure 5</a>). Objectives of 100X are often used, and allow examination of gross subcellular localization of proteins. Images collected
                        from such a microscope at high magnification suffer from an extensive background of fluorescent light emitted by cells not
                        in the plane of focus and of scattered light. This often severely cuts down on the resolving power of this microscope. However,
                        for first pass examination of samples, or for examining highly localized signals, this microscope is often the instrument
                        of choice (for a more detailed description see <a href="http://www.microscopyu.com/articles/fluorescence/fluorescenceintro.html" target="_top">http://www.microscopyu.com/articles/fluorescence/fluorescenceintro.html</a>).
                     </p>
                     <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="243">
<tr>
  <td align="center"><p><a href="cellfig5.jpg"><img src="cellfig5_s.jpg" border="2" align="middle" alt=" figure 5"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 5.&nbsp;</b>Standard fluorescence microscope image of the amphid ASE neuron and sheath cell (top). DIC view of same image (bottom) (Elliot
                              Perens and Shai Shaham).
                           </p>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e761"></a>2.3.4.3.&nbsp;Deconvolution fluorescent microscope
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>This instrument is essentially a compound fluorescence microscope equipped with a sensitive light detection system. Light
                        collected from optical sections through the sample is processed using sophisticated computer software, which assigns out-of-focus
                        light to its correct focal plane (<a href="#bib36">Wallace et al., 2001</a>). Images generated by this microscope consist of optical sections through a sample, and allow 3-dimensional reconstruction
                        of the fluorescent signal with little background. Sections can also be projected onto a single image, giving highly resolved
                        artificial two-dimensional images (<a href="#figure6">Figure 6</a>). All images are viewed on a computer screen. Time-resolution of the technique is, in general, inversely proportional to
                        spatial resolution. Thus, the more optical sections imaged, the larger the time intervals between image collections at any
                        given focal plane. This type of microscope is not efficient for examining processes that occur over intervals of seconds or
                        faster.
                     </p>
                     <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="cellfig6.jpg"><img src="cellfig6_s.jpg" border="2" align="middle" alt=" figure 6"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 6.&nbsp;</b>Deconvolution microscope image of amphid sheath cell (red) and ASE neuron (green) (Max Heiman and Shai Shaham).
                           </p>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e785"></a>2.3.4.4.&nbsp;Laser scanning confocal microscope
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>This instrument consists of a compound microscope equipped with a laser for fluorophore excitation, and a special detection
                        set up consisting of a small pinhole through which emitted fluorescence light produced only near the focal plane of observation
                        can pass. Light passing through the pinhole is collected in a sensitive light detector, and images are produced on a computer
                        screen. This technique eliminates out-of-focus fluorescence, giving sharp images with little background (for more details
                        see <a href="http://www.microscopyu.com/articles/confocal/" target="_top">http://www.microscopyu.com/articles/confocal/</a>). As with the deconvolution microscope, images consist of optical sections through the sample. Time-resolution of this microscopy
                        is affected by several factors, including the time required for the laser to scan the field of view, and the amount of light
                        that needs to be collected to visualize fluorescence. Unlike the deconvolution microscope, which uses all the light emitted
                        by the sample to calculate the point of emission, a confocal microscope only collects light traveling through the pinhole,
                        necessitating longer exposures or more sensitive light-detection systems.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e793"></a>2.3.4.5.&nbsp;Spinning disc confocal microscope
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>This instrument is a variation of the laser scanning confocal microscope. Here, many pinholes are arranged in a spiral pattern
                        on a disc that can rotate at high speed. Thus, time resolution can be excellent (on the order of l00 msec or less), and photobleaching
                        is generally not a significant issue as with other microscopes (<a href="#bib26">Nakano, 2002</a>). Image display is essentially as with the standard confocal microscope (<a href="#figure7">Figure 7</a>).
                     </p>
                     <div class="mediaobject" align="center"><a name="figure7"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="cellfig7.jpg"><img src="cellfig7_s.jpg" border="2" align="middle" alt=" figure 7"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 7.&nbsp;</b>Spinning disc confocal microscope 2-dimensional projection of the amphid sheath cell (green) and the AWC and AWB amphidial
                              neurons (red) (Taulant Bacaj and Shai Shaham).
                           </p>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e817"></a>2.3.4.6.&nbsp;Multiphoton microscope
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>The output from this microscope is similar to that from deconvolution or confocal microscopes. Thus, optical slices of images
                        are generated that can be reconstructed to form a three dimensional image. Multiphoton microscopy eliminates out-of-focus
                        light by directing excitation to the focal plane. This is accomplished by shining photons of long wavelengths and at high
                        density onto the sample. At the focal plane long wavelength photons can superimpose to become, effectively, a shorter wavelength
                        photon. This photon can now excite the fluorophore and induce fluorescence. Superposition of photons occurs significantly
                        only at the focal plane. Multiphoton microscopy is also useful for looking at samples that are much thicker than confocal
                        or deconvolution samples, with nearly equivalent resolution (<a href="#bib20">Helmchen and Denk, 2002</a>; <a href="#bib23">Michalet et al., 2003</a>).
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e830"></a>2.4.&nbsp;Electron microscopy
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Electron microscopy (EM) is currently the method of choice for examining subcellular structures in <span class="emphasis"><em>C. elegans</em></span>. The resolution offered by the technique is unparalleled, allowing objects as small as, or even smaller than ribosomes to
                  be viewed. Although EM is technically demanding, it is well worth the effort if resolution of small structures is of importance.
                  Transmission EM (TEM) generally involves fixation of animals with any combination of gluteraldehyde, paraformaldehyde, or
                  osmium tetraoxide (OsO<sub>4</sub>), embedding of fixed animals in a special resin, sectioning animals (sections are usually 50&#8211;100 nm thick), staining, and
                  imaging on an electron microscope (<a href="#bib18">Hall, 1995</a>; <a href="#figure8">Figures 8</a> and <a href="#figure9">9</a>). Fixation conditions are generally the key to a successful EM experiment. Standard treatments with fixatives are adequate
                  for most applications, however, they have the disadvantage that cellular structures are often slightly distorted because of
                  osmotic imbalance and other damage that occurs during the fixation procedure. High Pressure Freezing (HPF) has recently been
                  employed to avoid these issues. Tissues and cells are generally rounder, and the ultrastructure seems to reflect the <span class="emphasis"><em>in vivo</em></span> structures more accurately (<a href="#figure10">Figure 10</a>). Because HPF fixation has only recently come to prominence in <span class="emphasis"><em>C. elegans</em></span>, a number of issues still remain to be addressed with the technique. Specifically, results are often more variable than using
                  standard fixation. In addition, embryos and L1 larvae seem to fix better than older stages. Interestingly, embryo fixation
                  using standard methods is fairly difficult, thus, HPF and standard techniques are somewhat complementary.
               </p>
               <div class="mediaobject" align="center"><a name="figure8"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="cellfig8.jpg"><img src="cellfig8_s.jpg" border="2" align="middle" alt=" figure 8"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 8.&nbsp;</b>TEM micrograph of a longitudonal section of the <span class="emphasis"><em>C. elegans</em></span> nose (Erika Hartwieg and Bob Horvitz).
                     </p>
                  </div>
               </div>
               <div class="mediaobject" align="center"><a name="figure9"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="cellfig9.jpg"><img src="cellfig9_s.jpg" border="2" align="middle" alt=" figure 9"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 9.&nbsp;</b>TEM micrograph of cross-section of <span class="emphasis"><em>C. elegans</em></span> nose (Erika Hartwieg and Bob Horvitz).
                     </p>
                  </div>
               </div>
               <div class="mediaobject" align="center"><a name="figure10"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="cellfig10.jpg"><img src="cellfig10_s.jpg" border="2" align="middle" alt=" figure 10"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 10.&nbsp;</b>TEM section through an embryo fixed using HPF (Rick Fetter and Cori Bargmann).
                     </p>
                  </div>
               </div>
               <p>Because fixation is so critical for good ultrastructure, we present below several alternative protocols for TEM. At the end
                  of the section we present a method for scanning EM (SEM) of worms. In this method, whole-mount animals or structures are viewed
                  by EM, offering both broad-scale and highly resolved images (<a href="#figure11">Figure 11</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure11"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="cellfig11.jpg"><img src="cellfig11_s.jpg" border="2" align="middle" alt=" figure 11"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 11.&nbsp;</b>SEM of a dissected <span class="emphasis"><em>C. elegans</em></span> gonad (David Greenstein).
                     </p>
                  </div>
               </div>
               <p>The protocols below are intended to give a sense of the types of fixation procedures used for EM and should be used as guidlines
                  for those with previous EM experience. These protocols are not intended for teaching EM from scratch.
               </p>
               <div class="sidebar">
                  <p class="title"><b>Protocol 8: Conventional two-step fixation for TEM (Dave Hall)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Wash animals in M9 and anaesthetize in 8% alcohol in M9.</p>
                        </li>
                        <li>
                           <p>Fix in 2.5% glutaraldehyde, 1% paraformaldehyde in 0.1M sucrose, 0.05M cacodylate on ice, 2 hours.</p>
                        </li>
                        <li>
                           <p>Rinse 3X in 0.2M cacodylate on ice, 10 min. each.</p>
                        </li>
                        <li>
                           <p>Fix in 0.5% OsO<sub>4</sub>, 0.5% KFe(CN)<sub>6</sub> in 0.1M cacodylate, 90 min. on ice.
                           </p>
                        </li>
                        <li>
                           <p>Wash 3X, 10 min. each, in 0.1 cacodylate; Wash 3X, 10 min. each in 0.1M NaAcetate.</p>
                        </li>
                        <li>
                           <p>Stain in 1% UAc in 0.1M NaAcetate, pH 5.2 at room temperature for 60 min.</p>
                        </li>
                        <li>
                           <p>Rinse 3X in 0.1 NaAcetate, 5 min. each.</p>
                        </li>
                        <li>
                           <p>Rinse 3X in distilled water, leave overnight, embed cut pieces in 3% seaplaque agarose.</p>
                        </li>
                        <li>
                           <p>Cut out small agarose blocks containing worms; should be large enough to be excluded from pipette tip, but small enough to
                              fit into embedding mold.
                           </p>
                        </li>
                        <li>
                           <p>Dehydrate through ethanol series: 5 min. in 30% EtOH, 5 min. in 50% EtOH, 5 min. in 75% EtOH, 5 min. in 95% EtOH, 3 X 10 min.
                              in 100% EtOH, 3 X 10 min. ethanol, 3 X 10 min in 100% propylene oxide; all at room temperature.
                           </p>
                           <p><span class="underline">Infiltration:</span></p>
                           <p>First change is in 1 part resin to 2 parts propylene oxide, allow to sit for 3 hours.</p>
                           <p>Second change is in 2 parts resin to 1 part propylene oxide, for 5 hours.</p>
                           <p>Then change into fresh resin at least 3&#8211;4 times, 2&#8211;4 hours per change.</p>
                        </li>
                        <li>
                           <p>Place samples in fresh resin at room temp; place samples in air-tight containers and agitate or rotate samples slowly. Rotation
                              or agitation of the samples helps with the infiltration.
                           </p>
                        </li>
                        <li>
                           <p>Place samples in fresh resin in embedding mold, and position carefully.</p>
                        </li>
                        <li>
                           <p>Cure samples in 60&deg;C oven for 3 days.</p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="bold"><strong>Resin formula</strong></span></p>
                  <p>45 ml Scipoxy 812. Other brands include Embed 812, Medcast, Epon</p>
                  <p>30 ml DDSA</p>
                  <p>25 ml NMA</p>
                  <p>Mix components well with wooden applicator, add accelerator, 1.5 ml DMP-30, and mix resin thoroughly again. Place resin into
                     dessicator and put under vacuum for 5&#8211;15 min to remove gasses from mixture before using
                  </p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 9: TEM fixation and sectioning (Erika Hartwieg and Bob Horvitz)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Fixation</p>
                           <p>Rinse worms off plates with M9, then substitute with fixative: 0.8% Glutaraldhyde, 0.8% OsO<sub>4</sub> in 0.1 M Cacodylate buffer pH7.4 (with HCl).
                           </p>
                           <p>Let sit on ice for 1 hr in the dark, cut worms open close to tissue of interest, rinse 3x with 0.1M Cacodylate buffer.</p>
                        </li>
                        <li>
                           <p>Fixation</p>
                           <p>2% OsO<sub>4</sub> in Cacodylate buffer overnight in fridge in wet chamber
                           </p>
                        </li>
                        <li>
                           <p>Embedding in agar</p>
                           <p>Place 5&#8211;6 worms in a very small drop of buffer, withdraw the buffer and quickly place an agar drop over the worms and align
                              worms with a hot worm-pick as closely as possible.
                           </p>
                        </li>
                        <li>
                           <p>Dehydration and infiltration of resin</p>
                           <div class="informaltable">
                              <table border="1">
                                 <colgroup>
                                    <col align="left">
                                    <col align="center">
                                 </colgroup>
                                 <tbody>
                                    <tr>
                                       <td align="left">enbloc staining in 1% Uranyl acetate in 50% alcohol</td>
                                       <td align="center">1hr</td>
                                    </tr>
                                    <tr>
                                       <td align="left">50% alcohol</td>
                                       <td align="center">10 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">70% alcohol</td>
                                       <td align="center">10 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">70% alcohol</td>
                                       <td align="center">20 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">80% alcohol</td>
                                       <td align="center">15 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">96% alcohol</td>
                                       <td align="center">20 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">100% alcohol</td>
                                       <td align="center">45 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">100% alcohol</td>
                                       <td align="center">45 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">Propylene Oxide/100% alcohol</td>
                                       <td align="center">20 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">Propylene</td>
                                       <td align="center">10 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">Propylene</td>
                                       <td align="center">10 min</td>
                                    </tr>
                                    <tr>
                                       <td align="left">25% Epon mixture/Propylene</td>
                                       <td align="center">2 hrs</td>
                                    </tr>
                                    <tr>
                                       <td align="left">50% Epon mixture/Propylene</td>
                                       <td align="center">4 hrs</td>
                                    </tr>
                                    <tr>
                                       <td align="left">Full strength Epon mixture</td>
                                       <td align="center">6&#8211;8 hrs</td>
                                    </tr>
                                    <tr>
                                       <td align="left">Freshly mixed full strength Epon mixture</td>
                                       <td align="center">8 hrs</td>
                                    </tr>
                                 </tbody>
                              </table>
                           </div>
                           <p><span class="underline">Epon mixture:</span></p>
                           <p>13.0 g Eponate 812</p>
                           <p>5.5 g NMA</p>
                           <p>7.5 g DDSA</p>
                           <p>0.5 ml DMP30</p>
                        </li>
                        <li>
                           <p>Embedding for blocks to be cut</p>
                           <p>Place gelatin blocks into flat transparent embedding molds, blocks can be aligned according to cutting requirements, place
                              mutant no. (label) into the mold.
                           </p>
                        </li>
                        <li>
                           <p>Polymerize the resin at 65&deg;C. for 48 hrs</p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Sectioning:</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Trim blocks and cut worms to the desired level with glass knives. Place 1 micron sections onto a coated grid and on to a drop
                              of water on a glass slide, place on hotplate and stain with 1 % tolluidin blue+ 15% borax to find desired level.
                           </p>
                        </li>
                        <li>
                           <p>Trim excess plastic around the worms into a trapazoid form and dab on top and bottom sides of the block some &#8220;sticky wax&#8221;
                              (wax found in graphic arts stores Charette ASTM D4236) especially for serial sections. Cut thin sections of light gray to
                              white color. Long continuous ribbon of sections can then be picked up with a coated grid, where the sections according to
                              the length of the slotted grid can be separated by eyelashes. Hold the ribbon of sections with eyelash in the left hand and
                              pick up sections by submerging the grid underneath the water surface in the trough and then slowly come up at an angle.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>Staining grids in a grid staining machine: 90 min in Stain No. 1</p>
                  <p>30 min in Stain No.2</p>
                  <p>Coating slotted grids: 0.5% Pioloform in very clean Chloroform</p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 10: Osmium + Aldehyde one-step fixation for TEM (Dave Hall)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Wash animals in M9 buffer and chill to 4&deg;C.</p>
                        </li>
                        <li>
                           <p>Fix in 0.5% Osmium, 1.25% glutaraldehyde in 0.1M HEPES, pH 7.4 + 0.14M sucrose, 4&deg;C [CUT OPEN IN FIX ASAP WITH RAZOR BLADE].
                              After 30 min. allow samples to warm, 40&#8211;120 min.
                           </p>
                        </li>
                        <li>
                           <p>Wash in 0.2M HEPES buffer, 0.1M NaAcetate.</p>
                        </li>
                        <li>
                           <p>Stain in 1% UAc in 0.1M NaAcetate, pH 5.2, room temperature for 45&#8211;75 min.</p>
                        </li>
                        <li>
                           <p>Wash in 0.1M NaAcetate, 2 X 5 min; in 0.2M HEPES.</p>
                        </li>
                        <li>
                           <p>Embed cut pieces in 3% agarose, positioning worms into clusters; allow to harden at 4&deg;C for 1 hour, or overnight under buffer.</p>
                        </li>
                        <li>
                           <p>Cut out small agarose blocks containing worms; should be large enough to be excluded from pipette tip, but small enough to
                              fit into embedding mold.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>Dehydrate through ethanol series; 1&#8211;5 min in 30% EtOH, 1&#8211;5 min in 50% EtOH, 1&#8211;5min in 75% EtOH, 2&#8211;5 min in 95% EtOH, 3 X 10
                     min in 100% EtOH, 3 X 10 min in propylene oxide.
                  </p>
                  <p><span class="underline">Infiltration</span></p>
                  <p>First change is in 1 part resin to 2 parts propylene oxide, allow to sit 3&#8211;4 hours or more.</p>
                  <p>Second change is in 2 parts resin to 1 part propylene oxide, for 3&#8211;4 hours or more, 1 part propylene oxide to 2 parts resin
                     overnight, rotating.
                  </p>
                  <p>The next day, place samples in fresh resin at room temperature; place samples in air-tight containers and agitate or rotate
                     samples slowly during infiltration. Then change into fresh resin at least 3&#8211;4 times, 2&#8211;4 hours per change.
                  </p>
                  <p>Embed samples in fresh resin in embedding mold, and position carefully.</p>
                  <p>Cure samples in 60&deg;C oven for 3 days</p>
                  <p><span class="bold"><strong>Resin formula</strong></span> [use pre-dessicated plastic beaker with gradations for easy measurements]:
                  </p>
                  <p>45 ml Scipoxy 812 Other brands include Embed 812, Medcast, Epon</p>
                  <p>30 ml DDSA</p>
                  <p>25 ml NMA</p>
                  <p>Mix components well with wooden applicator, add accelerator, 1.5 ml DMP-30 and mix resin thoroughly again. Place resin into
                     dessicator and put under vacuum for 5&#8211;15 min to remove gasses from mixture before using.
                  </p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 11: Microwave aldehyde fixation for TEM (Dave Hall)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Wash animals in M9 buffer.</p>
                        </li>
                        <li>
                           <p>Add fixative (2.5% glutaraldehyde, 1.0% paraformaldehyde in 0.1M HEPES) and microwave at &#8220;hotspot&#8221; according to the following
                              schedule: 2 min ON, 2 min OFF, 2 min ON, 2 min OFF, 2 min ON, refreshing ice bath between &#8220;ON&#8221; sessions to keep samples cold
                              throughout. After microwave bouts, continue soaking animals in same fix at room temp for one hour.
                           </p>
                        </li>
                        <li>
                           <p>Wash 3 &times; 10 min in 0.2M HEPES buffer.</p>
                        </li>
                        <li>
                           <p>Fix in 1% OsO<sub>4</sub>, 0.5% KFe(CN)<sub>6</sub> in 0.1M HEPES pH 5.6, room temperature for 60 min.
                           </p>
                        </li>
                        <li>
                           <p>Wash in 0.2M HEPES, 2 &times; 10 min; wash in 0.1M NaAcetate pH 5.2, 2 &times; 10 min.</p>
                        </li>
                        <li>
                           <p>Stain in 1% UAc in 0.1M NaAcetate, pH 5.2, room temperature for 45 min.</p>
                        </li>
                        <li>
                           <p>Wash in 0.1M NaAcetate, 2 &times; 10 min; wash in 0.2M HEPES, 2 &times; 10 min.</p>
                        </li>
                        <li>
                           <p>Embed cut pieces in 3% agarose, positioning worms into clusters; allow to harden at 4&deg;C for 1 hour, or overnight under buffer.</p>
                        </li>
                        <li>
                           <p>Cut out small agarose blocks containing worms; should be large enough to be excluded from pipette tip, but small enough to
                              fit into embedding mold.
                           </p>
                        </li>
                        <li>
                           <p>Dehydrate through ethanols; 5 min in 30% EtOH, 5 min in 50% EtOH, 5 min in 75% EtOH, 5 min in 95% EtOH, 3 &times; 10 min in 100%
                              EtOH, 3 &times; 10 min in propylene oxide.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Infiltration:</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Change is in 1 part resin to 2 parts propylene oxide, allow to sit 3&#8211;4 hours or more.</p>
                        </li>
                        <li>
                           <p>Change is in 2 parts resin to 1 part propylene oxide, for 3&#8211;4 hours or more, 1 part propylene oxide to 2 parts resin overnight,
                              rotating.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>The next day, place samples in fresh resin at room temperature; place samples in air-tight containers and agitate or rotate
                     samples slowly during infiltration. Then change into fresh resin at least 3&#8211;4 times, 2&#8211;4 hours per change
                  </p>
                  <p>Embed samples in fresh resin in embedding mold, and position carefully.</p>
                  <p>Cure samples in 60&deg;C oven for 3 days</p>
                  <p><span class="bold"><strong>Resin formula</strong></span> [use pre-dessicated plastic beaker with gradations for easy measurements]:
                  </p>
                  <p>45 ml Scipoxy 812 Other brands include Embed 812, Medcast, Epon</p>
                  <p>30 ml DDSA</p>
                  <p>25 ml NMA</p>
                  <p>Mix components well with wooden applicator, add accelerator, 1.5 ml DMP-30, and mix resin thoroughly again. Place resin into
                     dessicator and put under vacuum for 5&#8211;15 min to remove gasses from mixture before using
                  </p>
                  <p><span class="bold"><strong>Special Equipment Needed</strong></span></p>
                  <p>Scientific microwave oven (we use a model from Ted Pella); kitchen ovens do not properly control the microwave energy pulses
                     for this purpose. See (<a href="#bib27">Paupard et al., 2001</a>) for further details on use of the microwave oven.
                  </p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 12: High pressure freezing fixation (Dave Hall)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Scoop animals off an NGM plate in a coating of <span class="emphasis"><em>E. coli</em></span> using a pick. Place the animals and bacteria into a metal hat until interior compartment is full to the brim. Place a blank
                              hat on top to seal the compartment and load into HPF machine to freeze. There should be no air space surrounding the sample
                              within the metal hat [to obtain maximum pressure].
                           </p>
                           <p><span class="underline">Freeze Substitution</span></p>
                        </li>
                        <li>
                           <p>Hold frozen sample at <span class="symbol">&#8722;</span>90&deg;C for 3 days in 2.0% OsO<sub>4</sub>, 0.1% Uranyl Acetate in acetone fixative.
                           </p>
                        </li>
                        <li>
                           <p>Warm sample to <span class="symbol">&#8722;</span>74&deg;C in same fixative and then to <span class="symbol">&#8722;</span>60&deg;C over 14-hour period.
                           </p>
                        </li>
                        <li>
                           <p>Warm sample to <span class="symbol">&#8722;</span>30&deg;C in same fixative over 14-hour period.
                           </p>
                        </li>
                        <li>
                           <p>Warm sample to 0&deg;C in same fixative over 1-hour period.</p>
                        </li>
                        <li>
                           <p>Transfer sample to 100% acetone three times at room temp, 20 min total.</p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Infiltration</span></p>
                  <p>First change is in 1 part resin to 1 part acetone, allow to sit 30 min.</p>
                  <p>Second change is in 2 parts resin to 1 part acetone, allow to sit for 30 min.</p>
                  <p>Third change is into pure resin plus accelerator and hold one hour.</p>
                  <p>Fourth change is into pure resin plus accelerator and hold overnight.</p>
                  <p>The next day, place samples in fresh resin plus accelerator at room temperature and place infiltrated worms in eppendorf tubes
                     or gelatin capsules. Cure samples under UV irradiation overnight or until hard; keep at <span class="symbol">&#8722;</span>20&deg;C while curing in Pelco Cryobox. If desired, cut best worms out of embedment and re-embed in resin at desired orientation
                     and re-cure in sealed flat mold.
                  </p>
                  <p><span class="underline">Resin formula</span></p>
                  <p>LR Gold resin plus 0.5% BME accelerator</p>
                  <p><span class="bold"><strong>Special Equipment needed</strong></span></p>
                  <p>High Pressure Freeze apparatus; a very expensive item, but they are available at several national facilities including those
                     in Madison, Minneapolis, Berkeley, and Albany. Freeze substitution apparatus; can be home-built or purchased
                  </p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 13: High pressure freezing and freeze substitution for TEM (Richard Fetter, Kent McDonald, Cori Bargmann)</b></p>
                  <p><span class="underline">HPF procedure with a Bal-Tec HPM 010:</span></p>
                  <p>As with any ultra-rapid freezing method, the principal concern is optimizing the specimen geometry to maximize the heat transfer
                     rate with the coolant used. Increasing the heat transfer rate is often best achieved by minimizing the size of the specimen
                     chamber in the direction that the coolant is being applied. The Bal-Tec HPM 010 high-pressure freezing device has specimen
                     holders that can be used to create specimen chambers that closely match the diameter of the specific <span class="emphasis"><em>C. elegans</em></span> stage being frozen through the use of appropriate spacers.
                  </p>
                  <p>The most suitable spacers found so far are slot, hole and Chien EM grids (ordered from Ted Pella, but probably available from
                     other suppliers; thicknesses were measured with a Fowler 52-223-005 micrometer). Multiple grids can be used in combination
                     to create specimen chambers of other thicknesses as needed:
                  </p>
                  <div class="informaltable">
                     <table border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <thead>
                           <tr>
                              <th align="left">Grid</th>
                              <th align="left">Ave. thickness (&#956;m)</th>
                              <th align="left">Range (&#956;m)</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td align="left">9GC20H (&#8220;Chien&#8221;)</td>
                              <td align="left">27</td>
                              <td align="left">27&#8211;28</td>
                           </tr>
                           <tr>
                              <td align="left">1GC42S</td>
                              <td align="left">35</td>
                              <td align="left">33&#8211;37</td>
                           </tr>
                           <tr>
                              <td align="left">1GC12H</td>
                              <td align="left">48</td>
                              <td align="left">47&#8211;50</td>
                           </tr>
                           <tr>
                              <td align="left">1GN42S</td>
                              <td align="left">27</td>
                              <td align="left">26&#8211;29</td>
                           </tr>
                           <tr>
                              <td align="left">1GN12H</td>
                              <td align="left">58</td>
                              <td align="left">54&#8211;65</td>
                           </tr>
                           <tr>
                              <td align="left">SynapTek DOT 4511</td>
                              <td align="left">100</td>
                              <td align="left">100&#8211;102</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>On the flat side of the Type B holder (300 m planchette), place a grid suitable in thickness to the worms being frozen. The
                              new space within the hole/slot of the EM grid is the specimen chamber where the embryos/worms will go. Place a dab of <span class="emphasis"><em>E. coli</em></span> in the chamber. (Briefly touching the grid to a thin <span class="emphasis"><em>E. coli</em></span> lawn before placing it on the flat surface of the Type B holder will help keep the spacer in place for subsequent steps.)
                           </p>
                        </li>
                        <li>
                           <p>Pick the embryos/worms you want and place them in the <span class="emphasis"><em>E. coli</em></span> dab in the specimen chamber. I try to put about 10 embryos/worms in a sample to be frozen. That's plenty for sectioning in
                              case the freezing is good, and not too many that if the freezing is bad that a lot effort has been wasted.
                           </p>
                        </li>
                        <li>
                           <p>Fill the rest of the well with <span class="emphasis"><em>E. coli</em></span> (preferred) or yeast paste as packing material, so that the level is flush with the top surface of the spacer. The consistency
                              of the packing material is important. You want to avoid packing material that is too soupy because the higher water content
                              does affect the freezing rate.
                           </p>
                        </li>
                        <li>
                           <p>Place the flat side of a second Type B holder (300 m) on top of the embryos/worms.</p>
                        </li>
                        <li>
                           <p>Place this &#8220;sandwich&#8221; in the specimen freezing holder of the Bal-Tec HPM 010 and freeze.</p>
                        </li>
                        <li>
                           <p>Store the frozen sample intact in LN2 until processed.</p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Freeze substitution protocol using a Leica AFS</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Under LN2, the frozen sample is split open with the tips of a pair of forceps previously cooled in LN2.</p>
                        </li>
                        <li>
                           <p>Transfer the holder(s) containing the sample to a 2.0 ml Nalgene cryovial containing 1&#8211;1.5 ml of frozen substitution medium
                              (0.5&#8211;1.0% OsO<sub>4</sub>/0.1% UAc in 2&#8211;5%MeOH/95-98% acetone for conventional EM; low glut and/or OsO<sub>4</sub> in 2&#8211;5%MeOH/95-98% acetone, or other cocktail, for immuno-EM) under LN2 and screw the cap on.
                           </p>
                        </li>
                        <li>
                           <p>Transfer the vial to your freeze-substitution device. A Leica AFS is the easiest to use since the temperatures and durations
                              of each step in the substitution can be programmed, however, perfectly adequate results can be obtained with &#8220;custom&#8221; freeze-substitution
                              devices based on picnic coolers and dry ice.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>For the Leica AFS device, I generally use the following protocol for conventional EM:</p>
                  <div class="informaltable">
                     <table border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <thead>
                           <tr>
                              <th align="left">Stage</th>
                              <th align="left">Temp. or warming rate</th>
                              <th align="left">Duration (hrs)</th>
                              <th align="left">Total time (hrs)</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td align="left">T1</td>
                              <td align="left"><span class="symbol">&#8722;</span>90&deg; C
                              </td>
                              <td align="left">72</td>
                              <td align="left">72</td>
                           </tr>
                           <tr>
                              <td align="left">S1</td>
                              <td align="left">5&deg; C/hr</td>
                              <td align="left">13</td>
                              <td align="left">85</td>
                           </tr>
                           <tr>
                              <td align="left">T2</td>
                              <td align="left"><span class="symbol">&#8722;</span>25&deg; C
                              </td>
                              <td align="left">13.5</td>
                              <td align="left">98.5</td>
                           </tr>
                           <tr>
                              <td align="left">S2</td>
                              <td align="left">10&deg; C/hr</td>
                              <td align="left">4.5</td>
                              <td align="left">103</td>
                           </tr>
                           <tr>
                              <td align="left">T3</td>
                              <td align="left">+20&deg; C</td>
                              <td align="left">2&#8211;17</td>
                              <td align="left">105&#8211;120</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <p>This typically gives good preservation and enhanced membrane contrast with well-frozen samples. There is, however, wide latitude
                     in temperatures, warming rates, and durations of each step that give excellent results.
                  </p>
                  <p><span class="bold"><strong>Freeze-substitution using a &#8220;custom&#8221; freeze-substitution device</strong></span></p>
                  <p>When an automated freeze-substitution device, such as a Leica AFS, is not available, completely adequate freeze-substitution
                     can be performed using a &#8220;custom&#8221; device and dry ice. In this case it is helpful to have a:
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p><span class="bold"><strong>digital thermometer/thermocouple</strong></span> (Omega Engineering) to monitor temperature during the substitution process,
                           </p>
                        </li>
                        <li>
                           <p><span class="bold"><strong>heat sink</strong></span> to buffer temperature changes, such as an aluminum block with holes just larger than the sample vials,
                           </p>
                        </li>
                        <li>
                           <p><span class="bold"><strong>insulated container</strong></span> to hold the samples, heat sink and about 40 pounds of dry ice (such as a Rubbermaid model 1827 picnic cooler), and
                           </p>
                        </li>
                        <li>
                           <p><span class="bold"><strong>orbital shaker</strong></span> capable of holding up to 50 lbs (Cole-Parmer model 51704 orbital shaker, 3/4'' stroke).
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>Neoprene SCUBA diving gloves work great for manipulating the specimens and aluminum holder at LN2 and dry ice temps.</p>
                  <p>The insulated container should be allowed to cool down prior to loading samples for substitution, by filling it with dry ice
                     about 30 minutes before use.
                  </p>
                  <p><span class="bold"><strong>Procedure:</strong></span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>As with the Leica AFS procedure above, under LN2, the frozen sample &#8220;sandwich&#8221; is split open with the tips of a pair of forceps
                              previously cooled in LN2.
                           </p>
                        </li>
                        <li>
                           <p>Transfer the holder(s) containing the sample to a 2.0 ml Nalgene cryovial containing 1.5 ml of frozen substitution medium
                              (0.5&#8211;1.0% OsO<sub>4</sub>/0.1% UAc in 2&#8211;5%MeOH/95-98% acetone for conventional EM; low glut and/or OsO<sub>4</sub> in 2&#8211;5%MeOH/95-98% acetone, or other cocktail, for immuno-EM) under LN2 and screw the cap on.
                           </p>
                        </li>
                        <li>
                           <p>The cryovials are transferred to the aluminum holder in a styrofoam container, or dewar, previously equilibrated with LN2.
                              Once all the sample vials are loaded, the aluminum holder is placed at a slight angle in the picnic container holding the
                              dry ice, and some aluminum foil is wrapped around the block to keep the sample vials secured during substitution. The thermocouple
                              is positioned through a small hole in the aluminum foil to a spot close to the sample vials, and the lid of the picnic cooler
                              closed.
                           </p>
                        </li>
                        <li>
                           <p>The picnic cooler is placed on the orbital shaker and fastened to the platform of the shaker with bungee cords. The shaker
                              is turned on and set at 50&#8211;100 rpm. The substitution with this agitation is performed for 3&#8211;4 days, with replenishment of
                              dry ice as needed to keep the specimen covered with dry ice.
                           </p>
                        </li>
                        <li>
                           <p>Fill the cooler for the last time with dry ice 48 hours before the planned end of the substitution time. Over the 48 hours
                              the dry ice will gradually disappear and the aluminum block will gradually warm. Remove the specimens at the desired end point
                              temperature of the substitution and rinse the specimens with acetone at that temperature.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="bold-underline">Infiltration and embedding:</span></p>
                  <p><span class="bold"><strong>Resins</strong></span></p>
                  <p>For conventional EM I've been using a 1A:3B mixture of Eponate 12 or other Epon replacement resin. For immuno-EM, a number
                     of resins are available that are amenable to UV polymerization at low temperature, including LR White, LR Gold, Unicryl and
                     Lowicryl. Infiltration should be performed using a tissue rotator whenever possible.
                  </p>
                  <p>Conventional EM embedding</p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>At the completion of the freeze-substitution program, rinse the samples 2x with dry acetone, and then exchange the acetone
                              for propylene oxide.
                           </p>
                        </li>
                        <li>
                           <p>Infiltrate the samples with 1:2 propylene oxide:resin for 30 minutes followed by 2:1 resin:propylene oxide for 30 minutes,
                              and then transfer to pure resin containing catalyst. An additional 2&#8211;3 changes of resin, 1&#8211;2 hours each, followed by an overnight
                              change in resin will ensure complete infiltration of the worm. All infiltration steps should be performed with a tissue rotator.
                           </p>
                           <p>In 100% resin, the embryo/worm plus <span class="emphasis"><em>E.coli</em></span>/yeast pancake usually comes out of the holder. A sharpened tungsten needle can be used to CAREFULLY and GENTLY chip the packing
                              material away from the embryos/worms. It is fine to leave the worms in the packing material, but it is a little harder to
                              see the them to orient for sectioning.
                           </p>
                        </li>
                        <li>
                           <p>In order to have maximum control over the orientation of the worms for sectioning, I prepare blocks for sectioning in 2 stages.
                              First, I flat embed the worms between 2 glass slides previously coated with spray-on Teflon (Cat. No. MS-122DF from the Miller-Stephenson
                              Chemical Co., George Washington Hwy, Danbury, CT 06810, (203) 743-4447. Previously coat one side of each pair of glass slides,
                              allow to dry, and then wipe off the white residue with a Kimwipe). A doubled-over piece of Parafilm at each end of the slide
                              works well as a spacer to prevent the worms from being crushed. After polymerization, a razor blade is used to gently pry
                              the slides apart.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>Using a razor blade, cut out the worms to be sectioned and re-embed them in regular molds in the desired orientation. Polymerize
                     as usual (48 hrs at 60&#8211;70 &deg; C). The teflon does not seem to interfere with the adhesion of the resin containing the worm with
                     the fresh resin it is re-embedded in. I have not experienced separation at this junction during thin sectioning.
                  </p>
                  <p><span class="bold"><strong>Immuno-EM embedding</strong></span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Rinse the samples 2x with dry acetone and then infiltrate with your immuno-EM resin of choice as usual.</p>
                           <p>As a general rule, infiltration and embedding at low temperature (<span class="symbol">&#8722;</span>20&deg; C or lower) is preferable to room temperature for preserving antigenicity. For an immuno-EM resin, I have used LR White
                              since it was more &#8220;fluid&#8221; at <span class="symbol">&#8722;</span>20&deg; C than Lowicryl and can be UV polymerized using 1.5 mg/ml resin of benzoin ethyl ether as the UV catalyst. The trick is
                              to add and dissolve 1.5 mg/ml of benzoin ethyl ether in the LR White at room temperature, then de-gas the resin (since UV
                              polymerization forces the dissolved gas out of the resin and unfortunately around the tissue) before you cool it down to <span class="symbol">&#8722;</span>20&deg; C for infiltration.
                           </p>
                        </li>
                        <li>
                           <p>Following infiltration, place the worms in molds and polymerize with UV for 24 hrs at <span class="symbol">&#8722;</span>20&deg; C followed by 2&#8211;10 hrs at room temperature.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>LR White polymerization is very sensitive to oxygen. To avoid getting an unpolymerized film of resin at the top of the mold,
                     I place a strip of ACLAR film (Ted Pella) on top of the mold, taking care not to trap a bubble of air, after I put the worm
                     in and fill the mold with resin. Unlike most plastic films, ACLAR has very low permeability to oxygen.
                  </p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 14: Fixation of embryos for TEM (Carolyn Norris, Dave Hall)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Mount embryos on slides as if for DIC except agar has fixative.</p>
                           <p><span class="bold"><strong>Agarose Fixative</strong></span></p>
                           <p>3 ml 2% Agarose : 1.5% LGT Agarose + .05% HGT Agarose</p>
                           <p>5 ul 1M MgCl<sub>2</sub></p>
                           <p>400 ul 0.2 M NaCacodylate (pH 7.4)</p>
                           <p>400 ul sucrose (68.46g/100ml)</p>
                           <p>500 ul 25% Glutaraldehyde</p>
                           <p>Make up above mixture immediately before mounting embryos. The pad should be thick, so use triple thickness tape when preparing
                              slide. Transfer the eggs to pads with minimal fluid (to preserve osmolarity). Transfer with a pick or mouth pipette. Use an
                              eyebrow hair to push the eggs to the center and arrange them so that they are close to each other without touching.
                           </p>
                        </li>
                        <li>
                           <p>Overlay a <span class="emphasis"><em>small</em></span> amount of slightly cooled agarose mix on slide and place a coverslip on top. Try either 12 mm or 18 mm coverslips.
                           </p>
                        </li>
                        <li>
                           <p>Make a map of the embryos- their arrangement on the pad, their orientation, and their relative stages. Mount a bunch of 1,
                              2, and 4 cell embryos on a pad, then watch them go through landmark events, like gastrulation, the migration of the dorsal
                              <a href="http://www.wormbase.org/db/get?name=hyp;class=Cell" target="_blank">hyp</a> nuclei, or cell deaths to stage them.
                           </p>
                        </li>
                        <li>
                           <p>When an embryo reaches the stage you want, permeabilize the egg shell and vitelline membrane with a laser. Shoot for edge
                              of shell. You can see the break in the shell and the membrane. The laser needs to be at full power. Also kill (explode) any
                              embryos that you don't want and record their positions on your &#8220;map&#8221;. Before fixing the final specimens, try several trial
                              embryos with differing amounts of sucrose in the final fixative. While watching under the microscope, shoot several laser
                              holes in the eggshell. If the embryo collapses rapidly (implodes), then the sucrose concentration is too high. If the embryo
                              blows up rapidly (explodes), then the sucrose is too low. Under optimal conditions, the embryo will be seen to swell in volume
                              very slowly after a laser hole is put through the eggshell. Thus the final fixative should be tested empirically on the day
                              of fixation, to optimize the exact osmolarity.
                           </p>
                        </li>
                        <li>
                           <p>Place the slide in a humidified chamber at room temperature for 2 hours to allow fixation.</p>
                        </li>
                        <li>
                           <p>Remove the coverslip by pipetting 25 mM NaHEPES (pH 7.5) around the base of the pad, then gently slide the coverslip off.
                              Eggs will usually stay in place.
                           </p>
                        </li>
                        <li>
                           <p>Begin dehydrations and washes. Either leave the group in their original agar pads, or cut up the pad and place individual
                              eggs in separate wells of a 9-well glass plate. Either way, trim the agarose with a clean new razor blade.
                           </p>
                        </li>
                        <li>
                           <p>Rinse in NaHEPES: 2 &times; 30<span class="symbol">&#8243;</span>, then 1 &times; 2<span class="symbol">&#8242;</span>.
                           </p>
                        </li>
                        <li>
                           <p>Wash in 0.2M Na Cacodylate pH 7.4, 2 &times; 30<span class="symbol">&#8243;</span>, 1 &times; 5<span class="symbol">&#8242;</span>, 1 &times; 10<span class="symbol">&#8242;</span>, 2 &times; 1 hr.
                           </p>
                        </li>
                        <li>
                           <p>Post-Fix in 1% OsO<sub>4</sub> +1%KFe(CN)<sub>6</sub> in .2 M NaCacodylate for 1 to 2 hours.
                           </p>
                        </li>
                        <li>
                           <p>Wash in 0.1M NaCacodylate: 2 &times; 30<span class="symbol">&#8243;</span>, 1 &times; 5<span class="symbol">&#8242;</span>, 2 &times; 10<span class="symbol">&#8242;</span>.
                           </p>
                        </li>
                        <li>
                           <p>Post-Fix in 0.2% tannic acid in 0.1M Cacodylate for 15<span class="symbol">&#8242;</span>.
                           </p>
                        </li>
                        <li>
                           <p>Wash in 0.1M NaCacodylate 2 &times; 30<span class="symbol">&#8243;</span>, 1 &times; 5<span class="symbol">&#8242;</span>, 2 &times; 10<span class="symbol">&#8242;</span>.
                           </p>
                        </li>
                        <li>
                           <p>Wash in 0.1M NaAcetate, 2 &times; 30<span class="symbol">&#8243;</span>, 1 &times; 5<span class="symbol">&#8242;</span>, 1 &times; 10<span class="symbol">&#8242;</span>, 1 &times; 30<span class="symbol">&#8242;</span>.
                           </p>
                        </li>
                        <li>
                           <p>Stain with 1% Uranyl Acetate in 0.1M NaAcetate for 3 hours.</p>
                        </li>
                        <li>
                           <p>Wash in in NaAcetate 2 &times; 30<span class="symbol">&#8243;</span>, 2 &times; 10<span class="symbol">&#8242;</span>.
                           </p>
                        </li>
                        <li>
                           <p>Wash in 0.1M HEPES 1 &times; 30<span class="symbol">&#8243;</span>, 2 &times; 5<span class="symbol">&#8242;</span>.
                           </p>
                        </li>
                        <li>
                           <p>Before the dehydration series, remount the embryos into large chunks (~1 cc) of agar (2%) and then cut each block into a distinct
                              shape so that every embryo can be distinguished from all the other embryos. The larger blocks help protect the embryos as
                              you go through all the steps and they are also much easier to see (and therefore to avoid sucking into your pipette and discarding).
                              The distinct shapes allow you to dehydrate all the embryos in the same vial without confusing them.
                           </p>
                        </li>
                        <li>
                           <p>Rinse the blocks in dH<sub>2</sub>0 and transfer to a vial such as a scintillation vial.
                           </p>
                        </li>
                        <li>
                           <p>Dehydration: 6' 30% EtOH, 10' 50% EtOH, 10' 70% EtOH, [if you need to take a break, this is a safe place to halt processing
                              overnight], 6' 90% EtOH, 6' 95% EtOH, 3 &times; 20' 100% EtOH, 20' 1:1 Propylene Oxide:EtOH, 2 &times; 8' Propylene Oxide, 30' Propylene
                              Oxide:Araldite 2:1, 60' Propylene Oxide:Araldite 1:2, 2 &times; 30' Araldite Room Temperature.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>Trim the blocks and return embryos to separate wells of a 9 well plate.</p>
                  <p>24 hrs Araldite RT.</p>
                  <p>Transfer blocks to whatever mold you will use to hold the embryo and try your best to orient embryo for easiest trimming and
                     cutting.
                  </p>
                  <p>3 days Araldite @60&deg;C.</p>
                  <p><span class="bold"><strong>Buffer Recipes</strong></span></p>
                  <p><span class="underline">0.2M NaCacodylate</span></p>
                  <div class="orderedlist">
                     <ol type="A">
                        <li>
                           <p>42.8g Na(CH<sub>3</sub>)<sub>2</sub>AsO<sub>2</sub> 3H<sub>2</sub>O in 1000ml dH<sub>2</sub>O
                           </p>
                        </li>
                        <li>
                           <p>0.2 M HCl [10 ml conc. HCl + 603 ml dH<sub>2</sub>O]
                           </p>
                           <p>50ml <span class="bold"><strong>A</strong></span> + <span class="bold"><strong>B</strong></span>/18.3 <span class="symbol">&#8594;</span> pH/6.4 Q.S. to 200 ml
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 15: Preparing worm gonads for SEM (David Greenstein)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Select L4 hermaphrodites <span class="emphasis"><em>en masse</em></span>.
                           </p>
                        </li>
                        <li>
                           <p>Grow overnight (12&#8211;16 hrs) to adult stage.</p>
                        </li>
                        <li>
                           <p>Transfer to unseeded plate to remove bacteria.</p>
                        </li>
                        <li>
                           <p>Transfer to a watch glass with 0.5 ml of 1X PBS.</p>
                        </li>
                        <li>
                           <p>Wash worms with one change of 1X PBS.</p>
                        </li>
                        <li>
                           <p>Dissect gonads in 1x PBS containing 0.05% tricane/0.005% tetramisole.</p>
                        </li>
                        <li>
                           <p>Transfer to a 6 x 50 mm borosilicate glass tube.</p>
                        </li>
                        <li>
                           <p>Recover the gonads by centrifugation at 1000 rpm in a IEC clinical centrifuge for 2 min. at room temperature.</p>
                        </li>
                        <li>
                           <p>Digest the gonadal basal lamina by incubation with 20&#8211;40 units/ml of Type II collagenase in 1x PBS for 2&#8211;12 min at room temperature.</p>
                        </li>
                        <li>
                           <p>After digestion place gonads on ice and gently remove supernatant from gonads which had settled during incubation period.</p>
                        </li>
                        <li>
                           <p>Because gonads are fragile until fixed, add all solutions slowly. Wash gonads with cold 1x PBS, allowing the gonads to settle
                              by gravity.
                           </p>
                        </li>
                        <li>
                           <p>Fix the gonads with 3% glutaraldehyde in 0.1 M sodium phosphate buffer (pH 7.2) for 4 hrs on ice.</p>
                        </li>
                        <li>
                           <p>After they are fixed, wash the gonads three times with cold 0.1 M sodium phosphate buffer (pH 7.0) containing 5% sucrose,
                              incubating on ice for 15 min. during each wash.
                           </p>
                        </li>
                        <li>
                           <p>After the final wash pipette the gonads into a watch glass and hand select intact gonads under a dissecting scope.</p>
                        </li>
                        <li>
                           <p>Postfix the gonads in 1% osmium tetroxide in 0.1 M sodium phosphate buffer (pH 7.2) for 1 hr at room temperature.</p>
                        </li>
                        <li>
                           <p>Rinse gonads in several changes of 0.1 M sodium phosphate buffer (pH 7.2).</p>
                        </li>
                        <li>
                           <p>Dehydrate in an ethanol series (50, 70, 95, 100%).</p>
                        </li>
                        <li>
                           <p>Dehydrate again in pure ethanol.</p>
                        </li>
                        <li>
                           <p>Place gonads in the cooled chamber of a critical point drier with the samples being covered with ethanol. To prevent the gonads
                              from being damaged when the solvent enters or is vented from the chamber, place the gonads between several layers of filter
                              paper cut with a standard hole punch.
                           </p>
                        </li>
                        <li>
                           <p>Transfer dried samples to aluminum stubs covered with double stick tape using a camel-hair brush.</p>
                        </li>
                        <li>
                           <p>Desiccate overnight and then coat with gold using a sputter coat.</p>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1909"></a>3.&nbsp;Protein-protein and protein-DNA interactions
                     </h2>
                  </div>
               </div>
            </div>
            <p>An important task in deciphering protein function is the identification of other entities with which it interacts. Although
               <span class="emphasis"><em>C. elegans</em></span> has not been exploited as an organism for biochemical analysis it is clearly amenable to such studies. Below are protocols
               describing immunoprecipitation (IP) and chromatin immunoprecipitation (ChIP) that should serve as general guidelines for <span class="emphasis"><em>in vivo</em></span> interaction studies in <span class="emphasis"><em>C. elegans</em></span>. These interactions can often be confirmed by standard <span class="emphasis"><em>in vitro</em></span> techniques such as two-hybrid, GST pull-down studies, and electrophoresis mobility shift assays (EMSA).
            </p>
            <div class="sidebar">
               <p class="title"><b>Protocol 16: Immunoprecipitation from embryo lysates (Ray Chan and Barbara Meyer)</b></p>
               <p><span class="underline">Worm Liquid Culture:</span></p>
               <p><span class="underline">Seeding asynchronous liquid worm culture</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Float adults worms off 9-cm NGM plates with 5 ml of M9. Approximately 6&#8211;8 plates saturated with asynchronous population of
                           worms, but not starved, will be needed to seed each liter of liquid culture.
                        </p>
                     </li>
                     <li>
                        <p>Add 10&#8211;15 ml of saturated HB101 and monitor the food supply at least once per day. Spot 1 drop of culture onto a 5-cm NGM
                           plate and allow the liquid to evaporate for about 1&#8211;2 minutes until the worms can crawl. If there is sufficient food, the
                           worms should not scatter or forage. Alternatively, starved worms appear somewhat translucent.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Harvesting asynchronous culture and seeding synchronous cultures</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Filter culture through a 35-&#956;m miracloth (Calbiochem) to collect gravid hermaphrodites.</p>
                     </li>
                     <li>
                        <p>Wash with approximately 0.5 L of dH<sub>2</sub>O.
                        </p>
                     </li>
                     <li>
                        <p>Treat each liter of culture with 100 ml of freshly made alkaline-bleach solution. Mix on a stir-plate for 5&#8211;10 minutes (for
                           mutant worms, it may be advisable to bleach for less than 5 minutes). Stop the bleaching process when the adult worms start
                           to break open.
                        </p>
                        <p>
                           
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td><span class="bold-underlune">Alkaline-bleach solution (100 mL):</span></td>
                           </tr>
                           <tr>
                              <td>75 mL H<sub>2</sub>O
                              </td>
                           </tr>
                           <tr>
                              <td>20 mL commercial bleach</td>
                           </tr>
                           <tr>
                              <td>5 mL 10 N NaOH</td>
                           </tr>
                        </table>
                        <p>
                           
                        </p>
                     </li>
                     <li>
                        <p>Weigh a 250-mL conical centrifuge tube. Record weight __________ g.</p>
                     </li>
                     <li>
                        <p>Centrifuge the bleached worms at 1&#8211;2,000 rpm in a tabletop centrifuge. Stop the centrifuge as soon as the speed reaches 2,000
                           rpm. (It takes several minutes for the rotor to come to a complete stop).
                        </p>
                     </li>
                     <li>
                        <p>Resuspend the worm in M9 and centrifuge as in step 5. Repeat this wash step once more for a total of two M9 washes.</p>
                     </li>
                     <li>
                        <p>Weigh the centrifuge tube with the washed embryos.</p>
                        <p>Combined weight of centrifuge tube and embryos __________ g.</p>
                        <p>Weight of the embryos __________ g.</p>
                     </li>
                     <li>
                        <p>Seed 0.5 to 2 grams of embryos per 1-L completed S-basal medium. Generally use 0.5&#8211;1 g for N2 and 1&#8211;2 g for mutant worms.
                           Allow the embryos to hatch overnight <span class="underline">without</span> food and feed the synchronized L1 larvae the next day.
                        </p>
                     </li>
                     <li>
                        <p>Wash unused embryos in homogenization buffer, resuspend in 1 mL of homogenization buffer per gram of embryo and store in <span class="symbol">&#8722;</span>80&deg;C freezer.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Embryo Lysates</span></p>
               <p>All steps are performed at 4&deg;C.</p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Thaw 10&#8211;15 mL of embryos frozen in homogenization buffer.</p>
                     </li>
                     <li>
                        <p>Place the tube on ice and sonicate it with ten 30-second bursts at 15% power on a Heat System XL2020 sonicator with a standard
                           tapered microtip (cat.#419, 3 mm diameter). Wait 1 minute in between bursts for cooling.
                        </p>
                     </li>
                     <li>
                        <p>Pellet debris by centrifugation at 6,500 rpm (5,000 &times; g) in a SS-34 rotor for 20 minutes. Collect the supernatant in a clean
                           50-mL tube.
                        </p>
                     </li>
                     <li>
                        <p>Sonicate as described in step 2 to shear the DNA. Pellet debris by centrifugation at 14,500 rpm (25,000 &times; g) in a SS-34 rotor
                           for 20 minutes. Collect and quick-freeze the supernatant in 0.5&#8211;1 mL aliquots.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Immunoprecipitation</span></p>
               <p>All steps performed at 4&deg;C.</p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Thaw embryo lysates on ice. Incubate lysates with Protein A Sepharaose beads (100 &#956;L per mL lysates; Amersham) or IgGsorb
                           (100 &#956;L per mL lysates; The Enzyme Center) for 10&#8211;30 minutes. Spin in a microcentrifuge for 2 min. at 500 &times; <span class="emphasis"><em>g</em></span> to pellet the Sepharose beads or IgGsorb. Transfer the supernatant to a new microfuge tube and spin in a microcentrifuge
                           at top speed for 10 min. Use the supernatant for IP.
                        </p>
                     </li>
                     <li>
                        <p>Incubate approximately 5 &#956;g of affinity purified antibodies with 3 mg total protein of embryo lysates for 2 hrs. Bring the
                           final volume up to 1.0 to 1.4 mL using ChIP buffer with 140 mM KCl.
                        </p>
                     </li>
                     <li>
                        <p>Pellet non-specific precipitates by spinning in a microcentrifuge at top speed (approximately 16,000 &times; <span class="emphasis"><em>g</em></span>) for 10 min.
                        </p>
                     </li>
                     <li>
                        <p>Transfer the supernatant to a new microfuge tube with 25 &#956;L of Protein A Sepharose and place on a rocker/nutator for 30 min.</p>
                     </li>
                     <li>
                        <p>Pellet the antibody-antigen complexes captured on the Protein A Sepharose beads by spinning in a microcentrifuge for 2 min.
                           at 500 &times; <span class="emphasis"><em>g</em></span>. Remove the supernatant.
                        </p>
                     </li>
                     <li>
                        <p>Wash the beads by adding 1 mL of ChIP buffer and spinning in a microcentrifuge for 2 min. at 500 &times; <span class="emphasis"><em>g</em></span>. Remove the wash buffer. Repeat this step for a total of four washes.
                        </p>
                     </li>
                     <li>
                        <p>Elute by boiling the beads with 1 x SDS sample buffer and loading directly onto an SDS-PAGE gel. Alternatively, incubate the
                           beads with 200 &#956;L of 0.1 M glycine (pH 3.0) at room temp. Pellet the beads as described above, remove and save the supernatant.
                           Precipitate the eluate with trichloroacetic acids.
                        </p>
                     </li>
                  </ol>
               </div>
               <p>For buffer recipes see the Chromatin Immunoprecipitation protocol below.</p>
            </div>
            <div class="sidebar">
               <p class="title"><b>Protocol 17: Chromatin Immunoprecipitation From Embryo Lysates (Ray Chan and Barbara Meyer)</b></p>
               <p><span class="underline">Formaldehyde cross-linking:</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Harvest embryos by bleaching gravid hermaphrodites. [A typical yield for N2 is 2&#8211;5 grams from 25 to 30 9-cm NGM plates].</p>
                     </li>
                     <li>
                        <p>Wash the embryos extensively with 1x M9 to remove the alkaline bleach solution.</p>
                     </li>
                     <li>
                        <p>Prepare 100 mL of formaldehyde solution [1 x M9 solution with 2% (v/v) formaldehyde]</p>
                     </li>
                     <li>
                        <p>Wash the embryos once in the formaldehyde solution. Aspirate away the wash solution. Add fresh formaldehyde solution to 50
                           mL and gently shake (using a nutator) at room temp for 30 minutes.
                        </p>
                     </li>
                     <li>
                        <p>Wash the cross-linked embryos once with 50 mL of 0.1 M Tris-HCl (pH 7.5), followed by two 50-mL washes of 1x M9.</p>
                     </li>
                     <li>
                        <p>Wash the embryos once with homogenization buffer. Add 1 mL of homogenization buffer per gram embryo (based on the starting
                           amount), quick-freeze and store at <span class="symbol">&#8722;</span>80&deg;C.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Extract prep:</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Thaw 10&#8211;15 mL of cross-linked embryos and add fresh protease inhibitors.</p>
                     </li>
                     <li>
                        <p>Place the tube on ice and sonicate it with ten 30-second bursts at 15% power on a Heat System XL2020 sonicator with a standard
                           tapered microtip (cat.#419, 3 mm diameter). Wait 1 minute in between bursts for cooling.
                        </p>
                     </li>
                     <li>
                        <p>Pellet debris by centrifugation at 6,500 rpm (5,000 &times; g) in a SS-34 rotor for 20 minutes. Collect the supernatant in a clean
                           50-mL tube.
                        </p>
                     </li>
                     <li>
                        <p>Sonicate as described in step 2 to shear the DNA. Pellet debris by centrifugation at 14,500 rpm (25,000 &times; g) in a SS-34 rotor
                           for 20 minutes. Collect and quick-freeze the supernatant in 0.5&#8211;1 mL aliquots.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">IP:</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Thaw embryo lysates on ice. Incubate lysates with Protein A Sepharaose beads (100 &#956;L per mL lysates; Amersham) or IgGsorb
                           (100 &#956;L per mL lysates; The Enzyme Center) for 10&#8211;30 minutes. Spin in a microcentrifuge for 2 min. at 2,000 &times; <span class="emphasis"><em>g</em></span> to pellet the Sepharose beads or IgGsorb. Transfer the supernatant to a new microfuge tube and spin in a microcentrifuge
                           at top speed for 10 min. Use the supernatant for IP.
                        </p>
                     </li>
                     <li>
                        <p>Incubate approximately 5 &#956;g of affinity purified antibodies with 3 mg total protein of embryo lysates for 2 hrs.</p>
                     </li>
                     <li>
                        <p>Pellet non-specific precipitates by spinning in a microcentrifuge at top speed (approximately 16,000 &times; <span class="emphasis"><em>g</em></span>) for 10 min.
                        </p>
                     </li>
                     <li>
                        <p>Transfer the supernatant to a new microfuge tube with 25 &#956;L of Protein A Sepharose and place on a rocker/nutator for 30 min.</p>
                     </li>
                     <li>
                        <p>Pellet the antibody-antigen complexes captured on the Protein A Sepharose beads by spinning in a microcentrifuge for 2 min.
                           at 500 &times; <span class="emphasis"><em>g</em></span>. Remove the supernatant.
                        </p>
                     </li>
                     <li>
                        <p>Wash the beads as follows:</p>
                        <p>2x 1-mL of ChIP buffer with 100 mM KCl</p>
                        <p>2x 1-mL of ChIP buffer with 1 M KCl</p>
                        <p>2x 1-mL of TE</p>
                     </li>
                     <li>
                        <p>Add 200 &#956;L elution buffer [10 mM Tris-HCl (pH8), 1% (w/v) SDS]. Spin to pellet the beads. Transfer eluate to a clean microfuge
                           tube. Repeat elution step once more. Combine the eluates (400 &#956;L total vol).
                        </p>
                     </li>
                     <li>
                        <p>Add 16 &#956;L of 5 M NaCl and heat overnight at 65&deg;C to reverse the formaldehyde cross-links. [Use a PCR machine with a heated
                           top or a Hybaid oven to avoid condensation at top of the tube].
                        </p>
                     </li>
                     <li>
                        <p>Adding 8 &#956;L of 0.5 M EDTA and 16 &#956;L of 1 M Tris-HCl (pH 6.8). Mix. Digest proteins with 20 &#956;g of proteinase K (Boehringer
                           Mannheim) for 1 hour at 45&deg;C.
                        </p>
                     </li>
                     <li>
                        <p>Phenol-chloroform extractions. Add 10&#8211;20 &#956;g of glycogen. Mix. EtOH precipitate. Resuspend in 100 &#956;L TE.</p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Buffer Recipes</strong></span></p>
               <p><span class="underline">M9 buffer:</span> 6 g Na<sub>2</sub>HPO<sub>4</sub>, 3 g KH<sub>2</sub>PO<sub>4</sub>, 5 g NaCl, 0.25 g of MgSO<sub>4</sub>7H<sub>2</sub>O per liter. Autoclave.
               </p>
               <p><span class="underline">1 M potassium citrate, pH 6.0:</span> Per liter solution, add 268.8 g tripotassium citrate, 26.3 g citric acid monohydrate and adjust pH with KOH. Autoclave to
                  sterilize.
               </p>
               <p><span class="underline">Trace metals solution:</span> Per liter solution, add 1.86 g Na<sub>2</sub>EDTA, 0.69 g FeSO<sub>4</sub>&middot;7H<sub>2</sub>O, 0.2 g MnCl<sub>2</sub>&middot;4H<sub>2</sub>O, 0.29 g ZnSO<sub>4</sub>&middot;7H<sub>2</sub>O, 0.016g CuSO<sub>4</sub>. Autoclave to sterilize and store in the dark.
               </p>
               <p><span class="underline">1 M potassium phosphate, pH 6.0:</span> Per liter solution, dissolve 136 g KH<sub>2</sub>PO<sub>4</sub> in 900 mL dH<sub>2</sub>O and adjust pH with KOH. Autoclave to sterilize.
               </p>
               <p><span class="underline">10 x S basal medium:</span> Per liter solution, add 59 g of NaCl, 500 mL of 1 M potassium phosphate (pH 6), 10 mL of cholesterol (5 mg/mL in EtOH). Autoclave
                  to sterilize. [Note: the cholesterol will not go into solution].
               </p>
               <p><span class="underline">Complete S medium:</span> Per liter, add 100 mL of 10 x S basal medium, 10 mL 1 M potassium citrate (pH 6), 10 mL trace metals solution, 3 mL 1 M MgSO<sub>4</sub>, 3 mL 1 M CaCl<sub>2</sub>. Autoclave to sterilize.
               </p>
               <p><span class="underline">Homogenization buffer:</span> 50 mM HEPES-KOH, pH 7.6; 1 mM EDTA; 140 mM KCl; <span class="bold"><strong>0.5%</strong></span> NP-40; 10% glycerol. Add fresh protease inhibitors (aprotinin, pepstatin A, leupeptin, PMSF) and 5 mM DTT before use.
               </p>
               <p><span class="underline">ChIP buffer:</span> 50 mM HEPES-KOH, pH 7.6; 1 mM EDTA; <span class="bold"><strong>0.05%</strong></span> NP-40. Add KCl to the desired concentration. Add fresh protease inhibitors (aprotinin, pepstatin A, leupeptin, PMSF) and
                  1 mM DTT before use. [This buffer contains less NP-40 and no glycerol compared to the homogenization buffer].
               </p>
            </div>
            <div class="sidebar">
               <p class="title"><b>Protocol 18: Chromatin immunoprecipitation (Johnathan Whetstine and Yang Shi)</b></p>
               <p>This protocol is a modified version of a protocol from Upstate Biotechnologies (<a href="http://www.upstate.com" target="_top">www.upstate.com</a>).
               </p>
               <p>Chromatin is isolated from 3&times;10<sup>5</sup> embryos/ChIP reaction, which equates to no less than 400 &#956;g chromatin per immunoprecipitation (IP). There is no harm in scaling
                  up for cleaner results, especially with poor antibodies.
               </p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Adult N2 worms grown on either HB101 or RNAi expressing bacteria are bleached and washed before being immersed and rotated
                           in 1.5% Formaldehyde/M9 buffer for 30 minutes at 16 &deg;C.
                        </p>
                     </li>
                     <li>
                        <p>Embryos are washed extensively with M9 (at least 3 times) and gently centrifuged. Be careful not to rupture embryos.</p>
                     </li>
                     <li>
                        <p>Add warm chromatin SDS lysis solution (30 &deg;C; Upstate biotechnologies cat. # 20-163) to the embryos (minimum of 3&times;10<sup>5</sup> embryos per 200 &#956;l solution) and dounce homogonize at least 30 times.
                        </p>
                     </li>
                     <li>
                        <p>Place the slurry on ice for at least 20&#8211;30 minutes.</p>
                     </li>
                     <li>
                        <p>Combine the total amount of embryos or adults and sonicate 25 times per 2ml of extract used for an approximately 500bp smear
                           (15 sec constant, 20% output on some sonicators). Do not over heat or allow to foam excessively. Keep on ice.
                        </p>
                     </li>
                     <li>
                        <p>Centrifuge the samples, and keep the supernatant for DNA quantification before processing.</p>
                     </li>
                     <li>
                        <p>Aliquot the supernatant (200 &#956;l) and dilute 10-fold into Dilution buffer (Upstate Biotechnologies cat. # 20-153).</p>
                     </li>
                     <li>
                        <p>Pre-clear each sample twice with 80 &#956;l of ssDNA/protein A agarose beads from Upstate Biotechnologies (cat. # 16-157).</p>
                     </li>
                     <li>
                        <p>Incubate samples with the indicated antibody overnight at 4 &deg;C with constant rotation.</p>
                     </li>
                     <li>
                        <p>The next morning add 60 &#956;l of beads and incubate for 1 hour. Centrifuge and wash the beads once at room temperature with constant
                           rotation with each of the following buffers (Note: You can make these buffers, see Upstate recipe, but I find that the beads
                           are cleaner when these products are used): 1.0 ml High Salt Solution (Upstate Biotechnologies cat. # 20-155), 1.0 ml Low Salt
                           buffer Solution (Upstate Biotechnologies cat. # 20-154), 1.0 ml LiCl Solution (Upstate Biotechnologies cat. # 20-156). After
                           the last wash, the beads are washed three times with 1X TE, pH 8.0, which makes the beads slightly translucent.
                        </p>
                     </li>
                     <li>
                        <p>Prepare and add fresh elution buffer (1%SDS and 0.1M NaHCO3; 250 &#956;l) to the beads at room temperature with constant agitation.
                           Keep the supernatant and repeat this step once more.
                        </p>
                     </li>
                     <li>
                        <p>Reverse the elution with 0.2 M NaCl for 4 hours at 65 &deg;C.</p>
                     </li>
                     <li>
                        <p>Treat the samples with proteinase K (10 &#956;l 0.5M EDTA, 20 &#956;l 1M Tris-HCl, pH 6.5, and 2 &#956;l 10mg/ml proteinase K) for 1 hour
                           at 45 &deg;C and then extract with phenol:chloroform:isoamyl alcohol.
                        </p>
                     </li>
                     <li>
                        <p>Precipitate the extracted solution with 20 &#956;g of yeast tRNA, and resuspend each sample in 50 &#956;l warm 10 mM Tris-HCl, pH 8.0.
                           The samples are now ready for PCR reactions.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e2311"></a>4.&nbsp;Specific methods in <span class="emphasis"><em>C. elegans</em></span> cell biology
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e2317"></a>4.1.&nbsp;Endocytosis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Endocytosis has been studied in two cell types in <span class="emphasis"><em>C. elegans</em></span>: coelomocytes, and oocytes (<a href="#bib10">Fares and Greenwald, 2001</a>; <a href="#bib16">Grant and Hirsh, 1999</a>). Coelomocyte endocytosis is usually assayed by uptake of proteins conjugated to dyes that are injected into the body cavity.
                  Oocyte endocytosis is measured by uptake of GFP-tagged yolk protein secreted by the intestine. Below are protocols for both
                  assays.
               </p>
               <div class="sidebar">
                  <p class="title"><b>Protocol 19: Pulse-chase analysis of coelomocyte endocytosis (Miyuki Sato and Barth Grant)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Mount adult worms under oil on a dried agarose pad as standard for microinjection.</p>
                        </li>
                        <li>
                           <p>Microinject Texas Red conjugated-BSA (Sigma; 0.2 to 1 mg/ml in Egg buffer) into the body cavity. Insert a needle into the
                              body cavity in the pharyngeal region. You should see liquid flowing in the body cavity.
                           </p>
                        </li>
                        <li>
                           <p>Recover worms onto seeded plates with Egg buffer and incubate for appropriate time. In wild type, Texas Red-BSA reaches lysosomes
                              within 60 min.
                           </p>
                        </li>
                        <li>
                           <p>After incubation, mount worms on ice-chilled agarose pads (2% agarose) with 1% formaldehyde to stop trafficking of Texas Red-BSA.
                              Incubate slides on ice for at least 20 min. and observe under microscope. Slides are kept on ice until just before starting
                              observation.
                           </p>
                           <p>Usually, about 10 worms are injected for each time point. At early time points it is difficult to find labeled coelomocytes
                              because the TR-BSA signal in coelomocytes is weak compared to Texas Red-BSA fluorescence in the body cavity. Using transgenic
                              worms expressing GFP fusions of endocytic markers in coelomocytes helps to find coelomocytes.
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 20: Gonad/Intestine Immunostaining (Adapted From R. Francis, Schedl Lab and Sarah Crittenden, Kimble Lab by Barth
                        Grant)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Pick adults and/or L4s to an unseeded NGM plate. Alternatively, wash worms off a seeded plate with 1.5 ml of egg buffer and
                              spin down in an eppendorf tube, then wash once with 1.5 ml of egg buffer. Transfer as many worms as can be dissected in 15
                              min (~200). If preparing multiple batches of dissected gonads/intestines, hatch bleached eggs in M9 buffer, then aliquot them
                              to plates to grow up as a synchronized population (about 200 worms/plate). L1 progeny of these worms present during dissection
                              do not interfere with anything.
                           </p>
                        </li>
                        <li>
                           <p>Suspend washed worms from one plate in 0.2&#8211;0.3 ml of egg buffer + 0.2 mM levamisole and transfer to a glass dish. Depression
                              slides work well. Use the deepest you can find.
                           </p>
                        </li>
                        <li>
                           <p>As paralysis sets in, begin cutting off heads at level of the pharynx. To cut off the head, place the head between two 25
                              gauge syringe needles and decapitate by moving needles in a scissors motion (avoid needles with bent tips). For most animals,
                              at least one gonad arm and the intestine should extrude completely.
                           </p>
                        </li>
                        <li>
                           <p>Transfer cut worms with a 100 ul mouth pipette to a presiliconized 1.7 ml eppendorf tube.</p>
                        </li>
                        <li>
                           <p>Fix the cut worms by adding 1 ml of 1.25% freshly made paraformaldehyde in egg buffer. Rock at RT for 10 min. Some people
                              post fix in <span class="symbol">&#8722;</span>20&deg;C methanol for 5 min. Cut worms can be stored up to two weeks in <span class="symbol">&#8722;</span>20&deg;C methanol. GFP autofluorescence survives the aldehyde fix well, but probably not the methanol.
                           </p>
                        </li>
                        <li>
                           <p>Preincubate in 1 ml PTB at RT for at least one hour. Change buffer at least 2X. Cut worms are spun down for 30 sec to 1 min
                              at 4,000 rpm between buffer changes. Worms can be stored at 4&deg;C for a few days in PTB. The Tween20 in PTB appears to very
                              quickly abolish fluorescence of autofluorecent gut granules (lipofucin), but has no effect on GFP.
                           </p>
                        </li>
                        <li>
                           <p>Spin down cut worms, remove supernatant, add primary Ab diluted in PTB. Incubate 4 hours at RT or overnight at 4&deg;C with rocking.</p>
                        </li>
                        <li>
                           <p>Wash for 1&#8211;4 hours at RT in PTC (four buffer changes, 1 ml each).</p>
                        </li>
                        <li>
                           <p>Preincubate in PTB (+ optional 10% serum) at RT for 30 min. Incubate in secondary antibody diluted in PTB for at least two
                              hours at RT. Use 1:500 dilutions of Cy3 or Alexa 488 conjugated antibodies from Jackson Immunoresearch or Molecular Probes.
                           </p>
                        </li>
                        <li>
                           <p>Wash for 1&#8211;4 hours at RT in PTC (four buffer changes, 1 ml each). You can never wash too much. An extra overnight wash at
                              4&deg;C can give nice results.
                           </p>
                        </li>
                        <li>
                           <p>Spin down cut worms, remove supernatant, resuspend in a glycerol antifade reagent such as Slowfade Light from Molecular Probes.
                              Store at <span class="symbol">&#8722;</span>20&deg;C. Mount on pads of Permanent Springtime Agarose just before viewing. Can seal with nail polish. Slides don't last long,
                              but can be stored briefly at 4&deg;C.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>
                     
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td><span class="underline">Egg Buffer:</span></td>
                     </tr>
                     <tr>
                        <td>NaCl (118 mM)</td>
                     </tr>
                     <tr>
                        <td>KCl (48 mM)</td>
                     </tr>
                     <tr>
                        <td>CaCl<sub>2</sub>&middot;2H<sub>2</sub>O (2 mM)
                        </td>
                     </tr>
                     <tr>
                        <td>MgCl<sub>2</sub>&middot;6H<sub>2</sub>O (2 mM)
                        </td>
                     </tr>
                     <tr>
                        <td>Hepes (25 mM)</td>
                     </tr>
                     <tr>
                        <td>Adjust pH to 7.3 with 1N NaOH.</td>
                     </tr>
                     <tr>
                        <td>Filter Sterilize</td>
                     </tr>
                  </table>
                  <p>
                     
                  </p>
                  <p>
                     
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td><span class="underline">PTC:</span></td>
                     </tr>
                     <tr>
                        <td>0.1% BSA</td>
                     </tr>
                     <tr>
                        <td>1X PBS</td>
                     </tr>
                     <tr>
                        <td>0.1% Tween20</td>
                     </tr>
                     <tr>
                        <td>0.05% Na Azide</td>
                     </tr>
                     <tr>
                        <td>1mM EDTA</td>
                     </tr>
                  </table>
                  <p>
                     
                  </p>
                  <p>
                     
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td><span class="underline">PTB:</span></td>
                     </tr>
                     <tr>
                        <td>1% BSA</td>
                     </tr>
                     <tr>
                        <td>1X PBS</td>
                     </tr>
                     <tr>
                        <td>0.1% Tween20</td>
                     </tr>
                     <tr>
                        <td>0.05% Na Azide</td>
                     </tr>
                     <tr>
                        <td>1mM EDTA</td>
                     </tr>
                     <tr>
                        <td>(Can also add 10% serum for preincubations, e.g. Goat serum if using a goat secondary).</td>
                     </tr>
                  </table>
                  <p>
                     
                  </p>
                  <p>
                     
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td>Permanent Springtime Agarose:</td>
                     </tr>
                     <tr>
                        <td>50 mM Tris Cl pH 9.5</td>
                     </tr>
                     <tr>
                        <td>5 mM MgCl<sub>2</sub></td>
                     </tr>
                     <tr>
                        <td>2% agarose</td>
                     </tr>
                  </table>
                  <p>
                     
                  </p>
                  <p><span class="underline">Fresh 1.25% Paraformaldehyde/PBS:</span></p>
                  <p>Mix 0.625g powdered paraformaldehyde with 25ml dH2O and 1&#8211;2 drops 1M NaOH. Heat to 65&deg;C. Mix until dissolved. Add dilute HCl
                     until pH drops to between 6 and 8 as judged by pH paper. Add 5 ml 10X PBS. Bring up to 50ml final volume with dH2O. Use the
                     same day. Fresh dilutions of concentrated EM grade formaldehyde from sealed ampules is also acceptable.
                  </p>
                  <p>
                     
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td><span class="underline">Best <a href="http://www.wormbase.org/db/seq/protein?name=RME-1;class=protein" target="_blank">RME-1</a>/<a href="http://www.wormbase.org/db/seq/protein?name=RME-2;class=protein" target="_blank">RME-2</a> Immunostain Conditions:</span></td>
                     </tr>
                     <tr>
                        <td>Dissect adults in egg buffer + 100 ug/ml (42 mM) levamisole.</td>
                     </tr>
                     <tr>
                        <td>Fix 10 min at RT in 1.5% PFA in egg buffer.</td>
                     </tr>
                     <tr>
                        <td>Wash 4&times;30min in PTB (PBS+0.1% Tween20+1% BSA).</td>
                     </tr>
                     <tr>
                        <td>Block 1&times; 30&#8211;60 min in PTB + 5% Goat Serum.</td>
                     </tr>
                     <tr>
                        <td>Incubate O/N at 4 degrees C with primary Ab 1:100 dil in PTB.</td>
                     </tr>
                     <tr>
                        <td>Wash 4 &times; 1 hour RT, PBST (PBS+0.1% Tween20).</td>
                     </tr>
                     <tr>
                        <td>Block 1&times; 30&#8211;60 min in PTB + 5% Goat Serum.</td>
                     </tr>
                     <tr>
                        <td>Incubate O/N at 4 degrees C with Goat anti-rabbit Alexa 488 1:1000 in PTB.</td>
                     </tr>
                     <tr>
                        <td>Wash 4 &times; 1 hour RT, PBST.</td>
                     </tr>
                     <tr>
                        <td>Resuspend in Slowfade Light with Glycerol.</td>
                     </tr>
                     <tr>
                        <td>Store at <span class="symbol">&#8722;</span>20 degrees C.
                        </td>
                     </tr>
                     <tr>
                        <td>Mount on agarose pads for microscopy.</td>
                     </tr>
                  </table>
                  <p>
                     
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e2525"></a>4.2.&nbsp;Chromatin cell biology (Gy&ouml;rgyi Csankovszki and Barbara Meyer)
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Specific chromosomal sequences, as well as chromosome-associated proteins can be elegantly labeled in <span class="emphasis"><em>C. elegans</em></span> using immunofluorescence and fluorescence <span class="emphasis"><em>in situ</em></span> hybridization (FISH) (<a href="#figure12">Figure 12</a>). In general, signals are easier to detect and resolve in larger nuclei, thus, best results are obtained by looking at germ
                  cells, intestinal cells, and cells in the early embryo. A variety of protocols related to chromatin visualization are presented
                  below.
               </p>
               <div class="mediaobject" align="center"><a name="figure12"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="cellfig12.jpg"><img src="cellfig12_s.jpg" border="2" align="middle" alt=" figure 12"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 12.&nbsp;</b>Pachetene stage germ cells stained using FISH. Red, DAPI; Green, 5S rDNA FISH (Annette Chan and Barbara Meyer).
                     </p>
                  </div>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 21: Antibody staining of <span class="emphasis"><em>C. elegans</em></span> gonads (Annette Chan and Barbara Meyer)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Use 1%, 1.5%, and 2% PFA with 5-minute fix:</p>
                           <div class="orderedlist">
                              <ol type="a">
                                 <li>
                                    <p>Place worms in 5 &#956;l 1X sperm salts on a positively-charged glass slide.</p>
                                 </li>
                                 <li>
                                    <p>Using a needle (Precision Glide<sup>&reg;</sup> Needle, Becton Dickinson &amp; Co., 22 G 1 1/2), make an incision between the pharynx and gonad (closer to the gonad than the
                                       pharynx) or between the tip of the tail and gonad of each worm. To release the embryos, cut at the vulva.
                                    </p>
                                 </li>
                                 <li>
                                    <p>Add 5 &#956;l of 2%, 3%, or 4% PFA (Electron Microscopy Sciences, cat. #15710) in 1X sperm salts.</p>
                                    <div class="table"><a name="d0e2574"></a><p class="title"></p>
                                       <table summary="" border="1">
                                          <colgroup>
                                             <col align="left">
                                             <col align="left">
                                          </colgroup>
                                          <tbody>
                                             <tr>
                                                <td align="left">To prepare 2% PFA:</td>
                                                <td align="left">125 &#956;l 16% PFA</td>
                                             </tr>
                                             <tr>
                                                <td align="left">&nbsp;</td>
                                                <td align="left">200 &#956;l 5X sperm salts</td>
                                             </tr>
                                             <tr>
                                                <td align="left">&nbsp;</td>
                                                <td align="left">675 &#956;l water</td>
                                             </tr>
                                          </tbody>
                                       </table>
                                    </div>
                                    <div class="table"><a name="d0e2593"></a><p class="title"></p>
                                       <table summary="" border="1">
                                          <colgroup>
                                             <col align="left">
                                             <col align="left">
                                          </colgroup>
                                          <tbody>
                                             <tr>
                                                <td align="left">To prepare 3% PFA:</td>
                                                <td align="left">187.5 &#956;l 16% PFA</td>
                                             </tr>
                                             <tr>
                                                <td align="left">&nbsp;</td>
                                                <td align="left">200 &#956;l 5X sperm salts</td>
                                             </tr>
                                             <tr>
                                                <td align="left">&nbsp;</td>
                                                <td align="left">612.5 &#956;l water</td>
                                             </tr>
                                          </tbody>
                                       </table>
                                    </div>
                                    <div class="table"><a name="d0e2612"></a><p class="title"></p>
                                       <table summary="" border="1">
                                          <colgroup>
                                             <col align="left">
                                             <col align="left">
                                          </colgroup>
                                          <tbody>
                                             <tr>
                                                <td align="left">To prepare 4% PFA:</td>
                                                <td align="left">250 &#956;l 16% PFA</td>
                                             </tr>
                                             <tr>
                                                <td align="left">&nbsp;</td>
                                                <td align="left">200 &#956;l 5X sperm salts</td>
                                             </tr>
                                             <tr>
                                                <td align="left">&nbsp;</td>
                                                <td align="left">550 &#956;l water</td>
                                             </tr>
                                          </tbody>
                                       </table>
                                    </div>
                                 </li>
                                 <li>
                                    <p>Place the slide in a humid chamber for 5 minutes. Place an 18-mm &times; 18-mm cover glass on top of the worms. Freeze the slide
                                       on a piece of dry ice. Let the slide sit on the dry ice for at least 10 minutes.
                                    </p>
                                 </li>
                              </ol>
                           </div>
                        </li>
                        <li>
                           <p>Remove the slide from the dry ice and quickly pop off the cover glass with a quick downward stroke of a single-edged razor.</p>
                        </li>
                        <li>
                           <p>Immediately place the slide in 95% ethanol for 1 minute. Be careful or the worms will fall off.</p>
                        </li>
                        <li>
                           <p>Incubate the slide in a vessel containing PBST for at least 10 minutes. Repeat this step twice.</p>
                        </li>
                        <li>
                           <p>Wick off most of the PBST, but do not allow the worms to dry. Add 20 to 30 &#956;l primary antibody diluted in PBST. Carefully
                              cover the worms with a 20 &times; 20-mm piece of parafilm. Place the slide in a humid chamber. Incubate overnight at room temperature.
                           </p>
                        </li>
                        <li>
                           <p>Incubate the slide in a vessel containing PBST for at least 10 minutes. Carefully remove the piece of parafilm.</p>
                        </li>
                        <li>
                           <p>Incubate the slide in a vessel containing fresh PBST for at least 10 minutes. Repeat this step once.</p>
                        </li>
                        <li>
                           <p>Wick off most of the PBST, but do not allow the worms to dry. Add 20 to 30 &#956;l secondary antibody diluted in PBST. Carefully
                              cover the worms with a 20 &times; 20-mm piece of parafilm. Place the slide in a humid chamber. Incubate overnight at room temperature.
                              Keep the slides in the dark whenever possible.
                           </p>
                        </li>
                        <li>
                           <p>Incubate the slide in a vessel containing PBST for at least 10 minutes. Carefully remove the piece of parafilm.</p>
                        </li>
                        <li>
                           <p>Incubate the slide in a vessel containing fresh PBST for at least 10 minutes. Repeat this step once.</p>
                        </li>
                        <li>
                           <p>Wick off most of the PBST, but do not allow the worms to dry. Add 10 &#956;l of 1 &#956;g/ml DAPI or 1 &#956;g/ml propidium iodide to the
                              slide.
                           </p>
                        </li>
                        <li>
                           <p>Mount the slide with 20 &#956;l anti-fade solution and a 22 &times; 40-mm cover glass. Seal the slide with fingernail polish.</p>
                        </li>
                        <li>
                           <p>Observe the slide. Store the slides at <span class="symbol">&#8722;</span>20&deg;C. Keep the slides in the dark whenever possible.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <div class="table"><a name="d0e2673"></a><p class="title">Buffers/reagents</p>
                     <table summary="Buffers/reagents" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <tbody>
                           <tr>
                              <td align="left">Sperm Salts:</td>
                              <td align="left">50 mM PIPES, pH 7.0</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">25 mM KCl</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">1 mM MgSO<sub>4</sub></td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">45 mM NaCl</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">2 mM CaCl<sub>2</sub></td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="table"><a name="d0e2705"></a><p class="title"></p>
                     <table summary="" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <tbody>
                           <tr>
                              <td align="left">PBST:</td>
                              <td align="left">10 ml 10X PBS</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">2.5 ml 20% Triton X-100</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">0.2 ml 0.5 M EDTA, pH 8</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">87.3 ml ddH<sub>2</sub>O
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="table"><a name="d0e2731"></a><p class="title"></p>
                     <table summary="" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <tbody>
                           <tr>
                              <td align="left">DABCO:</td>
                              <td align="left">50 ml total</td>
                           </tr>
                           <tr>
                              <td align="left">5 g DABCO</td>
                              <td align="left">&nbsp;</td>
                           </tr>
                           <tr>
                              <td align="left">5 ml PBS</td>
                              <td align="left">&nbsp;</td>
                           </tr>
                           <tr>
                              <td align="left">45 ml glycerin</td>
                              <td align="left">&nbsp;</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <p>Notes:</p>
                  <p>When trying an antibody for the first time, it is often good to try all three fixative concentrations for 5 minutes each (Step
                     #1). This gives you a baseline for adjusting the protocol for your specific antibody. You might notice certain trends. For
                     example, when you use lower fixative concentrations, you might find that the antibody staining becomes brighter, but the structure
                     of the chromosomes becomes worse. As you use higher fixative concentrations, the antibody staining might be worse (dimmer,
                     penetration not as good), but the structure might be better. If this is the case and you need the chromosomes to have better
                     structure, you might try incubating in a lower fixative concentration for a longer time period or in a higher fixative concentration
                     for less time. If penetration is not good with the lowest fixative concentration that you tried, you might try an even lower
                     concentration, or you might try less time in the lower fixative concentration (e.g., 2 minutes, instead of 5 minutes). You
                     can try higher concentrations of the PFA and longer incubation times in the PFA for better structure.
                  </p>
                  <p>The fixative concentration, fixation time, and freeze-cracking step (Step #2) are very important. You can see great differences
                     in staining with small changes in fixative concentration and fixation time, and it is important to get a good freeze-crack.
                  </p>
                  <p>You can tell that the penetration of the antibody was not good when nothing stains or just the outer surface or outer layer
                     of cells stains.
                  </p>
                  <p>If you see that worms on one part of a slide stain but worms on another part do not stain, you may be having freeze-cracking
                     problems (Step #2).
                  </p>
                  <p>If you leave the slides on the dry ice for too long (Step #1d), the slides can get icy, and you might not get a good freeze-crack
                     (Step #2).
                  </p>
                  <p>For a new antibody, try a concentration gradient with the antibody. The staining is sometimes greatly improved with lower
                     concentrations of some antibodies.
                  </p>
                  <p>Removing the ethanol step (Step #3) can produce dramatic changes in the antibody staining. It is good to try the protocol
                     with and without the ethanol step. Ethanol is a coagulating fixative that denatures proteins and changes the protoplasm into
                     an artificially interconnected network. A short dip in ethanol can coagulate proteins and boost signal, but it might also
                     increase &#8220;background.&#8221; Ethanol can also cause signal to &#8220;disappear&#8221; in some cases.
                  </p>
                  <p>This protocol does not have any block. First, try a new antibody without block to see everything that the antibody sticks
                     to without block. Then you can try adding block, such as BSA or fish gelatin (Cold Water Fish Skin Gelatin 40%, catalog #25560,
                     Electron Microscopy Sciences). You might find that block is not needed or that block causes greater background staining rather
                     than lower background staining.
                  </p>
                  <p>The &#8220;humid chamber&#8221; can be any container that can be sealed with wet paper towels inside.</p>
                  <p>Swirl the slide around a little after adding the fixative in Step 1c. You can place the cover glass on the slide shortly after
                     mixing the fixative and the sperm salt solution together in Step 1d. This can help to get the worms to stick to the slides
                     better.
                  </p>
                  <p>The incubation times with both the primary and the secondary antibodies can often be less. For example, you can try incubating
                     in secondary antibody for four hours at room temperature.
                  </p>
                  <p>Several different anti-fade solutions can be used, e.g., VECTASHIELD, ProLong, and DABCO. DABCO does not work well with Cy5.
                     It may not be good to use DABCO with any of the Cy dyes; however, I have not had a problem with DABCO and Cy3.
                  </p>
                  <p>You can use smaller cover glasses. Just use less DNA stain and anti-fade solution.</p>
                  <p>People often get variable results when they first use this protocol. Some slides have good antibody staining, and some do
                     not. This may be because they are still learning the technique. For example, if you dissect the worms too slowly in Step 1b,
                     sperm salt solution will evaporate, changing the final fixative concentration. As you get better and better at dissecting
                     the worms, the final fixative concentration is closer to the desired concentration, and the antibody staining becomes more
                     consistent.
                  </p>
                  <p>You can add levamisole or sodium azide to help you to immobilize the worms before dissecting them, but be careful with how
                     these chemicals might affect structure.
                  </p>
                  <p>This protocol was originally designed to look at worm gonads, but it was found to work with gut nuclei as well. To look at
                     structures other than gonads, you may need to adjust the protocol. For example, egg buffer might be better for embryos. This
                     protocol was not designed for embryo staining, although embryos do sometimes stain with antibody with this procedure. Often
                     few embryos stain, and the condition of the embryos is not good. Embryos often do not stick to the slides well with this protocol
                     and fall off during the procedure. Using <span class="symbol">&#8722;</span>20&deg;C 95% ethanol, instead of room temperature 95% ethanol, in Step #3 might help with this problem. You could also try omitting
                     the ethanol step.
                  </p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 22: Observing chromosomes in dissected worms (Annette Chan and Barbara Meyer)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Use 2% PFA with 5-minute fix:</p>
                           <div class="orderedlist">
                              <ol type="a">
                                 <li>
                                    <p>Place worms in 5 &#956;l 1X sperm salts on a positively-charged glass slide.</p>
                                 </li>
                                 <li>
                                    <p>Using a needle (Precision Glide<sup>&reg;</sup> Needle, Becton Dickinson &amp; Co., 22 G 1 1/2), make an incision between the pharynx and gonad (closer to the gonad than the
                                       pharynx) or between the tip of the tail and gonad of each worm. To release the embryos, cut at the vulva.
                                    </p>
                                 </li>
                                 <li>
                                    <p>Add 5 &#956;l of 4% PFA (Electron Microscopy Sciences, cat. #15710) in 1X sperm salts.</p>
                                    <div class="table"><a name="d0e2811"></a><p class="title"></p>
                                       <table summary="" border="1">
                                          <colgroup>
                                             <col align="left">
                                             <col align="left">
                                          </colgroup>
                                          <tbody>
                                             <tr>
                                                <td align="left">To prepare 4% PFA:</td>
                                                <td align="left">250 &#956;l 16% PFA</td>
                                             </tr>
                                             <tr>
                                                <td align="left">&nbsp;</td>
                                                <td align="left">200 &#956;l 5X sperm salts</td>
                                             </tr>
                                             <tr>
                                                <td align="left">&nbsp;</td>
                                                <td align="left">550 &#956;l water</td>
                                             </tr>
                                          </tbody>
                                       </table>
                                    </div>
                                 </li>
                                 <li>
                                    <p>Place the slide in a humid chamber for 5 minutes. Place an 18-mm &times; 18-mm cover glass on top of the worms. Freeze the slide
                                       on a piece of dry ice. Let the slide sit on the dry ice for at least 10 minutes.
                                    </p>
                                 </li>
                              </ol>
                           </div>
                        </li>
                        <li>
                           <p>Remove the slide from the dry ice and quickly pop off the cover glass with a quick downward stroke of a single-edged razor.</p>
                        </li>
                        <li>
                           <p>Immediately place the slide in 95% ethanol for 1 minute. Be careful or the worms will fall off.</p>
                        </li>
                        <li>
                           <p>Incubate the slide in a vessel containing PBST for at least 10 minutes. Repeat this step twice.</p>
                        </li>
                        <li>
                           <p>Wick off most of the PBST, but do not allow the worms to dry. Add 10 &#956;l of 1 &#956;g/ml DAPI or propidium iodide to the slide.</p>
                        </li>
                        <li>
                           <p>Mount the slide with 20 &#956;l anti-fade solution and a 22 &times; 40-mm cover glass. Seal the slide with fingernail polish.</p>
                        </li>
                        <li>
                           <p>Observe the slide. Store the slides at <span class="symbol">&#8722;</span>20&deg;C. Keep the slides in the dark whenever possible.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <div class="table"><a name="d0e2854"></a><p class="title">Solutions/Reagents</p>
                     <table summary="Solutions/Reagents" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <tbody>
                           <tr>
                              <td align="left">Sperm Salts:</td>
                              <td align="left">50 mM PIPES, pH 7.0</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">25 mM KCl</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">1 mM MgSO<sub>4</sub></td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">45 mM NaCl</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">2 mM CaCl<sub>2</sub></td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="table"><a name="d0e2886"></a><p class="title"></p>
                     <table summary="" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <tbody>
                           <tr>
                              <td align="left">PBST:</td>
                              <td align="left">10 ml 10X PBS</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">2.5 ml 20% Triton X-100</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">0.2 ml 0.5 M EDTA, pH 8</td>
                           </tr>
                           <tr>
                              <td align="left">&nbsp;</td>
                              <td align="left">87.3 ml ddH<sub>2</sub>O
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="table"><a name="d0e2912"></a><p class="title"></p>
                     <table summary="" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <tbody>
                           <tr>
                              <td align="left">DABCO:</td>
                              <td align="left">50 ml total</td>
                           </tr>
                           <tr>
                              <td align="left">5 g DABCO</td>
                              <td align="left">&nbsp;</td>
                           </tr>
                           <tr>
                              <td align="left">5 ml PBS</td>
                              <td align="left">&nbsp;</td>
                           </tr>
                           <tr>
                              <td align="left">45 ml glycerin</td>
                              <td align="left">&nbsp;</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <p><span class="bold"><strong>Notes:</strong></span></p>
                  <p>You can try higher concentrations of the PFA and longer incubation times in the PFA for better structure.</p>
                  <p>Several different anti-fade solutions can be used, e.g., VECTASHIELD, ProLong, and DABCO.</p>
                  <p>You can use smaller cover glasses. Just use less DAPI and anti-fade solution.</p>
                  <p>This protocol works well with the gonad and the gut. Embryos often don't stick to the slides well with this protocol and fall
                     off during the procedure. Using <span class="symbol">&#8722;</span>20&deg;C 95% ethanol, instead of room temperature 95% ethanol, in Step #3 might help with this problem. You could also try omitting
                     the ethanol step.
                  </p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 23: Observing chromosomes in whole worms using Streck tissue fixative (Annette Chan and Barbara Meyer)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Pick several worms and place them in 200 &#956;l Streck Tissue Fixative (S.T.F., Streck Laboratories, Inc., catalog #265180) in
                              a glass-welled plate.
                           </p>
                        </li>
                        <li>
                           <p>Incubate worms in the fixative overnight at room temperature in a humid chamber.</p>
                        </li>
                        <li>
                           <p>Carefully remove the fixative from the glass-welled plate with a tip for a 1&#8211;200 ul pipettor. Add about 200 &#956;l PBS. Rinse
                              the worms. Repeat this wash step until the wash solution is clear.
                           </p>
                        </li>
                        <li>
                           <p>Remove the PBS and add about 200 &#956;l 95% ethanol. Let sit for 1 minute. Swirl the worms around to get them to sink to the bottom
                              of the well.
                           </p>
                        </li>
                        <li>
                           <p>Remove the ethanol and add 200 &#956;l PBS. Rinse the worms. Repeat this wash step at least twice.</p>
                        </li>
                        <li>
                           <p>Make sure the worms sink to the bottom of the well before attempting to pick them up. Place the worms on a positively-charged
                              glass slide using a glass Pasteur pipette. Wick away the liquid, but do not allow the worms to dry.
                           </p>
                        </li>
                        <li>
                           <p>Add 10 &#956;l of 1 &#956;g/ml DAPI to the slide.</p>
                        </li>
                        <li>
                           <p>Mount the slide with 10 &#956;l anti-fade solution and a 22 &times; 40-mm cover glass. Seal slide with fingernail polish.</p>
                        </li>
                        <li>
                           <p>Observe the slide. Store at <span class="symbol">&#8722;</span>20&deg;C.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>Notes:</p>
                  <p>Avoid pipetting the worms into the tip because they tend to stick to plastic</p>
                  <p>For Step #4, it can be hard to remove all of the PBS and then add ethanol. If this is the case, remove as much PBS as possible
                     in Step #4, and then add ethanol. Swirl the ethanol/PBS mixture around a little. Remove as much liquid as possible, and then
                     add fresh 95% ethanol. Now, let the worms sit for 1 minute.
                  </p>
                  <p>Worms are often lost during Step #6 while trying to wick away liquid. If this is a problem, instead of doing Steps #6 to #8,
                     put DAPI and anti-fade solution on a slide, pick worms into the DAPI and anti-fade solution, mount the slide with a cover
                     glass, and then seal the slide with fingernail polish. You can also put DAPI on a slide, pick worms into the DAPI, add anti-fade
                     solution, swirl a little, mount the slide with a cover glass, and then seal the slide with fingernail polish.
                  </p>
                  <p>Several different anti-fade solutions can be used, e.g., VECTASHIELD, ProLong, and DABCO.</p>
                  <p>You can use smaller cover glasses. Just use less DAPI and anti-fade solution.</p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 24: Observing chromosomes in whole worms using paraformaldehyde (Annette Chan and Barbara Meyer)</b></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Pick several worms and place them in 200 &#956;l 8% PFA (Electron Microscopy Sciences, cat. #15710) in a glass-welled plate.</p>
                           <div class="table"><a name="d0e3002"></a><p class="title"></p>
                              <table summary="" border="1">
                                 <colgroup>
                                    <col align="left">
                                    <col align="left">
                                 </colgroup>
                                 <tbody>
                                    <tr>
                                       <td align="left">To prepare 8% PFA:</td>
                                       <td align="left">100 &#956;l 16% PFA</td>
                                    </tr>
                                    <tr>
                                       <td align="left">&nbsp;</td>
                                       <td align="left">10 &#956;l 20X M9 buffer</td>
                                    </tr>
                                    <tr>
                                       <td align="left">&nbsp;</td>
                                       <td align="left">90 &#956;l water</td>
                                    </tr>
                                 </tbody>
                              </table>
                           </div>
                        </li>
                        <li>
                           <p>Incubate worms in the fixative for 1 hour at room temperature in a humid chamber.</p>
                        </li>
                        <li>
                           <p>Carefully remove the fixative from the glass-welled plate with a tip for a 1&#8211;200 ul pipettor. Add about 200 &#956;l PBS. Rinse
                              the worms. Repeat this wash step until the wash solution is clear.
                           </p>
                        </li>
                        <li>
                           <p>Remove the PBS and add about 200 &#956;l 95% ethanol. Let sit for 1 minute. Swirl the worms around to get them to sink to the bottom
                              of the well.
                           </p>
                        </li>
                        <li>
                           <p>Remove the ethanol and add 200 &#956;l PBS. Rinse the worms. Repeat this wash step twice.</p>
                        </li>
                        <li>
                           <p>Make sure the worms sink to the bottom of the well before attempting to pick them up. Place the worms on a positively-charged
                              glass slide using a glass Pasteur pipette. Wick away the liquid, but do not allow the worms to dry.
                           </p>
                        </li>
                        <li>
                           <p>Add 10 &#956;l of 1 &#956;g/ml DAPI to the slide.</p>
                        </li>
                        <li>
                           <p>Mount the slide with 10 &#956;l anti-fade solution and a 22 &times; 40-mm cover glass. Seal slide with fingernail polish.</p>
                        </li>
                        <li>
                           <p>Observe the slide. Store at <span class="symbol">&#8722;</span>20&deg;C.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>Notes:</p>
                  <p>Avoid pipetting the worms into the tip because they tend to stick to plastic.</p>
                  <p>For Step #4, it can be hard to remove all of the PBS and then add ethanol. If this is the case, remove as much PBS as possible
                     in Step #4, and then add ethanol. Swirl the ethanol/PBS mixture around a little. Remove as much liquid as possible, and then
                     add fresh 95% ethanol. Now, let the worms sit for 1 minute.
                  </p>
                  <p>Worms are often lost during Step #6 while trying to wick away liquid. If this is a problem, instead of doing Steps #6 to #8,
                     put DAPI and anti-fade solution on a slide, pick worms into the DAPI and anti-fade solution, mount the slide with a cover
                     glass, and then seal the slide with fingernail polish. You can also put DAPI on a slide, pick worms into the DAPI, add anti-fade
                     solution, swirl a little, mount the slide with a cover glass, and then seal the slide with fingernail polish.
                  </p>
                  <p>Several different anti-fade solutions can be used, e.g., VECTASHIELD, ProLong, and DABCO.</p>
                  <p>You can use smaller cover glasses. Just use less DAPI and anti-fade solution.</p>
               </div>
               <div class="sidebar">
                  <p class="title"><b>Protocol 25: DNA FISH and DNA FISH followed by Immunofluorescence (Gy&ouml;rgyi Csankovszki and Barbara Meyer)</b></p>
                  <p><span class="underline">Preparation of FISH probe.</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Label probe by nick translation or random priming using fluorescent nucleotides, such as cy3-dCTP, cy5-dCTP (Amersham), or
                              fluorescein-dUTP (Roche).
                           </p>
                        </li>
                        <li>
                           <p>Remove unincorporated nucleotides using a spin column.</p>
                        </li>
                        <li>
                           <p>Ethanol precipitate probe together with 40 &#956;L salmon sperm DNA (10mg/mL).</p>
                        </li>
                        <li>
                           <p>Wash pellet in 70% ethanol, and dry well (air dry or speed vac).</p>
                        </li>
                        <li>
                           <p>Resuspend pellet in 160 &#956;L formamide.</p>
                        </li>
                        <li>
                           <p>Denature probe at 70&#8211;74&deg;C, 10 minutes.</p>
                        </li>
                        <li>
                           <p>Chill on ice.</p>
                        </li>
                        <li>
                           <p>Add 160 &#956;L 2X hyb (1 part 20X SSC, 2 parts 10 mg/mL BSA, 2 parts 50% dextran sulfate)</p>
                        </li>
                        <li>
                           <p>The probe is ready to use as is and can be stored at <span class="symbol">&#8722;</span>20&deg;C for months.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Fixation of worms for staining the gut or the germline.</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Pick about 40 animals onto a drop (5 &#956;L) of sperm salts (50 mM PIPES pH7, 25 mM KCl, 1 mM MgSO<sub>4</sub>, 45 mM NaCl, 2 mM CaCl<sub>2</sub>) on a microscope slide. Cut to release gonads and gut. Add 5 &#956;L of fixative (2&#8211;4% PFA in sperm salts) and fix for 5 minutes
                              in a humid chamber.
                           </p>
                        </li>
                        <li>
                           <p>Place an 18&times;18 mm coverslip on worms and place the slide on a dry ice block for 10 minutes.</p>
                        </li>
                        <li>
                           <p>Remove coverslip with a single-edged razor blade and immediately place the slide in PBST (1&times; PBS, I mM EDTA, 0.5% Triton X-100)
                              at RT in a coplin jar.
                           </p>
                        </li>
                        <li>
                           <p>Wash slides in PBST 3 &times; 10 minutes.</p>
                        </li>
                        <li>
                           <p>Place slides in 70% ethanol for 2 minutes.</p>
                        </li>
                        <li>
                           <p>80% ethanol for 2 minutes, 95% ethanol for 2 minutes, 100% ethanol for 2 minutes.</p>
                        </li>
                        <li>
                           <p>Air dry. Proceed with FISH.</p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Fixation of embryos.</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Bleach gravid hermaphrodites, wash and pellet embryos using standard methods.</p>
                        </li>
                        <li>
                           <p>Add and equal volume of fixative (4% PFA, 1&times; sperm salts) to the embryo pellet.</p>
                        </li>
                        <li>
                           <p>Pipette 10 &#956;L of embryos to each slide and cover with an 18 mm &times; 18 mm coverslip. Incubate in humid chamber for a total of
                              5 minutes from the time fixative was added in step 2.
                           </p>
                        </li>
                        <li>
                           <p>Place slides on a block of dry ice for 10 minutes.</p>
                        </li>
                        <li>
                           <p>Using a razor blade, remove coverslip and immerse slides in RT PBST.</p>
                        </li>
                        <li>
                           <p>Wash slides in PBST 3 times, 10 minutes. Handle slides carefully, otherwise embryos will fall off.</p>
                        </li>
                        <li>
                           <p>Place slides in 70% ethanol for 2 minutes.</p>
                        </li>
                        <li>
                           <p>80% ethanol for 2 minutes, 95% ethanol for 2 minutes, 100% ethanol for 2 minutes.</p>
                        </li>
                        <li>
                           <p>Air dry. Proceed with FISH.</p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">FISH</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Add 10&#8211;20 &#956;L of FISH probe and cover with coverslip.</p>
                        </li>
                        <li>
                           <p>Place slide on a 95&deg;C heat block for 3 minutes to denature the sample.</p>
                        </li>
                        <li>
                           <p>Incubate in humid chamber at 37&deg;C overnight.</p>
                        </li>
                        <li>
                           <p>Wash slide in 2X SSC/50% formamide, 5 minutes at 39&deg;C, 3 times.</p>
                        </li>
                        <li>
                           <p>Wash in 2X SSC, 5 minutes at 39&deg;C, 3 times.</p>
                        </li>
                        <li>
                           <p>Wash in 1X SSC, 10 minutes at 39&deg;C, once.</p>
                        </li>
                        <li>
                           <p>For immunofluorescence, proceed to the next section, otherwise wash slides once more in 4X SSC/0.01 &#956;g/mL DAPI. Mount the
                              slides using Vectashield (Vector Labs) and store them at <span class="symbol">&#8722;</span>20&deg;C.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Immunofluorescence.</span></p>
                  <p>(This part follows the Meyer lab staining protocol developed by Annette Chan.)</p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Wash slides in PBST.</p>
                        </li>
                        <li>
                           <p>Without letting the worms dry, wick off most of the PBST, add 30 &#956;L primary antibody diluted in PBST, cover sample with a
                              small piece of parafilm. Incubate in humid chamber at RT overnight, or 37&deg;C for 4 hours. (Temperature and duration of incubation
                              is antibody dependent.)
                           </p>
                        </li>
                        <li>
                           <p>Wash slides 3 times in PBST, 10 minutes each.</p>
                        </li>
                        <li>
                           <p>Add 30 &#956;L secondary antibody diluted in PBST, cover worms with parafilm, incubate at RT for 4 hours or 37&deg;C for 1 hour in
                              a humid chamber.
                           </p>
                        </li>
                        <li>
                           <p>Wash slides 3 times in PBST, 10 minutes each. In the last wash, to 50 &#956;L of PBST add 0.5 &#956;L of 1 mg/mL DAPI to counterstain
                              DNA.
                           </p>
                        </li>
                        <li>
                           <p>Mount the slides using Vectashield, and store them at <span class="symbol">&#8722;</span>20&deg;C.
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e3214"></a>4.3.&nbsp;Programmed cell death (Barbara Conradt, Julia Hatzold, Claus Schertl)
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e3217"></a>4.3.1.&nbsp;Detecting dying cells
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Dying cells can be detected by their morphology and refractility using differential interference contrast microscopy (DIC;
                     see above; <a href="#bib31">Sulston and Horvitz, 1977</a>). A cell corpse appears as a highly refractile, button-like structure (<a href="#figure13">Figure 13</a>), which is rapidly engulfed and degraded by neighboring cells. Cell death occurs in the <span class="emphasis"><em>C. elegans</em></span> soma mainly during embryogenesis and in the female germ line as a normal course of development (<a href="#bib17">Gumienny et al., 1999</a>). Germ-cell death can also be induced by DNA-damaging agents or by pathogenic bacteria (<a href="#bib1">Aballay and Ausubel, 2001</a>; <a href="#bib14">Gartner et al., 2000</a>). Below we present several protocols for detecting dying cells in the soma and germ line.
                  </p>
                  <div class="mediaobject" align="center"><a name="figure13"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="cellfig13.jpg"><img src="cellfig13_s.jpg" border="2" align="middle" alt=" figure 13"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 13.&nbsp;</b>DIC image of wild-type embryo at comma stage of development. Arrows point to the round, button-like corpses (Julia Hatzold
                           and Barbara Conradt).
                        </p>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e3256"></a>4.3.1.1.&nbsp;TUNEL labeling
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>TUNEL (<span class="emphasis"><em>T</em></span>dT-mediated d<span class="underline">U</span>TP <span class="underline">N</span>ick <span class="underline">E</span>nd <span class="underline">L</span>abeling) labels DNA ends, and therefore also dying cells, in which DNA becomes degraded (<a href="#bib15">Gavrieli et al., 1992</a>). TUNEL can be used as a marker for dying cells and to analyze DNA degradation. DNA degradation occurs very rapidly during
                        cell death, thus, TUNEL stains apoptotic cells only during a transient stage of DNA degradation. For this reason, usually
                        only a small subset of corpses is labeled with this technique. However, in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=nuc-1" target="_blank">nuc-1</a></em></span> mutants, which are defective in some aspects of DNA degradation, the nuclei of many dying cells can be labeled using this
                        method (<a href="#bib37">Wu et al., 2000</a>).
                     </p>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 26: TUNEL staining for embryos (based on <a href="#bib37">Wu et al., 2000</a>)</b></p>
                        <p>Animals are washed from one to three 15-mm petri plates with water and treated with hypochlorite to obtain the embryo.</p>
                        <p>A total of 1 ml of fixation solution is added to the embryos, which are immediately frozen in liquid nitrogen. The frozen
                           embryos are thawed in a water bath at room temperature for 2 min and rocked at room temperature for 25 min. Fixed embryos
                           are washed once in 1 ml of Tris-Triton buffer and three times in 1 ml of PBST for 10 min each. Approximately 3 &#956;l of packed
                           embryos are preincubated with 25 &#956;l of TdT reaction buffer for 5 min at room temperature. TdT reaction buffer is then replaced
                           with TdT reaction buffer containing 0.2 units of TdT (Boehringer Mannheim), 6.6 nM dUTP (Boehringer Mannheim) and 3.3 nM fluorescein-11-dUTP
                           (Boehringer Mannheim), and incubated for 2 hr at 37&deg;C. After incubation, embryos are washed in PBST three times for 10 min.
                           Embryos are mounted in Vectashield mounting medium (Vector Laboratories).
                        </p>
                        <p>
                           
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td><span class="underline">fixation solution:</span></td>
                           </tr>
                           <tr>
                              <td>80 mM KCl</td>
                           </tr>
                           <tr>
                              <td>20 mM NaCl</td>
                           </tr>
                           <tr>
                              <td>1.3 mM EGTA</td>
                           </tr>
                           <tr>
                              <td>3.2 mM spermine</td>
                           </tr>
                           <tr>
                              <td>7.5 mM sodium HEPES (pH 6.5)</td>
                           </tr>
                           <tr>
                              <td>25% methanol</td>
                           </tr>
                           <tr>
                              <td>2% paraformaldehyde</td>
                           </tr>
                           <tr>
                              <td>0.4% glutaraldehyde 98</td>
                           </tr>
                        </table>
                        <p>
                           
                        </p>
                        <p>
                           
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td><span class="underline">Tris-Triton buffer :</span></td>
                           </tr>
                           <tr>
                              <td>1% Triton X-100</td>
                           </tr>
                           <tr>
                              <td>100 mM Tris at pH 7.4</td>
                           </tr>
                        </table>
                        <p>
                           
                        </p>
                        <p>
                           
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td><span class="underline">PBST</span></td>
                           </tr>
                           <tr>
                              <td>PBS containing 0.5% Triton X-100</td>
                           </tr>
                        </table>
                        <p>
                           
                        </p>
                        <p>
                           
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td><span class="underline">TdT reaction buffer:</span></td>
                           </tr>
                           <tr>
                              <td>200 mM sodium cacodylate</td>
                           </tr>
                           <tr>
                              <td>25 mM Tris-HCl</td>
                           </tr>
                           <tr>
                              <td>0.25 mg/ml BSA</td>
                           </tr>
                           <tr>
                              <td>0.1% Triton X-100</td>
                           </tr>
                           <tr>
                              <td>1.5 mM cobalt chloride at pH 6.6</td>
                           </tr>
                        </table>
                        <p>
                           
                        </p>
                     </div>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 27: TUNEL labeling of dying germ cells (based on <a href="#bib13">Francis et al., 1995</a>)</b></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Pick clean worms into a drop of 180 &#956;l PBLE, in a watch glass.</p>
                              </li>
                              <li>
                                 <p>Cut off heads using 28 G needles.</p>
                              </li>
                              <li>
                                 <p>Add 19 &#956;l 37% formaldehyde.</p>
                              </li>
                              <li>
                                 <p>Fix for 60 minutes at room temp. in humidified chamber.</p>
                              </li>
                              <li>
                                 <p>Remove most of liquid using a finely drawn capillary.</p>
                              </li>
                              <li>
                                 <p>Add 500 &#956;l of <span class="symbol">&#8722;</span>20&deg;C 100% methanol.
                                 </p>
                              </li>
                              <li>
                                 <p>Fix at room temperature for 3 minutes.</p>
                              </li>
                              <li>
                                 <p>Remove ~1/2 of the methanol and add 1 ml PBS.</p>
                              </li>
                              <li>
                                 <p>Remove most of the liquid and add 1 ml PBS. Repeat once more.</p>
                              </li>
                              <li>
                                 <p>Add 1 ml PBST and then remove most of liquid (leaving ~20 &#956;l).</p>
                              </li>
                           </ol>
                        </div>
                        <p>For the labeling of corpses in dissected gonads the <span class="emphasis"><em>In Situ Cell Death Detection Kit, Fluorescein</em></span> (Roche) was used.
                        </p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Mix 5 &#956;l of Enzyme Solution with 45 &#956;l of Label Solution (for a negative control use Label Solution only)</p>
                              </li>
                              <li>
                                 <p>Incubate samples in this mix for 1 hour in the dark at 37&deg;C</p>
                              </li>
                              <li>
                                 <p>Wash twice with PBS</p>
                              </li>
                           </ol>
                        </div>
                        <p>It is possible to do an additional DAPI staining to visualize the DNA in neighboring healthy cells. Two additional washing
                           steps that include the DAPI reagent are necessary right before the gonads are mounted on slides:
                        </p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Remove most liquid and add 1 ml PBST and 2 &#956;l 100 &#956;g/ml DAPI.</p>
                              </li>
                              <li>
                                 <p>Incubate at room temperature for 10 min</p>
                              </li>
                              <li>
                                 <p>Transfer animals onto microscope slide</p>
                              </li>
                              <li>
                                 <p>Remove almost all liquid using drawn capillary</p>
                              </li>
                              <li>
                                 <p>Immediately add 10 &#956;l VECTASHIELD antifade mounting medium (Vector Laboratories, H-1000)</p>
                              </li>
                              <li>
                                 <p>Center worms in drop and add cover glass</p>
                              </li>
                              <li>
                                 <p>Seal with nail polish</p>
                              </li>
                           </ol>
                        </div>
                        <p>
                           
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td><span class="underline">PBS: (stored at 4&deg;C)</span></td>
                           </tr>
                           <tr>
                              <td>1.35 ml 5M NaCl</td>
                           </tr>
                           <tr>
                              <td>250 &#956;l 2M K<sub>2</sub>HPO<sub>4</sub></td>
                           </tr>
                           <tr>
                              <td>90 &#956;l 1M KH<sub>2</sub>PO<sub>4</sub></td>
                           </tr>
                           <tr>
                              <td>135 &#956;l 1M KCl</td>
                           </tr>
                           <tr>
                              <td>Plus H<sub>2</sub>O to 50 ml
                              </td>
                           </tr>
                           <tr>
                              <td>25 &#956;l 1 M NaN<sub>3</sub> (to 0.5mM)
                              </td>
                           </tr>
                        </table>
                        <p>
                           
                        </p>
                        <p>
                           
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td><span class="underline">PBST: (stored at 4&deg;)</span></td>
                           </tr>
                           <tr>
                              <td>5 ml PBS</td>
                           </tr>
                           <tr>
                              <td>5 &#956;l Tween 20</td>
                           </tr>
                        </table>
                        <p>
                           
                        </p>
                        <p>
                           
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td><span class="underline">FCST: (stored at 4&deg;)</span></td>
                           </tr>
                           <tr>
                              <td>4 ml PBS</td>
                           </tr>
                           <tr>
                              <td>500 &#956;l Fetal Bovine Serum</td>
                           </tr>
                        </table>
                        <p>
                           
                        </p>
                        <p>
                           
                        </p>
                        <table class="simplelist" border="0" summary="Simple list">
                           <tr>
                              <td><span class="underline">PBLE: (make fresh each time)</span></td>
                           </tr>
                           <tr>
                              <td>1 ml PBS</td>
                           </tr>
                           <tr>
                              <td>1 &#956;l 250mM Levamisole</td>
                           </tr>
                           <tr>
                              <td>20 &#956;l 0.5M EDTA</td>
                           </tr>
                        </table>
                        <p>
                           
                        </p>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e3509"></a>4.3.1.2.&nbsp;SYTO dyes
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>SYTO dyes (Molecular Probes) clearly label DNA in all cells. However, condensed DNA in apoptotic cells (&#8220;dead nuclei&#8221;) stains
                        more brightly than DNA in living cells. SYTO11, SYTO12, and Acridine Orange have given the best results (<a href="#bib17">Gumienny et al., 1999</a>; <a href="#bib37">Wu et al., 2000</a>; <a href="#figure14">Figure 14</a>).
                     </p>
                     <div class="mediaobject" align="center"><a name="figure14"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="cellfig14.jpg"><img src="cellfig14_s.jpg" border="2" align="middle" alt=" figure 14"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 14.&nbsp;</b>A. wild-type DIC view of X-ray induced germ-cell death. B. SYTO12 staining of A. C. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span>(<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n717;class=Variation" target="_blank">n717</a></em></span>) DIC. D. SYTO12 staining of C. Arrows point at corpses (Shai Shaham).
                           </p>
                        </div>
                     </div>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 28: SYTO11 staining of dying somatic cells (based on <a href="#bib37">Wu et al., 2000</a>)</b></p>
                        <p>Worms are washed off plates with M9 solution, collected in a 1.5 ml eppendorf tube, and washed once in M9. The washed worms
                           are incubated with 1 ml of 10 nM SYTO 11 (Molecular Probes) in M9 and rocked at room temperature for 1.5 hrs. The stained
                           worms are washed once in M9 and spotted onto a plate for recovery for at least 30 mins. Worms are mounted on 4% agar pads
                           with 20 mM NaN<sub>3</sub> and visualized using fluorescence microscopy and Nomarski optics to score cell corpses. Staining is observed with a DAPI
                           or FITC filter set.
                        </p>
                     </div>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 29: SYTO staining of dying germ cells (based on <a href="#bib17">Gumienny et al., 1999</a>)</b></p>
                        <p>SYTO dyes (Molecular Probes) stain DNA in the nucleus of all cells. Staining in apoptotic cells is usually much brighter as
                           the DNA is more condensed in these cells. A good signal to noise ratio was reported for SYTO 11 (Molecular Probes S7574) and
                           12 (Molecular Probes S7574).
                        </p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Adults are incubated for 4&#8211;5 hours at 23&deg;C in a 33 &#956;M aqueous solution of SYTO 12 in the dark.</p>
                              </li>
                              <li>
                                 <p>Transfer for 1 hour to NGM plates seeded with <span class="emphasis"><em>E. coli</em></span> OP50.
                                 </p>
                              </li>
                              <li>
                                 <p>Mount worms on slides</p>
                              </li>
                           </ol>
                        </div>
                        <p>In wild-type animals SYTO 12 has a granular cytoplasmic staining pattern in mid- to late stage oocytes and embryos. SYTO 12
                           stains both corpses visible by Nomarski optics as well as late stage germ cell corpses that are not visible by Nomarski optics.
                        </p>
                     </div>
                     <div class="sidebar">
                        <p class="title"><b>Protocol 30: Acridin Orange (AO) staining of dying germ cells (based on <a href="#bib17">Gumienny et al., 1999</a>; <a href="#bib22">Lettre et al., 2004</a>)</b></p>
                        <p>AO is a fluorescent dye that specifically labels apoptotic cells. The dye becomes incorporated by ingestion of AO-stained
                           <span class="emphasis"><em>Escherichia coli</em></span> by <span class="emphasis"><em>C. elegans</em></span>.
                        </p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Adults are incubated for 1 hours in the dark on seeded NGM plates with 500 &#956;l of M9 buffer containing AO (Molecular probes;
                                    0.02 mg/ml).
                                 </p>
                              </li>
                              <li>
                                 <p>Transfer the stained worms back to regular NGM plates (lacking AO) for one hour.</p>
                              </li>
                              <li>
                                 <p>Mount worms on slides and analyze by fluorescence microscopy.</p>
                              </li>
                           </ol>
                        </div>
                        <p>This staining does not work for engulfment defective mutants.</p>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e3612"></a>4.3.1.3.&nbsp;Dying germ cells visualized using <a href="http://www.wormbase.org/db/seq/protein?name=CED-1;class=protein" target="_blank">CED-1</a>::GFP
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>P<span class="emphasis"><em><sub><a href="http://www.wormbase.org/db/gene/gene?name=lim-7" target="_blank">lim-7</a></sub></em></span><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span>::gfp expresses a <a href="http://www.wormbase.org/db/seq/protein?name=CED-1;class=protein" target="_blank">CED-1</a>::GFP fusion protein under the control of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lim-7" target="_blank">lim-7</a></em></span> promoter (<a href="#bib42">Zhou et al., 2001</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lim-7" target="_blank">lim-7</a></em></span> promoter drives expression specifically in the sheath cells that form the gonadal tube. These cells are responsible for the
                        engulfment of dying germ cells. The <a href="http://www.wormbase.org/db/seq/protein?name=CED-1;class=protein" target="_blank">CED-1</a> protein is a transmembrane protein that is expressed on phagocytotic cells. After contact with an apoptotic cell, the <a href="http://www.wormbase.org/db/seq/protein?name=CED-1;class=protein" target="_blank">CED-1</a> protein clusters around the apoptotic cell (<a href="#bib42">Zhou et al., 2001</a>; see <a href="#figure15">Figure 15</a>). This reporter makes it easier to score germ cell death. Even cells at an early apoptotic stage that would hardly be recognizable
                        by DIC are already surrounded by the <a href="http://www.wormbase.org/db/seq/protein?name=CED-1;class=protein" target="_blank">CED-1</a>::GFP protein. One disadvantage of P<span class="emphasis"><em><sub><a href="http://www.wormbase.org/db/gene/gene?name=lim-7" target="_blank">lim-7</a></sub></em></span><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span>::gfp is the fact that it only weakly labels corpses in genetic backgrounds that cause massive germ cell death, like in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span>(lf) animals, in which most, if not all, germ cells die.
                     </p>
                     <div class="mediaobject" align="center"><a name="figure15"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="cellfig15.jpg"><img src="cellfig15_s.jpg" border="2" align="middle" alt=" figure 15"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 15.&nbsp;</b>A. DIC image of a wild-type adult hermaphoridite gonad arm showing a single corpse (arrow). B. DIC image of an engulfment
                              defective mutant. Dying cells are indicated by arrow. C and D. Fluorescence images of the P<sub><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lim-7" target="_blank">lim-7</a></em></span></sub><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-1" target="_blank">ced-1</a></em></span>::gfp reporter corresponding to A and B, respectively (Claus Schertl and Barbara Conradt).
                           </p>
                        </div>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e3691"></a>4.3.2.&nbsp;Detecting a block in cell death
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A structure that is particularly convenient for analyzing in appropriate cell survival is the anterior pharynx. Nuclei in
                     the anterior pharynx are relatively easy to identify in L3 or L4 larvae by DIC microscopy (<a href="#figure16">Figure 16</a>; <a href="#bib7">Ellis and Horvitz, 1991</a>). During the development of the anterior pharynx, 16 cells undergo programmed cell death in wild-type animals. In <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-1" target="_blank">egl-1</a></em></span>(lf), <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span>(lf), <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span>(lf) and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-9" target="_blank">ced-9</a></em></span>(gf) animals, in which cell death is blocked, 11&#8211;12 extra nuclei can be detected on average in the anterior pharynx. Extra
                     cells in the anterior pharynx appear in distinct positions as shown in <a href="#figure16">Figure 16</a>. The number of extra cells is a measure of the extent of the cell death defect.
                  </p>
                  <div class="mediaobject" align="center"><a name="figure16"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="585">
<tr>
  <td align="center"><p><a href="cellfig16.jpg"><img src="cellfig16_s.jpg" border="2" align="middle" alt=" figure 16"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 16.&nbsp;</b>A block in cell death, e.g. caused by a loss-of-function in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span>, results in the appearance of extra cells in the anterior part of the pharynx. A. schematic representation of the anterior
                           part of a wild-type pharynx. B. Schematic representation of the anterior part of a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span>(lf) pharynx. In red, extra nuclei are indicated that can be scored as an assay to evaluate a block in cell death. C-E. Nomarski
                           pictures of pharynxes of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span>(lf) animals. Arrowheads indicate extra nuclei (Julia Hatzold, Barbara Conradt, Ron Ellis).
                        </p>
                     </div>
                  </div>
                  <p>Inappropriate cell survival can also be scored using gfp reporter transgenes. For example, P<sub><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tph-1" target="_blank">tph-1</a></em></span></sub><span class="emphasis"><em>gfp</em></span>, is expressed in the NSM neurons and in their sisters if the latter are allowed to survive (<a href="#bib34">Thellmann et al., 2003</a>; <a href="#figure17">Figure 17</a>).
                  </p>
                  <div class="mediaobject" align="center"><a name="figure17"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="cellfig17.jpg"><img src="cellfig17_s.jpg" border="2" align="middle" alt=" figure 17"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 17.&nbsp;</b>A <span class="emphasis"><em><sub><a href="http://www.wormbase.org/db/gene/gene?name=tph-1" target="_blank">tph-1</a></sub></em></span>::gfp reporter is expressed in the NSMs in the anterior part of the pharynx. A. One fluorescent cell can be detected on each
                           side (left and right) of a wild-type animal. B. In a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ces-1" target="_blank">ces-1</a></em></span>(gf) animal, the surviving NSM sister cell can be visualized by gfp expression posterior to the NSM (Julia Hatzold and Barbara
                           Conradt).
                        </p>
                     </div>
                  </div>
                  <p>Most somatic cell deaths occur during embryogenesis and in wild-type animals the cell corpses are engulfed and rapidly degraded.
                     Therefore, cell corpses are rarely found in the head region of freshly hatched L1 larvae (<a href="#bib31">Sulston and Horvitz, 1977</a>; <a href="#bib32">Sulston et al., 1983</a>; <a href="#bib33">Sulston et al., 1983</a>). In <span class="emphasis"><em>ced-1(<a href="http://www.wormbase.org/db/get?name=e1735;class=Variation" target="_blank">e1735</a>)</em></span> animals, however, engulfment is blocked, resulting in the accumulation of cell corpses in L1 larvae (about 23 corpses). If
                     cell death is blocked in this mutant background, for example by a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span>(lf) mutation, few or no cell corpses are detected (<a href="#bib8">Ellis et al., 1991</a>; (Ellis et al., 1991). Therefore, this assay can also be used to determine if cells that are destined to undergo programmed
                     cell death failed to die.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e3811"></a>4.3.3.&nbsp;Detecting cell corpse engulfment defects
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The analysis of cell corpses in L1 larvae can give information about mutations that result in an engulfment defect. Since
                     113 out of 131 cell deaths occur during wild-type embryogenesis, no cell corpses are detected in the pharynx of L1 larvae
                     <a href="#bib31">Sulston and Horvitz, 1977</a>; <a href="#bib32">Sulston et al., 1983</a>; <a href="#bib33">Sulston et al., 1983</a>). Therefore, mutants, in which persistent corpses can be detected in freshly hatched L1 larvae, might be defective in engulfment.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e3828"></a>5.&nbsp;Embryonic cell culture
                     </h2>
                  </div>
               </div>
            </div>
            <p>Studying cellular processes in a living <span class="emphasis"><em>C. elegans</em></span> offers the advantage of a physiological environment in which these processes occur. However, because of the small size of
               the animal, and because the cuticle of the animal is a significant permeability barrier, many experiments are difficult or
               impossible to perform. Thus, for example, rapid modulation of cellular components using drugs is nearly impossible. Furthermore,
               studies of a particular process may be hampered by other physiological events surrounding the cell being examined, making
               results sometimes difficult to interpret. Cell culture offers an alternative to <span class="emphasis"><em>in vivo</em></span> studies, where problems such as those discussed above are obviated (<a href="#bib4">Christensen et al., 2002</a>; <a href="#bib39">Zhang et al., 2002</a>). Interpretation of experiments using cultured cells, of course, suffers from the non-physiological conditions under which
               the cells are grown. Thus, culture studies and <span class="emphasis"><em>in vivo</em></span> studies can be complementary, and together can yield significant insight into cell biological questions.
            </p>
            <p>Particular cell types can be labeled with GFP, sorted using fluorescence activated cell sorting (FACS), and studied as a homogeneous
               population (e.g. to examine expression using gene arrays). Furthermore, RNA interference can be used to inactivate gene expression
               in cultured cells, allowing for a quasi-genetic dissection of processes in culture. Finally, cells in culture are readily
               accessible to electrophysiological studies. Methods for culturing embryonic cells were recently developed (<a href="#bib4">Christensen et al., 2002</a>; <a href="#bib39">Zhang et al., 2002</a>) and protocols are presented below (<a href="#figure18">Figure 18</a>). The procedure has been mostly tried with embryonic cells. In general, differentiation of cells in culture seems to proceed
               through what would be the L1 stage. Markers specific for later stages are not expressed.
            </p>
            <div class="mediaobject" align="center"><a name="figure18"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="cellfig18.jpg"><img src="cellfig18_s.jpg" border="2" align="middle" alt="figure 18"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 18.&nbsp;</b>Profiling GFP-labeled embryonic cells. Intact embryos are isolated from gravid adults by hypochlorite treatment. 1. Treatment
                     with chitinase releases embryonic blastomeres. 2. Cells marked with GFP reporters expressed in early embryos (i.e., before
                     morphogenesis at ~400 min) are isolated by FACS (e.g. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=myo-3" target="_blank">myo-3</a>::GFP</em></span> body muscle cells) (Green arrows). 3. Alternatively, dissociated blastomeres can be cultured for 24 hr to generate GFP-labeled
                     cells that emerge later in embryonic development (e.g. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a>::GFP</em></span> in DA motor neurons) (brown arrows). 4. GFP-marked cells are isolated by FACS and RNA extracted for application to the <span class="emphasis"><em>C. elegans</em></span> Affymetrix Gene Chip. 5. (Rebecca M. Fox, Susan J. Ruff, and David M. Miller III)
                  </p>
               </div>
            </div>
            <div class="sidebar">
               <p class="title"><b>Protocol 31: Primary culture of embryonic cells (Kevin Strange and Rebecca Morrison)</b></p>
               <p><span class="bold"><strong>Isolation of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>eggs</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Cultures of synchronized worms are prepared using standard methods</p>
                     </li>
                     <li>
                        <p>Wash synchronized adult worms off agar plates with Milli-Q water into a 15 ml conical tube. Pellet worms by centrifugation
                           at 1200 rpm (~450g) for 3 min in a swinging bucket rotor in a tabletop centrifuge.
                        </p>
                     </li>
                     <li>
                        <p>Wash pelleted worms with Milli-Q water 1-3X until the supernatant is clear of bacteria.</p>
                     </li>
                     <li>
                        <p>After the last wash, lyse the worm pellet with the following fresh mixture of bleach and NaOH (1.0 ml bleach, 0.25 ml 10 N
                           NaOH, 3.75 ml sterile H<sub>2</sub>O) prepared immediately prior to use.
                        </p>
                     </li>
                     <li>
                        <p>Rock worms gently during the lysis. The progress of the lysis reaction is monitored by viewing the worm suspension with a
                           dissecting microscope and the lysis reaction should be stopped when ~50% of the worms are lysed. Lysis time is critical. Under
                           no conditions should the lysis reaction exceed 5 min.
                        </p>
                     </li>
                     <li>
                        <p>Stop the lysis reaction by filling the tube with egg buffer (118 mM NaCl, 48 mM KCl, 2 mM CaCl<sub>2</sub>, 2 mM MgCl<sub>2</sub>, 25 mM Hepes, pH 7.3, 340 mOsm) and immediately pellet the eggs and lysed worms by centrifugation at 1200 rpm for 3 min.
                        </p>
                     </li>
                     <li>
                        <p>Remove the supernatant using a sterile plastic transfer pipette and wash 3X with egg buffer. Make sure the pellet is completely
                           re-suspended in the egg buffer during each wash.
                        </p>
                     </li>
                     <li>
                        <p>After the last centrifugation, carefully remove the buffer by sterile plastic transfer pipette.</p>
                     </li>
                     <li>
                        <p>Eggs are separated from debris by centrifugation in a 30% sucrose solution. Resuspend the pelleted eggs and lysed worms in
                           5 ml of sterile water and then add 5 ml of a sterile 60% sucrose stock. Mix this solution well.
                        </p>
                     </li>
                     <li>
                        <p>Centrifuge the suspension at 1200 rpm (~450g) for 5&#8211;6 min using a swinging bucket rotor in a tabletop centrifuge.</p>
                     </li>
                     <li>
                        <p>Eggs should float in the 30% sucrose solution and will collect at the solution meniscus. Some eggs may appear in the region
                           just below the meniscus. Using a sterile plastic transfer pipet (eggs will stick to glass pipets), transfer the eggs at the
                           meniscus into a sterile 15 ml conical tube. Eggs that stick to the tube can be recovered by gently washing down the sides
                           with a small volume of the sucrose solution and then removing them with a plastic transfer pipette.
                        </p>
                     </li>
                     <li>
                        <p>Collect no more than 3&#8211;4 ml of the egg/sucrose suspension. Fill the tube with sterile water and re-pellet to remove the sucrose.</p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Preparation of dissociated embryo cells</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>After egg isolation, all subsequent steps should be carried out in a laminar flow hood under sterile conditions.</p>
                     </li>
                     <li>
                        <p>Remove embryo eggshells by incubating approximately 50&#8211;200 &#956;l of pelleted eggs in 500 &#956;l of chitinase (Sigma Chemical Co.,
                           St. Louis, MO) solution (1U/ml in sterile egg buffer) in a sterile Eppendorf tube.
                        </p>
                     </li>
                     <li>
                        <p>Mix the egg suspension in the Eppendorf tube by rocking gently at room temperature for 20&#8211;80 minutes. Note that each lot of
                           chitinase tends to vary in potency. Consequently, incubation times will have to be assessed for each lot. The progress of
                           eggshell digestion should be monitored by viewing the egg suspension under a microscope.
                        </p>
                     </li>
                     <li>
                        <p>Isolated cells are washed and cultured in L-15 cell culture medium (Life Technologies, Grand Island, NY) containing 10% heat
                           inactivated fetal bovine serum (HYCLONE, Logan, UT), 50 U/ml penicillin and 50 &#956;g/ml streptomycin. The osmolality of the culture
                           medium should be adjusted to 340 mOsm with sucrose and filter sterilized. Osmolality can be measured with a vapor pressure
                           or freezing point osmometer. If these instruments are not available, the osmolality of the L-15 cell culture medium is listed
                           on the bottle label. As a first approximation, 1 mM sucrose <span class="symbol">&#8776;</span> 1 mOsm. Therefore, osmolality can be increased to ~340 mOsm by adding an equimolar amount of sucrose.
                        </p>
                     </li>
                     <li>
                        <p>When approximately 80% of the eggshells have been lysed, add 800 &#956;l of L-15 cell culture medium to the Eppendorf tube. Pellet
                           the eggs by centrifugation at ~3500 rpm (~900g) for 3 min at 4&deg;C in a microcentrifuge.
                        </p>
                     </li>
                     <li>
                        <p>Carefully remove the supernatant and add 800 &#956;l of fresh L-15 cell culture medium.</p>
                     </li>
                     <li>
                        <p>Gently dissociate the cells by repeatedly pipetting the cell suspension using a 1 ml Eppendorf pipettor. Monitor the degree
                           of dissociation by periodically placing a drop of the suspension on a microscope slide and viewing at 20X. Continue the dissociation
                           until you have a large number of single cells present. The preparation will also contain some undissociated embryos, clumps
                           of cells and possibly hatched larvae.
                        </p>
                     </li>
                     <li>
                        <p>Pellet the dissociated cell suspension by centrifugation at ~3500 rpm (~900g) and 4&deg;C for 3 min. Remove the supernatant, which
                           may be somewhat cloudy, and resuspend the pellet in 500 &#956;l of L-15 cell culture medium.
                        </p>
                     </li>
                     <li>
                        <p>The cell suspension is filtered to remove hatched larvae, cell clumps and very large cells. Filtration is carried out using
                           a 5.0 micron Durapore filter (Millipore Corporation, Bedford MA) and a 3 ml sterile syringe as follows:
                        </p>
                        <div class="orderedlist">
                           <ol type="a">
                              <li>
                                 <p>A sterile pipette tip for a 10&#8211;100 &#956;l Eppendorf pipettor is cut off just below the top &#8220;nut&#8221;. The cut tip is inserted onto
                                    a sterile syringe. Alternatively, you can use a sterile 18 gauge needle.
                                 </p>
                              </li>
                              <li>
                                 <p>1 ml of L-15 medium is drawn up into the syringe.</p>
                              </li>
                              <li>
                                 <p>The cell suspension is then drawn up slowly into the pipette tip and into the bottom of the syringe. Care is taken so that
                                    the cell suspension does not mix with the L-15 medium in the syringe.
                                 </p>
                              </li>
                              <li>
                                 <p>The pipette tip is removed and the 5.0 micron Durapore filter is placed securely onto the syringe.</p>
                              </li>
                              <li>
                                 <p>The cell suspension is forced through the filter with &#8220;medium&#8221; pressure into a sterile Eppendorf tube.</p>
                              </li>
                              <li>
                                 <p>To maximize the yield of single cells, the filter is rinsed with L-15 medium as follows. Remove the filter unit from the syringe
                                    and attach a new, sterile pipet tip. Draw an additional 1.0&#8211;1.5 ml of L-15 medium into the syringe. Remove the tip from the
                                    syringe and replace with the filter.
                                 </p>
                              </li>
                              <li>
                                 <p>Gently force the L-15 medium through the filter into a second sterile Eppendorf tube.</p>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p>Pellet cells by centrifugation at ~3500 rpm (~900g) and 4&deg;C for 3 min.</p>
                     </li>
                     <li>
                        <p>Remove the supernatants and resuspend the cells in 25&#8211;200 &#956;l (the actual volume depends on the size of the cell pellet) of
                           L-15 cell culture medium. After resuspension, pool the two samples.
                        </p>
                     </li>
                     <li>
                        <p>Prepare appropriate dilutions of the cell suspension in L-15 medium and determine cell density using a hemacytometer. Useful
                           counting dilutions range from 1:10 to 1:200. For our experimental purposes, we count only &#8220;large&#8221; and &#8220;medium&#8221; cells and ignore
                           any very small cells.
                        </p>
                     </li>
                     <li>
                        <p>Plating density depends on the nature of the experiments that are going to be performed. For patch clamp and optical experiments,
                           we find that a seeding density of ~230,000 cells/cm<sup>2</sup> is optimal.
                        </p>
                     </li>
                     <li>
                        <p>Cells must adhere tightly to the growth substrate in order for differentiation to occur. Cells are grown on glass coverslips
                           or plastic dishes with glass coverslip bottoms. The coverslips are coated with peanut lectin (Sigma) to promote cell adhesion.
                        </p>
                     </li>
                     <li>
                        <p>Prepare a peanut lectin solution in sterile water at a concentration of 0.5 mg/ml. For patch clamp studies, cells are grown
                           on 12 mm diameter acid-washed glass coverslips. Pipette a small volume of the peanut lectin solution onto the coverslip so
                           that it covers about 80% of the surface. Incubate the coverslips with the lectin solution for 10&#8211;20 min and then remove <span class="underline">completely</span>. It is important to completely remove the lectin solution. Excess lectin on the coverslips causes cell clumping.
                        </p>
                     </li>
                     <li>
                        <p>Sterilize the coverslips under UV light for several hours. The coverslips can be stored for weeks under sterile conditions.
                           The lectin can also be sterilized by gamma irradiation prior to placing it in solution.
                        </p>
                     </li>
                     <li>
                        <p>Plate the cells onto the coverslips in a minimal volume of L-15 medium. Allow the cells to settle and attach for 2 h and then
                           add additional L-15 medium.
                        </p>
                     </li>
                     <li>
                        <p>Culture vessels are kept in small sealed Tupperware containers lined with wet paper towels. Prevention of medium evaporation
                           is critical. The Tupperware containers are kept in a sterile, humidified incubator at room temperature and ambient air.
                        </p>
                     </li>
                     <li>
                        <p>In our hands, morphological differentiation of cells is largely complete within 24 h. Cell survival is excellent for 2&#8211;3 weeks.
                           However, these are primary cultures and they may dedifferentiate with time. We typically do not use cultures for physiology
                           experiments that are older than 5&#8211;8 days.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sidebar">
               <p class="title"><b>Protocol 32: MAPCeL (Micro Array Profiling C. elegans Cells): Embryonic cell culture and FACS for <span class="emphasis"><em>in vitro</em></span> and gene array studies (Rebecca M. Fox, Susan J. Barlow, Stephen E. Von Stetina and David M. Miller III)</b></p>
               <p>These methods can be used to generate transcriptional profiles of specific cell populations (<a href="#bib12">Fox et al., 2005</a>).
               </p>
               <div class="orderedlist">
                  <ol type="A">
                     <li>
                        <p><span class="bold"><strong>Embryo isolation</strong></span></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Synchronize worms by allowing them to exhaust the food supply and starve Remove a small square of agar to transfer dauer larvae
                                    to 150 mm 8P plates inoculated with Na22. Grow for one generation (~3 days) until plates are crowded with gravid adults. Average
                                    yield is ~10<sup>7</sup> embryonic cells per 150 mm plate. The procedure outlined below is appropriate for a cell preparation from 5&#8211;10 plates of
                                    worms.
                                 </p>
                              </li>
                              <li>
                                 <p>Wash the worms off of each plate with sterile water into a separate 15 ml conical tube.</p>
                              </li>
                              <li>
                                 <p>Centrifuge for 2.5 minutes at 2500 rpm (250xg).</p>
                              </li>
                              <li>
                                 <p>Treat with 5 ml of Bleach Solution for 5 minutes.</p>
                              </li>
                              <li>
                                 <p>Add 10 ml of egg buffer to stop the reaction and centrifuge for 2.5 minutes at 2500 rpm.</p>
                              </li>
                              <li>
                                 <p>Wash 3 times in quick succession with egg buffer, or until you can no longer smell the bleach.</p>
                              </li>
                              <li>
                                 <p>To separate embryos from carcasses and debris, resuspend pellet in 10 ml of 30% sucrose solution (5 ml 60% sucrose plus 5
                                    ml water) and centrifuge for 2.5 minutes at 2500 rpm.
                                 </p>
                              </li>
                              <li>
                                 <p>Remove embryos (floating on top) with a transfer pipette along with <span class="bold"><strong>no more</strong></span> than 5 ml of sucrose solution.
                                 </p>
                              </li>
                              <li>
                                 <p>Dilute to 15 ml with Milli-Q water and centrifuge at 2500 for 2.5 minutes to pellet the embryos.</p>
                              </li>
                              <li>
                                 <p>Remove supernatant and proceed to laminar flow hood for additional steps.</p>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Dissociating embryos to release early embryonic cells.</strong></span></p>
                        <p>Early embryos (before morphogenesis, &lt; 400 min) are largely dispersed to single cell suspensions by these methods whereas
                           older embryos are not. In fact, late stage embryos can be seen crawling around in these preparations prior to their removal
                           in the filtration step (see below).
                        </p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Resuspend embryos in 0.5 ml of sterile chitinase (0.5 U/ml).</p>
                              </li>
                              <li>
                                 <p>Incubate at RT for ~45 minutes with gentle rocking on nutator. Examine under a dissecting scope after 30 minutes to check
                                    for dissociation of embryos. Some batches of chitinase are more potent than others.
                                 </p>
                              </li>
                              <li>
                                 <p>To stop the reaction, top off the tube (~1 ml) with L-15-10 media (see below) and centrifuge at 12,000 rpm (1300xg) for 2
                                    minutes in microfuge.
                                 </p>
                              </li>
                              <li>
                                 <p>Aspirate supernatant and resuspend pellet in 1 ml of L-15-10 media with P1000 pipetman.</p>
                              </li>
                              <li>
                                 <p>Preload 3cc syringe with 0.5 ml media and draw up cell suspension into barrel. Gently push sample through a 5 &#956;m syringe filter
                                    into a 1.5 ml microfuge tube. To recover more embryonic cells, reload the syringe with ~1.5 ml media and push it through the
                                    filter again into a second microfuge tube. The filtration step is crucially important for removing cell clumps and undissociated
                                    embryos.
                                 </p>
                              </li>
                              <li>
                                 <p>Centrifuge for 3 minutes at 6000 rpm (350xg) in microfuge at RT.</p>
                              </li>
                              <li>
                                 <p>Remove supernatant. For preparations that will be sorted immediately to isolate GFP-labeled cells, resuspend the pellet in
                                    egg buffer (~10<sup>7</sup> cells/ml; See FACS protocol, Section D. below). For cells that will be maintained in culture, resuspend pellet in L15-10
                                    media (~10<sup>7</sup> cells/ml). (See cell culture protocol, Section C. below.)
                                 </p>
                              </li>
                              <li>
                                 <p>Count cells with a hemocytometer.</p>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Primary culture of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>embryonic cells for</strong></span> <span class="emphasis"><em><span class="bold"><strong>in vitro</strong></span></em></span> <span class="bold"><strong>experiments.</strong></span></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>For cells that will be characterized in culture (e.g. patch clamping, immunofluorescence staining, etc.), apply the suspension
                                    of embryonic blastomeres (~10<sup>7</sup> cells/ml in L15-10 media) to peanut agglutinin-coated glass surfaces (see below).
                                 </p>
                              </li>
                              <li>
                                 <p>Cells are cultured at 25&deg;C. Place the culture chamber inside a covered petri dish with a moist paper towel to retard evaporation.</p>
                              </li>
                              <li>
                                 <p>Cells are tightly adherent to peanut agglutinin-coated glass. Within 24 hr of plating, body muscle cells assume a bipolar
                                    spindle shaped morphology (<a href="#figure19">Figure 19 D,E</a>) and neurons extend elongated processes (<a href="#figure20">Figure 20</a>). Media is changed at 3&#8211;4 day intervals and cultures maintained for ~ 2 weeks.
                                 </p>
                                 <div class="mediaobject" align="center"><a name="figure19"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="585">
<tr>
  <td align="center"><p><a href="cellfig19.jpg"><img src="cellfig19_s.jpg" border="2" align="middle" alt="figure 19"></a>
  </td>
</tr>
                                    </table>
                                    <div class="caption">
                                       <p><b>Figure 19.&nbsp;</b>Isolation of embryonic muscle cells by FACS. A. Fluorescence intensity profile of wildtype (non-GFP) cells. Boxed areas exclude
                                          autofluorescent cells (arrow). B. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=myo-3" target="_blank">myo-3</a></em></span>::GFP cells are gated to exclude propidium iodide-stained (non-viable) cells. C. Light scattering gate(R2) for GFP-positive
                                          cells (circle). D. Mixed population of embryonic cells from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=myo-3" target="_blank">myo-3</a></em></span>::GFP embryo before sorting. E. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=myo-3" target="_blank">myo-3</a></em></span>::GFP cells after FACS enrichment (~90%). Scale bars are 5 mm. (Rebecca M. Fox, Susan J. Ruff, and David M. Miller III)
                                       </p>
                                    </div>
                                 </div>
                                 <div class="mediaobject" align="center"><a name="figure20"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="cellfig20.jpg"><img src="cellfig20_s.jpg" border="2" align="middle" alt=" figure 20"></a>
  </td>
</tr>
                                    </table>
                                    <div class="caption">
                                       <p><b>Figure 20.&nbsp;</b>Cultured <span class="emphasis"><em>C. elegans</em></span> neurons. A. Touch neuron expressing <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-4" target="_blank">mec-4</a></em></span>::GFP. B. ASE neuron expressing <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gcy-5" target="_blank">gcy-5</a>::</em></span>GFP. (Erica Keen and Shai Shaham).
                                       </p>
                                    </div>
                                 </div>
                                 <div class="mediaobject" align="center"><a name="figure21"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="cellfig21.jpg"><img src="cellfig21_s.jpg" border="2" align="middle" alt=" figure 21"></a>
  </td>
</tr>
                                    </table>
                                    <div class="caption">
                                       <p><b>Figure 21.&nbsp;</b>Example of a FACS sorting experiment (Rebecca M. Fox, Susan J. Ruff, and David M. Miller III).
                                       </p>
                                    </div>
                                 </div>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Primary culture of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>embryonic cells for isolation by FACS.</strong></span></p>
                        <p>GFP markers expressed in late stage embryos (&gt; morphogenesis; e.g. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a>::GFP</em></span>) are rarely observed in freshly prepared suspensions of blastomeres but do emerge with overnight culture. To recover these
                           cells for isolation by FACS, we plate them on poly-L-lysine from which they can be readily dislodged the next day with gentle
                           mechanical disruption.
                        </p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Plate cells in L15-10 media (~10<sup>7</sup> cells/ml) on poly-L-lysine coated slides and allow to differentiate overnight. Single-well chamber coverglasses work well
                                    for this application.
                                 </p>
                              </li>
                              <li>
                                 <p>Remove media and replace with 1 ml Egg Buffer.</p>
                              </li>
                              <li>
                                 <p>Wash cells from slide using P1000 pipetteman. Cells are loosely adherent and are easily dislodged.</p>
                              </li>
                              <li>
                                 <p>Pool cell suspensions (~3 ml) and pass through Durapore syringe filter (5 &#956;) into 15 ml conical tube.</p>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Prepare cells for FACS isolation.</strong></span></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Cells from freshly dissociated embryos or from 24 hr primary cultures are dispersed in egg buffer as described above.</p>
                              </li>
                              <li>
                                 <p>You will need four separate preparations of cells for each sorting experiment (see below; <a href="#figure20">Figure 20</a>).
                                 </p>
                                 <div class="orderedlist">
                                    <ol type="a">
                                       <li>
                                          <p><span class="bold"><strong>N2 only</strong></span>&#8212;non-GFP cells to identify autofluorescent cells
                                          </p>
                                       </li>
                                       <li>
                                          <p><span class="bold"><strong>N2 + PI</strong></span>&#8212;non-GFP cells plus Propidium iodide to set gate for PI-stained cells
                                          </p>
                                       </li>
                                       <li>
                                          <p><span class="bold"><strong>GFP only</strong></span>&#8212;Cells from GFP transgenic embryos to set GFP gate
                                          </p>
                                       </li>
                                       <li>
                                          <p><span class="bold"><strong>Experimental</strong></span>&#8212;Cells from GFP transgenic embryos + PI. Sort to isolate GFP cells
                                          </p>
                                       </li>
                                    </ol>
                                 </div>
                              </li>
                              <li>
                                 <p><span class="bold"><strong>N2 only</strong></span>. Prepare cells (~10<sup>6</sup>) from wildtype (N2) embryos.
                                 </p>
                              </li>
                              <li>
                                 <p><span class="bold"><strong>N2 + PI</strong></span>. Add propidium iodide (PI) to a separate tube of N2 cells (~10<sup>6</sup>).
                                 </p>
                              </li>
                              <li>
                                 <p><span class="bold"><strong>GFP only</strong></span>. Set aside a small aliquot (e.g., ~10<sup>6</sup> cells in 250 ul) of cells prepared from the GFP transgenic line.
                                 </p>
                              </li>
                              <li>
                                 <p><span class="bold"><strong>Experimental</strong></span>. Transfer the remainder of cells from the GFP transgenic line (2&#8211;2.5 ml) to another FACS tube and label, &#8220;Experimental.&#8221;
                                    The cell suspension should be clear. If cloudy, add egg buffer to achieve the ideal concentration of ~5&#8211;10 million cells/ml.
                                 </p>
                              </li>
                              <li>
                                 <p>Add propidium iodide (PI) to the &#8220;Experimental&#8221; tube immediately before sorting to a final concentration of 5 &#956;g/ml. (e.g.
                                    1 ml of 15 &#956;g/ml PI stock added to 2 ml of cell suspension.) PI stains dead or damaged cells instantly but will also label
                                    viable cells after prolonged exposure (2&#8211;3 hr).
                                 </p>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Preparing the FACS machine for sorting</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>cells.</strong></span></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>It is critically important to flush the FACS machine with egg buffer prior to sorting as viability is reduced by exposure
                                    to the lower osmolarity of PBS solutions typically used for sorting mammalian cells.
                                 </p>
                              </li>
                              <li>
                                 <p>Use 2 &#956; fluorescent beads to calibrate light scattering gates to match the relatively small size of <span class="emphasis"><em>C. elegans</em></span> embryonic cells.
                                 </p>
                              </li>
                              <li>
                                 <p>Use the <span class="bold"><strong>N2 only</strong></span> sample to generate a fluorescence intensity scatter plot for the PI and GFP channels. Because autofluorescent gut cells are
                                    detected in both channels they are largely restricted to the diagonal axis of this scatter plot (<a href="#figure19">Figure 19A</a>).
                                 </p>
                              </li>
                              <li>
                                 <p>Next, <span class="bold"><strong>N2 + PI</strong></span> cells are used to establish a sorting gate for dead or damaged cells (<a href="#figure19">Figure 19B</a>).
                                 </p>
                              </li>
                              <li>
                                 <p>Flow cytometry of the <span class="bold"><strong>GFP only</strong></span> sample is used to discriminate between autofluorescent and GFP-labeled cells (<a href="#figure19">Figure 19B</a>). We typically use all of these controls prior to every FACS run to re-establish the sorting gate for viable, GFP cells.
                                 </p>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Isolating GFP cells by FACS</strong></span></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>The <span class="bold"><strong>Experimental</strong></span> sample containing GFP + PI-stained cells is now sorted and viable GFP cells collected (<a href="#figure19">Figure 19B</a>).
                                 </p>
                              </li>
                              <li>
                                 <p>We have also found it helpful to simultaneously gate GFP cells by light scattering parameters to remove debris and cell clumps
                                    (<a href="#figure19">Figure 19C</a>). This gate is empirically defined (see below)
                                 </p>
                              </li>
                              <li>
                                 <p>Cells are collected in 15 ml conical tubes preloaded with 1 ml of L-15-10 media.</p>
                              </li>
                              <li>
                                 <p>For microarray experiments, we collect at least 100,000 GFP cells for each hybridization. For less abundant cell types (e.g.
                                    <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a></em></span>::GFP) it is necessary to pool cells from multiple runs to reach this figure. In other cases (e.g. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=myo-3" target="_blank">myo-3</a>::</em></span>GFP) more than enough cells are generated in a typical 1&#8211;2 hr sorting run.
                                 </p>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p>Adjusting FACS sorting gates</p>
                        <p>Sorting gates are empirically defined for each new GFP line by replating initial batches of sorted cells (on peanut lectin)
                           and then counting GFP vs non-fluorescent cells on the following day. This step is especially important for defining light
                           scattering parameters. Using this approach, we have achieved ~90% enrichment of GFP cells for a wide range of neuron and muscle-specific
                           markers (R. Fox, S. Von Stetina, S. Ruff, D. M. Miller, manuscripts in preparation).
                        </p>
                        <p>All steps are performed in a laminar flow hood.</p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Transfer sorted cells (~3ml volume) into 1.5ml Eppendorf tubes.</p>
                              </li>
                              <li>
                                 <p>Centrifuge at 6000 rpm (350xg) for 5 min in a microfuge.</p>
                              </li>
                              <li>
                                 <p>Remove most of the supernatant. It is important to be VERY careful because you cannot see a pellet. We generally leave ~100ul
                                    of solution in the tube. This can then be transferred to a peanut lectin-coated slide for visualization. MatTek round dishes
                                    work well.
                                 </p>
                              </li>
                              <li>
                                 <p>Determine the fraction of GFP-positive cells 24hr after replating by direct inspection in inverted microscope.</p>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Isolating RNA for Microarray experiments.</strong></span></p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Transfer sorted cells (~3 ml volume) into 1.5 ml Eppendorf tubes.</p>
                              </li>
                              <li>
                                 <p>Centrifuge at 6000 rpm for 5 min in a microfuge.</p>
                              </li>
                              <li>
                                 <p>Isolate RNA with Stratagene micro RNA isolation kit (200344) &#8211; use volumes suggested for 10<sup>6</sup> cells.
                                 </p>
                              </li>
                              <li>
                                 <p>Yields are typically ~ 1pg total RNA/cell.</p>
                              </li>
                              <li>
                                 <p>Lyophilize pellets and store at <span class="symbol">&#8722;</span>80&deg;C.
                                 </p>
                              </li>
                           </ol>
                        </div>
                     </li>
                  </ol>
               </div>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td>Equipment</td>
                  </tr>
                  <tr>
                     <td>Tissue Culture Hood</td>
                  </tr>
                  <tr>
                     <td>Clinical Centrifuge</td>
                  </tr>
                  <tr>
                     <td>Microfuge</td>
                  </tr>
                  <tr>
                     <td>Inverted compound microscope with fluorescence and DIC optics.</td>
                  </tr>
                  <tr>
                     <td>Must have at least 40x and preferably 63x, 100x objectives. (e.g. Zeiss Axiovert)</td>
                  </tr>
                  <tr>
                     <td>Osmometer (Wescor)</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>FACS Machine:</p>
               <p>Sorting experiments were performed in a FACStar Plus flow cytometer (Becton Dickinson, San Jose, CA) equipped with a 488 nm
                  argon laser and FITC filter set. Emission filters were 530&plusmn;30nm for GFP fluorescence and 585&plusmn;22nm for PI fluorescence. Sorting
                  rates were 4000&#8211;5000 cells per second through a 70&#956;m nozzle.
               </p>
               <p><span class="bold"><strong>REAGENTS AND MATERIALS</strong></span></p>
               <p><span class="bold"><strong>Peanut lectin coated slides</strong></span> &#8211; We routinely use <span class="symbol">&#947;</span> irradiation (overnight) to sterilize peanut agglutinin powder. Dissolve in sterile dH<sub>2</sub>0 at 0.5 mg/ml and use to coat entire surface of slide (e.g., for 1-well chamber coverglass add 2ml). Incubate for ~20 min
                  in a tissue culture hood. It is important to remove excess liquid prior to adding cells to avoid clumping. As an alternative
                  to <span class="symbol">&#947;</span> irradiation, slides may also be sterilized by exposure to UV light for 30&#8211;60 min after application of the peanut agglutinin
                  solution.
               </p>
               <p><span class="bold"><strong>Poly-L-lysine</strong></span> &#8211;poly-L-lysine solution (0.01% in sterile H<sub>2</sub>0) is sterilized by filtration (0.2 micron). Apply to coat the entire surface of slide. Incubate for ~30 minutes with chamber
                  covered. Wash slide thoroughly with sterile water; residual poly-L-lysine is toxic to cells. Allow to dry uncovered under
                  tissue culture hood.
               </p>
               <div class="table"><a name="d0e4502"></a><p class="title"><span class="bold">Preparation 8P plates:</span></p>
                  <table summary="Preparation 8P plates:" border="1">
                     <colgroup>
                        <col align="left">
                        <col align="left">
                     </colgroup>
                     <tbody>
                        <tr>
                           <td align="left">For 1L</td>
                           <td align="left">&nbsp;</td>
                        </tr>
                        <tr>
                           <td align="left">NaCl</td>
                           <td align="left">3g</td>
                        </tr>
                        <tr>
                           <td align="left">Agar</td>
                           <td align="left">25g</td>
                        </tr>
                        <tr>
                           <td align="left">Bactopeptone</td>
                           <td align="left">20g</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
               <div class="table"><a name="d0e4529"></a><p class="title">After Autoclaving add:</p>
                  <table summary="After Autoclaving add:" border="1">
                     <colgroup>
                        <col align="left">
                        <col align="left">
                     </colgroup>
                     <tbody>
                        <tr>
                           <td align="left">Cholesterol</td>
                           <td align="left">1ml</td>
                        </tr>
                        <tr>
                           <td align="left">1M CaCl<sub>2</sub></td>
                           <td align="left">1ml</td>
                        </tr>
                        <tr>
                           <td align="left">1M MgSO<sub>4</sub></td>
                           <td align="left">1ml</td>
                        </tr>
                        <tr>
                           <td align="left">KPO<sub>4</sub> pH6.0 Buffer
                           </td>
                           <td align="left">25ml</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td>Add 60 ml agar/150 mm plastic petri plate</td>
                  </tr>
                  <tr>
                     <td>Seed with Na22 bacteria</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td>Bleach Solution</td>
                  </tr>
                  <tr>
                     <td>For 10 ml:</td>
                  </tr>
                  <tr>
                     <td>7.5 ml Milli-Q water</td>
                  </tr>
                  <tr>
                     <td>2 ml Clorox</td>
                  </tr>
                  <tr>
                     <td>0.5 ml 10N NaOH</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <div class="table"><a name="d0e4585"></a><p class="title"></p>
                  <table summary="" border="1">
                     <colgroup>
                        <col align="left">
                        <col align="center">
                        <col align="center">
                     </colgroup>
                     <tbody>
                        <tr>
                           <td align="left"><span class="emphasis"><em><span class="bold"><strong>Egg Buffer</strong></span></em></span></td>
                           <td align="center">500ml</td>
                           <td align="center">1000ml</td>
                        </tr>
                        <tr>
                           <td align="left">NaCl (118 mM)</td>
                           <td align="center">3.448g</td>
                           <td align="center">6.896g</td>
                        </tr>
                        <tr>
                           <td align="left">KCl (48 mM)</td>
                           <td align="center">1.789g</td>
                           <td align="center">3.578g</td>
                        </tr>
                        <tr>
                           <td align="left">CaCl<sub>2</sub>&#8226; 2H<sub>2</sub>O (2 mM)
                           </td>
                           <td align="center">0.147g</td>
                           <td align="center">0.294g</td>
                        </tr>
                        <tr>
                           <td align="left">MgCl<sub>2</sub>&#8226; 6H<sub>2</sub>O (2 mM)
                           </td>
                           <td align="center">0.203g</td>
                           <td align="center">0.406g</td>
                        </tr>
                        <tr>
                           <td align="left">HEPES (25 mM)</td>
                           <td align="center">2.979g</td>
                           <td align="center">5.958g</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td>It is important that these weights are exact because of the osmotic sensitivity of <span class="emphasis"><em>C. elegans</em></span> embryonic cells.
                     </td>
                  </tr>
                  <tr>
                     <td>Adjust to pH 7.30 with 1N NaOH (no more than 10 ml).</td>
                  </tr>
                  <tr>
                     <td>The optimum osmolarity is 335&#8211;345. Check in osmometer.</td>
                  </tr>
                  <tr>
                     <td>Sterilize with 0.2 &#956;m bottle top filters.</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td><span class="emphasis"><em><span class="bold"><strong>Adjust osmolarity of L-15 media</strong></span></em></span></td>
                  </tr>
                  <tr>
                     <td>Sucrose is added to increase the osmolarity of L-15 media to 340 mOsm as below:</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td>Read L-15 media in osmometer. (~315 mOsm)</td>
                  </tr>
                  <tr>
                     <td>Calculate final concentration of sucrose needed for adjusting L-15 to 340 mOsm:</td>
                  </tr>
                  <tr>
                     <td>340 mOsm &#8211; 315 mOsm = 25 mM Sucrose</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td>Calculate sucrose (solid) for 500 ml L-15 media:</td>
                  </tr>
                  <tr>
                     <td>(MW Sucrose = 342.3 g)</td>
                  </tr>
                  <tr>
                     <td>0.5 L &times; 0.025 mole/l &times; 342.3 g/mole = 4.28 g sucrose</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>Confirm final osmolarity of 340&#8211;345 mOsm in osmometer.</p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td>L-15-10 Media</td>
                  </tr>
                  <tr>
                     <td>500 ml L-15 media adjusted to 340&#8211;345 mOsm (see above)</td>
                  </tr>
                  <tr>
                     <td>50 ml Heat inactivated FBS (10%)</td>
                  </tr>
                  <tr>
                     <td>(Heat inactivate FBS at at 56&deg;C for 30 min)</td>
                  </tr>
                  <tr>
                     <td>2.5 ml of Pen/Strep (1:200)</td>
                  </tr>
                  <tr>
                     <td>Sterilize with Nalgene 0.2 &#956;m filter.</td>
                  </tr>
                  <tr>
                     <td>Store at 4 C.</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td><span class="bold"><strong>Propidium iodide (3x)</strong></span></td>
                  </tr>
                  <tr>
                     <td>15 ug/ml in egg buffer. Good for 1 week at RT.</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td>Catalog numbers:</td>
                  </tr>
                  <tr>
                     <td><span class="bold"><strong>L-15</strong></span> &#8211; Gibco 21083-027
                     </td>
                  </tr>
                  <tr>
                     <td><span class="bold"><strong>Chitinase</strong></span> &#8211; Sigma C-1525
                     </td>
                  </tr>
                  <tr>
                     <td>Peanut Agglutinin &#8211; Sigma L-0881</td>
                  </tr>
                  <tr>
                     <td><span class="bold"><strong>Poly-L-Lysine</strong></span> &#8211; Sigma P4832
                     </td>
                  </tr>
                  <tr>
                     <td><span class="bold"><strong>Fetal Bovine Serum &#8211;</strong></span>VWR 16777-014; HyClone SH30071.03
                     </td>
                  </tr>
                  <tr>
                     <td><span class="bold"><strong>Pen/Strep-</strong></span> Gibco 15140-122
                     </td>
                  </tr>
                  <tr>
                     <td>Propidium Iodide-Sigma P-4170</td>
                  </tr>
                  <tr>
                     <td><span class="bold"><strong>Durapore filter</strong></span> (5 &#956;m) &#8211; Millipore, SLSV R25 LS
                     </td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td><span class="emphasis"><em><span class="bold"><strong>Plates etc:</strong></span></em></span></td>
                  </tr>
                  <tr>
                     <td>MatTek 35 mm Glass Bottom microwell dishes - uncoated</td>
                  </tr>
                  <tr>
                     <td>P35G-0-14-C</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td>LabTek Chambered Coverglass</td>
                  </tr>
                  <tr>
                     <td>4 well - 155383</td>
                  </tr>
                  <tr>
                     <td>2 well - 155380</td>
                  </tr>
                  <tr>
                     <td>1 well - 155361</td>
                  </tr>
               </table>
               <p>
                  
               </p>
               <p>LabTekII Chamber slide with Cover (4 well) - 154526</p>
               <p>
                  
               </p>
               <table class="simplelist" border="0" summary="Simple list">
                  <tr>
                     <td><span class="bold"><strong>Tubes</strong></span></td>
                  </tr>
                  <tr>
                     <td>1.5 ml microfuge tubes</td>
                  </tr>
                  <tr>
                     <td>5ml Polystyrene Round-Bottom Tube (FACS tube) Falcon 352054 (Fisher)</td>
                  </tr>
               </table>
               <p>
                  
               </p>
            </div>
            <div class="sidebar">
               <p class="title"><b>Protocol 33: Embryonic cell culture (Yun Zhang and Marty Chalfie)</b></p>
               <p><span class="underline">Growing synchronized gravid adults</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Grow worms on fifty 6 cm diameter NGM plates at 20&deg;C for 3 to 4 days until the bacterial lawns are consumed.</p>
                     </li>
                     <li>
                        <p>Transfer agar from the plates to twenty 15 cm NGM agarose plates (these differ from the agar plates in having 10 g/L agarose
                           instead of agar and 12 g/L peptone) and grow for 2 days until the plates contain a large number of eggs.
                        </p>
                     </li>
                     <li>
                        <p>Wash larvae and adults from the plates with M9 buffer, leaving the essentially synchronized eggs. Grow animals with concentrated
                           OP50 <span class="emphasis"><em>E. coli</em></span> for 3 to 4 days to get gravid adults. 1 ml/plate of concentrated OP50 is used for the first two days and 2&#8211;3 ml/plate is
                           used for each succeeding day.
                        </p>
                     </li>
                     <li>
                        <p>Harvest animals when the plates have large numbers of gravid adults, but still have some bacteria (starved worms and dauers
                           are more resistant to the hypochlorite solution in the next step, and will, thus, result in a lower yield of embryos or contamination
                           of the embryo preparation). Worms are collected and washed 3X (or until most bacteria are removed) with M9 buffer; pellet
                           worms in conventional tabletop centrifuge at 2,000 rpm for 2 minutes.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Collection and staging of embryos</span></p>
               <p>These and all subsequent steps should be done under sterile conditions.</p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Collect embryos by dissolving adults in 2&#8211;4 volumes of freshly-prepared hypochlorite solution. Gently shake the solution and
                           at the same time closely observe one drop of solution under a dissection microscope to determine when all the worm bodies
                           have dissolved and the embryos are exposed (usually 5 min). Extended exposure of embryos to the hypochlorite solution damages
                           the embryos and decreases cell yield.
                        </p>
                     </li>
                     <li>
                        <p>Wash the isolated embryos 3X with M9 buffer and pellet in a conventional tabletop centrifuge at 2,000 rpm for 1 min.</p>
                     </li>
                     <li>
                        <p>Resuspend the embryos in M9 buffer and maintain at 21&deg;C for 2&#8211;4 hrs to allow embryos to develop to the 300&#8211;400 cell stage.
                           Monitor the age of the embryos by observing aliquots using differential interference contrast (DIC). (The 300&#8211;400 cell stage
                           is the optimum time for culturing the embryonic touch receptor neurons; growth of other cells may require embryos at a different
                           stage.)
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Isolation of cells from the embryos</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Divide the staged embryos between two 50 ml conical tubes, wash embryos 1X with 25 ml egg buffer, and remove as much of the
                           buffer as possible.
                        </p>
                     </li>
                     <li>
                        <p>Resuspend the embryos in 2 volumes (usually 20 ml per tube) of chitinase/chymotrypsin solution and gently shake at room temperature.
                           Monitor the embryos by DIC to determine that the eggshells have been digested (usually 5 min - extended treatment damages
                           the cells).
                        </p>
                     </li>
                     <li>
                        <p>Wash the embryos with egg buffer and pellet them at 2,000 rpm for 1 min.</p>
                     </li>
                     <li>
                        <p>Add two volumes of a 0.25% trypsin, 1 mM EDTA (Gibco) solution to the pelleted embryos and incubate at room temperature for
                           5 min with gentle shaking. Further dissociate cells by pipeting the resuspended embryos gently up and down 10X with a Pasteur
                           pipet (drawn out to have a 0.3&#8211;0.5 mm diameter opening that has been fire polished). The suspension, viewed with DIC, should
                           contain mostly separated cells, very few cell clumps, and some debris from lysed cells. If the dissociation is incomplete,
                           repeat the 5 min trypsin/EDTA incubation and 10X pipeting.
                        </p>
                     </li>
                     <li>
                        <p>Stop the trypsin treatment by adding ten volumes of L-15CM medium to the cell suspension and pellet the separated cells at
                           2,000 rpm for 5 minutes.
                        </p>
                     </li>
                     <li>
                        <p>Wash cells 2X with L-15CM medium and resuspend cells in L-15SFM medium.</p>
                     </li>
                     <li>
                        <p>Place the cell suspension on ice for 10 min to allow big cell clumps to settle.</p>
                     </li>
                     <li>
                        <p>Collect the supernatant and pellet cells at 1,000 rpm for 10 min.</p>
                     </li>
                     <li>
                        <p>Wash the cells once with L-15SFM and resuspend in 2&#8211;3 volumes of L-15SFM.</p>
                     </li>
                     <li>
                        <p>Plate cells on TESPA treated microscopic slides. Place each slide in a 10 cm diameter culture dish that is placed in water
                           in a 15 cm diameter culture dish to prevent the slides from dehydrating. Incubate the cells at 21&deg;C for two hours to allow
                           them to attach to the slides.
                        </p>
                     </li>
                     <li>
                        <p>Gently add large amounts of L-15CM to cover the cells and let the cells grow in the dark at 21&deg;C for 8&#8211;12 hr. The twenty 15
                           cm plates of animals generate several billion cells, which we usually put on 30 microscopic slides.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Solutions</span></p>
               <p><span class="underline">Concentrated OP50:</span> Culture OP50 overnight at 37&deg;C in 12 L of Superbroth, pellet, and resuspend in 60 ml of M9 buffer (final volume approximately
                  120 ml). Store at 4&deg;C until needed.
               </p>
               <p><span class="underline">Hypochlorite solution:</span> 1 ml sodium hypochlorite (Sigma), 2.5 ml 1 M NaOH, 1.5 ml dH<sub>2</sub>O.
               </p>
               <p><span class="underline">Egg buffer:</span> 11.8 ml 1 M NaCl, 4.8 ml 1 M KCl, 0.34 ml 1 M CaCl<sub>2</sub>, 0.34 ml 1 M MgCl<sub>2</sub>, 2.0 ml 0.25 M HEPES (pH 7.4), 82.5 ml dH<sub>2</sub>O. Filter-sterilize and store at 4&deg;C.
               </p>
               <p><span class="underline">Chitinase/chymotrypsin:</span> 5 mg chitinase (Sigma C-6137), 5 mg alpha- chymotrypsin (ICN 152272), 1 ml egg buffer, 10 &#956;l penicillin-streptomycin [Cat.
                  No. 15140-148, 10,000 units of Penicillin (base) and 10,000 &#956;g of Streptomycin base/ml, Gibco]. Freeze in 10 ml aliquots at
                  <span class="symbol">&#8722;</span>20&deg;C.
               </p>
               <p><span class="underline">L-15CM:</span> 24 ml L-15 (Gibco), 6 ml inulin [ICN 102055; 7.5 mg/ml egg salts, autoclaved and stored at 4&deg;C], 0.4 ml base mix (D8), 200
                  mg PVP (polyvinylpyrrolidone, Sigma), 0.4 ml penicillin-streptomycin (Gibco). Filter-sterilize, add 8 ml fetal bovine serum
                  (Heat-inactivated, Sigma), and store at 4&deg;C until needed on the same day.
               </p>
               <p><span class="underline">L-15SFM:</span> 24 ml L-15, 6 ml inulin, 0.4 ml base mix, 10 ml egg buffer, 0.4 ml penicillin-streptomycin, 100 mg PVP. Filter-sterilize
                  and store at 4&deg;C.
               </p>
               <p><span class="underline">TESPA-coated slides:</span> Soak microscopic slides in 70% ethanol and 1% HCl for 1 to 2 hours. Rinse 3X with ddH<sub>2</sub>O. Air-dry and sterilize slides under a culture hood UV lamp for 1 hr (all subsequent steps should be performed under sterile
                  conditions). Soak slide in 4% TESPA (3-aminopropyltriethoxysilane, Sigma)/acetone solution for 5 min. Wash slides in acetone
                  twice. Air dry. Soak slides in 4% paraformaldhyde/PBS solution for 30 min at room temperature. Wash slides in sterile water
                  for four times. Air dry and store at room temperature. Treated slides have to be used within 3 days.
               </p>
               <p><span class="underline">Egg salts:</span> 11.8 ml 1 M NaCl, 4.8 ml 1 M KCl, 83.6 ml H<sub>2</sub>O.
               </p>
               <p><span class="underline">Base mix:</span> 100 ml H<sub>2</sub>O, 100 mg adenine, 10 mg ATP, 3 mg guanine, 3 mg hypoxanthine, 3 mg thymine, 3 mg xanthine, 3 mg uridine, 5 mg ribose, 5 mg
                  deoxyribose; autoclave and store at 4&deg;C.
               </p>
            </div>
            <div class="sidebar">
               <p class="title"><b>Protocol 34: Linear Amplification of mRNA from cultured cells (Yun Zhang and Marty Chalfie based on <a href="#bib39">Zhang et al., 2002</a>)</b></p>
               <p><span class="underline">Isolation of mRNA</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Pellet sorted cells in an Eppendorf centrifuge at 12,000 rpm for 8 min and lyse them using the RTL buffer from the RNeasy
                           Mini kit (Ambion).
                        </p>
                     </li>
                     <li>
                        <p>Vortex the lysate for 1 min and pass through a QIAshredder column (Ambion). The lysate can be stored at <span class="symbol">&#8722;</span>80&deg;C overnight. RNA quality decreases, however, with the extended storage. Isolate total RNA with the RNeasy Mini kit with
                           the DNase I set (Ambion).
                        </p>
                     </li>
                     <li>
                        <p>Purify mRNA with the mRNA oligotex minikit (Ambion) according to manufacturer's instructions.</p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">First round amplification</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p><span class="underline">Reverse transcription</span>. Add 1 &#956;l 100 ng/&#956;l T7-promoter linked oligo dT (TTGTAATACGACTCACTATAGGGAGG24T; HPLC purified) to 13.75 &#956;l (~100 pg) of isolated
                           mRNA. This shorter oligo helps prevent template-independent amplification. Heat to 65&deg;C for 5 min and chill on ice for 3 minutes
                           before adding:
                        </p>
                        <div class="table"><a name="d0e4962"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="left">
                                 <col align="left">
                              </colgroup>
                              <tbody>
                                 <tr>
                                    <td align="left">2 &#956;l</td>
                                    <td align="left">10 &times; SensiScript RT reaction buffer (Ambion)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">2 &#956;l</td>
                                    <td align="left">10 &times; d-NTP mix (with SensiScript RT)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">0.25 &#956;l</td>
                                    <td align="left">RNase Inhibitor (Stratagene)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">1 &#956;l</td>
                                    <td align="left">SensiScript RT (Ambion)</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                        <p>Mix thoroughly and incubate at 37&deg;C for 1 hour. Stop the reaction by putting it on ice.</p>
                     </li>
                     <li>
                        <p><span class="underline">Formation of double-stranded cDNA</span>. Mix the following thoroughly and incubate at 16&deg;C for 2 hour.
                        </p>
                        <div class="table"><a name="d0e4995"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="left">
                                 <col align="center">
                              </colgroup>
                              <tbody>
                                 <tr>
                                    <td align="left">20 &#956;l</td>
                                    <td align="center">Reverse transcription reaction mix</td>
                                 </tr>
                                 <tr>
                                    <td align="left">16 &#956;l</td>
                                    <td align="center">5 &times; second strand buffer (D1)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">3.2 &#956;l</td>
                                    <td align="center">10 &times; d-NTP mix (5mM each)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">8 &#956;l</td>
                                    <td align="center">100 mM DTT (Gibco)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">2.4 &#956;l</td>
                                    <td align="center">DNA polymerase I (NEB, 10 unit/&#956;l)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">0.5 &#956;l</td>
                                    <td align="center">DNA ligase (NEB, 10 unit/&#956;l)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">0.3 &#956;l</td>
                                    <td align="center">RNase H (Gibco, 3 unit/&#956;l)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">29.6 &#956;l</td>
                                    <td align="center">RNase- and DNase-free H<sub>2</sub>O (Ambion)
                                    </td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                        <p>Add 2 &#956;l T4 DNA polymerase (NEB, 3 unit/&#956;l) and continue the incubation at 16&deg;C for 30 minutes. Add 80 &#956;l phenol/chloroform
                           (25:24) mix, vortex, and spin at 14,000 rpm in an Eppendorf centrifuge at room temperature for 3 minutes. Transfer the supernatant
                           to a new tube. Repeat extraction once. Recover the double-stranded cDNA from reaction solution, using 10 ng DstNI digested
                           pBR322 as carrier, by GENECLEAN II (BIO 101) according to manufacturer's instruction. Elute cDNA into 29 &#956;l Tris (pH 8.0).
                        </p>
                     </li>
                     <li>
                        <p><span class="underline"><span class="emphasis"><em>in vitro</em></span> transcription.</span> In the following we use the <span class="emphasis"><em>in vitro</em></span> Transcription Kit for T7 RNA Polymerase from Ambion, except that we substitute the Epicenter T7 RNA polymerase (2,500 unit/&#956;l).
                           This polymerase is at a higher concentration and gives better amplification. Mix the following thoroughly
                        </p>
                        <div class="table"><a name="d0e5056"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="left">
                                 <col align="center">
                              </colgroup>
                              <tbody>
                                 <tr>
                                    <td align="left">29 &#956;l</td>
                                    <td align="center">double stranded cDNA</td>
                                 </tr>
                                 <tr>
                                    <td align="left">8 &#956;l</td>
                                    <td align="center">10 &times; reaction buffer</td>
                                 </tr>
                                 <tr>
                                    <td align="left">32 &#956;l</td>
                                    <td align="center">NTP mix (8 &#956;l for each)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">8 &#956;l</td>
                                    <td align="center">Enzyme mix</td>
                                 </tr>
                                 <tr>
                                    <td align="left">3 &#956;l</td>
                                    <td align="center">T7 RNA polymerase</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                        <p>8 &#956;l of the reaction can be mixed in another tube with 0.8 &#956;l <span class="symbol">&#945;</span>-P<sup>32</sup>-CTP. Incubate both tubes at 37&deg;C for 18 hours. Purify transcripts from cold reaction using the RNeasy Minikit. Precipitate
                           the hot reaction with 5% trichloroacetic acid and 20 mM sodium pyrophosphate to determine the amount of transcription<sup>2</sup>.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Second round amplification</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p><span class="underline">Reverse transcription</span><span class="bold"><strong>.</strong></span> Mix 10 &#956;l of the first round amplified RNA with random hexamer (Gibco, 18 ng primer/&#956;g RNA), adjust the volume to 11.7 &#956;l,
                           and heat to 70&deg;C for 10 min. Chill on ice for 2 min and add
                        </p>
                        <div class="table"><a name="d0e5109"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="left">
                                 <col align="center">
                              </colgroup>
                              <tbody>
                                 <tr>
                                    <td align="left">4 &#956;l</td>
                                    <td align="center">5X reaction buffer (Gibco)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">0.8 &#956;l</td>
                                    <td align="center">dNTP mix (25 mM for each)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">2 &#956;l</td>
                                    <td align="center">100 mM DTT (Gibco)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">0.5 &#956;l</td>
                                    <td align="center">Superase In (Ambion)</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                        <p>Incubate at 45&deg;C for 10 min and add 1 &#956;l SuperScript RT (Gibco). Incubate at 37&deg;C for 1 hr. Recover the nucleic acid from
                           solution by GENECLEAN II and elute with 12 &#956;l Tris (pH 8.0).
                        </p>
                     </li>
                     <li>
                        <p><span class="underline">Formation of double-stranded cDNA</span><span class="bold"><strong>.</strong></span> Add 1 &#956;l of 1 &#956;g/&#956;l T7-promoter linked oligo dT to 12 &#956;l of the single stranded cDNA and heat for 3 min at 95&#8211;100 &deg;C to degrade
                           RNA. Chill on ice and then add
                        </p>
                        <div class="table"><a name="d0e5144"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="left">
                                 <col align="center">
                              </colgroup>
                              <tbody>
                                 <tr>
                                    <td align="left">2 &#956;l</td>
                                    <td align="center">10 &times; KFI buffer (D2)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">2 &#956;l</td>
                                    <td align="center">d-NTP mix (2.5 mM each)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">1 &#956;l</td>
                                    <td align="center">100 mM DTT (Gibco)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">1 &#956;l</td>
                                    <td align="center">T4 DNA polymerase (NEB, 10 unit/&#956;l)</td>
                                 </tr>
                                 <tr>
                                    <td align="left">1 &#956;l</td>
                                    <td align="center">Klenow fragment (NEB, 50 unit/&#956;l)</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                        <p>Incubate at 14&deg;C for 12 hours and recover double-stranded cDNA by GENECLEAN II.</p>
                     </li>
                     <li>
                        <p><span class="underline"><span class="emphasis"><em>in vitro</em></span> transcription reaction</span>. Follow the procedure for the first round amplification except carry out the reaction for 10 hr. Recover the amplified RNA
                           using the RNeasy Minikit.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Solutions</span></p>
               <p><span class="underline">5X second stranded buffer:</span> 500 mM KCl, 50 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 25 mM MgCl<sub>2</sub>, 0.75 mM NAD<sup>+</sup> (nicotinamide adenine dinucleotide), 100 mM Tris&middot;Cl (pH 7.5), and 0.25 mg/ml BSA.
               </p>
               <p><span class="underline">10X KFI buffer:</span> 200 mM Tris (pH 7.5), 100 mM MgCl<sub>2</sub>, 50mM NaCl.
               </p>
            </div>
            <div class="sidebar">
               <p class="title"><b>Protocol 35: Modified Affymetrix small sample amplification (Steve Von Stetina, Rebecca Fox, and David Miller)</b></p>
               <p>Affymetrix hybridizations require 10&#8211;15 ug of labeled probe. Because RNA in this amount is rarely available from FACS isolated
                  cells, we amplify initial RNA preparations with two rounds of a T7 RNA polymerase-dependent amplification protocol. The following
                  bench protocol produces high-quality cRNA from 100 ng of total RNA. This protocol is a modified version of an amplification
                  strategy originally described by Affymetrix (GeneChip Eukaryotic Small Sample Target Labeling Technical Note, Part 701129
                  Rev 1).
               </p>
               <p><span class="bold"><strong>Significant changes include:</strong></span></p>
               <div class="itemizedlist">
                  <ul type="disc">
                     <li>
                        <p>6 hr for the RNA transcription step provides higher yields.</p>
                     </li>
                     <li>
                        <p>For cRNA cleanup, 40% EtOH and two passes through the column dramatically enhanced yields.</p>
                     </li>
                  </ul>
               </div>
               <p><span class="bold"><strong>Other important points:</strong></span></p>
               <div class="itemizedlist">
                  <ul type="disc">
                     <li>
                        <p>The protocol can be divided into two long days or four shorter days; in all, the protocol takes ~24 hours to complete.</p>
                     </li>
                     <li>
                        <p>We obtain 30&#8211;70 ug of labeled cRNA from 100 ng total RNA after two rounds of amplification.</p>
                     </li>
                     <li>
                        <p>The commercially available fragmentation buffer was more reliable than &#8220;home-brewed&#8221; solutions. The reasons for this difference
                           are unknown.
                        </p>
                     </li>
                     <li>
                        <p>It is important to assess the quality and size distribution of amplified cRNA and subsequent fragmented cRNA label by gel
                           electrophoresis (Agilent Bioanalyzer).
                        </p>
                     </li>
                  </ul>
               </div>
               <div class="table"><a name="d0e5244"></a><p class="title">MATERIALS</p>
                  <table summary="MATERIALS" border="1">
                     <colgroup>
                        <col align="center">
                        <col align="center">
                        <col align="center">
                     </colgroup>
                     <thead>
                        <tr>
                           <th align="center">Reagent Name</th>
                           <th align="center">Vendor</th>
                           <th align="center">Catalog #</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="center">
                              
                              <table class="simplelist" border="0" summary="Simple list">
                                 <tr>
                                    <td>T7-(dT)<sub>24</sub> primer, 2-step HPLC, MALDI-TOF
                                    </td>
                                 </tr>
                                 <tr>
                                    <td>5'-GGCCAGTGAATTGTAA- TACGACTCACT</td>
                                 </tr>
                                 <tr>
                                    <td>ATAGGGAGGCGG-(dT)<sub>24</sub> -3'
                                    </td>
                                 </tr>
                              </table>
                              
                           </td>
                           <td align="center">Operon (Qiagen)</td>
                           <td align="center">&nbsp;</td>
                        </tr>
                        <tr>
                           <td align="center">
                              
                              <table class="simplelist" border="0" summary="Simple list">
                                 <tr>
                                    <td>Superscript II, 200 U/ul 40,000 U</td>
                                 </tr>
                                 <tr>
                                    <td>5X first strand buffer, DTT included</td>
                                 </tr>
                              </table>
                              
                           </td>
                           <td align="center">Invitrogen</td>
                           <td align="center">18064-071</td>
                        </tr>
                        <tr>
                           <td align="center">5X second strand buffer, 500 ul</td>
                           <td align="center">Invitrogen</td>
                           <td align="center">10812-014</td>
                        </tr>
                        <tr>
                           <td align="center"><span class="emphasis"><em>E. coli</em></span> DNA Polymerase, 10 U/ul, 1000 U
                           </td>
                           <td align="center">Invitrogen</td>
                           <td align="center">18010-025</td>
                        </tr>
                        <tr>
                           <td align="center"><span class="emphasis"><em>E. coli</em></span> DNA Ligase, 10 U/ul, 100 U
                           </td>
                           <td align="center">Invitrogen</td>
                           <td align="center">18052-019</td>
                        </tr>
                        <tr>
                           <td align="center">RNase H, 2 U/ul, 120 U</td>
                           <td align="center">Invitrogen</td>
                           <td align="center">18021-071</td>
                        </tr>
                        <tr>
                           <td align="center">T4 DNA Polymerase, 5 U/ul, 250 U</td>
                           <td align="center">Invitrogen</td>
                           <td align="center">18005-025</td>
                        </tr>
                        <tr>
                           <td align="center">Random Primers</td>
                           <td align="center">Invitrogen</td>
                           <td align="center">48190-011</td>
                        </tr>
                        <tr>
                           <td align="center">dNTP mix, 100 mM each</td>
                           <td align="center">NEB</td>
                           <td align="center">N0446S</td>
                        </tr>
                        <tr>
                           <td align="center">rRNasin, 40 U/ul 10,000 U</td>
                           <td align="center">Promega</td>
                           <td align="center">N2515</td>
                        </tr>
                        <tr>
                           <td align="center">Glycogen, 5 mg/ml, 5 ml</td>
                           <td align="center">Ambion</td>
                           <td align="center">9510</td>
                        </tr>
                        <tr>
                           <td align="center">Ammonium Acetate, 7.5 M, 100 ml</td>
                           <td align="center">Sigma</td>
                           <td align="center">A-2706</td>
                        </tr>
                        <tr>
                           <td align="center">MEGAscript T7 kit</td>
                           <td align="center">Ambion</td>
                           <td align="center">1334</td>
                        </tr>
                        <tr>
                           <td align="center">Affymetrix GeneChip Sample Cleanup Module</td>
                           <td align="center">Affymetrix</td>
                           <td align="center">900371</td>
                        </tr>
                        <tr>
                           <td align="center"><span class="symbol">&#946;</span> - mercaptoethanol, 25 ml
                           </td>
                           <td align="center">Sigma</td>
                           <td align="center">M-3148</td>
                        </tr>
                        <tr>
                           <td align="center">Enzo BioArray HighYield RNA Transcript Labeling kit, 10 reactions</td>
                           <td align="center">Affymetrix</td>
                           <td align="center">900182</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
               <p><span class="bold"><strong>Solutions (USE DEPC-H<sub>2</sub>O FOR ALL SOLUTIONS):</strong></span></p>
               <p><span class="underline">DEPC-H<sub>2</sub>O:</span> Fill 4L Flask in RNase-free cabinet with 3.5L of Milli-Q water. In hood, add 3.5 ml DEPC (final concentration is 0.1%). Leave
                  stirring at RT o/n. Autoclave the following day.
               </p>
               <p><span class="underline">5M Ammonium Acetate:</span> Dilute 7.5M NH<sub>4</sub>Ac to 5M using DEPC-H<sub>2</sub>O.
               </p>
               <p><span class="underline">Random Primers:</span> Dilute 3 ug/ul stock to 1 ug/ul using DEPC-H<sub>2</sub>O; store in 5 ul aliquots at <span class="symbol">&#8722;</span>20&deg;C.
               </p>
               <p><span class="underline">10 mM dNTP mix:</span> Mix 100 ul of each dNTP with 600 ul DEPC-H<sub>2</sub>O. Store in 5 ul aliquots at <span class="symbol">&#8722;</span>20&deg;C.
               </p>
               <p><span class="bold"><strong>METHODS</strong></span></p>
               <p><span class="bold"><strong>First Amplification Cycle (can split into 2 days if desired)</strong></span></p>
               <p><span class="bold"><strong>Step 1. First cycle, first strand cDNA synthesis</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Combine100 ng total RNA and the following components in a 1.5 ml RNase free microfuge tube.</p>
                        <div class="table"><a name="d0e5453"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center">Volume/reaction</th>
                                    <th align="center">Concentration</th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>DEPC-H<sub>2</sub>O</strong></span></td>
                                    <td align="center">9 ul</td>
                                    <td align="center">&nbsp;</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>Total RNA</strong></span></td>
                                    <td align="center">1 ul</td>
                                    <td align="center">100 ng/ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>T7-(dT)<sub>24</sub> primer</strong></span></td>
                                    <td align="center">1 ul</td>
                                    <td align="center">100 ng/ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em>total volume</em></span></td>
                                    <td align="center">11 ul</td>
                                    <td align="center">&nbsp;</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Vortex tube then quick spin in small benchtop microfuge (picofuge).</p>
                     </li>
                     <li>
                        <p>Place tube in 70&deg;C heat block for 10 min.</p>
                     </li>
                     <li>
                        <p>Chill on ice for 2 min.</p>
                     </li>
                     <li>
                        <p>Add the following reagents to microfuge tube:</p>
                        <div class="table"><a name="d0e5515"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center">Volume/reaction</th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>(RNA/primer mixture)</strong></span></td>
                                    <td align="center">(11 ul)</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>5X first strand buffer</strong></span></td>
                                    <td align="center">4 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>DTT, 0.1M</strong></span></td>
                                    <td align="center">2 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>dNTP mix, 10 mM</strong></span></td>
                                    <td align="center">1 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>RNasin, 40 U/ul</strong></span></td>
                                    <td align="center">1 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em>total volume</em></span></td>
                                    <td align="center">19 ul</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Vortex, spin. Incubate in 42&deg;C water bath for 2 min.</p>
                        <p>Add 1 ul of Superscript II (200 U/ul) to the preheated reaction, vortex and quick spin.</p>
                        <p>Incubate at 42&deg;C for 1 hour.</p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Step 2. First Cycle, second strand cDNA synthesis</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Spin down first strand reaction, then cool on ice for 2 min.</p>
                     </li>
                     <li>
                        <p>Transfer solution to a 0.5 ml RNase-free microfuge tube.</p>
                     </li>
                     <li>
                        <p>Add following reagents to the first strand synthesis reaction:</p>
                        <div class="table"><a name="d0e5582"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center">Volume/reaction</th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>(First strand synthesis reaction)</strong></span></td>
                                    <td align="center">(20 ul)</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>DEPC-H<sub>2</sub>O</strong></span></td>
                                    <td align="center">91 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>5X second strand buffer</strong></span></td>
                                    <td align="center">30 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>dNTP mix</strong></span></td>
                                    <td align="center">3 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em><span class="bold"><strong>E.coli</strong></span></em></span> <span class="bold"><strong>DNA ligase</strong></span></td>
                                    <td align="center">1 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em><span class="bold"><strong>E. coli</strong></span></em></span> <span class="bold"><strong>DNA polymerase</strong></span></td>
                                    <td align="center">4 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>RNase H</strong></span></td>
                                    <td align="center">1 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em>total volume</em></span></td>
                                    <td align="center">150 ul</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Vortex, spin. Incubate at 16&deg;C in a PCR machine for 2 hr.</p>
                     </li>
                     <li>
                        <p>Add 2 ul of T4 DNA polymerase to the reaction and incubate for an additional 15 min at 16&deg;C. (this step polishes the ends
                           of the dsDNA)
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Step 3. First cycle, double-stranded cDNA clean-up by ethanol precipitation.</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Following second strand synthesis, transfer contents of 0.5 ml microfuge into a 1.5 ml RNase-free microfuge tube.</p>
                     </li>
                     <li>
                        <p>Add following to tube (all items stored in RNase-free freezer):</p>
                        <div class="table"><a name="d0e5668"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">Reagent</th>
                                    <th align="center">Volume/reaction</th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>5 mg/ml glycogen</strong></span></td>
                                    <td align="center">1 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>5 M NH<sub>4</sub>OAc</strong></span></td>
                                    <td align="center">92 ul (0.6 volume)</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>Ice cold absolute EtOH</strong></span></td>
                                    <td align="center">612 ul (2.5 volumes)</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Mix by inversion. Centrifuge for 20 min at 4&deg;C at 14000 rpm. **You should see a pellet!**</p>
                     </li>
                     <li>
                        <p>REMOVE ETHANOL BY ASPIRATION USING RNase-free ROUND GEL-LOADING TIPS (Marsh Biomedical Products, Inc #T-3000).</p>
                     </li>
                     <li>
                        <p>Wash with 70% cold ethanol (stored in freezer); spin for 14,000 rpm in microfuge for 5 min at 4&deg;C.</p>
                     </li>
                     <li>
                        <p>Remove ethanol as above. Speed-vac for 5 min to dry pellet.</p>
                     </li>
                  </ol>
               </div>
               <p>THIS IS A POTENTIAL STOPPING POINT. STORE PELLET AT <span class="symbol">&#8722;</span>20&deg;C IF NOT CONTINUING ON TO IVT REACTION.
               </p>
               <p><span class="bold"><strong>Step 4. First cycle,</strong></span> <span class="emphasis"><em><span class="bold"><strong>in vitro</strong></span></em></span> <span class="bold"><strong>transcription (IVT) for cRNA using Ambion MEGAscript T7 kit</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Resuspend cDNA pellet in 8 ul DEPC-H<sub>2</sub>O. Vortex and pipet up and down to resuspend pellet.
                        </p>
                     </li>
                     <li>
                        <p>Add the following (can make master mix):</p>
                        <div class="table"><a name="d0e5738"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center">Volume/reaction</th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>(cDNA)</strong></span></td>
                                    <td align="center">(8 ul)</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>dNTP mix</strong></span></td>
                                    <td align="center">8 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>10X reaction buffer</strong></span></td>
                                    <td align="center">2 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>Ambion enzyme mix</strong></span></td>
                                    <td align="center">2 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em>total volume</em></span></td>
                                    <td align="center">20 ul</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                        <p>Vortex, spin. Incubate in 37&deg;C water bath for 6 hours.</p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Step 5. First cycle, cRNA clean-up with Affy Cleanup Kit</strong></span></p>
               <p>****Add 10 ul <span class="symbol">&#946;</span>-ME to every 1 ml IVT cRNA Binding Buffer before starting clean-up****
               </p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Add 80 ul DEPC-H<sub>2</sub>O to IVT reaction (total volume = 100 ul)
                        </p>
                     </li>
                     <li>
                        <p>Add 350 ul IVT cRNA Binding Buffer; mix. Add 300 ul 100% EtOH. Mix by pipeting.</p>
                     </li>
                     <li>
                        <p>Add sample (750 ul) to a supplied mini-column. Pulse in microfuge for 15 sec.</p>
                     </li>
                     <li>
                        <p>Pipet flow-through from collection tube and pulse through mini-column again.</p>
                     </li>
                     <li>
                        <p>Discard collection tube. Transfer column to a new 2.0 ml collection tube.</p>
                     </li>
                     <li>
                        <p>Add 500 ul IVT cRNA Wash Buffer. Pulse in microfuge for 15 sec. Discard flow-through.</p>
                     </li>
                     <li>
                        <p>Add 500 ul 80% EtOH. Spin for 2 min in microfuge at full speed. Discard collection tube.</p>
                     </li>
                     <li>
                        <p>Transfer column to a kit-supplied 1.5 ml collection tube.</p>
                     </li>
                     <li>
                        <p>Add 32 ul DEPC-H<sub>2</sub>O to mini-column. Incubate at RT for 1 min. Spin for 1 min at full-speed.
                        </p>
                     </li>
                     <li>
                        <p>Determine concentration of cRNA using spectrophotometer.</p>
                        <div class="orderedlist">
                           <ol type="a">
                              <li>
                                 <p>Add 2 ul cRNA to 98 ul DEPC-H<sub>2</sub>O (1:50 dilution).
                                 </p>
                              </li>
                              <li>
                                 <p>Use lab spec (DNA/RNA calculator) to determine A260.</p>
                                 <div class="orderedlist">
                                    <ol type="1">
                                       <li>
                                          <p>Determine concentration using conversion factor of 40 ug/ml (for RNA).</p>
                                       </li>
                                       <li>
                                          <p>If the concentration is less than 10 ng/ul, speed-vac to reduce volume of cRNA from 30 ul to 10 ul (DO NOT DRY ALL THE WAY!)
                                             and use all the cRNA for 2<sup>nd</sup> round.
                                          </p>
                                       </li>
                                       <li>
                                          <p>If the concentration is greater than 10 ng/ul, use 100 ng for next step.</p>
                                       </li>
                                    </ol>
                                 </div>
                              </li>
                           </ol>
                        </div>
                     </li>
                     <li>
                        <p>Store cRNA at <span class="symbol">&#8722;</span>80&deg;C o/n.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Second Amplification Cycle</strong></span></p>
               <p><span class="bold"><strong>Step 6. Second Cycle, first strand cDNA synthesis</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Start with 100 ng of first round cRNA (if you have less, use all of your first round sample).</p>
                     </li>
                     <li>
                        <p>Mix the following in an RNase-free 1.5 ml microfuge tube:</p>
                        <div class="table"><a name="d0e5868"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center">Volume/reaction</th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>100 ng cRNA</strong></span></td>
                                    <td align="center">up to 10 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>Random primers, 1 ug/ul</strong></span></td>
                                    <td align="center">1 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>DEPC-H<sub>2</sub>O</strong></span></td>
                                    <td align="center">Add to final volume of 11 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em>total volume</em></span></td>
                                    <td align="center">11 ul</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Vortex and spin.</p>
                     </li>
                     <li>
                        <p>Incubate in 70&deg;C heat block for 10 min (to denature).</p>
                     </li>
                     <li>
                        <p>Cool on ice for 2 min.</p>
                     </li>
                     <li>
                        <p>Add the following to the tube for first strand synthesis (make master mix if more than 1 sample):</p>
                        <div class="table"><a name="d0e5918"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center">Volume/reaction</th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>(cRNA/primer mixture)</strong></span></td>
                                    <td align="center">(11 ul)</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>5X first strand buffer</strong></span></td>
                                    <td align="center">4 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>DTT, 0.1 M</strong></span></td>
                                    <td align="center">2 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>dNTP mix, 10 mM</strong></span></td>
                                    <td align="center">1 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>RNasin</strong></span></td>
                                    <td align="center">1 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em>total volume</em></span></td>
                                    <td align="center">19 ul</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Incubate in 42&deg;C water bath for 2 min</p>
                     </li>
                     <li>
                        <p>Add 1 ul SuperScript II. Vortex, spin.</p>
                     </li>
                     <li>
                        <p>Incubate at 42&deg;C for 1 hour.</p>
                     </li>
                     <li>
                        <p>Add 1 ul RNase H.</p>
                     </li>
                     <li>
                        <p>Incubate in 37 &deg;C water bath for 20 min, then incubate in 95&deg;C heat block for 5 min,</p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Step 7. Second cycle, second strand cDNA synthesis with T7-(dT)<sub>24</sub> primer</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Spin down first strand reaction, then add 1 ul T7-(dT)<sub>24</sub> primer.
                        </p>
                     </li>
                     <li>
                        <p>Incubate in 70&deg;C heat block for 10 minutes, then cool on ice.</p>
                     </li>
                     <li>
                        <p>Transfer solution to a 0.5ml RNase-free microfuge tube.</p>
                     </li>
                     <li>
                        <p>Add the following reagents to the mix (make master mix if more than 1 sample)</p>
                        <div class="table"><a name="d0e6002"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center"><span class="bold"><strong>Volume/reaction</strong></span></th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>(First strand mix/T7-(dT)<sub>24</sub>)</strong></span></td>
                                    <td align="center">(22 ul)</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>DEPC-H<sub>2</sub>O</strong></span></td>
                                    <td align="center">91 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>5X second strand buffer</strong></span></td>
                                    <td align="center">30 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>dNTP mix, 10 mM</strong></span></td>
                                    <td align="center">3 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em><span class="bold"><strong>E. coli</strong></span></em></span> <span class="bold"><strong>DNA Polymerase</strong></span></td>
                                    <td align="center">4 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em>total volume</em></span></td>
                                    <td align="center">150 ul</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Vortex, spin. Incubate at 16&deg;C in a PCR machine for 2 hr.</p>
                     </li>
                     <li>
                        <p>Add 2 ul of T4 DNA polymerase to the reaction and incubate for an additional 15 min at 16&deg;C.</p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Step 8. Second cycle, double-stranded cDNA clean-up by ethanol precipitation.</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Following second strand synthesis, transfer contents of 0.5 ml microfuge to 1.5 ml RNase-free microfuge tube.</p>
                     </li>
                     <li>
                        <p>Add following to tube (all items stored in RNase-free freezer):</p>
                        <div class="table"><a name="d0e6076"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center"><span class="bold"><strong>Volume/reaction</strong></span></th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>5 mg/ml glycogen</strong></span></td>
                                    <td align="center">1 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>5 M NH<sub>4</sub>OAc</strong></span></td>
                                    <td align="center">92 ul (0.6 volume)</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>Ice cold absolute EtOH</strong></span></td>
                                    <td align="center">612 ul (2.5 volumes)</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Mix by inversion. Centrifuge for 20 min at 4&deg;C at 14000 rpm. **You should have a pellet!**</p>
                     </li>
                     <li>
                        <p>REMOVE ETHANOL BY ASPIRATION USING RNase-free ROUND GEL-LOADING TIPS.</p>
                     </li>
                     <li>
                        <p>Wash with 70% cold ethanol (stored in freezer); spin for 14,000 rpm for 5 min at 4&deg;C.</p>
                     </li>
                     <li>
                        <p>Remove ethanol as above. Speed-vac for 5 min to dry pellet.</p>
                     </li>
                  </ol>
               </div>
               <p>THIS IS A POTENTIAL STOPPING POINT. STORE PELLET AT <span class="symbol">&#8722;</span>20&deg;C IF NOT CONTINUING ON TO IVT-LABELING REACTION.
               </p>
               <p><span class="bold"><strong>Step 9. Second cycle, IVT for cRNA amplification and labeling with Enzo kit.</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Resuspend pellet in 22 ul DEPC-H<sub>2</sub>O. Vortex and pipet up and down to resuspend pellet.
                        </p>
                     </li>
                     <li>
                        <p>Add the following to the resuspended pellet: DO NOT MAKE MASTER MIX!</p>
                        <div class="table"><a name="d0e6139"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center"><span class="bold"><strong>Volume/reaction</strong></span></th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>(cDNA)</strong></span></td>
                                    <td align="center">(22 ul)</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>10X HY reaction buffer (tube 1)</strong></span></td>
                                    <td align="center">4 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>10X Biotin labeled ribonucleotides (tube 2)</strong></span></td>
                                    <td align="center">4 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>10X DTT (tube 3)</strong></span></td>
                                    <td align="center">4 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>10X RNase inhibitor mix (tube 4)</strong></span></td>
                                    <td align="center">4 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>20X T7 RNA Polymerase</strong></span></td>
                                    <td align="center">2 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em>total volume</em></span></td>
                                    <td align="center">40 ul</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Incubate reaction in 37&deg;C water bath for 6 hours.</p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Step 10. Second cycle, labeled cRNA target clean-up with Affy Cleanup Kit.</strong></span></p>
               <p>*****Add 10 ul <span class="symbol">&#946;</span>-ME to every 1 ml IVT cRNA Binding Buffer before starting clean-up*****
               </p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Add 60 ul DEPC-H<sub>2</sub>O to IVT reaction (total volume = 100 ul)
                        </p>
                     </li>
                     <li>
                        <p>Add 350 ul IVT cRNA Binding Buffer; mix. Add 300 ul 100% EtOH. Mix by pipeting.</p>
                     </li>
                     <li>
                        <p>Add sample (750 ul) to a supplied mini-column. Pulse in microfuge for 15 sec.</p>
                     </li>
                     <li>
                        <p>Pipet flow-through from collection tube and pulse through mini-column again.</p>
                     </li>
                     <li>
                        <p>Discard collection tube. Transfer column to a new 2.0 ml collection tube.</p>
                     </li>
                     <li>
                        <p>Add 500 ul IVT cRNA Wash Buffer (to wash). Pulse in microfuge for 15 sec. Discard flow-through.</p>
                     </li>
                     <li>
                        <p>Add 500 ul 80% EtOH. Spin for 2 min in microfuge at full speed. Discard collection tube.</p>
                     </li>
                     <li>
                        <p>Transfer column to a supplied 1.5 ml collection tube.</p>
                     </li>
                     <li>
                        <p>Add 32 ul DEPC-H<sub>2</sub>O to mini-column. Incubate at RT for 1 min. Spin for 1 min at full-speed.
                        </p>
                     </li>
                     <li>
                        <p>Keep eluate, discard column.</p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Step 11. Determine concentration, remove aliquots for bioanalysis, fragment labeled cRNA.</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Add 2 ul cRNA to 98 ul DEPC-H<sub>2</sub>O (1:50 dilution).
                        </p>
                     </li>
                     <li>
                        <p>Use lab spec (DNA/RNA calculator) to determine A260. cRNA should be 0.5&#8211;3 ug/ul.</p>
                     </li>
                     <li>
                        <p>Dilute a portion of labeled cRNA to 100 ng/ul. Remove 3 ul for for gel electrophoresis (Agilent Bioanalyzer)</p>
                        <p>Fragment RNA in 10&#8211;15 ug aliquots.</p>
                        <div class="table"><a name="d0e6259"></a><p class="title"></p>
                           <table summary="" border="1">
                              <colgroup>
                                 <col align="center">
                                 <col align="center">
                                 <col align="center">
                              </colgroup>
                              <thead>
                                 <tr>
                                    <th align="center">&nbsp;</th>
                                    <th align="center"><span class="bold"><strong>10 ug</strong></span></th>
                                    <th align="center"><span class="bold"><strong>15 ug</strong></span></th>
                                 </tr>
                              </thead>
                              <tbody>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>cRNA</strong></span></td>
                                    <td align="center">Variable amount (<span class="symbol">&#8804;</span> than 12 ul)
                                    </td>
                                    <td align="center">Variable amount (<span class="symbol">&#8804;</span> than 22 ul)
                                    </td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>5X Fragmentation buffer (Affymetrix)</strong></span></td>
                                    <td align="center">4 ul</td>
                                    <td align="center">6 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="bold"><strong>DEPC-H<sub>2</sub>O</strong></span></td>
                                    <td align="center">up to 20 ul</td>
                                    <td align="center">up to 30 ul</td>
                                 </tr>
                                 <tr>
                                    <td align="center"><span class="emphasis"><em>total volume</em></span></td>
                                    <td align="center">20 ul</td>
                                    <td align="center">30 ul</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </li>
                     <li>
                        <p>Incubate at 95&deg;C for 35 min.</p>
                     </li>
                     <li>
                        <p>Place on ice following incubation.</p>
                     </li>
                     <li>
                        <p>Analyze unfragmented and fragmented samples on Bioanalyzer. If samples look good (<a href="#figure22">Figure 22</a>), you may apply to a <span class="emphasis"><em>C. elegans</em></span> Affy chip.
                        </p>
                        <div class="mediaobject" align="center"><a name="figure22"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="585">
<tr>
  <td align="center"><p><a href="cellfig22.jpg"><img src="cellfig22_s.jpg" border="2" align="middle" alt="figure 22"></a>
  </td>
</tr>
                           </table>
                           <div class="caption">
                              <p><b>Figure 22.&nbsp;</b>Assessing quality of RNA amplified from cultured cells for gene array hybridization. A. The prefragmented RNA should show
                                 a broad size distribution (X-axis). Smaller fragments should be more abundant (Y-axis, arrow), as the amplification process
                                 is biased to the 3' end. The gel image should be a large smear, with no intense high molecular weight products. Fragmented
                                 RNA should be 200 bp or less. B. These plots demonstrate a sample that is not hybridizable. The prefragmented RNA distribution
                                 includes a long tail indicative of non-template dependent high molecular weight products (open arrow). The presence of unfragmented
                                 peaks are evident (arrowhead). For patterns like this, we either repeat the amplification/fragmentation or discard that RNA
                                 sample (Steve Von Stetina, Rebecca Fox, and David Miller).
                              </p>
                           </div>
                        </div>
                     </li>
                     <li>
                        <p>Alternatively, you may store amplified and fragmented RNA samples at <span class="symbol">&#8722;</span>80&deg;C for hybridization at a later time.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sidebar">
               <p class="title"><b>Protocol 36: RNA interference in cultured embryonic cells (Kevin Strange and Rebecca Morrison)</b></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>dsRNA is synthesized and purified using well established protocols.</p>
                     </li>
                     <li>
                        <p>Equal numbers of cells from a cell isolate are plated in either control L-15 cell culture medium or L-15 medium containing
                           15 &#956;g/ml dsRNA.
                        </p>
                     </li>
                     <li>
                        <p>Two to three hours after plating, the dsRNA is diluted to a final concentration of 5 &#956;g/ml using L-15 culture medium. The
                           time course for gene expression knockdown is dependent on the half-life of the protein encoded by the targeted gene and on
                           the cell type in which it is expressed (<a href="#bib4">Christensen et al., 2002</a>).
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e6366"></a>6.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     <span class="bibliomisc">Aballay, A., and Ausubel, F. M. (2001). Programmed cell death mediated by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span> protects <span class="emphasis"><em>Caenorhabditis elegans</em></span> from Salmonella typhimurium-mediated killing. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>98</em></span>, 2735&#8211;2739.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11226309&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.041613098" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     <span class="bibliomisc">Brenner, S. (1974). The genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>77</em></span>, 71&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4366476&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     <span class="bibliomisc">Chalfie, M., Tu, Y., Euskirchen, G., Ward, W.W., and Prasher, D.C. (1994). Green fluorescent protein as a marker for gene
                        expression. Science <span class="emphasis"><em>263</em></span>, 802&#8211;805.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8303295&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     <span class="bibliomisc">Christensen, M., Estevez, A., Yin, X., Fox, R., Morrison, R., McDonnell, M., Gleason, C., Miller, D.M., 3rd, and Strange,
                        K. (2002). A primary culture system for functional analysis of <span class="emphasis"><em>C. elegans</em></span> neurons and muscle cells. Neuron <span class="emphasis"><em>33</em></span>, 503&#8211;514.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11856526&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(02)00591-3" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     <span class="bibliomisc">Collet, J., Spike, C.A., Lundquist, E.A., Shaw, J.E., and Herman, R.K. (1998). Analysis of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-6" target="_blank">osm-6</a></em></span>, a gene that affects sensory cilium structure and sensory neuron function in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>148</em></span>, 187&#8211;200.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9475731&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     <span class="bibliomisc">Costa, M., Draper, B.W., and Priess, J.R. (1997). The role of actin filaments in patterning the <span class="emphasis"><em>Caenorhabditis elegans</em></span> cuticle. Dev. Biol. <span class="emphasis"><em>184</em></span>, 373&#8211;384.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9133443&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/dbio.1997.8530" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ellis, R.E., and Horvitz, H.R. (1991). Two <span class="emphasis"><em>C. elegans</em></span> genes control the programmed deaths of specific cells in the pharynx. Development <span class="emphasis"><em>112</em></span>, 591&#8211;603.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1794327&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ellis, R.E., Jacobson, D.M., and Horvitz, H.R. (1991). Genes required for the engulfment of cell corpses during programmed
                        cell death in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>129</em></span>, 79&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1936965&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ellis, R.E., Jacobson, D.M., and Horvitz, H.R. (1991). Genes Required for the Engulfment of Cell Corpses During Programmed
                        Cell Death in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>129</em></span>, 79&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1936965&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     <span class="bibliomisc">Fares, H., and Greenwald, I. (2001). Genetic analysis of endocytosis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: coelomocyte uptake defective mutants. Genetics <span class="emphasis"><em>159</em></span>, 133&#8211;145.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11560892&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     <span class="bibliomisc">Fire, A., Xu, S., Montgomery, M.K., Kostas, S.A., Driver, S.E., and Mello, C.C. (1998). Potent and specific genetic interference
                        by double-stranded RNA in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>391</em></span>, 806&#8211;811.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9486653&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35888" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     <span class="bibliomisc">Fox, R.M., et al. (2005). A gene expression fingerprint of <span class="emphasis"><em>C. elegans</em></span> embryonic motor neurons. BMC Genomics 6(1), p. 42.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     <span class="bibliomisc">Francis, R., Barton, M.K., Kimble, J., and Schedl, T. (1995). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gld-1" target="_blank">gld-1</a></em></span>, a tumor suppressor gene required for oocyte development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>139</em></span>, 579&#8211;606.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7713419&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     <span class="bibliomisc">Gartner, A., Milstein, S., Ahmed, S., Hodgkin, J., and Hengartner, M.O. (2000). A conserved checkpoint pathway mediates DNA
                        damage&#8211;induced apoptosis and cell cycle arrest in <span class="emphasis"><em>C. elegans</em></span>. Mol. Cell <span class="emphasis"><em>5</em></span>, 435&#8211;443.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10882129&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)80438-4" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     <span class="bibliomisc">Gavrieli, Y., Sherman, Y., and Ben-Sasson, S.A. (1992). Identification of programmed cell death <span class="emphasis"><em>in situ</em></span> via specific labeling of nuclear DNA fragmentation. J. Cell. Biol. <span class="emphasis"><em>119</em></span>, 493&#8211;501.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1400587&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.119.3.493" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     <span class="bibliomisc">Grant, B., and Hirsh, D. (1999). Receptor-mediated endocytosis in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> oocyte. Mol. Biol. Cell <span class="emphasis"><em>10</em></span>, 4311&#8211;4326.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10588660&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     <span class="bibliomisc">Gumienny, T.L., Lambie, E., Hartwieg, E., Horvitz, H.R., and Hengartner, M.O. (1999). Genetic control of programmed cell death
                        in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> hermaphrodite germline. Development <span class="emphasis"><em>126</em></span>, 1011&#8211;1022.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9927601&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hall, D.H. (1995). Electron microscopy and three-dimensional image reconstruction. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 395&#8211;436.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531736&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hedgecock, E.M., Culotti, J.G., Thomson, J.N., and Perkins, L.A. (1985). Axonal guidance mutants of <span class="emphasis"><em>Caenorhabditis elegans</em></span> identified by filling sensory neurons with fluorescein dyes. Dev. Biol. <span class="emphasis"><em>111</em></span>, 158&#8211;170.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3928418&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(85)90443-9" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     <span class="bibliomisc">Helmchen, F., and Denk, W. (2002). New developments in multiphoton microscopy. Curr. Opin. Neurobiol. <span class="emphasis"><em>12</em></span>, 593&#8211;601.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12367641&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0959-4388(02)00362-8" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     <span class="bibliomisc">Jorgensen, E.M., and Mango, S.E. (2002). The art and design of genetic screens: <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Rev. Genet. <span class="emphasis"><em>3</em></span>, 356&#8211;369.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11988761&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg794" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lettre, G., Kritikou, E.A., Jaeggi, M., Calixto, A., Fraser, A.G., Kamath, R.S., Ahringer, J., and Hengartner, M.O. (2004).
                        Genome-wide RNAi identifies p53-dependent and -independent regulators of germ cell apoptosis in <span class="emphasis"><em>C. elegans</em></span>. Cell Death Differ.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15272318&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/sj.cdd.4401488" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     <span class="bibliomisc">Michalet, X., Kapanidis, A.N., Laurence, T., Pinaud, F., Doose, S., Pflughoefft, M., and Weiss, S. (2003). The power and prospects
                        of fluorescence microscopies and spectroscopies. Ann. Rev. Biophys. Biomol. Struct. <span class="emphasis"><em>32</em></span>, 161&#8211;182.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12598370&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.biophys.32.110601.142525" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     <span class="bibliomisc">Miller, D.M., 3rd, Desai, N.S., Hardin, D.C., Piston, D.W., Patterson, G.H., Fleenor, J., Xu, S., and Fire, A. (1999). Two-color
                        GFP expression system for <span class="emphasis"><em>C. elegans</em></span>. Biotechniques <span class="emphasis"><em>26</em></span>, 914&#8211;918, 920&#8211;911.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10337485&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     <span class="bibliomisc">Miyawaki, A. (2002). Green fluorescent protein-like proteins in reef Anthozoa animals. Cell Struct. Funct. <span class="emphasis"><em>27</em></span>, 343&#8211;347.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12502888&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1247/csf.27.343" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     <span class="bibliomisc">Nakano, A. (2002). Spinning-disk confocal microscopy &#8211; a cutting-edge tool for imaging of membrane traffic. Cell Struct. Funct.
                        <span class="emphasis"><em>27</em></span>, 349&#8211;355.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12502889&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1247/csf.27.349" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     <span class="bibliomisc">Paupard, M.C., Miller, A., Grant, B., Hirsh, D., and Hall, D.H. (2001). Immuno-EM localization of GFP-tagged yolk proteins
                        in <span class="emphasis"><em>C. elegans</em></span> using microwave fixation. J. Histochem. Cytochem. <span class="emphasis"><em>49</em></span>, 949&#8211;956.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11457923&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     <span class="bibliomisc">Perkins, L.A., Hedgecock, E.M., Thomson, J.N., and Culotti, J.G. (1986). Mutant sensory cilia in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>117</em></span>, 456&#8211;487.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2428682&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(86)90314-3" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     <span class="bibliomisc">Shaham, S., and Bargmann, C.I. (2002). Control of neuronal subtype identity by the <span class="emphasis"><em>C. elegans</em></span> ARID protein <a href="http://www.wormbase.org/db/seq/protein?name=CFI-1;class=protein" target="_blank">CFI-1</a>. Genes Dev. <span class="emphasis"><em>16</em></span>, 972&#8211;983.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11959845&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.976002" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     <span class="bibliomisc">Strome, S. (1986). Fluorescence visualization of the distribution of microfilaments in gonads and early embryos of the nematode
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Cell. Biol. <span class="emphasis"><em>103</em></span>, 2241&#8211;2252.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3782297&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.103.6.2241" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sulston, J.E., and Horvitz, H.R. (1977). Post-embryonic cell lineages of the nematode, <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>56</em></span>, 110&#8211;156.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=838129&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(77)90158-0" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sulston, J.E., Schierenberg, E., White, J.G., and Thomson, J.N. (1983). The embryonic cell lineage of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>100</em></span>, 64&#8211;119.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6684600&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(83)90201-4" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sulston, J.E., Schierenberg, E., White, J.G., and Thomson, J.N. (1983). The Embryonic Cell Lineage of the Nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>100</em></span>, 64&#8211;119.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6684600&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(83)90201-4" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                     <span class="bibliomisc">Thellmann, M., Hatzold, J., and Conradt, B. (2003). The Snail-like <a href="http://www.wormbase.org/db/seq/protein?name=CES-1;class=protein" target="_blank">CES-1</a> protein of <span class="emphasis"><em>C. elegans</em></span> can block the expression of the BH3-only cell-death activator gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-1" target="_blank">egl-1</a></em></span> by antagonizing the function of bHLH proteins. Development <span class="emphasis"><em>130</em></span>, 4057&#8211;4071.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12874127&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.00597" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                     <span class="bibliomisc">Timmons, L., and Fire, A. (1998). Specific interference by ingested dsRNA. Nature <span class="emphasis"><em>395</em></span>, 854.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9804418&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/27579" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wallace, W., Schaefer, L.H., and Swedlow, J.R. (2001). A workingperson's guide to deconvolution in light microscopy. Biotechniques
                        <span class="emphasis"><em>31</em></span>, 1076&#8211;1078, 1080, 1082 passim.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11730015&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wu, Y.C., Stanfield, G.M., and Horvitz, H.R. (2000). <a href="http://www.wormbase.org/db/seq/protein?name=NUC-1;class=protein" target="_blank">NUC-1</a>, a <span class="emphasis"><em>Caenorhabditis elegans</em></span> DNase II homolog, functions in an intermediate step of DNA degradation during apoptosis. Genes Dev. <span class="emphasis"><em>14</em></span>, 536&#8211;548.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10716942&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                     <span class="bibliomisc">Yang, T.T., Kain, S.R., Kitts, P., Kondepudi, A., Yang, M.M., and Youvan, D.C. (1996). Dual color microscopic imagery of cells
                        expressing the green fluorescent protein and a red-shifted variant. Gene <span class="emphasis"><em>173</em></span>, 19&#8211;23.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8707051&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0378-1119(95)00781-4" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zhang, J., Campbell, R.E., Ting, A.Y., and Tsien, R.Y. (2002). Creating new fluorescent probes for cell biology. Nat. Rev.
                        Mol. Cell. Biol. <span class="emphasis"><em>3</em></span>, 906&#8211;918.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12124626&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature00891" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zhang, S., Ma, C., and Chalfie, M. (2004). Combinatorial marking of cells and organelles with reconstituted fluorescent proteins.
                        Cell <span class="emphasis"><em>119</em></span>, 137&#8211;144.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15454087&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2004.09.012" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zhang, Y., Ma, C., Delohery, T., Nasipak, B., Foat, B.C., Bounoutas, A., Bussemaker, H.J., Kim, S.K., and Chalfie, M. (2002).
                        Identification of genes expressed in <span class="emphasis"><em>C. elegans</em></span> touch receptor neurons. Nature <span class="emphasis"><em>418</em></span>, 331&#8211;335.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12461557&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrm976" target="_blank"> Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zhou, Z., Hartwieg, E., and Horvitz, H.R. (2001). <a href="http://www.wormbase.org/db/seq/protein?name=CED-1;class=protein" target="_blank">CED-1</a> is a transmembrane receptor that mediates cell corpse engulfment in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>104</em></span>, 43&#8211;56.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11163239&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(01)00190-8" target="_blank"> Article</a>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised August 12, 2005. Published January 02, 2006. This chapter should be cited as: Shaham,
                  S., ed., WormBook: Methods in Cell Biology (January 02, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.49.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Shai Shaham and contributors. This is an open-access article distributed under the terms of the Creative Commons Attribution
                  License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source
                  are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e33" href="#d0e33">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:shaham@rockefeller.edu" target="_top">shaham@rockefeller.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

