<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Karyotype, ploidy, and gene dosage</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#Karyotype_ArticleID" title="Karyotype, ploidy, and gene dosage">
      <link rel="next" href="#d0e68" title="1.&nbsp;Normal karyotype">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="karyotype.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="Karyotype_ArticleID"></a>Karyotype, ploidy, and gene dosage<sup><a name="d0e10" href="#ftn.d0e10">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Jonathan</span> <span class="surname">Hodgkin<em><span class="remark"><sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="address">Genetics Unit, Department of Biochemistry, University of Oxford, Oxford OX1 3QU, UK
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_geneticsgenomics.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Genetics and genomics<br />>> Genetics
		</a></div><hr />

<a href="http://www.wormbase.org/forums/index.php?board=191.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Karyotype,+ploidy+and+gene+dosage" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e68">1. Normal karyotype</a></span></dt>
               <dt><span class="sect1"><a href="#d0e95">2. Overview of chromosome organization</a></span></dt>
               <dt><span class="sect1"><a href="#d0e194">3. Abnormal karyotype</a></span></dt>
               <dt><span class="sect1"><a href="#d0e236">4. Tetraploids, triploids and haploids</a></span></dt>
               <dt><span class="sect1"><a href="#d0e303">5. Aneuploids</a></span></dt>
               <dt><span class="sect1"><a href="#d0e514">6. Chromosomal duplications</a></span></dt>
               <dt><span class="sect1"><a href="#d0e542">7. Chromosomal deficiencies</a></span></dt>
               <dt><span class="sect1"><a href="#d0e739">8. Dosage sensitive genes</a></span></dt>
               <dt><span class="sect1"><a href="#d0e823">9. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>The normal karyotype of</strong></span> <span class="emphasis"><em><span class="bold"><strong>Caenorhabditis
                                       elegans,</strong></span></em></span> <span class="bold"><strong>with its five pairs of autosomes and single pair of X chromosomes,
                                 is described. General features of chromosomes and global differences between different
                                 chromosomal regions are discussed. Abnormal karyotypes, including duplications,
                                 deficiencies, inversions, translocations and chromosome fusions are reviewed. The
                                 effects of varying ploidy and of varying gene dosage are summarized. Dosage-sensitive
                                 genes seem to be rare in</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans,</strong></span></em></span> <span class="bold"><strong>and the
                                 organism is able to tolerate substantial levels of aneuploidy. However, autosomal
                                 hemizygosity for more than about 3<span class="symbol">%</span> of the
                                 total genome may be incompatible with viability.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e68"></a>1.&nbsp;Normal karyotype
                     </h2>
                  </div>
               </div>
            </div>
            <p>In the earliest genetic investigations of <span class="emphasis"><em>C. elegans</em></span>, <a href="#bib25">Nigon (1949a)</a> showed that hermaphrodites and males both had five pairs of autosomes, but differed in the presence of two X chromosomes
               in hermaphrodites (<span class="emphasis"><em>2A;2X</em></span>) as opposed to one X chromosome in males (<span class="emphasis"><em>2A;1X</em></span>). Cytologically, all chromosomes appear as short rods at mitosis or meiosis, with no obvious substructure.
            </p>
            <p>The six chromosomes are roughly equal in size, both in terms of DNA content [ranging from 12.8 Mb (Linkage Group III, abbreviated
               LGIII) to 20.8 Mb (LGV)] and in terms of recombinational length, at about 50 centimorgans each. The 50 cM genetic map length
               reflects the fact that each chromosome usually experiences one and only one crossover at meiosis (see <a href="http://www.wormbook.org/chapters/www_meiosis/meiosis.html" target="_blank">Meiosis</a>).
            </p>
            <p>The mitochondrial genome is a small circular molecule of 13.8 kb, and is discussed at greater length elsewhere (see <a href="http://www.wormbook.org/chapters/www_mitogenetics/mitogenetics.html" target="_blank">Mitochondrial genetics</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e95"></a>2.&nbsp;Overview of chromosome organization
                     </h2>
                  </div>
               </div>
            </div>
            <p>Each of the five linkage groups corresponding to the five autosomes has a discernible
                           organization, with a central region or cluster, containing a higher density of genes,
                           flanked by arms with fewer genes. These clusters were apparent in the first genetic map
                           of <span class="emphasis"><em>C. elegans</em></span>, worked out by Brenner (<a href="#bib5">Brenner,1974</a>).
                           With the availability of genomic information, it became clear that part of the
                           clustering results from long-range variation in recombination rate, with a metric (ratio
                           of physical distance to genetic distance) of 100 - 500 kb/cM on the arms, as compared to
                           1500 kb/cM in the cluster regions (<a href="#bib4">Barnes et al., 1995</a>). Various general
                           sequence features also distinguish the arm regions from the clusters: the arm regions
                           have, overall, a slightly lower gene density (5.12 kb per gene as compared to 4.56 kb
                           per gene in the clusters) and a different distribution of repeat sequences (<a href="#bib6"><span class="emphasis"><em>C.
                                    elegans</em></span> Sequencing Consortium, 1998</a>). Initial annotations also
                           indicated that the cluster regions contain relatively more conserved and essential genes
                           than do the arms (<a href="#bib6"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>; <a href="#bib15">Hutter et al., 2000</a>). Large-scale functional analyses of the
                           genome by RNAi have confirmed and extended these regional distinctions (<a href="#bib17">Kamath et al. 2003</a>; <a href="#bib21">Miller et al., 2004</a>).
            </p>
            <p>The X chromosome exhibits much less dramatic variation in recombination across its
                           length, so that there is no obvious gene cluster in the X linkage map, and it also has a
                           significantly lower gene density (6.54 kb per gene) (<a href="#bib6"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>). The X chromosome is also unusual from a
                           functional point of view, in that essential genes are significantly under-represented on
                           this chromosome as compared to the autosomes (<a href="#bib28">Piano et al., 2000</a>; <a href="#bib29">Piano et al. 2002</a>; <a href="#bib30">Reinke et al., 2000</a>; <a href="#bib18">Kelly et al. 2002</a>).
            </p>
            <p><span class="emphasis"><em>C. elegans</em></span> chromosomes, like those of nematodes in general, do not
                           have a single centromere. At mitosis, spindle microtubules attach at multiple points
                           along the chromosome (<a href="#bib2">Albertson and Thomson, 1982</a>). It is not clear
                           whether there are preferred points on the chromosomes for spindle attachment; if so,
                           these chromosomes should be regarded as polycentric rather than holocentric.
                           Heterologous DNA, introduced as a transgenic array, seems to be handled fairly
                           adequately by the mitotic machinery (<a href="#bib34">Stinchcomb et al., 1985</a>), which suggests that there are no special sequence requirements for spindle attachment (see
                               <a href="http://www.wormbook.org/chapters/www_celldivision/celldivision.html" target="_blank">Cell division</a>).
            </p>
            <p>At meiosis, there is a necessary requirement for a single attachment site on each
                           chromosome to allow resolution and segregation of recombined chromatids. It appears that
                           either end of each chromosome can be used for this purpose (<a href="#bib3">Albertson and Thomson, 1993</a>). With respect to the initial pairing of chromosomes at meiosis, the ends do differ: there appears to be a preferred pairing
               site close to one end of each chromosome (see <a href="http://www.wormbook.org/chapters/www_meiosis/meiosis.html" target="_blank">Meiosis</a>; right end of LGI and LGV; left end of LGII, LGIII, LGIV, LGX).
            </p>
            <p>The absence of a defined centromere means that the genetic maps for each chromosome
                           were plotted around arbitrarily chosen zero points, in the approximate middle of each
                           chromosome. Genes to the left of this point were given negative coordinates, and are
                           drawn on the upper arm in vertical display; genes to the right were given positive
                           coordinates, and are drawn on the lower arm in vertical display. With the advent of
                           complete sequence information, it has become possible to correlate these genetic map
                           coordinates with absolute sequence coordinates.
            </p>
            <p>The telomeres of <span class="emphasis"><em>C. elegans</em></span> conform to the usual eukaryotic
                           pattern of a long series of hexanucleotide repeats, with a repeat sequence TTAGGC
                           similar to the vertebrate repeat TTAGGG (<a href="#bib36">Wicky et al., 1996</a>). In contrast to some organisms, however, there are no obviously specialized sub-telomeric regions.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e194"></a>3.&nbsp;Abnormal karyotype
                     </h2>
                  </div>
               </div>
            </div>
            <p> Many alterations to the normal karyotype have been generated and examined. In
                           addition to changes in whole chromosome number, discussed in the following section, a
                           variety of sub-chromosomal alterations are possible, including duplications,
                           deficiencies, inversions, and translocations.
            </p>
            <p>A special class of translocations is those involving the fusion of whole chromosomes.
                           The first of these to be reported was <span class="emphasis"><em>mnT12</em></span>, a fusion between the left end of LGIV and the right end of the X chromosome (<a href="#bib33">Sigurdson et al., 1984</a>). A stable and fairly healthy hermaphrodite line having only five pairs of chromosomes could be generated using <span class="emphasis"><em>mnT12</em></span>. Males of constitution <span class="emphasis"><em>IV/mnT12/0</em></span> are fertile, and can be regarded as having a neo-X
                           (mnT12) neo-Y (LGIV) constitution.
            </p>
            <p>Similar chromosomal fusions can now be readily generated by growing mutants with
                           telomere maintenance defects (<a href="#bib1">Ahmed and Hodgkin, 2000</a>). After many
                           generations, these mutants experience telomere erosion, which leads to promiscuous
                           chromosome fusion. The resulting fusion chromosomes are propagated reasonably stably,
                           because the holocentric nature of nematode chromosomes is permissive for such
                           structures. A variety of different fusions have been generated so far, and it is not
                           clear if there is any limit to the possible combinations that can be generated, or to
                           the associated variant karyotypes. A fusion of three chromosomes has been used to
                           examine control of crossover distribution (<a href="#bib9">Hillers and Villeneuve, 2003</a>).
            </p>
            <p>An attached-X chromosome (two X chromosomes fused at their left ends, symbolized <span class="emphasis"><em>X^X</em></span>)
                           has also been obtained in a different kind of experiment; this can be propagated but
                           tends to break down, reverting to a normal XX constitution (<a href="#bib13">Hodgkin and Albertson, 1995</a>). In comparison, attached X (and attached autosome) chromosomes that have
                           been generated for <span class="emphasis"><em>Drosophila melanogaster</em></span> are relatively stable.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e236"></a>4.&nbsp;Tetraploids, triploids and haploids
                     </h2>
                  </div>
               </div>
            </div>
            <p>Nigon was also able to establish and examine tetraploid lines of the species, although
                           these were subsequently lost (<a href="#bib26">Nigon 1949b</a>, <a href="#bib23">1951a</a>, <a href="#bib24">1951b</a>). He found two types of
                           tetraploid hermaphrodite, both larger in size than normal diploids; the two types
                           differed in progeny production, one producing <span class="symbol">&lt;</span>1<span class="symbol">%</span> males and the other more than 40<span class="symbol">%</span> males. He concluded that these were of
                           constitution 4A;4X and 4A;3X respectively, and that the males were of karyotype 4A;2X.
                           Madl and Herman confirmed and extended these conclusions (<a href="#bib19">Madl and Herman, 1979</a>). They reported that 4A;4X hermaphrodites were of lower fertility than diploids
                           (average tetraploid brood 84 eggs as compared to about 300 for diploids). Tetraploid egg
                           viability was lower, with 87<span class="symbol">%</span> of eggs
                           hatching compared to <span class="symbol">&gt;</span>99<span class="symbol">%</span> for diploids.
            </p>
            <p>Triploids were also generated and examined by these authors, by mating diploid males
                           and tetraploid hermaphrodites and vice versa. 3A;3X animals were found to be
                           hermaphrodite and 3A;2X animals were male. The 3A;3X hermaphrodites exhibited high
                           levels of embryonic inviability in their broods, with only 15<span class="symbol">%</span> of eggs hatching, which presumably results from
                           the production of many aneuploid progeny.
            </p>
            <p>The implications of these observations on the effect of X chromosome dosage and
                           autosomal dosage on sexual phenotype are discussed more extensively in the chapters on
                           Sex determination.
            </p>
            <p>Haploids of <span class="emphasis"><em>C. elegans</em></span>, generated by extrusion of the maternal
                           pronucleus after fertilization, are unable to complete embryogenesis although they
                           produce several hundred cells. (<a href="#bib31">Schierenberg and Wood, 1985</a>). It was
                           suggested that a female pronucleus might be necessary for successful embryogenesis,
                           despite the fact that no imprinting appears to occur in <span class="emphasis"><em>C. elegans</em></span>,
                           as discussed below. Chromosomal balance should be identical between a normal diploid
                           (<span class="emphasis"><em>2A;2X</em></span>) and a haploid (<span class="emphasis"><em>1A;1X</em></span>). Another
                           possible explanation for haploid inviability is that excess dosage compensation occurs,
                           resulting in insufficient X chromosome expression and therefore lethality. A third
                           possibility is that the nuclear volume of a haploid <span class="emphasis"><em>C. elegans</em></span> cell
                           is simply too small to sustain normal gene regulation and expression. Nuclear volume
                           seems to be largely determined by DNA content, and the 50 megabase DNA content of a
                           haploid <span class="emphasis"><em>C. elegans</em></span> nucleus would be smaller than any known animal genome.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e303"></a>5.&nbsp;Aneuploids
                     </h2>
                  </div>
               </div>
            </div>
            <p>In principle, there are six possible trisomies and six possible monosomies for the
                               <span class="emphasis"><em>C. elegans</em></span> diploid genome, of which one, monosomy X or XO,
                           occurs naturally in the form of the male sex. XO individuals are entirely viable as a
                           consequence of the X chromosome dosage compensation system (see <a href="http://www.wormbook.org/chapters/www_dosagecomp/dosagecomp.html" target="_blank">X-chromosome dosage compensation</a>), which equalizes gene expression between the XX sex (normally hermaphrodite)
                           and the XO sex (normally male). Trisomy X is also viable but abnormal:
                           <span class="emphasis"><em>2A;3X</em></span> individuals, otherwise known as triplo-X animals, are
                           hermaphrodites with a dumpy body morphology, slower growth rate and lower fertility than
                           normal <span class="emphasis"><em>2A;2X</em></span> hermaphrodites (<a href="#bib14">Hodgkin et al., 1979</a>). Since they are
                           abnormal, it seems that the dosage compensation mechanism is not proportionately more
                           active in a triplo-X animal than in a diplo-X animal. Triplo-X animals with partial
                           defects in dosage compensation, such as <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-21" target="_blank">dpy-21</a></em></span> mutants (which are
                           viable but dumpy as diplo-X individuals) are inviable (<a href="#bib10">Hodgkin, 1983</a>). Individuals of
                           karyotype <span class="emphasis"><em>2A;4X</em></span> appear to be inviable, as inferred from the
                           proportion of dead eggs produced in the progeny of <span class="emphasis"><em>2A;3X</em></span>
                           hermaphrodites. In all these cases, the inviabilities presumably result from cumulative
                           over-expression of sex-linked genes. It is unlikely that any single gene on the X
                           chromosome contributes a major part of the lethality, because screens for suppressors of
                           the lethality due to defective dosage compensation have not yielded any candidates.
                           Reduction in X chromosome dose has also been examined: individuals of karyotype
                           <span class="emphasis"><em>3A;1X</em></span> are viable males, but they are noticeably abnormal and
                           unhealthy, suggesting that any further reduction would be lethal (<a href="#bib12">Hodgkin, 1987</a>).
            </p>
            <p>Among autosomal aneuploids, one trisomy (for LGIV) is known to be viable. Trisomy IV
                           was generated by means of the IV-X fusion chromosome, <span class="emphasis"><em>mnT12</em></span>, and
                           also by using a meiotic mutant, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-6" target="_blank">him-6</a></em></span> (<a href="#bib33">Sigurdson et al., 1984</a>). The triplo-IV animals have surprisingly normal appearance and viability,
                           although they produce 50<span class="symbol">%</span> fewer progeny
                           than normal hermaphrodites. The inviability of most of the other possible aneuploids was
                           initially suggested by the very high level of embryonic lethality observed in some
                           meiotic mutants, which have general defects in recombination and chromosome
                           nondisjunction. Lethality is most probably due to extensive nondisjunction resulting in
                           many nullisomic and disomic gametes and hence to aneuploid progeny. Consistent with this
                           interpretation, males of such meiotic mutants sire many inviable progeny, when crossed
                           with normal hermaphrodites (<a href="#bib14">Hodgkin et al., 1979</a>).
            </p>
            <p>The general meiotic mutant <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-6" target="_blank">him-6</a></em></span> was explicitly tested for the
                           production of nullisomic and disomic gametes for each chromosome in turn (<a href="#bib8">Haack and
                              Hodgkin, 1991</a>). The results showed that all of these possible abnormal
                           gamete types were being generated, at roughly equal frequencies. These experiments also
                           tested for gross chromosomal imprinting effects, which appear to be absent, because it
                           was possible to recover normal diploids that had received a pair of autosomes only from
                           the maternal parent or only from the paternal parent.
            </p>
            <p>The inviability of all autosomal monosomies can be inferred independently from the
                           failure to recover them in any of the many screens for genetic deficiencies that have
                           been carried out. Such screens have concentrated on specific regions on each chromosome,
                           and as a result large partial deficiencies have been recovered for each of the five
                           autosomes (these are discussed further below), but no cases of monosomy have been seen.
                           The fact that large deficiencies are usually conspicuously unhealthy suggests that there
                           is a limit to how much haploidy can be tolerated in the genome, and that a deficiency as
                           large as a whole chromosome would be well above this limit. It follows that there is no
                           general mechanism for autosomal dosage compensation.
            </p>
            <p>Autosomal trisomies have not been recovered from screens for duplications, although
                           trisomy for LGIV might have been obtained, and the large duplication
                           <span class="emphasis"><em>mnDp37</em></span> appears to duplicate almost all of LGIII (<a href="#table1" title="Table 1. Large free duplications. The table lists some of the larger free&#xA;            duplications that have been obtained so far, together with their approximate extent as&#xA;            inferred from genetic data, and minimum size in Mb, based on the physical distance&#xA;            between loci known to be duplicated. Exact endpoints are not known for most&#xA;            duplications. Some of the duplications may however contain internal gaps, and therefore&#xA;            be somewhat smaller than indicated.">Table 1</a>). Some of the other autosomal trisomies (for LGI, LGII and LGV) may possibly be viable, but no explicit crosses designed
               to generate them have yet been reported.
            </p>
            <div class="table"><a name="table1"></a><p class="title"><span class="bold">Table 1. Large free duplications.</span> The table lists some of the larger free
                              duplications that have been obtained so far, together with their approximate extent as
                              inferred from genetic data, and minimum size in Mb, based on the physical distance
                              between loci known to be duplicated. Exact endpoints are not known for most
                              duplications. Some of the duplications may however contain internal gaps, and therefore
                              be somewhat smaller than indicated.
               </p>
               <table summary="Table 1. Large free duplications. The table lists some of the larger free&#xA;            duplications that have been obtained so far, together with their approximate extent as&#xA;            inferred from genetic data, and minimum size in Mb, based on the physical distance&#xA;            between loci known to be duplicated. Exact endpoints are not known for most&#xA;            duplications. Some of the duplications may however contain internal gaps, and therefore&#xA;            be somewhat smaller than indicated." border="1">
                  <colgroup>
                     <col align="center">
                     <col align="center">
                     <col align="center">
                     <col align="center">
                  </colgroup>
                  <thead>
                     <tr>
                        <th align="center">Duplication name</th>
                        <th align="center">Location</th>
                        <th align="center">Approximate extent (centimorgans)</th>
                        <th align="center">Minimum size (megabases)</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td align="center"><span class="emphasis"><em>sDp2</em></span></td>
                        <td align="center"><span class="emphasis"><em>I, L</em></span></td>
                        <td align="center">21.4</td>
                        <td align="center">7.2</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>sDp1</em></span></td>
                        <td align="center"><span class="emphasis"><em>I, R</em></span></td>
                        <td align="center">31.7</td>
                        <td align="center">10.6</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>mnDp34</em></span></td>
                        <td align="center"><span class="emphasis"><em>II, R</em></span></td>
                        <td align="center">21.3</td>
                        <td align="center">3.8</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>sDp3</em></span></td>
                        <td align="center"><span class="emphasis"><em>III, L</em></span></td>
                        <td align="center">23.0</td>
                        <td align="center">6.6</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>mnDp37</em></span></td>
                        <td align="center"><span class="emphasis"><em>III</em></span></td>
                        <td align="center">40.3</td>
                        <td align="center">11.6</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>eDp6</em></span></td>
                        <td align="center"><span class="emphasis"><em>III, R</em></span></td>
                        <td align="center">15.7</td>
                        <td align="center">2.8</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>mDp4</em></span></td>
                        <td align="center"><span class="emphasis"><em>IV, L</em></span></td>
                        <td align="center">31.7</td>
                        <td align="center">9.1</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>yDp1</em></span></td>
                        <td align="center"><span class="emphasis"><em>IV, R+V, L</em></span></td>
                        <td align="center">26.0</td>
                        <td align="center">7.0</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>yDp13</em></span></td>
                        <td align="center"><span class="emphasis"><em>X, L</em></span></td>
                        <td align="center">15.7</td>
                        <td align="center">2.4</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>mnDp3</em></span></td>
                        <td align="center"><span class="emphasis"><em>X, R</em></span></td>
                        <td align="center">14.9</td>
                        <td align="center">4.3</td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e514"></a>6.&nbsp;Chromosomal duplications
                     </h2>
                  </div>
               </div>
            </div>
            <p>The holocentric nature of nematode chromosomes means that chromosomal fragments can be
                           propagated readily through mitosis, as long as they retain a sufficient amount of
                           centric material to capture spindle microtubules. As a result, chromosomal fragments
                           have been generated and preserved for many parts of the <span class="emphasis"><em>C. elegans</em></span>
                           genome, as free duplications. Some of the larger free duplications are listed in <a href="#table1" title="Table 1. Large free duplications. The table lists some of the larger free&#xA;            duplications that have been obtained so far, together with their approximate extent as&#xA;            inferred from genetic data, and minimum size in Mb, based on the physical distance&#xA;            between loci known to be duplicated. Exact endpoints are not known for most&#xA;            duplications. Some of the duplications may however contain internal gaps, and therefore&#xA;            be somewhat smaller than indicated.">Table 1</a>. Additional duplications have also been recovered as stable translocations to other chromosomes.
            </p>
            <p>The limits to generation of free chromosomal duplications are not known,
                           but very small free duplications have never been reported.
                           Extrachromosomal transgenic arrays which behave much like free duplications can be
                           generated after microinjection; these arrays appear always to carry at least 40 copies
                           of the injected sequences, probably adding up to a megabase or more in total length
                           (<a href="#bib34">Stinchcomb et al., 1985</a>). Smaller arrays and small chromosomal fragments
                           may therefore not be successfully handled by the mitotic machinery.
            </p>
            <p>More than 80<span class="symbol">%</span> the <span class="emphasis"><em>C.
                                 elegans</em></span> genome has been duplicated by one means or another, implying that a
                               50<span class="symbol">%</span> increase in gene dosage for any of
                           the genes concerned has no seriously deleterious effect. Two large regions, on the left
                           arm of LGII and the right arm of LGV, are not represented in the set of duplications,
                           probably because no serious attempts have yet been made to generate duplications for
                           these regions.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e542"></a>7.&nbsp;Chromosomal deficiencies
                     </h2>
                  </div>
               </div>
            </div>
            <p>Numerous chromosomal deficiencies have been generated in the course of many different
                           investigations, such that more than 75<span class="symbol">%</span> of
                           the genome is now covered by at least one deficiency. These deficiencies provide
                           extremely useful tools for analysis of mutations, for manipulation of gene dosage and
                           for mapping. The majority of deficiencies appear to be continuous, with breakpoints that
                           can be exactly defined on both genetic and molecular maps. However, it should be noted
                           that some deficiencies, like some duplications, appear to have more complex structures,
                           with discontinuities or rearrangements within the overall span apparently deleted.
            </p>
            <p>A few large segments of the genetic map are currently lacking in any deficiencies,
                           notably on the right arm of LGII (17 centimorgans) and the left arm of LGIV (26
                           centimorgans). These absences may simply result from a lack of any systematic screens
                           for deficiencies in the relevant segments, rather than from failure to recover them.
                           However, in some small regions of the genome, deficiencies appear to be difficult or
                           impossible to generate, presumably because one or more haplo-insufficient loci
                           (discussed below) are present in the intervals concerned. A conspicuous example is
                           provided by a region on LG II, genetic map interval 0.79 to 0.81. Numerous deficiencies
                           extend leftward or rightward from this interval, but none crosses it (<a href="#bib33">Sigurdson et al., 1984</a>). One or more genes in this gap are likely to be
                           haplo-insufficient, so that a deletion would have dominant lethal effects. The only gene
                           in the gap that has been studied in any detail is <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=zyg-9" target="_blank">zyg-9</a></em></span>, which
                           encodes a conserved spindle component, but mutations in this gene show no dominant
                           effects (<a href="#bib20">Matthews et al., 1998</a>), so <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=zyg-9" target="_blank">zyg-9</a></em></span> does not
                           appear to be needed in more than one copy. Some other possible haplo-insufficient
                           regions can be tentatively identified on other chromosomes, but their existence is hard
                           to prove, and the whole genome may contain fewer than ten.
            </p>
            <p>The large-scale structure of chromosomes in <span class="emphasis"><em>C. elegans</em></span> probably
                           also contributes to the size and distribution of known deficiencies. As discussed above,
                           the five autosomes all have a structure with a central cluster region, with slightly
                           higher gene density, reduced recombination, and more conserved and essential genes, as
                           compared to the flanking arm regions. <a href="#table2" title="Table 2. Large deficiencies. The table lists a set of large deficiencies, two for each chromosome. Approximate genetic extents have been inferred from genetic data. Physical sizes are estimated from the distance between the last sequence-positioned genes that are known to be deleted, at either end of each deficiency. Therefore, these sizes may be underestimates. Exact endpoints are known for few deficiencies. Also, some deficiencies may be discontinuous, and therefore remove less of the genome than their predicted length suggests.">Table 2</a> lists some of the largest deficiencies
                           that have been obtained so far, together with estimates of their genetic length in
                           centimorgans and minimum physical extent in Mb. All the largest deficiencies affect
                           mainly arm regions, and none removes an entire cluster or more than about 4 Mb. Most or
                           all of these deficiencies exhibit significantly reduced viability and fertility, as
                           heterozygotes (<span class="emphasis"><em>Df/+</em></span>). These deleterious effects probably result from the cumulative effect of reducing gene dosage. Consequently, it is expected
               that deficiencies larger
                           than those generated so far would be dominantly lethal, although all of the genes
                           affected may be individually haplo-sufficient. This effect will be greater for the
                           cluster regions, because of the higher density of essential and non-redundant genes. One
                           can predict that any deficiency, or combination of deficiencies, that removes more than
                           about 3<span class="symbol">%</span> of the total gene content of
                               <span class="emphasis"><em>C. elegans</em></span>, will be dominantly lethal. Tests of this prediction
                           by combining smaller deficiencies on different chromosomes are feasible, but have not
                           yet been carried out.
            </p>
            <div class="table"><a name="table2"></a><p class="title"><span class="bold">Table 2. Large deficiencies.</span> The table lists a set of large deficiencies, two for each chromosome. Approximate genetic extents have been inferred from
                  genetic data. Physical sizes are estimated from the distance between the last sequence-positioned genes that are known to
                  be deleted, at either end of each deficiency. Therefore, these sizes may be underestimates. Exact endpoints are known for
                  few deficiencies. Also, some deficiencies may be discontinuous, and therefore remove less of the genome than their predicted
                  length suggests.
               </p>
               <table summary="Table 2. Large deficiencies. The table lists a set of large deficiencies, two for each chromosome. Approximate genetic extents have been inferred from genetic data. Physical sizes are estimated from the distance between the last sequence-positioned genes that are known to be deleted, at either end of each deficiency. Therefore, these sizes may be underestimates. Exact endpoints are known for few deficiencies. Also, some deficiencies may be discontinuous, and therefore remove less of the genome than their predicted length suggests." border="1">
                  <colgroup>
                     <col align="center">
                     <col align="center">
                     <col align="center">
                     <col align="center">
                  </colgroup>
                  <thead>
                     <tr>
                        <th align="center">Deficiency name</th>
                        <th align="center">Location</th>
                        <th align="center">Approximate extent (centimorgans)</th>
                        <th align="center">Minimum size (megabases)</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td align="center"><span class="emphasis"><em>hDf10</em></span></td>
                        <td align="center"><span class="emphasis"><em>I, L</em></span></td>
                        <td align="center">6.5</td>
                        <td align="center">1.8</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dxDf2</em></span></td>
                        <td align="center"><span class="emphasis"><em>I, R</em></span></td>
                        <td align="center">15.0</td>
                        <td align="center">3.1</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>ccDf11</em></span></td>
                        <td align="center"><span class="emphasis"><em>II, L</em></span></td>
                        <td align="center">14.1</td>
                        <td align="center">4.6</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>mnDf89</em></span></td>
                        <td align="center"><span class="emphasis"><em>II, RC</em></span></td>
                        <td align="center">3.7</td>
                        <td align="center">2.5</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>nDf11</em></span></td>
                        <td align="center"><span class="emphasis"><em>III, LC</em></span></td>
                        <td align="center">5.1</td>
                        <td align="center">1.1</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>eDf2</em></span></td>
                        <td align="center"><span class="emphasis"><em>III, R</em></span></td>
                        <td align="center">15.9</td>
                        <td align="center">2.9</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>mDf9</em></span></td>
                        <td align="center"><span class="emphasis"><em>IV, LC</em></span></td>
                        <td align="center">2.6</td>
                        <td align="center">1.1</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>nDf27</em></span></td>
                        <td align="center"><span class="emphasis"><em>IV, C</em></span></td>
                        <td align="center">5.3</td>
                        <td align="center">2.1</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>sDf45</em></span></td>
                        <td align="center"><span class="emphasis"><em>V, L</em></span></td>
                        <td align="center">11.5</td>
                        <td align="center">3.7</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>ozDf1</em></span></td>
                        <td align="center"><span class="emphasis"><em>V, R</em></span></td>
                        <td align="center">12.0</td>
                        <td align="center">3.0</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>syDf1</em></span></td>
                        <td align="center"><span class="emphasis"><em>X, L</em></span></td>
                        <td align="center">4.6</td>
                        <td align="center">2.0</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>mnDf1</em></span></td>
                        <td align="center"><span class="emphasis"><em>X, R</em></span></td>
                        <td align="center">8.0</td>
                        <td align="center">3.4</td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e739"></a>8.&nbsp;Dosage sensitive genes
                     </h2>
                  </div>
               </div>
            </div>
            <p>Most genes display no obvious phenotype when their dosage is reduced by 50<span class="symbol">%</span>, in hemizygotes, or increased by 50<span class="symbol">%</span>, in duplication-bearing animals. In general, it
                           is likely that such alterations in dosage do have a small impact on viability, which is
                           cumulatively deleterious in large deficiency heterozygotes, but the reduction in
                           viability is not easily perceptible at the level of a single gene. Moreover,
                           considerations from metabolic control theory indicate that genes encoding enzymes in
                           metabolic pathways are expected in general to be insensitive to variations in dose
                           (<a href="#bib16">Kacser and Burns, 1981</a>).
            </p>
            <p>As noted above, the absence of hemizygously viable deficiencies for certain parts of
                           the genome suggests that they may contain haplo-lethal genes, but no specific case of
                           haplo-lethality has yet been demonstrated. A few genes do appear to be conspicuously
                           haplo-insufficient, such that hemizygotes exhibit a recognizable phenotype. Most of
                           these seem to be genes encoding structural proteins or regulatory molecules. An example
                           of the former is the collagen gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-13" target="_blank">dpy-13</a></em></span> (von Mende et al., 1988). Examples of the latter are provided by the sex determination genes
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-2" target="_blank">tra-2</a></em></span> (<a href="#bib35">Trent et al., 1983</a>) and
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a></em></span> (<a href="#bib11">Hodgkin, 1986</a>). The expression of both of
                           these genes is partly regulated by translational control, in order to achieve proper
                           development of the hermaphrodite germ line (see <a href="http://www.wormbook.org/chapters/www_sexgermline/sexgermline.html" target="_blank">Sex-determination in the germ line</a>) so it may be that transcription levels for these genes are necessarily limiting, in order to facilitate
                           the translational modulation. The paucity of haplo-insufficient genes in <span class="emphasis"><em>C.
                                 elegans</em></span> is also evident from a general screen for dominant mutations in this
                           organism (<a href="#bib27">Park and Horvitz, 1986</a>), which yielded many gain-of-function
                           mutations but only one candidate for a gene with a dominant null phenotype, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-108" target="_blank">unc-108</a></em></span>.
            </p>
            <p>Haplo-insufficiency is rare when genes are considered alone and on a wildtype
                           background, but it can become much more frequent on a sensitized background. Effects of
                           this type are described and discussed further in the Chapters on <a href="http://www.wormbook.org/chapters/www_complementation/complementation.html" target="_blank">Complementation</a> and <a href="http://www.wormbook.org/chapters/www_geneticenhancers/geneticenhancers.html" target="_blank">Genetic enhancers</a>.
            </p>
            <p>No examples of triplo-lethal genes have yet been reported in <span class="emphasis"><em>C. elegans</em></span>. However, further increases in the dosage of many individual genes
                           can be deleterious. Such increases are routinely achieved in the generation of multicopy
                           extrachromosomal transgene arrays, which may have 50 or more copies of an individual
                           gene. There has been no systematic study of copy number and corresponding expression
                           levels for these arrays, and it is unlikely that these arrays express gene products as
                           efficiently as the normal chromosomal copies do, but several-fold over-expression seems
                           probable. Creation of multi-copy arrays is difficult or impossible for some cosmid
                           transgenes, probably as a result of such over-expression lethality. A few specific cases
                           have been demonstrated, where multicopy transgene arrays of a particular gene have been
                           impossible to generate, but integrated single-copy transgenes have proved viable and
                           functional. One example is <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rom-1" target="_blank">rom-1</a></em></span>, encoding the rhomboid protease (<a href="#bib7">Dutt et al., 2004</a>); another is the collagen gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-17" target="_blank">dpy-17</a></em></span> (Jacopo Novelli and J. H., unpublished observations).
            </p>
            <p>Genes encoding proteins that are needed in stoichiometric amounts, as part of a
                           multi-protein complex, might be expected to show strong dosage effects, but these genes often exhibit a specific
               auto-regulatory mechanism, which ensures that
                           protein levels do not go too high or too low. An example is provided by some of the
                           ribosomal proteins, such as RPL-12, which appears able to inhibit productive splicing of
                           its own pre-mRNA, as a negative feedback control (<a href="#bib22">Mitrovich and Anderson, 2000</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e823"></a>9.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                             <span class="bibliomisc">Ahmed, S., and Hodgkin, J. (2000). MRT-2 checkpoint protein is required for germline immortality and telomere replication
                        in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>403</em></span>, 159&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10646593&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35003120" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                             <span class="bibliomisc">Albertson, D.G., and Thomson, J.N. (1982). The kinetochores of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Chromosoma <span class="emphasis"><em>86</em></span>, 409&#8211;428.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7172865&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00292267" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                             <span class="bibliomisc">Albertson, D.G., and Thomson, J.N. (1993). Segregation of holocentric chromosomes at meiosis in the nematode, <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Chromosome Res., 15&#8211;26.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8143084&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00710603" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                             <span class="bibliomisc">Barnes, T.M., Kohara, Y., Coulson, A., and Hekimi, S. (1995). Meiotic recombination, noncoding DNA and genomic organization
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>141</em></span>, 159&#8211;179.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8536965&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                             <span class="bibliomisc">Brenner, S. (1974). The genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>77</em></span>, 71&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4366476&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                             <span class="bibliomisc"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium (1998). Genome sequence of the nematode <span class="emphasis"><em>C. elegans</em></span>: a platform for investigating biology. Science <span class="emphasis"><em>282</em></span>, 2012&#8211;2018.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851916&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.282.5396.2012" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                             <span class="bibliomisc">Dutt, A., Canevascini, S., Froehli-Hoier, E., and Hajnal, A. (2004). EGF signal propagation during <span class="emphasis"><em>C. elegans</em></span> vulval development mediated by ROM-1 rhomboid. PLoS Biol. <span class="emphasis"><em>2</em></span>, e334.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15455032&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0020334" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                             <span class="bibliomisc">Haack, H., and Hodgkin, J. (1991). Tests for parental imprinting in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Gen. Genet. <span class="emphasis"><em>228</em></span>, 482&#8211;485.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1896016&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00260643" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hillers, K.J., and Villeneuve, A.M. (2003). Chromosome-wide control of meiotic crossing over in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>13</em></span>, 1641&#8211;1647.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=13678597&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2003.08.026" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hodgkin, J. (1983). X chromosome dosage and gene expression in <span class="emphasis"><em>C. elegans</em></span>: Two unusual dumpy genes. Mol. Gen. Genet. <span class="emphasis"><em>192</em></span>, 452&#8211;458.</span>
                     <a href="http://dx.doi.org/10.1007/BF00392190" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hodgkin, J. (1986). Sex determination in the nematode <span class="emphasis"><em>C. elegans</em></span>: analysis of <span class="emphasis"><em>tra-3</em></span> suppressors and characterization of <span class="emphasis"><em>fem</em></span> genes. Genetics <span class="emphasis"><em>114</em></span>, 15&#8211;52.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3770465&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hodgkin, J. (1987). Primary sex determination in the nematode <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>101 (Suppl.)</em></span>, 5&#8211;16.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3503722&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hodgkin, J., and Albertson, D.G. (1995). Isolation of dominant XO-feminizing mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: new regulatory <span class="emphasis"><em>tra</em></span> alleles and an X chromosome duplication with implications for primary sex determination. Genetics <span class="emphasis"><em>141</em></span>, 527&#8211;542.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8647390&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hodgkin, J., Horvitz, H.R., and Brenner, S. (1979). Nondisjunction mutants of the nematode <span class="emphasis"><em>C. elegans</em></span>. Genetics <span class="emphasis"><em>91</em></span>, 67&#8211;94.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hutter, H.,Vogel, B.E., Plenefisch, J.D., Norris, C.R., Proenca, R.B., Spieth, J., Guo, C., Mastwal, S., Zhu, X., Scheel,
                        J., et al. (2000). Conservation and novelty in the evolution of cell adhesion and extracellular matrix genes. Science <span class="emphasis"><em>287</em></span>, 989&#8211;994.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10669422&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.287.5455.989" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kacser, H., and Burns, J.A. (1981). The molecular basis of dominance. Genetics <span class="emphasis"><em>97</em></span>, 639&#8211;666.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7297851&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kamath, R.S., Fraser, A.G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., Kanapin, A., Le Bot, N., Moreno, S., Sohrmann, M.,
                        et al. (2003). Systematic functional analysis of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome using RNAi. Nature <span class="emphasis"><em>421</em></span>, 231&#8211;237.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12529635&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature01278" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kelly, W.G., Schaner, C.E., Dernburg, A.F., Lee, M.H., Kim, S.K., Villeneuve, A.M., and Reinke, V. (2002). X-chromosome silencing
                        in the germline of <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>129</em></span>, 479&#8211;492.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11807039&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                             <span class="bibliomisc">Madl, J.E, and Herman, R.K. (1979). Polyploids and sex determination in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>93</em></span>, 393&#8211;402.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=295035&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                             <span class="bibliomisc">Matthews, L.R., Carter, P., Thierry-Mieg, D., and Kemphues, K. (1998). ZYG-9, a <span class="emphasis"><em>Caenorhabditis elegans</em></span> protein required for microtubule organization and function, is a component of meiotic and mitotic spindle poles. J. Cell
                        Biol. <span class="emphasis"><em>141</em></span>, 1159&#8211;1168.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9606208&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.141.5.1159" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                             <span class="bibliomisc">Miller, M.A., Cutter, A.D., Yamamoto, I., Ward, S., and Greenstein, D. (2004). Clustered organization of reproductive genes
                        in the <span class="emphasis"><em>C. elegans</em></span> genome. Curr. Biol. <span class="emphasis"><em>14</em></span>, 1284&#8211;1290.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15268860&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.07.025" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                             <span class="bibliomisc">Mitrovich, Q.M., and Anderson, P. (2000). Unproductively spliced ribosomal protein mRNAs are natural targets of mRNA surveillance
                        in <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>14</em></span>, 2173&#8211;2184.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10970881&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.819900" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                             <span class="bibliomisc">Nigon, V. (1951a). Polyploidie experimentale chez un nematode libre, Rhabditis elegans Maupas. Bulletin Biologique de la France
                        et de la Belgique <span class="emphasis"><em>95</em></span>, 187&#8211;225.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                             <span class="bibliomisc">Nigon, V. (1951b). La gametogenese d'un nematode tetraploide obtenu par voie experimentale. Bull Soc. Hist. Natl. Toulouse
                        <span class="emphasis"><em>86</em></span>, 195&#8211;200.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                             <span class="bibliomisc">Nigon, V. (1949a). Les modalites de la reproduction et le determinisme du sexe chez quelques nematodes libres. Annales de
                        Sciences Naturelles - Zool. Biol. Anim. <span class="emphasis"><em>11</em></span>, 1&#8211;132.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                             <span class="bibliomisc">Nigon, V. (1949b). Effects de la polyploidie chez un nematode libre. C.R. des Seances de l'Academie des Sciences Serie D 228,
                        1161-1162</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                             <span class="bibliomisc">Park, E-C., and Horvitz, H.R. (1986). Mutations with dominant effects on the behavior and morphology of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>113</em></span>, 821&#8211;852.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3744028&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                             <span class="bibliomisc">Piano, F., Schetter, A.J., Mangone, M., Stein, L., and Kemphues, K.J. (2000). RNAi analysis of genes expressed in the ovary
                        of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Curr. Biol. <span class="emphasis"><em>10</em></span>, 1619&#8211;1622.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11137018&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(00)00869-1" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                             <span class="bibliomisc">Piano, F., Schetter, A.J., Morton, D.G., Gunsalus, K.C., Reinke, V., Kim, S.K., and Kemphues, K.J. (2002). Gene clustering
                        based on RNAi phenotypes of ovary-enriched genes in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>12</em></span>, 1959&#8211;1964.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12445391&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)01301-5" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                             <span class="bibliomisc">Reinke, V., Smith, H.E., Nance, J., Wang, J., Van Doren, C., Begley, R., Jones, S.J., Davis, E.B., Scherer, S., Ward, S.,
                        et al. (2000). A global profile of germline gene expression in <span class="emphasis"><em>C. elegans</em></span>. Mol. Cell <span class="emphasis"><em>6</em></span>, 605&#8211;616.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11030340&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)00059-9" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                             <span class="bibliomisc">Schierenberg, E., and Wood, W.B. (1985). Control of cell-cycle timing in early embryos of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>107</em></span>, 337&#8211;354.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3972159&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(85)90316-1" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sigurdson, D.C., Herman, R.K., Horton, C.A., Kari, C.K., and Pratt S.E. (1986). An X-autosome fusion chromosome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Gen. Genet. <span class="emphasis"><em>202</em></span>, 212&#8211;228.</span>
                     <a href="http://dx.doi.org/10.1007/BF00331639" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sigurdson, D.C., Spanier, G.J., and Herman, R.K. (1984). <span class="emphasis"><em>Caenorhabditis elegans</em></span> deficiency mapping. Genetics <span class="emphasis"><em>108</em></span>, 331&#8211;345.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6500256&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                             <span class="bibliomisc">Stinchcomb, D.T., Shaw, J.E., Carr, S.H., and Hirsh, D. (1985). Extrachromosomal DNA transformation of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Cell Biol. <span class="emphasis"><em>5</em></span>, 3484&#8211;3496.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3837845&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                             <span class="bibliomisc">Trent, C., Tsung, N., and Horvitz, H.R. (1983). Egg-laying defective mutants of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>104</em></span>, 619&#8211;647.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11813735&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                             <span class="bibliomisc">Wicky, C., Villeneuve A.M., Lauper, N., Codourey, L., Tobler, H., and Muller, F. (1996). Telomeric repeats (TTAGGC)n are sufficient
                        for chromosome capping function in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>93</em></span>, 8983&#8211;8988.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8799140&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.93.17.8983" target="_blank">Article</a>
                         
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e10" href="#d0e10">*</a></sup>Edited by Philip Anderson. Last revised July 18, 2005. Published June 25, 2005. This chapter should be cited as: Hodgkin,
                  J., Karyotype, ploidy and gene dosage (June 25, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.3.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2005 Jonathan Hodgkin. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:jah@bioch.ox.ac.uk" target="_top">jah@bioch.ox.ac.uk</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

