<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>The putative chemoreceptor families of C. elegans
         
         
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#chemorecep_ArticleID" title="The putative chemoreceptor families of C. elegans&#xA;            &#xA;        ">
      <link rel="next" href="#d0e156" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="putativechemoreceptorfam.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="chemorecep_ArticleID"></a>The putative chemoreceptor families of <span class="emphasis"><em>C. elegans</em></span>
                                 <sup><a name="d0e11" href="#ftn.d0e11">*</a></sup>
                             
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Hugh M.</span> <span class="surname">Robertson<em><span class="remark">
                                                     <sup><a name="d0e41" href="#ftn.d0e41">&sect;</a></sup>
                                                 </span></em>
                                       </span><br></span><span class="affiliation"><span class="address">Department of Entomology, University of Illinois at Urbana-Champaign, Urbana, IL 61801 USA
                           <p></p></span></span></div>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">James H.</span> <span class="surname">Thomas<em><span class="remark">
                                                     <sup><a name="d0e41" href="#ftn.d0e41">&sect;</a></sup>
                                                 </span></em></span><br></span><span class="affiliation"><span class="address">Department of Genome Sciences, University of Washington, Seattle, WA 98195-7360 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_signaltrans.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Signal transduction
		</a></div><hr />

<a href="http://www.wormbase.org/forums/index.php?board=236.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=The+putative+chemoreceptor+families+of+C.+elegans" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e156">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#d0e293">2. Summary of the chemoreceptor families</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e688">2.1. The Str superfamily</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e804">2.2. The Sra superfamily</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e887">2.3. The Srg superfamily</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1131">2.4. The <span class="emphasis"><em>srw </em></span>family</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e1212">3. Gene expansions</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1308">4. Gene clusters</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1362">5. Function and expression patterns</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1462">6. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1499">7. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>Chemoreception of environmental stimuli is a major sensory system in small soil
                                 nematodes like</strong></span> <span class="bold-italic">C. elegans.</span> <span class="bold"><strong>As in other animals, chemoreception is
                                 mediated in</strong></span> <span class="bold-italic">C. elegans</span> <span class="bold"><strong>by members of the seven-transmembrane
                                 G-protein-coupled receptor class (7TM GPCRs). We summarize the many large putative
                                 chemoreceptor gene families, including the</strong></span> <span class="emphasis"><em><span class="bold"><strong>str</strong></span></em></span> <span class="bold"><strong>family (which includes</strong></span>
                           <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=odr-10" target="_blank">odr-10</a>,</strong></span></em></span> <span class="bold"><strong>the only receptor with an identified ligand), and the</strong></span>
                           <span class="emphasis"><em><span class="bold"><strong>sra, srab, srb, srbc, srd, sre, srg, srh, sri, srj, srm, srr, srsx, srt, sru</strong></span>,</em></span>
                           <span class="emphasis"><em><span class="bold"><strong>srv, srw, srx, srxa,</strong></span></em></span> <span class="bold"><strong>and</strong></span> <span class="emphasis"><em><span class="bold"><strong>srz</strong></span></em></span> <span class="bold"><strong>families. Together these comprise <span class="symbol">&plusmn;</span>1280 apparently intact genes and <span class="symbol">&plusmn;</span>420 apparent pseudogenes, about 7% of the total
                                 gene count of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans.</strong></span></em></span> <span class="bold"><strong>These genes are unusually clustered on
                                 chromosomes, both within and between families, and are enigmatically concentrated on the
                                 large chromosome V. Comparative studies with</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. briggsae</strong></span></em></span> <span class="bold"><strong>have
                                 revealed extraordinary divergence of the chemoreceptor repertoire between the two
                                 species, including frequent amplifications of subfamilies in</strong></span> <span class="emphasis"><em><span class="bold"><strong>C.
                                       elegans</strong></span></em></span> <span class="bold"><strong>and positive selection in the</strong></span> <span class="emphasis"><em><span class="bold"><strong>srz</strong></span></em></span> <span class="bold"><strong>family. The
                                 size and complexity of the chemoreceptor gene families also facilitate studies of
                                 promoter elements using paralogous and orthologous comparisons, as well as other aspects
                                 of gene family and genome evolution.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e156"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>Chemoperception is a central sense of soil nematodes like <span class="emphasis"><em>Caenorhabditis
                                 elegans</em></span>, as it is for most other animals (reviewed in <a href="#bib28">Mori, 1999</a>; <a href="#bib36">Sengupta, 1997</a>; <a href="#bib3">Bargmann, 1997</a>). Following the
                           Nobel-Prize-winning discovery of olfactory receptors (ORs) in mammals (<a href="#bib6">Buck and Axel,
                              1991</a>) as members of the rhodopsin superfamily of G-protein-coupled
                           receptors (GPCRs) with seven transmembrane domains (7TM), Troemel et al. identified a
                           series of disparate GPCRs as candidate olfactory receptors in <span class="emphasis"><em>C.
                                 elegans</em></span> (<a href="#bib43">Troemel et al., 1995</a>). These identifications were made
                           from some of the earliest sequences reported by the genome project and proved to be the
                           tip of the iceberg. The genes were classified into small gene families and given names
                           starting with sr for serpentine receptor (<span class="emphasis"><em>sra</em></span>,
                           <span class="emphasis"><em>srb</em></span>, <span class="emphasis"><em>srd</em></span>, <span class="emphasis"><em>sre</em></span>, and
                           <span class="emphasis"><em>sro</em></span>). Their probable chemosensory function was based on transgene
                           expression patterns in one or more known pairs of chemosensory neurons. Subsequently,
                           Sengupta et al. used a genetic screen to identify <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=odr-10" target="_blank">odr-10</a></em></span>, which
                           they showed encodes a GPCR that mediates olfactory response to diacetyl (<a href="#bib37">Sengupta et
                              al., 1996</a>). With completion of the genome sequence it became possible to
                           flesh out the true extent of these and related gene families encoding GPCRs that are
                           candidate chemoreceptors. Robertson (<a href="#bib31">Robertson, 1998</a>; <a href="#bib33">Robertson, 2001</a>) worked up the relatives of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=odr-10" target="_blank">odr-10</a></em></span> as the large <span class="emphasis"><em>str</em></span>
                           family along with the related <span class="emphasis"><em>sri</em></span> and <span class="emphasis"><em>srj</em></span>
                           families, and then also worked up the large related <span class="emphasis"><em>srh</em></span> family (<a href="#bib32">Robertson, 2000</a>). Annotation and naming of the smaller <span class="emphasis"><em>sra</em></span> family was
                           completed by HMR as part of the comparison with <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib38">Stein
                              et al., 2003</a>), while Chen et al. described the related
                           <span class="emphasis"><em>srab</em></span> family (<a href="#bib7">Chen et al., 2005</a>), and Thomas et al.
                           described the <span class="emphasis"><em>srz</em></span> family (<a href="#bib42">Thomas et al., 2005</a>) and
                           the <span class="emphasis"><em>srt</em></span> family (<a href="#bib46">Thomas, 2006</a> [in press]). A decade after their first recognition, we describe here
                           all the remaining families and relate them in a series of superfamilies that comprise a
                           remarkable array of roughly 1300 potentially functional candidate chemoreceptor genes,
                           along with about 400 apparent pseudogenes. Together they comprise roughly 7% of the gene
                           complement of this tiny animal, more than any other sequenced genome. This large genetic
                           investment might result from an extreme dependence on chemosensory abilities in the
                           absence of visual and auditory systems. On the basis of genetic complexity, this system
                           probably constitutes the bulk of signal transduction that occurs in <span class="emphasis"><em>C. elegans</em></span>.
            </p>
            <p>We first discuss these families below in superfamily groupings, loosely defined as
                           consisting of readily recognizable groupings based on protein sequence similarity, as
                           well as sometimes shared intron locations in their genes. We then discuss various
                           aspects of their biology, including new expression pattern results that support their
                           likely function as chemoreceptors. For the rest of this chapter, while fully
                           acknowledging the formal status of all but <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=odr-10" target="_blank">odr-10</a></em></span> as putative or
                           candidate chemoreceptors, we will refer to them as chemoreceptors. Genetic analysis of
                           sensory-neuron specific G proteins indicates that olfaction, nociception, and pheromone
                           responses are mediated predominantly by GPCRs (e.g., <a href="#bib16">Hilliard et al., 2004</a>;
                           <a href="#bib17">Jansen et al., 1999</a>; <a href="#bib18">Jansen et al., 2002</a>; <a href="#bib30">Roayaie et al.,
                              1998</a>) and it is difficult to see what else these huge families might
                           generally mediate. Nevertheless, it seems likely that some of the genes have been
                           subsequently recruited to other roles such as monitoring of internal chemistry, so the
                           function of any specific gene should be interpreted with caution.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e293"></a>2.&nbsp;Summary of the chemoreceptor families
                     </h2>
                  </div>
               </div>
            </div>
            <p><a href="#table1" title="Table 1. Gene and pseudogene numbers in chemoreceptor families and superfamilies">Table 1</a> lists the named chemoreceptor families by superfamily group with the numbers
                           of apparently intact genes and of apparent pseudogenes in each. Annotation of the genes
                           is not nearly as straightforward as for mammalian olfactory receptors, because the
                               <span class="emphasis"><em>C. elegans</em></span> genes have complex gene structures with 1&#8211;8 introns
                           each, and the intron locations are commonly different between gene families. EST data
                           are of little assistance due to uniformly low expression levels (e.g., of ~160,000 ESTs
                           on WS138, only 81 were from chemoreceptor genes). In the absence of functional
                           information it is not possible to be confident of the functionality of any specific
                           gene, but each has been annotated to encode a seemingly intact full-length protein that
                           aligns well with other family members. It is always possible that particular amino acid
                           changes or small in-frame insertions or deletions might inactivate an otherwise
                           apparently intact receptor. On the other hand, amongst the apparent pseudogenes from N2
                           is a subset with probable functional alleles in other wild strains of <span class="emphasis"><em>C.
                                 elegans</em></span> (<a href="#bib39">Stewart et al., 2005</a>). In part because of this we have
                           chosen to name both genes and pseudogenes in these families. We have arbitrarily chosen
                           to ignore gene fragments that encode less than 50% of the amino acid length typical for
                           a gene family. The curated annotations and gene names for all families should be correct
                           in WormBase release WS141, though ongoing studies will undoubtedly refine their structure.
            </p>
            <div class="table"><a name="table1"></a><p class="title">Table 1. Gene and pseudogene numbers in chemoreceptor families and superfamilies</p>
               <table summary="Table 1. Gene and pseudogene numbers in chemoreceptor families and superfamilies" border="1">
                  <colgroup>
                     <col align="left">
                     <col align="left">
                     <col align="left">
                     <col align="left">
                     <col align="left">
                     <col align="left">
                  </colgroup>
                  <thead>
                     <tr>
                        <th align="left">Superfamily</th>
                        <th align="left">Family</th>
                        <th align="left">Genes</th>
                        <th align="left">Pseudogenes</th>
                        <th align="left">% Pseudo</th>
                        <th align="left">Total loci</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td align="left">Str</td>
                        <td align="left"><span class="emphasis"><em>srh</em></span></td>
                        <td align="left">217</td>
                        <td align="left">91</td>
                        <td align="left">30</td>
                        <td align="left">308</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>str</em></span></td>
                        <td align="left">192</td>
                        <td align="left">75</td>
                        <td align="left">28</td>
                        <td align="left">267</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>sri</em></span></td>
                        <td align="left">60</td>
                        <td align="left">18</td>
                        <td align="left">23</td>
                        <td align="left">78</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srd</em></span></td>
                        <td align="left">64</td>
                        <td align="left">12</td>
                        <td align="left">16</td>
                        <td align="left">76</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srj</em></span></td>
                        <td align="left">39</td>
                        <td align="left">18</td>
                        <td align="left">32</td>
                        <td align="left">57</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srm</em></span></td>
                        <td align="left">5</td>
                        <td align="left">1</td>
                        <td align="left">17</td>
                        <td align="left">6</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srn</em></span></td>
                        <td align="left">1</td>
                        <td align="left">0</td>
                        <td align="left">0</td>
                        <td align="left">1</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left">all Str</td>
                        <td align="left">578</td>
                        <td align="left">215</td>
                        <td align="left">27</td>
                        <td align="left">793</td>
                     </tr>
                     <tr>
                        <td align="left">Sra</td>
                        <td align="left"><span class="emphasis"><em>sre</em></span></td>
                        <td align="left">51</td>
                        <td align="left">5</td>
                        <td align="left">9</td>
                        <td align="left">56</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>sra</em></span></td>
                        <td align="left">32</td>
                        <td align="left">7</td>
                        <td align="left">18</td>
                        <td align="left">39</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srab</em></span></td>
                        <td align="left">22</td>
                        <td align="left">5</td>
                        <td align="left">19</td>
                        <td align="left">27</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srb</em></span></td>
                        <td align="left">14</td>
                        <td align="left">5</td>
                        <td align="left">26</td>
                        <td align="left">19</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left">all Sra</td>
                        <td align="left">119</td>
                        <td align="left">22</td>
                        <td align="left">16</td>
                        <td align="left">141</td>
                     </tr>
                     <tr>
                        <td align="left">Srg</td>
                        <td align="left"><span class="emphasis"><em>srx</em></span></td>
                        <td align="left">94</td>
                        <td align="left">44</td>
                        <td align="left">32</td>
                        <td align="left">138</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srt</em></span></td>
                        <td align="left">59</td>
                        <td align="left">15</td>
                        <td align="left">20</td>
                        <td align="left">74</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srg</em></span></td>
                        <td align="left">59</td>
                        <td align="left">9</td>
                        <td align="left">13</td>
                        <td align="left">68</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>sru</em></span></td>
                        <td align="left">39</td>
                        <td align="left">9</td>
                        <td align="left">19</td>
                        <td align="left">48</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srv</em></span></td>
                        <td align="left">30</td>
                        <td align="left">6</td>
                        <td align="left">17</td>
                        <td align="left">36</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left"><span class="emphasis"><em>srxa</em></span></td>
                        <td align="left">17</td>
                        <td align="left">2</td>
                        <td align="left">11</td>
                        <td align="left">19</td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left">all Srg</td>
                        <td align="left">298</td>
                        <td align="left">85</td>
                        <td align="left">22</td>
                        <td align="left">383</td>
                     </tr>
                     <tr>
                        <td align="left">Solo</td>
                        <td align="left"><span class="emphasis"><em>srw</em></span></td>
                        <td align="left">100</td>
                        <td align="left">45</td>
                        <td align="left">31</td>
                        <td align="left">145</td>
                     </tr>
                     <tr>
                        <td align="left">Solo</td>
                        <td align="left"><span class="emphasis"><em>srz</em></span></td>
                        <td align="left">64</td>
                        <td align="left">39</td>
                        <td align="left">38</td>
                        <td align="left">103</td>
                     </tr>
                     <tr>
                        <td align="left">Solo</td>
                        <td align="left"><span class="emphasis"><em>srbc</em></span></td>
                        <td align="left">71</td>
                        <td align="left">13</td>
                        <td align="left">15</td>
                        <td align="left">84</td>
                     </tr>
                     <tr>
                        <td align="left">Solo</td>
                        <td align="left"><span class="emphasis"><em>srsx</em></span></td>
                        <td align="left">37</td>
                        <td align="left">3</td>
                        <td align="left">8</td>
                        <td align="left">40</td>
                     </tr>
                     <tr>
                        <td align="left">Solo</td>
                        <td align="left"><span class="emphasis"><em>srr</em></span></td>
                        <td align="left">9</td>
                        <td align="left">1</td>
                        <td align="left">10</td>
                        <td align="left">10</td>
                     </tr>
                     <tr>
                        <td align="left">All family totals:</td>
                        <td align="left">&nbsp;</td>
                        <td align="left">1276</td>
                        <td align="left">423</td>
                        <td align="left">25</td>
                        <td align="left">1699</td>
                     </tr>
                     <tr>
                        <td colspan="6" align="left">Pseudogenes and functional genes are distinguished by the presence of apparent defects in coding potential (mostly deletions,
                           stop codons, and frameshifts) and are occasionally subjective; these numbers should be taken as current best estimates. The
                           <span class="emphasis"><em>srbc</em></span> and <span class="emphasis"><em>srsx</em></span> families are probably distantly related to each other and perhaps to the <span class="emphasis"><em>srx</em></span> family (JHT, unpublished), but these relationship are not yet sufficiently clear to justify inclusion in the table. An additional
                           ~150 seven-pass receptors may also function as chemoreceptors, but they are either in very small families or are unique in
                           the genome and their status is unclear.
                        </td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e688"></a>2.1.&nbsp;The Str superfamily
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The Str superfamily consists of the <span class="emphasis"><em>str</em></span>,
                                  <span class="emphasis"><em>srd</em></span>, <span class="emphasis"><em>srh</em></span>, <span class="emphasis"><em>sri</em></span>,
                                  <span class="emphasis"><em>srj</em></span>, <span class="emphasis"><em>srm</em></span>, and <span class="emphasis"><em>srn</em></span>
                                  genes, with a total of 795 genes and pseudogenes. The superfamily includes the <span class="emphasis"><em>str</em></span>
                                  and <span class="emphasis"><em>srh</em></span> families, which are the two largest gene families in <span class="emphasis"><em>C.
                                        elegans</em></span>. Robertson was able to extract considerable information about
                                  gene family, intron, and chromosomal evolution from studies of these large families
                                  (<a href="#bib31">Robertson, 1998</a>; <a href="#bib32">Robertson, 2000</a>; <a href="#bib33">Robertson, 2001</a>). Similar analysis has subsequently been extended to all of the other
                                  families. The high frequency of pseudogenes in these families was novel at the time
                                  for <span class="emphasis"><em>C. elegans</em></span>, but is now more commonly recognized in the
                                  genome (e.g., <a href="#bib29">Mounsey et al., 2002</a>). Rampant gain and loss of introns
                                  was inferred from phylogenetic analysis and has been confirmed by genome-wide and
                                  comparative analyses with <span class="emphasis"><em>C. briggsae</em></span> (e.g., <a href="#bib10">Coghlan and Wolfe,
                                     2004</a>; <a href="#bib22">Kiontke et al., 2004</a>). Processes of genome flux
                                  have moved these genes around a lot, although somewhat mysteriously retaining most
                                  of them on the large chromosome V, which might reflect functional clustering with
                                  each other and other chemoreceptor genes, similar to that inferred for several other
                                  gene classes (e.g., <a href="#bib27">Miller et al., 2004</a>). Comparisons across the entire
                                  superfamily show that the srd family is a highly divergent and basal family (<a href="#figure1">Figure 1</a>), with just one intron location ancestrally shared in the family
                                  and across the entire superfamily. A few genes and pseudogenes have been added to
                                  the larger of these families since their original description with completion of the
                                  genome sequence.
               </p>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="540">
<tr>
  <td align="center"><p><a href="chemorecepfig1.jpg"><img src="chemorecepfig1_s.jpg" border="2" align="middle" alt="figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. Phylogenetic tree of the <span class="emphasis"><em>srd</em></span> family, rooted at the
                                       midpoint.&nbsp;</b>Receptor amino acid sequences were aligned using CLUSTALX (<a href="#bib8">Chenna et al., 2003</a>) after refinement of gene annotations. Phylogenetic analysis was performed
                                    using the minimum distance heuristic algorithm in PAUP<sup>
                                           <span class="symbol">&#8727;</span>
                                       </sup> v4.0b10 (<a href="#bib41">Swofford, 1998</a>), with distances corrected for multiple changes
                                    using the maximum likelihood model in TREE-PUZZLE v5 (<a href="#bib34">Schmidt et al., 2002</a>) with the BLOSUM62 matrix (<a href="#bib15">Henikoff and Henikoff, 1992</a>).
                                    Pseudogenes are indicated by the following symbols after their cosmid gene names
                                        (<span class="symbol">#</span> - frameshifting or large indel or
                                    truncation; <sup>
                                           <span class="symbol">&#8727;</span>
                                       </sup> - in-frame stop codon; ? -unacceptable intron boundary or missing start
                                    codon). Right click or control-click for larger image.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e804"></a>2.2.&nbsp;The Sra superfamily
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The Sra superfamily consists of the <span class="emphasis"><em>sra</em></span>,
                                  <span class="emphasis"><em>srab</em></span>, <span class="emphasis"><em>srb</em></span>, and <span class="emphasis"><em>sre</em></span>
                                  families, totaling 141 genes and pseudogenes. Comparison of the
                                  <span class="emphasis"><em>sra</em></span> and <span class="emphasis"><em>srab</em></span> families with their homologs
                                  in <span class="emphasis"><em>C. briggsae</em></span> revealed that these families are considerably
                                  expanded in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib7">Chen et al., 2005</a>; <a href="#bib38">Stein
                                     et al., 2003</a>), something that appears to be true for many of the
                                  chemoreceptor gene families. Using computational gene-building methods, Chen et al.
                                  (<a href="#bib7">Chen et al., 2005</a>) did not note any pseudogenes in the
                                  <span class="emphasis"><em>srab</em></span> family in <span class="emphasis"><em>C. elegans</em></span>, although they
                                  did note them in the <span class="emphasis"><em>srab</em></span> family in <span class="emphasis"><em>C.
                                        briggsae</em></span> and the <span class="emphasis"><em>sra</em></span> family in both species. Chen
                                  and HMR have reevaluated this family in <span class="emphasis"><em>C. elegans</em></span>, and now
                                  recognize four pseudogenes and added one gene. Phylogenetic analysis of the
                                  superfamily reveals that the <span class="emphasis"><em>sre</em></span> family is the most divergent,
                                  with the <span class="emphasis"><em>sra</em></span>, <span class="emphasis"><em>srab</em></span>, and
                                  <span class="emphasis"><em>srb</em></span> families being clustered together. The last three named
                                  members of the <span class="emphasis"><em>sra</em></span> family (<span class="emphasis"><em>sra</em></span>-37-39,
                                  T21H8.2-4) cluster with the <span class="emphasis"><em>srb</em></span> family in some analyses, but
                                  are best left in the <span class="emphasis"><em>sra</em></span> family with this caveat.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e887"></a>2.3.&nbsp;The Srg superfamily
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The Srg superfamily consists of the <span class="emphasis"><em>srg</em></span>,
                                  <span class="emphasis"><em>srt</em></span>, <span class="emphasis"><em>sru</em></span>, <span class="emphasis"><em>srv</em></span>,
                                  <span class="emphasis"><em>srx</em></span>, and <span class="emphasis"><em>srxa</em></span> families, totaling 310 genes
                                  and pseudogenes, with the latter being the most divergent. Evolutionary analysis of
                                  these families has been limited, and virtually nothing is known about their specific
                                  functions, though expression patterns are consistent with a role in chemosensation
                                  (<a href="#table2" title="Table 2. Summaries of known GFP-fusion expression patterns for chemoreceptors">Table 2</a>).
               </p>
               <div class="table"><a name="table2"></a><p class="title">Table 2. Summaries of known GFP-fusion expression patterns for chemoreceptors</p>
                  <table summary="Table 2. Summaries of known GFP-fusion expression patterns for chemoreceptors" border="1">
                     <colgroup>
                        <col align="left">
                        <col align="left">
                        <col align="left">
                        <col align="left">
                        <col align="left">
                        <col align="left">
                        <col align="left">
                        <col align="left">
                        <col align="left">
                     </colgroup>
                     <thead>
                        <tr>
                           <th align="left">Family</th>
                           <th align="left">Expressed</th>
                           <th align="left">Unexpr- essed</th>
                           <th align="left">Sensory</th>
                           <th align="left">Amphid</th>
                           <th align="left">Phasmid</th>
                           <th align="left">Other sensory</th>
                           <th align="left">Other neurons</th>
                           <th align="left">Non- neuronal</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="left"><span class="emphasis"><em>sra</em></span></td>
                           <td align="left">10</td>
                           <td align="left">3</td>
                           <td align="left">7/10</td>
                           <td align="left">5</td>
                           <td align="left">1</td>
                           <td align="left">3</td>
                           <td align="left">4</td>
                           <td align="left">2</td>
                        </tr>
                        <tr>
                           <td align="left"><span class="emphasis"><em>srab</em></span></td>
                           <td align="left">5</td>
                           <td align="left">0</td>
                           <td align="left">5/5</td>
                           <td align="left">3</td>
                           <td align="left">1</td>
                           <td align="left">1</td>
                           <td align="left">1</td>
                           <td align="left">0</td>
                        </tr>
                        <tr>
                           <td align="left"><span class="emphasis"><em>srd</em></span></td>
                           <td align="left">5</td>
                           <td align="left">4</td>
                           <td align="left">4/5</td>
                           <td align="left">4</td>
                           <td align="left">1</td>
                           <td align="left">1</td>
                           <td align="left">1</td>
                           <td align="left">0</td>
                        </tr>
                        <tr>
                           <td align="left"><span class="emphasis"><em>srg</em></span></td>
                           <td align="left">4</td>
                           <td align="left">0</td>
                           <td align="left">3/4</td>
                           <td align="left">2</td>
                           <td align="left">1</td>
                           <td align="left">0</td>
                           <td align="left">0</td>
                           <td align="left">1</td>
                        </tr>
                        <tr>
                           <td align="left"><span class="emphasis"><em>srh/sri</em></span></td>
                           <td align="left">15</td>
                           <td align="left">1</td>
                           <td align="left">12/15</td>
                           <td align="left">12</td>
                           <td align="left">1</td>
                           <td align="left">3</td>
                           <td align="left">3</td>
                           <td align="left">8</td>
                        </tr>
                        <tr>
                           <td align="left"><span class="emphasis"><em>str/srj</em></span></td>
                           <td align="left">16</td>
                           <td align="left">1</td>
                           <td align="left">10/16</td>
                           <td align="left">10</td>
                           <td align="left">0</td>
                           <td align="left">0</td>
                           <td align="left">7</td>
                           <td align="left">2</td>
                        </tr>
                        <tr>
                           <td align="left">other</td>
                           <td align="left">8</td>
                           <td align="left">5</td>
                           <td align="left">7/8</td>
                           <td align="left">7</td>
                           <td align="left">1</td>
                           <td align="left">1</td>
                           <td align="left">2</td>
                           <td align="left">1</td>
                        </tr>
                        <tr>
                           <td align="left">total</td>
                           <td align="left">63</td>
                           <td align="left">14</td>
                           <td align="left">48/63</td>
                           <td align="left">43</td>
                           <td align="left">6</td>
                           <td align="left">9</td>
                           <td align="left">18</td>
                           <td align="left">14</td>
                        </tr>
                        <tr>
                           <td colspan="9" align="left">Some genes are expressed in more than one listed tissue. Data were gathered from <a href="#bib5">Battu et al., 2003</a>; <a href="#bib7">Chen et al., 2005</a>; <a href="#bib24">McCarroll et al., 2005</a>; <a href="#bib37">Sengupta et al., 1996</a>; <a href="#bib43">Troemel et al., 1995</a>; <a href="#bib44">Wenick and Hobert, 2004</a>, and generous personal communications from Cori Bargmann and Piali Sengupta.
                           </td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1131"></a>2.4.&nbsp;The <span class="emphasis"><em>srw </em></span>family
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The <span class="emphasis"><em>srw</em></span> family of 145 genes and pseudogenes is unusual among
                                  the candidate chemoreceptors because its origins from another family of otherwise
                                  conserved GPCRs can be recognized. This family of highly divergent genes and many
                                  pseudogenes is related to families of apparent FMRFamide and other peptide
                                  receptors, which have relatives in vertebrates and insects (e.g., <a href="#bib13">Egerod et al.,
                                     2003</a>; <a href="#bib25">Meeusen et al., 2002</a>; <a href="#bib26">Mertens et al., 2004</a>; <a href="#figure2">Figure 2</a>). The candidate FMRFamide/peptide receptors
                                  are dispersed in the genome, but almost 90% of <span class="emphasis"><em>srw</em></span> genes are on
                                  chromosome V, most in large clusters on the chromosomal arms (<a href="#figure3">Figure 3</a>B). Given the similarities of their gene family evolution to that
                                  of the other families, and their common genomic organization, we hypothesize that
                                  the <span class="emphasis"><em>srw</em></span> family represents a novel origin of chemoreceptors within the nematode
                                  lineage, perhaps serving as environmental peptide receptors.
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="585">
<tr>
  <td align="center"><p><a href="chemorecepfig2.jpg"><img src="chemorecepfig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Phylogenetic tree of the srw family, rooted with a group of FMRF-amide and
                                       other peptide receptors from nematodes and insects.&nbsp;</b>Large arrows indicate a <span class="emphasis"><em>Drosophila</em></span> FRMFamide receptor (<a href="#bib25">Meeusen et al., 2002</a>), a <span class="emphasis"><em>Drosophila</em></span> myosuppressin receptor (<a href="#bib13">Egerod et al., 2003</a>),
                                    and one of the three <span class="emphasis"><em>C. elegans</em></span> FMRFamide receptors (<a href="#bib26">Mertens et al., 2004</a>). See Figure 1 legend for details of methods and symbols. Right click or control-click for larger image.
                     </p>
                  </div>
               </div>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="540">
<tr>
  <td align="center"><p><a href="chemorecepfig3.jpg"><img src="chemorecepfig3_s.jpg" border="2" align="middle" alt="figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 3. SR genome positions.&nbsp;</b>Panel A shows the positions in the entire genome of all analyzed members of
                                    chemoreceptor families (those listed in <a href="#table1" title="Table 1. Gene and pseudogene numbers in chemoreceptor families and superfamilies">Table 1</a>). Panel B shows the positions of all
                                    members of three specific families on chromosome V.
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1212"></a>3.&nbsp;Gene expansions
                     </h2>
                  </div>
               </div>
            </div>
            <p>An unusual characteristic of the chemoreceptor families is their highly dynamic gene
                           number, reflected within <span class="emphasis"><em>C. elegans</em></span> by frequent duplications and
                           gene loss. Systematic comparisons with <span class="emphasis"><em>C. briggsae</em></span> are only
                           beginning, but evidence thus far indicates that the familiar concept of orthology is
                           largely useless for these families due to their dynamic evolution. <a href="#figure4">Figure 4</a> shows a phylogenetic tree of two of the three families for which a
                           completed <span class="emphasis"><em>C. briggsae</em></span> annotation is available (<a href="#bib7">Chen et al., 2005</a>). Several large expansions in gene number in <span class="emphasis"><em>C.
                                 elegans</em></span> are apparent and only 7 pairs of genes are convincing one-to-one
                           orthologs. Several additional reciprocal best match pairs (marked "a") are substantially
                           more divergent in protein sequence, which probably indicates that they duplicated well
                           before <span class="emphasis"><em>elegans-briggsae</em></span> speciation (followed by loss of one copy in
                           both species) and are not orthologs in the strict sense. Similar dynamics are even more
                           extreme in the <span class="emphasis"><em>srz</em></span> family (<a href="#bib42">Thomas et al., 2005</a>) and
                           the <span class="emphasis"><em>srh</em></span> families (JHT, unpublished).
            </p>
            <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="495">
<tr>
  <td align="center"><p><a href="chemorecepfig4.jpg"><img src="chemorecepfig4_s.jpg" border="2" align="middle" alt="figure 4"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 4. <span class="emphasis"><em>sra</em></span> and <span class="emphasis"><em>srab</em></span> protein tree.&nbsp;</b>Unrooted neighbor-joining tree of all putative functional SRA and SRAB proteins from <span class="emphasis"><em>C. elegans</em></span> (blue) and <span class="emphasis"><em>C. briggsae</em></span> (red). Data are adapted from <a href="#bib7">Chen et al. (2005)</a>. Arrows point to <span class="emphasis"><em>C. elegans</em></span> gene expansions. "orth" <span class="symbol">=</span> putative one-to-one orthologs. "a" <span class="symbol">=</span> possible paralogs with reciprocal gene loss. The distinction between orthologs and paralogs is based only on protein distance
                     and is somewhat arbitrary, though in approximate agreement with the average divergence of apparent orthologs in other chemoreceptor
                     families.
                  </p>
               </div>
            </div>
            <p>Much work remains to be completed, but the broad picture is that the chemoreceptor
                           gene families are in a dynamic state, with fixation of new gene duplicates approximately
                           balancing gene loss (reflected in the large number of defective genes, and in the
                           evidence for ultimate deletion of many genes entirely). Over relatively short periods
                           this dynamic is probably near equilibrium, resulting in a stable number of genes.
                           However, over longer periods unknown factors can change the relative rates of gene gain
                           and gene loss, giving rise to the dramatically different gene numbers in <span class="emphasis"><em>C.
                                 elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>. This pattern is strikingly
                           reminiscent of recent findings in mammalian odorant and vomeronasal gene families, in
                           which gene duplications, gene loss, and lineage specific expansions are common (<a href="#bib12">Del
                              Punta et al., 2000</a>; <a href="#bib14">Glusman et al., 2001</a>; <a href="#bib23">Matsunami et al.,
                              2000</a>). It will be interesting to determine the chemosensory gene
                           distributions for the other <span class="emphasis"><em>Caenorhabditis</em></span> species currently in the
                           genome sequencing pipeline.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1308"></a>4.&nbsp;Gene clusters
                     </h2>
                  </div>
               </div>
            </div>
            <p>The putative chemosensory genes are strongly clustered in the genome (<a href="#figure3">Figure 3</a>). Most families are heavily enriched on chromosome V, and
                           significant enrichment is also found on chromosomes II and IV. Furthermore, the
                           positions of genes on all of these chromosomes are strongly biased toward the chromosome
                           arms (as defined by increased recombination frequency; <a href="#bib5">Barnes et al., 1995</a>). In addition to these broad patterns of gene clustering, there is a strong
                           tendency for closely related genes to be physically close in the genome (<a href="#bib7">Chen et al.,
                              2005</a>; <a href="#bib32">Robertson, 2000</a>; <a href="#bib31">Robertson, 2001</a>). These
                           patterns are likely attributable to patterns of gene duplication in <span class="emphasis"><em>C.
                                 elegans</em></span>. New duplications are almost always local, often tandem or inverted
                           repeats with little or no intervening sequence between the duplicates (<a href="#bib20">Katju and Lynch, 2003</a>; <a href="#bib35">Semple and Wolfe, 1999</a>). The current gene positions are
                           consistent with a history of frequent local gene duplication and rare large chromosomal
                           rearrangements, which occasionally break up clusters of local related genes. These large
                           rearrangements are rarely if ever interchromosomal translocations; we suspect that most
                           are inversions.
            </p>
            <p>Detailed analysis of the distribution of <span class="emphasis"><em>srh</em></span> and <span class="emphasis"><em>str</em></span> genes not on chromosome V showed
                           that small interchromosomal transposition events occur occasionally (<a href="#bib31">Robertson, 1998</a>; <a href="#bib32">Robertson, 2000</a>; <a href="#bib33">Robertson, 2001</a>). These events
                           are followed in some cases by a duplication and diversification process in the new
                           genomic location. As with chromosome V, residence on other chromosomal arms seems to
                           favor the duplication process. The mechanisms underlying this pattern are not known, but
                           are likely to involve the higher frequency of complex DNA repeats found on almost every
                           autosomal arm.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1362"></a>5.&nbsp;Function and expression patterns
                     </h2>
                  </div>
               </div>
            </div>
            <p>Analysis of the function of <span class="emphasis"><em>C. elegans</em></span> chemoreceptors has been
                           remarkably resistant to genetic methods. Only <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=odr-10" target="_blank">odr-10</a></em></span> has been
                           identified in classical forward genetic screens (<a href="#bib37">Sengupta et al., 1996</a>) and
                           the fraction of genes assigned gross phenotypes in the initial global RNAi screen is
                           close to zero (<a href="#bib19">Kamath et al., 2003</a>). This resistance may result from a
                           combination of the expected narrow phenotype range for the genes (and a relative paucity
                           of genetic screens directed toward this) and genetic redundancy among closely related
                           receptors. A few additional chemoreceptor genes have more limited phenotypic
                           information. Deletion or overexpression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sra-13" target="_blank">sra-13</a> </em></span>affects olfactory
                           response to isoamyl alcohol and also affects vulval induction, probably as part of a
                           sensory response to starvation (<a href="#bib5">Battu et al., 2003</a>). One sign of hope is
                           that about 25 chemoreceptor genes were identified as affecting fat accumulation in a
                           global RNAi screen testing specifically for this phenotype (<a href="#bib2">Ashrafi et al., 2003</a>). In addition, RNAi knockdown of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=str-2" target="_blank">str-2</a></em></span> is reported to
                           extend lifespan, though the effect is small (<a href="#bib1">Alcedo and Kenyon, 2004</a>).
                           These RNAi results are uncorroborated by other approaches, but it is possible that
                           environmental chemoperception or internal physiological monitoring by chemoreceptors
                           plays a prominent role in energy resource allocation.
            </p>
            <p>The cellular expression patterns of the putative chemosensory receptor genes are of
                           interest for two reasons. First, their classification as chemosensory receptors is
                           currently best assessed by testing whether expression is predominantly in chemosensory
                           neurons. Second, because of the large set of genes, their patterns of expression provide
                           a remarkable opportunity to study the relationship between cell fate, transcription
                           factors, and <span class="emphasis"><em>cis</em></span> regulatory sequences. Only one gene in the entire
                           SR class has been rigorously shown to function as a chemosensory receptor (<a href="#bib37">Sengupta et
                              al., 1996</a>; <a href="#bib45">Zhang et al., 1997</a>). Are the other ~1,300 genes
                           really chemosensory receptors, or are they a mixture of chemoreceptors and other types
                           of receptors? Transgenic GFP-fusion expression data bearing on this question are
                           summarized in <a href="#table2" title="Table 2. Summaries of known GFP-fusion expression patterns for chemoreceptors">Table 2</a>. Though the coverage of the superfamily is insufficient to draw
                           detailed inferences, a broad picture can be painted. Many genes are expressed only in
                           chemosensory neurons, consistent with a strictly chemosensory function. There are also exceptions to this rule,
               which might result from real function in other types
                           of neurons and non-neuronal tissue or from transgenic misexpression in ectopic tissues,
                           possibilities that it is currently not possible to distinguish. In addition to GFP
                           fusion data, one study used microarrays to examine gene expression from cell-sorted AWB
                           neurons (<a href="#bib11">Colosimo et al., 2004</a>), a method that is likely to gain
                           importance in the future. SR genes were prominent among the genes identified, including
                               <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=str-220" target="_blank">str-220</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=srd-16" target="_blank">srd-16</a></em></span>,
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=srd-23" target="_blank">srd-23</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sru-38" target="_blank">sru-38</a></em></span>, and
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=srsx-5" target="_blank">srsx-5</a></em></span>. Analysis of SR expression by microarray studies using total
                           worm mRNA is problematic because of their generally low abundance transcript levels;
                           though some genes were assigned to expression mountains (<a href="#bib21">Kim et al., 2001</a>), these assignments should be viewed with some skepticism because absolute
                           expression levels are typically at or close to noise levels.
            </p>
            <p>A final method that shows promise in assigning expression patterns to SR genes is
                           identification and analysis of specific promoter sequence motifs that may be binding
                           sites for transcription factors. Using this approach, one study identified a
                           well-defined E-box motif in the promoters of about 200 SR genes, mostly from the <span class="emphasis"><em>srh</em></span> and
                           <span class="emphasis"><em>sri</em></span> families (<a href="#bib24">McCarroll et al., 2005</a>). It was further
                           shown that the E-box is correlated with expression in the ADL chemosensory neuron, and
                           that it was sufficient to direct expression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=odr-10" target="_blank">odr-10</a></em></span> in ADL.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1462"></a>6.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>HMR thanks Kim Walden (<span class="emphasis"><em>sru</em></span>, <span class="emphasis"><em>srv</em></span>,
                           <span class="emphasis"><em>srx</em></span> families), Kim Ly (<span class="emphasis"><em>srxa</em></span> and
                           <span class="emphasis"><em>srz</em></span> families), Colin Stoetzner (<span class="emphasis"><em>srb</em></span> and
                           <span class="emphasis"><em>srbc</em></span> families), Lindsey Schmidt (<span class="emphasis"><em>srw</em></span> family),
                           and Greg New (<span class="emphasis"><em>srx</em></span> family) for assistance with annotation of some of
                           the gene families, all the WormBase annotators and Jonathan Hodgkin for their assistance
                           over the past decade, and Cori Bargmann for her enthusiastic support. JHT annotated the
                           <span class="emphasis"><em>srt</em></span> family and thanks Mary Stewart, Emily Rocke, Evan Galloway, and Joanna Kelley for
                           assistance. HMR was supported by NSF grant IBN-96-04095 and NIH grant 1AI056081; JHT was
                           supported by NIH grant RO1GM48700.
            </p>
            <p>&nbsp;</p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1499"></a>7.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                             <span class="bibliomisc">Alcedo, J., and Kenyon, C. (2004). Regulation of <span class="emphasis"><em>C. elegans</em></span> longevity by specific gustatory and olfactory neurons. Neuron <span class="emphasis"><em>41</em></span>, 45&#8211;55.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14715134&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(03)00816-X" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                             <span class="bibliomisc">Ashrafi, K., Chang, F.Y., Watts, J.L., Fraser, A.G., Kamath, R.S., Ahringer, J., and Ruvkun, G. (2003). Genome-wide RNAi analysis
                        of <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em>fat</em></span> regulatory genes. Nature <span class="emphasis"><em>421</em></span>, 268&#8211;272.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12529643&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature01279" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                             <span class="bibliomisc">Bargmann, C.I., Mori, I. (1997). Chemotaxis and thermotaxis. In <span class="emphasis"><em>C. elegans</em></span> <span class="emphasis"><em>II</em></span>, D.L. Riddle, Blumenthal, T., Meyer, B.J., Priess, J.R., eds. (Cold Spring Harbor, New York: Cold Spring Harbor Laboratory
                        Press), pp. 717&#8211;737.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                             <span class="bibliomisc">Barnes, T.M., Kohara, Y., Coulson, A., and Hekimi, S. (1995). Meiotic recombination, noncoding DNA and genomic organization
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>141</em></span>, 159&#8211;179.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8536965&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                             <span class="bibliomisc">Battu, G., Hoier, E.F., and Hajnal, A. (2003). The <span class="emphasis"><em>C. elegans</em></span> G-protein-coupled receptor <a href="http://www.wormbase.org/db/seq/protein?name=SRA-13;class=protein" target="_blank">SRA-13</a> inhibits RAS/MAPK signalling during olfaction and vulval development. Development <span class="emphasis"><em>130</em></span>, 2567&#8211;2577.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12736202&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.00497" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                             <span class="bibliomisc">Buck, L., and Axel, R. (1991). A novel multigene family may encode odorant receptors: a molecular basis for odor recognition.
                        Cell <span class="emphasis"><em>65</em></span>, 175&#8211;187.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1840504&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(91)90418-X" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                             <span class="bibliomisc">Chen, N., Pai, S., Zhao, Z., Mah, A., Newbury, R., Johnsen, R.C., Altun, Z., Moerman, D.G., Baillie, D.L., and Stein, L.D.
                        (2005). Identification of a nematode chemosensory gene family. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>102</em></span>, 146&#8211;151.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15618405&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0408307102" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                             <span class="bibliomisc">Chenna, R., Sugawara, H., Koike, T., Lopez, R., Gibson, T.J., Higgins, D.G., and Thompson, J.D. (2003). Multiple sequence
                        alignment with the Clustal series of programs. Nucleic Acids Res. <span class="emphasis"><em>31</em></span>, 3497&#8211;3500.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12824352&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkg500" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                             <span class="bibliomisc">Choi, S.S., and Lahn, B.T. (2003). Adaptive evolution of MRG, a neuron-specific gene family implicated in nociception. Genome
                        Res. <span class="emphasis"><em>13</em></span>, 2252&#8211;2259.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14525927&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.1431603" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                             <span class="bibliomisc">Coghlan, A., and Wolfe, K.H. (2004). Origins of recently gained introns in <span class="emphasis"><em>Caenorhabditis</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 11362&#8211;11367.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15243155&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0308192101" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                             <span class="bibliomisc">Colosimo, M.E., Brown, A., Mukhopadhyay, S., Gabel, C., Lanjuin, A.E., Samuel, A.D., and Sengupta, P. (2004). Identification
                        of thermosensory and olfactory neuron-specific genes via expression profiling of single neuron types. Curr. Biol. <span class="emphasis"><em>14</em></span>, 2245&#8211;2251.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15620651&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.12.030" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                             <span class="bibliomisc">Del Punta, K., Rothman, A., Rodriguez, I., and Mombaerts, P. (2000). Sequence diversity and genomic organization of vomeronasal
                        receptor genes in the mouse. Genome Res. <span class="emphasis"><em>10</em></span>, 1958&#8211;1967.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11116090&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.10.12.1958" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                             <span class="bibliomisc">Egerod, K., Reynisson, E., Hauser, F., Cazzamali, G., Williamson, M., and Grimmelikhuijzen, C.J. (2003). Molecular cloning
                        and functional expression of the first two specific insect myosuppressin receptors. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>100</em></span>, 9808&#8211;9813.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12907701&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.1632197100" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                             <span class="bibliomisc">Glusman, G., Yanai, I., Rubin, I., and Lancet, D. (2001). The complete human olfactory subgenome. Genome Res. <span class="emphasis"><em>11</em></span>, 685&#8211;702.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11337468&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.171001" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                             <span class="bibliomisc">Henikoff, S., and Henikoff, J.G. (1992). Amino acid substitution matrices from protein blocks. Proc. Natl. Acad. Sci. USA
                        <span class="emphasis"><em>89</em></span>, 10915&#8211;10919.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1438297&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hilliard, M.A., Bergamasco, C., Arbucci, S., Plasterk, R.H., and Bazzicalupo, P. (2004). Worms taste bitter: ASH neurons,
                        <a href="http://www.wormbase.org/db/seq/protein?name=QUI-1;class=protein" target="_blank">QUI-1</a>, <a href="http://www.wormbase.org/db/seq/protein?name=GPA-3;class=protein" target="_blank">GPA-3</a> and <a href="http://www.wormbase.org/db/seq/protein?name=ODR-3;class=protein" target="_blank">ODR-3</a> mediate quinine avoidance in<span class="emphasis"><em>Caenorhabditis elegans</em></span>. EMBO J. <span class="emphasis"><em>23</em></span>, 1101&#8211;1111.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14988722&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/sj.emboj.7600107" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                             <span class="bibliomisc">Jansen, G., Thijssen, K.L., Werner, P., van der Horst, M., Hazendonk, E., and Plasterk, R.H. (1999). The complete family of
                        genes encoding G proteins of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Genet. <span class="emphasis"><em>21</em></span>, 414&#8211;419.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10192394&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/7753" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                             <span class="bibliomisc">Jansen, G., Weinkove, D., and Plasterk, R.H. (2002). The G-protein gamma subunit <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gpc-1" target="_blank">gpc-1</a></em></span> of the nematode <span class="emphasis"><em>C.elegans</em></span> is involved in taste adaptation. EMBO J. <span class="emphasis"><em>21</em></span>, 986&#8211;994.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11867526&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/emboj/21.5.986" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kamath, R.S., Fraser, A.G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., Kanapin, A., Le Bot, N., Moreno, S., Sohrmann, M.,
                        et al. (2003). Systematic functional analysis of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome using RNAi. Nature <span class="emphasis"><em>421</em></span>, 231&#8211;237.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12529635&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature01278" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                             <span class="bibliomisc">Katju, V., and Lynch, M. (2003). The structure and early evolution of recently arisen gene duplicates in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome. Genetics <span class="emphasis"><em>165</em></span>, 1793&#8211;1803.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14704166&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kim, S.K., Lund, J., Kiraly, M., Duke, K., Jiang, M., Stuart, J.M., Eizinger, A., Wylie, B.N., and Davidson, G.S. (2001).
                        A gene expression map for <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>293</em></span>, 2087&#8211;2092.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11557892&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1061603" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kiontke, K., Gavin, N.P., Raynes, Y., Roehrig, C., Piano, F., and Fitch, D.H. (2004).<span class="emphasis"><em>Caenorhabditis</em></span> phylogeny predicts convergence of hermaphroditism and extensive intron loss. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 9003&#8211;9008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15184656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0403094101" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                             <span class="bibliomisc">Matsunami, H., Montmayeur, J.P., and Buck, L.B. (2000). A family of candidate taste receptors in human and mouse. Nature <span class="emphasis"><em>404</em></span>, 601&#8211;604.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10766242&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35007072" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                             <span class="bibliomisc">McCarroll, S.A., Li, H., and Bargmann, C. (2005). Identification of transcriptional regulatory elements in chemosensory receptor
                        genes by probabilistic segmentation. Curr. Biol. <span class="emphasis"><em>15</em></span>, 347&#8211;352.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15723796&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.02.023" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                             <span class="bibliomisc">Meeusen, T., Mertens, I., Clynen, E., Baggerman, G., Nichols, R., Nachman, R. J., Huybrechts, R., De Loof, A., and Schoofs,
                        L. (2002). Identification in <span class="emphasis"><em>Drosophila melanogaster</em></span> of the invertebrate G protein-coupled FMRFamide receptor. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>99</em></span>, 15363&#8211;15368.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12438685&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.252339599" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                             <span class="bibliomisc">Mertens, I., Vandingenen, A., Meeusen, T., Janssen, T., Luyten, W., Nachman, R. J., De Loof, A., and Schoofs, L. (2004). Functional
                        characterization of the putative orphan neuropeptide G-protein coupled receptor <a href="http://www.wormbase.org/db/get?name=C26F1.6;class=Gene" target="_blank">C26F1.6</a> in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. FEBS Lett. <span class="emphasis"><em>573</em></span>, 55&#8211;60.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15327975&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.febslet.2004.07.058" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                             <span class="bibliomisc">Miller, M.A., Cutter, A.D., Yamamoto, I., Ward, S., and Greenstein, D. (2004). Clustered organization of reproductive genes
                        in the <span class="emphasis"><em>C. elegans</em></span> genome. Curr. Biol. <span class="emphasis"><em>14</em></span>, 1284&#8211;1290.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15268860&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.07.025" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                             <span class="bibliomisc">Mori, I. (1999). Genetics of chemotaxis and thermotaxis in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Annu. Rev. Genet. <span class="emphasis"><em>33</em></span>, 399&#8211;422.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10690413&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.genet.33.1.399" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                             <span class="bibliomisc">Mounsey, A., Bauer, P., and Hope, I.A. (2002). Evidence suggesting that a fifth of annotated<span class="emphasis"><em>Caenorhabditis elegans</em></span> genes may be pseudogenes. Genome Res. <span class="emphasis"><em>12</em></span>, 770&#8211;775.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11997343&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr208802. Article published online before print in April 2002" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                             <span class="bibliomisc">Roayaie, K., Crump, J.G., Sagasti, A., and Bargmann, C.I. (1998). The G<span class="emphasis"><em><span class="symbol">&#945;</span></em></span> protein <a href="http://www.wormbase.org/db/seq/protein?name=ODR-3;class=protein" target="_blank">ODR-3</a> mediates olfactory and nociceptive function and controls cilium morphogenesis in <span class="emphasis"><em>C. elegans</em></span> olfactory neurons. Neuron <span class="emphasis"><em>20</em></span>, 55&#8211;67.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9459442&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(00)80434-1" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                             <span class="bibliomisc">Robertson, H.M. (1998). Two large families of chemoreceptor genes in the nematodes <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span> reveal extensive gene duplication, diversification, movement, and intron loss. Genome Res. <span class="emphasis"><em>8</em></span>, 449&#8211;463.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9582190&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11238245&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                             <span class="bibliomisc">Robertson, H.M. (2000). The large<span class="emphasis"><em>srh</em></span> family of chemoreceptor genes in<span class="emphasis"><em>Caenorhabditis</em></span> nematodes reveals processes of genome evolution involving large duplications and deletions and intron gains and losses. Genome
                        Res. <span class="emphasis"><em>10</em></span>, 192&#8211;203.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10673277&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.10.2.192" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                             <span class="bibliomisc">Robertson, H.M. (2001). Updating the<span class="emphasis"><em>str</em></span> and <span class="emphasis"><em>srj</em></span> (<span class="emphasis"><em>stl</em></span>) families of chemoreceptors in<span class="emphasis"><em>Caenorhabditis</em></span> nematodes reveals frequent gene movement within and between chromosomes. Chem. Senses <span class="emphasis"><em>26</em></span>, 151&#8211;159.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11238245&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/chemse/26.2.151" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                             <span class="bibliomisc">Schmidt, H.A., Strimmer, K., Vingron, M., and von Haeseler, A. (2002). TREE-PUZZLE: maximum likelihood phylogenetic analysis
                        using quartets and parallel computing. Bioinformatics <span class="emphasis"><em>18</em></span>, 502&#8211;504.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11934758&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/bioinformatics/18.3.502" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                             <span class="bibliomisc">Semple, C., and Wolfe, K.H. (1999). Gene duplication and gene conversion in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome. J. Mol. Evol. <span class="emphasis"><em>48</em></span>, 555&#8211;564.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10198121&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sengupta, P. (1997). Cellular and molecular analyses of olfactory behavior in <span class="emphasis"><em>C. elegans</em></span>. Semin. Cell Dev. Biol. <span class="emphasis"><em>8</em></span>, 153&#8211;161.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15001091&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/scdb.1996.0137" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sengupta, P., Chou, J.H., and Bargmann, C.I. (1996). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=odr-10" target="_blank">odr-10</a></em></span> encodes a seven transmembrane domain olfactory receptor required for responses to the odorant diacetyl. Cell <span class="emphasis"><em>84</em></span>, 899&#8211;909.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8601313&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(00)81068-5" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                             <span class="bibliomisc">Stein, L.D., Bao, Z., Blasiar, D., Blumenthal, T., Brent, M.R., Chen, N., Chinwalla, A., Clarke, L., Clee, C., Coghlan, A.,
                        et al. (2003). The genome sequence of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>: a platform for comparative genomics. PLoS Biol. <span class="emphasis"><em>1</em></span>, E45.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14624247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                             <span class="bibliomisc">Stewart, M.K., Clark, N.L., Merrihew, G., Galloway, E., and Thomas, J.H. (2005). High genetic diversity in the chemoreceptor
                        superfamily of <span class="emphasis"><em>C. elegans</em></span>. Genetics <span class="emphasis"><em>169</em></span>, 1985&#8211;1996.</span>
                     <a href="http://dx.doi.org/10.1534/genetics.104.035329" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                             <span class="bibliomisc">Swanson, W.J., and Vacquier, V.D. (2002). The rapid evolution of reproductive proteins. Nat. Rev. Genet. <span class="emphasis"><em>3</em></span>, 137&#8211;144.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11836507&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg733" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                             <span class="bibliomisc">Swofford, D.L. (1998). <span class="emphasis"><em>PAUP<span class="symbol">&#8727;</span>: Phylogenetic analysis using parsimony and other methods, Version 4</em></span> (New York, NY: Sinauer Press).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                             <span class="bibliomisc">Thomas, J.H., Kelley, J.L., Robertson, H.M., Ly, K., and Swanson, W.J. (2005). Adaptive evolution in the SRZ chemoreceptor
                        families of <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>102</em></span>, 4476&#8211;4481.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15761060&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0406469102" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                             <span class="bibliomisc">Thomas, J.H. (2006). Analysis of Homologous Gene Clusters in <span class="emphasis"><em>C. elegans</em></span> Reveals Striking Regional Cluster Domains. Genetics. In press</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                             <span class="bibliomisc">Troemel, E.R., Chou, J.H., Dwyer, N.D., Colbert, H.A., and Bargmann, C.I. (1995). Divergent seven transmembrane receptors
                        are candidate chemosensory receptors in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>83</em></span>, 207&#8211;218.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7585938&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(95)90162-0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                             <span class="bibliomisc">Wenick, A.S., and Hobert, O. (2004). Genomic <span class="emphasis"><em>cis</em></span>-regulatory architecture and <span class="emphasis"><em>trans</em></span>-acting regulators of a single interneuron-specific gene battery in <span class="emphasis"><em>C. elegans</em></span>. Dev. Cell <span class="emphasis"><em>6</em></span>, 757&#8211;770.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15177025&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2004.05.004" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                             <span class="bibliomisc">Zhang, Y., Chou, J.H., Bradley, J., Bargmann, C.I., and Zinn, K. (1997). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> seven-transmembrane protein <a href="http://www.wormbase.org/db/seq/protein?name=ODR-10;class=protein" target="_blank">ODR-10</a> functions as an odorant receptor in mammalian cells. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>94</em></span>, 12162&#8211;12167.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9342380&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.94.22.12162" target="_blank">Article</a>
                         
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e11" href="#d0e11">*</a></sup>Edited by Iva Greenwald. Last revised April 13, 2005. Published January 06, 2006. This chapter should be cited as: Robertson,
                  H. M. and Thomas, J. H. The putative chemoreceptor families of <span class="emphasis"><em>C. elegans</em></span> (January 06, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.66.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Hugh M. Robertson and James H. Thomas. This is an open-access article distributed under the terms of the Creative
                  Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original
                  author and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e41" href="#d0e41">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:hughrobe@uiuc.edu" target="_top">hughrobe@uiuc.edu</a> or <a href="mailto:jht@u.washington.edu" target="_top">jht@u.washington.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All journal content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

