<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>RNA in situ hybridization of dissected gonads	
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#RNAish_ArticleID" title="RNA in situ hybridization of dissected gonads&#xA;">
      <link rel="next" href="#d0e65" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="RNAishdissectedgonads.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="RNAish_ArticleID"></a>RNA <span class="emphasis"><em>in situ</em></span> hybridization of dissected gonads<sup><a name="d0e11" href="#ftn.d0e11">*</a></sup>	
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Min-Ho Lee<em><span class="remark">
                                 					<sup><a name="d0e39" href="#ftn.d0e39">&sect;</a></sup>
                                 				</span></em>
                           				<em><span class="remark">
                                 					<sup><a name="d0e52" href="#ftn.d0e52">&#8224;</a></sup>
                                 				</span></em>
                           , Tim Schedl<em><span class="remark"><sup><a name="d0e39" href="#ftn.d0e39">&sect;</a></sup></span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Department of Genetics, Washington University School of Medicine, St. Louis, MO 63110 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Gene expression
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=308.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=RNA+in+situ+hybridization+of+dissected+gonads" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e65">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#d0e209">2. General expression pattern</a></span></dt>
               <dt><span class="sect1"><a href="#d0e243">3. Materials</a></span></dt>
               <dt><span class="sect1"><a href="#d0e352">4. Procedure</a></span></dt>
               <dt><span class="sect1"><a href="#d0e793">5. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#d0e798">6. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e65"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>Determination of the temporal and spatial pattern of gene expression is an important step in understanding gene function during
               <span class="emphasis"><em>C. elegans</em></span> development. The expression pattern of a gene can be examined in several ways including transgenic reporter fusions (e.g.,
               gfp, <span class="symbol">&#946;</span>-galactosidase), as well as immunohistochemistry and RNA <span class="emphasis"><em>in situ</em></span> hybridization for analysis of the endogenous protein and mRNA, respectively. While reporter fusions and immnohistochemistry
               require time and effort to generate reagents (reporter construction and generation of transgenic lines or antibody production),
               RNA <span class="emphasis"><em>in situ</em></span> hybridization only requires the transcribed region of a gene of interest. In addition, as a complement to protein expression
               patterns of a gene, RNA <span class="emphasis"><em>in situ</em></span> allows one to determine whether the mRNA is subjected to post-transcriptional controls such as differential RNA localization,
               stability, and/or translational regulation (<a href="#bib3">Evans et al., 1994</a>; <a href="#bib13">Seydoux and Fire 1994</a>; <a href="#bib6">Jones et al., 1996</a>; <a href="#bib8">Lee and Schedl 2001</a>,<a href="#bib9"> 2004</a>).
            </p>
            <p>Several <span class="emphasis"><em>in situ</em></span> hybridization protocols have been developed to detect RNA in <span class="emphasis"><em>C. elegans</em></span>. RNA <span class="emphasis"><em>in situ</em></span> in embryos is well described previously (<a href="#bib14">Seydoux and Fire, 1995</a>. Protocols for whole-mount <span class="emphasis"><em>in situ</em></span> of intact animals and embryos can be also found at  <a href="http://nematode.lab.nig.ac.jp/method/index.php" target="_blank">Yuji Kohara's website</a>. Fluorescence RNA <span class="emphasis"><em>in situ</em></span> hybridizations have been also described (<a href="#bib1">Albertson et al., 1995</a>; <a href="#bib5">Graves, et al., 1999</a>; <a href="#bib11">Pitt et al., 2000</a>). In this chapter, a protocol for detection of mRNA by <span class="emphasis"><em>in situ</em></span> hybridization to dissected gonads is described. This protocol is primarily based on the method of <a href="#bib14">Seydoux and Fire (1995)</a> as adapted for dissected gonads.
            </p>
            <p>In contrast to an earlier method described for dissected gonads wherein the gonads were affixed to slides (<a href="#bib2">Crittenden et al., 1994</a>), the various manipulations described below (gonad dissections, hybridizations, and washes) are all performed in solution,
               either in glass dishes (dissection) or in small glass tubes (hybridization). Washes are done by low-speed spins and all solutions
               (except fixatives) contain 0.1% Tween 20 to prevent dissected gonads from sticking to glass. A key to make this protocol work
               consistently is to fix dissected gonads in a combination of paraformaldehyde (3%) and glutaraldehyde (0.25%). The addition
               of glutaraldehyde results in a lower background, as compared to fixation with paraformaldehyde alone and makes gonads both
               less sticky and less susceptible to breakage.
            </p>
            <p>It is important to perform RNA <span class="emphasis"><em>in situ</em></span> hybridization using both sense and anti-sense probes for a gene of interest, at least for initial experiments. RNA <span class="emphasis"><em>in situ</em></span> hybridization with a sense probe should give little or no signal compared to that with an anti-sense probe (<a href="#figure1">Figure 1</a>). The sense probe background control is particularly important for detecting transcripts that are expressed at very low levels,
               which requires an extended developing time with the alkaline phosphatase-mediated detection system. In general, we can easily
               detect most mRNAs that have five or more EST clones reported in WormBase. For mRNAs that have less than two EST clones (therefore
               likely expressed at very low levels), it is helpful to concentrate the probe either by using more template in the asymmetric
               PCR (probe synthesis step) or by dissolving the digoxigenin (DIG) labeled probe in a smaller volume. RNA <span class="emphasis"><em>in situ</em></span> hybridization with a concentrated anti-sense probe usually results in increased signal with decreased developing time of
               the alkaline phosphatase-mediated detection system. In addition, concentration of the sense probe does not increase the background
               signal significantly.
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="RNAishfig1.jpg"><img src="RNAishfig1_s.jpg" border="2" align="middle" alt="Gemline RNA in situ hybridization with an anti-sense and a sense probes figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1. Gemline RNA <span class="emphasis"><em>in situ</em></span> hybridization with an anti-sense and a sense probes.&nbsp;</b>Representative photomicrographs of RNA <span class="emphasis"><em>in situ</em></span> hybridizations showing <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cep-1" target="_blank">cep-1</a></em></span>/p53 mRNA accumulation in wild-type (A) and <span class="emphasis"><em>gld-1</em></span>(null) (B) hermaphrodites, and wild-type male (C). Each panel has images from gonads hybridized with an anti-sense probe and
                     a sense probe otherwise treated equally. Each image shows a surface view of the gonad where the distal region is on the left
                     and the proximal region is on the right. Sense probe gave no or very little signals (the proximal signal is likely due to
                     endogenous phosphatases). In wild-type hermaphrodites, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cep-1" target="_blank">cep-1</a></em></span> mRNA accumulates throughout the germ line and its levels increase toward the proximal region of the germ line. In <span class="emphasis"><em>gld-1</em></span>(null), <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cep-1" target="_blank">cep-1</a></em></span> mRNA levels are slightly lower as compared with wild type. In wild-type males, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cep-1" target="_blank">cep-1</a></em></span> mRNA accumulates at higher levels only in the distal meiotic germ cells, showing sexual dimorphism.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e209"></a>2.&nbsp;General expression pattern
                     </h2>
                  </div>
               </div>
            </div>
            <p>The levels of most maternal mRNAs become higher in the middle of the pachytene region and remain high in the developing oocytes
               in the proximal region because the bulk of RNA synthesis for oogenesis occurs in pachytene stage germ cells, with progressively
               less RNA synthesis as germ cells proceed through the diplotene stage. As chromosomes become highly condensed in late stage
               oocytes (diakinesis), almost no <span class="emphasis"><em>de novo</em></span> RNA synthesis occurs (<a href="#bib4">Gibert et al., 1984</a>; <a href="#bib12">Schisa et al., 2001</a>) and RNA polymerase II becomes inactive (<a href="#bib7">Kelly et al., 2002</a>). Therefore, most maternal mRNAs are transcribed primarily in pachytene stage germ cells in the distal region and transported
               into the common cytoplasmic core. In the proximal region, maternal mRNAs are packaged into growing oocytes. These processes
               result in extensive RNA accumulation throughout pachytene stage germ cells and developing oocytes (<a href="#figure1">Figure 1</a>).
            </p>
            <p>Some mRNAs are expressed throughout the germline including the distal mitotic region (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span>) while others are only expressed in meiotic cells (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rme-2" target="_blank">rme-2</a></em></span>). For a number of transcripts expressed throughout the germline, the relative levels observed in the distal mitotic region
               are lower, compared to the levels in the meiotic pachytene cells and proximal oocytes (<a href="#figure1">Figure 1</a>). The difference is likely, at least in part, a reflection of a lower level of the mRNA/protein being required for function
               in the distal mitotic region, while a higher level is required for late oogenesis and/or maternally for early embryogenesis.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e243"></a>3.&nbsp;Materials
                     </h2>
                  </div>
               </div>
            </div>
            <p>Solutions do not need to be treated with diethyl pyrocarbonate.</p>
            <p>PBS: Make a 10x stock solution according to <a href="#bib15">Sambrook et al. (1989)</a>.
            </p>
            <p>PBTw: 1x PBS containing 0.1% Tween 20.</p>
            <p>Paraformaldehyde / glutaraldehyde fix:Paraformaldehyde/Glutaraldehyde fixation works well for germline RNA <span class="emphasis"><em>in situ</em></span>: gonads remain intact through an extensive protease treatment (50 <span class="symbol">&#956;</span>g/ml Proteinase K for 30 min at room temperature (RT) that would destroy gonads fixed in paraformaldehyde alone. This fixation
               can also be used with good results for histological staining with some antibodies.
                               
            </p>
            <div class="orderedlist">
               <p class="title">To make 3% paraformaldehyde / 0.25% glutaraldehyde / 0.1M K<sub>2</sub>HPO<sub>4</sub> (pH7.2), mix
               </p>
               <ol type="1">
                  <li>
                     <p>10 ml 16% formaldehyde (sealed ampoules from EM Sciences #15710)</p>
                  </li>
                  <li>
                     <p>0.53 ml 25% glutaraldehyde (sealed ampoules from EM Sciences #16220)</p>
                  </li>
                  <li>
                     <p>25 ml 0.2 M K<sub>2</sub>HPO<sub>4</sub> (pH7.2)
                     </p>
                     <p>and dispense in 10 ml aliquots and store at <span class="symbol">&#8722;</span>20<span class="symbol">&deg;</span>C.
                     </p>
                  </li>
               </ol>
            </div>
            <p>
                              
            </p>
            <p>Proteinase K: 20 mg/ml. Dispense in 10 <span class="symbol">&#956;</span>l aliquots and store at <span class="symbol">&#8722;</span>80<span class="symbol">&deg;</span>C. Do not refreeze, discard after use.
            </p>
            <p>Hybridization buffer: 5X SSC, 50 % deionized formamide, 100 <span class="symbol">&#956;</span>g/ml autoclaved Herring sperm DNA, 50 <span class="symbol">&#956;</span>g/ml Heparin and 0.1 % Tween-20. We make 50 ml, dispense in 10 ml aliquots and store at <span class="symbol">&#8722;</span>20<span class="symbol">&deg;</span>C.
            </p>
            <p>Alkaline-phosphatase-conjugated anti-DIG (Fab2 fragment) from Roche (#1 093 274): An optimal dilution of this antibody needs
               to be determined empirically as lot-to-lot variations may exist. We have been using dilutions from 1:1000 to 1:2500.
            </p>
            <p>Staining Solution: 100 mM NaCl; 5 mM MgCl<sub>2</sub>; 100 mM Tris (pH 9.5); 0.1% Tween 20; 1mM Levamisole. Levamisole is a potent inhibitor of endogenous phosphatases.
            </p>
            <p>Sigma Fast BCIP/NBT tablet (#B5655): Dissolve one tablet in 10 ml staining solution. If non-specific background signal is
               high, you may want to use more staining solution (up to 20 ml) to dissolve the tablet.
            </p>
            <p>Anti-Fade solution: Make 0.2M Dabco (Sigma #D2522) in 20 mM Tris (pH 8.0), which can be stored at <span class="symbol">&#8722;</span>20<span class="symbol">&deg;</span>C. Then dilute this solution (10%) in glycerol (90%) and store <span class="symbol">&#8722;</span>20<span class="symbol">&deg;</span>C.
            </p>
            <p>Culture tubes: Borosilicate glass tubes with plain end. These tubes can be baked to decontaminate RNase. Available from Fisher
               as 6 mm (O.D.) x 50 mm (length) in either Pyrex (Fisher #14-957AA) or Kimax (Fisher #14-925B) brands.
            </p>
            <p>Dissection dishes: We use Square Watch Glass from Carolina Biological Supply (#ER-74-2300).</p>
            <p>QIAquick PCR Purification kit from Qiagen (#28104): Works well to remove unincorporated dNTP and PCR primers.</p>
            <p>DIG DNA Labeling Mix, 10 x conc.: Roche (#1 277 065).</p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e352"></a>4.&nbsp;Procedure
                     </h2>
                  </div>
               </div>
            </div>
            <p>Single-stranded probes from cloned cDNA can be prepared anytime. Following dissections and fixation, fixed gonads can be stored
               in 100% MeOH at <span class="symbol">&#8722;</span>20<span class="symbol">&deg;</span>C for at least one week. Once the gonads are subjected to permeabilization, the remaining procedure has to be completed.
            </p>
            <div class="procedure"><a name="d0e363"></a><p class="title">Procedure&nbsp;1.&nbsp;Preparation of single-stranded probes from cloned cDNA</p>
               <p>Based on <a href="#bib10">Patel and Goodman (1992)</a>, with slight modifications.
               </p>
               <ol type="1">
                  <li>
                     <p class="title"><b>PCR amplification of a cDNA of interest with any set of primers.</b></p>
                     <p>A cloned cDNA template is optimal because amplified products from genomic DNA usually generate high background.</p>
                     <p>Use less than 1<span class="symbol">&#956;</span>l (10 pmole/<span class="symbol">&#956;</span>l) of primers because QIAquick PCR Purification kit will not effectively remove primers if the concentration is higher.
                     </p>
                     <p>We usually set up two 50 <span class="symbol">&#956;</span>l PCR reaction with 35 cycles (94<span class="symbol">&deg;</span>C 30 seconds, 55<span class="symbol">&deg;</span>C 30 seconds and 72<span class="symbol">&deg;</span>C, 1 minute/1 kb).
                     </p>
                     <table class="simplelist" border="0" summary="Simple list">
                        <tr>
                           <td>10x Taq Buffer</td>
                           <td>5.0 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                        <tr>
                           <td>dNTP mix (2.5 mM each)</td>
                           <td>2.0 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                        <tr>
                           <td>5'Primer (10 pmole/<span class="symbol">&#956;</span>l)
                           </td>
                           <td>1.0 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                        <tr>
                           <td>3'Primer (10 pmole/<span class="symbol">&#956;</span>l)
                           </td>
                           <td>1.0 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                        <tr>
                           <td>a cloned cDNA (1:50 dilution of miniprep DNA) or 5&#8211;10 ng of purified plasmid</td>
                           <td>2.0 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                        <tr>
                           <td>Taq Polymerase</td>
                           <td>2 units</td>
                        </tr>
                        <tr>
                           <td>ddH<sub>2</sub>O
                           </td>
                           <td>to 50 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                     </table>
                     <p>Run 3 <span class="symbol">&#956;</span>l each in an agarose gel; the PCR product must be an intense single band.
                     </p>
                  </li>
                  <li>
                     <p class="title"><b>Purification of PCR product</b></p>
                     <p>Use the QIAquick PCR Purification kit to purify the PCR product away from unincorporated dNTP and PCR primers. Elute with
                        25 <span class="symbol">&#956;</span>l Elution buffer. The concentration should be above 200 ng/<span class="symbol">&#956;</span>l.
                     </p>
                  </li>
                  <li>
                     <p class="title"><b>Asymmetric (one-way) PCR</b></p>
                     <p>To generate DIG labeled probe (35 cycles, 94<span class="symbol">&deg;</span>C 30 seconds, 55<span class="symbol">&deg;</span>C 30 seconds and 72<span class="symbol">&deg;</span>C, 1 minute/1 kb):
                     </p>
                     <table class="simplelist" border="0" summary="Simple list">
                        <tr>
                           <td>10x Taq Buffer</td>
                           <td>2.5 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                        <tr>
                           <td>DIG DNA Labeling Mix, 10 x conc</td>
                           <td>2.5 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                        <tr>
                           <td>Primer<sup><a name="d0e501" href="#ftn.d0e501">&#8225;</a></sup> (10 pmole/<span class="symbol">&#956;</span>l)
                           </td>
                           <td>1.0 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                        <tr>
                           <td>Purified PCR product (at least 200 ng/<span class="symbol">&#956;</span>l)
                           </td>
                           <td>4.0 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                        <tr>
                           <td>Taq Polymerase</td>
                           <td>2 units</td>
                        </tr>
                        <tr>
                           <td>ddH<sub>2</sub>O
                           </td>
                           <td>to 25 <span class="symbol">&#956;</span>l
                           </td>
                        </tr>
                     </table>
                  </li>
                  <li>
                     <p>75 <span class="symbol">&#956;</span>l of H<sub>2</sub>O is added to the reaction and 90&#8211;95 <span class="symbol">&#956;</span>l of the diluted reaction is transferred to a new tube.
                     </p>
                  </li>
                  <li>
                     <p>10 <span class="symbol">&#956;</span>l of 1M NaCl and 3 vols of 100% EtOH are added to the diluted reaction. After 30 min at <span class="symbol">&#8722;</span>70<span class="symbol">&deg;</span>C or overnight at <span class="symbol">&#8722;</span>20<span class="symbol">&deg;</span>C, the reaction is centrifuged at 14,000 rpm for 10 min. The pellet is washed in 70% ethanol, dried, and resuspended in 400
                        <span class="symbol">&#956;</span>l of hybridization buffer.
                     </p>
                  </li>
                  <li>
                     <p>The probe is boiled for 1 hr. This step reduces the length of the probe for efficient penetration.</p>
                  </li>
                  <li>
                     <p>Probes can be stored at <span class="symbol">&#8722;</span>20<span class="symbol">&deg;</span>C for a long period. We have used two year-old probe without any obvious reduction of signal.
                     </p>
                  </li>
               </ol>
            </div>
            <div class="procedure"><a name="d0e585"></a><p class="title">Procedure&nbsp;2.&nbsp;Dissections and fixation</p>
               <ol type="1">
                  <li>
                     <p>Wear latex gloves (optional), prepare dissected gonads (see <a href="http://www.genetics.wustl.edu/tslab/Protocols/gonad_dissections.html" target="_top">http://www.genetics.wustl.edu/tslab/gonad_dissections.html</a> for dissection) and fix in 3% paraformaldehyde / 0.25% glutaraldehyde / 0.1 M K<sub>2</sub>HPO<sub>4</sub> (pH7.2) for 1.5 to 2 hr at RT. We routinely dissect about 150 animals per each hybridization to obtain 40 or more gonads
                        that are completely extruded, well stained, and unbroken at the end of the entire procedure.
                     </p>
                  </li>
                  <li>
                     <p>Transfer entire contents to a 5 ml glass conical tube and spin in clinical table top centrifuge for 1 min at setting 3 (All
                        subsequent spins are also for 1 min at setting 3).
                     </p>
                  </li>
                  <li>
                     <p>Discard the fixative in the appropriate chemical waste container and resuspend in 2-4 ml PBTw.</p>
                  </li>
                  <li>
                     <p>Spin and discard PBTw as much as possible.</p>
                  </li>
                  <li>
                     <p>Add about 2 ml of 100% cold (-20<span class="symbol">&deg;</span>C) Methanol (MeOH). At this point, dissected gonads can be stored in 100% MeOH at <span class="symbol">&#8722;</span>20<span class="symbol">&deg;</span>C for at least one week with no obvious problems.
                     </p>
                  </li>
               </ol>
            </div>
            <div class="procedure"><a name="d0e621"></a><p class="title">Procedure&nbsp;3.&nbsp;Permeabilization: Proteinase K digestion</p>
               <ol type="1">
                  <li>
                     <p>Add 2-3 ml of PBTw to the MeOH solution, and spin. Wash twice in PBTw. Now split the sample into 2 or more tubes (either into
                        5 ml conical tubes or into 6 mm x 50 mm culture tubes).
                     </p>
                  </li>
                  <li>
                     <p>Using a 20 mg/ml stock of Proteinase K (PrK), make solutions of 50 <span class="symbol">&#956;</span>g/ml PrK in PBTw. Add 2ml PrK / PBTw and digest for 30 min at RT.
                     </p>
                  </li>
                  <li>
                     <p>Wash three times in PBTw.</p>
                  </li>
                  <li>
                     <p>Fix 15 min in 3% paraformaldehyde / 0.25% glutaraldehyde / 0.1M K<sub>2</sub>HPO<sub>4</sub> (pH7.2)
                     </p>
                  </li>
                  <li>
                     <p>Incubate 15 min in PBTw containing 2 mg/ml glycine.</p>
                  </li>
                  <li>
                     <p>Wash three times in PBTw.</p>
                  </li>
                  <li>
                     <p>Divide gonads among several 6 x 50 mm culture tubes, one tube per hybridzation.</p>
                  </li>
               </ol>
            </div>
            <div class="procedure"><a name="d0e654"></a><p class="title">Procedure&nbsp;4.&nbsp;Hybridization and wash</p>
               <ol type="1">
                  <li>
                     <p>Place gonads in 400 <span class="symbol">&#956;</span>l of 50% PBTw / 50% hybridization buffer (HB) and incubate 5 min at 48<span class="symbol">&deg;</span>C (in water bath).
                     </p>
                  </li>
                  <li>
                     <p>Pre-hybridize in 400 <span class="symbol">&#956;</span>l HB for 1 hr at 48<span class="symbol">&deg;</span>C.
                     </p>
                  </li>
                  <li>
                     <p>Dilute boiled probe in HB to give 100 <span class="symbol">&#956;</span>l of solution. Try a 1:2 dilution for probes to messages of unknown abundance. Experience with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gld-1" target="_blank">gld-1</a></em></span> probes suggests that probe dilution is not critical &#8211; a strong, clean signal was obtained at dilutions between 1:2 and 1:5,
                        but it began to fall off at 1:10. Probe to major sperm protein RNA, which is very abundant during spermatogenesis, could be
                        diluted 1:20 without any fall off in signal.
                     </p>
                  </li>
                  <li>
                     <p>Boil diluted probe 5 min, cool to 48<span class="symbol">&deg;</span>, and add to gonads.
                     </p>
                  </li>
                  <li>
                     <p>Hybridize for 12 to 36 hr at 48<span class="symbol">&deg;</span>C (longer hybridization times tend to give increased signal and lower background). We routinely hybridize for 36 hrs.
                     </p>
                  </li>
                  <li>
                     <p>Wash three times in PBTw.</p>
                  </li>
                  <li>
                     <p>Divide gonads among several 6 x 50 mm culture tubes, one tube per hybridzation.</p>
                  </li>
                  <li>
                     <p>Add 400 <span class="symbol">&#956;</span>l of HB that is pre-warmed to 48<span class="symbol">&deg;</span>, spin down, and remove supernatant.
                     </p>
                  </li>
                  <li>
                     <p>Divide gonads among several 6 x 50 mm culture tubes, one tube per hybridzation.</p>
                  </li>
                  <li>
                     <p>Wash 4 x 15 min in HB at 48<span class="symbol">&deg;</span>C.
                     </p>
                  </li>
                  <li>
                     <p>Then perform the following washes:</p>
                     <ol type="a">
                        <li>
                           <p>1 &times; 10 min in 50% HB / 50% PBTw at 48<span class="symbol">&deg;</span></p>
                        </li>
                        <li>
                           <p>2 &times; 10 min in PBTw at 48<span class="symbol">&deg;</span></p>
                        </li>
                        <li>
                           <p>1 &times; 5 min in PBTw at RT</p>
                        </li>
                     </ol>
                  </li>
               </ol>
            </div>
            <div class="procedure"><a name="d0e738"></a><p class="title">Procedure&nbsp;5.&nbsp;Probe detection</p>
               <ol type="1">
                  <li>
                     <p>Incubate 15 min in PBTw /0.5 mg/ml BSA (either fraction V or restriction enzyme grade).</p>
                  </li>
                  <li>
                     <p>Dilute alkaline-phosphatase-conjugated anti-DIG (Fab2 fragment) to optimal conc. in PBTw /BSA. Add 500 <span class="symbol">&#956;</span>l to each tube and incubate overnight at 4<span class="symbol">&deg;</span>C or at RT for 2 hr. For gonads, overnight at 4<span class="symbol">&deg;</span>C is definitely better than 2 hr at RT.
                     </p>
                  </li>
                  <li>
                     <p>Wash at RT as follows:</p>
                     <ol type="a">
                        <li>
                           <p>2 x 2 min PBTw</p>
                        </li>
                        <li>
                           <p>3 x 20 min PBTw / BSA</p>
                        </li>
                        <li>
                           <p>2 x 5 min in Staining solution.</p>
                        </li>
                     </ol>
                  </li>
                  <li>
                     <p>Transfer gonads to staining solution with BCIP/NDT and 100 ng/ml DAPI and cover to protect from light. Signal will come up
                        anywhere between 5 min and 6 hr. For convenience, the reaction can be monitored by transferring tube contents to a glass dish
                        and inspecting periodically under the dissecting scope. However, stain will appear darker in the dissecting scope than under
                        Nomarski optics with a 40X or higher lens.
                     </p>
                  </li>
                  <li>
                     <p>Stop the reaction by washing three times in PBTw. Finally place gonads in PBS containing 100 ng/ml DAPI.</p>
                  </li>
                  <li>
                     <p>After removing DAPI solution as much as possible, add 35 <span class="symbol">&#956;</span>l anti-fade solution.
                     </p>
                  </li>
               </ol>
            </div>
            <div class="procedure"><a name="d0e781"></a><p class="title">Procedure&nbsp;6.&nbsp;Mounting gonads for viewing</p>
               <ul>
                  <li>
                     <p>Using a drawn capillary pipette, transfer settled worms onto a large 2% agar pad that covers most of a slide. After drawing
                        off excess liquid with a capillary, an eyelash hair can be used to push gonads and intestines away from one another. Cover
                        with a large (24 x 50 mm) coverslip, taking care not to move the coverslip once in place. Also do not seal the coverslip immediately
                        &#8211; image may improve as liquid evaporates and gonads become somewhat flattened. We usually take images after storing the slides
                        at 4<span class="symbol">&deg;</span>C overnight without sealing. Bright field (Nomarski) images must be taken before DAPI images because the strong UV light focused
                        by the microscope literally burns gonads brown. Slides can be stored at 4<span class="symbol">&deg;</span>C for a week or more, particularly if sealed with nail polish around the periphery of the coverslip. Because the stain is
                        purplish, images are better in color than in black and white.
                     </p>
                  </li>
               </ul>
            </div>
            <div class="footnotes"><br><hr width="100" align="left">
               <div class="footnote">
                  <p><sup><a name="ftn.d0e501" href="#d0e501">&#8225;</a></sup>Use only one primer. We set up two reactions, one to generate a sense probe with 5'primer and the other to generate an anti-sense
                     probe with 3'primer. RNA <span class="emphasis"><em>in situ</em></span> hybridization with the sense probe will be a background control.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e793"></a>5.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>We are grateful to Ross Francis for establishing this protocol and members of the Schedl lab for comments on this review.
               Work in the TS lab is supported by NIH grant GM63310.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e798"></a>6.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Albertson, D.G., Fishpool, R.M., and Birchall, P.S. (1995). Fluorescence <span class="emphasis"><em>in situ</em></span> hybridization for the detection of DNA and RNA. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 339&#8211;364.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531734&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Crittenden, S.L., Troemel, E.R., Evans, T.C., and Kimble, J. (1994). <a href="http://www.wormbase.org/db/seq/protein?name=GLP-1;class=protein" target="_blank">GLP-1</a> is localized to the mitotic region of the <span class="emphasis"><em>C. elegans</em></span> germ line. Development <span class="emphasis"><em>120</em></span>, 2901&#8211;2911.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7607080&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Evans, T.C., Crittenden, S.L., Kodoyianni, V., and Kimble, J. (1994). Translational control of maternal <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> mRNA establishes an asymmetry in the <span class="emphasis"><em>C. elegans</em></span> embryo. Cell <span class="emphasis"><em>77</em></span>, 183&#8211;194.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8168128&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(94)90311-5" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Gibert, M.A., Starck, J., and Beguet, B. (1984). Role of the gonad cytoplasmic core during oogenesis of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Biol. Cell <span class="emphasis"><em>50</em></span>, 77&#8211;85.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6234041&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Graves, L.E., Segal, S., and Goodwin, E.B. (1999). <a href="http://www.wormbase.org/db/seq/protein?name=TRA-1;class=protein" target="_blank">TRA-1</a> regulates the cellular distribution of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-2" target="_blank">tra-2</a></em></span> mRNA in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>399</em></span>, 802&#8211;805.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10391248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/21682" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jones, A.R., Francis, R., and Schedl, T. (1996). <a href="http://www.wormbase.org/db/seq/protein?name=GLD-1;class=protein" target="_blank">GLD-1</a>, a cytoplasmic protein essential for oocyte differentiation, shows stage- and sex-specific expression during <span class="emphasis"><em>Caenorhabditis elegans</em></span> germline development. Dev. Biol. <span class="emphasis"><em>180</em></span>, 165&#8211;183.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8948583&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/dbio.1996.0293" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kelly, W.G., Schaner, C.E., Dernburg, A.F., Lee, M.H., Kim, S.K., Villeneuve, A.M., and Reinke, V. (2002). X-chromosome silencing
                        in the germline of <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>129</em></span>, 479&#8211;492.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11807039&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lee, M.H., and Schedl, T. (2001). Identification of <span class="emphasis"><em>in vivo</em></span> mRNA targets of <a href="http://www.wormbase.org/db/seq/protein?name=GLD-1;class=protein" target="_blank">GLD-1</a>, a maxi-KH motif containing protein required for <span class="emphasis"><em>C. elegans</em></span> germ cell development. Genes Dev. <span class="emphasis"><em>15</em></span>, 2408&#8211;2420.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11562350&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.915901" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lee, M.H., and Schedl, T. (2004). Translation repression by <a href="http://www.wormbase.org/db/seq/protein?name=GLD-1;class=protein" target="_blank">GLD-1</a> protects its mRNA targets from nonsense-mediated mRNA decay in <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>18</em></span>, 1047&#8211;1059.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15105376&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1188404" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Patel, N.H., and Goodman, C.S. (1992). Preparation of digoxigenin-labeled single-stranded DNA probes. In: <span class="emphasis"><em>Non-Radioactive Labeling and Detection of Biomolecules</em></span>, C. Kessler, ed. (Berlin: Springer-Verlag).</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Pitt, J.N., Schisa, J.A., and Priess, J.R. (2000). P granules in the germ cells of <span class="emphasis"><em>Caenorhabditis elegans</em></span> adults are associated with clusters of nuclear pores and contain RNA. Dev. Biol. <span class="emphasis"><em>219</em></span>, 315&#8211;333.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10694425&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/dbio.2000.9607" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                         <span class="bibliomisc">Sambrook, J., Fritsch, T., and Maniatis, T. (1989). Molecular cloning: a laboratory manual. 2nd edition. (New York: Cold Spring
                        Harbor Laboratory Press).</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Schisa, J.A., Pitt, J.N., and Priess, J.R. (2001). Analysis of RNA associated with P granules in germ cells of <span class="emphasis"><em>C. elegans</em></span> adults. Development <span class="emphasis"><em>128</em></span>, 1287&#8211;1298.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11262230&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Seydoux, G., and Fire, A. (1994). Soma-germline asymmetry in the distributions of embryonic RNAs in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>120</em></span>, 2823&#8211;2834.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7607073&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Seydoux, G., and Fire, A. (1995). Whole-mount <span class="emphasis"><em>in situ</em></span> hybridization for the detection of RNA in <span class="emphasis"><em>Caenorhabditis elegans</em></span> embryos. Methods Cell. Biol. <span class="emphasis"><em>48</em></span>, 323&#8211;337.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531733&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e11" href="#d0e11">*</a></sup>Edited by Oliver Hobert. WormMethods editor, Victor Ambros. Last revised May 3, 2005. Published June 14, 2006. This chapter
                  should be cited as: Lee, M.-H. and Schedl, T. RNA <span class="emphasis"><em>in situ</em></span> hybridization of dissected gonads (June 14, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.107.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Min-Ho Lee and Tim Schedl. This is an open-access article distributed under the terms of the Creative Commons Attribution
                  License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source
                  are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e39" href="#d0e39">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:mhlee@albany.edu" target="_top">mhlee@albany.edu</a> or <a href="mailto:ts@genetics.wustl.edu" target="_top">ts@genetics.wustl.edu</a>
                  						
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e52" href="#d0e52">&#8224;</a></sup>Current address: Department of Biological Sciences, University at Albany, SUNY, Albany, NY 12222
               </p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

