<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Molecular evolution inferences from the C. elegans genome
         
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#molecular_evolution_articleid" title="Molecular evolution inferences from the C. elegans genome&#xA;">
      <link rel="next" href="#sec1" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="molecularevol.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="molecular_evolution_articleid"></a>Molecular evolution inferences from the <span class="emphasis"><em>C. elegans</em></span> genome<sup><a name="d0e9" href="#ftn.d0e9">*</a></sup>
                     
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Asher D. Cutter<em><span class="remark"><sup><a name="d0e34" href="#ftn.d0e34">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="orgname">Department of Ecology &amp; Evolutionary Biology, University of Toronto; </span><span class="address">25 Willcocks St., Toronto, ON, M5S 3B2, Canada
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_nematodeevolecol.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Evolution and ecology
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=332.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Molecular+evolution+inferences+from+the+C.+elegans+genome" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. Mutation</a></span></dt>
               <dt><span class="sect1"><a href="#sec3">3. Recombination</a></span></dt>
               <dt><span class="sect1"><a href="#sec4">4. Natural Selection</a></span></dt>
               <dt><span class="sect1"><a href="#sec5">5. Genetic Drift</a></span></dt>
               <dt><span class="sect1"><a href="#sec6">6. Population dynamics</a></span></dt>
               <dt><span class="sect1"><a href="#sec7">7. Summary</a></span></dt>
               <dt><span class="sect1"><a href="#sec8">8. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#sec9">9. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p>An understanding of evolution at the molecular level requires the simultaneous consideration of the 5 fundamental evolutionary
               processes: mutation, recombination, natural selection, genetic drift, and population dynamic effects. Experimental, comparative
               genomic, and population genetic work in <span class="emphasis"><em>C. elegans</em></span> has greatly expanded our understanding of these core processes, as well as of <span class="emphasis"><em>C. elegans</em></span> biology. This chapter presents a brief overview of some of the most salient features of molecular evolution elucidated by
               the <span class="emphasis"><em>C. elegans</em></span> system.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>Molecular evolution is the study of change-over-time in DNA sequence-based features, be they nucleotide variant frequencies
               in a population or divergence of loci between reproductively isolated lineages. This field has a rich theoretical history,
               and intertwines with classical and molecular population genetics and comparative genomics (<a href="#bib4">Avise, 2004</a>; <a href="#bib49">Fisher, 1958</a>; <a href="#bib52">Gibson and Muse, 2002</a>; <a href="#bib57">Hartl and Clark, 2007</a>; <a href="#bib76">Li, 1997</a>; <a href="#bib80">Lynch, 2007</a>). Facilitated by <span class="emphasis"><em>C. elegans</em></span>' experimental tractability and the early publication of its genome sequence, this species has contributed greatly to our
               understanding of general processes of molecular evolution &#8211; and molecular evolutionary analyses have contributed greatly to
               our understanding of <span class="emphasis"><em>C. elegans</em></span> biology. Like all descriptions of evolution, molecular evolution may be decomposed into 5 fundamental ingredients: mutation,
               recombination, natural selection, genetic drift, and population dynamic effects. In this brief overview to <span class="emphasis"><em>C. elegans</em></span> molecular evolution, I will consider each of these features in turn to provide an introduction to the molecular organization
               and change-over-time of <span class="emphasis"><em>C. elegans</em></span> genome. One broad review (<a href="#bib27">Cutter et al., 2009</a>), several targeted reviews (<a href="#bib20">Coghlan et al., 2006</a>; <a href="#bib113">Thomas, 2008</a>), and WormBook chapters (<a href="http://dx.doi.org/10.1895/wormbook.1.43.1" target="_top">Natural variation and population genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span></a>, <a href="http://dx.doi.org/10.1895/wormbook.1.15.1" target="_top">Nematode genome evolution</a>, <a href="http://dx.doi.org/10.1895/wormbook.1.11.1" target="_top">The phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> and other rhabditids</a> and <a href="http://dx.doi.org/10.1895/wormbook.1.2.1" target="_top">Gene duplications and genetic redundancy in <span class="emphasis"><em>C. elegans</em></span></a>) provide complementary in-depth perspectives on aspects of <span class="emphasis"><em>C. elegans</em></span> molecular evolution, in addition to the articles describing the genomes and genetic maps of <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib7">Barnes et al., 1995</a>; <a href="#bib14"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>) and <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib58">Hillier et al., 2007</a>; <a href="#bib110">Stein et al., 2003</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Mutation
                     </h2>
                  </div>
               </div>
            </div>
            <p>Mutation provides the raw material for evolutionary change. Consequently, a comprehensive understanding of molecular evolution
               requires the characterization of mutational rates, mechanisms, biases, and heterogeneity. Accurate measures of mutational
               properties also are critical for molecular clock approaches to inferring the timing of historical evolutionary events that
               are recorded in patterns of polymorphism and divergence, such as coalescent times within populations, splitting times among
               populations, speciation events, and the timing of gene duplications (<a href="#bib21">Coghlan and Wolfe, 2002</a>; <a href="#bib24">Cutter, 2008</a>; <a href="#bib32">Cutter et al., 2006</a>; <a href="#bib81">Lynch and Conery, 2000</a>).
            </p>
            <p>Among eukaryotes, <span class="emphasis"><em>C. elegans</em></span>' mutation rates and mechanisms are among the most well-characterized, in large-part due to mutation accumulation (MA) experiments
               (<a href="#bib5">Baer et al., 2007</a>). Laboratory MA experiments that enable even moderately deleterious mutations to accrue in worm genomes over the course of
               dozens or hundreds of generations permit the indirect (<a href="#bib6">Baer et al., 2006</a>; <a href="#bib68">Keightley and Bataillon, 2000</a>; <a href="#bib69">Keightley and Caballero, 1997</a>; <a href="#bib115">Vassilieva et al., 2000</a>; <a href="#bib116">Vassilieva and Lynch, 1999</a>) or direct (<a href="#bib38">Denver et al., 2009</a>; <a href="#bib40">Denver et al., 2004a</a>; <a href="#bib41">Denver et al., 2004b</a>; <a href="#bib42">Denver et al., 2000</a>) inference of the average mutation rate across the genome. The most recent incarnation of this work, using whole-genome re-sequencing
               of several MA lines, indicates an average per-generation single-nucleotide mutation rate in the nuclear genome of 2.7 &times; 10<sup>&#8722;9</sup> (<a href="#bib38">Denver et al., 2009</a>), which is roughly 3-times lower than a previous direct-sequencing analysis of MA lines (<a href="#bib41">Denver et al., 2004b</a>). Small indels in the nuclear genome are thought to arise at an average rate per generation that is comparable to the single-nucleotide
               rate (<a href="#bib41">Denver et al., 2004b</a>), although it is now unclear whether this result holds, in light of the most recent point-mutation rate estimates. <a href="#bib41">Denver et al. (2004b)</a> estimated that an average of roughly 1 deleterious mutation arises in the diploid genome per generation, which corroborates
               previous work that suggested that phenotypic estimates of the genomic deleterious mutation rate were much too low (<a href="#bib35">Davies et al., 1999</a>). Different DNA repair pathways influence mutation rates to varying degrees, with a particularly strong role of mismatch
               repair (<a href="#bib39">Denver et al., 2006</a>). In the mitochondrial genome, the mutation rate is inferred to average 9.7 &times; 10<sup>&#8722;8</sup> mutations per site per generation (<a href="#bib42">Denver et al., 2000</a>). Curiously, mitochondrial nucleotide mutational biases differ strikingly among yeast, fly, mouse and worm (<a href="#bib85">Montooth and Rand, 2008</a>).
            </p>
            <p>Short tandem repeat loci (STRs, or, microsatellites) are popular molecular markers (<a href="#bib10">Barri&egrave;re and F&eacute;lix, 2007</a>; <a href="#bib55">Haber et al., 2005</a>; <a href="#bib107">Sivasundar and Hey, 2003</a>), because typically they have high mutation rates that generate relatively high heterozygosity and because most are thought
               to be selectively neutral. STRs mutate by a mechanism distinct from single nucleotide changes, such that slippage of the replication
               machinery generates mutant alleles that differ in the number of repeats by one or more units. Studies of dysfunctional repair
               mutants (<a href="#bib37">Degtyareva et al., 2002</a>) and MA experiments in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib40">Denver et al., 2004a</a>; <a href="#bib93">Phillips et al., 2009</a>; <a href="#bib105">Seyfert et al., 2008</a>) both have informed mutational dynamics of STRs. Longer STRs (loci with more repeats, regardless of the length of the repeat
               unit) mutate faster, and also typically exhibit higher population genetic variation (<a href="#bib93">Phillips et al., 2009</a>). Nearly 30% of observed mutations involve multi-step changes, and longer STRs may have a greater fraction of multi-step
               changes &#8211; i.e., mutation is not consistent with a strict stepwise-mutation model, SMM (<a href="#bib88">Ohta and Kimura, 1973</a>). Because most mutational studies focus on long STRs (which mutate faster and are easier to study), yet most STRs in the
               worm genome are quite short, the incidence of single-step changes for typical STRs may actually be substantially more than
               70% and more closely approximate the SMM.
            </p>
            <p>Copy number variants originate by mutation from duplications, insertions, deletions, and the special case of transposable
               element activity. Such mutations are now recognized to contribute to substantial differences among wild isolates (<a href="#bib83">Maydan et al., 2007</a>) and between species (<a href="#bib110">Stein et al., 2003</a>). Most duplications occur intra-chromosomally, generally are less than 2kb in length (<a href="#bib15">Cavalcanti et al., 2003</a>; <a href="#bib67">Katju and Lynch, 2003</a>; <a href="#bib104">Semple and Wolfe, 1999</a>; <a href="#bib117">Vergara et al., 2009</a>), and arise at a rate much higher than in other taxa (<a href="#bib27">Cutter et al., 2009</a>; <a href="#bib81">Lynch and Conery, 2000</a>). More detailed discussion of duplication in the worm genome is provided elsewhere (<a href="#bib27">Cutter et al., 2009</a>; <a href="http://dx.doi.org/10.1895/wormbook.1.66.1" target="_top">The putative chemoreceptor families of <span class="emphasis"><em>C. elegans</em></span></a>, <a href="http://dx.doi.org/10.1895/wormbook.1.2.1" target="_top">Gene duplications and genetic redundancy in <span class="emphasis"><em>C. elegans</em></span></a>). Rearrangements form another important class of mutation, which, in <span class="emphasis"><em>C. elegans</em></span>, occur primarily within rather than between chromosomes and at a rate substantially higher than in other taxonomic groups
               (<a href="#bib21">Coghlan and Wolfe, 2002</a>; <a href="#bib58">Hillier et al., 2007</a>; <a href="#bib110">Stein et al., 2003</a>; <a href="#bib117">Vergara et al., 2009</a>). Thus, <span class="emphasis"><em>Caenorhabditis</em></span> displays higher mutation rates for several classes of mutations (single-nucleotide, duplication, rearrangement) relative
               to other taxa.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;Recombination
                     </h2>
                  </div>
               </div>
            </div>
            <p>Recombination generates new combinations of alleles from heterozygous individuals, and can facilitate natural selection by
               reducing interference between linked alleles that are subject to selection simultaneously. Consequently, recombination should
               be important for the effective elimination of deleterious alleles (via negative directional, purifying selection) and for
               the fixation of beneficial alleles (via positive directional selection). Because genetically effective recombination requires
               heterozygosity, and inbreeding by self-fertilization and biparental inbreeding destroys heterozygosity, understanding the
               role of recombination for highly selfing <span class="emphasis"><em>C. elegans</em></span> in nature is a major issue for molecular evolution (<a href="#bib94">Phillips, 2006</a>).
            </p>
            <p>Crossover rates vary along <span class="emphasis"><em>C. elegans</em></span> chromosomes in a predictable manner (see <a href="http://dx.doi.org/10.1895/wormbook.1.3.1" target="_top">Karyotype, ploidy and gene dosage</a>): the central half of each chromosome has distinctly low rates of crossover, the arms flanking the chromosome centers experience
               high rates of crossover, and the chromosome tips have virtually no crossover (<a href="#bib7">Barnes et al., 1995</a>; <a href="#bib100">Rockman and Kruglyak, 2009</a>). This pattern is most pronounced on the autosomes, but still detectable on the X sex-chromosome. <span class="emphasis"><em>C. briggsae</em></span>'s chromosomes exhibit a comparable pattern of crossover (Cutter and Choi, unpublished data; <a href="#bib58">Hillier et al., 2007</a>), so this may represent the ancestral and universal chromosomal pattern of crossover in <span class="emphasis"><em>Caenorhabditis</em></span> &#8211; but confirmation in gonochoristic species like <span class="emphasis"><em>C. remanei</em></span> is necessary. Chromosomes experience essentially complete interference, such that just a single crossover occurs per chromosome
               in meiosis (excepting the hemizygous X in males) (<a href="#bib124">Zetka, 2009</a>), and these crossovers do not appear to occur at recombination &#8220;hotspots&#8221; as in yeast and mammals.
            </p>
            <p>The chromosomal distribution of crossover rates in <span class="emphasis"><em>C. elegans</em></span> has important implications for nucleotide polymorphism and divergence. Notably, single-nucleotide differences and indel polymorphisms
               among strains are more common in regions of high recombination (<a href="#bib29">Cutter and Payseur, 2003</a>; <a href="#bib74">Koch et al., 2000</a>; <a href="#bib83">Maydan et al., 2007</a>). It has been proposed that this might reflect higher rates of mutation in regions of high recombination and/or that natural
               selection more dramatically impacts regions of low recombination (due to longer spans of linkage associated with selected
               sites), resulting in depressed levels of polymorphism. Indeed, divergence relative to <span class="emphasis"><em>C. briggsae</em></span> at silent sites is greater in high recombination regions (<a href="#bib29">Cutter and Payseur, 2003</a>), as are rates of chromosomal rearrangement (<a href="#bib58">Hillier et al., 2007</a>; <a href="#bib110">Stein et al., 2003</a>), consistent with a higher mutation rate in regions experiencing elevated crossover rates. However, whole-genome sequencing
               of MA lines identified no significant heterogeneity along the length of chromosomes with respect to mutation rate, based on
               &#8764;400 detected mutations (<a href="#bib38">Denver et al., 2009</a>). Natural selection could potentially shape chromosomal distributions of genetic variation in <span class="emphasis"><em>C. elegans</em></span> as well (<a href="#bib29">Cutter and Payseur, 2003</a>). &#8220;Background selection&#8221; (<a href="#bib17">Charlesworth et al., 1993</a>; <a href="#bib62">Hudson and Kaplan, 1995</a>) against deleterious mutations was proposed as a plausible selective explanation for low polymorphism in regions that experience
               little recombination, though genetic hitchhiking due to positive directional selection also might contribute to the pattern
               (<a href="#bib1">Andolfatto, 2001</a>; <a href="#bib109">Smith and Haigh, 1974</a>; <a href="#bib121">Wiehe and Stephan, 1993</a>). The proteins encoded in chromosomal centers (i.e. low crossover rate regions) tend to be more highly conserved across taxa
               (<a href="#bib14"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>; <a href="#bib92">Parkinson et al., 2004</a>), although the rate of amino acid substitution (K<sub>A</sub>) does not strongly correlate with crossover rate once the substitution rate at synonymous sites (K<sub>S</sub>) is properly accounted for (including adjustment for selection on synonymous sites). Analysis of this latter point should
               be revisited with species comparisons more closely related than <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>, between which divergence at synonymous sites is saturated.
            </p>
            <p>Gene conversion is an important process shaping linkage disequilibrium at very small scales within genomes (<a href="#bib2">Andolfatto and Nordborg, 1998</a>). Gene conversion as a mechanism of recombination remains fairly poorly understood generally, and in <span class="emphasis"><em>C. elegans</em></span> in particular. Specific examples of allelic gene conversion have been reported (<a href="#bib84">Moerman and Baillie, 1979</a>; <a href="#bib97">Rattray and Rose, 1988</a>; <a href="#bib101">Rose and Baillie, 1980</a>), as well as evidence of gene conversion among gene duplicates or multi-gene families in the genome (<a href="#bib66">Katju et al., 2008</a>; <a href="#bib104">Semple and Wolfe, 1999</a>), and from population polymorphism data (in <span class="emphasis"><em>C. remanei</em></span>) (<a href="#bib25">Cutter, 2008</a>). Gene conversion is now widely implemented for experimental purposes in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib51">Fr&oslash;kjaer-Jensen et al., 2008</a>; <a href="#bib98">Robert et al., 2008</a>), which holds promise for furthering our understanding of the process of gene conversion and its implications for molecular
               evolution. However, no genome-wide contrast of rates of crossover versus non-crossover (i.e. gene conversion) recombination
               have been conducted, as in yeast (<a href="#bib82">Mancera et al., 2008</a>).
            </p>
            <p>No conclusive evidence of mitochondrial recombination has been reported for <span class="emphasis"><em>C. elegans</em></span>, despite support for mitochondrial recombination in other nematodes (<a href="#bib79">Lunt and Hyman, 1997</a>; <a href="#bib95">Piganeau et al., 2004</a>) and a potential example in <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib61">Howe and Denver, 2008</a>).
            </p>
            <p>Genetically effective recombination occurs only rarely in nature, such that large blocks of chromosomes experience linkage
               disequilibrium among wild isolates (<a href="#bib8">Barri&egrave;re and F&eacute;lix, 2005</a>; <a href="#bib23">Cutter, 2006</a>; <a href="#bib55">Haber et al., 2005</a>; <a href="#bib100">Rockman and Kruglyak, 2009</a>). Linkage disequilibrium (LD) takes an average of 3.3Mb to decay by half, and significant LD occurs between loci on different
               chromosomes. Very low rates of outcrossing in nature (on the order of 10<sup>&#8722;3</sup> to 10<sup>&#8722;4</sup> per generation) are thought to be the primary cause of this LD, although complex population structure may also contribute.
               Despite the strong LD across the genome, clearly some recombination has occurred among multi-locus genotypes, meaning that
               there is no single bifurcating gene-tree that describes the population history of wild isolates of <span class="emphasis"><em>C. elegans</em></span>. It should also be noted that microsatellite heterozygosity yields higher estimates of outcrossing rates, and there might
               be important variation among local populations in outcrossing rates (<a href="#bib108">Sivasundar and Hey, 2005</a>), perhaps a consequence of harsh or variable local conditions (<a href="#bib86">Morran et al., 2009</a>; <a href="#bib87">Morran et al., 2009</a>). However, selection may generally disfavor recombinant genotypes (<a href="#bib10">Barri&egrave;re and F&eacute;lix, 2007</a>; <a href="#bib43">Dolgin et al., 2007</a>). Extreme levels of self-fertilization and lack of effective recombination, such as observed in <span class="emphasis"><em>C. elegans</em></span>, can have dire consequences for the long-term persistence of populations, in the absence of compensatory mutations (<a href="#bib78">Loewe and Cutter, 2008</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Natural Selection
                     </h2>
                  </div>
               </div>
            </div>
            <p>A principle aim of molecular evolutionists is to identify and characterize the targets of adaptive evolution across the genome.
               However, to infer the action of selection, one must first refute the possibility that neutral evolutionary forces (i.e. the
               4 evolutionary processes other than natural selection) could have caused the observed patterns. This is the reason that molecular
               evolutionists and molecular population geneticists so often are interested in the evolutionary dynamics of loci that are expected
               to be unaffected by selection; understanding the evolution of such neutral sites provides a null model against which to test
               for evidence of selection.
            </p>
            <p>Population genetic polymorphism and inter-specific divergence are the two main tools for inferring selection at the molecular
               level. Unfortunately, the low diversity coupled with high LD due to the complex population processes in <span class="emphasis"><em>C. elegans</em></span> (selfing, structured populations) render polymorphism as largely uninformative for localizing the gene targets of selection
               in this species. Consequently, most work has focused on analyses of divergence. When assessing nucleotide divergence between
               species, it is common practice to quantify the number of substitutions per basepair in the site class under consideration
               (e.g. replacement sites versus synonymous sites within coding regions) and to account for the potential for multiple mutational
               hits at a given site to have occurred. This is done with the various ways of computing K<sub>A</sub> (or, d<sub>N</sub>; the number of replacement substitutions per replacement site) and K<sub>S</sub> (or, d<sub>S</sub>; the number of synonymous substitutions per synonymous site) statistics (<a href="#bib123">Yang and Bielawski, 2000</a>). Dividing K<sub>A</sub> by K<sub>S</sub> provides a metric that relates the rate of change at amino acid-changing sites relative to the rate expected for putatively
               neutrally evolving synonymous sites. In other words, differences in K<sub>S</sub> are meant to reflect differences among loci in the underlying local rate of mutation that could, in itself, give rise to
               differences in rates of protein divergence. However, because synonymous sites for highly expressed genes have not been selectively
               neutral in the lineages leading to species of <span class="emphasis"><em>Caenorhabditis</em></span>, it is further necessary to correct K<sub>S</sub> values for the effects of selection on codon usage (<a href="#bib24">Cutter, 2008</a>).
            </p>
            <p>It is assumed implicitly in most divergence calculations that within-species polymorphism is negligible relative to fixed
               differences between species, which is a reasonable assumption for comparisons of known <span class="emphasis"><em>Caenorhabditis</em></span> species (which are highly divergent at the molecular level), but may not be reasonable in other taxa (e.g. some <span class="emphasis"><em>Drosophila</em></span> species pairs, human-chimpanzee) and may become an important issue as new, closely-related species of <span class="emphasis"><em>Caenorhabditis</em></span> are discovered. Note also that some methods of computing divergence are pairwise, whereas others use a phylogenetic tree
               to generate lineage-specific rate estimates separately for each branch in the tree (<a href="#figure1">Figure 1</a>). Such lineage-specific measures of divergence require that the phylogenetic relationships among species are well-defined
               (<a href="#bib71">Kiontke et al., 2007</a>; <a href="http://dx.doi.org/10.1895/wormbook.1.11.1" target="_top">The phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> and other rhabditids</a>); as genome sequences become available for additional <span class="emphasis"><em>Caenorhabditis</em></span> and other nematodes, new developments in multi-locus species-tree inference should prove useful in delineating the tempo
               of speciation (<a href="#bib36">Degnan and Rosenberg, 2009</a>).
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="540">

<tr>
  <td align="center"><p><a href="molecularevol_fig1.jpg"><img src="molecularevol_fig1_s.jpg" border="2" align="middle" alt="figure 1"></a>
  </td>
</tr>

               </table>
               <div class="caption">
                  <p><b>Figure 1. Diagrammatic representation of polymorphism and divergence.&nbsp;</b>Dotted lines delimit independently evolving lineages (species); solid lines trace the genealogy of an ortholog between species.
                     Polymorphism at a locus is shown within species B and C, as represented by a genealogy in each species that coalesces into
                     the single common ancestor of the extant polymorphism within that lineage. Only a single representative copy of the locus
                     is represented for species A. The expected coalescent time for extant polymorphism is 4N<sub>e</sub> generations. Divergence between distantly-related species typically involves comparison of a single representative sequence
                     for a locus from each species. For example, if the lengths of branches are proportional to the number of mutations that have
                     accumulated along them, then divergence between A and B would be the sum of the lengths of the dark and light blue, purple,
                     and orange branches. Using the colored copies of the locus as representatives from each of these 3 species, one could calculate
                     lineage-specific rates for species B and C. Lineage-specific divergence for C (orange) would be equivalent to summing the
                     lengths of the red and pink branches, if the position of their common ancestry could be inferred accurately. Differences in
                     mutation rates and/or generation times can result in variable lineage-specific branch-lengths.
                  </p>
               </div>
            </div>
            <p>The dominant form of natural selection in the genome, of course, is purifying selection (also termed negative directional
               selection), which acts to conserve sequence features and, in coding sequences, yields K<sub>A</sub>/K<sub>S</sub> &lt; 1 (<a href="#bib110">Stein et al., 2003</a>). For example, purifying selection exerts particularly potent effects on genes that confer sterility phenotypes when knocked-down
               by RNAi (<a href="#bib30">Cutter et al., 2003</a>). Also, adult-expressed genes tend to evolve more rapidly than those expressed predominantly in larvae, which is consistent
               with stronger purifying selection on pre-reproductive stages, as predicted by some theoretical models of aging (<a href="#bib31">Cutter and Ward, 2005</a>). Significant portions of non-coding sequence also are subject to purifying selection in <span class="emphasis"><em>Caenorhabditis</em></span> (<a href="#bib70">Kent and Zahler, 2000</a>; <a href="#bib106">Shabalina and Kondrashov, 1999</a>; <a href="#bib120">Webb et al., 2002</a>), which has been applied to successfully identify conserved regulatory elements (<a href="#bib20">Coghlan et al., 2006</a>). However, many functionally important sequences may evolve rapidly, and therefore be missed in genomic surveys of conserved
               elements.
            </p>
            <p>Detecting positive selection can be a difficult task, particularly in <span class="emphasis"><em>C. elegans</em></span>, because many of the most powerful methods are not accessible or robust in this system due to the high sequence divergence
               between species in combination with <span class="emphasis"><em>C. elegans</em></span>' complex demographic history (<a href="#bib75">Kreitman, 2000</a>). Comparative genomic analysis with newly identified species with closer phylogenetic positions to known species will help
               rectify this problem in the future. Nevertheless, a variety of intriguing general molecular evolutionary patterns resulting
               from natural selection reveal themselves under close inspection.
            </p>
            <p>Within multi-gene families, it is possible to identify genes with an excess of replacement-site substitutions relative to
               synonymous-site substitutions (i.e. K<sub>A</sub>/K<sub>S</sub> &gt; 1), which is indicative of repeated positive directional selection on peptide sequence (<a href="#bib123">Yang and Bielawski, 2000</a>). Specifically, the D subfamily of the ATP-binding cassette family of proteins shows evidence of such adaptive evolution
               in its history (<a href="#bib125">Zhao et al., 2007</a>), as do some members of the SRZ subfamily of putative chemoreceptors (<a href="#bib114">Thomas et al., 2005</a>), some lysozyme genes (<a href="#bib103">Schulenburg and Boehnisch, 2008</a>), and some antimicrobial peptides (<a href="#bib96">Pujol et al., 2008</a>). Transcription factors also tend to evolve more rapidly than other gene classes (<a href="#bib56">Haerty et al., 2008</a>; <a href="#bib63">Jovelin, 2009</a>), although strictly speaking the data for <span class="emphasis"><em>Caenorhabditis</em></span> make it difficult to rule out relaxed selection as a possible cause of elevated rates of replacement-site divergence. Similarly,
               sperm-related genes tend to have more rapid rates of protein evolution than other types of genes, and also experience disproportionate
               gene loss and duplication (<a href="#bib3">Artieri et al., 2008</a>; <a href="#bib31">Cutter and Ward, 2005</a>); this could be a signature of sexual selection in the obligately outbreeding ancestors of <span class="emphasis"><em>C. elegans</em></span>. As gene networks and genetic pathways become better-elucidated, molecular evolution analysis in the frameworks of systems
               biology and evo-devo may provide interesting insights (<a href="#bib63">Jovelin, 2009</a>; <a href="#bib64">Jovelin et al., 2009</a>; <a href="#bib126">Zou et al., 2008</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;Genetic Drift
                     </h2>
                  </div>
               </div>
            </div>
            <p>Stochastic change in allele frequency describes the process of genetic drift (<a href="#bib57">Hartl and Clark, 2007</a>), and this mechanism of evolutionary change is important for alleles that are very rare and/or subject to very weak natural
               selection. Drift at a given locus is most powerful when populations are composed of few breeding individuals; the effect of
               genetic drift on an allele is inversely proportional to N<sub>e</sub>s (selection coefficient s). N<sub>e</sub>, the effective population size, is a unifying concept in evolution that defines the size of populations not in terms of census
               size but in terms of the rate of genetic change due to drift (<a href="#bib16">Charlesworth, 2009</a>). Since the origin of the highly-selfing form of hermaphroditism that <span class="emphasis"><em>C. elegans</em></span> now experiences in nature, the role of genetic drift in molecular evolution will have been increased dramatically throughout
               <span class="emphasis"><em>C. elegans</em></span>' genome (a decrease in N<sub>e</sub> relative to obligately outcrossing ancestors). Phrased another way, the relative importance of selection has been relaxed
               in the <span class="emphasis"><em>C. elegans</em></span> lineage since the origin of extreme selfing.
            </p>
            <p>The evolution of biased codon usage provides an exceptional example of the interplay between mutation, selection, and drift.
               These three forces operate with similar magnitudes of effect to determine the evolutionary fate of alternative synonymous
               codons (<a href="#bib13">Bulmer, 1991</a>; <a href="#bib46">Duret, 2002</a>). Mutational pressures and natural selection clearly both play important roles in the biased usage of synonymous codons in
               <span class="emphasis"><em>Caenorhabditis</em></span> (<a href="#bib33">Cutter et al., 2008</a>; <a href="#bib45">Duret, 2000</a>; <a href="#bib47">Duret and Mouchiroud, 1999</a>; <a href="#bib111">Stenico et al., 1994</a>), and nematodes generally (<a href="#bib28">Cutter et al., 2006</a>). However, the drastic reduction in the effective number of breeding individuals in <span class="emphasis"><em>C. elegans</em></span>, following the origin of a highly selfing lifestyle from obligately outbreeding ancestors (<a href="#bib18">Cho et al., 2004</a>; <a href="#bib73">Kiontke et al., 2004</a>), will have raised the relative importance of genetic drift in mediating molecular evolution. This yields the prediction
               that adaptive codon usage bias should be less prevalent in the <span class="emphasis"><em>C. elegans</em></span> genome compared to its relatives, with the extent of codon bias decay being proportional to the time over which such relaxed
               selection has elapsed. Based on this logic, and the observation that codon bias differs only subtly between <span class="emphasis"><em>C. elegans</em></span> and other species in the genus, <a href="#bib33">Cutter et al. (2008)</a> concluded that the extreme form of selfing seen today in <span class="emphasis"><em>C. elegans</em></span> must have originated in the not-too-distant past.
            </p>
            <p>The activity of transposable elements (TEs) most typically exerts detrimental consequences on organismal fitness (<a href="#bib12">Begin and Schoen, 2007</a>), but the ability of natural selection to eliminate such deleterious mutational events depends on the relative role of genetic
               drift. Again, the drastically reduced population size of <span class="emphasis"><em>C. elegans</em></span> relative to its ancestor is predicted to have relaxed selection against the elimination of TE insertions. Population frequencies
               for 32 Tc1 insertion sites are consistent with relaxed selection in <span class="emphasis"><em>C. elegans</em></span> (i.e. a dominant role for genetic drift in contemporary natural populations), in stark contrast to evidence of purifying
               selection against mTcre1 elements in <span class="emphasis"><em>C. remanei</em></span> (<a href="#bib44">Dolgin et al., 2008</a>).
            </p>
            <p>The Bergerac strains of <span class="emphasis"><em>C. elegans</em></span> (e.g. BO = RW7000) were isolated from nature by Nigon, and exhibit an exceptionally high load of Tc1 elements (&gt;300) in their
               genomes (<a href="#bib48">Emmons et al., 1983</a>; <a href="#bib60">Hodgkin and Doniach, 1997</a>; <a href="#bib77">Liao et al., 1983</a>). Several other <span class="emphasis"><em>C. elegans</em></span> strains at the <span class="emphasis"><em>Caenorhabditis</em></span> Genetics Center similarly have high loads of Tc1 (<a href="#bib60">Hodgkin and Doniach, 1997</a>), although all of these appear to reflect spurious mislabeling and/or lab crosses with Bergerac during their history in the
               lab (<a href="#bib100">Rockman and Kruglyak, 2009</a>). It is tempting to speculate that relaxed selection on transposon activity in <span class="emphasis"><em>C. elegans</em></span> enabled the proliferation of Tc1 in Bergerac, which presumably eventually led to its extinction, as other strains with high
               genomic loads of Tc1 have not been isolated in recent years of extensive wild sampling effort. However, the isolation of Bergerac
               in the 1940's followed by decades of laboratory culture (<a href="#bib60">Hodgkin and Doniach, 1997</a>) raises the distinct possibility that the high abundance of Tc1 elements in its genome occurred after its isolation from
               nature.
            </p>
            <p>Relaxed selection on chemoreceptor genes similarly might be responsible for the segregation of premature stop codon alleles
               being common in nature (<a href="#bib112">Stewart et al., 2005</a>). The evolution of genes underlying traits associated with outcrossing and sexual selection, such as copulatory plugs (<a href="#bib89">Palopoli et al., 2008</a>), also likely currently are governed largely by genetic drift.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;Population dynamics
                     </h2>
                  </div>
               </div>
            </div>
            <p>In addition to the important impacts of mutation, recombination, selection, and drift on the evolution of inherited molecules,
               the details of population demography (population subdivision, migration, changes in size of the breeding population, mode
               of reproduction) also influence the ultimate evolutionary fate of alleles: loss or fixation. Molecular population genetics
               has been instrumental in tracing the ancestry of humans across the globe (<a href="#bib22">Conrad et al., 2006</a>; <a href="#bib102">Rosenberg et al., 2002</a>), and similar approaches have yielded generous insights into the dynamics of contemporary populations for <span class="emphasis"><em>C. elegans</em></span> and its relatives (<a href="http://dx.doi.org/10.1895/wormbook.1.43.1" target="_top">Natural variation and population genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span></a>, <a href="#bib27">Cutter et al., 2009</a>; <a href="#bib50">Fitch, 2005</a>; <a href="#bib94">Phillips, 2006</a>).
            </p>
            <p><span class="emphasis"><em>C. elegans</em></span> in nature exhibits strong evidence of subdivided population structure, with many subpopulations that display little-if-any
               isolation by distance (<a href="#bib8">Barri&egrave;re and F&eacute;lix, 2005</a>; <a href="#bib23">Cutter, 2006</a>; <a href="#bib55">Haber et al., 2005</a>; <a href="#bib108">Sivasundar and Hey, 2005</a>). This implies migration across both local and global scales, perhaps mediated by humans or other animal vectors. Migration
               and/or colonization of the globe may occur sufficiently frequently or recently that no obvious patterns of local adaptation
               are now evident for <span class="emphasis"><em>C. elegans</em></span>, as inferred from molecular population genetic analysis and lack of associations between phenotype, genotype, and geography
               (<a href="#bib60">Hodgkin and Doniach, 1997</a>; <a href="#bib107">Sivasundar and Hey, 2003</a>). Some local subpopulations appear to be stable over time, while others go extinct and are recolonized (<a href="#bib10">Barri&egrave;re and F&eacute;lix, 2007</a>). These observations make metapopulation dynamic models and many-demes models of coalescent genealogical history relevant
               to understanding its genetic variation (<a href="#bib23">Cutter, 2006</a>; <a href="#bib90">Pannell, 2003</a>; <a href="#bib91">Pannell and Charlesworth, 1999</a>; <a href="#bib118">Wakeley, 1999</a>; <a href="#bib119">Wakeley and Aliacar, 2001</a>). However, it remains unclear precisely how to define a &#8220;subpopulation&#8221; for these nematodes in an ecological sense: do the
               worms in an ephemeral rotting apple comprise a subpopulation? Or is the soil under the apple tree, present for years or decades,
               home to a subpopulation? Or the grounds of the orchard in which the apple tree resides? Integrating <span class="emphasis"><em>C. elegans</em></span> ecological context &#8211; which presently remains poorly known &#8211; with evolutionary analysis should prove helpful in elucidating
               the dynamics of population processes.
            </p>
            <p>While molecular approaches find no clear support for global population expansion, and only limited support for population
               contraction (<a href="#bib107">Sivasundar and Hey, 2003</a>), the power to detect changes in effective population size in <span class="emphasis"><em>C. elegans</em></span> very recent past is constrained by the complex population structure. Nevertheless, the substantially lower levels of genetic
               diversity (~20-fold) relative to obligately outcrossing species imply that it has experienced a strong genetic bottleneck
               since the origin of highly selfing hermaphroditism, possibly due to both demographic and selective causes (<a href="#bib27">Cutter et al., 2009</a>; <a href="#bib53">Graustein et al., 2002</a>).
            </p>
            <p>Comparisons of molecular variation for multiple populations across species of <span class="emphasis"><em>Caenorhabditis</em></span> that exhibit similar (e.g., <span class="emphasis"><em>C. briggsae</em></span>, <span class="emphasis"><em>C</em></span>. sp. 11) or different (e.g., <span class="emphasis"><em>C. remanei</em></span>, <span class="emphasis"><em>C. brenneri</em></span>, <span class="emphasis"><em>C</em></span>. sp. 5) modes of reproduction to <span class="emphasis"><em>C. elegans</em></span> also will help illuminate population dynamic processes. For example, despite the presence of selfing hermaphroditism and
               a global distribution like <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em>C. briggsae</em></span> exhibits strong patterns of phylogeography (reviewed in <a href="#bib27">Cutter et al., 2009</a>; <a href="#bib28">Cutter et al., 2006</a>; <a href="#bib34">Cutter et al., 2010</a>; <a href="#bib61">Howe and Denver, 2008</a>). Current evidence also points to much greater levels of genetic variation in obligately outbreeding female-male species
               (<a href="#bib11">Barriere et al., 2009</a>; reviewed in <a href="#bib27">Cutter et al., 2009</a>; <a href="#bib63">Jovelin, 2009</a>; <a href="#bib64">Jovelin et al., 2009</a>). These observations point to important differences in the demography of these species that confer unique opportunities to
               exploit the peculiarities of a given species to inform the processes of molecular evolution and the biology of <span class="emphasis"><em>Caenorhabditis</em></span>.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec7"></a>7.&nbsp;Summary
                     </h2>
                  </div>
               </div>
            </div>
            <p>Study of <span class="emphasis"><em>C. elegans</em></span> has greatly informed understanding of the fundamental forces of evolution operating at the molecular level, and the application
               of evolutionary theory to <span class="emphasis"><em>C. elegans</em></span> genome has benefited understanding of this exquisite organism. Mutation perhaps is the best characterized aspect of molecular
               evolution in <span class="emphasis"><em>C. elegans</em></span>. However, the great wealth of knowledge on mutation is matched by a relatively nascent understanding of the generality of
               mutational characteristics across species &#8211; and across greater phylogenetic depths. Despite the advances to molecular evolution
               made possible by research on <span class="emphasis"><em>C. elegans</em></span>, it is now clear that investigation of other species of <span class="emphasis"><em>Caenorhabditis</em></span> will provide complimentary and novel insights that are not possible by studying <span class="emphasis"><em>C. elegans</em></span> alone (<a href="#bib54">Haag et al., 2007</a>; <a href="#bib65">Kammenga et al., 2008</a>). In particular, the roles of ecology and adaptation in shaping genetic variation, genomic heterogeneity, phenotypic variation,
               and phylogeography in the lineage that gave rise to <span class="emphasis"><em>C. elegans</em></span> and its relatives will benefit profoundly from studies of obligately outcrossing species and from further comparative analysis
               of the repeated evolution of the selfing hermaphroditic lifestyle. The feasibility of resequencing entire <span class="emphasis"><em>Caenorhabditis</em></span> genomes and transcriptomes with next-generation technologies promises advances in molecular evolutionary understanding with
               this group. For example, we should all anticipate rapid genome data acquisition for newly discovered species with closer phylogenetic
               ties to other known taxa (e.g., <span class="emphasis"><em>C</em></span>. sp. 9), population genomics analysis of outcrossing species (e.g., <span class="emphasis"><em>C. remanei</em></span>) to infer targets of natural selection from patterns of population polymorphism, and large-scale functional assays in non-<span class="emphasis"><em>elegans</em></span> species. The scope of such advances, however, currently is limited by taxon sampling in <span class="emphasis"><em>Caenorhabditis</em></span> to species that are rather distantly-related &#8211; collection and identification of new species in the genus is paramount to
               long-term progress in characterizing molecular evolutionary phenomena with <span class="emphasis"><em>Caenorhabditis</em></span>.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec8"></a>8.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>A.D.C. is supported by a Canada Research Chair in Evolutionary Genomics and the Natural Sciences and Engineering Research
               Council of Canada. Thanks to R. Jovelin and two anonymous reviewers for their comments on the manuscript.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec9"></a>9.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     <span class="bibliomisc">Andolfatto, P. (2001). Adaptive hitchhiking effects on genome variability. Curr. Opin. Genet. Dev. <span class="emphasis"><em>11</em></span>, 635-641.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11682306&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     <span class="bibliomisc">Andolfatto, P., and Nordborg, M. (1998). The effect of gene conversion on intralocus associations. Genetics <span class="emphasis"><em>148</em></span>, 1397-1399.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9539452&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     <span class="bibliomisc">Artieri, C. G., Haerty, W., Gupta, B. P., and Singh, R. S. (2008). Sexual selection and maintenance of sex: evidence from
                        comparisons of rates of genomic accumulation of mutations and divergence of sex-related genes in sexual and hermaphroditic
                        species of <span class="emphasis"><em>Caenorhabditis</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>25</em></span>, 972-979.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18281268&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msn046" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     <span class="bibliomisc">Avise, J. C. (2004). Molecular Markers, Natural History, and Evolution, 2nd ed. (Sunderland, MA, Sinauer).</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     <span class="bibliomisc">Baer, C. F., Miyamoto, M. M., and Denver, D. R. (2007). Mutation rate variation in multicellular eukaryotes: causes and consequences.
                        Nat. Rev. Genet. <span class="emphasis"><em>8</em></span>, 619-631.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17637734&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg2158" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     <span class="bibliomisc">Baer, C. F., Phillips, N., Ostrow, D., Avalos, A., Blanton, D., Boggs, A., Keller, T., Levy, L., and Mezerhane, E. (2006).
                        Cumulative effects of spontaneous mutations for fitness in <span class="emphasis"><em>Caenorhabditis</em></span>: role of genotype, environment and stress. Genetics <span class="emphasis"><em>174</em></span>, 1387-1395.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16888328&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.106.061200" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barnes, T. M., Kohara, Y., Coulson, A., and Hekimi, S. (1995). Meiotic recombination, noncoding DNA and genomic organization
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>141</em></span>, 159-179.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8536965&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barri&egrave;re, A., and F&eacute;lix, M.-A. (2005). High local genetic diversity and low outcrossing rate in <span class="emphasis"><em>Caenorhabditis elegans</em></span> natural populations. Curr. Biol. <span class="emphasis"><em>15</em></span>, 1176-1184.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16005289&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.06.022" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barri&egrave;re, A., and F&eacute;lix, M.-A. (2005). Natural variation and population genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. In WormBook, The <span class="emphasis"><em>C. elegans</em></span> Research Community, ed., <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barri&egrave;re, A., and F&eacute;lix, M.-A. (2007). Temporal dynamics and linkage disequilibrium in natural <span class="emphasis"><em>Caenorhabditis elegans</em></span> populations. Genetics <span class="emphasis"><em>176</em></span>, 999-1011.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17409084&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.106.067223" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     <span class="bibliomisc">Barri&egrave;re, A., Yang, S. P., Pekarek, E., Thomas, C. G., Haag, E. S., and Ruvinsky, I. (2009). Detecting heterozygosity in shotgun
                        genome assemblies: Lessons from obligately outcrossing nematodes. Genome Res. <span class="emphasis"><em>19</em></span>, 470-480.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19204328&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.081851.108" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     <span class="bibliomisc">Begin, M., and Schoen, D. J. (2007). Transposable elements, mutational correlations, and population divergence in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Evolution <span class="emphasis"><em>61</em></span>, 1062-1070.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17492961&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1111/j.1558-5646.2007.00097.x" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     <span class="bibliomisc">Bulmer, M. (1991). The selection-mutation-drift theory of synonymous codon usage. Genetics <span class="emphasis"><em>129</em></span>, 897-907.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1752426&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     <span class="bibliomisc"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium (1998). Genome sequence of the nematode <span class="emphasis"><em>C. elegans</em></span>: a platform for investigating biology. Science <span class="emphasis"><em>282</em></span>, 2012-2018.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851916&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cavalcanti, A. R., Ferreira, R., Gu, Z., and Li, W. H. (2003). Patterns of gene duplication in <span class="emphasis"><em>Saccharomyces cerevisiae</em></span> and <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Mol. Evol. <span class="emphasis"><em>56</em></span>, 28-37.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12569420&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s00239-002-2377-2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     <span class="bibliomisc">Charlesworth, B. (2009). Fundamental concepts in genetics: Effective population size and patterns of molecular evolution and
                        variation. Nat. Rev. Genet. <span class="emphasis"><em>10</em></span>, 195-205.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19204717&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg2526" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     <span class="bibliomisc">Charlesworth, B., Morgan, M. T., and Charlesworth, D. (1993). The effect of deleterious mutations on neutral molecular variation.
                        Genetics <span class="emphasis"><em>134</em></span>, 1289-1303.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8375663&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cho, S., Jin, S. W., Cohen, A., and Ellis, R. E. (2004). A phylogeny of <span class="emphasis"><em>Caenorhabditis</em></span> reveals frequent loss of introns during nematode evolution. Genome Res. <span class="emphasis"><em>14</em></span>, 1207-1220.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15231741&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.2639304" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     <span class="bibliomisc">Coghlan, A. (September 7, 2005). Nematode genome evolution. In WormBook, The <span class="emphasis"><em>C. elegans</em></span> Research Community, ed., <a href="http://www.wormbook.org/" target="_top">http://www.wormbook.org</a>.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     <span class="bibliomisc">Coghlan, A., Stajich, J. E., and Harris, T. W. (2006). Comparative genomics in <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em>C. briggsae</em></span>, and other <span class="emphasis"><em>Caenorhabditis</em></span> species. In <span class="emphasis"><em>C. elegans</em></span>: Methods and Applications, K. Strange, ed. (Totowa, NJ, Humana Press Inc.), pp. 13-29.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     <span class="bibliomisc">Coghlan, A., and Wolfe, K. H. (2002). Fourfold faster rate of genome rearrangement in nematodes than in <span class="emphasis"><em>Drosophila</em></span>. Genome Res. <span class="emphasis"><em>12</em></span>, 857-867.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12045140&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.172702" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     <span class="bibliomisc">Conrad, D. F., Jakobsson, M., Coop, G., Wen, X. Q., Wall, J. D., Rosenberg, N. A., and Pritchard, J. K. (2006). A worldwide
                        survey of haplotype variation and linkage disequilibrium in the human genome. Nat. Genet. <span class="emphasis"><em>38</em></span>, 1251-1260.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17057719&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1911" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D. (2006). Nucleotide polymorphism and linkage disequilibrium in wild populations of the partial selfer <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>172</em></span>, 171-184.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16272415&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.105.048207" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D. (2008). Divergence times in <span class="emphasis"><em>Caenorhabditis</em></span> and <span class="emphasis"><em>Drosophila</em></span> inferred from direct estimates of the neutral mutation rate. Mol. Biol. Evol. <span class="emphasis"><em>25</em></span>, 778-786.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18234705&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msn024" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D. (2008). Multilocus patterns of polymorphism and selection across the X-chromosome of <span class="emphasis"><em>Caenorhabditis remanei</em></span>. Genetics <span class="emphasis"><em>178</em></span>, 1661-1672.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18245859&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.107.085803" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D., Dey, A., and Murray, R. L. (2009). Evolution of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome. Mol. Biol. Evol. <span class="emphasis"><em>26</em></span>, 1199-1234.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19289596&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msp048" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D., F&eacute;lix, M.-A., Barriere, A., and Charlesworth, D. (2006). Patterns of nucleotide polymorphism distinguish temperate
                        and tropical wild isolates of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Genetics <span class="emphasis"><em>173</em></span>, 2021-2031.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16783011&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.106.058651" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D., and Payseur, B. A. (2003). Selection at linked sites in the partial selfer <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>20</em></span>, 665-673.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12679551&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msg072" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D., Payseur, B. A., Salcedo, T., Estes, A. M., Good, J. M., Wood, E., Hartl, T., Maughan, H., Strempel, J., Wang,
                        B., <span class="emphasis"><em>et al.</em></span> (2003). Molecular correlates of genes exhibiting RNAi phenotypes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome Res. <span class="emphasis"><em>13</em></span>, 2651-2657.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14656969&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.1659203" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D., and Ward, S. (2005). Sexual and temporal dynamics of molecular evolution in <span class="emphasis"><em>C. elegans</em></span> development. Mol. Biol. Evol. <span class="emphasis"><em>22</em></span>, 178-188.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15371532&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msh267" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D., Wasmuth, J., and Blaxter, M. L. (2006). The evolution of biased codon and amino acid usage in nematode genomes.
                        Mol. Biol. Evol. <span class="emphasis"><em>23</em></span>, 2303-2315.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16936139&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msl097" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D., Wasmuth, J. D., and Washington, N. L. (2008). Patterns of molecular evolution in <span class="emphasis"><em>Caenorhabditis</em></span> preclude ancient origins of selfing. Genetics <span class="emphasis"><em>178</em></span>, 2093-2104.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18430935&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.107.085787" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                     <span class="bibliomisc">Cutter, A. D., Yan, W., Tsvetkov, N., Sunil, S., and F&eacute;lix, M. A. (2010). Molecular population genetics and phenotypic sensitivity
                        to ethanol for a globally diverse sample of the nematode <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Mol. Ecol.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                     <span class="bibliomisc">Davies, E. K., Peters, A. D., and Keightley, P. D. (1999). High frequency of cryptic deleterious mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>285</em></span>, 1748-1751.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10481013&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                     <span class="bibliomisc">Degnan, J. H., and Rosenberg, N. A. (2009). Gene tree discordance, phylogenetic inference and the multispecies coalescent.
                        Trends Ecol. Evol. <span class="emphasis"><em>24</em></span>, 332-340.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19307040&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.tree.2009.01.009" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                     <span class="bibliomisc">Degtyareva, N. P., Greenwell, P., Hofmann, E. R., Hengartner, M. O., Zhang, L., Culotti, J. G., and Petes, T. D. (2002). <span class="emphasis"><em>Caenorhabditis elegans</em></span> DNA mismatch repair gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=msh-2;class=Gene" target="_blank">msh-2</a></em></span> is required for microsatellite stability and maintenance of genome integrity. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>99</em></span>, 2158-2163.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11830642&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.032671599" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                     <span class="bibliomisc">Denver, D. R., Dolan, P. C., Wilhelm, L. J., Sung, W., Lucas-Lledo, J. I., Howe, D. K., Lewis, S. C., Okamoto, K., Thomas,
                        W. K., Lynch, M., and Baer, C. F. (2009). A genome-wide view of <span class="emphasis"><em>Caenorhabditis elegans</em></span> base-substitution mutation processes. Proc. Natl. Acad. of Sci. USA <span class="emphasis"><em>106</em></span>, 16310-16324.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19805298&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0904895106" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                     <span class="bibliomisc">Denver, D. R., Feinberg, S., Steding, C., Durbin, M., and Lynch, M. (2006). The relative roles of three DNA repair pathways
                        in preventing <span class="emphasis"><em>Caenorhabditis elegans</em></span> mutation accumulation. Genetics <span class="emphasis"><em>174</em></span>, 57-65.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16783005&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.106.059840" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                     <span class="bibliomisc">Denver, D. R., Morris, K., Kewalramani, A., Harris, K. E., Chow, A., Estes, S., Lynch, M., and Thomas, W. K. (2004a). Abundance,
                        distribution, and mutation rates of homopolymeric nucleotide runs in the genome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Mol. Evol. <span class="emphasis"><em>58</em></span>, 584-595.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15170261&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s00239-004-2580-4" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                     <span class="bibliomisc">Denver, D. R., Morris, K., Lynch, M., and Thomas, W. K. (2004b). High mutation rate and predominance of insertions in the
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span> nuclear genome. Nature <span class="emphasis"><em>430</em></span>, 679-682.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15295601&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02697" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                     <span class="bibliomisc">Denver, D. R., Morris, K., Lynch, M., Vassilieva, L. L., and Thomas, W. K. (2000). High direct estimate of the mutation rate
                        in the mitochondrial genome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>289</em></span>, 2342-2344.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11009418&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                     <span class="bibliomisc">Dolgin, E. S., Charlesworth, B., Baird, S. E., and Cutter, A. D. (2007). Inbreeding and outbreeding depression in <span class="emphasis"><em>Caenorhabditis</em></span> nematodes. Evolution <span class="emphasis"><em>61</em></span>, 1339-1352.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17542844&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1111/j.1558-5646.2007.00118.x" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                     <span class="bibliomisc">Dolgin, E. S., Charlesworth, B., and Cutter, A. D. (2008). Population frequencies of transposable elements in selfing and
                        outcrossing <span class="emphasis"><em>Caenorhabditis</em></span> nematodes. Genet. Res. <span class="emphasis"><em>90</em></span>, 317-329.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18840306&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1017/S0016672308009440" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                     <span class="bibliomisc">Duret, L. (2000). tRNA gene number and codon usage in the <span class="emphasis"><em>C. elegans</em></span> genome are co-adapted for optimal translation of highly expressed genes. Trends Genet. <span class="emphasis"><em>16</em></span>, 287-289.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10858656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                     <span class="bibliomisc">Duret, L. (2002). Evolution of synonymous codon usage in metazoans. Curr. Opin. Genet. Dev. <span class="emphasis"><em>12</em></span>, 640-649.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12433576&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                     <span class="bibliomisc">Duret, L., and Mouchiroud, D. (1999). Expression pattern and, surprisingly, gene length shape codon usage in <span class="emphasis"><em>Caenorhabditi</em></span>s, <span class="emphasis"><em>Drosophila</em></span>, <span class="emphasis"><em>Arabidopsis</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>96</em></span>, 4482-4487.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10200288&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                     <span class="bibliomisc">Emmons, S. W., Yesner, L., Ruan, K. S., and Katzenberg, D. (1983). Evidence for a transposon in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Cell <span class="emphasis"><em>32</em></span>, 55-65.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6297788&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed">
                     <span class="bibliomisc">Fisher, R. A. (1958). The Genetical Theory of Natural Selection (New York: Dover Publications).</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed">
                     <span class="bibliomisc">Fitch, D. H. A. (2005). Evolution: An ecological context for <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>15</em></span>, R655-R658.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16139194&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.08.028" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed">
                     <span class="bibliomisc">Fr&oslash;kjaer-Jensen, C., Davis, M. W., Hopkins, C. E., Newman, B. J., Thummel, J. M., Olesen, S. P., Grunnet, M., and Jorgensen,
                        E. M. (2008). Single-copy insertion of transgenes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Genet. <span class="emphasis"><em>40</em></span>, 1375-1383.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18953339&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng.248" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed">
                     <span class="bibliomisc">Gibson, G., and Muse, S. V. (2002). A Primer of Genome Science (Sunderland, MA, Sinauer).</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed">
                     <span class="bibliomisc">Graustein, A., Gaspar, J. M., Walters, J. R., and Palopoli, M. F. (2002). Levels of DNA polymorphism vary with mating system
                        in the nematode genus <span class="emphasis"><em>Caenorhabditis</em></span>. Genetics <span class="emphasis"><em>161</em></span>, 99-107.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12019226&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed">
                     <span class="bibliomisc">Haag, E. S., Chamberlin, H., Coghlan, A., Fitch, D. H., Peters, A. D., and Schulenburg, H. (2007). <span class="emphasis"><em>Caenorhabditis</em></span> evolution: if they all look alike, you aren't looking hard enough. Trends Genet. <span class="emphasis"><em>23</em></span>, 101-104.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17275130&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.tig.2007.01.002" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed">
                     <span class="bibliomisc">Haber, M., Schungel, M., Putz, A., Muller, S., Hasert, B., and Schulenburg, H. (2005). Evolutionary history of <span class="emphasis"><em>Caenorhabditis elegans</em></span> inferred from microsatellites: Evidence for spatial and temporal genetic differentiation and the occurrence of outbreeding.
                        Mol. Biol. Evol. <span class="emphasis"><em>22</em></span>, 160-173.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15371529&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msh264" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed">
                     <span class="bibliomisc">Haerty, W., Artieri, C., Khezri, N., Singh, R. S., and Gupta, B. P. (2008). Comparative analysis of function and interaction
                        of transcription factors in nematodes: extensive conservation of orthology coupled to rapid sequence evolution. BMC Genomics
                        <span class="emphasis"><em>9</em></span>, 399.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18752680&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-9-399" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hartl, D. L., and Clark, A. G. (2007). Principles of Population Genetics, 4th ed. (Sunderland, MA, Sinauer Associates).</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hillier, L. W., Miller, R. D., Baird, S. E., Chinwalla, A., Fulton, L. A., Koboldt, D. C., and Waterston, R. H. (2007). Comparison
                        of <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> genome sequences reveals extensive conservation of chromosome organization and synteny. PLoS Biology <span class="emphasis"><em>5</em></span>, e167.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17608563&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0050167" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hodgkin, J. (June 25, 2005). Karyotype, ploidy and gene dosage. In WormBook, The <span class="emphasis"><em>C. elegans</em></span> Research Community, ed., <a href="http://www.wormbook.org/" target="_top">http://www.wormbook.org</a>.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hodgkin, J., and Doniach, T. (1997). Natural variation and copulatory plug formation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>146</em></span>, 149-164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9136008&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed">
                     <span class="bibliomisc">Howe, D. K., and Denver, D. R. (2008). Muller's ratchet and compensatory mutation in <span class="emphasis"><em>Caenorhabditis briggsae</em></span> mitochondrial genome evolution. BMC Evol. Biol. <span class="emphasis"><em>8</em></span>, 62.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18302772&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2148-8-62" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed">
                     <span class="bibliomisc">Hudson, R. R., and Kaplan, N. L. (1995). Deleterious background selection with recombination. Genetics <span class="emphasis"><em>141</em></span>, 1605-1617.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8601498&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed">
                     <span class="bibliomisc">Jovelin, R. (2009). Rapid sequence evolution of transcription factors controlling neuron differentiation in <span class="emphasis"><em>Caenorhabditis</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>26</em></span>, 2373-2386.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19589887&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msp142" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed">
                     <span class="bibliomisc">Jovelin, R., Dunham, J. P., Sung, F. S., and Phillips, P. C. (2009). High nucleotide divergence in developmental regulatory
                        genes contrasts with the structural elements of olfactory pathways in <span class="emphasis"><em>Caenorhabditis</em></span>. Genetics <span class="emphasis"><em>181</em></span>, 1387-1397.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19001295&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.107.082651" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kammenga, J. E., Phillips, P. C., De Bono, M., and Doroszuk, A. (2008). Beyond induced mutants: using worms to study natural
                        variation in genetic pathways. Trends Genet. <span class="emphasis"><em>24</em></span>, 178-185.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18325626&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.tig.2008.01.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed">
                     <span class="bibliomisc">Katju, V., LaBeau, E. M., Lipinski, K. J., and Bergthorsson, U. (2008). Sex change by gene conversion in a <span class="emphasis"><em>Caenorhabditis elegans <a href="http://www.wormbase.org/db/get?name=fog-2;class=Gene" target="_blank">fog-2</a></em></span> mutant. Genetics <span class="emphasis"><em>180</em></span>, 669-672.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18757925&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.108.090035" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed">
                     <span class="bibliomisc">Katju, V., and Lynch, M. (2003). The structure and early evolution of recently arisen gene duplicates in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome. Genetics <span class="emphasis"><em>165</em></span>, 1793-1803.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14704166&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed">
                     <span class="bibliomisc">Keightley, P. D., and Bataillon, T. M. (2000). Multigeneration maximum-likelihood analysis applied to mutation-accumulation
                        experiments in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>154</em></span>, 1193-1201.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10757763&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed">
                     <span class="bibliomisc">Keightley, P. D., and Caballero, A. (1997). Genomic mutation rates for lifetime reproductive output and lifespan in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>94</em></span>, 3823-3827.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9108062&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kent, W. J., and Zahler, A. M. (2000). Conservation, regulation, synteny, and introns in a large-scale <span class="emphasis"><em>C. briggsae</em></span> - <span class="emphasis"><em>C. elegans</em></span> genomic alignment. Genome Res. <span class="emphasis"><em>10</em></span>, 1115-1125.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10958630&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kiontke, K., Barri&eacute;re, A., Kolotuev, I., Podbilewicz, B., Sommer, R., Fitch, D. H., and F&eacute;lix, M. A. (2007). Trends, stasis,
                        and drift in the evolution of nematode vulva development. Curr. Biol. <span class="emphasis"><em>17</em></span>, 1925-1937.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18024125&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2007.10.061" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib72"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kiontke, K., and Fitch, D. H. A. (August 11, 2005). The phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> and other rhabditids. In WormBook, The <span class="emphasis"><em>C. elegans</em></span> Research Community, ed., <a href="http://www.wormbook.org/" target="_top">http://www.wormbook.org</a>.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib73"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kiontke, K., Gavin, N. P., Raynes, Y., Roehrig, C., Piano, F., and Fitch, D. H. A. (2004). <span class="emphasis"><em>Caenorhabditis</em></span> phylogeny predicts convergence of hermaphroditism and extensive intron loss. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 9003-9008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15184656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0403094101" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib74"></a><p class="bibliomixed">
                     <span class="bibliomisc">Koch, R., van Luenen, H. G. A. M., van der Horst, M., Thijssen, K. L., and Plasterk, R. H. A. (2000). Single nucleotide polymorphisms
                        in wild isolates of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome Res. <span class="emphasis"><em>10</em></span>, 1690-1696.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11076854&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib75"></a><p class="bibliomixed">
                     <span class="bibliomisc">Kreitman, M. (2000). Methods to detect selection in populations with applications to the human. Annual Review of Genomics
                        and Human Genetics <span class="emphasis"><em>1</em></span>, 539-559.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11701640&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.genom.1.1.539" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib76"></a><p class="bibliomixed">
                     <span class="bibliomisc">Li, W. H. (1997). Molecular Evolution (Sunderland, MA, Sinauer).</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib77"></a><p class="bibliomixed">
                     <span class="bibliomisc">Liao, L. W., Rosenzweig, B., and Hirsh, D. (1983). Analysis of a transposable element in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>80</em></span>, 3585-3589.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6304721&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib78"></a><p class="bibliomixed">
                     <span class="bibliomisc">Loewe, L., and Cutter, A. D. (2008). On the potential for extinction by Muller's ratchet in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. BMC Evol. Biol. <span class="emphasis"><em>8</em></span>, 125.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18447910&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2148-8-125" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib79"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lunt, D. H., and Hyman, B. C. (1997). Animal mitochondrial DNA recombination. Nature <span class="emphasis"><em>387</em></span>, 247.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9153388&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/387247a0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib80"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lynch, M. (2007). The Origins of Genome Architecture (Sunderland, MA, Sinauer).</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib81"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lynch, M., and Conery, J. S. (2000). The evolutionary fate and consequences of duplicate genes. Science <span class="emphasis"><em>290</em></span>, 1151-1155.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11073452&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib82"></a><p class="bibliomixed">
                     <span class="bibliomisc">Mancera, E., Bourgon, R., Brozzi, A., Huber, W., and Steinmetz, L. M. (2008). High-resolution mapping of meiotic crossovers
                        and non-crossovers in yeast. Nature <span class="emphasis"><em>454</em></span>, 479-485.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18615017&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature07135" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib83"></a><p class="bibliomixed">
                     <span class="bibliomisc">Maydan, J. S., Flibotte, S., Edgley, M. L., Lau, J., Selzer, R. R., Richmond, T. A., Pofahl, N. J., Thomas, J. H., and Moerman,
                        D. G. (2007). Efficient high-resolution deletion discovery in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by array comparative genomic hybridization. Genome Res. <span class="emphasis"><em>17</em></span>, 337-347.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17267812&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.5690307" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib84"></a><p class="bibliomixed">
                     <span class="bibliomisc">Moerman, D. G., and Baillie, D. L. (1979). Genetic organization in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: fine-structure analysis of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-22;class=Gene" target="_blank">unc-22</a></em></span> gene. Genetics <span class="emphasis"><em>91</em></span>, 95-103.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17248882&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib85"></a><p class="bibliomixed">
                     <span class="bibliomisc">Montooth, K. L., and Rand, D. M. (2008). The spectrum of mitochondrial mutation differs across species. PLoS Biology <span class="emphasis"><em>6</em></span>, 1634-1637.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18752353&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0060213" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib86"></a><p class="bibliomixed">
                     <span class="bibliomisc">Morran, L. T., Cappy, B. J., Anderson, J. L., and Phillips, P. C. (2009). Sexual partners for the stressed: facultative outcrossing
                        in the self-fertilizing nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Evolution <span class="emphasis"><em>63</em></span>, 1473-1482.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19210536&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1111/j.1558-5646.2009.00652.x" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib87"></a><p class="bibliomixed">
                     <span class="bibliomisc">Morran, L. T., Parmenter, M. D., and Phillips, P. C. (2009). Mutation load and rapid adaptation favour outcrossing over self-fertilization.
                        Nature <span class="emphasis"><em>462</em></span>, 350-352.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19847164&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature08496" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib88"></a><p class="bibliomixed">
                     <span class="bibliomisc">Ohta, T., and Kimura, M. (1973). Model of mutation appropriate to estimate number of electrophoretically detectable alleles
                        in a finite population. Genet. Res. <span class="emphasis"><em>22</em></span>, 201-204.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4777279&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib89"></a><p class="bibliomixed">
                     <span class="bibliomisc">Palopoli, M. F., Rockman, M. V., Tinmaung, A., Ramsay, C., Curwen, S., Aduna, A., Laurita, J., and Kruglyak, L. (2008). Molecular
                        basis of the copulatory plug polymorphism in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>454</em></span>, 1019-1022.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18633349&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature07171" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib90"></a><p class="bibliomixed">
                     <span class="bibliomisc">Pannell, J. R. (2003). Coalescence in a metapopulation with recurrent local extinction and recolonization. Evolution <span class="emphasis"><em>57</em></span>, 949-961.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12836814&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib91"></a><p class="bibliomixed">
                     <span class="bibliomisc">Pannell, J. R., and Charlesworth, B. (1999). Neutral genetic diversity in a metapopulation with recurrent local extinction
                        and recolonization. Evolution <span class="emphasis"><em>53</em></span>, 664-676.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib92"></a><p class="bibliomixed">
                     <span class="bibliomisc">Parkinson, J., Mitreva, M., Whitton, C., Thomson, M., Daub, J., Martin, J., Schmid, R., Hall, N., Barrell, B., Waterston,
                        R. H., <span class="emphasis"><em>et al.</em></span> (2004). A transcriptomic analysis of the phylum Nematoda. Nat. Genet. <span class="emphasis"><em>36</em></span>, 1259-1267.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15543149&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1472" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib93"></a><p class="bibliomixed">
                     <span class="bibliomisc">Phillips, N., Salomon, M., Custer, A., Ostrow, D., and Baer, C. F. (2009). Spontaneous mutational and standing genetic (co)variation
                        at dinucleotide microsatellites in <span class="emphasis"><em>Caenorhabditis briggsae</em></span> and <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>26</em></span>, 659-669.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19109257&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msn287" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib94"></a><p class="bibliomixed">
                     <span class="bibliomisc">Phillips, P. C. (2006). One perfect worm. Trends Genet. <span class="emphasis"><em>22</em></span>, 405-407.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16806564&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.tig.2006.06.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib95"></a><p class="bibliomixed">
                     <span class="bibliomisc">Piganeau, G., Gardner, M., and Eyre-Walker, A. (2004). A broad survey of recombination in animal mitochondria. Mol. Biol.
                        Evol. <span class="emphasis"><em>21</em></span>, 2319-2325.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15342796&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msh244" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib96"></a><p class="bibliomixed">
                     <span class="bibliomisc">Pujol, N., Zugasti, O., Wong, D., Couillault, C., Kurz, C. L., Schulenburg, H., and Ewbank, J. J. (2008). Anti-fungal innate
                        immunity in <span class="emphasis"><em>C. elegans</em></span> is enhanced by evolutionary diversification of antimicrobial peptides. PLoS Pathogens <span class="emphasis"><em>4</em></span>, e1000105.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18636113&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.ppat.1000105" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib97"></a><p class="bibliomixed">
                     <span class="bibliomisc">Rattray, B., and Rose, A. M. (1988). Increased intragenic recombination and non-disjunction in the Rec-1 strain of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genet. Res. <span class="emphasis"><em>51</em></span>, 89-93.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib98"></a><p class="bibliomixed">
                     <span class="bibliomisc">Robert, V. J., Davis, M. W., Jorgensen, E. M., and Bessereau, J. L. (2008). Gene conversion and end-joining-repair double-strand
                        breaks in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> germline. Genetics <span class="emphasis"><em>180</em></span>, 673-679.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18757928&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.108.089698" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib99"></a><p class="bibliomixed">
                     <span class="bibliomisc">Robertson, H. M., and Thomas, J. H. (January 06, 2006). The putative chemoreceptor families of <span class="emphasis"><em>C. elegans</em></span>. In WormBook, The <span class="emphasis"><em>C. elegans</em></span> Research Community, ed., <a href="http://www.wormbook.org/" target="_top">http://www.wormbook.org</a>.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib100"></a><p class="bibliomixed">
                     <span class="bibliomisc">Rockman, M. V., and Kruglyak, L. (2009). Recombinational landscape and population genomics of <span class="emphasis"><em>C. elegans</em></span>. PLoS Genetics <span class="emphasis"><em>5</em></span>, e1000419.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19283065&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000419" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib101"></a><p class="bibliomixed">
                     <span class="bibliomisc">Rose, A. M., and Baillie, D. L. (1980). Genetic organization of the region around <a href="http://www.wormbase.org/db/get?name=UNC-15;class=Protein" target="_blank">UNC-15</a> (I), a gene affecting paramyosin in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>96</em></span>, 639-648.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7262541&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib102"></a><p class="bibliomixed">
                     <span class="bibliomisc">Rosenberg, N. A., Pritchard, J. K., Weber, J. L., Cann, H. M., Kidd, K. K., Zhivotovsky, L. A., and Feldman, M. W. (2002).
                        Genetic structure of human populations. Science <span class="emphasis"><em>298</em></span>, 2381-2385.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12493913&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1078311" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib103"></a><p class="bibliomixed">
                     <span class="bibliomisc">Schulenburg, H., and Boehnisch, C. (2008). Diversification and adaptive sequence evolution of <span class="emphasis"><em>Caenorhabditis</em></span> lysozymes (Nematoda: Rhabditidae). BMC Evol. Biol. <span class="emphasis"><em>8</em></span>, 114.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18423043&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2148-8-114" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib104"></a><p class="bibliomixed">
                     <span class="bibliomisc">Semple, C., and Wolfe, K. H. (1999). Gene duplication and gene conversion in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome. J. Mol. Evol. <span class="emphasis"><em>48</em></span>, 555-564.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10198121&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib105"></a><p class="bibliomixed">
                     <span class="bibliomisc">Seyfert, A. L., Cristescu, M. E., Frisse, L., Schaack, S., Thomas, W. K., and Lynch, M. (2008). The rate and spectrum of microsatellite
                        mutation in <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Daphnia pulex</em></span>. Genetics <span class="emphasis"><em>178</em></span>, 2113-2121.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18430937&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.107.081927" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib106"></a><p class="bibliomixed">
                     <span class="bibliomisc">Shabalina, S. A., and Kondrashov, A. S. (1999). Pattern of selective constraint in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> genomes. Genet. Res. <span class="emphasis"><em>74</em></span>, 23-30.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10505405&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib107"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sivasundar, A., and Hey, J. (2003). Population genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span>: the paradox of low polymorphism in a widespread species. Genetics <span class="emphasis"><em>163</em></span>.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12586703&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib108"></a><p class="bibliomixed">
                     <span class="bibliomisc">Sivasundar, A., and Hey, J. (2005). Sampling from natural populations using RNAi reveals high outcrossing and population structure
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Curr. Biol. <span class="emphasis"><em>15</em></span>, 1598-1602.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16139217&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.08.034" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib109"></a><p class="bibliomixed">
                     <span class="bibliomisc">Smith, J. M., and Haigh, J. (1974). Hitch-hiking effect of a favorable gene. Genet. Res. <span class="emphasis"><em>23</em></span>, 23-35.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4407212&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib110"></a><p class="bibliomixed">
                     <span class="bibliomisc">Stein, L. D., Bao, Z., Blasiar, D., Blumenthal, T., Brent, M. R., Chen, N., Chinwalla, A., Clarke, L., Clee, C., Coghlan,
                        A., <span class="emphasis"><em>et al.</em></span> (2003). The genome sequence of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>: A platform for comparative genomics. PLoS Biology <span class="emphasis"><em>1</em></span>, 166-192.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14624247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib111"></a><p class="bibliomixed">
                     <span class="bibliomisc">Stenico, M., Lloyd, A. T., and Sharp, P. M. (1994). Codon usage in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: delineation of translational selection and mutational biases. Nucleic Acids Res. <span class="emphasis"><em>22</em></span>, 2437-2446.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8041603&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib112"></a><p class="bibliomixed">
                     <span class="bibliomisc">Stewart, M. R., Clark, N. L., Merrihew, G., Galloway, E. M., and Thomas, J. H. (2005). High genetic diversity in the chemoreceptor
                        superfamily of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>169</em></span>, 1985-1996.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15520260&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.035329" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib113"></a><p class="bibliomixed">
                     <span class="bibliomisc">Thomas, J. H. (2008). Genome evolution in <span class="emphasis"><em>Caenorhabditis</em></span>. Brief. Func. Genom. Proteom. <span class="emphasis"><em>7</em></span>, 211-216.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18573804&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/bfgp/eln022" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib114"></a><p class="bibliomixed">
                     <span class="bibliomisc">Thomas, J. H., Kelley, J. L., Robertson, H. M., Ly, K., and Swanson, W. J. (2005). Adaptive evolution in the SRZ chemoreceptor
                        families of <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>102</em></span>, 4476-4481.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15761060&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0406469102" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib115"></a><p class="bibliomixed">
                     <span class="bibliomisc">Vassilieva, L. L., Hook, A. M., and Lynch, M. (2000). The fitness effects of spontaneous mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Evolution <span class="emphasis"><em>54</em></span>, 1234-1246.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11005291&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib116"></a><p class="bibliomixed">
                     <span class="bibliomisc">Vassilieva, L. L., and Lynch, M. (1999). The rate of spontaneous mutation for life-history traits in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>151</em></span>, 119-129.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9872953&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib117"></a><p class="bibliomixed">
                     <span class="bibliomisc">Vergara, I., Mah, A., Huang, J., Tarailo-Graovac, M., Johnsen, R., Baillie, D., and Chen, N. (2009). Polymorphic segmental
                        duplication in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. BMC Genomics <span class="emphasis"><em>10</em></span>, 329.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19622155&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-10-329" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib118"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wakeley, J. (1999). Nonequilibrium migration in human history. Genetics <span class="emphasis"><em>153</em></span>, 1863-1871.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10581291&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib119"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wakeley, J., and Aliacar, N. (2001). Gene genealogies in a metapopulation. Genetics <span class="emphasis"><em>159</em></span>, 893-905.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11606561&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib120"></a><p class="bibliomixed">
                     <span class="bibliomisc">Webb, C. T., Shabalina, S. A., Ogurtsov, A. Y., and Kondrashov, A. S. (2002). Analysis of similarity within 142 pairs of orthologous
                        intergenic regions of <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Nucleic Acids Res. <span class="emphasis"><em>30</em></span>, 1233-1239.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11861916&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib121"></a><p class="bibliomixed">
                     <span class="bibliomisc">Wiehe, T. H. E., and Stephan, W. (1993). Analysis of a genetic hitchhiking model, and its application to DNA polymorphism
                        data from <span class="emphasis"><em>Drosophila melanogaster</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>10</em></span>, 842-854.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8355603&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib122"></a><p class="bibliomixed">
                     <span class="bibliomisc">Woollard, A. (June 25, 2005). Gene duplications and genetic redundancy in <span class="emphasis"><em>C. elegans</em></span>. In WormBook, The <span class="emphasis"><em>C. elegans</em></span> Research Community, ed., (<a href="http://www.wormbook.org/" target="_top">http://www.wormbook.org</a>).</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib123"></a><p class="bibliomixed">
                     <span class="bibliomisc">Yang, Z., and Bielawski, J. P. (2000). Statistical methods for detecting molecular adaptation. Trends Ecol. Evol. <span class="emphasis"><em>15</em></span>, 496-503.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11114436&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib124"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zetka, M. (2009). Homologue pairing, recombination and segregation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome Dynamics <span class="emphasis"><em>5</em></span>, 43-55.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18948706&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1159/000166618" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib125"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zhao, Z., Thomas, J. H., Chen, N., Sheps, J. A., and Baillie, D. L. (2007). Comparative genomics and adaptive selection of
                        the ATP-binding-cassette gene family in <span class="emphasis"><em>Caenorhabditis</em></span> species. Genetics <span class="emphasis"><em>175</em></span>, 1407-1418.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17194779&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.106.066720" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib126"></a><p class="bibliomixed">
                     <span class="bibliomisc">Zou, L., Sriswasdi, S., Ross, B., Missiuro, P. V., Liu, J., and Ge, H. (2008). Systematic analysis of pleiotropy in <span class="emphasis"><em>C. elegans</em></span> early embryogenesis. PLoS Comput Biol <span class="emphasis"><em>4</em></span>, e1000003.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18463698&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pcbi.1000003" target="_blank">Article</a>
                     
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e9" href="#d0e9">*</a></sup>Edited by David H. A. Fitch. Last revised December 3, 2009. Published March 5, 2010. This chapter should be cited as:
                  Cutter, A. D. Molecular evolution inferences from the <span class="emphasis"><em>C. elegans</em></span> genome (March 5, 2010), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.149.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2010 Asher D. Cutter. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e34" href="#d0e34">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:asher.cutter@utoronto.ca" target="_top">asher.cutter@utoronto.ca</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>
