<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genomics and biology of the nematode Caenorhabditis briggsae
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#genbiolbriggsae_ArticleID" title="Genomics and biology of the nematode Caenorhabditis briggsae&#xA;">
      <link rel="next" href="#sec1" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="genomesCbriggsae.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="genbiolbriggsae_ArticleID"></a>Genomics and biology of the nematode <span class="emphasis"><em>Caenorhabditis briggsae</em></span><sup><a name="d0e10" href="#ftn.d0e10">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Bhagwati P. Gupta<sup>1</sup><em><span class="remark"><sup><a name="d0e37" href="#ftn.d0e37">&sect;</a></sup></span></em>, Robert Johnsen<sup>2</sup>, Nansheng Chen<sup>2</sup></span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup>Department of Biology, McMaster University, </span><span class="address">Hamilton, ON L8S 4K1, Canada
                           <p></p></span></span><span class="affiliation"><span class="orgname"><sup>2</sup>Department of Molecular Biology and Biochemistry, </span><span class="address">Simon Fraser University, Burnaby, BC V5A 1S6, Canada
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_geneticsgenomics.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Genetics and genomics<br />>> Genomics<br />>>> Biology and genome of nematodes other than <em>C. elegans</em>
		</a></div><hr />

<a href="http://www.wormbase.org/forums/index.php?board=198.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Genomics+and+biology+of+the+nematode+Caenorhabditis.+briggsae" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. Overview of the <span class="emphasis"><em>C. briggsae</em></span> genome</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec2_1">2.1. Genome sequencing</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_2">2.2. Protein-coding genes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_3">2.3. Orthologs between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span></a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_4">2.4. Evolutionary distance between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span></a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_5">2.5. Intron evolution</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_6">2.6. Genomic sequence alignment and colinearity analysis</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_7">2.7. Operons, neighboring genes, and overlapping genes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_8">2.8. Non-coding RNAs</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_9">2.9. Gene families</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec3">3. Evolutionary studies</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec3_1">3.1. Development of the excretory duct cell</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_2">3.2. Sex determination</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_3">3.3. Development of the vulva</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_4">3.4. Development of the male tail</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_5">3.5. Thermal sensitivity</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_6">3.6. Specification of neural asymmetry</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec4">4. Genetic approach to study functions of <span class="emphasis"><em>C. briggsae</em></span> genes</a></span></dt>
               <dt><span class="sect1"><a href="#sec5">5. Tools for genetic studies</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec5_1">5.1. RNAi</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_2">5.2. Mutant strains</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_3">5.3. Genetic linkage maps</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_4">5.4. Mos1 mutagenesis</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_5">5.5. Collection of wild isolates and recombinant inbred lines</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_6">5.6. Genomic clones</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec6">6. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#d0e2982">7. Bibliography</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>The soil nematode</strong></span> <span class="emphasis"><em><span class="bold"><strong>Caenorhabditis briggsae</strong></span></em></span> <span class="bold"><strong>is an attractive model system for studying evolution of both animal development and behavior. Being a close relative of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans, C. briggsae</strong></span></em></span> <span class="bold"><strong>is frequently used in comparative studies to infer species-specific function of the orthologous genes and also for studying
                     the dynamics of chromosome evolution. The genome sequence of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. briggsae</strong></span></em></span> <span class="bold"><strong>is valuable in reverse genetics and genome-wide comparative studies. This review discusses resources and tools, which are
                     currently available, to facilitate study of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. briggsae</strong></span></em></span> <span class="bold"><strong>in order to unravel mechanisms of gene function that confer morphological and behavioral diversity.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p><span class="emphasis"><em>C. briggsae</em></span> is a natural companion to <span class="emphasis"><em>C. elegans</em></span> for research because of the similarities in morphology, hermaphroditic life style and ease of cultivation. The two species
               diverged from a common ancestor roughly 100 million years ago (see <a href="#sec2_4" title="2.4.&nbsp;Evolutionary distance between C. briggsae and C. elegans">Section 2.4</a>; <a href="#bib46">Stein et al., 2003</a>) yet appear almost identical in morphology and behavior. The phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> nematodes reveals that <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> are not sibling species and that <span class="emphasis"><em>C. remanei</em></span>, a gonochoristic species, is the closest relative of <span class="emphasis"><em>C. briggsae</em></span> (see <a href="http://www.wormbook.org/chapters/www_phylogrhabditids/phylorhab.html" target="_top">The phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> and other rhabditids</a>). The seemingly identical morphology of the two nematodes permits straightforward interpretation of results involving conserved
               genes and pathways. The majority of the currently available genetic and reverse genetic tools in <span class="emphasis"><em>C. elegans</em></span> can be used in <span class="emphasis"><em>C. briggsae</em></span> with no major modification. These together with its whole genome sequence make <span class="emphasis"><em>C. briggsae</em></span> an ideal model for comparative studies.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Overview of the <span class="emphasis"><em>C. briggsae</em></span> genome
                     </h2>
                  </div>
               </div>
            </div>
            <p>The whole genome sequencing project (<a href="#bib46">Stein et al., 2003</a>) revealed that the genomes of <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> have much in common (Summarized in <a href="#table1" title="Table 1. Comparison between genomes of C. briggsae and C. elegans (primarily based on Stein et al. (2003))">Table 1</a>). For example, both worms have the same number of chromosomes (six chromosomes each), similar genome size, and similar numbers
               of protein coding and non-protein coding genes. Further analysis demonstrated that about 62% of the protein coding genes in
               <span class="emphasis"><em>C. briggsae</em></span> have orthologs in <span class="emphasis"><em>C. elegans</em></span>. Nevertheless, many interesting species-specific features including species-specific genes exist, which serve as the foundation
               for comparative analysis. In the following subsections, we will describe the <span class="emphasis"><em>C. briggsae</em></span> genome and compare it with the <span class="emphasis"><em>C. elegans</em></span> genome. Data described here are all available at <a href="http://www.wormbase.org" target="_blank">WormBase</a> (<a href="#bib6">Chen et al., 2005</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_1"></a>2.1.&nbsp;Genome sequencing
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A hybrid approach combining whole-genome assembly with whole-genome shotgun sequencing (WGS) was adopted to sequence the <span class="emphasis"><em>C. briggsae</em></span> genome (<a href="#bib46">Stein et al., 2003</a>). The <span class="emphasis"><em>C. briggsae</em></span> strain <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> was targeted for physical map construction (<a href="#bib33">Marra et al., 1997</a>) and whole genome sequencing. A total number of 2.068 million reads, representing an approximately 10-fold coverage of the
                  entire genome, were assembled into 5,341 contigs and linked together into 889 gapped supercontigs containing 105.6 Mbp of
                  DNA sequence. This assembly (named cb25.agp8) consists of 463 sequence superconfigs placed into 142 FPCs and covers 102,431,873
                  bp and is publicly available at the <a href="ftp://ftp.wormbase.org/pub/wormbase/genomes/briggsae/assembly/cb25.agp8" target="_blank">WormBase FTP site</a>. This assembly is estimated to cover 98% of the entire <span class="emphasis"><em>C. briggsae</em></span> genome with a sequencing accuracy of approximately 99.98%. Therefore the <span class="emphasis"><em>C. briggsae</em></span> genome size is approximately 104 Mbp, a size similar to but slightly larger than that of the <span class="emphasis"><em>C. elegans</em></span> genome (102 Mbp, according to WormBase WS160). Recently, the construction of a genetic map using single nucleotide polymorphic
                  (SNP) markers has positioned 100 Mb along the six chromosomes and refined the sequence map (<a href="#bib21">Hillier et al., 2005</a>).
               </p>
               <div class="table"><a name="table1"></a><p class="title"><span class="bold"><strong>Table 1. Comparison between genomes of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. briggsae</strong></span></em></span> <span class="bold"><strong>and</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> (primarily based on <a href="#bib46">Stein et al. (2003)</a>)
                  </p>
                  <table summary="Table 1. Comparison between genomes of C. briggsae and C. elegans (primarily based on Stein et al. (2003))" border="1">
                     <colgroup>
                        <col align="left">
                        <col align="left">
                        <col align="center">
                        <col align="center">
                     </colgroup>
                     <thead>
                        <tr>
                           <th colspan="2" align="center">Species</th>
                           <th align="center"><span class="emphasis"><em>C. briggsae</em></span></th>
                           <th align="center"><span class="emphasis"><em>C. elegans</em></span></th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td colspan="2" align="left">Genome size</td>
                           <td align="center">104 Mb</td>
                           <td align="center">102 Mb</td>
                        </tr>
                        <tr>
                           <td colspan="2" align="left">Sequencing approach</td>
                           <td align="center">Combined physical mapping and WGS</td>
                           <td align="center">Entirely clone-based</td>
                        </tr>
                        <tr>
                           <td colspan="2" align="left">Reads</td>
                           <td align="center">2.068</td>
                           <td align="center">-</td>
                        </tr>
                        <tr>
                           <td colspan="2" align="left">Most recent assembly</td>
                           <td align="center">Cb25.agp8</td>
                           <td align="center">WormBase WS160</td>
                        </tr>
                        <tr>
                           <td colspan="2" align="left">Assembly</td>
                           <td align="center">142 FPCs</td>
                           <td align="center">6 chromosomes</td>
                        </tr>
                        <tr>
                           <td colspan="2" align="left">Coming assembly</td>
                           <td align="center">6 chromosomes</td>
                           <td align="center">-</td>
                        </tr>
                        <tr>
                           <td colspan="2" align="left">Repetitive elements content</td>
                           <td align="center">22.4%</td>
                           <td align="center">16.5%</td>
                        </tr>
                        <tr valign="middle">
                           <td rowspan="6" align="left" valign="middle">Protein-coding genes<sup>*</sup></td>
                           <td align="left" valign="middle">Number of genes</td>
                           <td align="center" valign="middle">19,507</td>
                           <td align="center" valign="middle">20,621</td>
                        </tr>
                        <tr>
                           <td align="left">Exons</td>
                           <td align="center">114,339</td>
                           <td align="center">125,702</td>
                        </tr>
                        <tr>
                           <td align="left">Introns</td>
                           <td align="center">94,832</td>
                           <td align="center">105,081</td>
                        </tr>
                        <tr>
                           <td align="left">Median length</td>
                           <td align="center">1.90 kbp</td>
                           <td align="center">1.83 kbp</td>
                        </tr>
                        <tr>
                           <td align="left">Summed length</td>
                           <td align="center">55.7 Mbp</td>
                           <td align="center">55.6 Mbp</td>
                        </tr>
                        <tr>
                           <td align="left">Gene density</td>
                           <td align="center">5.4 kbp per gene</td>
                           <td align="center">4.9 kbp per gene</td>
                        </tr>
                        <tr valign="middle">
                           <td rowspan="16" align="left" valign="middle">Non-coding genes</td>
                           <td align="left" valign="middle">tRNA</td>
                           <td align="center" valign="middle">777 (181)</td>
                           <td align="center" valign="middle">609 (210)</td>
                        </tr>
                        <tr>
                           <td align="left">5S rRNA</td>
                           <td align="center">7</td>
                           <td align="center">15</td>
                        </tr>
                        <tr>
                           <td align="left">5.8S rRNA</td>
                           <td align="center">0</td>
                           <td align="center">1</td>
                        </tr>
                        <tr>
                           <td align="left">18S rRNA</td>
                           <td align="center">0 (3)</td>
                           <td align="center">2 (1)</td>
                        </tr>
                        <tr>
                           <td align="left">26S rRNA</td>
                           <td align="center">0 (7)</td>
                           <td align="center">1 (1)</td>
                        </tr>
                        <tr>
                           <td align="left">SRP</td>
                           <td align="center">4</td>
                           <td align="center">5</td>
                        </tr>
                        <tr>
                           <td align="left">U3 snoRNA</td>
                           <td align="center">4</td>
                           <td align="center">6</td>
                        </tr>
                        <tr>
                           <td align="left">U1</td>
                           <td align="center">11</td>
                           <td align="center">12</td>
                        </tr>
                        <tr>
                           <td align="left">U2</td>
                           <td align="center">15</td>
                           <td align="center">20</td>
                        </tr>
                        <tr>
                           <td align="left">U4</td>
                           <td align="center">5</td>
                           <td align="center">5</td>
                        </tr>
                        <tr>
                           <td align="left">U5</td>
                           <td align="center">10</td>
                           <td align="center">15</td>
                        </tr>
                        <tr>
                           <td align="left">U6</td>
                           <td align="center">40</td>
                           <td align="center">23</td>
                        </tr>
                        <tr>
                           <td align="left">miRNA</td>
                           <td align="center">70</td>
                           <td align="center">105</td>
                        </tr>
                        <tr>
                           <td align="left">RNAaseP</td>
                           <td align="center">1</td>
                           <td align="center">1</td>
                        </tr>
                        <tr>
                           <td align="left">SL1</td>
                           <td align="center">0</td>
                           <td align="center">0</td>
                        </tr>
                        <tr>
                           <td align="left">SL2</td>
                           <td align="center">18</td>
                           <td align="center">18</td>
                        </tr>
                        <tr>
                           <td colspan="4" align="left">Notes: tRNA pseudogene predictions and other fragmentary matches are indicated in parentheses. rRNA genes and SL1 were found
                              in tandemly duplicated arrays that were largely excluded from the genomic assembly.
                           </td>
                        </tr>
                        <tr>
                           <td colspan="4" align="left">*Protein-coding genes are &#8220;hybrid&#8221; genes that were predicted using four gene prediction programs (Genefinder, Fgenesh, Twinscan,
                              and Ensembl pipeline).
                           </td>
                        </tr>
                     </tbody>
                  </table>
               </div>
               <p>An interesting difference between the genomes of <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>, which was revealed by comparative genomic analysis, is that the difference in the genome size is mainly due to the larger
                  non-genic repetitive DNA content in <span class="emphasis"><em>C. briggsae</em></span>'s genome, which accounts for 22.4% (or 23.3 Mbp) of the <span class="emphasis"><em>C. briggsae</em></span> genome compared to 16.5% (or 16.5 Mbp) of the <span class="emphasis"><em>C. elegans</em></span> genome (see <a href="#table1" title="Table 1. Comparison between genomes of C. briggsae and C. elegans (primarily based on Stein et al. (2003))">Table 1</a>). Notably, a single <span class="emphasis"><em>C. briggsae</em></span> repeat family, Cb000047, is present more than 20,000 times and covers 4.7 Mbp of the <span class="emphasis"><em>C. briggsae</em></span> genome, while it is present only a few hundred times in the <span class="emphasis"><em>C. elegans</em></span> genome. Despite the striking similarities between the genomes and coding sequences of <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> as summarized in <a href="#table1" title="Table 1. Comparison between genomes of C. briggsae and C. elegans (primarily based on Stein et al. (2003))">Table 1</a>, only a few orthologous repetitive elements were identified between these two sister species, suggesting that essentially
                  all extant dispersed repeat elements postdate the divergence of these two species.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_2"></a>2.2.&nbsp;Protein-coding genes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Before a detailed description of the comparative analysis between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> protein-coding genes are made, some caveats should be pointed out. First, while the <span class="emphasis"><em>C. briggsae</em></span> genome is still at a draft stage, the <span class="emphasis"><em>C. elegans</em></span> genome sequencing is complete. Second, while the <span class="emphasis"><em>C. briggsae</em></span> gene set was generated entirely by computational gene finding, the <span class="emphasis"><em>C. elegans</em></span> gene set was generated by computational gene finding, followed by extensive manual curation with various experimental evidences
                  including ESTs, OSTs, and full length cDNAs. Third, while only a small number of laboratories use <span class="emphasis"><em>C. briggsae</em></span> as a major experimental organism, a significantly larger number of laboratories use <span class="emphasis"><em>C. elegans</em></span>, which not only provides more data for accurately curating <span class="emphasis"><em>C. elegans</em></span> gene modes, but also creates a strong pressure to establish a high-quality <span class="emphasis"><em>C. elegans</em></span> gene set. In fact, the <span class="emphasis"><em>C. briggsae</em></span> gene set (the "hybrid" gene set) remained the same for three years following the publication of the <span class="emphasis"><em>C. briggsae</em></span> genome analysis paper (<a href="#bib46">Stein et al., 2003</a>) until recently when WormBase started to curate <span class="emphasis"><em>C. briggsae</em></span> genes (WormBase Consortium, personal communication). In the meantime, the <span class="emphasis"><em>C. elegans</em></span> gene set has been improved and updated in WormBase in each of its tri-weekly releases.
               </p>
               <p>As shown in <a href="#table1" title="Table 1. Comparison between genomes of C. briggsae and C. elegans (primarily based on Stein et al. (2003))">Table 1</a>, there are similar numbers of protein-coding genes in the genomes of <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>. The unspliced lengths of protein-coding genes and the total length of the protein-coding genes, including the length of
                  their introns, are roughly the same between the two species, further suggesting that the larger size of the <span class="emphasis"><em>C. briggsae</em></span> genome is not contributed by the number or size of the protein-coding genes. The <span class="emphasis"><em>C. briggsae</em></span> genome has contributed to the improved predictions of <span class="emphasis"><em>C. elegans</em></span> protein-coding genes. Notably, <span class="emphasis"><em>C. briggsae</em></span> gene prediction has helped to predict 1,275 new <span class="emphasis"><em>C. elegans</em></span> genes (<a href="#bib46">Stein et al., 2003</a>). So far, about 300 genes of these have been confirmed by OSTs of the ORFeome project. Despite the similarity of the number
                  of genes shared by the two species, <span class="emphasis"><em>C. briggsae</em></span> has about 800 genes that have not be found in <span class="emphasis"><em>C. elegans</em></span>, while <span class="emphasis"><em>C. elegans</em></span> has 1,061 genes that have not been found in <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib46">Stein et al., 2003</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_3"></a>2.3.&nbsp;Orthologs between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span></h3>
                     </div>
                  </div>
               </div>
               <p>One way <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> orthologs were identified was by searching for <span class="emphasis"><em>C. briggsae</em></span>/<span class="emphasis"><em>C. elegans</em></span> pairs that were each other's best BLASTP (<a href="#bib1">Altschul et al., 1997</a>) match in the opposite species. Synteny was subsequently used as a criterion to search for more orthologous pairs. These
                  combined approaches produced altogether 12,155 ortholog pairs, or 62% of the <span class="emphasis"><em>C. briggsae</em></span> gene set or 65% of the <span class="emphasis"><em>C. elegans</em></span> gene set (<a href="#bib46">Stein et al., 2003</a>). After the publication of the initial <span class="emphasis"><em>C. briggsae</em></span> genome analysis paper, a recently developed program <a href="http://inparanoid.cgb.ki.se/" target="_blank">InParanoid</a> (<a href="#bib38">O'Brien et al., 2005</a>) was applied to identify orthologs between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>. The InParanoid database (April 2005 release), using a recent <span class="emphasis"><em>C. elegans</em></span> gene set, lists 12,858 <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> ortholog pairs. Therefore the numbers of orthologs obtained by these two different approaches agree well with each other.
                  However, the number of identified orthologs will change over time as the genome assembly and gene annotation, especially for
                  <span class="emphasis"><em>C. briggsae</em></span>, improve.
               </p>
               <p>Currently the <span class="emphasis"><em>C. elegans</em></span> ortholog of a <span class="emphasis"><em>C. briggsae gene</em></span> is displayed in the Homology section on its gene page in WormBase and <span class="emphasis"><em>vice versa</em></span>. Users can follow the link to the gene page of the <span class="emphasis"><em>C. elegans</em></span> ortholog of a <span class="emphasis"><em>C. briggsae</em></span> gene. Right next to this ortholog link, there is another link termed &#8220;syntenic alignment&#8221;, which directs users to a corresponding
                  &#8220;Synteny Browser&#8221; page (see <a href="#figure1">Figure 1</a>). The Synteny Browser provides a comparative view of two genomic regions, a <span class="emphasis"><em>C. elegans</em></span> genomic region that contains the <span class="emphasis"><em>C. elegan</em></span> ortholog of the <span class="emphasis"><em>C. briggsae</em></span> gene, a <span class="emphasis"><em>C. briggsae</em></span> genomic region that is syntenic to the <span class="emphasis"><em>C. elegans</em></span> region. The <span class="emphasis"><em>C. briggsae</em></span> genomic region contains the <span class="emphasis"><em>C. briggsae</em></span> ortholog of the <span class="emphasis"><em>C. elegans</em></span> gene (see <a href="#figure1">Figure 1</a>). Users can browse along the genomic sequences by modifying the genomic coordinates.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="630">
<tr>
  <td align="center"><p><a href="genbiolbriggsaefig1.jpg"><img src="genbiolbriggsaefig1_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. WormBase synteny browser.&nbsp;</b>A comparative view of syntenic regions of two genomes-<span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> genomes-is displayed in the browser. Also displayed are gene models contained within these genomic regions in both genomes
                        and alignment details. Users can browse along genomic sequences by clicking on the Zoom/Scroll errors or modifying the genomic
                        coordinates.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_4"></a>2.4.&nbsp;Evolutionary distance between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span></h3>
                     </div>
                  </div>
               </div>
               <p>Before the <span class="emphasis"><em>C. briggsae</em></span> genome sequencing project, the divergence of <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> had been estimated to be 20&#8211;120 MYA (million years ago; <a href="#bib9">Coghlan and Wolfe, 2002</a>; <a href="#bib19">Heschl and Baillie, 1990</a>; <a href="#bib25">Kennedy et al., 1993</a>; <a href="#bib31">Lee et al., 1992</a>; <a href="#bib40">Prasad and Baillie, 1989</a>). The <span class="emphasis"><em>C. briggsae</em></span> genome sequencing project helped to refine the divergence date to 80&#8211;110 MYA (<a href="#bib46">Stein et al., 2003</a>). The new estimation, based on 338 sets of orthologs, is more precise because of the large gene sample size and improved
                  <span class="emphasis"><em>C. briggsae</em></span> gene predictions and ortholog assignments.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_5"></a>2.5.&nbsp;Intron evolution
                        </h3>
                     </div>
                  </div>
               </div>
               <p>There are more introns per gene in <span class="emphasis"><em>C. elegans</em></span> than in <span class="emphasis"><em>C. briggsae</em></span> (see <a href="#table1" title="Table 1. Comparison between genomes of C. briggsae and C. elegans (primarily based on Stein et al. (2003))">Table 1</a>). Comparative analysis, by aligning orthologous protein-coding genes between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>, revealed 6,579 species-specific introns among the 60,775 introns in the orthologous pairs of protein-coding genes. Roughly
                  two-thirds (4,379) are <span class="emphasis"><em>C. elegans</em></span>-specific, while one-third (2,200) are <span class="emphasis"><em>C. briggsae</em></span>-specific, which suggest rates of intron gains or losses of about 0.5 per gene in the 80&#8211;110 million years since the split
                  between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>. This rate is dramatically higher than that of mouse and human, which is 0.01 intron gains or losses per gene in 75 million
                  years. Active intron gains and losses have also been observed by comparing many smaller gene sets in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>, for example, the chemosensory gene families (<a href="#bib41">Robertson, 1998</a>; <a href="#bib42">Robertson, 2000</a>; <a href="#bib43">Robertson, 2001</a>), a sets of five genes (four <span class="emphasis"><em>CPEB</em></span> genes and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fog-3;class=Gene" target="_blank">fog-3</a></em></span>; <a href="#bib8">Cho et al., 2004</a>), and another set of five genes (SSU rDNA, LSU rDNA, part of <span class="emphasis"><em>RNAP2</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=par-6;class=Gene" target="_blank">par-6</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=pkc-3;class=Gene" target="_blank">pkc-3</a></em></span>; <a href="#bib29">Kiontke et al., 2004</a>). These studies demonstrated dramatically higher rates of intron gain and loss in <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> compared to that in mammals, and furthermore, significantly higher rates of intron loss compared to intron gain, which happen
                  only rarely. Recently, a study identified 81 introns gained in <span class="emphasis"><em>C. elegans</em></span> and 41 in <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib10">Coghlan and Wolfe, 2004</a>). Among these 122 new introns, twenty-eight show significant DNA sequence similarity to other introns in the same genomes,
                  including three that are similar to other introns in the same gene, suggesting that some of the introns were gained via reverse
                  splicing of preexisting intron.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_6"></a>2.6.&nbsp;Genomic sequence alignment and colinearity analysis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Whole genome sequence alignment of <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>, using the pair-wise alignment tool WABA (<a href="#bib26">Kent and Zahler, 2000</a>), revealed that 50.1% of the <span class="emphasis"><em>C. briggsae</em></span> genome aligns to 52.3% of the <span class="emphasis"><em>C. elegans</em></span> genome (<a href="#bib46">Stein et al., 2003</a>). Merged WABA alignments, 4,837 in total, cover 84.6% of the <span class="emphasis"><em>C. briggsae</em></span> genome and 80.8% of the <span class="emphasis"><em>C. elegans</em></span> genome. Classification of these WABA alignments by searching for different types of breakpoints, which are defined as junctions
                  between adjacent colinear blocks with <span class="emphasis"><em>C. briggsae</em></span> supercontigs, resulted in 1,384 putative inversion events, 244 putative translocation events, and 2,735 putative transposition
                  events between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> genomes. By examining <span class="emphasis"><em>C. elegans</em></span> sequences, matching each end of the breakpoint junctions on <span class="emphasis"><em>C. briggsae</em></span> supercontigs we found a clear bias toward rearrangements being within the same ancestral chromosome (see <a href="#table2" title="Table 2. Chromosomal partners at colinearity breakpoints">Table 2</a>).
               </p>
               <div class="table"><a name="table2"></a><p class="title"><span class="bold"><strong>Table 2. Chromosomal partners at colinearity breakpoints</strong></span></p>
                  <table summary="Table 2. Chromosomal partners at colinearity breakpoints" border="1">
                     <colgroup>
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                     </colgroup>
                     <thead>
                        <tr>
                           <th align="center">&nbsp;</th>
                           <th align="center">I</th>
                           <th align="center">II</th>
                           <th align="center">III</th>
                           <th align="center">IV</th>
                           <th align="center">V</th>
                           <th align="center">X</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="center">I</td>
                           <td align="center">610</td>
                           <td align="center">77</td>
                           <td align="center">92</td>
                           <td align="center">123</td>
                           <td align="center">111</td>
                           <td align="center">52</td>
                        </tr>
                        <tr>
                           <td align="center">II</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">503</td>
                           <td align="center">90</td>
                           <td align="center">126</td>
                           <td align="center">131</td>
                           <td align="center">68</td>
                        </tr>
                        <tr>
                           <td align="center">III</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">469</td>
                           <td align="center">138</td>
                           <td align="center">122</td>
                           <td align="center">45</td>
                        </tr>
                        <tr>
                           <td align="center">IV</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">526</td>
                           <td align="center">165</td>
                           <td align="center">74</td>
                        </tr>
                        <tr>
                           <td align="center">V</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">732</td>
                           <td align="center">83</td>
                        </tr>
                        <tr>
                           <td align="center">X</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">&nbsp;</td>
                           <td align="center">229</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_7"></a>2.7.&nbsp;Operons, neighboring genes, and overlapping genes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Essentially all (96%) of the approximately 1,000 identified trans-spliced operons in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib4">Blumenthal and Gleason, 2003</a>) are conserved in <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib46">Stein et al., 2003</a>), suggesting that operons are under strong purifying selection pressure. Recent high throughput functional genomic projects
                  including microarray analysis (<a href="#bib27">Kim et al., 2001</a>) and SAGE (serial analysis of gene expression; <a href="#bib34">McKay et al., 2003</a>) have demonstrated that genes within operons in <span class="emphasis"><em>C. elegans</em></span> have positively correlated gene expression (<a href="#bib7">Chen and Stein, 2006</a>; <a href="#bib32">Lercher et al., 2003</a>). Whether genes within operons in <span class="emphasis"><em>C. briggsae</em></span> are positively correlated in gene expression remains to be examined.
               </p>
               <p>Correlation of gene expression is not limited to genes within operons. Recently, we have demonstrated that expression of neighboring
                  genes is also positively correlated (<a href="#bib7">Chen and Stein, 2006</a>). Using tissue-specific SAGE libraries generated by the <span class="emphasis"><em>C. elegans</em></span> Gene Expression Consortium, Chen and Stein (<a href="#bib7">Chen and Stein, 2006</a>) found that divergent neighboring gene pairs and parallel neighboring gene pairs are positively correlated in gene expression,
                  while convergent neighboring gene pairs show a greatly reduced correlation in gene expression that is lower than the background
                  correlation (see <a href="#figure2">Figure 2</a>; <a href="#bib7">Chen and Stein, 2006</a>). All of these correlations are dependent on the distances between neighboring gene pairs, the correlations approach background
                  as the distances become longer than 10kbp (see <a href="#figure2">Figure 2</a>). As extreme examples of neighboring genes, we analyzed genes that physically overlap in their gene models (introns, exons
                  and UTRs). Nested gene pairs and convergent gene pairs demonstrated lack of correlation in gene expression, while piggyback
                  gene pairs showed positive correlation in gene expression. Since no large-scale functional genomics projects such as microarray
                  analysis or SAGE have been carried out in <span class="emphasis"><em>C. briggsae</em></span>, correlation of gene expression for its neighboring genes and overlapping genes are unknown. However, similar types and extends
                  of correlation in gene expression likely exist in <span class="emphasis"><em>C. briggsae</em></span> because overlapping genes are well conserved between the two nematodes (<a href="#bib7">Chen and Stein, 2006</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_8"></a>2.8.&nbsp;Non-coding RNAs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>There are similar numbers of program-predicted non-coding RNA genes in <span class="emphasis"><em>C. briggsae</em></span> (962 and 191 fragmentary matches) and <span class="emphasis"><em>C. elegans</em></span> (838 and 212 fragmentary matches). A detailed list of various types of RNA genes is included in <a href="#table1" title="Table 1. Comparison between genomes of C. briggsae and C. elegans (primarily based on Stein et al. (2003))">Table 1</a>, which shows that both species have similar number of each type of non-coding RNAs. For example, both species have exactly
                  18 SL2 spliced leader genes, and 70 out of 105 identified <span class="emphasis"><em>C. elegans</em></span> miRNAs have close homologs in <span class="emphasis"><em>C. briggsae</em></span>.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="genbiolbriggsaefig2.jpg"><img src="genbiolbriggsaefig2_s.jpg" border="2" align="middle" alt="figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Distance-dependent correlation in gene expression.&nbsp;</b>x-axis is the distance between neighboring genes and y-axis is the Pearson correlation coefficient values. (Parallel) Parallel
                        gene pairs; (Divergent) divergent neighboring gene pairs; (Convergent) convergent neighboring gene pairs. Each point in the
                        figure represents a median value for the Pearson correlation coefficient for each group. Pearson correlation coefficient values
                        for cis- and trans-random gene pairs are essentially the same and are represented by a horizontal line.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_9"></a>2.9.&nbsp;Gene families
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Most gene families have similar sizes between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>, as suggested by TRIBE-MCL clustering analysis (<a href="#bib46">Stein et al., 2003</a>). However, there are exceptions in that some gene families are larger in <span class="emphasis"><em>C. briggsae</em></span> than in <span class="emphasis"><em>C. elegans</em></span> and vice versa. For example, there are many more putative chemosensory genes in <span class="emphasis"><em>C. elegans</em></span> than in <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib5">Chen et al., 2005</a>; <a href="#bib46">Stein et al., 2003</a>). Breaking down the putative chemosensory superfamily into smaller families, shows that some families have similar sizes
                  in <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>, while others, including the <span class="emphasis"><em>srab</em></span> family (<a href="#bib5">Chen et al. 2005b</a>) and the <span class="emphasis"><em>srz</em></span> family (<a href="#bib48">Thomas et al., 2005</a>), have twice as many genes in <span class="emphasis"><em>C. elegans</em></span> than in <span class="emphasis"><em>C. briggsae</em></span>. Such gene family size differences resulted largely from massive tandem duplications of individual genes in the <span class="emphasis"><em>C. elegans</em></span> genome. Tandem duplication were also observed in <span class="emphasis"><em>C. briggsae</em></span> but both the ratio and frequency of expansion was not as dramatic in <span class="emphasis"><em>C. briggsae</em></span> as in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib5">Chen et al., 2005</a>). Altogether, TRIBE-MCL clustering analysis demonstrated that for gene clusters with five or more <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> genes, there were 202 clusters (corresponding to gene families) with gene number differences of at least 2-fold between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib46">Stein et al., 2003</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;Evolutionary studies
                     </h2>
                  </div>
               </div>
            </div>
            <p>Morphologically <span class="emphasis"><em>C. briggsae</em></span> is almost identical to <span class="emphasis"><em>C. elegans</em></span>. However, as described above, a comparison of the genome sequences between the two species has revealed a significant number
               of species-specific genes (~2,000 in total; <a href="#bib46">Stein et al., 2003</a>). Hence, an obvious question is: why are these changes not reflected in significant morphological or behavioral differences?
               Perhaps such features are under strong selective pressure and have resulted in compensatory changes in the genome. Alternatively,
               it is possible that the two species exhibit differences in other developmental and physiological processes that have yet to
               be revealed. In the following subsections we summarize studies focusing on known differences between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> (as well as other <span class="emphasis"><em>Caenorhabditis</em></span> nematodes). With increasing research into nematode evolution, future studies are expected to reveal additional differences
               and the roles of species-specific genes in mediating some of these changes.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_1"></a>3.1.&nbsp;Development of the excretory duct cell
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The excretory system in <span class="emphasis"><em>C. elegans</em></span> plays a critical role in osmoregulation. It is made up of just three cells: the canal cell, the duct cell and the pore cell.
                  The duct and pore cells connect the canal to the outside world and help eliminate excess body fluid. In <span class="emphasis"><em>C. briggsae</em></span> the excretory system looks morphologically identical, however <span class="emphasis"><em>C. briggsae</em></span> animals have more anterior duct opening than <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib50">Wang and Chamberlin, 2002</a>). This difference in the placement of duct opening is the result of the altered expression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-48;class=Gene" target="_blank">lin-48</a></em></span>, a <span class="emphasis"><em>Drosophila ovo</em></span> family member (<a href="#bib50">Wang and Chamberlin, 2002</a>). In <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-48;class=Gene" target="_blank">lin-48</a></em></span> is expressed in the excretory duct cell, a subset of the hindgut cells and a few neurons. In contrast, <span class="emphasis"><em>C. briggsae <a href="http://www.wormbase.org/db/get?name=lin-48;class=Gene" target="_blank">lin-48</a></em></span> does not express in the duct cell due to an evolutionary change in the regulation of <span class="emphasis"><em>Cbr-lin-48</em></span>. This demonstrates that evolutionary changes in the regulation of <span class="emphasis"><em>C. briggsae</em></span> genes (or <span class="emphasis"><em>C. elegans</em></span> genes) can give rise to morphological differences.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_2"></a>3.2.&nbsp;Sex determination
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Regulation of sex-specific developmental programs is of prime importance for the propagation and diversity of multicellular
                  organisms. Experimental findings in metazoans have revealed that, in spite of their critical role, many sex determination
                  genes and genetic pathways have rapidly evolved. Because of their morphological diversity and two different modes of reproduction
                  (hermaphroditic, e.g., <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>, and gonochoristic, e.g., <span class="emphasis"><em>C. remanei, C. japonica</em></span> and <span class="emphasis"><em>C. sp.CB5161</em></span>), nematodes provide excellent opportunity to study the molecular basis of sexual differentiation and evolution. In this respect,
                  the <span class="emphasis"><em>C. briggsae</em></span> genome has facilitated identification of the sex-determination genes and study of their functional differences. Given that
                  hermaphroditism has evolved independently in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib29">Kiontke et al., 2004</a>), it is conceivable that the underlying regulatory mechanisms have diverged between the two species. A survey of the <span class="emphasis"><em>C. elegans</em></span> sex pathway genes in <span class="emphasis"><em>C. briggsae</em></span> genome has revealed significant sequence divergence in some genes suggesting functional differences in the mode of sex determination.
                  Consistent with this observation, <a href="http://www.wormbase.org/db/get?name=FEM-3;class=Protein" target="_blank">FEM-3</a> (novel) and <a href="http://www.wormbase.org/db/get?name=TRA-2;class=Protein" target="_blank">TRA-2</a> (transmembrane domain) proteins have rapidly evolved in <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span> (<a href="#bib17">Haag and Ackerman, 2005</a>; <a href="#bib18">Haag et al., 2002</a>). In <span class="emphasis"><em>C. elegans</em></span>, the two proteins are known to physically interact with each other (<a href="#bib35">Mehra et al., 1999</a>). While this interaction appears to be conserved in <span class="emphasis"><em>C. briggsae</em></span>, the two proteins function in strictly species-specific manners. Furthermore, RNA interference (RNAi)-mediated inactivation
                  of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fem-1;class=Gene" target="_blank">fem-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fem-2;class=Gene" target="_blank">fem-2</a></em></span> genes in <span class="emphasis"><em>C. briggsae</em></span> reveal functional differences from <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib47">Stothard and Pilgrim, 2003</a>). In a separate study, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fog-2;class=Gene" target="_blank">fog-2</a></em></span>, which is required for hermaphrodite spermatogenesis in <span class="emphasis"><em>C. elegans</em></span>, has been shown to be absent in <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib37">Nayak et al., 2005</a>). Hence, the genome sequence of <span class="emphasis"><em>C. briggsae</em></span> promises new understanding of the evolution of sex-determination pathways.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_3"></a>3.3.&nbsp;Development of the vulva
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Nematode vulval development is an established model for comparative developmental biology. In <span class="emphasis"><em>C. elegans</em></span>, six vulval precursor cells (VPCs) (Pn.p, where n=3&#8211;8) respond to a spatially graded inductive signal from the gonadal anchor
                  cell (AC) and give rise to a 3&deg;-3&deg;-2&deg;-1&deg;-2&deg;-3&deg; cell fate pattern. While the 1&deg; and 2&deg; VPCs divide to give rise to vulval tissue,
                  the 3&deg; lineage cells fuse into the surrounding hypodermal syncytial cell (hyp7). The cell ablation experiments have shown
                  that all six VPCs are equally capable of adopting 1&deg; and 2&deg; fates (<a href="#bib23">Horvitz and Sternberg, 1991</a>). For example, ablation of all VPCs but the <a href="http://www.wormbase.org/db/get?name=P3.p;class=Cell" target="_blank">P3.p</a>, allows it to generate vulval progeny although it does not do so in an intact animal. By contrast, in <span class="emphasis"><em>C. briggsae</em></span> <a href="http://www.wormbase.org/db/get?name=P3.p;class=Cell" target="_blank">P3.p</a> does not appear to be competent by this assay (<a href="#bib12">Delattre and Felix, 2001</a>). However, in certain multivulva mutants of <span class="emphasis"><em>C. briggsae</em></span> <a href="http://www.wormbase.org/db/get?name=P3.p;class=Cell" target="_blank">P3.p</a> is capable of being induced and give rise to vulval progeny suggesting that negative regulation may restrict the fate of
                  <a href="http://www.wormbase.org/db/get?name=P3.p;class=Cell" target="_blank">P3.p</a> in wild-type animals (B. Gupta, unpublished). Thus, the competence of <a href="http://www.wormbase.org/db/get?name=P3.p;class=Cell" target="_blank">P3.p</a> appears to have evolved between the two species.
               </p>
               <p>Rudel and Kimble (<a href="#bib44">Rudel and Kimble, 2001</a>; <a href="#bib45">Rudel and Kimble, 2002</a>) have examined evolution of Notch-family members, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-12;class=Gene" target="_blank">lin-12</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span>, in <span class="emphasis"><em>Caenorhabditis</em></span> nematodes (<span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei)</em></span>. Their results suggest that in <span class="emphasis"><em>C. briggsae</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-12;class=Gene" target="_blank">lin-12</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span> have acquired novel functions. Furthermore, there appears to be two <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-12;class=Gene" target="_blank">lin-12</a></em></span>-like genes in <span class="emphasis"><em>C. briggsae</em></span> compared to only one in <span class="emphasis"><em>C. elegans</em></span>. In <span class="emphasis"><em>C. elegans</em></span> <a href="http://www.wormbase.org/db/get?name=LIN-12;class=Protein" target="_blank">LIN-12</a> and <a href="http://www.wormbase.org/db/get?name=GLP-1;class=Protein" target="_blank">GLP-1</a> are functionally interchangeable and in wild-type animals perform some overlapping functions, but they are uniquely required
                  in some tissues (see <a href="http://www.wormbook.org/chapters/www_lin12Notch/lin12notch.html" target="_top">LIN-12/Notch signaling in <span class="emphasis"><em>C. elegans</em></span></a>). <a href="http://www.wormbase.org/db/get?name=GLP-1;class=Protein" target="_blank">GLP-1</a> is necessary during early embryogenesis and later in the development and proliferation of the germline whereas <a href="http://www.wormbase.org/db/get?name=LIN-12;class=Protein" target="_blank">LIN-12</a>, but not <a href="http://www.wormbase.org/db/get?name=GLP-1;class=Protein" target="_blank">GLP-1</a>, is required during lateral signaling events in some somatic tissues such as the vulva. In <span class="emphasis"><em>C. elegans</em></span> loss of function alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-12;class=Gene" target="_blank">lin-12</a></em></span> exhibit protruding vulva (Pvul) phenotype as a result of the fate conversion of the 2&deg; lineage cells <a href="http://www.wormbase.org/db/get?name=P5.p;class=Cell" target="_blank">P5.p</a> and <a href="http://www.wormbase.org/db/get?name=P7.p;class=Cell" target="_blank">P7.p</a> into the 1&deg; type cell lineage (see <a href="http://www.wormbook.org/chapters/www_lin12Notch/lin12notch.html" target="_top">LIN-12/Notch signaling in <span class="emphasis"><em>C. elegans</em></span></a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span> loss of function alleles, on the other hand, do not affect vulval development. However, in <span class="emphasis"><em>C. briggsae</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span> is required for vulval development. <span class="emphasis"><em>Cbr-glp-1</em></span>(RNAi) animals exhibit a multivulva phenotype suggesting that wild-type <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span> functions to regulate the competence of the VPCs. Likewise examination of the role of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-12;class=Gene" target="_blank">lin-12</a></em></span> revealed that, unlike <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-12;class=Gene" target="_blank">lin-12</a></em></span> in <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-12;class=Gene" target="_blank">lin-12</a></em></span> in <span class="emphasis"><em>C. briggsae</em></span> is essential for larval viability. <span class="emphasis"><em>Cbr-lin-12</em></span>(RNAi) animals arrest during L<sub>1</sub> larval stage, a phenotype that is not observed in <span class="emphasis"><em>Cel-lin-12</em></span>(RNAi) animals. Functional differences between <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-12;class=Gene" target="_blank">lin-12</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span> in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> are supported by the low level of amino acid conservation (68% and 61% identities, respectively) and may result from evolutionary
                  changes in protein-protein interactions.
               </p>
               <p>In a separate study, Kirouac and Sternberg (<a href="#bib30">Kirouac and Sternberg, 2003</a>) have examined regulation of the three orthologs that are expressed in vulval cells (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-17;class=Gene" target="_blank">egl-17</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=zmp-1;class=Gene" target="_blank">zmp-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=cdh-3;class=Gene" target="_blank">cdh-3</a></em></span>) in <span class="emphasis"><em>C. briggsae</em></span> and have identified many key regulatory elements that are conserved between the two species. In spite of this conservation,
                  the dynamics of the expression of these genes in vulval cells have evolved. Hence, for example, the <span class="emphasis"><em>C. elegans</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-17;class=Gene" target="_blank">egl-17</a>::GFP</em></span> reporter, which is always expressed in vulC and vulD cell types in <span class="emphasis"><em>C. elegans</em></span>, often fails to express in the same set of cells in <span class="emphasis"><em>C. briggsae</em></span>. The expression pattern of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=cdh-3;class=Gene" target="_blank">cdh-3</a>::GFP</em></span> reporter, on the other hand, is significantly diverged in <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib30">Kirouac and Sternberg, 2003</a>). This finding is somewhat similar to the regulatory differences reported for the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=pes-1;class=Gene" target="_blank">pes-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fkh-2;class=Gene" target="_blank">fkh-2</a></em></span> forkhead family members (<a href="#bib36">Molin et al., 2000</a>). While both these genes are functionally redundant in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> embryogenesis, <span class="emphasis"><em>C. briggsae <a href="http://www.wormbase.org/db/get?name=fkh-2;class=Gene" target="_blank">fkh-2</a></em></span> has acquired additional novel function.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_4"></a>3.4.&nbsp;Development of the male tail
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The male tail, involved in copulation, exhibits considerable differences in morphology among the Rhabditidae family (including
                  <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>). This variability is one criteria used by researchers to distinguish between and classify various nematodes (<a href="#bib13">Fitch, 1997</a>). <span class="emphasis"><em>C. briggsae</em></span>'s male tail has a subtle difference, relative to <span class="emphasis"><em>C. elegans</em></span>, in its pattern of ray morphology, specifically there is a tendency of ray 3 to fuse with ray 4. Baird and colleagues (<a href="#bib2">Baird et al., 2005</a>) have used recombinant inbred lines (RILs) to map at least two loci (linked to <span class="emphasis"><em>Cbr-unc-22</em></span>) that appear to play a role in this morphological variation.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_5"></a>3.5.&nbsp;Thermal sensitivity
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Studies focusing on the ability of nematodes to survive in different environmental conditions have shown evolutionary differences
                  between <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>. Examination of the thermal sensitivity of <span class="emphasis"><em>C. elegans</em></span> has revealed significant reduction in the brood size at temperatures above 25&deg;C and almost complete sterility above 27&deg;C
                  (M. Ailion and J. Thomas, personal communication; B. Gupta, unpublished). By contrast, <span class="emphasis"><em>C. briggsae</em></span> grows well at 27.5&deg;C with no significant reduction in brood size (<a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a>: 200 +/<span class="symbol">&#8722;</span> 35 compared to 234 +/<span class="symbol">&#8722;</span> 25 at 20&deg;C; <a href="#bib14">Fodor et al., 1983</a>). In fact, <span class="emphasis"><em>C. briggsae</em></span> can grow and develop to adulthood at 30&deg;C (M. Ailion and J. Thomas, personal communication; RJ, unpublished). These studies
                  have been done using multiple isolates of <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> and all isolates of each species behave similarly. The formation of dauer larva, a developmentally arrested diapause stage,
                  is another temperature-dependent trait that exhibits differences between <span class="emphasis"><em>C. briggsae</em></span> and other <span class="emphasis"><em>Caenorhabditis</em></span> nematodes (M. Ailion and J. Thomas, personal communication; T. Inoue and P. Sternberg, personal communication). <span class="emphasis"><em>C. elegans</em></span> when grown at temperature above 25&deg;C forms dauers even when well-fed, and analysis of mutants show these temperatures to
                  be a strong dauer-inducing condition. In contrast, well-fed <span class="emphasis"><em>C. briggsae</em></span> does not form dauers at these temperatures and analysis of mutants have failed to detect temperature-induced dauer induction
                  at temperatures above 25&deg;C (T. Inoue, personal communication).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_6"></a>3.6.&nbsp;Specification of neural asymmetry
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In multicellular organisms many cells and tissues exhibit morphological and functional asymmetries. <span class="emphasis"><em>C. elegans</em></span> has been successfully used to study function of some of the genes that confer asymmetry on neural cells (<a href="#bib22">Hobert et al., 2002</a>). Ortiz and colleagues (<a href="#bib39">Ortiz et al., 2006</a>) have carried out a detailed study of the sequence and expression pattern of the guanylyl cyclase (<span class="emphasis"><em>gcy</em></span>) genes in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>. A total of 23 (out of 33) <span class="emphasis"><em>C. briggsae gcy</em></span> genes have clear one-to-one orthologs in <span class="emphasis"><em>C. elegans</em></span> whereas the non-orthologous family members (10 in <span class="emphasis"><em>C. briggsae</em></span> and 11 in <span class="emphasis"><em>C. elegans</em></span>) appear to have arisen as a result of species-specific gene duplication events. The expression patterns of two <span class="emphasis"><em>gcy</em></span> genes (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=gcy-4;class=Gene" target="_blank">gcy-4</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=gcy-19;class=Gene" target="_blank">gcy-19</a></em></span>) in <span class="emphasis"><em>C. briggsae</em></span> ASE neurons (<a href="http://www.wormbase.org/db/get?name=ASEL;class=Cell" target="_blank">ASEL</a> and <a href="http://www.wormbase.org/db/get?name=ASER;class=Cell" target="_blank">ASER</a>) show striking evolutionary differences that resulted from changes in <span class="emphasis"><em>cis</em></span>-acting elements and <span class="emphasis"><em>trans</em></span>-acting factors. Since the <span class="emphasis"><em>C. elegans</em></span> ASE neurons exhibit functional left/right asymmetry and express distinct set of <span class="emphasis"><em>gcy</em></span> genes, the mechanism controlling regulation of the ASE laterality appears to have evolved in <span class="emphasis"><em>C. briggsae</em></span>.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Genetic approach to study functions of <span class="emphasis"><em>C. briggsae</em></span> genes
                     </h2>
                  </div>
               </div>
            </div>
            <p>The developmental and behavioral differences between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> summarized above are just the beginning. With increasing number of laboratories focusing on <span class="emphasis"><em>briggsae-elegans</em></span> comparative studies, many more differences are likely to be revealed in future. This argues for a need to develop resources
               and tools to facilitate experimental studies in <span class="emphasis"><em>C. briggsae</em></span>.
            </p>
            <p>Given the availability of the <span class="emphasis"><em>C. briggsae</em></span> genome sequence, RNAi could be one approach to examine the role of known genes in a given biological process. The success
               of the RNAi approach in <span class="emphasis"><em>C. elegans</em></span> has made it a method of choice for large-scale functional studies in other nematodes. However, it should be pointed out that
               while RNAi-based screen is likely to identify some of the functional genes, factors such as weak RNAi penetrance, pleotropy
               and genetic redundancy might not reveal function of many genes. If there is a difference in the RNAi phenotype of a gene it
               is unclear if it is a quantitative difference in the use of that gene, or in its susceptibility to RNAi. In this respect,
               genetic approaches promise success. Studies of genetic mutations have revealed a spectrum of phenotypes associated with hypomorphic
               (loss-of-function), hypermorphic (gain-of-function) and neomorphic (novel) alleles. Achieving these types of results are not
               possible by using standard RNAi or gene knock-out screens. Highly evolving genes (e.g., GPCR family, sex-determination genes)
               may have acquired novel functions that can best be identified by isolating genetic mutations, then studying their phenotypes
               and genetic interactions. To facilitate forward genetic experiments in <span class="emphasis"><em>C. briggsae</em></span>, mutations with visible phenotypes such as <a href="http://www.wormbase.org/db/get?name=Unc;class=Phenotype" target="_blank">Unc</a> (Uncoordinated), <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a> (Dumpy) and <a href="http://www.wormbase.org/db/get?name=Rol;class=Phenotype" target="_blank">Rol</a> (Roller) have been isolated in different laboratories (B. Gupta, P. Sternberg and D. Baillie, unpublished; <a href="#table3" title="Table 3. The list of genetically characterized mutations in C. briggsae">Table 3</a>; <a href="http://wormlab.caltech.edu/briggsae" target="_blank">the <span class="emphasis"><em>C. briggsae</em></span> website</a>). A number of these have been mapped leading to the definition of six independent linkage groups (see <a href="#figure3">Figure 3</a>; <a href="http://wormlab.caltech.edu/briggsae" target="_blank">the <span class="emphasis"><em>C. briggsae</em></span> website</a>). The linkage map will facilitate genetic analysis of the <span class="emphasis"><em>C. briggsae</em></span> genes. Some of the studies that are currently ongoing involve development of the vulva (B. Gupta, unpublished), sex determination
               (<a href="#bib20">Hill et al., 2006</a>), ray pattern formation (<a href="#bib2">Baird et al., 2005</a>), and the excretory system features and salt tolerance behavior (H. Chamberlin, personal communication).
            </p>
            <div class="table"><a name="table3"></a><p class="title"><span class="bold"><strong>Table 3. The list of genetically characterized mutations in</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. briggsae</strong></span></em></span></p>
               <table summary="Table 3. The list of genetically characterized mutations in C. briggsae" border="1">
                  <colgroup>
                     <col align="center">
                     <col align="center">
                     <col align="center">
                     <col align="center">
                     <col align="center">
                     <col align="left">
                  </colgroup>
                  <thead>
                     <tr>
                        <th align="center">Locus</th>
                        <th align="center">AKA</th>
                        <th align="center">LG</th>
                        <th align="center">Reference allele</th>
                        <th align="center">Alleles</th>
                        <th align="left">Phenotype</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td align="center"><span class="emphasis"><em>bli(sy5259)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">II</td>
                        <td align="center"><span class="emphasis"><em>sy5259</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Blister along the entire body, symmetrical</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>Cbr-daf-3</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sy5417</em></span></td>
                        <td align="center">1</td>
                        <td align="left">daf-d</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>Cbr-daf-4</em></span></td>
                        <td align="center"><span class="emphasis"><em>dau-1</em></span></td>
                        <td align="center">III</td>
                        <td align="center"><span class="emphasis"><em>sa973</em></span></td>
                        <td align="center">1</td>
                        <td align="left">ts for small size and dauer (daf-c) at 27&deg;C</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(<a href="http://www.wormbase.org/db/get?name=s1281;class=Variation" target="_blank">s1281</a>)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-1</em></span></td>
                        <td align="center">III</td>
                        <td align="center"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=s1281;class=Variation" target="_blank">s1281</a></em></span></td>
                        <td align="center">3</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(s1282)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-2</em></span></td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>s1282</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a>, slightly paralyzed
                        </td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5039)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-3</em></span></td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sy5039</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(s1272)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-4</em></span></td>
                        <td align="center">III</td>
                        <td align="center"><span class="emphasis"><em>s1272</em></span></td>
                        <td align="center">19</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(s1286)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-5</em></span></td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em>s1286</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5027)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-7</em></span></td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em>sy5027</em></span></td>
                        <td align="center">2</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a>, weak roller
                        </td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5017)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-8</em></span></td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em>sy5017</em></span></td>
                        <td align="center">4</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a>, slow moving
                        </td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5048)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-9; <a href="http://www.wormbase.org/db/get?name=rot-1;class=Gene" target="_blank">rot-1</a></em></span></td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sy5048</em></span></td>
                        <td align="center">2</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a>, Adults are weak rollers
                        </td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5064)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-10</em></span></td>
                        <td align="center">II</td>
                        <td align="center"><span class="emphasis"><em>sy5064</em></span></td>
                        <td align="center">2</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a>, very slow and often sick
                        </td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5057)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-12</em></span></td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sy5057</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5005)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-13</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center"><span class="emphasis"><em>sy5005</em></span></td>
                        <td align="center">3</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a>, weakly <a href="http://www.wormbase.org/db/get?name=Egl;class=Phenotype" target="_blank">Egl</a>, weakly <a href="http://www.wormbase.org/db/get?name=Rol;class=Phenotype" target="_blank">Rol</a> when backing
                        </td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5076)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-14</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center"><span class="emphasis"><em>sy5076</em></span></td>
                        <td align="center">5</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5025)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-16</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center"><span class="emphasis"><em>sy5025</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sy5075)</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-17</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center"><span class="emphasis"><em>sy5075</em></span></td>
                        <td align="center">2</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a>, poor moving
                        </td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sa971)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sa971</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sa995)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sa995</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(sa996)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em>sa996</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(nm3)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>nm3</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>dpy(nm4)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">II</td>
                        <td align="center"><span class="emphasis"><em>nm4</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>lev(sy5436)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">I</td>
                        <td align="center"><span class="emphasis"><em>sy5436</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Resistant to 1mM levamisole, <a href="http://www.wormbase.org/db/get?name=Unc;class=Phenotype" target="_blank">Unc</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>lev(sy5437)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">I</td>
                        <td align="center"><span class="emphasis"><em>sy5437</em></span></td>
                        <td align="center">3</td>
                        <td align="left">Resistant to 1mM levamisole, <a href="http://www.wormbase.org/db/get?name=Unc;class=Phenotype" target="_blank">Unc</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>lev(<a href="http://www.wormbase.org/db/get?name=sy5440;class=Variation" target="_blank">sy5440</a>)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">I</td>
                        <td align="center"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sy5440;class=Variation" target="_blank">sy5440</a></em></span></td>
                        <td align="center">1</td>
                        <td align="left">Resistant to 1mM levamisole, <a href="http://www.wormbase.org/db/get?name=Unc;class=Phenotype" target="_blank">Unc</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>Cbr-lin-11</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">I</td>
                        <td align="center"><span class="emphasis"><em>sy5336</em></span></td>
                        <td align="center">2</td>
                        <td align="left">Pvul, <a href="http://www.wormbase.org/db/get?name=Egl;class=Phenotype" target="_blank">Egl</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>lin(sy5216)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em>sy5216</em></span></td>
                        <td align="center">2</td>
                        <td align="left">Multivulva</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>lin(sy5342)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">II</td>
                        <td align="center"><span class="emphasis"><em>sy5342</em></span></td>
                        <td align="center">2</td>
                        <td align="left">Multivulva</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>lon(sy5227)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sy5227</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Thin and long, males mate poorly</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>Cbr-pry-1</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">I</td>
                        <td align="center"><span class="emphasis"><em>sy5353</em></span></td>
                        <td align="center">2</td>
                        <td align="left">Multivulva</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>rol(sy5143)</em></span></td>
                        <td align="center"><span class="emphasis"><em>rot-2</em></span></td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sy5143</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Right-handed roller</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>rol(sy5412)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">&nbsp;</td>
                        <td align="center"><span class="emphasis"><em>sy5412</em></span></td>
                        <td align="center">1</td>
                        <td align="left">All stages right-handed roller</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>Cbr-sma-6</em></span></td>
                        <td align="center"><span class="emphasis"><em>cby-15</em></span></td>
                        <td align="center">II</td>
                        <td align="center"><span class="emphasis"><em>sy5148</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a>, Small size, fairly active
                        </td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>sma(sa985)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">&nbsp;</td>
                        <td align="center"><span class="emphasis"><em>sa985</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Small size, active</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>sma(sy5002)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">&nbsp;</td>
                        <td align="center"><span class="emphasis"><em>sy5002</em></span></td>
                        <td align="center">2</td>
                        <td align="left">Small size, active</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>sma(sy5159)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sy5159</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="left">Small size</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(<a href="http://www.wormbase.org/db/get?name=s1270;class=Variation" target="_blank">s1270</a>)</em></span></td>
                        <td align="center"><span class="emphasis"><em>mip-1</em></span></td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=s1270;class=Variation" target="_blank">s1270</a></em></span></td>
                        <td align="center">11</td>
                        <td align="left">Twicher, thin</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(ge19)</em></span></td>
                        <td align="center"><span class="emphasis"><em>mip-3</em></span></td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em>ge19</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Almost paralyzed, tends to coil</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sy5077)</em></span></td>
                        <td align="center"><span class="emphasis"><em>mip-5</em></span></td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sy5077</em></span></td>
                        <td align="center">1</td>
                        <td align="left">almost paralyzed, kinker</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sy5019)</em></span></td>
                        <td align="center"><span class="emphasis"><em>mip-6</em></span></td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em>sy5019</em></span></td>
                        <td align="center">1</td>
                        <td align="left">A coiler, weak kinker, paralyzed</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sa984)</em></span></td>
                        <td align="center"><span class="emphasis"><em>mip-10</em></span></td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em>sa984</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Slow moving, weak <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>Cbr-unc-4</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">II</td>
                        <td align="center"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sy5341;class=Variation" target="_blank">sy5341</a></em></span></td>
                        <td align="center">1</td>
                        <td align="left">Active, fails to back when touched</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sa972)</em></span></td>
                        <td align="center"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=csp-1;class=Gene" target="_blank">csp-1</a></em></span></td>
                        <td align="center">III</td>
                        <td align="center"><span class="emphasis"><em>sa972</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Slow moving, defective in backward movement, slightly smaller than wild-type</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sa986)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">V</td>
                        <td align="center"><span class="emphasis"><em>sa986</em></span></td>
                        <td align="center">1</td>
                        <td align="left">kinker, slight curler</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sa987)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">IV</td>
                        <td align="center"><span class="emphasis"><em>sa987</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sa988)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sa988</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Kinker, weak <a href="http://www.wormbase.org/db/get?name=Dpy;class=Phenotype" target="_blank">Dpy</a></td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sa997)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">V</td>
                        <td align="center"><span class="emphasis"><em>sa997</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Slightly thin, slow movement in adults</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sy5094)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">V</td>
                        <td align="center"><span class="emphasis"><em>sy5094</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Kinker, curler</td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sy5329)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">X</td>
                        <td align="center"><span class="emphasis"><em>sy5329</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Medium <a href="http://www.wormbase.org/db/get?name=Unc;class=Phenotype" target="_blank">Unc</a>, poor in backward movement
                        </td>
                     </tr>
                     <tr>
                        <td align="center"><span class="emphasis"><em>unc(sy5422)</em></span></td>
                        <td align="center">&nbsp;</td>
                        <td align="center">III</td>
                        <td align="center"><span class="emphasis"><em>sy5422</em></span></td>
                        <td align="center">1</td>
                        <td align="left">Strong <a href="http://www.wormbase.org/db/get?name=Unc;class=Phenotype" target="_blank">Unc</a>, small size, does not form dauer
                        </td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;Tools for genetic studies
                     </h2>
                  </div>
               </div>
            </div>
            <p>Several tools are currently available to study evolutionary changes in the function of <span class="emphasis"><em>C. briggsae</em></span> genes and carry out <span class="emphasis"><em>C. briggsae</em></span>-<span class="emphasis"><em>C. elegans</em></span> comparative studies.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_1"></a>5.1.&nbsp;RNAi
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Among different approaches to deliver double-stranded RNA (dsRNA) to intact <span class="emphasis"><em>C. elegans</em></span> cells, RNAi by feeding (systemic RNAi) is a rapid method to study gene function (<a href="#bib49">Timmons et al., 2001</a>). While the RNAi by microinjection technique has been successfully used to examine function of some of the genes in <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib47">Stothard and Pilgrim, 2003</a>), wild-type <span class="emphasis"><em>C. briggsae</em></span> <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> is resistant to systemic RNAi. This is due to the lack of a functional <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sid-2;class=Gene" target="_blank">sid-2</a></em></span> gene since the <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> worms expressing <span class="emphasis"><em>C. elegans</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sid-2;class=Gene" target="_blank">sid-2</a></em></span> (<span class="emphasis"><em>Cel-sid-2</em></span>) are susceptible to systemic RNAi (C. Hunter, personal communication). Hence, a <span class="emphasis"><em>C. briggsae</em></span> transgenic strain carrying integrated <span class="emphasis"><em>Cel-sid-2</em></span> array can be used in functional genomic studies.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="genbiolbriggsaefig3.jpg"><img src="genbiolbriggsaefig3_s.jpg" border="2" align="middle" alt="figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 3. The phenotypic marker-based genetic linkage map of <span class="emphasis"><em>C. briggsae</em></span> (v9.0).&nbsp;</b>The <span class="emphasis"><em>C. elegans</em></span> orthologs, that have been experimentally confirmed, are shown below the lines. The question marked loci represent candidates
                        based on the genetic properties (phenotype, allele frequency and map position) and require further confirmation.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_2"></a>5.2.&nbsp;Mutant strains
                        </h3>
                     </div>
                  </div>
               </div>
               <p>To facilitate genetic studies of <span class="emphasis"><em>C. briggsae</em></span>, a large collection of mutant strains (more than 200) is freely available to the community (see <a href="#table3" title="Table 3. The list of genetically characterized mutations in C. briggsae">Table 3</a>). These strains not only allow genetic and molecular analysis of the involved loci but also serve as useful markers in phenotypic
                  analyses and linkage mapping of new mutants. While most of these are curated in three different laboratories (B. Gupta, P.
                  Sternberg and D. Baillie), some of the strains are available at the <a href="http://wormbase.org/db/gene/strain" target="_blank"><span class="emphasis"><em>Caenorhabditis</em></span> Genetics Center</a> for centralized distribution. A comprehensive list of the mutants can be found at the <a href="http://wormlab.caltech.edu/briggsae" target="_blank"><span class="emphasis"><em>C. briggsae</em></span> website</a>.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_3"></a>5.3.&nbsp;Genetic linkage maps
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The phenotypic marker-based genetic linkage map is currently under construction (see <a href="#figure3">Figure 3</a>; also see <a href="http://wormlab.caltech.edu/briggsae" target="_blank">the <span class="emphasis"><em>C. briggsae</em></span> website</a>; B. Gupta, T. Inoue, RJ, A. Mah, D. Ballie and P. Sternberg, unpublished). This has involved standard 2 and 3-factor genetic
                  mapping of mutations that exhibit visible phenotypes and cloning of some of the loci. In parallel, R. Waterston and colleagues
                  (personal communication) are working to construct a single nucleotide polymorphism (SNP)-based <a href="http://snp.wustl.edu/snp-research/c-briggsae/" target="_blank">genetic linkage map of <span class="emphasis"><em>C. briggsae</em></span></a>. This approach involves careful mapping of SNPs identified by partial sequencing of two <span class="emphasis"><em>C. briggsae</em></span> isolates <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a> and <a href="http://www.wormbase.org/db/get?name=VT847;class=Strain" target="_blank">VT847</a>. The current version of the SNP-based genetic linkage map (v3.1) is based on the 248 genotyped SNPs that are roughly equally
                  distributed over the six chromosomes. Integration of the SNP-based and mutant-based maps, which is ongoing (B. Gupta and R.
                  Miller, unpublished), will allow researchers to place their gene of interest on different linkage groups, examine synteny
                  with <span class="emphasis"><em>C. elegans</em></span>, carry out epistasis and genetic interaction experiments. This will facilitate cloning of the <span class="emphasis"><em>C. briggsae</em></span> genes, in particular those with the novel mutant phenotypes i.e., ones for which there are no candidate genes proposed from
                  our knowledge of <span class="emphasis"><em>C. elegans</em></span>, and study evolutionary changes in gene function.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_4"></a>5.4.&nbsp;Mos1 mutagenesis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Mos1 is a <span class="emphasis"><em>Drosophila</em></span> transposon that has been successfully used in <span class="emphasis"><em>C. elegans</em></span> to generate mutations (<a href="#bib3">Bessereau et al., 2001</a>). The advantage of this system is that it can be used as a sequence tag to rapidly identify the mutated gene by PCR and therefore
                  circumvent the need for time-consuming genetic mapping and gene cloning experiments. Dr. Marie-Anne Felix has tested the applicability
                  of the system in <span class="emphasis"><em>C. briggsae</em></span> and successfully recovered mutations in three genes (<a href="http://snp.wustl.edu/snp-research/c-briggsae/mos-mutagenesis.html" target="_blank">http://snp.wustl.edu/snp-research/c-briggsae/mos-mutagenesis.html</a>). While the experimental protocol is in the early phase of development in <span class="emphasis"><em>C. briggsae</em></span>, it has potential of rapidly isolating alleles of the <span class="emphasis"><em>C. elegans</em></span> orthologs (M.-A. Felix, personal communication; B. Gupta, unpublished).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_5"></a>5.5.&nbsp;Collection of wild isolates and recombinant inbred lines
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The wild populations of <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> serve as useful resources for studies involving natural variation and ecology of the species. Comparison of the genomic DNA
                  sequences in these strains has shown greater nucleotide diversity in <span class="emphasis"><em>C. briggsae</em></span> compared to <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib15">Graustein et al., 2002</a>; <a href="#bib24">Jovelin et al., 2003</a>). Currently more than 60 wild isolates of <span class="emphasis"><em>C. briggsae</em></span> are available, they have been collected from various regions of the world (S. Baird and M.-A. Felix, personal communications).
                  These include 57 new isolates that are part of a recent study looking at the pattern of polymorphisms in 63 strains across
                  the three continents (Asia, Europe and America; <a href="#bib11">Cutter et al., 2006</a>). In addition, a large number of recombinant inbred lines (RILs) have been generated. These are derived from two sets of
                  crosses between <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a>, <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a> and <a href="http://www.wormbase.org/db/get?name=VT847;class=Strain" target="_blank">VT847</a> (set 1: <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> and <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a>, set 2: <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> and <a href="http://www.wormbase.org/db/get?name=VT847;class=Strain" target="_blank">VT847</a>; S. Baird, personal communication). Besides their use in mapping quantitative trait loci (QTLs), the RILs are playing key
                  role in construction of the SNP-based genetic linkage map (R. Miller and S. Baird, personal communications).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_6"></a>5.6.&nbsp;Genomic clones
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The genomic clones of <span class="emphasis"><em>C. briggsae</em></span> (in the form of BAC and fosmid clones) were generated to construct a high resolution physical map to help assemble the sequenced
                  genome. The BAC clones are available from the <a href="http://bacpac.chori.org/library.php?id=26" target="_blank">Children's Hospital Oakland Research Institute</a>.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>We are grateful to David Baillie, Marie-Anne Felix, Craig Hunter, Raymond Miller, Scott Baird, Helen Chamberlin, Michael Ailion,
               James Thomas, Takao Inoue and Robert Waterston for sharing their unpublished work. We thank David Baillie for sharing his
               ideas and reviewing the manuscript. B. Gupta is a tier II Canada Research Chair and supported by grants from NSERC and NIH.
               NC is supported by a grant from NSERC, a faculty startup fund from Simon Fraser University, and the Simon Fraser University
               President's Research Grants Fund. RJ is supported by NSERC and CIHR.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e2982"></a>7.&nbsp;Bibliography
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., and Lipman, D.J. (1997). Gapped BLAST and
                        PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. <span class="emphasis"><em>25</em></span>, 3389&#8211;3402.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9254694&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Baird, S.E., Davidson, C.R., and Bohrer, J.C. (2005). The genetics of ray pattern variation in <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. BMC Evol. Biol. <span class="emphasis"><em>5</em></span>, 3.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15634358&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2148-5-3" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bessereau, J.L., Wright, A., Williams, D.C., Schuske, K., Davis, M.W., and Jorgensen, E.M. (2001). Mobilization of a Drosophila
                        transposon in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> germ line. Nature <span class="emphasis"><em>413</em></span>, 70&#8211;74.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11544527&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35092567" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Blumenthal, T., and Gleason, K.S. (2003). <span class="emphasis"><em>Caenorhabditis elegans</em></span> operons: form and function. Nat. Rev. Genet. <span class="emphasis"><em>4</em></span>, 112&#8211;120.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12560808&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg995" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chen, N., Harris, T.W., Antoshechkin, I., Bastiani, C., Bieri, T., Blasiar, D., Bradnam, K., Canaran, P., Chan, J., Chen,
                        C.K.<span class="emphasis"><em>, et al.</em></span> (2005). WormBase: a comprehensive data resource for <span class="emphasis"><em>Caenorhabditis biology</em></span> and genomics. Nucleic Acids Res. <span class="emphasis"><em>33</em></span>, D383&#8211;389.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15618405&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0408307102" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chen, N., Pai, S., Zhao, Z., Mah, A., Newbury, R., Johnsen, R.C., Altun, Z., Moerman, D.G., Baillie, D.L., and Stein, L.D.
                        (2005). Identification of a nematode chemosensory gene family. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>102</em></span>, 146&#8211;151.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15608221&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gki066" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chen, N., and Stein, L.D. (2006). Conservation and functional significance of gene topology in the genome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome Res. <span class="emphasis"><em>16</em></span>, 606&#8211;617.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16606698&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.4515306" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Cho, S., Jin, S.W., Cohen, A., and Ellis, R.E. (2004). A phylogeny of <span class="emphasis"><em>caenorhabditis</em></span> reveals frequent loss of introns during nematode evolution. Genome Res. <span class="emphasis"><em>14</em></span>, 1207&#8211;1220.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15231741&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.2639304" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Coghlan, A., and Wolfe, K.H. (2002). Fourfold faster rate of genome rearrangement in nematodes than in Drosophila. Genome
                        Res. <span class="emphasis"><em>12</em></span>, 857&#8211;867.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12045140&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.172702" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Coghlan, A., and Wolfe, K.H. (2004). Origins of recently gained introns in <span class="emphasis"><em>Caenorhabditis</em></span>. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>101</em></span>, 11362&#8211;11367.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15243155&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0308192101" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Cutter, A.D., F&eacute;lix, M.A., Barri&egrave;re, A., and Charlesworth, D. (2006). Patterns of nucleotide polymorphism distinguish temperate
                        and tropical wild isolates of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Genetics <span class="emphasis"><em>in press</em></span>.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Delattre, M., and Felix, M.A. (2001). Polymorphism and evolution of vulval precursor cell lineages within two nematode genera,
                        <span class="emphasis"><em>Caenorhabditis</em></span> and <span class="emphasis"><em>Oscheius</em></span>. Curr. Biol. <span class="emphasis"><em>11</em></span>, 631&#8211;643.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11369226&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(01)00202-0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Fitch, D.H. A. (1997). Evolution of male tail morphology and development in rhabditid nematodes related to <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Syst. Biol. <span class="emphasis"><em>46</em></span>, 145&#8211;179.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11975351&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.2307/2413641" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Fodor, A., Riddle, D.L., Nelson, F.K., and Golden, J.W. (1983). Comparison of a new wild-type <span class="emphasis"><em>Caenorhabditis briggsae</em></span> with laboratory strains of <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>. Nematologica <span class="emphasis"><em>29</em></span>, 203&#8211;217.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Graustein, A., Gaspar, J.M., Walters, J.R., and Palopoli, M.F. (2002). Levels of DNA polymorphism vary with mating system
                        in the nematode genus <span class="emphasis"><em>caenorhabditis</em></span>. Genetics <span class="emphasis"><em>161</em></span>, 99&#8211;107.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12019226&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Haag, E.S., and Ackerman, A.D. (2005). Intraspecific variation in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fem-3;class=Gene" target="_blank">fem-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene" target="_blank">tra-2</a></em></span>, two rapidly coevolving nematode sex-determining genes. Gene <span class="emphasis"><em>349</em></span>, 35&#8211;42.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15780968&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.gene.2004.12.051" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Haag, E.S., Wang, S., and Kimble, J. (2002). Rapid coevolution of the nematode sex-determining genes <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fem-3;class=Gene" target="_blank">fem-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene" target="_blank">tra-2</a></em></span>. Curr. Biol. <span class="emphasis"><em>12</em></span>, 2035&#8211;2041.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12477393&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)01333-7" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Heschl, M.F., and Baillie, D.L. (1990). Functional elements and domains inferred from sequence comparisons of a heat shock
                        gene in two nematodes. J. Mol. Evol. <span class="emphasis"><em>31</em></span>, 3&#8211;9.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2116528&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF02101786" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hill, R.C., de Carvalho, C.E., Salogiannis, J., Schlager, B., Pilgrim, D., and Haag, E.S. (2006). Genetic flexibility in the
                        convergent evolution of hermaphroditism in <span class="emphasis"><em>Caenorhabditis</em></span> nematodes. Dev. Cell <span class="emphasis"><em>10</em></span>, 531&#8211;538.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16580997&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2006.02.002" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hillier, L.W., Coulson, A., Murray, J.I., Bao, Z., Sulston, J.E., and Waterston, R.H. (2005). Genomics in <span class="emphasis"><em>C. elegans</em></span>: so many genes, such a little worm. Genome Res. <span class="emphasis"><em>15</em></span>, 1651&#8211;1660.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16339362&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.3729105" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hobert, O., Johnston, R.J., Jr., and Chang, S. (2002). Left-right asymmetry in the nervous system: the <span class="emphasis"><em>Caenorhabditis elegans</em></span> model. Nat. Rev. Neurosci. <span class="emphasis"><em>3</em></span>, 629&#8211;640.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12154364&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrn897" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Horvitz, H.R., and Sternberg, P.W. (1991). Multiple intercellular signalling systems control the development of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> vulva. Nature <span class="emphasis"><em>351</em></span>, 535&#8211;541.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1646401&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/351535a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jovelin, R., Ajie, B.C., and Phillips, P.C. (2003). Molecular evolution and quantitative variation for chemosensory behaviour
                        in the nematode genus <span class="emphasis"><em>Caenorhabditis</em></span>. Mol. Ecol. <span class="emphasis"><em>12</em></span>, 1325&#8211;1337.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12694294&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1046/j.1365-294X.2003.01805.x" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kennedy, B.P., Aamodt, E.J., Allen, F.L., Chung, M.A., Heschl, M.F., and McGhee, J.D. (1993). The gut esterase gene (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ges-1;class=Gene" target="_blank">ges-1</a></em></span>) from the nematodes <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. J. Mol. Biol. <span class="emphasis"><em>229</em></span>, 890&#8211;908.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8445654&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/jmbi.1993.1094" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kent, W.J., and Zahler, A.M. (2000). Conservation, regulation, synteny, and introns in a large-scale <span class="emphasis"><em>C. briggsae</em></span>-<span class="emphasis"><em>C. elegans</em></span> genomic alignment. Genome Res. <span class="emphasis"><em>10</em></span>, 1115&#8211;1125.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10958630&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.10.8.1115" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kim, S.K., Lund, J., Kiraly, M., Duke, K., Jiang, M., Stuart, J.M., Eizinger, A., Wylie, B.N., and Davidson, G.S. (2001).
                        A gene expression map for <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>293</em></span>, 2087&#8211;2092.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11557892&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1061603" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kiontke, K., Gavin, N.P., Raynes, Y., Roehrig, C., Piano, F., and Fitch, D.H. (2004). <span class="emphasis"><em>Caenorhabditis</em></span> phylogeny predicts convergence of hermaphroditism and extensive intron loss. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>101</em></span>, 9003&#8211;9008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15184656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0403094101" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kirouac, M., and Sternberg, P.W. (2003). cis-Regulatory control of three cell fate-specific genes in vulval organogenesis
                        of <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>. Dev. Biol. <span class="emphasis"><em>257</em></span>, 85&#8211;103.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12710959&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0012-1606(03)00032-0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lee, Y.H., Huang, X.Y., Hirsh, D., Fox, G.E., and Hecht, R.M. (1992). Conservation of gene organization and trans-splicing
                        in the glyceraldehyde-3-phosphate dehydrogenase-encoding genes of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Gene <span class="emphasis"><em>121</em></span>, 227&#8211;235.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1446820&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0378-1119(92)90126-A" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lercher, M.J., Blumenthal, T., and Hurst, L.D. (2003). Coexpression of neighboring genes in <span class="emphasis"><em>Caenorhabditis elegans</em></span> is mostly due to operons and duplicate genes. Genome Res. <span class="emphasis"><em>13</em></span>, 238&#8211;243.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12566401&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.553803" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Marra, M.A., Kucaba, T.A., Dietrich, N.L., Green, E.D., Brownstein, B., Wilson, R.K., McDonald, K.M., Hillier, L.W., McPherson,
                        J.D., and Waterston, R.H. (1997). High throughput fingerprint analysis of large-insert clones. Genome Res. <span class="emphasis"><em>7</em></span>, 1072&#8211;1084.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9371743&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                     		<span class="bibliomisc">McKay, S.J., Johnsen, R., Khattra, J., Asano, J., Baillie, D.L., Chan, S., Dube, N., Fang, L., Goszczynski, B., Ha, E.<span class="emphasis"><em>, et al.</em></span> (2003). Gene expression profiling of cells, tissues, and developmental stages of the nematode <span class="emphasis"><em>C. elegans</em></span>. Cold Spring Harb. Symp. Quant. Biol. <span class="emphasis"><em>68</em></span>, 159&#8211;169.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15338614&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/sqb.2003.68.159" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mehra, A., Gaudet, J., Heck, L., Kuwabara, P.E., and Spence, A.M. (1999). Negative regulation of male development in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by a protein-protein interaction between TRA-2A and <a href="http://www.wormbase.org/db/get?name=FEM-3;class=Protein" target="_blank">FEM-3</a>. Genes Dev. <span class="emphasis"><em>13</em></span>, 1453&#8211;1463.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10364161&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Molin, L., Mounsey, A., Aslam, S., Bauer, P., Young, J., James, M., Sharma-Oates, A., and Hope, I.A. (2000). Evolutionary
                        conservation of redundancy between a diverged pair of forkhead transcription factor homologues. Development <span class="emphasis"><em>127</em></span>, 4825&#8211;4835.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11044397&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Nayak, S., Goree, J., and Schedl, T. (2005). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fog-2;class=Gene" target="_blank">fog-2</a></em></span> and the evolution of self-fertile hermaphroditism in <span class="emphasis"><em>Caenorhabditis</em></span>. PLoS Biol. <span class="emphasis"><em>3</em></span>, e6.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15630478&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0030006" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                     		<span class="bibliomisc">O'Brien, K.P., Remm, M., and Sonnhammer, E.L. (2005). Inparanoid: a comprehensive database of eukaryotic orthologs. Nucleic
                        Acids Res. <span class="emphasis"><em>33</em></span>, D476&#8211;480.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15608241&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gki107" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ortiz, C.O., Etchberger, J.F., Posy, S.L., Frokjaer-Jensen, C., Lockery, S., Honig, B., and Hobert, O. (2006). Searching for
                        neuronal left/right asymmetry: genomewide analysis of nematode receptor-type guanylyl cyclases. Genetics <span class="emphasis"><em>173</em></span>, 131&#8211;149.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16547101&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.106.055749" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Prasad, S.S., and Baillie, D.L. (1989). Evolutionarily conserved coding sequences in the <span class="emphasis"><em>dpy-20-unc-22</em></span> region of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genomics <span class="emphasis"><em>5</em></span>, 185&#8211;198.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2793177&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0888-7543(89)90045-1" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Robertson, H.M. (1998). Two large families of chemoreceptor genes in the nematodes <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span> reveal extensive gene duplication, diversification, movement, and intron loss. Genome Res. <span class="emphasis"><em>8</em></span>, 449&#8211;463.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9582190&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Robertson, H.M. (2000). The large srh family of chemoreceptor genes in <span class="emphasis"><em>Caenorhabditis</em></span> nematodes reveals processes of genome evolution involving large duplications and deletions and intron gains and losses. Genome
                        Res. <span class="emphasis"><em>10</em></span>, 192&#8211;203.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10673277&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.10.2.192" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Robertson, H.M. (2001). Updating the <span class="emphasis"><em>str</em></span> and <span class="emphasis"><em>srj (stl)</em></span> families of chemoreceptors in <span class="emphasis"><em>Caenorhabditis</em></span> nematodes reveals frequent gene movement within and between chromosomes. Chem. Senses <span class="emphasis"><em>26</em></span>, 151&#8211;159.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11238245&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/chemse/26.2.151" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rudel, D., and Kimble, J. (2001). Conservation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span> regulation and function in nematodes. Genetics <span class="emphasis"><em>157</em></span>, 639&#8211;654.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11156985&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rudel, D., and Kimble, J. (2002). Evolution of discrete Notch-like receptors from a distant gene duplication in <span class="emphasis"><em>Caenorhabditis</em></span>. Evol. Dev. <span class="emphasis"><em>4</em></span>, 319&#8211;333.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12356262&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1046/j.1525-142X.2002.02027.x" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Stein, L.D., Bao, Z., Blasiar, D., Blumenthal, T., Brent, M.R., Chen, N., Chinwalla, A., Clarke, L., Clee, C., Coghlan, A.<span class="emphasis"><em>, et al.</em></span> (2003). The genome sequence of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>: a platform for comparative genomics. PLoS Biol. <span class="emphasis"><em>1</em></span>, E45.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14624247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Stothard, P., and Pilgrim, D. (2003). Sex determination gene and pathway evolution in nematodes. Bioessays <span class="emphasis"><em>25</em></span>, 221&#8211;231.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12596226&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/bies.10239" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Thomas, J.H., Kelley, J.L., Robertson, H.M., Ly, K., and Swanson, W.J. (2005). Adaptive evolution in the SRZ chemoreceptor
                        families of <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>102</em></span>, 4476&#8211;4481.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15761060&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0406469102" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Timmons, L., Court, D.L., and Fire, A. (2001). Ingestion of bacterially expressed dsRNAs can produce specific and potent genetic
                        interference in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Gene <span class="emphasis"><em>263</em></span>, 103&#8211;112.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11223248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0378-1119(00)00579-5" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wang, X., and Chamberlin, H.M. (2002). Multiple regulatory changes contribute to the evolution of the <span class="emphasis"><em>Caenorhabditis <a href="http://www.wormbase.org/db/get?name=lin-48;class=Gene" target="_blank">lin-48</a></em></span> ovo gene. Genes Dev. <span class="emphasis"><em>16</em></span>, 2345&#8211;2349.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12231624&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.996302" target="_blank">Article</a>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e10" href="#d0e10">*</a></sup>Edited by Jonathan Hodgkin. Last revised January 10, 2007. Published May 3, 2007. This chapter should be cited as: Gupta,
                  B.P., et al. Genomics and biology of the nematode <span class="emphasis"><em>Caenorhabditis briggsae</em></span> (May 3, 2007), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.136.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2007 Bhagwati P. Gupta, et al. This is an open-access article distributed under the terms of the Creative Commons Attribution
                  License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source
                  are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e37" href="#d0e37">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:guptab@mcmaster.ca" target="_top">guptab@mcmaster.ca</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>
