<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic mapping and manipulation: Chapter 6-Mapping with deficiencies and duplications
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMgmmddch6_ArticleID" title="Genetic mapping and manipulation: Chapter 6-Mapping with deficiencies and duplications&#xA;">
      <link rel="next" href="#d0e46" title="1.&nbsp;Basics">
      <script language="javascript" type="text/javascript" src="/js/history_list.js"></script>
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="mappingdeficienduplicat.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMgmmddch6_ArticleID"></a>Genetic mapping and manipulation: Chapter 6-Mapping with deficiencies and duplications<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                  </h1>
		  <h3>This chapter has been revised.  See Publication History in side bar.</h3>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">David</span> <span class="surname">Fay<em><span class="remark">
                                 					<sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Department of Molecular Biology, University of Wyoming, Laramie, Wyoming 82071-3944 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward genetics and genetic mapping<br />>>> Genetic mapping and manipulation
		</a></div><hr />

          <p><a style="color:blue;cursor:pointer" onclick="return kadabra('history');" title="click to expand ..." >Publication History</a></p>
           <ul id="history" style="display:none">
            <li><a href="/chapters/www_mappingdeficienduplicat/mappingdeficienduplicat.html">version 1</a></li>
            <li><a href="/chapters/www_mappingdeficienduplicat.2/mappingdeficienduplicat.html">latest version</a></li>
           </ul>

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e46">1. Basics</a></span></dt>
               <dt><span class="sect1"><a href="#d0e139">2. Setting up crosses with deficiencies</a></span></dt>
               <dt><span class="sect1"><a href="#d0e267">3. Mapping with duplications</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e46"></a>1.&nbsp;Basics
                     </h2>
                  </div>
               </div>
            </div>
            <p>Deficiency (<span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span>) mapping works great, when it works! Deficiencies refer to specific deleted regions within chromosomes. The sizes of <span class="emphasis"><em><span class="bold"><strong>Dfs</strong></span></em></span> vary greatly from just a few cosmids wide to the absence of a large portion of the chromosome. The endpoints of the <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> may have been determined precisely using molecular techniques or may be rough guesses based on genetic tests with various
               mutations. Homozygous <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> animals are almost always embryonic lethals, as removal of multiple genes usually includes some that are necessary during
               early development. One of the traditional strengths of <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> mapping is that it can provide clear-cut irrefutable endpoints for ones region of interest (as opposed to the statistical
               arguments often associated with three-point mapping). The problem in mapping with <span class="emphasis"><em><span class="bold"><strong>Dfs</strong></span></em></span> is that although positive results are generally unambiguous, negative results can be more difficult to nail down. In addition,
               the availability of SNPs to define true endpoints has to some extent removed much of the allure of working with deficiencies.
               Nevertheless, <span class="emphasis"><em><span class="bold"><strong>Dfs</strong></span></em></span> are still extremely useful for certain types of genetic analyses, such as determining whether or not mutations are null (also
               see <a href="http://www.wormbook.org/chapters/www_dominantmutations/dominantmutations.html" target="_top">Dominant mutations</a>). The basics of <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> mapping strategy are shown in <a href="#figure1">Figure 1</a>.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
                  <tr>
                     <td align="center"><img src="gmmddch6-1.jpg" align="middle" width="360" alt=" Mapping with deficiencies and duplications Figure 6-1"></td>
                  </tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1.&nbsp;</b></p>
               </div>
            </div>
            <p>In the case on the left, the deficiency (indicated by the dashed line) deletes the gene of interest and as a result fails
               to complement (does not rescue) the mutant allele on the opposite chromosome. Animals with such a genetic configuration will
               generally show the original mutant (<span class="bold"><strong>M</strong></span>) phenotype. An exception to this can occur when the mutation is a hypomorph (partial loss of function). In this case, <span class="emphasis"><em><span class="bold"><strong>m/Df</strong></span></em></span> animals may show a new "<span class="bold"><strong>M</strong></span>" phenotype that is more severe than the phenotype of <span class="emphasis"><em><span class="bold"><strong>m/m</strong></span></em></span> animals. In the case on the right, the mutation is outside the <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> and the <span class="bold"><strong>M</strong></span> phenotype will not be displayed. If the breakpoint of the <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> is near <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>, one may have generated a well-balanced heterozygote, as recombination is often reduced in the immediate vicinity of the
               <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span>. Nevertheless, it is important to keep in mind that recombination might occur between the right breakpoint and <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>, thereby destroying the balanced stock.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e139"></a>2.&nbsp;Setting up crosses with deficiencies
                     </h2>
                  </div>
               </div>
            </div>
            <p>The way most deficiency mapping is done is as follows: You set up a situation where you are looking for a mutant phenotype
               in the F1 generation of the cross. For example, as shown in <a href="#figure2">Figure 2</a>, your mutation, which is linked to an <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> or some other visible marker in cis, is crossed to male animals (step 2) that were created by mating N2 males into a balanced
               deficiency strain (step 1; provided <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> is not on X). In this case, the <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutation linked to your mutation is known to be outside this particular <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span>. In this scenario, if the mutation is within the <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span>, you will observe <span class="bold"><strong>non-Unc</strong></span> animals displaying the <span class="bold"><strong>M</strong></span> phenotype in the F1 generation (step 3). The presence of the linked <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> marker is necessary here to clearly identify cross-progeny. For <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> mapping, it is important to set up as many mating plates as possible (10-15 is not an excessive number) to guarantee generation
               and detection of the <span class="emphasis"><em><span class="bold"><strong>m/Df</strong></span></em></span> genotype. Failure to observe the <span class="bold"><strong>M</strong></span> phenotype would indicate that <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> is outside the <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span>, provided that on most of your mating plates you observe good numbers of <span class="bold"><strong>non-Unc</strong></span> cross-progeny. However, to ensure a correct interpretation of negative results, it is necessary to test whether animals of
               the genotype <span class="emphasis"><em><span class="bold"><strong>m/Df</strong></span></em></span> have a wild-type phenotype. Such an animal can be identified by the segregation of <span class="bold"><strong>M</strong></span> and <span class="bold"><strong>Df</strong></span> (usually embryonic lethal) phenotypes in the F1.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
                  <tr>
                     <td align="center"><img src="gmmddch6-2.jpg" align="middle" width="360" alt=" Mapping with deficiencies and duplications Figure 6-2"></td>
                  </tr>
               </table>
               <div class="caption">
                  <p><b>Figure 2.&nbsp;</b></p>
               </div>
            </div>
            <p>As an alternative approach, shown in <a href="#figure3">Figure 3</a>, you can also mate your mutation via the male into the <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> strain (step 2) and look for your <span class="bold"><strong>M</strong></span> phenotype in the first generation of cross-progeny (step 3). In this case, it is best to have the linked <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutation (or other cis marker) inside the <span class="bold"><strong>Df</strong></span> so that cross-progeny of the desired phenotype can easily be identified via the <span class="bold"><strong>Unc</strong></span> phenotype. This approach can also be used without any linked marker where one just looks directly for the appearance of the
               M phenotype in the F1 generation. As with the previous example, if the result appears negative (the <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span> is outside the <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span>), it is important to try and verify this by cloning supposed F1 cross-progeny and making sure that some of the phenotypically
               wild-type F1s throw progeny of both <span class="bold"><strong>M</strong></span> and <span class="bold"><strong>Df</strong></span> phenotypes.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="306">
                  <tr>
                     <td align="center"><img src="gmmddch6-3.jpg" align="middle" width="306" alt=" Mapping with deficiencies and duplications Figure 6-3"></td>
                  </tr>
               </table>
               <div class="caption">
                  <p><b>Figure 3.&nbsp;</b></p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e267"></a>3.&nbsp;Mapping with duplications
                     </h2>
                  </div>
               </div>
            </div>
            <p>Duplications (the <span class="emphasis"><em><span class="bold"><strong>Dps</strong></span></em></span>) are used less frequently than <span class="emphasis"><em><span class="bold"><strong>Dfs</strong></span></em></span> and are probably of less utility. Free duplications are autonomous pieces of DNA derived from normal chromosomes. They are
               usually relatively small as compared with full-length chromosomes and exhibit segregation properties that are independent
               of other chromosomes, including the chromosome from which they were derived. In many ways they most resemble extrachromosomal
               arrays and, like arrays, tend to be significantly less stable (especially meiotically) than normal chromosomes. <span class="emphasis"><em><span class="bold"><strong>Dps</strong></span></em></span> will vary significantly in their genetic stability, and some published information exists describing the properties of various
               <span class="emphasis"><em><span class="bold"><strong>Dps</strong></span></em></span>. Animals that contain a <span class="emphasis"><em><span class="bold"><strong>Dp</strong></span></em></span> will effectively be triploid for the genes that lie within the <span class="emphasis"><em><span class="bold"><strong>Dp</strong></span></em></span>. <span class="emphasis"><em><span class="bold"><strong>Dps</strong></span></em></span> are often used to balance a homozygous lethal mutation.
            </p>
            <p>The idea in mapping with free <span class="emphasis"><em><span class="bold"><strong>Dps</strong></span></em></span> is to determine whether or not the mutation of interest lies within the duplicated region. If it does, then one would observe
               "rescue" of the mutant phenotype. To do this, one will set up crosses that ultimately lead to the isolation of the genotype
               shown in <a href="#figure4">Figure 4</a>. This can in theory be accomplished using several approaches but usually takes a number of steps as one must re-homozygose
               animals for <span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>. Having a marker linked to (<span class="emphasis"><em><span class="bold"><strong>m</strong></span></em></span>) but outside the region covered by the <span class="emphasis"><em><span class="bold"><strong>Dp</strong></span></em></span> can be useful for identifying candidate (<span class="emphasis"><em><span class="bold"><strong>m/m</strong></span></em></span>) animals. As with <span class="emphasis"><em><span class="bold"><strong>Dfs</strong></span></em></span>, it will generally be quite obvious when one's mutation lies within the <span class="emphasis"><em><span class="bold"><strong>Dp</strong></span></em></span> but rather more difficult to prove that it definitely lies outside. Creating a chromosome where the mutation is flanked by
               two visible markers (one outside and one inside the <span class="emphasis"><em><span class="bold"><strong>Dp</strong></span></em></span>) can help to clarify this issue.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="162">
                  <tr>
                     <td align="center"><img src="gmmddch6-4.jpg" align="middle" width="162" alt=" Mapping with deficiencies and duplications Figure 6-4"></td>
                  </tr>
               </table>
               <div class="caption">
                  <p><b>Figure 4.&nbsp;</b></p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised May 5, 2005. Published February 17, 2006. This chapter should be cited as: Fay, D. Genetic
                  mapping and manipulation: Chapter 6-Mapping with deficiencies and duplications (February 17, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.95.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 David Fay. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:davidfay@uwyo.edu" target="_top">davidfay@uwyo.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>
