<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Natural variation and population genetics of Caenorhabditis elegans
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#genet_ArticleID" title="Natural variation and population genetics of Caenorhabditis elegans&#xA;">
      <link rel="next" href="#sec1" title="1.&nbsp;Molecular polymorphisms and population genetics">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="naturalvariationgenetics.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="genet_ArticleID"></a>Natural variation and population genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span><sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Antoine Barri&egrave;re and Marie-Anne F&eacute;lix<em><span class="remark">
                                 			<sup><a name="d0e34" href="#ftn.d0e34">&sect;</a></sup>
                                 			</span></em>
                           </span><br></span><span class="affiliation"><span class="address">Institut Jacques Monod, CNRS - Universities of Paris, 75251 Paris cedex 05, France
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_nematodeevolecol.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Evolution and ecology
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=282.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Natural+variation+and+population+genetics+of+C.+elegans" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Molecular polymorphisms and population genetics</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec1_1">1.1. Low molecular diversity of <span class="emphasis"><em>C. elegans</em></span></a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_2">1.2. Population structure and geographic differentiation</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_3">1.3. Comparison with <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span></a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_4">1.4. Outcrossing versus selfing in <span class="emphasis"><em>C. elegans</em></span></a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec2">2. Phenotypic diversity</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec2_1">2.1. Phenotypic change upon mutation accumulation</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_2">2.2. Polymorphic phenotypes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_3">2.3. Genetic analysis of a natural phenotypic variation</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec3">3. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>presents a low level of molecular diversity, which may be explained by its selfing mode of reproduction. Recent work on the
                     genetic structure of natural populations of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>indeed suggests a low level of outcrossing, and little geographic differentiation because of migration. The level and pattern
                     of molecular diversity among wild isolates of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>are compared with those found after accumulation of spontaneous mutations in the laboratory. The last part of the chapter
                     reviews phenotypic differences among wild isolates of</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans.</strong></span></em></span></p>
         </div>
         <p>Most studies on <span class="emphasis"><em>C. elegans</em></span> use a single genetic background, that of the N2 strain. Its biological features as unravelled in the laboratory may be better
            interpreted with more knowledge of their evolutionary origin and ecological significance. Moreover, the diversity of wild
            isolates is a source of phenotypic variations that can be studied like mutant phenotypes.
         </p>
         <p>In addition, the mode of reproduction of <span class="emphasis"><em>C. elegans</em></span> makes it a great system to study the regulation, consequences and evolution of outcrossing versus selfing breeding systems.
            Obvious questions concern the frequency of outcrossing, the geographic structure of populations (are populations locally clonal?)
            and the fate of deleterious mutations. Although evolutionary and ecological studies of <span class="emphasis"><em>C. elegans</em></span> have been scarce until a few years ago, its short generation time, the ease of obtaining isogenic lines by selfing and the
            ability to retain frozen stocks make it a model system with a future in evolutionary biology.
         </p>
         <p>We will successively review what is currently known about I. its molecular diversity and the genetic structure of populations,
            and II. its phenotypic diversity.
         </p>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Molecular polymorphisms and population genetics
                     </h2>
                  </div>
               </div>
            </div>
            <p>Early work on molecular polymorphisms in <span class="emphasis"><em>C. elegans</em></span> was based on a set of natural isolates that were collected worldwide over many years, each kept as a single selfing isogenic
               strain (available at CGC). These isolates come from North America (mostly California), Western Europe (including N2, the reference
               strain, isolated in Bristol, U.K.) and Adelaide (Australia); one isolate, CB4856, was sampled in Hawaii and is used for SNP
               mapping (see <a href="http://www.wormbook.org/chapters/www_SNPsintrotwopointmap/SNPsintrotwopointmap.html" target="_top">SNPs: introduction and 2-point mapping</a>; <a href="#bib35">Hodgkin and Doniach, 1997</a>; <a href="#figure1">Figure 1</a>). These isolates are used as representatives of the molecular and phenotypic variation within the species, even though they
               may miss a signifcant part of it, due to geographically restricted sampling. <span class="emphasis"><em>C. elegans</em></span> could never be isolated in samples from Asia (unlike <span class="emphasis"><em>C. briggsae</em></span>), but not much sampling was attempted in Africa nor South America.
            </p>
            <p>&nbsp;</p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="585">
<tr>
  <td align="center"><p><a href="genetfig1.jpg"><img src="genetfig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1. Geographic origin of available <span class="emphasis"><em>C. elegans</em></span> wild isolates.&nbsp;</b>Red dots: origin of available wild isolates, either through the <a href="http://biosci.umn.edu/CGC/Strains/strains.htm" target="_top">CGC</a> or at <a href="http://www2.ijm.jussieu.fr/worms/index.php" target="_top">http://www2.ijm.jussieu.fr/worms/index.php</a>. Green dots: sampling sites where <span class="emphasis"><em>C. elegans</em></span> was also found, but not conserved, in Algeria (original isolates by <a href="#bib47">Maupas, 1900</a>) and Qu&eacute;bec (<a href="#bib1">Abdul Kader and C&ocirc;t&eacute;, 1996</a>).
                  </p>
               </div>
            </div>
            <p>Recently, several sets of wild isolates were collected from Western Europe and California (plus one strain from Madeira; <a href="#bib31">Haber et al., 2005</a>; <a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>; <a href="#bib57">Sivasundar and Hey, 2005</a>). Several strains were established from a given sample, giving an insight of diversity <span class="emphasis"><em>within</em></span> a local population.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_1"></a>1.1.&nbsp;Low molecular diversity of <span class="emphasis"><em>C. elegans</em></span></h3>
                     </div>
                  </div>
               </div>
               <p>The level and pattern of genetic diversity depend on the rate and pattern of mutation, which can be measured after accumulation
                  of spontaneous mutations in the laboratory. Mutation accumulation (MA) lines are established by picking a single individual
                  at each generation, therefore limiting natural selection as much as possible (<a href="#figure2">Figure 2</a>). They represent the raw mutational process, whereas the diversity observed among wild isolates is the result of mutation
                  as well as population structure and phenotypic selection. We first review for mitochondrial and nuclear DNA the observed molecular
                  diversity in wild isolates, then the mutation rate and pattern in MA lines, and compare them to reveal the action of selection
                  in natural populations.
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="522">
<tr>
  <td align="center"><p><a href="genetfig2.jpg"><img src="genetfig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Mutation accumulation lines.&nbsp;</b>A. Mutation accumulation lines are obtained by transferring a single larva at each generation for many generations. Spontaneous
                        mutations accumulate, with minimal selection. Many lines are kept in parallel, and can be frozen at intervals to keep a record
                        of their evolution. Due to the accumulation of highly deleterious mutations, some lines are lost: from 100 original lines,
                        only 58 were kept up to 396 generations (<a href="#bib65">Vassilieva et al., 2000</a>). B. Frequency distribution between the different mutation accumulation lines of a fitness measure (the intrinsic rate of
                        increase) after 214 generations (<a href="#bib65">Vassilieva et al., 2000</a>). C. Frequency distribution of body size between the same lines at an average of 152 generations (<a href="#bib3">Azevedo et al., 2002</a>).
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_1_1"></a>1.1.1.&nbsp;Polymorphism in the mitochondrial genome
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><a href="#bib21">Denver et al. (2003)</a> sequenced the nearly complete mitochondrial genome of 27 <span class="emphasis"><em>C. elegans</em></span> natural isolates. Mitochondrial sequences were found to cluster in two well-defined clades, called I and II (<a href="#figure3">Figure 3</a>). N2 and CB4856 both belong to clade I. Several isolates have the same mitochondrial DNA sequence as N2.
                  </p>
                  <p>The mutation rate and pattern of mitochondrial DNA were assessed in mutation accumulation lines (<a href="#bib20">Denver et al., 2000)</a>. 26 mutations were recorded, 16 of which were base substitutions, yielding a rate of 9.7 &times; 10<sup>-8</sup> mutation per site per generation (<a href="#table1" title="Table 1. Mutation rates in mutation accumulation lines (MA) and mutational pattern in MA lines versus wild isolates">Table 1</a>).
                  </p>
                  <div class="table"><a name="table1"></a><p class="title">Table 1. Mutation rates in mutation accumulation lines (MA) and mutational pattern in MA lines versus wild isolates</p>
                     <table summary="Table 1. Mutation rates in mutation accumulation lines (MA) and mutational pattern in MA lines versus wild isolates" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="center">
                           <col align="left">
                           <col align="left">
                           <col align="left">
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <thead>
                           <tr>
                              <th align="left">DNA</th>
                              <th align="center">Strains</th>
                              <th align="left">Mutation rate<sup>*</sup></th>
                              <th align="left">Subst/Indel (total)</th>
                              <th align="left">Ts/Tv</th>
                              <th align="left">Repl : Syn</th>
                              <th align="left">Reference</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td rowspan="2" align="left" valign="top"><span class="bold"><strong>Mitochondrial</strong></span></td>
                              <td align="center"><span class="bold"><strong>MA</strong></span></td>
                              <td align="left" valign="middle">1.6 &times; 10<sup>-7</sup></td>
                              <td align="left">1.6 (26)</td>
                              <td align="left">4.3</td>
                              <td align="left">9 : 6</td>
                              <td align="left"><a href="#bib20">Denver et al., 2000</a></td>
                           </tr>
                           <tr>
                              <td align="center"><span class="bold"><strong>27 wild isolates</strong></span></td>
                              <td align="left">&nbsp;</td>
                              <td align="left">237 (238)</td>
                              <td align="left">2.6</td>
                              <td align="left">33 : 172</td>
                              <td align="left"><a href="#bib21">Denver et al., 2003</a></td>
                           </tr>
                           <tr>
                              <td rowspan="3" align="left" valign="top"><span class="bold"><strong>Nuclear</strong></span></td>
                              <td align="center"><span class="bold"><strong>MA</strong></span></td>
                              <td align="left" valign="middle">2.1 &times; 10<sup>-8</sup></td>
                              <td align="left">0.76 (30)</td>
                              <td align="left">1.6</td>
                              <td align="left">&nbsp;</td>
                              <td align="left"><a href="#bib19">Denver et al., 2004</a></td>
                           </tr>
                           <tr>
                              <td align="center"><span class="bold"><strong>CB4856 vs. N2</strong></span></td>
                              <td align="left">&nbsp;</td>
                              <td align="left">2.6</td>
                              <td align="left">1.3</td>
                              <td align="left">&nbsp;</td>
                              <td align="left"><a href="#bib62">Swan et al., 2002</a> &amp; <a href="#bib69">Wicks et al., 2001</a></td>
                           </tr>
                           <tr>
                              <td align="center"><span class="bold"><strong>18 wild isolates</strong></span></td>
                              <td align="left">&nbsp;</td>
                              <td align="left">3.9 (54)</td>
                              <td align="left">1.7</td>
                              <td align="left">&nbsp;</td>
                              <td align="left"><a href="#bib21">Denver et al., 2003</a></td>
                           </tr>
                           <tr>
                              <td align="left"><span class="bold"><strong>Homopolymers</strong></span></td>
                              <td align="center"><span class="bold"><strong>MA</strong></span></td>
                              <td align="left" valign="middle">4.5 &times; 10<sup>-6</sup> to 1.5 &times; 10<sup>-4</sup></td>
                              <td align="left">&#8211;</td>
                              <td align="left">&#8211;</td>
                              <td align="left">&nbsp;</td>
                              <td align="left"><a href="#bib18">Denver et al., 2004</a></td>
                           </tr>
                           <tr>
                              <td align="left"><span class="bold"><strong>Microsatellites</strong></span></td>
                              <td align="center">MA</td>
                              <td align="left" valign="middle">8.9 &times; 10<sup>-5</sup> to 1.8 &times; 10<sup>-2</sup></td>
                              <td align="left">&#8211;</td>
                              <td align="left">&#8211;</td>
                              <td align="left">&nbsp;</td>
                              <td align="left"><a href="#bib28">Frisse, 1999</a></td>
                           </tr>
                           <tr>
                              <td colspan="7" align="left">&#8220;Subst/Indel&#8221;: ratio of nucleotide substitutions over insertion-deletion events. &#8220;Ts/Tv&#8221;: ratio of transitions over transversions
                                 among substitutions. &#8220;Repl : Syn&#8221;: number of amino-acid replacements and synonymous nucleotide substitutions.
                              </td>
                           </tr>
                           <tr>
                              <td colspan="7" align="left">*: mutation rate is per base pair per generation for mitochondrial and nuclear sequences, and per locus per generation for
                                 homopolymers and microsatellites.
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="genetfig3.jpg"><img src="genetfig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 3. Phylogenetic analysis of <span class="emphasis"><em>C. elegans</em></span> hermaphrodite lineages using nuclear and mitochondrial DNA sequences.&nbsp;</b>Phylogenetic analysis of <span class="emphasis"><em>C. elegans</em></span> hermaphrodite lineages using mtDNA and nuclear sequences. Bootstrap values (maximum parsimony and maximum likelihood) are
                           indicated at each node. AB1 is the only strain whose placement differs in the two trees, suggesting that it is the product
                           of recombination between two divergent genotypes. A. Bootstrap consensus tree for <span class="emphasis"><em>C. elegans</em></span> natural isolates, using 70 nuclear variable sites from <a href="#bib21">Denver et al. (2003)</a> and <a href="#bib43">Koch et al. (2000)</a>. Note that in the latter set, most SNPs were found by sequencing CB4857. B. Unrooted phylogram of the <span class="emphasis"><em>C. elegans</em></span> natural isolates using 11,443 bp of mitochondrial DNA sequences. From <a href="#bib21">Denver et al. (2003)</a>.</p>
                     </div>
                  </div>
                  <p>The comparison of the mutation pattern in MA lines versus wild isolates suggests that the excess of transitions over transversions
                     in wild isolates is due to a mutational bias. In contrast, indels and replacement mutations in coding regions are less frequent
                     among wild isolates than in MA lines, suggesting that they are counterselected (<a href="#table1" title="Table 1. Mutation rates in mutation accumulation lines (MA) and mutational pattern in MA lines versus wild isolates">Table 1</a>; <a href="#bib20">Denver et al., 2000</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_1_2"></a>1.1.2.&nbsp;Polymorphism in the nuclear genome revealed by shotgun sequencing or AFLP
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The nuclear polymorphism level was first measured by sequencing random regions of the genome of four wild isolates: AB1, CB4857,
                     RC301 and TR403 (<a href="#bib43">Koch et al., 2000</a>). Extensive shotgun data (12 Mb) were then obtained for CB4856, which harbours variant polymorphisms distributed over the
                     whole genome (compared to N2), allowing its use for genetic mapping of mutations (see <a href="http://www.wormbook.org/chapters/www_SNPsintrotwopointmap/SNPsintrotwopointmap.html" target="_top">SNPs: introduction and 2-point mapping</a>; <a href="#bib62">Swan et al., 2002</a>; <a href="#bib69">Wicks et al., 2001</a>). A random 44 kb set was also sequenced in 18 strains (those in <a href="#figure3">Figure 3</a>; <a href="#bib21">Denver et al., 2003</a>). CB4856 and N2 differ by one polymorphism per 840 bp or 1.2 &times; 10<sup>-3</sup> per nucleotide (<a href="#bib62">Swan et al., 2002</a>; <a href="#bib69">Wicks et al., 2001</a>). The other sequenced strains are less divergent (<a href="#table2" title="Table 2. Pairwise molecular distances between N2 and other wild isolates">Table 2</a>). Overall, the level of polymorphism is low in <span class="emphasis"><em>C. elegans</em></span>.
                  </p>
                  <div class="table"><a name="table2"></a><p class="title">Table 2. Pairwise molecular distances between N2 and other wild isolates</p>
                     <table summary="Table 2. Pairwise molecular distances between N2 and other wild isolates" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <thead>
                           <tr>
                              <th align="left">Strain</th>
                              <th align="left">Distance to N2 (&times; 10<sup>3</sup> bp<sup>-1</sup>)
                              </th>
                              <th align="left">Reference</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td align="left">CB4856</td>
                              <td align="left">2.7</td>
                              <td align="left"><a href="#bib21">Denver et al., 2003</a></td>
                           </tr>
                           <tr>
                              <td align="left">CB4856</td>
                              <td align="left">1.2</td>
                              <td align="left"><a href="#bib62">Swan et al., 2002</a>; <a href="#bib69">Wicks et al., 2001</a></td>
                           </tr>
                           <tr>
                              <td align="left">CB4857</td>
                              <td align="left">0.7</td>
                              <td align="left"><a href="#bib43">Koch et al., 2000</a></td>
                           </tr>
                           <tr>
                              <td align="left">AB1</td>
                              <td align="left">0.4</td>
                              <td align="left"><a href="#bib43">Koch et al., 2000</a></td>
                           </tr>
                           <tr>
                              <td align="left">RC301</td>
                              <td align="left">0.13</td>
                              <td align="left"><a href="#bib43">Koch et al., 2000</a></td>
                           </tr>
                           <tr>
                              <td align="left">TR403</td>
                              <td align="left">0.11</td>
                              <td align="left"><a href="#bib43">Koch et al., 2000</a></td>
                           </tr>
                           <tr>
                              <td align="left">TR403</td>
                              <td align="left">0.07</td>
                              <td align="left"><a href="#bib21">Denver et al., 2003</a></td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <p>Whole-genome diversity was also investigated using AFLP (Amplified Fragment Length Polymorphism), a whole-genome fingerprinting
                     technique, on four populations from France (<a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>). For sampling, individuals (12&#8211;19/location) were picked as they crawled out of the substrate, avoiding as much as possible
                     post-sampling selection and ensuring a true representation of natural population diversity. Across the four French populations,
                     the AFLP fragment diversity is Hj = 4.9 (SE 0.9). An approximate conversion to nucleotide diversity gives a <span class="symbol">&#960;</span> = 0.81 &times; 10<sup>-3</sup> per nucleotide (<a href="#table3" title="Table 3. Nucleotide diversity">Table 3</a>).
                  </p>
                  <div class="table"><a name="table3"></a><p class="title">Table 3. Nucleotide diversity</p>
                     <table summary="Table 3. Nucleotide diversity" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                           <col align="left">
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <thead>
                           <tr>
                              <th align="left">Locus</th>
                              <th align="left">Wild isolates</th>
                              <th align="left">Nucleotide diversity (&times; 10<sup>3</sup>)
                              </th>
                              <th align="left">Replacement : synonymous</th>
                              <th align="left">Reference</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td rowspan="2" align="left"><span class="emphasis"><em>COII</em></span> (696 bp; mitochondrial)
                              </td>
                              <td align="left">11 (worldwide)</td>
                              <td align="left">4.6</td>
                              <td align="left">&nbsp;</td>
                              <td align="left"><a href="#bib63">Thomas and Wilson, 1991</a></td>
                           </tr>
                           <tr>
                              <td align="left">20 (worldwide)</td>
                              <td align="left">10.2</td>
                              <td align="left">1 : 14</td>
                              <td align="left"><a href="#bib29">Graustein et al., 2002</a></td>
                           </tr>
                           <tr>
                              <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cal-1" target="_blank">cal-1</a></em></span> (838 bp)
                              </td>
                              <td align="left">11 (worldwide)</td>
                              <td align="left">0.0</td>
                              <td align="left">0 : 0</td>
                              <td align="left"><a href="#bib63">Thomas and Wilson, 1991</a></td>
                           </tr>
                           <tr>
                              <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=odr-3" target="_blank">odr-3</a></em></span> (2652 bp)
                              </td>
                              <td align="left">10 (worldwide)</td>
                              <td align="left">0.08</td>
                              <td align="left">0 : 2</td>
                              <td align="left"><a href="#bib40">Jovelin et al., 2003</a></td>
                           </tr>
                           <tr>
                              <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> (547 bp)
                              </td>
                              <td align="left">20 (worldwide)</td>
                              <td align="left">0.9</td>
                              <td align="left">0 : 1</td>
                              <td align="left"><a href="#bib29">Graustein et al., 2002</a></td>
                           </tr>
                           <tr>
                              <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=spe-9" target="_blank">spe-9</a></em></span> (1869 bp)
                              </td>
                              <td align="left">20 (worldwide)</td>
                              <td align="left">0.3</td>
                              <td align="left">2 : 4</td>
                              <td align="left"><a href="#bib29">Graustein et al., 2002</a></td>
                           </tr>
                           <tr>
                              <td align="left"><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-2" target="_blank">tra-2</a></em></span> (670 bp)
                              </td>
                              <td align="left">20 (worldwide)</td>
                              <td align="left">0.0</td>
                              <td align="left">0 : 0</td>
                              <td align="left"><a href="#bib29">Graustein et al., 2002</a></td>
                           </tr>
                           <tr>
                              <td align="left">Whole genome AFLP</td>
                              <td align="left">several populations (France)</td>
                              <td align="left">0.81</td>
                              <td align="left">&#8211;</td>
                              <td align="left"><a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a></td>
                           </tr>
                           <tr>
                              <td align="left">Whole genome AFLP</td>
                              <td align="left">within populations</td>
                              <td align="left">0.02 to 0.43</td>
                              <td align="left">&#8211;</td>
                              <td align="left"><a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a></td>
                           </tr>
                           <tr>
                              <td colspan="5" align="left">Molecular diversity can be expressed as a pairwise distance between two genomes (<a href="#table2" title="Table 2. Pairwise molecular distances between N2 and other wild isolates">Table 2</a>). When more than two strains are compared, the standard measure of genetic diversity (Nei's genetic diversity) is the probability
                                 of two randomly sampled alleles to be different for the type of polymorphism considered (for example, nucleotide diversity,
                                 noted <span class="symbol">&#960;</span>). This measure takes into account the frequency of each allele in the population of samples. The conversion of AFLP fragment
                                 diversity to nucleotide diversity is approximate, but does not affect the ratio of local vs. global diversity estimates.
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <p>The nuclear mutation rate was estimated in MA lines to 2.1 &times; 10<sup>-8</sup> per nucleotide site (or 2.1 mutations per haploid genome) per generation (<a href="#bib19">Denver et al., 2004</a>). 17 out of 30 observed mutations were indels, among which insertions were dominant (13 out of 17). This contrasts with the
                     pattern observed among wild isolates in putatively neutral regions such as transposon pseudogenes, where deletions are 2.8
                     times more frequent than insertions (<a href="#bib70">Witherspoon and Robertson, 2003</a>); therefore, data obtained on putative pseudogenes should be taken carefully, as selection may not be fully suppressed. It
                     is also possible that the MA data, obtained in the N2 genetic background, are not representative of the mutational pattern
                     that gave rise to the present diversity in the species.
                  </p>
                  <p>The comparison of MA lines versus wild isolates shows that indels are counterselected (<a href="#table1" title="Table 1. Mutation rates in mutation accumulation lines (MA) and mutational pattern in MA lines versus wild isolates">Table 1</a>). Also, polymorphisms in exons are two-fold underrepresented in wild isolates (<a href="#bib43">Koch et al., 2000</a>; <a href="#bib21">Denver et al., 2003</a>). Polymorphisms in coding regions show a clear bias towards third base position, more prone to silent mutations, reflecting
                     the selection on coding sequences.
                  </p>
                  <p>At the chromosomal level, polymorphisms are more frequent in distal arm regions than in the central clusters (<a href="#bib43">Koch et al., 2000</a>; <a href="#bib69">Wicks et al., 2001</a>; <a href="#bib62">Swan et al., 2002</a>; see <a href="http://www.wormbook.org/chapters/www_karyotype/karyotype.html" target="_top">Karyotype, ploidy and gene dosage</a>). At a finer scale, different genes show a wide diversity of nucleotide polymorphism levels (<a href="#bib21">Denver et al., 2003</a>; <a href="#bib60">Stewart et al., 2004</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_1_3"></a>1.1.3.&nbsp;Polymorphism in specific genes
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A few genes have been sequenced in a large set of wild isolates (<a href="#bib63">Thomas and Wilson, 1991</a>; <a href="#bib29">Graustein et al., 2002</a>; <a href="#bib40">Jovelin et al., 2003</a>). The overall level of polymorphism found is very low. In some cases, no polymorphism was found at all over hundreds of base
                     pairs (<a href="#table3" title="Table 3. Nucleotide diversity">Table 3</a>).
                  </p>
                  <p><a href="#bib60">Stewart et al. (2004)</a> sequenced in 20 wild isolates 31 chemoreceptor genes from the <span class="emphasis"><em>srh</em></span> and <span class="emphasis"><em>str</em></span> families that are pseudogenes in the N2 strain. For 10 of these genes, functional alleles are present in some wild isolates.
                     Half of these genes apparently became non-functional by a substitution creating a stop codon, and half through a deletion
                     affecting the reading frame. Five of the putatively functional alleles were only found in CB4856 and JU258. The other polymorphisms
                     observed in the same gene sequences also make CB4856 and JU258 the most divergent strains compared to N2, with 25 and 14 polymorphisms,
                     respectively (compared to 4 polymorphisms in Bergerac/RW7000, the third most divergent strain). Overall, chemoreceptor genes
                     appear to have a 5- to 10-fold higher level of polymorphism than the whole genome average, and different genes showed widely
                     different polymorphism levels, from 15 polymorphisms in 9.4 kb to none in 17 kb.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_1_4"></a>1.1.4.&nbsp;Microsatellite polymorphism
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Microsatellites are short repeated sequences (ranging from two to six nucleotides), which are highly prone to slippage during
                     replication, and therefore highly mutable. Microsatellite mutation rates in MA lines range from 8.93 &times; 10<sup>-5</sup> to 1.85 &times; 10<sup>-2</sup> per locus per generation, depending of the size and nature of the repeats (<a href="#bib28">Frisse, 1999</a>; <a href="#bib56">Sivasundar and Hey, 2003</a>). Mononucleotide microsatellites, also called homopolymeric loci, are sequences of eight or more repeats of a single nucleotide;
                     30 length change mutations and one base substitution were identified in homopolymers in MA lines (<a href="#bib18">Denver et al., 2004</a>), giving a mutation rate range from 4.5 &times; 10<sup>-6</sup> per generation for A/T homopolymers to 1.5 &times; 10<sup>-4</sup> per generation for long G/C homopolymers (<a href="#table1" title="Table 1. Mutation rates in mutation accumulation lines (MA) and mutational pattern in MA lines versus wild isolates">Table 1</a>). 2% of homopolymers are found in protein-coding sequences, which results in an average of 0.03 homopolymer mutation in exon
                     sequence per genome per generation.
                  </p>
                  <p><a href="#bib56">Sivasundar and Hey (2003)</a> genotyped 20 dinucleotide microsatellite loci in 23 wild isolates of <span class="emphasis"><em>C. elegans</em></span>. Only half of these loci were polymorphic, with a mean of 6.2 alleles (range 3&#8211;12). The low diversity (or lack of it) in
                     some microsatellites correlates with the low number of true repeats, as pointed out by <a href="#bib31">Haber et al. (2005)</a>.
                  </p>
                  <p><a href="#bib31">Haber et al. (2005)</a> sequenced 10 tri- or tetra-nucleotide repeats in 58 natural isolates. All but one were found to be polymorphic, with an average
                     allele number of seven.
                  </p>
                  <p>While data from <a href="#bib31">Haber et al. (2005)</a> fit the infinite allele mutation model (every mutation creates a new allele), those from <a href="#bib56">Sivasundar and Hey (2003)</a> better fit the stepwise mutation model (mutations occur by insertion or deletion of a single repeat), which is also suggested
                     by the MA line pattern (<a href="#bib28">Frisse, 1999</a>; <a href="#bib56">Sivasundar and Hey, 2003</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_1_5"></a>1.1.5.&nbsp;Effective population size
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>An important parameter in population genetics is the effective population size (N<sub>e</sub>), which represents the number of individuals of an equivalent ideal random-mating population under mutation and genetic drift,
                     or the number of individuals contributing to the next generation. It is calculated from the mutation rate &#956; and the observed
                     molecular diversity <span class="symbol">&#960;</span> from <span class="symbol">&#960;</span> = 4N<sub>e</sub>&#956;. For a given mutation rate, it is thus equivalent to a measure of molecular diversity. Practically, N<sub>e</sub> is reduced by inbreeding or non-random mating due to geographical structure, and is almost always smaller than the number
                     of breeding individuals (<a href="#bib32">Hartl and Clark, 1997</a>; <a href="#bib9">Charlesworth et al., 2003</a>).
                  </p>
                  <p>Estimates of N<sub>e</sub> from microsatellites for the whole species range from 200 to 44,000, depending on which locus is considered (<a href="#bib56">Sivasundar and Hey, 2003</a>). From AFLP data, it was estimated to be 9,600 for the four samples over France (<a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>). These numbers are surprisingly small for such a widespread species with large local populations (densities of <span class="emphasis"><em>C. elegans</em></span> can reach 10 individuals / g in a compost heap; <a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>; and on decaying mushroom compost numbers are far larger; A. Cutter and E. Dolgin, pers. comm.).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_1_6"></a>1.1.6.&nbsp;Conclusion: Low genetic diversity of <span class="emphasis"><em>C. elegans</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>At least in the locations sampled so far, <span class="emphasis"><em>C. elegans</em></span> presents a low genetic diversity, comparable to that of humans and 20-fold lower than that found in <span class="emphasis"><em>Drosophila melanogaster</em></span>. Its molecular mutation rate is not particularly low. The level of genetic diversity depends on the structure of natural
                     populations. For example, <span class="emphasis"><em>Homo sapiens</em></span> has a low nucleotide diversity, due to a recent bottleneck in the population. The pattern of microsatellite diversity of
                     <span class="emphasis"><em>C. elegans</em></span> does not show evidence for a recent global bottleneck followed by population expansion (<a href="#bib56">Sivasundar and Hey, 2003</a>). The reproductive mode also influences genetic diversity, which is expected to be reduced in partial selfers such as <span class="emphasis"><em>C. elegans</em></span> or <span class="emphasis"><em>Arabidopsis thaliana</em></span>: selfing results in a two-fold reduction in effective population size, and genetic diversity is further reduced by the fact
                     that selection at one locus results in the selection of the entire genome under genetic linkage (<a href="#bib32">Hartl and Clark, 1997</a>). We will now review the geographic structuring and the reproductive mode of <span class="emphasis"><em>C. elegans</em></span>.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_2"></a>1.2.&nbsp;Population structure and geographic differentiation
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_2_1"></a>1.2.1.&nbsp;Diversity at different spatial scales
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>When the natural isolates are partitioned by world region, most of the genetic variation is found within a continent: for
                     example, the microsatellite genetic diversities within North America and Europe are of 0.83 and 0.85, respectively, which
                     is close to the 0.92 diversity over the whole dataset (<a href="#bib31">Haber et al., 2005</a>).
                  </p>
                  <p>At a local scale, <a href="#bib31">Haber et al. (2005)</a> and <a href="#bib57">Sivasundar and Hey (2005)</a> found microsatellite polymorphism within compost heaps in Germany, and soil samples in California, respectively. Effective
                     population size estimates range from 750 to 12,000 in different local populations. In these two sets of samples, individuals
                     were isolated after several (five to 24) days in the laboratory and may therefore have been the progeny (F1&#8211;F5) of wild individuals
                     and thus not faithfully represent diversity in the corresponding natural populations.
                  </p>
                  <p>Barri&egrave;re and F&eacute;lix picked independent wild-born individuals directly from the substrate. Using AFLP analysis, estimates for
                     within-population nucleotide diversity range from 0.02 to 0.43 &times; 10<sup>-3</sup> per nucleotide for different populations (N<sub>e</sub> = 200&#8211;5,100; <a href="#table2" title="Table 2. Pairwise molecular distances between N2 and other wild isolates">Table 2</a>). The within-sample diversity is spread over a large range, from almost null to a level of diversity comparable to the global
                     diversity (and to the distance between N2 and CB4856). In cases where the intra-sample diversity was low, a diversity of 0.80
                     &times; 10<sup>-3</sup> was recovered when three samples were sampled 10&#8211;30 meters away (<a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>).
                  </p>
                  <p>Thus, local diversity seems to remain high until the scale of a few cm<sup>3</sup>. In some samples, it then breaks down, probably as a consequence of local bottlenecks followed by selfing of a single individual
                     (see <a href="#sec1_4" title="1.4.&nbsp;Outcrossing versus selfing in C. elegans">section 1.4</a>). The large variance in within-sample diversity may also reflect a lack of demographic stability in some populations, with
                     transient bottlenecks that reduce genetic diversity. Together with known ecological features of <span class="emphasis"><em>C. elegans</em></span>, this suggests that a metapopulation dynamics (many small populations, experiencing frequent extinctions and <span class="emphasis"><em>de novo</em></span> colonizations) could apply to its natural populations.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_2_2"></a>1.2.2.&nbsp;Geographic structure of populations
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Local diversity may be the result of local mutation or of input of alleles through migration, which can be distinguished by
                     measuring the frequency of different alleles in different populations.
                  </p>
                  <p>Allelic frequencies correlate only weakly with geographic origin among the sampled world regions (Western Europe, North America,
                     Australia, Hawaii, Madeira; <a href="#bib56">Sivasundar and Hey, 2003</a>; <a href="#bib31">Haber et al., 2005</a>). Indeed, isolates from the same region can be significantly divergent, whereas very similar genotypes are found on different
                     continents (<a href="#bib35">Hodgkin and Doniach, 1997</a>; <a href="#bib43">Koch et al., 2000</a>). At a smaller scale in the four populations sampled in France, for more than half of polymorphisms within each population,
                     both alleles are also found outside this population (<a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>), suggesting that local diversity is fostered by either a large population size at population foundation, or later migration.
                     Moreover, no correlation was found between geographical and genetic distances (<a href="#bib56">Sivasundar and Hey, 2003</a>; <a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>), thus further suggesting that the migration rate is large enough that isolation by distance does not contribute substantially.
                  </p>
                  <p>These results suggest that <span class="emphasis"><em>C. elegans</em></span> is a better migrant than previously thought. The dauer larva is likely to be a dispersal stage, through its association with
                     other animals (see <a href="http://www.wormbook.org/chapters/www_phylogrhabditids/phylorhab.html" target="_top">The phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> and other rhabditids</a>). Because <span class="emphasis"><em>C. elegans</em></span> is easily found within gardens and compost heaps, human activity may also be a major factor in its present migration patterns
                     and distribution.
                  </p>
                  <p>Two wild isolates however stand out in terms of level of divergence and specific alleles, namely CB4856 and JU258. 70% of
                     the SNPs identified in CB4856/Hawaii were not found in nine other strains, indicating that this strain has significantly diverged
                     (<a href="#bib43">Koch et al., 2000</a>). JU258/Madeira was not included in the earlier analyses, but now appears to share specific polymorphisms with CB4856 (<a href="#bib60">Stewart et al., 2004</a>; <a href="#bib31">Haber et al., 2005</a>). Much diversity may still be in store in undersampled parts of the world.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3"></a>1.3.&nbsp;Comparison with <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span></h3>
                     </div>
                  </div>
               </div>
               <p>Among the <span class="emphasis"><em>Caenorhabditis</em></span> species that are most closely related to <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib42">Kiontke et al., 2004</a>; see <a href="http://www.wormbook.org/chapters/www_phylogrhabditids/phylorhab.html" target="_top">The phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> and other rhabditids</a> and <a href="http://www.wormbook.org/chapters/www_evolutionsexdetermin/evolutionsexdetermin.html" target="_top">The evolution of nematode sex determination</a>), the level of genetic diversity appears to correlate with the mode of reproduction, being higher in the obligate outcrosser
                  <span class="emphasis"><em>C. remanei</em></span>.
               </p>
               <p><a href="#bib29">Graustein et al. (2002)</a> compared nucleotide diversity of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-2" target="_blank">tra-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> and <span class="emphasis"><em>COII</em></span> in three species (<a href="#figure4">Figure 4</a>). The two selfing species, <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>, show low levels of diversity, with <span class="emphasis"><em>C. briggsae</em></span> being more polymorphic than <span class="emphasis"><em>C. elegans</em></span>, while the outcrossing species, <span class="emphasis"><em>C. remanei</em></span>, is highly polymorphic, from one to two orders of magnitude above <span class="emphasis"><em>C. elegans</em></span>.
               </p>
               <p>&nbsp;</p>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="genetfig4.jpg"><img src="genetfig4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 4. Silent site nucleotide diversity for three genes (<span class="emphasis"><em>COII, <a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a>,</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-2" target="_blank">tra-2</a></em></span>) in <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em>C. briggsae</em></span>, and <span class="emphasis"><em>C. remanei</em></span>.&nbsp;</b>Open circles represent estimates of silent site nucleotide diversity that are based on pairwise sequence comparisons. Error
                        bars represent the 95% confidence intervals for this estimate. The self-fertilizing species (<span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>) always harbor less variation than the cross-fertilizing species (<span class="emphasis"><em>C. remanei</em></span>). This difference between mating systems is more pronounced for the two nuclear genes (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-2" target="_blank">tra-2</a></em></span>) than for the mitochondrial gene (<span class="emphasis"><em>COII</em></span>). From <a href="#bib29">Graustein et al. (2002)</a>.
                     </p>
                  </div>
               </div>
               <p>Similarly for the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=odr-3" target="_blank">odr-3</a></em></span> gene, nucleotide diversity was found to be 0.08 &times; 10<sup>-3</sup> per nucleotide in 10 <span class="emphasis"><em>C. elegans</em></span> isolates, 1.2 &times; 10<sup>-3</sup> in <span class="emphasis"><em>C. briggsae</em></span>, and 12.9 &times; 10<sup>-3</sup> in <span class="emphasis"><em>C. remanei</em></span>, including one amino-acid replacement. The diversity within a single local <span class="emphasis"><em>C. remanei</em></span> population is two orders of magnitude higher than in the whole <span class="emphasis"><em>C. elegans</em></span> species (<a href="#bib40">Jovelin et al., 2003</a>).
               </p>
               <p>Interestingly, other more distant hermaphroditic nematode species, <span class="emphasis"><em>Pristionchus pacificus</em></span> (<a href="#bib58">Srinivasan et al., 2001</a>) and <span class="emphasis"><em>Oscheius tipulae</em></span> (D. Ba&iuml;lle and M.-A. F., unpublished), show much higher nucleotide diversity than <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>. The mode of reproduction is clearly not the sole determinant of genetic diversity, and population dynamics is likely to
                  be an important factor.
               </p>
               <p>This low intraspecific diversity contrasts with a large divergence between the three species: 9.4% between <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>, and 11.6 (<a href="#bib40">Jovelin et al., 2003</a>) to 11.9% (<a href="#bib63">Thomas and Wilson, 1991</a>) between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span>, about 10-fold higher than intraspecific variation of <span class="emphasis"><em>C. remanei</em></span>. The turnover of nucleotides at neutral sites is almost saturated between these different species (<a href="#bib55">Shabalina and Kondrashov, 1999</a>), which renders molecular evolution tests based on synonymous mutation rate difficult. On the other hand, polymorphism within
                  <span class="emphasis"><em>C. elegans</em></span> is so low that the amount of sequence information needed to look at DNA polymorphism patterns is large, and that for a given
                  gene may not be very meaningful.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_4"></a>1.4.&nbsp;Outcrossing versus selfing in <span class="emphasis"><em>C. elegans</em></span></h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. elegans</em></span> reproduces through self-fertile XX hermaphrodites and facultative XO males (see <a href="http://www.wormbook.org/chapters/www_introsexdetermination/introsexdetermination.html" target="_top">Introduction to sex determination</a>). Hermaphrodites first produce sperm, then oocytes, through meiotic divisions; hermaphrodites cannot cross-fertilize and
                  males are required for outcrossing to occur. In laboratory conditions, hermaphrodites are limited by their sperm production
                  to about 300 self-progeny, whereas males may have a much larger number of progeny. In outcrossing events, male sperm is more
                  efficient in mating than hermaphrodite sperm, and in <span class="emphasis"><em>C. briggsae</em></span> (AF16 strain), X-bearing male sperm is more efficient than non-X-bearing male sperm (<a href="#bib44">LaMunyon and Ward, 1995</a>; <a href="#bib45">LaMunyon and Ward, 1997</a>).
               </p>
               <p>The outcrossing frequency depends on the rate of spontaneous male production and the mating efficiency. Males can be produced
                  by i. non-disjunction of the X chromosome at meiosis; ii. mating of a XO male with a XX hermaphrodite.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_4_1"></a>1.4.1.&nbsp;Outcrossing in the laboratory
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In laboratory conditions, spontaneous non-disjunction of the X chromosome occurs at a low rate (about 1/1,000), which is subject
                     to genetic variation (<a href="#bib35">Hodgkin and Doniach, 1997</a>) and is altered by environmental factors such as temperature (<a href="#bib49">Nigon, 1949</a>).
                  </p>
                  <p>When N2 males are placed in culture with hermaphrodites at an initial 50% frequency, their proportion is rapidly reduced to
                     the non-disjunction equilibrium, due to a low mating ability and the apparent absence of an advantage for outcrossed over
                     selfed progeny (<a href="#bib33">Hodgkin, 1983</a>; <a href="#bib10">Chasnov and Chow, 2002</a>; <a href="#bib59">Stewart and Phillips, 2002</a>; <a href="#bib12">Cutter and Payseur, 2003</a>).
                  </p>
                  <p>Males from other strains of <span class="emphasis"><em>C. elegans</em></span>, such as CB4855 and CB4856, have higher mating ability (<a href="#bib35">Hodgkin and Doniach, 1997</a>; J. Hodgkin, pers. comm.). Moreover, we cannot generalize from these observations to wild individuals in their natural environment.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_4_2"></a>1.4.2.&nbsp;Outcrossing in natural populations
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Outcrossing in the wild can be detected by several methods:</p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Recombination of alleles at different loci</p>
                           <p>The most sensitive method for detecting outcrossing is based on reassortments of alleles at different loci in the genome.
                              If <span class="emphasis"><em>C. elegans</em></span> only reproduced by selfing, different loci would follow the same tree lineage as mitochondrial DNA, and the whole genome
                              would behave as a single genetic linkage block. Indeed, a partial but significant correlation between phylogenies built from
                              either nuclear or mitochondrial sequence data (<a href="#figure3">Figure 3</a>) suggests that outcrossing occurs, but is rare in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib21">Denver et al., 2003</a>).
                           </p>
                           <p>Outcrossing can be inferred when all four combinations of two alleles at two loci are found (4-gamete test; <a href="#bib37">Hudson and Kaplan, 1985</a>), or by mapping the occurrence of an allele onto an independently established tree, provided there is no homoplasy (that
                              is, no convergent mutations). Many such putative recombination events were found using sequence (<a href="#bib43">Koch et al., 2000</a>; <a href="#bib21">Denver et al., 2003</a>; <a href="#bib31">Haber et al., 2005</a>) or AFLP data (<a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>). Yet large chromosomal regions appear to remain linked, suggesting little outcrossing (<a href="#bib24">Egilmez et al., 1995</a>; <a href="#bib43">Koch et al., 2000</a>; <a href="#bib21">Denver et al., 2003</a>).
                           </p>
                           <p>Linkage disequilibrium is a quantitative measure of association between polymorphic loci in a population; since the recombination
                              frequency increases with outcrossing, linkage disequilibrium decreases. In <span class="emphasis"><em>Drosophila melanogaster</em></span> populations, linkage disequilibrium only extends for 100 bp-1 kb. Within local <span class="emphasis"><em>C. elegans</em></span> populations, linkage disequilibrium was found to be high for randomly chosen loci in the samples from Northwest Germany (<a href="#bib31">Haber et al., 2005</a>) and France (<a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>). The outcrossing rate can be estimated from linkage disequilibrium and effective population size. In two populations sampled
                              in France, it was estimated to one outcrossing every 7,400 (95% confidence interval: 6,000&#8211;19,000) and 17,400 (95% c. i.:
                              1/3,200-complete selfing) generations, suggesting exceptional outcrossing.
                           </p>
                           <p>At the global scale however, no linkage disequilibrium could be detected using random microsatellite markers (<a href="#bib56">Sivasundar and Hey, 2003</a>). It may be that microsatellites mutate too fast to detect the chromosomal linkage blocks suggested by <a href="#bib24">Egilmez et al. (1995)</a>, <a href="#bib43">Koch et al. (2000)</a>, <a href="#bib21">Denver et al. (2003)</a> at this scale of the whole species.
                           </p>
                        </li>
                        <li>
                           <p>Presence of heterozygotes</p>
                           <p>Given the frequency of two alleles at a given locus in a population, the outcrossing frequency can be calculated from the
                              observed frequency of heterozygotes (assuming non-assortative mating of different genotypes).
                           </p>
                           <p>(<a href="#bib57">Sivasundar and Hey, 2005</a>) isolated <span class="emphasis"><em>C. elegans</em></span> individuals using a RNA interference-based assay (performed several days after soil sample collection) and measured heterozygote
                              frequency using microsatellite polymorphisms. From allele frequencies and the frequency of heterozygotes, they estimated the
                              outcrossing rate in the wild ancestors at the time of sampling. For different populations sampled in Southern California,
                              estimates range from complete selfing to one outcrossing every 5 generations, with a rate of one outcrossing every 7 generations
                              over the whole dataset (i.e. an outcrossing rate of 14%).
                           </p>
                           <p>Among individuals directly sampled from four natural populations from France, heterozygotes could also be detected using microsatellites,
                              and the overall outcrossing rate estimated to 1% (<a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>). The difference between both studies may possibly result from variations between natural populations, either in the ecological
                              context (for example seasonal or geographical variations in outcrossing rates) or in genetic variation in spontaneous-male-occurrence
                              or fertility.
                           </p>
                           <p>These heterozygote-based estimates do not seem compatible with the much lower estimates inferred from linkage disequilibrium
                              data, and with the partial congruence of mitochondrial and nuclear phylogenies. However, the latter methods measure outcrossing
                              over a longer time period, taking into account population structure, changes in selfing rate over time and possible mating
                              preferences.
                           </p>
                        </li>
                        <li>
                           <p>Presence of males</p>
                           <p>Outcrossing results in a high frequency of males in the next generation, since half of the cross-progeny is male. A survey
                              of individual worms sampled directly from the wild (13 samples on 3 sampling sites over 6 months) found only two males out
                              of 1,135 individuals (<a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>). This very low occurrence of males is fully compatible with the rate of spontaneous male production in the laboratory. These
                              populations thus reproduce mostly by selfing, but occasional bursts of sexual reproduction would remain undetected by this
                              criterion.
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec1_4_3"></a>1.4.3.&nbsp;Conclusion: A world traveler with rare sexual encounters
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Overall, these results suggest that <span class="emphasis"><em>C. elegans</em></span> overwhelmingly reproduces by selfing in the wild and that outcrossing is only occasional. Most likely, males produced by
                     spontaneous X-chromosome non-disjunction only rarely mate. Since male mating results in more males, sexual reproduction in
                     small populations is likely to occur by unstable bursts of variable duration. Recombination of genotypes is rendered possible
                     by the coexistence of diverse genotypes in the same location. Outcrossing is sufficiently infrequent relative to the mutation
                     rate that the genomes remain incompletely mixed over large chromosomal segments.
                  </p>
                  <p>On an evolutionary timescale, the level of outcrossing has consequences. Since males of most <span class="emphasis"><em>C. elegans</em></span> wild isolates retain the ability to mate (<a href="#bib35">Hodgkin and Doniach, 1997</a>), even if not very efficiently, the maintenance of genes required for male-specific development and behavior seems to imply
                     the existence of outcrossing in the wild. On the other hand, <span class="emphasis"><em>C. elegans</em></span> hermaphrodites produce much less male-attracting pheromone than females of other <span class="emphasis"><em>Caenorhabditis</em></span> species, consistent with the fact that outcrossing is not required (<a href="#bib10">Chasnov and Chow, 2002</a>; <a href="#bib46">Lipton et al., 2004</a>).
                  </p>
                  <p>Other theoretical consequences of a high degree of selfing are the rapid elimination of highly deleterious recessive mutations
                     (because they are not masked in the heterozygous state when they appear) and on the other hand the accumulation of slightly
                     deleterious mutations (&#8220;M&uuml;ller's ratchet&#8221;). In addition to outcrossing, the rapid evolution of compensatory mutations (see
                     below) could help to eliminate the phenotypic effect of deleterious mutations. The evolutionary feedback of phenotypes on
                     genotypes becomes highly relevant at this point.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Phenotypic diversity
                     </h2>
                  </div>
               </div>
            </div>
            <p>Phenotypic evolution is the product of two processes, 1. the effect of mutational and environmental change on the phenotype,
               followed by 2. the sorting of these phenotypes in the wild by selection and drift. As with DNA evolution, the effect of random
               mutation on the rate and pattern of phenotypic variation can then be compared with the actual natural diversity.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_1"></a>2.1.&nbsp;Phenotypic change upon mutation accumulation
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2_1_1"></a>2.1.1.&nbsp;&#8220;Deleterious&#8221; mutations: a fitness measure in the laboratory
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Any operational definition of fitness is only partial and may not include the most relevant parameters in the wild, which
                     vary with ecological contexts. In the laboratory, a &#8220;fitness&#8221; parameter can be computed from measures of progeny number and
                     generation time. This laboratory &#8220;fitness&#8221; decreases at a rate of about 0.1% per generation in mutation accumulation lines,
                     which is slower than in <span class="emphasis"><em>Drosophila</em></span> (<a href="#bib41">Keightley and Caballero, 1997</a>; <a href="#bib67">Vassilieva and Lynch, 1999</a>; <a href="#bib66">Vassilieva et al., 2000</a>; <a href="#figure2">Figure 2</a>). Knowing the fitness decrease the molecular mutation rate and the proportion of constrained nucleotides as estimated from
                     a genome comparison with <span class="emphasis"><em>C. briggsae</em></span>, it appears that most deleterious mutations that are eliminated in the wild cannot be detected in the laboratory, either
                     because they have a very mild deleterious effect that would nevertheless be relevant in large populations or because they
                     have a stronger effect in another environment (<a href="#bib14">Davies et al., 1999</a>; <a href="#bib18">Denver et al., 2004</a>; <a href="#bib27">Estes et al., 2004</a>).
                  </p>
                  <p>Very strikingly, mutation accumulation lines recover remarkably rapidly when mutation and competition between resulting genotypes
                     are allowed through a population expansion. This rapid recovery over 10-80 generations most likely occurs through compensatory
                     extragenic mutations (<a href="#bib26">Estes and Lynch, 2003</a>). A population undergoing successive bottlenecks and expansions is thus able to recover despite the fixation of deleterious
                     alleles by random drift during the bottlenecks.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2_1_2"></a>2.1.2.&nbsp;Specific phenotypes
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Body size changes rapidly upon mutation of the N2 genome and most mutation accumulation lines show a decrease in body size
                     compared to N2 (<a href="#figure2">Figure 2</a>), as also observed after EMS-mediated mutagenesis, systematic RNAi screens or artificial selection experiments (<a href="#bib3">Azevedo et al., 2002</a>). Thus, from a given genotype, some phenotypic changes are easier than others. Such patterns of spontaneous variation can
                     then be taken into account when studying variations in natural populations (<a href="#bib22">Dichtel-Danjoy and F&eacute;lix, 2004</a>).
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_2"></a>2.2.&nbsp;Polymorphic phenotypes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Even though <span class="emphasis"><em>C. elegans</em></span> is not very diverse at the molecular level, wild strains show phenotypic variation among them in laboratory conditions. Because
                  of the paucity of ecological data, their relevance in the natural context is at this point poorly understood.
               </p>
               <p>Note that the N2 strain has been maintained in the laboratory for years before being frozen and may not be the best representative
                  of a natural <span class="emphasis"><em>C. elegans</em></span> isolate. The Bergerac derivatives were useful when molecular polymorphisms were mostly detected by transposon-tagging, yet
                  are not the current best choice, because their (sick) phenotypes may not be more representative of natural variation than
                  those induced in the laboratory. Indeed, cultures have been maintained for over 20 years before freezing and show different
                  phenotypes (<a href="#bib35">Hodgkin and Doniach, 1997</a>). More generally, any phenotype found in a single wild isolate may not be very relevant.
               </p>
               <div class="itemizedlist">
                  <ul type="disc">
                     <li>
                        <p><span class="emphasis"><em>&#8220;Laboratory fitness&#8221; and a tradeoff between progeny number and generation time</em></span></p>
                        <p>Progeny number and generation time under standard laboratory conditions vary among wild isolates (<a href="#bib1">Abdul Kader and C&ocirc;t&eacute;, 1996</a>; <a href="#bib35">Hodgkin and Doniach, 1997</a>). The brood size of the selfing hermaphrodite corresponds to the number of sperm produced prior to the irreversible onset
                           of oogenesis. This results in a tradeoff between sperm number and generation time: the more sperm produced, the longer the
                           generation time (<a href="#bib34">Hodgkin and Barnes, 1991</a>; <a href="#bib1">Abdul Kader and C&ocirc;t&eacute;, 1996</a>; <a href="#bib11">Cutter, 2004</a>). It remains to be assessed whether these phenotypic measures are relevant in the wild; moreover, the selection pressures
                           on brood size may vary with ecological conditions.
                        </p>
                        <p>Many male-female species (including <span class="emphasis"><em>Caenorhabditis</em></span> spp.) show an inbreeding depression when low-frequency recessive deleterious mutations are rendered homozygous by inbreeding,
                           and conversely, hybrid vigor (heterosis) when two strains are crossed. At this day, such a phenomenon was not observed in
                           F1 hybrids between N2 and Bergerac for growth rate, fertility or longevity (<a href="#bib38">Johnson and Hutchinson, 1993</a>). Conversely, it would be interesting to study whether the divergence of some isolated populations (for example on islands)
                           may result in an &#8220;outbreeding depression&#8221; (partial speciation) if the genomes are artificially recombined.
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Egg-laying</em></span>: When well-fed, N2 hermaphrodites lay embryos at mid to late gastrulation, whereas many other isolates lay their eggs at
                           an earlier stage (M.-A. F&eacute;lix, unpublished).
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Frequency of spontaneous males and male fertility</em></span> (<a href="#bib35">Hodgkin and Doniach, 1997</a>).
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Hermaphrodite and male longevity</em></span> (<a href="#bib48">McCulloch and Gems, 2003</a>).
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Body size</em></span>: Many <span class="emphasis"><em>C. elegans</em></span> strains are shorter than N2. The size variation between N2 and CB4855 has been mapped to chromosomes V and X (<a href="#bib1">Abdul Kader and C&ocirc;t&eacute;, 1996</a>; <a href="#bib35">Hodgkin and Doniach, 1997</a>).
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Clumping, oxygen and ethanol sensation</em></span>: Behavioral traits, such as the olfactory repertoire (<a href="#bib69">Wicks et al., 2001</a>; <a href="#bib60">Stewart et al., 2004</a>), are likely to be under diversifying selection pressures depending on ecological conditions.
                        </p>
                        <p>The best studied polymorphism in <span class="emphasis"><em>C. elegans</em></span> concerns the Clumping and Bordering behaviors: in some <span class="emphasis"><em>C. elegans</em></span> isolates such as CB4856, adults tend to aggregate on the border of the <span class="emphasis"><em>E. coli</em></span> lawn of standard culture plates (whereas N2 adults aggregate once the plate is just starved; <a href="#bib35">Hodgkin and Doniach, 1997</a>; <a href="#bib15">de Bono and Bargmann, 1998</a>; <a href="#figure5">Figure 5</a>; see <a href="http://www.wormbook.org/chapters/www_aggregation/aggregation.html" target="_top">Aggregation</a>). This phenotypic polymorphism is caused by a genetic polymorphism at a single locus, called <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=npr-1" target="_blank">npr-1</a></em></span>, which encodes a neuropeptide Y receptor homolog. The solitary feeders such as N2 possess a form of the receptor with reduced
                           activity, whereas the Clumpers have a dominant gain-of-function variation in the amino-acid sequence (<a href="#bib15">de Bono and Bargmann, 1998</a>). Other <span class="emphasis"><em>Caenorhabditis</em></span> species carry the N2 form of the receptor (<a href="#bib52">Rogers et al., 2003</a>), although some clump, suggesting that the same phenotype may be obtained through variation at another locus (M. de Bono,
                           pers. comm.). Most strikingly, the evolutionary change within <span class="emphasis"><em>C. elegans</em></span> is a gain of function of the <a href="http://www.wormbase.org/db/seq/protein?name=NPR-1;class=protein" target="_blank">NPR-1</a> protein. The ecological relevance of the Clumping phenotype as seen in the laboratory is not clear at this point. It is likely
                           that the effect is mediated by the lower oxygen levels at the border of the bacterial lawn and in a worm group (<a href="#bib30">Gray et al., 2004</a>). Variation in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=npr-1" target="_blank">npr-1</a></em></span> also affects recovery of worms upon exposure to high ethanol levels (<a href="#bib13">Davies et al., 2004</a>).
                        </p>
                        <p>&nbsp;</p>
                        <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="genetfig5.jpg"><img src="genetfig5_s.jpg" border="2" align="middle" alt=" figure 5"></a>
  </td>
</tr>
                           </table>
                           <div class="caption">
                              <p><b>Figure 5. Phenotypic polymorphisms: Clumping (Clp) and plugging (Plg).&nbsp;</b>Left: N2. Right: CB4857. Pictures: Marie Delattre, IJM.
                              </p>
                           </div>
                        </div>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>RNAi and transposons</em></span>: The Hawaiian strain CB4856 is partially resistant to RNAi inactivation of germ line-expressed genes, due to variation at
                           one major locus, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ppw-1" target="_blank">ppw-1</a></em></span>, which encodes a PAZ/PIWI-domain-containing protein (<a href="#bib64">Tijsterman et al., 2002</a>; see <a href="http://www.wormbook.org/chapters/www_RNAimechanisms/RNAimechanisms.html" target="_top">RNAi mechanisms</a>). CB4856 displays several molecular polymorphisms in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ppw-1" target="_blank">ppw-1</a></em></span> gene, including a one-base pair deletion that results in an early stop codon (<a href="#bib64">Tijsterman et al., 2002</a>). The phenotypic significance in the wild is unknown, but may be related to transposon activity.
                        </p>
                        <p>Transposon number and/or activity in the germ line are elevated in some &#8220;mutator&#8221; wild strains compared to N2 (<a href="#bib2">Anderson, 1995</a>; <a href="#bib24">Egilmez et al., 1995</a>; see <a href="http://www.wormbook.org/chapters/www_transposons/transposons.html" target="_top">Transposons in <span class="emphasis"><em>C. elegans</em></span></a>). Some of the chromosomal regions with high transposon number are retained in several wild strains, suggesting that they
                           are not necessarily an evolutionary deadend.
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Resistance to various environmental hazards</em></span>: Resistance to a toxic environment is likely to be a fast-evolving and polymorphic character. For example, a population sampled
                           in a South California desert shows an unusual resistance to high temperatures, apparently due to the balancing of one heterozygote
                           locus (<a href="#bib35">Hodgkin and Doniach, 1997</a>). This natural population has unfortunately not be resampled since.
                        </p>
                        <p>Pathogens are likely to be a major survival issue for <span class="emphasis"><em>C. elegans</em></span> in the wild (see <a href="http://www.wormbook.org/chapters/www_intermicrobpath/intermicrobpath.html" target="_top">Interactions with microbial pathogens</a> and <a href="http://www.wormbook.org/chapters/www_ecolCaenorhabditis/ecolCaenorhabditis.html" target="_top">Ecology of Caenorhabditis species</a>). <span class="emphasis"><em>C. elegans</em></span> strains differ in their response to the pathogenic bacterium <span class="emphasis"><em>Bacillus thuringiensis</em></span> in terms of survival rate, infection levels, pumping rate and evasion behavior (<a href="#bib54">Schulenburg and Muller, 2004</a>). Resistance to the bacterium <span class="emphasis"><em>Serratia marcescens</em></span> is found to be polymorphic within a local population of <span class="emphasis"><em>C. elegans</em></span> and different strains of <span class="emphasis"><em>Serratia marcescens</em></span> specifically affect different <span class="emphasis"><em>C. elegans</em></span> strains (<a href="#bib53">Schulenburg and Ewbank, 2004</a>): it is possible that both species coevolve in an evolutionary arms race.
                        </p>
                        <p>The cuticle composition varies between wild isolates of <span class="emphasis"><em>C. elegans</em></span>, as can be detected by specific antibody staining. The variation maps at a locus called <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=srf-1" target="_blank">srf-1</a></em></span> (<a href="#bib50">Politz et al., 1987</a>) and a variant allele results in a weaker response to the cuticle-bound pathogenic bacterium <span class="emphasis"><em>Microbacterium nematophilum</em></span> (<a href="#bib36">Hodgkin et al., 2000</a>).
                        </p>
                        <p>Phenotypic variations in resistance to phenol-derived toxic compounds (<a href="#bib1">Abdul Kader and C&ocirc;t&eacute;, 1996</a>) and to the volatile anesthetic halothane (<a href="#bib65">van Swinderen et al., 1997</a>) have also been described.
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Plugging</em></span>. In many animal species, males leave a copulatory plug on the female's genitalia after mating, which is often thought to
                           prevent remating. N2 males do not leave a plug, whereas males of several other wild isolates of <span class="emphasis"><em>C. elegans</em></span> do (<a href="#bib35">Hodgkin and Doniach, 1997</a>; <a href="#figure5">Figure 5</a>). This plug does not fully prevent remating by another male, but makes it less probable (<a href="#bib7">Barker, 1994</a>); it is possible that the plug has other unknown consequences. Polymorphisms in Plugging are found within local populations
                           (<a href="#bib6">Barri&egrave;re and F&eacute;lix, 2005</a>). The phenotypic polymorphism between N2 and several Plugging strains is the result of allelic variation at one major locus
                           called <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=plg-1" target="_blank">plg-1</a></em></span>, which is yet to be identified at the molecular level; the N2 phenotype is recessive (<a href="#bib35">Hodgkin and Doniach, 1997</a>).
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Dauer entry and exit.</em></span> Entry into the dauer stage and exit from it are key developmental decisions in the life of <span class="emphasis"><em>C. elegans</em></span> (see <a href="http://www.wormbook.org/chapters/www_ecolCaenorhabditis/ecolCaenorhabditis.html" target="_top">Ecology of <span class="emphasis"><em>Caenorhabditis</em></span> species</a>). The sets of environmental cues that influence them and the doses at which they operate and interact with each other vary:
                           for example, N2 is more sensitive to dauer pheromone than some other strains (<a href="#bib68">Viney et al., 2003</a>).
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Development.</em></span> One of the few non-invariant cell division patterns in N2 is that of <a href="http://www.wormbase.org/db/get?name=P3.p;class=Cell" target="_blank">P3.p</a>, a ventral epidermal cell: in about half of the animals, it fuses to the hypodermal syncytium hyp7 in the L2 stage, whereas
                           in the other half, it divides in the L3 stage and its daughters then fuse with hyp7 (<a href="#bib25">Eisenmann et al., 1998</a>; <a href="#bib61">Sulston and Horvitz, 1977</a>). This division frequency is polymorphic in <span class="emphasis"><em>C. elegans</em></span> (5&#8211;60%) and varies even more within the <span class="emphasis"><em>Caenorhabditis</em></span> genus (0&#8211;100%; <a href="#bib16">Delattre and F&eacute;lix, 2001</a>; <a href="#bib17">Delattre and F&eacute;lix, 2001</a>; <a href="#bib8">Bonnaud et al., 2002</a>).
                        </p>
                        <p>In <span class="emphasis"><em>C. briggsae</em></span>, the morphology of the male tail sensory rays varies within and among different strains: in the reference strain AF16, ray
                           3 is displaced posteriorly (compared with the <span class="emphasis"><em>C. elegans</em></span> pattern) and rays 3 and 4 are fused in nearly all animals, whereas in some other strains of <span class="emphasis"><em>C. briggsae</em></span> only about half of the males show ray 3&#8211;4 fusion (<a href="#bib4">Baird, 2001</a>).
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Penetrance of mutations</em></span></p>
                        <p>When laboratory mutations are mapped using SNPs of the CB4856 strain, it is a frequent finding that the penetrance is greatly
                           modified by the CB4856 background, which is usually bad news for the mapper, but demonstrates an underlying evolutionary change
                           in relevant genes.
                        </p>
                     </li>
                  </ul>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_3"></a>2.3.&nbsp;Genetic analysis of a natural phenotypic variation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Phenotypic variation between two wild isolates may be due to variation at a single major locus, in which case it can be studied
                  genetically like a laboratory mutant phenotype. It is then relatively easy to determine the nature of the gene and the type
                  of alteration (<span class="emphasis"><em>cis</em></span>-regulatory or protein change, loss or gain-of-function), as was done for the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=npr-1" target="_blank">npr-1</a></em></span> locus.
               </p>
               <p>The phenotypic variation may however be due to genetic variation at more than one major locus. This can be deduced either
                  from a phenotypic analysis of the F2s of a cross, or using Recombinant Inbred Lines (RIL; <a href="#figure6">Figure 6</a>). Each RI line has a random combination of the two parental genomes in the homozygous state - a great simplification compared
                  to F2 analysis because it removes all heterozygous combinations and allows the study of frequency traits on an isogenic population.
                  If the variation is monogenic, each parental phenotype will be found in half of the RI lines. Otherwise, the proportions may
                  be altered, or intermediate or extreme phenotypes be found. With several loci, the different alleles may interact additively
                  or epistatically. For example, the variation in <a href="http://www.wormbase.org/db/get?name=P3.p;class=Cell" target="_blank">P3.p</a> division frequency between the N2 and CB4857 or that between the <span class="emphasis"><em>C. briggsae</em></span> strains AF16 and HK104 in male ray position are the consequences of genetic variation at several loci (<a href="#bib8">Bonnaud et al., 2002</a>; <a href="#bib5">Baird et al., 2005</a>; <a href="#figure6">Figure 6</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="324">
<tr>
  <td align="center"><p><a href="genetfig6.jpg"><img src="genetfig6_s.jpg" border="2" align="middle" alt=" figure 6"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 6. Recombinant inbred lines and near isogenic lines.&nbsp;</b>A. A set of <span class="emphasis"><em>n</em></span> Recombinant Inbred Lines (RIL) is produced by picking <span class="emphasis"><em>n</em></span> individuals in the F2 generation of a cross between two wild isolates, and letting them self for 10&#8211;15 generations until
                        the resulting strains are isogenic. RIL have been built between the reference strain N2 and a number of other wild isolates,
                        including Bergerac (<a href="#bib39">Johnson and Wood, 1982</a>; <a href="#bib23">Ebert et al., 1993</a>; <a href="#bib64">van Swinderen et al., 1997</a>), CB4856 (S. Wicks and R. Plasterk; M. Hammarlund &amp; E. Jorgensen, pers. comm.), DR1350 (<a href="#bib67">Viney et al., 2003</a>), CB4857 (<a href="#bib8">Bonnaud et al., 2002</a>), etc. B. Frequency distribution of male ray 3 position (black bars) and ray 3-4 fusion (red bars) among Recombinant Inbred
                        Lines of the <span class="emphasis"><em>C. briggsae</em></span> AF16 and HK104 strains (<a href="#bib5">Baird et al., 2005</a>). The phenotypes of the parental lines are indicated by arrows. The phenotypic variation is due to variation at several loci,
                        with transgressive segregation. In addition, the only partial correlation of the two phenotypes between the different lines
                        (not shown) show that they can evolve independently. C. A Near Isogenic Line is obtained by moving a chromosomal segment from
                        one genetic background to another by repeated crosses, which can be assisted by flanking markers. For example, a Plugging
                        <span class="emphasis"><em>plg-1(<a href="http://www.wormbase.org/db/get?name=e2001;class=Variation" target="_blank">e2001</a>)III</em></span> strain in the N2 background was built in this manner (<a href="#bib35">Hodgkin and Doniach, 1997</a>).
                     </p>
                  </div>
               </div>
               <p>Even if two wild isolates display a similar phenotype, the phenotype of some RI lines may lie outside the range of the phenotypic
                  distribution of both parents. This so-called transgressive segregation indicates that alleles from both parents at different
                  loci act in the same direction on the trait when combined (<a href="#bib65">van Swinderen et al., 1997</a>; <a href="#bib17">Delattre and F&eacute;lix, 2001</a>) and thus indicates evolution in the molecular mechanisms that underlie the phenotype. For example, if the two parental isolates
                  have similar phenotypes, some of the lines may show a distinct phenotype, thus revealing cryptic genetic variation.
               </p>
               <p>The genetic structure of populations (see above) and the genetic architecture of a polymorphic trait bear on how a polymorphism
                  appears, spreads and is maintained in wild populations. A polymorphism found in extant populations may be a transient quasi-neutral
                  polymorphism (which may be maintained for a long time because of population isolation), a recent and currently spreading variant,
                  or a polymorphism that is maintained because of selective advantages of both forms in specific ecological conditions.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Abdul Kader, N., and C&ocirc;t&eacute;, M. G. (1996). Isolement, identification et caract&eacute;risation de souches qu&eacute;b&eacute;coises du n&eacute;matode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Fund. app. Nemat. <span class="emphasis"><em>19</em></span>, 381&#8211;389.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Anderson, P. (1995). Mutagenesis. In: <span class="emphasis"><em>Caenorhabditis elegans: modern biological analysis of an organism</em></span>, H. F. Epstein, and D. C. Shakes, eds., San Diego, Academic Press, pp. 31&#8211;58.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Azevedo, R.B., Keightley, P.D., Lauren-Maatta, C., Vassilieva, L.L., Lynch, M., and Leroi, A.M. (2002). Spontaneous mutational
                        variation for body size in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>162</em></span>, 755&#8211;765.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12399386&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Baird, S.E. (2001). Strain-specific variation in the pattern of caudal papillae in <span class="emphasis"><em>Caenorhabditis briggsae</em></span> (Nematoda: Rhabditidae); implications for species identification. Nematology <span class="emphasis"><em>3</em></span>, 373&#8211;376.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Baird, S.E., Davidson, C.R., and Bohrer, J.C. (2005). The genetics of ray pattern variation in <span class="emphasis"><em>Caenorhabditis</em></span> briggsae. BMC Evol. Biol. <span class="emphasis"><em>5</em></span>, 3.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15634358&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2148-5-3" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Barri&egrave;re, A., and F&eacute;lix, M.-A. (2005). High local genetic diversity and low outcrossing rate in <span class="emphasis"><em>Caenorhabditis elegans</em></span> natural populations. Curr. Biol. <span class="emphasis"><em>15</em></span>, 1176&#8211;1184.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16005289&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.06.022" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Barker, D.M. (1994). Copulatory plugs and paternity assurance in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Anim. Behav. <span class="emphasis"><em>48</em></span>, 147&#8211;156.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bonnaud, L., Delattre, M., and F&eacute;lix, M.-A. (2002). Division and fate of vulval precursor cells: is polymorphism an image
                        of interspecific divergence? Paper presented at: Fourth International Congress of Nematology, Tenerife, Spain, Brill, Leiden,
                        The Netherlands.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Charlesworth, B., Charlesworth, D., and Barton, N.H. (2003). The effects of genetic and geographic structure on neutral variation.
                        Ann. Rev. Ecol. Evol. Syst. <span class="emphasis"><em>34</em></span>, 99&#8211;125.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chasnov, J.R., and Chow, K.L. (2002). Why are there males in the hermaphroditic species <span class="emphasis"><em>Caenorhabditis elegans</em></span>? Genetics <span class="emphasis"><em>160</em></span>, 983&#8211;994.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11901116&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Cutter, A.D. (2004). Sperm-limited fecundity in nematodes: how many sperm are enough? Evolution Int. J. Org. Evolution <span class="emphasis"><em>58</em></span>, 651&#8211;655.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15119448&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Cutter, A.D., Aviles, L., and Ward, S. (2003). The proximate determinants of sex ratio in <span class="emphasis"><em>C. elegans</em></span> populations. Genet. Res. <span class="emphasis"><em>81</em></span>, 91&#8211;102.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12872911&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Davies, A.G., Bettinger, J.C., Thiele, T.R., Judy, M.E., and McIntire, S.L. (2004). Natural variation in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=npr-1" target="_blank">npr-1</a></em></span> gene modifies ethanol responses of wild strains of <span class="emphasis"><em>C. elegans</em></span>. Neuron <span class="emphasis"><em>42</em></span>, 731&#8211;743.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15182714&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.neuron.2004.05.004" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Davies, E.K., Peters, A.D., and Keightley, P.D. (1999). High frequency of cryptic deleterious mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>285</em></span>, 1748&#8211;1751.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10481013&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">de Bono, M., and Bargmann, C.I. (1998). Natural variation in a neuropeptide Y receptor homolog modifies social behavior and
                        food response in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>94</em></span>, 679&#8211;689.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9741632&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Delattre, M., and F&eacute;lix, M.A. (2001). Microevolutionary studies in nematodes: a beginning. Bioessays <span class="emphasis"><em>23</em></span>, 807&#8211;819.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11536293&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Delattre, M., and F&eacute;lix, M.A. (2001). Polymorphism and evolution of vulval precursor cell lineages within two nematode genera,
                        <span class="emphasis"><em>Caenorhabditis</em></span> and <span class="emphasis"><em>Oscheius</em></span>. Curr. Biol. <span class="emphasis"><em>11</em></span>, 631&#8211;643.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11369226&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Denver, D.R., Morris, K., Kewalramani, A., Harris, K.E., Chow, A., Estes, S., Lynch, M., and Thomas, W.K. (2004). Abundance,
                        distribution, and mutation rates of homopolymeric nucleotide runs in the genome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Mol. Evol. <span class="emphasis"><em>58</em></span>, 584&#8211;595.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15170261&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s00239-004-2580-4" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Denver, D.R., Morris, K., Lynch, M., and Thomas, W.K. (2004). High mutation rate and predominance of insertions in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> nuclear genome. Nature <span class="emphasis"><em>430</em></span>, 679&#8211;682.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15295601&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02697" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Denver, D.R., Morris, K., Lynch, M., Vassilieva, L.L., and Thomas, W.K. (2000). High direct estimate of the mutation rate
                        in the mitochondrial genome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>289</em></span>, 2342&#8211;2344.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11009418&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Denver, D.R., Morris, K., and Thomas, W.K. (2003). Phylogenetics in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: an analysis of divergence and outcrossing. Mol. Biol. Evol. <span class="emphasis"><em>20</em></span>, 393&#8211;400.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12644560&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Dichtel-Danjoy, M.L., and F&eacute;lix, M.A. (2004). Phenotypic neighborhood and micro-evolvability. Trends Genet. <span class="emphasis"><em>20</em></span>, 268&#8211;276.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15109782&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.tig.2004.03.010" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ebert, R.H., 2nd, Cherkasova, V.A., Dennis, R.A., Wu, J.H., Ruggles, S., Perrin, T.E., and Shmookler Reis, R.J. (1993). Longevity-determining
                        genes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: chromosomal mapping of multiple noninteractive loci. Genetics <span class="emphasis"><em>135</em></span>, 1003&#8211;1010.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8307318&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Egilmez, N.K., Ebert, R.H., 2nd, and Shmookler Reis, R.J. (1995). Strain evolution in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: transposable elements as markers of interstrain evolutionary history. J. Mol. Evol. <span class="emphasis"><em>40</em></span>, 372&#8211;381.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7769614&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Eisenmann, D.M., Maloof, J.N., Simske, J.S., Kenyon, C., and Kim, S.K. (1998). The beta-catenin homolog <a href="http://www.wormbase.org/db/seq/protein?name=BAR-1;class=protein" target="_blank">BAR-1</a> and <a href="http://www.wormbase.org/db/seq/protein?name=LET-60;class=protein" target="_blank">LET-60</a> Ras coordinately regulate the Hox gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-39" target="_blank">lin-39</a></em></span> during <span class="emphasis"><em>Caenorhabditis elegans</em></span> vulval development. Development <span class="emphasis"><em>125</em></span>, 3667&#8211;3680.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9716532&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Estes, S., and Lynch, M. (2003). Rapid fitness recovery in mutationally degraded lines of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Evolution <span class="emphasis"><em>57</em></span>, 1022&#8211;1030.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Estes, S., Phillips, P.C., Denver, D.R., Thomas, W.K., and Lynch, M. (2004). Mutation accumulation in populations of varying
                        size: the distribution of mutational effects for fitness correlates in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>166</em></span>, 1269&#8211;1279.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15082546&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Frisse, L. (1999) Understanding the Mechanisms of Microsatellite Formation and Mutation using the Model Organism <span class="emphasis"><em>Caenorhabditis elegans</em></span>, Ph.D. thesis, University of Missouri, Kansas City.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Graustein, A., Gaspar, J.M., Walters, J.R., and Palopoli, M.F. (2002). Levels of DNA polymorphism vary with mating system
                        in the nematode genus Caenorhabditis. Genetics <span class="emphasis"><em>161</em></span>, 99&#8211;107.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12019226&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Gray, J.M., Karow, D.S., Lu, H., Chang, A.J., Chang, J.S., Ellis, R.E., Marletta, M.A., and Bargmann, C.I. (2004). Oxygen
                        sensation and social feeding mediated by a <span class="emphasis"><em>C. elegans</em></span> guanylate cyclase homologue. Nature <span class="emphasis"><em>430</em></span>, 317&#8211;322.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15220933&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02714" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Haber, M., Schungel, M., Putz, A., Muller, S., Hasert, B., and Schulenburg, H. (2005). Evolutionary History of <span class="emphasis"><em>Caenorhabditis elegans</em></span> Inferred from Microsatellites: evidence for spatial and temporal genetic differentiation and the occurrence of outbreeding.
                        Mol. Biol. Evol. <span class="emphasis"><em>22</em></span>, 160&#8211;173.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15371529&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msh264" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hartl, D.L., and Clark, A.G. (1997). Principles of Population Genetics (3rd edition) Sunderland, MA, Sinauer Associates, Inc.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J. (1983). Male phenotypes and mating efficiency in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>103</em></span>, 43&#8211;64.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J., and Barnes, T.M. (1991). More is not better: brood size and population growth in a self-fertilizing nematode.
                        Proc. R. Soc. Lond. B Biol. Sci. <span class="emphasis"><em>246</em></span>, 19&#8211;24.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1684664&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J., and Doniach, T. (1997). Natural variation and copulatory plug formation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>146</em></span>, 149&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9136008&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J., Kuwabara, P.E., and Corneliussen, B. (2000). A novel bacterial pathogen, <span class="emphasis"><em>Microbacterium nematophilum</em></span>, induces morphological change in the nematode <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>10</em></span>, 1615&#8211;1618.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11137017&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hudson, R.R., and Kaplan, N.L. (1985). Statistical properties of the number of recombination events in the history of a sample
                        of DNA sequences. Genetics <span class="emphasis"><em>111</em></span>, 147&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4029609&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Johnson, T.E., and Hutchinson, E.W. (1993). Absence of strong heterosis for life span and other life history traits in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>134</em></span>, 465&#8211;474.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8325483&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Johnson, T.E., and Wood, W.B. (1982). Genetic analysis of life-span in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. <span class="emphasis"><em>79</em></span>, 6603&#8211;6607.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jovelin, R., Ajie, B.C., and Phillips, P.C. (2003). Molecular evolution and quantitative variation for chemosensory behaviour
                        in the nematode genus <span class="emphasis"><em>Caenorhabditis</em></span>. Mol. Ecol. <span class="emphasis"><em>12</em></span>, 1325&#8211;1337.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12694294&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Keightley, P.D., and Caballero, A. (1997). Genomic mutation rates for lifetime reproductive output and lifespan in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>94</em></span>, 3823&#8211;3827.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9108062&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kiontke, K., Gavin, N.P., Raynes, Y., Roehrig, C., Piano, F., and Fitch, D.H. (2004). <span class="emphasis"><em>Caenorhabditis</em></span> phylogeny predicts convergence of hermaphroditism and extensive intron loss. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 9003&#8211;9008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15184656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0403094101" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Koch, R., van Luenen, H.G., van der Horst, M., Thijssen, K.L., and Plasterk, R.H. (2000). Single nucleotide polymorphisms
                        in wild isolates of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome Res. <span class="emphasis"><em>10</em></span>, 1690&#8211;1696.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11076854&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                     		<span class="bibliomisc">LaMunyon, C.W., and Ward, S. (1995). Sperm precedence in a hermaphroditic nematode (<span class="emphasis"><em>Caenorhabditis elegans</em></span>) is due to competitive superiority of male sperm. Experientia <span class="emphasis"><em>51</em></span>, 817&#8211;823.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7649241&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                     		<span class="bibliomisc">LaMunyon, C.W., and Ward, S. (1997). Increased competitiveness of nematode sperm bearing the male X chromosome. Proc. Natl.
                        Acad. Sci. USA <span class="emphasis"><em>94</em></span>, 185&#8211;189.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8990183&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lipton, J., Kleemann, G., Ghosh, R., Lints, R., and Emmons, S.W. (2004). Mate searching in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: a genetic model for sex drive in a simple invertebrate. J. Neurosci. <span class="emphasis"><em>24</em></span>, 7427&#8211;7434.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15329389&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1523/JNEUROSCI.1746-04.2004" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Maupas, E. (1900). Modes et formes de reproduction des n&eacute;matodes. Arch. Zool. exp. <span class="emphasis"><em>8</em></span>, 463&#8211;624.</span>
                     <a href="http://www.wormbase.org/papers/1900-maupas/index.html" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                     		<span class="bibliomisc">McCulloch, D., and Gems, D. (2003). Evolution of male longevity bias in nematodes. Aging Cell <span class="emphasis"><em>2</em></span>, 165&#8211;173.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12882409&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Nigon, V. (1949). Modalit&eacute;s de la reproduction et d&eacute;terminisme du sexe chez quelques n&eacute;matodes libres. Ann. des. Sc. Nat.
                        Zool. <span class="emphasis"><em>11</em></span>, 1&#8211;132.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Politz, S.M., Chin, K.J., and Herman, D.L. (1987). Genetic analysis of adult-specific surface antigenic differences between
                        varieties of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>117</em></span>, 467&#8211;476.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3692138&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rogers, C., Reale, V., Kim, K., Chatwin, H., Li, C., Evans, P., and de Bono, M. (2003). Inhibition of <span class="emphasis"><em>Caenorhabditis elegans</em></span> social feeding by FMRFamide-related peptide activation of <a href="http://www.wormbase.org/db/seq/protein?name=NPR-1;class=protein" target="_blank">NPR-1</a>. Nat. Neurosci. <span class="emphasis"><em>6</em></span>, 1178&#8211;1185.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14555955&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nn1140" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Schulenburg, H., and Ewbank, J.J. (2004). Diversity and specificity in the interaction between <span class="emphasis"><em>Caenorhabditis elegans</em></span> and the pathogen <span class="emphasis"><em>Serratia marcescens</em></span>. BMC Evol. Biol. <span class="emphasis"><em>4</em></span>, 49.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15555070&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2148-4-49" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Schulenburg, H., and Muller, S. (2004). Natural variation in the response of <span class="emphasis"><em>Caenorhabditis elegans</em></span> towards <span class="emphasis"><em>Bacillus thuringiensis</em></span>. Parasitology <span class="emphasis"><em>128</em></span>, 433&#8211;443.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15151149&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Shabalina, S.A., and Kondrashov, A.S. (1999). Pattern of selective constraint in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> genomes. Genet. Res. <span class="emphasis"><em>74</em></span>, 23&#8211;30.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10505405&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sivasundar, A., and Hey, J. (2003). Population genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span>: the paradox of low polymorphism in a widespread species. Genetics <span class="emphasis"><em>163</em></span>, 147&#8211;157.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12586703&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sivasundar, A., and Hey, J. (2005). Sampling from natural populations with RNAi reveals high outcrossing and population structure
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Curr. Biol. <span class="emphasis"><em>15</em></span>, 1598&#8211;1602.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16139217&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.08.034" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Srinivasan, J., Pires-daSilva, A., Gutierrez, A., Zheng, M., Jungblut, B., Witte, H., Schlak, I., and Sommer, R.J. (2001).
                        Microevolutionary analysis of the nematode genus <span class="emphasis"><em>Pristionchus</em></span> suggests a recent evolution of redundant developmental mechanisms during vulva formation. Evol. Dev. <span class="emphasis"><em>3</em></span>, 229&#8211;240.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11478520&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Stein, L.D., Bao, Z., Blasiar, D., Blumenthal, T., Brent, M.R., Chen, N., Chinwalla, A., Clarke, L., Clee, C., Coghlan, A.,
                        Coulson, A., D'Eustachio, P., Fitch, D.H.A., Fulton, L.A., Fulton, R.E., Griffiths-Jone, S., Harris, T.W., Hillier, L.W.,
                        Kamath, R., Kuwabara, P., Mardis, E.R., Marra, M.A., Miner, T.L., Minx, P., Mullikin, J.C., Plumb, R.W., Rogers, J., Schein,
                        J.E., Sohrmann, M., Spieth, R., Stajich, J.E., Wei, C., Willey, D., Wilson, R.K., Dubin, R., and Waterston, R.H. (2003). The
                        genome sequence of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>: a platform for comparative genomics. PLOS Biology <span class="emphasis"><em>1</em></span>, 166&#8211;192.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14624247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Stewart, A.D., and Phillips, P.C. (2002). Selection and maintenance of androdioecy in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>160</em></span>, 975&#8211;982.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11901115&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Stewart, M.K., Clark, N.L., Merrihew, G., Galloway, E.M., and Thomas, J.H. (2004). High Genetic Diversity in the Chemoreceptor
                        Superfamily of <span class="emphasis"><em>C. elegans</em></span>. Genetics.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15520260&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.035329" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sulston, J., and Horvitz, H.R. (1977). Postembryonic cell lineages of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>56</em></span>, 110&#8211;156.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Swan, K.A., Curtis, D.E., McKusick, K.B., Voinov, A.V., Mapa, F.A., and Cancilla, M.R. (2002). High-throughput gene mapping
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome Res. <span class="emphasis"><em>12</em></span>, 1100&#8211;1105.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12097347&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.208902" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Thomas, W.K., and Wilson, A.C. (1991). Mode and tempo of molecular evolution in the nematode <span class="emphasis"><em>Caenorhabditis</em></span>: cytochrome oxidase II and calmodulin sequences. Genetics <span class="emphasis"><em>128</em></span>, 269&#8211;279.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1649066&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Tijsterman, M., Okihara, K.L., Thijssen, K., and Plasterk, R.H. (2002). <a href="http://www.wormbase.org/db/seq/protein?name=PPW-1;class=protein" target="_blank">PPW-1</a>, a PAZ/PIWI protein required for efficient germline RNAi, is defective in a natural isolate of <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>12</em></span>, 1535&#8211;1540.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12225671&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed">
                     		<span class="bibliomisc">van Swinderen, B., Shook, D.R., Ebert, R.H., Cherkasova, V.A., Johnson, T.E., Shmookler Reis, R.J., and Crowder, C.M. (1997).
                        Quantitative trait loci controlling halothane sensitivity in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>94</em></span>, 8232&#8211;8237.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9223344&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Vassilieva, L.L., Hook, A.M., and Lynch, M. (2000). The fitness effects of spontaneous mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Evolution Int. J. Org. Evolution <span class="emphasis"><em>54</em></span>, 1234&#8211;1246.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11005291&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Vassilieva, L.L., and Lynch, M. (1999). The rate of spontaneous mutation for life-history traits in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>151</em></span>, 119&#8211;129.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9872953&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Viney, M.E., Gardner, M.P., and Jackson, J.A. (2003). Variation in <span class="emphasis"><em>Caenorhabditis elegans</em></span> dauer larva formation. Dev. Growth Differ. <span class="emphasis"><em>45</em></span>, 389&#8211;396.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12950280&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wicks, S.R., Yeh, R.T., Gish, W.R., Waterston, R.H., and Plasterk, R.H. (2001). Rapid gene mapping in <span class="emphasis"><em>Caenorhabditis elegans</em></span> using a high density polymorphism map. Nat. Genet. <span class="emphasis"><em>28</em></span>, 160&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11381264&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/88878" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Witherspoon, D.J., and Robertson, H.M. (2003). Neutral evolution of ten types of mariner transposons in the genomes of <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. J. Mol. Evol. <span class="emphasis"><em>56</em></span>, 751&#8211;769.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12911038&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s00239-002-2450-x" target="_blank">Article</a>
                     	
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by David H.A. Fitch. Last revised May 24, 2005. Published December 26, 2005. This chapter should be cited as: Barri&egrave;re,
                  A. and F&eacute;lix, M.-A. Natural variation and population genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span> (December 26, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.43.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2005 Antoine Barri&egrave;re and Marie-Anne F&eacute;lix, This is an open-access article distributed under the terms of the Creative
                  Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original
                  author and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e34" href="#d0e34">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:felix@ijm.jussieu.fr" target="_top">felix@ijm.jussieu.fr</a>.
               </p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All journal content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

