<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Single molecule fluorescent in situ hybridization (smFISH) of C. elegans worms and embryos</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#Miller_and_Chin-Sang_Ephsignaling_id" title="Single molecule fluorescent in situ hybridization (smFISH) of C. elegans worms and embryos">
      <link rel="next" href="#sec1" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="smFISH.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="Miller_and_Chin-Sang_Ephsignaling_id"></a>Single molecule fluorescent <span class="emphasis"><em>in situ</em></span> hybridization (smFISH) of <span class="emphasis"><em>C. elegans</em></span> worms and embryos<sup><a name="d0e12" href="#ftn.d0e12">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Ni Ji<sup>1</sup> and Alexander van Oudenaarden<sup>2</sup><em><span class="remark"><sup><a name="d0e44" href="#ftn.d0e44">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup>Department of Brain and Cognitive Sciences, Massachusetts Institute of Technology, Cambridge, MA 02139 USA </span></span><span class="affiliation"><span class="orgname"><sup>2</sup>Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139 USA, Hubrecht Institute, Royal Netherlands
                           Academy of Arts and Science and University Medical Center Utrecht, Uppsalalaan 8, 3584 CT, Utrecht, The Netherlands; </span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Methods in nematode species other than <em>C. elegans</em>
		</a></div><hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. Protocols</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec2-1">2.1. Probe design and synthesis</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-2">2.2. Fixation of <span class="emphasis"><em>C. elegans</em></span> worms and embryos</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-3">2.3. Hybridization</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-4">2.4. Imaging</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-5">2.5. Data analysis</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2-6">2.6. Controls and troubleshooting</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec3">3. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>In <span class="emphasis"><em>C. elegans</em></span>, the expression pattern of a gene provides important clues to understanding its biological function. To accurately depict
               endogenous transcriptional activity, a highly sensitive method is required to measure transcript levels in the intact tissue
               across various developmental stages. Conventional RNA <span class="emphasis"><em>in situ</em></span> hybridization methods using hapten- (biotin or digoxygenin) labeled RNA probes rely on antibody binding for visualization,
               and are thus only semi-quantitative at best (<a href="#bib8">Raap <span class="emphasis"><em>et al.</em></span> 1995</a>; <a href="#bib5">Levsky <span class="emphasis"><em>et al.</em></span> 2003</a>). Additionally, hapten-labeled probes are prone to diffuse localization (when conjugated with alkaline phosphatase), low
               sensitivity (when conjugated with fluorescent molecules), and non-specific probe binding. Here, we introduce a recently developed
               mRNA <span class="emphasis"><em>in situ</em></span> hybridization method (<a href="#bib11">Raj <span class="emphasis"><em>et al.</em></span> 2008</a>) that circumvents the above difficulties to give single molecule resolution of transcript detection.
            </p>
            <p>The single molecule fluorescent <span class="emphasis"><em>in situ</em></span> hybridization (smFISH) method differs from conventional approaches by using many short (about 20 base pairs long) oligonucleotide
               probes to target different regions of the same mRNA transcript (<a href="#bib11">Raj <span class="emphasis"><em>et al.</em></span> 2008</a>; <a href="#bib1">Femino <span class="emphasis"><em>et al.</em></span> 1998</a>). Each oligo is conjugated with only one fluorophore and thus faintly visible by itself. Binding of multiple oligos to the
               same transcript yields a bright spot, indicative of a single mRNA transcript. Since mis-bound probes are unlikely to co-localize,
               this method effectively reduces false-positive signal from non-specific probe binding. The small oligo size allows the probes
               to efficiently penetrate through target tissue, yielding robust detection of even low-abundant transcripts. Subsequently,
               the total number of fluorescent spots within a single cell or region can be unambiguously counted and compared across different
               developmental stages and genetic backgrounds.
            </p>
            <p>Given its many advantages, the smFISH method is a powerful tool to study transcriptional regulation. Its high sensitivity
               allows accurate characterization of the spatio-temporal patterns of endogenous gene expression. Its single-molecule resolution
               enables precise quantification of gene expression levels. Such quantitative information can in turn be used to assess, for
               example: 1) tissue-specific correlations in gene expression, 2) similarity and difference in gene expression across strains,
               3) variability in gene expression, and 4) tissue-specific signaling dynamics. To date, smFISH has been successfully applied
               to study a variety of questions in <span class="emphasis"><em>C. elegans</em></span> biology (<a href="#bib9">Raj <span class="emphasis"><em>et al.</em></span> 2010</a>; <a href="#bib3">Harterink <span class="emphasis"><em>et al.</em></span> 2011</a>; <a href="#bib4">Korzelius <span class="emphasis"><em>et al.</em></span> 2011</a>; <a href="#bib7">Middelkoop <span class="emphasis"><em>et al.</em></span> 2012</a>; <a href="#bib13">Saffer <span class="emphasis"><em>et al.</em></span> 2011</a>; <a href="#bib14">Seidel <span class="emphasis"><em>et al.</em></span> 2011</a>; <a href="#bib15">Topalidou <span class="emphasis"><em>et al.</em></span> 2011</a>). Here we provide the necessary technical details to set up and perform smFISH, from sample preparation to data analysis.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Protocols
                     </h2>
                  </div>
               </div>
            </div>
            <p>The following protocol covers the 5 major steps of smFISH: <span class="bold"><strong>A.</strong></span> Probe design and synthesis, <span class="bold"><strong>B.</strong></span> Fixation of <span class="emphasis"><em>C. elegans</em></span> worms and embryos, <span class="bold"><strong>C.</strong></span> Hybridization, <span class="bold"><strong>D.</strong></span> Image acquisition, <span class="bold"><strong>E.</strong></span> Data analysis. This protocol is largely adapted from the general smFISH protocol detailed in <a href="#bib10">Raj and Tyagi (2010</a>), with notes and modifications specific to <span class="emphasis"><em>C. elegans</em></span>. Unless otherwise noted, all reagents listed can be made in bulk ahead of time and stored at room temperature (RT).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-1"></a>2.1.&nbsp;Probe design and synthesis
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-1-1"></a>2.1.1.&nbsp;Design
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>smFISH probes are DNA oligonucleotide probes designed according to the follow criteria:</p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Probe length: 17-22 base pairs</p>
                        </li>
                        <li>
                           <p>Probe spacing: no less than 2 base pairs</p>
                        </li>
                        <li>
                           <p>Probe GC content: ideally around 45% to ensure uniform binding efficiency</p>
                        </li>
                        <li>
                           <p>Number of probes: ideally between 30 and 96 depending on target transcript length. Typically 48 probes are used to ensure
                              good signal quality, while as few as 20 have yielded satisfactory signal (e.g., <a href="#figure1">Figure 1</a> illustrates the result from 27 probes designed to target <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mab-5;class=Gene" target="_blank">mab-5</a></em></span> mRNA). To increase effective target length, part or all of the 3&#8217;UTR sequences can be combined with the cDNA sequence for
                              probe design.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

<tr>
  <td align="center"><p><a href="smFISH_fig01.jpg"><img src="smFISH_fig01_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 1.&nbsp;</b>Comparison of immunohistochemistry and smFISH results detecting <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mab-5;class=Gene" target="_blank">mab-5</a></em></span> expression in the <a href="http://www.wormbase.org/db/get?name=QL;class=Cell" target="_blank">QL</a> neuroblasts. <span class="bold"><strong>A.</strong></span> <a href="http://www.wormbase.org/db/get?name=MAB-5;class=Protein" target="_blank">MAB-5</a> immunostaining of a wild-type L1 larva. <a href="http://www.wormbase.org/db/get?name=MAB-5;class=Protein" target="_blank">MAB-5</a> expression is seen in the two <a href="http://www.wormbase.org/db/get?name=QL;class=Cell" target="_blank">QL</a> daughters above V5. (Adapted from <a href="#bib6">Maloof <span class="emphasis"><em>et al.</em></span> 1999</a>) <span class="bold"><strong>B.</strong></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mab-5;class=Gene" target="_blank">mab-5</a></em></span> mRNA detected by smFISH in L1 larva around the same age. Note the asymmetry in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mab-5;class=Gene" target="_blank">mab-5</a></em></span> mRNA abundance between the two <a href="http://www.wormbase.org/db/get?name=QL;class=Cell" target="_blank">QL</a> daughters. Scale bars are 2.5 &#956;m.
                        </p>
                     </div>
                  </div>
                  <p>A web-based probe design software developed by <a href="#bib11">Raj <span class="emphasis"><em>et al.</em></span> (2008</a>) is available free of charge at: <a href="http://www.biosearchtech.com/stellarisdesigner/" target="_top">http://www.biosearchtech.com/stellarisdesigner/</a>. By supplying the target RNA sequence and the above criteria, one obtains a list of probe sequences with optimized GC contents.
                     These sequences can then be directly submitted for probe synthesis.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2-1-2"></a>2.1.2.&nbsp;Synthesis, coupling and purification.
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="bold"><strong>Synthesis:</strong></span> Based on the designed oligonucleotide sequences, smFISH probes are generated en masse using a 96-well DNA synthesizer. Biosearch
                     Technologies (Novato, CA) offers synthesizing service for custom-designed smFISH probes. The probes can be ordered in both
                     coupled and uncoupled forms. While ordering coupled probes minimizes the work involved in probe preparation, obtaining uncoupled
                     probes allows the flexibility to choose fluorophores that are optimally compatible with the researcher's microscope setup
                     (e.g., filter cubes) and other experimental needs. Furthermore, coupled probes are delivered as a mix, which precludes the
                     option of using subsets of the probe library to 1) selectively target different parts of the transcript, or 2) perform negative
                     controls using deletion mutants (see additional comment in the following section and details of suggested controls in the
                     Controls and Troubleshooting section). In the following section, we further suggest a list of factors to consider when deciding
                     whether to pursue in-house coupling.
                  </p>
                  <p>Given the small amount of probes needed for each hybridization experiment, synthesis can be carried out at a small scale (Biosearch
                     now delivers 5 nmol of each probe per custom order. This amount is typically sufficient for hundreds of experiments). When
                     ordering uncoupled probes, the oligonucleotide probes should be synthesized with a 3&#8217;amine group to allow subsequent oligo-fluorophore
                     conjugation. Additionally the oligos should be desalted and resuspended in water as opposed to Tris EDTA (TE), which can interfere with subsequent amine-coupling reactions.
                  </p>
                  <p><span class="bold"><strong>Coupling:</strong></span></p>
                  <p><span class="emphasis"><em>Note: When deciding whether to perform in-house coupling or order readily coupled probes, we suggest considering the following:</em></span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p><span class="emphasis"><em>Is it clear which fluorophore to choose? This could vary depending on probe set and the optical setup.</em></span></p>
                        </li>
                        <li>
                           <p><span class="emphasis"><em>Is it necessary to split the probe sets to target different parts of a transcript? Pre-coupling requires mixing of all probes
                                    in a library, thus excluding the possibility of alternative probe combinations.</em></span></p>
                        </li>
                        <li>
                           <p><span class="emphasis"><em>Is in-house coupling feasible? Is there access to a nearby HPLC facility?</em></span></p>
                        </li>
                        <li>
                           <p><span class="emphasis"><em>Does experimental schedule allow an additional 3-4 days of probe preparation?</em></span></p>
                        </li>
                        <li>
                           <p><span class="emphasis"><em>Is budget a concern? A single probe set of 10 nmol quantity can be coupled to multiple fluorophores (up to 3-4 coupling per probe set is possible),
                                    avoiding the need to purchase multiple libraries.</em></span></p>
                        </li>
                     </ol>
                  </div>
                  <p>Prior to coupling, one needs to decide on the desired type of fluorophore. The current protocol uses succinimidyl ester derivatives
                     to couple to the 3&#8217; end of the oligonucleotide probes. Three types of commercially available fluorophores are commonly used:
                     Cy5 (GE Amersham), Alexa 594 (Molecular Probes, Invitrogen), and tetramethylrhodamine (TMR) (Molecular Probes, Invitrogen).
                     Biosearch also offers equivalent fluorophores (Quasar 670, and CAL Fluor 610 and 590) for their couple probe sets. In general
                     we have found that using fluorophores with shorter emission wavelengths (such as Alexa 488) does not yield reliable signal
                     due to high cellular autofluorescence. <a href="#table1" title="Table 1. Strengths and weaknesses of fluorophores commonly used in smFISH.">Table 1</a> summarizes the strengths and weaknesses of each of the fluorophores as observed in our hands.
                  </p>
                  <div class="table"><a name="table1"></a><p class="title"><span class="bold">Table 1. Strengths and weaknesses of fluorophores commonly used in smFISH.</span></p>
                     <table summary="Table 1. Strengths and weaknesses of fluorophores commonly used in smFISH." border="1">
                        <colgroup>
                           <col>
                           <col>
                           <col>
                           <col>
                           <col>
                           <col>
                        </colgroup>
                        <thead>
                           <tr valign="bottom">
                              <th align="center" valign="bottom">&nbsp;</th>
                              <th align="center" valign="bottom">Peak Excitation/Emission Wavelength</th>
                              <th align="center" valign="bottom">Photostability</th>
                              <th align="center" valign="bottom">Autofluorescence level</th>
                              <th align="center" valign="bottom">RFP compatible</th>
                              <th align="center" valign="bottom">GFP compatible</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td align="left"><span class="bold"><strong>Cy5</strong></span></td>
                              <td align="left">649/670 nm</td>
                              <td align="left">Low</td>
                              <td align="left">Low</td>
                              <td align="left">Yes</td>
                              <td align="left">Yes</td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top"><span class="bold"><strong>Alexa 594</strong></span></td>
                              <td align="left" valign="top">590/617 nm</td>
                              <td align="left" valign="top">High</td>
                              <td align="left" valign="top">Medium</td>
                              <td align="left" valign="top">Affected</td>
                              <td align="left" valign="top">Yes</td>
                           </tr>
                           <tr valign="top">
                              <td align="left" valign="top"><span class="bold"><strong>TMR</strong></span></td>
                              <td align="left" valign="top">564/570 nm</td>
                              <td align="left" valign="top">High</td>
                              <td align="left" valign="top">High</td>
                              <td align="left" valign="top">No</td>
                              <td align="left" valign="top">Yes</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <p>For multiplexing assays, we have obtained good results with combinations of all three dyes (provided that each probe set works
                     well when assayed on its own). We generally prefer to pair Cy5 with Alexa 594, as the two fluorophores tend to give better
                     signal quality than TMR. For initial testing, it is important to compare results from multiplex and single-fluorophore assays
                     to ensure consistency. Also keep an eye out for &#8220;suspiciously similar&#8221; spot patterns accross channels, which is indicative of cross-talk between
                     fluorescent channels.
                  </p>
                  <p><span class="underline">Reagents for coupling:</span></p>
                  <div class="itemizedlist">
                     <ul type="none">
                        <li>
                           <p>DMSO (if coupling to TMR)</p>
                        </li>
                        <li>
                           <p>*0.1 M Sodium Bicarbonate (in RNase free water, pH 8.0)</p>
                        </li>
                        <li>
                           <p>*1 M Sodium Bicarbonate (in RNase free water, pH 8.0)</p>
                        </li>
                        <li>
                           <p>Ethanol (&gt;95% pure)</p>
                        </li>
                        <li>
                           <p>3 M Sodium Acetate, pH 5.2</p>
                        </li>
                        <li>
                           <p>Fluorophore with succinimidyl ester group</p>
                        </li>
                     </ul>
                  </div>
                  <p>*Note: it is desirable to make sodium bicarbonate fresh, or check before use to make sure the pH level is correct.</p>
                  <p><span class="underline">Day 1:</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>From the uncoupled probe stock, combine 1 nmol (e.g., 10 &#956;l from a 100 &#956;M stock) of each probe into a single microcentrifuge
                              tube.
                           </p>
                        </li>
                        <li>
                           <p>Add 0.11 volumes 1 M sodium bicarbonate to give a final concentration of 0.1 M. If the total volume at this stage is &lt; 0.3
                              mL, add some 0.1 M sodium bicarbonate to bring total volume to 0.3 mL.
                           </p>
                        </li>
                        <li>
                           <p>Dissolve a small amount (roughly 0.2 mg) of dye into 50 &#956;l of 0.1 M sodium bicarbonate.</p>
                           <div class="orderedlist">
                              <ol type="a">
                                 <li>
                                    <p>Note: TMR can be hard to dissolve in aqueous solution, so one must first dissolve it in a small volume (&lt;5 &#956;l) of DMSO before
                                       adding 50 &#956;l of 0.1 M sodium bicarbonate.
                                    </p>
                                 </li>
                              </ol>
                           </div>
                        </li>
                        <li>
                           <p>Add the dissolved fluorophore to the oligos.</p>
                        </li>
                        <li>
                           <p>Cover the tube in foil to prevent photo-bleaching and let the reaction proceed overnight at room temperature with gentle rocking.</p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Day 2:</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>In the morning, precipitate the oligos by adding 10% volume/volume of 3M sodium acetate and then adding 2.5 volumes of 100%
                              EtOH. Store at &#8722;70&deg;C for at least 1 hour (up to overnight).
                           </p>
                        </li>
                        <li>
                           <p>Spin down the sample in a 4&deg;C microcentrifuge for at least 15 minutes at maximum speed (~16K RCF).</p>
                        </li>
                        <li>
                           <p>After centrifugation, one should see a small colored pellet at the bottom of the tube. Aspirate away the fluorescent supernatant
                              (containing uncoupled dye molecules) as completely as possible. If purification is not performed right away, the pellet is
                              stable and can be stored at &#8722;20&deg;C.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="bold"><strong>Purification:</strong></span> The pellet obtained from the coupling steps contains a mixture of coupled and uncoupled oligonucleotides. To separate the
                     two species, we take advantage of the fact that coupled probes experience a large increase in hydrophobicity compared to the
                     uncoupled ones. High-pressure liquid chromatography (HPLC) can thus be used to enrich for coupled probes.
                  </p>
                  <p><span class="underline">Reagents and equipments for purification:</span></p>
                  <div class="itemizedlist">
                     <ul type="none">
                        <li>
                           <p>0.1 M Triethylammonium acetate, pH 6.5, filtered and degassed (Buffer A)</p>
                        </li>
                        <li>
                           <p>Acetonitrile for HPLC (Buffer B)</p>
                        </li>
                        <li>
                           <p>HPLC with a dual wavelength detector to measure both DNA and fluorophore absorbtion</p>
                        </li>
                        <li>
                           <p>C18 Column for HPLC, 218TP104</p>
                        </li>
                        <li>
                           <p>SpeedVac rated for acetonitrile</p>
                        </li>
                     </ul>
                  </div>
                  <p><span class="underline">Day 2 (if continued immediately after coupling step 8):</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Resuspend pellet in appropriate volume (0.1&#8594;0.5 mL nuclease free water, depending on your HPLC)</p>
                        </li>
                        <li>
                           <p>Inject coupled probe into HPLC and run a program in which the percentage of buffer B varies from 7% to 30% over the course
                              of around 30 minutes with a flow rate of 1 mL/minute.
                           </p>
                           <div class="itemizedlist">
                              <ul type="none">
                                 <li>
                                    <p>Note: Set the detector to monitor DNA absorption (260 nm) and the absorption of the coupled fluorophore (e.g., 555 nm for
                                       TMR).
                                    </p>
                                 </li>
                              </ul>
                           </div>
                        </li>
                        <li>
                           <p>One will observe two broad peaks. The first contains uncoupled probes and will only show a peak in the 260 nm channel. The
                              second contains pure coupled probes and will show peaks in both channels. The two peaks will typically be separated by a few
                              minutes of time or longer, with TMR having narrowly separated peaks and Cy5 having broadly separated peaks. With a series
                              of microcentrifuge tubes, collect the entire second peak as soon as the signal begins rise and until it drops back to baseline.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p><span class="underline">Day 3:</span></p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Dry the purified probes in a SpeedVac vacuum concentrator (~ 3-5 hours for 0.5 mL).</p>
                           <div class="itemizedlist">
                              <ul type="none">
                                 <li>
                                    <p>Note: Be sure to prevent any light from hitting the probes during the drying process to prevent photobleaching, especially
                                       for photolabile dyes such as Cy5.
                                    </p>
                                 </li>
                              </ul>
                           </div>
                        </li>
                        <li>
                           <p>Resuspend all tubes together in a total of 50-100 &#956;l of TE, pH 8.0 (equivalent to roughly 0.1&#8211;1 &#956;M). This is the concentrated
                              probe stock.
                           </p>
                        </li>
                        <li>
                           <p>(Optional) Dilute this probe 1:10, 1:20, 1:50 and 1:100 in TE to make working stocks for testing probe concentration.</p>
                        </li>
                     </ol>
                  </div>
                  <p>At this point, probe synthesis is complete. Probes can be stored in TE at &#8722;20&deg;C for years.</p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-2"></a>2.2.&nbsp;Fixation of <span class="emphasis"><em>C. elegans</em></span> worms and embryos
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="underline">Reagents for fixation:</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Fixation solution: 4% paraformaldehyde (PFA) in 1x PBS</p>
                     </li>
                     <li>
                        <p>Bleaching solution for embryos (per 40 mL, store at 4&deg;C):</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>40 mL distilled water</p>
                              </li>
                              <li>
                                 <p>7.2 mL 5 N NaOH</p>
                              </li>
                              <li>
                                 <p>4.5 mL 6% NaHOCl</p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>M9 buffer (per 1L):</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>5.8 g Na<sub>2</sub>HPO4
                                 </p>
                              </li>
                              <li>
                                 <p>3.0 g KH<sub>2</sub>PO4
                                 </p>
                              </li>
                              <li>
                                 <p>0.5 g NaCl</p>
                              </li>
                              <li>
                                 <p>1.0 g NH<sub>4</sub>Cl
                                 </p>
                              </li>
                              <li>
                                 <p>Dissolve in distilled water (dH<sub>2</sub>O) to 1 L final volume
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Fixation protocol for worms (larvae and adult):</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Grow worms on plates seeded with OP50. Synchronize worms to the desired stage if needed.</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Note: The appropriate number of plates may vary depending on need, with the minimum requirements that 1) there will be enough
                                    worms to form a visible pellet in a microcentrifuge tube, and 2) each plate is not too crowded.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>Add 3-5 mL M9 buffer to the plate and swirl to release worms from surface, then transfer worms to a 15 mL conical centrifuge
                           tube.
                        </p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Note: Distilled or deionized water may be used instead of M9 in this and subsequent steps.</p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>Spin down to form a pellet and aspirate the supernatant.</p>
                     </li>
                     <li>
                        <p>Wash and spin down with 3-5 mL M9 buffer to rinse away bacteria and other debris.</p>
                     </li>
                     <li>
                        <p>Resuspend in 1 mL fixation solution and transfer to microcentrifuge tube. Keep rotating at room temperature for 45 min.</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Note: Since autofluorescence levels increase with fixation time, incubation in fixation solution should be kept short. This
                                    is especially important for older worms where tissue autofluorescence is relatively strong.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>Wash twice with 1 mL 1x PBS.</p>
                     </li>
                     <li>
                        <p>Resuspend in 1 mL of 70% EtOH. Keep rotating for overnight (or longer) at 4&deg;C. Store at 4&deg;C for up to a month.</p>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Fixation protocol for embryos:</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Grow worms on OP50 plates till there are plenty of gravid hermaphrodites.</p>
                     </li>
                     <li>
                        <p>Wash worms off the plates with M9 into a 15 mL conical centrifuge tube.</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Note: Distilled or deionized water may be used instead of M9 in this and subsequent steps.</p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>Spin down and resuspend in bleaching solution. Vortex or shake rigorously for 4-8 minutes until worms disappear and only embryos
                           remain.
                        </p>
                     </li>
                     <li>
                        <p>Spin down and aspirate. Then wash twice with M9 buffer.</p>
                     </li>
                     <li>
                        <p>Resuspend in 1 mL fixation solution and transfer to a microcentrifuge tube. Keep rotating at room temperature for 15 minutes.</p>
                     </li>
                     <li>
                        <p>Vortex and then immediately submerge tube in liquid nitrogen for 1 minute to freeze crack the eggshells.</p>
                     </li>
                     <li>
                        <p>Thaw in water at room temperature.</p>
                     </li>
                     <li>
                        <p>Once thawed, vortex and place on ice for 20 minutes.</p>
                     </li>
                     <li>
                        <p>Wash twice with 1 mL of 1x PBS.</p>
                     </li>
                     <li>
                        <p>Resuspend in 1 mL of 70% EtOH and keep rotating for overnight or longer at 4&deg;C. Embryos can be hybridized up to a week following
                           fixation.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-3"></a>2.3.&nbsp;Hybridization
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="underline">Reagents for hybridization:</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Hybridization buffer (per 10 mL, store at &#8722;20&deg;C freezer):</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>*1 g dextran sulfate</p>
                              </li>
                              <li>
                                 <p>10 mg <span class="emphasis"><em>Escherichia coli</em></span> tRNA
                                 </p>
                              </li>
                              <li>
                                 <p>100 &#956;l 200 mM vanadyl ribonucleoside complex (New England Biolabs, Inc.)</p>
                              </li>
                              <li>
                                 <p>40 &#956;l 50 mg/mL RNase free BSA (Ambion)</p>
                              </li>
                              <li>
                                 <p>**Formamide (deionized, Ambion)</p>
                              </li>
                              <li>
                                 <p>Add nuclease free (NF) water (Ambion) to 10 mL final volume</p>
                              </li>
                           </ul>
                        </div>
                        <p>* Note: Dextran sulfate is viscous and hard to dissolve at room temperature. One solution is to mix dextran sulfate with 4
                           mL NF water, then sonicate till no clumps are visible.
                        </p>
                        <p>** Note: Determine the amount based on the desired formamide concentration, e.g., 1 mL for 10% final concentration. Higher
                           concentration yields higher probe binding stringency.
                        </p>
                     </li>
                     <li>
                        <p>Wash buffer (per 50 mL):</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Formamide (deionized, Ambion), use same concentration as determined for hybridization buffer</p>
                              </li>
                              <li>
                                 <p>5 mL 20x SSC (RNase free, Ambion)</p>
                              </li>
                              <li>
                                 <p>Add NF water (Ambion) to 50 mL final volume</p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>DAPI stain: Prepare working stock of 5 ng/mL in RNase free water, store in the freezer.</p>
                     </li>
                     <li>
                        <p>2x SSC: Prepare from 20x SSC (RNase free, Ambion) in RNase free water (Ambion).</p>
                     </li>
                     <li>
                        <p>Antifade buffer (to be made fresh)</p>
                        <div class="orderedlist">
                           <ol type="1">
                              <li>
                                 <p>Glox buffer (per 50 mL):</p>
                                 <div class="itemizedlist">
                                    <ul type="none">
                                       <li>
                                          <p>2 mL 10% glucose in NF water</p>
                                       </li>
                                       <li>
                                          <p>250 &#956;l 2M Tris-HCl, pH 8.0</p>
                                       </li>
                                       <li>
                                          <p>5 mL 20x SSC (RNase free, Ambion)</p>
                                       </li>
                                    </ul>
                                 </div>
                              </li>
                              <li>
                                 <p>Glycosidase stock: Dilute commercial Glox (glucose oxidase, Sigma-Aldrich) in 50 mM sodium acetate to working stock of 3.7
                                    mg/mL, adjust to pH 5.0. Aliquot to 100 &#956;l units and store at &#8722;20&deg;C.
                                 </p>
                              </li>
                              <li>
                                 <p>Catalase (Sigma-Aldrich): store at 4&deg;C.</p>
                                 <p>Prior to imaging, mix 1 &#956;l each of the glycosidase stock and catalase into 100 &#956;l of Glox buffer. Each 100 &#956;l antifade mix
                                    is enough for 1&#8211;2 hybridization samples.
                                 </p>
                              </li>
                           </ol>
                        </div>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Day 1:</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Prepare the hybridization solution: to 100 &#956;l of hybridization buffer, add 1 &#956;l of each probe at the appropriate concentration,
                           then vortex and centrifuge.
                        </p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Note: For the initial test of probe concentration, one may perform four parallel hybridizations with probes diluted at 1:10,
                                    1:20, 1:50 and 1:100 in TE. In our hands, 1:20 works well for Cy5 coupled probes and 1:50 appears sufficient for Alexa 594
                                    coupled probes.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>Centrifuge the fixed sample and aspirate away the ethanol.</p>
                     </li>
                     <li>
                        <p>Resuspend in 1 mL wash buffer that contains the same concentration of formamide as the hybridization buffer. Let stand for
                           2&#8211;5 min.
                        </p>
                     </li>
                     <li>
                        <p>Centrifuge sample and aspirate away the wash buffer.</p>
                     </li>
                     <li>
                        <p>Add the hybridization solution prepared in step 1. Incubate overnight at 30&deg;C in the dark.</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Note: The modified protocol on the Biosearch website now recommends 4 hour incubation at 37&deg;C. We plan to try out this modification
                                    and compare the results carefully with our existing data sets. The result will be included in a future edition of this protocol.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                  </ol>
               </div>
               <p><span class="underline">Day 2:</span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>In the morning, add 1 mL wash buffer to the sample. Vortex, spin down and aspirate.</p>
                     </li>
                     <li>
                        <p>Resuspend in another 1 mL wash buffer and incubate at 30&deg;C for 30 min.</p>
                     </li>
                     <li>
                        <p>Spin down and aspirate. Resuspend in 1 mL wash buffer</p>
                     </li>
                     <li>
                        <p>Add 1.5 &#956;l DAPI stain (5 ng/mL) for nuclear counterstaining. Vortex and incubate at 30&deg;C for 30 min.</p>
                     </li>
                     <li>
                        <p>Spin down and aspirate the wash buffer. Resuspend in 2x SSC, vortex, spin down and aspirate.</p>
                     </li>
                     <li>
                        <p>If imaging without glox antifade solution (possible for Alexa 594 and TMR), resuspend in a small amount (enough the cover
                           the samples) of 2x SSC and proceed to imaging.
                        </p>
                     </li>
                     <li>
                        <p>If imaging with the antifade solution, resuspend in a small amount of glox buffer and proceed to imaging.</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Note: While hybridized samples can be stored temporarily in 2x SSC at 4&deg;C. Prolonged storage increases the risk of signal
                                    degradation. It is thus advisable to image immediately following hybridization.
                                 </p>
                              </li>
                              <li>
                                 <p>Note: The hybridization protocol posted by Biosearch (<a href="http://www.biosearchtech.com/assets/bti_custom_stellaris_celegans_protocol.pdf" target="_top">http://www.biosearchtech.com/assets/bti_custom_stellaris_c<span class="emphasis"><em>elegans</em></span>_protocol.pdf</a>) differs from ours in the following: 1) omission of the blocking reagents (tRNA and BSA) from the hybridization buffer; 2)
                                    hybridize at 37&deg;C (instead of 30&deg;C) for 4 hours (instead of overnight). At the moment, we have not tried or systematically
                                    compared their protocol with our results. We plan to do so and will report our results in a future edition of this protocol.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-4"></a>2.4.&nbsp;Imaging
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="bold"><strong>Microscopy equipment:</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Microscope: Standard wide-field fluorescence microscope (e.g., Nikon TE2000 or Ti, Zeiss Axiovert).</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Note: Confocal microscopes, while excellent in spatial precision, use high light intensity and cause rapid bleaching of smFISH
                                    signals. They are especially problematic when taking multiple z sections and are thus not recommended for smFISH imaging.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>Light source: A strong light source is essential for spot detection. Mercury or metal-halide lamp (e.g., ExFo Excite, Prior
                           Lumen 200) are both good candidates. The metal-halide lamps are generally brighter and thus ideally suited for far-red dyes
                           such as Cy5.
                        </p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Note: With a metal-halide lamp, the exposure time we use for smFISH signal detection is generally around 1-2s, while only
                                    100&#8211;200 ms is needed for GFP and DAPI.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>Filter sets: Three filters, excitation, dichroic, and emission filters, are needed for each fluorescent channel. Choice of
                           filters should be made based on the optical features of the corresponding fluorophores used (see <a href="#table2" title="Table 2. Examples of optical filter sets compatible with multiplex mRNA detection">Table 2</a> for a list of the filters we use).
                        </p>
                     </li>
                     <li>
                        <p>Camera: Standard cooled CCD camera optimized for low-light level imaging rather than speed (preferably with 13 mm pixel size
                           or less; e.g., CoolSNAP HQ from Pixis, Princeton Instruments).
                        </p>
                     </li>
                     <li>
                        <p>High numerical aperture (NA&gt; 1.3) 100x DIC objective (be sure to check transmission properties when using far red dyes such
                           as Cy5 or Cy5.5). We have also seen spots using an oil-immersion 60x objective, but the reduced spatial resolution makes the
                           spots somewhat more difficult to identify computationally.
                        </p>
                     </li>
                  </ol>
               </div>
               <div class="table"><a name="table2"></a><p class="title"><span class="bold">Table 2. Examples of optical filter sets compatible with multiplex mRNA detection</span></p>
                  <table summary="Table 2. Examples of optical filter sets compatible with multiplex mRNA detection" border="1">
                     <colgroup>
                        <col>
                        <col>
                        <col>
                        <col>
                        <col>
                     </colgroup>
                     <thead>
                        <tr valign="bottom">
                           <th align="center" valign="bottom">&nbsp;</th>
                           <th align="center" valign="bottom">Excitation</th>
                           <th align="center" valign="bottom">Dichroic</th>
                           <th align="center" valign="bottom">Emission</th>
                           <th align="center" valign="bottom">Supplier</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="left"><span class="bold"><strong>Cy5</strong></span></td>
                           <td align="left">HQ620/60x</td>
                           <td align="left">Q660LP</td>
                           <td align="left">HQ700/75m</td>
                           <td align="left">Chroma</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="bold"><strong>Alexa 594</strong></span></td>
                           <td align="left" valign="top">590DF10</td>
                           <td align="left" valign="top">610DRLP</td>
                           <td align="left" valign="top">620DF30</td>
                           <td align="left" valign="top">Omega</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="bold"><strong>TMR</strong></span></td>
                           <td align="left" valign="top">546DF10</td>
                           <td align="left" valign="top">555DRLP</td>
                           <td align="left" valign="top">580DF30</td>
                           <td align="left" valign="top">Omega</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
               <p><span class="bold"><strong>Software:</strong></span> Standard microscopy software capable of memorizing sample positions and imaging through z optical stacks. We currently use
                  MetaMorph (Molecular Devices), which offers easily programmable user interface.
               </p>
               <p><span class="bold"><strong>Imaging chamber</strong></span>: There are two purposes of the imaging chamber: 1) to affix the sample in a small region and prevent drying and 2) to minimize
                  the thickness of the sample by flattening, thereby reducing out of focus light. To assemble the chamber:
               </p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Pipette 2-5 &#956;l of the sample in antifade solution onto a clean 8mm round cover glass (Electron Microscopy Sciences, #1.5 thickness).</p>
                     </li>
                     <li>
                        <p>Gently tap a clean 22 &times; 22 mm square cover glass (VWR, #1 thickness) onto the drop of sample solution. This will cause the
                           round cover glass to quickly adhere to the square glass.
                        </p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Optional: Clean cover glasses before hand by rinsing with 70% EtOH and let dry.</p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p>Immediately flip the square glass so that the round glass is on top. Let sit for half a minute or so while covered in dark.</p>
                     </li>
                     <li>
                        <p>While waiting, adhere a square Silicone Isolator (Grace Biolabs, 20 mm diameter &times; 0.5 depth) to a regular microscope slide.</p>
                     </li>
                     <li>
                        <p>Gently remove excess antifade solution from the rim of the round cover glass with a Kimwipes wiper.</p>
                     </li>
                     <li>
                        <p>Adhere the square cover glass to the Silicone Isolator with the round cover glass facing towards the microscope slide. Press
                           down on the edges of the square glass to create a tight seal. This constitutes the imaging chamber.
                        </p>
                     </li>
                     <li>
                        <p>Affix the imaging chamber to the microscope stage. Make sure to position the microscope slide correctly so that the imaging
                           chamber faces the objective. Proceed to locate individual worms or embryos.
                        </p>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>What to expect:</strong></span> A successful hybridization and sample preparation should yield clear fluorescent spots roughly 200-500 nm in size. These
                  spots are so called &#8220;diffraction limited spots&#8221; based on the fact that the size of an mRNA molecule (nm) is far below the
                  optical limit of the widefield microscope (&#956;m). On a digitally acquired smFISH image, spot intensities usually vary within
                  2 fold of one another (for the same gene detected in the same fluorescent channel), with the exception of transcription centers
                  in the nuclei (where large amounts of nascent transcripts accumulate), which can be much brighter. Optimal smFISH signal should
                  be more than 1.5 times in intensity above the tissue background. When viewed on an image, the spots should be distinct in
                  shape (i.e., not blurred) and readily identifiable by eye. <a href="#figure2">Figure 2</a> illustrates two smFISH experiments, one probing <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=elt-2;class=Gene" target="_blank">elt-2</a></em></span> (a transcription factor involved in specifying the future intestine, <a href="#bib2">Fukushige <span class="emphasis"><em>et al.</em></span> 1998</a>) in embryos (from <a href="#bib9">Raj <span class="emphasis"><em>et al.</em></span> 2010</a>), the other staining three Wnt ligands expressed in the posterior body of the L1 larvae (from <a href="#bib3">Harterink <span class="emphasis"><em>et al.</em></span> 2011</a>).
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

<tr>
  <td align="center"><p><a href="smFISH_fig02.jpg"><img src="smFISH_fig02_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2.&nbsp;</b>Examples of smFISH staining from embryos and L1 larvae. <span class="bold"><strong>A.</strong></span> Detection of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=elt-2;class=Gene" target="_blank">elt-2</a></em></span> mRNA (cyan spots) in mixed stage embryos. Scale bar is 5 &#956;m long. (Adapted from <a href="#bib9">Raj <span class="emphasis"><em>et al.</em></span> 2010</a>) <span class="bold"><strong>B.</strong></span> Detection of three Wnt ligands, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=cwn-1;class=Gene" target="_blank">cwn-1</a>, <a href="http://www.wormbase.org/db/get?name=egl-20;class=Gene" target="_blank">egl-20</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-44;class=Gene" target="_blank">lin-44</a>,</em></span> in the posterior body of staged L1 larvae. Scale bar is 10 &#956;m long. (Adapted from <a href="#bib3">Harterink <span class="emphasis"><em>et al.</em></span> 2011</a>).
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-5"></a>2.5.&nbsp;Data analysis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Successful smFISH experiments can simultaneously provide high quality information on the location, abundance, and transcriptional
                  states of multiple mRNA species. To extract this wealth of information in an efficient and unbiased way, we recommend analyzing
                  smFISH data using custom written computer software. While the exact analysis procedure may vary depending on experimental
                  goals, we suggest the following analysis streamline (<a href="#figure3">Figure 3</a>) and briefly outline its implementation:
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

<tr>
  <td align="center"><p><a href="smFISH_fig03.jpg"><img src="smFISH_fig03_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 3.&nbsp;</b>Recommended steps to computationally analyze smFISH data.
                     </p>
                  </div>
               </div>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p><span class="bold"><strong>Image conversion:</strong></span> Common imaging softwares such as MetaMorph and NIH-image save images in .tiff or .stk format. These files are not directly
                           analyzable in computing software such as MATLAB, and must be converted to a readable format.
                        </p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>*Note: A MATLAB code that reads data from .tiff files is available online at MATLAB Central: <a href="http://www.mathworks.com/matlabcentral/fileexchange/10298" target="_top">http://www.mathworks.com/matlabcentral/fileexchange/10298</a></p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Image segmentation:</strong></span> Image segmentation is a useful step in localizing mRNA expression to specific cell or tissue-types. Oftentimes, it is possible
                           to take advantage of DAPI staining (for demarcating the body axes and major tissue groups, i.e., body wall muscles, ventral
                           nerve cord, intestinal cells, etc.), GFP reporter systems, and smFISH signals (by staining for genes previously known to be
                           expressed in a given cell or tissue) to accurately assign smFISH spots to their tissue of origin. Computational software can
                           facilitate this process by overlaying images from multiple channels (e.g., Cy5, Alexa 594, GFP and DAPI), followed by manual
                           or automatic annotation of the relevant spatial landmarks. These annotations can later be overlaid onto the smFISH images
                           to allow tissue-specific transcript quantification. Additional analysis, such as measurement of tissue length or size, can
                           also be performed computationally at this step. If performing segmentation using computational software (e.g., MATLAB), the
                           resulting traces and annotations can be saved as a standalone data file and used for region-specific spot counting (step 5)
                           later.
                        </p>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Image sharpening:</strong></span> To facilitate subsequent automatic spot detection, it is useful to first enhance the signal-to-noise contrast of the smFISH
                           images. This can be done by convolving each image file (i.e., a matrix consisting of individual pixel values) with two mathematical
                           functions (kernels), a Gaussian followed by a Laplacian. In practice the two functions can be convolved first to form a computational
                           filter called the Laplacian of Gaussian (LoG). As shown in <a href="#figure4">Figure 4</a>, the effect of convolving the original image with the LoG filter is twofold: 1) the Gaussian filter smoothens the background
                           to reduce small speckles that could be confused with real spots, 2) the Laplacian filter then performs edge detection by amplifying
                           the contrast between adjacent dim and bright pixels. The width of the Gaussian filter should be picked to resemble the size
                           of a typical spot. Alternatively, this can be done by trial and error (we recommend starting around 1.5) by visually comparing
                           the filtered and original images. The following is an example of MATLAB code used for image sharpening:
                           
                        </p>
                        <div class="literallayout">
                           <p><br>
                              H&nbsp;=&nbsp;-fspecial('log',15,1.2);&nbsp;%&nbsp;H&nbsp;is&nbsp;the&nbsp;LoG&nbsp;filter&nbsp;with&nbsp;a&nbsp;standard&nbsp;deviation&nbsp;of&nbsp;1.2&nbsp;and&nbsp;width&nbsp;around&nbsp;5<br>
                              g&nbsp;=&nbsp;imfilter(im,H);&nbsp;&nbsp;&nbsp;&nbsp;%&nbsp;im&nbsp;is&nbsp;the&nbsp;original&nbsp;smFISH&nbsp;image
                           </p>
                        </div>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Spot detection and semi-automated threshold picking:</strong></span> Since smFISH signals are significantly brighter than the background, individual spots are detected by identifying regions
                           in the image with pixel values higher than a pre-specified threshold. One way of determining this threshold is to systematically
                           screen threshold values in incremental steps from the minimum to the maximum pixel intensities. For each threshold value tested,
                           the software identifies filled regions (or &#8220;connected objects&#8221;) wherein all pixel values are above threshold. The total number
                           of filled regions is then summed up to yield the total spot count corresponding to the given threshold.
                        </p>
                        <p>After spot detection and counting has been carried out for all thresholds, a plot can be generated showing total spot counts
                           as a function of different threshold values. With high quality imaging data this curve is expected to start high and drop
                           rapidly to reach a brief plateau, before it finally approaches zero (<a href="#figure5">Figure 5</a>). The observed plateau corresponds to all threshold values above the background auto-fluorescence and below the real signal
                           intensity. The total spot count is approximately constant in this range and corresponds reliably to the number of actual smFISH
                           spots. Seemingly convenient, automatic detection of this plateau is sometimes difficult (since the size and flatness of this
                           plateau varies from image to image). Especially during initial testing of the software, we recommend to manually pick the
                           threshold, and plot the computationally detected spots over the original image to ensure that no over- or under-counting has
                           occurred.
                        </p>
                     </li>
                     <li>
                        <p><span class="bold"><strong>Region-specific spot counting:</strong></span> Spot quantification within a specific cell or tissue can be conveniently achieved by aligning the results from image segmentation
                           and spot detection. Computationally this can be done by generating a binary map for the region of interest (ROI) and using
                           this map to filter out detected spots that are outside the ROI. MATLAB's Image Processing Toolbox provides many built-in functions
                           that can greatly facilitate the image analysis steps described here. A sample of such MATLAB based software is available at
                           the Arjun Raj lab website <a href="http://rajlab.seas.upenn.edu/pdfs/raj_nat_meth_2008_software.zip" target="_top">http://rajlab.seas.upenn.edu/pdfs/raj_nat_meth_2008_software.zip</a>. Additional details for using MATLAB to perform image processing and single-molecule identification can be found in a recent
                           paper by <a href="#bib12">Scott Rifkin (2011</a>). Other programming platforms, e.g., Python and Image J, also hold great promise in generating fast and accurate analysis
                           software.
                        </p>
                     </li>
                  </ol>
               </div>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

<tr>
  <td align="center"><p><a href="smFISH_fig04.jpg"><img src="smFISH_fig04_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 4.&nbsp;</b>Effects of LoG filtering on smFISH images. <span class="bold"><strong>A.</strong></span> Raw image data staining <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-17;class=Gene" target="_blank">lin-17</a></em></span> mRNA in an L1 larva (shown is a single image from a stack). <span class="bold"><strong>B.</strong></span> The same image after processing by a LoG filter with the above MATLAB commands. Note dramatic reduction in background and
                        enhancement in spot signal.
                     </p>
                  </div>
               </div>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">

<tr>
  <td align="center"><p><a href="smFISH_fig05.jpg"><img src="smFISH_fig05_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 5.&nbsp;</b>Threshold picking and automatic detection of smFISH spots. <span class="bold"><strong>A.</strong></span> Total spot counts as a function of pixel intensity threshold. The correct threshold should be placed where the spot count
                        is insensitive to threshold value (where the &#8220;plateau&#8221; is). <span class="bold"><strong>B.</strong></span> Spot detection when correct threshold (solid vertical line in <span class="bold"><strong>A</strong></span>) is chosen. Red circles indicate computationally identified spots for a single image from a stack. Note out-of-focus spots
                        are not counted. <span class="bold"><strong>C.</strong></span> Spot detection when threshold is chosen too low (dotted line in <span class="bold"><strong>A</strong></span>). Note many out-of-focus spots and background speckles are mistakenly included as spots.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2-6"></a>2.6.&nbsp;Controls and troubleshooting
                        </h3>
                     </div>
                  </div>
               </div>
               <p>As the power of smFISH method lies in the accurate detection of single mRNA molecules, it is important to perform proper controls
                  for each new smFISH probe library generated. Here we outline a number of control experiments that address common concerns
                  regarding smFISH results. Additionally, while the smFISH protocols described here worked robustly for many <span class="emphasis"><em>C. elegans</em></span> genes (including Wnt and Notch pathway genes, endoderm specification genes, etc., in both worms and embryos), we did notice
                  a number of technical factors that can affect data quality. We thus provide a list of troubleshooting tips below to facilitate
                  further optimization of the protocols.
               </p>
               <p><span class="bold"><strong>Recommended controls:</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p><span class="emphasis"><em>Does a single smFISH spot represent a single mRNA molecule?</em></span></p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Solution 1: To confirm single molecule resolution, one may randomly sample the peak pixel intensities of individual smFISH
                                    molecules within a given series of images (a process automatable by computer software). The spot intensities should then be
                                    plotted on a histogram. The resulting distribution is expected to be unimodal with one narrow peak (this would not be the
                                    case if mRNA molecules frequently occur as groups of two or more) (<a href="#bib16">Vargas <span class="emphasis"><em>et al.</em></span> 2005</a>). Large mRNA aggregates in the nucleus or specialized organelles (as in the case of transcription centers, P bodies and stress
                                    granules) are often many-fold brighter and can be easily excluded from analysis by pixel thresholding. To control for the
                                    rare case where mRNA of the same species dimerize (an example being the <span class="emphasis"><em>Drosophila</em></span> <span class="emphasis"><em>bicoid</em></span> mRNA, <a href="#bib17">Wagner <span class="emphasis"><em>et al.</em></span> 2004</a>), one may proceed to perform quantitative reverse-transcription PCR (qRT-PCR) as described below.
                                 </p>
                              </li>
                              <li>
                                 <p>Solution 2: To independently check mRNA level, a control qRT-PCR experiment can be performed on bulk collected <span class="emphasis"><em>C. elegans</em></span> tissue. As it is currently difficult to enrich for particular tissue or cell types in a bulk collection (with the exception
                                    of dissected gonads), this approach would only apply to whole worm or embryo measurement. qRT-PCR measurement from bulk collected
                                    tissue then needs to be normalized by the estimated total number of worms or embryos. The obtained average expression level
                                    per animal can then be compared to average smFISH measurements, and the two are expected to be largely consistent.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Are we robustly detecting all mRNA transcripts of a given species?</em></span> (Positive control)
                        </p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Solution 1: Separate the uncoupled probe library into two halves containing non-overlapping sets of probes. Couple these two
                                    probe sets respectively with two different fluorophores and hybridize both probe sets to the same sample. If the majority
                                    of mRNA transcripts are reliably detected, most spots should appear double labeled, and the overall degree of co-localization
                                    should be high. We have consistently detected around 85% co-localization, suggesting the majority of the mRNA transcripts
                                    are being detected by smFISH. Additionally, it is recommended to partition the probe library in a way that allows probes with
                                    one fluorophore to interleave probes with the other fluorophore. Compared with having the two sets of probes targeting two
                                    different regions of the endogenous mRNA, this approach avoids differences in probe binding efficiency along different parts
                                    of the mRNA. A potential weakness is that this approach is non-robust for mRNAs that are inaccessible by the smFISH protocol
                                    (e.g., mRNAs that are localized in specialized organelles or embodied by protein complexes). The following approach provides
                                    an independent and complementary assessment of total mRNA quantity.
                                 </p>
                              </li>
                              <li>
                                 <p>Solution 2: To determine whether the number of transcripts detected by smFISH is accurate, one may perform, in parallel, quantitative
                                    reverse-transcription PCR (qRT-PCR) on a known number of synchronized worms. The total and average number of transcripts should
                                    agree between the two approaches. However this approach may not work well when transcript number within a specific tissue
                                    or region is of interest. qRT-PCR measurement is also subject to imperfect worm synchronization and may yield average counts
                                    that are lower or higher than the smFISH result (<a href="#bib16">Vargas <span class="emphasis"><em>et al.</em></span> 2005</a>, <a href="#bib11">Raj <span class="emphasis"><em>et al.</em></span> 2008</a>).
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Are we detecting mRNA transcripts other than the species of interest?</em></span> (Negative control)
                        </p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Solution: If possible, obtain a mutant strain with a large deletion in the coding sequence. Design probes that specifically
                                    target the deleted region, and hybridize the same probe set to both the wild-type strain and the mutant. While the probe should
                                    yield detectable signal in wild-type worms, no spots should be detected for the mutant. If a deletion allele is not available
                                    or the deletion is too short, consider using a strain harboring a nonsense mutation. Confirm first by RT-PCR that the mRNA
                                    is greatly reduced compared to the wild-type (likely resulting from nonsense-mediated decay). Then perform smFISH to confirm
                                    that similar reduction in transcripts is observed in the mutant.
                                 </p>
                              </li>
                              <li>
                                 <p>General note: When designing probes, it is good practice to blast the probe library against the <span class="emphasis"><em>C. elegans</em></span> genome to ensure that majority of the targeted sequences are unique (especially important when multiple paralogs exist in
                                    the genome). It is also important to keep the GC content of the probes around below 70% as GC rich probes are more prone to
                                    non-specific binding.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>How do we differentiate smFISH spots from auto-fluorescent speckles?</em></span> (Negative control)
                        </p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Solution 1: Auto-fluorescent speckles tend to show up in multiple fluorescent channels while smFISH spots light up only in
                                    the channel determined by the coupled fluorophore. Overlay images from different fluorescent channels to differentiate auto-fluorescence
                                    from the real signal.
                                 </p>
                              </li>
                              <li>
                                 <p>Solution 2: If auto-fluorescence is a strong concern, one may perform the hybridization procedure without adding the coupled
                                    probes. In the unlikely case that many fluorescent spots show up in the image, we know for sure that there is significant
                                    interfering signal from auto-fluorescence.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Is there bleed-through between channels (in the case of multi-color smFISH)?</em></span></p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>Solution: Bleed through between channels can happen when a signal in one channel is extremely strong. This is the case with
                                    intra-nuclear transcription centers, densely-packed transcripts (sometimes localized in organelles), and highly-over-expressed
                                    transgenes. To check for bleed-through, one may check if the spots detected in different channels (expected to represent two
                                    different transcript species) are highly co-localized. Alternatively, one may perform single-color FISH to see if spots are
                                    detected outside the expected channel. To avoid bleed-through choose channels that are as far apart in emission wavelength
                                    as possible and choose optical filters that separate well between the channels (<a href="#table2" title="Table 2. Examples of optical filter sets compatible with multiplex mRNA detection">Table 2</a>).
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                  </ol>
               </div>
               <p><span class="bold"><strong>Troubleshooting tips:</strong></span></p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p><span class="emphasis"><em>Low signal intensity or no signal</em></span></p>
                        <p>Whenever possible, use a previously tested probe set (coupled to the same fluorophore) as positive control to make sure all
                           equipment (light source, camera, software, etc.) are working properly. For issues specific to the particular probe set, consider
                           checking the following:
                        </p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>1) <span class="emphasis"><em>Probe concentration</em></span>. Test higher concentrations (e.g., start with one order of magnitude higher) to see if the signal quality improves. Based
                                    on experience, we recommend diluting in TE 1:20 for Cy5 coupled probes, 1:50 for Alexa, and 1:10 for TMR, followed by 1:100
                                    dilution in hybridization buffer.
                                 </p>
                              </li>
                              <li>
                                 <p>2) <span class="emphasis"><em>Tissue penetration</em></span>. Make sure the sample has been incubated in EtOH for ample time (e.g., 24 hours).
                                 </p>
                              </li>
                              <li>
                                 <p>3) <span class="emphasis"><em>Hybridization stringency</em></span>. Formamide concentration of the hybridization buffer directly controls binding efficiency. While 10% formamide generally
                                    works well for all <span class="emphasis"><em>C. elegans</em></span> stages, one may try decreasing the concentration to see if signal improves. One should be cautious with this approach as
                                    low formamide concentration also ups the chance of non-specific binding.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>High fluorescent background</em></span></p>
                        <p>Multiple causes are likely:</p>
                        <div class="itemizedlist">
                           <ul type="none">
                              <li>
                                 <p>1) <span class="emphasis"><em>Unhealthy worms</em></span>. Starvation and over-crowded culture lead to high auto-fluorescence in the worm tissue. This is the most common cause of
                                    high background and should be avoided by proper worm culture maintenance.
                                 </p>
                              </li>
                              <li>
                                 <p>2) <span class="emphasis"><em>Over-fixation</em></span>. The second common cause and especially problematic for older worms. Keep fixation time under an hour and adjust fixation
                                    duration to troubleshoot. Also make sure to thoroughly wash off the fixative with 1x PBS to terminate fixation.
                                 </p>
                              </li>
                              <li>
                                 <p>3) <span class="emphasis"><em>High probe concentration or insufficient washing</em></span>. This can be checked by testing lower dilutions and incubating longer during the two wash steps.
                                 </p>
                              </li>
                              <li>
                                 <p>4) <span class="emphasis"><em>Low probe binding stringency</em></span>. This could be a result of either low formamide concentration or high probe GC content. In both cases, probes bind non-specifically
                                    leading to increased background fluorescence. Unlike tissue auto-fluorescence, this phenomenon should be specific to the channel
                                    of the fluorophore whereas auto-fluorescence affects all channels.
                                 </p>
                              </li>
                              <li>
                                 <p>5) <span class="emphasis"><em>Out-of-focus light</em></span>. Real FISH signals that are outside the focal plane appear as diffuse background fluorescence in the focal plane. This problem
                                    is often dramatically improved by sucking away any excess fluid in the imaging chamber and maximally flattening the sample.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Nonspecific signal (due to auto-fluorescence or bleed-through)</em></span></p>
                        <p>First make sure that the sample is collected from a clean, healthy batch of worms to reduce overall background. Sometimes,
                           signal in the Alexa 594 channel can bleed into the Cy5 channel if the two optical filters are not optimally separated in spectrum.
                           Extremely high signal intensity (often in the case of over-expressed transgenes) in one channel can also broadly show up as
                           &#8220;pseudo-spots&#8221; in multiple other channels. To check, test each probe set in a separate hybridization to see if problem persists.
                        </p>
                     </li>
                     <li>
                        <p><span class="emphasis"><em>Irregular or diffuse spot morphology interfering with spot identification and counting</em></span></p>
                        <p>Successful smFISH experiment should yield spots circular in 2D and with sizes similar to one another. However liquid between
                           the sample and the objective can obscure the expected optical properties of the spots. In such case, try to maximally flatten
                           the sample to reduce sample thickness and out-of-focus light. Additionally, highly local spot density (which occurs with highly
                           expressed genes or transgenes) can lead to inevitable difficulty in resolving individual spots. When processing through computational
                           software, try to identify local maxima in addition to &#8220;connected regions&#8221;. This allows sub-diffraction detection of the (in
                           principal) exact location of the mRNA transcript and thereby enhances the resolution of the smFISH signal.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed"><span class="bibliomisc">Femino A.M., Fay F.S., Fogarty K., and Singer R.H. (1998). Visualization of single RNA transcripts <span class="emphasis"><em>in situ</em></span>. Science <span class="emphasis"><em>280</em></span>, 585-590.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9554849&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.280.5363.585" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed"><span class="bibliomisc">Fukushige T., Hawkins M.G., and McGhee J.D. (1998). The GATA-factor <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=elt-2;class=Gene" target="_blank">elt-2</a></em></span> is essential for formation of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> intestine. Dev. Biol. <span class="emphasis"><em>198</em></span>, 286&#8211;302.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9659934&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0012-1606(98)80006-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed"><span class="bibliomisc">Harterink M., Kim D., Middelkoop T.C., Doan T.D., van Oudenaarden A., and Korswagen H.C. (2011). Neuroblast migration along
                        the anteroposterior axis of <span class="emphasis"><em>C. elegans</em></span> is controlled by opposing gradients of Wnts and a secreted Frizzled-related protein. Development. <span class="emphasis"><em>138</em></span>, 2915-2924.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21653614&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.064733" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed"><span class="bibliomisc">Korzelius J., The I., Ruijtenberg S., Prinsen M.B., Portegijs V., Middelkoop T.C., Groot Koerkamp M.J., Holstege F.C., Boxem
                        M., and van den Heuvel S. (2011). <span class="emphasis"><em>Caenorhabditis elegans</em></span> cyclin D/CDK4 and cyclin E/CDK2 induce distinct cell cycle re-entry programs in differentiated muscle cells. PLoS Genet.
                        <span class="emphasis"><em>7</em></span>, e1002362.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22102824&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002362" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed"><span class="bibliomisc">Levsky J.M., and Singer R.H. (2003). Fluorescence <span class="emphasis"><em>in situ</em></span> hybridization: past, present and future. J. Cell Sci. <span class="emphasis"><em>116,</em></span> 2833&#8211;2838.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12808017&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/jcs.00633" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed"><span class="bibliomisc">Maloof, J.N., Whangbo J., Harris J.M., Jongeward G.D., and Kenyon C. (1999). A Wnt signaling pathway controls Hox gene expression
                        and neuroblast migration in <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>126</em></span>, 37-49.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9834184&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed"><span class="bibliomisc">Middelkoop T.C., Williams L., Yang P.T., Betist M.C., Ji N., van Oudenaarden A., Kenyon C., and Korswagen H.C. (2012). The
                        thrombospondin repeat containing protein <a href="http://www.wormbase.org/db/get?name=MIG-21;class=Protein" target="_blank">MIG-21</a> controls a left-right asymmetric Wnt signaling response in migrating <span class="emphasis"><em>C. elegans</em></span> neuroblasts. Dev. Biol. <span class="emphasis"><em>361</em></span>, 338-348.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22074987&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2011.10.029" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed"><span class="bibliomisc">Raap A.K., van de Corput M.P., Vervenne R.A., van Gijlswijk R.P., Tanke H.J., and Wiegant J. (1995). Ultra-sensitive FISH
                        using peroxidase-mediated deposition of biotin- or fluorochrome tyramides. Hum. Mol. Genet. <span class="emphasis"><em>4</em></span>, 529-534.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7633400&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/hmg/4.4.529" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed"><span class="bibliomisc">Raj A., Rifkin S.A., Anderson E., and van Oudenaarden A. (2010). Variability in gene expression underlies incomplete penetrance.
                        Nature <span class="emphasis"><em>463</em></span>, 913-918.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20164922&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature08781" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed"><span class="bibliomisc">Raj A. and Tyagi S. (2010). Detection of individual endogenous RNA transcripts <span class="emphasis"><em>in situ</em></span> using multiple singly labeled probes. Methods Enzymol. <span class="emphasis"><em>472</em></span>, 365-386.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20580972&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0076-6879(10)72004-8" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed"><span class="bibliomisc">Raj A., van den Bogaard P., Rifkin SA, van Oudenaarden A., and Tyagi S. (2008). Imaging individual mRNA molecules using multiple
                        singly labeled probes. Nat. Methods. <span class="emphasis"><em>5</em></span>, 877&#8211;879.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18806792&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1253" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed"><span class="bibliomisc">Rifkin S.A. (2011). Identifying fluorescently labeled single molecules in image stacks using machine learning. Molecular Methods
                        for Evolutionary Genetics. (Orgogozo &amp; Rockman, eds.). Methods Mol. Biol. <span class="emphasis"><em>772</em></span>, 329-348.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22065448&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/978-1-61779-228-1_20" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed"><span class="bibliomisc">Saffer A.M., Kim D., van Oudenaarden A., and Horvitz H.R. (2011). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> synthetic multivulva genes prevent Ras pathway activation by tightly repressing global ectopic expression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-3;class=Gene" target="_blank">lin-3</a></em></span> EGF. PLoS Genet. <span class="emphasis"><em>7</em></span>, e1002418.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22242000&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002418" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed"><span class="bibliomisc">Seidel H.S., Ailion M., Li J., van Oudenaarden A., Rockman, M.V., and Kruglyak L. (2011). A novel sperm-delivered toxin causes
                        late-stage embryo lethality and transmission ratio distortion in <span class="emphasis"><em>C.</em></span> <span class="emphasis"><em>elegans</em></span>. PLoS Biol. <span class="emphasis"><em>9</em></span>, e1001115.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21814493&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.1001115" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed"><span class="bibliomisc">Topalidou I., van Oudenaarden A.,and Chalfie M. (2011). <span class="emphasis"><em>Caenorhabditis elegans</em></span> aristaless/Arx gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=alr-1;class=Gene" target="_blank">alr-1</a></em></span> restricts variable gene expression. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>108</em></span>, 4063-4068.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21368126&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.1101329108" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed"><span class="bibliomisc">Vargas D.Y. Raj A., Marras S.A., Kramer F.R. and Tyagi S. (2005). Mechanism of mRNA transport in the nucleus. Proc. Natl.
                        Acad. Sci. U. S. A. <span class="emphasis"><em>102</em></span>, 17008-17013.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16284251&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0505580102" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed"><span class="bibliomisc">Wagner C., Ehresmann C., Ehresmann B., and Brunel C. (2004). Mechanism of dimerization of Bicoid mRNA: Initiation and stabilization.
                        J. Biol. Chem. <span class="emphasis"><em>279,</em></span> 4560-4569.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14607826&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1074/jbc.M306511200" target="_blank">Article</a>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e12" href="#d0e12">*</a></sup>Edited by Oliver Hobert. Last revised May 24, 2012. Published December 13, 2012. This chapter should be cited as: Ji N. and
                  van Oudenaarden A. Single molecule fluorescent <span class="emphasis"><em>in situ</em></span> hybridization (smFISH) of <span class="emphasis"><em>C. elegans</em></span> worms and embryos (December 13, 2012), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.153.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2012 Ni Ji and Alexander van Oudenaarden. This is an open-access article distributed under the terms of the Creative Commons
                  Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author
                  and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e44" href="#d0e44">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="a.vanoudenaarden@hubrecht.eu" target="_top">a.vanoudenaarden@hubrecht.eu.</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --> </body>
</html>