<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Caenorhabditis briggsae methods
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#Cbrigmeth_ArticleID" title="Caenorhabditis briggsae methods&#xA;">
      <link rel="next" href="#d0e62" title="1.&nbsp;The Caenorhabditis briggsae model system">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="Cbriggsaemethods.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="Cbrigmeth_ArticleID"></a><span class="emphasis"><em>Caenorhabditis briggsae</em></span> methods<sup><a name="d0e10" href="#ftn.d0e10">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Scott E.</span> <span class="surname">Baird<em><span class="remark"><sup>&sect;</sup></span></em></span><br></span><span class="affiliation"><span class="address">Department of Biological Sciences, Wright State University, Dayton, OH 45435-0001 USA
                           <p></p></span></span></div>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Helen M.</span> <span class="surname">Chamberlin<em><span class="remark"><sup><a name="d0e49" href="#ftn.d0e49">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="address">Department of Molecular Genetics, The Ohio State University, Columbus, OH 43210-1292 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Methods in nematode species other than <em>C. elegans</em>
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=310.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Caenorhabditis+briggsae+methods" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e62">1. The <span class="emphasis"><em>Caenorhabditis briggsae</em></span> model system</a></span></dt>
               <dt><span class="sect1"><a href="#d0e144">2. Natural history and population biology</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e147">2.1. Natural history</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e170">2.2. Biogeography and population biology</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e247">3. Genetics</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e250">3.1. Genetic nomenclature</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e346">3.2. Single nucleotide polymorphisms</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e363">3.3. Recombination maps</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e407">3.4. Bulk segregant analysis</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e484">4. Reverse genetic methods</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e487">4.1. Identification of <span class="emphasis"><em>C. elegans&#8211;C. briggsae</em></span> orthologs</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e538">4.2. Transgenesis and heterologous gene expression</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e628">4.3. RNA-mediated interference</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e697">4.4. Targeted deletion screens</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e726">4.5. <span class="emphasis"><em>Mos</em></span> mutagenesis</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e795">5. Online resources for <span class="emphasis"><em>C. briggsae</em></span> genetics and genomics</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e809">5.1. The <span class="emphasis"><em>Caenorhabditis briggsae</em></span> Research Page</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e829">5.2. The <span class="emphasis"><em>C. briggsae</em></span> SNP map</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e843">5.3. WormBase</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e868">5.4. The WashU <span class="emphasis"><em>C. briggsae</em></span> Genome server</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e885">5.5. The <span class="emphasis"><em>Caenorhabditis</em></span> Genetics Center</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e899">6. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e62"></a>1.&nbsp;The <span class="emphasis"><em>Caenorhabditis briggsae</em></span> model system
                     </h2>
                  </div>
               </div>
            </div>
            <p><span class="emphasis"><em>Caenorhabditis briggsae</em></span> is being developed in parallel to <span class="emphasis"><em>C. elegans</em></span> as a model system, primarily for the study of evolution. Like <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em>C. briggsae</em></span> is a protandrous hermaphrodite (<a href="#bib24">Nigon and Dougherty, 1949</a>) and like <span class="emphasis"><em>C. elegans</em></span>, its genome has been sequenced (<a href="#bib26">Stein et al., 2003</a>). From this point, these two model systems diverge. The development, behavior, and physiology of <span class="emphasis"><em>C. elegans</em></span> have been characterized through tens of thousands of genetic and molecular studies. Genetic and molecular characterizations
               of <span class="emphasis"><em>C. briggsae</em></span> are relatively few. Experimental resources in <span class="emphasis"><em>C. elegans</em></span> include a high density recombination map that is well integrated with the genome sequence. The <span class="emphasis"><em>C. briggsae</em></span> recombination map has yet to be published and attempts to integrate it with the genome sequence are in their infancy. Despite
               these deficiencies, <span class="emphasis"><em>C. briggsae</em></span> is attractive for several reasons. First as a parallel system, it can be used to test the generality of results obtained
               in <span class="emphasis"><em>C. elegans</em></span>. Second, it appears that the structure of the <span class="emphasis"><em>C. briggsae</em></span> world-wide population is qualitatively different from that of <span class="emphasis"><em>C. elegans</em></span> and that <span class="emphasis"><em>C. briggsae</em></span> may be more amenable to studies of gene flow, genome evolution, and speciation (<a href="#bib7">Cutter et al., 2006</a>; Baird and Hampton, unpubl. results). Finally, <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span> are sister species with <span class="emphasis"><em>C. elegans</em></span> as an outgroup, making the <span class="emphasis"><em>C. briggsae&#8211;C. remanei</em></span> species better for some aspects of comparative genomics (<a href="#bib20">Kiontke et al., 2004</a>; <a href="#bib6">Cho et al., 2004</a>; see <a href="http://www.wormbook.org/chapters/www_phylogrhabditids/phylorhab.html">The phylogenetic relationships of <span class="emphasis"><em>Caenorhabditis</em></span> and other rhabditids</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e144"></a>2.&nbsp;Natural history and population biology
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e147"></a>2.1.&nbsp;Natural history
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Most <span class="emphasis"><em>Caenorhabditis</em></span> species form phoretic and/or necromenic associations with other soil invertebrates (see <a href="http://www.wormbook.org/chapters/www_ecolCaenorhabditis/ecolCaenorhabditis.html">Ecology of <span class="emphasis"><em>Caenorhabditis</em></span> species</a>). In phoretic associations, hosts are used for transport between microenvironments. In necromenic associations, nematodes
                  resume development and reproduce in the bacterial bloom that occurs following the host death. A variety of snails have been
                  identified as hosts for <span class="emphasis"><em>C. briggsae</em></span> dauer larvae (S. Baird, unpubl. obs.). <span class="emphasis"><em>C. briggsae</em></span> dauers also have been obtained from soil and compost (<a href="#bib7">Cutter et al., 2006</a>). As dauers can persist for several months without feeding, generation times in wild populations are not known and may vary
                  widely between and within populations.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e170"></a>2.2.&nbsp;Biogeography and population biology
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. briggsae</em></span> is a cosmopolitan species, having been obtained from collections in Asia, North America, and Europe (<a href="#figure1">Figure 1A</a>; <a href="#bib7">Cutter et al., 2006</a>). Phylogentic reconstructions of strains derived from these collections strongly support the division of <span class="emphasis"><em>C. briggsae</em></span> into distinct tropical and temperate clades (<a href="#figure1">Figure 1B</a>; <a href="#bib11">Graustein et al., 2002</a>; <a href="#bib7">Cutter et al., 2006</a>; Baird and Hampton, unpubl. results). This clade structure should be considered when designing experiments, as tropical and
                  temperate strains exhibit morphological and developmental differences. Consequently, different experimental results sometimes
                  are obtained for tropical and temparate strains (<a href="#bib1">Baird, 2001</a>; <a href="#bib2">2002</a>; <a href="#bib9">Dellatre and F&eacute;lix, 2001</a>). Few collections have been made in the Southern hemisphere. However, <span class="emphasis"><em>C. briggsae</em></span> strains have been obtained from Kenya and South Africa (E. Dolgin, personal communication).
               </p>
               <p>The division of <span class="emphasis"><em>C. briggsae</em></span> into tropical and temperate clades probably results from a recent expansion of the <span class="emphasis"><em>C. briggsae</em></span> range into temperate latitudes. Nucleotide diversity within the temperate clade is very low and the coalescence time for
                  this clade has been estimated to be 700 years (<a href="#bib7">Cutter et al., 2006</a>). This estimate was based on an effective population size (N<sub>e</sub>) for the temaperate clade of 1,000 and a 60 day generation time (assuming that in wild populations, <span class="emphasis"><em>C. briggsae</em></span> individuals spend most of their time as dauers).
               </p>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="585">
<tr>
  <td align="center"><p><a href="Cbrigmethfig1.jpg"><img src="Cbrigmethfig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. Biogeography of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>.&nbsp;</b>a) A map of tropical and temparate collection sites from which <span class="emphasis"><em>C. briggsae</em></span> strains have been obtained. b) An unrooted reconstruction of <span class="emphasis"><em>C. briggsae</em></span> phylogeny based on sequence comparisons at six loci. Adapted from <a href="#bib7">Cutter et al. (2006)</a>.
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e247"></a>3.&nbsp;Genetics
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e250"></a>3.1.&nbsp;Genetic nomenclature
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Nomenclatural rules for <span class="emphasis"><em>C. briggsae</em></span>, which currently are in flux, reflect the conflicting goals of providing unique durable gene names for mutationally identified
                  genes while at the same time allowing for the identification of orthologs to <span class="emphasis"><em>C. elegans</em></span> genes. The history of <span class="emphasis"><em>C. briggsae</em></span> nomenclature rules and detailed nomenclatural suggestions can be found at the <a href="http://wormlab.caltech.edu/briggsae/nomenclature/message28-20031016.html" target="_blank"><span class="emphasis"><em>Caenorhabditis briggsae</em></span> Research Page</a>. One feature of this system is that <span class="emphasis"><em>C. briggsae</em></span> gene names may change as orthology is determined. For example, the <span class="emphasis"><em>C. briggsae</em></span> <span class="emphasis"><em>mip-1</em></span> gene was identified based on mutations that cause mutant animals to twitch uncontrollably. In <span class="emphasis"><em>C. elegans</em></span>, this phenotype results commonly from mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-22;class=Gene" target="_blank">unc-22</a></em></span>. Hence, <span class="emphasis"><em>mip-1</em></span> may be the <span class="emphasis"><em>C. briggsae</em></span> ortholog of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-22;class=Gene" target="_blank">unc-22</a></em></span>. If this orthology is confirmed, <span class="emphasis"><em>mip-1</em></span> will be renamed <span class="emphasis"><em>Cb-unc-22</em></span>. Confusion caused by changes in <span class="emphasis"><em>C. briggsae</em></span> gene names is expected to be minimal as gene name histories will be archived in <a href="http://www.wormbase.org/" target="_blank">WormBase</a>.
               </p>
               <p>Alleles, strains, polymorphisms, rearrangements, transgenes, and other variants will be given unique identifiers following
                  <span class="emphasis"><em>C. elegans</em></span> nomenclatural conventions (<a href="#bib14">Horvitz et al., 1979</a>; <a href="http://biosci.umn.edu/CGC/Nomenclature/nomenguid.htm" target="_blank">CGC</a>). These identifiers will include a lab strain or allele designation followed by a unique strain or allele number for that
                  lab. For example, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=bd101;class=Variation" target="_blank">bd101</a></em></span> is a <span class="emphasis"><em>C. briggsae</em></span> mutation that is present in several strains including <a href="http://www.wormbase.org/db/get?name=PB101;class=Strain" target="_blank">PB101</a>, <a href="http://www.wormbase.org/db/get?name=PB107;class=Strain" target="_blank">PB107</a> and <a href="http://www.wormbase.org/db/get?name=PB108;class=Strain" target="_blank">PB108</a>. These names will be unique and durable.
               </p>
               <p>Finally, it should be noted that a different convention is used by <a href="http://www.wormbase.org/" target="_blank">WormBase</a> to name predicted <span class="emphasis"><em>C. briggsae</em></span> genes. In this convention, each gene prediction gets a CBG (<span class="emphasis"><em>C. briggsae</em></span> gene) prefix followed by a unique five digit identifier, e.g., CBG00123. Gene predictions in WormBase are continually refined
                  as algorithms are improved. Therefore, predicted CBG gene names also may change over time.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e346"></a>3.2.&nbsp;Single nucleotide polymorphisms
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The genome sequence of <span class="emphasis"><em>C. briggsae</em></span> is that of the tropical strain, <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> (<a href="#bib26">Stein et al., 2003</a>). To facilitate genetic studies, 13,000 randomly selected clones derived from a temperate strain, <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a>, also have been sequenced. From these sequences over 26,000 single nucleotide polymorphisms (SNPs) have been identified giving
                  a SNP density of greater than 0.2% (Hillier, Miller, Chinwalla, Fulton, Kolboldt, and Waterston, personal communication).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e363"></a>3.3.&nbsp;Recombination maps
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Two recombination maps have been constructed for <span class="emphasis"><em>C. briggsae</em></span>. One is a <a href="http://snp.wustl.edu/snp-research/c-briggsae/" target="_blank">SNP-based map</a> that was constructed based on the segregation of SNP alleles in recombinant inbred lines (RIL) (Hillier, Miller, Chinwalla,
                  Fulton, Kolboldt, and Waterston, personal communication). These RIL were derived from a cross between <span class="emphasis"><em>C. briggsae</em></span> strains <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> and <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a> (<a href="#bib3">Baird et al., 2005</a>). Each RIL was initiated with a single L4 F2 hermaphrodite and inbred through a single hermaphrodite per generation to F11.
                  The current draft SNP map (v3.3) includes segregation data for 290 SNPs in 93 RIL. RIL genotypes for each SNP were determined
                  using a fluorescent polarization-terminal dye incorporation (FP-TDI) assay (<a href="#bib15">Hsu et al., 2001</a>). This assay does not require any post-reaction product purification and has been adapted for the high-throughput genotyping
                  of <span class="emphasis"><em>C. briggsae</em></span> RIL. The other is a <a href="http://wormlab.caltech.edu/briggsae/" target="_blank">mutation-based map</a> that was constructed based on the segregation of visible phenotypes in F2 progeny (B. Gupta, P. Sternberg, and D. Baillie,
                  personal communication). Both maps identified six linkage groups. Integration of the SNP- and mutation-based maps is in progress
                  (R. Miller and B. Gupta, personal communication). Nearly all <span class="emphasis"><em>C. briggsae</em></span> induced mutations were derived from strain <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a>. Therefore, map integration will be accomplished by mapping mutant alleles relative to <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a> SNPs. This will entail the genotyping of individual mutant F2s for multiple SNPs. To facilitate this, template DNA from F2
                  single-worm lysates will be amplified by whole genome amplification (<a href="#bib8">Dean et al., 2001</a>). Preliminary experiments have demonstrated the whole genome amplification provides template sufficient for several thousand
                  genotyping reactions (S. Baird, unpubl results; B. Gupta and R. Miller, personal communication).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e407"></a>3.4.&nbsp;Bulk segregant analysis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In <span class="emphasis"><em>C. briggsae</em></span>, as in other organisms, most SNPs do not affect restriction endonuclease recognition sites. Therefore it is difficult and/or
                  expensive to distinguish between <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> and <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a> SNP alleles with using commonly available lab equipment. One way around this, at least for linkage determination, is bulk
                  segregant analysis. In <span class="emphasis"><em>C. briggsae</em></span> a bulk segregant assay is initiated by crossing an AF16-derived mutant strain to <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a>. Pools of 10&#8211;25 mutant F2 progeny then are collected and lysed following a scaled-up (25 &#956;l) <a href="http://www.wormbook.org/chapters/www_SNPsintrotwopointmap/SNPsintrotwopointmap.html">single-worm protocol</a>. Primers for these reactions are designed to amplify short (200&#8211;500 bp) products that contain a polymorphic nucleotide at
                  least 50 nucleotides from either end. Amplification products are sequenced and polymorphisms are detected as ambiguities in
                  the sequence traces. For unlinked loci, both alleles, i.e. an ambiguity, will be apparent in the sequence trace. For linked
                  loci, only the <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> allele will be present. In control reactions, a strong allelic bias was observed from lysates made from a 9:1 ratio of <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a> to <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> hermaphrodites (<a href="#figure2">Figure 2</a>). In F2 progeny, a 9:1 ratio would be expected for loci located 10 mu from the selected mutation. Thus, two-fold redundant
                  coverage of the <span class="emphasis"><em>C. briggsae</em></span> genome can be achieved with as few as four SNPs per chromosome. This approach has been used successfully to identify loci
                  involved in fitness loss in <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a>::HK104 F2 hybrids (Hampton and Baird, unpubl. results).
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="Cbrigmethfig2.jpg"><img src="Cbrigmethfig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Bulk segregant analysis of <span class="emphasis"><em>C. briggsae</em></span> SNPs.&nbsp;</b>Sequence traces from C. <span class="emphasis"><em>briggsae</em></span> strains a) <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> and b) <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a>, showing an A/T polymorphisms. Sequence traces from lysates made from c) a mixture of five <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> and five <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a> hermaphrodites and d) one <a href="http://www.wormbase.org/db/get?name=AF16;class=Strain" target="_blank">AF16</a> hermaphrodite and nine <a href="http://www.wormbase.org/db/get?name=HK104;class=Strain" target="_blank">HK104</a> hermaphrodites.
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e484"></a>4.&nbsp;Reverse genetic methods
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e487"></a>4.1.&nbsp;Identification of <span class="emphasis"><em>C. elegans&#8211;C. briggsae</em></span> orthologs
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> share a similar number of protein-coding genes (just under 20,000), with an estimate of 12,200 orthologous genes. Another
                  6,500 genes in <span class="emphasis"><em>C. briggsae</em></span> have one or more <span class="emphasis"><em>C. elegans</em></span> homolog, leaving about 800 genes as <span class="emphasis"><em>C. briggsae</em></span>-specific (<a href="#bib26">Stein et al., 2003</a>). <span class="emphasis"><em>C. elegans</em></span>&#8211;<span class="emphasis"><em>C. briggsae</em></span> orthologs are annotated in Wormbase (<a href="http://www.wormbase.org/" target="_blank">http://www.wormbase.org/</a>) according to the criteria of <a href="#bib26">Stein et al. (2003)</a>. Each ortholog was identified either as the reciprocal top BLASTP match when queried against the genome of the other species,
                  or was identified as a homolog (but not the reciprocal best match) with conserved synteny (gene order) with respect to other
                  genes on the chromosome.
               </p>
               <p>Although ortholog predictions by Stein et al. are fairly robust, false positives can result if best matches are not orthologous
                  due to gene loss or divergence, or gene amplification events. Reciprocally, false negatives can result in the cases of rapid
                  gene evolution or from errors in genome annotation. Consequently, researchers interested in a specific gene or set of genes
                  will want to perform additional analysis to confirm orthology. Commonly, this can be achieved by phylogenetic reconstruction
                  using all sequences of homologs available from different <span class="emphasis"><em>Caenorhabditis</em></span> species, as well as an outgroup species. Homologs can be identified using BLAST to identify conserved domains as well as
                  other genes (or translated gene products) that contain the domains. The relationship among the genes/proteins can be determined
                  using a neighbor joining program with confidence assessed by bootstrapping methods (e.g., PHYLIP). True orthologs should exhibit
                  fewer sequence differences and thus join, or cluster, with each other (e.g., <a href="#bib17">Jovelin and Phillips, 2005</a>; <a href="#bib23">Nayak et al., 2005</a>; <a href="#bib29">Thomas, 2006</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e538"></a>4.2.&nbsp;Transgenesis and heterologous gene expression
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Introduction of transgenes into <span class="emphasis"><em>C. briggsae</em></span> allows for mutant rescue experiments, reporter transgene assays, and heterologous gene expression, both for ectopic expression
                  studies, and inter-specific functional assays (e.g., <a href="#bib28">Streit et al., 1999</a>; <a href="#bib30">Wang and Chamberlin, 2002</a>; <a href="#bib31">Wang and Chamberlin, 2004</a>). High copy number extrachromosomal transgenes can be introduced into <span class="emphasis"><em>C. briggsae</em></span> by microinjection of DNA into the hermaphrodite germline (see <a href="http://www.wormbook.org/chapters/www_transformationmicroinjection/transformationmicroinjection.html">Transformation and microinjection</a>) following protocols developed for <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib22">Mello et al., 1991</a>). Transformation markers from <span class="emphasis"><em>C. elegans</em></span> that are effective in <span class="emphasis"><em>C. briggsae</em></span> include <span class="emphasis"><em>ce-rol-6(d)</em></span>, <span class="emphasis"><em>ce-myo-2::gfp</em></span>, <span class="emphasis"><em>ce-myo-3::gfp</em></span>, <span class="emphasis"><em>ce-hsp-16-48::gfp</em></span>, and <span class="emphasis"><em>ce-sur-5::gfp</em></span> (<a href="#bib28">Streit et al., 1999</a>; <a href="#bib30">Wang and Chamberlin, 2002</a>) M.-A. Felix, personal communication). Extrachromosomal arrays can be integrated (see <a href="http://www.wormbook.org/chapters/www_transformationmicroinjection/transformationmicroinjection.html">Transformation and microinjection</a>) into the chromosome using 3000&#8211;4000 rad <span class="symbol">&#947;</span>-irradiation and following <span class="emphasis"><em>C. elegans</em></span> protocols (<a href="#bib16">Inoue et al., 2002</a>).
               </p>
               <p><span class="emphasis"><em>C. briggsae</em></span> exhibits differences from <span class="emphasis"><em>C. elegans</em></span> with respect to the efficacy of transgene formation, and they exhibit greater transgene silencing and mosaicism. In general,
                  similar DNA mixtures and injection conditions result in 3-10 fold fewer stable transgenes in <span class="emphasis"><em>C. briggsae</em></span> than in <span class="emphasis"><em>C. elegans</em></span>, with reduced efficiency at both F1 and F2 generations (M.-A. Felix, personal communication). Although some of these differences
                  may reflect the use of heterologous [usually <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib21">Li et al., 2006</a>)] DNA in transformation experiments, these results suggest that producing transgenes in <span class="emphasis"><em>C. briggsae</em></span> will benefit from further optimization of protocols.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e628"></a>4.3.&nbsp;RNA-mediated interference
                        </h3>
                     </div>
                  </div>
               </div>
               <p>As in <span class="emphasis"><em>C. elegans</em></span>, double stranded RNA-mediated gene interference (RNAi; see <a href="http://www.wormbook.org/chapters/www_introreversegenetics/introreversegenetics.html">Reverse genetics</a>) can provide a relatively easy method to assess the function of <span class="emphasis"><em>C. briggsae</em></span> genes <span class="emphasis"><em>in vivo</em></span> (e.g., <a href="#bib12">Haag et al., 2002</a>; <a href="#bib25">Rudel and Kimble, 2001</a>; <a href="#bib27">Stothard et al., 2002</a>; <a href="#bib28">Streit et al., 1999</a>). For this method, introduction of double-stranded RNA corresponding to a gene of interest results in the specific degradation
                  of the corresponding mRNA, allowing phenotypic analysis of animals depleted for the gene. However, in contrast to <span class="emphasis"><em>C. elegans</em></span>, an effective RNAi response is generally obtained in <span class="emphasis"><em>C. briggsae</em></span> only when double stranded RNA is introduced directly into animals, such as by microinjection. <span class="emphasis"><em>C. briggsae</em></span> is resistant to RNAi introduced by soaking or bacterial feeding methods (M. K. Montgomery, personal communication; C. Hunter,
                  personal communication). This resistance reflects a difference between <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span> animals in availability of RNAi uptake and spreading mechanisms that allow RNAi transport into the animal and across epithelial
                  boundaries. Consequently, RNAi can frequently be less effective in <span class="emphasis"><em>C. briggsae</em></span> than <span class="emphasis"><em>C. elegans</em></span>, and negative results or differences in RNAi phenotypes between the species must be interpreted with caution. Recent work
                  has shown that the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sid-2;class=Gene" target="_blank">sid-2</a></em></span> gene required for environmental uptake of RNA by <span class="emphasis"><em>C. elegans</em></span> is altered and not functional in <span class="emphasis"><em>C. briggsae</em></span>. Consequently, introduction of <span class="emphasis"><em>ce-sid-2</em></span> as part of a transgene enhances RNAi uptake in <span class="emphasis"><em>C. briggsae</em></span>, and confers sensitivity to RNAi by feeding onto this species (W. M. Winston, M. Sutherlin, A. J. Wright, E. Feinberg, and
                  C. Hunter, unpublished results). We anticipate that these transgenic animals or other genetically modified or mutant <span class="emphasis"><em>C. briggsae</em></span> strains will allow for enhanced RNAi response and a greater range of methods to introduce RNAi.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e697"></a>4.4.&nbsp;Targeted deletion screens
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Although functional studies using RNAi are efficient, the gold standard in genetic analysis is chromosomal mutations. Strategies
                  developed to isolate deletion alleles (see <a href="http://www.wormbook.org/chapters/www_introreversegenetics/introreversegenetics.html">Reverse genetics</a>) in <span class="emphasis"><em>C. elegans</em></span> genes have been applied effectively to <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib10">Edgley et al., 2002</a>), although the frequency of allele recovery may be somewhat lower than in <span class="emphasis"><em>C. elegans</em></span>. <a href="#bib13">Hill et al. (2006)</a> report a single deletion allele each of <span class="emphasis"><em>Cb-fem-2</em></span> and <span class="emphasis"><em>Cb-fem-3</em></span> recovered from a library of 1.1 million EMS mutagenized gametes.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e726"></a>4.5.&nbsp;<span class="emphasis"><em>Mos</em></span> mutagenesis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The Drosophila transposon <span class="emphasis"><em>Mos1</em></span> has been shown to mobilize and cause heritable mutations in <span class="emphasis"><em>C. elegans</em></span>, providing a method for rapid tagging and recovery of mutant genes (<a href="#bib32">Williams et al., 2005</a>). Following a modified protocol (below), M.-A. Felix and her group have shown that <span class="emphasis"><em>Mos1</em></span> mutagenesis can be applied to <span class="emphasis"><em>C. briggsae</em></span> as well. Although the SNP-based polymorphism map (v3.3) combined with genomic sequence will provide the tools for positional
                  cloning of genes in <span class="emphasis"><em>C. briggsae</em></span>, transposon tagging may allow a more efficient strategy for gene cloning in some cases.
               </p>
               <p>In <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em>Mos1</em></span> mutagenesis is achieved using two transgenes: an extrachromosomal array that expresses the transposase under control of a
                  heat-shock promoter, and a second transgene that includes the transposon (<a href="#bib4">Bessereau et al., 2001</a>). Expression of the transposase is induced in parental (P0) hermaphrodites, and then offspring are screened in the F1 or
                  F2 for mutant phenotypes. Following outcross of the mutant allele, the tagged gene can be isolated using PCR. A detailed protocol
                  for <span class="emphasis"><em>Mos1</em></span> mutagenesis is available (<a href="#bib5">Bessereau et al., 2006</a>).
               </p>
               <p>In <span class="emphasis"><em>C. briggsae</em></span>, <span class="emphasis"><em>Mos1</em></span> mutagenesis can be achieved following the <span class="emphasis"><em>C. elegans</em></span> protocol, substituting transgenes <span class="emphasis"><em>mfEx18</em></span> (transposase) and <span class="emphasis"><em>mfEx24</em></span> (transposon) (M.A. Felix, personal communication). Heat-shock conditions are also modified to allow for the greater heat-tolerance
                  of <span class="emphasis"><em>C. briggsae</em></span> animals. Mos1 germline insertions and mutant alleles have been obtained using the following heat-shock treatment of P0 young
                  adults:
                  
               </p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>37&deg; C for one hour, 20&deg; C for one hour, 37&deg; for one hour, and 15&deg; overnight.</p>
                     </li>
                     <li>
                        <p>37&deg; C for 2.5 hours, 20&deg; C for 35 minutes, 37&deg; for one hour, and 25&deg; overnight.</p>
                     </li>
                  </ol>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e795"></a>5.&nbsp;Online resources for <span class="emphasis"><em>C. briggsae</em></span> genetics and genomics
                     </h2>
                  </div>
               </div>
            </div>
            <p>As the development of <span class="emphasis"><em>C. briggsae</em></span> as a model system is in its infancy, information on its genetics and genomics is changing rapidly. The most valuable resources
               for staying up to date are a set of <span class="emphasis"><em>C. briggsae</em></span> web sites that are the repository of considerable published and unpublished data.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e809"></a>5.1.&nbsp;The <span class="emphasis"><em>Caenorhabditis briggsae</em></span> Research Page
                        </h3>
                     </div>
                     <div>
                        <h4 class="subtitle"><a href="http://wormlab.caltech.edu/briggsae" target="_blank">wormlab.caltech.edu/briggsae</a></h4>
                     </div>
                  </div>
               </div>
               <p>This is the best site for <span class="emphasis"><em>C. briggsae</em></span> experimental genetics. Of primary importance are archival lists of <span class="emphasis"><em>C. briggsae</em></span> mutations, linkage data, and recombination maps based on visible mutations. This site also has links to a variety of other
                  resources including a list of <span class="emphasis"><em>C. briggsae</em></span> labs.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e829"></a>5.2.&nbsp;The <span class="emphasis"><em>C. briggsae</em></span> SNP map
                        </h3>
                     </div>
                     <div>
                        <h4 class="subtitle"><a href="http://snp.wustl.edu/snp-research/c-briggsae" target="_blank">snp.wustl.edu/snp-research/c-briggsae</a></h4>
                     </div>
                  </div>
               </div>
               <p>In addition to the SNP map, this site is a download site for <span class="emphasis"><em>C. briggsae</em></span> sequence data, including strain-specific nucleotide polymorphisms.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e843"></a>5.3.&nbsp;WormBase
                        </h3>
                     </div>
                     <div>
                        <h4 class="subtitle"><a href="http://www.wormbase.org/" target="_blank">www.wormbase.org</a></h4>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. briggsae</em></span> genomics data have been incorporated into WormBase. Features include <span class="emphasis"><em>C. briggsae</em></span> gene preditions, the identification of <span class="emphasis"><em>C. briggsae</em></span>-<span class="emphasis"><em>C. elegans</em></span> orthologs and syntenic alignments of <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. elegans</em></span>.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e868"></a>5.4.&nbsp;The WashU <span class="emphasis"><em>C. briggsae</em></span> Genome server
                        </h3>
                     </div>
                     <div>
                        <h4 class="subtitle"><a href="http://genome.wustl.edu/genome.cgi?GENOME=Caenorhabditis%20briggsae" target="_blank">http://genome.wustl.edu/genome.cgi?GENOME=Caenorhabditis%20briggsae</a></h4>
                     </div>
                  </div>
               </div>
               <p>The <span class="emphasis"><em>C. briggsae</em></span> genome server offers sequence data downloads and a link to <span class="emphasis"><em>C. briggsae</em></span>-specific BLAST server.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e885"></a>5.5.&nbsp;The <span class="emphasis"><em>Caenorhabditis</em></span> Genetics Center
                        </h3>
                     </div>
                     <div>
                        <h4 class="subtitle"><a href="http://biosci.umn.edu/CGC/CGChomepage.htm" target="_blank">biosci.umn.edu/CGC/CGChomepage.htm</a></h4>
                     </div>
                  </div>
               </div>
               <p>The CGC is a repository of <span class="emphasis"><em>C. briggsae</em></span> wild isolates and mutant strains.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e899"></a>6.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Baird, S.E. (2001). Strain-specific variation in the pattern of caudal papillae in <span class="emphasis"><em>Caenorhabditis briggsae</em></span> (Nematoda: Rhabditidae); implications for species identification. Nematology <span class="emphasis"><em>3</em></span>, 373&#8211;376.</span>
                     <a href="http://dx.doi.org/10.1163/156854101317020295" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Baird, S.E. (2002). Haldane's Rule by sexual transformation in <span class="emphasis"><em>Caenorhabditis</em></span>. Genetics <span class="emphasis"><em>161</em></span>, 1349&#8211;1353.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12136036&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://www.genetics.org/cgi/content/full/161/3/1349" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Baird, S.E., Davidson, C.R. and Bohrer, J.C. (2005). The genetics of ray pattern variation in <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. BMC Evol. Biol. <span class="emphasis"><em>5</em></span>, 3.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15634358&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2148-5-3" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bessereau, J.L., Wright, A., Williams, D.C., Schuske, K., Davis, M. W., and Jorgensen, E.M. (2001). Mobilization of a <span class="emphasis"><em>Drosophila</em></span> transposon in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> germ line. Nature <span class="emphasis"><em>413</em></span>, 70&#8211;04.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11544527&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35092567" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bessereau, J.L. (2006). Insertional mutagenesis in <span class="emphasis"><em>C. elegans</em></span> using the <span class="emphasis"><em>Drosophila</em></span> transposon Mos1: a method for the rapid identification of mutated genes. Methods Mol. Biol. <span class="emphasis"><em>351</em></span>, 59&#8211;73.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16988426&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Cho, S., Jin, S.-W., Cohen, A. and Ellis, R.E. (2004). A phylogeny of <span class="emphasis"><em>Caenorhabditis</em></span> reveals frequent loss of introns during nematode evolution. Genome Res. <span class="emphasis"><em>14</em></span>, 1207&#8211;1220.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15231741&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.2639304" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Cutter, A.D., F&eacute;lix, M.-A., Barriere, A. and Charlesworth, D. (2006). Patterns of nucleotide polymorphism distinguish temperate
                        and tropical wild isolates of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Genetics <span class="emphasis"><em>174</em></span>, 2021&#8211;2031.</span>
                     <a href="http://dx.doi.org/10.1534/genetics.106.058651" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Dean, F.B., Nelson, J.R., Giesler, T.L., and Lasken, R.S. (2001). Rapid Amplification of plasmid and phage DNA using Phi29
                        DNA polymerase and multiply-primed rolling circle amplification. Genome Res. <span class="emphasis"><em>11</em></span>, 1095&#8211;1099.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11381035&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.180501" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Dellatre, M. and F&eacute;lix, M.-A. (2001). Polymorphism and evolution of vulval precursor cell lineages within two nematode genera,
                        <span class="emphasis"><em>Caenorhabditis</em></span> and <span class="emphasis"><em>Oscheius</em></span>. Curr. Biol. <span class="emphasis"><em>11</em></span>, 631&#8211;643.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11369226&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(01)00202-0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Edgley, M., D'Souza, A., Moulder, G., McKay, S., Shen, B., Gilchrist, E., Moerman, D., and Barstead, R. (2002). Improved detection
                        of small deletions in complex pools of DNA. Nucleic Acids Res. <span class="emphasis"><em>30</em></span>, e52.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12060690&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gnf051" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Graustein, A., Gasper, J.M., Walters, J.R. and Palopoli, M.F. (2002). Levels of nucleotide polymorphism vary with mating system
                        in the nematode genus <span class="emphasis"><em>Caenorhabditis</em></span>. Genetics <span class="emphasis"><em>161</em></span>, 99&#8211;107.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12019226&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://www.genetics.org/cgi/content/full/161/1/99" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Haag, E.S., Wang, S., and Kimble, J. (2002). Rapid coevolution of the nematode sex-determining genes <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fem-3;class=Gene" target="_blank">fem-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=tra-2;class=Gene" target="_blank">tra-2</a></em></span>. Curr. Biol. <span class="emphasis"><em>12</em></span>, 2035&#8211;2041.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12477393&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)01333-7" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hill, R.C., de Carvalho, C.E., Salogiannis, J., Schlager, B., Pilgrim, D., and Haag, E.S. (2006). Genetic flexibility in the
                        convergent evolution of hermaphroditism in <span class="emphasis"><em>Caenorhabditis</em></span> nematodes. Dev. Cell <span class="emphasis"><em>10</em></span>, 531&#8211;538.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16580997&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2006.02.002" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Horvitz, H.R., Brenner, S., Hodgkin, J. and Herman, R.K. (1979). A uniform genetic nomenclature for the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Gen. Genet. <span class="emphasis"><em>175</em></span>, 129&#8211;133.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=292825&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00425528" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hsu, T.M., Chen, X., Duan, S., Miller, R.D. and Kwok, P.-Y. (2001). Universal SNP genotyping assay fluorescence polarization
                        detection. BioTechniques <span class="emphasis"><em>31</em></span>, 560&#8211;570.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11570500&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Inoue, T., Sherwood, D.R., Aspock, G., Butler, J.A., Gupta, B.P., Kirouac, M., Wang, M., Lee, P.Y., Kramer, J.M., Hope, I.,
                        Burglin, T.R., and Sternberg, P.W. (2002). Gene expression markers for <span class="emphasis"><em>Caenorhabditis elegans</em></span> vulval cells. Gene Expr. Patterns <span class="emphasis"><em>2</em></span>, 235&#8211;241.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12617807&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1567-133X(02)00055-8" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jovelin, R., and Phillips, P.C. (2005). Functional constraint and divergence in the G protein family in <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. Mol. Genet. Genomics <span class="emphasis"><em>273</em></span>, 299&#8211;310.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15856303&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s00438-004-1105-6" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kiontke, K., Gavin, N.P., Raynes, Y., Roehrig, C., Piano, F. and Fitch, D.H.A. (2004). <span class="emphasis"><em>Caenorhabditis</em></span> phylogeny predicts convergence of hermaphroditism and extensive intron loss. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>101</em></span>, 9003&#8211;9008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15184656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0403094101" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Li, X., Massey, H.C., Jr, Nolan, T.J., Schad, G.A., Kraus, K., Sundaram, M., and Lok, J.B. (2006). Successful transgenesis
                        of the parasitic nematode Strongyloides stercoralis requires endogenous non-coding control elements. Int. J. Parasitol. <span class="emphasis"><em>36</em></span>, 671&#8211;679.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16500658&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ijpara.2005.12.007" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mello, C.C., Kramer, J.M., Stinchcomb, D., and Ambros, V. (1991). Efficient gene transfer in <span class="emphasis"><em>C. elegans</em></span>: extrachromosomal maintenance and integration of transforming sequences. EMBO J. <span class="emphasis"><em>10</em></span>, 3959&#8211;3970.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1935914&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=453137" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Nayak, S., Goree, J., and Schedl, T. (2005). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fog-2;class=Gene" target="_blank">fog-2</a></em></span> and the evolution of self-fertile hermaphroditism in <span class="emphasis"><em>Caenorhabditis</em></span>. PLOS Biol. <span class="emphasis"><em>3</em></span>, e6.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15630478&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0030006" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Nigon, V. and Dougherty, E.C. (1949). Reproductive patterns and attempts at reciprocal crossing of <span class="emphasis"><em>Rhabditis elegans</em></span> Maupas, 1900 and <span class="emphasis"><em>Rhabditis briggsae</em></span>. J. Exp. Zool. <span class="emphasis"><em>112</em></span>, 485&#8211;503.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15404785&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/jez.1401120307" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rudel, D., and Kimble, J. (2001). Conservation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span> regulation and function in nematodes. Genetics <span class="emphasis"><em>157</em></span>, 639&#8211;54.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11156985&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://www.genetics.org/cgi/content/full/157/2/639" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Stein, L.D., Bao, Z., Blasiar, D., Blumenthal, T., Brent, M.R., Chen, N., Chinwalla, A., Clarke, L., Clee, C., Coghlan, A.,
                        Coulson, A., D'Eustachio, P., Fitch, D.H., Fulton, L.A., Fulton, R.E., Griffiths-Jones, S., Harris, T.W., Hillier, L. W.,
                        Kamath, R.S., Kuwabara, P.E., Mardis, E.R., Marra, M.A., Miner, T.L., Minx, P., Mullikin, J.C., Plumb, R.W., Rogers, J., Schein,
                        J.E., Sohrmann, M., Spieth, J., Stajich, J.E., Wei, C., Willey, D., Wilson, R.K., Durbin, R., and Waterston, R. (2003). The
                        Genome Sequence of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>: A Platform for Comparative Genomics. PLOS Biol. <span class="emphasis"><em>1</em></span>.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14624247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Stothard, P., Hansen, D., and Pilgrim, D. (2002). Evolution of the PP2C Family in <span class="emphasis"><em>Caenorhabditis</em></span>: Rapid Divergence of the Sex-Determining Protein <a href="http://www.wormbase.org/db/get?name=FEM-2;class=Protein" target="_blank">FEM-2</a>. J. Mol. Evol. <span class="emphasis"><em>54</em></span>, 267&#8211;282.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11821919&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s0023901-0008-y" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Streit, A., Li, W., Robertson, B., Schein, J., Kamal, I.H., Marra, M., and Wood, W.B. (1999). Homologs of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> masculinizing gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=her-1;class=Gene" target="_blank">her-1</a></em></span> in <span class="emphasis"><em>C. briggsae</em></span> and the filarial parasite <span class="emphasis"><em>Brugia malayi</em></span>. Genetics <span class="emphasis"><em>152</em></span>, 1573&#8211;0084.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10430584&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://www.genetics.org/cgi/content/full/152/4/1573" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Thomas, J.H. (2006). Concerted evolution of two novel protein families in <span class="emphasis"><em>Caenorhabditis</em></span> species. Genetics <span class="emphasis"><em>172</em></span>, 2269&#8211;2281.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16415360&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.105.052746" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wang, X., and Chamberlin, H.M. (2002). Multiple regulatory changes contribute to the evolution of the <span class="emphasis"><em>Caenorhabditis <a href="http://www.wormbase.org/db/get?name=lin-48;class=Gene" target="_blank">lin-48</a> ovo</em></span> gene. Genes Dev. <span class="emphasis"><em>16</em></span>, 2345&#8211;2349.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12231624&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.996302" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wang, X., and Chamberlin, H.M. (2004). Evolutionary innovation of the excretory system in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Genet. <span class="emphasis"><em>36</em></span>, 231&#8211;232.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14758362&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1301" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Williams, D.C., Boulin, T., Ruaud, A.F., Jorgensen, E.M., and Bessereau, J.L. (2005). Characterization of Mos1-mediated mutagenesis
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: a method for the rapid identification of mutated genes. Genetics <span class="emphasis"><em>169</em></span>, 1779&#8211;1785.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15654093&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.038265" target="_blank">Article</a>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e10" href="#d0e10">*</a></sup>Edited by Ralf J. Sommer. WormMethods editor Victor Ambros. Last revised November 6, 2006. Published December 18, 2006. This
                  chapter should be cited as: Baird, S.E. and Chamberlin, H.M. <span class="emphasis"><em>Caenorhabditis briggsae</em></span> methods (December 18, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.128.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Scott E. Baird and Helen M. Chamberlin. This is an open-access article distributed under the terms of the Creative
                  Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original
                  author and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e49" href="#d0e49">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:scott.baird@wright.edu" target="_top">scott.baird@wright.edu</a> or <a href="mailto:chamberlin.27@osu.edu" target="_top">chamberlin.27@osu.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>
