<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic mapping and manipulation: Chapter 8-Dominant mutations
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMgmdmch8_ArticleID" title="Genetic mapping and manipulation: Chapter 8-Dominant mutations&#xA;">
      <link rel="next" href="#d0e61" title="1.&nbsp;Basics">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="dominantmutations.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMgmdmch8_ArticleID"></a>Genetic mapping and manipulation: Chapter 8-Dominant mutations<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">David</span> <span class="surname">Fay<em><span class="remark">
                                 					<sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Department of Molecular Biology, University of Wyoming, Laramie, Wyoming 82071-3944 USA
                           <p></p></span></span></div>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Andy</span> <span class="surname">Spencer<em><span class="remark">
                                 					<sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Departments of Chemistry and Biochemistry, University of Wisconsin, Madison, Madison, WI 53706 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward genetics and genetic mapping<br />>>> Genetic mapping and manipulation
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=292.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Dominant+mutations" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e61">1. Basics</a></span></dt>
               <dt><span class="sect1"><a href="#d0e152">2. Isolating dominant alleles</a></span></dt>
               <dt><span class="sect1"><a href="#d0e199">3. Mapping a dominant mutation</a></span></dt>
               <dt><span class="sect1"><a href="#d0e431">4. Different types of dominant mutations</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e445">4.1. Haploinsufficiency</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e484">4.2. Dominant-negative alleles</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e518">4.3. Dominant gain-of-function (GOF) alleles</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e549">4.4. Semi-dominant alleles</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e578">5. Genetic tests for dominance classes</a></span></dt>
               <dt><span class="sect1"><a href="#d0e625">6. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e61"></a>1.&nbsp;Basics
                     </h2>
                  </div>
               </div>
            </div>
            <p>Dominant alleles were first described by Mendel to account for the patterns he observed with respect to flower color. For
               example, red flowers are produced when the R allele is present in one or more copies (i.e., genotypes <span class="bold"><strong>R/r</strong></span>  and <span class="bold"><strong>R/R</strong></span>), versus white flowers, which are produced only when the <span class="bold"><strong>r</strong></span>  allele is homozygous (<span class="bold"><strong>r/r</strong></span>). In this case, the <span class="bold"><strong>R</strong></span>  allele is said to be dominant to the <span class="bold"><strong>r</strong></span>  allele, which is recessive. As the study of genetics has matured, the common definition of <span class="emphasis"><em><span class="bold"><strong>dominance</strong></span></em></span> has come to refer to alleles whose phenotype is manifest when present as a heterozygote (<span class="bold"><strong>R/r</strong></span>). Dominant alleles may also be phenotypically deterministic when present along with two or more recessive alleles (e.g.,
               <span class="bold"><strong>R/r/r</strong></span>), a situation sometimes encountered in transgenic strains or with free duplications.
            </p>
            <p>A good example of a dominant allele in <span class="emphasis"><em>C. elegans</em></span> is the <span class="emphasis"><em><span class="bold"><strong>rol-6(<a href="http://www.wormbase.org/db/get?name=su1006;class=Variation" target="_blank">su1006</a>)</strong></span></em></span> allele, which causes a roller (<span class="bold"><strong>Rol</strong></span>) phenotype. <span class="emphasis"><em><span class="bold"><strong>rol-6(<a href="http://www.wormbase.org/db/get?name=su1006;class=Variation" target="_blank">su1006</a>)</strong></span></em></span> animals exhibit the <span class="bold"><strong>Rol</strong></span>  phenotype when they are of the following genotypes: <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a>/rol-6; <a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a>/+;</strong></span></em></span> or +/<span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a></strong></span></em></span>. Because the <span class="bold"><strong>Rol</strong></span> phenotype is observed when a single mutant copy of <span class="emphasis"><em><span class="bold"><strong>rol-6(su10060)</strong></span></em></span> is present, the <span class="emphasis"><em><span class="bold"><strong>rol-6(<a href="http://www.wormbase.org/db/get?name=su1006;class=Variation" target="_blank">su1006</a>)</strong></span></em></span> allele is said to be dominant. Keep in mind that not all alleles of a particular gene will be dominant; there are several
               <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a></strong></span></em></span> alleles that exhibit recessive phenotypes. Dominance or recessivity are allele-specific properties. They are not gene-specific
               properties.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e152"></a>2.&nbsp;Isolating dominant alleles
                     </h2>
                  </div>
               </div>
            </div>
            <p>Depending on the particular developmental question in which you are interested, the systematic isolation of dominant alleles
               may be desirable. If you decide this is the case, the isolation of dominant alleles is straightforward. Although the typical
               genetic screen in <span class="emphasis"><em>C. elegans</em></span> often aims to isolate recessive, loss-of-function alleles, as shown in <a href="#figure1">Figure 1</a>, the isolation of a dominant mutation requires one to simply screen the F1 generation (i.e., the self-progeny of mutagenized
               <a href="http://www.wormbase.org/db/get?name=P0;class=Cell" target="_blank">P0</a> worms), as shown in <a href="#figure2">Figure 2</a>.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="gmdmch8-1.jpg"><img src="gmdmch8-1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1.&nbsp;</b></p>
               </div>
            </div>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="gmdmch8-2.jpg"><img src="gmdmch8-2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 2.&nbsp;</b></p>
               </div>
            </div>
            <p>This is trivial. One can also isolate dominant alleles in the first screen, because dominant mutations will exhibit the mutant
               phenotype in the F2 generation as well as the F1 generation. In a non-clonal screen, where F2 worms on a plate will derive
               from several different <a href="http://www.wormbase.org/db/get?name=P0;class=Cell" target="_blank">P0</a> animals, one may not notice that a particular allele is dominant until outcrossing fails to eliminate the mutant phenotype
               in cross-progeny. When outcrossing any newly isolated mutation, one should carefully observe the genetic behavior of an allele
               to determine whether it is dominant or recessive.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e199"></a>3.&nbsp;Mapping a dominant mutation
                     </h2>
                  </div>
               </div>
            </div>
            <p>Whether your dominant mutation was isolated on purpose or by chance, the next step will be to map it to a chromosome. You'll
               recall from earlier sections that recessive alleles are initially crossed to wild type and then progeny of the heterozygous
               animals are then scored for segregation of the mutations and known genetic markers. If an allele is dominant, however, it
               is necessary to change our thinking slightly when scoring the segregation of phenotypes in mapping strains. Because we cannot
               determine whether a phenotypically mutant animal is heterozygous or homozygous for the dominant allele by simple observation,
               we use the alternative strategy of mapping the absence of our dominant allele. Once one has thought about it carefully, it
               can often be easier to map true dominant mutations than recessive mutations.
            </p>
            <p>The phenotype of a recessive mutation disappears when crossed into a mapping strain. Consider <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=lin-1" target="_blank">lin-1</a></strong></span></em></span>, which causes a multivulval (<span class="bold"><strong>Muv</strong></span>) phenotype when crossed into a strain as shown in <a href="#figure3">Figure 3</a>. We can then score the co-segregation of <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=lin-1" target="_blank">lin-1</a></strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=dpy-17" target="_blank">dpy-17</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-32" target="_blank">unc-32</a></strong></span></em></span> in the normal manner by picking <span class="bold"><strong>Muv non-Dpy Uncs</strong></span>  and noting how often the <span class="bold"><strong>Dpy Unc</strong></span>  phenotypes co-segregate with <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=lin-1" target="_blank">lin-1</a></strong></span></em></span><span class="bold"><strong>Muv</strong></span>.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="gmdmch8-3.jpg"><img src="gmdmch8-3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 3.&nbsp;</b></p>
               </div>
            </div>
            <p>Now consider a dominant mutation that, in contrast to <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=lin-1" target="_blank">lin-1</a></strong></span></em></span>, exhibits the mutant phenotype when the allele is present as a heterozygote. When we cross into our chromosomal mapping strains,
               all the heterozygous cross progeny will exhibit the mutant phenotype. Let's consider an imaginary dominant mutation, <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span>, as shown in <a href="#figure4">Figure 4</a>, which we'll say causes a spiked-head phenotype. These <span class="emphasis"><em><span class="bold"><strong>dom-1/+</strong></span></em></span> heterozygotes will display the spiked-head phenotype and will be indistinguishable from <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> homozygotes. Thus, we will be unable to score the segregation of <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> with <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=dpy-17" target="_blank">dpy-17</a></strong></span></em></span>, <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=unc-32" target="_blank">unc-32</a></strong></span></em></span> by following spiked-head animals because we won't know whether the animals are <span class="emphasis"><em><span class="bold"><strong>dom-1/+</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1</strong></span></em></span>.
            </p>
            <p>The trick to mapping such true dominant mutations is to follow the animals that do not display the dominant phenotype. In
               other words, ignore the <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> phenotype and look only for those animals that are wild type. The reason for this is as follows: as we can always tell when
               <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> is present because of the dominant spiked-head phenotype, we follow the absence of <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> and note how often the markers segregate with <span class="bold"><strong>non-spiked-head</strong></span>  animals.
            </p>
            <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="gmdmch8-4.jpg"><img src="gmdmch8-4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 4.&nbsp;</b></p>
               </div>
            </div>
            <p>The results from chromosomal mapping of dominant mutations are apparent in the F2 generation. Because you are following the
               absence of <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span>, at this stage you are looking for <span class="bold"><strong>non-spiked-head</strong></span>  animals. If <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> is on the same chromosome as your markers and is relatively close (as shown in <a href="#figure5">Figure 5</a>; case #1), then the only animals with normal heads will be the <span class="bold"><strong>Dpy Uncs</strong></span>. Also if you were to pick <span class="bold"><strong>spiked-head</strong></span> animals and look for those that throw 100% spiked-head animals in their progeny (<span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1</strong></span></em></span>), few or none will throw <span class="bold"><strong>Dpy Uncs</strong></span>. If, however, <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> is on a different chromosome from the markers (as shown in <a href="#figure5">Figure 5</a>; case #2), then it will segregate independently from the markers, and 3/4 of the <span class="bold"><strong>Dpy Unc</strong></span> progeny will display <span class="bold"><strong>spiked heads</strong></span>. Also in this case, 2/3 of the <span class="bold"><strong>spiked-head</strong></span>  (<span class="bold"><strong>non-Dpy Unc</strong></span>) progeny that throw exclusively <span class="bold"><strong>spiked-head</strong></span>  progeny (<span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1</strong></span></em></span>) will throw <span class="bold"><strong>Dpy Uncs</strong></span>.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="gmdmch8-5.jpg"><img src="gmdmch8-5_s.jpg" border="2" align="middle" alt=" figure 5"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 5.&nbsp;</b></p>
               </div>
            </div>
            <p>Once we have our dominant mutation mapped to a chromosome, it is similarly easy to collect data for three-point mapping. We'll
               start again with our balanced strain that is heterozygous for both <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> and our markers. This strain will have the spiked-head phenotype. In this example, we'll assume <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> lies between <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=dpy-17" target="_blank">dpy-17</a></strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=unc-32" target="_blank">unc-32</a></strong></span></em></span>. When looking for recombinant <span class="bold"><strong>Unc-non-Dpy</strong></span> or <span class="bold"><strong>Dpy-non-Unc</strong></span>  animals, we will know immediately whether or not the recombinant picked up the mutant <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> allele because of its dominance. Recombinants that pick up the <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> allele will have a spiked head, and recombinants that don't pick it up will have wild-type heads. Quick and easy.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e431"></a>4.&nbsp;Different types of dominant mutations
                     </h2>
                  </div>
               </div>
            </div>
            <p>Why do some mutations act in a dominant fashion? Below we examine some different mechanisms through which a mutation can confer
               a dominant phenotype. In certain situations, different dominant alleles may require different mapping strategies. These situations
               must be managed on a case-by-case basis. In each example below, we will consider the fictional <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> gene and imagine different situations that could give rise to various types of dominant alleles in <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span>.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e445"></a>4.1.&nbsp;Haploinsufficiency
                        </h3>
                     </div>
                  </div>
               </div>
               <p>This describes a situation in which one copy (<span class="emphasis"><em>haplo</em></span>) of a wild-type gene is not enough to provide wild-type function when the other copy is compromised. This can only occur
                  for loss-of-function alleles. Consider again our fictional dominant mutation, <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span>. Let's assume that a certain threshold of <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> activity is required to avoid the abnormal spiked-head phenotype; two copies of the wild-type gene are required to achieve
                  that threshold, and any drop below that threshold allows the mutant spiked head to form. Mutations in <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> that reduce or eliminate its activity would therefore behave dominantly because in heterozygous animals, the single remaining
                  wild-type copy of the <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> gene would be <span class="emphasis"><em>insufficient</em></span> to provide the wild-type levels of gene activity. Thus, the loss-of-function <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> mutant allele may produce a similar phenotype whether present in one or two copies and behaves in a dominant fashion. Alternatively,
                  <span class="emphasis"><em><span class="bold"><strong>dom-1/+</strong></span></em></span>  heterozygous animals may display a phenotype that is quantitatively or qualitatively different from homozygous <span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1</strong></span></em></span> animals, since the former would still retain half the normal gene dose.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e484"></a>4.2.&nbsp;Dominant-negative alleles
                        </h3>
                     </div>
                  </div>
               </div>
               <p>These typically occur when the mutant allele does not function normally <span class="emphasis"><em>and</em></span> either directly inhibits the activity of the wild-type protein (usually through dimerization) or inhibits the activity of
                  another protein that is required for the normal function of the wild-type protein (such as an activator or downstream component
                  in a pathway). Although this situation can effectively result in the loss of function of the wild-type protein, it differs
                  markedly from haploinsufficiency. Consider an animal that is heterozygous for a dominant-negative allele of <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span>. In this case, we'll also imagine that the single wild-type copy of <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> would normally provide enough <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> activity to avoid the spiked-head phenotype. However, because of a dominant-negative version of <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> would actually <span class="emphasis"><em><span class="bold"><strong>interfere</strong></span></em></span> with the function of wild-type <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> its activity is further reduced and a mutant phenotype results.
               </p>
               <p>A well-known example of a gene that can incur dominant-negative mutations is the small GTPase Ras. These dominant-negative
                  alleles of Ras are not functional themselves because they preferentially bind GDP and stay locked in the inactive state. In
                  addition, they also prevent the Ras exchange factor (which binds Ras-GDP and catalyzes GDP/GTP exchange and subsequent Ras
                  activation) from acting on wild-type Ras, essentially killing all Ras activity.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e518"></a>4.3.&nbsp;Dominant gain-of-function (GOF) alleles
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Also termed hypermorphs, these refer to mutations that result in elevated levels of gene activity. In some cases, dominant
                  GOF mutations may acquire novel biochemical activities, in which case they may be referred to as neomorphs. It is possible
                  to imagine numerous scenarios that might lead to the removal of normal regulatory constraints and the enhancement of protein
                  activity. For example, a mutation in the promoter region could lead to overexpression of the gene and the saturation of negative
                  regulatory pathways. Alternatively, point mutations in a region of a gene important for its regulation could lead to inappropriate
                  activity and mutant phenotypes. Let's revisit <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> and imagine it is an enzyme whose activity promotes head development. Assume that normal levels of <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> activity result in normal head development and any <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> activity above normal levels results in a spiked head. Also assume that a negative regulatory phosphate group is added to
                  an N-terminal serine when <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> activity gets to the threshold required for normal development. A point mutation that makes this serine phosphorylation impossible
                  (e.g., Ser <span class="symbol">&#8594;</span> Ala) could remove the negative regulation of <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> and allow its activity to proceed unchecked, thus leading to the <span class="bold"><strong>spiked-head</strong></span>  phenotype. In short, too much of a good thing can lead to developmental abnormalities.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e549"></a>4.4.&nbsp;Semi-dominant alleles
                        </h3>
                     </div>
                  </div>
               </div>
               <p>It is actually quite typical for dominant alleles to behave in a partially dominant fashion. Alleles are designated semi-dominant
                  when the penetrance of the phenotype in heterozygous animals (<span class="emphasis"><em><span class="bold"><strong>dom-1/+</strong></span></em></span>) is less than that observed for homozygous animals (<span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1</strong></span></em></span>). For <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span>, this would be the case if <span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1</strong></span></em></span> animals were 100% spiked head and <span class="emphasis"><em><span class="bold"><strong>dom-1/+</strong></span></em></span> animals were 60% spiked head. This is an important point, as the basic mapping strategies outlined above were assuming 100%
                  dominance. In practice, this is not necessarily that difficult to deal with, as the presence of the mutation will always be
                  seen by the next generation, as is the case for any recessive allele. Thus mapping a semi-dominant mutation will simply require
                  following progeny for an extra generation to distinguish between <span class="emphasis"><em><span class="bold"><strong>dom-1/+</strong></span></em></span> and +/+ animals.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e578"></a>5.&nbsp;Genetic tests for dominance classes
                     </h2>
                  </div>
               </div>
            </div>
            <p>To attempt to distinguish between various classes of dominant mutations, a number of genetic tests can be performed. For example,
               to determine if a mutant phenotype observed in a heterozygous animal is due to haploinsufficiency, one can directly examine
               animals that are heterozygous for a chromosomal deficiency that removes the entire gene (as well as a number of other genes
               presumably). Alternatively, if a deletion or null allele of the gene exists, placing this mutation over the wild-type chromosome
               could provide an even cleaner answer. In addition, to distinguish haploinsufficieny effects from hypermorphic mutations, one
               can further compare homozygous mutant animals (<span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1</strong></span></em></span>) with animals that are heterozygous for the mutation and the deficiency (<span class="emphasis"><em><span class="bold"><strong>dom-1/Df</strong></span></em></span>). If the homozygous mutants show a more severe phenotype than the mutant allele over the deficiency, then it is likely that
               the mutation is at least partially dominant, although one can have both dominance and haploinsufficient effects for the same
               allele.
            </p>
            <p>In addition, a hypermorphic mutation would be expected to exert an effect even in the presence of two normal copies of the
               gene. Thus, a genetic test of this can be carried out using a free duplication that contains a wild-type copy of the gene,
               which is examined in the background of the heterozygous mutant (e.g., <span class="emphasis"><em><span class="bold"><strong>dom-1/+; Dp</strong></span></em></span>). A further test is to examine <span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1; Dp</strong></span></em></span> animals. In this case, if the mutant allele is not hypermorphic (only LOF associated with haploinsufficieny), the phenotype
               of this animal should be no more severe than <span class="emphasis"><em><span class="bold"><strong>dom-1/+</strong></span></em></span> animals and may even be less severe if the <span class="emphasis"><em><span class="bold"><strong>dom-1</strong></span></em></span> allele contains some residual activity.
            </p>
            <p>Other questions may be more difficult to answer genetically, particularly in the absence of knowing or understanding the molecular
               functions of the gene. For example, distinguishing dominant negatives from dominant gain of function alleles may be difficult
               in a vacuum. The ability of RNAi to phenocopy or enhance a dominant mutation would suggest that the mutation is a dominant
               negative, although a negative result in this case is difficult to interpret. Also, if the gene is cloned, then attempts to
               overexpress the wild-type version of the gene product may be informative in this regard, as phenocopy would indicate a hypermorphic
               mutation. Also, a dominant negative might be expected to be less penetrant in a background that contains one or more copies
               of the wild-type gene (e.g., <span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1</strong></span></em></span> versus; <span class="emphasis"><em><span class="bold"><strong>dom-1/dom-1; Dp</strong></span></em></span>), although a number of hand-waving explanations can theoretically weaken these types of arguments.
            </p>
            <p>In closing, we refer readers to a number of published papers dealing with various types of dominant mutations in <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> (see below). We hope you enjoyed this discussion of dominant alleles. Now get back to work, dammit (A. Spencer).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e625"></a>6.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Greenwald, I.S., and Horvitz, H.R. (1982). Genetics <span class="emphasis"><em>101</em></span>, 211&#8211;225.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7173602&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Greenwald, I.S., Sternberg, P.W., and Horvitz, H.R. (1983). Cell <span class="emphasis"><em>34</em></span>, 435&#8211;444.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6616618&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(83)90377-X" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Han, M., and Sternberg, P.W. (1991). Genes Dev. <span class="emphasis"><em>5</em></span>, 2188&#8211;2198.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1748278&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Han, M., Aroian, R.V., and Sternberg, P.W. (1990). Genetics <span class="emphasis"><em>126</em></span>, 899&#8211;913.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2076820&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Han, M., and Sternberg, P.W. (1990). Cell <span class="emphasis"><em>63</em></span>, 921&#8211;931.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2257629&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(90)90495-Z" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mitenko, N.L., Eisner, J.R., Swiston, J.R., and Mains, P.E. (1997). Genetics <span class="emphasis"><em>147</em></span>, 1665&#8211;1674</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9409829&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised May 6, 2005. Published February 17, 2006. This chapter should be cited as: Fay, D. and
                  Spencer, A. Genetic mapping and manipulation: Chapter 8-Dominant mutations (February 17, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.97.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 David Fay and Andy Spencer. This is an open-access article distributed under the terms of the Creative Commons Attribution
                  License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source
                  are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:davidfay@uwyo.edu" target="_top">davidfay@uwyo.edu</a> or <a href="mailto:aspencer@biochem.wisc.edu" target="_top">aspencer@biochem.wisc.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

