<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Pristionchus pacificus genomics: from genetics to genome sequence
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#Ppagenomics_ArticleID" title="Pristionchus pacificus genomics: from genetics to genome sequence&#xA;">
      <link rel="next" href="#d0e70" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="ppagenomics.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="Ppagenomics_ArticleID"></a><span class="emphasis"><em>Pristionchus pacificus</em></span> genomics: from genetics to genome sequence<sup><a name="d0e10" href="#ftn.d0e10">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Christoph Dieterich<sup>1</sup><em><span class="remark"><sup><a name="d0e37" href="#ftn.d0e37">&sect;</a></sup></span></em>, Waltraud Roeseler<sup>1</sup>, Jagan Srinivasan<sup>2</sup><em><span class="remark"><sup>&sect;</sup></span></em></span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup>Department of Evolutionary Biology, Max Planck Institute for Developmental Biology; </span><span class="address">Tuebingen, D-72076 Germany
                           <p></p></span></span><span class="affiliation"><span class="orgname"><sup>2</sup>Division of Biology, California Institute of Technology; </span><span class="address">Pasadena, CA-91125, USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Methods in nematode species other than <em>C. elegans</em>
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=313.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Pristionchus+pacificus+genomics:+from+genetics+to+genome+sequence" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e70">1. Introduction</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e110">1.1. General biology and genetics</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e153">1.2. Biogeography and polymorphisms in <span class="emphasis"><em>Pristionchus pacificus</em></span></a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e243">2. Genomic resources for <span class="emphasis"><em>P. pacificus</em></span></a></span></dt>
               <dt><span class="sect1"><a href="#d0e308">3. Genetic linkage map of <span class="emphasis"><em>P. pacificus</em></span></a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e322">3.1. Generation of polymorphic markers</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e425">3.2. Preparation of a meiotic mapping panel and linkage analysis</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e439">3.3. Chromosomal allocation and fine mapping of mutants in <span class="emphasis"><em>P. pacificus</em></span></a></span></dt>
                     <dt><span class="sect2"><a href="#d0e870">3.4. Mapping mutations using morphological markers in <span class="emphasis"><em>P. pacificus</em></span></a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e931">4. Physical map of <span class="emphasis"><em>P. pacificus</em></span></a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e942">4.1. AFLP-based BAC fingerprinting</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e958">4.2. Image analysis of AFLP fingerprints and contig assembly</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e972">4.3. Integration of the genetic linkage map and physical map</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e1024">5. Bioinformatic resources for <span class="emphasis"><em>P. pacificus</em></span></a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e1094">5.1. Genome viewer</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1099">5.2. Similarity searches</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1126">5.3. Genetic and physical map</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1162">5.4. Data retrieval</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1167">5.5. WikiOnchus</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e1175">6. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1180">7. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e70"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <div class="blockquote">
               <table border="0" width="100%" cellspacing="0" cellpadding="0" class="blockquote" summary="Block quote">
                  <tr>
                     <td width="10%" valign="top">&nbsp;</td>
                     <td width="80%" valign="top">
                        <div class="literallayout">
                           <p>&#8216;Nothing&nbsp;in&nbsp;biology&nbsp;makes&nbsp;sense&nbsp;except&nbsp;in&nbsp;the&nbsp;light&nbsp;of&nbsp;evolution&#8217;</p>
                        </div>
                     </td>
                     <td width="10%" valign="top">&nbsp;</td>
                  </tr>
                  <tr>
                     <td width="10%" valign="top">&nbsp;</td>
                     <td colspan="2" align="right" valign="top">--<span class="attribution">Theodosius Dobzhansky (The American Biology Teacher, 1973)</span></td>
                  </tr>
               </table>
            </div>
            <p>Satellite systems are proving to be important players in understanding the differences between ontogenies during evolution.
               The satellite system <span class="emphasis"><em>P. pacificus</em></span>, since its discovery nearly a decade ago (<a href="#bib18">Sommer et al., 1996</a>), has provided developmental biologists with new insights into various developmental processes (<a href="#bib1">Eizinger and Sommer, 1997</a>; <a href="#bib5">Jungblut and Sommer, 1998</a>; <a href="#bib16">Sigrist and Sommer, 1999</a>; <a href="#bib25">Zheng et al., 2005</a>). Further understanding of these developmental processes necessitated a genomic framework. Generation of an integrated genetic
               linkage map and a physical map for mapping of mutants has set a good foundation for such molecular studies (<a href="#bib20">Srinivasan et al., 2003</a>; <a href="#bib21">Srinivasan et al., 2002</a>). With the impending genome sequence being generated by the Washington Genome sequencing center (WUGSC) funded by NHGRI,
               <span class="emphasis"><em>P. pacificus</em></span> is bound to cross the &#8216;next frontier&#8217; in our understanding of evolution of developmental and behavioral processes. This review
               highlights the current status of <span class="emphasis"><em>P. pacificus</em></span> genomics.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e110"></a>1.1.&nbsp;General biology and genetics
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>Pristionchus pacificus</em></span> is a free-living nematode belonging to the family Diplogastridae. It is 1 mm long and has two sexes <span class="emphasis"><em>viz</em></span>. self-fertilizing hermaphrodites and cross fertilizing males, making it amenable to genetic analysis. Adult hermaphrodites
                  are larger than males. In <span class="emphasis"><em>P. pacificus</em></span>, like <span class="emphasis"><em>C. elegans</em></span>, animals with two X chromosomes develop as hermaphrodites, whereas XO animals develop as males. Males are rare, occurring
                  at a low frequency, but can be induced under low food or stress conditions. The life cycle of <span class="emphasis"><em>P. pacificus</em></span> at 20&deg;C is 4 days (<a href="#bib18">Sommer et al., 1996</a>). A single self-fertilizing hermaphrodite lays on average 190 eggs over a period of several days. <span class="emphasis"><em>P. pacificus</em></span>, has the four larval stages seen in many different nematodes including <span class="emphasis"><em>C. elegans</em></span>. However, we observe a divergence in hatching time with respect to the molt between these species, suggesting that such heterochronic
                  changes play a major role in developmental evolution (<a href="#bib3">Felix et al., 1999</a>). The potential of <span class="emphasis"><em>P. pacificus</em></span> as a genetic model system has been proven by the generation of a diverse range of mutants affecting several developmental
                  processes such as vulva development, sex determination and gonad development (<a href="#bib12">Pires-daSilva and Sommer, 2004</a>; <a href="#bib14">Rudel et al., 2005</a>; <a href="#bib18">Sommer et al., 1996</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e153"></a>1.2.&nbsp;Biogeography and polymorphisms in <span class="emphasis"><em>Pristionchus pacificus</em></span></h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>P. pacificus</em></span> is highly cosmopolitan in distribution and has been isolated from several continents. At present, there are about 15 isolates
                  of <span class="emphasis"><em>P. pacificus</em></span> (<a href="#figure1">Figure 1</a>). The two commonly used <span class="emphasis"><em>P. pacificus</em></span> strains PS312 and PS1843 are available at the CGC (<a href="http://wormbase.org/db/gene/strain" target="_top">http://wormbase.org/db/gene/strain</a>). The other strains can be requested from the Sommer Lab in Germany (<a href="http://www.eb.tuebingen.mpg.de/departments/4-evolutionary-biology" target="_top">http://www.eb.tuebingen.mpg.de/dept4/home.html</a>). Of these, the North American strains, <span class="emphasis"><em>viz</em></span>. California (PS312), Washington (PS1843), and Hawaii (JU138), and the European strain RS106 from Poland, have identical ITS
                  sequences, representing different isolates from <span class="emphasis"><em>Pristionchus pacificus</em></span> (<a href="#bib19">Srinivasan et al., 2001</a>; <a href="#figure1">Figure 1</a>). This data was further confirmed by mating experiments between these four species. Males from any of these strains can be
                  mated with hermaphrodites of the other strains (<a href="#bib19">Srinivasan et al., 2001</a>).
               </p>
               <p>Amplified restriction fragment length polymorphism (AFLP) using multiple primer combinations and restriction fragment length
                  polymorphism studies confirmed that the three American strains of <span class="emphasis"><em>P. pacificus</em></span> differ substantially from one another (<a href="#bib15">Schlak, 1997</a>; <a href="#bib19">Srinivasan et al., 2001</a>). However, the developmentally distinct strain RS106 from Poland exhibited an AFLP pattern that is identical to the one from
                  <span class="emphasis"><em>P. pacificus var</em></span>. California (PS312) (<a href="#bib19">Srinivasan et al., 2001</a>). Sequence analysis of the polymorphisms indicated that between the polymorphic strains of <span class="emphasis"><em>P. pacificus</em></span> (PS312 and PS1843), 73.7% were substitutions and 26.3% were insertions/deletions. Of the substitutions, 47.3% were transitions
                  and 52.7% were transversions, a finding that is exactly converse to that of <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib7">Koch et al., 2000</a>; <a href="#bib21">Srinivasan et al., 2002</a>; <a href="#bib23">Wicks et al., 2001</a>). Data from these polymorphisms provided the necessary prerequisite for further mapping studies to clone genes defined by
                  mutations.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="495">
<tr>
  <td align="center"><p><a href="Ppagenomicsfig1.jpg"><img src="Ppagenomicsfig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. List of <span class="emphasis"><em>P. pacificus</em></span> strains isolated from different parts of the world.&nbsp;</b>A representative AFLP fingerprint showing the different laboratory strains of <span class="emphasis"><em>P. pacificus</em></span>; C= California, P= Poland, H= Hawaii, W= Washington.
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e243"></a>2.&nbsp;Genomic resources for <span class="emphasis"><em>P. pacificus</em></span></h2>
                  </div>
               </div>
            </div>
            <p>To get a more complete idea of the genome of <span class="emphasis"><em>P. pacificus</em></span>, a combination of different libraries were generated:
            </p>
            <div class="orderedlist">
               <ol type="a">
                  <li>
                     <p><span class="bold"><strong>EST library</strong></span>: Initially a mixed-stage library of <span class="emphasis"><em>P. pacificus var</em></span>. California (PS312) was generated and later on stage-specific libraries were made and sequenced from the oligodT ends (<a href="http://www.nematode.net" target="_top">http://www.nematode.net</a>). A total of 14,663 sequences were generated which were then clustered into 2603 clusters. The EST data are accessible at
                        the following links (<a href="http://www.ncbi.nlm.nih.gov/dbEST/" target="_top">http://www.ncbi.nlm.nih.gov/dbEST</a>, <a href="http://www.nematode.net" target="_top">http://www.nematode.net</a>).
                     </p>
                  </li>
                  <li>
                     <p><span class="bold"><strong>BAC library</strong></span>: Two <span class="emphasis"><em>P. pacificus</em></span> var. California BAC libraries were generated in collaboration with KeyGene NV Netherlands: one library was constructed by
                        partial digestion of genomic DNA with HindIII and the other was constructed with EcoRI. The average size of the BAC clones
                        from the HindIII library was 126 kb and that from the EcoRI library was 70 kb. Around 700 bp from the 5<span class="symbol">&#8242;</span> and 3<span class="symbol">&#8242;</span> ends of each BAC was sequenced (end sequences) at the Genome Centre at the Max Planck Institute for Developmental Biology,
                        Tuebingen. The end sequences were checked for repeats and were blasted against the non-redundant database of NCBI and the
                        <span class="emphasis"><em>C. elegans</em></span> Wormpep databases. There are currently <span class="bold"><strong>31,387 BAC end sequences</strong></span> from both the libraries and they are accessible at NCBI (<a href="http://www.ncbi.nlm.nih.gov/entrez" target="_top">http://www.ncbi.nlm.nih.gov/entrez</a>).
                     </p>
                  </li>
                  <li>
                     <p><span class="bold"><strong>Fosmid library</strong></span>: A fosmid library of <span class="emphasis"><em>P. pacificus var</em></span>. California (PS312) was generated containing 50,000 clones in collaboration with California Institute of Technology, Pasadena.
                        From the 50,000 clones, 25,000 clones were end sequenced from both ends at the Genome Sciences Centre, Vancouver, Canada.
                        On average, 500 bp were sequenced from the fosmid ends. <span class="bold"><strong>48,568 fosmid end sequences</strong></span> are accessible at NCBI (<a href="http://www.ncbi.nlm.nih.gov/entrez" target="_top">http://www.ncbi.nlm.nih.gov/entrez</a>).
                     </p>
                  </li>
               </ol>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e308"></a>3.&nbsp;Genetic linkage map of <span class="emphasis"><em>P. pacificus</em></span></h2>
                  </div>
               </div>
            </div>
            <p>To construct a genetic linkage map, the sequence information from the BAC ends was used to search for polymorphisms between
               the working &#8216;wild type&#8217; <span class="emphasis"><em>P. pacificus</em></span> var. California and the polymorphic Washington strain. We used the &#8220;single stranded conformational polymorphism&#8221; (SSCP) technique
               (<a href="#bib10">Orita et al., 1989a</a>), to generate polymorphic markers and map them on the meiotic mapping panel.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e322"></a>3.1.&nbsp;Generation of polymorphic markers
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The strategy involved designing PCR primers to randomly amplify 180 to 200 bp amplicons in sequenced BAC ends and EST sequences
                  using PrimeArray(c) (<a href="#bib13">Raddatz et al., 2001</a>) and analysing the PCR products for polymorphisms using the SSCP technique (<a href="#figure2">Figure 2</a>; <a href="#bib11">Orita et al., 1989b</a>). As of now we have generated 577 SSCP markers.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="468">
<tr>
  <td align="center"><p><a href="Ppagenomicsfig2.jpg"><img src="Ppagenomicsfig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Strategy for SNP detection: PCR primers were designed using the program PrimeArray to amplify a 180- to 200-bp amplicons
                           (<a href="#bib13">Raddatz et al., 2001</a>).&nbsp;</b>The amplicons were tested for polymorphisms using the SSCP technique. Reprinted with permission from <a href="#bib21">Srinivasan et al. (2002)</a>. Copyright &copy; 2002 the Genetics Society of America.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e355"></a>3.1.1.&nbsp;PCR mix for generation of polymorphic markers
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Genomic DNA was extracted from <span class="emphasis"><em>P. pacificus</em></span> var. California and <span class="emphasis"><em>P. pacificus</em></span> var. Washington strains using the BioRad genomic DNA extraction kit (BioRad). For a typical PCR assay, 50 ng DNA was used
                     as a template in the following reaction:
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td>Distilled Water</td>
                        <td>6.95 <span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Forward Primer (20 <span class="symbol">&#956;</span>M)
                        </td>
                        <td>0.5 <span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Reverse Primer (20 <span class="symbol">&#956;</span>M)
                        </td>
                        <td>0.5 <span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>10X PCR Buffer</td>
                        <td>1.0 <span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>dNTPs (2 mM)</td>
                        <td>1.0 <span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Taq Polymerase</td>
                        <td>0.05 <span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                  </table>
                  <p>Thermocycling was done in a Perkin Elmer Gene Amp9700 PCR machine under standard conditions consisting of an initial denaturation
                     at 94&deg;C for 3 min, followed by 30 cycles of 94&deg;C for 1 min, 50&deg;C for 1 min, and 72&deg;C for 1 min and a final incubation at 72&deg;C
                     for 7 min. The PCR products were checked on a 1% agarose gel for the right size before loading them onto the SSCP gels.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e417"></a>3.1.2.&nbsp;Gel electrophoresis and polymorphism detection
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>For SSCP detection (<a href="#figure2">Figure 2</a>), the samples were diluted 1:1 in denaturing solution (95% formamide, 0.1% xylene cyanol, 0.1% bromophenol blue, 10mM NaOH),
                     denatured at 95&deg;C for 5 min and loaded onto a GeneGel Excel prepoured 6% acrylamide gel (Pharmacia Biotech). Gels were run
                     for 10 min at 200V, 10 mA, 5W and subsequently 2.45 hours at 375 V, 15mA, 10W. After running, the gels were fixed for 30 min
                     and then stained for 30 min. The gels were washed in water for a minute and then treated with developing solution for 6 min
                     and 30 min in stopping solution to detect the DNA. Mobility differences between the California and Washington PCR products
                     were checked both at the single and double stranded DNA level.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e425"></a>3.2.&nbsp;Preparation of a meiotic mapping panel and linkage analysis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The meiotic mapping panel was prepared from 46 randomly picked F2 animals from a cross between phenotypically marked hermaphrodites
                  of the California strain and Washington strain males. F2 animals were cloned and genomic DNA extracted from them. The polymorphic
                  markers generated were tested on the mapping panel and the markers were assigned linkage groups using the Map Manager Software
                  using the Kosambi mapping function (<a href="http://www.mapmanager.org/" target="_top">http://www.mapmanager.org</a>; <a href="#bib9">Manly et al., 2001</a>). Local order was then determined by manually placing the marker at the location that minimised the number of double recombinants.
                  Data analysis ordered 512 loci of the 577 loci into six linkage groups that were supported by LOD scores <span class="symbol">&#8805;</span>5.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e439"></a>3.3.&nbsp;Chromosomal allocation and fine mapping of mutants in <span class="emphasis"><em>P. pacificus</em></span></h3>
                     </div>
                  </div>
               </div>
               <p>To assign mutants to chromosomes on the genetic linkage group, two representative markers per linkage group were chosen for
                  analyses. The chosen SSCP markers were from the two opposite ends of each linkage group. Homozygous mutant animals, generated
                  in the California background, were crossed with wild-type Washington strain males and the heterozygous F1 progeny was selfed.
                  DNA from these 21 homozygote animals was isolated and tested for segregation on the representative markers. In order to assign
                  a linkage group, the selected chromosomal markers were tested on these 21 clones and the segregation of the paternal Washington
                  versus the maternal California pattern was tested. The segregation was measured, as a unitless map ratio Washington/California
                  (W/C).
               </p>
               <p>Unlinked markers are expected to be represented equally and henceforth will have a ratio of approximately 1. Linked markers
                  on the other hand, will have a predominant maternal segregation pattern and therefore will yield a value close to 0. To further
                  fine map the mutants, the SSCP markers near to the chromosomal marker showing the least Washingtion/California ratio were
                  tested for segregation with the selfed progeny. The current version of the genetic linkage map can be seen at <a href="http://www.pristionchus.org/cgi-bin/cmap.pl" target="_top">http://www.pristionchus.org/cgi-bin/cmap.pl</a>.
               </p>
               <p>The genetic linkage map of <span class="emphasis"><em>P. pacificus</em></span> has six chromosomes with five autosomes and one sex chromosome (<a href="#figure3">Figure 3</a>). Sequence similarity comparisons between <span class="emphasis"><em>P. pacificus</em></span> and <span class="emphasis"><em>C. elegans</em></span> at a global scale suggest that the six chromosomes can be homologized between the two species (<a href="#bib26">Hong and Sommer, 2006</a>). However, unlike <span class="emphasis"><em>C. elegans</em></span>, the lengths of the linkage groups exceeds more than 50 cM in length in <span class="emphasis"><em>P. pacificus</em></span> (<a href="#bib26">Hong and Sommer, 2006</a>). The reason for the increased chromosome length is due to the high incidence of double crossovers per chromosome in <span class="emphasis"><em>P. pacificus</em></span> compared to one crossover in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib4">Hodgkin, 1988</a>; <a href="#bib21">Srinivasan et al., 2002</a>). Molecular studies indicated many intra- but only limited inter-chromosomal rearrangements between the genomes of both species.
                  Additionally, limited microsynteny is seen within a contiguous 126 kb genomic region limiting the likelihood of identifying
                  orthologous genes of <span class="emphasis"><em>P. pacificus</em></span> and <span class="emphasis"><em>C. elegans</em></span> based on positional information within the two genomes (<a href="#bib8">Lee et al., 2003</a>; <a href="#bib21">Srinivasan et al., 2002</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="585">
<tr>
  <td align="center"><p><a href="Ppagenomicsfig3.jpg"><img src="Ppagenomicsfig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 3. Genetic linkage map of <span class="emphasis"><em>P. pacificus</em></span> showing the different linkage groups.&nbsp;</b>The lengths of the linkage groups are not 50cM like <span class="emphasis"><em>C. elegans</em></span>, due to multiple crossovers.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e521"></a>3.3.1.&nbsp;PCR mix for chromosomal allocation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Mix for 4 Primers (50 <span class="symbol">&#956;</span>M):
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td></td>
                        <td><span class="bold"><strong>1 sample</strong></span></td>
                        <td><span class="bold"><strong>26 samples</strong></span></td>
                        <td><span class="bold"><strong>28 samples</strong></span></td>
                        <td><span class="bold"><strong>55 samples</strong></span></td>
                     </tr>
                     <tr>
                        <td>Distilled Water</td>
                        <td>10.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>265.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>285.6<span class="symbol">&#956;</span>l
                        </td>
                        <td>561<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Primer 1</td>
                        <td>0.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>10.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>11.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>22<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Primer 2</td>
                        <td>0.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>10.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>11.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>22<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Primer 3</td>
                        <td>0.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>10.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>11.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>22<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Primer 4</td>
                        <td>0.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>10.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>11.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>22<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>10x PCR Buffer</td>
                        <td>2.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>52.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>56.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>110<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>dNTPs 2mM</td>
                        <td>2.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>52.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>56.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>110<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Taq Polymerase</td>
                        <td>0.2 <span class="symbol">&#956;</span>l
                        </td>
                        <td>5.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>5.6<span class="symbol">&#956;</span>l
                        </td>
                        <td>11<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                  </table>
                  <p>Mix for 2 Primers (50 <span class="symbol">&#956;</span>M):
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td></td>
                        <td><span class="bold"><strong>1 sample</strong></span></td>
                        <td><span class="bold"><strong>26 samples</strong></span></td>
                        <td><span class="bold"><strong>28 samples</strong></span></td>
                        <td><span class="bold"><strong>48 samples</strong></span></td>
                     </tr>
                     <tr>
                        <td>Distilled Water</td>
                        <td>11<span class="symbol">&#956;</span>l
                        </td>
                        <td>286<span class="symbol">&#956;</span>l
                        </td>
                        <td>308<span class="symbol">&#956;</span>l
                        </td>
                        <td>528<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Primer 1</td>
                        <td>0.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>10.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>11.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>19.2<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Primer 2</td>
                        <td>0.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>10.4<span class="symbol">&#956;</span>l
                        </td>
                        <td>11.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>19.2<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>10x PCR Buffer</td>
                        <td>2.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>52.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>56<span class="symbol">&#956;</span>l
                        </td>
                        <td>96.0<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>dNTPs 2mM</td>
                        <td>2.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>52.0<span class="symbol">&#956;</span>l
                        </td>
                        <td>56<span class="symbol">&#956;</span>l
                        </td>
                        <td>96.0<span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                     <tr>
                        <td>Taq Poymerase</td>
                        <td>0.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>5.2<span class="symbol">&#956;</span>l
                        </td>
                        <td>5.6<span class="symbol">&#956;</span>l
                        </td>
                        <td>9.6 <span class="symbol">&#956;</span>l
                        </td>
                     </tr>
                  </table>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e870"></a>3.4.&nbsp;Mapping mutations using morphological markers in <span class="emphasis"><em>P. pacificus</em></span></h3>
                     </div>
                  </div>
               </div>
               <p>In addition to the molecular SSCP markers, we have isolated 40 morphological markers which can be used to map mutations to
                  the respective chromosomes (<a href="#bib6">Kenning et al., 2004</a>; <a href="#figure4">Figure 4</a>). The morphological markers were isolated in screens for mutations altering body morphology and were classified as &#8216;Dumpy&#8217;,
                  &#8216;Roller&#8217; and &#8216;Uncoordinated&#8217; based on their morphological phenotypes. To date, we have isolated 43 Dumpy mutations representing
                  12 genes and one roller gene. Though the screen is far from being saturated, these genes represent markers for each of the
                  <span class="emphasis"><em>P. pacificus</em></span> chromosomes. For nomenclature purposes, these dumpy and roller genes are described as <span class="emphasis"><em>Ppa-pdl-n or Ppa-prl-n</em></span>, where Ppa indicates the species &#8216;<span class="emphasis"><em>P. pacificus</em></span>&#8217; and &#8216;<span class="emphasis"><em>pdl</em></span> and <span class="emphasis"><em>prl</em></span>&#8217; stands for <span class="emphasis"><em>Pristionchus dumpy-like</em></span> and <span class="emphasis"><em>Pristionchus roller-like</em></span> respectively (<a href="#bib6">Kenning et al., 2004</a>; <a href="#bib18">Sommer et al., 1996</a>).
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="558">
<tr>
  <td align="center"><p><a href="Ppagenomicsfig4.jpg"><img src="Ppagenomicsfig4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 4. Morphological markers for each linkage group in <span class="emphasis"><em>P. pacificus</em></span> allowing mapping of mutations in <span class="emphasis"><em>P. pacificus</em></span>. Reprinted with permission from <a href="#bib6">Kenning et al. (2004)</a>. Copyright 2004, John Wiley and Sons.&nbsp;</b></p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e931"></a>4.&nbsp;Physical map of <span class="emphasis"><em>P. pacificus</em></span></h2>
                  </div>
               </div>
            </div>
            <p>The physical map of the <span class="emphasis"><em>P. pacificus</em></span> genome was generated by AFLP fingerprint analysis of BAC clones. Since most of the SSCP markers used to generate the genetic
               linkage map were derived from BAC ends, the physical genome map and the genetic map could be integrated.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e942"></a>4.1.&nbsp;AFLP-based BAC fingerprinting
                        </h3>
                     </div>
                  </div>
               </div>
               <p>To generate a physical map of the genome of <span class="emphasis"><em>P. pacificus</em></span>, 9000 BAC clones from the Hind III library were selected, including those BAC clones that had been used to generate the SSCP
                  markers for the genetic linkage map. All the BAC clones were AFLP fingerprinted using the primer combination Hind III/MseI
                  (+0/+0; i.e., no selective bases in the AFLP reaction).
               </p>
               <p>The resulting fingerprints were digitized and scanned for quality control using the program BACxtractor and scanned manually
                  for repetitive elements as these produce excessive numbers of bands (<a href="#bib20">Srinivasan et al., 2003</a>). To optimize contig assembly, a screening window of 260-825 bp was chosen to create the physical map from the AFLP fingerprint
                  data (<a href="#figure5">Figure 5A</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e958"></a>4.2.&nbsp;Image analysis of AFLP fingerprints and contig assembly
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The AFLP based fingerprints for more than 9,000 BAC clones generated were filtered for repeat sequences and bad fingerprinting.
                  On average, 25 AFLP bands per BAC clone were used for fingerprint analysis. Using image analysis, we chose 7747 BAC clones
                  out of the 9,000 clones as suitable for further analysis. These clones were then assembled using FPC v6.2 (<a href="#bib17">Soderlund et al., 1997</a>) to generate the physical map of <span class="emphasis"><em>P. pacificus</em></span> (<a href="#figure5">Figure 5B</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e972"></a>4.3.&nbsp;Integration of the genetic linkage map and physical map
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The genetic linkage map of <span class="emphasis"><em>P. pacificus</em></span> was generated using SSCP markers from BAC-end and EST sequences (<a href="#bib21">Srinivasan et al., 2002</a>) and the physical map of the <span class="emphasis"><em>P. pacificus</em></span> genome was assembled using AFLP fingerprints of individual BAC clones. This allows many of the SSCP markers on the genetic
                  map can be anchored to the physical map and vice versa. This integration led to 53 contigs containing one or more SSCP markers
                  on the <span class="emphasis"><em>P. pacificus</em></span> physical map (<a href="#figure6">Figure 6</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0">
<tr style="height: 738px">
  <td align="center"><p><a href="Ppagenomicsfig5.jpg"><img src="Ppagenomicsfig5_s.jpg" border="2" align="middle" alt=" figure 5"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 5.&nbsp;</b>A) AFLP fingerprint pattern of individual BAC clones using HindIII/Mse-I primer combination. Each lane represents the fingerprint
                        for a single BAC clone. The arrows at the top and bottom of the gel indicates the region of the fingerprint used for analysis.
                        B) FPC v6.2 contig window displaying a horizontal representation of a representative contig (<a href="#bib17">Soderlund et al., 1997</a>). Reprinted from <a href="#bib20">Srinivasan et al. (2003)</a>, Copyright 2003, with kind permission of Springer Science and Business Media.
                     </p>
                  </div>
               </div>
               <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="585">
<tr>
  <td align="center"><p><a href="Ppagenomicsfig6.jpg"><img src="Ppagenomicsfig6_s.jpg" border="2" align="middle" alt=" figure 6"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 6. An integrated map of <span class="emphasis"><em>P. pacificus</em></span> combining the genetic markers and the physical contigs.&nbsp;</b></p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1024"></a>5.&nbsp;Bioinformatic resources for <span class="emphasis"><em>P. pacificus</em></span></h2>
                  </div>
               </div>
            </div>
            <p>Two large-scale sequencing efforts catapulted <span class="emphasis"><em>P. pacificus</em></span> into the Genomic era. First, a high-coverage whole genome-sequencing project has been launched and will be completed in late
               2006. Second, <span class="emphasis"><em>P. pacificus</em></span> is part of a large-scale EST sequencing effort at the WUGSC (<a href="http://www.nematode.net" target="_top">www.nematode.net</a>; <a href="#bib24">Wylie et al., 2004</a>). These two projects constitute a steady flow of new genomics data. An integrated user-friendly view on these data plus complementary
               information such as the Genetic Map and Physical Map is provided via our genome portal: <a href="http://www.pristionchus.org" target="_top">www.pristionchus.org</a>.
            </p>
            <p>Our genome portal is the primary access point for all genome-centred data on <span class="emphasis"><em>P. pacificus</em></span>. The online resource is divided into six sections:
            </p>
            <div class="orderedlist">
               <ol type="1">
                  <li>
                     <p><span class="bold"><strong>Genome</strong></span> - This section offers a graphical interface to sequence assemblies of the <span class="emphasis"><em>Pristionchus</em></span> genome and feature annotation thereof. The Generic Genome Browser environment (<a href="#bib22">Stein et al., 2001</a>) is used for this purpose, as it is well known in the nematode community.
                     </p>
                  </li>
                  <li>
                     <p><span class="bold"><strong>BLAST</strong></span> - Similarity searches and single sequence retrieval is easily done on this part of the website.
                     </p>
                  </li>
                  <li>
                     <p><span class="bold"><strong>Genome Maps</strong></span> - Genetic and physical maps are located here. Both are interlinked and will be lined up with the genome assembly once it
                        is available.
                     </p>
                  </li>
                  <li>
                     <p><span class="bold"><strong>Batch Retrieval</strong></span> - Large user-defined data sets are made available via this interface.
                     </p>
                  </li>
                  <li>
                     <p><span class="bold"><strong>WikiOnchus</strong></span> - An online forum for the <span class="emphasis"><em>Pristionchus</em></span> community.
                     </p>
                  </li>
                  <li>
                     <p><span class="bold"><strong>People</strong></span> - an overview of labs and people working with/on <span class="emphasis"><em>Pristionchus pacificus</em></span>.
                     </p>
                  </li>
               </ol>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1094"></a>5.1.&nbsp;Genome viewer
                        </h3>
                     </div>
                  </div>
               </div>
               <p>This section will become available when the first genome annotation is released.</p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1099"></a>5.2.&nbsp;Similarity searches
                        </h3>
                     </div>
                  </div>
               </div>
               <p>All existing genomic sequence data are deposited in a searchable manner on our website. Genomic sequences can be searched
                  for local similarities with the BLAST form (<a href="#figure7">Figure 7</a>). Search parameters are explained by following the corresponding links. Search parameters can be altered to tailor an individual
                  search approach. Genome assembly constraints are easily checked with the Mapping Tool, which is found below the BLAST form
                  (<a href="#figure7">Figure 7</a>). Here, one obtains information related to genome order, for example like which reads form a Contig and why Contigs are linked
                  into a Supercontig. Single sequence records may be obtained by using the Download tool at the bottom of the page.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure7"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="Ppagenomicsfig7.jpg"><img src="Ppagenomicsfig7_s.jpg" border="2" align="middle" alt=" figure 7"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 7. BLAST search form.&nbsp;</b>Query page of <span class="emphasis"><em>Pristionchus</em></span> sequence databases. Search parameter settings may be changed in the corresponding fields. Parameter explanations are given
                        by following the blue links.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1126"></a>5.3.&nbsp;Genetic and physical map
                        </h3>
                     </div>
                  </div>
               </div>
               <p>We employ the CMap software (<a href="#bib2">Fang et al., 2003</a>) to display our genetic and physical map. The genetic map currently comprises 512 SSCP markers divided up into 6 large linkage
                  groups. The physical map constitutes a second map set encompassing 397 contigs. It was anchored on the genetic map since a
                  subset of genetic markers were derived from BAC ends (<a href="#figure8">Figure 8</a>).
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure8"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="Ppagenomicsfig8.jpg"><img src="Ppagenomicsfig8_s.jpg" border="2" align="middle" alt=" figure 8"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 8. Assembly mapping and single sequence download.&nbsp;</b>The Mapping tool delivers information as to the position of a sequence entity on the genome assembly. Individual assembly
                        contigs can be obtained with the Download function. This is particularly useful to retrieve target sequences from BLAST searches.
                     </p>
                  </div>
               </div>
               <div class="mediaobject" align="center"><a name="figure9"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0">
<tr style="height: 765px">
  <td align="center"><p><a href="Ppagenomicsfig9.jpg"><img src="Ppagenomicsfig9_s.jpg" border="2" align="middle" alt=" figure 9"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 9. Genetic map of chromosome I linked to contigs of the physical map.&nbsp;</b>The CMap software realizes an integrated view on the physical and genetic map. Features in red have correspondences in both
                        maps.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1162"></a>5.4.&nbsp;Data retrieval
                        </h3>
                     </div>
                  </div>
               </div>
               <p>This section will start to operate with the release of the first whole genome annotation.</p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1167"></a>5.5.&nbsp;WikiOnchus
                        </h3>
                     </div>
                  </div>
               </div>
               <p>WikiOnchus is an online forum on our favourite animal. Everybody is free to view forum contents ranging from methods and protocols
                  to basic introductory texts in a variety of languages. Web users are encouraged to register for active membership to be granted
                  access to edit pages. WikiOnchus is based on the MediaWiki system for which further information are available at <a href="http://www.mediawiki.org" target="_top">http://www.mediawiki.org</a>.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1175"></a>6.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>We would like to thank the whole nematode community for their continued interest in a &#8216;satellite&#8217; model system and Paul W.
               Sternberg and the anonymous reviewers for critical comments on the manuscript.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1180"></a>7.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Eizinger, A., and Sommer, R.J. (1997). The homeotic gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-39" target="_blank">lin-39</a></em></span> and the evolution of nematode epidermal cell fates. Science <span class="emphasis"><em>278</em></span>, 452&#8211;455.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9334302&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.278.5337.452" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Fang, Z., Polacco, M., Chen, S., Schroeder, S., Hancock, D., Sanchez, H., and Coe, E. (2003). cMap: the comparative genetic
                        map viewer. Bioinformatics <span class="emphasis"><em>19</em></span>, 416&#8211;417.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12584129&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/bioinformatics/btg012" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Felix, M.A., Hill, R.J., Schwarz, H., Sternberg, P.W., Sudhaus, W., and Sommer, R.J. (1999). <span class="emphasis"><em>Pristionchus pacificus</em></span>, a nematode with only three juvenile stages, displays major heterochronic changes relative to <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. R. Soc. Lond. B. Biol. Sci. <span class="emphasis"><em>266</em></span>, 1617&#8211;1621.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10501036&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1098/rspb.1999.0823" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Hodgkin, J. (1988). Sexual dimorphism and sex determination. In <span class="emphasis"><em>The nematode Caenorhabditis elegans.</em></span>, W. Wood, ed., Cold Spring Harbor Laboratory Press, pp. 243&#8211;279.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Hong, R.L., and Sommer, R.J. (2006) <span class="emphasis"><em>Pristionchus pacificus</em></span>: a well-rounded nematode. Bioessays <span class="emphasis"><em>28</em></span>, 651&#8211;659.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16700067&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/bies.20404" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Jungblut, B., and Sommer, R.J. (1998). The <span class="emphasis"><em>Pristionchus pacificus <a href="http://www.wormbase.org/db/gene/gene?name=mab-5" target="_blank">mab-5</a></em></span> gene is involved in the regulation of ventral epidermal cell fates. Curr. Biol. <span class="emphasis"><em>8</em></span>, 775&#8211;778.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9651682&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(98)70301-X" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Kenning, C., Kipping, I., and Sommer, R.J. (2004). Isolation of mutations with dumpy-like phenotypes and of collagen genes
                        in the nematode <span class="emphasis"><em>Pristionchus pacificus</em></span>. Genesis <span class="emphasis"><em>40</em></span>, 176&#8211;183.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15493014&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/gene.20084" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Koch, R., van Luenen, H.G., van der Horst, M., Thijssen, K.L., and Plasterk, R.H. (2000). Single nucleotide polymorphisms
                        in wild isolates of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome Res. <span class="emphasis"><em>10</em></span>, 1690&#8211;1696.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11076854&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.GR-1471R" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Lee, K.Z., Eizinger, A., Nandakumar, R., Schuster, S.C., and Sommer, R.J. (2003). Limited microsynteny between the genomes
                        of <span class="emphasis"><em>Pristionchus pacificus</em></span> and <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>31</em></span>, 2553&#8211;2560.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12736304&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkg359" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Manly, K.F., Cudmore, R.H., Jr., and Meer, J.M. (2001). Map Manager QTX, cross-platform software for genetic mapping. Mamm.
                        Genome <span class="emphasis"><em>12</em></span>, 930&#8211;932.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11707780&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s00335-001-1016-3" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Orita, M., Iwahana, H., Kanazawa, H., Hayashi, K., and Sekiya, T. (1989a). Detection of polymorphisms of human DNA by gel
                        electrophoresis as single-strand conformation polymorphisms. Proc. Natl. Acad. Sci. U.S.A. <span class="emphasis"><em>86</em></span>, 2766&#8211;2770.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2565038&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Orita, M., Suzuki, Y., Sekiya, T., and Hayashi, K. (1989b). Rapid and sensitive detection of point mutations and DNA polymorphisms
                        using the polymerase chain reaction. Genomics <span class="emphasis"><em>5</em></span>, 874&#8211;879.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2687159&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0888-7543(89)90129-8" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Pires-daSilva, A., and Sommer, R.J. (2004). Conservation of the global sex determination gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> in distantly related nematodes. Genes Dev. <span class="emphasis"><em>18</em></span>, 1198&#8211;1208.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15155582&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.293504" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Raddatz, G., Dehio, M., Meyer, T.F., and Dehio, C. (2001). PrimeArray: genome-scale primer design for DNA-microarray construction.
                        Bioinformatics <span class="emphasis"><em>17</em></span>, 98&#8211;99.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11222267&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/bioinformatics/17.1.98" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Rudel, D., Riebesell, M., and Sommer, R.J. (2005). Gonadogenesis in <span class="emphasis"><em>Pristionchus pacificus</em></span> and organ evolution: development, adult morphology and cell-cell interactions in the hermaphrodite gonad. Dev. Biol. <span class="emphasis"><em>277</em></span>, 200&#8211;221.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15572150&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2004.09.021" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Schlak, I., Eizinger,A., and Sommer, R.J. (1997). High rate of restriction fragment length polymorphisms between two populations
                        of the nematode <span class="emphasis"><em>Pristionchus pacificus</em></span> (Diplogastridae). J. Zool. Syst. Evol. <span class="emphasis"><em>35</em></span>, 137&#8211;142.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Sigrist, C.B., and Sommer, R.J. (1999). Vulva formation in <span class="emphasis"><em>Pristionchus pacificus</em></span> relies on continuous gonadal induction. Dev. Genes Evol. <span class="emphasis"><em>209</em></span>, 451&#8211;459.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10415322&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s004270050278" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Soderlund, C., Longden, I., and Mott, R. (1997). FPC: a system for building contigs from restriction fingerprinted clones.
                        Comput. Appl. Biosci. <span class="emphasis"><em>13</em></span>, 523&#8211;535.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9367125&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Sommer, R.J., Carta, L.K., Kim, S.Y., and Sternberg, P.W. (1996). Morphological, genetic and molecular description of <span class="emphasis"><em>Pristionchus pacificus</em></span> sp. n. (Nematoda: Neodiplogastridae). Fundam. Appl. Nematol. <span class="emphasis"><em>19</em></span>, 511&#8211;521.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Srinivasan, J., Pires-daSilva, A., Gutierrez, A., Zheng, M., Jungblut, B., Witte, H., Schlak, I., and Sommer, R.J. (2001).
                        Microevolutionary analysis of the nematode genus <span class="emphasis"><em>Pristionchus</em></span> suggests a recent evolution of redundant developmental mechanisms during vulva formation. Evol. Dev. <span class="emphasis"><em>3</em></span>, 229&#8211;240.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11478520&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1046/j.1525-142x.2001.003004229.x" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Srinivasan, J., Sinz, W., Jesse, T., Wiggers-Perebolte, L., Jansen, K., Buntjer, J., van der Meulen, M., and Sommer, R.J.
                        (2003). An integrated physical and genetic map of the nematode <span class="emphasis"><em>Pristionchus pacificus</em></span>. Mol. Genet. Genomics <span class="emphasis"><em>269</em></span>, 715&#8211;722.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12884007&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s00438-003-0881-8" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Srinivasan, J., Sinz, W., Lanz, C., Brand, A., Nandakumar, R., Raddatz, G., Witte, H., Keller, H., Kipping, I., Pires-DaSilva,
                        A. et al. (2002). A Bacterial Artificial Chromosome-Based Genetic Linkage Map of the Nematode <span class="emphasis"><em>Pristionchus pacificus</em></span>. Genetics <span class="emphasis"><em>162</em></span>, 129&#8211;134.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12242228&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Stein, L., Sternberg, P., Durbin, R., Thierry-Mieg, J., and Spieth, J. (2001). WormBase: network access to the genome and
                        biology of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>29</em></span>, 82&#8211;86.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11125056&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/29.1.82" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Wicks, S.R., Yeh, R.T., Gish, W.R., Waterston, R.H., and Plasterk, R.H. (2001). Rapid gene mapping in <span class="emphasis"><em>Caenorhabditis elegans</em></span> using a high density polymorphism map. Nat. Genet. <span class="emphasis"><em>28</em></span>, 160&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11381264&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/88878" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Wylie, T., Martin, J.C., Dante, M., Mitreva, M.D., Clifton, S.W., Chinwalla, A., Waterston, R.H., Wilson, R.K., and McCarter,
                        J.P. (2004). Nematode. net: a tool for navigating sequences from parasitic and free-living nematodes. Nucleic Acids Res. <span class="emphasis"><em>32</em></span>, D423&#8211;D426.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14681448&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkh010" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     	<span class="bibliomisc">Zheng, M., Messerschmidt, D., Jungblut, B., and Sommer, R.J. (2005). Conservation and diversification of Wnt signaling function
                        during the evolution of nematode vulva development. Nat. Genet. <span class="emphasis"><em>37</em></span>, 300&#8211;304.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15696167&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1512" target="_blank">Article</a>
                     
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e10" href="#d0e10">*</a></sup>Edited by Ralf J. Sommer. WormMethods editor, Victor Ambros. Last revised May 16, 2006. Published August 14, 2006. This chapter
                  should be cited as: Dieterich C., et al. <span class="emphasis"><em>Pristionchus pacificus</em></span> genomics: from genetics to genome sequence (August 14, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.116.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Christoph Dieterich et al. This is an open-access article distributed under the terms of the Creative Commons Attribution
                  License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source
                  are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e37" href="#d0e37">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:christoph.dieterich@tuebingen.mpg.de" target="_top">christoph.dieterich@tuebingen.mpg.de</a> or <a href="mailto:jsrini@caltech.edu" target="_top">jsrini@caltech.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

