<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Biochemistry and molecular biology
         	
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#biochem_ArticleID" title="Biochemistry and molecular biology&#xA;">
      <link rel="next" href="#d0e58" title="1.&nbsp;The yeast two-hybrid system">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="biochemmolbio.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="biochem_ArticleID"></a>Biochemistry and molecular biology<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     	
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Albertha J. M. Walhout, ed.<em><span class="remark">
                                 				<sup><a name="d0e31" href="#ftn.d0e31">&sect;</a></sup>
                                 			</span></em>
                           		</span><br></span><span class="affiliation"><span class="address">Program in Gene Function and Expression and Program in Molecular Medicine, University of Massachusetts Medical School, Worcester,
                           MA 01605 USA
                           <p></p></span></span></div>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Simon J. Boulton, ed.<em><span class="remark"><sup>&sect;</sup></span></em></span><br></span><span class="affiliation"><span class="address">DNA damage Response Laboratory, London Research Institute, Cancer Research UK, Blanche Lane, South Mimms, UK, EN6 3LD
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=288.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Biochemistry+and+molecular+biology" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e58">1. The yeast two-hybrid system</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e156">1.1. Creating DB-X baits</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e242">1.2. Testing self-activation by DB-X baits</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e278">1.3. Performing yeast two-hybrid screens</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e341">1.4. Yeast transformations</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e428">1.5. <span class="emphasis"><em>lacZ</em></span> assays</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e472">1.6. Identifying interacting preys</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e484">2. Tandem immuno-affinity purification</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e516">2.1. Creating the transgene</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e569">2.2. Growth and harvesting the transgenic line</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e617">2.3. Generating lysates</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e683">2.4. Preparing antibody coupled beads</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e704">2.5. Tandem immuno-affinity purification of protein complexes</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e751">2.6. Preparation of the sample for mass-spectrometry</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e776">3. Acknowledgments</a></span></dt>
               <dt><span class="sect1"><a href="#d0e784">4. References</a></span></dt>
            </dl>
         </div>
         <p>Several forward and reverse proteomic approaches are available that can be used to identify interaction partners for a protein
            of interest. Here we provide methods for identifying interacting partners by the yeast two-hybrid system (a reverse proteomic
            method) and by tandem immuno-affinity purification of protein complexes combined with mass spectrometry (a forward proteomic
            method).
         </p>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e58"></a>1.&nbsp;The yeast two-hybrid system
                     </h2>
                  </div>
               </div>
            </div>
            <p>The yeast two-hybrid system is a convenient reverse proteomic approach to identify and study protein-protein interactions
               (<a href="#bib4">Fields and Song, 1989</a>; <a href="#bib17">Vidal and Legrain, 1999</a>; <a href="#bib22">Walhout and Vidal, 2001</a>). The yeast two-hybrid system can be used with a single bait protein of interest. However, large datasets in which many <span class="emphasis"><em>C. elegans</em></span> proteins were used as baits are now also available (<a href="#bib1">Boulton et al., 2002</a>; <a href="#bib2">Davy et al., 2001</a>; <a href="#bib8">Li et al., 2004</a>; <a href="#bib15">Tewari et al., 2004</a>; <a href="#bib19">Walhout et al., 2002</a>; <a href="#bib18">Walhout et al., 2000</a>).
            </p>
            <p>The yeast two-hybrid system is based on the functional reconstitution of an intact transcription factor that activates reporter
               gene expression (<a href="#figure1">Figure 1</a>). Reporter gene expression can efficiently be selected for in yeast. As indicated by the name, the yeast two-hybrid system
               utilizes two hybrid proteins. The bait protein (X; <a href="#figure1">Figure 1</a>) is fused to the DNA binding domain (DB; <a href="#figure1">Figure 1</a>) of a transcription factor. The prey protein (Y; <a href="#figure1">Figure 1</a>) is fused to the transcription activation domain (AD; <a href="#figure1">Figure 1</a>) of a transcription factor. In our system, we use the yeast transcription factor Gal4. When two hybrid proteins are co-expressed
               in yeast and when X and Y can physically interact with each other, a functional transcription factor is reconstituted. This
               reconstituted transcription factor activates the expression of a set of reporter genes. To minimize the number of false positives
               identified, we use 3 reporter genes: <span class="emphasis"><em>HIS3</em></span>, <span class="emphasis"><em>URA3</em></span> and <span class="emphasis"><em>lacZ</em></span>. Expression of these genes is measured by growth on minimal media lacking histidine (for <span class="emphasis"><em>HIS3</em></span>), or uracil (<span class="emphasis"><em>URA3</em></span>), or by a colorimetric (&#8220;blue-white&#8221;) assay.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="biochemfig1.jpg"><img src="biochemfig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1. The yeast two-hybrid system.&nbsp;</b></p>
               </div>
            </div>
            <p>Performing yeast two-hybrid screens consists of the following steps: 1) creating a DB-X bait, 2) testing self-activation by
               the DB-X bait, 3) performing the yeast two-hybrid screen and 4) identifying the prey. These steps are described below in more
               detail. For additional information see <a href="#bib17">Vidal and Legrain (1999)</a> and <a href="#bib22">Walhout and Vidal (2001)</a>.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e156"></a>1.1.&nbsp;Creating DB-X baits
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Bait-encoding ORFs can be cloned into a DB-containing vector by regular, restriction-enzyme based cloning or by Gateway cloning.
                  For regular cloning, we use the plasmid pPC97 that has the DNA binding domain of Gal4 immediately upstream of a multiple cloning
                  site (<a href="#bib17">Vidal and Legrain, 1999</a>). We generally prefer to use Gateway cloning because many DB-ORF baits can be created simultaneously and in a standardized
                  and high-throughput manner (<a href="#bib6">Hartley et al., 2000</a>; <a href="#bib20">Walhout et al., 2000</a>; <a href="#bib21">Walhout et al., 2000</a>). After creating the DB-ORF clone(s) of interest, they need to be transformed into the yeast strain MaV103 (<a href="#bib16">Vidal, 1997</a>; <a href="#bib17">Vidal and Legrain, 1999</a>; <a href="#bib22">Walhout and Vidal, 2001</a>). A protocol for high-efficiency yeast transformations is provided below. Because DB-ORF-containing plasmids contain a <span class="emphasis"><em>LEU2</em></span> marker gene, transformants need to be selected on plates lacking leucine (Sc-Leu). Before proceeding with the actual yeast
                  two-hybrid screen, bait strains need to be examined for self-activation, i.e. their ability to activate reporter gene expression
                  in the absence of an AD-Y interaction partner (see below).
               </p>
               <p>For Gateway LR cloning the following reagents are required:</p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>An Entry clone from the ORFeome that contains the ORF(s) of interest (<a href="#bib23">Reboul et al., 2003</a>). These can be obtained from Dr. Marc Vidal's laboratory, see <a href="http://worfdb.dfci.harvard.edu/" target="_blank">http://worfdb.dfci.harvard.edu/</a>.
                        </p>
                     </li>
                     <li>
                        <p>A Destination vector containing the DNA binding domain of Gal4: (pDEST<sub>DB</sub>; <a href="#bib20">Walhout et al., 2000</a>). This vector can be obtained from Invitrogen, as part of their Gateway-compatible yeast two-hybrid system (catalog number:
                           10835-031).
                        </p>
                     </li>
                     <li>
                        <p>Gateway LR enzyme (Invitrogen, catalog number: 11791&#8211;020).</p>
                     </li>
                     <li>
                        <p>Competent DH5<span class="symbol">&#945;</span> bacteria with a transformation efficiency of &gt;10<sup>7</sup> colonies/<span class="symbol">&#956;</span>gDNA.
                        </p>
                     </li>
                     <li>
                        <p>LB/ampicillin plates (50 <span class="symbol">&#956;</span>g/ml).
                        </p>
                     </li>
                  </ol>
               </div>
               <p>Protocols for Gateway cloning reactions are described elsewhere (<a href="#bib20">Walhout et al., 2000</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e242"></a>1.2.&nbsp;Testing self-activation by DB-X baits
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>After transformation to create the DB-X bait strain, pick 12 individual yeast colonies and patch them onto a fresh Sc-Leu
                           plate. We patch 5 control strains at the bottom of each plate for comparison (see <a href="#bib22">Walhout and Vidal (2001)</a> for details).
                        </p>
                     </li>
                     <li>
                        <p>Incubate overnight at 30&deg;C.</p>
                     </li>
                     <li>
                        <p>Next day, replica plate the patches onto 5 Sc-Leu,-His + 3-amintotriazole (3AT) plates (containing 20, 40, 60, 80 and 100
                           mM 3AT). 3AT is a competitive inhibitor of the His3 enzyme and only baits that exhibit self-activation will be able to grow
                           in the presence of this compound. After replica plating, the 3AT-containing plates need to be replica cleaned 2 or 3 times,
                           until no yeast is visible (<a href="#bib22">Walhout and Vidal, 2001</a>).
                        </p>
                     </li>
                     <li>
                        <p>Next day, perform <span class="emphasis"><em>lacZ</em></span> assay (see below for protocol).
                        </p>
                     </li>
                     <li>
                        <p>Look at the 3AT plates on days 3, 4 and 5 after replica plating. Bait strains that grow on plates containing 100 mM 3AT cannot
                           be used in yeast two-hybrid experiments. For baits that can be used, use the lowest 3AT concentration at which no growth is
                           observed after 5 days in subsequent yeast two-hybrid screens.
                        </p>
                     </li>
                     <li>
                        <p>We also perform <span class="emphasis"><em>lacZ</em></span> assays to examine self-activation (see below). Patches that are blue correspond to self-activating baits.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e278"></a>1.3.&nbsp;Performing yeast two-hybrid screens
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>For yeast two-hybrid screens, the transformation protocol indicated below should be followed. For screens, we use 30 <span class="symbol">&#956;</span>g of a <span class="emphasis"><em>C. elegans</em></span> AD-Y cDNA library (<a href="#bib20">Walhout et al., 2000</a>). This library contains cDNAs obtained from worms of all developmental stages, males and dauer larvae and is highly comprehensive
                           (<a href="#bib11">Reboul et al., 2001</a>). The vector in which the cDNAs were cloned (pPC86, <a href="#bib16">Vidal, 1997</a>) contains a <span class="emphasis"><em>TRP1</em></span> marker for selection of yeast transformants on media lacking tryptophan. The transformation reactions should be directly
                           plated on Sc-Leu,-His,-Trp + 3AT plates to select for colonies in which a protein-protein interaction between a bait and prey
                           protein takes place.
                        </p>
                     </li>
                     <li>
                        <p>To estimate the transformation efficiency of the reaction, serial dilutions of one reaction (1/10; 1/100 and 1/1000) should
                           be plated on Sc-Leu,-Trp media (<a href="#bib22">Walhout and Vidal, 2001</a>). After incubating for 3 days at 30&deg;C, the number of colonies should be counted and the total number of transformants calculated.
                           We aim to screen at least 1 million colonies per bait.
                        </p>
                     </li>
                     <li>
                        <p>Incubate the screen plates for 4&#8211;5 days at 30&deg;C.</p>
                     </li>
                     <li>
                        <p>Pick any growing colonies (that are clearly bigger than background colonies, this varies from bait to bait) and patch with
                           a sterile toothpick on a fresh Sc-Leu,-Trp,-His + 3AT plate. Incubate overnight at 30&deg;C.
                        </p>
                     </li>
                     <li>
                        <p>Next day, replica plate the positives onto a fresh Sc-Leu,-Trp,-His + 3AT plate, a YEPD plate containing a nitrocellulose
                           filter for <span class="emphasis"><em>lacZ</em></span> assays (see below) and a Sc-Leu,-Trp,-Ura plate. Replica clean the 3AT containing plate and the Sc-Leu, -Trp,-Ura plate until
                           no yeast is visible. Incubate at 30&deg;C.
                        </p>
                     </li>
                     <li>
                        <p>Next day, perform the <span class="emphasis"><em>lacZ</em></span> assay (see below).
                        </p>
                     </li>
                     <li>
                        <p>Examine growth of potential positives on the other two plates after incubating for 3&#8211;5 days. Any growing patches are considered
                           positives. We only consider colonies that score positive for at least 2 reporter genes, e.g., they grow on 3AT-containing
                           media and are blue.
                        </p>
                     </li>
                     <li>
                        <p>Retest each interaction in fresh bait-containing yeast cells by PCR/Gap-repair (<a href="#bib22">Walhout and Vidal, 2001</a>) to minimize the number of false positives.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e341"></a>1.4.&nbsp;Yeast transformations
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Patch the required yeast strain on a YEPD plate and incubate overnight at 30&deg;C.</p>
                     </li>
                     <li>
                        <p>Next day, inoculate 500 ml liquid YEPD media with the relevant yeast strain to a density of OD<sub>600</sub> = ~0.1.
                        </p>
                     </li>
                     <li>
                        <p>Incubate at 30&deg;C while shaking vigorously until the culture reaches an OD<sub>600</sub> between 0.4&#8211;0.5.
                        </p>
                     </li>
                     <li>
                        <p>Harvest the cells by centrifugation for 5 minutes at 1,800 rpm at room temperature.</p>
                     </li>
                     <li>
                        <p>Decant the medium and wash the cells with 50 ml sterile water.</p>
                     </li>
                     <li>
                        <p>Resuspend the cells by pipetting up and down. Centrifuge 5min. at 1,800 rpm and RT - decant water.</p>
                     </li>
                     <li>
                        <p>Wash the cells in 10 ml TE/LiAc (10 mM Tris-HCl, pH 8.0; 1 mM EDTA; 0.1 M lithium acetate), centrifuge as described above.</p>
                     </li>
                     <li>
                        <p>Resuspend the cells in a volume of TE/LiAc solution corresponding to 10 times the final OD<sub>600</sub> value (i.e. for a 500 ml culture with an OD<sub>600</sub> = 0.5, 5 ml TE/LiAc solution should be used).
                        </p>
                     </li>
                     <li>
                        <p>Add 0.1 volume boiled (bring to a boil, and put on ice) salmon sperm DNA (10 mg/ml stock solution; Sigma, catalog number:
                           D9156).
                        </p>
                     </li>
                     <li>
                        <p>For each tranformation reaction put 50 <span class="symbol">&#956;</span>l of the cell suspension in a 1.5 ml eppendorf tube.
                        </p>
                     </li>
                     <li>
                        <p>Add the DNA to be transformed. For screens we use 30 reactions, each with 1<span class="symbol">&#956;</span>g library DNA.
                        </p>
                     </li>
                     <li>
                        <p>Add the DNA to be transformed and 300 <span class="symbol">&#956;</span>l TE/LiAc/PEG solution (made from a 10X stock solutions of TE and LiAc with the remaining volume being polyethylene glycol
                           Mw 3350).
                        </p>
                     </li>
                     <li>
                        <p>Resuspend carefully by pipetting.</p>
                     </li>
                     <li>
                        <p>Incubate at least 30 minutes at 30&deg;C.</p>
                     </li>
                     <li>
                        <p>Heat-shock the reactions for exactly 20 minutes at 42&deg;C.</p>
                     </li>
                     <li>
                        <p>Quickly centrifuge the tubes in a microcentrifuge, 5 seconds is sufficient.</p>
                     </li>
                     <li>
                        <p>Remove the supernatant.</p>
                     </li>
                     <li>
                        <p>Resuspend the cells in 350 <span class="symbol">&#956;</span>l sterile water and plate on the appropriate selective media (i.e., Sc-Leu,-Trp,-His with the appropriate concentration of
                           3AT; <a href="#bib22">Walhout and Vidal, 2001</a>).
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e428"></a>1.5.&nbsp;<span class="emphasis"><em>lacZ</em></span> assays
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Yeast should be grown overnight at 30&deg;C on a nitrocellulose filter that is placed on top of a YEPD plate (<a href="#bib22">Walhout and Vidal, 2001</a>). We use filters from Osmonics (catalog number: WP4HY13700).
                        </p>
                     </li>
                     <li>
                        <p>Next day, place 2 Whatman paper filters (Whatman catalog number: 1454125) in an empty 15 cm Petri dish.</p>
                     </li>
                     <li>
                        <p>Soak the filters with 6 ml Z-buffer (<a href="#bib22">Walhout and Vidal, 2001</a>) containing 11 <span class="symbol">&#956;</span>l <span class="symbol">&#946;</span>-mercaptoethanol and 100 <span class="symbol">&#956;</span>l 4% X-gal solution (dissolved in dimethylformamide). This should be done in a chemical hood.
                        </p>
                     </li>
                     <li>
                        <p>Take the nitrocellulose filter containing the yeast with a pair of tweezers and quick-freeze for 10 seconds in liquid nitrogen.
                           Thaw while holding with the tweezers.
                        </p>
                     </li>
                     <li>
                        <p>Carefully place the nitrocellulose on top of the Whatman filters, with the yeast patches facing up.</p>
                     </li>
                     <li>
                        <p>Incubate overnight at 37&deg;C.</p>
                     </li>
                     <li>
                        <p>Blue yeast patches correspond to self-activating baits (when only DB-X is used), or to potential positives (when DB-X is screened
                           against an AD-Y cDNA library).
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e472"></a>1.6.&nbsp;Identifying interacting preys
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The ORFs encoding interacting preys can be obtained from yeast by colony PCR using AD and Term primers. For a protocol see:
                  <a href="#bib22">Walhout and Vidal (2001)</a>. PCR products can be used in PCR/Gap-repair to retest potential interactions. PCR products corresponding to retesting positives
                  should be purified to remove the primers. Subsequently, PCR products can be sequenced using the AD primer to obtain an IST
                  (interaction sequence tag). ISTs can be blasted against Wormpep in Wormbase (<a href="http://www.wormbase.org/db/searches/blat" target="_blank">http://www.wormbase.org/db/searches/blat</a>) to identify the prey.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e484"></a>2.&nbsp;Tandem immuno-affinity purification
                     </h2>
                  </div>
               </div>
            </div>
            <p>Multi-protein complexes are the functional units of many cellular processes. Defining the components of these complexes, how
               they associate and their intrinsic biochemical activities provides a wealth of information about the context in which proteins
               operate (<a href="#bib3">Deshaies et al., 2002</a>; <a href="#bib5">Gavin et al., 2002</a>; <a href="#bib7">Ho et al., 2002</a>; <a href="#bib12">Rigaut et al., 1999</a>; <a href="#bib13">Shevchenko et al., 2002</a>). Here we describe a standardized method for purification of protein complexes from <span class="emphasis"><em>C. elegans</em></span> extracts by tandem immuno-affinity (<a href="#bib9">Polanowska et al., 2004</a>). This method describes the following steps: 1). Creating the transgene for expression of an epitope tagged form of your
               protein of interest (P.O.I.), 2). Growth and harvesting the transgenic line, 3). Generating lysates, 4) Preparing antibody
               coupled beads, 5) Tandem immuno-affinity purification of protein complexes, and 6) Preparation of the sample for mass-spectrometry
               (MS).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e516"></a>2.1.&nbsp;Creating the transgene
                        </h3>
                     </div>
                  </div>
               </div>
               <p>We have constructed pSB_GW::TAG, a Gateway destination vector that can be used to generate transgenic animals that express
                  any P.O.I. fused at the C-terminus to HA_8xHis_TEV_Myc epitopes, enabling associated protein complexes to be purified from
                  whole worm or embryo extracts by tandem immuno-affinity (<a href="#figure2">Figure 2A</a>; <a href="#bib9">Polanowska et al., 2004</a>). See (<a href="#bib21">Walhout et al., 2000</a>) for details on Gateway cloning. The promoter and coding region of a gene of interest (G.O.I.) are cloned into the Gateway
                  entry vector, sequenced, and then transferred by Gateway LR cloning into pSB_GW::TAG as previously described (<a href="#bib9">Polanowska et al., 2004</a>; <a href="#figure2">Figure 2A</a>). Stable transgenic animals generated by microparticle bombardment (<a href="#bib10">Praitis et al., 2001</a>) of <span class="emphasis"><em>unc-119(ed3)</em></span> mutants with pSB_G.O.I::TAG are then tested for transgene expression by western blotting using antibodies against either
                  1) the HA epitope (MAb12CA5), 2) 8xHis (anti-His antibody, Amersham BioSciences), or 3) the Myc epitope (MAb9E10).
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0">
                     <tr style="height: 702px">
  <td align="center"><p><a href="biochemfig2.jpg"><img src="biochemfig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Protein complex purification from <span class="emphasis"><em>C. elegans</em></span>.&nbsp;</b>(A) Gateway recombination is used to clone the promoter and coding region for a gene of interest (G.O.I) into p221 using the
                        BP reaction, and from p221 into pSB_GW::TAG using the LR reaction. An example of this is shown on the right in which universal
                        primers have been used to PCR amplify the promotor and coding region (P+C) for <a href="http://www.wormbase.org/db/get?name=Y41E3.9;class=Gene" target="_blank">Y41E3.9</a> from pSB_Y41E3.9::TAG (B) Microparticle bombardment is used to integrate the construct into <span class="emphasis"><em>unc-119(ed3)</em></span> mutants at low copy to generate a transgenic line expressing the protein of interest (P.O.I.) fused to C-terminal epitope
                        tags. The transgenic line is grown to high density in a BioFlo5000&reg; fermenter, harvested and extracts generated for purification
                        purposes(C). Tandem immuno-affinity chromatography over protein-A-MAb12CA5 and then protein-A-MAb9E10 is used to purify the
                        P.O.I.-associated protein complex, before complex components are identified by mass-spectrometry. An example on the right
                        shows a silver stained 4&#8211;12% gradient gel comparing the mock purification after (1.) HA and (2.) HA + Myc to the (3.) <a href="http://www.wormbase.org/db/get?name=Y41E3.9;class=Gene" target="_blank">Y41E3.9</a> associated protein complex after HA + MYC immunoaffinity.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e569"></a>2.2.&nbsp;Growth and harvesting the transgenic line
                        </h3>
                     </div>
                  </div>
               </div>
               <p>We have optimized large-scale production of transgenic lines in BioFlo5000&reg; fermenters (New Brunswick Scientific) or alternatively,
                  smaller scale cultures grown in S-basal may yield sufficient material for purification purposes. See elsewhere for details
                  (<a href="#bib9">Polanowska et al., 2004</a>). The weight of worms required for purifying a given protein complex to sufficient levels for protein identification by MS
                  will depend on the level of expression from the genes promotor and must be determined for each transgene. The complexes we
                  have studied so far function in the DNA damage response and are expressed at low levels in all proliferating cells, but are
                  undetectable in somatic tissues. For all complexes (n=5) studied approximately 50g of biomass was required to obtained sufficient
                  protein for MS.
               </p>
               <p>To prepare the worms for lysis:</p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Wash the worms twice in 4x the pellet volume of cold 0.1M NaCl.</p>
                     </li>
                     <li>
                        <p>Centrifugate at 4,000 rpm in 30% sucrose at 4&deg;C for 5 minutes to remove residual bacteria and dead worms (<a href="#figure2">Figure 2B</a>).
                        </p>
                     </li>
                     <li>
                        <p>Collect the live worms from the top of the sucrose solution.</p>
                     </li>
                     <li>
                        <p>Wash a third time with 4x the pellet volume of cold 0.1M NaCl.</p>
                     </li>
                     <li>
                        <p>Resuspend the pellet in twice the worm pellet volume of CSK lysis buffer [100mM Pipes (pH 6.0), 100mM NaCl, 3mM MgCl<sub>2</sub>, 1mM EGTA, 1mM DTT, 1mM PMSF, 0.3M sucrose, 0.5% triton X 100, complete (Roche) protease inhibitor tablets, and phosphatase
                           inhibitors: 2mM <span class="symbol">&#946;</span>-glycerophosphate, 5mM NaF, 1mM Na<sub>3</sub>VO<sub>4</sub>, 0.1 <span class="symbol">&#956;</span>M okadaic acid].
                        </p>
                     </li>
                     <li>
                        <p>Freeze down in 50 ml aliquots on dry ice and store at -80&deg;C.</p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e617"></a>2.3.&nbsp;Generating lysates
                        </h3>
                     </div>
                  </div>
               </div>
               <p>To generate whole worm extracts perform the following:</p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Rapidly thaw the aliquot(s) in a 37&deg;C water bath.</p>
                     </li>
                     <li>
                        <p>Re-freeze on dry ice then re-thaw as before.</p>
                     </li>
                     <li>
                        <p>Dounce the suspension 6 times in a homogenizer to break up the material.</p>
                     </li>
                     <li>
                        <p>Clarify the mixture by centrifugation at 4,500 rpm and collect the soluble supernatant fraction (S).</p>
                     </li>
                     <li>
                        <p>To extract the chromatin fraction (C), resuspend the pellet in half the pellet volume of CSK.</p>
                     </li>
                     <li>
                        <p>Add micrococcal nuclease (Roche) to a final concentration of 3U/<span class="symbol">&#956;</span>l.
                        </p>
                     </li>
                     <li>
                        <p>Incubate at 20&deg;C with gentle shaking for 30 minutes.</p>
                     </li>
                     <li>
                        <p>Stop the reaction by adding EDTA and EGTA to a final concentration of 5mM.</p>
                     </li>
                     <li>
                        <p>Collection the chromatin fraction supernatant by high-speed centrifugation at 40,000 rpm.</p>
                     </li>
                  </ol>
               </div>
               <p>Lysates can also be prepared from embryos:</p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Resuspend the embryo pellet in twice the pellet volume of homogenization buffer (50 mM HEPES at pH 7.6, 200 mM KCl, 1 mM EDTA,
                           1 mM EGTA, 0.2% Triton X-100, 5% glycerol, complete (Roche) protease inhibitor tablets, and phosphatase inhibitors: 2mM <span class="symbol">&#946;</span>-glycerophosphate, 5mM NaF, 1mM Na<sub>3</sub>VO<sub>4</sub>, 0.1 <span class="symbol">&#956;</span>M okadaic acid).
                        </p>
                     </li>
                     <li>
                        <p>Sonicate on ice until the mixture has lost its viscosity.</p>
                     </li>
                     <li>
                        <p>Centrifuge at 5,000<span class="emphasis"><em>g</em></span> at 4&deg;C for 5 min and collect the soluble extract (S).
                        </p>
                     </li>
                     <li>
                        <p>The chromatin bound fraction (C) is prepared as above.</p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e683"></a>2.4.&nbsp;Preparing antibody coupled beads
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Incubate 1 ml sepharose protein-A beads (Pharmacia) at 20&deg;C with mouse ascite (2 mg of MAb12CA5(HA) and MAb9E10 (Myc) antibodies).</p>
                     </li>
                     <li>
                        <p>Wash the beads with 10 volumes of 0.2 M sodium borate (pH 9.0).</p>
                     </li>
                     <li>
                        <p>Add dimethylpimelimidate dihydrochloride (Sigma) to a final concentration of 2 mM to couple the antibody to the beads.</p>
                     </li>
                     <li>
                        <p>Incubate for 30 minutes at 20&deg;C.</p>
                     </li>
                     <li>
                        <p>Wash the beads with 5 volumes of 0.1 M Tris-HCl (pH 8.0) and 10 volumes of PBS to stop the coupling reaction.</p>
                     </li>
                  </ol>
               </div>
               <p>The coupled beads are stored at 4&deg;C in two volumes of PBS in 0.5% azide. Prior to purification, the beads should be washed
                  in B0.1 buffer [50 mM Tris (pH 7.3), 0.1 M KCl, 1 mM EDTA, 10% glycerol].
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e704"></a>2.5.&nbsp;Tandem immuno-affinity purification of protein complexes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The first immuno-affinity step:</p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Batch absorb at 4&deg;C for 6 hours with 60 <span class="symbol">&#956;</span>l HA coupled protein-A beads for each ml of extract (<a href="#figure1">Figure 1</a>C).
                        </p>
                     </li>
                     <li>
                        <p>Transfer the beads to a BioRad column.</p>
                     </li>
                     <li>
                        <p>Wash three times with 5 volumes of B0.1 buffer [supplemented with 1 mM DTT, 1mM PMSF, 2mM <span class="symbol">&#946;</span>-glycerophosphate, 5 mM NaF, 1 mM Na3VO4, 1 <span class="symbol">&#956;</span>M okadaic acid and complete protease inhibitor tablets].
                        </p>
                     </li>
                     <li>
                        <p>Elute the bound complex by resuspending the beads in one volume of B0.1 buffer containing 5 <span class="symbol">&#956;</span>g/ml of double HA peptide.
                        </p>
                     </li>
                     <li>
                        <p>Incubate at 4&deg;C for 2 hrs.</p>
                     </li>
                  </ol>
               </div>
               <p>Using the same conditions as above, the HA eluate is subjected to a second immuno-affinity step using Myc coupled protein-A
                  beads (<a href="#figure2">Figure 2</a>C). Elution from Myc-beads is inefficient using Myc-peptide so recombinant TEV protease (Invitrogen) is used to cleave off
                  the Myc epitope. The cleavage reaction is performed at 20&deg;C for 4 hours at a concentration of 1 <span class="symbol">&#956;</span>l TEV per 50 <span class="symbol">&#956;</span>l of HA eluate. The HA-Myc eluate can be further concentrated by binding to Ni-agarose beads followed by elution using a 20&#8211;500mM
                  imidazole gradient in B0.1 buffer.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e751"></a>2.6.&nbsp;Preparation of the sample for mass-spectrometry
                        </h3>
                     </div>
                  </div>
               </div>
               <p>To minimize contamination with keratins replace gloves frequently. Also, resolve purified protein complexes on Nu-PAGE 4&#8211;12%
                  pre-poured gels (Invitrogen) and stain with either Coomassie stain or silver stain (SilverQuest, Invitrogen). Bands are excised
                  from the gel and prepared for MS analysis as described previously (<a href="#bib14">Shevchenko et al., 1996</a>). Coomassie stainable levels of protein should be sufficient for protein identification by MS but it is important to discuss
                  sample preparation with your MS lab/facility as they may have specific requirements. To obtain a protein ID, search the databases
                  at <a href="http://wolf.bms.umist.ac.uk/mapper" target="_blank">PepMAPPER</a> and <a href="http://www.matrixscience.com/" target="_blank">MASCOT</a>  with the peak masses obtained from MS. Note that mock tandem immuno-affinity purification from extracts made from an untagged
                  <span class="emphasis"><em>C. elegans</em></span> strain occasionally gives two bands, one of which is <a href="http://www.wormbase.org/db/seq/protein?name=HSP-70;class=protein" target="_blank">HSP-70</a> (<a href="http://www.wormbase.org/db/get?name=C12C8.1;class=Gene" target="_blank">C12C8.1</a>; data not shown).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e776"></a>3.&nbsp;Acknowledgments
                     </h2>
                  </div>
               </div>
            </div>
            <p>SB would like to thank Biotechiques for permitting the reproducion of some of the work published in Polanowska et al, 2004,
               Biotechniques <span class="emphasis"><em>36</em></span>:778&#8211;780, 782. AJMW is funded by NCI (R33 CA097516&#8211;02), the Worcester Foundation and a Diabetes Endocrinology Research Center
               Grant 5 P30 DK32520. SB is funded by Breast Cancer Campaign (GA3221) and Cancer Research UK.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e784"></a>4.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Boulton, S.J., Gartner, A., Reboul, J., Vaglio, P., Dyson, N., Hill, D.E., and Vidal, M. (2002). Combined functional genomic
                        maps of the <span class="emphasis"><em>C. elegans</em></span> DNA damage response. Science <span class="emphasis"><em>295</em></span>, 127&#8211;131.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11778048&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1065986" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Davy, A., Bello, P., Thierry-Mieg, N., Vaglio, P., Hitti, J., Doucette-Stamm, L., Thierry-Mieg, D., Reboul, J., Boulton, S.,
                        Walhout, A.J., et al. (2001). A protein-protein interaction map of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> 26S proteasome. EMBO Rep. <span class="emphasis"><em>2</em></span>, 821&#8211;828.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11559592&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/embo-reports/kve184" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Deshaies, R.J., Seol, J.H., McDonald, W.H., Cope, G., Lyapina, S., Shevchenko, A., Verma, R., and Yates, J.R., III (2002).
                        Charting the protein complexome in yeast by mass spectrometry. Mol. Cell Proteomics <span class="emphasis"><em>1</em></span>, 3&#8211;10.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12096135&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1074/mcp.R100001-MCP200" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Fields, S., and Song, O. (1989). A novel genetic system to detect protein-protein interactions. Nature <span class="emphasis"><em>340</em></span>, 245&#8211;246.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2547163&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/340245a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Gavin, A.C., Bosche, M., Krause, R., Grandi, P., Marzioch, M., Bauer, A., Schultz, J., Rick, J.M., Michon, A.M., Cruciat,
                        C.M., et al. (2002). Functional organization of the yeast proteome by systematic analysis of protein complexes. Nature <span class="emphasis"><em>415</em></span>, 141&#8211;147.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11805826&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/415141a" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hartley, J.L., Temple, G.F., and Brasch, M.A. (2000). DNA cloning using in vitro site-specific recombination. Genome Res.
                        <span class="emphasis"><em>10</em></span>, 1788&#8211;1795.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11076863&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.143000" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ho, Y., Gruhler, A., Heilbut, A., Bader, G.D., Moore, L., Adams, S.L., Millar, A., Taylor, P., Bennett, K., Boutilier, K.,
                        et al. (2002). Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry. Nature <span class="emphasis"><em>415</em></span>, 180&#8211;183.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11805837&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/415180a" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Li, S., Armstrong, C.M., Bertin, N., Ge, H., Milstein, S., Boxem, M., Vidalain, P.O., Han, J.D., Chesneau, A., Hao, T., et
                        al. (2004). A map of the interactome network of the metazoan <span class="emphasis"><em>C. elegans</em></span>. Science <span class="emphasis"><em>303</em></span>, 540&#8211;543.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14704431&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1091403" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Polanowska, J., Martin, J.S., Fisher, R., Scopa, T., Rae, I., and Boulton, S.J. (2004). Tandem immunoaffinity purification
                        of protein complexes from <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Biotechniques <span class="emphasis"><em>36</em></span>, 778&#8211;780, 782.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15152596&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Praitis, V., Casey, E., Collar, D., and Austin, J. (2001). Creation of low-copy integrated transgenic lines in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>157</em></span>, 1217&#8211;1226.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11238406&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                              <span class="bibliomisc">Reboul, J., Vaglio, P., Rual, J.F., Lamesch, P., Martinez, M., Armstrong, C.M., Li, S., Jacotot, L., Bertin, N., Janky, R.,
                        Moore, T., Hudson, J.R., Jr., Hartley, J.L., Brasch, M.A., Vandenhaute, J., Boulton, S., Endress, G.A., Jenna, S., Chevet,
                        E., Papasotiropoulos, V., Tolias, P.P., Ptacek, J., Snyder, M., Huang, R., Chance, M.R., Lee, H., Doucette-Stamm, L., Hill,
                        D.E., and Vidal, M. (2003). <span class="emphasis"><em>C. elegans</em></span> ORFeome version 1.1: experimental verification of the genome annotation and resource for proteome-scale protein expression.
                        Nat. Genet. <span class="emphasis"><em>34</em></span>, 35&#8211;41.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12679813&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1140" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Reboul, J., Vaglio, P., Tzellas, N., Thierry-Mieg, N., Moore, T., Jackson, C., Shin-i, T., Kohara, Y., Thierry-Mieg, D., Thierry-Mieg,
                        J., et al. (2001). Open-reading-frame sequence tags (OSTs) support the existence of at least 17, 300 genes in <span class="emphasis"><em>C. elegans</em></span>. Nat. Genet. <span class="emphasis"><em>27</em></span>, 332&#8211;336.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11242119&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/85913" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rigaut, G., Shevchenko, A., Rutz, B., Wilm, M., Mann, M., and Seraphin, B. (1999). A generic protein purification method for
                        protein complex characterization and proteome exploration. Nat. Biotechnol. <span class="emphasis"><em>17</em></span>, 1030&#8211;1032.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10504710&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/13732" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Shevchenko, A., Schaft, D., Roguev, A., Pijnappel, W.W., and Stewart, A.F. (2002). Deciphering protein complexes and protein
                        interaction networks by tandem affinity purification and mass spectrometry: analytical perspective. Mol. Cell Proteomics <span class="emphasis"><em>1</em></span>, 204&#8211;212.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12096120&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1074/mcp.M200005-MCP200" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Shevchenko, A., Wilm, M., Vorm, O., and Mann, M. (1996). Mass spectrometric sequencing of proteins silver-stained polyacrylamide
                        gels. Anal. Chem. <span class="emphasis"><em>68</em></span>, 850&#8211;858.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8779443&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1021/ac950914h" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Tewari, M., Hu, P.J., Ahn, J.S., Ayivi-Guedehoussou, N., Vidalain, P.O., Li, S., Milstein, S., Armstrong, C.M., Boxem, M.,
                        Butler, M.D., et al. (2004). Systematic interactome mapping and genetic perturbation analysis of a <span class="emphasis"><em>C. elegans</em></span> TGF-beta signaling network. Mol. Cell. <span class="emphasis"><em>13</em></span>, 469&#8211;482.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14992718&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(04)00033-4" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Vidal, M. (1997). The reverse two-hybrid system. In: The Yeast Two-Hybrid System, P. Bartels and S. Fields, Eds. Oxford University
                        Press: New-York, p. 109&#8211;147.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Vidal, M., and Legrain, P. (1999). Yeast forward and reverse &#8216;n&#8217;-hybrid systems. Nucleic Acids Res. <span class="emphasis"><em>27</em></span>, 919&#8211;929.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9927722&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/27.4.919" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Walhout, A.J., Boulton, S.J., and Vidal, M. (2000). Yeast two-hybrid systems and protein interaction mapping projects for
                        yeast and worm. Yeast <span class="emphasis"><em>17</em></span>, 88&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10615043&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Walhout, A.J., Reboul, J., Shtanko, O., Bertin, N., Vaglio, P., Ge, H., Lee, H., Doucette-Stamm, L., Gunsalus, K.C., Schetter,
                        A.J., et al. (2002). Integrating interactome, phenome, and transcriptome mapping data for the <span class="emphasis"><em>C. elegans</em></span> germline. Curr. Biol. <span class="emphasis"><em>12</em></span>, 1952&#8211;1958.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12445390&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)01279-4" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Walhout, A.J., Sordella, R., Lu, X., Hartley, J.L., Temple, G.F., Brasch, M.A., Thierry-Mieg, N., and Vidal, M. (2000). Protein
                        interaction mapping in <span class="emphasis"><em>C. elegans</em></span> using proteins involved in vulval development. Science <span class="emphasis"><em>287</em></span>, 116&#8211;122.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10900455&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.287.5450.116" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Walhout, A.J., Temple, G.F., Brasch, M.A., Hartley, J.L., Lorson, M.A., van den Heuvel, S., and Vidal, M. (2000). GATEWAY
                        recombinational cloning: application to the cloning of large numbers of open reading frames or ORFeomes. Methods Enzymol.
                        <span class="emphasis"><em>328</em></span>, 575&#8211;592.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11075367&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Walhout, A.J., and Vidal, M. (2001). High-throughput yeast two-hybrid assays for large-scale protein interaction mapping.
                        Methods <span class="emphasis"><em>24</em></span>, 297&#8211;306.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11403578&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/meth.2001.1190" target="_blank">Article</a>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised April 11, 2005. Published February 13, 2006. This chapter should be cited as: Walhout,
                  A.J.M. and Boulton, S.J. Biochemistry and molecular biology (February 13, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.86.1, <a href="http://www.wormbook.org" target="_blank">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Albertha J. M. Walhout and Simon J. Boulton. This is an open-access article distributed under the terms of the Creative
                  Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original
                  author and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e31" href="#d0e31">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:marian.walhout@umassmed.edu" target="_blank">marian.walhout@umassmed.edu</a> or <a href="mailto:simon.boulton@cancer.org.uk" target="_blank">simon.boulton@cancer.org.uk</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

