<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic balancers
         			
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#EdB_ArticleID" title="Genetic balancers&#xA;&#xA;">
      <link rel="next" href="#sec1" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="geneticbalancers.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="EdB_ArticleID"></a>Genetic balancers<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     			<sup><a name="d0e21" href="#ftn.d0e21">&sect;</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Mark L. Edgley,<sup>1,</sup><em><span class="remark"><sup><a name="d0e59" href="#ftn.d0e59">&#8224;</a></sup></span></em> David L. Baillie,<sup>2,</sup><em><span class="remark"><sup><a name="d0e59" href="#ftn.d0e59">&#8224;</a></sup></span></em> Donald L. Riddle,<sup>1,</sup><em><span class="remark"><sup><a name="d0e59" href="#ftn.d0e59">&#8224;</a></sup></span></em> and Ann M. Rose<sup>3,</sup><em><span class="remark">
                                 					<sup><a name="d0e59" href="#ftn.d0e59">&#8224;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup>Michael Smith Laboratories, University of British Columbia, </span><span class="address">Vancouver, BC Canada V6T 1Z4
                           <p></p></span></span><span class="affiliation"><span class="orgname"><sup>2</sup>Institute of Molecular Biology and Biochemistry, Simon Fraser University Burnaby, </span><span class="address">BC Canada V5A 1S6
                           <p></p></span></span><span class="affiliation"><span class="orgname"><sup>3</sup>Department of Medical Genetics, University of British Columbia Vancouver, </span><span class="address">British Columbia Canada V6T lZ3
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward genetics and genetic mapping
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=299.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Genetic+balancers" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. Types of balancers</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec2_1">2.1. Translocations</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_2">2.2. Duplications</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_3">2.3. Transgenes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_4">2.4. Inversions</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_5">2.5. Other crossover suppressors</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_6">2.6. Deficiencies</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec3">3. Practical considerations of balancer use</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec3_1">3.1. Characteristics of good balancers</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_2">3.2. Genetically marked balancers</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_3">3.3. Crossing schemes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_4">3.4. Accumulation of spontaneous mutations</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_5">3.5. Exceptional segregants and balancer breakdown</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_6">3.6. Use of balancers in mutant screens</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3_7">3.7. Screens for transposon-induced mutations</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec4">4. The balancer map</a></span></dt>
               <dt><span class="sect1"><a href="#sec5">5. A field guide to balancers</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec5_1">5.1. Explanation of descriptors</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_2">5.2. The balancers</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec6">6. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>Genetic balancers are genetic constructs or chromosomal rearrangements that allow lethal or sterile mutations to be stably
               maintained in heterozygotes. In this chapter we use the term balancer primarily to refer to chromosomal duplications or rearrangements
               that suppress crossing over. In addition, we define lethal as any mutation that blocks survival or reproduction. Phenotypes
               associated with lethal mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span> range from egg or larval lethality to adult sterility and maternal effect lethality, and can include conditional effects
               such as temperature sensitivity. The number of essential genes in <span class="emphasis"><em>C. elegans</em></span> (those identified by lethal mutations) may range as high as 7000 according to genetic estimates (<a href="#bib12">Clark, D.V., Rogalski, T.M., Donati, L.M., and Baillie, D.L., 1988</a>; <a href="#bib33">Howell, A.M., and Rose, A.M., 1990</a>; <a href="#bib36">Johnsen, R.C., and Baillie, D.L., 1991</a>). Thus, lethal mutations constitute a rich source of information about basic biological processes in this nematode.
            </p>
            <p>This chapter assumes the reader is somewhat familiar with <span class="emphasis"><em>C. elegans</em></span> genetic nomenclature. A few basic definitions are given here,<sup><a name="d0e126" href="#ftn.d0e126">&#8225;</a></sup> but for full reference see <a href="#bib31">Horvitz, H.R., Brenner, S., Hodgkin, J., and Herman, R.K. (1979)</a>.
            </p>
            <p>Maintenance of mutations is an important aspect of <span class="emphasis"><em>C. elegans</em></span> genetics. Genetic strains carrying non-conditional recessive lethal mutations, at present the largest class of lethal, cannot
               be kept as homozygotes, and in heterozygotes the mutations can be lost easily through segregation unless there is a means
               to identify the heterozygotes that carry them. Maintenance of the heterozygous genotype from one generation to the next requires
               selection of heterozygous individuals, a task that becomes burdensome if more than a few strains must be maintained. For example,
               the self progeny of an unmarked recessive lethal heterozygote (<span class="emphasis"><em>let-x/+</em></span>) are either arrested <span class="emphasis"><em>let-x</em></span> homozygotes (<span class="emphasis"><em>let-x/let-x</em></span>), <span class="emphasis"><em>let-x</em></span> heterozygotes (<span class="emphasis"><em>let-x/+</em></span>), or wild-type animals (<span class="emphasis"><em>+/+</em></span>) that cannot be distinguished phenotypically from heterozygotes. Many phenotypically wild-type animals must be selected each
               generation to ensure the propagation of heterozygotes, and the presence of the lethal mutation must be confirmed through direct
               observation of <span class="emphasis"><em>let-x</em></span> homozygotes. If the normal homologue carries a morphological marker, <span class="emphasis"><em>in trans</em></span> to the lethal mutation, heterozygotes can be reliably identified; however, loss of the lethal can still occur through recombination
               and subsequent segregation. For example, in a <span class="emphasis"><em>let+/+dpy</em></span> heterozygote, a crossover between the <span class="emphasis"><em>let</em></span> and <span class="emphasis"><em>dpy</em></span> loci produces two recombinant chromosomes, one that carries both mutations and one that is completely wild type. A zygote
               carrying the wild-type recombinant chromosome and a parental <span class="emphasis"><em>dpy</em></span> chromosome (<span class="emphasis"><em>+/dpy</em></span>) will have a wild-type phenotype, and will produce wild-type and Dpy progeny just as the original heterozygote, but the desired
               lethal will no longer be present in the strain. Thus, the presence of the lethal in each generation still must be confirmed
               through direct observation or be deduced through the ratio of wild-type to Dpy progeny, which is very tedious (about 2:1 in
               <span class="emphasis"><em>let+/+dpy</em></span> heterozygotes and 3:1 in <span class="emphasis"><em>+/dpy</em></span> heterozygotes).
            </p>
            <p>Clearly, any sort of large-scale isolation and analysis of lethal mutations requires more effective methods to prevent their
               loss. The investigator must have an easy way to distinguish between progeny heterozygous for a lethal-bearing chromosome and
               those homozygous for the nonlethal homologue and, further, must be able either to detect directly when recombination has occurred
               between the lethal mutation and marker mutations or to reduce the incidence of recombination to a negligible level. A balancer
               provides these functions.
            </p>
            <p>Many different genetic constructs (genotypes) have been used to balance lethal mutations, but they have the same basic characteristics:
               (1) heterozygotes possess a unique phenotype (wild type or mutant) so that they can be selected reliably; (2) progeny phenotypes
               allow the investigator to tell when recombination has occurred; (3) close proximity of markers increases the degree of balancing
               by decreasing the frequency of recombination between lethals and markers; and (4) in the majority of cases, common visible
               markers linked to the lethals ease all phases of analysis.
            </p>
            <p>Linking lethals to morphological markers is especially helpful for analysis, as it is much easier to score for the absence
               of a particular morphological phenotype than to score arrested progeny. Furthermore, linked markers facilitate two-factor
               mapping, and markers common to an entire set of lethals allow rapid <span class="emphasis"><em>inter se</em></span> complementation tests. For example, recombination frequency can be measured between a lethal mutation and a linked marker
               mutation by scoring viable marker homozygotes among the progeny segregated from a heterozygote. Complementation between two
               lethal mutations can be assayed by performing a cross to construct animals heterozygous for both lethals and the same marker
               mutation, and scoring for marker homozygotes. Presence of these homozygotes indicates complementation, and absence indicates
               failure to complement.
            </p>
            <p>Three examples of the use of simple genotypes to balance lethal mutations follow. To maintain newly generated deficiencies
               on linkage group (LG) III that were lethal as homozygotes, <a href="#bib18">Greenwald, I.S., and Horvitz, H.R. (1980</a>) kept them as heterozygotes over an <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a> <a href="http://www.wormbase.org/db/gene/gene?name=dpy-17" target="_blank">dpy-17</a></em></span> chromosome. These animals were Unc-93 in phenotype because the deficiencies deleted <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a></em></span>, and they segregated Unc-93, Unc-93 Dpy-17, and lethal (deficiency homozygote) progeny. The deficiencies could be maintained
               by picking Unc animals, as the recombination frequency between <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-17" target="_blank">dpy-17</a></em></span> and the deficiency endpoints was low. In a second example, <a href="#bib49">Moerman, D.G., and Baillie, D.L. (1979</a>) isolated lethals linked to a conditionally dominant <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a></em></span> mutation, which causes dominant twitching in the presence of 1% nicotine. Conditional dominance results in a unique heterozygous
               phenotype. In screens for new mutations, worms heterozygous for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a></em></span> were screened for the absence or near absence of fertile adult Unc-22 progeny. Once obtained, these lethals could be maintained
               by picking animals that moved normally but were induced to twitch in 1% nicotine. Crossing over between a lethal and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a></em></span> was easily detected by scoring the rare Unc-22 progeny in the absence of nicotine. The closer the lethal was to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a></em></span>, the less often recombinational loss occurred. In a third example, <a href="#bib58">Rose, A.M., and Baillie, D.L. (1980</a>) mutagenized a phenotypically wild-type strain of genotype <span class="emphasis"><em>+unc-15+/dpy-14+unc-13</em></span> and screened for lethal mutations linked to the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-14" target="_blank">dpy-14</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a> I</em></span>chromosome. The new lethals were maintained simply by picking wild-type animals (heterozygotes) and checking to see that they
               gave wild-type and Unc-l5 progeny but no Dpy-14 Unc-13 progeny. Lethals were balanced because of the short genetic distances.
               In all cases, recombination was easily detected by the presence of Dpy, Unc, or DpyUnc recombinant animals among the self
               progeny of a heterozygote. The types and numbers of recombinant progeny gave distance and gene order relative to the markers.
            </p>
            <p>A drawback of these methods is that they are not easily adapted for the isolation of large numbers of lethal mutations over
               large genetic regions. They may require many different sets of appropriate marker mutations, which either may be too difficult
               to score or may not exist. A more sophisticated approach makes use of heterozygous chromosomal rearrangements adapted for
               use as balancers. There are two types of balancing rearrangements: (1) those that reduce or eliminate recombination between
               a lethal-bearing chromosome and a homologue carrying a wild-type allele of the locus, and (2) those that provide an extrachromosomal
               or integrated wild-type allele that complements a homozygous lethal mutation. As balancers, these rearrangements have several
               advantages over simple marked chromosomes. They can balance large genomic regions, and the choice of markers to which lethals
               can be linked is much broader, requiring only that the markers lie in the balanced region. By virtue of their effects on recombination,
               they can be heritably stable vehicles for the maintenance of large numbers of lethal mutations. Finally, genetic variants
               of the rearrangements themselves can be generated to aid in strain construction, mutant screens, and analysis of balancer
               structure.
            </p>
            <p>Rearrangements as balancers are the focus of this chapter. We describe the features and behavior of each class of balancer
               currently used in <span class="emphasis"><em>C. elegans</em></span>, using a well-characterized member of the class; present a discussion of the practical aspects of day-to-day balancer use;
               give an overview of the state of balancing for each chromosome; and lay out in guidebook form information for understanding
               and maintaining the most commonly used <span class="emphasis"><em>C. elegans</em></span> balancers and their genetic variants. A great deal of detailed analysis beyond the scope of this chapter has been accumulated
               on balancers (see citations for individual rearrangements), and we encourage the reader to seek out the published works for
               a more complete understanding of the behavior of particular balancers. Information contained in the cited work is especially
               helpful when designing mutant screens and carrying out complementation and mapping protocols.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Types of balancers
                     </h2>
                  </div>
               </div>
            </div>
            <p>In <span class="emphasis"><em>C. elegans</em></span>, <a href="#bib24">Herman, R.K., Albertson, D.G., and Brenner, S. (1976)</a> first characterized duplications with the intention of using them to recover recessive lethal mutations on the X chromosome.
               Subsequent work by many different groups has resulted in the isolation and characterization of an array of rearrangement types
               that together balance approximately 70% of the <span class="emphasis"><em>C. elegans</em></span> genome. These rearrangements, including translocations, duplications, and inversions, constitute a powerful set of tools
               for investigators. In addition, other dominant crossover suppressors and transgenes have been adapted for use as balancers.
               The physical structures of several balancers have been deduced through genetic analysis. Recent work using fluorescence <span class="emphasis"><em>in situ</em></span> hybridization (FISH) analysis (<a href="#bib2">Albertson, D.G., Fishpool, R.M., and Birchall, P.S., 1995</a>) shows the potential for determining balancer structure through molecular means. The following discussion gives an overview
               of the different classes of balancers.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_1"></a>2.1.&nbsp;Translocations
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Translocations are chromosomal rearrangements in which parts of nonhomologous chromosomes are exchanged (<a href="#figure1">Figure 1</a>). Translocations in <span class="emphasis"><em>C. elegans</em></span> have been recovered in specific screens for X-chromosome nondisjunction (<a href="#bib26">Herman, R.K., Kari, C.K., and Hartman, P.S., 1982</a>), suppression of recombination between widely spaced linked markers (<a href="#bib20">Herman, R.K., 1978</a>; <a href="#bib17">Fodor, A., and Deak, P., 1985</a>; <a href="#bib14">Edgley, M.L., and Riddle, D.L., 2001</a>), linked lethals (<a href="#bib62">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L., 1985</a>; <a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>), and pseudolinkage of normally unlinked marker mutations (<a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>; <a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>). A number of existing translocations have been characterized genetically, and several have proven to be reliable balancers
                  for lethal mutant screens (<span class="emphasis"><em>eT1</em></span>: <a href="#bib60">Rosenbluth, R.E., and Baillie, D.L., 1981</a>; <a href="#bib61">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L., 1983</a>; <span class="emphasis"><em>nTl</em></span>: <a href="#bib12">Clark, D.V., Rogalski, T.M., Donati, L.M., and Baillie, D.L., 1988</a>; <span class="emphasis"><em>hTl</em></span>: <a href="#bib33">Howell, A.M., and Rose, A.M., 1990</a>). They balance large genomic regions, and they are easily manipulated in genetic crosses.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="EdBfig1.jpg"><img src="EdBfig1_s.jpg" border="2" align="middle" alt=" Diagram of reciprocal translocation eT1, showing mutations present in the reference heterozygous strain figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. Diagram of reciprocal translocation <span class="emphasis"><em>eT1</em></span>, showing mutations present in the reference heterozygous strain.&nbsp;</b>The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-46" target="_blank">unc-46</a></em></span> mutations are carried on the normal chromosomes, and the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span> mutation is caused by the breakpoint of the translocation on LG III. Recombination is suppressed in the region to the right
                        of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span> on LG III, balancing <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a></em></span>, and in the region to the left of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-42" target="_blank">unc-42</a></em></span> on LG V (not shown, see Figure 7), balancing <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-46" target="_blank">unc-46</a></em></span>. <span class="emphasis"><em>eT1(III)</em></span> recombines with the normal LG III in the region to the left of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span>. <span class="emphasis"><em>eT1(V)</em></span> recombines with the normal LG V in the region to the right of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-23" target="_blank">unc-23</a></em></span> (not shown, see Figure 7).
                     </p>
                  </div>
               </div>
               <p>The first <span class="emphasis"><em>C. elegans</em></span> translocation for which reciprocal exchange of chromosomal segments was demonstrated is <span class="emphasis"><em>eT1(III;V)</em></span> (<a href="#bib60">Rosenbluth, R.E., and Baillie, D.L., 1981</a>), which is viable as a homozygote and was originally called <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-72" target="_blank">unc-72</a></em></span> (<a href="#bib7">Brenner, S., 1974</a>). Marker mutations crossed onto or induced on <span class="emphasis"><em>eT1</em></span> were employed to show linkage between appropriate normally unlinked markers on both half-translocations. Several other translocations
                  have since been shown to be reciprocal using similar protocols, for example, <span class="emphasis"><em>mnT2</em></span> and <span class="emphasis"><em>mnT10</em></span> (<a href="#bib26">Herman, R.K., Kari, C.K., and Hartman, P.S., 1982</a>); <span class="emphasis"><em>szT1</em></span> and <span class="emphasis"><em>hT1</em></span> (<a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>); and <span class="emphasis"><em>hT2</em></span> (<a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>). Segregation data for <span class="emphasis"><em>nT1</em></span> (<a href="#bib15">Ferguson, E.L., and Horvitz, H.R., 1985</a>; <a href="#bib12">Clark, D.V., Rogalski, T.M., Donati, L.M., and Baillie, D.L., 1988</a>), <span class="emphasis"><em>hT3</em></span> (<a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>) and <span class="emphasis"><em>mT1</em></span> (<a href="#bib14">Edgley, M.L., and Riddle, D.L., 2001</a>) are consistent with their being reciprocal translocations, but rigorous proof is lacking.
               </p>
               <p>In the case of <span class="emphasis"><em>eT1</em></span>, the left portion of LG V is translocated to the left portion of LG III, and the right portion of III is translocated to
                  the right portion of V (<a href="#figure1">Figure 1</a>). In heterozygotes, the half-translocation carrying the left portion of III [referred to as <span class="emphasis"><em>eT1(III)</em></span>] recombines with the normal III in the region to the left of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span>, and the half-translocation carrying the right portion of V [<span class="emphasis"><em>eT1(V)</em></span>] recombines with the normal V in the region to the right of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-23" target="_blank">unc-23</a></em></span> (refer to <a href="#figure7">Figure 7</a>b and <a href="#figure7">Figure 7</a>c for position of markers). The regions that recombine segregate from each other during meiosis (<a href="#bib60">Rosenbluth, R.E., and Baillie, D.L., 1981</a>; <a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>; <a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>). Thus, <span class="emphasis"><em>eT1(III)</em></span> segregates from the normal III, and <span class="emphasis"><em>eT1(V)</em></span> segregates from the normal LG V. Recombination is suppressed along the length of the translocated portion of each chromosome,
                  from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span> to the right end of LG III, and from between <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-23" target="_blank">unc-23</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-42" target="_blank">unc-42</a></em></span> to the left end of LG. V. The boundaries of crossover suppression thus correspond to the translocation breakpoints. Recombination
                  is not suppressed in the regions that segregate from their normal homologues, from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span> to the left end of LG III, and from between <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-23" target="_blank">unc-23</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-42" target="_blank">unc-42</a></em></span> to the right end of LG V; nor is it suppressed in <span class="emphasis"><em>eT1</em></span> homozygotes. These observations have led to the proposal that these regions do not recombine in heterozygotes due to their
                  inability to pair with normal homologues (<a href="#bib60">Rosenbluth, R.E., and Baillie, D.L., 1981</a>). To maintain the translocation as a heterozygote, the normal homologues are marked with morphological mutations in the recombination-suppressed
                  region [in the case of <span class="emphasis"><em>eT1</em></span>, the most frequently used strain carries <span class="emphasis"><em>dpy-18(III)</em></span> and <span class="emphasis"><em>unc-46(V)</em></span>]. In this way, heterozygotes (which have a wild-type phenotype) can be distinguished from normal homologue homozygotes. Self
                  progeny of <span class="emphasis"><em>eT1</em></span> heterozygotes are wild-type heterozygotes; <span class="emphasis"><em>eT1</em></span> homozygotes (which have an Unc-36 phenotype because the translocation breakpoint lies in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span> gene on LG III); Dpy-18 Unc-46 homozygotes; and a large percentage of aneuploid progeny (10/16) that arrest development (<a href="#figure2">Figure 2</a>) as embryos or early larvae (<a href="#bib1">Adames, K.A., Gawne, J., Wicky, C., Muller, F., and Rose, A.M., 1998</a>; Turner and Baillie, unpublished results). Aneuploid progeny are those with an abnormal complement of chromosomes. <a href="#figure2">Figure 2</a> illustrates how they arise. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-46" target="_blank">unc-46</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a></em></span> mutations appear to be linked (they are pseudolinked) because all singly mutant homozygotes are aneuploid, and this arrangement
                  cannot break down in the absence of recombination. If a strain carries a recessive lethal mutation in the recombination-suppressed
                  region of either normal homologue, all of the homozygous DpyUnc progeny will die. To characterize these arrested animals,
                  they must not be confused with aneuploid progeny segregated by translocation heterozygotes (See 3.6.6, <a href="#sec3_6_6" title="3.6.6.&nbsp;Linked markers">Linked markers</a>, for more detailed discussion).
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="EdBfig2.jpg"><img src="EdBfig2_s.jpg" border="2" align="middle" alt=" Punnett square showing normal progeny genotypes and phenotypes that result from selfing a dpy-18/eT1 III; unc-46/eT1 Vheterozygote figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Punnett square showing normal progeny genotypes and phenotypes that result from selfing a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a>/eT1 III</em></span>; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-46" target="_blank">unc-46</a>/eT1 V</em></span> heterozygote.&nbsp;</b>Phenotypes are indicated for each progeny class. Genotypes for gametes and zygotes are given in gene and rearrangement names
                        according to standard nomenclature, and in drawings representing normal and translocation chromosomes with genetic markers
                        in place. The normal LG III is shown as a black bar, with a vertical line indicating the position of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a></em></span> mutation. The normal LG V is shown as a shaded bar, with a vertical line indicating the position of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-46" target="_blank">unc-46</a></em></span> mutation. The half-translocation <span class="emphasis"><em>eT1 (III)</em></span> is shown as a half-black, half-shaded bar with a vertical line indicating the position of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span> mutation caused by the translocation breakpoint on LG III. The half-translocation <span class="emphasis"><em>eT1 (V)</em></span> is shown as a half-black, half-shaded bar with no vertical line. Boxes of the square representing viable progeny are unshaded.
                        Boxes representing aneuploid progeny, all of which arrest during development, are shaded. All wild-type progeny are heterozygous
                        for the translocation chromosomes and the normal chromosomes. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span> progeny are <span class="emphasis"><em>eT1</em></span> homozygotes, and Dpy Unc progeny are <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a>; <a href="http://www.wormbase.org/db/gene/gene?name=unc-46" target="_blank">unc-46</a></em></span> homozygotes. Aneuploid progeny account for 10/16ths of the total.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_2"></a>2.2.&nbsp;Duplications
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Duplications are chromosomal segments that are present in the nucleus in addition to a full complement of normal chromosomes
                  (<a href="#figure3">Figure 3</a>). Duplications have been generated by mutagenesis with ionizing radiation or UV light, and recovered as genetic elements
                  that rescue the phenotypic expression of particular mutations (<a href="#bib24">Herman, R.K., Albertson, D.G., and Brenner, S., 1976</a> <a href="#bib60">Rosenbluth, R.E., and Baillie, D.L., 1981</a>; <a href="#bib27">Herman, R.K., Madl, J.E., and Kari, C.K., 1979</a>; <a href="#bib26">Herman, R.K., Kari, C.K., and Hartman, P.S., 1982</a>; <a href="#bib28">Hodgkin, J., 1980</a>; <a href="#bib19">Greenwald, I.S., Sternberg, P.W., and Horvitz, H.R., 1983</a>; <a href="#bib21">Herman, R.K., 1984</a>; <a href="#bib22">Herman, R.K., 1987</a>; <a href="#bib48">Meneely, P.M. and Wood, W.B., 1984</a>; <a href="#bib59">Rose, A.M., Baillie, D.L., and Curran, J., 1984</a>; <a href="#bib62">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L., 1985</a>; <a href="#bib64">Rosenbluth, R.E., Rogalski, T.M., Johnsen, R.C., Addison, L.M., and Baillie, D.L., 1988</a>; <a href="#bib6">Austin, J., and Kimble, J., 1989</a>; <a href="#bib13">DeLong, L., Casson, L.P., and Meyer, B.J., 1987</a>; <a href="#bib47">Meneely, P.M., and Nordstrom, K.D., 1988</a>; <a href="#bib57">Rogalski, T.M., and Riddle, D.L., 1988</a>; <a href="#bib25">Herman, R.K., and Kari, C.K., 1989</a>; <a href="#bib33">Howell, A.M., and Rose, A.M., 1990</a>; <a href="#bib34">Hunter, C.P., and Wood, W.B., 1990</a>; <a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>; <a href="#bib69">Yuan, J.Y., and Horvitz, H.R., 1990</a>; <a href="#bib66">Stewart, H.I., Rosenbluth, R.E., and Baillie, D.L., 1991</a>; <a href="#bib39">Marra, M.A., and Baillie, D.L., 1994</a>). They have also been recovered as exceptional segregants from translocation strains (<span class="emphasis"><em>mnDp11</em></span>: <a href="#bib26">Herman, R.K., Kari, C.K., and Hartman, P.S., 1982</a>; <span class="emphasis"><em>szDp1</em></span>: <a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>; <span class="emphasis"><em>hDp133, hDp134, hDp135</em></span>: <a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>), and as products of a rare recombination event in an inversion strain (<a href="#bib70">Zetka, M.C., and Rose, A.M., 1992</a>). Most duplications in <span class="emphasis"><em>C. elegans</em></span> are "free" elements that segregate in a non-Mendelian fashion. They have been described as extrachromosomal fragments of
                  genetic material, and they can be lost both mitotically and meiotically during gametogenesis (<a href="#bib24">Herman, R.K., Albertson, D.G., and Brenner, S., 1976</a>). Mitotic chromosomes exhibit multiple spindle attachment points (<a href="#bib3">Albertson, D.G., and Thomson, J.N., 1982</a>), whereas meiotic chromosomes exhibit a localized spindle attachment point (<a href="#bib4">Albertson, D.G., and Thomson, J.N., 1993</a>). Hence, mitotic segregation of chromosome fragments is more easily understood than is meiotic segregation. The mechanism
                  of free duplication transmission during meiosis is unknown. Each duplication has a characteristic frequency of transmission
                  to gametes. Generally, the larger the duplication, the more stable it is in the germ line (<a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>). Mitotic loss of certain free duplications from somatic cells during development has made possible mosaic analysis of the
                  gene function (<a href="#bib23">Herman, R.K., 1995</a>). A small percentage of duplications exist as integrants into otherwise normal chromosomes, in which case segregation becomes
                  Mendelian. Many duplications have been used as balancers (e.g., <span class="emphasis"><em>mnDpl</em></span>: <a href="#bib45">Meneely, P.M., and Herman, R.K., 1979</a>; <span class="emphasis"><em>sDp2</em></span>: <a href="#bib32">Howell, A.M., Gilmour, S.G., Mancebo, R.A., and Rose, A.M., 1987</a>; <span class="emphasis"><em>qDp3</em></span>: <a href="#bib8">Bucher, E.A., and Greenwald, I., 1991</a>; <span class="emphasis"><em>sDp3</em></span>: Stewart and Baillie, unpublished results). There are two basic classes of duplication, those that do not recombine with
                  the normal homologues and those that do. Discussion of the significance of this observation can be found in <a href="#bib27">Herman, R.K., Madl, J.E., and Kari, C.K. (1979</a>) and <a href="#bib59">Rose, A.M., Baillie, D.L., and Curran, J. (1984</a>). Members of the nonrecombining class can be used as effective balancers (<a href="#bib59">Rose, A.M., Baillie, D.L., and Curran, J., 1984</a>; Stewart and Baillie, unpublished results). For example, <span class="emphasis"><em>sDp2</em></span> has been used extensively in screens for lethal mutations, and it has been shown to permit identification of the same range
                  of essential genes as does a translocation (<span class="emphasis"><em>hT1</em></span>) that balances the same region (<a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>). Rarely, free duplications have been observed to shorten spontaneously, which reduces the extent of the balanced region
                  (<a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>).
               </p>
               <p>Free duplications generally exist as a single copy, but strains carrying two copies have been documented in some cases (<a href="#bib27">Herman, R.K., Madl, J.E., and Kari, C.K., 1979</a>; <a href="#bib57">Rogalski, T.M., and Riddle, D.L., 1988</a>). The reference strain carrying <span class="emphasis"><em>sDp2</em></span> is homozygous for mutant alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> and, in addition, has a single copy of <span class="emphasis"><em>sDp2</em></span> (<a href="#figure3">Figure 3</a>). The strain has an Unc-13 phenotype, as the duplication carries a wild-type allele of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> but not of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span>. It segregates Unc-13 progeny (carrying the duplication) and Dpy-5 Unc-13 homozygotes. The possibility of losing either marker
                  is eliminated by the homozygous state of the marked chromosomes and lack of recombination with the duplication. If a strain
                  is also homozygous for a lethal mutation on the same chromosome in the duplicated region, then the Dpy Unc progeny class will
                  die and the only viable progeny will be those carrying the duplication. The lethal mutation is thus effectively balanced.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="387">
<tr>
  <td align="center"><p><a href="EdBfig3.jpg"><img src="EdBfig3_s.jpg" border="2" align="middle" alt="Diagram of genotype of the reference sDp2 strain, showing progeny genotypes and phenotypes that result from selfing figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 3. Diagram of genotype of the reference <span class="emphasis"><em>sDp2</em></span> strain, showing progeny genotypes and phenotypes that result from selfing.&nbsp;</b>The strain is homozygous for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> mutations, and carries one copy of <span class="emphasis"><em>sDp2</em></span>. Animals of this genotype are Unc-13 in phenotype. Progeny not carrying the duplication are Dpy-5 Unc-13.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_3"></a>2.3.&nbsp;Transgenes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A transgene is a segment of cloned DNA containing the complete coding element and control elements of a gene, present in the
                  nucleus in addition to a complete complement of normal chromosomes. Direct transformation of mutant strains using cloned DNA
                  is performed in <span class="emphasis"><em>C. elegans</em></span> by germline microinjection (<a href="#bib16">Fire, A., 1986</a>; <a href="#bib43">Mello, C., and Fire, A., 1995</a>). The microinjected molecules (containing one or more transgenes) assemble into tandem arrays that can be maintained extrachromosomally.
                  Expression of wild-type transgenes in the array can complement, or "rescue," a mutant phenotype. An important advance in these
                  procedures involved coinjecting the DNA to be tested for rescue along with clones of a selectable marker mutation, such as
                  the dominant <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a></em></span> mutation (<a href="#bib44">Mello, C.C., Kramer, J.M., Stinchcomb, D., and Ambros, V., 1991</a>), which causes animals carrying it to roll along their long axes. Transgenic progeny displaying the marker phenotype were
                  scored for rescue of the strain's mutant phenotype.
               </p>
               <p>Germline microinjection becomes laborious if one wishes to assay the rescuing ability of a group of cosmids in a set of mutants.
                  A more general approach takes advantage of the ability to manipulate transgenes genetically, as if they were free duplications.
                  McKay, McDowall, and Rose (unpublished results) have used transgenes as genetic elements in large-scale experiments to rescue
                  lethal mutations in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-14" target="_blank">dpy-14</a></em></span> regions of chromosome I. Copies of a marker plasmid containing <span class="emphasis"><em>rol-6(<a href="http://www.wormbase.org/db/get?name=su1006;class=Variation" target="_blank">su1006</a>)</em></span> were coinjected with sets of two or three overlapping cosmids from the genomic region of interest into wild-type animals
                  or animals homozygous for a morphological mutation. Transformants were selected on the basis of their Rol-6 phenotype, and
                  individual lines were established that exhibited a high frequency of transmission of the transgene. Standard duplication complementation
                  tests were then used to assay for rescue of a mutation. Rescue was confirmed by establishment of a nonmutant line from the
                  complementation test that exhibited the Rol-6 phenotype, with the transgene as an effective balancer, and that segregated
                  nonrolling progeny (not carrying the transgenic array) that were not rescued. Since the sequence of the whole genome has been
                  deduced (<a href="#bib9"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>), this method can be used with virtually any gene.
               </p>
               <p>Complementation tests with extrachromosomal arrays containing more than one cosmid increase the efficiency of the analysis,
                  as every mutation does not have to be tested separately against every cosmid. If the transgenic arrays are properly designed,
                  a small number of crosses can establish exactly which individual cosmid is responsible for the rescue. Arrays containing single
                  rescuing cosmids can then be generated to confirm the result.
               </p>
               <p>Transgenic arrays have also been used as balancers in screens for new mutations. Labouesse and Horvitz (pers. comm.) used
                  a transgenic strain carrying a cosmid that rescues <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-26" target="_blank">lin-26</a></em></span> in a noncomplementation screen for new mutations mapping to the region balanced by the cosmid. A wild-type clone of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span> was coinjected with the cosmid into a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span> mutant strain as a marker to identify transformants. The resulting non-Dpy transgenic animals were mutagenized, and their
                  F2 progeny screened for the presence of new <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-26" target="_blank">lin-26</a></em></span> mutations or other mutations balanced by the cosmid. These were identified in the Dpy segregants that had spontaneously lost
                  the transgenes. One new mutation, in the gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-253" target="_blank">let-253</a></em></span>, was recovered among the progeny of 2727 F1's. This technique allows balancers to be used in tightly focused screens for
                  mutations in very small genomic regions. <a href="#bib53">Plasterk, R.H.A. (1995</a>) also used such arrays to screen for lethal transposon-induced deficiencies.
               </p>
               <p>Green fluorescent protein (GFP) transgenes (<a href="#bib10">Chalfie M., Tu, Y., Euskirchen, G., Ward W.W., and Prasher, D.C., 1994</a>) also have utility as balancers and chromosome markers. When animals are transformed by microinjection of the GFP coding
                  region under control of a <span class="emphasis"><em>C. elegans</em></span> promoter, GFP is expressed in the corresponding tissue. For example, a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=myo-2" target="_blank">myo-2</a></em></span>::GFP construct results in GFP expression in the pharynx. If such a transgene is integrated into the genome, GFP expression
                  becomes a marker for the chromosome into which the element is integrated. The expression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=myo-2" target="_blank">myo-2</a></em></span>::GFP constructs typically is semi-dominant, allowing one integrated copy to be distinguished from two by the relative brightness
                  of the GFP signal (<a href="#bib14">Edgley, M.L., and Riddle, D.L., 2001</a>; Edgley, unpublished results).
               </p>
               <p>Several researchers have performed integration experiments with a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=myo-2" target="_blank">myo-2</a></em></span>::GFP construct, resulting in GFP-tagged variants of several existing balancers and many integrations into non-balancer chromosomes.
                  The former group is very useful, often allowing unequivocal identification of lethal homozygotes by virtue of the absence
                  of GFP expression (see the handbook section for GFP variants of <span class="emphasis"><em>mIn1</em></span>, <span class="emphasis"><em>nT1</em></span>, and <span class="emphasis"><em>hT2</em></span>). Simple chromosomal insertions are also valuable, as they are semi-dominant markers for particular chromosome regions, and
                  they usually exhibit local recombination suppression over a few map units. A number of these elements have been roughly mapped
                  genetically and are in use as balancers for regions where no other balancer exists (Edgley, unpublished results). Large-scale
                  integration of promoter::GFP elements and subsequent SNP mapping could yield local balancer elements for several regions that
                  are not currently covered by balancers.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_4"></a>2.4.&nbsp;Inversions
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Inversions are chromosome segments that break from the intact chromosome and rejoin it in the original location but in opposite
                  orientation. There are two proven examples of inversion in <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em>hIn1 I</em></span> and <span class="emphasis"><em>mIn1 II</em></span>. In <span class="emphasis"><em>hIn1</em></span> (<a href="#figure4">Figure 4</a>) a portion of the right half of LG I, from about <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-75" target="_blank">unc-75</a></em></span> to about <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span>, is inverted (<a href="#bib70">Zetka, M.C., and Rose, A.M., 1992</a>). In <span class="emphasis"><em>mIn1</em></span>, a large section in the middle of the chromosome is inverted, from about <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-31" target="_blank">lin-31</a></em></span> to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-1" target="_blank">rol-1</a></em></span>. For both elements, mapping of mutations in inversion homozygotes revealed the inverted gene order relative to wild type.
                  Both were recovered in screens for rearrangements that suppress crossing over specifically in previously unbalanced regions.
                  Recombination between the breakpoints is almost completely suppressed in heterozygotes, presumably because of absence of DNA
                  alignment or interruption of sites required for pairing. Both inversions are viable as homozygotes, with normal brood sizes,
                  and in the homozygous state exhibit normal levels of recombination in the inverted interval. <span class="emphasis"><em>hIn1</em></span> has been used in screens for lethal mutations on LG I to the right of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-75" target="_blank">unc-75</a></em></span> (Ho and Rose, unpublished results); <span class="emphasis"><em>mIn1</em></span> has been used in screens to isolate lethals and deletions across its extents, especially to the left of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span>.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="EdBfig4.jpg"><img src="EdBfig4_s.jpg" border="2" align="middle" alt="Diagram of genotype of reference hIn1 inversion strain. The unc-75 and unc-101 mutations are carried on the normal LG I. figure 4"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 4. Diagram of genotype of reference <span class="emphasis"><em>hIn1</em></span> inversion strain.&nbsp;</b>The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-75" target="_blank">unc-75</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span> mutations are carried on the normal LG I. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> mutation was induced secondarily on the inversion chromosome. Recombination is suppressed along the length of the inverted
                        segment in heterozygotes (boundaries of suppression correspond to breakpoints). Terminal sequences are most likely not included
                        in the inverted interval, as a deficiency of the ribosomal gene cluster (<span class="emphasis"><em>eDf24</em></span>) to the right of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> on normal LG I is apparently not balanced.
                     </p>
                  </div>
               </div>
               <p>The original <span class="emphasis"><em>hIn1</em></span> was generated on a wild-type chromosome, making it impossible to distinguish between inversion heterozygotes and homozygotes.
                  To circumvent this difficulty and to make the balancer more useful for mutant screens, new <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-75" target="_blank">unc-75</a></em></span> (two lethal alleles), <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span> mutations were induced on <span class="emphasis"><em>hIn1</em></span> (<a href="#bib37">Lee, J., Jongeward, G.D., and Sternberg, P.W., 1994</a>; <a href="#bib70">Zetka, M.C., and Rose, A.M., 1992</a>). The reference strain is a phenotypically wild-type heterozygote in which an <span class="emphasis"><em>hIn1[<a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a>]</em></span> chromosome balances <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-75" target="_blank">unc-75</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span>. Selfing the strain produces wild-type heterozygotes, balancer homozygotes (Unc-54 phenotype), and Unc-75 Unc-101 homozygotes.
                  <span class="emphasis"><em>hIn1</em></span> may not include the physical end of the chromosome, as <span class="emphasis"><em>eDf24</em></span> (a deletion of part of the ribosomal gene cluster that lies to the right of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span>) apparently is not balanced by the inversion (M. Zetka, pers. comm.).
               </p>
               <p>The <span class="emphasis"><em>mIn1</em></span> inversion was generated on a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span> chromosome. Several marker variants, including unmarked, doubly-marked, and marked with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-1" target="_blank">rol-1</a></em></span> were used to establish gene order. In addition, a variant carrying an integrated pharyngeal GFP in addition to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span> was generated by gamma-ray mutagenesis.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_5"></a>2.5.&nbsp;Other crossover suppressors
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Several chromosomal rearrangements exist in <span class="emphasis"><em>C. elegans</em></span> that dominantly suppress crossing over but that remain uncharacterized with regard to their structures. Two examples are
                  <span class="emphasis"><em>mnC1</em></span> (<a href="#bib20">Herman, R.K., 1978</a>), which balances the right half of LG II from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span> to around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span>, and <span class="emphasis"><em>sC1</em></span> (Stewart and Baillie, unpublished results), which balances an approximately 15-map-unit segment of the left half of LG III,
                  from around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-45" target="_blank">unc-45</a></em></span> to the region left of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-17" target="_blank">dpy-17</a></em></span>. Crossover suppression in these two cases is presumably restricted to a single chromosome, in contrast to translocations.
                  The patterns of crossover suppression, meiotic properties, brood size, and progeny phenotypes are consistent with their being
                  inversions, but proof is lacking. Both have been used successfully in screens for lethal mutations in the regions they balance.
                  <span class="emphasis"><em>mnC1</em></span>, marked with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span> mutations, has been especially useful in isolating and maintaining a high-resolution set of deficiencies (<a href="#bib65">Sigurdson, D.C., Spanier, G.J., and Herman, R.K., 1984</a>). It is phenotypically wild type as a heterozygote, gives broods of relatively normal size, and is viable as a DpyUnc homozygote
                  but with very small broods. <span class="emphasis"><em>sC1</em></span> was generated on a wild-type chromosome, is viable as a homozygote, and morphologically marked variants have been generated
                  to facilitate its use (Stewart and Baillie, unpublished results).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_6"></a>2.6.&nbsp;Deficiencies
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Deficiencies are segments of chromosomes entirely missing from the genome. Deficiencies are detected after mutagenesis by
                  various methods (e.g., <a href="#bib55">Riddle, D.L., and Brenner, S., 1978</a>; <a href="#bib65">Sigurdson, D.C., Spanier, G.J., and Herman, R.K., 1984</a>; <a href="#bib50">Moerman, D.G., and Baillie, D.L, 1981</a>; <a href="#bib18">Greenwald, I.S., and Horvitz, H.R., 1980</a>; <a href="#bib62">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L., 1985</a>; <a href="#bib68">Yandell, M.D., Edgar, L.S., and Wood, W.B., 1994</a>). They are useful as genetic mapping tools, but some deficiencies dominantly reduce recombination frequency in particular
                  genetic regions. Several of the latter class were described by <a href="#bib63">Rosenbluth, R.E., Johnsen, R.C., and Baillie, D.L. (1990</a>). A chromosome carrying a small deficiency in the gene cluster of LG I, <span class="emphasis"><em>hDf8</em></span>, was also associated with reduced recombination frequency, from around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-14" target="_blank">dpy-14</a></em></span> to around the left end of the chromosome (<a href="#bib42">McKim, K.S., Starr, T., and Rose, A.M., 1992</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;Practical considerations of balancer use
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_1"></a>3.1.&nbsp;Characteristics of good balancers
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Balancers are used for a variety of tasks, including balancing existing mutations, facilitating strain construction, and screening
                  for new mutations. At a minimum, a balancer should have the properties already described: heterozygotes must have a phenotype
                  distinguishable from that of each homozygote; recombination should be virtually eliminated; progeny phenotypes must allow
                  ready detection of rare recombination; and the construct should be stable (balanced mutations should not spontaneously become
                  unbalanced). It is also helpful if the balancer can be manipulated easily in genetic crosses, particularly if it can be passed
                  through male sperm, and if morphologically marked or lethal variants exist.
               </p>
               <p>Balancing existing mutations is probably the least demanding of these tasks, requiring only that a suitable balancer exists
                  and that the mutation/balancer heterozygote is viable. A minor degree of instability in the heterozygote is acceptable, as
                  long as the investigator checks the strain routinely. Strain construction can be aided by particular qualities of a few balancers,
                  and often requires balancers marked with particular mutations (see <a href="#sec3_3" title="3.3.&nbsp;Crossing schemes">Section 3.3</a>).
               </p>
               <p>By far the most stringent test of a balancer is to use it to screen for lethal mutations. The major additional requirement
                  imposed on the balancer is that the apparent spontaneous mutation frequency should be low (no higher than in the reference
                  wild-type strain, N2). Studies using two different reciprocal translocations (<span class="emphasis"><em>eT1</em></span> and <span class="emphasis"><em>szT1</em></span>) illustrate this point.
                  				
               </p>
               <div class="informalexample">
                  <p>1. <span class="emphasis"><em>eT1(III;V)</em></span> This translocation is an extremely stable balancer that has been used extensively for lethal screens. The size of the balanced
                     region is about 40 map units (mu). <a href="#bib61">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L. (1983</a>) conducted a dose-response study for balanced lethal mutations using ethyl methanesulfonate (<a href="http://www.wormbase.org/db/get?name=EMS;class=Cell" target="_blank">EMS</a>) and gamma irradiation, and determined the spontaneous mutation frequency for lethal mutations. A strain of genotype <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a>/eT1 III; <a href="http://www.wormbase.org/db/gene/gene?name=unc-46" target="_blank">unc-46</a>/eT1 V</em></span> was used to screen for new lethal mutations linked to either <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-46" target="_blank">unc-46</a></em></span>. Lethals were recovered at reasonable frequency with both mutagens (6.6% at 12 mM <a href="http://www.wormbase.org/db/get?name=EMS;class=Cell" target="_blank">EMS</a>, and 4.5% at 1500 R gamma). The frequency of accumulation of spontaneous mutations was very low. Two control experiments,
                     involving 3198 F1 heterozygous progeny of two <a href="http://www.wormbase.org/db/get?name=P0;class=Cell" target="_blank">P0</a> animals, resulted in a total of two balanced lethal mutations for a frequency of 0.06%. In addition, <span class="emphasis"><em>eT1</em></span> seems to be completely stable. It has been used to screen many thousands of mutagenized chromosomes, is currently used to
                     balance hundreds of lethal mutations, and it has never been observed to break down.
                  </p>
               </div>
               <p>
                  				
               </p>
               <div class="informalexample">
                  <p>2. <span class="emphasis"><em>szT1(I;X)</em></span> This translocation is very stable, and it has been useful in strain constructions and for balancing existing mutations. It
                     has been used to a limited extent to recover new lethal mutations. Experiments to determine the spontaneous lethal mutation
                     frequency in the region balanced by <span class="emphasis"><em>szT1</em></span> gave results different from those obtained with <span class="emphasis"><em>eT1</em></span>. <a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M. (1988)</a> detected unusual segregants among the progeny of a strain of genotype <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a>/szT1[<a href="http://www.wormbase.org/db/gene/gene?name=lon-2" target="_blank">lon-2</a>] I; <a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a>/szT1 X</em></span>. This strain has a wild-type phenotype and normally segregates wild types, Dpy-5 Unc-13 Unc-3 homozygotes, a few Lon-2 males
                     (as a result of meiotic nondisjunction of the X chromosome), and aneuploids that arrest as embryos or young larvae. From among
                     the progeny of 1104 unmutagenized wild-type heterozygotes, 34 animals were recovered that did not give Dpy Unc-13 Unc-3 progeny
                     and thus appeared to carry a new lethal in one of the balanced regions. These animals, however, did not give progeny appropriate
                     for an <span class="emphasis"><em>szT1</em></span> heterozygote and, on further analysis, appeared to carry a compound chromosome consisting of <span class="emphasis"><em>szT1 (X)</em></span> fused to the normal X carrying the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a></em></span> mutation. These types of spontaneous rearrangements reduce the utility of <span class="emphasis"><em>szT1</em></span> as a balancer for lethal screens, although it has been used successfully after mutagenesis to recover lethals linked to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> (<a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>).
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_2"></a>3.2.&nbsp;Genetically marked balancers
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In properly designed strains, animals heterozygous for a balancer and a balanced mutation (and marker mutations), or homozygous
                  for the mutations and carrying a balancing duplication, possess a unique phenotype. These strains can be maintained by selecting
                  animals with this phenotype and checking their progeny to be sure that appropriate phenotypic classes are segregated. The
                  genetic details vary among individual balancers, and a good understanding of these details is required to maintain a given
                  strain. Specific information on normal progeny genotypes and phenotypes for each class of balancer can be found <a href="#sec2" title="2.&nbsp;Types of balancers">Section 2 (Types of balancers)</a>, and information on homozygous balancer phenotypes and standard marker mutations for particular balancers can be found <a href="#sec5" title="5.&nbsp;A field guide to balancers">Section 5 (A field guide to balancers)</a>.
               </p>
               <p>The original isolates of some balancers carry mutations that were in the genetic background of the strains from which the
                  balancers were generated (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span> in the case of <span class="emphasis"><em>mnC1</em></span>) or carry mutations in genes interrupted by rearrangement breakpoints (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span> in <span class="emphasis"><em>eT1</em></span>). In addition, existing balancers have been specially marked with lethal or morphological mutations for particular tasks.
                  These variants can be useful for strain construction (<a href="#bib60">Rosenbluth, R.E., and Baillie, D.L., 1981</a>), maintenance of balanced mutations (Johnsen and Baillie, unpublished results), and analysis of mutations and balancer structure
                  (<a href="#bib60">Rosenbluth, R.E., and Baillie, D.L., 1981</a>). They can be generated by different means, including spontaneous mutation (<a href="#bib56">Rogalski, T.M., Bullerjahn, A.M.E., and Riddle, D.L., 1988</a>), mutagenesis or recombination (<a href="#bib60">Rosenbluth, R.E., and Baillie, D.L., 1981</a>), and DNA fusion (<a href="#bib34">Hunter, C.P., and Wood, W.B., 1990</a>). The new mutations they carry may be recessive or dominant and may be conditional or non-conditional.
               </p>
               <p>Lethal variants of homozygous viable balancers can be used when it is desirable to remove balancer homozygotes from a population,
                  leaving heterozygotes as the only viable class of progeny. For example, Johnsen and Baillie (unpublished results) used a lethal
                  derivative of <span class="emphasis"><em>eT1</em></span> to maintain lethal mutations in strains in which balancer homozygotes were the most vigorous progeny class. Lethal variants
                  are useful to keep balancer homozygotes from overgrowing a population under any conditions in which heterozygotes cannot be
                  selected routinely, such as large-scale preparations for biochemical analysis or DNA extraction.
               </p>
               <p>Morphologically marked balancer variants are used when a homozygous viable balancer does not already have a unique phenotype,
                  when its phenotype is similar to that of one of the progeny classes, or when a second scorable phenotype is needed in addition
                  to the balancer's native homozygous phenotype. For example, <a href="#bib60">Rosenbluth, R.E., and Baillie, D.L. (1981</a>) induced new mutations on <span class="emphasis"><em>eT1</em></span> to determine the segregation patterns of the half-translocations. <a href="#bib70">Zetka, M.C., and Rose, A.M. (1992</a>) induced a new <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> mutation on the inversion <span class="emphasis"><em>hIn1</em></span>, which originally had a wild-type homozygous phenotype, to distinguish homozygotes from heterozygotes.
               </p>
               <p>Most of the lethal or morphological balancer variants in common use were generated by mutagenesis with <a href="http://www.wormbase.org/db/get?name=EMS;class=Cell" target="_blank">EMS</a> and are thus not very likely to be structurally altered. That is, they may generally be expected to balance the same regions
                  as their parent balancers. On the other hand, balancers subjected to mutagenesis with ionizing radiation could be expected
                  to carry new rearrangements (<a href="#bib62">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L., 1985</a>; <a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>; <a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>). Thus, it may be necessary to test the extent of balancing for particular variants.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_3"></a>3.3.&nbsp;Crossing schemes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Crosses to move mutations into or out of different genetic backgrounds are basic to genetic analysis. For example, crosses
                  are necessary to genetically map mutations; to complement them against deficiencies, duplications, or other mutations; to
                  study the effects of different mutations on each other; and to remove unwanted deleterious mutations after mutagenesis. If
                  one wishes to cross a balanced mutation into another genetic background, males that carry the mutation and are able to mate
                  may be needed. Three methods can be employed to accomplish this. First, a mixed population of heterozygous males, half of
                  which carry the balancer and half of which carry the balanced mutation, can be made by crossing heterozygous hermaphrodites
                  with wild-type males. Second, males can arise in balanced strains by spontaneous X-chromosome nondisjunction, the frequency
                  of which can be increased by a <span class="emphasis"><em>him</em></span> mutation in the genetic background (<a href="#bib30">Hodgkin, J.A., Horvitz, H.R., and Brenner, S., 1979</a>) or by heat stock (<a href="#bib67">Sulston, J., and Hodgkin, J., 1988</a>), and these males can be used to propagate balanced male stocks. Third, and most desirable, balanced phenotypically unique
                  males can be made from each balanced strain on demand by crossing to a standard male balancer strain.
               </p>
               <p>The first method complicates analysis by requiring F2 progeny testing to ensure that the correct marker has been transferred.
                  The second method mayor may not require progeny testing, depending on whether the desired F2 cross-progeny have a unique phenotype;
                  however, it requires that a male stock be maintained for each balanced mutation. Strain construction and complementation tests
                  are made much simpler if the genotypes and corresponding phenotypes resulting from crosses are unique, and if the required
                  males can be generated as needed.
               </p>
               <p>The following discussion presents a few examples of useful crossing protocols of the latter class for particular balancers.
                  The original published works, or the investigators themselves, are still the best source for specific detail and advice.
                  				
               </p>
               <div class="informalexample">
                  <p>1. <span class="emphasis"><em>hIn1 (I)</em></span> Ho and Rose (unpublished results) devised a simple scheme to facilitate the complementation testing of a set of lethal mutations
                     induced using the inversion <span class="emphasis"><em>hIn1</em></span> as a balancer. A single cross for each balanced lethal generated wild-type males of a single genotype that could be used
                     in complementation tests against deficiencies and other lethal mutations. For example, males of a standard balancer strain,
                     of genotype <span class="emphasis"><em>hIn1[<a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a>]/<a href="http://www.wormbase.org/db/gene/gene?name=unc-75" target="_blank">unc-75</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span>, were crossed to <span class="emphasis"><em>hIn1[<a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-75" target="_blank">unc-75</a>]/<a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a> <a href="http://www.wormbase.org/db/gene/gene?name=lev-11" target="_blank">lev-11</a> let-x</em></span> hermaphrodites (<a href="#figure5">Figure 5</a>). The only phenotypically wild-type male progeny, of genotype <span class="emphasis"><em>hIn1[<a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a>]/<a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a> <a href="http://www.wormbase.org/db/gene/gene?name=lev-11" target="_blank">lev-11</a> let-x</em></span>, were then crossed to other identically balanced lethals in complementation tests (e.g., <span class="emphasis"><em>hIn1[<a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a>]/<a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a> <a href="http://www.wormbase.org/db/gene/gene?name=lev-11" target="_blank">lev-11</a> let-y</em></span>). In this case, complementation of <span class="emphasis"><em>let-x</em></span> by <span class="emphasis"><em>let-y</em></span> was indicated by the presence of viable, fertile Unc-101 Lev-11 progeny (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a> <a href="http://www.wormbase.org/db/gene/gene?name=lev-11" target="_blank">lev-11</a> let-x/<a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a> <a href="http://www.wormbase.org/db/gene/gene?name=lev-11" target="_blank">lev-11</a> let-y</em></span>). Failure to complement was indicated by the absence of this progeny class. The same scheme was used to map the lethals against
                     deficiencies that deleted both <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lev-11" target="_blank">lev-11</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> (such as <span class="emphasis"><em>eDf3</em></span>). In these cases, complementation or its failure was indicated by the presence or absence of Lev-11 progeny, as none of the
                     deficiencies deleted <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span>.
                  </p>
               </div>
               <p>
                  				
               </p>
               <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="EdBfig5.jpg"><img src="EdBfig5_s.jpg" border="2" align="middle" alt=" Example of a crossing scheme using hIn1 to generate genotypically and phenotypically unique balanced males carrying a desired lethal mutation figure 5"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 5. Example of a crossing scheme using <span class="emphasis"><em>hIn1</em></span> to generate genotypically and phenotypically unique balanced males carrying a desired lethal mutation.&nbsp;</b>The standard male balancer strain and a sample balanced lethal strain are represented by genotypes in the boxes on the left.
                        The genotypes and phenotypes of the four classes of cross progeny resulting are shown on the right. Both male and hermaphrodite
                        cross progeny will be present. This example illustrates a basic principle of marker manipulation, which can be adapted for
                        other balancers and markers.
                     </p>
                  </div>
               </div>
               <p>
                  				
               </p>
               <div class="informalexample">
                  <p>2. <span class="emphasis"><em>szT1(I;X)</em></span> An advantage of <span class="emphasis"><em>szT1</em></span> in crossing protocols is its spontaneous segregation of fertile males that have a distinct Lon-2 phenotype and are hemizygous
                     for the translocation. These males have been used effectively for strain construction and complementation tests (<a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>) and elucidation of the structure of the translocation (<a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>). The translocation was induced in a strain carrying <span class="emphasis"><em>lon-2(<a href="http://www.wormbase.org/db/get?name=e678;class=Variation" target="_blank">e678</a>)</em></span>, and this mutation is thus present on one half-translocation (<a href="#bib17">Fodor, A., and Deak, P., 1985</a>). Males from <span class="emphasis"><em>szT1</em></span> strains carrying <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lon-2" target="_blank">lon-2</a></em></span> must have a Lon-2 phenotype, as they are hemizygous for the X chromosome. These males carry both half-translocations and
                     one copy of a normal LG I. As males arise spontaneously in <span class="emphasis"><em>szT1</em></span> strains at a frequency of approximately 10% (<a href="#bib17">Fodor, A., and Deak, P., 1985</a>; <a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>), it is not necessary to make males that carry either the half-translocations or the balanced LG I markers by crossing balanced
                     hermaphrodites with wild-type males.
                  </p>
               </div>
               <p>
                  			The three basic uses of the Lon males depend on their exclusive transfer of the normal LG I to all viable male progeny
                  and of the translocation chromosomes to all viable hermaphrodite progeny (<a href="#figure6">Figure 6</a>). First, the males can be used to balance existing mutations. For example, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> could be balanced with <span class="emphasis"><em>szT1</em></span> simply by crossing Lon-2 males to a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> homozygote. As all wild-type hermaphrodite progeny resulting from the cross must be heterozygous for both <span class="emphasis"><em>szT1</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span>, the mutation is balanced in a single step. Second, the males can be used to generate wild-type males heterozygous for <span class="emphasis"><em>szT1</em></span>-balanced lethal mutations for use in complementation tests against other <span class="emphasis"><em>szT1</em></span>-balanced lethals, assuming the lethals have a morphological marker in common and assuming that failure to complement can
                  be scored reliably in male progeny. Third, they can be used for purposes that require F3 hermaphrodite progeny carrying two
                  copies of the normal LG I. For example, <a href="#bib40">McKim, K.S., and Rose, A.M. (1990</a>) used <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-11" target="_blank">unc-11</a> <a href="http://www.wormbase.org/db/gene/gene?name=dpy-14" target="_blank">dpy-14</a>/szT1 (I); szT1 (X)/O</em></span> males to generate wild-type males of genotype <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-11" target="_blank">unc-11</a> <a href="http://www.wormbase.org/db/gene/gene?name=dpy-14" target="_blank">dpy-14</a>/<a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a> let-x <a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span>. These were then crossed to other strains to test for complementation of <span class="emphasis"><em>let-x</em></span> by a set of duplications. Presence of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-11" target="_blank">unc-11</a> <a href="http://www.wormbase.org/db/gene/gene?name=dpy-14" target="_blank">dpy-14</a></em></span> chromosome allowed them to select progeny of the desired genotype without progeny testing.
                  				
               </p>
               <div class="informalexample">
                  <p>3. <span class="emphasis"><em>mnDp1 (X;V)</em></span> This duplication of part of the right end of LG X is translocated to LG V. It is sterile as a homozygote, but heterozygous
                     males are fertile (<a href="#bib24">Herman, R.K., Albertson, D.G., and Brenner, S., 1976</a>). Normally, analysis of X-linked mutations is complicated because males are hemizygous for the X chromosome and thus express
                     the mutant phenotype. For example, males resulting from crossing wild-type males to a homozygous mutant <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a></em></span> (LG X) hermaphrodite would have the genotype <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a>/0 (X)</em></span>. These would have an Unc-3 phenotype and be unable to mate or mate poorly. The use of <span class="emphasis"><em>mnDp1</em></span> (and similar duplications for other parts of the X chromosome) solves this problem. A hermaphrodite strain carrying two mutant
                     copies of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a></em></span> and one copy of <span class="emphasis"><em>mnDp1</em></span> has a wild-type phenotype, as the duplication provides a wild-type copy of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a></em></span> gene. Males resulting from crossing wild-type males with this strain have one of two genotypes and correspondingly unique
                     phenotypes. Half will have the genotype <span class="emphasis"><em>+/+ V; <a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a>/O</em></span> and be Unc-3, and half will have the genotype <span class="emphasis"><em>mnDp1/+ V; <a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a>/0</em></span> and a wild-type phenotype. The latter are able to mate and transfer both the duplication and the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a></em></span> mutation in crosses. <a href="#bib45">Meneely and Herman (1979</a>) used these properties of <span class="emphasis"><em>mnDp1</em></span> to analyze recessive lethal mutations balanced by <span class="emphasis"><em>mnDp1</em></span> that are either linked to or delete <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a></em></span>. Wild-type males of genotype <span class="emphasis"><em>mnDp1/+ V; <a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a> let-x/O</em></span> were used to transfer the two X mutations into different genetic backgrounds for two-and three-factor mapping and for complementation
                     tests with other balanced lethals.
                  </p>
               </div>
               <p>
                  				
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="495">
<tr>
  <td align="center"><p><a href="EdBfig6.jpg"><img src="EdBfig6_s.jpg" border="2" align="middle" alt="Example of a crossing scheme using spontaneous Lon-2 males from an szT1 strain to balance a dpy-5 mutation figure 6"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 6. Example of a crossing scheme using spontaneous Lon-2 males from an <span class="emphasis"><em>szT1</em></span> strain to balance a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> mutation.&nbsp;</b>The male and hermaphrodite parents are represented by genotypes in the boxes on the left. The genotypes and phenotypes of
                        the two classes of viable cross-progeny are shown on the right. Male parents are hemizygous for LG X and, because of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lon-2" target="_blank">lon-2</a></em></span> mutation carried on <span class="emphasis"><em>szT1</em></span>, must have a Lon phenotype. All wild-type male progeny will be heterozygous for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> and will not carry the translocation. All wild-type hermaphrodite progeny will be heterozygous for both <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> and the translocation. All other cross-progeny are aneuploid, and arrest during development.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_4"></a>3.4.&nbsp;Accumulation of spontaneous mutations
                        </h3>
                     </div>
                  </div>
               </div>
               <p><a href="#bib61">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L. (1983)</a> measured the spontaneous lethal mutation frequency in the 40-mu region balanced by <span class="emphasis"><em>eT1</em></span> and found it to be 0.06%. Extrapolated to the whole genome, approximately 1 in every 119 selfed hermaphrodites may be expected
                  to be heterozygous for a new lethal mutation. These mutations would quickly be lost by segregation if they appeared in strains
                  that carried no balancer and were maintained in culture. If, however, the mutations occur in a balanced region, segregational
                  and recombinational loss will be eliminated. Indeed, spontaneous mutations of all types (not only lethal) tend to accumulate
                  in balancer strains. Great care must therefore be taken by the investigator to confirm that a given balanced strain has not
                  acquired any additional mutations subsequent to the original mutagenesis. It is preferable to keep strains frozen if they
                  are not actually being used in experiments, and strains that have been passaged should be checked closely. The effects of
                  an additional lethal mutation in a strain that already carries a lethal can be especially subtle. For example, if the original
                  mutation results in arrest at a mid-larval developmental stage, an additional mutation that causes the animal to arrest slightly
                  earlier may not be noticed. New mutations causing later developmental arrest are phenotypically undetectable in a lethal that
                  arrests as an embryo. These unrecognized secondary mutations may confound analyses.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_5"></a>3.5.&nbsp;Exceptional segregants and balancer breakdown
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The value of a balancer is reduced if balanced mutations spontaneously become unbalanced, or if balancer strains appear to
                  acquire new mutations at high frequency. The point at which a balancer is no longer considered a balancer is when it cannot
                  be used successfully for its intended purpose. This is the decision of the particular investigator. Although it is possible
                  for any balancer to undergo recombination that results in loss of balanced mutations, it is important to distinguish between
                  exceptional progeny that result from recombination and those that may result from chromosome nondisjunction, as described
                  below. The determination of the nature of a putative breakdown event must always be documented through recovery and analysis
                  of diagnostic progeny.
               </p>
               <p>Exceptional segregants may lead the investigator to believe that recombination has occurred in a balancer strain. For example,
                  the normal progeny of a strain carrying <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a> I</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a> X</em></span> mutations balanced by <span class="emphasis"><em>szT1</em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a>/szT1[<a href="http://www.wormbase.org/db/gene/gene?name=lon-2" target="_blank">lon-2</a>] I; <a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a>/szT1 X</em></span>) are wild-type heterozygotes, Dpy-5 Unc-3 homozygotes, Lon-2 males, lethal <span class="emphasis"><em>szT1</em></span> homozygotes, and lethal aneuploid progeny. Dpy non-Unc and Unc non-Dpy progeny are not usually produced because the markers
                  are pseudolinked. Rare Unc-3 progeny do arise (<a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>), which at first glance could be the product of a recombination event between the normal LG I and <span class="emphasis"><em>szT1(X)</em></span> that resulted in loss of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> from the normal LG I. Careful analysis of one such event, however, revealed that no recombination had occurred. Instead,
                  one of the half-translocations, <span class="emphasis"><em>szT1(X)</em></span>, carrying a portion of LG I, was present as a duplication in addition to two normal copies of LG I (marked with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span>) and two normal copies of LG X (marked with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a></em></span>). As this duplication, named <span class="emphasis"><em>szDp1</em></span>, carried a wild-type copy of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> but not of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a></em></span>, the strain had an Unc-3 phenotype. Analogous exceptional segregants have been recovered from a number of translocation strains,
                  for example, <span class="emphasis"><em>mnDp11</em></span> from <span class="emphasis"><em>mnT2</em></span> (<a href="#bib26">Herman, R.K., Kari, C.K., and Hartman, P.S., 1982</a>), <span class="emphasis"><em>hDp133</em></span> from <span class="emphasis"><em>hT1</em></span>, <span class="emphasis"><em>hDp134</em></span> from <span class="emphasis"><em>hT2</em></span>, and <span class="emphasis"><em>hDp135</em></span> from <span class="emphasis"><em>hT3</em></span> (<a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>). Thus, the investigator must be aware of the possibility that a translocation heterozygote will give rise to apparent recombinant
                  progeny and must analyze the exceptional progeny before concluding that a crossover has occurred. Physical breakdown of a
                  few balancers has been documented. For example, some duplications of LG I have been observed to shorten spontaneously, resulting
                  in exposure of previously balanced mutations (<a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>). Also, <a href="#bib70">Zetka, M.C., and Rose, A.M. (1992</a>) documented two rare recombination events in <span class="emphasis"><em>hIn1</em></span> heterozygotes that resulted in deficiencies. In all these cases, the fact that physical rearrangement had occured was documented
                  through the recovery and analysis of exceptional progeny.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_6"></a>3.6.&nbsp;Use of balancers in mutant screens
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Balancers have been used in mutant screens most commonly for recovery of lethal mutations, but they can also be used to recover
                  nonlethal mutations. For example, they can be used in noncomplementation screens for new alleles of existing morphological
                  mutations. The work summarized below is concerned mostly with lethal mutations, but it provides practical information for
                  anyone contemplating a mutant screen. The following information will not substitute for a thorough knowledge of balancer genetics
                  and the published works, but the points are good guides for avoiding unnecessary work and confusion.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_6_1"></a>3.6.1.&nbsp;Mutagenizing balancer strains
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Ideally, nonbalancer strains are mutagenized and the new mutations are captured in crosses to balancers, to prevent inducing
                     mutations on the balancer itself. Such crosses also replace approximately half the mutagenized genome with unmutagenized chromosomes,
                     helping to remove unwanted second-site mutations. For large screens, however, routine backcrossing is not practical, and it
                     is much more efficient to mutagenize balancer strains directly. Genetic mapping and <span class="emphasis"><em>inter se</em></span> complementation of the new mutations can be accomplished rapidly with strains that are already balanced and appropriately
                     marked, but that have not been backcrossed, as discussed previously. The problem of second-site mutations can be minimized
                     by the use of low mutagen doses (see below). Before more extensive characterization is undertaken for particular mutations,
                     back-crossing can be done using schemes similar to those presented in <a href="#sec3" title="3.&nbsp;Practical considerations of balancer use">Section 3 (Practical considerations of balancer use)</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_6_2"></a>3.6.2.&nbsp;Mutagen dose
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Screens should be done at low mutagen doses for all mutagens to decrease the incidence of multiple mutational events. Studies
                     undertaken by <a href="#bib61">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L. (1983</a>; <a href="#bib62">1985</a>) demonstrated the utility of low doses of <a href="http://www.wormbase.org/db/get?name=EMS;class=Cell" target="_blank">EMS</a> and gamma irradiation, and characterized a number of rearrangements generated in these experiments. Based on these studies,
                     we recommend doses in the range 1500 to 3000 R for gamma irradiation and 6 to 18 mM for <a href="http://www.wormbase.org/db/get?name=EMS;class=Cell" target="_blank">EMS</a>. It is tempting to use higher doses to increase the efficiency of recovery, but the benefits are more than outweighed by
                     the increased complication of analysis. At the very least, the use of lower doses can help keep the amount of work down by
                     reducing the need for extensive backcrossing prior to analysis. Rare multiple events are usually spotted in the process of
                     mapping, although this is not always the case (<a href="#bib5">Anderson, P., 1995</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_6_3"></a>3.6.3.&nbsp;Mutation types
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Mutagenesis of balancer strains has yielded point mutations, deficiencies, duplications, and translocations. Some of these
                     can be very complicated to analyze. The investigator should be conversant with all the published work involving a prospective
                     balancer before using it to recover lethals, and should also be conversant with the detailed study by <a href="#bib62">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L. (1985</a>) of the types and frequency of mutations recovered in screens with <span class="emphasis"><em>eT1</em></span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_6_4"></a>3.6.4.&nbsp;Chromosomal location of balanced lethals
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Screens using reciprocal translocations will recover lethals in the two genomic regions balanced by the two half-translocations.
                     Figuring out on which chromosome they reside requires linkage tests or mapping (e.g., with deficiencies). Linkage mapping
                     is fairly trivial if the original balanced strain carries morphological markers on both balanced chromosomes. For example,
                     refer to <a href="#figure2">Figure 2</a>. Mapping to chromosome can be done by outcrossing to remove the translocation and examining segregating progeny for reduced
                     numbers of either Dpy-18 (indicating a lethal on LG III) or Unc-46 (indicating a lethal on LG V). Subsequent higher-resolution
                     positioning can be accomplished using recombination mapping and complementation tests with deficiencies or duplications.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_6_5"></a>3.6.5.&nbsp;Mutations outside the balanced region
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Translocation and inversion screens can yield lethal mutations that are balanced because they are close to the rearrangement
                     breakpoints, but they lie outside the rearrangement and may be lost through recombination. Duplications have a precise recovery
                     boundary, as they balance by complementation of the lethal (presence of a wild-type allele). A duplication screen will only
                     yield lethals that lie inside the genetic extents of the duplication. Deficiencies spanning the duplication endpoint will
                     not be recovered if they delete any essential genes inside the endpoint.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_6_6"></a>3.6.6.&nbsp;Linked markers
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The presence of morphological mutations linked to lethals in balancer strains has several benefits. First among these is the
                     ease of recovery of the lethals. It much easier to screen for the absence of a particular morphological mutant class than
                     it is to screen for arrested embryos or larvae. The second major advantage is that presence of the markers eases genetic mapping,
                     complementation analysis, and arrest-stage determination of the lethals. A disadvantage is that the investigator may encounter
                     marker effects. The presence of particular markers may result in a lethal arresting at a different developmental stage than
                     it would without the markers, or a given mutation may be lethal in the presence of a marker but completely viable in its absence.
                     For example, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-14" target="_blank">dpy-14</a></em></span> exhibits a synthetic lethal phenotype in doubly mutant constructs involving a number of muscle mutants, such as <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-15" target="_blank">unc-15</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> (<a href="#bib58">Rose, A.M., and Baillie, D.L., 1980</a>). It is best remove the markers prior to phenotypic analysis.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_6_7"></a>3.6.7.&nbsp;Selecting lethal homozygotes
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Care must be taken when selecting homozygous progeny for further developmental or molecular characterization to understand
                     the correspondence of phenopes and genotypes segregated from a balanced lethal heterozygote. For example, translocation strains
                     segregate a large proportion of aneuploid progeny (<a href="#figure2">Figure 2</a>) that arrest as embryos or early larvae, but these are not the lethal homozygotes of interest. To be certain that the arrested
                     embryos or animals are of the correct genotype, the lethal-bearing chromosome must be crossed to a nonbalanced genetic background
                     and resegregated as a homozygote (<a href="#bib36">Johnsen, R.C., and Baillie, D.L., 1991</a>).
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_7"></a>3.7.&nbsp;Screens for transposon-induced mutations
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The advantages of balancers for lethal mutant screens seem ideal for extension to screens conducted in mutator genetic backgrounds
                  that exhibit a high frequency of Tc1 transposition (<a href="#bib51">Moerman, D.G., and Waterston, R.H., 1989</a>; <a href="#bib5">Anderson, P., 1995</a>). The ease of maintenance and crossing is especially attractive given the necessity of removing the mutator from the background
                  once a mutation has been induced. Although these screens may produce desired mutations, they may not be as generally useful
                  as are screens using other mutagens. Using <span class="emphasis"><em>mut-4(st7000)I</em></span>, <a href="#bib11">Clark, D.V., Johnsen, R.C., McKim, K.S., and Baillie, D.L. (1990</a>) analyzed 28 spontaneous lethal mutations (probably Tc1-induced) in the 49-mu region balanced by <span class="emphasis"><em>nT1</em></span>. The distribution of these mutations was skewed: 86% of them mapped to LG V, as compared with 43% of EMS-induced lethal mutations
                  recovered over <span class="emphasis"><em>nT1</em></span>. Furthermore, the types of mutations obtained were different from those obtained using <a href="http://www.wormbase.org/db/get?name=EMS;class=Cell" target="_blank">EMS</a>, formaldehyde, or gamma irradiation. Two were new mutations in previously unidentified genes, six were mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-40" target="_blank">lin-40</a></em></span>, and seven were deficiencies. All of the latter deleted the left end of the chromosome, and five of them had right endpoints
                  in or near <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-40" target="_blank">lin-40</a></em></span>.
               </p>
               <p>The distribution of transposon-induced mutations is changing with identification methods that do not require that an insertion
                  result in a scorable phenotype (<a href="#bib53">Plasterk, R.H.A., 1995</a>). Also, analysis of secondary deletion mutants derived from strains carrying transposon insertions is resulting in information
                  about the null phenotypes of these genes. Balancers may prove useful in such schemes for maintaining heterozygosity of induced
                  mutations and identifying animals homozygous for the insertions or their spontaneous derivatives.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;The balancer map
                     </h2>
                  </div>
               </div>
            </div>
            <p>Over half the <span class="emphasis"><em>C. elegans</em></span> genome is covered by reliable, tested balancers (<a href="#figure7">Figure 7</a>). Well-balanced regions include virtually all of LG I, the right 65% of LG II, all of LG III, the right half of LG IV, the
               left 66% of LG V, and the right 80% of LG X (based on genetic, not physical, extents of each chromosome). Incompletely balanced
               regions include most of the remaining areas. Balancers may or may not include the physical ends of chromosomes. When drawn
               or referred to as including the end, it means only that the most distal genetic marker appears to be balanced.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center">
               <table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="EdBfig7a.jpg"><img src="EdBfig7a_s.jpg" border="2" align="middle"></a>
  </td>
</tr>
               </table>
            </div>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center">
               <table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="EdBfig7b.jpg"><img src="EdBfig7b_s.jpg" border="2" align="middle"></a>
  </td>
</tr>
               </table>
            </div>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure7"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="EdBfig7c.jpg"><img src="EdBfig7c_s.jpg" border="2" align="middle" alt="Map of the balanced regions of C. elegans chromosomes Map of the balanced regions of C. elegans chromosomes figure 7a"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 7. Map of the balanced regions of <span class="emphasis"><em>C. elegans</em></span> chromosomes.&nbsp;</b>Each chromosome is represented by a vertical bar with cross lines indicating genetic positions of a subset of identified genes
                     (some genes defining the endpoints of minor balancers are not shown where space is limited). By convention, the top end of
                     each vertically drawn chromosome corresponds to the left end when drawn horizontally, and the bottom of each corresponds to
                     the right end. Shading of the chromosomes indicates the degree of balancing. Black represents well-balanced regions, crosshatching
                     represents regions balanced by unstable or uncharacterized rearrangements, and no shading represents regions for which no
                     balancers exist. Approximately 70% of the genome is well balanced. Balancers are drawn to the left of each chromosome as double-line
                     bars (duplications) or bars with different shading patterns (other balancers). Endpoint lines on each balancer indicate its
                     genetic extents. Some rearrangements that provide balancing of limited reliability are not shown. Also not shown are duplications
                     that balance small intervals.
                  </p>
               </div>
            </div>
            <p>Certain chromosomal regions not covered by translocations have proven somewhat resistant to efforts to obtain effective balancers.
               For example, several balancers for the right arm of LG V have been recovered, but all to date have exhibited unacceptable
               levels of crossing over and thus are of limited value for lethal screens (Stewart and Baillie, unpublished results). Such
               incomplete balancers may still provide a useful degree of balancing for existing mutations, and their use should be evaluated
               case by case.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;A field guide to balancers
                     </h2>
                  </div>
               </div>
            </div>
            <p>This guide presents alphabetically the most fully characterized and commonly used <span class="emphasis"><em>C. elegans</em></span> balancers. It is by no means an exhaustive list. The "field guide" format is an attempt to present information critical for
               the use of each balancer in a brief and consistent manner. Standardized descriptors help compare and contrast the choices
               for a particular genetic region.
            </p>
            <p>Most of the balancers listed here, and some of the derivatives and marked variants, are available from the Caenorhabditis
               Genetics Center (CGC). Those variants not stocked by the CGC generally have not been the subject of wide interest, but they
               may prove useful for certain screens or strain constructions. For more information about whether a particular variant is available,
               the investigator should contact the originating laboratory.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_1"></a>5.1.&nbsp;Explanation of descriptors
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Most of the descriptors are self-explanatory, but a few require more detail. For translocations, inversions, and other crossover
                  suppressors, <span class="emphasis"><em>stability</em></span> refers generally to the frequency at which unusual progeny may be segregated. Measures of stability are necessarily subjective,
                  as very little data has been accumulated on the frequency of such progeny and the nature of the events that produced them.
                  The stability descriptor therefore represents a consensus opinion based on published information and the experiences of the
                  authors' laboratories. <span class="emphasis"><em>Extremely stable</em></span> means that neither recombination nor unusual segregants have been documented. <span class="emphasis"><em>Very stable</em></span> means that rare unusual segregants have been recovered, but the utility of the balancer is not generally affected. <span class="emphasis"><em>Stable</em></span> means that unusual segregants have been recovered at somewhat higher frequency, and utility of the balancer is potentially
                  compromised. Details are given where the nature of unusual segregants has been deduced through further analysis.
               </p>
               <p>For all balancers, <span class="emphasis"><em>Recommended use</em></span> is broken down into four categories, presented in increasing stringency of characteristics required in the balancer. <span class="emphasis"><em>General balancing</em></span> means that it is useful for small-scale use. <span class="emphasis"><em>Strain construction</em></span> is listed only for balancers known to be especially useful in crossing protocols because of specific characteristics or the
                  existence of marked variants. By definition, males carrying these balancers must be able to mate. <span class="emphasis"><em>Strain maintenance</em></span> means that a balancer is stable enough for large-scale use, although it may be unsuitable for mutant screens. <span class="emphasis"><em>Mutant screens</em></span> means that a balancer has been proven reliable in screens for lethal mutations.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_2"></a>5.2.&nbsp;The balancers
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_1"></a>5.2.1.&nbsp;<span class="emphasis"><em>eDp6(III;f)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Free duplication, moderately well characterized, not observed to recombine with normal homologs. Very effective balancer
                     for right portion of LG III from right end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=vab-7" target="_blank">vab-7</a></em></span>.
                  </p>
                  <p>Origin: Acetaldehyde mutagenesis.</p>
                  <p>Recommended use: General balancing, strain maintenance.</p>
                  <p>Reference strain: CB1517, <span class="emphasis"><em>eDf2 III; eDp6(III;f)</em></span>.
                  </p>
                  <p>Phenotype: Unc-119.</p>
                  <p>Segregants: Unc-119.</p>
                  <p>Handling: Unc phenotype of reference strain is characteristic of the combination of two copies of <span class="emphasis"><em>eDf2</em></span> and one copy of <span class="emphasis"><em>eDp6</em></span>. This is due to deletion of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> locus by <span class="emphasis"><em>eDf2</em></span> and incomplete complementation of the deletion by the duplication (D. Pilgrim and M. Maduro, pers. comm.).
                  </p>
                  <p>References: <a href="#bib28">Hodgkin, J. (1980</a>; <a href="#bib29">1987</a>); <a href="#bib34">Hunter, C.P., and Wood, W.B., 1990</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_2"></a>5.2.2.&nbsp;<span class="emphasis"><em>eT1(III;V)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Reciprocal translocation, well characterized, extremely stable. Very effective balancer for left portion of LG V
                     from left end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-23" target="_blank">unc-23</a></em></span>, and right portion of LG III from right end to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-36" target="_blank">unc-36</a></em></span>. <span class="emphasis"><em>eT1(III)</em></span> is LG V (left) translocated to LG III (left), disjoins from normal LG III. <span class="emphasis"><em>eT1(V)</em></span> is LG III (right) translocated to LG V (right), disjoins from normal LG V.
                  </p>
                  <p>Origin: 32P mutagenesis of N2.</p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance, mutant screens.</p>
                  <p>Reference strain: BC2200, <span class="emphasis"><em>dpy-18(<a href="http://www.wormbase.org/db/get?name=e364;class=Variation" target="_blank">e364</a>)/eT1 III; unc-46(<a href="http://www.wormbase.org/db/get?name=e177;class=Variation" target="_blank">e177</a>)/eT1 V</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-36 <span class="emphasis"><em>eT1</em></span> homozygotes, Dpy-18 Unc-46, and large numbers of arrested aneuploid progeny.
                  </p>
                  <p>Growth characteristics: Original isolate homozygous viable with Unc-36 phenotype caused by the translocation breakpoint. Brood
                     size in heterozygotes ~ 100, in homozygotes ~160. <span class="emphasis"><em>eT1</em></span> homozygotes can overgrow a population, especially in balanced strains carrying certain lethal mutations or deficiencies.
                  </p>
                  <p>Handling: Easy to manipulate. Heterozygous male stocks mate well.</p>
                  <p>Marked variants: Morphological: <span class="emphasis"><em>eT1[bli-5(<a href="http://www.wormbase.org/db/get?name=s277;class=Variation" target="_blank">s277</a>)]/eT1 [dpy-11(<a href="http://www.wormbase.org/db/get?name=s287;class=Variation" target="_blank">s287</a>)]</em></span>, <span class="emphasis"><em>eT1[sma-2(<a href="http://www.wormbase.org/db/get?name=s262;class=Variation" target="_blank">s262</a>)]</em></span>, <span class="emphasis"><em>eT1[sma-3(<a href="http://www.wormbase.org/db/get?name=e491;class=Variation" target="_blank">e491</a>)]</em></span>, <span class="emphasis"><em>eT1[unc-42(<a href="http://www.wormbase.org/db/get?name=e270;class=Variation" target="_blank">e270</a>)]</em></span>. Lethal: <span class="emphasis"><em>eT1[let-500(<a href="http://www.wormbase.org/db/get?name=s2165;class=Variation" target="_blank">s2165</a>)]</em></span>.
                  </p>
                  <p>References: <a href="#bib60">Rosenbluth, R.E., and Baillie, D.L., 1981</a>; <a href="#bib61">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L. (1983</a>; <a href="#bib62">1985</a>); <a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>; <a href="#bib52">Nelson, G.A., Schubert, W.W., Marshall, T.M., Benton, E.R., and Benton, E.V., 1989</a>; <a href="#bib1">Adames, K.A., Gawne, J., Wicky, C., Muller, F., and Rose, A.M., 1998</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_3"></a>5.2.3.&nbsp;<span class="emphasis"><em>hIn1(I)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Inversion, well characterized, very stable. Very effective balancer for right portion of LG I from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-75" target="_blank">unc-75</a></em></span> through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span>. Extreme right end of chromosome apparently not balanced.
                  </p>
                  <p>Origin: Gamma irradiation of N2 males.</p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance, mutant screens.</p>
                  <p>Reference strain: KR2267, <span class="emphasis"><em>hIn1[unc-54(<a href="http://www.wormbase.org/db/get?name=h1040;class=Variation" target="_blank">h1040</a>)]/unc-75(<a href="http://www.wormbase.org/db/get?name=e950;class=Variation" target="_blank">e950</a>) unc-101(m1) I</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-54 <span class="emphasis"><em>hIn1</em></span> homozygotes, Unc-75 Unc-101.
                  </p>
                  <p>Growth characteristics: Original isolate homozygous viable with wild-type phenotype; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> mutation in the reference strain was induced secondarily. Brood size similar to N2 as homozygote and as heterozygote.
                  </p>
                  <p>Handling: Easy to manipulate. Heterozygous males, and homozygous males of original isolate, mate well. Very rare recombination
                     can occur, which generates deficiencies and duplications.
                  </p>
                  <p>Marked variants: Morphological: <span class="emphasis"><em>hIn1[unc-54(<a href="http://www.wormbase.org/db/get?name=h1040;class=Variation" target="_blank">h1040</a>)]</em></span>, <span class="emphasis"><em>hIn1[unc-101(<a href="http://www.wormbase.org/db/get?name=sy241;class=Variation" target="_blank">sy241</a>)]</em></span>. Lethal: <span class="emphasis"><em>hIn1[unc-75(h1041)]</em></span>, <span class="emphasis"><em>hIn1[unc-75(h1042)]</em></span>, <span class="emphasis"><em>hIn1[unc-54(<a href="http://www.wormbase.org/db/get?name=h1040;class=Variation" target="_blank">h1040</a>) unc-75(h1041)]</em></span>.
                  </p>
                  <p>Derivatives: <span class="emphasis"><em>hDf11</em></span>, <span class="emphasis"><em>hDf12</em></span>, <span class="emphasis"><em>hDp131</em></span>, <span class="emphasis"><em>hDp132</em></span>.
                  </p>
                  <p>References: <a href="#bib37">Lee, J., Jongeward, G.D., and Sternberg, P.W., 1994</a>; <a href="#bib70">Zetka, M.C., and Rose, A.M., 1992</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_4"></a>5.2.4.&nbsp;<span class="emphasis"><em>hT1(I;V)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Reciprocal translocation, well characterized, very stable. Very effective balancer for left portion of LG I from
                     the left end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-80" target="_blank">let-80</a></em></span>, and the left portion of LG V from the left end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-11" target="_blank">dpy-11</a></em></span>. <span class="emphasis"><em>hT1(I)</em></span> is LG V (left) translocated to LG I (right), disjoins from normal LG I. <span class="emphasis"><em>hT1(V)</em></span> is LG I (left) translocated to LG V (right), disjoins from normal LG V.
                  </p>
                  <p>Origin: Gamma irradiation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-23" target="_blank">unc-23</a> + / + <a href="http://www.wormbase.org/db/gene/gene?name=unc-42" target="_blank">unc-42</a></em></span> males.
                  </p>
                  <p>Recommended use: General balancing, strain maintenance, mutant screens.</p>
                  <p>Reference strain: KR1037, <span class="emphasis"><em>unc-13(<a href="http://www.wormbase.org/db/get?name=e51;class=Variation" target="_blank">e51</a>)/hT1 I; dpy-11(<a href="http://www.wormbase.org/db/get?name=e224;class=Variation" target="_blank">e224</a>)/hT1[unc-42(<a href="http://www.wormbase.org/db/get?name=e270;class=Variation" target="_blank">e270</a>)] V</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, arrested <span class="emphasis"><em>hT1</em></span> homozygotes, Unc-13 Dpy-11, and large numbers of arrested aneuploid progeny. Recombination occurs occasionally between the
                     <span class="emphasis"><em>hT1</em></span> breakpoint and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-42" target="_blank">unc-42</a></em></span> on <span class="emphasis"><em>hT1(V)</em></span>, giving rise to Unc-42 animals. These are of genotype <span class="emphasis"><em>unc-13(<a href="http://www.wormbase.org/db/get?name=e51;class=Variation" target="_blank">e51</a>)/hT1 I; dpy-11(<a href="http://www.wormbase.org/db/get?name=e224;class=Variation" target="_blank">e224</a>) unc-42(<a href="http://www.wormbase.org/db/get?name=e270;class=Variation" target="_blank">e270</a>)/hT1[unc-42(<a href="http://www.wormbase.org/db/get?name=e270;class=Variation" target="_blank">e270</a>)]V</em></span>.
                  </p>
                  <p>Growth characteristics: Homozygous inviable, cause unknown. Arrests at L3. Brood size in heterozygotes ~75.</p>
                  <p>Handling: Easy to manipulate. Heterozygous males mate well. Rare exceptional progeny carry one half-translocation as a complex
                     free duplication. Recombination frequency in the unbalanced <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span> - <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> interval on LG I is increased twofold.
                  </p>
                  <p>Marked variants: <span class="emphasis"><em>hT1[unc-29(<a href="http://www.wormbase.org/db/get?name=e403;class=Variation" target="_blank">e403</a>)]</em></span>.
                  </p>
                  <p>Derivatives: <span class="emphasis"><em>hDp133(I;V;f)</em></span>.
                  </p>
                  <p>References: <a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>; <a href="#bib33">Howell, A.M., and Rose, A.M., 1990</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_5"></a>5.2.5.&nbsp;<span class="emphasis"><em>hT2(I;III)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Reciprocal translocation, well characterized, stable. Effective balancer for left portion of LG I from left end through
                     <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span>, and right portion of LG III from right end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-17" target="_blank">dpy-17</a></em></span>. <span class="emphasis"><em>hT2(I)</em></span> is LG III (right) translocated to LG I (right), disjoins from normal LG I. <span class="emphasis"><em>hT2(III)</em></span> is LG I (left) translocated to LG III (left), disjoins from normal LG III.
                  </p>
                  <p>Origin: Gamma irradiation of <span class="emphasis"><em>bli-4(<a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a>)I</em></span> males.
                  </p>
                  <p>Recommended use: General balancing, strain maintenance.</p>
                  <p>Reference strain: KR1235, <span class="emphasis"><em>unc-13(<a href="http://www.wormbase.org/db/get?name=e51;class=Variation" target="_blank">e51</a>)/hT2 I; dpy-18(<a href="http://www.wormbase.org/db/get?name=e364;class=Variation" target="_blank">e364</a>)/hT2[bli-4(<a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a>)] III</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Bli-4 <span class="emphasis"><em>hT2</em></span> homozygotes, Unc-13 Dpy-18, and large numbers of arrested aneuploid progeny.
                  </p>
                  <p>Growth characteristics: Original isolate homozygous viable with Bli-4 phenotype.</p>
                  <p>Handling: Easy to manipulate. Heterozygous males, and homozygous males of original isolate, mate well. Homozygous Bli-4 phenotype
                     completely suppressed in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a></em></span> variants. Rare exceptional progeny carry one half-translocation as a complex free duplication. Mutations have been observed
                     to become unbalanced at low frequency, but the mechanism is not fully understood. The <span class="emphasis"><em>dpy-18(<a href="http://www.wormbase.org/db/get?name=h662;class=Variation" target="_blank">h662</a>)</em></span> variant is very mildly Dpy as a homozygote. <span class="emphasis"><em>hT2[bli-4(<a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a>) let-?(<a href="http://www.wormbase.org/db/get?name=q782;class=Variation" target="_blank">q782</a>) <a href="http://www.wormbase.org/db/misc/etree?name=qIs48;class=Transgene" target="_blank">qIs48</a>]</em></span> carries an integrated pharyngeal GFP element; the lethal mutation in this variant has been observed to recombine away, leaving
                     <span class="emphasis"><em>hT2[bli-4(<a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a>) <a href="http://www.wormbase.org/db/misc/etree?name=qIs48;class=Transgene" target="_blank">qIs48</a>]</em></span> that apparently retains balancer activity.
                  </p>
                  <p>Marked variants: <span class="emphasis"><em>hT2[dpy-5(h659)]</em></span>, <span class="emphasis"><em>hT2[dpy-18(<a href="http://www.wormbase.org/db/get?name=h662;class=Variation" target="_blank">h662</a>)]</em></span>, <span class="emphasis"><em>hT2[unc-54(<a href="http://www.wormbase.org/db/get?name=e190;class=Variation" target="_blank">e190</a>)]</em></span>, <span class="emphasis"><em>hT2[bli-4(<a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a>) unc-29(h1011)]</em></span>, <span class="emphasis"><em>hT2[dpy-18(<a href="http://www.wormbase.org/db/get?name=h662;class=Variation" target="_blank">h662</a>) unc-59(<a href="http://www.wormbase.org/db/get?name=e261;class=Variation" target="_blank">e261</a>)]</em></span>, <span class="emphasis"><em>hT2[bli-4(<a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a>) let-?(<a href="http://www.wormbase.org/db/get?name=q782;class=Variation" target="_blank">q782</a>) <a href="http://www.wormbase.org/db/misc/etree?name=qIs48;class=Transgene" target="_blank">qIs48</a>]</em></span>.
                  </p>
                  <p>Derivative: <span class="emphasis"><em>hDp134 (I;III;f)</em></span>.
                  </p>
                  <p>References: <a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>; J. Kimble, pers. comm.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_6"></a>5.2.6.&nbsp;<span class="emphasis"><em>hT3(I;X)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Translocation (rigorous proof of reciprocity lacking), moderately well characterized, very stable. Very effective
                     balancer for left portion of LG I from left end to around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-363" target="_blank">let-363</a></em></span>, and the right portion of LG X from the right end to between <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-7" target="_blank">dpy-7</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a></em></span>. <span class="emphasis"><em>hT3(I)</em></span>, which disjoins from normal LG I, is probably LG X (right) translocated to LG I (right). <span class="emphasis"><em>hT3(X)</em></span>,which disjoins from normal LG X, is probably LG I (left) translocated to LG X (left).
                  </p>
                  <p>Origin: Gamma irradiation mutagenesis of <span class="emphasis"><em>dpy-5(<a href="http://www.wormbase.org/db/get?name=e61;class=Variation" target="_blank">e61</a>) unc-29(<a href="http://www.wormbase.org/db/get?name=e403;class=Variation" target="_blank">e403</a>)/hT2[dpy-18(<a href="http://www.wormbase.org/db/get?name=h662;class=Variation" target="_blank">h662</a>)] I; +/hT2[bli-4(<a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a>)] III</em></span>. Isolated as a lethal mutation balanced by <span class="emphasis"><em>hT2</em></span>, now maintained as a homozygote balanced by <span class="emphasis"><em>szDp1</em></span>.
                  </p>
                  <p>Recommended use: General balancing, strain maintenance.</p>
                  <p>Reference strain: KR1879, <span class="emphasis"><em>hT3[dpy-5(<a href="http://www.wormbase.org/db/get?name=e61;class=Variation" target="_blank">e61</a>) unc-29(<a href="http://www.wormbase.org/db/get?name=e403;class=Variation" target="_blank">e403</a>)](I;X); szDp1(I;X;f)</em></span>.
                  </p>
                  <p>Phenotype: Unc-29 (<span class="emphasis"><em>szDp1</em></span> complements <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> but not <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-29" target="_blank">unc-29</a></em></span>).
                  </p>
                  <p>Segregants: Unc-29, arrested <span class="emphasis"><em>hT3</em></span> homozygotes.
                  </p>
                  <p>Growth characteristics: Original isolate marked with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-29" target="_blank">unc-29</a></em></span>. Homozygous inviable, probably breaks in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-363" target="_blank">let-363</a> (I)</em></span>. Heterozygotes exhibit reduced viability, low level of X chromosome nondisjunction (1.2%).
                  </p>
                  <p>Handling: Easy to manipulate. Rare exceptional progeny carry one half-translocation as a free duplication. Recombination frequency
                     in unbalanced intervals increased on both LG I and LG X.
                  </p>
                  <p>Marked variants: <span class="emphasis"><em>hT3[dpy-5(<a href="http://www.wormbase.org/db/get?name=e61;class=Variation" target="_blank">e61</a>)]</em></span>.
                  </p>
                  <p>Derivative: <span class="emphasis"><em>hDp135(I;X;f)</em></span>.
                  </p>
                  <p>Reference: <a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_7"></a>5.2.7.&nbsp;<span class="emphasis"><em>mIn1(II)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Inversion, well characterized, very stable. Very effective balancer for center portion of LG II from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-31" target="_blank">lin-31</a></em></span> through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-1" target="_blank">rol-1</a></em></span>.
                  </p>
                  <p>Origin: Gamma irradiation of <span class="emphasis"><em>dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>)/unc-85(<a href="http://www.wormbase.org/db/get?name=e1414;class=Variation" target="_blank">e1414</a>) rol-1(<a href="http://www.wormbase.org/db/get?name=e91;class=Variation" target="_blank">e91</a>)II</em></span>.
                  </p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance, mutant screens.</p>
                  <p>Reference strain: DR1785, <span class="emphasis"><em>mIn1[dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>)]/unc-4(<a href="http://www.wormbase.org/db/get?name=e120;class=Variation" target="_blank">e120</a>)II</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Dpy-10 <span class="emphasis"><em>mIn1</em></span> homozygotes, Unc-4.
                  </p>
                  <p>Growth characteristics: Original isolate marked with <span class="emphasis"><em>dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>)</em></span>. Brood size similar to N2 as heterozygote. Homozygotes do not survive freezing well.
                  </p>
                  <p>Handling: Easy to manipulate. Heterozygous males mate well. <span class="emphasis"><em>mIn1[dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>) <a href="http://www.wormbase.org/db/misc/etree?name=mIs14;class=Transgene" target="_blank">mIs14</a>]</em></span> carries an integrated pharyngeal GFP element; expression is semi-dominant, such that one copy of <span class="emphasis"><em><a href="http://www.wormbase.org/db/misc/etree?name=mIs14;class=Transgene" target="_blank">mIs14</a></em></span> can be distinguished from two by GFP signal brightness. Rearrangement may have occurred in the integration event, as lethal
                     mutations in the far left extent of the balanced region are not as stable over the GFP variant as they are over the original
                     <span class="emphasis"><em>mIn1</em></span>.
                  </p>
                  <p>Marked variants: Morphological: <span class="emphasis"><em>mIn1[unc-4(<a href="http://www.wormbase.org/db/get?name=e120;class=Variation" target="_blank">e120</a>) dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>)], mIn1(+)</em></span>, <span class="emphasis"><em>mIn1[unc-4(<a href="http://www.wormbase.org/db/get?name=e120;class=Variation" target="_blank">e120</a>)]</em></span>, <span class="emphasis"><em>mIn1[rol-1(<a href="http://www.wormbase.org/db/get?name=e91;class=Variation" target="_blank">e91</a>)]</em></span>, <span class="emphasis"><em>mIn1[rol-1(<a href="http://www.wormbase.org/db/get?name=e91;class=Variation" target="_blank">e91</a>) dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>)]</em></span>, <span class="emphasis"><em>mIn1[dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>) mIs14(<a href="http://www.wormbase.org/db/gene/gene?name=myo-2" target="_blank">myo-2</a>::GFP)]</em></span>. Lethal: <span class="emphasis"><em>mIn1[dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>) let-?(<a href="http://www.wormbase.org/db/get?name=m727;class=Variation" target="_blank">m727</a>)]</em></span>.
                  </p>
                  <p>Reference: <a href="#bib14">Edgley, M.L., and Riddle, D.L., 2001</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_8"></a>5.2.8.&nbsp;<span class="emphasis"><em>mT1(II;III)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Reciprocal translocation, well characterized, very stable. Very effective balancer for right portion of LG II from
                     the right end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span>, and the right portion of LG III from the right end to between <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=daf-2" target="_blank">daf-2</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a></em></span>. <span class="emphasis"><em>mT1(II)</em></span> is probably LGII (right) translocated to LG III (left), disjoins from normal LG III. <span class="emphasis"><em>mT1(III)</em></span> is probably LG III (right) translocated to LG II (left), disjoins from normal LG II.
                  </p>
                  <p>Origin: Gamma irradiation of <span class="emphasis"><em>dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>)/unc-85(<a href="http://www.wormbase.org/db/get?name=e1414;class=Variation" target="_blank">e1414</a>) rol-1(<a href="http://www.wormbase.org/db/get?name=e91;class=Variation" target="_blank">e91</a>) II</em></span>.
                  </p>
                  <p>Recommended use: General balancing, strain maintenance, mutant screens.</p>
                  <p>Reference strain: DR1832, <span class="emphasis"><em>mT1/unc-4(<a href="http://www.wormbase.org/db/get?name=e120;class=Variation" target="_blank">e120</a>)II; mT1[dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>)]/dpy-17(<a href="http://www.wormbase.org/db/get?name=e164;class=Variation" target="_blank">e164</a>)III</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, sterile Dpy <span class="emphasis"><em>mT1</em></span> homozygotes, Unc-4 Dpy-17, and large numbers of arrested aneuploid progeny.
                  </p>
                  <p>Growth characteristics: Original isolate marked with <span class="emphasis"><em>dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>)</em></span>. Translocation probably breaks in an essential gene, as <span class="emphasis"><em>mT1</em></span> homozygotes are sterile.
                  </p>
                  <p>Handling: Easy to manipulate. Heterozygous males mate reasonably well.</p>
                  <p>Reference: <a href="#bib14">Edgley, M.L., and Riddle, D.L., 2001</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_9"></a>5.2.9.&nbsp;<span class="emphasis"><em>mnC1(II)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Dominant crossover suppressor, uncharacterized with regard to structure, very stable. Very effective balancer for
                     right portion of LG II from around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a></em></span> to around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span>.
                  </p>
                  <p>Origin: X-ray mutagenesis of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a>/<a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span>.
                  </p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance, mutant screens.</p>
                  <p>Reference strain: SP127, <span class="emphasis"><em>unc-4(<a href="http://www.wormbase.org/db/get?name=e120;class=Variation" target="_blank">e120</a>)/mnC1[dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>) unc-52(<a href="http://www.wormbase.org/db/get?name=e444;class=Variation" target="_blank">e444</a>)]</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-4, Dpy-10 Unc-52 <span class="emphasis"><em>mnC1</em></span> homozygotes. The homozygotes are short, fat, and paralyzed, whereas the Unc-4 segregants are large and healthy and move forward
                     well, but cannot back up.
                  </p>
                  <p>Growth characteristics: Homozygotes viable with extremely small broods (average: 7 progeny). Brood size in heterozygotes ~200,
                     with 100% egg hatching.
                  </p>
                  <p>Handling: Easy to manipulate. Heterozygous male stocks mate well. Very rare recombination gives rise to Dpy non-Unc and Unc
                     non-Dpy progeny. The recombinant chromosomes carried by these progeny are homozygous lethal.
                  </p>
                  <p>References: <a href="#bib20">Herman, R.K., 1978</a>; <a href="#bib65">Sigurdson, D.C., Spanier, G.J., and Herman, R.K., 1984</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_10"></a>5.2.10.&nbsp;<span class="emphasis"><em>mnDp1(X;V)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Translocated duplication, well characterized, does not recombine with normal homologues. Very effective balancer
                     for right portion of LG X from right end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-4" target="_blank">let-4</a></em></span>.
                  </p>
                  <p>Origin: ~7500-R X-ray mutagenesis of N2 males.</p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance, mutant screens.</p>
                  <p>Reference strain: SP219, <span class="emphasis"><em>mnDp1(X;V)/+ V; unc-3(<a href="http://www.wormbase.org/db/get?name=e151;class=Variation" target="_blank">e151</a>)X</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-3, sterile <span class="emphasis"><em>mnDp1</em></span> homozygotes.
                  </p>
                  <p>Growth characteristics: Brood size of reference strain ~235. Unc-3 progeny constitute 25% of total, and about a third of total
                     are slow-growing, sterile duplication homozygotes.
                  </p>
                  <p>Handling: Easy to manipulate. Males mate well. <span class="emphasis"><em>mnDp1</em></span> is attached to LG V near the left end, and severely reduces recombination frequency from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-60" target="_blank">unc-60</a></em></span> to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-11" target="_blank">dpy-11</a></em></span>.
                  </p>
                  <p>References: <a href="#bib24">Herman, R.K., Albertson, D.G., and Brenner, S., 1976</a>; <a href="#bib27">Herman, R.K., Madl, J.E., and Kari, C.K., 1979</a>; <a href="#bib46">Meneely P.M., and Herman, R.K., 1981</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_11"></a>5.2.11.&nbsp;<span class="emphasis"><em>mnDp3(X;f)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Free duplication, well characterized, does not recombine with normal LG X. Effective balancer for right portion of
                     LG X from around the right end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-9" target="_blank">unc-9</a></em></span>.
                  </p>
                  <p>Origin: ~7500-R X-ray mutagenesis of N2 males.</p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance.</p>
                  <p>Reference strain: SP123, <span class="emphasis"><em>unc-3(<a href="http://www.wormbase.org/db/get?name=e151;class=Variation" target="_blank">e151</a>)X; mnDp3(X;f)</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-3.</p>
                  <p>Growth characteristics: Brood size ~275, duplication transmitted to ~65% of progeny.</p>
                  <p>Handling: Easy to manipulate. Males carrying duplication mate well. Shows some tendency for somatic loss. Marked variant:
                     <span class="emphasis"><em>mnDp3[sup-10(<a href="http://www.wormbase.org/db/get?name=mn338;class=Variation" target="_blank">mn338</a>)]</em></span>.
                  </p>
                  <p>References: <a href="#bib24">Herman, R.K., Albertson, D.G., and Brenner, S., 1976</a>; <a href="#bib27">Herman, R.K., Madl, J.E., and Kari, C.K., 1979</a>; <a href="#bib21">Herman, R.K., 1984</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_12"></a>5.2.12.&nbsp;<span class="emphasis"><em>mnDp10(X;I)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Translocated duplication, well characterized, does not recombine with normal LG X. Effective balancer for right portion
                     of LG X from around the right end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-2" target="_blank">lin-2</a></em></span>.
                  </p>
                  <p>Origin: ~7500-R X-ray mutagenesis of N2 males.</p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance.</p>
                  <p>Reference strain: SP117, <span class="emphasis"><em>mnDp10(X;I); unc-3(<a href="http://www.wormbase.org/db/get?name=e151;class=Variation" target="_blank">e151</a>)X</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, rare Unc-3.</p>
                  <p>Growth characteristics: Duplication is homozygous viable. Unc-3 animals arise from somatic loss of duplication.</p>
                  <p>Handling: Easy to manipulate. Males carrying the duplication mate well. <span class="emphasis"><em>mnDp10</em></span> shows some tendency to segregate from the X chromosome in male spermatogenesis.
                  </p>
                  <p>References: <a href="#bib27">Herman, R.K., Madl, J.E., and Kari, C.K., 1979</a>; <a href="#bib46">Meneely P.M., and Herman, R.K., 1981</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_13"></a>5.2.13.&nbsp;<span class="emphasis"><em>mnDp33(X;IV)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Translocated duplication, well characterized, does not recombine with normal X. Effective balancer for small region
                     of left portion of LG X, from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-18" target="_blank">lin-18</a></em></span> to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-5" target="_blank">osm-5</a></em></span> (inclusive).
                  </p>
                  <p>Origin: ~7500 R X-ray mutagenesis of N2 males.</p>
                  <p>Recommended use: General balancing; strain construction, strain maintenance.</p>
                  <p>Reference strain: SP309, <span class="emphasis"><em>mnDp33(X;IV); unc-20(<a href="http://www.wormbase.org/db/get?name=e112;class=Variation" target="_blank">e112</a>)X</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-20, arrested L1 or L2 larvae (duplication homozygotes).</p>
                  <p>Growth characteristics: Homozygous inviable. ~100% egg hatching, but 25% arrest.</p>
                  <p>Handling: Easy to manipulate. Males carrying the duplication mate well.</p>
                  <p>Reference: <a href="#bib27">Herman, R.K., Madl, J.E., and Kari, C.K., 1979</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_14"></a>5.2.14.&nbsp;<span class="emphasis"><em>mnDp34-36(II)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Independently isolated free duplications with apparently identical genetic extents. Well characterized, do not recombine
                     with normal homologues. Very effective balancers for right portion of LG II from around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span> through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-53" target="_blank">unc-53</a></em></span>.
                  </p>
                  <p>Origin: ~7500-R X-ray mutagenesis of <span class="emphasis"><em>mnC1[dpy-10(<a href="http://www.wormbase.org/db/get?name=e128;class=Variation" target="_blank">e128</a>) unc-52(<a href="http://www.wormbase.org/db/get?name=e444;class=Variation" target="_blank">e444</a>)]/ unc-4(<a href="http://www.wormbase.org/db/get?name=e120;class=Variation" target="_blank">e120</a>)</em></span>.
                  </p>
                  <p>Recommended use: General balancing, strain maintenance.</p>
                  <p>Reference strains:
                     					
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td><span class="emphasis"><em>mnDp34</em></span>: SP306, <span class="emphasis"><em>mnC1[<a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>]/<a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>; mnDp34</em></span>.
                        </td>
                     </tr>
                     <tr>
                        <td><span class="emphasis"><em>mnDp35</em></span>: SP307, <span class="emphasis"><em>mnC1[<a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>]/<a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>; mnDp35</em></span>.
                        </td>
                     </tr>
                     <tr>
                        <td><span class="emphasis"><em>mnDp36</em></span>: SP308, <span class="emphasis"><em>mnC1[<a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>]/<a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>; mnDp36</em></span>.
                        </td>
                     </tr>
                  </table>
                  <p>
                     				
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Dpy-l0 Unc-52 (<span class="emphasis"><em>mnC1[<a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>]</em></span> homozygotes), Unc-52 (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span> homozygotes and <span class="emphasis"><em>mnC1[<a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>]/<a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span>), Unc-4 (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>; mnDpx</em></span>), Dpy-l0 (<span class="emphasis"><em>mnC1[<a href="http://www.wormbase.org/db/gene/gene?name=dpy-10" target="_blank">dpy-10</a> <a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a>]; mnDpx</em></span>).
                  </p>
                  <p>Growth characteristics: Egg hatching nearly 100%. About 20% of these arrest development as early larvae, and are presumably
                     duplication homozygotes. The duplications are transmitted to about 40% of progeny.
                  </p>
                  <p>Handling: These duplications tend to segregate from the X chromosome in male spermatogenesis.</p>
                  <p>Reference: <a href="#bib27">Herman, R.K., Madl, J.E., and Kari, C.K., 1979</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_15"></a>5.2.15.&nbsp;<span class="emphasis"><em>nT1(IV;V)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Translocation, moderately well characterized, very stable. Very effective balancer for right portion of LG IV from
                     right end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span>, and for left portion of LG V from left end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-76" target="_blank">unc-76</a></em></span>.
                  </p>
                  <p>Origin: Spontaneous.</p>
                  <p>Recommended use: General balancing, strain maintenance, mutant screens.</p>
                  <p>Reference strain: MTl000, <span class="emphasis"><em>unc-5(<a href="http://www.wormbase.org/db/get?name=e53;class=Variation" target="_blank">e53</a>)/nTl IV; dpy-11(<a href="http://www.wormbase.org/db/get?name=e224;class=Variation" target="_blank">e224</a>)/nTl V</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, vulvaless <span class="emphasis"><em>nT1</em></span> homozygotes that become bags of worms, Unc-5 Dpy-11, and large numbers of arrested aneuploid progeny.
                  </p>
                  <p>Growth characteristics: Brood size in <span class="emphasis"><em>nT1</em></span> heterozygotes ~100.
                  </p>
                  <p>Handling: Easy to manipulate. Heterozygous males mate well. Vulvaless homozygous hermaphrodites completely unable to mate.
                     Cause of vulvaless phenotype unknown. Translocation may break down spontaneously, but analysis of such events is lacking.
                     Heterozygous strains occasionally begin to segregate large numbers of sick-looking progeny while appearing to remain heterozygous,
                     or they occasionally begin to give larger broods (Schein and Baillie, unpublished results). <span class="emphasis"><em>nT1[<a href="http://www.wormbase.org/db/misc/etree?name=qIs51;class=Transgene" target="_blank">qIs51</a>]</em></span> carries an integrated pharyngeal GFP element, and is homozygous inviable; this variant appears to be transferred to male
                     cross progeny more frequently than to hermaphrodite cross progeny.
                  </p>
                  <p>Marked variants: <span class="emphasis"><em>nT1[unc(n754dm) let]</em></span> (previously called <span class="emphasis"><em>DnT1</em></span>, dominant Unc, recessive Let); <span class="emphasis"><em>nT1[<a href="http://www.wormbase.org/db/misc/etree?name=qIs51;class=Transgene" target="_blank">qIs51</a>]; nTl[let(<a href="http://www.wormbase.org/db/get?name=m435;class=Variation" target="_blank">m435</a>)]</em></span>.
                  </p>
                  <p>References: <a href="#bib15">Ferguson, E.L., and Horvitz, H.R., 1985</a>; <a href="#bib12">Clark, D.V., Rogalski, T.M., Donati, L.M., and Baillie, D.L., 1988</a>; <a href="#bib57">Rogalski, T.M., and Riddle, D.L., 1988</a>; J. Kimble, pers. comm.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_16"></a>5.2.16.&nbsp;<span class="emphasis"><em>qC1(III)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Dominant crossover suppressor, uncharacterized with regard to structure, very stable. Very effective balancer for
                     the left portion of LG III from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> to at least <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-1" target="_blank">dpy-1</a></em></span>.
                  </p>
                  <p>Origin: 7200-R gamma-irradiation mutagenesis of <span class="emphasis"><em>unc-32(<a href="http://www.wormbase.org/db/get?name=e189;class=Variation" target="_blank">e189</a>)/dpy-19(<a href="http://www.wormbase.org/db/get?name=e1259;class=Variation" target="_blank">e1259</a>)</em></span>.
                  </p>
                  <p>Recommended use: General balancing, strain maintenance.</p>
                  <p>Reference strain: JK1122, <span class="emphasis"><em>dpy-17(<a href="http://www.wormbase.org/db/get?name=e164;class=Variation" target="_blank">e164</a>) unc-32(<a href="http://www.wormbase.org/db/get?name=e189;class=Variation" target="_blank">e189</a>)/qC1[dpy-19(<a href="http://www.wormbase.org/db/get?name=e1259;class=Variation" target="_blank">e1259</a>) glp-1(<a href="http://www.wormbase.org/db/get?name=q339;class=Variation" target="_blank">q339</a>)] III</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Dpy-17 Unc-32 homozygotes, sterile <span class="emphasis"><em>ts</em></span> Dpy-19.
                  </p>
                  <p>Handling: Relatively easy to manipulate. Heterozygous males mate and transfer <span class="emphasis"><em>qC1</em></span>, though perhaps at low frequency. <span class="emphasis"><em>qC1</em></span> fails to complement <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mog-1" target="_blank">mog-1</a></em></span> mutations.
                  </p>
                  <p>Marked variants: <span class="emphasis"><em>qC1[dpy-19(<a href="http://www.wormbase.org/db/get?name=e1259;class=Variation" target="_blank">e1259</a>) glp-1(<a href="http://www.wormbase.org/db/get?name=q339;class=Variation" target="_blank">q339</a>) <a href="http://www.wormbase.org/db/misc/etree?name=qIs26;class=Transgene" target="_blank">qIs26</a>]</em></span>.
                  </p>
                  <p>References: <a href="#bib6">Austin, J., and Kimble, J., 1989</a>; J. Austin, E. Goodwin, and J. Kimble, pers. comm.; Ha and Baillie, unpublished results.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_17"></a>5.2.17.&nbsp;<span class="emphasis"><em>sC1(III)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Dominant crossover suppressor, uncharacterized with regard to structure, very stable. Very effective balancer for
                     an approximately 15-mu portion of LG III from around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-45" target="_blank">unc-45</a></em></span> to near <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=daf-2" target="_blank">daf-2</a></em></span>.
                  </p>
                  <p>Origin: 2000-R gamma-irradiation mutagenesis of N2 males.</p>
                  <p>Recommended use: General balancing, strain maintenance, mutant screens.</p>
                  <p>Reference strain: BC4279, <span class="emphasis"><em>sC1[dpy-l (<a href="http://www.wormbase.org/db/get?name=s2170;class=Variation" target="_blank">s2170</a>)] III</em></span>.
                  </p>
                  <p>Phenotype: Dpy-l.</p>
                  <p>Segregants: Dpy-l (homozygous strain).</p>
                  <p>Growth characteristics: Original isolate wild type. Broods approximately wild type in size.</p>
                  <p>Handling: Easy to manipulate. Heterozygous males, and homozygous males of original isolate, mate well.</p>
                  <p>Marked variants: <span class="emphasis"><em>sC1[dpy-1(<a href="http://www.wormbase.org/db/get?name=s2171;class=Variation" target="_blank">s2171</a>)]; sC1[dpy-1(<a href="http://www.wormbase.org/db/get?name=s2171;class=Variation" target="_blank">s2171</a>) let]</em></span>.
                  </p>
                  <p>Reference: Stewart and Baillie, unpublished results.</p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_18"></a>5.2.18.&nbsp;<span class="emphasis"><em>sC4(V)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Dominant crossover suppressor, uncharacterized with regard to structure, moderately stable. Balances right portion
                     of LG V from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-9" target="_blank">rol-9</a></em></span> to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-76" target="_blank">unc-76</a></em></span>.
                  </p>
                  <p>Origin: 2000-R gamma-irradiation mutagenesis of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-21" target="_blank">dpy-21</a>/<a href="http://www.wormbase.org/db/gene/gene?name=unc-76" target="_blank">unc-76</a> <a href="http://www.wormbase.org/db/gene/gene?name=rol-9" target="_blank">rol-9</a></em></span>.
                  </p>
                  <p>Recommended use: General balancing.</p>
                  <p>Reference strain: BC4586, <span class="emphasis"><em>sC4[<a href="http://www.wormbase.org/db/gene/gene?name=dpy-21" target="_blank">dpy-21</a>]/<a href="http://www.wormbase.org/db/gene/gene?name=unc-76" target="_blank">unc-76</a> <a href="http://www.wormbase.org/db/gene/gene?name=rol-9" target="_blank">rol-9</a></em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-76 Rol-9, arrested <span class="emphasis"><em>sC4</em></span> homozygotes. Presence of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-21" target="_blank">dpy-21</a></em></span> on <span class="emphasis"><em>sC4</em></span> not confirmed.
                  </p>
                  <p>Growth characteristics: Homozygous inviable. Average brood size 117. Cause of homozygous lethal phenotype unknown. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-76" target="_blank">unc-76</a></em></span> - <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-9" target="_blank">rol-9</a></em></span> genetic distance reduced to 1.8%.
                  </p>
                  <p>Handling: Easy to manipulate. Heterozygous males mate well.</p>
                  <p>Reference: Stewart and Baillie, unpublished results.</p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_19"></a>5.2.19.&nbsp;<span class="emphasis"><em>sDp2(I;f)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Free duplication, well characterized, does not recombine with normal homologues. Very effective balancer for the
                     left portion of LG I from the left end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-15" target="_blank">unc-15</a></em></span> (just left of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span>).
                  </p>
                  <p>Origin: 7500-R gamma-irradiation mutagenesis of N2 males.</p>
                  <p>Recommended use: General balancing, strain maintenance, mutant screens.</p>
                  <p>Reference strain: KR236, <span class="emphasis"><em>dpy-5(<a href="http://www.wormbase.org/db/get?name=e61;class=Variation" target="_blank">e61</a>) unc-13(<a href="http://www.wormbase.org/db/get?name=e450;class=Variation" target="_blank">e450</a>)I; sDp2(I,f)</em></span>.
                  </p>
                  <p>Phenotype: Unc-13.</p>
                  <p>Segregants: Unc-13, Dpy-5 Unc-13.</p>
                  <p>Growth characteristics: Both Unc-13 and Dpy-5 Unc-13 animals are slow growing (generation time at 20<span class="symbol">&deg;</span>C nearly 5 days). Animals carrying two copies of sDp2 have never been recovered.
                  </p>
                  <p>Handling: sDp2-bearing males mate and give some progeny, but are slow growing and do not compete well with non-Dp males in
                     mating.
                  </p>
                  <p>Derivatives: <span class="emphasis"><em>hDp2-hDp30; hDp59; hDp74-hDp77</em></span>.
                  </p>
                  <p>References: <a href="#bib59">Rose, A.M., Baillie, D.L., and Curran, J., 1984</a>; <a href="#bib32">Howell, A.M., Gilmour, S.G., Mancebo, R.A., and Rose, A.M., 1987</a>; <a href="#bib33">Howell, A.M., and Rose, A.M., 1990</a>; <a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>; <a href="#bib42">McKim, K.S., Starr, T., and Rose, A.M., 1992</a>; <a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_20"></a>5.2.20.&nbsp;<span class="emphasis"><em>sDp3(III;f)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Free duplication, well characterized, does not recombine with normal homologues. Very effective balancer for left
                     portion of LG III from around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-86" target="_blank">unc-86</a></em></span> through at least <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-1" target="_blank">dpy-1</a></em></span> (does not extend to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-45" target="_blank">unc-45</a></em></span>).
                  </p>
                  <p>Origin: 1500-R gamma-irradiation mutagenesis of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a>/eT1 III; <a href="http://www.wormbase.org/db/gene/gene?name=unc-46" target="_blank">unc-46</a>/ eT1 V</em></span>.
                  </p>
                  <p>Recommended use: General balancing, strain maintenance, mutant screens.</p>
                  <p>Reference strain: BC986, <span class="emphasis"><em>eT1(III;V); sDp3(III,f).</em></span></p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-36 <span class="emphasis"><em>eT1</em></span> homozygotes.
                  </p>
                  <p>Handling: Easy to manipulate. Males mate well. Duplication homozygotes are probably inviable (never recovered).</p>
                  <p>Reference: <a href="#bib62">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L., 1985</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_21"></a>5.2.21.&nbsp;<span class="emphasis"><em>stDp2(X;II)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Translocated duplication, moderately well characterized, not observed to recombine with normal LG X. Very effective
                     balancer for small region in the center of LG X, from around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-58" target="_blank">unc-58</a></em></span> to around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-6" target="_blank">unc-6</a></em></span>.
                  </p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance.</p>
                  <p>Reference strain: RW6002, <span class="emphasis"><em>+/stDp2 II; unc-18(<a href="http://www.wormbase.org/db/get?name=e81;class=Variation" target="_blank">e81</a>)X</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-18.</p>
                  <p>Growth characteristics: Homozygous inviable.</p>
                  <p>Handling: Easy to manipulate. Males carrying one copy of the duplication mate well.</p>
                  <p>Reference: <a href="#bib48">Meneely, P.M. and Wood, W.B., 1984</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_22"></a>5.2.22.&nbsp;<span class="emphasis"><em>szDp1(I;X;f)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Complex free duplication, well characterized, does not recombine with normal LG I. Very effective balancer for the
                     left portion of LG I from the left end through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span>. Consists of one half-translocation [<span class="emphasis"><em>szT1(X)</em></span>] from <span class="emphasis"><em>szT1</em></span> maintained in addition to a normal chromosome complement.
                  </p>
                  <p>Origin: Exceptional Unc-3 segregant from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-5" target="_blank">dpy-5</a>/szT1[<a href="http://www.wormbase.org/db/gene/gene?name=lon-2" target="_blank">lon-2</a>] I; <a href="http://www.wormbase.org/db/gene/gene?name=unc-3" target="_blank">unc-3</a>/szT1 X</em></span>.
                  </p>
                  <p>Reference strain: KRI577, <span class="emphasis"><em>dpy-5(<a href="http://www.wormbase.org/db/get?name=e61;class=Variation" target="_blank">e61</a>) unc-13(<a href="http://www.wormbase.org/db/get?name=e450;class=Variation" target="_blank">e450</a>)I; szDpI(I;x,f)</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild-type hermaphrodites, small percentage wild-type males, Dpy-5 Unc-13.</p>
                  <p>Growth characteristics: Animals carrying two copies of <span class="emphasis"><em>szDp1</em></span> apparently inviable. Duplication strains give rise to spontaneous males through meiotic nondisjunction of the X chromosome.
                  </p>
                  <p>Handling: <span class="emphasis"><em>szDp1</em></span>-bearing males either do not mate or are infertile.
                  </p>
                  <p>Derivatives: <span class="emphasis"><em>hDp31-hDp58; hDp60-hDp73</em></span>.
                  </p>
                  <p>References: <a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>; <a href="#bib42">McKim, K.S., Starr, T., and Rose, A.M., 1992</a>; <a href="#bib41">McKim, K.S., Peters, K., and Rose, A.M., 1993</a>; <a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_23"></a>5.2.23.&nbsp;<span class="emphasis"><em>szT1(I;X)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Reciprocal translocation, well characterized, very stable. Effective balancer for left portion of LG I through <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span>, nearly all of LG X from right end to around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-3" target="_blank">dpy-3</a></em></span>. <span class="emphasis"><em>szT1(I)</em></span> is large segment of LG X (right) translocated to LG I, disjoins from normal LG I. <span class="emphasis"><em>szT1(X)</em></span> is LG I (left) translocated to fragment of LG X (left), disjoins from normal LG X.
                  </p>
                  <p>Origin: 7000-R gamma-ray mutagenesis of <span class="emphasis"><em>lon-2(<a href="http://www.wormbase.org/db/get?name=e678;class=Variation" target="_blank">e678</a>)/dpy-8(<a href="http://www.wormbase.org/db/get?name=e1321;class=Variation" target="_blank">e1321</a>) unc-3(<a href="http://www.wormbase.org/db/get?name=e151;class=Variation" target="_blank">e151</a>)</em></span>hermaphrodites.
                  </p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance.</p>
                  <p>Reference strain: AFl, <span class="emphasis"><em>+/szT1[lon-2(<a href="http://www.wormbase.org/db/get?name=e678;class=Variation" target="_blank">e678</a>) I; dpy-8(<a href="http://www.wormbase.org/db/get?name=e1321;class=Variation" target="_blank">e1321</a>) unc-3(<a href="http://www.wormbase.org/db/get?name=e151;class=Variation" target="_blank">e151</a>)/szT1 X</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Dpy-8 Unc-3 hermaphrodites and males, embryonic lethal <span class="emphasis"><em>szT1</em></span> homozygotes, large numbers of embryonic or early larval arrest aneuploid progeny, and about 10% Lon-2 males.
                  </p>
                  <p>Growth characteristics: Homozygous inviable; breakpoint resulting in lethality must interrupt a gene on LG I, as hemizygous
                     males are viable and fertile. Lon-2 males arise through meiotic nondisjunction of X chromosome. Brood size in heterozygotes
                     ~100.
                  </p>
                  <p>Handling: Easy to manipulate. Lon-2 <span class="emphasis"><em>szT1</em></span> males mate well. Rare exceptional progeny carry one half-translocation as a complex free duplication. Gives rise spontaneously
                     to rare apparent lethal mutations that may represent fusion of <span class="emphasis"><em>szT1(X)</em></span> and the normal X. Shows threefold enhanced recombination frequency immediately adjacent to right of LG I breakpoint and about
                     twofold enhanced frequency in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span> - <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> interval.
                  </p>
                  <p>Marked variant: <span class="emphasis"><em>szT1[lon-2(<a href="http://www.wormbase.org/db/get?name=e678;class=Variation" target="_blank">e678</a>) unc-29(<a href="http://www.wormbase.org/db/get?name=e403;class=Variation" target="_blank">e403</a>)]</em></span>.
                  </p>
                  <p>References: <a href="#bib17">Fodor, A., and Deak, P., 1985</a>; <a href="#bib71">McKim, K.S., Howell, A.M., and Rose, A.M., 1988</a>; <a href="#bib33">Howell, A.M., and Rose, A.M., 1990</a>; <a href="#bib40">McKim, K.S., and Rose, A.M., 1990</a>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_24"></a>5.2.24.&nbsp;<span class="emphasis"><em>yDp1(IV;V;f)</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Summary: Complex free duplication, moderately well characterized, not observed to recombine with either normal homologue.
                     Very effective balancer for right portion of LG IV from around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-4" target="_blank">dpy-4</a></em></span> to around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a></em></span>, and left portion of LG V from around <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-34" target="_blank">unc-34</a></em></span> through <span class="emphasis"><em>yDp1</em></span>.
                  </p>
                  <p>Origin: <a href="http://www.wormbase.org/db/get?name=EMS;class=Cell" target="_blank">EMS</a> mutagenesis of <span class="emphasis"><em>unc-22(s7) dpy-26(<a href="http://www.wormbase.org/db/get?name=n199;class=Variation" target="_blank">n199</a>)IV/nT1[unc(<a href="http://www.wormbase.org/db/get?name=n754;class=Variation" target="_blank">n754</a>) let](IV;V)</em></span>.
                  </p>
                  <p>Recommended use: General balancing, strain construction, strain maintenance.</p>
                  <p>Reference strain: TY156, <span class="emphasis"><em>unc-30(<a href="http://www.wormbase.org/db/get?name=e191;class=Variation" target="_blank">e191</a>) dpy-4(<a href="http://www.wormbase.org/db/get?name=e1166;class=Variation" target="_blank">e1166</a>) IV; yDp1(IV;V,f)</em></span>.
                  </p>
                  <p>Phenotype: Wild type.</p>
                  <p>Segregants: Wild type, Unc-30 Dpy-4.</p>
                  <p>Growth characteristics: Brood size ~250. Animals carrying two copies of <span class="emphasis"><em>yDp1</em></span> apparently inviable. The duplication segregates from X chromosome in male meiosis.
                  </p>
                  <p>Handling: Easy to manipulate. Males carrying the duplication mate well.</p>
                  <p>References: <a href="#bib13">DeLong, L., Casson, L.P., and Meyer, B.J., 1987</a>; <a href="#bib54">Plenefisch, J.D., DeLong, L., and Meyer, B.J., 1989</a>; <a href="#bib69">Yuan, J.Y., and Horvitz, H.R., 1990</a>.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Adames, K.A., Gawne, J., Wicky, C., Muller, F., and Rose, A.M. (1998). Mapping a telomere using the translocation <span class="emphasis"><em>eT1(III;V)</em></span> in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>150</em></span>, 1059&#8211;1066.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9799258&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Albertson, D.G., Fishpool, R.M., and Birchall, P.S. (1995). Fluorescence <span class="emphasis"><em>in situ</em></span> hybridization for the detection of DNA and RNA. In: <span class="emphasis"><em>Methods in Cell Biology, Vol. 48</em></span>, H.F. Epstein and D.C. Shakes, eds. (San Diego, CA: Academic Press), pp. 339&#8211;364.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531734&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Albertson, D.G., and Thomson, J.N. (1982). The kinetochores of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Chromosoma <span class="emphasis"><em>86</em></span>, 409&#8211;428.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7172865&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00292267" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Albertson, D.G., and Thomson, J.N. (1993). Segregation of holocentric chromosomes at meiosis in the nematode, <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Chromosome Res. <span class="emphasis"><em>1</em></span>, 15&#8211;26.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8143084&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00710603" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Anderson, P. (1995). Mutagenesis. In: <span class="emphasis"><em>Methods in Cell Biology, Vol. 48</em></span>, H.F. Epstein and D.C. Shakes, eds. (San Diego, CA: Academic Press), pp. 31&#8211;54.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531732&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     <span class="bibliomisc">Austin, J., and Kimble, J. (1989). Transcript analysis of <span class="emphasis"><em>glp-l</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span>, homologous genes required for cell interactions during development of <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>58</em></span>, 565&#8211;571.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2758467&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(89)90437-6" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Brenner, S. (1974). The genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>77</em></span>, 71&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4366476&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bucher, E.A., and Greenwald, I. (1991). A genetic mosaic screen of essential zygotic genes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>128</em></span>, 281&#8211;292.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2071016&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium (1998). Genome sequence of the nematode <span class="emphasis"><em>C. elegans</em></span>: a platform for investigating biology. Science <span class="emphasis"><em>282</em></span>, 2012&#8211;2018.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851916&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.282.5396.2012" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chalfie M., Tu, Y., Euskirchen, G., Ward W.W., and Prasher, D.C. (1994). Green fluorescent protein as a marker for gene expression.
                        Science <span class="emphasis"><em>263</em></span>, 802&#8211;805</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8303295&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Clark, D.V., Johnsen, R.C., McKim, K.S., and Baillie, D.L. (1990). Analysis of lethal mutations induced in a mutator strain
                        that activates transposable elements in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome <span class="emphasis"><em>33</em></span>, 109&#8211;114.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2158925&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Clark, D.V., Rogalski, T.M., Donati, L.M., and Baillie, D.L. (1988). The <span class="emphasis"><em>unc-22(IV)</em></span> region of <span class="emphasis"><em>Caenorhabditis elegans</em></span>: genetic analysis of lethal mutations. Genetics <span class="emphasis"><em>119</em></span>, 345&#8211;353.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3396868&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">DeLong, L., Casson, L.P., and Meyer, B.J. (1987). Assessment of X-chromosome dosage compensation in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by phenotypic analysis of <span class="emphasis"><em>lin-l4</em></span>. Genetics <span class="emphasis"><em>117</em></span>, 657&#8211;670.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3428573&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Edgley, M.L., and Riddle, D.L. (2001). LG II balancer chromosomes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: <span class="emphasis"><em>mT1(II;III)</em></span> and the <span class="emphasis"><em>mIn1</em></span> set of dominantly and recessively marked inversions. Mol. Genet. Genomics <span class="emphasis"><em>266</em></span>(3), 385&#8211;395.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11713668&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s004380100523" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ferguson, E.L., and Horvitz, H.R. (1985). Identification and characterization of 22 genes that affect the vulval cell lineages
                        of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>110</em></span>, 17&#8211;72.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3996896&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Fire, A. (1986). Integrative transformation of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. EMBO J. <span class="emphasis"><em>5</em></span>, 2673&#8211;2680.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Fodor, A., and Deak, P. (1985). The isolation and genetic analysis of a <span class="emphasis"><em>Caenorhabditis elegans</em></span> translocation (<span class="emphasis"><em>szT1</em></span>) strain bearing an X-chromosome balancer. J. Genet. <span class="emphasis"><em>64</em></span>, 143&#8211;157.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3224815&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Greenwald, I.S., and Horvitz, H.R. (1980). <span class="emphasis"><em>unc-93(<a href="http://www.wormbase.org/db/get?name=e1500;class=Variation" target="_blank">e1500</a>)</em></span>: a behavioral mutant of <span class="emphasis"><em>Caenorhabditis elegans</em></span> that defines a gene with a wild-type null phenotype. Genetics <span class="emphasis"><em>96</em></span>, 147&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6894129&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Greenwald, I.S., Sternberg, P.W., and Horvitz, H.R. (1983). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> locus specifies cell fates in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>34</em></span>, 435&#8211;444.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6616618&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(83)90377-X" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Herman, R.K. (1978). Crossover suppressors and balanced recessive lethals in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>88</em></span>, 49&#8211;65.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=631558&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     <span class="bibliomisc">Herman, R.K. (1984). Analysis of genetic mosaics of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>108</em></span>, 165&#8211;180.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6434374&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Herman, R.K. (1987). Mosaic analysis of two genes that affect nervous system structure in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>116</em></span>, 377&#8211;388.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3609726&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Herman, R.K. (1995). Mosaic analysis. In: <span class="emphasis"><em>Methods in Cell Biology, Vol. 48</em></span>, H.F. Epstein and D.C. Shakes, eds. (San Diego, CA: Academic Press), pp. 123&#8211;143.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531723&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Herman, R.K., Albertson, D.G., and Brenner, S. (1976). Chromosome rearrangements in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>83</em></span>, 91&#8211;105.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1269921&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Herman, R.K., and Kari, C.K. (1989). Recombination between small X chromosome duplications and the X chromosome in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>121</em></span>, 723&#8211;737.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2721932&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Herman, R.K., Kari, C.K., and Hartman, P.S. (1982). Dominant X-chromosome nondisjunction mutants of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>102</em></span>, 379&#8211;400.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6890921&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Herman, R.K., Madl, J.E., and Kari, C.K. (1979). Duplications in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>92</em></span>, 419&#8211;435.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=573721&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J. (1980). More sex-determination mutants of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>96</em></span>, 649&#8211;664.</span>
                     		<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7262542&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J. (1987). A genetic analysis of the sex-determining gene, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span>, in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genes Dev. <span class="emphasis"><em>1</em></span>, 731&#8211;745.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3428597&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J.A., Horvitz, H.R., and Brenner, S. (1979). Nondisjunction mutants of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>91</em></span>, 67&#8211;94.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Horvitz, H.R., Brenner, S., Hodgkin, J., and Herman, R.K. (1979). A uniform genetic nomenclature for the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Gen. Genet. <span class="emphasis"><em>175</em></span>, 129&#8211;133.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=292825&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00425528" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Howell, A.M., Gilmour, S.G., Mancebo, R.A., and Rose, A.M. (1987). Genetic analysis of a large autosomal region in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by the use of a free duplication. Genet. Res. <span class="emphasis"><em>49</em></span>, 207&#8211;213.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Howell, A.M., and Rose, A.M. (1990). Essential genes in the <span class="emphasis"><em>hDf6</em></span> region of chromosome I in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>126</em></span>, 583&#8211;592.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2249758&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hunter, C.P., and Wood, W.B. (1990). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> gene determines sexual phenotype cell-autonomously in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>63</em></span>, 1193&#8211;1204.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2261640&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(90)90415-B" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Johnsen, R.C., and Baillie, D.L. (1988). Formaldehyde mutagenesis of the <span class="emphasis"><em>eTl</em></span> balanced region in .. <span class="emphasis"><em>Caenorhabditis elegans</em></span>: dose-response curve and the analysis of mutational events. Mutat. Res. <span class="emphasis"><em>201</em></span>, 137&#8211;147.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Johnsen, R.C., and Baillie, D.L. (1991). Genetic analysis of a major segment [LGV(left)] of the genome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>129</em></span>, 735&#8211;752.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1752418&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lee, J., Jongeward, G.D., and Sternberg, P.W. (1994). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-101" target="_blank">unc-101</a></em></span>, a gene required for many aspects of <span class="emphasis"><em>Caenorhabditis elegans</em></span> development and behavior, encodes a clathrin-associated protein. Genes Dev. <span class="emphasis"><em>8</em></span>, 60&#8211;73.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8288128&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Howell, A.M., and Kose, A.M. (1988). The effects of translocations on recombination frequency in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>120</em></span>, 987&#8211;1001.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Marra, M.A., and Baillie, D.L. (1994). Recovery of duplications by drug resistance selection in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genome <span class="emphasis"><em>37</em></span>, 701&#8211;705.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7958827&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed">
                     		<span class="bibliomisc">McKim, K.S., Howell, A.M., and Rose, A.M. (1988). The effects of translocations on recombination frequency in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>120</em></span>, 987&#8211;1001.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3224815&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                     		<span class="bibliomisc">McKim, K.S., and Rose, A.M. (1990). Chromosome I duplications in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>124</em></span>, 115&#8211;132.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2307351&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                               <span class="bibliomisc">McKim, K.S., Peters, K., and Rose, A.M. (1993). Two types of sites required for meiotic chromosome pairing in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>134</em></span>, 749&#8211;768.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8349107&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                     		<span class="bibliomisc">McKim, K.S., Starr, T, and Rose, A.M. (1992). Genetic and molecular analysis of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-14" target="_blank">dpy-14</a></em></span> region in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Gen. Genet. <span class="emphasis"><em>233</em></span>, 241&#8211;25l.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1603066&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00587585" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mello, C., and Fire, A. (1995). DNA transformation. In: <span class="emphasis"><em>Methods in Cell Biology, Vol. 48</em></span>, H.F. Epstein and D.C. Shakes, eds. (San Diego, CA: Academic Press), pp. 452&#8211;480.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531738&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mello, C.C., Kramer, J.M., Stinchcomb, D., and Ambros, V. (1991). Efficient gene transfer in <span class="emphasis"><em>C. elegans</em></span>: extrachromosomal maintenance and integration of transforming sequences. EMBO J. <span class="emphasis"><em>10</em></span>, 3959&#8211;3970.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1935914&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Meneely, P.M., and Herman, R.K. (1979). Lethals, steriles and deficiencies in a region of the X chromosome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>92</em></span>, 99&#8211;115.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=574105&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Meneely, P.M., and Herman, R.K. (1981). Suppression and function of X-linked lethal and sterile mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>97</em></span>, 65&#8211;84.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7196363&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Meneely, P.M., and Nordstrom, K.D. (1988). X chromosome duplications affect a region of the chromosome they do not duplicate
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>119</em></span>, 365&#8211;375.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3396870&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Meneely, P.M., and Wood, W.B. (1984). An autosomal gene that affects X chromosome expression and sex determination in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>106</em></span>, 29&#8211;44.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6537930&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Moerman, D.G., and Baillie, D.L. (1979). Genetic organization in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: fine-structure analysis of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a></em></span> gene. Genetics <span class="emphasis"><em>91</em></span>, 95&#8211;104.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Moerman, D.G., and Baillie, D.L. (1981). Formaldehyde mutagenesis in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mutat. Res. <span class="emphasis"><em>80</em></span>, 273&#8211;279.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7207485&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Moerman, D.G., and Waterston, R.H. (1989). Mobile elements in <span class="emphasis"><em>Caenorhabditis elegans</em></span> and other nematodes. In Mobile DNA, D.E. Berg and M.M. Howe, eds. (Washington, D. C: American Society for Microbiology),
                        pp. 537&#8211;556.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Nelson, G.A., Schubert, W.W., Marshall, T.M., Benton, E.R., and Benton, E.V. (1989). Radiation effects in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, mutagenesis by high and low LET ionizing radiation. Mutat. Res. <span class="emphasis"><em>212</em></span>, 181&#8211;192.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2733713&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Plasterk, R.H.A. (1995). Reverse genetics: From gene sequence to mutant worm. In: <span class="emphasis"><em>Methods in Cell Biology, Vol. 48</em></span>, H.F. Epstein and D.C. Shakes, eds. (San Diego, CA: Academic Press), pp. 59&#8211;78.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531745&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Plenefisch, J.D., DeLong, L., and Meyer, B.J. (1989). Genes that implement the hermaphrodite mode of dosage compensation in
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>121</em></span>, 57&#8211;76.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2917714&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Riddle, D.L., and Brenner, S. (1978). Indirect suppression in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>89</em></span>, 299&#8211;314.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=669254&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rogalski, T.M., Bullerjahn, A.M.E., and Riddle, D.L. (1988). Lethal and amanitin-resistance mutations in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ama-1" target="_blank">ama-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ama-2" target="_blank">ama-2</a></em></span> genes. Genetics <span class="emphasis"><em>120</em></span>, 409&#8211;422.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3197954&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rogalski, T.M., and Riddle, D.L. (1988). A <span class="emphasis"><em>Caenorhabditis elegans</em></span> RNA polymerase II gene, <span class="emphasis"><em>ama-I IV</em></span>, and nearby essential genes. Genetics <span class="emphasis"><em>118</em></span>, 61&#8211;74.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8608933&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rose, A.M., and Baillie, D.L. (1980). Genetic organization of the region around <span class="emphasis"><em>unc-15(I)</em></span>, a gene affecting paramyosin in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>96</em></span>, 639&#8211;648.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7262541&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rose, A.M., Baillie, D.L., and Curran, J. (1984). Meiotic pairing behavior of two free duplications of linkage group I in
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Gen. Genet. <span class="emphasis"><em>195</em></span>, 52&#8211;56.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6593563&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00332723" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rosenbluth, R.E., and Baillie, D.L. (1981). The genetic analysis of a reciprocal translocation, <span class="emphasis"><em>eT1(III;V)</em></span>, in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>99</em></span>, 415&#8211;428.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6953041&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rosenbluth, R.E., Cuddeford, C., and Baillie, D.L. (1983). Mutagenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. 1. A rapid eukaryotic mutagen test system using the reciprocal translocation <span class="emphasis"><em>eTI(III;V)</em></span>. Mutat. Res. <span class="emphasis"><em>110</em></span>, 39&#8211;48.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rosenbluth, R.E., Cuddeford, C, and Baillie, D.L. (1985). Mutagenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. II. A spectrum of mutational events induced with 1500 Rofgamma-radiation. Genetics <span class="emphasis"><em>109</em></span>, 493&#8211;511.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3979812&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rosenbluth, R.E., Johnsen, R.C., and Baillie, D.L. (1990). Pairing for recombination in LG V of <span class="emphasis"><em>Caenorhabditis elegans</em></span>: a model based on recombination in deficiency heterozygotes. Genetics <span class="emphasis"><em>124</em></span>, 615&#8211;625.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2311918&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rosenbluth, R.E., Rogalski, T.M., Johnsen, R.C., Addison, L.M., and Baillie, D.L. (1988). Genomic organization in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: deficiency mapping on linkage group V (left). Genet. Res. <span class="emphasis"><em>52</em></span>, 105&#8211;118.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sigurdson, D.C., Spanier, G.J., and Herman, R.K. (1984). <span class="emphasis"><em>Caenorhabditis elegans</em></span> deficiency mapping. Genetics <span class="emphasis"><em>108</em></span>, 331&#8211;345.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6500256&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Stewart, H.I., Rosenbluth, R.E., and Baillie, D.L. (1991). Most ultraviolet irradiation induced mutations in the nematode
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span> are chromosomal rearrangements. Mutat. Res. <span class="emphasis"><em>249</em></span>, 37&#8211;54.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2067542&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sulston, J., and Hodgkin, J. (1988). Methods. In The Nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>, W. B. Wood, ed. (New York: Cold Spring Harbor Laboratory), pp. 592.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Yandell, M.D., Edgar, L.S., and Wood, W.B. (1994). Trimethylpsoralen induces small deletion mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>91</em></span>, 1381&#8211;1385.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7906415&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Yuan, J.Y., and Horvitz, H.R. (1990). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> genes <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-3" target="_blank">ced-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ced-4" target="_blank">ced-4</a></em></span> act cell autonomously to cause programmed cell death. Dev. Biol. <span class="emphasis"><em>138</em></span>, 33&#8211;41.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2307287&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(90)90174-H" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Zetka, M.-C., and Rose, A.M. (1992). The meiotic behavior of an inversion in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>131</em></span>, 321&#8211;332.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1644275&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by David Fay. WormMethods editor, Victor Ambros. Last revised February 25, 2005. Published April 6, 2006. This chapter
                  should be cited as: Edgley, M. L. et al. Genetic balancers (April 6, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.89.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e21" href="#d0e21">&sect;</a></sup>Adapted from: Edgley, M., Baillie, D.L., Riddle, D.L., and Rose, A.M. (1995). Genetic Balancers. In Methods Cell Biol. aka
                  <span class="emphasis"><em>C. elegans</em></span>: Modern Biological Analysis of an Organism, H.F. Epstein and D.C. Shakes, eds. (San Diego: Academic Press), <span class="emphasis"><em>48</em></span>, 148&#8211;184.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e59" href="#d0e59">&#8224;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:edgley@interchange.ubc.ca" target="_top">edgley@interchange.ubc.ca</a>, <a href="mailto:baillie@sfu.ca" target="_top">baillie@sfu.ca</a>, <a href="mailto:driddle@genomebc.ca" target="_top">driddle@genomebc.ca</a> or <a href="mailto:arose@gene.nce.ubc.ca" target="_top">arose@gene.nce.ubc.ca</a></p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e126" href="#d0e126">&#8225;</a></sup>Gene names are italicized and consist of three lowercase letters, followed by a hyphen and isolation number for individual
                  genes. The name is usually a mnemonic referring to the nature of mutations in the gene. For example, the name <span class="emphasis"><em>dpy</em></span> refers to a class of mutation characterized by a short, fat body, and stands for <span class="emphasis"><em>dumpy</em></span>. The name <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a></em></span> refers to a particular <span class="emphasis"><em>dpy</em></span> gene. Some other names used throughout the chapter are <span class="emphasis"><em>let</em></span> (lethal), <span class="emphasis"><em>unc</em></span> (uncoordinated), <span class="emphasis"><em>lon</em></span> (long), <span class="emphasis"><em>rol</em></span> (roller), and <span class="emphasis"><em>bli</em></span> (blistered). Phenotypes are represented by nonitalicized gene names with the first letter capitalized. For example, the general
                  phenotype for all <span class="emphasis"><em>dpy</em></span> mutations is written Dpy, and the phenotype for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-18" target="_blank">dpy-18</a></em></span> mutations is written Dpy-18. Rearrangement names are italicized and consist of a lower case laboratory allele prefix followed
                  by a capitalized abbreviation indicating the type of rearrangement, an isolation number, and optional chromosomal origin and
                  location information in parentheses. For example, the name <span class="emphasis"><em>sDp2(I;f)</em></span> represents the second chromosomal duplication isolated in the Baillie laboratory (allele prefix "s"). The "I;f" indicates
                  that it is a duplication of part of LG I and that it is "free" (not attached to another chromosome). For translocations, the
                  parenthetical information indicates the chromosomes involved. For example, <span class="emphasis"><em>eT1(III;V)</em></span> indicates a translocation between LG III and LG V. The rearrangement abbreviations used here are <span class="emphasis"><em>C</em></span> (dominant crossover supressor), <span class="emphasis"><em>Df</em></span> (deficiency), <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=Dp;class=Cell" target="_blank">Dp</a></em></span> (duplication), <span class="emphasis"><em>In</em></span> (inversion), and <span class="emphasis"><em>T</em></span> (translocation).
               </p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

