<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Endogenous RNAi pathways in C. elegans</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WB_Billi_etal_text_jm" title="Endogenous RNAi pathways in C. elegans">
      <link rel="next" href="#sec1" title="1.&nbsp;Introduction">
      <script language="javascript" type="text/javascript" src="/js/history_list.js"></script>
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="endoRNAipathwys.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WB_Billi_etal_text_jm"></a>Endogenous RNAi pathways in <span class="emphasis"><em>C. elegans</em></span><sup><a name="d0e8" href="#ftn.d0e8">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Allison C. Billi<sup>1</sup>, Sylvia E. J. Fischer<sup>2</sup>, and John K. Kim<sup>1</sup><em><span class="remark"><sup><a name="d0e40" href="#ftn.d0e40">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup>Life Sciences Institute, Department of Human Genetics, University of Michigan, Ann Arbor <a href="http://www.wormbase.org/db/get?name=MI;class=Cell" target="_blank">MI</a>, USA; </span></span><span class="affiliation"><span class="orgname"><sup>2</sup>Department of Molecular Biology, Massachusetts General Hospital, Department of Genetics, Harvard Medical School, Boston MA,
                           USA </span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">
         
	   <div id="section_locator"><a href="/toc_molecularbiology.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Molecular biology
		</a></div><hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. Features and targets of 26G RNAs</a></span></dt>
               <dt><span class="sect1"><a href="#sec3">3. Biogenesis of 26G RNAs: The ERI complex</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec3-1">3.1. RRF-3</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-2">3.2. DRH-3</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-3">3.3. ERI-5</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-4">3.4. DCR-1</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-5">3.5. DCR-1 helicase domain</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-6">3.6. ERI-1b</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-7">3.7. ERI-3</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-8">3.8. RDE-4</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec4">4. Other factors implicated in 26G RNA accumulation</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec4-1">4.1. ERI-9</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4-2">4.2. ERI-6/7</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4-3">4.3. MUT-16</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4-4">4.4. Other Mutator (MUT) proteins</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec5">5. 26G RNA Argonaute loading</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec5-1">5.1. ALG-3/4</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5-2">5.2. ERGO-1</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5-3">5.3. HENN-1</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec6">6. 26G RNA effector function</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec6-1">6.1. Sperm-origin ts sterility due to loss of ALG-3/4 class 26G RNAs</a></span></dt>
                     <dt><span class="sect2"><a href="#sec6-2">6.2. Enhanced RNAi sensitivity due to loss of ERGO-1 class 26G RNAs</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec7">7. 22G RNAs comprise two major small RNA pathways</a></span></dt>
               <dt><span class="sect1"><a href="#sec8">8. 22G RNA biogenesis: The 22G RNA RdRP module</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec8-1">8.1. RRF-1</a></span></dt>
                     <dt><span class="sect2"><a href="#sec8-2">8.2. DRH-3</a></span></dt>
                     <dt><span class="sect2"><a href="#sec8-3">8.3. EKL-1</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec9">9. Triggering of WAGO 22G RNAs</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec9-1">9.1. Triggering by 26G RNAs</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9-2">9.2. Triggering by primary exo-siRNAs</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9-3">9.3. Triggering by RDE-1-scavenged small RNAs</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9-4">9.4. Triggering by 21U RNAs</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec10">10. Germline amplification of WAGO 22G RNAs in perinuclear Mutator foci</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec10-1">10.1. Dynamics of Mutator foci</a></span></dt>
                     <dt><span class="sect2"><a href="#sec10-2">10.2. Assembly of Mutator foci</a></span></dt>
                     <dt><span class="sect2"><a href="#sec10-3">10.3. Function of Mutator foci</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec11">11. Additional amplification of WAGO 22G RNAs by RDE-10/RDE-11</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec11-1">11.1. RDE-10/RDE-11 amplifies 22G RNAs triggered by exo-RNAi and ERGO-1</a></span></dt>
                     <dt><span class="sect2"><a href="#sec11-2">11.2. RDE-10/RDE-11-mediated target degradation</a></span></dt>
                     <dt><span class="sect2"><a href="#sec11-3">11.3. RSD-2, RSD-6, and/or HAF-6 may potentiate RDE-10/RDE-11 function</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec12">12. WAGO 22G RNA Argonaute loading</a></span></dt>
               <dt><span class="sect1"><a href="#sec13">13. WAGO 22G RNA-mediated post-transcriptional gene silencing</a></span></dt>
               <dt><span class="sect1"><a href="#sec14">14. WAGO 22G RNA-mediated transcriptional gene silencing: The nuclear RNAi pathway</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec14-1">14.1. Early evidence for transcriptional gene silencing</a></span></dt>
                     <dt><span class="sect2"><a href="#sec14-2">14.2. Early evidence for heritable and multigenerational gene silencing</a></span></dt>
                     <dt><span class="sect2"><a href="#sec14-3">14.3. NRDE-3 and HRDE-1 WAGO Argonautes mediate nuclear RNAi</a></span></dt>
                     <dt><span class="sect2"><a href="#sec14-4">14.4. NRDE-3 collaborates with other WAGO Argonautes to silence somatic targets</a></span></dt>
                     <dt><span class="sect2"><a href="#sec14-5">14.5. HRDE-1 transgenerationally silences diverse targets to promote germline immortality</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec15">15. NRDE-3 target recognition</a></span></dt>
               <dt><span class="sect1"><a href="#sec16">16. The downstream nuclear RNAi machinery</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec16-1">16.1. NRDE-2</a></span></dt>
                     <dt><span class="sect2"><a href="#sec16-2">16.2. NRDE-1</a></span></dt>
                     <dt><span class="sect2"><a href="#sec16-3">16.3. NRDE-4</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec17">17. Nuclear RNAi inhibits transcriptional elongation and drives H3K9 trimethylation</a></span></dt>
               <dt><span class="sect1"><a href="#sec18">18. NRDE-3 directs single-generation epigenetic inheritance of silencing</a></span></dt>
               <dt><span class="sect1"><a href="#sec19">19. HRDE-1 directs multigenerational epigenetic inheritance of silencing</a></span></dt>
               <dt><span class="sect1"><a href="#sec20">20. Features and targets of CSR-1 22G RNAs</a></span></dt>
               <dt><span class="sect1"><a href="#sec21">21. The CSR-1 22G RNA pathway molecular machinery</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec21-1">21.1. CSR-1</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec22">22. Phenotypes associated with CSR-1 22G RNA pathway compromise</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec22-1">22.1. Germline phenotypes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec22-2">22.2. Embryonic phenotypes</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec23">23. Localization and assembly of the CSR-1 22G RNA complex</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec23-1">23.1. Germline localization</a></span></dt>
                     <dt><span class="sect2"><a href="#sec23-2">23.2. Embryo localization</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec24">24. CSR-1 22G RNA effector function</a></span></dt>
               <dt><span class="sect1"><a href="#sec25">25. CDE-1 regulates the CSR-1 22G RNA pathway</a></span></dt>
               <dt><span class="sect1"><a href="#sec26">26. 21U RNA features and loci</a></span></dt>
               <dt><span class="sect1"><a href="#sec27">27. 21U RNA targets</a></span></dt>
               <dt><span class="sect1"><a href="#sec28">28. 21U RNA biogenesis</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec28-1">28.1. The 21U RNA large motif and Forkhead family transcription factors</a></span></dt>
                     <dt><span class="sect2"><a href="#sec28-2">28.2. The 21U RNA small motif and precursor</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec29">29. 21U RNA precursor processing and Argonaute loading</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec29-1">29.1. PRG-1</a></span></dt>
                     <dt><span class="sect2"><a href="#sec29-2">29.2. HENN-1</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec30">30. TSS-associated 21U RNAs</a></span></dt>
               <dt><span class="sect1"><a href="#sec31">31. Future directions</a></span></dt>
               <dt><span class="sect1"><a href="#sec32">32. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>In addition to several hundred microRNAs, <span class="italic">C. elegans</span> produces thousands of other small RNAs targeting coding genes, pseudogenes, transposons, and other noncoding RNAs. Here we
                     review what is currently known about these endogenous small interfering RNAs (siRNAs) and piwi-interacting RNAs (piRNAs),
                     providing an overview of their biogenesis, their associated protein factors, and their effects on mRNA dynamics and chromatin
                     structure. Additionally, we describe how the molecular actions of these classes of endogenous small RNAs connect to their
                     physiological roles in the organism.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>The phenomenon of RNA interference (RNAi), wherein introduction of dsRNA results in silencing of homologous genes, was first
               observed and reported in <span class="emphasis"><em>C. elegans</em></span>. In addition, when the first small RNA, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002993;class=Gene" target="_blank">lin-4</a></em></span> microRNA, was identified, it was thought to be a curiosity of <span class="emphasis"><em>C. elegans</em></span> biology until the discovery of the conserved <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002285;class=Gene" target="_blank">let-7</a></em></span> family of microRNAs. Since then, thousands of unique endogenous small RNAs have been found in <span class="emphasis"><em>C. elegans</em></span> falling into three distinct classes: microRNAs, endogenous small interfering RNAs (endo-siRNAs), and Piwi-interacting RNAs
               (piRNAs). All three types of small RNAs bind to Argonaute effector proteins, recognize target transcripts exhibiting partial
               or perfect complementarity, and direct target regulation that is primarily inhibitory in nature. Yet microRNAs, endo-siRNAs,
               and piRNAs vary greatly in biogenesis mechanisms, specific protein cofactors, and even effector function. The details of <span class="emphasis"><em>C. elegans</em></span> microRNA biology have been previously described (<a href="#bib36">Fischer, 2010</a>) and are addressed in this review only insofar as they intersect with endo-siRNA and piRNA biology. Similarly, a thorough
               examination of exogenously-derived siRNAs and exogenous RNAi (exo-RNAi), and its function as an anti-viral silencing response,
               is beyond the scope of this review; however, the gene silencing mechanisms initiated by exo-RNAi are discussed, as <span class="emphasis"><em>C. elegans</em></span> exo-RNAi engages a downstream endo-RNAi amplification pathway also shared by primary endo-siRNAs and piRNAs.
            </p>
            <p>The gene silencing capacity of antisense RNA was first described in <span class="emphasis"><em>C. elegans</em></span> over two decades ago (<a href="#bib34">Fire et al., 1991</a>). The effective interfering agent was subsequently determined to be double-stranded RNA (dsRNA), and its incredible potency
               suggested the existence of a catalytic or amplification mechanism engaged by exogenous dsRNA (<a href="#bib35">Fire et al., 1998</a>). This dsRNA is processed into primary exo-siRNAs by the conserved RNase III enzyme Dicer (<a href="#bib146">Zamore et al., 2000</a>; <a href="#bib11">Bernstein et al., 2001</a>), increasing the ratio of trigger to target, but still to a degree insufficient to explain the potency of exo-RNAi. Further
               studies of the small RNA effector populations during exo-RNAi in <span class="emphasis"><em>C. elegans</em></span> revealed that primary exo-siRNAs are not the ultimate effectors of interference; rather, they trigger vigorous production
               of secondary siRNAs by RNA-dependent RNA polymerases (RdRPs), amplifying the signal and focusing the interference response
               on expressed sequences (<a href="#bib115">Sijen et al., 2001</a>). Before the discovery of endogenous silencing pathways in <span class="emphasis"><em>C. elegans</em></span>, isolation of mutations causing deficiencies in both exo-RNAi and endogenous silencing phenomena such as transposon silencing
               suggested a shared mechanism (<a href="#bib125">Tabara et al., 1999</a>; <a href="#bib66">Ketting et al., 1999</a>; <a href="#bib64">Ketting and Plasterk, 2000</a>).  Mutations in core molecular machinery of RNAi also were found to result in phenotypes indicating essential endogenous
               roles. Loss of DCR-1, the sole <span class="emphasis"><em>C. elegans</em></span> ortholog of Dicer, results in profound cell fate specification defects and germline abnormalities leading to lethality and
               sterility (<a href="#bib44">Grishok et al., 2001</a>; <a href="#bib65">Ketting et al., 2001</a>; <a href="#bib68">Knight and Bass, 2001</a>); the RdRP EGO-1 is required for germline development (<a href="#bib119">Smardon et al., 2000</a>); and mutants defective in transposon silencing and RNAi show compromised fertility (<a href="#bib66">Ketting et al., 1999</a>; <a href="#bib128">Tijsterman et al., 2002a</a>; <a href="#bib134">Vastenhouw et al., 2003</a>; <a href="#bib20">Chen et al., 2005</a>).
            </p>
            <p>The discovery of the first microRNA, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002993;class=Gene" target="_blank">lin-4</a></em></span>, in <span class="emphasis"><em>C. elegans</em></span> suggested that endogenous small RNAs also initiate transcript regulation through antisense mechanisms (<a href="#bib73">Lee et al., 1993</a>; <a href="#bib143">Wightman et al., 1993</a>). After the identification of a second microRNA, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002285;class=Gene" target="_blank">let-7</a></em></span>, in 2000 (<a href="#bib103">Reinhart et al., 2000</a>), cloning approaches were successfully applied to expand dramatically the repertoire of microRNAs in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib71">Lau et al., 2001</a>; <a href="#bib74">Lee and Ambrose 2001</a>). It was not until a decade after the discovery of the first microRNA that other endogenous small RNAs were identified in
               <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib2">Ambros et al., 2003</a>; <a href="#bib77">Lim et al., 2003</a>), and only in 2006 did small RNA deep sequencing first reveal the incredible diversity of <span class="emphasis"><em>C. elegans</em></span> small RNAs (<a href="#bib108">Ruby et al., 2006</a>). Among the species identified were a large pool of 5&#8217; guanosine antisense small RNAs identified as endo-siRNAs that appeared
               to represent distinct 26- and 22-nucleotide (nt) subpopulations, later determined to correspond to primary and secondary endo-siRNAs,
               respectively (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib91">Pak and Fire, 2007</a>; <a href="#bib116">Sijen et al., 2007</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>). Subsequent dissection of these 26G and 22G RNA populations identified unique subgroups with largely overlapping biogenesis
               requirements but that engage different effector pathways distinguished by the particular Argonaute protein(s) interacting
               with the small RNAs. <span class="emphasis"><em>C. elegans</em></span> encodes 27 Argonaute proteins. Roughly half of these proteins belong to a worm-specific Argonaute (WAGO) clade and bind 22G
               RNAs of particular subclasses, the WAGO 22G RNAs and CSR-1 22G RNAs; the former represent secondary siRNAs that effect target
               silencing, whereas the latter are a class of siRNAs complementary to germline-expressed transcripts that do not silence target
               genes but rather promote proper chromosome organization during mitosis (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). Also identified in the initial deep sequencing dataset were the 21U RNAs, 5&#8217; uridine 21-nt small RNAs later determined
               to represent the piRNAs of <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>). This review summarizes the literature contributing to our current understanding of the <span class="emphasis"><em>C. elegans</em></span> 26G RNAs, WAGO 22G RNAs, CSR-1 22G RNAs, and 21U RNAs, discussing mechanisms of triggering, biogenesis, and effector function,
               where known. Whereas microRNAs are required for diverse developmental and physiological processes in the soma, endo-siRNAs
               and 21U RNAs serve as the guardians of the immortal germline, constituting a complex, interconnected, and tremendously robust
               system for surveillance of the <span class="emphasis"><em>C. elegans</em></span> genome.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Features and targets of 26G RNAs
                     </h2>
                  </div>
               </div>
            </div>
            <p>Early deep sequencing of <span class="emphasis"><em>C. elegans</em></span> small RNAs revealed a distinct population of 26 nt species that are largely anti-sense to annotated genes and therefore classified
               as endo-siRNAs (<a href="#bib108">Ruby et al., 2006</a>). Like endo-siRNAs sequenced earlier in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib2">Ambros et al., 2003</a>), these 26 nt species show a 5&#8217; guanosine bias and are thus termed 26G RNAs (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib5">Asikainen et al., 2008</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib123">Stoeckius et al., 2009</a>; <a href="#bib42">Gent et al., 2010</a>) (<a href="#table1" title="Table 1. Features of different classes of endogenous small RNAs.">Table 1</a>). 26G RNAs are 5&#8217; monophosphorylated (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib123">Stoeckius et al., 2009</a>; <a href="#bib42">Gent et al., 2010</a>). 26G RNAs are quite enriched in male and female germlines (<a href="#bib52">Han et al., 2009</a>) where they comprise two distinct subpopulations that are temporally isolated and bound by unique, germline-specific effector
               complexes. 26G RNAs in the spermatogenic gonad are bound by redundant Argonautes ALG-3 and ALG-4, whereas 26G RNAs in oogenic
               gonad are bound by the ERGO-1 Argonaute (<a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>). ERGO-1 class 26G RNAs are also highly abundant in embryos (<a href="#bib52">Han et al., 2009</a>; <a href="#bib123">Stoeckius et al., 2009</a>) and perdure through early larval development (<a href="#bib42">Gent et al., 2010</a>).
            </p>
            <div class="table"><a name="table1"></a><p class="title">Table 1. Features of different classes of endogenous small RNAs.</p>
               <table summary="Table 1. Features of different classes of endogenous small RNAs." border="1">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr valign="bottom">
                        <th align="left" valign="bottom"><span class="bold"><strong>Small RNA</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>5&#8217; terminus</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>3&#8217; terminus</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>Polymerase</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>Biogenesis factors</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>Cofactors regulating accumulation</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>Argonaute(s) and RISC components</strong></span></th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr valign="top">
                        <td align="left" valign="top">ERGO-1 class 26G RNA</td>
                        <td align="left" valign="top">Monophosphorylated</td>
                        <td align="left" valign="top">2&#8217;-O-methylated</td>
                        <td align="left" valign="top">RdRP RRF-3</td>
                        <td align="left" valign="top">DRH-3, ERI-5, DCR-1, ERI-1b, ERI-3, RDE-4</td>
                        <td align="left" valign="top">HENN-1, ERI-9, ERI-6/7, MUT-16</td>
                        <td align="left" valign="top">ERGO-1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">ALG-3/4 class 26G RNA</td>
                        <td align="left" valign="top">Monophosphorylated</td>
                        <td align="left" valign="top">Hydroxylated</td>
                        <td align="left" valign="top">RdRP RRF-3</td>
                        <td align="left" valign="top">DRH-3, ERI-5, DCR-1, ERI-1b, ERI-3, RDE-4</td>
                        <td valign="top">&nbsp;</td>
                        <td align="left" valign="top">ALG-3/4</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">WAGO 22G RNA</td>
                        <td align="left" valign="top">Triphosphorylated</td>
                        <td align="left" valign="top">Hydroxylated</td>
                        <td align="left" valign="top">RdRPs RRF-1/EGO-1</td>
                        <td align="left" valign="top">RRF-1/EGO-1, DRH-3, EKL-1</td>
                        <td align="left" valign="top">MUT-16, MUT-7, MUT-8, MUT-2, MUT-15, MUT-14, RDE-10, RDE-11</td>
                        <td align="left" valign="top">WAGO Argonautes</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">CSR-1 22G RNA</td>
                        <td align="left" valign="top">Triphosphorylated</td>
                        <td align="left" valign="top">Hydroxylated</td>
                        <td align="left" valign="top">RdRP</td>
                        <td align="left" valign="top">EGO-1</td>
                        <td align="left" valign="top">EGO-1, DRH-3, EKL-1</td>
                        <td align="left" valign="top">CDE-1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">21U RNA (Type I)</td>
                        <td align="left" valign="top">Monophosphorylated</td>
                        <td align="left" valign="top">2&#8217;-O-methylated</td>
                        <td align="left" valign="top">RNA Pol II</td>
                        <td align="left" valign="top">FKH-3/4/5, UNC-130, (large and small motifs)</td>
                        <td align="left" valign="top">HENN-1</td>
                        <td align="left" valign="top">PRG-1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">TSS-associated 21U RNA (Type II)</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                        <td align="left" valign="top">RNA Pol II</td>
                        <td align="left" valign="top">(Small motif)</td>
                        <td valign="top">&nbsp;</td>
                        <td align="left" valign="top">PRG-1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">microRNA</td>
                        <td align="left" valign="top">Monophosphorylated</td>
                        <td align="left" valign="top">Hydroxylated</td>
                        <td align="left" valign="top">RNA Pol II</td>
                        <td align="left" valign="top">DCR-1, DRSH-1, PASH-1</td>
                        <td align="left" valign="top">XRN-2, DCS-1</td>
                        <td align="left" valign="top">ALG-1/2, AIN-1/2, NHL-2, CGH-1, TSN-1, VIG-1</td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <p>Little is known about how transcripts are selected for targeting by 26G RNAs. 26G RNAs map primarily to protein-coding genes
               with a strong antisense bias, although some target unannotated loci (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib123">Stoeckius et al., 2009</a>). They are transcribed from spliced mRNA templates by the RNA-dependent RNA polymerase RRF-3 (<a href="#bib43">Gent et al., 2009</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>), as indicated by the sequencing of rare species spanning exon-exon junctions (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib42">Gent et al., 2010</a>) and by the loss of complementary 26G RNAs upon nonsense-mediated decay of template mRNA (<a href="#bib52">Han et al., 2009</a>).
            </p>
            <p>Targets of ALG-3/4 class 26G RNAs are highly enriched for transcripts classified as spermatogenesis-enriched (<a href="#bib104">Reinke et al., 2004</a>; <a href="#bib43">Gent et al., 2009</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>), explaining earlier observations connecting endo-siRNAs primarily with regulation of sperm function genes (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib6">Asikainen et al., 2007</a>). ALG-3/4 class 26G RNAs map across the lengths of their target sequences, but preferentially target transcript 5&#8217; and 3&#8217;
               termini; transcripts exhibiting higher 5&#8217; UTR targeting appear to be more efficiently silenced (<a href="#bib24">Conine et al., 2010</a>).
            </p>
            <p>Although biogenesis of ERGO-1 class 26G RNAs initiates in oogenesis, their targets are depleted of germline-intrinsic transcripts
               (<a href="#bib104">Reinke et al., 2004</a>; <a href="#bib52">Han et al., 2009</a>); accordingly, ERGO-1 class 26G RNAs primarily regulate zygotic targets throughout development (<a href="#bib52">Han et al., 2009</a>; <a href="#bib42">Gent et al., 2010</a>). The majority of abundant ERGO-1 class 26G RNAs map to within 5 Mb of chromosome ends at more gene-poor regions (<a href="#bib133">Vasale et al., 2010</a>; <a href="#bib38">Fischer et al., 2011</a>). Unlike ALG-3/4 class 26G RNAs, ERGO-1 class 26G RNAs are excluded from the first ~100 nt of target transcripts (<a href="#bib133">Vasale et al., 2010</a>). Roughly half of ERGO-1 class 26G RNAs map to coding loci, lincRNAs, or pseudogenes and half to loci that likely correspond
               to unannotated transcription units (<a href="#bib133">Vasale et al., 2010</a>). Many of their targets represent recently duplicated, non-conserved genes, suggesting ERGO-1 class 26G RNAs may buffer expression
               of newly acquired genes (<a href="#bib133">Vasale et al., 2010</a>; <a href="#bib38">Fischer et al., 2011</a>). A large-scale proteomics dataset (<a href="#bib109">Schrimpf et al., 2009</a>) shows peptides corresponding to less than 20% of ERGO-1 class 26G RNA targets, but 54% of all annotated coding genes, indicating
               that very few of these targets represent functional, coding loci (<a href="#bib38">Fischer et al., 2011</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;Biogenesis of 26G RNAs: The ERI complex
                     </h2>
                  </div>
               </div>
            </div>
            <p>The ERI complex mediates 26G RNA biogenesis and is named for the <span class="underline">e</span>nhanced (exogenous) <span class="underline">R</span>NA<span class="underline">i</span> (<a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a>) phenotype that is associated with compromise of ERGO-1 class 26G RNA function (<a href="#table2" title="Table 2. Protein factors acting in endogenous small RNA pathways.">Table 2</a> and <a href="#figure1">Figure 1</a>). Loss of ERI complex members also results in temperature-sensitive (ts) sterility at 25&deg;C due to defective spermatogenesis
               as well as a <span class="underline">h</span>igh <span class="underline">i</span>ncidence of <span class="underline">m</span>ale progeny (<a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a>) phenotype, indicative of increased X chromosome nondisjunction, due to compromised ALG-3/4 class 26G RNA function. The <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> and ts sterile phenotypes are explained in greater detail below. At the heart of the ERI complex is a core RdRP module consisting
               of RRF-3, DRH-3, and ERI-5. Associated with this module are DCR-1, accessory factors ERI-1b, ERI-3, RDE-4, and possibly the
               RNA phosphatase PIR-1. Analyses of DCR-1, ERI-1b, and ERI-5 complexes immuno-purified from gravid adult and embryo have revealed
               interactions with each other member of the ERI complex (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib127">Thivierge et al., 2012</a>), and the ~850 kD estimated mass of the ERI complex is similar to the summed mass of these proteins, ~810 kD (<a href="#bib127">Thivierge et al., 2012</a>). The ERI complex shares only DCR-1 and RDE-4 with the RDE (exogenous <span class="underline">R</span>NA<span class="underline">i</span> <span class="underline">de</span>fective) complex, which mediates processing of dsRNA trigger to initiate the exo-RNAi pathway (<a href="#bib126">Tabara et al., 2002</a>). DCR-1 and ERI-1 also both interact with ERI-9, another factor implicated in ERGO-1 class 26G RNA biogenesis (<a href="#bib127">Thivierge et al., 2012</a>), suggesting that the ERI complex includes other transient interactors.
            </p>
            <div class="table"><a name="table2"></a><p class="title">Table 2. Protein factors acting in endogenous small RNA pathways.</p>
               <table summary="Table 2. Protein factors acting in endogenous small RNA pathways." border="1" width="100%">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr valign="bottom">
                        <th colspan="2" align="left" valign="bottom" width="40%"><span class="bold"><strong>26G RNA pathway factors</strong></span></th>
                        <th align="left" valign="bottom" width="30%"><span class="bold"><strong>Protein/domain(s)</strong></span></th>
                        <th align="left" valign="bottom" width="30%"><span class="bold"><strong>Orthologous Hs protein(s)</strong></span></th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr valign="top">
                        <td rowspan="7" align="left" valign="top" width="25%">ERI complex</td>
                        <td align="left" valign="top">RRF-3</td>
                        <td align="left" valign="top">RdRP</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">DRH-3</td>
                        <td align="left" valign="top">Dicer-related helicase</td>
                        <td align="left" valign="top">DDX58/IFIH1/DHX58</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">ERI-5</td>
                        <td align="left" valign="top">Tudor domain protein</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">DCR-1/ERI-4</td>
                        <td align="left" valign="top">Helicase, PAZ, RNase III</td>
                        <td align="left" valign="top">DICER1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">ERI-1b</td>
                        <td align="left" valign="top">DEDDh-like 3&#8217; to 5&#8217; exonuclease</td>
                        <td align="left" valign="top">ERI1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">ERI-3</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">RDE-4</td>
                        <td align="left" valign="top">dsRNA-binding domains</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td rowspan="4" align="left" valign="top">Accumulation regulators</td>
                        <td align="left" valign="top">ERI-9</td>
                        <td align="left" valign="top">Putative RNA transferase</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">ERI-6/7</td>
                        <td align="left" valign="top">Superfamily I helicase</td>
                        <td align="left" valign="top">IGHMBP2, MOV10 (ERI-7)</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">MUT-16</td>
                        <td align="left" valign="top">P-rich domain, Q/N-rich domains</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">HENN-1</td>
                        <td align="left" valign="top">Methyltransferase</td>
                        <td align="left" valign="top">HENMT</td>
                     </tr>
                     <tr valign="top">
                        <td rowspan="2" align="left" valign="top">Argonautes</td>
                        <td align="left" valign="top">ALG-3/4</td>
                        <td align="left" valign="top">Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">ERGO-1</td>
                        <td align="left" valign="top">Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                  </tbody>
               </table>
               <table summary="Table 2. Protein factors acting in endogenous small RNA pathways." border="1" width="100%">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr valign="bottom">
                        <th colspan="2" align="left" valign="bottom" width="40%"><span class="bold"><strong>22G RNA pathway factors</strong></span></th>
                        <th align="left" valign="bottom" width="30%"><span class="bold"><strong>Protein/domain(s)</strong></span></th>
                        <th align="left" valign="bottom" width="30%"><span class="bold"><strong>Orthologous Hs protein(s)</strong></span></th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr valign="top">
                        <td rowspan="4" align="left" valign="top" width="25%">RdRP module</td>
                        <td align="left" valign="top">RRF-1</td>
                        <td align="left" valign="top">RdRP</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">EGO-1</td>
                        <td align="left" valign="top">RdRp</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">DRH-3</td>
                        <td align="left" valign="top">Dicer-related helicase</td>
                        <td align="left" valign="top">DDX58/IFIH1/DHX58</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">EKL-1</td>
                        <td align="left" valign="top">Tudor domain protein</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td rowspan="6" align="left" valign="top">WAGO 22G RNA accumulation regulators: Mutator foci</td>
                        <td align="left" valign="top">MUT-16</td>
                        <td align="left" valign="top">P-rich domain, Q/N-rich domains</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">MUT-7</td>
                        <td align="left" valign="top">3&#8217; to 5&#8217; exonuclease</td>
                        <td align="left" valign="top">EXD3</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">MUT-8/RDE-2</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">MUT-2/RDE-3</td>
                        <td align="left" valign="top">Beta-nucleotidyltransferase</td>
                        <td align="left" valign="top">TUT1/MTPAP</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">MUT-15</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">MUT-14</td>
                        <td align="left" valign="top">DEAD box RNA helicase</td>
                        <td align="left" valign="top">DDX4</td>
                     </tr>
                     <tr valign="top">
                        <td rowspan="5" align="left" valign="top">WAGO 22G RNA accumulation regulators: RDE-10/RDE-11 complex</td>
                        <td align="left" valign="top">RDE-10</td>
                        <td align="left" valign="top">Maelstrom domain</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">RDE-11</td>
                        <td align="left" valign="top">RING-type zinc finger domain</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">RSD-2</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">RSD-6</td>
                        <td align="left" valign="top">Tudor domain protein</td>
                        <td align="left" valign="top">TDRD5</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">HAF-6</td>
                        <td align="left" valign="top">ABC transporter</td>
                        <td align="left" valign="top">ABCB8</td>
                     </tr>
                     <tr valign="top">
                        <td rowspan="8" align="left" valign="top">Cytoplasmic WAGO Argonautes</td>
                        <td align="left" valign="top">WAGO-1</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">WAGO-2</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">PPW-2/WAGO-3</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">WAGO-4</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">WAGO-5</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">WAGO-6</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">PPW-1/WAGO-7</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">WAGO-8</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td rowspan="4" align="left" valign="top">Nuclear WAGO Argonautes</td>
                        <td align="left" valign="top">HRDE-1/WAGO-9</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">WAGO-10</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">WAGO-11</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">NRDE-3/WAGO-12</td>
                        <td align="left" valign="top">WAGO Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td rowspan="3" align="left" valign="top">Core nuclear RNAi machinery</td>
                        <td align="left" valign="top">NRDE-2</td>
                        <td align="left" valign="top">Ser/Arg-rich domain, HAT-like domain, DUF1740</td>
                        <td align="left" valign="top">NRDE2</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">NRDE-1</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">NRDE-4</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td rowspan="5" align="left" valign="top">Chromatin-level nuclear RNAi effectors</td>
                        <td align="left" valign="top">HPL-2</td>
                        <td align="left" valign="top">Heterochromatin protein</td>
                        <td align="left" valign="top">CBX1/CBX3</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">MES-3</td>
                        <td align="left" valign="top">Polycomb complex protein</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">MES-4</td>
                        <td align="left" valign="top">Trithorax protein</td>
                        <td align="left" valign="top">several</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">SET-32</td>
                        <td align="left" valign="top">SET domain</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">SET-25</td>
                        <td align="left" valign="top">Putative H3K9 methyltransferase</td>
                        <td align="left" valign="top">SETBP1</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">CSR-1 22G RNA Argonaute</td>
                        <td align="left" valign="top">CSR-1</td>
                        <td align="left" valign="top">Argonaute</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">CSR-1 22G RNA accumulation regulator</td>
                        <td align="left" valign="top">CDE-1</td>
                        <td align="left" valign="top">poly(U) polymerase</td>
                        <td align="left" valign="top">ZCCHC6/ZCCHC11</td>
                     </tr>
                  </tbody>
               </table>
               <table summary="Table 2. Protein factors acting in endogenous small RNA pathways." border="1" width="100%">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr valign="bottom">
                        <th colspan="2" align="left" valign="bottom" width="40%"><span class="bold"><strong>21U RNA pathway factors</strong></span></th>
                        <th align="left" valign="bottom" width="30%"><span class="bold"><strong>Protein/domain(s)</strong></span></th>
                        <th align="left" valign="bottom" width="30%"><span class="bold"><strong>Orthologous Hs protein(s)</strong></span></th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr valign="top">
                        <td rowspan="2" align="left" valign="top" width="25%">Transcription factors</td>
                        <td align="left" valign="top">FKH-3/4/5</td>
                        <td align="left" valign="top">Forkhead transcription factor</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">UNC-130</td>
                        <td align="left" valign="top">Forkhead transcription factor</td>
                        <td align="left" valign="top">FOXD2/FOXD3/FOXD4/FOXD4L1-L6/FOXE1/FOXE3</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">Argonaute</td>
                        <td align="left" valign="top">PRG-1</td>
                        <td align="left" valign="top">Piwi Argonaute</td>
                        <td align="left" valign="top">PIWIL1-4</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">Accumulation regulator</td>
                        <td align="left" valign="top">HENN-1</td>
                        <td align="left" valign="top">Methyltransferase</td>
                        <td align="left" valign="top">HENMT</td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="endoRNAipathwys_fig1.jpg"><img src="endoRNAipathwys_fig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1. ERGO-1 and ALG-3/4 26G siRNA pathways.&nbsp;</b>The templates for 26G siRNA generation are ~500 mRNAs and lincRNAs. A complex containing an RdRP and DCR-1 generates 26G siRNAs
                     in both oocytes/embryos and spermatocytes/sperm. These 26G siRNAs associate with the Argonaute proteins ERGO-1 and ALG-3/4
                     respectively. 22G siRNA generation is triggered and requires another RdRP and multiple cofactors. These 22G siRNAs associate
                     with WAGO class Argonautes to silence their targets in the soma through nuclear and cytoplasmic silencing pathways.
                  </p>
               </div>
            </div>
            <p>ERI complex factors show some interdependence and hierarchy of assembly. Most notably, ERI-5 is proposed to tether the RdRP
               module to DCR-1 to potentiate 26G RNA biogenesis. While the other factors are required for biogenesis of 26G RNAs, loss of
               ERI-5 merely attenuates their expression due to partial compensation by paralog EKL-1 (<a href="#bib127">Thivierge et al., 2012</a>). EKL-1 interacts with DRH-3 (<a href="#bib49">Gu et al., 2009</a>) and substitutes for ERI-5 in the RdRP module, but it does not interact with DCR-1 (<a href="#bib127">Thivierge et al., 2012</a>). Untethered to DCR-1, this EKL-1 RdRP module still produces 26G RNAs of normal genomic distribution and in a DCR-1-dependent
               manner, but 26G RNA accumulation is impaired (<a href="#bib127">Thivierge et al., 2012</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-1"></a>3.1.&nbsp;RRF-3
                        </h3>
                     </div>
                  </div>
               </div>
               <p>RRF-3 is an RNA-dependent RNA polymerase (RdRP) (<a href="#bib119">Smardon et al., 2000</a>) required for 26G RNA biogenesis (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib75">Lee et al., 2006</a>; <a href="#bib43">Gent et al., 2009</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>). <span class="emphasis"><em>C. elegans</em></span> RdRPs and homologs catalyze primer-independent synthesis of antisense siRNAs (<a href="#bib81">Makeyev and Bamford, 2002</a>; <a href="#bib3">Aoki et al., 2007</a>; <a href="#bib91">Pak and Fire, 2007</a>; <a href="#bib116">Sijen et al., 2007</a>), with a preference for initiating transcription with GTP (<a href="#bib3">Aoki et al., 2007</a>) (see <a href="#sec8-1" title="8.1.&nbsp;RRF-1">Section 8.1</a>). The RdRP activity of RRF-3 is necessary for 26G RNA accumulation and function (<a href="#bib96">Pavelec et al., 2009</a>), suggesting that RRF-3 directly transcribes 26G RNAs. Each 26G RNA is likely generated as an independent transcript, as
                  homologous RdRPs exhibit very low processivity (<a href="#bib81">Makeyev and Bamford, 2002</a>; <a href="#bib3">Aoki et al., 2007</a>), and 26G RNAs show irregular phasing and occasional overlap (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib133">Vasale et al., 2010</a>; <a href="#bib38">Fischer et al., 2011</a>). The N terminus of RRF-3 is divergent and may confer target specificity to the ERI RdRP module (<a href="#bib127">Thivierge et al., 2012</a>), as DRH-3 is also involved in biogenesis of other endo-siRNAs. In addition to loss of 26G RNAs, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004510;class=Gene" target="_blank">rrf-3</a></em></span> mutants show depletion of dependent secondary siRNAs, called WAGO 22G RNAs, whose biogenesis is triggered by 26G RNA targeting
                  (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib75">Lee et al., 2006</a>; <a href="#bib43">Gent et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>). Loss of RRF-3 results in the characteristic sperm-origin ts sterile, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a>, and <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotypes associated with compromise of the 26G RNA pathway (<a href="#bib115">Sijen et al., 2001</a>; <a href="#bib118">Simmer et al., 2002</a>; <a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib96">Pavelec et al., 2009</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-2"></a>3.2.&nbsp;DRH-3
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The conserved Dicer-related helicase genes include two paralogs, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001090;class=Gene" target="_blank">drh-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span>, and a probable pseudogene, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001091;class=Gene" target="_blank">drh-2</a></em></span>. These genes are named for the similarity of their encoded DExD/H box helicase domains to that of DCR-1 and likely act as
                  helicases on dsRNA intermediates during biogenesis of siRNAs (<a href="#bib126">Tabara et al., 2002</a>). The functions of DRH-1 and DRH-3 in siRNA biogenesis complexes differ. DRH-1 interacts with DCR-1, RDE-4, and Argonaute
                  RDE-1 within the RDE complex to process primary exo-siRNAs from dsRNA trigger (<a href="#bib126">Tabara et al., 2002</a>; <a href="#bib127">Thivierge et al., 2012</a>). DRH-1 interacts directly with DCR-1 but does not participate in an RdRP module, whereas DRH-3 does not interact directly
                  with DCR-1 but appears to represent an essential component of all <span class="emphasis"><em>C. elegans</em></span> RdRP modules (<a href="#bib49">Gu et al., 2009</a>; <a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib3">Aoki et al., 2007</a>; <a href="#bib89">Nakamura et al., 2007</a>). Thus, loss of DRH-3 results in loss of 26G RNAs (<a href="#bib133">Vasale et al., 2010</a>), although many other populations of small RNAs are also lost. Accordingly, the phenotypes associated with loss of DRH-3
                  are pleiotropic, with more severe phenotypes masking defects attributable to loss of 26G RNAs, and are discussed below with
                  the relevant WAGO and CSR-1 22G RNA pathways.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-3"></a>3.3.&nbsp;ERI-5
                        </h3>
                     </div>
                  </div>
               </div>
               <p>ERI-5 is a protein containing two Tudor domains that putatively tethers the ERI complex RdRP module to DCR-1 (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib127">Thivierge et al., 2012</a>). <span class="emphasis"><em>In vitro</em></span>, recombinant ERI-5 binds recombinant DCR-1 but fails to bind recombinant RRF-3, suggesting that RRF-3 may be modified <span class="emphasis"><em>in vivo</em></span> to generate Tudor domain-binding sites (<a href="#bib127">Thivierge et al., 2012</a>). Loss of ERI-5 results in the sperm-origin ts sterility and <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype of ERI complex mutants (<a href="#bib30">Duchaine et al., 2006</a>). However, due to redundancy with EKL-1, 26G RNAs are only depleted two-fold and the dependent (or secondary) WAGO 22G RNAs
                  are depleted nine-fold in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00021419;class=Gene" target="_blank">eri-5</a></em></span> mutant embryos, resulting in a two-fold up-regulation of 26G RNA target mRNAs (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib127">Thivierge et al., 2012</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-4"></a>3.4.&nbsp;DCR-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Dicer contains a helicase domain, a PAZ domain, and dual RNase III motifs and is a conserved member of a family of RNase III
                  nucleases that cleave dsRNA (<a href="#bib11">Bernstein et al., 2001</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00000939;class=Gene" target="_blank">dcr-1</a></em></span> encodes the sole Dicer homolog of <span class="emphasis"><em>C. elegans</em></span> and acts in microRNA, primary exo-siRNA, and primary endo-siRNA biogenesis (<a href="#bib44">Grishok et al., 2001</a>; <a href="#bib65">Ketting et al., 2001</a>; <a href="#bib68">Knight and Bass, 2001</a>; <a href="#bib52">Han et al., 2009</a>). Loss of DCR-1 results in sterility, with abnormal oocyte formation and absence of fertilization (<a href="#bib44">Grishok et al., 2001</a>; <a href="#bib65">Ketting et al., 2001</a>; <a href="#bib68">Knight and Bass, 2001</a>), as well as heterochronic phenotypes due to defects in microRNA processing (<a href="#bib44">Grishok et al., 2001</a>; <a href="#bib65">Ketting et al., 2001</a>). Maternal inheritance of DCR-1 somewhat ameliorates somatic <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00000939;class=Gene" target="_blank">dcr-1</a></em></span> null mutant phenotypes. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00000939;class=Gene" target="_blank">dcr-1</a></em></span> mutant germline, however, shows an exogenous <span class="underline">R</span>NAi-<span class="underline">de</span>fective (<a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a>) phenotype and impaired transgene silencing, a related phenotype that reports on integrity of exo- and endo-RNAi pathways
                  (<a href="#bib44">Grishok et al., 2001</a>; <a href="#bib65">Ketting et al., 2001</a>; <a href="#bib68">Knight and Bass, 2001</a>). In <span class="emphasis"><em>C. elegans</em></span> extracts, DCR-1 dices long dsRNA into 23 bp duplexes (<a href="#bib65">Ketting et al., 2001</a>). Immuno-purified DCR-1 complexes digest dsRNA processively into siRNAs in the presence of ATP but terminate processing when
                  ssRNA is encountered, demonstrating the specificity for dsRNA substrate (<a href="#bib65">Ketting et al., 2001</a>). Although 26G RNAs are transcribed by RRF-3, they do not show the signature 5&#8217; triphosphate of unprimed synthesis by an
                  RdRP (<a href="#bib81">Makeyev and Bamford, 2002</a>). Rather, 26G RNAs show a 5&#8217; monophosphate characteristic of Dicer products, and indeed DCR-1 is required for their biogenesis
                  (<a href="#bib52">Han et al., 2009</a>), as well as accumulation of dependent WAGO 22G RNAs (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib75">Lee et al., 2006</a>). However, dsRNA substrates in embryo extract are cleaved with similar efficiency by DCR-1 regardless of the 5&#8217; nt identity,
                  indicating that the 5&#8217; guanosine bias of 26G RNAs and other endo-siRNAs is not imposed by preferential DCR-1 processing (<a href="#bib141">Welker et al., 2011</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-5"></a>3.5.&nbsp;DCR-1 helicase domain
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The helicase domain of DCR-1 appears to play a specific role in biogenesis of primary siRNAs. The <span class="emphasis"><em>dcr-1(<a href="http://www.wormbase.org/db/get?name=WBVar00088953;class=Variation" target="_blank">mg375</a>)</em></span> allele encodes a missense mutation in the helicase domain that does not affect ERI complex formation but abrogates biogenesis
                  of 26G RNAs and dependent 22G RNAs (<a href="#bib96">Pavelec et al., 2009</a>, <a href="#bib142">Welker et al., 2010</a>). The <span class="emphasis"><em>dcr-1(<a href="http://www.wormbase.org/db/get?name=WBVar00088953;class=Variation" target="_blank">mg375</a>)</em></span> helicase mutant shows sperm-origin ts sterile, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a>, and <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotypes, but is viable, fertile, and does not show heterochronic defects, indicating that DCR-1 helicase activity is dispensable
                  for microRNA production (<a href="#bib96">Pavelec et al., 2009</a>, <a href="#bib142">Welker et al., 2010</a>). The RNAi sensitivity of the <span class="emphasis"><em>dcr-1(<a href="http://www.wormbase.org/db/get?name=WBVar00088953;class=Variation" target="_blank">mg375</a>)</em></span> helicase mutant is less enhanced than that of other ERI complex mutants (<a href="#bib96">Pavelec et al., 2009</a>), suggesting that the helicase activity of DCR-1 may enhance primary exo-siRNA production from dsRNA triggers. This would
                  be consistent with the theory that the helicase domain also functions as a translocase, allowing DCR-1 to catalyze multiple,
                  processive cleavage events before dissociation from a dsRNA substrate (<a href="#bib142">Welker et al., 2010</a>). Study of the function of DCR-1 in cell-free embryo extracts suggests that the DCR-1 helicase domain facilitates cleavage
                  of dsRNAs with blunt or 5&#8217;-overhanging ends and promotes processive cleavage along the substrate. 3&#8217;-overhanging termini are
                  cleaved to 21-23 nt products without engaging the helicase domain, explaining the intact microRNA levels in the <span class="emphasis"><em>dcr-1(<a href="http://www.wormbase.org/db/get?name=WBVar00088953;class=Variation" target="_blank">mg375</a>)</em></span> helicase mutant (<a href="#bib141">Welker et al., 2011</a>). The <span class="emphasis"><em>C. elegans</em></span> Dicer homolog appears to measure from the 3&#8217; terminus of dsRNA substrate. Cleavage of short (~40 bp) blunt-ended dsRNA yields
                  a 26 nt small RNA with a 22-23 nt passenger strand. Processing of longer (~100 bp) blunt-ended dsRNA in the presence of ATP
                  yields both 26 and 27 nt species, with subsequent internal cleavages yielding ~23 nt duplexes with 3&#8217; overhangs. Failure to
                  produce 26 nt species from internal cleavages strongly supports independent processing of 26G RNAs from short RdRP products
                  rather than sequential cleavage of a long dsRNA precursor. Consistent with these <span class="emphasis"><em>in vitro</em></span> data, deep sequencing libraries show evidence of shorter passenger strands antisense to 26G RNAs. Most commonly, the 5&#8217; nt
                  of the sense read maps to nt 23 of the 26G RNA, corresponding to a 3 nt 3&#8217; overhang for the 26G RNA (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib38">Fischer et al., 2011</a>; <a href="#bib139">Warf et al., 2012</a>). While these sense fragments may be temporarily stabilized in the dsRNA duplex, the 3&#8217; ends of sense reads are highly variable,
                  suggesting 3&#8217; to 5&#8217; degradation (<a href="#bib38">Fischer et al., 2011</a>; <a href="#bib139">Warf et al., 2012</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-6"></a>3.6.&nbsp;ERI-1b
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> encodes two isoforms of a conserved RNase that contains a DEDDh-like 3&#8217; to 5&#8217; exonuclease and a SAP/SAF-box domain (<a href="#bib63">Kennedy et al., 2004</a>). <span class="emphasis"><em>In vitro</em></span>, ERI-1 partially degrades siRNAs with 2-nt 3&#8217; overhangs, but not ssRNA or siRNA internally hybridized to a long RNA (<a href="#bib63">Kennedy et al., 2004</a>). <span class="emphasis"><em>In vivo</em></span>, ERI-1 is required for 26G RNA biogenesis and degrades the 3&#8217; end of the 5.8S rRNA, which pairs with the 5&#8217; end of the 25-28S
                  rRNA. Loss of ERI-1 results in sperm-origin ts sterile, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a>, and <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotypes with loss of 26G RNAs and dependent 22G RNAs (<a href="#bib63">Kennedy et al., 2004</a>; <a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib40">Gabel and Ruvkun, 2008</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>). The significance of 5.8S rRNA processing is unknown but conserved in mammals. Both ERI-1 isoforms are cytoplasmically localized
                  (<a href="#bib63">Kennedy et al., 2004</a>; <a href="#bib40">Gabel and Ruvkun, 2008</a>). Whereas either ERI-1 protein isoform is capable of rescuing 5.8S rRNA processing <span class="emphasis"><em>in vivo</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> mutant phenotypes and 26G RNA accumulation are only rescued by expression of ERI-1b (<a href="#bib63">Kennedy et al., 2004</a>; <a href="#bib40">Gabel and Ruvkun, 2008</a>). ERI-1a may be insufficient for rescue because only ERI-1b interacts with DCR-1 (<a href="#bib30">Duchaine et al., 2006</a>), possibly via its extended C-terminal domain (<a href="#bib63">Kennedy et al., 2004</a>; <a href="#bib40">Gabel and Ruvkun, 2008</a>). ERI-1 exonuclease activity is required for 26G RNA accumulation and rRNA processing (<a href="#bib40">Gabel and Ruvkun, 2008</a>), but the precise function of ERI-1b in 26G RNA biogenesis is unclear. Possibly, ERI-1b recognizes 3&#8217; stem-loop structures
                  in mRNA targets and removes excess nucleotides to generate a suitable RNA substrate for RRF-3-dependent synthesis (<a href="#bib30">Duchaine et al., 2006</a>). Alternatively, RRF-3 may synthesize dsRNA with short 3&#8217; overhangs that must be processed by ERI-1b to create blunt termini
                  that engage DCR-1-mediated production of 26 nt species (<a href="#bib141">Welker et al., 2011</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-7"></a>3.7.&nbsp;ERI-3
                        </h3>
                     </div>
                  </div>
               </div>
               <p>ERI-3 is a protein without identifiable domains (<a href="#bib30">Duchaine et al., 2006</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00021103;class=Gene" target="_blank">eri-3</a></em></span> is encoded in an operon with <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006387;class=Gene" target="_blank">taf-6.1</a></em></span> and can be expressed as a single polypeptide or as a fusion protein with TAF-6.1, which is also detected in immuno-purified
                  DCR-1 complexes. Loss of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00021103;class=Gene" target="_blank">eri-3</a></em></span> results in the characteristic phenotypes of ERI complex compromise, and ERI-3 may serve only to recruit ERI-1b to DCR-1 (<a href="#bib30">Duchaine et al., 2006</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-8"></a>3.8.&nbsp;RDE-4
                        </h3>
                     </div>
                  </div>
               </div>
               <p>RDE-4 contains two dsRNA-binding motifs and binds long dsRNA preferentially <span class="emphasis"><em>in vitro</em></span> without specificity for sequence or overhang structure (<a href="#bib126">Tabara et al., 2002</a>; <a href="#bib93">Parker et al., 2006</a>). RDE-4 protein mediates exo-RNAi in both germline and soma by interacting with DCR-1, DRH-1, and the Argonaute RDE-1 (<a href="#bib125">Tabara et al., 1999</a>). <span class="emphasis"><em>In vitro</em></span> and <span class="emphasis"><em>in vivo</em></span> studies of recombinant RDE-4 have dissected the contributions of the constituent domains to RDE-4 function: The C-terminal
                  domain of RDE-4 directs its dimerization in solution (<a href="#bib93">Parker et al., 2006</a>). The 3&#8217; dsRNA-binding motif is important for dsRNA binding and mediates interaction with DCR-1 along with the linker between
                  the motifs (<a href="#bib94">Parker et al., 2008</a>; <a href="#bib14">Blanchard et al., 2011</a>). RDE-4 binds dsRNA cooperatively (<a href="#bib93">Parker et al., 2006</a>; <a href="#bib94">Parker et al., 2008</a>) and promotes its accumulation <span class="emphasis"><em>in vivo</em></span> (<a href="#bib126">Tabara et al., 2002</a>). During exo-RNAi, RDE-4 binds long trigger dsRNA <span class="emphasis"><em>in vivo</em></span>, but not mRNA or amplified siRNAs (<a href="#bib126">Tabara et al., 2002</a>). RDE-4 is required for primary exo-siRNA and dependent 22G RNA production, but its absence can be partially bypassed by
                  injection of prepared 24-25 nt siRNA duplexes (<a href="#bib95">Parrish and Fire, 2001</a>; <a href="#bib126">Tabara et al., 2002</a>). RDE-4 is required for full production of ERGO-1 class 26G RNAs and dependent 22G RNAs (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib75">Lee et al., 2006</a>; <a href="#bib133">Vasale et al., 2010</a>; <a href="#bib142">Welker et al., 2010</a>). Although its role in ALG-3/4 class 26G RNA production has not been explicitly tested, RDE-4 appears to be required for
                  detection of an endo-siRNA corresponding to ALG-3/4 class 26G target mRNA <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006044;class=Gene" target="_blank">ssp-16</a></em></span> (<a href="#bib142">Welker et al., 2010</a>), and microarray profiling indicates that mRNA levels of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006044;class=Gene" target="_blank">ssp-16</a></em></span> are elevated in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004326;class=Gene" target="_blank">rde-4</a></em></span> mutant adult  (<a href="#bib140">Welker et al., 2007</a>). In spite of this, loss of RDE-4 does not appear to result in the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype and highly penetrant sperm-origin sterility associated with compromise of ALG-3/4 class 26G RNAs (<a href="#bib142">Welker et al., 2010</a>; <a href="#bib14">Blanchard et al., 2011</a>). Perhaps loss of RDE-4 does not fully abrogate 26G RNA biogenesis, as target desilencing is less profound in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004326;class=Gene" target="_blank">rde-4</a></em></span> than <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> mutant embryo (<a href="#bib148">Zhang et al., 2011</a>). The function of RDE-4 in 26G RNA biogenesis remains somewhat unclear. The preference shown by RDE-4 for binding of long
                  dsRNA may promote exo-RNAi by aiding release of dsRNA siRNA duplexes after DCR-1 processing (<a href="#bib93">Parker et al., 2006</a>), but argues against a role for RDE-4 in binding and stabilizing the likely very short dsRNA 26G RNA precursor.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Other factors implicated in 26G RNA accumulation
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-1"></a>4.1.&nbsp;ERI-9
                        </h3>
                     </div>
                  </div>
               </div>
               <p>ERI-9 is a worm-specific putative RNA transferase (<a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib150">Zhuang and Hunter, 2012</a>) that interacts with DCR-1 (<a href="#bib30">Duchaine et al., 2006</a>) and is required only for ERGO-1 class 26G RNA and dependent 22G RNA accumulation (<a href="#bib96">Pavelec et al., 2009</a>). Loss of ERI-9 results in the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype associated with loss of ERGO-1 class 26G RNAs, but neither the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype nor the sperm-origin ts sterility associated with loss of ALG-3/4 class 26G RNAs (<a href="#bib96">Pavelec et al., 2009</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-2"></a>4.2.&nbsp;ERI-6/7
                        </h3>
                     </div>
                  </div>
               </div>
               <p>ERI-6/7 is a helicase protein required only for ERGO-1 class 26G RNA and dependent 22G RNA accumulation (<a href="#bib37">Fischer et al., 2008</a>; <a href="#bib38">Fischer et al., 2011</a>). The ERI-6/7 protein is encoded by antiparallel <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00016561;class=Gene" target="_blank">eri-6</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00016566;class=Gene" target="_blank">eri-7</a></em></span> pre-mRNAs that are trans-spliced to generate a fusion mRNA; these two genes constitute a single, contiguous gene in <span class="emphasis"><em>C. briggsae</em></span> and the <span class="emphasis"><em>C. elegans</em></span> <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain" target="_blank">CB4856</a> isolate (<a href="#bib37">Fischer et al., 2008</a>). Although <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00016566;class=Gene" target="_blank">eri-7</a></em></span> is classified as oogenesis-enriched (<a href="#bib104">Reinke et al., 2004</a>), transcriptional fusion reporters indicate that these genes are also somatically expressed (<a href="#bib37">Fischer et al., 2008</a>). ERI-6/7 is predominantly cytoplasmically localized, suggesting RNA helicase function (<a href="#bib37">Fischer et al., 2008</a>). Loss of ERI-6/7 results in an <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype (<a href="#bib37">Fischer et al., 2008</a>), but only moderately decreased fertility (<a href="#bib38">Fischer et al., 2011</a>). ALG-3/4 class 26G RNAs and all other major classes of <span class="emphasis"><em>C. elegans</em></span> small RNAs are not decreased by loss of ERI-6/7 (<a href="#bib38">Fischer et al., 2011</a>). Interaction between ERI-6/7 and the ERI complex has not been detected, suggesting that ERI-6/7 may function at a later
                  step in ERGO-1 class 26G RNA accumulation, such as Argonaute loading.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-3"></a>4.3.&nbsp;MUT-16
                        </h3>
                     </div>
                  </div>
               </div>
               <p>MUT-16 is a worm-specific protein with proline-rich and glutamine/asparagine-rich regions (<a href="#bib134">Vastenhouw et al., 2003</a>) that functions critically in Mutator foci in the WAGO 22G RNA pathway (<a href="#bib98">Phillips et al., 2012</a>) and will therefore be further discussed below (<a href="#sec10" title="10.&nbsp;Germline amplification of WAGO 22G RNAs in perinuclear Mutator foci">Section 10</a>). Loss of MUT-16 results in severe depletion of ERGO-1 class 26G RNAs and dependent 22G RNAs, but not ALG-3/4 class 26G RNAs
                  (<a href="#bib148">Zhang et al., 2011</a>). Some <span class="emphasis"><em>C. elegans</em></span> laboratory strains, such as the <span class="emphasis"><em>dcr-1(<a href="http://www.wormbase.org/db/get?name=WBVar00088953;class=Variation" target="_blank">mg375</a>)</em></span> helicase mutant, contain a <span class="emphasis"><em>mut-16(<a href="http://www.wormbase.org/db/get?name=WBVar00601255;class=Variation" target="_blank">mg461</a>)</em></span> mutation; this allele fails to express somatic MUT-16 fully, potentially compromising somatic 26G RNA production and complicating
                  interpretation of experimental results pertaining to the ERGO-1 class 26G RNA pathway or somatic WAGO pathway-dependent gene
                  silencing (<a href="#bib148">Zhang et al., 2011</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4-4"></a>4.4.&nbsp;Other Mutator (MUT) proteins
                        </h3>
                     </div>
                  </div>
               </div>
               <p>ERGO-1 class 26G RNAs also require MUT-2, MUT-7, and MUT-15 and show minor dependence on MUT-8 and MUT-14 (<a href="#bib148">Zhang et al., 2011</a>). These proteins are recruited by MUT-16 to form Mutator foci in the germline for WAGO 22G RNA amplification and target silencing
                  (see <a href="#sec10" title="10.&nbsp;Germline amplification of WAGO 22G RNAs in perinuclear Mutator foci">Section 10</a>) (<a href="#bib98">Phillips et al., 2012</a>). ALG-3/4 class 26G RNAs are modestly decreased by loss of MUT-7, but appear intact upon loss of the other Mutator foci MUT
                  proteins (<a href="#bib148">Zhang et al., 2011</a>). The dependence of ERGO-1 class 26G RNAs on this set of factors suggests that Mutator foci may also serve as amplification
                  centers for primary endo-siRNA biogenesis.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;26G RNA Argonaute loading
                     </h2>
                  </div>
               </div>
            </div>
            <p>The ERGO-1 and ALG-3/4 Argonautes are required for accumulation of their respective 26G RNA populations (<a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>), but they are not found in the ERI complex and are thought to act downstream of biogenesis. While many of the Argonautes
               encoded by <span class="emphasis"><em>C. elegans</em></span> do not show conservation of the three critical catalytic residues mediating slicer activity, all of the Argonautes that bind
               primary small RNAs (ERGO-1, ALG-3/4, PRG-1/2, RDE-1, ALG-1/2, and, putatively, CSR-1) show intact catalytic triads in their
               RNase H-related PIWI domains (<a href="#bib145">Yigit et al., 2006</a>). Although the slicer activity of an Argonaute refers generally to the ability to catalyze target cleavage, it appears also
               to play a critical role in effector complex maturation through passenger strand removal. The catalytic activity of RDE-1,
               the Argonaute that binds primary exo-siRNAs, is required only for efficient removal of the passenger strain; thus, the catalytic
               mutant of RDE-1 shows only a partial <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype attributable to impaired target mRNA interaction (<a href="#bib122">Steiner et al., 2009</a>). A putative ERGO-1 catalytic mutant exhibits an <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype (<a href="#bib38">Fischer et al., 2011</a>). This indicates that catalytic activity is required for ERGO-1-mediated triggering of WAGO 22G RNAs, but 26G RNA and passenger
               strand levels have not been assessed. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019971;class=Gene" target="_blank">ergo-1</a></em></span> mutant shows vastly depleted ERGO-1 class 26G RNA levels but two-fold increased passenger strand levels (<a href="#bib38">Fischer et al., 2011</a>), supporting a role for ERGO-1 slicer activity in liberating the passenger strand from a 26G RNA dsRNA intermediate. In comparison,
               both mature ERGO-1 class 26G RNAs and passenger strands are depleted by loss of ERI-1 or ERI-6/7, suggesting function upstream
               of 26G RNA duplex formation (<a href="#bib38">Fischer et al., 2011</a>). Similar analyses of catalytically inactive ALG-3 or ALG-4 have not been reported.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-1"></a>5.1.&nbsp;ALG-3/4
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The highly homologous Argonautes ALG-3 and ALG-4 redundantly bind and stabilize 26G RNAs generated in spermatogenic germline
                  and mediate their effector functions (<a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>). <span class="emphasis"><em>alg-3</em></span> mRNA and protein are enriched in male worms and depleted in female worms (<a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>). In the spermatogenic germline, ALG-3 expression begins in postpachytene spermatocytes, showing cytoplasmic localization
                  with enrichment in P granules (<a href="#bib24">Conine et al., 2010</a>). After spermatogenesis, ALG-3 is detected only in the spermatheca, where it is confined to residual bodies after mature,
                  postmeiotic spermatids have budded off (<a href="#bib24">Conine et al., 2010</a>). The fertility of the <span class="emphasis"><em>alg-3</em></span> or <span class="emphasis"><em>alg-4</em></span> single mutants does not differ significantly from that of wild type (<a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>), reflecting their redundancy, but loss of both ALG-3 and ALG-4 impairs fertility at 20&deg;C and results in the characteristic
                  sperm-origin ts sterility at 25&deg;C (<a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>). The RNAi sensitivity of the single and double mutants is wild type (<a href="#bib52">Han et al., 2009</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-2"></a>5.2.&nbsp;ERGO-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The Argonaute ERGO-1 binds and stabilizes 26G RNAs generated in oogenic germline, embryo, and likely beyond and mediates their
                  effector function (<a href="#bib52">Han et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>). ERGO-1 is nearly absent from L3 and L4 larva and young adult  (<a href="#bib133">Vasale et al., 2010</a>), paralleling decreased detection of ERGO-1 class 26G RNAs during these stages (<a href="#bib52">Han et al., 2009</a>; <a href="#bib12">Billi et al., 2012</a>). In the hermaphrodite oogenic germline, ERGO-1 expression begins at pachytene exit and persists into embryo, showing cytoplasmic
                  localization throughout (<a href="#bib12">Billi et al., 2012</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019971;class=Gene" target="_blank">ergo-1</a></em></span> mutant exhibits the characteristic <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype associated with loss of ERGO-1 class 26G RNAs, but only a minor fertility defect and no <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype (<a href="#bib145">Yigit et al., 2006</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-3"></a>5.3.&nbsp;HENN-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>HENN-1 is the <span class="emphasis"><em>C. elegans</em></span> ortholog of HEN1 methyltransferase (<a href="#bib92">Park et al., 2002</a>). HENN-1 catalyzes 2&#8217;-O-methylation of the 3&#8217; terminus of small RNAs associated with Argonautes of the PIWI clade (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>), namely, ERGO-1 class 26G RNAs and 21U RNAs (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>). Direct interaction between HENN-1 and either ERGO-1 or PRG-1 has not been demonstrated (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>), and recombinant HENN-1 is capable of methylating RNA oligomers in the presence of S-adenosyl methionine (<a href="#bib58">Kamminga et al., 2012</a>). Nevertheless, HENN-1 appears to methylate small RNAs only following Argonaute binding, as loss of ERGO-1 results in loss
                  of methylation for the rare residual ERGO-1 class 26G RNAs (<a href="#bib12">Billi et al., 2012</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00015349;class=Gene" target="_blank">henn-1</a></em></span> mRNA and protein are detected at all stages in both germline and soma with strongest expression in germline and embryo (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>). HENN-1 is detected throughout male and female germlines, with proximal oocytes showing cytoplasmic and intense nucleoplasmic
                  signal; the nucleoplasmic enrichment is lost upon fertilization (<a href="#bib12">Billi et al., 2012</a>). During sperm maturation, HENN-1 becomes enriched in residual bodies, suggesting possible exclusion from mature spermatids
                  (<a href="#bib12">Billi et al., 2012</a>). In embryos, HENN-1, like ERGO-1, is abundant and diffusely cytoplasmic (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>). HENN-1-mediated methylation is critical for ERGO-1 class 26G RNA stability and inheritance into the embryo; in the absence
                  of HENN-1, ERGO-1 class 26G RNAs show size heterogeneity (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>) and increased levels of non-templated nucleotide additions (<a href="#bib87">Montgomery et al., 2012</a>). However, this trimming and tailing activity is not limited to unmethylated 26G RNAs. Analysis of trimming and tailing rates
                  by small RNA class reveals that the methylated 21U RNAs show the lowest frequency and ALG-3/4 class 26G RNAs the highest,
                  but ERGO-1 class 26G RNAs show a trimming and tailing rate nearly as high as that of ALG-3/4 class 26G RNAs (<a href="#bib87">Montgomery et al., 2012</a>). 21U RNAs show significantly decreased perdurance in the absence of HENN-1, but their initial accumulation is less severely
                  affected than that of ERGO-1 class 26G RNAs (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>). The relevance of HENN-1 to 21U RNA accumulation and stability are discussed further below. It is unclear how HENN-1 affects
                  22G RNA levels. Global levels of both WAGO and CSR-1 22G RNAs are decreased in the absence of HENN-1 by ~30% without major
                  changes in size or frequency of addition of non-templated nucleotides (<a href="#bib58">Kamminga et al., 2012</a>), contributing to decreased detection of 22G RNAs dependent upon methylated primary siRNAs (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>). The effects on target mRNAs are difficult to interpret, as global mRNA analysis shows general downregulation of germline-expressed
                  genes upon loss of HENN-1; however, ERGO-1 class 26G RNA target mRNAs are not downregulated, hinting at impaired target silencing
                  (<a href="#bib58">Kamminga et al., 2012</a>), and another report identifies significant upregulation of several ERGO-1 class 26G RNA targets (<a href="#bib87">Montgomery et al., 2012</a>). It is possible that the general depletion of germline mRNAs causes decreased 22G RNA levels due to decreased template-dependent
                  synthesis, but another explanation may be that mRNA levels of critical 22G RNA pathway factors such as PPW-2 and MUT-7 are
                  decreased by loss of HENN-1 (<a href="#bib58">Kamminga et al., 2012</a>). This may also explain a curious phenotype of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00015349;class=Gene" target="_blank">henn-1</a></em></span> mutants: while the soma shows an <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype, presumably due to somatic ERGO-1 class 26G RNA depletion, the germline is <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>). Loss of HENN-1 also results in slightly decreased fertility at 25&deg;C and a mild <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;26G RNA effector function
                     </h2>
                  </div>
               </div>
            </div>
            <p>Loss of factors involved in ERGO-1 and/or ALG-3/4 class 26G RNA biogenesis or accumulation results in desilencing of the relevant
               population of complementary target mRNAs (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib75">Lee et al., 2006</a>; <a href="#bib43">Gent et al., 2009</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>) (<a href="#table3" title="Table 3.  Functions of endogenous small RNAs.">Table 3</a>). Factors that play lesser roles in accumulation such as ERI-5 or Mutator foci MUT proteins show less pronounced silencing
               defects (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib148">Zhang et al., 2011</a>). While several factors have been identified that appear to contribute exclusively to ERGO-1 class 26G RNA accumulation (<a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib38">Fischer et al., 2011</a>; <a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>; <a href="#bib149">Zhuang and Hunter, 2011</a>), ALG-3 and ALG-4 remain the only factors known to be specifically required for accumulation of ALG-3/4 class 26G RNAs (<a href="#bib52">Han et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>). 26G RNAs repress target mRNA expression in their cognate cell types; loss of 26G RNAs does not result in ectopic expression
               within other cell types of the native male or female germline or inappropriate target expression in the opposite germline
               (<a href="#bib52">Han et al., 2009</a>).
            </p>
            <div class="table"><a name="table3"></a><p class="title">Table 3.  Functions of endogenous small RNAs.</p>
               <table summary="Table 3.  Functions of endogenous small RNAs." border="1">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr valign="bottom">
                        <th align="left" valign="bottom"><span class="bold"><strong>Small RNA</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>Features of targets</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>Targeting Mechanism</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>Cofactors involved in function</strong></span></th>
                        <th align="left" valign="bottom"><span class="bold"><strong>Phenotypes of loss</strong></span></th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr valign="top">
                        <td align="left" valign="top">ERGO-1 class 26G RNA</td>
                        <td align="left" valign="top">Duplicated genes, lincRNA genes, noncoding loci</td>
                        <td align="left" valign="top">Silencing in cis and in trans (mismatch-tolerant)</td>
                        <td align="left" valign="top">WAGO (NRDE-3) 22G RNAs, RDE-10/RDE-11, RSD-2, RSD-6, HAF-6</td>
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">ALG-3/4 class 26G RNA</td>
                        <td align="left" valign="top">Spermatogenesis-enriched genes</td>
                        <td align="left" valign="top">Unknown</td>
                        <td align="left" valign="top">WAGO 22G RNAs</td>
                        <td align="left" valign="top">ts Ste (male origin), <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">NRDE-3-associated 22G RNA</td>
                        <td align="left" valign="top">Somatic targets of exo-siRNAs and ERGO-1 class 26G RNAs</td>
                        <td align="left" valign="top">Silencing in cis and in trans (mismatch-tolerant)</td>
                        <td align="left" valign="top">NRDE-1, NRDE-2, NRDE-4</td>
                        <td align="left" valign="top">Nrde</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">HRDE-1-associated 22G RNA</td>
                        <td align="left" valign="top">Germline targets of exo-siRNAs and 21U RNAs</td>
                        <td valign="top">&nbsp;</td>
                        <td align="left" valign="top">NRDE-1, NRDE-2, NRDE-4, HPL-2, MES-4, SET-32, SET-25</td>
                        <td align="left" valign="top">Nrde, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a>, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000966;class=Phenotype" target="_blank">Mrt</a></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">Cytoplasmic WAGO-associated 22G RNA</td>
                        <td align="left" valign="top">Targets of 26G RNAs, 21U RNAs, RDE-1-associated siRNAs</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a>, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001222;class=Phenotype" target="_blank">Mut</a></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">CSR-1 22G RNA</td>
                        <td align="left" valign="top">Germline-expressed coding genes</td>
                        <td align="left" valign="top">Unknown</td>
                        <td valign="top">&nbsp;</td>
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001187;class=Phenotype" target="_blank">Emb</a></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">21U RNA (Type I)</td>
                        <td align="left" valign="top">Transposon genes, pseudogenes  (depleted of protein-coding genes)</td>
                        <td align="left" valign="top">Silencing in trans (mismatch-tolerant)</td>
                        <td align="left" valign="top">HRDE-1-bound 22G RNAs, cytoplasmic WAGO 22G RNAs</td>
                        <td align="left" valign="top">ts Ste, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">TSS-associated 21U RNA (Type II)</td>
                        <td align="left" valign="top">Targets and function unknown</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                        <td valign="top">&nbsp;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top">microRNA</td>
                        <td align="left" valign="top">Diverse</td>
                        <td align="left" valign="top">Silencing in trans (seed sequence pairing)</td>
                        <td valign="top">&nbsp;</td>
                        <td align="left" valign="top">microRNA-specific</td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <p>During 26G RNA biogenesis, DCR-1 likely catalyzes the cleavage of the mRNA template in generating the dsRNA intermediate (<a href="#bib141">Welker et al., 2011</a>), thereby reducing mRNA levels. Subsequently, 26G RNA Argonaute recognition may also trigger target cleavage. However, neither
               is likely to represent a significant means of target silencing, as WAGO 22G RNAs triggered by 26G RNA targeting are the major
               effectors of the 26G RNA endo-RNAi pathway. These secondary siRNAs, discussed in depth below, are strictly required for 26G
               RNA target silencing (<a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>), as well as silencing of other small RNA pathway targets (<a href="#figure2">Figure 2</a>). It is not known precisely how 26G RNA RISC association triggers 22G formation, but target transcripts show 26G RNA-dependent
               accumulation of WAGO 22G RNAs (<a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>), and many 22G and 26G RNAs originate from the same 5&#8217; nt (<a href="#bib42">Gent et al., 2010</a>).
            </p>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="endoRNAipathwys_fig2.jpg"><img src="endoRNAipathwys_fig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 2. Different classes of primary small RNAs trigger secondary siRNA generation.&nbsp;</b>26G siRNAs and 21U/piRNAs trigger secondary 22G siRNA generation by the RdRPs RRF-1 and EGO-1, in conjunction with the helicase
                     DRH-3 and the Tudor domain protein EKL-1. Six mutator proteins that form perinuclear Mutator foci are required for 22G siRNA
                     production in the germline. The RDE-10/11 complex promotes secondary 22G siRNA amplification in the soma.
                  </p>
               </div>
            </div>
            <p>22G RNAs amplify the silencing signal of 26G RNAs and increase the perdurance of the repression. ERGO-1 class 26G RNA levels
               peak in embryos and decline significantly throughout larval development (<a href="#bib52">Han et al., 2009</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>); however, ERGO-1 class 26G RNAs primarily target transcripts that are not germline-intrinsic (<a href="#bib104">Reinke et al., 2004</a>; <a href="#bib52">Han et al., 2009</a>), and triggering the production of WAGO 22G RNAs enables their silencing influence to persist through larval development
               (<a href="#bib52">Han et al., 2009</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>). Similarly, 26G RNAs and ALG-3/4 are depleted in mature sperm, but WAGO 22G RNA Argonaute WAGO-1 is abundant in mature sperm
               (<a href="#bib24">Conine et al., 2010</a>). This precludes significant inheritance of paternal 26G RNAs, but evidently not dependent 22G RNAs: an ALG-3/4 class 26G
               RNA-dependent 22G RNA can be detected even at the L1 larval stage (<a href="#bib52">Han et al., 2009</a>).
            </p>
            <p>Analysis of 26G RNA-dependent 22G RNAs further suggests that 26G RNAs target transcripts in <span class="emphasis"><em>trans</em></span> as well as in <span class="emphasis"><em>cis</em></span>. The X-cluster (<a href="#bib2">Ambros et al., 2003</a>) describes a region of the X chromosome from which abundant 22Gs, but none or very few 26Gs, are generated (<a href="#bib133">Vasale et al., 2010</a>). Accumulation of X-cluster 22G RNAs requires ERGO-1 (<a href="#bib145">Yigit et al., 2006</a>) and a particular ERGO-1 class 26G RNA derived from the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=K02E2.11;class=Gene" target="_blank">K02E2.11</a></em></span> transcript, which shares no other significant homology with the X-cluster (<a href="#bib87">Montgomery et al., 2012</a>). This particular 26G RNA shows multiple putative target sites throughout the X-cluster. None is perfectly complementary,
               and the best sites contain a minimum of three adjacent, central mismatches and two wobble pairs (<a href="#bib87">Montgomery et al., 2012</a>). This indicates that 26G RNA targeting of mRNA targets to initiate 22G RNA production in <span class="emphasis"><em>trans</em></span> is mismatch-tolerant, suggesting immense targeting capacity.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6-1"></a>6.1.&nbsp;Sperm-origin ts sterility due to loss of ALG-3/4 class 26G RNAs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>As indicated above, compromise of ALG-3/4 class 26G RNA pathway activity results in sterility at 25&deg;C due to defective spermatogenesis
                  and spermiogenesis (<a href="#bib52">Han et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>). The <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype also results from X chromosome missegregation during compromised spermatogenesis. <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00021103;class=Gene" target="_blank">eri-3</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004510;class=Gene" target="_blank">rrf-3</a></em></span> mutants do not show maternal rescue of this fertility defect, unlike the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype (<a href="#bib149">Zhuang and Hunter, 2011</a>). 26G RNA and dependent 22G RNA production is lost in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00021103;class=Gene" target="_blank">eri-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00021419;class=Gene" target="_blank">eri-5</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004510;class=Gene" target="_blank">rrf-3</a></em></span> mutants at both elevated and permissive temperatures (<a href="#bib30">Duchaine et al., 2006</a>), indicating that it is not siRNA production that is sensitive to temperature but rather that these siRNAs are only required
                  for functional spermatogenesis at elevated temperatures. The temperature-sensitive period for fertility in the absence of
                  ALG-3/4 class 26G RNAs coincides with spermatogenesis at the L4 larval stage (<a href="#bib43">Gent et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>).
               </p>
               <p>The defects occurring during spermatogenesis at 25&deg;C in the absence of 26G RNAs have been detailed in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004510;class=Gene" target="_blank">rrf-3</a></em></span> and <span class="emphasis"><em>dcr-1(<a href="http://www.wormbase.org/db/get?name=WBVar00088953;class=Variation" target="_blank">mg375</a>)</em></span> helicase mutants as well as in the <span class="emphasis"><em>alg-3; alg-4</em></span> double mutant, which lacks only ALG-3/4 class 26G RNAs (<a href="#bib43">Gent et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>). The phenotypes are highly similar, suggesting that compromise of only the male germline-expressed class of 26G RNAs is
                  sufficient to cause these defects; consistent with this, loss of ERGO-1 ERI-9, or ERI-6/7 results on only very mild decreases
                  in fertility at 25&deg;C (<a href="#bib52">Han et al., 2009</a>; <a href="#bib38">Fischer et al., 2011</a>).
               </p>
               <p>The spermatogenesis defects associated with compromised ALG-3/4 class 26G RNA target silencing are as follows: (1) mutant
                  gonads show delayed spermatogenesis and decreased production of sperm-like nuclei; (2) nuclear abnormalities first become
                  evident in primary spermatocytes; (3) some late-stage spermatocytes show chromatin bridges and arrest as multinucleate masses
                  or produce spermatids with large, misshapen, or multiple nuclei; (4) spermatocytes that develop without obvious morphological
                  abnormalities produce smaller numbers of residual bodies and sperm; (5) mutant sperm show abnormal wreaths of microtubules
                  surrounding nuclei; and (6) activation of these mutant sperm is impaired&#8212;many fail to form pseudopods and instead show long
                  spike structures and impaired motility (<a href="#bib43">Gent et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>).
               </p>
               <p>Precisely how ALG-3/4 class 26G RNAs promote thermotolerant fertility is unknown. It is unlikely that desilencing of a single
                  target results in these sperm development and motility defects, as individual depletion of 68 spermatogenesis-enriched transcripts
                  desilenced in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004510;class=Gene" target="_blank">rrf-3</a></em></span> mutants failed to suppress the ts sterility and <a href="http://www.wormbase.org/db/get?name=EMS;class=Cell" target="_blank">EMS</a> mutagenesis failed to identify any suppressors among several million genomes screened (<a href="#bib96">Pavelec et al., 2009</a>). Many of these defects are observed with variable penetrance at 20&deg;C, but nearly all sperm show the abnormal spike structures during activation at 25&deg;C, correlating with the penetrant ts sterility. Many ALG-3/4 class 26G RNAs target transcripts
                  encoding major sperm proteins (<a href="#bib6">Asikainen et al., 2007</a>; <a href="#bib52">Han et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>), which assemble into filamentous fibers in the pseudopod. One function of 26G RNAs may be to promote male fertility by limiting
                  excess accumulation of major sperm proteins. Additionally, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004510;class=Gene" target="_blank">rrf-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> mutant hermaphrodites cultured at 23&deg;C generate embryos with significant spindle structure abnormalities including tripolar
                  spindles or male pronuclei with supernumerary microtubule asters (<a href="#bib43">Gent et al., 2009</a>). These defects, as well as the abnormal microtubule wreaths in spermatids (<a href="#bib43">Gent et al., 2009</a>) and the chromosome segregation defects in spermatocytes (<a href="#bib43">Gent et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>), suggest a role for ALG-3/4 class 26G RNAs in regulating microtubule organization during spermatogenesis. Alternatively,
                  these microtubule defects may reflect abnormalities in chromosome complements resulting from dysfunctional spermatogenesis.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6-2"></a>6.2.&nbsp;Enhanced RNAi sensitivity due to loss of ERGO-1 class 26G RNAs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In <span class="emphasis"><em>C. elegans</em></span>, exo-RNAi and endo-RNAi engage a common downstream pathway to effect target silencing: the WAGO 22G RNAs. The earliest evidence
                  for intersection between these two pathways was the identification of mutants that show both <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> and Mutator (<a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001222;class=Phenotype" target="_blank">Mut</a>, loss of transposon silencing) phenotypes (<a href="#bib66">Ketting et al., 1999</a>; <a href="#bib125">Tabara et al., 1999</a>), indicating that common mechanisms mediate exo-RNAi and germline transposon silencing. Discovery of distinct primary and
                  secondary phases of exo-RNAi first prompted the hypothesis that loss of RRF-3 might release limiting, common cofactors for
                  function in the amplification phase of RNAi (<a href="#bib115">Sijen et al., 2001</a>). The WAGO Argonautes appear to represent these limiting factors in RNAi: loss of one can impair sensitivity to exogenous
                  RNAi (<a href="#bib130">Tijsterman et al., 2002b</a>; <a href="#bib145">Yigit et al., 2006</a>), and overexpression of one enhances accumulation of 22G RNAs and RNAi sensitivity (<a href="#bib145">Yigit et al., 2006</a>). Accordingly, loss of ERGO-1 class 26G RNAs results in an <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype due to decreased competition for WAGO Argonaute occupancy by secondary siRNAs (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib75">Lee et al., 2006</a>; <a href="#bib145">Yigit et al., 2006</a>; <a href="#bib49">Gu et al., 2009</a>), in line with an earlier observation that loss of ERI-1 increases accumulation of exo-RNAi-triggered siRNAs (<a href="#bib63">Kennedy et al., 2004</a>). The converse is also true: in the absence of exogenous dsRNA, RDE-1 scavenges diverse small RNAs and triggers endogenous
                  somatic 22G RNA production (<a href="#bib140">Welker et al., 2007</a>; <a href="#bib49">Gu et al., 2009</a>; <a href="#bib25">Corr&ecirc;a et al., 2010</a>); loss of RDE-1 and these dependent endo-siRNAs enhances accumulation of an ERGO-1-dependent somatic 22G RNA (<a href="#bib145">Yigit et al., 2006</a>).
               </p>
               <p>Loss of ERGO-1 class 26G RNAs, but not ALG-3/4 class 26G RNAs, results in the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype of ERI complex mutants (<a href="#bib52">Han et al., 2009</a>). Only ERGO-1 class 26G RNAs are inherited by offspring to generate abundant 22G RNAs and effect target silencing (<a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>). <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> mutants show strong maternal, but not paternal, rescue of the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype in the soma (<a href="#bib149">Zhuang and Hunter, 2011</a>), consistent with this phenotype arising due to liberation of WAGO Argonautes in the absence of maternally inherited ERGO-1
                  class 26G RNAs and dependent 22G RNAs. It is possible that upregulation of target mRNAs encoding factors such as helicases
                  and dsRNA-binding proteins may also contribute to the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001258;class=Phenotype" target="_blank">Eri</a> phenotype associated with loss of ERGO-1 class 26G RNAs (<a href="#bib38">Fischer et al., 2011</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec7"></a>7.&nbsp;22G RNAs comprise two major small RNA pathways
                     </h2>
                  </div>
               </div>
            </div>
            <p>Early capture of <span class="emphasis"><em>C. elegans</em></span> small RNAs using a protocol insensitive to 5&#8217; structure indicated that a majority of antisense small RNAs are 22 nt in length
               (<a href="#bib2">Ambros et al., 2003</a>). Protocols selective for 5&#8217; monophosphorylated species failed to recover these sequences effectively (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib77">Lim et al., 2003</a>), revealing a 5&#8217; structure different from that of the 26G RNAs. Subsets of these small RNAs were subsequently shown to exhibit
               a 5&#8217; triphosphate (<a href="#bib91">Pak and Fire, 2007</a>) and a 3&#8217; hydroxyl (<a href="#bib91">Pak and Fire, 2007</a>; <a href="#bib116">Sijen et al., 2007</a>; <a href="#bib49">Gu et al., 2009</a>). Finally, 22 nt siRNAs in <span class="emphasis"><em>C. elegans</em></span> show a prominent 5&#8217; guanosine bias (<a href="#bib2">Ambros et al., 2003</a>; <a href="#bib108">Ruby et al., 2006</a>; <a href="#bib49">Gu et al., 2009</a>), distinguishing these as 22G RNAs.
            </p>
            <p>As a whole, 22G RNAs target about ~50% of the annotated coding genome, with most 22G RNAs targeting unique genome sequences
               (<a href="#bib49">Gu et al., 2009</a>). 22G RNAs mapping antisense to mRNAs show enrichment primarily at transcript 3&#8217; ends, consistent with RdRP engagement at
               the mRNA 3&#8217; terminus, but some also show enrichment at 5&#8217; ends (<a href="#bib49">Gu et al., 2009</a>). Most 22G RNAs are germline expressed and deposited into embryo, potentially coinherited with their mRNA targets (<a href="#bib49">Gu et al., 2009</a>). The 22G RNAs comprise two distinct classes of small RNAs that are synthesized by similar RdRP modules but engage unique
               pathways mediated by specific Argonautes and cofactor proteins to effect entirely distinct outcomes. The WAGO 22G RNAs bind
               semi-redundant Argonautes of the worm-specific WAGO clade to mediate silencing of certain protein-coding genes, transposons,
               pseudogenes, and cryptic loci through both transcriptional and post-transcriptional mechanisms (<a href="#bib51">Guang et al., 2008</a>; <a href="#bib49">Gu et al., 2009</a>; <a href="#bib17">Burkhart et al., 2011</a>; <a href="#bib18">Burton et al., 2011</a>; <a href="#bib4">Ashe et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib47">Gu, S.G., et al. 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). The CSR-1 22G RNAs bind CSR-1, another WAGO Argonaute, and target germline-expressed genes to fulfill an essential role
               in promoting chromosome segregation (<a href="#bib22">Claycomb et al., 2009</a>). Thus, 22G RNAs target both silent and expressed loci genome-wide to maintain the germline.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec8"></a>8.&nbsp;22G RNA biogenesis: The 22G RNA RdRP module
                     </h2>
                  </div>
               </div>
            </div>
            <p>RdRPs RRF-1 and EGO-1 both contribute to biogenesis of 22G RNAs (<a href="#figure2">Figure 2</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a>; <a href="http://www.wormbase.org/db/get?name=WBGene00004508;class=Gene" target="_blank">rrf-1</a></em></span> double mutant lacks 22G RNAs synthesized <span class="emphasis"><em>de novo</em></span> in the germline (<a href="#bib49">Gu et al., 2009</a>). EGO-1 and RRF-1 proteins show greater than 50% sequence identity (<a href="#bib119">Smardon et al., 2000</a>) and collaborate to generate WAGO 22G RNAs, but EGO-1 alone is required for CSR-1 22G RNAs (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). Accordingly, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> transcript is classified as germline-intrinsic (<a href="#bib104">Reinke et al., 2004</a>), and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> mRNA and protein are enriched in young adult, adult, and embryo but nearly absent during larval development (<a href="#bib119">Smardon et al., 2000</a>; <a href="#bib136">Vought et al., 2005</a>; <a href="#bib22">Claycomb et al., 2009</a>). EGO-1 protein is also enriched in the nuclear fraction of cell lysates (<a href="#bib80">Maine et al., 2005</a>). In contrast, a transgene expressing GFP::RRF-1 that rescues the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004508;class=Gene" target="_blank">rrf-1</a></em></span> mutant phenotype shows robust somatic expression, with prominent cytoplasmic and weak nuclear localization (<a href="#bib3">Aoki et al., 2007</a>). Despite their different expression patterns, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004508;class=Gene" target="_blank">rrf-1</a></em></span> gene is encoded directly downstream of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> in an operon (<a href="#bib82">Mangone et al., 2010</a>).
            </p>
            <p>Because exo-RNAi produces WAGO 22G RNAs to mediate transcript knockdown (<a href="#bib115">Sijen et al., 2001</a>; <a href="#bib91">Pak and Fire, 2007</a>; <a href="#bib116">Sijen et al., 2007</a>), the RNAi sensitivity phenotypes associated with loss of RRF-1 versus EGO-1 reveal their respective contributions to the
               WAGO pathway. RRF-1 is required in somatic tissues for 22G RNA accumulation and therefore exo-RNAi (<a href="#bib115">Sijen et al., 2001</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>), whereas no role for EGO-1 is detected during RNAi in the soma (<a href="#bib119">Smardon et al., 2000</a>; <a href="#bib115">Sijen et al., 2001</a>). Accordingly, 22G RNAs mapping to ERGO-1 class 26G RNA targets are largely RRF-1-dependent (<a href="#bib133">Vasale et al., 2010</a>). In the germline, however, EGO-1 and RRF-1 are partially redundant for WAGO 22G RNA biogenesis. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004508;class=Gene" target="_blank">rrf-1</a></em></span> mutant germline is sensitive to exo-RNAi of germline transcripts (<a href="#bib115">Sijen et al., 2001</a>), revealing compensation by EGO-1. This explains why loss of RRF-1 does not recapitulate the ts sterility and other germline
               phenotypes associated with total compromise of WAGO 22G RNA target silencing. Loss of EGO-1 results in a partial germline
               <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype (<a href="#bib119">Smardon et al., 2000</a>). Interestingly, injection of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> dsRNA enhances the germline defects of an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> null mutant (<a href="#bib119">Smardon et al., 2000</a>), suggesting the possibility that RRF-1 may be recruited, although insufficiently, to CSR-1 22G RNA biogenesis in the absence
               of EGO-1. EGO-1 and phenotypes associated with its loss are discussed further with CSR-1 22G RNAs, below (<a href="#sec22" title="22.&nbsp;Phenotypes associated with CSR-1 22G RNA pathway compromise">Section 22</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec8-1"></a>8.1.&nbsp;RRF-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In a cell-free system for analyzing secondary siRNA production triggered by exo-RNAi, RRF-1 accounts for 90% of RdRP activity
                  (<a href="#bib3">Aoki et al., 2007</a>). Secondary siRNAs generated by immuno-purified GFP::RRF-1 complexes are complementary to a supplied RNA template, and production
                  is inhibited by addition of RNA chain elongation terminator. RRF-1 also exhibits terminal transferase activity in cell extracts.
                  93% of RdRP products generated by GFP::RRF-1 complexes show a 5&#8217; guanosine. This indicates that the 5&#8217; guanosine bias of 22G
                  RNAs, and perhaps also 26G RNAs, reflects an RdRP preference for initiation of transcription with GTP. GFP::RRF-1 complexes
                  show very low processivity when incubated with ribonucleotides and mRNA template, generating products 22-23 nt in length;
                  this length is not imposed by DCR-1-mediated cleavage, as DCR-1 is not recovered in GFP::RRF-1 complexes. DRH-3 immuno-purifies
                  with GFP::RRF-1 and is required for RdRP activity. GFP::RRF-1 complexes fail to generate RdRP products from long dsRNA template,
                  but polyadenylated and non-polyadenylated mRNAs are equally effective templates <span class="emphasis"><em>in vitro</em></span>; in cell extract, the non-polyadenylated template stimulates more robust RdRP activity. Neither in cell extract nor <span class="emphasis"><em>in vitro</em></span> do GFP::RRF-1 complexes catalyze extension of an RNA primer complementary to a template, arguing against primer-dependent
                  synthesis. In contrast, GFP::RRF-1 complexes successfully incorporate labeled GTP as the 5&#8217; nucleotide of RdRP products <span class="emphasis"><em>in vitro</em></span>, demonstrating unprimed synthesis (<a href="#bib3">Aoki et al., 2007</a>). The 5&#8217; ends of these RdRP products are also sensitive to capping, suggesting these products show the 5&#8217; triphosphate of
                  22G RNAs generated <span class="emphasis"><em>in vivo</em></span> (<a href="#bib91">Pak and Fire, 2007</a>). Unprimed synthesis also occurs <span class="emphasis"><em>in vivo</em></span>, as secondary siRNAs triggered by a mismatch-containing primary siRNA show perfect complementarity to the endogenous target
                  (<a href="#bib116">Sijen et al., 2007</a>). Furthermore, the rare secondary siRNAs that are generated 3&#8217; to the trigger sequence could not originate via extension
                  of exo-siRNA-derived primers (<a href="#bib91">Pak and Fire, 2007</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec8-2"></a>8.2.&nbsp;DRH-3
                        </h3>
                     </div>
                  </div>
               </div>
               <p>As mentioned above (see <a href="#sec3-2" title="3.2.&nbsp;DRH-3">Section 3.2</a>), DRH-3 is an essential component of all known <span class="emphasis"><em>C. elegans</em></span> RdRP modules (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib3">Aoki et al., 2007</a>; <a href="#bib89">Nakamura et al., 2007</a>; <a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). DRH-3 is detected at all developmental stages and is not restricted to germline (<a href="#bib22">Claycomb et al., 2009</a>), but the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> transcript is classified as germline-intrinsic (<a href="#bib104">Reinke et al., 2004</a>), and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> mRNA levels are three-fold higher in young adult and adult than in larva (<a href="#bib89">Nakamura et al., 2007</a>). The most severe phenotypes associated with loss of DRH-3 result from compromise of the CSR-1 22G RNA pathway (<a href="#bib22">Claycomb et al., 2009</a>) and will be discussed below (<a href="#sec22" title="22.&nbsp;Phenotypes associated with CSR-1 22G RNA pathway compromise">Section 22</a>). Nevertheless, partial loss of DRH-3 function results in phenotypes and target upregulation attributable to loss of WAGO
                  22G RNAs: point mutations in the helicase domain yield viable <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> hypomorphic mutants that show variable germline and somatic <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a>, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001222;class=Phenotype" target="_blank">Mut</a>, ts sterile, and <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotypes as well as embryonic lethality possibly due to reduced CSR-1 22G and transposon 22G RNA accumulation (<a href="#bib49">Gu et al., 2009</a>). Additionally, reduction of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> by RNAi results in hypersensitivity to DNA-damaging agents (<a href="#bib89">Nakamura et al., 2007</a>). This phenotype could be the result of defective selection of repair templates because of chromosome segregation defects
                  (CSR-1 pathway). Alternatively, it may be that <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> mutant animals already experience an increase in DNA double-strand breaks because of defects in transposon silencing (WAGO
                  pathway) and are therefore hypersensitive to DNA-damaging agents. Finally, DRH-3 could play a more direct role in DNA double-strand
                  break repair. A sterile <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> null mutant is insensitive to RNAi in germline but not soma (<a href="#bib30">Duchaine et al., 2006</a>), presumably due to rescue by maternal inheritance. Deep sequencing of this mutant using a 5&#8217; monophosphate-independent protocol
                  shows dramatically reduced endo-siRNA reads mapping to protein-coding genes, pseudogenes, repetitive elements, and unannotated
                  loci but intact microRNAs and 21U RNAs (<a href="#bib49">Gu et al., 2009</a>). Residual 22G RNAs detected in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> null mutant map primarily to the 3&#8217; ends of transcripts, suggesting that the RdRP module may be loaded at transcript 3&#8217; ends,
                  with DRH-3 possibly acting to promote sequential initiation of polymerization (<a href="#bib49">Gu et al., 2009</a>). Proposed roles for DRH-3 within the RdRP module include relaxing template secondary structure and promoting 22G RNA dissociation
                  from the transcript for Argonaute loading (<a href="#bib49">Gu et al., 2009</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec8-3"></a>8.3.&nbsp;EKL-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>EKL-1 is a Tudor domain protein (<a href="#bib105">Robert et al., 2005</a>) paralogous to ERI-5. Like ERI-5, EKL-1 interacts with DRH-3, but only ERI-5 interacts DCR-1 (<a href="#bib127">Thivierge et al., 2012</a>). This is consistent with 22G RNA biogenesis occurring through a DCR-1-independent mechanism (<a href="#bib3">Aoki et al., 2007</a>; <a href="#bib49">Gu et al., 2009</a>; <a href="#bib96">Pavelec et al., 2009</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene" target="_blank">ekl-1</a></em></span> transcript is classified as germline-intrinsic (<a href="#bib104">Reinke et al., 2004</a>), but EKL-1 is detected at all developmental stages and does not show germline restriction (<a href="#bib22">Claycomb et al., 2009</a>). Like RRF-1/EGO-1 and DRH-3, EKL-1 is also required for 22G RNA biogenesis (<a href="#bib49">Gu et al., 2009</a>) and exhibits phenotypes characteristic of compromise of CSR and WAGO 22G RNA pathways. Loss of EKL-1 results in sterility
                  as well as somatic and germline defects in exo-RNAi and related phenomena (<a href="#bib67">Kim et al., 2005</a>; <a href="#bib105">Robert et al., 2005</a>; <a href="#bib106">Rocheleau et al., 2008</a>; <a href="#bib22">Claycomb et al., 2009</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec9"></a>9.&nbsp;Triggering of WAGO 22G RNAs
                     </h2>
                  </div>
               </div>
            </div>
            <p>WAGO 22G RNAs serve as the critical amplification pathway upon which most primary small RNAs in <span class="emphasis"><em>C. elegans</em></span> converge. Accordingly, WAGO 22G RNAs are generated from and target transcripts targeted by 26G RNAs, primary exo-siRNAs,
               RDE-1-scavenged small RNAs, and 21U RNAs. The requirements for triggering of WAGO 22G RNAs by each of these primary small
               RNA types are addressed below (<a href="#sec9-1" title="9.1.&nbsp;Triggering by 26G RNAs">Sections 9.1</a>, <a href="#sec9-2" title="9.2.&nbsp;Triggering by primary exo-siRNAs">9.2</a>, <a href="#sec9-3" title="9.3.&nbsp;Triggering by RDE-1-scavenged small RNAs">9.3</a>, and <a href="#sec9-4" title="9.4.&nbsp;Triggering by 21U RNAs">9.4</a>).
            </p>
            <p>Secondary siRNAs in <span class="emphasis"><em>C. elegans</em></span> were first identified during exo-RNAi as a population of small RNAs that are antisense to a target mRNA but not derived from
               the initial dsRNA trigger (<a href="#bib115">Sijen et al., 2001</a>). However, the existence of some means of amplification had been hypothesized several years prior. It had been noted that
               a few molecules of dsRNA are sufficient to deplete a cellular pool of target mRNAs, arguing against simple stoichiometric
               interaction and indicating the involvement of a catalytic and/or amplification mechanism (<a href="#bib35">Fire et al., 1998</a>; <a href="#bib85">Montgomery and Fire, 1998</a>). Although trigger dsRNA cleavage generates some measure of amplification, this was recognized as still insufficient to explain
               the potent silencing ability of exogenous dsRNA (<a href="#bib115">Sijen et al., 2001</a>). The essential contribution of RdRP activity to this amplification was subsequently established by demonstrating complete
               insensitivity of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004508;class=Gene" target="_blank">rrf-1</a></em></span> mutant soma to injection of prepared siRNA duplexes (<a href="#bib115">Sijen et al., 2001</a>).
            </p>
            <p>WAGO 22G RNAs are generated from spliced transcripts targeted by the triggering primary small RNAs (<a href="#bib91">Pak and Fire, 2007</a>). RdRP initiation appears to be non-random however as 22G RNAs show some phasing across a target transcript, and similar
               sets of 22G RNAs are produced in different transgenic lines (<a href="#bib116">Sijen et al., 2007</a>). 22G RNA biogenesis shows a limited degree of spreading from the target site of the primary small RNA (<a href="#bib115">Sijen et al., 2001</a>). During exo-RNAi, spreading occurs primarily 5&#8217; of the primary RNA target site with respect to the sense of the target transcript
               (<a href="#bib115">Sijen et al., 2001</a>; <a href="#bib91">Pak and Fire, 2007</a>; <a href="#bib116">Sijen et al., 2007</a>). ERGO-1 class 26G RNAs rarely target the first ~100 nt of a transcript, but 22G RNAs show robust coverage of this interval
               at these same targets (<a href="#bib133">Vasale et al., 2010</a>). Similarly, 21U RNAs trigger 22G RNA production primarily 5&#8217; of the target site (<a href="#bib72">Lee et al., 2012</a>). This may have implications for transgenic construct design, as transgenes encoding foreign sequence 3&#8217; to endogenous sequence
               are more likely to evade silencing by 21U RNAs (<a href="#bib114">Shirayama et al., 2012</a>); presumably, 5&#8217; spread of 22G RNA biogenesis from foreign to endogenous sequence could be detrimental, resulting in selection
               for evaders. The range of the spreading is relatively modest and may vary by primary small RNA abundance. Exo-RNAi appears
               to generate secondary siRNAs with a range of 100-180 nt 5&#8217; to the region targeted by dsRNA (<a href="#bib1">Alder et al., 2003</a>). 21U RNAs trigger production of 22G RNAs within the surrounding 40-100 nt (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9-1"></a>9.1.&nbsp;Triggering by 26G RNAs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>As described above, 26G RNAs trigger robust production of WAGO 22G RNAs at target sites, and these secondary siRNAs are essential
                  for target silencing (<a href="#bib24">Conine et al., 2010</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>). However, loss of 26G RNAs does not significantly deplete 22G RNA levels for all loci (<a href="#bib43">Gent et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib133">Vasale et al., 2010</a>), indicating the existence of other primary triggers of WAGO 22G RNA biogenesis.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9-2"></a>9.2.&nbsp;Triggering by primary exo-siRNAs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Competition between endo- and exo-RNAi implicates primary exo-siRNAs and the associated Argonaute RDE-1 in triggering 22G
                  RNAs as well (<a href="#bib115">Sijen et al., 2001</a>; <a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib75">Lee et al., 2006</a>; <a href="#bib145">Yigit et al., 2006</a>). Exo-RNAi triggers production of antisense secondary siRNAs that share all of the features and dependencies of endogenously
                  triggered WAGO 22G RNAs (<a href="#bib115">Sijen et al., 2001</a>; <a href="#bib128">Tijsterman et al., 2002a</a>; <a href="#bib3">Aoki et al., 2007</a>; <a href="#bib91">Pak and Fire, 2007</a>; <a href="#bib116">Sijen et al., 2007</a>; <a href="#bib49">Gu et al., 2009</a>). Expression of a single hairpin-derived 22 nt siRNA (22siR) targeting the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006759;class=Gene" target="_blank">unc-22</a></em></span> mRNA 3&#8217; UTR reveals some of the requirements for RDE-1-triggered 22G RNA biogenesis (<a href="#bib116">Sijen et al., 2007</a>): a 22siR showing perfect target site complementarity depletes <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006759;class=Gene" target="_blank">unc-22</a></em></span> mRNA levels by 50% and generates abundant secondary siRNAs upstream of the target site. A mutated 22siR bearing target site
                  mismatches at positions 10-12 binds the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006759;class=Gene" target="_blank">unc-22</a></em></span> mRNA but does not trigger secondary siRNA biogenesis. Single mismatches at position 11 or position 21 weaken target silencing,
                  but do not alter the range of secondary siRNAs triggered. The lack of an effect of mismatch at the putative cleavage site
                  (position 11) on the distribution of secondary siRNAs suggests that RDE-1-mediated target cleavage is not necessary for triggering
                  secondary siRNA production (<a href="#bib116">Sijen et al., 2007</a>), consistent with a restricted role RDE-1 cleavage activity in passenger strand removal (<a href="#bib122">Steiner et al., 2009</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9-3"></a>9.3.&nbsp;Triggering by RDE-1-scavenged small RNAs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In the absence of exogenous dsRNA, RDE-1 binds a variety of DCR-1 products; among these are microRNAs and dsRNA-derived siRNAs
                  cleaved from endogenous hairpins or bidirectionally transcribed genomic regions (<a href="#bib25">Corr&ecirc;a et al., 2010</a>). Some of these scavenged siRNAs initiate significant 22G RNA production at target loci: the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=Y47H10A.5;class=Gene" target="_blank">Y47H10A.5</a></em></span> transcript generates abundant 22G RNAs triggered by miR-243-loaded RDE-1, for which a perfectly complementary target site
                  22 nt in length appears on <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=Y47H10A.5;class=Gene" target="_blank">Y47H10A.5</a></em></span> transcript (<a href="#bib140">Welker et al., 2007</a>; <a href="#bib49">Gu et al., 2009</a>; <a href="#bib25">Corr&ecirc;a et al., 2010</a>). Like primary exo-siRNAs, miR-243-mediated triggering of these 22G RNAs requires RDE-4 in addition to RDE-1 (<a href="#bib49">Gu et al., 2009</a>). In contrast, the chief microRNA pathway Argonaute ALG-1 does not trigger secondary siRNA biogenesis at target sites (<a href="#bib121">Steiner et al., 2007</a>); this may explain why microRNAs fail to elicit dramatic target knockdown despite the vast abundances of individual microRNAs
                  relative to individual endo-siRNAs. It is important to note that 22G RNAs themselves are rarely bound by RDE-1 (<a href="#bib115">Sijen et al., 2001</a>), prohibiting triggering of further siRNA biogenesis from aberrant tertiary targets in <span class="emphasis"><em>trans</em></span>. Similarly, 22G RNAs bound by WAGO Argonautes also do not appear to trigger further amplification or spreading at target
                  sites: a sensor transgene with a single target site for an abundant 22G RNA shows confined loss of secondary siRNAs at the
                  target site when the triggering 22G RNA is removed (<a href="#bib87">Montgomery et al., 2012</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9-4"></a>9.4.&nbsp;Triggering by 21U RNAs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A fourth population of primary small RNAs converging on the WAGO 22G RNAs are 21U RNAs, the piRNAs of <span class="emphasis"><em>C. elegans</em></span>. These small RNAs are bound by and require the PIWI clade Argonaute PRG-1 for accumulation and targeting (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>). 21U RNAs constitute a library of mismatch-tolerant sequences that are depleted for targeting of protein-coding genes but
                  capable of silencing non-self transcripts by triggering WAGO 22G RNAs (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). 21U RNA properties, biogenesis, and function are discussed in depth below. Initial studies identified 21U RNA-dependent
                  secondary siRNAs (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>), but only recently were these established as WAGO 22G RNAs and the targeting requirements reported. 21U RNAs trigger 22G
                  RNA biogenesis at engineered sensor targets with up to two target site mismatches and at endogenous targets with up to three
                  or four mismatches (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>). Seed site pairing, which plays an important role in microRNA-mediated target repression, may also be important for 21U
                  RNA targeting. One study found that nearly perfect pairing is required at positions 2-8, with a maximum of one G-U wobble
                  pair across the interval, to trigger 22G RNA biogenesis (<a href="#bib72">Lee et al., 2012</a>); however, another study found that transgenic 21U RNA sensor transcripts are effectively targeted regardless of the position
                  of dinucleotide mismatches across the length of the 21U RNA (<a href="#bib8">Bagijn et al., 2012</a>). Interestingly, 21U RNAs seem less effective at stimulating a secondary siRNA response, as less than 5% of target sites
                  exhibiting perfect complementarity to 21U RNAs show unambiguous triggering of 22G RNAs (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>). This may be in part due to the low abundances of many 21U RNA species, as levels of 22G RNAs triggered correlate with both
                  21U RNA abundance and degree of complementarity (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>). Recombinant PRG-1 shows some slicing activity <span class="emphasis"><em>in vitro</em></span>, but this activity is dispensable for triggering 22G RNA production as well as target silencing (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>). Also in support of a cleavage-independent mechanism, 21U RNA target site mismatches at positions 10 and 11 do not abrogate
                  22G RNA triggering (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec10"></a>10.&nbsp;Germline amplification of WAGO 22G RNAs in perinuclear Mutator foci
                     </h2>
                  </div>
               </div>
            </div>
            <p>The observation that some natural isolates of <span class="emphasis"><em>C. elegans</em></span> are mutators (strains that have a high spontaneous germ line mutation rate) not only led to the use of these strains in genetic
               screens but also to research into the causal &#8220;mutagen&#8221;, DNA transposons of the Tc1/<span class="emphasis"><em>mariner</em></span> family, including Tc1 and Tc3. To understand how activity of these transposons is regulated, genetic screens were performed
               to identify mutants that display increased Tc1 or Tc3 transposition in the germline (<a href="#bib23">Collins et al., 1987</a>; <a href="#bib66">Ketting et al., 1999</a>; <a href="#bib134">Vastenhouw et al., 2003</a>). The description of the first mutator (<a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001222;class=Phenotype" target="_blank">Mut</a>) gene identified, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003504;class=Gene" target="_blank">mut-7</a></em></span> (<a href="#bib66">Ketting et al., 1999</a>), and of the first genetic screen for exo-RNAi-defective (<a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a>) mutants (<a href="#bib125">Tabara et al., 1999</a>), provided the first evidence of a role for RNAi in transposon silencing in the germline: many <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> mutants displayed <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001222;class=Phenotype" target="_blank">Mut</a> phenotypes and many <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001222;class=Phenotype" target="_blank">Mut</a> mutants displayed <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotypes. These and subsequent screens for related phenomena (<a href="#bib66">Ketting et al., 1999</a>; <a href="#bib134">Vastenhouw et al., 2003</a>; <a href="#bib45">Grishok et al., 2005</a>; <a href="#bib67">Kim et al., 2005</a>; <a href="#bib105">Robert et al., 2005</a>) recurrently identified members of a group of proteins that have now been implicated in formation of Mutator foci. These
               perinuclear processing compartments interact with RRF-1 and are required for WAGO 22G RNA amplification and target silencing
               in the germline (<a href="#bib98">Phillips et al., 2012</a>). The six proteins that assemble to form Mutator foci are MUT-16, MUT-7, MUT-8/RDE-2, MUT-2/RDE-3, MUT-15, and MUT-14. Consistent
               with their critical, non-redundant role in the WAGO 22G RNA pathway that silences transposons, loss of any of these proteins
               results in mutator (<a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001222;class=Phenotype" target="_blank">Mut</a>), ts sterile, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a>, and <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotypes (<a href="#bib23">Collins et al., 1987</a> <a href="#bib66">Ketting et al., 1999</a>; <a href="#bib128">Tijsterman et al., 2002a</a>; <a href="#bib134">Vastenhouw et al., 2003</a>; <a href="#bib20">Chen et al., 2005</a>; <a href="#bib131">Tops et al., 2005</a>; <a href="#bib49">Gu et al., 2009</a>; <a href="#bib98">Phillips et al., 2012</a>), although a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003507;class=Gene" target="_blank">mut-14</a></em></span> mutant carrying a missense mutation shows only a germline <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype (<a href="#bib128">Tijsterman et al., 2002a</a>; <a href="#bib98">Phillips et al., 2012</a>).
            </p>
            <p>MUT-16 is a worm-specific protein containing proline-rich and glutamine/asparagine-rich regions (<a href="#bib134">Vastenhouw et al., 2003</a>). A translational MUT-16::GFP fusion protein is expressed broadly in cytoplasm and nuclei (<a href="#bib134">Vastenhouw et al., 2003</a>). MUT-16 is also required for accumulation of ERGO-1 class 26G RNAs, whereas other Mutator foci components are not strictly
               required (<a href="#bib148">Zhang et al., 2011</a>). MUT-7 is a 3&#8217; to 5&#8217; exonuclease (<a href="#bib66">Ketting et al., 1999</a>) that is conserved in animals (<a href="#bib98">Phillips et al., 2012</a>). A <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003504;class=Gene" target="_blank">mut-7</a>::gfp</em></span> transcriptional fusion shows expression in many different cell types in larva and adult (<a href="#bib55">Hunt-Newbury et al., 2007</a>). MUT-8/RDE-2 contains no known domains and may only exist to recruit MUT-7 (<a href="#bib131">Tops et al., 2005</a>; <a href="#bib98">Phillips et al., 2012</a>). MUT-2/RDE-3 is a &#946; nucleotidyltransferase; mutations of conserved residues at the active site produce <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotypes, suggesting polymerase activity is required for function (<a href="#bib20">Chen et al., 2005</a>). MUT-2/RDE-3 expressed in frog oocytes shows no polymerase activity, unlike homolog CDE-1 (<a href="#bib70">Kwak and Wickens, 2007</a>). MUT-15 contains no known domains. MUT-14 is a DEAD box RNA helicase (<a href="#bib128">Tijsterman et al., 2002a</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec10-1"></a>10.1.&nbsp;Dynamics of Mutator foci
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Germline Mutator foci are most prominent in the mitotic region and transition zone, showing perinuclear localization adjacent
                  to P granules until late meiosis (<a href="#bib98">Phillips et al., 2012</a>). Somatic Mutator foci components are cytoplasmically localized but still required for efficient WAGO 22G RNA amplification
                  and silencing function (<a href="#bib98">Phillips et al., 2012</a>), as most of the component proteins are required for exo-RNAi in both germline and soma (<a href="#bib66">Ketting et al., 1999</a>; <a href="#bib134">Vastenhouw et al., 2003</a>; <a href="#bib20">Chen et al., 2005</a>; <a href="#bib67">Kim et al., 2005</a>; <a href="#bib131">Tops et al., 2005</a>; <a href="#bib49">Gu et al., 2009</a>; <a href="#bib98">Phillips et al., 2012</a>).  Failure to assemble Mutator foci in somatic tissues likely decreases the effectiveness of the components in promoting
                  genome surveillance, as transposons efficiently silenced in germline can be active in the soma (<a href="#bib32">Emmons and Yesner, 1984</a>). Inheritance of maternal Mutator foci may be important for transposon suppression in offspring (<a href="#bib66">Ketting et al., 1999</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec10-2"></a>10.2.&nbsp;Assembly of Mutator foci
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Mutator foci may associate directly with the nuclear pore complex (<a href="#bib98">Phillips et al., 2012</a>), much like P granules (<a href="#bib99">Pitt et al., 2000</a>). However, Mutator foci are not required for P granule formation, nor are P granules required for Mutator focus assembly
                  (<a href="#bib98">Phillips et al., 2012</a>). Consistent with the many associations of the nuclear pore with RNA processing bodies, nuclear pore complex proteins routinely
                  emerge from screens for factors involved in RNAi (<a href="#bib134">Vastenhouw et al., 2003</a>; <a href="#bib67">Kim et al., 2005</a>). Mutator focus assembly is a regulated process: MUT-16, which contains a Q/N-rich domain that may mediate protein-protein
                  interactions, recruits MUT-2/RDE-3, MUT-15, MUT-14, and MUT-8/RDE-2, which itself recruits MUT-7 (<a href="#bib131">Tops et al., 2005</a>; <a href="#bib98">Phillips et al., 2012</a>). In the absence of MUT-16, the other five components do not interact (<a href="#bib98">Phillips et al., 2012</a>). Intriguingly, formation of Mutator foci appears not to require the presence of WAGO 22G RNAs, as loss of DRH-3, EKL-1,
                  or EGO-1 and RRF-1 simultaneously or alone does not affect their formation, nor does loss of 26G RNAs, CSR-1 22G RNAs, or
                  several individual WAGO Argonautes (<a href="#bib98">Phillips et al., 2012</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec10-3"></a>10.3.&nbsp;Function of Mutator foci
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Mutator foci serve as amplification centers for WAGO 22G RNAs in the germline (<a href="#bib98">Phillips et al., 2012</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00011323;class=Gene" target="_blank">mut-15</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003508;class=Gene" target="_blank">mut-16</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003499;class=Gene" target="_blank">mut-2</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00003499;class=Gene" target="_blank">rde-3</a></em></span> mutants show defects in secondary siRNA accumulation in response to exo-RNAi (<a href="#bib116">Sijen et al., 2007</a>), and deep sequencing of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003499;class=Gene" target="_blank">mut-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003504;class=Gene" target="_blank">mut-7</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003508;class=Gene" target="_blank">mut-16</a></em></span> mutants reveals general defects in accumulation of WAGO 22G RNAs (<a href="#bib49">Gu et al., 2009</a>; <a href="#bib148">Zhang et al., 2011</a>). Somatic X-cluster 22G RNAs are depleted in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00011323;class=Gene" target="_blank">mut-15</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003499;class=Gene" target="_blank">mut-2</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00003499;class=Gene" target="_blank">rde-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003504;class=Gene" target="_blank">mut-7</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003508;class=Gene" target="_blank">mut-16</a></em></span> mutants (<a href="#bib148">Zhang et al., 2011</a>; <a href="#bib98">Phillips et al., 2012</a>). A <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003507;class=Gene" target="_blank">mut-14</a></em></span> mutant shows intact X-cluster 22G RNAs, but loss of germline 22G RNAs (<a href="#bib98">Phillips et al., 2012</a>). Therefore, the germline and somatic <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotypes of Mutator foci mutants reflect their respective capacities to support 22G RNA biogenesis. In Mutator foci mutants,
                  however, loss of WAGO 22G RNAs is incomplete, and depletion does not appear proportionate for all species. For example, the
                  targets showing greatest 22G RNA depletion upon loss of MUT-16, MUT-2, or MUT-7 are those that generate very high levels of
                  22G RNAs (<a href="#bib49">Gu et al., 2009</a>; <a href="#bib148">Zhang et al., 2011</a>; <a href="#bib98">Phillips et al., 2012</a>). Furthermore, MUT-2/RDE-3 is required for full target knockdown during dsRNA feeding, but not when dsRNA is transgenically
                  expressed (<a href="#bib20">Chen et al., 2005</a>). These data suggest that Mutator foci are most important for heavy siRNA amplification and are less critical when primary
                  small RNA triggers are abundant. The mechanisms by which Mutator foci promote WAGO 22G RNA amplification are not known, although
                  recruitment of RdRP to target transcripts is likely involved. Germline RRF-1 localizes exclusively to Mutator foci, whereas
                  EGO-1 localizes both to P granules and Mutator foci (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib98">Phillips et al., 2012</a>), reflecting the dual function of EGO-1 in the WAGO and CSR-1 22G RNA pathways (<a href="#bib49">Gu et al., 2009</a>). The adjacency of Mutator foci and P granules suggests possible interaction between the two (<a href="#bib98">Phillips et al., 2012</a>). P granules contain ALG-3 and PRG-1 (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib138">Wang and Reinke, 2008</a>; <a href="#bib24">Conine et al., 2010</a>), two Argonautes that require WAGO 22G RNA-mediated amplification to effect target silencing (<a href="#bib27">Das et al., 2008</a>; <a href="#bib49">Gu et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>; <a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>). When a nascent mRNA passes through the nuclear pore, the presence of the hydrophobic P granule may retard its diffusion
                  to enhance recognition by regulatory molecules such as loaded Argonautes (<a href="#bib112">Sheth et al., 2010</a>). Once a target transcript enters a P granule and is recognized by a primary small RNA Argonaute, the adjacent Mutator focus
                  may engage the transcript for secondary siRNA amplification. Notably, CSR-1 22G RNAs do not accumulate to high levels or trigger
                  gene silencing (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>) and do not require Mutator foci for amplification, as a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003508;class=Gene" target="_blank">mut-16</a></em></span> mutant does not show depletion of 22G RNAs targeting a validated set of CSR-1 22G RNA pathway targets (<a href="#bib148">Zhang et al., 2011</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec11"></a>11.&nbsp;Additional amplification of WAGO 22G RNAs by RDE-10/RDE-11
                     </h2>
                  </div>
               </div>
            </div>
            <p>The RDE-10/RDE-11 protein complex amplifies 22G RNAs and potentiates degradation of the targets that serve as their templates
               (<a href="#bib144">Yang et al., 2012</a>; <a href="#bib147">Zhang et al., 2012</a>). RDE-10 and RDE-11 are worm-specific genes (<a href="#bib144">Yang et al., 2012</a>). <a href="http://www.wormbase.org/#012-3-6" target="_blank">WormBase</a> reports RDE-10 as containing a Maelstrom domain, which adopts an RNase H fold that may confer nuclease or RNA-binding functionality.
               RDE-11 contains a RING-type zinc finger domain (<a href="#bib144">Yang et al., 2012</a>). Mutation of a key zinc-coordinating residue in RDE-11 abrogates its interaction with RDE-10 <span class="emphasis"><em>in vitro</em></span> and results in failure to rescue <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00023421;class=Gene" target="_blank">rde-11</a></em></span> mutant phenotypes (<a href="#bib144">Yang et al., 2012</a>). Thus, zinc binding facilitates the formation of the RDE-10/RDE-11 complex, possibly by promoting folding of RDE-11.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec11-1"></a>11.1.&nbsp;RDE-10/RDE-11 amplifies 22G RNAs triggered by exo-RNAi and ERGO-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The RDE-10/RDE-11 complex appears to play a significant role in amplification of WAGO 22G RNAs triggered by exo-RNAi (<a href="#bib144">Yang et al., 2012</a>; <a href="#bib147">Zhang et al., 2012</a>). The absence of RDE-10 or RDE-11 results in a partial germline and somatic <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype, but no fertility defect or <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype (<a href="#bib144">Yang et al., 2012</a>; <a href="#bib147">Zhang et al., 2012</a>). The exo-RNAi deficiency of these mutants is dosage-sensitive and results from impaired accumulation of secondary WAGO 22G
                  RNAs: loss of RDE-10/RDE-11 depletes exo-RNAi-triggered 22G RNAs by ~80% (<a href="#bib144">Yang et al., 2012</a>; <a href="#bib147">Zhang et al., 2012</a>). ERGO-1 class 26G RNAs also variably require RDE-10/RDE-11 for maximal amplification of secondary siRNAs and target silencing.
                  ERGO-1 is detected in immuno-purified RDE-10 complexes (<a href="#bib144">Yang et al., 2012</a>; <a href="#bib147">Zhang et al., 2012</a>), and embryonic 22G RNAs targeting ERGO-1 26G RNA targets are depleted by more than half upon loss of RDE-10 (<a href="#bib147">Zhang et al., 2012</a>). 22G RNAs are most abundant in the germline (<a href="#bib49">Gu et al., 2009</a>; <a href="#bib42">Gent et al., 2010</a>), and some primary small RNAs are restricted to germline tissues (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>; <a href="#bib138">Wang and Reinke, 2008</a>; <a href="#bib24">Conine et al., 2010</a>). The more pronounced effects of RDE-10/RDE-11 on exo-RNAi and ERGO-1-mediated silencing therefore suggest that the RDE-10/RDE-11
                  complex may be particularly important in somatic 22G RNA amplification. Indeed, the somatic X-cluster and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=Y47H10A.5;class=Gene" target="_blank">Y47H10A.5</a></em></span> 22G RNAs triggered by ERGO-1 and RDE-1, respectively, showed strong dependence on RDE-10 and RDE-11 for accumulation (<a href="#bib147">Zhang et al., 2012</a>). In comparison, accumulation of a germline-enriched, ERGO-1-independent 22G RNA targeting <span class="emphasis"><em>T01A4.3</em></span> (<a href="#bib30">Duchaine et al., 2006</a>) does not require RDE-10/11 (<a href="#bib147">Zhang et al., 2012</a>), nor apparently do many other WAGO 22G RNAs, microRNAs, 21U RNAs, 26G RNAs, primary exo-siRNAs, or CSR-1 22G RNAs (<a href="#bib144">Yang et al., 2012</a>; <a href="#bib147">Zhang et al., 2012</a>). Perhaps the RDE-10/RDE-11 complex supports WAGO 22G RNA amplification in the soma, in the absence of perinuclear Mutator
                  foci.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec11-2"></a>11.2.&nbsp;RDE-10/RDE-11-mediated target degradation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In addition to amplifying 22G RNAs, the RDE-10/RDE-11 complex may directly promote target deadenylation and degradation. RDE-10
                  binds mRNAs targeted by exo-RNAi, even in the absence of RDE-11 (<a href="#bib144">Yang et al., 2012</a>). This binding activity is lost in the absence of RDE-1, but intact in the absence of RRF-1, indicating that primary exo-siRNAs
                  are guiding association of RDE-10/RDE-11 with mRNA targets independently of secondary 22G RNAs (<a href="#bib144">Yang et al., 2012</a>). After recognition, RDE-10-bound mRNAs are deadenylated and degraded in an RDE-11-dependent manner (<a href="#bib144">Yang et al., 2012</a>). Targets undergoing RDE-10/RDE-11-mediated degradation retain 5&#8217; caps, providing a transiently stable substrate for continued
                  secondary siRNA production (<a href="#bib144">Yang et al., 2012</a>). Thus, the RDE-10/RDE-11 complex may promote secondary siRNA biogenesis by binding exo-RNAi targets to promote RRF-1 binding
                  as deadenylation and degradation occur (<a href="#bib144">Yang et al., 2012</a>). The RDE-10/RDE-11-mediated degradation pathway likely acts in parallel to target silencing mediated by somatic nuclear
                  Argonaute NRDE-3 and other WAGO Argonautes, as many NRDE-3-bound 22G RNAs are intact in the absence of RDE-10/RDE-11 (<a href="#bib144">Yang et al., 2012</a>; <a href="#bib147">Zhang et al., 2012</a>). In somatic exo-RNAi in particular, the RDE-10/RDE-11 complex and NRDE-3 may represent the two main pathways for target
                  silencing: <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00021634;class=Gene" target="_blank">rde-10</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00023421;class=Gene" target="_blank">rde-11</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019862;class=Gene" target="_blank">nrde-3</a></em></span> mutants all show partial sensitivity to <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006759;class=Gene" target="_blank">unc-22</a></em></span> dsRNA feeding, but loss of RDE-10 or RDE-11 in a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019862;class=Gene" target="_blank">nrde-3</a></em></span> mutant background results in complete insensitivity to <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006759;class=Gene" target="_blank">unc-22</a></em></span> dsRNA and a defect in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006759;class=Gene" target="_blank">unc-22</a></em></span> mRNA knockdown as profound as observed upon complete loss of exo-RNAi in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004323;class=Gene" target="_blank">rde-1</a></em></span> mutant (<a href="#bib144">Yang et al., 2012</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec11-3"></a>11.3.&nbsp;RSD-2, RSD-6, and/or HAF-6 may potentiate RDE-10/RDE-11 function
                        </h3>
                     </div>
                  </div>
               </div>
               <p>RSD-2 and Tudor domain protein RSD-6 are worm-specific proteins; RSD-2 contains no annotated domains (<a href="#bib129">Tijsterman et al., 2004</a>) HAF-6 is a half-molecule ATP-binding cassette transporter detected with a reticular pattern in intestinal, muscle, and germline
                  cells, with perinuclear localization in germline (<a href="#bib124">Sundaram et al., 2006</a>). Like <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00021634;class=Gene" target="_blank">rde-10</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00023421;class=Gene" target="_blank">rde-11</a></em></span> mutants, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004681;class=Gene" target="_blank">rsd-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004684;class=Gene" target="_blank">rsd-6</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001816;class=Gene" target="_blank">haf-6</a></em></span> mutants also show dosage-sensitive <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotypes (<a href="#bib129">Tijsterman et al., 2004</a>; <a href="#bib124">Sundaram et al., 2006</a>; <a href="#bib53">Han et al., 2008</a>) due to defects in WAGO 22G RNA accumulation (<a href="#bib53">Han et al., 2008</a>; <a href="#bib147">Zhang et al., 2012</a>). These three proteins also regulate accumulation and target silencing of a subset of WAGO 22G RNAs, many of which are ERGO-1-
                  and RDE-10/RDE-11-dependent (<a href="#bib147">Zhang et al., 2012</a>). RSD-2, RSD-6, and HAF-6 are all required for accumulation of X-cluster 22G RNAs (<a href="#bib147">Zhang et al., 2012</a>). The shared dosage-sensitive <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype and functional overlap of RDE-10/RDE-11 and RSD-2, RSD-6, and HAF-6 suggests the possibility of a common complex.
                  Supporting this, RSD-2 interacts with both RDE-10 and RSD-6 (<a href="#bib129">Tijsterman et al., 2004</a>; <a href="#bib147">Zhang et al., 2012</a>). However, loss of RSD-2 also results in a <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001222;class=Phenotype" target="_blank">Mut</a> phenotype at 25&deg;C, and loss of RSD-6 causes ts sterile and <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotypes (<a href="#bib53">Han et al., 2008</a>). These more severe WAGO 22G RNA pathway phenotypes indicate that RSD-2 and RSD-6 may play more critical or additional roles
                  in WAGO 22G RNA accumulation or function. The perinuclear germline localization of HAF-6 (<a href="#bib124">Sundaram et al., 2006</a>) suggests that HAF-6 could also interact with Mutator foci, and there is some evidence for genetic interaction between HAF-6
                  and Mutator foci components: double heterozygous <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004324;class=Gene" target="_blank">mut-8</a>/rde-2(+/-); mut-7(+/-)</em></span> mutants show impaired exo-RNAi, and heterozygosity for a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001816;class=Gene" target="_blank">haf-6</a></em></span> mutation in either a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004324;class=Gene" target="_blank">mut-8</a>/rde-2(+/-)</em></span> or <span class="emphasis"><em>mut-7(+/-)</em></span> heterozygous background also results in defects in exo-RNAi (<a href="#bib124">Sundaram et al., 2006</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec12"></a>12.&nbsp;WAGO 22G RNA Argonaute loading
                     </h2>
                  </div>
               </div>
            </div>
            <p>Early analysis of <span class="emphasis"><em>C. elegans</em></span> Argonautes revealed significant expansion of the protein family (<a href="#bib44">Grishok et al., 2001</a>). A large subset of these cluster phylogenetically into a WAGO clade that is roughly equally distant from the conserved PIWI
               and AGO clades (<a href="#bib145">Yigit et al., 2006</a>). Included in these are CSR-1, the Argonaute that mediates the CSR-1 22G RNA pathway (<a href="#bib22">Claycomb et al., 2009</a>), and related protein <a href="http://www.wormbase.org/db/get?name=C04F12.1;class=Gene" target="_blank">C04F12.1</a>. Also variably included is the divergent RDE-1, the first <span class="emphasis"><em>C. elegans</em></span> Argonaute to be linked to RNAi (<a href="#bib125">Tabara et al., 1999</a>). The other 12 Argonaute proteins in this clade constitute the WAGO Argonautes of the WAGO 22G RNA pathway. These 12 Argonautes
               bind and stabilize WAGO 22G RNAs semiredundantly to mediate target silencing. Simultaneous loss of all 12 Argonautes in the
               MAGO12 Argonaute mutant results in penetrant ts sterile, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a>, and <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotypes and massive depletion of 22G RNAs that are not dependent on CSR-1 (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). The ts sterility of the MAGO12 mutant can only be partially rescued by wild type sperm, likely because loss of 21U RNAs,
               one of the triggers of 22G RNA biogenesis, results in ts fertility defects in female germline as well as male (<a href="#bib138">Wang and Reinke, 2008</a>). These WAGO-dependent 22G RNAs are also globally depleted in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003504;class=Gene" target="_blank">mut-7</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003499;class=Gene" target="_blank">mut-2</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00003499;class=Gene" target="_blank">rde-3</a></em></span> mutants (<a href="#bib49">Gu et al., 2009</a>), reflecting the contribution of Mutator foci components to WAGO 22G RNA accumulation (<a href="#bib98">Phillips et al., 2012</a>).
            </p>
            <p>Although the phylogenetic relationships between the Argonautes can vary depending on whether full-length protein or specific
               domains were used for phylogenetic analysis, the 12 WAGO Argonautes of the cognate 22G RNA pathway (<a href="#bib44">Grishok et al., 2001</a>; <a href="#bib145">Yigit et al., 2006</a>; <a href="#bib87">Montgomery et al., 2012</a>) are usually clustered into three subclades referred to by phylogenetic branch: Branch 1 comprises WAGO-1 (<a href="http://www.wormbase.org/db/get?name=R06C7.1;class=Gene" target="_blank">R06C7.1</a>), WAGO-2 (<a href="http://www.wormbase.org/db/get?name=F55A12.1;class=Gene" target="_blank">F55A12.1</a>), WAGO-3 (PPW-2/<a href="http://www.wormbase.org/db/get?name=Y110A7A.18;class=Gene" target="_blank">Y110A7A.18</a>), WAGO-4 (<a href="http://www.wormbase.org/db/get?name=F58G1.1;class=Gene" target="_blank">F58G1.1</a>), WAGO-5 (<a href="http://www.wormbase.org/db/get?name=ZK1248.7;class=Gene" target="_blank">ZK1248.7</a>); Branch 2 comprises WAGO-6/8 (SAGO-2/F56A6.1), WAGO-7 (PPW-1/C18E3.7), WAGO-8/6 (SAGO-1/<a href="http://www.wormbase.org/db/get?name=K12B6.1;class=Gene" target="_blank">K12B6.1</a>); and Branch 3 comprises WAGO-9 (HRDE-1/<a href="http://www.wormbase.org/db/get?name=C16C10.3;class=Gene" target="_blank">C16C10.3</a>), WAGO-10 (<a href="http://www.wormbase.org/db/get?name=T22H9.3;class=Gene" target="_blank">T22H9.3</a>), WAGO-11 (<a href="http://www.wormbase.org/db/get?name=Y49F6A.1;class=Gene" target="_blank">Y49F6A.1</a>), WAGO-12 (NRDE-3/<a href="http://www.wormbase.org/db/get?name=R04A9.2;class=Gene" target="_blank">R04A9.2</a>) (<a href="#bib49">Gu et al., 2009</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00011061;class=Gene" target="_blank">wago-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00018862;class=Gene" target="_blank">wago-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004094;class=Gene" target="_blank">wago-3</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00004094;class=Gene" target="_blank">ppw-2</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00022877;class=Gene" target="_blank">wago-5</a></em></span> transcripts are classified as germline-intrinsic (<a href="#bib104">Reinke et al., 2004</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00010263;class=Gene" target="_blank">wago-4</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00007624;class=Gene" target="_blank">wago-9</a>/<a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00007624?query=hrde-1#0-9e-3" target="_blank">hrde-1</a></em></span> transcripts are classified as oogenesis-enriched, whereas <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00020707;class=Gene" target="_blank">wago-10</a></em></span> is spermatogenesis-enriched (<a href="#bib104">Reinke et al., 2004</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019862;class=Gene" target="_blank">wago-12</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00019862;class=Gene" target="_blank">nrde-3</a></em></span> transcript is classified as mixed oogenesis-somatic (<a href="#bib104">Reinke et al., 2004</a>). The others are not assigned germline enrichment classifications (<a href="#bib104">Reinke et al., 2004</a>). WAGO-12/NRDE-3 and WAGO-9/HRDE-1 will be discussed in greater detail below in the context of the nuclear RNAi pathway.
            </p>
            <p>Loss of single WAGO Argonautes can cause mild defects in exo-RNAi sensitivity and germline transposon silencing, such as the
               germline <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype observed in a natural <span class="emphasis"><em>C. elegans</em></span> isolate (<a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain" target="_blank">CB4856</a>) that lacks functional WAGO-3/PPW-2 (<a href="#bib130">Tijsterman et al., 2002b</a>; <a href="#bib134">Vastenhouw et al., 2003</a>; <a href="#bib145">Yigit et al., 2006</a>). Simultaneous loss of multiple WAGO Argonautes within the same phylogenetic branch produces more significant impairment.
               The MAGO strain lacks Branch 2 and WAGO-4 Argonautes and shows a somatic and germline <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype and a ts fertility defect (<a href="#bib145">Yigit et al., 2006</a>). The Quadruple mutant lacks all of the Branch 1 Argonautes but WAGO-1 and shows an <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype (<a href="#bib49">Gu et al., 2009</a>). Similarly, WAGO Argonautes bind and stabilize WAGO 22G RNAs semiredundantly. In exo-RNAi of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006759;class=Gene" target="_blank">unc-22</a></em></span> transcript, Branch 2 Argonautes bind secondary siRNAs redundantly, and overexpression of any Branch 2 Argonaute rescues the
               RNAi defect of the MAGO strain, which otherwise fails to accumulate <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006759;class=Gene" target="_blank">unc-22</a></em></span> secondary siRNAs (<a href="#bib145">Yigit et al., 2006</a>). Still, particular 22G RNAs in germline and soma show different dependencies upon different WAGO Argonautes. The Quadruple
               mutant, while showing a germline <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype, still generates wild-type levels of a germline <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=F37D6.3;class=Gene" target="_blank">F37D6.3</a></em></span> 22G RNA, whereas loss of the remaining Branch 1 Argonaute, WAGO-1, severely depletes this species (<a href="#bib49">Gu et al., 2009</a>). Loss of all five Branch 1 Argonautes in the Quintuple mutant results in dramatic reduction of germline 22G RNAs, whereas
               a MAGO+2 mutant lacking Branch 2, WAGO-3, and WAGO-4 Argonautes (<a href="#bib49">Gu et al., 2009</a>) shows near wild type levels of germline 22G RNAs, but loss of a somatic <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=Y47H10A.5;class=Gene" target="_blank">Y47H10A.5</a></em></span> 22G RNA triggered by RDE-1 (<a href="#bib49">Gu et al., 2009</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec13"></a>13.&nbsp;WAGO 22G RNA-mediated post-transcriptional gene silencing
                     </h2>
                  </div>
               </div>
            </div>
            <p>Early evidence suggested that exo-RNAi occurred primarily through a post-transcriptional silencing mechanism. Injection of
               dsRNA segments corresponding to intron and promoter sequences does not elicit interference (<a href="#bib35">Fire et al., 1998</a>), and injection of dsRNA targeting a transcript encoded in an operon generally does not decrease levels of transcripts encoded
               downstream of the target within the operon appreciably (<a href="#bib85">Montgomery et al., 1998</a>). Little progress has been made in understanding cytoplasmic, post-transcriptional target silencing by WAGO 22G RNAs. It
               is unlikely to involve Argonaute-mediated target cleavage, as WAGO Argonautes lack the catalytic triad residues required for
               slicer activity (<a href="#bib145">Yigit et al., 2006</a>); furthermore, target cleavage induced by introduction of 5&#8217; triphosphorylated siRNAs into cell lysate is predominantly mediated
               by CSR-1 (<a href="#bib3">Aoki et al., 2007</a>), which is not involved in the WAGO 22G RNA pathway (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). This suggests that other nucleases may be recruited to the target mRNA to mediate mRNA degradation in the WAGO pathway.
               It is possible that RDE-10/RDE-11-mediated target deadenylation and degradation represents the main post-transcriptional mechanism
               for target silencing (<a href="#bib144">Yang et al., 2012</a>). In the germline, post-transcriptional target silencing may occur at P granules, as GFP::WAGO-1 is expressed in the germline
               and localizes to perinuclear P granules (<a href="#bib49">Gu et al., 2009</a>), like Argonautes ALG-3, PRG-1, and CSR-1 (<a href="#bib9">Batista et al., 2008</a>, <a href="#bib138">Wang and Reinke, 2008</a>; <a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib24">Conine et al., 2010</a>). Earlier data connecting the nonsense-mediated decay (NMD) pathway with exo-RNAi likely resulted from the presence of the
               <span class="emphasis"><em>mut-16(<a href="http://www.wormbase.org/db/get?name=WBVar00601255;class=Variation" target="_blank">mg461</a>)</em></span> mutation in the mutant strains of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004880;class=Gene" target="_blank">smg-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004883;class=Gene" target="_blank">smg-5</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004884;class=Gene" target="_blank">smg-6</a></em></span> genes, which encode NMD pathway proteins (<a href="#bib148">Zhang et al., 2011</a>). This explanation is consistent with an initial report that failed to demonstrate a role for the SMG proteins in exo-RNAi
               (<a href="#bib85">Montgomery et al., 1998</a>), although a transgenic system for studying transcriptional gene silencing showed impairment of repression upon depletion
               of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004883;class=Gene" target="_blank">smg-5</a></em></span> by RNAi (<a href="#bib45">Grishok et al., 2005</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec14"></a>14.&nbsp;WAGO 22G RNA-mediated transcriptional gene silencing: The nuclear RNAi pathway
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec14-1"></a>14.1.&nbsp;Early evidence for transcriptional gene silencing
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A complex of MES-2, MES-3 and MES-6 is functionally analogous to the Polycomb-repressive complex 2 (PRC2), a complex that
                  catalyzes H3K27 trimethylation, a repressive chromatin mark. MES-4 generates H3K36me3 marks. MES-2, MES-3, MES-6, and MES-4
                  are required for germline silencing of high-copy arrays (<a href="#bib61">Kelly and Fire, 1998</a>), and silenced arrays carry histone modifications consistent with a heterochromatinized state (<a href="#bib62">Kelly et al., 2002</a>). A screen for factors involved in cosuppression, silencing in <span class="emphasis"><em>trans</em></span> of an endogenous locus in response to a cognate transgenic array, identified numerous factors that regulate chromatin structure
                  and transcription (<a href="#bib105">Robert et al., 2005</a>). In this system, silencing occurs at the transcriptional level, as relieving cosuppression through loss of MUT-16 or MUT-8/RDE-2
                  results in proportional increases in levels of both spliced and unspliced transcripts (<a href="#bib105">Robert et al., 2005</a>). Another transgene was found to show transcriptional silencing upon dsRNA feeding that occurs through a mechanism that requires
                  both RNAi factors and proteins involved in chromatin regulation (<a href="#bib45">Grishok et al., 2005</a>). Transcriptional silencing of this transgene correlates with decreased RNA polymerase (Pol) II occupancy as well as increased
                  H4 histone acetylation. Furthermore, this transgene shows spontaneous silencing upon loss of RRF-3 (<a href="#bib45">Grishok et al., 2005</a>), suggesting convergence of endo- and exo-RNAi upon a common a transcriptional silencing pathway.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec14-2"></a>14.2.&nbsp;Early evidence for heritable and multigenerational gene silencing
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The effects of dsRNA injection are often observed in progeny, but rarely heritable beyond the first generation (<a href="#bib35">Fire et al., 1998</a>; <a href="#bib85">Montgomery et al., 1998</a>), suggesting the existence of an epigenetic heritable interfering agent of limited longevity. However, hermaphrodites injected
                  with dsRNA targeting transcripts expressed in the maternal germline show multigenerational inheritance of silencing mediated
                  by a dominant factor that can be inherited through either sperm or oocyte (<a href="#bib46">Grishok et al., 2000</a>). Similarly, a germline-expressed transgene transcriptionally silenced by cosuppression takes several generations to recover
                  expression after relief of cosuppression (<a href="#bib105">Robert et al., 2005</a>). In contrast, transcriptional silencing of a transgene expressed in the soma is not inherited (<a href="#bib45">Grishok et al., 2005</a>). These data suggest the existence of two different forms of inherited RNAi: single-generation inheritance of somatic RNAi
                  and multigenerational inheritance of germline RNAi.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec14-3"></a>14.3.&nbsp;NRDE-3 and HRDE-1 WAGO Argonautes mediate nuclear RNAi
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Two WAGO Argonautes, WAGO-12/NRDE-3 and WAGO-9/HRDE-1, mediate somatic and germline nuclear RNAi, respectively, through what
                  is referred to as the Nrde (<span class="underline">N</span>uclear <span class="underline">R</span>NAi-<span class="underline">de</span>fective) pathway (<a href="#figure3">Figure 3</a>). Although both NRDE-3 and HRDE-1 were reported to be completely dispensable for RNAi-mediated knockdown of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004078;class=Gene" target="_blank">pos-1</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002280;class=Gene" target="_blank">let-2</a></em></span> (<a href="#bib145">Yigit et al., 2006</a>), these Branch 3 WAGO Argonautes appear to be the sole mediators of transcriptional gene silencing. As WAGO Argonautes, neither
                  has an intact catalytic triad conferring slicer activity (<a href="#bib145">Yigit et al., 2006</a>), but each contains a bipartite nuclear localization signal (NLS) and shows predominantly nuclear localization (<a href="#bib51">Guang et al., 2008</a>; <a href="#bib4">Ashe et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>). NRDE-3 and HRDE-1 are triggered by WAGO 22G RNA binding to enter the nucleus and associate with nascent pre-mRNA targets,
                  where they recruit NRDE-2, NRDE-1, and NRDE-4 to inhibit Pol II elongation and deposit the repressive H3K9me3 chromatin mark
                  (<a href="#bib51">Guang et al., 2008</a>; <a href="#bib50">Guang et al., 2010</a>; <a href="#bib17">Burkhart et al., 2011</a>; <a href="#bib18">Burton et al., 2011</a>; <a href="#bib4">Ashe et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib47">Gu, S.G., et al. 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>).  The two Argonautes appear to use common silencing mechanisms and machinery (<a href="#bib17">Burkhart et al., 2011</a>; <a href="#bib4">Ashe et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>), but their protein expression patterns are distinct: a rescuing GFP::NRDE-3 translational fusion protein shows expression
                  in most somatic cells after the ~80-cell embryo stage (<a href="#bib51">Guang et al., 2008</a>), whereas GFP::HRDE-1 is expressed in male and female germ cells (<a href="#bib16">Buckley et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). Consequently, NRDE-3 mediates somatic nuclear RNAi, and HRDE-1 mediates germline nuclear RNAi.
               </p>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="endoRNAipathwys_fig3.jpg"><img src="endoRNAipathwys_fig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 3. Nuclear RNAi pathways required for trans-generational silencing.&nbsp;</b>Two pathways, silencing somatic targets and germline targets, converge on the same NRDE proteins but require distinct nuclear
                        Argonaute proteins: cytoplasmic secondary 22G siRNAs associate with NRDE-3 in the soma and with HRDE-1 in the germline. The
                        Argonaute:siRNA complex translocates to the nucleus where it recruits NRDE-2 and NRDE-1 to the nascent pre-mRNA. NRDE-1 also
                        associates with chromatin in a NRDE-4-dependent manner. NRDE-1 and NRDE-4 promote histone H3K9 trimethylation by histone methyltransferases
                        and thus inhibit transcription. In addition, the NRDE proteins inhibit RNA polymerase II. Nuclear RNAi is heritable across
                        generations: silencing signals, most likely siRNAs generated in the germline of the parent, are inherited and direct H3K9
                        trimethylation in the offspring. 
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec14-4"></a>14.4.&nbsp;NRDE-3 collaborates with other WAGO Argonautes to silence somatic targets
                        </h3>
                     </div>
                  </div>
               </div>
               <p>NRDE-3-mediated silencing is not always a significant component of the silencing response triggered by exo-RNAi or primary
                  endogenous small RNAs (<a href="#bib51">Guang et al., 2008</a>; <a href="#bib50">Guang et al., 2010</a>). While NRDE-3 likely does not play an important role in silencing of cytoplasmic mRNAs (<a href="#bib51">Guang et al., 2008</a>), it is required for exo-RNAi of somatic nuclear-localized mRNAs including polycistronic pre-mRNAs such as <span class="emphasis"><em>lin-15a-lin15b</em></span> and <span class="emphasis"><em>lir-1-lin-26</em></span> (<a href="#bib51">Guang et al., 2008</a>). Loss of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019862;class=Gene" target="_blank">nrde-3</a></em></span> also results in increased pre-mRNA and mRNA levels of select endogenous targets (<a href="#bib51">Guang et al., 2008</a>); however, a majority of NRDE-3 targets show far less upregulation in the absence of NRDE-3 than in the absence of RRF-1/EGO-1,
                  DRH-3, or EKL-1 (<a href="#bib49">Gu et al., 2009</a>). This indicates that other somatic Argonautes such as SAGO-1 and SAGO-2 (<a href="#bib145">Yigit et al., 2006</a>) collaborate with NRDE-3 to mediate WAGO 22G-RNA target silencing, potentially through the RDE-10/RDE-11 complex (<a href="#bib144">Yang et al., 2012</a>; <a href="#bib147">Zhang et al., 2012</a>). Accordingly, loss of NRDE-3 abrogates somatic nuclear RNAi entirely, but does not result in a classical somatic <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotype (<a href="#bib51">Guang et al., 2008</a>). NRDE-3 is also required for single-generation inheritance of somatic target silencing triggered by exo-RNAi; in progeny,
                  NRDE-3 promotes continued accumulation of secondary WAGO 22G RNAs to reestablish H3K9me3 marks (see <a href="#sec18" title="18.&nbsp;NRDE-3 directs single-generation epigenetic inheritance of silencing">Section 18</a>) (<a href="#bib18">Burton et al., 2011</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec14-5"></a>14.5.&nbsp;HRDE-1 transgenerationally silences diverse targets to promote germline immortality
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In contrast to NRDE-3, HRDE-1 engages the Nrde pathway in germ cells to direct silencing that can be inherited over many generations
                  (also called RNAe for <span class="underline">RNA</span>-induced <span class="underline">e</span>pigenetic silencing) (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib47">Gu, S.G., et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). This transgenerational silencing can be initiated by exo-RNAi to establish transcript knockdown that is stable for several
                  generations without additional trigger exposure (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib47">Gu, S.G., et al., 2012</a>). Endogenous small RNA pathways also robustly engage germline nuclear RNAi. 22G RNAs bound by HRDE-1 and WAGO-1 largely overlap
                  (<a href="#bib114">Shirayama et al., 2012</a>), affirming that nuclear and cytoplasmic WAGO Argonautes share a common set of siRNA cofactors and targets. These include
                  WAGO 22G RNAs triggered by 21U RNAs and 26G RNAs that maintain germline integrity. In particular, 21U RNAs encode an epigenetic
                  memory of non-self critical for genome surveillance (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). These fertility-promoting small RNAs associate with HRDE-1 to initiate transgenerational silencing of targets that pose
                  a threat to germline integrity; accordingly, loss of HRDE-1 or downstream Nrde factors causes progressive target desilencing
                  and emergence of pleiotropic germline defects including a <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype, production of nonfunctional gametes, and failure to generate mature oocytes or sperm in late generations at 25&deg;C
                  (<a href="#bib16">Buckley et al., 2012</a>). Effectively, loss of HRDE-1 results in a <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000966;class=Phenotype" target="_blank">Mrt</a>, or <span class="underline">m</span>o<span class="underline">rt</span>al germline, phenotype. While this phenotype is shared by mutants of downstream Nrde pathway members (NRDE-2, NRDE-1, and
                  NRDE-4), it is not observed with loss of the somatic nuclear Argonaute NRDE-3 (<a href="#bib16">Buckley et al., 2012</a>). After engaging HRDE-1 to initiate transgenerational silencing, the triggering primary small RNA may not be required, as
                  suggested by the dispensability of PRG-1 for maintenance of transgene silencing initiated by 21U RNAs (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). For silencing of some targets, however, HRDE-1 and the Nrde pathway collaborate with other, likely cytoplasmic WAGO Argonautes:
                  some transgenes are incompletely silenced by HRDE-1 and show increased desilencing upon loss of additional Argonautes including
                  WAGO-10 (<a href="#bib114">Shirayama et al., 2012</a>). Additional requirements for transgenerational silencing are discussed further below.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec15"></a>15.&nbsp;NRDE-3 target recognition
                     </h2>
                  </div>
               </div>
            </div>
            <p>Like other WAGO Argonautes, NRDE-3 binds WAGO 22G RNAs. Immuno-purified NRDE-3 complexes contain primarily 5&#8217; triphosphorylated
               ~22 nt endo-siRNAs (<a href="#bib51">Guang et al., 2008</a>). Consistent with production by a cytoplasmic RdRP acting on spliced transcripts, NRDE-3-associated siRNAs map to exonic
               sequences, even when RNAi is triggered by dsRNA containing introns (<a href="#bib51">Guang et al., 2008</a>). NRDE-3 primarily binds somatic 22G RNAs triggered by ERGO-1 class 26G RNAs, such as <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=E01G4.5;class=Gene" target="_blank">E01G4.5</a></em></span> 22G RNAs, but also binds secondary siRNAs during exo-RNAi targeting a somatic transcript (<a href="#bib51">Guang et al., 2008</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib50">Guang et al., 2010</a>). Due to redundancy among the WAGO Argonautes, loss of NRDE-3 does not significantly impair stability of these 22G RNAs (<a href="#bib51">Guang et al., 2008</a>).
            </p>
            <p>Upon binding a cytoplasmic WAGO 22G RNA, NRDE-3 translocates to the nucleus to execute target silencing (<a href="#bib51">Guang et al., 2008</a>). Nuclear localization is required for NRDE-3 function: an NLS-defective NRDE-3 mutant protein that binds siRNAs at wild
               type levels but does not redistribute to the nucleus fails to rescue <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019862;class=Gene" target="_blank">nrde-3</a></em></span> mutant phenotypes (<a href="#bib51">Guang et al., 2008</a>). siRNA binding is required for translocation to the nucleus, as mutating the NRDE-3 PAZ domain to abrogate siRNA binding
               also results in failure to localize to the nucleus (<a href="#bib51">Guang et al., 2008</a>). Similarly, loss of siRNA cofactors abolishes nuclear localization. As NRDE-3 binds primarily somatic 22G RNAs triggered
               by ERGO-1 class 26G RNAs, GFP::NRDE-3 is cytoplasmic in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019971;class=Gene" target="_blank">ergo-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003504;class=Gene" target="_blank">mut-7</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004326;class=Gene" target="_blank">rde-4</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003499;class=Gene" target="_blank">mut-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00016143;class=Gene" target="_blank">eri-9</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00016561;class=Gene" target="_blank">eri-6</a></em></span>, and <span class="emphasis"><em>dcr-1(<a href="http://www.wormbase.org/db/get?name=WBVar00088953;class=Variation" target="_blank">mg375</a>)</em></span> helicase mutants (<a href="#bib51">Guang et al., 2008</a>; <a href="#bib96">Pavelec et al., 2009</a>; <a href="#bib38">Fischer et al., 2011</a>). The nuclear localization of GFP::NRDE-3 is rescued upon introduction of dsRNA in an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> mutant, but not an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004326;class=Gene" target="_blank">rde-4</a></em></span> mutant, which cannot initiate exo-RNAi (<a href="#bib51">Guang et al., 2008</a>); this suggests that siRNA binding is necessary and sufficient for nuclear redistribution of nuclear Argonautes.
            </p>
            <p>After translocation to the nucleus, the loaded WAGO 22G RNA directs NRDE-3 to associate with unspliced target pre-mRNA (<a href="#bib51">Guang et al., 2008</a>). NRDE-3 targeting recognition appears to require very high sequence complementarity (<a href="#bib87">Montgomery et al., 2012</a>): A reporter transgene with a single target site for an X-cluster 22G RNA is efficiently silenced in a wild type background
               and shows primarily NRDE-3-dependent silencing. Silencing is abrogated by mismatches at positions 1-3, 4-5, 12-14, or deletion
               of a nucleotide at position 4. Silencing is impaired by mismatches at positions 9-11 or deletion or insertion of a nucleotide
               at position 13. Mismatches at position 13 or 20-22 are tolerated.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec16"></a>16.&nbsp;The downstream nuclear RNAi machinery
                     </h2>
                  </div>
               </div>
            </div>
            <p>The proteins NRDE-2, NRDE-1, and NRDE-4 mediate the transcription silencing effects of nuclear Argonautes. After NRDE-3 binds
               a WAGO 22G RNA, translocates to the nucleus, and recognizes a nascent pre-mRNA target, NRDE-3 associates with NRDE-2 to recruit
               NRDE-1 to the transcript and direct its deposition on chromatin through a mechanism that requires NRDE-4 (<a href="#bib51">Guang et al., 2008</a>; <a href="#bib50">Guang et al., 2010</a>; <a href="#bib17">Burkhart et al., 2011</a>). NRDE-2, NRDE-1, and NRDE-4 are also engaged by HRDE-1 to mediate nuclear RNAi in the germline (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>). Accordingly, loss of any of these factors results in the same <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000966;class=Phenotype" target="_blank">Mrt</a> phenotype observed with loss of HRDE-1 (<a href="#bib16">Buckley et al., 2012</a>), as well as loss of sensitivity to nuclear exo-RNAi (<a href="#bib50">Guang et al., 2010</a>; <a href="#bib17">Burkhart et al., 2011</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec16-1"></a>16.1.&nbsp;NRDE-2
                        </h3>
                     </div>
                  </div>
               </div>
               <p>NRDE-2 is a predominantly nuclear-localized, evolutionarily conserved protein encoding a domain of unknown function, a Serine/Arginine-rich
                  domain, and a HAT-like domain, a domain consisting of repeats of three aromatic residues that is found in several RNA processing
                  proteins (<a href="#bib50">Guang et al., 2010</a>). Immuno-purified NRDE-2 complexes contain NRDE-3, but only when the NRDE-3 NLS is intact, indicating an exclusively nuclear
                  interaction (<a href="#bib50">Guang et al., 2010</a>). NRDE-2 is dispensable for NRDE-3 to transport siRNAs from cytoplasm to nucleus and for NRDE-3 to associate with the target
                  pre-mRNA (<a href="#bib50">Guang et al., 2010</a>). Within the nucleus, NRDE-3 recruits NRDE-2 to the targeted nascent transcript. It is interesting to note that siRNA counts
                  appear to be elevated two-fold by loss of NRDE-2 (<a href="#bib47">Gu, S.G., et al., 2012</a>); possibly, failure to silence target transcripts in the nucleus via the Nrde pathway results in increased cytoplasmic transcript
                  accumulation, enhancing WAGO 22G RNAs production by RdRPs.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec16-2"></a>16.2.&nbsp;NRDE-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>NRDE-1 is a nematode-specific, nuclear-localized protein containing no obvious protein domains (<a href="#bib17">Burkhart et al., 2011</a>). NRDE-1 is recruited by NRDE-2/NRDE-3 to pre-mRNA near the target site, where it is required for nuclear RNAi-mediated inhibition
                  of transcription elongation; a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00007577;class=Gene" target="_blank">nrde-1</a></em></span> mutant fails to show 5&#8217; transcription inhibition of <span class="emphasis"><em>lin-15b</em></span> upon RNAi and survives RNAi targeting of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003044;class=Gene" target="_blank">lir-1</a></em></span> polycistron in a sensitizing <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> mutant background (<a href="#bib17">Burkhart et al., 2011</a>). NRDE-1 also associates with chromatin at the targeted genomic locus in a NRDE-4-dependent manner and links nuclear RNAi
                  to chromatin regulation by promoting H3K9 trimethylation at genomic sites targeted by WAGO 22G RNAs.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec16-3"></a>16.3.&nbsp;NRDE-4
                        </h3>
                     </div>
                  </div>
               </div>
               <p>NRDE-4 is a nematode-specific protein with a predicted bipartite NLS and no other obvious domains (<a href="#bib17">Burkhart et al., 2011</a>). NRDE-4 does not recruit NRDE-3, NRDE-2, or NRDE-1 to pre-mRNA but is required for recruitment of NRDE-1 to chromatin and
                  therefore for transcriptional inhibition and H3K9 trimethylation (<a href="#bib17">Burkhart et al., 2011</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec17"></a>17.&nbsp;Nuclear RNAi inhibits transcriptional elongation and drives H3K9 trimethylation
                     </h2>
                  </div>
               </div>
            </div>
            <p>Nuclear RNAi terminates transcription and promotes establishment of a repressed chromatin state at genomic loci targeted by
               exo-RNAi as well as multiple loci targeted by endo-siRNAs and 21U RNAs (<a href="#bib17">Burkhart et al., 2011</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib47">Gu, S.G., et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). Nuclear RNAi requires interaction with target pre-mRNA, as targeting 5&#8217; to the 5&#8217; UTR does not result in silencing (<a href="#bib47">Gu, S.G., et al., 2012</a>). NRDE-3, NRDE-2, and NRDE-1 bind to pre-mRNA at the target site but are not detected in association with pre-mRNA 5&#8217; to
               the target site, presumably due to co-transcriptional splicing of nascent transcripts (<a href="#bib88">Moore and Proudfoot, 2009</a>; <a href="#bib50">Guang et al., 2010</a>; <a href="#bib17">Burkhart et al., 2011</a>). Nrde targeting inhibits transcription elongation, as Pol II occupancy increases near the target site but not at initiation
               sites (<a href="#bib50">Guang et al., 2010</a>). Nuclear run-on assays show Pol II inhibition occurring ~2 kb downstream of the target site, correlating with decreased
               Pol II occupancy by chromatin immunoprecipitation 3&#8217; to the target site (<a href="#bib50">Guang et al., 2010</a>; <a href="#bib17">Burkhart et al., 2011</a>). Because transcription is inhibited, NRDE-3, NRDE-2, and NRDE-1 do not bind pre-mRNA sequences encoded 3&#8217; to the target
               site; however, Nrde factors appear to remain associated with pre-mRNA fragments after silencing (<a href="#bib50">Guang et al., 2010</a>; <a href="#bib17">Burkhart et al., 2011</a>).
            </p>
            <p>Nrde pathway targeting results in enrichment of H3K9 trimethylation at genomic target sites that correlates with the transcription
               inhibition (<a href="#bib17">Burkhart et al., 2011</a>), consistent with the established repressive role of H3K9 methylation. Whereas NRDE-3 and NRDE-2 recruit NRDE-1 to the RNA
               target near or 5&#8217; to the target site, NRDE-1 requires NRDE-4 for chromatin interaction and associates with chromatin predominantly
               3&#8217; to the site of RNAi (<a href="#bib17">Burkhart et al., 2011</a>). Accordingly, H3K9 trimethylation occurs across the gene but peaks 3&#8217; to the target site (<a href="#bib17">Burkhart et al., 2011</a>). The chromatin response to nuclear RNAi occurs at the subgenic level, as shifting the RNAi trigger site shifts the interval
               of H3K9me3 deposition (<a href="#bib47">Gu, S.G., et al., 2012</a>). During exo-RNAi-triggered nuclear RNAi, secondary siRNAs accumulate mainly at exonic regions proximal to the dsRNA trigger
               site, but both exons and introns show high H3K9me3 levels after targeting (<a href="#bib47">Gu, S.G., et al., 2012</a>).
            </p>
            <p>Although many of the studies of NRDE-3 function in nuclear RNAi were conducted under exo-RNAi, NRDE-3-mediated nuclear RNAi
               is also important for transcriptional silencing triggered by a subset of endogenous small RNAs in the soma (<a href="#bib17">Burkhart et al., 2011</a>). As mentioned above (see <a href="#sec15" title="15.&nbsp;NRDE-3 target recognition">Section 15</a>), NRDE-3 binds somatic WAGO 22G RNAs, many of which are triggered by ERGO-1 class 26G RNAs (<a href="#bib51">Guang et al., 2008</a>; <a href="#bib42">Gent et al., 2010</a>; <a href="#bib50">Guang et al., 2010</a>). 22G RNAs targeting the ERGO-1 target <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=E01G4.5;class=Gene" target="_blank">E01G4.5</a></em></span> are abundantly bound by NRDE-3 (<a href="#bib51">Guang et al., 2008</a>) and direct chromatin modification at this locus through the somatic nuclear RNAi pathway (<a href="#bib17">Burkhart et al., 2011</a>). Loss of members of the ERI complex, NRDE-3, or downstream Nrde pathway factors results in a subtle but detectable increase
               in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=E01G4.5;class=Gene" target="_blank">E01G4.5</a></em></span> transcription inhibition, increased transcript levels, and depletion of H3K9me3 marks at the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=E01G4.5;class=Gene" target="_blank">E01G4.5</a></em></span> locus (<a href="#bib17">Burkhart et al., 2011</a>). The strong influence of NRDE-3 on H3K9me3 deposition at the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=E01G4.5;class=Gene" target="_blank">E01G4.5</a></em></span> locus is uncommon, and levels of many ERGO-1 targets show little or no effect of loss of the somatic Nrde pathway (<a href="#bib17">Burkhart et al., 2011</a>), consistent with significant overlap between cytoplasmic and nuclear WAGO Argonautes in silencing of somatic targets (<a href="#bib49">Gu et al., 2009</a>). Nonetheless, in the absence of ERGO-1 class 26G RNAs, the somatic Nrde pathway can be heavily engaged by exo-RNAi, as indicated
               by the emergence of RNAi phenotypes for polycistronic transcripts in an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> mutant (<a href="#bib15">Bosher et al., 1999</a>; <a href="#bib63">Kennedy et al., 2004</a>). In fact, exo-RNAi of some monocistronic transcripts also engages NRDE-3 in the absence of ERGO-1 class 26G RNAs, as exo-RNAi
               targeting <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006805;class=Gene" target="_blank">unc-73</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001074;class=Gene" target="_blank">dpy-13</a></em></span> results in less severe phenotypes in an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a>; <a href="http://www.wormbase.org/db/get?name=WBGene00019862;class=Gene" target="_blank">nrde-3</a></em></span> double mutant than in an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> mutant (<a href="#bib51">Guang et al., 2008</a>). While loss of ERGO-1 class 26G RNAs does sensitize the somatic nuclear RNAi system for phenotypic analysis, an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> or equivalent mutant background is not necessary for the transcription- and chromatin-level effects of nuclear RNAi (<a href="#bib17">Burkhart et al., 2011</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec18"></a>18.&nbsp;NRDE-3 directs single-generation epigenetic inheritance of silencing
                     </h2>
                  </div>
               </div>
            </div>
            <p>Although NRDE-3 acts primarily in somatic cells, the silencing effects of somatic nuclear RNAi are transmissible for one generation
               (<a href="#bib18">Burton et al., 2011</a>). Exo-RNAi in the parental generation generates secondary siRNAs abundantly bound by NRDE-3; inheriting progeny also show
               high levels of NRDE-3-bound secondary siRNAs from embryo through adulthood, although these are not inherited into F2 embryos
               (<a href="#bib18">Burton et al., 2011</a>). In the absence of NRDE-1 or NRDE-4, these secondary siRNAs are still inherited and associate with NRDE-3, but levels of
               NRDE-3-bound secondary siRNAs are not maintained throughout development (<a href="#bib18">Burton et al., 2011</a>). This indicates that somatic nuclear RNAi is not required for inheritance of secondary siRNAs but is required for continued
               expression of these siRNAs in progeny.
            </p>
            <p>H3K9me3 marks established in the parental generation at RNAi target loci are lost in progeny during the embryo stage, possibly
               due to the absence of NRDE-3 in early embryogenesis (<a href="#bib51">Guang et al., 2008</a>; <a href="#bib18">Burton et al., 2011</a>). Silencing is then reestablished through NRDE-3 and downstream Nrde factors during development and strengthened to exceed
               the H3K9me3 levels in the parental generation (<a href="#bib18">Burton et al., 2011</a>). The mechanism is likely the same in progeny as in the parental generation as NRDE-3 siRNA binding and nuclear translocation
               are still required for inherited silencing (<a href="#bib18">Burton et al., 2011</a>). However, the increased H3K9me3 deposition in progeny suggests that the inheritance of a silencing signal may further promote
               the ability of nuclear RNAi to repress chromatin, possibly by enhancing coupling of siRNA generation and H3K9me3 deposition
               in progeny. Importantly, this reestablishment of silencing requires the activity of NRDE-3 in progeny: homozygous <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019862;class=Gene" target="_blank">nrde-3</a></em></span> mutant progeny of heterozygous <a href="http://www.wormbase.org/db/get?name=WBGene00019862;class=Gene" target="_blank">nrde-3</a> hermaphrodites subjected to exo-RNAi do not show inheritance of silencing (<a href="#bib18">Burton et al., 2011</a>). The converse has not been tested; it is not known whether somatic nuclear RNAi is required in the parental generation for
               RNAi inheritance and chromatin repression in progeny. Furthermore, it is as yet unclear whether somatic nuclear RNAi is required
               for establishment or maintenance of transcriptional silencing during RNAi inheritance in the soma.
            </p>
            <p>Somatic nuclear RNAi in the parental generation directs siRNA expression and H3K9 trimethylation in inheriting progeny through
               one of two possible mechanisms. NRDE-3-bound secondary siRNAs may be directly deposited into germ cells to direct embryonic
               silencing in association with other WAGO Argonautes, then re-bind NRDE-3 to promote maintenance of siRNA expression through
               nuclear RNAi during development. Alternatively, a different silencing signal such as primary small RNAs may be deposited in
               germ cells to promote expression of siRNAs in progeny that bind NRDE-3 and re-engage nuclear RNAi (<a href="#bib18">Burton et al., 2011</a>). In support of the latter mechanism, NRDE-3-bound WAGO 22G RNAs do not appear to initiate their own amplification at target
               sites (<a href="#bib87">Montgomery et al., 2012</a>), suggesting that primary small RNA trigger may be required for secondary siRNA re-amplification in progeny. One likely possibility
               is that primary exo-siRNAs are directly inherited from hermaphrodites exposed to dsRNA. Indeed, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004323;class=Gene" target="_blank">rde-1</a></em></span> transcript is abundant in hermaphrodite germline and detected at high levels in embryos of all stages (NEXTDB clone 26b3),
               suggesting that primary exo-siRNAs may be inherited bound to RDE-1 to trigger NRDE-3-mediated nuclear RNAi in inheriting progeny.
               Such a mechanism would explain why somatic nuclear RNAi is heritable for only a single generation.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec19"></a>19.&nbsp;HRDE-1 directs multigenerational epigenetic inheritance of silencing
                     </h2>
                  </div>
               </div>
            </div>
            <p>Like NRDE-3, HRDE-1 engages the Nrde downstream machinery to direct transcriptional silencing and H3K9me3 deposition at loci
               targeted by exo-RNAi and endogenous small RNAs; however, HRDE-1 binds germline WAGO 22G RNAs and engages the downstream Nrde
               pathway to direct transgenerational silencing of germline transcripts (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib47">Gu, S.G., et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). Loss of HRDE-1, NRDE-2, NRDE-1, or NRDE-4 results in the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000966;class=Phenotype" target="_blank">Mrt</a> phenotype described above (<a href="#bib16">Buckley et al., 2012</a>), likely due to desilencing of germline targets such as transposons that pose a threat to genome integrity when active. Further
               testing of mutants for various upstream and downstream 21U RNA, 26G RNA, and WAGO 22G RNA pathway factors is necessary to
               determine the respective contributions of these pathways to promoting germline immortality. Curiously, the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000966;class=Phenotype" target="_blank">Mrt</a> phenotype of <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00007624?query=hrde-1#0-9e-3" target="_blank">hrde-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00011333;class=Gene" target="_blank">nrde-2</a></em></span> mutants manifests only at 25&deg;C, whereas <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00007577;class=Gene" target="_blank">nrde-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001584;class=Gene" target="_blank">nrde-4</a></em></span> mutants are <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000966;class=Phenotype" target="_blank">Mrt</a> at 25&deg;C and 20&deg;C (<a href="#bib16">Buckley et al., 2012</a>), suggesting the chromatin machinery of the Nrde pathway may have additional roles in germline maintenance.
            </p>
            <p>Multigenerational inheritance of transcriptional silencing requires germline expression of the targeted transcript (<a href="#bib45">Grishok et al., 2005</a>). Most likely, an mRNA template must be present in the germline for RdRP to generate siRNAs capable of propagating silencing
               across each generation; this may account for the limited perdurance of NRDE-3-mediated somatic nuclear RNAi (<a href="#bib18">Burton et al., 2011</a>). In contrast, HRDE-1-mediated nuclear RNAi in the germline results in continued siRNA production and silencing over multiple
               generations that maintain H3K9 trimethylation (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib47">Gu, S.G., et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). Accordingly, factors such as MUT-7 and RDE-3 that are required for 22G RNA accumulation are also required for maintenance
               of transgenerational silencing (<a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). Screens using transgenerationally silenced exo-RNAi and 21U RNA sensor transgenes have identified a set of chromatin factors
               necessary for maintenance of transgenerational RNAi: the chromo domain protein HPL-2 (semiredundantly with HPL-1), Polycomb
               complex protein MES-3, Trithorax complex protein MES-4, SET domain SET-32, and putative H3K9 methyltransferase SET-25 (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). These proteins may be recruited by NRDE-1 to mediate H3K9 methylation or otherwise promote a chromatin state favorable
               for continued siRNA biogenesis to maintain nuclear RNAi.
            </p>
            <p>In the parental generation exposed to dsRNA, HRDE-1 is dispensable for germline silencing; however, HRDE-1 is required in
               the F1 generation for inheritance of parental germline silencing, in the F2 generation for inheritance from the F1, and so
               on, as indicated by a failure of homozygous <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00007624?query=hrde-1#0-9e-3" target="_blank">hrde-1</a></em></span> mutant offspring to inherit germline transgene silencing from a <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00007624?query=hrde-1#0-9e-3" target="_blank">hrde-1</a></em></span> heterozygous hermaphrodite (<a href="#bib16">Buckley et al., 2012</a>). This result differs somewhat from another study, which shows that loss of HRDE-1 or MUT-7 reactivates a silenced locus
               only in the second generation of <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00007624?query=hrde-1#0-9e-3" target="_blank">hrde-1</a></em></span> homozygosity (<a href="#bib78">Luteijn et al., 2012</a>), suggesting that epigenetic silencing may be established in the parental germline.
            </p>
            <p>Surprisingly, the germline nuclear RNAi pathway appears to require cytoplasmic WAGO Argonautes for initiation: during exo-RNAi
               targeting the oogenesis-enriched <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004879;class=Gene" target="_blank">smg-1</a></em></span> transcript (<a href="#bib104">Reinke et al., 2004</a>), the target site fails to become enriched for H3K9me3 marks in the MAGO mutant strain (<a href="#bib47">Gu, S.G., et al., 2012</a>). This definitively indicates that the nuclear and cytoplasmic WAGO Argonautes do not compete for siRNA cofactors. Rather,
               cytoplasmic WAGO Argonautes promote nuclear RNAi through an unknown mechanism. It is not yet established whether WAGO Argonautes
               bind 22G RNAs irreversibly or reversibly; if the latter, perhaps HRDE-1 appropriates WAGO 22G RNAs from the cytoplasmic Argonautes
               for nuclear silencing. NRDE-3 does not appear to be required for establishment or maintenance of transgenerational silencing
               (<a href="#bib16">Buckley et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>), nor is a sensitizing mutation in endo-RNAi factors such as ERI-1 or RRF-3 required (<a href="#bib47">Gu, S.G., et al., 2012</a>). Other WAGO Argonautes do, however, contribute variably to HRDE-1 target silencing, as some 21U RNA sensor transgenes show
               dependence on additional Argonautes for complete silencing (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>).
            </p>
            <p>While establishment of transgenerational silencing requires factors involved in the formation of the initial trigger, such
               as RDE-1 in the case of primary exo-siRNAs (<a href="#bib16">Buckley et al., 2012</a>; <a href="#bib47">Gu, S.G., et al., 2012</a>) or PRG-1 in the case of 21U RNAs (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>), maintenance of transgenerational silencing does not (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib16">Buckley et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). Once established, HRDE-1-mediated nuclear silencing is stable for multiple generations and can be inherited from either
               the male or female germline as a dominant signal, although silencing may show a generational lag when inherited from the male
               (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). This silencing in <span class="emphasis"><em>trans</em></span> is mediated by a diffusible agent, namely, secondary 22G RNAs, capable of repressing homologous DNA regardless of whether
               the primary small RNA target site is present (<a href="#bib78">Luteijn et al., 2012</a>). These 22G RNAs map further upstream of the initiating 21U RNA target site than observed during early, PRG-1-dependent silencing
               (<a href="#bib8">Bagijn et al., 2012</a>), suggesting that transgenerational RNAi may entail additional spreading during maintenance (<a href="#bib78">Luteijn et al., 2012</a>). 22G RNAs whose expression is stimulated at transgenerationally silenced 21U RNA targets to maintain silencing may not correspond
               precisely to the species triggered by 21U RNA targeting, as the latter appear to be PRG-1-dependent and are lost when transgenerationally
               silenced loci are introduced into a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004178;class=Gene" target="_blank">prg-1</a></em></span> mutant background (<a href="#bib4">Ashe et al., 2012</a>).
            </p>
            <p>After exposure to dsRNA, secondary siRNA accumulation and association with HRDE-1 decrease progressively across generations
               (<a href="#bib16">Buckley et al., 2012</a>). Transgenerational silencing initiated by endogenous small RNAs shows a similar effect, as endogenous targets of HRDE-1
               silencing show progressive loss of H3K9me3 enrichment and increased expression over successive generations in a <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00007624?query=hrde-1#0-9e-3" target="_blank">hrde-1</a></em></span> mutant background (<a href="#bib16">Buckley et al., 2012</a>). This suggests that endogenous primary small RNA triggers may be required for periodic reinforcement of established transgenerational
               silencing. It is intriguing to speculate what function the nuclear RNAi pathways serve. While nuclear RNAi may simply provide
               a heritable signal to direct silencing of aberrant and potentially dangerous transcripts, it has also been suggested that
               targeting of these endogenous loci by nuclear RNAi may also fulfill a larger function in regulating chromatin dynamics.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec20"></a>20.&nbsp;Features and targets of CSR-1 22G RNAs
                     </h2>
                  </div>
               </div>
            </div>
            <p>The 22G RNAs that are not bound by WAGO Argonautes are the CSR-1 22G RNAs (<a href="#figure4">Figure 4</a>). Named for their sole Argonaute effector, CSR-1, these 22G RNAs do not induce target silencing but rather associate with
               chromatin to promote proper organization or assembly of targets within the holocentric chromosomes of <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib22">Claycomb et al., 2009</a>). CSR-1 22G RNAs are produced in the germline by the same DRH-3- and EKL-1-containing RdRP module as WAGO 22G RNAs, but require
               EGO-1 and not RRF-1 for accumulation (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). Although CSR-1 22G RNAs share the features and biogenesis machinery of the secondary siRNAs engaged by WAGO Argonautes,
               there is no evidence that CSR-1 22G RNAs are themselves secondary siRNAs.
            </p>
            <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="endoRNAipathwys_fig4.jpg"><img src="endoRNAipathwys_fig4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 4. The CSR-1 22G siRNA pathway.&nbsp;</b>In the germline P granules, the RdRP EGO-1, in complex with the helicase DRH-3 and the Tudor domain protein EKL-1, generates
                     22G siRNAs from over 4000 protein-coding, germline-expressed genes. These 22G siRNAs associate with the Argonaute CSR-1. CSR-1:siRNA
                     complexes do not silence their targets, but rather localize to chromosomes to ensure proper meiotic and mitotic chromosome
                     segregation and to protect against silencing. The poly(U)polymerase CDE-1 regulates the abundance of CSR-1 associated 22G
                     siRNAs by uridylation. CSR-1 is thought to protect its targets against PRG-1 recruitment.
                  </p>
               </div>
            </div>
            <p>CSR-1 22G RNAs largely target germline-expressed genes. Immuno-purified CSR-1 complexes are enriched for a set of 22G RNAs
               that are antisense to over 4,000 protein-coding genes, whereas microRNAs, 21U RNAs, and 22G RNAs targeting repetitive sequences,
               pseudogenes, and intergenic or unannotated loci are depleted (<a href="#bib22">Claycomb et al., 2009</a>). Similarly, comparison of deep sequencing libraries generated from <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> mutants (<a href="#bib22">Claycomb et al., 2009</a>) with <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene" target="_blank">ekl-1</a></em></span> mutants (<a href="#bib49">Gu et al., 2009</a>) reveals that CSR-1 and EGO-1 are required for accumulation of 22G RNAs antisense to protein-coding genes, many of which
               also require DRH-3 and EKL-1 (<a href="#bib22">Claycomb et al., 2009</a>). Most CSR-1 22G RNAs are depleted in the absence of germline (<a href="#bib49">Gu et al., 2009</a>), consistent with the germline-restricted expression pattern of EGO-1 (<a href="#bib119">Smardon et al., 2000</a>; <a href="#bib80">Maine et al., 2005</a>; <a href="#bib136">Vought et al., 2005</a>; <a href="#bib22">Claycomb et al., 2009</a>). Less than 1% of 22G RNAs recovered in immuno-purified CSR-1 complexes target repetitive elements and pseudogenes (<a href="#bib22">Claycomb et al., 2009</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec21"></a>21.&nbsp;The CSR-1 22G RNA pathway molecular machinery
                     </h2>
                  </div>
               </div>
            </div>
            <p>The 22G RNA RdRP module is discussed in detail above; the following sections focus on the features and functions of EGO-1,
               DRH-3, and EKL-1 only as they pertain to the CSR-1 22G RNA pathway. <span class="emphasis"><em>In situ</em></span> hybridization indicates that concentrations of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene" target="_blank">ekl-1</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> transcripts are highest in gonad and early embryo (<a href="#bib111">She et al., 2009</a>), in line with other transcript and protein localization data (<a href="#bib119">Smardon et al., 2000</a>; <a href="#bib104">Reinke et al., 2004</a>; <a href="#bib80">Maine et al., 2005</a>;  <a href="#bib136">Vought et al., 2005</a>; <a href="#bib89">Nakamura et al., 2007</a>; <a href="#bib22">Claycomb et al., 2009</a>). DRH-3 and EKL-1 are not germline-restricted, as they also function in WAGO 22G RNA biogenesis in germline and soma (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec21-1"></a>21.1.&nbsp;CSR-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The CSR-1 Argonaute clusters phylogenetically within the WAGO clade (<a href="#bib145">Yigit et al., 2006</a>), but does not bind WAGO 22G RNAs (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). Nevertheless, loss of CSR-1 results in partially defective germline and early zygotic transgene silencing (<a href="#bib145">Yigit et al., 2006</a>), although this may be due to defective localization of P granules in the absence of CSR-1 (<a href="#bib22">Claycomb et al., 2009</a>). CSR-1 contains an intact catalytic triad (<a href="#bib145">Yigit et al., 2006</a>). In a cell-free system, addition of single-stranded, 5&#8217;triphosphorylated 23 nt siRNA and target mRNA results in significant
                  target cleavage that is 90% depleted in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> null mutant adult lysate; substitution of 5&#8217; monophosphorylated siRNA decreased cleavage efficiency considerably (<a href="#bib3">Aoki et al., 2007</a>). Recombinant CSR-1 is also capable of catalyzing robust cleavage <span class="emphasis"><em>in vitro</em></span> when incubated with single-stranded 5&#8217; triphosphorylated, but not 5&#8217; monophosphorylated, siRNA and target (<a href="#bib3">Aoki et al., 2007</a>). Like the slicer activity of RISC in other organisms (<a href="#bib31">Elbashir et al., 2001</a>; <a href="#bib110">Schwarz et al., 2004</a>), CSR-1-mediated cleavage in cell lysate requires magnesium ions and is abolished by mutation of the two nucleotides flanking
                  the target cleavage site (<a href="#bib3">Aoki et al., 2007</a>). Although these <span class="emphasis"><em>in vitro</em></span> findings demonstrate the catalytic competence of CSR-1, no direct function for its cleavage activity has been demonstrated.
                  The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> gene encodes two isoforms. Both are expressed at all developmental stages, but <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> mRNA and protein are most enriched in young adult, gravid adult, and embryo (<a href="#bib22">Claycomb et al., 2009</a>). The larger CSR-1 protein isoform is expressed throughout larval development and is not germline restricted (<a href="#bib22">Claycomb et al., 2009</a>). CSR-1 does not require EGO-1, DRH-3, or EKL-1 for stability, nor do any of those proteins require CSR-1 or another member
                  of the RdRP module for stability (<a href="#bib22">Claycomb et al., 2009</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec22"></a>22.&nbsp;Phenotypes associated with CSR-1 22G RNA pathway compromise
                     </h2>
                  </div>
               </div>
            </div>
            <p>EGO-1, DRH-3, EKL-1, and CSR-1 represent the main components of the CSR-1 22G RNA pathway (<a href="#bib22">Claycomb et al., 2009</a>). Earlier phenotypic analyses and focused or genome-wide RNAi screens have identified an overlapping set of phenotypes associated
               with depletion of these proteins that include embryonic lethality, larval arrest, and sterility  (<a href="#bib100">Qiao et al., 1995</a>; <a href="#bib39">Fraser et al., 2000</a>; <a href="#bib119">Smardon et al., 2000</a>; <a href="#bib79">Maeda et al., 2001</a>; <a href="#bib151">Zipperlen et al., 2001</a>; <a href="#bib57">Kamath et al., 2003</a>; <a href="#bib117">Simmer et al., 2003</a>; <a href="#bib107">Rual et al., 2004</a>; <a href="#bib33">Fernandez et al., 2005</a>; <a href="#bib120">Sonnichsen et al, 2005</a>; <a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib76">Lehner et al., 2006</a>; <a href="#bib145">Yigit et al., 2006</a>). These four factors were first implicated in a common pathway by their shared <span class="underline">e</span>nhancement of <span class="emphasis"><em><span class="underline">k</span>sr-1</em></span> <span class="underline">l</span>ethality (Ekl) phenotype, which describes a failure to specify excretory duct cell fate in late embryogenesis (<a href="#bib106">Rocheleau et al., 2008</a>). Transcripts whose loss produces an Ekl phenotype are expressed in maternal germline to regulate embryonic gene expression
               (<a href="#bib106">Rocheleau et al., 2008</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec22-1"></a>22.1.&nbsp;Germline phenotypes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Mutation or RNAi-mediated depletion of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene" target="_blank">ekl-1</a></em></span>, or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> results in common mitotic and meiotic defects leading to sterility (<a href="#bib100">Qiao et al., 1995</a>; <a href="#bib117">Simmer et al., 2003</a>; <a href="#bib105">Robert et al., 2005</a>; <a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib145">Yigit et al., 2006</a>). In general, loss of these factors results in underproliferation of the germline, with nuclei of abnormal shape and size,
                  (<a href="#bib136">Vought et al., 2005</a>; <a href="#bib80">Maine et al., 2005</a>; <a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib111">She et al., 2009</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> mutants show moderately reduced germ cell number, enlargement of the transition zone with large, diffuse transition zone
                  nuclei, reduction of the pachytene zone, and overall delayed gametogenesis (<a href="#bib100">Qiao et al., 1995</a>; <a href="#bib119">Smardon et al., 2000</a>; <a href="#bib136">Vought et al., 2005</a>). The spermatogenesis-to-oogenesis transition is also delayed, resulting in production of excess sperm (<a href="#bib119">Smardon et al., 2000</a>). Sperm produced by an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> mutant are incapable of fertilization, and oocytes are small, nonfunctional, and show unpaired homologous chromosomes (<a href="#bib100">Qiao et al., 1995</a>; <a href="#bib119">Smardon et al., 2000</a>). Like <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> mutants, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene" target="_blank">ekl-1</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> mutant germlines show protracted transition zones that include some large nuclei with diffuse chromosome morphology (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib111">She et al., 2009</a>). These mutants also show aberrantly segregated chromosomes in diakinesis oocyte nuclei, suggesting defects in pairing, synapsis,
                  or recombination (<a href="#bib89">Nakamura et al., 2007</a>; <a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib111">She et al., 2009</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene" target="_blank">ekl-1</a></em></span> mutants produce sperm with abnormally large and variably sized nuclei, indicating impaired chromatin condensation or segregation;
                  a similar phenotype is observed in an <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a>; <a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> double mutant (<a href="#bib111">She et al., 2009</a>). These phenotypes suggest pervasive chromosome segregation defects.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec22-2"></a>22.2.&nbsp;Embryonic phenotypes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Loss of EGO-1, DRH-3, EKL-1, or CSR-1 results in embryonic lethality (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib145">Yigit et al., 2006</a>; <a href="#bib106">Rocheleau et al., 2008</a>). Fertilized <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> mutant oocytes produce eggshells, but arrest as balls of 20-50 cells without undergoing morphogenesis or gastrulation (<a href="#bib119">Smardon et al., 2000</a>). Similarly, RNAi-mediated depletion of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene" target="_blank">ekl-1</a></em></span>, or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> results in production of embryos that show normal prophase chromosome condensation, poor metaphase alignment, chromosomal
                  bridging at anaphase, and bisection of lagging chromosomes by the cleavage furrow at cytokinesis during every cell division;
                  these defects cause accumulation of abnormally shaped nuclei with non-wild-type chromosomal complements and result in embryonic
                  lethality due to arrest between gastrulation and early morphogenesis (<a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib145">Yigit et al., 2006</a>; <a href="#bib89">Nakamura et al., 2007</a>; <a href="#bib22">Claycomb et al., 2009</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> hypomorphic mutants and a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> mutant partially rescued by transgenic CSR-1 expression both produce dead embryos arrested at various points in embryogenesis
                  that contain many nuclei with abnormal DNA content (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). A <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype is also observed in these mutants, confirming impairment of chromosome segregation (<a href="#bib22">Claycomb et al., 2009</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec23"></a>23.&nbsp;Localization and assembly of the CSR-1 22G RNA complex
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec23-1"></a>23.1.&nbsp;Germline localization
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In the germline, EGO-1, DRH-3, and CSR-1, but not EKL-1, localize to P granules and promote or maintain P granule structure
                  and association with the nuclear periphery (<a href="#bib22">Claycomb et al., 2009</a>). As oocytes mature, EGO-1 is lost from P granules, but some DRH-3 and CSR-1 continue to associate with P granules in germ
                  cells throughout the life cycle. DRH-3 recruits EGO-1 to the P granule, which recruits CSR-1 in an EKL-1-dependent manner.
                  This assemblage then drives P granule association with the nuclear periphery, possibly by binding mRNA targets as they exit
                  through nuclear pores; such a mechanism would be consistent with loss of P granule perinuclear localization in states of low
                  transcription such as in early germ cells or oocytes (<a href="#bib22">Claycomb et al., 2009</a>). Mutation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene" target="_blank">ekl-1</a></em></span>, or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> disrupts perinuclear localization of P granules (<a href="#bib136">Vought et al., 2005</a>; <a href="#bib22">Claycomb et al., 2009</a>), possibly contributing to the <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000743;class=Phenotype" target="_blank">Rde</a> phenotypes exhibited by these mutants (<a href="#bib119">Smardon et al., 2000</a>; <a href="#bib67">Kim et al., 2005</a>; <a href="#bib105">Robert et al., 2005</a>; <a href="#bib30">Duchaine et al., 2006</a>; <a href="#bib145">Yigit et al., 2006</a>; <a href="#bib49">Gu et al., 2009</a>). These four factors may also promote normal nuclear pore structure. In wild type germlines, P granules are detected near
                  nuclear pore clusters, although not every nuclear pore is associated with a P granule (<a href="#bib99">Pitt et al., 2000</a>). Pachytene chromosomes are not located adjacent to nuclear pores and may not be able to attach to the nuclear envelope in
                  regions of high nuclear pore density (<a href="#bib99">Pitt et al., 2000</a>). Because the nuclear envelope assembles around chromatin, the chromatin may therefore influence distribution of nuclear
                  pores (<a href="#bib135">Vasu and Forbes, 2001</a>). Loss of EGO-1 results in a patchy distribution of nuclear pore marker nucleoporin in mitotic and transition zones that
                  might reflect abnormal chromatin dynamics  (<a href="#bib136">Vought et al., 2005</a>). In sperm and oocytes, CSR-1 is detected in chromatin fractions (<a href="#bib21">Chu et al., 2006</a>), consistent with immunofluorescence data indicating that CSR-1 and EGO-1 are enriched in mature oocyte nuclei, with CSR-1
                  associating with diakinetic chromosomes (<a href="#bib22">Claycomb et al., 2009</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec23-2"></a>23.2.&nbsp;Embryo localization
                        </h3>
                     </div>
                  </div>
               </div>
               <p>EGO-1, DRH-3, EKL-1, and CSR-1 localize to mitotic chromosomes in embryos (<a href="#bib22">Claycomb et al., 2009</a>). In embryonic cells, all four factors show cytoplasmic localization. During mitosis, however, they show nuclear enrichment
                  beginning at prophase. As embryonic mitotic chromosomes condense, EGO-1, DRH-3, and EKL-1 become enriched along the length
                  of each chromosome, but CSR-1 remains nuclear. At the metaphase plate, DRH-3 and CSR-1 localize to chromosomes in a pattern
                  similar to that of cohesins, whereas EKL-1 and, to a lesser degree, EGO-1 localize to chromosomes in a pattern like that of
                  kinetochore proteins. EKL-1 stays associated with chromosomes during anaphase, while the others are more difficult to detect.
                  The factors assemble on chromosomes in a hierarchical order, with DRH-3 recruiting EGO-1 and EKL-1, which in turn recruit
                  CSR-1 (<a href="#bib22">Claycomb et al., 2009</a>). In embryos, CSR-1 pathway components are found to associate directly with chromatin in a CSR-1 22G RNA-dependent manner,
                  supporting a direct role for CSR-1 and associated 22G RNAs in promoting chromosome segregation: CSR-1 shows enrichment at
                  CSR-1 22G RNA target loci, but not WAGO 22G RNA target loci, by chromatin immunoprecipitation; this association is lost in
                  a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> mutant (<a href="#bib22">Claycomb et al., 2009</a>). As CSR-1 22G RNA target loci are distributed relatively uniformly along chromosomes, CSR-1 associates with chromatin genome-wide
                  to influence chromosome segregation (<a href="#bib22">Claycomb et al., 2009</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec24"></a>24.&nbsp;CSR-1 22G RNA effector function
                     </h2>
                  </div>
               </div>
            </div>
            <p>Although CSR-1 shows cleavage activity in cell lysate (<a href="#bib3">Aoki et al., 2007</a>), CSR-1 does not appear to silence CSR-1 22G RNA targets: these targets are expressed in germline, oocyte, and embryo and
               do not show changes in expression upon loss of CSR-1 or DRH-3 (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). Rather, EGO-1, DRH-3, EKL-1, and CSR-1 localize to chromosomes to promote proper organization and alignment of metaphase
               chromosomes and proper orientation of kinetochores to opposing spindle poles (<a href="#bib22">Claycomb et al., 2009</a>). Loss of these factors disrupts chromosome condensation and cohesion: depletion of any by RNAi results in highly disorganized
               condensin and cohesin protein loading onto mitotic chromosomes in embryos (<a href="#bib22">Claycomb et al., 2009</a>). Loss of these factors also disrupts kinetochore assembly. Whereas inner centromeric proteins HCP-3/CENP-A and HCP-4/CENP-C
               should be poleward-localized on both sides of metaphase plate, embryos depleted of CSR-1 22G RNA pathway factors show dramatically
               disorganized loading of these proteins on metaphase chromosomes; outer kinetochore protein KLP-7/MCAK and spindle checkpoint
               protein BUB-1 are also misloaded onto mitotic chromosomes (<a href="#bib22">Claycomb et al., 2009</a>). Thus, the CSR-1 22G RNA pathway directly influences chromosome organization to promote proper segregation. The X chromosome
               is depleted of genes expressed in the germline (<a href="#bib104">Reinke et al., 2004</a>) and therefore of CSR-1 22G RNA targets. Nevertheless, the X chromosome usually segregates correctly in mitosis, suggesting
               that other mechanisms may govern X chromosome segregation (<a href="#bib22">Claycomb et al., 2009</a>).
            </p>
            <p>The CSR-1 22G RNA pathway also regulates H3K9me2 distribution. H3K9me2 becomes enriched on unpaired chromosomes and high-copy
               extrachromosomal arrays during meiosis (<a href="#bib62">Kelly et al., 2002</a>; <a href="#bib10">Bean et al., 2004</a>). Loss of EGO-1, DRH-3, EKL-1, or CSR-1 results in reduced H3K9me2 accumulation on unpaired chromosomes such as the male
               X chromosome and ectopic H3K9me2 accumulation on paired and synapsed chromosomes (<a href="#bib80">Maine et al., 2005</a>; <a href="#bib111">She et al., 2009</a>). Whereas wild-type male pachytene nuclei show a single focus of H3K9me2 labeling, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene" target="_blank">ekl-1</a></em></span>, or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> mutant nuclei with normal pachytene morphology lack a single strong focus of H3K9me2 labeling, and some show multiple bright
               H3K9me2 foci with higher overall levels of H3K9me2 (<a href="#bib80">Maine et al., 2005</a>; <a href="#bib111">She et al., 2009</a>). Mutant nuclei, however, may be large, morphologically abnormal, and polyploid, with diffuse chromosome morphology and multiple
               H3K9me2 foci and high overall H3K9me2 (<a href="#bib111">She et al., 2009</a>). This demonstrates a role for CSR-1 22G RNAs in specifying chromatin modification. However, as CSR-1 22G RNAs target protein-coding
               genes expressed in germline (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>), it is likely that CSR-1 and associated 22G RNAs are excluding repressive H3K9me2 marks from target genomic loci rather
               than depositing them.
            </p>
            <p>Another function of CSR-1 22G RNAs may be to define areas of germline transcription. This is consistent with a proposed role
               for CSR-1 22G RNAs in acting as a self-recognition system in opposition to 21U RNA-directed silencing of DNA perceived as
               non-self; targeting of transcripts by CSR-1 may therefore represent a licensing process that helps to maintain expression
               of <span class="emphasis"><em>bona fide</em></span> germline transcripts (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). As CSR-1 22G RNAs are generated in germline through the activity of RdRP EGO-1, CSR-1 22G RNA targets are robustly expressed
               in maternal germline and may therefore engage CSR-1 as nascent transcripts during gametogenesis; this may define CSR-1 chromatin
               domains that could perdure through embryogenesis independent of transcription (<a href="#bib22">Claycomb et al., 2009</a>). Such a mechanism would account for the prominent localization of CSR-1 to chromatin in mature oocytes (<a href="#bib22">Claycomb et al., 2009</a>). Presumably, CSR-1 establishes euchromatic genomic regions and defines the boundaries of adjacent kinetochores, potentially
               by mediating deposition of activating chromatin marks and HCP-3/CENP-A incorporation, respectively. In embryos, CENP-A is
               loaded at low density onto about half of the genome, inverse to regions transcribed in germline and early embryo, and defines
               centromeric regions (<a href="#bib41">Gassmann et al., 2012</a>). Pre-existing CENP-A nucleosomes are not necessary to guide recruitment of new CENP-A nucleosomes, as CENP-A is not transmitted
               by sperm during fertilization and is unloaded and reloaded during oogenic meiotic prophase (<a href="#bib41">Gassmann et al., 2012</a>). Germline transcription of a genomic region may exclude CENP-A incorporation in progeny, with reinforcement by zygotic transcription
               in early embryogenesis. Domains targeted by the CSR-1 22G RNA pathway and regions enriched for the centromeric histone variant
               CENP-A appear to be mutually exclusive (<a href="#bib22">Claycomb et al., 2009</a>); therefore, the CSR-1 22G RNA pathway may serve to transmit patterns of germline transcription to early embryo. Alternatively,
               H3K36 methylation may fulfill this role, as its pattern also correlates inversely with CENP-A occupancy (<a href="#bib102">Rechtsteiner et al., 2010</a>; <a href="#bib41">Gassmann et al., 2012</a>).
            </p>
            <p>Finally, the CSR pathway also appears to play a critical role in maturation of replication-dependent core histone mRNAs. Processing
               of replication-dependent core histone pre-mRNAs requires endonucleolytic cleavage in the 3&#8217;UTR just downstream of the stem-loop
               that in many organisms is directed by the U7 snRNA (<a href="#bib83">Marzluff et al., 2008</a>). Nematodes, however, lack the U7 snRNA (<a href="#bib28">D&aacute;vila L&oacute;pez and Samuelsson, 2008</a>), and CSR-1 22G RNAs may fulfill this role in its stead (<a href="#bib7">Avgousti et al., 2012</a>): a majority of histone genes (66%) are depleted by loss of CSR-1. Immuno-purified CSR-1 complexes are enriched for 22G RNAs
               targeting all core histone genes, and EGO-1-dependent 22G RNAs map to histone pre-mRNAs at the 3&#8217; end sites of mature histone
               mRNAs (<a href="#bib7">Avgousti et al., 2012</a>). CSR-1 binds these 22G RNAs to target histone pre-mRNAs and either directly cleaves them or facilitates their cleavage.
               Loss of CSR-1 22G RNA pathway factors results in the accumulation of unprocessed histone mRNAs, with a corresponding decrease
               in histone production that could explain the phenotypes of these mutants (<a href="#bib7">Avgousti et al., 2012</a>). For example, histone biogenesis defects in cultured cells result in S-phase extension and cell cycle delay (<a href="#bib137">Wagner et al., 2005</a>; <a href="#bib56">Ideue et al., 2009</a>) reminiscent of the extended transition zone observed in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> mutant germlines (<a href="#bib119">Smardon et al., 2000</a>). Additionally, RNAi-mediated knockdown of a single core histone gene causes chromosome segregation defects and sterility
               similar to the phenotypes of CSR-1 22G RNA pathway mutants (<a href="#bib69">Kodama et al., 2002</a>; <a href="#bib120">Sonnichsen et al, 2005</a>). Consistent with this explanation, overexpression of transgenic histone pre-mRNAs that do not require cleavage rescues lethality
               associated with <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> depletion by RNAi (<a href="#bib7">Avgousti et al., 2012</a>). Phenotypes similar to those shown by CSR-1 22G RNA pathway mutants also result upon depletion of the stem-loop-binding
               protein (SLBP), CDL-1. SLBP is required for histone mRNA processing, stabilization, and efficient translation (<a href="#bib83">Marzluff et al., 2008</a>). In the germline, CDL-1::GFP shows nuclear enrichment in developing oocytes, much like CSR-1 (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib7">Avgousti et al., 2012</a>). RNAi-mediated knockdown of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00000411;class=Gene" target="_blank">cdl-1</a></em></span> results in accumulation of unprocessed histone mRNA, severely depletes histone levels, and causes adult sterility and chromosome
               segregation defects in early embryo (<a href="#bib69">Kodama et al., 2002</a>; <a href="#bib97">Pettitt et al., 2002</a>; <a href="#bib7">Avgousti et al., 2012</a>). In contrast, RNAi-mediated knockdown of condensin component <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004874;class=Gene" target="_blank">smc-4</a></em></span> does not result in histone processing and accumulation defects, showing that this is not a general feature associated with
               compromised chromosome segregation (<a href="#bib7">Avgousti et al., 2012</a>). These results suggest that CSR-1 and CDL-1 may collaborate to promote histone maturation in oocytes, ensuring that stores
               of histones will be sufficient to sustain embryonic division until zygotic transcription begins. It is interesting to note
               that the sterility associated with compromise of the CSR-1 22G RNA pathway may be independent of the P granule defects, as
               loss of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00000411;class=Gene" target="_blank">cdl-1</a></em></span> does not disrupt P granules (<a href="#bib7">Avgousti et al., 2012</a>). Finally, intestinal cell endoreduplication also requires high levels of histone proteins (see WormBook chapter <a href="http://www.wormbook.org/chapters/www_intestine/intestine.html" target="_blank">The <span class="emphasis"><em>C. elegans</em></span> intestine</a>
               ). Possibly related to somatic expression observed for the longer <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> isoform (<a href="#bib22">Claycomb et al., 2009</a>), CSR-1 protein is detected in adult intestinal nuclei (<a href="#bib7">Avgousti et al., 2012</a>). These cells show depletion of histone H2B upon RNAi knockdown of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span>, suggesting that CSR-1 function in histone processing may not be restricted to the germline (<a href="#bib7">Avgousti et al., 2012</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec25"></a>25.&nbsp;CDE-1 regulates the CSR-1 22G RNA pathway
                     </h2>
                  </div>
               </div>
            </div>
            <p>The CSR-1 22G RNA pathway requires the conserved &#946; nucleotidyltransferase CDE-1 for uridylation of 22G RNAs to restrict their
               accumulation (<a href="#bib132">van Wolfswinkel et al., 2009</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019629;class=Gene" target="_blank">cde-1</a></em></span> transcript (<a href="#bib104">Reinke et al., 2004</a>) is detected by <span class="emphasis"><em>in situ</em></span> hybridization throughout the entire gonad but not in other tissues (<a href="#bib132">van Wolfswinkel et al., 2009</a>). In the male or hermaphrodite spermatogenic germline, CDE-1 localizes to bright granules in close proximity to the condensing
               DNA of mature sperm. In the oogenic germline, CDE-1 localization is cytoplasmic with perinuclear enrichment. Within the embryonic
               P lineage, CDE-1 localizes to perinuclear granules, colocalizing mainly with PGL-1; in mitotic cells, CDE-1 shows EGO-1-dependent
               localization to the outer edges of condensing chromosomes at prometaphase and the poleward sides of chromosomes at metaphase,
               with expression detectable through anaphase. CDE-1 immuno-purifies with EGO-1 in embryo extract, but CSR-1 is not required
               for EGO-1 accumulation. Loss of EGO-1 results in dispersed localization of CDE-1 over the whole metaphase plate and a CDE-1
               halo surrounding the DNA in condensing sperm. Depletion of CSR-1 results in very faint but correct CDE-1 localization to the
               metaphase plate, but no association of CDE-1 granules with DNA in sperm. Loss of EGO-1 or CSR-1 does not affect CDE-1 localization
               to P-granules, and EGO-1 and CSR-1 show normal localization in the absence of CDE-1. CDE-1 protein is not detected in intestine
               (<a href="#bib132">van Wolfswinkel et al., 2009</a>).
            </p>
            <p>CDE-1 is a catalytically active nucleotidyltransferase. Immuno-purified CDE-1 complexes preferentially catalyze the addition
               of uridines to small RNA 3&#8217; ends <span class="emphasis"><em>in vitro</em></span> (<a href="#bib132">van Wolfswinkel et al., 2009</a>), consistent with earlier findings expressing CDE-1 in <span class="emphasis"><em>Xenopus</em></span> oocytes (<a href="#bib70">Kwak and Wickens, 2007</a>). Uridine tails added <span class="emphasis"><em>in vitro</em></span> are short, suggesting low processivity, and can be blocked by 2&#8217;-O-methylation of the substrate small RNA 3&#8217; end (<a href="#bib132">van Wolfswinkel et al., 2009</a>). <span class="emphasis"><em>In vivo</em></span>, CDE-1 uridylates a portion of CSR-1 22G RNAs. Endo-siRNAs, but not microRNAs, show addition of one or more untemplated 3&#8217;
               uridines that are lost in the absence of CDE-1 in adults; however, some endo-siRNAs in young adult still show untemplated
               uridines in a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019629;class=Gene" target="_blank">cde-1</a></em></span> mutant, suggesting redundant terminal uridylation activity at earlier stages (<a href="#bib132">van Wolfswinkel et al., 2009</a>). It is not clear whether CDE-1 uridylates 22G RNAs already bound by CSR-1; however, no association between the two proteins
               is detected in embryo extract (<a href="#bib132">van Wolfswinkel et al., 2009</a>). Uridylation by CDE-1 appears to destabilize CSR-1 22G RNAs, as those frequently found uridylated in wild type are increased
               upon loss of CDE-1, whereas levels of microRNAs, 21U RNAs, and other antisense small RNAs are not (<a href="#bib132">van Wolfswinkel et al., 2009</a>). Accordingly, immuno-purification of CSR-1 complexes from a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019629;class=Gene" target="_blank">cde-1</a></em></span> mutant recovers many more siRNAs, although the distribution of targets does not change (<a href="#bib132">van Wolfswinkel et al., 2009</a>). This suggests that CDE-1 acts to reduce the half-life of CSR-1 22G RNAs, ensuring that CSR-1 is only partially loaded,
               and, indeed, CSR-1 22G RNAs are rarer by far than WAGO 22G RNAs (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). Their modest accumulation may also explain why CSR-1 22G RNA targets are not silenced, as only the more abundantly WAGO
               22G RNA-targeted loci are appreciably silenced (<a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib49">Gu et al., 2009</a>). CDE-1-mediated uridylation and partial loading of CSR-1 may serve to restrict CSR-1 22G RNAs to the CSR-1 22G RNA pathway,
               as loss of CDE-1 results in erroneous gene silencing likely due to misloading of CSR-1 22G RNAs onto WAGO 22G RNA Argonautes
               (<a href="#bib132">van Wolfswinkel et al., 2009</a>). This may explain the phenotypes of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019629;class=Gene" target="_blank">cde-1</a></em></span> mutants, which show defects in cosuppression, increased Tc1 transposon activity, and partial insensitivity to RNAi in the
               germline (<a href="#bib105">Robert et al., 2005</a>; <a href="#bib132">van Wolfswinkel et al., 2009</a>). In the absence of CDE-1-mediated uridylation, excess CSR-1 22G RNAs compete with WAGO 22G RNAs for Argonatute binding,
               compromising exo-RNAi in the germline. Similarly, Tc1-targeting siRNAs are decreased, possibly due to decreased stabilization
               by available WAGO Argonautes, resulting in Tc1 activation and increased dsDNA breaks and apoptosis (<a href="#bib132">van Wolfswinkel et al., 2009</a>).
            </p>
            <p>Loss of CDE-1 leads to meiotic and embryonic mitotic segregation defects with a resulting <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001175;class=Phenotype" target="_blank">Him</a> phenotype, presumably due to erroneous targeting of CSR-1 22G RNA targets by WAGO Argonautes (<a href="#bib132">van Wolfswinkel et al., 2009</a>). Germline nuclei do not show mitotic missegregation, but diakinetic oocytes often show univalents, suggesting impaired pairing
               of homologous chromosomes during meiosis (<a href="#bib132">van Wolfswinkel et al., 2009</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019629;class=Gene" target="_blank">cde-1</a></em></span> mutant embryos show metaphase plate disorganization similar to that observed in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> mutant embryos, with abnormal CENP-A deposition, and occasional severe spindle defects and polar body retention (<a href="#bib119">Smardon et al., 2000</a>; <a href="#bib145">Yigit et al., 2006</a>; <a href="#bib22">Claycomb et al., 2009</a>; <a href="#bib132">van Wolfswinkel et al., 2009</a>). In analysis of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019629;class=Gene" target="_blank">cde-1</a></em></span> mutant hermaphrodite x wild-type male hybrid embryos, both parental chromosomes are affected by major segregation defects,
               indicating that missegregation is not limited to the parental germline and persists into embryonic development (<a href="#bib132">van Wolfswinkel et al., 2009</a>). Consistent with the collaborative but antagonistic functions of CDE-1 and CSR-1 22G RNA pathway factors, the embryonic
               lethality of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00019629;class=Gene" target="_blank">cde-1</a></em></span> mutants is less severe than that of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00017641;class=Gene" target="_blank">csr-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001214;class=Gene" target="_blank">ego-1</a></em></span> mutants (<a href="#bib132">van Wolfswinkel et al., 2009</a>). CDE-1 also regulates processing of pre-let-7 through uridylation (<a href="#bib54">Heo et al., 2008</a>) and may act in additional roles in other small RNA pathways.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec26"></a>26.&nbsp;21U RNA features and loci
                     </h2>
                  </div>
               </div>
            </div>
            <p>The following pertains to canonical or &#8220;Type 1&#8221; 21U RNAs. &#8220;Type 2&#8221; 21U RNAs are associated with transcription start sites
               (TSSs) of protein-coding mRNAs and other Pol II transcripts and are discussed below.
            </p>
            <p>Early deep sequencing of <span class="emphasis"><em>C. elegans</em></span> small RNAs identified a highly diverse population of 21 nt small RNAs with a 5&#8217; uridine bias and no other common sequence
               features (<a href="#bib108">Ruby et al., 2006</a>). These 21U RNAs are 5&#8217; monophosphorylated (<a href="#bib108">Ruby et al., 2006</a>) and terminally methylated by HENN-1 methyltransferase (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>). Unlike endo-siRNAs, 21U RNAs are not generated from mRNA templates by RdRPs but rather transcribed directly from genomic
               loci primarily clustered in two gene-depleted regions of chromosome IV (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib48">Gu, W., et al., 2012</a>). 21U RNA loci are vastly depleted for overlap with exons. Those that do overlap show no sense or antisense bias (<a href="#bib108">Ruby et al., 2006</a>). 21U RNA loci do not show a prominent strand bias and overlap less than expected by chance (<a href="#bib108">Ruby et al., 2006</a>), consistent with their independent transcription from short precursors, (<a href="#bib48">Gu, W., et al., 2012</a>; <a href="#bib13">Billi et al, 2013</a>). Although the larger genomic clusters of 21U RNA loci show synteny across rhabditids, the 21U RNAs themselves show no sequence
               conservation (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib29">de Wit et al., 2009</a>). This lack of conservation suggests that evolutionary pressure has maximized the diversity of 21U RNA sequences, in line
               with their purported role as a genetic immune system capable of selectively targeting non-self transcripts (<a href="#bib4">Ashe et al., 2012</a>;  <a href="#bib72">Lee et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). Consistent with their free evolution, the sequence content of a 21U RNA appears to be irrelevant for expression, with the
               exception of the 5&#8217; uridine (<a href="#bib13">Billi et al, 2013</a>).
            </p>
            <p>21U RNAs are robustly expressed in the germline (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>) and detected in early embryo, but their levels decline across development (<a href="#bib123">Stoeckius et al., 2009</a>). A majority of 21U RNAs show differential expression in male and female germlines (<a href="#bib13">Billi et al, 2013</a>; <a href="#bib113">Shi et al., 2013</a>). These male and female germline-enriched subpopulations of 21U RNAs direct the production of nonoverlapping 22G RNAs that
               target distinct transcripts. Targets of male germline-enriched 21U RNAs are depleted for transcripts enriched in spermatogenesis
               but not oogenesis (<a href="#bib104">Reinke et al., 2004</a>), suggesting selection against evolution of male-expressed 21U RNAs that target male germline genes (<a href="#bib13">Billi et al, 2013</a>; <a href="#bib113">Shi et al., 2013</a>). Female germline-enriched 21U RNAs do not show depletion of spermatogenesis-enriched transcripts and are paradoxically enriched
               for oogenesis-enriched transcripts. While the significance of this is uncertain, the distinct enrichment patterns suggest
               that male and female germline-enriched 21U RNAs are subject to different evolutionary pressures (<a href="#bib13">Billi et al, 2013</a>; <a href="#bib113">Shi et al., 2013</a>). Female germline-enriched 21U RNAs are preferentially detected in embryos (<a href="#bib13">Billi et al, 2013</a>; <a href="#bib113">Shi et al., 2013</a>), similar to the pattern observed for ERGO-1 class 26G RNAs (<a href="#bib52">Han et al., 2009</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec27"></a>27.&nbsp;21U RNA targets
                     </h2>
                  </div>
               </div>
            </div>
            <p>21U RNAs target transcripts by directing Argonaute PRG-1 to imperfectly complementary sites with up to three, or possibly
               even four, mismatches (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>). 21U RNA targets are depleted of protein-coding transcripts but not pseudogene or transposon transcripts (<a href="#bib8">Bagijn et al., 2012</a>). As 21U RNA targeting triggers local production of WAGO 22G RNAs (<a href="#bib27">Das et al., 2008</a>; <a href="#bib72">Lee et al., 2012</a>), 21U RNA targets are enriched for WAGO 22G RNA targets and depleted for CSR-1 22G RNA targets (<a href="#bib72">Lee et al., 2012</a>). As 21U RNAs are capable of triggering transgenerational silencing that persists for tens of generations in the absence
               of PRG-1 and additional 21U RNA trigger (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>), identification of target transcripts is challenging. Targets can be approximated as those transcripts that show robust
               21U RNA-dependent 22G RNA accumulation or those that show upregulation upon loss of PRG-1, although the latter are more likely
               to report a specific subset of targets whose silencing is more heavily weighted toward cytoplasmic than nuclear RNAi. Possibly,
               the rapid evolution of 21U RNA sequences and the profoundly stable silencing exerted by 21U RNAs renders definition of a specific
               target set irrelevant, as many previously silenced targets have likely been eroded through genetic drift. So may be the eventual
               fate of sequences not protected from 21U RNA targeting by a self-recognition system such as CSR-1 22G RNAs are proposed to
               represent (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec28"></a>28.&nbsp;21U RNA biogenesis
                     </h2>
                  </div>
               </div>
            </div>
            <p>21U RNAs are not transcribed by RdRP complexes, nor does their biogenesis appear to require DCR-1 or any of the factors directly
               involved in endo-siRNA biogenesis (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>). Rather, 21U RNAs are encoded as independent Pol II transcriptional units (<a href="#bib48">Gu, W., et al., 2012</a>; <a href="#bib13">Billi et al, 2013</a>) (<a href="#figure5">Figure 5</a>). In <span class="emphasis"><em>C. elegans</em></span> and other rhabditids, 21U RNA loci show a common upstream sequence arrangement of two motifs, the large and small motifs,
               that together drive the regulated expression of 21U RNAs in the germline (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib19">Cecere et al., 2012</a>; <a href="#bib48">Gu, W., et al., 2012</a>; <a href="#bib13">Billi et al, 2013</a>). In <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>, 21U RNA loci distribution shows a pattern of enrichment at the start and end of full-length DNA transposon loci in sense
               and antisense orientations, respectively, suggesting that 21U RNA upstream motifs may serve as traps for triggering 21U RNA
               biogenesis against recent integrations (<a href="#bib8">Bagijn et al., 2012</a>).
            </p>
            <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="endoRNAipathwys_fig5.jpg"><img src="endoRNAipathwys_fig5_s.jpg" border="2" align="middle" alt=" figure 5"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 5. The 21U/piRNA pathway.&nbsp;</b>piRNAs are generated from two types of loci, from piRNA genes mostly found in large clusters on chromosome IV and from protein
                     coding genes and other Pol II transcripts. These two types of loci share a common small motif, but piRNA genes show a large
                     motif that is bound by Forkhead transcription factors and is required to drive piRNA expression. piRNAs derived from protein
                     coding genes map to transcription start sites and may reflect promoter proximal pausing. Both types of piRNA loci produce
                     capped 26 nt precursors that are subsequently trimmed and 2&#8217;-O-methylated by HENN-1 to produce mature 21U RNAs. 21U RNAs associate
                     with the Piwi proteins PRG-1/2 and trigger secondary 22G siRNA production.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec28-1"></a>28.1.&nbsp;The 21U RNA large motif and Forkhead family transcription factors
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The large motif, occupying ~60-25 bp upstream of the 21U RNA locus, consists of A/T-richness and a central 8 bp core motif
                  of consensus sequence CTGTTTCA (<a href="#bib108">Ruby et al., 2006</a>). The A/T-richness of this region repels nucleosomes, resulting in nucleosome depletion upstream of 21U RNA locus; accordingly,
                  the genomic 21U RNA clusters of chromosome IV show general depletion of nucleosomes (<a href="#bib19">Cecere et al., 2012</a>). The core motif is conserved in other rhabditids, but more divergent in <span class="emphasis"><em>Pristionchus pacificus</em></span>; however, the spacing is conserved, suggesting mechanistic conservation (<a href="#bib29">de Wit et al., 2009</a>; <a href="#bib113">Shi et al., 2013</a>). The core motif is required for expression of the downstream 21U RNA, as its deletion or mutation results in loss of the
                  associated 21U RNA (<a href="#bib19">Cecere et al., 2012</a>; <a href="#bib13">Billi et al, 2013</a>). A subset of Forkhead family transcription factors, including at least FKH-3, FKH-4, FKH-5, and UNC-130, binds this core
                  motif to promote 21U RNA expression: 21U RNA upstream regions immuno-purify with UNC-130::GFP, and recombinant Forkhead proteins
                  preferentially bind dsDNA probes containing the core motif octamer (<a href="#bib19">Cecere et al., 2012</a>). An <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00006853;class=Gene" target="_blank">unc-130</a></em></span> mutant shows ~50% depletion of individual 21U RNA species; similar depletion is observed upon partial, simultaneous RNAi
                  knockdown of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001435;class=Gene" target="_blank">fkh-3</a>/4/5</em></span> (<a href="#bib19">Cecere et al., 2012</a>). It is not yet understood whether these Forkhead proteins are semiredundantly required for expression of 21U RNAs or individually
                  contribute to expression. Consistent with the core motif acting as a promoter element for independent transcription of 21U
                  RNAs, the spacing between core motifs and 21U RNA loci affects 21U RNA expression, as does the sequence content. Plotting
                  21U RNA abundances by spacer length reveals a strong correlation, with spacer lengths of 37-40 nt showing the highest abundances
                  and tailing down to either side (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib13">Billi et al, 2013</a>). The tolerance in the spacer length affords expression of multiple, genomically miniclustered 21U RNAs in association with
                  the same core motif (<a href="#bib59">Kato et al., 2009</a>; <a href="#bib13">Billi et al, 2013</a>). Core motifs that more closely match the consensus sequence, and in particular those that contain the central GTTTC, are
                  correlated with higher abundance 21U RNAs and are more likely to be associated with 21U RNA miniclusters than solitary 21U
                  RNAs (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib13">Billi et al, 2013</a>). Miniclustered 21U RNA loci necessarily lie at different genomic distances from the core motif, and their respective abundances
                  recapitulate the correlation between optimal spacer length and expression level (<a href="#bib13">Billi et al, 2013</a>). The sequence content of the core motif also influences germline enrichment. Core motifs whose first nucleotide is cytidine
                  are preferentially enriched in male germline, whereas female germline-enriched 21U RNAs show no preference at this motif position
                  and in general show core motifs with poorer consensus match (<a href="#bib13">Billi et al, 2013</a>). A simple explanation for differential germline expression of 21U RNAs according to their motifs would be selective binding
                  by Forkhead transcription factors; however, neither UNC-130 nor FKH-3/4/5 appears to be selectively required for 21U RNAs
                  expressed in male germline or showing a cytidine-intial motif (Allison Billi, Mallory Freeberg, and John Kim., unpublished
                  data), suggesting that other factors confer germline enrichment.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec28-2"></a>28.2.&nbsp;The 21U RNA small motif and precursor
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The small motif shows a YRNT consensus sequence with the thymidine situated at position 1 of the 21U RNA locus and the YR
                  pair at 3 and 2 nt upstream, resembling the initiator element required for Pol II transcription initiation (<a href="#bib108">Ruby et al., 2006</a>; <a href="#bib48">Gu, W., et al., 2012</a>). Global profiling of endogenous capped small RNAs (csRNAs) reveals that the R position of the small motif corresponds to
                  the first nucleotide of the 21U RNA precursor (<a href="#bib48">Gu, W., et al., 2012</a>). These csRNA precursors show blocked 5&#8217; ends consistent with Pol II-mediated capping and an average length of 26 nt, corresponding
                  to the m<sup>7</sup>GpppAmNm <a href="http://www.wormbase.org/db/get?name=WBGene00000293;class=Gene" target="_blank">cap-2</a> structure upstream and ~3 nt downstream that are removed during 21U RNA maturation (<a href="#bib48">Gu, W., et al., 2012</a>). The abundances of csRNA precursors correlate significantly with abundances of their cognate 21U RNAs, supporting their
                  status as precursors (<a href="#bib48">Gu, W., et al., 2012</a>). A very small proportion of 21U RNAs show evidence of longer, &gt;70 nt capped sequences (<a href="#bib19">Cecere et al., 2012</a>); however, these are unlikely to represent csRNA/21U RNA precursors, as their abundances do not correlate with abundances
                  of cognate 21U RNAs (<a href="#bib48">Gu, W., et al., 2012</a>). The importance of the YR consensus match is indicated by the respective abundances of 21U RNAs encoded at loci staggered
                  by 1 nt. Only one 21U RNA of each pair can show a YRNT motif; those that do are ten-fold more abundant than their non-YRNT
                  sister species (<a href="#bib48">Gu, W., et al., 2012</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec29"></a>29.&nbsp;21U RNA precursor processing and Argonaute loading
                     </h2>
                  </div>
               </div>
            </div>
            <p>In association with an upstream large motif or at a TSS, the YR pair of the YRNT small motif appears sufficient to direct
               csRNA expression, whereas the thymidine is required for 21U RNA accumulation. The accumulation of csRNAs, but not mature 21U
               RNAs, associated with YNRV small motifs suggests that the 5&#8217; uridine of 21U RNAs is dispensable for transcription but plays
               a role in mature 21U RNA stabilization or processing (<a href="#bib48">Gu, W., et al., 2012</a>). In support of this, mutation of the thymidine to G or A abrogates accumulation of a cognate 21G or 21A species (<a href="#bib13">Billi et al, 2013</a>). Furthermore, comparison of other <span class="emphasis"><em>C. elegans</em></span> isolates with YRNT motifs where the N2 isolate shows YRNV reveals that substitution for a genomic thymidine results in accumulation
               the associated 21U RNA (<a href="#bib48">Gu, W., et al., 2012</a>; <a href="#bib13">Billi et al, 2013</a>). The presence of a uridine at position 3 of the csRNA may be required for removal of the m<sup>7</sup>GpppAmNm <a href="http://www.wormbase.org/db/get?name=WBGene00000293;class=Gene" target="_blank">cap-2</a> structure to generate the mature 21U RNA 5&#8217; end; however, recombinant PRG-1 shows preferential target cleavage in association
               with 5&#8217; uridine ssRNA (<a href="#bib8">Bagijn et al., 2012</a>), suggesting PRG-1 may purify candidate small RNAs processed from csRNAs by selectively binding and stabilizing those with
               5&#8217; uridines. Identification of decapping machinery responsible for csRNA 5&#8217; end processing will address this question definitively.
               Trimming of the csRNA 3&#8217; end to achieve the mature 21 nt length may occur after PRG-1 binding. While the vast majority of
               small RNA deep sequencing reads corresponding to 21U RNA loci are 21 nt in length, a small portion show extensions, with a
               significant bias toward 3&#8217; extensions (<a href="#bib59">Kato et al., 2009</a>). These 3&#8217; extended reads may represent biogenesis intermediates and support the hypothesis that 3&#8217; trimming occurs after
               5&#8217; processing. In silkworm, PIWI Argonaute-bound small RNAs are trimmed after Argonaute loading by a 3&#8217; to 5&#8217; exonuclease
               until the length is sufficiently short for anchoring of the 3&#8217; end in the Argonaute (<a href="#bib60">Kawaoka et al., 2011</a>). While the 26 nt length of 26G RNAs is determined by DCR-1 processing, both 22G RNAs and 21U RNAs are DCR-1-independent
               (<a href="#bib3">Aoki et al., 2007</a>; <a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>; <a href="#bib96">Pavelec et al., 2009</a>). Perhaps a common exonuclease trims 22G RNAs and 21U RNAs.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec29-1"></a>29.1.&nbsp;PRG-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>PRG-1 was first identified as a factor required for germline stem cell proliferation and/or self-renewal: injection of antisense
                  <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004178;class=Gene" target="_blank">prg-1</a></em></span> RNA into the germline results in significant shortening of the mitotic proliferative zone, reduced sperm production, and
                  decreased production of offspring (<a href="#bib26">Cox et al., 1998</a>). Loss of PRG-1 results in decreased brood size at 20&deg;C and sterility at 25&deg;C (<a href="#bib145">Yigit et al., 2006</a>). This is likely due to defective sperm activation and fertilization. Spermatocytes are present at 25&deg;C, but mature spermatids
                  are rare; those that do form fail to produce pseudopodia upon activation <span class="emphasis"><em>in vitro</em></span> (<a href="#bib138">Wang and Reinke, 2008</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004178;class=Gene" target="_blank">prg-1</a></em></span> mutant fertility is rescued to wild-type levels by mating to wild-type males at 20&deg;C, but still shows an average of half
                  the wild-type brood size when out-crossed at 25&deg;C (<a href="#bib138">Wang and Reinke, 2008</a>), suggesting compromised female fertility as well. PRG-1 is one of three PIWI clade Argonautes encoded by the <span class="emphasis"><em>C. elegans</em></span> genome: ERGO-1 is highly divergent, whereas PRG-1 and PRG-2 show 91% amino acid identity and are likely the result of a recent
                  gene duplication (<a href="#bib26">Cox et al., 1998</a>; <a href="#bib145">Yigit et al., 2006</a>; <a href="#bib27">Das et al., 2008</a>). Despite their nearly identical sequences, only PRG-1 is required for 21U RNA accumulation (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>). Expression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004178;class=Gene" target="_blank">prg-1</a></em></span> mRNA is highly germline-restricted and detected primarily in young adult and gravid adult (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>). PRG-1 protein is abundant in young adult, gravid adult, and embryo, suggesting maternal inheritance (<a href="#bib9">Batista et al., 2008</a>). Throughout the germline, PRG-1 associates consistently with P granules, with perinuclear localization in mitotic and meiotic
                  zones, loss of expression during spermatid maturation, and dissociation from the nuclear periphery in mature oocytes (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib138">Wang and Reinke, 2008</a>). Embryonic PRG-1 is also associated with P granules in the P lineage (<a href="#bib9">Batista et al., 2008</a>). PRG-1 is required for stabilizing 21U RNAs, and therefore accumulation of 21U RNAs parallels PRG-1 protein expression (<a href="#bib104">Reinke et al., 2004</a>; <a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>). PRG-1 binds 21U RNAs to target transcripts that are often imperfectly complementary by triggering biogenesis in <span class="emphasis"><em>trans</em></span> of WAGO 22G RNAs (<a href="#bib27">Das et al., 2008</a>; <a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>). These 21U RNA-dependent 22G RNAs can trigger transgenerational silencing by binding HRDE-1 and engaging the germline nuclear
                  RNAi pathway (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). Loss of PRG-1 results in target derepression due to failure to trigger dependent 22G RNAs: in particular, levels of Tc3
                  transposase increase and WAGO 22G RNAs targeting Tc3 decrease (<a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>; <a href="#bib49">Gu et al., 2009</a>). However, transcripts upregulated by loss of PRG-1 may not represent the full set of targets silenced by 21U RNAs, as transgenerational
                  silencing engaged by 21U RNA-dependent WAGO 22G RNAs can be effectively maintained in the absence of 21U RNA trigger (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>; <a href="#bib78">Luteijn et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>). This, in addition to the fact that PRG-1-mediated targeting may not trigger WAGO 22G RNA biogenesis as effectively as other
                  primary small RNA triggers (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>), likely explains the subtle effect of 21U RNA loss upon global transcript expression (<a href="#bib9">Batista et al., 2008</a>). The catalytic triad of PRG-1 is intact, and recombinant PRG-1 is capable of cleaving target RNA when incubated with complementary
                  5&#8217; uridine ssRNA; cleavage is abrogated by mutating a key catalytic residue or by introducing a mismatch at the cleavage site
                  (<a href="#bib8">Bagijn et al., 2012</a>). Nevertheless, introduction of catalytically inactive PRG-1 appears to rescue <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004178;class=Gene" target="_blank">prg-1</a></em></span> mutant 21U RNA accumulation, 22G RNA triggering, and target silencing, although the rescue of fertility may be imperfect
                  (<a href="#bib8">Bagijn et al., 2012</a>; <a href="#bib72">Lee et al., 2012</a>). This suggests that, much like ERGO-1 class 26G RNAs, 21U RNAs silence targets primarily by triggering biogenesis of WAGO
                  22G RNAs that engage cytoplasmic and nuclear Argonautes.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec29-2"></a>29.2.&nbsp;HENN-1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>HENN-1 protein and loss of function phenotypes are described in greater detail above. HENN-1 terminal methylates 21U RNAs
                  universally (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>). Although HENN-1 does not obviously localize to P granules (<a href="#bib12">Billi et al., 2012</a>) and stable interaction between PRG-1 and HENN-1 could not be detected (<a href="#bib58">Kamminga et al., 2012</a>), 21U RNA methylation presumably occurs after Argonaute loading, in analogy to ERGO-1, and 3&#8217; end trimming (<a href="#bib12">Billi et al., 2012</a>). HENN-1 is detected in a larger complex of ~100 kD, suggesting that HENN-1 may act as a complex or associate with another
                  factor, such as perhaps a trimmer exonuclease (<a href="#bib58">Kamminga et al., 2012</a>). HENN-1 is required for robust 21U RNA inheritance into embryo and perdurance in offspring (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>), but loss of HENN-1 shows a lesser effect on 21U RNA accumulation in the adult germline (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>). In contrast, ERGO-1 class 26G RNAs are quite significantly affected (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>). This discrepancy does not appear to be due to differences in targeting complementarity preferentially triggering 26G RNA
                  degradation, however, as the presence of a perfectly complementary 21U RNA target does not appear to stimulate 21U RNA trimming
                  and tailing upon loss of methylation (<a href="#bib58">Kamminga et al., 2012</a>). Furthermore, both classes of 26G RNAs show roughly equivalent frequency, and 21U RNAs the lowest, of trimming and tailing
                  among major small RNA classes in a wild-type background (<a href="#bib87">Montgomery et al., 2012</a>), suggesting that terminal methylation in <span class="emphasis"><em>C. elegans</em></span> may not simply exist to oppose trimming and tailing. Nevertheless, loss of HENN-1 does result in increased levels of 20 nt
                  reads corresponding to 21U RNAs (<a href="#bib58">Kamminga et al., 2012</a>). The proportion of a particular 21U RNA species that is trimmed is highly correlated across different <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00015349;class=Gene" target="_blank">henn-1</a></em></span> mutant libraries, and 21U RNAs that show higher trimming frequencies are more likely to be depleted in <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00015349;class=Gene" target="_blank">henn-1</a></em></span> mutant libraries, indicating that 21U RNA trimming is related to greater instability and varies by species (<a href="#bib58">Kamminga et al., 2012</a>). The presence of HENN-1 appears to enhance 21U RNA-mediated silencing of a reporter transgene, but this may be an indirect
                  consequence of decreased expression of factors involved in the WAGO 22G RNA pathway (<a href="#bib58">Kamminga et al., 2012</a>). Loss of HENN-1 results in extremely modest, if any, effect on levels of Tc3 transposase (<a href="#bib12">Billi et al., 2012</a>; <a href="#bib58">Kamminga et al., 2012</a>; <a href="#bib87">Montgomery et al., 2012</a>) and no increase in Tc3 transposition (<a href="#bib58">Kamminga et al., 2012</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec30"></a>30.&nbsp;TSS-associated 21U RNAs
                     </h2>
                  </div>
               </div>
            </div>
            <p>Mapping of csRNAs reveals that a majority are not associated with upstream core CTGTTTCA sequences; rather, many of these
               map bidirectionally to TSSs of protein-coding and other Pol II transcripts (<a href="#bib48">Gu, W., et al., 2012</a>). Like chromosome IV csRNAs associated with large motifs, these TSS-associated csRNAs are also processed into mature 21U
               RNAs when encoded by YRNT motifs (<a href="#bib48">Gu, W., et al., 2012</a>). The abundance of csRNAs at a particular promoter correlates with the abundance of longer capped reads originating from
               that promoter, suggesting that these csRNAs are products of Pol II initiation at active promoters and may reflect promoter-proximal
               pausing (<a href="#bib48">Gu, W., et al., 2012</a>). The significance of these TSS-associated 21U RNAs is as yet unknown. Much as Pol II initiation at canonical TSSs produces
               both csRNAs and longer pre-mRNA transcripts, a small fraction of 21U RNA loci show both csRNAs and corresponding longer reads
               (<a href="#bib19">Cecere et al., 2012</a>; <a href="#bib48">Gu, W., et al., 2012</a>). This suggests that promoter elements and 21U RNA upstream motifs may both promote Pol II association and transcription
               initiation, but 21U RNA loci generally lack sequence elements or chromatin configuration necessary to promote further transcript
               elongation. csRNA distribution does reveal differences in 21U RNA and canonical promoters, as bidirectional transcription
               is not reported to occur at 21U RNA loci. 21U RNAs are required for full fertility (<a href="#bib145">Yigit et al., 2006</a>; <a href="#bib9">Batista et al., 2008</a>; <a href="#bib27">Das et al., 2008</a>; <a href="#bib138">Wang and Reinke, 2008</a>), so it seems possible that csRNAs mapping to canonical promoters are incidental byproducts of selection for Pol II activity
               that produces 21U RNA precursors. However, promoter-associated csRNAs are observed in other metazoan genomes  (<a href="#bib101">Rasmussen and Lis, 1995</a>; <a href="#bib90">Nechaev et al., 2010</a>), suggesting rather that 21U RNA production may originally have arisen incidental to this more ancient process of unknown
               significance.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec31"></a>31.&nbsp;Future directions
                     </h2>
                  </div>
               </div>
            </div>
            <p>As a model organism, <span class="emphasis"><em>C. elegans</em></span> rose to fame for its utility in the study of cell differentiation and organ development, yet this nematode has proven one
               of the richest models for study of small RNAs. The earlier challenges&#8212;developing methods to capture small RNAs of unknown
               properties, distinguishing the unique subclasses, identifying the factors involved in their biogenesis and effector mechanisms&#8212;have
               been met due to the ingenuity of pioneers in the field. Advances in sequencing technology have allowed us to catalogue the
               small RNAs of <span class="emphasis"><em>C. elegans</em></span> to greater depth than ever anticipated, and new approaches continue to reveal additional complexity, such as the recently
               identified TSS-associated 21U RNAs found through the novel CapSeq method (<a href="#bib48">Gu, W., et al., 2012</a>). Genetic screens have also been invaluable in defining the protein factors that act in these pathways. Early screens identified
               suites of candidate small RNA pathway factors using complex RNAi-related phenotypes as readouts for incompletely understood
               underlying mechanisms. The roles of these candidates are still being elucidated, for example, chromatin factors previously
               implicated in transgene or transposon silencing (<a href="#bib61">Kelly and Fire, 1998</a>; <a href="#bib134">Vastenhouw et al., 2003</a>; <a href="#bib45">Grishok et al., 2005</a>; <a href="#bib67">Kim et al., 2005</a>) have only just been positioned downstream of the nuclear RNAi pathway and transgenerational silencing (<a href="#bib4">Ashe et al., 2012</a>; <a href="#bib114">Shirayama et al., 2012</a>).
            </p>
            <p>In this young field of small RNAs, large, fundamental questions at every step remain unanswered. In biogenesis: How are transcripts
               selected for targeting by 26G RNAs? Currently, there is only speculation that secondary structure may somehow contribute to
               recruitment of biogenesis factors. In amplification: How does primary small RNA recognition trigger local biogenesis of secondary
               siRNAs? The key may lie in Mutator foci or other such physical cellular substructures that serve as amplification factories.
               In silencing: What are the chief post-transcriptional mechanisms by which siRNAs repress target mRNAs? The cleavage activity
               of many <span class="emphasis"><em>C. elegans</em></span> Argonautes appears dispensable or non-conserved, contradicting the RNAi canon of Argonaute-mediated target cleavage. And
               particularly in broader function: What are the molecular mechanisms by which CSR-2 22G RNAs promote chromosome segregation?
               Why does loss of male 26G RNAs lead to defects in spermatogenesis? How does loss of the nuclear RNAi pathway compromise germline
               immortality? The final questions that bear upon us are those of conservation. Mammalian piRNAs show entirely unique biogenesis
               mechanisms, although downstream aspects of piRNA biology are shared with the 21U RNAs. This does not exclude the possibility
               that mammals express small RNAs from tiny transcriptional units like 21U RNA loci; discovery of small RNA classes and mechanisms
               in mammals is not yet at saturation. Although many of the pathways found in <span class="emphasis"><em>C. elegans</em></span> are unlikely to be conserved in higher animals lacking the greatly expanded Argonaute family and its worm-specific clade,
               analogous or partially overlapping pathways may function in the germlines of other organisms. microRNAs, once regarded as
               a peculiarity of <span class="emphasis"><em>C. elegans</em></span> genetics, have emerged as master regulators of most developmental processes. Endo-siRNAs and piRNAs appear to occupy a similar
               position of prominence in the <span class="emphasis"><em>C. elegans</em></span> germline and embryo. It remains to be seen how this role translates in higher organisms.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec32"></a>32.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed"><span class="bibliomisc">Alder, M.N., Dames, S., Gaudet, J., and Mango, S.E. (2003). Gene silencing in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by transitive RNA interference. RNA <span class="emphasis"><em>9</em></span>, 25-32.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12554873&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.2650903" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed"><span class="bibliomisc">Ambros, V., Lee, R.C., Lavanway, A., Williams, P.T., and Jewell, D. (2003). MicroRNAs and other tiny endogenous RNAs in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>13</em></span>, 807-818.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12747828&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(03)00287-2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed"><span class="bibliomisc">Aoki, K., Moriguchi, H., Yoshioka, T., Okawa, K., and Tabara, H. (2007). <span class="emphasis"><em>In vitro</em></span> analyses of the production and activity of secondary small interfering RNAs in <span class="emphasis"><em>C. elegans</em></span>. EMBO J. <span class="emphasis"><em>26</em></span>, 5007-5019.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18007599&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed"><span class="bibliomisc">Ashe, A., Sapetschnig, A., Weick, E.M., Mitchell, J., Bagijn, M.P., Cording, A.C., Doebley, A.L., Goldstein, L.D., Lehrbach,
                        N.J., Le Pen, J. et al. (2012). piRNAs can trigger a multigenerational epigenetic memory in the germline of <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>150</em></span>, 88-99.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22738725&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2012.06.018" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed"><span class="bibliomisc">Asikainen, S., Heikkinen, L., Wong, G., and Storvik, M. (2008). Functional characterization of endogenous siRNA target genes
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. BMC Genomics <span class="emphasis"><em>9</em></span>, 270.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18522735&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-9-270" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed"><span class="bibliomisc">Asikainen, S., Storvik, M., Lakso, M., and Wong, G. (2007). Whole genome microarray analysis of <span class="emphasis"><em>C. elegans</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004510;class=Gene" target="_blank">rrf-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001332;class=Gene" target="_blank">eri-1</a></em></span> mutants. FEBS Lett. <span class="emphasis"><em>581</em></span>, 5050-5054.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17919598&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed"><span class="bibliomisc">Avgousti, D.C., Palani, S., Sherman, Y., and Grishok, A. (2012). CSR-1 RNAi pathway positively regulates histone expression
                        in <span class="emphasis"><em>C. elegans</em></span>. EMBO J. <span class="emphasis"><em>31</em></span>, 3821-3832.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22863779&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/emboj.2012.216" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed"><span class="bibliomisc">Bagijn, M.P., Goldstein, L.D., Sapetschnig, A., Weick, E.M., Bouasker, S., Lehrbach, N.J., Simard, M.J., and Miska, E.A. (2012).
                        Function, targets, and evolution of <span class="emphasis"><em>Caenorhabditis elegans</em></span> piRNAs. Science <span class="emphasis"><em>337</em></span>, 574-578.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22700655&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1220952" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed"><span class="bibliomisc">Batista, P.J., Ruby, J.G., Claycomb, J.M., Chiang R., Fahlgren, N., Kasschau, K.D., Chaves, D.A., Gu, W., Vasale, J.J., Duan,
                        S., et al. (2008). PRG-1 and 21U-RNAs interact to form the piRNA complex required for fertility in <span class="emphasis"><em>C. elegans</em></span>. Mol. Cell <span class="emphasis"><em>31</em></span>, 67-78.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18571452&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.molcel.2008.06.002" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed"><span class="bibliomisc">Bean, C.J., Schaner, C.E., and Kelly, W.G., (2004). Meiotic pairing and imprinted X chromatin assembly in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature Genet. <span class="emphasis"><em>36</em></span>, 100-105.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14702046&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed"><span class="bibliomisc">Bernstein, E., Caudy, A.A., Hammond, S.M., and Hannon, G.J. (2001). Role for a bidentate ribonuclease in the initiation step
                        of RNA interference. Nature <span class="emphasis"><em>409</em></span>, 363-366.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11201747&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35053110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed"><span class="bibliomisc">Billi, A.C., Alessi, A.F., Khivansara, V., Han, T., Freeberg, M., and. (2012). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> HEN1 ortholog, HENN-1, methylates and stabilizes select subclasses of germline small RNAs. PLoS Genet. <span class="emphasis"><em>8</em></span>, e1002617.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22548001&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002617" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed"><span class="bibliomisc">Billi, A.C., Freeberg, M.A., Day, A.M., Chun, S.Y., Khivansara, V., and Kim, J.K. (2013). A conserved upstream motif orchestrates
                        autonomous, germline-enriched expression of <span class="emphasis"><em>Caenorhabditis elegans</em></span> piRNAs. PLoS Genet. <span class="emphasis"><em>9</em></span>, e1003392.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23516384&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1003392" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed"><span class="bibliomisc">Blanchard, D., Parameswaran, P., Lopez-Molina, J., Gent, J., Saynuk, J.F., and Fire, A. (2011). On the nature of <span class="emphasis"><em>in vivo</em></span> requirements for <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004326;class=Gene" target="_blank">rde-4</a></em></span> in RNAi and developmental pathways in <span class="emphasis"><em>C. elegans</em></span>. RNA Biol. <span class="emphasis"><em>8</em></span>, 458-467.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21519199&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.4161/rna.8.3.14657" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed"><span class="bibliomisc">Bosher, J.M., Dufourcq, P., Sookhareea, S., and Labouesse, M. (1999). RNA interference can target pre-mRNA, consequences for
                        gene expression in a <span class="emphasis"><em>Caenorhabditis elegans</em></span> operon. Genetics <span class="emphasis"><em>153</em></span>, 1245-1256.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10545456&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed"><span class="bibliomisc">Buckley, B.A., Burkhart, K.B., Gu, S.G., Spracklin, G., Kershner, A., Fritz, H., Kimble, J., Fire, A., and Kenedy, S. (2012).
                        A nuclear Argonaute promotes multigenerational epigenetic inheritance and germline immortality. Nature <span class="emphasis"><em>489</em></span>, 447-451.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22810588&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature11352" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed"><span class="bibliomisc">Burkhart, K.B., Guang, S., Buckley, B.A., Wong, L., Bochner, A.F., and Kennedy, S. (2011). A pre-mRNA-associating factor links
                        endogenous siRNAs to chromatin regulation. PLoS Genet. <span class="emphasis"><em>7</em></span>, e1002249.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21901112&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002249" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed"><span class="bibliomisc">Burton, N.O., Burkhart, K.B., and Kennedy, S. (2011). Nuclear RNAi maintains heritable gene silencing in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>108</em></span>, 19683-19688.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22106253&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.1113310108" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed"><span class="bibliomisc">Cecere, G., Zheng, G.X., Mansisidor, A.R., Klymko, K.E., and Grishok, A. (2012). Promoters recognized by forkhead proteins
                        exist for individual 21U-RNAs. Mol. Cell <span class="emphasis"><em>47</em></span>, 734-745.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22819322&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.molcel.2012.06.021" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed"><span class="bibliomisc">Chen, C.C., Simard, M.J., Tabara, H., Brownell, D.R., McCollough, J.A., and Mello, C.C. (2005). A member of the polymerase
                        &#946; nucleotidyltransferase superfamily is required for RNA interference in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>15</em></span>, 378-383.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15723801&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.01.009" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed"><span class="bibliomisc">Chu, D.S., Liu, H., Nix, P., Wu, T.F., Ralston, E.J., Yates, J.R., 3rd, and Meyer, B.J. (2006). Sperm chromatin proteomics
                        identifies evolutionarily conserved fertility factors. Nature <span class="emphasis"><em>443</em></span>, 101-105.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16943775&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature05050" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed"><span class="bibliomisc">Claycomb, J.M., Batista, P.J., Pang, K.M., Gu, W., Vasale, J.J., van Wolfswinkel, J.C., Chaves, D.A., Shirayama, M., Mitani,
                        S., Ketting, R.F., et al. (2009). The Argonaute CSR-1 and its 22G-RNA cofactors are required for holocentric chromosome segregation.
                        Cell <span class="emphasis"><em>139</em></span>, 123-134.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19804758&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2009.09.014" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed"><span class="bibliomisc">Collins, J., Saari, B., and Anderson, P. (1987). Activation of a transposable element in the germ line but not the soma of
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>328</em></span>, 726-728.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3039378&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed"><span class="bibliomisc">Conine, C.C., Batista, P.J., Gu, W., Claycomb, J.M., Chaves, D.A., Shirayama M., and Mello, C.C. (2010). Argonautes ALG-3
                        and ALG-4 are required for spermatogenesis-specific 26G-RNAs and thermotolerant sperm in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>107</em></span>, 3588-3593.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20133686&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0911685107" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed"><span class="bibliomisc">Corr&ecirc;a, R.L., Steiner, F.A., Berezikov, E., Ketting, R.F. (2010). MicroRNA&#8211;directed siRNA biogenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. PLoS Genet. <span class="emphasis"><em>6</em></span>, e1000903.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20386745&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000903" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed"><span class="bibliomisc">Cox, D.N., Chao, A., Baker, J., Chang, L., Qiao, D., and Lin, H. (1998). A novel class of evolutionarily conserved genes defined
                        by piwi are essential for stem cell self-renewal. Genes Dev. <span class="emphasis"><em>12</em></span>, 3715-3727.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851978&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.12.23.3715" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed"><span class="bibliomisc">Das, P.P., Bagijn, M.P., Goldstein, L.D., Woolford, J.R., Lehrbach, N.J., Sapetschnig, A., Buhecha, H.R., Howe, K.L., Startk,
                        R., Matthews, N., et al. (2008). Piwi and piRNAs act upstream of an endogenous siRNA pathway to suppress Tc3 transposon mobility
                        in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> germline. Mol. Cell <span class="emphasis"><em>31</em></span>, 79-90.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18571451&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.molcel.2008.06.003" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed"><span class="bibliomisc">D&aacute;vila L&oacute;pez, M., and Samuelsson, T. (2008). Early evolution of histone mRNA 3&#8217; end processing. RNA <span class="emphasis"><em>14</em></span>, 1-10.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17998288&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.782308" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed"><span class="bibliomisc">de Wit, E., Linsen, S.E., Cuppen, E., and Berezikov, E. (2009). Repertoire and evolution of miRNA genes in four divergent
                        nematode species. Genome Res. <span class="emphasis"><em>19</em></span>, 2064-2074.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19755563&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.093781.109" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed"><span class="bibliomisc">Duchaine, T.F., Wohlschlegel, J.A., Kennedy, S., Bei, Y., Conte, D. Jr., Pang, K., Brownell, D.R., Harding, S., Mitani, S.,
                        Ruvkun, G., et al. (2006). Functional proteomics reveals the biochemical niche of <span class="emphasis"><em>C. elegans</em></span> DCR-1 in multiple small-RNA-mediated pathways. Cell <span class="emphasis"><em>124</em></span>, 343-354.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16439208&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2005.11.036" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed"><span class="bibliomisc">Elbashir, S.M., Lendeckel, W., and Tuschl, T. (2001). RNA interference is mediated by 21- and 22-nucleotide RNAs. Genes Dev.
                        <span class="emphasis"><em>15</em></span>, 188-200.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11157775&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.862301" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed"><span class="bibliomisc">Emmons, S.W., and Yesner, L. (1984). High-frequency excision of transposable element Tc 1 in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span> is limited to somatic cells. Cell <span class="emphasis"><em>36</em></span>, 599-605.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6321037&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0092-8674(84)90339-8" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed"><span class="bibliomisc">Fernandez, A.G., Gunsalus, K.C., Huang, J., Chuang, L.S., Ying, N., Liang, H.L., Tang, C., Schetter, A.J., Zegar, C., Rual,
                        J.F. et al. (2005). New genes with roles in the <span class="emphasis"><em>C. elegans</em></span> embryo revealed using RNAi of ovary-enriched ORFeome clones. Genome Res. <span class="emphasis"><em>15</em></span>, 250-259.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15687288&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.3194805" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed"><span class="bibliomisc">Fire, A., Albertson, D., Harrison, S.W., and Moerman, D.G. (1991). Production of antisense RNA leads to effective and specific
                        inhibition of gene expression in <span class="emphasis"><em>C. elegans</em></span> muscle. Development <span class="emphasis"><em>113</em></span>, 503-514.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1782862&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed"><span class="bibliomisc">Fire, A., Xu, S., Montgomery, M.K., Kostas, S.A., Driver, S.E., and Mello, C.C. (1998). Potent and specific genetic interference
                        by double-stranded RNA in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>391</em></span>, 806-811.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9486653&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35888" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed"><span class="bibliomisc">Fischer, S.E. (2010). Small RNA-mediated gene silencing pathways in <span class="emphasis"><em>C. elegans</em></span>. Int. J. Biochem. Cell Biol. <span class="emphasis"><em>42</em></span>, 1306-1315.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20227516&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed"><span class="bibliomisc">Fischer, S.E., Butler, M.D., Pan, Q., and Ruvkun, G. (2008). Trans-splicing in <span class="emphasis"><em>C. elegans</em></span> generates the negative RNAi regulator ERI-6/7. Nature <span class="emphasis"><em>455</em></span>, 491-496.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18784652&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature07274" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed"><span class="bibliomisc">Fischer, S.E., Montgomery, T.A., Zhang, C., Fahlgren, N., Breen, P.C., Hwang, A., Sullivan, C.M., Carrington, J.C., and Ruvkun,
                        G. (2011). The ERI-6/7 helicase acts at the first stage of an siRNA amplification pathway that targets recent gene duplications.
                        PLoS Genet. <span class="emphasis"><em>7</em></span>, e1002369.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22102828&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002369" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed"><span class="bibliomisc">Fraser, A.G., Kamath, R.S., Zipperlen, P., Martinez-Campos, M., Sohrmann, M., and Ahringer, J. (2000). Functional genomic
                        analysis of <span class="emphasis"><em>C. elegans</em></span> chromosome I by systematic RNA interference. Nature <span class="emphasis"><em>408</em></span>, 325-330.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11099033&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35042517" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed"><span class="bibliomisc">Gabel, H.W., and Ruvkun, G. (2008). The exonuclease ERI-1 has a conserved dual role in 5.8S rRNA processing and RNAi. Nat.
                        Struct. Mol. Biol. <span class="emphasis"><em>15</em></span>, 531-533.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18438419&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nsmb.1411" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed"><span class="bibliomisc">Gassmann, R., Rechtsteiner, A., Yuen, K.W., Muroyama, A., Egelhofer, T., Gaydos, L., Barron, F., Maddox, P., Essex, A., Monen,
                        J. et al. (2012). An inverse relationship to germline transcription defines centromeric chromatin in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>484</em></span>, 534-537.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22495302&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature10973" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed"><span class="bibliomisc">Gent, J.I., Lamm, A.T., Pavelec, D.M., Maniar, J.M., Parameswaran, P., Tao, L., Kennedy, S., and Fire, A.Z. (2010). Distinct
                        phases of siRNA synthesis in an endogenous RNAi pathway in <span class="emphasis"><em>C. elegans</em></span> soma. Mol. Cell <span class="emphasis"><em>37</em></span>, 679-689.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20116306&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.molcel.2010.01.012" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed"><span class="bibliomisc">Gent, J.I., Schvarzstein, M., Villeneuve, A.M., Gu, S.G., Jantsch, V., Fire, A.Z., and Baudrimont, A. (2009). A <span class="emphasis"><em>Caenorhabditis elegans</em></span> RNA-directed RNA polymerase in sperm development and endogenous RNA interference. Genetics <span class="emphasis"><em>183</em></span>, 1297-1314.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19805814&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.109.109686" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed"><span class="bibliomisc">Grishok, A., Pasquinelli, A.E., Conte, D., Li, N., Parrish, S., Ha, I., Baillie, D.L., Fire, A., Ruvkun, G., and Mello, C.C.
                        (2001). Genes and mechanisms related to RNA interference regulate expression of the small temporal RNAs that control <span class="emphasis"><em>C. elegans</em></span> developmental timing. Cell <span class="emphasis"><em>106</em></span>, 23-34.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11641272&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed"><span class="bibliomisc">Grishok, A., Sinskey, J.L., and Sharp, P.A. (2005). Transcriptional silencing of a transgene by RNAi in the soma of <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>19</em></span>, 683-696.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15741313&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1247705" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed"><span class="bibliomisc">Grishok, A., Tabara, H., and Mello, C.C. (2000). Genetic requirements for inheritance of RNAi in <span class="emphasis"><em>C. elegans</em></span>. Science 287, 2494-2497.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10741970&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.287.5462.2494" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed"><span class="bibliomisc">Gu, S.G., Pak, J., Guang, S., Maniar, J.M., Kennedy, S., and Fire, A. (2012). Amplification of siRNA in <span class="emphasis"><em>Caenorhabditis elegans</em></span> generates a transgenerational sequence-targeted histone H3 lysine 9 methylation footprint. Nature Genet. <span class="emphasis"><em>44</em></span>, 157-164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22231482&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng.1039" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed"><span class="bibliomisc">Gu, W., Lee, H.C., Chaves, D., Youngman, E.M., Pazour, G.J., Conte, D., Jr., and Mello, C.C. (2012). CapSeq and CIP-TAP identify
                        Pol II start sites and reveal capped small RNAs as <span class="emphasis"><em>C. elegans</em></span> piRNA precursors. Cell <span class="emphasis"><em>151</em></span>, 1488-1500.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23260138&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2012.11.023" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed"><span class="bibliomisc">Gu, W., Shirayama, M., Conte, D., Jr., Vasale, J., Batista, P.J., Claycomb, J.M., Moresco, J.J., Youngman, E.M., Keys, J.,
                        Stoltz, M.J., et al. (2009). Distinct argonaute-mediated 22G-RNA pathways direct genome surveillance in the <span class="emphasis"><em>C. elegans</em></span> germline. Mol. Cell <span class="emphasis"><em>36</em></span>, 231-244.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19800275&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.molcel.2009.09.020" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed"><span class="bibliomisc">Guang, S., Bochner, A.F., Burkhart, K.B., Burton, N., Pavelec, D.M., and Kennedy, S. (2010). Small regulatory RNAs inhibit
                        RNA polymerase II during the elongation phase of transcription. Nature <span class="emphasis"><em>465</em></span>, 1097-1101.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20543824&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature09095" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed"><span class="bibliomisc">Guang, S., Bochner, A.F., Pavelec, D.M., Burkhart, K.B., Harding, S., Lachowiec, J., and Kennedy, S. (2008). An Argonaute
                        transports siRNAs from the cytoplasm to the nucleus. Science <span class="emphasis"><em>321</em></span>, 537-541.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18653886&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1157647" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed"><span class="bibliomisc">Han, T., Manoharan, A.P., Harkins, T.T., Bouffard, P., Fitzpatrick, C., Fitzpatrick, C., Chu, D.S., Thierry-Mieg, D., Thierry-Mieg,
                        J., and Kim, J.K. (2009). 26G endo-siRNAs regulate spermatogenic and zygotic gene expression in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>106</em></span>, 18674-18679.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19846761&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0906378106" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed"><span class="bibliomisc">Han, W., Sundaram, P., Kenjale, H., Grantham, J., and Timmons, L. (2008). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004681;class=Gene" target="_blank">rsd-2</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004684;class=Gene" target="_blank">rsd-6</a></em></span> genes are required for chromosome functions during exposure to unfavorable environments. Genetics <span class="emphasis"><em>178</em></span>, 1875-1893.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18430922&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.107.085472" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed"><span class="bibliomisc">Heo, I., Joo, C., Cho, J., Ha, M., Han, J., and Kim, V.N. (2008). Lin28 mediates the terminal uridylation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002285;class=Gene" target="_blank">let-7</a></em></span> precursor MicroRNA. Mol. Cell <span class="emphasis"><em>32</em></span>, 276-284.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18951094&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.molcel.2008.09.014" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed"><span class="bibliomisc">Hunt-Newbury, R., Viveiros, R., Johnsen, R., Mah, A., Anastas, D., Fang, L., Halfnight, E., Lee, D., Lin, J., Lorch, A. et
                        al. (2007). High-throughput <span class="emphasis"><em>in viv</em></span>o analysis of gene expression in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. PLoS Biol. <span class="emphasis"><em>5</em></span>, e237.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17850180&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0050237" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed"><span class="bibliomisc">Ideue, T. Hino, K., Kitao, S., Yokoi, T., and Hirose, T. (2009). Efficient oligonucleotide-mediated degradation of nuclear
                        noncoding RNAs in mammalian cultured cells. RNA <span class="emphasis"><em>15</em></span>, 1578-1587.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19535462&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.1657609" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed"><span class="bibliomisc">Kamath, R.S., Fraser, A.G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., Kanapin, A., Le Bot, N., Moreno, S., Sohrmann, M.
                        et al. (2003). Systematic functional analysis of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome using RNAi. Nature <span class="emphasis"><em>421</em></span>, 231-237.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12529635&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature01278" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed"><span class="bibliomisc">Kamminga, L.M., van Wolfswinkel, J.C., Luteijn, M.J., Kaaij, L.J., Bagijn, M.P., Sapetschnig, W., Miska,E.A., Berezikov, E.,
                        and Ketting, R.F. (2012). Differential impact of the HEN1 homolog HENN-1 on 21U and 26G RNAs in the germline of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. PLoS Genet. <span class="emphasis"><em>8</em></span>, e1002702.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22829772&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002702" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed"><span class="bibliomisc">Kato, M., de Lencastre, A., Pincus, Z., and Slack, F.J. (2009). Dynamic expression of small non-coding RNAs, including novel
                        microRNAs and piRNAs/21U-RNAs, during <span class="emphasis"><em>Caenorhabditis elegans</em></span> development. Genome Biol. <span class="emphasis"><em>10</em></span>, R54.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19460142&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2009-10-5-r54" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed"><span class="bibliomisc">Kawaoka, S., Izumi, N., Katsuma, S., and Tomari, Y. (2011). 3&#8217; end formation of PIWI-interacting RNAs <span class="emphasis"><em>in vitro</em></span>. Mol. Cell <span class="emphasis"><em>43</em></span>, 1015-1022.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21925389&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.molcel.2011.07.029" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed"><span class="bibliomisc">Kelly, W.G., and Fire., A. (1998). Chromatin silencing and the maintenance of a functional germline in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>125</em></span>, 2451-2456.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9609828&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed"><span class="bibliomisc">Kelly, W.G., Schaner, C.E., Dernburg, A.F., Lee, M.H., Kim, S.K., Villeneuve, A.M., and Reinke, V. (2002). X-chromosome silencing
                        in the germline of <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>129</em></span>, 479-492.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11807039&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed"><span class="bibliomisc">Kennedy, S., Wang, D., and Ruvkun, G. (2004). A conserved siRNA-degrading RNase negatively regulates RNA interference in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>427</em></span>, 645-649.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14961122&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02302" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed"><span class="bibliomisc">Ketting, R.F., and Plasterk, R.H. (2000). A genetic link between co-suppression and RNA interference in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>404</em></span>, 296-298.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10749214&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35005113" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed"><span class="bibliomisc">Ketting, R.F., Fischer, S.E., Bernstein, E., Sijen, T., Hannon, G.J., and Plasterk, R.H. (2001). Dicer functions in RNA interference
                        and in synthesis of small RNA involved in developmental timing in <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>15</em></span>, 2654-2659.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11641272&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.927801" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed"><span class="bibliomisc">Ketting, R.F., Haverkamp, T.H., van Luenen, H.G., and Plasterk, R.H. (1999). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003504;class=Gene" target="_blank">mut-7</a></em></span> of <span class="emphasis"><em>C. elegans</em></span>, required for transposon silencing and RNA interference, is a homolog of Werner syndrome helicase and RNaseD. Cell <span class="emphasis"><em>99</em></span>, 133-141.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10535732&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0092-8674(00)81645-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed"><span class="bibliomisc">Kim, J.K., Gabel, H.W., Kamath, R.S., Tewari, M., Pasquinelli, A., Rual, J.F., Kennedy, S., Dybbs, M., Bertin, N., Kaplan,
                        J.M. et al. (2005). Functional genomic analysis of RNA interference in <span class="emphasis"><em>C. elegans</em></span>. Science <span class="emphasis"><em>308</em></span>, 1164-1167.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15790806&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1109267" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed"><span class="bibliomisc">Knight, S.W., and Bass, B.L. (2001). A role for the RNase III enzyme DCR-1 in RNA interference and germ line development in
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>293</em></span>, 2269-2271.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11486053&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1062039" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed"><span class="bibliomisc">Kodama, Y., Rothman, J.H., Sugimoto, A., and Yamamoto, M. (2002). The stem-loop binding protein CDL-1 is required for chromosome
                        condensation, progression of cell death and morphogenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>129</em></span>, 187-196.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11782412&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed"><span class="bibliomisc">Kwak, J.E., and Wickens, M. (2007). A family of poly(U) polymerases. RNA <span class="emphasis"><em>13</em></span>, 860-867.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17449726&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.514007" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed"><span class="bibliomisc">Lau, N.C., Lim, L.P., Weinstein, E.G., and Bartel, D.P. (2001), An abundant class of tiny RNAs with probable regulatory roles
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>294</em></span>, 858-862.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11679671&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1065062" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib72"></a><p class="bibliomixed"><span class="bibliomisc">Lee, H.C., Gu, W., Shirayama, M., Youngman, E., Conte, D., Jr., and Mello, C.C. (2012). <span class="emphasis"><em>C. elegans</em></span> piRNAs mediate the genome-wide surveillance of germline transcripts. Cell <span class="emphasis"><em>150</em></span>, 78-87.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22738724&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2012.06.016" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib73"></a><p class="bibliomixed"><span class="bibliomisc">Lee, R., Feinbaum, R., and Ambros, V. (1993). The <span class="emphasis"><em>C. elegans</em></span> heterochronic gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002993;class=Gene" target="_blank">lin-4</a></em></span> encodes small RNAs with antisense complementarity to <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003003;class=Gene" target="_blank">lin-14</a></em></span>. Cell <span class="emphasis"><em>75</em></span>, 843-854.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8252621&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0092-8674(93)90529-Y" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib74"></a><p class="bibliomixed"><span class="bibliomisc">Lee, R.C., and Ambros, V. (2001). An extensive class of small RNAs in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>294</em></span>, 862-864.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11679672&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1065329" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib75"></a><p class="bibliomixed"><span class="bibliomisc">Lee, R.C., Hammell, C.M., and Ambros, V. (2006). Interacting endogenous and exogenous RNAi pathways in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. RNA <span class="emphasis"><em>12</em></span>, 589-597.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16489184&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.2231506" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib76"></a><p class="bibliomixed"><span class="bibliomisc">Lehner, B., Crombie, C., Tischler, J., Fortunato, A., and Fraser, A.G. (2006). Systematic mapping of genetic interactions
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span> identifies common modifiers of diverse signaling pathways. Nature Genet. <span class="emphasis"><em>38</em></span>, 896-903.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16845399&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1844" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib77"></a><p class="bibliomixed"><span class="bibliomisc">Lim, L.P., Lau, N.C., Weinstein, E.G., Abdelhakim, A., Yekta, S., Rhoades, M.W., Burge, C.B., and Bartel, D.P. (2003). The
                        microRNAs of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genes Dev. <span class="emphasis"><em>17</em></span>, 991-1008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12672692&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1074403" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib78"></a><p class="bibliomixed"><span class="bibliomisc">Luteijn, M.J., van Bergeijk, P., Kaaij, L.J., Almeida, M.V., Roovers, E.F., Berezikov, E. and Ketting, R.F. (2012). Extremely
                        stable Piwi-induced gene silencing in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. EMBO J. <span class="emphasis"><em>31</em></span>, 3422-3430.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22850670&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/emboj.2012.213" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib79"></a><p class="bibliomixed"><span class="bibliomisc">Maeda, I., Kohara, Y., Yamamoto, M., and Sugimoto, A. (2001). Large-scale analysis of gene function in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by high-throughput RNAi. Curr. Biol. <span class="emphasis"><em>11</em></span>, 171-176.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11231151&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0960-9822(01)00052-5" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib80"></a><p class="bibliomixed"><span class="bibliomisc">Maine, E.M., Hauth, J., Ratliff, T., Vought, V.E., She, X., and Kelly, W.G. (2005). EGO-1, a putative RNA-dependent RNA polymerase,
                        is required for heterochromatin assembly on unpaired DNA during <span class="emphasis"><em>C. elegans</em></span> meiosis. Curr. Biol. <span class="emphasis"><em>15</em></span>, 1972-1978.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16271877&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2005.09.049" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib81"></a><p class="bibliomixed"><span class="bibliomisc">Makeyev, E.V., and Bamford, D.H. (2002). Cellular RNA-dependent RNA polymerase involved in posttranscriptional gene silencing
                        has two distinct activity modes. Mol. Cell <span class="emphasis"><em>10</em></span>, 1417-1427.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12504016&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S1097-2765(02)00780-3" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib82"></a><p class="bibliomixed"><span class="bibliomisc">Mangone, M., Manoharan, A.P., Thierry-Mieg, D., Thierry-Mieg, J., Han, T., Mackowiak, S.D., Mis, E., Zegar, C., Gutwein, M.R.,
                        Khivansara, V. et al. (2010). The landscape of <span class="emphasis"><em>C. elegans</em></span> 3'UTRs. Science <span class="emphasis"><em>329</em></span>, 432-435.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20522740&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1191244" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib83"></a><p class="bibliomixed"><span class="bibliomisc">Marzluff, W.F., Wagner, E.J., and Duronio, R.J. (2008). Metabolism and regulation of canonical histone mRNAs, life without
                        a poly(A) tail. Nat. Rev. Genet. <span class="emphasis"><em>9</em></span>, 843-854.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18927579&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg2438" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib84"></a><p class="bibliomixed"><span class="bibliomisc">McGhee, J.D. The <span class="emphasis"><em>C. elegans</em></span> intestine (March 27, 2007), WormBook, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.133.1, http://www.wormbook.org. </span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib85"></a><p class="bibliomixed"><span class="bibliomisc">Montgomery, M.K., and Fire., A. (1998). Double-stranded RNA as a mediator in sequence-specific genetic silencing and co-suppression.
                        Trends Genet. <span class="emphasis"><em>14</em></span>, 255-258.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9676523&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35078107" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib86"></a><p class="bibliomixed"><span class="bibliomisc">Montgomery, M.K., Xu, S., and Fire., A. (1998). RNA as a target of double-stranded RNA-mediated genetic interference in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>95</em></span>, 15502-15507.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9860998&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.95.26.15502" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib87"></a><p class="bibliomixed"><span class="bibliomisc">Montgomery, T.A., Rim, Y.S., Zhang, C., Dowen, R.H., Phillips, C.M., Fischer, S.E., and Ruvkun, G. (2012). PIWI associated
                        siRNAs and piRNAs specifically require the <span class="emphasis"><em>Caenorhabditis elegans</em></span> HEN1 ortholog <a href="http://www.wormbase.org/db/get?name=WBGene00015349;class=Gene" target="_blank">henn-1</a>. PLoS Genet. <span class="emphasis"><em>8</em></span>, e1002616.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22536158&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002616" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib88"></a><p class="bibliomixed"><span class="bibliomisc">Moore, M.J., and Proudfoot, N.J. (2009). Pre-mRNA processing reaches back to transcription and ahead to translation. Cell
                        <span class="emphasis"><em>136</em></span>, 688-700.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19239889&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2009.02.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib89"></a><p class="bibliomixed"><span class="bibliomisc">Nakamura, M., Ando, R., Nakazawa, T., Yudazono, T., Tsutsumi, N., Hatanaka, N., Ohgake, T., Hanaoka, F., and Eki, T. (2007).
                        Dicer-related <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00008400;class=Gene" target="_blank">drh-3</a></em></span> gene functions in germ-line development by maintenance of chromosomal integrity in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genes Cells <span class="emphasis"><em>12</em></span>, 997-1010.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17825044&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1111/j.1365-2443.2007.01111.x" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib90"></a><p class="bibliomixed"><span class="bibliomisc">Nechaev, S., Fargo, D.C., dos Santos, G., Liu, L., Gao, Y., and Adelman, K. (2010). Global analysis of short RNAs reveals
                        widespread promoter-proximal stalling and arrest of Pol II in <span class="emphasis"><em>Drosophila</em></span>. Science <span class="emphasis"><em>327</em></span>, 335-338.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20007866&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1181421" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib91"></a><p class="bibliomixed"><span class="bibliomisc">Pak, J., and Fire, A. (2007). Distinct populations of primary and secondary effectors during RNAi in <span class="emphasis"><em>C. elegans</em></span>. Science <span class="emphasis"><em>315</em></span>, 241-244.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17124291&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/ 10.1126/science.1132839" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib92"></a><p class="bibliomixed"><span class="bibliomisc">Park, W., Li, J., Song, R., Messing, J., and Chen, X. (2002). CARPEL FACTORY, a Dicer homolog, and HEN1, a novel protein,
                        act in microRNA metabolism in Arabidopsis thaliana. Curr. Biol. <span class="emphasis"><em>12</em></span>, 1484-1495.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12225663&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0960-9822(02)01017-5" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib93"></a><p class="bibliomixed"><span class="bibliomisc">Parker, G.S., Eckert, D.M., and Bass, B.L. (2006). RDE-4 preferentially binds long dsRNA and its dimerization is necessary
                        for cleavage of dsRNA to siRNA. RNA <span class="emphasis"><em>12</em></span>, 807-818.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16603715&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.2338706" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib94"></a><p class="bibliomixed"><span class="bibliomisc">Parker, G.S., Maity, T.S., and Bass, B.L. (2008). dsRNA binding properties of RDE-4 and TRBP reflect their distinct roles
                        in RNAi. J. Mol. Biol. <span class="emphasis"><em>384</em></span>, 967-979.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18948111&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.jmb.2008.10.002" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib95"></a><p class="bibliomixed"><span class="bibliomisc">Parrish, S., and Fire, A. (2001). Distinct roles for RDE-1 and RDE-4 during RNA interference in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. RNA <span class="emphasis"><em>7</em></span>, 1397-1402.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11680844&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib96"></a><p class="bibliomixed"><span class="bibliomisc">Pavelec, D.M., Lachowiec, J., Duchaine, T.F., Smith, H.E., and Kennedy, S. (2009). Requirement for the ERI/DICER complex in
                        endogenous RNA interference and sperm development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>183</em></span>, 1283-1295.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19797044&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.109.108134" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib97"></a><p class="bibliomixed"><span class="bibliomisc">Pettitt, J., Crombie, C., Schumperli, D., Muller, B. (2002). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> histone hairpin-binding protein is required for core histone gene expression and is essential for embryonic and postembryonic
                        cell division. J. Cell Sci. <span class="emphasis"><em>115</em></span>, 857-866.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11865041&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib98"></a><p class="bibliomixed"><span class="bibliomisc">Phillips, C.M., Montgomery, T.A., Breen, P.C., and Ruvkun, G. (2012). MUT-16 promotes formation of perinuclear mutator foci
                        required for RNA silencing in the <span class="emphasis"><em>C. elegans</em></span> germline. Genes Dev. <span class="emphasis"><em>26</em></span>, 1433-1444.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22713602&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.193904.112" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib99"></a><p class="bibliomixed"><span class="bibliomisc">Pitt, J.N., Schisa, J.A., and Priess, J.R. (2000). P granules in the germ cells of <span class="emphasis"><em>Caenorhabditis elegans</em></span> adults are associated with clusters of nuclear pores and contain RNA. Dev. Biol. <span class="emphasis"><em>219</em></span>, 315-333.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10694425&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1006/dbio.2000.9607" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib100"></a><p class="bibliomixed"><span class="bibliomisc">Qiao, L., Lissemore, J.L., Shu, P., Smardon, A., Gelber, M.B., and Maine E.M. (1995). Enhancers of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001609;class=Gene" target="_blank">glp-1</a></em></span>, a gene required for cell-signaling in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, define a set of genes required for germline development. Genetics <span class="emphasis"><em>141</em></span>, 551-569.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8647392&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib101"></a><p class="bibliomixed"><span class="bibliomisc">Rasmussen, E.B., and Lis, J.T. (1995). Short transcripts of the ternary complex provide insight into RNA polymerase II elongational
                        pausing. J. Mol. Biol. <span class="emphasis"><em>252</em></span>, 522-535.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7563071&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib102"></a><p class="bibliomixed"><span class="bibliomisc">Rechtsteiner, A., Ercan, S., Takasaki, T., Phippen, T.M., Egelhofer, T.A., Wang, W., Kimura, H., Lieb, J.D., and Strome, S.
                        (2010). The histone H3K36 methyltransferase MES-4 acts epigenetically to transmit the memory of germline gene expression to
                        progeny. PLoS Genet. <span class="emphasis"><em>6</em></span>, e1001091.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20824077&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1001091" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib103"></a><p class="bibliomixed"><span class="bibliomisc">Reinhart, B.J., Slack, F.J., Basson, M., Pasquinelli, A.E., Bettinger, J.C., Rougvie, A.E., Horvitz, H.R., and Ruvkun, G.
                        (2000). The 21-nucleotide <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002285;class=Gene" target="_blank">let-7</a></em></span> RNA regulates developmental timing in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>403</em></span>, 901-906.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10706289&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35002607" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib104"></a><p class="bibliomixed"><span class="bibliomisc">Reinke, V., Gil, I.S., Ward, S., and Kazmer, K. (2004). Genome-wide germline-enriched and sex-biased expression profiles in
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>131</em></span>, 311-323.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14668411&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.00914" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib105"></a><p class="bibliomixed"><span class="bibliomisc">Robert, V.J., Sijen, T., van Wolfswinkel, J., and Plasterk, R.H. (2005). Chromatin and RNAi factors protect the <span class="emphasis"><em>C. elegans</em></span> germline against repetitive sequences. Genes Dev <span class="emphasis"><em>19</em></span>, 782-787.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15774721&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.332305" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib106"></a><p class="bibliomixed"><span class="bibliomisc">Rocheleau, C.E., Cullison, K., Huang, K., Bernstein, Y., Spilker, A.C., and Sundaram, M.V. (2008). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em>ekl</em></span> (enhancer of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002239;class=Gene" target="_blank">ksr-1</a></em></span> lethality). genes include putative components of a germline small RNA pathway. Genetics <span class="emphasis"><em>178</em></span>, 1431-1443.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18245826&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.107.084608" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib107"></a><p class="bibliomixed"><span class="bibliomisc">Rual, J.F., Ceron, J., Koreth, J., Hao, T., Nicot, A.S., Hirozane-Kishikawa, T., Vandenhaute, J., Orkin, S.H., Hill, D.E.,
                        van den Heuvel, S., and Vidal, M. (2004). Toward improving <span class="emphasis"><em>Caenorhabditis elegans</em></span> phenome mapping with an ORFeome-based RNAi library. Genome Res. <span class="emphasis"><em>14</em></span>, 2162-2168.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15489339&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.2505604" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib108"></a><p class="bibliomixed"><span class="bibliomisc">Ruby, J.G., Jan, C., Player, C., Axtell, M.J., Lee, W., Nusbaum, C., Ge, H., and Bartel, D.P. (2006). Large-scale sequencing
                        reveals 21U-RNAs and additional microRNAs and endogenous siRNAs in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>127</em></span>, 1193-1207.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17174894&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2006.10.040" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib109"></a><p class="bibliomixed"><span class="bibliomisc">Schrimpf, S.P., Weiss, M., Reiter, L., Ahrens, C.H., Jovanovic, M., Malmstrom, J., Brunner, E., Mohanty, S., Lercher, M.J.,
                        Hunziker, P.E., et al. (2009). Comparative functional analysis of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Drosophila melanogaster</em></span> proteomes. PLoS Biol. <span class="emphasis"><em>7</em></span>, e48.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19260763&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.1000048" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib110"></a><p class="bibliomixed"><span class="bibliomisc">Schwarz, D.S., Tomari, Y., and Zamore, P.D. (2004). The RNA-induced silencing complex is a Mg2+-dependent endonuclease. Curr.
                        Biol. <span class="emphasis"><em>14</em></span>, 787-791.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15120070&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2004.03.008" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib111"></a><p class="bibliomixed"><span class="bibliomisc">She, X., Xu, X., Fedotov, A., Kelly, W.G., and Maine, E.M. (2009). Regulation of heterochromatin assembly on unpaired chromosomes
                        during <span class="emphasis"><em>Caenorhabditis elegans</em></span> meiosis by components of a small RNA-mediated pathway. PLoS Genet. <span class="emphasis"><em>5</em></span>, e1000624.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19714217&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/ 10.1371/journal.pgen.1000624" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib112"></a><p class="bibliomixed"><span class="bibliomisc">Sheth, U., Pitt, J., Dennis, S., and Priess, J.R. (2010). Perinuclear P granules are the principal sites of mRNA export in
                        adult <span class="emphasis"><em>C. elegans</em></span> germ cells. Development <span class="emphasis"><em>137</em></span>, 1305-1314.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20223759&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.044255" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib113"></a><p class="bibliomixed"><span class="bibliomisc">Shi, Z., Montgomery, T.A., Qi, Y., and Ruvkun, G. (2013). High-throughput sequencing reveals extraordinary fluidity of miRNA,
                        piRNA, and siRNA pathways in nematodes. Genome Res. <span class="emphasis"><em>23</em></span>, 497-508.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=23363624&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.149112.112" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib114"></a><p class="bibliomixed"><span class="bibliomisc">Shirayama, M., Seth, M., Lee, H.C., Gu, W., Ishidate, T., Conte, D. Jr., and Mello, C.C. (2012). piRNAs initiate an epigenetic
                        memory of nonself RNA in the <span class="emphasis"><em>C. elegans</em></span> germline. Cell <span class="emphasis"><em>150</em></span>, 65-77.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22738726&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2012.06.015" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib115"></a><p class="bibliomixed"><span class="bibliomisc">Sijen, T., Fleenor, J., Simmer, F., Thijssen, K.L., Parrish, S., Timmons, L., Plasterk, R.H., and Fire, A. (2001). On the
                        role of RNA amplification in dsRNA-triggered gene silencing. Cell <span class="emphasis"><em>107</em></span>, 465-476.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11719187&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0092-8674(01)00576-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib116"></a><p class="bibliomixed"><span class="bibliomisc">Sijen, T., Steiner, F.A., Thijssen, K.L., and Plasterk, R.H. (2007). Secondary siRNAs result from unprimed RNA synthesis and
                        form a distinct class. Science <span class="emphasis"><em>315</em></span>, 244-247.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17158288&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1136699" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib117"></a><p class="bibliomixed"><span class="bibliomisc">Simmer, F., Moorman, C., van der Linden, A.M., Kuijk, E., van den Berghe, P.V., Kamath, R.S., Fraser, A.g., Ahringer, J.,
                        and Plasterk, R.H. (2003). Genome-wide RNAi of <span class="emphasis"><em>C. elegans</em></span> using the hypersensitive <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004510;class=Gene" target="_blank">rrf-3</a></em></span> strain reveals novel gene functions. PLoS Biol. <span class="emphasis"><em>1</em></span>, e12.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14551910&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0000012" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib118"></a><p class="bibliomixed"><span class="bibliomisc">Simmer, F., Tijsterman, M., Parrish, S., Koushika, S.P., Nonet, M.L., Fire, A., Ahringer, J., and Plasterk, R.H. (2002). Loss
                        of the putative RNA-directed RNA polymerase RRF-3 makes <span class="emphasis"><em>C. elegans</em></span> hypersensitive to RNAi. Curr. Biol. <span class="emphasis"><em>12</em></span>, 1317-1319.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12176360&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0960-9822(02)01041-2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib119"></a><p class="bibliomixed"><span class="bibliomisc">Smardon, A., Spoerke, J.M., Stacey, S.C., Klein, M.E., Mackin, N., and Maine, E.M. (2000). EGO-1 is related to RNA-directed
                        RNA polymerase and functions in germ-line development and RNA interference in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>10</em></span>, 169-178.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10704412&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib120"></a><p class="bibliomixed"><span class="bibliomisc">Sonnichsen, B., Koski, L.B., Walsh, A., Marschall, P., Neumann, B., Brehm, M., Alleaume, A.M., Artelt, J., Bettencourt, P.,
                        Cassin, E. et al. (2005). Full-genome RNAi profiling of early embryogenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>434</em></span>, 462-469.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15791247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature03353" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib121"></a><p class="bibliomixed"><span class="bibliomisc">Steiner, F.A., Hoogstrate, S.W., Okihara, K.L., Thijssen, K.L., Ketting, R.F., Ketting, R.F., Plasterk, R.H., and Sijen, T.
                        (2007). Structural features of small RNA precursors determine Argonaute loading in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Struct. Mol. Biol.<span class="emphasis"><em>14</em></span>, 927-933.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17891148&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nsmb1308" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib122"></a><p class="bibliomixed"><span class="bibliomisc">Steiner, F.A., Okihara, K.L., Hoogstrate, S.W., Sijen, T., and Ketting, R.F. (2009). RDE-1 slicer activity is required only
                        for passenger-strand cleavage during RNAi in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Struct. Mol. Biol.<span class="emphasis"><em>16</em></span>, 207-211.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19151723&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nsmb.1541" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib123"></a><p class="bibliomixed"><span class="bibliomisc">Stoeckius, M., Maaskola, J., Colombo, T., Rahn, H.P., Friedlander, M.R., Li, N., Chen, W., Piano, F., and Rajewsky, N. (2009).
                        Large-scale sorting of <span class="emphasis"><em>C. elegans</em></span> embryos reveals the dynamics of small RNA expression. Nat. Methods <span class="emphasis"><em>6</em></span>, 745-751.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19734907&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1370" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib124"></a><p class="bibliomixed"><span class="bibliomisc">Sundaram, P., Echalier, B., Han, W., Hull, D., and Timmons, L. (2006). ATP-binding cassette transporters are required for
                        efficient RNA interference in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Biol. Cell <span class="emphasis"><em>17</em></span>, 3678-3688.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16723499&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1091/mbc.E06-03-0192" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib125"></a><p class="bibliomixed"><span class="bibliomisc">Tabara, H., Sarkissian, M., Kelly, W.G., Fleenor, J., Grishok, A., Timmons, L., Fire, A., and Mello, C.C. (1999). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00004323;class=Gene" target="_blank">rde-1</a></em></span> gene, RNA interference, and transposon silencing in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>99</em></span>, 123-132.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10535731&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0092-8674(00)81644-X" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib126"></a><p class="bibliomixed"><span class="bibliomisc">Tabara, H., Yigit, E., Siomi, H., and Mello, C.C. (2002). The dsRNA binding protein RDE-4 interacts with RDE-1, DCR-1, and
                        a DExH-box helicase to direct RNAi in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>109</em></span>, 861-871.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12110183&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0092-8674(02)00793-6" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib127"></a><p class="bibliomixed"><span class="bibliomisc">Thivierge, C., Makil, N., Flamand, M., Vasale, J.J., Mello, C.C., Wohlschlegel, J., Conte, D. Jr., and Duchaine, T.F. (2012).
                        Tudor domain ERI-5 tethers an RNA-dependent RNA polymerase to DCR-1 to potentiate endo-RNAi. Nat. Struct. Mol. Biol. <span class="emphasis"><em>19</em></span>, 90-97.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22179787&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nsmb.2186" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib128"></a><p class="bibliomixed"><span class="bibliomisc">Tijsterman, M., Ketting, R.F., Okihara, K.L., Sijen, T., and Plasterk, R.H. (2002a). RNA helicase MUT-14-dependent gene silencing
                        triggered in <span class="emphasis"><em>C. elegans</em></span> by short antisense RNAs. Science <span class="emphasis"><em>295</em></span>, 694-697.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11809977&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1067534" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib129"></a><p class="bibliomixed"><span class="bibliomisc">Tijsterman, M., May, R.C., Simmer, F., Okihara, K.L. and Plasterk, R.H. (2004). Genes required for systemic RNA interference
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Curr. Biol. <span class="emphasis"><em>14</em></span>, 111-116.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14738731&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2003.12.029" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib130"></a><p class="bibliomixed"><span class="bibliomisc">Tijsterman, M., Okihara, K.L., Thijssen, K., and Plasterk, R.H. (2002b). PPW-1, a PAZ/PIWI protein required for efficient
                        germline RNAi, is defective in a natural isolate of <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>12</em></span>, 1535-1540.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12225671&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0960-9822(02)01110-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib131"></a><p class="bibliomixed"><span class="bibliomisc">Tops, B.B., Tabara, H., Sijen, T., Simmer, F., Mello, C.C., Plasterk, R.H., and Ketting, R.F. (2005). RDE-2 interacts with
                        MUT-7 to mediate RNA interference in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>33</em></span>, 347-355.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15653635&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gki183" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib132"></a><p class="bibliomixed"><span class="bibliomisc">van Wolfswinkel, J.C., Claycomb, J.M., Batista, P.J., Mello, C.C., Berezikov, E., and Ketting, R.F. (2009). CDE-1 affects
                        chromosome segregation through uridylation of CSR-1-bound siRNAs. Cell <span class="emphasis"><em>139</em></span>, 135-148.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19804759&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2009.09.012" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib133"></a><p class="bibliomixed"><span class="bibliomisc">Vasale, J.J., Gu, W., Thivierge, C., Batista, P.J., Claycomb, J.M., Youngman, E.M., Duchaine, T.F., Mello, C.C., and Conte,
                        D. Jr. (2010). Sequential rounds of RNA-dependent RNA transcription drive endogenous small-RNA biogenesis in the ERGO-1/Argonaute
                        pathway. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>107</em></span>, 3582-3587.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20133583&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0911908107" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib134"></a><p class="bibliomixed"><span class="bibliomisc">Vastenhouw, N.L., Fischer, S.E., Robert, V.J., Thijssen, K.L., Fraser, A.G., Kamath, R.S., Ahringer, J., and Plasterk, R.H.
                        (2003). A genome-wide screen identifies 27 genes involved in transposon silencing in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>13</em></span>, 1311-1316.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12906791&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0960-9822(03)00539-6" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib135"></a><p class="bibliomixed"><span class="bibliomisc">Vasu, S.K., and Forbes, D.J. (2001). Nuclear pores and nuclear assembly. Curr. Opin. Cell Biol. <span class="emphasis"><em>13</em></span>, 363-375.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11343909&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0955-0674(00)00221-0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib136"></a><p class="bibliomixed"><span class="bibliomisc">Vought, V.E., Ohmachi, M., Lee, M.H., and Maine, E.M. (2005). EGO-1, a putative RNA-directed RNA polymerase, promotes germline
                        proliferation in parallel with GLP-1/Notch signaling and regulates the spatial organization of nuclear pore complexes and
                        germline P granules in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>170</em></span>, 1121-1132.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15911573&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.105.042135" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib137"></a><p class="bibliomixed"><span class="bibliomisc">Wagner, E.J., Berkow, A., and Marzluff, W.F. (2005). Expression of an RNAi-resistant SLBP restores proper S-phase progression.
                        Biochem. Soc. Trans. <span class="emphasis"><em>33</em></span>, 471-473.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15916543&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib138"></a><p class="bibliomixed"><span class="bibliomisc">Wang, G., and Reinke, V. (2008). A <span class="emphasis"><em>C. elegans</em></span> Piwi, PRG-1, regulates 21U-RNAs during spermatogenesis. Curr. Biol. <span class="emphasis"><em>18</em></span>, 861-867.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18501605&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2008.05.009" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib139"></a><p class="bibliomixed"><span class="bibliomisc">Warf, M.B., Shepherd, B.A., Johnson, W.E., and Bass, B.L. (2012). Effects of ADARs on small RNA processing pathways in <span class="emphasis"><em>C. elegans</em></span>. Genome Res. <span class="emphasis"><em>22</em></span>, 1488-1498.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22673872&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.134841.111" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib140"></a><p class="bibliomixed"><span class="bibliomisc">Welker, N.C., Habig, J.W., and Bass, B.L. (2007). Genes misregulated in <span class="emphasis"><em>C. elegans</em></span> deficient in Dicer, RDE-4, or RDE-1 are enriched for innate immunity genes. RNA <span class="emphasis"><em>13</em></span>, 1090-1102.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17526642&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.542107" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib141"></a><p class="bibliomixed"><span class="bibliomisc">Welker, N.C., Maity, T.S., Ye, X., Aruscavage, P.J., Krauchuk, A.A., Liu, Q., and Bass, B.L. (2011). Dicer's helicase domain
                        discriminates dsRNA termini to promote an altered reaction mode. Mol. Cell <span class="emphasis"><em>41</em></span>, 589-599.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21362554&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.molcel.2011.02.005" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib142"></a><p class="bibliomixed"><span class="bibliomisc">Welker, N.C., Pavelec, D.M., Nix, D.A., Duchaine, T.F., Kennedy, S., and Bass, B.L. (2010). Dicer's helicase domain is required
                        for accumulation of some, but not all, <span class="emphasis"><em>C. elegans</em></span> endogenous siRNAs. RNA <span class="emphasis"><em>16</em></span>, 893-903.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20354150&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.2122010" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib143"></a><p class="bibliomixed"><span class="bibliomisc">Wightman, B., Ha, I., and Ruvkun, G. (1993). Posttranscriptional regulation of the heterochronic gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003003;class=Gene" target="_blank">lin-14</a></em></span> by <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00002993;class=Gene" target="_blank">lin-4</a></em></span> mediates temporal pattern formation in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>75</em></span>, 855-862</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8252622&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/0092-8674(93)90530-4" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib144"></a><p class="bibliomixed"><span class="bibliomisc">Yang, H., Zhang, Y., Vallandingham, J., Li, H., Florens, L., and Mak, H.Y. (2012). The RDE-10/RDE-11 complex triggers RNAi-induced
                        mRNA degradation by association with target mRNA in <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>26</em></span>, 846-856.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22508728&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.180679.111" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib145"></a><p class="bibliomixed"><span class="bibliomisc">Yigit, E., Batista, P.J., Bei, Y., Pang, K.M., Chen, C.C., Toila, N.H., Joshua-Tor, L., Mitani, S., Simard, M.J. and Mello,
                        C.C. (2006). Analysis of the <span class="emphasis"><em>C. elegans</em></span> Argonaute family reveals that distinct Argonautes act sequentially during RNAi. Cell <span class="emphasis"><em>127</em></span>, 747-757.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17110334&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cell.2006.09.033" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib146"></a><p class="bibliomixed"><span class="bibliomisc">Zamore, P.D., Tuschl, T., Sharp, P.A., and Bartel, D.P. (2000). RNAi, double-stranded RNA directs the ATP-dependent cleavage
                        of mRNA at 21 to 23 nucleotide intervals. Cell <span class="emphasis"><em>101</em></span>, 25-33.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10778853&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/S0092-8674(00)80620-0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib147"></a><p class="bibliomixed"><span class="bibliomisc">Zhang, C., Montgomery, T.A., Fischer, S.E., Garcia, S.M., Riedel, C.G., Fahlgren, N., Sullivan, C., Carrington, J.C., and
                        Ruvkun, G. (2012). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> RDE-10/RDE-11 complex regulates RNAi by promoting secondary siRNA amplification. Curr. Biol. <span class="emphasis"><em>22</em></span>, 881-890.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22542102&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.cub.2012.04.011" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib148"></a><p class="bibliomixed"><span class="bibliomisc">Zhang, C., Montgomery, T.A., Gabel, H.W., Fischer, S.E., Phillips, C.M., Fahlgren, N., Sullivan, C.M., Carrington, J.C., and
                        Ruvkun, G. (2011). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003508;class=Gene" target="_blank">mut-16</a></em></span> and other mutator class genes modulate 22G and 26G siRNA pathways in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>108</em></span>, 1201-1208.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21245313&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.1018695108" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib149"></a><p class="bibliomixed"><span class="bibliomisc">Zhuang, J.J., and Hunter, C.P. (2011). Tissue specificity of <span class="emphasis"><em>Caenorhabditis elegans</em></span> enhanced RNA interference mutants. Genetics <span class="emphasis"><em>188</em></span>, 235-237.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21385728&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.111.127209" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib150"></a><p class="bibliomixed"><span class="bibliomisc">Zhuang, J.J., and Hunter, C.P. (2012). RNA interference in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, uptake, mechanism, and regulation. Parasitology <span class="emphasis"><em>139</em></span>, 560-573.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22075748&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/http://dx.doi.org/10.1017/S0031182011001788" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib151"></a><p class="bibliomixed"><span class="bibliomisc">Zipperlen, P., Fraser, A.G., Kamath, R.S., Martinez-Campos, M., and Ahringer, J. (2001). Roles for 147 embryonic lethal genes
                        on <span class="emphasis"><em>C. elegans</em></span> chromosome I identified by RNA interference and video microscopy. EMBO J. <span class="emphasis"><em>20</em></span>, 3984-3992.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11483502&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/emboj/20.15.3984" target="_blank">Article</a>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Julie Ahringer. Last revised Novermber 10, 2013. Published May 7, 2014. This chapter should be cited as: Billi
                  et al. Endogenous RNAi pathways in <span class="emphasis"><em>C. elegans</em></span> (May 7, 2014), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.170.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2014 Allison C. Billi, Sylvia E. J. Fischer, John K. Kim. This is an open-access article distributed under the terms of
                  the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided
                  the original author and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e40" href="#d0e40">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:jnkim@umich.edu" target="_top">jnkim@umich.edu</a></p>
 <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>