<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Complementation</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#complementation_ArticleID" title="Complementation">
      <link rel="next" href="#sec1_1" title="1.&nbsp;A simple test for assigning a mutation to a genetic locus">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF" style="border:0;overflow:visible">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="complementation.pdf">Download PDF version</a> 
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="complementation_ArticleID"></a>Complementation<sup><a name="d0e10" href="#ftn.d0e10">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Karen</span> <span class="surname">Yook<em><span class="remark"><sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="address">Department of Biochemistry, Genetics Unit, University of Oxford, Oxford OX1 3QU, UK
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_geneticsgenomics.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Genetics and genomics<br />>> Genetics
		</a></div><hr />

<a href="http://www.wormbase.org/forums/index.php?board=185.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Complementation" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1_1">1. A simple test for assigning a mutation to a genetic locus</a></span></dt>
               <dt><span class="sect1"><a href="#sec1_2">2. When the complementation test is not so simple</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec2_1">2.1. Intragenic complementation</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_2">2.2. Non-allelic non-complementation</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec1_3">3. Conclusion</a></span></dt>
               <dt><span class="sect1"><a href="#sec1_4">4. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#sec1_5">5. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>Mutations in many genes can result in a similar phenotype. Finding a number of mutants with the same phenotype tells you little
                     about how many genes you are dealing with, and how mutable those genes are until you can assign those mutations to genetic
                     loci. The genetic assay for gene assignment is called the complementation test. The simplicity and robustness of this test
                     makes it a fundamental genetic tool for gene assignment. However, there are occasional unexpected outcomes from this test
                     that bear explanation. This chapter reviews the complementation test and its various outcomes, highlighting  relatively rare
                     but nonetheless interesting exceptions such as intragenic complementation and non-allelic non-complementation.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_1"></a>1.&nbsp;A simple test for assigning a mutation to a genetic locus
                     </h2>
                  </div>
               </div>
            </div>
            <p>During the complementation test, if a phenotype is observed when a recessive mutation
                           is combined in <span class="emphasis"><em>trans</em></span> with another recessive mutation that has been mapped to the same area, it is concluded that these mutations are alleles of
               the same gene; neither allele  produces a product that can restore wild-type function (<a href="#figure1">Figure 1</a>). However, if the double heterozygote exhibits a wild-type phenotype, it is concluded that the two mutations are alleles
               of different genes; each mutant has a functional version of the other gene. This assay requires that the mutation in question
               is genetically linked to the test mutation and that both mutations are recessive. The usefulness of this test can be most
               appreciated after large mutagenesis experiments, as demonstrated by Sydney Brenner in his pioneering mutagenesis screen in
               <span class="emphasis"><em>C. elegans</em></span>. Through mapping and complementation tests, he was able to place 256 autosomal ethylmethane sulphonate (EMS)  induced mutations
               into 77 complementation groups, making the genetic analysis of <span class="emphasis"><em>C. elegans</em></span> much easier to approach
                           (<a href="#bib6">Brenner, 1974</a>).
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="468">
<tr>
  <td align="center"><p><a href="complementationfig1.jpg"><img src="complementationfig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1. The complementation test.&nbsp;</b><span class="emphasis"><em>m1</em></span> and <span class="emphasis"><em>m2</em></span> are fictional alleles representing recessive mutations. Historical Note: The <span class="emphasis"><em>cis</em></span>-<span class="emphasis"><em>trans</em></span> test. In a more rigorous application of the complementation test; if the mutations do not complement one another when they
                     are present in the <span class="emphasis"><em>trans</em></span> configuration, the mutations should then be combined in <span class="emphasis"><em>cis</em></span> and tested over the wild-type allele (<a href="#bib45">Pontecorvo, 1958</a>). In this case, the <span class="emphasis"><em>cis</em></span> heterozygote should exhibit a wild-type phenotype. If a mutant phenotype is still observed, then the <span class="emphasis"><em>trans</em></span> complementation test was leading you to the wrong conclusion. The correct conclusion is that the mutations do not define
                     alleles of the same gene, rather they are exhibiting non-allelic non-complementation (see text). This result is rare and the
                     <span class="emphasis"><em>cis</em></span> test is not usually performed and may be quite difficult to carry out.
                  </p>
               </div>
            </div>
            <p>In essence, the complementation test is a very simple test and can quite satisfyingly
                           give you an idea of the number of loci involved in a process after a successful
                           mutagenesis experiment. Having said that, there are four considerations to keep in mind
                           (<a href="#bib17">Hawley and Walker, 2003</a>). First, both mutations must be recessive. Second, the mutations can cause different homozygous phenotypes. Third, the trans-heterozygote
               can
                           exhibit a more severe phenotype than either homozygote alone. Fourth, a mutation can
                           affect more than one gene product in a complex locus.
            </p>
            <div class="orderedlist">
               <ol type="1">
                  <li>
                     <p>Both mutations must be recessive. The complementation test will not work if
                                            either mutation exhibits dominance over the wild-type allele. In this case the
                                            phenotype of the dominant mutation will be exhibited whether or not the other
                                            allele is a mutation in the same gene; the mutations will appear to not
                                            complement. However, semi-dominant mutations can be put into complementation
                                            groups if the heterozygous phenotype is noticeably different from the homozygous
                                            phenotype (for examples see <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-58" target="_blank">unc-58</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a></em></span>; <a href="#bib6">Brenner, 1974</a>; <a href="#bib15">Greenwald and Horvitz, 1980</a>; <a href="#bib42">Park
                                               and Horvitz, 1986</a>). Assigning a dominant mutation to a locus requires other methods (see <a href="http://www.wormbook.org/chapters/www_geneticsuppression/geneticsuppression.html" target="_blank">Genetic suppression</a>).
                     </p>
                  </li>
                  <li>
                     <p>Mutations in the same gene can cause different homozygous phenotypes. If a gene functions in many processes, then mutations
                        in this gene may impair each function independently. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> is such a pleiotropic gene. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> encodes the <span class="emphasis"><em>C. elegans</em></span> member of the epidermal growth factor family and is an essential gene that is required in at least three different processes;
                        vulval induction, male spicule development and hermaphrodite fertility (<a href="#bib12">Ferguson and Horvitz, 1985</a>; <a href="#bib19">Hill and Sternberg, 1992</a>; <a href="#bib29">Liu et al., 1999</a>). Mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> disrupt these functions to different degrees; homozygotes of one allele, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1417;class=Variation" target="_blank">e1417</a></em></span>, are Vul whereas homozygotes of another allele, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n1058;class=Variation" target="_blank">n1058</a></em></span>, exhibit defects in male spicule development, hermaphrodite fertility, viability, and only very weakly, vulval induction
                        (<a href="#figure2">Figure 2</a>A). Thus initially, it would appear that <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1417;class=Variation" target="_blank">e1417</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n1058;class=Variation" target="_blank">n1058</a></em></span> are alleles of different genes. To further complicate the issue, these alleles exhibit complex complementation interactions
                        (see <a href="#sec2_1" title="2.1.&nbsp;Intragenic complementation">Intragenic complementation</a> below). 
                     </p>
                     <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="495">
<tr>
  <td align="center"><p><a href="complementationfig2.jpg"><img src="complementationfig2_s.jpg" border="2" align="middle" alt=" Yook complementation fig 2"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 2.&nbsp;</b>(A) LIN-3 is required in at least four processes, viability, vulval development, male spicule development and hermaphrodite
                               fertility. <span class="emphasis"><em>lin-3</em></span> alleles exhibit a different allelic series for each process, suggesting that each process has a different, independently
                              regulated requirement for LIN-3. Adapted from <a href="#bib29">Liu et al., 1999</a>. (B) Domain structure of LIN-3 and location of the lesions in <span class="emphasis"><em>lin-3</em></span> alleles. SP-signal peptide cleavage site, EGF-epidermal growth factor domain, TM-transmembrane domain, dotted vertical lines-intron
                              boundries. Adapted from <a href="#bib29">Liu et al., 1999</a>.  Information for <span class="emphasis"><em>e1417</em></span> is from <a href="#bib20">Hwang and Sternberg, 2004</a>.
                           </p>
                        </div>
                     </div>
                  </li>
                  <li>
                     <p>The trans-heterozygote can exhibit a phenotype, that is more severe than either homozygote
                                            alone. This is also termed "negative complementation". This result contrasts with
                                            the expected phenotype of the trans-heterozygote having the phenotype of the
                                            weaker allele. Although rare, negative complementation has been noted for the
                                            <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sqt-1" target="_blank">sqt-1</a></em></span> locus. In this case, animals homozygous for
                                            <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc100;class=Variation" target="_blank">sc100</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc103;class=Variation" target="_blank">sc103</a></em></span> are wild-type, but
                                            <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc101;class=Variation" target="_blank">sc101</a>/<a href="http://www.wormbase.org/db/get?name=sc103;class=Variation" target="_blank">sc103</a></em></span> male progeny have a weak Dumpy (Dpy) phenotype. Additionally, when <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc100;class=Variation" target="_blank">sc100</a></em></span> is put in <span class="emphasis"><em>trans</em></span> to a deficiency, the
                                            trans-heterozygotes have a weak Dpy phenotype (<a href="#bib28">Kusch and Edgar, 1986</a>). In general, mutations in many of the collagen genes result in
                                            interesting allelic interactions as discussed below.
                     </p>
                  </li>
                  <li>
                     <p>A mutation can affect more than one gene product if the genes are in a complex locus. A classic example of
                                            this type of genetic relationship is the
                                            <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span>/<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span> locus (<a href="#figure3">Figure 3</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span>
                                            are tightly linked and mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span> and
                                            <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span> result in very similar, severe behavioral and developmental defects (<a href="#bib49">Rand and Russell, 1984</a>; <a href="#bib48">Rand, 1989</a>). Although most alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span> complemented
                                            <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span>, three hypomorphic <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span>
                                            alleles failed to complement <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span>. Molecular
                                            characterization finally demonstrated that <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span> and
                                            <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span> encode independent mature transcripts produced by
                                            alternative splicing (<a href="#bib2">Alfonso et al., 1993</a>; <a href="#bib3">Alfonso et al., 1994</a>). The non-complementing mutations fall into two groups, <span class="symbol">&#945;</span> and <span class="symbol">&#946;</span>.
                                            The one <span class="symbol">&#945;</span> mutation is a deletion of the non-coding region between
                                            <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span> that removes
                                            sequences required for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span> RNA processing and most
                                            likely for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span> expression and/or splicing. The two <span class="symbol">&#946;</span> mutations have yet to be identified, but they have been mapped to the shared
                                            5' untranslated region of the locus and thus probably disrupt transcriptional
                                            regulation of both <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span>
                                            (<a href="#bib3">Alfonso et al., 1994</a>). Thus the non-complementing alleles are
                                            actually alleles of both genes.
                     </p>
                     <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="complementationfig3.jpg"><img src="complementationfig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>

                        </table>
                        <div class="caption">
                           <p><b>Figure 3. <span class="emphasis"><em>cha-1</em></span> and <span class="emphasis"><em>unc-17</em></span> form a complex locus.&nbsp;</b>(A) The complementation matrix of alleles of <span class="emphasis"><em>cha-1</em></span> and <span class="emphasis"><em>unc-17</em></span>. "+" indicates complementation, "&#8212;" indicates non-complementation. <span class="emphasis"><em>p1156</em></span>, <span class="emphasis"><em>e113</em></span> and <span class="emphasis"><em>e876</em></span> do not complement alleles of <span class="emphasis"><em>cha-1</em></span> or <span class="emphasis"><em>unc-17</em></span> (boxed in red). <span class="emphasis"><em>unc-33(e204)</em></span> was used as a complementation control. Reprinted with permission from <a href="#bib49">Rand and Russell, 1984</a>. Copyright &copy;1984 the Genetics Society of America. (B) <span class="emphasis"><em>p1156</em></span>,<span class="emphasis"><em> e113</em></span> and <span class="emphasis"><em>e876</em></span> were fine mapped to two separate regions of the <span class="emphasis"><em>cha-1/unc-17</em></span> locus. <span class="emphasis"><em>p1156</em></span> defines the alpha region, to the left of the other <span class="emphasis"><em>unc-17</em></span> alleles, while <span class="emphasis"><em>e113</em></span> and <span class="emphasis"><em>e876</em></span> define the beta region, to the right of <span class="emphasis"><em>unc-17</em></span>
                                          alleles. Reprinted with permission from <a href="#bib48">Rand, 1989</a>. Copyright &copy;1989 the Genetics Society of America. (C) <a href="#bib3">Alfonso et al. (1994)</a> molecularly determined
                                          that <span class="emphasis"><em>unc-17</em></span> lies within the first intron of the <span class="emphasis"><em>cha-1</em></span> locus, the beta region is a
                                          non-coding region of the complex locus shared with <span class="emphasis"><em>cha-1</em></span>, and the alpha region lies
                                          between the 3' end of <span class="emphasis"><em>unc-17</em></span> and splice site junction of <span class="emphasis"><em>cha-1</em></span>. Top line corresponds to
                                          genetic boundaries between <span class="emphasis"><em>cha-1</em></span> and <span class="emphasis"><em>unc-17</em></span>, with mutations noted. Middle line corresponds to genomic DNA with <span class="emphasis"><em>SalI (S)</em></span> and <span class="emphasis"><em>EcoRI</em></span> sites noted. Bottom line corresponds
                                          to cDNA structures of <span class="emphasis"><em>cha-1</em></span> and <span class="emphasis"><em>unc-17</em></span>. Reprinted with permission from <a href="#bib3">Alfonso et al. (1994)</a>, Copyright (1994), with permission from Elsevier. See reference for more details.
                           </p>
                        </div>
                     </div>
                  </li>
               </ol>
            </div>
            <div class="literallayout">
               <p></p>
            </div>
            <p>In summary, the complementation test is used to assign mutant alleles to specific genetic loci. Mutant alleles of the same
               gene fail to complement one another, while alleles of different genes do complement each other. In general, keeping in mind
               the considerations above, the interpretation of complementation test is quite straightforward. However, we are occasionally
               faced with complex patterns of complementation.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_2"></a>2.&nbsp;When the complementation test is not so simple
                     </h2>
                  </div>
               </div>
            </div>
            <p>There are two ways the complementation test can mislead you. First, alleles of the
                           same gene can sometimes complement each other, termed "intragenic complementation".
                           Second, mutations of two separate genes can sometimes not complement one another, termed  "non-allelic non-complementation".
               In both instances, complex allelic interactions are
                           observed that can lead to problems  knowing when and how to assign the mutant alleles
                           to complementation groups. Although these allelic conundrums are found in many organisms,
                           this discussion is focused on examples found in <span class="emphasis"><em>C. elegans</em></span>.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_1"></a>2.1.&nbsp;Intragenic complementation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>During intragenic complementation, alleles of the same gene complement one
                                  another, even though both alleles produce a faulty gene product. There are different
                                  means by which mutant alleles of the same gene can mutually correct one another.
                                  First, one mutant gene product may reduce the dosage of the other mutant product.
                                  Second, a faulty complex formed by one mutant gene product may be stabilized by the
                                  presence of an alternatively mutant gene product. Third, a gene product containing a
                                  mutation that affects one function may provide the function missing from an
                                  alternatively altered gene product.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_1"></a>2.1.1.&nbsp;Reducing dosage of a mutant product
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Alleles of genes required for cuticle formation exhibit intragenic
                                         complementation (<a href="#bib11">De Melo et al., 2002</a>; <a href="#bib27">Kramer et al., 1988</a>; <a href="#bib28">Kusch and Edgar, 1986</a>). In the case of the cuticle
                                         collagen <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sqt-1" target="_blank">sqt-1</a></em></span>, the recessive missense mutation,
                                         <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc101;class=Variation" target="_blank">sc101</a></em></span>, is alleviated in <span class="emphasis"><em>trans</em></span> by a
                                         null mutation, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc103;class=Variation" target="_blank">sc103</a></em></span> (<a href="#table2" title="Table 2. Mutated domains of the non-allelic, non-complementing SQT gene products">Table 2</a>  and <a href="#table3" title="Table 3. Complex interactions are observed between collagen genes,&#xA;                        sqt-1, 2, 3&#xA;                        and rol-8">Table 3</a>; <a href="#bib26">Kramer and Johnson, 1993</a>). Although both <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc101;class=Variation" target="_blank">sc101</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc103;class=Variation" target="_blank">sc103</a></em></span> homozygotes have tail
                                         defects, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc101;class=Variation" target="_blank">sc101</a></em></span>/<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc101;class=Variation" target="_blank">sc101</a></em></span> is a long weak left roller, the
                                         trans-heterozygotes are wild-type (<a href="#bib28">Kusch and Edgar, 1986</a>; <a href="#bib26">Kramer and Johnson, 1993</a>). This result suggests that the presence of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sc101;class=Variation" target="_blank">sc101</a></em></span> product has a negative effect on normal collagen associations, since decreasing the amount of the mutant product (e.g. replacing
                     one allele with an allele that produces no product) restores a wild-type phenotype. The poisoning effect of the Gly-X-Y mutations
                     seems to be especially detrimental to collagen proteins and cuticle formation as they are often found in alleles that exhibit
                     non-allelic non-complementation with other collagen genes (see <a href="#sec2_2" title="2.2.&nbsp;Non-allelic non-complementation">Non-allelic non-complementation</a>).
                  </p>
                  <div class="table"><a name="table1"></a><p class="title">Table 1. Phenotypes of <span class="emphasis"><em>lin-3</em></span> homozygous and heteroallelic strains
                     </p>
                     <table summary="Table 1. Phenotypes of lin-3 homozygous and heteroallelic strains" border="1">
                        <colgroup>
                           <col align="left">
                           <col align="left">
                           <col align="left">
                           <col align="left">
                           <col align="left">
                        </colgroup>
                        <tbody>
                           <tr>
                              <td align="left"><span class="emphasis"><em>lin-3</em></span> allele
                              </td>
                              <td align="left"><span class="emphasis"><em>n378</em></span></td>
                              <td align="left"><span class="emphasis"><em>e1417</em></span></td>
                              <td align="left"><span class="emphasis"><em>n1058</em></span></td>
                              <td align="left"><span class="emphasis"><em>n1059</em></span></td>
                           </tr>
                           <tr>
                              <td align="left"><span class="emphasis"><em>n378</em></span></td>
                              <td align="left">97% Vul (<span class="emphasis"><em>n</em></span>=266)
                              </td>
                              <td align="left">88% Vul (<span class="emphasis"><em>n</em></span>=226)
                              </td>
                              <td align="left">78% Vul (<span class="emphasis"><em>n</em></span>=364)
                              </td>
                              <td align="left">100% Vul (<span class="emphasis"><em>n</em></span>=665)
                              </td>
                           </tr>
                           <tr>
                              <td align="left"><span class="emphasis"><em>e1417</em></span></td>
                              <td align="left">89% Vul (<span class="emphasis"><em>n</em></span>=351)
                              </td>
                              <td align="left">&nbsp;</td>
                              <td align="left">59% Vul (<span class="emphasis"><em>n</em></span>=414)
                              </td>
                              <td align="left">99.8% Vul (<span class="emphasis"><em>n</em></span>=471)
                              </td>
                           </tr>
                           <tr>
                              <td align="left"><span class="emphasis"><em>n1058</em></span></td>
                              <td align="left">&nbsp;</td>
                              <td align="left">&nbsp;</td>
                              <td align="left">Sterile; occasional arrested larvae</td>
                              <td align="left">Arrested larvae</td>
                           </tr>
                           <tr>
                              <td align="left"><span class="emphasis"><em>n1059</em></span></td>
                              <td align="left">&nbsp;</td>
                              <td align="left">&nbsp;</td>
                              <td align="left">&nbsp;</td>
                              <td align="left">Arrested larvae</td>
                           </tr>
                           <tr>
                              <td colspan="5" align="left"><span class="emphasis"><em>lin-3</em></span> alleles exhibit intragenic complementation. <span class="emphasis"><em>e1417</em></span>/<span class="emphasis"><em>n1058</em></span> and<span class="emphasis"><em> n378</em></span>/<span class="emphasis"><em>n1058</em></span> trans-heterozygotes are viable and are less likely to be Vul than <span class="emphasis"><em>e1417</em></span> or <span class="emphasis"><em>n378</em></span> homozygotes. Abbreviations:Vul-Vulvaless. Reprinted with permission from Ferguson and Horvitz, 1985. Copyright &copy;1985 the
                                 Genetics Society of America.
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="table"><a name="table2"></a><p class="title">Table 2. Mutated domains of the non-allelic, non-complementing SQT gene products</p>
                     <table summary="Table 2. Mutated domains of the non-allelic, non-complementing SQT gene products" border="1">
                        <colgroup>
                           <col align="center">
                           <col align="center">
                           <col align="center">
                           <col align="center">
                        </colgroup>
                        <thead>
                           <tr valign="middle">
                              <th align="center" valign="middle">&nbsp;</th>
                              <th align="center" valign="middle">Allele</th>
                              <th align="center" valign="middle">Collagen domain distrupted</th>
                              <th align="center" valign="middle">References</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr valign="middle">
                              <td rowspan="4" align="center" valign="middle">
                                                                             
                                 <p><span class="emphasis"><em>sqt-1</em></span></p>
                              </td>
                              <td align="center" valign="middle"><span class="emphasis"><em>e1350dm</em></span></td>
                              <td align="center" valign="middle">N-terminal propeptide cleavage site</td>
                              <td rowspan="4" align="center" valign="middle">
                                                                             
                                 <p><a href="#bib26">Kramer and Johnson, 1993</a></p>
                              </td>
                           </tr>
                           <tr>
                              <td align="center"><span class="emphasis"><em>sc13</em></span></td>
                              <td align="center">Cysteine domain /C-terminal propeptide cleavage site</td>
                           </tr>
                           <tr>
                              <td align="center"><span class="emphasis"><em>sc101</em></span></td>
                              <td align="center">Gly-X-Y repeat</td>
                           </tr>
                           <tr>
                              <td align="center"><span class="emphasis"><em>sc103(n)</em></span></td>
                              <td align="center">All (N-terminal stop)</td>
                           </tr>
                           <tr>
                              <td rowspan="4" align="center" valign="middle">
                                                                             
                                 <p><span class="emphasis"><em>sqt-3</em></span></p>
                              </td>
                              <td align="center"><span class="emphasis"><em>sc8</em></span></td>
                              <td align="center">C-terminal propeptide cleavage site</td>
                              <td rowspan="2" align="center" valign="middle">
                                 <p><a href="#bib39">Novelli et al., 2004</a></p>
                              </td>
                           </tr>
                           <tr>
                              <td align="center"><span class="emphasis"><em>sc42</em></span></td>
                              <td align="center">Signal peptide</td>
                           </tr>
                           <tr>
                              <td align="center"><span class="emphasis"><em>sc63ts</em></span></td>
                              <td align="center">Gly-X-Y repeat</td>
                              <td rowspan="2" align="center" valign="middle">
                                 <p><a href="#bib56">van der Keyl et al., 1994</a></p>
                              </td>
                           </tr>
                           <tr>
                              <td align="center"><span class="emphasis"><em>e24ts,sd</em></span></td>
                              <td align="center">Gly-X-Y repeat</td>
                           </tr>
                           <tr>
                              <td align="center">
                                 <p><span class="emphasis"><em>rol-8</em></span></p>
                              </td>
                              <td align="center">&nbsp;</td>
                              <td align="center">mutations not determined</td>
                              <td align="center">&nbsp;</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="table"><a name="table3"></a><p class="title">Table 3. Complex interactions are observed between collagen genes,
                                                <span class="emphasis"><em>sqt-1</em></span>, <span class="emphasis"><em>2</em></span>, <span class="emphasis"><em>3</em></span>
                                                and <span class="emphasis"><em>rol-8</em></span></p>
                     <table summary="Table 3. Complex interactions are observed between collagen genes,&#xA;                        sqt-1, 2, 3&#xA;                        and rol-8" border="1">
                        <colgroup>
                           <col align="left">
                        </colgroup>
                        <tbody>
<tr>
  <td align="center"><p><span class="inlinemediaobject"><a href="complementationtable3.jpg"><img src="complementationtable3_s.jpg" border="2" align="middle" alt=" figure 2"></a></span>
  </td>
</tr>
                           <tr>
                              <td align="left">For <span class="emphasis"><em>sqt-3</em></span> interactions, trans-heterozygotes were
                                                         assayed at 20<span class="symbol">&deg;</span> and 25<span class="symbol">&deg;</span> C. Only the results from the
                                                             25<span class="symbol">&deg;</span> assay are included here. See references for more instances of non-allelic  non-complementation. In some instances, complementation
                                 was also observed among alleles of the
                                                         same gene (highlighted in yellow). Abbreviations are as follows. (n)-null
                                                         allele, RRol-Right Roller, Lon-Long, Dpy-Dumpy, LRol-Left Roller,
                                                         Rol-Roller, Tal-abnormal tail, ND-not discussed. Modifiers; x-extreme,
                                                         w-weak. a Data from <a href="#bib28">Kusch and Edgar (1986)</a>. b Data from <a href="#bib26">Kramer and Johnson (1993)</a>.
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_2"></a>2.1.2.&nbsp;Stabilizing a complex
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In 1964, Crick and Orgel proposed that inter-allelic complementation may be
                                         due to a "good corrects bad" mechanism (<a href="#bib10">Crick and Orgel, 1964</a>). In this scenario, a mutation in a subunit of a homomer or multimer would result in the mis-folding of the monomer subunit
                     and disrupt the activity of the whole complex. Likewise, a subunit with a different mutation would also produce a
                                         faulty complex. However, a functional complex can be restored if one mutant
                                         subunit is able to structurally compensate for the other mutant subunit.
                  </p>
                  <p>One example of this type of mutual correction may be demonstrated by alleles
                                         of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-2" target="_blank">let-2</a></em></span>. The initial identification of
                                         <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-2" target="_blank">let-2</a></em></span> occurred in a screen for X-linked lethals and
                                         steriles (<a href="#bib34">Meneely and Herman, 1979</a>). In this screen, four alleles
                                         of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-2" target="_blank">let-2</a></em></span> were identified and were shown to exhibit a
                                         complex complementation pattern. In fact, every allele complemented at least one
                                         other allele, including two alleles identified independently in other labs. A second screen published
                     a few years later identified another eight alleles, which also exhibited the same complex pattern, giving rise to a complicated
                     web of allelic interactions (<a href="#figure4">Figure 4</a>A; <a href="#bib35">Meneely and Herman, 1981</a>). These alleles also complement at least one other allele (see <a href="#table4" title="Table 4. Complementation tests among let-2 allele*">Table 4</a>; <a href="#bib35">Meneeley and Herman, 1981</a>). The strongest allele, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/variation?name=mn153;class=Variation" target="_blank">mn153</a></em></span>, is a mutation in an N-terminal splice site and does not complement any other allele except <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=b246;class=Variation" target="_blank">b246</a></em></span>. <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=b246;class=Variation" target="_blank">b246</a></em></span> exhibits the least severe phenotype of all the alleles; it is embryonic lethal only at non-permissive temperatures. Unexpectedly,
                      <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=b246;class=Variation" target="_blank">b246</a></em></span> complements all other <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-2" target="_blank">let-2</a></em></span> alleles (see <a href="#table4" title="Table 4. Complementation tests among let-2 allele*">Table 4</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-2" target="_blank">let-2</a></em></span> encodes an <span class="symbol">&#945;</span>2(IV) collagen chain, which is a major component of basement membrane (see <a href="http://www.wormbook.org/chapters/www_basementmb/basementmb.html" target="_blank">Basement membranes</a>; <a href="#bib52">Sibley et al., 1993</a>). Like other collagen molecules, <a href="http://www.wormbase.org/db/seq/protein?name=LET-2;class=protein" target="_blank">LET-2</a> has an extensive Gly-X-Y repeat domains, making up about three-quarters of the molecule, and conserved non-collagenous regions
                     (<a href="#figure4">Figure 4</a>B). Type IV collagen molecules are made up of heterotrimers of one <span class="symbol">&#945;</span>2(IV) chain and 2 <span class="symbol">&#945;</span>1(IV) chains (<a href="#figure4">Figure 4</a>C). These heterotrimers dimerize via their conserved C-terminal NC1 domains and can tetramerize via their C-termini to form
                     a collagen lattice that provides structural support for basement membranes. The strong mutations in the Gly-X-Y regions can
                     disrupt trimer formation and cause  the accumulation of misfolded heterotrimers in the cell in a temperature dependent manner
                     (<a href="#bib16">Gupta et al., 1997</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=b246;class=Variation" target="_blank">b246</a></em></span> mutation occurs in a region of relatively high thermal stability and is therefore unlikely to have as drastic an effect on
                     triple helical formation as mutations in regions of lower thermal stability (such as the mutations noted in bold in <a href="#figure4">Figure 4</a>B) in subsequent complex formations (<a href="#bib52">Sibley et al., 1993</a>). The presence of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=b246;class=Variation" target="_blank">b246</a></em></span> mutant product in the type IV collagen heterotrimers may help to stabilize the presence of other more severely altered heterotrimers
                     in trans-heterozygotes.
                  </p>
                  <div class="table"><a name="table4"></a><p class="title">Table 4. Complementation tests among <span class="emphasis"><em>let-2</em></span> allele*
                     </p>
                     <table summary="Table 4. Complementation tests among let-2 allele*" border="0">
                        <colgroup>
                           <col align="center">
                        </colgroup>
                        <tbody>
                           <tr>
<tr>
  <td align="center"><p><span class="inlinemediaobject"><a href="complementationtable4.jpg"><img src="complementationtable4_s.jpg" width="700" border="2" align="middle"></a></span>
  </td>
</tr>
                           <tr>
                              <td align="left">Table of complementation interactions among the <span class="emphasis"><em>let-2</em></span> alleles, including <span class="emphasis"><em>mn153</em></span>, which does not complement any of the other alleles except <span class="emphasis"><em>b246</em></span>, which complements every allele. Reprinted with permssion from <a href="#bib35">Meneely and Herman, 1981</a>. Copyright &copy;1981 the Genetics Society of America.
                              </td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="495">
<tr>
  <td align="center"><p><a href="complementationfig4.jpg"><img src="complementationfig4_s.jpg" border="2" align="middle" alt=" Yook Complementation figure 4"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 4. Intragenic complementation of <span class="emphasis"><em>let-2</em></span> alleles.&nbsp;</b>(A) <span class="emphasis"><em>let-2</em></span> alleles exhibit a very complex pattern of complementation. Lines between
                                       alleles indicate non-complementation (the expected outcome). Alleles that are not joined
                                       by a line complement one another to some extent. Reprinted with permission from <a href="#bib35">Meneely and Herman, 1981</a>. Copyright &copy;1981 the Genetics Society of America. (B) A schematic of the LET-2 (<span class="symbol">&#945;</span>2)IV collagen, adapted from <a href="#bib52">Sibley et al. (1993)</a>. Conserved type IV collagen features include extensive Gly-X-Y repeats (in blue) with interspersed interruptions (vertical
                                       black lines), non-collagenous termini (in grey) with a conserved NC1 domain at the
                                       C-terminus. Cross-hatched area denotes an alternative splice region. Locations of
                                       mutations are indicated with the strongest mutations (&gt;90% homozygous embryonic
                                       lethality at 20<span class="symbol">&deg;</span>) in bold. <span class="emphasis"><em>mn153</em></span> (boxed in red), is the only mutation that does not
                                       complement any other allele aside from <span class="emphasis"><em>b246</em></span> (boxed in green), which complements all
                                       other alleles to some extent. The interruptions in the Gly-X-Y repeat domains lower the
                                       thermal stability of the region resulting in more severe phenotypes of Gly-X-Y mutations
                                       (noted in bold). (C) Type IV collagen is composed of a heterotrimer of 1 <span class="symbol">&#945;</span>2(IV) collagen
                                       chain and 2 <span class="symbol">&#945;</span>1(IV) chains. These heterotrimers can dimerize at their C-terminal NC1
                                       domains, and then form a complex lattice through tetramerization of their N-terminal
                                       regions and lateral interactions along their triple helical domains. Mutations in the
                                       Gly-X-Y repeats would disrupt heterotrimer formation and subsequent associations.
                        </p>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_3"></a>2.1.3.&nbsp;Providing the missing function
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Many examples of intragenic complementation have been reported for genes that encode products with independently functioning
                     domains, or loci that are  involved in independently regulated functions. In these cases, a mutation can result in a gene
                     product that specifically disrupts one process while functioning relatively normally in others. Trans-heterozygotes made with
                     such alleles exhibit intragenic complementation. This appears to be the most frequently reported type of intragenic complementation
                     in <span class="emphasis"><em>C. elegans</em></span>, perhaps since it results in some very interesting interactions. This type of intragenic complementation has been observed
                     for a number of genes, including <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gld-1" target="_blank">gld-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-5" target="_blank">unc-5</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-84" target="_blank">unc-84</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=bli-4" target="_blank">bli-4</a></em></span>.
                  </p>
                  <p>As mentioned before, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> is required in different developmental processes. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> is also independently mutable, such that a mutation in one part of the gene can disrupt one process while having minimal
                     effects on other  <a href="http://www.wormbase.org/db/seq/protein?name=LIN-3;class=protein" target="_blank">LIN-3</a> requiring processes. For example, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1417;class=Variation" target="_blank">e1417</a></em></span> mutation only disrupts vulval induction,  having no effect on male spicule development, while the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n1058;class=Variation" target="_blank">n1058</a></em></span> mutation only weakly disrupts vulval induction, but strongly disrupts male spicule development (<a href="#bib29">Liu et al., 1999</a>). These homozygous phenotypes suggest that each allele has residual function in certain tissues. Furthermore, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n1058;class=Variation" target="_blank">n1058</a></em></span> mutation can complement two other <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> alleles, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1417;class=Variation" target="_blank">e1417</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n378;class=Variation" target="_blank">n378</a></em></span>, for vulval induction (see <a href="#table1" title="Table 1. Phenotypes of lin-3 homozygous and heteroallelic strains">Table 1</a> and  <a href="#table3" title="Table 3. Complex interactions are observed between collagen genes,&#xA;                        sqt-1, 2, 3&#xA;                        and rol-8">Table 3</a> in <a href="#bib29">Liu et al., 1999</a>). The intragenic complementation in this case suggests that residual function from each allele is enough to complement the
                     missing tissue specific function of the other allele, especially since the null mutation, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n1059;class=Variation" target="_blank">n1059</a></em></span>, does not complement any of these alleles (<a href="#table1" title="Table 1. Phenotypes of lin-3 homozygous and heteroallelic strains">Table 1</a>). Molecular analysis of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> mutations supports these genetic observations. Specifically, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1417;class=Variation" target="_blank">e1417</a></em></span> has a mutation in an anchor-cell specific enhancer, which is required for <a href="http://www.wormbase.org/db/seq/protein?name=LIN-3;class=protein" target="_blank">LIN-3</a> function in vulval development, but would not affect <a href="http://www.wormbase.org/db/seq/protein?name=LIN-3;class=protein" target="_blank">LIN-3</a> function in other tissues (<a href="#bib20">Hwang and Sternberg, 2004</a>; <a href="#figure2">Figure 2</a>B). Additionally, <span class="emphasis"><em>e378</em></span> has a missense mutation resulting in reduction-of-function and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n1058;class=Variation" target="_blank">n1058</a></em></span> has a splice site mutation, which may cause a truncated <a href="http://www.wormbase.org/db/seq/protein?name=LIN-3;class=protein" target="_blank">LIN-3</a> product or lowered amount of the <a href="http://www.wormbase.org/db/seq/protein?name=LIN-3;class=protein" target="_blank">LIN-3</a> product to be made (<a href="#bib12">Ferguson and Horvitz, 1985</a>; <a href="#bib20">Hwang and Sternberg, 2004</a>; <a href="#bib29">Liu et al., 1999</a>). Neither of these mutations would completely remove <a href="http://www.wormbase.org/db/seq/protein?name=LIN-3;class=protein" target="_blank">LIN-3</a> function.
                  </p>
                  <p>Such functional compensation has also been observed for alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gld-1" target="_blank">gld-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-5" target="_blank">unc-5</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-84" target="_blank">unc-84</a></em></span> (<a href="#bib13">Francis et al., 1995</a>; <a href="#bib36">Merz et al., 2001</a>; <a href="#bib31">Malone et al., 1999</a>). For example, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gld-1" target="_blank">gld-1</a></em></span> encodes an RNA binding protein required for several aspects of cell cycle progression during gametogenesis (<a href="#bib23">Jones and Schedl, 1995</a>). <a href="http://www.wormbase.org/db/seq/protein?name=GLD-1;class=protein" target="_blank">GLD-1</a> is required in a temporally and spatially regulated manner during germline development, and these requirements are reflected
                     in the range of mutant phenotypes exhibited by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gld-1" target="_blank">gld-1</a></em></span> mutants. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gld-1" target="_blank">gld-1</a></em></span> mutations fall into five classes, A-E (<a href="#bib13">Francis et al., 1995</a>). Class C mutations result in the masculinization of the germline so that only sperm and no oocytes are produced. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gld-1" target="_blank">gld-1</a></em></span> class D mutations result in the opposite phenotype of feminization of the germline so that only oocytes and no sperm are
                     produced. Trans-heterozygotes between class C and D alleles produce a wild-type phenotype presumably because gene products
                     carrying the class C mutation can provide <a href="http://www.wormbase.org/db/seq/protein?name=GLD-1;class=protein" target="_blank">GLD-1</a> function during spermatogenesis while products carrying the class D mutation provide <a href="http://www.wormbase.org/db/seq/protein?name=GLD-1;class=protein" target="_blank">GLD-1</a> function during oogenesis.
                  </p>
                  <p>Finally, if a mutation results in the lack of expression of gene product in a subset of cells, a compensating mutation can
                     be one that restores that expression pattern. One example of compensation by expression is illustrated by mutations of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=bli-4" target="_blank">bli-4</a></em></span>. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=bli-4" target="_blank">bli-4</a></em></span> encodes an essential Kex2/substilin-like protease that cleaves the N-terminus of pre-pro-peptides (<a href="#bib54">Thacker et al., 1995</a>). Two lethal alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=bli-4" target="_blank">bli-4</a></em></span>, <span class="emphasis"><em>sy90</em></span> and <span class="emphasis"><em>h754</em></span>, which do not complement other lethal alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=bli-4" target="_blank">bli-4</a></em></span>, were found to complement the viable Blister allele, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a></em></span> (<a href="#bib43">Peters et al., 1991</a>; <a href="#bib51">Rose and Baillie, 1980</a>). Other lethal alleles exacerbate the Blister phenotype of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a></em></span>. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=bli-4" target="_blank">bli-4</a></em></span> encodes nine splice variants of the Kex2/substilin-like protease. All variants share the first 12 exons, including the coding
                     region for the protease domain. Mutations in the protease domain affect all nine isoforms and result in embryonic arrest.
                     <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a></em></span> is a deletion that affects 5 of the 9 isoforms (<a href="#bib55">Thacker et al., 2000</a>). The <span class="emphasis"><em>sy90</em></span> and <span class="emphasis"><em>h754</em></span> mutations have not been identified, suggesting that these mutations may lie in regulatory elements. In addition, <span class="emphasis"><em>sy90</em></span> and <span class="emphasis"><em>h754</em></span> homozygous mutants arrest development during L1, which is later than the other lethal mutants suggesting that these mutations
                     alter expression or function of <a href="http://www.wormbase.org/db/seq/protein?name=BLI-4;class=protein" target="_blank">BLI-4</a> after the embryonic period. Thus in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a></em></span>/<span class="emphasis"><em>sy90</em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a></em></span>/<span class="emphasis"><em>h754</em></span> trans-heterozygote it appears that enough <a href="http://www.wormbase.org/db/seq/protein?name=BLI-4;class=protein" target="_blank">BLI-4</a> is supplied by the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e937;class=Variation" target="_blank">e937</a></em></span> allele to proceed past L1 development while the <span class="emphasis"><em>sy90</em></span> or <span class="emphasis"><em>h754</em></span> alleles may provide enough <a href="http://www.wormbase.org/db/seq/protein?name=BLI-4;class=protein" target="_blank">BLI-4</a> in later stages to abrogate the Blister phenotype.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_4"></a>2.1.4.&nbsp;Other examples
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>One last example of intragenic complementation, which could occur through stabilizing a complex, or by providing a compensatory
                     function, is exhibited by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=eat-2" target="_blank">eat-2</a></em></span> alleles. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=eat-2" target="_blank">eat-2</a></em></span> encodes a subunit of the nicotinic acetylcholine gated ion channel required for pharyngeal pumping (<a href="#bib46">Raizen et al., 1995</a>). Nicotinic acetylcholine gated ion channels are hetero-pentamers that are activated by endogenous acetylcholine or drugs
                     such as nicotine. The extracellular ligand binding sites are formed at the interfaces between two different subunits (<a href="#bib8">Changeux and Edelstein, 1998</a>). Alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=eat-2" target="_blank">eat-2</a></em></span> fall into five classes, A-E. Although class A alleles do not complement alleles in any class, alleles in class B-E exhibit
                     intragenic complementation (<a href="#figure5">Figure 5</a>A). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=eat-2" target="_blank">eat-2</a></em></span> mutations were sequenced and complementing mutations of class B-E were located in the extracellular domain of the subunit
                     (<a href="#figure5">Figure 5</a>B; <a href="#bib33">McKay  et al., 2004</a>). In general, disruptions in extracellular domains would be expected to alter ligand-binding sites. One can imagine then
                     that a channel composed of either faulty subunit would not form any stable ligand binding sites; however, two differently
                     altered subunits might stabilize the channel and the ligand binding sites to allow some increase in channel activity. Alternatively,
                     in an <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=eat-2" target="_blank">eat-2</a></em></span> trans-heterozygote with two faulty <a href="http://www.wormbase.org/db/seq/protein?name=EAT-2;class=protein" target="_blank">EAT-2</a> subunits, two different channels might be made with compensatory ligand specificities, thus restoring the functions of the
                     <a href="http://www.wormbase.org/db/seq/protein?name=EAT-2;class=protein" target="_blank">EAT-2</a> nicotinic acetylcholine receptor.
                  </p>
                  <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="342">
<tr>
  <td align="center"><p><a href="complementationfig5.jpg"><img src="complementationfig5_s.jpg" border="2" align="middle" alt=" Yook Complementation figure 5"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 5.&nbsp;</b>(A) Intragenic complementation of <span class="emphasis"><em>eat-2</em></span> alleles. The number of alleles in each class is noted in parentheses below the class letter along the bottom row. Intragenic
                           complementation results are highlighted in yellow. *These trans-heterozygotes are not fully wild-type; however, thay are less
                           defective than either homozygote.  Data was extrapolated from <a href="#bib46">Raizen et al. (1995)</a>. (B) A schematic of the EAT-2 subunit with approximate locations of the mutations that can complement other <span class="emphasis"><em>eat-2</em></span> mutations
                                       (highlighted in yellow). Class A non-complementing mutations are highlighted in purple.
                                       Data extrapolated from <a href="#bib33">McKay et al. (2004)</a>. Abbreviations: Pmp-Pumping defective.
                        </p>
                     </div>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_2"></a>2.2.&nbsp;Non-allelic non-complementation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Non-allelic non-complementation, also referred to as second-site non-complementation<sup><a name="d0e1392" href="#ftn.d0e1392">&#8224;</a></sup>, intergenic non-complementation, unlinked non-complementation, or second-site dominant enhancer, occurs when alleles of two
                  different loci behave as if they are alleles of the same  locus, e.g. the double heterozygote <span class="emphasis"><em>m1/+</em></span>; <span class="emphasis"><em>+/m2</em></span> looks like either homozygote <span class="emphasis"><em>m1 +/m1 +</em></span> or <span class="emphasis"><em>+ m2/+ m2</em></span>. Such interactions are not routinely encountered during complementation testing, because such assays are usually done between
                  mutations that have been mapped to the same region and thus have a physical connection on the chromosome. By contrast, this
                  interaction between mutations does not result from their physical location in the genome, but instead reflects a functional
                  connection between different gene products, and often implies that the gene products physically interact. Other experimental
                  approaches may however reveal  that this effect is not uncommon. Specifically, screens for new alleles of a gene  have the
                  potential to uncover non-allelic non-complementing mutations (<a href="#bib18">Hays et al., 1989</a>). Hawley and Walker extensively review second-site non-complementation giving in depth examples of this event found in <span class="emphasis"><em>Drosophila</em></span> and other organisms (<a href="#bib17">Hawley and Walker, 2003</a>). The following examples from <span class="emphasis"><em>C. elegans</em></span> complement their review. Non-allelic non-complementation can be due to two situations: First, one or both mutations act as
                  poisons to protein complexes, thus functional protein complexes become a limiting factor in a process. Second, both mutations
                  reduce a threshold level of gene product, thus the dosage of the individual gene product becomes a limiting factor in a process.
                  These situations, referred to as The Poison and Dosage Models, were put forth by M. Fuller, T. Stearns and D. Botstein and
                  are discussed in more detail below (<a href="#bib14">Fuller et al., 1989</a>; <a href="#bib53">Stearns and Botstein, 1988</a>).
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_5"></a>2.2.1.&nbsp;Non-allelic non-complementation by poison
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In the Poison Model of non-allelic non-complementation, an altered gene product  impairs the protein complex with which it
                     normally associates. Although the complex-poisoning effect of this mutation does not result in a visible phenotype on its
                     own in heterozygotes, a simultaneous mutation in another member of the protein complex can reveal a visible defect. Such interactions
                     have been observed between <span class="symbol">&#945;</span> and <span class="symbol">&#946;</span> tubulin genes in <span class="emphasis"><em>Drosophila</em></span> and yeast, where altered a tubulin act as poisons by either sequestering <span class="symbol">&#946;</span> tubulin or by disrupting the polymerization of the microtubule (<a href="#bib14">Fuller et al., 1989</a>; <a href="#bib18">Hays et al., 1989</a>; <a href="#bib53">Stearns and Botstein, 1988</a>).
                  </p>
                  <p>Such poisonous interactions have also been observed in <span class="emphasis"><em>C. elegans</em></span>; most notably, Kusch and Edgar observed many instances of unconventional allelic interactions in their study of mutations
                     affecting body shape and morphology (<a href="#bib28">Kusch and Edgar, 1986</a>). Specifically, it was noted that <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sqt-1" target="_blank">sqt-1</a></em></span> alleles did not complement certain <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sqt-3" target="_blank">sqt-3</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-8" target="_blank">rol-8</a></em></span> mutations (<a href="#table3" title="Table 3. Complex interactions are observed between collagen genes,&#xA;                        sqt-1, 2, 3&#xA;                        and rol-8">Table 3</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sqt-1" target="_blank">sqt-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sqt-3" target="_blank">sqt-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-8" target="_blank">rol-8</a></em></span> all encode cuticle collagens (<a href="#bib9">Cox et al., 1989</a>; <a href="#bib27">Kramer et al., 1988</a>; <a href="#bib39">Novelli et al., 2004</a>; <a href="#bib56">van der Keyl et al., 1994</a>). Collagens undergo significant amounts of homo- and heteromeric interactions, propeptide processing and crosslinking that
                     result in permanent associations (<a href="#figure6">Figure 6</a>B; <a href="#bib22">Johnstone, 2000</a>; <a href="#bib38">Myllyharju and Kivirikko, 2004</a>; <a href="#bib40">Page and Winter, 2003</a>). Thus, an aberrant collagen monomer can potentially disrupt the cuticle matrix at many stages of cuticle assembly. The prevalence
                     of non-allelic non-complementation among alleles of cuticle collagen genes supports this notion. In addition, mutations among
                     collagen alleles that exhibit non-allelic, non-complementation interactions tend to disrupt a Gly-X-Y repeat domain, which
                     is proposed to be required for monomer associations (<a href="#table3" title="Table 3. Complex interactions are observed between collagen genes,&#xA;                        sqt-1, 2, 3&#xA;                        and rol-8">Table 2</a>, <a href="#figure6">Figure 6</a>; <a href="#bib26">Kramer and Johnson, 1993</a>; <a href="#bib39">Novelli  et al., 2004</a>; <a href="#bib56">van der Keyl et al., 1994</a>). This suggests that mutations in the Gly-X-Y repeats act as poisons by causing disruptions in trimer formation and by sequestering
                     wild-type monomers in non-functional protein interactions thereby decreasing the amount of functional trimers.
                  </p>
                  <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="585">
<tr>
  <td align="center"><p><a href="complementationfig6.jpg"><img src="complementationfig6_s.jpg" border="2" align="middle" alt=" figure 6"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 6. Cuticle collagens have a conserved domain structure.&nbsp;</b>(A) Collagens in vertebrates have been demonstrated to form propeptide homo- or
                                       heteromeric helical trimers. Trimer formation is preceded by the association of the
                                       cysteine containing domains that are specific for each family of collagen. The
                                       association of the cysteine domains also puts the monomers in the right registration for
                                       trimer formation, which is carried out by the Gly-X-Y repeats in a zipping fashion
                                       starting from the C-terminal end. (B) The helical trimer propeptide is further processed
                                       by protease cleavage of the N-terminus by BLI-4 (in SQT-3 the C-terminus may also be a
                                       site of propeptide cleavage; <a href="#bib39">Novelli et al., 2004</a>). After proteolytic
                                       processing, the trimers are cross-linked to other trimers via N-terminal and C-terminal
                                       cross-linking sites to form the collagen matrix.
                        </p>
                     </div>
                  </div>
                  <p>Non-allelic non-complementation has also been observed among genes required for synaptic vesicle fusion (<a href="#bib58">Yook et al., 2001</a>). In particular, a hypomorphic allele of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n2813;class=Variation" target="_blank">n2813</a></em></span>, acts as a poison in synaptic transmission. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> <span class="emphasis"><em>(<a href="http://www.wormbase.org/db/get?name=n2813;class=Variation" target="_blank">n2813</a>)</em></span> is a recessive allele that causes a jerky, Unc phenotype due to decreased synaptic transmission. However, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n2813;class=Variation" target="_blank">n2813</a></em></span> as a trans-heterozygote with mutations in other synaptic function loci exhibits an Unc phenotype even when wild-type copies
                     of both loci are present (E. Jorgensen, pers. comm.). Furthermore, in a sensitive drug assay, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n2813;class=Variation" target="_blank">n2813</a></em></span> exhibits a poisonous effect on synaptic transmission. <a href="http://www.wormbase.org/db/seq/protein?name=UNC-13;class=protein" target="_blank">UNC-13</a> is a diacylglycerol binding protein with multiple C2 Ca<sup>++</sup> binding domains that physically  interact with <a href="http://www.wormbase.org/db/seq/protein?name=UNC-64;class=protein" target="_blank">UNC-64</a>/syntaxin to prime vesicles for fusion to the plasma membrane (<a href="#bib1">Ahmed et al., 1992</a>; <a href="#bib32">Maruyama and Brenner, 1991</a>; <a href="#bib50">Richmond and Broadie, 2002</a>). It was demonstrated that <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span><span class="emphasis"><em>(<a href="http://www.wormbase.org/db/get?name=n2813;class=Variation" target="_blank">n2813</a>)</em></span> did not complement a null or hypomorphic allele of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-64" target="_blank">unc-64</a></em></span>, although a null allele of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> fully complemented the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-64" target="_blank">unc-64</a></em></span> null allele. These results suggest that synaptic transmission is not sensitive to the amount of its individual <a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">UNC-13</a> or <a href="http://www.wormbase.org/db/gene/gene?name=unc-64" target="_blank">UNC-64</a> protein molecules; however, the number of functional <a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">UNC-13</a>/<a href="http://www.wormbase.org/db/gene/gene?name=unc-64" target="_blank">UNC-64</a> protein complexes is a limiting factor. It is possible that <a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">UNC-13</a> <span class="emphasis"><em>(<a href="http://www.wormbase.org/db/get?name=n2813;class=Variation" target="_blank">n2813</a>)</em></span> acts as a poison to synaptic transmission by sequestering wild-type <a href="http://www.wormbase.org/db/gene/gene?name=unc-64" target="_blank">UNC-64</a> into non-functional complexes, thus making the amount of <a href="http://www.wormbase.org/db/seq/protein?name=UNC-64;class=protein" target="_blank">UNC-64</a> a limiting factor to synaptic transmission, following the Poison Model of non-allelic non-complementation (<a href="#figure7">Figure 7</a>A).
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure7"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="477">
<tr>
  <td align="center"><p><a href="complementationfig7.jpg"><img src="complementationfig7_s.jpg" border="2" align="middle" alt=" figure 7"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 7. The Poison and Dosage Models of vesicle priming at the synapse.&nbsp;</b>(A) A schematic of primed vesicles along the active zone at a wild-type synapse. In The
                                       Dosage Model, the <span class="emphasis"><em>unc-13(null)/+</em></span> ; <span class="emphasis"><em>unc-64(null)/+</em></span> trans-heterozygote would have half the
                                       amount of UNC-13 and UNC-64, which would result in a decrease in the number of primed
                                       vesicles ready for synaptic release. In The Poison Model, the <span class="emphasis"><em>unc-13(poison)/+</em></span>;
                                       <span class="emphasis"><em>unc-64(poison)/+</em></span> trans-heterozygote would produce products that would interfere with the
                                       formation of functional complexes. Altered UNC-13 and altered UNC-64 can participate in
                                       the same complex and render it non-functional. In addition these altered products can
                                       form complexes with wild-type partners, sequestering them in non-functional complexes.
                                       These interactions would reduce the level of functional complexes to below half the
                                       normal amount. (B) The synaptic vesicle cycle at a cholinergic synapse. Synaptic vesicles
                                       and associated proteins are transported to the synapse from the cell body by the
                                       synaptic vesicle kinesin protein UNC-104. Acetylcholine, made by CHA-1, is packaged into vesicles by the acetylcholine
                           transporter, UNC-17. Mature synaptic vesicles must be
                                       docked and primed at the synapse so that the vesicle can rapidly fuse with the plasma
                                       membrane when the neuron is depolarized. Docking and priming of the vesicle requires the
                                       dissociation of UNC-18 from UNC-64/syntaxin and the association of UNC-13 with UNC-64. The vesicle is fully primed
                           when SNB-1 joins the UNC-13/UNC-64 complex. After vesicle fusion, the vesicle and its associated proteins are recovered from
                           the plasma membrane through clathrin-mediated endocytosis, which utilizes the <span class="symbol">&#956;</span>2/DPY-23 containing AP-2
                                       adaptor complex (W.S. Davis et al., WBPaper00023077). Recycling
                                       of the synaptic vesicles and their associated proteins is required for maintaining a
                                       readily releasable pool.
                        </p>
                     </div>
                  </div>
                  <p>These results support the view that non-allelic non-complementation signifies a physical interaction between the mutant gene
                     products; however, in the same study it was demonstrated that this is not always the case. Specifically, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> <span class="emphasis"><em>(<a href="http://www.wormbase.org/db/get?name=n2813;class=Variation" target="_blank">n2813</a>)</em></span> failed to complement mutations in other genes required for synaptic vesicle dynamics (<a href="#figure7">Figure 7</a>B). Whereas the greatest degree of non-complementation occurred between mutations in genes whose products are known to have
                     a physical interaction (i.e. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-64" target="_blank">unc-64</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-18" target="_blank">unc-18</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-64" target="_blank">unc-64</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=snb-1" target="_blank">snb-1</a></em></span>), non-complementation was also observed between genes whose products are not known to have a physical interaction (i.e. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=snb-1" target="_blank">snb-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-23" target="_blank">dpy-23</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-104" target="_blank">unc-104</a></em></span>). These observations suggest that the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span>(<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n2813;class=Variation" target="_blank">n2813</a></em></span>) aberrant gene product makes synaptic transmission sensitive to perturbations in other synaptic function loci. In particular,
                     the effects of the <a href="http://www.wormbase.org/db/seq/protein?name=UNC-13;class=protein" target="_blank">UNC-13</a> poison extends to those loci that affect the concentration of synaptic components at the synapse, such as <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-104" target="_blank">unc-104</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-23" target="_blank">dpy-23</a></em></span>. However, there is a limit to this interaction as non-complementation was not observed for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=syd-1" target="_blank">syd-1</a></em></span> trans-heterozygotes, as <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cha-1" target="_blank">cha-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-17" target="_blank">unc-17</a></em></span>  and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=syd-1" target="_blank">syd-1</a></em></span> do not play a direct role in the synaptic vesicle cycle (see references in <a href="#bib58">Yook et al., 2001</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_6"></a>2.2.2.&nbsp;Non-allelic non-complementation by dosage (i.e. combined-haplo-insufficiency)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In the Dosage Model, the limiting factor in a process is the total amount of gene product such that a simultaneous decrease
                     in the levels of expression of both genes results in a mutant phenotype. Dosage-sensitive processes have been reported for
                     developmental pathways where events are controlled by protein gradients, such as observed in <span class="emphasis"><em>Drosophila</em></span> and vertebrates (<a href="#bib21">Jackson and Berg, 1999</a>; <a href="#bib24">Kidd et al., 1999</a>; <a href="#bib47">Rancourt et al., 1995</a>). Few examples of dosage sensitive processes have been reported in <span class="emphasis"><em>C. elegans</em></span>; however, combined-haplo-insufficiency has been reported among <span class="emphasis"><em>ram</em></span>  genes required in male tail ray morphogenesis (<a href="#bib4">Baird and Emmons, 1990</a>). Baird and Emmons demonstrated that null-like mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ram-4" target="_blank">ram-4</a></em></span> do not complement presumed null mutations in any of the other <span class="emphasis"><em>ram</em></span> loci. Specifically, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ram-4" target="_blank">ram-4</a>(bx25ts)</em></span> behaves like the deficiency, <span class="emphasis"><em>mDf9</em></span>, in its trans-heterozygous interactions with mutations at other <span class="emphasis"><em>ram</em></span> loci. Unfortunately, these studies were  limited to assaying gene interactions between <span class="emphasis"><em>ram</em></span> alleles that were not necessarily nulls. Putative null alleles of the <span class="emphasis"><em>ram</em></span> genes have since been obtained and verified and combined-haplo-insufficiency has been observed for some but not all of the
                     <span class="emphasis"><em>ram</em></span> loci (K.L. Chow, pers. comm.). Specifically, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ram-1" target="_blank">ram-1</a>(wx71)</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ram-2" target="_blank">ram-2</a>(<a href="http://www.wormbase.org/db/get?name=bx76;class=Variation" target="_blank">bx76</a>)</em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ram-4" target="_blank">ram-4</a>(bx48)</em></span> exhibit non-allelic non-complementation with one another, but not with <span class="emphasis"><em>ram-6(wx66)</em></span> (K.L. Chow, pers. comm.). Further work has demonstrated that <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ram-1" target="_blank">ram-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ram-2" target="_blank">ram-2</a>/<a href="http://www.wormbase.org/db/gene/gene?name=ram-3" target="_blank">ram-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ram-4" target="_blank">ram-4</a> </em></span>encode cuticular collagens (Tam et al., International <span class="emphasis"><em>C. elegans</em></span> Meeting 2003, 81; Tam et al., in prep.; Yu and Chow, International <span class="emphasis"><em>C. elegans</em></span> Meeting 2001, 424; Yu et al., in prep.).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_7"></a>2.2.3.&nbsp;Other examples of non-allelic non-complementation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Non-allelic non-complementation has been noted in other processes. In particular, in a screen for suppressors of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sog-1" target="_blank">sog-1</a></em></span> alleles were demonstrated to not complement alleles of five other <span class="emphasis"><em>sog</em></span> loci for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> suppression (<a href="#bib30">Maine and Kimble, 1993</a>). In addition, in a screen for suppressors of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-3" target="_blank">rol-3</a></em></span> lethality, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=srl-2" target="_blank"> srl-2</a>(<a href="http://www.wormbase.org/db/get?name=s2506;class=Variation" target="_blank">s2506</a>)</em></span> was demonstrated to not complement mutations  in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=srl-1" target="_blank">srl-1</a></em></span> (<a href="#bib5">Barbazuk et al., 1994</a>). Unfortunately, molecular information is currently unavailable for these genes and mutations.
                  </p>
                  <p>More recently, Chang et al. have reported non-allelic non-complementation between genes required for establishing left-right
                     asymmetry of the ASE chemosensory neurons (<a href="#bib7">Chang et al., 2003</a>). These researchers identified a number of <span class="emphasis"><em>lsy</em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lim-6" target="_blank">lim-6</a></em></span> symmetry) mutants that had altered asymmetric expression of <a href="http://www.wormbase.org/db/get?name=ASEL;class=Cell" target="_blank">ASEL</a> or <a href="http://www.wormbase.org/db/get?name=ASER;class=Cell" target="_blank">ASER</a> specific reporter constructs. Mutations in one class were identified as alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-37" target="_blank">unc-37</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cog-1" target="_blank">cog-1</a></em></span>. Mutations in this class exhibited ectopic expression of an <a href="http://www.wormbase.org/db/get?name=ASEL;class=Cell" target="_blank">ASEL</a> specific reporter, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gcy-7" target="_blank">gcy-7</a>::gfp</em></span>, in <a href="http://www.wormbase.org/db/get?name=ASER;class=Cell" target="_blank">ASER</a>. These  researchers also observed ectopic expression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gcy-7" target="_blank">gcy-7</a>::gfp</em></span>  in <a href="http://www.wormbase.org/db/get?name=ASER;class=Cell" target="_blank">ASER</a> in an <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-37" target="_blank">unc-37</a>(<a href="http://www.wormbase.org/db/get?name=e262;class=Variation" target="_blank">e262</a>)/+</em></span>;<span class="emphasis"><em>+/<a href="http://www.wormbase.org/db/gene/gene?name=cog-1" target="_blank">cog-1</a>(<a href="http://www.wormbase.org/db/get?name=ot28;class=Variation" target="_blank">ot28</a>)</em></span>   trans-heterozygote whereas there is only <a href="http://www.wormbase.org/db/get?name=ASEL;class=Cell" target="_blank">ASEL</a> expression in either heterozygote  alone. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-37" target="_blank">unc-37</a></em></span> encodes the <span class="emphasis"><em>C. elegans</em></span> ortholog of the Groucho transcription co-factor and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cog-1" target="_blank">cog-1</a></em></span> encodes the ortholog of vertebrate Nkx6 type homeobox genes (<a href="#bib41">Palmer et al., 2002</a>; <a href="#bib44">Pflugrad et al., 1997</a>). Biochemical studies   have demonstrated that vertebrate  <a href="http://www.wormbase.org/db/seq/protein?name=COG-1;class=protein" target="_blank">COG-1</a> ortholog interacts with Drosophila Groucho through a conserved engrailed homolog (eh1) domain, thus it is likely  that <a href="http://www.wormbase.org/db/seq/protein?name=UNC-37;class=protein" target="_blank">UNC-37</a> and  <a href="http://www.wormbase.org/db/seq/protein?name=COG-1;class=protein" target="_blank">COG-1</a> physically interact (<a href="#bib37">Muhr et al., 2001</a>). Other <span class="emphasis"><em>lsy</em></span> mutants in the same class exhibit non-allelic non-complementation with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-37" target="_blank">unc-37</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cog-1" target="_blank">cog-1</a></em></span>; however, these have yet to be characterized (Oliver  Hobert pers. comm.).
                  </p>
               </div>
            </div>
            <div class="footnotes"><br><hr100" align="left">
               <div class="footnote">
                  <p><sup><a name="ftn.d0e1392" href="#d0e1392">&#8224;</a></sup>In <span class="emphasis"><em>Drosophila</em></span>, "second-site non-complementation" or SSNC is the predominant title for this interaction, however to avoid confusion with
                     intragenic mutations that modify allelic mutations as in the case of "second-site suppressors", we have opted to stay with
                     the "nonallelic non-complementation" title.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_3"></a>3.&nbsp;Conclusion
                     </h2>
                  </div>
               </div>
            </div>
            <p>Complementation occurs when two mutations together result in a wild-type phenotype. Non-complementation occurs when two mutations
               together result in a mutant phenotype. The complementation test is a simple and fundamental assay in genetics used to assign
               a mutation to a gene. However, if this test does not give a straightforward answer, valuable inferences about gene function
               and interaction may still be obtained if the situation is properly understood.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_4"></a>4.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>This chapter was completed after valuable feedback from members of the Hodgkin Lab, D. Williams, M. Best, K. L. Chow and an
               anonymous reviewer, all to whom I am very appreciative. I am indebted to O. Hobert, K. L. Chow and J. Kimble for  sharing
               unpublished data. KJY was supported by a Ruth L. Kirchstein NRSA (F32A1050333) from NIH.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_5"></a>5.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                             <span class="bibliomisc">Ahmed, S., Maruyama, I.N., Kozma, R., Lee, J., Brenner, S., and Lim, L. (1992). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em>unc-13</em></span> gene product is a phospholipid-dependent high-affinity phorbol ester receptor. Biochem. J. <span class="emphasis"><em>287</em></span> (Pt 3), 995&#8211;999.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1445255&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                             <span class="bibliomisc">Alfonso, A., Grundahl, K., Duerr, J.S., Han, H.P., and Rand, J.B. (1993). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em>unc-17</em></span> gene: a putative vesicular acetylcholine transporter. Science <span class="emphasis"><em>261</em></span>, 617&#8211;619.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8342028&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                             <span class="bibliomisc">Alfonso, A., Grundahl, K., McManus, J.R., Asbury, J.M., and Rand, J.B. (1994). Alternative splicing leads to two cholinergic
                        proteins in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Mol. Biol. <span class="emphasis"><em>241</em></span>, 627&#8211;630.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8057385&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/jmbi.1994.1538" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                             <span class="bibliomisc">Baird, S.E., and Emmons, S.W. (1990). Properties of a class of genes required for ray morphogenesis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>126</em></span>, 335&#8211;344.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2245913&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                             <span class="bibliomisc">Barbazuk, W.B., Johnsen, R.C., and Baillie, D.L. (1994). The generation and genetic analysis of suppressors of lethal mutations
                        in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em>rol-3(V)</em></span> gene. Genetics <span class="emphasis"><em>136</em></span>, 129&#8211;143.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8138151&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                             <span class="bibliomisc">Brenner, S. (1974). The genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>77</em></span>, 71&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4366476&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                             <span class="bibliomisc">Chang, S., Johnston, R.J., Jr., and Hobert, O. (2003). A transcriptional regulatory cascade that controls left/right asymmetry
                        in chemosensory neurons of <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>17</em></span>, 2123&#8211;2137.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12952888&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1117903" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                             <span class="bibliomisc">Changeux, J.P., and Edelstein, S.J. (1998). Allosteric receptors after 30 years. Neuron <span class="emphasis"><em>21</em></span>, 959&#8211;980.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9856454&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(00)80616-9" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                             <span class="bibliomisc">Cox, G.N., Fields, C., Kramer, J.M., Rosenzweig, B., and Hirsh, D. (1989). Sequence comparisons of developmentally regulated
                        collagen genes of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Gene <span class="emphasis"><em>76</em></span>, 331&#8211;344.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2753356&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0378-1119(89)90173-X" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                             <span class="bibliomisc">Crick, F.H., and Orgel, L.E. (1964). The theory of inter-allelic complementation. J. Mol. Biol. <span class="emphasis"><em>8</em></span>, 161&#8211;165.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14149958&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                             <span class="bibliomisc">De Melo, J.V., De Souza, W., and Peixoto, C.A. (2002). Ultrastructural analyses of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> DR 847 <span class="emphasis"><em>bli-1(n361)</em></span> mutant which produces abnormal cuticle blisters. J. Submicrosc. Cytol. Pathol. <span class="emphasis"><em>34</em></span>, 291&#8211;297.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12408362&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                             <span class="bibliomisc">Ferguson, E.L., and Horvitz, H.R. (1985). Identification and characterization of 22 genes that affect the vulval cell lineages
                        of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>110</em></span>, 17&#8211;72.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3996896&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                             <span class="bibliomisc">Francis, R., Barton, M.K., Kimble, J., and Schedl, T. (1995). <span class="emphasis"><em>gld-1</em></span>, a tumor suppressor gene required for oocyte development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>139</em></span>, 579&#8211;606.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7713419&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fuller, M.T., Regan, C.L., Green, L.L., Robertson, B., Deuring, R., and Hays, T.S. (1989). Interacting genes identify interacting
                        proteins involved in microtubule function in <span class="emphasis"><em>Drosophila</em></span>. Cell Motil Cytoskeleton <span class="emphasis"><em>14</em></span>, 128&#8211;135.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2684419&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/cm.970140122" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                             <span class="bibliomisc">Greenwald, I.S., and Horvitz, H.R. (1980). <span class="emphasis"><em>unc-93(e1500)</em></span>: a behavioral mutant of <span class="emphasis"><em>Caenorhabditis elegans</em></span> that defines a gene with a wild-type null phenotype. Genetics <span class="emphasis"><em>96</em></span>, 147&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6894129&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                             <span class="bibliomisc">Gupta, M.C., Graham, P.L., and Kramer, J.M. (1997). Characterization of alpha1(IV) collagen mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span> and the effects of alpha1 and alpha2(IV) mutations on type IV collagen distribution. J. Cell. Biol. <span class="emphasis"><em>137</em></span>, 1185&#8211;1196.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9166417&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.137.5.1185" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hawley, R.S., and Walker, M.Y. (2003). Advanced genetic analysis: finding meaning in a genome (Malden, Massachusetts: Oxford,
                        Blackwell).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hays, T.S., Deuring, R., Robertson, B., Prout, M., and Fuller, M.T. (1989). Interacting proteins identified by genetic interactions:
                        a missense mutation in alpha-tubulin fails to complement alleles of the testis-specific beta-tubulin gene of <span class="emphasis"><em>Drosophila melanogaster</em></span>. Mol. Cell Biol. <span class="emphasis"><em>9</em></span>, 875&#8211;884.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2498648&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hill, R.J., and Sternberg, P.W. (1992). The gene <span class="emphasis"><em>lin-3</em></span> encodes an inductive signal for vulval development in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>358</em></span>, 470&#8211;476.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1641037&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/358470a0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hwang, B.J., and Sternberg, P.W. (2004). A cell-specific enhancer that specifies <span class="emphasis"><em>lin-3</em></span> expression in the <span class="emphasis"><em>C. elegans</em></span> anchor cell for vulval development. Development <span class="emphasis"><em>131</em></span>, 143&#8211;151.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14660442&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.00924" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                             <span class="bibliomisc">Jackson, S.M., and Berg, C.A. (1999). Soma-to-germline interactions during <span class="emphasis"><em>Drosophila</em></span> oogenesis are influenced by dose-sensitive interactions between cut and the genes <span class="emphasis"><em>cappuccino</em></span>, <span class="emphasis"><em>ovarian tumor</em></span> and <span class="emphasis"><em>agnostic</em></span>. Genetics <span class="emphasis"><em>153</em></span>, 289&#8211;303.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10471713&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                             <span class="bibliomisc">Johnstone, I.L. (2000). Cuticle collagen genes. Expression in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Trends Genet. <span class="emphasis"><em>16</em></span>, 21&#8211;27.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10637627&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0168-9525(99)01857-0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                             <span class="bibliomisc">Jones, A.R., and Schedl, T. (1995). Mutations in <span class="emphasis"><em>gld-1</em></span>, a female germ cell-specific tumor suppressor gene in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, affect a conserved domain also found in Src-associated protein Sam68. Genes Dev. <span class="emphasis"><em>9</em></span>, 1491&#8211;1504.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7601353&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kidd, T., Bland, K.S., and Goodman, C.S. (1999). Slit is the midline repellent for the robo receptor in <span class="emphasis"><em>Drosophila</em></span>. Cell <span class="emphasis"><em>96</em></span>, 785&#8211;794.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10102267&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(00)80589-9" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                             <span class="bibliomisc">Killeen, M., Tong, J., Krizus, A., Steven, R., Scott, I., Pawson, T., and Culotti, J. (2002). UNC-5 function requires phosphorylation
                        of cytoplasmic tyrosine 482, but its UNC-40-independent functions also require a region between the ZU-5 and death domains.
                        Dev. Biol. <span class="emphasis"><em>251</em></span>, 348&#8211;366.</span>
                     <a href="http://dx.doi.org/10.1006/dbio.2002.0825" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kramer, J.M., and Johnson, J.J. (1993). Analysis of mutations in the <span class="emphasis"><em>sqt-1</em></span> and <span class="emphasis"><em>rol-6</em></span> collagen genes of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>135</em></span>, 1035&#8211;1045.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8307321&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kramer, J.M., Johnson, J.J., Edgar, R.S., Basch, C., and Roberts, S. (1988). The <span class="emphasis"><em>sqt-1</em></span> gene of <span class="emphasis"><em>C. elegans</em></span> encodes a collagen critical for organismal morphogenesis. Cell <span class="emphasis"><em>55</em></span>, 555&#8211;565.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3180220&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(88)90214-0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kusch, M., and Edgar, R.S. (1986). Genetic studies of unusual loci that affect body shape of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span> and may code for cuticle structural proteins. Genetics <span class="emphasis"><em>113</em></span>, 621&#8211;639.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3732788&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                             <span class="bibliomisc">Liu, J., Tzou, P., Hill, R.J., and Sternberg, P.W. (1999). Structural requirements for the tissue-specific and tissue-general
                        functions of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> epidermal growth factor LIN-3. Genetics <span class="emphasis"><em>153</em></span>, 1257&#8211;1269.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10545457&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                             <span class="bibliomisc">Maine, E.M., and Kimble, J. (1993). Suppressors of<span class="emphasis"><em> glp-1</em></span>, a gene required for cell communication during development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, define a set of interacting genes. Genetics <span class="emphasis"><em>135</em></span>, 1011&#8211;1022.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8307319&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                             <span class="bibliomisc">Malone, C.J., Fixsen, W.D., Horvitz, H.R., and Han, M. (1999). UNC-84 localizes to the nuclear envelope and is required for
                        nuclear migration and anchoring during <span class="emphasis"><em>C. elegans</em></span> development. Development <span class="emphasis"><em>126</em></span>, 3171&#8211;3181.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10375507&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                             <span class="bibliomisc">Maruyama, I.N., and Brenner, S. (1991). A phorbol ester/diacylglycerol-binding protein encoded by the <span class="emphasis"><em>unc-13</em></span> gene of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>88</em></span>, 5729&#8211;5733.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2062851&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                             <span class="bibliomisc">McKay, J.P., Raizen, D.M., Gottschalk, A., Schafer, W.R., and Avery, L. (2004). <span class="emphasis"><em>eat-2</em></span> and <span class="emphasis"><em>eat-18</em></span> are required for nicotinic neurotransmission in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> pharynx. Genetics <span class="emphasis"><em>166</em></span>, 161&#8211;169.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15020415&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.166.1.161" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                             <span class="bibliomisc">Meneely, P.M., and Herman, R.K. (1979). Lethals, steriles and deficiencies in a region of the X chromosome of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>92</em></span>, 99&#8211;115.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=574105&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                             <span class="bibliomisc">Meneely, P.M., and Herman, R.K. (1981). Suppression and function of X-linked lethal and sterile mutations in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>97</em></span>, 65&#8211;84.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7196363&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                             <span class="bibliomisc">Merz, D.C., Zheng, H., Killeen, M.T., Krizus, A., and Culotti, J.G. (2001). Multiple signaling mechanisms of the UNC-6/netrin
                        receptors UNC-5 and UNC-40/DCC <span class="emphasis"><em>in vivo</em></span>. Genetics <span class="emphasis"><em>158</em></span>, 1071&#8211;1080.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11454756&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                             <span class="bibliomisc">Muhr, J., Andersson, E., Persson, M., Jessell, T.M., and Ericson, J. (2001). Groucho-mediated transcriptional repression establishes
                        progenitor cell pattern and neuronal fate in the ventral neural tube. Cell <span class="emphasis"><em>104</em></span>, 861&#8211;873.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11290324&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(01)00283-5" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                             <span class="bibliomisc">Myllyharju, J., and Kivirikko, K.I. (2004). Collagens, modifying enzymes and their mutations in humans, flies and worms. Trends
                        Genet. <span class="emphasis"><em>20</em></span>, 33&#8211;43.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14698617&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.tig.2003.11.004" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                             <span class="bibliomisc">Novelli, J., Ahmed, S., and Hodgkin, J. (2004). Gene interactions in <span class="emphasis"><em>Caenorhabditis elegans</em></span> define DPY-31 as a candidate procollagen C-proteinase and SQT-3/ROL-4 as its predicted major target. Genetics <span class="emphasis"><em>168</em></span>, 1259&#8211;1273.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15579684&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.027953" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                             <span class="bibliomisc">Page, A.P., and Winter, A.D. (2003). Enzymes involved in the biogenesis of the nematode cuticle. Adv. Parasitol. <span class="emphasis"><em>53</em></span>, 85&#8211;148.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14587697&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                             <span class="bibliomisc">Palmer, R.E., Inoue, T., Sherwood, D.R., Jiang, L.I., and Sternberg, P.W. (2002). <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em>cog-1</em></span> locus encodes GTX/Nkx6.1 homeodomain proteins and regulates multiple aspects of reproductive system development. Dev. Biol.
                        <span class="emphasis"><em>252</em></span>, 202&#8211;213.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12482710&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/dbio.2002.0850" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                             <span class="bibliomisc">Park, E.C., and Horvitz, H.R. (1986). <span class="emphasis"><em>C. elegans unc-105</em></span> mutations affect muscle and are suppressed by other mutations that affect muscle. Genetics <span class="emphasis"><em>113</em></span>, 853&#8211;867.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3744029&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                             <span class="bibliomisc">Peters, K., McDowall, J., and Rose, A.M. (1991). Mutations in the <span class="emphasis"><em>bli-4 (I)</em></span> locus of <span class="emphasis"><em>Caenorhabditis elegans</em></span> disrupt both adult cuticle and early larval development. Genetics <span class="emphasis"><em>129</em></span>, 95&#8211;102. </span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1936966&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                             <span class="bibliomisc">Pflugrad, A., Meir, J.Y., Barnes, T.M., and Miller, D.M., III. (1997). The Groucho-like transcription factor UNC-37 functions
                        with the neural specificity gene <span class="emphasis"><em>unc-4</em></span> to govern motor neuron identity in <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>124</em></span>, 1699&#8211;1709.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9165118&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                             <span class="bibliomisc">Pontecorvo, G. (1958). Trends in Genetic Analysis. (New York: Columbia University Press).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                             <span class="bibliomisc">Raizen, D.M., Lee, R.Y., and Avery, L. (1995). Interacting genes required for pharyngeal excitation by motor neuron MC in
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>141</em></span>, 1365&#8211;1382.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8601480&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                             <span class="bibliomisc">Rancourt, D.E., Tsuzuki, T., and Capecchi, M.R. (1995). Genetic interaction between <span class="emphasis"><em>hoxb-5</em></span> and <span class="emphasis"><em>hoxb-6</em></span> is revealed by nonallelic noncomplementation. Genes Dev. <span class="emphasis"><em>9</em></span>, 108&#8211;122.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7828847&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                             <span class="bibliomisc">Rand, J.B. (1989). Genetic analysis of the <span class="emphasis"><em>cha-1-unc-17</em></span> gene complex in <span class="emphasis"><em>Caenorhabditis</em></span>. Genetics <span class="emphasis"><em>122</em></span>, 73&#8211;80.</span>
                             
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed">
                             <span class="bibliomisc">Rand, J.B., and Russell, R.L. (1984). Choline acetyltransferase-deficient mutants of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>106</em></span>, 227&#8211;248.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6698395&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed">
                             <span class="bibliomisc">Richmond, J.E., and Broadie, K.S. (2002). The synaptic vesicle cycle: exocytosis and endocytosis in <span class="emphasis"><em>Drosophila</em></span> and <span class="emphasis"><em>C. elegans</em></span>. Curr. Opin. Neurobiol <span class="emphasis"><em>12</em></span>, 499&#8211;507.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12367628&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0959-4388(02)00360-4" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed">
                             <span class="bibliomisc">Rose, A.M., and Baillie, D.L. (1980). Genetic organization of the region around UNC-15 (I), a gene affecting paramyosin in
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>96</em></span>, 639&#8211;648.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7262541&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sibley, M.H., Johnson, J.J., Mello, C.C., and Kramer, J.M. (1993). Genetic identification, sequence, and alternative splicing
                        of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> alpha 2(IV) collagen gene. J. Cell Biol. <span class="emphasis"><em>123</em></span>, 255&#8211;264.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7691828&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.123.1.255" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed">
                             <span class="bibliomisc">Stearns, T., and Botstein, D. (1988). Unlinked noncomplementation: isolation of new conditional-lethal mutations in each of
                        the tubulin genes of <span class="emphasis"><em>Saccharomyces cerevisiae</em></span>. Genetics <span class="emphasis"><em>119</em></span>, 249&#8211;260.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3294100&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed">
                             <span class="bibliomisc">Thacker, C., Peters, K., Srayko, M., and Rose, A.M. (1995). The <span class="emphasis"><em>bli-4</em></span> locus of <span class="emphasis"><em>Caenorhabditis elegans</em></span> encodes structurally distinct kex2/subtilisin-like endoproteases essential for early development and adult morphology. Genes
                        Dev. <span class="emphasis"><em>9</em></span>, 956&#8211;971.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7774813&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed">
                             <span class="bibliomisc">Thacker, C., Srayko, M., and Rose, A.M. (2000). Mutational analysis of <span class="emphasis"><em>bli-4/kpc-4</em></span> reveals critical residues required for proprotein convertase function in <span class="emphasis"><em>C. elegans</em></span>. Gene <span class="emphasis"><em>252</em></span>, 15&#8211;25.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10903434&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0378-1119(00)00211-0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed">
                             <span class="bibliomisc">van der Keyl, H., Kim, H., Espey, R., Oke, C.V., and Edwards, M.K. (1994). <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em>sqt-3</em></span> mutants have mutations in the <span class="emphasis"><em>col-1</em></span> collagen gene. Dev. Dyn. <span class="emphasis"><em>201</em></span>, 86&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7803850&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/aja.1002010109" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed">
                             <span class="bibliomisc">Wood, W.B. (1988). The nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>, (Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed">
                             <span class="bibliomisc">Yook, K.J., Proulx, S.R., and Jorgensen, E.M. (2001). Rules of nonallelic noncomplementation at the synapse in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>158</em></span>, 209&#8211;220.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11333231&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e10" href="#d0e10">*</a></sup>Edited by Jonathan Hodgkin and Philip Anderson. Last revised October 05, 2005. Published October 06, 2005. This chapter should
                  be cited as: Yook, K. Complementation (October 06, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.24.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2005 Karen Yook. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:karen.yook@bioch.ox.ac.uk" target="_top">karen.yook@bioch.ox.ac.uk</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

