<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Profiling C. elegans gene expression with DNA microarrays	
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMmicroarray_ArticleID" title="Profiling C. elegans gene expression with DNA microarrays&#xA;">
      <link rel="next" href="#d0e138" title="1.&nbsp;Planning your experiment">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="DNAmicroarrays.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMmicroarray_ArticleID"></a>Profiling <span class="emphasis"><em>C. elegans</em></span> gene expression with DNA microarrays<sup><a name="d0e11" href="#ftn.d0e11">*</a></sup>	
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Douglas S.</span> <span class="surname">Portman<em><span class="remark">
                                 					<sup><a name="d0e41" href="#ftn.d0e41">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Center for Aging and Developmental Biology, Department of Biomedical Genetics, University of Rochester School of Medicine
                           and Dentistry, Rochester, NY 14642 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Gene expression
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=303.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=DNA+microarrays" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e138">1. Planning your experiment</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e141">1.1. Test a hypothesis</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e148">1.2. Experimental paradigms</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e268">1.3. Alternatives to DNA microarrays</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e318">1.4. Sources of arrays</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e411">1.5. Determining the scale of your experiment</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e427">2. Protocols</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e430">2.1. Large-scale <span class="emphasis"><em>C. elegans</em></span> culture</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e495">2.2. Sucrose floatation</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e522">2.3. Preparation of synchronous cultures</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e549">2.4. RNA isolation</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e618">2.5. Poly(A)<sup>+</sup> selection</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e626">2.6. RNA Amplification</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e631">2.7. Labeling and hybridization</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e642">3. Following up</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e645">3.1. Data analysis</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e661">3.2. Validation</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e670">3.3. Sharing data</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e694">4. References</a></span></dt>
            </dl>
         </div>
         <p>DNA microarrays, along with high-throughput RNAi screens, are the first fruits of the "functional genomics" movement available
            to <span class="emphasis"><em>C. elegans</em></span> researchers. By immobilizing miniaturized DNA probes that span the genome, DNA microarrays allow the rapid and economical&#8212;at
            least when compared with performing 20,000 Northern blots&#8212;assessment of the level of expression of essentially every <span class="emphasis"><em>C. elegans</em></span> gene. Microarrays have been successfully used by <span class="emphasis"><em>C. elegans</em></span> researchers to probe gene expression in specific tissues including the germline (<a href="#bib24">Reinke et al., 2000</a>), pharynx (<a href="#bib9">Gaudet and Mango, 2002</a>), muscle (<a href="#bib27">Roy et al., 2002</a>) and neurons (<a href="#bib6">Cinar et al., 2005</a>; <a href="#bib7">Colosimo et al., 2004</a>; <a href="#bib8">Fox et al., 2005</a>; <a href="#bib15">Kunitomo et al., 2005</a>; <a href="#bib23">Portman and Emmons, 2004</a>), and to investigate changes in gene expression associated with development (<a href="#bib11">Hill et al., 2000</a>; <a href="#bib12">Jiang et al., 2001</a>), aging (<a href="#bib10">Golden and Melov, 2004</a>; <a href="#bib18">Lund et al., 2002</a>; <a href="#bib19">McCarroll et al., 2004</a>; <a href="#bib20">McElwee et al., 2004</a>; <a href="#bib21">Murphy et al., 2003</a>) and other processes. For a very basic primer on microarray technology, see <a href="http://www.ncbi.nlm.nih.gov/About/primer/microarrays.html" target="_blank">http://www.ncbi.nlm.nih.gov/About/primer/microarrays.html</a>.
         </p>
         <p>At the same time, the seductive power of DNA microarrays can obscure their potential drawbacks. A successful DNA microarray
            experiment requires careful experimental design, a strategy for the validation of hits, and an underlying hypothesis. For
            most <span class="emphasis"><em>C. elegans</em></span> microarray experiments, arrays themselves are merely the means to an end, namely, the identification of specific regulated
            genes. Beware the temptation to carry out a microarray study simply because it is possible: this is an excellent way to become
            mired in a sea of data without learning much about biology.
         </p>
         <p>Because of their highly technical nature, microarray experiments are best done in collaboration with an on-site core facility
            or other group of experts. Most institutions now have functional genomics facilities that can provide the equipment and expertise
            needed. If satisfactory local facilities are not available, it is sometimes possible to "outsource" the work to another institution's
            facility. The choice of a group of collaborators is essential to the success of these experiments; be sure that the group
            you choose has the experience your studies will require before making a commitment.
         </p>
         <p>Because the specific techniques necessary for microarray experiments will depend heavily on experimental design, the source
            of RNA and the microarray platform to be used, this section cannot be an exhaustive source for all methods. In some cases,
            readers are referred other relevant sources. In particular, close consultation with a core facility is advised before, during
            and after carrying out microarray work. Here we provide a discussion of the options for carrying out microarray experiments
            and offer some practical advice for experimental design and for dealing with <span class="emphasis"><em>C. elegans</em></span>-specific issues of nematode culture and RNA isolation. For more general information on constructing, using and interpreting
            data generated by DNA microarrays, refer to one of the many recent texts on the subject.
         </p>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e138"></a>1.&nbsp;Planning your experiment
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e141"></a>1.1.&nbsp;Test a hypothesis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Not surprisingly, the most successful microarray experiments are those that are carefully designed and carried out with attention
                  to detail. Ideally, microarray studies are undertaken to test a specific underlying hypothesis. A hypothesis along the lines
                  of "We believe that specific patterns of gene expression are associated with X" may not seem especially fulfilling, but it's
                  a good starting point. Only if the underlying biology of the system remains the top priority will a microarray experiment
                  fulfill its potential, and only if a strategy exists for interpreting and following up on the data, whether this be at the
                  individual-gene or genome-wide level, should it be considered. At the same time, it's important not to be blinded by a pre-existing
                  hypothesis, as microarray experiments may point you in a direction you hadn't considered previously.
               </p>
               <p>Resist the temptation to be too ambitious. Keep experimental design as simple as possible, ideally changing only one parameter.
                  If your long-term goal is to carry out multivariate analysis&#8212;for example, examining multiple environmental changes at multiple
                  developmental stages in both sexes&#8212;restrict your initial experiments to just one variable. Then vary another dimension while
                  holding the first constant at some optimal value. Varying multiple conditions simultaneously will lead not only to the use
                  of a huge number of arrays, it will excessively complicate data analysis.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e148"></a>1.2.&nbsp;Experimental paradigms
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Two general paradigms have been used for gene expression profiling studies in <span class="emphasis"><em>C. elegans</em></span> using DNA microarrays. In the first, organism-wide changes in gene expression resulting from developmental time, age, sex,
                  environmental factors, or other experimental manipulations are examined. This general approach has been very successful for
                  identifying changes in gene expression patterns without specific focus on one cell or tissue type. Note that this approach
                  certainly does not prevent the identification of genes expressed tissue-specifically, but rather that the emphasis is at an
                  organismal, rather than tissue-specific, level. In these experiments, cultures with sufficient numbers of animals to yield
                  enough RNA for labeling and hybridization are grown. Particular attention must be given to ensuring that the only difference
                  between the samples is the one that the experimenter intends, since very subtle differences in culture conditions, age and
                  other factors can lead to an unacceptably high number of biologically false positives.
               </p>
               <p>The second type of microarray experiment carried out by <span class="emphasis"><em>C. elegans</em></span> biologists aims to identify patterns of gene expression associated with a specific cell or organ type. Because, with the
                  exception of the germline, <span class="emphasis"><em>C. elegans</em></span> tissues are not easily dissected out, and because of the very small amounts of RNA obtained from individual animals, it is
                  not generally possible to isolate particular cell types from intact worms for transcript profiling. Unfortunately, efforts
                  to dissociate cells from larvae or adults and purify specific cell types based on the expression of fluorescent markers have
                  not been successful. Because of this, three alternatives can be considered to carry out these sorts of studies.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e164"></a>1.2.1.&nbsp;Mutant comparison
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>mRNA profiles from intact animals can be compared between wild-type and mutant strains (or between two mutants) in which the
                     tissue of interest is absent or otherwise affected. This approach has been successfully used to identify large numbers of
                     genes expressed specifically in the <span class="emphasis"><em>C. elegans</em></span> germline (<a href="#bib24">Reinke et al., 2000</a>), pharynx (<a href="#bib9">Gaudet and Mango, 2002</a>) and male tail rays (<a href="#bib23">Portman and Emmons, 2004</a>), among others. This approach requires the availability of a viable mutant in which the cells of interest are specifically
                     compromised but the rest of the animal is intact. An advantage of this strategy is that it does not require any complex experimental
                     manipulations to obtain RNA. However, this approach is highly sensitive to any unexpected differences between the two samples
                     (<a href="#bib23">Portman and Emmons, 2004</a>) and to artifactual differences arising from slight differences in the developmental stage or culture conditions in two strains.
                     Moreover, a mutation that affects the tissue of interest with sufficiently high specificity may not be available. Even if
                     one is found, differences in small numbers of cells may be difficult or impossible to detect when set against a constant high
                     background from the rest of the organism.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e188"></a>1.2.2.&nbsp;Sorting cultured cells
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The recent development of embryonic primary cell culture techniques for <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib5">Christensen et al., 2002</a>) provides an excellent opportunity to profile gene expression in specific cell types. The development of cells expressing
                     specific differentiation markers for neurons and muscle cells has been observed <span class="emphasis"><em>in vitro</em></span>, and the generation of other cell types may be possible under the appropriate conditions. By culturing cells isolated from
                     transgenic embryos carrying a fluorescent marker expressed exclusively in the cell type of interest, FACS technology can be
                     used to purify cells carrying the marker. RNA is then isolated from these cells, amplified, and used to produce microarray
                     hybridization probes. This technique has recently been successfully used to profile patterns of gene expression in touch neurons
                     (<a href="#bib26">Zhang et al., 2002</a>), olfactory neurons (<a href="#bib7">Colosimo et al., 2004</a>) and motor neurons (<a href="#bib6">Cinar et al., 2005</a>; <a href="#bib8">Fox et al., 2005</a>). Because of the complexity and continuing optimization of this strategy, specific protocols are not given here for cell
                     culture or sorting; refer to the primary literature for guidance (<a href="#bib5">Christensen et al., 2002</a>; <a href="#bib6">Cinar et al., 2005</a>; <a href="#bib7">Colosimo et al., 2004</a>; <a href="#bib8">Fox et al., 2005</a>; <a href="#bib26">Zhang et al., 2002</a>). The obvious advantage of this approach lies in its potential to identify a large number of specifically expressed genes
                     with a relatively low rate of false positives. Drawbacks, however, include the technically intensive nature of primary cell
                     culture, cell sorting, and the small amount of RNA that is obtained, necessitating amplification procedures that can introduce
                     bias. Additionally, this approach necessitates the availability of a specifically-expressed marker, and requires that the
                     cell type of interest can be efficiently generated in culture (post-embryonically derived cells, for example, do not generally
                     arise in primary embryonic cell cultures; <a href="#bib5">Christensen et al., 2002</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e243"></a>1.2.3.&nbsp;RNA immunopurification
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A third approach to profiling gene expression in a single cell type is the elegant strategy known as "mRNA-tagging." In this
                     approach, cytoplasmic mRNA is epitope-tagged by expressing a FLAG-tagged RNA-binding protein (generally the poly-A binding
                     protein <a href="http://www.wormbase.org/db/seq/protein?name=PAB-1;class=protein" target="_blank">PAB-1</a>) under the control of a cell-type-specific promoter. Animals carrying this transgene are then lysed; mRNAs present in the
                     cell type of interest can be specifically immunopurified using antibodies against the epitope tag. This approach has been
                     successfully used to identify a large number of muscle-specific transcripts (<a href="#bib27">Roy et al., 2002</a>). Recently, this approach has also been productively applied to identify expression profiles in ciliated sensory neurons
                     (<a href="#bib15">Kunitomo et al., 2005</a>). Opportunities for increasing the specificity of mRNA-tagging may come from the use of improved epitope tags, the use of
                     other RNA-binding or ribosomal proteins, and careful control over the expression levels of the tagged transgene.
                  </p>
                  <p>In principle, more complex experiments could be designed that incorporate features of both of these general paradigms for
                     transcriptional profiling. For example, one can imagine an experimental strategy designed to detect changes in gene expression
                     in a particular tissue type that occur with development, aging, or changes in environmental conditions. Clearly, the complexity
                     of these experiments makes them significantly more technically challenging. Lastly, another potential use for DNA microarrays
                     is in the study of DNA-protein interaction. DNA fragments isolated by chromatin immunoprecipitation (ChIP) can be hybridized
                     to a whole-genome "tiling" array to determine the targets of a particular DNA-binding protein at a genomic level (<a href="#bib4">Buck and Lieb, 2004</a>). A <span class="emphasis"><em>C. elegans</em></span> tiling array is not yet available to the community, though this is an area in which we can expect future progress.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e268"></a>1.3.&nbsp;Alternatives to DNA microarrays
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The time, cost, and effort involved in carrying out an array experiment makes the consideration of alternatives worthwhile
                  in some cases.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e273"></a>1.3.1.&nbsp;SAGE
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>This technique, "Serial Analysis of Gene Expression," is the primary competitor to DNA microarrays for the large-scale profiling
                     of transcription patterns (<a href="#bib25">Velculescu et al., 1995</a>). Rather than hybridizing cDNA to immobilized probes, fragments from the 3' ends of cDNAs are ligated together and the resulting
                     concatamers are sequenced. These sequenced tags can then be mapped to infer the specific mRNA from which they arose. Detailed
                     information about this technique is available at: <a href="http://www.sagenet.org" target="_blank">www.sagenet.org</a>. A large-scale SAGE project in <span class="emphasis"><em>C. elegans</em></span> is currently underway at the Genome Sequence Centre in Vancouver (<a href="#bib13">Jones et al., 2001</a>). Advantages to SAGE are that no special equipment other than high-throughput DNA sequencing apparatus is required, unannotated
                     genes can be detected (whereas microarrays can only detect genes for which probes are present), and SAGE may be better at
                     detecting low-abundance mRNAs. SAGE also allows direct quantitation of the relative expression levels of different genes.
                     On the other hand, SAGE can be slower than other methods, since a great deal of sequencing may be required to achieve the
                     coverage provided by a microarray experiment. Moreover, it is difficult to control for biological reproducibility in a SAGE
                     experiment, since pairwise comparisons between biological replicates involve significant effort and have not yet been generally
                     reported. In terms of cost, SAGE is flexible, since it allows control over the amount of sequence information that investigators
                     wish to obtain.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e292"></a>1.3.2.&nbsp;Mine existing data
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Consider the possibility that data already generated by members of the <span class="emphasis"><em>C. elegans</em></span> community might be able to answer some of your questions of interest. A great deal of microarray data is available publicly
                     from the Stanford Microarray Database (<a href="http://genome-www5.stanford.edu/" target="_blank">genome-www5.stanford.edu</a>), <a href="http://www.wormbook.org" target="_blank">Wormbase</a>, and directly from researchers who have performed these experiments. As discussed below, the abundance of data generated
                     by a single array experiment is both a blessing and a curse, and it is virtually certain that creative analysis of existing
                     datasets will reveal new information. It is quite possible that at least some of the answers you seek are already lurking
                     in the dusty corners of someone else's data. Also, consider taking advantage of meta-analyses of array data, clustering approaches
                     that can be helpful in identifying new genes whose expression patterns are related to those of interest (<a href="#bib14">Kim et al., 2001</a>; <a href="#bib17">Lee and Batzoglou, 2003</a>; <a href="#bib22">Owen et al., 2003</a>).
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e318"></a>1.4.&nbsp;Sources of arrays
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The <span class="emphasis"><em>C. elegans</em></span> community was introduced to microarrays through the efforts of Stuart Kim. His group at Stanford developed the first <span class="emphasis"><em>C. elegans</em></span> arrays in the late 1990s, followed by whole-genome arrays shortly thereafter. With funding from NCRR, their facility generously
                  provided hybridization and scanning services free of charge to the <span class="emphasis"><em>C. elegans</em></span> community, a resource which was taken advantage of by many groups. Data from many of these early array experiments are available
                  from the Stanford Microarray Database, (<a href="http://genome-www5.stanford.edu/" target="_blank">genome-www5.stanford.edu</a>). Because of the evolution of array technology and the availability of arrays from other sources, the Stanford group no longer
                  provides a no-cost service to the community at large. However, the Stanford Functional Genomics Facility continues to provide
                  these arrays, as well as hybridization and data analysis services, for a fee; for more information on this option, consult
                  their website, <a href="http://www.microarray.org/sfgf/jsp/home.jsp" target="_blank">http://www.microarray.org/sfgf/jsp/home.jsp</a>. In addition to this service, several alternative options exist for those who wish to take advantage of microarray technology.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e338"></a>1.4.1.&nbsp;Affymetrix
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>This company, one of the pioneers of commercial microarrays, produces "GeneChips" for use with their proprietary hybridization
                     and scanning equipment. The <span class="emphasis"><em>C. elegans</em></span> gene chip, introduced in 2002, contains probes for over 22,500 transcripts, and the precise genomic location of each probe
                     is available from Affymetrix online (<a href="http://www.affymetrix.com" target="_blank">www.affymetrix.com</a>). Several published works have resulted from the use of GeneChip technology (<a href="#bib6">Cinar et al., 2005</a>; <a href="#bib7">Colosimo et al., 2004</a>; <a href="#bib8">Fox et al., 2005</a>; <a href="#bib11">Hill et al., 2000</a>). The GeneChip platform takes advantage of technology whereby short oligos are directly synthesized <span class="emphasis"><em>in situ</em></span> on a glass slide; by constructing a panel of short probe and control oligos for each gene, these arrays are thought to be
                     able to sensitively discriminate between genuine hybridization signals and background noise. In addition, data from these
                     arrays is highly quantitative. Affymetrix arrays are not used for the hybridization of dual-labeled probes on a single slide;
                     rather, because of their high reproducibility, samples are compared between chips. A large user community and extensive experience
                     with this platform at many facilities make the GeneChip a good choice for many users. Additionally, the company provides extensive
                     protocols and software for data analysis. One downside is the cost; Affymetrix chips are not inexpensive and can be used only
                     once and only for one sample. Specific protocols for using GeneChip arrays are not given here; refer to the Affymetrix web
                     site and/or consult your local facility for more information.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e368"></a>1.4.2.&nbsp;Printed long-oligo arrays
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Advances in oligonucleotide-synthesis technology have made long single-stranded oligos a popular choice for use as immobilized
                     probes (<a href="#bib1">Barczak et al., 2003</a>). A number of groups have printed long-oligo chips that are available for a fee to the <span class="emphasis"><em>C. elegans</em></span> community. With support from NHGRI and HHMI, the Genome Sequencing Center at Washington University in St. Louis has produced
                     long-oligo whole-genome <span class="emphasis"><em>C. elegans</em></span> arrays that are available to academic researchers worldwide. While researchers' experience with these arrays is still limited,
                     initial results have been quite promising. Arrays are supplied ready to use; cDNA labeling, hybridization and scanning are
                     performed at the local user's facility-preferably in close consultation with staff that are familiar with the use of printed-oligo
                     chips. Detailed labeling protocols (using Genisphere 3DNA technology) and instructions for hybridization are available from
                     the GSC, though users are free to use the techniques of their choice. For more information on the GSC arrays and recommended
                     protocols, consult their website at: <a href="http://genome.wustl.edu/genome/celegans/microarray/ma_gen_info.cgi" target="_blank">http://genome.wustl.edu/genome/celegans/microarray/ma_gen_info.cgi</a>. For those that do not have a suitable on-site facility, the Microarray Core at Washington University will, for a fee, accept
                     outside users' RNA samples and perform amplification, labeling, hybridization, scanning and data analysis using these arrays.
                  </p>
                  <p>In collaboration with Operon, the Microarray Laboratory at the Samuel Lunenfeld Research Institute of Mount Sinai Hospital
                     in Toronto is also producing whole-genome long-oligo arrays. More information on the "<span class="emphasis"><em>C. elegans</em></span> 20K array" and recommended protocols are available at: <a href="http://www.mshri.on.ca/microarray/" target="_blank">http://www.mshri.on.ca/microarray</a>. Initial results indicate that this chip performs quite well when compared side-by-side to other platforms. This facility
                     will also perform array services to members of the academic community on a fee-for-service basis. As more institutions generate
                     arrays, the resources available to the <span class="emphasis"><em>C. elegans</em></span> research community is likely to grow.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e397"></a>1.4.3.&nbsp;Homemade arrays
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>It is also possible to construct your own microarrays in-house. This is of course not an insignificant undertaking; guidance
                     for this option is well beyond the scope of this report. Currently, at least two commercial sources for premade long oligos
                     covering the <span class="emphasis"><em>C. elegans</em></span> genome exist: the oligos used in the Washington University array are available from Illumina (<a href="http://www.illumina.com" target="_blank">www.illumina.com</a>), and Operon (Qiagen) supplies the oligos used in the Samuel Lunenfeld Research Institute chip (<a href="http://www.operon.com/arrays/oligosets_elegans.php?" target="_blank">www.operon.com/arrays/oligosets_elegans.php?</a>). Given the availability of premade chips from several sources and the effort required to produce arrays in-house, the homebrew
                     option seems ill-advised for most researchers. However, making one's own arrays does allow for the construction of a custom
                     chip containing probes for genes (e.g., miRNAs) not represented on other arrays.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e411"></a>1.5.&nbsp;Determining the scale of your experiment
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e414"></a>1.5.1.&nbsp;How many chips will I need to use?
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>This is a question worth serious thought before undertaking a microarray experiment. Multiple chips are required in order
                     to control for two sources of variability: biological variability arising between independently-prepared RNA samples, and
                     technical variability. No simple answer can be given to the question of chip number since it depends on the design and goals
                     of the experiment being undertaken as well as the array platform being used. The statistical power of the analysis increases
                     with increasing numbers of arrays, as does the cost and amount of sample required. Consulting with your array facility about
                     this the point at which one maximizes experimental value is essential. As a starting point, plan on using at least three arrays
                     to replicate each biological sample; multiply this number by the number of conditions and biological replicates you plan to
                     analyze. When doing two-color hybridizations, many workers also carry out dye-swap controls, as artifacts can arise from the
                     fluorophores themselves; this doubles the number of chips necessary. Dye-swap controls are not necessary with Affymetrix chips&#8212;since
                     each chip is used for only one sample, the same dye can be used to label all cDNA samples. Therefore, experiments using single-channel
                     GeneChips will require the same number of arrays as those using printed two-color arrays with dye-swap controls.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e419"></a>1.5.2.&nbsp;How much RNA do I need?
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>This issue also requires careful consideration. Generally, a single array hybridization will require approximately 2&#8211;5<span class="symbol">&#956;</span>g of total RNA. For more complex methods, such as cell-sorting or RNA tagging, it is difficult to predict the amount of RNA
                     that will be obtained, but it will likely be small enough so that one or two rounds of amplification will be necessary; see
                     below for more information about this technique.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e427"></a>2.&nbsp;Protocols
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e430"></a>2.1.&nbsp;Large-scale <span class="emphasis"><em>C. elegans</em></span> culture
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In general, this is a multi-step procedure. First, a culture containing large numbers of gravid hermaphrodites is grown either
                  in liquid or on plates specially designed to support large populations. Next, worms are recovered from these cultures and
                  treated with alkaline hypochlorite to isolate fertilized eggs; eggs are hatched in food-free medium and arrest as L1 larvae.
                  Upon plating, worms recover from arrest, develop synchronously and are harvested for RNA isolation at the desired stage and
                  under the appropriate conditions. Careful attention to the details of synchronous culture are essential; any small artifactual
                  difference between the samples to be compared can lead to large numbers of false positives in the resulting data.
               </p>
               <div class="procedure"><a name="d0e438"></a><p class="title">Procedure&nbsp;1.&nbsp;Large-scale liquid culture</p>
                  <p>Liquid culture can be a difficult technique; worms grown in liquid are not generally as healthy or fertile as those grown
                     on plates, and cultures are easily contaminated. Nevertheless, some researchers prefer this approach for growing large-scale
                     cultures. Worms grown in liquid should not be used directly for array analysis because the large degree of variability in
                     culture conditions can cause dramatic changes in gene expression patterns.
                  </p>
                  <p>(protocol adapted from Kim lab website)</p>
                  <ol type="1">
                     <li>
                        <p>Prepare a large quantity of <span class="emphasis"><em>E. coli</em></span> NA22 (available from the CGC) using standard culture methods or a fermentor. Add a small amount of M9 or S-basal to the bacterial
                           pellet to form a paste; this can be stored refrigerated or frozen until use. You can expect to obtain a mass of <span class="emphasis"><em>C. elegans</em></span> of roughly half of the amount of bacteria used to feed the worm culture.
                        </p>
                     </li>
                     <li>
                        <p>Wash off worms from 5 to 10 100-mm plates approaching starvation. Transfer these to 500 mL of S-basal medium in a baffled
                           flask and add 1 to 3 ml of bacterial paste. Shake at 20<span class="symbol">&deg;</span>C at 240 rpm.
                        </p>
                     </li>
                     <li>
                        <p>Monitor the progress of the culture daily by removing a small aliquot to an unseeded plate. As the culture becomes less turbid,
                           supplement with an additional 1 to 3 ml of bacterial paste. Adding too much food can cause the cultures to become oxygen-deprived;
                           conversely, too little food will cause starvation, slow growth and dauer formation.
                        </p>
                     </li>
                     <li>
                        <p>As cultures grow, debris will accumulate in the form of small brown "pellets." Once a large number of gravid adults are present
                           (generally 4-5 days), worms are ready to harvest.
                        </p>
                     </li>
                  </ol>
               </div>
               <div class="procedure"><a name="d0e466"></a><p class="title">Procedure&nbsp;2.&nbsp;Large-scale plate culture</p>
                  <p>Plates prepared with an <span class="emphasis"><em>E. coli</em></span>-egg yolk mixture can support the growth of large numbers of worms; however, large numbers of dauers can form under these
                     conditions because of crowding. Nevertheless, the abundance of gravid hermaphrodites may make this an acceptable alternative
                     to liquid culture.
                  </p>
                  <ol type="1">
                     <li>
                        <p>Autoclave 50 ml of LB medium in a flask with a stirring bar and cool to ~60<span class="symbol">&deg;</span>C (just cool enough to touch for a few seconds). Don gloves and rinse with ethanol. Rinse the outside of a supermarket-bought
                           chicken egg with ethanol, crack it on the side of a sterile beaker, and transfer the yolk repeatedly between halves of the
                           shell to remove the egg white while retaining the yolk. (The presence of too much egg white in the media causes the formation
                           of debris that is difficult to remove.) Transfer the yolk to the LB while stirring and stir the mixture at ~60<span class="symbol">&deg;</span>C for 10 min on a heating plate. Cool the mixture to 30<span class="symbol">&deg;</span>C, add 5 ml of an overnight culture of <span class="emphasis"><em>E. coli</em></span> NA22, and stir at room temperature for 15 min. Dispense 5 ml of this mixture onto each of 15 100-mm plates containing 2%
                           Agar-NGM (to prevent burrowing). Allow the plates to dry in a sterile hood until still moist but not "runny".
                        </p>
                     </li>
                     <li>
                        <p>Seed each plate with washed worms from a nearly-starved 100 mm culture. Grow worms at 20<span class="symbol">&deg;</span>C and monitor the status of the culture regularly. These plates should support the growth of at least one generation (~4 days);
                           cultures can be supplemented with NA22 paste as necessary, but too much density can lead to unhealthy worms and dauer formation.
                           Try to harvest once nearly all of the food is exhausted and the surface of the plate is a shimmering, writhing mass of worms.
                           To harvest, rinse plates with M9 or S-basal and spin in a clinical centrifuge. This may need to be done several times to remove
                           bacterial and egg debris.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e495"></a>2.2.&nbsp;Sucrose floatation
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="procedure">
                  <p>Depending on the cleanliness of the culture, worms may need to be cleaned from debris by sucrose floatation before isolating
                     embryos.
                  </p>
                  <ol type="1">
                     <li>
                        <p>Prepare 100 mM NaCl and 60% sucrose and chill to 4<span class="symbol">&deg;</span>C.
                        </p>
                     </li>
                     <li>
                        <p>Spin down worms recovered from liquid or plate culture in a clinical centrifuge using 50 ml conical tubes. Generally 1-2 minutes
                           at a moderate setting is sufficient to yield a pellet; the necessary spin time and speed can be assessed by determining whether
                           significant numbers of gravid hermaphrodites remain in the supernatant.
                        </p>
                     </li>
                     <li>
                        <p>Resuspend pellets in 100 mM cold NaCl and consolidate to 1 or 2 tubes, spinning again if necessary to reduce volume below
                           15 mL total in each tube.
                        </p>
                     </li>
                     <li>
                        <p>Add cold 100 mM NaCl to each tube to bring volume to 22 mL. Mix to resuspend worms and let sit on ice 15 min.</p>
                     </li>
                     <li>
                        <p>Add 22 mL cold 60% sucrose to each tube, invert to mix, and spin 5 min at the highest setting on the clinical centrifuge.</p>
                     </li>
                     <li>
                        <p>With a wide-bore glass pipette (a flamed broken Pasteur pipette works well), carefully but immediately remove the light brown
                           layer of worms floating at the top of the tube (prolonged exposure to 30% sucrose will damage worms). Transfer worms to a
                           new 50 ml tube containing 35 ml cold 100 mM NaCl. Pellet worms by spinning 1 min. Wash with 100 mM NaCl two or three more
                           times to remove residual sucrose.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e522"></a>2.3.&nbsp;Preparation of synchronous cultures
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Pelleted gravid hermaphrodites can be treated with a solution of alkaline hypochlorite; all but the eggshell-protected embryos
                  will be destroyed by this solution. Care must be taken to incubate long enough to destroy all larval and adult carcasses but
                  not so long that embryos are damaged.
               </p>
               <div class="procedure">
                  <ol type="1">
                     <li>
                        <p>Prepare alkaline hypochlorite solution by adding 20 ml of fresh supermarket bleach (4-6% NaOCl) to 50 ml of 1 N NaOH. This
                           solution should be prepared freshly for each use.
                        </p>
                     </li>
                     <li>
                        <p>Add at least 2.5 volumes of alkaline hypochlorite solution to the pelleted worms. Disperse the pellet by pipetting and incubate
                           at room temperature on a rocking platform. Transfer a sample to an unseeded plate with a Pasteur pipette every two or three
                           minutes to monitor progress of treatment. By about 5 minutes, most carcasses should be nearly completely disintegrated; treatment
                           can continue several minutes longer if necessary but eggs will become affected after about ten minutes.
                        </p>
                     </li>
                     <li>
                        <p>Spin 1 minute at high speed in a clinical centrifuge to recover eggs (a soft pale pellet). Wash 3 to 4 times with M9 or S-basal,
                           consolidating eggs into one tube after first wash.
                        </p>
                     </li>
                     <li>
                        <p>Add resuspended, washed egg pellet to 100 ml of S-basal or M9 medium. Incubate on a rotary shaker for 18 to 24 hours at 20<span class="symbol">&deg;</span>C with vigorous shaking to allow embryos to hatch and arrest in L1.
                        </p>
                     </li>
                     <li>
                        <p>Transfer culture to 50 ml tubes and spin down larvae in clinical centrifuge. Resuspend pellet in a small amount of S-basal
                           or M9 medium and transfer to 100 mm OP50-seeded NGM or MYOB plates. Use one plate for each 50<span class="symbol">&#956;</span>l of pelleted larvae. Grow cultures until ready to harvest for RNA isolation. Do not allow cultures to approach starvation;
                           supplement with additional bacterial paste if necessary.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e549"></a>2.4.&nbsp;RNA isolation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Several methods have been used to isolate RNA from intact worms. The primary concerns are to disrupt the cuticle and solubilize
                  RNAs rapidly while preventing degradation by endogenous RNAses. Older methods using guanidinium isothiocyanate solutions have
                  been successful; today many researchers prefer the commercially available reagent Trizol, whose use is described here. Other
                  kits, such as those sold by Qiagen, can also be used; however, worms must be efficiently lysed before RNA can be recovered.
                  When working with RNA, be meticulous about the cleanliness of all reagents and equipment used to avoid degradation. Many researchers
                  use DEPC-treated water to prevent RNAse contamination; however, if any residual DEPC remains after autoclaving, it can create
                  more problems than it solves.
               </p>
               <div class="procedure">
                  <ol type="1">
                     <li>
                        <p>Harvest worms by washing plates with room-temperature M9 medium, using 2-3 mL per plate. Swirl by hand to release worms from
                           the bacterial lawn (check progress under a dissecting microscope) and transfer supernatant to 50 ml conical tubes. Spin down
                           briefly in a clinical centrifuge and wash several times with room-temperature M9 to remove as much bacteria as possible. Because
                           gene expression patterns are sensitive to environmental stresses, minimize the manipulations performed before RNA isolation
                           and be sure that all samples are treated equivalently.
                        </p>
                     </li>
                     <li>
                        <p>Immediately add 5 to 10 volumes of Trizol per volume of pelleted worms. Vortex well to resuspend pellet. Lysis can be achieved
                           either through grinding in liquid nitrogen or by freeze cracking.
                        </p>
                     </li>
                     <li>
                        <ul>
                                                   
                           <li>
                              <p class="title"><b>Grinding</b></p>
                              <p>Transfer Trizol suspension to an RNAse-free porcelain mortar of an appropriate size that has been chilled in a -80<span class="symbol">&deg;</span>C freezer. Gently pour a small amount of liquid nitrogen into the mortar; once bubbling subsides, break up and grind the frozen
                                 suspension with a chilled pestle. Grind the suspension to a fine powder, adding liquid nitrogen to keep the powder cold, about
                                 10 to 15 minutes. Scrape the frozen powder into a conical centrifuge tube and allow it to warm to room temperature.
                              </p>
                           </li>
                                                   
                           <li>
                              <p class="title"><b>Freeze-cracking</b></p>
                              <p>Freeze the Trizol suspension completely in liquid nitrogen; then transfer to a 37<span class="symbol">&deg;</span>C water bath to thaw completely. Repeat this process several times. Though easier on the arms, this method may not solubilize
                                 RNA as effectively as the mortar-and-pestle method.
                              </p>
                           </li>
                                                   
                        </ul>
                     </li>
                     <li>
                        <p>Vortex the suspension vigorously at room temperature for 30 sec; let rest 30 sec and repeat 5 times. Let the mixture stand
                           at room temperature for 5 minutes to disrupt all RNA-protein complexes.
                        </p>
                     </li>
                     <li>
                        <p>Add 0.2 ml of chloroform for each 1ml of Trizol used. Shake tubes vigorously by hand for 15 sec; incubate at room temperature
                           for 2-3 minutes. Spin tubes at 4<span class="symbol">&deg;</span>C at either 10000-12000 g for 15 min or 2600 g for 30-60 min, depending on the properties of the tubes being used.
                        </p>
                     </li>
                     <li>
                        <p>Remove the aqueous (upper) phase containing the RNA; transfer to a new tube and add 0.5 ml isopropanol for each 1 ml of Trizol
                           originally used. Mix and incubate at room temperature for 10 min. Centrifuge again as in step 4. The RNA will form a gel-like
                           pellet on the side and bottom of the tube.
                        </p>
                     </li>
                     <li>
                        <p>Carefully remove the supernatant and wash the pellet with 75% ethanol, using 1 ml 75% ethanol per 1 ml of Trizol. Vortex the
                           sample and spin at 4<span class="symbol">&deg;</span>C at 7500 g for 5 minutes or 2500g for 15 minutes.
                        </p>
                     </li>
                     <li>
                        <p>Carefully remove the supernatant and air-dry the pellet until all traces of ethanol are removed. Do not vacuum-dry the pellet,
                           as this denatures RNA and makes it difficult or impossible to resuspend. Dissolve the RNA pellet in RNAse-free water. Pipetting
                           up and down and/or briefly incubating at 55<span class="symbol">&deg;</span>C can aid in solubilizing the pellet.
                        </p>
                     </li>
                     <li>
                        <p>Assess the quality and quantity of RNA by running an agarose gel and taking spectrophotometric readings at 260 and 280 nm.
                           RNA concentration can be quantitated by assuming that 1 OD corresponds to 40 <span class="symbol">&#956;</span>g/ml. (Alternatively, the Agilent Bioanalyzer, owned by many nucleic acid core facilities, can quantitate and assess the quality
                           of very small RNA samples.) Expected yields of total RNA are in the neighborhood of 1 to 4 mg RNA per gram of worms. On an
                           agarose gel, 18S and 28S ribosomal RNA bands should be clear and crisp with little smearing. An OD<sub>260/280</sub> ratio of 1.8 to 2.0 indicates high-quality RNA; lower readings may indicate RNA unfit for array analysis. High-quality RNA
                           is essential to the generation of high-quality microarray data. Store purified RNA in aliquots at -80<span class="symbol">&deg;</span>C.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e618"></a>2.5.&nbsp;Poly(A)<sup>+</sup> selection
                        </h3>
                     </div>
                  </div>
               </div>
               <p>While not always necessary depending on the labeling method used, the purification of mRNA from total RNA can help to increase
                  the specificity of microarray signals. Several commercial kits are available for this purpose (Ambion, Qiagen, etc.).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e626"></a>2.6.&nbsp;RNA Amplification
                        </h3>
                     </div>
                  </div>
               </div>
               <p>If only limited quantities of RNA are available (as in the cases of mRNA-tagging or cell sorting) amplification of RNA may
                  be necessary. A number of commercial kits are available for this purpose from vendors such as Arcturus and Genisphere. Two
                  rounds of amplification can provide enough material for array experiments from as little as picogram amounts of RNA. The primary
                  concern associated with these techniques is that amplification can be biased; that is, that the relative ratios of mRNA species
                  may not be preserved after amplification. For this reason, it is essential that all samples to be compared undergo identical
                  amplification procedures starting with equivalent amounts of RNA.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e631"></a>2.7.&nbsp;Labeling and hybridization
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Because a variety of strategies for fluorescently labeling cDNA are available, specific methods are not given here. The choice
                  of method depends on the platform of use (Affymetrix or spotted-oligo array) and the experience of your local facility. Consult
                  with your local facility for guidance; often facilities will perform these manipulations themselves to ensure standardization.
                  Recommended labeling protocols for spotted-oligo arrays produced by Washington University and the Samuel Lunenfeld Research
                  Institute are available at their respective websites (<a href="http://genome.wustl.edu/genome/celegans/microarray/ma_gen_info.cgi" target="_blank">http://genome.wustl.edu/genome/celegans/microarray/ma_gen_info.cgi</a> and <a href="http://www.mshri.on.ca/microarray/" target="_blank">www.mshri.on.ca/microarray</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e642"></a>3.&nbsp;Following up
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e645"></a>3.1.&nbsp;Data analysis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The analysis of microarray data is a continuously evolving field that occupies legions of statisticians. A detailed description
                  of methods is beyond the scope of this section; fortunately, however, your local facility is likely to be able to provide
                  you expertise in this area. The particulars of data analysis will depend on the platform you have used as well as the design
                  of your experiment and the nature of the specific questions you would like to ask. A number published and on-line works are
                  good starting points for appreciating the complexity of microarray data analysis, for example, (<a href="#bib2">Bolstad et al., 2004</a>; <a href="#bib26">Storey and Tibshirani, 2003</a>) and (<a href="http://discover.nci.nih.gov/microarrayAnalysis/Microarray.Home.jsp" target="_blank">discover.nci.nih.gov/microarrayAna lysis/Microarray.Home.jsp</a>). Several software packages,  including GenePix, QuantArray, ArrayPro, ArrayVision, are widely used for microarray data analysis;
                  again; your facility can offer guidance, or may perform this analysis for you.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e661"></a>3.2.&nbsp;Validation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The avalanche of data that is supplied by a microarray experiment can be daunting to biologists used to thinking about individual
                  genes. For this reason, it is imperative to have a strategy to deal with your data before you carry out your experiment.
               </p>
               <p>First, test the expression of control genes in your dataset. Are there genes already known to respond to the specific manipulations
                  you are testing? If these genes do not emerge as significantly changed from an analysis of the data, it suggests that you
                  may have many false negatives, or that your analysis is not sufficiently sensitive. Negative controls are also helpful: if
                  housekeeping genes consistently arise as being significantly regulated, it may indicate a large number of false positives.
               </p>
               <p>Second, test on a gene-by-gene level the accuracy of some of the genes identified as highly significant. The ways to go about
                  this will of course depend on the question you are asking; quantitative RT-PCR from independently-isolated RNA samples, or
                  GFP reporters to determine cellular expression patterns, can be useful tools in validating hits from microarray analysis.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e670"></a>3.3.&nbsp;Sharing data
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Making microarrays datasets public has become a precondition for publication for many journals; moreover, it is in keeping
                  with the open-access spirit of the <span class="emphasis"><em>C. elegans</em></span> community. Your data may be useful to others for reasons you may not have forseen, and powerful statistical meta-analyses
                  can take advantage of multiple datasets to address genome-wide issues. The <a href="http://www.ncbi.nlm.nih.gov/geo" target="_blank">NCBI "Geo" database</a> accepts microarray data for deposition, and your local facility may have a means for making your data public. Additionally,
                  several <span class="emphasis"><em>C. elegans</em></span> investigators provide access to their data via laboratory web sites. Finally, <a href="http://www.wormbook.org" target="_blank">Wormbase</a> is an ideal repository for <span class="emphasis"><em>C. elegans</em></span> array data across different platforms. Public databases may require that your data conform to the "MIAME" standard (<a href="#bib3">Brazma et al., 2001</a>); your local facility can advise you on how to prepare your data for distribution to the rest of the scientific world.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e694"></a>4.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Barczak, A., Rodriguez, M.W., Hanspers, K., Koth, L.L., Tai, Y.C., Bolstad, B.M., Speed, T.P., and Erle, D.J. (2003). Spotted
                        long oligonucleotide arrays for human gene expression analysis. Genome Res. <span class="emphasis"><em>13</em></span>, 1775&#8211;1785.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12805270&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.1048803" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">	
                     		<span class="bibliomisc">Bolstad, B.M., Collin, F., Simpson, K.M., Irizarry, R.A., and Speed, T.P. (2004). Experimental design and low-level analysis
                        of microarray data. Int. Rev. Neurobiol. <span class="emphasis"><em>60</em></span>, 25&#8211;58.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15474586&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0074-7742(04)60002-X" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Brazma, A., Hingamp, P., Quackenbush, J., Sherlock, G., Spellman, P., Stoeckert, C., Aach, J., Ansorge, W., Ball, C.A., Causton,
                        H.C., et al. (2001). Minimum information about a microarray experiment (MIAME)-toward standards for microarray data. Nat.
                        Genet. <span class="emphasis"><em>29</em></span>, 365&#8211;371.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11726920&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1201-365" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Buck, M.J., and Lieb, J.D. (2004). ChIP-chip: considerations for the design, analysis, and application of genome-wide chromatin
                        immunoprecipitation experiments. Genomics <span class="emphasis"><em>83</em></span>, 349&#8211;360.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14986705&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ygeno.2003.11.004" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Christensen, M., Estevez, A., Yin, X., Fox, R., Morrison, R., McDonnell, M., Gleason, C., Miller, D.M., III, and Strange,
                        K. (2002). A primary culture system for functional analysis of <span class="emphasis"><em>C. elegans</em></span> neurons and muscle cells. Neuron <span class="emphasis"><em>33</em></span>, 503&#8211;514.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11856526&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(02)00591-3" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Cinar, H., Keles, S., and Jin, Y. (2005). Expression profiling of GABAergic motor neurons in Caenorhabditis elegans. Curr.
                        Biol. <span class="emphasis"><em>15</em></span>, 340&#8211;346.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15723795&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.02.025" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Colosimo, M.E., Brown, A., Mukhopadhyay, S., Gabel, C., Lanjuin, A.E., Samuel, A.D., and Sengupta, P. (2004). Identification
                        of Thermosensory and olfactory neuron-specific genes via expression profiling of single neuron types. Curr. Biol. <span class="emphasis"><em>14</em></span>, 2245&#8211;2251.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15620651&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.12.030" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Fox, R.M., Von Stetina, S.E., Barlow, S.J., Shaffer, C., Olszewski, K.L., Moore, J.H., Dupuy, D., Vidal, M., and Miller, D.M.,
                        III (2005). A gene expression fingerprint of <span class="emphasis"><em>C. elegans</em></span> embryonic motor neurons. BMC Genomics <span class="emphasis"><em>6</em></span>, 42.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15780142&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-6-42" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Gaudet, J., and Mango, S.E. (2002). Regulation of organogenesis by the Caenorhabditis elegans FoxA protein <a href="http://www.wormbase.org/db/seq/protein?name=PHA-4;class=protein" target="_blank">PHA-4</a>. Science <span class="emphasis"><em>295</em></span>, 821&#8211;825.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11823633&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1065175" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Golden, T.R., and Melov, S. (2004). Microarray analysis of gene expression with age in individual nematodes. Aging Cell <span class="emphasis"><em>3</em></span>, 111&#8211;124.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15153179&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1111/j.1474-9728.2004.00095.x" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hill, A.A., Hunter, C.P., Tsung, B.T., Tucker-Kellogg, G., and Brown, E.L. (2000). Genomic analysis of gene expression in
                        <span class="emphasis"><em>C. elegans</em></span>. Science <span class="emphasis"><em>290</em></span>, 809&#8211;812.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11052945&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.290.5492.809" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jiang, M., Ryu, J., Kiraly, M., Duke, K., Reinke, V., and Kim, S.K. (2001). Genome-wide analysis of developmental and sex-regulated
                        gene expression profiles in Caenorhabditis elegans. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>98</em></span>, 218&#8211;223.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11134517&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.011520898" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jones, S.J., Riddle, D.L., Pouzyrev, A.T., Velculescu, V.E., Hillier, L., Eddy, S.R., Stricklin, S.L., Baillie, D.L., Waterston,
                        R., and Marra, M.A. (2001). Changes in gene expression associated with developmental arrest and longevity in Caenorhabditis
                        elegans. Genome Res. <span class="emphasis"><em>11</em></span>, 1346&#8211;1352.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11483575&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.184401" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kim, S.K., Lund, J., Kiraly, M., Duke, K., Jiang, M., Stuart, J.M., Eizinger, A., Wylie, B.N., and Davidson, G.S. (2001).
                        A gene expression map for Caenorhabditis elegans. Science <span class="emphasis"><em>293</em></span>, 2087&#8211;2092.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11557892&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1061603" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kunitomo, H., Uesugi, H., Kohara, Y., and Iino, Y. (2005). Identification of ciliated sensory neuron-expressed genes in Caenorhabditis
                        elegans using targeted pull-down of poly(A) tails. Genome Biol. <span class="emphasis"><em>6</em></span>, R17.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15693946&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2005-6-2-r17" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lee, S.I., and Batzoglou, S. (2003). Application of independent component analysis to microarrays. Genome Biol. <span class="emphasis"><em>4</em></span>, R76.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14611662&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2003-4-11-r76" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     <span class="bibliomisc">Lund, J., Tedesco, P., Duke, K., Wang, J., Kim, S.K., and Johnson, T.E. (2002). Transcriptional profile of aging in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>12</em></span>, 1566&#8211;1573.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12372248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)01146-6" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     		<span class="bibliomisc">McCarroll, S.A., Murphy, C.T., Zou, S., Pletcher, S.D., Chin, C.S., Jan, Y.N., Kenyon, C., Bargmann, C.I., and Li, H. (2004).
                        Comparing genomic expression patterns across species identifies shared transcriptional profile in aging. Nat. Genet. <span class="emphasis"><em>36</em></span>, 197&#8211;204.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14730301&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1291" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     		<span class="bibliomisc">McElwee, J.J., Schuster, E., Blanc, E., Thomas, J.H., and Gems, D. (2004). Shared transcriptional signature in Caenorhabditis
                        elegans Dauer larvae and long-lived <a href="http://www.wormbase.org/db/gene/gene?name=daf-2" target="_blank">daf-2</a> mutants implicates detoxification system in longevity assurance. J. Biol. Chem. <span class="emphasis"><em>279</em></span>, 44533&#8211;44543.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15308663&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1074/jbc.M406207200" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Murphy, C.T., McCarroll, S.A., Bargmann, C.I., Fraser, A., Kamath, R.S., Ahringer, J., Li, H., and Kenyon, C. (2003). Genes
                        that act downstream of <a href="http://www.wormbase.org/db/seq/protein?name=DAF-16;class=protein" target="_blank">DAF-16</a> to influence the lifespan of Caenorhabditis elegans. Nature <span class="emphasis"><em>424</em></span>, 277&#8211;283.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12845331&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature01789" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Owen, A.B., Stuart, J., Mach, K., Villeneuve, A.M., and Kim, S. (2003). A gene recommender algorithm to identify coexpressed
                        genes in <span class="emphasis"><em>C. elegans</em></span>. Genome Res. <span class="emphasis"><em>13</em></span>, 1828&#8211;1837.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12902378&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.1125403" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Portman, D.S., and Emmons, S.W. (2004). Identification of <span class="emphasis"><em>C. elegans</em></span> sensory ray genes using whole-genome expression profiling. Dev. Biol. <span class="emphasis"><em>270</em></span>, 499&#8211;512.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15183729&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2004.02.020" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Reinke, V., Smith, H.E., Nance, J., Wang, J., Van Doren, C., Begley, R., Jones, S.J., Davis, E.B., Scherer, S., Ward, S.,
                        et al. (2000). A global profile of germline gene expression in <span class="emphasis"><em>C. elegans</em></span>. Mol. Cell <span class="emphasis"><em>6</em></span>, 605&#8211;616.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11030340&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)00059-9" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Roy, P.J., Stuart, J.M., Lund, J., and Kim, S.K. (2002). Chromosomal clustering of muscle-expressed genes in Caenorhabditis
                        elegans. Nature <span class="emphasis"><em>418</em></span>, 975&#8211;979.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12214599&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature01012" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Storey, J.D., and Tibshirani, R. (2003). Statistical methods for identifying differentially expressed genes in DNA microarrays.
                        Methods Mol. Biol. <span class="emphasis"><em>224</em></span>, 149&#8211;157.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12710672&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Velculescu, V.E., Zhang, L., Vogelstein, B., and Kinzler, K.W. (1995). Serial analysis of gene expression. Science <span class="emphasis"><em>270</em></span>, 484&#8211;487.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7570003&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Zhang, Y., Ma, C., Delohery, T., Nasipak, B., Foat, B.C., Bounoutas, A., Bussemaker, H.J., Kim, S.K., and Chalfie, M. (2002).
                        Identification of genes expressed in <span class="emphasis"><em>C. elegans</em></span> touch receptor neurons. Nature <span class="emphasis"><em>418</em></span>, 331&#8211;335.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12124626&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature00891" target="_blank">Article</a>
                     	
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e11" href="#d0e11">*</a></sup>Edited by Oliver Hobert. WormMethods editor, Victor Ambros. Last revised May 3, 2005. Published January 20, 2006. This chapter
                  should be cited as: Portman, D. S. Profiling <span class="emphasis"><em>C. elegans</em></span> gene expression with DNA microarrays (January 20, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.104.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Douglas S. Portman. This is an open-access article distributed under the terms of the Creative Commons Attribution
                  License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source
                  are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e41" href="#d0e41">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:douglas.portman@rochester.edu" target="_top">douglas.portman@rochester.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

