<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic mapping and manipulation: Chapter 9-Synthetic and enhancer mutations
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMgmsemch9_ArticleID" title="Genetic mapping and manipulation: Chapter 9-Synthetic and enhancer mutations&#xA;">
      <link rel="next" href="#d0e46" title="1.&nbsp;Basics">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="syntheticenhancermutat.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMgmsemch9_ArticleID"></a>Genetic mapping and manipulation: Chapter 9-Synthetic and enhancer mutations<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">David</span> <span class="surname">Fay<em><span class="remark">
                                 					<sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Department of Molecular Biology, University of Wyoming, Laramie, Wyoming 82071-3944 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Forward genetics and genetic mapping<br />>>> Genetic mapping and manipulation
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=298.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Synthetic+and+enhancer+mutations" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e46">1. Basics</a></span></dt>
               <dt><span class="sect1"><a href="#d0e95">2. Making mapping strains</a></span></dt>
               <dt><span class="sect1"><a href="#d0e209">3. Two-point mapping with genetic markers</a></span></dt>
               <dt><span class="sect1"><a href="#d0e322">4. Three-point mapping with genetic markers</a></span></dt>
               <dt><span class="sect1"><a href="#d0e467">5. Deficiency mapping</a></span></dt>
               <dt><span class="sect1"><a href="#d0e531">6. Using SNPs to map synthetic mutants</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e46"></a>1.&nbsp;Basics
                     </h2>
                  </div>
               </div>
            </div>
            <p>Unfortunately for the field of eukaryotic genetics, mutations in most genes do not appear to cause any obvious or penetrant
               defects (at least under laboratory conditions). Thus, as occurs in the case of <span class="emphasis"><em>C. elegans</em></span> one is left wondering about the cellular and developmental functions of <span class="symbol">&lt;</span>70% of the genes in the genome. In some cases, such genes may ultimately be identified in screens for suppressors or enhancers
               of previously characterized mutations. In addition, our lab and others have developed systematic approaches for identifying
               synthetic genetic interactions (see <a href="http://www.wormbook.org/chapters/www_geneticenhancers/geneticenhancers.html" target="_blank">Genetic enhancers</a>). The difference between a synthetic mutation and an enhancer mutation is largely semantic. At its most extreme, a synthetic
               double-mutant strain <span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span>) may exhibit the phenotype <span class="bold"><strong>C</strong></span> at a very high frequency, whereas neither single mutant ever shows this defect. Alternatively, if both the <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> single mutants display the <span class="bold"><strong>C</strong></span> phenotype 20% of the time, whereas the <span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span> double mutant exhibits the defect 90% of the time, we would probably refer to <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> as an enhancer of <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> or vice versa. Where one draws the line between synthetic mutants and enhancers is arbitrary, although in both cases, the
               combined effects are always more than additive. Finally, keep in mind that both synthetic and enhancer phenomena are completely
               allele dependent. In fact, such hidden functions for genes as revealed by synthetic screens may depend heavily on the isolation
               of rare mutations or weak alleles.
            </p>
            <p>The focus of this section will be on the specific methods our lab has developed to clone bona fide synthetic mutations. Our
               strategy assumes that the molecular identity of one of the two interacting genes is known and that the presence of the known
               mutation can be verified by an observable plate phenotype, a molecular lesion, or through a synthetic interaction with another
               known gene. Some of these approaches will undoubtedly apply to cloning enhancer mutations, particularly when the penetrance
               of the phenotype in the single mutants is very low. In situations where the penetrance of the phenotype in either single mutant
               is reasonably high (<span class="symbol">&lt;</span>25%), however, it may be possible to clone the gene by more straightforward methods. Another significant issue is whether
               or not double-mutant animals are viable (as is the case for synthetic multivulval mutants). Again, our approach is designed
               to address the worst-case scenario where the double mutants are not viable and may not even display an easily recognizable
               plate phenotype. In addition, our method avoids complications resulting from the contribution of maternal products that may
               significantly reduce the penetrance of the phenotype in the progeny of heterozygous animals. The key to the methods described
               below is that although labor intensive, unambiguous results can be obtained that will steadily move the mapping process forward.
               In fact, cloning such synthetic mutations need only be slightly more time consuming than cloning any straightforward mutation.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e95"></a>2.&nbsp;Making mapping strains
                     </h2>
                  </div>
               </div>
            </div>
            <p>Mapping a synthetic mutation requires building marker strains that contain the known mutation in the background. If the known
               mutation has a phenotype, then this should be quite straightforward. In situations where the known mutation has no phenotype
               as a single mutant, however, the question arises: how do you follow its presence or absence? The best way around this is to
               make use of the opposite chromosome as shown in <a href="#figure1">Figure 1</a> (<a href="http://www.wormbook.org/chapters/www_makingcompdmutants.2/makingcompdmutants.html" target="_blank">Making compound mutants</a>). A visible marker is chosen that maps close to the synthetic mutation <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span>, in this case an <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span>. Following mating to N2 males (step 1), the <span class="emphasis"><em><span class="bold"><strong>unc/+</strong></span></em></span> (heterozygous) male is next mated to the desired marker strain (step 2) to generate trans-het males, which are then mated
               into the <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> homozygous strain (step 3). We now identify cross-progeny animals that throw both <span class="bold"><strong>Unc</strong></span> and <span class="bold"><strong>A B</strong></span> progeny (step 4). By identifying an animal that has lost the <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> mutation in the next generation (step 5), we have effectively selected for the <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> homozygous mutation. In step 6, the <span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span> mutations are also homozygosed. In the construction of a <span class="emphasis"><em><span class="bold"><strong>dpy unc</strong></span></em></span> mapping strain, <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> counter markers may be used, although it is necessary that they have phenotypes that are distinguishable from those of the
               <span class="emphasis"><em><span class="bold"><strong>dpy</strong></span></em></span> or <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span> being used as markers.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="gmsemch9-1.jpg"><img src="gmsemch9-1_s.jpg" border="2" align="middle" alt="Synthetic and enhancer mutations figure 9-1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1.&nbsp;</b></p>
               </div>
            </div>
            <p>Although this method requires a fair amount of picking to guarantee selection of the <span class="emphasis"><em><span class="bold"><strong>unc/syn</strong></span></em></span><span class="bold"><strong>A</strong></span>; <span class="emphasis"><em><span class="bold"><strong>ab</strong></span></em></span><span class="bold"><strong>/+</strong></span> animal, it is mostly foolproof, assuming that the counter-marker (in this case <span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span>) is close to the synthetic mutation (<span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span>). Nevertheless, it is wise to generate at least two independent mapping strains to ensure that the correct strain (<span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> homozygous) is obtained. If known synthetic interactors of <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> already exist, these mutations (or <span class="emphasis"><em>RNAi</em></span>) can be used to test for the presence of <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> in the mapping strain. One can also sequence several isolates to verify the presence of the lesion. Alternatively, if the
               mutation creates a polymorphism or is itself a deletion, PCR methods can be effectively used.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e209"></a>3.&nbsp;Two-point mapping with genetic markers
                     </h2>
                  </div>
               </div>
            </div>
            <p>Our described methods for two- and three-point mapping with synthetic mutants require the use of an extrachromosomal array
               that contains rescuing sequences for the known mutation (<span class="emphasis"><em><span class="bold"><strong>syn A</strong></span></em></span>) and a widely expressed GFP marker such as <span class="emphasis"><em>sur-5.</em></span>::GFP. In our case, the arrays we use for mapping are also integral to our isolation of the synthetic mutations. However,
               such an array could also be derived after the fact. The key is that in homozygous double-mutant strains, only those animals
               that segregate the rescuing array are viable. Thus, the array allows one to infer the genotype of the animals with respect
               to the unknown mutation (<span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span>; <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> is homozygous throughout). The basic method for two-point mapping is outlined in <a href="#figure2">Figure 2</a>.
            </p>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="gmsemch9-2.jpg"><img src="gmsemch9-2_s.jpg" border="2" align="middle" alt="Synthetic and enhancer mutations figure 9-2"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 2.&nbsp;</b></p>
               </div>
            </div>
            <p>In step 1, homozygous <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> males are crossed into the mapping strain to generate trans-het males, which are mated to the double-mutant <span class="emphasis"><em><span class="bold"><strong>syn</strong></span></em></span> strain containing the extrachromosomal array (step 2). Depending on whether or not the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation is on the same chromosome as <span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span>, we have two scenarios. In #1, they are on different chromosomes. Therefore, when we identify progeny where <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> is once again homozygous (step 4), 67% of these will throw <span class="bold"><strong>A B</strong></span> progeny. If <span class="emphasis"><em><span class="bold"><strong>syn X</strong></span></em></span> is on the same chromosome and close to the markers, rehomozygosed <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> animals will fail to throw appreciable <span class="bold"><strong>A B</strong></span> progeny. All the basic rules of two-point mapping apply here. In this case the frequency of recombinants will be about twice
               that of the actual map distance (see <a href="http://www.wormbook.org/chapters/www_twopointmapmarkers.2/twopointmapmarkers.html" target="_blank">Two-point mapping</a> for further details; link). Generally speaking, one will want to pick about 100 animals for each chromosome as only 1/4 will
               be rehomozygosed to score. <span class="emphasis"><em>Note that throughout these steps we will pick only Ex+ animals.</em></span></p>
            <p>A second source of information comes from the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> <span class="bold"><strong>non-rehomozygosed</strong></span> plates. When the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation lies on the same chromosome and is close to the markers <span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span>, <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> is essentially balanced. Thus nearly all non-rehomozygosed animals for <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> will throw <span class="bold"><strong>A B</strong></span> animals. (In this case the percentage of recombinants will directly equal the map distance.) In contrast, if <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span> are on separate chromosomes, only 2/3 will throw <span class="bold"><strong>A B</strong></span> progeny.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e322"></a>4.&nbsp;Three-point mapping with genetic markers
                     </h2>
                  </div>
               </div>
            </div>
            <p>Once assigned to a chromosome, three-point mapping can be undertaken. The approach is reminiscent of the method used for two-point
               mapping in that we will seek to rehomozygose the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation using the extrachromosomal array after picking recombinants. <span class="emphasis"><em>An important point in mapping a mutation with no phenotype on its own is that we have to be absolutely sure that the synX
                     mutation is actually present in the generation of animals from which we will pick the recombinants.</em></span> There are essentially two ways to ensure this, as shown in <a href="#figure3">Figure 3</a>.
            </p>
            <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="540">
<tr>
  <td align="center"><p><a href="gmsemch9-3.jpg"><img src="gmsemch9-3_s.jpg" border="2" align="middle" alt="Synthetic and enhancer mutations figure 9-3"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 3.&nbsp;</b></p>
               </div>
            </div>
            <p>One is to pick recombinants only immediately after obtaining the trans-heterozygous strain (scheme #1). This guarantees the
               presence of both <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> and the <span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span> marker. This is an effective way to do three-point mapping initially, provided the markers <span class="emphasis"><em><span class="bold"><strong>a</strong></span></em></span> and <span class="emphasis"><em><span class="bold"><strong>b</strong></span></em></span> are reasonably far apart (i.e., several map units or more). For markers that are closely spaced, it is often necessary to
               pick recombinants over several generations or more to get sufficient numbers (scheme #2). Here, however, we run the risk that
               a recombination event will lead to the loss of the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation, and that this may go undetected. This can happen even though the markers have effectively balanced the silent <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation. To ensure that recombinants are picked only from plates where the parent is a true trans-het, we must make certain
               that ~25% of sibling plates are re-homozygosed. For example, we pick 40 animals (step 4) and 10 turn out to be homozygous
               for <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> (step 5). We then have confidence to pick recombinants off the 30 non-re-homozygosed sibling plates. This strategy can be
               carried out indefinitely (step 6) until sufficient recombinants are obtained. <span class="emphasis"><em>Note that in picking recombinants as well as propagating the trans-het strain, we will only pick Ex+ animals.</em></span></p>
            <p>Once a recombinant has been picked, we need to determine whether or not the recombinant chromosome has acquired the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation, as shown in <a href="#figure4">Figure 4</a>. This turns out to be quite straightforward. In the example on the left, an <span class="bold"><strong>A non-B</strong></span> recombinant has picked up the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation (step 1). This animal will throw additional <span class="bold"><strong>A non-B</strong></span> progeny, 1/3 of which will be homozygous for the recombinant <span class="emphasis"><em><span class="bold"><strong>synX a</strong></span></em></span> chromosome (step 2). These animals will require the presence of the rescuing array and can therefore be scored positively
               for the presence of <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span>. On the right, <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> is not present on the recombinant chromosome and progeny <span class="bold"><strong>will never throw re-homozygosed</strong></span> <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> (step 3). In a typical situation we might pick 15 (Ex +) progeny from an F1 recombinant animal. If the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation has been acquired, 5 animals on average will be homozygous for the recombinant chromosome. For those that are homozygous
               (based on the absence of <span class="emphasis"><em><span class="bold"><strong>a b</strong></span></em></span> progeny) we then determine whether these worms also homozygous for <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> (i.e, they require the array). Although fairly laborious, this approach will generally give unambiguous data points.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="gmsemch9-4.jpg"><img src="gmsemch9-4_s.jpg" border="2" align="middle" alt="Synthetic and enhancer mutations figure 9-4"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 4.&nbsp;</b></p>
               </div>
            </div>
            <p><span class="emphasis"><em>It should be noted that if the mutation lies outside the markers and at some distance, the chance for a second recombination
                     occurring where the syn X mutation is lost in some percentage of the progeny becomes substantial.</em></span> Thus we might have a situation where five animals are clearly homozygous for the <span class="bold"><strong>A non-B</strong></span> chromosome, but only four are rehomozygosed with respect to <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span>. In this case we would count the recombinant as positive for acquiring <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> and would also conclude that <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> is unlikely to lie between the two markers. This added complexity is not a factor when <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> is not initially acquired by the recombinant, as such progeny will never contain the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e467"></a>5.&nbsp;Deficiency mapping
                     </h2>
                  </div>
               </div>
            </div>
            <p>The good news is that we need not construct <span class="bold"><strong>Df</strong></span> strains that are homozygous for the <span class="emphasis"><em><span class="bold"><strong>syn A</strong></span></em></span> mutation. The scheme for <span class="bold"><strong>Df</strong></span> mapping is outlined in <a href="#figure5">Figure 5</a>. Here the <span class="bold"><strong>Df</strong></span> is crossed into the double-mutant strain containing the array (step 2). The key is to unambiguously determine cross-progeny
               at this stage. In the example shown above, the chromosome with <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> contains a visible marker (<span class="emphasis"><em><span class="bold"><strong>unc</strong></span></em></span>) in cis, which is outside of the region covered by the deficiency. Thus, the cross-progeny are <span class="bold"><strong>non-Unc</strong></span>. Alternatively, the double-mutant strain can harbor visible markers on a separate chromosome (such strains are readily obtained
               during two-point mapping). In the scheme above, 1/8 of the final cross-progeny will be both homozygous for <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> and trans-heterozygous for <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> and the <span class="emphasis"><em><span class="bold"><strong>Df</strong></span></em></span> (step 3). Such strains would be identified as <span class="bold"><strong>non-Uncs</strong></span> that throw <span class="bold"><strong>Uncs</strong></span> and (most likely) dead eggs and require the array for viability. As discussed in <a href="http://www.wormbook.org/chapters/www_mappingdeficienduplicat.2/mappingdeficienduplicat.html" target="_blank">chapter 6</a>, a positive result in deficiency mapping is always much more meaningful than a negative one.
            </p>
            <div class="literallayout">
               <p></p>
            </div>
            <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="gmsemch9-5.jpg"><img src="gmsemch9-5_s.jpg" border="2" align="middle" alt="Synthetic and enhancer mutations figure 9-5"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 5.&nbsp;</b></p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e531"></a>6.&nbsp;Using SNPs to map synthetic mutants
                     </h2>
                  </div>
               </div>
            </div>
            <p>Thankfully, SNPs can be fully employed for both two- and three-point mapping of synthetic mutants. The first critical step
               is to construct a strain that harbors your mutation of interest in the CB4856 (Hawaiian) background (<a href="#figure6">Figure 6</a>A). This can be a bit tricky, as CB4856 is not sensitive to RNAi feeding and even germline injection of dsRNA is somewhat
               less potent than it is in N2. The first step is to cross CB4856 males to your known mutation (<span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span>) and then blindly re-isolate candidate homozygous <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> mutants. These isolates can then be tested for the presence of homozygous <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> by dsRNA injections using a known genetic interactor, PCR-based methods if the mutant in <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> causes a RFLP, or direct sequencing. Depending on how many synthetic mutations you anticipate mapping in the future, it may
               be prudent to successively backcross your mutation to CB4856 to obtain <span class="emphasis"><em><span class="bold"><strong>synA</strong></span></em></span> strains that have <span class="symbol">&lt;</span>95% CB4856 DNA content (5 or 6 backcrosses). This is also essential if you are to use SNPs for two-point mapping of your synthetic
               mutations. Alternatively, if you already have your mutation (<span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span>) mapped to a relatively small region, you could theoretically isolate multiple independent mutant strains after just a single
               backcross to CB4856. You would then identify one or more isolates that contain CB4856 sequences throughout your region of
               interest. Of course, all backcrossed strains need to be thoroughly tested to ensure that specific regions are indeed composed
               exclusively of CB4856 sequence.
            </p>
            <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="342">
<tr>
  <td align="center"><p><a href="gmsemch9-6.jpg"><img src="gmsemch9-6_s.jpg" border="2" align="middle" alt="Synthetic and enhancer mutations figure 9-6"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 6.&nbsp;</b></p>
               </div>
            </div>
            <p>Both two-point and three-point mapping methods are largely identical to those already outlined in <a href="http://www.wormbook.org/chapters/www_SNPsintrotwopointmap/SNPsintrotwopointmap.html" target="_blank">SNPs: introduction and two-point mapping</a> and <a href="http://www.wormbook.org/chapters/www_SNPsintrothreepointmap/SNPsintrothreepointmap.html" target="_blank">SNPs: three-point mapping</a>. <a href="#figure6">Figure 6</a>B shows the basic outline for three-point mapping with SNPs. As for the mapping procedures described above, one always picks
               Ex+ animals at each step, and the presence of the <span class="emphasis"><em><span class="bold"><strong>synX</strong></span></em></span> mutation is inferred by the requirement for the array. You may also want to use a two-tiered screening approach as outlined
               in <a href="http://www.wormbook.org/chapters/www_SNPsintrothreepointmap/SNPsintrothreepointmap.html" target="_blank">SNPs: three-point mapping</a> to maximize your efficiency.
            </p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised May 7, 2005. Published February 17, 2006. This chapter should be cited as: Fay, D. Genetic
                  mapping and manipulation: Chapter 9-Synthetic and enhancer mutations (February 17, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.98.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 David Fay. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:davidfay@uwyo.edu" target="_top">davidfay@uwyo.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

