<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Nematode genome evolution
         
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#NematodeGenomeEvol_ArticleID" title="Nematode genome evolution&#xA;       ">
      <link rel="next" href="#sec1_1" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="genomevol.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="NematodeGenomeEvol_ArticleID"></a>Nematode genome evolution<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                            
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Avril</span> <span class="surname">Coghlan<em><span class="remark"><sup><a name="d0e34" href="#ftn.d0e34">&sect;</a></sup><sup><a name="d0e39" href="#ftn.d0e39">&#8224;</a></sup></span></em> 
                                      </span><br></span><span class="affiliation"><span class="address">Conway Institute of Biomolecular and Biomedical Research, University College Dublin, Dublin 4, Ireland
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_nematodeevolecol.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Evolution and ecology
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=283.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Nematode+genome+evolution" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1_1">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#sec1_2">2. Evolution of genome size</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec1_2_1">2.1. The range of genome size across the Nematoda</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_2_2">2.2. What causes genome size to change?</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_2_3">2.3. Does genome size reflect gene count?</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec1_3">3. The evolution of chromosome number and structure</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec1_3_1">3.1. The range of haploid chromosome numbers among nematodes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_3_2">3.2. Have ancient linkage groups been conserved in nematodes?</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_3_3">3.3. Different evolutionary patterns in the arms and centers of nematode chromosomes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_3_4">3.4. The effect of co-expressed genes on chromosomal evolution</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_3_5">3.5. The nematode HOX gene cluster</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_3_6">3.6. Evolution of X and Y chromosomes in nematodes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_3_7">3.7. The effect of the reproductive system on genome stability</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_3_8">3.8. The effect of kinetochore organization on genome stability</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec1_4">4. Evolution of gene content</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec1_4_1">4.1. What genes are necessary to make a nematode?</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_4_2">4.2. Proliferation and loss of gene families</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_4_3">4.3. Species-specific genes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_4_4">4.4. Horizontal gene transfer in the Nematoda</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_4_5">4.5. Identifying "parasitism genes"</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec1_5">5. The rate of evolution in nematodes</a></span></dt>
               <dt><span class="sect1"><a href="#sec1_6">6. Summary</a></span></dt>
               <dt><span class="sect1"><a href="#sec1_7">7. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#sec1_8">8. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>Nematodes are the most abundant type of animal on earth, and live in hot springs, polar ice, soil, fresh and salt water, and
                     as parasites of plants, vertebrates, insects, and other nematodes. This extraordinary ability to adapt, which hints at an
                     underlying genetic plasticity, has long fascinated biologists. The fully sequenced genomes of</strong></span> <span class="emphasis"><em><span class="bold"><strong>Caenorhabditis elegans</strong></span></em></span> <span class="bold"><strong>and</strong></span> <span class="emphasis"><em><span class="bold"><strong>Caenorhabditis briggsae,</strong></span></em></span> <span class="bold"><strong>and ongoing sequencing projects for eight other nematodes, provide an exciting opportunity to investigate the genomic changes
                     that have enabled nematodes to invade many different habitats. Analyses of the</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span> <span class="bold"><strong>and</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. briggsae</strong></span></em></span> <span class="bold"><strong>genomes suggest that these include major changes in gene content; as well as in chromosome number, structure and size. Here
                     I discuss how the data set of ten genomes will be ideal for tackling questions about nematode evolution, as well as questions
                     relevant to all eukaryotes.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>In terms of the numbers of individuals, nematodes are the most abundant type of animal on earth (<a href="#bib77">Platt, 1994</a>). So far 25,000 species have been classified, and there could be 100 million species (<a href="#bib15">Blaxter, 2003</a>; <a href="#bib52">Lambshead, 1993</a>). This abundance results from their ability to adapt, as well as their small size, resistant cuticle, and simple body plan.
               Small changes to their body plan have allowed invasion of many different habitats. Nematodes live in hot springs, polar ice,
               soil, fresh and salt water, and as parasites of plants, vertebrates, insects, and other nematodes (<a href="#bib4">Andr&aacute;ssy and Zombori, 1976</a>). This evolutionary plasticity, which hints at an underlying genetic plasticity, has long fascinated biologists. In 1965,
               the German zoologist Alfred Kaestner wrote "our knowledge concerning the evolution of nematodes is next to nothing." Happily,
               with the genome sequences of the nematodes <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> in hand, and those of <span class="emphasis"><em>C. remanei</em></span>, <span class="emphasis"><em>C. japonica</em></span>, <span class="emphasis"><em>C.</em></span> sp. PB2801, <span class="emphasis"><em>Pristionchus pacificus</em></span>, <span class="emphasis"><em>Haemonchus contortus</em></span>, <span class="emphasis"><em>Meloidogyne hapla</em></span>, <span class="emphasis"><em>Brugia malayi</em></span>, and <span class="emphasis"><em>Trichinella spiralis</em></span> soon to follow, our knowledge is now growing fast. This data set of ten genomes will be ideal for tackling questions about
               nematode evolution, but can also be studied to address questions relevant to all eukaryotes.
            </p>
            <p>In this chapter I will discuss some of the major evolutionary forces that have shaped nematode genomes: how selection seems
               to have acted to preserve operons and clusters of co-expressed genes (see <a href="#sec1_3_4" title="3.4.&nbsp;The effect of co-expressed genes on chromosomal evolution">3.4</a> and <a href="#sec1_3_5" title="3.5.&nbsp;The nematode HOX gene cluster">3.5</a>); how the involvement of the X and Y chromosomes in sex determination may have affected the evolution of their sequence and
               structure (see <a href="#sec1_3_6" title="3.6.&nbsp;Evolution of X and Y chromosomes in nematodes">3.6</a>); how different reproductive strategies may have affected the evolution of chromosome number, structure, and sequence (see
               <a href="#sec1_3_1" title="3.1.&nbsp;The range of haploid chromosome numbers among nematodes">3.1</a>, <a href="#sec1_3_4" title="3.4.&nbsp;The effect of co-expressed genes on chromosomal evolution">3.4</a> and <a href="#sec1_3_7" title="3.7.&nbsp;The effect of the reproductive system on genome stability">3.7</a>); and how a large number of novel genes have arisen and acquired new functions, probably enabling adaptation to new environmental
               niches, such as parasitism (see <a href="#sec1_4_3" title="4.3.&nbsp;Species-specific genes">4.3</a>, <a href="#sec1_4_4" title="4.4.&nbsp;Horizontal gene transfer in the Nematoda">4.4</a> and <a href="#sec1_4_5" title="4.5.&nbsp;Identifying &#34;parasitism genes&#34;">4.5</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_2"></a>2.&nbsp;Evolution of genome size
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_2_1"></a>2.1.&nbsp;The range of genome size across the Nematoda
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Most nematodes have genomes ranging from 50&#8211;250 Mb (<a href="#bib56">Leroy et al., 2003</a>; <a href="#figure1">Figure 1</a>). Among the nematodes being sequenced, sizes vary from 53 Mb for <span class="emphasis"><em>Haemonchus contortus</em></span> (<a href="#bib56">Leroy et al., 2003</a>) to 240 Mb for <span class="emphasis"><em>Trichinella spiralis</em></span> (<a href="#bib42">Hammond and Bianco, 1992</a>). A few nematodes even have genomes as large as those of mammals, such as the ~2100 Mb genome of <span class="emphasis"><em>Parascaris univalens</em></span> (<a href="#bib68">Niedermaier and Moritz, 2000</a>). Other nematode genomes are tiny, such as the ~30 Mb <span class="emphasis"><em>Bursaphelenchus mucronatus</em></span> genome (<a href="#bib56">Leroy et al., 2003</a>). The variation in genome size across the phylum is probably even larger, since sizes have only been estimated for approximately
                  50 species (<a href="#bib56">Leroy et al., 2003</a>; Gregory, 2005, <a href="http://www.genomesize.com" target="_top">Animal Genome Size Databse</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_2_2"></a>2.2.&nbsp;What causes genome size to change?
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Nematode genomes are similar in size to those of flatworms, annelids, and insects (~60-100 Mb upwards), but are smaller than
                  those of some invertebrates such as molluscs and echinoderms (~400-500 Mb upwards; Gregory, 2005, <a href="http://www.genomesize.com" target="_blank">Animal Genome Size Database</a>). The compact nature of nematode genomes may due to a high rate of large, spontaneous deletions (<a href="#bib97">Witherspoon and Robertson, 2003</a>), and perhaps to selection for deletions (<a href="#bib29">Denver et al., 2004</a>).
               </p>
               <p>The <span class="emphasis"><em>C. briggsae</em></span> genome is slightly (~4 Mb) larger than the <span class="emphasis"><em>C. elegans</em></span> genome, due to a larger amount of repetitive DNA in the <span class="emphasis"><em>C. briggsae</em></span> genome (<a href="#bib86">Stein et al., 2003</a>). This must be due to proliferation of repeat families in the <span class="emphasis"><em>C. briggsae</em></span> genome, or loss of repetitive DNA from <span class="emphasis"><em>C. elegans</em></span>. Comparison of the <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> genomes to those of closely related nematodes will shed light on the relative importance of deletions (which will decrease
                  the genome size), versus insertions and proliferation of repeats (which will both increase the genome size).
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="630">
<tr>
  <td align="center"><p><a href="genomeevolutionfig1.jpg"><img src="genomeevolutionfig1_s.jpg" border="2" align="middle" alt=" Nematode genome size figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. Genome size and chromosome number variation across the phylum Nematoda.&nbsp;</b>Species for which whole-genome sequences will soon be available are highlighted in
                                    purple. The phylogeny was adapted from <a href="#bib28">De Ley and Blaxter (2002)</a>. Estimates of genome
                                    size and chromosome number where taken from: <span class="emphasis"><em>Haemonchus contortus</em></span>:John Gilleard (pers. comm.) and <a href="#bib56">Leroy et al. (2003)</a>; <span class="emphasis"><em>Caenorhabditis elegans</em></span>: <a href="#bib86">Stein et al. (2003)</a>; <span class="emphasis"><em>Pristionchus pacificus</em></span>: <a href="#bib83">Sommer et al. (1996)</a>; clade V nematodes: <a href="#bib14">Blaxter (2000)</a>; <span class="emphasis"><em>Meloidogyne hapla</em></span>: David Bird (pers. comm.); <span class="emphasis"><em>Brugia malayi</em></span>: <a href="#bib37">Ghedin et al. (2004)</a>; <span class="emphasis"><em>Trichinella spiralis</em></span>: <a href="#bib42">Hammond and Bianco (1992)</a>
                                    and <a href="#bib66">Mutafova et al. (1982)</a>; other species: <a href="#bib56">Leroy et al. (2003)</a>.
                     </p>
                  </div>
               </div>
               <p><a href="#bib59">Lynch and Conery (2003)</a> suggested that species with smaller effective population sizes (a smaller number of individuals that contribute different
                  alleles to the next generation) have larger genomes, because they tend to accumulate repetitive DNA and genomic duplications.
                  Thus, we expect the genome of <span class="emphasis"><em>C. remanei</em></span> which is currently being sequenced, to be smaller those of <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>, because its effective population size seems to be larger. That is, in a study of two nuclear genes, the diversity in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> was just 6-13% of the diversity seen in <span class="emphasis"><em>C. remanei</em></span> (<a href="#bib39">Graustein et al., 2002</a>). The effective population sizes of parasitic nematodes probably depend on those of their hosts, so parasites of herbivores
                  may have larger effective population sizes than parasites of carnivores or omnivores (<a href="#bib59">Lynch and Conery, 2003</a>). Thus, one could speculate that this explains why the sheep parasite <span class="emphasis"><em>Haemonchus contortus</em></span> has such a small genome (53 Mb) compared to the human parasite <span class="emphasis"><em>Brugia malayi</em></span> (85&#8211;95 Mb) or the pig parasite <span class="emphasis"><em>Trichinella spiralis</em></span> (240 Mb).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_2_3"></a>2.3.&nbsp;Does genome size reflect gene count?
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Since the size difference between the 104 Mb <span class="emphasis"><em>C. briggsae</em></span> and 100 Mb <span class="emphasis"><em>C. elegans</em></span> genomes is due to repetitive DNA, they both have ~19,500 genes (<a href="#bib86">Stein et al., 2003</a>). The <span class="emphasis"><em>Brugia malayi</em></span> genome has a similar size to the <span class="emphasis"><em>Caenorhabditis</em></span> genomes, ~85-95 Mb, and a similar number of genes, ~18,500 genes (<a href="#bib37">Ghedin et al., 2004</a>; <a href="#bib96">Whitton et al., 2004</a>). The <span class="emphasis"><em>Haemonchus contortus</em></span> genome is just 53 Mb, but it is not yet clear whether it contains half as many genes as <span class="emphasis"><em>C. elegans</em></span>, or rather has the same number of genes but half as much non-coding DNA.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_3"></a>3.&nbsp;The evolution of chromosome number and structure
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3_1"></a>3.1.&nbsp;The range of haploid chromosome numbers among nematodes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Most nematodes have haploid chromosome numbers of n=4&#8211;12 (<a href="#bib92">Walton, 1959</a>). The karyotypes of just ~300 species have been studied, but nematodes display a lot of karyotypic variation (<a href="#bib84">&#352;pakulov&aacute; and Casanova, 2004</a>). The lowest haploid number is n=1 in <span class="emphasis"><em>Parascaris univalens</em></span>, but very high counts are seen in polyploid species in the Tylenchomorpha. For example, the race of <span class="emphasis"><em>Meloidogyne hapla</em></span> being sequenced is diploid and has n=14, but another race of <span class="emphasis"><em>M. hapla</em></span> is polyploid with 2n=45&#8211;48 (David Bird, pers. comm.; <a href="#bib88">Triantaphyllou, 1984</a>). Many tylenchomorphs including <span class="emphasis"><em>M. hapla</em></span>, are parthenogens, in which unfertilized eggs develop into new individuals. Animal species that reproduce in this way seem
                  to be susceptible to polyploidization (<a href="#bib70">Otto and Whitton, 2000</a>). The <span class="emphasis"><em>M. hapla</em></span> race being sequenced has twice as many chromosomes as most rhabditines, so could reveal traces of an ancient genome duplication
                  in the Tylenchomorpha.
               </p>
               <p>In contrast to the tylenchomorphs, most rhabditines have n=5-6 (<a href="#bib14">Blaxter, 2000</a>). Indeed, <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> both have n=6, even though their chromosomes have undergone ~4000 rearrangements since they diverged (<a href="#bib86">Stein et al., 2003</a>). The lack of fissions or fusions suggests that there could be selection for a stable chromosome number in the Rhabditina.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3_2"></a>3.2.&nbsp;Have ancient linkage groups been conserved in nematodes?
                        </h3>
                     </div>
                  </div>
               </div>
               <p>When <a href="#bib86">Stein et al. (2003)</a> compared the genome of <span class="emphasis"><em>C. elegans</em></span> to that of <span class="emphasis"><em>C. briggsae</em></span>, they identified ~4800 conserved segments, with an average size of 37 kb. They estimated that there have been 3.6 interchromosomal
                  rearrangements per Mb in the <span class="emphasis"><em>C. briggsae</em></span> genome (<a href="#bib86">Stein et al., 2003</a>). Thus, an average <span class="emphasis"><em>C. briggsae</em></span> chromosome of ~10-20 Mb consists of a mosaic of ~35-70 chunks that match several <span class="emphasis"><em>C. elegans</em></span> chromosomes. However, some of these segments are very small, so it may be possible to detect ancient <span class="emphasis"><em>Caenorhabditis</em></span> linkage groups by considering just the largest conserved segments. A genetic map for <span class="emphasis"><em>C. briggsae</em></span> is currently underway, and should allow us to match each <span class="emphasis"><em>C. briggsae</em></span> chromosome to the <span class="emphasis"><em>C. elegans</em></span> chromosome(s) with which it shares common ancestry (Robert Waterston, Raymond Miller, Scott Baird  and Asif Chinwalla, unpublished
                  data).
               </p>
               <p>Sequencing of random regions of the <span class="emphasis"><em>Pristionchus pacificus</em></span> and <span class="emphasis"><em>Brugia malayi</em></span> genomes suggests that despite the frequent occurrence of reciprocal translocations, ancient secernentean linkage groups may
                  still be detectable. In an 11-gene region sequenced from <span class="emphasis"><em>P. pacificus</em></span> chromosome III, 10/11 genes had orthologs on <span class="emphasis"><em>C. elegans</em></span> chromosome III (<a href="#bib53">Lee et al., 2003</a>). This led Lee et al. to suggest that  <span class="emphasis"><em>P. pacificus</em></span> chromosome III and <span class="emphasis"><em>C. elegans</em></span> chromosome III shared a common ancestor. If this  is true, there must have been a lot of intrachromosomal rearrangement since
                  just three pairs of the <span class="emphasis"><em>P. pacificus</em></span> genes are closely linked in <span class="emphasis"><em>C. elegans</em></span>, but these pairs are scattered over 12 Mb. <a href="#bib96">Whitton et al. (2004)</a> found evidence suggesting that <span class="emphasis"><em>B. malayi</em></span> chromosmes can be matched to their <span class="emphasis"><em>C. elegans</em></span> homologs. They sequenced BAC ends containing 8 Mb of <span class="emphasis"><em>Brugia malayi</em></span> sequence, and found that 60% of the BACs matched the same <span class="emphasis"><em>C. elegans</em></span> chromosome at both ends. However, large rearrangements seem to have occurred within chromosomes, because the average distance
                  between two matches was 4 Mb.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3_3"></a>3.3.&nbsp;Different evolutionary patterns in the arms and centers of nematode chromosomes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Each of <span class="emphasis"><em>Caenorhabditis elegans</em></span>' chromosomes is divided into a repeat-poor "central cluster" that rarely undergoes meiotic exchange, and two repeat-rich
                  "arms" that have a ~7-fold higher recombination rate (<a href="#bib7">Barnes et al., 1995</a>; <a href="#bib20"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>). Intriguingly, the arms are evolving far more rapidly than the centers of chromosomes, in terms of both substitutions and
                  chromosomal rearrangements such as translocations, inversions, and duplications (<a href="#bib20"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>; <a href="#bib86">Stein et al., 2003</a>). This may reflect a lower tolerance to mutation in the central clusters, which contain most of the essential genes and operons
                  (<a href="#bib17">Blumenthal et al., 2002</a>; <a href="#bib48">Kamath et al., 2003</a>). Alternatively, the arms may simply have a higher mutation rate, since the high recombination rate may provoke substitutions
                  (<a href="#bib24">Cutter and Payseur, 2003</a>), while the abundance of repeats probably triggers chromosomal rearrangements (<a href="#bib22">Coghlan and Wolfe, 2002</a>).
               </p>
               <p>Analysis of the <span class="emphasis"><em>Pristionchus pacificus</em></span> genetic map indicates that at least four of the six chromosomes have genetically-defined central clusters and arms (Ralf
                  Sommer, pers. comm.). Are all nematode chromosomes split into genetically defined central clusters and arms, and do the arms
                  always evolve faster than the centers? Comparison of <span class="emphasis"><em>C. elegans</em></span> chromosomes to those other nematodes will allow us to address this question; as well as to test the interesting hypothesis
                  of <a href="#bib7">Barnes et al. (1995)</a> that an ancestor of <span class="emphasis"><em>C. elegans</em></span> had smaller chromosomes consisting of just "clusters", which later sprouted arms due to proliferation of repetitive DNA.
               </p>
               <p><a href="#bib7">Barnes et al. (1995)</a> noticed that the recombination rate in most <span class="emphasis"><em>C. elegans</em></span> autosomes differs by a factor of ~7&#8211;12 between the arms and central clusters. However, in chromosome V, the recombination
                  rate differs by a factor of just four between the arms and cluster. The relatively higher  recombination rate in the central
                  cluster of chromosome V may be a cause (or possibly a result) of its "arm-like" characteristics: its high density of gene
                  families (<a href="#bib20"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium, 1998</a>),  low number of essential genes (<a href="#bib48">Kamath et al., 2003</a>), scarcity of  operons (<a href="#bib17">Blumenthal et al., 2002</a>), abundant species-specific genes (<a href="#bib72">Parkinson et al., 2004</a>), and low probability of sequence matches to <span class="emphasis"><em>Brugia malayi</em></span> BAC end sequences (<a href="#bib96">Whitton et al., 2004</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3_4"></a>3.4.&nbsp;The effect of co-expressed genes on chromosomal evolution
                        </h3>
                     </div>
                  </div>
               </div>
               <p>There are ~1000 operons in the <span class="emphasis"><em>C. elegans</em></span> genome, of which 96% are conserved in <span class="emphasis"><em>C. briggsae</em></span>, far more than expected if selection  selection did not act to preserve operons (60%; <a href="#bib86">Stein et al., 2003</a>). Gene order in ~15% of the genome is stabilized by selection against rearrangements of operons, since 15% of <span class="emphasis"><em>C. elegans</em></span> genes are part of operons (<a href="#bib17">Blumenthal et al., 2002</a>). In fact, operons are concentrated in the central clusters of <span class="emphasis"><em>C. elegans</em></span> chromosomes, so probably contribute to the lower rearrangement rate in the centers compared to the arms (<a href="#bib17">Blumenthal et al., 2002</a>). One <span class="emphasis"><em>C. elegans</em></span> operon is conserved in the closely related rhabditine <span class="emphasis"><em>Oscheius</em></span> (<a href="#bib32">Evans et al., 1997</a>), but at least one <span class="emphasis"><em>C. elegans</em></span> operon has been broken in <span class="emphasis"><em>Pristionchus pacificus</em></span> (<a href="#bib54">Lee and Sommer, 2003</a>). Operons probably exist in the Rhabditina, Tylenchina, and Spirurina, since <span class="emphasis"><em>trans</em></span>-splicing has been observed in <span class="emphasis"><em>Haemonchus contortus</em></span>, <span class="emphasis"><em>Panagrellus redivivus</em></span>, <span class="emphasis"><em>Ascaris suum</em></span>, <span class="emphasis"><em>Anisakis</em></span> spp. and <span class="emphasis"><em>Brugia malayi</em></span> (<a href="#bib8">Bektesh et al., 1988</a>; <a href="#bib87">Takacs et al., 1988</a>). Two unresolved questions are whether <span class="emphasis"><em>Trichinella spiralis</em></span> has <span class="emphasis"><em>trans</em></span>-splicing and operons, and whether nematode operons are related to those in flatworms (<a href="#bib27">Davis and Hodgson, 1997</a>).
               </p>
               <p><span class="emphasis"><em>C. elegans</em></span> chromosomes also contain small clusters of ~2-5 genes that are co-expressed in muscle, even though they do not belong to
                   operons; as well as clusters co-expressed in the germline, intestine, and neurons (<a href="#bib80">Roy et al., 2002</a>, Table 1: intestine=Mountain 08 and neurons=Mountain 06). Furthermore, <span class="emphasis"><em>C. elegans</em></span> genes seem to be partitioned between chromosomes according to their general role: essential genes are  mostly found in the
                  centers of chromosomes I, II and III; while genes involved in  worm behavior and morphology are concentrated on chromosomes
                  II and X (<a href="#bib48">Kamath et al., 2003</a>). This organization may permit co-regulation of genes that sit inside the same chromatin domain (<a href="#bib48">Kamath et al., 2003</a>). There may have been selection against rearrangement of one large cluster of <span class="emphasis"><em>C. elegans</em></span> genes on chromosome II that are expressed during spermatogenesis, since it is part of the largest conserved segment between
                  the <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span>   (<a href="#bib63">Miller et al., 2004</a>). In contrast, two other  spermatogenesis clusters on <span class="emphasis"><em>C. elegans</em></span> chromosome IV are not found in <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib63">Miller et al., 2004</a>). An intriguing possibility is that these two clusters were assembled just in <span class="emphasis"><em>C. elegans</em></span>, by rearrangements that were adaptive because they permitted co-regulation of reproductive genes (<a href="#bib63">Miller et al., 2004</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3_5"></a>3.5.&nbsp;The nematode HOX gene cluster
                        </h3>
                     </div>
                  </div>
               </div>
               <p>HOX genes are transcription factors that are closely clustered in the genomes of most animals (<a href="#bib18">B&uuml;rglin, 1994</a>). They control the expression of anterior-posterior patterning along the body axis during early embryogenesis collinearly
                  with their arrangement on the chromosome. The HOX cluster has been conserved in most animal phyla over hundreds of millions
                  of years of evolution (<a href="#bib33">Ferrier and Holland, 2001</a>), but the nematode HOX cluster is surprisingly poorly preserved. The ancestral bilaterian probably had a cluster of nine
                  HOX genes (nine ortholog groups), but all nematodes have lost at least three ortholog groups (<a href="#bib1">Aboobaker and Blaxter, 2003</a>). A further two ortholog groups were lost in the lineage leading to <span class="emphasis"><em>C. elegans</em></span>, after the Spirurina-Rhabditina-Tylenchina clade diverged from other nematodes. <a href="#bib1">Aboobaker and Blaxter (2003)</a> point out that these two HOX ortholog groups were lost around the time when <span class="emphasis"><em>C. elegans</em></span>' ancestor switched from a regulative mode of development to a deterministic lineage-driven mode. They suggest that perhaps
                  the transition freed the two HOX ortholog groups from their role in anterior-posterior patterning, making their loss tolerable.
                  Interestingly, the HOX cluster has been broken up in <span class="emphasis"><em>C. elegans</em></span>: its six HOX genes (belonging to four ortholog groups) are arranged in three pairs scattered over 5 Mb of chromosome III.
                  <span class="emphasis"><em>Trichinella spiralis</em></span> probably has a regulative mode of development, but it is not yet known whether its HOX genes are clustered. However, even
                  though <span class="emphasis"><em>Brugia malayi</em></span> has lineage-driven development, most of its HOX cluster seems to have been preserved intact (<a href="#bib1">Aboobaker and Blaxter, 2003</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3_6"></a>3.6.&nbsp;Evolution of X and Y chromosomes in nematodes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In <span class="emphasis"><em>Caenorhabditis elegans</em></span>, sex determination acts through an X-chromosome dosage mechanism: animals with two X chromosomes develop as hermaphrodites,
                  whereas XO animals develop as males (<a href="#bib67">Nicoll et al., 1997</a>). XX/XO sex determination is very common across the Nematoda (<a href="#bib91">Walton, 1940</a>), suggesting that the first nematode possibly had XX/XO sex determination. Even if the <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>Trichinella spiralis</em></span>  XX/XO systems did share common ancestry, the traces will be hard to find, since sex  determination pathways and genes are
                  evolving very quickly both in terms of sequence change and gene regulation (reviewed in <a href="#bib41">Haag and Doty, 2005</a>). However, at least one key gene is conserved in the XX/XO sex determination pathways of <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>Pristionchus pacificus</em></span> (<a href="#bib76">Pires-daSilva and Sommer, 2004</a>), so it should be possible to determine whether the  <span class="emphasis"><em>P. pacificus</em></span>, <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>Haemonchus contortus</em></span> XX/XO systems are orthologous.
               </p>
               <p>Only a handful of nematodes have Y chromosomes: <span class="emphasis"><em>Brugia malayi</em></span> (<a href="#bib90">Underwood and Bianco, 1999</a>), <span class="emphasis"><em>Onchocerca volvulus</em></span> (<a href="#bib45">Hirai et al., 1987</a>), <span class="emphasis"><em>Baylisascaris transfuga</em></span> (<a href="#bib65">Mutafova, 1995</a>), <span class="emphasis"><em>Contracaecum incurvum</em></span> (<a href="#bib95">White, 1973</a>), and <span class="emphasis"><em>Trichuris muris</em></span> (<a href="#bib85">&#352;pakulov&aacute; et al., 1994</a>). Since Ys are only known in these few distantly related nematodes, <a href="#bib95">White (1973)</a> suggested that they probably emerged recently. In papaya the sequence of the Y chromosome betrays its recent origin from
                  autosomes (<a href="#bib57">Liu et al., 2004</a>), and it will be interesting to see if the <span class="emphasis"><em>Brugia malayi </em></span> Y arose in a similar way.
               </p>
               <p>The involvement of the <span class="emphasis"><em>C. elegans</em></span> X chromosome in sex determination may have restrained its pace of structural evolution. Since <span class="emphasis"><em>C. elegans</em></span> diverged from <span class="emphasis"><em>C. briggsae</em></span>, its X chromosome has undergone about half as many rearrangements as its autosomes  (<a href="#bib86">Stein et al., 2003</a>). Indeed, two of the three largest conserved segments between the two genomes are on <span class="emphasis"><em>C. elegans</em></span> X (<a href="#bib86">Stein et al., 2003</a>). Furthermore, a genetic linkage map of <span class="emphasis"><em>Pristionchus pacificus</em></span> suggests that the X chromosome may have been preserved largely intact since the divergence of <span class="emphasis"><em>P. pacificus</em></span> from <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib53">Lee et al., 2003</a>; Ralf Sommer, pers. comm.). Thus, although the genes involved in sex determination tolerate high substitution rates, structural
                  rearrangements of the sex chromosome may be more detrimental to the mechanism of sex determination. A possible reason for
                  a lower rate of rearrangement of the X chromosome compared to autosomes is selection against translocations of the regions
                  of the X chromosome from which dosage compensation is initiated (<a href="#bib23">Csankovszki et al., 2004</a>), and/or of regions that contain the "X-signal elements" whose dosage determines the sex of an embryo (<a href="#bib62">Meyer, 2000</a>). X-autosomal translocations may also be deleterious because they upset the dosage of the translocated genes (<a href="#bib69">Ohno, 1967</a>).
               </p>
               <p>In nematode species in which the Y chromosome determines sex, the rate of chromosomal rearrangement in X may be similar to
                  that in autosomes, or perhaps even higher. In <span class="emphasis"><em>Brugia malayi</em></span>, which has a Y chromosome, the X chromosome seems to have undergone more interchromosomal rearrangements than the autosomes.
                  That is, among <span class="emphasis"><em>B. malayi</em></span> BACs that match the <span class="emphasis"><em>C. elegans</em></span> X chromosome at one end, an unusually high number match an autosome at the other end (<a href="#bib96">Whitton et al., 2004</a>). Perhaps the emergence of a Y chromosome in <span class="emphasis"><em>B. malayi</em></span> has liberated its X from any function in sex determination, resulting in relaxation of selection against rearrangements of
                  X.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3_7"></a>3.7.&nbsp;The effect of the reproductive system on genome stability
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The most common reproductive strategy among nematodes is sexual reproduction between males and females (amphimixis), which
                  is seen in <span class="emphasis"><em>Caenorhabditis remanei</em></span>,  <span class="emphasis"><em>Caenorhabditis</em></span> sp. PB2801, <span class="emphasis"><em>Caenorhabditis japonica</em></span>, <span class="emphasis"><em>Haemonchus  contortus</em></span>, <span class="emphasis"><em>Brugia malayi</em></span>, and <span class="emphasis"><em>Trichinella  spiralis</em></span>. However, alternative reproductive strategies have arisen in some nematode groups, including hermaphroditism, parthenogenesis,
                  and haplo-diploidy. For example, <span class="emphasis"><em>C. elegans</em></span>, <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>Pristionchus pacificus</em></span> are hermaphroditic (see <a href="http://www.wormbook.org/chapters/www_evolutionsexdetermin/evolutionsexdetermin.html" target="_blank">The evolution of nematode sex determination</a>), while the strain of <span class="emphasis"><em>Meloidogyne hapla</em></span> being sequenced is a facultative meiotic parthenogen (David Bird, pers. comm.). Because hermaphroditic species (and perhaps
                  parthenogenetic species: <a href="#bib5">Archetti, 2004</a>) have a smaller effective population size than amphimictic species, they will tend to accumulate deleterious mutations, resulting
                  in a faster substitution rate and rate of chromosomal rearrangement (<a href="#bib21">Charlesworth, 1992</a>; <a href="#bib24">Cutter and Payseur, 2003</a>). This may explain why substitution rates in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>Meloidogyne</em></span> seem to be high compared to most nematodes  (<a href="#bib16">Blaxter et al., 1998</a>). We will soon be able to test whether they also have an accelerated rate of chromosomal rearrangement.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3_8"></a>3.8.&nbsp;The effect of kinetochore organization on genome stability
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Since <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> diverged, their chromosomes have been splintered by ~250 reciprocal translocations, ~1400 inversions and ~2700 transpositions
                  (<a href="#bib86">Stein et al., 2003</a>). Intrachromosomal rearrangement is about four times more frequent than interchromosomal rearrangement. Even so, translocations
                  are surprisingly common in <span class="emphasis"><em>Caenorhabditis</em></span> compared to flies, in which translocations are extremely rare (<a href="#bib79">Ranz et al., 2001</a>; <a href="#bib82">Sharakhov et al., 2002</a>). This may be because almost all dipterans have "monocentric" chromosomes, in which the kinetochores assemble on a localized
                  region in each chromosome. In contrast, "holocentric" species such as <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> have diffuse kinetochores that form along the length of their chromosomes during mitosis. Since the kinetochores are the
                  primary chromosomal attachment site for spindle microtubules, they play a key role in ensuring high fidelity chromosome transmission
                  in both monocentric and holocentric species. However, little is known of the relationship between the distribution of kinetic
                  activity along chromosomes and the pattern of chromosomal rearrangement. In species with monocentric chromosomes, many translocations
                  will be lethal because they will give rise to acentric or dicentric chromosomes; while species with holocentric chromosomes
                  may be more tolerant of translocations (<a href="#bib30">Dernburg, 2001</a>). Most nematodes have holocentric chromosomes, but <span class="emphasis"><em>Trichinella spiralis</em></span> and some other  trichinellids have monocentric chromosomes (<a href="#bib66">Mutafova et al., 1982</a>; <a href="#bib85">&#352;pakulov&aacute; et al., 1994</a>). Thus, comparison of the <span class="emphasis"><em>T. spiralis</em></span> genome to that of <span class="emphasis"><em>C. elegans</em></span> may provide clues as to whether holocentric chromosomes are more susceptible to rearrangement, and whether the first nematode
                  had holocentric chromosomes.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_4"></a>4.&nbsp;Evolution of gene content
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_4_1"></a>4.1.&nbsp;What genes are necessary to make a nematode?
                        </h3>
                     </div>
                  </div>
               </div>
               <p><a href="#bib72">Parkinson et al. (2004)</a> sequenced ESTs from 30 different nematode species across the phylum, and defined ~94,000 genes from ~60,000 families. Surprisingly,
                  only about 15,000 (15%) of the ~94,000 genes are found in all four clades of nematodes studied (Rhabditina, Tylenchina, Spirurina,
                  Dorylaimia). These 15,000 genes are probably involved in core metabolic or structural pathways, since most of them (91%) have
                  sequence matches outside the Nematoda. In addition, they  identified ~1300 genes that are nematode-specific but that are found
                  in most nematodes. These ~1300 genes probably have roles that are important for nematode  body plan and life history, and
                  so may shed light on the early evolution of the phylum.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_4_2"></a>4.2.&nbsp;Proliferation and loss of gene families
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Since <span class="emphasis"><em>C. elegans</em></span> has diverged from <span class="emphasis"><em>C. briggsae</em></span>, chemoreceptors have proliferated in the <span class="emphasis"><em>C. elegans</em></span> genome, so that it now has almost twice as many as <span class="emphasis"><em>C. briggsae</em></span> (718 versus 429; <a href="#bib86">Stein et al., 2003</a>). Duplication and divergence of extra chemoreceptors may have allowed <span class="emphasis"><em>C. elegans</em></span> to adopt a slightly different ecological niche than <span class="emphasis"><em>C. briggsae</em></span>, since it uses chemoreceptors to find food, and to avoid predators, pathogens and toxins (<a href="#bib89">Troemel, 1999</a>). On the other hand, <span class="emphasis"><em>C. elegans</em></span> seems to have lost several genes (~30 genes)  that are found in both <span class="emphasis"><em>Pristionchus pacificus</em></span> and <span class="emphasis"><em>Haemonchus contortus</em></span> (<a href="#bib72">Parkinson et al., 2004</a>). For example, <span class="emphasis"><em>C. elegans</em></span> has lost a DNA methyltransferase gene that is found in <span class="emphasis"><em>P. pacificus</em></span> &#8212; a loss that probably led to the abolition of DNA methylation in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib40">Gutierrez and Sommer, 2004</a>). Contrasting the gene families that have been duplicated or lost in each of the ten nematode genomes may reveal selection
                  for different gene contents in different species.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_4_3"></a>4.3.&nbsp;Species-specific genes
                        </h3>
                     </div>
                  </div>
               </div>
               <p> <span class="emphasis"><em>C. elegans</em></span> has ~1000 genes that are not found in <span class="emphasis"><em>C. briggsae</em></span>, and that lack any match in sequence databases (<a href="#bib86">Stein et al., 2003</a>). Of these, ~200 have been confirmed by EST or cDNA data, so are definitely not gene prediction errors. These genes may have
                  diverged so rapidly that their <span class="emphasis"><em>C. briggsae</em></span> homolog is unrecognizable; or may have been assembled <span class="emphasis"><em>de novo</em></span> via chromosomal rearrangements in the  <span class="emphasis"><em>C. elegans</em></span> genome (<a href="#bib58">Long, 2001</a>). Duplications, chromosomal rearrangements and transposable elements are known to play a role in the  birth of novel genes
                  (<a href="#bib11">Betran and Long, 2002</a>; <a href="#bib35">Ganko et al., 2003</a>; <a href="#bib58">Long, 2001</a>). Thus, the abundance of species-specific genes in the arms of <span class="emphasis"><em>C. elegans</em></span> chromosomes probably results from the arms' high rate of rearrangement (<a href="#bib86">Stein et al., 2003</a>).
               </p>
               <p><span class="emphasis"><em>C. elegans</em></span> is not alone in having so many species-specific genes. In a survey of ESTs from 30 nematode species, as many as 30-50% of
                  genes in each species seemed to be species-specific (<a href="#bib72">Parkinson et al., 2004</a>). Among the nematodes whose genomes are being sequenced, the fraction ranged from 19% for <span class="emphasis"><em>Haemonchus contortus</em></span> to 37% for <span class="emphasis"><em>Trichinella spiralis</em></span>. This is far higher than the fraction of <span class="emphasis"><em>C. elegans</em></span> genes that are species-specific (5%). However, some of the putative species-specific <span class="emphasis"><em>T. spiralis</em></span> genes may not be truly species-specific, but rather are just very divergent; if so, we might be able to identify their <span class="emphasis"><em>C. elegans</em></span> orthologs by using synteny information. Furthermore, some <span class="emphasis"><em>T. spiralis</em></span> EST consensus sequences were probably too short to find sequences matches; we may be able to detect <span class="emphasis"><em>C. elegans</em></span> orthologs of these <span class="emphasis"><em>T. spiralis</em></span> genes once we know their full-length sequences.
               </p>
               <p>Nematode genespace is probably far larger than 60,000 gene families. Firstly, Parkinson et al.'s survey probably missed many
                  genes expressed at low levels and genes whose expression is restricted to a small number of specialized cells, which are difficult
                  to detect in an EST survey. Secondly, they only sampled 30 species out of at least 25,000 species. Furthermore, all of the
                  species sampled so far are terrestrial, and do not yet include any of the large number of poorly sampled enoplid and chromadorid
                  marine nematodes (<a href="#bib52">Lambshead, 1993</a>). Since the first nematode was probably a marine animal (<a href="#bib78">Poinar, 1983</a>), perhaps an enoplid (<a href="#bib15">Blaxter, 2003</a>), even more diversity in gene content may be found in the oceans.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_4_4"></a>4.4.&nbsp;Horizontal gene transfer in the Nematoda
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Horizontal gene transfer occurs frequently in prokaryotes, but seems to be rare in eukaryotes. For example, ~1% of the gene
                  repertoire in the nematode <span class="emphasis"><em>Meloidogyne</em></span> probably originated by horizontal transfer (<a href="#bib81">Scholl et al., 2003</a>), compared to 1-5% of single-copy genes and at least 22% of gene duplicates in Y-Proteobacteria (<a href="#bib55">Lerat et al., 2005</a>) <span class="emphasis"><em>Meloidogyne hapla</em></span>, a plant parasitic nematode, seems to have gained at least a dozen genes by horizontal gene transfer from bacteria that occupy
                  similar niches in the soil and roots (<a href="#bib81">Scholl et al., 2003</a>). Those genes gained are useful for the nematode's parasitic lifestyle, such as cellulases for digesting plant material,
                  and signaling molecules that induce morphological changes in the plant, facilitating invasion (McCarter et al., 2003; <a href="#bib94">Weerasinghe et al., 2005</a>). A distantly related plant parasite, <span class="emphasis"><em>Bursaphelenchus xylophilus</em></span>, seems to have independently acquired a cellulase gene from a fungus (<a href="#bib50">Kikuchi et al., 2004</a>). Perhaps horizontal transfer can spur the transition to parasitism (<a href="#bib94">Weerasinghe et al., 2005</a>)?
               </p>
               <p>Several groups of parasitic nematodes, including <span class="emphasis"><em>Brugia malayi</em></span>, live in symbiosis with specific bacteria carried by the nematodes (see Table 1 in <a href="#bib15">Blaxter, 2003</a>). Some of these are extracellular symbionts, but others are intracellular, such as <span class="emphasis"><em>Wolbachia</em></span> living in <span class="emphasis"><em>B. malayi</em></span> and other filarial nematodes. The capture of the <span class="emphasis"><em>Wolbachia</em></span> gene set seems to have been adaptive for filarial nematodes, since killing<span class="emphasis"><em> Wolbachia</em></span> with antibiotics reduces the growth and fecundity of the nematodes (<a href="#bib34">Foster et al., 2005</a>; <a href="#bib46">Hoerauf et al., 2001</a>).
               </p>
               <p>Free-living nematodes may also have pinched genes from organisms that live nearby Many nematodes use other animals, often
                  arthropods and molluscs, as transport hosts. For example, <span class="emphasis"><em>C. remanei</em></span> lives in close association  with molluscs and isopods (<a href="#bib6">Baird, 1999</a>). Indeed, <span class="emphasis"><em>C. elegans</em></span> has four genes, including an alcohol dehydrogenase, that have stronger sequence matches to fungi than to other animals (<a href="#bib71">Parkinson and Blaxter, 2003</a>). These <span class="emphasis"><em>C. elegans</em></span> genes group with fungal genes in phylogenetic trees. Similar phylogenetic analyses will allow us to scan the eight new genomes
                  for stolen genes.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_4_5"></a>4.5.&nbsp;Identifying "parasitism genes"
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Parasitism of plants and animals has evolved independently at least nine times in the history of the nematodes (<a href="#bib31">Dorris et al., 1999</a>). Four of the nematodes whose genomes are being sequenced are parasites: <span class="emphasis"><em>Haemonchus contortus</em></span>, <span class="emphasis"><em>Meloidogyne hapla</em></span>,  <span class="emphasis"><em>Brugia malayi</em></span> and <span class="emphasis"><em>Trichinella spiralis</em></span>. The adoption of parasitism in nematodes probably required adaptation of genes present in their free-living ancestors (<a href="#bib15">Blaxter, 2003</a>). For example, modification of nutrient-acquisition genes found in <span class="emphasis"><em>C. elegans</em></span>, such as digestive enzymes or secreted hydrolases, are likely to have been important for the evolution of parasitism (<a href="#bib36">Geary  and Thompson, 2001</a>). The ability of parasitic nematodes to survive  immunological attack, some living in an infected individual for years, has
                  long been a puzzle. The cuticle is the main site of interaction between a nematode and its environment, and many nematode
                  genes so far implicated in evading host defenses are secreted or cuticle proteins (<a href="#bib26">Davis et al., 2004</a>; <a href="#bib60">Maizels et al., 2001</a>). For example, the main soluble surface glycoprotein of filarial  nematodes, a secreted glutathione peroxidase (GPX-1), is
                  hypothesized to have a role in immune evasion (<a href="#bib98">Zvelebil et al., 1993</a>). In viral, bacterial and protozoan parasites, genes involved in host immune evasion or recognition are often under positive
                  selection, and so show patterns of rapid amino acid substitution (<a href="#bib61">McInerney et al., 2003</a>). Indeed, <span class="emphasis"><em>B. malayi</em></span> GPX-1 shows signs of positive selection (<a href="#bib98">Zvelebil et al., 1993</a>). By scanning for <span class="emphasis"><em>Haemonchus contortus</em></span> genes that have diverged sharply in sequence from their <span class="emphasis"><em>Pristionchus</em></span> and <span class="emphasis"><em>Caenorhabditis</em></span> orthologs, and that bear secretory signals (<a href="#bib43">Harcus et al., 2004</a>), it may be possible to identify <span class="emphasis"><em>H. contortus</em></span> genes that have adapted for a parasitic lifestyle.
               </p>
               <p>Some genes essential for parasitism in worms may be novel genes. One possible source is gene duplication, which allows one
                  duplicate to keep the original role, and the other duplicate to take on a parasitic role. For example, the  <span class="emphasis"><em>alt</em></span> gene family of filarial nematodes, which has been implicated in establishing infection, has a single <span class="emphasis"><em>C. elegans</em></span> ortholog (<a href="#bib38">Gomez-Escobar et al., 2002</a>). On the other hand, other novel genes adapted for parasitism may have been assembled <span class="emphasis"><em>de novo</em></span>, or have been gained by horizontal gene transfer: plant parasitic nematodes seem to have acquired "parasitism genes" from
                  bacteria in their environment (<a href="#bib12">Bird et al., 2003</a>).
               </p>
               <p>Some "parasitism genes" may by identifiable by examining the expression pattern of their <span class="emphasis"><em>C. elegans</em></span> orthologs. In <span class="emphasis"><em>Haemonchus contortus</em></span> and <span class="emphasis"><em>Brugia malayi</em></span>, the infective stage of  the life cycle is the third larval (L3) developmental stage (<a href="#bib15">Blaxter, 2003</a>). In <span class="emphasis"><em>C. elegans</em></span> the L3 developmental stage is an alternative developmental pathway adopted when food is scarce, called the dauer larva. Thus,
                  identifying the orthologs of <span class="emphasis"><em>C. elegans</em></span> genes expressed in the dauer larva (<a href="#bib93">Wang and Kim, 2003</a>) may be a route to pinpointing <span class="emphasis"><em>Brugia</em></span> and <span class="emphasis"><em>Haemonchus</em></span> genes involved in infection (<a href="#bib19">B&uuml;rglin et al., 1998</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_5"></a>5.&nbsp;The rate of evolution in nematodes
                     </h2>
                  </div>
               </div>
            </div>
            <p><a href="#bib64">Mushegian et al. (1998)</a> compared 36 <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>Drosophila</em></span> protein orthologs to their yeast counterparts, and found that many <span class="emphasis"><em>C. elegans</em></span> genes have evolved twice as fast as their <span class="emphasis"><em>Drosophila</em></span> orthologs. Nematode rRNA genes also seem to have a substitution rate that is 2-3 times that of other animal phyla (<a href="#bib3">Aguinaldo et al., 1997</a>). For example, the rRNA gene divergence between <span class="emphasis"><em>Caenorhabditis</em></span> species is comparable to that between vertebrate species (<a href="http://www.wormbook.org/chapters/www_phylogrhabditids/phylorhab.html" target="_blank">The phylogenetic relationships of <span class="emphasis"><em>C. elegans</em></span> and other Rhabditids</a>; <a href="#bib51">Kiontke et al., 2004</a>)!
            </p>
            <p>To accurately estimate the evolutionary rate in nematodes, ideally we would divide the number of mutations between two closely
               related species by their divergence date. However, estimating divergence dates between nematode species is extremely difficult,
               because nematode fossils are scarce (<a href="#bib78">Poinar, 1983</a>). One solution is to calibrate the molecular clock using the date that nematodes diverged from other animals. This allows
               evolutionary rates and speciation dates to be estimated from the subset of genes that have evolved at the same rate in all
               animals (for example, see <a href="#bib86">Stein et al., 2003</a>). Unfortunately, estimates made using this approach have large margins of error, due to uncertainties in the date of divergence
               of nematodes from other animals (estimates range from 600-1300 million years ago: <a href="#bib9">Benton and Ayala, 2003</a>; <a href="#bib13">Blair and Hedges, 2005</a>; <a href="#bib44">Hedges et al., 2004</a>; <a href="#bib73">Peterson et al., 2004</a>), and in the relationship of nematodes to other animals (<a href="#bib74">Philip et al., 2005</a>; <a href="#bib75">Philippe et al., 2005</a>). Ongoing whole-genome sequencing projects for members of eight more animal phyla (<a href="#bib10">Bernal et al., 2001</a>: <a href="http://www.genomesonline.org/" target="_blank">http://www.genomesonline.org/</a>) may lead to a more accurate phylogenetic tree of animal phyla, reducing these sources of error.
            </p>
            <p>Another solution is to directly measure the mutation rate in laboratory animals. In an ingenious experiment, <a href="#bib29">Denver et al. (2004)</a> estimated the mutation rate in <span class="emphasis"><em>C. elegans</em></span> by sequencing random stretches of DNA in mutation accumulation lines. They estimated the genomic mutation rate to be ~2.1
               mutations per genome per generation. Of the mutations observed, 43% were substitutions, 43% insertions, and 13% deletions.
               Comparable estimates for other animals do not yet exist, as estimates using earlier techniques were probably less accurate
               (reviewed by <a href="#bib49">Keightley and Charlesworth, 2005</a>). The future use of this method for detecting mutations in arthropods and vertebrates will provide insight into the difference
               between the rate and types of mutations in nematodes and that in other animals.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_6"></a>6.&nbsp;Summary
                     </h2>
                  </div>
               </div>
            </div>
            <p>Many mysteries remain in eukaryotic genome evolution. We will soon have a data set of ten nematode genome sequences that will
               be ideal for investigating unresolved questions, such as what are the forces governing the evolution of chromosome number,
               size and structure; how does sex chromosome evolution differ from that of autosomes; how do differences in life history traits
               and reproductive strategy affect genome evolution; and what are the major genomic changes that enable species to adapt to
               new ecological niches such as parasitism. Looking forward, it seems very possible that once again these tiny animals will
               be first in revealing some of nature&#8217;s deepest secrets.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_7"></a>7.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>I am very grateful to David Bird, John Speith, Ralf Sommer,
                       John Gilleard, Robert Waterston, Raymond Miller and David Baillie for useful information and discussion. I am grateful
               to David Fitch and two anonymous reviewers for their very helpful comments on the text. I thank Richard Durbin and Des Higgins
               for generously allowing me to complete this work in their labs. Avril Coghlan is supported by the Wellcome Trust.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1_8"></a>8.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                             <span class="bibliomisc">Aboobaker, A., and Blaxter, M. (2003a). Hox gene evolution in nematodes: novelty conserved. Curr. Opin. Genet. Dev. <span class="emphasis"><em>13</em></span>, 593&#8211;598.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14638320&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.gde.2003.10.009" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                             <span class="bibliomisc">Aboobaker, A.A., and Blaxter, M.L. (2003b). Hox gene loss during dynamic evolution of the nematode cluster. Curr. Biol. <span class="emphasis"><em>13</em></span>, 37&#8211;40.</span>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)01399-4" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                             <span class="bibliomisc">Aguinaldo, A.M., Turbeville, J.M., Linford, L.S., Rivera, M.C., Garey, J.R., Raff, R.A., and Lake, J.A. (1997). Evidence for
                        a clade of nematodes, arthropods and other moulting animals. Nature <span class="emphasis"><em>387</em></span>, 489&#8211;493.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9168109&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/387489a0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                             <span class="bibliomisc">Andr&aacute;ssy, I., and Zombori, L. (1976). Evolution as a basis for the systematization of nematodes. (London: Pitman).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                             <span class="bibliomisc">Archetti, M. (2004). Recombination and loss of complementation: a more than two-fold cost for parthenogenesis. J. Evol. Biol.
                        <span class="emphasis"><em>17</em></span>, 1084&#8211;1097.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15312080&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1111/j.1420-9101.2004.00745.x" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                             <span class="bibliomisc">Baird, S.E. (1999). Natural and experimental association of <span class="emphasis"><em>Caenorhabditis  remanei</em></span> with <span class="emphasis"><em>Trachelipus rathkii</em></span> and other terrestrial isopods. Nematology <span class="emphasis"><em>1</em></span>, 471&#8211;475.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                             <span class="bibliomisc">Barnes, T.M., Kohara, Y., Coulson, A., and Hekimi, S. (1995). Meiotic recombination, noncoding DNA and genomic organization
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>  Genetics <span class="emphasis"><em>141</em></span>, 159&#8211;179.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8536965&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                             <span class="bibliomisc">Bektesh, S., Van Doren, K., and Hirsh, D. (1988). Presence of the <span class="emphasis"><em>Caenorhabditis  elegans</em></span> spliced leader on different mRNAs and in different genera of nematodes. Genes Dev. <span class="emphasis"><em>2</em></span>, 1277&#8211;1283.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3203906&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                             <span class="bibliomisc">Benton, M.J., and Ayala, F.J. (2003). Dating the tree of life. Science <span class="emphasis"><em>300</em></span>, 1698&#8211;1700.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12805535&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1077795" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                             <span class="bibliomisc">Bernal, A., Ear, U., and Kyrpides, N. (2001). Genomes OnLine Database (GOLD): a monitor of genome projects world-wide. Nucleic Acids Res.
                        <span class="emphasis"><em>29</em></span>, 126&#8211;127.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11125068&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/29.1.126" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                             <span class="bibliomisc">Betran, E., and Long, M. (2002). Expansion of genome coding regions by acquisition of new genes. Genetica <span class="emphasis"><em>115</em></span>, 65&#8211;80.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12188049&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1023/A:1016024131097" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                             <span class="bibliomisc">Bird, D.M., Opperman, C.H., and Davies, K.G. (2003). Interactions between bacteria and plant-parasitic nematodes: now and
                        then. Int. J. Parasitol. <span class="emphasis"><em>33</em></span>, 1269&#8211;1276.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=13678641&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0020-7519(03)00160-7" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                             <span class="bibliomisc">Blair, J.E., and Hedges, S.B. (2005). Molecular clocks do not support the Cambrian explosion. Mol. Biol. Evol. <span class="emphasis"><em>22</em></span>, 387&#8211;390.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15537810&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msi039" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                             <span class="bibliomisc">Blaxter, M. (2000). Genes and genomes of <span class="emphasis"><em>Necator americanus</em></span> and related hookworms. Int. J. Parasitol. <span class="emphasis"><em>30</em></span>, 347&#8211;355.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10731559&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0020-7519(99)00198-8" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                             <span class="bibliomisc">Blaxter, M.L. (2003). Nematoda: genes, genomes and the evolution of parasitism. Adv. Parasitol. <span class="emphasis"><em>54</em></span>, 101&#8211;195.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14711085&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0065-308X(03)54003-9" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                             <span class="bibliomisc">Blaxter, M.L., De Ley, P., Garey, J.R., Liu, L.X., Scheldeman, P., Vierstraete, A., Vanfleteren, J.R., Mackey, L.Y., Dorris,
                        M., Frisse, L.M., et al. (1998). A molecular evolutionary framework for the phylum Nematoda. Nature <span class="emphasis"><em>392</em></span>, 71&#8211;75.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9510248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/32160" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                             <span class="bibliomisc">Blumenthal, T., Evans, D., Link, C.D., Guffanti, A., Lawson, D., Thierry-Mieg, J., Thierry-Mieg, D., Chiu, W.L., Duke, K.,
                        Kiraly, M., et al. (2002). A global analysis of <span class="emphasis"><em>Caenorhabditis elegans</em></span> operons. Nature <span class="emphasis"><em>417</em></span>, 851&#8211;854.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12075352&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature00831" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                             <span class="bibliomisc">B&uuml;rglin, T.R. (1994). A comprehensive classification of homeobox genes. In Guidebook to the homeobox genes, D. Duboule, ed.
                        (Oxford, United Kingdom: Oxford University Press), pp. 25&#8211;71.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                             <span class="bibliomisc">B&uuml;rglin, T.R., Lobos, E., and Blaxter, M.L. (1998). <span class="emphasis"><em>Caenorhabditis elegans</em></span> as a model for parasitic nematodes. Int. J. Parasitol. <span class="emphasis"><em>28</em></span>, 395&#8211;411.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9559358&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0020-7519(97)00208-7" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                             <span class="bibliomisc"><span class="emphasis"><em>C. elegans</em></span> Sequencing Consortium (1998). Genome sequence of the nematode <span class="emphasis"><em>C. elegans</em></span>: a platform for investigating biology. Science <span class="emphasis"><em>282</em></span>, 2012&#8211;2018.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851916&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.282.5396.2012" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                             <span class="bibliomisc">Charlesworth, B. (1992). Evolutionary rates in partially self-fertilizing species. Am. Nat. <span class="emphasis"><em>140</em></span>, 126&#8211;148.</span>
                     <a href="http://dx.doi.org/10.1086/285406" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                             <span class="bibliomisc">Coghlan, A., and Wolfe, K.H. (2002). Fourfold faster rate of genome rearrangement in nematodes than in <span class="emphasis"><em>Drosophila</em></span>. Genome Res. <span class="emphasis"><em>12</em></span>, 857&#8211;867.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12045140&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.172702" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                             <span class="bibliomisc">Csankovszki, G., McDonel, P., and Meyer, B.J. (2004). Recruitment and spreading of the <span class="emphasis"><em>C. elegans</em></span> dosage compensation complex along X chromosomes. Science <span class="emphasis"><em>303</em></span>, 1182&#8211;1185.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14976312&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1092938" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                             <span class="bibliomisc">Cutter, A.D., and Payseur, B.A. (2003a). Rates of deleterious mutation and the evolution of sex in <span class="emphasis"><em>Caenorhabditis</em></span>. J. Evol. Biol. <span class="emphasis"><em>16</em></span>, 812&#8211;822.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14635896&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1046/j.1420-9101.2003.00596.x" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                             <span class="bibliomisc">Cutter, A.D., and Payseur, B.A. (2003b). Selection at linked sites in the partial selfer <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>20</em></span>, 665&#8211;673.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12679551&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msg072" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                             <span class="bibliomisc">Davis, E.L., Hussey, R.S., and Baum, T.J. (2004). Getting to the roots of parasitism by nematodes. Trends Parasitol. <span class="emphasis"><em>20</em></span>, 134&#8211;141.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15036035&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.pt.2004.01.005" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                             <span class="bibliomisc">Davis, R.E., and Hodgson, S. (1997). Gene linkage and steady state RNAs suggest  trans-splicing may be associated with a polycistronic
                        transcript in <span class="emphasis"><em>Schistosoma mansoni</em></span>. Mol. Biochem. Parasitol. <span class="emphasis"><em>89</em></span>, 25&#8211;39.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9297698&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0166-6851(97)00097-2" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                             <span class="bibliomisc">De Ley, P., and Blaxter, M.L. (2002). Systematic position and phylogeny. In The Biology of Nematodes, D. Lee, ed. (Reading:
                        Harwood Academic Publishers).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                             <span class="bibliomisc">Denver, D.R., Morris, K., Lynch, M., and Thomas, W.K. (2004). High mutation rate and predominance of insertions in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> nuclear genome. Nature <span class="emphasis"><em>430</em></span>, 679&#8211;682.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15295601&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02697" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                             <span class="bibliomisc">Dernburg, A.F. (2001). Here, there, and everywhere: kinetochore function on holocentric chromosomes. J. Cell. Biol. <span class="emphasis"><em>153</em></span>, F33&#8211;F38.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11402076&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.153.6.F33" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                             <span class="bibliomisc">Dorris, M., De Ley, P., and Blaxter, M.L. (1999). Molecular analysis of nematode diversity and the evolution of parasitism.
                        Parasitol Today <span class="emphasis"><em>15</em></span>, 188&#8211;193.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10322352&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0169-4758(99)01439-8" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                             <span class="bibliomisc">Evans, D., Zorio, D., MacMorris, M., Winter, C.E., Lea, K., and Blumenthal, T. (1997). Operons and SL2 trans-splicing exist
                        in nematodes outside the genus <span class="emphasis"><em>Caenorhabditis</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>94</em></span>, 9751&#8211;9756.</span> <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9275196&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                             <span class="bibliomisc">Ferrier, D.E., and Holland, P.W. (2001). Ancient origin of the Hox gene cluster. Nat. Rev. Genet. <span class="emphasis"><em>2</em></span>, 33&#8211;38.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11253066&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35047605" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                             <span class="bibliomisc">Foster, J., Ganatra, M., Kamal, I., Ware, J., Makarova, K., Ivanova, N., Bhattacharyya, A., Kapatral, V., Kumar, S., Posfai,
                        J., et al. (2005). The Wolbachia genome of <span class="emphasis"><em>Brugia malayi</em></span>: endosymbiont evolution within a human pathogenic nematode. PLoS. Biol. <span class="emphasis"><em>3</em></span>, e121.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15780005&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0030121" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                             <span class="bibliomisc">Ganko, E.W., Bhattacharjee, V., Schliekelman, P., and McDonald, J.F. (2003). Evidence for the contribution of LTR retrotransposons
                        to <span class="emphasis"><em>C. elegans</em></span> gene evolution. Mol. Biol. Evol. <span class="emphasis"><em>20</em></span>, 1925&#8211;1931.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12885961&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msg200" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                             <span class="bibliomisc">Geary, T.G., and Thompson, D.P. (2001). <span class="emphasis"><em>Caenorhabditis elegans</em></span>: how good a  model for veterinary parasites? Vet. Parasitol. <span class="emphasis"><em>101</em></span>, 371&#8211;386.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11707307&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0304-4017(01)00562-3" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                             <span class="bibliomisc">Ghedin, E., Wang, S., Foster, J.M., and Slatko, B.E. (2004). First sequenced genome of a parasitic nematode. Trends Parasitol.
                        <span class="emphasis"><em>20</em></span>, 151&#8211;153.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15099548&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.pt.2004.01.011" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                             <span class="bibliomisc">Gomez-Escobar, N., Gregory, W.F., Britton, C., Murray, L., Corton, C., Hall, N., Daub, J., Blaxter, M.L., and Maizels, R.M.
                        (2002). Abundant larval transcript-1 and -2 genes from <span class="emphasis"><em>Brugia malayi</em></span>: diversity of genomic environments but conservation of 5' promoter sequences functional in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Biochem. Parasitol. <span class="emphasis"><em>125</em></span>, 59&#8211;71.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12467974&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0166-6851(02)00219-0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                             <span class="bibliomisc">Graustein, A., Gaspar, J.M., Walters, J.R., and Palopoli, M.F. (2002). Levels of DNA polymorphism vary with mating system
                        in the nematode genus <span class="emphasis"><em>Caenorhabditis</em></span>. Genetics <span class="emphasis"><em>161</em></span>, 99&#8211;107.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12019226&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                             <span class="bibliomisc">Gutierrez, A., and Sommer, R.J. (2004). Evolution of <span class="emphasis"><em>dnmt-2</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mbd-2" target="_blank">mbd-2</a></em></span>-like genes in the free-living nematodes <span class="emphasis"><em>Pristionchus pacificus</em></span>, <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em> Caenorhabditis briggsae</em></span>. Nucleic Acids Res. <span class="emphasis"><em>32</em></span>, 6388&#8211;6396.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15576683&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkh982" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                             <span class="bibliomisc">Haag, E.S., and Doty, A.V. (2005). Sex determination across evolution: connecting the dots. PLoS. Biol. <span class="emphasis"><em>3</em></span>, e21.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15660158&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0030021" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hammond, M.P., and Bianco, A.E. (1992). Genes and genomes of parasitic nematodes. Parasitol Today <span class="emphasis"><em>8</em></span>, 299&#8211;305.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15463645&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0169-4758(92)90100-G" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                             <span class="bibliomisc">Harcus, Y.M., Parkinson, J., Fernandez, C., Daub, J., Selkirk, M.E., Blaxter, M.L., and Maizels, R.M. (2004). Signal sequence
                        analysis of expressed sequence tags from the nematode <span class="emphasis"><em>Nippostrongylus brasiliensis</em></span> and the evolution of secreted proteins in parasites. Genome Biol. <span class="emphasis"><em>5</em></span>, R39.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15186490&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2004-5-6-r39" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hedges, S.B., Blair, J.E., Venturi, M.L., and Shoe, J.L. (2004). A molecular timescale of eukaryote evolution and the rise
                        of complex multicellular life. BMC Evol. Biol. <span class="emphasis"><em>4</em></span>, 2.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15005799&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2148-4-2" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hirai, H., Tada, I., Takahashi, H., Nwoke, B.E., and Ufomadu, G.O. (1987). Chromosomes of <span class="emphasis"><em>Onchocerca volvulus</em></span> (Spirurida: Onchocercidae): a comparative study  between Nigeria and Guatemala. J. Helminthol. <span class="emphasis"><em>61</em></span>, 43&#8211;46.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3571919&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hoerauf, A., Mand, S., Adjei, O., Fleischer, B., and Buttner, D.W. (2001). Depletion of <span class="emphasis"><em>Wolbachia</em></span> endobacteria in <span class="emphasis"><em>Onchocerca volvulus</em></span> by doxycycline and microfilaridermia after ivermectin treatment. Lancet <span class="emphasis"><em>357</em></span>, 1415&#8211;1416.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11356444&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0140-6736(00)04581-5" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kaestner, A. (1965). Lehrbuch der Speziellen Zoologie. Band I: Wirbellose (Jena: Verlag).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kamath, R.S., Fraser, A.G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., Kanapin, A., Le Bot, N., Moreno, S., Sohrmann, M.,
                        et al. (2003). Systematic functional analysis of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome using RNAi. Nature <span class="emphasis"><em>421</em></span>, 231&#8211;237.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12529635&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature01278" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed">
                             <span class="bibliomisc">Keightley, P.D., and Charlesworth, B. (2005). Genetic instability of <span class="emphasis"><em>C. elegans</em></span> comes naturally. Trends Genet. <span class="emphasis"><em>21</em></span>, 67&#8211;70.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15661349&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.tig.2004.11.015" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kikuchi, T., Jones, J.T., Aikawa, T., Kosaka, H., and Ogura, N. (2004). A family of glycosyl hydrolase family 45 cellulases
                        from the pine wood nematode <span class="emphasis"><em>Bursaphelenchus xylophilus</em></span>. FEBS Lett. <span class="emphasis"><em>572</em></span>, 201&#8211;205.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15304348&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.febslet.2004.07.039" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kiontke, K., Gavin, N.P., Raynes, Y., Roehrig, C., Piano, F., and Fitch, D.H. (2004). <span class="emphasis"><em>Caenorhabditis</em></span> phylogeny predicts convergence of hermaphroditism and extensive intron loss. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 9003&#8211;9008.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15184656&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0403094101" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lambshead, P.J. (1993). Recent developments in marine benthic biodiversity research. Oceanis <span class="emphasis"><em>19</em></span>, 5&#8211;24.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lee, K.Z., Eizinger, A., Nandakumar, R., Schuster, S.C., and Sommer, R.J. (2003). Limited microsynteny between the genomes
                        of <span class="emphasis"><em>Pristionchus pacificus</em></span> and <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>31</em></span>, 2553&#8211;2560.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12736304&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkg359" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lee, K.Z., and Sommer, R.J. (2003). Operon structure and trans-splicing in the nematode <span class="emphasis"><em>Pristionchus pacificus</em></span>. Mol. Biol. Evol. <span class="emphasis"><em>20</em></span>, 2097&#8211;2103.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12949121&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msg225" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lerat, E., Daubin, V., Ochman, H., and Moran, N.A. (2005). Evolutionary origins of genomic repertoires in bacteria. PLoS.
                        Biol. <span class="emphasis"><em>3</em></span>, e130.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15799709&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0030130" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed">
                             <span class="bibliomisc">Leroy, S., Duperray, C., and Morand, S. (2003). Flow cytometry for parasite nematode genome size measurement. Mol. Biochem.
                        Parasitol. <span class="emphasis"><em>128</em></span>, 91&#8211;93.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12706802&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0166-6851(03)00023-9" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed">
                             <span class="bibliomisc">Liu, Z., Moore, P.H., Ma, H., Ackerman, C.M., Ragiba, M., Yu, Q., Pearl, H.M., Kim, M.S., Charlton, J.W., Stiles, J.I., et
                        al. (2004). A primitive Y chromosome in papaya marks incipient sex chromosome evolution. Nature <span class="emphasis"><em>427</em></span>, 348&#8211;352.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14737167&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02228" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed">
                             <span class="bibliomisc">Long, M. (2001). Evolution of novel genes. Curr. Opin. Genet. Dev. <span class="emphasis"><em>11</em></span>, 673&#8211;680.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11682312&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0959-437X(00)00252-5" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lynch, M., and Conery, J.S. (2003). The origins of genome complexity. Science <span class="emphasis"><em>302</em></span>, 1401&#8211;1404.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14631042&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1089370" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed">
                             <span class="bibliomisc">Maizels, R.M., Blaxter, M.L., and Scott, A.L. (2001). Immunological genomics of <span class="emphasis"><em>Brugia malayi</em></span>: filarial genes implicated in immune evasion and protective immunity. Parasite Immunol. <span class="emphasis"><em>23</em></span>, 327&#8211;344.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11472553&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1046/j.1365-3024.2001.00397.x" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed">
                             <span class="bibliomisc">McInerney, J.O., Littlewood, D.T., and Creevey, C.J. (2003). Detecting adaptive molecular evolution: additional tools for
                        the parasitologist. Adv. Parasitol. <span class="emphasis"><em>54</em></span>, 359&#8211;379.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14711091&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0065-308X(03)54009-X" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed">
                             <span class="bibliomisc">Meyer, B.J. (2000). Sex in the wormcounting and compensating X-chromosome dose. Trends Genet. <span class="emphasis"><em>16</em></span>, 247&#8211;253.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10827451&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0168-9525(00)02004-7" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed">
                             <span class="bibliomisc">Miller, M.A., Cutter, A.D., Yamamoto, I., Ward, S., and Greenstein, D. (2004). Clustered organization of reproductive genes
                        in the <span class="emphasis"><em>C. elegans</em></span> genome. Curr. Biol. <span class="emphasis"><em>14</em></span>, 1284&#8211;1290.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15268860&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.07.025" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed">
                             <span class="bibliomisc">Mushegian, A.R., Garey, J.R., Martin, J., and Liu, L.X. (1998). Large-scale taxonomic profiling of eukaryotic model organisms:
                        a comparison of orthologous proteins encoded by the human, fly, nematode, and yeast genomes. Genome Res. <span class="emphasis"><em>8</em></span>, 590&#8211;598.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9647634&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed">
                             <span class="bibliomisc">Mutafova, T. (1995). Meiosis and some aspects of cytological mechanisms of chromosomal sex determination in nematode species.
                        Int. J. Parasitol. <span class="emphasis"><em>25</em></span>, 453&#8211;462.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7635621&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0020-7519(94)00150-M" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed">
                             <span class="bibliomisc">Mutafova, T., Dimitrova, Y., and Komandarev, S. (1982). The karyotype of four <span class="emphasis"><em>Trichinella</em></span> species. Z. Parasitenkd. <span class="emphasis"><em>67</em></span>, 115&#8211;120.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7072318&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00929519" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed">
                             <span class="bibliomisc">Nicoll, M., Akerib, C.C., and Meyer, B.J. (1997). X-chromosome-counting mechanisms that determine nematode sex. Nature <span class="emphasis"><em>388</em></span>, 200&#8211;204.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9217163&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/40669" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed">
                             <span class="bibliomisc">Niedermaier, J., and Moritz, K.B. (2000). Organization and dynamics of satellite and telomere DNAs in <span class="emphasis"><em>Ascaris</em></span>: implications for formation and breakdown of compound chromosomes. Chromosoma <span class="emphasis"><em>109</em></span>, 439&#8211;452.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11151673&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed">
                             <span class="bibliomisc">Ohno, S. (1967). Sex chromosomes and sex-linked genes. In: Monographs on endocrinology, A. Labhart, ed. (Heidelberg: Springer-Verlag),
                        pp. 123&#8211;135.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed">
                             <span class="bibliomisc">Otto, S.P., and Whitton, J. (2000). Polyploid incidence and evolution. Annu. Rev. Genet. <span class="emphasis"><em>34</em></span>, 401&#8211;437.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11092833&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.genet.34.1.401" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed">
                             <span class="bibliomisc">Parkinson, J., and Blaxter, M. (2003). SimiTri-visualizing similarity relationships for groups of sequences. Bioinformatics
                        <span class="emphasis"><em>19</em></span>, 390&#8211;395.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12584125&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/bioinformatics/btf870" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib72"></a><p class="bibliomixed">
                             <span class="bibliomisc">Parkinson, J., Mitreva, M., Whitton, C., Thomson, M., Daub, J., Martin, J., Schmid, R., Hall, N., Barrell, B., Waterston,
                        R.H., et al. (2004). A transcriptomic analysis of the phylum Nematoda. Nat. Genet. <span class="emphasis"><em>36</em></span>, 1259&#8211;1267.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15543149&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1472" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib73"></a><p class="bibliomixed">
                             <span class="bibliomisc">Peterson, K.J., Lyons, J.B., Nowak, K.S., Takacs, C.M., Wargo, M.J., and McPeek, M.A. (2004). Estimating metazoan divergence
                        times with a molecular clock. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 6536&#8211;6541.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15084738&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0401670101" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib74"></a><p class="bibliomixed">
                             <span class="bibliomisc">Philip, G.K., Creevey, C.J., and McInerney, J.O. (2005). The Opisthokonta and  the Ecdysozoa may not be clades: stronger support
                        for the grouping of plant and animal than for animal and fungi and stronger support for the Coelomata than Ecdysozoa. Mol.
                        Biol. Evol.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15703245&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msi102" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib75"></a><p class="bibliomixed">
                             <span class="bibliomisc">Philippe, H., Lartillot, N., and Brinkmann, H. (2005). Multigene analyses of bilaterian animals corroborate the monophyly
                        of Ecdysozoa, Lophotrochozoa and Protostomia. Mol. Biol. Evol.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15703236&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/molbev/msi111" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib76"></a><p class="bibliomixed">
                             <span class="bibliomisc">Pires-daSilva, A., and Sommer, R.J. (2004). Conservation of the global sex determination gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> in distantly related nematodes. Genes Dev. <span class="emphasis"><em>18</em></span>, 1198&#8211;1208.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15155582&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.293504" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib77"></a><p class="bibliomixed">
                             <span class="bibliomisc">Platt, H.M. (1994). Foreward. In: The phylogenetic systematics of free-living nematodes, S. Lorenzen, ed. (London: The Ray
                        Society).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib78"></a><p class="bibliomixed">
                             <span class="bibliomisc">Poinar, G.O. (1983). The natural history of nematodes. (Englewood Cliffs, NJ: Prentice-Hall).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib79"></a><p class="bibliomixed">
                             <span class="bibliomisc">Ranz, J.M., Casals, F., and Ruiz, A. (2001). How malleable is the eukaryotic genome? Extreme rate of chromosomal rearrangement
                        in the genus <span class="emphasis"><em>Drosophila</em></span>. Genome Res. <span class="emphasis"><em>11</em></span>, 230&#8211;239.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11157786&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.162901" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib80"></a><p class="bibliomixed">
                             <span class="bibliomisc">Roy, P.J., Stuart, J.M., Lund, J., and Kim, S.K. (2002). Chromosomal clustering of muscle-expressed genes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>418</em></span>, 975&#8211;979.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12214599&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature01012" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib81"></a><p class="bibliomixed">
                             <span class="bibliomisc">Scholl, E.H., Thorne, J.L., McCarter, J.P., and Bird, D.M. (2003). Horizontally transferred genes in plant-parasitic nematodes:
                        a high-throughput genomic approach. Genome Biol. <span class="emphasis"><em>4</em></span>, R39.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12801413&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2003-4-6-r39" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib82"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sharakhov, I.V., Serazin, A.C., Grushko, O.G., Dana, A., Lobo, N., Hillenmeyer, M.E., Westerman, R., Romero-Severson, J.,
                        Costantini, C., Sagnon, N., et al. (2002). Inversions and gene order shuffling in <span class="emphasis"><em>Anopheles gambiae</em></span> and <span class="emphasis"><em>A. funestus</em></span>. Science <span class="emphasis"><em>298</em></span>, 182&#8211;185.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12364797&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1076803" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib83"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sommer, R.J., Carta, L.K., Kim, S.-Y., and Sternberg, P.W. (1996). Morphological, genetic and molecular description of <span class="emphasis"><em>Pristionchus pacificus sp. n.</em></span> (Nematoda: Neodiplogastridae). Fundam. Appl. Nematol. <span class="emphasis"><em>19</em></span>, 511&#8211;521.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib84"></a><p class="bibliomixed">
                             <span class="bibliomisc">&#352;pakulov&aacute;, M., and Casanova, J.C. (2004). Current knowledge on B chromosomes in natural populations of helminth parasites:
                        a review. Cytogenet. Genome Res. <span class="emphasis"><em>106</em></span>, 222&#8211;229.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15292595&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1159/000079291" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib85"></a><p class="bibliomixed">
                             <span class="bibliomisc">&#352;pakulov&aacute;, M., Kr&aacute;l'ov&aacute;, I., and Cutillas, C. (1994). Studies on the karyotype and gametogenesis in <span class="emphasis"><em>Trichuris muris</em></span>. J. Helminthol. <span class="emphasis"><em>68</em></span>, 67&#8211;72.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8006389&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib86"></a><p class="bibliomixed">
                             <span class="bibliomisc">Stein, L.D., Bao, Z., Blasiar, D., Blumenthal, T., Brent, M.R., Chen, N., Chinwalla, A., Clarke, L., Clee, C., Coghlan, A.,
                        et al. (2003). The genome sequence of <span class="emphasis"><em>Caenorhabditis briggsae</em></span>: a platform for comparative genomics. PLoS. Biol. <span class="emphasis"><em>1</em></span>, E45.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14624247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib87"></a><p class="bibliomixed">
                             <span class="bibliomisc">Takacs, A.M., Denker, J.A., Perrine, K.G., Maroney, P.A., and Nilsen, T.W. (1988). A 22-nucleotide spliced leader sequence
                        in the human parasitic nematode <span class="emphasis"><em>Brugia malayi</em></span> is identical to the trans-spliced leader exon in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>85</em></span>, 7932&#8211;7936.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3186698&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib88"></a><p class="bibliomixed">
                             <span class="bibliomisc">Triantaphyllou, A.C. (1984). Polyploidy in meiotic parthenogenetic populations of <span class="emphasis"><em>Meloidogyne hapla</em></span> and a mechanism of conversion to diploidy. Rev. Nematol. <span class="emphasis"><em>7</em></span>, 65&#8211;72.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib89"></a><p class="bibliomixed">
                             <span class="bibliomisc">Troemel, E.R. (1999). Chemosensory signaling in <span class="emphasis"><em>C. elegans</em></span>. Bioessays <span class="emphasis"><em>21</em></span>, 1011&#8211;1020.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10580986&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib90"></a><p class="bibliomixed">
                             <span class="bibliomisc">Underwood, A.P., and Bianco, A.E. (1999). Identification of a molecular marker for the Y chromosome of <span class="emphasis"><em>Brugia malayi</em></span>. Mol. Biochem. Parasitol. <span class="emphasis"><em>99</em></span>, 1&#8211;10.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10215019&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0166-6851(98)00180-7" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib91"></a><p class="bibliomixed">
                             <span class="bibliomisc">Walton, A.C. (1940). Gametogenesis. In: An Introduction to Nematology, G.B. Chitwood, and M.B. Chitwood, eds. (New York: Babylon),
                        pp. 205&#8211;215.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib92"></a><p class="bibliomixed">
                             <span class="bibliomisc">Walton, A.C. (1959). Some parasites and their chromosomes. J. Parasitol. <span class="emphasis"><em>45</em></span>, 1&#8211;20.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib93"></a><p class="bibliomixed">
                             <span class="bibliomisc">Wang, J., and Kim, S.K. (2003). Global analysis of dauer gene expression in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>130</em></span>, 1621&#8211;1634.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12620986&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.00363" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib94"></a><p class="bibliomixed">
                             <span class="bibliomisc">Weerasinghe, R.R., Bird, D.M., and Allen, N.S. (2005). Root-knot nematodes and bacterial Nod factors elicit common signal
                        transduction events in <span class="emphasis"><em>Lotus japonicus</em></span>. Proc. Natl. Acad. Sci. USA.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15710894&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0407926102" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib95"></a><p class="bibliomixed">
                             <span class="bibliomisc">White, M.J.D. (1973). Animal Cytology and Evolution, 3rd edn (New York: Cambridge University Press).</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib96"></a><p class="bibliomixed">
                             <span class="bibliomisc">Whitton, C., Daub, J., Quail, M., Hall, N., Foster, J., Ware, J., Ganatra, M., Slatko, B., Barrell, B., and Blaxter, M. (2004).
                        A genome sequence survey of the filarial nematode <span class="emphasis"><em>Brugia malayi</em></span>: repeats, gene discovery, and comparative genomics. Mol. Biochem. Parasitol. <span class="emphasis"><em>137</em></span>, 215&#8211;227.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15383292&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.molbiopara.2004.05.013" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib97"></a><p class="bibliomixed">
                             <span class="bibliomisc">Witherspoon, D.J., and Robertson, H.M. (2003). Neutral evolution of ten types of <span class="emphasis"><em>mariner</em></span> transposons in the genomes of <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Caenorhabditis briggsae</em></span>. J. Mol. Evol. <span class="emphasis"><em>56</em></span>, 751&#8211;769.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12911038&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/s00239-002-2450-x" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib98"></a><p class="bibliomixed">
                             <span class="bibliomisc">Zvelebil, M.J., Tang, L., Cookson, E., Selkirk, M.E., and Thornton, J.M. (1993). Molecular modelling and epitope prediction
                        of gp29 from <span class="emphasis"><em>lymphatic filariae</em></span>. Mol. Biochem. Parasitol. <span class="emphasis"><em>58</em></span>, 145&#8211;153</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7681545&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by David H.A. Fitch. Last revised September 07, 2005. Published September 07, 2005. This chapter should be cited as:
                  Coghlan, A. Nematode genome evolution (September 07, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.15.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2005 Avril Coghlan. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e34" href="#d0e34">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:alc@sanger.ac.uk" target="_top">alc@sanger.ac.uk</a></p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e39" href="#d0e39">&#8224;</a></sup>Current address: Wellcome Trust Sanger Institute, Cambridge, CB10 1SA, UK
               </p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

