<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Transcriptional regulation of gene expression in C. elegans</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WB_Reinke_etal" title="Transcriptional regulation of gene expression in C. elegans">
      <link rel="next" href="#sec1" title="1.&nbsp;Overview">
      <script language="javascript" type="text/javascript" src="/js/history_list.js"></script>
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="transcriptionalregulation.2.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WB_Reinke_etal"></a>Transcriptional regulation of gene expression in <span class="emphasis"><em>C. elegans</em></span><sup><a name="d0e8" href="#ftn.d0e8">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Valerie Reinke<sup>1&sect;</sup>, Michael Krause<sup>2</sup><em><span class="remark"><sup><a name="d0e37" href="#ftn.d0e37">&sect;</a></sup></span></em>, Peter Okkema<sup>3&sect;</sup></span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup> Department of Genetics, Yale University, New Haven, CT 06520; </span></span><span class="affiliation"><span class="orgname"><sup>2</sup>Laboratory of Molecular Biology, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of
                           Health, Bethesda, MD 20892; </span></span><span class="affiliation"><span class="orgname"><sup>3</sup>Department of Biological Sciences, University of Illinois Chicago, Chicago, IL 60607; </span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">
         
	   <div id="section_locator"><a href="/toc_molecularbiology.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Molecular biology
		</a></div><hr />

          <p><a style="color:blue;cursor:pointer" onclick="return kadabra('history');" title="click to expand ..." >Publication History</a></p>
           <ul id="history" style="display:none">
            <li><a href="/chapters/www_transcriptionalregulation/transcriptionalregulation.html">version 1</a></li>
            <li><a href="/chapters/www_transcriptionalregulation.2/transregulate.html">latest version</a></li>
           </ul>

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Overview</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. RNA Polymerase II and associated factors</a></span></dt>
               <dt><span class="sect1"><a href="#sec3">3. <span class="emphasis"><em>C. elegans</em></span> gene organization and regulation</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec3-1">3.1. Proximal control regions</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-2">3.2. Distal control regions</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-3">3.3. Transcriptional initiation and trans-splicing</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3-4">3.4. Core promoter elements</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec4">4. Promoter complexity</a></span></dt>
               <dt><span class="sect1"><a href="#sec5">5. Transcription factors</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec5-1">5.1. Transcription factor resources</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5-2">5.2. Transcription factor families</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5-3">5.3. Interspecies comparisons</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5-4">5.4. Transcription factor targets</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5-5">5.5. Transcription factor gene expression</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5-6">5.6. Post-transcriptional regulation of transcription factor function</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec6">6. Chromatin status and transcription</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec6-1">6.1. The NuRD complex</a></span></dt>
                     <dt><span class="sect2"><a href="#sec6-2">6.2. The MLL/COMPASS complex</a></span></dt>
                     <dt><span class="sect2"><a href="#sec6-3">6.3. The Polycomb Group complex</a></span></dt>
                     <dt><span class="sect2"><a href="#sec6-4">6.4. The SynMuv pathway</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec7">7. Systematic genome-scale analysis of transcription regulation</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec7-1">7.1. The modENCODE project</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7-2">7.2. modENCODE transcription factor ChIP studies</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7-3">7.3. modENCODE chromatin modification ChIP studies</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7-4">7.4. modENCODE transcription studies</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec8">8. Future Prospects</a></span></dt>
               <dt><span class="sect1"><a href="#sec9">9. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#sec10">10. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p>Protein coding gene sequences are converted to mRNA by the highly regulated process of transcription. The precise temporal
               and spatial control of transcription for many genes is an essential part of development in metazoans. Thus, understanding
               the molecular mechanisms underlying transcriptional control is essential to understanding cell fate determination during embryogenesis,
               post-embryonic development, many environmental interactions, and disease-related processes. Studies of transcriptional regulation
               in <span class="emphasis"><em>C. elegans</em></span> exploit its genomic simplicity and physical characteristics to define regulatory events with single-cell and minute-time-scale
               resolution. When combined with the genetics of the system, <span class="emphasis"><em>C. elegans</em></span> offers a unique and powerful vantage point from which to study how chromatin-associated proteins and their modifications
               interact with transcription factors and their binding sites to yield precise control of gene expression through transcriptional
               regulation.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Overview
                     </h2>
                  </div>
               </div>
            </div>
            <p>Every aspect of cellular function depends on the gene products expressed in that cell. The mechanisms regulating the expression
               of these gene products are diverse, and they can affect each of the steps necessary to make and maintain a steady-state level
               of functional RNA or protein. These mechanisms include those controlling RNA synthesis, processing, and stability, and, in
               the case of protein coding genes, protein translation, modification and degradation. Here we focus on the regulation of RNA
               transcription in <span class="emphasis"><em>C. elegans</em></span>. Transcription is a necessary first step in gene expression, and transcriptional regulation plays a central role in organismal
               development and evolution (<a href="#bib129">Levine and Tjian, 2003</a>; <a href="#bib30">Chen and Rajewsky 2007</a>). Indeed, the number of specific proteins involved in transcription and its regulation increases with increasing organismal
               complexity (<a href="#bib235">Vogel and Chothia, 2006</a>).
            </p>
            <p>Many of the general characteristics that make <span class="emphasis"><em>C. elegans</em></span> an excellent model system (invariant lineage, simple anatomy, effectiveness of RNAi, etc.) also make it an excellent system
               to study transcriptional regulation. However, two specific characteristics uniquely facilitate transcriptional regulation
               studies in the worm. First, <span class="emphasis"><em>C. elegans</em></span> is transparent throughout its entire life cycle making it an ideal system to use fluorescent protein reporter genes to monitor
               gene expression in live animals with single-cell resolution. Second, the relatively compact size of the <span class="emphasis"><em>C. elegans</em></span> genome facilitates identification of cis-acting transcriptional regulatory elements (CREs) controlling gene expression. Examples
               of the success of studying transcriptional control in <span class="emphasis"><em>C. elegans</em></span> are reviewed in WormBook and elsewhere, and they include the elucidation of the transcriptional cascade controlling specification
               and differentiation of the gut, nervous system, and pharynx (see <a href="http://dx.doi.org/10.1895/wormbook.1.133.1" target="_top">The <span class="emphasis"><em>C. elegans</em></span> intestine</a>; <a href="http://dx.doi.org/10.1895/wormbook.1.12.2" target="_top">Neurogenesis in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span></a>; and <a href="http://dx.doi.org/10.1895/wormbook.1.129.1" target="_top">The <span class="emphasis"><em>C. elegans</em></span> pharynx: a model for organogenesis</a>). These two physical characteristics are augmented by facile forward and reverse genetics in the <span class="emphasis"><em>C. elegans</em></span> system. Mutations affecting numerous transcription factors and even CREs have provided important insights into the transcriptional
               control of cell fate decisions during development, such as specifying which cells undergo apoptosis (<a href="#bib35">Conradt and Xue, 2005</a>), while characterization of dosage compensation (<a href="#bib155">Meyer, 2010</a>) and the synMuv mutants (<a href="#bib56">Fay and Yochem, 2007</a>) have broadened our general understanding gene regulation at the chromatin level. These advantages have consistently provided
               novel mechanistic insights into the transcription regulation of gene expression.
            </p>
            <p>This chapter provides an overview of RNA Pol II transcription in <span class="emphasis"><em>C. elegans</em></span>, focusing on what we have learned to date about gene expression in the somatic cells of the animal. Although much of the
               content of this review is also applicable to germline transcription, readers interested in germline gene expression are encouraged
               to see WormBook chapters <a href="http://dx.doi.org/10.1895/wormbook.1.74.1" target="_top">Germline genomics</a> and <a href="http://dx.doi.org/10.1895/wormbook.1.73.1" target="_top">Germline chromatin</a>, as important differences exist in the mechanisms controlling expression in these two tissue types. This chapter takes a
               broad-strokes approach to somatic transcription while providing references to serve as entry points for those wanting to explore
               particular topics in more detail. A brief introduction into the basics of gene organization and regulation sets the stage
               for those unfamiliar with <span class="emphasis"><em>C. elegans</em></span> gene expression. This is followed by a discussion of transcription factor function, and the chapter ends with the known roles
               for chromatin in <span class="emphasis"><em>C. elegans</em></span> gene regulation. Although some basic information may duplicate that found in previous reviews of transcriptional regulation
               (<a href="http://dx.doi.org/10.1895/wormbook.1.45.1" target="_top">Transcriptional regulation</a>; <a href="http://dx.doi.org/10.1895/wormbook.1.121.1" target="_top">Transcription mechanisms</a>; <a href="#bib67">Gaudet and McGhee, 2010</a>; <a href="#bib117">Krause, 1995</a>; <a href="#bib149">McGhee and Krause, 1997</a>; <a href="#bib231">Van Nostrand and Kim, 2011</a>), this review aims to augment rather than supersede these alternate overviews as each has valuable information and a unique
               perspective on transcriptional regulation.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;RNA Polymerase II and associated factors
                     </h2>
                  </div>
               </div>
            </div>
            <p>The regulation of RNA Polymerase II (Pol II)-mediated transcription in <span class="emphasis"><em>C. elegans</em></span> can be described as typical for eucaryotes. Pol II appears to act in concert with TATA Binding Protein (TBP) and TBP-Associated
               Factors (TAFs) at the core promoter of protein coding genes (<a href="#bib39">Dantonel et al., 2000</a>; <a href="#bib101">Kaltenbach et al., 2000</a>; <a href="#bib132">Lichtsteiner and Tjian, 1993</a>; <a href="#bib240">Walker et al., 2004</a>; see also <a href="http://dx.doi.org/10.1895/wormbook.1.121.1" target="_top">Transcription mechanisms</a>). As in other eucaryotyes, the large subunit of Pol II of <span class="emphasis"><em>C. elegans</em></span>, encoded by the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ama-1;class=Gene" target="_blank">ama-1</a></em></span> gene (<a href="#bib16">Bird and Riddle, 1989</a>), has an extended C-terminal domain (CTD) that likely serves as a binding site for protein complexes involved in co-transcriptional
               mRNA processing and histone modification (<a href="#figure1">Figure 1</a>; from <a href="#bib180">Phatnani and Greenleaf, 2006</a>). Active Pol II is phosphorylated on the CTD at serine 2 and 5 (Ser2P or Ser5P) of the conserved heptad repeat (YSTPSPS)
               and its variants, as it is in other eucaryotes (<a href="#bib200">Seydoux and Dunn, 1997</a>; <a href="#bib242">Wallenfang and Seydoux, 2002</a>; Zhang et al., 2003). The levels of Ser2P increase while the Ser5P levels decrease with transcriptional elongation in eucaryotes
               (<a href="#bib23">Buratowski, 2009</a>) and there is some evidence for the same in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib65">Garrido-Lecca and Blumenthal, 2010</a>). Interestingly, many of the antibodies used to distinguish these Pol II isoforms based on heptad repeat phosphorylation
               epitopes yield very similar patterns by chromatin immunoprecipitation (ChIP) in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib10">Baugh et al., 2009</a>; <a href="#bib179">Pferdehirt et al., 2011</a>). The functions of many of the other core transcription factors are similarly conserved with one of the only major differences
               being the absence of the negative elongation factor NELF (reviewed in <a href="http://dx.doi.org/10.1895/wormbook.1.121.1" target="_top">Transcription mechanisms</a>).
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="transregulate_fig1.jpg"><img src="transregulate_fig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>

               </table>
               <div class="caption">
                  <p><b><span class="bold"><strong>Figure 1: A hypothetical RNAPII elongation megacomplex.</strong></span>&nbsp;</b>RNAPII (including the extended CTD with SerPO4 knobs) is purple; the globular and CTD portions are drawn approximately to
                     scale for mammalian RNAPII.  Orange DNA is wrapped around yellow nucleosomal histones; nucleosomes modified by Set2 are shaded
                     darker. The nascent RNA transcript is green. Yeast names are used for PCAPs (e.g., <a href="#bib180">Phatnani and Greenleaf, 2006</a>), not all of which are shown. (CBC) cap-binding complex; (CRF) chromatin remodeling factor; (XF) processing/export factor.
                     (Used with permission <a href="#bib180">Phatnani and Greenleaf, 2006</a>)
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;<span class="emphasis"><em>C. elegans</em></span> gene organization and regulation
                     </h2>
                  </div>
               </div>
            </div>
            <p>Transcriptional regulation results from a complex organization of cis-acting sequences that serve as binding sites for a multitude
               of trans-acting factors that together determine if a gene will be active or silent. In higher eucaryotes, these cis-acting
               sequences are typically clustered into discrete functional modules, including the core promoter, extended proximal and downstream
               promoter regions, positive and negative enhancers, and insulators as diagramed in <a href="#figure2">Figure 2</a> (<a href="#bib129">Levine and Tjian, 2003</a>).
            </p>
            <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="transregulate_fig2.jpg"><img src="transregulate_fig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b><span class="bold"><strong>Figure 2: Metazoan regulatory modules controlling transcription.</strong></span>&nbsp;</b>Shown is a diagram of a typical metazoan gene illustrating the complex interactions among <span class="emphasis"><em>cis</em></span>-acting modules and <span class="emphasis"><em>trans</em></span>-acting factors regulating gene expression.  Note that both positive and negative control regions are interspersed with promoter
                     modules, all of which can be further influenced by distal regions regulating chromatin configuration, such as insulators.
                     (Used with permission, <a href="#bib129">Levine and Tjian, 2003</a>)
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-1"></a>3.1.&nbsp;Proximal control regions
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The majority of protein-coding genes in <span class="emphasis"><em>C. elegans</em></span> are within relatively gene-dense regions of the genome. Consequently, cis-acting regulatory regions are usually close to
                  the coding region. In fact, a good rule of thumb for <span class="emphasis"><em>C. elegans</em></span> is that the minimal set of cis-acting sequences sufficient to regulate proper gene expression is found within 2 kb upstream
                  of the translational start codon. Often, another gene is present on the same or opposite strand and located less than 2 kb
                  upstream of the gene of interest. Generally in these cases, one assumes the minimal promoter is restricted to the non-coding,
                  intergenic region. There are notable exceptions to this compact view of cis-acting sequences. For example, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-1;class=Gene" target="_blank">egl-1</a></em></span> expression is controlled, in part, by elements located both upstream and more than 4 kb downstream of the coding region (<a href="#bib226">Thellmann et al., 2003</a>; <a href="#bib252">Winn et al., 2011</a>). For <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-39;class=Gene" target="_blank">lin-39</a></em></span>, proper reporter gene expression required inclusion of &#8764;30 kb of genomic DNA that extends long distances upstream and downstream
                  of the protein coding region (<a href="#bib238">Wagmaister et al., 2006</a>). Clearly <span class="emphasis"><em>C. elegans</em></span> genes can have complex and distant control regions so the 2 kb rule of thumb should not be mistaken for dogma.
               </p>
               <p>It is important to remember that the minimal promoter region is not synonymous with the natural promoter. The natural promoter
                  may span a much larger region due to redundancy in the function of regulatory elements that ensure proper and robust regulation
                  of the endogenous gene. One common site of additional control elements is within the introns. Most <span class="emphasis"><em>C. elegans</em></span> introns are small (e.g., &lt;100 bp; see <a href="http://dx.doi.org/10.1895/wormbook.1.65.1" target="_top">Overview of gene structure</a>) and are thus unlikely to contain elements controlling expression. However, introns larger than several hundred base pairs
                  do often have such elements (e.g., <a href="#bib162">Nam et al., 2002</a>; <a href="#bib170">Okkema et al., 1993</a>; <a href="#bib115">Kostrouchova et al., 1998</a>). Therefore, intron size can provide a clue in searching for transcriptional control sequences.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-2"></a>3.2.&nbsp;Distal control regions
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The relatively compact <span class="emphasis"><em>C. elegans</em></span> genome may also underlie the apparent absence of long-range control mechanisms for gene regulation that are common to other
                  metazoa. For example, CTCF in vertebrates and flies plays a key role in long-range chromatin organization and can block enhancer-stimulated
                  gene expression, thus functioning as an insulator as shown in <a href="#figure2">Figure 2</a> (<a href="#bib241">Wallace and Felsenfeld, 2007</a>). To date, there is no evidence for an ortholog or functional equivalent of CTCF in <span class="emphasis"><em>C. elegans</em></span>. Thus, both the local and global organization of genes in <span class="emphasis"><em>C. elegans</em></span> appears to be relatively simple in comparison with other metazoans, presumably simplifying our understanding of transcriptional
                  regulation.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-3"></a>3.3.&nbsp;Transcriptional initiation and trans-splicing
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Another unusual aspect of the <span class="emphasis"><em>C. elegans</em></span> system to note when considering transcriptional regulation is the occurrence of trans-splicing, a process that replaces the
                  initial transcript 5&#8217; untranslated region (5&#8217; UTR) with a 22 nucleotide leader sequence (SL1) for the majority of Pol II messages
                  (<a href="#bib1">Allen et al., 2011</a>; <a href="#bib120">Krause and Hirsh, 1987</a>; <a href="http://dx.doi.org/10.1895/wormbook.1.5.2" target="_top">Trans-splicing and operons in <span class="emphasis"><em>C. elegans</em></span></a>). Therefore, mapping the 5&#8217; start site of the mature mRNA often only reveals the site of trans-splicing, not transcriptional
                  initiation, complicating analyses that are commonplace in other systems. The ability to trans-splice messages also provides
                  a processing mechanism for polycistronic messages, or operons, in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib214">Spieth et al., 1993</a>; reviewed in <a href="http://dx.doi.org/10.1895/wormbook.1.5.2" target="_top">Trans-splicing and operons in <span class="emphasis"><em>C. elegans</em></span></a>). The sequence of the spliced leader present on the mature message distinguishes the first gene of an operon (no SL or SL1
                  trans-spliced) versus internal messages of the operon (SL2 trans-spliced). Typically, the tightly clustered genes of an operon
                  are co-regulated due to their polycistronic nature, although the use of different promoters upstream and within operons can
                  result in independent transcriptional regulation of one or more of the mRNAs (<a href="#bib1">Allen et al., 2011</a>; <a href="#bib91">Huang et al., 2007</a>; <a href="#bib257">Yin et al., 2010</a>; <a href="#bib159">Morton and Blumenthal, 2011</a>; <a href="http://dx.doi.org/10.1895/wormbook.1.5.2" target="_top">Trans-splicing and operons in <span class="emphasis"><em>C. elegans</em></span></a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3-4"></a>3.4.&nbsp;Core promoter elements
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Ironically, the study of common sequences among messages trans-spliced provided the first systematic information on common
                  promoter sequences (<a href="#bib72">Graber et al., 2007</a>), revealing the presence of a consensus Kozak sequence regulating translational initiation (<a href="#bib116">Kozak, 1981</a>). A more recent study of promoter sequences has extended this analysis to define the core promoter sequence elements typically
                  present in <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib74">Grishkevich et al., 2011</a>). The five elements commonly observed are an Sp1 like site (CNCCGCCC), T-blocks that correlate with nucleosome eviction and
                  gene expression levels (TTTT[N/T]), TATA box (GTATA[TA][TA]AG), trans-splicing site (TTnCAG), and Kozak site that includes
                  the translation initiation codon ([CA]AA[CA]ATG) (<a href="#bib74">Grishkevich et al., 2011</a>) (<a href="#figure3">Figure 3</a>). The sensitivity and depth of coverage of next-generation sequencing-based techniques will allow fine scale mapping of primary
                  transcript start sites of outron-containing genes in the near future, aiding in our understanding of transcriptional initiation
                  and requisite regulatory sequences in the near future.
               </p>
               <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="transregulate_fig3.jpg"><img src="transregulate_fig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b><span class="bold"><strong>Figure 3:</strong></span> Core promoter motif composition among <span class="emphasis"><em>Caenorhabditis</em></span> <span class="bold"><strong>promoters.</strong></span>&nbsp;</b>Motif composition of the <span class="emphasis"><em>Caenorhabditis</em></span> core promoter. (A) Five conserved motifs in each of the five examined <span class="emphasis"><em>Caenorhabditis</em></span> species are shown as sequence logos. (*) In contrast to all other motifs that were found in the initial search, the <span class="emphasis"><em>Caenorhabditis japonica</em></span> TATA box motif was detected only in sequences whose orthologs contained the &#8220;TATA&#8221; motif. (B) Distribution of motifs relative
                        to the translational start codon. The gray box in each plot corresponds to the core promoter.  The area under the curve is
                        the total frequency of occurrence within the core promoter region with the line indicating the frequency at each position
                        as indicated by the scale to the left.  The <span class="emphasis"><em>C. japonica</em></span> SL1 motif was normalized to the length of the other species. (C)  Frequency table for each sequence motif.  (Figure and data
                        used with permission, <a href="#bib74">Grishkevich et al., 2011</a>)
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Promoter complexity
                     </h2>
                  </div>
               </div>
            </div>
            <p>Two general types of promoter organization have been described in <span class="emphasis"><em>C. elegans</em></span>, simple and complex. A simple promoter is defined here as one in which the cis-acting control elements necessary for proper
               expression are confined to a small region (a few hundreds of bp) of the genome. Housekeeping genes expressed in all tissues
               are good candidates for regulation by simple promoters, although few housekeeping genes in <span class="emphasis"><em>C. elegans</em></span> have been well characterized. Among the best-characterized simple promoters are those of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hsp-16;class=Gene" target="_blank">hsp-16</a></em></span> family of genes. This family consists of pairs of divergently transcribed genes with promoter regions sufficient for heat-regulated
               expression contained within the short (&#8764;350 bp) intergenic regions (<a href="#bib99">Jones et al., 1986</a> <a href="#bib196">Russnak and Candido, 1985</a>; <a href="#bib221">Stringham et al., 1992</a>). Despite these compact promoters, distinct tissue expression patterns are induced from different <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hsp-16;class=Gene" target="_blank">hsp-16</a></em></span> promoters (<a href="#bib221">Stringham et al., 1992</a>), suggesting the presence of multiple regulatory sites within these simple promoters. Another excellent example of simple
               promoters are in the vitellogenin (<span class="emphasis"><em>vit</em></span>) genes, which exhibit stage-, tissue- and sex-specific expression controlled, in the case of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=vit-2;class=Gene" target="_blank">vit-2</a></em></span>, by a 247 bp promoter (<a href="#bib137">MacMorris et al., 1992</a>; <a href="#bib138">MacMorris et al., 1994</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=vit-2;class=Gene" target="_blank">vit-2</a></em></span> promoter activity depends on GATA-factor binding sites and a novel VPE2 site (TGTCAAT) conserved in <span class="emphasis"><em>vit</em></span> gene promoters in <span class="emphasis"><em>C. elegans</em></span> and <span class="emphasis"><em>C. briggsae</em></span> (<a href="#bib215">Spieth et al., 1985</a>; Zucker-Aprison and Blumenthal, 1989). Certain cell cycle promoters have also been shown to be remarkably simple. Analysis
               of several genes expressed only in proliferative cells and encoding <a href="http://www.wormbase.org/db/get?name=G1;class=Cell" target="_blank">G1</a> phase regulators (e.g., cyclin D) revealed that proper regulation required as little as 67 base pairs from the promoter (<a href="#bib21">Brodigan et al., 2003</a>; <a href="#bib175">Park and Krause, 1999</a>).
            </p>
            <p>In contrast to the simple promoters, complex promoters contain dispersed regulatory elements in which the overall pattern
               of gene expression is the result of the composite action of several dispersed elements, each influencing or contributing to
               the overall expression pattern. Complex promoters are often associated with regulatory genes controlling a key developmental
               decision. For example, the piecemeal organization of the regulatory regions for <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hlh-1;class=Gene" target="_blank">hlh-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-26;class=Gene" target="_blank">lin-26</a></em></span> reflect a need for these promoters to integrate a variety of different cell lineage inputs to control proper cell fate specification
               in the correct cells at the appropriate time during development (see previous version of this chapter, <a href="http://dx.doi.org/10.1895/wormbook.1.45.1" target="_top">Trancriptional regulation</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;Transcription factors
                     </h2>
                  </div>
               </div>
            </div>
            <p>Proper spatial and temporal regulation of gene expression depends on the binding of transcription factors to specific gene
               cis-regulatory sequences (<a href="#bib129">Levine and Tjian, 2003</a>). A variety of <span class="emphasis"><em>C. elegans</em></span> transcription factors are well characterized and have important developmental roles; however it remains a challenge to accurately
               catalog all the transcription factors encoded in the <span class="emphasis"><em>C. elegans</em></span> genome. Automated searches for gene ontology terms associated with transcriptional regulation can result in inclusion of
               false positive hits for transcription factors (<a href="#bib186">Reece-Hoyes et al., 2005</a>; <a href="#bib232">Vaquerizas et al., 2009</a>). In some cases there is ambiguity as to whether a particular domain defines the protein as a DNA-binding factor. For example,
               various types of zinc finger domains can bind DNA but can also serve other functions, including RNA binding and protein-protein
               interactions (<a href="#bib64">Gamsjaeger et al., 2007</a>; <a href="#bib146">Matthews and Sunde, 2002</a>). Likewise, for factors that modify chromatin or participate in a transcription complex, the definition of a transcription
               factor often lies in the eyes of the investigator. Finally, gene annotations change as new information regarding gene structure
               is obtained and new classes of DNA-binding transcription factors are discovered. Thus any list of transcription factors must
               be manually curated and periodically updated to include the latest gene annotations.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-1"></a>5.1.&nbsp;Transcription factor resources
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Several groups have produced excellent catalogs of transcription factors in <span class="emphasis"><em>C. elegans</em></span> using predictions based on gene ontology terms associated with transcription and DNA-binding domain assignments, followed
                  by manual curation, to produce lists of transcription factor genes containing between 934 and 988 genes (<a href="#bib9">Barrasa et al., 2007</a>; <a href="#bib76">Haerty et al., 2008</a>; <a href="#bib186">Reece-Hoyes et al., 2005</a>; <a href="#bib250">Wilson et al., 2008</a>; <a href="#bib187">Reece-Hoyes et al., 2011</a>). While these lists are largely overlapping, they are not identical, so it may be useful to scan each of these lists for
                  your favorite genes (<a href="http://edgedb.umassmed.edu/" target="_top">http://edgedb.umassmed.edu/</a>; <a href="http://www.macwormlab.net/ntfdb/index.php" target="_top">http://www.macwormlab.net/ntfdb/index.php</a>; <a href="http://www.transcriptionfactor.org/index.cgi?Home" target="_top">http://www.transcriptionfactor.org/index.cgi?Home</a>).
               </p>
               <p>The most recent catalog is wTF2.2 (<a href="#bib187">Reece-Hoyes et al., 2011</a>), and we have updated and annotated this list with current gene names (link to <a href="Table_1_Ce_TF_list.xls" target="_blank">Table 1</a>. Catalog of transcription factors.).
                  Fifty-three transcription factor genes were removed from wTF2.2 either due to reannotation of the genome, or because there
                  was only weak evidence that motifs in these genes encoded DNA-binding domains [e.g., ZF - A20 (IPR002653), ZF - CCCH (IPR000571),
                  ZF - DHHC (IPR001594), and ZF - MIZ(IPR004181)]. We have also added forty transcription factor genes that had been identified
                  as DNA binding proteins in large scale yeast one-hybrid screens (<a href="#bib46">Deplancke et al., 2006</a>; <a href="#bib187">Reece-Hoyes et al., 2011</a>), or based on information from other transcription factor catalogs (<a href="#bib76">Haerty et al., 2008</a>; <a href="#bib250">Wilson et al., 2008</a>). Our catalog contains 924 transcription factor genes, which is &#8764;4.6% of all protein coding genes (WS220). This number is
                  slightly less than the frequency of transcription factor genes in the human genome (&#8764;6% of the protein coding genes) (<a href="#bib232">Vaquerizas et al., 2009</a>). Commercially available feeding RNAi clones targeting many of these genes are available (<a href="Table_1_Ce_TF_list.xls" target="_blank">Table 1</a>), facilitating functional
                  analysis of <span class="emphasis"><em>C. elegans</em></span> transcription factors.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-2"></a>5.2.&nbsp;Transcription factor families
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. elegans</em></span> contains representatives of most major transcription factor families found in other animals (Tables <a href="Table_1_Ce_TF_list.xls" target="_blank">1</a> and <a href="#table2" title="Table 2. DNA-binding families">2</a>), and 344 (37%) of the <span class="emphasis"><em>C. elegans</em></span> transcription factor genes have been matched with clear human orthologs by reciprocal BLAST analysis or using orthology prediction
                  programs (<a href="Table_1_Ce_TF_list.xls" target="_blank">Table 1</a>) (<a href="#bib186">Reece-Hoyes et al., 2005</a>; <a href="#bib202">Shaye and Greenwald, 2011</a>). Interestingly, some families of DNA-binding domains seem more highly conserved during animal evolution than others (<a href="#figure4">Figure 4</a>). For example, 19 out of 19 (100%) MYB-like factors and 30 out of 41 bHLH factors (73%) have human orthologs. In comparison,
                  only 6 out of 22 T-box genes (28%) and 0 out of 9 MADF-family genes (0%) have been conserved. As has been noted previously
                  (<a href="#bib186">Reece-Hoyes et al., 2005</a>; <a href="#bib212">Sluder et al., 1999</a>), the number of likely nuclear hormone receptors (NHRs) has expanded greatly in <span class="emphasis"><em>C. elegans</em></span> (272 members in <a href="#table2" title="Table 2. DNA-binding families">Table 2</a>) relative to humans (46 members; IPR001628; (<a href="#bib232">Vaquerizas et al., 2009</a>)).
               </p>
               <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="transregulate_fig4.jpg"><img src="transregulate_fig4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b><span class="bold"><strong>Figure 4: <span class="emphasis"><em>C. elegans</em></span> transcription factor genes with clear orthologs in the human genome.</strong></span>&nbsp;</b>The percentage of <span class="emphasis"><em>C. elegans</em></span> genes from each DNA-binding domain family with clear orthologs in the human genome  (<a href="#bib186">Reece-Hoyes et al., 2005</a>; <a href="#bib202">Shaye and Greenwald, 2011</a>).  Only DNA-binding domain families with 3 or more <span class="emphasis"><em>C. elegans</em></span> members are included.
                     </p>
                  </div>
               </div>
               <div class="table"><a name="table2"></a><p class="title">Table 2. DNA-binding families</p>
                  <table summary="Table 2. DNA-binding families" border="1">
                     <colgroup>
                        <col>
                        <col>
                     </colgroup>
                     <tbody>
                        <tr valign="bottom">
                           <td align="left" valign="bottom">DNA-binding Domain</td>
                           <td align="left" valign="bottom">Total members</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - NHR</td>
                           <td align="left" valign="top">272</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - C2H2</td>
                           <td align="left" valign="top">217</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">HD</td>
                           <td align="left" valign="top">101</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">bHLH</td>
                           <td align="left" valign="top">41</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">novel</td>
                           <td align="left" valign="top">34</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">bZIP</td>
                           <td align="left" valign="top">33</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">AT Hook</td>
                           <td align="left" valign="top">29</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">T-box</td>
                           <td align="left" valign="top">22</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">MYB</td>
                           <td align="left" valign="top">19</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">WH - Fork Head</td>
                           <td align="left" valign="top">18</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">HMG box</td>
                           <td align="left" valign="top">16</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - GATA</td>
                           <td align="left" valign="top">14</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">MADF</td>
                           <td align="left" valign="top">11</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - DM</td>
                           <td align="left" valign="top">11</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">WH - ETS</td>
                           <td align="left" valign="top">10</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">CBF</td>
                           <td align="left" valign="top">9</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - BED</td>
                           <td align="left" valign="top">9</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">HTH</td>
                           <td align="left" valign="top">7</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">MH1</td>
                           <td align="left" valign="top">7</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">Paired Domain</td>
                           <td align="left" valign="top">7</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - THAP</td>
                           <td align="left" valign="top">6</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ARID/BRIGHT</td>
                           <td align="left" valign="top">5</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">COLD BOX</td>
                           <td align="left" valign="top">5</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">AP-2</td>
                           <td align="left" valign="top">4</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">SAND</td>
                           <td align="left" valign="top">4</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">WH - TDP</td>
                           <td align="left" valign="top">4</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - FLYWCH</td>
                           <td align="left" valign="top">4</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">IPT/TIG</td>
                           <td align="left" valign="top">3</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">p53</td>
                           <td align="left" valign="top">3</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">TSC-22/dip/bun</td>
                           <td align="left" valign="top">3</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">WH - HSF</td>
                           <td align="left" valign="top">3</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">Brinker</td>
                           <td align="left" valign="top">2</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">GC-rich DNA-binding domain</td>
                           <td align="left" valign="top">2</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">MADS box</td>
                           <td align="left" valign="top">2</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">PUR</td>
                           <td align="left" valign="top">2</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">WH</td>
                           <td align="left" valign="top">2</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - NF-X1 - 10 domains</td>
                           <td align="left" valign="top">2</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=CP2;class=Cell" target="_blank">CP2</a></td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">p66 family</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">RNT</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">RPEL</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">STAT</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">TEA/ATTS</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">TRAP230 family</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">WH - DAC</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">WH - RFX</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">YL1 TF</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - C2CH</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top">ZF - C2HC</td>
                           <td align="left" valign="top">1</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-3"></a>5.3.&nbsp;Interspecies comparisons
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Genome sequencing of additional <span class="emphasis"><em>Caenorhabditis</em></span> species allows a comparison of the <span class="emphasis"><em>C. elegans</em></span> transcription factor gene family to those in other species (<a href="#bib76">Haerty et al., 2008</a>). <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span> contain similar numbers of transcription factor genes to <span class="emphasis"><em>C. elegans</em></span>, and approximately 72% of the <span class="emphasis"><em>C. elegans</em></span> transcription factor genes have detectable orthologs in both <span class="emphasis"><em>C. briggsae</em></span> and <span class="emphasis"><em>C. remanei</em></span>. This proportion of orthology is higher than that found overall for protein coding genes, suggesting transcription factor
                  genes are under strong selective pressure. Transcription factor genes are not uniformly distributed on the chromosomes in
                  <span class="emphasis"><em>C. elegans</em></span> or <span class="emphasis"><em>C. briggsae</em></span>, and many genes are located in clusters that are enriched for transcription factor genes compared to non-transcription factor
                  genes. Furthermore members of gene families such as NHR, HOX, and T-box are frequently clustered in tandem arrays (<a href="#bib76">Haerty et al., 2008</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-4"></a>5.4.&nbsp;Transcription factor targets
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A major goal in studying transcription is to make the link between transcription factors and their target genes. These links
                  have traditionally been made by identifying binding sites in experimentally verified targets of transcription factors by detailed
                  promoter analyses. While this approach is still valuable, more recent techniques such as PCR based binding site selection,
                  microarray analyses, yeast one-hybrid screens, and chromatin immunoprecipitation (ChIP) assays have expanded our ability to
                  identify transcription factor binding sites and candidate target genes on a genome-wide scale (e.g., <a href="#bib46">Deplancke et al., 2006</a>; <a href="#bib147">McElwee et al., 2003</a>; <a href="#bib166">Niu et al., 2011</a>; <a href="#bib261">Zhong et al., 2010</a>). Our knowledge of transcription factor binding site specificity will continue to increase, but we provide references to
                  find information about DNA binding site specificity and potential target genes for some <span class="emphasis"><em>C. elegans</em></span> transcription factors (link to <a href="Table_3_binding_refs.xls" target="_blank">Table 3</a>. Transcription factor targets.). As described more fully below (see <a href="#sec7" title="7.&nbsp;Systematic genome-scale analysis of transcription regulation">Section 7</a>), data from many of these ChIP analyses is available at <a href="http://www.modencode.org/" target="_top">modENCODE</a> and through <a href="http://www.wormbase.org/" target="_top">WormBase</a>. Likewise candidate transcription factor binding sites predicted from published data and user generated position weight matrices
                  can be visualized in the <a href="http://www.wormbase.org/tools/genome/gbrowse/c_elegans/" target="_top">Genome Browser</a> at <a href="http://www.wormbase.org/" target="_top">WormBase</a> and at <a href="http://www.modencode.org/" target="_top">modENCODE</a> by accessing the <a href="http://data.modencode.org/?Organism=C.%20elegans" target="_top">Sequence Motif track</a>.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-5"></a>5.5.&nbsp;Transcription factor gene expression
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Transcription factor gene expression is often highly regulated, and understanding the spatial and temporal expression patterns
                  of transcription factors is a key to determining their function. While a variety of transcription factors have been characterized
                  individually using reporter genes, antibodies and in situ hybridizations (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hlh-1;class=Gene" target="_blank">hlh-1</a>, <a href="http://www.wormbase.org/db/get?name=pha-4;class=Gene" target="_blank">pha-4</a>, <a href="http://www.wormbase.org/db/get?name=ceh-22;class=Gene" target="_blank">ceh-22</a>, <a href="http://www.wormbase.org/db/get?name=tbx-37;class=Gene" target="_blank">tbx-37</a>, <a href="http://www.wormbase.org/db/get?name=unc-86;class=Gene" target="_blank">unc-86</a>, <a href="http://www.wormbase.org/db/get?name=cnd-1;class=Gene" target="_blank">cnd-1</a>, <a href="http://www.wormbase.org/db/get?name=end-3;class=Gene" target="_blank">end-3</a></em></span>) (<a href="#bib57">Finney and Ruvkun, 1990</a>; <a href="#bib71">Good et al., 2004</a>; <a href="#bib78">Hallam et al., 2000</a>; <a href="#bib89">Horner et al., 1998</a>; <a href="#bib100">Kalb et al., 1998</a>; <a href="#bib118">Krause et al., 1990</a>; <a href="#bib139">Maduro et al., 2007</a>; <a href="#bib169">Okkema and Fire, 1994</a>), high throughput techniques such as cell type-specific microarrays and SAGE analyses are providing genome-wide gene expression
                  data (<a href="#bib60">Fox et al., 2005</a>; <a href="#bib151">McKay et al., 2003</a>; <a href="#bib152">Meissner et al., 2009</a>; <a href="#bib193">Roy et al., 2002</a>; <a href="#bib213">Spencer et al., 2011</a>; <a href="#bib236">Von Stetina et al., 2007</a>; Zhang et al., 2002). This useful data is available for transcription factor genes (and all other genes) online through <a href="http://www.wormbase.org/" target="_top">WormBase</a> and <a href="http://www.modencode.org/" target="_top">modENCODE</a>, while additional analyses regarding tissue specificity are available at <a href="http://www.vanderbilt.edu/wormdoc/wormmap/Welcome.html" target="_top">http://www.vanderbilt.edu/wormdoc/wormmap/Welcome.html</a> (<a href="#bib213">Spencer et al., 2011</a>). In addition, automated analyses of fluorescent protein reporter gene expression are accelerating our knowledge of transcription
                  factor gene expression during embryogenesis and in L1 larvae (<a href="#bib136">Liu et al., 2009</a>; <a href="#bib161">Murray et al., 2008</a>) (<a href="#figure5">Figure 5</a>). Lineage-based gene expression data generated using these high throughput approaches can be accessed online through <a href="http://epic.gs.washington.edu/" target="_top">EPIC</a> and <a href="http://www.computationalbio.com/Stanford/KimLab/wormDB/" target="_top">WormDB</a>. This data is useful to investigators interested in specific transcription factors, but it also opens the exciting possibility
                  of using computational approaches to overlay and identify correlations between gene expression patterns to understand how
                  networks of transcription factors control development (<a href="#figure5">Figure 5</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="transregulate_fig5.jpg"><img src="transregulate_fig5_s.jpg" border="2" align="middle" alt=" figure 5"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b><span class="bold"><strong>Figure 5: Automated analyses of transcription factor gene expression.</strong></span>&nbsp;</b>(A-D) Representative frames showing expression of mCherry reporters for the indicated transcription factors (red) ovelayed
                        on ubiquitous histone::gfp reporter expression marking all nuclei.  Anterior is left.  (E) Embryonic lineage trees showing
                        expression of the indicated transcription factor::mCherry transgenes (colored), or a merged image showing expression of all
                        four transgenes.  This data was acquired and visualized using StarryNite and AceTree (<a href="#bib161">Murray et al., 2008</a>).  Panels A-D were acquired from movies at http://epic.gs.washington.edu/.  Panel E was adapted with permission from <a href="#bib161">Murray et al., 2008</a>.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5-6"></a>5.6.&nbsp;Post-transcriptional regulation of transcription factor function
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Covalent post-translational modifications also play important roles in regulating transcription factor function (<a href="#bib228">Tootle and Rebay, 2005</a>), and a variety of <span class="emphasis"><em>C. elegans</em></span> transcription factors are regulated by modifications including phosphorylation, proteolysis, ubiquitination, and SUMOylation.
                  For example, nuclear localization and DNA-binding activity of the FoxO-family factor <a href="http://www.wormbase.org/db/get?name=DAF-16;class=Protein" target="_blank">DAF-16</a> is negatively regulated by the <a href="http://www.wormbase.org/db/get?name=DAF-2;class=Protein" target="_blank">DAF-2</a>/insulin like signaling pathway through phosphorylation at phylogenetically conserved sites (<a href="#bib24">Cahill et al., 2001</a>; <a href="#bib133">Lin et al., 2001</a>), whereas <a href="http://www.wormbase.org/db/get?name=DAF-16;class=Protein" target="_blank">DAF-16</a> protein stability is regulated by ubiquitination (<a href="#bib131">Li et al., 2007</a>). Likewise the Gli family factor <a href="http://www.wormbase.org/db/get?name=TRA-1;class=Protein" target="_blank">TRA-1</a> activity is regulated by specific proteolytic cleavage and by ubiquitin mediated targeting to the proteosomal degradation
                  pathway (<a href="#bib199">Schvarzstein and Spence, 2006</a>; <a href="#bib217">Starostina et al., 2007</a>). Both the ETS-family factor <a href="http://www.wormbase.org/db/get?name=LIN-1;class=Protein" target="_blank">LIN-1</a> and the T-box factor <a href="http://www.wormbase.org/db/get?name=TBX-2;class=Protein" target="_blank">TBX-2</a> require SUMOylation (<a href="#bib128">Leight et al., 2005</a>; <a href="#bib192">Roy Chowdhuri et al., 2006</a>), and at least 20 other transcription factors have been identified as possible SUMOylation targets (<a href="#bib102">Kaminsky et al., 2009</a>). SUMOylation of <a href="http://www.wormbase.org/db/get?name=LIN-1;class=Protein" target="_blank">LIN-1</a> promotes interaction with <a href="http://www.wormbase.org/db/get?name=MEP-1;class=Protein" target="_blank">MEP-1</a>, a component of the NuRD (see <a href="#sec6-1" title="6.1.&nbsp;The NuRD complex">Section 6.1</a>) transcriptional repression complex (<a href="#bib128">Leight et al., 2005</a>). Transcription factor SUMOylation has also been shown to recruit <span class="emphasis"><em>Drosophila</em></span> <a href="http://www.wormbase.org/db/get?name=MEP-1;class=Protein" target="_blank">MEP-1</a> and the NuRD complex, suggesting a conserved mechanism for SUMOylation dependent transcriptional repressors (<a href="#bib218">Stielow et al., 2008</a>).
               </p>
               <p>Non-covalent interactions of transcriptional co-activators or co-repressors with DNA-bound transcription factors are also
                  crucial for transcription factor function. The transcriptional co-activator p300/CBP is important for embryonic development
                  (<a href="#bib204">Shi and Mello, 1998</a>), while the co-repressors CtBP and <a href="http://www.wormbase.org/db/get?name=SIR-2.1;class=Protein" target="_blank">SIR-2.1</a> play important roles regulating life span (<a href="#bib31">Chen et al., 2009</a>; <a href="#bib227">Tissenbaum and Guarente, 2001</a>). The best-studied transcriptional co-repressor in <span class="emphasis"><em>C. elegans</em></span> is the Groucho-family factor <a href="http://www.wormbase.org/db/get?name=UNC-37;class=Protein" target="_blank">UNC-37</a>, which was first characterized based on its interaction with the homeodomain factor <a href="http://www.wormbase.org/db/get?name=UNC-4;class=Protein" target="_blank">UNC-4</a> (<a href="#bib253">Winnier et al., 1999</a>). Not surprisingly, <a href="http://www.wormbase.org/db/get?name=UNC-37;class=Protein" target="_blank">UNC-37</a>/Groucho has also been shown to function with a variety of transcription factors in <span class="emphasis"><em>C. elegans</em></span>, including <a href="http://www.wormbase.org/db/get?name=POP-1;class=Protein" target="_blank">POP-1</a>, <a href="http://www.wormbase.org/db/get?name=COG-1;class=Protein" target="_blank">COG-1</a>, <a href="http://www.wormbase.org/db/get?name=REF-1;class=Protein" target="_blank">REF-1</a>, <a href="http://www.wormbase.org/db/get?name=RNT-1;class=Protein" target="_blank">RNT-1</a> <a href="http://www.wormbase.org/db/get?name=UNC-30;class=Protein" target="_blank">UNC-30</a>, and <a href="http://www.wormbase.org/db/get?name=MLS-1;class=Protein" target="_blank">MLS-1</a> (<a href="#bib25">Calvo et al., 2001</a>; <a href="#bib29">Chang et al., 2003</a>; <a href="#bib157">Miller and Okkema, 2011</a>; <a href="#bib165">Neves and Priess, 2005</a>; <a href="#bib177">Peden et al., 2007</a>; <a href="#bib254">Xia et al., 2007</a>). The number of transcription factors that function as <a href="http://www.wormbase.org/db/get?name=UNC-37;class=Protein" target="_blank">UNC-37</a> dependent repressors is likely to be larger, as potential Groucho-interaction motifs are found in many <span class="emphasis"><em>C. elegans</em></span> transcription factors (<a href="#bib36">Copley, 2005</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;Chromatin status and transcription
                     </h2>
                  </div>
               </div>
            </div>
            <p>The initiation, elongation, and termination of transcription is influenced by both local and chromosome-wide chromatin configuration,
               and vice versa. Many excellent reviews provide in depth treatments of this topic (<a href="#bib14">Berger, 2010</a>; <a href="#bib258">Yun et al., 2011</a>; and <a href="#bib8">Bannister and Kouzarides, 2011</a>). Here, basic information that is generally applicable across species is briefly summarized prior to a discussion of chromatin
               regulatory complexes in <span class="emphasis"><em>C. elegans</em></span>.
            </p>
            <p>Nucleosomes, around which DNA is wound, are composed of a histone protein octamer (two each of H2A, H2B, H3, and H4) that
               can be post-translationally modified in a variety of ways. Typically, residues on the amino terminal tails of individual histones
               are modified by phosphorylation, methylation, acetylation, and ubiquitination. The specific amino acid residue of the histone
               protein that is targeted, the type of modification, and the location of the nucleosome relative to the gene body, can all
               have effects on transcription, or be altered by transcriptional events. These effects include altering nucleosome density
               and changing the level of chromatin compaction to either relax or condense a region, which has been predicted to facilitate
               or prohibit association of different transcriptional regulatory complexes. Additionally, histone modifications can provide
               specific recruitment sites for different transcriptional regulatory complexes. Moreover, the number of methyl groups modifying
               a particular residue can have distinct effects on gene expression. As one example, monomethylation of lysine 20 of histone
               H4 affects transcription, while dimethylation is associated with DNA repair, and trimethylation facilitates heterochromatin
               formation (<a href="#bib7">Balakrishnan and Milavetz, 2010</a>). Finally, protein variants of histones, such as H2A.z or CENPA, can be incorporated into the core histone octamer, with
               differential effects on chromatin configuration and function as well, through regulating noncoding RNA transcription in centromeric
               regions (reviewed in <a href="#bib220">Stimpson and Sullivan, 2010</a>).
            </p>
            <p>Over the past several years, the role of histone modifications, histone variants, and histone modifying enzymes in regulating
               gene expression during <span class="emphasis"><em>C. elegans</em></span> development has become clearer. Genome-wide studies of the distribution of individual chromatin marks provide a glimpse into
               the complex combinatorial &#8220;codes&#8221; that are possible and that are associated with gene expression (<a href="#figure6">Figure 6</a>). Core chromatin regulatory complexes that have been studied in other organisms, such as NuRD (<a href="#sec6-1" title="6.1.&nbsp;The NuRD complex">Section 6.1</a>), MLL/COMPASS (<a href="#sec6-2" title="6.2.&nbsp;The MLL/COMPASS complex">Section 6.2</a>), and PcG (<a href="#sec6-2" title="6.2.&nbsp;The MLL/COMPASS complex">Section 6.2</a>), are also present in recognizable form in <span class="emphasis"><em>C. elegans</em></span>. An extensive review of germline chromatin is available as a separate chapter in WormBook (<a href="http://dx.doi.org/10.1895/wormbook.1.73.1" target="_top">Germline chromatin</a>), so here we focus on somatic functions of these chromatin regulatory complexes.
            </p>
            <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="transregulate_fig6.jpg"><img src="transregulate_fig6_s.jpg" border="2" align="middle" alt=" figure 6"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b><span class="bold"><strong>Figure 6:  Examples of ChIP-chip data for RNA polymerase II (Pol II) and five different histone modifications.</strong></span>&nbsp;</b>The X axis represents a stretch of Chr IV from nt 12,341,610 to 12,472,625.  Coding genes are shown below, with arrows marking
                     the direction of transcription 5'&#8594;3'.  The Y axis represents the z-score of the log 2 ratios of IP/Input (mean centered and
                     scaled to stdev=1).  Note the opposing pattern of H3K27me, a mark associated with gene silencing, with that of activation
                     marks such as H3K4me and H3K36me.  Image courtesy of Susan Strome and Andreas Rechsteiner.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6-1"></a>6.1.&nbsp;The NuRD complex
                        </h3>
                     </div>
                  </div>
               </div>
               <p>NuRD (<span class="bold"><strong>Nu</strong></span>cleosome <span class="bold"><strong>R</strong></span>emodeling and <span class="bold"><strong>D</strong></span>eacetylase) is an evolutionarily conserved complex correlated with transcriptional repression that is required for proper
                  development in mammals (reviewed in <a href="#bib87">Ho and Crabtree, 2010</a>). The NuRD complex primarily targets lysine 9 of histone H3 for deacetylation, paving the way for a histone methyltransferase
                  (HMT) to methylate the residue. Methylated lysine 9 is then bound by HP1 (<span class="bold"><strong>h</strong></span>eterochromatin <span class="bold"><strong>p</strong></span>rotein 1), which induces gene silencing (<a href="#bib20">Brehm et al., 1998</a>). In <span class="emphasis"><em>C. elegans</em></span>, NuRD consists of <a href="http://www.wormbase.org/db/get?name=HDA-1;class=Protein" target="_blank">HDA-1</a> (histone deacetylase), <a href="http://www.wormbase.org/db/get?name=LIN-53;class=Protein" target="_blank">LIN-53</a> (RbAp48), <a href="http://www.wormbase.org/db/get?name=LIN-40;class=Protein" target="_blank">LIN-40</a> (MTA), and one of two Mi2 subunits, either <a href="http://www.wormbase.org/db/get?name=LET-418;class=Protein" target="_blank">LET-418</a> (with <a href="http://www.wormbase.org/db/get?name=MEP-1;class=Protein" target="_blank">MEP-1</a>) or <a href="http://www.wormbase.org/db/get?name=CHD-3;class=Protein" target="_blank">CHD-3</a>. The combinatorial nature of NuRD complex components allows both target gene and tissue specificity for repression. In vulval
                  development both complexes act redundantly (<a href="#bib237">von Zelewsky et al., 2000</a>), whereas the complex containing <a href="http://www.wormbase.org/db/get?name=LET-418;class=Protein" target="_blank">LET-418</a> and <a href="http://www.wormbase.org/db/get?name=MEP-1;class=Protein" target="_blank">MEP-1</a> also acts to suppress the germline fate in somatic tissues (<a href="#bib176">Passannante et al., 2010</a>; <a href="#bib229">Unhavaithaya et al., 2002</a>). The histone methyltransferases that likely methylate the deacetylated lysine 9 residue are <a href="http://www.wormbase.org/db/get?name=MET-1;class=Protein" target="_blank">MET-1</a> and/or <a href="http://www.wormbase.org/db/get?name=MET-2;class=Protein" target="_blank">MET-2</a> (<a href="#bib4">Andersen and Horvitz, 2007</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6-2"></a>6.2.&nbsp;The MLL/COMPASS complex
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The MLL/COMPASS (<span class="bold"><strong>M</strong></span>ixed <span class="bold"><strong>L</strong></span>ineage <span class="bold"><strong>L</strong></span>eukemia/<span class="bold"><strong>Com</strong></span>plex <span class="bold"><strong>P</strong></span>roteins <span class="bold"><strong>A</strong></span>ssociated with <span class="bold"><strong>S</strong></span>et1) complex is responsible for the methylation of lysine 4 of histone H3 (reviewed in <a href="#bib205">Shilatifard, 2008</a>). In contrast to lysine 9, methylation of lysine 4 is correlated with active gene expression. This complex generally contains
                  seven subunits, all of which are present in <span class="emphasis"><em>C. elegans</em></span>, including two MLL-like histone methyltransferases (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=set-2;class=Gene" target="_blank">set-2</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=set-16;class=Gene" target="_blank">set-16</a></em></span>), <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00012458?query=ash-2#0-9e-3" target="_blank">ash-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00017683?query=rbbp-5#0-9e-3" target="_blank">rbbp-5</a></em></span>, three WDR5-like proteins (<span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00006474?query=wdr-5.1#0-9e-3" target="_blank">wdr-5.1</a>, <a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00010572#0-9e-3" target="_blank">.2</a>,</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00013862#0-9e-3" target="_blank">.3</a></em></span>), <span class="emphasis"><em>cfp-1</em></span>, and either <span class="emphasis"><em>wdr-82</em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=dpy-30;class=Gene" target="_blank">dpy-30</a></em></span> (<a href="#bib130">Li and Kelly, 2011</a>). Various components have been implicated in dosage compensation (<a href="#bib179">Pferdehirt et al., 2011</a>), vulval development (<a href="#bib58">Fisher et al., 2010</a>), neuronal development (<a href="#bib182">Poole et al., 2011</a>), and aging (<a href="#bib73">Greer et al., 2010</a>). This complex potentially functions antagonistically to NuRD/HP1 in certain aspects, such as growth and somatic gonad development
                  (<a href="#bib211">Simonet et al., 2007</a>). Whether NuRD/HP1 and MLL/COMPASS target the same genes or distinct genes in these various developmental processes is currently
                  unknown.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6-3"></a>6.3.&nbsp;The Polycomb Group complex
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The Polycomb Group (PcG) of chromatin regulators were first uncovered in the classic studies of Edward B. Lewis in <span class="emphasis"><em>Drosophila</em></span> because of their critical roles in maintaining the repressed state of homeotic (Hox) genes regulating segmentation (reviewed
                  in <a href="#bib160">Muller and Kassis, 2006</a>). Subsequent work by numerous groups over the years revealed that the PcG includes two types of complexes, PRC1 and PRC2.
                  A major role for mammalian PRC2 is repression of Hox gene expression during development by promoting histone H3 lysine 27
                  methylation, a mark that is then bound by the silencing complex PRC1, which ubiquitylates histone H2A. The mechanism by which
                  this modification leads to Hox gene silencing is not clear. In <span class="emphasis"><em>C. elegans</em></span>, the PcG-related components were first identified in maternal effect sterile (MES) screens due to their role in the germline
                  (<a href="#bib26">Capowski et al., 1991</a>), a topic explored in detail in the WormBook chapter <a href="http://dx.doi.org/10.1895/wormbook.1.9.1" target="_blank">Specification of the germ line</a>. In the <span class="emphasis"><em>C. elegans</em></span> soma, at least some PcG-related proteins of the PRC2 complex (<a href="http://www.wormbase.org/db/get?name=MES-2;class=Protein" target="_blank">MES-2</a> and <a href="http://www.wormbase.org/db/get?name=MES-6;class=Protein" target="_blank">MES-6</a>) appear to have a role in regulating Hox gene expression, as is observed in other animals (<a href="#bib45">Deng et al., 2007</a>; <a href="#bib189">Ross and Zarkower, 2003</a>). Moreover, <a href="http://www.wormbase.org/db/get?name=MES-2;class=Protein" target="_blank">MES-2</a> is required for restricting the developmental plasticity of embryos through global changes to the chromatin state (<a href="#bib259">Yuzyuk et al., 2009</a>). Until recently, it was less clear whether <span class="emphasis"><em>C. elegans</em></span> contained a PRC1-like complex. However, certain PRC1-related components are present including <a href="http://www.wormbase.org/db/get?name=MIG-32;class=Protein" target="_blank">MIG-32</a>/Bmi-1 and <a href="http://www.wormbase.org/db/get?name=SPAT-3;class=Protein" target="_blank">SPAT-3</a>/Ring1B, which were shown to be required for H2A ubiquitylation in the soma (<a href="#bib103">Karakuzu et al., 2009</a>). Moreover, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mig-32;class=Gene" target="_blank">mig-32</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=spat-3;class=Gene" target="_blank">spat-3</a></em></span> mutants have somatic defects similar to <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mes-2;class=Gene" target="_blank">mes-2</a></em></span>, suggesting that they act to regulate the same genes. Intriguingly, however, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mig-32;class=Gene" target="_blank">mig-32</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=spat-3;class=Gene" target="_blank">spat-3</a></em></span> do not share the germline defects of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mes-2;class=Gene" target="_blank">mes-2</a></em></span>, suggesting that the mechanism of PRC2 activity in the germline is distinct from that of the soma (<a href="#bib103">Karakuzu et al., 2009</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6-4"></a>6.4.&nbsp;The SynMuv pathway
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Genetic studies in <span class="emphasis"><em>C. elegans</em></span> have also uncovered a series of intertwined, genetically-linked pathways involving chromatin modifications that affect developmental
                  pathways called the SynMuv pathway (reviewed in <a href="#bib56">Fay and Yochem, 2007</a>). Consisting of at least three genetically defined groups (A, B, and C), the SynMuv groups reveal the functional redundancy
                  underlying gene regulation (<a href="#figure7">Figure 7</a>). Under standard conditions, genes from at least two of the SynMuv groups must be disrupted by mutation before significant
                  effects in development occur, typically and historically read out as defects in vulva formation in the hermaphrodite. All
                  three groups contain genes that encode gene regulatory proteins, although the SynMuv A group is unique to <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib40">Davison et al., 2011</a>). SynMuv C genes encode proteins of the TipA/HAT regulatory complex that is associated with H3K4 acetylation and active gene
                  expression (<a href="#bib28">Ceol and Horvitz, 2004</a>). The best-studied pathway, SynMuv B, includes members of the DRM complex (<a href="#bib82">Harrison et al., 2006</a>), which is related to a repressor complex called dREAM in <span class="emphasis"><em>Drosophila</em></span> (<a href="#bib113">Korenjak et al., 2004</a>) and DREAM/Myb-MuvB/LINC in mammals (<a href="#bib110">Knight et al., 2009</a>). In <span class="emphasis"><em>C. elegans</em></span>, DRM includes the sequence-specific binding factor E2F called <a href="http://www.wormbase.org/db/get?name=EFL-1;class=Protein" target="_blank">EFL-1</a>, a pocket protein called <a href="http://www.wormbase.org/db/get?name=LIN-35;class=Protein" target="_blank">LIN-35</a> and other factors with less well-understood functions, but it does not contain a Myb-like protein as in other organisms (<a href="#bib82">Harrison et al., 2006</a>). DRM potentially acts with the deacetylase NuRD complex to promote transcriptional repression, and indeed microarray studies
                  of mutants in the SynMuv B group primarily show increased abundance of target transcripts (<a href="#bib108">Kirienko and Fay, 2007</a>). DRM components are widely expressed and appear to play diverse roles in many tissues, although in most cases the function
                  of the DRM complex is not solely essential for normal development. However, a second mutation that disrupts a tissue-specific
                  regulatory protein along with a mutation in a DRM complex member can cause a tissue-specific phenotype. For instance, mutation
                  of the C2H2 zinc finger gene <span class="emphasis"><em>slr-2</em></span> in conjunction with <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-35;class=Gene" target="_blank">lin-35</a></em></span> mutations disrupts intestinal function, while either mutation alone does not (<a href="#bib109">Kirienko et al., 2008</a>). Additionally, a recent report indicates that one SynMuvB component, <a href="http://www.wormbase.org/db/get?name=LIN-61;class=Protein" target="_blank">LIN-61</a>, which binds methylated lysine 9 of H3 (<a href="#bib111">Koester-Eiserfunke and Fischle, 2011</a>), can act as part of the DRM complex in vulval development but not in other processes, showing that this complex has tissue-specificity
                  (<a href="#bib83">Harrison et al., 2007</a>). Finally, environmental influences such as temperature can have an effect as well. <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-35;class=Gene" target="_blank">lin-35</a></em></span>, and certain other components of the DRM complex, are required to suppress the germline fate in somatic tissues (<a href="#bib243">Wang et al., 2005</a>), but only at higher temperatures is the onset of the germline fate sufficiently severe to lead to a larval arrest (<a href="#bib178">Petrella et al., 2011</a>).
               </p>
               <div class="mediaobject" align="center"><a name="figure7"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="transregulate_fig7.jpg"><img src="transregulate_fig7_s.jpg" border="2" align="middle" alt=" figure 7"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b><span class="bold"><strong>Figure 7: Outline of genes in each group of the SynMuv pathway.</strong></span>&nbsp;</b>SynMuv A and SynMuv B are the major groups in this pathway that are redundantly required for vulval development.  The relationship
                        of these pathways with the SynMuv C pathway are less clear.  Within the SynMuv B group, <a href="http://www.wormbase.org/db/get?name=LIN-53;class=Protein" target="_blank">LIN-53</a>, marked with an asterisk, is listed twice, as it is found in both NuRD and DRM complexes.
                     </p>
                  </div>
               </div>
               <p>As in other systems, the landscape of chromatin modifications and the complexes carrying them out are generally conserved
                  in <span class="emphasis"><em>C. elegans</em></span>. Thus, it is likely that information from any one experimental system will inform our general understanding of chromatin
                  influences in all systems. In the future investigating how these different chromatin regulatory complexes interact at common
                  target loci, and how they influence key sequence-specific transcription factors and the core transcriptional machinery, will
                  be critical for understanding gene regulatory mechanisms. The molecular and genetic advantages of <span class="emphasis"><em>C. elegans</em></span>, combined with a relatively simple and defined cell lineage, suggest that the worm will prove particularly important for
                  understanding the role of chromatin in developmental processes.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec7"></a>7.&nbsp;Systematic genome-scale analysis of transcription regulation
                     </h2>
                  </div>
               </div>
            </div>
            <p>Genome-wide analyses complement detailed single-gene studies by providing a global overview that can be used to determine
               how broadly observations of transcriptional regulatory mechanisms at individual genes are applicable. Over the last decade
               microarray analysis of gene expression has been widely used to examine gene expression changes upon perturbation of various
               transcriptional components. However, one limitation to this type of analysis is the inability to distinguish direct from indirect
               effects. The more recent development of techniques such as chromatin immunoprecipitation (ChIP), which maps the binding events
               of a given factor throughout the genome, can overcome this limitation and provide important information about the direct activity
               of the factor. Chromatin fragments that immunoprecipitate with chromatin or transcriptional regulatory proteins are identified
               by either hybridization to a microarray (ChIP-chip) or by deep sequencing (ChIP-seq). This approach has proved to be a very
               powerful tool for investigating and discovering transcriptional mechanisms. In <span class="emphasis"><em>C. elegans</em></span>, ChIP studies have been performed by individual labs focused on particular processes or factors including dosage compensation
               components (<a href="#bib53">Ercan et al., 2007</a>; <a href="#bib95">Jans et al., 2009</a>), the DRM complex component <a href="http://www.wormbase.org/db/get?name=LIN-54;class=Protein" target="_blank">LIN-54</a> (<a href="#bib223">Tabuchi et al., 2011</a>), the histone variant <a href="http://www.wormbase.org/db/get?name=HTZ-1;class=Protein" target="_blank">HTZ-1</a> (<a href="#bib249">Whittle et al., 2008</a>), and transcription factors such as <a href="http://www.wormbase.org/db/get?name=HLH-1;class=Protein" target="_blank">HLH-1</a> and <a href="http://www.wormbase.org/db/get?name=NFI-1;class=Protein" target="_blank">NFI-1</a> (<a href="#bib127">Lei et al., 2010</a>; <a href="#bib248">Whittle et al., 2009</a>).
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7-1"></a>7.1.&nbsp;The modENCODE project
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In addition to individual efforts, a large-scale effort by a multi-lab consortium has systematically utilized a genomics approach
                  to exploring <span class="emphasis"><em>C. elegans</em></span> gene expression. The modENCODE consortium (<span class="bold"><strong>mod</strong></span>el organism <span class="bold"><strong>Enc</strong></span>yclopedia <span class="bold"><strong>o</strong></span>f <span class="bold"><strong>D</strong></span>NA <span class="bold"><strong>E</strong></span>lements), funded by the National Human Genome Research Institute (NHGRI), has in the last few years produced a wealth of genome-wide
                  <span class="emphasis"><em>C. elegans</em></span> datasets. These studies explore many aspects of transcriptional regulation such as transcription factor binding sites, chromatin
                  modifications, and gene expression analysis of diverse RNAs, including small noncoding RNAs in addition to polyadenylated
                  RNAs (<a href="#bib68">Gerstein et al., 2010</a>). A similar effort to analyze the <span class="emphasis"><em>Drosophila melanogaster</em></span> genome is ongoing as well (<a href="#bib194">Roy et al., 2010</a>). The ultimate goal of these projects is to identify, as comprehensively as possible, all of the functional elements encoded
                  in the DNA that are responsible for the regulation and formation of that organism.
               </p>
               <p>A major effort of modENCODE is to determine the binding sites of sequence-specific transcription factors genome wide. These
                  elements direct the temporal and spatial control of transcription, which in turn dictates an organism's development, physiology
                  and response to the environment. Identifying these elements provides an important first step toward understanding how DNA
                  sequence is interpreted to form a three-dimensional body plan. At the beginning of the modENCODE project, almost nothing was
                  known about the direct targets of transcription factors in <span class="emphasis"><em>C. elegans</em></span>. By the time the project completed its fifth year in March 2012, the genome-wide binding profiles of over 120 factors from
                  diverse families of transcriptional regulators were collected and released to the <span class="emphasis"><em>C. elegans</em></span> community.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7-2"></a>7.2.&nbsp;modENCODE transcription factor ChIP studies
                        </h3>
                     </div>
                  </div>
               </div>
               <p>To systematically identify binding sites, transgenic lines expressing GFP-tagged transcription factors were subjected to ChIP-seq
                  using an antibody to GFP (<a href="#bib261">Zhong et al., 2010</a>). The GFP expression patterns of these lines largely recapitulated known endogenous expression patterns, and all lines that
                  were tested robustly rescued the mutant phenotype of that gene, indicating that the tagged transcription factors retain wild-type
                  function. All of the detailed protocols and datasets are freely available at <a href="http://www.modencode.org/" target="_top">http://www.modencode.org/</a>. Moreover, all the strains for which successful datasets have been produced are available in the Caenorhabditis Genetics
                  Center (<a href="http://www.cbs.umn.edu/cgc" target="_top">CGC</a>). So far, 77 completed datasets are available representing 46 transcription factors, some of which have been analyzed at
                  multiple developmental stages (<a href="#table4" title="Table 4. Expression of transcription factors.">Table 4</a>).
               </p>
               <div class="table"><a name="table4"></a><p class="title"><span class="bold">Table 4. Expression of transcription factors.</span></p>
                  <table summary="Table 4. Expression of transcription factors." border="1">
                     <colgroup>
                        <col>
                        <col>
                        <col>
                        <col>
                        <col>
                        <col>
                     </colgroup>
                     <tbody>
                        <tr valign="bottom">
                           <td align="left" valign="bottom">TF genes</td>
                           <td align="left" valign="bottom">stage(s)</td>
                           <td align="left" valign="bottom">Histone modifications</td>
                           <td align="left" valign="bottom">stage(s)</td>
                           <td align="left" valign="bottom">Chromatin proteins</td>
                           <td align="left" valign="bottom">stage(s)</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=alr-1;class=Gene" target="_blank">alr-1</a></em></span></td>
                           <td align="left" valign="top">L2</td>
                           <td align="left" valign="top">total H3</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=HPL-2;class=Protein" target="_blank">HPL-2</a></td>
                           <td align="left" valign="top">LE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=aly-2;class=Gene" target="_blank">aly-2</a></em></span></td>
                           <td align="left" valign="top">L1, L3</td>
                           <td align="left" valign="top">H3K27ac</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td align="left" valign="top">LIN-15B</td>
                           <td align="left" valign="top">LE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ama-1;class=Gene" target="_blank">ama-1</a></em></span></td>
                           <td align="left" valign="top">EE, LE, MxE, L1, starved L1, L2, L3, L4, L4/YA, YA</td>
                           <td align="left" valign="top">H3K27me1</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td align="left" valign="top">pol2</td>
                           <td align="left" valign="top">EE, LE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=blmp-1;class=Gene" target="_blank">blmp-1</a></em></span></td>
                           <td align="left" valign="top">L1</td>
                           <td align="left" valign="top">H3K27me3</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=ama-1;class=Gene" target="_blank">ama-1</a></td>
                           <td align="left" valign="top">MxE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=C01B12.2;class=Gene" target="_blank">C01B12.2</a></em></span></td>
                           <td align="left" valign="top">L2</td>
                           <td align="left" valign="top">H3K36me1</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=HCP-3;class=Protein" target="_blank">HCP-3</a></td>
                           <td align="left" valign="top">EE, MxE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ceh-14;class=Gene" target="_blank">ceh-14</a></em></span></td>
                           <td align="left" valign="top">L2</td>
                           <td align="left" valign="top">H3K36me2</td>
                           <td align="left" valign="top">EE</td>
                           <td align="left" valign="top">CBP</td>
                           <td align="left" valign="top">MxE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ceh-26;class=Gene" target="_blank">ceh-26</a></em></span></td>
                           <td align="left" valign="top">LE</td>
                           <td align="left" valign="top">H3K36me3</td>
                           <td align="left" valign="top">EE, LE, L1, L2, L3, L4, YA</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=HTZ-1;class=Protein" target="_blank">HTZ-1</a></td>
                           <td align="left" valign="top">MxE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ceh-30;class=Gene" target="_blank">ceh-30</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td align="left" valign="top">H3K4me1</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=DPY-26;class=Protein" target="_blank">DPY-26</a></td>
                           <td align="left" valign="top">MxE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ces-1;class=Gene" target="_blank">ces-1</a></em></span></td>
                           <td align="left" valign="top">L1, L4</td>
                           <td align="left" valign="top">H3K4me2</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=DPY-27;class=Protein" target="_blank">DPY-27</a></td>
                           <td align="left" valign="top">MxE, EE, L4</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=daf-12;class=Gene" target="_blank">daf-12</a></em></span></td>
                           <td align="left" valign="top">L3, L4</td>
                           <td align="left" valign="top">H3K4me3</td>
                           <td align="left" valign="top">EE, LE, L1, L2, L3, L4, YA</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=DPY-28;class=Protein" target="_blank">DPY-28</a></td>
                           <td align="left" valign="top">MxE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=daf-16;class=Gene" target="_blank">daf-16</a></em></span></td>
                           <td align="left" valign="top">L4</td>
                           <td align="left" valign="top">H3K79me1</td>
                           <td align="left" valign="top">MxE, L3</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=LEM-2;class=Protein" target="_blank">LEM-2</a></td>
                           <td align="left" valign="top">MxE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=dpl-1;class=Gene" target="_blank">dpl-1</a></em></span></td>
                           <td align="left" valign="top">L1, YA</td>
                           <td align="left" valign="top">H3K79me2</td>
                           <td align="left" valign="top">EE, MxE, L3</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=MES-4;class=Protein" target="_blank">MES-4</a></td>
                           <td align="left" valign="top">EE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=dpy-27;class=Gene" target="_blank">dpy-27</a></em></span></td>
                           <td align="left" valign="top">MxE</td>
                           <td align="left" valign="top">H3K79me3</td>
                           <td align="left" valign="top">EE, MxE, L3</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=MIX-1;class=Protein" target="_blank">MIX-1</a></td>
                           <td align="left" valign="top">MxE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=efl-1;class=Gene" target="_blank">efl-1</a></em></span></td>
                           <td align="left" valign="top">L1, YA</td>
                           <td align="left" valign="top">H3K9me1</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=MRG-1;class=Protein" target="_blank">MRG-1</a></td>
                           <td align="left" valign="top">EE</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-27;class=Gene" target="_blank">egl-27</a></em></span></td>
                           <td align="left" valign="top">L1</td>
                           <td align="left" valign="top">H3K9me2</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-5;class=Gene" target="_blank">egl-5</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td align="left" valign="top">H3K9me3</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=elt-3;class=Gene" target="_blank">elt-3</a></em></span></td>
                           <td align="left" valign="top">L1</td>
                           <td align="left" valign="top">total H4</td>
                           <td align="left" valign="top">L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=eor-1;class=Gene" target="_blank">eor-1</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td align="left" valign="top">H4acTetra</td>
                           <td align="left" valign="top">EE</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=F16B12.6;class=Gene" target="_blank">F16B12.6</a></em></span></td>
                           <td align="left" valign="top">L1</td>
                           <td align="left" valign="top">H4K16ac</td>
                           <td align="left" valign="top">EE</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=F45C12.2;class=Gene" target="_blank">F45C12.2</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td align="left" valign="top">H4K20me1</td>
                           <td align="left" valign="top">EE, LE, L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=fos-1;class=Gene" target="_blank">fos-1</a></em></span></td>
                           <td align="left" valign="top">L1, L2</td>
                           <td align="left" valign="top">H4K8ac</td>
                           <td align="left" valign="top">EE, L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=gei-11;class=Gene" target="_blank">gei-11</a></em></span></td>
                           <td align="left" valign="top">L2, L4</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hlh-1;class=Gene" target="_blank">hlh-1</a></em></span></td>
                           <td align="left" valign="top">MxE</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hlh-8;class=Gene" target="_blank">hlh-8</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-11;class=Gene" target="_blank">lin-11</a></em></span></td>
                           <td align="left" valign="top">L2</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-13;class=Gene" target="_blank">lin-13</a></em></span></td>
                           <td align="left" valign="top">MxE</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-15B;class=Gene" target="_blank">lin-15B</a></em></span></td>
                           <td align="left" valign="top">L3, L4</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-39;class=Gene" target="_blank">lin-39</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mab-5;class=Gene" target="_blank">mab-5</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mdl-1;class=Gene" target="_blank">mdl-1</a></em></span></td>
                           <td align="left" valign="top">L1</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mef-2;class=Gene" target="_blank">mef-2</a></em></span></td>
                           <td align="left" valign="top">L1</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mep-1;class=Gene" target="_blank">mep-1</a></em></span></td>
                           <td align="left" valign="top">MxE</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=nhr-105;class=Gene" target="_blank">nhr-105</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=nhr-111;class=Gene" target="_blank">nhr-111</a></em></span></td>
                           <td align="left" valign="top">L2</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=nhr-28;class=Gene" target="_blank">nhr-28</a></em></span></td>
                           <td align="left" valign="top">L3, L4</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=nhr-6;class=Gene" target="_blank">nhr-6</a></em></span></td>
                           <td align="left" valign="top">L2</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=pes-1;class=Gene" target="_blank">pes-1</a></em></span></td>
                           <td align="left" valign="top">L4</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=pha-4;class=Gene" target="_blank">pha-4</a></em></span></td>
                           <td align="left" valign="top">MxE, LE, L1, starved L1, L2, YA</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td class="auto-generated">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=pqm-1;class=Gene" target="_blank">pqm-1</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=R02D3.7;class=Gene" target="_blank">R02D3.7</a></em></span></td>
                           <td align="left" valign="top">L2</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sea-2;class=Gene" target="_blank">sea-2</a></em></span></td>
                           <td align="left" valign="top">L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=skn-1;class=Gene" target="_blank">skn-1</a></em></span></td>
                           <td align="left" valign="top">L1</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-130;class=Gene" target="_blank">unc-130</a></em></span></td>
                           <td align="left" valign="top">L1</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-62;class=Gene" target="_blank">unc-62</a></em></span></td>
                           <td align="left" valign="top">L1, L2, L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/species/c_elegans/gene/WBGene00021000#0-9e-3" target="_blank">W03F9.2</a></em></span></td>
                           <td align="left" valign="top">L4/YA</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=zag-1;class=Gene" target="_blank">zag-1</a></em></span></td>
                           <td align="left" valign="top">L1, L2, L3</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                           <td valign="top">&nbsp;</td>
                        </tr>
                        <tr valign="top">
                           <td colspan="6" align="left" valign="top">EE = early embryo, LE= late embryo, MxE= mixed stage embryos, L1-L4 = larval stages 1-4, YA = young adult</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
               <p>This pipeline was first utilized on the FoxA transcription factor <a href="http://www.wormbase.org/db/get?name=PHA-4;class=Protein" target="_blank">PHA-4</a> that has important roles in both organ development and environmental responses (<a href="#bib261">Zhong et al., 2010</a>). Subsequently, an analysis of the major characteristics of 22 transcription factors describes binding site features and
                  correlations for this larger set, including a preliminary regulatory network analysis (<a href="#bib166">Niu et al., 2011</a>). Most of these factors bind thousands of sites in the genome, and the majority of these binding sites near coding genes
                  lie within 500 bp of the predicted transcript start site. A significant insight from the properties of these 22 transcription
                  factors was the recognition that the genome contains hundreds of regions that are broadly permissive for non-specific binding
                  by transcription factors (termed high-occupancy target, or &#8220;HOT&#8221;, regions) (<a href="#bib68">Gerstein et al., 2010</a>). Recruitment to a HOT region does not require the sequence-specific binding property of the transcription factor, nor is
                  binding correlated with the regulation of nearby genes. How transcription factors are recruited to HOT sites, and the possible
                  role of chromatin or nuclear organization in this process, is unknown. For those interested in using these data to understand
                  gene regulation by a specific transcription factor, HOT sites should be carefully distinguished from other binding sites that
                  are either unique to or primarily bound by the factor of interest, which are more likely to result in direct regulatory events.
                  Another limitation users should realize when interpreting the genome-wide transcription factor binding profiles is that almost
                  all the experiments were performed in whole animals. Any binding profile for a broadly-expressed transcription factor will
                  therefore be an amalgam of binding in multiple tissues. Currently, tissue-specific profiling techniques are being applied
                  to circumvent this problem.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7-3"></a>7.3.&nbsp;modENCODE chromatin modification ChIP studies
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In addition to sequence-specific transcription factors, a second modENCODE project has focused on collecting genome-wide ChIP-chip
                  datasets for various chromatin-associated factors as well as for histone modifications. Over 20 histone modifications and
                  14 factors have been analyzed to date. Unlike the transcription factor studies that analyzed transgenic GFP-tagged proteins
                  these datasets were generated by using antibodies specific to each factor or modification to monitor their endogenous distribution;
                  many of these antibodies are commercially available. This global analysis of chromatin states has yielded various insights,
                  including the persistence of the germline-established chromatin state in the somatic tissues, highlighted by the existence
                  of chromosomal domains enriched for repressive histone modifications that correlate with increased meiotic recombination rates
                  (<a href="#bib68">Gerstein et al., 2010</a>). Additionally, the X chromosome exhibits several distinctive features relative to autosomes, including increased monomethylation
                  of H4K20 and H3K27, and increased nucleosome density (<a href="#bib54">Ercan et al., 2011</a>; <a href="#bib135">Liu et al., 2011</a>). Importantly, when the chromatin modification status was combined with transcription factor data in predictive algorithms
                  for gene regulatory events, the accuracy of the resulting predictions was greatly improved compared to either alone (<a href="#bib68">Gerstein et al., 2010</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7-4"></a>7.4.&nbsp;modENCODE transcription studies
                        </h3>
                     </div>
                  </div>
               </div>
               <p>All of these ChIP-based studies are complemented by modENCODE-based analysis of gene expression at many different developmental
                  stages, tissues, and environmental conditions through the use of deep sequencing and tiling microarrays to monitor transcript
                  identity and abundance. As of October 2011, more than 130 experiments have been analyzed and released to the community. These
                  include analysis of both poly-A-selected RNAs as well as small RNAs. Most of these data have been primarily analyzed with
                  the immediate goal of improving gene annotation, and have led to the identification of thousands of novel exons and splicing
                  events, new small RNAs, including many microRNAs, and improved 5&#8217; and 3&#8217; UTR mapping (<a href="#bib85">Hillier et al., 2009</a>; <a href="#bib106">Kato et al., 2009</a>; <a href="#bib145">Mangone et al., 2010</a>; see also <a href="#bib94">Jan et al., 2011</a>). Improved gene models lead to improved assignment of regulatory events to the correct target genes. Moreover, as additional
                  gene expression data is collected and analyzed, more conclusions will be drawn regarding correlations between regulatory factors
                  and changes in gene expression levels across stages, tissues, and conditions.
               </p>
               <p>Countless biological discoveries are embedded in these deep and complex modENCODE datasets. The published global observations
                  and analyses (<a href="#bib68">Gerstein et al., 2010</a>) are just the beginning. Investigation of the data by the larger research community, with their specialized expertise in
                  so many aspects of <span class="emphasis"><em>C. elegans</em></span> biology, is essential to plumb the full possibilities of the data. To facilitate such endeavors by the <span class="emphasis"><em>C. elegans</em></span> community, all modENCODE data, along with detailed descriptions of growth and collection conditions and protocols, are available
                  at <a href="http://www.modencode.org/" target="_top">http://www.modencode.org/</a>. The data are available for bulk download for large-scale analyses and comparisons, but the data for individual experiments
                  or individual genes can be examined as well, for those with a particular focus on a single pathway or process. Data of interest
                  can be selected as &#8220;tracks&#8221; for viewing on a genome browser as well. In the near future, additional changes will be made to
                  the interface to improve selection and analysis of all interested users. Ultimately, all the data on the modENCODE website
                  will be incorporated into <a href="http://www.wormbase.org/" target="_blank">WormBase</a>. Movement of these data to cloud-based storage to increase accessibility and facilitate downloads is also a likely possibility
                  in the near future.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec8"></a>8.&nbsp;Future Prospects
                     </h2>
                  </div>
               </div>
            </div>
            <p>There is little doubt that the field of transcriptional regulation in <span class="emphasis"><em>C. elegans</em></span> is in the midst of an information explosion. We are rapidly acquiring information concerning temporal and spatial patterns
               of gene expression using genome wide expression assays and automated analyses of fluorescent protein reporter expression.
               At the same time, we are identifying binding sites for transcription factors and chromatin regulatory factors throughout the
               genome, and recently developed techniques for isolating specific nuclei will enhance our ability for tissue specific chromatin
               profiling (<a href="#bib43">Deal and Henikoff, 2010</a>). Still, we have only begun to explore other areas of transcriptional regulation. How does higher order organization within
               nuclei affect gene expression (<a href="#bib153">Meister et al., 2010</a>; <a href="#bib93">Ikegami et al., 2010</a>)? What impact do post-transcriptional and post-translational mechanisms have on transcription factor activity? Overall, it
               is an exciting time to study transcriptional regulation in <span class="emphasis"><em>C. elegans</em></span>. Because of the relative simplicity of <span class="emphasis"><em>C. elegans</em></span> gene promoters, we can reasonably make connections between transcription factors and their target genes. Understanding this
               information will help decipher how the information in the genome controls every aspect of <span class="emphasis"><em>C. elegans</em></span> biology.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec9"></a>9.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>This work was supported, in part, by the NIDDK Intramural Research Program of the National Institutes of Health (NIH) (M.
               Krause), and extramural NIH support from the NHGRI (V. Reinke), and the NIGMS (P. Okkema).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec10"></a>10.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed"><span class="bibliomisc">Allen, M.A., Hillier, L.W., Waterston, R.H., and Blumenthal, T. (2011). A global analysis of <span class="emphasis"><em>C. elegans</em></span> trans-splicing. Genome Res. 21, 255-264.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177958&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.113811.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed"><span class="bibliomisc">Amin, N.M., Shi, H., and Liu, J. (2010). The FoxF/FoxC factor <a href="http://www.wormbase.org/db/get?name=LET-381;class=Protein" target="_blank">LET-381</a> directly regulates both cell fate specification and cell differentiation in <span class="emphasis"><em>C. elegans</em></span> mesoderm development. Development <span class="emphasis"><em>137</em></span>, 1451-1460.</span>
                     <a href="http://dx.doi.org/10.1242/dev.048496" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed"><span class="bibliomisc">An, J.H., and Blackwell, T.K. (2003). <a href="http://www.wormbase.org/db/get?name=SKN-1;class=Protein" target="_blank">SKN-1</a> links <span class="emphasis"><em>C. elegans</em></span> mesendodermal specification to a conserved oxidative stress response. Genes Dev. <span class="emphasis"><em>17</em></span>, 1882-1893.</span>
                     <a href="http://dx.doi.org/10.1101/gad.1107803" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed"><span class="bibliomisc">Andersen, E.C., and Horvitz, H.R. (2007). Two <span class="emphasis"><em>C. elegans</em></span> histone methyltransferases repress <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-3;class=Gene" target="_blank">lin-3</a></em></span> EGF transcription to inhibit vulval development. Development <span class="emphasis"><em>134</em></span>, 2991-2999.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17634190&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.009373" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed"><span class="bibliomisc">Ao, W., Gaudet, J., Kent, W.J., Muttumu, S., and Mango, S.E. (2004). Environmentally induced foregut remodeling by <a href="http://www.wormbase.org/db/get?name=PHA-4;class=Protein" target="_blank">PHA-4</a>/FoxA and <a href="http://www.wormbase.org/db/get?name=DAF-12;class=Protein" target="_blank">DAF-12</a>/NHR. Science <span class="emphasis"><em>305</em></span>, 1743-1746.</span>
                     <a href="http://dx.doi.org/10.1126/science.1102216" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed"><span class="bibliomisc">Arata, Y., Kouike, H., Zhang, Y., Herman, M.A., Okano, H., and Sawa, H. (2006). Wnt signaling and a Hox protein cooperatively
                        regulate <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=psa-3;class=Gene" target="_blank">psa-3</a></em></span>/Meis to determine daughter cell fate after asymmetric cell division in <span class="emphasis"><em>C. elegans</em></span>. Dev. Cell <span class="emphasis"><em>11</em></span>, 105-115.</span>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2006.04.020" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed"><span class="bibliomisc">Balakrishnan, L., and Milavetz, B. (2010). Decoding the histone H4 lysine 20 methylation mark. Crit. Rev. Biochem. Mol. Biol.
                        <span class="emphasis"><em>45</em></span>, 440-452.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20735237&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.3109/10409238.2010.504700" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed"><span class="bibliomisc">Bannister A.J., and Kouzarides, T. (2011).  Regulation of chromatin by histone modifications.  Cell Res. <span class="emphasis"><em>21</em></span>, 381-395.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21321607&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/cr.2011.22" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed"><span class="bibliomisc">Barrasa, M.I., Vaglio, P., Cavasino, F., Jacotot, L., and Walhout, A.J. (2007). EDGEdb: a transcription factor-DNA interaction
                        database for the analysis of <span class="emphasis"><em>C. elegans</em></span> differential gene expression. BMC Genomics <span class="emphasis"><em>8</em></span>, 21.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17233892&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-8-21" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed"><span class="bibliomisc">Baugh, L.R., Demodena, J., and Sternberg, P.W. (2009). RNA Pol II accumulates at promoters of growth genes during developmental
                        arrest. Science <span class="emphasis"><em>324</em></span>, 92-94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19251593&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1169628" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed"><span class="bibliomisc">Beaster-Jones, L., and Okkema, P.G. (2004). DNA binding and <span class="emphasis"><em>in vivo</em></span> function of <span class="emphasis"><em>C.elegans</em></span> <a href="http://www.wormbase.org/db/get?name=PEB-1;class=Protein" target="_blank">PEB-1</a> require a conserved FLYWCH motif. J. Mol. Biol. <span class="emphasis"><em>339</em></span>, 695-706.</span>
                     <a href="http://dx.doi.org/10.1016/j.jmb.2004.04.030" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed"><span class="bibliomisc">Berger, M.F., Philippakis, A.A., Qureshi, A.M., He, F.S., Estep, P.W., 3rd, and Bulyk, M.L. (2006). Compact, universal DNA
                        microarrays to comprehensively determine transcription-factor binding site specificities. Nat. Biotechnol. <span class="emphasis"><em>24</em></span>, 1429-1435.</span>
                     <a href="http://dx.doi.org/10.1038/nbt1246" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed"><span class="bibliomisc">Berger S.L. (2010). Cell signaling and transcriptional regulation via histone phsophorylation.  Cold Spring Harb. Symp. Quant.
                        Biol. <span class="emphasis"><em>75</em></span>, 23-26.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21467136&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/sqb.2010.75.044" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed"><span class="bibliomisc">Bertrand, V., and Hobert, O. (2009). Linking asymmetric cell division to the terminal differentiation program of postmitotic
                        neurons in <span class="emphasis"><em>C. elegans</em></span>. Dev. Cell <span class="emphasis"><em>16</em></span>, 563-575.</span>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2009.02.011" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed"><span class="bibliomisc">Bird, D.M., and Riddle, D.L. (1989). Molecular cloning and sequencing of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ama-1;class=Gene" target="_blank">ama-1</a></em></span>, the gene encoding the largest subunit of <span class="emphasis"><em>Caenorhabditis elegans</em></span> RNA polymerase II. Mol. Cell. Biol. <span class="emphasis"><em>9</em></span>, 4119-4130.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2586513&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed"><span class="bibliomisc">Blackwell, T.K., Bowerman, B., Priess, J.R., and Weintraub, H. (1994). Formation of a monomeric DNA binding domain by Skn-1
                        bZIP and homeodomain elements. Science <span class="emphasis"><em>266</em></span>, 621-628.</span>
                     <a href="http://dx.doi.org/10.1126/science.7939715" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed"><span class="bibliomisc">Blackwell, T. K. and Walker, A. K. Transcription mechanisms (September 5, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.121.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.121.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed"><span class="bibliomisc">Blumenthal T., Trans-splicing and operons in <span class="emphasis"><em>C. elegans</em></span> (November 20, 2012), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.5.2, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.5.2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed"><span class="bibliomisc">Brehm, A., Miska, E.A., McCance, D.J., Reid, J.L., Bannister, A.J., and Kouzarides, T. (1998). Retinoblastoma protein recruits
                        histone deacetylase to repress transcription. Nature <span class="emphasis"><em>391</em></span>, 597-601.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9468139&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35404" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed"><span class="bibliomisc">Brodigan, T.M., Liu, J., Park, M., Kipreos, E.T., and Krause, M. (2003). Cyclin E expression during development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>254</em></span>, 102-115.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12606285&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0012-1606(02)00032-5" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed"><span class="bibliomisc">Broitman-Maduro, G., Maduro, M.F., and Rothman, J.H. (2005). The noncanonical binding site of the <a href="http://www.wormbase.org/db/get?name=MED-1;class=Protein" target="_blank">MED-1</a> GATA factor defines differentially regulated target genes in the <span class="emphasis"><em>C. elegans</em></span> mesendoderm. Dev. Cell <span class="emphasis"><em>8</em></span>, 427-433.</span>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2005.01.014" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed"><span class="bibliomisc">Buratowski, S. (2009). Progression through the RNA polymerase II CTD cycle. Mol. Cell <span class="emphasis"><em>36</em></span>, 541-546.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19941815&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.molcel.2009.10.019" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed"><span class="bibliomisc">Cahill, C.M., Tzivion, G., Nasrin, N., Ogg, S., Dore, J., Ruvkun, G., and Alexander-Bridges, M. (2001). Phosphatidylinositol
                        3-kinase signaling inhibits <a href="http://www.wormbase.org/db/get?name=DAF-16;class=Protein" target="_blank">DAF-16</a> DNA binding and function via 14-3-3-dependent and 14-3-3-independent pathways. J. Biol. Chem. <span class="emphasis"><em>276</em></span>, 13402-13410.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11124266&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1074/jbc.M010042200" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed"><span class="bibliomisc">Calvo, D., Victor, M., Gay, F., Sui, G., Luke, M.P., Dufourcq, P., Wen, G., Maduro, M., Rothman, J., and Shi, Y. (2001). A
                        <a href="http://www.wormbase.org/db/get?name=POP-1;class=Protein" target="_blank">POP-1</a> repressor complex restricts inappropriate cell type-specific gene transcription during <span class="emphasis"><em>Caenorhabditis elegans</em></span> embryogenesis. Embo J. <span class="emphasis"><em>20</em></span>, 7197-7208.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11742996&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/emboj/20.24.7197" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed"><span class="bibliomisc">Capowski, E.E., Martin, P., Garvin, C., and Strome, S. (1991). Identification of grandchildless loci whose products are required
                        for normal germ-line development in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>129</em></span>, 1061-1072.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1783292&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed"><span class="bibliomisc">Carroll, A.S., Gilbert, D.E., Liu, X., Cheung, J.W., Michnowicz, J.E., Wagner, G., Ellenberger, T.E., and Blackwell, T.K.
                        (1997). <a href="http://www.wormbase.org/db/get?name=SKN-1;class=Protein" target="_blank">SKN-1</a> domain folding and basic region monomer stabilization upon DNA binding. Genes Dev. <span class="emphasis"><em>11</em></span>, 2227-2238.</span>
                     <a href="http://dx.doi.org/10.1101/gad.11.17.2227" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed"><span class="bibliomisc">Ceol, C.J., and Horvitz, H.R. (2004). A new class of <span class="emphasis"><em>C. elegans</em></span> synMuv genes implicates a Tip60/NuA4-like HAT complex as a negative regulator of Ras signaling. Dev. Cell <span class="emphasis"><em>6</em></span>, 563-576.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15068795&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1534-5807(04)00065-6" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed"><span class="bibliomisc">Chang, S., Johnston, R.J., Jr., and Hobert, O. (2003). A transcriptional regulatory cascade that controls left/right asymmetry
                        in chemosensory neurons of <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>17</em></span>, 2123-2137.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12952888&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1117903" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed"><span class="bibliomisc">Chen, K., and Rajewsky, N. (2007). The evolution of gene regulation by transcription factors and microRNAs. Nat. Rev. Genet.
                        <span class="emphasis"><em>8</em></span>, 93-103.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17230196&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nrg1990" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed"><span class="bibliomisc">Chen, S., Whetstine, J. R., Ghosh, S., Hanover, J.A., Gali, R.R., Grosu, P., and Shi, Y. (2009). The conserved NAD(H)-dependent
                        corepressor <a href="http://www.wormbase.org/db/get?name=CTBP-1;class=Protein" target="_blank">CTBP-1</a> regulates <span class="emphasis"><em>Caenorhabditis elegans</em></span> life span. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>106</em></span>, 1496-1501.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19164523&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0802674106" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed"><span class="bibliomisc">Choi, K.Y., Ji, Y. J., Jee, C., Kim do, H., and Ahnn, J. (2002). Characterization of CeHDA-7, a class II histone deacetylase
                        interacting with <a href="http://www.wormbase.org/db/get?name=MEF-2;class=Protein" target="_blank">MEF-2</a> in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Biochem.  Biophys. Res. Commun. <span class="emphasis"><em>293</em></span>, 1295-1300.</span>
                     <a href="http://dx.doi.org/10.1016/S0006-291X(02)00374-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed"><span class="bibliomisc">Christensen, S., Kodoyianni, V., Bosenberg, M., Friedman, L., and Kimble, J. (1996). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lag-1;class=Gene" target="_blank">lag-1</a></em></span>, a gene required for <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-12;class=Gene" target="_blank">lin-12</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=glp-1;class=Gene" target="_blank">glp-1</a></em></span> signaling in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, is homologous to human CBF1 and <span class="emphasis"><em>Drosophila</em></span> Su(H). Development <span class="emphasis"><em>122</em></span>, 1373-1383.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8625826&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed"><span class="bibliomisc">Conradt, B., and Horvitz, H.R. (1999). The TRA-1A sex determination protein of <span class="emphasis"><em>C. elegans</em></span> regulates sexually dimorphic cell deaths by repressing the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-1;class=Gene" target="_blank">egl-1</a></em></span> cell death activator gene. Cell <span class="emphasis"><em>98</em></span>, 317-327.</span>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(00)81961-3" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed"><span class="bibliomisc">Conradt, B. and Xue D. Programmed cell death (October 06, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.32.1, http://www.wormbook.org.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18061982&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.32.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed"><span class="bibliomisc">Copley, R.R. (2005). The EH1 motif in metazoan transcription factors. BMC Genomics <span class="emphasis"><em>6</em></span>, 169.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16309560&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-6-169" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed"><span class="bibliomisc">Corsi, A.K., Kostas, S.A., Fire, A., and Krause, M. (2000). <span class="emphasis"><em>Caenorhabditis elegans</em></span> twist plays an essential role in non-striated muscle development. Development <span class="emphasis"><em>127</em></span>, 2041-2051.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10769229&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed"><span class="bibliomisc">Cui, M., and Han, M. (2003). Cis regulatory requirements for vulval cell-specific expression of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> fibroblast growth factor gene <span class="emphasis"><em>egl-17.</em></span> Dev. Biol. <span class="emphasis"><em>257</em></span>, 104-116.</span>
                     <a href="http://dx.doi.org/0.1016/S0012-1606(03)00033-2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed"><span class="bibliomisc">Dantonel, J.C., Quintin, S., Lakatos, L., Labouesse, M., and Tora, L. (2000). TBP-like factor is required for embryonic RNA
                        polymerase II transcription in <span class="emphasis"><em>C. elegans</em></span>. Mol. Cell <span class="emphasis"><em>6</em></span>, 715-722.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11030350&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)00069-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed"><span class="bibliomisc">Davison, E.M., Saffer, A.M., Huang, L.S., DeModena, J., Sternberg, P.W., and Horvitz, H.R. (2011). The LIN-15A and <a href="http://www.wormbase.org/db/get?name=LIN-56;class=Protein" target="_blank">LIN-56</a> transcriptional regulators interact to negatively regulate EGF/Ras signaling in <span class="emphasis"><em>Caenorhabditis elegans</em></span> vulval cell-fate determination. Genetics <span class="emphasis"><em>187</em></span>, 803-815.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21196525&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.110.124487" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed"><span class="bibliomisc">De Masi, F., Grove, C.A., Vedenko, A., Alibes, A., Gisselbrecht, S.S., Serrano, L., Bulyk, M.L., and Walhout, A.J. (2011).
                        Using a structural and logics systems approach to infer bHLH-DNA binding specificity determinants. Nucleic Acids Res. <span class="emphasis"><em>39</em></span>, 4553-4563.</span>
                     <a href="http://dx.doi.org/10.1093/nar/gkr070" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed"><span class="bibliomisc">del Castillo-Olivares, A., Kulkarni, M., and Smith, H.E. (2009). Regulation of sperm gene expression by the GATA factor <a href="http://www.wormbase.org/db/get?name=ELT-1;class=Protein" target="_blank">ELT-1</a>. Dev. Biol. <span class="emphasis"><em>333</em></span>, 397-408.</span>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2009.06.044" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed"><span class="bibliomisc">Deal, R.B., and Henikoff, S. (2010). A simple method for gene expression and chromatin profiling of individual cell types
                        within a tissue. Dev. Cell <span class="emphasis"><em>18</em></span>, 1030-1040.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20627084&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2010.05.013" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed"><span class="bibliomisc">DeMeo, S.D., Lombel, R.M., Cronin, M., Smith, E.L., Snowflack, D.R., Reinert, K., Clever, S., and Wightman, B. (2008). Specificity
                        of DNA-binding by the <a href="http://www.wormbase.org/db/get?name=FAX-1;class=Protein" target="_blank">FAX-1</a> and <a href="http://www.wormbase.org/db/get?name=NHR-67;class=Protein" target="_blank">NHR-67</a> nuclear receptors of <span class="emphasis"><em>Caenorhabditis elegans</em></span> is partially mediated via a subclass-specific P-box residue. BMC Mol. Biol. <span class="emphasis"><em>9</em></span>, 2.</span>
                     <a href="http://dx.doi.org/10.1186/1471-2199-9-2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed"><span class="bibliomisc">Deng, H., Sun, Y., Zhang, Y., Luo, X., Hou, W., Yan, L., Chen, Y., Tian, E., Han, J., and Zhang, H. (2007). Transcription
                        factor NFY globally represses the expression of the <span class="emphasis"><em>C. elegans</em></span> Hox gene <span class="emphasis"><em>Abdominal-B</em></span> homolog <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-5;class=Gene" target="_blank">egl-5</a></em></span>. Dev. Biol. <span class="emphasis"><em>308</em></span>, 583-592.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17574230&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2007.05.021" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed"><span class="bibliomisc">Deplancke, B., Mukhopadhyay, A., Ao, W., Elewa, A.M., Grove, C.A., Martinez, N.J., Sequerra, R., Doucette-Stamm, L., Reece-Hoyes,
                        J.S., Hope, I.A.<span class="emphasis"><em>, et al.</em></span> (2006). A gene-centered <span class="emphasis"><em>C. elegans</em></span> protein-DNA interaction network. Cell <span class="emphasis"><em>125</em></span>, 1193-1205.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16777607&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2006.04.038" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed"><span class="bibliomisc">Dichoso, D., Brodigan, T., Chwoe, K.Y., Lee, J.S., Llacer, R., Park, M., Corsi, A.K., Kostas, S.A., Fire, A., Ahnn, J., and
                        Krause, M. (2000). The MADS-Box factor CeMEF2 is not essential for <span class="emphasis"><em>Caenorhabditis elegans</em></span> myogenesis and development. Dev. Biol. <span class="emphasis"><em>223</em></span>, 431-440.</span>
                     <a href="http://dx.doi.org/10.1006/dbio.2000.9758" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed"><span class="bibliomisc">Duggan, A., Ma, C., and Chalfie, M. (1998). Regulation of touch receptor differentiation by the <span class="emphasis"><em>Caenorhabditis elegans <a href="http://www.wormbase.org/db/get?name=mec-3;class=Gene" target="_blank">mec-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-86;class=Gene" target="_blank">unc-86</a></em></span> genes. Development <span class="emphasis"><em>125</em></span>, 4107-4119.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9735371&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed"><span class="bibliomisc">Eastman, C., Horvitz, H.R., and Jin, Y. (1999). Coordinated transcriptional regulation of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-25;class=Gene" target="_blank">unc-25</a></em></span> glutamic acid decarboxylase and the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-47;class=Gene" target="_blank">unc-47</a></em></span> GABA vesicular transporter by the <span class="emphasis"><em>Caenorhabditis elegans</em></span> <a href="http://www.wormbase.org/db/get?name=UNC-30;class=Protein" target="_blank">UNC-30</a> homeodomain protein. J. Neurosci. <span class="emphasis"><em>19</em></span>, 6225-6234.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10414952&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed"><span class="bibliomisc">Egan, C.R., Chung, M.A., Allen, F.L., Heschl, M.F., Van Buskirk, C.L., and McGhee, J.D. (1995). A gut-to-pharynx/tail switch
                        in embryonic expression of the <span class="emphasis"><em>Caenorhabditis elegans <a href="http://www.wormbase.org/db/get?name=ges-1;class=Gene" target="_blank">ges-1</a></em></span> gene centers on two GATA sequences. Dev. Biol. <span class="emphasis"><em>170</em></span>, 397-419.</span>
                     <a href="http://dx.doi.org/10.1006/dbio.1995.1225" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed"><span class="bibliomisc">Enoki, Y., and Sakurai, H. (2011). Diversity in DNA recognition by heat shock transcription factors (HSFs) from model organisms.
                        FEBS Lett. <span class="emphasis"><em>585</em></span>, 1293-1298.</span>
                     <a href="http://dx.doi.org/10.1016/j.febslet.2011.04.014" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed"><span class="bibliomisc">Ercan, S., Giresi, P.G., Whittle, C.M., Zhang, X., Green, R.D., and Lieb, J.D. (2007). X chromosome repression by localization
                        of the <span class="emphasis"><em>C. elegans</em></span> dosage compensation machinery to sites of transcription initiation. Nat. Genet. <span class="emphasis"><em>39</em></span>, 403-408.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17293863&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1983" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed"><span class="bibliomisc">Ercan, S., Lubling, Y., Segal, E., and Lieb, J.D. (2011). High nucleosome occupancy is encoded at X-linked gene promoters
                        in <span class="emphasis"><em>C. elegans</em></span>. Genome Res. <span class="emphasis"><em>21</em></span>, 237-244.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17293863&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.115931.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed"><span class="bibliomisc">Etchberger, J.F., Flowers, E.B., Poole, R.J., Bashllari, E., and Hobert, O. (2009). Cis-regulatory mechanisms of left/right
                        asymmetric neuron-subtype specification in <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>136</em></span>, 147-160.</span>
                     <a href="http://dx.doi.org/10.1242/dev.030064" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed"><span class="bibliomisc">Fay, D.S., and Yochem, J. (2007). The SynMuv genes of <span class="emphasis"><em>Caenorhabditis elegans</em></span> in vulval development and beyond. Dev. Biol. <span class="emphasis"><em>306</em></span>, 1-9.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17434473&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2007.03.016" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed"><span class="bibliomisc">Finney, M., and Ruvkun, G. (1990). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=unc-86;class=Gene" target="_blank">unc-86</a></em></span> gene product couples cell lineage and cell identity in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>63</em></span>, 895-905.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2257628&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(90)90493-X" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed"><span class="bibliomisc">Fisher, K., Southall, S.M., Wilson, J.R., and Poulin, G.B. (2010). Methylation and demethylation activities of a <span class="emphasis"><em>C. elegans</em></span> MLL-like complex attenuate RAS signalling. Dev. Biol. <span class="emphasis"><em>341</em></span>, 142-153.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20188723&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2010.02.023" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed"><span class="bibliomisc">Flames, N., and Hobert, O. (2009). Gene regulatory logic of dopamine neuron differentiation. Nature <span class="emphasis"><em>458</em></span>, 885-889.</span>
                     <a href="http://dx.doi.org/0.1038/nature07929" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed"><span class="bibliomisc">Fox, R.M., Von Stetina, S.E., Barlow, S.J., Shaffer, C., Olszewski, K.L., Moore, J.H., Dupuy, D., Vidal, M., and Miller, D.M.,
                        3rd (2005). A gene expression fingerprint of <span class="emphasis"><em>C. elegans</em></span> embryonic motor neurons. BMC Genomics <span class="emphasis"><em>6</em></span>, 42.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15780142&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-6-42" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed"><span class="bibliomisc">Fukushige, T., Goszczynski, B., Yan, J., and McGhee, J.D. (2005). Transcriptional control and patterning of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=pho-1;class=Gene" target="_blank">pho-1</a></em></span> gene, an essential acid phosphatase expressed in the <span class="emphasis"><em>C. elegans</em></span> intestine. Dev. Biol. <span class="emphasis"><em>279</em></span>, 446-461.</span>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2004.12.012" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed"><span class="bibliomisc">Furuyama, T., Nakazawa, T., Nakano, I., and Mori, N. (2000). Identification of the differential distribution patterns of mRNAs
                        and consensus binding sequences for mouse <a href="http://www.wormbase.org/db/get?name=DAF-16;class=Protein" target="_blank">DAF-16</a> homologues. Biochem. J. <span class="emphasis"><em>349</em></span>, 629-634.</span>
                     <a href="http://dx.doi.org/10.1042/0264-6021:3490629" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed"><span class="bibliomisc">Gamsjaeger, R., Liew, C.K., Loughlin, F.E., Crossley, M., and Mackay, J.P. (2007). Sticky fingers: zinc-fingers as protein-recognition
                        motifs. Trends Biochem. Sci <span class="emphasis"><em>32</em></span>, 63-70.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17210253&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.tibs.2006.12.007" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed"><span class="bibliomisc">Garrido-Lecca, A., and Blumenthal, T. (2010). RNA polymerase II C-terminal domain phosphorylation patterns in <span class="emphasis"><em>Caenorhabditis elegans</em></span> operons, polycistronic gene clusters with only one promoter. Mol. Cell. Biol. <span class="emphasis"><em>30</em></span>, 3887-3893.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20498277&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.00325-10" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed"><span class="bibliomisc">Gaudet, J., and Mango, S.E. (2002). Regulation of organogenesis by the <span class="emphasis"><em>Caenorhabditis elegans</em></span> FoxA protein <a href="http://www.wormbase.org/db/get?name=PHA-4;class=Protein" target="_blank">PHA-4</a>. Science <span class="emphasis"><em>295</em></span>, 821-825.</span>
                     <a href="http://dx.doi.org/10.1126/science.1065175" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed"><span class="bibliomisc">Gaudet, J., and McGhee, J.D. (2010). Recent advances in understanding the molecular mechanisms regulating <span class="emphasis"><em>C. elegans</em></span> transcription. Dev. Dyn. <span class="emphasis"><em>239</em></span>, 1388-1404.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20175193&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed"><span class="bibliomisc">Gerstein, M.B., Lu, Z.J., Van Nostrand, E.L., Cheng, C., Arshinoff, B.I., Liu, T., Yip, K.Y., Robilotto, R., Rechtsteiner,
                        A., Ikegami, K.<span class="emphasis"><em>, et al.</em></span> (2010). Integrative analysis of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> genome by the modENCODE project. Science <span class="emphasis"><em>330</em></span>, 17751787.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177976&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1196914" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed"><span class="bibliomisc">Ghai, V., and Gaudet, J. (2008). The CSL transcription factor <a href="http://www.wormbase.org/db/get?name=LAG-1;class=Protein" target="_blank">LAG-1</a> directly represses <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hlh-6;class=Gene" target="_blank">hlh-6</a></em></span> expression in <span class="emphasis"><em>C. elegans.</em></span> Dev. Biol. <span class="emphasis"><em>322</em></span>, 334-344.</span>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2008.07.018" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed"><span class="bibliomisc">Good, K., Ciosk, R., Nance, J., Neves, A., Hill, R.J., and Priess, J.R. (2004). The T-box transcription factors <a href="http://www.wormbase.org/db/get?name=TBX-37;class=Protein" target="_blank">TBX-37</a> and <a href="http://www.wormbase.org/db/get?name=TBX-38;class=Protein" target="_blank">TBX-38</a> link <a href="http://www.wormbase.org/db/get?name=GLP-1;class=Protein" target="_blank">GLP-1</a>/Notch signaling to mesoderm induction in <span class="emphasis"><em>C. elegans</em></span> embryos. Development <span class="emphasis"><em>131</em></span>, 1967-1978.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15056620&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.01088" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib72"></a><p class="bibliomixed"><span class="bibliomisc">Graber, J.H., Salisbury, J., Hutchins, L.N., and Blumenthal, T.(2007). <span class="emphasis"><em>C. elegans</em></span> sequences that control trans-splicing and operon pre-mRNA processing. RNA <span class="emphasis"><em>13</em></span>, 14091426.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17630324&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.596707" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib73"></a><p class="bibliomixed"><span class="bibliomisc">Greer, E.L., Maures, T.J., Hauswirth, A.G., Green, E.M., Leeman, D.S., Maro, G.S., Han, S., Banko, M.R., Gozani, O., and Brunet,
                        A. (2010). Members of the H3K4 trimethylation complex regulate lifespan in a germline-dependent manner in <span class="emphasis"><em>C. elegans.</em></span> Nature <span class="emphasis"><em>466</em></span>, 383-387.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20555324&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature09195" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib74"></a><p class="bibliomixed"><span class="bibliomisc">Grishkevich, V., Hashimshony, T., and Yanai, I. (2011). Core promoter T-blocks correlate with gene expression levels in <span class="emphasis"><em>C. elegans</em></span>. Genome Res. <span class="emphasis"><em>21</em></span>, 707-717.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21367940&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.113381.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib75"></a><p class="bibliomixed"><span class="bibliomisc">Grove, C.A., De Masi, F., Barrasa, M.I., Newburger, D.E., Alkema, M.J., Bulyk, M.L., and Walhout, A.J. (2009). A multiparameter
                        network reveals extensive divergence between <span class="emphasis"><em>C. elegans</em></span> bHLH transcription factors. Cell <span class="emphasis"><em>138</em></span>, 314-327.</span>
                     <a href="http://dx.doi.org/10.1016/j.cell.2009.04.058" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib76"></a><p class="bibliomixed"><span class="bibliomisc">Haerty, W., Artieri, C., Khezri, N., Singh, R.S., and Gupta, B.P. (2008). Comparative analysis of function and interaction
                        of transcription factors in nematodes: extensive conservation of orthology coupled to rapid sequence evolution. BMC Genomics
                        <span class="emphasis"><em>9</em></span>, 399.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18752680&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2164-9-399" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib77"></a><p class="bibliomixed"><span class="bibliomisc">Hahn-Windgassen, A., and Van Gilst, M.R. (2009). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> HNF4&#945; Homolog, <a href="http://www.wormbase.org/db/get?name=NHR-31;class=Protein" target="_blank">NHR-31</a>, mediates excretory tube growth and function through coordinate regulation of the vacuolar ATPase. PLoS Genet. <span class="emphasis"><em>5</em></span>, e1000553.</span>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000553" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib78"></a><p class="bibliomixed"><span class="bibliomisc">Hallam, S., Singer, E., Waring, D., and Jin, Y. (2000). The <span class="emphasis"><em>C. elegans</em></span> NeuroD homolog <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=cnd-1;class=Gene" target="_blank">cnd-1</a></em></span> functions in multiple aspects of motor neuron fate specification. Development <span class="emphasis"><em>127</em></span>, 4239-4252.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10976055&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib79"></a><p class="bibliomixed"><span class="bibliomisc">Harfe, B.D., and Fire, A. (1998). Muscle and nerve-specific regulation of a novel NK-2 class homeodomain factor in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>125</em></span>, 421-429.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9425137&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib80"></a><p class="bibliomixed"><span class="bibliomisc">Harfe, B.D., Vaz Gomes, A., Kenyon, C., Liu, J., Krause, M., and Fire, A. (1998). Analysis of a <span class="emphasis"><em>Caenorhabditis elegans</em></span> Twist homolog identifies conserved and divergent aspects of mesodermal patterning. Genes Dev. <span class="emphasis"><em>12</em></span>, 2623-2635.</span>
                     <a href="http://dx.doi.org/10.1101/gad.12.16.2623" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib81"></a><p class="bibliomixed"><span class="bibliomisc">Hargitai, B., Kutnyanszky, V., Blauwkamp, T.A., Stetak, A., Csankovszki, G., Takacs-Vellai, K., and Vellai, T. (2009). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=xol-1;class=Gene" target="_blank">xol-1</a>,</em></span> the master sex-switch gene in <span class="emphasis"><em>C. elegans</em></span>, is a transcriptional target of the terminal sex-determining factor <a href="http://www.wormbase.org/db/get?name=TRA-1;class=Protein" target="_blank">TRA-1</a>. Development <span class="emphasis"><em>136</em></span>, 3881-3887.</span>
                     <a href="http://dx.doi.org/10.1242/dev.034637" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib82"></a><p class="bibliomixed"><span class="bibliomisc">Harrison, M.M., Ceol, C.J., Lu, X., and Horvitz, H.R. (2006). Some <span class="emphasis"><em>C. elegans</em></span> class B synthetic multivulva proteins encode a conserved <a href="http://www.wormbase.org/db/get?name=LIN-35;class=Protein" target="_blank">LIN-35</a> Rb-containing complex distinct from a NuRD-like complex. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>103</em></span>, 16782-16787.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17075059&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0608461103" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib83"></a><p class="bibliomixed"><span class="bibliomisc">Harrison, M.M., Lu, X., and Horvitz, H.R. (2007). <a href="http://www.wormbase.org/db/get?name=LIN-61;class=Protein" target="_blank">LIN-61</a>, one of two <span class="emphasis"><em>Caenorhabditis elegans</em></span> malignant-brain-tumor-repeat-containing proteins, acts with the DRM and NuRD-like protein complexes in vulval development
                        but not in certain other biological processes. Genetics <span class="emphasis"><em>176</em></span>, 255-271.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17409073&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.106.069633" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib84"></a><p class="bibliomixed"><span class="bibliomisc">Heard, M., Maina, C.V., Morehead, B.E., Hoener, M.C., Nguyen, T.Q., Williams, C.C., Rowan, B.G., and Gissendanner, C.R. (2010).
                        A functional NR4A nuclear receptor DNA-binding domain is required for organ development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genesis <span class="emphasis"><em>48</em></span>, 485-491.</span>
                     <a href="http://dx.doi.org/10.1002/dvg.20646" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib85"></a><p class="bibliomixed"><span class="bibliomisc">Hillier, L.W., Reinke, V., Green, P., Hirst, M., Marra, M.A., and Waterston, R.H. (2009). Massively parallel sequencing of
                        the polyadenylated transcriptome of <span class="emphasis"><em>C. elegans</em></span>. Genome Res. <span class="emphasis"><em>19</em></span>, 657-666.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19181841&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.088112.108" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib86"></a><p class="bibliomixed"><span class="bibliomisc">Hirose, T., Galvin, B.D., and Horvitz, H.R. (2010). Six and Eya promote apoptosis through direct transcriptional activation
                        of the proapoptotic BH3-only gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-1;class=Gene" target="_blank">egl-1</a></em></span> in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>107</em></span>, 15479-15484.</span>
                     <a href="http://dx.doi.org/10.1073/pnas.1010023107" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib87"></a><p class="bibliomixed"><span class="bibliomisc">Ho, L., and Crabtree, G.R. (2010). Chromatin remodelling during development. Nature <span class="emphasis"><em>463</em></span>, 474-484.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20110991&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature08911" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib88"></a><p class="bibliomixed"><span class="bibliomisc">Hobert O. Neurogenesis in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span> (October 4, 2010), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.12.2, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.12.2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib89"></a><p class="bibliomixed"><span class="bibliomisc">Horner, M.A., Quintin, S., Domeier, M.E., Kimble, J., Labouesse, M., and Mango, S.E. (1998). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=pha-4;class=Gene" target="_blank">pha-4</a></em></span>, an HNF-3 homolog, specifies pharyngeal organ identity in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genes Dev. <span class="emphasis"><em>12</em></span>, 1947-1952.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9649499&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.12.13.1947" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib90"></a><p class="bibliomixed"><span class="bibliomisc">Hristova, M., Birse, D., Hong, Y., and Ambros, V. (2005). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> heterochronic regulator <a href="http://www.wormbase.org/db/get?name=LIN-14;class=Protein" target="_blank">LIN-14</a> is a novel transcription factor that controls the developmental timing of transcription from the insulin/insulin-like growth
                        factor gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ins-33;class=Gene" target="_blank">ins-33</a></em></span> by direct DNA binding. Mol. Cell Biol. <span class="emphasis"><em>25</em></span>, 11059-11072.</span>
                     <a href="http://dx.doi.org/10.1128/MCB.25.24.11059-11072.2005" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib91"></a><p class="bibliomixed"><span class="bibliomisc">Huang, P., Pleasance, E.D., Maydan, J.S., Hunt-Newbury, R., O'Neil, N.J., Mah, A., Baillie, D.L., Marra, M.A., Moerman, D.G.,
                        and Jones, S.J. (2007). Identification and analysis of internal promoters in <span class="emphasis"><em>Caenorhabditis elegans</em></span> operons. Genome Res. <span class="emphasis"><em>17</em></span>, 1478-1485.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17712020&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.6824707" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib92"></a><p class="bibliomixed"><span class="bibliomisc">Hwang, B.J., and Sternberg, P.W. (2004). A cell-specific enhancer that specifies <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-3;class=Gene" target="_blank">lin-3</a></em></span> expression in the <span class="emphasis"><em>C. elegans</em></span> anchor cell for vulval development. Development <span class="emphasis"><em>131</em></span>, 143-151.</span>
                     <a href="http://dx.doi.org/10.1242/dev.00924" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib93"></a><p class="bibliomixed"><span class="bibliomisc">Ikegami, K., Egelhofer, T.A., Strome, S., and Lieb, J.D. (2010). <span class="emphasis"><em>Caenorhabditis elegans</em></span> chromosome arms are anchored to the nuclear membrane via discontinuous association with <a href="http://www.wormbase.org/db/get?name=LEM-2;class=Protein" target="_blank">LEM-2</a>. Genome Biol. <span class="emphasis"><em>11</em></span>, R120.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2010&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2010-11-12-r120" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib94"></a><p class="bibliomixed"><span class="bibliomisc">Jan, C.H., Friedman, R.C., Ruby, J.G., and Bartel, D.P. (2011). Formation, regulation and evolution of <span class="emphasis"><em>Caenorhabditis elegans</em></span> 3'UTRs. Nature <span class="emphasis"><em>469</em></span>, 97-101.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21085120&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib95"></a><p class="bibliomixed"><span class="bibliomisc">Jans, J., Gladden, J.M., Ralston, E.J., Pickle, C.S., Michel, A.H., Pferdehirt, R.R., Eisen, M.B., and Meyer, B.J. (2009).
                        A condensin-like dosage compensation complex acts at a distance to control expression throughout the genome. Genes Dev. <span class="emphasis"><em>23</em></span>, 602-618.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19270160&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1751109" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib96"></a><p class="bibliomixed"><span class="bibliomisc">Jiang, Y., Shi, H., Amin, N.M., Sultan, I., and Liu, J. (2008). Mesodermal expression of the <span class="emphasis"><em>C. elegans</em></span> HMX homolog <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mls-2;class=Gene" target="_blank">mls-2</a></em></span> requires the PBC homolog <a href="http://www.wormbase.org/db/get?name=CEH-20;class=Protein" target="_blank">CEH-20</a>. Mech. Dev. <span class="emphasis"><em>125</em></span>, 451-461.</span>
                     <a href="http://dx.doi.org/10.1016/j.mod.2008.01.009" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib97"></a><p class="bibliomixed"><span class="bibliomisc">Jin, Y., Hoskins, R., and Horvitz, H.R. (1994). Control of type-D GABAergic neuron differentiation by <span class="emphasis"><em>C. elegans</em></span> <a href="http://www.wormbase.org/db/get?name=UNC-30;class=Protein" target="_blank">UNC-30</a> homeodomain protein. Nature <span class="emphasis"><em>372</em></span>, 780-783.</span>
                     <a href="http://dx.doi.org/10.1038/372780a0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib98"></a><p class="bibliomixed"><span class="bibliomisc">Johnson, A.D., Fitzsimmons, D., Hagman, J., and Chamberlin, H.M. (2001). <a href="http://www.wormbase.org/db/get?name=EGL-38;class=Protein" target="_blank">EGL-38</a> Pax regulates the ovo-related gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-48;class=Gene" target="_blank">lin-48</a></em></span> during <span class="emphasis"><em>Caenorhabditis elegans</em></span> organ development. Development <span class="emphasis"><em>128</em></span>, 2857-2865.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11532910&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib99"></a><p class="bibliomixed"><span class="bibliomisc">Jones, D., Russnak, R.H., Kay, R.J., and Candido, E.P. (1986). Structure, expression, and evolution of a heat shock gene locus
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span> that is flanked by repetitive elements. J. Biol. Chem <span class="emphasis"><em>261</em></span>, 12006-12015.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3017958&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib100"></a><p class="bibliomixed"><span class="bibliomisc">Kalb, J.M., Lau, K.K., Goszczynski, B., Fukushige, T., Moons, D., Okkema, P.G., and McGhee, J.D. (1998).  <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=pha-4;class=Gene" target="_blank">pha-4</a></em></span> is Ce-<span class="emphasis"><em>fkh-1</em></span>, a fork head/HNF-3&#945;,&#946;,&#947; homolog that functions in organogenesis of the <span class="emphasis"><em>C. elegans</em></span> pharynx. Development <span class="emphasis"><em>125</em></span>, 2171-2180.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9584117&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib101"></a><p class="bibliomixed"><span class="bibliomisc">Kaltenbach, L., Horner, M.A., Rothman, J.H., and Mango, S.E. (2000). The TBP-like factor CeTLF is required to activate RNA
                        polymerase II transcription during <span class="emphasis"><em>C. elegans</em></span> embryogenesis. Mol. Cell <span class="emphasis"><em>6</em></span>, 705-713.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11030349&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)00068-X" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib102"></a><p class="bibliomixed"><span class="bibliomisc">Kaminsky, R., Denison, C., Bening-Abu-Shach, U., Chisholm, A.D., Gygi, S.P., and Broday, L. (2009). SUMO regulates the assembly
                        and function of a cytoplasmic intermediate filament protein in <span class="emphasis"><em>C. elegans</em></span>. Dev. Cell <span class="emphasis"><em>17</em></span>, 724-735.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19922876&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2009.10.005" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib103"></a><p class="bibliomixed"><span class="bibliomisc">Karakuzu, O., Wang, D.P., and Cameron, S. (2009). <a href="http://www.wormbase.org/db/get?name=MIG-32;class=Protein" target="_blank">MIG-32</a> and SPAT-3A are PRC1 homologs that control neuronal migration in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>136</em></span>, 943-953.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19211678&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.029363" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib104"></a><p class="bibliomixed"><span class="bibliomisc">Karp, X., and Greenwald, I. (2003). Post-transcriptional regulation of the E/Daughterless ortholog <a href="http://www.wormbase.org/db/get?name=HLH-2;class=Protein" target="_blank">HLH-2</a>, negative feedback, and birth order bias during the AC/VU decision in <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>17</em></span>, 3100-3111.</span>
                     <a href="http://dx.doi.org/10.1101/gad.1160803" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib106"></a><p class="bibliomixed"><span class="bibliomisc">Kato, M., de Lencastre, A., Pincus, Z., and Slack, F.J. (2009). Dynamic expression of small non-coding RNAs, including novel
                        microRNAs and piRNAs/21U-RNAs, during <span class="emphasis"><em>Caenorhabditis elegans</em></span> development. Genome Biol. <span class="emphasis"><em>10</em></span>, R54.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19460142&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2009-10-5-r54" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib107"></a><p class="bibliomixed"><span class="bibliomisc">Kim, K., Colosimo, M.E., Yeung, H., and Sengupta, P. (2005). The <a href="http://www.wormbase.org/db/get?name=UNC-3;class=Protein" target="_blank">UNC-3</a> Olf/EBF protein represses alternate neuronal programs to specify chemosensory neuron identity. Dev. Biol. <span class="emphasis"><em>286</em></span>, 136-148.</span>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2005.07.024" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib108"></a><p class="bibliomixed"><span class="bibliomisc">Kirienko, N.V., and Fay, D.S. (2007). Transcriptome profiling of the <span class="emphasis"><em>C. elegans</em></span> Rb ortholog reveals diverse developmental roles. Dev. Biol. <span class="emphasis"><em>305</em></span>, 674-684.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17368442&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2007.02.021" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib109"></a><p class="bibliomixed"><span class="bibliomisc">Kirienko, N.V., McEnerney, J.D., and Fay, D.S. (2008). Coordinated regulation of intestinal functions in <span class="emphasis"><em>C. elegans</em></span> by <a href="http://www.wormbase.org/db/get?name=LIN-35;class=Protein" target="_blank">LIN-35</a>/Rb and SLR-2. PLoS Genet. <span class="emphasis"><em>4</em></span>, e1000059.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18437219&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000059" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib110"></a><p class="bibliomixed"><span class="bibliomisc">Knight, A.S., Notaridou, M., and Watson, R.J. (2009). A Lin-9 complex is recruited by B-Myb to activate transcription of <a href="http://www.wormbase.org/db/get?name=G2;class=Cell" target="_blank">G2</a>/M genes in undifferentiated embryonal carcinoma cells. Oncogene <span class="emphasis"><em>28</em></span>, 1737-1747.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19252525&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/onc.2009.22" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib111"></a><p class="bibliomixed"><span class="bibliomisc">Koester-Eiserfunke, N., and Fischle, W. (2011). H3K9me2/3 binding of the MBT domain protein <a href="http://www.wormbase.org/db/get?name=LIN-61;class=Protein" target="_blank">LIN-61</a> is essential for <span class="emphasis"><em>Caenorhabditis elegans</em></span> vulva development. PLoS Genet. <span class="emphasis"><em>7</em></span>, e1002017.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21437264&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002017" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib112"></a><p class="bibliomixed"><span class="bibliomisc">Koh, K., Peyrot, S.M., Wood, C.G., Wagmaister, J.A., Maduro, M.F., Eisenmann, D.M., and Rothman, J.H. (2002). Cell fates and
                        fusion in the <span class="emphasis"><em>C. elegans</em></span> vulval primordium are regulated by the <a href="http://www.wormbase.org/db/get?name=EGL-18;class=Protein" target="_blank">EGL-18</a> and <a href="http://www.wormbase.org/db/get?name=ELT-6;class=Protein" target="_blank">ELT-6</a> GATA factors &#8211; apparent direct targets of the <a href="http://www.wormbase.org/db/get?name=LIN-39;class=Protein" target="_blank">LIN-39</a> Hox protein. Development <span class="emphasis"><em>129</em></span>, 5171-5180.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12399309&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib113"></a><p class="bibliomixed"><span class="bibliomisc">Korenjak, M., Taylor-Harding, B., Binne, U.K., Satterlee, J.S., Stevaux, O., Aasland, R., White-Cooper, H., Dyson, N., and
                        Brehm, A. (2004). Native E2F/RBF complexes contain Myb-interacting proteins and repress transcription of developmentally controlled
                        E2F target genes. Cell <span class="emphasis"><em>119</em></span>, 181-193.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15479636&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2004.09.034" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib114"></a><p class="bibliomixed"><span class="bibliomisc">Kostas, S.A., and Fire, A. (2002). The T-box factor <a href="http://www.wormbase.org/db/get?name=MLS-1;class=Protein" target="_blank">MLS-1</a> acts as a molecular switch during specification of nonstriated muscle in <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>16</em></span>, 257-269.</span>
                     <a href="http://dx.doi.org/10.1101/gad.923102" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib115"></a><p class="bibliomixed"><span class="bibliomisc">Kostrouchova, M., Krause, M., Kostrouch, Z., and Rall, J.E. (1998). CHR3: a <span class="emphasis"><em>Caenorhabditis elegans</em></span> orphan nuclear hormone receptor required for proper epidermal development and molting. Development <span class="emphasis"><em>125</em></span>, 1617-1626.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9521900&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib116"></a><p class="bibliomixed"><span class="bibliomisc">Kozak, M. (1981). Possible role of flanking nucleotides in recognition of the AUG initiator codon by eukaryotic ribosomes.
                        Nucleic Acids Res. <span class="emphasis"><em>9</em></span>, 5233-5252.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7301588&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/9.20.5233" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib117"></a><p class="bibliomixed"><span class="bibliomisc">Krause, M. (1995). Transcription and translation. Methods Cell Biol. <span class="emphasis"><em>48</em></span>, 483-512.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8531739&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0091-679X(08)61400-4" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib118"></a><p class="bibliomixed"><span class="bibliomisc">Krause, M., Fire, A., Harrison, S.W., Priess, J., and Weintraub, H. (1990). CeMyoD accumulation defines the body wall muscle
                        cell fate during <span class="emphasis"><em>C. elegans</em></span> embryogenesis. Cell <span class="emphasis"><em>63</em></span>, 907-919.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2175254&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(90)90494-Y" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib119"></a><p class="bibliomixed"><span class="bibliomisc">Krause, M., Fire, A., White-Harrison, S., Weintraub, H., and Tapscott, S. (1992). Functional conservation of nematode and
                        vertebrate myogenic regulatory factors. J. Cell Sci. Suppl. <span class="emphasis"><em>16</em></span>, 111-115.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1338434&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib120"></a><p class="bibliomixed"><span class="bibliomisc">Krause, M., and Hirsh, D. (1987). A trans-spliced leader sequence on actin mRNA in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>49</em></span>, 753-761.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3581169&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(87)90613-1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib121"></a><p class="bibliomixed"><span class="bibliomisc">Krause, M., Park, M., Zhang, J.M., Yuan, J., Harfe, B., Xu, S.Q., Greenwald, I., Cole, M., Paterson, B., and Fire, A. (1997).
                        A <span class="emphasis"><em>C. elegans</em></span> E/Daughterless bHLH protein marks neuronal but not striated muscle development. Development <span class="emphasis"><em>124</em></span>, 2179-2189.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9187144&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib122"></a><p class="bibliomixed"><span class="bibliomisc">Kuchenthal, C.A., Chen, W., and Okkema, P.G. (2001). Multiple enhancers contribute to expression of the NK-2 homeobox gene
                        <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ceh-22;class=Gene" target="_blank">ceh-22</a></em></span> in <span class="emphasis"><em>C. elegans</em></span> pharyngeal muscle. Genesis <span class="emphasis"><em>31</em></span>, 156-166.</span>
                     <a href="http://dx.doi.org/10.1002/gene.10018" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib123"></a><p class="bibliomixed"><span class="bibliomisc">Lam, N., Chesney, M.A., and Kimble, J. (2006). Wnt signaling and CEH22/tinman/Nkx2.5 specify a stem cell niche in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>16</em></span>, 287-295.</span>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.12.015" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib124"></a><p class="bibliomixed"><span class="bibliomisc">Lazakovitch, E., Kalb, J.M., Matsumoto, R., Hirono, K., Kohara, Y., and Gronostajski, R.M. (2005). <span class="emphasis"><em>nfi-I</em></span> affects behavior and life-span in <span class="emphasis"><em>C. elegans</em></span> but is not essential for DNA replication or survival. BMC Dev. Biol. <span class="emphasis"><em>5</em></span>, 24.</span>
                     <a href="http://dx.doi.org/10.1186/1471-213X-5-24" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib126"></a><p class="bibliomixed"><span class="bibliomisc">Lee, S.S., Kennedy, S., Tolonen, A.C., and Ruvkun, G. (2003). <a href="http://www.wormbase.org/db/get?name=DAF-16;class=Protein" target="_blank">DAF-16</a> target genes that control <span class="emphasis"><em>C. elegans</em></span> life-span and metabolism. Science <span class="emphasis"><em>300</em></span>, 644-647.</span>
                     <a href="http://dx.doi.org/10.1126/science.1083614" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib127"></a><p class="bibliomixed"><span class="bibliomisc">Lei, H., Fukushige, T., Niu, W., Sarov, M., Reinke, V., and Krause, M. (2010). A widespread distribution of genomic CeMyoD
                        binding sites revealed and cross validated by ChIP-Chip and ChIP-Seq techniques. PLoS One <span class="emphasis"><em>5</em></span>, e15898.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21209968&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pone.0015898" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib128"></a><p class="bibliomixed"><span class="bibliomisc">Leight, E.R., Glossip, D., and Kornfeld, K. (2005). Sumoylation of <a href="http://www.wormbase.org/db/get?name=LIN-1;class=Protein" target="_blank">LIN-1</a> promotes transcriptional repression and inhibition of vulval cell fates. Development <span class="emphasis"><em>132</em></span>, 10471056.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15689373&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib129"></a><p class="bibliomixed"><span class="bibliomisc">Levine, M., and Tjian, R. (2003). Transcription regulation and animal diversity. Nature <span class="emphasis"><em>424</em></span>, 147-151.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12853946&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature01763" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib130"></a><p class="bibliomixed"><span class="bibliomisc">Li, T., and Kelly, W.G. (2011). A role for Set1/MLL-related components in epigenetic regulation of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> germ line. PLoS Genet. <span class="emphasis"><em>7</em></span>, e1001349.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21455483&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1001349" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib131"></a><p class="bibliomixed"><span class="bibliomisc">Li, W., Gao, B., Lee, S.M., Bennett, K., and Fang, D. (2007). <a href="http://www.wormbase.org/db/get?name=RLE-1;class=Protein" target="_blank">RLE-1</a>, an E3 ubiquitin ligase, regulates <span class="emphasis"><em>C. elegans</em></span> aging by catalyzing <a href="http://www.wormbase.org/db/get?name=DAF-16;class=Protein" target="_blank">DAF-16</a> polyubiquitination. Dev. Cell <span class="emphasis"><em>12</em></span>, 235-246.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17276341&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2006.12.002" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib132"></a><p class="bibliomixed"><span class="bibliomisc">Lichtsteiner, S., and Tjian, R. (1993). Cloning and properties of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> TATA-box-binding protein. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>90</em></span>, 9673-9677.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8415761&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.90.20.9673" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib133"></a><p class="bibliomixed"><span class="bibliomisc">Lin, K., Hsin, H., Libina, N., and Kenyon, C. (2001). Regulation of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> longevity protein <a href="http://www.wormbase.org/db/get?name=DAF-16;class=Protein" target="_blank">DAF-16</a> by insulin/IGF-1 and germline signaling. Nat. Genet. <span class="emphasis"><em>28</em></span>, 139-145.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11381260&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/88850" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib134"></a><p class="bibliomixed"><span class="bibliomisc">Liu, J., and Fire, A. (2000). Overlapping roles of two Hox genes and the <span class="emphasis"><em>exd</em></span> ortholog <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ceh-20;class=Gene" target="_blank">ceh-20</a></em></span> in diversification of the <span class="emphasis"><em>C. elegans</em></span> postembryonic mesoderm. Development <span class="emphasis"><em>127</em></span>, 5179-5190.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11060243&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib135"></a><p class="bibliomixed"><span class="bibliomisc">Liu, T., Rechtsteiner, A., Egelhofer, T.A., Vielle, A., Latorre, I., Cheung, M.S., Ercan, S., Ikegami, K., Jensen, M., Kolasinska-Zwierz,
                        P.<span class="emphasis"><em>, et al.</em></span> (2011). Broad chromosomal domains of histone modification patterns in <span class="emphasis"><em>C. elegans</em></span>. Genome Res. <span class="emphasis"><em>21</em></span>, 227-236.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177964&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.115519.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib136"></a><p class="bibliomixed"><span class="bibliomisc">Liu, X., Long, F., Peng, H., Aerni, S.J., Jiang, M., Sanchez-Blanco, A., Murray, J.I., Preston, E., Mericle, B., Batzoglou,
                        S.<span class="emphasis"><em>, et al.</em></span> (2009). Analysis of cell fate from single-cell gene expression profiles in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>139</em></span>, 623-633.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19879847&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2009.08.044" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib137"></a><p class="bibliomixed"><span class="bibliomisc">MacMorris, M., Broverman, S., Greenspoon, S., Lea, K., Madej, C., Blumenthal, T., and Spieth, J. (1992). Regulation of vitellogenin
                        gene expression in transgenic <span class="emphasis"><em>Caenorhabditis elegans</em></span>: short sequences required for activation of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=vit-2;class=Gene" target="_blank">vit-2</a></em></span> promoter. Mol. Cell. Biol. <span class="emphasis"><em>12</em></span>, 1652-1662.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1549118&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib138"></a><p class="bibliomixed"><span class="bibliomisc">MacMorris, M., Spieth, J., Madej, C., Lea, K., and Blumenthal, T. (1994). Analysis of the VPE sequences in the <span class="emphasis"><em>Caenorhabditis elegans <a href="http://www.wormbase.org/db/get?name=vit-2;class=Gene" target="_blank">vit-2</a></em></span> promoter with extrachromosomal tandem array-containing transgenic strains. Mol. Cell. Biol. <span class="emphasis"><em>14</em></span>, 484-491.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8264616&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib139"></a><p class="bibliomixed"><span class="bibliomisc">Maduro, M.F., Broitman-Maduro, G., Mengarelli, I., and Rothman, J.H. (2007). Maternal deployment of the embryonic SKN-1&#8211;&gt;<a href="http://www.wormbase.org/db/get?name=MED-1;class=Protein" target="_blank">MED-1</a>,2 cell specification pathway in <span class="emphasis"><em>C. elegans</em></span>. Dev. Biol. <span class="emphasis"><em>301</em></span>, 590-601.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16979152&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2006.08.029" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib141"></a><p class="bibliomixed"><span class="bibliomisc">Maduro, M.F., Meneghini, M.D., Bowerman, B., Broitman-Maduro, G., and Rothman, J.H. (2001). Restriction of mesendoderm to
                        a single blastomere by the combined action of <a href="http://www.wormbase.org/db/get?name=SKN-1;class=Protein" target="_blank">SKN-1</a> and a GSK-3&#946; homolog is mediated by <a href="http://www.wormbase.org/db/get?name=MED-1;class=Protein" target="_blank">MED-1</a> and -2 in <span class="emphasis"><em>C. elegans</em></span>. Mol. Cell <span class="emphasis"><em>7</em></span>, 475-485.</span>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(01)00195-2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib143"></a><p class="bibliomixed"><span class="bibliomisc">Mah, A.K., Armstrong, K.R., Chew, D.S., Chu, J.S., Tu, D.K., Johnsen, R.C., Chen, N., Chamberlin, H.M., and Baillie, D.L.
                        (2007). Transcriptional regulation of <a href="http://www.wormbase.org/db/get?name=AQP-8;class=Protein" target="_blank">AQP-8</a>, a <span class="emphasis"><em>Caenorhabditis elegans</em></span> aquaporin exclusively expressed in the excretory system, by the POU homeobox transcription factor <a href="http://www.wormbase.org/db/get?name=CEH-6;class=Protein" target="_blank">CEH-6</a>. J. Biol. Chem <span class="emphasis"><em>282</em></span>, 28074-28086.</span>
                     <a href="http://dx.doi.org/10.1074/jbc.M703305200" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib144"></a><p class="bibliomixed"><span class="bibliomisc">Mango, S.E. The <span class="emphasis"><em>C. elegans</em></span> pharynx: a model for organogenesis (January 22, 2007), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.129.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.129.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib145"></a><p class="bibliomixed"><span class="bibliomisc">Mangone, M., Manoharan, A.P., Thierry-Mieg, D., Thierry-Mieg, J., Han, T., Mackowiak, S.D., Mis, E., Zegar, C., Gutwein, M.R.,
                        Khivansara, V.<span class="emphasis"><em>, et al.</em></span> (2010). The landscape of <span class="emphasis"><em>C. elegans</em></span> 3'UTRs. Science <span class="emphasis"><em>329</em></span>, 432-435.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20522740&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1191244" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib146"></a><p class="bibliomixed"><span class="bibliomisc">Matthews, J.M., and Sunde, M. (2002). Zinc fingers&#8211;folds for many occasions. IUBMB Life <span class="emphasis"><em>54</em></span>, 351-355.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12665246&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1080/15216540216035" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib147"></a><p class="bibliomixed"><span class="bibliomisc">McElwee, J., Bubb, K., and Thomas, J.H. (2003). Transcriptional outputs of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> forkhead protein <a href="http://www.wormbase.org/db/get?name=DAF-16;class=Protein" target="_blank">DAF-16</a>. Aging Cell <span class="emphasis"><em>2</em></span>, 111-121.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12882324&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1046/j.1474-9728.2003.00043.x" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib148"></a><p class="bibliomixed"><span class="bibliomisc">McGhee, J.D. The <span class="emphasis"><em>C. elegans</em></span> intestine (March 27, 2007), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.133.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.133.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib149"></a><p class="bibliomixed"><span class="bibliomisc">McGhee, J.D., and Krause, M.W. (1997). Transcription Factors and Transcriptional Regulation. In <span class="emphasis"><em>C. elegans</em></span> II, D.L. Riddle, T. Blumenthal, B.J. Meyer, and J.R. Priess, eds. (Cold Spring Harbor, NY, CSHL Press), pp. 147-184.</span>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib150"></a><p class="bibliomixed"><span class="bibliomisc">McGhee, J.D., Sleumer, M.C., Bilenky, M., Wong, K., McKay, S.J., Goszczynski, B., Tian, H., Krich, N.D., Khattra, J., Holt,
                        R.A.<span class="emphasis"><em>, et al.</em></span> (2007). The <a href="http://www.wormbase.org/db/get?name=ELT-2;class=Protein" target="_blank">ELT-2</a> GATA-factor and the global regulation of transcription in the <span class="emphasis"><em>C. elegans</em></span> intestine. Dev. Biol. <span class="emphasis"><em>302</em></span>, 627-645.</span>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2006.10.024" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib151"></a><p class="bibliomixed"><span class="bibliomisc">McKay, S.J., Johnsen, R., Khattra, J., Asano, J., Baillie, D.L., Chan, S., Dube, N., Fang, L., Goszczynski, B., Ha, E.<span class="emphasis"><em>, et al.</em></span> (2003). Gene expression profiling of cells, tissues, and developmental stages of the nematode <span class="emphasis"><em>C. elegans</em></span>. Cold Spring Harb. Symp. Quant. Biol. <span class="emphasis"><em>68</em></span>, 159-169.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15338614&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/sqb.2003.68.159" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib152"></a><p class="bibliomixed"><span class="bibliomisc">Meissner, B., Warner, A., Wong, K., Dube, N., Lorch, A., McKay, S.J., Khattra, J., Rogalski, T., Somasiri, A., Chaudhry, I.<span class="emphasis"><em>, et al.</em></span> (2009). An integrated strategy to study muscle development and myofilament structure in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. PLoS Genet. <span class="emphasis"><em>5</em></span>, e1000537.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19557190&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000537" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib153"></a><p class="bibliomixed"><span class="bibliomisc">Meister, P., Towbin, B.D., Pike, B.L., Ponti, A., and Gasser, S.M. (2010). The spatial dynamics of tissue-specific promoters
                        during <span class="emphasis"><em>C. elegans</em></span> development. Genes Dev. <span class="emphasis"><em>24</em></span>, 766-782.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20395364&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.559610" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib154"></a><p class="bibliomixed"><span class="bibliomisc">Metzstein, M.M., and Horvitz, H.R. (1999). The <span class="emphasis"><em>C. elegans</em></span> cell death specification gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ces-1;class=Gene" target="_blank">ces-1</a></em></span> encodes a snail family zinc finger protein. Mol. Cell <span class="emphasis"><em>4</em></span>, 309-319.</span>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)80333-0" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib155"></a><p class="bibliomixed"><span class="bibliomisc">Meyer, B.J. (2010). Targeting X chromosomes for repression.  Curr. Opin. Genet. Dev. <span class="emphasis"><em>20</em></span>,179-189.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20381335&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.gde.2010.03.008" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib156"></a><p class="bibliomixed"><span class="bibliomisc">Miley, G.R., Fantz, D., Glossip, D., Lu, X., Saito, R.M., Palmer, R.E., Inoue, T., Van Den Heuvel, S., Sternberg, P.W., and
                        Kornfeld, K. (2004). Identification of residues of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> <a href="http://www.wormbase.org/db/get?name=LIN-1;class=Protein" target="_blank">LIN-1</a> ETS domain that are necessary for DNA binding and regulation of vulval cell fates. Genetics <span class="emphasis"><em>167</em></span>, 1697-1709.</span>
                     <a href="http://dx.doi.org/10.1534/genetics.104.029017" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib157"></a><p class="bibliomixed"><span class="bibliomisc">Miller, R.R., and Okkema, P.G. (2011). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> T-box factor MLS1 requires Groucho co-repressor interaction for uterine muscle specification. PLoS Genet. <span class="emphasis"><em>7</em></span>, e1002210.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21852953&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002210" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib158"></a><p class="bibliomixed"><span class="bibliomisc">Moilanen, L.H., Fukushige, T., and Freedman, J.H. (1999). Regulation of metallothionein gene transcription. Identification
                        of upstream regulatory elements and transcription factors responsible for cell-specific expression of the metallothionein
                        genes from <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Biol. Chem <span class="emphasis"><em>274</em></span>, 29655-29665.</span>
                     <a href="http://dx.doi.org/10.1074/jbc.274.42.29655" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib159"></a><p class="bibliomixed"><span class="bibliomisc">Morton, J.J., and Blumenthal, T. (2011). Identification of transcription start sites of trans-spliced genes: uncovering unusual
                        operon arrangements. RNA <span class="emphasis"><em>17</em></span>, 327-337.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21156961&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1261/rna.2447111" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib160"></a><p class="bibliomixed"><span class="bibliomisc">Muller, J., and Kassis, J.A. (2006). Polycomb response elements and targeting of Polycomb group proteins in <span class="emphasis"><em>Drosophila</em></span>. Curr. Opin. Genet. Dev. <span class="emphasis"><em>16</em></span>, 476-484.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16914306&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.gde.2006.08.005" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib161"></a><p class="bibliomixed"><span class="bibliomisc">Murray, J.I., Bao, Z., Boyle, T.J., Boeck, M.E., Mericle, B.L., Nicholas, T.J., Zhao, Z., Sandel, M.J., and Waterston, R.H.
                        (2008). Automated analysis of embryonic gene expression with cellular resolution in <span class="emphasis"><em>C. elegans</em></span>. Nat. Methods <span class="emphasis"><em>5</em></span>, 703-709.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18587405&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1228" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib162"></a><p class="bibliomixed"><span class="bibliomisc">Nam, S., Jin, Y.H., Li, Q.L., Lee, K.Y., Jeong, G.B., Ito, Y., Lee, J., and Bae, S.C. (2002). Expression pattern, regulation,
                        and biological role of runt domain transcription factor, run, in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Cell. Biol. <span class="emphasis"><em>22</em></span>, 547-554.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11756550&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.22.2.547-554.2002" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib163"></a><p class="bibliomixed"><span class="bibliomisc">Nash, B., Colavita, A., Zheng, H., Roy, P.J., and Culotti, J.G. (2000). The forkhead transcription factor <a href="http://www.wormbase.org/db/get?name=UNC-130;class=Protein" target="_blank">UNC-130</a> is required for the graded spatial expression of the UNC129 TGF-&#946; guidance factor in <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>14</em></span>, 2486-2500.</span>
                     <a href="http://dx.doi.org/10.1101/gad.831500" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib164"></a><p class="bibliomixed"><span class="bibliomisc">Neves, A., English, K., and Priess, J.R. (2007). Notch-GATA synergy promotes endoderm-specific expression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=ref-1;class=Gene" target="_blank">ref-1</a></em></span> in <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>134</em></span>, 4459-4468.</span>
                     <a href="http://dx.doi.org/10.1242/dev.008680" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib165"></a><p class="bibliomixed"><span class="bibliomisc">Neves, A., and Priess, J.R. (2005). The <a href="http://www.wormbase.org/db/get?name=REF-1;class=Protein" target="_blank">REF-1</a> family of bHLH transcription factors pattern <span class="emphasis"><em>C. elegans</em></span> embryos through Notch-dependent and Notch-independent pathways. Dev. Cell <span class="emphasis"><em>8</em></span>, 867-879.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15935776&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2005.03.012" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib166"></a><p class="bibliomixed"><span class="bibliomisc">Niu, W., Lu, Z.J., Zhong, M., Sarov, M., Murray, J.I., Brdlik, C.M., Janette, J., Chen, C., Alves, P., Preston, E.<span class="emphasis"><em>, et al.</em></span> (2011). Diverse transcription factor binding features revealed by genome-wide ChIP-seq in <span class="emphasis"><em>C. elegans</em></span>. Genome Res. <span class="emphasis"><em>21</em></span>, 245-254.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177963&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.114587.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib167"></a><p class="bibliomixed"><span class="bibliomisc">O'Meara, M.M., Bigelow, H., Flibotte, S., Etchberger, J.F., Moerman, D.G., and Hobert, O. (2009). Cis-regulatory mutations
                        in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> homeobox gene locus <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=cog-1;class=Gene" target="_blank">cog-1</a></em></span> affect neuronal development. Genetics <span class="emphasis"><em>181</em></span>, 1679-1686.</span>
                     <a href="http://dx.doi.org/10.1534/genetics.108.097832" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib168"></a><p class="bibliomixed"><span class="bibliomisc">Oh, S.W., Mukhopadhyay, A., Dixit, B.L., Raha, T., Green, M.R., and Tissenbaum, H.A. (2006). Identification of direct <a href="http://www.wormbase.org/db/get?name=DAF-16;class=Protein" target="_blank">DAF-16</a> targets controlling longevity, metabolism and diapause by chromatin immunoprecipitation. Nat. Genet. <span class="emphasis"><em>38</em></span>, 251-257.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16380712&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib169"></a><p class="bibliomixed"><span class="bibliomisc">Okkema, P.G., and Fire, A. (1994). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> NK-2 class homeoprotein <a href="http://www.wormbase.org/db/get?name=CEH-22;class=Protein" target="_blank">CEH-22</a> is involved in combinatorial activation of gene expression in pharyngeal muscle. Development <span class="emphasis"><em>120</em></span>, 2175-2186.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7925019&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib170"></a><p class="bibliomixed"><span class="bibliomisc">Okkema, P.G., Harrison, S.W., Plunger, V., Aryana, A., and Fire, A. (1993). Sequence requirements for myosin gene expression
                        and regulation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>135</em></span>, 385-404.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8244003&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib171"></a><p class="bibliomixed"><span class="bibliomisc">Okkema, P. G. and Krause, M. Transcriptional regulation (December 23, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.45.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.45.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib172"></a><p class="bibliomixed"><span class="bibliomisc">Oliveira, R.P., Porter Abate, J., Dilks, K., Landis, J., Ashraf, J., Murphy, C.T., and Blackwell, T.K. (2009). Condition-adapted
                        stress and longevity gene regulation by <span class="emphasis"><em>Caenorhabditis elegans</em></span> <a href="http://www.wormbase.org/db/get?name=SKN-1;class=Protein" target="_blank">SKN-1</a>/Nrf. Aging Cell <span class="emphasis"><em>8</em></span>, 524-541.</span>
                     <a href="http://dx.doi.org/10.1111/j.1474-9726.2009.00501.x" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib173"></a><p class="bibliomixed"><span class="bibliomisc">Oommen, K.S., and Newman, A.P. (2007). Co-regulation by Notch and Fos is required for cell fate specification of intermediate
                        precursors during <span class="emphasis"><em>C. elegans</em></span> uterine development. Development <span class="emphasis"><em>134</em></span>, 3999-4009.</span>
                     <a href="http://dx.doi.org/10.1242/dev.002741" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib174"></a><p class="bibliomixed"><span class="bibliomisc">Ow, M.C., Martinez, N.J., Olsen, P.H., Silverman, H.S., Barrasa, M.I., Conradt, B., Walhout, A.J., and Ambros, V. (2008).
                        The FLYWCH transcription factors FLH-1, FLH-2, and FLH-3 repress embryonic expression of microRNA genes in <span class="emphasis"><em>C. elegans</em></span>. Genes Dev. <span class="emphasis"><em>22</em></span>, 2520-2534.</span>
                     <a href="http://dx.doi.org/10.1101/gad.1678808" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib175"></a><p class="bibliomixed"><span class="bibliomisc">Park, M., and Krause, M.W. (1999). Regulation of postembryonic G(1) cell cycle progression in <span class="emphasis"><em>Caenorhabditis elegans</em></span> by a cyclin D/CDK-like complex. Development <span class="emphasis"><em>126</em></span>, 4849-4860.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10518501&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib176"></a><p class="bibliomixed"><span class="bibliomisc">Passannante, M., Marti, C.O., Pfefferli, C., Moroni, P.S., Kaeser-Pebernard, S., Puoti, A., Hunziker, P., Wicky, C., and Muller,
                        F. (2010). Different Mi-2 complexes for various developmental functions in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. PLoS One <span class="emphasis"><em>5</em></span>, e13681.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21060680&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pone.0013681" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib177"></a><p class="bibliomixed"><span class="bibliomisc">Peden, E., Kimberly, E., Gengyo-Ando, K., Mitani, S., and Xue, D. (2007). Control of sex-specific apoptosis in <span class="emphasis"><em>C. elegans</em></span> by the BarH homeodomain protein <a href="http://www.wormbase.org/db/get?name=CEH-30;class=Protein" target="_blank">CEH-30</a> and the transcriptional repressor <a href="http://www.wormbase.org/db/get?name=UNC-37;class=Protein" target="_blank">UNC-37</a>/Groucho. Genes Dev. <span class="emphasis"><em>21</em></span>, 3195-3207.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18056429&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1607807" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib178"></a><p class="bibliomixed"><span class="bibliomisc">Petrella, L.N., Wang, W., Spike, C.A., Rechtsteiner, A., Reinke, V., and Strome, S. (2011). synMuv B proteins antagonize germline
                        fate in the intestine and ensure <span class="emphasis"><em>C. elegans</em></span> survival. Development <span class="emphasis"><em>138</em></span>, 1069-1079.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21343362&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.059501" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib179"></a><p class="bibliomixed"><span class="bibliomisc">Pferdehirt, R.R., Kruesi, W.S., and Meyer, B.J. (2011). An MLL/COMPASS subunit functions in the <span class="emphasis"><em>C. elegans</em></span> dosage compensation complex to target X chromosomes for transcriptional regulation of gene expression. Genes Dev. <span class="emphasis"><em>25</em></span>, 499-515.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21363964&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.2016011" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib180"></a><p class="bibliomixed"><span class="bibliomisc">Phatnani, H.P., and Greenleaf, A.L. (2006). Phosphorylation and functions of the RNA polymerase II CTD. Genes Dev. <span class="emphasis"><em>20</em></span>, 2922-2936.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17079683&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1477006" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib181"></a><p class="bibliomixed"><span class="bibliomisc">Pocock, R., Ahringer, J., Mitsch, M., Maxwell, S., and Woollard, A. (2004). A regulatory network of T-box genes and the even-skipped
                        homologue <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=vab-7;class=Gene" target="_blank">vab-7</a></em></span> controls patterning and morphogenesis in <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>131</em></span>, 2373-2385.</span>
                     <a href="http://dx.doi.org/10.1242/dev.01110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib182"></a><p class="bibliomixed"><span class="bibliomisc">Poole, R.J., Bashllari, E., Cochella, L., Flowers, E.B., and Hobert, O. (2011).  A genome-wide RNAi screen for factors involved
                        in neuronal specification in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. PLoS Genet. <span class="emphasis"><em>7</em></span>, e1002109.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21698137&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002109" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib183"></a><p class="bibliomixed"><span class="bibliomisc">Powell-Coffman, J.A., Bradfield, C.A., and Wood, W.B. (1998). <span class="emphasis"><em>Caenorhabditis elegans</em></span> orthologs of the aryl hydrocarbon receptor and its heterodimerization partner the aryl hydrocarbon receptor nuclear translocator.
                        Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>95</em></span>, 2844-2849.</span>
                     <a href="http://dx.doi.org/10.1073/pnas.95.6.2844" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib184"></a><p class="bibliomixed"><span class="bibliomisc">Raharjo, W.H., Logan, B.C., Wen, S., Kalb, J.M., and Gaudet, J. (2010). In vitro and in vivo characterization of <span class="emphasis"><em>Caenorhabditis elegans</em></span> <a href="http://www.wormbase.org/db/get?name=PHA-4;class=Protein" target="_blank">PHA-4</a>/FoxA response elements. Dev. Dyn. <span class="emphasis"><em>239</em></span>, 2219-2232.</span>
                     <a href="http://dx.doi.org/10.1002/dvdy.22359" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib185"></a><p class="bibliomixed"><span class="bibliomisc">Reece-Hoyes, J.S., Deplancke, B., Barrasa, M.I., Hatzold, J., Smit, R.B., Arda, H.E., Pope, P.A., Gaudet, J., Conradt, B.,
                        and Walhout, A.J. (2009). The <span class="emphasis"><em>C. elegans</em></span> Snail homolog <a href="http://www.wormbase.org/db/get?name=CES-1;class=Protein" target="_blank">CES-1</a> can activate gene expression <span class="emphasis"><em>in vivo</em></span> and share targets with bHLH transcription factors. Nucleic Acids Res. <span class="emphasis"><em>37</em></span>, 3689-3698.</span>
                     <a href="http://dx.doi.org/10.1093/nar/gkp232" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib186"></a><p class="bibliomixed"><span class="bibliomisc">Reece-Hoyes, J.S., Deplancke, B., Shingles, J., Grove, C.A., Hope, I.A., and Walhout, A.J. (2005). A compendium of <span class="emphasis"><em>Caenorhabditis elegans</em></span> regulatory transcription factors: a resource for mapping transcription regulatory networks. Genome Biol. <span class="emphasis"><em>6</em></span>, R110.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16420670&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2005-6-13-r110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib187"></a><p class="bibliomixed"><span class="bibliomisc">Reece-Hoyes, J.S., Diallo, A., Lajoie, B., Kent, A., Shrestha, S., Kadreppa, S., Pesyna, C., Dekker, J., Myers, C.L., and
                        Walhout, A.J. (2011). Enhanced yeast one-hybrid assays for high-throughput gene-centered regulatory network mapping. Nat.
                        Methods <span class="emphasis"><em>8</em></span>, 1059-1064.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=22037705&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nmeth.1748" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib188"></a><p class="bibliomixed"><span class="bibliomisc">Reinke, V. Germline genomics (January 20, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.74.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.74.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib189"></a><p class="bibliomixed"><span class="bibliomisc">Ross, J.M., and Zarkower, D. (2003). Polycomb group regulation of Hox gene expression in <span class="emphasis"><em>C. elegans</em></span>. Dev. Cell <span class="emphasis"><em>4</em></span>, 891-901.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12791273&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1534-5807(03)00135-7" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib190"></a><p class="bibliomixed"><span class="bibliomisc">Rougvie, A.E., and Ambros, V. (1995). The heterochronic gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-29;class=Gene" target="_blank">lin-29</a></em></span> encodes a zinc finger protein that controls a terminal differentiation event in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>121</em></span>, 2491-2500.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7671813&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib191"></a><p class="bibliomixed"><span class="bibliomisc">Roush, S.F., and Slack, F.J. (2009). Transcription of the <span class="emphasis"><em>C. elegans <a href="http://www.wormbase.org/db/get?name=let-7;class=Gene" target="_blank">let-7</a></em></span> microRNA is temporally regulated by one of its targets, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=hbl-1;class=Gene" target="_blank">hbl-1</a></em></span>. Dev. Biol. <span class="emphasis"><em>334</em></span>, 523-534.</span>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2009.07.012" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib192"></a><p class="bibliomixed"><span class="bibliomisc">Roy Chowdhuri, S., Crum, T., Woollard, A., Aslam, S., and Okkema, P.G. (2006). The T-box factor <a href="http://www.wormbase.org/db/get?name=TBX-2;class=Protein" target="_blank">TBX-2</a> and the SUMO conjugating enzyme <a href="http://www.wormbase.org/db/get?name=UBC-9;class=Protein" target="_blank">UBC-9</a> are required for ABa-derived pharyngeal muscle in <span class="emphasis"><em>C. elegans</em></span>. Dev. Biol. <span class="emphasis"><em>295</em></span>, 664-677.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16701625&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2006.04.001" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib193"></a><p class="bibliomixed"><span class="bibliomisc">Roy, P.J., Stuart, J.M., Lund, J., and Kim, S.K. (2002). Chromosomal clustering of muscle-expressed genes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>418</em></span>, 975-979.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12214599&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib194"></a><p class="bibliomixed"><span class="bibliomisc">Roy, S., Ernst, J., Kharchenko, P.V., Kheradpour, P., Negre, N., Eaton, M.L., Landolin, J.M., Bristow, C.A., Ma, L., Lin,
                        M.F.<span class="emphasis"><em>, et al.</em></span> (2010). Identification of functional elements and regulatory circuits by <span class="emphasis"><em>Drosophila</em></span> modENCODE. Science <span class="emphasis"><em>330</em></span>, 1787-1797.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177974&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib196"></a><p class="bibliomixed"><span class="bibliomisc">Russnak, R.H., and Candido, E.P. (1985). Locus encoding a family of small heat shock genes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: two genes duplicated to form a 3.8-kilobase inverted repeat. Mol. Cell. Biol. <span class="emphasis"><em>5</em></span>, 1268-1278.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4033652&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib197"></a><p class="bibliomixed"><span class="bibliomisc">Schafer, J.C., Haycraft, C.J., Thomas, J.H., Yoder, B.K., and Swoboda, P. (2003). <a href="http://www.wormbase.org/db/get?name=XBX-1;class=Protein" target="_blank">XBX-1</a> encodes a dynein light intermediate chain required for retrograde intraflagellar transport and cilia assembly in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Biol. Cell <span class="emphasis"><em>14</em></span>, 2057-2070.</span>
                     <a href="http://dx.doi.org/10.1091/mbc.E02-10-0677" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib198"></a><p class="bibliomixed"><span class="bibliomisc">Schaner, C. E. and Kelly, W. G. Germline chromatin (January 24, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.73.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.73.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib199"></a><p class="bibliomixed"><span class="bibliomisc">Schvarzstein, M., and Spence, A.M. (2006). The <span class="emphasis"><em>C. elegans</em></span> sex-determining GLI protein TRA-1A is regulated by sex-specific proteolysis. Dev. Cell <span class="emphasis"><em>11</em></span>, 733-740.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17084364&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2006.09.017" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib200"></a><p class="bibliomixed"><span class="bibliomisc">Seydoux, G., and Dunn, M.A. (1997). Transcriptionally repressed germ cells lack a subpopulation of phosphorylated RNA polymerase
                        II in early embryos of <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Drosophila melanogaster</em></span>. Development <span class="emphasis"><em>124</em></span>, 2191-2201.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9187145&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib201"></a><p class="bibliomixed"><span class="bibliomisc">Shaham, S., and Bargmann, C.I. (2002). Control of neuronal subtype identity by the <span class="emphasis"><em>C. elegans</em></span> ARID protein <a href="http://www.wormbase.org/db/get?name=CFI-1;class=Protein" target="_blank">CFI-1</a>. Genes Dev. <span class="emphasis"><em>16</em></span>, 972-983.</span>
                     <a href="http://dx.doi.org/10.1101/gad.976002" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib202"></a><p class="bibliomixed"><span class="bibliomisc">Shaye, D.D., and Greenwald, I. (2011). OrthoList: a compendium of <span class="emphasis"><em>C. elegans</em></span> genes with human orthologs. PLoS One <span class="emphasis"><em>6</em></span>, e20085.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21647448&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pone.0020085" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib203"></a><p class="bibliomixed"><span class="bibliomisc">Shemer, G., and Podbilewicz, B. (2002). <a href="http://www.wormbase.org/db/get?name=LIN-39;class=Protein" target="_blank">LIN-39</a>/Hox triggers cell division and represses <a href="http://www.wormbase.org/db/get?name=EFF-1;class=Protein" target="_blank">EFF-1</a>/fusogen-dependent vulval cell fusion. Genes Dev. <span class="emphasis"><em>16</em></span>, 3136-3141.</span>
                     <a href="http://dx.doi.org/10.1101/gad.251202" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib204"></a><p class="bibliomixed"><span class="bibliomisc">Shi, Y., and Mello, C. (1998). A CBP/p300 homolog specifies multiple differentiation pathways in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genes Dev. <span class="emphasis"><em>12</em></span>, 943-955.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9531533&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.12.7.943" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib205"></a><p class="bibliomixed"><span class="bibliomisc">Shilatifard, A. (2008). Molecular implementation and physiological roles for histone H3 lysine 4 (H3K4) methylation. Curr.
                        Opin. Cell Biol. <span class="emphasis"><em>20</em></span>, 341-348.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18508253&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib206"></a><p class="bibliomixed"><span class="bibliomisc">Shim, J., and Lee, J. (2008). Regulation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=rnt-1;class=Gene" target="_blank">rnt-1</a></em></span> expression mediated by the opposing effects of <a href="http://www.wormbase.org/db/get?name=BRO-1;class=Protein" target="_blank">BRO-1</a> and <a href="http://www.wormbase.org/db/get?name=DBL-1;class=Protein" target="_blank">DBL-1</a> in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Biochem.  Biophys. Res. Commun. <span class="emphasis"><em>367</em></span>, 130-136.</span>
                     <a href="http://dx.doi.org/10.1016/j.bbrc.2007.12.097" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib207"></a><p class="bibliomixed"><span class="bibliomisc">Shim, Y.H. (1999). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=elt-1;class=Gene" target="_blank">elt-1</a></em></span>, a gene encoding a <span class="emphasis"><em>Caenorhabditis elegans</em></span> GATA transcription factor, is highly expressed in the germ lines with <span class="emphasis"><em>msp</em></span> genes as the potential targets. Mol. Cells <span class="emphasis"><em>9</em></span>, 535-541.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10597043&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib208"></a><p class="bibliomixed"><span class="bibliomisc">Shim, Y.H., Bonner, J.J., and Blumenthal, T. (1995). Activity of a <span class="emphasis"><em>C. elegans</em></span> GATA transcription factor, <a href="http://www.wormbase.org/db/get?name=ELT-1;class=Protein" target="_blank">ELT-1</a>, expressed in yeast. J. Mol. Biol. <span class="emphasis"><em>253</em></span>, 665-676.</span>
                     <a href="http://dx.doi.org/10.1006/jmbi.1995.0581" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib209"></a><p class="bibliomixed"><span class="bibliomisc">Shoichet, S.A., Malik, T.H., Rothman, J.H., and Shivdasani, R.A. (2000). Action of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> GATA factor <a href="http://www.wormbase.org/db/get?name=END-1;class=Protein" target="_blank">END-1</a> in <span class="emphasis"><em>Xenopus</em></span> suggests that similar mechanisms initiate endoderm development in ecdysozoa and vertebrates. Proc. Natl. Acad. Sci. U. S.
                        A. <span class="emphasis"><em>97</em></span>, 4076-4081.</span>
                     <a href="http://dx.doi.org/10.1073/pnas.97.8.4076" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib210"></a><p class="bibliomixed"><span class="bibliomisc">Shostak, Y., Van Gilst, M.R., Antebi, A., and Yamamoto, K.R. (2004). Identification of <span class="emphasis"><em>C. elegans</em></span> DAF-12-binding sites, response elements, and target genes. Genes Dev. <span class="emphasis"><em>18</em></span>, 2529-2544.</span>
                     <a href="http://dx.doi.org/10.1101/gad.1218504" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib211"></a><p class="bibliomixed"><span class="bibliomisc">Simonet, T., Dulermo, R., Schott, S., and Palladino, F. (2007). Antagonistic functions of <a href="http://www.wormbase.org/db/get?name=SET-2;class=Protein" target="_blank">SET-2</a>/SET1 and HPL/HP1 proteins in <span class="emphasis"><em>C. elegans</em></span> development. Dev. Biol. <span class="emphasis"><em>312</em></span>, 367-383.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17967446&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2007.09.035" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib212"></a><p class="bibliomixed"><span class="bibliomisc">Sluder, A.E., Mathews, S.W., Hough, D., Yin, V.P., and Maina, C.V. (1999). The nuclear receptor superfamily has undergone
                        extensive proliferation and diversification in nematodes. Genome Res. <span class="emphasis"><em>9</em></span>, 103-120.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10022975&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib213"></a><p class="bibliomixed"><span class="bibliomisc">Spencer, W.C., Zeller, G., Watson, J.D., Henz, S.R., Watkins, K.L., McWhirter, R.D., Petersen, S., Sreedharan, V.T., Widmer,
                        C., Jo, J.<span class="emphasis"><em>, et al.</em></span> (2011). A spatial and temporal map of <span class="emphasis"><em>C. elegans</em></span> gene expression. Genome Res. <span class="emphasis"><em>21</em></span>, 325-341.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21177967&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.114595.110" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib214"></a><p class="bibliomixed"><span class="bibliomisc">Spieth, J., Brooke, G., Kuersten, S., Lea, K., and Blumenthal, T. (1993). Operons in <span class="emphasis"><em>C. elegans</em></span>: polycistronic mRNA precursors are processed by trans-splicing of SL2 to downstream coding regions. Cell <span class="emphasis"><em>73</em></span>, 521-532.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8098272&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(93)90139-H" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib215"></a><p class="bibliomixed"><span class="bibliomisc">Spieth, J., Denison, K., Kirtland, S., Cane, J., and Blumenthal, T. (1985). The <span class="emphasis"><em>C. elegans</em></span> vitellogenin genes: short sequence repeats in the promoter regions and homology to the vertebrate genes. Nucleic Acids Res.
                        <span class="emphasis"><em>13</em></span>, 5283-5295.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4022780&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/13.14.5283" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib216"></a><p class="bibliomixed"><span class="bibliomisc">Spieth, J. and Lawson, D. Overview of gene structure (January 18, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.65.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.65.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib217"></a><p class="bibliomixed"><span class="bibliomisc">Starostina, N.G., Lim, J.M., Schvarzstein, M., Wells, L., Spence, A.M., and Kipreos, E.T. (2007). A <a href="http://www.wormbase.org/db/get?name=CUL-2;class=Protein" target="_blank">CUL-2</a> ubiquitin ligase containing three FEM proteins degrades <a href="http://www.wormbase.org/db/get?name=TRA-1;class=Protein" target="_blank">TRA-1</a> to regulate <span class="emphasis"><em>C. elegans</em></span> sex determination. Dev. Cell <span class="emphasis"><em>13</em></span>, 127-139.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17609115&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2007.05.008" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib218"></a><p class="bibliomixed"><span class="bibliomisc">Stielow, B., Sapetschnig, A., Kruger, I., Kunert, N., Brehm, A., Boutros, M., and Suske, G. (2008). Identification of SUMO-dependent
                        chromatin-associated transcriptional repression components by a genome-wide RNAi screen. Mol. Cell <span class="emphasis"><em>29</em></span>, 742-754.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18374648&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.molcel.2007.12.032" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib220"></a><p class="bibliomixed"><span class="bibliomisc">Stimpson, K.M., and Sullivan, B.A. (2010). Epigenomics of centromere assembly and function.  Curr. Opin. Cell Biol. <span class="emphasis"><em>22</em></span>, 772-280.</span>
                     <a href="http://dx.doi.org/10.1016/j.ceb.2010.07.002" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib221"></a><p class="bibliomixed"><span class="bibliomisc">Stringham, E.G., Dixon, D.K., Jones, D., and Candido, E.P. (1992). Temporal and spatial expression patterns of the small heat
                        shock (<span class="emphasis"><em>hsp16</em></span>) genes in transgenic <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Biol. Cell <span class="emphasis"><em>3</em></span>, 221-233.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1550963&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib222"></a><p class="bibliomixed"><span class="bibliomisc">Strome, S. Specification of the germ line (July 28, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.9.1, http://www.wormbook.org.</span>
                     <a href="http://dx.doi.org/10.1895/wormbook.1.9.1" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib223"></a><p class="bibliomixed"><span class="bibliomisc">Tabuchi, T.M., Deplancke, B., Osato, N., Zhu, L.J., Barrasa, M.I., Harrison, M.M., Horvitz, H.R., Walhout, A.J., and Hagstrom,
                        K.A. (2011). Chromosome-biased binding and gene regulation by the <span class="emphasis"><em>Caenorhabditis elegans</em></span> DRM complex. PLoS Genet. <span class="emphasis"><em>7</em></span>, e1002074.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21589891&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1002074" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib224"></a><p class="bibliomixed"><span class="bibliomisc">Thatcher, J.D., Fernandez, A.P., Beaster-Jones, L., Haun, C., and Okkema, P.G. (2001). The <span class="emphasis"><em>Caenorhabditis elegans <a href="http://www.wormbase.org/db/get?name=peb-1;class=Gene" target="_blank">peb-1</a></em></span> gene encodes a novel DNA-binding protein involved in morphogenesis of the pharynx, vulva, and hindgut. Dev. Biol. <span class="emphasis"><em>229</em></span>, 480-493.</span>
                     <a href="http://dx.doi.org/10.1006/dbio.2000.9978" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib225"></a><p class="bibliomixed"><span class="bibliomisc">Thatcher, J.D., Haun, C., and Okkema, P.G. (1999). The <a href="http://www.wormbase.org/db/get?name=DAF-3;class=Protein" target="_blank">DAF-3</a> Smad binds DNA and represses gene expression in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> pharynx. Development <span class="emphasis"><em>126</em></span>, 97-107.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9834189&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib226"></a><p class="bibliomixed"><span class="bibliomisc">Thellmann, M., Hatzold, J., and Conradt, B. (2003). The Snail-like <a href="http://www.wormbase.org/db/get?name=CES-1;class=Protein" target="_blank">CES-1</a> protein of <span class="emphasis"><em>C. elegans</em></span> can block the expression of the BH3-only cell-death activator gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=egl-1;class=Gene" target="_blank">egl-1</a></em></span> by antagonizing the function of bHLH proteins. Development <span class="emphasis"><em>130</em></span>, 4057-4071.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12874127&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.00597" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib227"></a><p class="bibliomixed"><span class="bibliomisc">Tissenbaum, H.A., and Guarente, L. (2001). Increased dosage of a <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sir-2;class=Gene" target="_blank">sir-2</a></em></span> gene extends lifespan in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>410</em></span>, 227-230.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11242085&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/35065638" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib228"></a><p class="bibliomixed"><span class="bibliomisc">Tootle, T.L., and Rebay, I. (2005). Post-translational modifications influence transcription factor activity: a view from
                        the ETS superfamily. Bioessays <span class="emphasis"><em>27</em></span>, 285-298.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15714552&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/bies.20198" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib229"></a><p class="bibliomixed"><span class="bibliomisc">Unhavaithaya, Y., Shin, T.H., Miliaras, N., Lee, J., Oyama, T., and Mello, C.C. (2002). <a href="http://www.wormbase.org/db/get?name=MEP-1;class=Protein" target="_blank">MEP-1</a> and a homolog of the NURD complex component Mi-2 act together to maintain germline-soma distinctions in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>111</em></span>, 991-1002.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12507426&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(02)01202-3" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib230"></a><p class="bibliomixed"><span class="bibliomisc">van der Linden, A.M., Nolan, K.M., and Sengupta, P. (2007). <a href="http://www.wormbase.org/db/get?name=KIN-29;class=Protein" target="_blank">KIN-29</a> SIK regulates chemoreceptor gene expression via an MEF2 transcription factor and a class II HDAC. EMBO J. <span class="emphasis"><em>26</em></span>, 358-370.</span>
                     <a href="http://dx.doi.org/10.1038/sj.emboj.7601479" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib231"></a><p class="bibliomixed"><span class="bibliomisc">Van Nostrand, E.L., and Kim, S.K. (2011). Seeing elegance in gene regulatory networks of the worm. Curr. Opin. Genet. Dev.
                        <span class="emphasis"><em>21</em></span>, 776-786.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21963133&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.gde.2011.08.007" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib232"></a><p class="bibliomixed"><span class="bibliomisc">Vaquerizas, J.M., Kummerfeld, S.K., Teichmann, S.A., and Luscombe, N.M. (2009). A census of human transcription factors: function,
                        expression and evolution. Nat. Rev. Genet. <span class="emphasis"><em>10</em></span>, 252-263.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19274049&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib233"></a><p class="bibliomixed"><span class="bibliomisc">Venkatesan, K., McManus, H.R., Mello, C.C., Smith, T.F., and Hansen, U. (2003). Functional conservation between members of
                        an ancient duplicated transcription factor family, LSF/Grainyhead. Nucleic Acids Res. <span class="emphasis"><em>31</em></span>, 4304-4316.</span>
                     <a href="http://dx.doi.org/10.1093/nar/gkg644" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib234"></a><p class="bibliomixed"><span class="bibliomisc">Vilimas, T., Abraham, A., and Okkema, P.G. (2004). An early pharyngeal muscle enhancer from the <span class="emphasis"><em>Caenorhabditis elegans <a href="http://www.wormbase.org/db/get?name=ceh-22;class=Gene" target="_blank">ceh-22</a></em></span> gene is targeted by the Forkhead factor <a href="http://www.wormbase.org/db/get?name=PHA-4;class=Protein" target="_blank">PHA-4</a>. Dev. Biol. <span class="emphasis"><em>266</em></span>, 388-398.</span>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2003.10.015" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib235"></a><p class="bibliomixed"><span class="bibliomisc">Vogel, C., and Chothia, C. (2006). Protein family expansions and biological complexity. PLoS Comput. Biol. <span class="emphasis"><em>2</em></span>, e48.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16733546&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib236"></a><p class="bibliomixed"><span class="bibliomisc">Von Stetina, S.E., Watson, J.D., Fox, R.M., Olszewski, K.L., Spencer, W.C., Roy, P.J., and Miller, D.M., 3rd (2007). Cell-specific
                        microarray profiling experiments reveal a comprehensive picture of gene expression in the <span class="emphasis"><em>C. elegans</em></span> nervous system. Genome Biol. <span class="emphasis"><em>8</em></span>, R135.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17612406&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/gb-2007-8-7-r135" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib237"></a><p class="bibliomixed"><span class="bibliomisc">von Zelewsky, T., Palladino, F., Brunschwig, K., Tobler, H., Hajnal, A., and Muller, F. (2000). The <span class="emphasis"><em>C. elegans</em></span> Mi-2 chromatin-remodelling proteins function in vulval cell fate determination. Development <span class="emphasis"><em>127</em></span>, 5277-5284.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11076750&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib238"></a><p class="bibliomixed"><span class="bibliomisc">Wagmaister, J.A., Miley, G.R., Morris, C.A., Gleason, J.E., Miller, L.M., Kornfeld, K., and Eisenmann, D.M. (2006). Identification
                        of cis-regulatory elements from the <span class="emphasis"><em>C. elegans</em></span> Hox gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=lin-39;class=Gene" target="_blank">lin-39</a></em></span> required for embryonic expression and for regulation by the transcription factors <a href="http://www.wormbase.org/db/get?name=LIN-1;class=Protein" target="_blank">LIN-1</a>, <a href="http://www.wormbase.org/db/get?name=LIN-31;class=Protein" target="_blank">LIN-31</a> and <a href="http://www.wormbase.org/db/get?name=LIN-39;class=Protein" target="_blank">LIN-39</a>. Dev. Biol. <span class="emphasis"><em>297</em></span>, 550-565.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16782085&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2006.05.008" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib239"></a><p class="bibliomixed"><span class="bibliomisc">Walker, A.K., See, R., Batchelder, C., Kophengnavong, T., Gronniger, J.T., Shi, Y., and Blackwell, T.K. (2000). A conserved
                        transcription motif suggesting functional parallels between <span class="emphasis"><em>Caenorhabditis elegans</em></span> <a href="http://www.wormbase.org/db/get?name=SKN-1;class=Protein" target="_blank">SKN-1</a> and Cap'n'Collar-related basic leucine zipper proteins. J. Biol. Chem. <span class="emphasis"><em>275</em></span>, 22166-22171.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10764775&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib240"></a><p class="bibliomixed"><span class="bibliomisc">Walker, A.K., Shi, Y., and Blackwell, T.K. (2004). An extensive requirement for transcription factor IID-specific <a href="http://www.wormbase.org/db/get?name=TAF-1;class=Protein" target="_blank">TAF-1</a> in <span class="emphasis"><em>Caenorhabditis elegans</em></span> embryonic transcription. J. Biol. Chem. <span class="emphasis"><em>279</em></span>, 15339-15347.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14726532&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib241"></a><p class="bibliomixed"><span class="bibliomisc">Wallace, J.A., and Felsenfeld, G. (2007). We gather together: insulators and genome organization. Curr. Opin. Genet. Dev.
                        <span class="emphasis"><em>17</em></span>, 400-407.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17913488&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.gde.2007.08.005" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib242"></a><p class="bibliomixed"><span class="bibliomisc">Wallenfang, M.R., and Seydoux, G. (2002). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=cdk-7;class=Gene" target="_blank">cdk-7</a></em></span> Is required for mRNA transcription and cell cycle progression in <span class="emphasis"><em>Caenorhabditis elegans</em></span> embryos. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>99</em></span>, 5527-5532.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11960010&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.082618399" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib243"></a><p class="bibliomixed"><span class="bibliomisc">Wang, D., Kennedy, S., Conte, D., Jr., Kim, J.K., Gabel, H.W., Kamath, R.S., Mello, C.C., and Ruvkun, G. (2005). Somatic misexpression
                        of germline P granules and enhanced RNA interference in retinoblastoma pathway mutants. Nature <span class="emphasis"><em>436</em></span>, 593-597.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16049496&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature04010" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib244"></a><p class="bibliomixed"><span class="bibliomisc">Wang, X., Jia, H., and Chamberlin, H.M. (2006). The bZip proteins <a href="http://www.wormbase.org/db/get?name=CES-2;class=Protein" target="_blank">CES-2</a> and <a href="http://www.wormbase.org/db/get?name=ATF-2;class=Protein" target="_blank">ATF-2</a> alter the timing of transcription for a cell-specific target gene in <span class="emphasis"><em>C. elegans</em></span>. Dev. Biol. <span class="emphasis"><em>289</em></span>, 456-465.</span>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2005.10.029" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib245"></a><p class="bibliomixed"><span class="bibliomisc">Wang, Y., and Levy, D.E. (2006). <span class="emphasis"><em>C. elegans</em></span> STAT: evolution of a regulatory switch. FASEB J. <span class="emphasis"><em>20</em></span>, 1641-1652.</span>
                     <a href="http://dx.doi.org/10.1096/fj.06-6051com" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib246"></a><p class="bibliomixed"><span class="bibliomisc">Way, J.C., and Chalfie, M. (1988). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mec-3;class=Gene" target="_blank">mec-3</a></em></span>, a homeobox-containing gene that specifies differentiation of the touch receptor neurons in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>54</em></span>, 5-16.</span>
                     <a href="http://dx.doi.org/10.1016/0092-8674(88)90174-2" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib247"></a><p class="bibliomixed"><span class="bibliomisc">Wenick, A.S., and Hobert, O. (2004). Genomic cis-regulatory architecture and transacting regulators of a single interneuron-specific
                        gene battery in <span class="emphasis"><em>C. elegans</em></span>. Dev. Cell <span class="emphasis"><em>6</em></span>, 757-770.</span>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2004.05.004" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib248"></a><p class="bibliomixed"><span class="bibliomisc">Whittle, C.M., Lazakovitch, E., Gronostajski, R.M., and Lieb, J.D. (2009). DNA-binding specificity and in vivo targets of
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span> nuclear factor I. Proc. Natl. Acad. Sci. U. S. A. <span class="emphasis"><em>106</em></span>, 12049-12054.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19584245&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0812894106" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib249"></a><p class="bibliomixed"><span class="bibliomisc">Whittle, C.M., McClinic, K.N., Ercan, S., Zhang, X., Green, R.D., Kelly, W.G., and Lieb, J.D. (2008). The genomic distribution
                        and function of histone variant <a href="http://www.wormbase.org/db/get?name=HTZ-1;class=Protein" target="_blank">HTZ-1</a> during <span class="emphasis"><em>C. elegans</em></span> embryogenesis. PLoS Genet <span class="emphasis"><em>4</em></span>, e1000187.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18787694&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000187" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib250"></a><p class="bibliomixed"><span class="bibliomisc">Wilson, D., Charoensawan, V., Kummerfeld, S.K., and Teichmann, S.A. (2008). DBD&#8212;taxonomically broad transcription factor predictions:
                        new content and functionality. Nucleic Acids Res. <span class="emphasis"><em>36</em></span>, D88-92.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=18073188&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/nar/gkm964" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib251"></a><p class="bibliomixed"><span class="bibliomisc">Winkelbauer, M.E., Schafer, J.C., Haycraft, C.J., Swoboda, P., and Yoder, B.K. (2005). The <span class="emphasis"><em>C. elegans</em></span> homologs of nephrocystin-1 and nephrocystin-4 are cilia transition zone proteins involved in chemosensory perception. J.
                        Cell Sci <span class="emphasis"><em>118</em></span>, 5575-5587.</span>
                     <a href="http://dx.doi.org/10.1242/jcs.02665" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib252"></a><p class="bibliomixed"><span class="bibliomisc">Winn, J., Carter, M., Avery, L., and Cameron, S. (2011). Hox and a newly identified E2F co-repress cell death in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>188</em></span>, 897-905.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21596899&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.111.128421" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib253"></a><p class="bibliomixed"><span class="bibliomisc">Winnier, A.R., Meir, J.Y., Ross, J.M., Tavernarakis, N., Driscoll, M., Ishihara, T., Katsura, I., and Miller, D.M., 3rd (1999).
                        <a href="http://www.wormbase.org/db/get?name=UNC-4;class=Protein" target="_blank">UNC-4</a>/UNC-37-dependent repression of motor neuron-specific genes controls synaptic choice in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genes Dev. <span class="emphasis"><em>13</em></span>, 2774-2786.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10557206&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib254"></a><p class="bibliomixed"><span class="bibliomisc">Xia, D., Zhang, Y., Huang, X., Sun, Y., and Zhang, H. (2007). The <span class="emphasis"><em>C. elegans</em></span> CBF&#946; homolog, <a href="http://www.wormbase.org/db/get?name=BRO-1;class=Protein" target="_blank">BRO-1</a>, regulates the proliferation, differentiation and specification of the stem cell-like seam cell lineages. Dev. Biol. <span class="emphasis"><em>309</em></span>, 259-272.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=17706957&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.ydbio.2007.07.020" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib255"></a><p class="bibliomixed"><span class="bibliomisc">Yi, W., Ross, J.M., and Zarkower, D. (2000). <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=mab-3;class=Gene" target="_blank">mab-3</a></em></span> is a direct <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=tra-1;class=Gene" target="_blank">tra-1</a></em></span> target gene regulating diverse aspects of <span class="emphasis"><em>C. elegans</em></span> male sexual development and behavior. Development <span class="emphasis"><em>127</em></span>, 4469-4480.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11003845&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib256"></a><p class="bibliomixed"><span class="bibliomisc">Yi, W., and Zarkower, D. (1999). Similarity of DNA binding and transcriptional regulation by <span class="emphasis"><em>Caenorhabditis elegans</em></span> <a href="http://www.wormbase.org/db/get?name=MAB-3;class=Protein" target="_blank">MAB-3</a> and <span class="emphasis"><em>Drosophila melanogaster</em></span> DSX suggests conservation of sex determining mechanisms. Development <span class="emphasis"><em>126</em></span>, 873-881.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9927589&amp;dopt=Abstract" target="_blank">Abstract</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib257"></a><p class="bibliomixed"><span class="bibliomisc">Yin, J., Yu, L., and Savage-Dunn, C. (2010). Alternative trans-splicing of <span class="emphasis"><em>Caenorhabditis elegans <a href="http://www.wormbase.org/db/get?name=sma-9;class=Gene" target="_blank">sma-9</a>/schnurri</em></span> generates a short transcript that provides tissue-specific function in BMP signaling. BMC Mol. Biol. <span class="emphasis"><em>11</em></span>, 46.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20565799&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2199-11-46" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib258"></a><p class="bibliomixed"><span class="bibliomisc">Yun, M., Wu, J., Workman, J.L., and Li, B. (2011).  Readers of histone modifications.  Cell Res. <span class="emphasis"><em>21</em></span>, 564-578.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=21423274&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/cr.2011.42" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib259"></a><p class="bibliomixed"><span class="bibliomisc">Yuzyuk, T., Fakhouri, T. H., Kiefer, J., and Mango, S. E. (2009). The polycomb complex protein <a href="http://www.wormbase.org/db/get?name=mes-2;class=Gene" target="_blank">mes-2</a>/E(z) promotes the transition from developmental plasticity to differentiation in <span class="emphasis"><em>C. elegans</em></span> embryos. Dev. Cell <span class="emphasis"><em>16</em></span>, 699-710.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=19460346&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2009.03.008" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib260"></a><p class="bibliomixed"><span class="bibliomisc">Zarkower, D., and Hodgkin, J. (1993). Zinc fingers in sex determination: only one of the two <span class="emphasis"><em>C. elegans</em></span> Tra-1 proteins binds DNA in vitro. Nucleic Acids Res. <span class="emphasis"><em>21</em></span>, 3691-3698.</span>
                     <a href="http://dx.doi.org/10.1093/nar/21.16.3691" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib261"></a><p class="bibliomixed"><span class="bibliomisc">Zhong, M., Niu, W., Lu, Z.J., Sarov, M., Murray, J.L., Janette, J., Raha, D., Sheaffer, K.L., Lam, H.Y., Preston, E., <span class="emphasis"><em>et al.</em></span> (2010). Genome-wide identification of binding sites defines distinct functions for <span class="emphasis"><em>Caenorhabditis elegans</em></span> PHA-4/FOXA in development and environmental response. PLoS Genet. <span class="emphasis"><em>6</em></span>, e1000848.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=20174564&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1371/journal.pgen.1000848" target="_blank">Article</a>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by: Thomas Blumenthal. Last revised March 28, 2012, Published June 4, 2013. This chapter should be cited as: Reinke
                  et al. Transcriptional regulation of gene expression in <span class="emphasis"><em>C. elegans</em></span> (June 4, 2013), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.45.2, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2013 Reinke et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e37" href="#d0e37">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:michaelkr@niddk.nih.gov" target="_top">michaelkr@niddk.nih.gov</a>, or <a href="mailto:okkema@uic.edu" target="_top">okkema@uic.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>