<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Caenorhabditis nomenclature</title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WB_Tuli_etal_Nomenclature" title="Caenorhabditis nomenclature">
      <link rel="next" href="#sec1" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="caenornomenclature.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WB_Tuli_etal_Nomenclature"></a><span class="italic">Caenorhabditis</span> nomenclature<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup></h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Mary Ann Tuli<sup>1,<span class="italic">&sect;</span></sup>, Aric Daul<sup>2</sup>, Tim Schedl<sup>3,<span class="italic">&sect;</span></sup><em><span class="remark"><sup><a name="d0e44" href="#ftn.d0e44">&sect;</a></sup></span></em></span><br></span><span class="affiliation"><span class="orgname"><sup>1</sup>Biology Division, California Institute of Technology, Pasadena CA 91125, USA; </span></span><span class="affiliation"><span class="orgname"><sup>2</sup>Caenorhabditis Genetics Center, University of Minnesota, Minneapolis MN 55455, USA; </span></span><span class="affiliation"><span class="orgname"><sup>3</sup>Department of Genetics, Washington University School of Medicine, St. Louis MO 63110, USA; </span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_geneticsgenomics.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Genetics and genomics
		</a></div><hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#sec2">2. Genes</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec2.1">2.1. Applying for new gene class and/or gene names</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2.2">2.2. Guidelines for proposing new gene names</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2.3">2.3. Approved gene name conflicts</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec3">3. Guidelines for variation data</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec3.1">3.1. Alleles/mutations from experimentally induced variation</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3.2">3.2. Gene knockouts</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3.3">3.3. Genome engineering</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3.4">3.4. Modifiers: suppressors, revertants, and enhancers</a></span></dt>
                     <dt><span class="sect2"><a href="#sec3.5">3.5. Chromosomal aberrations.</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec4">4. Polymorphisms and other genetic elements</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec4.1">4.1. SNPs and RFLPs</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4.2">4.2. Natural copy number variants</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4.3">4.3. Introgressed regions in near-isogenic lines.</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4.4">4.4. Transposons and transposon insertions</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4.5">4.5. Transgenes</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec5">5. Genotypes</a></span></dt>
               <dt><span class="sect1"><a href="#sec6">6. Phenotypes</a></span></dt>
               <dt><span class="sect1"><a href="#sec7">7. Proteins</a></span></dt>
               <dt><span class="sect1"><a href="#sec8">8. Strains</a></span></dt>
               <dt><span class="sect1"><a href="#sec9">9. Other nematodes</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec9.1">9.1. Species prefixes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9.2">9.2. Gene naming: homologous genes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9.3">9.3. Gene naming: non-homologous genes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9.4">9.4. Gene naming: forward genetics</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec10">10. Acknowledgements</a></span></dt>
               <dt><span class="sect1"><a href="#sec11">11. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>Genetic nomenclature for <span class="italic">Caenorhabditis</span> species and other nematodes is supervised by <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> in collaboration with the <a href="https://cgc.umn.edu/" target="_top"><span class="italic">Caenorhabditis</span> Genetics Center</a> (CGC) and with essential input from the community of scientists working on <span class="italic">C. elegans</span> and other nematodes.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>Genetic nomenclature allows the genetic features of an organism to be structured and described in a uniform and systematic
               way. Genetic features, including genes, variations (both natural and induced), and gene products, are assigned descriptors
               that inform on the nature of the feature. These nomenclature designations facilitate communication among researchers (in publications,
               presentations, and databases) to advance understanding of the biology of the genetic feature and the experimental utilization
               of organisms that contain the genetic feature.
            </p>
            <p>The nomenclature system that is used for <span class="italic">C. elegans</span> was first employed by <a href="#bib1">Sydney Brenner (1974)</a> in his landmark description of the genetics of this model organism, and then substantially extended and modified in <a href="#bib3">Horvitz et al., 1979</a>. The gene, allele, and chromosome rearrangement nomenclature, described below, is an amalgamation of that from bacteria and
               yeast, with the rearrangement types from <span class="italic">Drosophila</span>. The nomenclature avoids standard words, subscripts, superscripts, and Greek letters and includes a hyphen (-) to separate
               the gene name from gene number (distinct genes with similar phenotypes or molecular properties). As described by <a href="https://wormbase.org/resources/person/WBPerson261#013542--10" target="_top">Jonathan Hodgkin</a>, &#8216;the hyphen is about 1 mm in length in printed text and therefore symbolizes the 1 mm long worm&#8217;. These nomenclature properties
               make <span class="italic">C. elegans</span> publications highly suitable for informatic text mining, as there is minimal ambiguity. From the founding of the Caenorhabditis
               Genetics Center (<a href="https://cgc.umn.edu/" target="_top">CGC</a>) in 1979 until 1992, <a href="https://wormbase.org/resources/person/WBPerson521#013542--10" target="_top">Don Riddle</a> and <a href="https://wormbase.org/resources/person/WBPerson154#013542--10" target="_top">Mark Edgley</a> acted as the central repository for genetic nomenclature. <a href="https://wormbase.org/resources/person/WBPerson261#013542--10" target="_top">Jonathan Hodgkin</a> was nomenclature czar from 1992 through 2013; this was a pivotal period with the elucidation of the genome sequence of <span class="italic">C. elegans</span>, and later that of related nematodes, and the inception of <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a>. Thus, under the guidance of <a href="https://wormbase.org/resources/person/WBPerson261#013542--10" target="_top">Hodgkin</a>, the nomenclature system became a central feature of <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> and the number and types of genetic features significantly expanded. The nomenclature system remains dynamic, with recent
               additions including guidelines related to genome engineering, and continued reliance on the community for input.
            </p>
            <p><a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> assigns specific identifying codes to each laboratory engaged in dedicated long-term genetic research on <span class="italic">C. elegans</span>. Each laboratory is assigned a laboratory/strain code for naming strains, and an allele code for naming genetic variation
               (e.g., mutations) and transgenes. These designations are assigned to the laboratory head/PI who is charged with supervising
               their organization in laboratory databases and their associated biological reagents that are described on <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a>, in publications, and distributed to the scientific community on request. The laboratory/strain code is used: a) to identify
               the originator of community-supplied information on WormBase, which, in addition to attribution, facilitates communication
               between the community/curators and the originator if an issue related to the information should arise at a later date; and
               b) to provide a tracking code for activities at the <a href="https://cgc.umn.edu/" target="_top">CGC</a>. The laboratory/strain designation consists of 2-3 uppercase letters while the allele designation has 1-3 lowercase letters.
               The final letter of a laboratory code should not be an &#8220;O&#8221; or an &#8220;I&#8221; so as not to be mistaken for the numbers &#8220;0&#8221; or &#8220;1&#8221; respectively.
               Additionally, allele designations should also not end with the letter &#8220;l&#8221; which could also be mistaken for the number &#8220;1.&#8221;
               These codes are listed at the <a href="https://cgc.umn.edu/" target="_top">CGC</a> and in <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a>. Investigators generating strains, alleles, transgenes, and/or defining genes require these designations and should apply
               for them at <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a>.
            </p>
            <p>Information for several other nematode species, in addition to <span class="italic">C. elegans</span>, is curated at <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a>. All species are referred to by their Linnean binomial names (e.g,. <span class="italic">Caenorhabditis elegans</span> or <span class="italic">C. elegans</span>). Details of all the genomes available at <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> and the degree of their curation can be found at <a href="http://www.wormbase.org/species/all" target="_top">www.wormbase.org/species/all</a></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Genes
                     </h2>
                  </div>
               </div>
            </div>
            <p>Three different gene naming schemes are employed in <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a>: GeneIDs, Sequence Names, and Gene Names.
            </p>
            <p>First, for all genes <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> has assigned a unique accession of the form &#8216;WBGene#&#8217;, for example &#8216;WBGene00000024&#8217;. The accession uniquely refers to a locus
               at a specific location in the genome: either a protein coding sequence (CDS) and transcript structure, an uncloned gene, a
               non-coding RNA transcript, a defined sequence element, or pseudogene. This accession follows the gene through any changes
               that may be made to it based on bioinformatics predictions or experimentation. If the gene object is split into two genes,
               the original accession will usually apply to the 5&#8217; gene and a new accession will be assigned to the other half.
            </p>
            <p>Second, for nearly all <span class="italic">C. elegans</span> genes <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> has assigned a Sequence Name, which is derived from the cosmid, fosmid, or YAC clone on which they reside, and which was
               used to generate the reference Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> genome sequence (The <span class="italic">C. elegans</span> <a href="#bib4">Sequencing Consortium, 1998</a>). For example &#8216;AC3.3&#8217;, indicates that the gene is on cosmid <a href="http://www.wormbase.org/db/get?name=AC3;class=Clone" target="_blank">AC3</a>, and that there are at least 2 other genes (.1 and .2) on that cosmid.
            </p>
            <p>Finally, for a number of loci, a standard genetic gene name is employed, which usually refers to mutant phenotype or some
               aspect of sequence similarity. Researchers who are investigating such genes are encouraged to propose gene names based on
               the guidelines in <a href="#sec2.1" title="2.1.&nbsp;Applying for new gene class and/or gene names">Section 2.1</a> and <a href="#sec2.2" title="2.2.&nbsp;Guidelines for proposing new gene names">Section 2.2</a>. Genetic gene names consist of a three or four letter gene class name, e.g., &#8220;<span class="italic">abu</span>&#8221; (for <span class="bold"><strong>A</strong></span>ctivated in <span class="bold"><strong>B</strong></span>locked <span class="bold"><strong>U</strong></span>nfolded protein response), followed by a hyphen and a number: <span class="italic">&#8220;abu-1&#8221;</span>. A small number of gene objects defined by mutation are currently uncloned, and until associated with a sequence name will
               appear with just a standard genetic name. Publications commonly use the genetic name, as this designation provides information
               about function in biological processes, potential molecular function based on sequence similarity, or in some cases known
               biochemical activity. Sequence names for genes lacking a genetic name, of which there are many, are also used in publications,
               often with -omic level studies where functional information may be limited.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2.1"></a>2.1.&nbsp;Applying for new gene class and/or gene names
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Investigators wishing to propose new gene names for <span class="italic">C. elegans</span> should note the summary guidelines below and apply online via <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> or by email application to <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a>. It is highly recommended that investigators contact <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> with their gene name proposal prior to submission of a manuscript or abstract, as proposed gene names are not always used
                  as the main name on <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a>. Reasons for not using a proposed name as the main name include the gene already having a name (usually being held in confidence;
                  see <a href="#sec2.2.8" title="2.2.8.&nbsp;&#8195;">Section 2.2.8</a>.) or that the name is inappropriate based on incorrect sequence analysis or other type of ambiguity.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2.2"></a>2.2.&nbsp;Guidelines for proposing new gene names
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.1"></a>2.2.1.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Gene names must conform to the standard format of 3 or 4 letters gene class name, hyphen, number.</p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.2"></a>2.2.2.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Genes can be named on the basis of a mutant phenotype or on the basis of the predicted protein product or RNA product.</p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.3"></a>2.2.3.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>For genes defined by mutation, the approved gene names refer to the mutant phenotype originally detected or most easily scored
                     e.g., <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000583;class=Phenotype" target="_blank">dumpy</a> (<span class="italic">d</span>um<span class="italic">py</span>) in the case of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001067;class=Gene" target="_blank">dpy-5</a></span>, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000062;class=Phenotype" target="_blank">lethal</a> (<span class="italic">let</span>hal) in the case of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002312;class=Gene" target="_blank">let-37</a></span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.4"></a>2.2.4.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>For genes defined on the basis of sequence similarity or sequence features, the approved gene name refers to the predicted
                     protein product, RNA product, or gene causing a disease, e.g., <span class="italic">my</span>osin <span class="italic">l</span>ight <span class="italic">c</span>hain in the case of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003371;class=Gene" target="_blank">mlc-3</a></span>, <span class="italic">s</span>uper<span class="italic">o</span>xide <span class="italic">d</span>ismutase in the case of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004930;class=Gene" target="_blank">sod-1</a></span>, or <span class="italic">n</span>e<span class="italic">ph</span>rono<span class="italic">p</span>hthisis (human kidney disease) homolog, in the case of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00011261;class=Gene" target="_blank">nphp-4</a></span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.5"></a>2.2.5.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Gene names based on homology with a previously named gene in another well-studied organism (e.g., <span class="italic">Homo sapiens</span>) or that is the standard in the relevant field (e.g., DNA replication) are often appropriate and desirable, especially where
                     there is convincing orthology between genes (e.g., <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006944;class=Gene" target="_blank">wrn-1</a></span> is the <span class="italic">C. elegans</span> ortholog of the human gene WRN1, responsible for Werner's syndrome, or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004393;class=Gene" target="_blank">rnt-1</a></span> is the <span class="italic">C. elegans</span> ortholog of the <span class="italic">Drosophila</span> gene <span class="italic">runt</span>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.6"></a>2.2.6.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>If a new gene clearly belongs in an existing gene class, then a new gene number will be assigned after consultation with the
                     laboratory responsible for the gene class in question.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.7"></a>2.2.7.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>If the establishment of a new gene class name seems more appropriate, then approval for this gene name must be obtained, preferably
                     online via <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> or by email application to <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a>. To avoid the situation of multiple names for the same gene in the literature, approval should be sought prior to abstract
                     or manuscript submission.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.8"></a>2.2.8.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>New gene name classes can be assigned in confidence, prior to formal publication or disclosure in an abstract.</p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.9"></a>2.2.9.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Gene names that are memorable, informative, and simply explained are encouraged.</p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.10"></a>2.2.10.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Gene names and gene numbering schemes that conform to established nomenclature proposals for particular protein classes are
                     desirable.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.11"></a>2.2.11.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Gene names including <span class="italic">c</span> (for <span class="italic">Caenorhabditis</span>), <span class="italic">ce</span> (for <span class="italic">C. elegans</span>), <span class="italic">n</span> (for nematode) or <span class="italic">w</span> (for worm) are discouraged. <span class="italic">C. elegans</span> as the organism of origin can be specified with a prefix (<span class="italic">Cel</span>-) if desired, e.g., <span class="italic">Cel-fem-1</span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.12"></a>2.2.12.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Gene names based solely on RNAi phenotypes or high-throughput analysis of gene expression or protein interactions are discouraged.</p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.13"></a>2.2.13.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A limited number of genes have been given temporary <span class="italic">tag</span>- names (<span class="italic">tag</span> = <span class="italic">t</span>emporarily <span class="italic">a</span>ssigned <span class="italic">g</span>ene name). These are genes for which deletion alleles have been generated by reverse genetic methods, but which have not yet
                     been given more informative names based on sequence or mutant phenotype. When sufficient information becomes available, each
                     <span class="italic">tag</span> name will be replaced by an appropriate standard 3- or 4-letter name.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.14"></a>2.2.14.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>If a gene produces a protein that can be classified as a member of a family, the gene may also be assigned an approved name
                     consisting of three or four italicized letters, a hyphen, and an italicized Arabic number, e.g., <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003639;class=Gene" target="_blank">nhr-49</a></span>, indicating that this is the 49th member of the <span class="italic">nhr</span> (nuclear hormone receptor) gene family.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.15"></a>2.2.15.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>There are a few exceptions to this format. Genes in a paralogous set related to a single named gene are sometimes given the
                     same gene name and number, followed by a distinguishing decimal. For example, four <span class="italic">C. elegans</span> genes homologous to <span class="italic">SIR2</span> in <span class="italic">S. cerevisiae</span> have been given the names <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00004800;class=Gene" target="_blank">sir-2.1</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00004801;class=Gene" target="_blank">sir-2.2</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00004802;class=Gene" target="_blank">sir-2.3</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00004803;class=Gene" target="_blank">sir-2.4</a></span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.16"></a>2.2.16.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A limited number of genes, named on the basis of sequence homology, have been given non-standard names ending with alphanumeric
                     identifiers rather than with simple numbers in order to make these names closer to the generally accepted names used in other
                     organisms. For example, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001225;class=Gene" target="_blank">eif-3.B</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00001226;class=Gene" target="_blank">eif-3.C</a></span>, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001227;class=Gene" target="_blank">eif-3.D</a>,</span> etc. encode proteins of the conserved translation factor eIF3.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.17"></a>2.2.17.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The gene name may, on rare occasions, be followed by an italicized Roman numeral to indicate the chromosome (linkage group)
                     on which the gene resides, e.g., <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001067;class=Gene" target="_blank">dpy-5</a> I</span> (chromosome <span class="italic">I</span>) or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002312;class=Gene" target="_blank">let-37</a> X</span> or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003371;class=Gene" target="_blank">mlc-3</a> III</span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.18"></a>2.2.18.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Genes with related properties are often given the same 3 or 4 letter gene class name and different numbers. For example, there
                     are three known myosin light chain genes: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003369;class=Gene" target="_blank">mlc-1</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00003370;class=Gene" target="_blank">mlc-2</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00003371;class=Gene" target="_blank">mlc-3</a></span>, and more than twenty different <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000583;class=Phenotype" target="_blank">dumpy</a> genes: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001063;class=Gene" target="_blank">dpy-1</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00001064;class=Gene" target="_blank">dpy-2</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00001065;class=Gene" target="_blank">dpy-3</a>,</span> and so on.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2.2.19"></a>2.2.19.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>There is no specific nomenclature for pseudogenes. The Overview section on the <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> gene page indicates if the gene is designated as a pseudogene.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2.3"></a>2.3.&nbsp;Approved gene name conflicts
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Approved gene names that have been established in databases and have been used in a body of published literature should preferably
                  not be changed. In cases where a gene has received multiple names, one name will be adopted as the main name for the gene
                  on <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a>. The other names will continue to be listed in <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a>, where relevant associations can be found through search features. Whenever possible, name changes or the adoption of a single
                  main name should be made with the approval of all laboratories concerned.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;Guidelines for variation data
                     </h2>
                  </div>
               </div>
            </div>
            <p>Variation is defined as a change in genome sequence compared to the reference <span class="italic">C. elegans</span> Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> sequence. Variation can be experimentally induced (e.g., chemical mutagenesis, genome engineering) or found in <span class="italic">C. elegans</span> strains isolated from natural environments. The term allele is applied both to experimentally-induced change and natural
               variation whether the change produces a phenotype or not. The term mutation is often applied to experimentally-induced variation
               which can often result in phenotypic deviation. The term polymorphism is applied to natural variation, and often does not
               result in a phenotype. The Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> reference strain is defined by convention as wild type. However, during laboratory propagation Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> can accumulate variation resulting in differences from the reference Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> sequence. Therefore in cases where an investigator wants to associate phenotype with genotype, multiple lines of evidence
               are necessary to draw the conclusion that variation x leads to phenotype y. Every variation is assigned an accession of the
               form WBVariationDDDD. Some variations, for example mutant alleles and polymorphisms of certain types, are assigned an additional
               allele name.
            </p>
            <p>(Natural variation is discussed below in <a href="#sec5" title="5.&nbsp;Genotypes">Section 5</a>. Polymorphisms and other genetic elements.)
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3.1"></a>3.1.&nbsp;Alleles/mutations from experimentally induced variation
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.1.1"></a>3.1.1.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Every allele has a unique designation. Alleles are given names consisting of one to three italicized letters (the allele designation
                     for the assigning laboratory) followed by an italicized Arabic number, e.g., <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00142941;class=Variation" target="_blank">e61</a></span> or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00089058;class=Variation" target="_blank">mn138</a></span> or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00248701;class=Variation" target="_blank">st5</a></span>. The last letter should not be an &#8220;l&#8221; to avoid confusion with the number &#8220;1&#8221;. For example, <span class="italic">e</span> refers (originally) to the MRC Laboratory of Molecular Biology (Cambridge, U.K.), (currently) to the laboratory of <a href="https://wormbase.org/resources/person/WBPerson261#013542--10" target="_top">J. Hodgkin</a> (University of Oxford), and <span class="italic">st</span> refers to the laboratory of <a href="https://wormbase.org/resources/person/WBPerson1562#01--10" target="_top">R.H. Waterston</a> (originally at Washington University, St. Louis, MO, currently at the University of Washington, Seattle).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.1.2"></a>3.1.2.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>When gene and allele names are used together, the allele name is included in parentheses after the gene name, e.g., <span class="italic">dpy-5(<a href="http://www.wormbase.org/db/get?name=WBVar00142941;class=Variation" target="_blank">e61</a>)</span>, <span class="italic">let-37(<a href="http://www.wormbase.org/db/get?name=WBVar00089058;class=Variation" target="_blank">mn138</a>)</span>. When unambiguous (e.g., if only one allele is known for a given gene or if all work on a gene described in a publication
                     used a single allele cited in a Methods section), gene names can be used in preference to allele names (<span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002312;class=Gene" target="_blank">let-37</a></span> rather than <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00089058;class=Variation" target="_blank">mn138</a></span> or <span class="italic">let-37(<a href="http://www.wormbase.org/db/get?name=WBVar00089058;class=Variation" target="_blank">mn138</a>)</span>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.1.3"></a>3.1.3.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Optional suffixes indicating characteristics of an allele can follow the allele name. These are usually two-letter non-italicized
                     letters, e.g., <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00087750;class=Variation" target="_blank">hc17</a></span>ts, where ts stands for temperature-sensitive, or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00239157;class=Variation" target="_blank">pk15</a></span>te, where te stands for transposon-excision.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.1.4"></a>3.1.4.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The wild type allele (version) of a gene is defined as that present in the Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> reference strain, stored frozen at the <a href="https://cgc.umn.edu/" target="_top">CGC</a> and other locations. The wild type allele can be designated by a plus sign immediately after the gene name, <span class="italic">dpy-5+,</span> or, more commonly, by including the plus sign in parentheses, <span class="italic">dpy-5(</span><span class="bold"><strong><span class="italic">+</span></strong></span><span class="italic">)</span>.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3.2"></a>3.2.&nbsp;Gene knockouts
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.2.1"></a>3.2.1.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Many gene knockouts constructed to date are small deletions (&lt;5 kb) generated by chemical mutagenesis, by transposon excision,
                     or by genome engineering. These are named as alleles, sometimes with the optional suffix te (transposon-excision) or ko (knockout).
                     Example: <span class="italic">zyx-1(<a href="http://www.wormbase.org/db/get?name=WBVar00145597;class=Variation" target="_blank">gk190</a>)</span> is a 777 bp deletion in the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006999;class=Gene" target="_blank">zyx-1</a></span> gene.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.2.2"></a>3.2.2.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Some of the small deletions generated by chemical mutagenesis, by transposon excision, or by genome engineering may remove
                     parts of two adjacent genes. If only two genes appear to be affected, then the deletion is given a single allele name, but
                     the genotype is written with both gene names coupled with an ampersand (&amp;). Example: allele <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00091899;class=Variation" target="_blank">ok615</a></span> is a 1422 bp deletion of two adjacent genes, so it can be written <span class="italic">rad-54&amp;tag-157(<a href="http://www.wormbase.org/db/get?name=WBVar00091899;class=Variation" target="_blank">ok615</a>)</span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.2.3"></a>3.2.3.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Deletions that affect more than two genes are named as Deficiencies &#8220;(<span class="italic">Df</span>)&#8221;, as described in <a href="#sec3.5" title="3.5.&nbsp;Chromosomal aberrations.">Section 3.5</a> Chronmosomal aberrations.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3.3"></a>3.3.&nbsp;Genome engineering
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Genome engineering (CRISPR-Cas9, TALENs, etc.) is increasingly being applied to <span class="italic">C. elegans</span> and related nematodes. The resulting genomic alterations require some additional recommendations. The aim is to provide compact
                  and unambiguous ways of describing and referring to engineered changes to endogenous loci, as distinct from transgenic constructs
                  that are inserted elsewhere in the genome.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.3.1"></a>3.3.1.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Each engineered modification to an endogenous locus (point mutations, deletions, insertions, or combinations thereof) should
                     receive a unique allele designation, using the standard allele designation of the originating laboratory. For example: <span class="italic">bus-50(e5000).</span> A single allele designation is employed even if multiple changes are made to the locus.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.3.2"></a>3.3.2.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Optional brackets can be employed to provide additional information. Example: <span class="italic">bus-50(e5000</span>[T110E]) (an engineered missense mutation).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.3.3"></a>3.3.3.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A new allele designation is assigned for each independent, yet identical, engineered gene change. For example, if in a structure/function
                     analysis of <span class="italic">gene-x</span>, the co-conversion strategy used resulted in the R71C change in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene" target="_blank">dpy-10</a></span> in 15 strains containing modifications to <span class="italic">gene-x</span>, then 15 different allele designations need to be assigned to <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene" target="_blank">dpy-10</a></span> (in additional to relevant allele assignments for <span class="italic">gene-x</span>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.3.4"></a>3.3.4.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>An engineered fusion of GFP to the C-terminus of BUS-50 would be: <span class="italic">bus-50(e5001[bus-50::gfp])</span>. Similarly, an N-terminal fusion would be: <span class="italic">bus-50(e5100[gfp:bus-50])</span>. In the case of an engineered internal GFP fusion, the designation is the same as a C-terminal fusion, with a detailed description
                     of the structure in the relevant publication.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.3.5"></a>3.3.5.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>As a shorter and more convenient form, and where unambiguous, this could be referred to as: <span class="italic">bus-50::gfp.</span> Such abbreviations should be clearly defined where first used in a publication.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.3.6"></a>3.3.6.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>An engineered insertion of GFP plus the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a></span>(+) selectable marker, flanked by <span class="italic">loxP</span> sites, would be: <span class="italic">bus-50(e5002[bus-50::gfp + loxP unc-119(+) loxP]).</span></p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.3.7"></a>3.3.7.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Each additional engineering of the endogenous locus requires a new allele number. In the example of <span class="italic">bus-50(e5002)</span>, following Cre-mediated recombinase removal of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a></span>(+) so that a single loxP site remains, the new genotype would be <span class="italic">bus-50(e5003[bus-50::gfp +loxP])</span> or <span class="italic">bus-50(e5003)</span> for short. The original allele can be indicated in brackets with a preceding asterisk (*), in order to allow searches for
                     all derivatives from a given change. The above example would be <span class="italic">bus-50(e5003[*e5002]).</span></p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.3.8"></a>3.3.8.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Engineered insertions in apparent intergenic regions are given standard <span class="italic">Is</span> or <span class="italic">Si</span> (see Section 7.5) insertion names, for example <span class="italic">eIs2002</span>. Optional descriptors can include the nature of the insertion, e.g., <span class="italic">[<a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a>::gfp]</span> and the position in the genome, e.g., <span class="italic">[III:2992500]</span>, to give <span class="italic">eIs2002[<a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a>::gfp]</span> or <span class="italic">eIs2002[<a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a>::gfp, III:2992500].</span></p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.3.9"></a>3.3.9.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Engineered changes to existing integrated transgenes, either <span class="italic">Is</span> or <span class="italic">Si,</span> should receive new <span class="italic">Is</span> <span class="italic">or Si</span> numbers using originating lab's prefix. The original <span class="italic">Is</span> or <span class="italic">Si</span> insertion can be indicated in brackets with a preceding asterisk (*), in order to allow searches for all derivatives from
                     a given insertion. For example, an engineered change from GFP to mCherry in <span class="italic">eIs2002</span> might be named as <span class="italic">ozIs909</span>, or <span class="italic">ozIs909[<a href="http://www.wormbase.org/db/get?name=WBGene00006843;class=Gene" target="_blank">unc-119</a>::mCherry *eIs2002].</span></p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3.4"></a>3.4.&nbsp;Modifiers: suppressors, revertants, and enhancers
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.4.1"></a>3.4.1.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>There is no special nomenclature for modifier mutations. Many extragenic suppressor loci are called <span class="italic">sup</span> (40 <span class="italic">sup</span> loci defined so far, with a wide variety of properties and mechanisms). An increasing number of more specific modifier gene
                     classes have been established, such as <span class="italic">smu</span> (suppressor of <span class="italic">mec</span> and <span class="italic">unc</span>), and <span class="italic">smg</span> (<span class="italic">s</span>uppressor with <span class="italic">m</span>orphogenetic effect on <span class="italic">g</span>enitalia) and <span class="italic">sel</span> (<span class="italic">s</span>uppressor/<span class="italic">e</span>nhancer of <span class="italic">l</span>in-12).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.4.2"></a>3.4.2.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Intragenic suppressors or modifiers are indicated by adding a second allele name within parentheses; for example, <span class="italic">unc-17(e245e2608)</span> is an intragenic partial revertant of <span class="italic">unc-17(<a href="http://www.wormbase.org/db/get?name=WBVar00143061;class=Variation" target="_blank">e245</a>)</span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3.4.3"></a>3.4.3.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Mutations known to be chromosomal rearrangements, rather than intragenic lesions, are named differently, as described in the
                     <a href="#sec3.5" title="3.5.&nbsp;Chromosomal aberrations.">Section 3.5</a>.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3.5"></a>3.5.&nbsp;Chromosomal aberrations.
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="italic">D</span>u<span class="italic">p</span>lications (<span class="italic"><a href="http://www.wormbase.org/db/get?name=Dp;class=Cell" target="_blank">Dp</a></span>), <span class="italic">d</span>e<span class="italic">f</span>iciencies (<span class="italic">Df</span>), <span class="italic">in</span>versions (<span class="italic">In</span>), and <span class="italic">t</span>ranslocations (<span class="italic">T</span>) are known in <span class="italic">C. elegans</span> genetics and cytogenetics. These are given italicized names consisting of the laboratory mutation prefix, the relevant abbreviation,
                  and a number, optionally followed by the affected linkage groups in parentheses (e.g., <span class="italic">eT1(III;V), mnDp5(<a href="http://www.wormbase.org/db/get?name=X;class=Clone" target="_blank">X</a>;f)</span>, where <span class="italic">f</span> indicates a free duplication). If linkage groups are indicated in a translocation they should be in the order &#8216;transposed
                  from&#8217;; transposed to&#8217;. Chromosomal balancers of unknown structure can be designated using the abbreviation <span class="italic">C</span>, e.g., <span class="italic">mnC1(II)</span>. See WormBook chapter <a href="http://www.wormbook.org/chapters/www_geneticbalancers/geneticbalancers.html" target="_top">Genetic balancers</a> for a more detailed explanation.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Polymorphisms and other genetic elements
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4.1"></a>4.1.&nbsp;SNPs and RFLPs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Polymorphic sites, which are mostly SNPs (<span class="italic">s</span>ingle <span class="italic">n</span>ucleotide <span class="italic">p</span>olymorphisms) or RFLPs (<span class="italic">r</span>estriction <span class="italic">f</span>ragment <span class="italic">l</span>ength <span class="italic">p</span>olymorphisms) derived from natural isolates of <span class="italic">C. elegans</span> (and thus differ from the Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> reference) are designated by an italic letter <span class="italic">P</span> and an italic number, preceded by the allele designation for the laboratory responsible for identifying the site. For example,
                  <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00248812;class=Variation" target="_blank">stP17</a></span> and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00248840;class=Variation" target="_blank">stP196</a></span> are RFLPs identified in the laboratory of <a href="https://wormbase.org/resources/person/WBPerson1562#01--10" target="_top">R.H. Waterston</a>, and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00000159;class=Variation" target="_blank">amP6</a></span> and <span class="italic">amP15</span> are SNPs identified in the laboratory of <a href="https://wormbase.org/resources/person/WBPerson341#013542--10" target="_top">K. Kornfeld</a>.
               </p>
               <p>SNPs identified in whole genome sequencing projects from one or more natural isolates are not assigned a name but may be referred
                  to by their WBVariationID. For example, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar01710822;class=Variation" target="_blank">WBVar01710822</a></span> is an intronic SNP in the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003807;class=Gene" target="_blank">npr-1</a></span> gene found in various natural isolates. Other identifiers (such as those used in publications, internal IDs assigned by the
                  project or historical names) are incorporated into <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> to enable users to recover such entities through searches.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4.2"></a>4.2.&nbsp;Natural copy number variants
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Hundreds of independent natural isolates of <span class="italic">C. elegans</span> have been recovered from multiple locations around the world. The genomes of some of these isolates contain large (&gt;10 kb)
                  deletions, duplications, or insertions, relative to the reference wild-type strain, Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a>. Deletions are named with the prefix <span class="italic">niDf</span> (<span class="italic">n</span>atural <span class="italic">i</span>solate <span class="italic">d</span>e<span class="italic">f</span>iciency) followed by a number. Duplications and insertions are named with the prefix <span class="italic">niDp</span> (<span class="italic">n</span>atural <span class="italic">i</span>solate <span class="italic">d</span>u<span class="italic">p</span>lication or insertion), followed by a number. Numbers for <span class="italic">niDf</span> and <span class="italic">niDp</span> variants are assigned by application to: <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a></p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4.3"></a>4.3.&nbsp;Introgressed regions in near-isogenic lines.
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Genetic regions that have been introgressed from one natural isolate of <span class="italic">C. elegans</span> into the background of a different isolate are named in a manner similar to that used for deficiencies (<span class="italic">Df</span>) and duplications (<span class="italic"><a href="http://www.wormbase.org/db/get?name=Dp;class=Cell" target="_blank">Dp</a></span>). Each <span class="italic">I</span>ntrogressed <span class="italic">R</span>egion is given an italicized name consisting of the relevant laboratory allele designation, the letters <span class="italic">IR</span>, and a number. Thus, a region from the X chromosome of Hawaiian strain <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain" target="_blank">CB4856</a> crossed onto a Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> background, and created in the <a href="https://wormbase.org/resources/person/WBPerson5224#013542&#8212;10" target="_blank">Kruglyak</a> lab (allele code <span class="italic">qq</span>) has been given the name <span class="italic">qqIR1</span>. Additional information about genetic map location and strain origin can be provided in an optional parenthesis. So this
                  example could be more fully written as <span class="italic">qqIR1(X, <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain" target="_blank">CB4856</a>)</span>, with the implicit assumption that the strain background is Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a>. The strain background and the direction of introgression can also be specified, using the symbol &gt;, with this example being
                  written <span class="italic">qqR1(X, CB4856&gt;N2)</span>.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4.4"></a>4.4.&nbsp;Transposons and transposon insertions
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Types of <span class="italic">C. elegans</span> transposons are called Tc1, Tc2, etc., where each number represents a different family. Transposon names are not italicized
                  except when included in a genotype. Different natural isolates of <span class="italic">C. elegans</span> have different distributions of these transposons, which result in polymorphic differences from the reference wild-type strain
                  Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a>. The differences between natural isolates and Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> are given polymorphism names, as described below.
               </p>
               <p>The endogenous transposons of <span class="italic">C. elegans</span> can be mobilized to generate new insertional events. In addition, foreign transposons such as Mos1 can be introduced by transformation,
                  and then mobilized to create new insertions. All these newly generated transposon insertions can be named as simple alleles,
                  with an optional suffix indicating the nature of the transposon. They are treated as alleles of named genes if they are located
                  within the boundaries of a gene. Example: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00241309;class=Variation" target="_blank">r293</a></span> is a Tc1 insertion in the gene <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006789;class=Gene" target="_blank">unc-54</a></span>. An optional descriptor can also be added after a double colon to indicate the nature of the insertion. Example: <span class="italic">unc-54(<a href="http://www.wormbase.org/db/get?name=WBVar00241309;class=Variation" target="_blank">r293</a>::Tc1)</span>. Note that such insertions may often be silent in terms of gene activity, for example if an insertion occurs within an intron
                  and can be spliced out.
               </p>
               <p>Newly generated <span class="italic">T</span>ransposon <span class="italic">i</span>nsertions, especially those located in apparently intergenic regions, may also be given <span class="italic">Ti</span> (transposon insertion) names. These consist of the designation identifying the laboratory of origin, the two letters <span class="italic">Ti</span>, and a number, all italicized. Example: <span class="italic">eTi13</span> is an insertion of a Mos transposon into an intergenic region on <span class="italic">LGIII</span>.
               </p>
               <p>Transposon loci have ID names formed from &#8216;WBTransposon&#8217; followed by a unique number, like <span class="italic">WBTransposon00000623</span>.
               </p>
               <p>Their exon-like structure is curated as a Transposon_CDS (coding sequence) object with a name like <span class="italic"><a href="http://www.wormbase.org/db/get?name=C29E6.6;class=Gene" target="_blank"><a href="http://www.wormbase.org/db/get?name=WBGene00007803;class=Gene" target="_blank">C29E6.6</a></a></span> formed from the YAC or cosmid or clone they are on followed by a number which uniquely identifies it from the other CDS-like
                  objects on that clone, YAC or cosmid.
               </p>
               <p>Transposons and Transposon_CDS are not currently classed as genes in <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> and so do not have a parent gene object. The WBTransposon and representation on the Genome Browser should be viewed as analogous
                  to the WBGene and how it is displayed.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4.5"></a>4.5.&nbsp;Transgenes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Transformation of <span class="italic">C. elegans</span> with exogenous DNA by microinjection usually leads to the formation of a transmissible extrachromosomal array containing
                  many copies of the introduced DNA. Extrachromosomal arrays differ in their frequency of meiotic and mitotic transmission.
                  Extrachromosomal arrays can subsequently be integrated into the genome by irradiation. Direct integrative transformation with
                  exogenous DNA can be obtained by microparticle bombardment, mosSCI, or miniMos techniques. As these integrative events are
                  not associated with the endogenous locus corresponding to the exogenous DNA they are considered distinct from genome engineered
                  changes to the endogenous locus and thus have different designations.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4.5.1"></a>4.5.1.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Extrachromosomal arrays are given italicized names consisting of the laboratory allele prefix, the two letters <span class="italic">Ex</span>, and a number.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4.5.2"></a>4.5.2.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Integrated transgenes are designated by italicized names consisting of the laboratory allele prefix, the two letters <span class="italic">Is</span>, and a number. Single copy integrants, usually generated by the MosSCI or miniMos insertion techniques, are a subset of integrated
                     transgenes and are designated by italicized names consisting of the laboratory allele prefix, the two letters <span class="italic">Si</span>, and a number.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4.5.3"></a>4.5.3.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Transgene designations <span class="italic">Ex</span>, <span class="italic">Is,</span> and <span class="italic">Si</span> can optionally be followed by genotypic or molecular information describing the transgene in square brackets. For example,
                     <span class="italic">eEx3</span> or <span class="italic">eIs2</span> or <span class="italic">stEx5[sup-7(<a href="http://www.wormbase.org/db/get?name=WBVar00248701;class=Variation" target="_blank">st5</a>) unc-22(+)]</span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4.5.4"></a>4.5.4.&nbsp;&#8195;
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Gene fusions incorporated in transgenes that consist of a <span class="italic">C. elegans</span> gene or part thereof fused to a reporter such as <span class="italic">lacZ</span> or <span class="italic">GFP</span> are indicated by the <span class="italic">C. elegans</span> gene name followed by two colons and the reporter, all italicized: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003976;class=Gene" target="_blank">pes-1</a>::lacZ, <a href="http://www.wormbase.org/db/get?name=WBGene00003106;class=Gene" target="_blank">mab-9</a>::GFP</span>. To distinguish between transcriptional and translational fusions, a lowercase italicized <span class="italic">p</span> following the gene name has been used to indicate transcriptional fusions, e.g., <span class="italic">mab-9p::GFP</span>.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;Genotypes
                     </h2>
                  </div>
               </div>
            </div>
            <p>The genotype of an animal is specified by listing all known differences between its genotype and that of wild type, which
               is defined by convention as Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a>. Each such difference is assigned a unique name. <a href="#table1" title="Table 1. Nomenclature terms and usage in C. elegans research">Table 1</a> lists the currently recognized types of difference that have designations, described at greater length elsewhere in the chapter.
            </p>
            <div class="table"><a name="table1"></a><p class="title">Table 1. Nomenclature terms and usage in <span class="italic">C. elegans</span> research
               </p>
               <table summary="Table 1. Nomenclature terms and usage in C. elegans research" border="1">
                  <colgroup>
                     <col>
                     <col>
                  </colgroup>
                  <tbody>
                     <tr valign="top">
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=R107.8;class=Gene" target="_blank">R107.8</a></td>
                        <td align="left" valign="top">Systematic gene identification (the 8<sup>th</sup> predicted gene on cosmid <a href="http://www.wormbase.org/db/get?name=R107;class=Clone" target="_blank">R107</a>)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">lin</span></td>
                        <td align="left" valign="top">Gene class, &#8220;abnormal cell LINeage&#8221;</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene" target="_blank">lin-12</a></span></td>
                        <td align="left" valign="top">The 12<sup>th</sup> &#8220;abnormal cell LINeage&#8221; gene named
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00000218;class=Variation" target="_blank">ar170</a></span></td>
                        <td align="left" valign="top">Allele name (&#8220;<span class="italic">ar</span>&#8221; allele designation from the <a href="https://wormbase.org/resources/person/WBPerson220#01--10" target="_top">Greenwald</a> lab; the 170<sup>th</sup> allele generated in the <a href="https://wormbase.org/resources/person/WBPerson220#01--10" target="_top">Greenwald</a> lab)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=WP:CE00274;class=Protein" target="_blank">LIN-12</a></td>
                        <td align="left" valign="top">Protein name (product of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene" target="_blank">lin-12</a></span> gene)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000093;class=Phenotype" target="_blank">Lin</a></td>
                        <td align="left" valign="top">Phenotype (abnormal cell <span class="bold"><strong>lin</strong></span>eage phenotype)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00000218;class=Variation" target="_blank">ar170</a>)</span> or <span class="italic">lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00000218;class=Variation" target="_blank">ar170</a>)/lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00000218;class=Variation" target="_blank">ar170</a>)</span></td>
                        <td align="left" valign="top">Homozygous for <span class="italic">lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00000218;class=Variation" target="_blank">ar170</a>)</span> allele
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00000218;class=Variation" target="_blank">ar170</a>)/+</span></td>
                        <td align="left" valign="top">Heterozygous for <span class="italic">lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00000218;class=Variation" target="_blank">ar170</a>)</span> allele
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00000218;class=Variation" target="_blank">ar170</a>)/lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00089843;class=Variation" target="_blank">n941</a>)</span></td>
                        <td align="left" valign="top">Heterozygous for two different <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene" target="_blank">lin-12</a></span> alleles (also call a compound heterozygote) (&#8220;<span class="italic">n</span>&#8221; allele designation from the <a href="https://wormbase.org/resources/person/WBPerson268#01--10" target="_top">Horvitz</a> lab)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00089665;class=Variation" target="_blank">n676n930</a>)</span></td>
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00089835;class=Variation" target="_blank">n930</a></span> is an intragenic revertant of the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00089663;class=Variation" target="_blank">n676</a> <a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene" target="_blank">lin-12</a></span> gain of function allele
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00404594;class=Variation" target="_blank">gk181351</a></span></td>
                        <td align="left" valign="top">Allele of <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene" target="_blank">lin-12</a></span> from the Million mutant project, MMP (<span class="italic">&#8220;gk</span>&#8221; MMP allele numbering starts at 100000)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00070143;class=Variation" target="_blank">WBVar00070143</a></span></td>
                        <td align="left" valign="top">Single Nucleotide Polymorphism (SNP) in the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene" target="_blank">lin-12</a></span> 3&#8217;UTR found in the natural isolate <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain" target="_blank">CB4856</a> (Hawaiian strain)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00248812;class=Variation" target="_blank">stP17</a></span></td>
                        <td align="left" valign="top">Restriction Fragment Length Polymorphism (RFLP) (&#8220;<span class="italic">st</span>&#8221; allele designation from the <a href="https://wormbase.org/resources/person/WBPerson1562#01--10" target="_top">Waterston</a> lab)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">lin-12p::gfp</span></td>
                        <td align="left" valign="top">GFP transcriptional fusion (using only the promoter of the gene)</td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene" target="_blank">lin-12</a>::gfp</span></td>
                        <td align="left" valign="top">GFP translational fusion (in which <span class="italic">gfp</span> is inserted at the C-terminus of the <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene" target="_blank">lin-12</a></span> coding sequence)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><a href="http://www.wormbase.org/db/get?name=GS60;class=Strain" target="_blank">GS60</a></td>
                        <td align="left" valign="top">Strain name (&#8220;GS&#8221; laboratory/strain designation from the <a href="https://wormbase.org/resources/person/WBPerson220#01--10" target="_top">Greenwald</a> lab); full genotype <span class="italic">unc-32(<a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation" target="_blank">e189</a>) lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00089665;class=Variation" target="_blank">n676n930</a>) III</span>.
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">lin-41(tn1490)</span></td>
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003026;class=Gene" target="_blank">lin-41</a></span> allele generated by chemical mutagenesis (&#8220;<span class="italic">tn</span>&#8221; allele designation from the <a href="https://wormbase.org/resources/person/WBPerson219#01--10" target="_top">Greenstein</a> lab)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">lin-41(xe11)</span></td>
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003026;class=Gene" target="_blank">lin-41</a></span> allele generated by genome engineering (&#8220;<span class="italic">xe</span>&#8221; allele designation from the <a href="https://wormbase.org/resources/person/WBPerson2384#01--10" target="_top">Grosshans</a> lab)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">lin-41(tn1490</span>[G883E]<span class="italic">)</span></td>
                        <td align="left" valign="top">Amino acid change in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003026;class=Gene" target="_blank">lin-41</a></span> allele <span class="italic">tn1490</span> indicated
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">mnDp26</span></td>
                        <td align="left" valign="top">Duplication (<span class="italic"><a href="http://www.wormbase.org/db/get?name=Dp;class=Cell" target="_blank">Dp</a></span>) (&#8220;<span class="italic">mn</span>&#8221; allele designation from the <a href="https://wormbase.org/resources/person/WBPerson252#013542--10" target="_top">Herman</a> lab)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">nDf17</span></td>
                        <td align="left" valign="top">Deficiency (<span class="italic">Df</span>) (multi-gene deletion)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">nT1 (IV; V)</span></td>
                        <td align="left" valign="top">Translocation (<span class="italic">T</span>) involving chromosomes <span class="italic">IV</span> and <span class="italic">V</span></td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBTransgene00008917;class=Transgene" target="_blank">rtEx726</a></span></td>
                        <td align="left" valign="top">Extrachromosomal (<span class="italic">Ex</span>) transgene array (&#8220;<span class="italic">rt</span>&#8221; allele designation from the <a href="https://wormbase.org/resources/person/WBPerson242#013542--10" target="_top">Hart</a> lab)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBTransgene00000058;class=Transgene" target="_blank">arIs80</a></span></td>
                        <td align="left" valign="top">Integrated (<span class="italic">Is</span>) transgene
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic"><a href="http://www.wormbase.org/db/get?name=WBTransgene00015560;class=Transgene" target="_blank"><a href="http://www.wormbase.org/db/get?name=WBVar00604219;class=Variation" target="_blank">dotSi110</a></a></span></td>
                        <td align="left" valign="top">Single copy insertion (<span class="italic">Si</span>) (&#8220;<span class="italic">dot</span>&#8221; allele designation from the J. <a href="https://wormbase.org/resources/person/WBPerson2168#013542--10" target="_top">Chen</a> lab)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">glc-1(<a href="http://www.wormbase.org/db/get?name=WBVar00239197;class=Variation" target="_blank">pk54</a>::Tc1)</span></td>
                        <td align="left" valign="top">Transposon (Tc1) insertion in <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001591;class=Gene" target="_blank">glc-1</a></span> gene (&#8220;<span class="italic">pk</span>&#8221; allele designation from the <a href="https://wormbase.org/resources/person/WBPerson490#013542--10" target="_top">Plasterk</a> lab)
                        </td>
                     </tr>
                     <tr valign="top">
                        <td align="left" valign="top"><span class="italic">pgIR2</span></td>
                        <td align="left" valign="top">Introgressed region (<span class="italic">IR</span>); full genotype <span class="italic">(II, CB4856&gt;N2)</span>, which indicates that a region from chromosome <span class="italic">II</span> of the Hawaiian strain <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain" target="_blank">CB4856</a> has been crossed into the Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a> background (&#8220;<span class="italic">pg</span>&#8221; allele designation from the <a href="https://wormbase.org/resources/person/WBPerson209#013542--10" target="_top">M. Goodman</a> lab).
                        </td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <p>Where necessary, wild type sequence can be indicated using the symbol +. Because every genetic &#8220;feature&#8221; (i.e., difference
               from Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain" target="_blank">N2</a>) has a unique name, an animal's genotype is fully specified by listing all the named features that it carries. Example: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBVar00144583;class=Variation" target="_blank">e2123</a>; <a href="http://www.wormbase.org/db/get?name=WBTransgene00000956;class=Transgene" target="_blank">mdIs18</a></span>.
            </p>
            <p>For clarity and convenience, additional information about genes, chromosomes, transgene contents, etc., can be added as described
               elsewhere in this document, to produce a more informative genotype. Example: <span class="italic">pha-1(e2123ts) III; mdIs18[pha-1(+) <a href="http://www.wormbase.org/db/get?name=WBGene00006756;class=Gene" target="_blank">unc-17</a>::GFP]</span></p>
            <p>Strains carrying more than one mutation are designated by sequentially listing mutant genes or alleles according to the left-right
               (= up-down) order on the genetic map/genome sequence. Different chromosomes (linkage groups) are separated by a semicolon
               and given in the order <span class="italic">I, II, III, IV, V, X, f, M. I-V</span> are the five autosomes, <span class="italic">X</span> is the <span class="italic">X</span> chromosome, <span class="italic">f</span> refers to free duplications or chromosomal fragments, and <span class="italic">M</span> is the mitochondrial genome. For example: <span class="italic">dpy-5(<a href="http://www.wormbase.org/db/get?name=WBVar00142941;class=Variation" target="_blank">e61</a>) I; bli-2(<a href="http://www.wormbase.org/db/get?name=WBVar00143470;class=Variation" target="_blank">e768</a>) II; unc-32(<a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation" target="_blank">e189</a>) III</span>.
            </p>
            <p>Integrated transgenes (<span class="italic">Is</span> and <span class="italic">Si</span>) should be grouped with other mutant genes or alleles on the chromosome, if it is known into which chromosome the transgene
               is integrated. Extrachromosomal arrays (<span class="italic">Ex</span>) and unmapped integrated arrays should be included at the end of the genotype. Example: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBTransgene00017864;class=Transgene" target="_blank"><a href="http://www.wormbase.org/db/get?name=WBVar01473985;class=Variation" target="_blank">oxTi330</a></a> III; <a href="http://www.wormbase.org/db/get?name=WBTransgene00017865;class=Transgene" target="_blank">gaIs283</a></span>.
            </p>
            <p>Heterozygotes, with allelic differences between chromosomes are designated by separating mutations on the two homologous chromosomes
               with a slash. For example, the compound heterozygote <span class="italic">lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00089843;class=Variation" target="_blank">n941</a>)/lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00089283;class=Variation" target="_blank">n137</a>)</span>. Where unambiguous, wild type alleles can be designated by a plus sign alone, or even omitted. For example, <span class="italic">dpy-5(<a href="http://www.wormbase.org/db/get?name=WBVar00142941;class=Variation" target="_blank">e61</a>) unc-13(+)/dpy-5(+) unc-13(<a href="http://www.wormbase.org/db/get?name=WBVar00142935;class=Variation" target="_blank">e51</a>) I</span> can also be written <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00001067;class=Gene" target="_blank">dpy-5</a> +/+ <a href="http://www.wormbase.org/db/get?name=WBGene00006752;class=Gene" target="_blank">unc-13</a> or <a href="http://www.wormbase.org/db/get?name=WBGene00001067;class=Gene" target="_blank">dpy-5</a>/unc-13.</span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;Phenotypes
                     </h2>
                  </div>
               </div>
            </div>
            <p>Phenotypic characteristics can be described in words, e.g., <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000583;class=Phenotype" target="_blank">dumpy</a> animals or <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000643;class=Phenotype" target="_blank">uncoordinated</a> animals. If more convenient, a nonitalicized 3-letter or 4-letter abbreviation, which usually corresponds to a gene class
               or gene name, may be used. The first letter of a phenotypic abbreviation is capitalized, e.g., <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000643;class=Phenotype" target="_blank">Unc</a> for <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000643;class=Phenotype" target="_blank">uncoordinated</a>, <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000583;class=Phenotype" target="_blank">Dpy</a> for <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000583;class=Phenotype" target="_blank">dumpy</a>. If it is necessary to distinguish among related but distinguishable phenotypes, the relevant gene number can be added, e.g.,
               Unc-4 and Unc-13 to differentiate the distinct phenotypes produced by mutations in the two genes <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006744;class=Gene" target="_blank">unc-4</a></span> and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006752;class=Gene" target="_blank">unc-13</a></span>. <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> maintains a standard set of defined phenotype descriptors (the <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> Phenotype Ontology).
            </p>
            <p>Abbreviations that do not correspond to a gene class or gene name can also be used, e.g., <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000700;class=Phenotype" target="_blank">Muv</a> for <span class="italic">mu</span>ltiple <span class="italic">v</span>ulval development, and Daf-c for <span class="italic">da</span>uer-<span class="italic">f</span>ormation-<span class="italic">c</span>onstitutive. To avoid ambiguity and conflicts, phenotype abbreviations <span class="underline">not</span> corresponding to a gene name are controlled by <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> and requests for names should be made, prior to abstract or manuscript submission, via email to: <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a>.
            </p>
            <p>A common and accepted convention, when comparing a mutant with the wild type, is to use the prefix non- to refer to the wild
               type phenotype, for example, non-Lin (= wild type cell lineage) or <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000583;class=Phenotype" target="_blank">Dpy</a> non-Unc (= wild type with respect to movement, but <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0000583;class=Phenotype" target="_blank">dumpy</a> with respect to body shape).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec7"></a>7.&nbsp;Proteins
                     </h2>
                  </div>
               </div>
            </div>
            <p>The protein product of a gene can be referred to by the relevant gene name, written in non-italic capitals, e.g., the protein
               encoded by <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006752;class=Gene" target="_blank">unc-13</a></span> can be called UNC-13.
            </p>
            <p>In some cases the gene name and the protein products have distinct names, often in situations where the gene name is based
               on phenotype and the protein product is named based on sequence similarity or biochemical activity. For example, for the gene
               <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene" target="_blank">let-60</a></span> and the corresponding protein RAS, designation <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene" target="_blank">let-60</a></span>/RAS or <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene" target="_blank">let-60</a></span> RAS can be used.
            </p>
            <p>Where more than one protein product is predicted for a gene (usually as a result of alternative message processing), the different
               proteins are distinguished by adding &#8216;isoform&#8217; and then the isoform letter derived from the isoform letter of the name of
               the <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> CDS, e.g., the gene &#8216;tra-1&#8217; has two CDS isoforms: &#8216;Y47D3A.6a&#8217; and &#8216;Y47D3A.6b&#8217; which give rise to the protein isoforms: &#8216;TRA-1,
               isoform a&#8217; and &#8216;TRA-1, isoform b&#8217;.
            </p>
            <p>Mutant protein products can be named by the missense change, for example a mutant &#8216;TRA-1, isoform a&#8217; protein with a <a href="http://www.wormbase.org/db/get?name=WBPhenotype:0001033;class=Phenotype" target="_blank">Pro</a> to Leu change at codon 79 would be written: &#8216;TRA-1, isoform a (P79L)&#8217;.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec8"></a>8.&nbsp;Strains
                     </h2>
                  </div>
               </div>
            </div>
            <p>A strain is a set of individuals of a particular genotype with the capacity to produce more individuals of the same genotype.
               Strains are given non-italicized names consisting of two or three uppercase letters followed by a number. The strain letter
               prefixes refer to the laboratory of origin and are distinct from the allele (mutation) letter prefixes. Examples: CB1833 is
               a strain of genotype <span class="italic">dpy-5(<a href="http://www.wormbase.org/db/get?name=WBVar00142941;class=Variation" target="_blank">e61</a>) unc-13(<a href="http://www.wormbase.org/db/get?name=WBVar00142935;class=Variation" target="_blank">e51</a>)</span>, originally constructed by <a href="https://wormbase.org/resources/person/WBPerson77#01--10" target="_top">S. Brenner</a> at the MRC Laboratory of Molecular Biology (strain designation CB, allele designation <span class="italic">e</span>), and <a href="http://www.wormbase.org/db/get?name=MT688;class=Strain" target="_blank">MT688</a> is a strain of genotype <span class="italic">unc-32(<a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation" target="_blank">e189</a>) +/+ lin-12(<a href="http://www.wormbase.org/db/get?name=WBVar00089283;class=Variation" target="_blank">n137</a>) III; him-5(<a href="http://www.wormbase.org/db/get?name=WBVar00144023;class=Variation" target="_blank">e1467</a>) V</span>, constructed in the laboratory of <a href="https://wormbase.org/resources/person/WBPerson268#01--10" target="_top">H.R. Horvitz</a> at M.I.T. (strain designation MT, allele designation <span class="italic">n</span>).
            </p>
            <p>Strains can and should be preserved as frozen stocks at &#8722;70 &deg;C, or ideally in liquid nitrogen, in order to ensure long-term
               maintenance and to avoid drift or accumulation of modifier mutations.
            </p>
            <p>Bacterial strain names employ the two or three letter Laboratory/Strain designation, followed by &#8220;b&#8221;. For example, CBb#. This
               facilitates distinguishing nematode strains from bacterial strains. Please provide full information on species and relevant
               genotype of the bacteria.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec9"></a>9.&nbsp;Other nematodes
                     </h2>
                  </div>
               </div>
            </div>
            <p>Research and genomic analysis of non-<span class="italic">C. elegans</span> species is increasing rapidly. An important mission of <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> is to make available information for each species listed in the Overview section within the database structure developed
               for <span class="italic">C. elegans</span>. For these organisms <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> will also supervise gene naming in order to maximize consistency with <span class="italic">C. elegans</span>. It is recommended that nomenclature in general should follow the principles used for <span class="italic">C. elegans</span>, as far as possible. Gene name proposals and queries should be made online via <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> or sent to <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a></p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9.1"></a>9.1.&nbsp;Species prefixes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In order to specify unambiguously the nematode species-of-origin, an optional 3-letter standard prefix and hyphen can be added
                  to the gene name (<a href="#table2" title="Table 2. Species prefixes currently in use">Table 2</a>). Examples: the <span class="italic">C. briggsae</span> and <span class="italic">Pristionchus pacificus</span> orthologs of <span class="italic">C. elegans</span> <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00006604;class=Gene" target="_blank">tra-1</a></span> are called <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00033987;class=Gene" target="_blank">Cbr-tra-1</a></span> and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00114980;class=Gene" target="_blank">Ppa-tra-1</a></span>, respectively. <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> coordinates the species prefix designations to avoid the use of the same designation for more than one species; contact <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a> for prefix proposals.
               </p>
               <div class="table"><a name="table2"></a><p class="title">Table 2. Species prefixes currently in use</p>
                  <table summary="Table 2. Species prefixes currently in use" border="1">
                     <colgroup>
                        <col>
                        <col>
                     </colgroup>
                     <thead>
                        <tr valign="bottom">
                           <th align="left" valign="bottom"><span class="bold"><strong>Species</strong></span></th>
                           <th align="left" valign="bottom"><span class="bold"><strong>Prefix</strong></span></th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="italic">C. elegans</span></td>
                           <td align="left" valign="top"><span class="italic">Cel-</span></td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="italic">C. briggsae</span></td>
                           <td align="left" valign="top"><span class="italic">Cbr-</span></td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="italic">C. remanei</span></td>
                           <td align="left" valign="top"><span class="italic">Cre-</span></td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="italic">C. brenneri</span></td>
                           <td align="left" valign="top"><span class="italic">Cbn-</span></td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="italic">C. japonica</span></td>
                           <td align="left" valign="top"><span class="italic">Cpj-</span></td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="italic">Heterorhabditis bacteriophora</span></td>
                           <td align="left" valign="top"><span class="italic">Hba-</span></td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="italic">Oscheius tipulae</span></td>
                           <td align="left" valign="top"><span class="italic">Oti-</span></td>
                        </tr>
                        <tr valign="top">
                           <td align="left" valign="top"><span class="italic">Pristionchus pacificus</span></td>
                           <td align="left" valign="top"><span class="italic">Ppa-</span></td>
                        </tr>
                        <tr valign="top">
                           <td colspan="2" align="left" valign="top">For additional prefixes since publication of this article see <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a></td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9.2"></a>9.2.&nbsp;Gene naming: homologous genes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Genes predicted from whole genome sequences in other nematode species will, in many cases, have identifiable close homologs
                  in <span class="italic">C. elegans</span>, for which approved names already exist. In these cases, the same name should be used as in <span class="italic">C. elegans</span>, with the relevant species identifier. <span class="bold"><strong></strong></span> Possible scenarios are listed below.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9.2.1"></a>9.2.1.&nbsp;One-to-one
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Where one gene in <span class="italic">C. elegans</span> corresponds to a single gene in another nematode species, ortholog naming can be applied automatically. Example: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00020172;class=Gene" target="_blank">thoc-1</a></span> in <span class="italic">C. elegans</span> has a <span class="italic">C. briggsae</span> ortholog, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00042381;class=Gene" target="_blank">Cbr-thoc-1</a></span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9.2.2"></a>9.2.2.&nbsp;One-to-many
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Where one gene in <span class="italic">C. elegans</span> is related to multiple genes (paralogs) in another nematode species, these paralogs can be named using additional decimal
                     numbers. Example: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00009341;class=Gene" target="_blank">thoc-3</a></span> in <span class="italic">C. elegans</span> has two <span class="italic">C. briggsae</span> paralogs, <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00033360;class=Gene" target="_blank">Cbr-thoc-3.1</a></span> and <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00043048;class=Gene" target="_blank">Cbr-thoc-3.2</a></span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9.2.3"></a>9.2.3.&nbsp;Many-to-one
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Where multiple genes exist in <span class="italic">C. elegans</span> but only a single gene in another nematode species, it is recommended that either the most closely similar, or the lowest
                     numbered <span class="italic">C. elegans</span> gene, be used to name the single gene, as appropriate.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9.2.4"></a>9.2.4.&nbsp;Many-to-many.
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Where multiple closely related genes can be identified in both species but the phylogenetic relationships of the two sets
                     are complex, new gene numbers can be assigned to the set of genes in the other nematode species, after consultation with <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a>
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9.2.5"></a>9.2.5.&nbsp;A standard gene name has not yet been assigned in <span class="italic">C. elegans</span></h4>
                        </div>
                     </div>
                  </div>
                  <p>The gene can be referred to using the sequence name, such as <span class="italic">Hba-W01B11.3</span>. However, in such cases it will usually be both feasible and desirable to assign a standard name to the <span class="italic">C. elegans</span> gene as well, at the same time.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9.3"></a>9.3.&nbsp;Gene naming: non-homologous genes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>It is expected that many genes in other nematode species will lack obvious close homologs in <span class="italic">C. elegans</span>, because of loss or substantial divergence during the evolution of <span class="italic">C. elegans</span>. These genes can be given new gene numbers if they belong to an identifiable named class in <span class="italic">C. elegans</span>, or else new gene name classes can be established for them. In either case, assignment of an approved name should be made
                  after consultation with <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a></p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9.4"></a>9.4.&nbsp;Gene naming: forward genetics
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A significant amount of mutation-based forward genetic analysis is being pursued in nematodes other than <span class="italic">C. elegans</span>, in particular using other species of <span class="italic">Caenorhabditis</span> (<span class="italic">C. briggsae, C. remanei, C. brenneri</span>, and others), as well as species of <span class="italic">Oscheius</span> and <span class="italic">Pristionchus</span>. It is expected that most, but not all, of the mutationally-defined genes discovered in these species will prove to have
                  orthologs with equivalent or similar function in <span class="italic">C. elegans</span>, and hence that standard genetic names will have been approved already. Possible situations are described below.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9.4.1"></a>9.4.1.&nbsp;The molecular identity is known and orthology is obvious
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>It is recommended that the <span class="italic">C. elegans</span> name be used, with the appropriate species identifier prefix. Example: <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00195120;class=Gene" target="_blank">Ppa-mab-5</a></span> is the <span class="italic">Pristionchus pacificus</span> ortholog of <span class="italic">C. elegans</span> <span class="italic"><a href="http://www.wormbase.org/db/get?name=WBGene00003102;class=Gene" target="_blank">mab-5</a></span>.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9.4.2"></a>9.4.2.&nbsp;The molecular identity is not initially known but the mutant phenotype corresponds to a known <span class="italic">C. elegans</span> mutant phenotype
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>To reduce gene renaming as much as possible, it is recommended that: a) new mutations should be mapped as precisely as possible,
                     b) all relevant complementation tests should be done, and c) if the genome assembly for that species is sufficiently accurate,
                     potential <span class="italic">C. elegans</span> orthologs should be tested by sequencing or RNAi. If the gene still seems to be novel, then a new gene class name can be
                     established, following consultation with <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a> in order to ensure that the new name is available and appropriate. An appropriate name should describe the phenotype yet
                     be distinct from the corresponding <span class="italic">C. elegans</span> name. If the molecularly identity is subsequently found to be conserved in <span class="italic">C. elegans</span>, then the name would typically revert to the <span class="italic">C. elegans</span> name, but this would be determined on a case-by-case basis.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9.4.3"></a>9.4.3.&nbsp;The molecular identity is unknown and the mutant phenotype does not correspond to a known <span class="italic">C. elegans</span> mutant phenotype
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A new gene class name can be established, following consultation with <a href="mailto:genenames@wormbase.org" target="_top">genenames@wormbase.org</a> in order to ensure that the new name is available and appropriate. Example: <span class="italic">cov</span> = <span class="italic">C</span>ompetence and/or centering <span class="italic">O</span>f <span class="italic">V</span>ulva abnormal.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec10"></a>10.&nbsp;Acknowledgements
                     </h2>
                  </div>
               </div>
            </div>
            <p>This work is funded by U41 HG002223D to <a href="https://wormbase.org/#012-34-5" target="_top">WormBase</a> (M.A.T and T.S.) and NIH-ORIP (P40 OD010440) to the <a href="https://cgc.umn.edu/" target="_top">CGC</a> (A.D.). The following individuals made substantial contributions to the <span class="italic">C. elegans</span> nomenclature system and its utilization: <a href="https://wormbase.org/resources/person/WBPerson77#01--10" target="_top">Sydney Brenner</a>, <a href="https://wormbase.org/resources/person/WBPerson268#01--10" target="_top">Robert Horvitz</a>, <a href="https://wormbase.org/resources/person/WBPerson261#013542--10" target="_top">Jonathan Hodgkin</a>, <a href="https://wormbase.org/resources/person/WBPerson252#013542--10" target="_top">Robert Herman</a>, <a href="https://wormbase.org/resources/person/WBPerson21#013542--10" target="_top">Phil Anderson</a>, <a href="https://wormbase.org/resources/person/WBPerson154#013542--10" target="_top">Mark Edgley</a> and <a href="https://wormbase.org/resources/person/WBPerson521#013542--10" target="_top">Don Riddle</a>.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec11"></a>11.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed"><span class="bibliomisc">Brenner S. (1974). The genetics of <span class="italic">Caenorhabditis elegans</span>. Genetics <span class="italic">77</span>, 71-94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4366476&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://www.genetics.org/content/77/1/71.long" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed"><span class="bibliomisc">Harris T.W., Baran, J., Bieri, T., Cabunoc, A., Chan, J., Chen, W.J., Davis, P., Done, J., Grove, C., Howe, K., et al. (2014).
                        WormBase 2014: new views of curated biology. Nucleic Acids Res. <span class="italic">42</span> (Database Issue) D789-793.</span>
                     <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=WormBase+2014%3A+new+views+of+curated+biology" target="_blank">Abstract</a>
                     <a href="https://academic.oup.com/nar/article/42/D1/D789/1046684" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed"><span class="bibliomisc">Horvitz H.R., Brenner, S., Hodgkin, J., and Herman, R.K. (1979). A uniform genetic nomenclature for the nematode <span class="italic">Caenorhabditis elegans</span>. Mol. Gen. Genet. <span class="italic">175</span>, 129-133.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=292825&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://post.queensu.ca/~chinsang/a-worm-reader/horvitz-et-al-1979.pdf" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed"><span class="bibliomisc">The <span class="italic">C. elegans</span> Sequencing Consortium. (1998). Genome sequence of the nematode <span class="italic">C. elegans</span>: a platform for investigating biology. Science <span class="italic">282</span>, 2012-2018.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9851916&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://science.sciencemag.org/content/282/5396/2012.long" target="_blank">Article</a>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Paul Sternberg. Last revised November 25, 2015. Published in its final form August 8, 2018. This chapter should be cited as: Tuli M.A.,
                  Daul A., Schedl T. <span class="italic">Caenorhabditis</span> nomenclature. (August 8, 2018), <span class="italic">WormBook</span>, ed. The <span class="italic">C. elegans</span> Research Community, WormBook, doi/10.1895/wormbook.1.183.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2016 Mary Ann Tuli, Aric Daul, Tim Schedl. This is an open-access article distributed under the terms of the Creative Commons
                  Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author
                  and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e44" href="#d0e44">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:maryann.tuli@wormbase.org" target="_top">maryann.tuli@wormbase.org</a> or <a href="mailto:ts@genetics.wustl.edu" target="_top">ts@genetics.wustl.edu</a></p>
 <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" -->
   </body>
</html>