<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Reporter gene fusions
         			
         		
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMGFPReporter_ArticleID" title="Reporter gene fusions&#xA;&#xA;">
      <link rel="next" href="#d0e55" title="1.&nbsp;General considerations">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="reportergenefusions.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMGFPReporter_ArticleID"></a>Reporter gene fusions
                     			<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     		
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Thomas Boulin<em><span class="remark"><sup><a name="d0e37" href="#ftn.d0e37">&sect;</a></sup></span></em>, John F. Etchberger<em><span class="remark">
                                 					<sup><a name="d0e37" href="#ftn.d0e37">&sect;</a></sup>
                                 				</span></em>, Oliver Hobert<em><span class="remark">
                                 					<sup><a name="d0e44" href="#ftn.d0e44">&#8224;</a></sup>
                                 				</span></em>
                           			</span><br></span><span class="affiliation"><span class="address">Howard Hughes Medical Institute, Department of Biochemistry and Molecular Biophysics, Center for Neurobiology and Behavior,
                           Columbia University, New York, NY 10032 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods<br />>> Gene expression
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=307.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Reporter+gene+fusions" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e55">1. General considerations</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e58">1.1. Introduction</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e115">1.2. Categories of reporter gene constructs</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e199">1.3. Sequences to be included in reporter constructs</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e256">1.4. Genome-wide gene expression projects</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e285">1.5. Technical considerations</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e470">1.6. Multicolor GFP expression systems</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e487">1.7. Common artifacts</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e513">1.8. Fluorophore maturation of fluorescent proteins</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e543">1.9. Coinjection markers</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e613">1.10. Concluding remark</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e632">2. Protocols</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e635">2.1. Protocol 1: Generating GFP reporter constructs using standard cloning techniques</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e802">2.2. Protocol 2: Generating GFP reporters using a PCR fusion approach</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1018">2.3. Protocol 3: Generating GFP reporters using <span class="emphasis"><em>in vivo</em></span> recombination</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1323">2.4. Protocol 4: Generating GFP reporter constructs using Gateway<span class="symbol">&#63194;</span> cloning technology</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e1694">2.5. Protocol 5: Generating GFP reporter constructs using yeast homologous recombination</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e2015">2.6. Protocol 6: Cell type-specific GFP expression from reconstituted reporter genes</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e2139">3. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e55"></a>1.&nbsp;General considerations
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e58"></a>1.1.&nbsp;Introduction
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. elegans</em></span> has distinguishing characteristics that predestine this organism for the visualization of gene expression patterns using
                  the reporter genes green fluorescent protein (GFP; <a href="#bib2">Chalfie et al., 1994</a>) and <span class="symbol">&#946;</span>GAL (LacZ; <a href="#bib6">Fire et al., 1990</a>). The transparency of <span class="emphasis"><em>C. elegans</em></span> allows for microscopic analysis <span class="emphasis"><em>in vivo</em></span> without animal dissection. In addition, the relative thinness of animals (approximately 75 <span class="symbol">&#956;</span>m cross-section) often alleviates the need for high-powered confocal microscopy. Finally, germline transformation techniques
                  allow for the rapid generation of transgenic animals.
               </p>
               <p>The <span class="emphasis"><em>E. coli</em></span> gene <span class="emphasis"><em>LacZ</em></span>, encoding <span class="symbol">&#946;</span>-galactosidase, was first used for single-cell gene expression analysis in <span class="emphasis"><em>C. elegans</em></span> in 1990 (<a href="#bib6">Harrison et al., 1990</a>) and was the reporter gene of choice until the introduction of GFP in 1994 (<a href="#bib2">Chalfie et al., 1994</a>). The primary advantage of GFP over LacZ is the ability to visualize reporter gene expression in live animals rather than
                  in fixed preparations. Therefore, GFP can be more easily used for a variety of purposes including (1) expression pattern analysis,
                  (2) dissection of <span class="emphasis"><em>cis</em></span>-regulatory sequences, (3) protein localization, (4) visualization of cellular anatomy (e.g., neuroanatomy), (5) cell identification,
                  and (6) visualization of cellular and physiological processes (<a href="#bib2">Chalfie et al., 1994</a>; <a href="#bib27">Hobert and Loria, 2005</a>). For these reasons, GFP reporter transgenes have become the primary tool for gene expression analysis in <span class="emphasis"><em>C. elegans</em></span>. We will focus on GFP from now on, keeping in mind that in many cases LacZ could substitute for GFP. In this chapter, we
                  describe protocols for the generation of GFP reporter constructs and provide several technical considerations.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e115"></a>1.2.&nbsp;Categories of reporter gene constructs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A crucial aspect of gene expression studies is choosing an adequate type of reporter for the purposes of the experiment. Each
                  reporter differs in the amount of information it provides about the expression of a gene. The three general types of reporter
                  gene constructs are: 1) transcriptional reporters, 2) translational reporters, and 3) "<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span>-based" transcriptional reporters (<a href="#figure1">Figure 1</a>).
               </p>
               <p><span class="emphasis"><em><span class="bold"><strong>Transcriptional reporters</strong></span></em></span> consist of a promoter fragment from a gene of interest driving GFP (<a href="#figure1">Figure 1</a>A). Typically, promoter fragments of a few kilobases immediately upstream of the start codon contain a significant portion
                  of the <span class="emphasis"><em>cis</em></span>-regulatory information necessary to provide a tentative expression pattern of the endogenous gene under study.
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="GFPRfig1.jpg"><img src="GFPRfig1_s.jpg" border="2" align="middle" alt="Architecture of reporter gene constructs figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. Architecture of reporter gene constructs.&nbsp;</b>(A) Transcriptional reporter, (B) translational reporter, (C) "<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span>-based" reporter. Non-coding DNA sequences are indicated by black lines. Exon sequences are represented as blue boxes. GFP
                        is shown in green.
                     </p>
                  </div>
               </div>
               <p><span class="emphasis"><em><span class="bold"><strong>Translational reporters</strong></span></em></span> are in-frame gene fusions between GFP and a gene of interest (<a href="#figure1">Figure 1</a>B). Ideally, a translational reporter includes the entire genomic locus of a gene (5<span class="symbol">&#8242;</span> upstream region, exons, introns, 3<span class="symbol">&#8242;</span> UTR). GFP can be inserted at any point in the open reading frame, preferably at a site that does not disrupt protein function
                  or topology.
               </p>
               <p><span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></strong></span>-</em></span><span class="bold"><strong>based reporters</strong></span> are transcriptional reporters that aim to include all <span class="emphasis"><em>cis</em></span>-regulatory information of a gene. GFP, including its stop codon, is inserted between the promoter and the first exon (<a href="#figure1">Figure 1</a>C). Hence, the transcription of the transgene is controlled by the <span class="emphasis"><em>cis</em></span>-regulatory sequences found upstream, intronically or downstream of the gene of interest, yet only GFP is translated. However,
                  inclusion of the GFP stop codon targets the produced mRNA for nonsense-mediated mRNA decay (<a href="#bib21">Pulak and Anderson, 1993</a>; <a href="#bib25">Wilkinson et al., 1994</a>; <a href="#bib16">Mango, 2001</a>). Therefore, these reporters need to be injected into a genetic background deficient for nonsense-mediated mRNA decay, such
                  as <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span> (<a href="#bib16">Mango, 2001</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e199"></a>1.3.&nbsp;Sequences to be included in reporter constructs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The <span class="emphasis"><em>C. elegans</em></span> genome is very compact. Most <span class="emphasis"><em>cis</em></span>-regulatory information lies within several kilobases immediately upstream of a gene. However, there is a growing number of
                  examples for which crucial  <span class="emphasis"><em>cis</em></span>-regulatory sequences have been found within introns (especially genes containing large introns; <a href="#bib23">Wenick and Hobert, 2004</a>) or the 3<span class="symbol">&#8242;</span> UTR (<a href="#bib24">Wightman et al., 1993</a>). Rarer yet, <span class="emphasis"><em>cis</em></span>-regulatory sequences have been found at unusually long distances (beyond upstream or downstream flanking genes) from the
                  gene of interest (<a href="#bib3">Conradt and Horvitz, 1999</a>; <a href="#bib23">Wenick and Hobert, 2004</a>).
               </p>
               <p>Setting aside extreme examples, the most common reporter gene constructs simply include a large portion of the 5<span class="symbol">&#8242;</span> intergenic sequence (transcriptional reporter). Ideally,  reporters should contain as much <span class="emphasis"><em>cis</em></span>-regulatory sequence, from upstream to downstream gene, as technically possible (translational and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span>-based reporters).
               </p>
               <div class="sidebar">
                  <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                     <h4 class="title">Note</h4>
                     <p>Over the past decade, it has become clear that microRNAs play an important role in post-transcriptional gene regulation (via
                        the 3<span class="symbol">&#8242;</span> UTR; <a href="#bib1">Ambros, 2004</a>). Therefore, transcriptional reporters, which do not include the endogenous 3<span class="symbol">&#8242;</span> UTR, may not accurately represent the expression patterns of genes regulated by microRNAs.
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e256"></a>1.4.&nbsp;Genome-wide gene expression projects
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Given the availability of the full <span class="emphasis"><em>C. elegans</em></span> genome sequence, several groups have undertaken genome-wide gene expression projects. Three groups are systematically generating
                  promoter::GFP fusions and documenting the resulting expression patterns. Results and additional information are available
                  through their respective websites:
               </p>
               <p>Promoterome project: <a href="http://vidal.dfci.harvard.edu/promoteromedb/" target="_top">http://vidal.dfci.harvard.edu/promoteromedb/</a></p>
               <p>BC <span class="emphasis"><em>C. elegans</em></span> Gene Expression Consortium: <a href="http://elegans.bcgsc.ca/home/ge_consortium.html" target="_top">http://elegans.bcgsc.ca/home/ge_consortium.html</a></p>
               <p>Hope Lab Expression Patten Database: <a href="http://bgypc059.leeds.ac.uk/~web/" target="_top">http://bgypc059.leeds.ac.uk/~web/</a></p>
               <p><span class="emphasis"><em>C. elegans</em></span> Promoter/Marker Database: <a href="http://www.grs.nig.ac.jp/c.elegans/promoter/index.jsp?lang=english" target="_top">http://www.grs.nig.ac.jp/c.elegans/promoter/index.jsp?lang=english</a></p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e285"></a>1.5.&nbsp;Technical considerations
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e288"></a>1.5.1.&nbsp;Comments on different reporter types
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em><span class="bold"><strong>Transcriptional reporters</strong></span></em></span> can be used to rapidly establish a tentative expression pattern for a gene of interest. Fusing 5<span class="symbol">&#8242;</span> upstream sequences to GFP can be done in a number of ways and usually presents no technical challenge. Compared to translational
                     and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span>-based reporters, however, promoter fusions may not give a complete representation of the real expression pattern of a gene,
                     both spatially and temporally.
                  </p>
                  <p><span class="emphasis"><em><span class="bold"><strong>Translational reporters</strong></span></em></span> can provide a faithful representation of a gene's expression pattern because additional regulatory information that may be
                     present in introns or 3<span class="symbol">&#8242;</span>UTRs is included in such reporter constructs. When mutants are available, translational reporters can be used in rescue experiments.
                     Successfull rescue of the mutant phenotype lends support for the functional relevance of the resulting expression patterns.
                     In addition, translational gene fusions can also provide information about subcellular localization and the temporal aspects
                     of gene regulation. Translational fusions may appear less bright than transcriptional fusions due to the intrinsic instability
                     of the protein fused to GFP. However, insertion of GFP intragenically can sometimes disrupt protein function or even lead
                     to toxicity of the chimeric product. Finally, translational reporters that exhibit subcellular localization can make cell
                     type identification more difficult because the shape of the cell may not be visible (especially for neurons).
                  </p>
                  <p><span class="emphasis"><em><span class="bold"><strong><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a>-based reporters</strong></span></em></span> represent expression patterns as accurately as translational reporters. GFP produced by these reporters is not localized
                     which can facilitate cell type identification. The major disadvantage of  <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span>-based reporters is that they require a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span> mutation in the genetic background.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e326"></a>1.5.2.&nbsp;Vector backbones
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A large number of vector backbones and reporter gene derivatives are available from the Fire Vector Kit constructed by <a href="http://genome-www.stanford.edu/group/fire/" target="_top">Andrew Fire's lab</a>. In addition to the basic structural elements of the expression vectors (Multiple Cloning Site, artificial intron, reporter
                     gene, 3<span class="symbol">&#8242;</span> UTR; see Cloning) some vectors have been engineered to contain nuclear localization, membrane targeting or mitochondrial
                     targeting signals. The vectors, their sequence files, detailed descriptions and instructions for using these vectors are available
                     at: <a href="http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1" target="_top">http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1</a> or <a href="ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors" target="_top">ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors</a>. Some of the most useful vectors from the Fire Vector Kit are listed in <a href="#table1" title="Table 1. Useful vector backbones: GFP/LacZ/DsRed reporter variants">Table 1</a>.
                  </p>
                  <p>
                                             
                  </p>
                  <div class="table"><a name="table1"></a><p class="title"><span class="bold"><strong>Table 1. Useful vector backbones: GFP/LacZ/DsRed reporter variants</strong></span></p>
                     <table summary="Table 1. Useful vector backbones: GFP/LacZ/DsRed reporter variants" border="1">
                        <colgroup>
                           <col align="center">
                           <col align="center">
                           <col align="center">
                        </colgroup>
                        <thead>
                           <tr>
                              <th align="center">Vector</th>
                              <th align="center">Description</th>
                              <th align="center">Localization</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td align="center">pPD95.75 (L2463)</td>
                              <td align="center">GFP [S65C]</td>
                              <td align="center">None</td>
                           </tr>
                           <tr>
                              <td align="center">pPD16.43</td>
                              <td align="center">LacZ; 1xNLS</td>
                              <td align="center">Nucleus</td>
                           </tr>
                           <tr>
                              <td align="center">pDsRed</td>
                              <td align="center">DsRed; Clontech</td>
                              <td align="center">-</td>
                           </tr>
                           <tr>
                              <td align="center">pDsRed2</td>
                              <td align="center">DsRed2; Clontech</td>
                              <td align="center">-</td>
                           </tr>
                           <tr>
                              <td align="center">pPD135.83 (L4796)</td>
                              <td align="center">GFP [S65C]; 2xSV40-NLS</td>
                              <td align="center">Nucleolus and Nucleus</td>
                           </tr>
                           <tr>
                              <td align="center">pPD136.15 (L4809)</td>
                              <td align="center">GFP [S65C]; 3xSV40-NLS</td>
                              <td align="center">Nucleolus &gt; Nucleus</td>
                           </tr>
                           <tr>
                              <td align="center">pPD133.48 (L4663)</td>
                              <td align="center">CFP[Y66W, N146I, M153T, V163A]; 1xSV40-NLS</td>
                              <td align="center">Nucleus &gt; Cytoplasm</td>
                           </tr>
                           <tr>
                              <td align="center">pPD133.45 (L4660)</td>
                              <td align="center">CFP [Y66W, N146I, M153T, V163A]::lacZ; 1xSV40-NLS</td>
                              <td align="center">Nucleus only</td>
                           </tr>
                           <tr>
                              <td align="center">pPD132.112 (L4643)</td>
                              <td align="center">YFP [S65G V68A S72A T203Y]; 1xSV40-NLS</td>
                              <td align="center">Nucleus &gt; Cytoplasm</td>
                           </tr>
                           <tr>
                              <td align="center">pPD133.63 (L4671)</td>
                              <td align="center">YFP [S65G V68A S72A T203Y]::lacZ; 1xSV40-NLS</td>
                              <td align="center">Nucleus only</td>
                           </tr>
                           <tr>
                              <td align="center">pPD122.39 (L4058)</td>
                              <td align="center">PAT-3-GFP[S65C]; <a href="http://www.wormbase.org/db/seq/protein?name=PAT-3;class=protein" target="_blank">PAT-3</a> transmembrane domain
                              </td>
                              <td align="center">Plasma membrane</td>
                           </tr>
                           <tr>
                              <td align="center">pPD133.54 (L4665)</td>
                              <td align="center">Mt::CFP[Y66W, N146I, M153T, V163A]</td>
                              <td align="center">Mitochondria</td>
                           </tr>
                           <tr>
                              <td align="center">pPD133.60 (L4667)</td>
                              <td align="center">Mt::YFP [S65G V68A S72A T203Y]</td>
                              <td align="center">Mitochondria</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <div class="sidebar">
                     <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                        <h4 class="title">Note</h4>
                        <p>The Fire Vector Kit does not include red fluorescent protein variants, but DsRed and DsRed2 from Clontech have been successfully
                           substituted for GFP in reporter transgenes in <span class="emphasis"><em>C. elegans</em></span> (e.g., <a href="#bib23">Wenick and Hobert, 2004</a>).
                        </p>
                     </div>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e470"></a>1.6.&nbsp;Multicolor GFP expression systems
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The availability of GFP variants with non-overlapping emission spectra (Cyan-FP, Yellow-FP) has opened the possibility for
                  multi-color labeling of <span class="emphasis"><em>C. elegans</em></span> cells <span class="emphasis"><em>in vivo</em></span>. CFP and YFP variants can be separated using appropriate filter sets (<a href="#bib20">Miller et al., 1999</a>). Recently, triple color combinations (CFP, YFP, DsRed) have been used successfully to label separate classes of neurons
                  (<a href="#bib13">Hutter, 2003</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e487"></a>1.7.&nbsp;Common artifacts
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Of little consequence to the experimentalist, yet fascinating in its own right, is the autofluorescence exhibited by the gut
                  granules and nucleoli of hypodermal cells in <span class="emphasis"><em>C. elegans</em></span>. These phenomena are not a result of aberrant reporter transgene expression, but occur naturally in <span class="emphasis"><em>C. elegans</em></span>.
               </p>
               <p>In contrast to non-GFP-related autofluorescence, GFP reporters can sometimes induce non-specific fluorescence in posterior
                  gut cells. This observed effect is thought to be an indirect consequence of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> 3<span class="symbol">&#8242;</span> UTR, which is used in most vectors from the Fire Kit (see <a href="http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1" target="_top">http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1</a> or <a href="ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors" target="_top">ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors</a>).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e513"></a>1.8.&nbsp;Fluorophore maturation of fluorescent proteins
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Temporal gene regulation studies rely on the ability to detect gene expression fluctuations within a narrow time window. A
                  drawback to using fluorescent proteins for such experiments is the time required for the maturation of the fluorophore.
               </p>
               <p>Studies performed in <span class="emphasis"><em>E.coli</em></span> have found the  t<sub>0.5</sub> for maturation to be 27 minutes for the S65T variant of GFP (used in pPD95.75 and its derivatives; <a href="#bib10">Heim et al., 1995</a>). Evidence from <span class="emphasis"><em>C. elegans</em></span> experiments suggests that GFP S65T matures at a similar rate in the worm, as do CFP and YFP variants. The red fluorescent
                  proteins, DsRed and DsRed2, mature significantly slower. However, new variants of DsRed have been generated that are brigther,
                  mature faster (<a href="#bib22">Shaner et al., 2004</a>) and fluoresce in <span class="emphasis"><em>C. elegans</em></span> (Etchberger, unpublished).
               </p>
               <p>Sometimes expression from weak promoters may require a prolonged accumulation of protein before a fluorescent signal is detectable
                  <span class="emphasis"><em>in vivo</em></span>. In these cases, antibody staining can reveal expression of the fluorescent protein before a detectable fluorescent signal
                  is achieved.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e543"></a>1.9.&nbsp;Coinjection markers
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A number of standard coinjection markers are available to label reporter gene-containing extrachromosomal arrays in <span class="emphasis"><em>C. elegans</em></span>. The most commonly used marker is <span class="emphasis"><em>rol-6(<a href="http://www.wormbase.org/db/get?name=su1006;class=Variation" target="_blank">su1006</a>)</em></span> (<a href="#bib14">Kramer et al., 1990</a>). <span class="emphasis"><em>rol-6(<a href="http://www.wormbase.org/db/get?name=su1006;class=Variation" target="_blank">su1006</a>)</em></span> is a dominant allele that can be used to label arrays in most genetic backgrounds, allowing for easy selection and maintenance
                  of transgenic lines. Alternative injection markers that do require specific mutants background include: <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pha-1" target="_blank">pha-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-20" target="_blank">dpy-20</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-4" target="_blank">unc-4</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span> (see WormMethods section: <a href="http://www.wormbook.org/chapters/www_transformationmicroinjection/transformationmicroinjection.html" target="_top">Transformation and microinjection</a>). Notably, GFP reporters, such as <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ceh-22" target="_blank">ceh-22</a></em></span>::<span class="emphasis"><em>gfp</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-122" target="_blank">unc-122</a></em></span>::<span class="emphasis"><em>gfp</em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=elt-2" target="_blank">elt-2</a></em></span>::<span class="emphasis"><em>gfp</em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ttx-3" target="_blank">ttx-3</a></em></span>::<span class="emphasis"><em>gfp</em></span> have been used as injection markers in their own right.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e613"></a>1.10.&nbsp;Concluding remark
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Reporter transgenes may not represent the complete expression pattern for a gene, but still provide testable hypotheses about
                  the site of gene function. It has become standard to supplement these hypotheses with cell-type specific rescue experiments,
                  mosaic analysis and antibody staining. For early embryo studies, <span class="emphasis"><em>in situ</em></span> hybridization can also be informative.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e621"></a>1.10.1.&nbsp;Links
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Fire Lab Vector Kit: <a href="http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1" target="_top">http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1</a> or <a href="ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors" target="_top">ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors</a></p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e632"></a>2.&nbsp;Protocols
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e635"></a>2.1.&nbsp;Protocol 1: Generating GFP reporter constructs using standard cloning techniques
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e638"></a>2.1.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The most common approach for generating reporter genes in <span class="emphasis"><em>C. elegans</em></span> is to clone/subclone DNA fragments into reporter gene vectors. A large array of vectors have been made by <a href="http://genome-www.stanford.edu/group/fire/" target="_top">Andrew Fire's lab</a>. The &#8216;Fire Vector Kit&#8217; contains many vector variants used to study <span class="emphasis"><em>C. elegans</em></span> gene expression (see <a href="http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1" target="_top">http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1</a> or <a href="ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors" target="_top">ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors</a>). All vectors have a backbone based on the pUC19 plasmid and contain the <span class="emphasis"><em>E.coli</em></span> ampicilin resistance gene (AmpR) and origin of replication (<a href="#bib6">Fire et al., 1990</a>). They also have convenient multiple cloning sites (MCS) and provide a number of useful reporter gene variants. In addition
                     to the Fire Vector Kit, many expression vectors are available in the <span class="emphasis"><em>C. elegans</em></span> community.
                  </p>
                  <div class="itemizedlist">
                     <p class="title">Pros of using standard cloning techniques:</p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Creation of reusable reagents</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Availability of large sets of compatible vectors</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Verification of cloned DNA by sequencing</p>
                        </li>
                     </ul>
                  </div>
                  <div class="itemizedlist">
                     <p class="title">Cons of using standard cloning techniques:</p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Time intensive: minimum 2-3 days</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Dependence on restriction sites</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Size limits for DNA insert</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Toxicity to bacteria in rare cases</p>
                        </li>
                     </ul>
                  </div>
                  <div class="sidebar">
                     <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                        <h4 class="title">Note</h4>
                        <p>The reader is expected to be familiar with standard molecular cloning techniques. Protocols will be focusing on specific aspects
                           relating to <span class="emphasis"><em>C. elegans</em></span> reagents.
                        </p>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e701"></a>2.1.2.&nbsp;Choosing vector backbone
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The first step in designing GFP reporter constructs is to choose an adequate vector backbone. Many vectors are available in
                     the Fire Kit. Presented in <a href="#figure2">Figure 2</a> is the structure of pPD95.75, a vector that serves as an archetype for most Fire vectors. In addition, a variety of specialized
                     vectors are listed in <a href="#table1" title="Table 1. Useful vector backbones: GFP/LacZ/DsRed reporter variants">Table 1</a>.
                  </p>
                  <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="540">
<tr>
  <td align="center"><p><a href="GFPRfig2.jpg"><img src="GFPRfig2_s.jpg" border="2" align="middle" alt="Description of pPD95.75 figure 2"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 2. Description of pPD95.75.&nbsp;</b>The multiple cloning site (MCS) of pPD95.75 spans from HindIII to MlsI. The reading frame upstream of GFP is indicated by
                           alternating red and gray boxes. The artificial intron lies between the MCS and the GFP coding region (the ATG of GFP is indicated
                           in green).
                        </p>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e723"></a>2.1.3.&nbsp;Description of pPD95.75
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e726"></a>2.1.3.1.&nbsp;<span class="emphasis"><em>Multiple cloning site (MCS)</em></span></h5>
                           </div>
                        </div>
                     </div>
                     <div class="itemizedlist">
                        <ul type="bullet">
                           <li style="list-style-type: disc">
                              <p>The multiple cloning site of pPD95.75 stretches from HindIII to MlsI (<a href="#figure2">Figure 2</a>).
                              </p>
                           </li>
                           <li style="list-style-type: disc">
                              <p>Cloning a PCR amplified promoter: adapted restriction sites can be added to facilitate directional cloning.</p>
                           </li>
                           <li style="list-style-type: disc">
                              <p>Generating a C-terminal translational fusion: the sequence of interest has to be cloned into the proper reading frame. The
                                 reading frame of pPD95.75 is indicated by alternating gray and red boxes in <a href="#figure2">Figure 2</a>. Vectors with shifted reading frames are also available (pPD95.77, pPD95.79).
                              </p>
                           </li>
                        </ul>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e744"></a>2.1.3.2.&nbsp;gfp <span class="emphasis"><em>ORF</em></span></h5>
                           </div>
                        </div>
                     </div>
                     <p>The reporter gene in pPD95.75 is a S65C variant of GFP that includes artificial introns for enhanced expression (Fire Vector
                        Kit, 1995). Other reporter genes are listed in <a href="#table1" title="Table 1. Useful vector backbones: GFP/LacZ/DsRed reporter variants">Table 1</a>.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e754"></a>2.1.3.3.&nbsp;<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a> 3<span class="symbol">&#8242;</span> UTR</em></span></h5>
                           </div>
                        </div>
                     </div>
                     <p>The 3<span class="symbol">&#8242;</span> UTR in pPD95.75 is derived from the muscle myosin heavy chain gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> and ensures efficient processing of transcripts. Some recent Fire Vectors contain an alternative 3<span class="symbol">&#8242;</span> UTR from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-858" target="_blank">let-858</a></em></span>.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e779"></a>2.1.4.&nbsp;Discussion
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The primary advantage of standard cloning approaches is the generation of a reusable reporter gene construct. Plasmids can
                     be repeatedly amplified in bacteria and stored for long periods of time. In addition, constructs can be subjected to further
                     sub-cloning steps for <span class="emphasis"><em>cis</em></span>-regulatory analysis (promoter bashing) and/or functional analysis. Finally, cloned reagents can be used as a DNA source in
                     other GFPs fusion methods.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e788"></a>2.1.5.&nbsp;Links
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Vector Kit: <a href="http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1" target="_top">http://www.addgene.org/pgvec1?f=c&amp;cmd=showcol&amp;colid=1</a> or <a href="ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors" target="_top">ftp://ftp.wormbase.org/pub/wormbase/datasets/fire_vectors</a></p>
                  <p>Fire Lab: <a href="http://genome-www.stanford.edu/group/fire/" target="_top">http://genome-www.stanford.edu/group/fire/</a></p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e802"></a>2.2.&nbsp;Protocol 2: Generating GFP reporters using a PCR fusion approach
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e805"></a>2.2.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Traditional cloning strategies involving PCR-amplified DNA fragments and GFP expression vectors are time consuming and limited
                     by the size of DNA inserts. Current cloning/subcloning protocols require at least 2-3 days to obtain a final construct ready
                     for use. To obtain reporter constructs more quickly and circumvent the challenge of cloning large DNA fragments, DNA sequences
                     can be fused to GFP via the polymerase chain reaction (PCR) and injected into the <span class="emphasis"><em>C. elegans</em></span> gonad in a linear form (<a href="#bib11">Hobert, 2002</a>). This approach is referred to as PCR fusion. For more details, see <a href="#figure3">Figure 3</a> and <a href="#bib11">Hobert (2002)</a>.
                                                 
                  </p>
                  <div class="itemizedlist">
                     <p class="title">Pros of using the PCR fusion approach:</p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Rapid generation of reporter constructs</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Adaptable to any vector backbone</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Independent of presence/absence of restriction sites</p>
                        </li>
                     </ul>
                  </div>
                  <p>
                                                 
                  </p>
                  <div class="itemizedlist">
                     <p class="title">Cons of using the PCR fusion approach:</p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>PCR products are a depletable resource</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Sequencing validation of reporter construct impractical</p>
                        </li>
                     </ul>
                  </div>
                  <p>
                                                 
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e845"></a>2.2.2.&nbsp;Protocol
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>PCR fusions generate a single amplicon from two fragments that contain a small overlapping region of sequence homology (~24
                     bp; <a href="#figure3">Figure 3</a>). The overlap is engineered into the fragments by the primers. By this method, a sequence of interest can be fused to any
                     reporter gene.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e852"></a>2.2.2.1.&nbsp;Choosing an appropriate reporter gene
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Many derivatives of GFP expression vectors currently exist (Fire Vector Kit). Choose a reporter gene vector that suits the
                        needs of the experiment. pPD95.75 is a commonly used vector for PCR fusion. However, in some cases, a vector containing a
                        localized reporter gene or 3<span class="symbol">&#8242;</span>UTR variant may be needed (<a href="#table1" title="Table 1. Useful vector backbones: GFP/LacZ/DsRed reporter variants">Table 1</a>). Remember that primer design depends upon the sequence of the reporter gene vector.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="d0e863"></a>2.2.2.2.&nbsp;Designing the primers
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>The following guidelines for primer design correspond to situtations where GFP is fused to the 3<span class="symbol">&#8242;</span> end of a DNA fragment (e.g., promoter or genomic locus). Gene-specific primers are used to amplify the promoter and/or gene
                        of interest (<a href="#figure3">Figure 3</a>: primers A, A*, B; A* is nested relative to A; <a href="#table2" title="Table 2. Primer sequences used with pPD95.75">Table 2</a>). Likewise, vector-specific primers are used to amplify the expression vector of choice, encompassing the reporter gene and
                        3<span class="symbol">&#8242;</span> UTR (<a href="#figure3">Figure 3</a>: primers C, D, D*; D* is nested relative to D; <a href="#table2" title="Table 2. Primer sequences used with pPD95.75">Table 2</a>). The primer requiring the most careful design is the 3<span class="symbol">&#8242;</span> gene-specific primer (<a href="#figure3">Figure 3</a>: primer B). This primer contains a linking sequence that overlaps with the reporter gene amplicon. The gene-specific primer
                        linker should be at least 24 bp long and complementary to the MCS of the expression vector or to the coding region of the
                        reporter gene.
                     </p>
                     <div class="table"><a name="table2"></a><p class="title"><span class="bold">Table 2. Primer sequences used with pPD95.75</span></p>
                        <table summary="Table 2. Primer sequences used with pPD95.75" border="1">
                           <colgroup>
                              <col align="center">
                              <col align="center">
                           </colgroup>
                           <tbody>
                              <tr>
                                 <td align="center">Primer A</td>
                                 <td align="center">Gene-specific 5' 'outside' primer</td>
                              </tr>
                              <tr>
                                 <td align="center">Primer A*</td>
                                 <td align="center">Gene-specific 5' 'nested' primer</td>
                              </tr>
                              <tr>
                                 <td align="center">Primer B</td>
                                 <td align="center">5<span class="symbol">&#8242;</span>-AGTCGACCTGCAGGCATGCAAGCT-Gene-specific sequence-3<span class="symbol">&#8242;</span></td>
                              </tr>
                              <tr>
                                 <td align="center">Primer C</td>
                                 <td align="center">5<span class="symbol">&#8242;</span>-AGCTTGCATGCCTGCAGGTCG-3<span class="symbol">&#8242;</span></td>
                              </tr>
                              <tr>
                                 <td align="center">Primer D</td>
                                 <td align="center">5<span class="symbol">&#8242;</span>-AAGGGCCCGTACGGCCGACTA-3<span class="symbol">&#8242;</span></td>
                              </tr>
                              <tr>
                                 <td align="center">Primer D*</td>
                                 <td align="center">5<span class="symbol">&#8242;</span>-GGAAACAGTTATGTTTGGTATA-3<span class="symbol">&#8242;</span></td>
                              </tr>
                           </tbody>
                        </table>
                     </div>
                     <div class="sidebar">
                        <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                           <h4 class="title">Note</h4>
                           <p>Transcriptional reporters: Fusing a promoter to the MCS of a pPD95.75-type expression vector ensures that the intron between
                              the MCS and reporter gene is included. Inclusion of this intron greatly enhances reporter gene expression.
                           </p>
                           <p>Translational reporters: For translational reporters, the 3<span class="symbol">&#8242;</span> gene-specific primer (<a href="#figure3">Figure 3</a>: Primer B) must be in frame with the reporter gene.
                           </p>
                        </div>
                     </div>
                     <div class="literallayout">
                        <p></p>
                     </div>
                     <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="GFPRfig3.jpg"><img src="GFPRfig3_s.jpg" border="2" align="middle" alt="Generating a C-terminal translational PCR fusion figure 3"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 3. Generating a C-terminal translational PCR fusion.&nbsp;</b>Primers A and B amplify the genomic region (amplicon #1). Primer B adds a 24 bp overlap in frame to the GFP coding region.
                              Primers C and D amplify the reporter gene (e.g., GFP) and 3' UTR (amplicon #2). Primers A* and D* are used to fuse amplicon
                              #1 and amplicon #2 (gray box indicates 24 bp sequence overlap). The resulting fusion product (amplicon #3) can be directly
                              injected into <span class="emphasis"><em>C. elegans</em></span> without purification.
                           </p>
                        </div>
                     </div>
                     <p><span class="bold"><strong>Steps:</strong></span></p>
                     <div class="procedure">
                        <ol type="1">
                           <li>
                              <p class="title"><b>PCR amplification of genomic DNA and reporter gene</b></p>
                              <ol type="a">
                                 <li>
                                    <p>Amplify DNA fragment with gene-specific primers (Primers A and B) from a genomic DNA preparation, worm lysate, cosmid or other
                                       source.
                                    </p>
                                 </li>
                                 <li>
                                    <p>Amplify reporter gene from expression vector with vector-specific primers (for pPD95.75 Primers C and D).</p>
                                 </li>
                                 <li>
                                    <p>Estimate DNA concentration of PCR products by agarose gel electrophoresis.</p>
                                 </li>
                              </ol>
                           </li>
                           <li>
                              <p class="title"><b>Fusion PCR</b></p>
                              <ol type="a">
                                 <li>
                                    <p>Fuse products from Step 1 (1-50 ng each per reaction) using nested primers (Primer A*; for pPD95.75 Primer D*).</p>
                                 </li>
                                 <li>
                                    <p>Estimate DNA concentration of PCR product by agarose gel electrophoresis.</p>
                                 </li>
                              </ol>
                           </li>
                           <li>
                              <p class="title"><b>Sample preparation and injection</b></p>
                              <ol type="a">
                                 <li>
                                    <p>The PCR reaction from Step 2 can be used directly for injection if the expected size fusion product is present in sufficient
                                       quantity. If yield is low, the DNA sample can be concentrated by standard methods. Usually, reporters are injected at concentrations
                                       ranging from 10 to 50 ng/<span class="symbol">&#956;</span>L.
                                    </p>
                                 </li>
                              </ol>
                           </li>
                        </ol>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1013"></a>2.2.3.&nbsp;Discussion
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>PCR fusion technology is a good supplement to other strategies for the generation of reporter gene constructs. The primary
                     advantage to the PCR fusion approach is speed. The time required to make a PCR fusion product from start to finish is shorter
                     than traditional cloning approaches. In addition, many reactions can be carried out in parallel. However, PCR fusion technology
                     is not to replace all traditional cloning strategies. Most significantly, PCR fusion products are not amplifiable in bacteria.
                     Furthermore, additive rounds of PCR reactions increase the chances of acquiring point mutations in the amplicon. This problem
                     can be reduced by pooling independent fusion reactions.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1018"></a>2.3.&nbsp;Protocol 3: Generating GFP reporters using <span class="emphasis"><em>in vivo</em></span> recombination
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1024"></a>2.3.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Reporter gene constructs for specific genomic loci can be extremely long (&gt;20kb). The manipulation of large genomic fragments
                     containing complete or partial open reading frames as well as 5<span class="symbol">&#8242;</span> and 3<span class="symbol">&#8242;</span> intergenic regions can be difficult with current cloning or PCR approaches. To circumvent these limitations, DNA fragments
                     with overlapping sequence homology can be coinjected and will undergo homologous recombination <span class="emphasis"><em>in vivo</em></span> to produce a long linear fragment (<a href="#bib19">Mello et al., 1991</a>; <a href="#bib18">Maryon et al., 1996</a>; <a href="#bib28">Tsalik et al., 2003</a>). This approach is termed <span class="emphasis"><em>in vivo</em></span> recombination. Reporter transgenes made in this manner eliminate the time required for conventional cloning techniques and
                     bypass limitations associated with the PCR-mediated amplification of long DNA fragments.
                                         
                  </p>
                  <div class="itemizedlist">
                     <p class="title">Pros of using <span class="emphasis"><em>in vivo</em></span> recombination:
                     </p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Generation of large reporter transgenes</p>
                        </li>
                     </ul>
                  </div>
                  <p>
                                         
                  </p>
                  <div class="itemizedlist">
                     <p class="title">Cons of using <span class="emphasis"><em>in vivo</em></span> recombination:
                     </p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Validation of reporter transgene sequence difficult</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Multiple control injections required</p>
                        </li>
                     </ul>
                  </div>
                  <p>
                                     
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1074"></a>2.3.2.&nbsp;Protocol: generation of large GFP reporter transgenes
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>In vivo</em></span> recombination circumvents size limitations associated with cloning and PCR fusion approaches. Smaller fragments with a minimum
                     overlap of 350bp (<a href="#bib18">Maryon et al., 1996</a>), can be generated by various methods and coinjected to recombine into the full reporter transgene. All three types of reporter
                     trangenes (transcriptional, translational and "<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smg-1" target="_blank">smg-1</a></em></span>-based") can be generated by this method. As an example of this approach, the following protocol describes the generation
                     an N-terminally fused GFP translational reporter.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1088"></a>2.3.3.&nbsp;Generation of an N-terminal GFP translational reporter
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>To generate a translational GFP reporter for a large gene (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-41" target="_blank">lin-41</a></em></span>; pers. com. Frank Slack), the simplest approach is to generate <span class="emphasis"><em>5<span class="symbol">&#8242;</span>-upstream-GFP</em></span> and <span class="emphasis"><em>GFP-ORF-3<span class="symbol">&#8242;</span>-downstream</em></span> fragments (by PCR fusion in this case) which will recombine to form <span class="emphasis"><em>5<span class="symbol">&#8242;</span>-upstream-GFP-ORF-3<span class="symbol">&#8242;</span>-downstream</em></span> (see <a href="#figure4">Figure 4</a>). <span class="emphasis"><em>5<span class="symbol">&#8242;</span>-upstream-GFP</em></span> contains the promoter fused to the coding region of GFP. <span class="emphasis"><em>GFP-ORF-3<span class="symbol">&#8242;</span>-downstream</em></span> is a fusion of GFP to the genomic coding region and the complete 3<span class="symbol">&#8242;</span> UTR.
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="GFPRfig4.jpg"><img src="GFPRfig4_s.jpg" border="2" align="middle" alt="Generating a N-terminal translational reporter figure 4"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 4. Generating a N-terminal translational reporter.&nbsp;</b>The <span class="emphasis"><em>5'-upstream-GFP </em></span>and <span class="emphasis"><em>GFP-ORF-3'-downstream</em></span> fragments are generated in two independent PCR fusion steps (see protocol #2). The products are injected at equal molar ratios.
                           The sequence overlap between the two fragments (GFP coding region) allows for homologous recombination<span class="emphasis"><em> in vivo</em></span> to generate <span class="emphasis"><em>5'-upstream-GFP-ORF-3'-downstream</em></span>.
                        </p>
                     </div>
                  </div>
                  <p><span class="bold"><strong>Steps:</strong></span></p>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <ol type="a">
                              <li>
                                 <p class="title"><b><span class="emphasis"><em>5<span class="symbol">&#8242;</span>-upstream-GFP</em></span> fragment</b></p>
                                 <ol type="i">
                                    <li>
                                       <p>PCR amplify <span class="emphasis"><em>5<span class="symbol">&#8242;</span>-upstream</em></span> to include overlapping 3<span class="symbol">&#8242;</span> linker to the first 24nt of the GFP coding region (GFP sequence: ATGAGTAAAGGAGAAGAACTTTTC).
                                       </p>
                                    </li>
                                    <li>
                                       <p>PCR amplify GFP coding region (5<span class="symbol">&#8242;</span> primer: ATGAGTAAAGGAGAAGAACTTTTC; 3<span class="symbol">&#8242;</span> primer: GTCAGAGGCACGGGCGCGAGATG) from reporter gene vector (e.g., pPD95.75).
                                       </p>
                                    </li>
                                    <li>
                                       <p>PCR fuse <span class="emphasis"><em>5<span class="symbol">&#8242;</span>-upstream</em></span> and GFP coding region <span class="emphasis"><em>not including the STOP codon</em></span> (5<span class="symbol">&#8242;</span> nested primer; 3<span class="symbol">&#8242;</span> nested GFP primer: TTTGTATAGTTCATCCATGCC).
                                       </p>
                                    </li>
                                 </ol>
                              </li>
                              <li>
                                 <p class="title"><b><span class="emphasis"><em>GFP-ORF-3<span class="symbol">&#8242;</span>-downstream</em></span> fragment</b></p>
                                 <ol type="i">
                                    <li>
                                       <p>PCR amplify <span class="emphasis"><em>ORF-3<span class="symbol">&#8242;</span>-downstream</em></span> to include overlapping 5<span class="symbol">&#8242;</span> linker to the last 24nt of the GFP coding region <span class="emphasis"><em>minus the STOP codon</em></span> (GFP sequence: CATGGCATGGATGAACTATACAAA).
                                       </p>
                                    </li>
                                    <li>
                                       <p>PCR amplify GFP coding region (5<span class="symbol">&#8242;</span> primer: GTTTCGAATGATACTAACATAAC; 3<span class="symbol">&#8242;</span> primer: TTTGTATAGTTCATCCATGCC) from reporter gene vector (e.g., pPD95.75).
                                       </p>
                                    </li>
                                    <li>
                                       <p>PCR fuse GFP coding region and <span class="emphasis"><em>ORF-3'-downstream</em></span> region (5<span class="symbol">&#8242;</span> nested GFP primer: ATGAGTAAAGGAGAAGAACTTTTC; 3<span class="symbol">&#8242;</span> nested <span class="emphasis"><em>ORF-3<span class="symbol">&#8242;</span>-downstream</em></span> primer).
                                       </p>
                                    </li>
                                 </ol>
                              </li>
                           </ol>
                        </li>
                        <li>
                           <p class="title"><b>Co-injection of PCR fusion products</b></p>
                           <ol type="a">
                              <li>
                                 <p>Inject PCR fusion products (<span class="emphasis"><em>5<span class="symbol">&#8242;</span>-upstream-GFP</em></span> and <span class="emphasis"><em>GFP-ORF-3<span class="symbol">&#8242;</span>-downstream</em></span>) simultaneously at equal molar ratios.
                                 </p>
                              </li>
                           </ol>
                           <div class="sidebar">
                              <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                                 <h4 class="title">Note</h4>
                                 <p>Inject <span class="emphasis"><em>5<span class="symbol">&#8242;</span>-upstream-GFP</em></span> and <span class="emphasis"><em>GFP-ORF-3<span class="symbol">&#8242;</span>-downstream</em></span> individually for comparison to expression pattern from coinjected products.
                                 </p>
                              </div>
                           </div>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1301"></a>2.3.4.&nbsp;Discussion
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>In vivo</em></span> recombination technology provides a way to bypass the difficulties involved in the amplification and cloning of large DNA
                     fragments. However, the efficiency of homologous recombination in <span class="emphasis"><em>C. elegans</em></span> is largely uncharacterized. In addition, the <span class="emphasis"><em>5<span class="symbol">&#8242;</span>-upstream-GFP</em></span> or <span class="emphasis"><em>GFP-ORF-3<span class="symbol">&#8242;</span>-downstream</em></span> fragment may lead to erroneous expression on its own. To minimize undesirable background expression, truncated GFP sequences
                     that maintain sufficient overlap to trigger homologous recombination could be used.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1323"></a>2.4.&nbsp;Protocol 4: Generating GFP reporter constructs using Gateway<span class="symbol">&#63194;</span> cloning technology
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1329"></a>2.4.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In the wake of large scale genome sequencing projects, there has been an effort to make available the large number of functionally
                     relevant DNA sequences (e.g., ORFs, cDNAs, promoters, 3<span class="symbol">&#8242;</span> UTRs) in a versatile format. Gateway<span class="symbol">&#63194;</span> cloning technology <a href="http://www.invitrogen.com/content.cfm?pageid=4072" target="_top">http://www.invitrogen.com/content.cfm?pageid=4072</a> is a high-throughput method available for <span class="emphasis"><em>in vitro</em></span> cloning of DNA fragments into a variety of vector backbones.
                  </p>
                  <p>Presented here is a Gateway<span class="symbol">&#63194;</span> cloning strategy suitable for generating transcriptional  <span class="emphasis"><em>promoter::gfp</em></span> fusions and translational <span class="emphasis"><em>promoter::ORF::gfp</em></span> fusions (<a href="#bib4">Dupuy et al., 2004</a>; <a href="#bib12">Hope et al., 2004</a>). In addition, the promoter clones generated by this method can be used within the Gateway<span class="symbol">&#63194;</span> system for a variety of applications (e.g., tissue-specific RNAi and yeast one hybrid screens).
                  </p>
                  <div class="itemizedlist">
                     <p class="title">Pros of using Gateway&reg; technology:</p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Fast, efficient <span class="emphasis"><em>in vitro</em></span> reaction
                           </p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Compatible with large collection of vectors for other studies (<a href="#bib4">Dupuy et al., 2004</a>)
                           </p>
                        </li>
                     </ul>
                  </div>
                  <div class="itemizedlist">
                     <p class="title">Cons of using Gateway&reg; technology:</p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Decreased cloning efficiency above 2-3 kb (<a href="#bib17">Marsischky and LaBaer, 2004</a>)
                           </p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Proprietary cloning system</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Multistep procedure</p>
                        </li>
                     </ul>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1397"></a>2.4.2.&nbsp;Protocol: generation of <span class="emphasis"><em>promoter::gfp</em></span> and <span class="emphasis"><em>promoter::ORF::gfp</em></span> reporters
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The Gateway<span class="symbol">&#63194;</span> system takes advantage of the site-specific recombination properties of bacteriophage lambda (<a href="#bib15">Landy, 1989</a>; <a href="#bib8">Hartley et al., 2000</a>). Bacteriophage lambda integrase proteins recognize compatible recombination sites (<span class="emphasis"><em>att</em></span> sites), found in the cloning vector and flanking the DNA sequence of interest, to carry out the recombination event.
                  </p>
                  <p>For generating <span class="emphasis"><em>promoter::gfp</em></span> and <span class="emphasis"><em>promoter::ORF::gfp</em></span> reporters, a promoter is initially amplified with primers containing flanking <span class="emphasis"><em>att</em></span> sites (<span class="emphasis"><em>attB4</em></span> and <span class="emphasis"><em>attB1R</em></span>). The PCR product is recombined with pDONR-P4-P1R to generate pENTRY-prom (<a href="#figure5">Figure 5</a>). Finally, pENTRY-prom is recombined with the appropriate vectors to generate the intended product:
                                 
                  </p>
                  <table class="simplelist" border="0" summary="Simple list">
                     <tr>
                        <td><span class="emphasis"><em>promoter::gfp</em></span> reporter: recombine pENTRY-prom with pENTRY-gfp and pDEST-DD03 (<a href="#figure6">Figure 6</a>).
                        </td>
                     </tr>
                     <tr>
                        <td><span class="emphasis"><em>promoter::ORF::gfp</em></span> reporters: recombine pENTRY-prom with pENTRY-ORF and pDEST-MB14 (<a href="#figure7">Figure 7</a>).
                        </td>
                     </tr>
                  </table>
               <div class="literallayout">
                  <p></p>
               </div>
                  <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="252">
                        <tr style="height: 765px">
  <td align="center"><p><a href="GFPRfig5.jpg"><img src="GFPRfig5_s.jpg" border="2" align="middle" alt="Generating pENTRY-prom figure 5"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 5. Generating pENTRY-prom.&nbsp;</b>The <span class="emphasis"><em>attB4-prom-attB1R</em></span> fragment is amplified with appropriate primers containing <span class="emphasis"><em>attB4</em></span> and <span class="emphasis"><em>attB1R</em></span> sequences. <span class="emphasis"><em>attB4</em></span> and <span class="emphasis"><em>attB1R</em></span> recombine with <span class="emphasis"><em>attP4</em></span> and <span class="emphasis"><em>atP1R</em></span> of pDONR-P4-P1R, respectively, to generate pENTRY-prom in which the promoter is flanked by <span class="emphasis"><em>attL4</em></span> and <span class="emphasis"><em>attR1</em></span> sites.
                        </p>
                     </div>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="GFPRfig6.jpg"><img src="GFPRfig6_s.jpg" border="2" align="middle" alt="Generating pENTRY-prom figure 5"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 6. Generating <span class="emphasis"><em>promoter::gfp</em></span>.&nbsp;</b>The <span class="emphasis"><em>promoter::gfp</em></span> vector is generated by a multi-site LR reaction between pENTRY-prom, pENTRY-gfp and pDEST-DD03. <span class="emphasis"><em>attL</em></span> sites recombine with corresponding <span class="emphasis"><em>attR</em></span> sites. The resulting plasmid, <span class="emphasis"><em>promoter::gfp</em></span>, contains the promoter and GFP flanked by <span class="emphasis"><em>attB</em></span> sites.
                        </p>
                     </div>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure7"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="GFPRfig7.jpg"><img src="GFPRfig7_s.jpg" border="2" align="middle" alt="Generating pENTRY-prom figure 5"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 7. Generating <span class="emphasis"><em>promoter::ORF::gfp</em></span>.&nbsp;</b>The <span class="emphasis"><em>promoter::ORF::gfp</em></span> vector is generated by a multi-site LR reaction between pENTRY-prom, pENTRY-ORF and pDEST-MB14. In this reaction, <span class="emphasis"><em>attL</em></span> sites recombine with corresponding <span class="emphasis"><em>attR </em></span>sites. The resulting plasmid, <span class="emphasis"><em>promoter::ORF::gfp</em></span>, contains the promoter, ORF and GFP in frame with each other. pENTRY-ORF vectors are specifically designed to recombine in
                           frame with GFP from pDEST-MB14.
                        </p>
                     </div>
                  </div>
                  <p><span class="bold"><strong>Steps:</strong></span></p>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p class="title"><b>Generation of <span class="emphasis"><em>attB4-prom-attB1R</em></span></b></p>
                           <p>Design primers to amplify a promoter of interest. The primers must contain <span class="emphasis"><em>attB4</em></span> (5<span class="symbol">&#8242;</span> primer) and <span class="emphasis"><em>attB1R</em></span> (3<span class="symbol">&#8242;</span> primer) sequences in addition to the promoter specific sequences.
                           </p>
                           <p><span class="bold"><strong>5<span class="symbol">&#8242;</span> primer:</strong></span> (<span class="emphasis"><em>attB4</em></span> site)
                           </p>
                           <p>5<span class="symbol">&#8242;</span>-GGGGCAACTTTGTATAGAAAAGTTG---promoter-specific sequence-3<span class="symbol">&#8242;</span></p>
                           <p><span class="bold"><strong>3<span class="symbol">&#8242;</span> primer:</strong></span> (<span class="emphasis"><em>attB1R</em></span> site)
                           </p>
                           <p>5<span class="symbol">&#8242;</span>-GGGGACTGCTTTTTTGTACAAACTTGTC---promoter-specific sequence-3<span class="symbol">&#8242;</span></p>
                           <p>Amplify <span class="emphasis"><em>attB4-prom-attB1R</em></span> from an appropriate source (genomic DNA preparation, worm lysate, plasmid, cosmid, etc.).
                           </p>
                        </li>
                        <li>
                           <p class="title"><b>Generation of pENTRY-prom</b></p>
                           <p>Recombine <span class="emphasis"><em>attB4-prom-attB1R</em></span> with pDONR-P4-P1R <a href="http://www.invitrogen.com/search.cfm?searchTerm=12537023 category=Vectors" target="_top">(Invitrogen, cat. num. 12537023</a>) to generate an Entry Vector (pENTRY-prom) in which the promoter is flanked by <span class="emphasis"><em>attL4</em></span> and <span class="emphasis"><em>attR1</em></span> sites.
                           </p>
                           <div class="sidebar">
                              <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                                 <h4 class="title">Note</h4>
                                 <p>Cryptic <span class="emphasis"><em>att</em></span> sites may be present in genomic DNA sequences that could interfere with targeted recombination. However, this is found to
                                    be fairly unlikely (<a href="#bib8">Hartley et al., 2000</a>).
                                 </p>
                              </div>
                           </div>
                        </li>
                        <li>
                           <p class="title"><b>Generation of <span class="emphasis"><em>promoter::gfp</em></span> and <span class="emphasis"><em>promoter::ORF::gfp</em></span> reporter</b></p>
                           <p>
                                                  
                           </p>
                           <div class="orderedlist">
                              <ol type="i">
                                 <li>
                                    <p><span class="emphasis"><em>promoter::gfp</em></span> reporter: recombine pENTRY-prom with pENTRY-gfp and pDEST-DD03 (<a href="#figure6">Figure 6</a>).
                                    </p>
                                 </li>
                                 <li>
                                    <p><span class="emphasis"><em>promoter::ORF::gfp</em></span> reporter: recombine pENTRY-prom with pENTRY-ORF and pDEST-MB14 (<a href="#figure7">Figure 7</a>). The vector pDEST-MB14 already contains the <span class="emphasis"><em>gfp</em></span> gene and is engineered for the recombination product to be in frame.
                                    </p>
                                 </li>
                              </ol>
                           </div>
                           <p>In the study by Dupuy et al., the destination vectors (pDEST-DD03 <span class="symbol">&amp;</span> pDEST-MB14) contain the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> gene which is used in ballistic transformation protocols (Praitis, 2001). pENTRY-GFP, pENTRY-ORF, pDEST-DD03 and pDEST-MB14
                              can be obtained from Marc Vidal (<a href="mailto:marc_vidal@dfci.harvard.edu" target="_top">marc_vidal@dfci.harvard.edu</a>).
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1676"></a>2.4.3.&nbsp;Links
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Promoterome database: <a href="http://vidal.dfci.harvard.edu/promoteromedb/" target="_top">http://vidal.dfci.harvard.edu/promoteromedb/</a></p>
                  <p>Invitrogen: <a href="http://www.invitrogen.com" target="_top">http://www.invitrogen.com</a></p>
                  <p>Gateway<span class="symbol">&#63194;</span> cloning technology: <a href="http://www.invitrogen.com/content.cfm?pageid=4072" target="_top">http://www.invitrogen.com/content.cfm?pageid=4072</a></p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1694"></a>2.5.&nbsp;Protocol 5: Generating GFP reporter constructs using yeast homologous recombination
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1697"></a>2.5.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The analysis of gene expression from large genomic loci requires the construction of large reporter transgenes. Traditional
                     cloning strategies rely on available restriction sites for the insertion of a reporter gene within a gene of interest. The
                     multitude of restriction sites within large DNA fragments makes this a difficult task. One method to bypass the need for restriction
                     sites is yeast homologous recombination (<a href="#bib9">Hawkins et al., 2003</a>). This approach exploits the naturally occuring homologous recombination in <span class="emphasis"><em>S. cerevisiae</em></span> and can be used to generate precise insertions of a reporter gene into virtually any target sequence.
                                         
                  </p>
                  <div class="itemizedlist">
                     <p class="title">Pros of using yeast homologous recombination:</p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Generation of large (&gt; 18kb) reporter gene fusions</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>No requirement for restriction sites</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Limited use of PCR amplification</p>
                        </li>
                     </ul>
                  </div>
                  <p>
                                         
                  </p>
                  <div class="itemizedlist">
                     <p class="title">Cons of using yeast homologous recombination:</p>
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Requirement for yeast culturing skills</p>
                        </li>
                     </ul>
                  </div>
                  <p>
                                     
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1728"></a>2.5.2.&nbsp;Protocol
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Yeast homologous recombination utilizes the site-specific recombination properties of <span class="emphasis"><em>S. cerevisiae</em></span>. An initial PCR reaction amplifies a modified GFP reporter cassette (GFP-<span class="emphasis"><em>URA3</em></span>-GFP) with primers containing at least 50 base pairs of sequence homologous to the insertion site within the gene of interest
                     ("target sequence") (see <a href="#figure8">Figure 8</a>). The cassette is recombined into the target sequence that is carried by a bacterial shuttle vector (pRS415). Subsequent
                     selection steps generate the final GFP fusion construct (see <a href="#figure9">Figure 9</a>).
                  </p>
                  <p><span class="bold"><strong>Reagents:</strong></span></p>
                  <div class="itemizedlist">
                     <ul type="bullet">
                        <li style="list-style-type: disc">
                           <p>Target sequence cloned into the yeast-bacterial shuttle vector (pRS415)</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Primers to amplify (n<sub>50</sub>)-GFP-<span class="emphasis"><em>URA3</em></span>-GFP-(n<sub>50</sub>) cassette (see Step 2)
                           </p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Vector pCB182: contains two tandem repeats of GFP flanking <span class="emphasis"><em>URA3</em></span></p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Vector pRS415: yeast-bacterial shuttle vector</p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Yeast strain CBS17: MAT<span class="symbol">&#945;</span> <span class="emphasis"><em>his3</em></span><span class="symbol">&#948;</span><span class="emphasis"><em>200 leu2</em></span><span class="symbol">&#948;</span><span class="emphasis"><em>1 ura3-52 mal hap2</em></span></p>
                        </li>
                        <li style="list-style-type: disc">
                           <p>Yeast strain CBY668: MAT<span class="symbol">&#945;</span> <span class="emphasis"><em>ade2-101 leu2-3,112 ura3-52 cyh2 kar1-1</em></span></p>
                        </li>
                     </ul>
                  </div>
                  <p><span class="bold"><strong>Steps:</strong></span></p>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p class="title"><b>Generation of (n<sub>50</sub>)-GFP-<span class="emphasis"><em>URA3</em></span>-GFP-(n<sub>50</sub>) cassette</b></p>
                           <p>Design primers to amplify GFP-<span class="emphasis"><em>URA3</em></span>-GFP. The primers must contain at least 50bp overhangs homologous to the target site and the core sequences (complementary
                              to GFP) as follows:
                           </p>
                           <p>5<span class="symbol">&#8242;</span> primer: 5<span class="symbol">&#8242;</span>-(n<sub>50</sub>)-ATGAGTAAAGGAGAAGAACTTTTCAC-3<span class="symbol">&#8242;</span></p>
                           <p>3<span class="symbol">&#8242;</span> primer: 5<span class="symbol">&#8242;</span>-(n<sub>50</sub>)-GAATTCTTTGTATAGTTCATCCATGC-3<span class="symbol">&#8242;</span></p>
                           <p>Amplify (n<sub>50</sub>)-GFP-<span class="emphasis"><em>URA3</em></span>-GFP-(n<sub>50</sub>) from pCB182 (<a href="#figure8">Figure 8</a>).
                           </p>
                           <div class="sidebar">
                              <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                                 <h4 class="title">Note</h4>
                                 <p>A small band (~800 bp) might be generated during PCR amplification because of homology between tandem GFP repeats. This product
                                    will not interfere in the next step since it does not contain <span class="emphasis"><em>URA3</em></span>.
                                 </p>
                              </div>
                           </div>
                           <div class="mediaobject" align="center"><a name="figure8"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="GFPRfig8.jpg"><img src="GFPRfig8_s.jpg" border="2" align="middle" alt="Isolation of GFP fusion construct figure 9"></a>
  </td>
</tr>
                              </table>
                              <div class="caption">
                                 <p><b>Figure 8. Recombination of GFP-<span class="emphasis"><em>URA3</em></span>-GFP into target sequence.&nbsp;</b>The (n<sub>50</sub>)-GFP-<span class="emphasis"><em>URA3</em></span>-GFP-(n<sub>50</sub>) cassette is amplified from pCB182 and transformed into yeast carrying the bacterial shuttle vector pRS415 (containing the
                                    target sequence). The gene specific extensions (n<sub>50</sub>) flanking GFP-<span class="emphasis"><em>URA3</em></span>-GFP direct homologous recombination between (n<sub>50</sub>)-GFP-<span class="emphasis"><em>URA3</em></span>-GFP-(n<sub>50</sub>) and pRS415. Selection for vector recombinants is achieved by plating the cells in the absence of uracil and leucine. Adapted
                                    from <a href="#bib9">Hawkins et al. (2003)</a>.
                                 </p>
                              </div>
                           </div>
                        </li>
                        <li>
                           <p class="title"><b>Recombination of GFP-<span class="emphasis"><em>URA3</em></span>-GFP into target sequence</b></p>
                           <p>Transform (n<sub>50</sub>)-GFP-<span class="emphasis"><em>URA3</em></span>-GFP-(n<sub>50</sub>) into CBS17 (carrying target sequence-containing pRS415). Select on solid media lacking uracil (selection for GFP-<span class="emphasis"><em>URA3</em></span>-GFP) and leucine (selection for target sequence-containing pRS415; <a href="#figure9">Figure 9</a>).
                           </p>
                        </li>
                        <li>
                           <p class="title"><b>Selection of strains carrying a homogeneous population of GFP-<span class="emphasis"><em>URA3</em></span>-GFP containing plasmids</b></p>
                           <p>The GFP-<span class="emphasis"><em>URA3</em></span>-GFP cassette will recombine specifically into the target plasmid or unspecifically into the yeast genome. In addition, since
                              yeast strains carry multiple copies of CEN-based plasmids (e.g., pRS415), not all of the plasmids need to contain the GFP-<span class="emphasis"><em>URA3</em></span>-GFP insert for growth without uracil (<a href="#figure9">Figure 9</a>).
                           </p>
                           <div class="mediaobject" align="center"><a name="figure9"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="GFPRfig9.jpg"><img src="GFPRfig9_s.jpg" border="2" align="middle" alt="Isolation of GFP fusion construct figure 9"></a>
  </td>
</tr>
                              </table>
                              <div class="caption">
                                 <p><b>Figure 9: Isolation of GFP fusion construct.&nbsp;</b>To obtain yeast carrying only the pRS415-GFP-<span class="emphasis"><em>URA3</em></span>-GFP recombinant vector, plasmids are transfered by cytoduction into the karyogamy-deficient <span class="emphasis"><em>kar1-1</em></span> strain (CBY886). This step eliminates the presence of chromosomal insertions of GFP-<span class="emphasis"><em>URA3</em></span>-GFP or unrecombined pRS415. Selection for the excision of <span class="emphasis"><em>URA3</em></span> and generation of the final GFP fusion product occurs when yeast are grown on on solid medium containing 5-fluoroorotic acid
                                    and lacking leucine. Adapted from <a href="#bib9">Hawkins et al. (2003)</a>.
                                 </p>
                              </div>
                           </div>
                           <p>To obtain a yeast strain containing only plasmids carrying GFP-<span class="emphasis"><em>URA3</em></span>-GFP and no chromosomal insertions, replica-print yeast between different solid media. Transfer individual plasmids into karyogamy-deficient
                              recipient yeast strains (<span class="emphasis"><em>kar1-1</em></span>) by exceptional cytoduction (<a href="#bib5">Dutcher, 1981</a>). The <span class="emphasis"><em>kar1-1</em></span> mutation limits the exchange of chromosomal DNA between cells but does not affect exchange of plasmids or extranuclear material.
                              As a result, haploid cells retain their original nuclear genotype but receive plasmid from the mating partner
                           </p>
                           <p>Mate strains carrying GFP-<span class="emphasis"><em>URA3</em></span>-GFP <span class="emphasis"><em>en masse</em></span> with CBY668 spread out as a lawn on rich medium. Select <span class="emphasis"><em>kar1-1</em></span> recipients on solid medium containing cycloheximide but lacking leucine. Cycloheximide resistance confered by <span class="emphasis"><em>cyh2</em></span> to CBY668 is recessive. Therefore only haploid cells will grow (not diploids and not the original transformants). Only yeast
                              having received the plasmid grow without leucine and only yeast in which the GFP-<span class="emphasis"><em>URA3</em></span>-GFP cassette was integrated into the plasmid grow without uracil.
                           </p>
                        </li>
                        <li>
                           <p class="title"><b>Generation of GFP fusion construct</b></p>
                           <p>To select for the final GFP fusion construct, grow yeast on solid medium containing 5-fluoroorotic acid and lacking leucine
                              (<a href="#figure9">Figure 9</a>). 5-fluoroorotic acid selects against cells containing the <span class="emphasis"><em>URA3</em></span> gene.
                           </p>
                           <div class="sidebar">
                              <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                                 <h4 class="title">Note</h4>
                                 <p>The cytoduction step (step 3), previous to 5-fluoroorotic acid treatment, is essential to obtain plasmid recombinants.</p>
                              </div>
                           </div>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e2015"></a>2.6.&nbsp;Protocol 6: Cell type-specific GFP expression from reconstituted reporter genes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Conventional transcriptional and translational reporters rarely direct expression in a single cell type. Moreover, cell-specific
                  <span class="emphasis"><em>cis</em></span>-regulatory sequences are not available for all 959 somatic cells of <span class="emphasis"><em>C. elegans</em></span>. Advances in the molecular understanding of GFP have led to the development of reconstituted GFP expression systems. Partial
                  GFP peptides (N-GFP: residues 1-157 and C-GFP: residues 158-238) co-expressed in bacteria can form a stable fluorescent product.
                  In eukaryotic cells, GFP fluorescence occurs when N-GFP and C-GFP are fused to short peptides (e.g., leucine zippers) that
                  naturally interact <span class="emphasis"><em>in vivo</em></span> (<a href="#bib7">Ghosh et al., 2000</a>; <a href="#figure10">Figure 10</a>; <a href="#figure11">Figure 11</a>).
               </p>
               <div class="literallayout">
                  <p></p>
               </div>
               <div class="mediaobject" align="center"><a name="figure10"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="450">
<tr>
  <td align="center"><p><a href="GFPRfig10.jpg"><img src="GFPRfig10_s.jpg" border="2" align="middle" alt="NZGFP-CZGFP Reconstitution figure 11"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 10. NZGFP-CZGFP reconstitution.&nbsp;</b>The interaction of the leucine zipper motifs (NZ, CZ) leads to the dimerization of NZGFP and CZGFP monomers. Dimerization
                        results in the reconstitution of functional GFP (green). Adapted from <a href="#bib26">Zhang et al. (2004)</a>.
                     </p>
                  </div>
               </div>
               <div class="mediaobject" align="center"><a name="figure11"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="GFPRfig11.jpg"><img src="GFPRfig11_s.jpg" border="2" align="middle" alt="NZGFP-CZGFP Reconstitution figure 11"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 11. Domain structure of CZGFP and NZGFP.&nbsp;</b>NGFP(1-157) contains a C-terminal peptide linker to the NZ leucine zipper motif. Similarly, CGFP(158-238) has a N-terminal
                        peptide linker to the CZ leucine zipper motif. Adapted from <a href="#bib26">Zhang et al. (2004)</a>.
                     </p>
                  </div>
               </div>
               <p>To apply this approach to <span class="emphasis"><em>C. elegans</em></span>, two expression vectors have been developed that contain GFP polypeptides, N-GFP and C-GFP, fused via a small linker to sequences
                  that can form an antiparallel leucine zipper (NZ and CZ; <a href="#bib26">Zhang et al., 2004</a>). The vectors, refered to as NZGFP and CZGFP, are suited for the cloning of <span class="emphasis"><em>cis</em></span>-regulatory DNA fragments. By combining <span class="emphasis"><em>cis</em></span>-regulatory sequences yielding overlapping expression patterns, one can engineer worm strains that label specific subsets
                  of cells.
               </p>
               <div class="table"><a name="d0e2081"></a><p class="title">Reagents (Chalfie, pers. comm.)</p>
                  <table summary="Reagents (Chalfie, pers. comm.)" border="1">
                     <colgroup>
                        <col align="center">
                        <col align="center">
                        <col align="center">
                     </colgroup>
                     <thead>
                        <tr>
                           <th align="center">Vector</th>
                           <th align="center">Reporter gene</th>
                           <th align="center">Comments</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="center">TU#710</td>
                           <td align="center">NZGFP in pPD95.77</td>
                           <td align="center">GFP portion in NZGFP contains the introns from the original pPD95.77</td>
                        </tr>
                        <tr>
                           <td align="center">TU#711</td>
                           <td align="center">CZGFP in pPD95.77</td>
                           <td align="center">GFP portion in CZGFP contains the intron from the original pPD95.77</td>
                        </tr>
                        <tr>
                           <td align="center">TU#712</td>
                           <td align="center">NZYFP version of TU#710</td>
                           <td align="center">Missense mutations: S65G, V68L, and S72A</td>
                        </tr>
                        <tr>
                           <td align="center">TU#713</td>
                           <td align="center">CZYFP version of TU#711</td>
                           <td align="center">Missense mutations: T203Y</td>
                        </tr>
                        <tr>
                           <td align="center">TU#714</td>
                           <td align="center">NZCFP version of TU#7102</td>
                           <td align="center">Missense mutations: Y66W, N146I, and M153T</td>
                        </tr>
                        <tr>
                           <td align="center">TU#715</td>
                           <td align="center">CZCFP version of TU#711</td>
                           <td align="center">Missense mutations: V163A</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e2139"></a>3.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Ambros, V. (2004). The functions of animal microRNAs. Nature <span class="emphasis"><em>431</em></span>, 350&#8211;355.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15372042&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02871" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Chalfie, M., Y. Tu, et al. (1994). Green fluorescent protein as a marker for gene expression. Science <span class="emphasis"><em>263</em></span>, 802&#8211;805.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8303295&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Conradt, B., and Horvitz, H.R. (1999). The TRA-1A sex determination protein of <span class="emphasis"><em>C. elegans</em></span> regulates sexually dimorphic cell deaths by repressing the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-1" target="_blank">egl-1</a></em></span> cell death activator gene. Cell <span class="emphasis"><em>98</em></span>, 317&#8211;327.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10458607&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(00)81961-3" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Dupuy, D., Li, Q.R., et al. (2004). A first version of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> promoterome. Genome Res. <span class="emphasis"><em>14</em></span>, 2169&#8211;2175.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15489340&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.2497604" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Dutcher, S.K. (1981). Internuclear transfer of genetic information in kar1-1/KAR1 heterokaryons in <span class="emphasis"><em>Saccharomyces cerevisiae</em></span>. Mol. Cell Biol. <span class="emphasis"><em>1</em></span>, 245&#8211;253.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6765600&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Fire, A., Harrison, S.W., et al. (1990). A modular set of <span class="emphasis"><em>lacZ</em></span> fusion vectors for studying gene expression in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Gene <span class="emphasis"><em>93</em></span>, 189&#8211;198.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2121610&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0378-1119(90)90224-F" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Ghosh, I., Hamilton, A.D., et al. (2000). Antiparallel leucine zipper-directed protein reassembly: application to the green
                        fluorescent protein. J. Am. Chem. Soc. <span class="emphasis"><em>122</em></span>, 5658&#8211;5659.</span>
                     <a href="http://dx.doi.org/10.1021/ja994421w" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Hartley, J.L., Temple, G.F., et al. (2000). DNA cloning using in vitro site-specific recombination. Genome Res. <span class="emphasis"><em>10</em></span>, 1788&#8211;1795.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11076863&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.143000" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Hawkins, N.C., Garriga, G.,  et al. (2003). Creating precise GFP fusions in plasmids using yeast homologous recombination.
                        Biotechniques <span class="emphasis"><em>34</em></span>, 74&#8211;78, 80.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12545543&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Heim, R., Cubitt, A.B., et al. (1995). Improved green fluorescence. Nature <span class="emphasis"><em>373</em></span>, 663&#8211;664.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7854443&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/373663b0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Hobert, O. (2002). PCR fusion-based approach to create reporter gene constructs for expression analysis in transgenic <span class="emphasis"><em>C. elegans</em></span>. Biotechniques <span class="emphasis"><em>32</em></span>, 728&#8211;730.</span><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11962590&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hobert, O., and Loria, P.M. (2005). Uses of GFP in <span class="emphasis"><em>C. elegans</em></span>. In: <span class="emphasis"><em>Green Fluorescent Protein: Properties, Applications and Protocols</em></span>. Second edition. Chalfie and Kain, eds. (Wiley).</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Hope, I.A., Stevens, J. et al. (2004). Feasibility of genome-scale construction of promoter::reporter gene fusions for expression
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span> using a multisite gateway recombination system. Genome Res. <span class="emphasis"><em>14</em></span>, 2070&#8211;2075.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15489328&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.2463804" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Hutter, H. (2003). Extracellular cues and pioneers act together to guide axons in the ventral cord of <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>130</em></span>, 5307&#8211;5318.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=13129845&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1242/dev.00727" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Kramer, J.M., French, R.P., et al. (1990). The <span class="emphasis"><em>Caenorhabditis elegans <a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a></em></span> gene, which interacts with the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sqt-1" target="_blank">sqt-1</a></em></span> collagen gene to determine organismal morphology, encodes a collagen. Mol. Cell Biol. <span class="emphasis"><em>10</em></span>, 2081&#8211;2089.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1970117&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Landy, A. (1989). Dynamic, structural, and regulatory aspects of lambda site-specific recombination. Annu. Rev. Biochem. <span class="emphasis"><em>58</em></span>, 913&#8211;949.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2528323&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1146/annurev.bi.58.070189.004405" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Mango, S.E. (2001). Stop making nonsense: the <span class="emphasis"><em>C. elegans smg</em></span> genes. Trends Genet. <span class="emphasis"><em>17</em></span>, 646&#8211;653.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11672865&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0168-9525(01)02479-9" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Marsischky, G., and LaBaer J. (2004). Many paths to many clones: a comparative look at high-throughput cloning methods. Genome
                        Res. <span class="emphasis"><em>14</em></span>, 2020&#8211;2028.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15489321&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gr.2528804" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Maryon, E.B., Coronado, R., et al. (1996). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-68" target="_blank">unc-68</a></em></span> encodes a ryanodine receptor involved in regulating <span class="emphasis"><em>C. elegans</em></span> body-wall muscle contraction. J. Cell Biol. <span class="emphasis"><em>134</em></span>, 885&#8211;893.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8769414&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.134.4.885" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Mello, C.C., Kramer, J.M., et al. (1991). Efficient gene transfer in <span class="emphasis"><em>C.elegans</em></span>: extrachromosomal maintenance and integration of transforming sequences. EMBO J. <span class="emphasis"><em>10</em></span>, 3959&#8211;3970.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1935914&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Miller, D.M., III, Desai, N.S. et al. (1999). Two-color GFP expression system for <span class="emphasis"><em>C. elegans</em></span>. Biotechniques <span class="emphasis"><em>26</em></span>, 914&#8211;918, 920&#8211;921.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10337485&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Pulak, R., and Anderson P. (1993). mRNA surveillance by the <span class="emphasis"><em>Caenorhabditis elegans smg</em></span> genes. Genes Dev. <span class="emphasis"><em>7</em></span>, 1885&#8211;1897.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8104846&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Shaner, N.C., Campbell, R.E. et al. (2004). Improved monomeric red, orange and yellow fluorescent proteins derived from Discosoma
                        sp. red fluorescent protein. Nat. Biotechnol. <span class="emphasis"><em>22</em></span>, 1567&#8211;1572.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15558047&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nbt1037" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Tsalik, E.L., Niacaris, T., Wenick, A.S., Pau, K., Avery, L., and Hobert, O. (2003). LIM homeobox gene-dependent expression
                        of biogenic amine receptors in restricted regions of the <span class="emphasis"><em>C. elegans</em></span> nervous system. Dev. Biol. <span class="emphasis"><em>263</em></span>, 81&#8211;102.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14568548&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0012-1606(03)00447-0" target="_blank">Article</a>
                     		
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Wenick, A.S., and Hobert, O. (2004). Genomic <span class="emphasis"><em>cis</em></span>-regulatory architecture and <span class="emphasis"><em>trans</em></span>-acting regulators of a single interneuron-specific gene battery in <span class="emphasis"><em>C. elegans</em></span>. Dev. Cell <span class="emphasis"><em>6</em></span>, 757&#8211;770.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15177025&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2004.05.004" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Wightman, B., Ha, I., et al. (1993). Post transcriptional regulation of the heterochronic gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-14" target="_blank">lin-14</a></em></span> by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-4" target="_blank">lin-4</a></em></span> mediates temporal pattern formation in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>75</em></span>, 855&#8211;862.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8252622&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(93)90530-4" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Wilkinson, H.A., Fitzgerald, K., et al. (1994). Reciprocal changes in expression of the receptor <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> and its ligand <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lag-2" target="_blank">lag-2</a></em></span> prior to commitment in a <span class="emphasis"><em>C. elegans</em></span> cell fate decision. Cell <span class="emphasis"><em>79</em></span>, 1187&#8211;1198.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8001154&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(94)90010-8" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     			<span class="bibliomisc">Zhang, S., Ma, C.,  et al. (2004). Combinatorial marking of cells and organelles with reconstituted fluorescent proteins.
                        Cell <span class="emphasis"><em>119</em></span>, 137&#8211;144.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15454087&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2004.09.012" target="_blank">Article</a>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised April 18, 2005. Published April 5, 2006. This chapter should be cited as: Boulin, T.
                  et al. Reporter gene fusions (April 5, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.106.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Thomas Boulin et al. This is an open-access article distributed under the terms of the Creative Commons Attribution
                  License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source
                  are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e37" href="#d0e37">&sect;</a></sup>Both of these authors contributed equally
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e44" href="#d0e44">&#8224;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:or38@columbia.edu" target="_top">or38@columbia.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

