<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Genetic suppression
         
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#geneticsuppression_ArticleID" title="Genetic suppression&#xA;        ">
      <link rel="next" href="#d0e73" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="geneticsuppression.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="geneticsuppression_ArticleID"></a>Genetic suppression<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                             
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="firstname">Jonathan</span> <span class="surname">Hodgkin<em><span class="remark">
                                                     <sup><a name="d0e35" href="#ftn.d0e35">&sect;</a></sup>
                                                 </span></em>
                                       </span><br></span><span class="affiliation"><span class="address">Genetics Unit, Department of Biochemistry, University of Oxford, Oxford OX1 3QU, UK
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_geneticsgenomics.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > Genetics and genomics<br />>> Genetics
		</a></div><hr />

<a href="http://www.wormbase.org/forums/index.php?board=190.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Genetic+suppression" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e73">1. Introduction</a></span></dt>
               <dt><span class="sect1"><a href="#d0e97">2. Intragenic suppression by same site replacement</a></span></dt>
               <dt><span class="sect1"><a href="#d0e110">3. Intragenic suppression by compensatory second site mutation</a></span></dt>
               <dt><span class="sect1"><a href="#d0e126">4. Intragenic suppression by altered splicing</a></span></dt>
               <dt><span class="sect1"><a href="#d0e139">5. Intragenic suppression of dominant mutations by loss-of-function in <span class="emphasis"><em>cis</em></span>.</a></span></dt>
               <dt><span class="sect1"><a href="#d0e202">6. Extragenic suppression &#8212; overview</a></span></dt>
               <dt><span class="sect1"><a href="#d0e461">7. Informational suppression &#8212; overview</a></span></dt>
               <dt><span class="sect1"><a href="#d0e469">8. Nonsense suppression</a></span></dt>
               <dt><span class="sect1"><a href="#d0e818">9. Extragenic suppression by modified splicing</a></span></dt>
               <dt><span class="sect1"><a href="#d0e940">10. Suppression by loss of NMD (Nonsense Mediated Decay)</a></span></dt>
               <dt><span class="sect1"><a href="#d0e961">11. Extragenic suppression by non-specific physiological effects</a></span></dt>
               <dt><span class="sect1"><a href="#d0e982">12. Suppression by altered gene dosage</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1031">13. Bypass suppression</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1069">14. Extragenic suppression by interacting components</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1125">15. Suppression by removal of a toxic protein or protein complex</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1298">16. Suppression by modulators of activity</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1400">17. Suppression by epistasis</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1521">18. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title"><span class="bold"><strong>Abstract</strong></span></p>
            <p><span class="bold"><strong>Genetic suppression has provided a very powerful tool for analyzing</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans.</strong></span></em></span>
                           <span class="bold"><strong>Suppression experiments are facilitated by the ability to handle very large numbers of
                                 individuals and to apply powerful selections. Because the animal grows as a
                                 self-fertilizing diploid, both dominant and recessive suppressors can be recovered. Many
                                 different kinds of suppression have been reported. These are discussed by category, with
                                 examples, together with discussion of how suppressors can be used to interpret the
                                 underlying biology, and to enable further experimentation. Suppression phenomena can be
                                 divided into intragenic and extragenic classes, depending on whether the suppressor lies
                                 in the same gene as the starting mutation, or in a different gene. Intragenic types
                                 include same-site replacement, compensatory mutation, alteration in splicing, and
                                 reversion of dominant mutations by</strong></span> <span class="emphasis"><em><span class="bold"><strong>cis-</strong></span></em></span><span class="bold"><strong>knockout. Extragenic suppression can occur by a
                                 variety of informational mechanisms, such as alterations in splicing, translation or
                                 nonsense-mediated decay. In addition, extragenic suppression can occur by bypass, dosage
                                 effects, product interaction, or removal of toxic products. Within signaling pathways,
                                 suppression can occur by modulating the strength of signal transmission, or by epistatic
                                 interactions that can reveal the underlying regulatory hierarchies. In</strong></span> <span class="emphasis"><em><span class="bold"><strong>C. elegans</strong></span></em></span>
                           <span class="bold"><strong>biology, the processes of muscle development, vulva formation and sex determination have
                                 provided remarkably rich arenas for the investigation and exploitation of suppression.</strong></span></p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e73"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>Genetic suppressors provide some of the most powerful tools available for exploring
                           gene expression, function and interaction. A suppressor mutation can be defined as a
                           second mutation, at a site distinct from the mutation under study, that suppresses the
                           phenotypic effect of the first mutation. <span class="emphasis"><em>C. elegans</em></span> is especially suited for suppression
                           analysis, for several practical reasons: 
            </p>
            <div class="orderedlist">
               <ol type="1">
                  <li>
                     <p>Very large numbers of individuals can be handled, so that extremely rare
                                                mutational events can occur within a single experimental population.
                     </p>
                  </li>
                  <li>
                     <p>Strong selections can be applied, which may permit the detection and
                                                recovery of a single individual from a population of one hundred thousand or more.
                     </p>
                  </li>
                  <li>
                     <p>Mutations are easily generated, particularly by the chemical mutagen EMS.</p>
                  </li>
                  <li>
                     <p>The predominant mode of reproduction is by self-fertilization, so that
                                                both recessive and dominant suppressors can be recovered readily.
                     </p>
                  </li>
               </ol>
            </div>
            <p>
                       
            </p>
            <p>Genetic suppression can arise in a variety of ways, which are described in the
                           sections below. Depending on the nature of the suppressive interaction, different kinds
                           of information can be obtained about the initial gene studied.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e97"></a>2.&nbsp;Intragenic suppression by same site replacement
                     </h2>
                  </div>
               </div>
            </div>
            <p>In the simplest and least informative scenario, if a mutant carries a single base
                           change difference from wildtype, then reversion can occur simply by mutating the
                           sequence back to wildtype. However, intracodonic suppression can also occur, if a
                           different base change can result in a functional coding sequence. An example is
                           provided by a Gln-to-Pro mutant of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-31" target="_blank">dpy-31</a></em></span>, encoding a collagen-processing protease,
                           which has been repeatedly reverted either by back mutation, Pro-to-Gln (CCT -&gt;
                           CTT), or by the only other acceptable change in this codon, Pro-to-Ser (CCT -&gt;
                           TCT) (<a href="#bib42">Novelli et al., 2004</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e110"></a>3.&nbsp;Intragenic suppression by compensatory second site mutation
                     </h2>
                  </div>
               </div>
            </div>
            <p>In principle, missense mutations that result in a non-functional protein can sometimes
                           be suppressed by introducing another change elsewhere in the protein. Relatively few
                           cases of this effect have been studied so far in <span class="emphasis"><em>C. elegans</em></span>. An example is provided by
                           the reversion of temperature-sensitive missense mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span>, which encodes a cell
                           interaction protein; this experiment yielded a variety of second-site missense changes
                           that are able to correct the original defect (<a href="#bib34">Lissemore et al., 1993</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e126"></a>4.&nbsp;Intragenic suppression by altered splicing
                     </h2>
                  </div>
               </div>
            </div>
            <p>Sometimes intragenic reversion can be achieved by a second mutation that leads to
                           omission of a deleterious mutation by means of exon skipping. This is particularly
                           likely to happen with genes that produce multiple splice variants, as illustrated by
                           studies of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span>. This gene encodes multiple isoforms of perlecan, which are proteins
                           required for myofilament anchorage to the basement membrane underlying muscle cells.
                           Nonsense mutations located in some of the alternatively spliced exons can be suppressed
                           by a G-to-A change in the acceptor site adjoining the mutant exon, which leads to its
                           omission; the resulting RNA remains in frame and produces a functional protein (<a href="#bib46">Rogalski
                              et al., 1995</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e139"></a>5.&nbsp;Intragenic suppression of dominant mutations by loss-of-function in <span class="emphasis"><em>cis</em></span>.
                     </h2>
                  </div>
               </div>
            </div>
            <p>Most genes in <span class="emphasis"><em>C. elegans</em></span> are haplo-sufficient, so the majority of dominant mutations
                           are associated with gain-of-function effects, which result in a toxic or inappropriately
                           expressed gene product (<a href="#bib43">Park and Horvitz, 1986</a>). The dominant effect of
                           such a mutation can be suppressed by introducing a second mutation elsewhere in the
                           mutated gene that results in partial or complete loss-of-function in its product.
            </p>
            <p>An early example was provided by studies of the myosin gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span>. Some dominant
                           mutations of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> cause slow movement or paralysis in single dose, by poisoning thick
                           filament assembly. These mutations, such as the allele <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1152;class=Variation" target="_blank">e1152</a></em></span>, can be reverted by
                           introducing null or partial loss-of-function mutations in <span class="emphasis"><em>cis</em></span>. Heterozygotes carrying
                           just the dominant allele <span class="emphasis"><em>(<a href="http://www.wormbase.org/db/get?name=e1152;class=Variation" target="_blank">e1152</a>/+)</em></span> are slow, whereas <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1152;class=Variation" target="_blank">e1152</a> null / +</em></span> heterozygotes are
                           wildtype. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> gene is essential for movement, so <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1152;class=Variation" target="_blank">e1152</a> null</em></span> double homozygotes
                           are paralysed, but the mutations are initially recovered in healthy heterozygotes with
                           normal movement. Selection for improved movement of <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1152;class=Variation" target="_blank">e1152</a>/+</em></span> heterozygotes therefore
                           provided an efficient means of recovering many mutations in this gene (<a href="#bib1">Anderson and
                              Brenner, 1984</a>).
            </p>
            <p>Similar strategies have been used frequently and effectively to generate multiple
                           loss-of-function mutations in important or essential genes.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e202"></a>6.&nbsp;Extragenic suppression &#8212; overview
                     </h2>
                  </div>
               </div>
            </div>
            <p>Suppressing the effects of a mutant gene by changes elsewhere in the genome can occur
                           in a number of ways. Extragenic suppression is usually revealed by crossing a revertant
                           line with wildtype, and observing the re-appearance of the original mutant after
                           Mendelian segregation. Numerous cases of extragenic suppression have now been identified
                           and analysed in greater or lesser depth. Many different phenomena are involved in the
                           various cases. About 40 genes in the <span class="emphasis"><em>sup</em></span> class have been defined thus far, and in
                           addition separate gene classes have been established for many of the more specific
                           processes that have been studied. As shown in <a href="#table1" title="Table 1. Suppressor gene classes">Table 1</a>, research on 21 suppressor gene
                           classes has been published. At least 20 further classes have been established (see
                           <a href="http://www.wormbase.org/" target="_blank">Wormbase</a>) but their properties are as yet unpublished. It should be noted that
                           suppressor mutations frequently turn out to be special alleles of genes that have
                           already been well studied and given more informative names, so the names of suppressor
                           loci are more transient than most.
            </p>
            <div class="table"><a name="table1"></a><p class="title">Table 1. Suppressor gene classes</p>
               <table summary="Table 1. Suppressor gene classes" border="1">
                  <colgroup>
                     <col align="left">
                     <col align="left">
                     <col align="left">
                  </colgroup>
                  <thead>
                     <tr>
                        <th align="left">Gene name</th>
                        <th align="left">Explanation</th>
                        <th align="left">Initial references</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td align="left"><span class="emphasis"><em>cps</em></span></td>
                        <td align="left"><a href="http://www.wormbase.org/db/seq/protein?name=CED-3;class=protein" target="_blank">CED-3</a> Protease Suppressor
                        </td>
                        <td align="left"><a href="#bib56">Wang et al., 2002</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>des</em></span></td>
                        <td align="left">DEgeneration Suppressor</td>
                        <td align="left"><a href="#bib55">Treinin et al., 1998</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>eor</em></span></td>
                        <td align="left">Egl-1 suppressor/DiO uptake defect/Raf enhancer</td>
                        <td align="left"><a href="#bib19">Howard and Sundaram, 2002</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>ksr</em></span></td>
                        <td align="left">Kinase Suppressor of activated Ras</td>
                        <td align="left"><a href="#bib53">Sundaram and Han, 1995</a>; <a href="#bib27">Kornfeld et al., 1995</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>scd</em></span></td>
                        <td align="left">Suppressor of Constitutive Dauer formation</td>
                        <td align="left"><a href="#bib21">Inoue and Thomas, 2000</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sel</em></span></td>
                        <td align="left">Suppressor/Enhancer of Lin-12</td>
                        <td align="left"><a href="#bib52">Sundaram and Greenwald, 1993</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>seu</em></span></td>
                        <td align="left">Suppressor of Ectopic Unc-5</td>
                        <td align="left"><a href="#bib6">Colavita and Culotti, 1998</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sgs</em></span></td>
                        <td align="left">Suppressor of G protein S defect</td>
                        <td align="left"><a href="#bib28">Korswagen et al., 1998</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sli</em></span></td>
                        <td align="left">Suppressor of LIneage defect</td>
                        <td align="left"><a href="#bib23">Jongeward et al., 1995</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>smg</em></span></td>
                        <td align="left">Suppressor, Morphological effect on Genitalia</td>
                        <td align="left"><a href="#bib18">Hodgkin et al., 1989</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>smu</em></span></td>
                        <td align="left">Suppressor of Mec and Unc defects</td>
                        <td align="left"><a href="#bib36">Lundquist and Herman, 1994</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>soc</em></span></td>
                        <td align="left">Suppressor Of Clr</td>
                        <td align="left"><a href="#bib49">Selfors et al., 1998</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sog</em></span></td>
                        <td align="left">Suppressor Of Glp</td>
                        <td align="left"><a href="#bib38">Maine and Kimble, 1993</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sol</em></span></td>
                        <td align="left">Suppressor Of Lurcher movement defect</td>
                        <td align="left"><a href="#bib65">Zheng et al., 2004</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sop</em></span></td>
                        <td align="left">Suppressor Of Pal</td>
                        <td align="left"><a href="#bib64">Zhang and Emmons, 2000</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>spr</em></span></td>
                        <td align="left">Suppressor of PResenilin defect</td>
                        <td align="left"><a href="#bib60">Wen et al. 2000</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>srl</em></span></td>
                        <td align="left">Suppressor of Roller Lethality</td>
                        <td align="left"><a href="#bib3">Barbazuk et al., 1994</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup</em></span></td>
                        <td align="left">SUPpressor</td>
                        <td align="left"><a href="#bib45">Riddle and Brenner, 1978</a>; <a href="#bib58">Waterston and Brenner, 1978</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sur</em></span></td>
                        <td align="left">SUppressor of activated Ras</td>
                        <td align="left"><a href="#bib29">Lackner et al., 1994</a>; <a href="#bib62">Wu and Han, 1994</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sus</em></span></td>
                        <td align="left">SUppressor of Suppressor</td>
                        <td align="left"><a href="#bib5">Brown and Riddle, 1985</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>suv</em></span></td>
                        <td align="left">SUppressor of Vulvaless</td>
                        <td align="left"><a href="#bib15">Hajnal et al., 1997</a></td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <p>Space does not permit a full description of all the classes listed in <a href="#table1" title="Table 1. Suppressor gene classes">Table 1</a>, but
               		some of the major types of suppression that have been encountered are discussed below,
                           using particular examples.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e461"></a>7.&nbsp;Informational suppression &#8212; overview
                     </h2>
                  </div>
               </div>
            </div>
            <p>Alteration of the general machinery for transcription, RNA processing and translation
                           can sometimes ameliorate or suppress certain kinds of mutation in many different genes.
                           The hallmarks of informational suppression, as this phenomenon is known, are
                           allele-specificity and gene-non-specificity. Three kinds of informational suppression
                           have been described so far in <span class="emphasis"><em>C. elegans</em></span>: nonsense suppression; suppression by modified
                           splicing; suppression by loss of nonsense-mediated decay (NMD).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e469"></a>8.&nbsp;Nonsense suppression
                     </h2>
                  </div>
               </div>
            </div>
            <p>A large fraction of point mutations identified in <span class="emphasis"><em>C. elegans</em></span> are changes from sense to
                           nonsense codons, either UAG (amber), UAA (ochre) or UGA (opal), resulting in polypeptide
                           chain termination and (usually) complete or almost complete loss of gene function. These
                           nonsense mutants can potentially be suppressed by nonsense suppressor tRNAs, in which a
                           sense codon has been changed to be able to decode the nonsense triplet. In practice,
                           only amber suppressors have been recovered. Early work identified two suppressor loci,
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-5" target="_blank">sup-5</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-7" target="_blank">sup-7</a></em></span>, which could act to suppress certain mutations in a wide variety of
                           genes (<a href="#bib58">Waterston and Brenner, 1978</a>; <a href="#bib57">Waterston, 1981</a>). These
                           were subsequently shown to be amber suppressor mutations of two Trp tRNA genes, carrying
                           alterations in their anticodons from CCA to CTA (<a href="#bib61">Wills, et al., 1983</a>;
                           <a href="#bib4">Bolten et al., 1984</a>).
            </p>
            <p>There are twelve members of the Trp tRNA family in <span class="emphasis"><em>C. elegans</em></span>, but only two can be
                           mutated to yield strong amber suppressors. The same two mutations, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-5" target="_blank">sup-5</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-7" target="_blank">sup-7</a></em></span>, were
                           isolated repeatedly in independent experiments (<a href="#bib57">Waterston, 1981</a>). Weak
                           amber suppressors were subsequently obtained for six additional members of the set (see
                           <a href="#table2" title="Table 2. Amber nonsense suppressors">Table 2</a>), by selecting for suppressors using amber mutants that are very easily
                           suppressed such as several in the sex determination gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-3" target="_blank">tra-3</a></em></span> (<a href="#bib16">Hodgkin, 1985</a>;
                           <a href="#bib25">Kondo et al., 1988</a>; <a href="#bib26">Kondo et al., 1990</a>). Probably only a
                           trace of full length <a href="http://www.wormbase.org/db/seq/protein?name=TRA-3;class=protein" target="_blank">TRA-3</a> protein is sufficient to provide enough activity for normal
                           sexual development, so weak suppressors can be recovered. As indicated in <a href="#table2" title="Table 2. Amber nonsense suppressors">Table 2</a>, the
                           eight identified amber suppressors appear to be expressed at different levels, and they
                           also exhibit some differences between tissues in the strength of suppression (<a href="#bib26">Kondo et
                              al., 1990</a>). In vitro observations on the transcription of these genes
                           that are consistent with these different strengths have been reported (Li et al, 1988). The differences in strength
               show that flanking sequences can exert a
                           strong influence on expression level of these tRNAs, despite the fact that all eight are
                           identical in the internal promoter sequences believed to be sufficient for Pol III
                           transcription of tRNA genes.
            </p>
            <div class="table"><a name="table2"></a><p class="title">Table 2. Amber nonsense suppressors</p>
               <table summary="Table 2. Amber nonsense suppressors" border="1">
                  <colgroup>
                     <col align="left">
                     <col align="left">
                     <col align="left">
                     <col align="left">
                  </colgroup>
                  <thead>
                     <tr>
                        <th align="left">Gene name (allele)</th>
                        <th align="left">Class (other name)</th>
                        <th align="left">Approximate relative strength</th>
                        <th align="left">Reference</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-5(<a href="http://www.wormbase.org/db/get?name=e1464;class=Variation" target="_blank">e1464</a>)</em></span></td>
                        <td align="left">Trp (<span class="emphasis"><em>rtw-1</em></span>)
                        </td>
                        <td align="left">7</td>
                        <td align="left"><a href="#bib58">Waterston and Brenner (1978)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-7(st5)</em></span></td>
                        <td align="left">Trp (<span class="emphasis"><em>rtw-2</em></span>)
                        </td>
                        <td align="left">8</td>
                        <td align="left"><a href="#bib57">Waterston (1981)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-21(<a href="http://www.wormbase.org/db/get?name=e1957;class=Variation" target="_blank">e1957</a>)</em></span></td>
                        <td align="left">Trp (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rtw-10" target="_blank">rtw-10</a></em></span>)
                        </td>
                        <td align="left">3</td>
                        <td align="left"><a href="#bib16">Hodgkin (1985)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-24(<a href="http://www.wormbase.org/db/get?name=st354;class=Variation" target="_blank">st354</a>)</em></span></td>
                        <td align="left">Trp (<span class="emphasis"><em>rtw-4</em></span>)
                        </td>
                        <td align="left">4</td>
                        <td align="left"><a href="#bib25">Kondo et al. (1988)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-28(<a href="http://www.wormbase.org/db/get?name=e2058;class=Variation" target="_blank">e2058</a>)</em></span></td>
                        <td align="left">Trp (<span class="emphasis"><em>rtw-8</em></span>)
                        </td>
                        <td align="left">2</td>
                        <td align="left"><a href="#bib25">Kondo et al. (1988)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-29(<a href="http://www.wormbase.org/db/get?name=e1986;class=Variation" target="_blank">e1986</a>)</em></span></td>
                        <td align="left">Trp (<span class="emphasis"><em>rtw-9</em></span>)
                        </td>
                        <td align="left">1</td>
                        <td align="left"><a href="#bib25">Kondo et al. (1988)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-33(<a href="http://www.wormbase.org/db/get?name=st389;class=Variation" target="_blank">st389</a>)</em></span></td>
                        <td align="left">Trp (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rtw-11" target="_blank">rtw-11</a></em></span>)
                        </td>
                        <td align="left">4</td>
                        <td align="left"><a href="#bib26">Kondo et al. (1990)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-34(<a href="http://www.wormbase.org/db/get?name=e2227;class=Variation" target="_blank">e2227</a>)</em></span></td>
                        <td align="left">Trp (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rtw-12" target="_blank">rtw-12</a></em></span>)
                        </td>
                        <td align="left">3</td>
                        <td align="left"><a href="#bib26">Kondo et al. (1990)</a></td>
                     </tr>
                     <tr>
                        <td align="left">&nbsp;</td>
                        <td align="left">&nbsp;</td>
                        <td align="left">&nbsp;</td>
                        <td align="left">&nbsp;</td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-22(<a href="http://www.wormbase.org/db/get?name=e2057;class=Variation" target="_blank">e2057</a>)</em></span></td>
                        <td align="left">Non-Trp</td>
                        <td align="left">2</td>
                        <td align="left"><a href="#bib16">Hodgkin (1985)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-23(<a href="http://www.wormbase.org/db/get?name=e2059;class=Variation" target="_blank">e2059</a>)</em></span></td>
                        <td align="left">?Non-Trp</td>
                        <td align="left">2</td>
                        <td align="left"><a href="#bib16">Hodgkin (1985)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-32(<a href="http://www.wormbase.org/db/get?name=e1958;class=Variation" target="_blank">e1958</a>)</em></span></td>
                        <td align="left">Non-Trp</td>
                        <td align="left">2</td>
                        <td align="left"><a href="#bib16">Hodgkin (1985)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-?(st402)</em></span></td>
                        <td align="left">Non-Trp</td>
                        <td align="left">3</td>
                        <td align="left"><a href="#bib26">Kondo et al. (1990)</a></td>
                     </tr>
                     <tr>
                        <td align="left"><span class="emphasis"><em>sup-?(st414)</em></span></td>
                        <td align="left">Non-Trp</td>
                        <td align="left">2</td>
                        <td align="left"><a href="#bib26">Kondo et al. (1990)</a></td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <p>Too much suppression is clearly deleterious, because animals carrying two doses of
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-7" target="_blank">sup-7</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-5" target="_blank">sup-5</a></em></span> are inviable or subviable, especially at low temperature (<a href="#bib57">Waterston,
                              1981</a>). Presumably the deleterious effects result from too much
                           readthrough of natural stop codons. Selection for improved low-temperature viability of
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-7" target="_blank">sup-7</a></em></span> animals was used to generate deletions or loss-of-function mutations of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-7" target="_blank">sup-7</a></em></span>,
                           suggesting that the inviability is due to readthrough rather than loss of the normal
                           function of the gene. The same effect may explain why ochre or opal suppressors have not
                           been recovered.
            </p>
            <p>Additional weak amber suppressors have been isolated, which do not correspond to Trp
                           tRNA genes, but they have not yet been identified at the molecular level (<a href="#table2" title="Table 2. Amber nonsense suppressors">Table 2</a>). Some
                           of these are likely to be Gln tRNA genes, because a Gln CTG anticodon can be readily
                           mutated to read amber codons. Suppression by an amber-suppressing Ser tRNA from
                           Drosophila, introduced as a transgene, has been reported (<a href="#bib44">Pilgrim and Bell, 1993</a>), demonstrating that endogenous Ser tRNA genes might also yield amber
                       suppressors. As well as providing valuable tools for examining mutations in
                       particular genes and for studying tRNA gene expression, amber suppressors have been useful
                       in the development of methods for microinjection and transformation of <span class="emphasis"><em>C. elegans</em></span> (<a href="#bib24">Kimble et
                          al., 1982</a>; <a href="#bib9">Fire, 1986</a>) The dose-sensitivity of strong amber
                   suppressors also allows selection of single-copy transformants (<a href="#bib10">Fire et al., 1990</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e818"></a>9.&nbsp;Extragenic suppression by modified splicing
                     </h2>
                  </div>
               </div>
            </div>
            <p>Two kinds of suppression by modified splicing have been reported. The first kind
                           depends on compensatory mutations in splice-site recognition machinery, and the second
                           is associated with mutations that affect alternative splicing.
            </p>
            <p>The first kind involves allele-specific suppression of mutations that change the
                           conserved and essential GU donor sequence at the 5&#8217; end of an intron, by means of
                           compensatory changes in the sequence of a U1 snRNA gene. In the case of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-6" target="_blank">sup-6</a></em></span>, a donor
                           site mutation (AGgu --&gt; AGau) in allele <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e309;class=Variation" target="_blank">e309</a></em></span> of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-13" target="_blank">unc-13</a></em></span> is suppressed by a change
                           in one of the U1 snRNA genes (&#8230;UCCAUU&#8230; --&gt; UCUAUU) (<a href="#bib63">Zahler et al., 2004</a>). In a related case, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-39" target="_blank">sup-39</a></em></span>, a donor site mutation (AGgu --&gt; AGuu) in
                           allele <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e936;class=Variation" target="_blank">e936</a></em></span> of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-73" target="_blank">unc-73</a></em></span> is suppressed by a change in a different U1 snRNA gene (&#8230;UCCAUU&#8230;
                           --&gt;&#8230; UCAAUU&#8230;) (<a href="#bib48">Run et al., 1996</a>; <a href="#bib47">Roller et al., 2000</a>; <a href="#bib63">Zahler et al., 2004</a>). These effects are consistent with the proposed role of U1 in pairing at the
                           donor site during splicing.
            </p>
            <p>In <span class="emphasis"><em>C. elegans</em></span>, there are 12 genes encoding U1 snRNA, yet only two have yielded
                           suppressors of this type (<a href="#bib63">Zahler et al., 2004</a>). As with the failure to
                           find ochre or opal suppressors discussed above, it is possible that most U1 suppressor
                           mutations will be either too weak, because of redundancy, or too deleterious, because of
                           effects on normal genes. In support of the latter possibility, both <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-6" target="_blank">sup-6</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-39" target="_blank">sup-39</a></em></span>
                           exhibit defects when homozygous: <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-6" target="_blank">sup-6</a></em></span> homozygotes arrest in late-larval development,
                           and 50<span class="symbol">%</span> of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-39" target="_blank">sup-39</a></em></span> homozygotes die as embryos. In addition, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-39" target="_blank">sup-39</a></em></span> animals that escape
                           this embryonic lethality frequently exhibit two rows of oocytes rather than one in adult
                           hermaphrodites, an unusual and unexplained phenotype (<a href="#bib48">Run et al., 1996</a>).
            </p>
            <p>The second kind of extragenic suppression by altered splicing involves the regulation
                           of alternative splicing. As described above, some mutations of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span> perlecan gene
                           are located in alternatively spliced exons and can be suppressed by intragenic mutations
                           that cause exon-skipping. A subset of these, those located in exon 17, can also be
                           suppressed by loss-of-function mutations in two genes, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smu-1" target="_blank">smu-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smu-2" target="_blank">smu-2</a></em></span>, which encode
                           proteins homologous to mammalian spliceosome-associated factors (<a href="#bib50">Spartz et al., 2004</a>). Loss of these factors leads to enhanced skipping of exon 17, and
                           consequently to suppression of mutations affecting this exon. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smu-1" target="_blank">smu-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smu-2" target="_blank">smu-2</a></em></span>
                           mutations also weakly suppress the pleiotropic effects of mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-8" target="_blank">mec-8</a></em></span>, which
                           encodes another regulator of alternative splicing (<a href="#bib36">Lundquist and Herman, 1994</a>). This suggests that the <a href="http://www.wormbase.org/db/seq/protein?name=SMU-1;class=protein" target="_blank">SMU-1</a> and <a href="http://www.wormbase.org/db/seq/protein?name=SMU-2;class=protein" target="_blank">SMU-2</a> factors can affect the splicing of
                           multiple genes, although neither is essential for viability. It seems very likely that
                           further cases of suppression by modified splicing will be encountered in the future.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e940"></a>10.&nbsp;Suppression by loss of NMD (Nonsense Mediated Decay)
                     </h2>
                  </div>
               </div>
            </div>
            <p>Mutations resulting in a premature stop codon trigger the NMD system of message
                           degradation. If the
                           NMD system is absent, then mRNAs with such codons have longer lifetimes, increasing the
                           chance of readthrough at the stop codon and the production of a functional protein.
                           Alternatively, if the stop codon is located close to the end of the message, the
                           truncated polypeptide produced after translation of the stabilized message may be long
                           enough to have substantial activity. Nonsense mutations are frequently generated in <span class="emphasis"><em>C.
                                 elegans</em></span> forward mutagenesis screens, so a variety of suppressor screens have yielded
                           mutations in the <span class="emphasis"><em>C. elegans</em></span> NMD genes (<a href="#bib18">Hodgkin et al., 1989</a>). Seven genes
                           required for NMD have been identified in this way, which are called <span class="emphasis"><em>smg</em></span> (for Suppressor
                           with Morphogenetic effect on Genitalia). Loss of NMD does not lead to inviability in <span class="emphasis"><em>C.
                                 elegans</em></span>, but does result in an abnormal morphology of the hermaphrodite vulva and of the
                           male tail, hence the name. Suppression is generally weaker than with amber suppression.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e961"></a>11.&nbsp;Extragenic suppression by non-specific physiological effects
                     </h2>
                  </div>
               </div>
            </div>
            <p>Some cases of suppression probably result from general effects on the physiology of
                           the animal, which can alter the expression of a variety of different genes. An example
                           is provided by the recovery of mutations in eight or more cuticle collagen genes, all of
                           which cause a &#8216;dumpy left roller&#8217; phenotype, as weak suppressors of two quite different
                           mutant defects: germline proliferation, in the case of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> (<a href="#bib37">Maine and Kimble, 1989</a>), and embryonic gastrulation, in the case of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=emb-5" target="_blank">emb-5</a></em></span> (<a href="#bib41">Nishiwaki and Miwa,
                              1998</a>). The mechanism involved is unknown, but indirect effects via
                           alteration of extracellular matrix or of stress response have been suggested as possible explanations.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e982"></a>12.&nbsp;Suppression by altered gene dosage
                     </h2>
                  </div>
               </div>
            </div>
            <p>Sometimes the lack of a particular protein can be compensated for by increasing the
                           amount of an alternative, related protein. This is illustrated by the analysis of a gene
                           initially called <span class="emphasis"><em>sup-3</em></span> (<a href="#bib45">Riddle and Brenner, 1978</a>), identified by mutations
                           that acted to suppress <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-15" target="_blank">unc-15</a></em></span> mutants; these lack either the major body wall
                           myosin (<a href="http://www.wormbase.org/db/seq/protein?name=UNC-54;class=protein" target="_blank">UNC-54</a>) or another thick filament component, paramyosin (<a href="http://www.wormbase.org/db/seq/protein?name=UNC-15;class=protein" target="_blank">UNC-15</a>). Molecular
                           analysis of <span class="emphasis"><em>sup-3</em></span> showed that its mutant alleles are duplications or triplications of
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=myo-3" target="_blank">myo-3</a></em></span>, which encodes a minor myosin. Normally <a href="http://www.wormbase.org/db/seq/protein?name=MYO-3;class=protein" target="_blank">MYO-3</a> accounts for only 15<span class="symbol">%</span> of the body
                           wall myosin, but its level is increased in the duplication alleles, and can thereby
                           compensate for the lack of <a href="http://www.wormbase.org/db/seq/protein?name=UNC-54;class=protein" target="_blank">UNC-54</a> or <a href="http://www.wormbase.org/db/seq/protein?name=UNC-15;class=protein" target="_blank">UNC-15</a> (<a href="#bib39">Maruyama et al., 1989</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1031"></a>13.&nbsp;Bypass suppression
                     </h2>
                  </div>
               </div>
            </div>
            <p>Suppression by increased dosage can be regarded as a special case of bypass
                           suppression, wherein an alternative protein or pathway is up-regulated or de-regulated
                           to remove the need for a defective component. Mutations that activate alternative
                           pathways are often encountered as a result of suppressor analysis. Screens for
                           suppressors of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pal-1" target="_blank">pal-1</a></em></span>, the <span class="emphasis"><em>C. elegans</em></span> homolog of the developmental regulator caudal,
                           illustrate this effect (<a href="#bib64">Zhang and Emmons, 2000</a>). In normal development,
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pal-1" target="_blank">pal-1</a></em></span> expression in certain tail cells depends on a positive regulatory element located
                           in one of its introns. A screen was carried out for suppressors of a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pal-1" target="_blank">pal-1</a></em></span> mutation
                           affecting this intronic element, which led to the identification of three <span class="emphasis"><em>sop</em></span>
                           (Suppressor Of Pal) genes. These encode factors that act as general repressors of an
                           alternative mode of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pal-1" target="_blank">pal-1</a></em></span> transcription. Two of the <span class="emphasis"><em>sop</em></span> genes encode components of the
                           transcriptional Mediator complex. Loss of these factors results in expression of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pal-1" target="_blank">pal-1</a></em></span>
                           in the tail cells, and hence to suppression by bypassing the need for the intronic
                           regulatory element.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1069"></a>14.&nbsp;Extragenic suppression by interacting components
                     </h2>
                  </div>
               </div>
            </div>
            <p>An abnormal protein can sometimes be restored to greater functionality by changes in
                           an interacting protein. Suppression by such a mechanism is usually allele-specific, and
                           provides <span class="emphasis"><em>in vivo</em></span> evidence for protein interaction. An example is provided by mutations
                           of the muscle protein gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a></em></span>, which cause a &#8216;twitcher&#8217; abnormal movement phenotype.
                           Some revertants of certain <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-22" target="_blank">unc-22</a></em></span> alleles were found to be unusual missense alleles of
                           the body wall myosin gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span>, suggesting that the <a href="http://www.wormbase.org/db/seq/protein?name=UNC-22;class=protein" target="_blank">UNC-22</a> and <a href="http://www.wormbase.org/db/seq/protein?name=UNC-54;class=protein" target="_blank">UNC-54</a> proteins
                           interact directly (<a href="#bib40">Moerman et al., 1982</a>). Both these proteins are,
                           indeed, located in the thick filaments of the myofilament lattice (<a href="http://www.wormbook.org/chapters/www_SarcomereAssembCelegansmusc/SarcomereAssembCelegansmusc.html" target="_blank">Sarcomere assembly in <span class="emphasis"><em>C. elegans</em></span> muscle</a>).
            </p>
            <p>Another example is provided by suppression of certain dominant mutations in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-105" target="_blank">unc-105</a></em></span>,
                           a gene encoding a putative mechanosensory channel in muscle cells, by a specific
                           missense alteration in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-2" target="_blank">let-2</a></em></span>, which encodes a basement membrane collagen (<a href="#bib35">Liu et al.,
                              1996</a>). A plausible interpretation is that <a href="http://www.wormbase.org/db/seq/protein?name=UNC-105;class=protein" target="_blank">UNC-105</a> and <a href="http://www.wormbase.org/db/seq/protein?name=LET-2;class=protein" target="_blank">LET-2</a> interact
                           directly, providing anchorage for the sensory channel and allowing it to act as a
                           stretch sensor.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1125"></a>15.&nbsp;Suppression by removal of a toxic protein or protein complex
                     </h2>
                  </div>
               </div>
            </div>
            <p>A situation sometimes encountered in <span class="emphasis"><em>C. elegans</em></span> genetics is the appearance of an
                           abnormal phenotype as the result of a neomorphic change in a gene, resulting in the
                           production of a toxic protein. Such mutations are usually but not always dominant
                           (sometimes they are recessive because in heterozygotes the abnormal protein is
                           outcompeted by normal protein). Suppression studies have been carried out with such
                           mutations in a variety of different genes. Any alteration that removes the toxic
                           protein, or prevents it from acting, will behave as a suppressor.
            </p>
            <p>The simplest situation involves the introduction of a second inactivating mutation in
                           <span class="emphasis"><em>cis</em></span> to the dominant allele. This is exemplified by dominant mutations of the myosin gene
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span>, as already described.
            </p>
            <p>A more complex situation was encountered with the gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a></em></span>, which has been
                           subjected to extensive suppression analysis. A rare mutation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1500;class=Variation" target="_blank">e1500</a></em></span>, causes a
                           distinctive movement and egg-laying defect referred to as the &#8216;rubber-band&#8217; phenotype.
                           This phenotype is almost completely recessive, although genetic tests indicate that
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1500;class=Variation" target="_blank">e1500</a></em></span> is neomorphic in character. Null mutations introduced in <span class="emphasis"><em>cis</em></span> to <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1500;class=Variation" target="_blank">e1500</a></em></span> result in
                           normally moving animals, when homozygous. Therefore, the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a></em></span> gene is dispensable, but
                           can be mutated to generate a product that is able to poison muscle contraction
                           (<a href="#bib12">Greenwald and Horvitz, 1980</a>). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1500;class=Variation" target="_blank">e1500</a></em></span> phenotype can also be suppressed
                           by null mutations in three unlinked genes, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-9" target="_blank">sup-9</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-10" target="_blank">sup-10</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-18" target="_blank">sup-18</a></em></span>, which act as
                           recessive suppressors, as well as by specific dominant mutations of another gene, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-11" target="_blank">sup-11</a></em></span>
                           (<a href="#bib13">Greenwald and Horvitz, 1982</a>). Moreover, &#8216;rubber-band&#8217; alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-9" target="_blank">sup-9</a></em></span>
                           and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-10" target="_blank">sup-10</a></em></span> were subsequently found, which can be suppressed by null alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a></em></span>
                           (<a href="#bib14">Greenwald and Horvitz, 1986</a>; <a href="#bib30">Levin and Horvitz, 1992</a>).
                           Molecular analysis indicates that these genes encode components of a muscle-specific
                           potassium channel: the <a href="http://www.wormbase.org/db/seq/protein?name=SUP-9;class=protein" target="_blank">SUP-9</a> sequence reveals that it belongs to the family of two-pore
                           K<sup>+</sup> channel proteins, and <a href="http://www.wormbase.org/db/seq/protein?name=SUP-10;class=protein" target="_blank">SUP-10</a> and <a href="http://www.wormbase.org/db/seq/protein?name=UNC-93;class=protein" target="_blank">UNC-93</a> are membrane proteins that co-localize with
                           <a href="http://www.wormbase.org/db/seq/protein?name=SUP-9;class=protein" target="_blank">SUP-9</a> in muscle cells (<a href="#bib31">Levin and Horvitz, 1993</a>; <a href="#bib7">de la Cruz et al., 2003</a>). The current interpretation is therefore that this multi-protein channel
                           complex is dispensable for normal muscle function, but it can be mutated so as to
                           interfere severely with contraction, presumably by affecting K<sup>+</sup> flux. The other two
                           interacting genes, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-11" target="_blank">sup-11</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-18" target="_blank">sup-18</a></em></span>, likely encode additional components, but <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-11" target="_blank">sup-11</a></em></span> is
                           known to be an essential gene, in contrast to the other four (because null mutations of
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-11" target="_blank">sup-11</a></em></span> are embryonic lethal), so it must be involved in other processes as well.
            </p>
            <p>A comparable situation was found in the case of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=deg-3" target="_blank">deg-3</a></em></span> gene, which encodes a
                           nicotinic acetylcholine receptor subunit. Rare dominant alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=deg-3" target="_blank">deg-3</a></em></span> cause neuronal
                           degeneration (hence the name), but the degeneration can be suppressed by
                           loss-of-function mutations of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=des-2" target="_blank">des-2</a></em></span> (DEgeneration Suppressor). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=des-2" target="_blank">des-2</a></em></span> gene was found
                           to encode another receptor subunit, and co-expression of wildtype <a href="http://www.wormbase.org/db/seq/protein?name=DEG-3;class=protein" target="_blank">DEG-3</a> and <a href="http://www.wormbase.org/db/seq/protein?name=DES-2;class=protein" target="_blank">DES-2</a> in
                         <span class="emphasis"><em>Xenopus</em></span> oocytes resulted in formation of a functional acetylcholine-gated channel.
                           Co-expression of mutant <a href="http://www.wormbase.org/db/seq/protein?name=DEG-3;class=protein" target="_blank">DEG-3</a> with wildtype <a href="http://www.wormbase.org/db/seq/protein?name=DES-2;class=protein" target="_blank">DES-2</a> resulted in an abnormal channel.
                           Expression of either subunit alone, however, does not allow formation of a channel,
                           thereby explaining the mechanism of suppression (<a href="#bib55">Treinin et al., 1998</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1298"></a>16.&nbsp;Suppression by modulators of activity
                     </h2>
                  </div>
               </div>
            </div>
            <p>The efficiency of a signal transduction pathway is usually modulated by interacting
                           components that can increase or decrease the strength of a signal. This is especially
                           true in multicellular organisms, where the same pathway may be used for multiple
                           purposes in different tissues, and can be fine-tuned by these modulators so as to allow
                           optimal function in each context. The presence of modulators is often revealed by
                           suppression analysis. Good illustrations of this type of suppression are provided by
                           extensive work on the <a href="http://www.wormbase.org/db/seq/protein?name=LIN-12;class=protein" target="_blank">LIN-12</a>/NOTCH receptor, which is involved in numerous important
                           signaling events during development (see <a href="http://www.wormbook.org/chapters/www_lin12Notch/lin12notch.html" target="_blank">LIN-12/Notch signaling in <span class="emphasis"><em>C. elegans</em></span></a>). Suppression experiments
                           have been carried out using both hypomorphic, partial loss-of-function alleles (<a href="#bib52">Sundaram
                              and Greenwald, 1993</a>) and hypermorphic, gain-of-function alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span>
                           (<a href="#bib32">Levitan and Greenwald, 1995</a>; <a href="#bib54">Tax et al., 1997</a>). The first
                           kind of experiment yielded mutations in negative regulators of <a href="http://www.wormbase.org/db/seq/protein?name=LIN-12;class=protein" target="_blank">LIN-12</a> activity, such as
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sel-1" target="_blank">sel-1</a></em></span>, encoding an extracellular protein (<a href="#bib11">Grant and Greenwald, 1996</a>), and
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sel-10" target="_blank">sel-10</a></em></span>, encoding an F-box protein predicted to be involved in ubiquitin-mediated
                           turnover of <a href="http://www.wormbase.org/db/seq/protein?name=LIN-12;class=protein" target="_blank">LIN-12</a> (<a href="#bib20">Hubbard et al., 1997</a>). <a href="http://www.wormbase.org/db/seq/protein?name=SEL-10;class=protein" target="_blank">SEL-10</a> was independently
                           identified as a factor affecting sex determination, again partly as a result of
                           suppressor analysis (<a href="#bib22">Jager et al., 2004</a>).
            </p>
            <p>The second kind of experiment, suppressing dominant gain-of-function alleles of
                           <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span>, generated mutations in positive regulators. These included <a href="http://www.wormbase.org/db/seq/protein?name=SUP-17;class=protein" target="_blank">SUP-17</a>, a protein
                           belonging to the ADAM protease family, which perhaps acts by cleaving and activating the
                           extracellular domain of the <a href="http://www.wormbase.org/db/seq/protein?name=LIN-12;class=protein" target="_blank">LIN-12</a> receptor (<a href="#bib59">Wen et al., 1997</a>). Also
                           identified were <a href="http://www.wormbase.org/db/seq/protein?name=SEL-5;class=protein" target="_blank">SEL-5</a>, a serine/threonine kinase (<a href="#bib8">Fares and Greenwald, 1999</a>) and <a href="http://www.wormbase.org/db/seq/protein?name=SEL-12;class=protein" target="_blank">SEL-12</a>, which is related to mammalian presenilin proteins and therefore
                           of great medical interest (<a href="#bib32">Levitan and Greenwald, 1995</a>). Not
                           surprisingly, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sel-12" target="_blank">sel-12</a></em></span> mutations have an egg-laying defect in a <span class="emphasis"><em>lin-12(+)</em></span> background; this
                           in turn has allowed further suppressor screens for mutations correcting this defect, and
                           the identification of <span class="emphasis"><em>spr</em></span> (Suppressor of Presenilin) genes (<a href="#bib60">Wen et al. 2000</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1400"></a>17.&nbsp;Suppression by epistasis
                     </h2>
                  </div>
               </div>
            </div>
            <p>Suppression analysis has been invaluable in unraveling regulatory pathways in <span class="emphasis"><em>C.
                                 elegans</em></span>. Many important events in development depend on signal transduction events,
                           whereby an inductive signal emanating from one cell leads to a transcriptional response
                           in target cells or tissues. The transduction of the signal involves a chain of
                           interactions, with receptors, second messengers and transcription factors, and the
                           interactions may be either positive or negative at each step. Mutations that decrease or
                           increase signaling at one step can be epistatically suppressed by antagonistic mutations
                           acting at a downstream step, which in turn can be subjected to further analysis. If the
                           pathway involves a linear series of interactions, then the epistatic relationships of
                           mutations in the different genes can be used to deduce the order and nature of the steps
                           involved. More detailed discussion of the rules and interpretation of epistasis can be
                           found elsewhere (<a href="#bib2">Avery and Wasserman 1992</a>).
            </p>
            <p>Two good examples are provided by the vulval induction pathway (see <a href="http://www.wormbook.org/chapters/www_RTKRasMAPKsignaling/RTKRasMAPKsignaling.html" target="_blank">RTKRas/MAP kinase signaling</a> and the sex determination pathway (see <a href="http://www.wormbook.org/chapters/www_somaticsexdeterm/somaticsexdeterm.html" target="_blank">Somatic sex determination</a> and <a href="http://www.wormbook.org/chapters/www_sexgermline/sexgermline.html" target="_blank">Sex-determination in the germ line</a>). In
                           the first case, opposite mutant phenotypes result from either too little or too much
                           activation of the ERK MAP kinase pathway in the vulval precursor cells. These phenotypes
                           are called Vul (vulvaless, failure to induce or complete vulval differentation) and Muv
                           (multivulva, excess vulval differentiation) respectively. Both kinds of phenotype are
                           very conspicuous in adult hermaphrodites, and both are associated with egg-laying
                           defects and reduced fertility, so suppressors are easily detected. Mutations that
                           suppress a Vul phenotype often lead to a Muv phenotype on their own, and vice versa. For
                           example, screens for suppressors of a loss-of-function Vul mutation in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span>, encoding
                           the EGF receptor required for vulval induction, yielded a gain-of-function in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a></em></span>,
                           encoding the downstream Ras protein in this signal transduction pathway. The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-60" target="_blank">let-60</a>(gf)</em></span>
                           mutation has a Muv phenotype on its own, and screens for suppressors of the Muv
                           phenotype led to the isolation of mutations in ten or more genes encoding further
                           downstream components or modifiers of the pathway, most of which have a Vul phenotype on
                           their own. This kind of analysis has led to the identification of thirty or more genes
                           involved in EGF signaling, and has played a major role in the very detailed elucidation
                           of its properties and function (see <a href="http://www.wormbook.org/chapters/www_RTKRasMAPKsignaling/RTKRasMAPKsignaling.html" target="_top">RTKRas/MAP kinase signaling</a>). For a summary of the
                           suppressor screens that have been carried out, see <a href="#bib51">Sternberg and Han (1998</a>).
            </p>
            <p>In the case of sex determination, the opposite phenotypes caused by mutation are
                           Masculinized and Feminized. It is frequently possible (for example by use of
                           temperature-sensitive mutants) to set up a situation in which a large population of
                           worms develops only along one sexual path. Such a pure male or pure female population
                           will yield no progeny, so there is extremely effective selection for any suppressor
                           mutation that reverts or ameliorates the phenotype, and thereby allows cross- or
                           self-fertilization, and the production of progeny. An extensive series of experiments of
                           this type has been carried out in a number of different laboratories, as summarized in
                           <a href="#figure1">Figure 1</a>. Both loss-of-function and gain-of-function mutations have been isolated at
                           each of four regulatory steps, largely as a result of this approach. For example,
                           selections for suppressors of the masculinizing mutations of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-3" target="_blank">tra-3</a></em></span> yielded feminizing
                           (loss-of-function) alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-1" target="_blank">fem-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-2" target="_blank">fem-2</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=fem-3" target="_blank">fem-3</a></em></span>, as well as gain-of-function
                           alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-1" target="_blank">tra-1</a></em></span> (<a href="#bib17">Hodgkin, 1986</a>). Amber alleles of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tra-3" target="_blank">tra-3</a></em></span> were used in
                           some of these selections, which consequently also yielded many nonsense suppressors and
                           NMD (smg) suppressors (<a href="#bib16">Hodgkin, 1985</a>; <a href="#bib17">Hodgkin, 1986</a>).
            </p>
            <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="495">
<tr>
  <td align="center"><p><a href="geneticsuppressionfig1.jpg"><img src="geneticsuppressionfig1_s.jpg" width="580" border="2" align="middle" alt="Suppressor mutations figure 1"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 1. Suppressor screens and selections for sex determination mutants.&nbsp;</b>Part of the signal transduction cascade controlling sex in <span class="emphasis"><em>C. elegans</em></span> is shown at
                                 left. In hermaphrodites, the feminizing transcription factor <a href="http://www.wormbase.org/db/seq/protein?name=TRA-1;class=protein" target="_blank">TRA-1</a> promotes female
                                 development, and the membrane receptor <a href="http://www.wormbase.org/db/seq/protein?name=TRA-2;class=protein" target="_blank">TRA-2</a> prevents inhibition of <a href="http://www.wormbase.org/db/seq/protein?name=TRA-1;class=protein" target="_blank">TRA-1</a> by the FEM
                                 proteins. In males, <a href="http://www.wormbase.org/db/seq/protein?name=TRA-1;class=protein" target="_blank">TRA-1</a> is prevented from acting by the FEM proteins, because <a href="http://www.wormbase.org/db/seq/protein?name=TRA-2;class=protein" target="_blank">TRA-2</a>
                                 action is blocked by binding the extracellular male-specific protein <a href="http://www.wormbase.org/db/seq/protein?name=HER-1;class=protein" target="_blank">HER-1</a>. At right are
                                 shown some of the suppressive interactions that led to this model. Red arrows indicate
                                 isolation of feminizing suppressors that counteract masculinization, and blue arrows
                                 indicate isolation of masculinizing suppressors that counteract feminization. For more
                                 detailed explanation, see text and chapters on sex determination.
                  </p>
               </div>
            </div>
            <p>Almost every known control pathway in <span class="emphasis"><em>C. elegans</em></span> has been subjected to some kind of
                           suppressor analysis, and the results have frequently been critical in elucidating
                           properties and testing models. Genetic suppression must also happen naturally, and is
                           certain to play an important role in evolution. Whether new kinds of suppression will
                           become apparent, as further genes and processes are subjected to investigation, remains
                       to be seen.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1521"></a>18.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                             <span class="bibliomisc">Anderson, P., and Brenner. S. (1984). A selection for myosin heavy-chain mutants in the nematode <span class="emphasis"><em>C. elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>81</em></span>, 4470&#8211;4474.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6589606&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                             <span class="bibliomisc">Avery, L., and Wasserman, S. (1992). Ordering gene function: the interpretation of epistasis in regulatory hierarchies. Trends
                        Genet. <span class="emphasis"><em>8</em></span>, 312&#8211;316.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1365397&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                             <span class="bibliomisc">Barbazuk, W.B., Johnsen, R.C., and Baillie, D.L. (1994). The generation and genetic analysis of suppressors of lethal mutations
                        in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em>rol-3(V)</em></span> gene. Genetics <span class="emphasis"><em>136</em></span>, 129&#8211;143.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8138151&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                             <span class="bibliomisc">Bolten, S.L., Powell-Abel, P., Fischhoff, D.A., and Waterston, R.H. (1984). The <span class="emphasis"><em>sup-7(st5)</em></span> X gene of <span class="emphasis"><em>Caenorhabditis elegans</em></span> encodes a tRNATrpUAG amber suppressor. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>81</em></span>, 6784&#8211;6788.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6093119&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                             <span class="bibliomisc">Brown, S.J., and Riddle, D.L. (1985). Gene interactions affecting muscle organization in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>110</em></span>, 421&#8211;440.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4018568&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                             <span class="bibliomisc">Colavita, A., and Culotti, J.G. (1998). Suppressors of ectopic <a href="http://www.wormbase.org/db/seq/protein?name=UNC-5;class=protein" target="_blank">UNC-5</a> growth cone steering identify eight genes involved in axon guidance in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>194</em></span>, 72&#8211;85.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9473333&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/dbio.1997.8790" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                             <span class="bibliomisc">de la Cruz, I.P., Levin, J.Z., Cummins, C., Anderson, P., and Horvitz, H.R. (2003). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-9" target="_blank">sup-9</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-10" target="_blank">sup-10</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a></em></span> may encode components of a two-pore K+ channel that coordinates muscle contraction in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Neurosci. <span class="emphasis"><em>23</em></span>, 9133&#8211;9145.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14534247&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fares, H., and Greenwald, I. (1999). <a href="http://www.wormbase.org/db/seq/protein?name=SEL-5;class=protein" target="_blank">SEL-5</a>, a serine/threonine kinase that facilitates <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> activity in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>153</em></span>, 1641&#8211;1654.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10581273&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fire, A. (1986). Integrative transformation of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. EMBO J. <span class="emphasis"><em>5</em></span>, 2673&#8211;2680.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                             <span class="bibliomisc">Fire A, Kondo K, Waterston R. (1990). Vectors for low copy transformation of <span class="emphasis"><em>C. elegans</em></span>. Nucleic Acids Res. <span class="emphasis"><em>18</em></span>, 4269&#8211;4270.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2377476&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                             <span class="bibliomisc">Grant, B., and Greenwald, I. (1996). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sel-1" target="_blank">sel-1</a></em></span> gene, a negative regulator of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span>, encodes a predicted extracellular protein. Genetics <span class="emphasis"><em>143</em></span>, 237&#8211;247.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8722778&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                             <span class="bibliomisc">Greenwald, I.S., and Horvitz, H.R. (1980). <span class="emphasis"><em>unc-93(<a href="http://www.wormbase.org/db/get?name=e1500;class=Variation" target="_blank">e1500</a>)</em></span>: a behavioral mutant of <span class="emphasis"><em>Caenorhabditis elegans</em></span> that defines a gene with a wild-type null phenotype. Genetics <span class="emphasis"><em>96</em></span>, 147&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6894129&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                             <span class="bibliomisc">Greenwald, I.S., and Horvitz, H.R. (1982). Dominant suppressors of a muscle mutant define an essential gene of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>101</em></span>, 211&#8211;225.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7173602&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                             <span class="bibliomisc">Greenwald, I.S., and Horvitz, H.R. (1986). A visible allele of the muscle gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-10" target="_blank">sup-10</a> X</em></span> of <span class="emphasis"><em>C. elegans</em></span>. Genetics <span class="emphasis"><em>113</em></span>, 63&#8211;72.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3710144&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hajnal, A., Whitfield, C.W., and Kim, S.K. (1997). Inhibition of <span class="emphasis"><em>Caenorhabditis elegans </em></span>vulval induction by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=gap-1" target="_blank">gap-1</a></em></span> and by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> receptor tyrosine kinase. Genes Dev. <span class="emphasis"><em>11</em></span>, 2715&#8211;2728.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9334333&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hodgkin, J. (1985). Novel nematode amber suppressors. Genetics <span class="emphasis"><em>111</em></span>, 287&#8211;310.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9447966&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hodgkin, J. (1986). Sex determination in the nematode <span class="emphasis"><em>C. elegans</em></span>: analysis of suppressors and characterization of <span class="emphasis"><em>fem</em></span> genes. Genetics <span class="emphasis"><em>114</em></span>, 15&#8211;52.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3770465&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hodgkin, J., Papp, A., Pulak, R., Ambros, V., and Anderson, P. (1989). A new kind of informational suppression in the nematode
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>123</em></span>, 301&#8211;313.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2583479&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                             <span class="bibliomisc">Howard, R.M., and Sundaram, M.V. (2002). <span class="emphasis"><em>C. elegans</em></span> <a href="http://www.wormbase.org/db/seq/protein?name=EOR-1;class=protein" target="_blank">EOR-1</a>/PLZF and <a href="http://www.wormbase.org/db/seq/protein?name=EOR-2;class=protein" target="_blank">EOR-2</a> positively regulate Ras and Wnt signaling and function redundantly with <a href="http://www.wormbase.org/db/seq/protein?name=LIN-25;class=protein" target="_blank">LIN-25</a> and the <a href="http://www.wormbase.org/db/seq/protein?name=SUR-2;class=protein" target="_blank">SUR-2</a> mediator component. Genes Dev. <span class="emphasis"><em>16</em></span>, 1815&#8211;1827.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12130541&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.998402" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                             <span class="bibliomisc">Hubbard, E.J., Wu, G., Kitajewski, J., and Greenwald, I. (1997). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sel-10" target="_blank">sel-10</a></em></span>, a negative regulator of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> activity in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, encodes a member of the CDC4 family of proteins. Genes Dev. <span class="emphasis"><em>11</em></span>, 3182&#8211;3193.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9389650&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                             <span class="bibliomisc">Inoue, T., and Thomas, J.H. (2000). Suppressors of transforming growth factor-<span class="symbol">&#946;</span> pathway mutants in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> dauer formation pathway. Genetics <span class="emphasis"><em>156</em></span>, 1035&#8211;1046.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11063683&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                             <span class="bibliomisc">Jager, S., Schwartz, H.T., Horvitz, H.R., and Conradt, B. (2004). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> F-box protein <a href="http://www.wormbase.org/db/seq/protein?name=SEL-10;class=protein" target="_blank">SEL-10</a> promotes female development and may target <a href="http://www.wormbase.org/db/seq/protein?name=FEM-1;class=protein" target="_blank">FEM-1</a> and <a href="http://www.wormbase.org/db/seq/protein?name=FEM-3;class=protein" target="_blank">FEM-3</a> for degradation by the proteasome. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 12549&#8211;12554.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15306688&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0405087101" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                             <span class="bibliomisc">Jongeward, G.D., Clandinin, T.R, and Sternberg, P.W. (1995). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sli-1" target="_blank">sli-1</a></em></span>, a negative regulator of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span>-mediated signaling in <span class="emphasis"><em>C. elegans</em></span>. Genetics <span class="emphasis"><em>139</em></span>, 1553&#8211;1566.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7789760&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kimble, J., Hodgkin, J., Smith, T., Smith, J. (1982). Suppression of an amber mutation by microinjection of suppressor tRNA
                        in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>299</em></span>, 456&#8211;458.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7121584&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/299456a0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kondo, K., Hodgkin, J., and Waterston, R.H. (1988). Differential expression of five tRNA(UAGTrp) amber suppressors in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Cell Biol. <span class="emphasis"><em>8</em></span>, 3627&#8211;3635.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3221861&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kondo, K., Makovec, B., Waterston, R.H., and Hodgkin, J. (1990). Genetic and molecular analysis of eight tRNA(Trp) amber suppressors
                        in Caenorhabditis elegans. J. Mol. Biol. <span class="emphasis"><em>215</em></span>, 7&#8211;19.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2398498&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                             <span class="bibliomisc">Kornfeld, K., Hom, D.B., and Horvitz, H.R. (1995). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ksr-1" target="_blank">ksr-1</a></em></span> gene encodes a novel protein kinase involved in Ras-mediated signaling in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>83</em></span>, 9039&#8211;9013.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8521514&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(95)90206-6" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                             <span class="bibliomisc">Korswagen, H.C., van der Linden, A.M., and Plasterk, R.H. (1998). G protein hyperactivation of the <span class="emphasis"><em>Caenorhabditis elegans</em></span> adenylyl cyclase SGS-1 induces neuronal degeneration. EMBO J. <span class="emphasis"><em>17</em></span>, 5059&#8211;5065.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9724641&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/emboj/17.17.5059" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lackner, M.R., Kornfeld, K., Miller, L.M., Horvitz, H.R., and Kim, S.K. (1994). A MAP kinase homolog, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mpk-1" target="_blank">mpk-1</a></em></span>, is involved in ras-mediated induction of vulval cell fates in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genes Dev. <span class="emphasis"><em>8</em></span>, 160&#8211;173.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8299936&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                             <span class="bibliomisc">Levin, J.Z., and Horvitz, H.R. (1992). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-93" target="_blank">unc-93</a></em></span> gene encodes a putative transmembrane protein that regulates muscle contraction. J. Cell Biol. <span class="emphasis"><em>117</em></span>, 143&#8211;155.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1313436&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.117.1.143" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                             <span class="bibliomisc">Levin, J.Z., and Horvitz, H.R. (1993). Three new classes of mutations in the <span class="emphasis"><em>Caenorhabditis elegans</em></span> muscle gene <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-9" target="_blank">sup-9</a></em></span>. Genetics <span class="emphasis"><em>135</em></span>, 53&#8211;70.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8224828&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                             <span class="bibliomisc">Levitan, D., and Greenwald, I. (1995). Facilitation of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span>-mediated signalling by <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sel-12" target="_blank">sel-12</a></em></span>, a <span class="emphasis"><em>Caenorhabditis elegans</em></span> S182 Alzheimer's disease gene. Nature <span class="emphasis"><em>377</em></span>, 351&#8211;354.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7566091&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/377351a0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                             <span class="bibliomisc">Li, L., Linning, R.M., Kondo, K., and Honda, B.M. (1998). Differential expression of individual suppressor tRNA(Trp) gene
                        family members <span class="emphasis"><em>in vitro</em></span> and <span class="emphasis"><em>in vivo</em></span> in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Cell Biol. <span class="emphasis"><em>18</em></span>, 703&#8211;709.</span>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lissemore, J.L., Currie, P.D., Turk, C.M., and Maine, E.M. (1993). Intragenic dominant suppressors of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span>, a gene essential for cell-signaling in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, support a role for cdc10/SW16/Ankyrin motifs in <a href="http://www.wormbase.org/db/seq/protein?name=GLP-1;class=protein" target="_blank">GLP-1</a> function. Genetics <span class="emphasis"><em>135</em></span>, 1023&#8211;1034.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8307320&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                             <span class="bibliomisc">Liu, J., Schrank, B., and Waterston, R.H. (1996). Interaction between a putative mechanosensory membrane channel and a collagen.
                        Science <span class="emphasis"><em>273</em></span>, 361&#8211;364.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8662524&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                             <span class="bibliomisc">Lundquist, E.A., and Herman, R.K. (1994). The <a href="http://www.wormbase.org/db/gene/gene?name=mec-8" target="_blank">mec-8</a> gene of <span class="emphasis"><em>Caenorhabditis elegans</em></span> affects muscle and sensory neuron function and interacts with three other genes: <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smu-1" target="_blank">smu-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=smu-2" target="_blank">smu-2</a></em></span>. Genetics <span class="emphasis"><em>138</em></span>, 83&#8211;101.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8001796&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                             <span class="bibliomisc">Maine, E.M., and Kimble, J.E. (1989). Identification of genes that interact with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span>, a gene required for inductive cell interactions in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>106</em></span>, 133&#8211;143.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2627884&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                             <span class="bibliomisc">Maine, E.M., and Kimble, J.E. (1993). Suppressors of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span>, a gene required for cell communication during development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, define a set of interacting genes. Genetics <span class="emphasis"><em>135</em></span>, 1011&#8211;1122.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8307319&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                             <span class="bibliomisc">Maruyama, I.N., Miller, D.M., and Brenner, S. (1989). Myosin heavy chain gene amplification as a suppressor mutation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Gen. Genet. <span class="emphasis"><em>219</em></span>, 113&#8211;118.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2575705&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00261165" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                             <span class="bibliomisc">Moerman, D.G., Plurad, S., Waterston, R.H., and Baillie, D.L. (1982). Mutations in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-54" target="_blank">unc-54</a></em></span> myosin heavy chain gene of <span class="emphasis"><em>Caenorhabditis elegans</em></span> that alter contractility but not muscle structure. Cell <span class="emphasis"><em>29</em></span>, 773&#8211;781.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7151169&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(82)90439-1" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                             <span class="bibliomisc">Nishiwaki, K., and Miwa, J. (1998). Mutations in genes encoding extracellular matrix proteins suppress the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=emb-5" target="_blank">emb-5</a></em></span> gastrulation defect in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Gen. Genet. <span class="emphasis"><em>1259</em></span>, 2&#8211;12.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9738874&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                             <span class="bibliomisc">Novelli, J., Ahmed, S., and Hodgkin, J. (2004). Gene interactions in <span class="emphasis"><em>C. elegans</em></span> define <a href="http://www.wormbase.org/db/seq/protein?name=DPY-31;class=protein" target="_blank">DPY-31</a> as a candidate procollagen C-proteinase and <a href="http://www.wormbase.org/db/seq/protein?name=SQT-3;class=protein" target="_blank">SQT-3</a>/ROL-4 as its predicted major target. Genetics <span class="emphasis"><em>168</em></span>, 1259&#8211;1273.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15579684&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.027953" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                             <span class="bibliomisc">Park, E.C., and Horvitz, H.R. (1986). Mutations with dominant effects on the behavior and morphology of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>113</em></span>, 821&#8211;852.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3744028&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                             <span class="bibliomisc">Pilgrim, D.B., and Bell, J.B. (1993). Expression of a <span class="emphasis"><em>Drosophila melanogaster</em></span> amber suppressor tRNA(Ser) in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Mol. Gen. Genet. <span class="emphasis"><em>241</em></span>, 26&#8211;32.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8232208&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1007/BF00280197" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                             <span class="bibliomisc">Riddle, D.L., and Brenner, S. (1978). Indirect suppression in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>89</em></span>, 299&#8211;314.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=669254&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                             <span class="bibliomisc">Rogalski, T.M., Gilchrist, E.J., Mullen, G.P., and Moerman, D.G. (1995). Mutations in the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-52" target="_blank">unc-52</a></em></span> gene responsible for body wall muscle defects in adult <span class="emphasis"><em>Caenorhabditis elegans</em></span> are located in alternatively spliced exons. Genetics <span class="emphasis"><em>139</em></span>, 159&#8211;169.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7535716&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                             <span class="bibliomisc">Roller, A.B., Hoffman, D.C., and Zahler, A.M. (2000). The allele-specific suppressor <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-39" target="_blank">sup-39</a></em></span> alters use of cryptic splice sites in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>154</em></span>, 1169&#8211;1179.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10757761&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                             <span class="bibliomisc">Run, J.Q., Steven, R., Hung, M.S., van Weeghel, R., Culotti. J.G., and Way J.C. (1996). Suppressors of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-73" target="_blank">unc-73</a></em></span> gene of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>143</em></span>, 225&#8211;236.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8722777&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed">
                             <span class="bibliomisc">Selfors, L.M., Schutzman, J.L., Borland, C.Z., and Stern, M.J. (1998). <a href="http://www.wormbase.org/db/gene/gene?name=soc-2" target="_blank">soc-2</a> encodes a leucine-rich repeat protein implicated in fibroblast growth factor receptor signaling. Proc. Natl. Acad. Sci. USA
                        <span class="emphasis"><em>95</em></span>, 6903&#8211;6908.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9618511&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.95.12.6903" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed">
                             <span class="bibliomisc">Spartz, A.K., Herman, R.K., and Shaw, J.E. (2004). <a href="http://www.wormbase.org/db/seq/protein?name=SMU-2;class=protein" target="_blank">SMU-2</a> and <a href="http://www.wormbase.org/db/seq/protein?name=SMU-1;class=protein" target="_blank">SMU-1</a>, <span class="emphasis"><em>Caenorhabditis elegans</em></span> homologs of mammalian spliceosome-associated proteins RED and fSAP57, work together to affect splice site choice. Mol. Cell
                        Biol. <span class="emphasis"><em>24</em></span>, 6811&#8211;6823.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15254247&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.24.15.6811-6823.2004" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sternberg, P.W., and Han, M. (1998). Genetics of RAS signaling in <span class="emphasis"><em>C. elegans</em></span>. Trends Genet. <span class="emphasis"><em>14</em></span>, 466&#8211;472.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9825675&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0168-9525(98)01592-3" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sundaram, M., and Greenwald, I. (1993). Suppressors of a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> hypomorph define genes that interact with both<span class="emphasis"><em> <a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glp-1" target="_blank">glp-1</a></em></span> in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>135</em></span>, 765&#8211;783.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8293978&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed">
                             <span class="bibliomisc">Sundaram, M., and Han, M. (1995). The <span class="emphasis"><em>C. elegans</em></span> <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=ksr-1" target="_blank">ksr-1</a></em></span> gene encodes a novel Raf-related kinase involved in Ras-mediated signal transduction. Cell <span class="emphasis"><em>83</em></span>, 889&#8211;901.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8521513&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(95)90205-8" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed">
                             <span class="bibliomisc">Tax, F.E., Thomas, J.H., Ferguson, E.L., and Horvitz, H.R. (1997). Identification and characterization of genes that interact
                        with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-12" target="_blank">lin-12</a></em></span> in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>147</em></span>, 1675&#8211;1695.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9409830&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed">
                             <span class="bibliomisc">Treinin, M., Gillo, B., Liebman, L., and Chalfie, M. (1998). Two functionally dependent acetylcholine subunits are encoded
                        in a single <span class="emphasis"><em>Caenorhabditis elegans</em></span> operon. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>95</em></span>, 15492&#8211;15495.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9860996&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.95.26.15492" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed">
                             <span class="bibliomisc">Wang, X., Yang, C., Chai, J., Shi, Y., and Xue, D. (2002). Mechanisms of AIF-mediated apoptotic DNA degradation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Science <span class="emphasis"><em>298</em></span>, 1587&#8211;1592.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12446902&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1076194" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed">
                             <span class="bibliomisc">Waterston, R.H. (1981). A second informational suppressor, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-7" target="_blank">sup-7</a> X</em></span>, in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>97</em></span>, 307&#8211;325.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7274656&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed">
                             <span class="bibliomisc">Waterston, R.H., and Brenner, S. (1978). A suppressor mutation in the nematode acting on specific alleles of many genes. Nature
                        <span class="emphasis"><em>275</em></span>, 715&#8211;719.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=703836&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/275715a0" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed">
                             <span class="bibliomisc">Wen, C., Metzstein, M.M., and Greenwald, I. (1997). <a href="http://www.wormbase.org/db/seq/protein?name=SUP-17;class=protein" target="_blank">SUP-17</a>, a <span class="emphasis"><em>Caenorhabditis elegans</em></span> ADAM protein related to Drosophila KUZBANIAN, and its role in <a href="http://www.wormbase.org/db/seq/protein?name=LIN-12;class=protein" target="_blank">LIN-12</a>/NOTCH signalling. Development <span class="emphasis"><em>124</em></span>, 4759&#8211;4767.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9428412&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed">
                             <span class="bibliomisc">Wen, C., Levitan, D., Li, X., and Greenwald, I. (2000). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=spr-2" target="_blank">spr-2</a></em></span>, a suppressor of the egg-laying defect caused by loss of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sel-12" target="_blank">sel-12</a></em></span> presenilin in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, is a member of the SET protein subfamily. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>97</em></span>, 14524&#8211;14529.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11114162&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.011446498" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed">
                             <span class="bibliomisc">Wills, N., Gesteland, R.F., Karn, J., Barnett, L., Bolten, S., and Waterston, R.H. (1983). The genes <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-7" target="_blank">sup-7</a> X</em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=sup-5" target="_blank">sup-5</a> III</em></span> of <span class="emphasis"><em>C. elegans</em></span> suppress amber nonsense mutations <span class="emphasis"><em>via</em></span> altered transfer RNA. Cell <span class="emphasis"><em>33</em></span>, 575&#8211;583.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=6571695&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(83)90438-5" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed">
                             <span class="bibliomisc">Wu, Y., and Han, M. (1994). Suppression of activated Let-60 ras protein defines a role of <span class="emphasis"><em>Caenorhabditis elegans</em></span> Sur-1 MAP kinase in vulval differentiation. Genes Dev. <span class="emphasis"><em>8</em></span>, 147&#8211;159.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8299935&amp;dopt=Abstract" target="_blank">Abstract</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed">
                             <span class="bibliomisc">Zahler, A.M., Tuttle, J.D., Chisholm, A.D. (2004, August). Genetic suppression of intronic +1G mutations by compensatory U1
                        snRNA changes in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics. <span class="emphasis"><em>167</em></span>(4), 1689&#8211;1696.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15342508&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1534/genetics.104.028746" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed">
                             <span class="bibliomisc">Zhang, H., and Emmons, S.W. (2000). A <span class="emphasis"><em>C. elegans</em></span> mediator protein confers regulatory selectivity on lineage-specific expression of a transcription factor gene. Genes Dev.
                        <span class="emphasis"><em>14</em></span>, 2161&#8211;2172. (PMID: 10970880)</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10970880&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.814700" target="_blank">Article</a>
                         
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed">
                             <span class="bibliomisc">Zheng, Y., Mellem, J.E., Brockie, P.J., Madsen, D.M., and Maricq, A.V. (2004). <a href="http://www.wormbase.org/db/seq/protein?name=SOL-1;class=protein" target="_blank">SOL-1</a> is a CUB-domain protein required for <a href="http://www.wormbase.org/db/seq/protein?name=GLR-1;class=protein" target="_blank">GLR-1</a> glutamate receptor function in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>427</em></span>, 451&#8211;457. (PMID: 14749834)</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14749834&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02244" target="_blank">Article</a>
                     	
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Philip Anderson. Last revised March 29, 2005. Published December 27, 2005. This chapter should be cited as: Hodgkin,
                  J. Genetic suppression (December 27, 2005), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.59.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2005 Jonathan Hodgkin. This is an open-access article distributed under the terms of the Creative Commons Attribution License,
                  which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are
                  credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e35" href="#d0e35">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:jah@bioch.ox.ac.uk" target="_top">jah@bioch.ox.ac.uk</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All journal content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

