<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Germline chromatin
         
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#germlinechromatin_ArticleID" title="Germline chromatin&#xA;">
      <link rel="next" href="#d0e75" title="1.&nbsp;General considerations">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
<div class="html_to_pdf_link">
	<a href="germlinechromatin.pdf">Download PDF version</a>
</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="germlinechromatin_ArticleID"></a>Germline chromatin<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                     
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Christine E. Schaner, William G. Kelly<em><span class="remark">
                                 <sup><a name="d0e34" href="#ftn.d0e34">&sect;</a></sup>
                                 </span></em>
                           </span><br></span><span class="affiliation"><span class="address">Biology Department, Emory University, Atlanta, GA 30322 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_germline.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > The germ line
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=259.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Germline+chromatin" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#d0e75">1. General considerations</a></span></dt>
               <dt><span class="sect1"><a href="#d0e94">2. Histone modifications and chromatin structure</a></span></dt>
               <dt><span class="sect1"><a href="#d0e160">3. Embryonic germline chromatin</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e163">3.1. The P Lineage</a></span></dt>
                     <dt><span class="sect2"><a href="#d0e311">3.2. 
                              Z2/Z3: The primordial germ cells</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e706">4. Post-embryonic germline chromatin</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e722">4.1. X chromosome-specific chromatin regulation</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e1408">5. Completing the cycle: the zygote</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#d0e1413">5.1. X chromatin imprinting</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#d0e1447">6. Special circumstances: germline silencing of transgenes</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1526">7. Germline goals include genome protection </a></span></dt>
               <dt><span class="sect1"><a href="#d0e1576">8. Future directions</a></span></dt>
               <dt><span class="sect1"><a href="#d0e1584">9. References</a></span></dt>
            </dl>
         </div>
         <div class="abstract">
            <p class="title">
               <span class="bold"><strong>Abstract</strong></span>
               
            </p>
            <p>
               <span class="bold"><strong>The DNA in eukaryotes is wrapped around a histone octamer core, together comprising the main subunit of chromatin, the nucleosome.
                     Modifications of the nucleosomal histones in the genome correlate with the ability or inability of chromatin to form higher
                     order structures, that in turn influence gene activity. The genome in primordial germ cells in early</strong></span>
               <span class="emphasis"><em>
                     <span class="bold"><strong>C. elegans</strong></span>
                     </em></span>
               <span class="bold"><strong>germ cells carries a unique pattern of histone modifications that correlate with transcriptional repression in these cells,
                     and aspects of this chromatin regulation are conserved in</strong></span>
               <span class="emphasis"><em>
                     <span class="bold"><strong>Drosophila.</strong></span>
                     </em></span>
               <span class="bold"><strong>Loss of repression causes sterility in the adults, suggesting that chromatin-based repression is essential for germ line maintenance.
                     The post-embryonic germ line also exhibits unique and dynamic aspects of chromatin regulation, with chromosome-wide regulation
                     particularly evident on the X chromosome. Several properties of X-specific chromatin assembly are also sex-specific. These
                     properties appear to be responding to the meiotic pairing status of the X chromosome, rather than the sex of the germ cells.
                     Finally, gamete-specific chromatin regulation during gametogenesis impacts on X chromatin assembly in the offspring, leading
                     to an apparent sperm-imprinted X inactivation in the early embryo. Other potential roles for germline-specific modes of chromatin
                     assembly in genome regulation and protection are discussed.</strong></span>
               
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e75"></a>1.&nbsp;General considerations
                     </h2>
                  </div>
               </div>
            </div>
            <p>The germ line in <span class="emphasis"><em>C. elegans</em></span>, as in any organism, must be established in early embryogenesis and then maintained throughout development and in the adult
               gonad. Key processes for establishment and maintenance include activation and repression of the proper sets of genes, and
               the protection of genetic information from heritable mutation caused by mobile genetic elements. One can therefore consider
               these as two overlapping teleological goals for guardianship of the germ line: the maintenance of germ line character and
               the maintenance of genome integrity.
            </p>
            <p>In this chapter we will discuss the different phases of transcriptional regulation that have been observed in the <span class="emphasis"><em>C. elegans</em></span> germ line. Transcriptional regulation in embryonic germ cells is somewhat unique, because global "protective" repression
               of genes involved in somatic differentiation coexists with the activities of genes needed for germ cell survival. In <span class="emphasis"><em>C. elegans</em></span>, part of this is achieved by a strong reliance on maternally provided proteins and transcripts' maternal supplies of essential
               factors can initially replace a requirement for zygotic transcriptional activity. A genome-wide transcriptional repression
               that is mediated by the maternal protein <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> is thus made feasible in the early embryonic germ line by sufficient maternal support. Subsequent to PIE-1's degradation,
               another mode of embryonic transcriptional repression follows that involves regulated changes of chromatin structure. The post-embryonic
               germ line, unlike the primordial germ cells (PGCs) in the embryo, is both proliferative and highly transcriptionally engaged.
               Global repression is therefore not a viable strategy, and the processes are necessarily more refined. We will discuss the
               contributions of regulated chromatin assembly in maintaining and protecting the post-embryonic germ line.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e94"></a>2.&nbsp;Histone modifications and chromatin structure
                     </h2>
                  </div>
               </div>
            </div>
            <p>
               <span class="emphasis"><em>C. elegans</em></span> lacks cytosine methylation and thus relies mostly on modifications of histone proteins to dictate chromatin structure (<a href="#bib31">Simpson et al., 1986</a>
               ). In eukaryotes, two each of the four core histone proteins, H2A, H2B, H3 and H4, form an octamer, around which DNA is wrapped
               to form a nucleosome, the fundamental unit of chromatin. Varieties of a fourth histone protein, H1, associate with the "linker"
               DNA between each nucleosome. Each histone protein can be post-translationally modified in various ways, including methylation,
               acetylation, phosphorylation, ubiquitylation, and ADP-ribosylation. The combination of these modifications on the various
               histone proteins correlates with different biological functions (<a href="#bib23">Peterson and Laniel, 2004</a>
               ). For example, acetylation of lysine residues on histones H3 and H4 correlates with transcriptionally competent euchromatin
               (<a href="#bib7">Grunstein, 1997</a>
               ). Methylation of lysines 4 and 36 on histone H3 (H3methK4, H3methK36) are also enriched in transcriptionally competent euchromatin,
               and these modifications are especially abundant during transcriptional initiation and elongation (<a href="#bib8">Hampsey and Reinberg, 2003</a>
               ; <a href="#bib15">Kouzarides, 2002</a>
               ). In budding yeast the methyltransferases responsible for these modifications, Set1 and Set2 respectively, are recruited
               to sites of transcriptional elongation by the different phospho-epitopes of RNA polymerase II (RNAPII; <a href="#bib16">Krogan et al., 2003</a>
               ; <a href="#bib22">Ng et al., 2003</a>
               ; see also <a href="http://www.wormbook.org/chapters/www_specgermline/specgermline.html" target="_top">Specification of the germ line</a>). In contrast, methylation of lysines 9 and 27 on histone H3 (H3methK9, H3methK27) correspond to a transcriptionally repressed
               chromatin state (<a href="#bib17">Lachner and Jenuwein, 2002</a>
               ; <a href="#bib21">Muller et al., 2002</a>
               ). In general, while H3K9 hypermethylation is a conserved component of constitutive heterochromatin (e.g., pericentric chromatin
               in mammals), both H3K9 and H3K27 methylation can have variable representation in facultative heterochromatin (e.g., the inactivated
               X chromosome in female mammals) and in individually silenced loci (<a href="#bib32">Sims et al., 2003</a>
               ).
            </p>
            <p>
               <span class="emphasis"><em>Summary: Modifications of nucleosomal histones affect the function (e.g., transcriptional activity) of genomic regions. In
                     general, histone acetylation reduces histone interactions with DNA, and hence can increase accessibility to transcription
                     machinery. Histone methylation can correlate with silenced DNA (H3methK9, H3methK27), or with active DNA (H3methK4, H3methK36).</em></span>
               
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e160"></a>3.&nbsp;Embryonic germline chromatin
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e163"></a>3.1.&nbsp;The P Lineage
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In early <span class="emphasis"><em>C. elegans</em></span> embryos, the germ lineage (P lineage) is transcriptionally repressed by <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a>, a CCCH zinc finger protein. <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> is actively and asymmetrically segregated during early sequential cell divisions into the posterior blastomeres that comprise
                  the germ (P) lineage (<a href="#bib20">Mello et al., 1996</a>
                  ; <a href="#bib28">Seydoux et al., 1996</a>
                  ; see <a href="http://www.wormbook.org/chapters/www_specgermline/specgermline.html" target="_top">Specification of the germ line</a>). This transcriptional repression is necessary for germ cell identity, since the P<sub>2</sub> blastomere and its daughters are transformed into somatic blastomeres by loss of <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> activity (<a href="#bib19">Mello et al., 1992</a>
                  ; <a href="#bib28">Seydoux et al., 1996</a>
                  ). <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> is present until the division of P<sub>4</sub> into the primordial germ cells, <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a> and <a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a>, at approximately the 100-cell stage (<a href="#bib20">Mello et al., 1996</a>
                  ). In all stages leading up to this division, histone modifications corresponding to transcriptionally competent chromatin
                  are found in germline blastomeres at levels similar to their somatic neighbors (<a href="#bib26">Schaner et al., 2003</a>
                  ; <a href="#figure1">Figure 1</a>A-D). This indicates that <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> is not repressing transcription through a repressive chromatin architecture, and potentially supports the model proposed
                  by Zhang, et al. that <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> does not prevent binding of the transcription complex to promoters (<a href="#bib38">Zhang et al., 2003</a>
                  ). Also supporting this model is the observation that a phospho-epitope on the C-terminal domain repeat (CTD) sequence of
                  RNAP II that corresponds to transcriptional initiation, phospho-Ser5, is present in all cells of the early embryo, whereas
                  a phospho-epitope corresponding to transcriptional elongation, phospho-Ser2, is found only in the somatic blastomeres (<a href="#bib27">Seydoux and Dunn, 1997</a>
                  ). It therefore appears that chromatin in the early germ lineage is competent for transcriptional initiation, but <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> represses transcription by preventing transcriptional elongation. The <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> mode of germline repression seems limited to nematodes, since no obvious orthologs of <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> have been identified in other organisms.
               </p>
               <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="432">
<tr>
  <td align="center"><p><a href="germlinechromatinfig1.jpg"><img src="germlinechromatinfig1_s.jpg" border="2" align="middle" alt=" figure 1"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 1. A histone modification corresponding to a transcriptionally competent
                                       chromatin structure is present in all cells of the early embryo, but absent from <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a> and
                                       <a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a>.&nbsp;</b>Fixed, whole-mount embryos were stained with anti-H3methK4 (green), anti-PGL-1 (blue;
                                    marking the germ lineage), and counter-stained with DAPI (red). (A) H3dimethK4 is
                                    equally present in both <a href="http://www.wormbase.org/db/get?name=AB;class=Cell" target="_blank">AB</a> and P<sub>1</sub> (arrow) of 2-cell embryos, and
                                    persists in the germline blastomeres through (B) P<sub>2</sub>, (C)
                                        P<sub>3</sub>, and (D) P<sub>4</sub>. However, a specific
                                    chromatin reorganization occurs in the primordial germ cells <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a> and <a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a> (enlarged in
                                    insets), which are now the only cells to lose both H3methK4 (E) and H4acetylK8 (F). Bar
                                        <span class="symbol">=</span> 5 <span class="symbol">&#956;</span>m. (Reprinted from <a href="#bib26">Schaner et al. (2003)</a>, Copyright (2003), with permission from Elsevier.)
                     </p>
                  </div>
               </div>
               <p>
                  <span class="emphasis"><em>Summary: The early embryonic germline blastomeres that contribute to both germline and somatic lineages are globally quiescent
                        for transcription. This quiescence requires portioning of the maternal factor <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> into the germ line. The chromatin architecture in both somatic and germline cells, at these early stages, seems identical.</em></span>
                  
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e311"></a>3.2.&nbsp;
                           <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a>: The primordial germ cells
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The division of P<sub>4</sub> is the first symmetric division of the embryonic germ line, yielding the equipotent primordial germ cells, <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a> (PGCs). Their birth completes the separation of germ line and somatic lineages, and is accompanied by several dramatic changes
                  in germline regulation, including the loss of <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> and a specific change in chromatin structure.
               </p>
               <p>Coincident with <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> degradation, RNAPII phospho-Ser2 appears in <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a> (<a href="#bib27">Seydoux and Dunn, 1997</a>
                  ). Despite the appearance of this epitope, however, only a few mRNAs are known to be produced in these cells, and these are
                  detected at a later time in development (e.g., <span class="emphasis"><em>
                        <a href="http://www.wormbase.org/db/gene/gene?name=pgl-1" target="_blank">pgl-1</a>
                        </em></span> and <span class="emphasis"><em>
                        <a href="http://www.wormbase.org/db/gene/gene?name=nos-2" target="_blank">nos-2</a>
                        </em></span> mRNAs; <a href="#bib11">Kawasaki et al., 1998</a>
                  ; <a href="#bib33">Subramaniam and Seydoux, 1999</a>
                  ). These cells are mitotically quiescent until the embryo hatches, which may indicate continued transcriptional and/or post-transcriptional
                  repression, but an extensive analysis of this has not been done.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e372"></a>3.2.1.&nbsp;PGC-specific chromatin remodeling
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The DNA in <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a> is also noticeably more condensed than in surrounding somatic nuclei (<a href="#bib26">Schaner et al., 2003</a>
                     ). In addition, there is a change in three specific histone modifications, as detected by specific antibodies (<a href="#table1" title="Table 1. Histone modifications in the early germ line">Table 1</a>). Histone H3 di- and tri-methylated on lysine 4 and histone H4 acetylated on lysine 8 (H3methK4 and H4acetylK8, respectively)
                     are all globally reduced in the chromatin of the PGCs upon their birth (<a href="#figure1">Figure 1</a>E, F). In contrast, trimethylation of H3 lysine 27 (H3trimeK27) remains relatively unchanged, and in fact may actually increase
                     slightly (<a href="#bib2">Bender et al., 2004</a>
                     ; <a href="#bib26">Schaner et al., 2003</a>
                     ). The stabilization and/or slight increase in H3trimethK27 are thought to be due in part to conversion of H3dimethK27 to
                     the tri-methylated form in <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a>. Since H3methK4 and H4acetylK8 both correlate with transcriptional competence, and H3trimethK27 with transcriptional repression,
                     this suggests that transcriptional repression remains engaged in these cells, but has switched to a chromatin-based mechanism.
                     This may also indicate at least a partial switch from maternal control of germline repression to zygotic control, although
                     the mechanisms are not well defined for the chromatin mode. It is intriguing to note that H3dimethK27 appears specifically
                     in <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a> in the absence of maternal <a href="http://www.wormbase.org/db/seq/protein?name=MES-2;class=protein" target="_blank">MES-2</a>, which is a histone methyltransferase required for all other H3K27 methylation in the early embryo (<a href="#bib2">Bender et al., 2004</a>
                     ). This may further indicate a zygotic process engaging upon the birth of <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a> that involves activation of <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/Z3-restricted histone methyltransferase(s).
                  </p>
                  <div class="table"><a name="table1"></a><p class="title">Table 1. Histone modifications in the early germ line</p>
                     <table summary="Table 1. Histone modifications in the early germ line" border="1">
                        <colgroup>
                           <col align="center">
                           <col align="center">
                           <col align="center">
                           <col align="center">
                           <col align="center">
                           <col align="center">
                        </colgroup>
                        <thead>
                           <tr>
                              <th align="center">Histone modification</th>
                              <th align="center">Correlation</th>
                              <th align="center">P<sub>1</sub> - P<sub>4</sub>
                                 
                              </th>
                              <th align="center">
                                 <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a> (embryo)
                              </th>
                              <th align="center">
                                 <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a> (hatch + feed)
                              </th>
                              <th align="center">Somatic blastomeres</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td align="center">H4AcetylK16</td>
                              <td align="center">Active genes</td>
                              <td align="center">Present</td>
                              <td align="center">Present</td>
                              <td align="center">Present</td>
                              <td align="center">Present</td>
                           </tr>
                           <tr>
                              <td align="center">H4acetylK8</td>
                              <td align="center">Active genes</td>
                              <td align="center">Present</td>
                              <td align="center">Absent</td>
                              <td align="center">Present</td>
                              <td align="center">Present</td>
                           </tr>
                           <tr>
                              <td align="center">H3diacetylK9/ K14</td>
                              <td align="center">Active genes</td>
                              <td align="center">Present</td>
                              <td align="center">Present</td>
                              <td align="center">Present</td>
                              <td align="center">Present</td>
                           </tr>
                           <tr>
                              <td align="center">H3methK4 (di/tri)</td>
                              <td align="center">Active genes</td>
                              <td align="center">Present</td>
                              <td align="center">Absent</td>
                              <td align="center">Present</td>
                              <td align="center">Present</td>
                           </tr>
                           <tr>
                              <td align="center">H3dimethK9</td>
                              <td align="center">Inactive genes</td>
                              <td align="center">Absent</td>
                              <td align="center">Absent</td>
                              <td align="center">N/D</td>
                              <td align="center">Telomeres</td>
                           </tr>
                           <tr>
                              <td align="center">H3dimethK27</td>
                              <td align="center">Inactive genes</td>
                              <td align="center">Present</td>
                              <td align="center">Absent</td>
                              <td align="center">N/D</td>
                              <td align="center">Present</td>
                           </tr>
                           <tr>
                              <td align="center">H3trimeth27</td>
                              <td align="center">Inactive genes</td>
                              <td align="center">Present</td>
                              <td align="center">Present</td>
                              <td align="center">N/D</td>
                              <td align="center">Present</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e566"></a>3.2.2.&nbsp;H3methK4 dynamics in the PGCs
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The specific loss of the H3methK4 epitope is intriguing, since until recently there were no known histone demethylases and
                     this mark was thought to be enzymatically irreversible. A recent report, however, has described histone demethylation activity
                     in a human homolog of amine oxidases, of which <span class="emphasis"><em>C. elegans</em></span> has several potential orthologs, which raises the possibility of PGC-specific histone demethylase activity (<a href="#bib29">Shi, et al., 2004</a>
                     ). However, the chromatin remodeling could also be due to PGC-specific replacement of histone H3 coupled with prevention of
                     K4 methylation. In addition, since the observed change is most readily noticed as a loss of antibody binding, a cleavage of
                     the H3 tail containing the epitope, or masking of the antibody epitope through additional modifications of nearby residues
                     could explain the results. The mechanism that leads to the loss of H3methK4 in the PGCs remains to be elucidated.
                  </p>
                  <p>Once it disappears, H3methK4 remains absent from PGCs until after hatching, and even then does not return if the embryos hatch
                     into a foodless environment. Once food is available, however, H3methK4 can be observed to accumulate prior to the PGCs' post-embryonic
                     proliferation (<a href="#bib26">Schaner et al., 2003</a>
                     ). The absence of H3methK4 from PGCs upon their specification appears to be a conserved mark of these important cells, and
                     continued absence from PGC birth through hatching requires the <span class="emphasis"><em>Nanos</em></span> orthologues <a href="http://www.wormbase.org/db/seq/protein?name=NOS-1;class=protein" target="_blank">NOS-1</a> and <a href="http://www.wormbase.org/db/seq/protein?name=NOS-2;class=protein" target="_blank">NOS-2</a>. Both the absence of H3methK4 from PGCs and the requirement for NANOS for its maintenance are conserved in <span class="emphasis"><em>Drosophila</em></span>. Loss of both <span class="emphasis"><em>nos</em></span> gene products in worms, and the single NANOS in flies, results in premature return of H3methK4 and subsequent loss of the
                     primordial germ cells in both organisms (<a href="#bib26">Schaner et al., 2003</a>
                     ).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e606"></a>3.2.3.&nbsp;
                              <a href="http://www.wormbase.org/db/seq/protein?name=MEP-1;class=protein" target="_blank">MEP-1</a> and the PGCs
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A candidate for mediating the loss of H4acetylK8 (and perhaps indirectly H3methK4) is the worm counterpart of the nucleosome
                     remodeling and histone deacetylase (NURD) complex. This complex consists of <a href="http://www.wormbase.org/db/seq/protein?name=MEP-1;class=protein" target="_blank">MEP-1</a>, a Kr&auml;ppel-type zinc-finger protein, <a href="http://www.wormbase.org/db/seq/protein?name=LET-418;class=protein" target="_blank">LET-418</a>, a homologue of the NURD complex component Mi-2, and the histone deacetylase <a href="http://www.wormbase.org/db/seq/protein?name=HDA-1;class=protein" target="_blank">HDA-1</a>. Together they are involved in the repression of germ line-specific genes in the soma, presumably through their chromatin
                     remodeling activity. (<a href="#bib35">Unhavaithaya et al., 2002</a>
                     ; see <a href="http://www.wormbook.org/chapters/www_specgermline/specgermline.html" target="_top">Specification of the germ line</a>) <a href="http://www.wormbase.org/db/seq/protein?name=MEP-1;class=protein" target="_blank">MEP-1</a>, <a href="http://www.wormbase.org/db/seq/protein?name=HDA-1;class=protein" target="_blank">HDA-1</a>, and <a href="http://www.wormbase.org/db/seq/protein?name=LET-418;class=protein" target="_blank">LET-418</a> have been shown to interact with <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a>, suggesting that the NURD complex could be regulated in the embryonic germ line by <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a>. Once <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> is degraded at the 100-cell stage, the complex would then be free to remodel the chromatin in the PGCs. Indeed, loss of H4acetylK8
                     and H3methK4 from the PGCs is temporally linked to the loss of <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a>. This scenario, however, lacks experimental support since depletion of <span class="emphasis"><em>
                           <a href="http://www.wormbase.org/db/gene/gene?name=mep-1" target="_blank">mep-1</a>
                           </em></span>, <span class="emphasis"><em>
                           <a href="http://www.wormbase.org/db/gene/gene?name=let-418" target="_blank">let-418</a>
                           </em></span>, or <span class="emphasis"><em>
                           <a href="http://www.wormbase.org/db/gene/gene?name=hda-1" target="_blank">hda-1</a>
                           </em></span> by RNAi had no appreciable effect on loss of H3methK4 or H4acetylK8 in PGCs (<a href="#bib26">Schaner et al., 2003</a>
                     ). <span class="emphasis"><em>
                           <a href="http://www.wormbase.org/db/gene/gene?name=mep-1" target="_blank">mep-1</a>
                           </em></span> and <span class="emphasis"><em>
                           <a href="http://www.wormbase.org/db/gene/gene?name=let-418" target="_blank">let-418</a>
                           </em></span> mutants are viable but grow up as sterile adults, suggesting that the NURD complex does indeed play essential roles in the
                     germ line, but perhaps at a later stage (<a href="#bib35">Unhavaithaya et al., 2002</a>
                     ).
                  </p>
                  <p>
                     <span class="emphasis"><em>Summary: The first symmetric division of the embryonic germ line yields the PGCs, <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a>. Upon their birth, <a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a> degrades and there is a change in chromatin architecture specific to the PGCs. This is similar to what is observed in Drosophila
                           PGCs. In both organisms, NANOS activity is required to maintain this unique architecture, which correlates with transcriptional
                           quiescence. The factors required to establish the PGC-specific chromatin in C. elegans are not known.</em></span>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e706"></a>4.&nbsp;Post-embryonic germline chromatin
                     </h2>
                  </div>
               </div>
            </div>
            <p>As detailed above, H3methK4 normally returns to <a href="http://www.wormbase.org/db/get?name=Z2;class=Cell" target="_blank">Z2</a>/<a href="http://www.wormbase.org/db/get?name=Z3;class=Cell" target="_blank">Z3</a> prior to their proliferation, which happens after completion of embryogenesis and when the hatched larva has begun feeding.
               The return of this modification thus coincides with a presumed need to generally reactivate transcription in advance of germ
               cell proliferation (<a href="#bib26">Schaner et al., 2003</a>
               ). From this point on the genome in post-embryonic germ cells is transcriptionally active, as indicated both by germ cell
               expansion, bulk maternal gene expression (in later stages), and high levels of histone modifications that correlate with active
               transcription. Two striking exceptions to this active status exist: the X chromosome and extrachromosomal transgene arrays.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e722"></a>4.1.&nbsp;X chromosome-specific chromatin regulation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The X chromosome in immature germ cells of both males and hermaphrodites lacks most histone modifications corresponding to
                  transcriptionally competent and/or active chromatin including, H3methK4 (<a href="#figure2">Figure 2</a>A and B), H4acetyl-K5, -K8, -K12, and -K16, and H3phosphoS10 (<a href="#bib12">Kelly et al., 2002</a>
                  ; <a href="#bib25">Reuben and Lin, 2002</a>
                  ). Indeed, the Ser-2 phospho-epitope of elongating RNA POLII is also largely absent (<a href="#bib12">Kelly et al., 2002</a>
                  ; <a href="#bib5">Fong et al., 2002</a>
                  ). Both male and females gonads are gamete factories, with distal proliferative cells exiting mitosis and progressing into
                  and through meiotic stages in the more proximal regions (see <a href="http://www.wormbook.org/chapters/www_germlineprolifcontrol/germlineprolifcontrol.html" target="_blank">Germline proliferation and its control</a>). The X chromosome(s) is uniquely and strikingly depleted of the above modifications in all stages from the proliferative
                  zone through pachytene stage in both ovary and testis (compare <a href="#table2" title="Table 2. X chromosome histone modifications in the adult germ line">Table 2</a> to <a href="#table3" title="Table 3. Autosome histone modifications in the adult germ line">Table 3</a>). This pattern suggests that there is a wholesale inactivation of the X chromosome during germ cell maturation, and this
                  is consistent with genome-wide microarray analyses showing germline genes to be significantly under-represented on the X chromosome.
                  (<a href="#bib24">Reinke et al., 2000</a>
                  ; see <a href="http://www.wormbook.org/chapters/www_germlinegenomics/germlinegenomics.html" target="_blank">Germline genomics</a>). An interesting, yet unresolved question is whether the loss of germline genes from the X chromosome is an evolved consequence
                  of its unique chromatin status, or whether the unique chromatin is a simple consequence of the loss of genes that would be
                  active in this tissue.
               </p>
               <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="360">
<tr>
  <td align="center"><p><a href="germlinechromatinfig2.jpg"><img src="germlinechromatinfig2_s.jpg" border="2" align="middle" alt=" figure 2"></a>
  </td>
</tr>
                  </table>
                  <div class="caption">
                     <p><b>Figure 2. Different histone modification patterns on the hermaphrodite and male X
                                       chromosomes.&nbsp;</b>Adult pachytene stage nuclei were fixed and stained with antibodies to specific
                                    histone modifications (green) and counterstained with DAPI (red). In the hermaphrodite
                                    germline, the X chromosome pair (white arrows) does not stain with anti-H3methK4 (A) or
                                    anti-H3methK9 (C). H3methK4, but not H3methK9, eventually accumulates on the
                                    hermaphrodite X chromosome during diplotene (<a href="#bib12">Kelly et al., 2002</a>). In the male testis,
                                    the X chromosome never stains with anti-H3methK4 (B) but does stain strongly with
                                    anti-H3methK9 (D). Scale bar equals 5<span class="symbol">&#956;</span>m.
                     </p>
                  </div>
               </div>
               <div class="table"><a name="table2"></a><p class="title">Table 2. X chromosome histone modifications in the adult germ line</p>
                  <table summary="Table 2. X chromosome histone modifications in the adult germ line" border="1">
                     <colgroup>
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                     </colgroup>
                     <thead>
                        <tr>
                           <th colspan="6" align="left">A. Oogenesis</th>
                        </tr>
                        <tr>
                           <th align="center">Histone Modifications</th>
                           <th align="center">Distal <span class="symbol">&#8594;</span> transition
                           </th>
                           <th align="center">Pachytene</th>
                           <th align="center">Diplotene</th>
                           <th align="center">Diakinesis</th>
                           <th align="center">1-cell embryo</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="center">H4acetylK8, K16</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Present (low)</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3diacetylK9/ K14</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Present (low)</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3methK4</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Present (low)</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3dimethK9</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                        </tr>
                        <tr>
                           <td align="center">H3dimethK27</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3trimethK27</td>
                           <td align="center">Present (high)</td>
                           <td align="center">Present (high)</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                        </tr>
                     </tbody>
                  </table>
                  <table summary="Table 2. X chromosome histone modifications in the adult germ line" border="1">
                     <colgroup>
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                     </colgroup>
                     <thead>
                        <tr>
                           <th colspan="6" align="left">B. Spermatogenesis</th>
                        </tr>
                        <tr>
                           <th align="center">Histone Modifications</th>
                           <th align="center">Distal <span class="symbol">&#8594;</span> transition
                           </th>
                           <th align="center">Pachytene</th>
                           <th align="center">1&deg;/2&deg;</th>
                           <th align="center">Spermatids</th>
                           <th align="center">1-cell embryo</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="center">H4acetylK8, K16</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3diacetylK9/ K14</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                        </tr>
                        <tr>
                           <td align="center">H3methK4</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                        </tr>
                        <tr>
                           <td align="center">H3dimethK9</td>
                           <td align="center">Absent</td>
                           <td align="center">Present</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                        </tr>
                        <tr>
                           <td align="center">H3dimethK27</td>
                           <td align="center">Present</td>
                           <td align="center">N/D</td>
                           <td align="center">N/D</td>
                           <td align="center">N/D</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3trimethK27</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Absent</td>
                           <td align="center">Present</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
               <div class="table"><a name="table3"></a><p class="title">Table 3. Autosome histone modifications in the adult germ line</p>
                  <table summary="Table 3. Autosome histone modifications in the adult germ line" border="1">
                     <colgroup>
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                     </colgroup>
                     <thead>
                        <tr>
                           <th colspan="6" align="left">A. Oogenesis</th>
                        </tr>
                        <tr>
                           <th align="center">Histone modifications</th>
                           <th align="center">Distal <span class="symbol">&#8594;</span> transition
                           </th>
                           <th align="center">Pachytene</th>
                           <th align="center">Diplotene</th>
                           <th align="center">Diakinesis</th>
                           <th align="center">1-cell embryo</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="center">H4acetylK8, K16</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3diacetylK9/ K14</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3methK4</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3dimethK9</td>
                           <td align="center">Absent</td>
                           <td align="center">Present (late)</td>
                           <td align="center">Present</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                        </tr>
                        <tr>
                           <td align="center">H3methK27 (di/tri)</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                        </tr>
                     </tbody>
                  </table>
                  <table summary="Table 3. Autosome histone modifications in the adult germ line" border="1">
                     <colgroup>
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                        <col align="center">
                     </colgroup>
                     <thead>
                        <tr>
                           <th colspan="6" align="left">B. Spermatogenesis</th>
                        </tr>
                        <tr>
                           <th align="center">Histone modifications</th>
                           <th align="center">Distal <span class="symbol">&#8594;</span> transition
                           </th>
                           <th align="center">Pachytene</th>
                           <th align="center">1&deg;/2&deg;</th>
                           <th align="center">Spermatids</th>
                           <th align="center">1-cell embryo</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td align="center">H4acetylK8, K16</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present (low)</td>
                           <td align="center">Absent</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3diacetylK9/ K14</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present (low)</td>
                           <td align="center">Absent</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3methK4</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present (low)</td>
                           <td align="center">Absent</td>
                           <td align="center">Present</td>
                        </tr>
                        <tr>
                           <td align="center">H3dimethK9</td>
                           <td align="center">Absent</td>
                           <td align="center">Present (late)</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                           <td align="center">Absent</td>
                        </tr>
                        <tr>
                           <td align="center">H3methK27 (di/tri)</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Present</td>
                           <td align="center">Absent</td>
                           <td align="center">Present</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1188"></a>4.1.1.&nbsp;X chromatin and Polycomb proteins
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The transcriptional repression of the X chromosome is accomplished, at least in part, by the MES proteins. <a href="http://www.wormbase.org/db/seq/protein?name=MES-6;class=protein" target="_blank">MES-6</a> (the <span class="emphasis"><em>C. elegans</em></span> homologue of <span class="emphasis"><em>Drosophila</em></span> Extra Sex Combs), <a href="http://www.wormbase.org/db/seq/protein?name=MES-2;class=protein" target="_blank">MES-2</a> (the <span class="emphasis"><em>C. elegans</em></span> homologue of <span class="emphasis"><em>Drosophila</em></span> Enhancer of Zeste), and <a href="http://www.wormbase.org/db/seq/protein?name=MES-3;class=protein" target="_blank">MES-3</a> (a novel protein), act together in a complex that is maternally supplied and required for germ cell function (<a href="#bib37">Xu et al., 2001</a>
                     ). Enhancer of Zeste and Extra Sex Combs are members of the Polycomb group of transcriptional regulators, known for their
                     role in promoting a repressed chromatin state. The proposed repressive activity of the MES complex has recently been strengthened
                     by the demonstration of its function as a histone methyltransferase complex. The catalytic component <a href="http://www.wormbase.org/db/seq/protein?name=MES-2;class=protein" target="_blank">MES-2</a> di- and tri-methylates lysine 27 of histone H3 (<a href="#bib2">Bender et al., 2004</a>
                     ). H3methK27, particularly its tri-methylated form (H3trimethK27), has been shown to correspond with transcriptional repression
                     and H3trimethK27 is enriched on the X chromosome during pachytene. An additional MES protein, <a href="http://www.wormbase.org/db/seq/protein?name=MES-4;class=protein" target="_blank">MES-4</a>, is a SET-domain containing protein thought to exclude the repressive <a href="http://www.wormbase.org/db/seq/protein?name=MES-2;class=protein" target="_blank">MES-2</a>,-3,-6 complex from the autosomes. <a href="http://www.wormbase.org/db/seq/protein?name=MES-4;class=protein" target="_blank">MES-4</a> is found on all chromosomes in adult germ cells except for the X, supporting its role as an antagonist of repressive chromatin
                     structure (<a href="#bib5">Fong et al., 2002</a>
                     ).
                  </p>
                  <p>The role of the MES proteins in X chromatin regulation likely contributes, at least in part, to the X dosage-dependent severity
                     of phenotypes in <span class="emphasis"><em>mes</em></span> mutants, with severity increasing with X dosage (<a href="#bib6">Garvin et al., 1998</a>
                     ). As mentioned above, other sex-specific aspects of X chromatin regulation also exist in later meiotic stages of male and
                     female germ cells.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1252"></a>4.1.2.&nbsp;The X chromosome in oogenesis
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In hermaphrodites, &#8220;activating&#8221; histone modifications begin to accumulate on the X during diplotene, reaching the levels present
                     on autosomes by diakinesis stage (<a href="#table2" title="Table 2. X chromosome histone modifications in the adult germ line">Table 2A</a>). This accumulation corresponds with the onset of X-linked oocyte-specific transcription that is not detected until late
                     pachytene stage. These transcripts are observed to accumulate at high levels in maturing oocytes. The accumulation may represent
                     mRNA synthesized in, and transported from, nuclei in earlier stages, since the phospho-Ser5 epitope of RNA POLII is not detectable
                     on the chromatin in these cells (<a href="#bib12">Kelly et al., 2002</a>
                     ). After ovulation and fertilization, the oocyte chromosomes retain all of the histone modifications present in diakinesis,
                     even after the <a href="http://www.wormbase.org/db/get?name=MI;class=Cell" target="_blank">MI</a> and MII meiotic divisions. The oocyte genome therefore appears to arrive into the zygote in a chromatin state grossly similar
                     to its status in oogenesis, and this does not appreciably change in early stages.
                  </p>
                  <p>Despite the transcriptional repression of the X chromosome until late pachytene, it does not specifically accumulate global
                     H3methK9 during oogenesis. Indeed, whereas there is a transient accumulation of H3methK9 on all autosomes as the germ cells
                     exit pachytene stage of meiosis, the X chromosome seems to lack this addition (<a href="#table2" title="Table 2. X chromosome histone modifications in the adult germ line">Table 2A</a>; <a href="#bib12">Kelly et al., 2002</a>
                     ; <a href="#bib1">Bean et al., 2004</a>
                     ). The accumulation of H3methK9 on the autosomes coincides with the accumulation of activating modifications on the X chromosomes,
                     so there is likely a regulatory connection. One should note that these data (as well as those discussed for males below) represent
                     antibody staining with commercially available anti-H3dimethK9 antisera; no published studies using highly specific antibodies
                     against the trimethylated form in worms are currently available.
                  </p>
                  <p>The other &#8220;repressive&#8221; marks, H3di- and trimethK27, are both present on all chromosomes at all germ cell stages in the ovary,
                     but there is an enrichment of H3trimethK27 on the X chromosomes in pachytene nuclei (<a href="#table2" title="Table 2. X chromosome histone modifications in the adult germ line">Tables 2A</a> and <a href="#table3" title="Table 3. Autosome histone modifications in the adult germ line">3A</a>). In <span class="emphasis"><em>
                           <a href="http://www.wormbase.org/db/gene/gene?name=mes-2" target="_blank">mes-2</a>
                           </em></span> mutant ovaries, both forms of H3methK27 are absent from distal to medial stages, but both are present in more proximal stages
                     (<a href="#bib2">Bender et al., 2004</a>
                     ). This presumably reflects the activity of another K27 methyltransferase that is only active in mid-pachytene and later stages
                     of meiosis.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="d0e1302"></a>4.1.3.&nbsp;The X chromosome in spermatogenesis
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>There are two types of sperm in <span class="emphasis"><em>C. elegans</em></span>&#8212; those generated in hermaphrodite larvae, and those generated in males. We will primarily discuss male germ cells, but will
                     mention differences observed in hermaphrodite spermatogenesis where they occur. There are very distinct modes of chromatin
                     regulation that occur in the male germ line that are not normally observed in the female germ line. In the male testis, activating
                     modifications (e.g., H3 and H4 acetylation, H3methK4) never accumulate on the X chromosome in germ cells at any stage (<a href="#table2" title="Table 2. X chromosome histone modifications in the adult germ line">Table 2B</a>). The failure of the male X to accumulate these modifications indicates transcriptional repression throughout gametogenesis,
                     which is supported by the exceptional paucity reported for spermatogenesis genes on the X-chromosome (<a href="#bib24">Reinke et al., 2000</a>
                     ; see <a href="http://www.wormbook.org/chapters/www_germlinegenomics/germlinegenomics.html" target="_blank">Germline genomics</a>). Furthermore, all assayed modifications on the autosomes are observed to be shed as the germ cells exit diplotene during
                     spermatogenesis (<a href="#table2" title="Table 2. X chromosome histone modifications in the adult germ line">Table 2B</a>; <a href="#bib12">Kelly et al., 2002</a>
                     ). This is presumably due to sperm histone replacement that occurs in many organisms, but has not as yet been described in
                     <span class="emphasis"><em>C. elegans</em></span>.
                  </p>
                  <p>Other than the sperm-specific disappearance of H3di- and trimethK27 in diplotene nuclei (as is the case with all other assayed
                     modifications), male and female germ cells appear to have similar regulation of these marks (<a href="#table2" title="Table 2. X chromosome histone modifications in the adult germ line">Tables 2</a> and <a href="#table3" title="Table 3. Autosome histone modifications in the adult germ line">3</a>). This sharply contrasts with the H3methK9 pattern seen in male gametogenesis. The male X uniquely accumulates H3methK9 during
                     pachytene (<a href="#figure3">Figure 3</a>D) and maintains this modification until it disappears at spermatogenesis. Interestingly, this enrichment on the X does not
                     appear to happen in hermaphrodite spermatogenesis (<a href="#bib1">Bean et al., 2004</a>
                     ; <a href="#bib12">Kelly et al., 2002</a>
                     ). Why is the X chromosome targeted for H3methK9 enrichment only in male meiosis? One possibility is that this is a male-specific
                     process; however, targeting of the X for H3methK9 enrichment can occur in sexually transformed XO hermaphrodites and is absent
                     in transformed XX male spermatogenesis (<a href="#bib1">Bean et al., 2004</a>
                     ). Another possibility is that the targeting of the X is the result of its unpaired status during male meiosis. This seems
                     to be the case, since targeting of both X's occurs in <a href="http://www.wormbase.org/db/gene/gene?name=him-8" target="_blank">him-8</a> mutant hermaphrodites, in which X chromosome pairing is disrupted. In addition, transgene arrays and free autosomal duplications
                     are also targeted for H3methK9 addition during meiosis in both sexes (<a href="#bib1">Bean et al., 2004</a>
                     ). These results indicate that this process is neither X chromosome-specific or sex-specific, but instead may generally act
                     on DNA entering meiosis without a pairing partner.
                  </p>
                  <p>Three proteins have been identified that play a role in the enrichment of H3methK9 on the male X during meiosis. One, <a href="http://www.wormbase.org/db/seq/protein?name=HIM-17;class=protein" target="_blank">HIM-17</a>, is a chromatin-associated protein required for recombination, but not homolog alignment or synapsis, during meiosis (<a href="#bib40">Reddy and Villeneuve, 2004</a>). A second protein is <a href="http://www.wormbase.org/db/seq/protein?name=RHA-1;class=protein" target="_blank">RHA-1</a>. <a href="http://www.wormbase.org/db/seq/protein?name=RHA-1;class=protein" target="_blank">RHA-1</a> is a putative RNA helicase that is required for proper chromatin assembly on the male X, transgene silencing, and RNAi in
                     germ cells. Curiously, the chromatin and silencing defects are only observed at the elevated temperatures, and the animals
                     are sterile. In contrast, the animals are fertile, yet still defective in RNAi, at lower temperatures (<a href="#bib41">Walstrom et al., 2005</a>). Interestingly, the germline-specific RNA-dependent RNA polymerase (RdRP) <a href="http://www.wormbase.org/db/seq/protein?name=EGO-1;class=protein" target="_blank">EGO-1</a> is required for both RNAi and H3methK9 enrichment on unpaired DNA (<a href="#bib39">Maine et al., 2005</a>). In the filamentous fungus Neurospora crassa, repression of unpaired DNA ("meiotic silencing") also requires an RdRP, which
                     suggests that the process is conserved (discussed below).
                  </p>
                  <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="540">
<tr>
  <td align="center"><p><a href="germlinechromatinfig3.jpg"><img src="germlinechromatinfig3_s.jpg" border="2" align="middle" alt=" figure 3"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 3. Epigenetic imprint on the paternal X chromosome.&nbsp;</b>Adult hermaphrodite gonads and embryos were fixed and stained with anti-H3methK4
                                 (green) and counterstained with DAPI (red). All chromosomes of the mature oocyte stain
                                 with anti-H3methK4 (A). This modification (like all others tested: see text) is not
                                 detectable in spermatids (B). Upon fertilization, H3methK4 accumulates on all of the
                                 sperm-derived autosomes, but is absent from the X chromosome (white arrow) (C). The
                                 paternal X chromosome remains refractory to this histone modification even after
                                 extensive decondensation (white arrow in D), and multiple rounds of cell division (not
                                 shown; <a href="#bib1">Bean et al., 2004</a>). Note that panel C shows a fertilized oocyte shortly after
                                 fertilization and during the second meiotic division, while Panel D shows a slightly
                                 later staged one-cell zygote during pronuclear fusion.
                        </p>
                     </div>
                  </div>
                  <p>
                     <span class="emphasis"><em>Summary: The post-embryonic germline is transcriptionally engaged and proliferative. Instead of a global genomic repression,
                           as seen in the embryo, there are post-embryonic mechanisms that target repression of the X chromosome, and these can involve
                           assembly of repressive chromatin. Transgene and X chromosome repression are mediated, in part, by the mes Polycomb Group proteins.
                           Unpaired chromosomes going through meioisis in either sex can be targeted for repressive chromatin assembly. Male specific
                           targeting of the X chromosome for repressive chromatin assembly appears to be an evolved consequence of the unpaired status
                           of the X in males.</em></span>
                     
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1408"></a>5.&nbsp;Completing the cycle: the zygote
                     </h2>
                  </div>
               </div>
            </div>
            <p>The meiotic divisions in the oocyte occur after fertilization, and thus the 1-cell embryo is essentially still very much a
               germ cell in character with two separate genomes. As discussed above, each genome arrives with quite distinct histories of
               chromatin regulation: the oocyte genome is enriched in histones that carry activating modifications, whereas the sperm lacks
               any such marks, and indeed may be depleted of histones. Prior to genome fusion, the sperm pronuclear chromosomes accumulate
               histone modifications and begin to decondense&#8212; with a specific exception that is discussed below. After the meiotic divisions
               are complete, the maternal and paternal haploid genomes fuse to form the diploid zygote.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="d0e1413"></a>5.1.&nbsp;X chromatin imprinting
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Before and after fertilization, all chromosomes in the oocyte pronucleus contain H3methK4 and other epitopes consistent with
                  transcriptional competence (<a href="#figure3">Figure 3</a>A and data not shown). The sperm arrives devoid of all assayed modifications (<a href="#figure3">Figure 3B</a>), but quickly accumulates them on its autosomes as the chromosomes in the male pronucleus begin to decondense (<a href="#figure3">Figure 3</a>C). The sperm X (Xp), however, initially appears uniquely refractory to accumulating H3methK4 (<a href="#figure3">Figure 3</a>C and D), and remains significantly and heritably devoid of this and other H3-specific modifications for several rounds of
                  cell division (<a href="#table2" title="Table 2. X chromosome histone modifications in the adult germ line">Table 2B</a>; <a href="#bib1">Bean et al., 2004</a>
                  ). These modifications initially gradually appear with increased embryonic cell number, until the Xp is indistinguishable
                  from other chromosomes in 20-24-cell embryos. This delayed appearance is consistent with dilution of a persistently unmodified
                  histone H3, or H3-like molecule, during DNA replication. In addition, the timing of the appearance of H3methK4 on the Xp is
                  influenced by whether it had a pairing partner during meiosis. The apparent zygotic activation of an Xp from an XO parent
                  is significantly delayed relative to activation of an Xp from an XX parent (<a href="#bib1">Bean et al., 2004</a>
                  ). It is important to emphasize, however, that imprint establishment occurs in sperm from both XO and XX animals&#8212; meiotic
                  pairing in the parental germ line only influences the stability of the imprint in the offspring.
               </p>
               <p>The rate of disappearance of the imprint could theoretically be determined by its rate of replacement by S-phase histone H3,
                  which is a suitable substrate for lysine 4 methylation. A higher level of the imprint on an Xp could thus take longer (i.e.,
                  more cell divisions) to be replaced in the embryo. One obvious difference between these Xp's is that the Xp from an XO parent
                  is transiently marked with H3methK9 during pachytene stage, while the Xp from an XX parent is not, as discussed above. The
                  transient H3methK9 mark could direct the incorporation of increased levels of an unidentified &#8220;imprint&#8221; an Xp from an XO parent
                  during spermatogenesis. This higher level may subsequently take more rounds of replication in the embryo for effective S-phase
                  coupled histone replacement of the imprint, which would be observed as an increased perdurance of the mark.
               </p>
               <p>
                  <span class="emphasis"><em>Summary: Passage of an X chromosome through spermatogenesis establishes an epigenetic, metastable imprint on the X chromosome
                        that results in its temporary inactivation in XX offspring. The duration of the inactivation is influenced by whether the
                        X chromosome had a pairing partner in sperm meiosis. The nature of the imprint, at this time, is unclear.</em></span>
                  
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1447"></a>6.&nbsp;Special circumstances: germline silencing of transgenes
                     </h2>
                  </div>
               </div>
            </div>
            <p>In many ways, the silencing of repetitive transgene arrays mimics at least a subset of the repressive mechanisms regulating
               X chromatin assembly. Repetitive transgene arrays are silenced in all post-embryonic germ cells, including larval stages (<a href="#bib13">Kelly et al., 1997</a>
               ; <a href="#figure4">Figure 4</a>C). In adults, the stage at which most analyses have been done, this silencing correlates with an enrichment of H3methK9 in
               the array chromatin (<a href="#figure4">Figure 4</a>F), as well as an absence of H3methK4 (<a href="#figure4">Figure 4</a>D) and other &#8220;active chromatin&#8221; marks (<a href="#bib12">Kelly et al., 2002</a>
               ; <a href="#bib25">Reuben and Lin, 2002</a>
               ). Silencing of these arrays is dependent on the DNA content, as less repetitive (more &#8220;complex&#8221;) arrays are not silenced
               (<a href="#figure4">Figure 4</a>B) and consequently contain H3methK4 and can lack H3methK9 (<a href="#bib12">Kelly et al., 2002</a>
                and W. Kelly, unpublished). The C. elegans MES/ Polycomb Group proteins (discussed above) are required for efficient silencing
               of repetitive transgene arrays, as are <a href="http://www.wormbase.org/db/seq/protein?name=HPL-2;class=protein" target="_blank">HPL-2</a> (a highly conserved heterochromatin binding protein) and H1.1 (an H1 histone isoform). These findings strongly implicate
               a predominant role for chromatin structure in this silencing process (<a href="#bib3">Couteau et al., 2002</a>
               ; <a href="#bib10">Jedrusik and Schulze, 2001</a>
               ). Interestingly, a subset of mutants defective in RNAi also exhibit defective transgene silencing in germ cells, illustrating
               a potential contribution of dsRNAs toward post-transcriptional and/or transcriptional repression, as discussed below (<a href="#bib14">Ketting et al., 1999</a>
               ; <a href="#bib34">Tabara et al., 1999</a>
               ). A genome-wide screen for genes required for RNA-mediated silencing also identified 14 genes whose functions are also required
               for germline silencing, further strengthening a role for RNAi in this process (<a href="#bib42">Kim et al., 2005</a>).
            </p>
            <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="540">
<tr>
  <td align="center"><p><a href="germlinechromatinfig4.jpg"><img src="germlinechromatinfig4_s.jpg" border="2" align="middle" alt=" figure 4"></a>
  </td>
</tr>
               </table>
               <div class="caption">
                  <p><b>Figure 4. Extrachromosomal array expression and histone modifications in the germline.&nbsp;</b>Repetitive extrachromosomal transgenes are often silenced in adult germ cell nuclei. Silencing of transgene arrays can be alleviated by decreasing the copy number of the reporter transgene, and increasing the &#8220;complexity&#8221; of the array DNA. Panels A and B show a DIC image (A) and GFP fluorescence (B) of a transgenic animal with a &#8220;complex&#8221; array carrying a <span class="emphasis"><em>let-858::gfp</em></span> reporter-a gene that should express in all cells. Panel C shows the expression pattern of the same reporter in a repetitive transgene. The germ cell nuclei in the gonad are circled in A-C. The silenced, repetitive transgene in the strain depicted in C is depleted of H3methK4 (D) and enriched in H3methK9 (F); arrows in D-G point to the transgene. Panels D and G: Red is DAPI; Green is antibody signal for the respective histone modifications. Panels E and F are DAPI stains of Panels D and G, respectively.</p>
               </div>
            </div>
            <p>
               <span class="emphasis"><em>Summary: Multi-copy transgenes are strongly silenced in the post-embryonic germ line and mimic, in many respects, the regulation
                     of the X chromosome. Mutants that cause defects in X-specific chromatin assembly, or are required for assembly of repressive
                     chromatin, also have defects in silencing multi-copy transgenes in the germ line</em></span>.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1526"></a>7.&nbsp;Germline goals include genome protection 
                     </h2>
                  </div>
               </div>
            </div>
            <p>Why are extrachromosomal arrays and other unpaired DNA species preferentially targeted for H3methK9 addition during meiosis,
               and more specifically, during early pachytene stage? Meiotic pairing of homologous chromosomes is effectively complete upon
               entry into pachytene. Meiotic pairing not only allows for genetic recombination, but also affords a chance for the linear
               sequences of entire homologous chromosomes to be compared. Mobile genetic elements by their nature are mutagenic; the germ
               line is the one tissue where mutations are heritable and thus is a major site of the battles waged between these elements
               and host genomes. Significant deviations in linear sequence, such as those imposed by transposable element insertions, could
               perhaps be &#8220;noticed&#8221; at this point as looped out, or unpaired DNA, and targeted for repression to prevent further spreading.
               As mentioned above, similar phenomenon, meiotic silencing by unpaired DNA (MSUD, or simply meiotic silencing), has been identified
               in <span class="emphasis"><em>Neurospora</em></span> (<a href="#bib30">Shiu et al., 2001</a>
               ), and surprisingly, it requires conserved components of the RNA interference pathway (<a href="#bib9">Hynes and Todd, 2003</a>
               ). This suggests a role for RNAi in MSUD, in addition to its potential roles in other modes of genome surveillance.
            </p>
            <p>Transposons are major targets of silencing mechanisms in most, if not all, organisms, and are specifically repressed in the
               <span class="emphasis"><em>C. elegans</em></span> germ line by the RNAi machinery (Sijen and Plasterk, 2003). In fact, many of the mutator (mut) genes involved in germline
               silencing of transposable elements are also required for RNAi and vice versa (<a href="#bib14">Ketting et al., 1999</a>
               ; <a href="#bib34">Tabara et al., 1999</a>
               ). Moreover, defective transgene silencing in germ cells is observed in mutants defective in both RNAi and transposon repression,
               indicating that all three processes are mechanistically related. Though RNAi has been mostly appreciated for its role in post-transcriptional
               gene silencing, it is now recognized as serving highly conserved and ancient roles in transcriptional regulation (<a href="#bib18">Lippman and Martienssen, 2004</a>
               ). A major and conserved component of the transcriptional arm of RNAi is its role in targeting of H3methK9 to genomic sequences
               (<a href="#bib36">Volpe et al., 2002</a>
               ). It is thus not complete speculation to hypothesize that pairing-sensing mechanisms operating during worm meiosis may somehow
               utilize components of the RNAi machinery to direct assembly of heterochromatin to unpaired DNA. The natural targets of these
               processes are likely to be novel transposition events&#8212; the male X may be a happenstance target by virtue of its (evolved)
               unpaired status during male meiosis. The role of RNAi in these processes awaits further investigation.
            </p>
            <p>
               <span class="emphasis"><em>Summary: Some mutants that are defective in RNAi also exhibit defects in germline transposon and transgene repression. RNAi
                     has been shown in other organisms to direct repressive chromatin assembly. This may suggest that germline transgene silencing
                     may, in part, be revealing genome defense mechanisms that act through RNAi to target multi-copy genetic elements for repressive
                     chromatin assembly.</em></span>
               
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1576"></a>8.&nbsp;Future directions
                     </h2>
                  </div>
               </div>
            </div>
            <p>Histone modifications, like histones themselves, are exquisitely conserved among eukaryotes and represent a very ancient form
               of basal genetic regulation. This conservation largely extends to the function of each individual modification, and therefore
               the rules of the game as they are understood in any organisms are likely to apply to others. The game in question, &#8220;epigenetics&#8221;
               (defined as heritable changes in gene expression in the absence of changes in genomic sequence), is a rapidly growing field
               with tremendous biomedical implications. The genetics, cytology, and molecular biology of <span class="emphasis"><em>C. elegans</em></span> are likely to continue to contribute in a major way to our understanding of the rules, and perhaps how we can bend them in
               our favor.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="d0e1584"></a>9.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <p></p>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bean, C.J., Schaner, C.E., and Kelly, W.G. (2004). Meiotic pairing and imprinted X chromatin assembly in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nat. Genet. <span class="emphasis"><em>36</em></span>, 100&#8211;105.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14702046&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/ng1283" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bender, L.B., Cao, R., Zhang, Y., and Strome, S. (2004). The <a href="http://www.wormbase.org/db/seq/protein?name=MES-2;class=protein" target="_blank">MES-2</a>/<a href="http://www.wormbase.org/db/seq/protein?name=MES-3;class=protein" target="_blank">MES-3</a>/<a href="http://www.wormbase.org/db/seq/protein?name=MES-6;class=protein" target="_blank">MES-6</a> Complex and Regulation of Histone H3 Methylation in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>14</em></span>, 1639&#8211;1643.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15380065&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.08.062" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Couteau, F., Guerry, F., Muller, F., and Palladino, F. (2002). A heterochromatin protein 1 homologue in <span class="emphasis"><em>Caenorhabditis elegans</em></span> acts in germline and vulval development. EMBO Rep. <span class="emphasis"><em>3</em></span>, 235&#8211;241.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11850401&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/embo-reports/kvf051" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Emmons, S.W., Yesner, L., Ruan, K.S., and Katzenberg, D. (1983). Evidence for a transposon in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Cell <span class="emphasis"><em>32</em></span>, 55&#8211;65.</span>
                     <a href="http://dx.doi.org/10.1016/0092-8674(83)90496-8" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Fong, Y., Bender, L., Wang, W., and Strome, S. (2002). Regulation of the different chromatin states of autosomes and X chromosomes
                        in the germ line of <span class="emphasis"><em>C. elegans</em></span>. Science <span class="emphasis"><em>296</em></span>, 2235&#8211;2238.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12077420&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1070790" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Garvin, C., Holdeman, R., and Strome, S. (1998). The phenotype of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mes-2" target="_blank">mes-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mes-3" target="_blank">mes-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mes-4" target="_blank">mes-4</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mes-6" target="_blank">mes-6</a></em></span>, maternal-effect genes required for survival of the germline in <span class="emphasis"><em>Caenorhabditis elegans</em></span>, is sensitive to chromosome dosage. Genetics <span class="emphasis"><em>148</em></span>, 167&#8211;185.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9475730&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Grunstein, M. (1997). Histone acetylation in chromatin structure and transcription. Nature <span class="emphasis"><em>389</em></span>, 349&#8211;352.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9311776&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/38664" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hampsey, M., and Reinberg, D. (2003). Tails of intrigue: phosphorylation of RNA polymerase II mediates histone methylation.
                        Cell <span class="emphasis"><em>113</em></span>, 429&#8211;432.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12757703&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(03)00360-X" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hynes, M.J., and Todd, R.B. (2003). Detection of unpaired DNA at meiosis results in RNA-mediated silencing. Bioessays <span class="emphasis"><em>25</em></span>, 99&#8211;103.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12539234&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/bies.10241" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jedrusik, M.A., and Schulze, E. (2001). A single histone H1 isoform (H1.1) is essential for chromatin silencing and germline
                        development in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>128</em></span>, 1069&#8211;1080.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11245572&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kawasaki, I., Shim, Y.H., Kirchner, J., Kaminker, J., Wood, W.B., and Strome, S. (1998). <a href="http://www.wormbase.org/db/seq/protein?name=PGL-1;class=protein" target="_blank">PGL-1</a>, a predicted RNA-binding component of germ granules, is essential for fertility in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>94</em></span>, 635&#8211;645.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9741628&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(00)81605-0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kelly, W.G., Schaner, C.E., Dernburg, A.F., Lee, M.H., Kim, S.K., Villeneuve, A.M., and Reinke, V. (2002). X-chromosome silencing
                        in the germline of <span class="emphasis"><em>C. elegans</em></span>. Development <span class="emphasis"><em>129</em></span>, 479&#8211;492.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11807039&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kelly, W.G., Xu, S., Montgomery, M.K., and Fire, A. (1997). Distinct requirements for somatic and germline expression of a
                        generally expressed <span class="emphasis"><em>Caernorhabditis elegans</em></span> gene. Genetics <span class="emphasis"><em>146</em></span>, 227&#8211;238.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9136012&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ketting, R.F., Haverkamp, T.H., van Luenen, H.G., and Plasterk, R.H. (1999). <span class="emphasis"><em>Mut-7</em></span> of <span class="emphasis"><em>C. elegans</em></span>, required for transposon silencing and RNA interference, is a homolog of Werner syndrome helicase and RNaseD. Cell <span class="emphasis"><em>99</em></span>, 133&#8211;141.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10535732&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(00)81645-1" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kim, J.K., Gabel, H.W., Kamath, R.S., Tewari, M., Pasquinelli, A., Rual, J.F., Kennedy, S., Dybbs, M., Bertin, N., Kaplan,
                        J.M., Vidal, M., and Ruvkun, G. (2005). Functional genomic analysis of RNA interference in <span class="emphasis"><em>C. elegans</em></span>. Science <span class="emphasis"><em>308</em></span>, 1164&#8211;1167.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16049496&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1109267" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kouzarides, T. (2002). Histone methylation in transcriptional control. Curr. Opin. Genet. Dev. <span class="emphasis"><em>12</em></span>, 198&#8211;209.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11893494&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0959-437X(02)00287-3" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Krogan, N.J., Kim, M., Tong, A., Golshani, A., Cagney, G., Canadien, V., Richards, D.P., Beattie, B.K., Emili, A., Boone,
                        C., et al. (2003). Methylation of histone H3 by Set2 in <span class="emphasis"><em>Saccharomyces cerevisiae </em></span>is linked to transcriptional elongation by RNA polymerase II. Mol. Cell Biol. <span class="emphasis"><em>23</em></span>, 4207&#8211;4218.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12773564&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1128/MCB.23.12.4207-4218.2003" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lachner, M., and Jenuwein, T. (2002). The many faces of histone lysine methylation. Curr. Opin. Cell Biol. <span class="emphasis"><em>14</em></span>, 286&#8211;298.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12067650&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0955-0674(02)00335-6" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lippman, Z., and Martienssen, R. (2004). The role of RNA interference in heterochromatic silencing. Nature <span class="emphasis"><em>431</em></span>, 364&#8211;370.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15372044&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02875" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Maine, E.M., Hauth, J., Ratliff, T., Vought, V.E., She, X., and Kelly, W.G. (2005). <a href="http://www.wormbase.org/db/seq/protein?name=EGO-1;class=protein" target="_blank">EGO-1</a>, a putative RNA-dependent RNA polymerase, is required for heterochromatin assembly on unpaired dna during <span class="emphasis"><em>C. elegans</em></span> meiosis. Curr. Biol. <span class="emphasis"><em>15</em></span>, 1972&#8211;1978.</span> 
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=16271877&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2005.09.049" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mello, C.C., Draper, B.W., Krause, M., Weintraub, H., and Priess, J.R. (1992). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pie-1" target="_blank">pie-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mex-1" target="_blank">mex-1</a></em></span> genes and maternal control of blastomere identity in early <span class="emphasis"><em>C. elegans</em></span> embryos. Cell <span class="emphasis"><em>70</em></span>, 163&#8211;176.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1623520&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(92)90542-K" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mello, C.C., Schubert, C., Draper, B., Zhang, W., Lobel, R., and Priess, J.R. (1996). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a></em></span> protein and germline specification in <span class="emphasis"><em>C. elegans</em></span> embryos. Nature <span class="emphasis"><em>382</em></span>, 710&#8211;712.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8751440&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/382710a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Muller, J., Hart, C.M., Francis, N.J., Vargas, M.L., Sengupta, A., Wild, B., Miller, E.L., O'Connor, M.B., Kingston, R.E.,
                        and Simon, J.A. (2002). Histone methyltransferase activity of a <span class="emphasis"><em>Drosophila </em></span>Polycomb group repressor complex. Cell <span class="emphasis"><em>111</em></span>, 197&#8211;208.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12408864&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(02)00976-5" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ng, H.H., Robert, F., Young, R.A., and Struhl, K. (2003). Targeted recruitment of Set1 histone methylase by elongating Pol
                        II provides a localized mark and memory of recent transcriptional activity. Mol. Cell <span class="emphasis"><em>11</em></span>, 709&#8211;719.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12667453&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(03)00092-3" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Peterson, C.L., and Laniel, M.A. (2004). Histones and histone modifications. Curr. Biol. <span class="emphasis"><em>14</em></span>, R546&#8211;R551.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15268870&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.07.007" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Reddy, K.C., and Villeneuve, A.M. (2004). <span class="emphasis"><em>C. elegans</em></span> HIM-17 links chromatin modification and competence for initiation of meiotic recombination. Cell <span class="emphasis"><em>118</em></span>, 439&#8211;452.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15315757&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2004.07.026" target="_blank">Article</a>
                     
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Reinke, V. Smith, H.E., Nance, J., Wang, J., Van Doren, C., Begley, R., Jones, S.J.M., Davis, E.B., Scherer, S., Ward, S.,
                        and Kim, S.K. (2000). A global expression profile of germline gene expression in <span class="emphasis"><em>C. elegans</em></span>. Mol. Cell <span class="emphasis"><em>6</em></span>, 605&#8211;616.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11030340&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1097-2765(00)00059-9" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Reuben, M., and Lin, R. (2002). Germline X chromosomes exhibit contrasting patterns of histone H3 methylation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. 245:71-82.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11969256&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/dbio.2002.0634" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Schaner, C.E., Deshpande, G., Schedl, P.D., and Kelly, W.G. (2003). A conserved chromatin architecture marks and maintains
                        the restricted germ cell lineage in worms and flies. Dev. Cell <span class="emphasis"><em>5</em></span>, 747&#8211;757.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14602075&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S1534-5807(03)00327-7" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Seydoux, G., and Dunn, M.A. (1997). Transcriptionally repressed germ cells lack a subpopulation of phosphorylated RNA polymerase
                        II in early embryos of <span class="emphasis"><em>Caenorhabditis elegans</em></span> and <span class="emphasis"><em>Drosophila melanogaster</em></span>. Development <span class="emphasis"><em>124</em></span>, 2191&#8211;2201.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9187145&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Seydoux, G., Mello, C.C., Pettitt, J., Wood, W.B., Priess, J.R., and Fire, A. (1996). Repression of gene expression in the
                        embryonic germ lineage of <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>382</em></span>, 713&#8211;716.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8751441&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/382713a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Shi, Y., Lan, F., Matson, C., Mulligan, P., Whetstine, J.R., Cole, P.A., Casero, R.A., and Shi, Y. (2004). Histone demethylation
                        mediated by the nuclear amine oxidase homolog LSD1. Cell <span class="emphasis"><em>119</em></span>, 941&#8211;953.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15620353&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2004.12.012" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Shiu, P.K., Raju, N.B., Zickler, D., and Metzenberg, R.L. (2001). Meiotic silencing by unpaired DNA. Cell <span class="emphasis"><em>107</em></span>, 905&#8211;916.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11779466&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(01)00609-2" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Simpson, V.J., Johnson, T.E., and Hammen, R.F. (1986). <span class="emphasis"><em>Caenorhabditis elegans</em></span> DNA does not contain 5-methylcytosine at any time during development. Nucleic Acids Res. <span class="emphasis"><em>14</em></span>, 6711&#8211;6719.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3748820&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sims, R.J., Nishiola, K.,and Reinberg, D. (2003). Histone lysine methylation: a signature for chromatin function. Trends Genet.
                        <span class="emphasis"><em>19</em></span>, 629&#8211;639.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14585615&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.tig.2003.09.007" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Subramaniam, K., and Seydoux, G. (1999). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=nos-1" target="_blank">nos-1</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=nos-2" target="_blank">nos-2</a></em></span>, two genes related to <span class="emphasis"><em>Drosophila nanos</em></span>, regulate primordial germ cell development and survival in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Development <span class="emphasis"><em>126</em></span>, 4861&#8211;4871.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10518502&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Tabara, H., Sarkissian, M., Kelly, W.G., Fleenor, J., Grishok, A., Timmons, L., Fire, A., and Mello, C.C. (1999). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rde-1" target="_blank">rde-1</a></em></span> gene, RNA interference, and transposon silencing in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>99</em></span>, 123&#8211;132.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12110183&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(00)81644-X" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Unhavaithaya, Y., Shin, T.H., Miliaras, N., Lee, J., Oyama, T., and Mello, C.C. (2002). <span class="emphasis"><em><a href="http://www.wormbase.org/db/seq/protein?name=MEP-1;class=protein" target="_blank">MEP-1</a></em></span> and a homolog of the NURD complex component Mi-2 act together to maintain germline-soma distinctions in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>111</em></span>, 991&#8211;1002.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12507426&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(02)01202-3" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Volpe, T.A., Kidner, C., Hall, I.M., Teng, G., Grewal, S.I., and Martienssen, R.A. (2002). Regulation of heterochromatic silencing
                        and histone H3 lysine-9 methylation by RNAi. Science <span class="emphasis"><em>297</em></span>, 1833&#8211;1837.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12193640&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.1074973" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Walstrom, K.M., Schmidt, D., Bean, C.J., and Kelly, W.G. (2005). RNA helicase A is important for germline transcriptional
                        control, proliferation, and meiosis in <span class="emphasis"><em>C. elegans</em></span>. Mech. Dev. <span class="emphasis"><em>122</em></span>, 707&#8211;720.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15817227&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.mod.2004.12.002" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Xu, L., Fong, Y., and Strome, S. (2001). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> maternal-effect sterile proteins, <span class="emphasis"><em><a href="http://www.wormbase.org/db/seq/protein?name=MES-2;class=protein" target="_blank">MES-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/seq/protein?name=MES-3;class=protein" target="_blank">MES-3</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/seq/protein?name=MES-6;class=protein" target="_blank">MES-6</a></em></span>, are associated in a complex in embryos. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>98</em></span>, 5061&#8211;5066.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11320248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.081016198" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Zhang, F., Barboric, M., Blackwell, T.K., and Peterlin, B.M. (2003). A model of repression: CTD analogs and <span class="emphasis"><em><a href="http://www.wormbase.org/db/seq/protein?name=PIE-1;class=protein" target="_blank">PIE-1</a></em></span> inhibit transcriptional elongation by P-TEFb. Genes Dev. <span class="emphasis"><em>17</em></span>, 748&#8211;758.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12651893&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/gad.1068203" target="_blank">Article</a>
                     	
                  </p>
               </div>
            </div>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Judith Kimble and Susan Strome. Last revised March 25, 2009. Published January 24, 2006. This chapter should be
                  cited as: Schaner, C. E. and Kelly, W. G. Germline chromatin (January 24, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.73.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p>
                  <span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Christine E. Schaner and William G. Kelly. This is an open-access article distributed under the terms of the Creative
                  Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original
                  author and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e34" href="#d0e34">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:bkelly@biology.emory.edu" target="_top">bkelly@biology.emory.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>

            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>
