<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
   
      <title>Behavior
         
      </title>
      <meta name="generator" content="DocBook XSL Stylesheets V1.67.2">
      <link rel="start" href="#WMBehavior_ArticleID" title="Behavior&#xA;   ">
      <link rel="next" href="#sec1" title="1.&nbsp;Introduction">
      <!--#include virtual="/ssi/header_article.html" -->
   </head>
   <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
<!--#include virtual="/ssi/boilerplate.html" -->
	<div class="html_to_pdf_link">
		<a href="behavior.pdf">Download PDF version</a>
	</div>
      <div class="article" lang="en">
         <div class="titlepage">
            <div>
               <div>
                  <h1 class="title"><a name="WMBehavior_ArticleID"></a>Behavior<sup><a name="d0e8" href="#ftn.d0e8">*</a></sup>
                        
                  </h1>
               </div>
               <div>
                  <div class="author"><span class="author"><span class="othername">Anne C. Hart, ed.<em><span class="remark">
                                                      <sup><a name="d0e33" href="#ftn.d0e33">&sect;</a></sup>
                                                    </span></em>
                                          </span><br></span><span class="affiliation"><span class="address">Massachusetts General Hospital  Cancer Center, Charlestown, MA 02129 USA
                           <p></p></span></span></div>
               </div>
            </div>
            <hr>
         </div>
         <div class="toc">

	   <div id="section_locator"><a href="/toc_wormmethods.html">
	         <b>This chapter is in WormBook section:</b><br />
		       > WormMethods
		</a></div><hr />
<a href="http://www.wormbase.org/forums/index.php?board=287.0" title="Discuss WormBook" target="_blank"><font color="#9932cc"><b>View/Add Comments</b></font></a><br/>
<IFRAME  SRC="http://tazendra.caltech.edu/~acedb/cgi-bin/wormbook_comment.cgi?name=Behavior" HEIGHT=50 WIDTH=180 FRAMEBORDER=0 SCROLLING=NO allowTransparency="true"></IFRAME>

<hr />

            <p><b>Table of Contents</b></p>
            <dl>
               <dt><span class="sect1"><a href="#sec1">1. Introduction</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec1_1">1.1. Considerations for behavioral assays</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_2">1.2. Controls for behavioral assays</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_3">1.3. Feeding status and cultivation conditions</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_4">1.4. Ambient conditions and chemicals</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_5">1.5. Animals to assay</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_6">1.6. Scoring behavior</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_7">1.7. Statistical analysis and reporting results</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_8">1.8. Confirming results</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_9">1.9. Transgenic strains</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_10">1.10. RNAi</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_11">1.11. Summary</a></span></dt>
                     <dt><span class="sect2"><a href="#sec1_12">1.12. Acknowledgements</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec2">2. Mechanosensation</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec2_1">2.1. Gentle touch to the body</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_2">2.2. Harsh touch to the body</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_3">2.3. Precipice response</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_4">2.4. Nictation</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_5">2.5. Head withdrawal and foraging</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_6">2.6. Tap reflex and habituation to tap (including tap reflex).</a></span></dt>
                     <dt><span class="sect2"><a href="#sec2_7">2.7. Nose touch</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec3">3. Osmotic avoidance</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec3_1">3.1. Osmotic ring assay</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec4">4. Chemosensation</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec4_1">4.1. Chemotaxis, chemoaversion and plasticity-quadrant plate, version 1</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4_2">4.2. Chemotaxis and chemoaversion-quadrant assay, version 2</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4_3">4.3. Drop assay</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4_4">4.4. Chemotaxis to volatile point source</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4_5">4.5. Distinguishing between odorants during cross-saturation</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4_6">4.6. Decreased response after chronic exposure to attractant</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4_7">4.7. Oxygen sensing</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4_8">4.8. Polyunsaturated fatty acids avoidance; liquid drop assay</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4_9">4.9. Males attraction to hermaphrodites</a></span></dt>
                     <dt><span class="sect2"><a href="#sec4_10">4.10. Interaction of two chemosensory stimuli (interaction assay)</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec5">5. Learning, adaptation and habituation</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec5_1">5.1. Learning, habituation and adaptation to tap</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_2">5.2. State-dependent adaptation to volatile odorants</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_3">5.3. Ethanol intoxication</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_4">5.4. Attraction to chemicals previously paired with attractive stimuli and avoidance of chemicals previously paired with starvation</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_5">5.5. Chemotaxis to soluble point source</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_6">5.6. Plasticity in chemotaxis using NaCl</a></span></dt>
                     <dt><span class="sect2"><a href="#sec5_7">5.7. Context dependency in olfactory adaptation</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec6">6. Thermal responses</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec6_1">6.1. Thermotaxis-tracking themoclines</a></span></dt>
                     <dt><span class="sect2"><a href="#sec6_2">6.2. Microdroplet assay for thermotaxis</a></span></dt>
                     <dt><span class="sect2"><a href="#sec6_3">6.3. Linear thermal gradient assay for thermotaxis assay</a></span></dt>
                     <dt><span class="sect2"><a href="#sec6_4">6.4. Point source heat avoidance</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec7">7. Locomotion</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec7_1">7.1. Quantitation of locomotion rate by counting body bends</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7_2">7.2. Decreased locomotion on food</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7_3">7.3. Bordering at edges of bacterial lawns</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7_4">7.4. Spontaneous reversals</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7_5">7.5. Ivermectin sensitivity assays</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7_6">7.6. Pirouettes</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7_7">7.7. Dauer lethargy and molting lethargy</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7_8">7.8. Duration and velocity of spontaneous forward or backward locomotion</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7_9">7.9. Suppression of head oscillations during backwards locomotion</a></span></dt>
                     <dt><span class="sect2"><a href="#sec7_10">7.10. Immobilization in serotonin (liquid)</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec8">8. Feeding</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec8_1">8.1. Electropharyngeogram recordings</a></span></dt>
                     <dt><span class="sect2"><a href="#sec8_2">8.2. Pharyngeal pumping rate</a></span></dt>
                     <dt><span class="sect2"><a href="#sec8_3">8.3. Foraging</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec9">9. Egg-laying, males and mating</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec9_1">9.1. Assessing defective or constitutive egg-laying</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9_2">9.2. Quantitation of constitutive egg-laying</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9_3">9.3. Modulation of egg-laying in response by bacteria</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9_4">9.4. Stimulation of egg-laying with serotonin and other drugs</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9_5">9.5. Quantification of egg-laying with chitinase assays</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9_6">9.6. Analyzing the temporal dynamics of egg-laying</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9_7">9.7. Male mating behaviors</a></span></dt>
                     <dt><span class="sect2"><a href="#sec9_8">9.8. Nicotine adaptation with or without levamisole induction</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec10">10. Assays of <span class="emphasis"><em>C. elegans</em></span> reproductive behaviors</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec10_1">10.1. Hermaphrodite sexual behavior</a></span></dt>
                     <dt><span class="sect2"><a href="#sec10_2">10.2. Sprinting assay</a></span></dt>
                     <dt><span class="sect2"><a href="#sec10_3">10.3. Sperm expulsion assay</a></span></dt>
                     <dt><span class="sect2"><a href="#sec10_4">10.4. Male sexual behavior</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec11">11. Defecation</a></span></dt>
               <dd>
                  <dl>
                     <dt><span class="sect2"><a href="#sec11_1">11.1. Assaying the defecation motor program (DMP)</a></span></dt>
                  </dl>
               </dd>
               <dt><span class="sect1"><a href="#sec12">12. Galvanotaxis</a></span></dt>
               <dt><span class="sect1"><a href="#sec13">13. Circadian rhythms</a></span></dt>
               <dt><span class="sect1"><a href="#sec14">14. References</a></span></dt>
            </dl>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec1"></a>1.&nbsp;Introduction
                     </h2>
                  </div>
               </div>
            </div>
            <p>Behavior reflects nervous system activity and is dependent on multiple factors including external stimuli, past experience,
               neuronal structure and changes in the internal milieu of the animal. Alterations at the cellular or functional level can profoundly
               alter basal and evoked activity. Therefore, behavioral assays offer the researcher simple, sensitive and powerful tools to
               interrogate neuronal function. The sensitivity of behavioral assays can also be their greatest weakness; the behavior of animals
               can be dramatically affected by small changes or variations in culture or assay conditions. In this short introduction, basic
               considerations for behavioral assays are very briefly examined. Both general considerations and admonitions relevant to <span class="emphasis"><em>C. elegans</em></span> behavioral assays are included. Specific protocols for behavioral assays contributed by individual researchers follow this
               more general introduction.
            </p>
            <p>New behavioral assays are constantly under development and old assays are often revised. Your suggestions, comments and new
               assays are welcome as this chapter and the appended protocols will be updated on a regular basis.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_1"></a>1.1.&nbsp;Considerations for behavioral assays
                        </h3>
                     </div>
                  </div>
               </div>
               <p>In the following section various parameters that contribute to behavioral response are described. Not every variable will
                  affect every behavior. But, when troubleshooting or designing an assay, these variables should at least be considered.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_2"></a>1.2.&nbsp;Controls for behavioral assays
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Controls: Clearly, controls are absolutely essential for all behavioral experiments. Because responses can vary from day to
                  day it is important for control assays with appropriate strains or animals to be run every day in parallel with experimental
                  animals. Both negative and positive controls should be tested daily. One way is to alternate control and experimental trials
                  to control for extraneous variables. For example, a wild type <span class="emphasis"><em>C. elegans</em></span> strain (N2) and a defective strain (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glr-1" target="_blank">glr-1</a></em></span>) should be tested for nose touch response in addition to any experimental strains under evaluation each day. These daily
                  controls are reported in the literature as control results along with the corresponding experimental results.
               </p>
               <p>The results of an experiment can be subtly or dramatically changed by the expectations or bias of the researcher observing
                  the behavior. Experimenter bias is usually subconscious and exists despite the best intentions of researchers; determined
                  vigilance as to appropriate controls is required. Clearly, this entails considerable effort in both experimental design and
                  daily effort. Ideally, all strains or animals will be scored in the behavioral assay by an observer ignorant as to the genotype,
                  treatment and/or expected outcome. Animals or strains should be renamed and assigned code numbers prior to the experiment
                  by another researcher to avoid bias in scoring. In some cases, only a subset of the animals is scored under these most rigorous
                  conditions. These results of these behavioral assays are subjected to statistical analysis independent of animals tested under
                  less rigorous conditions. If the results from the two groups are statistically equivalent, then all of the data can be pooled
                  for reporting in the literature at the discretion of the authors and with the concordance of the reviewers. Note that these
                  two groups of data may need to be reported independently in &#8220;Supplementary Information&#8221; for each assay.
               </p>
               <p>Control strains should be raised in parallel under identical conditions this can go beyond just raising animals on the same
                  batch of NGM plates with the same bacteria in the same box in the same incubator. Consider the controls required for behavioral
                  assessment of animals after laser surgery. &#8220;Mock ablated&#8221; animals of the same genotype and larval stage should also be anesthetized
                  on azide pads for the same time interval as &#8220;ablated&#8221; animals. Both &#8220;mock ablated&#8221; and &#8220;ablated&#8221; animals should be coded before
                  behavioral testing as individual animals, not as groups of animals. This coding avoids experimenter bias toward a specific
                  group of animals. Additional control animals (i.e. negative and positive controls) are also evaluated in parallel with the
                  coded ablated animals to confirm that assay conditions are appropriate. The animals should also be examined to confirm that
                  the laser microsurgery was successful prior to &#8220;decoding&#8221; the results.
               </p>
               <p>Selecting the correct control animals is crucial. In the evaluation of transgenic strains, the correct control is usually
                  not the N2 wild type strain nor is it &#8220;non-transgenic&#8221; animals from the same plate. A more appropriate control is the scoring
                  of several independent transgenic strains generated in parallel using 1) the same transgenic marker such as GFP or phenotypic
                  rescue, 2) the same genetic background, 3) an &#8220;empty&#8221; version of the promoter used to drive the transgene of interest and
                  4) that are generated by microinjection by the same experimenter. This helps decrease the corresponding problems of genetic
                  background, marker effects on behavior, promoter copy number, and relative gene dosage. &#8220;Non-transgenic&#8221; animals from the
                  same plate are rarely used as controls because of the mosaic nature of animals carrying extrachromosomal arrays. Unless a
                  cell-specific marker is used to mark a specific cell of interest, &#8220;non-transgenic&#8221; animals that do not express the transgenic
                  marker may still maintain the array and express the transgene under analysis in other behaviorally relevant cells. Strains
                  in different labs can suffer from genetic drift resulting in altered behavior. If a transgenic strain or mutant strain is
                  obtained from another laboratory, it is a good idea to obtain the corresponding background strain from the same laboratory.
               </p>
               <p>Behavioral assessments should be made by more than one observer, should be repeated several times, and should occur on more
                  than one day. Using more than one observer helps to avoid subconscious bias. Repeating the behavioral assay multiple times
                  and on more than one day ensures reproducibility and consistency. Alternatively, videotaping or other optical recording of
                  behavioral experiments permits multiple researchers to score the behavior.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_3"></a>1.3.&nbsp;Feeding status and cultivation conditions
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Cultivation: <span class="emphasis"><em>C. elegans</em></span> are generally raised on OP50 bacterial lawns on NGM agar plates. Altering the size of the lawn, the type of bacteria, the
                  agar, and the chemicals in the NGM can affect behavioral response. Unless carefully sealed, agar plates and their bacterial
                  lawns lose water to the atmosphere over time. Very dry culture plates should be avoided. Contamination on the culture plates
                  can also dramatically affect behavior. Animals from contaminated plates should not be used for behavioral assays; the short
                  cut of letting contaminated animals feed on uncontaminated plates prior to the behavioral assay is not recommended. As a general
                  rule, mold is less deleterious than yeast or bacterial contamination, but both should be avoided depending on the assay in
                  question. A simple bleaching protocol is sufficient to generate uncontaminated animals whose offspring can be used for behavioral
                  assays.
               </p>
               <p>Crowding and feeding status: The crowding and feeding status of animals can dramatically or subtly affect behavior. For example,
                  animals from crowded plates perform poorly in many avoidance assays. Animals clearly behave differently depending on their
                  feeding status and history. Animals that have been starved during development or animals that have not gone through the dauer
                  stage can have altered behavior compared to animals that have never been crowded. Feeding status also alters behavior. Animals
                  on the bacterial lawn respond differently in behavioral assays compared to animals off the lawn. Along the same lines, increasing
                  time off food alters behavior.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_4"></a>1.4.&nbsp;Ambient conditions and chemicals
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Ambient conditions: The conditions under which assays occur can also affect the results of behavioral assays. Possible environmental
                  factors that can be controlled include room humidity, temperature, drafts, and vibration. The dryness of assay plates can
                  shift response. Some behavioral assays are more sensitive to this than others. Osmolarity and spontaneous reversal rate assays
                  are quite sensitive to plate dryness. Depending on the assay, this can be controlled by pouring plates fresh daily, weighing
                  plates poured at equal volume, or running controls on the same plates.
               </p>
               <p>Ambient conditions (time of day, temperature and humidity) should always be recorded, so their effect can be considered when
                  evaluating data or troubleshooting.
               </p>
               <p>Some behavioral assays are run with plate lids on; some with lids off.</p>
               <p>The number of animals used in population assays can affect the outcome; this should also be standardized. For example, introducing
                  too many animals into the circular osmotic barrier used in the classical osmotic avoidance assays can result in crowding and
                  the escape of normally defective animals.
               </p>
               <p>Chemicals: The results of behavioral assays can be dramatically different depending on reagent freshness, purity and supplier.
                  Some chemicals must be made up fresh or diluted on the day of the assay. And, impurities in chemicals (e.g., switching from
                  a 99% pure to a 95% pure chemical) or changing suppliers can alter behavioral results. These caveats apply both to the reagents
                  used for cultivation of the animals and the reagents used in the actual behavioral assays.
               </p>
               <p>Assessing the effect of drugs and other pharmacologically active compounds on behavior involves several more variables. In
                  addition to the preceding admonitions, the stability of the drug, the effect of any solvent, batch-to-batch variation in activity,
                  and the actual concentration of the drug in the assay or assay plate should be considered. Control animals and plates should
                  be treated exactly in parallel. For example, if the drug is dissolved in ethanol and then diluted in water, then the control
                  animals or plates should be treated with the same diluted ethanol solution (lacking the drug). Note that increased osmolarity
                  in the behavioral assay due to inclusion of the drug may alter behavior in some assays.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_5"></a>1.5.&nbsp;Animals to assay
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Stage of life cycle: The sex, age and life cycle stage of the animal tested in an assay is an important factor for behavior.
                  Males are generally used only for assessment of male-specific behaviors. All animals demonstrate lethargic behavior during
                  each molt; staging of young adults is unambiguous so adult hermaphrodites are generally used in assays. The stage of the life
                  cycle can alter behavior in hermaphrodites although many assays can be used on larval animals. Nervous system development
                  is largely complete by the end of the L1 stage, but some developmental changes are clearly coordinated with the L4/adult molt.
               </p>
               <p>Behavioral changes occur during adult stages as well. Some assays are indiscriminate as to the age of the adult, but other
                  behavioral assays are more sensitive. The age of animals can be standardized by &#8220;hours after molt to adulthood&#8221;, by hours
                  since an egg was laid at a specified temperature, or by the number of eggs in the uterus. The former is preferable in most
                  cases. Animals can also become less responsive to some sensory stimuli after several days of adulthood. Some researchers run
                  behavioral assays only on animals from synchronized plates where adult hermaphrodites laid eggs over a two hour period (and
                  then the parents were removed) to keep animal age constant.
               </p>
               <p>The treatment of animals during the assay can also alter behavior. Most assays require that the animals be treated as gently
                  as possible. Transfer by mouth pipette seems to be the most gentle although this in not recommended as a general lab practice.
                  Gently moving animals with a pick is sufficient for most assays. If an individual animal is damaged in transfer and unable
                  to move properly, then it usually is excluded from the data set. But, the rules for exclusion and the number of animals excluded
                  should be reported.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_6"></a>1.6.&nbsp;Scoring behavior
                        </h3>
                     </div>
                  </div>
               </div>
               <p>To avoid ambiguity and inconsistency, it is critical to precisely define the criteria for scoring behavior both in the laboratory
                  and in publications. For example, initiating backward locomotion is generally called a reversal. But, some researchers define
                  a minimum distance the animal has to move backward to score as a <span class="emphasis"><em>bona fide</em></span> reversal; some do not. Some researchers include omega turns as reversals. And, some researchers count increased backward
                  locomotion as a reversal or response; most do not. A precise definition of the behavior to be scored is critical for analysis.
               </p>
               <p>It is useful to have a &#8220;scoring worksheet&#8221; to record results for each behavioral assay. In addition to the actual scoring
                  data, the ambient conditions, date, time, genotype, cultivation conditions, researcher name, number of animals scored, and
                  other variables should be recorded.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_7"></a>1.7.&nbsp;Statistical analysis and reporting results
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The number of animals assayed is determined by the assay. As a general rule, 4 large-scale population based assays or 30 individual
                  assays with corresponding controls should be considered a bare minimum. To achieve statistical significance, many more trials
                  may be required.
               </p>
               <p>Data from experimental strains should always be compared to data from control strains analyzed in parallel.</p>
               <p>Statistical analysis is critical for drawing conclusions from behavioral data. At a minimum, n and p values should be presented
                  for critical data. Additionally, standard errors of the mean or standard errors should be presented within figures and tables
                  to allow readers to assess significance.
               </p>
               <p>Reproducibility can be difficult for behavioral assays. Control strains may vary in their response from day to day. Statistically,
                  it may be tempting to normalize the data to the control results each day to decrease variation in the reported results. Although
                  this has been done in some cases in the literature, it is clearly more persuasive to present data that has not been normalized.
                  If normalized data is presented, then the non-normalized data should be presented in the corresponding on-line Supplementary
                  Information.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_8"></a>1.8.&nbsp;Confirming results
                        </h3>
                     </div>
                  </div>
               </div>
               <p>It is wise to confirm the phenotype of strains, sequence plasmids and carefully check other reagents upon arrival. To avoid
                  ambiguity, order crucial strains of <span class="emphasis"><em>C. elegans</em></span> from the <span class="emphasis"><em>C. elegans</em></span> Genetics Center (CGC) when possible. When they arrive, freeze multiple copies for long-term storage. Most <span class="emphasis"><em>C. elegans</em></span> laboratories that specialize in behavior have their own &#8220;horror stories&#8221; in which genetic drift due to continuous passaging
                  or mislabeling have wasted months of effort. It is a wise idea to thaw an aliquot of any crucial control strains at least
                  once a year (e.g., the wild type N2 strain). Some labs thaw bi-monthly.
               </p>
               <p>Background mutations: In addition to the mutant allele, <span class="emphasis"><em>C. elegans</em></span> strains often contain other changes in their DNA. Some of these are spontaneous and some are background mutations arise from
                  the original mutagenesis. And, because backcrossing is most effective for unlinked genes, the remaining background mutations
                  may be tightly linked to the mutant allele and gene of interest. Historically, one strategy to confirm that the gene under
                  examination causes the phenotype of interest is to create a trans-heterozygote (examining the behavior of allele A/allele
                  B). If the alleles do not complement in the trans-heterozygous animal, then the mutant phenotypes are likely due to alterations
                  in the function of gene X. Multiple, independently derived alleles of the gene of interest should also be examined if they
                  are available.
               </p>
               <p>Transgenic rescue of a mutant phenotype is the best indication that the behavioral phenotype observed is due to altered function
                  of the gene of interest. For example, cDNA rescue using a heat shock promoter and a transformation marker (described below)
                  may restore normal function to mutant animals. This phenotypic rescue by the cDNA should be confirmed by examining the behavior
                  of transgenic mutant animals that carry the transformation marker and the &#8220;empty&#8221; heat shock promoter construct. Alternatively,
                  a genomic rescue construct can be used to restore normal behavior, but a mutant version of the genomic rescue construct should
                  not restore behavior.
               </p>
               <p>Integrated transgenic arrays are caused by insertion of exogenous DNA in the chromosome. Therefore the behavioral phenotype
                  of strain carrying an integrated array can arise either from the chromosomal DNA break or from the transgenes on the integrated
                  array. If the behavior of animals carrying the extrachromosomal array is the same as the integrated array, then the behavioral
                  changes are due to the transgene itself.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_9"></a>1.9.&nbsp;Transgenic strains
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Transgenic strains: Some markers for generating transgenic <span class="emphasis"><em>C. elegans</em></span> are preferred for behavioral analysis. GFP or dsRed reporter constructs can be used as transgenesis markers although these
                  usually require the use of a fluorescent dissection microscope for strain maintenance. Historically, transgenic rescue of
                  an unrelated mutant phenotype has been broadly used. Commonly used markers include <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pha-1" target="_blank">pha-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-20" target="_blank">dpy-20</a></em></span>, or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span>. Injection of a dominant <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rol-6" target="_blank">rol-6</a></em></span> transgene is frequently used for transgenesis <span class="emphasis"><em>C. elegans</em></span> but the rolling phenotype induced by this transgene interferes with most behavioral assays.
               </p>
               <p>It is important to determine if the transgenic marker used will interfere with the assessment of the behavioral phenotype.
                  GFP or dsRed expression can have deleterious effects. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pha-1" target="_blank">pha-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-15" target="_blank">lin-15</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-20" target="_blank">dpy-20</a></em></span>, or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-119" target="_blank">unc-119</a></em></span> mutant alleles or even rescued transgenic animals are defective in some behavioral assays.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_10"></a>1.10.&nbsp;RNAi
                        </h3>
                     </div>
                  </div>
               </div>
               <p>RNAi knockdown of gene expression is a powerful approach to assess the role of specific genes in neuronal function and behavior.
                  Several approaches can be used with varying efficacy to knockdown gene expression with double stranded RNA in <span class="emphasis"><em>C. elegans</em></span>. The magnitude of the RNAi knockdown varies dramatically from gene to gene; this is an important consideration in interpreting
                  experiments.
               </p>
               <p>Feeding <span class="emphasis"><em>C. elegans</em></span> bacteria expressing dsRNA for the gene of interest works in some cases. The nervous system is relatively refractory to RNAi
                  compared to some other tissues, but useful results may still be obtained. Mutant strains with increased sensitivity to RNAi
                  are often used to increase efficacy of gene knockdown; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rrf-3" target="_blank">rrf-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=eri-1" target="_blank">eri-1</a></em></span> are commonly used. Of course the behavior of control animals raised on control bacterial feeding strains should be examined.
                  For example, as control for a nose touch response RNAi experiment, a bacterial strain expressing the dsRNA corresponding to
                  <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-10" target="_blank">osm-10</a></em></span> could be fed to <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rrf-3" target="_blank">rrf-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rrf-3" target="_blank">rrf-3</a></em></span>; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glr-1" target="_blank">glr-1</a></em></span> animals. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-10" target="_blank">osm-10</a></em></span> is expressed in the ASH neurons that are critical for nose touch response, but loss of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-10" target="_blank">osm-10</a></em></span> function does not perturb nose touch response. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rrf-3" target="_blank">rrf-3</a></em></span> animals raised on <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-10" target="_blank">osm-10</a></em></span> RNAi bacteria should robustly respond to nose touch; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=rrf-3" target="_blank">rrf-3</a></em></span>; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glr-1" target="_blank">glr-1</a></em></span> animals should not as <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glr-1" target="_blank">glr-1</a></em></span> is required for nose touch response.
               </p>
               <p><span class="emphasis"><em>C. elegans</em></span> promoters can also be used to express dsRNA in transgenic animals. Usually an inverted repeat of coding sequence from the
                  targeted gene is used for expression of dsRNA, but co-expression of sense and antisense sequences from two separate plasmids
                  can induce RNAi knockdown for some genes. Ubiquitous and inducible gene knockdown can be attempted using common heat shock
                  vector promoters. Cell-specific knockdown is sometimes also possible as dsRNA and RNAi effects spread poorly (if at all) between
                  neurons.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_11"></a>1.11.&nbsp;Summary
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. elegans</em></span> behavioral assays encompass a wide range of behaviors and approaches. When combined with current molecular biological, electrophysiological
                  and optical recording techniques, behavioral assays have been remarkably successful in assessing the contribution of genes
                  and specific neurons to behavior. Generous researchers who specialize in each technique have contributed the following protocols.
                  We hope that by providing a resource for the community, that behavioral studies in <span class="emphasis"><em>C. elegans</em></span> will be made easier, more accessible and clearer for neuroscience researchers in general.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec1_12"></a>1.12.&nbsp;Acknowledgements
                        </h3>
                     </div>
                  </div>
               </div>
               <p>The comments and suggestions about this introduction by John Satterlee and Catharine Rankin are appreciated.</p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec2"></a>2.&nbsp;Mechanosensation
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_1"></a>2.1.&nbsp;Gentle touch to the body
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Martin Chalfie, Columbia University, New York, NY, USA, March 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2_1_1"></a>2.1.1.&nbsp;General considerations
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>C. elegans</em></span> senses a variety of mechanical stimuli. These stimuli include gentle touch stimulus delivered to the body (<a href="#bib14">Chalfie and Sulston, 1981</a>; <a href="#bib74">Sulston et al., 1975</a>), harsh touch to the midbody (<a href="#bib84">Way and Chalfie, 1989</a>), harsh touch to the head or tail (<a href="#bib16">Chalfie and Wolinsky, 1990</a>), nose touch (<a href="#bib51">Kaplan and Horvitz, 1993</a>) and texture (<a href="#bib72">Sawin et al., 2000</a>). Gentle touch to the body is sensed by six sensory neurons (<a href="http://www.wormbase.org/db/get?name=ALML;class=Cell" target="_blank">ALML</a>/R, <a href="http://www.wormbase.org/db/get?name=PLML;class=Cell" target="_blank">PLML</a>/R, <a href="http://www.wormbase.org/db/get?name=AVM;class=Cell" target="_blank">AVM</a>, and <a href="http://www.wormbase.org/db/get?name=PVM;class=Cell" target="_blank">PVM</a>). An important feature of any assay for gentle touch mediated by these cells is that it not be confused with the response
                     to other mechanical signals. Thus, care should be taken to touch the animals in a way that will not stimulate other mechanosensory
                     neurons. In practice one should avoid too harsh a stimulus and not touch the animals too near the tip of the head or tail.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2_1_2"></a>2.1.2.&nbsp;Qualitative assays for touch sensitivity
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2_1_2_1"></a>2.1.2.1.&nbsp;Stroking with an eyebrow hair
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>The initial and most generally used method to test for gentle touch sensitivity is to stroke the animals with an eyebrow hair
                        that has been glued to the end of a toothpick (<a href="#bib14">Chalfie and Sulston, 1981</a>; <a href="#figure1">Figure 1</a>). Eyebrow hairs are used because they are usually not cut and are, thus, finely tapered. They also can be obtained with a
                        minimum of discomfort. Some papers stated that the hairs are from eyelashes. I find, however, that removing eyelashes is a
                        much more painful process. Various glues can be used to secure the hairs, but the hair should be placed so that its shaft
                        extends straight from the end of the toothpick and does not curve away from the tip.
                     </p>
                     <p>The hair is sterilized by dipping it into a 70% ethanol solution and drying by shaking (don't flame the hair). Sometimes bacteria
                        accumulate on the hair; they can be removed by poking the hair into the agar on a spare plate.
                     </p>
                     <p>Animals are touched by stroking the hair across the body just behind the pharynx (for the anterior touch response) or just
                        before the anus (for the posterior touch response; see <a href="#figure1">Figure 1</a>). Actually touching the animals in any position along the touch receptor processes will generate a response. Touching the
                        animals near the middle of the body (near the vulva in adult hermaphrodites), however, yields ambiguous results because both
                        the anterior and posterior touch circuits can be activated. The animals should not be poked with the end of the hair, since
                        this provides a stronger stimulus and can sometimes evoke a response, even in touch-insensitive animals. Similarly, animals
                        should not be touched with the end of a platinum worm pick. Animals should not be touched at either the tip of the nose or
                        the tip of the tail as even animals lacking the six touch sensing cells often respond.
                     </p>
                     <p>The touch assay is really a differential assay, since animals that are dead or paralyzed will also not respond to the stroking.
                        Routinely, animals are said to be touch-sensitive if they respond to stroking with the eyebrow hair by stopping movement toward
                        the hair or move away from the hair (sometimes the touch stops moving animals without having them reverse their direction
                        of movement). Touch-insensitive (Mec) animals fail to respond to the hair, but do respond to prodding with a worm pick. A
                        partial response is one in which the animals move away from only some of the touches.
                     </p>
                     <p>Touch mediated by the anterior touch cells can be assayed in severely uncoordinated animals by monitoring pharyngeal pumping
                        (<a href="#bib15">Chalfie et al., 1985</a>; M. Chalfie, unpub. data). Touching animals in the anterior slow the rate of pharyngeal pumping. Another output of the touch
                        circuitry is the defecation rhythm, which is reset by stroking the anterior of the animal with the eyebrow hair and is dependent
                        on genes needed for the function of the touch receptor (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-4" target="_blank">mec-4</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-5" target="_blank">mec-5</a></em></span>, and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-9" target="_blank">mec-9</a></em></span>; <a href="#bib76">Thomas, 1990</a>).
                     </p>
                     <div class="literallayout">
                        <p></p>
                     </div>
                     <div class="mediaobject" align="center"><a name="figure1"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="WMBehavFig1.jpg"><img src="WMBehavFig1_s.jpg" border="2" align="middle" alt="WormBook Behavior figure 1"></a>
  </td>
</tr>
                        </table>
                        <div class="caption">
                           <p><b>Figure 1. Using an eyebrow hair to test gentle touch sensitivity.&nbsp;</b>(A) Positioning (top panel) and gluing (bottom panel) the eyebrow hair to the tooth pick. The thickened black line indicates
                              the shaft of the hair; the gray area indicates the location of the glue. (B) Animals should be touched by stroking the hair
                              across the body at the positions of the arrows. The six touch receptor neurons are indicated.
                           </p>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2_1_2_2"></a>2.1.2.2.&nbsp;Tapping the plate
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Wild-type animals will move (adults usually reverse direction) in response to their plate being tapped (<a href="#bib14">Chalfie and Sulston, 1981</a>). This stimulus often happens when plates are placed on the stage of a dissecting microscope. Touch-insensitive animals do
                        not respond to this tapping. Although not as accurate a measure of touch sensitivity as touching with an eyebrow hair, this
                        is a rapid assay that has been used to screen for touch-insensitive mutants (<a href="#bib13">Chalfie and Au, 1989</a>). Plates with F2 progeny of mutagenized parents were dropped from about 1-2 cm above the stage of the microscope and then
                        examined for animals that did not move. Candidate touch mutants were then tested with the eyebrow hair and worm pick to determine
                        if they were touch insensitive. Cathy Rankin and coworkers designed an electric tapper to deliver a taps at defined intervals
                        to plates for their studies examining the touch circuitry and habituation of the touch response (<a href="#bib64">Rankin, 1991</a>; <a href="#bib87">Wicks and Rankin, 1995</a>).
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2_1_3"></a>2.1.3.&nbsp;Quantitative assays for touch sensitivity
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The above assays treat touch sensitivity as an all-or-none phenomenon. Often, however, animals give a partial response to
                     the touch stimulus. In addition, even touch-insensitive animals sometimes respond to the first touch. Two general methods
                     provide a more quantitative measure of the touch response: counting the responses to multiple touches and touching with defined
                     stimuli.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2_1_3_1"></a>2.1.3.1.&nbsp;Scoring the result of multiple touches
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>The first instance of using multiple touches with an eyebrow hair to measure the degree of insensitivity to touch was by <a href="#bib93">Hobert et al. (1999)</a>. In this assay animals are touched ten times (alternating head and tail touches) and a score is giving for the number of
                        positive responses. At least thirty animals are examined and mean percentage score is obtained. Since individual touches are
                        not always the same, the values obtained in this assay are not truly quantitative. Nonetheless, by using multiple animals
                        relatively subtle difference in touch sensitivity can be revealed (e.g., <a href="#bib93">Hobert et al., 1999</a>; <a href="#bib95">Zhang and Chalfie, 2002</a>). In addition, by separately scoring head and tail responses differences between the two can also be uncovered (<a href="#bib94">Zhang et al., 2004</a>). In using this method one should be careful to distinguish between animals that habituate more rapidly than wild type from
                        those that respond less frequently to the touch stimulus. The animals with the former defect should show a response pattern
                        in which the animals respond less frequently to successive touches; animals with the latter defect should show no pattern
                        to the failures. The experimenter should also determine whether the animals are responding equally (in terms of number of
                        responses) to touches in the head or tail.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec2_1_3_2"></a>2.1.3.2.&nbsp;Using worm von Frey hairs
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>von Frey hairs have been used clinically for over a century to test human touch sensitivity. They consist of a series of flexible
                        fibers that are touched end-on to skin and bend once a specific force has been applied. The longer and thinner the fiber,
                        the less force is required to bend it. These calibrated fibers are then used to determine the forces needed to provoke a touch
                        sensation when placed on the skin. We have adapted this method to <span class="emphasis"><em>C. elegans</em></span> by using fine monofilament nylon sutures (7-0 and smaller; Iris Chin, Miriam Goodman, and M. Chalfie, unpub. data). A short
                        length of suture (Sharpoint, Surgical Specialties, Reading, PA) is attached perpendicularly to the end of a glass capillary
                        (100<span class="symbol">&#956;</span>L volume) with epoxy and calibrated using an analytical balance. The force (in <span class="symbol">&#956;</span>Newtons) required for bending is calculated as the weight in (mg) found for the bent fiber X 9.8 m/s2. A limitation of this
                        method is that fibers that deliver less than 10 <span class="symbol">&#956;</span>N of force are difficult to use because they bend with the air currents normally found in the lab.
                     </p>
                     <p>Typically, we mount the capillary on a manual micromanipulator with the fiber perpendicular to the agar surface, position
                        a worm underneath the fiber, and make contact by moving the z-axis of the micromanipulator. Animals are touched in the same
                        locations as described above for testing body touch with eyebrow hairs, and the response is noted. Ninety percent of wild-type
                        hermaphrodites respond to a 10 <span class="symbol">&#956;</span>N of force; virtually all animals respond to forces <span class="symbol">&gt;</span>100 <span class="symbol">&#956;</span>N. In contrast, hermaphrodites homozygous for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-3" target="_blank">mec-3</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-6" target="_blank">mec-6</a></em></span> null mutations fail to respond to a 100 <span class="symbol">&#956;</span>N force. Approximately 10% of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-4" target="_blank">mec-4</a></em></span><span class="emphasis"><em> (<a href="http://www.wormbase.org/db/get?name=u253;class=Variation" target="_blank">u253</a>)</em></span> animals respond to a 100 <span class="symbol">&#956;</span>N force (the difference may reflect some activity from the <a href="http://www.wormbase.org/db/get?name=PVD;class=Cell" target="_blank">PVD</a> cells, which are defective in <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-3" target="_blank">mec-3</a></em></span> animals).
                     </p>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_2"></a>2.2.&nbsp;Harsh touch to the body
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Extracted from the literature. Comments from researchers who use this assay would be appreciated.</p>
               <p>Harsh touch to the body is measured by prodding animals with a platinum wire in the midsection of the body (<a href="#bib14">Chalfie and Sulston, 1981</a>; <a href="#bib84">Way and Chalfie, 1989</a>). Nonmoving gravid adults are prodded at or just posterior to the vulva. Animals respond by initiating locomotion, usually
                  by backing up. Animals to be assayed should be grown in the continuous presence of food. Animals who are starved and animals
                  that have passed through the dauer stage often fail to respond to harsh touch regardless of the functional status of the <a href="http://www.wormbase.org/db/get?name=PVD;class=Cell" target="_blank">PVD</a> sensory neurons.
               </p>
               <p>Harsh touch is assessed in animals in which the function of the ALM and PLM neurons have been perturbed as these neurons mediate
                  response to both gentle and harsh touch. Response to harsh touch is mediated by the <a href="http://www.wormbase.org/db/get?name=PVD;class=Cell" target="_blank">PVD</a> sensory neurons and perhaps the FLP sensory neurons. The phenotype is called &#8220;touch aberrant&#8221; (abbreviated Tab) to distinguish
                  it from &#8220;Mechanosensory defective&#8221; (abbreviated Mec) animals. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-3" target="_blank">mec-3</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-86" target="_blank">unc-86</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-32" target="_blank">lin-32</a></em></span> animals are Mec and Tab; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-4" target="_blank">mec-4</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-2" target="_blank">mec-2</a></em></span> and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-6" target="_blank">mec-6</a></em></span> animals are nonTab and Mec (respond to harsh, but not gentle touch).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_3"></a>2.3.&nbsp;Precipice response
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Although the animals are reported to avoid the edge of an agar chunk, little is known about this behavior.</p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_4"></a>2.4.&nbsp;Nictation
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. elegans</em></span> dauers can occasionally be found flailing at the top of mold filament independent of plate orientation. (<a href="#bib21">Croll and Matthews, 1977</a>)  An overtly similar behavior called nictation is reported to play a role in the dispersal of parasitic nematodes. An assay
                  for this behavior has not been developed.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_5"></a>2.5.&nbsp;Head withdrawal and foraging
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Anne Hart, Massachusetts General Hospital and Harvard Medical School, Boston, MA USA, March 2005.</p>
               <p>The nose of <span class="emphasis"><em>C. elegans</em></span> moves in a rhythmic dorsal/ventral pattern as animals feed on a standard bacterial lawn. (<span class="emphasis"><em>C. elegans</em></span> lay on their side on agar plates.) This rhythmic nose motion is called foraging in this context. Aberrant foraging rates
                  and patterns of movement can be observed in some strains/animals. The rate of basal foraging is dependent on the RMD motorneurons
                  and on <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glr-1" target="_blank">glr-1</a></em></span> function (<a href="#bib39">Hart et al., 1995</a>).
               </p>
               <p>Animals respond to touch to the side of the nose by rapidly moving their nose away from the stimulus; this is called <span class="underline">head withdrawal</span>. Response is again mediated by the RMD motorneurons and requires <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glr-1" target="_blank">glr-1</a></em></span> function. (<a href="#bib39">Hart et al., 1995</a>)  A hair-similar in size to the hair used in the nose touch assay, is placed on the agar plate so that the side of the animal's
                  nose will just barely touch the hair at the maximal extent of the foraging motion. Animals respond by rapidly moving the nose
                  away from the hair. Significant practice is required learn where to lay down the hair and to learn to discriminate between
                  the rapid withdrawal motion <span class="emphasis"><em>versus</em></span> normal foraging motion. Foraging and head withdrawal assays should be undertaken by observers blind as to genotype or treatment.
               </p>
               <p>Thin bacterial lawns for these assays resemble those used in nose touch assays. They can be prepared by releasing a single
                  drop of an overnight OOP50 culture over a standard NGM plate. Allow the liquid to soak into the plate. To prevent bacterial
                  growth and too much thickening of the bacterial lawn, the plate should be used within a few hours or should be sealed with
                  Parafilm and stored at 4 degrees for weeks. Allow stored plates to return to room temperature before use.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_6"></a>2.6.&nbsp;Tap reflex and habituation to tap (including tap reflex).
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Catharine Rankin, University of British Columbia, Vancouver, BC, Canada, March 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2_6_1"></a>2.6.1.&nbsp;Tap reflex
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The tap-withdrawal protocol measures the responses of a worm to a single tap or a series of successive taps (trains of taps),
                     given to the side of a 4 cm Petri plate filled with 10 ml of NGM agar. The taps/trains cause the worm's environment to vibrate
                     briefly, and this causes the adult worm to swim backwards for some distance (termed a reversal). This technique produces a
                     quantifiable measure of the magnitude of this reversal response. For this assay worms are placed on the center of a Petri
                     plate and are videotaped through the lens of a dissecting microscope. The mechanical tapper is arranged to hit the center
                     of one side of the plate. The tapper is composed of an electromagnetic relay run by a stimulator. (<a href="#bib65">Rankin et al., 1990</a>)  The tapper delvers a force of approximately 1 to 2 Newtons per tap which are transmitted through the dish and the agar
                     to stimulate the worm. These experiments are usually run with the lids off of the plates while taps are being delivered. The
                     effects of drugs on the tap response can be assessed by adding the drug plus vehicle to the agar and comparing the results
                     to a group that was tested with just the vehicle for the drug. The length of the reversal to tap can be affected by several
                     variables: 1) The intensity of the tap; Chiba and Rankin (<a href="#bib18">Chiba and Rankin, 1990</a>) showed that the tap response is graded and that stronger stimuli produce larger reversals. 2) The age of the worm; Chiba
                     and Rankin (<a href="#bib18">Chiba and Rankin, 1990</a>), showed that as worms got larger so too did reversals. 3) The agar in the plates; as agar plates age they dry out and lead
                     to differences in the ease with which worms can move. This can be controlled for by only using plates within a certain age
                     range, or by weighing plates prior to use. 4) The reversal response can also be altered by the ambient temperature and humidity
                     in the room where the testing is done. This is probably the result of changes in the agar and ease of movement; it is an important
                     variable to control. At the very least, experimental and control animals must always be run together, on the same day, at
                     about the same time under all of the same conditions. By taking note of temperature and humidity conditions for each animal
                     it will be possible to determine whether variations in either variable affected the outcome of the experiment.
                  </p>
                  <p>Prior to undertaking tap reflex experiments, it would be advisable to also read the protocols in the <span class="emphasis"><em><a href="#sec5" title="5.&nbsp;Learning, adaptation and habituation">Learning, Adaptation and Habituation</a></em></span> section on the tap reflex.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec2_6_2"></a>2.6.2.&nbsp;Scoring responses to tap
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Once the experiment has been run and videotaped, then the behavior must be scored. Using stop-frame video analysis, each worm's
                     behavior is scored by tracing its body from the videosimage onto a transparent acetate sheet on the video monitor, going frame
                     by frame through the tape. The adult worm will usually (90% to 95% of the time) respond to the tap with a reversal response,
                     with the size and number of the responses decreasing as stimuli are repeated through the habituation run. In a typical reversal
                     response, a stopped or forward-moving worm moves backwards for a distance (usually less than 1 or 2 worm lengths) and then
                     either remains still or re-initiates forward movement in a new direction. To score the response, a person notes the pre-tap
                     position of the worm, and then traces the total distance the worm reversed (i.e. track length) onto the acetate sheet. Because
                     the head of the worm often makes small side-to12 side foraging movements, reversals are most accurately scored by tracing
                     the path of the tail. In order to have consensus a scoring manual should be developed with rules for scoring variations from
                     this behavior (i.e. the reversal must occur within 1 second of the tap, a pause is scored as 0 backward movement, if the worm
                     is already swimming backwards when the tap is delivered this is a &#8220;missing data point&#8221; etc). In some cases, a worm will respond
                     with acceleration forward rather than a reversal. In a typical acceleration response, a stopped or forward-moving worm either
                     initiates forward movement or increases its speed, respectively. Generally, in experiments with adult wild-type worms, these
                     acceleration responses are simply scored as missing data points (an adult worm will accelerate to tap only about 5% of the
                     time). Under other circumstances or protocols, such as with larval, mutant or laser-ablated animals, the acceleration response
                     occurs more often and is therefore scored. If a worm accelerates after the tap, the acceleration is scored by measuring the
                     distance the worm moves (i.e. track length) during the 1-sec interval before the tap and subtracting it from the distance
                     the worm moves during the 1-sec interval after the tap. If the measurement for the second after the tap is <span class="symbol">&#8805;</span>1.75 X the measurement for the second before the tap then the response is considered an acceleration. It is important to note
                     that the reversal response and the acceleration response are two qualitatively different outcomes, which cannot easily be
                     compared. These &#8220;scored&#8221; tracings are then scanned into a computer and the length of each is measured using NIH image. The
                     values from this program, representing the reversal/acceleration magnitude, can then be directly transported into a statistical
                     package for data analysis. The frequency of reversals and accelerations is also an important measure of the behavior of the
                     worm, and are scored simply by counting the number of instances of either response.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec2_7"></a>2.7.&nbsp;Nose touch
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Anne Hart, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA, March 2005.</p>
               <p><span class="emphasis"><em>C. elegans</em></span> responds to gentle touch to the nose by initiating backward locomotion. (<a href="#bib51">Kaplan and Horvitz, 1993</a>)  This response is the &#8220;nose touch response&#8221; or Not assay. Response to nose touch is mediated by the ASH, OLQ and FLP sensory
                  neurons. The two polymodal ASH sensory neurons mediated 60% of the wild type response to nose touch. Their ciliated, sensory
                  endings are located at the tip of the nose where they detect light touch to the nose, high osmolarity (<a href="#bib38">Hart et al., 1999</a>) and volatile repellent chemicals (<a href="#bib78">Troemel, 1999</a>).
               </p>
               <p>Nose touch is assayed by laying a hair on the surface of the plate in front of the animal. As an animal moves forward, it
                  contacts the hair with the tip of the nose perpendicular to the direction of movement. Some practice is required to anticipate
                  where the hair must be laid down for an animal to run into at 90 degrees. Normal animals immediately initiate backward locomotion.
                  Defective animals either hurdle over the hair or slide their nose along the hair. An individual animal is tested no more than
                  10 times in a row to avoid inducing habituation. You can drop the assay plate to induce lethargic animals to move or reverse
                  direction, but you cannot touch the animal to facilitate the assay.
               </p>
               <p>Nose touch can be assayed either on very thin bacterial lawns or off food. Response on food is more robust. Animals that are
                  moved to thin lawns recover within 10 minutes and respond in roughly 90% of trials. Response by animals who have wandered
                  off the bacterial lawn is slightly diminished; animals respond in roughly 80% of trials. Response is dramatically reduced
                  in animals who are physically moved by the investigator to a plate lacking food; animals respond in roughly 30% of trials
                  (<a href="#bib17">Chao et al., 2004</a>).
               </p>
               <p>The hair should be wiped off every few trials to eliminate bacteria and water that may adhere. Normal animals will ignore
                  a wet or slimy hair. A folded &#8220;Kimwipe&#8221; can be used or clean bare fingers. In the latter case, avoid using lotions, creams
                  or other surfactants. Don't flame the hair. Individual hairs can be reused used for weeks.
               </p>
               <p>The hair used in the Not assay must be of an appropriate size. Too thick will result in all animals responding; too thin will
                  cause even animals of mutant genotype to respond. Hair thickness has definite variation between individuals; arm hair from
                  a female researcher or an eye lash from a male researcher would be a good place to start. The root of the hair is taped to
                  the small end of a glass Pasteur pipette or to a thin wooden stick. Many hospitals stock cotton swabs on thin wooden sticks
                  that are ideal. Slightly thicker hairs are required for nose touch assays on food. Under the microscope it is clear that each
                  hair tapers to a very fine tip. Generally the very end of a hair is too thin and the animal must impact a thicker (and less
                  distal) part of the hair in the nose touch trial.
               </p>
               <p>Thin bacterial lawns for nose touch assays can be prepared by releasing a single drop of an overnight OOP50 culture over a
                  standard NGM plate. Allow the liquid to soak into the plate. To prevent bacterial growth and too much thickening of the bacterial
                  lawn, the plate should be used within a few hours or should be sealed with Parafilm and stored at 4 degrees for weeks. Allow
                  stored plates to return to room temperature before use. If the bacterial lawn is too thick, then the animals may crawl through
                  the food over or under the hair and avoid the nose touch stimulus.
               </p>
               <p>Each hair is tested by comparing the response of N2 and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glr-1" target="_blank">glr-1</a></em></span> animals. (<a href="#bib39">Hart et al., 1995</a>). N2 animals should respond robustly; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=glr-1" target="_blank">glr-1</a></em></span> animals should fail in 90% of trials. (Note that <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=eat-4" target="_blank">eat-4</a></em></span> animals are so defective in their response to nose touch that they are not useful as a control. <a href="#bib54">Lee et al., 1999</a>; <a href="#bib62">Raizen and Avery, 1994</a>)  At a bare minimum at least 5 animals should be tested at 10 trials each. Response is expressed as % response.
               </p>
               <p><span class="emphasis"><em>C. elegans</em></span> adapt to nose touch after a rapid series of repeated trials. The nose touch trials must be given in fairly rapid succession
                  to adapt the animals' response-without accidentally hitting the animal over the head with the hair. To measure habituation,
                  compare the % response in the first 10 trials to the % response in the trials 40 through 50.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec3"></a>3.&nbsp;Osmotic avoidance
                     </h2>
                  </div>
               </div>
            </div>
            <p>See also: &#8220;<a href="#sec4_1" title="4.1.&nbsp;Chemotaxis, chemoaversion and plasticity-quadrant plate, version 1">Chemotaxis and chemoaversion, quadrant assay, version 2</a>&#8221; and &#8220;<a href="#sec4_3" title="4.3.&nbsp;Drop assay">Drop assay</a>&#8221;.
            </p>
            <p>Contributed by Anne Hart, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA, March 2005.</p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec3_1"></a>3.1.&nbsp;Osmotic ring assay
                        </h3>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_1_1"></a>3.1.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Avoidance of high osmolarity (concentrated solutions) by <span class="emphasis"><em>C. elegans</em></span> was first described by <a href="#bib23">Culotti and Russell (1978)</a>. and revisited by various researchers over subsequent years. The original assay, described in an updated format below, tests
                     the rate at which <span class="emphasis"><em>C. elegans</em></span> will cross an osmotic barrier on the surface of the agar. The barrier is usually circular in shape-referred to as a ring.
                     Animals are placed in the center of the ring (an area of low osmolarity) so they are surrounded by the high osmolarity ring
                     which will they avoid as a noxious stimulus. Animals escape the ring by crossing the high osmolarity barrier. Outside the
                     ring is normal NGM agar, a region of low osmolarity. In practice, the rate at which animals escape is dependent on several
                     parameters. Particularly critical are plate dryness, the number of animals in the ring <span class="emphasis"><em>versus</em></span> the size of the ring, and the rate of diffusion of the osmotic barrier. The assay is dependent on the number of times that
                     animals encounter the osmotic barrier; special consideration must be taken when assessing osmotic avoidance for uncoordinated
                     or lethargic animals. Hermaphrodites are assayed.
                  </p>
                  <p>Control strains must be used each day. N2 animals are generally used as positive controls as they robustly avoid the ring
                     barrier. Generally, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=che-3" target="_blank">che-3</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-10" target="_blank">osm-10</a></em></span> animals are used for negative controls. Uncontaminated, uncrowded cultures must be used. Animals that have been starved during
                     their life, those close to running out of food, animals that have passed through dauer or animals from culture plates that
                     are very dry should not be assayed.
                  </p>
                  <p>The number of animals to be tested determines the size of the ring. A 5/8<sup>th</sup> inch diameter ring holds 30 to 100 animals for population assays; a 3/8<sup>th</sup>inch diameter ring holds 1 to 5 animals. Too many animals for a specific ring size leads to aberrantly high escape rates;
                     too large a ring for the number of animals leads to aberrantly low escape rates.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_1_2"></a>3.1.2.&nbsp;Preparing assay plates and reagents
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The dryness of the assay plates is a critical consideration and can be the critical variable to control. The objective is
                     to obtain a large number of plates of uniform dryness. Plate dryness is assessed by weighing the plates. Once you establish
                     the correct plate weight in your lab, then (if the plate volume poured remains unchanged) finding good plates for the assay
                     is relatively straight-forward.
                  </p>
                  <p>NGM agar plates are allowed to sit out on the bench overnight after a large batch is poured. It is important that an equal
                     volume of media is poured into each plate; an automated media dispenser is ideal. The next day the plates are inverted and
                     then placed in a plastic box. The length of time required for plates to dry depends on atmospheric humidity in the laboratory
                     and how the box is covered. In winter in Boston it can take only a few days for plates to dry in a box with a closed (but
                     not tightly sealed) lid. In summer, it can take weeks. To identify plates of uniform dryness, plates are removed from the
                     box and individually weighed. If a constant amount of plate media is dispensed during plate pouring, then the weight of each
                     plate is directly related to how dry the plate is (how much evaporation has taken place). If the amount of media is a constant
                     over weeks or months, then plates can be often selected based on weight alone-once the correct weight is determined in behavioral
                     assays. Be sure to remove any condensation from the plate lid before weighing.
                  </p>
                  <p>The only definitive test to determine the correct weight (and dryness) is to run the osmotic avoidance assay with the control
                     strains. A good rule of thumb to start with here is that animals crawling on the agar surface of an appropriately dried plate
                     should have a visible, but small meniscus.
                  </p>
                  <p>Alternative approaches: The drying process can be hastened by covering the box with a large Kimwipe (as opposed to a lid)
                     although uniformity of plate dryness may decrease. Or, plates can be dried in a sterile hood for a few hours to hasten the
                     drying process-but the plates must be allowed to equilibrate with the lids closed for several hours afterward.
                  </p>
                  <p>Prepare 8M glycerol in water. This corresponds to a 60% solution of pure glycerol in water; &#8220;pure glycerol&#8221; from Sigma is
                     a 13M solution. Add a tiny bit of bromophenol blue to color the 8M glycerol and the resulting rings blue. 8M glycerol aliquots
                     can be stored in the <span class="symbol">&#8722;</span>20&deg;C freezer for years; aliquots are stable at room temperature for days.
                  </p>
                  <p>Depending on the number of animals to be assayed, either ultra fine point Sharpie lids (<span class="symbol">&#8764;</span>3/8<sup>th</sup> inch) or metal test tube caps (<span class="symbol">&#8764;</span>5/8<sup>th</sup> inch) are obtained. <span class="emphasis"><em>C. elegans</em></span> S Basal is required for population assays.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_1_3"></a>3.1.3.&nbsp;Running population assays
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Osmotic barrier rings must be printed onto the agar of the assay plate so they will be ready as soon as the animals finish
                     rinsing in the S Basal (see next paragraph). Usually four assay plates can be run in one batch. Obtain 4 additional 6cm Petri
                     dish lids and four metal test tube caps. Pipette about 25 to 30<span class="symbol">&#956;</span>l of 8M glycerol into the surface of each inverted lid. Place the open end of the cap down onto the inverted lid bisecting
                     the drop of glycerol. Spin the cap against the surface of the lid to spread the glycerol around the circumference of the cap.
                     Lift the cap. If there is a bubble of glycerol closing the cap, then pop it. Place the lid with glycerol very gently onto
                     the agar of an assay plate. Try to avoid marring the surface of the agar. Let the glycerol soak into the agar; this usually
                     takes between 2 and 5 minutes. Lift the cap off. Check to make sure the annular osmotic barrier is uniform. Always run the
                     control animals in duplicate (e.g., N2 and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-10" target="_blank">osm-10</a></em></span>) first to confirm that the assay plates are working. It is a good idea to run control interspersed with experimental animals
                     during the assays.
                  </p>
                  <p>Rinse animals off culture plates and into a 1.5ml microfuge tube with 1ml of S Basal using a Pasteur pipette. No need to close
                     the tube lids. Let animals settle by gravity for 5 minutes, remove excess S Basal and add another 1ml. After 5 to 10 minutes,
                     move animals in a minimal amount of liquid to the center of the osmotic ring. Remove excess liquid by very gently blotting
                     with a rolled up Kimwipe. Once the excess liquid is removed and animals start crawling, the timing begins for that assay plate.
                     Record the time the assay starts. Replace plate lid and avoid disturbing animals by dropping or tapping the plate during the
                     assay. Repeat till all four assays are underway. When the assays are done, wipe off the caps and lids so they can be reused.
                  </p>
                  <p>Animals should move actively within the ring during the assay period. N2 animals should avoid entering the osmotic barrier
                     for at least 10 minutes; 50 to 80% of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-10" target="_blank">osm-10</a></em></span> or other control animals should either die in the osmotic barrier or cross it and escape the ring. Those who die in the ring
                     are counted as escapers. If too many N2 animals escape, then the plates are likely too wet. If too few <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-10" target="_blank">osm-10</a></em></span> animals escape, then the plates are likely too dry.
                  </p>
                  <p>The number of animals that have escaped at 10 minutes is recorded along with the number of animals that remain in the ring.
                     The % escaping animals is calculated. At least 5 assays per strain or genotype are suggested; these should be determined on
                     at least two different days.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec3_1_4"></a>3.1.4.&nbsp;Running assays with limited numbers of animals
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The rate at which animals encounter the osmotic barrier is important in determining their response. If limited animals are
                     available-or individual animals need to be tested, then smaller rings should be used.
                  </p>
                  <p>Rings can be generated using ultra fine point Sharpie lids and 20 <span class="symbol">&#956;</span>l of 8M glycerol as described above. Use the open end of the Sharpie lid. Animals to be assayed are gently (and rapidly) picked
                     from the bacterial lawn to an empty plate (to eliminate food) to the assay plate. Up to 5 animals can be used in a ring of
                     this size. At least 7 rings with at least 4 animals for each genotype is a good staring point. As described above, control
                     strains should be run before experiments begin.
                  </p>
                  <p>If the osmotic response of individual animals must be assessed (e.g., laser ablation experiments), then animals should be
                     assayed at least 4 times. Once an animal crosses the osmotic barrier, then the animal is immediately returned to the bacterial
                     lawn for at least 15 minutes.
                  </p>
                  <p>If small numbers of animals are used, then it is convenient, and perhaps more accurate, to record the time the animals cross
                     the barrier, not the % escaping. While the assay is running, plates are checked every minute and the number of animals who
                     have escaped is recorded. The average time to escape is then calculated.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec4"></a>4.&nbsp;Chemosensation
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_1"></a>4.1.&nbsp;Chemotaxis, chemoaversion and plasticity-quadrant plate, version 1
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Gert Jansen,  Erasmus Medical Center, Rotterdam, The Netherlands, March, 2005.</p>
               <p>We discriminate three different responses using the quadrant plate chemotaxis assay for salts as developed by Steven Wicks
                  (<a href="#bib86">Wicks et al., 2000</a>):
                                          
               </p>
               <div class="orderedlist">
                  <ol type="1">
                     <li>
                        <p>Chemo-attraction to 0.1&#8211;100 mM NaCl or other salts.</p>
                     </li>
                     <li>
                        <p>Chemo-aversion of 1 M NaCl.</p>
                     </li>
                     <li>
                        <p>Gustatory plasticity, i.e. the response of <span class="emphasis"><em>C. elegans</em></span> to 25 mM NaCl after 15 min pre-exposure to 100 mM NaCl (<a href="#bib48">Jansen et al., 2002</a>).
                        </p>
                     </li>
                  </ol>
               </div>
               <p>Briefly, pairs of opposite quadrants of four-quadrant Petri plates (Falcon X plate, Becton Dickinson Labware, USA) are filled
                  with 16 ml buffered agar (2% agar, 5 mM K<sub>2</sub>HPO<sub>4</sub>/KH<sub>2</sub>PO<sub>4</sub> MgSO<sub>4</sub>) either containing a dissolved attractant or no attractant. The plates are left open to solidify and dry for 60 min. Then
                  the plates are closed and stored for use on the same day. Just before the assays are performed, i.e. just before washing off
                  worms, adjacent quadrants of the assay plates are connected with a thin layer of molten agar (without attractant). Assays
                  are always performed in duplicate, try to control for environmental influences by placing the plates on the bench in different
                  orientations. In addition, assays should always be repeated on separate days.
               </p>
               <p>Animals are prepared by bleaching gravid adults and washing the eggs with M9 and growing the animals for approximately 72
                  hrs at 25&deg;C. During a 15 min wash, the well-fed, young adult nematodes are washed three times with CTX buffer (5 mM KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> pH 6, 1 mM CaCl<sub>2</sub> and 1 mM MgSO<sub>4</sub>) and 100&#8211;200 worms are placed at the intersection of the four quadrants in 5 ul CTX buffer. Most of the buffer can be removed
                  with a pipette, or a tissue. The distribution of the worms over the four quadrants is determined after 10 min. A chemotaxis
                  index (CI=(A-C)/A+C, where A is the number of worms over quadrants 1 and 3, C is the number of worms over quadrants 2 and
                  4) was calculated at each time point. As attractants we use 100 microM, 1, 10, and 100 mM NaAc, NaCl, NH<sub>4</sub> Ac and NH<sub>4</sub>Cl. As repellent we use 1 M NaCl.
               </p>
               <p>To test the plasticity of the response of <span class="emphasis"><em>C. elegans</em></span> to salts, animals are pre-exposed to the compound tested during the washing steps (<a href="#bib48">Jansen et al., 2002</a>). We found that the optimal concentration for pre-exposure was 100 mM NaCl. The 100 mM salt was simply added to the CTX washing
                  buffer. Optimal pre-exposure time was 15 min, longer pre-exposure was toxic, especially for some mutants, while shorter pre-exposure
                  had less effect. Subsequently, the animals are tested in a standard water-soluble compound chemotaxis assay for their attraction
                  to the salt used during pre-exposure. We found that chemotaxis to 25 mM NaCl worked best.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_2"></a>4.2.&nbsp;Chemotaxis and chemoaversion-quadrant assay, version 2
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Stephen Wicks, Boston College, Chestnut Hill, MA, USA, March, 2005.</p>
               <p>This assay was developed to assay attraction from or avoidance of soluble compounds, or to compare concentrations of a single
                  compound (<a href="#bib86">Wicks et al., 2000</a>). The assay is robust enough to characterize known mutants, or to screen for new mutants which fail to approach or avoid
                  soluble compounds such as salts, sugars, amino acids and other potentially biologically significant compounds. This assay
                  is effective (chemotaxis index of 0.99 from some compounds), probably because the gradients that develop between adjacent
                  quadrants are very high, and thus demonstrating a preference for one compound over another, or a discrimination of two concentrations,
                  is an easy task for the worm. The assay relies on the use of segmented plastic Petri plates. A variety of manufacturers sell
                  appropriate plates. Ideally, the plates should have no (or at least only a very small) raised arm - a recent innovation that
                  facilitates robotic handling of plates-restricted to the periphery. A plate will have 4 equally sized quadrants. Each quadrant
                  should have a volume capacity of 10-12 mls. and is filled with agar containing the dissolved compound of interest solublized
                  in molten agar. In practice we generally fill opposite pairs of quadrants with the same compound-containing agar. Thus, each
                  quadrant is adjacent to two quadrants containing the other compound or concentration of interest. Each quadrant is overfilled
                  such that surface tension provided by the sterilized plastic is just sufficient to keep the agar from spilling, and the agar
                  is allowed to dry. It is important to keep the volume in each quadrant the same as in all other quadrants. Once the agar has
                  solidified, the tops of each plastic wall that separates the four quadrants is coated with agar containing no attractant to
                  provide a smooth, continuous surface over which animals move. It is also allowed to dry and cool. A population of animals
                  is then spotted at the intersection of the quadrants in a small (4-10 microliter) volume of low salt buffer. The population
                  will segregate over the quadrants that contain the preferred taste. Animals present in each quadrant are counted at several
                  time points to assess adaptation and changing taste preferences.
               </p>
               <div class="procedure"><a name="d0e928"></a><p class="title">Procedure&nbsp;1.&nbsp;Quadrant plate preparation</p>
                  <ol type="1">
                     <li>
                        <p>Plates are available from Falcon. Cases of 500 plates (Falcon #351009 X PLATE Petri dish with lid) each of which is 100 mm
                           (diameter) by 15 mm (deep) are typically ordered.
                        </p>
                     </li>
                     <li>
                        <p>Taxis Agar is composed of 2% agar (MOPS buffered). We have found that for most salts, 10% of 0.1 M MOPS (we pH the MOPS to
                           7.2 with 25% ammonium) is sufficient to control the pH, although this does not appear to be true for amino acids or other
                           free bases. The desired attractants are dissolved in solution prior to autoclaving. After autoclaving, the solutions are brought
                           to 1 mM CaCl<sub>2</sub> and 1 mM MgSO<sub>4</sub>. Don't add TWEEN or any other detergents; these disrupt the surface tension of the agar too much. As an alterative we also
                           have been using phosphate buffered agar.
                        </p>
                     </li>
                     <li>
                        <p>After the agar cools to about the temperature that you could comfortably hand pour (55-60 degrees C), an autopipetter is used
                           to distribute 12.6 ml of agar (the exact volume will depend on the plate used) into each quadrant of the plate. Typically,
                           I place similar attractants into opposite quadrants of the plate. The surface on which the agar is distributed should be level,
                           as this volume will bring the agar level above the plastic spacers between quadrants; surface tension keeps the agar from
                           spilling from one quadrant into adjacent ones. (Also, don't shake or bump the bench or table!).
                        </p>
                     </li>
                     <li>
                        <p>Let the agar cool and harden. Then, a few minutes prior to testing a population, draw about 0.5 ml of buffered agar (with
                           CaCl<sub>2</sub> and MgSO<sub>4</sub>) into a glass pipette and gently but briskly fill the &#8220;valleys&#8221; between adjacent agar-filled quadrants on each plate. I have
                           tried using a lower agar concentration for this step to avoid building any significant &#8220;topography&#8221; for the worms and that
                           works well also, but, with care this is not necessary.
                        </p>
                        <p></p>
                     </li>
                     <li>
                        <p>A population of animals is washed off of plates and rinsed once with a low-salt CTX-buffer solution (1mM CaCl<sub>2</sub>, 1mM MgSO<sub>4</sub>, 5mM potassium phosphate, pH=6.6), placed over the centre of the plate in about 10ul of CTX Buffer and allowed to partition
                           over time. The worms will often show a preference almost immediately (within 30 s) even prior to the evaporation of the buffer.
                        </p>
                     </li>
                     <li>
                        <p>The preference for one kind of moiety (or concentration of moiety) over the other is generally expressed as fraction of worms
                           over that moiety divided by total number of worms per plate (although, obviously it could also be expressed as [# worms over
                           attractant - # worms over control] / Total # worms, as with the odorant assays from the Bargmann lab). This is calculated
                           at various time points (generally, I use 10 min 20 min, 30 min, 45 min, and 60 min-sometimes also 5 min if the buffer has
                           evaporated).
                        </p>
                     </li>
                  </ol>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_2_1"></a>4.2.1.&nbsp;Soluble compound aversion assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Genetic screens for animals defective for avoidance of aversive compounds were carried out on assay plates with 10<sup>3</sup>-10<sup>4</sup> worms per plate. The assay plates were prepared by pouring 5 ml of chemotaxis agar (1.5% agar, 75 mM NH<sub>4</sub>Cl, 10 mM MOPS, pH 7.2 with NH<sub>4</sub>OH, and 0.25% Tween 20) into a 9 cm Petri dish. A thin line (25 <span class="symbol">&#956;</span>l) of an aversive compound (e.g., copper as 150 mM CuSO<sub>4</sub>, 2 % SDS, or high osmotic pressure as 4 M fructose; <a href="#bib83">Ward, 1973</a>) was poured across the midline of the Petri dish and another 25 <span class="symbol">&#956;</span>l traced in a semicircle around one half of the dish (<a href="#figure2">Figure 2</a>). Young adult worms were washed twice with S-basal buffer to remove bacteria. Immediately after the compound was absorbed
                     into the chemotaxis agar, worms were placed on one side of the assay plate in a small volume of buffer. Excess buffer was
                     carefully removed with tissue paper. A volatile attractant (2 <span class="symbol">&#956;</span>l of isoamyl alcohol 1:10 dilution in ethanol) was placed across the assay plate 1 cm from the opposite edge with the aversive
                     semicircle. Normal, healthy worms approached the attractant, but stopped abruptly at the boundary formed by the repulsive
                     compound. Individuals that passed through the aversive barrier were isolated and allowed to produce self progeny. To quantify
                     the behavior of a given strain, an aversion index (A.I.) was calculated by dividing the number of worms that, one hour after
                     placing the worms on the plate, crawled through the aversive compound by the total number of worms on the plate (approximately
                     250&#8211;500). This index includes all animals that attempt to crawl up the side of the plate.
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure2"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="WMBehavFig2.jpg"><img src="WMBehavFig2_s.jpg" border="2" align="middle" alt="WormBook Behavior figure 2"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 2.&nbsp;</b></p>
                     </div>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_3"></a>4.3.&nbsp;Drop assay
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Paolo Bazzicalupo and Massimo Hilliard, Istituto Internazionale di Genetica e Biofisica, Naples, Italy, March,
                  2005.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_3_1"></a>4.3.1.&nbsp;Drop test and dry drop test
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec4_3_1_1"></a>4.3.1.1.&nbsp;Introduction
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>These two assays were developed to study the avoidance response of individual worms to water soluble repellents. They can
                        efficiently be used to score populations as well. Reference to and practical use of these tests can be found in <a href="#bib42">Hilliard et al. (2004)</a> and in <a href="#bib41">Hilliard et al. (2002)</a>. These papers also report some controls strains (especially mutants tested) that can help researchers understand and define
                        the usefulness of these assays. Main use of the assays: determine the phenotype of single animals to characterize existing
                        mutants, isolate new mutants, characterize laser-operated animals, study spatial resolution of sensory organ, study water-soluble
                        repellent adaptation. Possible other applications include response to volatile repellent.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec4_3_1_2"></a>4.3.1.2.&nbsp;Drop test
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>In this assay, a small drop of repellent is delivered near the tail of an animal while it moves forward. Once in contact with
                        the tail, the drop surrounds the entire animal by capillary action reaching the anterior sensory organs. If the substance
                        is sensed as repellent the animal stops moving forward and starts moving backward. If the substance is not sensed as a repellent
                        the animal will continue its forward movement.
                     </p>
                     <p>Animals: Well fed young adult hermaphrodites. With some experience and appropriate controls, younger larvae and males can
                        also be tested. Animals are washed off the plates were they have grown, put on unseeded NGM plates and allowed to rest at
                        room temp for 10-15 minutes before testing. We always challenge the worms to be tested with a few drops of buffer alone (and
                        they should not respond). Occasionally populations of worms appear hyper-reactive. In these cases the test should be postponed
                        to avoid false positives.
                     </p>
                     <p>Plates: The tests are conducted on unseeded NGM plates. In some cases a more controlled composition of the medium may be desirable
                        (e.g., without Ca and/or Mg ions, lower salt concentration etc.). NGM can certainly be substituted by other media but appropriate
                        controls are necessary. It is possible to use M13 plates (plates in which the agar is dissolved in M13 buffer, see below,
                        in which repellents are usually dissolved) to reduce possible interference due to the change in composition between buffer
                        and medium.
                     </p>
                     <p>Best results are obtained with fresh, non wet plates such that the tracks of the moving worms can be detected but will disappear
                        in few minutes. Assays are done at room temperature as long as it does not exceed 24-25 &deg;C. Plates should be allowed to equilibrate
                        to the room temp.
                     </p>
                     <p>Repellents:  All repellents are dissolved in: 30 mM Tris-HCl pH 7.0, 100 mM NaCl, 10 mM KCl (M13 buffer). We have used quinine,
                        other alkaloids, Denatonium etc. at 10mM final concentration. Cu<sup>++</sup> and Zn<sup>++</sup> were also used at 10mM. For Low pH avoidance we used 0.1 or 0.2 M Acetic acid. For osmotic avoidance we used Glycerol at
                        1M. It is advisable to determine dose response curves if new repellents are tested. With appropriate controls buffer composition,
                        repellent concentration and the composition of the medium can be modified.
                     </p>
                     <p>Delivery of the stimulus and scoring: Drops are delivered using 10<span class="symbol">&#956;</span>l glass capillaries (Blaubrand intraMARK or equivalent) pulled by hand on a flame to reduce the diameter of the tip. The capillary
                        is mounted in a holder with rubber tubing and operated by mouth pipetting. In case of toxic compounds, all the mouth operations
                        can be performed using a 1-5 ml syringe. We calculated the average drop size to be about 5 nanoliters. The size of the drop
                        has to be large enough such that the liquid surrounds the animal and can reach the anterior sensory organs but not so large
                        that the animal looses adherence to the substrate and swims in it. The animal usually starts a backward motion within 1 second
                        of the delivery of the drop. The response to each drop is scored as positive if the animal reacts within 4 seconds. The avoidance
                        index (a.i.) is the number of positive responses divided by the total number of trials. The a.i. will range between value
                        of 1 and 0 where higher values indicate stronger repellent response, lower values indicate weak response. As described below
                        the a.i. can be used to define the response of single animals as well as of populations.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec4_3_1_3"></a>4.3.1.3.&nbsp;Single animal assay
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>In the single animal assay (for example to determine the phenotype and thus the genotype of one animal derived from a cross-or
                        to determine the effect of cell ablation on one animal) repeated drops are delivered to the same animal. The single animal
                        is transferred from the growing plate to the test plate with a platinum wire or an eyelash and allowed to rest and adapt to
                        the new plate for 5-10 minutes before starting the assay. An ISI (Inter Stimuli Interval) of at least two minutes is used
                        between successive drops to the same animal. Each animal is tested with no more than 20 successive drops; the animal is then
                        transferred onto a new seeded plate and allowed to recover for one or two hours before starting a new set of experiments.
                        No more than 3 sets of 20 drops each were conducted per day on each animal. In most cases the phenotype is accurately determined
                        with just 10 drops. The avoidance index (a.i.) will be the number of positive responses divided by the total number of trials.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec4_3_1_4"></a>4.3.1.4.&nbsp;Population assay
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Two methods can be used to define the responses of populations of worms:
                        								
                     </p>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>10 to 50 (usually 30) well-fed adult animals of the same population are placed each on a small NGM unseeded agar plate and
                                 allowed to rest for 5-10 minutes. Each animal is then tested with several successive drops (5-30) as described in the single
                                 animal assay. The results of all animals belonging to the same population are then combined. This is our preferred population
                                 method since it provides data on the individual variation within the population and the results are amenable to more accurate
                                 statistic analysis.
                              </p>
                           </li>
                           <li>
                              <p>A population of well-fed adult animals is washed with buffer and placed on a large unseeded plate. The animals are allowed
                                 to rest for 15 minutes and then at least two series of 50 animals of the same population are challenged each with a single
                                 drop of the chosen substance. Each response is recorded as either positive or negative. The a.i. is the number of positive
                                 avoidance responses divided by the total of trials (drops delivered). This is obviously faster than the other method and often
                                 it is more than satisfactory.
                              </p>
                              <div class="sidebar">
                                 <div class="tip" style="margin-left: 0.5in; margin-right: 0.5in;">
                                    <h3 class="title">Tips</h3>
                                    <div class="itemizedlist">
                                       <ul type="disc">
                                          <li>
                                             <p>Experience, which can be acquired in a few days, is of great assistance especially with regard to judging the appropriate
                                                dryness of the plates, learning how to deliver a drop of the appropriate size and how to detect the response.
                                             </p>
                                          </li>
                                          <li>
                                             <p>CuCl<sub>2</sub> is a very strong repellent that can be used as reference substance to observe clearly the avoidance reflex.
                                             </p>
                                          </li>
                                          <li>
                                             <p><span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-10" target="_blank">osm-10</a></em></span> is a good mutant to look at during a training period since it is very specific with a very low response to high osmotic strength
                                                (glycerol 1M) and still a good response to copper and other repellents.
                                             </p>
                                          </li>
                                          <li>
                                             <p>Because animals of different genetic background do differ in the way they move and react, it is always a good idea to get
                                                acquainted with the movement and the reaction to the drops of animals of a new genetic make-up before testing them. Obviously
                                                animals that are severely Unc or paralyzed cannot be tested but after some experience it is possible to test animals that
                                                are dumpy, rollers, and some weak Uncs.
                                             </p>
                                          </li>
                                          <li>
                                             <p>In some cases a complete description of the avoidance phenotype of a mutant strain may require the determination of dose response
                                                curves.
                                             </p>
                                          </li>
                                       </ul>
                                    </div>
                                 </div>
                              </div>
                           </li>
                        </ol>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec4_3_1_5"></a>4.3.1.5.&nbsp;Dry drop test
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>The dry drop test resembles the drop test except that the animal encounters the repellent substance after the drop has been
                        absorbed into the agar. In this case the drop is delivered about 0.5-1 mm anterior to the animal (head stimulus). This assay
                        allows spatial resolution between anterior and posterior sensory organs since it prevents the capillary action (present in
                        the drop test) that brings the substance and thus the stimulus on the entire body surface. In the drop test both amphid and
                        phasmid neurons are stimulated, in the dry drop test only amphid neurons are stimulated. Thus a difference in response between
                        drop test and dry drop test may indicate a role of the phasmid neurons in determining the response.
                     </p>
                     <p>It is also possible to separate stimulation of anterior and posterior sensory neurons by confronting the animals with two
                        dry drops. The first just anterior to the animal nose; the second just posterior to the animal tail (0.3-0.5 mm). With this
                        strategy, the animal is forced to encounter the repellent first anteriorly with the head (e.g., amphids) and then, during
                        the backward movement, with the tail (e.g., phasmids). In this case, the duration of the backward response is the measure
                        of the effect of the posterior sensory organ. For instance with SDS as a repellent, the backward response is reduced from
                        an average of 3 seconds if only head stimulus is applied, to 2 seconds if the repellent is applied also on the tail as described
                        in <a href="#bib41">Hilliard et al. (2002)</a>.
                     </p>
                     <p>Other variations: Another interesting use of the dry drop test was done in the Maricq lab (<a href="#bib58">Mellem et al., 2002</a>). In their analysis, instead of simply scoring the response as either negative or positive, they measured the delay of the
                        response from the time when the animal encountered the repellent on the agar to the time when the backward response begins.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec4_3_1_6"></a>4.3.1.6.&nbsp;Adaptation using the drop assay
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>The drop test can be used to investigate the adaptation of animals to water soluble repellent substances. Two adaptation paradigms
                        have been used:
                        								
                     </p>
                     <div class="orderedlist">
                        <ol type="1">
                           <li>
                              <p>Worms are exposed to successive stimuli (drops) and the response to each drop is recorded as either positive or negative.
                                 If the ISI (inter-stimulus interval) is short enough (10 to 30 seconds) the avoidance index (determined by testing 10 or more
                                 worms) undergoes a decrease on the second stimulus and will undergo further reductions on successive stimuli. The adaptation
                                 is reversible and after an adequate period of rest the a.i. will go back to normal. The main parameter that can be varied
                                 in the assay is the length of the ISI. If the ISI is long enough no adaptation will be detected.
                              </p>
                           </li>
                           <li>
                              <p>Worms are exposed to the repellent continuously for a relatively long period of time and then the avoidance response to the
                                 repellent is tested by the drop test. The a.i. is calculated by testing 10 to 30 worms. In this test, Cu<sup>++</sup> exposure for 1 minute, is able to induce a strong reduction in the response to Cu<sup>++</sup>, which lasts several minutes but is again reversible. In practice a large drop of the repellent is deposited on the unseeded
                                 plate and the animal is kept swimming in it. After the desired time, the animal is moved to a clean section of the plate and
                                 after 1 minute, tested with the repellent in the same modality used in the drop test. Because adaptation is revealed by a
                                 reduced response it is necessary to distinguish it from general toxic effects. This can be done by testing the avoidance response
                                 to some different repellent stimulus (to which adaptation has not occurred). Determination of the recovery time may in same
                                 cases also be important.
                              </p>
                              <div class="sidebar">
                                 <div class="caution" style="margin-left: 0.5in; margin-right: 0.5in;">
                                    <h3 class="title">Caution</h3>
                                    <p>Some repellent are too toxic too quickly for the worms and thus the long exposure cannot be always applied. The repeated drops
                                       paradigm might in these cases be more appropriate.
                                    </p>
                                 </div>
                              </div>
                           </li>
                        </ol>
                     </div>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_4"></a>4.4.&nbsp;Chemotaxis to volatile point source
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Cori Bargmann,  The Rockefeller University, New York, N.Y. USA, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_4_1"></a>4.4.1.&nbsp;Strains
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Nematodes are grown on <span class="emphasis"><em>E. coli</em></span> strain HB101 at 20&deg;C (<a href="#bib11">Brenner, 1974</a>). Raise all animals with plentiful food in uncrowded conditions (ideal-place about 3 adults on one 6 cm plate with food,
                     or 10 adults on one 10 cm plate with food, wait 4 days at 20 and test their progeny; for mutants you will need to place down
                     more adults at the beginning-about twice as many for <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-6" target="_blank">osm-6</a></em></span> as for N2). Wild-type animals are <span class="emphasis"><em>C. elegans</em></span> variety Bristol, strain N2. Test animals in their first or second days as adults, larvae do not perform as well.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_4_2"></a>4.4.2.&nbsp;Chemotaxis assays
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Chemotaxis assays are based on the assay developed by <a href="#bib83">Ward (1973)</a>.
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Assay plates are 10 cm tissue culture dishes containing 10 ml of:
                                                                      
                           </p>
                           <table class="simplelist" border="0" summary="Simple list">
                              <tr>
                                 <td>1.6% BBL-agar (Benton-Dickinson) or 2% Difco-agar. Autoclave, then add:</td>
                              </tr>
                              <tr>
                                 <td>5 mM potassium phosphate, pH 6.0</td>
                              </tr>
                              <tr>
                                 <td>1 mM CaCl<sub>2</sub></td>
                              </tr>
                              <tr>
                                 <td>1 mM MgSO<sub>4</sub></td>
                              </tr>
                           </table>
                        </li>
                        <li>
                           <p>For volatile attractants, use plates 12-36 hours after pouring. If plates are contaminated with mold, do not use. To prepare
                              plates for assay, make two small x marks 180 degrees opposite each other on the bottom (plastic) plate near the edges of the
                              plate. Label one with the name of the attractant to be used. Place 1 microliter of 1 M Sodium Azide (POISON) on the agar above
                              each x. Let this soak in as you prepare the worms. (<a href="#bib75">Suzuki et al., 2003</a>; <a href="#bib80">Voisine and Hart, 2004</a>).
                           </p>
                        </li>
                        <li>
                           <p>Wash the worms off an unstarved plate with 1.5 ml S Basal (<a href="#bib11">Brenner, 1974</a>; we just use phosphate-buffered saline and omit the cholesterol). Place in an Eppendorf tube. Let the worms settle to the
                              bottom (3-5 minutes). Remove the supernatant  and wash with S basal 2 additional times, letting worms settle each time, and
                              then once with water. Do not skip any washes, since coli interfere with the assay. Work quickly here-if you let the animals
                              sit around for too long, they get sluggish and do not perform well. The washes enrich for larger animals (adults), which is
                              what you want.
                           </p>
                        </li>
                        <li>
                           <p>Pipette the worms in a small volume (10 microliters or so) onto the Petri plate, equidistant from the two xs, slightly off
                              center (see origin, <a href="#figure2">Figure 2</a>). Remove excess liquid with a Kimwipe, leaving worms on agar. Pipette 1 microliter of attractant on the agar at the labeled
                              x, 1 microliter of diluent (usually ethanol) at the unlabelled x. Close the lid of the plate and don't open in again if you
                              don't have to.
                           </p>
                        </li>
                        <li>
                           <p>After 10 minutes or so make sure the worms are not all clumped at the origin. If they are, disperse them with a wire pick.</p>
                        </li>
                        <li>
                           <p>Count the total number of worms.</p>
                        </li>
                        <li>
                           <p>At 60 minutes, count the worms anaesthetized at each of the two x marks.</p>
                        </li>
                        <li>
                           <p>Calculate the chemotaxis index:
                                                                  
                           </p>
                           <table class="simplelist" border="0" summary="Simple list">
                              <tr>
                                 <td>i=(# worms at attractant at 60'-#worms at control at 60')/total number of worms.</td>
                              </tr>
                           </table>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_5"></a>4.5.&nbsp;Distinguishing between odorants during cross-saturation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Using a variation of the assay above, response to a given odorant can be affected by saturating amounts of another odorant
                  (<a href="#bib5">Bargmann et al., 1993</a>). This saturation assay was used to group volatile odorants into seven classes. The saturating odorant is mixed into the
                  molten agar prior to assay plate pouring. In the published paper, 1<span class="symbol">&#956;</span>l of benzaldehyde or diacetyl was added to 10ml of agar.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_6"></a>4.6.&nbsp;Decreased response after chronic exposure to attractant
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Odorant exposure can decrease subsequent response. Each odorant saturates response to itself before saturating response to
                  other odorants. The above protocols are used for exposure and measuring response.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_7"></a>4.7.&nbsp;Oxygen sensing
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contact Cori Bargmann (The Rockefeller University, New York) for details and see <a href="#bib36">Gray et al. (2004)</a>.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_8"></a>4.8.&nbsp;Polyunsaturated fatty acids avoidance; liquid drop assay
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Cori Bargmann and Amanda Kahn, The Rockefeller University, New York, USA, March, 2005.</p>
               <p><span class="emphasis"><em>C. elegans</em></span> respond rapidly to exogenous polyunsaturated fatty acids. This response is dependent on TRPV channel function and mediated
                  by sensory neurons (<a href="#bib50">Kahn-Kirby et al., 2004</a>). Animals are tested off food on NGM plates that have aged overnight at room temperature in stacks of four plates. Pick animals
                  off food as young adults to a clean plate, then transfer without food to an assay plate. Let the animal crawl 15 minutes after
                  transfer with the lid off. For a given strain, animals are assayed with 25 drops of 1M glycerol, and ethanol/M13 controls
                  prior to lipid exposure to establish the baseline response. Then, for the experiment on another assay plate do 25 trials each
                  with M13/ethanol, diluted fatty acid #1 and diluted fatty acid #2. All trials should be done with a half hour of transfer.
                  M13 buffer contains 30 mM TRIS pH 7.0, 100 mM NaCl, and 10 mM KCl.
               </p>
               <p>Undiluted fatty acids were obtained from Nu-Chek Prep (Elysian, MN) and stored at -20&deg;C in the dark in sealed glass vials.
                  Glass vials containing fatty acids are readily opened using a glass chromatography column cutter  ceramic knife. Fatty acids
                  are aliquoted undiluted using a glass Pasteur pipette into small amber glass Kimble vials obtained from VWR (<span class="symbol">&#8764;</span>4ml, with small black caps, no plastic lid inserts) for storage. The vials are sealed under nitrogen by blowing a stream of
                  nitrogen over the neck of the vial, Parafilmed, and frozen.
               </p>
               <p>For lipid drop test assays, prepare a fresh 200 mM stock in ethanol fresh each day. Vortex to dissolve. This 200 mM stock
                  PUFA solution in ethanol is diluted into M13 immediately prior to the assay, vortexed for 30 seconds, and flushed with nitrogen.
                  Transfer a small amounts of the diluted fatty acids to microfuge tube for assays; close microfuge tube during assay.
               </p>
               <p>Two lipid solutions were tested during each assay, and the investigator was blind to lipid identity. Positive (<span class="emphasis"><em>N2</em></span>) and negative (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-9" target="_blank">osm-9</a></em></span>) control strains were assayed along mutant strains each day, and mutant data was collected only if positive and negative
                  control animals displayed normal performance on M13 and 1M glycerol assays. Fatty acids concentrations tested range between
                  2mM and 0.01mM (4mM is usually too high). EPA can be used as a canonical fatty acid.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_9"></a>4.9.&nbsp;Males attraction to hermaphrodites
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Extracted from the literature, see <a href="#bib73">Simon and Sternberg (2002)</a>.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_9_1"></a>4.9.1.&nbsp;Strains
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Unless indicated otherwise, all males come from the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1490;class=Variation" target="_blank">e1490</a></em></span>) mutant, which segregates XO male progeny by X chromosome nondisjunction during meiosis (<a href="#bib46">Hodgkin et al., 1079</a>). To construct cue source regions, the muscle mutant <span class="emphasis"><em>unc-52</em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e444;class=Variation" target="_blank">e444</a></em></span>), individuals of which become relatively motionless by the time they are young adults, was used. For the vulvaless experiment
                     we constructed a <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a></em></span> (<span class="emphasis"><em>lsy1</em></span>) <span class="emphasis"><em>unc-52</em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e444;class=Variation" target="_blank">e444</a></em></span>); <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=dpy-20" target="_blank">dpy-20</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1282;class=Variation" target="_blank">e1282</a></em></span>)<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lin-3" target="_blank">lin-3</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=n378;class=Variation" target="_blank">n378</a></em></span>); <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1490;class=Variation" target="_blank">e1490</a></em></span>) strain (PS3980; Aroian and Sternberg, 1991; <a href="#bib34">Ferguson and Horvitz, 1985</a>). <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lov-1" target="_blank">lov-1</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sy582;class=Variation" target="_blank">sy582<span class="symbol">&#948;</span></a></em></span>) and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pkd-2" target="_blank">pkd-2</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sy606;class=Variation" target="_blank">sy606<span class="symbol">&#948;</span></a></em></span>) have been described  (<a href="#bib8">Barr et al., 2001</a>). The sensory mutants <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-5" target="_blank">osm-5</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=p813;class=Variation" target="_blank">p813</a></em></span>) and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-6" target="_blank">osm-6</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=p811;class=Variation" target="_blank">p811</a></em></span>), as well as the diverse isolates CB4932 (Taunton, England), CB4555 (Pasadena, CA), and CB4856 (Hawaii) were obtained from
                     the <span class="emphasis"><em>Caenorhabditis</em></span> Genetics Center (St. Paul); isolates are described by Hodgkin and Doniach (<a href="#bib44">Hodgkin and Doniach, 1997</a>) and further characterized by de Bono and Bargmann  (<a href="#bib27">de Bono and Bargmann, 1998</a>). Similar to wild-type  [Bristol N2; <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> (<span class="emphasis"><em>+</em></span>)] strains, males from isolates occur spontaneously at an inconveniently low frequency in the self-progeny of hermaphrodites.
                     To obtain a constant supply of males, isolates were heat shocked and maintained by backcrossing. To test males from <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-6" target="_blank">osm-6</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-8" target="_blank">him-8</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1489;class=Variation" target="_blank">e1489</a></em></span>); <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-6" target="_blank">osm-6</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=p811;class=Variation" target="_blank">p811</a></em></span>) strains were made, and the presence of <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=osm-6" target="_blank">osm-6</a></em></span> was verified by use of fructose avoidance and dye-fill assays (<a href="http://cobweb.dartmouth.edu/_ambros/worms/16.html" target="_blank">http://cobweb.dartmouth.edu/_ambros/worms/16.html</a>; <a href="#bib23">Culotti and Russell, 1978</a>). All animal stocks were stored at 18&deg;C. All stocks and animals harvested for upcoming trials were grown on standard 5-cm
                     diameter NG agar plates inoculated with the <span class="emphasis"><em>Escherichia coli</em></span> strain OP50, grown in Luria-Bertani media (LB), as a food source.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_9_2"></a>4.9.2.&nbsp;Response assay protocol
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Bacterial lawns were grown on standard 5-cm diameter agar plates for all trials: three drops of OP50-inoculated LB, spread
                     thinly, with a <span class="symbol">&#8764;</span>0.25-centimeter gap between the edge of the lawns and the walls of the plates, to dissuade test animals from leaving trials.
                     Plates were then stored at <span class="symbol">&#8764;</span>22&deg;C for 2 days until used for trials. Source and test animals were harvested daily at the fourth larval stage (L4), and stored
                     at <span class="symbol">&#8764;</span>18&deg;C overnight with 10-20 animals per same-sex plate to be used the following day in trials as young adults. The muscle mutant
                     <span class="emphasis"><em>unc-52</em></span> was used to condition agar plates for at least 3 and up to 8 h (see Figure 1a in <a href="#bib73">Simon and Sternberg, 2002</a>). Five to 10 min before the onset of a particular trial, source animals were removed. Trials consisted of the introduction
                     of a single test animal on to a single trial plate (see Figure 1a in <a href="#bib73">Simon and Sternberg, 2002</a>), and movement of the animal was then documented for 5 min. Effort was made to orient animals in the direction of the 3 &times;
                     7-mm scoring regions. A motor-driven stage attached to a computer joystick was used to help videotape trials through a dissecting
                     microscope. Trials were videotaped, and the number of reversals per crossing and the time animals spent in conditioned region
                     were scored and are on record. Backwards movement equal to or greater than a body length was counted as &#8220;1 reversal&#8221;; backward
                     movement less than a body length was counted as &#8220;0.5 reversal.&#8221; We used the frequency of reversals as indication that males
                     detect a cue; it seems likely that when males pass over a mate-finding cue that they will move backwards, and by repeated
                     reversals, home in on a cue's source. Equal numbers of unconditioned and conditioned agar trials were run in parallel on <span class="symbol">&#8764;</span>5 plates each per day; moreover, trials were run blindly and interspersed at random. Isolate trials were run in parallel alongside
                     trials of the laboratory strain (Bristol N2). Unconditioned agar trials were meant to parallel conditioned agar trials as
                     closely as possible: (i) made from same day bacterial lawns, (ii) kept at the same temperature, and (iii) had bacteria picked
                     on and off to mimic introduction of source animals as on conditioned agar trials. Average temperature during trials was <span class="symbol">&#8764;</span>22&deg;C.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_9_3"></a>4.9.3.&nbsp;Holding assay protocol
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The holding assay was identical to the response assay with the following modifications: (i) source animals were introduced
                     in a pile in area A, (ii) test animals were introduced directly on this conditioned spot (see Figure 1b in <a href="#bib73">Simon and Sternberg, 2002</a>); (iii) in addition to area A, number of reversals was scored in outwardly stacked, concentric ring-areas (with radii of
                     1 mm, 5 mm, 16.5 mm, and <span class="symbol">&#8764;</span>25 mm; unequal sized areas were normalized by dividing outer ring-areas by 24, 247, and 330, respectively); and (iv) movements
                     of test animals were documented for 10 min.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_9_4"></a>4.9.4.&nbsp;Attraction assay protocol
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Design of the attraction assay was similar to the holding assay with the following modifications: (i) source animals were
                     left to condition a point source for 24 h; (ii) at the onset of each trial, individual hermaphrodites were placed atop conditioned
                     regions; and (iii) test males were introduced <span class="symbol">&#8764;</span>1.5 cm from the conditioned point source/unconditioned scoring region (see Figure 1c in <a href="#bib73">Simon and Sternberg, 2002</a>).
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec4_10"></a>4.10.&nbsp;Interaction of two chemosensory stimuli (interaction assay)
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Ikue Mori, Nagoya University, Nagoya, Japan, March, 2005.</p>
               <p>The experimental procedure of interaction assay was reported by <a href="#bib47">Ishihara et al. (2002)</a>.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_10_1"></a>4.10.1.&nbsp;Equipment and reagents
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The working bench should be horizontal. Avoid sources of wind or vibration-such as a centrifuge machine or a draft chamber
                     nearby. Do not use an incubator for the assay because it is strong source of wind of fan and vibration from the compressor.
                  </p>
                  <p>Room temperature should be kept at 25 &plusmn; 1 &deg;C.</p>
                  <p>6-cm seeded NGM plates. Use a 6-cm plate containing 14 ml of NGM, which is minimal volume required for getting sufficient
                     bacterial lawn. The NGM plates were spread with overnight-cultured <span class="emphasis"><em>E. coli</em></span>, incubated for a day at room temperature (or for overnight at 37&deg;C), and stored at room temperature. Do not use old seeded
                     NGM plates (over 2 weeks).
                  </p>
                  <p><span class="emphasis"><em>C. elegans</em></span>:  Use the healthy adult animals from uncrowded plates. Animals should be maintained by transferring 2 to 3 animals to a new
                     NGM plate before starving. Do not use the starved animals or those from a starved agar chunk.
                  </p>
                  <p>Assay plate: The medium consists of 2% agar, 50mM NaCl, 1mM CaCl<sub>2</sub>, 1mM MgSO<sub>4</sub>, and 10mM pH 7.0 HEPES. After autoclaving, pour 10 ml of the medium exactly into a 9-cm plate. Store at 4&deg;C and use within
                     2 weeks. Spread 25 ml of 100 mM (CH<sub>3</sub>COO)<sub>2</sub>Cu solution on the midline of an assay plates, and place at room temperature for 18-22 hr to allow diffusion before the assay.
                  </p>
                  <p>Wash buffer: 0.02% gelatin, 50mM NaCl, and 25mM pH 6.0 potassium phosphate.</p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec4_10_2"></a>4.10.2.&nbsp;Method
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p>Collect well-fed animals with wash buffer.</p>
                        </li>
                        <li>
                           <p>Wash animals with wash buffer three times.</p>
                        </li>
                        <li>
                           <p>Put animals as several drops on one side of the Cu<sup>2+</sup>barrier on the assay plate (30&#8211;90 animals) (See <a href="#figure2">Figure 2</a>).
                           </p>
                        </li>
                        <li>
                           <p>Leave the plates with lids for 90 min on the working bench.</p>
                        </li>
                        <li>
                           <p>Put several drops of chloroform on the lid to stop the animal.</p>
                        </li>
                        <li>
                           <p>Count the number of adult animals on each side of the barrier (<a href="#figure2">Figure 2</a>), and calculate the index. The index is defined as the percentage of animals on the odorant side divided by the total number
                              of animals.
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec5"></a>5.&nbsp;Learning, adaptation and habituation
                     </h2>
                  </div>
               </div>
            </div>
            <p>See also &#8220;<a href="#sec5_2_1" title="5.2.1.&nbsp;Adaptation assay">Adaptation</a>&#8221; within &#8220;<a href="#sec2_7" title="2.7.&nbsp;Nose touch">Nose touch response</a>&#8221; in the Mechanosensation section, &#8220;<a href="#sec4_3_1_6" title="4.3.1.6.&nbsp;Adaptation using the drop assay">Adaptation using the drop assay</a>&#8221; within the &#8220;<a href="#sec4_3" title="4.3.&nbsp;Drop assay">Drop assay</a>&#8221; in the Chemosensation section., &#8220;<a href="#sec5_6" title="5.6.&nbsp;Plasticity in chemotaxis using NaCl">Plasticity</a>&#8221; within &#8220;<a href="#sec4_1" title="4.1.&nbsp;Chemotaxis, chemoaversion and plasticity-quadrant plate, version 1">Chemotaxis, chemoaversion and plasticity, quadrant plate, version 1</a>&#8221; in the Chemosensation section., &#8220;<a href="#sec4_5" title="4.5.&nbsp;Distinguishing between odorants during cross-saturation">Distinguishing between odorants during cross-saturation</a>&#8221; and &#8220;<a href="#sec4_6" title="4.6.&nbsp;Decreased response after chronic exposure to attractant">Decreased response after chronic exposure to attractant</a>&#8221; within &#8220;<a href="#sec4_4" title="4.4.&nbsp;Chemotaxis to volatile point source">Chemotaxis to volatile point source</a>&#8221; in the Chemosensation section.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_1"></a>5.1.&nbsp;Learning, habituation and adaptation to tap
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Catharine Rankin, University of British Columbia, Vancouver, BC Canada, March, 2005.</p>
               <p>The following discussions are predicated on the description of the tap reflex in the Mechanosensation section.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_1_1"></a>5.1.1.&nbsp;Habituation to tap
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A number of decisions need to be made prior to running any experiment on habituation to tap: 1) The age of the worms to be
                     used; <a href="#bib18">Chiba and Rankin (1990)</a> showed that it is only after the end of the larval stages that worms consistently reverse to tap. It is best to use worms
                     of the same age for the experiments because for wild-type worms the length of the reversal response is directly related to
                     the length of the worm. 2) How many stimuli will be delivered; <a href="#bib66">Rankin and Broster (1992)</a> showed that the higher the number of stimuli the longer it took for spontaneous recovery to begin. 3) The interstimulus interval
                     (ISI or frequency) of the taps; over a series of experiments we have shown that habituation at short ISIs has different properties
                     from habituation at long ISIs and is probably mediated by some different mechanisms (reviewed in <a href="#bib68">Rose and Rankin (2001)</a>). 4) Whether there will be food (<span class="emphasis"><em>E. coli</em></span>) on the plate or not; for short-term habituation experiments there is not usually food on the plate; for long-term habituation
                     experiments that last 24 hours there usually is a small amount of food. And 5) at the end of the experiment will there be
                     tests for either spontaneous recovery or for dishabituation? Whatever decisions you make, the most important thing is to be
                     consistent. In addition it is important to always run control animals at the same time and under exactly the same conditions
                     as the experimental animals. That means run one experimental and then one control throughout the experiment. If you are testing
                     mutant strains the best control is the N2 strain from the lab where the mutant was made or into which it was backcrossed.
                     Alternatively a rescue strain might be a good control-however the high level of variability in some rescue strains makes this
                     sometimes problematic.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_1_2"></a>5.1.2.&nbsp;Spontaneous recovery
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>When worms that have been habituated to tap are left unstimulated for a period of time they show spontaneous recovery of the
                     tap response. To look at spontaneous recovery worms are left on the microscope stage and are given test stimuli at various
                     intervals after habituation training to determine the extent to which the response has recovered. Spontaneous recovery can
                     be used to distinguish habituation from sensory adaptation or motor fatigue. Worms habituated at short ISIs show great habituation,
                     but recover more rapidly (within <span class="symbol">&#8764;</span>10 min) than worms habituated at long ISIs (at least 30 min), which show less overall habituation. This rapid recovery following
                     habituation at short ISIs is the opposite of what would be predicted for adaptation or fatigue (where the most decremented
                     group should recover the most slowly).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_1_3"></a>5.1.3.&nbsp;Dishabituation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Dishabituation can also be used to distinguish habituation from sensory adaptation or motor fatigue. In dishabituation the
                     habituated response is rapidly recovered by the delivery of a dishabituating stimulus that is usually a strong novel or noxious
                     stimulus. In this protocol, the dishabituating stimulus is a brief train of electric shocks, generated by the Grass S88 stimulator.
                     The shocks are delivered using a hand-held spanning electrode (a bipolar electrode used for stimulating rat brains) with wires
                     placed into the agar on either side of the worm, about 3 to 4 worm widths apart. The 2 forks of the electrode thus span the
                     animal, piercing the agar about 1 worm width away on each side of the cuticle, at about the middle of the worm's length. Neither
                     fork of the electrode actually contacts the cuticle--the shock is transmitted solely through the agar. The train of shocks
                     usually consists of 10 ms shocks of 60 V at 10 PPS for 600 ms. It is important to decide on the appropriated delivery time
                     for the shock-the worms require about 30 seconds to recover from the shock, and so it is important to give them time to recover
                     before the first tap. However, if there is a long interval between the last tap of habituation and the first tap following
                     shock it is important to control for any spontaneous recovery that might have occurred in the interval by having a control
                     group in which the same interval is given, but no shock delivered. Following this dishabituating stimulus an additional set
                     of 10 taps at the same ISI as training, is given to assess the effect of the shock on the response that was habituated. Dishabituation
                     is shown by the increase in response amplitude above habituated levels following the shock.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_1_4"></a>5.1.4.&nbsp;Long-term memory for habituation
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>To assess long-term memory for habituation tap stimuli are delivered in a distributed or spaced training procedure. For training
                     approximately 20 young adult worms are pre-plated on an agar plate containing a thin lawn of E. coli 24 hours prior to training.
                     A second plate for control worms is pre-plated at the same time. Both plates are left on a vibration-free shelf in the lab
                     over night. Twenty-four hours after pre-plating the experimental plate is placed on the stage of the dissecting scope and
                     taps are delivered. The worms receive four blocks of 20 taps each. With in a block taps are delivered at a 60 s ISI, and between
                     blocks there is a 1-hour rest period (<a href="#bib67">Rose et al., 2002</a>). The control plate of worms receives a single tap at the end of the training period. One hour after the end of training
                     worms from the experimental and control plates are transferred to individual Petri plates (again with a thin lawn of E. coli)
                     and left on the vibration free shelf for approximately 22 hours. To test for long-term memory for habituation each worm (both
                     experimental and control) is given a set of 10 taps at a 60 s ISI. Testing can begin about 22 hours after training and continue
                     until 26 hours after training. Responses to the test taps are scored and long-term memory is considered to be present when
                     the mean response magnitude of the trained or experimental worms is significantly smaller than the mean response magnitude
                     of the control worms. In these experiments it is usual to have at least 20 worms per group. It is possible to test drug effects
                     on memory by delivering the drug through the agar that the worm is on during training (for drug experiments worms can be pre-plated
                     on drug containing plates 1 hour prior to training). For drug experiments there are four groups of worms tested: drug trained
                     and control groups, and vehicle trained and control groups. Worms are removed from the drug plates 1 hour after training is
                     completed and tested approximately 24 hours later on plates without drug. The drug is considered to have blocked memory if
                     the vehicle group shows normal long-term memory with trained worms having smaller responses to the 10 test taps than control
                     worms, and if the drug group shows no difference between trained and control worms. In addition, to show that the drug did
                     not damage the worms in some way it is important that the control worms for the drug and vehicle groups do not differ from
                     one another.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_2"></a>5.2.&nbsp;State-dependent adaptation to volatile odorants
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Jill Bettinger (<a href="#bib10">Bettinger and McIntire, 2004</a>) as a modification of Cori Bargmann's original adaptation protocol (<a href="#bib19">Colbert and Bargmann, 1995</a>), March, 2005.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_1"></a>5.2.1.&nbsp;Adaptation assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Adaptation plates are prepared as follows:</p>
                  <p>10 mLs of adaptation agar (3% agar, 5mM KPO<sub>4</sub> [pH6], 1 mM CaCl<sub>2</sub>, 1 mM MgSO<sub>4</sub>) or chemotaxis assay agar (2% agar, 5mM KPO<sub>4</sub> [pH6], 1 mM CaCl<sub>2</sub>, 1 mM MgSO<sub>4</sub>) is aliquoted into 10 cm Petri plates, and allowed to dry overnight. The day of the experiment, all plates are dried at 37&deg;C
                     without lids for 1 hour, and a representative plate of each type is melted to determine volume. Ethanol is added to appropriate
                     plates to a concentration of 200 mM, the plates are Parafilmed and allowed to soak for 1.5&#8211;2 hours. Odorant is aliquoted onto
                     5 agar plugs on the lid of adaptation plates (for benzaldehyde adaptation, 2 <span class="symbol">&#956;</span>L of 100% benzaldehyde is used; for butanone, 5 <span class="symbol">&#956;</span>L of 100% butanone is used). Between 100 and 300 animals are washed twice in S Basal and once in assay buffer (5mM KPO<sub>4</sub> [pH6], 1 mM CaCl<sub>2</sub>, 1 mM MgSO<sub>4</sub>), then placed on an adaptation plate, and the plate is sealed with Parafilm. Animals are incubated in adaptation conditions
                     for 90 minutes, then washed twice with S Basal and once with assay buffer and placed on chemotaxis plates. Chemotaxis is allowed
                     to proceed for 1 hour, worms are counted, and a chemotaxis index is calculated. Adaptation is observed as a decrease in chemotaxis
                     to an odorant after pretreatment with the odorant.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_2_2"></a>5.2.2.&nbsp;Chemotaxis assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Chemotaxis assay plates are prepared as follows: 10 mLs of assay agar (2% agar, 5mM KPO<sub>4</sub> [pH6], 1 mM CaCl<sub>2</sub>, 1 mM MgSO<sub>4</sub> is aliquoted into 10 cm Petri plates, and allowed to dry overnight. The day of the experiment, all plates are dried at 37&deg;C
                     without lids for 1 hour. If ethanol is to be used in the experiment, one representative plate is melted to determine final
                     volume, and ethanol is added to the desired final concentration (generally 200 mM). The ethanol plates are sealed with Parafilm,
                     and ethanol is allowed to soak into the agar for 1.5 -2 hours. Between 100 and 300 animals are washed twice in S Basal and
                     once in assay buffer (5 mM KPO<sub>4</sub> [pH6], 1 mM CaCl<sub>2</sub>, 1 mM MgSO<sub>4</sub>). A spot of diluted odorant (for benzaldehyde, 1 <span class="symbol">&#956;</span>L of 1:200 benzaldehyde: ethanol) is placed on one side of the plate. Exactly opposite is placed a spot of diluent (1 <span class="symbol">&#956;</span>L ethanol). To each spot is added the anesthetic sodium azide  (1 <span class="symbol">&#956;</span>L of 1 M sodium azide) to immobilize the worms when they reach a spot. Washed worms are then placed onto a spot on the plate
                     that is exactly between the odorant and diluent spots and slightly off center. Excess liquid is wicked up with the corner
                     of a Kimwipe. After 1 hour, worms are counted, and a chemotaxis index is calculated as follows: CI= (# of worms at the odorant
                     spot - # of worms at the diluent spot)/ # of worms in the assay. A high chemotaxis index (close to 1) indicates that the odorant
                     acts as a strong attractant, a lower chemotaxis index indicates that the odorant is a less effective attractant or does not
                     act as an attractant.
                  </p>
                  <div class="sidebar">
                     <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                        <h3 class="title">Notes</h3>
                        <p>I have used benzaldehyde, butanone and isoamyl-alcohol.</p>
                        <p>When making adaptation or chemotaxis agar, KP0<sub>4</sub>, CaCl<sub>2</sub> and MgSO<sub>4</sub> are prepared separately and added after autoclaving.
                        </p>
                        <p>I always dry plates in a single layer on the bench-plates in a stack dry unevenly.</p>
                        <p>State-dependency is extremely sensitive to overtraining. I found that I could only see complete state-dependency of adaptation
                           when the adapted chemotaxis index was between about 0.35 and 0.6. Below 0.35, there is definite evidence of state-dependency,
                           but the results are more difficult to interpret. I would aim for adapted CI values around 0.4&#8211;0.5.
                        </p>
                     </div>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_3"></a>5.3.&nbsp;Ethanol intoxication
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Andrew G. Davies and Steven L. McIntire, UCSF and Ernest Gallo Clinic and Research Center, San Francisco, CA,
                  USA, March, 2005.
               </p>
               <p>Cited in: <a href="#bib26">Davies et al. (2003)</a>, <a href="#bib24">Davies et al. (2004)</a>, and detailed in <a href="#bib25">Davies and McIntire (2004)</a>.
               </p>
               <p>Standard NGM agar plates (6 cm) are dried for 2 hours at 37&deg;C without their lids. For quantitative locomotion assays, unseeded
                  plates are preferable. Seeded plates should be used for egg-laying assays. For quantitative locomotion assays, the worms are
                  first placed on unseeded plates containing no ethanol, so twice the number of plates to be used in the assay should be dried
                  plus at least one more for use in calculating the volume of the media in each plate.
               </p>
               <p>Determine the volume of the media in one of the dried plates by collecting the media into a 15 ml tube and melting it in a
                  microwave for 10-15 seconds. Assume that all plates have the same volume of media (use plates poured in the same batch).
               </p>
               <p>It is possible to test multiple strains on the same ethanol plate by using copper rings as corrals (#48 cap thread gaskets,
                  1.6 cm inner diameter; STK#35583, PlumbMaster, Concordville, PA). This approach helps to reduce some of the plate-to-plate
                  variation. To do this, heat a copper ring in a flame for 1-2 seconds using forceps and place the ring on the surface of the
                  plate. Apply a small amount of downward pressure on the ring so that it melts into the surface of the plate evenly, creating
                  a seal when the agar solidifies. Three to four rings can be placed on a 6 cm Petri plate.
               </p>
               <p>Ethanol at 4&deg;C is then added to the dried plates. The volume of the agar from the melted plate is used to determine the volume
                  of ethanol to be added. For instance, 232 <span class="symbol">&#956;</span>l of 100% ethanol is added to 10 ml of agar media to achieve a final concentration of 400 mM ethanol. The ethanol should be
                  added outside of the copper rings, especially for seeded plates. Seal the plates with Parafilm and allow the ethanol to equilibrate
                  in the media for 2 hours at room temperature.
               </p>
               <p>Typically, 1st-day adult hermaphrodites are tested. Animals are synchronized by age by picking L4 stage animals 20-24 hours
                  prior to the assay.
               </p>
               <p>For locomotion assays, 10 worms of each genotype to be tested are transferred using minimal bacteria to one copper ring per
                  genotype on unseeded plates containing no ethanol. The animals are allowed 30 minutes to become accustomed to a lack of food
                  before being transferred without food to the assay plates. The plates are sealed with Parafilm and speed measurements are
                  collected 10-50 minutes after the animals are transferred, typically after 20 minutes exposure. 3-4 genotypes (10 worms each)
                  can be transferred in less than 2 minutes, which means that animals of different genotypes can be compared directly with minimal
                  difference in the time of intoxication.
               </p>
               <p>A quantitative measure of the speed of the intoxicated animals is measured by first creating a 2-minute time-lapse movie (1
                  frame per second) that includes all of the copper rings in the frame. This is don using a video camera attached to a computer
                  with a video frame grabber and appropriate software for collecting images for a movie such as Openlab (Improvision). On a
                  Apple computer, each genotype displayed in the movie can then be analyzed using the DIAS software application (Soll Technologies
                  Inc.), which tracks each worm as a separate object from frame to frame and measures the distance traveled over time (average
                  speed). As some genotypes move at different basal rates, calculating the speed of a strain on ethanol relative to its speed
                  off ethanol can be useful for comparing strains.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_4"></a>5.4.&nbsp;Attraction to chemicals previously paired with attractive stimuli and avoidance of chemicals previously paired with starvation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Eric Law and Derek Van Der Kooy, University of Toronto, Toronto, Canada, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_4_1"></a>5.4.1.&nbsp;Food-odor associative learning
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>We demonstrated that <span class="emphasis"><em>C. elegans</em></span> is able to form an association between the presence of the odorant benzaldehyde and the food content of its environment.
                     When exposed to 100% benzaldehyde for 1 h in the absence of food the naive attractive response is reduced. In contrast, the
                     benzaldehyde attractive response is strengthened when animals are exposed to 100% benzaldehyde for 1 h in the presence of
                     food. These results suggest that animals can form an association between the odorant benzaldehyde and food status (<a href="#bib92">Nuttley et al., 2002</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_4_2"></a>5.4.2.&nbsp;Strains and general methods
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Nematode strains were obtained from the <span class="emphasis"><em>Caenorhabditis</em></span> Genetics Center at the University of Minnesota (St. Paul). All experiments used well fed adults cultivated at 20&deg;C on standard
                     nematode growth medium (NGM plates) seeded with <span class="emphasis"><em>E. coli</em></span> strain OP50.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_4_3"></a>5.4.3.&nbsp;Chemotaxis assays
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Assays were performed with standard 100-mm Petri dishes containing 6 m20/15g/liter agar). Where necessary, odorants were diluted
                     in ethanol and l of CTX medium (10 mM Mops, pH 7.2/0.25% (vol/vol) Tween reported as percentages by volume. Plates were sealed
                     with a strip of Parafilm around the edge during all odorant exposures. Briefly, 15 min before the assay 1 ml of 1 M NaN<sub>3</sub> was applied to the centers of two test spots that were 6 cm apart. This acts as an anesthetic to immobilize any animals that
                     reach the spot during the assay. Individuals (100-300) then were placed at the center of the plate between the two spots,
                     1 ml of the test odorant was placed at one spot, and 1 ml of ethanol was applied to the control spot. After 1 h of chemotaxis,
                     animals within a 2-cm radius of either spot were counted, and a chemotactic index (CI) was calculated as the number of animals
                     at the test spot minus the number of animals at the control spot and divided by the total number of animals on the plate.
                     A positive CI indicates an attraction to the odor, and a negative CI indicates an aversion.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_4_4"></a>5.4.4.&nbsp;Benzaldehyde-starvation learning
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>To measure the effects of pre-exposing animals to starvation-benzaldehyde pairing, 500-1,000 animals were placed on a CTX
                     plate, and then 2 ml of the odorant were placed on a piece of Parafilm on the lid of the plate. The plate was sealed with
                     Parafilm and then left inverted for 1 h. The animals then were rinsed from the test plate in approximately 2 ml of water and
                     transferred to a conical centrifuge tube, where they were allowed to settle to the bottom of the tube. Within 5 min, the concentrated
                     animals were collected and transferred to a fresh CTX plate for testing.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_4_5"></a>5.4.5.&nbsp;Benzaldehyde-food learning
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>For assays involving food-benzaldehyde association on the CTX plates, cells from a 6X concentration of logarithmically growing
                     <span class="emphasis"><em>E. coli</em></span> culture were spread onto the plate and allowed to dry immediately before pre-exposure. A 1X concentration of <span class="emphasis"><em>E. coli</em></span> was defined as an amount equivalent to 100 ml of culture diluted to an OD600 of 0.5. As with many food-related behavior,
                     exogenous serotonin (5-HT) can substitute for food in this assay. For preparation, 5-HT salt (Sigma) was diluted in M9 buffer
                     for a final concentration of 100mM. 100<span class="symbol">&#956;</span>L of this solution was spread onto the plate and allowed to dry immediately before benzaldehyde pre-exposure.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_5"></a>5.5.&nbsp;Chemotaxis to soluble point source
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Makoto Koga, Kyushu University, Fukuoka, Japan, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_5_1"></a>5.5.1.&nbsp;Protocol
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Assays are undertaken with well-fed, 1 day old adult animals raised at 20 degrees Celsius. Plates are closed during the assay
                     <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=che-1" target="_blank">che-1</a></em></span> and N2 animals are used as controls. The  chemotaxis assay were performed essentially as described by <a href="#bib30">Dusenbery et al. (1975)</a> with some modifications (<a href="#bib79">Uchida et al., 2003</a>). A radial concentration gradient of NaCl is established by spotting 2<span class="symbol">&#956;</span>l of 5M NaCl to the center of 9cm plate containing 8 ml of agar medium [2% agar, 0.25% Tween20, 10mM HEPES(pH7.2)], and leaving
                     the plate at room temperature for 12-16 hr. For the assay, the animals are placed on the surface of agar, 1 cm distant from
                     the periphery and allowed to move freely for 1hr.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_5_2"></a>5.5.2.&nbsp;Recording results
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Put the agar assay plate 1cm away from frosted glass and light it from agar plate side using a slide projector set 2-3m distant.
                     Worm tracks appear as darkened tracks on the frosted glass. These are photographed using a digital camera.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_6"></a>5.6.&nbsp;Plasticity in chemotaxis using NaCl
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Ikue Mori, Nagoya University, Nagoya Japan, March, 2005.</p>
               <p>The experimental procedure of plasticity of chemotaxis to NaCl was reported by <a href="#bib70">Saeki et al. (2001)</a> and <a href="#bib47">Ishihara et al. (2002)</a>.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_6_1"></a>5.6.1.&nbsp;Equipment and reagents
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The working bench should be horizontal. Avoid sources of wind and vibration such as a centrifuge machine or a draft chamber
                     nearby. Do not use an incubator for the assay because it is strong source of wind of fan and vibration from compressor.
                  </p>
                  <p>Room temperature should be kept at 25&plusmn;1&deg;C.</p>
                  <p>6-cm seeded NGM plates. Use a 6-cm plate containing 14 ml of NGM, which is minimal volume required for getting sufficient
                     bacterial lawn. The NGM plates were spread with overnight-cultured E. coli, incubated for a day at room temperature (or for
                     overnight at 37&deg;C), and stored at room temperature. Do not use old seeded NGM plates (over 2 weeks).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_6_2"></a>5.6.2.&nbsp;<span class="emphasis"><em>C. elegans</em></span></h4>
                        </div>
                     </div>
                  </div>
                  <p>Use the healthy adult animals from uncrowded plates. Animals should be maintained by transferring 2 to 3 animals to a new
                     NGM plate before starving. Do not use the starved animals or those from a starved agar chunk.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_6_3"></a>5.6.3.&nbsp;Plates
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Conditioning plate: The medium consists of 3% agar, 50mM NaCl, and 10mM pH 7.2 MOPSxNH<sub>4</sub>. After autoclaving, pour 8 ml of the medium exactly into a 9-cm plate. Store at 4&deg;C and use in 3 weeks. (See <a href="#figure3">Figure 3</a>.)
                  </p>
                  <p>Mock conditioning plate: The medium consists of 3% agar and 10mM pH 7.2 MOPSxNH<sub>4</sub>. After autoclaving, pour 8 ml of the medium exactly into a 9-cm plate. Store at 4&deg;C and use in 3 weeks.
                  </p>
                  <p>Assay plate: The medium consists of 2% agar and 10mM pH 7.2 MOPSxNH<sub>4</sub>. After autoclaving, pour 3<span class="symbol">&#8764;</span>3.5 ml of the medium exactly into a 6-cm plate. Store at 4&deg;C and use in 3 weeks.
                  </p>
                  <p>NaCl plug: The medium consists of 2% agar, 50mM NaCl, and 10mM pH 7.2 MOPSxNH<sub>4</sub>. After autoclaving, pour 6 ml of the medium exactly into a 6-cm plate. Store at 4&deg;C and use in 3 weeks. To make NaCl-gradient
                     on an assay plate, cut a plug (5 mm in diameter) from the 50mM NaCl plate and put a plug on one side of an assay plate for
                     19<span class="symbol">&#8764;</span>23 hr (<a href="#figure4">Figure 4</a>). After making NaCl gradient, discard the plug from assay plate before chemotaxis assay. You may use the Pasteur pipette
                     (5 mm in diameter) to cut and put a NaCl-plug. Wash buffer: 0.5g per liter gelatin, 1mM CaCl<sub>2</sub>, 1mM MgSO<sub>4</sub>, and 25mM pH 6.0 potassium phosphate. Store at room temperature.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_6_4"></a>5.6.4.&nbsp;Method
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p>Collect well-fed animals with wash buffer.</p>
                        </li>
                        <li>
                           <p>Wash animals with wash buffer three times.</p>
                        </li>
                        <li>
                           <p>Transfer animals to the conditioning plate or mock-conditioning plate.</p>
                        </li>
                        <li>
                           <p>Place these plates with lids in the incubator at 20&deg;C for 4 hr.</p>
                        </li>
                        <li>
                           <p>Collect conditioned or mock-conditioned animals with wash buffer.</p>
                        </li>
                        <li>
                           <p>Put animals on the center of an assay plate (50<span class="symbol">&#8764;</span>300 animals).
                           </p>
                        </li>
                        <li>
                           <p>Leave the plates with lids for 15 min on the working bench.</p>
                        </li>
                        <li>
                           <p>Put several drops of chloroform on the lid to stop the animal.</p>
                        </li>
                        <li>
                           <p>Count the number of adult animals at each fraction, and calculate the chemotaxis index (<a href="#figure4">Figure 4</a>). You should not count clump worms on the midline of the assay plate because these clumps mainly contain wash-injured animals.
                              Chemotaxis index is calculated as (A <span class="symbol">&#8722;</span> B) / (A + B) where A was the number of animals on the NaCl side of the plate and B was the number of animals on the other
                              side.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure3"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="WMBehavFig3.jpg"><img src="WMBehavFig3_s.jpg" border="2" align="middle" alt="WormBook Behavior figure 3"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 3.&nbsp;</b></p>
                     </div>
                  </div>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure4"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="225">
<tr>
  <td align="center"><p><a href="WMBehavFig4.jpg"><img src="WMBehavFig4_s.jpg" border="2" align="middle" alt="WormBook Behavior figure 4"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 4.&nbsp;</b>Chemotaxis index is calculated as (A <span class="symbol">&#8722;</span> B)/(A + B) where A is the number of animals on the NaCl side of the plate and B is the number of animals on the other side.
                        </p>
                     </div>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec5_7"></a>5.7.&nbsp;Context dependency in olfactory adaptation
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Eric Law and Derek Van Der Kooy, University of Toronto, Toronto, Canada, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_7_1"></a>5.7.1.&nbsp;Brief description
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>We demonstrated that olfactory adaptation (benzaldehyde-starvation learning) is taste context dependent. (<a href="#bib53">Law et al., 2004</a>) Animals adapted in a sodium acetate context require the same sodium acetate taste context for adaptation response. The same
                     context effects can be observed if animals are trained in an ammonium chloride context or the standard NGM context.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_7_2"></a>5.7.2.&nbsp;General methods
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>All experiments used well-fed adults cultivated at 20&deg;C on normal growth medium (NGM; 50 mM NaCl, 15 g/l agar, 20 g/l peptone,
                     1 mM cholesterol, 1 mM CaCl<sub>2</sub>, 1 mM MgSO4, 1 mM KP<span class="emphasis"><em>i</em></span> [pH = 7.0]) seeded with the OP50 <span class="emphasis"><em>E. coli</em></span> strain. Minimal medium was 15 g/l agar, 20 g/l peptone, 1mM KP<span class="emphasis"><em>i</em></span> (pH = 7.0). CTX medium was 10 mM Mops (pH 7.2), 0.25% v/v Tween 20, 15 g/l agar with NaAc or NH<sub>4</sub>Cl added to a final concentration of 25 mM where indicated.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_7_3"></a>5.7.3.&nbsp;Na/Cl context effects
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>N2 worms received benzaldehyde adaptation training in either the NaAc or NH<sub>4</sub>Cl added to a CTX medium for a final concentration of 25 mM. Plates were poured one day before experiments and allowed to
                     dry.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec5_7_4"></a>5.7.4.&nbsp;NGM/minimal context effects
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>In a second set of experiments, we defined the contexts using the standard NGM context and a minimal context (with all the
                     cues in the NGM context removed except for 15 g/l agar and 20 g/l peptone). The olfactory adaptation learning assay was carried
                     out as described by <a href="#bib92">Nuttlet et al. (2002)</a>.
                  </p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec6"></a>6.&nbsp;Thermal responses
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6_1"></a>6.1.&nbsp;Thermotaxis-tracking themoclines
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Ikue Mori, Nagoya University, Nagoya, Japan, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_1_1"></a>6.1.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Thermotaxis is one of the interesting behaviors in <span class="emphasis"><em>C. elegans</em></span>. This behavior, reported first by <a href="#bib40">Hedgecock and Russell (1975)</a>, should require some mechanisms for sensing and memorizing temperature, which are still largely unknown. Thermotaxis is also
                     a behavior difficult to assay, because this behavior is influenced by the environmental conditions including temperature,
                     population density, and feeding status. Animals avoid temperatures at which they have been starved and accumulate at temperatures
                     associated with food. However, the critical control of the environmental conditions enable us analyze this challenging behavior.
                     This protocol describes a series of tips for thermotaxis (Ttx) assays (<a href="#bib59">Mori and Ohshima, 1995</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_1_2"></a>6.1.2.&nbsp;Radial thermotaxis (Ttx) assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Equipment and reagents: The working bench should be horizontal. There should be no source of wind and vibration nearby-such
                     as a centrifuge machine or a draft chamber. Do not use an incubator for Ttx assays because it is strong source of wind of
                     fan and vibration from the compressor.
                  </p>
                  <p>Room temperature should be kept at 25 &plusmn; 1 &deg;C. Avoid local temperature changes caused by sunlight. Low humidity (20 <span class="symbol">&#8764;</span> 30%) is preferred for assay.
                  </p>
                  <p>Incubator to grow animals:  A small volume (100 <span class="symbol">&#8764;</span> 150 L) incubator is preferred to get stable temperature.
                  </p>
                  <p>6-cm seeded NGM plates. Use a 6-cm plate containing 14 ml of NGM, which is minimal volume required for getting sufficient
                     bacterial lawn. The NGM plates were spread with overnight-cultured E. coli, incubated for a day at room temperature (or for
                     overnight at 37&deg;C), and stored at room temperature. Do not use old seeded NGM plates (over 2 weeks).
                  </p>
                  <p><span class="emphasis"><em>C. elegans</em></span>: Use the healthy adult animals from uncrowded plates. Animals should be maintained by transferring 2 to 3 animals to a new
                     NGM plate before starving. Do not use the starved animals or those from a starved agar chunk. To avoid crowding, you may transfer
                     <span class="symbol">&#8764;</span> 15 of L4 larvae to a fresh and pre-incubated NGM plate. Grow those for 8 <span class="symbol">&#8764;</span> 24 hr (depend on cultivation temperature) and use for assays.
                  </p>
                  <p>9-cm Ttx plates. Ttx medium consists of 2% agar, 0.3% NaCl and 25 mM potassium phosphate buffer (pH 6.0). After autoclaving,
                     pour 8 ml of the medium exactly into a 9-cm plate. Store at 4&deg;C and use within 2 weeks.
                  </p>
                  <p>Vials filled with frozen glacial acetic acid (stored at 4&deg;C). Glass vials are 10-cm long and 2.7-cm diameter. Freeze glacial
                     acetic acid at -20&deg;C and store at 4&deg;C.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_1_3"></a>6.1.3.&nbsp;Method
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p>Remove lids of Ttx plates and dry the plates for <span class="symbol">&#8764;</span> 30 min at room temperature. Do not dry Ttx plates in a windy place like a clean bench or a draft chamber. Overdrying Ttx
                              plates causes the plates' surface to become rough. The surface of Ttx plates should be smooth as a mirror.
                           </p>
                        </li>
                        <li>
                           <p>Place the Ttx plates with lids upside-down; mark center and a point of 1.5 cm from edge.</p>
                        </li>
                        <li>
                           <p>Take the vials containing frozen glacial acetic acid out from cold room and leave the vials for 10 min at room temperature
                              (25&deg;C) to allow acetic acid starting to melt. The melting point of glacial acetic acid is 16.7&deg;C.
                           </p>
                        </li>
                        <li>
                           <p>Put the vial on the center of Ttx plate.</p>
                        </li>
                        <li>
                           <p>Leave the plates for 10 <span class="symbol">&#8764;</span> 15 min to form stable radial thermal gradient on the surface of Ttx plate. The thermal gradient is kept over 1 hr under the
                              vial with melting glacial acetic acid. Renew the vials when the half of the glacial acid is melted.
                           </p>
                        </li>
                        <li>
                           <p>Remove the vial and place an animal on the point of Ttx plate. Do not scratch the surface of the Ttx plate. Avoid bringing
                              E. coli onto the Ttx plate. For example, move an animal to the place without bacterial lawn in NGM plate (or unseeded NGM
                              plate) and transfer the animal to the Ttx plate.
                           </p>
                        </li>
                        <li>
                           <p>Put the vial on the center of Ttx plate again.</p>
                        </li>
                        <li>
                           <p>Leave the plates for 50 <span class="symbol">&#8764;</span> 60 min. Do not walk around during assay to avoid wind and vibration.
                           </p>
                        </li>
                        <li>
                           <p>Remove the vial and put several drops of chloroform on the lid to stop the animal.</p>
                        </li>
                        <li>
                           <p>Place the Ttx plates at 4 &deg;C until photographed.</p>
                        </li>
                     </ol>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_1_4"></a>6.1.4.&nbsp;Preparation of starved animals
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="orderedlist">
                     <p class="title">Equipment and reagents</p>
                     <ol type="1">
                        <li>
                           <p>9-cm starvation plates. The medium consists of 2% agar, 1mM CaCl<sub>2</sub>, 1mM MgSO<sub>4</sub>, and 25mM pH 6.0 potassium phosphate. After autoclaving, pour 8 ml of the medium exactly into a 9-cm plate. Store at 4&deg;C
                              and use in a month.
                           </p>
                        </li>
                        <li>
                           <p>NG buffer. NG buffer consists of 0. 3% NaCl, 1mM CaCl<sub>2</sub>, 1mM MgSO<sub>4</sub>, and 25mM pH 6.0 potassium phosphate.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <div class="procedure"><a name="d0e2189"></a><p class="title">Procedure&nbsp;5.&nbsp;Starvation method 1</p>
                     <ol type="1">
                        <li>
                           <p>Incubate NG buffer and starvation plates at the temperature for assay over 3 hr.</p>
                        </li>
                        <li>
                           <p>Collect well-fed animals with pre-incubated NG buffer.</p>
                        </li>
                        <li>
                           <p>Wash animals with NG buffer twice in the water bath at the temperature for assay.</p>
                        </li>
                        <li>
                           <p>Transfer animals to the pre-incubated starvation plate.</p>
                        </li>
                        <li>
                           <p>Seal the starvation plate with Parafilm.</p>
                        </li>
                        <li>
                           <p>Incubate at the temperature for assay for appropriate time (at 17&deg;C for 2 <span class="symbol">&#8764;</span> 3hr or at 25&deg;C for 30 min).
                           </p>
                        </li>
                     </ol>
                  </div>
                  <div class="procedure"><a name="d0e2213"></a><p class="title">Procedure&nbsp;6.&nbsp;Starvation method 2</p>
                     <ol type="1">
                        <li>
                           <p>Incubate starvation plate at assay temperature over 3 hr.</p>
                        </li>
                        <li>
                           <p>Transfer 20 animals to the pre-incubated starvation plate.</p>
                        </li>
                        <li>
                           <p>Seal the starvation plate with Parafilm.</p>
                        </li>
                        <li>
                           <p>Incubate at 17&deg;C for 2 <span class="symbol">&#8764;</span> 3hr or at 25&deg;C for 30 min.
                           </p>
                        </li>
                        <li>
                           <p>Use animals for Ttx assay.</p>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6_2"></a>6.2.&nbsp;Microdroplet assay for thermotaxis
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Will Ryu, Chris Gabel, Aravi Samuel, Harvard University, Cambridge, MA, USA, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_2_1"></a>6.2.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>C. elegans</em></span> crawling is characterized by a series of forward movement punctuated by occasional backward movements and abrupt turns. A
                     swimming worm suspended in a microdroplet will also execute these movements spontaneously. There are several advantages to
                     studying worm behavior by examining swimming patterns in microdroplets: 1) The movements of a single worm can be monitored
                     for an extended period of time using a stationary video camera. 2) Thermal stimuli are easily delivered to the microdroplet.
                     3) The movements of the swimming worm can be quantified objectively using machine vision software.
                  </p>
                  <p>We have developed a microdroplet-based thermotactic assay that effectively quantifies the behavioral output of individual
                     swimming worms responding to defined thermal stimuli. (<a href="#bib69">Ryu and Samuel, 2002</a>; <a href="#bib71">Satterlee et al., 2004</a>; <a href="#figure5">Figure 5</a>).
                  </p>
                  <div class="mediaobject" align="center"><a name="figure5"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="WMBehavFig5.jpg"><img src="WMBehavFig5_s.jpg" border="2" align="middle" alt="WormBook Behavior figure 5"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 5.&nbsp;</b></p>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_2_2"></a>6.2.2.&nbsp;Thermal stage
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The thermal stage is designed for use on an inverted microscope. The worm swims freely in a microdroplet sandwiched between
                     a cover slip from above and a round sapphire window from below. The typical microdroplet contains <span class="symbol">&#8764;</span>0.5 <span class="symbol">&#956;</span>L of NGM buffer (same salt concentrations as NGM media) that flattens to a circular droplet <span class="symbol">&#8764;</span>2 mm in diameter and <span class="symbol">&#8764;</span>300 micrometers in thickness within which we place a single young adult worm. (For monitoring the swimming behavior of a single
                     worm in the microdroplet, a low magnification objective (1-4x) is useful. But this thermal stage can also be used for physiological
                     measurements of neuronal activity using high-numerical aperture objectives.)
                  </p>
                  <p>The single-crystal sapphire window (15.9 mm diameter, 1 mm thick; Rolyn Optics, Inc) is embedded in a slab of copper such
                     that the upper surfaces of the sapphire and copper are flush. We use copper and single-crystal sapphire because of their excellent
                     thermal conductivities. A microthermocouple - either embedded in the copper near the window or in contact with either the
                     sapphire or the droplet itself - is used to continuously monitor the actual temperature and to provide feedback to the control
                     electronics. A suitable thermocouple is the IT-23 Teflon-coated Type T thermocouple made by Physitemp Instruments Inc.
                  </p>
                  <p>A Peltier-effect 85 Watt thermoelectric element (5 cm x 5 cm) sandwiched between the copper stage and a brass, water-cooled
                     heat sink provides feedback-controlled heating/cooling. The thermoelectric controller we use is the Thorlabs TEC2000 (Thorlabs,
                     Newton, NJ), which provides adequate power and straightforward computer interfacing, but many other options exist. To minimize
                     the temperature gradient across the Peltier element (and thus the power the thermoelectric controller has to deliver), we
                     also use a temperature-controlled water bath (Model 1160S Heated/Refrigerated Circulator, VWR) to fix the temperature of the
                     heat sink to an intermediate value of the actual experiments.
                  </p>
                  <p>An alternate microdroplet configuration consists of suspending 50 <span class="symbol">&#956;</span>L of NGM buffer from a 4 mm diameter hole in an temperature controlled anodized aluminum plate. The top of the hanging droplet
                     is covered with a glass coverslip and the bottom of the droplet is left exposed. The microdroplet can be imaged from above
                     using a video camera with a zoom lens or a stereomicroscope. Worms swim naturally but remain in the center of the droplet
                     due to the droplet's curvature. There is some evaporative loss, but if the droplet volume is maintained, a single worm can
                     be studied for many hours.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_2_3"></a>6.2.3.&nbsp;Computer software
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>We have written software in the LabVIEW programming environment (National Instruments, Austin, TX) to simultaneously control
                     and monitor the temperature of the droplet and also capture images of the worm. Thermal stimulation of the swimming worm is
                     achieved via defined voltage waveforms sent to the thermoelectric controller, using a multifunction data acquisition card
                     [PCI-6024E]. Movements of the worm swimming in the microdroplet are imaged with a CCD camera and captured to the computer
                     with a PCI-1407 Image Acquisition card. We find that a simple and effective metric of behavioral performance is the nose-to-tail
                     distance of the swimming worm. During forward swimming, bending waves propagate from nose-to-tail and the nose-to-tail distance
                     oscillates just below the contour length of the worm. During reorientations (either sharp turns or reversals), the nose-to-tail
                     distance decreases significantly. (See <a href="#figure6">Figure 6</a> and <a href="#figure7">Figure 7</a>).
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure6"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="WMBehavFig6.jpg"><img src="WMBehavFig6_s.jpg" border="2" align="middle" alt="WormBook Behavior figure 6"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 6.&nbsp;</b></p>
                     </div>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure7"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="405">
<tr>
  <td align="center"><p><a href="WMBehavFig7.jpg"><img src="WMBehavFig7_s.jpg" border="2" align="middle" alt="WormBook Behavior figure 7"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 7.&nbsp;</b></p>
                     </div>
                  </div>
                  <p>We wrote image-processing software to calculate the nose-to-tail distance of a swimming worm using the LabVIEW Vision Development
                     Module. Since the software works best with a high-contrast image, we typically used dark-field illumination for the microdroplet
                     supplied by an LED ring or a fiber-optic gooseneck lamp set at an oblique angle. For each video frame, a background subtraction
                     and threshold was used to eliminate debris or uneven illumination from each image. The centerline of the worm was then calculated
                     using a skeletonize algorithm. The absolute distance between the end-to-end of the centerline is then calculated for each
                     video frame. Typically, we use a frame rate of 1-5 Hz. High frame rates are suitable if one wants to calculate the rate of
                     bending waves, which will emerge as the high frequency component in the oscillation of end-to-end distance. Reorientation
                     events are detected in each experiment by flagging significant dips in the nose-to-tail distance over the time course of each
                     experiment.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_2_4"></a>6.2.4.&nbsp;Experimental data
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The cryophilic response of the worm thermotaxis is straightforward to measure using the microdroplet assay. In earlier work,
                     we found that when worms navigate at temperatures warmer than their cultivation temperature (<span class="emphasis"><em>T</em></span><sub>cult</sub>), increasing temperature stimulates reorientations and decreasing temperature suppresses reorientation (1). The cryophilic
                     response dominates worm movements when worms swimming in a microdroplet are subjected to thermal oscillations.
                  </p>
                  <p>In order to set the memory of <span class="emphasis"><em>T</em></span><sub>cult</sub>, we grow worms overnight on NGM plates containing an ample lawn of OP50. A single worm is picked from the plate, rinsed in
                     an NGM buffer (containing the same salt concentrations as the plate), and then picked to the microdroplet. Thermal cycles
                     around ambient temperatures warmer than <span class="emphasis"><em>T</em></span><sub>cult</sub> (e.g., cycling temperatures between 23-24&deg;C for <span class="emphasis"><em>T</em></span>-<sub>cult</sub>-=20&deg;C) will stimulate reorientations during the warming phase and suppress reorientations during the cooling phase. By cycling
                     the temperature several times for each worm, it is possible to perform a robust measurement of the stimulus-response relationship
                     for an individual worm. By measuring the ambient temperatures at which the cryophilic drive is active, it is also possible
                     to use the microdroplet in a 'memory assay' to quantify the set-point of <span class="emphasis"><em>T</em></span>-<sub>cult</sub> in a single animal.
                  </p>
                  <p>The main limitation of the microdroplet assay for thermotaxis is that a correlate of isothermal tracking has not yet been
                     detected. During isothermal tracking, an individual worm persistently zigzags along each isotherm in a spatial thermal gradient.
                     One possibility is that the uniform temperature of the microdroplet could not support a correlate of isothermal tracking that
                     might require a spatial thermal gradient. Another possibility is that the behavioral response is subtle and might also require
                     a stimulus pattern that more closely matches the thermal experience of a worm tracking an isotherm (e.g., alternating upward
                     and downward steps of <span class="symbol">&#8764;</span>0.1&deg;C every 10-20 seconds).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_2_5"></a>6.2.5.&nbsp;Summary
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The automated analysis of behavioral responses of single swimming worms has accelerated studies of thermotaxis (2). The machine-vision
                     analysis of movements of single worms in microdroplets is a general technique, which can easily be implemented in other studies,
                     e.g., measurements of spontaneous reorientation, bending wave frequency.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6_3"></a>6.3.&nbsp;Linear thermal gradient assay for thermotaxis assay
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Will Ryu, Chris Gabel, Aravi Samuel,  Harvard University, Cambridge, MA, USA.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_3_1"></a>6.3.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Conventional methods for studying worm thermotactic migration utilize frozen glacial acetic acid as a cold source and an incubator
                     as a hot source to create spatial thermal gradients on agar plates. (See preceding protocol, <a href="#bib59">Mori and Ohshima, 1995</a>). We have developed an alternative approach utilizing a simple linear thermal gradient apparatus that utilizes Peltier-effect
                     thermoelectric elements to produce highly reproducible and stable gradients on agar plates with simultaneous monitoring of
                     worm movements using video microscopy.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_3_2"></a>6.3.2.&nbsp;Linear thermal stage
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>We establish a thermal gradient across an anodized aluminum slab (4"x9"x1/4") by affixing 160W thermoelectric elements on
                     each end. Each thermoelectric element is sandwiched between the aluminum slab and its own brass heat sink. Water is continuously
                     circulated through both heat sinks by a temperature-controlled water bath (Model 1160S Heated/Refrigerated Circulator, VWR).
                     The thermoelectric elements create a thermal gradient across the middle of the slab, atop which can be placed a 9 cm agar
                     plate. A few drops of glycerol between the plate and the top surface of the slab ensure thermal contact. The agar plate is
                     covered to isolate the surface from air currents. To prevent condensation build-up, the cover of the agar plate is coated
                     with a thin layer of liquid detergent (<a href="#figure8">Figure 8</a>).
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure8"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="315">
<tr>
  <td align="center"><p><a href="WMBehavFig8.jpg"><img src="WMBehavFig8_s.jpg" border="2" align="middle" alt="WormBook Behavior figure 8"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 8.&nbsp;</b></p>
                     </div>
                  </div>
                  <p>Separate feedback loops are needed for both thermoelectric elements. We use separate calibrated thermistors embedded in the
                     agar slab at the &#8220;hot&#8221; and &#8220;cold&#8221; edges of the gradient. Each thermistor and Peltier-effect thermoelectric element is connected
                     to its own thermoelectric controller. The amount of power needed for steep gradients (1&deg;C/cm) exceeds the operating range
                     of two Thorlabs TEC2000s (Thorlabs, Newton, NJ). One option is to use two Oven Industries Model 5C7-195 thermoelectric controllers
                     (Oven Industries, Mechanicsburg, PA) with external power supplies with the proper power rating (<span class="symbol">&gt;</span>160W). Another option is to use Wavelength electronics MPT10000 Chassis Mounted Thermoelectric modules (Wavelength electronics,
                     Bozeman, MT), although these modules are not 'stand-alone' devices and require assembly into an electronic instrument (e.g.,
                     RIS-549 made by the Electronic Engineering Laboratory at the Rowland Institute at Harvard).
                  </p>
                  <p>The temperature gradient at the agar surface will be less steep than the gradient across the aluminum slab, and needs to be
                     verified with each setting of the thermoelectric elements. This can easily be done using a hand-held digital thermometer (e.g.,
                     Fluke 50 Series) with a T-type thermocouple.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_3_3"></a>6.3.3.&nbsp;Video microscopy
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Uniform illumination and high-contrast images are necessary to visualize worms navigating the linear thermal gradients. We
                     have engineered a simple dark-field illuminator using a ring of 48  superbright light-emitting diodes in the same plane as
                     the agar plate. Using fresh, clean plates without surface defects, the only light-scattering objects are the worms themselves.
                     These high-contrast images of the worms can be recorded using a CCD camera (e.g., Panasonic WV-BP554) equipped with a zoom
                     lens (25 mm) that captures the image of the whole plate.
                  </p>
                  <p>We use a PCI-1407 Image Acquisition card to grab frames from the CCD camera of worms executing thermotactic migrations along
                     the surface of the agar plate. In this way, we can measure overall thermotactic drift upwards or downwards on the thermal
                     gradient as well as score individual movements like forward crawling, backward crawling, and turns. We have written simple
                     particle-tracking algorithms using LabVIEW Vision Assistant to measure individual crawling trajectories.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec6_3_4"></a>6.3.4.&nbsp;Experimental data
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Measurement of thermotactic migration requires setting the worms to a particular cultivation temperature (<span class="emphasis"><em>T</em></span><sub>cult</sub>). When placed on a spatial thermal gradient, worms will track isotherms in a band of temperatures near <span class="emphasis"><em>T</em></span><sub>cult</sub>. Worms can be set to a specific <span class="emphasis"><em>T</em></span><sub>cult</sub> by growing them overnight at <span class="emphasis"><em>T</em></span><sub>cult</sub> on an NGM plate with OP50. The day of the experiment, several young adult worms should be rinsed in NGM buffer and picked
                     to a fresh NGM plate, which should then be transferred immediately to the linear thermal gradient. It is necessary to cover
                     the plate. Usually, the thermal gradient will equilibrate after 3-4 minutes, and isothermal tracks should emerge among the
                     worms navigating at temperatures near their <span class="emphasis"><em>T</em></span><sub>cult</sub> (<a href="#figure9">Figure 9</a>).
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure9"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="WMBehavFig9.jpg"><img src="WMBehavFig9_s.jpg" border="2" align="middle" alt="WormBook Behavior figure 9"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 9.&nbsp;</b></p>
                     </div>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec6_4"></a>6.4.&nbsp;Point source heat avoidance
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. elegans</em></span> avoidance of very high temperatures can be induced by heating a scalpel or pick-then holding the heat source near the animals
                  nose. Animals rapidly initiate backward locomotion. A soldering iron can also be used. This response is called &#8220;Thermal avoidance&#8221;
                  (Tav). A diode laser has been used by the Baumeister laboratory to study thermal avoidance (<a href="#bib88">Wittenburg and Baumeister, 1999</a>).
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec7"></a>7.&nbsp;Locomotion
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_1"></a>7.1.&nbsp;Quantitation of locomotion rate by counting body bends
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Niels Ringstad and Bob Horvitz, written by Michael Koelle, and inspired by Beth Sawin, MIT, Cambridge, MA,
                  USA, March, 2005.
               </p>
               <p>To make results comparable, it is important that the assays always be done exactly the same way!</p>
               <div class="procedure">
                  <ol type="1">
                     <li>
                        <p>Pick 10 late L4 animals from an unstarved plate to fresh plates, one worm to a plate. By picking fat L4s with large white
                           vulval crescents with a tiny black dot in them (as viewed in the dissecting scope) you can get very accurately staged animals.
                           For sick genotypes, may need to pick more than 10 animals to have 10 healthy ones surviving the next day. It's best to use
                           freshly seeded plates with fairly thin lawns. I used plates that were seeded the previous afternoon, and left to grow overnight
                           at 37&deg; in a covered box.
                        </p>
                     </li>
                     <li>
                        <p>24 hours later (&plusmn;1 hour) begin the assays.</p>
                     </li>
                     <li>
                        <p>I've been doing the assays at room temperature (22&deg;) but doing them in the 20&deg; constant temperature room would probably be
                           more careful.
                        </p>
                     </li>
                     <li>
                        <p>Gently place a plate on the dissecting scope (so as not to mechanically stimulate the worm). Start a 3 minute timer, and begin
                           counting body bends. Every time the part of the worm just behind the pharynx reaches a maximum bend in the opposite direction
                           from the bend last counted, advance the count one. For example, if the worm moves forward, then spontaneously reverses, and
                           the region just behind the pharynx bends again the same direction it had just bent when the worm had moved forward, that doesn't
                           count! Wait until the worm actually bends the other way to advance the count.
                        </p>
                        <p>Worm movement changes if the animal reaches the edge of the bacterial lawn. I do not use measurements if the animal dawdles
                           at the edge of the lawn for more than about 15 seconds during the three minute period. If an animal is at the edge of the
                           lawn, put the plate aside for a few minutes and come back to it for measurement when the worm is away from the lawn edge.
                        </p>
                     </li>
                     <li>
                        <p>Count the other 9 worms the same way.</p>
                     </li>
                     <li>
                        <p>Calculate average body bends per minute for each worm by adding up the body bends counted for that worm over the entire 9
                           minutes of measurement, and dividing by 9. Calculate an average and standard deviation for the genotype being measured using
                           the 10 values obtained for the 10 individual worms measured. To give an idea of the size of the standard deviations I get
                           using this method, here are some values I measured (per 3 minutes):
                        </p>
                        <p>N2:  42&plusmn;12 <span class="emphasis"><em><a href="http://www.wormbase.org/db/misc/etree?name=nIs51;class=Transgene" target="_blank">nIs51</a></em></span>:	70&plusmn;16 <span class="emphasis"><em>egl-10 (<a href="http://www.wormbase.org/db/get?name=n692;class=Variation" target="_blank">n692</a>)</em></span>	10&plusmn;4.
                        </p>
                     </li>
                     <li>
                        <p>I prefer this measure of locomotion to others that have been used. The actual distance a worm moves per time may be influenced
                           by the number of eggs it carries, the thickness of the bacterial lawn it is on, number of direction reversals the worm makes,
                           etc. Counting the number of body bends seems to me to be a more direct measure of the effort the worm is making to move which
                           should be less influenced by these factors.
                        </p>
                        <p>I always carry out all assays to be compared on the same day, using the same batch of plates, so as to minimize any systematic
                           errors that might be introduced by using different batches of plates.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_2"></a>7.2.&nbsp;Decreased locomotion on food
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Niels Ringstad and Bob Horvitz, adapted from Sawin, Ranganathan and Horvitz (<a href="#bib72">Sawin et al., 2000</a>) by Dan Omura, MIT Cambridge, MA, USA, March, 2005.
               </p>
               <p>Assay plates are prepared by spreading the <span class="emphasis"><em>E. coli</em></span> strain HB101 in a ring with an inner diameter of <span class="symbol">&#8764;</span>1 cm and an outer diameter of <span class="symbol">&#8764;</span>3.5 cm on NGM agar in 5 cm Petri plates. Lawns can be spread using the smooth bottom of a small culture tube. Assay plates
                  are always freshly spread with bacteria, incubated overnight at 37&deg;C, and allowed to cool to room temperature before use.
                  Lawns should not be overgrown and should have a slightly textured, not glossy, appearance. Plates for measuring locomotory
                  rate in the absence of bacteria are also incubated at 37&deg;C. Only synchronized young adult hermaphrodites (12-21 hr after the
                  late L4 larval stage) are tested.
               </p>
               <p>For well-fed animals, locomotory rate is measured by gently picking 5 animals from plates with ample bacteria, washing the
                  animals twice in 1 ml S basal buffer in a 24 well flat bottom tissue culture plate using a capillary pipette, and transferring
                  them to an assay plate in a drop of buffer. If the assay plate contains a ring-shaped bacterial lawn, the animals are transferred
                  to the clear zone in the center of the ring. The buffer used to transfer the animals is carefully absorbed with a Kimwipe.
                  Five minutes after transfer, the number of body bends in 20 s intervals is measured for each of the 5 animals on the assay
                  plate.
               </p>
               <p>To food-deprived animals, 5-15 animals are washed free of bacteria in S basal buffer (two washes) and then transferred to
                  5 cm NGM agar plates with no bacteria and a ring of 4M fructose on the outer edge. The fructose ring prevents animals from
                  crawling off of the plate and is easily made by holding the plate at an angle, dripping 2-3 drops of fructose at the edge
                  of the plate, and rotating the plate. To make the high osmolarity ring visible, add a small amount of bromophenol blue to
                  the fructose solution. The animals are gently dried with a Kimwipe and incubated on these plates for 30 min at room temperature.
                  At the end of 30 min of food deprivation, five worms are gently transferred in a drop of S basal buffer to assay plates, and
                  locomotory rate is measured as described above for well-fed animals.
               </p>
               <p>Locomotory rate measurements for mock and laser-ablated animals are done in a similar fashion, except that only 1 animal is
                  transferred to each assay plate. Its locomotory rate is tested in the presence of bacteria, and it is then allowed to recover
                  for 2-3 hr on plates containing bacteria and then transferred, as described above, to food deprivation plates. After 5 min
                  on the food deprivation plates, the locomotory rate of these well-fed animals in the absence of food is recorded; these locomotory
                  rates are not different from locomotory rates on regular assay plates without bacteria that do not have the high osmolarity
                  ring (data not shown). After 30 min, the food-deprived animals are transferred to assay plates containing ring-shaped bacterial
                  lawns, and the locomotory rate is again recorded.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_3"></a>7.3.&nbsp;Bordering at edges of bacterial lawns
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. elegans</em></span> normally disperses across a bacterial lawn, but mutations in specific genes or specific environmental conditions can induce
                  animals to aggregate or form &#8220;clumps&#8221; just inside the edge of the bacterial lawn. This behavior is historically called &#8220;bordering&#8221;
                  or &#8220;social behavior&#8221;. Decreased function of the <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=npr-1" target="_blank">npr-1</a></em></span> neuropeptide receptor or associated ligands induces bordering. (<a href="#bib27">de Bono and Bargmann, 1998</a>)  Bordering can be induced by decreasing oxygen levels. Bordering can be quantified on NGM plates containing 2.1% agar seeded
                  2 days before the assay with 200 l of <span class="emphasis"><em>E. coli</em></span> OP50 in LB medium to generate a circular lawn about 25 mm in diameter. Approximately 120 well-fed adult worms from uncrowded
                  plates are picked onto this lawn and left at 20&deg;C for 3 hr. (Do not disturb the plates during the incubation period; avoid
                  mechanical perturbations like vibration or tap.) After 3 hr, clumping behavior can be measured by calculating the fraction
                  of animals in contact with two or more other animals along at least 50% of their body length. At the same time,bordering behavior
                  can be measured by calculating the fraction of animals that resided within 2 mm of the edge of the bacterial lawn.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_4"></a>7.4.&nbsp;Spontaneous reversals
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Joseph Dent, McGill University, Montreal, Quebec Canada, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec7_4_1"></a>7.4.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Reversal behavior refers to the tendency of the worms to change their mode of locomotion from forward to backward by changing
                     the direction of propagation of the sinusoidal body waves. Reversals can be induced, as when an acute stimulus such as touch
                     produces an immediate reversal, or spontaneous as when the worm reverses in the absence of an acute stimulus. Spontaneous
                     reversals reflect the integration of many aspects of both the environment and the internal state of the worm and therefore
                     can be a sensitive measure of behavioral state.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec7_4_2"></a>7.4.2.&nbsp;Reversal frequency
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>There are two methods for scoring reversals: human observation and computer vision. Human observation involves watching a
                     worm, typically under a dissecting microscope and counting the number of times they reverse. (<a href="#bib18">Chiba and Rankin, 1990</a>; <a href="#bib20">Croll, 1975</a>) The scoring can be aided by making a video record of the assay. Reversals run the gamut from a stutter in the forward locomotion
                     that lasts less than a second to several seconds of backward movement, so one must operationally define what constitutes a
                     reversal. The specific definition of a reversal is less important then its consistent application. One convenient definition
                     for human observation is any retrograde movement that is greater than the length of the pharynx. This has the virtue of being
                     easy to gauge and, because the pharynx is proportional to the size of the worm, is less dependent on the size of the worm.
                  </p>
                  <p>Computer vision applications typically involve a camera mounted on a microscope, connected to a frame-grabber and image analysis
                     software, which are in turn connected to a motorized stage. (<a href="#bib61">Pierce-Shimomura et al., 1999</a>) The image analysis software identifies the silhouette of the worm, determines how far the worm has moved since the last
                     image was taken and commands the stage to re-center the worm in the field of view. The record of the movements of the stage
                     is also a record of the movements of the worm. Updating the stage every second is generally a high enough sample rate to catch
                     most reversals although some very brief reversals may be missed. (<a href="#bib61">Pierce-Shimomura et al., 1999</a>; <a href="#bib89">Zhao et al., 2003</a>) Alternatively, the worms can be recorded under low magnification with no movement of the stage and their position in the
                     field of view recorded at each time point. To reduce computational demands, the position of the worm is usually abstracted
                     to its centroid. Reversals are defined as changes in the bearing of the centroid (determined by the angle between the line
                     segments defined by successive pairs of data points) greater than a certain angle, e.g., <span class="symbol">&gt;</span>50&deg; (<a href="#bib61">Pierce-Shimomura et al., 1999</a>). However, the centroid can be misleading, especially when the worm is significantly bent as in the case of an omega turn,
                     resulting in false positives and negatives. It is important to visually confirm the result of at least a sample of the reversals
                     scored by the computer. More recently, software has been developed that abstracts the worm as a series of line segments along
                     the midline (<a href="#bib3">Baek et al., 2002</a>; <a href="#bib33">Feng et al., 2004</a>). In principle, this should result in more accurate scoring of reversals. Various combinations of hardware have been employed
                     for worm tracking (<a href="#bib37">Hardaker et al., 2001</a>; <a href="#bib61">Pierce-Shimomura et al., 1999</a>; <a href="#bib89">Zhao et al., 2003</a>). Software is often written by the authors and available on request (<a href="#bib33">Feng et al., 2004</a>).
                  </p>
                  <p>Although a researcher can score one worm out a population on a plate, it is often advantageous to place worms individually
                     on an assay plate and score them one at a time. This is because, whether scoring by eye or by computer, it is essential to
                     control the state of the worm and its environment as much as possible. Among the variables that can affect reversal frequency
                     are: temperature, humidity, wetness of the agar, presence of food, age of the worm, sex of the worm and gravidity of the worm
                     (<a href="#bib89">Zhao et al., 2003</a>). A worm will also reverse if it bumps into another animal. Moreover, computer vision applications used so far only track
                     one worm at a time. By scoring L4 larvae one can minimize the effects of age and gravidity. To control for other effects,
                     it is useful to assay control and experimental populations on the same day using the same agar plate, preferably by alternating
                     experimental/mutant and control worms (<a href="#bib89">Zhao et al., 2003</a>; <a href="#bib90">Zheng et al., 1999</a>).
                  </p>
                  <p>Although one can measure reversal frequency simply as the number of reversals in a given period of time, it can be more informative
                     to score the reversal interval, i.e. the time between reversals. By plotting a histogram of reversal intervals, one finds
                     that the resulting distribution can best be fit by a lognormal function (<a href="#bib37">Hardaker et al., 2001</a>; <a href="#bib89">Zhao et al., 2003</a>), or the sum of two exponentials (<a href="#bib12">Brockie et al., 2001</a>; <a href="#bib61">Pierce-Shimomura et al., 1999</a>). The parameters of these distributions can vary significantly with the experimental conditions, providing information about
                     how the probability of reversal varies over time or among populations in different behavioral states.
                  </p>
                  <p>Reversal frequency can be measured in response to various experimental manipulations. These include: the presence of a source
                     of chemo-attractive compounds (<a href="#bib61">Pierce-Shimomura et al., 1999</a>), the presence of drugs (in the agar) and mechanical stimulation (<a href="#bib89">Zhao et al., 2003</a>). Mechanical stimulation with an eyelash or pick requires that the lid be left off the plate; in this case care must be taken
                     to avoid air currents that can stimulate the worms to reverse.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec7_4_3"></a>7.4.3.&nbsp;Reversal duration/distance
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Reversal duration/distance, how long the worm spends in retrograde locomotion for a given reversal and how far it moves backward,
                     are also sensitive indicators of the state of the worm. A video record of the animal's behavior traced onto acetate or a computer
                     vision application can be used to measure the duration of a reversal and its distance (<a href="#bib18">Chiba and Rankin, 1990</a>). Reversal duration can also be scored visually with a timer and the reversal distance estimated by counting the number of
                     body bends generated while the worm is reversing. Body bends are generally an accurate measure of distance for worms of approximately
                     equal size. As above, control of the environment is essential to get consistent results.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_5"></a>7.5.&nbsp;Ivermectin sensitivity assays
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Joseph Dent, McGill University, Montreal, Quebec, Canada, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec7_5_1"></a>7.5.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Ivermectin is an antiparasitic drug that affects nematodes including <span class="emphasis"><em>C. elegans</em></span>. It activates glutamate-gated chloride channels which in turn inhibit a variety of excitable cells including the pharyngeal
                     muscle and neurons involved in locomotion (<a href="#bib22">Cully et al., 1994</a>; <a href="#bib28">Dent et al., 2000</a>). Because ivermectin prevents pharyngeal pumping, one effect of ivermectin is to cause the animals to starve. Ivermectin-induced
                     paralysis of locomotion appears to be an effect of ivermectin on the neuronal circuit controlling locomotion rather than on
                     the muscles themselves. Both the locomotion and eating phenotypes have been used to measure sensitivity to ivermectin.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec7_5_2"></a>7.5.2.&nbsp;Growth assay for ivermectin sensitivity
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Prepare stock a solution of the ivermectin (e.g., 10 mg/ml, Sigma-Aldrich, St. Louis, MO.) in 100% DMSO. Prepare drug plates
                     at desired concentrations (EC<sub>50</sub> for wild type worms is <span class="symbol">&#8764;</span>1 ng/ml) by adding 1% DMSO and then ivermectin from a stock solution to melted NGM agar. For dose response curves, which require
                     preparing plates at many drug concentrations, we prefer to pour 35 mm plastic Petri dishes. Seed plates with bacteria a day
                     or two in advance. If only testing a few drug concentrations, pick eggs from the strain of interest directly onto the drug
                     plate. If many drug concentrations are to be tested, egg prep the strain of interest using alkaline bleach and resuspend eggs
                     from one 60 mm plate in <span class="symbol">&#8764;</span>1 ml sterile M9 buffer. Estimate the concentration of eggs by placing <span class="symbol">&#8764;</span>30 ml of suspension from a well shaken tube (always re-shake the suspension because eggs will settle), put on an agar plate
                     and count the number of eggs. Try to have between 50 and 100 eggs per 30 ml; dilute suspension if necessary. Pipette 30 ml
                     of egg suspension per drug plate. Be sure to include control plates lacking drugs (ideally with 1 % DMSO although in practice
                     the DMSO seems not to have much effect). Allow worms to grow for 3-4 days and score growth. A simple method of scoring growth
                     is to count the number of hermaphrodites that have become gravid adults, although there is any number of criteria one could
                     use to measure growth. It is essential however to include negative controls (no drug) against which to normalize the data.
                     We typically score 6 plates per ivermectin concentration using at least two different egg preps. The growth assay is most
                     sensitive to pharyngeal function and primarily reflects the effect of ivermectin on feeding (<a href="#bib28">Dent et al., 2000</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec7_5_3"></a>7.5.3.&nbsp;Motility assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>To measure effects of ivermectin on locomotion, a motility assay can be used. (<a href="#bib1">Arena et al., 1995</a>) Wash worms off of plates with 25 mM HEPES buffer and incubate in liquid with drug up to 16 hours. Score the percentage of
                     worms that are still thrashing in liquid.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_6"></a>7.6.&nbsp;Pirouettes
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Shawn Lockery, University of Oregon, Eugene, OR, USA, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec7_6_1"></a>7.6.1.&nbsp;Background
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>C. elegans</em></span> locomotion involves bouts of forward locomotion punctuated about twice a minute by episodes of turning that have been termed
                     &#8220;pirouettes&#8221; (<a href="#bib61">Pierce-Shimomura et al., 1999</a>). Pirouettes appear to be the final common path for approach behaviors such as chemotaxis, and avoidance behaviors such as
                     the response to high osmolarity. Because the efficiency of either type of behavior depends on the change in orientation produced
                     by individual pirouettes, it is often of considerable importance to quantify the effects of pirouettes on the direction of
                     locomotion.
                  </p>
                  <p>Pirouettes are defined in terms of omega turns and reversal turns, here collectively referred to as &#8220;sharp turns&#8221;, and the
                     intervals between them called &#8220;runs&#8221;. The study that originally proposed the concept of a pirouette (<a href="#bib61">Pierce-Shimomura et al., 1999</a>), noted that the distribution of run durations was well fit by the sum of two exponentials, suggesting that there are two
                     distinct kinetic states, one that emits mainly short runs and another that emits mainly long runs. Pirouettes were defined
                     as either a single sharp turn or a series of sharp turns each separated from its neighbors by short runs. Because either type
                     of run state is theoretically capable of generating a short or long run, it is not possible to definitively assign a run of
                     a particular duration to one state or the other. However, one can minimize the number of classification errors by computing
                     a critical run duration, called <span class="emphasis"><em>t</em></span><sub>crit</sub>, below which a run is assumed to be generated by the short run state. The equation for <span class="emphasis"><em>t</em></span><sub>crit</sub> is shown in <a href="#figure10">Figure 10</a> where <span class="emphasis"><em>A</em></span><sub>l</sub> and <span class="emphasis"><em>A</em></span> <sub>s</sub> are the respective amplitudes of the long-and short-run exponential components and <sub>l</sub> and <sub>s</sub> are their respective time constants. In the original study, <span class="emphasis"><em>t</em></span><sub>crit</sub> was found to be approximately 6 sec, but <span class="emphasis"><em>t</em></span><sub>crit</sub> is likely to be different in different types of experiments. A first step therefore in any study of turning behavior will
                     be to obtain the distribution of run durations under appropriate experimental conditions, and to compute the value to <span class="emphasis"><em>t</em></span><sub>crit</sub> to be used in the analysis of data generated under these conditions.
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure10"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="135">
<tr>
  <td align="center"><p><a href="WMBehavFig10.jpg"><img src="WMBehavFig10_s.jpg" border="" align="middle" alt="WormBook Behavior figure 10"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 10.&nbsp;</b></p>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec7_6_2"></a>7.6.2.&nbsp;Automated analysis of pirouettes
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The following summarizes the steps required to identify pirouettes and to measure their effects on orientation.</p>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p>Record the location of the worm's centroid at a rate of at least 1 sample per second. This requires an automated tracking
                              system with enough spatial resolution to clearly define the direction of locomotion at this sampling rate.
                           </p>
                        </li>
                        <li>
                           <p>Convert this record to instantaneous turning rate vs. time, <span class="emphasis"><em>R</em></span> (<span class="emphasis"><em>t</em></span>). This is done by computing the (minimum)  change in angle between the velocity vector defined by sample points at <span class="emphasis"><em>t</em></span> -1 and <span class="emphasis"><em>t</em></span>, and the velocity vector defined by sample points <span class="emphasis"><em>t</em></span> and <span class="emphasis"><em>t</em></span> + 1. The change in orientation is converted to a rate by dividing by the sample interval.
                           </p>
                        </li>
                        <li>
                           <p>Identify sharp turns in the recording by noting the times at which |<span class="emphasis"><em>R</em></span> (<span class="emphasis"><em>t</em></span>)| exceeds a threshold of 50 deg/sec. It is advisable to verify and possibly revise the threshold in accordance with different
                              strains and plate conditions. This is done by comparing sharp turns identified in videotapes by eye the <span class="emphasis"><em>R</em></span> (<span class="emphasis"><em>t</em></span>) trace of the same worm.
                           </p>
                        </li>
                        <li>
                           <p>Identify pirouettes. A pirouette is as an isolated sharp turn, or a sharp turn burst. A burst begins when the interval between
                              two consecutive sharp turns is <span class="symbol">&gt;</span> <span class="emphasis"><em>t</em></span><sub>crit</sub>. A burst ends when the interval to the next sharp turn is <span class="symbol">&#8805;</span> <span class="emphasis"><em>t</em></span><sub>crit</sub>. As discussed above, <span class="emphasis"><em>t</em></span><sub>crit</sub> depends on the distribution of run durations. Because this distribution is likely to be different in different experiments,
                              it is advisable to compute it anew for each data set.
                           </p>
                        </li>
                        <li>
                           <p>Compute than angle change associated with each pirouette. This computation depends on the reference frame appropriate to the
                              experiment. For chemotaxis in a radial gradient, for example, the reference frame is the gradient. The change in angle produced
                              by a pirouette is the change in the worm's bearing relative to the gradient peak. Bearing is defined as the angle between
                              the velocity vector of the worm and the vector defined by connecting the worm's location to gradient peak; the change in bearing
                              is defined as the difference between the bearing vectors immediately before and after the pirouette. For other studies, such
                              as those concerning spontaneous locomotion, the reference frame is likely to be the worm itself. In this case one merely takes
                              the difference between velocity vectors immediately before and after the pirouette.
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_7"></a>7.7.&nbsp;Dauer lethargy and molting lethargy
                        </h3>
                     </div>
                  </div>
               </div>
               <p><span class="emphasis"><em>C. elegans</em></span> spontaneous locomotion decreases during the dauer stage and decreases at specific intervals at each larval molt. Relatively
                  little is known about the mechanisms involved.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_8"></a>7.8.&nbsp;Duration and velocity of spontaneous forward or backward locomotion
                        </h3>
                     </div>
                  </div>
               </div>
               <p>This description is adapted from the literature. (<a href="#bib12">Brockie et al., 2001</a>; <a href="#bib90">Zheng et al., 1999</a>) Animals are grown at 20&deg;C-21&deg;C on <span class="emphasis"><em>E. coli</em></span> strain OP50 under standard conditions. Individual first day adults are transferred to assay plates using a glass pick from
                  culture plates to standard NGM agar plates lacking food. Care should be taken to ensure that no bacteria are transferred with
                  the animals. Each animal is observed for about 7 min, and a computer program keyed by the operator can be used to record the
                  duration of forward and backward movements. Average duration times can vary between experiments, presumably due to variables
                  like humidity, dryness of plates, <span class="emphasis"><em>etc</em></span>. All of the strains that needed to be scored should randomized and scored on the same day. And, all movement assays should
                  be scored by an observer who is blind as to genotype or treatment. Forward and backward durations can be reported as mean
                  &plusmn; SEM, and the significance of the differences can be evaluated using the Student's t test.
               </p>
               <p>Velocity can be determined from calibrated, videotaped recordings of movements on unseeded NGM agar plates. Average velocity
                  is defined as the total path distance (mm) per time (s).
               </p>
               <p>Reversal frequency is obtained by counting the total number of movement reversals per minute; it can be calculated from the
                  forward and backward duration: reversal frequency per MIN = 1/([forward time + backward time]/2).
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_9"></a>7.9.&nbsp;Suppression of head oscillations during backwards locomotion
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Niels Ringstad and Bob Horvitz, written by Mark Alkema, MIT, Cambridge, MA, USA, March, 2005.</p>
               <p>Attach a very fine eyelash or hair of a fine sable hair paint brush  (watercolor) to the very end of Pasteur pipette with
                  Scotch tape.
               </p>
               <p>Stroke young adults with the hair  behind the posterior bulb of the pharynx. The anterior touch induces a backing response.
                  Score the presence or absence of head oscillations during the backing response. Score animals that make at least 2 backward
                  body bends (one sinusoidal wave) to make sure that the animal responds to the anterior touch. Wild type animals suppress their
                  head oscillations during the backing response. In the wild type, head oscillations resume as soon as forward locomotion is
                  reinitiated. <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=tdc-1" target="_blank">tdc-1</a></em></span> mutants fail to suppress head oscillations during this backing response and can be used as negative control in this assay.
                  The suppression of head oscillations is most easily scored in animals that are outside the bacterial lawn or on a fresh thin
                  bacterial lawn.
               </p>
               <p>Animals that show no head oscillations during backward locomotion were scored as wild-type. A single animal can be tested
                  at least 3 consecutive times with no obvious extinction in the suppression of head oscillations.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec7_10"></a>7.10.&nbsp;Immobilization in serotonin (liquid)
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Niels Ringstad and Bob Horvitz, written by Megan Higginbotham, MIT, Cambridge, USA, MA, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec7_10_1"></a>7.10.1.&nbsp;Exogenous serotonin
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p>
                                                      Dissolve serotonin (serotonin creatinine sulfate complex, Sigma) in M9 to the desired concentration
                              by vortexing.
                           </p>
                        </li>
                        <li>
                           <p>
                                                      Aliquot 200 microliters per well (96 flat bottom well tissue culture plates, Falcon).
                           </p>
                        </li>
                        <li>
                           <p>
                                                      Pick 20 animals (staged, approx. 24 hrs post-L4) to well.
                           </p>
                        </li>
                        <li>
                           <p>
                                                      Count immobilized animals after each minute for desired number of minutes.
                           </p>
                           <div class="sidebar">
                              <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                                 <h3 class="title">Notes</h3>
                                 <p>Other people heat the solution to 70&deg;C to get the serotonin to dissolve.</p>
                                 <p>I use the serotonin solution immediately after making it, it's not clear how long it stays good.</p>
                                 <p>I use animals that are approx. 24 hours post L4-I haven't rigorously tested the effect of age on response to 5-HT, but in
                                    my experience older animals seem to be less sensitive.
                                 </p>
                                 <p>Carrying over bacteria to the assay well does not seem to have an effect on immobilization rate.</p>
                              </div>
                           </div>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec8"></a>8.&nbsp;Feeding
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec8_1"></a>8.1.&nbsp;Electropharyngeogram recordings
                        </h3>
                     </div>
                  </div>
               </div>
               <p>A detailed description of this can be found at <a href="http://eatworms.swmed.edu/Worm_labs/Avery/MiniRig/" target="_blank">http://eatworms.swmed.edu/Worm_labs/Avery/MiniRig/</a>.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec8_2"></a>8.2.&nbsp;Pharyngeal pumping rate
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Pumping rate are measured by counting visible movements of the grinder. (<a href="#bib63">Raizen et al., 1995</a>).
               </p>
               <p>Normal rates are almost too fast to count, but decreased pumping rates can be readily counted. Note that in normal animals
                  pharyngeal pumping is decreased off food.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec8_3"></a>8.3.&nbsp;Foraging
                        </h3>
                     </div>
                  </div>
               </div>
               <p>See &#8220;<a href="#sec2_5" title="2.5.&nbsp;Head withdrawal and foraging">Head withdrawal and foraging</a>&#8221; in the Mechanosensation section.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec9"></a>9.&nbsp;Egg-laying, males and mating
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9_1"></a>9.1.&nbsp;Assessing defective or constitutive egg-laying
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Michael Koelle, Yale University, New Haven, CT, USA, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_1_1"></a>9.1.1.&nbsp;Counting eggs inside a worm
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The number of eggs that accumulate inside an adult is a function both of the rate of egg production and the rate of egg-laying.
                     If you want to use this assay to quantify differences in the rates of egg-laying between various strains, you have to first
                     be certain that the strains you're assaying have a similar rates of egg production. In practice, I do this by measuring brood
                     sizes on all the strains being analyzed to make sure they are similar.
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>The number of eggs inside an adult varies with age, so staged adults must be used to get meaningful numbers. Stage the animals
                              by picking late L4 larvae to a new plate. They are recognized by a white crescent in the presumptive vulval region, which
                              acquires a black central dot in late L4. Also, late L4 are fatter than early L4. Animals thus staged should be within a few
                              hours of each other in development. Put the staged animals back in the incubator, and count after the desired aging has occurred.
                              The number of eggs in most genotypes peaks around 44 hours after late L4 at 20&deg;. I usually assay the animals at 36 hr after
                              late L4 at 20&deg;:  this is just before the time when severely Egl worms begin to have hatched larvae inside of them. These hatched
                              larvae tend to damage the mother's gonad, interfering with the production of eggs. The number of eggs inside a worm containing
                              hatched larvae therefore does not necessarily reflect the rate at which eggs have been laid; it is skewed by this damage to
                              the gonad. I never use egg counts from worms containing hatched larvae.
                           </p>
                        </li>
                        <li>
                           <p>To dissolve the worms, but not the fertilized eggs, use 20% sodium hypochlorite (NaOCl) in water. This is sold by Aldrich,
                              cat. #29,930-5. Store the 100% stock in the refrigerator. I keep 10 mls of 20% ready for use in a capped tube at my bench.
                              Dispense 50<span class="symbol">&#956;</span>l per well into a flat well bottom microtiter dish. I've been using &#8220;Costar&#8221; brand &#8220;cell culture cluster dishes&#8221;, which are
                              very clean. Note:  NaOCl goes bad after a while; the 20% solution should be thrown out after about a month, and even the refrigerated
                              concentrated stuff eventually goes bad.
                           </p>
                        </li>
                        <li>
                           <p>Put one worm in each well. Do this at the dissecting scope using the 12X objective, using a worm pick to transfer animals
                              from plates to the microtiter dishes. It takes 8-10 minutes for the hypochlorite to dissolve most adults without affecting
                              mature eggs. In practice, I pick 20 worms to wells, which takes about 8 minutes, count those worms, and then pick more. Don't
                              wait longer than <span class="symbol">&#8764;</span>20 minutes because eventually hypochlorite will dissolve the eggs too.
                           </p>
                        </li>
                        <li>
                           <p>To count the eggs, move to a higher power scope. This aids in identifying eggs versus dissolved bits of adult worm, and allows
                              eggs to be staged if early versus late eggs are being counted. Two types of scopes can be used. The Makroskop (high power
                              dissecting scope) is nice because you can use a zoom lens, it is in the right magnification range, and there is space above
                              the dish to use a worm pick to mechanically disrupt worms that haven't been lysed by the hypochlorite adequately. Unfortunately,
                              the optics around the edges of the microtiter wells are terrible with the Macroskop, and eggs on the edges of the wells will
                              be invisible. If the Makroskop is used, it is important to place the worms in the center of the wells, and be careful not
                              to bump the dish and move eggs to the edge. The better option is to use a low power inverted scope. A tissue culture style
                              scope is ideal, but worm labs don't have these. A microinjection scope can be used. Just remove the injection needle, and
                              the clips that are normally used to hold the slide on the stage. Then use 5X and 10X objectives, and swing out the polarized
                              light filter. A reasonable image of the whole microtiter well can be had.
                           </p>
                        </li>
                        <li>
                           <p>Ideally the adult will be almost completely dissolved, and a pile of eggs will remain to be counted. This works better in
                              some genotypes than others. For example, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cat-4" target="_blank">cat-4</a></em></span> adults dissolve almost immediately, many Egl mutants dissolve okay, and N2 is difficult to dissolve; often the N2 cuticle
                              remains around the bunch of eggs making it difficult to resolve exactly the number of eggs in the tightly packed cluster.
                              This problem is overcome by just letting these worms soak in hypochlorite for an extra 5 minutes.
                           </p>
                        </li>
                        <li>
                           <p>There is a fair amount of variation between individual worms, so I always count at least 10 worms of a given genotype to get
                              a meaningful average.
                           </p>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9_2"></a>9.2.&nbsp;Quantitation of constitutive egg-laying
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Michael Koelle, Yale University, New Haven, CT, USA, March, 2005.</p>
               <p>To make results comparable, it is important that the assays always be done exactly the same way!</p>
               <div class="procedure">
                  <ol type="1">
                     <li>
                        <p>Pick about 25 late L4 animals from an unstarved plate to a fresh plate.</p>
                     </li>
                     <li>
                        <p>36 hours later (+ or <span class="symbol">&#8722;</span> 0.5 hours), pick 20 animals to a fresh plate. You will necessarily transfer some eggs with the animals. Let the animals crawl
                           away from these eggs for a few minutes, and then pick them again to a fresh plate, this time making sure that no eggs are
                           transferred.
                        </p>
                     </li>
                     <li>
                        <p>Set the animals at 20&deg; for exactly 30 minutes. There should be 25-100 eggs on the plate.</p>
                     </li>
                     <li>
                        <p>Stage the freshly laid eggs using a Wild Makroskop with 20X eyepieces and the zoom objective. To help scan across the plate
                           systematically, it helps to place the plate inside a plate lid with parallel black lines drawn every 2.5 mm. I scan across
                           the plate with the zoom set at <span class="symbol">&#8764;</span>16X, and usually zoom up all the way to stage eggs when I find them.
                        </p>
                     </li>
                     <li>
                        <p>I use a multiple channel mechanical counter to keep track of the following categories:
                                               
                        </p>
                        <div class="literallayout">
                           <p><br>
                              &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1&nbsp;cell&nbsp;embryos<br>
                              &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2&nbsp;cell&nbsp;embryos<br>
                              &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3-4&nbsp;cell&nbsp;embryos<br>
                              &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5-8&nbsp;cell&nbsp;embryos<br>
                              &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;9&nbsp;or&nbsp;more&nbsp;cell&nbsp;embryos
                           </p>
                        </div>
                     </li>
                     <li>
                        <p>N2 lays more than 99% embryos at the 9 cell or later stage.</p>
                     </li>
                     <li>
                        <p>I always repeat the assay on two different days for each strain. I don't understand why, but sometimes the results can vary
                           as much as two-fold! Usually, the standard deviation is more like 20%. The bottom line is that you're only getting a ballpark
                           figure from a single assay.
                        </p>
                     </li>
                     <li>
                        <p>Results from this assay are only meaningful if you show that your strain undergoes embryonic development at about the same
                           rate as does N2. Pick a few early 1 or 2 cell embryos and follow the timing of their first few cell divisions, and the time
                           it takes to undergo morphogenesis and to hatch. Make sure these times are similar to what you measure for N2.
                        </p>
                     </li>
                  </ol>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9_3"></a>9.3.&nbsp;Modulation of egg-laying in response by bacteria
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Niels Ringstad and Bob Horvitz, MIT, Cambridge, MA, USA, March, 2005.</p>
               <p>The day before, spot fresh plates and incubate overnight at 37&deg;C in a covered container. Assay plates without bacterial lawns
                  should be handled the same way and also left overnight at 37&deg;C. Pick late L4 animals 24 hours before the assay and leave at
                  20&deg;C.
               </p>
               <p>The next day, bring worms and assay plates to room temperature 2-3 hours before the assay. Assay plates without bacteria are
                  ringed with 4M fructose to prevent animals from crawling off into the void (see Mod assay protocol for details) - do this
                  at least an hour before the assay to allow the fructose ring some time to get absorbed by the plate. I pick the worms into
                  S basal medium and then transfer single worms to assay plates with a mouth pipette. Worms are gently blotted with a Kimwipe
                  and then left for a period of time, usually 60 minutes. After the assay is over, the animal is picked off of the plate and
                  I count the eggs present on the plate. By noting the time of transfer and continuously transferring animals, one can score
                  30+ animals per hour.
               </p>
               <p>Animals can be assayed in the well-fed state or in a food-deprived state as with the locomotion assays.</p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9_4"></a>9.4.&nbsp;Stimulation of egg-laying with serotonin and other drugs
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by James J. Moresco and Michael R. Koelle, Yale University, New Haven, CT, USA, March, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_4_1"></a>9.4.1.&nbsp;Background
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>This assay can be used to determine if various compounds can stimulate egg-laying in liquid (<a href="#bib85">Weinshenker et al., 1995</a>). M9 buffer inhibits the egg-laying behavior of wild-type animals. Serotonin and other drugs can overcome this inhibition
                     to stimulate egg-laying in M9 buffer. This assay was originally used to determine if egg-laying defective (Egl) animals had
                     neuronal, vulval or muscle defects. (<a href="#bib77">Trent et al., 1983</a>) Egl animals with vulval muscle defects will not lay eggs when exposed to serotonin. If the muscles of an Egl animal could
                     be induced to contract to release eggs when exposed to exogenous serotonin then the Egl animal was thought to have functioning
                     muscles but defective HSN neurons that did not release serotonin.
                  </p>
                  <p>Treating animals with drugs to block serotonin removal from the synapse, such as fluoxetine, can potentiate serotonin signaling
                     to stimulate egg-laying. Mutant animals that do not lay eggs when exposed to fluoxetine are thought to have HSN neurons defective
                     in serotonin release.
                  </p>
                  <p>Imipramine was originally used potentiate serotonin, signaling but fluoxetine has since been shown to be a more selective
                     serotonin reuptake inhibitor (SSRI) and should be used instead of imipramine (<a href="#bib85">Weinshenker et al., 1995</a>).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_4_2"></a>9.4.2.&nbsp;Procedure
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p>Culture animals on standard NGM plates seeded with OP50 bacteria at 20&deg;C.</p>
                        </li>
                        <li>
                           <p>One day prior to the assay, pick 30 late L4 animals from an unstarved plate to a new plate. Under a dissecting scope, late
                              L4 hermaphrodites have a dark spot in the clear crescent formed by the developing vulva. Grow the selected animals at 20&deg;C
                              for 24&plusmn;1 hours, so they are precisely staged young adults.
                           </p>
                        </li>
                        <li>
                           <p>Compounds to be tested are dissolved in M9 buffer. As serotonin is not very stable in solution, prepare the serotonin solution
                              the day of the assay. We use 35 mM serotonin in M9 buffer. Serotonin is available in multiple forms. Serotonin hydrochloride
                              (Sigma product number H 9523) is more soluble then serotonin creatinine sulphate complex (Sigma product number H7752) but
                              also nearly ten times as expensive. To dissolve creatinine sulphate complex in M9 buffer heat to 80&deg;C for 5 minutes, cool
                              to room temperature before using. Fluoxetine (Sigma product number F 132) is assayed at 1 mg/ml.
                           </p>
                        </li>
                        <li>
                           <p>Distribute 50 <span class="symbol">&#956;</span>l of each test solution into 10 wells of a microtiter dish. For negative controls distribute 50 <span class="symbol">&#956;</span>l of M9 to an additional 10 wells.
                           </p>
                        </li>
                        <li>
                           <p>Pick one young adult to each well. Leave the plate at room temperature during the assay.</p>
                        </li>
                        <li>
                           <p>After 60 minutes the number of eggs laid by each animal is counted.</p>
                        </li>
                     </ol>
                  </div>
                  <p>Data can be presented as the mean eggs laid per animal.</p>
                  <p>N2 animals laid and average of 13 eggs in serotonin compared to very few in M9 (<a href="#bib29">Desai and Horvitz, 1989</a>).
                  </p>
                  <p>N2 animals in 1 mg/ml of fluoxetine laid 7.94&plusmn;1.16 eggs (<a href="#bib85">Weinshenker et al., 1995</a>).
                  </p>
                  <p>Alternatively data can be presented by binning animals into four categories based on the number of eggs laid, <span class="symbol">&gt;</span>7 eggs, 4-7 eggs, 1-3 eggs or zero eggs (<a href="#bib77">Trent et al., 1983</a>).
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9_5"></a>9.5.&nbsp;Quantification of egg-laying with chitinase assays
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Tim Geary, Pfizer, Kalamazoo, MI, USA, March, 2005.</p>
               <p>The goal was to identify an assay suitable for measuring egg-laying in a 96-well format, avoiding the necessity for visual
                  inspection of multiple cultures. This technique was published in <a href="#bib31">Ellerbrock et al. (2004)</a>.
               </p>
               <p>Based on work done in some species of parasitic nematodes, it appeared that the rate of release of chitinase into culture
                  medium could be used as a measure of egg hatching. We confirmed that chitinase accumulation in medium is directly correlated
                  with egg hatching in <span class="emphasis"><em>C. elegans</em></span> by experiments involving wt and a couple of Egl strains. The cultures do not have to be synchronous for the method to work.
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_5_1"></a>9.5.1.&nbsp;Methods
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Wells of a flat-bottomed 96-well plate received a total of 200 ul medium, including 167 ul culture medium (M9 buffer + <span class="emphasis"><em>E. coli</em></span> broth + 6 ul DMSO +/<span class="symbol">&#8722;</span> drug as appropriate) and 33 ul worm suspension (50&#8211;90 worms in M9 buffer). Chitinase activity was measured after a significant
                     proportion of the worms were gravid and L1 larvae were apparent (this is not time-dependent to any degree; the enzyme activity
                     is very stable in the culture supernatants).
                  </p>
                  <p>To measure enzyme activity, 10 ul of a 0.8 mM solution of chitinase substrate (4-methylumbelliferyl-beta-D-N,N<span class="symbol">&#8242;</span>N<span class="symbol">&#8243;</span>-triacetylchitotrioside, Sigma Chemical Co.) is added to each well. Plates are incubated at 37 C for 1 hr. The reaction is
                     terminated by addition of 100 ul alkaline buffer (1 M glycine/1 N NaOH, pH 10.6). Fluorescence is determined on a 96-well
                     fluorimeter at excitation 360/40, emission 460/40.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9_6"></a>9.6.&nbsp;Analyzing the temporal dynamics of egg-laying
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Bill Schafer, University of California, San Diego, La Jolla, CA, USA.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_6_1"></a>9.6.1.&nbsp;Introduction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Egg-laying behavior provides an important phenotypic assay for genetic analysis of nervous system function. In particular,
                     egg-laying is a useful behavioral readout for the activity of the Go/Gq signaling network in neurons and muscle cells, and
                     for neuromodulation by serotonin, acetylcholine, and neuropeptides. The most common assays for egg-laying phenotypes count
                     the number of eggs retained in the uterus, an indirect measure of overall egg-laying rate,. More detailed assays of the temporal
                     pattern of egg-laying can in principle make it possible to distinguish effects on different egg-laying signal transduction
                     pathways. In the presence of abundant food, wild-type animals lay eggs in a specific temporal pattern:  egg-laying events
                     tend to be clustered in short bursts, or active phases, which are separated by longer inactive phases during which eggs are
                     retained. This egg-laying pattern can be accurately modeled as a three-parameter probabilistic process, in which animals fluctuate
                     between discrete inactive, active, and egg-laying states. Mutations affecting different cellular or molecular components of
                     the egg-laying circuitry have different effects on this pattern. However, because egg-laying events are infrequent, direct
                     monitoring of the temporal dynamics of egg-laying requires the use of long-term video recording and analysis.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_6_2"></a>9.6.2.&nbsp;Materials
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The essential piece of equipment is a microscope equipped for video recording of worms at approximately 50 X magnification
                     (i.e. high enough to easily see eggs on a video image). Some mechanism for recording video (e.g., a VCR) is also required.
                     Most applications of this technique have also utilized a motorized stage controlled by an automated tracking system that allows
                     the worm to be continuously maintained in the field of view during a long recording. Several systems appropriate to this purpose
                     have been described. In the absence of such a tracking system, it is in principle to manually maintain the animals within
                     the field of view throughout the recording, though the required durations of these recordings would make this a tedious task.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_6_3"></a>9.6.3.&nbsp;Protocol (modified from <a href="#bib82">Waggoner et al., 1998</a>)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Assay conditions and growth media: Unless otherwise noted, nematodes are grown and assayed at room temperature on standard
                     nematode growth medium (NGM) seeded with <span class="emphasis"><em>E. coli</em></span> strain OP50 as a food source. Typically first-day adult hermaphrodites (obtained by picking L4 hermaphrodites the previous
                     day) are used. Drugs (e.g., serotonin or levamisole) can be added directly to the agar before pouring.
                  </p>
                  <p>Recording and analysis of <span class="emphasis"><em>C. elegans</em></span> egg-laying behavior:  Single animals are observed as they crawled across a standard 8 cm agar plate using a stereoscopic
                     dissecting microscope equipped with a computer-controlled motorized stage. A video camera records the animal's behavior at
                     standard video frame rate, and a time stamp is used to indicate the time of egg-laying events on the resulting videotape.
                     For the tracking systems commonly used for these analyses, egg-laying times are determined by manually replaying the videotapes.
                     An egg-laying event is defined as a muscle contraction that leads to the expulsion of one or more eggs. An automated algorithm
                     for egg-detection has been described (<a href="#bib35">Geng et al., 2005</a>), though it has not been extensively used for actual behavioral analysis.
                  </p>
                  <p><span class="emphasis"><em>ML estimates of model parameters from real data.</em></span> For wild-type N2 and most N2-derived mutant strains, egg-laying occurs in short bursts, or active phases. The onset of the
                     active phase can be modeled as an exponential random variable with a time constant on the order of 20-30 minutes, while egg-laying
                     events within the active phase model as an exponential random variable with a time constant of around 20 seconds.
                  </p>
                  <p>For egg-laying events that conform to this pattern, we can prove that the random variable governing the egg-laying interval
                     times has the following probability density function (p. d. f.) (<a href="#bib91">Zhou et al., 1998</a>):
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure11"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="WMBehavFig11.jpg"><img src="WMBehavFig11_s.jpg" border="" align="middle" alt="WormBook Behavior Formula figure 11"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 11.&nbsp;</b></p>
                     </div>
                  </div>
                  <p>One way these parameters can be estimated for a given set of data is using the maximum likelihood method. This involves maximizing
                     the likelihood function for the above probability density function for the observed interval data. Given N observations of
                     the intervals x = [x<sub>1</sub>,x<sub>2</sub>, &#8230;, x<sub>N</sub>], the likelihood function is given by:
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure12"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="WMBehavFig12.jpg"><img src="WMBehavFig12_s.jpg" border="" align="middle" alt="WormBook Behavior Formula figure 12"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 12.&nbsp;</b></p>
                     </div>
                  </div>
                  <p>which is a function of the parameter vector <span class="symbol">&#952;</span> = [p, <span class="symbol">&#955;</span><sub>1</sub>, <span class="symbol">&#955;</span><sub>2</sub>]. The maximum likelihood 'estimate' is defined as the <span class="symbol">&#952;</span> that maximizes f(x|<span class="symbol">&#952;</span>) over all possible <span class="symbol">&#952;</span>'s. The bi-modal peak information provided by the histogram of the log intervals was used to obtain a rough estimate of <span class="symbol">&#952;</span>  and initialize the non-linear ML algorithm, which adjusted the parameters to maximize the likelihood function.
                  </p>
                  <p>For some cases (e.g., N2 animals on serotonin, some Egl-c mutants, some non-N2 nematode strains) a pattern resembling a homogeneous
                     Poisson process is observed. The time constant for egg-laying in these cases can be estimated by performing a weighted least-squares
                     linear regression to the log tail distribution (see below).
                  </p>
                  <p><span class="emphasis"><em>Log tail distribution:</em></span> One useful method for assessing the temporal pattern of egg-laying is the log tail distribution. It can be shown that the
                     log tail probability (i.e., the log of the probability that a given interval will be longer than time x) for intervals generated
                     by animals fluctuating between active and inactive egg-laying states is given by:
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure13"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="WMBehavFig13.jpg"><img src="WMBehavFig13_s.jpg" border="" align="middle" alt="WormBook Behavior Formula figure 13"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 13.&nbsp;</b></p>
                     </div>
                  </div>
                  <p>When the interval duration t is plotted against the log of the frequency of intervals greater than t, a biphasic plot will
                     be obtained. For the long intervals (x <span class="symbol">&gt;</span>5/<span class="symbol">&#955;</span><sub>1</sub>), ln Pr(X<span class="symbol">&#8805;</span>x) becomes approximately linear with respect to x, with slope -p<span class="symbol">&#955;</span><sub>2</sub>. Thus, the slope of this plot is inversely proportional to the average duration of the inactive phase. If the pattern follows
                     a single exponential distribution, the log tail distribution will be linear with a negative slope equal to the time constant
                     for egg-laying.
                  </p>
                  <p><span class="emphasis"><em>Histogram of log intervals:</em></span> Another way to visualize the overall distribution is with a histogram of interval times. Since the intervals between eggs
                     laid are clustered at short intervals and sparse at long intervals, it is more meaningful for reasons of better dynamic range
                     to study the distribution of the log intervals (denoted by Y = ln X). Animals showing an N2-like clustered pattern exhibit
                     a bimodal distribution, which can be shown to be represented by the following p. d. f. of Y:
                  </p>
                  <div class="literallayout">
                     <p></p>
                  </div>
                  <div class="mediaobject" align="center"><a name="figure14"></a><table border="0" summary="manufactured viewport for HTML img" cellspacing="0" cellpadding="0" width="270">
<tr>
  <td align="center"><p><a href="WMBehavFig14.jpg"><img src="WMBehavFig14_s.jpg" border="" align="middle" alt="WormBook Behavior Formula figure 14"></a>
  </td>
</tr>
                     </table>
                     <div class="caption">
                        <p><b>Figure 14.&nbsp;</b></p>
                     </div>
                  </div>
                  <p>We can show that when <span class="symbol">&#955;</span>1 and p <span class="symbol">&#955;</span>2 are sufficiently different, fY(y) peaks at y = ln(1/<span class="symbol">&#955;</span>1) and ln(1/p<span class="symbol">&#955;</span>2) with corresponding peak heights k1/e and k2/e. When the values of <span class="symbol">&#955;</span>1 and p<span class="symbol">&#955;</span>2 are close (or when egg-laying events are unclustered), only a single peak is observed.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_6_4"></a>9.6.4.&nbsp;Considerations
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p><span class="emphasis"><em>Advantages:</em></span> One significant advantage of this type of analysis is the ability to more precisely classify the phenotypes of egg-laying
                     defective mutants. Some mutants with slow egg-laying rates (e.g., mutants defective in HSN function) show a lengthened time
                     constant for the long interburst intervals but normal or even a shorter time constant for the short, intraburst intervals.
                     In contrast, other mutants (e.g., mutants affecting muscle calcium channels) exhibit an unclustered, or less clustered, egg-laying
                     pattern. A second advantage is a greater sensitivity in detecting subtle egg-laying phenotypes. For example, mutants defective
                     in vulval muscle nicotinic acetylcholine receptors show increased time constants for the short intraburst intervals, even
                     though they are not visibly Egl. Finally, this method has the advantage of being a direct measure of egg-laying behavior.
                     Unless the uterus is depleted of eggs, it does not appear that the number of eggs in the uterus significantly alters the probability
                     of an egg-laying event; thus, as long as a mutant retains nearly normal numbers of eggs there is less concern that effects
                     on the rate of egg-production will confound the scoring of an egg-laying phenotype.
                  </p>
                  <p><span class="emphasis"><em>Disadvantages</em></span>: As noted above, mutants that do not retain many eggs are difficult to analyze with this method. Practically, the analysis
                     of video recordings is relatively time- and labor-intensive. To obtain reliable timing parameters, it is usually necessary
                     analyze at least 100 intervals between egg-laying events, a number that generally requires analysis of at least 30 hours of
                     recording. Thus, phenotyping methods based on evaluating the number or developmental stage of eggs retained in the uterus
                     are far more effective assays for identifying egg-laying defective mutants in screens.
                  </p>
                  <p>See also &#8220;<a href="#sec4_9" title="4.9.&nbsp;Males attraction to hermaphrodites">Leaving food by males, not hermaphrodites</a>&#8221; in  Chemosensation section.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9_7"></a>9.7.&nbsp;Male mating behaviors
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by Curtis Loer, University of San Diego, San Diego, CA, USA, March, 2005.</p>
               <p>Ideally, all these behaviors are scored blindly (<a href="#bib57">Loer and Kenyon, 1993</a>).
               </p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_7_1"></a>9.7.1.&nbsp;Male tail curling in liquid
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p>Pick L4 males to seeded plate, score the next day (Some loss may occur from male &#8220;leaving&#8221;).</p>
                        </li>
                        <li>
                           <p>Adult males picked with minimal bacteria and released by swishing into 2 ml M9 (+/<span class="symbol">&#8722;</span> drugs) in 24-well plate. Typically scored with about 10-20 males/well.
                           </p>
                        </li>
                        <li>
                           <p>After 15&#8211;20 minutes, observe each male with stereomicroscope for 10 sec. Males with the tail tightly curled ventrally for
                              <span class="symbol">&#8805;</span> 5 sec are scored as &#8220;curled.&#8221; At higher drug concentrations, some worms will be paralyzed in this position.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <div class="sidebar">
                     <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                        <h3 class="title">Notes</h3>
                        <div class="itemizedlist">
                           <ul type="disc">
                              <li>
                                 <p>Always make drug-containing solutions immediately prior to use, unless one is certain about stability long-term in solution.</p>
                              </li>
                              <li>
                                 <p>In this assay. baseline curling (M9 alone) is about 8-12% males curled <span class="symbol">&gt;</span>50% of the 10 sec assay; in 20 mM serotonin, about 75% (Figure 2B, <a href="#bib57">Loer and Kenyon, 1993</a>).
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_7_2"></a>9.7.2.&nbsp;Male 'turning' assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <div class="procedure">
                     <ol type="1">
                        <li>
                           <p>Pick L4 males to seeded plate, score the next day (Some loss may occur from male &#8220;leaving&#8221;).</p>
                        </li>
                        <li>
                           <p>Place four young adult <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-51" target="_blank">unc-51</a></em></span> hermaphrodites as partners scattered about the mating spot lawn away from the edge.
                           </p>
                        </li>
                        <li>
                           <p>Individual adult males are picked and placed in the spot as gently as possible. At least 5 min recovery is allowed prior to
                              observations.
                           </p>
                        </li>
                        <li>
                           <p>Observe each male worm for 5 min, beginning when the male's tail first contacts a hermaphrodite. One can gently shuffle plates
                              under the stereoscope until finding a male very near or in contact with a hermaphrodite. Plates are closed when observed.
                           </p>
                        </li>
                        <li>
                           <p>Turns are scored as good, sloppy, missed. Each male is scored for the percentage of each type of turn. See definitions below.</p>
                        </li>
                     </ol>
                  </div>
                  <div class="sidebar">
                     <div class="note" style="margin-left: 0.5in; margin-right: 0.5in;">
                        <h3 class="title">Notes</h3>
                        <div class="itemizedlist">
                           <ul type="disc">
                              <li>
                                 <p>Mating assay plates are NGM agar seeded with tiny spot of OP50, grown 2-3 days at RT.</p>
                              </li>
                              <li>
                                 <p>Use young adult <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-51" target="_blank">unc-51</a></em></span> (completely paralyzed) hermaphrodites as partners (a day or two post-L4 stage). Don't use older worms as they become flaccid.
                                    To maintain <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-51" target="_blank">unc-51</a></em></span> worms one must pick up and distribute worms around the lawn every few days. Otherwise, the worms will starve locally on plates
                                    with plenty of food. Without assistance, the worms &#8220;diffuse&#8221; very slowly from where they are born.
                                 </p>
                              </li>
                              <li>
                                 <p>At least 5 min recovery is allowed prior to observations, but plates of different conditions or genotypes should be mixed
                                    since some worms will have varying recovery times if all worms are placed at the same time prior to beginning observations.
                                 </p>
                              </li>
                              <li>
                                 <p>Strains of males that &#8220;ignore&#8221; hermaphrodites (defective in recognition upon tail contact) can be difficult to score for turning.
                                    They are less likely to initiate backing upon contact, and more likely to leave the hermaphrodite.
                                 </p>
                              </li>
                              <li>
                                 <p>Observations are suspended if a male copulates. One can return to observe the worm in 10-15 min and observe the worm for the
                                    remaining time, assuming copulation has been completed.
                                 </p>
                              </li>
                              <li>
                                 <p>We found no difference in this assay with N2 males generated by heat-shock (or maintained by mating) vs CB1490.</p>
                              </li>
                              <li>
                                 <p>[<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1490;class=Variation" target="_blank">e1490</a></em></span>)] males, so most of our assays were done with <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> strains that generate about 30% males.
                                 </p>
                              </li>
                              <li>
                                 <p>Males making fewer than 3 turns in 5 min are excluded from the results. Each worm is scored for the percentage of each type
                                    of turn: good, sloppy, missed. (Rather than pooling all turns from multiple worms, which would skew the results toward worms
                                    making many turns during the observation period. The number of turns in 5 min varies widely.
                                 </p>
                              </li>
                           </ul>
                        </div>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec9_7_3"></a>9.7.3.&nbsp;Definitions of types of turns
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>A &#8220;good&#8221; turn occurs when the male tail remains in contact with the hermaphrodite throughout the turn and continues backing
                     on her opposite side after the turn is completed. A &#8220;sloppy&#8221; turn occurs when the male's tail temporarily loses contact, but
                     regains contact on the opposite side quickly because of a favorable trajectory (weak or late curling occurred, but was sufficient).
                     A &#8220;missed&#8221; turn occurs when the male's tail sails off the end of the hermaphrodite, completely losing contact. This can occur
                     either because the tail curl is not as tight or because the tail curl is late. With missed turns, the male often moves off
                     away from the hermaphrodite and only resumes &#8220;turning&#8221; upon contact with this or another hermaphrodite in the mating spot
                     (See Figure 4 of <a href="#bib57">Loer and Kenyon, 1993</a>).
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec9_8"></a>9.8.&nbsp;Nicotine adaptation with or without levamisole induction
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Adapted from the literature (<a href="#bib81">Waggoner et al., 2000</a>): Acute egg-laying response to levamisole can be measured after 1 hr in M9 plus levamisole. This response can be suppressed
                  by pre-adaptation to nicotine. Animals are placed on 30 mM nicotine  seeded NGM plates for varying lengths of time, then tested
                  for response to levamisole in individual liquid M9 assays. To maximize the possibility that a levamisole-sensitive animal
                  would lay eggs, eggs were counted after 4 hr in levamisole in these assays.
               </p>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec10"></a>10.&nbsp;Assays of <span class="emphasis"><em>C. elegans</em></span> reproductive behaviors
                     </h2>
                  </div>
               </div>
            </div>
            <p>The follow section on reproductive behaviors was compiled and contributed by Scott Emmons, Albert Einstein College of Medicine,
               Bronx, NY, USA, May, 2005.
            </p>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec10_1"></a>10.1.&nbsp;Hermaphrodite sexual behavior
                        </h3>
                     </div>
                  </div>
               </div>
               <p>As may be expected for a self-fertile hermaphrodite, <span class="emphasis"><em>C. elegans</em></span> hermaphrodites resist mating by males, especially when they are young and have self-sperm. Males find it impossible to insert
                  their spicules into some individual hermaphrodites  (<a href="#bib56">Liu and Sternberg, 1995</a>). Two behaviors have been defined associated with mating resistance, <span class="emphasis"><em>sprinting</em></span> and <span class="emphasis"><em>sperm expulsion</em></span>, (<a href="#bib6">Barker, 1994a</a>; <a href="#bib7">Barker, 1994b</a>; <a href="#bib52">Kleemann, 2005</a>). In sprinting, hermaphrodites move forward quickly when touched by a male. After a male ejaculates into the uterus and swims
                  away, contractions of the uterus may cause the seminal fluid to be expelled out through the vulva.
               </p>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec10_2"></a>10.2.&nbsp;Sprinting assay
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by G. Kleemann and Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_2_1"></a>10.2.1.&nbsp;Reagents and equipment
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Plastic Petri plates (6-cm-diameter), NGM agar medium, <span class="emphasis"><em>Escherichia coli</em></span> strain OP50 and compound microscope with moveable stage, video camera, and monitor.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_2_2"></a>10.2.2.&nbsp;Method
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The day before, put 5 <span class="symbol">&#956;</span>l of bacteria from an overnight culture of <span class="emphasis"><em>E. coli</em></span> strain OP50 in the center of each plate, creating a 5 mm circle of bacteria. Leave the plates on the bench overnight. Move
                     individual larval (L4 stage) worms to separate plates about 12 hours prior to the assay. At the beginning of the experiment,
                     place a single hermaphrodite and a single male on opposite sides of a bacterial patch. Put the whole Petri dish on the microscope
                     stage. Trials can be videotaped through the 5X objective of a compound microscope. The male is kept in the field of view using
                     the moveable stage, and the behaviors of the worms are videotaped for later analysis.
                  </p>
                  <p>Hermaphrodite sprinting is defined as a visually distinct increase in the velocity of hermaphrodite swimming (at least 2 seconds
                     in duration) while the male is in contact but prior to sperm transfer.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec10_3"></a>10.3.&nbsp;Sperm expulsion assay
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by G. Kleemann and Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_3_1"></a>10.3.1.&nbsp;Reagents and equipment
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Same as for the Sprinting assay (above).</p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_3_2"></a>10.3.2.&nbsp;Method
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>The setup is the same as for the <span class="emphasis"><em>Sprinting assay</em></span> above. Video tape the activities of the worms for one hour or 15 minutes after successful sperm transfer. Fifteen minutes
                     is sufficient to allow sperm to migrate beyond the opening of the vulva making sperm ejection unlikely. After sperm transfer
                     is observed, keep the hermaphrodite vulva in the field of view for 15 minutes. The male ejaculate will displace eggs in the
                     hermaphrodite uterus and can be seen through the body wall as an area filled with clear fluid. Individual sperm cells may
                     be visible as small round cells. Score sperm ejection when fluid is expelled from the vulva. Often a singly mated hermaphrodite
                     ejects sperm multiple times, therefore the number of sperm ejection bouts can also be recorded.
                  </p>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec10_4"></a>10.4.&nbsp;Male sexual behavior
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Compiled and contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_1"></a>10.4.1.&nbsp;Mate-searching
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>An isolated male on a restricted food source on an agar plate eventually leaves the food and wanders about the agar surface.
                     This is a sex-specific and age-specific behavior not expressed by adult hermaphrodites or juveniles of either sex. Moreover,
                     males do not leave a food source if hermaphrodites are present (providing a means for assaying male detection and response
                     to hermaphrodites, see below). Since wandering behavior is not expressed in the presence of hermaphrodites, it appears to
                     be a mate-searching behavior. Mate-searching is assayed by measuring the rate at which isolated males leave a food spot in
                     the <span class="emphasis"><em>Leaving Assay</em></span> (Adapted with permission from <a href="#bib55">Lipton et al., 2004</a>, Copyright 2004 by the Society for Neuroscience). Mutant males that do not leave food are said to have a Las phenotype (leaving
                     assay defective).
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_2"></a>10.4.2.&nbsp;Leaving assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Adapted with permission from <a href="#bib55">Lipton et al. (2004)</a>. Contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_2_1"></a>10.4.2.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Plastic Petri plates (9-cm-diameter)
                                                    agar medium [17 gm of agar (Difco, Detroit, <a href="http://www.wormbase.org/db/get?name=MI;class=Cell" target="_blank">MI</a>), 2.7 gm of Bactopeptone (Difco), 0.55 gm of Tris base (Sigma, St. Louis, MO), 0.27 gm of Tris HCl (Sigma), 2.0 gm of NaCl
                        (Fisher Scientific, Pittsburgh, PA), and 1 ml of ethanol containing 5 mg/ml cholesterol (Sigma), per liter H<sub><sub>2</sub></sub>O] <span class="emphasis"><em>Escherichia coli</em></span> strain OP50 (OD=1.0).
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_2_2"></a>10.4.2.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Prepare plastic Petri plates with 10 ml agar medium and allow plates to dry (lids closed) overnight on a laboratory bench.
                        A smaller amount of agar per plate is used (10 ml) compared with the standard amount for genetics (23 ml) because it increases
                        the ease of scoring worm tracks on the agar surface.
                     </p>
                     <p>Inoculate each plate in the center with 18 <span class="symbol">&#956;</span>l of <span class="emphasis"><em>E. coli</em></span> strain OP50 grown to OD600 = 1.0 and allow plates to incubate for 12-16 hr at room temperature to establish a small circular
                        lawn of approximately 9 mm diameter. The dryness of the agar surface and degree of bacterial growth are the most critical
                        variables for reproducible results.
                     </p>
                     <p>Select approximately 20 L4 larval worms from worm cultures and hold on same-sex seeded plates for 12 hr to mature.</p>
                     <p>To start the assay, place around 20 single males individually on the bacterial lawns of assay plates. Keep the plates at room
                        temperature and examine them at intervals to determine if the male is a leaver or non-leaver. An animal is scored as a leaver
                        if it is present outside a 3.5 cm radius circle (1 cm from the edge of the plate) or if its track shows that it passed beyond
                        this circle during the preceding time interval. Tracks can often be most easily seen by observing the plate tilted at angles
                        to observe the reflections of laboratory lights on the agar surface.
                     </p>
                     <p>Leaving rate is calculated as the probability of leaving per hour, PL. Plots of log (fraction non-leavers) versus time yield,
                        during the first 24 hours, straight lines passing through the point representing 100% non-leavers at time 0. Such linear plots
                        reflect the constant probability per unit time that a male will leave. PL is estimated as the hazard obtained by fitting an
                        exponential parametric survival model to the censored data using maximum likelihood (software R; <a href="http://www.R-project.org" target="_blank">http://www.R-project.org</a>). Under normal conditions, wild type males leave with a PL value of around 0.17-about half the males will have left in the
                        first 5 hours of the assay. To control for fluctuations in conditions, the leaving rate of experimental animals should be
                        compared to that of wild type controls tested side-by-side on the same batch of plates.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_3"></a>10.4.3.&nbsp;Detection of hermaphrodites
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <p>Males can detect hermaphrodites and chemical substances (pheromones) produced by hermaphrodites. This ability can be assayed
                     using a variation of the <span class="emphasis"><em>Leaving assay</em></span> described above (<a href="#bib55">Lipton et al., 2004</a>) or by observing male behavior on agar conditioned by hermaphrodites (<a href="#bib73">Simon and Sternberg, 2002</a>). These assays may reveal responses to multiple signals, including non-sexual chemical cues, for example those involved in
                     social interactions (<a href="#bib27">de Bono and Bargmann, 1998</a>), as well as sex pheromones.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_4"></a>10.4.4.&nbsp;Staying assay (leaving assay with hermaphrodites present)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Adapted with permission from <a href="#bib55">Lipton et al. (2004)</a>, Copyright 2004 by the Society for Neuroscience. Contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx,
                     NY, USA, May, 2005.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_4_1"></a>10.4.4.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Same as for <span class="emphasis"><em>Leaving assay</em></span>.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_4_2"></a>10.4.4.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Carry out a <span class="emphasis"><em>Leaving assay</em></span> as described above, placing 2-5 wild type or paralyzed (e.g., <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-51" target="_blank">unc-51</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e364;class=Variation" target="_blank">e364</a></em></span>)) hermaphrodites on the bacterial lawn. Under these conditions, wild type males do not leave food (PL <span class="symbol">&#8764;</span>0).
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_5"></a>10.4.5.&nbsp;Response to pheromone assay, variation 1
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>From <a href="#bib73">Simon and Sternberg (2002)</a>, contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_5_1"></a>10.4.5.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>5-cm diameter agar plates, OP50 and video system attached to the dissecting microscope.</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_5_2"></a>10.4.5.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Prepare the plates: three drops of OP50-inoculated LB, spread thinly, with a <span class="symbol">&#8764;</span>0.25-centimeter gap between the edge of the lawns and the walls of the plates. Plates were stored at <span class="symbol">&#8764;</span>22&deg;C for 2 days until used for trials.
                     </p>
                     <p>Harvest source and test animals at the fourth larval stage (L4), and store at <span class="symbol">&#8764;</span>18&deg;C overnight with 10-20 animals per same-sex plate to be used the following day in trials as young adults.
                     </p>
                     <p>Condition agar plates for at least 3 and up to 8 hours with 10-20 source animals (paralyzed muscle mutant <span class="emphasis"><em>unc-52</em></span>). Spread out the source animals along a 1 cm line (see <a href="#bib73">Simon and Sternberg, 2002</a>). Remove the source animals 5 to 10 min before the onset of a trial.
                     </p>
                     <p>Introduce a single test animal on a single trial plate, 1 cm from the conditioned scoring region, orienting it in the direction
                        of the scoring region. Document (videotape) its movement for 5 min.
                     </p>
                     <p>Score and record the number of reversals made when the male crosses the conditioned region and the time animals spend in the
                        conditioned region. Backwards movement equal to or greater than a body length was counted as &#8220;1 reversal&#8221;; backward movement
                        less than a body length was counted as &#8220;0.5 reversal.&#8221;
                     </p>
                     <p>The frequency of reversals is used as an indication that males detect a cue.</p>
                     <p>Equal numbers of unconditioned and conditioned agar trials are run in parallel on <span class="symbol">&#8764;</span>5 plates each per day. Trials are run blindly and interspersed at random. Unconditioned agar trials are meant to parallel
                        conditioned agar trials as closely as possible: (i) made from same day bacterial lawns, (ii) kept at the same temperature,
                        and (iii) have bacteria picked on and off to mimic introduction of source animals as on conditioned agar trials.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_6"></a>10.4.6.&nbsp;Response to pheromone assay, variation 2
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Contributed by A. Barrios and Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_6_1"></a>10.4.6.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>3.5-cm diameter agar plates, OP50, digital video camera attached to the dissecting microscope and M9 buffer.</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_6_2"></a>10.4.6.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Prepare the plates: seed the plates the day before, spreading the OP50 thinly over the entire agar surface.</p>
                     <p>Harvest test L4 males the day before and keep at 20&deg;C overnight.</p>
                     <p>Incubate 40 <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> or N2 adult hermaphrodites in 40 <span class="symbol">&#956;</span>l of M9 buffer for 4hrs before the assay, room temperature.
                     </p>
                     <p>Place 2 <span class="symbol">&#956;</span>l of M9 buffer (control spot) and 2 <span class="symbol">&#956;</span>l of hermaphrodite-conditioned M9 buffer (conditioned spot) 1.5 cm distance away from each other on the centre of the plate.
                        Wait until the spots are dry before starting the trial.
                     </p>
                     <p>Place 10 to 12 males equidistant from the two spots near the edge of the plate (about 1.5 cm from each spot) and record their
                        locomotion for 20 min. After approximately 15 min, one can observe accumulation of worms in the conditioned spot. For each
                        worm that enters a spot (conditioned and control), score the time it spends in the spot before exiting it again (time leaving
                        spot - time entering spot). Calculate the ratio of average time on conditioned spot to average time on control spot. Typical
                        values for wild type worms are 148 sec &plusmn; 14 sec (s.e.m.) (conditioned), 47 sec &plusmn; 7 sec (s.e.m.) (unconditioned); conditioned/unconditioned
                        = 3.1.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_7"></a>10.4.7.&nbsp;Holding assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Compiled and contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <p>The <span class="emphasis"><em>Holding Assay</em></span> (<a href="#bib73">Simon and Sternberg, 2002</a>) is identical to the <span class="emphasis"><em>Response to Pheromone Assay, Variation 1</em></span>, with the following modifications:
                                                 
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Source animals are introduced in a pile in the center of a plate, rather than along a 1 cm line.</p>
                        </li>
                        <li>
                           <p>After removal of the source animals, test animals are introduced directly on this conditioned spot, rather than 1 cm away
                              from it.
                           </p>
                        </li>
                        <li>
                           <p>In addition to the center circle, number of reversals is scored in outwardly stacked, concentric ring-areas (with radii of
                              1 mm, 5 mm, 16.5 mm, and 25 mm; unequal sized areas are normalized by dividing outer ring-areas by 24, 247, and 330, respectively).
                           </p>
                        </li>
                        <li>
                           <p>Movements of test animals are documented for 10 min and compared in each area. A difference between scores on conditioned
                              and unconditioned plates indicates the male's behavior is modified persistently and not just on first encounter of a signal.
                              Moreover, the alteration in male behavior results in his remaining close to the hermaphrodite.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>
                                            
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_8"></a>10.4.8.&nbsp;Attraction assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Compiled and contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <p>(Similar to the <span class="emphasis"><em>Tendency to Mate Assay</em></span> below.)
                  </p>
                  <p>The <span class="emphasis"><em>Attraction Assay</em></span> (<a href="#bib73">Simon and Sternberg, 2002</a>) is similar to the <span class="emphasis"><em>Holding Assay</em></span> with the following modifications:
                                                 
                  </p>
                  <div class="orderedlist">
                     <ol type="1">
                        <li>
                           <p>Source animals are left to condition a point source for 24h.</p>
                        </li>
                        <li>
                           <p>At the onset of each trial, a single hermaphrodite is placed on top of conditioned region.</p>
                        </li>
                        <li>
                           <p>Test males are introduced 1.5 cm from the conditioned point source/unconditioned scoring region.</p>
                        </li>
                        <li>
                           <p>Determine the time it takes for the male to find the hermaphrodite. This assay can be used to show that the male finds the
                              hermaphrodite more quickly on conditioned than on unconditioned medium. In other words, the modification in male behavior
                              consequent on detecting a signal has the result of more rapid mate finding.
                           </p>
                        </li>
                     </ol>
                  </div>
                  <p>
                                            
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_9"></a>10.4.9.&nbsp;Copulation (Cod phenotype)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Compiled and contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <p>Male copulatory behavior can be broken down into a series of sub-behaviors (<a href="#bib9">Barr and Sternberg, 1999</a>; <a href="#bib32">Emmons and Sternberg, 1997</a>; <a href="#bib56">Liu and Sternberg, 1995</a>; <a href="#bib57">Loer and Kenyon, 1993</a>). Assays for each of the main subdivisions of the behavioral series are given below. These subdivisions may themselves be
                     composed of further definable sub-steps. In addition to copulation itself, the time it takes for a male to begin copulation
                     when in the presence of hermaphrodites (<span class="emphasis"><em>Tendency to Mate</em></span>) can be determined.
                  </p>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_10"></a>10.4.10.&nbsp;Mating efficiency (Mab phenotype)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Compiled and contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <p>This assay really measures male fertility rather than behavior. It gives an overall indication of mating ability if the gonad
                     and morphology of the male are normal. It has been traditionally used to score the mating ability of mutant males (<a href="#bib43">Hodgkin, 1983</a>) and is useful for determining whether a particular genotype may be used as the male in a genetic cross (<a href="#bib45">Hodgkin et al., 1988</a>).
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_10_1"></a>10.4.10.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Standard 5-cm diameter agar plates and OP50.</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_10_2"></a>10.4.10.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Prepare cross plates: spread <span class="symbol">&#8764;</span>20 <span class="symbol">&#956;</span>l OP50 on standard 5 cm diameter agar plate to make a 1 cm bacteria lawn and store 2 days before use.
                     </p>
                     <p>Place 6 test males and 6 Unc or Dpy hermaphrodites at late L4 stage together on the bacterial lawn of a cross plate.</p>
                     <p>Remove males after 24 hour at 20&deg;C.</p>
                     <p>Transfer hermaphrodites to fresh regular agar plates each day for 4-5 days until they lay no eggs.</p>
                     <p>Count cross progeny (nonUnc or nonDpy) and self progeny (Unc or Dpy) each day.</p>
                     <p>Total cross progeny is divided by total progeny (cross plus self progeny) and the percentage is used as an indication of male
                        mating efficiency (ME): ME0 = never mating; ME1 = almost never mating (less than 1% of wild type); ME2 = poor mating (1-10%
                        of wild type); ME3 = fair-to-excellent mating (10-100% of wild type).
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_11"></a>10.4.11.&nbsp;Tendency to copulate
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>(Similar to the <span class="emphasis"><em>Attraction assay</em></span> above and the <span class="emphasis"><em>Response assay</em></span> below.)
                  </p>
                  <p>Contributed by L. Jia and Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_11_1"></a>10.4.11.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Standard 5-cm diameter agar plates, OP50 and dissecting microscope (with attached video system, optional).</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_11_2"></a>10.4.11.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Prepare mating plates: a small drop (18<span class="symbol">&#956;</span>l) of OP-50 bacteria spread on the center of a standard 5 cm diameter agar plate; dry the plates on the laboratory bench overnight
                        to make a <span class="symbol">&#8764;</span> cm diameter bacterial lawn.
                     </p>
                     <p>Isolate both hermaphrodites and males at the L4 stage and keep in isolation until used the following day in all observations
                        as healthy young adults.
                     </p>
                     <p>To begin the assay, place 4 <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-51" target="_blank">unc-51</a></em></span> hermaphrodites on one edge of the bacterial lawn and a single tester male on the other side of the bacterial lawn.
                     </p>
                     <p>Observe or document the male's movement for 10 min under a dissecting microscope.</p>
                     <p>Determine the interval from the start of the assay to the time when the male presses the ventral side of his tail against
                        the surface of a hermaphrodite in the Response step of copulation (see below). Plot a histogram (typically 20 males) of the
                        number that respond in less than 5 min, 5-10 min, <span class="symbol">&gt;</span>10 min (i.e. do not respond during the 10 min assay). Typically for wild type males, 90% will give a Response behavior (stop
                        forward locomotion, place ventral side of fan against hermaphrodite, back up) in the first 5 min, whereas the most severely
                        deficient mutants ignore the hermaphrodites altogether and never give a response within 10 min.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_12"></a>10.4.12.&nbsp;Response and vulva location assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>By K. Knobel and M. Barr, a variation of <a href="#bib56">Liu and Sternberg (1995)</a> and <a href="#bib9">Barr and Sternberg (1999)</a> (Lov phenotype),  contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.
                  </p>
                  <p>Response is a complex behavior. It begins when a male touches a hermaphrodite with his tail and stops his forward locomotion.
                     He next assumes a specific posture in which the posterior body curves slightly ventrally and the fan is pressed against the
                     side of the hermaphrodite. He then begins to back up by propagating anteriorly-moving waves through the anterior part of the
                     body, while maintaining the posterior body pressed against the hermaphrodite. All of these steps must be achieved to score
                     a positive response.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_12_1"></a>10.4.12.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Strains: <span class="emphasis"><em>unc-31 (<a href="http://www.wormbase.org/db/get?name=e169;class=Variation" target="_blank">e169</a>)</em></span> hermaphrodites and mating controls such as CB1490 <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1490;class=Variation" target="_blank">e1490</a></em></span>) (wild type control) and PT9 (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=pkd-2" target="_blank">pkd-2</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=sy606;class=Variation" target="_blank">sy606</a></em></span>); <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1490;class=Variation" target="_blank">e1490</a></em></span>) (Lov mutant).
                     </p>
                     <p>NGM plates, laboratory timer, liquid OP-50 culture, and dissecting microscope.</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_12_2"></a>10.4.12.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Day before mating assays: Set up worms to use for mating: Select L4 males from plate of the strains to be charac-terized and
                        place them on a pre-seeded plate. Make sure there are no hermaphrodites on the plate. Put worms at 15&deg;C (20&deg;C) overnight.
                     </p>
                     <div class="procedure"><a name="d0e4057"></a><p class="title">Procedure&nbsp;14.&nbsp;Mating day</p>
                        <ol type="1">
                           <li>
                              <p>Carefully drop 10 <span class="symbol">&#956;</span>l of concentrated OP-50 into the center of a worm plate. (Spin down 2 tubes of 1.5 ml ea OP-50 culture and resuspend both
                                 pellets in a total of about 100&#8211;150 <span class="symbol">&#956;</span>l.) Let plates dry at room temp (keep tops off until dry, around 30 minutes, ok at room temp or 15&deg;C overnight).
                              </p>
                           </li>
                           <li>
                              <p>Take worms out of 15&deg;C incubator and let warm up to room temp (about 30 minutes).</p>
                           </li>
                           <li>
                              <p>Place 12-20 adult <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-31" target="_blank">unc-31</a></em></span> hermaphrodites onto the lawn of a mating plate.
                              </p>
                           </li>
                           <li>
                              <p>Place 2-3 males of each strain onto the mating plate with the hermaphrodites and start the timer.</p>
                           </li>
                           <li>
                              <p>Observe each worm and score male behavior with respect to time for 4 minutes in the following categories:
                                                                     
                              </p>
                              <div class="orderedlist">
                                 <ol type="1">
                                    <li>
                                       <p>Touch (o): touch (note at what time this occurs) (x)-touch but no r.</p>
                                    </li>
                                    <li>
                                       <p>Response (r): male wraps fan around hermaphrodite like a suction cup and initiates backing.</p>
                                    </li>
                                    <li>
                                       <p>Back (b): male backs along length of hermaphrodite body.</p>
                                    </li>
                                    <li>
                                       <p>Forward (f): male moves forward along length of hermaphrodite body.</p>
                                    </li>
                                    <li>
                                       <p>Turn (t): male turns around head or tail of hermaphrodite  (note: t10= turn at 10% of herm body where 0 is head and 100 is
                                          tail tip) and t90 =turn at 90% of herm body-Also note if turning occurs at other locations).
                                       </p>
                                    </li>
                                    <li>
                                       <p>Locate (l) (stopping for greater than 1 sec) or pass (p) or hesitate (h) (pause for less than 1 sec) at the hermaphrodite
                                          vulva (note time).
                                       </p>
                                    </li>
                                    <li>
                                       <p>Off (&#8230;): worm leaves hermaphrodite momentarily and returns.: male leaves the hermaphrodite.</p>
                                    </li>
                                 </ol>
                              </div>
                              <p>For example: wild-type (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1490;class=Variation" target="_blank">e1490</a></em></span>) might look like: r/b/t/l (r= 1, l= 1/1 or 100%) or r/b/t/h/b/f/l (r=1, l= 1/2 or 50%)     <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=lov-1" target="_blank">lov-1</a></em></span> mutants: x/x/r/./r/./r/b/t/p/t/b/t/p/t/b/t/l (r=1, L=1/3 or 30%).
                              </p>
                           </li>
                           <li>
                              <p>After trial is over remove the males and start over with 2 more males. Collect data from at least 10-20 worms during 3-4 trials
                                 on different days.
                              </p>
                           </li>
                           <li>
                              <p>Data analysis  %Response: (# responding within 4 minutes/total # worms evaluated) x100.</p>
                              <p>%Vulva location: (# l /(# l+# p+# h) x100.</p>
                           </li>
                        </ol>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_13"></a>10.4.13.&nbsp;Response assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>By E. Peden and M. Barr (<a href="#bib60">Peden and Barr, 2005</a>), contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_13_1"></a>10.4.13.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Same as for <span class="emphasis"><em>Response and Vulva location assay</em></span>.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_13_2"></a>10.4.13.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Day before assay: segregate <span class="symbol">&gt;</span>50 L4 males O/N.
                     </p>
                     <p>Day of assay:  Prepare mating plates with 50-70 young adult hermaphrodites on a patch of bacteria made from a 10-12 microliter
                        OP50 drop (see <span class="emphasis"><em>Response and Vulva Location Assay</em></span> for making mating plates).
                     </p>
                     <p>Add 5 adult males to the hermaphrodite-rich mating plate.</p>
                     <p>Score # males that respond to a hermaphrodite in 3 minutes  (a single male may only be counted once).</p>
                     <p>Response is defined when the male stops at hermaphrodite and places ventral tail on a potential mate for <span class="symbol">&gt;</span> 20 consecutive seconds).
                     </p>
                     <p>Response Efficiency (RE) = [# males responding / (5 males)] X 100% (e.g., if 4 males respond, RE = 4/5 = 80%).</p>
                     <p>Run at least 5 trials per strain per day (=25 males) for at least 3 different days (at least 75 males total); compare at least
                        3 different strains per day.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_14"></a>10.4.14.&nbsp;Vulva location assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>By E. Peden and M. Barr, contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_14_1"></a>10.4.14.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Same as for Response and Vulva location assay.</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_14_2"></a>10.4.14.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Day before assay, segregate <span class="symbol">&gt;</span>50 L4 males O/N.
                     </p>
                     <p>Day of Assay: Prepare mating plates with <span class="symbol">&#8764;</span> 25 <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-31" target="_blank">unc-31</a></em></span> hermaphrodites.
                     </p>
                     <p>Add and observe 1 male at a time. Score behavior as for <span class="emphasis"><em>Response and Vulva location assay</em></span>.
                     </p>
                     <p>Count the number of passes or hesitations (encounters) at the vulva until the male FIRST stops at vulva.</p>
                     <p>Stop trial after male first stops at the vulva.</p>
                     <p>Location Efficiency (L.E.) = 1 / # encounters to stop  (e.g., if male stops at vulva on first encounter, then LE = 1/1; if
                        male stops at vulva on third encounter, then LE=1/3).
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_15"></a>10.4.15.&nbsp;Turning
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>By T. Liu and M. Barr, contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <p>Turning is a complex behavior expressed during the response phase of copulation when the male encounters either end of the
                     hermaphrodite. Males first must detect the end (often they detect its approach before they actually reach it, at about 10%
                     or 90% of body length). Detecting the end stops backing and triggers a deep ventral bend or arch of the posterior body region
                     so that the fan slides around to the other side of the hermaphrodite. Next, the deep ventral bend or arch is propagated along
                     the body to the head, while the posterior body region is held in the posture typical of backing and the fan slides along the
                     body. When the entire body is around the hermaphrodite, normal backing is resumed.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_15_1"></a>10.4.15.1.&nbsp;Background information for scoring male turning defects
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>See also <a href="#bib57">Loer and Kenyon (1993)</a>, who scored turns into three categories: &#8220;good,&#8221; &#8220;sloppy,&#8221; and &#8220;missed.&#8221;
                     </p>
                     <div class="orderedlist">
                        <p class="title">Step 1</p>
                        <ol type="1">
                           <li>
                              <p>Wild type turning:  stops backing as his tail reaches the end of hermaphrodite.</p>
                           </li>
                           <li>
                              <p>Defective phenotype: (A) continues backing though the tail has passed the end of hermaphrodite.</p>
                           </li>
                           <li>
                              <p>Phenotypic categorizations and representative mutants: &#8220;swim-off&#8221;, example: males lacking rays 7-9, males having reduced or
                                 absent dopamine/serotonin synthesis (<span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cat-1" target="_blank">cat-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cat-2" target="_blank">cat-2</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=bas-1" target="_blank">bas-1</a></em></span>).
                              </p>
                           </li>
                        </ol>
                     </div>
                     <div class="orderedlist">
                        <p class="title">Step 2</p>
                        <ol type="1">
                           <li>
                              <p>Wild type turning: makes a sharp ventral bend in the tail (i.e. the tail is pressed against hermaphrodite all the time) and
                                 turns to the other side of hermaphrodite.
                              </p>
                           </li>
                           <li>
                              <p>Defective phenotype: (B) makes a wide turn (i.e. at some point, the tail loses contact with hermaphrodite completely).</p>
                           </li>
                           <li>
                              <p>Phenotypic categorizations and representative mutants: &#8220;wide turn&#8221;, example: <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cat-1" target="_blank">cat-1</a></em></span>, <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cat-2" target="_blank">cat-2</a></em></span>(C) halts right before/after turning, then moves forward along hermaphrodite for a distance (ranges from 1/10&#8211;2/3 of hermaphrodite
                                 body length) and repeats backing.
                              </p>
                              <p>Phenotypic categorizations and representative mutants: &#8220;stutter turning&#8221;, example: <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=egl-3" target="_blank">egl-3</a></em></span>.
                              </p>
                           </li>
                        </ol>
                     </div>
                     <div class="orderedlist">
                        <p class="title">Step 3</p>
                        <ol type="1">
                           <li>
                              <p>Wild type turning: slides fan along body and propagates the deep ventral bend or arch along the body to the head, then continues
                                 backing along the other side of hermaphrodite.
                              </p>
                           </li>
                           <li>
                              <p>Defective phenotype: (D) the fan does not slide but stops on the other side of hermaphrodite immediately after it turns around,
                                 the rest of the male body begins arching until the male head reaches the hermaphrodite end, then the tail continues to lead
                                 backing.
                              </p>
                           </li>
                           <li>
                              <p>Phenotypic categorizations and representative mutants:, example: <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=cat-1" target="_blank">cat-1</a></em></span>.
                              </p>
                           </li>
                        </ol>
                     </div>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_15_2"></a>10.4.15.2.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Same as for <span class="emphasis"><em>Response and Vulva Location Assay</em></span>.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_15_3"></a>10.4.15.3.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Mating assay plates: inoculate 5cm NGM plates with 10ul OP50 (OD=1.0) and leave them on bench for 24 hours. Then they are
                        ready for immediate use or can be sealed and kept at 4&deg;C for up to 1 month.
                     </p>
                     <p>Male subjects: isolate 30-40 L4 males 12-20 hours prior to behavioral observation and keep them at 20&deg;C until use.</p>
                     <p>Hermaphrodite subjects: on the day of observation, pick 20 <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-31" target="_blank">unc-31</a></em></span> adult hermaphrodites to the mating plate.
                     </p>
                     <div class="procedure"><a name="d0e4314"></a><p class="title">Procedure&nbsp;15.&nbsp;Behavioral observation</p>
                        <ol type="1">
                           <li>
                              <p>Pick one male to mating plate each time.</p>
                           </li>
                           <li>
                              <p>If the strain is wild type in response, do a 5-minute observation, otherwise, do a 10-minute observation.</p>
                           </li>
                           <li>
                              <p>For each male observed, take notes for all the turns he makes during the entire observation time, i.e. wild type, swim-off,
                                 wide turn, or stutter turn (use m/n expression for stutter turn: m = how many times the male actually turns his tail to the
                                 other side of hermaphrodite (usually 1 or zero); n = how many times the male repeats the forward-backward motion for this
                                 single turn).
                              </p>
                           </li>
                           <li>
                              <p>Transform first-hand notes into statistically manageable data: for each male subject observed, give a numerical value &#8220;1&#8221;
                                 if a defective turning is recorded and &#8220;0&#8221; if none.
                              </p>
                           </li>
                        </ol>
                     </div>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_16"></a>10.4.16.&nbsp;Spicule prodding and insertion
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Contributed by R. Garcia  and Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <p>Spicule insertion is probably the most difficult step of mating behavior for the male to accomplish. This behavioral step
                     consists of two motor behaviors: the spicules prodding the vulval slit rhythmically, and the spicules penetrating the vulva.
                     Both of these behaviors require high magnification and cooperative hermaphrodites for consistent measurement.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_16_1"></a>10.4.16.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Standard 5-cm diameter agar plates</p>
                     <p>M9 buffer</p>
                     <p>Aerated overnight culture of OP50</p>
                     <p><span class="emphasis"><em>unc-52 (<a href="http://www.wormbase.org/db/get?name=e444;class=Variation" target="_blank">e444</a></em></span>) or <span class="emphasis"><em>unc-38 (<a href="http://www.wormbase.org/db/get?name=sy576;class=Variation" target="_blank">sy576</a>);egl-19 (<a href="http://www.wormbase.org/db/get?name=n582;class=Variation" target="_blank">n582</a>) unc-31 (<a href="http://www.wormbase.org/db/get?name=e169;class=Variation" target="_blank">e169</a>)</em></span> or <span class="emphasis"><em>unc-64 (<a href="http://www.wormbase.org/db/get?name=e246;class=Variation" target="_blank">e246</a>)</em></span>.
                     </p>
                     <p>Stereomicroscope with at least 300X total magnification with attached video and recording system or Compound microscope with
                        a long working distance 40X objective with attached video and recording system.
                     </p>
                     <p>Microscope slides.</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_16_2"></a>10.4.16.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Prepare plates: spread <span class="symbol">&#8764;</span> 40<span class="symbol">&#956;</span>l OP50 on standard 5 cm diameter agar plates to make a 2 cm bacterial lawn; store 2 days before use.
                     </p>
                     <p>Place 5 to 20 late L4 males together on the bacterial lawn. (Do not place more than 20 males together because they will aggregate
                        and burrow into the agar. Placing less than 5 males may cause the males to leave the bacteria and desiccate on the side of
                        the Petri plate.) Incubate at 20&deg;C for 24 hours.
                     </p>
                     <p>Spin down 1 ml of a fresh aerated overnight culture of OP50. Remove the media and resuspend the pellet in 100 ml of M9 buffer.</p>
                     <p>Add 10 ml of cells to 5 cm diameter agar plate. Let the liquid absorb into the agar. The OP50 should be concentrated enough
                        to form an instant lawn.
                     </p>
                     <p>To observe long runs of spicule prodding behavior, use a severely paralyzed egg-laying defective, <span class="symbol">&gt;</span> 24 hr adult hermaphrodite; <span class="emphasis"><em>unc-52</em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e444;class=Variation" target="_blank">e444</a></em></span>) or <span class="emphasis"><em>unc-38 (<a href="http://www.wormbase.org/db/get?name=sy576;class=Variation" target="_blank">sy576</a>); egl-19 (<a href="http://www.wormbase.org/db/get?name=n582;class=Variation" target="_blank">n582</a>)</em></span> double mutants work well. To observe spicule insertion, use a 48 hour paralyzed hermaphrodites such as <span class="emphasis"><em>unc-31 (<a href="http://www.wormbase.org/db/get?name=e169;class=Variation" target="_blank">e169</a>)</em></span> or <span class="emphasis"><em>unc-64 (<a href="http://www.wormbase.org/db/get?name=e246;class=Variation" target="_blank">e246</a>)</em></span>. Place 5 to 10 hermaphrodites on the lawn; let the hermaphrodites acclimate for 15 minutes before putting the males on the
                        lawn.
                     </p>
                     <p>Gently put the male on the lawn using a pick or a mouth tube. If using a mouth tube, try not to get too much liquid on the
                        bacterial lawn.
                     </p>
                     <p>If using a high magnification stereomicroscope, the animals can be immediately observed. If using a compound microscope with
                        a long working distance 40X objective, before adding the male, cut the block of agar containing the bacterial lawn from the
                        plate and put it on a standard microscope slide. Add the male to the bacterial lawn, put the slide on the microscope stage,
                        using the 10X objective to focus on the male; follow the male by adjusting the stage controls. When the male chooses a hermaphrodite,
                        focus on her vulva and switch the objective to 40X. During matings with <span class="symbol">&gt;</span>24 hr adult <span class="emphasis"><em>unc-52</em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-38" target="_blank">unc-38</a>; <a href="http://www.wormbase.org/db/gene/gene?name=egl-19" target="_blank">egl-19</a></em></span> hemaphrodites, the spicules of wild-type males should prod the vulva at frequency of 7 to 9 Hz (video recordings need to
                        be slowed down to count the spicule movements). During matings with 2 day old <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-31" target="_blank">unc-31</a></em></span> or <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-64" target="_blank">unc-64</a></em></span> hermaphrodites, the males should insert within a few seconds of vulva contact.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_17"></a>10.4.17.&nbsp;Mutation-induced spicule protraction (Prc phenotype)
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Contributed by R. Garcia  and Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <p>Normally, a male keeps his copulatory spicules within his tail prior to and between mating events. Certain mutations may disrupt
                     this regulation and cause males to protract their spicules spontaneously in the absence of hermaphrodites. The spicule muscles
                     and neurons are quite sensitive cells and precocious spicule protraction can provide a readout to determine if a mutation
                     can enhance cell excitability.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_17_1"></a>10.4.17.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Standard 5-cm diameter agar plates, OP50 and dissecting microscope.</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_17_2"></a>10.4.17.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Prepare plates: spread <span class="symbol">&#8764;</span> 40<span class="symbol">&#956;</span>l OP50 on standard 5 cm diameter agar plates to make a 2 cm bacteria lawn; store 2 days before use.
                     </p>
                     <p>Place 5 to 20 late L4 males together on the bacterial lawn. (Do not place more than 20 males together because they will aggregate
                        and burrow into the agar. Placing less than 5 males may cause the males to leave the bacteria and desiccate on the side of
                        the Petri plate.)
                     </p>
                     <p>After 24 hour at 20&deg;C, count the number of males that have at least one spicule protruding from their cloaca. Generally, anywhere
                        between 0 to 12% and 0 to 5% of virgin <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-5" target="_blank">him-5</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1490;class=Variation" target="_blank">e1490</a></em></span>) and <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=him-8" target="_blank">him-8</a></em></span> (<span class="emphasis"><em><a href="http://www.wormbase.org/db/get?name=e1489;class=Variation" target="_blank">e1489</a></em></span>) males (that is, wild type or nonPrc), respectively, will have their spicules protracted.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_18"></a>10.4.18.&nbsp;Drug-induced spicule protraction
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>Contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <p>Acetylcholine agonists such as levamisole, arecoline, nicotine and oxotremorine M can cause adult males to protract their
                     spicules. These drugs act on the spicule muscles and some drugs such as oxotremorine M also act on the neurons that innervate
                     them. Each of these drugs stimulate spicule protraction via a different mechanism; thus drug-induced spicule protraction can
                     be used as a readout to determine if a certain mutation may interrupt different types of pre-and post synaptic cholinergic
                     signaling.
                  </p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_18_1"></a>10.4.18.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Standard 5-cm diameter agar plates, OP50, Pyrex 3 well round bottom dish, Levamisol, arecoline, nicotine and oxotremorine
                        M (Sigma) and dissecting microscope.
                     </p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_18_2"></a>10.4.18.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Prepare plates: spread <span class="symbol">&#8764;</span> 40<span class="symbol">&#956;</span>l OP50 on standard 5 cm diameter agar plates to make a 2 cm bacteria lawn; store 2 days before use.
                     </p>
                     <p>Dissolve the drugs in distilled water. Stock concentrations are.</p>
                     <p>1mM for levamisole, 10mM for nicotine, 100mM for arecoline and oxotremorine. Drugs can be frozen at <span class="symbol">&#8722;</span>20&deg;C, but should not be repeatedly frozen and thawed more than 3 times. Keep arecoline and oxotremorine M in the dark.
                     </p>
                     <p>Place 5 to 20 late L4 males together on the bacterial lawn. (Do not place more than 20 males together because they will aggregate
                        and burrow into the agar. Placing less than 5 males may cause the males to leave the bacteria and desiccate on the side of
                        the Petri plate.)
                     </p>
                     <p>Dilute drugs appropriately in distilled water (do not use M9 buffer, somehow this interferes with drug response, and one must
                        use 10X more drug to get the same response curve). The range of working concentrations for the drugs are: levamisole (100nM
                        to 1mM), nicotine (1mM to 5mM), arecoline (10mM to 10mM). Oxotremorine M (100mM to 50mM).
                     </p>
                     <p>Add 1 ml of the drug to 1 well of the Pyrex 3 well round bottom dish.</p>
                     <p>Add 1 to 5 adult males to the drug using a worm pick. Watch the males for 5 minutes, score the male as sensitive if he keeps
                        his spicules protracted <span class="symbol">&#8764;</span> 10 secs.
                     </p>
                     <p>After one set of males has been scored, the males can be sucked out of the drug using a pipetter, and new males can be added
                        to the drug bath. Change the drug bath after every 3 trials or if the bath gets too dirty with bacteria.
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_19"></a>10.4.19.&nbsp;Sperm transfer assay
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>By G. Schindelman and P. Sternberg.</p>
                  <p>Compiled and contributed by Scott Emmons, Albert Einstein College of Medicine, Bronx, NY, USA, May, 2005.</p>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_19_1"></a>10.4.19.1.&nbsp;Reagents and equipment
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Standard 5-cm diameter agar plates, OP50, dissecting microscope with good optics, and laboratory timer.</p>
                  </div>
                  <div class="sect4" lang="en">
                     <div class="titlepage">
                        <div>
                           <div>
                              <h5 class="title"><a name="sec10_4_19_2"></a>10.4.19.2.&nbsp;Method
                              </h5>
                           </div>
                        </div>
                     </div>
                     <p>Prepare the plates: OP50 is spotted onto plates to make a 1-2 cm bacterial lawn and plates are stored at <span class="symbol">&#8764;</span>22&deg;C for at least 2 days until used for assay. Plates older then 7 days tend to cause aberrant mating behavior.
                     </p>
                     <p>Males to be tested are picked 18-24 hrs before assay at the L4 stage and moved to fresh plates in groups of 10-20 before observation.</p>
                     <p>For sperm transfer assays a virgin adult male (18-24 hrs post L4 lethargus) is placed on the 1-2 cm bacterial lawn with multiple
                        <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=unc-31" target="_blank">unc-31</a></em></span> hermaphrodite adults (approximately 50). The hermaphrodites used are between 3 and 4 days post L4, as these are partners
                        with which spicule insertion occurs almost instantaneously and therefore facilitates the analysis of sperm transfer. It appears
                        to help if the staged hermaphrodites are place on the bacterial lawn 1 to 2 hours before assay. After tonic spicule insertion,
                        a stopwatch is started. Time points can be taken when the sperm are released into the hermaphrodite (release) and when the
                        spicules were retracted after sperm transfer (cessation).
                     </p>
                  </div>
               </div>
               <div class="sect3" lang="en">
                  <div class="titlepage">
                     <div>
                        <div>
                           <h4 class="title"><a name="sec10_4_20"></a>10.4.20.&nbsp;Acknowledgement
                           </h4>
                        </div>
                     </div>
                  </div>
                  <p>I thank the contributors listed above for their help in assembling this list of behavioral assays (Scott Emmons).</p>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec11"></a>11.&nbsp;Defecation
                     </h2>
                  </div>
               </div>
            </div>
            <div class="sect2" lang="en">
               <div class="titlepage">
                  <div>
                     <div>
                        <h3 class="title"><a name="sec11_1"></a>11.1.&nbsp;Assaying the defecation motor program (DMP)
                        </h3>
                     </div>
                  </div>
               </div>
               <p>Contributed by James Thomas, University of Washington, Seattle, WA, USA, March, 2005.</p>
               <div class="procedure">
                  <ol type="1">
                     <li>
                        <p>Pick one (or a few) young to mid adults to a fresh NG plate seeded with OP50. You can assay larvae and older adults as well,
                           but defecation periodicity may be more variable or harder to see. Males are quite difficult to assay.
                        </p>
                        <p>The assay plate should be relatively fresh with a lawn of bacteria that isn't too thick or old.</p>
                        <p>Only assay an animal that is on the lawn and feeding.</p>
                        <p>Avoid air currents, plate taps, or anything else that might disturb the worm, as these cause the cycle period to be more variable.
                           Temperature should ideally be 20&deg; though most aspects of the assay will be quite similar from about 20&deg; to 25&deg;.
                        </p>
                     </li>
                     <li>
                        <p>Let the worms settle down for at least 5 minutes. Picking the worms upsets them and they won't feed or defecate normally for
                           a minute or two. To be safe, wait at least 5 minutes.
                        </p>
                     </li>
                     <li>
                        <p>Observe a single animal closely at about 25X magnification using a high-quality dissecting microscope.</p>
                     </li>
                     <li>
                        <p>A typical assay goes for about 10 minutes and a DMP occurs every 45 to 50 seconds.</p>
                     </li>
                     <li>
                        <p>You have to watch continuously to be sure you don't miss anything (though if the assay is with the wild type under standard
                           conditions, you can relax a bit after each DMP since another one is unlikely to happen for the next 30 seconds or so).
                        </p>
                     </li>
                  </ol>
               </div>
               <div class="sidebar">
                  <div class="tip" style="margin-left: 0.5in; margin-right: 0.5in;">
                     <h3 class="title">Tips</h3>
                     <p>Each defecation in wild-type animals starts with a slow peristaltic contraction of the posterior body wall muscle (pBoc, posterior
                        body contraction), running anteriorly almost half way up the worm over a period of about _ second (it looks as if the tail
                        scrunches up). The pBoc is followed about a second later by relaxation of the same muscles. About 2 seconds after the pBoc
                        relaxation, specialized anal muscles contract rapidly and a plume-like slurry of feces appears at the anus (EMC, enteric muscle
                        contraction, or Exp, expulsion). The expulsion is preceded very slightly by a sharp contraction in the head of the animal
                        (aBoc, anterior body contraction), but this step is much harder to score reliably, and trying to observe it distracts from
                        detecting the expulsion. For most purposes, you should ignore the aBoc and focus on the pBoc and Exp steps.
                     </p>
                     <p>A movie of the DMP is available at: <a href="ftp://calliope.gs.washington.edu/movies/simpleDMP.avi" target="_top">ftp://calliope.gs.washington.edu/movies/simpleDMP.avi</a></p>
                     <p>The Etho program, which lets you collect timed events as keystrokes (and record and analyze them in a fairly flexible way),
                        is available for download at: <a href="http://calliope.gs.washington.edu/software/otherSoftware.html" target="_blank">http://calliope.gs.washington.edu/software/other Software.html</a>.
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec12"></a>12.&nbsp;Galvanotaxis
                     </h2>
                  </div>
               </div>
            </div>
            <p><span class="emphasis"><em>C. elegans</em></span> responds to electrical fields. This response is under active investigation but no protocols have been published.
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec13"></a>13.&nbsp;Circadian rhythms
                     </h2>
                  </div>
               </div>
            </div>
            <p>As <span class="emphasis"><em>C. elegans</em></span> normally dwell in the soil, their lack of circadian rhythms is not surprising. However, <span class="emphasis"><em>C. elegans</em></span> homologs of <span class="emphasis"><em>Per, Tim</em></span> and other proteins involved in circadian rhythms in other species are coordinately expressed with the <span class="emphasis"><em>C. elegans</em></span> larval molts suggesting possible related function (<a href="#bib4">Banerjee et al., 2005</a>; <a href="#bib49">Jeon et al., 1999</a>).
            </p>
         </div>
         <div class="sect1" lang="en">
            <div class="titlepage">
               <div>
                  <div>
                     <h2 class="title" style="clear: both"><a name="sec14"></a>14.&nbsp;References
                     </h2>
                  </div>
               </div>
            </div>
            <div class="bibliography">
               <div class="bibliomixed"><a name="bib1"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Arena, J.P., Liu, K.K., Paress, P.S., Frazier, E.G., Cully, D.F., Mrozik, H., and Schaeffer, J.M. (1995). The mechanism of
                        action of avermectins in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: correlation between activation of glutamate-sensitive chloride current, membrane binding, and biological activity. J. Parasitol.
                        <span class="emphasis"><em>81</em></span>, 286&#8211;294.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7707209&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib2"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Aroian, R.V., and Sternberg, P.W. (1991). Multiple functions of <a href="http://www.wormbase.org/db/gene/gene?name=let-23" target="_blank">let-23</a>, a <span class="emphasis"><em>Caenorhabditis elegans</em></span> receptor tyrosine kinase gene required for vulval induction. Genetics <span class="emphasis"><em>128</em></span>, 251&#8211;267.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib3"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Baek, J.H., Cosman, P., Feng, Z., Silver, J., and Schafer, W.R. (2002). Using machine vision to analyze and classify <span class="emphasis"><em>Caenorhabditis elegans</em></span> behavioral phenotypes quantitatively. J. Neurosci. Methods <span class="emphasis"><em>118</em></span>, 9&#8211;21.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12191753&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0165-0270(02)00117-6" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib4"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Banerjee, D., Kwok, A., Lin, S.Y., and Slack, F.J. (2005). Developmental timing in <span class="emphasis"><em>C. elegans</em></span> is regulated by <a href="http://www.wormbase.org/db/gene/gene?name=kin-20" target="_blank">kin-20</a> and <a href="http://www.wormbase.org/db/gene/gene?name=tim-1" target="_blank">tim-1</a>, homologs of core circadian clock genes. Dev. Cell <span class="emphasis"><em>8</em></span>, 287&#8211;295.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15691769&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.devcel.2004.12.006" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib5"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bargmann, C.I., Hartwieg, E., and Horvitz, H.R. (1993). Odorant-selective genes and neurons mediate olfaction in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>74</em></span>, 515&#8211;527.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8348618&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0092-8674(93)80053-H" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib6"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Barker, D.M. (1994a). Copulatory plugs and paternity assurance in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Animal Behavior <span class="emphasis"><em>48</em></span>, 147&#8211;156.</span>
                     <a href="http://dx.doi.org/10.1006/anbe.1994.1221" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib7"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Barker, D.M. (1994b). Copulatory plugs and paternity assurance in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Animal Behaviour <span class="emphasis"><em>48</em></span>, 147&#8211;156.</span>
                     <a href="http://dx.doi.org/10.1006/anbe.1994.1221" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib8"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Barr, M.M., DeModena, J., Braun, D., Nguyen, C.Q., Hall, D.H., and Sternberg, P.W. (2001). The <span class="emphasis"><em>Caenorhabditis elegans</em></span> autosomal dominant polycystic kidney disease gene homologs <a href="http://www.wormbase.org/db/gene/gene?name=lov-1" target="_blank">lov-1</a> and <a href="http://www.wormbase.org/db/gene/gene?name=pkd-2" target="_blank">pkd-2</a> act in the same pathway. Curr. Biol. <span class="emphasis"><em>11</em></span>, 1341&#8211;1346.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11553327&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(01)00423-7" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib9"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Barr, M.M., and Sternberg, P.W. (1999). A polycystic kidney-disease gene homologue required for male mating behaviour in <span class="emphasis"><em>C. elegans</em></span>. Nature <span class="emphasis"><em>401</em></span>, 386&#8211;389.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10517638&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/43916" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib10"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Bettinger, J.C., and McIntire, S.L. (2004). State-dependency in <span class="emphasis"><em>C. elegans</em></span>. Genes Brain Behav. <span class="emphasis"><em>3</em></span>, 266&#8211;272.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15344920&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1111/j.1601-183X.2004.00080.x" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib11"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Brenner, S. (1974). The genetics of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>77</em></span>, 71&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4366476&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib12"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Brockie, P.J., Mellem, J.E., Hills, T., Madsen, D.M., and Maricq, A.V. (2001). The <span class="emphasis"><em>C. elegans</em></span> glutamate receptor subunit <a href="http://www.wormbase.org/db/seq/protein?name=NMR-1;class=protein" target="_blank">NMR-1</a> is required for slow NMDA-activated currents that regulate reversal frequency during locomotion. Neuron <span class="emphasis"><em>31</em></span>, 617&#8211;630.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11545720&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(01)00394-4" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib13"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chalfie, M., and Au, M. (1989). Genetic control of differentiation of the <span class="emphasis"><em>C. elegans</em></span> touch receptor neurons. Science <span class="emphasis"><em>243</em></span>, 1027&#8211;1033.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2646709&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib14"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chalfie, M., and Sulston, J. (1981). Developmental genetics of the mechanosensory neurons of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>82</em></span>, 358&#8211;370.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7227647&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0012-1606(81)90459-0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib15"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chalfie, M., Sulston, J.E., White, J.G., Southgate, E., Thomson, J.N., and Brenner, S. (1985). The neural circuit for touch
                        sensitivity in <span class="emphasis"><em>C. elegans</em></span>. J. Neurosci. <span class="emphasis"><em>5</em></span>, 956&#8211;964.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3981252&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib16"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chalfie, M., and Wolinsky, E. (1990). The identification and suppression of inherited neurodegeneration in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>345</em></span>, 410&#8211;416.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2342572&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/345410a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib17"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chao, M.Y., Komatsu, H., Fukuto, H.S., Dionne, H.M., and Hart, A.C. (2004). Feeding status and serotonin rapidly and reversibly
                        modulate a <span class="emphasis"><em>Caenorhabditis elegans</em></span> chemosensory circuit. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>101</em></span>, 15512&#8211;15517.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15492222&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.0403369101" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib18"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Chiba, C.M., and Rankin, C.H. (1990). A developmental analysis of spontaneous and reflexive reversals in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Neurobiol. <span class="emphasis"><em>21</em></span>, 543&#8211;554.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2376729&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/neu.480210403" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib19"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Colbert, H.A., and Bargmann, C.I. (1995). Odorant-specific adaptation pathways generate olfactory plasticity in <span class="emphasis"><em>C. elegans</em></span>. Neuron <span class="emphasis"><em>14</em></span>, 803&#8211;812.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7718242&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0896-6273(95)90224-4" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib20"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Croll, N.A. (1975). Components and patterns in the behavior of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Zool. <span class="emphasis"><em>176</em></span>, 159&#8211;176.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib21"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Croll, N.A., and Matthews, B.E. (1977). Biology of Nematodes (New York: John Wiley).</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib22"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Cully, D.F., Vassilatis, D.K., Liu, K.K., Paress, P.S., Van der Ploeg, L.H., Schaeffer, J.M., and Arena, J.P. (1994). Cloning
                        of an avermectin-sensitive glutamate-gated chloride channel from <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>371</em></span>, 707&#8211;711.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7935817&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/371707a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib23"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Culotti, J.G., and Russell, R.L. (1978). Osmotic avoidance defective mutants of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>90</em></span>, 243&#8211;256.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=730048&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib24"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Davies, A.G., Bettinger, J.C., Thiele, T.R., Judy, M.E., and McIntire, S.L. (2004). Natural variation in the <a href="http://www.wormbase.org/db/gene/gene?name=npr-1" target="_blank">npr-1</a> gene modifies ethanol responses of wild strains of <span class="emphasis"><em>C. elegans</em></span>. Neuron <span class="emphasis"><em>42</em></span>, 731&#8211;743.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15182714&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.neuron.2004.05.004" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib25"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Davies, A.G., and McIntire, S.L. (2004). Using <span class="emphasis"><em>C. elegans</em></span> to screen for targets of ethanol and behavior-altering drugs. Biol. Proced. Online <span class="emphasis"><em>6</em></span>, 113&#8211;119.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15192754&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=15192754" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib26"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Davies, A.G., Pierce-Shimomura, J.T., Kim, H., VanHoven, M.K., Thiele, T.R., Bonci, A., Bargmann, C.I., and McIntire, S.L.
                        (2003). A central role of the BK potassium channel in behavioral responses to ethanol in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>115</em></span>, 655&#8211;666.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14675531&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(03)00979-6" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib27"></a><p class="bibliomixed">
                     		<span class="bibliomisc">de Bono, M., and Bargmann, C.I. (1998). Natural variation in a neuropeptide Y receptor homolog modifies social behavior and
                        food response in <span class="emphasis"><em>C. elegans</em></span>. Cell <span class="emphasis"><em>94</em></span>, 679&#8211;689.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9741632&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(00)81609-8" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib28"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Dent, J.A., Smith, M.M., Vassilatis, D.K., and Avery, L. (2000). The genetics of ivermectin resistance in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>97</em></span>, 2674&#8211;2679.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10716995&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.97.6.2674" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib29"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Desai, C., and Horvitz, H.R. (1989). <span class="emphasis"><em>Caenorhabditis elegans</em></span> mutants defective in the functioning of the motor neurons responsible for egg-laying. Genetics <span class="emphasis"><em>121</em></span>, 703&#8211;721.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2721931&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib30"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Dusenbery, D.B., Sheridan, R.E., and Russell, R.L. (1975). Chemotaxis-defective mutants of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>80</em></span>, 297&#8211;309.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1132687&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib31"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ellerbrock, B.R., Coscarelli, E.M., Gurney, M.E., and Geary, T.G. (2004). Screening for presenilin inhibitors using the free-living
                        nematode, <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Biomol. Screen. <span class="emphasis"><em>9</em></span>, 147&#8211;152.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15006138&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1177/1087057103261038" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib32"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Emmons, S.W., and Sternberg, P.W. (1997). Male development and mating behavior. In <span class="emphasis"><em>C. elegans</em></span> II, D.L. Riddle, T. Blumenthal, B.J. Meyer, and J.R. Priess, eds. (Cold Spring Harbor, NY: Cold Spring Harbor Laboratory
                        Press), pp. 295&#8211;334.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib33"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Feng, Z., Cronin, C.J., Wittig, J.H., Jr., Sternberg, P.W., and Schafer, W.R. (2004). An imaging system for standardized quantitative
                        analysis of <span class="emphasis"><em>C. elegans</em></span> behavior. BMC Bioinformatics <span class="emphasis"><em>5</em></span>, 115.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15331023&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1186/1471-2105-5-115" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib34"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ferguson, E.L., and Horvitz, H.R. (1985). Identification and characterization of 22 genes that affect the vulval cell lineages
                        of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>110</em></span>, 17&#8211;72.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=3996896&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib35"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Geng, W., Cosman, P., Palm, M., and Schafer, W. (2005). <span class="emphasis"><em>C. elegans</em></span> egg-laying detection and behavior study using image analysis. EURASIP J. App. Sig. Proc. in press.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib36"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Gray, J.M., Karow, D.S., Lu, H., Chang, A.J., Chang, J.S., Ellis, R.E., Marletta, M.A., and Bargmann, C.I. (2004). Oxygen
                        sensation and social feeding mediated by a <span class="emphasis"><em>C. elegans</em></span> guanylate cyclase homologue. Nature <span class="emphasis"><em>430</em></span>, 317&#8211;322.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15220933&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/nature02714" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib37"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hardaker, L.A., Singer, E., Kerr, R., Zhou, G., and Schafer, W.R. (2001). Serotonin modulates locomotory behavior and coordinates
                        egg-laying and movement in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Neurobiol. <span class="emphasis"><em>49</em></span>, 303&#8211;313.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11745666&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/neu.10014" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib38"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hart, A.C., Kass, J., Shapiro, J.E., and Kaplan, J.M. (1999). Distinct signaling pathways mediate touch and osmosensory responses
                        in a polymodal sensory neuron. J. Neurosci. <span class="emphasis"><em>19</em></span>, 1952&#8211;1958.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10066248&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib39"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hart, A.C., Sims, S., and Kaplan, J.M. (1995). Synaptic code for sensory modalities revealed by <span class="emphasis"><em>C. elegans</em></span> <a href="http://www.wormbase.org/db/seq/protein?name=GLR-1;class=protein" target="_blank">GLR-1</a> glutamate receptor. Nature <span class="emphasis"><em>378</em></span>, 82&#8211;85.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7477294&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/378082a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib40"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hedgecock, E.M., and Russell, R.L. (1975). Normal and mutant thermotaxis in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>72</em></span>, 4061&#8211;4065.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1060088&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib41"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hilliard, M.A., Bargmann, C.I., and Bazzicalupo, P. (2002). <span class="emphasis"><em>C. elegans</em></span> responds to chemical repellents by integrating sensory inputs from the head and the tail. Curr. Biol. <span class="emphasis"><em>12</em></span>, 730&#8211;734.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12007416&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0960-9822(02)00813-8" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib42"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hilliard, M.A., Bergamasco, C., Arbucci, S., Plasterk, R.H., and Bazzicalupo, P. (2004). Worms taste bitter: ASH neurons,
                        <a href="http://www.wormbase.org/db/seq/protein?name=QUI-1;class=protein" target="_blank">QUI-1</a>, <a href="http://www.wormbase.org/db/seq/protein?name=GPA-3;class=protein" target="_blank">GPA-3</a> and <a href="http://www.wormbase.org/db/seq/protein?name=ODR-3;class=protein" target="_blank">ODR-3</a> mediate quinine avoidance in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. EMBO J. <span class="emphasis"><em>23</em></span>, 1101&#8211;1111.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14988722&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/sj.emboj.7600107" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib93"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hobert, O., Moerman, D.G., Clark, K.A., Beckerle, M.C., and Ruvkun, G. (1999). A conserved LIM protein that affects muscular
                        adherens junction integrity and mechanosensory function in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Cell Biol. <span class="emphasis"><em>144</em></span>, 45&#8211;57.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9885243&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1083/jcb.144.1.45" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib43"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J. (1983). Male phenotypes and mating efficiency in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>103</em></span>, 43&#8211;64.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib44"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J., and Doniach, T. (1997). Natural variation and copulatory plug formation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>146</em></span>, 149&#8211;164.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9136008&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib45"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J., Edgley, M., Riddle, D.L., and Albertson, D.G. (1988). Genetics. In: <span class="emphasis"><em>The Nematode C. elegans</em></span>, W.B. Wood, ed.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib46"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Hodgkin, J., Horvitz, H.R., and Brenner, S. (1079). Nondisjunction mutants of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>91</em></span>, 67&#8211;94.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2071015&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib47"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ishihara, T., Iino, Y., Mohri, A., Mori, I., Gengyo-Ando, K., Mitani, S., and Katsura, I. (2002). <a href="http://www.wormbase.org/db/seq/protein?name=HEN-1;class=protein" target="_blank">HEN-1</a>, a secretory protein with an LDL receptor motif, regulates sensory integration and learning in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Cell <span class="emphasis"><em>109</em></span>, 639&#8211;649.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12062106&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0092-8674(02)00748-1" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib48"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jansen, G., Weinkove, D., and Plasterk, R.H. (2002). The G-protein gamma subunit <a href="http://www.wormbase.org/db/gene/gene?name=gpc-1" target="_blank">gpc-1</a> of the nematode <span class="emphasis"><em>C. elegans</em></span> is involved in taste adaptation. EMBO J. <span class="emphasis"><em>21</em></span>, 986&#8211;994.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11867526&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1093/emboj/21.5.986" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib49"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Jeon, M., Gardner, H.F., Miller, E.A., Deshler, J., and Rougvie, A.E. (1999). Similarity of the <span class="emphasis"><em>C. elegans</em></span> developmental timing protein <a href="http://www.wormbase.org/db/seq/protein?name=LIN-42;class=protein" target="_blank">LIN-42</a> to circadian rhythm proteins. Science <span class="emphasis"><em>286</em></span>, 1141&#8211;1146.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10550049&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1126/science.286.5442.1141" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib50"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kahn-Kirby, A.H., Dantzker, J.L., Apicella, A.J., Schafer, W.R., Browse, J., Bargmann, C.I., and Watts, J.L. (2004). Specific
                        polyunsaturated fatty acids drive TRPV-dependent sensory signaling <span class="emphasis"><em>in vivo</em></span>. Cell <span class="emphasis"><em>119</em></span>, 889&#8211;900.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15607983&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cell.2004.11.005" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib51"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kaplan, J.M., and Horvitz, H.R. (1993). A dual mechanosensory and chemosensory neuron in <span class="emphasis"><em>C. elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>90</em></span>, 2227&#8211;2231.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8460126&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib52"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Kleemann, G. (2005) Mating behavior and intersexual mating interactions in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Lincoln, NB: University of Nebraska.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib53"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Law, E., Nuttley, W.M., and van der Kooy, D. (2004). Contextual taste cues modulate olfactory learning in <span class="emphasis"><em>C. elegans</em></span> by an occasion-setting mechanism. Curr. Biol. <span class="emphasis"><em>14</em></span>, 1303&#8211;1308.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15268863&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.06.066" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib54"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lee, R.Y., Sawin, E.R., Chalfie, M., Horvitz, H.R., and Avery, L. (1999). <a href="http://www.wormbase.org/db/seq/protein?name=EAT-4;class=protein" target="_blank">EAT-4</a>, a homolog of a mammalian sodium-dependent inorganic phosphate cotransporter, is necessary for glutamatergic neurotransmission
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Neurosci. <span class="emphasis"><em>19</em></span>, 159&#8211;167.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9870947&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib55"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Lipton, J., Kleemann, G., Ghosh, R., Lints, R., and Emmons, S.W. (2004). Mate searching in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: a genetic model for sex drive in a simple invertebrate. J. Neurosci. <span class="emphasis"><em>24</em></span>, 7427&#8211;7434.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15329389&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1523/JNEUROSCI.1746-04.2004" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib56"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Liu, K.S., and Sternberg, P.W. (1995). Sensory regulation of male mating behavior in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Neuron <span class="emphasis"><em>14</em></span>, 79&#8211;89.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7826644&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0896-6273(95)90242-2" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib57"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Loer, C.M., and Kenyon, C.J. (1993). Serotonin-deficient mutants and male mating behavior in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Neurosci. <span class="emphasis"><em>13</em></span>, 5407&#8211;5417.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8254383&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib58"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mellem, J.E., Brockie, P.J., Zheng, Y., Madsen, D.M., and Maricq, A.V. (2002). Decoding of polymodal sensory stimuli by postsynaptic
                        glutamate receptors in <span class="emphasis"><em>C. elegans</em></span>. Neuron <span class="emphasis"><em>36</em></span>, 933&#8211;944.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12467596&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(02)01088-7" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib59"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Mori, I., and Ohshima, Y. (1995). Neural regulation of thermotaxis in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Nature <span class="emphasis"><em>376</em></span>, 344&#8211;348.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7630402&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1038/376344a0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib92"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Nuttley WM, Atkinson-Leadbeater KP, Van Der Kooy D. (2002). Serotonin mediates food-odor associative learning in the nematode
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>99</em></span>, 12449&#8211;12454.</span>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib60"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Peden, E.M., and Barr, M.M. (2005). The <a href="http://www.wormbase.org/db/seq/protein?name=KLP-6;class=protein" target="_blank">KLP-6</a> kinesin is required for male mating behaviors and polycystin localization in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Curr. Biol. <span class="emphasis"><em>15</em></span>, 394&#8211;404.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15753033&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.12.073" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib61"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Pierce-Shimomura, J.T., Morse, T.M., and Lockery, S.R. (1999). The fundamental role of pirouettes in <span class="emphasis"><em>Caenorhabditis elegans</em></span> chemotaxis. J. Neurosci. <span class="emphasis"><em>19</em></span>, 9557&#8211;9569.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10531458&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib62"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Raizen, D.M., and Avery, L. (1994). Electrical activity and behavior in the pharynx of <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Neuron <span class="emphasis"><em>12</em></span>, 483&#8211;495.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8155316&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0896-6273(94)90207-0" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib63"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Raizen, D.M., Lee, R.Y., and Avery, L. (1995). Interacting genes required for pharyngeal excitation by motor neuron MC in
                        <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>141</em></span>, 1365&#8211;1382.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8601480&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib64"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rankin, C.H. (1991). Interactions between two antagonistic reflexes in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Comp. Physiol. [A] <span class="emphasis"><em>169</em></span>, 59&#8211;67.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1941719&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib65"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rankin, C.H., Beck, C.D., and Chiba, C.M. (1990). <span class="emphasis"><em>Caenorhabditis elegans</em></span>: a new model system for the study of learning and memory. Behav. Brain Res. <span class="emphasis"><em>37</em></span>, 89&#8211;92.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2310497&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/0166-4328(90)90074-O" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib66"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rankin, C.H., and Broster, B.S. (1992). Factors affecting habituation and recovery from habituation in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Behav. Neurosci. <span class="emphasis"><em>106</em></span>, 239&#8211;249.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1590951&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1037/0735-7044.106.2.239" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib67"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rose, J.K., Kaun, K.R., and Rankin, C.H. (2002). A new group-training procedure for habituation demonstrates that presynaptic
                        glutamate release contributes to long-term memory in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Learn. Mem. <span class="emphasis"><em>9</em></span>, 130&#8211;137.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12075001&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/lm.46802" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib68"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Rose, J.K., and Rankin, C.H. (2001). Analyses of habituation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Learn. Mem. <span class="emphasis"><em>8</em></span>, 63&#8211;69.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11274251&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1101/lm.37801" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib69"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ryu, W.S., and Samuel, A.D. (2002). Thermotaxis in <span class="emphasis"><em>Caenorhabditis elegans</em></span> analyzed by measuring responses to defined Thermal stimuli. J. Neurosci. <span class="emphasis"><em>22</em></span>, 5727&#8211;5733.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12097525&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/20026542" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib70"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Saeki, S., Yamamoto, M., and Iino, Y. (2001). Plasticity of chemotaxis revealed by paired presentation of a chemoattractant
                        and starvation in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Exp. Biol. <span class="emphasis"><em>204</em></span>, 1757&#8211;1764.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11316496&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib71"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Satterlee, J.S., Ryu, W.S., and Sengupta, P. (2004). The <a href="http://www.wormbase.org/db/seq/protein?name=CMK-1;class=protein" target="_blank">CMK-1</a> CaMKI and the <a href="http://www.wormbase.org/db/seq/protein?name=TAX-4;class=protein" target="_blank">TAX-4</a> Cyclic nucleotide-gated channel regulate thermosensory neuron gene expression and function in <span class="emphasis"><em>C. elegans</em></span>. Curr. Biol. <span class="emphasis"><em>14</em></span>, 62&#8211;68.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=14711416&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2003.12.030" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib72"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sawin, E.R., Ranganathan, R., and Horvitz, H.R. (2000). <span class="emphasis"><em>C. elegans</em></span> locomotory rate is modulated by the environment through a dopaminergic pathway and by experience through a serotonergic pathway.
                        Neuron <span class="emphasis"><em>26</em></span>, 619&#8211;631.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10896158&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(00)81199-X" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib73"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Simon, J.M., and Sternberg, P.W. (2002). Evidence of a mate-finding cue in the hermaphrodite nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>99</em></span>, 1598&#8211;1603.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11818544&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.032225799" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib74"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Sulston, J., Dew, M., and Brenner, S. (1975). Dopaminergic neurons in the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Comp. Neurol. <span class="emphasis"><em>163</em></span>, 215&#8211;226.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=240872&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1002/cne.901630207" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib75"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Suzuki, H., Kerr, R., Bianchi, L., Frokjaer-Jensen, C., Slone, D., Xue, J., Gerstbrein, B., Driscoll, M., and Schafer, W.R.
                        (2003). <span class="emphasis"><em>In vivo</em></span> imaging of <span class="emphasis"><em>C. elegans</em></span> mechanosensory neurons demonstrates a specific role for the <a href="http://www.wormbase.org/db/seq/protein?name=MEC-4;class=protein" target="_blank">MEC-4</a> channel in the process of gentle touch sensation. Neuron <span class="emphasis"><em>39</em></span>, 1005&#8211;1017.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12971899&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.neuron.2003.08.015" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib76"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Thomas, J.H. (1990). Genetic analysis of defecation in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>124</em></span>, 855&#8211;872.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2323555&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib77"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Trent, C., Tsuing, N., and Horvitz, H.R. (1983). Egg-laying defective mutants of the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Genetics <span class="emphasis"><em>104</em></span>, 619&#8211;647.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11813735&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib78"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Troemel, E.R. (1999). Chemosensory signaling in <span class="emphasis"><em>C. elegans</em></span>. Bioessays <span class="emphasis"><em>21</em></span>, 1011&#8211;1020.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10580986&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib79"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Uchida, O., Nakano, H., Koga, M., and Ohshima, Y. (2003). The <span class="emphasis"><em>C. elegans</em></span> <a href="http://www.wormbase.org/db/gene/gene?name=che-1" target="_blank">che-1</a> gene encodes a zinc finger transcription factor required for specification of the ASE chemosensory neurons. Development <span class="emphasis"><em>130</em></span>, 1215&#8211;1224.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12588839&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib80"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Voisine, C., and Hart, A.C. (2004). <span class="emphasis"><em>Caenorhabditis elegans</em></span> as a model system for triplet repeat diseases. Methods Mol. Biol. <span class="emphasis"><em>277</em></span>, 141&#8211;160.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15201454&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib81"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Waggoner, L.E., Dickinson, K.A., Poole, D.S., Tabuse, Y., Miwa, J., and Schafer, W.R. (2000). Long-term nicotine adaptation
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span> involves PKC-dependent changes in nicotinic receptor abundance. J. Neurosci. <span class="emphasis"><em>20</em></span>, 8802&#8211;8811.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11102488&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib82"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Waggoner, L.E., Zhou, G.T., Schafer, R.W., and Schafer, W.R. (1998). Control of alternative behavioral states by serotonin
                        in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Neuron <span class="emphasis"><em>21</em></span>, 203&#8211;214.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=9697864&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(00)80527-9" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib83"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Ward, S. (1973). Chemotaxis by the nematode <span class="emphasis"><em>Caenorhabditis elegans</em></span>: identification of attractants and analysis of the response by use of mutants. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>70</em></span>, 817&#8211;821.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4351805&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib84"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Way, J.C., and Chalfie, M. (1989). The <span class="emphasis"><em><a href="http://www.wormbase.org/db/gene/gene?name=mec-3" target="_blank">mec-3</a></em></span> gene of <span class="emphasis"><em>Caenorhabditis elegans</em></span> requires its own product for maintained expression and is expressed in three neuronal cell types. Genes Dev. <span class="emphasis"><em>3</em></span>, 1823&#8211;1833.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=2576011&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib85"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Weinshenker, D., Garriga, G., and Thomas, J.H. (1995). Genetic and pharmacological analysis of neurotransmitters controlling
                        egg-laying in <span class="emphasis"><em>C. elegans</em></span>. J. Neurosci. <span class="emphasis"><em>15</em></span>, 6975&#8211;6985.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7472454&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib86"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wicks, S.R., de Vries, C.J., van Luenen, H.G., and Plasterk, R.H. (2000). <a href="http://www.wormbase.org/db/seq/protein?name=CHE-3;class=protein" target="_blank">CHE-3</a>, a cytosolic dynein heavy chain, is required for sensory cilia structure and function in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. Dev. Biol. <span class="emphasis"><em>221</em></span>, 295&#8211;307.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10790327&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1006/dbio.2000.9686" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib87"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wicks, S.R., and Rankin, C.H. (1995). Integration of mechanosensory stimuli in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. J. Neurosci. <span class="emphasis"><em>15</em></span>, 2434&#8211;2444.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7891178&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib88"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Wittenburg, N., and Baumeister, R. (1999). Thermal avoidance in <span class="emphasis"><em>Caenorhabditis elegans</em></span>: an approach to the study of nociception. Proc. Natl. Acad. Sci. USA <span class="emphasis"><em>96</em></span>, 10477&#8211;10482.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10468634&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1073/pnas.96.18.10477" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib94"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Zhang, S., Arnadottir, J., Keller, C., Caldwell, G.A., Yao, C.A., and Chalfie, M. (2004). MEC-2 is recruited to the putative
                        mechanosensory complex in <span class="emphasis"><em>C. elegans</em></span> touch receptor neurons through its stomatin-like domain. Curr. Biol. <span class="emphasis"><em>14</em></span>, 1888&#8211;1896.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=15530389&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/j.cub.2004.10.030" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib95"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Zhang, Y., and Chalfie, M. (2002). MTD-1, a touch-cell-specific membrane protein with a subtle effect on touch sensitivity.
                        Mech. Dev. <span class="emphasis"><em>119</em></span>, 3&#8211;7.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12385749&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0925-4773(02)00293-9" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib89"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Zhao, B., Khare, P., Feldman, L., and Dent, J.A. (2003). Reversal frequency in <span class="emphasis"><em>Caenorhabditis elegans</em></span> represents an integrated response to the state of the animal and its environment. J. Neurosci. <span class="emphasis"><em>23</em></span>, 5319&#8211;5328.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=12832557&amp;dopt=Abstract" target="_blank">Abstract</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib90"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Zheng, Y., Brockie, P.J., Mellem, J.E., Madsen, D.M., and Maricq, A.V. (1999). Neuronal control of locomotion in <span class="emphasis"><em>C. elegans</em></span> is modified by a dominant mutation in the <a href="http://www.wormbase.org/db/seq/protein?name=GLR-1;class=protein" target="_blank">GLR-1</a> ionotropic glutamate receptor. Neuron <span class="emphasis"><em>24</em></span>, 347&#8211;361.</span>
                     <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=10571229&amp;dopt=Abstract" target="_blank">Abstract</a>
                     <a href="http://dx.doi.org/10.1016/S0896-6273(00)80849-1" target="_blank">Article</a>
                     	
                  </p>
               </div>
               <div class="bibliomixed"><a name="bib91"></a><p class="bibliomixed">
                     		<span class="bibliomisc">Zhou, G.T., Schafer, W.R., and Schafer, R.W. (1998). Control of alternative behavioral states by serotonin in <span class="emphasis"><em>Caenorhabditis elegans</em></span>. IEEE Trans Signal Process 46.</span>
                     	
                  </p>
               </div>
            </div>
            <p></p>
         </div>
         <div class="footnotes"><br><hr width="100" align="left">
            <div class="footnote">
               <p><sup><a name="ftn.d0e8" href="#d0e8">*</a></sup>Edited by Victor Ambros. Last revised June 02, 2005. Published July 3, 2006. This chapter should be cited as: Hart, Anne C.,
                  ed. Behavior (July 3, 2006), <span class="emphasis"><em>WormBook</em></span>, ed. The <span class="emphasis"><em>C. elegans</em></span> Research Community, WormBook, doi/10.1895/wormbook.1.87.1, <a href="http://www.wormbook.org" target="_top">http://www.wormbook.org</a>.
               </p>
               <p><span class="bold"><strong>Copyright:</strong></span> &copy; 2006 Anne C. Hart and contributors. This is an open-access article distributed under the terms of the Creative Commons
                  Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author
                  and source are credited.
               </p>
            </div>
            <div class="footnote">
               <p><sup><a name="ftn.d0e33" href="#d0e33">&sect;</a></sup>To whom correspondence should be addressed. E-mail: <a href="mailto:hart@helix.mgh.harvard.edu" target="_top">hart@helix.mgh.harvard.edu</a></p>
			   <p><img src="somerights20.gif" alt="Creative Commons License" align="middle" border="0"> All WormBook content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License" target="_blank">Creative Commons Attribution License</a>.</p>
            </div>
         </div>
      </div>
   <!--#include virtual="/ssi/footer.html" --></body>
</html>

